<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/911" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/911.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/911.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/911.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/911.atom" />
<meta name="HW.pisa" content="ggg;5/5/911" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" />
<meta name="DC.Identifier" content="10.1534/g3.115.017178" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2015 Andersen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genetic variants underlying complex traits are often elusive even in powerful model organisms such as Caenorhabditis elegans with controlled genetic backgrounds and environmental conditions. Two major contributing factors are: (1) the lack of statistical power from measuring the phenotypes of small numbers of individuals, and (2) the use of phenotyping platforms that do not scale to hundreds of individuals and are prone to noisy measurements. Here, we generated a new resource of 359 recombinant inbred strains that augments the existing C. elegans N2xCB4856 recombinant inbred advanced intercross line population. This new strain collection removes variation in the neuropeptide receptor gene [npr-1][1] , known to have large physiological and behavioral effects on C. elegans and mitigates the hybrid strain incompatibility caused by [zeel-1][2] and [peel-1][3] , allowing for identification of quantitative trait loci that otherwise would have been masked by those effects. Additionally, we optimized highly scalable and accurate high-throughput assays of fecundity and body size using the COPAS BIOSORT large particle nematode sorter. Using these assays, we identified quantitative trait loci involved in fecundity and growth under normal growth conditions and after exposure to the herbicide paraquat, including independent genetic loci that regulate different stages of larval growth. Our results offer a powerful platform for the discovery of the genetic variants that control differences in responses to drugs, other aqueous compounds, bacterial foods, and pathogenic stresses.

 [1]: http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene
 [3]: http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene" />
<meta name="DC.Contributor" content="Erik C. Andersen" />
<meta name="DC.Contributor" content="Tyler C. Shimko" />
<meta name="DC.Contributor" content="Jonathan R. Crissman" />
<meta name="DC.Contributor" content="Rajarshi Ghosh" />
<meta name="DC.Contributor" content="Joshua S. Bloom" />
<meta name="DC.Contributor" content="Hannah S. Seidel" />
<meta name="DC.Contributor" content="Justin P. Gerke" />
<meta name="DC.Contributor" content="Leonid Kruglyak" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genetic variants underlying complex traits are often elusive even in powerful model organisms such as &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; with controlled genetic backgrounds and environmental conditions. Two major contributing factors are: (1) the lack of statistical power from measuring the phenotypes of small numbers of individuals, and (2) the use of phenotyping platforms that do not scale to hundreds of individuals and are prone to noisy measurements. Here, we generated a new resource of 359 recombinant inbred strains that augments the existing &lt;i&gt;C. elegans&lt;/i&gt; N2xCB4856 recombinant inbred advanced intercross line population. This new strain collection removes variation in the neuropeptide receptor gene &lt;i&gt;npr-1&lt;/i&gt;, known to have large physiological and behavioral effects on &lt;i&gt;C. elegans&lt;/i&gt; and mitigates the hybrid strain incompatibility caused by &lt;i&gt;zeel-1&lt;/i&gt; and &lt;i&gt;peel-1&lt;/i&gt;, allowing for identification of quantitative trait loci that otherwise would have been masked by those effects. Additionally, we optimized highly scalable and accurate high-throughput assays of fecundity and body size using the COPAS BIOSORT large particle nematode sorter. Using these assays, we identified quantitative trait loci involved in fecundity and growth under normal growth conditions and after exposure to the herbicide paraquat, including independent genetic loci that regulate different stages of larval growth. Our results offer a powerful platform for the discovery of the genetic variants that control differences in responses to drugs, other aqueous compounds, bacterial foods, and pathogenic stresses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/911" />
<meta name="citation_id" content="5/5/911" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/911" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/911.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/911.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/911.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017178" />
<meta name="citation_pmid" content="25770127" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="911" />
<meta name="citation_lastpage" content="920" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Erik C. Andersen" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_email" content="erik.andersen@northwestern.edu" />
<meta name="citation_author_email" content="lkruglyak@mednet.ucla.edu" />
<meta name="citation_author" content="Tyler C. Shimko" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author" content="Jonathan R. Crissman" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Rajarshi Ghosh" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Joshua S. Bloom" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095" />
<meta name="citation_author" content="Hannah S. Seidel" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Justin P. Gerke" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Leonid Kruglyak" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095" />
<meta name="citation_author_email" content="erik.andersen@northwestern.edu" />
<meta name="citation_author_email" content="lkruglyak@mednet.ucla.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. C. Andersen;citation_author=J. S. Bloom;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=A variant in the neuropeptide receptor npr-1 is a major determinant of Caenorhabditis elegans growth and physiology.;citation_pages=e1004156;citation_volume=10;citation_year=2014;citation_pmid=24586193;citation_doi=10.1371/journal.pgen.1004156" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Ayyadevara;citation_author=R. Ayyadevara;citation_author=S. Hou;citation_author=J. J. Thaden;citation_author=R. J. Shmookler Reis;citation_title=Genetic Mapping of Quantitative Trait Loci Governing Longevity of Caenorhabditis elegans in Recombinant-Inbred Progeny of a Bergerac-BO  RC301 Interstrain Cross;citation_pages=655-666;citation_volume=157;citation_year=2001;citation_issue=2;citation_pmid=11156986" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Ayyadevara;citation_author=R. Ayyadevara;citation_author=A. Vertino;citation_author=A. Galecki;citation_author=J. J. Thaden;citation_title=Genetic Loci Modulating Fitness and Life Span in Caenorhabditis elegans: Categorical Trait Interval Mapping in CL2a x Bergerac-BO Recombinant-Inbred Worms;citation_pages=557-570;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618395" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Bendesky;citation_author=M. Tsunozaki;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=C. I. Bargmann;citation_title=Catecholamine receptor polymorphisms affect decision-making in C. elegans.;citation_pages=313-318;citation_volume=472;citation_year=2011;citation_issue=7343;citation_pmid=21412235;citation_doi=10.1038/nature09821" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Bendesky;citation_author=J. Pitts;citation_author=M. V. Rockman;citation_author=W. C. Chen;citation_author=M.-W. Tan;citation_title=Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in Caenorhabditis elegans.;citation_pages=e1003157;citation_volume=8;citation_year=2012;citation_pmid=23284308;citation_doi=10.1371/journal.pgen.1003157" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. S. Bloom;citation_author=I. M. Ehrenreich;citation_author=W. T. Loo;citation_author=T.-L. V. Lite;citation_author=L. Kruglyak;citation_title=Finding the sources of missing heritability in a yeast cross.;citation_pages=234-237;citation_volume=494;citation_year=2013;citation_pmid=23376951;citation_doi=10.1038/nature11867" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=W. A. Boyd;citation_author=M. V. Smith;citation_author=G. E. Kissling;citation_author=J. R. Rice;citation_author=D. W. Snyder;citation_title=Application of a mathematical model to describe the effects of chlorpyrifos on Caenorhabditis elegans development.;citation_pages=e7024-e7024;citation_volume=4;citation_year=2009;citation_issue=9;citation_pmid=19753116;citation_doi=10.1371/journal.pone.0007024" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. B. Brem;citation_author=J. D. Storey;citation_author=J. Whittle;citation_author=L. Kruglyak;citation_title=Genetic interactions between polymorphisms that affect gene expression in yeast.;citation_pages=701-703;citation_volume=436;citation_year=2005;citation_issue=7051;citation_pmid=16079846;citation_doi=10.1038/nature03865" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Doroszuk;citation_author=L. B. Snoek;citation_author=E. Fradin;citation_author=J. Riksen;citation_author=J. Kammenga;citation_title=A genome-wide library of CB4856/N2 introgression lines of Caenorhabditis elegans.;citation_volume=37;citation_year=2009;citation_pmid=19542186;citation_doi=10.1093/nar/gkp528" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. E. Gaertner;citation_author=P. C. Phillips;citation_title=Caenorhabditis elegans as a platform for molecular quantitative genetics and the systems biology of natural variation.;citation_pages=331-348;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429266;citation_doi=10.1017/S0016672310000601" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Gaertner;citation_author=M. D. Parmenter;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=P. C. Phillips;citation_title=More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in Caenorhabditis elegans.;citation_volume=192;citation_year=2012;citation_pmid=23086219;citation_doi=10.1534/genetics.112.142877" />
<meta name="citation_reference" content="citation_journal_title=Science (New York, NY);citation_author=R. Ghosh;citation_author=E. C. Andersen;citation_author=J. A. Shapiro;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=Natural variation in a chloride channel subunit confers avermectin resistance in C. elegans.;citation_pages=574-578;citation_volume=335;citation_year=2012;citation_doi=10.1126/science.1214318" />
<meta name="citation_reference" content="citation_title=Multigenic natural variation underlies Caenorhabditis elegans olfactory preference for the bacterial pathogen Serratia marcescens.;citation_author=E. E. Glater;citation_author=M. V. Rockman;citation_author=C. I. Bargmann;citation_pages=265-276;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Free radical research;citation_journal_abbrev=Free Radic Res;citation_author=N. Ishii;citation_title=Oxidative stress and aging in Caenorhabditis elegans.;citation_pages=857-864;citation_volume=33;citation_year=2000;citation_issue=6;citation_pmid=11237107;citation_doi=10.1080/10715760000301371" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. E. Kammenga;citation_author=A. Doroszuk;citation_author=J. A. G. Riksen;citation_author=E. Hazendonk;citation_author=L. Spiridon;citation_title=A Caenorhabditis elegans wild type defies the temperature-size rule owing to a single nucleotide polymorphism in tra-3.;citation_pages=e34-e34;citation_volume=3;citation_year=2007;citation_issue=3;citation_pmid=17335351;citation_doi=10.1371/journal.pgen.0030034" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Li;citation_author=O. A. Alvarez;citation_author=E. W. Gutteling;citation_author=M. Tijsterman;citation_author=J. Fu;citation_title=Mapping determinants of gene expression plasticity by genetical genomics in C. elegans.;citation_pages=e222-e222;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17196041;citation_doi=10.1371/journal.pgen.0020222" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=P. T. McGrath;citation_author=M. V. Rockman;citation_author=M. Zimmer;citation_author=H. Jang;citation_author=E. Z. Macosko;citation_title=Quantitative mapping of a digenic behavioral trait implicates globin variation in C. elegans sensory behaviors.;citation_pages=692-699;citation_volume=61;citation_year=2009;citation_pmid=19285466;citation_doi=10.1016/j.neuron.2009.02.012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. F. Palopoli;citation_author=M. V. Rockman;citation_author=A. TinMaung;citation_author=C. Ramsay;citation_author=S. Curwen;citation_title=Molecular basis of the copulatory plug polymorphism in Caenorhabditis elegans.;citation_pages=1019-1022;citation_volume=454;citation_year=2008;citation_issue=7207;citation_pmid=18633349;citation_doi=10.1038/nature07171" />
<meta name="citation_reference" content="citation_title=Resistance to germline RNA interference in a Caenorhabditis elegans wild isolate exhibits complexity and nonadditivity.;citation_author=D. A. Pollard;citation_author=M. V. Rockman;citation_pages=941-947;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. C. Reddy;citation_author=E. C. Andersen;citation_author=L. Kruglyak;citation_author=D. H. Kim;citation_title=A Polymorphism in npr-1 Is a Behavioral Determinant of Pathogen Susceptibility in C. elegans;citation_pages=382-384;citation_volume=323;citation_year=2009;citation_issue=5912;citation_pmid=19150845;citation_doi=10.1126/science.1166527" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Breeding designs for recombinant inbred advanced intercross lines.;citation_volume=179;citation_year=2008;citation_pmid=18505881;citation_doi=10.1534/genetics.107.083873" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Recombinational landscape and population genomics of Caenorhabditis elegans.;citation_pages=e1000419-e1000419;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19283065;citation_doi=10.1371/journal.pgen.1000419" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. V. Rockman;citation_author=S. S. Skrovanek;citation_author=L. Kruglyak;citation_title=Selection at Linked Sites Shapes Heritable Phenotypic Variation in C. elegans;citation_pages=372-376;citation_volume=330;citation_year=2010;citation_issue=6002;citation_pmid=20947766;citation_doi=10.1126/science.1194208" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=H. S. Seidel;citation_author=M. Ailion;citation_author=J. Li;citation_author=A. van Oudenaarden;citation_author=M. V. Rockman;citation_title=A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in C. elegans.;citation_pages=e1001115-e1001115;citation_volume=9;citation_year=2011;citation_issue=7;citation_pmid=21814493;citation_doi=10.1371/journal.pbio.1001115" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. C. Shimko;citation_author=E. C. Andersen;citation_title=COPASutils: an R package for reading, processing, and visualizing data from COPAS large-particle flow cytometers.;citation_pages=e111090;citation_volume=9;citation_year=2014;citation_pmid=25329171;citation_doi=10.1371/journal.pone.0111090" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. V. Smith;citation_author=W. A. Boyd;citation_author=G. E. Kissling;citation_author=J. R. Rice;citation_author=D. W. Snyder;citation_title=A discrete time model for the analysis of medium-throughput C. elegans growth data.;citation_pages=e7018-e7018;citation_volume=4;citation_year=2009;citation_issue=9;citation_pmid=19753303;citation_doi=10.1371/journal.pone.0007018" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=A. Vertino;citation_author=S. Ayyadevara;citation_author=J. J. Thaden;citation_author=R. J. S. Reis;citation_title=A narrow quantitative trait locus in C. elegans coordinately affects longevity, thermotolerance, and resistance to paraquat.;citation_pages=63-63;citation_volume=2;citation_year=2011;citation_pmid=22303358" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="The genetic variants underlying complex traits are often elusive even in powerful model organisms such as Caenorhabditis elegans with controlled genetic backgrounds and environmental conditions. Two major contributing factors are: (1) the lack of statistical power from measuring the phenotypes of small numbers of individuals, and (2) the use of phenotyping platforms that do not scale to hundreds of individuals and are prone to noisy measurements. Here, we generated a new resource of 359 recombinant inbred strains that augments the existing C. elegans N2xCB4856 recombinant inbred advanced intercross line population. This new strain collection removes variation in the neuropeptide receptor gene [npr-1][1] , known to have large physiological and behavioral effects on C. elegans and mitigates the hybrid strain incompatibility caused by [zeel-1][2] and [peel-1][3] , allowing for identification of quantitative trait loci that otherwise would have been masked by those effects. Additionally, we optimized highly scalable and accurate high-throughput assays of fecundity and body size using the COPAS BIOSORT large particle nematode sorter. Using these assays, we identified quantitative trait loci involved in fecundity and growth under normal growth conditions and after exposure to the herbicide paraquat, including independent genetic loci that regulate different stages of larval growth. Our results offer a powerful platform for the discovery of the genetic variants that control differences in responses to drugs, other aqueous compounds, bacterial foods, and pathogenic stresses.

 [1]: http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene
 [3]: http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene" />
<meta name="og-title" property="og:title" content="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/911" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genetic variants underlying complex traits are often elusive even in powerful model organisms such as Caenorhabditis elegans with controlled genetic backgrounds and environmental conditions. Two major contributing factors are: (1) the lack of statistical power from measuring the phenotypes of small numbers of individuals, and (2) the use of phenotyping platforms that do not scale to hundreds of individuals and are prone to noisy measurements. Here, we generated a new resource of 359 recombinant inbred strains that augments the existing C. elegans N2xCB4856 recombinant inbred advanced intercross line population. This new strain collection removes variation in the neuropeptide receptor gene [npr-1][1] , known to have large physiological and behavioral effects on C. elegans and mitigates the hybrid strain incompatibility caused by [zeel-1][2] and [peel-1][3] , allowing for identification of quantitative trait loci that otherwise would have been masked by those effects. Additionally, we optimized highly scalable and accurate high-throughput assays of fecundity and body size using the COPAS BIOSORT large particle nematode sorter. Using these assays, we identified quantitative trait loci involved in fecundity and growth under normal growth conditions and after exposure to the herbicide paraquat, including independent genetic loci that regulate different stages of larval growth. Our results offer a powerful platform for the discovery of the genetic variants that control differences in responses to drugs, other aqueous compounds, bacterial foods, and pathogenic stresses.

 [1]: http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene
 [3]: http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xEBABNJ1De5x8g-32P8e-4BuVpBqj4zUZ49wY2hDtUE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459887","apath":"\/ggg\/5\/5\/911.atom","pisa":"ggg;5\/5\/911","ac":{"\/ggg\/5\/5\/911.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/911.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/911"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/911"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/911":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CZ93ELVoL+DYRkO41SqhxELI+P6MNcVEeHf9VOVlZNQP4g2b\\\\\\\/fx1KkpbRtOMY\\\\\\\/22YrKvlsDCVcVUvlG4Iix+00vlW\\\\\\\/lCQkYJxRrOkbWDqw++MFN2oEUSiSZgF4gTHPG\\\\\\\/orRRcmXv+i\\\\\\\/W772uPDipLy+9iEbVcZtV6RgMBR+P85ryhhm5HY4MIPYfF\\\\\\\/TJ\\\\\\\/iYpsV8\\\\\\\/px7ZWeLOX4GCcWX1pr9umnD0dnPYYLbjcb23Be93nQw7Qsrd\\\\\\\/En6atFVTS8cvM3iGEpTDIyxGVoleHEas\\\\\\\/aW+3fCt0xuckBSqRBo23VZALX9Mv+zbllrWO+UQw2Cs9nWRRUIlfILu6ZiW1V043Kr4oKeTTHU+jDLDJnsjODqVTqASf+tQvtvPwKwLiHDxWRrhYOZ+iFbiC4emM9SbpiWCa26Kn+2hMNK8xBl+zrKoICQ3am65+tjSj0S9AcnA7mFYRvexUQjPiCZpuBqLknN00CsoB05PKOCUy8YA2Kkzew25MZDHZMMjkvZOPHLKRBsefY1cFgYwHJahHkXoTkic5w8auafu6lAd+wtminkhH0QaSiGlEPTN7lgbnTm6ahO3g7rPENPcjLO8Ai6+Rm\\\\\\\/ViAGDONWnq0yvVq9RJ36c66BEUeqBn6ouEGGe0rAvMU0viWRFI1ObbD5OZ1EtGkuksQfOxAkGv\\\\\\\/cPPWLKnNgS+uoykER5KbrfBfsGPM00Kcl8s5I35DwiUUoLSjrTyd1a8k0I5rLMmgBMl1QLiu1Mki3W8G1YnO4nJYMgZMf78NQmt5y0OUNZ5Cegqs2e7z8jIqlIyE0SbGqQx0ybdCMbfYpM7A5VDru8e4+NXSaNJFHxEHE15HFyExEsyzKMeAg4A\\\\\\\/39IRtNvzhMy9wc4EzIu03cop2opBXpV93cudR2nDP+hZEJGhsZEa+DcB8vSzPFgBeHt9Gzip49oSyxpk2p1EbvL2LjwJNiy5GIvnsN2NSHJmzWdt4x5mQg1INV0YYuouDse9F4RW0HGDN2Zp9GAWWG3dKLzENVuNVCBsUkpRo7ChelM\\\\\\\/iZ2++vAv7Mszsnso+P8HCN2E7hWaev85Umaqkphwy1nbZhiEfQ83gkrCb+09rDQ+Pf9n7bIyHsgi1dKarQ9mo2q\\\\\\\/mARYGENi8N0XlJf\\\\\\\/l2JDnfo\\\\\\\/ESaR2Xv5uBOQ\\\\\\\/JNfm2XVVDU7f+f+YqtEJa2xyUjLz0X4vnlm1WFZnyCdDFKSVX3UmjKG1PNaulYgpHdb08yjtl0u1Dd6U5Euz3nTECl5saTvLF7LVJ+X9eAbggsIVyWVgFbyg\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ik3CrdSdjqOHu\\\\\\\/amtXNc1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227a56647da232f431997d1f31a70837a9b454d56c5293d99a09b1f291005d24d1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Bu40eCP6UU\\\\\\\/SnjVd6VEFC5u36s4iIOf+BSNvrJMvdSYT1bciFS3gYwxrGnP2ucSsnk60OqK4eSHChnYAAxNa0r6wACHJtb75BMFBbL44Gmee1wCR0n7EJvzzzULE3oohICb+2aNIUp4bE3O1KH86NjiQsoG3B3i+pvFv9GdrA8jh487YF178WyzDJhKs1\\\\\\\/GHkgXoJfajZ4oiZ+ETRsYOvI0IR7hNP\\\\\\\/o7eHjXb6MYLRIviClbP+pc7rfwixFHGUBZ8auqEGxBtenlh1MyF97qEoSWHdJ7g5rqOLNKpS2zO7PI2IKoRLmYmx17oIATWaFHiblGKwZxgUOnbFNdXoHvBNsbskiW6Ylw3t5xZKk2tfRMW9Fy1\\\\\\\/mLEcrfZilAsGlaN0NCe\\\\\\\/LY\\\\\\\/TpDz03Pjcc3\\\\\\\/qQyCZzFR0dOK5rplQLKCHAmXrUsQCMrs7tJvKMT8VdmWftvG1MYEehMPvvdRL5RACR1L6n0+Uf9GRGRy0uTFvjHyFGgOeTSbJv4Jugp3gl\\\\\\\/QmNQjq0aTfDzzXrrXlNHDCF0G\\\\\\\/kCIYxdZw5nlUNEqtM4QGhEqZr7rf+GQzC1ztTuOYNH1xr4hwvM53AyELtyWeSMJ7iww5UOSPV2bBs3jBKIMdI6dnWUn7Mz2OtduZv37LSRmGF0tM1HT3KCtkBpBoUpIFtVhX5ssVjMGjW\\\\\\\/jeISL7FWOMJWm7MSnuLzqlC5g84lZ47aS7P8lrwwX+2Qx0bz\\\\\\\/aUXHKE5mQVCZxw7pVbFC48W6emxwwmFZE+i8trfiSVLeVT5lcFwtvkXJg+V5wdN\\\\\\\/Ury1UypIDxWETFCDcOpIGWa6e9ZGJtQgoqO\\\\\\\/8zdoMTk3dpNzT1utU\\\\\\\/M1wRd6S5ToFzOzdfuE1V84YCvMlLvjwgn6eFxPDRVr+b1dCcR\\\\\\\/iBrmvPCKKPBamhfV3XgxghKwnS5zNdIqZxm0dy\\\\\\\/hO6xqRlrqSWVUR41gxXEViaBiIvd82YGwHMVN+F+2cdE\\\\\\\/khCwebfAr3vw1uVYigk1GsIzcem1Bl5rmhXWEQtTmdxbhqL4OkT8DwUsBwZUCC5NEqdedi6SuTZrcYAyVTT50NUetHIPo8gvl3Qjr6kAGJJxV0RMjXYqiKZ\\\\\\\/qN91vbmuzMXVD7D5RfcJ7CdFSqDy6WkISL+9Slw1AWxFdwZWwz+tIU2NZKBSxMslgKXMpKItfrPpxi2r6JUYq2upJZOjrtuDtY4k2kopa0EL3eDFukvwglX4agZY59PtYkD1xI8YdMMH6cxVFMg2hsZOdmLwg9bUnQJC20ECurmUuQ7U1BZ366npFtZyP3sGMsXtlPTgOEVdk4oKu0AMxA7hpSfCJPoic0RRLaxFxtdv+1xLhuwyNHwg3SmAqDTWAMBTcj2xkD0t++tYje5DEdbfr4MjxGAZr9LmPqH9pxY3XqdAOrsNo\\\\\\\/+q1O\\\\\\\/omxitwfgB7QKtYYn5FYBfiEsZixK5zf4770CKRDJdb10LIomfKa0Q6ukjRhP1fKEzfumcDz+bDsuwMqiLmtulOghnVGFBL49jXMDr+3DkjImvCsg349sPZoJDCJhFEIIgCQDstsyYHYJN05AJrcLGwlQFJhf9vAGejqxLOZ1OEETLpXavxLK+iwccw6K2yGdTW5og52Cqwbe9FimfH8Gk+rOzNft4LHOcUAcMTa7osYnEmmuTdAZuTBwRVi+yXV54JALQNArldXwJDKqIR00oalB5ukDVjsndfCBomMtlYip+BDdQBSS6bCYcb4KxpoJD2mUbAouu1guZihkJpM4fbxfEE6TfwDCeSJbw9iV9mWK7qeRgJEtjg\\\\\\\/pXXALgsGzPqEfSx6ZaMNZbv9lAGCgieYtp1jQzcnYzTeNRwZwPFHhhMoyKpjlKH6hHb8lA7HDa3BXMIIMuh63pIqAuIxxbhTSqtdIpYepmkz\\\\\\\/mak5GJiSe06phn63RH2MMdvzmdYT0oa5hDeog+7XkNdxmR0tKNG02xxNPCDug8XesTPfZ1V9QsEHd5Z88kreLTcEgiujk8JavG7nJJ+EJSB3FHxPXlz7maLY3ZgNZNpzsG58gK4t41eiM4zjTvbtKoaTZ8KPFlLwfqwQzeZIeDmVV1DHvXEHm2U5B7DwciDMFJ3ashE1\\\\\\\/IcPAboF80u8veTabEx21w4WTivVhV2Q3WaDMJJjzcyaM8XI7qm1OF2aet3ALh146VismFLep09CJ5n8\\\\\\\/zMBiJQInfAR8p9RBBoGfW8j7Ac9s68dI9t1ZTs2QjVa6FYE4Bnpviv6xdOkkbZTOL+JXs\\\\\\\/+Kx7LuetKF8V0bVnNSc\\\\\\\/VIhpUjdLtTJ7So3K6HQ5WjOuAE\\\\\\\/Wz1EIZ+HUyWhn6xs1yj7jk0qYOGhEplUa6HxoZanUR2hdPWGWAF4IaumoqgsrM+zL5ddIHzZmQIbT4xvd8LwZoOsI6ffHWXImb6m0Pn69LAR+xZvJIEhtBWOAeHiIo\\\\\\\/RmsrNfu9ZlMz3AQWB9vCvTixQegfM+urBttm1RJHL0a3r00lkEZm8xn6o9aaylzwHg9nMVTPrKQcVGlbGHXbe+e3Z47+cRgCFRm6W6vdmcmzFwkFIkcOa9aZdrQp5juwLEKDAtWpX+\\\\\\\/uDJynzRMVhjqMyi4b3C5m3IehQVibpa2JUyQQhjJpgdQhO7cZtyrMSKEzm7peEP+5bQodpc4UF1pGIphE00s03eTaxsVPugBG1HgtoQs7e35vSidS4PDBZvSRy9ffv\\\\\\\/1ZQ7QfPjK5Ia7eCWtUro5H84sLYcTmDlwqPujiJhiiNj\\\\\\\/FxEv09gUwNk8xUbL8zfHWNDtrswqntxNymZr93f7iOFVQv3J4hMjDVqYP07pQNkoybvLkFE+\\\\\\\/JEauEnmxGq7fewqFsiaGPiMl7b2fu\\\\\\\/Ll5MBubZhulu1yvkvE6e36X1WUQ432iWrcsvvKb6rNtOWEwaILmHUXkW4DBc41QhHtUmJG7Px2+AorVSX8wnoFLYcsUL9ayFj5s9x6fylV7h00jviaZOkf0wjl1YBh7FjcvXKM0BPgwUnhkm2wI1kWE7fV\\\\\\\/Dpev\\\\\\\/FTTZf8PDLYhy+sb66Lewq9XVs02eVuFRkFs3DSQWn6ZkkH11FvbIZsn+gXJV2a56BmpuQK9sAzmer+N8Vj2GrlkFpkGXU3YnnSJXYd5kZ7vdrM2ZEmKVyK5H66foPLZOjheQ1IJnak8DfPeGNzndauCXrajJ75fqjGaGGY8YVCdGDtR1oyTj7S0FoLxkDoeQt0bSi0tevenATZ3kos02nIO\\\\\\\/niAkTsRISnwf6uVSbev8Dta8l0ab9WGz+gccTQxk7YlcUejg6Cw9sfq5Ttr5lJP7Oyj\\\\\\\/qlwPpc4zrGmBlNrUnP0jolUuhtkAmC0cwC4pEY71meL\\\\\\\/SksPVio2zX2bYE1BsGnza91Opr1Df9zlE\\\\\\\/voFUV4RwVF1yesACFsYPoQ0MgGL1+rTQpidyx4JhbHjEqoEsrdpiXFlbhE7WavlYEOYchv+rC4GVzGZBymcFK+eqGq26i6h4DwqaaM10w26MGKZWNMO\\\\\\\/4hIxe3YIoPstDWsQgT4pKoQe+bz6Jrojigu3ZOZBRB8EeunkQkSi9cUoGb7ZYuu4dhkIzGnwVEPW+S587kwLfAmslQQKda4yN4q53ipulJQIzWnQvX8\\\\\\\/onKmc59Fa1ew9wVrhmoeH1GqrjiPjUzMlJzna5S40dtKv13bay2PK3GeXeXKdExdDI2b1OhcRT+tz6NVOtz6hXPGmbgz8\\\\\\\/WwFbe6jANjHxkaC0wdeNbuIQlKQcUSNGuGENodMcRD6MjGJ58F2OxpMmDhQjBDe5F8OBOCrYMDz48Bfei63mZKGUOoDVb2pylhatay76bZSYKsBwvC\\\\\\\/FcrAzIxyexuRaa\\\\\\\/mmCG6MF\\\\\\\/mZs4GIRkUFD+cF25HZT0AkuI7Cp0FaV\\\\\\\/ujoAbjTnyP\\\\\\\/eVmRGGROSSlmYA7u\\\\\\\/P0VFQupNU3eTPhoSwZsIbVSWqZnIlqFUvGm6ELrF8AD6HSZp6jKN6xKoycGW2cz1JfXyKaVTCkeHi9NAzAb0ubhietlKmTv9ryoILwPd4IC4UB+FX3repnwfIe30s9vmWhV3w1QEX4FyCRZQnyUKRmQFnzFHMJ4WqW2NFdrRVycY\\\\\\\/HShBnyLoh5msTrGTu4FTXkG2KQzBoNSyBaBdAtMUiq4LUSmfF6FjqFewNgn0ezNZpHIITr3hhJI9L6pyDO8L3RYReUKN+LkAZ9V6GjiLf7J7JJruzeEgxem1zfP6soe378L42d8YwOloJMKhkEHRuDMT7kU3VwwFDmapMCavhGAuhGt5Ka2h4apEV0igY1Y41NPUqt7owRHUddLn3GkPFTpRTLXfQuaPoqk8\\\\\\\/sXqZQs9mx5tMYcoN0+WF1hdaL1JZrrHv64jNFoOh1excaxC9BXIx5Va6JFIT\\\\\\\/UJaqM55vH8YJr9lyabMBG14mUcofCYjcmXuvnMn5pmeaUnygchdotkPy\\\\\\\/ZGm\\\\\\\/FjswW0MA\\\\\\\/mkFmT+2GVYEviE1IZz9sR1GQJo1VG+W5RcdXWtiHMej+Kj1kNaVneDk5mZ3NW3ZHw4wqiGe52ywVrG\\\\\\\/zLWZ+kl2+p5I9QbV0xbEfrYaSbFe8qNHQdH00lZI9WavqVBLPo7aKmnb2WHP+yYZsIZ5CNmEwsPUnnkK6nb82GCOhYHQHyikM4RKDE8LeZSSxKO6qEqM2tlm+JZNNHqmVOmXiIcxZjCN5LEtbMwAMKr1YjPe79q1wI79rYnL+rvPf8P2JmsM2vOuqJpIJkCi5augWDR1M3dhLCnO0BtTL0WbsLNXdiXY47Ji8JgpPw8jHT32i19ndy6U21fJmQxmZMeacD\\\\\\\/jCzMskk6k0KhXJVxhJtmGLnfNpew1i+0kbGv1E\\\\\\\/wqRBX48lyd6jopCF\\\\\\\/0eUd\\\\\\\/36e\\\\\\\/ZFpQcK+HZSx1jOvVeEMF7hhpnPU+632y3kX6yxMlEOugNblaz8Ksq5MY62nkL4H9A+zFsYD59mVdmuo99QP27+VNyoc1IhVZ7UCtcfH9g0EaNXRNRLvKx7h\\\\\\\/UL79Nl88kwbpboeyDIRSEXgNoHueY+oqoxPQpdZL\\\\\\\/6Xt4i+dTSp3i00ufeXEjQNfq9KT6Y\\\\\\\/+xsPhLr5V\\\\\\\/Eob3R+Zu2lBG\\\\\\\/fygzZaGjWKGz+J6SJERpBZzfBmXdMhgRGOpJmxr7IATbwRWSG6sppTNJEz9u5gFA9RbrToK+yucjqXUq\\\\\\\/Ap1bhwC1KTs6Tq0JVnPM3XveUnB3P41LWXwKj9rYw95eg4i3cZ\\\\\\\/tmGLyqhTS3fZlIJLgv6CjlHE3mD3hRmTAP3lAaTrHLXaIkCFgC9yqZuudnehtv79JXfsr2JknYBujls4cpENs+OFtSJGN3MH9PfU\\\\\\\/Sh+CQWKxf5N7GyOYr64VE5t2jsaVV1HBktC93AADX3Pd1H+\\\\\\\/W5zUUiM\\\\\\\/RzpnrW0KKa6biwV+sm4+wHkF2\\\\\\\/zVj2TY\\\\\\\/UcASfJAU4FxHa+vcCtoYx3T4DjiDgihafxpVY4v3vla\\\\\\\/JqB5Azv9NfQ+kQWIlN1eTzupKiDCSZiVRGH63hIIMEIiwU0V5pm4H2RMckJ0o1tNfYtR+MWxLMu5MWU5EwFQdRs2VD8UO3f4yNcpc875sga+2pP835P22kXzzCvPP7zwUFZ1QzJ6IgMZjHJokFbOFv9vSN\\\\\\\/Y8dzOEDVMM+YFSu4QCHhbLJwrtcoStyWruzxbU\\\\\\\/JWxTn7si6g0hBzyA84434JmBW7CGK\\\\\\\/Do7UP14wyYIJhgkp3RtMne2xiUjDUyiDILhp1Uur41vN0VqnkfO0YWf0N9U2mtKhtS+v8fxtzyzE5ikWmaz8kZG0uX\\\\\\\/7hLsAwSmIVByJEQYN6rHa4SxOCwFHSyx5Hm9AEKTMs69UQT8vX6HGJptyopNPgypo88ppEQmvvwG3Jcq9EFr\\\\\\\/pwZNhHJ8o9LELK9qh3GcKQZmIrTa7uBmdV43NXq2MEwN6wT6hWoaC\\\\\\\/5ZMUKjtk5MkXmmtc8qY5UmMPOqzWXx9hlK9H8c9gvh+pYavjVQFeRBEl5sC2D2zx7te0FhggR8XKEtvQwXHGj5YwIhCV5rqqcYFK2jvRS1viV6GL+wZpHz+W192yhH50D0zFpDEtf17sZAcfT2n7Ofx+v8idJK35Sld+M9wjcEo28\\\\\\\/oZriPhj8rwGKM4Ddyz6Hm6oCHeByRMdM6XDgYvxbUED3FNQ1FXaqcA7j0lHq3r9w14gIzb8M3J2rPQz4U1obB7Zavxwf06+CzrnU0wlJFnI7Ge0YB0N0iO0ssC0GhqXm9kcIPYMGuxOgGmI9Us6cgQuE\\\\\\\/UmtbjxwDnIUwNVdWY68W4+ryuY25TfxQ+kKV8M8NybCtB+XsKCMDKjxLlwWRiwU7YQi2njg67Lse48HeDMZNuvVabGNX7Ka4iO+ZVIWUPcfVfW65Mde84fxjN+xJsrkS9DCuK\\\\\\\/Q2zs\\\\\\\/BF9od18Trpc7O3\\\\\\\/YeAjEYBtbn0kdf46+0YWYqNAqGOVKiv32xWOm0ytWV3S229AH6qeWAH4Nfiwa1CpBs6d+WrVmPUeWKiKy6GRdX+HL4Pmo3uY8ulGiCbJvGzDcEX9OUTvFlmS6\\\\\\\/3cY5M\\\\\\\/Q8hA5EPWF+LHZd85wlv1NV8d2xxv62ia+t0Sm9o1aqoXiy3k9Kx7wuL+NCURCQpNCjjlygkEvL7UVyPPpD55hThjmmU6AYCSEX60M7dzSgXxwLNjfV\\\\\\\/G5jLqE4MhiQxAT2yIQEPkafhgZoX2ZBRq70IM8LJK3Q8GWKyEf\\\\\\\/FTA8GRE1PWLSEp6fmH7MBj+0odMKMvlE5P\\\\\\\/5vtG4sliduNf4246hLjRMyPz7aVyoBzIzFgKu09f5VK2cAwmNGHnrTzMzLiLlO0+g8NX9KDK9GSlyUbw7RXAwOZCwTsv4Il\\\\\\\/tQXTlHDjDHnAFo6MrvYw3bkwGltHmDzReFS1VpcQHI447hafiutPkT5CjexX+M1aUtOCyrlC1a1cVFJ8HJNgNCRd1cwfCwXfIN3jxZIli6tJEyRGgn4ibdXGL4YjM9LRAvgPqJO1l1EbwKZ6\\\\\\\/CjzqImI1hmilbyHXGnqGO9uV+qExgvjEHSnVuaKbpqlxn46GaTblnVIpltP\\\\\\\/kvoRsu+KS5MRkjBy\\\\\\\/eXAX3aRiMdf8w6P4rc1qu\\\\\\\/yzO2Ow+8gmNtqKg\\\\\\\/bwBb2btdwvYah4yf1neKasjC93OVPPJKJ6cpvu+Y5\\\\\\\/yUYn\\\\\\\/FmWBXTBl2M5RNgmVAicNpgpJNGMLi2Od2TePg5iVH9OwYHamqcrro1UZ1pvKXtQe1vuKNZUVcQb9ysLC+v2yg+XioeBmDGb2sgzuKGOaBdUaL2sLqsY8iXiIdUNbf+AP2y1CtrBn0n+M8oOxyncVObY1sy5i2A+q83DBmxj2aP1nbQ3g\\\\\\\/voBFB\\\\\\\/RCdQsABNeaQij3P+9lHCFZQYkgt4h6p2B1G27SZfQsO5MdvgXOQuW9c8RQua+o7IlpGkL1T4F5vOC22fQFoA3g8cKmGnA+yqU\\\\\\\/jpdHcqWyffJVTPR5Khn38332X6ZKBFrJHpdN05IEbps2BCjhSV8UFiNK+saTqI+cAJUWzj2K7AC\\\\\\\/ZTa8JMu7EiUJVV6HDG\\\\\\\/bu25OIrnYfBLqfcANjBBPCa9wOX2Ep2Z1gn9QC6HyVvSp8j6jwdFcrBqUY9QGJ7bC8pfzt\\\\\\\/8nNy0JQdRyBRBM26MV49W769EWuZ7bokLT2LCKjLQjHvXxcAKZF4WUS\\\\\\\/TzZfgP8Op0snhY5MRDJsNqXxNdkBYkerXO\\\\\\\/7uRVgkA29HqUPZF5PV2CNC5qN0Pmh2ZaTKHvkAnGWfrfLC7YWL\\\\\\\/yAwunkjS7bh6vJ\\\\\\\/m23nHCUQDkxBfgTyLdWSxuTv4Zc7NkLMU28LATMcPeYqT\\\\\\\/M0gPRmVNl9aLgWZPRopvGQG22HkxM61Pjoj4ODm3msp8uIMhNkBGfOcIZRYVQU8B6KqZmY30nYltg6JeglK+u6h+PBDjhOP8aGnvYXElRPJgwPOdBLQIYedWmq5bn8Dlf00iWflfkbaa3yjfFuNnGblm5sz60r7JI5qkWIJKxj6yeyGFWPyjq3S15\\\\\\\/PZwh0pb0\\\\\\\/vdCqRGvx8C36AttjSd99PnjnlRs4lfcrLsfVLJmL+90yDoYuxKrnTNwDXPC0kADdPMYb6taVCy\\\\\\\/P5l390e8+HKstvHwsEt5u5AKBKLFnScbKViFI4e8xHHbU4fP6x4JbQ12BnsgGZyUBZ7ByvLuwqaL0Josnhil321VG4hJK0DK3fl4dHcDOiymU5IfVSCj5nCor5ohwikfvD3UH\\\\\\\/TdC3bFHYvTjdMJOmOJOg+BqdwXXnSsxgfhfSxwHkhozrJvy+ULTN3bHtStaCGCY66toJLe5o1VPlSvO73O6n1wCcEDM4LDTFHN3JrF61SLndjlR\\\\\\\/F5sjQPOOf7mDtYnzLO1Xm9qwzmQwOvpEvX4TSXh\\\\\\\/HBpGfhCPCShPdFleFFEQV3OhVndVIsK43AjFNnvOw7YaYd4xSf8YvfN1qGGcAmFs5SlaeFJzA7rAvhh5GGDUjMN7MlAt4X6Gzyi3wa8NG6F8BrESsyYou8pmRKab4ThAFBD3KeK5toZEG6VCNh3cN6LWenK+nNNg1EUnuHdiuaS4BDzK1\\\\\\\/WYMDDKnRJvTcJojgiKNSwUM92AZftslOqp5m+RqNOh6ZiVbMA8tXIbmTlHO2xB9WeMUMtJAUj04zIWwbFtVWSrkf2+A7uLqauyDOGWYKgJ3EtfeWW5OrYAch5fN0sxg7842JJ+HTMkGbHoZrB7PUQe41vP4zcTF5DB0AN1oy0lMZhEamHe2tW9Y4DwrGahMHBsCqZPK1Vs60dQn1rjIxRpqa9ffMe1q0Vp4bCcqWMBG1ZpbIbr0ccdGA9fRm+LLRCKsCuM371vm5JWUsap2brSPvomvEdjoZyTmmUmcP+o8HzvpYDUPIiSlTa9\\\\\\\/y4k8Q6Jus5u2RWPNXKVzRVgiYs94DM\\\\\\\/QVuUOfpZWjtJEyICdzofx+vNfQYW59wpMQ6B6E0ZJ3nQVKXf2nYTqNZNqLWkIl3symEGJKA6Al43OUju6rqxYWWnHShI9p9RYAewfiEWhtcllD7h6Izne2DYPZgNWemgsFAeC\\\\\\\/3t4tBK9JfgX56XOlck+pdAMg1fvJQjEPHJNc+bF0U1rEEnlKVUYSiem6vePR2mB6DFIt9o3qd1D+F5HpjCstEBTGerXyfWl+6JjbSjYwatQZGMfZgEFUeORD7XxAhEy4yqkno+04QGbmrXJuR6qzSpPVIqZf79hiQtfw7om\\\\\\\/9D4GafSUpk1KSJ31ZtDMLwj321JnjYFNffpAeCnJxA5hniz7Lfr5LgCbJTAFflqlKzxAAPDkpkVSynQrUSn+R0CeTybJ13n7B2FOw9VTbt6vg\\\\\\\/DaYym\\\\\\\/512ROjMn0HDFhJ4c9JiLTXCCs9UwVAyHxA9VOnKDD2oGb\\\\\\\/IA40KORz31B+IP37HH6qXw9BVWR3MbwkE7aWg6cmiJUvTHdqat0tcjl7qgbbHxuqhUQcd+jaHrIxsjQP5lWsuf5+7WgxhsT3BhFS3lEKd38j60p+CxQ3hHTmDkaO43rKJBZTTWeSMqyqfcrFzb4n1J2W5y3\\\\\\\/\\\\\\\/ht5txxsKcaVDONaMVIhGM5c\\\\\\\/pzfOnBs1+bBeSoMENwOhWrcfBnR3gsB0P4hOjyy3ThO46\\\\\\\/h5PLdEKiW+HfQxAMSEqSUbSULkOvzqona3HadVive2sHQP2Q+v1UWnxci8JO6S8qPZvyeaOOpqj3g0sE99LGrQ0XxPavLiVvRQhNIKbVbq3kLYyKj1EPyua6zuMhY3cvrwkBC4bngin15kGbjqpCTS7K9qj70QVCuzO70znZXB8fxc33nrfVbHVmZy2Njp8SYk3sFnHipLQnU3g6Jjmjm1Ao2c\\\\\\\/6bsN1ldgxE\\\\\\\/7PgGFjZPdZsVui0DhKhEt6HALqn4T\\\\\\\/04VtWlCKTXMaKGN+sURyb4JeQ6eKjW3+g22N1BIUMJFUhd5hDtUszs7YpbOHIVbNzp9xc\\\\\\\/vQnkwHZsYtwADRc4JLI2EXRfVsHQHMvb47JOvfpwOPr02fAXwk43H6s3falGDRC5HgWEcmYLnZip52xCt1NgyliX+0Cinmb3YurhLUJ499RY7Sm8CSgIyVASEjg5zJddgkGDah\\\\\\\/iOT99arU+HCvyUsRmSqD\\\\\\\/ApWvq\\\\\\\/SqN5Zc\\\\\\\/M0qIjU5gIzG7jhcuahWqbPYZZXlH4V0MMSLsX59naf\\\\\\\/6JmF+oHmuSC5bjl3mc5wi2lC90jkN0s+aaFF3eOQL07feovmCkqcs1PcMdcXojWuyuXRCGuxLd6WYbV7Ip3rGitj9hGh8v2bFw3dx99jYFSMbmuChbTPMH713UU9t2FiwJkk6a7aU4zafrBwcfOux3HD8vfezS36jR8dDj6zigKfoldfaInwVz5FClqaPx0vLnnWT4pXBNjDeYkx4N1OQhYRC3FcPPkZbcSzKNCdCZN4yy6kcOhIhh7K8y8o5gL9ItWaDJHUq93ePkFB8THDv8I4jfH6fAAW2ZRaqwzKljR9oRbpT1f2S8Y2PH+zKI7EQiUgrgD+0dCk4GW3No+98r4+fss76Giumu1sZMRHgsRKwNTEXmdpcDJ\\\\\\\/fc9+yuSDqjHvp5XWYQkjsZGf\\\\\\\/voo4C\\\\\\\/50xYkOxFGMdaiEAVvbZ86LRh9aoVHhbMvuULH9lyWWzle3SigjKgMIIxMBagHYkP7zOw6XAO2db6eMT7BJtZxyki9nVEWE04Pg9mhlKINqjNZgjisGv4Pp2lNg25B7jColoTSe7uof1DbinuTDiBb\\\\\\\/\\\\\\\/ZE7oCzxmbZ\\\\\\\/4mxWyyWLUcHxyrt5r6TCSPW4iyl8wtuC2qzSpoN0RyjpQ6r\\\\\\\/ls0vd4piGVeuzAl49HuDw1jWDrlcMiKvIcdbe2HyB4Um4+eTNFPoJbRvv6uuWL5xaLPDBqhLKYYrcnD6dNiWGSD6+oy7jF7QKXECPsLthtH4sgRZ4lO0Hx8HTWE4E+CenGRbx8H5bS10N+3P16oVtksI70IuR\\\\\\\/nOl46mVzQ1ETm6AJhOhRJ0\\\\\\\/fmBJyNAAeChZtbF\\\\\\\/wWYDMytThirDjm+gtmv1YgFsT8kzmz1tr1WGxdpIslR7dzMW\\\\\\\/fO\\\\\\\/wYb1qYlsdFHqwS7rj4eoEpZmY9+WktV6NK7w\\\\\\\/ofGCgKO\\\\\\\/K\\\\\\\/6RC\\\\\\\/AqtWDkH16+aMmoUhoY6Cr2FUGqKxO8BnE7GVPqgj+yuks\\\\\\\/750R+9csBStOSptvoEF2PZtLFOhxy1BIJfxRGvC+7hZBl0SEEA+njVdb6ywaRg7QMoF2RLoxIaHslOrjK8y84W\\\\\\\/FSYrO0bLIyFk+nMXu60p81ruTPOCkmatlNNcviDoDTqQbrvQBFEeRSnm8imqApxU5s3JIp1o0ogrJplUCEG\\\\\\\/gmgD+U0WQ50hzL+0V4+9HcON9yV1EbeaGL9h2NOD\\\\\\\/VrqJKnIntUo+bjoHvIt4S1\\\\\\\/TooSOOxqp1vXM\\\\\\\/cDNWtfLAd4JSm363wEtQOovt4sf4eA9KuiL5RndPQJ3KYKwsdvQ6+z13\\\\\\\/Hq1xf4PFx2No8FW7luCZK1UlriB26eq4peddftAO0IKjFd1q5AgoYI815eJPWjvT3KCB5w\\\\\\\/4d2LLeHM3Yf63GVEvae5R0ygy6tjAU9x0rYKCEBSdQMyapWHoszRF1kNmUnJYR8rAR93Nx5X7EyhoJqFHOjNyH02G6gcS4nDe7+aa\\\\\\\/5\\\\\\\/gxfCSw94omXpziyC03+jQ\\\\\\\/yZUKY7WWuy+SClfWZ8jovLOPY8DIsznJzDI71KWUnqG5OMakr8X3Tv+hTMPMcDqtzqWD1\\\\\\\/HmYM\\\\\\\/\\\\\\\/0y1TsuzHs4syf8Z60B8gELHDkA8A6lRv5RlDHIsr28y7SqBG\\\\\\\/Q8c\\\\\\\/zbzNbo0Fyisz+Pddcalx68uJBetDkdsAIMt4QIj8nifibL0GcY5eI7MVsiHoWnHIow3yeuxgZzJNZSeftvQaK7ZMwDI2vHVW1nGQChZdRy\\\\\\\/SbCBlukGrAk7vqnEEV+R1HUHN3E2lZVL7Davj0wi8qUhx\\\\\\\/VDaw7GrNkSaCK6yIwF1NIynUOriBj9gNnMBEX0jESmtIVukFToHBH2WiSTTWH9ZKEs4BT8fhhSmp+jlgp+f4\\\\\\\/iUIPl3TRdeYQDKPpRm5m3oz4vqC2zx48rKlFLLaj4IjkNMxkZGToCwsWVn2b67eVLLES2gqikg3oow2tTCM9jJsxMHu36tF1IYJVZhsi2OGkC\\\\\\\/teWuEQm2cPgp1d7\\\\\\\/M7NfXClptTdl967Ah7rDAlfFw4v08pExbIR6koQe2GS3aufzh6SwlzWM5siwXpYrSFjIM5y2Z5VgX0ulu0r8INboP6DohDUMtn958wqk7Z0TuxmwjyTmkb4OTOxiiB6T0mGdmwu\\\\\\\/Yb4jSch5rN\\\\\\\/az0YRF\\\\\\\/TbnHLC+BPQr\\\\\\\/hZG1C36b1TXQnBEoXuYNS10JWeacyrPp+xdOvkExirZytMfwGdgkJ6akSPgNNk7QI9XtRq69heB6EojdPN5q1x2NKq+3AnHfkkFSlQmMNDIPR\\\\\\\/w1h6kIsJpANlqKpwaZiPe1+HVxl4g8oaDjCaF\\\\\\\/B39smX1Oqlf8BmSLpz6fOOFRshWLlLU9hBBNl0hovle2RWBGqpkiwC9C0KXdjJczeOGFk5HVBUo7Jhy\\\\\\\/TMeSSAbsdLOV1tCBjecWsc0rVTACRLIokGj9VoTLNg79IJ8vVwzou1C4VyGgbt43rV\\\\\\\/omR2pszvKoIOw58YVPnjkorpI95fbGHg29MdpQspW8trBWCx0gjGJqWhZzln7svCji635ZpLzc9KGITjP91w4EyVz6QE4XRnY1WGjYorOqsH96n6PeDwxddwOnk+1TnTOvAHvY6HXGnwUgSE9UgGgz0k0bPyqLCyDmqECCriJZTOWFfcYG8sD\\\\\\\/gPUWjjlHsckbh8+Uq4ztm66yrFRP2dNDXA22xvJ2MK9PunFPmOt04KRlI9+m3CqJS6XZMIouNcs33FMwfASJvXQETfNT1fBmg4Y+nyFC\\\\\\\/DUhqkxPvs1iSpWCd9Nk8+SJq1EHT3Ui2S5\\\\\\\/zeIfilUfrSesHlLeF4Mw2IYvqez2YqgVpcNeDFdjkGrpYMasK9i9giKgE7bIDlI4Dv47om0FaKKVaHpHQDE1SIRRwn+ln1swawPUuf7q68Fm7Dg4yQyncKV+Xw0jPAYHdq3CDR1eQ0Suhh\\\\\\\/1OHh4Exj4jrXlOjkl0r1EVuXEUKgosu\\\\\\\/IB0qmdKEDpfxcSklLR2G5AV3uJT4x\\\\\\\/NEb2b04cdOLHai1VU4SKpH5o+9EfBtNYDqZDP\\\\\\\/7nMg1msvU\\\\\\\/5S5ttPSPkKPGMQK2H1uY6q8XTHOfx4BBqESBTUPo1olKT8JTioiDTB06236wOfGUxr93DIZLMU2yeUDh7gkK9dFFjeyC+Tpp9\\\\\\\/TV6J2xuqcEARPQfNsOQv6SMhFuLcf20q6Qqgu2+Qg2vq4Q1d++FwJbnCFf1baKi9HGuS0E2uVsUa7P4esMjZQZL0qtlHsVtXr\\\\\\\/AHjV\\\\\\\/UNsEfNkg1wyX6Sus\\\\\\\/iXJjKnLvvuE+UiKd9JjbrfDcrNJdOLutFZuim6ezfgB58KS7MxiJxh9vA3mtqIHpMRYwqasPg8ArNWysKasXU+YgyEcnwHGZFJRLr8XWeSq9v532fFcWYEGn7bwYh1qwOM2JTx4udOOQEv4+5udYgqdWLqh652hMLSGuC4PI9tOLCectBe4yu1vH9+mnRSSxp9YtPdHS8Ln8ffUuUle8epDg459o+m9ysWHBMq5SbAuVsJSiV0n996EhhE+0WMZMe7kWKcfs0UL10Ez6rV1vGVKIdutIzKBk\\\\\\\/Cf5ALLrQsnPibe6fV+TMzMlUcvsWKtzeARK2H6Lqx1uiIPBrTSUD8k8fMOy2RQ+nLh5hS\\\\\\\/auQIJtnrpAmzwLfRZ9nK\\\\\\\/jgt6fxlXSLWcsFYfJ807I+oad0CiAKtwO9p9KMut551PK7SFPv32qSuEFmUHJg9+0\\\\\\\/Hk4tttrDXWkMuNG0dP2Mx6P1xZ9IpSbOTYpJsT3gDzFKmXj3eZ6naJVSO8vtHwj4oseX4hEqdIvD5wTILzWM9hHtVtQJhh7Lkm3o\\\\\\\/RRcabYiOfYzmmJTvjwYdvkx8KqkGJJmnBcmmgswoYtVbKMEOnxY65dCcU5BYfwkxkXFEeI6piR5OqA1DFSvliOBZeuSRYFkQ9\\\\\\\/oJzySrgtLQq9lCQy+thgPu49L1KzOQgSCgVDGeoVk3Kh3cBixH79CWkr1KvIvTYmgPsviVggP4jmTGXOAEiYEe+JPW+P+Xz0BO5m8GjDprxBmqsKzfQQk2HFjQxgCVeKg\\\\\\\/6K51iPKw7fiD8Jd26sAzOi1CW3eoRqRasHbvM1VHaRejw5pKFT9PWVDgnHgO9oXC2ehDCYWyeq5Pj+YvdubJXIWd1KOWoD0jduMy2bPf42QhY9VaRLHrBDOrYierDAkBm1Ls6G0lSlzJ4WbiewziskTQgGGtSbdeiUE2eaxXBeQMTetsw+nLT8Y+fxtJ+yXDL\\\\\\\/rD+1um2Krlcl4e\\\\\\\/corkGuKykn5pNVAvD4wihFBTVxeI6UGxgGPACS+a1rYGFBdM8Zuqs7mCF3ROGdzP3vEK0Qsav3pYXGKvRdV\\\\\\\/fPRpdgT4xIfGRQ4wgWpVC9ANUDWeieeonchm7P0gwnqETD4HpY7EQDo2uEcU\\\\\\\/14R+5mMQOv+JDIGyNK7Snt9KU58htcGWDn5FDzFUrXwQUTQMw0Cq5\\\\\\\/qoXZhurhKmTrA66WF39hZroa+ET\\\\\\\/6ZTBxA1+H+7kBJPnK4sAc0D8Gp+zMrti\\\\\\\/gMh1JLKR\\\\\\\/x7SUB\\\\\\\/iRL5NLSta2une+AKjEP\\\\\\\/4qma781hGBTJtL6NY5b5Iyd6bbeHNa9yAIjdQOUX40yOCU5uM8l7B3MCGkeX9jiPQ8PGpWT4DFYOoP+ORTIZsyWsIAnJmAD\\\\\\\/romyazkdsvW3dkoXmKDHMHmBb+Psh5VlVNzeX8XaH0Y+aHKlR0gDT8Hez5vOeXlkl1m84bq9PIRRO04yG7nh7X3j+kVYHslSx5W2KHyoW\\\\\\\/qAKXQWqvr67UCfDQGWae1\\\\\\\/sqgLooW4p7xf7cMyNTCXwzIBzeiIEmJ\\\\\\\/bDMYUbsUbhYwsT5\\\\\\\/KfUFaut+5ijN20AYqA3l6\\\\\\\/CVqvlFy64xwDIMVLEMbNDRsH0ADOm7w0uYI9wCMFkOZV9r3fdkphcJrhDqitnAwQ\\\\\\\/Y58wGM31TsemGhJNNABVwZAIi0RUHVvnq3h7rq6g+ulbbqFvZPnxRrzlHhLsay9SMOmomSy9p8ET2WjIAeL4H1ylZuGdgJ+Vb9MdQeYjhxAIrc2U22ZWIUeX\\\\\\\/9hbh1F7S2Brz0uu5cEO7tuautrNdF8WNGDb269gf\\\\\\\/NbhXXW9aMPZKc5PZ6DGjyX8j3ErkOLk5WVgl1yBfuMukmzJ4UJ3jzlq\\\\\\\/bjKw+iXBB7Uh67sgQrFJqpgM96XbspFxnvE1Elo\\\\\\\/dJ8A6y7KFqUr6IVpQUbz9YABsHcxHeLKRQmaKxyvWLNQP5GSjY5\\\\\\\/s\\\\\\\/ZRoaN7VrrCxXyyHWG1AbQB7W4pTGKPPI5hXaTamhZOykh+eJxeKKDC4eLrlAnBiKtVDkuFERH5QIJfteaBmMtObmw1V8OBRo97WzNiEnRNzMbkvWGSay\\\\\\\/RiwGjYee5tqzcYWZe6Lf+r9d0SRE1VpdAD\\\\\\\/wuxITPKo657NfoR3EofDthHdcpf9Ke5H4mu7AZEHpZ9G0AzHfQCdwvZpFAq53u27xTldYhFuGGJhCFIq4dS3zjlXyekD1DM8YJK2jIkbJA7pJs0hdljqEztcGs5v7sDXLH+USlWEyge\\\\\\\/9rxY2j2PcIadGxJTYkkua5aJXINCO8zGagRAA1pRAB\\\\\\\/srn7ZYBkrcWK4mPsVKbpuZCkEYnTltc7yQjJCRfZpqwI7rN6vgINzoS03QNSLTndNACQkA3PviYGMR+dfUCJ6JumtN0mjQjcWGeXwnrmkoZlVKN6XmF8J4wDxsGGTdEAZvCVeVj1T9VfPJHimPjJ1Ed7kNKNILgtdJVRcijxOY9xeJAOPwlUoPK4XWVkNCLT5hOtezKzNLchbZi3e47cOhIWDTGg3hPX+fN6nrsVBPKEDYlzG51HQqwjdQ2Q2phvu0oYJ\\\\\\\/iXEnze1E4CcjexrS+No5eohJrNpJeghxHRBj+wKpdfw9oaqSaN4w+VZTIH5rfUxy4xlFITRdCYPZPvKAB01k\\\\\\\/2ZcRiCDD11KwfBAAi+8RSJaPAz6ow7\\\\\\\/7srG+40E5pg+8oAm1j75J1uKK\\\\\\\/EjrnsyEpuZWYjdWBkn+gCg9GnOgoUk141pMjaWKI+BkPDsiwqx6hu1uEX8yAJmCCRdezpZChi1dHT8nQNyYvkZyG0VzHNgE9ahKA+qeSONq9LseRO62FT8WOu1x\\\\\\\/Ea+ZMjp5z+BYmzklVkjYKgXQBTP++4+9SNzviKrGHw545ZqHpQTub4wNUqoLZp3UijtlRo6owmGvuBpyYwz7VgWVOpOPe4i8q2enhoPedcF8xNg3SaZN01NYl5IcWL+1ah9LoRBYmFnKaU08+BWkRPaFB5e2raglwNvP5kQ2iWWQ6BYh5R2sRRh\\\\\\\/8DB8Yp5lSNY5AVkb13C1zPs8atTmfprpn2h\\\\\\\/KtbmCfRjYgnLvo8zeVnlyccBw80tVVi+VteUY4W6iwcBAnVH35vUgd+g0UTZcIqu\\\\\\\/P1jxbjsf7enUWzKrb1hCWLSx2GqRI1nJ\\\\\\\/zEztL4pxjwH+F\\\\\\\/zAtjK7fNu5u+EGkqu5BjyhtWoKupRL1yrLIP3UP7TWp3dsCC\\\\\\\/ynjkM7Z7Eh2z6zR3JQCTcO\\\\\\\/qyf4srQyjmGQOduawfsdjN8WW9rq7eXnweBE0P0g4aFbfbwGiAH+f9Is4Q3C5NAvupvQDOxHCvpQMxCLgg+C\\\\\\\/nwxmfnKCy0\\\\\\\/mQ4v8gw\\\\\\\/hVjinopt74rH4RZA+C\\\\\\\/Fz6yCgwvDIcAVOXaE8m06keZJzt4OKvJOikXbB9\\\\\\\/QviqyY0yBIMjy+Wnief9e9OhRIcxD0dCJ1HQ7YzdxYaIQIGTi7b+XFf2bWnP2VPR9rJgETYpIkP\\\\\\\/djrXRRDn1irbtF5hHVzMRDg4uXkebUIpy4deI2sCyu936JN4jKLTMeYyKOmrMQUR\\\\\\\/QseJZAHDRJOh1\\\\\\\/Jv6czxgM+A7CMN0nm\\\\\\\/H2uYQjeiOMYZW3G4L\\\\\\\/3Upbs+lXxQKrViGflG9v9nazArzuqNhlH\\\\\\\/7sonqo6sxYEewR96bdbRsRDioYoCv0Rts+NtdnXg8SFHK1qkYm9E1MUqU8\\\\\\\/E5k\\\\\\\/p7VZg8uXEz9+pglAwc2PUGU3wVsBKDo80Q16f9F3ba6fxNL734ZLZlyTJRgvYuQF7sGosQJ1EF2NzZU2R\\\\\\\/xIV7DJf4ASAPQB9P40zQgazzQbsoG8d4oK1djBY\\\\\\\/q7ji6\\\\\\\/dcuoEdaolOEl4u3cL+YErLI5PGLEIzKLLgSllKX\\\\\\\/oK8LVbqCQjDNCeINZqs581vhHEG08ipaI6aYE5ENgpL9mE1eXSqPJjSBQPDcz12NXxYWzQadRZiDzrcbNhgQhjAw1oZbFYvnsfjNAx778fFpJ6TFjen0TssClDJtvr7jd\\\\\\\/8clLpjHpXh8RwGKgP\\\\\\\/NbVa2hucNeg9TMFSo771j1SaSuNwM3e6oaLJuO9RIpBujEkBWhgqQnVsGAqgVEpHeCFbcEZKQ9RVqdaHoYAoqN3lXfYtQ9gifFtZsQhGCM46hUYBOIoXPP33mww2ONRv2khjTdJqmHB6+ZydhkNxyJhzhaRzI1AjDiwwSXnkRvtLgsGuFGjTUyRcldmWM3ILXBlX4lIbST0xYZ7qsqpFJeRPgltt1IfSiWJRoEyip+5EkPD1XrVXWmWhrY77Qboe0v44ULL\\\\\\\/+hNi8mw+8GqyvX+VT7Flf\\\\\\\/f2h0hDrDh2tirRmyicQ2vDGzEWcZE60dHZ7EA8wqJ3M0ukqzfLC3wJw+KMvHFlBjBYnKxUpMd\\\\\\\/5xeGcje1yZnZbB4tkzA14e8UPcXsW12z+0pHvzDOAqAGeBftOw3kRgRSbIEM+3\\\\\\\/7FKlpPknaiHLEf4EYVSwXhbJKanNPtjVf3ywoNfWmvQBb9\\\\\\\/6Qp6cmuIFd5L7CVs2QmFfFa5gKaUeqydTQj2omUnxFKHY4JYPbud1Vq9V3O67doCpzpy1\\\\\\\/2eZUCV\\\\\\\/uwIPBbE9J2SSLCrLyV3m\\\\\\\/kAfGG+bO\\\\\\\/iclMUOI2oEg+7P9HGOVgtzW+7FvSJugToO6bvMUq69GJKOaAPwhxn06Ewe5WJql5tJLV4cXUpDEPUu5IKjvBMdZHd+rGCnjq3OpBqwHsigetEG05Z0OgxCpHxj8QSAoAUjozQ4ruk0lO2CZh34gKzgfaV44zYSLQAtegN8S52v3tIiEO6A27hUYmITGMVVxh8t25AVZupwC6Msmt\\\\\\\/bRAbWgyUULrygHIwAPSXCLHZ0dMAXwWZiYgMupQecRyPs6+GGOfh3ExdUI0nhCPuy1yDQTiBQSNtiE\\\\\\\/NAYr8JPlmTktkrfW0KYBxVUH+FJyPUQHnO0EJfQSmu8S+ycJXAdNb4Fnde8nqP\\\\\\\/lvqPgxNQc7DDqTl7oF5Ot8ZQFFuA21nLY8cmwLWhe8tFWxn40tTqJmmca\\\\\\\/Ef\\\\\\\/uyuJJ1Ent0GdQ1fOeONXi4t0mQCMDbsI1KpTlrU+jG+Ot+wMQuP0tTgq6seInzbo3s+AVCbu3A2Nmp8nz+au7doVZO9hr4zxnkeef8zcMHEptvQ9mvYZMoOPeom7Og5IBgjkoluxG7MM8NkvpvPR\\\\\\\/4DSPGzYBDNi6bqQlzRJm0rlkuBBoWdXyqTm9KFje2RLe8or73fCPGhB3FRrK6aw7Z5JFoT6g3POM1EG5ABOw7v9Xkq4EQWlea4qI2c8SFG0UQf9uVVObSopI8Kmi1EysU34H+\\\\\\\/KGZK6IMKiEavqmTIPSJLnJKllXYiGTj2ptUOle11ycfZ97XhtLIbNDNBn+WvZjb0DHt\\\\\\\/6v1yaFW5D0m+\\\\\\\/CrrnFaOvyCzkdDvZPoCKgLKKA2uAuMO3METCmaNfBSCkEezVKDZ++kcqNvTf47PmO11EZhQqRieU\\\\\\\/XS0FkQFr0ryfERdwH1hrs4vsLZitaWQzruOwQNvfEp0qET6R7eFtCop3f\\\\\\\/MO9E1LJOVMZ0t0R3QwEONslvFRNXyjCts2Nu3KuyDIDhw3AsXGL6IcqeMKS69LXNz3RDKD9IZ15rEmMRGS+cbPQ6Gjm6ce791cNHApOkAhg4OScxE2exSYMvexxK+ooshSCiQwxQOVIrey6tZ1vDmesSoUKtof7S3Bv3dTsRHWPD1NaUFjy9TZE9DiHXtGqkJ\\\\\\\/kdMKNHGoP\\\\\\\/mKrqsIawBxQIVokA5llNYCnZ+7bFdxst8+95VSKjFh6yuGIoF8BWfSaXwCrv4EcCBZKf8gDipgiVOgrqxXNTsjJxkOZUiQP\\\\\\\/htkcBn3USIn\\\\\\\/kLt+H08sK7nbMlF++1ETPCN12rnRavGrpGUAT2yErJHey1FlRqI9JV0mzVv0WCILNGdlykIjP1xfXUsh3\\\\\\\/KSfTy70zr7pNvkZYJCjsDm4dxcRn1QPi703Cac4VxP67csmGFnAoMYFm8\\\\\\\/hLEJw3+dsLH2748zfmON2qRFOgT0M\\\\\\\/ajp7J2IpS21erMj+y\\\\\\\/t1BDigq103uV03YHiIzvFJqzs2Fw\\\\\\\/jc3ga1yll1Yl4+tJvaqVHb30yyC6NBXG6Scmc07nQgxRMm3PXq+IsJzJSs8zPvdMCU0r0I6l1cUPGf8k9ltu4GSfxvyc+x63kPdTi563TCsq0jyIB3N1nH8LMHeufH65c9yTg0KAdRbM+OAtwgQ21pj7Hqp9RcZ9\\\\\\\/s+AhSqUXMkx8mH2z5rUGYuTyyN+hYJI8HvbX5QV36HiIf6E9lXy3Z8gJLKzMcWKt2T1B66w90Bpir+RPkyGzxaD3f8WjXv8YqgQt92Mq\\\\\\\/iPYjS47+iKaSQTlRe\\\\\\\/U6rYEhBZwoL1DTXzWIxF7DLxLPWuDDHHKADUtF4VwBdbwmIhwNL\\\\\\\/prCEcb0QUIP5QA\\\\\\\/fr6rPSJVacGvJJreBxj7LkaNizBLmnhqFueBZRv4x2sabYqr0RdgjOhVHXxybbpDpcnkmRY1mz9yK\\\\\\\/EddFIIeNMEttLNDkg6JZkQNkycJBrB5QzXGK4CVk3Ppu+DAQ66BxEbGea2T76DrU71buHdCigMgD43\\\\\\\/bSKVAzc3DzFyhja+XxGVVqVI65EHGviRhoSJ7wC19FEXfEBm2S5Qc3pSHEkwoQ2PoFMJtpvG+V6ukwxcplrEwXl3eW6lGa0mMeQGzdfmdfVa+bA++XWxH0kgaXB3nOk1\\\\\\\/0eWAZURGIz1X4RfSkt1Pqn2z2qsSwdP5e4WvybwU3yeAzCSXPDjKzGca5aX9G6Z7eBe1TK54GwM9l7x2p8YsV4IrkDDq6IoKnxizK\\\\\\\/5inJnio\\\\\\\/1Ge+lImzDrRcKaaFqhoWqNK+71R1AqDLTsBdT+u1Pl3eeLxzInAtz97YQYgWJz5omQex0bbAY\\\\\\\/kN+rqifDZHpIyxAjHrjUnCRQsl4L76fOIGPbarMo\\\\\\\/v6D24uKbP0ooMN6sF0VRwutURw1SVRRThCv6Li1AGnsxGtE+jyH+pqzWBZnjUN96\\\\\\\/78ucLVDhV7DzU1v1slwu0MM8IxEAoUqgb72XMcz+fslRyCexbwMNklmsaO8EhQqrLivcUYVuGygl\\\\\\\/R5Cy2\\\\\\\/IcvuJOZvObwzMARsavLVZNj0zSF6fYzSMdILBCiLe4NegAiZAR1BheRbD+bCtty1No2qbm32cyIZXY6E5y17irXCNobgAb1jdDWpjJG9ECVUkD2b53L8Pf+O8X5v3BffnBM2PL+NrDarL2c4DN7Y9pqrechJ2neEFKGq9+14RXMVIc5WHAZ3Sc2mNrFGNk9zpw1rH6Ty9ct2wMwdyvdhDH7E78BukcnK0YzajhZl6g3nRajn3smjaANhvLTXJu4guB46ajezGDqXaQYx9dgisaEWDX5iwvUE9p0Cb8OfzLQ3v0JfjTbzvfx2XLvHDKDBooN64Msbf\\\\\\\/uuf7PISZaMnJpNEQCxWws5RQZjzVspclR+Pud9yo0ny\\\\\\\/z55iAbnUt9C1C6+VY\\\\\\\/UcZuxzGfuWUZNwRIv4rK7G\\\\\\\/dJ1n\\\\\\\/W6Bt1jw2PrC86uKjkgE6YJWmbNQwW0Tgj9PsBjLiAeRahKtDNAg6h+l8A+yKzmhtlJaBsNzJOMM4ngYTazU+QltQaNI7bmljs42p8t8ZxIkf9Pu6lHYE2gBe6HwfM58DmZSGgKc3uf\\\\\\\/a9VqJlmUh\\\\\\\/jIcORugRMkH8B8fzA\\\\\\\/K4S8c\\\\\\\/ZQUCpOaBars92Pq63ifryn+LvtaeZnhT8u5LfSDbB+cLdCvfXxdLgdmXmFwBwX\\\\\\\/QKxXttNe+Sv1ZzKWFvHvcKwRV1QTWyubGF+Hlv1gnB4bQSrhoY1vqFBhjBy7+WZk5u6bbPPhkhGUJtwT3rIRLe1RkAqA2DIHmvZJOtmgr79BUiAWDn87XKki2HGjc1UVXc13qbaaJeHnIzlEusACUqYVEloGEaDRFJvWgGYKWG0NkvbGwvbBKlALwaMqMVscUuXkXCrgZInRro76w\\\\\\\/cFHJufxXGPbvyBoPGuaOLekP02DCDuL80WKTRgRE2cIsONRskXtjcTz6BHMIjWk+E85wkDh2xCQNE+G8ZHvRsa6HvJ5Nu49IVVG6Sa9FRNQqM8UYVyV4Hbjz\\\\\\\/K2gIW0oUTY7iMMYM9zc5uaL9RZHchYRZqofmMyJ5YSU3brFaEy\\\\\\\/WMmT32TudMoma7Kd9DkZPBddLwwZvbTNZP\\\\\\\/kS+mlUx2c7YPsknci0ib8lAKEJj83DkxQntXKiHgH2BGUXe9KlH\\\\\\\/iSaIxfLOzrjY9TkqkPyClsb4+yN2lnJZtaPAfZqYyd3Jv95M7f7klYnINcuUN6kUQs6LeZEO+VzM\\\\\\\/8\\\\\\\/NOsxaGB5AB\\\\\\\/GCO05purlOdP0G6tDrfOZvp\\\\\\\/pw8lHfBNWrRT1BSa76SxEfTwqMm8rdPtrI9x8ZlptBq6ppB5vYXu2Al8rvtsJtQrroSncvyYiX+YS71utlQd87+3v\\\\\\\/nPgDKhnUfvhVjCBNGafOurAn4iNvQEYeKWHHkS69eTy4HrLLcU8o+11z2Pgr\\\\\\\/NiTMK35pJG\\\\\\\/rpR0LunhB5dvU+\\\\\\\/gFZBnPq7k6+V0ZLzSe7ELP9dxI+CNZh8YtC9iel+iznDVsSyDcqr3m\\\\\\\/q\\\\\\\/2AvWVJNBKRXcZMrNNExgeLbRxv5ZdwFTeH179RijzosEQ7gzVj\\\\\\\/uIU1afA0XdehWvORtqSqnEfqCEyX775GYUEXtb0R1CCAkAqw0im8L3FMEQWKk4KKahxO8csotNoCpgzQeeMOJFElW58IpQY14NXTBnisU\\\\\\\/cU8d5sYd1sUNSTQQK1Qa95zjE1o2sFtPWCzEl+1x8a4ph0V3go9Nh95QqUH1cuZL53Dqx7+dBzt2nA9eKxjWkA8AwF4ZAjRAonSnz5WbKWOjdWh1tJV2JuFDWirmwArlVPXtY7qxChP7r4LXyOLHiJeCDUTD7IM7im+p+e6acBQ8V9pYtn6pxFpqpViDynqpXvnHJdF3NIVpS8bqasa1LgA\\\\\\\/2IzZbB3ug8qmfZBIFwXoZzugt2P2\\\\\\\/KeVy+pwNdEJfgzxwmpZnIkWxaqtqJckWLn0sWSIVBm1n9NU3PAbkDQiZncfXbzk74\\\\\\\/xBVPGndfK\\\\\\\/5N2HNStqFbUerlSP7GaZx1YPGoSUbZqVZrfUGryv8QXNJ9I9rKARg0yp6s3jzWWdP045vOqoWS2URnkdNqYoRTdOlF7dtTVgEYDkJFZGEM2Bk9Yx8m13xF5Gkx6CMSXMm2rr3Ui4pgKTB9Y9VwAMu+VJlZdrnpwhq22dEEJr5NnicoUfLHAlkzkHclhwZZXzH3uDZvgDrfAfR1u3QcJq49ksgOZdr9a\\\\\\\/YwoZMPBZSL5Nh4VG4epEIonhtlx04As6jzIKRk1SKthbWirMBMHUaOrPPTTEczaiMY2BiG73FS08MokiytQmG1VzJafQnFmFmDHbWaS2vFYTtMI67tb3V4+nVEhFCIhWHeqRFhvNddUfPWN\\\\\\\/YTfSAd+kdrNXbpMv4p7AHrQ1Ifh8+toGUXl2jNdzgi3KBztrLQvgXh+Cr5iOFnvTLmScqszPRVC7F0LECy9lRSpGsjg7F6klanCfPp8ZipKicr+MFMzhiQa3AOD9rrd0FjWV2yshSkUYTNm4GqBBK\\\\\\\/VAelIw7kMI0PPyHYPWwftjHf\\\\\\\/HilTA6oZ6RVEiC3VFGPPsojMPPKNU1Se5TWyVGT439uXmhzyQTRoBuX8QvZhlLCwk8iZk\\\\\\\/LdrvRBzAOpCEfo2NxKk\\\\\\\/v+71MajslKA+iG\\\\\\\/b5j3\\\\\\\/aF59X7H3h4xSxroRBaSa2iIjmIkpWz2Z6FQysvrwPbZD\\\\\\\/TUW2RWZam3aNiDUMxBWXh5Jlb8zUbzDM78Qsm3oNM4ZuoCa7+LDLwpc68bCuOC8GvBF6vyJbN8fHEsqecepvLXLqxn9k+6n5cxRS4GWn9WbOLHIA0Qv+D5mmJ\\\\\\\/GC69CcPhhcuYnp2m\\\\\\\/aixX4DD9PhI\\\\\\\/6ht0VTdNacNlw3\\\\\\\/jzZi7eRRGYeQrv50Te\\\\\\\/cw4AFpuPa7azLkQR+h75nnHxAf2iZriHxJ7+e\\\\\\\/lUJ0J5h4mEwBYxqTvfPyAcliSDCq+EkEHw9TllrBH5aYtBVF+goQxLWQTQEbZ1nLh8bO58V4knl6912PGCBAguhE1xY7gkTwR7u\\\\\\\/bRwO0TWv+wyMzlp7LyoxxxDNRcPzm4PvRhrSjVwww4R5EMGh7vf\\\\\\\/W\\\\\\\/bzGKIe+ATvlc4McYpSbuKfKDNW3yLb9KM8oTAqnFeBrkPRjPdXatYlxoqmiz7FGkSZv\\\\\\\/trpdTnZeenN7pKWceemAPt423mQo4S8Eiw940cqvAAU3lzRC9boeRY+63yyQVbTbmw5PtLPO5eqbUexSZI3J\\\\\\\/0bH8\\\\\\\/WglFbJfXz82GwL3lz922XBgVZeOY1AuhwCdk2fs16qmQ2cayhDjegBdEEJXvlxyxr2walvS4MO0kDxUbpdFCr3PdrkdyOlEAH0+30TPbofNVggnMqL9ytk\\\\\\\/8TVKVvfLTp5DcTsTXqC+tkJNSImAO98GeX7U6RqYshvLYI\\\\\\\/5arT0lRbU5VuanTRFbaZ5uE6hvUsi7rH1mXJCkyuzurDHws6twbyLlxgsfiPnZUthnIiqZGwUyVab1tv0RHXUeXa6YEcuI6xrnE71Nl6DQ9TVct6dx4iP9Pk66Oa6lymQjjj4OLyzV+M52+vsZNaWvda6+WHaW4qzUzafN51DLtDT3sI1fz9zzeMg3FfWhLBqZLa\\\\\\\/aP1zsCv73Lue4xNspmB5P\\\\\\\/iPHz0RiXir8xu0ypC2\\\\\\\/MaFnyTv5cjKD2igjg4zesnEUyD62AVZjylFucQ\\\\\\\/TrfwFQV13Z7AvMMsIWSBgXOXjbGNsoylloUiYdmAxOLt4SJz7G72Y9w3TEpa3BlH3K9UEs7bCW7eXFKsePL9Z3QjRkSVxOyGDL6q0bjMtZbr0IZTIM4ojLBpdkJ+0u+rUc\\\\\\\/UKeN453cnM+i3L1GdFMMYzlOQm4lNAGU91EY4buwVyZQQPkHsQhTO6dV+8CMyJ+zaz0\\\\\\\/K5z6Gsv+PwscP\\\\\\\/uAm3uPT\\\\\\\/VEJHZCaXqL6I5KeDYkF3UnIb1rka6tb4LUk9s7wpZw1OkEYVpBosMgaf6cMUzIPafCceNNsckIo6MCda2ywE22ElADer4KLwgoxbigYdQuEuzT5uxO2xW4RhGTgMFm8f885Idr+m97YPvuN48qtb5hkGRxMOSNPw2QOEFhfENBeGEWMJVup8SW+vithetKq1VrOP+8mgPWPdOk9NoJqQaU05tRgo6WaDNSTCpVEROquuSTMcjiA8Lx01a\\\\\\\/NxBeSzNPFoMlN2l10JluuKoE6SBokxltbrRjfVkh7y5vm+CZTsrq7udehr2xBned0cwKkqO9tSIGzkEqWcifsvvltBQbef\\\\\\\/IZV\\\\\\\/L3eiDwyR8lGNDbCoEWehbNobKGlHVP28d\\\\\\\/6Lqcj3KbV4Y+ESv+Rsnc92tOK6gKfQovtIrkZ3k3Ef5bJ8Nf4+iJlunGmv4Bzcvoi\\\\\\\/IHdU3rpVkgABjJxPeeAYj\\\\\\\/kFE+i3OQCu5UcOIg3S51aZAqZCyOiiGXDtr3b9pv0AUVo8WCB6v\\\\\\\/ZcAmxrXPCEtb0gVYiFZEL8qq8\\\\\\\/55KFV1N+iecR+l\\\\\\\/EuGhCX74S+huK+JysYayNkps24FQuFLIJaOTS4JR9eMfFmSlJqe3nYiuFL+A42MHjOp71ht9T0GClHqjO+\\\\\\\/hdZLh4fWqu7vvAtjeXNwBpUwq8\\\\\\\/0ifyWKe92J4rHSl56suqLGYtC4PUnNTNtPdIsh3i5zIJKgIMbtss02VsdmH7HI5nkQeHPthXYRaxa0w94H1c3nEwelzKcVySBR+VNZ6z9r2S8FurquSJPVkN18u+N8w8kiEqYdAhr51GB9PAADiYzwS3Y9O3ysvzH\\\\\\\/sFFGMW5LJ7a3zJOCEDRi4atRxrA22zTq97et4CO1GFwx6DC6aVS7+Iyv0tUCatCthfmbQp13mZqKyqyabGjUWfaR2bfDMdFKNSuNgIvudQDTGR8oPOL141RmiMAyf\\\\\\\/UuVGShTafWE\\\\\\\/ZmIrGnVe\\\\\\\/37HE9gQC3qHl8lVvYt3GKv1GP7hoaJSMuBAeE5+1MqWuqlB8Y2FidLmjR07wZ60CnegLkMsKv\\\\\\\/8JHCWaAAigEzXn9icGqR0S3djA20B9Ty0ruR4zSrwyEDPZpHwlgwY4SQYU\\\\\\\/DdvgmjdA\\\\\\\/RRYiMdkRieNWJVVl9k4MM8FNIh7ncDHkCh03b+FNoei52MBhIyqAST\\\\\\\/KcKZ8xuQXcOhHFAMe9HLxHI\\\\\\\/bPh50DwADcOO03k7Ek1tb7vvqjIZbJSJzlHKjpa8pHZp6QAv1q6Vk+C7uKUxzlHAambtfLV\\\\\\\/YO+jd2TDavYCZLDuywj+uxsO3DcPKsg3mzJXCKrMTPBintUpYKHWfHLrSndzsQjTE5fi059l3PlXG\\\\\\\/Ek3bD7IXqIAeFx+bKOxiKve9ZZFQ26EElsGQ6GMggx70GdwzhM1wlRdYA5F5AGHMnz8GUZZp9jluqZj8w0nJD0DGNvyjrF1buRZH+ponrlxv7gJiunKsqP9l9tZlyR3Bn0Y4UeGpnqziWw2AnSFu2HHinyIGE057oYa\\\\\\\/mN4CW5DkIoS3JxYbKW2XaK3dEerwGUuK4RcsSkRfCMkgu2WH4hJqp+k1RwiFYfrfpZqAf+hEfaQXYnr0FRA2WIXG77SjHBvb2Vt4bkyVaoQH3h0N8cSh197Unc555j0raHta+QYjRMdaktEge2rNmVSt36HF3QP5g6L\\\\\\\/cnvuk4Aw7bgglLiYjOO7hfkvBRpU0wJOscSGDezQ\\\\\\\/TeZ70+tKIyjvNr9xMFOa0W+NnToicrG30L8Ap38RR9Z\\\\\\\/iRPVY\\\\\\\/MkgosBr2hElpalaZMt1b7EVX1ctUy+kRQbZQn50I1YoDSAXW6oBsv99KQ6zUv2oYaE6ibYVxA8dw32yZcBXlk1OnnwsiM4T0n5Jxwc05a7IcHcv8nJjBNnlIMXSDsxoCEs\\\\\\\/jltr3cS4fX0DNSsULJFZRtejWEtFKkKek37RlxsCS05sWWXN5YFAJbHeyD4duno9fZDLFWqt6WXxxFCSOAEMTEzEg997emyTJ0c6SerEl0+CWL+ohM9oDxzFasSxMcpRIRnbbuDrnWd+SPmAD\\\\\\\/Xf77O4R17gY7Oz0C9aoZ7UDNPzisHTXcAlEusfxSJ0H1VD3ZN0EP4afzh2U4oN+Rjz4wmZ0IAJFOjuxwpyjPrkfp955iel6OmZlo7mGfHS5B303lL7ILdiB4dn8dbSnWO9ucEKaU5Gf4\\\\\\\/BLTCUItssQWLyE6zKXCknGmOq2ixoduNdeqiqyuy8ZVXnkpKZezsej7OkQMaOpaGbewAkOv+3+sbI+TnJ58yo8fmV8qfFgC3ufj6BjLvYARCtlRYvj+ZbZ7KAx6yg\\\\\\\/O\\\\\\\/vu+XwbjUI2jIxv2yH2D3Q6KiSixeeFlTEsv85hkAkI3lATe3OJzgBDpc9dHSfjNj9\\\\\\\/KaPSU5o36KORn1gljTqabCXix79Yb8PXXOhosDPzt7e+bKJC78yyp5MSa67X90v+WshH4btpYjJVfQHKjRY\\\\\\\/jxS9FWQsO68jOI8DY9UXakxZ9jJFQd9\\\\\\\/ngbGESNNEuRqHam+XPmlEeVMPkpy2BU3L62bsBrZvdwLVjB19\\\\\\\/5gOmO29O9NdvjykfF59sVsw1mgoGd+Np1vjTpunlUHbHSHYeJJX6LdXXcuVrizeNt3UtUIJCWRskpkhJfrDPLwiz4LuwOVvVPatfqZFO\\\\\\\/YpTD0JkZMWeAHeCtTSs9wV3r4cP3y50M6YK8x5MTJaLn+o6RaLjJVSW8pNAHuGUa7xRipJ5hNafEpYExs0G1fgcFcX4xqgruXpuAKWJXS0yOCJf0RLXHAxdIPLawl\\\\\\\/v2SCHHlElnkw9yeFYbzhTungl9uC\\\\\\\/Foocf8KHjPnOy\\\\\\\/vM67\\\\\\\/1wvRbmKHksAoBv9uLcmgqRVBcJWpsJK3k6fPI6JIZbwEyT2whqkBuHYt0b5CPKuv47\\\\\\\/xLeGRfm\\\\\\\/agl05QORWLu3RA+RHiEME9LaQBvkyIHhVzUHqB24R3d1rUkqWbMx0HHiM6Qmo4HksLD8f7p4ax+\\\\\\\/fygKqXqkWOjMogPaoCXMvQUm68MKctAy56HGz4i5vvhRAwXeK\\\\\\\/CYfRXOdEbPp+5aQRECzvCLZMlVMaHdwzu7MJzka44\\\\\\\/MW1FP13NzDBGcWjG3r2dbhhUH+61fSHRJThWkexLqO8FILV9VZK7xFHtkq19LGYs7A2nryMTvfZDNGTg7sgwaGfy4jI2URnavl7\\\\\\\/Sfxf+ZoJBCDhIlYdZKyVS8btnlGALK2DOEjEYPdj29HgFyY9vIVstUPImoZeupsv70jjj6gCJC8p8PeVZ6\\\\\\\/3jWTPnIxlnIv1yDzyXC889\\\\\\\/C8vAYv+3guTKAGbEoYyrphCpCB3o543fSp5Rgbq3UC19RzUQOeHXi1thtfKLkYxHVFHAz63bx7PfLQuU63pDDnMQVfvhrptF4U0udYAl2\\\\\\\/MbgGtNlfehTsZNMrG6xv3xIsV3Jdsyd1cJS2b1EHzvqwwCA2RdicrQKGYrrCZAV0oKUuXEgtBp3Y\\\\\\\/AL3pt70ty5N6cryjQc6uhuKJyaUSCXU94bu8w\\\\\\\/OsLWWPGCoH8Jx699yPLYOXIiXy0mMhnEZRp7TjtgiBFRSnSUFXeatQ93TLjHqJhDvtAPtWl7x6paxsAdFLGFruYkeiwOI3ikkD0Xpi8bKgrZKpgQsllj6JBYRnl9lU+84XqndNRL889kbRsmFMJc9UQ1mjyvOD+aY6HTNJwHZlyO2GX2heFRHbtTBCw2lkcER0kFVVfZnIdAJLeR4r1hMZqYSMp2\\\\\\\/AeJ\\\\\\\/JrOw3BPHD8zCOUmBOKvnsmHPHkubqA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sOHzYuiTADMwsKgcdddOog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e775718b75539eec8e5f39fa694c8f9d51d3e763baf262199d1d714b872483a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223dBD9g8LrYb+WSHBkVl+W1fO+m\\\\\\\/7mo8J1dSYlQ\\\\\\\/yvFOsd5qFPuHQ5MYlBPbCbWQDTT6HZR1Xrfp6A6Pwqe4BDBXY30h+2K1RyjnfBYAf8OHJ8M1ktFPiVBIvvkiI1vyC+\\\\\\\/wu3Cb808hgQQM8HOtmVU0r\\\\\\\/d7vnNfrycriCvXIu7kTfladRnnlq\\\\\\\/oJSepzcjLzWufReTTQwQmoCkumm4OzCZWmlm6Ngnl0sGZBBML6n6Qb3Wjd\\\\\\\/PZaapjmFBA+T3P0\\\\\\\/D34Q5Fb\\\\\\\/59E4yyWhmwfiDkGn6tcO4UBb6zbijrHSIvfRwPcETeyqIgkv5OdVPB1G2oR8TqXra0ORu3g44FJzDACu8dWQEvOJ5OGrkvPDhWu4YWncLpuoABfyYBggOxfNroTl38UZIafRqVufajJoGMTcdnZvHvdRAXFCTRBWhPh5FbHSnbmSFPLoGBVzEVuXML4MwIkINUDgU\\\\\\\/MJsPfEgSn4LuvWzJwZVpb1JDyXaBEqbVRuOo5LNucPjwm8wggJq\\\\\\\/8ek6ZdfqIsEyBhGlCZkpbdRnxZ0pJ46cTE6rjd1htp64lZizcHOCaT33\\\\\\\/gNHiP+kuwlIulb9oeL\\\\\\\/QRNerZ0rvrXoc\\\\\\\/PzeGoW+\\\\\\\/oplchax4tAD1ohjOd5K0bj+u8drEKvP4X5Ko7HzmcLHZBeN35HSxy\\\\\\\/s4FoqVLo3oy1VOGZCRseF+iTmZj966h2q4k+CwFHUzMV4aNDvRlVC8MMBJPd8kF+KjrvgovsuwzNfFUbjyVFc3MUFDfWcn\\\\\\\/B5GdWSnCZl6vtuc+uw3PatoWGqeDPqc5rDujli4axKF6mp6aiX1E3bEj7wKAREUulHGIBy78qB1zkq3WmD2cka\\\\\\\/FgpAX8\\\\\\\/SNA2F3I8mFbOpfk45l3vz9AtauzGn\\\\\\\/enB10TBHegJUHQT7tpFxb3WdkliTZM9n6OahSoD+hSLMXw97oe6E30mLY0HKe38tSpXqaRgetBt5z4EDn\\\\\\\/dJDBEF6k5UeTDOtFc8PRoURMaLJ3Kf\\\\\\\/0Ei3r4dBQqp0WrS828yNtG1ll9Uci83WA9GtfPgRUnB0R9qQqrqffg1YtIOiA5d28R9oVwJv8NVZT7\\\\\\\/T6luGfUFMtua\\\\\\\/hBvn+0BZ3PqmBmodu6bIqXfTqNy6kkMSPLPlywCxFhkQx0wNZnyIXKEV18x8j432yEHAoqAnlCBaJiqmWVDz81TF\\\\\\\/VQbhYH4CAVyfHrqPO1F3yVpqERGvTPZz5+QpMX2Qq7c6Ke9vpKuC6shZcR5tq47793z1AOa1mKF6dwUlUgHSRdbKVHFMdF6iEaJgHOPAK968LDBipC7j3vkjykz081dmLFROUYlCEN5dfTUEDoZNsb4nggDnl11Q7N5jd4JSdhcNrA\\\\\\\/tgfSgWcP3VO9jFW3iecWbDafivNDiOx5jssv3TFQJgsdhtabcUgJ8lVFUs8\\\\\\\/PMySW7S5tk7v+rYLMAbU7\\\\\\\/9PXmK\\\\\\\/N+nA3K03iwMcGyH4P7EVHGcJsx6hBJkuJZHg1hczvm3QLvs9yz80i4Ps8AlFNVuPu9XKaf7I9\\\\\\\/piMuNsBre+26t7uqi2myz2Cu5CXHv+qMOU9LqLMGJ\\\\\\\/Bp\\\\\\\/v7YKtsEyAnZe+NC1vHKcWJ0GbeGmDZU3Tb+r7Oqy8WGO6AdtUXXpYCf1eKZoBBiAEmcUgfGigUrhkxNU1uLPj1bLMMrUPKRktC3mpUh++PlXsfnBe1fnsxswqErm13HfLWu\\\\\\\/vmrUbCfmGiJPyT4Qf2IwWVG4Mk0yy38ZjLmMGcLaX3PXOW8fvdGv\\\\\\\/7pDD9xV+u500LUB5mhlABwxm0Fg\\\\\\\/EimdfcEumKMRRlUUVofy3OwwoZtCVx3LJV4h0Au2JRmbQt0qkDYb400SpOsJd6JCOaxmpjtN+rgoWFr1lSGmWSTKjOqZnX\\\\\\\/GWquRk+qkCCicLJY9pJTIzhDyK1roz2JEWNFll6\\\\\\\/P9ZZpoydlVIO4H2\\\\\\\/L46knbB6w6AgcMqdkhCb2LP3+kpschz5Y3ZL7Tov0S2LUErhlKUq9NoTX4IzkbRFITOtxxL0uBunz0JQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ff6b0ENKOmt8HBLRlo7Ojw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f025087c80de46704142d95fc5f60677c506d0c48ab7da211dbba5bd2678338f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459887 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/911" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eSiWvpK_CfqlNJ9GlDKQSCGA908ihHSMIKwoZxVCLxs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/911" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LJnJH5xiHpApuXRlF3RbA1V5GcfmBYbawr1k7_ocuGo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/891" title="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/921" title="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459887" id="node459887" data-pisa="ggg;5/5/911" data-pisa-master="ggg;g3.115.017178" data-apath="/ggg/5/5/911.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Powerful New Quantitative Genetics Platform, Combining <em>Caenorhabditis elegans</em> High-Throughput Fitness Assays with a Large Collection of Recombinant Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erik C. Andersen</span>, <span class="highwire-citation-author" data-delta="1">Tyler C. Shimko</span>, <span class="highwire-citation-author" data-delta="2">Jonathan R. Crissman</span>, <span class="highwire-citation-author" data-delta="3">Rajarshi Ghosh</span>, <span class="highwire-citation-author" data-delta="4">Joshua S. Bloom</span>, <span class="highwire-citation-author" data-delta="5">Hannah S. Seidel</span>, <span class="highwire-citation-author" data-delta="6">Justin P. Gerke</span> and <span class="highwire-citation-author" data-delta="7">Leonid Kruglyak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">911-920;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017178 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459887" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erik C. Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BC.%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2BC.%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler C. Shimko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BC.%2BShimko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shimko%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BC.%2BShimko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan R. Crissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BR.%2BCrissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crissman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BR.%2BCrissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajarshi Ghosh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajarshi%2BGhosh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ghosh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajarshi%2BGhosh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joshua S. Bloom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BS.%2BBloom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bloom%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BS.%2BBloom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hannah S. Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BS.%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHannah%2BS.%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Justin P. Gerke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BGerke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerke%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BP.%2BGerke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonid Kruglyak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonid%2BKruglyak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kruglyak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonid%2BKruglyak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/911" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459887" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/911.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459887" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/911.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459887" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/911.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459887" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/911.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459887" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">The genetic variants underlying complex traits are often elusive even in powerful model organisms such as <em>Caenorhabditis elegans</em> with controlled genetic backgrounds and environmental conditions. Two major contributing factors are: (1) the lack of statistical power from measuring the phenotypes of small numbers of individuals, and (2) the use of phenotyping platforms that do not scale to hundreds of individuals and are prone to noisy measurements. Here, we generated a new resource of 359 recombinant inbred strains that augments the existing <em>C. elegans</em> N2xCB4856 recombinant inbred advanced intercross line population. This new strain collection removes variation in the neuropeptide receptor gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>, known to have large physiological and behavioral effects on <em>C. elegans</em> and mitigates the hybrid strain incompatibility caused by <em><a href="http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene">zeel-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em>, allowing for identification of quantitative trait loci that otherwise would have been masked by those effects. Additionally, we optimized highly scalable and accurate high-throughput assays of fecundity and body size using the COPAS BIOSORT large particle nematode sorter. Using these assays, we identified quantitative trait loci involved in fecundity and growth under normal growth conditions and after exposure to the herbicide paraquat, including independent genetic loci that regulate different stages of larval growth. Our results offer a powerful platform for the discovery of the genetic variants that control differences in responses to drugs, other aqueous compounds, bacterial foods, and pathogenic stresses.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C. elegans</em></a></li><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/high-throughput-phenotyping-0" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-phenotyping" rel="nofollow">high-throughput phenotyping</a></li><li class="kwd"><a href="/keyword/fitness-assays" class="hw-term hw-article-keyword hw-article-keyword-fitness-assays" rel="nofollow">fitness assays</a></li></ul><p id="p-7">The nematode <em>Caenorhabditis elegans</em> is emerging as a powerful model to connect quantitative traits to genetic variants (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaertner and Phillips 2010</a>). These quantitative trait genes are easily identified because of favorable attributes of the organism, including a selfing hermaphroditic lifestyle, the ability to generate large numbers of recombinant offspring, the ability to cryopreserve and revive strains indefinitely, and ample phenotypic variation.</p><p id="p-8">To facilitate quantitative genetic mapping, a variety of strain resources have been created. Two crosses between the laboratory-adapted Bristol (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) strain and a wild strain from Hawaii (<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>) are available, comprising one panel of 200 F<sub>2</sub> recombinant lines (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2006</a>) and one panel of 239 F<sub>10</sub> advanced intercross lines (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>). These two sets of strains and a set of nearly isogenic lines, or NILs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doroszuk <em>et al.</em> 2009</a>), are proving to be powerful community resources to connect phenotype to genotype because they are freely available and can be used indefinitely. Additionally, a growing collection of wild <em>C. elegans</em> strains facilitates genome-wide association studies (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>).</p><p id="p-9">Unfortunately, these recombinant inbred lines and NILs are not optimal for quantitative trait mapping for two reasons. First, before the creation of these strain resources, the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strain underwent a long period of laboratory adaptation. During this time, alleles accumulated that promoted survival under laboratory conditions, including an allele of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> gene that encodes a neuropeptide receptor with large physiological and behavioral effects (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). Because of the widespread effects of this allele under laboratory conditions, the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> gene is often detected as causal when mapping quantitative traits in any panel of recombinant strains that include the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> version of the gene. Second, both panels of recombinant strains constructed between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> have skewed allele frequencies on the left arm of chromosome I (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2006</a>). This skew is the result of a genetic incompatibility that causes strains without protection of the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-provided <em><a href="http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene">zeel-1</a></em> gene to be killed by the toxic effects of the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>-provided <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> gene (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Seidel <em>et al.</em> 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2008</a>). Mapping panels that eliminate <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> variation and the allele frequency skew are highly desirable.</p><p id="p-10">To identify many of the alleles underlying complex traits, the phenotypes of a large number of independent strains need to be measured accurately. Recent efforts in yeast have shown that cross populations as large as one thousand individuals can identify most of the loci that contribute additively to phenotypic differences in a population (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bloom <em>et al.</em> 2013</a>). The <em>C. elegans</em> community requires a large panel of recombinant strains to increase the statistical power to detect more alleles of ever decreasing effect sizes. However, the majority of assays used in this model organism are optimized for the measurement of a small number of strains, because most research groups focus on the laboratory strain. To connect phenotype to genotype for complex traits, we require a larger panel of recombinant strains and high-throughput assays to measure the phenotypes of a large number of independent strains.</p><p id="p-11">Here, we describe the creation of 359 additional N2xCB4856 recombinant inbred advanced intercross lines (RIAILs) and the optimization of high-throughput fitness assays to measure phenotypes from hundreds of independent strains in parallel. The strains in our new RIAIL panel all have the Hawaiian version of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> and have a transposon insertion into the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> gene encoding the incompatibility toxin. These strains do not have the large phenotypic differences caused by variation in the laboratory-adapted allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> and have a reduced allele frequency skew at the incompatibility region on chromosome I. Additionally, we took advantage of the large particle nematode sorter made by Union Biometrica to measure the numbers, lengths, optical densities, and fluorescence of large populations of offspring derived from over a hundred independent lines per day. We optimized liquid growth conditions to massively scale the assays and to significantly reduce environmental heterogeneity normally present in standard laboratory agar plate assays. Using these high-throughput fitness assays, we measured a large number of parameters in our new panel of 359 RIAILs in control conditions and after exposure to the herbicide paraquat. We identified quantitative trait loci (QTL) that individually explain as little as 4% of trait variance, showing the statistical power of this large mapping panel. We believe that this new strain resource, along with a scalable and accurate high-throughput phenotyping platform, will further strengthen <em>C. elegans</em> as a powerful organism to understand the functional consequences of natural variation in metazoans.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-12">Animals were cultured and all crosses performed at 20Â° with the bacterial strain <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> on modified nematode growth medium containing 1% agar and 0.7% agarose to prevent burrowing of strains with the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> version of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>. For each assay, strains were grown for at least five generations with no strain entering starvation or encountering dauer-inducing conditions. QX1430 contains the nearly isogenic region <em>qgIR1 [X</em>, <em><a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> &gt; <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a></em>, <em>4,754,307-4,864,273]</em> with the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> version of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> and a transposon insertion <em><a href="http://www.wormbase.org/db/get?name=WBVar00256987;class=Variation">ttTi12715</a></em>, which reduces the function of <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> rendering the strain compatible with <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>. Other transposon insertions present in the parent strain <a href="http://www.wormbase.org/db/get?name=IE12715;class=Strain">IE12715</a> were outcrossed to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and confirmed lost by inverse PCR. Throughout the manuscript and this analysis, QX1430 was used in place of the Bristol strain. The Bristol (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) strain has many other laboratory-derived variants, including <em><a href="http://www.wormbase.org/db/get?name=WBGene00015964;class=Gene">glb-5</a></em>. Because most have unknown effects on the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> phenotype, we did not and could not correct them all.</p></div><div id="sec-3" class="subsection"><h3>Recombinant inbred line construction by advanced intercross</h3><p id="p-13">We constructed RIAILs from reciprocal crosses between QX1430 and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>. The reciprocal crosses yielded two classes each of male and hermaphrodite progeny that differed with respect to their mitochondrial and X chromosomes. Each of the four possible crosses among the four F1 classes was performed, yielding two classes of hermaphrodite and two classes of male F2 progeny. Each of the four possible crosses from the F2 classes was then performed. F3 progeny were randomly single-pair mated each generation for 10 generations. Each pair contributed equally to each generation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Rockman and Kruglyak 2008</a>).</p><p id="p-14">Each cross plate constituted a single-pair mating between one male and one hermaphrodite. With each generation, a proportion of the crosses failed due to poor male mating. An absence of males in the population of progeny indicated the failure of a cross. Additionally, crosses failed when animals crawled to the edge of the plate and were lost. The proportion of failed crosses was reduced through preparation of three redundant plates for each cross. One line resulting from each cross was then propagated for ten generations through the random selection and selfing of a hermaphrodite. Each randomly selected hermaphrodite was picked with twofold redundancy to reduce the loss of lines during the selfing phase. These strains are available upon request from the Andersen laboratory.</p></div><div id="sec-4" class="subsection"><h3>Genotyping recombinant inbred lines</h3><p id="p-15">After RIAIL strains were generated, we froze a strain copy and collected DNA in parallel from the same culture plates. We collected and cleaned DNA using the DNeasy Blood and Tissue Kit (QIAGEN) from populations of animals grown on three 10-cm plates seeded with <a href="http://www.wormbase.org/db/get?name=X1666;class=Strain">X1666</a> <em>E. coli</em> bacteria. The strains were genotyped using the Illumina GoldenGate assay as described previously (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>). Raw data from the Illumina reader were read into and processed in R. We used normal mixture modeling via the expectation-maximization (EM) algorithm in the <em>mclust</em> R package to determine the genotype of each RIAIL at each single-nucleotide polymorphism (SNP) assuming two groups. This method assigned genotype data for all 1536 SNPs. SNP UCE2-2369 was excluded from analyses because of poor genotyping on the GoldenGate platform. A total of 7017 of 589,824 genotype calls were set to not applicable (NA) because of poor differentiation into either an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> call. To keep our new RIAIL panel compatible with the previous panel, we selected the same 1454 SNPs genotyped in the original set of N2xCB4856 RIAILs and processed those SNPs. SNPs with allele frequencies greater than 65% <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or 60% <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> were set to NA. We imputed missing and poorly assigned genotypes using the hidden Markov model implemented in the <em>qtl</em> package in R (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Broman <em>et al.</em> 2003</a>). Five SNPs (UCE1-1433, UCE5-3001, CE5-218, UCE6-686, UCE6-1285) were set to NA because the allele frequency skew and later imputation made assignment errors. Re-imputation led to uniform recombination maps. Our genotyping code, raw files, processed data (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FileS1.csv">File S1</a>), a new R/qtl cross object, a marker liftover file, and code to merge phenotypes into this cross object can be found on the Andersen lab GitHub page.</p></div><div id="sec-5" class="subsection"><h3>High-throughput fitness assays of fecundity and body size</h3><p id="p-16">Animals were grown in 96-well microtiter plates to allow for automated liquid handling to speed assay set up and phenotyping. In each well, the final volume was 50 ÂµL of S medium solution and <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> bacterial food. <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> was prepared in large batches (greater than 20 L) to reduce assay-to-assay variability. The bacteria were prepared from cultures grown for 24 hr in Superbroth and then centrifuged to concentrate. <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> bacteria were resuspended at 20% volume per volume in S medium and frozen at â80Â° (without glycerol) in 1-mL aliquots. Bacteria were thawed and fed to animals at a final concentration of 2% volume per volume. This amount of food was sufficient to sustain the number of offspring derived from a single fourth larval stage hermaphrodite with food left over at the end of the assay. For any mapping or causality experiment, we used the same <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> culture to reduce the substantial variation that arises from bacteria grown in independent preparations. Solutions of S medium and bacterial food were prepared and then split for control and paraquat conditions.</p><p id="p-17">After assay plates were prepared, one mid to late fourth larval stage animal was singled using a platinum wire to each 50-ÂµL well. In the plate set up, we separated different genotypes with a wash well containing only S medium to decrease carry-over and mixing of independent genotypes from one well to the next. After all wells were populated with animals, the microtiter plate was sealed with a Breath-Easy film (USA Scientific) and placed in a humidity chamber lined with damp towels, closed, and then sealed with parafilm. We observed less than 5% of the well volume evaporated after four days under these conditions. Humidity chambers were placed into incubators set to 20Â° and shaking at 180 rpm. These conditions ensured that cultures and bacterial food was constantly mixing so animals would never enter hypoxia nor encounter regions of depleted food in the well. Animals were grown for 96 hr in these conditions and then prepared for measurement on the COPAS BIOSORT (Union Biometrica).</p><p id="p-18">Two minutes before the animals were loaded on the COPAS BIOSORT, 200 ÂµL of M9 with 50 mM sodium azide was added to the wells. The sodium azide kills and straightens the animals to ensure proper measurement of body length as the paralyzed animal passes through the flow cell. The COPAS BIOSORT sheath flow rate was kept constant at 9.8 mL per minute to decrease variability in length measurements as much as possible. Then, 96-well microtiter plates were aspirated using the ReFLx module with the BISORT set to âno-bubble-trapâ mode. When the system is run in âno-bubble-trapâ mode, all wells from a single microtiter plate can be measured in approximately 25 min and well-to-well contamination is less than 1%, which is further mitigated by the use of wash wells between unique genotypes. Extinction and time of flight minimums were 50 and 20, respectively. Green, yellow, and red photomultiplier tubes were set to 700, 700, and 900, respectively. Signal multipliers were set to 1.0 and signal gains set to 3.0. The COPAS BIOSORT software had Profiler II enabled. Flat comma-separated value files were analyzed using custom scripts in the R statistical computing environment. The COPAS BIOSORT software can not differentiate bubbles from animals as objects pass through the flow cell. For this reason, we trained a support vector machine to differentiate these two types of objects with 99.97% accuracy (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/017178SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS1.pdf">Figure S1</a>). The support vector machine primarily uses optical density raw values to generate a binary linear classifier separating bubbles from animals. The raw data were read in, processed, and plotted using the <em>COPASutils</em> R package (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Shimko and Andersen 2014</a>). All of the raw and processed data (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FileS2.csv">File S2</a>) can be accessed on the Andersen lab GitHub page. High-throughput assay plates with paraquat were prepared at 1.5 mM paraquat (Chem Service, Inc.) or no paraquat from the same dilution of <a href="http://www.wormbase.org/db/get?name=HB101;class=Strain">HB101</a> and S medium.</p></div><div id="sec-6" class="subsection"><h3>Linkage mapping</h3><p id="p-19">A total of 357 RIAILs were phenotyped in the high-throughput assays described previously for both control and paraquat conditions. The phenotype data and genotype data were entered into R and scaled to have a mean of zero and a variance of one for linkage analysis. QTL were detected by calculating logarithm of odds (LOD) scores for each marker and each trait as -n(ln(1-r^2)/2ln(10)), where r is the Pearson correlation coefficient between RIAIL genotypes at the marker and phenotype trait values (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bloom <em>et al.</em> 2013</a>). The maximum LOD score for each chromosome for each trait was retained from three iterations of linkage mappings (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FileS3.csv">File S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FileS4.docx">File S4</a>). We randomly permuted the phenotype values of each RIAIL while maintaining correlation structure among phenotypes 1000 times to estimate significance empirically. The ratio of expected peaks to observed peaks was calculated to determine the 5% false discovery rate of LOD 2.97.</p><p id="p-20">Broad-sense heritability was calculated as the fraction of phenotypic variance explained by strain from fit of a linear mixed-model of repeat phenotypic measures of the parents and RIAILs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> 2005</a>). The total variance explained by each QTL was divided by the broad-sense heritability to determine how much of the heritability is explained by each QTL. Confidence intervals were defined as the regions contained within a 1.5 LOD drop from the maximum LOD score.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>An expanded collection of RIAILs</h3><p id="p-21"><em>C. elegans</em> quantitative genetics was accelerated by the creation and use of an advanced intercross recombinant inbred line collection between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (Bristol) and Hawaii (<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>) strains or N2xCB4856 RIAILs (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>). This collection of strains facilitated the linkage mapping of a large number of diverse traits (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bendesky <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gaertner and Phillips 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gaertner <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ghosh <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Glater <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">McGrath <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Palopoli <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pollard and Rockman 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Reddy <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Seidel <em>et al.</em> 2008</a>). Despite these results, this RIAIL panel had three distinct disadvantages. First, a genetic incompatibility between the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and Hawaii strains led to a large genotype skew on chromosome I, where nearly all of the strains in the panel have the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allele. Second, because the cross was performed at 25Â°, the strain collection had high levels of sickness and some strains were lost. To combat the strain loss, each strain was duplicated after the intercrossing phase to make a panel where half of the population is highly related to the other half. Third, the laboratory-derived allele at <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> has a large effect on the growth and physiology of <em>C. elegans</em> because of behavioral changes. These changes do not represent the natural behavior, growth, or physiology of the species (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). We set out to augment this existing resource while also correcting these three disadvantages.</p><p id="p-22">To deal with the strain incompatibility and the laboratory-derived allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>, we created a new strain to use as the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> parent. For the strain incompatibility, the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> gene is necessary and sufficient to kill animals that do not have a functioning <em><a href="http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene">zeel-1</a></em> gene product (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Seidel <em>et al.</em> 2011</a>). If we eliminate the function of <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em>, the strain will be compatible with any other strain, including the Hawaii strain. We took advantage of an existing transposon insertion into the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> gene, which is predicted to eliminate the function of <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em>. We found this strain to be compatible with both the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and Hawaii strains (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS2.pdf">Figure S2</a>). For the laboratory-derived allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>, we used the nearly isogenic region present in <em>qgIR1</em>, where the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> genomic region from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> is replaced with the equivalent genomic region from the Hawaii strain. This new strain, named QX1430, is compatible with the Hawaii strain and has the natural allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em>.</p><p id="p-23">After the creation of QX1430, we performed an advanced intercross identical to the design previously implemented (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>), except we crossed animals at 20Â° to reduce temperature-sensitive sickness. In parallel, these strains were frozen for permanent storage and DNA prepared for genotyping. We genotyped the strains using the same polymorphic markers as the previous set. We determined the genotypes of the new recombinant inbred line panel using custom scripts described previously (see <em>Materials and Methods</em>). Our new collection of RIAILs comprises 359 independent strains. We refer to this panel as a N2xCB4856 panel to indicate the primary genetic background of the QX1430 parental strain.</p><p id="p-24">Surprisingly, we still observed an allele frequency skew on chromosome I that was less pronounced than in the previous N2xCB4856 RIAIL collection (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS3.pdf">Figure S3</a>) but greater than expected from random drift (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>). We showed that the skew is caused by a reduced but still present genetic incompatibility (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS2.pdf">Figure S2</a>). These results suggest that other genes besides <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> might weakly contribute to the incompatibility at that genomic location or that the function of <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> is not completely abrogated by the transposon insertion. We also noted that other weaker allele frequency skews on other chromosomes in the original RIAIL collection showed the opposite skew in the new collection. These skews are within the expectations for random drift (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>).</p></div><div id="sec-9" class="subsection"><h3>Implementation of high-throughput assays to measure fecundity and body size</h3><p id="p-25">As a scientific community, <em>C. elegans</em> researchers have few assays that can measure phenotypes for hundreds of independent strains quickly and reproducibly. Most phenotypic assays use a small number of strains in a large number of conditions because the majority of investigators only use the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genetic background. To understand the underlying allelic variants that contribute to complex traits, we require assays that can measure a large number of independent strains. Most chemical perturbations to an organism affect the number of offspring produced and the growth rate of animals. For these reasons, we created high-throughput assays to measure fecundity and body size for hundreds of independent strains quickly and accurately.</p><p id="p-26">Previously, we found that measuring fecundity and body size by using standard laboratory agar plate-based methods would not work well for a large number of independent strains (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). We aimed to develop a protocol with little human input for preparation of the assay and no human input in the measurement of the traits. For these reasons, we optimized a protocol in which researchers prepare the assay in microtiter plates and the COPAS BIOSORT platform (Union Biometrica) automatedly measures the traits of interest (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In brief, hermaphrodites are singled into wells of a microtiter plate. Four days later, the clonal offspring of those single hermaphrodites are aspirated by the COPAS BIOSORT system outfitted with a ReFLx probe. The length (time of flight), optical density (extinction), and three fluorescence parameters of every nematode in every well are measured. From these measurements, the growth distribution among the offspring is determined. Additionally, the number of offspring per original hermaphrodite can be obtained by summing the total number of objects in each well of the microtiter plate. Raw data from the COPAS platform are read in, processed, and analyzed using a new R package called <em>COPASutils</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Shimko and Andersen 2014</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An overview of the high-throughput analysis pipeline is shown. Animals are grown on plates for five generations without starvation and then single hermaphrodites are picked to wells of a 96-well microtiter plate. The microtiter plate either has no compound (control) or paraquat added. Animals are grown for 96 hr at which point M9 solution with sodium azide is added to kill the nematodes and straighten them out. Each well is aspirated using the ReFLx module of the COPAS BIOSORT and animals are measured for length (time of flight), optical density (extinction), and three fluorescence measures. Raw data are read in, processed, and plotted using the COPASutils R package. Example length and optical density data are shown from one 96-wel plate with animals in every well except wells in columns 4, 8, and 12." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;An overview of the high-throughput analysis pipeline is shown. Animals are grown on plates for five generations without starvation and then single hermaphrodites are picked to wells of a 96-well microtiter plate. The microtiter plate either has no compound (control) or paraquat added. Animals are grown for 96 hr at which point M9 solution with sodium azide is added to kill the nematodes and straighten them out. Each well is aspirated using the ReFLx module of the COPAS BIOSORT and animals are measured for length (time of flight), optical density (extinction), and three fluorescence measures. Raw data are read in, processed, and plotted using the COPASutils R package. Example length and optical density data are shown from one 96-wel plate with animals in every well except wells in columns 4, 8, and 12.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F1.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/911/F1.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466126" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">An overview of the high-throughput analysis pipeline is shown. Animals are grown on plates for five generations without starvation and then single hermaphrodites are picked to wells of a 96-well microtiter plate. The microtiter plate either has no compound (control) or paraquat added. Animals are grown for 96 hr at which point M9 solution with sodium azide is added to kill the nematodes and straighten them out. Each well is aspirated using the ReFLx module of the COPAS BIOSORT and animals are measured for length (time of flight), optical density (extinction), and three fluorescence measures. Raw data are read in, processed, and plotted using the COPASutils R package. Example length and optical density data are shown from one 96-wel plate with animals in every well except wells in columns 4, 8, and 12.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">This liquid-handling platform worked well for our purposes, but parameters needed to be optimized to make it a useful quantitative assay. We prepared bacterial food in large batches and froze aliquots to make the food source reproducible across many months. Plates were housed in a humidity chamber with constant mixing and temperature. Our fecundities and growth rates are highly correlated with measurements from agar plates, indicating that <em>C. elegans</em> can thrive when grown in liquid conditions as long as bacterial food is distributed evenly. We found that intra-assay and inter-assay correlations for fecundity were substantially greater than fecundity assays using standard agar plate-based methods (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS4.pdf">Figure S4</a>). These results indicate that these fecundity and body size assays are much more reproducible, more accurate, and higher throughput than standard <em>C. elegans</em> laboratory methods.</p></div><div id="sec-10" class="subsection"><h3>High-throughput assays of fecundity and animal size generate rich multivariate traits with distinct underlying genetic causes</h3><p id="p-29">The raw data from the high-throughput assays described above provide a variety of measurements relating to fecundity and body size. We generated summary statistics from these measurements and treated the summary statistics as traits for quantitative trait mapping. For fecundity, we mapped the total number of animals in each well. For body size, the raw data provide five relevant parameters: time of flight, optical density, and red, yellow, and green fluorescence. Because fluorescence values were low, we did not analyze those data further. Time of flight is a direct measurement of body length, whereas the optical density parameter is influenced by body length, thickness, and composition. For each parameter, we calculated eight summary statistics: the average, variance, interquartile range, 10th, 25th, 50th (median), 75th, and 90th quantiles. Next, because body length is highly correlated with optical density (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS5.pdf">Figure S5</a>), we also normalized the latter for body length and re-calculated the summary statistics. These normalized values better capture the contributions of body thickness and body composition to optical density. Additionally, the normalized optical density summary statistics are less correlated with length and optical density parameters. This data processing pipeline produces a single measurement of fecundity and 24 summary statistics related to body size (two raw parameters plus one normalized parameters times eight summary statistics). Finally, when chemical perturbations were measured along with paired controls (see next section), we performed an additional analysis where we used linear regression to reduce the contribution of the control trait to the perturbed trait.</p><p id="p-30">We measured fecundity and all 24 body size traits for 357 of our N2xCB4856 RIAILs and a large number of QX1430 and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> parent replicates. Many of the 24 body size traits were highly correlated with each other (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS5.pdf">Figure S5</a>). For fecundity, 49.5% of trait variance in the population can be ascribed to genetic causes. Strains QX1430 and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> differed in their fecundity, and the phenotypes of the RIAILs mostly ranged between the two parents (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). This difference maps to a single significant QTL on chromosome IV, which explains 12% of the phenotypic variance (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Tukey box plots of N2 (orange), CB4856 (blue), and recombinant inbred advanced intercross lines (RIAILs; gray) normalized fecundity data. The total number of offspring from twenty replicates of each parent and 357 RIAILs were counted. The fecundities of the RIAILs show transgressive segregation. (B) Linkage mapping results of normalized fecundity in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Tukey box plots of N2 (orange), CB4856 (blue), and recombinant inbred advanced intercross lines (RIAILs; gray) normalized fecundity data. The total number of offspring from twenty replicates of each parent and 357 RIAILs were counted. The fecundities of the RIAILs show transgressive segregation. (B) Linkage mapping results of normalized fecundity in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F2.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/911/F2.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465988" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">(A) Tukey box plots of N2 (orange), CB4856 (blue), and recombinant inbred advanced intercross lines (RIAILs; gray) normalized fecundity data. The total number of offspring from twenty replicates of each parent and 357 RIAILs were counted. The fecundities of the RIAILs show transgressive segregation. (B) Linkage mapping results of normalized fecundity in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Next, we mapped each of the body size traits separately and obtained 47 significant QTL, which correspond to at least six distinct loci (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS6.pdf">Figure S6</a>). Some loci were detected multiple times for sets of correlated traits. As an example of the type of data we observe, representative distributions of body lengths are shown (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Importantly, we find that the genetic architectures of traits change as different summary statistics for the same parameter are mapped. For example, as mappings of body length proceed from the 10th through the 90th quantiles, QTL appear and disappear on chromosomes III, IV, and V: the 10th quantile of body length mapped to a significant QTL on chromosome III (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>); in mappings of the 25th and 50th quantiles, a different QTL emerges (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>); and in mappings of the 75th and 90th quantiles, yet a third QTL emerges (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, E and F</a>). Thus, different QTL influence different subpopulations of the body length distribution. Because body length is a convenient proxy for developmental stage, these results likely indicate that distinct genetic loci influence different developmental stages of animals in a population. As another example of genetic architecture changing as different summary statistics are mapped, we find that QTL for optical density and body length largely overlap (as expected, given their tight correlation), but new QTL emerge when we map a normalized optical density trait (the optical density of each animal divided by its length, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Taken together, these data show that using these high-throughput assays and our new RIAIL collection, we can rapidly and accurately map variants that control a variety of growth rate-related traits in <em>C. elegans</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary statistics of body length map to different genomic regions implicating unique genetic variants. (A) Histograms of the length of every animal measured for 10 randomly chosen RIAILs are shown. The data for each RIAIL are in separate boxes with the strain name in bold above. The colored vertical lines denote population summary statistics, 10th quantile (red), 25th quantile (orange), median (yellow), 75th quantile (blue), 90th quantile (purple), and interquartile range (gray). The color of the vertical line matches the color of the chromosome label titles on top of the linkage mapping results for each trait. The genetic architectures differ when variation of size trait changes. B. Linkage mapping results of the 10th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Linkage mapping results of the 25th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (D) Linkage mapping results of the 50th quantile or median of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. E. Linkage mapping results of the 75th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. F. Linkage mapping results of the 90th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. G. Linkage mapping results of the variance of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary statistics of body length map to different genomic regions implicating unique genetic variants. (A) Histograms of the length of every animal measured for 10 randomly chosen RIAILs are shown. The data for each RIAIL are in separate boxes with the strain name in bold above. The colored vertical lines denote population summary statistics, 10th quantile (red), 25th quantile (orange), median (yellow), 75th quantile (blue), 90th quantile (purple), and interquartile range (gray). The color of the vertical line matches the color of the chromosome label titles on top of the linkage mapping results for each trait. The genetic architectures differ when variation of size trait changes. B. Linkage mapping results of the 10th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Linkage mapping results of the 25th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (D) Linkage mapping results of the 50th quantile or median of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. E. Linkage mapping results of the 75th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. F. Linkage mapping results of the 90th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. G. Linkage mapping results of the variance of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F3.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/911/F3.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Summary statistics of body length map to different genomic regions implicating unique genetic variants. (A) Histograms of the length of every animal measured for 10 randomly chosen RIAILs are shown. The data for each RIAIL are in separate boxes with the strain name in bold above. The colored vertical lines denote population summary statistics, 10th quantile (red), 25th quantile (orange), median (yellow), 75th quantile (blue), 90th quantile (purple), and interquartile range (gray). The color of the vertical line matches the color of the chromosome label titles on top of the linkage mapping results for each trait. The genetic architectures differ when variation of size trait changes. B. Linkage mapping results of the 10th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Linkage mapping results of the 25th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (D) Linkage mapping results of the 50th quantile or median of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. E. Linkage mapping results of the 75th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. F. Linkage mapping results of the 90th quantile of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. G. Linkage mapping results of the variance of body length in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Histogram of the normalized median body lengths of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (B) Linkage mapping results of median body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Histogram of the normalized median optical densities of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (D) Linkage mapping results of median optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (E) Histogram of the median normalized optical densities (optical density of each animal divided by its length) of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position before normalizing for body length. (F) Linkage mapping results of median normalized optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. After normalizing for body length additional variation is revealed and maps to chromosomes III and X. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Histogram of the normalized median body lengths of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (B) Linkage mapping results of median body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Histogram of the normalized median optical densities of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (D) Linkage mapping results of median optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (E) Histogram of the median normalized optical densities (optical density of each animal divided by its length) of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position before normalizing for body length. (F) Linkage mapping results of median normalized optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. After normalizing for body length additional variation is revealed and maps to chromosomes III and X. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F4.medium.gif" width="372" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/911/F4.medium.gif" width="372" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466088" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">(A) Histogram of the normalized median body lengths of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (B) Linkage mapping results of median body length in control conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (C) Histogram of the normalized median optical densities of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position. (D) Linkage mapping results of median optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. (E) Histogram of the median normalized optical densities (optical density of each animal divided by its length) of the RIAIL population in control conditions. Linear regression was used to reduce the effects of assay plate and plate position before normalizing for body length. (F) Linkage mapping results of median normalized optical density in control conditions are shown with genomic position (Mb) on the x-axis and LOD score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. After normalizing for body length additional variation is revealed and maps to chromosomes III and X. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3><em>C. elegans</em> strains vary in their responses to the herbicide paraquat, and multiple QTL contribute to this difference</h3><p id="p-35">We extended our high-throughput assays to map fecundity and body size traits in response to the chemical stressor paraquat. This widely used herbicide catalyzes the formation of reactive oxygen species and interferes with the electron transport chain in animal cells. Previous research in <em>C. elegans</em> indicated that the species is sensitive to paraquat (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ishii 2000</a>) and that wild strains differ in responses to it (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Vertino <em>et al.</em> 2011</a>). To define an appropriate paraquat dose for our high-throughput assay, we measured fecundity and each of the body size traits for four divergent strains (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>) exposed to increasing concentrations of paraquat (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The broad-sense heritability of fecundity and body size traits in paraquat conditions was 60% on average, indicating that the majority of phenotypic variance in the population can be attributed to genetic causes. Using these results, we chose 1.5 mM paraquat as an appropriate concentration for mapping paraquat responses in our RIAIL population because it had the largest difference between the parent strains along with high broad-sense heritability (0.56).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The normalized brood sizes of four strains in different concentrations of the herbicide paraquat are shown. N2 (orange), CB4856 (blue), JU258 (red), and LKC34 (black) have decreased growth rates and body sizes when exposed to increasing concentration of paraquat. The means of four replicates of each strain are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The normalized brood sizes of four strains in different concentrations of the herbicide paraquat are shown. N2 (orange), CB4856 (blue), JU258 (red), and LKC34 (black) have decreased growth rates and body sizes when exposed to increasing concentration of paraquat. The means of four replicates of each strain are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F5.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/911/F5.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466048" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">The normalized brood sizes of four strains in different concentrations of the herbicide paraquat are shown. N2 (orange), CB4856 (blue), JU258 (red), and LKC34 (black) have decreased growth rates and body sizes when exposed to increasing concentration of paraquat. The means of four replicates of each strain are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Next, using our high-throughput assays, we exposed 357 of the 359 N2xCB4856 RIAILs and replicates of the parent strains to 1.5 mM paraquat and measured fecundity and each of the body size traits. As expected, fecundity was reduced after paraquat exposure, as compared to control conditions (data not shown). For QTL mapping of paraquat-specific responses, we used linear regression to reduce the contributions of control conditions to the paraquat phenotype data and then mapped the residual traits in the same way as the control traits. Nine additional QTL were detected that represent two distinct paraquat-specific genomic regions on chromosomes IV and V (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). These two loci each control approximately 5% of the phenotypic variance for traits related to optical density (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). Moreover, these loci do not overlap QTL detected in control conditions (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS7.pdf">Figure S7</a>), indicating that separate loci control growth in paraquat beyond any differences observed in control growth conditions. Finally, when considering QTL detected in any condition, we found that the proportion of phenotypic variance explained by each QTL was on average 5% (<a href="http://www.g3journal.org/content/suppl/2015/03/13/g3.115.017178.DC1/FigureS8.pdf">Figure S8</a>). When considered additively for any particular trait, the detected QTL explained 10% of the phenotypic variance on average. Thus, our high-throughput assays and this panel of new strains enabled the detection of QTL with small phenotypic effects affecting a variety of traits.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/911/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Histograms of mean normalized optical density in control (black) and in paraquat (red) conditions for the 357 RIAILs are shown. (B) Linkage mapping results of the residual normalized mean optical density in paraquat conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-344331151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Histograms of mean normalized optical density in control (black) and in paraquat (red) conditions for the 357 RIAILs are shown. (B) Linkage mapping results of the residual normalized mean optical density in paraquat conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/911/F6.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/5/911/F6.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/911/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/911/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466050" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">(A) Histograms of mean normalized optical density in control (black) and in paraquat (red) conditions for the 357 RIAILs are shown. (B) Linkage mapping results of the residual normalized mean optical density in paraquat conditions are shown with genomic position (Mb) on the x-axis and logarithm of odds (LOD) score on the y-axis. The tick marks on the x-axis denote every 5 Mb. Each chromosome is in its own box labeled on top. The dotted red line is the LOD threshold for 5% FDR obtained by permuting the phenotype data and mapping 1000 times. The red triangle denotes the peak QTL marker, and the blue bar shows the 95% QTL confidence interval. FDR, false discovery rate; QTL, quantitative trait loci; RIAIL, recombinant inbred advanced intercross lines.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-39">To supplement an existing collection of recombinant inbred lines between the laboratory strain (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) and a wild strain from Hawaii (<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>), we created an additional 359 RIAILs, more than doubling the total size of this resource. We also optimized high-throughput assays to measure fecundity and size parameters for populations of nematodes. These assays enable studies of quantitative trait variation for a large number of fitness-related traits. We mapped these traits in normal growth conditions and also after exposure to the herbicide paraquat. These mappings identified 56 QTL, representing eight distinct loci for sets of correlated traits. Together, these resources constitute a powerful new experimental platform to study phenotypic effects of diverse treatments in a large number of independent strains in a reproducible and scalable manner.</p><div id="sec-13" class="subsection"><h3>Persistence of allele frequency skews in the new RIAIL panel</h3><p id="p-40">This new collection of RIAILs was constructed using the same cross design as the previous collection (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Rockman and Kruglyak 2009</a>). However, we used strain QX1430 instead of <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> to eliminate the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a> <a href="http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene">zeel-1</a></em> incompatibility (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Seidel <em>et al.</em> 2011</a>, <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">2008</a>) and the large phenotypic effects of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). Surprisingly, we still see allele frequency skews on chromosome I and other chromosomes, suggesting that subtle incompatibilities still exist. The remaining skew on chromosome I may have been caused by genes other than the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em>; alternatively, the transposon insertion into <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> might not cause complete loss of gene function. We carried out our crosses at 20Â°, rather than 25Â°, which could have allowed different alleles to have been selected during the RIAIL panel construction. Larger RIAIL or NIL panels might allow us to map these subtle incompatibilities further.</p></div><div id="sec-14" class="subsection"><h3>Multivariate data uncover multiple contributors to growth rate</h3><p id="p-41">The high-throughput phenotyping platform generates a large amount of data from populations of independent strains. Along with counting each animal in every well of a microtiter plate, the length, optical density, and three fluorescent parameters of each individual are recorded. Summary statistics from distributions of these parameters, like quantiles, means, interquartile ranges, and variances, are calculated. Many of these traits are correlated with each other. For example, small body size as indicated by the 10th quantile of length is correlated with the 10<sup>th</sup> quantile of optical density. However, we found that different summary statistics recorded for the RIAILs sometimes map to different regions of the genome. The population of animals derived from the hermaphrodite mother varies in life stage from early larval animals to young adults. In control conditions without any paraquat applied, we find that the summary statistics that describe distributions from the RIAILs are different, indicating that growth rates (as measured by the length of offspring or optical density) vary in our strain population. Other researchers have used these size parameters to fit models of growth (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boyd <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Smith <em>et al.</em> 2009</a>). Interestingly, our data suggest that life-stage progression is controlled by different genetic mechanisms because we often identify distinct QTL for small size <em>vs.</em> large size. Our results indicate that the COPAS BIOSORT platform is a powerful tool to obtain multivariate data from large numbers of independent strains and observe relatively small phenotypic effects.</p></div><div id="sec-15" class="subsection"><h3>Comparison with previous QTL studies in <em>C. elegans</em></h3><p id="p-42">Our mapping results recapitulate and expand upon previous QTL studies in <em>C. elegans</em>. Previously, we mapped adult body size to two QTL on chromosomes V and X (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). The QTL on chromosome V also was detected in this study as a regulator of body length but only when looking at larger animals in the distribution (75th and 90th quantiles), suggesting that this genetic locus may influence body length in later developmental stages. As expected, we did not identify the QTL on chromosome X because the new mapping population no longer varies for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> gene. We also mapped fecundity to two QTL on chromosomes II and X (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2014</a>). Like the previous mapping result, the lack of variation at <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> likely eliminates the QTL on chromosome X. The other QTL on chromosome II was at the threshold of detection and could have been falsely discovered in our previous analysis. In this study, we identified a QTL on chromosome IV for fecundity that was not identified in the earlier analysis. Because a QTL with this effect size is near the level of detection in the previous study, we believe that it was possibly a false negative. In two independent crossesâBergeracxRC301 and CB4857xRC301âa QTL was detected just right of center on chromosome IV that contributed to variation in longevity (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ayyadevara <em>et al.</em> 2003</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">2001</a>). This QTL also contributed to variation in responses to paraquat (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Vertino <em>et al.</em> 2011</a>). An overlapping genomic region was detected in our current study as a regulator of optical density in paraquat conditions. Further investigation will determine whether these QTL are separate loci and whether the genomic region we identified also affects variation in longevity. Using imaging, investigators scored another collection of N2xCB4856 recombinant strains for variation in body size and found a QTL on chromosome IV but did not detect QTL overlapping with our results (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kammenga <em>et al.</em> 2007</a>). This discrepancy is likely caused by differences in the body size assay. Notably, recombinant inbred line panels constructed using the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> strain or derivatives likely harbor numerous laboratory-derived alleles in addition to <em><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene">npr-1</a></em> and care should be taken while interpreting evolutionary conclusions.</p><p id="p-43">This new collection of RIAILs will drastically increase statistical power beyond the current set of N2xCB4856 RIAILs. When phenotyped as a complete set, the combined panel should allow researchers to detect QTL that individually explain as little as 2% of the phenotypic variance for quantitative traits. When combined with our new high-throughput phenotyping assays, the panel will enable mapping of multivariate phenotype measurements for responses to any aqueous compound, altered bacterial food sources, or biotic stresses. We believe that these developments increase the power of <em>C. elegans</em> quantitative genetics in substantial ways.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We are grateful to Daniel Cook, Sam Rosenberg, Robyn Tanny, and Stefan Zdraljevic for helpful discussions and comments on the manuscript. We also thank Matthew Rockman along with the <em>Caenorhabditis</em> Genetics Center, which is funded by the National Institutes of Health National Center for Research Resources (NCRR), for some nematode strains used in this work. We are grateful for the assistance from Sarah Gerke in cross management during the RIAIL construction project. Additionally, we thank The Rockefeller University Genomics Facility for GoldenGate genotyping and Christina DeCoste from the Princeton Flow Cytometry Resource Facility for her invaluable assistance with the COPAS BIOSORT. This work was supported by an NIH Ruth L. Kirschstein National Research Service Award F32-GM089007 (E.C.A.) and a National Cancer Institute training grant T32-CA009528 (E.C.A.), a Merck Fellowship of the Life Science Research Foundation (J.P.G.), a Department of Defense (DoD) National Defense Science &amp; Engineering Graduate Fellowship (NDSEG) Program fellowship (H.S.S.), a James S. McDonnell Foundation Centennial Fellowship (L.K.), Howard Hughes Medical Institute (L.K.), and NIH grant R01-HG004321 (L.K.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017178/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017178/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-46"><em>Communicating editor: P. C. Phillips</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-05"><span class="received-label">Received </span>February 5, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-12"><span class="accepted-label">Accepted </span>March 12, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2015 Andersen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.1" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crissman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>â<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.2" data-doi="10.1371/journal.pgen.1004156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A variant in the neuropeptide receptor npr-1 is a major determinant of <em>Caenorhabditis elegans</em> growth and physiology.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004156%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004156%26rft_id%253Dinfo%253Apmid%252F24586193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586193&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayyadevara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyadevara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaden</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shmookler Reis</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Genetic mapping of quantitative trait loci governing longevity of <em>Caenorhabditis elegans</em> in recombinant-inbred progeny of a Bergerac-BO x RC301 interstrain cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">655</span>â<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAyyadevara%26rft.auinit1%253DS.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D655%26rft.epage%253D666%26rft.atitle%253DGenetic%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BGoverning%2BLongevity%2Bof%2BCaenorhabditis%2Belegans%2Bin%2BRecombinant-Inbred%2BProgeny%2Bof%2Ba%2BBergerac-BO%2B%2BRC301%2BInterstrain%2BCross%26rft_id%253Dinfo%253Apmid%252F11156986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvNjU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayyadevara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyadevara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vertino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaden</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Genetic loci modulating fitness and life span in <em>Caenorhabditis elegans</em>: categorical trait interval mapping in CL2a x Bergerac-BO recombinant-inbred worms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">557</span>â<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAyyadevara%26rft.auinit1%253DS.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D557%26rft.epage%253D570%26rft.atitle%253DGenetic%2BLoci%2BModulating%2BFitness%2Band%2BLife%2BSpan%2Bin%2BCaenorhabditis%2Belegans%253A%2BCategorical%2BTrait%2BInterval%2BMapping%2Bin%2BCL2a%2Bx%2BBergerac-BO%2BRecombinant-Inbred%2BWorms%26rft_id%253Dinfo%253Apmid%252F12618395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.5" data-doi="10.1038/nature09821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsunozaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Catecholamine receptor polymorphisms affect decision-making in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">472</span>: <span class="cit-fpage">313</span>â<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBendesky%26rft.auinit1%253DA.%26rft.volume%253D472%26rft.issue%253D7343%26rft.spage%253D313%26rft.epage%253D318%26rft.atitle%253DCatecholamine%2Breceptor%2Bpolymorphisms%2Baffect%2Bdecision-making%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09821%26rft_id%253Dinfo%253Apmid%252F21412235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21412235&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289724600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.6" data-doi="10.1371/journal.pgen.1003157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">M.-W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003157%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003157%26rft_id%253Dinfo%253Apmid%252F23284308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284308&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.7" data-doi="10.1038/nature11867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lite</span>  <span class="cit-name-given-names">T.-L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Finding the sources of missing heritability in a yeast cross.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">234</span>â<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11867%26rft_id%253Dinfo%253Apmid%252F23376951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315137700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.8" data-doi="10.1371/journal.pone.0007024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kissling</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Application of a mathematical model to describe the effects of chlorpyrifos on <em>Caenorhabditis elegans</em> development.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBoyd%26rft.auinit1%253DW.%2BA.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De7024%26rft.epage%253De7024%26rft.atitle%253DApplication%2Bof%2Ba%2Bmathematical%2Bmodel%2Bto%2Bdescribe%2Bthe%2Beffects%2Bof%2Bchlorpyrifos%2Bon%2BCaenorhabditis%2Belegans%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007024%26rft_id%253Dinfo%253Apmid%252F19753116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19753116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.9" data-doi="10.1038/nature03865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Genetic interactions between polymorphisms that affect gene expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">701</span>â<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrem%26rft.auinit1%253DR.%2BB.%26rft.volume%253D436%26rft.issue%253D7051%26rft.spage%253D701%26rft.epage%253D703%26rft.atitle%253DGenetic%2Binteractions%2Bbetween%2Bpolymorphisms%2Bthat%2Baffect%2Bgene%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03865%26rft_id%253Dinfo%253Apmid%252F16079846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16079846&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230964500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.10" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>â<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.11" data-doi="10.1093/nar/gkp528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doroszuk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fradin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A genome-wide library of CB4856/N2 introgression lines of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp528%26rft_id%253Dinfo%253Apmid%252F19542186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xNi9lMTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.12" data-doi="10.1017/S0016672310000601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaertner</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title"><em>Caenorhabditis elegans</em> as a platform for molecular quantitative genetics and the systems biology of natural variation.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">331</span>â<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DGaertner%26rft.auinit1%253DB.%2BE.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D331%26rft.epage%253D348%26rft.atitle%253DCaenorhabditis%2Belegans%2Bas%2Ba%2Bplatform%2Bfor%2Bmolecular%2Bquantitative%2Bgenetics%2Band%2Bthe%2Bsystems%2Bbiology%2Bof%2Bnatural%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000601%26rft_id%253Dinfo%253Apmid%252F21429266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429266&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.13" data-doi="10.1534/genetics.112.142877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaertner</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1533</span>â<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142877%26rft_id%253Dinfo%253Apmid%252F23086219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.14" data-doi="10.1126/science.1214318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Natural variation in a chloride channel subunit confers avermectin resistance in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">574</span>â<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BNY%2529%26rft.volume%253D335%26rft.spage%253D574%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.1214318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.911.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glater</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Multigenic natural variation underlies <em>Caenorhabditis elegans</em> olfactory preference for the bacterial pathogen Serratia marcescens.</span> G3 (Bethesda) <span class="cit-vol">4</span>: <span class="cit-fpage">265</span>â<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.16" data-doi="10.1080/10715760000301371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Oxidative stress and aging in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Free Radic. Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">857</span>â<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFree%2Bradical%2Bresearch%26rft.stitle%253DFree%2BRadic%2BRes%26rft.aulast%253DIshii%26rft.auinit1%253DN.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D857%26rft.epage%253D864%26rft.atitle%253DOxidative%2Bstress%2Band%2Baging%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10715760000301371%26rft_id%253Dinfo%253Apmid%252F11237107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10715760000301371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237107&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167185200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.17" data-doi="10.1371/journal.pgen.0030034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doroszuk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J. A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazendonk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiridon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A <em>Caenorhabditis elegans</em> wild type defies the temperature-size rule owing to a single nucleotide polymorphism in tra-3.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKammenga%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De34%26rft.epage%253De34%26rft.atitle%253DA%2BCaenorhabditis%2Belegans%2Bwild%2Btype%2Bdefies%2Bthe%2Btemperature-size%2Brule%2Bowing%2Bto%2Ba%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bin%2Btra-3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030034%26rft_id%253Dinfo%253Apmid%252F17335351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17335351&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.18" data-doi="10.1371/journal.pgen.0020222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutteling</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mapping determinants of gene expression plasticity by genetical genomics in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De222%26rft.epage%253De222%26rft.atitle%253DMapping%2Bdeterminants%2Bof%2Bgene%2Bexpression%2Bplasticity%2Bby%2Bgenetical%2Bgenomics%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020222%26rft_id%253Dinfo%253Apmid%252F17196041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17196041&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.19" data-doi="10.1016/j.neuron.2009.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Quantitative mapping of a digenic behavioral trait implicates globin variation in <em>C. elegans</em> sensory behaviors.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">692</span>â<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D61%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19285466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285466&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264366900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.20" data-doi="10.1038/nature07171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palopoli</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">TinMaung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curwen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Molecular basis of the copulatory plug polymorphism in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">1019</span>â<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPalopoli%26rft.auinit1%253DM.%2BF.%26rft.volume%253D454%26rft.issue%253D7207%26rft.spage%253D1019%26rft.epage%253D1022%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bthe%2Bcopulatory%2Bplug%2Bpolymorphism%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07171%26rft_id%253Dinfo%253Apmid%252F18633349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18633349&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258591000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.911.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Resistance to germline RNA interference in a <em>Caenorhabditis elegans</em> wild isolate exhibits complexity and nonadditivity.</span> G3 (Bethesda) <span class="cit-vol">3</span>: <span class="cit-fpage">941</span>â<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.22" data-doi="10.1126/science.1166527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A polymorphism in npr-1 is a behavioral determinant of pathogen susceptibility in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">382</span>â<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BC.%26rft.volume%253D323%26rft.issue%253D5912%26rft.spage%253D382%26rft.epage%253D384%26rft.atitle%253DA%2BPolymorphism%2Bin%2Bnpr-1%2BIs%2Ba%2BBehavioral%2BDeterminant%2Bof%2BPathogen%2BSusceptibility%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166527%26rft_id%253Dinfo%253Apmid%252F19150845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMi8zODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.23" data-doi="10.1534/genetics.107.083873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Breeding designs for recombinant inbred advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1069</span>â<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083873%26rft_id%253Dinfo%253Apmid%252F18505881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.24" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Recombinational landscape and population genomics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.25" data-doi="10.1126/science.1194208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skrovanek</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Selection at linked sites shapes heritable phenotypic variation in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">372</span>â<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D372%26rft.epage%253D376%26rft.atitle%253DSelection%2Bat%2BLinked%2BSites%2BShapes%2BHeritable%2BPhenotypic%2BVariation%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194208%26rft_id%253Dinfo%253Apmid%252F20947766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.26" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>â<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.27" data-doi="10.1371/journal.pbio.1001115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ailion</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oudenaarden</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSeidel%26rft.auinit1%253DH.%2BS.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253De1001115%26rft.epage%253De1001115%26rft.atitle%253DA%2Bnovel%2Bsperm-delivered%2Btoxin%2Bcauses%2Blate-stage%2Bembryo%2Blethality%2Band%2Btransmission%2Bratio%2Bdistortion%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001115%26rft_id%253Dinfo%253Apmid%252F21814493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814493&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.28" data-doi="10.1371/journal.pone.0111090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimko</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">COPASutils: an R package for reading, processing, and visualizing data from COPAS large-particle flow cytometers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e111090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De111090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0111090%26rft_id%253Dinfo%253Apmid%252F25329171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0111090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25329171&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.29" data-doi="10.1371/journal.pone.0007018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kissling</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A discrete time model for the analysis of medium-throughput <em>C. elegans</em> growth data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSmith%26rft.auinit1%253DM.%2BV.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De7018%26rft.epage%253De7018%26rft.atitle%253DA%2Bdiscrete%2Btime%2Bmodel%2Bfor%2Bthe%2Banalysis%2Bof%2Bmedium-throughput%2BC.%2Belegans%2Bgrowth%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007018%26rft_id%253Dinfo%253Apmid%252F19753303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19753303&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.911.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vertino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyadevara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaden</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reis</span>  <span class="cit-name-given-names">R. J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A narrow quantitative trait locus in <em>C. elegans</em> coordinately affects longevity, thermotolerance, and resistance to paraquat.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DVertino%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.spage%253D63%26rft.epage%253D63%26rft.atitle%253DA%2Bnarrow%2Bquantitative%2Btrait%2Blocus%2Bin%2BC.%2Belegans%2Bcoordinately%2Baffects%2Blongevity%252C%2Bthermotolerance%252C%2Band%2Bresistance%2Bto%2Bparaquat.%26rft_id%253Dinfo%253Apmid%252F22303358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22303358&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F911.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/911.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/891" title="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/921" title="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/911" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7JG_XQJ5ckpYoZtuxIYBqpWBzL6cMHJJSGYYLVP7BMQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/911" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/911" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/911" class="active" data-icon-position="" data-hide-link-title="0">A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459887" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-houDZrVTcB6ZKM8gUgVJ7DdP4huyuIcVtt-CcMjvDA0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/911.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/911" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-t6UgnhtUFxxiAcPuyjTsQp388Ti1HHwXJhmwyFW1DtE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/911" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/911.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459887" id="node459887--2" data-pisa="ggg;5/5/911" data-pisa-master="ggg;g3.115.017178" data-apath="/ggg/5/5/911.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Powerful New Quantitative Genetics Platform, Combining <em>Caenorhabditis elegans</em> High-Throughput Fitness Assays with a Large Collection of Recombinant Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erik C. Andersen</span>, <span class="highwire-citation-author" data-delta="1">Tyler C. Shimko</span>, <span class="highwire-citation-author" data-delta="2">Jonathan R. Crissman</span>, <span class="highwire-citation-author" data-delta="3">Rajarshi Ghosh</span>, <span class="highwire-citation-author" data-delta="4">Joshua S. Bloom</span>, <span class="highwire-citation-author" data-delta="5">Hannah S. Seidel</span>, <span class="highwire-citation-author" data-delta="6">Justin P. Gerke</span> and <span class="highwire-citation-author" data-delta="7">Leonid Kruglyak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">911-920;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017178 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459887--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erik C. Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BC.%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2BC.%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler C. Shimko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BC.%2BShimko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shimko%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BC.%2BShimko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan R. Crissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BR.%2BCrissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crissman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BR.%2BCrissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajarshi Ghosh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajarshi%2BGhosh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ghosh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajarshi%2BGhosh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joshua S. Bloom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BS.%2BBloom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bloom%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BS.%2BBloom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hannah S. Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BS.%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHannah%2BS.%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Justin P. Gerke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BGerke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerke%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BP.%2BGerke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonid Kruglyak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonid%2BKruglyak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kruglyak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonid%2BKruglyak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/911" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/911&amp;text=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/911&amp;t=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/911&amp;title=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459887/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459887" id="node459887--3" data-pisa="ggg;5/5/911" data-pisa-master="ggg;g3.115.017178" data-apath="/ggg/5/5/911.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Powerful New Quantitative Genetics Platform, Combining <em>Caenorhabditis elegans</em> High-Throughput Fitness Assays with a Large Collection of Recombinant Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erik C. Andersen</span>, <span class="highwire-citation-author" data-delta="1">Tyler C. Shimko</span>, <span class="highwire-citation-author" data-delta="2">Jonathan R. Crissman</span>, <span class="highwire-citation-author" data-delta="3">Rajarshi Ghosh</span>, <span class="highwire-citation-author" data-delta="4">Joshua S. Bloom</span>, <span class="highwire-citation-author" data-delta="5">Hannah S. Seidel</span>, <span class="highwire-citation-author" data-delta="6">Justin P. Gerke</span> and <span class="highwire-citation-author" data-delta="7">Leonid Kruglyak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">911-920;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017178 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459887--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erik C. Andersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BC.%2BAndersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andersen%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2BC.%2BAndersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tyler C. Shimko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biosciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BC.%2BShimko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shimko%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BC.%2BShimko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan R. Crissman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BR.%2BCrissman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crissman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BR.%2BCrissman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajarshi Ghosh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajarshi%2BGhosh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ghosh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajarshi%2BGhosh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joshua S. Bloom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BS.%2BBloom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bloom%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BS.%2BBloom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hannah S. Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BS.%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHannah%2BS.%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Justin P. Gerke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BGerke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerke%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BP.%2BGerke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leonid Kruglyak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Howard Hughes Medical Institute, Departments of Human Genetics and Biological Chemistry, David Geffen School of Medicine, U.C.L.A., Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonid%2BKruglyak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kruglyak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonid%2BKruglyak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erik.andersen@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">erik.andersen@northwestern.edu</a>
<a href="mailto:lkruglyak@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">lkruglyak@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=911&amp;rft.epage=920&amp;rft.atitle=A%20Powerful%20New%20Quantitative%20Genetics%20Platform%2C%20Combining%20Caenorhabditis%20elegans%20High-Throughput%20Fitness%20Assays%20with%20a%20Large%20Collection%20of%20Recombinant%20Strains&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Andersen%2C+Erik+C.&amp;rft.au=Shimko%2C+Tyler+C.&amp;rft.au=Crissman%2C+Jonathan+R.&amp;rft.au=Ghosh%2C+Rajarshi&amp;rft.au=Bloom%2C+Joshua+S.&amp;rft.au=Seidel%2C+Hannah+S.&amp;rft.au=Gerke%2C+Justin+P.&amp;rft.au=Kruglyak%2C+Leonid"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459887/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459887/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459887/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459887/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459887/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459887/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459887/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459887/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459887/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459887/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459887/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459887/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/911" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/911.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/911.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/911.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/911.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
