<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/971" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/971.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/971.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/971.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/971.atom" />
<meta name="HW.pisa" content="ggg;5/5/971" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function" />
<meta name="DC.Identifier" content="10.1534/g3.114.016121" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Kevei et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A recombinant in-bred line population derived from a cross between Solanum lycopersicum var. cerasiforme (E9) and S. pimpinellifolium (L5) has been used extensively to discover quantitative trait loci (QTL), including those that act via rootstock genotype, however, high-resolution single-nucleotide polymorphism genotyping data for this population are not yet publically available. Next-generation resequencing of parental lines allows the vast majority of polymorphisms to be characterized and used to progress from QTL to causative gene. We sequenced E9 and L5 genomes to 40- and 44-fold depth, respectively, and reads were mapped to the reference Heinz 1706 genome. In L5 there were three clear regions on chromosome 1, chromosome 4, and chromosome 8 with increased rates of polymorphism. Two other regions were highly polymorphic when we compared Heinz 1706 with both E9 and L5 on chromosome 1 and chromosome 10, suggesting that the reference sequence contains a divergent introgression in these locations. We also identified a region on chromosome 4 consistent with an introgression from S. pimpinellifolium into Heinz 1706. A large dataset of polymorphisms for the use in fine-mapping QTL in a specific tomato recombinant in-bred line population was created, including a high density of InDels validated as simple size-based polymerase chain reaction markers. By careful filtering and interpreting the SnpEff prediction tool, we have created a list of genes that are predicted to have highly perturbed protein functions in the E9 and L5 parental lines." />
<meta name="DC.Contributor" content="Zoltan Kevei" />
<meta name="DC.Contributor" content="Robert C. King" />
<meta name="DC.Contributor" content="Fady Mohareb" />
<meta name="DC.Contributor" content="Martin J. Sergeant" />
<meta name="DC.Contributor" content="Sajjad Z. Awan" />
<meta name="DC.Contributor" content="Andrew J. Thompson" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A recombinant in-bred line population derived from a cross between &lt;i&gt;Solanum lycopersicum&lt;/i&gt; var. &lt;i&gt;cerasiforme&lt;/i&gt; (E9) and &lt;i&gt;S. pimpinellifolium&lt;/i&gt; (L5) has been used extensively to discover quantitative trait loci (QTL), including those that act via rootstock genotype, however, high-resolution single-nucleotide polymorphism genotyping data for this population are not yet publically available. Next-generation resequencing of parental lines allows the vast majority of polymorphisms to be characterized and used to progress from QTL to causative gene. We sequenced E9 and L5 genomes to 40- and 44-fold depth, respectively, and reads were mapped to the reference Heinz 1706 genome. In L5 there were three clear regions on chromosome 1, chromosome 4, and chromosome 8 with increased rates of polymorphism. Two other regions were highly polymorphic when we compared Heinz 1706 with both E9 and L5 on chromosome 1 and chromosome 10, suggesting that the reference sequence contains a divergent introgression in these locations. We also identified a region on chromosome 4 consistent with an introgression from &lt;i&gt;S. pimpinellifolium&lt;/i&gt; into Heinz 1706. A large dataset of polymorphisms for the use in fine-mapping QTL in a specific tomato recombinant in-bred line population was created, including a high density of InDels validated as simple size-based polymerase chain reaction markers. By careful filtering and interpreting the SnpEff prediction tool, we have created a list of genes that are predicted to have highly perturbed protein functions in the E9 and L5 parental lines.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/971" />
<meta name="citation_id" content="5/5/971" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/971" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/971.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/971.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/971.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016121" />
<meta name="citation_pmid" content="25809074" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="971" />
<meta name="citation_lastpage" content="981" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zoltan Kevei" />
<meta name="citation_author_institution" content="Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom" />
<meta name="citation_author_email" content="z.l.kevei@cranfield.ac.uk" />
<meta name="citation_author" content="Robert C. King" />
<meta name="citation_author_institution" content="Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom" />
<meta name="citation_author_institution" content="Rothamsted Research, Harpenden, Hertfordshire, AL5 2JQ, United Kingdom" />
<meta name="citation_author" content="Fady Mohareb" />
<meta name="citation_author_institution" content="Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom" />
<meta name="citation_author" content="Martin J. Sergeant" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom" />
<meta name="citation_author" content="Sajjad Z. Awan" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom" />
<meta name="citation_author" content="Andrew J. Thompson" />
<meta name="citation_author_institution" content="Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Sci;citation_author=A. Ambrosone;citation_author=A. Costa;citation_author=A. Leone;citation_author=S. Grillo;citation_title=Beyond transcription: RNA-binding proteins as emerging regulators of plant response to environmental constraints.;citation_pages=12-18;citation_volume=182;citation_year=2012;citation_pmid=22118611;citation_doi=10.1016/j.plantsci.2011.02.004" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. J. Asins;citation_author=M. C. Bolarin;citation_author=F. Perez-Alfocea;citation_author=M. T. Estan;citation_author=C. Martinez-Andujar;citation_title=Genetic analysis of physiological components of salt tolerance conferred by Solanum rootstocks. What is the rootstock doing for the scion?;citation_pages=105-115;citation_volume=121;citation_year=2010;citation_issue=1;citation_pmid=20180091;citation_doi=10.1007/s00122-010-1294-9" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=M. J. Asins;citation_author=I. Villalta;citation_author=M. M. Aly;citation_author=R. Olias;citation_author=D. E. Alvarez;citation_author=P. A. de Morales;citation_title=Two closely linked tomato HKT coding genes are positional candidates for the major tomato QTL involved in Na+ /K+ homeostasis.;citation_pages=1171-1191;citation_volume=36;citation_year=2013;citation_doi=10.1111/pce.12051" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=W. Barrantes;citation_author=A. Fernández-del-Carmen;citation_author=G. López-Casado;citation_author=M. Á. González-Sánchez;citation_author=R. Fernández-Muñoz;citation_title=Highly efficient genomics-assisted development of a library of introgression lines of Solanum pimpinellifolium;citation_pages=1817-1831;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0141-0" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=B. Çakir;citation_author=A. Gül;citation_author=L. Yolageldi;citation_author=H. Özaktan;citation_title=Response to Fusarium oxysporum f.sp. radicis-lycopersici in tomato roots involves regulation of SA- and ET-responsive gene expressions.;citation_pages=373-385;citation_volume=139;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Causse;citation_author=N. Desplat;citation_author=L. Pascual;citation_author=M. C. Le Paslier;citation_author=C. Sauvage;citation_title=Whole genome resequencing in tomato reveals variation associated with introgression and breeding events.;citation_pages=791;citation_volume=14;citation_year=2013;citation_pmid=24228636;citation_doi=10.1186/1471-2164-14-791" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=S. B. Chang;citation_author=T. J. Yang;citation_author=E. Datema;citation_author=J. Van Vugt;citation_author=B. Vosman;citation_title=FISH mapping and molecular organization of the major repetitive sequences of tomato.;citation_pages=919-933;citation_volume=16;citation_year=2008;citation_issue=7;citation_pmid=18688733;citation_doi=10.1007/s10577-008-1249-z" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. Wang le;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. T. Estan;citation_author=I. Villalta;citation_author=M. C. Bolarin;citation_author=E. A. Carbonell;citation_author=M. J. Asins;citation_title=Identification of fruit yield loci controlling the salt tolerance conferred by solanum rootstocks.;citation_pages=305-312;citation_volume=118;citation_year=2009;citation_issue=2;citation_pmid=18846361;citation_doi=10.1007/s00122-008-0900-6" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. Evans;citation_author=R. F. McCormick;citation_author=D. Morishige;citation_author=S. N. Olson;citation_author=B. Weers;citation_title=Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.;citation_pages=e79192;citation_volume=8;citation_year=2013;citation_pmid=24265758;citation_doi=10.1371/journal.pone.0079192" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=R. Fernández-Muñoz;citation_author=M. Salinas;citation_author=M. Álvarez;citation_author=J. Cuartero;citation_title=Inheritance of resistance to two-spotted spider mite and glandular leaf trichomes in wild tomato Lycopersicon pimpinellifolium (Jusl.) Mill.;citation_pages=188-195;citation_volume=128;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=M. R. Foolad;citation_author=H. L. Merk;citation_author=H. Ashrafi;citation_title=Genetics, genomics and breeding of late blight and early blight resistance in tomato.;citation_pages=75-107;citation_volume=27;citation_year=2008;citation_doi=10.1080/07352680802147353" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Grandillo;citation_author=H. M. Ku;citation_author=S. D. Tanksley;citation_title=Identifying the loci responsible for natural variation in fruit size and shape in tomato.;citation_pages=978-987;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051405" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=M. Hashimoto;citation_author=J. Negi;citation_author=J. Young;citation_author=M. Israelsson;citation_author=J. I. Schroeder;citation_title=Arabidopsis HT1 kinase controls stomatal movements in response to CO2.;citation_pages=391-397;citation_volume=8;citation_year=2006;citation_issue=4;citation_pmid=16518390;citation_doi=10.1038/ncb1387" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=H. Hirakawa;citation_author=K. Shirasawa;citation_author=A. Ohyama;citation_author=H. Fukuoka;citation_author=K. Aoki;citation_title=Genome-wide SNP genotyping to infer the effects on gene functions in tomato.;citation_volume=20;citation_year=2013;citation_pmid=23482505;citation_doi=10.1093/dnares/dst005" />
<meta name="citation_reference" content="citation_journal_title=Plant physiology and biochemistry :  PPB / SociÃ©tÃ© franÃ§aise de physiologie vÃ©gÃ©tale;citation_journal_abbrev=Plant Physiol Biochem;citation_author=Y. W. Huang;citation_author=W. S. Tsay;citation_author=C. C. Chen;citation_author=C. W. Lin;citation_author=H. J. Huang;citation_title=Increased expression of the rice C-type cyclin-dependent protein kinase gene, Orysa;CDKC;1, in response to salt stress.;citation_pages=71-81;citation_volume=46;citation_year=2008;citation_issue=1;citation_pmid=18054244;citation_doi=10.1016/j.plaphy.2007.10.013" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=M. Kobayashi;citation_author=H. Nagasaki;citation_author=V. Garcia;citation_author=D. Just;citation_author=C. Bres;citation_title=Genome-wide analysis of intraspecific DNA polymorphism in ‘micro-tom’, a model cultivar of tomato (Solanum lycopersicum);citation_volume=55;citation_year=2014;citation_pmid=24319074;citation_doi=10.1093/pcp/pct181" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. Lin;citation_author=G. Zhu;citation_author=J. Zhang;citation_author=X. Xu;citation_author=Q. Yu;citation_title=Genomic analyses provide insights into the history of tomato breeding.;citation_pages=1220-1226;citation_volume=46;citation_year=2014;citation_pmid=25305757;citation_doi=10.1038/ng.3117" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Mol. Biol. Plants;citation_author=P. Malekzadeh;citation_author=J. Khara;citation_author=R. Heydari;citation_title=Alleviating effects of exogenous Gamma-aminobutiric acid on tomato seedling under chilling stress.;citation_pages=133-137;citation_volume=20;citation_year=2014;citation_pmid=24554847;citation_doi=10.1007/s12298-013-0203-5" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. L. McNally;citation_author=K. L. Childs;citation_author=R. Bohnert;citation_author=R. M. Davidson;citation_author=K. Zhao;citation_title=Genomewide SNP variation reveals relationships among landraces and modern varieties of rice.;citation_volume=106;citation_year=2009;citation_pmid=19597147;citation_doi=10.1073/pnas.0900992106" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Mizuno;citation_author=Y. Kawahara;citation_author=H. Sakai;citation_author=H. Kanamori;citation_author=H. Wakimoto;citation_title=Massive parallel sequencing of mRNA in identification of unannotated salinity stress-inducible transcripts in rice (Oryza sativa L.).;citation_pages=683-683;citation_volume=11;citation_year=2010;citation_pmid=21122150;citation_doi=10.1186/1471-2164-11-683" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. J. Monforte;citation_author=M. J. Asins;citation_author=E. A. Carbonell;citation_title=Salt tolerance in Lycopersicon species. V. Does genetic variability at quantitative trait loci affect their analysis?;citation_pages=284-293;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050561" />
<meta name="citation_reference" content="citation_journal_title=Syst. Bot. Monogr.;citation_author=I. E. Peralta;citation_author=D. M. Spooner;citation_author=S. Knapp;citation_title=SYST BOT MONOGR;citation_pages=1-186;citation_volume=84;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=M. J. Rodríguez-López;citation_author=E. Garzo;citation_author=J. P. Bonani;citation_author=A. Fereres;citation_author=R. Fernández-Muñoz;citation_title=Whitefly resistance traits derived from the wild tomato Solanum pimpinellifolium affect the preference and feeding behavior of Bemisia tabaci and reduce the spread of Tomato yellow leaf curl virus.;citation_pages=1191-1201;citation_volume=101;citation_year=2011;citation_issue=10;citation_pmid=21615206;citation_doi=10.1094/PHYTO-01-11-0028" />
<meta name="citation_reference" content="citation_journal_title=Florida Agr. Expt. Sta. Circ.;citation_author=J. W. Scott;citation_author=B. K. Harbaugh;citation_title=Micro-Tom. A miniature dwarf tomato.;citation_pages=1-6;citation_volume=370;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=S. Isobe;citation_author=H. Hirakawa;citation_author=E. Asamizu;citation_author=H. Fukuoka;citation_title=SNP discovery and linkage map construction in cultivated tomato.;citation_pages=381-391;citation_volume=7;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=H. Fukuoka;citation_author=H. Matsunaga;citation_author=Y. Kobayashi;citation_author=I. Kobayashi;citation_title=Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato.;citation_volume=20;citation_year=2013;citation_pmid=23903436;citation_doi=10.1093/dnares/dst033" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Sim;citation_author=G. Durstewitz;citation_author=J. Plieske;citation_author=R. Wieseke;citation_author=M. W. Ganal;citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.;citation_pages=e40563-e40563;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22802968;citation_doi=10.1371/journal.pone.0040563" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Sim;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=D. S. Douches;citation_author=D. Zarka;citation_title=High-density SNP genotyping of tomato (Solanum lycopersicum L.) reveals patterns of genetic variation due to breeding.;citation_pages=e45520-e45520;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029069;citation_doi=10.1371/journal.pone.0045520" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=N. Ranc;citation_author=S. Muños;citation_author=S. Santoni;citation_author=M. Causse;citation_title=A clarified position for Solanum lycopersicum var. cerasiforme in the evolutionary history of tomatoes (solanaceae).;citation_pages=130-130;citation_volume=8;citation_year=2008;citation_pmid=19099601;citation_doi=10.1186/1471-2229-8-130" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. D. Tanksley;citation_author=S. Grandillo;citation_author=T. M. Fulton;citation_author=D. Zamir;citation_author=Y. Eshed;citation_title=Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative L. pimpinellifolium.;citation_pages=213-224;citation_volume=92;citation_year=1996;citation_pmid=24166170;citation_doi=10.1007/s001220050116" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. Thorstensen;citation_author=P. E. Grini;citation_author=R. B. Aalen;citation_title=SET domain proteins in plant development.;citation_pages=407-420;citation_volume=1809;citation_year=2011;citation_issue=8;citation_pmid=21664308;citation_doi=10.1016/j.bbagrm.2011.05.008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=Proteomics;citation_journal_abbrev=Proteomics;citation_author=S. Valadkhan;citation_author=Y. Jaladat;citation_title=The spliceosomal proteome: at the heart of the largest cellular ribonucleoprotein machine.;citation_pages=4128-4141;citation_volume=10;citation_year=2010;citation_issue=22;citation_pmid=21080498;citation_doi=10.1002/pmic.201000354" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Villalta;citation_author=A. Reina-Sánchez;citation_author=J. Cuartero;citation_author=E. A. Carbonell;citation_author=M. J. Asins;citation_title=Comparative microsatellite linkage analysis and genetic structure of two populations of F6 lines derived from Lycopersicon pimpinellifolium and L. cheesmanii.;citation_pages=881-894;citation_volume=110;citation_year=2005;citation_issue=5;citation_pmid=15690174;citation_doi=10.1007/s00122-004-1906-3" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Villalta;citation_author=G. P. Bernet;citation_author=E. A. Carbonell;citation_author=M. J. Asins;citation_title=Comparative QTL analysis of salinity tolerance in terms of fruit yield using two Solanum populations of F7 lines.;citation_pages=1001-1017;citation_volume=114;citation_year=2007;citation_issue=6;citation_pmid=17394031;citation_doi=10.1007/s00122-006-0494-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Villalta;citation_author=A. Reina-Sanchez;citation_author=M. C. Bolarin;citation_author=J. Cuartero;citation_author=A. Belver;citation_title=Genetic analysis of Na(+) and K (+) concentrations in leaf and stem as physiological components of salt tolerance in Tomato.;citation_pages=869-880;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18251001;citation_doi=10.1007/s00122-008-0720-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. V. Vinogradova;citation_author=G. G. Malanina;citation_author=J. S. Waitzman;citation_author=S. E. Rice;citation_author=R. J. Fletterick;citation_title=Plant kinesin-like calmodulin binding protein employs its regulatory domain for dimerization.;citation_pages=e66669;citation_volume=8;citation_year=2013;citation_pmid=23805258;citation_doi=10.1371/journal.pone.0066669" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Viquez-Zamora;citation_author=B. Vosman;citation_author=H. van de Geest;citation_author=A. Bovy;citation_author=R. G. Visser;citation_title=Tomato breeding in the genomics era: insights from a SNP array.;citation_pages=354;citation_volume=14;citation_year=2013;citation_pmid=23711327;citation_doi=10.1186/1471-2164-14-354" />
<meta name="citation_reference" content="citation_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=L. Wang;citation_author=C. Xu;citation_author=C. Wang;citation_author=Y. Wang;citation_title=Characterization of a eukaryotic translation initiation factor 5A homolog from Tamarix androssowii involved in plant abiotic stress tolerance.;citation_pages=118-118;citation_volume=12;citation_year=2012;citation_pmid=22834699;citation_doi=10.1186/1471-2229-12-118" />
<meta name="citation_reference" content="citation_title=The Draft Genome Sequence of Solanum pimpinellifolium LA1589;citation_author=D. Ware;citation_author=W. R. McCombie;citation_author=Z. B. Lippman;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. I. Weller;citation_author=M. Soller;citation_author=T. Brody;citation_title=Linkage analysis of quantitative traits in an interspecific cross of tomato (lycopersicon esculentum x lycopersicon pimpinellifolium) by means of genetic markers.;citation_pages=329-339;citation_volume=118;citation_year=1988;citation_issue=2;citation_pmid=17246412" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=W. Q. Yang;citation_author=Y. Lai;citation_author=M. N. Li;citation_author=W. Y. Xu;citation_author=Y. B. Xue;citation_title=A novel C2-domain phospholipid-binding protein, OsPBP1, is required for pollen fertility in rice.;citation_volume=1;citation_year=2008;citation_pmid=19825580;citation_doi=10.1093/mp/ssn035" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=J. Yang;citation_author=Y. Wang;citation_author=H. Shen;citation_author=W. Yang;citation_title=In Silico identification and experimental validation of insertion-deletion polymorphisms in tomato genome.;citation_volume=21;citation_year=2014;citation_pmid=24618211;citation_doi=10.1093/dnares/dsu008" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=S. Aflitos;citation_author=E. Schijlen;citation_author=H. de Jong;citation_author=D. de Ridder;citation_title=Exploring genetic variation in the tomato (Solanum section Lycopersicon) clade by whole-genome sequencing.;citation_pages=136-148;citation_volume=80;citation_year=2014;citation_pmid=25039268;citation_doi=10.1111/tpj.12616" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="A recombinant in-bred line population derived from a cross between Solanum lycopersicum var. cerasiforme (E9) and S. pimpinellifolium (L5) has been used extensively to discover quantitative trait loci (QTL), including those that act via rootstock genotype, however, high-resolution single-nucleotide polymorphism genotyping data for this population are not yet publically available. Next-generation resequencing of parental lines allows the vast majority of polymorphisms to be characterized and used to progress from QTL to causative gene. We sequenced E9 and L5 genomes to 40- and 44-fold depth, respectively, and reads were mapped to the reference Heinz 1706 genome. In L5 there were three clear regions on chromosome 1, chromosome 4, and chromosome 8 with increased rates of polymorphism. Two other regions were highly polymorphic when we compared Heinz 1706 with both E9 and L5 on chromosome 1 and chromosome 10, suggesting that the reference sequence contains a divergent introgression in these locations. We also identified a region on chromosome 4 consistent with an introgression from S. pimpinellifolium into Heinz 1706. A large dataset of polymorphisms for the use in fine-mapping QTL in a specific tomato recombinant in-bred line population was created, including a high density of InDels validated as simple size-based polymerase chain reaction markers. By careful filtering and interpreting the SnpEff prediction tool, we have created a list of genes that are predicted to have highly perturbed protein functions in the E9 and L5 parental lines." />
<meta name="og-title" property="og:title" content="Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/971" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A recombinant in-bred line population derived from a cross between Solanum lycopersicum var. cerasiforme (E9) and S. pimpinellifolium (L5) has been used extensively to discover quantitative trait loci (QTL), including those that act via rootstock genotype, however, high-resolution single-nucleotide polymorphism genotyping data for this population are not yet publically available. Next-generation resequencing of parental lines allows the vast majority of polymorphisms to be characterized and used to progress from QTL to causative gene. We sequenced E9 and L5 genomes to 40- and 44-fold depth, respectively, and reads were mapped to the reference Heinz 1706 genome. In L5 there were three clear regions on chromosome 1, chromosome 4, and chromosome 8 with increased rates of polymorphism. Two other regions were highly polymorphic when we compared Heinz 1706 with both E9 and L5 on chromosome 1 and chromosome 10, suggesting that the reference sequence contains a divergent introgression in these locations. We also identified a region on chromosome 4 consistent with an introgression from S. pimpinellifolium into Heinz 1706. A large dataset of polymorphisms for the use in fine-mapping QTL in a specific tomato recombinant in-bred line population was created, including a high density of InDels validated as simple size-based polymerase chain reaction markers. By careful filtering and interpreting the SnpEff prediction tool, we have created a list of genes that are predicted to have highly perturbed protein functions in the E9 and L5 parental lines." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F_afCS2l-H4QYkGrkKkUscE3v97DKPm_Gzp91cVb0yw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459839","apath":"\/ggg\/5\/5\/971.atom","pisa":"ggg;5\/5\/971","ac":{"\/ggg\/5\/5\/971.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/971.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/971"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/971"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/971.supplemental":true,"\/content\/5\/5\/971":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jzNf0yUdt\\\\\\\/+nG+z6P26+a2Xwi7O9ChYP\\\\\\\/F65NbA0C4BkFYifjD4ZppCMFKu0VvD7EDVR5oqK3rKzquZ8zToKe92fjNWDklfwY6jgREeWsN+zE5ivSmFNh9HsKuh5vYnZ7JIX0dvT241VxZkdRjH\\\\\\\/xLBTQ96ifrs8AbvyFO18SBFzk+koN26L9lz\\\\\\\/RBwad7tWq0pXwxJ5053opc9Psn6SOdi60j\\\\\\\/hdiYTW5iqTxzQK4JEhV6C4SHBpDcWJdqrAOGRw2NTCaK89ERqYe0GCIJszM8f0UT6B2\\\\\\\/9fv1gKoNTORE9oaVMmr7AvjsP8r7N7bjdKeSnktEIhimdKkxwhVY1psH+a8VhDPkgeFtI34NgmmCR4na6oViGKliUYoXWWvSjfyd8nOX6EUfqV91d98\\\\\\\/bQj\\\\\\\/UZuQ7QAcz2ih5ktZYbIVQcpGa8MYyR1LGnuTuykaAtOxynmrBunGF\\\\\\\/P+q0RGRuSg1r52Kr4vOijVNEyuf4VqiAsNRNWbKobvMGrJvE2V5sloyi\\\\\\\/H8CnV4DDamuQp1mkLZ0+iwsOyLx07IFSyQ7+dkWlPpp7Zykj7fo+AtebLvASE5xqs\\\\\\\/erpBhg6oM7pnCkftCI4a3MQQ9m1Vs+rLk8nkdsTfQwM8tEUaLQJSsZJ4HJeXFJsC8N7tqTSnFFz+wrtX2KNnF6Qm4viyZwBTjmohuVZGXf+kjn58ATjFEMgzTjvHMwMh1Ijh\\\\\\\/0dXZTZNRO+fgzzuEuiTEP+Ard\\\\\\\/tZ7EgkRjahPLEZfENWv5WrpwtWfY11XaxAJWVHOVz3LHbN2eSQ6GzT54\\\\\\\/ntG+suh7gL6forcaDzVDJEE9ArSlRa\\\\\\\/mnpkqoJBiA9EDK4ncEMiEKuS8duJ4rc4h0rlJX4NIw0r+HGzcEog6nAvA\\\\\\\/tAR\\\\\\\/yJjLY5jMkCddhyv0dc9P5S+v9DtoEc2Qjt1DGnFWhm\\\\\\\/PWE0J2EBD+peKE8qku7QonfxitLj8hGNtabQDMmudzDb6Yxf54jdQqIkIiv53HqWKaOyUvLYEnt8Lz+dPVkYxxQYYJNBlr43BMfmB955qEDd4g\\\\\\\/ZlDw2zNJp6NFN2uZD+Dc8d2BvIa4S\\\\\\\/IyYV08hIejpK3iAaHFC9TvTUzu1\\\\\\\/ZkLBhzz4O4\\\\\\\/MbBWZ0AksKygGoFXRUhPjLZeeweq\\\\\\\/cS58VmqH6sbipiCwccD6z+\\\\\\\/dLL7s8YcLJ19Dpb0JllY2m92mnQRXAVFaxN7FX4S8cweXtsVR5T7aCBLtBGX98vORUvNDUXRHgwO8nhSPbQ\\\\\\\/gxocwpy04U\\\\\\\/+ZH6mzpTdg2GK3fDj\\\u0022,\\\u0022iv\\\u0022:\\\u0022bX+5P6rDSc0KQcPaE6pe5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289136819892276dc9b271f971f974cd0b88b9224edf01a90b9afdbffc1be310f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ec3eoXQ3uV3vl3Ch8CZHmdOkMKYF0lFY3fKqbu0sMZr\\\\\\\/gTBttKdaF08x9VuBg\\\\\\\/AEP+DMOrld+YShvMNxKBMO8RwmyAdUWgAqryVqxPyKzCyrYoR\\\\\\\/OVBNJA7E5IWfSQ4jNH6z+UnBas40Gs3L0gi9pH6K9rN6ARlTL9p+8XKELvzd3x1G1FFUlJfSRG7h6yHq3b93aZhE074KbpHXEVKyhPfEKOv5I3rr8\\\\\\\/HIop1hWNkNu9MuxZOxQ8vbP9x58PK6shswb3uyUnU07+m\\\\\\\/jqVCkzpPBqkIHs5oGxrq0Sn9Zo\\\\\\\/hfRva1uaARtAOoJP\\\\\\\/Uhc5r\\\\\\\/2EqriHeauzSyXKRuVsXOVHHVcZQn9aTtl\\\\\\\/vXtFYg8RgkuLrsRb4YoYhhNLt8l1+TbOh\\\\\\\/+gF81A4N4WoXURVXGpbymyUzN9g8lQXozT7IWozVyP+xUj23wWzpRCGNDoQV4eBfhLFJXHQENdlC\\\\\\\/A9ptVn4xr29eyFhQAOypk2Qc7GaC8BioDN2AINL1GpeTR1jfqP7hGeA+yrELLCDekPaLkj5ZmzoYLtv4ctCCstQ+\\\\\\\/6USoiVGmr0cY8o3GShAoenpCEHWBGA8KWEBsNroVUhwT\\\\\\\/Y06j6acGJ\\\\\\\/eILiUDifNKUs4Ln5tf0a3uRjSgcS5cmWZ9anjZCYXAqx\\\\\\\/K99yHQGme2V9CuWSUdTtdPXCiiOb0FISsybS+ZWYCNz9CWhE3mnloUQdOoqi77excDKqbL4vHswL6tREwt+StRKpGgukPI8LDjvdCN3Kwhl7F0DOVPjZhSw7yWDh5qyN0EDtOOvoxQeWmKtFWeZBFT3dke1ZgdRFlbZcCA7DEoSAZaNdJ4A6hs9kVm1ZzqeYEuXSVGQbT0RUeypMsMGdw5tGPQYgVLnfbpXbs77d8CBVpwP5txjRHCD1HlpI1VKpedvbkoF\\\\\\\/Pqt\\\\\\\/1fSVx4lVh8FvYOYIdHI7\\\\\\\/PNFFxOKthlQryIZLj9AeXSeWDDqyoaS6QT1bwTXroi87cbzUTsaUldt4B4UJE5Zeu7gFj5Q2KhyFS0xhp1H9ZEhw05sb2ucEqAzJnEMRBK2uig1XAAx\\\\\\\/LS9hG3num7vHRL1XVc0MakwtvxVK9uCAfJVAf++Gs2mSc5vPc49EPjrZ3XJcdBPYZz5OeaMQPw4fEYGnperK7BkubtyaiNVekxKMUP7kmCsLGERJSEt6M5yOoMtlBAdslqJMY5icvTtXT2XqJDtx9OEbg\\\\\\\/Nv24cMxImYhLVagbtxHcQ0IIyrV2iOiAspmxV6hsi1STaFId3bWhD4hner89ZDjHehdO8qDUBhUOspPa11hTDX5Z3Oihh0nzE5Q8+UBAXCZDlSlI+HRHTkXRzUzqalfWvoiLaWA8TpoIrj5\\\\\\\/g5noC2DoFHLUHN3q2tk\\\\\\\/BmZ9dOmUNFSLpLU0Tk3UuNN\\\\\\\/AAh10QS0+sVH7IujZansyL0Yb9V\\\\\\\/GjWQGGJf1BFUi8NlgLamIap4ZRdYQyFRJPq725TN9T4dBDkf2Lsv1bWkrI52z9yfklrR1drGn6rSt+9Dv8fFb3r7EuZ9KWagdCPTu\\\\\\\/\\\\\\\/TL7rejR\\\\\\\/UYVk3koNoNQJdCK\\\\\\\/HuK2QGEDhsJLjph+3ulHhaFipG6Pm8cZ2hufOAiYgMS34CEfJNeHHE2b1Cuurw4vkizQENfTyVKrbSpR0MVs7WZuP5y0o+3FUke4wgKV4V+RdtYj2jBFsv7u09NblLNGuG37ctorfWo4xUKoGmaqZ2VNZCDAGt6A7sd3AdIt5waZkCn+oCg3N599FLNzdknjV2CdGC6vNxLXCb4EBo2nav5VhWb9SQs8una8pwZSQ9nxwMA\\\\\\\/gMw2xf9zn1gWYQOh4dWCQ61A79zQWW1ywXHFYrs2qt4vN7l8Ki5aoo4kISz8BZDbq9qVOkoY5UoP+mmNnOTp4z+e8e9LK90OozV+uRcem3B92OzYklrT7KN3pHuGFEOLJdGbOyJYADW+Gi3+waT4jWuGvYFbdCTRl3wOg9CCQ9qdTWST\\\\\\\/k2WOLpGjpcUUr4MqZU+tV1EXyC+3EQsaj9ylFGoi29IIw2N9iE8Jt2FqdUlPAvDZzdwlplik4pbtznhPPNm0wEU\\\\\\\/b7qwt4tXIP+TXrNvsjVZ46e9Yzy6Z5sEguXOqRpHb2PfyQd+WdFaJO0gJl6NDC75sguCQkpneH+uNTmWXsFXd9VCpnHYTIxxYsOdJ5uD2FQNR9Gw8vqErOf3PF5ukgTG8DlQ9UjABXfXE2343vn1P8LkqNLHATPUmKRjFsIhX5EjiOk0tv1pYHNwTNQatmftI65wjgUFD1SkNUphLc3oGvy6zGbivMujiTtiv6a3dGNxaRsVLclGVyfaV2BpX41VcGmVn8d3oY4Jc7An9kQBBaBVnOquATsQKPQYurXlKFVAew7+ExsvyyZO6AMzBl4eLeh3mIC8EMTDbdqfVAtjnlxs1dcX4Tsrm+zChc03v6GNGi0cCzrZTZTZkwunoyTgIlNlQtSmiKV1fXW8gnci3CL0ZURObIJcQmye70sMgfRvFYTXMeoj8XE6Ot1Z4knEVnxaPk7CUL+hQkRPPcuQ3Fx3zXeG+VmqIj25f8OotHevek1YEnFlV\\\\\\\/KelwM+XFuM9np33QDjSFJCwcyXomjv2eKJ0HrsjN8AOFkmIHx59NrIEm36w8kY4Ymt7KDi4wCAW78o4hWluP9+qUdZELbm4xjTIH5egHtRjN6fhW8mV2C6RfPQchpeX7QzjV0S+sN\\\\\\\/lVwJlAHiEAKkHymYr1ocOLxyf2bWlwAMqLJAvkiPRC5pFhFmVS38fISp19gBarCSaolrG7B1WdV\\\\\\\/4KF1Ue925w9czrR2zG88cth79OkZ77N+zUuYmFIfX6IK70LLiFgqLtux3zFpzIxC9+BWW3w2Kzcz+qtUn1PlW7ksapMjAvFeDgPI10lacQF\\\\\\\/GRiWbPdQnhEfx91Y1fuxgwfJhe8gBYRtZ8SpKKxonTTsUT1142pG6XucabWbBhxmuuMBsK4g69lGHXswjDQ0PkFK9BMqV+lGSaAAsLl\\\\\\\/i6caUz9hpMSBMdovY3zJI2WtTb+CKmw8SYQQRJkKxVmRmKixk3hggoLmOA8ZDsTsXq4TQlwNv1wVCiMtKKoKzd0rFWrWcdtowFfLv5FiiFNn\\\\\\\/eyp+P4Hq9DyD3Vsk048ueks5uMO6mmzFC688Ko0M+onaQQrclage4dUned4\\\\\\\/lnviC8QQNbFl7U+srIfmUY8V6hp7QeXpsEXRAxIeEwNp48bp4bvpK2H8AlQ13SfP4cCaoBRUfSLhhl4CuiZ8B\\\\\\\/whq+crqlK4sErXkOhAQbnLca3AuiqOw1TryJIcmMVA3tlEk4qFbiGeKtILglwVdKBcCRP1KuPRNVhIB0kkznnJEKBn0gcBjQofH6piiaYP3o7kaRVNL6+9tVRy++o\\\\\\\/kW+NdfcM6RgfdNbgkGRMcYutBDzcbFRqMZoBUDlvVfLI4fBoKvrlxtkp1rtRJTYOJJZ3+I6lVe5TfN0eWpJxPRy0pteBh74U+3FSG72+GUFW2FMjxzW+6HXUGn33Nmo+a+zA+kTnGvwVKd5dFIdeeyqtDas+6fbjPhDNniiwqSelCVOhMdOyRjhGac+hec4gR85ZCtYMqsH2fegL7lD6zCxOZZA2GBIpaiIVyWBZjZeL9VjBsgl+OSWNe77cV\\\\\\\/rnoQWJuL3OoqW9dsDqnZMBhknTdYwsixu73xy6PzvCPHAsS+8nohwIIeKnWIyxn\\\\\\\/c05fA8WCPMc2waLzYbyTgFVQcukYeDarNMBzh+rXQOarSYb5z5YVIHXIlY0P0AnZrudo3oHh0yPD1VD2SsSpLNP72mSpdzyQFx2X\\\\\\\/fCYIfcJMUMPhDX8rpfstvWChAkFQNDOfR+B+EZGZQLv6WFeBTw3r67adxWvlNHJ2RbzAgGSAK9vIYwDlUUquUYUidM3fENv64hodqGAwa+wUKATont+fw5d841hKG24dfYUOG1HGG95uRcdidQn\\\\\\\/LvubiSFKkZ89OhWCSoNxW8Z9jwGfEY4vVA6mx8qCogLrTJHhXfp\\\\\\\/furyCSn7hw\\\\\\\/tBnUShbGat3hP3EWWqQP9ruGHQovp397OIX41k8neWLgCWBLJMhvHniPH+\\\\\\\/f0+VIwfKSYVJIfkA44w\\\\\\\/GSenLfNHfjNA9gFl73yIP4xtMqvFCYPzam9m1RFljjaMIa+9ihrT3Vk9O7ozHthhOz\\\\\\\/3cdsVbaZ6kNxRZbcAEx6n+qIocidNbcMlaO1wpTaJJ2pxCs47XW3SgpjfxPNjAVU8VQWb14az2lVdPjTPiPg7Khj05jAi6roeaLdFEiFsak5CedzpknM5mv9w0\\\\\\\/Ze7ZpTYOeCFtwIJRjJafXDOoSN7BATz3gIlfKc14pSW8NWSM8fhoedJ5ZEhcVVW92BtjiyVcIThP+rnoVbmwL9tqUP\\\\\\\/NbO9Pta79NWkXSR+UWveeUb4lxdttKWAAae8XdP\\\\\\\/FjbYztZsL6yn+4MU2DF7D87LU0fdGalm58NOfuEafr4MtvW6L57krcSEFBfc15+xgZW20vfZuukRAPW7Af941FtZJdxu8ihdks3qwiysPuyTle\\\\\\\/FAK1XSey9\\\\\\\/4oSZWGBhq0a6pjmDLkgWaN14Hz2GzhDYun818gcNBPtcUdYeReQROkPnqAvBvtPERQU7mef3FxYYYnbeLtiWSJBhIA7WmJaU4fczfwmgqciino9e5v5S2I7rsz7FiNCtEXflYSxmo2oGHF0U7tMOWg9wlu2tfSeoVAnwhtNbc4eBP+uMB32xgamnamVPr+YD7XSus9Xv4sEkPqbwJqqTCaHYhbNYYNHxDHVRLVU4A1eTOE1vua9ULyAh48o\\\\\\\/B\\\\\\\/+uDcBYbATzolxU1oN2Q3N7G8b2bqvRUrNpXS4EC\\\\\\\/x7nq8r9z0DgYqwj0C5nNiqha+\\\\\\\/vmNsRlsQJKoILG36HTP8uXa\\\\\\\/C\\\\\\\/Q8VFy5WDibhTPURbyWByW+SFdl1xqZ8CTQ2SlzXSsQmLXErYAUZLMbig8hNddiWP9+A94+69Cx6sqNySXkqvVUrtlQPSZQyZrkob7LGqikeSezSzZZ9cQtsoGI\\\\\\\/OgmoXz1KC9kNCIGGGp8dyH2kRk1pPt9VribzjMk1mxvzgie\\\\\\\/bj5NCQX7Lj\\\\\\\/C2YCEYQwm7cWBl9amR2obAkkssnol2iYWuwAzwyASkRFk9By2fQH79HPa\\\\\\\/LyvA2P6MRvg9BUzS4jruvAy5zVuELTSxlYSwAXKUpe+UfIA4TyZQsOM57DscY+Ecuhrtda+HdxhZ\\\\\\\/Fxunz27I\\\\\\\/+lh0Sj9ciX6KkzCTFM3ybq6ErsB+pBmpZur0\\\\\\\/Q+7sAd8hSzfHSy5YgWx5isAyamnOneBd1WQhjcHSqLOknDPApIq1\\\\\\\/HRgq\\\\\\\/ojNvNhEt8SgnEvSyaLXRRCeIJ\\\\\\\/ii1n95nD+YLbnCg9qEyBzGI2iTkX8\\\\\\\/epFo9NeOeekAJQ6LNlxHcN8FORpHoCwUiPecsS6\\\\\\\/80NU\\\\\\\/uTP6ViOEnWOGB7Uh+L8GL1Xj7Dn+\\\\\\\/aNEwpwxGRhlcXCX6NDxoUWwkNo9jgjxlNQotwSJTkmuW4APDrkD7XpwzARsxt\\\\\\\/dEqLuuI700fMX0zrhD2Ss6HjP2nJDcoNk8d5zxofSmffvGb7jNZoJ0ET0KEtDUhHxlAjrzzogEmyDngIgZsll+ABAxcirO8mygiXqKf6iHY8YYJ9PpvvJzam9LUIt88RFMiH\\\\\\\/TsEyCBQrE4mfdxvt3K1gmy0OcrkB8VIDHfHwte2RYuoM5\\\\\\\/xqfGKMKu76ymsE2QrCQQuXTx6eAxF05R2yxbDyI\\\\\\\/I\\\\\\\/IMlfDO3GZpA9F8AJinozKptUFK9v3xaZm\\\\\\\/s1bpp\\\\\\\/U+4N3ssFjIaCc0S6hVE5tDdkFrVnXbXGKKlnTqkd6crUBJz3b9F5IkvSxVv1a4CoPCEKnhcsRYjtw2UGVEwYCrsXxtNaxMx5kHdRAeRN8A+1o3ApvZ6RTMz+xC5NKH664TuhH\\\\\\\/zC14JOlCZUrNnDCW0KnBq+6DfqZJDaBBc+NfvdbFS2Bjo3qeXB9gk2RLev\\\\\\\/lI4XQJnT2zLx+AkZRCX9gAcJUrgKfmli9PHQ27OequBBopj6z351mYuM3QLn1EdksNkHtNT7+ysMxTB\\\\\\\/nuCbambDGzMjqFMe7vmsUfgozuNms2fijIEM0jWUaRFe0FxEmffdI+giliWPWuJnseVV\\\\\\\/tg333EGzsQRYr6sKsgE1M0BwUBGhX1NN5v+rArIyONUReiPRyF3LJDycn2yafa1Sdh4ZY1nWlAZz6P9AbHN21qsds7AophYydoXw2NQ7i45E2jW2m85PuJx+oUsarcISfMVPZ6Am+NGPdhw7HfS\\\\\\\/Vwm009nEo\\\\\\\/dYl+UQeruhzhxFigYvoAym7anyuL8mswP1eRfmeBfXXCIJllAm3avWD4S3ERJEcmK+jBCC+ubZZmKn4HzQt0UAPsPHBAfs78pWZrvgv5IbRnbdfDr4kZ5kQZfiKQvzJDYWTdYfgDKKwRCE1olXECN5wCE0xWylqvPPN++TTLrhT\\\\\\\/KXYrd6WXrI+dh46uTU7333912GNpqBicXPVK33JkYt8KQel\\\\\\\/gqRMyRHGyE9UiENtfSOjls3+09Mh7zKkTEfVS2jbpX7WpmIKifs6C9y36qTmiyhPFP7WLkP1qg11ouOQVKhrjLHpI63oPect+HTWCJ3FVD2IrUO1qotohAiPjJn0MgeXU1ccGcs2A9Bp6rdPcc86yzS6NJXF\\\\\\\/RSrQ+F8JIawLBu8dKXR4HqZql2X7CImIlvSClFh2mGjx+6aJYDRwbLvMg7j1HEHj7hQsdsYUx0qsXtS4tPRhLFHRuqlMa8Hr3GaeoP9SntI3nxIWsETqGR0l32YgD5vWHixTpTmSPjDd4rZcoINShofYCPFyT7LNxX4OabQJs0G7q9n72dNmuu7FIH+IUtqnRYC8btYi1rOrrij0C4Q2ubHkNiIOJn+I+NyMZLv\\\\\\\/52lAvRGcdtf7OYN\\\\\\\/kirnQoRD4QQbQLgR8+Mp2WhL8I8p9yABo8cHXrwmrJkNvZRZ+QKNTnWMAsLmkhjbuxDGcfTV8+Ck2fkrf\\\\\\\/tf3WHBannhOvFHAzkYeA63ue3EB+\\\\\\\/Nh5o1XD+Q0twm6fZ9LjPS7APZhTPiHo55jOclKwcQielOoHhPNsRQqefyOJeeBV0GEefLcGWyMLcXdjx92GBLu60a29T6Ut+wXpMYgNQYinGH86XDAp5KUW42OTBNsR6GbwquWvh0L+fTOHXMbbrwoqlKRxLMFbvmqqt9fD6g5O+so8J\\\\\\\/Hgw3rzJiM0tux1ubM0pHX\\\\\\\/C\\\\\\\/2PomgiX2Lk4NW75AFBU1D2WJoXBU+8IvHriJQekhxCFuie6DnDuAbmBFttSdf6mtsBIHkiCLLXKaVHApdS0obUQj7fnglOf6phG\\\\\\\/eRt33J9P3IsAzvG5bIBSDqE9jqTk\\\\\\\/lJfPzrAcGNtCycEoOOeXvuq6iEJk76nVi+xy+gKmXrolsjvdJO13F6pF5O0TV5Mm8j3Mn7sDtY+1O4WVlXA0++rwpJ5nTqYPYdqRn3cYyly7\\\\\\\/76K93C64n1ejmQWdjjuCsWs9OYUgaYN4RuvltKr+WCsFq4scNpUIv1m\\\\\\\/yd\\\\\\\/kyVQUmbLN\\\\\\\/m0yH\\\\\\\/LPKXpOnsq8BuvYtE1bFUvTijcElQsz4rH8oGZymCGYN\\\\\\\/RCQE8uoXq0KApp7ZrfzTQlIgDXgXMw9sKnbiZocuIlYOoUXXcDzwC3KT9h4x39wgCitaZwqNLXAcABzhLQ+M4sR5PN+qKKCPhKVvfLLk+MbJ+oIHoWCadEOnPZIYE5mzLh6k7s3meydwHbq9jc3HImBxNNbdzBtB4gVsU\\\\\\\/vxFJICECsFJp94AteO65z4T5iXP0nRcUPkA1Mymiv5O8pC9enXjm12dgPM5aQf+WBogv4noKnOb35C8lgLQcPZdGEg4evz8SQJb41MqRKWSnpaW9iiz+mEIE+m5s2woZYbh9cjfNYtFqaTxmZTRbvScn82YcchSHSMSLAeSSfh7HMoXZVCuHHfmnv29058mZOWMcnheKdufISFwJCiyflly3ZUIjUI12mLJ9z\\\\\\\/HX7XD9l9NjZ1nmPfgQGq0M9mBJk+9DdOxsthb8bK2+Qdkl5IB5\\\\\\\/UotQHDWfiRZ4JGTv\\\\\\\/HQga5bCnCkjRFkbqsNXvdGgwzvZotBFQ2nhkcYG4u1heQelW6n2XgPsPXexEDLFTeBFqLD5CHvY77+scodBxwkrzyJAznHFpWOC89E54bOlyqA0ieIIab0lDAbh39GyeOnfGvAx5HhoOjU15Mma0LQU8kqfjNPmsj279J8N2DVvGArkF5A\\\\\\\/XzY8AwTdrCFEdc6vy31MRaJMDnOnSJqKMHjWaoQ\\\\\\\/mgrMWsA4sM\\\\\\\/9A1QlsATT+BoU4EZ3\\\\\\\/c3DKzW7lok2KxQmrpVpcdDyZfx+oUvAd7LbGOdCLlHmfS7YBrOGbS9kAiyndpx8KGkRdcXsdfZrRbV82oY2Yyd52kJP1lOhGw0dpKR3083OuGovWvxd0QLrOcr14NMQQ8STxufRPcvuijUITt21ZKongtdsnkR75\\\\\\\/yW3xIr7k4XbShz5\\\\\\\/vLjPzlUdSTvwmoE6CUoTcifNWD8bfd2VMk9Nr2Csz7dz+pvzhwveRTN1UcVZO5mJKjIS33uraOlOa+EqUCMOoflPRQQULeT11zicS2Is0Kv7xacJSiw8Z3o6cyEtrhAALN7hkfcxACsoAsH8AlVZR7sWf4tPIPM0AC07AplliTkQsLywq4YX\\\\\\\/71jfjQrcMytTL9QOGJUoPXvQutwx+A\\\\\\\/pF8od0SqDxRk7F9YzSHlxnvsd2846quM8m3J7ZmYXTjP8jvUGYr+zeuBI+ctvk9bOZpJqhfhPe+8qlU+4gUSBZTOGN1m\\\\\\\/gtPhJJcjcV3lbOQEp++7huKaJTdNSUreIVBZ\\\\\\\/Ht2Z3ttMQzujFCXhocoH0a1J\\\\\\\/ATN5PcJGa8eokFBI83oyup0VvDxfo55LxZFyRj14T+DwOUkSyvlURBHWiwTjNkctOf7VgdNaRtKBbMpqkrRzCLJTnBsctHdfm9h7QCxS3isD0PagiETZN4s1XZetskqxDpLIGEf7KsJUfXOkkvJEI8VHyF\\\\\\\/AtaNd0nL\\\\\\\/Nn+85GTefPe+0UEe6x0Mpzlap8t3HFdhH804d2RoIQ9HWjaHQqK++7FQH2NVcEmWAC6bIK+DrnrkCC5OgAk\\\\\\\/ZvZTKSDQdG5cNHLexziAAHntjsV0f9k3mcxKYZNQhmARruLVQES3IXCEulr7D5leoANvHqmH+TRzKX+8WtZAla7f39VziGObybKV7dEoxFg+6TxDHsgRsHU8QeBXzO54\\\\\\\/NlkqcoziSDp0XJyVKmLWln51hkfxRUDiM5W8FujknEWb9LqtRtyXHCV4U917CC+D\\\\\\\/YuWMKWhTnvsFQbQK+935qprbx066i8g5LkBNC\\\\\\\/FQ9TeFOzBVo3PMdsmmJimmrfWe3NnQOnnxa6CVpSe6THNzCLFgXubhuQkxWTb8ttz227QIThCJVmkyPI6omA0Uz\\\\\\\/mvmiILAqevhZGJZibGrA0hQdo3Gn0sngZQAS\\\\\\\/6cbGbrdD+uwDzgiTakrVY6\\\\\\\/RNyL6ZR+msI60rACoyDIrMKXywHuwoOm4HasNAj4oIM6XQ8b9Lz1YaQ3NHWN12os696m+eox5QaWV7SDZ2V2RvpHfWcsHHXFRyI4VOQYVy2N9+uplBFjwW5FDNifO7B60C3dYQmnfoM5BRfOyzuq1Q9QQu6VTgEiARlgs4AspG3ZzvaaWgRRJbYQQT7oKsEJ8LyKd99h+GVE+F+JbYFzIN9CI6TD\\\\\\\/JtxPnWLozbGep72Zqaxk1egkTG4YNAKrC5AomlCXKopZ4TbG0GH+voWexJd0m6ubkVVcUVH7qlSuOhNHndFxCMyrhKIAehCWJ5TNKRDPGQtdgERMMAGCyClFYkS89gQZlknYpmI9xSc1XICnZZlUPxEYndWTp\\\\\\\/ZjOk2S9jew872RrU7istol8DgqK3B1Sh2Fvr4akw9QBui8yEp5WyuDUP7tgIXWNbHqmLU02652\\\\\\\/GsKaBFLdbBEYkeTWcAPfZSUXx20KdFffHbgbG6Hs+juh4imHhQ1oju9Ib3lsJXFfwSSkUJCGN498QgDDhEzyc9TAFV6JsFrAUnD0MWmvIzAnlYrah9EQh9zIXhDApxfvQQQ1MTxwOO\\\\\\\/0YmgZ34amLXj1PjlWeIdh6v8CbK5fIFPMwSG9\\\\\\\/gTKZZbaqFojbxGmj3QysN9IOa9rxKsbrqHJzz8OWMCR9EDkNGy2YAHr6sLHdNt27Wg3amRICL7lqQ5EYR7E0t+AqKfnMN4lG9RRjWCvEmoQ1Jz\\\\\\\/XCLVXDB1KWJJZJw1EiTEWEPtT9X\\\\\\\/vssIY9ypmfInITC4e25vAJye5dIoBaHw4r+AlklWBPFCUbYGxD4SXE7Ip75py9llWeYP1ccowUuoGCgeo0pTP7Cs0Vq0c+BJDScCfRC2+8gx5vuvkzXrFsPyUYVLr0zgwtoUZC\\\\\\\/5uqcl1Z5695WwQnIcVxvhsP80oV5SBMLhU3sOeFdMC00XbIlAvldGDHtnq8fbs8aF6sqfCUrezECGaSqSnlqzO5k+hfSqAt\\\\\\\/W9Oy3v4pGxxUIl\\\\\\\/HxvjXuhh236XR+nEg8mgSUzzub5iWHLpl0fc0oKXu51ajZnDVA3AUNLEAbi4PNoRYkQAnfUQcmLXpAilQXJJ9v6TkJbT457padLltoKScLnd+wpgyG55xDw6t1oN5Zxz0lP4ziWA7xQE7LNwEVI1Qa4csxtqAsGtOshmAfbEqVvXdDadt9lnRkY54O2pA5G+4Rv0KNZozMgWw9ua4dUqrl50DTNIW7cOf5Tl\\\\\\\/W1QXpGuG9fICVtLFYic0BZ21HR\\\\\\\/otGbFdX0oDoBd1kSnygpMszModTv1Wr6CRTsz2oagViCjK4pLaCbhKCa8njiSK7LUd5ea0vlcfO2r2T9GfQpgdyNvOSB0NgbL4PIp2Kb3T1kUPOftFZoXYXchi5SPai0kanLD4ZnSqS4+N6HPNcWw9PZiPsoBVck5E93AniL4HzXHTKEAEpOzr8w9ZUo8I\\\\\\\/Wxiq\\\\\\\/a8r4Gk1sDMt7n6ZbR89qcl5He2kOLOgeHo2vBTSZyV4KgNq3VfsrbtYCFUjVZMto8hSm+q0HkgppQaL9sUKTFlLNWsQTVwBDXOI2Bm5XpdRfKkgr7IpH4rrdsCWnGrg42wnXK3WLC1r125Zj\\\\\\\/70RfIpimO1F9kKIBDXNPB01JYat4go0mbMGU\\\\\\\/3AC0J58I7xyqPr\\\\\\\/PkOFyRyScsV8nSTH2f\\\\\\\/XLIJ5a7l+82lzI\\\\\\\/UKzyYNdwiAZynJoVQ+VDeGmuCY8dK\\\\\\\/7KRUofQ76Ov3tUIlZif6s0GoKqWloTec0o8wsUVBlUUETQneQebaCX\\\\\\\/ou\\\\\\\/kETveNPbMTTed6n8gVj8S2L1kzRpjHCcVxzi7WhWEZLJ5kHKb+JOUrG\\\\\\\/34ayhWf6tajHPDKVxiDUNWmJUnd3WRup29gxe5UPlXP1D3ng5j0n4jnJTOsqedpFmQTZRejM1oGDtVs1Ndcr\\\\\\\/E8Fuwoguu5aTCu+7Snw+4dnNgjMvXLTNszx2IsIXlkZJSEyj65FjjJ2eCAYM0CWS43mYK+vEpHTQZ1DV3\\\\\\\/ith4X31Zb02\\\\\\\/PcVEsEsnzQoPJsXAMVbJj0CGGMd3alAwJ8NMYCUUm0tBlyLKCPuLgqOo\\\\\\\/z\\\\\\\/mE+Z+2Y6kRCHT+mLByY\\\\\\\/pYyUFTgJGe7HN1oISF324tAp74GSiDVHwzGd4a+cxl9eUV5M4S0m3wUFlG8zYUqr8CjOPrlvh4xvOLRt11nbISjY93Vrc19aakh1Sh3jUMymbT9vflqR09KGxAycscNTRt7ZaZHFD4gedAJR1g38wS1uJdIXn\\\\\\\/JvfIefxoHb7DXfWVT\\\\\\\/I8Q+f\\\\\\\/q+mJNL0EeV2OxnSf6zhs6Uvi0py76xaWmuJHWtMaiy\\\\\\\/uc5Pa18qbHAx3LNeAAHMMYPCcgG8kQhrmpMYDR967rVws0Mc7H3WXAS0THI\\\\\\\/+lTo0SxQe6FoATLG6XRiEMbrcwk2IN3rhilZI\\\\\\\/EXTFCPF1FBnLxp2zv4jMInOEy23W84q9Hf0S7JvS+WWFHwzWoVtRMT4DdGmY\\\\\\\/tcP4VKCRSnjKi31Bn\\\\\\\/gydIL8eCts2IlLQLwzXPlAI7eBXDAfrkm57\\\\\\\/Rfz0NychtlJhBX5XcjpKeKj5KYWLctNtcNxvA9Ho+kwxxlgN3lZ3HvDvQ2OXGAQNvJdsr1nVsyVYS6SLfw1b6PuIdQ22BIkJBueFLa97BD5ApE\\\\\\\/8xeJagnHrfzn8kjZahDAs47VGK0tmhErp5txFz80DdRbJPDi1Hy+tfrlp3dcKpcGz+CS8Yy5qr1Jc0sG16hkBkdUDbQKqLCyUz+xVYrpfkvuCpUHfxoDqo0KoHjLpPcnSpuim9d7j7eBYpEvygXzPUfEDa5TdoX6\\\\\\\/ZXO4hjj1G8b9YSGhrC1EcbaMaFikiW+1BAb0gBipR2R30ZU+w3+qhQPOQNgPjo+Z1ZQMz5hQOwyBvGxnWM6DemASYwmp2kANcwkhH1JQYeNNrXi\\\\\\\/OhZQT4GDoWlgi+hj6cQZgwyHEFmsRM2s8YqwSkK\\\\\\\/axnNKo0+XZers5ClSWBw4t3A7MeEZXmuHDNZ88NgKLTLXc2HfhzNTDIOGW2PdsfZ\\\\\\\/3i7OlxrKuF8W1jLZRsOztAdFWuvPy6TN98B+Th0djGHffuzA8XOyWzI3\\\\\\\/pKyUNVhEShdL2Uen3RV4qdxjceLbAcCJoLdpaMYp0ZabvLYgWoF2W0YWpMTX8iaQYRQG5Irk5iSkIurVcDVSgzoJp6\\\\\\\/OUgWkMO55UPSf8ntYw2M5chBHHJRv3kxlJx+IFw4za1\\\\\\\/lkkPomD0qzEq1UDCMxLX2B5y1OtdMWu5MaQQ6zDfz3DzhJkJ0obzjKZWfQxnvmh7qzEtD8SPUbQRDFG4h6bw2mPebCrxBzx5xoB1ea7gEqn7wqc7ZevAd9OYvSE31TAl9HPRtfOXH\\\\\\\/RMQxCNZtaKiqgrK5nJI\\\\\\\/oS2dbZ1hcnWRt8QG6CtKt7P+3eKrgPNU2\\\\\\\/ZboEFcf2gJgPjKTm9eKn0vJ8kF2vSCN3J2XLBUYVxetc0U0gQt7hUQNdWI2Irgu3urLzzsyQmP+5BsH7dKi53mxOWL\\\\\\\/qkrHhNzWlskgtm6wzh4cIUyygyJia76QLPDq31jPZr6R2S6G8vtyVK5yXmbSb70Kx6KueUqxazo6Fsi4vfIwOvXtRJsQXaezwR9VN8zDK\\\\\\\/rJw6Z38J8Id6Ys3lAZRUfadQA8cFcqCK4W9m6tBdHtkiAMqktVmSFfik9eyOvg+zxUIfR43UdXbk78QotkNGL1rIFRHw9AfbKAfvEiz\\\\\\\/bjtPp6jdWgkNl87Z963h9KpL\\\\\\\/WwUpn5J0mOD0x0MvCHE09cuIvKbyRGkO0JrcnT\\\\\\\/E3NBbtGUMBI+dDtvO\\\\\\\/qytbe0W4TRPjlu9aortllEMC7zuzhdRh42JWM0bPdpzdT6wF6YECrpYIN0wG181RsUoOE0OfuCnEgBODFiiUdDX3gJ6fDd5bXSCEpM\\\\\\\/hIRaqrk3+06WZMqI5mX46vFGraszvRRnLaetHFgOWpQGA90crZWc8\\\\\\\/HKipsJjtEBEY+nonwJrwlEWEQH3RnDp97qINiHJmlv4LnqCBxzQzoeeh1Nkvqg0rvNh1Lg3sBRL6dAUEdwiMtQi+8\\\\\\\/k7U53PxzyOnqvcOe+oithTCdyl34E\\\\\\\/iRTgphvJpDHk30CsxVa6bUoemdSDq8wPvpKAPfUEYaJcm1NKtN7DjOTclvUyuDauNksY2H44kH3vC8DKQ+erENZgRVqq2jJ0l5mVdbiS0HEJbNWy4uqNZmHTykasbV+A+Ml0Y7fbfq3nH2p2Celqrv\\\\\\\/pcASs85SAcJ0gpFgfKBdVcWNkNnBwwDuIlUZGbxDisP3dxSpljPmnKuIBMJORRBdYtEErYtI+9HznKLgKf7cNqdoSWwRhzjaBo\\\\\\\/bI\\\\\\\/7Y5kvrhMSbi0t\\\\\\\/ek\\\\\\\/8iPDZMsbqNe+dq8jUgF8leDg1+sVylmGmh1LmFKX4HkirsNRHKsdJ4Z8jyQatXlxsYDNn\\\\\\\/P7mx4yVnmfa0YUrxt3F7SMomZZzz6UNn7AiZ0NCUia238skbQD+QR9VQ5JT+yOD9fUU6qTZe7\\\\\\\/a5hLBrxXvE4FuzWED8e7aPpGoSMi2MwmQ4lpeirmtB4\\\\\\\/xYyyFQ5EPeup2OY8xow6xA\\\\\\\/hdMM1W1mnuD+JWPzhGap65BI5v8MzheysJ5CQgF+J5X1nvq2hmr8P5OAMQWxyqCDknvQyQMpPcVcJ2708hS96HJCIIx30OEEhwpUGTlahdJ9EzaPKDpiCzpxL6iJRyauCoSwuoJsGG+S\\\\\\\/ghJlOvF+Qd\\\\\\\/liN2dWBCnP4h5+pFDn7GM5KRnsBc9o1auIyWcScrGd63I8APeCXMk\\\\\\\/lDsuQO3vLDlJRtd\\\\\\\/m2BPyBwDJisYAwmEdkbQIfdzJNB7BgETg+s3QioBb031TS88DwZd5TQyOJhuWYf\\\\\\\/D4qm8UKduJO9VhHJ6bB0xg4g8I8b\\\\\\\/L67QlbTspDvVtUp\\\\\\\/cS2ISvmAynSkJsKas5CppKodQ7BAfS92RUaMRYbsp5\\\\\\\/0w9pQcTZaf\\\\\\\/XHMWiPKNvcrwyhe8DVPgR7KH7nr5wrddTnG4Fon+ZO+dMmgmp6p5lsUbMlEpdrM7\\\\\\\/NZ36pXiVuxPMv3g4XYHmwLBKmsMcI6PLwgLBJMag6gHjQJCd1YBqp+Y8UmRK\\\\\\\/+ggsmJ1rz\\\\\\\/aQHkDSmSTB\\\\\\\/3rmtcVs5bIHFboK5h2cMUl8\\\\\\\/O8YrWCWLwqn6DDKJkeLrP2l9txXR0+hvIDpTZ0yu2nreL7jHPuWXfgf6t8SYjX4cjzIuPd9PUu3kJyTaUit5k2R5oTv9nTRgN5btm32641HgStYuVRNAELvTTJ5HTbUEZRSmr4xkE+9W9i0117Bpi2j6RIBafTSc2axp6yvFJFatEkSZCBinYy2eWEPD+dUqRyTFfIMy4G6Nd882eHsrnS5zRNvspb8OmA9wryHMvNO8A0\\\\\\\/IWIN5JvVX14at\\\\\\\/TNHbEGXXPp\\\\\\\/86dtmzh0KH+8jNaKGU9roe69vk6IzUhduvnZP4RBP2OVhQu\\\\\\\/83EczvuXeXduLvOg3ZOHPEmEqlof3oaRkKGoAhv0mLiedkLhlc6Nk7w2inj2EFiomx9GAVcSIJAeUboqjGTYXgdw5SxWfWZmWJQFn7RSWrfCsAODY+MVwPu2flxxXbMYZenrBhAnDe1EzflSEU6pq6EPVflOBDRpO9CbS5r\\\\\\\/NGe2M4TqLSjw3+EOGyuPF+rKEQ45NU6WEl3aL5aAmGHqv\\\\\\\/68mFGNFWJq020ZbZc2IUsW64TYME2E8tk0\\\\\\\/Asacmx3+tN021BM7v0QMvyYuSVnPbkUKlO40z2onx8VEGiXCxEJ7wfuNugDqD1GBGRBbYyWpmCzFe5q8L0RgpByj6RBrYHdMwjPE60uFnBQCZo\\\\\\\/m2FsaoRcw2F7K+KQ4wPtFM4RoOHwYJc8TvvrFi6u+kWMHz5uHS5lYXWIC0JMv\\\\\\\/yXNGncGHthB+Ix8oveKaZHbhglkfzfXy\\\\\\\/YweWN0YDcSZfCC7JqOzANaf7wjaGJWH3dVFnkwqNw2W4GGJXeOY1yvwzagrWtkdc\\\\\\\/R3ovG7xBg\\\\\\\/8DT8cJpnO9uuXV9w5xjvivFYm4SWuPn\\\\\\\/BMACqG\\\\\\\/NhQuihUbbitb61T5USAsAMEfZSZfhg9FC8y1PVfAuHuICFKPf4dg90wEGhVnnZ2q+iCWGjEcgTOmSVQL+lpx8w5Ir4lQqaq8MeGGtNBc99Nomm8U+zQyaIObivb9p7P0HXaZQqWn2hOX\\\\\\\/7gkYVRHxxkFphM+yLWHUb3Y1KLXRMxknIj5gWP9gsGG3a99qeRSmmHB5\\\\\\\/oQjIevQUVgWFkEbA2dgTEDhEZS2m8rtYmDELI1hGKInkXNWyAwBsZAN9ms\\\\\\\/41LbaHiKKh78Z5VSM1scFZ\\\\\\\/UHuoKxE8WIOULWqRCADbDdmFB2aZ8d+bzqSKmUJBtCSNP4zml2jrgMkYrockyvtRpnl3\\\\\\\/H4EASiaqjy+muPkidVD6ovmNm7hYQYcG\\\\\\\/sxCICScAQ1lnhbWK1xZetaVF5OZi3i1Ja72rS++PVUyYj1zKf\\\\\\\/HFLw8hcwRw7OsH52Je4IupQCAi4E+D3M7FyxjX0kxnLoD4tkCg8kpomPDCcqWwL0b0yxsrtKDFlhKOx2JyHcI7EhaAywAUVRu1T\\\\\\\/WWLpI9oDC3nFfuKTUbCGJVgJxB082LkHmFnxFV5zNSgyuB1g0Ptdzcv\\\\\\\/uO+aAAhA4kosNM3HMI\\\\\\\/4QNd4XXQjJkABR4+Uta5DHqeeApa4YJm9Q6La69ne7UR6zgL3fQVIwMKG4gT3TwivbvIgIwiysQ4p7o\\\\\\\/Wi1ucBVAJVa6nRtIEXYrTKngKUb9PDmDQO0J\\\\\\\/mE0dRO44XlJlGpSVULb\\\\\\\/41FfzZkUrZxQaieLSHbbg18o7zhGNA4yuRSRMU0Yl+QnqKCif\\\\\\\/x+I93zenAIYtajRxulYLeYGarbYgJseiwcARdmS\\\\\\\/7\\\\\\\/K76DoOS3Ak+BRHX77TYg9mJqjJSf412ntuNmz7EztW9vJjWHixb2Tte106luNlzCB71dIib4Mx+L2f30GkjqdsQdo8s3fReEhwjsYsq9a0Pkf\\\\\\\/hOqScU7V4YT2Lqh2rcGQGHEDnEqaeQwJpZXmNXh0zGhTqPtv9CRTZLsHLFcu0Th\\\\\\\/lOzIr2s4nCJk3DQT2PWq3a5x14gqrmaZzCpSvPYiaDHyjXo1JTF0aeytpA9IkHJCh7i7wzfux1iYyJAMjuvGf9tagk161fY3NccK8nC1cISD1q2uPm2Z2ldXWf4BXx3Z4EoZ18fshQW3375W1lV2tM+vYPkMI5SZo4aTnGSB1Ib4HWo9zJaKpv7HtJGxqBMIoiE3lWdcZaEMUgWekObAp261OXzSUvrHA5C9ZzY8Dk40GUnEpxpoUsf4y4n3gQPHLxeJqZxmUlVf56b1qnr\\\\\\\/5nubFHLXS8RZT9UdqdhLUAJ29vX+S2aWFmARj2OTkQmmDKTh86KSsF00XkmqXZp4GoXrXqSAHZ3fLcoIt0yCrdDf7nTcwGngUc7Etz3nKOGG9LD6zGRfKC2TWSSYV9O3uw8kZrlzjCafsBKW38n40Q1K1Ox4rThcn8wlXVQoT5ay+FRZBmJvZJwSvRmCnIdiJ5JSz4+CbQRlPe6c6TQv51SBOr9gANyaKPGFTL\\\\\\\/ssQ5Fs0mtVUvnh21+xtCkSDw2K6+r8DsQk05eYROvtAiwcY2aY\\\\\\\/OEXuAnWwoMJolHI\\\\\\\/IV74ZBjKmNL5pGyELYYPZSj66jIoGA49x7BqN8YLRYj+EelwmH1Fk8tpCCvcxmy+Nocf1N3M0RIAJs0fF1NYxQYct3nRfja9Mu2pYqoybHmrZ0tLGUH+D66TaTMfX5YyAqSS8+T\\\\\\\/qTxAlmubS+o7\\\\\\\/Azp0w4KqbNx2Zc9uvLi7uk5Dg+xBpKM\\\\\\\/CcWZdLif50SQTfUM7uejeDIRRAxX\\\\\\\/DHXO26dt0XMoI2alJ16wk8bOrqlN2C9Yiz6nmg8aQbIv6sHFrfZ4GheSD0PsiUJWQXe92Ew5\\\\\\\/6OL\\\\\\\/qCkADxc2TiCohHDtDxW5OB8jPf\\\\\\\/A5X+W\\\\\\\/g7cQEiq46dpVc7bDSmMfl3b4SEVmOWavo8R1Vh496tl++SoTcjvlnqlk6b2nMmhv2QT6kU0vfnllz8oSaBFohqq4tdhl+nhj11tQTciuPwKQDrFKIm7I881aU3VhiREoBj2CcqErB9Co0qEdxa4xvn52RYssMGkrS4cMtlwakY0rYV5WgEnBGEjkj+7LxVIoK0i1JHrxubiJfTrKU+avmwz8T8iKDD43d5cUZ2VqWZFTjaouQTNyMBRaZSb5Q8MgmYsaPM48t9Zo14QRW9nJ0IPz+oaT7nEh+FSa44u9tEiClKtpF3XYs82\\\\\\\/HKb9K2I+Z8TXIoFXgucet9PoAHni5Cx5nAnJ4Mp3GCyMYFAPIKq3nMkfX0k32M2kjPUmCj+qqEJwbvi5Gs2bU+ABR5RJAzNnAaQpfDDgh0bqXuaxfkDzQ4OXp48FkWyggXhHcEj4JrL6YH9Grb\\\\\\\/Xmx8H593Sd3ed5sG5fbD6u0nEg3SISymOYy9kCq94jWUm2xapQKKWfuOEid7yWarff4oy60jwzmT51MeVMRXGRmWyRQ6AT8gm6aUJPxRD68StDj3b1PsmV\\\\\\\/2X5TPGgZoRTnvpIPcRSkoascQEhezlUOZdBsHtjymcl1d1qlqqsQpwzYyF9+3alsr5YBLMzQMOspP4D7vh1LnZrOP0kqK7w7fqStrda4DTI4lQC\\\\\\\/RbUA7qboRB5sLxFk3e0GXA9HX1h+yr+mh7HDiOvX62d9y13zTGAgxNOa9y1zZZHDJFeW6phwCQVGrAVRt0FyKlGWmx6HycxAvBoBxb9mNLZ6BkU\\\\\\\/ezdGuV3hoV7mCSDHdK2\\\\\\\/EjT+RRABaIzIWy5LEs\\\\\\\/b7UaqD0+Q3F8UXZnpHZwAC9AXyMVBGwc8C4u9dYEKgd8jCbe4pYyryG2T83ksD7beaZJtzIkIGZM0+lNNV12igsdhHaVq++gtyEldlaOPIY0lPcXSxsdUwxwj4IY0uHuMp2f30QJKxWrkycimqKiW1iOseX9BHKUJx5S4lwZslliN45dqpiXLttZMUG+I62DRX\\\\\\\/H4clPARw6dytemhTh5kXhLaN\\\\\\\/PF1byJxK8RylTkNdis2UzU2a08EoTtIk8OMwHY8rW8CA9C9O8D8t4gR6zaR27hhDYG+eO21a8ZKrk7qmFCrcRljNMGVd9KB3iXmXGLVVXNsQ2sWQU+BL0aW8cfnsaUgvSUke1LgjLU2cmhql+8Na33ZDO1msewjAn3pxIdJl0WsJfJMrd+Rf8oSeDbqGVD5iafnZYmz1iPEBHiwdKTONl5+HJ1EmXPHrZ2+pEDRYcN3xfMP5rH+1kZaFon\\\\\\\/zcElLmvA4ih74viitJZIFUuPa6vNEajWvLRrs+WVI0QxqF\\\\\\\/2n5Hz179n9M9Wh1KGd5vIT92QHEeVq4zqtCwxpymMkxIGT2SzaY3J\\\\\\\/ZXniQUqqkvJAgCu4dp86CPgrrWpPQWu5NpS4E4TI0e8\\\\\\\/LsYPDHLQEVmfopHu2vFW4Dnz0Rr1XT1OanFKZDtmYhgk+iMFKSlkTvwKG0hpyjWIj16oMUe9PBQXjhfuO9j5IDXnfuMzac047P15HYhwOSLlzDnK8YoVALxf781OtxizxtwcNTDZPyPY\\\\\\\/rRxwYFA+6waMJMrb45qYVeTNhtPZo\\\\\\\/gysF2vHdczi0PH9Ce5rEUuGTj2lvu8TUVNBIC+xV4j6ovz\\\\\\\/bx7ScVC5UiEsPrOtKf8T9z80f6PS\\\\\\\/jEeW5oEFpDUyXky10I0xjlPsfpbTQPfsnLGkPj9dZEeUJRkRDlKopaD7hWJasmTFVg6yeYEyukPPlzlWhLkILHWbqR4RqoQYTs3M2+oSWa8GPP2uUn6QRidWyk3BFqtm\\\\\\\/Y+SshitDgRgvXmGTpn5NDVvUcCX\\\\\\\/y17UzGtvfdYpKEc7QV7MspAq7Raizi+tvIDTeD3Z8cQAqLMGFjDBPUbqRCGwVEZWbPiCXQkkIf0SjEdtLDy1U4oAoXgvrLJ1nbq+beca0zYLem4E6wQJQcjDSyTj8pEVk5UN4FyDdp+OkgY5vKpwAunI+xfRasnP+SSCixoz0\\\\\\\/VfQ1dv0ZmgL7rVc6juVLT7dUg+aOnp0DdgVCI1Yg22xMy8X37nTA95KdzZ1bFR7quDxY16vXrDgadtKGb0HM1SuuZaAvJI0sgh24rSA4mfEvUBjktl4e\\\\\\\/AgPDi4581s0Xt1h0W7xfgCyY8NbE1pcJ2BiSPijbxS2vXu6MhVWAzYGeZFMwqPSCr5uNYros6TwIZvs5skyWeQHNZv4ClRs46gn\\\\\\\/ThDph4q3LYcibCITzhKkTBx7amyPh6IsudDLFKBzHY3DLV025F\\\\\\\/IQy4mWHeOCcJuw321rjHI7pkWVMqMm5qpSDNoLNdeY1QcULXZSwTdBRvnd384z3+YAOJYwuZSJRxz\\\\\\\/NNVcYB9OJnlgjVNY2CwgN6G5GgkzoAagVNyGbVoicxaDnarLqy2zPk4dhjMz\\\\\\\/7VNg5HIaXkPDqJzvm9ZIwvK3IIsIycqSDOC0klIAwzYjX2H7wkSdKxNMIq7qnogEUZDqBJ9K4v7C1cHCWWJ0kd20jz7AZfcYV8CH+rpC2fM8b2\\\\\\\/mGKVxYM802DWUXqyXjMqd\\\\\\\/LUsO6na47lzGwzmoLuzLNXPtTDnoFmHcjjer4yoBcJ8Kl1nRQPJIrAK7fc3DdfOLJfEmdMzJEoTz5azxQTTwU+PW0D\\\\\\\/Wo2hOiSCpwEilflocv2b9zKuotk5oqI1eDD4JkMecFrqdeW0DJKjvv\\\\\\\/dO8ajjz9ba7xhPFoLEF+3+4fAchf\\\\\\\/I4tjmP5BJ2kYQSK92SZ3v5bDME+dqZcaile1egBmZwJsIMvd7rohS3X1l9QB+5wyzcMmbddno4hE73+9SEl9ry+xf5zbjPcuuJYr1JAcreSMI9OSTbFf5aciSb\\\\\\\/POuQsMYG7+RJ1CMADLki3VIh5ASaAGJDMBnENTzkieW93dptKRO32j16xj4MYm\\\\\\\/5lsSbui1O9pDV3\\\\\\\/tw1nAvJA3wWinO8VLaCMGt9xV716MJ\\\\\\\/ywyPHaLjy9HqE\\\\\\\/Y\\\\\\\/u8KLOk6VE8dE3UjLhyuf5\\\\\\\/SXOtj\\\\\\\/tCbhLQ4VNBAiwijR+LdPJ0cv0vKDVtjmhVl6BERWgagWEpP2uDknBR6f3XEWmhblXvLxLlpwDj6EO0iANT+N33isjPqjjRQ1qlnaeWqzyBM0d9SKRoJGu416gg9jnwmfmnPQZuKkFh8uIZEzPXWxDk9r4IPGk1YjoYPMp7UWLYLryWnQMQbvW2irzYIVdMdgTuVQCIWfD3J9aEhpCqZ4h+PssqzsHVICwdTT50+X0xrq5HrTTusqZHdoP8xlgWhXQUIp4Gt2OwGyt8iX1oiuoinKIu08fjLRa5g21V3x4n8t3hXp\\\\\\\/RfBuh5SVHmlCghHtWiCSsH2wKj+ZeOJVpCs7AyIf3UgRINHtI3T99x60F+7943HJ+IZq+Ywx\\\\\\\/Iln95ug9fpllGRj4Ft22T+Vuk\\\\\\\/VL3GFQal0JSJjTkaeRMO47nnQd3xpXVFknWMtBKPxiC0adA8CckNtqjcwVM9lR5fnrOeyICwu7VQs7c4XIKHi9y9KeJmeCHcSjSUzIIXGFpAY8iZ9gY5UpgMWALpO98L8D\\\\\\\/a9h2Ft58jhf5mdFb04YP91\\\\\\\/fvPQImoygGRGXPVvUzBNGgCD5IPc3CyBtzT\\\\\\\/vo8Ea9kbC2PI\\\\\\\/uVTqYnPfDMDq0+2JxJuCK9z8M7mYoaUbvIY9D1AiCS0vMXXgJASRl7L1iIz7e2cs9TUesdBcTE4HRdB2AjQEXnmPU5KEL63dS6JB6m3yjrshr5ipE1TFJCiJXkWc\\\\\\\/n6pdLuJLZL82QlSt1IlC5qm8WGDmo1S\\\\\\\/9s\\\\\\\/Nv+h7\\\\\\\/ivyuRtL13FSuOmR4dbW3Zwhd03Uzub7S5T0gad3caA2XosMSt7bPfOjnI9DkLSO80upxJ1iCJWFNE93fJVEgjLy3UpDd294ABwjU3392Xa7YHS1jbqkoY2044yWF0hmSPbxOjQwtuW6811PtMK1dKjdXZVq9IXmdkg37DOQO0GJjB5whv6sedgxPRuMkRemdvSo+RRQixd+icMMPxxKfHXZABW\\\\\\\/X1KMFRoAB3uf46z\\\\\\\/McwsITg7oKo+gJZZkADxyMzn2kMAH3RglHnFRejbFskc46hSaqzYVTQqQdqdeWW2gP13YQ0sB7elZYxWzmxLiGS4neUDMWRiqi\\\\\\\/+X14WRPZAnoz1PkDrb+gDTZxMlkUd7aMuv4ubBL4sYyhOaamDkvSYR5Q5q3\\\\\\\/Ogci5CBTPKDlguzl6POmVt3mY++\\\\\\\/eT2st5+i9JWpSGXQeW7+95iy9V1GgPDBm6YZ9kCm\\\\\\\/SAQhdD7Y9pLv4j1lBaK9oAUX517Kr5UbjC8hv0U\\\\\\\/lATcCgYyjnipDulGEErsMoZnDixJm6KJeOAGwj5qj5N1kNSzgErS6A2FJyVqBbVaoO2KoxBI2c\\\\\\\/xkfJ9hfbu1GfeYOY481VeSy5vRFIg\\\\\\\/oJPdgm\\\\\\\/S0LMq9qpU2fIMgN6CV49lDbYuvYHtJ4QjKk9qCnSZbcYRXtWZGzzWDvR4Fmp0sRT0I6bUsuLdD7YtkNBmD1ptK6h2sE9FeVLTP3wBhvDMwfjS9OuTBNpFnt+EJOudhuck1BR8kAyLNgkuc4paNwKn0wa+44\\\\\\\/2b8rccCSfv0L1V6VUGo45gKABorBKVh6eIRK8JsjBFG0YM3L7vhAyt\\\\\\\/nWQ1Ei++QSgIy2+j4YcHY8IkpyU7GckmrHiOYskZ15zAyZPJYpZ6871Byd8d7Hpw1w7aC1Bg4j0Zk80DRmqYITIda4WlJae9dM2ktQK7PqyGoT6HhIXb1EUnkXdRcRJEQofKF3thrFiBt9XuzBkY629ABbS2t7ywDH0WJty3z9t95bmzd0HvOgRjoqsPiwaRtA1+SmP3jOJ4le9pgIcd1weMDEIBQurljKKVTAoDwnuUNVGeJbsU3Rnmo1LGAHCdOpw1mFuMRzKl4gWklKAMS8qmm2A68HPblJ+OUlBZtpR7HFHanESVODw5q4XP\\\\\\\/gu2VE9Xou0Fvtu+2fJ0nbiOHeExyVFw1oEWFOsn+PTX5WXS9wJg6Z3+v6S9xb9R2qmoqSZU50Z8X\\\\\\\/y1rvPNSNWyrtYYsoREDgwIisWw8G2qYi78TGN\\\\\\\/hgmY\\\\\\\/4Yp674oLyj7Q9TOLRazaDCr5vATjtMj9HsVApKkiGwu3ChfgY21pOdgtTXYBhEplWhZ7nJ\\\\\\\/6xaZAozuc88bY+RU4Lw6LU1u4lSEmbaL3riPde8equTk4vZS6y1l5OLTRXgy3JYH25uHs8cl6kRexGqKlADM5uOeWa49Zk4Q9PDqa7gxrYfEonDY7ne4NhpbUYCTuGXK5lBfrfM86Ip80AJQzGzzEAYaOO+MNgwwIaI5Gmi2yT3v3QT0AuDuB6RyGSd3jBlASutyNUsVHnoBR10jCu6asHyUyfqr2g+FHkCvpEOCxPnsitthWy\\\\\\\/qLdYjXJ6moF22h7jsqvq8CnN8RlNm+\\\\\\\/ABUj\\\\\\\/pBq9jGvzAvfASi1b1QsYElToFNFoOckELtNDKnBGWvApaOYPjlkwo1kjv2ZGJ3cK9N0NAW\\\\\\\/J4Flu+0nCUOqdFsFYMHKwWpXA\\\\\\\/mZkXPauvLE9L7E1dvq4DAN+IvW3KAdHyfPzaYNlCCsTXl8BpHvNK0zRqti5q9ny5zzvI62tAiiPlik7lqLGhBIFhZ781mIVqdTl+jHrgVfpwsQF0GqeGltaMD19K5pr308tuaZv3ONGOOCaZ7fMKWDRH80MIy1vM9EkfslyfBRMBhR7rJg0fR21IeLzVc4uqy0yDaxZtXD55RwivsJD3jWHKZyAVBW3VuhzJ1QUy6dXN7MmY7G2pNpH1T+IEaih4qozJh7xzf90H7mVN81jG0bhZUS3ZWbjgK0bRnyjkF7NcyY42Qojz58iSFnCm1TqidRyzZQYS1Vj2hHSQwLUiFIYosF3saB3vE+jyUkqWoG5WwOohOm\\\\\\\/eHE9sNxXssCehYvjvPaQ9DyXT+\\\\\\\/AoP0+yZ8\\\\\\\/krWFRxKaQA4Sofmae03LHomm\\\\\\\/6I08Tk5CDiou7jgpjaBC\\\\\\\/wCpZojYBdptD2ZmnrOYwVVwU1eEvFQLXvvm0LWSXuYNc36aEj1NWWmQ\\\\\\\/cuhA6HMArrbIiBIeDULtQWz8qDjXTSo6CeEEuxOgZRBIC40sCQt7brNkP7RwlqDKMFWyUisOuZOwd9dwOrVHJMW5UDSCcCy7W9ENkF6OaaBXHwUugfhkJ8USu343flMGgy5gkgo6Nxql+oumonVBOS7uO38JGoftMrc7+d2SKcbdcOCuIy5HT0ULo1PQiJUU9i8UQLXQRUCYUFDvEDTNlnro5cVRNxhC22HK9w6Qdo6OyYmVrEreoYqnHAVSvc8cINm7mfO05TN2BKPfOaHKFk5+IU5ZjYxvn39MqRlcedgPdHVU4kU6sU\\\\\\\/dTBB4eMa0jaONwLVy42uq2dIo341ord2GcUfCllN6xrJqDwntboedeTTU2KsZ3cUCQnF4kZC2INs4IqfYMbWc\\\\\\\/HvlcZqDAgNf+\\\\\\\/sIyIc3IiOf7\\\\\\\/KBLDeQEwTqKO1ReR7vz0PsH3G0K144P0T61Hz9ovf9tCEKHG2LK5jvYnCNPvtwn9cd1V8iyHIwkxf+3CPXVIXf63nUuXZxOZBrPPk3P3YFC2yCj\\\\\\\/KuSE\\\\\\\/RDPSNHkKhK0BtMF560ZnZXwT6+tkPgzk42e2\\\\\\\/s5muQ2OU8NYjzqAM2TA491PRF7Qa7pN2dSLTb+AHHVlblYcY3kuQ6mSLhjI6WMxehskQ4lyxVAWlOv42V5kggsgMy+EBd\\\\\\\/Xf3sJLdyxHEcdwYCHYd\\\\\\\/cCu7UvedNtjxoLfJXBvewepqNQCYKsiXAHvbpCM2ZxKmWqxPgfwkYghEX0KbH2FsMgCLa0oyTMhoPrzdgLnhzrKx313IcMS\\\\\\\/4HGzHijvYG9GTG4CAlpWNexqDjRgtilAPPe6yItpf5WNiPUSE0R5Va7p3ko9EHDQ6hEdqpNXBpiUqtgzT96\\\\\\\/9gD0ZJABU+7SeLn2\\\\\\\/FBpGmqFKerOU\\\\\\\/jkgU9wKrfVjRsZsFWDbRIU3UtLrQ3cN8y4DLD9QUOKMN7iRsV+Pxm3SQfRe2N4iQgibAzkM2gHYGHtLkY3xOz0WQRmeum4dA+HE+27OBWPrugwdwPz9ufOzGYQHG834M9mOaLXAgpFsXd1ygnaNvuDi6YNFbeqtMRGqTqPfZdEPG0eVT2yjFHOBRmNLXt602ikuxNgF1L3kNbIc8T4gNlEJI3NdvNRqLs8kpptnTYhlb0Zh72ljz3ohd4WCvuY5aNcQ1MruQjJyxbuuJpomlevMXo51uN2ILRqe5fTf3UWQMHnH4bXIyG412yP7TWNmCgVQcn2H11tT+toqeubAp5oP9PMNLJWPvEqTF3JOz12E2rF+bHBU3bRjA244RpCFIUAb+a7p7rUskLBcBIpJz86blbgfzAoUALUukcXf4Cmto88Pn5O0ZpCeEDwG1hiQGe3fS6JBIi6GXfcAIreiFtO43WcM4oVTEPQo38d47yfF7pc+CsUSK9fvnCZPChSiC5WxI6H5X01zbwRzRbmNS1GK2AT79EP4xskwBXXd8mNqOMROZURoMsa3Jjj15ApAdifiRCOcLUJIiOLYAbYSUXZR4b9cdIBbBYV8tLEG9zJQwMpNimyJyjDHB5LQHpKucVrKWKhbjvtV4jQe\\\\\\\/mdgvrMEHqLD8Gy3n1Js8z1E\\\\\\\/oFJje+Wksji5yq5nesmmZDW9EhnPHk4JlY1mfCJmIK7zJCJCggBRfxOLw9o8ZoQknUUb\\\\\\\/jcJXxh7cMAfZvJHi+zHeUxoTdWLAo3Rp9rp1Uza\\\\\\\/xNj36boEViJecaM9+Z91iWDiLm5GlPp2X\\\\\\\/KE1DsyNIXQlvb+5\\\\\\\/3N090qztyFhBj04H0iAe0FKzeucTezkPn6STLE+ZHcH543QKzR4q4zZZeK45qY5gsUMD\\\\\\\/Ztia950iVpAFttKijefR\\\\\\\/ywptw+io9RkguUuL3QtAFYX6TRctgSbIe+45eq1r1SJs2EVjgGEN2XIZrFo05WnaUw+zgu4h\\\\\\\/jsDmpKF8tr79UtQeEOH+FaCA2MMhuWp3XnXpCJDKoDrBrVCijotgqcw0Rz9pwgL7M3EXYJAVJxJEvy2ezYOlp\\\\\\\/MRibxMdTsXhrvx\\\\\\\/2yQbVubGX0eowCEP3EPSkZCMuuo4RBKVRe1JBM5A53y0xlkmgEt6uvD7FkcYY3tjPW\\\\\\\/+XirfkDYr2wb\\\\\\\/I50tZ8ZXXf3IMvs4CZ48jfbjFWznUoOXWhRLteQ1Dsrnam5CcmCvcUWpBNN\\\\\\\/yolKLy3pGU5Wgw8M0H+vTjUpisVet7c9pwIIGE7DxBMUTXhMMdErKUqxGn5y+mhVL8BUWmTvmZ1nRxIKkVRCzH0qiMLLLOsA1c+Uyhr7HAUL\\\\\\\/2BOl\\\\\\\/ZvDvjvd8RsPL1lwNJCtFqujioKfxnyeyOde8d\\\\\\\/zYs0A4Kvwl7cYYqbj+dc0fpaLvCb3vnfEC25EZSreHS43HWIx1WPop\\\\\\\/\\\\\\\/U4WnOA8K\\\\\\\/hoRb1hLYCPMC472wh5tCrj2iKU21IOcaMUNalnbPpFeNXTiLGaOmBF\\\\\\\/dPlL4lZwxZVKPBb9u4GIbpmQj+qWVkhg2Rkvy8IaENbvJgObNLrd8DEomcRAx9JQ\\\\\\\/xRfNPan0i0VnIO9zr+7oRGrzJuftrIA4koT3ydIo7XmH6uIttKzs+OKJ6wxEVX0tvMrh0rosyI\\\\\\\/pofPBgWZpWVMef776EEyMhdCRQTslz4vdk8ZVwRE3Djv2UlvixFl7NcVxMlQ1Z14Q2xWhGEJzQ3EjQzAmJS1eAXhIg\\\\\\\/IWM3lCQRmo8AioeAArwS8KvMy1BiTtz+qpRjIQ5jUOLthwyzVJZwOpDJhe2faMwREwoRGkVsouH7ykfXPrhTOKLHpjdka81oNoDQKbR2jAsojKkEufRPq0RU+FG4nDdVngcWUp2U47Z6Lkp6brY1drC1kjMqRobYuNUjQg97X3iR\\\\\\\/+Obmm11WoCQIMw55iOMqCGWO9AmhVAYUNpoykW6jehNBtu4L3hoYNW0LfFX47K903RUJE3nEgP8DYw0MfDDtwXOb1wOve6PxNTmYs02wn\\\\\\\/0IW6xAbhJce3BFTbol3u4oRgZ2oQmKPXJD8y4\\\\\\\/hYMDbiQK54gc3AQY3nNu+zU7YtDNVWAp\\\\\\\/UVK9HH2ivn7bt6QQTldCTqRAOXwped7FT7WDrzmNmJYJP7KjXq98wK23ylXcHImqxmqVdEEeQ2MbhwZeyWJJIS7jH0lwe9+DNahE2gulVke3EgqxDgHD\\\\\\\/NilfeTwSjtaUcrPE+073dt3oaHX\\\\\\\/U06xlqbdDGtfEDW12F1xX2vuFe9AX8y8f3jocuZB1uaiVusMsLLB0hdDmWPWVD3E8Pa4GERdz3MJyD3uSZ+lyWuBjbUIHHUl56E6L8PxjvomCzT+fST7Ceio\\\\\\\/kN8Mz8QU2i531yC5\\\\\\\/wUALfecbgwJwXrJTl9tmNwWkdjaSVEAQT0iJKVsq6lx74r1zRPWYXm6cl8i4N089Pthvto9m5bxh+7IwboLpD6CC96GQZuzIRbTrUBNQOmndCq8gsPvY+pzo7UcvgFog1YdHbLKp0+D0gyjLGL\\\\\\\/qIjvGuSQNY5PGF35ylZGh5KxcC4weON1g5vjdKcSlNTZHid5UnkiC52VS22xOdpp+3WlzL5s1z\\\\\\\/GBnSJRDVWN+5H1mhAq6pI+yYuSL9k7A+aESSLnI6++EOWtGRaVnzTnqW4lap0tw8gaiVXaDy4S0LRWhY9Kd5Hwbo05dW\\\\\\\/gfS+8RaRglirxExTiAYMT1DNq21AP9CY0O4iOwfPqmJd33R9aK2KiBQuz8RfMgcD4x9xlq9hZYB8DNDzYXAf7ADLDW860hl6\\\\\\\/CDgcHIiXqSygGC6i7z8dt\\\\\\\/umTyKIgFWeS9eZyT3RBizIMpfT38DUfasg2il1YmMSlW7TkLS\\\\\\\/hNI35xr1TXApAbq3UojEaurLAuMprncUvBhYYuRICnDaGRHa\\\\\\\/LzBSDVHM62IW4VMd18TEqGoOCtXX4NisKDrNEHJSqMgZFUYLPgT9J6amtZLdxp+Rimo1gksnUP54UgVLZaMikwG+YJ0NLgRYKZWgRxom9djNVNhMrKcT6tvosU3MRCuV895SpsrnQLEoRQ5U9lk8vHRJf9yLq3NiOCWdMFCSz9zssJkwV7LRdOQnHSNoHRtwuTMBQVBLcX2N0xOnuYHKDEl\\\\\\\/b\\\\\\\/59xH7SacrH24ibdLnKkNa25zFx0ZCIhymSCqDg0JFjVL\\\\\\\/B2vziHkZ5sLzJvPb+DycSvrk4I2SyxwiHXJr6BzVKhFAMNBAMR2gFr1f+nG2mRo0PP36n1852owJTGZeYCy8pgkSj72HTeC2jrLJuJoEskiEPWyIiM4UUt+QZ41PHC55S6FBNH3vXAe3tBv+7X\\\\\\\/2PbkKvhLz+\\\\\\\/PW6VOzvhTOejVJV5zzXfJsIqtiiI6Fz\\\\\\\/xydLjUgiXlSKHfy4cICyCR\\\\\\\/X59Jyz\\\\\\\/l1+ucaS1MW57LrAxNrd0q26VSZrCghQczddbfMba32nUcUaSFhf6FCkLKDNcZpuSBDJd\\\\\\\/2yLKir6D8ThbCMblLm4e9T2k0DsYl7FkHcjVU1V5LXlE1VN94ljzQ\\\\\\\/a\\\\\\\/0tB1+oRw+M2dbHWBkQglMuw\\\\\\\/NSycY3XEE7jNuH0lr3kWnXrmia6\\\\\\\/W+TkE4FyNsJ91O0SMTQpen+oq+TiQ4ZvXYA5wosAyRK11gkVe+qlnALo5z0U6mebL3ojD8uSB7BbHbOh2p3tEy2gmh\\\\\\\/mc4mU5MrhIdJxZjW5CahSLVkifKQOfzjtc5\\\\\\\/Sgl6X4CI7lLT+WlnN\\\\\\\/ZCWeYYK\\\\\\\/oDCNSHod2aKYD9477atAhxD2K\\\\\\\/jRn8s9nvNO63YUBWLeDjqWxGwXYL32uA=\\\u0022,\\\u0022iv\\\u0022:\\\u00220rqOZ9xcMB7s+E71szUACg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd2d2dee715611bc9112c15645c64933bd67978989494f917bfe1124b32d8e46\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zoyOmfYze3453kC7CmG5jZfHhGvjrSRp3yoSQhIDonC3ar\\\\\\\/zFeGd\\\\\\\/TTNcvBBbt9a2jKoexAYAKE75\\\\\\\/JW3uP9MuT4NlUVO4tJkCFj2HDwZQLYLTMy6RSQboF4ycQhmoMznrmo2mb5gm6PQtIfC9G3Tg0iaih3HLDMueuombmSKcKVrL6+yxDhmDtPaDwoEwEK72UvfF96BBNqNkLO9YULb3Z7QuygXn3DyxtF1EgHj3a+YS6aORNRZYaYF8Pau9HxykgTHQMmBbpMY3mxScnPUHEJSF+SYAt1\\\\\\\/4bO7x4GuDilrQj9T0kzS8uaHzIO2o6IyoCEs17Kp+rREGxpLwv\\\\\\\/WXCa9rKYNeXI\\\\\\\/zE1\\\\\\\/pM+2d8VGG8et0eIbIt8Rfkbce3kg6yJBEGww7paRQHshZmV3DntRjaVUADqEAUypY0E4n6jYgmKKq2inhAYHU1ScO2YnJP30wCVSe+CHw0nWc1JY9Ils5rXLjnyRBPFm4pJa1EjMMEehiov+23JilE6HCZBvdCaarFnKiO+XcuON5IoemZL7VlOk+wNdlsMpnkB9XVlHGCUGvD7CHZiRVcrJ9uB9yph6DG6OK0NrmNgPfbrcV+1W9IRy8LjK0cXUpz1LQSk0hjlE+SGXgfI2apdsTzISMiwYPTBzoYwouSe8Rf4IRj0GReQ2nm1qPiN2lHGOZm9irNkemEJaR1+yjzTcFUoo92OpNvVZ17GuOn\\\\\\\/RhkypnDFRLpWHIAgJh76ITee1TdkgGgAggZUWRyHy+r37skGPFUOfySLtcJ67NsjW1yfTxiwu5jbVFGjqHcwQdp7T7sGvQrEPYWPcO5CxtIsjbxhkcCt9w6GGJZynpHSuXIjAe2whtFb5GiTABiiuXpdraYQexqNnIJwo7uE1W1WXunETw5ySkxg0A3H1pePRus6E1R9kc5LSGPhwZCYz6wBPNENcNjaMdwpYPuKFPJuykBi\\\\\\\/5vRNfV64kJigYdPKRs\\\\\\\/\\\\\\\/8Xua1EqqPCsIlNCU91MSz7StO7lnBCMoOE27AABo6144rCUyHjzS1zfBbDXqkd9lExhomEogh8Cq9QlIk40iUgxZvwo4r3NAxOWUSzcyQJD7wVR44Jj+85xAhpE576CLab4S2gVT5UGx74Ons8Vdify691bjBdNhKAoFKItfSBXrD0fmhDUmyHRhUk9FWoAaNuj705SC1gvO8mMlWDb8D\\\\\\\/2mz9sonE\\\\\\\/7BJeRKpg2fJkWV3uuxwA2PjL1M+WYLrQLY3CdMKz2z5DsLI55YnCHNgGPF9AhomQYWMfMn9tkZ7q7da3KZpQi06nFvU7snx1rRF6249u+tuQQive0MkqJE3nPYCDvvUnpLCtJF+wn7vMwj+FjPG8PlkboNvno5LgZEDSUndOJWdcJgFpCspyVSRkiMrsSSOMLKcs432Z0IHsje4NQ61T8AvnlX1Ndi1\\\\\\\/lx7rYhDbpZca9ahT1\\\\\\\/5vbCYGLexcTP\\\\\\\/iT7m+ClWRtaKXSLaPldNETU9uIDNhb9f+gPG3m3gkc3CZfU4ZXWtdf\\\\\\\/y4xrdiKcua0\\\\\\\/roOAV7K8AkExdzhQKGCGqOofjsMFnJozXEWY1X8dflkTm5GJbBFkc\\\\\\\/n0A+YMKybX5I\\\\\\\/4VuoDsF14WtZ9GlNxaSaduGWkH\\\\\\\/HRGpueFPb\\\\\\\/0Tc6cq3JGK\\\\\\\/K3X9nacR+PVZoUYO1OZknulBVx7bUyxk\\\\\\\/n6UHwh8WgJGVNTITu4D1W94XfyY34rhj+wfhGEqzDMq7Dq\\\\\\\/9u79jJ0kp08AspMY45CTodpAZSJPHmS+5l+YYcC831YoiO+IE\\\\\\\/s\\\\\\\/aDbAD4d6pgvFxGextp1R8Qfa018S4yVTPZ27eeMUfTP85g5NCx2396U\\\\\\\/kItZ78QlUHq6QaMC1tcFONl7ooeNU\\\\\\\/+gbuTCCGzM0u6cN77Q06vXOZzX\\\\\\\/35iLrtrJAQ8GLfNYBrXRFeBJ+zMlI4GO5MAgB5z2O23qQkHlayf6ogkCgBW0041g++ThB51alebgCt9UaUWHA6Xk5p854UK6b7D8OTRHsLXnWgkZwHW4dqqNYuMQzuLzitd4OdqZkurKfwC+06vT9t3rjqQ24nHeJ8adbrisJPUDfmsK8KIlaKrM9I7h1dnVCnZFh8QQlZxTeJtEmaw9nok+FE\\\\\\\/EpdVapFAE2zP6Diz6JMhU5+eD8Ujw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022I6AOyUxrglfUL16tl83bIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e006513013e1a385709e72c729dc733e0d28cd2d2cc3d7cbd3d1fc73dd0f3ff4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459839 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/971" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-J6J86_-srH9KESIEJrGWcK_j4NGS4DssYV1Ef_l_i10" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/971" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2Kr5A3gNEDWPfyxpISRX2twfI8XV7Q2vjjxF49sq_IY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/963" title="The Maintenance of Single-Locus Polymorphism by Maternal Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/983" title="Comparative Analysis of Wolbachia Genomes Reveals Streamlining and Divergence of Minimalist Two-Component Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459839" id="node459839" data-pisa="ggg;5/5/971" data-pisa-master="ggg;g3.114.016121" data-apath="/ggg/5/5/971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing at ≥40-Fold Depth of the Parental Genomes of a <em>Solanum lycopersicum</em> × <em>S. pimpinellifolium</em> Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zoltan Kevei</span>, <span class="highwire-citation-author" data-delta="1">Robert C. King</span>, <span class="highwire-citation-author" data-delta="2">Fady Mohareb</span>, <span class="highwire-citation-author" data-delta="3">Martin J. Sergeant</span>, <span class="highwire-citation-author" data-delta="4">Sajjad Z. Awan</span> and <span class="highwire-citation-author" data-delta="5">Andrew J. Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">971-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459839" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zoltan Kevei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BKevei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kevei%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZoltan%2BKevei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:z.l.kevei@cranfield.ac.uk" class="" data-icon-position="" data-hide-link-title="0">z.l.kevei@cranfield.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert C. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Rothamsted Research, Harpenden, Hertfordshire, AL5 2JQ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fady Mohareb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fady%2BMohareb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohareb%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFady%2BMohareb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin J. Sergeant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BJ.%2BSergeant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeant%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BJ.%2BSergeant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sajjad Z. Awan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sajjad%2BZ.%2BAwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Awan%20SZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASajjad%2BZ.%2BAwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew J. Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/971" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459839" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/971.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459839" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/971.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459839" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/971.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459839" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/971.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459839" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A recombinant in-bred line population derived from a cross between <em>Solanum lycopersicum</em> var. <em>cerasiforme</em> (E9) and <em>S. pimpinellifolium</em> (L5) has been used extensively to discover quantitative trait loci (QTL), including those that act via rootstock genotype, however, high-resolution single-nucleotide polymorphism genotyping data for this population are not yet publically available. Next-generation resequencing of parental lines allows the vast majority of polymorphisms to be characterized and used to progress from QTL to causative gene. We sequenced E9 and L5 genomes to 40- and 44-fold depth, respectively, and reads were mapped to the reference Heinz 1706 genome. In L5 there were three clear regions on chromosome 1, chromosome 4, and chromosome 8 with increased rates of polymorphism. Two other regions were highly polymorphic when we compared Heinz 1706 with both E9 and L5 on chromosome 1 and chromosome 10, suggesting that the reference sequence contains a divergent introgression in these locations. We also identified a region on chromosome 4 consistent with an introgression from <em>S. pimpinellifolium</em> into Heinz 1706. A large dataset of polymorphisms for the use in fine-mapping QTL in a specific tomato recombinant in-bred line population was created, including a high density of InDels validated as simple size-based polymerase chain reaction markers. By careful filtering and interpreting the SnpEff prediction tool, we have created a list of genes that are predicted to have highly perturbed protein functions in the E9 and L5 parental lines.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/solanum-lycopersicum" class="hw-term hw-article-keyword hw-article-keyword-solanum-lycopersicum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Solanum lycopersicum</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/s-pimpinellifolium" class="hw-term hw-article-keyword hw-article-keyword-s-pimpinellifolium" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">S. pimpinellifolium</em></a></li><li class="kwd"><a href="/keyword/sl250" class="hw-term hw-article-keyword hw-article-keyword-sl250" rel="nofollow">SL2.50</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/indel" class="hw-term hw-article-keyword hw-article-keyword-indel" rel="nofollow">InDel</a></li><li class="kwd"><a href="/keyword/introgression" class="hw-term hw-article-keyword hw-article-keyword-introgression" rel="nofollow">introgression</a></li><li class="kwd"><a href="/keyword/large-effect-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-large-effect-polymorphisms" rel="nofollow">large effect polymorphisms</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-lines" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-lines" rel="nofollow">recombinant inbred lines</a></li></ul><p id="p-3">Tomato is the most important noncereal crop species after potato. It has simple diploid genetics and a genome of approximately 900 Mb, which has been widely studied. The reference genome (~820 Mb) of <em>Solanum lycopersicum</em> (Heinz 1706) is publically available (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tomato Genome Consortium 2012</a>) and has been used extensively for the identification of single-nucleotide polymorphisms (SNPs) in range of cultivars and related species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sim <em>et al.</em> 2012a</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">b</a>) and in genome resequencing projects (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Viquez-Zamora <em>et al.</em> 2013</a>). SNP markers of expressed sequence tags were used to create diverse tomato linkage maps and for comparative analyses among cultivars (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Shirasawa <em>et al.</em> 2010</a>), another study of 40 tomato lines revealed many SNPs that may affect gene function (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hirakawa <em>et al.</em> 2013</a>). Genotyping using SNP arrays confirmed that SNPs from wild-species had been introgressed into domesticated cultivars (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Viquez-Zamora <em>et al.</em> 2013</a>). <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shirasawa <em>et al.</em> (2013)</a> showed that detailed SNP analysis of six tomato lines coupled with genome wide association studies can directly associate SNP data with agronomical traits. The resequencing of eight tomato accessions by next-generation sequencing demonstrated the presence of introgression regions between varieties and revealed intragenic polymorphisms, including SNPs, insertion−deletion mutations (InDels) and copy number variation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> 2013</a>). Moreover, the available “150 tomato genome re-sequencing project” (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">100 Tomato Genome Sequencing Consortium <em>et al.</em> 2014</a>), which was augmented very recently with SNP data from another 360 accessions (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lin <em>et al.</em> 2014</a>), offers a huge selection of resequenced genomes allowing a myriad of comparisons of SNPs and InDels between various tomato landraces, heritage cultivars, and wild relatives.</p><p id="p-4">A tomato recombinant inbred line (RIL) population was developed from a cross between the <em>S. lycopersicum</em> var. <em>cerasiforme</em> (cherry tomato) line “E9” and the wild species <em>S. pimpinellifolium</em> line “L5” (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Monforte <em>et al.</em> 1997</a>). The <em>S. lycopersicum</em> var. <em>cerasiforme</em> falls within the same species as cultivated tomato, but possesses considerable morphological diversity and is believed to be an admixture of wild and cultivated tomatoes that has perhaps arisen from reversion of cultivated forms (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Peralta <em>et al.</em> 2008</a>). <em>S. pimpinellifolium</em> is a wild tomato species closely related to <em>S. lycopersicum</em> that is self-compatible, and that can be crossed with the domesticated lines without hybrid incompatibility in subsequent generations. Various agronomic observations have highlighted the importance of <em>S. pimpinellifolium</em> germplasm in tomato breeding: it has natural resistance against the two-spotted spider mite (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fernández-Muñoz <em>et al.</em> 2003</a>), whitefly (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rodríguez-López <em>et al.</em> 2011</a>), and late blight (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Foolad <em>et al.</em> 2008</a>). <em>S. pimpinellifolium</em> has been widely used as a parent in QTL studies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Weller <em>et al.</em> 1988</a>), including the localization of QTL for fruit shape and size (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Tanksley <em>et al.</em> 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Grandillo <em>et al.</em> 1999</a>). As the <em>S. pimpinellifolium</em> LA1589 has been recently sequenced (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ware <em>et al.</em> 2014</a>) the resultant InDel sequence data were used to calculate the genetic distance between different cherry tomatoes and other <em>S. pimpinellifolium</em> lines (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yang <em>et al.</em> 2014</a>). The existing 7720 SolCAP SNP markers were used to select a substantial number of introgression lines in the pest resistant <em>S. pimpinellifolium</em> TO-937 line (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrantes <em>et al.</em> 2014</a>).</p><p id="p-5">The RIL population generated from the cross of E9 × L5 has been phenotyped with the aim of exploring salinity tolerance and rootstock-specific traits. The microsatellite coverage of the F<sub>6</sub> RILs from the cross E9 × L5 (P population) were compared to that of F<sub>6</sub> RILs derived from the cross of E9 with another salt tolerant tomato wild species accession, <em>S. cheesmanii</em> (C population) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Villalta <em>et al.</em> 2005</a>). The F<sub>7</sub> RILs of the P and the C populations also were used for comparative analyses and mapping of QTL responsible for salt tolerance in relation to fruit yield (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Villalta <em>et al.</em> 2007</a>). The physiology of salinity stress was then analyzed in the F<sub>8</sub> populations via the investigation of QTL for Na<sup>+</sup> and K<sup>+</sup> concentrations in stems and leaves (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Villalta <em>et al.</em> 2008</a>). The effect of rootstock genotype on salt tolerance was analyzed in the F<sub>9</sub> RILs of the P and C populations, revealing a QTL which had a direct rootstock effect on fruit weight under high salinity conditions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Estan <em>et al.</em> 2009</a>). Furthermore, this fruit weight QTL of the P population (gFW9.1) cosegregated with a leaf water-content QTL (g5LW9.1) on chromosome 9 increasing the agronomic importance of these rootstock dependent QTL for breeding (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asins <em>et al.</em> 2010</a>). The detailed analyses of one of the above mentioned QTL related to Na<sup>+</sup> and K<sup>+</sup> homeostasis in leaves identified two closely linked HKT transporter genes on chr 7 which are good candidates to be causative genes underlying a salt tolerance QTL (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Asins <em>et al.</em> 2013</a>). The P population has been genotyped for 7720 SolCAP SNP markers (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Asins <em>et al.</em> 2013</a>), and 278 of these were used to map the region containing the two HKT genes, but the SolCAP genotype scores are not yet publically available.</p><p id="p-6">To allow rapid progress in fine-mapping of QTL discovered in the P population, we have resequenced the genomes of the parental lines E9 and L5 to a depth of 40-fold by mapping reads against the Heinz 1706 reference. Subsequently we have made a detailed analysis of the discovered polymorphisms, focusing on introgressions and mutations that are likely to perturb protein functions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material, sequencing, and polymerase chain reaction (PCR)</h3><p id="p-7">Total genomic DNA of <em>S. lycopersicum</em> var. <em>cerasiforme</em> line E9, <em>S. pimpinellifolium</em> line L5, P population individuals (F6 generation) and Micro-Tom line was isolated from leaf tissue using DNeasy 96 Plant Kit (QIAGEN). Whole-genome sequencing was carried out by Illumina HiSeq (L5 and E9, one lane each) using 100 bp paired-end reads, or by Illumina Genome Analyzer using 76-bp paired-end reads with an average insert size of 533 bps (Micro-Tom). Specific oligonucleotides were used in standard PCR conditions for the closely linked PCR InDel marker of gFW9.1. 3% agarose gel and 0.5x TBE buffer was used for visualization of PCR amplifications.</p></div><div id="sec-3" class="subsection"><h3>Alignment and variant calling</h3><p id="p-8">Burrows-Wheeler aligner (BWA-MEM algorithm; version 0.7.4) was used to align the sequencing reads, with default parameters, to the <em>S. lycopersicum</em> reference genome sequence version SL2.50 (Sol Genomics Network). Alignments were converted to sequence alignment map format followed by binary alignment map, sorted and indexed (Samtools version 0.1.19). Picard tools were used to mark duplicate reads post mapping. GATK software tools (version 3.3.0; Broad Institute) were used for InDel realignment and variant calling (HaplotypeCaller), using default settings.</p></div><div id="sec-4" class="subsection"><h3>Annotation and filtering</h3><p id="p-9">Resulting “VCF” files were annotated using SnpEff (version 4.0e) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cingolani <em>et al.</em> 2012</a>) with ITAG2.40 annotation (Sol Genomic Network). Filtering was performed using a combination of SAMtools to remove the polymorphisms with coverage of above 100 (Quality Depth &lt; 2 / Fisher Strand &gt; 60 / Mapping Quality &lt; 40 / ReadPosRankSum 8 / Mapping QualityRankSum 12.5), SnpSift (version 4.0e) to remove heterozygous polymorphisms and filter frame shifts (FS), and GATK SelectVariants to filter for InDels or SNPs. Polymorphism physical positions from the zipped and indexed VCF files were compared using vcf-compare from VCFtools (version 0.1.12b).</p></div><div id="sec-5" class="subsection"><h3>Gene expression levels</h3><p id="p-10">RNA expression levels were obtained from the RNA-seq data (<a href="ftp://ftp.solgenomics.net/genomes/Solanum_lycopersicum/RNA_seq/">ftp://ftp.solgenomics.net/genomes/Solanum_lycopersicum/RNA_seq/</a>) of Sol Genomics Network (SL2.40_all_rna_seq.bigwig). <em>ITAG2.3</em>/SL2.40 RNA-seq coverage data on genes with relative maximum value &lt;200 (blue color) were classified as having a “nondetectable” gene expression level.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-11">The sequence data were displayed in “Genoverse” (<a href="http://www.genoverse.org/">http://www.genoverse.org/</a>) interactive genome browser to visualize and localize SNPs, InDels, and homozygous FS of L5 and E9, and the SNPs of Micro-Tom: <a href="http://elvis.misc.cranfield.ac.uk/genoverseSol/examples/tomato/chromosome01.html">http://elvis.misc.cranfield.ac.uk/genoverseSol/examples/tomato/chromosome01.html</a>. Note that the GATK SelectVariants tool provides the E9 and L5 variants with the greatest mapped-read frequency for display on Genoverse: minor variants at a particular locus are not displayed as they are assumed to be due to false mapping rather than genuine heterozygosity in these inbred lines. Variants are color coded on Genoverse to indicate quality score: red indicates the highest quality, green the lowest and orange intermediate. In addition, the Micro-Tom SNPs are available from the Sol Genomics Network genome browser (<a href="http://solgenomics.net/gb2/gbrowse/ITAG2.4_genomic/">http://solgenomics.net/gb2/gbrowse/ITAG2.4_genomic/</a>).</p></div></div><div class="section discussion" id="sec-7"><h2 class="">Result and Discussion</h2><div id="sec-8" class="subsection"><h3>Sequence analyses: SNPs and InDels</h3><p id="p-12">For sequence polymorphism discovery, we isolated the genomic DNA from the two parental lines, <em>S. lycopersicum</em> var. <em>cerasiforme</em> line E9 and <em>S. pimpinellifolium</em> line L5. Illumina sequencing resulted in 370 million reads for E9 and 380 million reads for L5. The sequencing reads were mapped to the latest version (SL2.50) of the Heinz 1706 reference tomato genome (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Tomato Genome Consortium 2012</a>) with a 40- and 44-fold coverage for E9 and L5, respectively. This depth of coverage is greater than in many other tomato genome comparisons, in which 8- to 20-fold coverage of sequences was used (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Shirasawa <em>et al.</em> 2013</a>). The larger depth of coverage gives more reliable polymorphic data reducing the impact of low quality reads and the problems of mapping to regions of repetitive DNA. The SnpEff software was used to determine polymorphisms between the parental lines (summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/016121SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS1.xlsx">Table S1</a>). In addition, in an earlier study, we had determined SNPs between <em>S. lycopersicum</em> cv. Micro-Tom (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Scott and Harbaugh 1989</a>), and Heinz 1706 after sequencing the Micro-Tom genome to approximately 10-fold depth of coverage; therefore we included this cultivar as well in the following analysis.</p><p id="p-13">In comparison with Heinz 1706, E9 possesses many fewer sequence changes than L5; this is expected because the cherry tomato is a closer relative of the cultivated variant (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hirakawa <em>et al.</em> 2013</a>). Approximately 900,000 changes were found in E9 with an average of ∼67,000 per chromosome (~69,000 if the nonassembled chr 0 data are included); this represents approximately one sequence change every 1100 base pairs (bp). Interestingly there is a higher accumulation of sequence changes on chr 10 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). With around 150,000 changes, it has triple the number found on chr 4, 5, or 8, which are of a similar length. Furthermore, chr 1, which is 1.5-fold longer than chr 10, has considerably fewer sequence alterations in total. In contrast, the particularly high number of changes on chr 1, chr 10, and chr 12 in E9 was not found in other cherry tomato lines where the rate of polymorphisms was highest on chr 4, 5, and 8 (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> 2013</a>). This finding suggests that there are major differences in the genome structures of different accessions, which are all classed as <em>S. lycopersicum</em> var. <em>cerasiforme</em>. It is proposed that the lines used by <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> (2013)</a> are an admixture of <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em> genomes, whereas the origin of <em>S. lycopersicum</em> var. <em>cerasiforme</em> line E9 appears to be a different admixture, consistent with multiple events in which reversion of cultivars back to the wild creates many independent forms of <em>S. lycopersicum</em> var. <em>cerasiforme</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ranc <em>et al.</em> 2008</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/971/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative distribution of single-nucleotide polymorphisms (SNPs) and insertion&#x2212;deletion mutations (InDels) on the E9 and L5 chromosomes. The number of SNPs and InDels per unit length of chromosome are given for each chromosome. For both genomes, values are expressed relative to the chromosome with the maximum linear density of polymorphisms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1644300039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative distribution of single-nucleotide polymorphisms (SNPs) and insertion&#x2212;deletion mutations (InDels) on the E9 and L5 chromosomes. The number of SNPs and InDels per unit length of chromosome are given for each chromosome. For both genomes, values are expressed relative to the chromosome with the maximum linear density of polymorphisms.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/971/F1.medium.gif" width="440" height="115"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/971/F1.medium.gif" width="440" height="115"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/971/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/971/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465982" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Relative distribution of single-nucleotide polymorphisms (SNPs) and insertion−deletion mutations (InDels) on the E9 and L5 chromosomes. The number of SNPs and InDels per unit length of chromosome are given for each chromosome. For both genomes, values are expressed relative to the chromosome with the maximum linear density of polymorphisms.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">In the <em>S. pimpinellifolium</em> L5 line, more than seven million sequence changes were detected with an average of ~610,000 per chromosome (~590,000 if the nonassembled chr 0 changes are included); this represents one sequence change every 111 base pairs. In general, therefore, the number of polymorphisms between L5 and Heinz 1706 is approximately 10 times greater than the number between E9 and Heinz 1706. In contrast to E9, the sequence changes in L5 were relatively evenly distributed between the 12 chromosomes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The most notable variation was that chr 8 had twice the linear density of polymorphisms as chr 9. It is worth mentioning that chr 5 and chr 8, which are highly divergent L5 chromosomes, also show prominent sequence changes in the cherry-type tomato admixture lines of <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em> (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> 2013</a>), suggesting that some of the admixture from <em>S. pimpinellifolium</em> is present on these two chromosomes (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ranc <em>et al.</em> 2008</a>).</p></div><div id="sec-9" class="subsection"><h3>Intrachromosomal SNP and InDel distributions and putative introgression sites</h3><p id="p-16">SNP and InDel distributions on each chromosome were analyzed to reveal the regions which are the most polymorphic to the reference genome. Most chromosomal sections in L5 had between 2000 and 20,000 SNPs per megabase (Mb) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). However, there were three chromosomal segments where the number of SNPs was particularly high (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>); chr 1 (9.63−11.56 Mb), chr 4 (9.15−11.60 Mb) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS1.pdf">Figure S1</a>) and chr 8 (48.48−48.76 Mb) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS2.pdf">Figure S2</a>). Because these peaks are absent from E9 (so that E9 and Heinz 1706 are similar in these regions) the most likely hypothesis is that there were ancestral introgression events that occurred in the progenitor of E9 and Heinz 1706 where the donor species was highly divergent from L5. Alternatively the peaks in L5 may simply reflect different ancient origins of the <em>S. pimpinellifolium</em> DNA and consequent variable divergence from Heinz 1706 and E9.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/971/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear density of single-nucleotide polymorphisms (SNPs) along the chromosomes of E9 and L5. The number of SNPs within each 1-Mb window (Y axis) is plotted against chromosomal positions for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9 specific SNP accumulations, red stars stand for L5 specific increase. Black stars stand for similar SNP score in both lines; green stars show the regions with a zero SNP value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1644300039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Linear density of single-nucleotide polymorphisms (SNPs) along the chromosomes of E9 and L5. The number of SNPs within each 1-Mb window (Y axis) is plotted against chromosomal positions for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9 specific SNP accumulations, red stars stand for L5 specific increase. Black stars stand for similar SNP score in both lines; green stars show the regions with a zero SNP value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/971/F2.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/971/F2.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/971/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/971/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466046" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Linear density of single-nucleotide polymorphisms (SNPs) along the chromosomes of E9 and L5. The number of SNPs within each 1-Mb window (Y axis) is plotted against chromosomal positions for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9 specific SNP accumulations, red stars stand for L5 specific increase. Black stars stand for similar SNP score in both lines; green stars show the regions with a zero SNP value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Similar to the situation in L5, there are also chromosomal regions in E9 possessing noticeably more SNPs than the surrounding DNA (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>): chr 5 (60.11−61.24 Mb) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS2.pdf">Figure S2</a>) and chr 11 (3.51−4.62 Mb) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS3.pdf">Figure S3</a>). In these locations we detected almost as many changes as observed in L5, with approximately 5000 SNPs within 1 Mb. This finding suggests introgression regions that are specific for E9 because the changes are rather different from the SNPs found in L5 or Micro-Tom. In addition to these two E9 regions, there are another two chromosome sections where there were peaks in genetic differences, where approximately more than 10,000 SNPs were found (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), which was much higher than the typical ∼1000 mutations per Mb in E9. Interestingly, both regions (chr 1; 10.55−11.14 Mb and chr 10; 42.11−42.37 Mb) are also highly polymorphic in L5 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS4.pdf">Figure S4</a>); the similar positions and identities of polymorphisms in these two regions suggests an introgression in Heinz 1706 derived from sources divergent to both L5 and E9. Micro-Tom also shows high polymorphism in the chr 1 region (10.55−11.14 Mb), further supporting the idea of a divergent introgression in Heinz 1706 in this position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS3.pdf">Figure S3</a>), whereas in the chr 10 region (42.11−42.37 Mb) Micro-Tom is almost identical to the reference genome, signifying that this probable introgression is present in both Heinz and Micro-Tom (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS4.pdf">Figure S4</a>). A further possibility is that an introgression occurred in E9 or L5 which was then transferred to the other during breeding or natural hybridizations. A region on chr 4 between 60.32−62.03 Mb shows a unique property in E9: it possesses more SNPs on average than L5, making it the only region in E9 with this property. The high accumulation of changes in E9 and the almost entire absence of SNPs and InDels in L5 strongly indicate an introgression event occurred in Heinz from <em>S. pimpinellifolium</em>. Moreover, Micro-Tom possesses highly similar SNPs and InDels to E9 in this region which also supports the introgression of <em>S. pimpinellifolium</em> in Heinz (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS4.pdf">Figure S4</a>). Interestingly, these data seem to differ from the results of another sequencing project where the distal part of chr 4 showed high SNPs between Heinz 1706 and S. <em>pimpinellifolium</em> lines (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Tomato Genome Consortium 2012</a>). The origin of this particular region is of interest because numerous, potentially disturbed genes are located within it (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-19">The InDels showed similar chromosomal distributions to the SNPs. The obvious difference between them is the lower (5−10 times) amount of InDels in the same region. Similarly to the SNP data, chr 1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS1.pdf">Figure S1</a>) and chr 8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS2.pdf">Figure S2</a>) contain a prominent number of InDels in L5. This is not always true, however, because in the case of chr 4, the InDel number is lower than might be expected considering the SNP linear density. On chr 5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS2.pdf">Figure S2</a>) and chr 11 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS3.pdf">Figure S3</a>) of E9, there are also significantly more InDels compared with the SNP density in these regions; however it is not possible to determine whether these alterations are significant or due to variations in the mapping quality in these two cases. The SNP-rich region of E9 on chr 1 and chr 10 (42.11-42.37 Mb) also possess large number of InDels. Moreover, the region chr 4 between 60.32-62.03 Mb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS4.pdf">Figure S4</a>) carrying E9 specific SNPs has also elevated number of InDels. Thus, most InDel values reflect the corresponding SNP pattern along the chromosomes.</p><p id="p-20">Surprisingly, there is a significant difference in the SNP and InDel distributions at the chromosome scale in both E9 and L5. For SNPs, apart from the above discussed specific regions with high or low SNP accumulations, the rest of the SNPs are relatively evenly distributed along the chromosomes in both lines. In contrast, the InDels have higher density close to the proximal and distal part of the chromosomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The difference between the SNP (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>) and InDel (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) accumulation is rather apparent in the case of chromosome 9 and 10. Indeed, the pattern of InDel density follows the accumulation of genes in the same region (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kobayashi <em>et al.</em> 2014</a>). This overlap between genes and InDels suggests that the accumulation and/or selection of InDels is related to the euchromatin, the gene-rich regions, where recombination events may predominantly occur, while the SNP distributions are more evenly distributed between euchromatin and heterochromatin in the tomato genome. These findings are in accordance with the investigations of SNP and InDel distributions in sorghum chromosomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Evans <em>et al.</em> 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/971/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear density of insertion&#x2212;deletion mutations (InDels) along the chromosomes of E9 and L5. The number of InDels within each 1-Mb window (Y axis) is plotted against chromosomal position for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9-specific single-nucleotide polymorphisms (SNPs) accumulation, red star stands for L5 specific increase. Black stars stand for high SNP score in both lines; green stars show the regions with a zero InDel value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1644300039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Linear density of insertion&#x2212;deletion mutations (InDels) along the chromosomes of E9 and L5. The number of InDels within each 1-Mb window (Y axis) is plotted against chromosomal position for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9-specific single-nucleotide polymorphisms (SNPs) accumulation, red star stands for L5 specific increase. Black stars stand for high SNP score in both lines; green stars show the regions with a zero InDel value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/971/F3.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/971/F3.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/971/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/971/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Linear density of insertion−deletion mutations (InDels) along the chromosomes of E9 and L5. The number of InDels within each 1-Mb window (Y axis) is plotted against chromosomal position for each of the 12 chromosomes (X axis). Blue: E9; red: L5. Blue star marks the E9-specific single-nucleotide polymorphisms (SNPs) accumulation, red star stands for L5 specific increase. Black stars stand for high SNP score in both lines; green stars show the regions with a zero InDel value because of gaps in the SL2.50 reference sequence. Gene rich regions (more than 50 genes in 0.5 Mb) are marked with a brown line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Beside the high sequence variations, both E9 and L5 lines possessed regions were the SNPs and InDels were at a low level or completely absent within a 1 Mb window (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). These regions correspond to large gaps in the reference genomic sequence. Most likely they overlap with the highly heterochromatic tomato centromeres (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chang <em>et al.</em> 2008</a>) containing repetitive sequences which impeded the <em>de novo</em> assembly of the reference sequence.</p><p id="p-23">These observations on the SNP and InDel distributions give important information about the potential introgressions and can contribute to our understanding of the QTL variations and their ecological significance in the genetic backgrounds in which they underwent natural selection. The data on the frequency of polymorphisms are important in designing strategies to fine map QTL to identify the genes responsible for influencing the trait.</p></div><div id="sec-10" class="subsection"><h3>InDel frequency and sizes</h3><p id="p-24">In addition to the 742,963 SNPs, 93,712 insertions and 55,702 deletions were found in E9. L5 had 432,545 insertions and 380,701 deletions beside the 6,934,608 SNPs. The size of detected insertions and deletions compared to Heinz 1706 were up to ~200−300 bp, even in the case of the more closely related E9. The longest natural deletions found were 212 bp in E9 and 238 bp in L5, whereas the longest insertions were 319 bp (E9) and 326 bp (L5). Compared with the reference genome, E9 has 394 deletions and 345 insertions longer than 50 bp, whereas the equivalent figures for L5 were 3445 deletions and 4106 insertions. This large number of InDels and their relatively even distribution along gene-rich regions facilitates the generation of simply-detectable polymorphic InDel-based molecular markers that can be used for fine genetic mapping of traits in the investigated RIL population. We have used even shorter (between 10-32 bp) InDel-based PCR markers in the process of fine mapping fruit weight QTL gFW9.1 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Estan <em>et al.</em> 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/FigureS5.pdf">Figure S5</a>). As InDels can be found mostly in the gene rich regions (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) we were interested to analyze the actual coding regions in further detail to see potential gene effects on gene function.</p></div><div id="sec-11" class="subsection"><h3>Resolution of SNPs and InDels in coding sequences</h3><p id="p-25">A further aim of this study was to search for SNPs and InDels that would have major effects on the function of gene coding sequences, and that might be expected to confer interesting and significant phenotypes. We observed that E9 and L5 showed very similar distributions between the different types of changes related to either coding or noncoding genomic sequences, but L5 has approximately 10 times more differences in each category. E9 has 41,593, whereas L5 has 410,684 changes in intron sequences. In the exons, E9 had 23,023 SNPs and InDels, whereas L5 had 150,623. The intergenic regions of E9 contain ~800,000 sequence changes while L5 has ~7 million. Thus the ratios of coding to noncoding sequence changes were comparable between the two parent lines: 35 and 45 times more SNPs and InDels were detected in noncoding <em>vs.</em> coding regions of genes in E9 and L5, respectively.</p><p id="p-26">We carefully investigated the distinct change types within the coding regions to determine the degree to which they were expected to alter protein structure and function. The category with the highest number of changes is the non-synonymous amino acid (aa) differences where E9 has more than 13,000 while L5 has approximately 83,000 changes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS2.xlsx">Table S2</a>). These numbers, particularly in the <em>S. pimpinellifolium</em> L5 line, are notably high because the number of detected nonsynonymous aa changes was more than the double of the total genes number (~36,000) in the investigated plant genome. Without functional genetic or transgenic studies it is difficult to conclude whether these aa changes result in any alteration in phenotype. However, the study of InDel data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS2.xlsx">Table S2</a>) to discover more severe protein sequence changes could be valuable in finding genes underlying QTL or monogenic mutations. Frame shift modifications may result in unquestionable protein sequence modification and consequently altered or null functional characteristics.</p></div><div id="sec-12" class="subsection"><h3>Genes influenced by InDels causing FS</h3><p id="p-27">Using SnpEff software we found 1296 and 5461 FS InDels in E9 and L5, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS2.xlsx">Table S2</a>). This raises the question of whether it is plausible to have such a large number of disruptive gene changes without rendering a genome unviable. Therefore, we investigated these FS InDels further using a systematic manual analysis protocol to determine which persuasive, large effect gene changes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">McNally <em>et al.</em> 2009</a>) were most likely to cause functional alterations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). First, we eliminated the genes which have heterozygous InDels. Given that the E9 and L5 lines are the results of inbreeding, they must possess very low residual heterozygosity, and potential sequencing and read mapping errors are the most likely origin of heterozygous polymorphisms. Therefore, we chose to study the coding regions possessing only homozygous FS changes. This resulted in 494 genes in E9 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS3.xlsx">Table S3</a>) and 2904 in L5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS4.xlsx">Table S4</a>). Afterward, based on RNA-seq results within existing databases, the genes without detectable RNA expression levels were also filtered out to focus on only genes with experimentally verified transcriptional activity. Of the 494 E9 genes containing FS InDels, only 30 seems to be considerably active, the others are likely nonfunctional or pseudogenes, or possibly expressed in rare conditions, or at a very low level. L5 has 2904 FSs in genes, but only 142 have reported RNA expression level. Next we discovered that several insertions or deletions mark the start or stop codon sites or the exon-intron borders, where the change of one base pair provoked no genuine FS. These are mostly insertions and deletions of a base pair that is identical to its neighbor resulting in no actual FS. These can be considered pseudo FS, detected by an imprecise algorithm in the SnpEff software. If we exclude these pseudo FS, E9 has now only three remaining genes containing FS InDels (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These three InDels have large and clear effects on the three proteins because they miss entire functional domains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS5.xlsx">Table S5</a>). Excluding the pseudo FS in L5, reduced the number of FS InDels from 142 to 39 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and these 39 mutations were studied more comprehensively. Most seemed to result in a small shortening of the N or C terminal part of the proteins by only a few aa which may presumably not provoke major functional modifications (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS6.xlsx">Table S6</a>), for example in the case of <em>Solyc01g058160</em>, <em>Solyc01g095620</em> or <em>Solyc09g082630</em> genes in L5 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). By aligning with BLAST the FS InDel-containing L5 protein sequences with available database sequences, it was apparent that some of them were more likely to represent complete, functional L5 genes, and the allelic annotated gene of the reference genome was therefore either functionally compromised, or wrongly predicted during the annotation. For instance, the analysis showed that in some cases the L5 protein structure was more similar to the potato ortholog than to the Heinz 1706 reference allele. This is likely the case for <em>Solyc01g050040</em>, <em>Solyc06g005080</em>, <em>Solyc06g084160</em> and <em>Solyc11g005420</em> genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/971/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Filtering process for frame shift (FS) mutated genes. The different steps of the flowchart show the process of establishing which FS insertion&#x2212;deletion mutations (InDels) led to genuine, large effect changes in the protein structure. SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1644300039" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Filtering process for frame shift (FS) mutated genes. The different steps of the flowchart show the process of establishing which FS insertion&#x2212;deletion mutations (InDels) led to genuine, large effect changes in the protein structure. SNP, single-nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/971/F4.medium.gif" width="372" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/971/F4.medium.gif" width="372" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/971/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/971/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466148" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Filtering process for frame shift (FS) mutated genes. The different steps of the flowchart show the process of establishing which FS insertion−deletion mutations (InDels) led to genuine, large effect changes in the protein structure. SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466121/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466121/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The frame shift altered proteins in E9 compared with reference genome</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465968/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465968/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The frame shift altered proteins in L5 compared with reference genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Despite the measurable RNA expression levels, a few of the L5 genes containing FS InDels appeared to encode incomplete proteins both in the reference genome and in L5, <em>e.g.</em>, <em>Solyc01g017050</em> or <em>Solyc01g091150</em> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), based on comparisons with orthologous proteins; this decreases the likelihood of their functional requirement in tomato, so these genes were also considered to be of less interest in our functional analysis. After eliminating these uncertain FS InDel changes still 7 genes exhibit altered protein sequence in L5 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). These FSs are undoubtedly changing the protein structures as entire domains, motifs are absent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-32">The final group of FS InDel genes to consider are those that affected protein structure similarly in both in E9 and L5 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). By searching for homologous genes we could divide these FS InDels in two categories: the ones that are mutated from the normal protein structure in both E9 and L5, and, second, the others that appear to be mutated in Heinz 1706 but not in E9 or L5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS7.xlsx">Table S7</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466147/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466147/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The frame shift altered proteins where E9 and L5 are similar and they both differ from the reference genome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Functions of genes containing disruptive InDels in E9 and L5</h3><p id="p-34">To verify the specificity of the determined nine FS InDels responsible for significantly altered proteins in these lines, we next made a comparison with another 12 related cultivars and wild species from the list of 150 tomato accessions which have been re-sequenced (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">100 Tomato Genome Sequencing Consortium <em>et al.</em> 2014</a>). As expected, most of the revealed InDels are not uniquely present in the E9 and L5 lines, but were represented within a limited number of other cultivars (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). As the E9 and L5 derived P population was generated to localize QTL responsible for salt tolerance in stress conditions we were interested whether any of the putatively influential InDels could somehow be involved in regulation of abiotic stresses responses. The E9-disrupted gene Solyc03g115650, a translation initiation factor 5A-1, is a stress-responsive protein involved in the abscisic acid (ABA) signal transduction pathway in <em>Tamarix androssowii</em>, a tropical plant possessing high drought and salt tolerance (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2012</a>). Beside E9 line, this gene has the same FS InDel in <em>S. lycopersicum</em> cv “Rutgers” (SLR), <em>S. lycopersicum</em> cv “Katinka Cherry” (SLKC) and <em>S. lycopersicum</em> cv “Black Cherry” (SLBC) lines (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), from which the last two are closely related cherry tomato cultivars. The <em>Solyc12g038920</em> gene, having FS deletion, codes for a serine-threonine kinase and shows high homology for a rice C-type cyclin-dependent protein kinase, which is induced by salt stress and ABA treatments, probably involved in the plant developmental program and ABA-signaling pathway (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> 2008</a>). This FS seems to be very specific for the E9 line as it is not present in any other line (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). The E9 gene Solyc12g100290 containing an FS InDel encodes a histone-lysine N-methyltransferase homolog, which is related to the <em>SUVH</em> gene family in <em>Arabidopsis thaliana</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Thorstensen <em>et al.</em> 2011</a>). This family is responsible for epigenetic silencing of chromatin histones and participates in the regulation of genes and transposons, which can be induced by biotic or abiotic stresses. This mutation is present also in the heritage cultivar <em>S. lycopersicum</em> cv Ailsa Craig (SLAC) and the SLKC line.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466014/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466014/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The occurrence of selected E9 and L5 frame shift alterations in other tomato accessions</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Our FS InDel data on L5 also was investigated in other <em>S. pimpinellifolium</em> sequences. The three available lines from the 150 tomato genome resequencing project (LYC2798, LA1584, LA1578) have also these InDels; however, only the LYC2798 line possess the same InDel distribution as L5 (probably a very close line), whereas LA1584 and LA1578 have fewer of the investigated InDels (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). <em>Solyc01g090610</em> codes for a gibberellin 2-beta-dioxygenase homolog which is induced by salinity stress in rice (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mizuno <em>et al.</em> 2010</a>); this enzyme is responsible for inactivating gibberellins. Solyc03g098240 is a glutamate decarboxylase homolog, which catalyses the conversion of glutamate into gamma-aminobutyric acid. Exogenous gamma-aminobutyric acid treatment protects tomato seedlings from chilling stress by enhancing antioxidant enzymes activities and maintaining membrane integrity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Malekzadeh <em>et al.</em> 2014</a>). These two genes have exclusively changed in L5 and LYC2798 line. The <em>Solyc04g010040</em> gene codes for an RNA recognition motif-containing protein and belongs to the large RNA binding protein family whose members are known to be involved in regulatory processes in response to different abiotic stress conditions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ambrosone <em>et al.</em> 2012</a>). Interestingly, while the other <em>S. pimpinellifolium</em> lines LA1584 and LA1578 have no InDel in <em>Solyc04g010040</em>, the SLBC and SLKC tomato lines do. Moreover, another cherry line, <em>S. lycopersicum</em> var. <em>cerasiforme</em> “Cervil” (SLCC) also has this InDel change. This tomato variant possesses similarities with L5 as the SLCC line is an admixture of <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em> lines (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Causse <em>et al.</em> 2013</a>). Solyc09g089690 is an aminocyclopropane-1-carboxylate oxidase protein, which is induced in tomato by attack of a certain <em>Fusarium oxysporum f.sp. radicis-lycopersici</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Çakir <em>et al.</em> 2014</a>) and is an enzyme in ethylene biosynthesis. The same InDel is present in SLKC and SLCC lines. Solyc11g012050 is a HT1-like serine-threonine-protein kinase, which regulates the stomatal closure in <em>A. thaliana</em> leaves via the ABA-signaling pathway (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hashimoto <em>et al.</em> 2006</a>). The FS InDel within this gene is only present in <em>S. pimpinellifolium</em> lines L5, LYC2798 and LA1578. Solyc11g067080 is a protein kinase which shows high homology to kinesin-like calmodulin-binding proteins that are primarily responsible for the microtubule-based movement of the plant cell (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vinogradova <em>et al.</em> 2013</a>). Nonetheless, this FS InDel seems to be less specific as it can be found in several other lines including <em>S. lycopersicum</em> cv “Moneymaker,” SLAC, SLR, SLKC, and SLBC tomato as well. The function of Solyc09g018670, a transmembrane protein 56B is not yet analyzed in plants, however, it seems to possess a highly homologous, conservative sequence in <em>A. thaliana</em> and soybean. The FS InDel of this gene is also present in the SLCC admixture line.</p><p id="p-37">The FS InDels of E9 and L5 show rather specific presence in a few lines. Interestingly, most of them are present in SLKC and SLCC lines suggesting common progenitors between them. The function of most genes were directly related to abiotic stresses, including salt tolerance properties, which are the target traits in the E9 × L5 RIL population.</p></div><div id="sec-14" class="subsection"><h3>Important InDels that occur in both E9 and L5</h3><p id="p-38">From the significant FS InDels, by homology searches we concluded that the <em>Solyc02g085420</em>, <em>Solyc05g054640</em>, <em>Solyc07g065630</em>, <em>Solyc09g007770</em>, <em>Solyc10g083190</em> and <em>Solyc10g083870</em> genes have sequence alteration both in E9 and L5 compared to the functional one in Heinz 1706 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1/TableS7.xlsx">Table S7</a>). Although, in these cases the non-functional FS InDels seems to be not limited only to the two parental lines, most investigated cultivars carry these InDels, which suggests that the functions of the affected genes are likely less significant and not critical. Only the gene of the U1 small nuclear ribonucleoprotein 70 (Solyc02g085420), a member of the spliceosomal machinery (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Valadkhan and Jaladat 2010</a>) seems to have the FS only in limited number of lines, in SLR and SLKC in addition to E9 and L5 (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-39">On the other hand, the FS InDels genes of <em>Solyc07g065220</em>, <em>Solyc09g005580</em> show truncated and probably non-functional gene sequence in Heinz 1706. These FS changes seem to be rather specific for the reference genome as most other investigated lines have the obviously correct gene sequence (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). Solyc07g065220 is a calcium-dependent lipid-binding protein that shows high homology to the C2-domain containing phospholipid-binding protein of rice, which is required for pollen fertility (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2008</a>). The function of Solyc09g005580 orthologs is described only in the animal kingdom so far. The Heinz 1706 <em>Solyc12g011030</em> gene carries a different exon sequence than the rest of the lines, however, both types can be functional. All these InDel data may be highly useful during possible molecular analyses of these particular genes, or as candidate genes in QTL mapping.</p><p id="p-40">Next-generation sequencing−based resequencing greatly facilitates investigations in plant genetics, speeding up genetic mapping projects and breeding programs. In our study, we used deep sequencing data of the two parental lines to discover SNPs and InDels for use in the fine-mapping of QTL in the descendant RIL populations. The sequence data are now publically available in the interactive and decipherable Genoverse format to search for essential SNPs and InDels which can be particularly useful for studies related to the <em>S. pimpinellifolium</em> genome. We have revealed regions which may harbor new introgressions in both E9 and L5 and determined which InDels are most likely to strongly influence gene functions. The data generated have particular importance for genetic analysis: several salinity and rootstock related QTL of the E9 and L5 derived P population are under investigation in different laboratories where these data will be particularly valuable. The detailed SNP and InDel data provide very convenient PCR markers for fine mapping, the InDel functional analysis provides many genes for further functional investigation, particularly where co-localization with QTL is observed. The FS InDel changes that we characterized undoubtedly represent only a small portion of the numerous probable gene function changes, but our investigation of this class of mutation underlined the limitations of existing algorithms which generated many pseudo FS InDels.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank Maria Jose Asins, IVIA, Valencia, Spain, for providing seeds of E9 and L5 and a seed population showing residual heterozygocity at gFW9.1. We thank Lázaro E. P. Peres, University of São Paulo, Brazil, for the Micro-Tom seeds. We thank Eugene Bragin, Genome Research, Ltd, Cambridge, for assistance with Genoverse. The 150 tomato genome data were produced by the 150 tomato genome consortium in collaboration with the user community. The research leading to these results has received funding from the European Union’s Seventh Framework Program (<em>FP7/2007-2013</em>) under grant agreement n° 289365. We are grateful for The Genome Analysis Centre, Norwich for Illumina sequencing of L5 and E9, and University of Warwick Genomics Resource Centre for Illumina sequencing of Micro-Tom. The authors declare that they have no competing interests.</p><p id="p-42">Z.K. wrote the article. A.J.T. extracted the genomic DNA and edited the article. R.C.K. and F.M. assembled and analyzed the E9 and L5 Illumina sequence data and set up the genome browser. M.J.S. and S.Z.A. generated and analyzed the Micro-Tom sequence data. Z.K. analyzed the SnpEff data and ran InDel PCR markers. F.M. and A.J.T. conceived and directed the research. All authors read and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-43">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016121/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-44"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-10"><span class="received-label">Received </span>December 10, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-22"><span class="accepted-label">Accepted </span>March 22, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Kevei <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.1" data-doi="10.1016/j.plantsci.2011.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ambrosone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grillo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Beyond transcription: RNA-binding proteins as emerging regulators of plant response to environmental constraints.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BSci%26rft.aulast%253DAmbrosone%26rft.auinit1%253DA.%26rft.volume%253D182%26rft.spage%253D12%26rft.epage%253D18%26rft.atitle%253DBeyond%2Btranscription%253A%2BRNA-binding%2Bproteins%2Bas%2Bemerging%2Bregulators%2Bof%2Bplant%2Bresponse%2Bto%2Benvironmental%2Bconstraints.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2011.02.004%26rft_id%253Dinfo%253Apmid%252F22118611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2011.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118611&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.2" data-doi="10.1007/s00122-010-1294-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolarin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Alfocea</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Andujar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of physiological components of salt tolerance conferred by Solanum rootstocks. What is the rootstock doing for the scion?</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAsins%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D115%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bphysiological%2Bcomponents%2Bof%2Bsalt%2Btolerance%2Bconferred%2Bby%2BSolanum%2Brootstocks.%2BWhat%2Bis%2Bthe%2Brootstock%2Bdoing%2Bfor%2Bthe%2Bscion%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1294-9%26rft_id%253Dinfo%253Apmid%252F20180091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1294-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20180091&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.3" data-doi="10.1111/pce.12051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aly</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olias</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Morales</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Two closely linked tomato HKT coding genes are positional candidates for the major tomato QTL involved in Na<sup>+</sup> /K<sup>+</sup> homeostasis.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D36%26rft.spage%253D1171%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000318176800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.4" data-doi="10.1007/s11032-014-0141-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrantes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-del-Carmen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Casado</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Sánchez</span>  <span class="cit-name-given-names">M. Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Muñoz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Highly efficient genomics-assisted development of a library of introgression lines of <em>Solanum pimpinellifolium</em></span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D1817%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0141-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0141-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Çakir</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gül</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yolageldi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Özaktan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Response to <em>Fusarium oxysporum</em> f.sp. <em>radicis-lycopersici</em> in tomato roots involves regulation of SA- and ET-responsive gene expressions.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D139%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.6" data-doi="10.1186/1471-2164-14-791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desplat</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Paslier</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole genome resequencing in tomato reveals variation associated with introgression and breeding events.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-791%26rft_id%253Dinfo%253Apmid%252F24228636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24228636&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.7" data-doi="10.1007/s10577-008-1249-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datema</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Vugt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FISH mapping and molecular organization of the major repetitive sequences of tomato.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DChang%26rft.auinit1%253DS.%2BB.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D919%26rft.epage%253D933%26rft.atitle%253DFISH%2Bmapping%2Band%2Bmolecular%2Borganization%2Bof%2Bthe%2Bmajor%2Brepetitive%2Bsequences%2Bof%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-008-1249-z%26rft_id%253Dinfo%253Apmid%252F18688733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-008-1249-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18688733&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260510400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.8" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang le</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster strain w1118</em>; <em>iso-2</em>; <em>iso-3</em>.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.9" data-doi="10.1007/s00122-008-0900-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolarin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of fruit yield loci controlling the salt tolerance conferred by solanum rootstocks</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DEstan%26rft.auinit1%253DM.%2BT.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D312%26rft.atitle%253DIdentification%2Bof%2Bfruit%2Byield%2Bloci%2Bcontrolling%2Bthe%2Bsalt%2Btolerance%2Bconferred%2Bby%2Bsolanum%2Brootstocks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0900-6%26rft_id%253Dinfo%253Apmid%252F18846361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0900-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846361&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.10" data-doi="10.1371/journal.pone.0079192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e79192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De79192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0079192%26rft_id%253Dinfo%253Apmid%252F24265758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0079192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24265758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Muñoz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Álvarez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuartero</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inheritance of resistance to two-spotted spider mite and glandular leaf trichomes in wild tomato <em>Lycopersicon pimpinellifolium</em> (Jusl.) Mill.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D128%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.12" data-doi="10.1080/07352680802147353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foolad</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merk</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetics, genomics and breeding of late blight and early blight resistance in tomato.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D27%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352680802147353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352680802147353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.13" data-doi="10.1007/s001220051405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Identifying the loci responsible for natural variation in fruit size and shape in tomato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D978%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083542000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.14" data-doi="10.1038/ncb1387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Israelsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>Arabidopsis</em> HT1 kinase controls stomatal movements in response to CO<sub>2</sub>.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DHashimoto%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D391%26rft.epage%253D397%26rft.atitle%253DArabidopsis%2BHT1%2Bkinase%2Bcontrols%2Bstomatal%2Bmovements%2Bin%2Bresponse%2Bto%2BCO2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1387%26rft_id%253Dinfo%253Apmid%252F16518390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16518390&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236560000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.15" data-doi="10.1093/dnares/dst005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aoki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide SNP genotyping to infer the effects on gene functions in tomato.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdst005%26rft_id%253Dinfo%253Apmid%252F23482505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMjIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.16" data-doi="10.1016/j.plaphy.2007.10.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsay</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Increased expression of the rice C-type cyclin-dependent protein kinase gene, Orysa;<em>CDKC;1</em>, in response to salt stress.</span> <abbr class="cit-jnl-abbrev">Plant Physiol. Biochem.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bphysiology%2Band%2Bbiochemistry%2B%253A%2B%2BPPB%2B%252F%2BSoci%25C3%25A9t%25C3%25A9%2Bfran%25C3%25A7aise%2Bde%2Bphysiologie%2Bv%25C3%25A9g%25C3%25A9tale%26rft.stitle%253DPlant%2BPhysiol%2BBiochem%26rft.aulast%253DHuang%26rft.auinit1%253DY.%2BW.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D81%26rft.atitle%253DIncreased%2Bexpression%2Bof%2Bthe%2Brice%2BC-type%2Bcyclin-dependent%2Bprotein%2Bkinase%2Bgene%252C%2BOrysa%253BCDKC%253B1%252C%2Bin%2Bresponse%2Bto%2Bsalt%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plaphy.2007.10.013%26rft_id%253Dinfo%253Apmid%252F18054244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plaphy.2007.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18054244&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.17" data-doi="10.1093/pcp/pct181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagasaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Just</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide analysis of intraspecific DNA polymorphism in ‘micro-tom’, a model cultivar of tomato (<em>Solanum lycopersicum</em>)</span>. <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpct181%26rft_id%253Dinfo%253Apmid%252F24319074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjg6IjU1LzIvNDQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.18" data-doi="10.1038/ng.3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic analyses provide insights into the history of tomato breeding.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1220</span>–<span class="cit-lpage">1226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3117%26rft_id%253Dinfo%253Apmid%252F25305757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25305757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.19" data-doi="10.1007/s12298-013-0203-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malekzadeh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khara</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heydari</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Alleviating effects of exogenous Gamma-aminobutiric acid on tomato seedling under chilling stress.</span> <abbr class="cit-jnl-abbrev">Physiol. Mol. Biol. Plants</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BMol.%2BBiol.%2BPlants%26rft.volume%253D20%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12298-013-0203-5%26rft_id%253Dinfo%253Apmid%252F24554847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12298-013-0203-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24554847&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.20" data-doi="10.1073/pnas.0900992106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohnert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide SNP variation reveals relationships among landraces and modern varieties of rice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">12273</span>–<span class="cit-lpage">12278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0900992106%26rft_id%253Dinfo%253Apmid%252F19597147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMwLzEyMjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.21" data-doi="10.1186/1471-2164-11-683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mizuno</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawahara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakimoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Massive parallel sequencing of mRNA in identification of unannotated salinity stress-inducible transcripts in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMizuno%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.spage%253D683%26rft.epage%253D683%26rft.atitle%253DMassive%2Bparallel%2Bsequencing%2Bof%2BmRNA%2Bin%2Bidentification%2Bof%2Bunannotated%2Bsalinity%2Bstress-inducible%2Btranscripts%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-683%26rft_id%253Dinfo%253Apmid%252F21122150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21122150&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.22" data-doi="10.1007/s001220050561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monforte</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonell</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Salt tolerance in <em>Lycopersicon</em> species. V. Does genetic variability at quantitative trait loci affect their analysis?</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peralta</span>  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spooner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The taxonomy of tomatoes: a revision of wild tomatoes (<em>Solanum</em> section <em>Lycopersicon</em>) and their outgroup relatives in sections <em>Juglandifolium</em> and <em>Lycopersicoides</em>.</span> <abbr class="cit-jnl-abbrev">Syst. Bot. Monogr.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBot.%2BMonogr.%26rft.volume%253D84%26rft.spage%253D1%26rft.atitle%253DSYST%2BBOT%2BMONOGR%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.24" data-doi="10.1094/PHYTO-01-11-0028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-López</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garzo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonani</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fereres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Muñoz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whitefly resistance traits derived from the wild tomato <em>Solanum pimpinellifolium</em> affect the preference and feeding behavior of <em>Bemisia tabaci</em> and reduce the spread of <em>Tomato yellow leaf curl virus</em>.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1191</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DRodriguez-Lopez%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D101%26rft.issue%253D10%26rft.spage%253D1191%26rft.epage%253D1201%26rft.atitle%253DWhitefly%2Bresistance%2Btraits%2Bderived%2Bfrom%2Bthe%2Bwild%2Btomato%2BSolanum%2Bpimpinellifolium%2Baffect%2Bthe%2Bpreference%2Band%2Bfeeding%2Bbehavior%2Bof%2BBemisia%2Btabaci%2Band%2Breduce%2Bthe%2Bspread%2Bof%2BTomato%2Byellow%2Bleaf%2Bcurl%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-01-11-0028%26rft_id%253Dinfo%253Apmid%252F21615206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-01-11-0028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21615206&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295185300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbaugh</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Micro-Tom. A miniature dwarf tomato.</span> <abbr class="cit-jnl-abbrev">Florida Agr. Expt. Sta. Circ.</abbr> <span class="cit-vol">370</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFlorida%2BAgr.%2BExpt.%2BSta.%2BCirc.%26rft.volume%253D370%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SNP discovery and linkage map construction in cultivated tomato.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft.volume%253D7%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.27" data-doi="10.1093/dnares/dst033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsunaga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdst033%26rft_id%253Dinfo%253Apmid%252F23903436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvNTkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.28" data-doi="10.1371/journal.pone.0040563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40563%26rft.epage%253De40563%26rft.atitle%253DDevelopment%2Bof%2Ba%2Blarge%2BSNP%2Bgenotyping%2Barray%2Band%2Bgeneration%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040563%26rft_id%253Dinfo%253Apmid%252F22802968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22802968&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.29" data-doi="10.1371/journal.pone.0045520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">High-density SNP genotyping of tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of genetic variation due to breeding.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45520%26rft.epage%253De45520%26rft.atitle%253DHigh-density%2BSNP%2Bgenotyping%2Bof%2Btomato%2B%2528Solanum%2Blycopersicum%2BL.%2529%2Breveals%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bdue%2Bto%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045520%26rft_id%253Dinfo%253Apmid%252F23029069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029069&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.30" data-doi="10.1186/1471-2229-8-130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muños</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A clarified position for <em>Solanum lycopersicum</em> var. <em>cerasiforme</em> in the evolutionary history of tomatoes (Solanaceae).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DRanc%26rft.auinit1%253DN.%26rft.volume%253D8%26rft.spage%253D130%26rft.epage%253D130%26rft.atitle%253DA%2Bclarified%2Bposition%2Bfor%2BSolanum%2Blycopersicum%2Bvar.%2Bcerasiforme%2Bin%2Bthe%2Bevolutionary%2Bhistory%2Bof%2Btomatoes%2B%2528solanaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-8-130%26rft_id%253Dinfo%253Apmid%252F19099601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-8-130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19099601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.31" data-doi="10.1007/s001220050116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative <em>L. pimpinellifolium</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D92%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050116%26rft_id%253Dinfo%253Apmid%252F24166170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166170&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TZ82500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.32" data-doi="10.1016/j.bbagrm.2011.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorstensen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grini</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aalen</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">domain proteins in plant development.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1809</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DThorstensen%26rft.auinit1%253DT.%26rft.volume%253D1809%26rft.issue%253D8%26rft.spage%253D407%26rft.epage%253D420%26rft.atitle%253DSET%2Bdomain%2Bproteins%2Bin%2Bplant%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2011.05.008%26rft_id%253Dinfo%253Apmid%252F21664308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2011.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21664308&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.33" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.34" data-doi="10.1002/pmic.201000354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valadkhan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaladat</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The spliceosomal proteome: at the heart of the largest cellular ribonucleoprotein machine.</span> <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">4128</span>–<span class="cit-lpage">4141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DValadkhan%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D22%26rft.spage%253D4128%26rft.epage%253D4141%26rft.atitle%253DThe%2Bspliceosomal%2Bproteome%253A%2Bat%2Bthe%2Bheart%2Bof%2Bthe%2Blargest%2Bcellular%2Bribonucleoprotein%2Bmachine.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201000354%26rft_id%253Dinfo%253Apmid%252F21080498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201000354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21080498&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285164500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.35" data-doi="10.1007/s00122-004-1906-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reina-Sánchez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuartero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative microsatellite linkage analysis and genetic structure of two populations of F6 lines derived from <em>Lycopersicon pimpinellifolium</em> and <em>L. cheesmanii</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVillalta%26rft.auinit1%253DI.%26rft.volume%253D110%26rft.issue%253D5%26rft.spage%253D881%26rft.epage%253D894%26rft.atitle%253DComparative%2Bmicrosatellite%2Blinkage%2Banalysis%2Band%2Bgenetic%2Bstructure%2Bof%2Btwo%2Bpopulations%2Bof%2BF6%2Blines%2Bderived%2Bfrom%2BLycopersicon%2Bpimpinellifolium%2Band%2BL.%2Bcheesmanii.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1906-3%26rft_id%253Dinfo%253Apmid%252F15690174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1906-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15690174&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.36" data-doi="10.1007/s00122-006-0494-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernet</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asins</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparative QTL analysis of salinity tolerance in terms of fruit yield using two solanum populations of F7 lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVillalta%26rft.auinit1%253DI.%26rft.volume%253D114%26rft.issue%253D6%26rft.spage%253D1001%26rft.epage%253D1017%26rft.atitle%253DComparative%2BQTL%2Banalysis%2Bof%2Bsalinity%2Btolerance%2Bin%2Bterms%2Bof%2Bfruit%2Byield%2Busing%2Btwo%2BSolanum%2Bpopulations%2Bof%2BF7%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0494-9%26rft_id%253Dinfo%253Apmid%252F17394031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0494-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17394031&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245362000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.37" data-doi="10.1007/s00122-008-0720-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villalta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reina-Sanchez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolarin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuartero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belver</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analysis of Na<sup>+</sup> and K<sup>+</sup> concentrations in leaf and stem as physiological components of salt tolerance in tomato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVillalta%26rft.auinit1%253DI.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D869%26rft.epage%253D880%26rft.atitle%253DGenetic%2Banalysis%2Bof%2BNa%2528%252B%2529%2Band%2BK%2B%2528%252B%2529%2Bconcentrations%2Bin%2Bleaf%2Band%2Bstem%2Bas%2Bphysiological%2Bcomponents%2Bof%2Bsalt%2Btolerance%2Bin%2BTomato.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0720-8%26rft_id%253Dinfo%253Apmid%252F18251001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0720-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18251001&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.38" data-doi="10.1371/journal.pone.0066669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinogradova</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malanina</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waitzman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fletterick</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Plant kinesin-like calmodulin binding protein employs its regulatory domain for dimerization.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e66669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66669%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066669%26rft_id%253Dinfo%253Apmid%252F23805258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23805258&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.39" data-doi="10.1186/1471-2164-14-354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viquez-Zamora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Geest</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Tomato breeding in the genomics era: insights from a SNP array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D354%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-354%26rft_id%253Dinfo%253Apmid%252F23711327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23711327&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.971.40" data-doi="10.1186/1471-2229-12-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterization of a eukaryotic translation initiation factor 5A homolog from <em>Tamarix androssowii</em> involved in plant abiotic stress tolerance.</span> <span class="cit-source">BMC Plant Biol.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.spage%253D118%26rft.epage%253D118%26rft.atitle%253DCharacterization%2Bof%2Ba%2Beukaryotic%2Btranslation%2Binitiation%2Bfactor%2B5A%2Bhomolog%2Bfrom%2BTamarix%2Bandrossowii%2Binvolved%2Bin%2Bplant%2Babiotic%2Bstress%2Btolerance.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-118%26rft_id%253Dinfo%253Apmid%252F22834699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22834699&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.971.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippman</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">The Draft Genome Sequence of Solanum pimpinellifolium LA1589</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brody</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Linkage analysis of quantitative traits in an interspecific cross of tomato (<em>Lycopersicon esculentum</em> x <em>Lycopersicon pimpinellifolium</em>) by means of genetic markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeller%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D329%26rft.epage%253D339%26rft.atitle%253DLinkage%2Banalysis%2Bof%2Bquantitative%2Btraits%2Bin%2Ban%2Binterspecific%2Bcross%2Bof%2Btomato%2B%2528lycopersicon%2Besculentum%2Bx%2Blycopersicon%2Bpimpinellifolium%2529%2Bby%2Bmeans%2Bof%2Bgenetic%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F17246412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE4LzIvMzI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.43" data-doi="10.1093/mp/ssn035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Y. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A novel C2-domain phospholipid-binding protein, OsPBP1, is required for pollen fertility in rice.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">770</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssn035%26rft_id%253Dinfo%253Apmid%252F19825580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssn035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825580&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259104500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.44" data-doi="10.1093/dnares/dsu008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>In Silico</em> identification and experimental validation of insertion-deletion polymorphisms in tomato genome.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu008%26rft_id%253Dinfo%253Apmid%252F24618211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNDI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.971.45" data-doi="10.1111/tpj.12616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">100 Tomato Genome Sequencing Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aflitos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schijlen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ridder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Exploring genetic variation in the tomato (<em>Solanum</em> section Lycopersicon) clade by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D80%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12616%26rft_id%253Dinfo%253Apmid%252F25039268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25039268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342721900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/971.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/963" title="The Maintenance of Single-Locus Polymorphism by Maternal Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/983" title="Comparative Analysis of Wolbachia Genomes Reveals Streamlining and Divergence of Minimalist Two-Component Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/971.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bn0Hqwx4hntwKhuKtcPrH16xYjx9uj2NhCskSMae9n4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/971" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/971" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/971" class="active" data-icon-position="" data-hide-link-title="0">Resequencing at ≥40-Fold Depth of the Parental Genomes of a Solanum lycopersicum × S. pimpinellifolium Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459839" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IUr6pDX-DJgOgyxjMiaBTFK3QZMzYmSzz9lt9aLWic8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/971.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/971" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-D116R7kpzMpji6eVtwuAwO5N5-Zvk4sbRzY-jtazNr0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/971" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/971.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459839" id="node459839--2" data-pisa="ggg;5/5/971" data-pisa-master="ggg;g3.114.016121" data-apath="/ggg/5/5/971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing at ≥40-Fold Depth of the Parental Genomes of a <em>Solanum lycopersicum</em> × <em>S. pimpinellifolium</em> Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zoltan Kevei</span>, <span class="highwire-citation-author" data-delta="1">Robert C. King</span>, <span class="highwire-citation-author" data-delta="2">Fady Mohareb</span>, <span class="highwire-citation-author" data-delta="3">Martin J. Sergeant</span>, <span class="highwire-citation-author" data-delta="4">Sajjad Z. Awan</span> and <span class="highwire-citation-author" data-delta="5">Andrew J. Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">971-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459839--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zoltan Kevei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BKevei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kevei%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZoltan%2BKevei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:z.l.kevei@cranfield.ac.uk" class="" data-icon-position="" data-hide-link-title="0">z.l.kevei@cranfield.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert C. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Rothamsted Research, Harpenden, Hertfordshire, AL5 2JQ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fady Mohareb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fady%2BMohareb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohareb%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFady%2BMohareb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin J. Sergeant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BJ.%2BSergeant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeant%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BJ.%2BSergeant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sajjad Z. Awan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sajjad%2BZ.%2BAwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Awan%20SZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASajjad%2BZ.%2BAwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew J. Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/971" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/971&amp;text=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/971&amp;t=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/971&amp;title=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459839/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459839" id="node459839--3" data-pisa="ggg;5/5/971" data-pisa-master="ggg;g3.114.016121" data-apath="/ggg/5/5/971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing at ≥40-Fold Depth of the Parental Genomes of a <em>Solanum lycopersicum</em> × <em>S. pimpinellifolium</em> Recombinant Inbred Line Population and Characterization of Frame-Shift InDels That Are Highly Likely to Perturb Protein Function</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zoltan Kevei</span>, <span class="highwire-citation-author" data-delta="1">Robert C. King</span>, <span class="highwire-citation-author" data-delta="2">Fady Mohareb</span>, <span class="highwire-citation-author" data-delta="3">Martin J. Sergeant</span>, <span class="highwire-citation-author" data-delta="4">Sajjad Z. Awan</span> and <span class="highwire-citation-author" data-delta="5">Andrew J. Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">971-981;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459839--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zoltan Kevei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zoltan%2BKevei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kevei%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZoltan%2BKevei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:z.l.kevei@cranfield.ac.uk" class="" data-icon-position="" data-hide-link-title="0">z.l.kevei@cranfield.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert C. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Rothamsted Research, Harpenden, Hertfordshire, AL5 2JQ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BC.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BC.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fady Mohareb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fady%2BMohareb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohareb%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFady%2BMohareb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin J. Sergeant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BJ.%2BSergeant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeant%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BJ.%2BSergeant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sajjad Z. Awan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, University of Warwick, Gibbet Hill Campus, Coventry, CV4 7AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sajjad%2BZ.%2BAwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Awan%20SZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASajjad%2BZ.%2BAwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew J. Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cranfield Soil and AgriFood Institute, School of Energy, Environment and Agrifood, Cranfield University, Cranfield, Bedfordshire, MK43 0AL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=971&amp;rft.epage=981&amp;rft.atitle=Resequencing%20at%20%E2%89%A540-Fold%20Depth%20of%20the%20Parental%20Genomes%20of%20a%20Solanum%20lycopersicum%20%C3%97%20S.%20pimpinellifolium%20Recombinant%20Inbred%20Line%20Population%20and%20Characterization%20of%20Frame-Shift%20InDels%20That%20Are%20Highly%20Likely%20to%20Perturb%20Protein%20Function&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kevei%2C+Zoltan&amp;rft.au=King%2C+Robert+C.&amp;rft.au=Mohareb%2C+Fady&amp;rft.au=Sergeant%2C+Martin+J.&amp;rft.au=Awan%2C+Sajjad+Z.&amp;rft.au=Thompson%2C+Andrew+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459839/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459839/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459839/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459839/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459839/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459839/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459839/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459839/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459839/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459839/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459839/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459839/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/971" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Result and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/971.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/971.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/971.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/971.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
