<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/891" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/891.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/891.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/891.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/5/891/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/891.atom" />
<meta name="HW.pisa" content="ggg;5/5/891" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" />
<meta name="DC.Identifier" content="10.1534/g3.115.017533" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2015 Ramstein et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping by sequencing allows for large-scale genetic analyses in plant species with no reference genome, but sets the challenge of sound inference in presence of uncertain genotypes. We report an imputation-based genome-wide association study (GWAS) in reed canarygrass ( Phalaris arundinacea L., Phalaris caesia Nees), a cool-season grass species with potential as a biofuel crop. Our study involved two linkage populations and an association panel of 590 reed canarygrass genotypes. Plants were assayed for up to 5228 single nucleotide polymorphism markers and 35 traits. The genotypic markers were derived from low-depth sequencing with 78% missing data on average. To soundly infer marker-trait associations, multiple imputation (MI) was used: several imputes of the marker data were generated to reflect imputation uncertainty and association tests were performed on marker effects across imputes. A total of nine significant markers were identified, three of which showed significant homology with the Brachypodium dystachion genome. Because no physical map of the reed canarygrass genome was available, imputation was conducted using classification trees. In general, MI showed good consistency with the complete-case analysis and adequate control over imputation uncertainty. A gain in significance of marker effects was achieved through MI, but only for rare cases when missing data were &lt;45%. In addition to providing insight into the genetic basis of important traits in reed canarygrass, this study presents one of the first applications of MI to genome-wide analyses and provides useful guidelines for conducting GWAS based on genotyping-by-sequencing data." />
<meta name="DC.Contributor" content="Guillaume P. Ramstein" />
<meta name="DC.Contributor" content="Alexander E. Lipka" />
<meta name="DC.Contributor" content="Fei Lu" />
<meta name="DC.Contributor" content="Denise E. Costich" />
<meta name="DC.Contributor" content="Jerome H. Cherney" />
<meta name="DC.Contributor" content="Edward S. Buckler" />
<meta name="DC.Contributor" content="Michael D. Casler" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping by sequencing allows for large-scale genetic analyses in plant species with no reference genome, but sets the challenge of sound inference in presence of uncertain genotypes. We report an imputation-based genome-wide association study (GWAS) in reed canarygrass (&lt;i&gt;Phalaris arundinacea&lt;/i&gt; L., &lt;i&gt;Phalaris caesia&lt;/i&gt; Nees), a cool-season grass species with potential as a biofuel crop. Our study involved two linkage populations and an association panel of 590 reed canarygrass genotypes. Plants were assayed for up to 5228 single nucleotide polymorphism markers and 35 traits. The genotypic markers were derived from low-depth sequencing with 78% missing data on average. To soundly infer marker-trait associations, multiple imputation (MI) was used: several imputes of the marker data were generated to reflect imputation uncertainty and association tests were performed on marker effects across imputes. A total of nine significant markers were identified, three of which showed significant homology with the &lt;i&gt;Brachypodium dystachion&lt;/i&gt; genome. Because no physical map of the reed canarygrass genome was available, imputation was conducted using classification trees. In general, MI showed good consistency with the complete-case analysis and adequate control over imputation uncertainty. A gain in significance of marker effects was achieved through MI, but only for rare cases when missing data were &amp;lt;45%. In addition to providing insight into the genetic basis of important traits in reed canarygrass, this study presents one of the first applications of MI to genome-wide analyses and provides useful guidelines for conducting GWAS based on genotyping-by-sequencing data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/891" />
<meta name="citation_id" content="5/5/891" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/891" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/891.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/891.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/891.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017533" />
<meta name="citation_pmid" content="25770100" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="891" />
<meta name="citation_lastpage" content="909" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Guillaume P. Ramstein" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="ramstein@wisc.edu" />
<meta name="citation_author" content="Alexander E. Lipka" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Fei Lu" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Denise E. Costich" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), 56237 Texcoco, Mexico" />
<meta name="citation_author" content="Jerome H. Cherney" />
<meta name="citation_author_institution" content="Soil and Crops Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Edward S. Buckler" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Agricultural Research Service, United States Department of Agriculture, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Michael D. Casler" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_institution" content="Agricultural Research Service, United States Department of Agriculture, Madison, Wisconsin 53706" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. SchÃ¤ffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=F. J. Alway;citation_title=Early Trials and Use of Reed Canary Grass as a Forage Plant;citation_pages=64-66;citation_volume=23;citation_year=1931;citation_issue=1;citation_doi=10.2134/agronj1931.00021962002300010008x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. Asay;citation_author=I. Carlson;citation_author=C. Wilsie;citation_title=Genetic Variability in Forage Yield, Crude Protein Percentage, and Palatability in Reed Canarygrass, Phalaris arundinacea L.;citation_pages=568-571;citation_volume=8;citation_year=1968;citation_issue=5;citation_doi=10.2135/cropsci1968.0011183X000800050018x" />
<meta name="citation_reference" content="citation_journal_title=Webbia;citation_author=R. M. Baldini;citation_title=Revision of the genus Phalaris L. (Gramineae).;citation_pages=265-329;citation_volume=49;citation_year=1995;citation_doi=10.1080/00837792.1995.10670587" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=J. Barnard;citation_author=D. B. Rubin;citation_title=Miscellanea. Small-sample degrees of freedom with multiple imputation;citation_pages=948-955;citation_volume=86;citation_year=1999;citation_issue=4;citation_doi=10.1093/biomet/86.4.948" />
<meta name="citation_reference" content="citation_title=Reed canarygrass - a production guide;citation_author=S. Bittman;citation_author=J. Waddington;citation_author=B. E. Coulman;citation_author=S. G. Bonin;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Fuel;citation_author=A. Boateng;citation_author=H. Jung;citation_author=P. Adler;citation_title=Pyrolysis of energy crops including alfalfa stems, reed canarygrass, and eastern gamagrass.;citation_pages=2450-2457;citation_volume=85;citation_year=2006;citation_doi=10.1016/j.fuel.2006.04.025" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=J. F. Bobb;citation_author=D. O. Scharfstein;citation_author=M. J. Daniels;citation_author=F. S. Collins;citation_author=S. Kelada;citation_title=Multiple imputation of missing phenotype data for QTL mapping.;citation_pages=1-27;citation_volume=10;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. Boe;citation_author=D. L. Beck;citation_title=Yield Components of Biomass in Switchgrass;citation_pages=1306-1311;citation_volume=48;citation_year=2008;citation_issue=4;citation_doi=10.2135/cropsci2007.08.0482" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=W. Boerjan;citation_author=J. Ralph;citation_author=M. Baucher;citation_title=Lignin biosynthesis.;citation_volume=54;citation_year=2003;citation_pmid=14503002;citation_doi=10.1146/annurev.arplant.54.031902.134938" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=Y. Bouchenak-Khelladi;citation_author=N. Salamin;citation_author=V. Savolainen;citation_author=F. Forest;citation_title=Large multi-gene phylogenetic trees of the grasses (Poaceae): progress towards complete tribal and generic level sampling.;citation_pages=488-505;citation_volume=47;citation_year=2008;citation_issue=2;citation_pmid=18358746;citation_doi=10.1016/j.ympev.2008.01.035" />
<meta name="citation_reference" content="citation_title=Classification and regression trees;citation_author=L. Breiman;citation_author=J. Friedman;citation_author=C. J. Stone;citation_author=R. A. Olshen;citation_year=1984" />
<meta name="citation_reference" content="citation_title=Switchgrass production in Iowa: economic analysis, soil suitability, and varietal performance;citation_author=E. Brummer;citation_author=C. Burras;citation_author=M. Duffy;citation_author=K. Moore;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Epidemiol.;citation_author=L. F. Burgette;citation_author=J. P. Reiter;citation_title=Multiple imputation for missing data via sequential regression trees.;citation_volume=172;citation_year=2010;citation_pmid=20841346;citation_doi=10.1093/aje/kwq260" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. R. Burton;citation_author=D. G. Clayton;citation_author=L. R. Cardon;citation_author=N. Craddock;citation_author=P. Deloukas;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual;citation_author=D. Butler;citation_author=B. R. Cullis;citation_author=A. Gilmour;citation_author=B. Gogel;citation_year=2007" />
<meta name="citation_reference" content="citation_title=Cool-season forage grasses;citation_title=Reed canarygrass and other Phalaris species;citation_author=I. T. Carlson;citation_author=R. N. Oram;citation_author=J. Surprenant;citation_pages=569-604;citation_year=1996" />
<meta name="citation_reference" content="citation_title=Genetics, breeding, and ecology of reed canarygrass;citation_author=M. Casler;citation_pages=30-36;citation_volume=4;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Casler;citation_author=M. Phillips;citation_author=A. Krohn;citation_title=DNA Polymorphisms Reveal Geographic Races of Reed Canarygrass;citation_pages=2139-2148;citation_volume=49;citation_year=2009;citation_issue=6;citation_doi=10.2135/cropsci2009.02.0055" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Research;citation_author=M. D. Casler;citation_author=J. H. Cherney;citation_author=E. C. Brummer;citation_title=Biomass yield of naturalized populations and cultivars of reed canary grass.;citation_pages=165-173;citation_volume=2;citation_year=2009;citation_doi=10.1007/s12155-009-9043-0" />
<meta name="citation_reference" content="citation_journal_title=Biomass;citation_author=J. Cherney;citation_author=K. Johnson;citation_author=J. Volenec;citation_author=K. Anliker;citation_title=Chemical composition of herbaceous grass and legume species grown for maximum biomass production.;citation_pages=215-238;citation_volume=17;citation_year=1988;citation_doi=10.1016/0144-4565(88)90105-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of Environmental Quality;citation_journal_abbrev=J. Environ. Qual.;citation_author=P. Cureton;citation_author=P. Groenevelt;citation_author=R. McBride;citation_title=Landfill Leachate Recirculation: Effects on Vegetation Vigor and Clay Surface Cover Infiltration;citation_pages=17-24;citation_volume=20;citation_year=1991;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Y. Dai;citation_author=I. Ruczinski;citation_author=M. LeBlanc;citation_author=C. Kooperberg;citation_title=Imputation methods to improve inference in SNP association studies.;citation_pages=690-702;citation_volume=30;citation_year=2006;citation_issue=8;citation_pmid=16986162;citation_doi=10.1002/gepi.20180" />
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenergy;citation_author=B. S. Dien;citation_author=H.-J. G. Jung;citation_author=K. P. Vogel;citation_author=M. D. Casler;citation_author=J. F. Lamb;citation_title=Chemical composition and response to dilute-acid pretreatment and enzymatic saccharification of alfalfa, reed canarygrass, and switchgrass.;citation_pages=880-891;citation_volume=30;citation_year=2006;citation_doi=10.1016/j.biombioe.2006.02.004" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=L. Doove;citation_author=S. Van Buuren;citation_author=E. Dusseldorp;citation_title=Recursive partitioning for missing data imputation in the presence of interaction effects.;citation_pages=92-104;citation_volume=72;citation_year=2014;citation_doi=10.1016/j.csda.2013.10.025" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=A. Gelman;citation_author=T. E. Raghunathan;citation_title=Using conditional distributions for missing-data imputation.;citation_pages=268-269;citation_volume=3;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Practical issues in imputation-based association mapping.;citation_pages=e1000279-e1000279;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19057666;citation_doi=10.1371/journal.pgen.1000279" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Jaiswal;citation_author=J. Ni;citation_author=I. Yap;citation_author=D. Ware;citation_author=W. Spooner;citation_title=Gramene: a bird's eye view of cereal genomes;citation_pages=D717-D723;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381966;citation_doi=10.1093/nar/gkj154" />
<meta name="citation_reference" content="citation_journal_title=Biol. Invasions;citation_author=A. R. Jakubowski;citation_author=M. D. Casler;citation_author=R. D. Jackson;citation_title=Genetic evidence suggests a widespread distribution of native North American populations of reed canarygrass.;citation_pages=261-268;citation_volume=15;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=A. R. Jakubowski;citation_author=R. D. Jackson;citation_author=R. Johnson;citation_author=J. Hu;citation_author=M. D. Casler;citation_title=Genetic diversity and population structure of Eurasian populations of reed canarygrass: cytotypes, cultivars, and interspecific hybrids.;citation_pages=982-991;citation_volume=62;citation_year=2011;citation_doi=10.1071/CP11232" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Epidemiol.;citation_author=M. A. Klebanoff;citation_author=S. R. Cole;citation_title=Use of multiple imputation in the epidemiologic literature.;citation_volume=168;citation_year=2008;citation_pmid=18591202;citation_doi=10.1093/aje/kwn071" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=K.-H. Li;citation_author=T. E. Raghunathan;citation_author=D. B. Rubin;citation_title=Large-sample significance levels from multiply imputed data using moment-based statistics and an F reference distribution.;citation_pages=1065-1073;citation_volume=86;citation_year=1991;citation_doi=10.2307/2290525" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=J. McWilliam;citation_author=C. Neal-Smith;citation_title=Tetraploid and hexaploid chromosome races of Phalaris arundinacea L.;citation_pages=1-9;citation_volume=13;citation_year=1962;citation_doi=10.1071/AR9620001" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Olmstead;citation_author=M. D. Casler;citation_author=E. C. Brummer;citation_title=Genetic variability for biofuel traits in a circumglobal reed canarygrass collection.;citation_pages=524-531;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.01.0003" />
<meta name="citation_reference" content="citation_journal_title=Ind. Crops Prod.;citation_author=K. Pahkala;citation_author=M. Pihala;citation_title=Different plant parts as raw material for fuel and pulp production.;citation_pages=119-128;citation_volume=11;citation_year=2000;citation_doi=10.1016/S0926-6690(99)00050-3" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Pasaniuc;citation_author=N. Rohland;citation_author=P. J. McLaren;citation_author=K. Garimella;citation_author=N. Zaitlen;citation_title=Extremely low-coverage sequencing and imputation increases power for genome-wide association studies.;citation_pages=631-635;citation_volume=44;citation_year=2012;citation_issue=6;citation_pmid=22610117;citation_doi=10.1038/ng.2283" />
<meta name="citation_reference" content="citation_journal_title=Bioresource Technology;citation_journal_abbrev=Bioresource Technology;citation_author=C. R. Picard;citation_author=L. H. Fraser;citation_author=D. Steer;citation_title=The interacting effects of temperature and plant community type on nutrient removal in wetland microcosms.;citation_pages=1039-1047;citation_volume=96;citation_year=2005;citation_issue=9;citation_pmid=15668200;citation_doi=10.1016/j.biortech.2004.09.007" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Research;citation_author=D. L. Price;citation_author=M. D. Casler;citation_title=Divergent selection for secondary traits in upland tetraploid switchgrass and effects on sward biomass yield.;citation_pages=329-337;citation_volume=7;citation_year=2014;citation_doi=10.1007/s12155-013-9374-8" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=A. Quintanar;citation_author=S. Castroviejo;citation_author=P. CatalÃ¡n;citation_title=Phylogeny of the tribe Aveneae (Pooideae, Poaceae) inferred from plastid trnT-F and nuclear ITS sequences;citation_pages=1554-1569;citation_volume=94;citation_year=2007;citation_issue=9;citation_pmid=21636521;citation_doi=10.3732/ajb.94.9.1554" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Soil Water Conserv.;citation_author=J. Rice;citation_author=B. Pinkerton;citation_title=Reed canarygrass survival under cyclic inundation.;citation_pages=132-135;citation_volume=48;citation_year=1993" />
<meta name="citation_reference" content="citation_title=Multiple imputation for nonresponse in surveys;citation_author=D. B. Rubin;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=D. B. Rubin;citation_title=Multiple imputation after 18+ years.;citation_pages=473-489;citation_volume=91;citation_year=1996;citation_doi=10.2307/2291635" />
<meta name="citation_reference" content="citation_title=Statistical analysis with missing data;citation_author=D. B. Rubin;citation_author=R. J. Little;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=D. B. Rubin;citation_author=N. Schenker;citation_title=Multiple imputation for interval estimation from simple random samples with ignorable nonresponse.;citation_pages=366-374;citation_volume=81;citation_year=1986;citation_doi=10.2307/2289225" />
<meta name="citation_reference" content="citation_journal_title=Bioresour. Technol.;citation_author=M. Sanderson;citation_author=R. Reed;citation_author=S. McLaughlin;citation_author=S. Wullschleger;citation_author=B. Conger;citation_title=Switchgrass as a sustainable bioenergy crop.;citation_pages=83-93;citation_volume=56;citation_year=1996;citation_doi=10.1016/0960-8524(95)00176-X" />
<meta name="citation_reference" content="citation_title=Analysis of incomplete multivariate data;citation_author=J. L. Schafer;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Servin;citation_author=M. Stephens;citation_title=Imputation-based analysis of association studies: candidate regions and quantitative traits.;citation_pages=e114-e114;citation_volume=3;citation_year=2007;citation_issue=7;citation_pmid=17676998;citation_doi=10.1371/journal.pgen.0030114" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Shenk;citation_author=M. Westerhaus;citation_title=Population Definition, Sample Selection, and Calibration Procedures for Near Infrared Reflectance Spectroscopy;citation_pages=469-474;citation_volume=31;citation_year=1991;citation_issue=2;citation_doi=10.2135/cropsci1991.0011183X003100020049x" />
<meta name="citation_reference" content="citation_journal_title=BMJ;citation_author=J. A. Sterne;citation_author=I. R. White;citation_author=J. B. Carlin;citation_author=M. Spratt;citation_author=P. Royston;citation_title=Multiple imputation for missing data in epidemiological and clinical research: potential and pitfalls.;citation_volume=338;citation_year=2009;citation_pmid=19564179;citation_doi=10.1136/bmj.b2393" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=Grass Forage Sci.;citation_author=J. Tilley;citation_author=R. Terry;citation_title=A twoâstage technique for the in vitro digestion of forage crops.;citation_pages=104-111;citation_volume=18;citation_year=1963;citation_doi=10.1111/j.1365-2494.1963.tb00335.x" />
<meta name="citation_reference" content="citation_title=Biomass as feedstock for a bioenergy and bioproducts industry: the technical feasibility of a billion-ton annual supply;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Statistical Methods in Medical Research;citation_journal_abbrev=Stat Methods Med Res;citation_author=S. Van Buuren;citation_title=Multiple imputation of discrete and continuous data by fully conditional specification;citation_pages=219-242;citation_volume=16;citation_year=2007;citation_issue=3;citation_pmid=17621469;citation_doi=10.1177/0962280206074463" />
<meta name="citation_reference" content="citation_title=Flexible imputation of missing data;citation_author=S. Van Buuren;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Software;citation_author=S. Van Buuren;citation_author=K. Groothuis-Oudshoorn;citation_title=MICE: Multivariate imputation by chained equations in R.;citation_pages=1-67;citation_volume=45;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Comput. Simul.;citation_author=S. Van Buuren;citation_author=J. P. Brand;citation_author=C. G. M. Groothuis-Oudshoorn;citation_author=D. B. Rubin;citation_title=Fully conditional specification in multivariate imputation.;citation_pages=1049-1064;citation_volume=76;citation_year=2006;citation_doi=10.1080/10629360600810434" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=J. Van Keulen;citation_author=B. A. Young;citation_title=Evaluation of Acid-Insoluble Ash as a Natural Marker in Ruminant Digestibility Studies;citation_pages=282-287;citation_volume=44;citation_year=1977;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=K. P. Vogel;citation_author=H. J. G. Jung;citation_title=Genetic modification of herbaceous plants for feed and fuel.;citation_pages=15-49;citation_volume=20;citation_year=2001;citation_doi=10.1016/S0735-2689(01)80011-3" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Research;citation_author=K. P. Vogel;citation_author=B. S. Dien;citation_author=H. G. Jung;citation_author=M. D. Casler;citation_author=S. D. Masterson;citation_title=Quantifying actual and theoretical ethanol yields for switchgrass strains using NIRS analyses.;citation_pages=96-110;citation_volume=4;citation_year=2011;citation_doi=10.1007/s12155-010-9104-4" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=S. N. Wood;citation_title=Thin-plate regression splines.;citation_pages=95-114;citation_volume=65;citation_year=2003;citation_doi=10.1111/1467-9868.00374" />
<meta name="citation_reference" content="citation_journal_title=Acta Agricultur. Scand. B Plant Soil Sci.;citation_author=C. Wrobel;citation_author=B. E. Coulman;citation_author=D. L. Smith;citation_title=The potential use of reed canarygrass (Phalaris arundinacea L.) as a biofuel crop.;citation_pages=1-18;citation_volume=59;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Zheng;citation_author=Y. Li;citation_author=G. R. Abecasis;citation_author=P. Scheet;citation_title=A comparison of approaches to account for uncertainty in analysis of imputed genotypes.;citation_pages=102-110;citation_volume=35;citation_year=2011;citation_issue=2;citation_pmid=21254217;citation_doi=10.1002/gepi.20552" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_journal_abbrev=The Plant Genome;citation_author=C. Zhu;citation_author=M. Gore;citation_author=E. S. Buckler;citation_author=J. Yu;citation_title=Status and Prospects of Association Mapping in Plants;citation_pages=5-20;citation_volume=1;citation_year=2008;citation_issue=1;citation_doi=10.3835/plantgenome2008.02.0089" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping by sequencing allows for large-scale genetic analyses in plant species with no reference genome, but sets the challenge of sound inference in presence of uncertain genotypes. We report an imputation-based genome-wide association study (GWAS) in reed canarygrass ( Phalaris arundinacea L., Phalaris caesia Nees), a cool-season grass species with potential as a biofuel crop. Our study involved two linkage populations and an association panel of 590 reed canarygrass genotypes. Plants were assayed for up to 5228 single nucleotide polymorphism markers and 35 traits. The genotypic markers were derived from low-depth sequencing with 78% missing data on average. To soundly infer marker-trait associations, multiple imputation (MI) was used: several imputes of the marker data were generated to reflect imputation uncertainty and association tests were performed on marker effects across imputes. A total of nine significant markers were identified, three of which showed significant homology with the Brachypodium dystachion genome. Because no physical map of the reed canarygrass genome was available, imputation was conducted using classification trees. In general, MI showed good consistency with the complete-case analysis and adequate control over imputation uncertainty. A gain in significance of marker effects was achieved through MI, but only for rare cases when missing data were &lt;45%. In addition to providing insight into the genetic basis of important traits in reed canarygrass, this study presents one of the first applications of MI to genome-wide analyses and provides useful guidelines for conducting GWAS based on genotyping-by-sequencing data." />
<meta name="og-title" property="og:title" content="Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/891" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping by sequencing allows for large-scale genetic analyses in plant species with no reference genome, but sets the challenge of sound inference in presence of uncertain genotypes. We report an imputation-based genome-wide association study (GWAS) in reed canarygrass ( Phalaris arundinacea L., Phalaris caesia Nees), a cool-season grass species with potential as a biofuel crop. Our study involved two linkage populations and an association panel of 590 reed canarygrass genotypes. Plants were assayed for up to 5228 single nucleotide polymorphism markers and 35 traits. The genotypic markers were derived from low-depth sequencing with 78% missing data on average. To soundly infer marker-trait associations, multiple imputation (MI) was used: several imputes of the marker data were generated to reflect imputation uncertainty and association tests were performed on marker effects across imputes. A total of nine significant markers were identified, three of which showed significant homology with the Brachypodium dystachion genome. Because no physical map of the reed canarygrass genome was available, imputation was conducted using classification trees. In general, MI showed good consistency with the complete-case analysis and adequate control over imputation uncertainty. A gain in significance of marker effects was achieved through MI, but only for rare cases when missing data were &lt;45%. In addition to providing insight into the genetic basis of important traits in reed canarygrass, this study presents one of the first applications of MI to genome-wide analyses and provides useful guidelines for conducting GWAS based on genotyping-by-sequencing data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SUByigYblGNI3FaBTlZcX3QWLoWiaUb4CznYbXtjCAY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459905","apath":"\/ggg\/5\/5\/891.atom","pisa":"ggg;5\/5\/891","ac":{"\/ggg\/5\/5\/891.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/891.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/891"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/891"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/891.abstract":true,"\/content\/5\/5\/891":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AIu7WcQd0T9F5OP5q8+H7dSd00lapdNfJEdJucgl5cIXCXc8L2pwWmkPD0a5i\\\\\\\/paDqvfaHGSrnMrPiGToPQZ2tpeA79aJaBn1+XDZHVJczl\\\\\\\/tbHV20lufQ3K5bq2J8oe9unm4WUVX\\\\\\\/xgb27q0fRW5VSF9nwcd9rVW7j3Udll0BXH4PzCzqsAGZcD9znw5GCK4mAmJU\\\\\\\/Zmok1wrlIdY2uGwpsI17BzUFBL9Sh3GER96R\\\\\\\/BXUWNl9LE5aroMC1jyxQg+8zIhWKktSCNKAmA3aACemVkFyJWD0REFQONlbYO7Fx5UkmJ3Vv4rU9V1e8+HuC6DlUx8G8mm1f30NjmMlPR8Lb8i1SMtJqliZXKVmJyfkNV0d596fuIoEXcizRa8zn4hbNWp8xnsXxEsv\\\\\\\/cYIEOb8gdHLkG8jX+xnn0iMjsnVANItDF7i4cVKeIMjM+4XGYs2+4unMKo8ncJnp56GEXhU6H92dhQBbob\\\\\\\/SEMYo39BWLT5IYVSSbhekUVDE+16bOisXtUd4jpn6J6KpRxRi8J7a8h\\\\\\\/jgNYFVreUU6ocffMS1kObSqaVWseOgxPPMqvDxuv6jj4uus4zTskcbbl3Ak7yTdInSyVOWaFaWN2f9X5F+c2Jt0+Cp5mczcC9V\\\\\\\/ZIG5TvZWgfWcL+iWHf\\\\\\\/3KQmIrVyaQ4hwW2huPPOGNNmc8MzgXF7l2ZUTnX+Leu8yiD+Hmf5uaRN4oMIvomX85bZVCOr0u63FH6d1Re6HXIBSVZRVWgkbk7Gru1cNwtXJOrzBh6J3LCHEYnAxUai1TSsdMM086MWYZ6jPJXz1gJW3sbThW+hHwqyvHiTjTkusx1+hUNsryaR+5UMvLEc8PWVAIiihxEklMaG9OtT0h\\\\\\\/4WA\\\\\\\/HR3v4uRJ3ad9joUQoq3y0\\\\\\\/FqYoZog9n9MR6OlG\\\\\\\/NPZm3Ymr1zNDhDpq0HlBjSm9hw\\\\\\\/EH6ZDI8CXoGWgtEj2cw6j6gH1D7dn8mAMAPRE9jB1Ow++vWaCT42OT8JkFz1NAOeeocxQN9thktVJV52L22B5YGUDM8LfdwLb86oFrV589Bazuy8cF4ui1+DxqopaEfSuVJkqwZVpgY0KTa9ySnKtz\\\\\\\/29z4cXPjXRa+Yqhh9D6qJhNq8YLcksQGBakbrIaPL70fPjcPNMF54ctMURq2JghGSWUYfxo+taz84HK7UBBQgTo9Z19is7UUlfmx8GbxypWQaGDquzCZJ0K6gG2hU+27AoBZLziUx6LT1MpoxYNakhArk\\\\\\\/pm1Ayp\\\\\\\/isksmr8DnQzNja54FVn1zSbBaY\\\u0022,\\\u0022iv\\\u0022:\\\u002220HsAr4u2++Kq6y0wmeOJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ed929e8b8328060711e9daa70bde1399e0c4242196397bffc4da8ad6fce0313\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022d\\\\\\\/APm5yxKDB5HPNw4A9O7ZDHaSKVcfZMOiI1BuGQQy+XiwJh6qga\\\\\\\/qi4Ufir+wXSWJseSY6wjZZ1A7\\\\\\\/nN7yGoOFIh8E\\\\\\\/lyqeWJ6lazetxRF6HAPOqlg2b0X12p8CfWm0I+Dp+ezP5Msz2L4YLt\\\\\\\/1+9pSTgxMHrZQnRKHA2TuiFmbeByOm9uWh0uHbpNYsqgfoGMSyuhK4MyP\\\\\\\/sMJiFFRHkN43aRF6xOlIoEksVpC\\\\\\\/rgH8l+OrVUZXQT0G9tPx4murhf5yQFSJdem9c7BdmAHGLviT1xRDEX2efmh3GEIyFbp0f\\\\\\\/xhZS4rDdI1J6Mgpbha0avVzksvmoedXJr1gbg7Dt8BmhTey0MjYRKdJlzLSZZHFwZ2QhugY0hkIPQfzuS0mdRHTqcW+76QeX1qCWpOIceLeJao5MsW\\\\\\\/yD5Q27Mo729qgRVbFF+HOYUgnkl+TA23aC5MosasLSppTGfEIwZgXhN9oFCqyeYhl2xgYv2uVqRvGC1T5SgX09c9JRTv3F3LkmkIbx7iOMVftkA5xOWc3VC\\\\\\\/S\\\\\\\/DxcBoISiHaX9RsrCrGjsBRnaTdtG+joVoprJG7axzeM4JlKHws8+wu\\\\\\\/y\\\\\\\/JHKdWlJKfRsVzYFcLBS4Dd94adLzI10NddUb6Vn5bA2TxFg4+nbj+wHBhkrrS1Al0\\\\\\\/r0H\\\\\\\/M3SnoRdCDaULNWVXw7E6C7ZoPye7ObKjIHrXbxpJC2\\\\\\\/AMnEyO1XVmpUqmDfPX+RWsk+o\\\\\\\/kOradvaSPcXXzdgioZOPjqAekJ5jr\\\\\\\/idFVhvxOIRSdLjjMwj2Z52G+pXedKt5OAaoZ6CO5Jupir1rJ+yeOjbL6cCU1wB9Oj4ZmxayknvVFFLic7gvR7L+hLeQvACAaCx\\\\\\\/+glDgt6xNNMPefSnSekAtoMez9CyEVHyAQSEIFisVWHSIn5\\\\\\\/snwI2VebyPrZKF\\\\\\\/JeKBILcYVQ25yibLRm2rjeY7uTcVbbtUTAuyCh55Hw0XUXYQT4IoHBauvVP2rMfHfXQ8uevCjOp7X9iRSUxREB\\\\\\\/OEwKllueeU\\\\\\\/z5nndvARQ4us7jyKROHXpI3Mof0rbzUg7wcSncUPRGMWfxq+5DXbJb5HyaA3MLPGi76VAEyLQEOKNY3PpYPGVeEjA0RyZOiF\\\\\\\/fFnP9cAHXg8Fq89+zv3dj0\\\\\\\/tNZdAxWyu5OwPe6SCcpcIxfueXaSWI8IDUcnw\\\\\\\/UMNBxA2AmTCybojwe9ndfR3nlB9Y6lG9BG7FMyjCd6yBAGagyePQkrLg16fmwTy6tC7l7tAm\\\\\\\/A50iVEW+YK+6VAnfuSmxPIJ16Ofsxnnn+VZrbGsXuenxpFW5NWfRP9zEu7Mhnk6rLqyoI9tJwwQghOwrKBsWAGLeKxX+vzWC873Q4nhK9hAniH8jEof82fPDLBQm4Q5JkpsicyDqWXE\\\\\\\/e8bc42yuQwTzm2hMCP2A1eIsc9KbIyJbfpNkkofIAvd1PKOn8v3XhKVzriBAzRDXIfYrOyMnWVmlr\\\\\\\/\\\\\\\/l8FkNfjuggGiVAUJLh4QGINzz+MhUOaI25xgJPl9A+gT\\\\\\\/9d2fjOVhsoXNkKuWyiP5e5KOgDPSKGyubEgrxM32lVE4mcDu6mWZoOX6xWM8Y\\\\\\\/0qppzthJa8ZciMLEaEe+Bgbt6XyiAn+4vc7tlwEYFlS+iZSNKVydnIQFSQD4T\\\\\\\/Pn\\\\\\\/cfXW5\\\\\\\/mcK8VfwprAFtZaEgMfZ8UTbSin3HF8y9569c3rwTLlsMfNGynAhBtCJkZ62Q2c\\\\\\\/sP8IjIoihU+07LNqcczBvEj0\\\\\\\/bFrc5+OCWNJnezqBzIzcRSGDsNKVxNPH\\\\\\\/NVeX3ZNJ5li2d+8zpCRnUN9lhUHPDrTvl6njgQ8BGdPzhO+RXP4DyVKkuEwHYOmvo24U3A93l\\\\\\\/TPrnYZ9prXuGENvwPsUh620vLmyIYPH5xD+T7sWN5tAVrpGX7Zi\\\\\\\/lDCs9qKip5v4aefEQtdiMJDiMTbYy4hAho1Tn\\\\\\\/5SXhCBv05LKf+aGsRmGMEYCsmq2pZ5BzSQkb1k2QA6NQmWllnNCxtIV5vSNRzHI10lpFjv6lBTyKVQHvdOOF+vRfnNgyWV+vOeaqYuEpagFAmv8AgXk0Uro0GLlpLzhsKgi++BRvs+H51M8wOllXiU9nmaEod67n1ZJ+KTz3+mYSljBCPBjFz5wKG0Jtkatymes+mv2pwFeYXZ+2xMuIQiOoa6kkwVvnGvUFdNqvp3HRQQFBM+0zBs3iOyBxoriSGrnFTgZfpW+apl+jOE8LOfVecx\\\\\\\/z2+NrbNMLjL06mTBbDNkqK+s3ip88jN9d65H0nPcdlVnJqahb1cRmrYG1YxHR35bHfvV2fF6e3WeZdORqTV5rfiJRvAVr7YYl30Gj+UQJWOvGZbNSkZfGh39PajxqeRQdouqul7x6O5GVnvQNQerqThmC1+9nnds\\\\\\\/D5GaJK7fGdqIDv3\\\\\\\/bWgHSwbE1DjjCfqLhZR1c5yrsz1elQUWYPqOH+lJGyNK2cEp5\\\\\\\/5mgAAZT0XW9AKZUkb7O3Khnk6guTBZO4IujglE4Yub\\\\\\\/3LWCNhWA+MdqZEZB\\\\\\\/Igy+ryUO8DXrpeIvNJx8yLcprGtWpkBDkipeWjYmyn0DFGxvWN3NoPWXT6ohIFws3VnIpcrKJLKkKdOlAckhL3Vuem7dQiA4mkICK5BB1q7Zw6G91JB5R0ACf4KJRsgdgB7Lp8d3ef8T2+Ocy9BY4jPMmsLeeV8gRVr4UAghFuV7JlHFpRB38WcCULoOgLUKf0CIaupNwaNqX9FQvlVqDM9yYzF5zxt8czD\\\\\\\/liZjoK7uloju6k2WPIj0BgOrDYH7Crdkw1NKk4A3qcU\\\\\\\/nhZQYcc7Mqh\\\\\\\/pFAyTdExpHEKtDnxngQROmAnkE7MnQiN59xZJJcg2W+KceUo6b0UANO0IgFUyhZZrUwjVMQSzwhBEjEL6nwvDKWFBMJvAWV2zlxoB\\\\\\\/RSmmT8\\\\\\\/w9KEd0VpcZZ\\\\\\\/eBMwpx3Dh+w\\\\\\\/PAybbiLON55SzJnEtHlYEt+T5\\\\\\\/TiNgbMiFEOb+tbKhetiQo0SKL34JrP73UHrqreeVblrlKwmQaX4NOd0HvU11meZclS4\\\\\\\/PAn56r6RiiyTU1mn1GI9wdQ+cWxW\\\\\\\/jAXzVHerIBR9py9wh5isFVuZYqT54sL4\\\\\\\/d\\\\\\\/07xb34T9i38Thz2+n0Pm1oV\\\\\\\/C8TrrUKU5314KlmXLaT\\\\\\\/nxKJafhx2IqHiGFep2QoJuBHCmJ\\\\\\\/L3EMsKWS3PVk8RAxuCxKhAen9siHbrwm5rPITQEM91yA6W\\\\\\\/8SEuNDTYQqbVDRoXYhzZtYKhp3reRaO00DT9u1Byxzm9nFPplIZJRTstPW3tjhORXSXxjkLdAQhhaKLuzgTp4yNEYqVWnqz3ErR3ipD6vTV8RoagRyoK4WL8ZNeCGhXb8p3P40gDnBupPAsAp4RvxRomHNFdxfDBSTF0Zl1QX0wiJRduJt6EMKTvrym+onil3ZGRdX1zTUjGp7mi4gQ4nHSlq2D4kVh4jj5hYUGFRvZ0OZZ9djsrxJ4OkfMv1ocln9hX7Lp21KQnNpUbIHKIIPWyOxiAhRZltdOvNwYb0cKbsZ4nTLN1BqZBJDChB6a1pskDkXnM6DycXiy4PCA9Umj2ESiIRJgpkoYfwCTm1LVU+vZv1OqK2nS+y\\\\\\\/lcqUdokmAlv\\\\\\\/RXm7PxrEbIFjbBR+LM7DegZTGNXdUduTtiDQGTyMl5ZphXTiV6SrEm1T\\\\\\\/VzQz+05tCwQsVI7SuyuOuYOcE3DiuHyFTXjS0jytUdGEdC51DO4nFCk00s4YPyTE9UKzGYmTtMoVlyrXgMvTL6WmODbJuqu6tIjhLNk7P00Hj\\\\\\\/Jksu79BOGt8sQX8t7KPnrXLVrRPbGO\\\\\\\/OFjzefYiIMSrMQidQfrk6lGexqHt2RZPtQ4Ylw56H2ci2dDR1d3CdL3R40\\\\\\\/SSbQfyseXa8P1z8foeouMu5eeV6Vdi2Xmoie2gyueghD5Q\\\\\\\/rINpMJP7IHwJmHgbO7svci5v9PZntrtFoNEBPmuKMTnI7h8goqXyHGEpocRI6m1mHdiopEMmjL\\\\\\\/Cq8et6FpNw73yXPSrYW7xSGB49zer9Qju6ank4MQfH1J9598C8yCbfnKqVLtioG2FHY9I0d4D3itDZ6j1uK5wvs86xSuSi7sWyoVDmagv\\\\\\\/Gfn4PbGRG0crJGBZTJbNWyqQJ3hwz9F5ZBYMOLK4G1DkKfA885tSsIfqsQby4V+\\\\\\\/8d4RIRBempvDlCwpOSH\\\\\\\/6RVNdBjKeXHLdwkaG7xs6lC\\\\\\\/Qd8lmL3Ru6C9LP4oGsWNGXw6DXxAt8KgRNwhuXIUwYMNXJw+JhXD0IusPbkcqMbqzj54G3dw7AeyBcEYk+OyBft+V\\\\\\\/col5XficPN\\\\\\\/8duUPE01usLSz1SCwQdzV8fe2Nxjz+oo9dgQzHKe5lGYxWlRa88TQOKgD3gVfSvF2JpwlrOyOVPwAsn140tl8a2R+EnPEH0elSZfgtYJd72NPoYKGx6Nj7sPTZkEjehranKNgNiI+tMnC7Fc90cBBE3qYl+8vqiRGqmLbpK\\\\\\\/ugRAB+NBtXWlrc4ffBEzVtl6f5yhrgFoROPAo5UIX4qF7P\\\\\\\/aKXopuJ3U60fzs3513w8bR0XEG3bBSInBwZ5gofp4BgzCubR6oa2KVu0pH3ix55R5SRjd4HUExphZcyI+h3sF8u48cpLWWUuKQtDPb8VtyPwXbrglLdVMfOeDCzUj9EfveoA0al8eNdW+oYcGmoYw7hNBOtfK5RE50KzXBAajcGn30UJQASIgmlFx7fx+BQ08F2hlkInyush\\\\\\\/PA+0RJflJQUohyVswVnGN7VTUdVlhr75rxrMM3KAvfrn+tqE6BH2V3Ed4+A5xPVB2CM5b8AcdiI5oTMBdrtGBHgOlLkElapdk6wBYeRJIu7HoWTHFlcfUunD+zF0jq46GjXkrLVIPI9tWQn1lNCD\\\\\\\/1xF4EsMmjmMPa5KVdl5uLhLbSYLdt7pbllSz1uzfwuySc3GZOtdML5AGCW0SXpZV1zUdjECpKb2W2aExuT9MRpX3VK\\\\\\\/U3mzs0pVY3BYLOq3t3TSi3ylLJwna\\\\\\\/GsOAsjBezj21wtblXAEN5Ts4+Awo9\\\\\\\/bSi13EigCGWaFHMYZ4H9rx\\\\\\\/D57nEEaJPH6zHFxvLJsBYFqVWFEVVRExcwCa3joIFRCBpJcig5UxTIzEKNqKJtNZlfU3MUG+x6mxveKLGQ4qC2YB9BNjKF5pygKLfXRkqlzWfMnyu+iCq1dBZCbd3O4A5XVLxnL9URswmW6bDPhm+fQukP3zcrS+aa2L3DPNHiXQnNeZ0KfVdEMKMZ7EWG1RXPZBK8ut6AE4tMcD+vC0lQmKHYdyvK\\\\\\\/9L7bxc4vOx5NpHTKbfqM3dU2Rokmyb4hS2mAQqn+ZMRWuQp+khkiGFJ2a1NpehonnzThA585JSuTmtN\\\\\\\/qVTA3NA1BqJs4l\\\\\\\/DYWgBMnlPBCItTeJ8PaAfm0rPacgqU\\\\\\\/+qbNBN5y9XsLNj74U7Dl\\\\\\\/Okkw1IUGxqv2TSrGvZeUXhZglbJvHJCNSxIYzD2GFioMKzj6v4TzirWrejWnXRyghC2eOWxtEGU0AtGy4B4nvONWUigZCNgTg7wcWPWW\\\\\\\/dSRCcVrLNqtyz8qGo377Aw4qyhDIIDYBFbTA9Uwy8ihbaRfIU+ab34vgGiOhnFahUaUVb8FY2DtzbTnn3zJ0tbil42wYKBZjP8EwJQ\\\\\\\/YPUYuPheQk0xXp9J5C9E3iWFTbJXZshSm25Oa\\\\\\\/Od+FhJL2xgjCLpE1gm1mhw67hfRFxL+23wLPd52aO5pe8d7dMd8U5xQiiotwJynSu3TjoUr+r81IV689dRQ\\\\\\\/+LoxlC0I3XEzniAcr4WQNgfZiU0VakskCnVi7Raec7hb8B4pp5r4rATo8G8UXUBg+zJKXDa2wmMYJn1mSkVR7k1208wUCTHOWN\\\\\\\/roGLseNwffu62oGZmFfUkfILkXEr1yxXap+AThRwLkIix4M5w5r7xhQ4cd2d\\\\\\\/D9t0hwSlkC0RP+e7nNX3Ku5dfnh9pbLzMeQG5KpzmDdk8SUkVCdE\\\\\\\/NU37KuSoKjvR0TwcQk61JNs+G6zEmwUcfI0ZOE\\\\\\\/6MaMiTv5aoQSi0dseA7ahQ\\\\\\\/pwE5oLTJHjEV+FAN9sJgN497rUg2LP8z7HK80dSsb1ZOcwKpbk5nb2ooe5ehXk+a+C5hvAlZ9rx3OSl5tQJbJsFXBQSmjA49PkMVvQl\\\\\\\/GboasnV8QKjfrSL8xD15XcuyGmysAjIRCiNBdfDZdyCXX3AY0MKOboRECDUTJJ+W0rzywiHnHvEi1WrOrllBvvHuM0DfkpYKdJnmfJ01Z+XL6Q4EQjVtV\\\\\\\/WL9ji8l7enlaK4GE\\\\\\\/Jo1HM+JR+XRyWXehlK5w2Pg3WCKALuFR\\\\\\\/b8d\\\\\\\/Usjwja8NMzSuFGhkR0EHK3yN3pxlrldEJnpQ7dzHw481yVCthEb4i7DtQ5A8D\\\\\\\/gwsXYiMH\\\\\\\/V6pdKmTL2o4SNt+6ptROqqob6woRqv48yvhThnwuyGJKvpNuGsL6LdrB24Ko0XrT5K6S2IIJtvC\\\\\\\/FU\\\\\\\/kCRYiif\\\\\\\/DJHk8HtBNr5+8X\\\\\\\/yUNaCngP4b\\\\\\\/ebrejg7TQ+qZCZc9BzEImRJhuj0F\\\\\\\/PSolPyuQ8rUt9uhs+cbedA\\\\\\\/oNkXfcW+O+BfY6PyKPi7TDg0sgFcMEIXi2KXmPjYo3uG5gq6vqjfNRqafLbgDRGlrfFmXwGvp+YYqpf29eaJWAGkFz4oXquoIWQkrZYSSKUWD0hphVnSjI+k+HvFmUV0mCHjMJ5Y0g5ega69aPZVykA9mUVN0e+BPDhktuK62TWKdClRFrXWrWubDdk329U\\\\\\\/z3ROXcpZQXffqP5NhshsA6PFKlNZzDQPiDiZDJ6VpZjXIGkDrwBANKJ7gCDR3bbd50\\\\\\\/icNtTVcgi6m9TEH6C5VKlLEUGhYUQ6M556LrpZ1J4ppuoCN\\\\\\\/6bsVju7fVTIhJM9coJOUqkCyTi6hYJM80lHdPlXvQMf3xkWyxPqgPUBeGo5Rk4eqHfrKSWfYgGJxJWaSecRhKgdVnyaxwKNOj8IAsN3dtw63clXQMFWrHc8bdMuVdxd5nNmG5TOPrKCes1xksxIt4fd8\\\\\\\/+T8LIeDTmeR1c3mz3ybgok8PZOTKZiJBXwcd\\\\\\\/aafY1jgUtwAI6tSycKqDyw1OipVl\\\\\\\/Xs20xnIGS0LA8OXVHwQceDdmCUl5Bt5gHhfISEtZSueQptVCcc0vxGhqsIu1odhiUqUX7qavLhL7O11jdCRjZw6klBsrPgHwry\\\\\\\/1FJObm2bTUrQ\\\\\\\/oLAu9aDcaUAxs\\\\\\\/z6d9fr4v1Q91ZCpVTC2Ddlr+k0jHvWgbY1pxCN6MaeqhC4zWR+\\\\\\\/7EcznlEVKYEl\\\\\\\/RexNJ2uWMm+KsnRFBpTWhNZCfgjtO5Z19LE2gfTzFuCUXofi8e2OtS\\\\\\\/7t8SbjUd25QW7aez4j\\\\\\\/NmepA4mBU7FBs6UvKkZEoqVGtqZuUsw\\\\\\\/NwH+Dzby1CcHV84RTYQmHPEnZwkAWScNSky0lj40WoNj6JXIPL8Z2toaiWJWsxH3u7aI45thZ06w2c1KizOZnKpcso31iJBfGuIarCMkX38634QaHUkkckIwJXD0ZZcu5O9GWEUHacgLGXmYNNJzXs1rxpaNUoBLVVOjrbawNS2KZkvTyOeL7oM6CxxcodC3zqSsVW6nO+hX+ZYz83WpCzR6Dkz8vKnVU+yvcw087XYGEEWnpotvITdgGlCRnpHYVny6d4Tk2zIQjj4Z2VuVR5quXrDakA2JXgRAu\\\\\\\/2Rsnp5XTdJKq\\\\\\\/n1C23bg51fmrjhNvlwsTWBcOovpn0zkxLVdjCrE1dY76FhifGNBgjnR58YS3kyECItTd8ySiVZkZs\\\\\\\/Znx4wOzfuAdnT9zxYrsrNUd\\\\\\\/Tbt7lE9xLBC\\\\\\\/lfN9reNstgKRIXu5b\\\\\\\/A8Jqr\\\\\\\/2ZT5A6sBFMB8pUKUcYQ8evjbQw7BjW6O3uOrxABSqA2bIB+Hd+zRyxAqv2O5r4oRVPiD0FY24jJcBD\\\\\\\/nWO0+hPwCrVnrvCqFZEbjsBZu52Pn8aq05m8K3h5wv9GzUzcauwyV5rbPgpDit0ed7ZjC+htPsQIMPWWOayM9qZuO5SCls4xnSJrJ2IWkI8cdioKIMhBObeat3Gimxd5xCn5QhuywJdcs5zRoDPNjGYF9KVS0xR6RTLDWmsVA5f05+7DLiJgFyVrtGHWElCIIpWKz+uPRCqjJEupzHBX9711RnGnqxnQro0Rk++hm5dY20gcORFuB0Z16JDFbmsEgaQgAUCxB9Lzk7oMqsb8awn7qwEejMuXUJeKLPNsH8hQ4ER4J\\\\\\\/lIYm565QbvQ5liGCC3f9+3BcEdyFKPKCvPYrgpKqXtbl4v\\\\\\\/bxD0Z15YLMh3QPOxM307AxDjIwrW53pkI8gj9ABg6iE0Zl6Enwb3HacrAxc5OIVmtXwTNmJ9xWX5k3otnJRze\\\\\\\/XcWayrXjjRuOYrF1cWGcmkh687sHsSjetMyB1OM7+sTV7ARZzI85idtyN4BaEMqcM9CEsfG\\\\\\\/SbIHJahZAsRLWrPBbhq8G6w+MqGUahFsIv3fT7Gy85u9zRqQiH7Wa6FMqDevqadEOGX5bH4zQnwx6Duz5L7i+IJ7PefhR8AWT8uwHf74t3yYkmLIoE2WDrhfLXrY3dKj80dppGtOWTOh3XdG7+YVWPEwx7aWxhKZfP8oCcMeCI+Tnq6et\\\\\\\/bgyCA5SZekHXm9L8GUrRIE4T7y4kmFb54aToQr2oJETJPEwAve+Ka\\\\\\\/vlkcVWqg+ZEdP4RrzGYW7s5YL8q+jngMtB0ocv2vWbHkPKREjSBojaKBsj37695rffMyjXdFAUA3Pu2euLrt7ekBGmhPEowocKkU3tVb12pYztPh5g7fZaYxF1FTM7OU3Kh16B6DmapjmSfuPuQvNFWxLd\\\\\\\/GnHaWmABz7qe8KkVMESHknU4Ps2Q1xHtMbKR1zB3L\\\\\\\/S3HA6kJZkyc6HTsM5l\\\\\\\/jPPSwHV0v8mkNDGvJoKJjIoBgn6XNYRWhlsYLon9O0tJmO4+RJqPVHAAZzuQYtA7xOifaAq8AD1u75kQFzdhHjnSODlRArWmVTh3TVGQYRgEvjh2uNMwUrZZXZdFNgb2t3t8CAy0uqKIMV0pqgHjnVuIXynmzmBL7d1PfFFgfFbgxL4vm3KWJOtmDFrcsA3z8k3AtChwbPNlzxRGeciJUdBQvsLJPNXeWkEkRwR\\\\\\\/H7+Xqid0mfPtGPKyMgVT1ou+xIITOLQCQRZt81ldE7bF7FDayqeGRSnSgYFMq38S9n1A3B1uxErZMnNpBNHUuTRs73N2KXXU+h1fqIZWi4EokY6zryb78FIOQn0W6Yyac7xG5NEDqAATj2t4F3TXfvLFPDUGmL03qmRvqJtQNAHV8o8MU0Pop2+v4E68wF9IATxhJ1426SlkQQmXjPoxcp2LdnM5clYTD44E55R2Jb69Mk1L32hT1QQ5tUbN1saxISGaGMnyg5pqxL9m0czjWVkixJ3N9geeionSHTBD24U\\\\\\\/50dFKMnEOqJBGzuGe1HZe3PAgw8A3Eh2FeJSvfU\\\\\\\/81D6GQ7WE6gu0+A6EdOKwdnmCWcR56co70K\\\\\\\/ZzZqArcg3rlYBZvXXCQSUeSWk5wIUfcE\\\\\\\/5JhJDfPDQEF\\\\\\\/UcK+sEUOmcfx\\\\\\\/pUPkXYJt+t3MuK+meZUawQvEaLqpq4R78y12KnRDWDZ75nmVHj96o11dKMvIcyrUriLtlIaXh4qqwoDGTnw6xJDxllem6YSk2oLbG4gII3Pn1nnE1rGW+Pmw6UmQzlTh9hZWyU+AFBD8bgQAjHGm4Tk9yqOMmycfQYeL5BVNhlFRN3Aup7rcuaPodplwpNOxWHpJOc21lWOdeIk3ther8rmggqb7abQwXkdSGvIVQrYq5klmblFtP5KMOQWS3HFM3rxZnVyKirmvfat2ShVdKl+vXeLEfRmm92+6H0+XI5JZYCncFAsjMMPESd9QaAHANTGV7w3blYBBnKfyXmhTb2kpzWM+m7hIBJYEJpN1+LtOQvI3ca6CXR8a9Q6FDNFjV46uOQUku2sunCXNVj+Mdi3Ye9bpWy6XFbmCPAzkhevO+woRjqiwoR\\\\\\\/IszIlTI2d8UByA8M0A0N11E6GmohBlrR2B4ss\\\\\\\/1weq7ZOb9vQXIKPFEbmBiXa3pGC3+BM+bIGwR9nHjaewA2SyKivKHd\\\\\\\/sKHenQfx46jBi34m4IhcWOJlmQIaV77KIBoi\\\\\\\/3nsAkhJMY+KAZnpxyOo6NcAXzyJd02smf42xhA9cyjTwkq3w6NU5nEB3IiI\\\\\\\/tUbON0IdxIRzwFIlZr6fENsQgNZBN\\\\\\\/eM8CjmeOxOQ6XFC\\\\\\\/rvou20OwMztvyk0t\\\\\\\/pEgWsGoxU2kgXnadblYGpwKLPVdsSUIEmI4gCHUJPVJWYbnSec2ydR0ddsKw8gzzAwt70qlQe5E81a7pmHelUcAMvLVA\\\\\\\/mmuyWHoWQrzy8PnurTCQ8gX6TVVlCxfwBxbDea0TMKs6G9ChFVWK3JAs8+VVI5ERC3WpWUiaV\\\\\\\/mSygbVzSM8Pql03B\\\\\\\/32+OV2bc30hBJiBnQyYSEjN+iKw0XFyZNcc2joxcAqPRHZnkenhtC3Bhey2X3MS8SCIhoA4NKlcQ1FDvzCwV8KyZUlHK0mIxtN8QLgdgBqP9VLEIczKtk67R2oXa7qreXhrbJsKIRa1K0vQDXAxgQb9P6gjUVbd8mBJMjiaV71teU6Pp3nprnKx1YaEGkA\\\\\\\/XFkxmQZGdS9sw7T+szWPReeVFDYwFjYaPJu9FUq4ApKSh9BgbCGgVwayYpNuTHLy8HlcL41+fulp\\\\\\\/uq\\\\\\\/7vLwUkjXqzrE51if3QvdPCfrfA+lAD3fB\\\\\\\/78BOsQPS7NbQ\\\\\\\/admsVMfG9RwCXW0m\\\\\\\/eq66D9Q+szps9JT6yYPIv+GTPF8uhqLWcGnPlllLfx2cVI2siWaaHV6rab152pbJg9BH5l+VuzvnLB6RbaHcrZM35PgC5L7pPFKf8UQ9\\\\\\\/kvKDg4bWhBoXbiHozAiEoZ95cFk7lPrPxlZdQZsTD+b6T2pxFmj5q2nCCYr6Y6L5AZhaPKqr+GVwWfcY\\\\\\\/Dr0b7Tof4dch5Wv+tDwn0xC4arAx59djrNShl\\\\\\\/Ht6BxPqXtZawEOwiBQaU5ujXseBuxlyc3eiPbdDYLrvDtd1jDn6g9REFxQkt80mxgc6JTqnzkgtoSWw2pNryegPOJMpk8\\\\\\\/lVxFYMMGzQum05Lvyhz85I48Be0L3I6r5I2eLqh2DFk5JiLghCQ8Zf1dLu0+nR38fy7jE4VBn2Zs6QJabgFoTHTTptpshDQ8Q3anbB5fCUjeMaXsJ0WwM+CgRjBYkKLi\\\\\\\/dZmDQbYTJTkA4VGGCsKKJtSmRurOXAg5\\\\\\\/ogSyQgvlc+mukZPKWQj+hxkWSO4uqDJLroYk8rvUFl5rzYl3qouMl1clu0PNswy6PrWkgaAjwtOjsRg4BXOGBZEAE1KcvUcO1B+MtDgCCLRNT9AwO2MrAvIvGvArcbtQZ762jDayttBu7w0YHxgrOBo+shse6uNEgFVt1VxP8caRUlWzi+PVlIU4mi8TBtqxRamhIsTUwBu6toCiqTRx5lxO6DeJrM1XUy1U3PMe3GAqNDATrvw\\\\\\\/5Du4BZlMQkLxoA4iTxoyXucyth2bQlrHbQEYrKC3JVs+D17v3Yj2GaUIkMbq7i5ycxQA4paY9W+kzmgcjb4gcG\\\\\\\/wVbTgEAqT4ow2v4WZ6V99f9g+sidr9aG2IYrp9wIfQhHWs5+\\\\\\\/LxRX+EYhMGn6EkM40JZQvxRd5XsDTFILmBVWO4UWrywdqF6Z04l3V1e90Kn+nDr1T2ghthzUmTcWigtUIa8WhQp\\\\\\\/KkZebJx2e9IQ+e07umS3ko8Fm63vBjGkPF1WujYSxaeLDroEmiHTErJs7fondt39fPdpwnRcDW09YqH2RM7uLyFMSH37TlQVALKJKq8xwlhpUkgjjQkiJUmh0rQg4c4iZxQMwgCa1dzwGn5dqW6bRot\\\\\\\/dwwFsIUX\\\\\\\/N2WnR4pvg8THCPynLnWt94dy18V3Ql81vt5vX3QxRv9KJUbVvMSHFkbxKtdOwzuVSP1OkNQiwSxoKlPAB+oGgH6pnMckt3hqwRxnMhDLmg5j1fYxIlkhHqUYdJEHMKKfh9gPbGtsTmM+KR+xDw2wzr6Qe8Oxnx4bACWrsTeyMfcETu805EJm2k98G\\\\\\\/n4WyyLX0Lqv7Hh7XggHahgwjiVXCL83cOETeHiE9awyX0RriBSA9ra+E1UN2hNc4\\\\\\\/Kd\\\\\\\/RSybQou3bEtI9fNhIhJ7sRAol90wrW1pfQScRQM672phinz0g5io0Kpl6MpW0QpISqLRvS5RrsrvDU7oeOF2YtoByHPoZuzBzL0sAgCWPm2KYAJKmBsK\\\\\\\/wa5eX\\\\\\\/FIVUzPESJ+GYp\\\\\\\/1pg6T1MTaYJIT2LMykmUEvObEqEDmWcU1vF1n+ja6sHlJa8ChHP+pIbvjAEYTEQGlHvie1z3sVn336bl2+VcLCUaMERUjLzoB9tbUQxkOF886mG9MPeCQe+0Su2dgJrex49Awflx1Uab9\\\\\\\/Eo+sYIAKvpJe4X4SYd8z\\\\\\\/0ZbdAft4ug1pgAfEgfhAN1C9+2Iu6eRtwIPCRenpkBoWsE5sV\\\\\\\/mx07k6JfYKO90Xg8gWbv8Gm7cMO9zYoHPBAL1mGyXefFMrsB7Qv3vqtc+WehsXWgFtM0Wd8ypQmGeB2q\\\\\\\/w9jccMOs3Mnt3AlAZPYDLV7YCnkjDgHOMcZeG1zdYzOmcZF93Loiq0LczPKjjNAtlw5ZrFyRl7oo9cywSBzmMqYEJjFCGtGNTZtk6UAoAPp9rebNcMJf6GXnT2jvwCue5h\\\\\\\/NyMqvpDqerfnxAoAO3k1Mkb3lPkACGSXalg+7A4x+AgMbIzrb7UcqCb0AUn++tbEr41zSiWa\\\\\\\/988wiPOeAwnnJ\\\\\\\/jhaGgkcoTy09aC2XcBrRzVzpTgZssm4hNqfYZ5Nf\\\\\\\/F31vbRunIBIiwAb9pU40h6dvB8MuXImAfyAWsp\\\\\\\/iwFUbyTCgf5umxhb48kj92HvNovxSFs9DM3X3V8rLdVQ4ej9aNP69A3QoBx5PPNhGG7Q8i2j7Ez+0XHcA1pQmHLoLoHI9dmm3YJp39GtoTQDreT5HNaFLCXy9T7z\\\\\\\/Z4eqPSrjXJybqXw5Cm5uSg25znHd+pcr+W8w8tLQves9OujJHZK9D1j7DD0tB57g4OQJcF\\\\\\\/PBU\\\\\\\/15aPIJt\\\\\\\/RGzTS2Vzfj\\\\\\\/seXQ\\\\\\\/d5VGkJpgCdd38BVv28cv\\\\\\\/cAaxy3J\\\\\\\/DX1ltQCyspb3XVV98OyrgbLO54ZHi825aT48AkwU6T4L0qRqTe0IH\\\\\\\/uP+sxA0lIYDnSqrO2FxkDRdyBETCuO8V\\\\\\\/uXW+si9lWW3hjlPwJA43TpM2ZcxOCMsBGAxZevqEo4orLi5SPbiAZ9Mt6gLy36R7zjhSMTuygQ8t4ueTC03Pn+X4A0keVsxFgnLU+3wNcdEaCch2R4EOASLREIxc2lHl3ZnaVeKCVGyNzw+ps5NAI2rbfmH+tZGC3TpMJZ26YAeml\\\\\\\/uqpDMJji39RBsx6cF4r2uIKODk3fOsRaPT9bRjyoohnhfjG4n8XbvBVRV8VlF\\\\\\\/tsUIFrqmyEeHulu79RpFfr+oOnX7UVnfnLTV0Y\\\\\\\/r8dW9H0H++GNDl5V6rPRA8AHej3p9YZW5h17+TT\\\\\\\/olEGqRdomBDEnTnP5ZOlLUWNcnMpjr+8kxKjH63zyNpxyKiRiPt1FHKCwAYjRzvFsHFFGUzDArd9xOBugTWvAkd6af4A2Q1NTcSf1gRzkOXC7ObPUm8Tc3UlUXjRHgNivDDh\\\\\\\/grJZBTLPfpRmOZh9LUNt1h\\\\\\\/VFm1gkgcXqC95wyTxVvaGOLhuzmegltknOfEpoZnjnUXQrWwfE3xiN54Ff5nrAwyne5O1XvAnik3NE\\\\\\\/dco7HipqxcB0+Sz8Xy1oPxcOVB9vprDblVir1ez+gTPnLZQAgi4ztmvA\\\\\\\/pp0yymUJd1BbLJL08HiJ6INND\\\\\\\/1myTxN0yieYZ8Al8BXPnGP3VM8dW9RkkZE9Edz7tryLYUlxQIe2fAFh43ip7mOuR4elJPc9nui\\\\\\\/SjzeB3RTPYkxSyUlRONMvDDMBcOfpIhG2C91PGxg2nGcYbvxsFDlj1XvWq9bveIhvsvipZpE7+X19hPh4gpVDr0m0vcvHUN5Ufxgm6EGAKrBxgK9L9DVXEo3BrYyDWEVxQN1iZnbXikss0HyMTSlHsfy9UTsROwX10dbtuIGHJpoLORYGhc1Wv1N3NWaYqWGI8pDpuIP3kMrusU+y9S2uwtNHBNrMeOIVx4aCHCriBZO4nVD1BOKYPa\\\\\\\/3OLeJlwuGaSAThOicCfLJTSYgVvxmRu7WGwOZlC4mIKvl\\\\\\\/Dkhxs3ZteIRRZlYZxuI\\\\\\\/hBMWw0nT5VMIXIaUDIRy8G3lvv9th7HM7Ke6wqVLHKbZQh0OwdncW3GUCoB\\\\\\\/oyldM6oGRSRPP2D6bQXHSgxMHs51VbDRd7e2nKGS+CJN2yjGEK13zOHgSYcbqqD7U\\\\\\\/K+g3mBiwbzFmMfIy5fXGTRAE2yb8E8Eg6UJ+GRkvGPu38+bEAhyGzNdAVTnH\\\\\\\/VujsoWupJ89y8M5zDuVrxRLLINZXrc5DKp6pranMVGLvCV3yI6Y+rSINIJEYUZWz2QIdM7t6qlYFp29SqiCkg+rAJUa8GUip0GdYfN+pdTTQcBDiEHSUUUsegsc9yRV9rqhnerDYpLH4TL7WBbI0G70HJjBdKzRpqAEhbDkqRbGsGeChOeYhI5OhVh4GRG6+GkMlGraTFHzR43VzsZU8bBA8HzlPU7vPwUXcyTuDYzKYgbJkSADFXxrKorybu4lkoLGxRpb\\\\\\\/XUguVrR4kbycttrEuMKSc9QsTXYnHLWtbGg9vwd\\\\\\\/t20S1GHxdfWhaWKY3WfzxjXVJ8JeXGmiTyb3lxQHbF0PMIyqfVLaJXF6aGY6Q\\\\\\\/mGdfxTY09P0hal7z9vVeKMcsrD9XmNcgC4zuwe6Ca89TOt9tc7Z+L9x\\\\\\\/7mSLm5D9M1cKaaN1jrqcW95nuzJcvaSFYlRsxfgzCLaTL+Mf+cYksgpAdyoto6ncx872yMTr91X9k1mp+wSLV7Y3FZVio+CJ0FL\\\\\\\/vbHAkJM50Y+T6EuR+xxpsvYdCqMOdNSqyL1dTY7uw43xEXzPFqLkRrcG7TdgQnsm7rVECdkLhZDs5BbD5aXHQRyMBOZg6KXSw8\\\\\\\/Vyftd25LTiy5Wwx0r0IqdC1cnoxIhglHOTykyxpO4HDb\\\\\\\/6QpSwGVBKy1+nmE9G0eaUeGbpe6xe+7w8XZSDNVQtMFTMUdlLeTCt4I51cUfPPk1d1u\\\\\\\/yFiUbdZioUVMmnyXjwF7VuwBU0xtWUIfvUw8uA5q98C5JxuE5vUTI1PtXQvTpqVzVqLmfFxrMBXx5Kx4Xl73+jFzOR0fi20\\\\\\\/cjafpujqwTMBRNiPiDJMO4jjETreTttTfoJg06vjLV6XUbvqvc5JflZBru6K\\\\\\\/3ixrwTa72OwW9tPD92WkbaR970NN7hqAFy3E3Q\\\\\\\/eLonZ0LpTRQNX4P18xd87l1\\\\\\\/s1zb3sy25No38laGsZzsfF6uow5UQVU5zG5yyrkJxDbVbXrFxg9I4BR57PMyMmyOYzOxkd4JRV+TZTjJSAQtnDYlhDpeQgq6zeSNfUyk3Gwuj5XCVbdKx4K8iqfJLuyBApwKKTeD1ewcWABQ0gcp4s3C4fjvYXy8iBjkzni+pYD3hnSeiWb7RR7SdZr+CU81m8s1AkdfMIL6MsF0tdGAg+z6sojeCYaWSdyYaTN61+8UzD9\\\\\\\/KjoxvcEA3OND1P6UIEiYHANh3sFVyL8\\\\\\\/UN0beasvu5NDNttFCpPFeAAdxyXV9EyQ6YQDEWtQmi36RgLNWs2l1Ms5P1xzH23Ii3qrWqe3GoFjdT7Cjuz3WStL\\\\\\\/d7VwkW+LPYYnweMzZd59puMsJwaEUPw6uZVMgZjcKftfyh3YBTIEEpW7PFjCKJG416ZaiDpsL5\\\\\\\/ihh7TnvkPIucJSnpgKG2\\\\\\\/9mgmud9WX2AagdqVbTR+QAJeR+zTsFkCnXyTn7MUlg4NCagZipGPKNM4rQPS1519v3tg\\\\\\\/6qiG2o5n\\\\\\\/Do5PP8n7S\\\\\\\/xeaNZD0NXG3fXniIgvFu9rYXvrhHn6SVMPIbVG7jkT6E5RObsLdVWh16G+Md5wLpFVzMyhozb+ZJjnqJ+k7wws8wanXCdKBGPSvegTV45OGNnK+kgTBfmOxT0+BQ006X4P8Hhxng0775gWPdGN9SlFfyk2vak4QCpUVb2SQz9GRFRUk0dHtTvxMTdKKyllgDxzHO9W8sO9zRXRCvja+qW6iKZ1f0md95BpLpcDCEvfek9D2sFMDaT2D7fYG0pqPFBKx\\\\\\\/\\\\\\\/6QCiXUfSY5SluWdzpPK1p4zVI86TEeqThyCLlouVZUSN+sADcFZzwMb6fo1Ln0aIykoHptwZHsU73pfShPzfbOzlzQI3FrJJlnCv1dB8tewoxHWFeeMNPBJQH6dn+zQR+T1955Qi\\\\\\\/VNE36GxNgtqGoWARkRDlPbf1AVnHSb6\\\\\\\/iGFS2\\\\\\\/1GmRv2pKegdz3d3EAWwGRHxAxYm6G88dTMMfu9BNlPsVdMIoan51eeYdJX0m\\\\\\\/LmQZZP907tu7wd3GVOHV259KExxhUWhVNpn4kvm\\\\\\\/YhLmem5Dozy56azRjzkERkmJh5dnd\\\\\\\/y5DKQivNoP8fo3+O\\\\\\\/wo7xE03NnG5cLZOWS+ZGbv4NlXd4Vh8mIB5qpBplQM6zI8PYf1lHGjN9b3KZHk069MaBb\\\\\\\/e1gsvjjM2z86OfTtZiZA+bMtUHXfwHgrW9QvvXtS+EXbwj3ZSVqDeIDpwdNqq0gJNq\\\\\\\/9SrV+hYBQ\\\\\\\/Om8ahwwXg4Bok82fFh139OX0uVX2aVwOwmB428jXjIKsNx8Cf76wnprBDMzU1IeEeveFiMLPQnLfuCDb+2fr20OPHl46QGpPfIHI2OFg\\\\\\\/Kqscy3AIj7bMyy\\\\\\\/OiQpagI+9sf19JTP7LzopkCRZ5ws73rrtMtL+nZeVIKV5V7Tp8EXL8pMXIF6i1MK9Ko1SKWklYAa60\\\\\\\/N5XJNrQxqqeyl58YDDPN+mtYtbJu2tbKx605YIy6\\\\\\\/OSdP+GEejrWQ5BHKyMSScA0UIq4YqqgjxSvSkQOT2KAC4wfFqCxFhwptMptpU0EstZcfUKfdL47bAYLcdARxuUvCzuB0ATrjlxNxxs8SLRfUcY59ZVJ+3ycU5zJeVjNfHv0LtEcW4zSUQT2mQU4VMknqyAjv84gpZF81VQNP4TAJXQlvyfs77fF1iGOvOdDb94mxCIogixlBCP22DtYXEO0n22MJYnVvtjqA6n7tCNM6PcBpBXyNrPqMhBYbpYXYfmKqN0oYYuAXSoeQqi\\\\\\\/6VRbLPN8BDG5aT4KeuQdz2NgSZ9NhCb6kZAHZquW8DE7VVHc9Zc+X7Qgq68gUTEhS6T+gmygt+xjAJlxH6BkaXnuMSEaL0+RP7OGeHlW1hcH3uEsZS5Mwtei6b+6zGQL\\\\\\\/c3r1TQwB1L31bJJdq7H6qBQX4VrbRHcyo988X4MzVr4Uq1NK6v1Nzr5tpVuhbXM479BVvY6LKN69p7I1lLY\\\\\\\/219A216zG7NnkBoCLr9WylIZCj+oUDk9m1ggvlCn8aAPHsFNRNSwz5nnabeuUxk5XlWIq0jgsaqlfRoh04sYuA5n+4s7BTbWUFMPMdY62Z5UUiK7yPBqdgV+cViObm0NYboyIqim06aPyeBTMDEQYXCd+8AWCjyLsXRViiOIx8HAlLrmBNldiUdOnH6iGMWV9l68B2O5wXko3cmEsMrmMyACKp6xe7OvdIX0eGl+nY7zCgqDdqujDUzgfjn3MEF7KXum3+ID9yHeHGgzj9m5uGxI+9n+XVXVehviNJrkqTipvNYdodGvkIeV\\\\\\\/BbtS8erz23iHejM+ON917QkKJj0ji2S55TMwBpi9GA4nB\\\\\\\/EgCNtnRXKt8slpwc0y0Yp8+Pbm0RtZIqyR\\\\\\\/iV1xZ+vmlGk00Bi2whqTWuXxVRFC+Z4QKyaal4CpsAmLdOLNRPCN7L6ry4j\\\\\\\/pqEqpNuEYTx3Z5miXGRDylVJfp7N8i6o6IQWyc5Ubu09fM3yL5oONhpbzHFmMjb9ClcTIJbZmDhfuMwIJBsKSsHF+arjRNNCwbIQrmv0Pu1\\\\\\\/o6CKTgVQIdfdtULKYYn10aYKp9taAZ+2bBcul5okCtlTdgC82ryH6GKS16gSVUd0WMu469JQQ+NUxEXNOeoHovlxCJwuvh\\\\\\\/bfnz99EDfFJwD1kg4KcLCwdYH716hBGYeAtSHCBxufd7MGTTRFv+OjjxCzvTLVcBuj5WXVYrINJ6un0LjaXC0F1jeIjE+s9E6z56lbIxMS1hIuKEUX5\\\\\\\/juua1H2\\\\\\\/J3xxeln495lMKIXZZzTqsXNB50Rx+jX8JDsyBjPHe2LfcQQU4d86fwBB3iD\\\\\\\/iQ+Zyj9Bl9ooaNYCbYWkfwTm7YRdXzis7pOFmNs4V5u0pWzGKJeiy\\\\\\\/+MdrFrAVB5CvGXvf01zSIYSErGUG7WyDi5TA8tCS+lTwa1BHXkaftLO5SlQHSpQ91UwhBD+tb3EnhH6dwhqJOj91+1+IHDcF19+51Bz5mef+Zf21WdLiv5PIMa9YdUvUUte4cH80eZMMtBIBErbnprPP1YjyRQcR3yEQk0CC0wUTfokOFEDK0RIvLnzgP6ZrZzQxZ+0tyyxwvb07hz2P+krRfpsbgcFm4zaGDcHq1uqu1koYL0HKfcEj7fOm9v5zL\\\\\\\/zstpUhbU9AIT2R0Z+RiRX5nSbXK8KPzqiumwLAvv4ZmKytyxH6UFXcMKaBB1k6425kbYr4Lhse179wNFVxzCA1\\\\\\\/JaEfqknsTpnaQy0rSJcCZyeha2Q1A1\\\\\\\/deJIpxWA\\\\\\\/LlsFWShTGUv2fUZL6vtGjChOP5klNKYOgiSCT97u8cj\\\\\\\/5WmXdEejnquOCkeBFHA74uzvuZ\\\\\\\/Rgd0w1sgULsELbqzm0KqbZASOlf+9OLzetse3hGh2C10m0KlXjrqWlfSJMuXNuktmjUccpdwkwHRyi+hoMpOFI0nZ4xJsYqvYc1HsFMMpx4TVm4zn3XX0VPbHrrcKADnpd419xCHGr1Mdk78\\\\\\\/C+6FhXvLcPnRrA9eIKcIri\\\\\\\/MXey9aFiydOF8duxeg7vlALPmOu\\\\\\\/pJ5dGlucudv9Li43AuIrTNFZ9Z4u5MGNeMNB1pU4pvC9Wz\\\\\\\/wdqQO2lt83eiQPaYPsQP9OKZOoJw+czbLumcocUaNNk\\\\\\\/wLK1Pk2II4YG+TA2NGInsn+IUJfgk5E4p8l0Sv3SrgVjE99NNpmzXJ3FcUfxzml+ePZK7NwzIQV\\\\\\\/VBhI0MY7JDTwT+qE6buqWU4mxQ0HiCbGzokPP7gnondnp167GQon\\\\\\\/W7gYUJTj8\\\\\\\/Ijg6Xco1+jfv9sU0zOeApeHcQgqNRXEbUfZlNQSBG0TVgcFsC2pETUAJCcpm+w9JzdtOWEZjEP61VAE+39sgGoIfvm9tjzuiP48gdE8ki6W+f5YTAkKuMtZrcstRskdbrv0nSa\\\\\\\/5Z5EORriiIZ8u03MGSTBJNpb3tCZhFyzAhSOyZ3vkm4MlZMAbD1zf8eBmGwi5MAs7aM7hEoxK95X3GqWdZ+geL1em7kb\\\\\\\/837m1+ovi+fvBA8liT\\\\\\\/eXC2oxjqPqA8ekT5xAE7\\\\\\\/sNkHeZsMSwKVUyItk0uKnalkGURAGuktFHgegODOhAxjzPHxR+87eOShe0U8PqOCuqJ2zQYUMmVKuAxnpWaUtIWekBU88s77taP3pVjIy1ecdHsEFv\\\\\\\/u+GJelcJ4nkHx8FyhQAvWIlvurSRyK0scJqwGOdJNupPPbBRV6RyMQX7MFXaIv3y9BEHJDfyPFEM\\\\\\\/0OVkUoQCF79vpXYBZSjBHX97hKZcIRf\\\\\\\/YcEtLpLtqOzM7RzJ9G5u2R+Zzijf0S4BjDpORYASWdosWAXGECdsmjgIQIGE9XlNtZsPK\\\\\\\/1uv2XvbduXuiVU9e6vnjKXd8mHge7HBlJxKuh84qCc7rMJH6g8ZRQaE6Qb4FYPovh5fAgNinsVBSg2K8OBPE5sd3f+C8gdeQ410USUjx+EHCS9QdXIIBNlOZEYd7rv6j+PIHQLWuox6WhgwZBZoTRCkmHsBjLLa7YIo\\\\\\\/g0DVbJUssZ7vGkQ2r16o9NKj2Q1RVrrrx0whRvpsW\\\\\\\/UxihfBtkXdvK1Hh6JbI6CIzagkFUHy1TZ9cYM5AHtT\\\\\\\/2zGeR2xLUMnxZq31VGfLoc7CkT\\\\\\\/8W2p821\\\\\\\/yY1VNsMxQjcM5fhSXQssQm9n4DmzBgfQkZ5Z76O\\\\\\\/Gz1Ce0kx8vPfAN66F2diiaQfsoi9A\\\\\\\/pnW8fnlHdQxDQPu4B54OFDPVvzGiWRF\\\\\\\/x4fpT0YjGbxY1tpJ0\\\\\\\/EM1h6GIkBgABWIbrjG7qtj3iNXW+CfqQr56Dbrk7mX9Q\\\\\\\/03uxqvgCEAJt79W9V9MfT4IztvixjxyPRaPmJSNClq36tvOQ3gkIG\\\\\\\/0ZrWM2fxwdnUhb3gARJAFF2p84RCDNsHKGiKoM0c1xr7vpDaLKqoMHKJlh7p\\\\\\\/fszCO7ihICuj6vw108LZzBINIqkbBxYWcRmIpkukSDWdENZhwB\\\\\\\/93R457jx7xG0skgKTJssGba\\\\\\\/dmXv2KOsa+AbC\\\\\\\/O55H5PK3GbH0V746GnJiJBwlHY9w4ZGCsjvJ5SqrqGh5h4tGhRqtfKCW1rN5JyYcnj2z\\\\\\\/1k7HhcAH95yf8incyy275j1X3DWCekR+6pqCuM4OOblFlmVzUXBSZqBzqJRRdomglIulFtuR9QZcsbUIlJ7hwG7lYRS3xDbmFzUworgT1Da0YZ7gxAiXdf8UYwdvti6JxRjWIvUUIWqWN0Io8yfLA5xix835m0JVq8aCFTrPR\\\\\\\/zC4jcG0N0x5hBfEQP3RpffiLxCc7SgH\\\\\\\/fALTWx3WurkIpDFtost45KZswxA436OFT1HWdP6SQjG20KUi8ETsaCBrDjztr6vT9lpQYWKZgF1y0WtW\\\\\\\/zXnr9kVa\\\\\\\/+KvRhu9pzuOASTuOZecFGER9Fwfvl1YhTS2lweughjrblywfgRnjjKw\\\\\\\/0Qc0B2uumUqSTjOxMN0Td2dTwg3c6OMN0LqrFfACE0uTtICirDCZ8c6UAXL456tdwCimX\\\\\\\/rzhsRY5Nt3LRP+ehiinTGavH8l8v+KjHD1rgsErUhdwaznllgg+czrLFKfs0kX7vJlL+aamo42Lvlne0\\\\\\\/sfOxYAiKAUA1mCUDQAT3gdPMxiudwbtIg+SVHYCK0v3gLbQBFAroEX0toy46JxzxXAIMloKDYq+z3b43R7IPvQsqqSfp4u3zrzLJG33rPdhj2HI3xjuDUrzaDss+TZt6S\\\\\\\/R9A3K340rWDzHtY0EDkN\\\\\\\/1M9P9s0e7sNdBxcPxoK\\\\\\\/dDHAbgCT\\\\\\\/LkCphZJR9ui70dopu6C7PZbNy\\\\\\\/vZzg3JVDaKfruPbfhi6qVqnhK1jDTP347ZmlBPsZrvDQXkY4ygDLxsSv7FBvZX+lZK2nNQ2f3ptxEhmVZaXjulwBPZrfVmFqdjiodUiUefOWo8b1NEse98GyMk+Zp2IoG\\\\\\\/zC+3Z4OiSSAAx2BeLJVmpxcWMp27MYppn1Ob8ieG3qlpUJygbBJTjk3nEd4cLHn9NRdc9EXJzE0VT\\\\\\\/QjWdfIrgOHCr5f\\\\\\\/fjojLzHuLFiX+S4qr8E8eXE\\\\\\\/17XxeaHZwEyrQOe\\\\\\\/s7fFQDZBaMQnECYphPl2uxAnTmFEl\\\\\\\/wH4czFUE+u6RS0ZWPY+BsJGhzAmDuJpfZ4Q9Zswns4WJQl0Wb2BGhLiPiaIb\\\\\\\/Exfp5zvsKJV21olil\\\\\\\/0XMFzNnF7D8RTP0jkI54NMt0Hs3nKezY0B7NyNb7z2oaiSeheJ4ENR35+R171YTSNs2+Nmwh4BpfABlyQ+dujvDqpZ6BSHo+UbV3zNmEdhps5FOCHXBaEwRU25lwXjLRIeDYcMCmopK1jeDzE3tdnF9IDIAloJOsogsDRIeBoLn\\\\\\\/hotddAn4rQyizvkmfPKRhED2\\\\\\\/1fyJbKD\\\\\\\/e6FqxQCQMiyv52kfmxNPm5LlCPSqjrqEiPisaoj36OFakQq0A8I3gfvsZ4DceuQkLKSb3vym7Si6fCO0tvf860YEvJgQBiVGiGjQh7IRKnCrWRL2DgCjgMh2VW9czHdpUQ1ulVTMC\\\\\\\/vmDXCSlExvOsrj6yY\\\\\\\/eHhF\\\\\\\/f5ka15lreS4P0yRB6GVtpA2+hVJutKIsTMu6Ui2UI4bqlLqXq5TJAvIrv4oUEBDoYuwdHQ89oKENV2Z67qhohCIfEC46GyFlYz2f4hKMcz1zL+k17HMxv5uybWX+QP9wohNQhZ+brhjdAesRJkTdIF7gWRy7LbxNHJ1YvAiRoEHjj+RKMozyLyXDC5KtkIyBTeWhwE0YLPJLkKz9CQhHAg+TNxQFgK+aCrMpcjtx0aaTee5BCVQpCSrxj8CCGJ2tkmCdYP44U7SNe6+35Y5O8CfNwXNQGECIZQnSCBjwklsOAz5Kg9o6hTX6tep\\\\\\\/OlOzepsOD8brUbyv7buuIX097TrHjBLV\\\\\\\/C3\\\\\\\/JQdduvW59i\\\\\\\/a1cn0GkXAArj7lK4XKZEze4NuaLvF5Z97w4pUViJ0fUZy0WNBMmYIDwsyIwl+5I84bbEDMzNcNADTjCiD3zUWk6WTYk62KN+gT3juI2apN0dWRm2r\\\\\\\/96dBExqwPRonUSuGLvj29DLtG1UZqrhOiteHpHutB1isF5xTBr+Vo\\\\\\\/clh4rOFt3tKDtZEVFkvFt4JMNP6Bk79W7aHjuAuojAkMxPVn+RyVS+m6xys6WJk8djz+p9mxHITjYZ3wW\\\\\\\/k\\\\\\\/mWL92da9cxxd78b4Yq7Y12SS1znUb2Zm\\\\\\\/Rv5D8kUEcYjhvA1JvteDIpQts6VI\\\\\\\/1Qtwq5TrpvUoGYukwB3A8MqvtrZ0R32FWRhPbOQ\\\\\\\/K9xphPm68GFl8q7P8C2nyS+3XqvRb80mKTpCDiHzmxJUFyDY56EFY3f+ApJXlkmkad1nWh+4r6yBFHqKuVSS5lxOEhU19FHriIdaoOM+bO0Qa7gA2OQdXX2Z4mvKbhPuU3Rh53wnbbXv28YcRYrWmkPr2ltbODh0vlmXC+lAapXzbCOvpoFMdwLf0Mll01RzRlCMBCyF0SUcWOJ5zieXFD2X2yIqByecmF8dUmbajjnmi8qhUewqp1Gp6CiqZ1vaEpfeUjOG\\\\\\\/6GtvDJqaDeF1fXThU7i8eq18P5AxSFBowjaxUghJmpe5bzzQ3EAneeVinJSW404476OEu1+NtBSi7HKD1kk8KejUFORxhYyHpO56dITnF6cbeHK6azM1+U0gMO3aa9qYBBMWzHacy5sCnnNrd4M22hz1QalsGBDyiy3ilNStE3pH57YrlQAawrJDKmjykDzttVh2Vsd9zMZ34jC\\\\\\\/oRn2eYExu3AAAIuJ2tSIYkpCXdxzlzEqAr6PGrdnmSWQs4Ew56A16hTMZwOl\\\\\\\/RX5JBtCc5nVY6K6eQS7afoVaRZZ3qwuEOoDhWQst7pBsmFsSkCAkqo5N7tzQJs7TfElr0ziloNkoqFBPDSq9WdsSpgmSJv8i2bwNNn3oR0Wzkms6Ie6ParEm1wXz\\\\\\\/+m6exdsX34hJdEdHPDSJxBb2pH9+UvJPPtZnWC3\\\\\\\/huMFPYpd\\\\\\\/VeEQUUNM1bSHzZgcghRQMrPzOzEZ3z4t9e2whjfjFcZXPHxV0MQqKvvqp\\\\\\\/aOrWJWpiqKa0QyxDPrUUzsfpFrGfh7kGjCJB9DSsBXuU\\\\\\\/2ynCg7RMFU5MFnVgQKFYVPjPXo9wpHhAI3xjvEQa8i0GLjXUOi+40KwgRrI4uSxKIeSA8vP68ecyWRjrEtV8cQhQTJ8KqrSO70gUbmPJPDNx4+qisEgwjJkglyDc3T2ze2dnUUdnuBtg6+hyt1B+jNYkc\\\\\\\/o5wVYpu1nwe3pgwU4rGWJQ+Uo7MAfQtHGHAwuw7CVwbAJU\\\\\\\/rA8rmhqZE8qEoWb9DndC0TWpMy7T6\\\\\\\/EulsLq0z8BZCKCl5+7HQVAmc+KzMZ\\\\\\\/1DdrDN4udy7kQB76aTyTAiE9yFxtRIhW7Rdl+K3G6CcTuvCbu+doIokpX0EhAGNlm52jbpUyWywREMgL6AiJTOqTytYlGO7bwfawkuXnUMJobK8NhqoNnMFVwkmutuwgXRCBd9syPejaiwnAAhx2VQc1H2yzlRtaQVVyzngv5NnYhMqNmQb6WWxB0wu36xEinO1MlGhKrzcOTGYLdLGTuQjO8i5w9wLqtFnUpyIXtP7sV7FSshx4JiCeKcPdhdc6dhyt8aoPHDDRbZa8vpiLUKETABXdGEbb0aE+qJwuConKRMi9J\\\\\\\/6ovZ9fppykwOLFC\\\\\\\/RCNsIaEiErESeGm3Wp3DHRpjdNy8Mx7gJCaAWo8HVB\\\\\\\/b+sKcl20xXcfjkK9Nil7SKK\\\\\\\/mmoDShbYNyE4MRhX7qUMoJS7Wuwv\\\\\\\/lZtaO4x4HIYnWGOkpBn8V8bMYZMeyWZcfNs6N5F7gNyv+RC5PpuE4b83\\\\\\\/gJvn+ntXEDlTW19RYrlDjHhb6jzkZWKKjh4gzMpgydmF3IoAF0+LFl7wK1xWm39faJ4ZLnCClo3sog2vBn32LclivocfdSCk0pNc+qSKTqB4vbe9smURGJkavPDi3uu23KvalECy9S534In5+XUXpMzl1tYYq5TgdnisHnTjRhJuE4Ktj0l+oWL2f7ZkhExCG\\\\\\\/1FIsamWqD4tUC5zM1vT\\\\\\\/0DgoRGA+Bwy0MNaTX97bwLiwK6Ht3BdYhQRUY5BwpNRzofRTntsPAhMZVBrdRbwRUSt7oRj5ur5Vv0BmCV4eY59ETQX9FM9s7UyBOcY1KclK8hGVD7WHfhSWvoATgFSAQSCZS+ulKfUcLMEnbP6OPA5iJnzkIxHwurHH4dNUy75pXW7z6QaY\\\\\\\/Mb3unDWuyPewjdiHrg5\\\\\\\/ZTwUl4wjud3xzN1dIzvV1+uFShZoFc4UgONSYR1P1DiHCtzlPBxvrOo8d1TGjs4XoHhUT90ao+\\\\\\\/C1FgfD5PEOeVWpcZjAm5J9KPreODOVZtiLzn3BcVpyZbAHueB9HYxrdJUBuWbNtRmXY\\\\\\\/CJ\\\\\\\/7hsEClnFO41pqp98lpGyCwqIOUvwYV0qcaVYJJOCnFDmzRq\\\\\\\/EL1ATSlRjZ1Fq2xfb1bK3lJRs6gSwtUQw0+hlfTGtKhRP4725jpwFVLMEGdMhhl2YGNmbFw+K00EX\\\\\\\/uB+Ca5OFnXgcEJcyqVqXmkmbxhcsI6m3\\\\\\\/wxXtgHuK6jOHyD9K988AOdIoJyFhT5fgXf5So\\\\\\\/ZS9+2FwUpDQgNgqosfkTXcXZZZb86CzanjJsUdp51rpXWY3kUGgQ7IyOzHYhUWmiMSJbouboPM6uU\\\\\\\/p+c3iZdLklQtHJfqbJ\\\\\\\/8WQcsqJu1hRYF3ey1g0yNi6eI5fVAao+PwDhAakDdGrSWnMDeMTT1zrjiIeW7xil20cix5R4dZlmXksUt6rVRd3ObwDqNl4lUK3NtKd4B\\\\\\\/RKZntkskv\\\\\\\/PCyldPqlIEBegntI9qZMJnOGU4lWp53epKVMU7TiAA3+5qPDwbnnJx82wHezV8cvxKzmVB4jJnd0RmZcqYrOt6A7DXWRFHhrgEHUVUJ5gTSm4LsMP2N0O7U8oQOzDWtRl8uquyI0rbbHDhQY0FAPg35djuOA3fAqGb+wdXKKYBzaEqaOneYdoDBjLJw87HFPT\\\\\\\/aWuP5JxSry2YyWjOv\\\\\\\/oT2qn3KYK1m64dHy9\\\\\\\/f4E0LJhPS4AdUHdblOISW\\\\\\\/SIAvYZENzqztOtyX4pENUcKLKbTbjye6OzEeNSb1rZ2bVanOurSLei9DTQoFOfDvvwTsUVEF1lhMm2MD4KwPfQwozwPv4P7i8G19FF9zsIbJ0cgv8ms3T\\\\\\\/9Rq\\\\\\\/pb0xBiLT5wD6wimP2LeIcgkJT08Za3Aqn6VK8Htg3wDbU6Dp9Uq7SRq40OKDb+dxFcpp1o2iqP670NAIrAOGsTe+ChLLRAzMIHX4wR5fECSNW6YZscSG8p+jqZ4A+u3ru4wc7co0eIdOLkdvNsFxhruZS25Vjr+O49p\\\\\\\/oRfh\\\\\\\/DlO+vTWVuGNx7xKbfsXoGycb2nUqGzbTOoR+8OBfiYExBrIRYmrKOzb1mMcAHQYmiJ6WWXUZYmTF1LHL80t\\\\\\\/T6zb5QcNA\\\\\\\/f8bJhBYWTFZoCaGApZHN6bJ8LHBPbSTK6tHisn5KQBqeAxnqEEmNFUP0oWrjXSzuW\\\\\\\/olTOu1YsCCSMdOMh7Wpu5dBzBj+1MIvcgFnBdwMD2x6qSwXxq8zEksfYy2qK+0h4MvoE8Qu\\\\\\\/BtcE7KALrReJfAWGIP5Cs4+WSX9zcEPWmEMiPF3jUpza0W6B8\\\\\\\/vz3EPQFa14PXdsfnJMgmOWyTbKCV4DgouhXtmcEWDn9BUn10\\\\\\\/KzlNm+m2tmx7sl11PshINYq3cU7m6nKdm9lEKGVFJiJm\\\\\\\/1CAWy1dfzpsCFb1whKbqVMaU+LdNaJH2d+wTq6c5aRDtlwHa7zGZ4\\\\\\\/u+nen7dQcTsrZlISqLx7YfISKJI1EX+jpjlFLpKEMk+zB\\\\\\\/fm2WIgXbgltBDeHjj7yWIuHwIy1apso6GnFW1fMfPkwxe3keO3UZvDwD5yvcjl7PxWegjGZ6NVwTrhzxnCHM30M3QHvFWc1FvwP6CKJY5qbNnPEW6xf9EpaOKp\\\\\\\/xqpBd8MCJZbY1j2Fb0A\\\\\\\/ckZfxx8tPgmCyoxcXgQoo5J7Oui4l6iP9ZKeKa720nAYAye\\\\\\\/16PMynU2aHkglITVL2Qy3a50+UF0BOzJlwXXp+wFXdhBZCnD0FEJ3N08RU7BLe+gC9TJ+aT3UlX6gpaoKi5vyJXeZOcIsYBWfIb8xQFg8lMZA+dAT17pK7+G0qBIBb85tV7c+vItghnhY0nF\\\\\\\/fJU+104vr6HfPareydjoZ1JehfiuTM6Fyd6mhxuph2Cr1X4AfI9bRU+PHnStcCZDs2vj+0a0OyWzdaWUmCmNWzBr9hjzlwZxJKqLl3Jx8PGcAqaWe2u7b70RY6ICDLYpJnHIHb6yh1nwXdAXtMsSqYfiyGB1uEucWFRHgJ4o9C\\\\\\\/VZloLnVVTCkFciC5GqhuANfgrrR8Y0tDNbTpMzgK8Zd7uxOVYUth3CVTl0PIHdrU5+C7Po9Y2\\\\\\\/jLvOoAjLUTohEKcbJ1zOuQAfLM0eg1WkfKlAF5tphyMuF0q+DSlDJxxrp+JgyQwg+nTxsiApflQ6j09S0+ZKLlmx\\\\\\\/+Ylh+vKwK0SNKpyhIq4bi2zgM6mxAQysC19\\\\\\\/RkrTVGOtZw2MWJwtpK67SzH26YWMpvdIxcEu5U+N52\\\\\\\/5BC0DpFx1BITIoV5+oQSvYfijwRG8ACDeTDtP+NjNdRKlXLvND1q4qS9SqkQns6gcx\\\\\\\/rKcc1wkHq9pqo7vCIoWG3vyA\\\\\\\/xwLdYwYUod0I0kXQpxS870GxtEHSWFIT5CB81pRhB5ogWF2F3Cs9mnO1X0BczR2UXnKHnYEc9NF8Bm6R2567UEfTMen8JJal+CEKBXj2EaAsuSTlomfgwh6pF1rspzMm7zmjBWzafcUd3FTfnAmW+EE4wNQNBtSo841zxBLIrBZJEGkG7EIlUigg6kJM5bNbRIGrvRRkfTsJ9Q+h8WLCCMclAATo0rlcu\\\\\\\/MbP6MPyBGW3OKiAVdymrWoH7uJF0sJjB8P+vhaxAuz80168v2DFvgcIGdHM8AnJ7ssqD2Opegb58GPruIiiqI3HD2tXdb7wKWCQd8+AMEeE6Ny5g92Gxd+9tq7Aw9x7rv1Dkua9GdHlBEgdbLMWx9tlMh0qdNF8D3k631HOhGigDiRWOlhfO4uUiiDmxoUDHPiM4grE8E1Wk1tvfyfwov6DorykSQvn4Mj\\\\\\\/iKTt7xrVi4x1ORhtRrko0TkX3i1xqVKWiQgP\\\\\\\/aG9mu0qcRY4MheIi2e4WJQRHIT7dY564zRjZRh+Vc+eMrEuT+E4wgXDaHQqgGvdmfnN3REkxjbQUm89INk2qponx9kPTvqvZMwP2sJOFvJCfbQCUn2RP2h9opzVcWvNteJPGgsnk\\\\\\\/b4OCBxkT6sJasmypnQgcQ274sPpqHJADZSNZi2isDF9g1O7En1Fq9gMooG7+ttjVpbM\\\\\\\/\\\\\\\/FAF8qxcfyAXJuPmGizA4GJ2xvTtVa29F1J+H\\\\\\\/1tNdq6DOl9UtIMHHA6BTq7kjEstckDq2yPmmKgaK1Z0lMUmADS26S2Y1KiDNH6gAUIFSHge\\\\\\\/2SqpmX5UeQmw2txvItweGxOo5FjlfScN7eTDJoPSwKzIzCcM1G+cryC6NMrGlme5tFCGfJD8DiuTUFT+jiq655i9XglkLXp\\\\\\\/+YAsa4GShCIDydbhr4S3LcHKOA7+Yi+fKpCdxLfFECCBV8XJvBrzv3JZKZCzAMi9UJN7SgSDd6RSiol2ql+eftpXFwso8miIf7+L8Hbiyc\\\u0022,\\\u0022iv\\\u0022:\\\u0022q4cZRoMjL3xBoSyf90JDZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022015ed61e156dcb96ef55e9533db5cb69deb783c9403dc721f55ffe4c9a04ad6d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I5DTbm0dXP2VJ8wZGzULZUT2mJbBptLsPSl4z+3sMIp25uvUnxS42Lb8O8LG1cEmf2\\\\\\\/NJiUoQPD0m8O7Dobn5pDOLAYZWxmpAP5MvY+tN3Y3XdgjJGV1vioXAzFZyK4axybkVkuD5sLSi9zmhgiyhWeN\\\\\\\/LupDY11bL6OIwuSzSKewJ2rEOuBctkoCt7rB\\\\\\\/i3cMBDXdrpCZZbG2fSzISlecc4yfM2viC0mZQ3eGYkbSqowBdZVB4KUEQht0A6c1fLuqatIheqb46K0\\\\\\\/cEnRji5Gb2qsrGtKXMzZ6aXXqHmKwhDB9YPGpK3fcw25X79lNNnqAeB61G2H4XPkw+f1q8r8TnrxhBQp90csacXXyN4I9tz6NpixODR1K11K0PCAgzDBhmNN+XlXh9Hgs9c36CHhOyMRA195+k6NA84Juetbq3O0ouEhwEJoGOofZVVpxdgZTFoTMqblbld1X\\\\\\\/O7+USS5LyyiFiPO5QbhS6+3JRtM+caK7jD+mGDbibL20QZFfQ6f6vMj877O5wAWJvJnvoyyV95XLxvHmtNxYeMM5ksv7+grgPN+AQOXK6z4CMakNkLPrPZiRaePkR509lCz4s0G1flwvkVGds9+8mXwPpvvlqKIEF\\\\\\\/db0Y4BlE9Xnp6CVDYHVkVWoKO3rPzOc3huxg1ELUHMCCaPpPdcIG5VV0Kz\\\\\\\/UshKVFd6WJbawrkyVI\\\\\\\/+g51+7vHf9+xCCie327mXagBIgKgc9kg1zlqchWHOTFHF4awBDenrhAQV25ZD1RaoHmh61vxkGY8zhXMpONwWtrzsyzb8giOC4SBdxa3CmEeuHVXaQj\\\\\\\/lgn6UFy3u6rxA\\\\\\\/nHjAdVN95DIMuxfZirbOXQAKZRD6oQUKMwMlab\\\\\\\/8dFCeqVEGEumkPVc+wIGGAcQ3VrHOMkBlmtPyKjimuwtvTOixolUC68vod5FTZJ3hklSTJ\\\\\\\/M3CGHzLK6OoNl7G5+JOYkt8\\\\\\\/oA2b4gpXUI1lgWvpw\\\\\\\/0qFhaKBRnAHX7xNARBlm42X\\\\\\\/dVSUeSivGMUiSz7\\\\\\\/E4Tm9LEkuFIb3cE+xegfxWFDVJBH89te1vHo4Gz0OG8vVy4CfnAgNZfUG6JB5TKH4TF\\\\\\\/S+SAgKanTadTOHXMsJqJy8JjbtDrCCYLxyApuhif8FNTxwHmgJvPKst\\\\\\\/MP3MOFkce16fDDZfyAHFKGPJWi7EppTp24tw1mTTH2mrVa7C7dKiTOIk82bOxsQ24YupDcWSwtKaWnPfzWMoBidUq7LabgC9qOu3RHvtCzG5xzTZPDJ2VjMlJ8ZjFw45V2HRFtGeYfevXT\\\\\\\/fWdZqIGnTzl7WLsBgJxt\\\\\\\/g7sA\\\\\\\/FyJvxkW45ufg1cKaSQwRQZ+ruaGXjFIyIcKozX9xrCTs4hxgEqUomVRc6Xwx5vlzaWQdYw7pRvNuNY8DJ85TPr4d3YQDjq5XNNXR0jH3xMm73FrZE6QN4E6mpoR2n7VTIZEsO\\\\\\\/pa4JSVzdemkwDvEAHmo83AdLwT72z3m6TkPvujRAOWL5J5CLq0eC9lQPV4GAxNHwtYeh1FZjMRVnzEGEFeEui3aZvWl0+MMgRiN0yXP\\\\\\\/Cerc+3MkOTLWxrJofn\\\\\\\/URDLVFvP81dkZ6UDwJ5QesDX3Aete20Nn6NN7UnVY82riWCKzQ94HEQ1gdn+GZ9gmvS5I6qG\\\\\\\/tbVu9JRgkWOa+mUlywGGmhBIJXw4m0AjkZb3xFvhb4AhRoPeK++f0beLwZrrFAGLk3PMbi21QJRTWd\\\\\\\/E4y+uEViF9ydXeOqYmI5dBZEvNvI6IfEYj8F03g\\\\\\\/IJZlU6aRUifTmP5ALEEXb6thEdE9vPJ2dOlktpPMHOFbU0lOBF+y\\\\\\\/A4RSfF\\\\\\\/cDdZRZiLisFPbLOrXnfc9OjKVhle29KDHjn6RSdnsRYlZ8zKnh1jyosk2dyk0Gnw\\\\\\\/KpFB\\\\\\\/nGof4jGQeUjL5EPeRuVhD8pQhZf5wFP1H2eubPlxsFCYCazlVFf9oLJRxuGdZ+k1yWILugiNYJkKZv4JmAAVxNxLjK671evhVhJvfFEakvaa0ZqUXPeUECoDebBkHmmUo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jnV1Gh5AqIeIiLjBbiDr7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022196754431d44a1c0d84be1b3ceb4a2594541b84bd1ef918161ff7434aa48b850\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459905 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/891" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c39atpViBjSymH4UR40P4pu6UwgmZI6TUbVWKVymjEk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/891" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cdG9_HXNRjMTMStU4DHxBJXp5AKD55W_VgHKaQkf0GU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/873" title="Quantitative Genetics of Migration-Related Traits in Rainbow and Steelhead Trout" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/911" title="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459905" id="node459905" data-pisa="ggg;5/5/891" data-pisa-master="ggg;g3.115.017533" data-apath="/ggg/5/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Alexander E. Lipka</span>, <span class="highwire-citation-author" data-delta="2">Fei Lu</span>, <span class="highwire-citation-author" data-delta="3">Denise E. Costich</span>, <span class="highwire-citation-author" data-delta="4">Jerome H. Cherney</span>, <span class="highwire-citation-author" data-delta="5">Edward S. Buckler</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017533 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459905" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Denise E. Costich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>International Maize and Wheat Improvement Center (CIMMYT), 56237 Texcoco, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denise%2BE.%2BCostich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costich%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenise%2BE.%2BCostich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jerome H. Cherney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Soil and Crops Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jerome%2BH.%2BCherney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cherney%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJerome%2BH.%2BCherney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Agricultural Research Service, United States Department of Agriculture, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Agricultural Research Service, United States Department of Agriculture, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/891" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459905" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459905/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/891.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459905" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459905/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/891.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459905" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459905/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/891.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459905" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459905/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/891.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459905" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459905/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genotyping by sequencing allows for large-scale genetic analyses in plant species with no reference genome, but sets the challenge of sound inference in presence of uncertain genotypes. We report an imputation-based genome-wide association study (GWAS) in reed canarygrass (<em>Phalaris arundinacea</em> L., <em>Phalaris caesia</em> Nees), a cool-season grass species with potential as a biofuel crop. Our study involved two linkage populations and an association panel of 590 reed canarygrass genotypes. Plants were assayed for up to 5228 single nucleotide polymorphism markers and 35 traits. The genotypic markers were derived from low-depth sequencing with 78% missing data on average. To soundly infer marker-trait associations, multiple imputation (MI) was used: several imputes of the marker data were generated to reflect imputation uncertainty and association tests were performed on marker effects across imputes. A total of nine significant markers were identified, three of which showed significant homology with the <em>Brachypodium dystachion</em> genome. Because no physical map of the reed canarygrass genome was available, imputation was conducted using classification trees. In general, MI showed good consistency with the complete-case analysis and adequate control over imputation uncertainty. A gain in significance of marker effects was achieved through MI, but only for rare cases when missing data were &lt;45%. In addition to providing insight into the genetic basis of important traits in reed canarygrass, this study presents one of the first applications of MI to genome-wide analyses and provides useful guidelines for conducting GWAS based on genotyping-by-sequencing data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">genome-wide association study</a></li><li class="kwd"><a href="/keyword/multiple-imputation" class="hw-term hw-article-keyword hw-article-keyword-multiple-imputation" rel="nofollow">multiple imputation</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping by sequencing</a></li><li class="kwd"><a href="/keyword/bioenergy" class="hw-term hw-article-keyword hw-article-keyword-bioenergy" rel="nofollow">bioenergy</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/phalaris-spp" class="hw-term hw-article-keyword hw-article-keyword-phalaris-spp" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Phalaris</em> spp.</a></li></ul><p id="p-4">Perennial crops, which include herbaceous energy crops (HEC), are increasingly studied as potential significant sources of energy because of their environmental benefits and the increase in prices of petroleum. In its 2005 billion-ton supply report, the US Department of Agriculture (USDA) and the US Department of Energy (USDOE) set the goal of a 30% replacement of US petroleum consumption with biofuels by 2030. This goal implies a production of approximately 1 billion dry matter tons of biomass per year from forest and agricultural lands. According to the reportâs projections and assumptions, achieving that objective will require the following: (i) the conversion of active croplands, pasture land, and lands under the conservation reserve program (CRP) into perennial crop lands and (ii) achieving biomass yields ranging between 5.5 and 8 dry tons per acre from perennial crops (between 12.4 and 19.8 Mg ha<sup>-1</sup>) (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">US Department of Agriculture and US Department of Energy, 2005</a>). Plant breeding has an important part to play in achieving such yields. To efficiently select for higher biomass yield, selection may act on secondary traits such as plant height and flowering time (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Price and Casler 2014</a>), resistance to biotic stress, tiller density (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boe and Beck 2008</a>), leaf area, and plant architecture. Biomass quality considerations, for conversion into bioenergy, bring another suite of traits to bear in HEC breeding (<em>e.g.</em>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Vogel <em>et al.</em> 2011</a>). Common methods for transforming biomass feedstock into energy include direct combustion, pyrolysis, and fermentation of sugars (soluble sugars, starch, cellulose, and hemicellulose) into ethanol (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wrobel <em>et al.</em> 2009</a>).</p><p id="p-5">Reed canarygrass (<em>Phalaris arundinacea</em> L., <em>Phalaris caesia</em> Nees) is a promising HEC in North America. It belongs to the tribe <em>Avenae</em> (sub-family <em>Pooideae</em>, family <em>Poaceae</em>), which includes the oat genus <em>Avena</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quintanar <em>et al.</em> 2007</a>). Of the grass model species, such as <em>Brachypodium distachyon</em> (Brachypodium), <em>Oryza sativa</em> (Rice), <em>Zea mays</em> (maize), and <em>Sorghum bicolor</em> (Sorghum), Brachypodium is the most closely related to reed canarygrass (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bouchenak-Khelladi <em>et al.</em> 2008</a>). Reed canarygrass is a species complex that comprises two chromosomal races: the tetraploid race, <em>Phalaris arundinacea</em> L. (2n = 4x = 28), which is thought to be native to Europe, Asia, and North America (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jakubowski <em>et al.</em> 2012</a>), and the hexaploid race, <em>Phalaris caesia</em> Nees (2n = 6x = 42) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McWilliam and Neal-Smith 1962</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baldini 1995</a>). Most cultivars and wild accessions of <em>P. arundinacea</em> found in North America are of European ancestry (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Casler <em>et al.</em> 2009a</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jakubowski <em>et al.</em> 2011</a>). Reports of breeding efforts in <em>P. arundinacea</em> trace back to the early 20<sup>th</sup> century in North America (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Casler 2010</a>), but this species was already cultivated in the 18<sup>th</sup> century in Europe (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alway 1931</a>). Some noncrop uses for reed canarygrass have been reported, such as phytoremediation (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Picard <em>et al.</em> 2005</a>), erosion control (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rice and Pinkerton 1993</a>), and paper production (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pahkala and Pihala 2000</a>). However, reed canarygrass has mostly been used as a forage crop. Consistently, most recent breeding efforts have focused on low alkaloid content for palatability to livestock, not on biomass yield (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Wrobel <em>et al.</em> 2009</a>).</p><p id="p-6">Most of the research on HEC in the US has been focused on switchgrass (<em>Panicum virgatum</em> L.) (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sanderson <em>et al.</em> 1996</a>), but reed canarygrass presents characteristics that may complement those of switchgrass; it is particularly tolerant to northern climates (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Casler <em>et al.</em> 2009a</a>), as well as to soil acidity, alkalinity, and moisture content (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bittman <em>et al.</em> 1980</a>) and high levels of metals and minerals (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cureton <em>et al.</em> 1991</a>). However, biomass yields in reed canarygrass are not very high. Based on trials in the US Midwest involving wild accessions and cultivars, genotype means for dry matter yield have been estimated to range from 7.6 to 10 Mg.ha<sup>-1</sup> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Casler <em>et al.</em> 2009b</a>). Also, quality tends to be lower in reed canarygrass than it is in switchgrass (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cherney <em>et al.</em> 1988</a>). Nonetheless, judging from the significant genotypic variation observed in both biomass yield (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Asay <em>et al.</em> 1968</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Casler <em>et al.</em> 2009b</a>) and quality (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Carlson <em>et al.</em> 1996</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Olmstead <em>et al.</em> 2013</a>), improvement by selection of these primary biofuel traits is feasible.</p><p id="p-7">Because no linkage map and no genome sequence are available in reed canarygrass, association studies have not been conducted in this species complex. However, genotyping by sequencing (GBS) provides the opportunity to call polymorphisms without prior marker development. With this technology, a genome-wide association study (GWAS) can be performed on reed canarygrass, but because no reference sequence is available, one limitation of such study is the inability to assign single nucleotide polymorphisms (SNP) to specific physical positions in the reed canarygrass genome. This has two important implications: (i) SNP-trait associations cannot be mapped to a particular region of the genome and (ii) no imputation method based on marker sequences, <em>e.g.</em>, hidden Markov models (HMM) or sliding-window algorithms, may be used. However, other methods are available for imputing unordered marker data (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rutkoski <em>et al.</em> 2013</a>). Another limitation of using SNPs derived from GBS is the high amount of missing values in marker data and, therefore, the importance of accounting for uncertainty in the imputation of marker genotypes.</p><p id="p-8">The purpose of this study was to make statistically sound inferences about associations between GBS markers with a high frequency of missing values and biofuel traits (related to biomass yield and quality) in reed canarygrass. GWAS was performed under the assumption of disomic inheritance in reed canarygrass, which is suggested by the alloploidy of a closely related species, <em>Phalaris aquatica</em> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Carlson <em>et al.</em> 1996</a>). To perform statistically valid association tests, it was necessary to avoid false positives, not only by controlling for population structure and familial relatedness (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Yu <em>et al.</em> 2006</a>) but also by accounting for imputation uncertainty. Our study exemplifies the use of multiple imputation, initially proposed by <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rubin (1987)</a>, to account for this source of variability when making inferences. Classification tree models, shown by <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> (2012)</a> and <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rutkoski <em>et al.</em> (2013)</a> to be promising, were used under this framework to impute missing values without information on markersâ genomic location and order. After a description of the variability present in the panels under study, we examine the general behavior of inferences in multiple imputation compared to more traditional methods that would not fully account for imputation uncertainty. Then, we present our results of GWAS performed in a multiple-imputation framework and describe the significant markers in light of where they map onto the <em>B. distachyon</em> genome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Populations</h3><p id="p-9">Three panels were assessed in this study: two randomly segregating populations derived from biparental crosses (LP1 and LP2) comprising 177 and 189 clones, respectively (<a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/017533SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS1.pdf">Table S1</a>), and one association panel (AP) comprising 590 clones originating from North America and Europe (<a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS1.pdf">Table S1</a>). LP1 and LP2 were derived from two distinct crosses involving genotypes of WR00, a tetraploid cultivar originating from Wisconsin. In AP, four populations (AR Upland, Superior, PI-284179, and PI-236525) were accessions of <em>P. caesia</em> (hexaploid). The AP subset consisting of only the 550 <em>P. arundinaceae</em> clones is referred to as AP-4x.</p><p id="p-10">Clones were assessed in spaced-plant trials arranged in a Sets-in-Reps design with two replicates. The trials were performed in two locations: Arlington, Wisconsin (43.3Â°N, 89.4Â°W) and Ithaca, New York (42.6Â°N, 76.4Â°W). Soil type was Plano silt loam (fine-silty, mixed, mesic Typic Argiudoll) in Arlington and was Niagara silt loam (fine-silty, mixed, active, mesic Aeric Endoaqualf) in Ithaca.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-11">Traits related to disease resistance (Ds), morphology [standability (St); leaf width (LW); leaf length (LL); total stem count (STC); plant height (PH); full height (FH)], phenology [heading date (HD); anthesis date (AD)], and quality [dry matter percentage (DM); neutral detergent fiber (NDF); acid detergent fiber (ADF); NDF digestibility (NDFD); acid insoluble ash (AIA); Klason lignin (Lignin); crude protein (CP); Ash (ASH); calcium (Ca); chlorine (Cl); copper (Cu); iron (Fe); potassium (K); magnesium (Mg); manganese (Mn); sodium (Na); phosphorus (P); sulfur (S); zinc (Zn); glucose (GLC); galactose (GAL); xylose (XYL); arabinose (ARA); GLC conversion efficiency (GLC_Eff); XYL conversion efficiency (XYL_Eff); energy content (BTU)] were determined on plants grown in Ithaca in 2009â2011 and in Arlington in 2010â2011 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Quality traits were predicted from near-infrared reflectance spectroscopy (NIRS) measurement, using methodology described by <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Vogel <em>et al.</em> (2011)</a>. The NIRS prediction equations were developed on a diverse set of 110 reed canarygrass samples from the experiments described by <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Casler <em>et al.</em> (2009b)</a>. Wet-laboratory traits were determined for these samples using the procedures described by <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Vogel <em>et al.</em> (2011)</a>. The validity of predictions from the predictive models was verified by the extremely low frequency of outliers: nine biomass-quality samples with Mahalanobis distance more than three out of a total of &gt;3300 samples (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shenk and Westerhaus 1991</a>). The raw phenotypic data are available for download from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466002/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466002/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of field traits and quality traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">Association analyses were performed on best linear unbiased estimations (BLUEs) of genotypesâ performance for a given trait, inferred from the following linear mixed model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-2.gif"/></noscript></span></span> is the measurements at one of the 35 traits considered, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-3.gif"/></noscript></span></span> is the genotypic value of clone <em>i</em>, modeled as fixed (to guarantee convergence of the fitting algorithm and to avoid assumptions about genotypesâ sampling). For all other terms, the corresponding effects were considered random, independent, and identically normally distributed: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-4.gif"/></noscript></span></span> is the effect of location <em>j</em>; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-5.gif"/></noscript></span></span> is the effect of year <em>l</em> within location <em>j</em>; <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-6.gif"/></noscript></span></span> is the effect of replicate <em>m</em> within location <em>j</em>; Ã indicates interactions; and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-7.gif"/></noscript></span></span> is the effect of the plot at row <em>r</em> and column <em>c</em>, within an environment (year <em>l</em> within location <em>j</em>). Plot effects within environments were modeled as<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-8.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-9.gif"/></noscript></span></span> is the Kronecker product of the first-order autoregressive covariance matrices on rows and on columns, respectively. The mixed models were fitted using ASREML-R (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Butler <em>et al.</em> 2007</a>).</p><p id="p-16">The matrix of genotypesâ BLUEs, computed as described above, can be downloaded from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p></div><div id="sec-4" class="subsection"><h3>Marker data and quality control</h3><p id="p-17">Genome reduction, by <em>ApeKI</em> restriction, and sequencing were performed according to <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Elshire <em>et al.</em> (2011)</a>. Reduced DNA samples were sequenced on the Illumina HiSequation 2000, with 95 samples plus one negative control per lane. To simultaneously discover SNP markers and call genotypes, the UNEAK pipeline was used (<a href="http://www.maizegenetics.net/gbs-bioinformatics">http://www.maizegenetics.net/gbs-bioinformatics</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lu <em>et al.</em> 2013</a>). This pipeline trims reads to a 64-bp length to limit sequencing errors and speedup computation, and discards markers based on a network filter designed to detect and eliminate markers showing complex relationships with others, which suggests paralogy and/or sequencing errors (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lu <em>et al.</em> 2013</a>). A total of 29,313 SNPs were called out of the UNEAK pipeline. Marker genotypes were coded as allelic dosages: â1 for homozygotes at the reference allele, 0 for heterozygotes, and 1 for homozygotes at the alternate allele, assuming disomic inheritance in reed canarygrass (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Carlson <em>et al.</em> 1996</a>). The raw genotypic data are available for download from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p><p id="p-18">Markers were selected based on proportion of missing values (PMV) &lt;0.90 in each of three panels, separately. It is typical to filter out GBS SNPs by a predetermined low missing rate (<em>e.g.</em>, PMV &lt;0.20). However, we did not filter these markers to avoid removing potentially useful information, and also because a rigorous study of the behavior of the MI approach at different missing rates is merited. After this first filtering step, 18,818 markers were retained. Marker variables were then discarded if they met one of the following criteria: (i) being âconstantâ, <em>i.e.</em>, having a variance close to 0 (only 24 markers were discarded based on that criterion) or (ii) being âcollinearâ, <em>i.e.</em>, being correlated by at least 0.999 (in absolute value) to some other marker variable with a smaller amount of missing values in the dataset. This filtering step was recommended by <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">van Buuren and Groothuis-Oudshoorn (2011)</a> and was implemented in the mice R package. This avoided overly conservative tests in GWAS due to inadequate adjustment for multiple testing on highly correlated test statistics and also ensured that multiple imputations were not biased, as a result of strong correlations among predictors, and appropriately reflected imputation uncertainty. At this point, 6138 markers were considered for further analyses. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the distributions of PMV and MAF for marker data after the first filtering step (18,818 markers) and after the second filtering step (6138 markers). As expected, filtering for variability and noncollinearity preferentially discarded markers with high PMV and low MAF. Note that filtering out markers on PMV &lt;0.80 across panels (which is still a very lenient criterion) would have resulted in only 3419 markers being considered for subsequent analyses (<a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS2.pdf">Table S2</a>). Also, for markers not meeting the criteria of <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">van Buuren and Groothuis-Oudshoorn (2011)</a>, estimates of marker effects from averaged imputed data ((AD); see <em>Association analyses</em>) tended to show larger deviations from those obtained based on nonmissing data only ((CC); see <em>Association analyses</em>), no matter what the imputation uncertainty was (<a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/FigureS2.pdf">Figure S2</a>). Finally, effects of discarded markers as estimated from multiple imputes ((MI); see <em>Association analyses</em>) tended to show excessively strong shrinkage in comparison with (AD) estimates, with little adjustment of estimates in response to imputation uncertainty (<a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary statistics and distributions of proportions of missing values (PMV) and minor allele frequency (MAF) on markers retained after the first filtering step (&#x201C;on PMV only&#x201D;, i.e. PMV &lt; 0.9 by panel; 18,818 markers) or after both filtering steps [&#x201C;on PMV + variability and noncollinearity&#x201D;, i.e. PMV &lt; 0.9 by panel + filtering step recommended by van Buuren and Groothuis-Oudshoorn (2011), in which collinear and constant marker variables are discarded; 6138 markers]. Statistics are the mean, SD, and range for &#x201C;on PMV only&#x201D; / &#x201C;on PMV + variability and noncollinearity&#x201D;. Filtering for variability and noncollinearity preferentially discarded markers with high PMV and low MAF." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary statistics and distributions of proportions of missing values (PMV) and minor allele frequency (MAF) on markers retained after the first filtering step (&#x201C;on PMV only&#x201D;, &lt;em&gt;i.e.&lt;/em&gt; PMV &lt; 0.9 by panel; 18,818 markers) or after both filtering steps [&#x201C;on PMV + variability and noncollinearity&#x201D;, &lt;em&gt;i.e.&lt;/em&gt; PMV &lt; 0.9 by panel + filtering step recommended by van Buuren and Groothuis-Oudshoorn (2011), in which collinear and constant marker variables are discarded; 6138 markers]. Statistics are the mean, SD, and range for &#x201C;on PMV only&#x201D; / &#x201C;on PMV + variability and noncollinearity&#x201D;. Filtering for variability and noncollinearity preferentially discarded markers with high PMV and low MAF.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F1.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/891/F1.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Summary statistics and distributions of proportions of missing values (PMV) and minor allele frequency (MAF) on markers retained after the first filtering step (âon PMV onlyâ, <em>i.e.</em> PMV &lt; 0.9 by panel; 18,818 markers) or after both filtering steps [âon PMV + variability and noncollinearityâ, <em>i.e.</em> PMV &lt; 0.9 by panel + filtering step recommended by <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">van Buuren and Groothuis-Oudshoorn (2011)</a>, in which collinear and constant marker variables are discarded; 6138 markers]. Statistics are the mean, SD, and range for âon PMV onlyâ / âon PMV + variability and noncollinearityâ. Filtering for variability and noncollinearity preferentially discarded markers with high PMV and low MAF.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The high values of PMV among selected markers (78% of missing values on average) reflect low sequencing depth of the GBS. From the quality control step, the marker data matrix <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-10.gif"/></noscript></span></span> was produced, in which missing values were coded as NA. Matrix <strong><span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-11.gif"/></noscript></span></span></strong>, used as input to the imputation procedure, is available for download from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p></div><div id="sec-5" class="subsection"><h3>Marker data imputation</h3><div id="sec-6" class="subsection"><h4>General principle of multiple multivariate imputation:</h4><p id="p-21">Consider some incomplete data set <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-12.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-14.gif"/></noscript></span></span> denoting the observed and missing data, respectively. Given an estimand of interest <em>Î¸</em> (<em>e.g.</em>, a model parameter), multiple imputation (MI) aims at producing a correct Monte Carlo (MC) approximation of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-15.gif"/></noscript></span></span> from <em>m</em> imputes of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-16.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-17.gif"/></noscript></span></span> denoting some imputation of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-18.gif"/></noscript></span></span>. One implication of the MC approximation being correct is that, given an impute <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-19.gif"/></noscript></span></span>, the distribution <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-20.gif"/></noscript></span></span> is well-approximated, <em>i.e.</em>, the sampling model is correct. Another implication that is of particular concern when performing MI is that the distribution <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-21.gif"/></noscript></span></span> is also well-approximated, <em>i.e.</em>, the imputation procedure is proper (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rubin 1987</a>), which is defined as follows: let <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-22.gif"/></noscript></span></span> be an estimate of <em>Î¸</em> from a given impute, let <em>W</em> be the within-impute estimation variance of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-23.gif"/></noscript></span></span>, and let <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-24.gif"/></noscript></span></span> be the among-impute variance of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-25.gif"/></noscript></span></span> (corrected for a finite number of imputes): (i) <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-26.gif"/></noscript></span></span>, <em>i.e.</em>, the average of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-27.gif"/></noscript></span></span> over imputes <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-28.gif"/></noscript></span></span> is an unbiased estimator of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-29.gif"/></noscript></span></span>, the estimate of <em>Î¸</em> from the hypothetically complete dataset; (ii) <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-30.gif"/></noscript></span></span>; and (iii) <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-31.gif"/></noscript></span></span> (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Rubin 1987</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">van Buuren 2012</a>). Condition (i) implies correct assumptions regarding the imputation model and the source of missingness in the data. Typically, the missing-at-random assumption (no factor other than those accounted for in the imputation model caused missingness) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rubin and Little 2002</a>) is necessary to guarantee that condition (i) holds. Conditions (ii) and (iii) imply that inferences from MI are confidence-valid (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">van Buuren 2012</a>), <em>i.e.</em>, that the total variance of <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-32.gif"/></noscript></span></span> is realized in a conservative way: <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-33.gif"/></noscript></span></span>. In this study, we assess imputation models for unbiasedness, under specific assumptions [condition (i)], but we did not test imputation models for their ability to preserve variability in the data [condition (ii)] or to correctly reflect among-impute variability [condition (iii)].</p><p id="p-22">When performing MI on data sets containing missing values at several variables (SNP markers), one must sample from the joint distribution of missing values at all variables. To achieve this, two strategies have been proposed: joint modeling (JM) and fully conditional specification (FCS). JM consists of sampling from the joint distribution directly, by ordinary MC (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rubin and Schafer 1990</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Schafer 2010</a>). This method is theoretically sound, but it requires complex model specifications and assumptions that, if not correct, may result in imputation bias. JM cannot accommodate tree-based approaches that have the advantage of being flexible and not requiring any model specification. FCS, however, implicitly samples from some hypothetical joint distribution by repeatedly sampling from the fully conditional distributions at each variable of interest using a Gibbs sampling scheme (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">van Buuren <em>et al.</em> 2006</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">van Buuren 2007</a>). Because it relies on Markov chain MC (MCMC), FCS can be computationally costly. Also, there is no guarantee that the joint distribution actually exists, so FCS is usually described as a pseudo-Gibbs sampling procedure (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gelman and Raghunathan 2001</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">van Buuren <em>et al.</em> 2006</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">van Buuren 2007</a>). However, simulation studies have suggested that FCS is robust to such issues (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">van Buuren <em>et al.</em> 2006</a>). A critical incentive for using FCS rather than JM is that FCS allows for much more flexibility in the imputation procedure: each fully conditional distribution can be derived separately, using either parametric (<em>e.g.</em>, linear regression) or nonparametric models (<em>e.g.</em>, classification trees). This flexibility is particularly valuable when dealing with missing values at many variables, in which case JM may simply not be practical (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gelman and Raghunathan 2001</a>).</p></div><div id="sec-7" class="subsection"><h4>Implementation of multiple imputation:</h4><p id="p-23">In this study, we sampled missing values from <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-34.gif"/></noscript></span></span> by FCS. In the pseudo-Gibbs sampling process, missing values at the <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-35.gif"/></noscript></span></span> SNP were sampled from <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-36.gif"/></noscript></span></span> based on a classification and regression tree model (CART) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Breiman <em>et al.</em> 1984</a>). For unbiasedness, imputation of <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-37.gif"/></noscript></span></span> relied on the missing-completely-at-random (MCAR) assumption, stating that missingness occurred at random, with no factor causing SNP genotypes to be systematically missing (Rubin and <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Little 2002</a>). CART presents the advantages of not requiring explicit model specification and conveniently accommodating nonlinear effects.</p><p id="p-24">The general algorithm was:</p><p id="p-25">For impute <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-38.gif"/></noscript></span></span></p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-26">For <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-39.gif"/></noscript></span></span> fill in missing values at the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-40.gif"/></noscript></span></span> marker by random draws <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-41.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-42.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p>For <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-43.gif"/></noscript></span></span></p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-28">For <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-44.gif"/></noscript></span></span></p></li><li id="list-item-4"><p id="p-29">sample <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-45.gif"/></noscript></span></span> from<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-46.gif"/></noscript></span></span></p></li></ul></li><li id="list-item-5"><p id="p-30">Return <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-47.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-48.gif"/></noscript></span></span>,</p></li></ol><p id="p-31">where <em>m</em> is the total number of imputes of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-49.gif"/></noscript></span></span>, <em>q</em> is the number of marker variables, and <em>L</em> is the number of MCMC iterations.</p><p id="p-32">The number of imputes <em>m</em> was set to 20, based on available computational and memory resources. The number of iterations <em>L</em> up to which the actual sampling occurred (<em>i.e.</em>, the burn-in period) was set to 10, based on previous studies (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dai <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Burgette and Reiter 2010</a>) and available computational resources.</p><p id="p-33">MI, based on CART, was implemented according to <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Doove <em>et al.</em> (2014)</a> with packages mice (<a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">van Buuren and Groothuis-Oudshoorn 2011</a>) and rpart (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Therneau and Atkinson 1997</a>) in the R programming language (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">R Development Core Team 2014</a>). CARTs were fitted with pruning up to at least five donors per terminal node (<em>i.e.</em>, no less than five observations at each leaf in the tree). For the MI algorithm to be computationally tractable, only a subset of SNPs was considered for fitting the CART models: for each SNP <em>k</em>, only the 500 SNPs showing the highest marginal mutual information with SNP <em>k</em> were considered as potential predictors.</p><p id="p-34">MI implemented as described above took 1 d per imputation (chain of 10 iterations) on a workstation consisting of 24 Intel Xeon X7460 CPU processors at 2.66 GHz with 264 Gb of RAM. The procedure was parallelized on five <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-50.gif"/></noscript></span></span> threads.</p><p id="p-35">Two types of marker data were generated from the multiple imputation procedure: the MI set of matrices <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-51.gif"/></noscript></span></span> and the average-dosage (AD) matrix <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-52.gif"/></noscript></span></span> (<em>i.e.</em>, each element in <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-53.gif"/></noscript></span></span> was the mean of genotype codes across the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-54.gif"/></noscript></span></span> imputes). The array <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-55.gif"/></noscript></span></span> and the matrix <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-56.gif"/></noscript></span></span> from the imputation procedure can be downloaded from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p></div></div><div id="sec-8" class="subsection"><h3>Association analyses</h3><div id="sec-9" class="subsection"><h4>Data subsets:</h4><p id="p-36">Only tetraploid samples were considered for GWAS, and the following subsets of the data were examined separately when testing marker-trait associations: LP1, LP2, and AP-4x (AP panel with only <em>P. arundinacea</em> samples) consisting of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-57.gif"/></noscript></span></span> individuals, respectively. After discarding marker variables with MAF &lt;0.05 (as estimated from <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-58.gif"/></noscript></span></span>), in each subset separately, 5024, 5096, and 5228 markers were assayed in association testing, respectively. <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/FigureS1.pdf">Figure S1</a> shows, for each subset separately, the distribution of PMV and MAF for markers selected prior to conducting GWAS. The distributions are equivalent across subsets and similar to that observed with the larger set of 6138 markers, with the exception that the average MAF increased from 0.28 to 0.32.</p></div><div id="sec-10" class="subsection"><h4>Association model:</h4><p id="p-37">Within each of the three data subsets (LP1, LP2, and AP-4x), the linear mixed model of <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Yu <em>et al.</em> (2006)</a> was fitted for each SNP <em>k</em> retained, using the P3D (population parameters previously determined) approximation for computational efficiency (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kang <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-59.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-60.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-61.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-62.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-63.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-64.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-65.gif"/></noscript></span></span> is the <em>n</em>-vector of clonesâ genotypic values as described above; <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-66.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-67.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-68.gif"/></noscript></span></span> are the vectors of allelic dosage for SNP <em>k</em> and correspond to the three types of marker data described above; <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-69.gif"/></noscript></span></span> is the effect of SNP <em>k</em>; <strong>Q</strong> is the matrix of the first <em>t</em> components used to account for population structure (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Price <em>et al.</em> 2006</a>) obtained from a principal component analysis (PCA) performed either on <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-70.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-71.gif"/></noscript></span></span>; <strong>v</strong> is the vector of their effects; <strong>Z</strong> is the design matrix for relating observations to clones; <strong>u</strong> is the vector of random polygenic effects; <strong>K</strong> is the realized genetic relationship matrix estimated from either <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-72.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-73.gif"/></noscript></span></span>; <strong>e</strong> is the vector of residuals; and <strong>I</strong> is the identity matrix. <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-74.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-75.gif"/></noscript></span></span> were estimated by restricted maximum likelihood (REML). The R package rrBLUP (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Endelman 2011</a>) was used to calculate the realized relationship matrix <strong>K</strong> and estimate <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-77.gif"/></noscript></span></span>. Subscript <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-78.gif"/></noscript></span></span> refers to the subset of individuals for which there was no missing value at SNP <em>k</em>.</p><p id="p-38">(CC) is the analysis restricted to complete cases (nonmissing values) for a given marker tested, with the approximation of <strong>Q</strong> and <strong>K</strong> estimated from <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-79.gif"/></noscript></span></span> used as fixed, to account for population structure and relatedness. In (AD), marker effects are assessed from <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-80.gif"/></noscript></span></span>, considered as fixed. In (MI) and (MI*), marker effects are assessed from <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-81.gif"/></noscript></span></span> and the variability across imputes <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-82.gif"/></noscript></span></span> is accounted for. Relying on the MCAR assumption, (CC) served as a reference for assessing the consistency of estimates from (AD) or (MI): (AD) estimates departing too much from (CC) estimates were considered unreliable, especially when imputation uncertainty (<em>Î³</em>; see below) was high. Estimates from (MI) were expected to show appropriate control over imputation uncertainty, <em>i.e.</em>, shrinkage toward zero as <em>Î³</em> increases. In (MI*), marker effects are assessed from <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-83.gif"/></noscript></span></span> as in (MI), but variability across imputes for estimates of <strong>Q</strong> and <strong>K</strong> is also accounted for, through <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-84.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-85.gif"/></noscript></span></span>, thus allowing for full control over imputation uncertainty in inferences regarding <em>Î²</em>. Although (CC), (AD), and (MI) are convenient to assess the impact of imputation uncertainty on <em>Î²</em> estimates, (MI*) should be the method permitting the most statistically sound inferences regarding marker-trait associations.</p><p id="p-39">In AP-4x, population structure and relatedness were, respectively, accounted for by one principal component and by matrix <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-86.gif"/></noscript></span></span> calculated on AP-4x individuals. In LP1 and LP2, no principal component and no genetic relationship matrix were included in the GWAS model, then equivalent to a single-marker analysis model.</p></div><div id="sec-11" class="subsection"><h4>Combination of parameter estimates in MI and calculation of p values:</h4><p id="p-40">In (CC) and (AD), significance of marker-trait associations was assessed by performing an F-test for the regression coefficient estimate <em>Î²</em>, as described in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kang <em>et al.</em> (2008)</a>. In (MI) and (MI*), an F-test was performed in which the <em>p</em> value was (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rubin and Schenker 1986</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">van Buuren 2012</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-87.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-88.gif"/></noscript></span></span> is the average estimate of<em>Î²</em> over imputations, with <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-89.gif"/></noscript></span></span> being the estimate of <em>Î²</em> based on the <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-90.gif"/></noscript></span></span> imputation; <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-91.gif"/></noscript></span></span> is the (estimated) total variance of <em>Î²</em> estimates, partitioned into <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-92.gif"/></noscript></span></span>, the average within-impute variance of <em>Î²</em> estimates, and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-93.gif"/></noscript></span></span>, the among-impute variance of <em>Î²</em> estimates, corrected by <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-94.gif"/></noscript></span></span> for unbiasedness (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-95.gif"/></noscript></span></span>means âasymptotically equalâ). <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barnard and Rubin (1999)</a> derived a formula for the number of denominator degrees of freedom <em>v</em> of the F-statistic:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-96.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-97.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-98.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-99.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-100.gif"/></noscript></span></span> being the number of degrees of freedom for the hypothetically complete dataset.</p><p id="p-41">For a particular coefficient <em>Î²</em>, the uncertainty in its estimate due to imputation is characterized by <em>Î³</em>, the fraction of information about <em>Î²</em> lacking due to missingness:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-101.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-102.gif"/></noscript></span></span> (<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Rubin 1987</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">van Buuren 2012</a>). <em>Î³</em> reflects the dependency of the inferences about <em>Î²</em> on the imputation procedure. Although quite complex, the <em>Î³</em>-statistic may simply be interpreted as <em>Î»</em>, the proportion of among-impute variance in inferences, adjusted for a finite number of imputes (<a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">van Buuren 2012</a>). In this study, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-103.gif"/></noscript></span></span>. According to <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> (1991)</a>, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-104.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-105.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-106.gif"/></noscript></span></span> indicate a âmodest,â âmoderately large,â and âhighâ missing data problem, respectively.</p></div><div id="sec-12" class="subsection"><h4>False discovery rate and significance for marker-trait association:</h4><p id="p-42">The <em>p</em> values obtained from (CC), (AD), or (MI) were transformed into adjusted false discovery rates (FDR) using the method of <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Storey and Tibshirani (2003)</a>. Marker-trait associations for which FDR &lt;0.1 in (CC) or (MI*) were deemed significant and considered for further analyses, although evidence from (MI*) was preferred because (CC) relied on the approximation of <strong>Q</strong> and <strong>K</strong> estimated from <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-107.gif"/></noscript></span></span> used as fixed in the GWAS model and also because (CC) was considered more prone to false positives or false negatives due to smaller sample sizes, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-108.gif"/></noscript></span></span> (see <em>Discussion</em>). The R package qvalue (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dabney <em>et al.</em> 2013</a>) was used to compute FDR.</p></div></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Genotypic variability in the panels</h3><div id="sec-15" class="subsection"><h4>Phenotypic traits:</h4><p id="p-43">Traits were measured in varying numbers of years and locations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Some traits were measured in only one location (Ds, TSC, St, and Quality traits in AP, measured/predicted in Ithaca only) and/or during only 1 yr (Ds only in 2009 and Quality traits only in 2011). As a result, they would show an inflated genotypic variance. Ds, St, LL, LW, and all quality traits showed a much higher ratio of genotypic-to-phenotypic variance (H) in AP compared to LP1 and LP2. PH showed a higher H in LP1 and AP compared to LP2. AD showed a higher H in LP2 and AP compared to LP1 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466056/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466056/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics on the field traits (9) and quality traits (26) in each panel</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Population structure:</h4><p id="p-46">To analyze population structure, principal component analysis (PCA) was performed on the average dosage matrix <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-115.gif"/></noscript></span></span>. For the whole dataset (panels LP1, LP2, and AP combined), the first four principal components (PC) were deemed relevant for describing population structure based on proportions of variance explained and grouping patterns on PCs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Grouping patterns in the whole dataset are consistent with race (first PC, distinguishing accessions that are <em>Phalaris arundinacea</em> from those that are <em>Phalaris caesia</em>), panel (second and third PCs), and geographical originâEastern Europe <em>vs.</em> North America or Western Europe (fourth PC). The fifth and sixth PCs do not seem to reflect any grouping and explain a small proportion of the total genotypic variation compared with the first four PCs.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis (PCA) on the three panels combined: first six components and the proportion of marker variation explained, in parentheses on axis labels. Colors refer to location of origin (as in Table S1). Shapes refer to the panel (as in Table S1). Data points for P. caesia clones are circled in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis (PCA) on the three panels combined: first six components and the proportion of marker variation explained, in parentheses on axis labels. Colors refer to location of origin (as in Table S1). Shapes refer to the panel (as in Table S1). Data points for &lt;em&gt;P. caesia&lt;/em&gt; clones are circled in black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F2.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/891/F2.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466149" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-47" class="first-child">Principal component analysis (PCA) on the three panels combined: first six components and the proportion of marker variation explained, in parentheses on axis labels. Colors refer to location of origin (as in <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS1.pdf">Table S1</a>). Shapes refer to the panel (as in <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/TableS1.pdf">Table S1</a>). Data points for <em>P. caesia</em> clones are circled in black.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">PCA was also performed for four other subsets of the data: LP1, LP2, AP, and AP-4x (AP panel with only <em>P. arundinacea</em> samples). The numbers of selected PCs for those subsets were 0, 0, 2, and 1, respectively (data not shown). The relevant PCs in AP were consistent with race and geographical origin, respectively, in the same way as in the whole data set. The relevant PC in AP-4x was consistent with geographical origin.</p></div></div><div id="sec-17" class="subsection"><h3>Analysis of imputation uncertainty</h3><div id="sec-18" class="subsection"><h4>Consistency of PCs and genetic relationship coefficients across imputes:</h4><p id="p-49">Consistency of a variable in MI is defined as the correlation with the same variable in (AD), in absolute value, averaged over imputes. Such value aims at assessing to what extent it is appropriate to use <strong>Q</strong> and <strong>K</strong>, estimated from <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-116.gif"/></noscript></span></span>, as fixed, in (CC), (AD), and (MI) association analyses. In the whole dataset, <strong>K</strong> estimates seem very consistent over imputes (consistency of 0.90 Â± 0.0002; <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), but although estimates of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-117.gif"/></noscript></span></span> (first PC) are quite consistent, the estimates at the subsequent PCs lose coherence across imputes, with the fourth PC having a consistency of only 0.043 (Â± 0.032). This result suggests that accounting for imputation uncertainty with respect to population structure variablesâwhen those are estimated from marker dataâis important, which implies that (MI*) should be the most appropriate type of analysis for assessing marker-trait associations. In the AP-4x subset, the consistency of the only relevant PC is 0.076 (Â± 0.067) and the consistency of genetic relationship coefficients is 0.934 (Â± 0.0002).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465932/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465932/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Consistency of population structure and genetic relationship variables across imputes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Consistency across imputation schemes of marker effects and significance:</h4><div id="sec-20" class="subsection"><h5>âFrom complete case (CC) to average dosage (AD):</h5><p id="p-51">Under the MCAR assumption, the relationship between <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-131.gif"/></noscript></span></span> and the trait of interest adequately reflects the effect of markers in the whole sample. That is, for any marker-trait association, the estimate <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-132.gif"/></noscript></span></span> based on complete cases is unbiased with respect to <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-133.gif"/></noscript></span></span>, the estimate of <em>Î²</em> based on the hypothetically complete data set. It follows from the MCAR assumption that imputations are unbiased if estimates of marker effects from (AD), <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-134.gif"/></noscript></span></span>, are unbiased with respect to those from (CC), <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-135.gif"/></noscript></span></span>. The plot (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and the regression analysis (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) of <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-136.gif"/></noscript></span></span> on <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-137.gif"/></noscript></span></span>, across traits and markers, strongly suggest that there is no bias from (CC) to (AD). For marker-trait associations with low imputation uncertainty <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-138.gif"/></noscript></span></span>, there is a close relationship between <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-139.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-140.gif"/></noscript></span></span> (R<sup>2</sup> = 0.94). However, for marker-trait associations with moderate-to-high imputation uncertainty <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-141.gif"/></noscript></span></span>, large differences between <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-143.gif"/></noscript></span></span> can be observed, but at random. Such differences indicate noise in inferred values of <em>Î²</em> and opportunities for false positives or false negatives (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance in inferences from (CC) to (AD) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log10(p)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (p &lt; 9.56 &#xD7; 10&#x2212;6)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Concordance in inferences from (CC) to (AD) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (&lt;em&gt;p&lt;/em&gt; &lt; 9.56 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F3.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/891/F3.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465983" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-52" class="first-child">Concordance in inferences from (CC) to (AD) in the AP-4x subset of the data. The pointsâ size refers to the stability of inferences across imputes (1âÎ³). (A) Concordance across procedures and over all traits in marker effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [âlog<sub>10</sub>(<em>p</em>)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (<em>p</em> &lt; 9.56 Ã 10<sup>â6</sup>).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466100/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466100/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Regression analysis of the relationship between <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T4/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T4/embed/mml-math-144.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T4/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T4/embed/mml-math-145.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h5>From average dosage (AD) to multiple imputation (MI):</h5><p id="p-54">The purpose of MI is to make sound inferences when one bases statistical analyses on imputed data. Therefore, MI should produce more conservative estimates <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-154.gif"/></noscript></span></span> when imputation uncertainty regarding marker-trait associations is high, in order to avoid declaring as significant associations that are due to âfortuitousâ imputations. The plot (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) and the regression analysis (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) of <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-155.gif"/></noscript></span></span> on <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-156.gif"/></noscript></span></span>, across traits and markers, show that as <em>Î³</em> increases, estimates of <em>Î²</em> tend to shrink toward zero, from (AD) to (MI). This behavior results in higher <em>p</em> values (lower significance) for those associations in which imputation uncertainty is high: whether associations inferred in (AD) are true or false, their <em>p</em> values will tend to fall above the significance threshold in (MI) if their among-impute variability is too large, but there is good agreement between (AD) and (MI) for low values of <em>Î³</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Only for <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-157.gif"/></noscript></span></span> (no variation among impute) would there be no adjustment on <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-159.gif"/></noscript></span></span>. In other words, one should rely on (AD) for inferences only if it can be assumed that the situation is âclose enoughâ to the case where <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-160.gif"/></noscript></span></span>. Clearly, this is not the case for these data. In <a href="http://www.g3journal.org/content/suppl/2015/03/12/g3.115.017533.DC1/FigureS4.pdf">Figure S4</a>, the consistency of marker effects from (CC) to (MI) is shown.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance in inferences from (AD) to (MI) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log10(p)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (p &lt; 9.56 &#xD7; 10&#x2212;6)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Concordance in inferences from (AD) to (MI) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (&lt;em&gt;p&lt;/em&gt; &lt; 9.56 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/891/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466055" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">Concordance in inferences from (AD) to (MI) in the AP-4x subset of the data. The pointsâ size refers to the stability of inferences across imputes (1âÎ³). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [âlog<sub>10</sub>(<em>p</em>)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (<em>p</em> &lt; 9.56 Ã 10<sup>â6</sup>).</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465936/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465936/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Regression analysis of the relationship between <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T5/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T5/embed/mml-math-161.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T5/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/T5/embed/mml-math-162.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h5>From multiple imputation (MI) to multiple imputation with full account of imputation uncertainty (MI<sup>*</sup>):</h5><p id="p-57">In (MI*), the marker data matrix <strong>X</strong>, but also the principal component and the genetic relationship matrices <strong>Q</strong> and <strong>K</strong> (both estimated from <strong>X</strong>) are allowed to vary over imputes. Although considering <strong>Q</strong> and <strong>K</strong> as fixed (estimated by <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-172.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-173.gif"/></noscript></span></span><strong>)</strong> was convenient for studying the behavior of inferences from (CC) to (AD) to (MI), (MI*) should permit the safest inferences by appraising the imputation uncertainty in the imputed genotypes as well as in the resulting <strong>Q</strong> and <strong>K</strong> estimates. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> suggests concordance of <em>Î²</em> estimates from (MI) to (MI*), despite the relatively low consistency of <strong>Q</strong> estimates across imputes (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Also, marker effects with low imputation uncertainty seem less shrunk toward zero in (MI*), where the variability in <strong>Q</strong> and <strong>K</strong> estimates is accounted for. As a result, more significant associations could be detected in (MI*) than in (MI) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Values of <em>Î³</em> were quite consistent from (MI) to (MI*), with increased noise around perfect concordance for values of <em>Î³</em> above 0.2 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance in inferences from (MI) to (MI*) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log10(p)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (p &lt; 9.56 &#xD7; 10&#x2212;6)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Concordance in inferences from (MI) to (MI*) in the AP-4x subset of the data. The points&#x2019; size refers to the stability of inferences across imputes (1&#x2212;&#x3B3;). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [&#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (&lt;em&gt;p&lt;/em&gt; &lt; 9.56 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F5.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/891/F5.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466053" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-58" class="first-child">Concordance in inferences from (MI) to (MI*) in the AP-4x subset of the data. The pointsâ size refers to the stability of inferences across imputes (1âÎ³). (A) Concordance across procedures and over all traits in marker-effect estimates, standardized within each trait. (B) Concordance across procedures and over all traits in significance [âlog<sub>10</sub>(<em>p</em>)]; green areas correspond to significant associations according to a Bonferroni correction on a single-trait basis (<em>p</em> &lt; 9.56 Ã 10<sup>â6</sup>).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance in imputation uncertainty, as reflected by the &#x3B3;-statistic, over all traits in the AP-4x subset from (MI*) to (MI). The blue dashed lines correspond to the seemingly critical threshold of &#x3B3; &gt; 0.2, beyond which the &#x3B3;-statistic loses coherence across procedures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Concordance in imputation uncertainty, as reflected by the &#x3B3;-statistic, over all traits in the AP-4x subset from (MI*) to (MI). The blue dashed lines correspond to the seemingly critical threshold of &#x3B3; &gt; 0.2, beyond which the &#x3B3;-statistic loses coherence across procedures.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F6.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/5/891/F6.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465979" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-59" class="first-child">Concordance in imputation uncertainty, as reflected by the Î³-statistic, over all traits in the AP-4x subset from (MI*) to (MI). The blue dashed lines correspond to the seemingly critical threshold of Î³ &gt; 0.2, beyond which the Î³-statistic loses coherence across procedures.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-23" class="subsection"><h4>Imputation uncertainty and potential gains in power:</h4><p id="p-60">As <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a> suggests, for <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-174.gif"/></noscript></span></span> (<em>i.e.</em>, markers with a modest missing data problem) there were generally small differences in significance <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-175.gif"/></noscript></span></span> from (CC) to (MI*), but some opportunity for a few outstanding gains in significance, with increases of up to 8.08 in <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-176.gif"/></noscript></span></span> (for the association between marker TP87762 and TSC; see next paragraph and <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). Assuming that the associations detected are true positives, this increase in significance may be considered an increase in power. This higher sensitivity in GWAS would logically come from a higher effective sample size due to imputation, with a modest missing data problem at the same time. For <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-177.gif"/></noscript></span></span>, however, there was little benefit from imputation, with very few opportunities for higher sensitivity from (CC) to (MI*): most values for the differential in <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-178.gif"/></noscript></span></span> are actually negative. Those results show again that a threshold of 0.2 for <em>Î³</em> is a good guideline for setting apart inferences with excessively high imputation uncertainty.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Relationship between &#x3B3;, in (MI*), and the difference in significance from (CC) to (MI*) [&#x394;{&#x2212;log10(p)}]. Decrease in significance [&#x2212;log10(p)] seems to occur more often and with higher intensity for &#x3B3; &gt; 0.2; presumably, there would to be more opportunities for gaining detection power with &#x3B3; &lt; 0.2. (B&#x2013;D) Potential factors affecting imputation uncertainty (&#x3B3;): relationship between &#x3B3;, in (MI*), and (B) PMV (proportion of missing values), (C) MAF (minor allele frequency), and (D) the average mutual information between one given marker and all other markers in the dataset. In purple are the smoothed curves obtained from thin plate regression (default smoother in mgcv R package; Wood 2003). The &#x3B3;-statistics and p values are from analyses on the AP-4x subset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Relationship between &#x3B3;, in (MI*), and the difference in significance from (CC) to (MI*) [&#x394;{&#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;)}]. Decrease in significance [&#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;)] seems to occur more often and with higher intensity for &#x3B3; &gt; 0.2; presumably, there would to be more opportunities for gaining detection power with &#x3B3; &lt; 0.2. (B&#x2013;D) Potential factors affecting imputation uncertainty (&#x3B3;): relationship between &#x3B3;, in (MI*), and (B) PMV (proportion of missing values), (C) MAF (minor allele frequency), and (D) the average mutual information between one given marker and all other markers in the dataset. In purple are the smoothed curves obtained from thin plate regression (default smoother in mgcv R package; Wood 2003). The &#x3B3;-statistics and &lt;em&gt;p&lt;/em&gt; values are from analyses on the AP-4x subset.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F7.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/5/891/F7.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-61" class="first-child">(A) Relationship between Î³, in (MI*), and the difference in significance from (CC) to (MI*) [Î{âlog<sub>10</sub>(<em>p</em>)}]. Decrease in significance [âlog<sub>10</sub>(<em>p</em>)] seems to occur more often and with higher intensity for Î³ &gt; 0.2; presumably, there would to be more opportunities for gaining detection power with Î³ &lt; 0.2. (BâD) Potential factors affecting imputation uncertainty (Î³): relationship between Î³, in (MI*), and (B) PMV (proportion of missing values), (C) MAF (minor allele frequency), and (D) the average mutual information between one given marker and all other markers in the dataset. In purple are the smoothed curves obtained from thin plate regression (default smoother in mgcv R package; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wood 2003</a>). The Î³-statistics and <em>p</em> values are from analyses on the AP-4x subset.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Potential factors influencing imputation uncertainty:</h4><p id="p-62">Because there is good consistency between <em>Î³</em> and the apparent usefulness of imputation for increasing power, it would be helpful to identify the factors that allow the analyst to effectively predict <em>Î³</em> and determine how likely imputation is to generate gains in power. <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, BâD</a>, respectively, show the marginal relationships between <em>Î³</em> and proportion of missing values (PMV), MAF, or the average mutual information (AMI) between given markers and all other markers in the data set. There seems to be a strong positive relationship between <em>Î³</em> and PMV, with values of <em>Î³</em> that are below 0.2 for markers that have up to approximately 25% of missing values. As PMV increases, the slope for <em>Î³</em> decreases and the variability around the conditional mean, as determined by a smoothing curve, increases. There seems to be a weak correlation between <em>Î³</em> and MAF, with markers having high MAF showing slightly higher imputation uncertainty, especially for MAF &lt;0.1. However, this relationship might be an artifact from markers with low PMV, which tended, by happenstance, to have lower MAF. There seems to be some (nonlinear) relationship between <em>Î³</em> and AMI, but it is not clear whether it is due to the fact that markers with very low AMI tended to have lower PMV on average: for AMI &lt;0.02, average PMV is 0.45 (SD 0.32); for AMI &gt;0.02, average PMV is 0.80 (SD 0.071). A simple additive model fitted to the data with <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-179.gif"/></noscript></span></span> regressed on PMV, MAF, and AMI suggests that that all three factors considered have a significant effect on imputation uncertainty: MAF (usually equivalent to marker genotype variance) andâto a much larger extentâPMV generate imputation uncertainty, whereas AMI reduces imputation noise (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). The model fitted has some predictive value (r<sup>2</sup><sub>CV</sub>, prediction reliability in 10-fold cross-validation, was 0.21), but a rather large part of the variation could not be accounted for (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Regression analysis on potential factors affecting imputation uncertainty</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-25" class="subsection"><h3>Association analyses</h3><div id="sec-26" class="subsection"><h4>Significance of marker-trait associations:</h4><p id="p-64">Significant associations, for which FDR &lt;0.1 in (CC) or (MI*), were detected in the AP-4x subset and in the LP2 subset; no significant association was detected in LP1. These involved nine markers (thereafter âsignificant markersâ): TP140584; TP184396; TP191264; TP217634; TP268059; TP341988; TP477925; TP521945; and TP87762. The one association involving TP341988 was detected only in LP2 (TP341988 was not included in the analysis in AP-4x due to the threshold of MAF &gt;0.05), whereas the others were detected only in AP-4x. As shown in <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>, behavior of inferences from (CC) to (MI*) was highly dependent on <em>Î³</em>: for values of <em>Î³</em> above 0.3 (associations involving TP184396, TP191264, TP217634, TP268059, TP341988, and TP521945), marker effects estimated from (MI*) were closer to zero and significance of associations decreased, certainly as a result of the shrinkage of marker effects as well as the high among-impute variance. These marker-trait associations that lost significance from (CC) to (MI*) generally had a high PMV (above 0.69). However, associations with <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-187.gif"/></noscript></span></span> (involving TP140584, TP477925, and TP87762), characterized by lower PMV (below 0.34), were more prone to show higher significance from (CC) to (MI*), with similar (sometimes larger) estimated marker effects. In some cases, such as TP140584-TSC, TP477925-ARA, TP477925-Mg, TP477925-Ds, TP477925-P, TP477925-TSC,TP87762-ARA, and TP87762-Mg, associations were deemed significant in (MI*) but not in (CC). Such behavior suggests an increase in detection power despite the uncertainty associated with imputation. Those associations generally show consistency from (CC) to (MI*), which would bring evidence for unbiasedness of the imputation procedure. However, for associations TP477925-Mn <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-188.gif"/></noscript></span></span> and TP477925-Cu <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-189.gif"/></noscript></span></span>, marker effects were estimated to be weaker in (MI*). Under the assumption that the imputation procedure is proper, such results suggest that the associations detected in (CC) were actually false positives.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465964/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465964/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Results of association analyses based on (CC) and (MI*) procedures</span><div class="sb-div caption-clear"></div></div></div><p id="p-67">The good overall concordance between observed and expected quantiles of <em>p</em> values for (CC) suggests that potential confounders (due to population structure and relatedness in AP-4x) were well accounted for in the GWAS models (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Regarding (MI*), the strong overall deflation of <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-192.gif"/></noscript></span></span> suggests that <em>p</em> values simply do not follow a <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-193.gif"/></noscript></span></span> distribution under the null hypothesis (no significant marker-trait association) because of the extra variability due to imputation uncertainty, which was large for the majority of markers (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Q-Qplot of p values from (CC), for St, TSC, AD, CP, Lignin (in AP-4x subset), and GLC (in LP2 subset), with corresponding p values from (MI*). Expected quantiles are from the (CC) analysis; the corresponding values from (MI*) are shown unordered to reflect how consistent quantiles are across analyses. Open circles, &#x2212;log10(p) based on (CC); full circles, &#x2212;log10(p) based on (MI*). Green symbols indicate associations for which FDR " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Q-Qplot of &lt;em&gt;p&lt;/em&gt; values from (CC), for St, TSC, AD, CP, Lignin (in AP-4x subset), and GLC (in LP2 subset), with corresponding &lt;em&gt;p&lt;/em&gt; values from (MI*). Expected quantiles are from the (CC) analysis; the corresponding values from (MI*) are shown unordered to reflect how consistent quantiles are across analyses. Open circles, &#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;) based on (CC); full circles, &#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;p&lt;/em&gt;) based on (MI*). Green symbols indicate associations for which FDR &lt;0.1. Gray areas correspond to the 95% C.I. of quantiles under the null hypothesis that &lt;em&gt;p&lt;/em&gt; values follow a &lt;em&gt;Uniform&lt;/em&gt;(0,1). The &lt;em&gt;p&lt;/em&gt; values from (CC) seem to follow a &lt;em&gt;Uniform&lt;/em&gt;(0,1), indicating good control for potential confounders (population structure and relatedness). However, &lt;em&gt;p&lt;/em&gt; values from (MI*) do not follow a &lt;em&gt;Uniform&lt;/em&gt;(0,1) because of losses of significance caused by imputation uncertainty.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F8.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/5/891/F8.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465998" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-68" class="first-child">Q-Qplot of <em>p</em> values from (CC), for St, TSC, AD, CP, Lignin (in AP-4x subset), and GLC (in LP2 subset), with corresponding <em>p</em> values from (MI*). Expected quantiles are from the (CC) analysis; the corresponding values from (MI*) are shown unordered to reflect how consistent quantiles are across analyses. Open circles, âlog<sub>10</sub>(<em>p</em>) based on (CC); full circles, âlog<sub>10</sub>(<em>p</em>) based on (MI*). Green symbols indicate associations for which FDR &lt;0.1. Gray areas correspond to the 95% C.I. of quantiles under the null hypothesis that <em>p</em> values follow a <em>Uniform</em>(0,1). The <em>p</em> values from (CC) seem to follow a <em>Uniform</em>(0,1), indicating good control for potential confounders (population structure and relatedness). However, <em>p</em> values from (MI*) do not follow a <em>Uniform</em>(0,1) because of losses of significance caused by imputation uncertainty.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h4>Correlation among significant markers:</h4><p id="p-69">In the absence of haplotypic information, the squared correlation between markersâ allelic dosages <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-194.gif"/></noscript></span></span> was used to reflect linkage disequilibrium (LD) between markers. The <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-195.gif"/></noscript></span></span> values were calculated based on <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-196.gif"/></noscript></span></span>. Among the nine significant markers, there seemed to be one group of four markers in moderately strong association with each other (TP140584, TP217634, TP477925, and TP87762) and one group of two markers in mild association (TP521945 and TP268059) (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). This grouping is consistent with the associations inferred from GWAS (<em>i.e.</em>, markers in one group tend to be associated to similar traits; <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>), except for TP217634, which is not associated with the same traits as TP140584, TP477925, or TP87762.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/891/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Squared coefficients of correlation among significant markers based on  (below diagonal elements). Above diagonal elements indicate significance; *: p &lt; 0.05; **: p &lt; 0.01; ***: p &lt; 0.001. TP140584, TP217634, TP477925, and TP87762 form a group of fairly correlated marker loci. TP521945 and TP268059 are in very mild correlation with each other. TP191264, TP184396, and TP341988 each seem independent of other significant markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1402902453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Squared coefficients of correlation among significant markers based on &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-197&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (below diagonal elements). Above diagonal elements indicate significance; *: &lt;em&gt;p&lt;/em&gt; &lt; 0.05; **: &lt;em&gt;p&lt;/em&gt; &lt; 0.01; ***: &lt;em&gt;p&lt;/em&gt; &lt; 0.001. TP140584, TP217634, TP477925, and TP87762 form a group of fairly correlated marker loci. TP521945 and TP268059 are in very mild correlation with each other. TP191264, TP184396, and TP341988 each seem independent of other significant markers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/891/F9.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/5/5/891/F9.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/891/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/891/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466161" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-70" class="first-child">Squared coefficients of correlation among significant markers based on <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/F9/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/F9/embed/mml-math-197.gif"/></noscript></span></span> (below diagonal elements). Above diagonal elements indicate significance; *: <em>p</em> &lt; 0.05; **: <em>p</em> &lt; 0.01; ***: <em>p</em> &lt; 0.001. TP140584, TP217634, TP477925, and TP87762 form a group of fairly correlated marker loci. TP521945 and TP268059 are in very mild correlation with each other. TP191264, TP184396, and TP341988 each seem independent of other significant markers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h4>Homology of SNP 64-bp sequences with the Brachypodium distachyon genome:</h4><p id="p-71">For each marker, a 64-bp read containing the corresponding SNP was obtained from the UNEAK pipeline. The reads corresponding to the nine significant markers were analyzed for homology with the <em>Brachypodium distachyon</em> genome (v1.0) (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). Three of the nine marker reads significantly match regions of the <em>Brachypodium distachyon</em> genome: (i) TP184396, shown to be negatively associated to Lignin (the nonreference allele has a negative effect on the trait), is in a region homologous to the last intron of an arginine-tRNA ligase gene (<em>e</em>-value = 1.2E-6); (ii) TP268059, shown to be negatively and positively associated with CP and XYL_Eff, respectively, is in a region homologous to the only exon in a phenylalanine/histidine-ammonia-lyase gene (<em>e</em>-value = 3.4E-12); and (iii) TP341988, shown to be negatively associated with GLC in LP2, is in a region homologous to the second exon of a translation elongation factor G gene (<em>e</em>-value = 4.3E-19). The moderate significance of TP184396's alignment to the <em>B. distachyon</em> genome may be due to the fact that the marker is located in an intron, which is usually more likely than exons to be under low selection pressure and diverge substantially across species.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466040/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466040/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Sequence information on significant markers</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-29"><h2 class="">Discussion</h2><div id="sec-30" class="subsection"><h3>Genotype calling uncertainty</h3><p id="p-73">Genotype calling uncertainty in GBS typically arises from sequencing error, which generates miscalls in SNP alleles, and low sequencing depth, which causes heterozygotes to be miscalled as homozygotes. Genotype calling uncertainty may be accounted for by applying cut-offs on proportion of reads (<em>e.g.</em>, calling individuals homozygous for a SNP if &gt;80% of their reads is of a particular allele) or probabilistic methods (returning posterior probabilities of genotypes), which rely on some estimates of error rate and population allele frequency at SNPs (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nielsen <em>et al.</em> 2011</a>). In this study, sequencing depth was very low; there were, on average, 0.29 reads per SNP called (SD across SNPs: 0.38). As a result, none of the aforementioned methods could be used conveniently (probabilistic methods were not used because of the high amounts of missing values, particularly prohibitive for estimating population allele frequencies). Genotype calling uncertainty was therefore not accounted for, and the genotypes were used as returned by the UNEAK pipeline in the imputation procedure and association tests. Because genotype calling uncertainty translates into random error under a GWAS model, not accounting for it typically results in shrunk marker effects and loss of power (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nielsen <em>et al.</em> 2011</a>). Because there was presumably no systematic loss in precision (true discovery rate), any GWAS inference made from (CC) or (MI*) was still deemed valid in this study. Nonetheless, false positives may have arisen at random from noise (due to genotype-calling uncertainty or other sources), particularly in (CC), which usually had low sample size <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-198.gif"/></noscript></span></span>.</p></div><div id="sec-31" class="subsection"><h3>The use of tree models for imputation</h3><p id="p-74">Here, no reference genome or genetic map was available for imputing the GBS marker data. Although it was possible to generate genetic maps on LP1 and LP2, the proportion of missing values was such that, in both populations, parental SNP genotypes could not be determined clearly (in particular, 37.6% and 25.6% of selected markers had missing values at both parents in LP1 and LP2, respectively) and imputation uncertainty would have to be accounted to produce reliable genetic maps, which would have been a serious statistical and computational challenge. Consequently, imputation methods based on HMM, which can be very accurate, were not used. In presence of unordered marker data with a general pattern of missingness and no reference panel, the strategy described here to impute missing values (<em>i.e.</em>, FCS based on tree models), should be pertinent: although computationally intensive, the proposed imputation procedure was flexible, easy to implement, and appropriate for modeling marker data, as was suggested by <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Dai <em>et al.</em> (2006)</a>.</p></div><div id="sec-32" class="subsection"><h3>Structure in the panels</h3><p id="p-75">Analysis of structure in the three panels combined revealed stratification by panels, race, and geographical origin. The observed stratification by race and geographical origin was consistent with the results from <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Jakubowski <em>et al.</em> (2011)</a>, <em>i.e.</em>, <em>P. caesia</em> being genetically distinct from <em>P. arundinacea</em> and, within <em>P. arundinacea</em>, East European strains being distinct from West European and North American strains. Even though accessions native to North America with a genetic background distinct from that of European accessions do exist (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jakubowski <em>et al.</em> 2012</a>), most accessions of reed canarygrass found in North America, including all those evaluated in this study, share some common ancestry with West European accessions.</p></div><div id="sec-33" class="subsection"><h3>Results of imputation-based association tests</h3><p id="p-76">In this association study, for the rare cases in which <em>Î³</em> was low enough <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-199.gif"/></noscript></span></span>, there were gains of significance from (CC) to (MI*) (and, presumably, a gain of power) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>, <a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7</a>). Another interesting outcome from MI was the decreased significance, given one marker (TP477925), for only a subset of the associations detected in (CC) (<a id="xref-table-wrap-7-5" class="xref-table" href="#T7">Table 7</a>). Such results suggest a possible gain of precision in MI-based association tests. Unfortunately, no novel significant markers could be detected from (CC) to (MI*). As expected, for high values of <em>Î³</em> <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-200.gif"/></noscript></span></span>, MI had the desired property of decreasing significance of associations for the sake of precision.</p><p id="p-77">Values of <em>Î³</em> lower on average than 0.3 would correspond to PMV lower than approximately 0.45 according to the model presented in <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>, with values of MAF and AMI set to 0.1 and 0.05, respectively. PMV lower than 0.45 corresponded to 1.6%, 1.8%, and 1.6% of the markers considered for GWAS in the LP1, LP2, and AP-4x subsets, respectively. As discussed below, if a reference panel had been available for imputation, then there would have been many more opportunities for gains in power from MI.</p></div><div id="sec-34" class="subsection"><h3>GWAS results and similarity of marker sequences to <em>Brachypodium distachyon</em></h3><p id="p-78">This GWAS revealed associations of markers with multiple traits, which may indicate pleiotropy of one single tagged causal variant, LD between distinct causal variants affecting different traits, or genetic correlation among traits. For example, TP87762 was negatively associated with TSC and positively associated with Ds, St, ARA, and Mg. This result would probably suggest that, through lower tiller density, reed canarygrass plants carrying the nonreference allele at TP87762 were less prone to disease and lodging and had higher concentration of arabinose and magnesium.</p><p id="p-79">Significant markers could not be mapped to a particular region of the reed canarygrass genome, because no reference map or genome sequence is available for that species. However, the significant markers identified here may be used in further studies making use of DNA sequences in reed canarygrass. Potentially, our studies may bring some insight about the function of genes in related species such as Brachypodium or oat. However, actual causal genes were probably not directly tagged by significant markers. Markers that did not map to the Brachypodium genomes (TP140584, TP191264, TP217634, TP477925, TP521945, and TP87762; because of absence of homology or because of evolutionary divergence) were probably in LD with unmarked functional regions, which possibly could have been mapped to the Brachypodium genome. TP184396 and TP 341988 could be mapped to the Brachypodium genome, but within genes with very general purposes (both matched genes involved in mRNA translation). Such results could be âtrue hits,â but this does not seem very likely. However, TP268059, negatively associated to CP and positively associated to XYL_Eff, was mapped to the exon of a gene coding for Phenylalanine/Histidine-Ammonia-Lyase. Although it is entirely possible that TP268059 did not directly tag the true causal gene, it is plausible that TP268059 is a true hit: Phenylalanine-Ammonia-Lyase (PAL) is involved in the very first step of the monolignol biosynthetic pathway leading to the synthesis of lignin (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boerjan <em>et al.</em> 2003</a>), and a decreased ability to synthesize lignin is consistent with lower crude protein content and higher xylose conversion efficiency because lignin inhibits fermentation (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Vogel and Jung 2001</a>). However, one would then expect to find significant associations involving Lignin (lignin content) and GLC_Eff (glucose conversion efficiency), which was not the case here. That said, the association between TP268059 and lignification still makes good sense from the homology of the marker in the Brachypodium genome and the relative directions of the detected effects.</p></div><div id="sec-35" class="subsection"><h3>A genome-wide association study based on multivariate MI</h3><p id="p-80">Historically, MI has been prevalent in epidemiological studies, probably because of the frequent and prohibitive occurrence of nonresponse in survey data (<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Rubin 1987</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rubin 1996</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Klebanoff and Cole 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sterne <em>et al.</em> 2009</a>). Here, the large amount of missing values in our GBS data motivated us to use MI to account for imputation uncertainty and make sound inferences about marker-trait associations. There have been previous quantitative genetics studies that used MI to increase the precision of significance tests for marker-associated effects: <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Dai <em>et al.</em> (2006)</a> used an imputation procedure similar to that presented here and compared it with expectation-maximization techniques. However, their study dealt with small marker data (10 SNPs in real data and 4 SNPs in simulated data) and limited PMV (10% or 20% of missing data). <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bobb <em>et al.</em> (2011)</a> used MI to gain precision in QTL mapping, but they generated multiple imputes of the phenotypic data, not the genomic data. To our knowledge, this study is the first report of MI applied in a genome-wide context (with thousands of markers over the genome). We believe the methodology presented here could be useful in large-scale genetic analyses involving hypothesis testing in two ways. First, it exemplifies the use of tree models to impute unordered marker data in GWAS and, in the context of multivariate MI, approximate the distribution <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-201.gif"/></noscript></span></span> by following the methodologies developed by <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Dai <em>et al.</em> (2006)</a> and <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Burgette and Reiter (2010)</a>. As multiplexed GBS, which trades sequencing depth (and therefore genotyping costs) for uncertainty in genotype calling and imputation, is increasingly used for association mapping (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Poland and Rife 2012</a>), this type of imputation procedures in a MI context should be particularly useful in species where no or only part of a reference genome is available, like wheat or switchgrass. Second, it shows how estimates from different imputes about marker effects in the unified linear mixed model (<a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Yu <em>et al.</em> 2006</a>) could be pooled using the rules developed by <a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">Rubin (1987)</a> to conveniently account for imputation uncertainty and perform statistically valid association tests. That said, MI is not the only method that has been developed to explicitly account for imputation uncertainty. A basic approach would be to use expected genotype counts based on some distribution <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-202.gif"/></noscript></span></span> and to perform association tests as if the marker data were fixed, as was done in (AD) here (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Guan and Stephens 2008</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zheng <em>et al.</em> 2011</a>). As noted previously (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Guan and Stephens 2008</a>), such simplification may yield erratic behavior in association testing if imputation uncertainty is high <em>and</em> marker effects are large, which is consistent with the results obtained here (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). A more sophisticated approach would be to use tests with explicit account for increased variance of parameter estimates due to imputation uncertainty that are based on some approximation of <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-203.gif"/></noscript></span></span> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Guan and Stephens 2008</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Zheng <em>et al.</em> 2011</a>). In this framework, frequentist tests include score tests and likelihood ratio tests, implemented in SNPTEST (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Marchini <em>et al.</em> 2007</a>). The Bayesian counterparts of this type of tests, implemented in SNPTEST and BIMBAM (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Servin and Stephens 2007</a>), feature some interesting advantages compared to the frequentist tests; in particular, they avoid an inflation in significance arising from high imputation uncertainty (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Guan and Stephens 2008</a>). Unfortunately, both the frequentist and Bayesian methods described above, in the current state of their implementations, are limited in the type of models that can be fitted to the data: the tests do not apply to linear mixed models (accounting for relatedness through the <strong>K</strong> matrix), and uncertainty in covariates cannot be conveniently accounted for, as was done in (MI*) here. Although in human GWAS such models may be appropriate (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Burton <em>et al.</em> 2007</a>), the strong population stratification in plant GWAS panels call for mixed models that incorporate information on population structure and relatedness (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Zhu <em>et al.</em> 2008</a>). We believe the approach used here was particularly useful in that it allowed accounting for imputation uncertainty when using linear mixed models for testing associations.</p></div><div id="sec-36" class="subsection"><h3>Properness of imputations</h3><p id="p-81">An imputation procedure is proper if it is unbiased and it is confidence-valid, <em>i.e.</em>, the variability among imputed values is equivalent to what it would be if the data had been complete. Here, we made the MCAR assumption stating that no factor (in particular, not the phenotypes of interest) influenced missingness at the marker data (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Rubin and Little 2002</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sterne <em>et al.</em> 2009</a>). Because marker effect estimates based on complete cases are unbiased under the MCAR assumption, (CC) analyses served as the reference to assess consistency of imputation-based tests. Judging from regression analyses on marker-effect estimates from (AD) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>) and from (MI) (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>), it seemed that imputations did not generate bias, with shrinkage in marker-effect estimates from MI occurring only because of imputation uncertainty in (MI) [and (MI*)], which is desirable. Imputation bias may have occurred if the MCAR assumptionâmore generally, the MAR assumptionâhad not been valid. This may occur in survey data but is unlikely to occur, systematically, in GBS data; the probability of a marker read being available is unlikely to vary systematically across individuals. Confidence validity could only be assumed here. For other imputation methods, which do not preserve the variability present in the original dataset, this assumption cannot be made. Such methods, which include mean imputation and major allele imputation, should therefore be avoided in inference studies.</p><p id="p-82">In the future, a simulation study might be conducted to characterize the unbiasedness and confidence validity of our imputation procedure in a setting where both marker effects and missing values are known <em>a priori</em>. Similar analyses were already performed on SNP data (<em>e.g.</em>, <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Dai <em>et al.</em> 2006</a>), but assessing the properness of MI based on GBS data in a genome-wide context could certainly be useful.</p></div><div id="sec-37" class="subsection"><h3>Practical issues for MI in GWAS</h3><p id="p-83">Here, given the amount of missing data (78% on average) and the information available for imputation (no reference panel and no genetic map of markers), imputation uncertainty was too prohibitive for detecting novel significant markers when performing imputation-based association tests. Gains of significance were nonetheless possible, mostly for low values of <em>Î³</em> (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a>), corresponding approximately to PMV &lt;0.45 (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>). <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pasaniuc <em>et al.</em> (2012)</a> showed that a very substantial gain in imputation accuracy on low-depth GBS data in humans could be achieved when using a reference panel (the 1000-genomes data). For example, with 0.1Ã sequencing depth, they reported an increase in imputation accuracy from approximately 0.05 to more than 0.70 when the reference panel was used. In light of the results from <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pasaniuc <em>et al.</em> (2012)</a> and this article, we recommend imputation-based association studies on GBS data with sporadically missing values only when coverage is high enough to produce few missing values <em>or</em> when a reference panel is available for imputation.</p><p id="p-84">When dealing with marker datasets that are much larger than the one considered here (numbers of markers <em>q</em> on the order of <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-204.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-205.gif"/></noscript></span></span>), implementing MI in GWAS would be a challenge. Computational and memory requirements will of course increase and parallelization should be devised accordingly. When dealing with unordered marker data, MI based on CART should be a good choice with regard to computationalâand statisticalâefficiency. However, the mice package uses a <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-206.gif"/></noscript></span></span> predictor adjacency matrix to keep record of the candidate predictors for each marker. With many threads and/or high <em>q</em>, memory usage can be reduced by replacing the adjacency matrix with a <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-207.gif"/></noscript></span></span> adjacency list (<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-208.gif"/></noscript></span></span>: number of variables selected as potential predictors; <em>e.g.</em>, 500 in this study) or determining the set of candidate predictors at each iteration with no storage, hence trading computational efficiency for memory efficiency. Both of these measures would involve modifications of the mice code. Importantly, if a (completely typed) reference panel <strong>H</strong> is available for imputation, then one can base imputations of <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-209.gif"/></noscript></span></span> on <strong>H</strong> only, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-210.gif"/></noscript></span></span> becomes <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-211.gif"/></noscript></span></span>. In such settings, when imputing a given variable <em>k</em>, there would be no uncertainty about values at the predictors to account for (supposedly, genotypes or haplotypes in <strong>H</strong> have been perfectly called). As a result, the multiple imputed datasets in MI could be sampled by ordinary MC instead of MCMC. This would dramatically reduce the computational and memory requirements when implementing MI. Note also that when imputing at markers that are completely untyped in <strong>X</strong> (<em>in silico</em> genotyping), basing imputations on <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-212.gif"/></noscript></span></span> rather than <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/891/embed/mml-math-213.gif"/></noscript></span></span> may actually yield more accurate imputations (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Guan and Stephens 2008</a>). Thus, in presence of a reference panel, MI should be not only more useful (imputations being more accurate) but also more tractable (probably applicable when dealing with hundreds of thousands of markers).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">The authors thank two anonymous reviewers for remarks and suggestions that greatly helped with improving the manuscript. This research was supported by the US Department of Energy &amp; Department of Agriculture Plant Feedstock Genomics for Bioenergy Program Project Number DE-A-102-07ER64454, by Agriculture and Food Research Initiative Competitive Grant No. 2011-68005-30411 from the USDA National Institute of Food and Agriculture (CenUSA), and by USDA-ARS Congressionally allocated funds. Mention of commercial products and organizations in this manuscript is solely to provide specific information. The USDA is an equal opportunity provider and employer. G.P.R. was supported by the Gabelman-Shippo Wisconsin Distinguished Graduate Fellowship at the University of WisconsinâMadison.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-14"><p id="p-86">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017533/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017533/-/DC1</a></p></li><li class="fn" id="fn-15"><p id="p-87"><em>Communicating editor: K. M. Devos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-17"><span class="received-label">Received </span>February 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-10"><span class="accepted-label">Accepted </span>March 10, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2015 Ramstein <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SchÃ¤ffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>â<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.2" data-doi="10.2134/agronj1931.00021962002300010008x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alway</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span>â<span class="cit-article-title">Early trials and use of reed canary grass as a forage plant.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">64</span>â<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DAlway%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D66%26rft.atitle%253DEarly%2BTrials%2Band%2BUse%2Bof%2BReed%2BCanary%2BGrass%2Bas%2Ba%2BForage%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1931.00021962002300010008x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1931.00021962002300010008x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.3" data-doi="10.2135/cropsci1968.0011183X000800050018x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asay</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilsie</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span>â<span class="cit-article-title">Genetic Variability in Forage Yield, Crude Protein Percentage, and Palatability in Reed Canarygrass, Phalaris arundinacea L.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">568</span>â<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DAsay%26rft.auinit1%253DK.%2BH.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D568%26rft.epage%253D571%26rft.atitle%253DGenetic%2BVariability%2Bin%2BForage%2BYield%252C%2BCrude%2BProtein%2BPercentage%252C%2Band%2BPalatability%2Bin%2BReed%2BCanarygrass%252C%2BPhalaris%2Barundinacea%2BL.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1968.0011183X000800050018x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1968.0011183X000800050018x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.4" data-doi="10.1080/00837792.1995.10670587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baldini</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Revision of the genus Phalaris L. (Gramineae).</span> <abbr class="cit-jnl-abbrev">Webbia</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">265</span>â<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWebbia%26rft.volume%253D49%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00837792.1995.10670587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00837792.1995.10670587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.5" data-doi="10.1093/biomet/86.4.948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Miscellanea. Small-sample degrees of freedom with multiple imputation.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">948</span>â<span class="cit-lpage">955</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DBarnard%26rft.auinit1%253DJ%26rft.volume%253D86%26rft.issue%253D4%26rft.spage%253D948%26rft.epage%253D955%26rft.atitle%253DMiscellanea.%2BSmall-sample%2Bdegrees%2Bof%2Bfreedom%2Bwith%2Bmultiple%2Bimputation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F86.4.948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6Ijg2LzQvOTQ4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bittman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulman</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-source">Reed canarygrass - a production guide</span>, <span class="cit-publ-name">Agriculture Canada</span>, <span class="cit-publ-loc">Ottawa, Ontario</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.7" data-doi="10.1016/j.fuel.2006.04.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boateng</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adler</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Pyrolysis of energy crops including alfalfa stems, reed canarygrass, and eastern gamagrass.</span> <abbr class="cit-jnl-abbrev">Fuel</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">2450</span>â<span class="cit-lpage">2457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFuel%26rft.volume%253D85%26rft.spage%253D2450%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fuel.2006.04.025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fuel.2006.04.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bobb</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharfstein</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniels</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelada</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multiple imputation of missing phenotype data for QTL mapping.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.9" data-doi="10.2135/cropsci2007.08.0482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Yield components of biomass in switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1306</span>â<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBoe%26rft.auinit1%253DA.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D1306%26rft.epage%253D1311%26rft.atitle%253DYield%2BComponents%2Bof%2BBiomass%2Bin%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.08.0482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.08.0482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257974100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.10" data-doi="10.1146/annurev.arplant.54.031902.134938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boerjan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baucher</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Lignin biosynthesis.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">519</span>â<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.54.031902.134938%26rft_id%253Dinfo%253Apmid%252F14503002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.54.031902.134938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14503002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185094100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.11" data-doi="10.1016/j.ympev.2008.01.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouchenak-Khelladi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forest</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, M.v.d. Bank <em>et al</em>., <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Large multi-gene phylogenetic trees of the grasses (Poaceae): progress towards complete tribal and generic level sampling.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">488</span>â<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DBouchenak-Khelladi%26rft.auinit1%253DY.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D488%26rft.epage%253D505%26rft.atitle%253DLarge%2Bmulti-gene%2Bphylogenetic%2Btrees%2Bof%2Bthe%2Bgrasses%2B%2528Poaceae%2529%253A%2Bprogress%2Btowards%2Bcomplete%2Btribal%2Band%2Bgeneric%2Blevel%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2008.01.035%26rft_id%253Dinfo%253Apmid%252F18358746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2008.01.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18358746&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256110100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olshen</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-source">Classification and regression trees</span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-5.5.891.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burras</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-source"><em>Switchgrass production in Iowa: economic analysis</em>, <em>soil suitability</em>, <em>and varietal performance</em></span>, <span class="cit-publ-name">Iowa State University</span>, <span class="cit-publ-loc">Ames, Iowa</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.14" data-doi="10.1093/aje/kwq260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgette</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiter</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Multiple imputation for missing data via sequential regression trees.</span> <abbr class="cit-jnl-abbrev">Am. J. Epidemiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1070</span>â<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BEpidemiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faje%252Fkwq260%26rft_id%253Dinfo%253Apmid%252F20841346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYW1qZXBpZCI7czo1OiJyZXNpZCI7czoxMDoiMTcyLzkvMTA3MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.15" data-doi="10.1038/nature05911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craddock</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>â<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-source">ASReml-R reference manual</span>, <span class="cit-publ-name">Queensland Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oram</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surprenant</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Reed canarygrass and other Phalaris species</span>, pp. <span class="cit-fpage">569</span>â<span class="cit-lpage">604</span> in <span class="cit-source">Cool-season forage grasses</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">L. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Buxton</span>  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>. <span class="cit-publ-name">American Society of Agronomy Inc.</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-source"><em>Genetics</em>, <em>breeding</em>, <em>and ecology of reed canarygrass</em></span>. <span class="cit-publ-name">Intl J Plant Breeding</span> <span class="cit-vol">4</span>: <span class="cit-fpage">30</span>â<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bbreeding%252C%2Band%2Becology%2Bof%2Breed%2Bcanarygrass%26rft.volume%253D4%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.19" data-doi="10.2135/cropsci2009.02.0055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krohn</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span>â<span class="cit-article-title">DNA polymorphisms reveal geographic races of reed canarygrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">2139</span>â<span class="cit-lpage">2148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D49%26rft.issue%253D6%26rft.spage%253D2139%26rft.epage%253D2148%26rft.atitle%253DDNA%2BPolymorphisms%2BReveal%2BGeographic%2BRaces%2Bof%2BReed%2BCanarygrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.02.0055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.02.0055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000271607300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.20" data-doi="10.1007/s12155-009-9043-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span>â<span class="cit-article-title">Biomass yield of naturalized populations and cultivars of reed canary grass.</span> <abbr class="cit-jnl-abbrev">BioEnergy Research</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">165</span>â<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BResearch%26rft.volume%253D2%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-009-9043-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-009-9043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.21" data-doi="10.1016/0144-4565(88)90105-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volenec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anliker</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Chemical composition of herbaceous grass and legume species grown for maximum biomass production.</span> <abbr class="cit-jnl-abbrev">Biomass</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">215</span>â<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomass%26rft.volume%253D17%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0144-4565%252888%252990105-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0144-4565(88)90105-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cureton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenevelt</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">Landfill leachate recirculation: effects on vegetation vigor and clay surface cover infiltration.</span> <abbr class="cit-jnl-abbrev">J. Environ. Qual.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">17</span>â<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEnvironmental%2BQuality%26rft.stitle%253DJ.%2BEnviron.%2BQual.%26rft.aulast%253DCureton%26rft.auinit1%253DP.%2BM.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D24%26rft.atitle%253DLandfill%2BLeachate%2BRecirculation%253A%2BEffects%2Bon%2BVegetation%2BVigor%2Band%2BClay%2BSurface%2BCover%2BInfiltration%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1991EY99600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-other" id="cit-5.5.891.23"><div class="cit-metadata unstructured">Dabney, A., J.D. Storey, and G.R. Warnes, 2013âqvalue: Q-value estimation for false discovery rate control. R package version 1.34.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.24" data-doi="10.1002/gepi.20180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruczinski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeBlanc</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kooperberg</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Imputation methods to improve inference in SNP association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">690</span>â<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DDai%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D30%26rft.issue%253D8%26rft.spage%253D690%26rft.epage%253D702%26rft.atitle%253DImputation%2Bmethods%2Bto%2Bimprove%2Binference%2Bin%2BSNP%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20180%26rft_id%253Dinfo%253Apmid%252F16986162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16986162&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242383400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.25" data-doi="10.1016/j.biombioe.2006.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dien</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.-J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Chemical composition and response to dilute-acid pretreatment and enzymatic saccharification of alfalfa, reed canarygrass, and switchgrass.</span> <abbr class="cit-jnl-abbrev">Biomass Bioenergy</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">880</span>â<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomass%2BBioenergy%26rft.volume%253D30%26rft.spage%253D880%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biombioe.2006.02.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biombioe.2006.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241067500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.26" data-doi="10.1016/j.csda.2013.10.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doove</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Buuren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dusseldorp</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Recursive partitioning for missing data imputation in the presence of interaction effects.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">92</span>â<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D72%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2013.10.025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2013.10.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.27" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.28" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>â<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghunathan</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Using conditional distributions for missing-data imputation.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">268</span>â<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D3%26rft.spage%253D268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.30" data-doi="10.1371/journal.pgen.1000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Practical issues in imputation-based association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000279%26rft.epage%253De1000279%26rft.atitle%253DPractical%2Bissues%2Bin%2Bimputation-based%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000279%26rft_id%253Dinfo%253Apmid%252F19057666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19057666&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.31" data-doi="10.1093/nar/gkj154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaiswal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spooner</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Gramene: a birdâs eye view of cereal genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D717</span>â<span class="cit-lpage">D723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DJaiswal%26rft.auinit1%253DP.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD717%26rft.epage%253DD723%26rft.atitle%253DGramene%253A%2Ba%2Bbird%2527s%2Beye%2Bview%2Bof%2Bcereal%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj154%26rft_id%253Dinfo%253Apmid%252F16381966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0Q3MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genetic evidence suggests a widespread distribution of native North American populations of reed canarygrass.</span> <abbr class="cit-jnl-abbrev">Biol. Invasions</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">261</span>â<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BInvasions%26rft.volume%253D15%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.33" data-doi="10.1071/CP11232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genetic diversity and population structure of Eurasian populations of reed canarygrass: cytotypes, cultivars, and interspecific hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">982</span>â<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D62%26rft.spage%253D982%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP11232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP11232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.34" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>â<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.35" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.36" data-doi="10.1093/aje/kwn071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klebanoff</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Use of multiple imputation in the epidemiologic literature.</span> <abbr class="cit-jnl-abbrev">Am. J. Epidemiol.</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">355</span>â<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BEpidemiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faje%252Fkwn071%26rft_id%253Dinfo%253Apmid%252F18591202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYW1qZXBpZCI7czo1OiJyZXNpZCI7czo5OiIxNjgvNC8zNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.37" data-doi="10.2307/2290525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghunathan</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">Large-sample significance levels from multiply imputed data using moment-based statistics and an F reference distribution.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1065</span>â<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D86%26rft.spage%253D1065%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GU05000031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.38" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.39" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>â<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.40" data-doi="10.1071/AR9620001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neal-Smith</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">Tetraploid and hexaploid chromosome races of Phalaris arundinacea L.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D13%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9620001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9620001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.41" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>â<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.42" data-doi="10.2135/cropsci2012.01.0003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genetic variability for biofuel traits in a circumglobal reed canarygrass collection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">524</span>â<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.01.0003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.01.0003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.43" data-doi="10.1016/S0926-6690(99)00050-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pahkala</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pihala</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Different plant parts as raw material for fuel and pulp production.</span> <abbr class="cit-jnl-abbrev">Ind. Crops Prod.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">119</span>â<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInd.%2BCrops%2BProd.%26rft.volume%253D11%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0926-6690%252899%252900050-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0926-6690(99)00050-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.44" data-doi="10.1038/ng.2283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Extremely low-coverage sequencing and imputation increases power for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">631</span>â<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPasaniuc%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D631%26rft.epage%253D635%26rft.atitle%253DExtremely%2Blow-coverage%2Bsequencing%2Band%2Bimputation%2Bincreases%2Bpower%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2283%26rft_id%253Dinfo%253Apmid%252F22610117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22610117&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.45" data-doi="10.1016/j.biortech.2004.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The interacting effects of temperature and plant community type on nutrient removal in wetland microcosms.</span> <abbr class="cit-jnl-abbrev">Bioresour. Technol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1039</span>â<span class="cit-lpage">1047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioresource%2BTechnology%26rft.stitle%253DBioresource%2BTechnology%26rft.aulast%253DPicard%26rft.auinit1%253DC.%2BR.%26rft.volume%253D96%26rft.issue%253D9%26rft.spage%253D1039%26rft.epage%253D1047%26rft.atitle%253DThe%2Binteracting%2Beffects%2Bof%2Btemperature%2Band%2Bplant%2Bcommunity%2Btype%2Bon%2Bnutrient%2Bremoval%2Bin%2Bwetland%2Bmicrocosms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biortech.2004.09.007%26rft_id%253Dinfo%253Apmid%252F15668200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biortech.2004.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15668200&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.46" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>â<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.47" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>â<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312663100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.48" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>â<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.49" data-doi="10.1007/s12155-013-9374-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Divergent selection for secondary traits in upland tetraploid switchgrass and effects on sward biomass yield.</span> <abbr class="cit-jnl-abbrev">BioEnergy Research</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">329</span>â<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BResearch%26rft.volume%253D7%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-013-9374-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-013-9374-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.50" data-doi="10.3732/ajb.94.9.1554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quintanar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castroviejo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">CatalÃ¡n</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Phylogeny of the Tribe Avenae (Pooideae, Poaceae) inferred from plastid Trn-T and nuclear ITS sequences.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1554</span>â<span class="cit-lpage">1569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DQuintanar%26rft.auinit1%253DA.%26rft.volume%253D94%26rft.issue%253D9%26rft.spage%253D1554%26rft.epage%253D1569%26rft.atitle%253DPhylogeny%2Bof%2Bthe%2Btribe%2BAveneae%2B%2528Pooideae%252C%2BPoaceae%2529%2Binferred%2Bfrom%2Bplastid%2BtrnT-F%2Band%2Bnuclear%2BITS%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.9.1554%26rft_id%253Dinfo%253Apmid%252F21636521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6Ijk0LzkvMTU1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinkerton</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Reed canarygrass survival under cyclic inundation.</span> <abbr class="cit-jnl-abbrev">J. Soil Water Conserv.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">132</span>â<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSoil%2BWater%2BConserv.%26rft.volume%253D48%26rft.spage%253D132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-source">Multiple imputation for nonresponse in surveys</span>, <span class="cit-publ-name">John Wiley and Sons</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.54" data-doi="10.2307/2291635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Multiple imputation after 18+ years.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">473</span>â<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D91%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UP55200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-source">Statistical analysis with missing data</span>, <span class="cit-publ-name">J Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-confproc" id="cit-5.5.891.56"><div class="cit-metadata unstructured">Rubin, D. B., and J. L. Schafer, 1990âEfficiently creating multiple imputations for incomplete multivariate normal data, pp. 88 in <em>Proceedings of the Statistical Computing Section of the American Statistical Association</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.57" data-doi="10.2307/2289225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenker</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">Multiple imputation for interval estimation from simple random samples with ignorable nonresponse.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">366</span>â<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D81%26rft.spage%253D366%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C648000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-other" id="cit-5.5.891.58"><div class="cit-metadata unstructured">Rutkoski, J.E., J. Poland, J.-L. Jannink, and M.E. Sorrells, 2013âImputation of unordered markers and the impact on genomic selection accuracy. G3 (Bethesda) 3: 427â439.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.59" data-doi="10.1016/0960-8524(95)00176-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Switchgrass as a sustainable bioenergy crop.</span> <abbr class="cit-jnl-abbrev">Bioresour. Technol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">83</span>â<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioresour.%2BTechnol.%26rft.volume%253D56%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0960-8524%252895%252900176-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0960-8524(95)00176-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VG11600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-source">Analysis of incomplete multivariate data</span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.61" data-doi="10.1371/journal.pgen.0030114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Imputation-based analysis of association studies: candidate regions and quantitative traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DServin%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De114%26rft.epage%253De114%26rft.atitle%253DImputation-based%2Banalysis%2Bof%2Bassociation%2Bstudies%253A%2Bcandidate%2Bregions%2Band%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030114%26rft_id%253Dinfo%253Apmid%252F17676998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676998&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.62" data-doi="10.2135/cropsci1991.0011183X003100020049x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shenk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westerhaus</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">Population definition, sample selection, and calibration procedures for near infrared reflectance spectroscopy.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">469</span>â<span class="cit-lpage">474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DShenk%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D469%26rft.epage%253D474%26rft.atitle%253DPopulation%2BDefinition%252C%2BSample%2BSelection%252C%2Band%2BCalibration%2BProcedures%2Bfor%2BNear%2BInfrared%2BReflectance%2BSpectroscopy%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1991.0011183X003100020049x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1991.0011183X003100020049x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FZ64300048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.63" data-doi="10.1136/bmj.b2393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sterne</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royston</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Multiple imputation for missing data in epidemiological and clinical research: potential and pitfalls.</span> <abbr class="cit-jnl-abbrev">BMJ</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">b2393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMJ%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fbmj.b2393%26rft_id%253Dinfo%253Apmid%252F19564179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYm1qIjtzOjU6InJlc2lkIjtzOjE3OiIzMzgvanVuMjlfMS9iMjM5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.64" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>â<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">âµ</a><div class="cit ref-cit ref-other" id="cit-5.5.891.65"><div class="cit-metadata unstructured">Therneau, T.M., and E.J. Atkinson, 1997 An introduction to recursive partitioning using the RPART routines.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.66" data-doi="10.1111/j.1365-2494.1963.tb00335.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tilley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terry</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span>â<span class="cit-article-title">A twoâstage technique for the in vitro digestion of forage crops.</span> <abbr class="cit-jnl-abbrev">Grass Forage Sci.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">104</span>â<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGrass%2BForage%2BSci.%26rft.volume%253D18%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2494.1963.tb00335.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2494.1963.tb00335.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">US Department of Agriculture and US Department of Energy</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-source">Biomass as feedstock for a bioenergy and bioproducts industry: the technical feasibility of a billion-ton annual supply</span>, <span class="cit-publ-name">Oak Ridge National Laboratory</span>, <span class="cit-publ-loc">Oak Ridge, TN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.68" data-doi="10.1177/0962280206074463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Buuren</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Multiple imputation of discrete and continuous data by fully conditional specification.</span> <abbr class="cit-jnl-abbrev">Stat. Methods Med. Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">219</span>â<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BMethods%2Bin%2BMedical%2BResearch%26rft.stitle%253DStat%2BMethods%2BMed%2BRes%26rft.aulast%253Dvan%2BBuuren%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D219%26rft.epage%253D242%26rft.atitle%253DMultiple%2Bimputation%2Bof%2Bdiscrete%2Band%2Bcontinuous%2Bdata%2Bby%2Bfully%2Bconditional%2Bspecification%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0962280206074463%26rft_id%253Dinfo%253Apmid%252F17621469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3BzbW0iO3M6NToicmVzaWQiO3M6ODoiMTYvMy8yMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">âµ</a><div class="cit ref-cit ref-book" id="cit-5.5.891.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Buuren</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-source">Flexible imputation of missing data</span>, <span class="cit-publ-name">CRC press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Buuren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groothuis-Oudshoorn</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">MICE: Multivariate imputation by chained equations in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Software</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftware%26rft.volume%253D45%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.71" data-doi="10.1080/10629360600810434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Buuren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groothuis-Oudshoorn</span>  <span class="cit-name-given-names">C. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Fully conditional specification in multivariate imputation.</span> <abbr class="cit-jnl-abbrev">J. Stat. Comput. Simul.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1049</span>â<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BComput.%2BSimul.%26rft.volume%253D76%26rft.spage%253D1049%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10629360600810434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10629360600810434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242208900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.891.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Keulen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span>â<span class="cit-article-title">Evaluation of acid-insoluble ash as a natural marker in ruminant digestibility studies.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">282</span>â<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DVan%2BKeulen%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D282%26rft.epage%253D287%26rft.atitle%253DEvaluation%2Bof%2BAcid-Insoluble%2BAsh%2Bas%2Ba%2BNatural%2BMarker%2Bin%2BRuminant%2BDigestibility%2BStudies%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1977CU65800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-2" title="View reference  in text" id="ref-73">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.73" data-doi="10.1016/S0735-2689(01)80011-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H. J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Genetic modification of herbaceous plants for feed and fuel.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">15</span>â<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D20%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0735-2689%252801%252980011-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0735-2689(01)80011-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000167167500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.74" data-doi="10.1007/s12155-010-9104-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dien</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masterson</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Quantifying actual and theoretical ethanol yields for switchgrass strains using NIRS analyses.</span> <abbr class="cit-jnl-abbrev">BioEnergy Research</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">96</span>â<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BResearch%26rft.volume%253D4%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-010-9104-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-010-9104-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000289246400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.75" data-doi="10.1111/1467-9868.00374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Thin-plate regression splines.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">95</span>â<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D65%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wrobel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulman</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The potential use of reed canarygrass (Phalaris arundinacea L.) as a biofuel crop.</span> <abbr class="cit-jnl-abbrev">Acta Agricultur. Scand. B Plant Soil Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgricultur.%2BScand.%2BB%2BPlant%2BSoil%2BSci.%26rft.volume%253D59%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.77" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.78" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>â<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.79" data-doi="10.1002/gepi.20552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A comparison of approaches to account for uncertainty in analysis of imputed genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">102</span>â<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DZheng%26rft.auinit1%253DJ.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D110%26rft.atitle%253DA%2Bcomparison%2Bof%2Bapproaches%2Bto%2Baccount%2Bfor%2Buncertainty%2Bin%2Banalysis%2Bof%2Bimputed%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20552%26rft_id%253Dinfo%253Apmid%252F21254217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21254217&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">âµ</a><div class="cit ref-cit ref-journal" id="cit-5.5.891.80" data-doi="10.3835/plantgenome2008.02.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Status and prospects of association mapping in plants.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">5</span>â<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.stitle%253DThe%2BPlant%2BGenome%26rft.aulast%253DZhu%26rft.auinit1%253DC.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D20%26rft.atitle%253DStatus%2Band%2BProspects%2Bof%2BAssociation%2BMapping%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2008.02.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2008.02.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/891.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/873" title="Quantitative Genetics of Migration-Related Traits in Rainbow and Steelhead Trout" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/911" title="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/891.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qShgS68v5CV31QMmHadTXWQy0chZVUqj-g63_h7PezQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/891" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/891" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/891" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459905" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-V03mZBBaRZQXqdcRDPeaBiQh85yTIXnsDtwwIVS2cP4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/891.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/891" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CN29xwNbcTB-tn7NokrznfAvWC0z-auT0nqCt8--6mo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/891" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/891.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459905" id="node459905--2" data-pisa="ggg;5/5/891" data-pisa-master="ggg;g3.115.017533" data-apath="/ggg/5/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Alexander E. Lipka</span>, <span class="highwire-citation-author" data-delta="2">Fei Lu</span>, <span class="highwire-citation-author" data-delta="3">Denise E. Costich</span>, <span class="highwire-citation-author" data-delta="4">Jerome H. Cherney</span>, <span class="highwire-citation-author" data-delta="5">Edward S. Buckler</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017533 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459905--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Denise E. Costich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>International Maize and Wheat Improvement Center (CIMMYT), 56237 Texcoco, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denise%2BE.%2BCostich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costich%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenise%2BE.%2BCostich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jerome H. Cherney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Soil and Crops Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jerome%2BH.%2BCherney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cherney%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJerome%2BH.%2BCherney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Agricultural Research Service, United States Department of Agriculture, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Agricultural Research Service, United States Department of Agriculture, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/891" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/891&amp;text=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/891&amp;t=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/891&amp;title=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459905/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459905" id="node459905--3" data-pisa="ggg;5/5/891" data-pisa-master="ggg;g3.115.017533" data-apath="/ggg/5/5/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study Based on Multiple Imputation with Low-Depth Sequencing Data: Application to Biofuel Traits in Reed Canarygrass</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Alexander E. Lipka</span>, <span class="highwire-citation-author" data-delta="2">Fei Lu</span>, <span class="highwire-citation-author" data-delta="3">Denise E. Costich</span>, <span class="highwire-citation-author" data-delta="4">Jerome H. Cherney</span>, <span class="highwire-citation-author" data-delta="5">Edward S. Buckler</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-909;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017533 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459905--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fei Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Denise E. Costich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>International Maize and Wheat Improvement Center (CIMMYT), 56237 Texcoco, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denise%2BE.%2BCostich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costich%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenise%2BE.%2BCostich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jerome H. Cherney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Soil and Crops Section, School of Integrative Plant Science, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jerome%2BH.%2BCherney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cherney%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJerome%2BH.%2BCherney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Agricultural Research Service, United States Department of Agriculture, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â â </span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>â¡â¡</span>Agricultural Research Service, United States Department of Agriculture, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=891&amp;rft.epage=909&amp;rft.atitle=Genome-Wide%20Association%20Study%20Based%20on%20Multiple%20Imputation%20with%20Low-Depth%20Sequencing%20Data%3A%20Application%20to%20Biofuel%20Traits%20in%20Reed%20Canarygrass&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ramstein%2C+Guillaume+P.&amp;rft.au=Lipka%2C+Alexander+E.&amp;rft.au=Lu%2C+Fei&amp;rft.au=Costich%2C+Denise+E.&amp;rft.au=Cherney%2C+Jerome+H.&amp;rft.au=Buckler%2C+Edward+S.&amp;rft.au=Casler%2C+Michael+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459905/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459905/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459905/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459905/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459905/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459905/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459905/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459905/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459905/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459905/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459905/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459905/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/891" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-29" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/891.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/891.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/891.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/891.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
