<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3745" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3745.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3745.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3745.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3745.atom" />
<meta name="HW.pisa" content="ggg;8/12/3745" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" />
<meta name="DC.Identifier" content="10.1534/g3.118.200552" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 De-Kayne, Feulner. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic datasets continue to increase in number due to the ease of production for a wider selection of species including non-model organisms. For many of these species, especially those with large or polyploid genomes, highly contiguous and well-annotated genomes are still rare due to the complexity and cost involved in their assembly. As a result, a common starting point for genomic work in non-model species is the production of a linkage map. Dense linkage maps facilitate the analysis of genomic data in a variety of ways, from broad scale observations regarding genome structure e.g. , chromosome number and type or sex-related structural differences, to fine scale patterns e.g. , recombination rate variation and co-localization of differentiated regions. Here we present both sex-averaged and sex-specific linkage maps for Coregonus sp. “Albock ”, a member of the European whitefish lineage ( C. lavaretus spp. complex), containing 5395 single nucleotide polymorphism (SNP) loci across 40 linkage groups to facilitate future investigation into the genomic basis of whitefish adaptation and speciation. The map was produced using restriction-site associated digestion (RAD) sequencing data from two wild-caught parents and 156 F1 offspring. We discuss the differences between our sex-averaged and sex-specific maps and identify genome-wide synteny between C. sp. “Albock ” and Atlantic Salmon ( Salmo salar ), which have diverged following the salmonid-specific whole genome duplication. Our analysis confirms that many patterns of synteny observed between Atlantic Salmon and Oncorhynchus and Salvelinus species are also shared by members of the Coregoninae subfamily. We also show that regions known for their species-specific rediploidization history can pose challenges for synteny identification since these regions have diverged independently in each salmonid species following the salmonid-specific whole genome duplication. The European whitefish map provided here will enable future studies to understand the distribution of loci of interest, e.g. , FST outliers, along the whitefish genome as well as assisting with the de novo assembly of a whitefish reference genome." />
<meta name="DC.Contributor" content="Rishi De-Kayne" />
<meta name="DC.Contributor" content="Philine G. D. Feulner" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic datasets continue to increase in number due to the ease of production for a wider selection of species including non-model organisms. For many of these species, especially those with large or polyploid genomes, highly contiguous and well-annotated genomes are still rare due to the complexity and cost involved in their assembly. As a result, a common starting point for genomic work in non-model species is the production of a linkage map. Dense linkage maps facilitate the analysis of genomic data in a variety of ways, from broad scale observations regarding genome structure &lt;i&gt;e.g.&lt;/i&gt;, chromosome number and type or sex-related structural differences, to fine scale patterns &lt;i&gt;e.g.&lt;/i&gt;, recombination rate variation and co-localization of differentiated regions. Here we present both sex-averaged and sex-specific linkage maps for &lt;i&gt;Coregonus sp. “Albock&lt;/i&gt;”, a member of the European whitefish lineage (&lt;i&gt;C. lavaretus&lt;/i&gt; spp. complex), containing 5395 single nucleotide polymorphism (SNP) loci across 40 linkage groups to facilitate future investigation into the genomic basis of whitefish adaptation and speciation. The map was produced using restriction-site associated digestion (RAD) sequencing data from two wild-caught parents and 156 F1 offspring. We discuss the differences between our sex-averaged and sex-specific maps and identify genome-wide synteny between &lt;i&gt;C. sp. “Albock&lt;/i&gt;” and Atlantic Salmon (&lt;i&gt;Salmo salar&lt;/i&gt;), which have diverged following the salmonid-specific whole genome duplication. Our analysis confirms that many patterns of synteny observed between Atlantic Salmon and &lt;i&gt;Oncorhynchus&lt;/i&gt; and &lt;i&gt;Salvelinus&lt;/i&gt; species are also shared by members of the Coregoninae subfamily. We also show that regions known for their species-specific rediploidization history can pose challenges for synteny identification since these regions have diverged independently in each salmonid species following the salmonid-specific whole genome duplication. The European whitefish map provided here will enable future studies to understand the distribution of loci of interest, &lt;i&gt;e.g.&lt;/i&gt;, F&lt;sub&gt;ST&lt;/sub&gt; outliers, along the whitefish genome as well as assisting with the &lt;i&gt;de novo&lt;/i&gt; assembly of a whitefish reference genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3745" />
<meta name="citation_id" content="8/12/3745" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3745" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3745.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3745.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3745.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200552" />
<meta name="citation_pmid" content="30297382" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="3745" />
<meta name="citation_lastpage" content="3755" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rishi De-Kayne" />
<meta name="citation_author_institution" content="Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland" />
<meta name="citation_author_institution" content="Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland" />
<meta name="citation_author" content="Philine G. D. Feulner" />
<meta name="citation_author_institution" content="Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland" />
<meta name="citation_author_institution" content="Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland" />
<meta name="citation_author_email" content="philine.feulner@eawag.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8078-1788" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=L. Bernatchez;citation_author=J. J. Dodson;citation_title=Allopatric origin of sympatric populations of lake whitefish (Coregonus clupeaformis) as revealed by mitochondrial-DNA restriction analysis.;citation_pages=1263-1271;citation_volume=44;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Brawand;citation_author=C. E. Wagner;citation_author=Y. I. Li;citation_author=M. Malinsky;citation_author=I. Keller;citation_title=The genomic substrate for adaptive radiation in African cichlid fish.;citation_pages=375-381;citation_volume=513;citation_year=2014;citation_pmid=25186727;citation_doi=10.1038/nature13726" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. A. Christensen;citation_author=J. S. Leong;citation_author=D. Sakharani;citation_author=C. A. Biagi;citation_author=D. R. Minkley;citation_title=Chinook salmon (Oncorhynchus tshawytscha) genome and transcriptome.;citation_pages=e0195461;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0195461" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=K. A. Christensen;citation_author=E. B. Rondeau;citation_author=D. R. Minkley;citation_author=J. S. Leong;citation_author=C. M. Nugent;citation_title=The Arctic charr (Salvelinus alpinus) genome and transcriptome assembly.;citation_pages=e0204076;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0204076" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=C. J. Doenz;citation_author=D. Bittner;citation_author=P. Vonlanthen;citation_author=C. E. Wagner;citation_author=O. Seehausen;citation_title=Rapid buildup of sympatric species diversity in Alpine whitefish.;citation_year=2018;citation_doi=10.1002/ece3.4375" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. W. Doerge;citation_title=Mapping and analysis of quantitative trait loci in experimental populations.;citation_pages=43-52;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823790;citation_doi=10.1038/nrg703" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. R. Douglas;citation_author=P. C. Brunner;citation_author=L. Bernatchez;citation_title=Do assemblages of Coregonus (Teleostei: Salmoniformes) in the Central Alpine region of Europe represent species flocks?;citation_pages=589-603;citation_volume=8;citation_year=1999;citation_doi=10.1046/j.1365-294x.1999.00581.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. C. Dunn;citation_title=LINKAGE IN MICE AND RATS;citation_pages=325-343;citation_volume=5;citation_year=1920;citation_issue=3;citation_pmid=17245945" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Ekblom;citation_author=J. Galindo;citation_title=Applications of next generation sequencing in molecular ecology of non-model organisms.;citation_pages=1-15;citation_volume=107;citation_year=2011;citation_issue=1;citation_pmid=21139633;citation_doi=10.1038/hdy.2010.152" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=H. Ellegren;citation_title=GENOME SEQUENCING AND POPULATION GENOMICS IN NON-MODEL ORGANISMS;citation_pages=51-63;citation_volume=29;citation_year=2014;citation_pmid=24139972;citation_doi=10.1016/j.tree.2013.09.008" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=H. Ellegren;citation_title=The Evolutionary Genomics of Birds.;citation_pages=239-259;citation_volume=44;citation_year=2013;citation_doi=10.1146/annurev-ecolsys-110411-160327" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=P. G. D. Feulner;citation_author=R. De-Kayne;citation_title=Genome evolution, structural rearrangements and speciation.;citation_pages=1488-1490;citation_volume=30;citation_year=2017;citation_doi=10.1111/jeb.13101" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. G. D. Feulner;citation_author=J. Schwarzer;citation_author=M. P. Haesler;citation_author=J. I. Meier;citation_author=O. Seehausen;citation_title=A dense linkage map of Lake Victoria cichlids improved the Pundamilia genome assembly and revealed a major QTL for sex-determination.;citation_volume=8;citation_year=2018;citation_pmid=29760203;citation_doi=10.1534/g3.118.200207" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Fierst;citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.;citation_pages=220;citation_volume=6;citation_year=2015;citation_pmid=26150829;citation_doi=10.3389/fgene.2015.00220" />
<meta name="citation_reference" content="citation_journal_title=Mar. Genomics;citation_author=R. R. da Fonseca;citation_author=A. Albrechtsen;citation_author=G. E. Themudo;citation_author=J. Ramos-Madrigal;citation_author=J. A. Sibbesen;citation_title=Next-generation biology: Sequencing and data analysis approaches for non-model organisms.;citation_pages=3-13;citation_volume=30;citation_year=2016;citation_doi=10.1016/j.margen.2016.04.012" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=P.-A. Gagnaire;citation_author=S. A. Pavey;citation_author=E. Normandeau;citation_author=L. Bernatchez;citation_title=The genetic architecture of reproductive isolation during speciation-with-gene-flow in lake whitefish species pairs assessed by RAD sequencing.;citation_pages=2483-2497;citation_volume=67;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. R. Garvin;citation_author=K. Saitoh;citation_author=A. J. Gharrett;citation_title=Application of single nucleotide polymorphisms to non-model species: a technical review.;citation_pages=915-934;citation_volume=10;citation_year=2010;citation_doi=10.1111/j.1755-0998.2010.02891.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Gharbi;citation_author=A. Gautier;citation_author=R. G. Danzmann;citation_author=S. Gharbi;citation_author=T. Sakamoto;citation_title=A Linkage Map for Brown Trout (Salmo trutta): Chromosome Homeologies and Comparative Genome Organization With Other Salmonid Fish.;citation_volume=172;citation_year=2006;citation_pmid=16452148;citation_doi=10.1534/genetics.105.048330" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. B. Gross;citation_author=M. Protas;citation_author=M. Conrad;citation_author=P. E. Scheid;citation_author=O. Vidal;citation_title=Synteny and candidate gene prediction using an anchored linkage map of Astyanax mexicanus.;citation_volume=105;citation_year=2008;citation_pmid=19104060;citation_doi=10.1073/pnas.0806238105" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Gu;citation_author=L. Gu;citation_author=R. Eils;citation_author=M. Schlesner;citation_author=B. Brors;citation_title=circlize implements and enhances circular visualization in R.;citation_volume=30;citation_year=2014;citation_pmid=24930139;citation_doi=10.1093/bioinformatics/btu393" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. C. Hale;citation_author=G. J. McKinney;citation_author=C. L. Bell;citation_author=K. M. Nichols;citation_title=Using Linkage Maps as a Tool To Determine Patterns of Chromosome Synteny in the Genus Salvelinus.;citation_volume=7;citation_year=2017;citation_pmid=28963166;citation_doi=10.1534/g3.117.300317" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. A. Hornett;citation_author=C. W. Wheat;citation_title=Quantitative RNA-Seq analysis in non-model species: assessing transcriptome assemblies as a scaffold and the utility of evolutionary divergent genomic reference species.;citation_pages=361-361;citation_volume=13;citation_year=2012;citation_pmid=22853326;citation_doi=10.1186/1471-2164-13-361" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=A. G. Hudson;citation_author=B. Lundsgaard-Hansen;citation_author=K. Lucek;citation_author=P. Vonlanthen;citation_author=O. Seehausen;citation_title=Managing cryptic biodiversity: Fine-scale intralacustrine speciation along a benthic gradient in Alpine whitefish (Coregonus spp.).;citation_pages=251-266;citation_volume=10;citation_year=2017;citation_doi=10.1111/eva.12446" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=A. G. Hudson;citation_author=P. Vonlanthen;citation_author=O. Seehausen;citation_title=Rapid parallel adaptive radiations from a single hybridogenic ancestral population.;citation_volume=278;citation_year=2011;citation_pmid=20685705;citation_doi=10.1098/rspb.2010.0925;citation_georef=2013048753" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. E. Johnston;citation_author=J. Huisman;citation_author=P. A. Ellis;citation_author=J. M. Pemberton;citation_title=A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus).;citation_volume=7;citation_year=2017;citation_pmid=28667018;citation_doi=10.1534/g3.117.044198" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Kawakami;citation_author=L. Smeds;citation_author=N. Backström;citation_author=A. Husby;citation_author=A. Qvarnström;citation_title=A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.;citation_pages=4035-4058;citation_volume=23;citation_year=2014;citation_pmid=24863701;citation_doi=10.1111/mec.12810" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Kodama;citation_author=M. S. O. Brieuc;citation_author=R. H. Devlin;citation_author=J. J. Hard;citation_author=K. A. Naish;citation_title=Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event.;citation_volume=4;citation_year=2014;citation_pmid=25053705;citation_doi=10.1534/g3.114.012294" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. F. Koop;citation_author=K. R. von Schalburg;citation_author=J. Leong;citation_author=N. Walker;citation_author=R. Lieph;citation_title=A salmonid EST genomic study: genes, duplications, phylogeny and microarrays.;citation_pages=545-545;citation_volume=9;citation_year=2008;citation_pmid=19014685;citation_doi=10.1186/1471-2164-9-545" />
<meta name="citation_reference" content="citation_title=Handbook of European freshwater fishes;citation_author=M. Kottelat;citation_author=J. Freyhof;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=S. Kubota;citation_author=A. Longloy;citation_author=A. Singhabun;citation_author=W. Khammee;citation_author=K. Kessuwan;citation_title=Quantitative trait locus mapping of growth-related traits in inter-specific F 1 hybrid grouper (Epinephelus fuscoguttatus × E. lanceolatus) in a tropical climate.;citation_pages=5913-5927;citation_volume=48;citation_year=2017;citation_doi=10.1111/are.13415" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lamichhaney;citation_author=J. Berglund;citation_author=M. S. Almén;citation_author=K. Maqbool;citation_author=M. Grabherr;citation_title=Evolution of Darwin’s finches and their beaks revealed by genome sequencing.;citation_pages=371-375;citation_volume=518;citation_year=2015;citation_pmid=25686609;citation_doi=10.1038/nature14181" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. S. Lander;citation_author=P. Green;citation_title=Construction of multilocus genetic linkage maps in humans.;citation_pages=2363-2367;citation_volume=84;citation_year=1987;citation_issue=8;citation_pmid=3470801;citation_doi=10.1073/pnas.84.8.2363" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Lander;citation_author=N. J. Schork;citation_title=Genetic dissection of complex traits;citation_pages=2037-2048;citation_volume=265;citation_year=1994;citation_issue=5181;citation_pmid=8091226;citation_doi=10.1126/science.8091226" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Leitwein;citation_author=B. Guinand;citation_author=J. Pouzadoux;citation_author=E. Desmarais;citation_author=P. Berrebi;citation_title=A Dense Brown Trout (Salmo trutta) Linkage Map Reveals Recent Chromosomal Rearrangements in theSalmoGenus and the Impact of Selection on Linked Neutral Diversity.;citation_volume=7;citation_year=2017;citation_pmid=28235829;citation_doi=10.1534/g3.116.038497" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Lenormand;citation_author=J. Dutheil;citation_title=Recombination difference between sexes: a role for haploid selection.;citation_pages=e63-e63;citation_volume=3;citation_year=2005;citation_issue=3;citation_pmid=15736976;citation_doi=10.1371/journal.pbio.0030063" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=M. P. Kent;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_volume=533;citation_year=2016;citation_doi=10.1038/nature17164" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=G. Lu;citation_author=L. Bernatchez;citation_title=Correlated trophic specialization and genetic divergence in sympatric lake whitefish ecotypes (Coregonus clupeaformis): support for the ecological speciation hypothesis.;citation_pages=1491-1505;citation_volume=53;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.;citation_volume=21;citation_year=2011;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. J. Macqueen;citation_author=I. A. Johnston;citation_title=A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.;citation_volume=281;citation_year=2014;citation_pmid=24452024;citation_doi=10.1098/rspb.2013.2881" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. V. Matz;citation_title=Fantastic Beasts and How To Sequence Them: Ecological Genomics for Obscure Model Organisms.;citation_pages=121-132;citation_volume=34;citation_year=2018;citation_doi=10.1016/j.tig.2017.11.002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. J. McKinney;citation_author=L. W. Seeb;citation_author=W. A. Larson;citation_author=D. Gomez-Uchida;citation_author=M. T. Limborg;citation_title=An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=769-783;citation_volume=16;citation_year=2016;citation_doi=10.1111/1755-0998.12479" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=T. Moen;citation_author=B. Hoyheim;citation_author=H. Munck;citation_author=L. Gomez-Raya;citation_title=A linkage map of Atlantic salmon (Salmo salar) reveals an uncommonly large difference in recombination rate between the sexes.;citation_pages=81-92;citation_volume=35;citation_year=2004;citation_issue=2;citation_pmid=15025566;citation_doi=10.1111/j.1365-2052.2004.01097.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. J. Near;citation_author=R. I. Eytan;citation_author=A. Dornburg;citation_author=K. L. Kuhn;citation_author=J. A. Moore;citation_title=Resolution of ray-finned fish phylogeny and timing of diversification.;citation_volume=109;citation_year=2012;citation_pmid=22869754;citation_doi=10.1073/pnas.1206625109" />
<meta name="citation_reference" content="citation_title=Fishes of the world;citation_author=J. S. Nelson;citation_author=T. Grande;citation_author=M. V. H. Wilson;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. M. Nugent;citation_author=A. A. Easton;citation_author=J. D. Norman;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication.;citation_volume=7;citation_year=2017;citation_pmid=27986793;citation_doi=10.1534/g3.116.038026" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. Østbye;citation_author=L. Bernatchez;citation_author=T. F. Naesje;citation_author=K.-J. M. Himberg;citation_author=K. Hindar;citation_title=Evolutionary history of the European whitefish Coregonus lavaretus (L.) species complex as inferred from mtDNA phylogeography and gill-raker numbers.;citation_pages=4371-4387;citation_volume=14;citation_year=2005;citation_issue=14;citation_pmid=16313599;citation_doi=10.1111/j.1365-294X.2005.02737.x" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=R. Phillips;citation_author=P. Rab;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2007;citation_doi=10.1111/j.1469-185X.2000.tb00057.x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Poplin;citation_author=V. Ruano-Rubio;citation_author=M. A. DePristo;citation_author=T. J. Fennell;citation_author=M. O. Carneiro;citation_title=Scaling accurate genetic variant discovery to tens of thousands of samples.;citation_year=2017;citation_doi=10.1101/201178" />
<meta name="citation_reference" content="R Core Team, 2014 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL http://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_title=Lep-MAP3: robust linkage mapping even for low-coverage whole genome sequencing data.;citation_pages=3726-3732;citation_volume=33;citation_year=2017;citation_doi=10.1093/bioinformatics/btx494" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of Ultradense Linkage Maps with Lep-MAP2: Stickleback F2 Recombinant Crosses as an Example.;citation_volume=8;citation_year=2015;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_author=L. Paulin;citation_author=I. Hanski;citation_author=R. Lehtonen;citation_author=P. Auvinen;citation_title=Lep-MAP: fast and accurate linkage map construction for large SNP datasets.;citation_volume=29;citation_year=2013;citation_pmid=24078685;citation_doi=10.1093/bioinformatics/btt563" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=F. M. Robertson;citation_author=M. K. Gundappa;citation_author=F. Grammes;citation_author=T. R. Hvidsten;citation_author=A. K. Redmond;citation_title=Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.;citation_pages=111;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1241-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. M. Rogers;citation_author=L. Bernatchez;citation_title=The genetic architecture of ecological speciation and the association with signatures of selection in natural lake whitefish (Coregonus sp. Salmonidae) species pairs.;citation_volume=24;citation_year=2007;citation_pmid=17404398;citation_doi=10.1093/molbev/msm066" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. M. Rogers;citation_author=L. Bernatchez;citation_title=FAST-TRACK: Integrating QTL mapping and genome scans towards the characterization of candidate loci under parallel selection in the lake whitefish (Coregonus clupeaformis).;citation_pages=351-361;citation_volume=14;citation_year=2004;citation_doi=10.1111/j.1365-294X.2004.02396.x" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=S. M. Rogers;citation_author=D. Campbell;citation_author=S. J. E. Baird;citation_author=R. G. Danzmann;citation_author=L. Bernatchez;citation_title=Combining the analyses of introgressive hybridisation and linkage mapping to investigate the genetic architecture of population divergence in the lake whitefish (Coregonus clupeaformis, Mitchill).;citation_pages=25-41;citation_volume=111;citation_year=2001;citation_issue=1-3;citation_pmid=11841170;citation_doi=10.1023/A:1013773600304" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Sakamoto;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_author=P. Howard;citation_author=A. Ozaki;citation_title=A Microsatellite Linkage Map of Rainbow Trout (Oncorhynchus mykiss) Characterized by Large Sex-Specific Differences in Recombination Rates;citation_pages=1331-1345;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880492" />
<meta name="citation_reference" content="citation_author=J. M. Sardell;citation_author=C. Cheng;citation_author=A. J. Dagilis;citation_author=A. Ishikawa;citation_author=J. Kitano;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Biochem Physiol Part D Genomics Proteomics;citation_author=E. Sarropoulou;citation_title=Comparative genomics in teleost species: Knowledge transfer by linking the genomes of model and non-model fish species.;citation_pages=92-102;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=20961822;citation_doi=10.1016/j.cbd.2010.09.003" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. Smukowski;citation_author=M. Noor;citation_title=Recombination rate variation in closely related species.;citation_pages=496-508;citation_volume=107;citation_year=2011;citation_issue=6;citation_pmid=21673743;citation_doi=10.1038/hdy.2011.44" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. Stapley;citation_author=P. G. D. Feulner;citation_author=S. E. Johnston;citation_author=A. W. Santure;citation_author=C. M. Smadja;citation_title=Variation in recombination frequency and distribution across eukaryotes: patterns and processes.;citation_volume=372;citation_year=2017;citation_pmid=29109219;citation_doi=10.1098/rstb.2016.0455" />
<meta name="citation_reference" content="citation_journal_title=Schweizerische Zeitschrift für Hydrol.;citation_author=P. Steinmann;citation_title=Monographie der schweizerischen Koregonen.;citation_pages=340-491;citation_volume=12;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Sun;citation_author=Y. Niu;citation_author=X. Ye;citation_author=J. Dong;citation_author=W. Hu;citation_title=Construction of a high-density linkage map and mapping of sex determination and growth-related loci in the mandarin fish (Siniperca chuatsi).;citation_pages=446;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3830-3" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. J. G. Sutherland;citation_author=T. Gosselin;citation_author=E. Normandeau;citation_author=M. Lamothe;citation_author=N. Isabel;citation_title=Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.;citation_volume=8;citation_year=2016;citation_doi=10.1093/gbe/evw262" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Vonlanthen;citation_author=D. Bittner;citation_author=A. G. Hudson;citation_author=K. A. Young;citation_author=R. Müller;citation_title=Eutrophication causes speciation reversal in whitefish adaptive radiations.;citation_pages=357-362;citation_volume=482;citation_year=2012;citation_issue=7385;citation_pmid=22337055;citation_doi=10.1038/nature10824" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. B. W. Wolf;citation_author=H. Ellegren;citation_title=Making sense of genomic islands of differentiation in light of speciation.;citation_pages=87-100;citation_volume=18;citation_year=2017;citation_doi=10.1038/nrg.2016.133" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=A. V. Zhigunov;citation_author=P. S. Ulianich;citation_author=M. V. Lebedeva;citation_author=P. L. Chang;citation_author=S. V. Nuzhdin;citation_title=Development of F1 hybrid population and the high-density linkage map for European aspen (Populus tremula L.) using RADseq technology.;citation_pages=180;citation_volume=17;citation_year=2017;citation_doi=10.1186/s12870-017-1127-y" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Genomic datasets continue to increase in number due to the ease of production for a wider selection of species including non-model organisms. For many of these species, especially those with large or polyploid genomes, highly contiguous and well-annotated genomes are still rare due to the complexity and cost involved in their assembly. As a result, a common starting point for genomic work in non-model species is the production of a linkage map. Dense linkage maps facilitate the analysis of genomic data in a variety of ways, from broad scale observations regarding genome structure e.g. , chromosome number and type or sex-related structural differences, to fine scale patterns e.g. , recombination rate variation and co-localization of differentiated regions. Here we present both sex-averaged and sex-specific linkage maps for Coregonus sp. “Albock ”, a member of the European whitefish lineage ( C. lavaretus spp. complex), containing 5395 single nucleotide polymorphism (SNP) loci across 40 linkage groups to facilitate future investigation into the genomic basis of whitefish adaptation and speciation. The map was produced using restriction-site associated digestion (RAD) sequencing data from two wild-caught parents and 156 F1 offspring. We discuss the differences between our sex-averaged and sex-specific maps and identify genome-wide synteny between C. sp. “Albock ” and Atlantic Salmon ( Salmo salar ), which have diverged following the salmonid-specific whole genome duplication. Our analysis confirms that many patterns of synteny observed between Atlantic Salmon and Oncorhynchus and Salvelinus species are also shared by members of the Coregoninae subfamily. We also show that regions known for their species-specific rediploidization history can pose challenges for synteny identification since these regions have diverged independently in each salmonid species following the salmonid-specific whole genome duplication. The European whitefish map provided here will enable future studies to understand the distribution of loci of interest, e.g. , FST outliers, along the whitefish genome as well as assisting with the de novo assembly of a whitefish reference genome." />
<meta name="og-title" property="og:title" content="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3745" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic datasets continue to increase in number due to the ease of production for a wider selection of species including non-model organisms. For many of these species, especially those with large or polyploid genomes, highly contiguous and well-annotated genomes are still rare due to the complexity and cost involved in their assembly. As a result, a common starting point for genomic work in non-model species is the production of a linkage map. Dense linkage maps facilitate the analysis of genomic data in a variety of ways, from broad scale observations regarding genome structure e.g. , chromosome number and type or sex-related structural differences, to fine scale patterns e.g. , recombination rate variation and co-localization of differentiated regions. Here we present both sex-averaged and sex-specific linkage maps for Coregonus sp. “Albock ”, a member of the European whitefish lineage ( C. lavaretus spp. complex), containing 5395 single nucleotide polymorphism (SNP) loci across 40 linkage groups to facilitate future investigation into the genomic basis of whitefish adaptation and speciation. The map was produced using restriction-site associated digestion (RAD) sequencing data from two wild-caught parents and 156 F1 offspring. We discuss the differences between our sex-averaged and sex-specific maps and identify genome-wide synteny between C. sp. “Albock ” and Atlantic Salmon ( Salmo salar ), which have diverged following the salmonid-specific whole genome duplication. Our analysis confirms that many patterns of synteny observed between Atlantic Salmon and Oncorhynchus and Salvelinus species are also shared by members of the Coregoninae subfamily. We also show that regions known for their species-specific rediploidization history can pose challenges for synteny identification since these regions have diverged independently in each salmonid species following the salmonid-specific whole genome duplication. The European whitefish map provided here will enable future studies to understand the distribution of loci of interest, e.g. , FST outliers, along the whitefish genome as well as assisting with the de novo assembly of a whitefish reference genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RV93PvoFU4sJDVd1D3Ku9Jxp0QAV8QDgsR1JjBq-SOE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491561","apath":"\/ggg\/8\/12\/3745.atom","pisa":"ggg;8\/12\/3745","ac":{"\/ggg\/8\/12\/3745.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3745.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3745"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3745"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3745":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022p7V+VKNlHiLZf0qs3ZJnZ9rE1lNm7wLIrPTYL\\\\\\\/pEudJkNaodd3\\\\\\\/lht9zibTlxcaJivZaWeXtV+L5tNDYYOq1fmjbGJTDsVea2Sp0oDi1v2bYt26PsIIPHiY4YX0h5DTt+K\\\\\\\/aSRcPyTlY8Hz06A9Qhy0qjX+H40Gy01G1SOUVgmf3GvTr+IahJ7wJci2lDjpTQEuIVib+6pp0Y6bvyeKtt69rMjf2h5EcN\\\\\\\/okH1rJBmClnpSEhl2anoxebGd22hL0xMIkOIlWjBRiIGNW89\\\\\\\/6g9ZwOiBLRKNQ34Y6HmGv19Td9E9XmBZ1+XqHwgERIo1K6VLQrBN1Of8Mxhuvyceijc++C9lp+lVMVR9oKzgEH0WRVyRE92XaTOdpGUsY+w4tECS0zzuEhcmTZIRD+CeYxqDR1AjXUfaXuxDWFq96IXn2Oc7Z312ZyJRYMYkpvKhfooWQ8wRWEw\\\\\\\/6Uh3LeZieJAv+U\\\\\\\/nqOQGYIUTvmoyne0SvUEtAW1SpiEKKUXQq6dTy9\\\\\\\/jr2ekJ2EbubMD\\\\\\\/jXDc5lMSPMrtlixIlPhPD2sKTlB\\\\\\\/2Uqe93ZrgHFdCNYSV9AEpemfoZ+FiNr2nMxmgzSd5YbWAobxTyqAvS5BdlkudAw8r5EWPaJLNx8NDX04WdcJ76vcirMPRDksI2vxqSd6nIDpOv9Aev+QxRkgIUOXGitBQCknWA4uKW9UODkoo4GBYjdP9IgUm+PtQCoV9\\\\\\\/SbR2UCA03i5HB8PRqszy+276mGt\\\\\\\/1BmxaoPVmfk3p74tbDWLra+UcpiCQ\\\\\\\/qlCmt\\\\\\\/3qpR7j89dYgzxiA1LOAtm8sRBwmtPjXGedmAQoCSLmk+wZgE11F8LQEWt55ttS56UUAvdbiwSMT1UiL7+RXygwb6eAeiB5GVtB6UgWjejvPQMPmvXDyUyX4nlrZRlUL9WVBwHX7ZrdWNj\\\\\\\/8aYxu4snYx0O7wtRWzWc3nN52thv8Z5H7neeFqK42QkbMBBOLJ1mLZdoMgjP9BO91qo+3hrkNCOl8jrqrN91ZtRX90GXg7CsMsTO\\\\\\\/jGXRY\\\\\\\/m1mCqOmSm\\\\\\\/7GgsZW0HSZN9fMrJMqfaD2iRAmtcQjZ9Mi58sLEBHxTuUCUK82vJ834m5vka\\\\\\\/TFC7kBoT6ApJZeqKVIpD8XhdsyYAxMhJYpFwHT9k\\\\\\\/OSDB4JaxqsskXTAtPbqGlFokC07j2bkFtlDQYmS2lQpTZeM1wdTGQUieWGVEvZgwxpTmsgI6351hnXj40LO+bUReYPub\\\\\\\/5cAHwucMbvDcDyxLMWtYCjSRl7ktplrX3LVwqP6U\\\u0022,\\\u0022iv\\\u0022:\\\u0022Dfn\\\\\\\/yzCnu4PQs7ILiRi6qA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222dacc3255e0b5c0a7cb71c884767a025a8d8b756db53d2133c20ced24593c01c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kodNYVM\\\\\\\/MrZxYMiZ7fLVOXEpywPhUmqC+JLnGi1\\\\\\\/R4W\\\\\\\/OUGufe\\\\\\\/fnUd0IiAhGxoHBcmqoDEwkLtDWsHaLld5KlYAN9Ic18lAfULymvZXWlkAEqt1VfzXQBAaaLWk99h9hSJd6R3dqBBO+xZGWZT5BO\\\\\\\/hqJG4mEoegGDft51t+hhjRYhD88y90bcASjuWNi+u3+QhPLsPADPt\\\\\\\/Q\\\\\\\/IBv\\\\\\\/IHDHH0zC2BxYBnYqVBLWkwnMtmlHLfpZSE+7dSb387iwo8ZLiOH7JzlrVUS8GiM9KEPQI7x1GP7HXQMMHEL9ispywJXss3PMHj9YIbYb7ZUpT5Zvu6jtPuW1YX\\\\\\\/\\\\\\\/Gq42lCHlfzyYOdxkeWr6U\\\\\\\/vu8jrVhmLxE0ogBvGh0YPx87gEhdInLcNRgEBjk2XuRvTD+DgecE09OqDIutCbV\\\\\\\/ST6\\\\\\\/k1mH3JvczEoZzE4GkyCC+4OW4UrsP9SSznOM5plvtXcbMGK9nmdavjTrG4lncomPYB6rRQCLjyO7RykUvcFEN9ocgSIPKFdy5Dg\\\\\\\/RSkXTcAFs2\\\\\\\/RdnS7H1kJwY+B6S65\\\\\\\/OYvQqIYmFlcAd4IATdty4R19VWO16GWEexPelWGGYiygAlUcnavpEvOc9AlmcPawOHxhODhV1hUWC2ocdR44IQ8jRxxvhlvBGSb7Lb4kL3ExmN4pGR1OQ1KdDpjFvwRwfQ8BbIPIss3Pm\\\\\\\/mJACl6bpO3pcIHHOljtz+eQimYNA\\\\\\\/Bv15rzgGkRiK7w1LBAbpi0SdAMfzSZn\\\\\\\/4qYSRNeRVgbeyJ5Vss1ABD+kbujYX\\\\\\\/L9CdwXeA1wmBRSIfR0iscmWUEOwVKaQI65f710X+UpHfUHASoWYpluj9E5B94zrRxJn+rtlK13\\\\\\\/kORidUqH0d+p8xXzmvOIwDalfcfpJ9MIxlYcpwL\\\\\\\/kny9ZKPI8\\\\\\\/DwLUMg+H3ZIPFvi7igahXEn2GAedOWe5NpTaWxFcWKaivu2RRrj65WfeJLfb0dxytpYnJE+s5nl5ZT12GNWIDsvLShIUZYDUjV34ETPFlRqKCZ9AwtsIEEUkkP2MjlyA6pU+mrK6LyVxiStNqkeQQwI5nfFyo4EWdT3GlI+YEaaN0pQQbkfrWa7hdi3s6t9ugccuJYGFqmQHzES5uHA0Piz4Q2+AaJkxrFAMNUcnsCzfr8KcN1XYvWDKgdT\\\\\\\/hu1PzO3t+GCqPXAmNHTexA8TXxmxHgzGRpGuo9rndDGFlgsxUzDwEw3J7lbAvJ6FdRUuTJ3PC+\\\\\\\/mku\\\\\\\/ph8TRR2HJc3Hx\\\\\\\/qYo2RDASmBhO3Ixk\\\\\\\/C2zJdx0IcyQE2skqGiQBcUrmTySbKjQVhx\\\\\\\/W1565GUpK4DyxQsQvkgaR1vZD7fHTVHoeRsx4mqxqJchL4UTz89V08Grpchgcc3oR6s5\\\\\\\/YSWdUmT2s3x0cIeEKGdRW+gbA5I9NULyQ\\\\\\\/0T23xYSg0i71KT8ebij2XSXUo91nn3w5IRe5bJ\\\\\\\/mKzpNwPVRruJ60FyX7429OhtOV1zIpUFWFGN87ocf8U3C\\\\\\\/7fVkstVvtYMouQtW9yY2QQuuNYCw+mMogt7TItRy8vJ0agdWJEQuj3zmrn+XcFZlvx69AHoDWasUV45D9sXlTrabYdc5MWkD0PAmNnDGOeGtZdahG8lnoKnWAgSy12VgOY8aCq1jx76PNfo6fEnEZ3hHsI9g2J2kLdy+Cmpi2QHYZQ9D5eY5w7mVNDMynGNecHheC5dKtgXXKMPPKavEoNF+TZkB34x29vpKjCHsNbZFe+NIl5+E6LqmouxaoixD9tHg5Cj\\\\\\\/z7GcSQ9YvVAwB2AJJpemmYGh59Amcg8VycnDvZSFHAkjK6WAD4G3T1b1kD7k6IhKzwkSAlEHv+xGcloqpwV3ec8weT+7ffNvnmqQQCgerb0TO0Bsj1LXX0uOx49mHdO7QDyGEhx3dqrIP8FH25vT+ulLe82PyZbufYuMSZpeRS2mR8vC8LGY522SUTFmHo+TfAlvFsXg6LYsQCJOB12MKlmznuBKsLa1+Vm9ZHklJ4MTaSM7BSyTOYKdPbFbFGevg6OoWCJL18DFgO9BcfJPQASwg28nBC9FIWQ\\\\\\\/VsZzh3Tg+hFu1I1asrVXJqzUJTBCrXWXVCjAGtWg7DZLY2tpnQWRzwAs9mBKU1hSGs3th+8fPTbys83rXvln15D7rslE2fcekXZa4dK4JS1lrMUOTL6ut83Gsvy2Uch6xK7a0pm2g8SsNHNT5YYiOdUeygowtjOQCQB6xReUSCjoE4ZFw5aaguSkbLNMDeQT\\\\\\\/xdlMj1N34hgYpI2CUAPSsMVA8guvi+3auPrpY6q0ee\\\\\\\/O7kLyUP8BqHIrkJdd4+s\\\\\\\/6+iRPgJva3UCcqtHwbIbDpSSz20A5MPuiNbFNGEMBZHZmFlfX7C5besM5xJDOom3JIv8XcBXpKujmrQt9vJpau3yPlMswxZQGjhO2T+RkG+s4Aifg5JDRO3NQvXcwTnDzSHhC5U25JYuz9uuOSgm0aF0bWG\\\\\\\/u+YGa4tWOxq9MOuJ\\\\\\\/aP6eXGP8BRM0nweFx7xp1dBtNZBJPSSiZxXvnkf4LMWdXXSNBFja3Yg6VshsHVl1wCiQPVovt7Fm1yTEVCY2umK0QTtDk4XPDSMXDebyDKXKDcoJyBBR+o0uUfHidHAe7Ov7\\\\\\\/lORSBUoPK6roYrncXjgCvd0\\\\\\\/XQdff1Pq37YyTXVLDuYX+x4lEzkRybH\\\\\\\/5T7mAvSg+76mGAzeskspP8DPms1AlYzQ32eytYT3ELlOvvCmVdeurOFbw8\\\\\\\/kvIrYB5\\\\\\\/PDftRQl9w\\\\\\\/AkmGFqEsKCTbhdPpm6rXiJQ4NfsW1t7N7NbzDR1765PXpELtYPRCwotN1p8uY4Fy086FtZTBL6ZNnocFsZ8Y5NLrZND8yVLJWgRWSicckf8V\\\\\\\/MaNjX0syFz8yxlA+OZXVOto7DVbO6Tqm6O7l7zak\\\\\\\/3SBG3fW3ncAoOxGSaytNlPvmhlkj3670wjtFBC\\\\\\\/OZoKmMnkRYRe7LaXVQtXsLMZ+QK0QuvwLKtnvXrRdupyx74HtnbzifBbVQVyZHz11LI6URryIhhQ8LxaSSnh+ocuAuHGQp2Q+BA97vJoBKR21\\\\\\\/L6Y947CBgmJV6LxWJ6GGQhO+\\\\\\\/MujZNd0SwXVTzIsfTmsc6j9QhjKFvtO1zLyQaxtNHayYTkUifzQrfDAA5N6IuYEpf9WcpxcOtD5TRdUSLrcE2ASIfexSl\\\\\\\/mIJVVoF\\\\\\\/dUjxxi2\\\\\\\/21z7s2ID7kkFeDVoRTs2uNFrpYm0baX0w+FHREIWAYruWY+MRHgDCucrJKBwQ3EvAYRwzD8q+\\\\\\\/rweWHPAKmLqPAcaH6Kl+oTspByZBVqwa3\\\\\\\/xp+Q9DZ7Ya+t5\\\\\\\/K+pnHcPnM0QKIduZcEUhnoP43+pBJAc8z9h4HeP09Yyv2LHj8N\\\\\\\/\\\\\\\/sILRAYmQIg72bUsZ2+G+NIalNZKHgUIT8fXBG9h9aKYGANWTnsSecohhZDnUHD3u\\\\\\\/jmmi+CnOF441q7LUGTOrAwfQmkXiwrfF+YdzL08r+qpfoWOsIrgCFUNjnwGZgnylMqx84p+qebE2D\\\\\\\/zIUlKydogLYBnTuEV907XgPZ6uw\\\\\\\/NiexY\\\\\\\/mKOqQIl4LtP5rfrJ\\\\\\\/OABpc6qEj9ScnfR6MaG0mYqJAaQ1TUKWX\\\\\\\/dxmqgfIXE50rJmdBIgX2WkpGw1UGoyZqzDmlTwmEWoMfIDRQ5g7bdY4G4Zdr2SraHOtuSSWlIUtzmUTQBfVhDQR6RO7vLowqIG0tfMVutNRFfzbZsCuVK\\\\\\\/9yZ0j7PMMkuy74dMouMHZgGDaReTkw73PTtvmu3whGS9jOWxh2SROGy4YkA7PR79VzD4G9Kic\\\\\\\/V1TNJc7FSRoK2qMBD6jvi6PGB2p9SVsZmNBBw1BZ5axEO3SZGy4+M9odO1bDyRZsVvYuJBtoqhN+wcW0RdFJjwCAEkVD\\\\\\\/CGtLkrR8F0odsP+iJwqUDRceuJ2O4U8T\\\\\\\/+9tkcEQ76UvuNh9MIeVaTiw9o9b7ZAiJPycHf\\\\\\\/WBo0JZwtyqKH6hEtq8bgEY5yu5Il7eca+2WCtfTm++qOCLI5pMiQai\\\\\\\/\\\\\\\/ru2ica2zCOjjeYTXD1R\\\\\\\/wQgolhhp1+v1K2zFiwY+K38LFTyxCzXh+EghzyjtCbrM1Fp\\\\\\\/83kPQWs\\\\\\\/zEgQrVvr2\\\\\\\/EX52sL3sdCuZhXl4pKWGYDsEib0TVPbbsgWnYVIoIYHVcrYIu089XmwopzvAtY6Ye3W6Ou1LT6l\\\\\\\/PNVw7cVIQ0Aovrri\\\\\\\/rvx38u48DDHBLczsAaxNOht3qFDBTWAbNsmd9dGM3OnDjrybclq0+f3OFGrfI+TnUkWprrjTv4mVKehaGC2duZSRQw4VC9vF0cjKmi1sQlPyL6KDGYHTWfwPfCvi6ixhm73xUKQPLBlHQZbobAC7\\\\\\\/y1aPt05wr9UL3dIjykqoPQ0UMTjNogoLiZ1K\\\\\\\/+P2dTm7tI9m9MsXqkPEDGwGNg8IrMzXPAr2jshTSwN27RGfjtw63U0AYQFgdUhqtiO+JYINykN8\\\\\\\/hs+LsoJLT5WPCEwukjFpTKlM0YyUcsQoH\\\\\\\/3dRPLo2vcbLG5QszZrP78HjJC7tREGqC5T23igRSOextgl5VbNGyJ5gXmrOjI2pf8aeRSmmLKbfWLbJF2j6F20ETTnhSMouc5LjXUlZyW3mqVmzjfd6gNWSFDlrHY\\\\\\\/9xKvFbgCHCPAfwRJAy+DSrzc4O6l74Mw3kjVgIuhxQtvUnfD1OHtzenPLtR1gTjo+VJgUjg+xWSWUhBXYx8v0kJa991ckeTxd2NeE99\\\\\\\/JSXlaWhbPlggbyqR\\\\\\\/o6uoYUvUK3xEAt8tZiF8dC9YltzDYfc1BLXrxXsuXVKhpMINCaum47YyVcsr7ok6FrHSmMiy9hyAB0ooR0tP7IKJtctk3fybq\\\\\\\/h74mnPnTYwqz7GEZ6j+lKAjEcSRSvCDYj43r3yAPLhtpjFhZwnlqRMTZkHI2Rz\\\\\\\/xiiJXmiQZq5NzEC2L4Xs94XSPOBookUHEJ+z2bnSXI+RlS5OHtlYAqMln3j1XLcEMSHik+464\\\\\\\/u4flqupuCEK3S\\\\\\\/ug8pmedJoIsa+FaeivDNUMzqmbwDeyHza22jEwyHyUby65tB9WuxEmZXSPV1+2tOIwSkB5mO4YxV5MIJitmFm2Jm1gy5YOxTtW8XjfXRI\\\\\\\/oHi5+lkxl9Hoqe86rjbjegCFpXrhqMHOQAklm+pEd21trvPXl8jlghyU1bLz2A9wjtWJPsV0isfk9BapPNwvN1vBl7Jb6A7UryNc0oJg2kj8YJGo122AraVL3iIlB4fFry1fT7RETYupwKIN5usVwvYclTuu23+m\\\\\\\/Zq3F00kAscVVyGuNTiBzgqF7pR9+akhB4ImxWdGt0F3cAkIZuo3Utpfv2qJTv6fwY45KSHbKUj0g1S0eRAI4pwrDv+p\\\\\\\/4tlUgD7IoJrtz1bP5iAjZFZdxzbxKr3D11eS05lVclg6fFwt0LaR1q2n7RPOalI8wC935qY+X43k77XGrD5YgmiQRIPRdtPhdA94PLvJuhNTtW9xqKnJfRpGrzdmKfHpl3Yt69D5v2mzQar5w\\\\\\\/F4blDxOkeAQEEytoVkDt1InL3SrpNOLH0eJABitwcShfELVJZtBzFrn\\\\\\\/VHIlIrDtG2eZUOKZij+dsgbhby\\\\\\\/D\\\\\\\/52Gbm6J2anGq\\\\\\\/Yy64KxvZ7rRZ5SQpvf0uayVeAA5d+Vi4W3kXq56O5Z\\\\\\\/lB\\\\\\\/BigXDR9MCely1o12lJsGUnKB5StwUrvJbsXarYSX0bOY4n1ffO7ldtGU6Su7fXNdK1t2KYSM6u+0xaUFCHoW+JRXbRTdxVqgtBPEImx+XlTJGfg\\\\\\\/b\\\\\\\/ZRklPnxnmS8KD4vXoN84hN\\\\\\\/vBvK9+QcDmmVXHHin5AthQ9saVXicuNytnK8JFvjCpqISaMKtfDo1ycSr+QOhaLaKBQais6HIqfFVBcRfyd8DXJ9MuiZRS2S0AhNBwipIitU7ybWF69hYAIdy1NeYAiPEdOHSron7mPXUp8CHR+EvLxyShH9uKs8hYiHW8XYExHZk9xXS3zL+ykqo1bfXxFndfuKGEgclIZk9rsxjpzXOTciuzPE\\\\\\\/sS98ZW3TehMDNtZTKPretYptCV78JmuLJiVjsZw3M6MsS9qT9zMOsdCEamMzEBUBovza504\\\\\\\/k5acJuenEHcJNpbOMZJ+6NXLoXObyleW1yq+OQlgLrj0jzEHRkL3UEOWDWZHpgr4lLUQfs7ObltTJo\\\\\\\/s6pXuyws4ohPZ1fIqYnqBlwigdwcfcUAqhBXw7Ip83EecoYkbFJKCIEL26MHv4nP5YkL+6a0+z+L6Mv0lnrpAhV5wbVJyD1ZbvRl7wgKmK79oyZLuPbzdBSp7BFOKBJ21Q7LCURTPNdLZoFgxeFd6LS3CIeaz+9MDYDeIyd\\\\\\\/0wwu0SoH2RuCWFOxC061BCnM\\\\\\\/WGlP8lrDtTjl3\\\\\\\/7H2GuiJ4zk+1H+leX3b1Ixhq5d6J7NjHzqOwg81DXWbRTzFMQ9VkeGIb8\\\\\\\/yZoUFaGfYtpuzgTfzAYtrGA27uS0FFTBevvlqKuB2lEvWjVhfUyMFN+yQQwiWmzDvvmmFQPTpqaxQ0CPMvoJUM+l2fp1JhgpvpFZT2I+37ZtH4Hf7kPILQ3VVYQ7Ke3OZXn2gulVJD\\\\\\\/kaVJ05FUTx\\\\\\\/LzfbH6p6Q4lg23SXbjMjC2bvsgGpgIx7V01msyapzrCK\\\\\\\/KBARz+lSuV4szQQk4inZYQuNZOfAkRQMjA0t\\\\\\\/i+7pmS7JlwRQ37uD6HvQsZjeNtoUVZSp8R+h\\\\\\\/36vDMLwn511N4X4ZUq+zxTtWn7O+KzvWwSCDNBzGbFFEuMSjUU\\\\\\\/vp+PzmVHzHzpKQlp0zAq4WZI7HXpn2XZXDUFzHkzWtbn6faOFYw+ZQ5ENyb5HeCyGO2cbrm0EYSI+hzQC6\\\\\\\/BT5aitYXFcgytiqt0uAW00IhZS3RJRGmh5t+zqYjjPlN2OkrvPkoX4AKtvw9MwOHLMtPyuNzeWGvileTkcqAs8thh5l7WmmmhuDNoE6kfGsEpnimI1l8ZdzRdmBQURN6WKpBWKQI7CuXL3pZGAeIiJTRiCArjv9MdA0rZqQ1GKrSy83X6cES32DuuSIvcrWnVK42lIe\\\\\\\/D+XQ5C9VRyN2I\\\\\\\/bgyEv5CtTbXx8T6penMQbfsz\\\\\\\/8uBdvLqvsUL7myfY7CZoNGo9VLpf27\\\\\\\/K4CVbNYRN1x1em\\\\\\\/SyY9IroKLC3YRMtFkbOlAwWW23QQwd\\\\\\\/7lPlaJvjTjkZYL6VkUtHfoLah11+w8vRRU9ZQ3CU1FYUp1YU5RP8+NvJ4hGHvWA9n4kRYpHw+DuKHHeRMwUv+JidOzwnhogWgg+X7ihE0QuKHGAdHPYQhJpWPjxyW2ux9PuBGbyfs3yoW3UVPsieKhCBaKxy91+4apC6v1PDkL\\\\\\\/ZqgGdn\\\\\\\/YY2vfg4i24WuKrSBGy7YtGEhmfYQlKmbqsZdyXm32UlApuct8q7+X\\\\\\\/PL8sA4tXj6GBOxsznBpKqDXt\\\\\\\/Rn86ccaIOfScihhrwQMBPIcDMA0vGn5l6n\\\\\\\/8FclSmv9SRGB\\\\\\\/SVIDv\\\\\\\/EF4gfUi3aol22j7u0ym0VNMZP+pQm\\\\\\\/FBQJ\\\\\\\/O0hHIwl1MzAWYH4eqgIJOzwoRlsCzxFUHQliiHn0zkbOA5xjZHUigVjXuTXVkHH9EtYG+8y\\\\\\\/tvuxVgWcgyNWbVfpMIG++nS4c7vDDySbm8MmkD59aLlF7UAUYAQauR\\\\\\\/0eL9wDv+8tmHdkL6tuIYg5QlI3WsgFNp6tpYAiBqTXb9d9MP0zvuZnk\\\\\\\/bfKFCFJ8F6ibgnPIwjbIBj+UaGpkGRqcj3w\\\\\\\/9pSvKMNwtvwNRGWyOBovRijQLhxYIBylagv6KSym5PxP9AsXUgj1RjXpfNqizuSbKphzY14L7iP7rPp68Mi2vaY4T2wh8aQN0uAMatwM2s+fzI3unn7L2yfuwC2b3czsIO6iLQamOf\\\\\\\/bLGB8D\\\\\\\/EUixA9kGu1bxBGlu1cDM\\\\\\\/4OH9NvF778\\\\\\\/6o6Yr4m2\\\\\\\/1nskn7s\\\\\\\/X2BtnrJ+Lutal6E2M7\\\\\\\/F5i3+N7we\\\\\\\/Klrc0uBERjzJv\\\\\\\/9Hhnw9T+gsoWCGCIYoQohKQP0QV0C1TMdbwxg4yx+Iw7gd53S9KcIeG3+ER3TOXyakI1jXnBFxZvBGbXoEh\\\\\\\/oJpP9OxECn3vUs+Dk9Bbj0Z5Ycv3UbSoWPpE8bBPNcSYBeYf8S0\\\\\\\/pEPcLmUkBGcWJ4TSo8LqrJCW9YTTFHBXUlV\\\\\\\/KPUI6Yd1xusEQZ+\\\\\\\/+V0Leh40Es7xBe+ucfFw2TZtuOVyN27MYpDmdKEo+g\\\\\\\/3zsuow7pTqlQlY11eFKDCtd6x0SmYKOvocFe9riDZayJvSXkt9QXRQ\\\\\\\/zAhjBmr2evCnvvlhqrIGqDAT5ncnnpnXtFbvXVaF9n0XkfGcCmE4UvUGSFsp+pKOe9C7HMl6QYvE4h\\\\\\\/ES\\\\\\\/vP4CLhzopKSs2SsvcI4NWmDqSNjEiaB9Pzz8uQupaoi5MqBezmWwaucHmT0EA9FicCHoJbRHcwrsjTWXb7LXrCql9QUo1\\\\\\\/1KuSESsNmBZyQ3IwmodFxasb\\\\\\\/MCxDuLiehpc56az2FJKcKMMv5ujMY793wbYC9Jzaj\\\\\\\/7GQhPiEFDuqEb09fHt+9lxBNbwJqluCK14m7v7Aj4W+sTU8duQvs\\\\\\\/Kmkmp1\\\\\\\/XwEM7x4xMpP6UFXk8VZM73bzYX3gc2ws1wXcK6Gs+eTaedwzNSA25OBU182W2K\\\\\\\/VIequGkaqVRDIceXDgUBd\\\\\\\/dqQd0FdqtiFqGtmW45skhoNu4p+FnqvnsRoeNMeZalDS7X5GGL+H+Hbi7qg1qkYnhJvIaLG\\\\\\\/kOuF7zGnay+6atyvP2itlP+ouZCYcfIIeNZ4+ZNTO8OUoSNZxHnufuFwecPlRCvcCqNrBKo4Cp8LvFbkcath06Y2RrW1VjZJ4yCZ0kXHTNJzvzIGg1fQIJH9ynRQIsWwYva6\\\\\\\/\\\\\\\/l+Gcj\\\\\\\/XyKL06sEt\\\\\\\/sagUXbAfAXsqQ44Cm427Lle\\\\\\\/WfvqQDTfeAoFro0tqYeRvUb25iYZHlL6NcvFF5Ht7tZV5o9Y6E7UplHtsg2DURrJ2rjmjMtCQJN57plb3UYy9gsBIP2XBiaSG+ZD0pE4n+gzP1VU2XSQPrOxDQZ1sWNrjwwbDc3cBamjAiENxW6D6hXNPpQ4RohK8y5oUnJv\\\\\\\/DNFqzvciGC8wJxKA0e2meWKjiDxj14veeclfpyOLMv5LDJpZeM4DkDNaU9HlOuIFIVrtArSwOMjOuJ8HhRAWqQXx1YuG69XE+SMH86KC4vfny3upYeiehLgoZf6HkougfKzhWG04ZcTnqdkfYuht3vQ8meOiq3X8Lro6jYj0ae4+Tn\\\\\\\/1xGWyg6GczbffO09wauU78lAt6iOGDA24rIgfcA3RxKS\\\\\\\/YbChl7FFWiFTzxvrZyy8cr7D\\\\\\\/FpV3C\\\\\\\/DAMYXchSTRGfsscS3g9qXpIjgkEYdmskg0IxroAbs3Xck9qcTrRaxk6MOhrU2\\\\\\\/\\\\\\\/y4s\\\\\\\/MxMntscrbsQO3KBejGZvLoc2Q6AyDI279muUbiiF57xmtdogRkfG8qdlDOOS0NSKra\\\\\\\/WSsf9YAv0rs4uoGPaN9fdYkQFtSy2PhI7D\\\\\\\/blLFWy7RBuubKezuw2WgOifVf8r1BjeiTNtATrerTJkB7AZ71OEtYMue0bNj6\\\\\\\/myWcA2kE7hO9FqbjOGftwqPiKbOSrpFH2zNuYuvieLALtV+fJP9\\\\\\\/VWz3ALSRbUhPeNzbrebyk5BxRnz8dbvkdZqJBVFL7bjH2aE6SuxO5GAHnL8Pw9gO7H\\\\\\\/zKcYJFCwscoVKqxaGOPkk9QsRCXt7qOm+3nCWS1vkHpom5Zud10ZnTmWQ+AdAdsC23Pnr\\\\\\\/Nn4GS7ssj\\\\\\\/LFZoBoEToXYjQxEpNZ5ME5oXe2cL7oKmcjNIllQSG++8tuQe2H\\\\\\\/UqjCMAX\\\\\\\/KgjfVNMDKV6OfT4HSrIKjRFMQZ8GufrFxiIYhcLJEXtnCDSVFoUlPACMOje0TSb2SAlVNcxlIcktUHRfr9ap5aEqT2FXvnvP6fDeZYlk9X52h3Z1+6RDxyrl41VxJ606GPUkjN2PtuTUojCPnjL1H+dZvp5Kxc54DE6KVU8Ct4wq0iQZSpStH2Tw0kSqSDubhYcFhv+QVmJH75TKdRelLBk7TuoZ9v\\\\\\\/JI14KSSvEYSfg5xECCf6oXlp4Ivj0m8KMIh9LfNEKhZyjdMCSX++m27Yuk5REA6taPcpg56mEZEi7QWlJUcwnjDVPBpqnXbRciBAyy5\\\\\\\/RAFmLu7OPGEwv4xqGoPIJ1OJPewN0JXPQCmr4IeZZlQbXWlRfLWBqaRpEdRRq+4e1ruCDW3irt++eq8iOLneW6ojWZIg5meKM0hDhnBCJSGFc7iRugbS2WmtOZo9qhRKxwsCoxt4RIVA10BRvboTsyUASfEU3QMoy6VHjNU8A1DSCF8s1SrDYST4a5KKueKgA2YEt9PRNG2yHJBZA330REnLuTN7RJBy6k1I2osAxpYJDPz0a2SSOEusv6uWShUgXap7KSh3Wzrbsal6FKqGWO4w6MRW2H7wzIwup03Xq67WqxACkgdqZ3VkQuSjpC9JAWG0NeT6AeiHMNYT00tot7q7KiipSbWZeJCIPISMBLlqo6+asOkkrsE+5h4rtKWWvuSOyzNoC1biUi\\\\\\\/y4YiDkSWQnbrBdDgJiNSHL7UOlEqVvWCRLdu3zT+0LUKBh+kkqqPd\\\\\\\/GDpXWDQmwYitMnb15uDoc6t\\\\\\\/aaxu3WnctaNysqyJ9RP34uBYes9Mn5yeKNpFJURJX6O04sZDgc09eScbrKwnMlkfxiJM2duRomLl8CVP9WKZPYCRZruwSPRryM+2PWmXTFw4iUdKzqGNxfClOnHdDMd1cCeUJf0TsK2tO5THzS5gd\\\\\\\/DQue2XMkOS1kejSpo78NKi77kSJ1hXZ5ioudmodbS4rac2yQ\\\\\\\/yWCa3\\\\\\\/etfGMagTY9XjXj\\\\\\\/ws\\\\\\\/ggoHNPrniSY+Gg9VIkCfZL1023DcmSNDWokqnZLds7X8l\\\\\\\/TJemgUyc92iZQlfBgw9cONOfhN9u6WDsDtanjtIfuHaRWEIY3NLVfIWrQYlB6iO1t+\\\\\\\/EGS5EugIapC0SzMlzG4nq\\\\\\\/NRlTKnXK\\\\\\\/wy7VDcu0YmsZmZ4WSueNdwnfpGsFUtOzdrgzKlM2cNMxo8zGvrHgqRgBTHbxfdaw\\\\\\\/1MeqC8cz7gYXYhg83woGUJbusiHvPypoY4wZsCA0BCDMfSun1HM7wIQkz6v3ZIRI1YeSNaJs2PIqC1obiQ8T6h3tj73zHRUNAHFef4iGGUw1B5Q\\\\\\\/Mi9I271lvJfxPNL8bPLFo4fmkBEqWwglo19Fr\\\\\\\/tHtfYmJaEPkvWz9gkUhgm2qffR7jOAhxUWlmzcPQ7TJD41qKR3TLuFQs8cVpd9cbPyGqY8ZG0It5VwJaaQjL6K8lVqYg4bMJ591t9u6K6v6kke5MUVtOJOnOgIeuq46RQVSqYxwq2n3ehQhhQZE+byYyBju9EK\\\\\\\/jWxMdMuJ2xaehKHPAuGrVCYAb1ROl9AFAlElzogZlXepmvlRx2R06Z1Zv74Fybbix+GVuAN7azLzcvIajYWer2iwPT8N+Z9GzQiOxUnjx4Y1tUD+ch+U9xd50L9BOt7SfWo9h+dzyUu8X3ct38uAEEonh0Anfu7l8g3bpCsSVPydKva2mHeTQ7CYSYpfQpLMIILdEriPoXegGIHtjZumbi63byiqszKjqe\\\\\\\/5oeSLHBq57ZPqF7X\\\\\\\/sDFnkMVjYqlhAKBRILpNg14TuHzJLT1oa\\\\\\\/QZjKRe+TFG5CS2LFFlJKKK5V2MG3DpXSNSA89SZqF5xrlWyBRow5KVCcSjJXrH33QbQq8uGcBgOZbGbKyZNUZVCiEJxeY+LsUCeSPTR0srAruJKuVhw8BoS8xPmd\\\\\\\/4+Kn0oIEyZ2WLDpsaW6b9v5Hk\\\\\\\/DAd9lx1Uy3ZkfPxXxrkyjz+tP4GvB+hkdV2bbOTn+VD0CqKOJR3qgiEWzewsplkgUP5ij+TEpEJQ6pk081urdgG2zd7FQk\\\\\\\/\\\\\\\/OgCxlg0NBIXisLc+j5HxkyldlBr+lCH8fdxf40NRjpWiCybkqtR4iALMwjp\\\\\\\/nRGm6+LiHhga+IebyKFAD3dyaaJ0XYynsPcnmjWFUi1IylYFdXZbNte0Dj0oYdK41Sxz7\\\\\\\/kweb5ykSsT1N7oL12szPpsgxwDp8X0dD8NAshU\\\\\\\/M\\\\\\\/ijsc9X6kmWRN3vIRXTVlo85M2UKtDt4R6CGbmVP0T4Yq1ZAETod9T0\\\\\\\/W1KTkDTEBICjNjaznve\\\\\\\/\\\\\\\/QaEhUSgBJaD\\\\\\\/pf9P2LyGT6Hn72tItAEg0o7bdIHZ20ftMz5L01+1QQ4563LDxrVONlRaJ4y\\\\\\\/2juIcDU+ee+qAH1YZfJuWkl7cXedOzq\\\\\\\/+PaHdQooFw1Uzx40vNMpLRL6PY9o7gXZ7hGefWmgb9qm3AlDW7WjhUua51tnYGcO2ES6sqR7xX8CyM+X5ezgWyexgqO6RVO5SqNzE+xzr\\\\\\\/oxWbZhfsTkwRTCyr6EWPcNuVQh2C7q5viYYbEnKLSZNgT\\\\\\\/kWLeeIdtxS+wBpm\\\\\\\/2fbOKOmYMiuypMAwzSrdP1ETdmKbzK0rNRxqeIAEV9yAs312gGFSUT3XUqCvaKQAh3Y\\\\\\\/o07xEBLzPOsdgWeV9exUOhdaxfthPkzIvE\\\\\\\/v\\\\\\\/As2sg5ptFjVRzITnXELjuXEuj74C5D8WE5ltcaBN12oUnKVhbClheUKhC8hj1moqtJ710rkLYRGEHWZTwDBWKyPsNJo3luJ3r\\\\\\\/ZcFbhNhKyIOC5LZHxegrREW9Ir4hCwju2giXQNAOwDtZXLWcecXj5LlQ1MCo5sxcysUGbvzHchmCceVQUcZqLhbSLHzYV9pDYPs+RH+LTNg1Qek5ii\\\\\\\/UzlFau3iUToFlLltJSy58RHCaFCmNDuZXyqP9cozNXKntOuNpKwv9FOf6rA1R3WOPXZlPAjWa2hfPW7yXfr3+Cd57+RNJdb3GGIDgEo\\\\\\\/FyLVpGd0Z8kbWTiHUry59hKxk3FQZtVp7Y3N6Gh\\\\\\\/\\\\\\\/cA+p1t49ydXIv95kCSbcpUmwGtyj\\\\\\\/zQ37i4pX3hUaaWl+Jy1DcZ7WSQJ+kg8aOGtBPMUQk3qK2kG3dyDz4CYkhlb6i2Vzk4FE\\\\\\\/UfYzTFLBNLJGh5\\\\\\\/LXqHcaK7IWtUk3JpP8NBmf8yaPVO\\\\\\\/vYAgVNv\\\\\\\/GneWAigBpzcw3ykvuymFLsl+3X7u2Rq+EZE5jrgBPByAnoCmAqwU+4q2v0cAvGrv3S0r29vVFGHTgiRI\\\\\\\/KlH5n71RN0UFeLxMVhZ\\\\\\\/Wy4oWTTh+q4kkYq+ULprFIizcQzaeErbr7PclxwDzlPut+Ui3mfXzhGDTVJyf7N7rauQKx4qfyJ3FD2ITvFULrWiYqV2SAcjPFUA8+z0R0QMuEiLK+q3+pJsAMIr39zrRKs6FchQZs4qBR29FJZ3KdAzHsjICdQGOaJoJOKzmVQHtjR\\\\\\\/o19z\\\\\\\/Z4Zefvi2BFZwtV62vicODejiUvXdbmd\\\\\\\/BMfQayIjUsfa+QLl9rQ5jtbm6djjlw9Hke61j4NOrkyWyi5xLKagKUEyno5l3FozKcrOaaeLDIZtApGyYRxrA9yJ7MI5yCCU7UuOnyOm\\\\\\\/ssdDqntiakUd+Z\\\\\\\/sDmEq63bxfQGPAPs7vsb7YwLbZ2dapz6O3bD0TM8EwP0hGp\\\\\\\/JuTlF+\\\\\\\/IGvuhUGx0Qi4B9ch\\\\\\\/vlUgIKKgsk0SMu2ay5YMwBx+4WvfYIxuxNuZ2Sg5BNHWcEXe6MNFYUKT72t8pcVk2L\\\\\\\/MvxfzfsibwEbXM37tufdgqcQYJ03Dd5HaNF9eSaVIhmIUpLLuPmJzJWnXWXidk0F71\\\\\\\/yrJkuBYGVFfnVeO0rUxE2fDTgj5XtX5cTlca6aQ23R5u+mwTUZOBnJJJQtpmIZX\\\\\\\/UFol6g3BTZOLl0NnISJDFj+xeiDSXfTOevebbHZGKG6iwbgzSxI9zaO+jMnvGDlAbOSjc8VNkfL+nqmjELYaQ9sCvnWrt9\\\\\\\/slxvZ1JNu+7gXJT27HvGKTnSQ3\\\\\\\/4C7SQHVklFwmBMGZr\\\\\\\/JC7BDeSLY0UgTgHdqjfKi08AblD3mDucCz3cGVVtP1s5lJtUH7BxFo7Eyi8Vx72splha7RGhVi8h1InQrD0hIZyDkUmJBOEayKwzjIjI1NUYxhbei4r0ppzl170Ie26Z9UF3Op89eHmNByEDR3tV+BQypTH9uPTw15\\\\\\\/Ibz4EgiUheODJG4+4mY61ViKp6iYEpotmT\\\\\\\/uVy6obac\\\\\\\/k1p0Z4jXnF3TssVZgBb0XX5KbEkyh7BRLxHhrgLqGQVVHoseOEwivFsUw2SBC3Z0wxJkCRAbCrVZtXUm8r3XZOtp9SB7xUGAZaSxxZzVgLAxVeAt+rwv1vMqbtLWdW39faD4gywuAkHEdA8uPJVOJDqgw3KRaMYhwzVnbxaVCDUIF6bjLnCl+4wVESfEREK5bhZA0jzgpYV5f\\\\\\\/OfFTEwrRQnYN0LdYlVWBd2K0GUCxYQl9MDxFrFm0A+abDvW40dEoAemyJLreYrU04wCsdHDtzjxApawi1\\\\\\\/AeGF7DUiHAe0ifcNoBtHGF\\\\\\\/mStyLj6EVhRYgVsAcNVtVLkPpNBILK0hpxBgyvO+F6ygWY+SInGVjXXUtkhWvX+6ac\\\\\\\/pLPn1biFIOGaub4xXOqCiDsQXfo2rTcO3Qy6eQV0mBEhZr60I1QMSBmCreNK\\\\\\\/qYjvSOgnWdijDoH7JCa8vxq8vtv22pDr556K3ReULPgPoiWcBCi7LOlba5RQ1CHtHA\\\\\\\/bdy6p8CcsVqf9RS64x5odD3a0jYDqBWvF+v6a0V8077KHW1gfcgzY7sqUb\\\\\\\/wcqqi+qhn4UzetvrIp0gHzHutqWYAl6oIJXqWOgKCwlelWVFLy16EDKqJsQmw+l6p7Vh020o8K4k5qIcwjmEW91mXW4j7u4+BT3nfvTcqdlKPXZQQr6pDQKDVrvSC05By19Aw1ZQPVUSZuSWUK1AiT6e9zAiF4TFe+OvZWtBhXHnz4ZJdTY41VchNB9U5kbrkz7dufW9uIRRMSUsgIJO1KN3UY0+q3HZP8eZvK80TDJtJvdkAVVUC7H9V15L3zw05vxfHOpL4MSsxrBCo6BUcSbEfye6VazTif8M07dE7F+jAA4uTPHp5qt8rI\\\\\\\/6EYtgyTb\\\\\\\/V9ef91i517994\\\\\\\/q\\\\\\\/IEi\\\\\\\/3UDBs6m+8WCw\\\\\\\/XNwnESLVFU2Y8iHpXocwdN7qpKY6gdbHcNqV7F\\\\\\\/M\\\\\\\/TlR\\\\\\\/3s57UJB1akoKRx0GD6PdFnYeFxK17BQO3pm0YhD0c1Svu0rbffL9WG\\\\\\\/rtd\\\\\\\/bQol\\\\\\\/w\\\\\\\/mUFNHO4j4decBeljYYboJ4+o1rrjWoGMQZT2NpXS0geaPOFTIZH26FWqiBXgBGHClzB6iXONrrrPU6RRczrUi7f8pveOY0SF7rcB0Ub1G8yCKQ1QagSgB9GNa7TjULy8D322ZbhzAAOvTrjRcO+Xgd0\\\\\\\/9WHUEXrnLpKCRf7e6BUx3Yu1PPM32APQGWw+b7o1uKKb82DfkHYynpdQbGrdI\\\\\\\/orZWhnQQERojFNwri8mYEWvgOGAB7f4rjOwLb1hzm91QqWsySHYEhhWWgcMrCRDVlGTTSP2gWzmjvFagKdUEN+LNjgQyviIsB6Hb54M2wv3ybpU4lKWgZHCIRNoo77JCdyhGGxWfWBmB0bq\\\\\\\/dBohXmLfeUrWG633nqvMkO9t3iQsH20lxoRdUoHuNWJGndkf7h19D2CMkE+s5s7IDD0zySYdNqhY+V+ZuAn\\\\\\\/dLW9xJ+2s6JV5\\\\\\\/Hkjt0bTuCEKhCVzBgfa+C7qlUI\\\\\\\/b166cn8lMlltcc8FJW8aw7eqqm04LVAIDS+PAAhXewgjm5XXBHcA62TTftB5YfFYHXYCpm+rKB9d0hKW7\\\\\\\/BFCfpAWykDhdpg4zQHRnqDd73TWwyobI7Qf4yFbcGm3yFIcBAMEf4VYzCn3af25rqAO+IlccjmT3vDRzh\\\\\\\/+jMAjydwwk6GyUwuXaDiVOppYlmr6QZ12iafWF1mlJ6ONf2SYm\\\\\\\/NqMYxBEo1\\\\\\\/AB3hr85okiJ7fTPSXeQxOgz62frZsvi22CtXbRyqcjdv1M\\\\\\\/yiABUZ932RtzBvnkWsBVe\\\\\\\/wYZCWOIaGgXviATw2QAZeBK+t4NKeONTp2xWPhWyLoKmFjDA1EytDO1zxQJDAF8hN66tnhm6Dl4\\\\\\\/Yuw8\\\\\\\/TVZI6y9tV1jj\\\\\\\/yyrvmximOB4XWjKiu7yLrfRriXGgXC1xePwyNX\\\\\\\/zpHf9sQVrBWtvwX4DgeAT8Mx51Hkw8tP41\\\\\\\/+c6TW\\\\\\\/VnpsvVSbxX9jKnzcONrg7iMn\\\\\\\/gOnbOtralksEBYrv6mgPRP6V0cNOvZvMS7vvIpHqlNjZRdWMYSg0Kxpai0XK3+1Hyls8uISKJAO4fBwTmnBHccM0cs9MeB6u+Pa0Y\\\\\\\/zucFqaw\\\\\\\/u8vWJdMOH47Rzt\\\\\\\/C8mnzMkq\\\\\\\/Xpqw3uWZbd5BTd2BWBxPCjxGuoddkBiRofFupaz3HhOvIXHbDavLfp11HZCDf\\\\\\\/oXXUXZQA6I22cL585tthr\\\\\\\/Hh4bs3KE8zsy7LcPhvoyNBEo83e11grsi4ebxLJig+40EWmpyJ2Nj4YDwUxNoLsO2go16A\\\\\\\/HpFlHgHS+t70reK3cCk5QgTgqhsKh82aQqsA\\\\\\\/vK2Ib31WsjxUTlYei4IQKSHTFf4WdHWS94IF\\\\\\\/SGo3wT1X7R+njlDdBA5VN\\\\\\\/5g\\\\\\\/L1NzbfpSDGEHRDqayA1PuvZMDBkPeax4GFMgD6GOuRAtpVwqiJ5V+1RD+czL3A3wT5d1hzgt8i+AQdMHTNLXukCalgrzuE2Acf8m1fJFKVFmbSSb6T5edy1v9yee\\\\\\\/a4SQz15mZgDTgshnt1A\\\\\\\/WPMiZslyDyakauuZCjWSTVSlRnyqnX8QfYmVSuntLAzfprCqoxBxilk8tAETjnxl\\\\\\\/qvjYmqYWiquIWs+Gh1pQOVropqDWAqEK5hE7GJqV49TtVbxPb14ro8e3n2CTvbb90CLR7mpZqyCTdebKFxxUMavq81auSEYA7r14vXPHnk0mmJOfes6Yhfaqb9KX9\\\\\\\/1A\\\\\\\/6pXNMQQ6zt+\\\\\\\/x4BaV5u25C0iXP7KHwppdY\\\\\\\/tkcKLC0LBfKC3iHJsuvzTKwj06OGCY4f8drvG3YSmmL8Hni4AK2WdW1lwVi25cGydVD\\\\\\\/PQ4no61AFKvrtrJ76aTdgewwmG76jvF+VrPU8aHQPzRNTGWq+qxd3CpE4DE+O47IsGZsHpHx4l0\\\\\\\/JrYw8B6c0snqkdUweFRNbEs6d2XsT5SDxeT38DPgId8NS4fuauAL21VsMoYkn4WcsXtqSXzOiRhmA1Zo4AB5QWOMfEmj+INhc+TvE5fUnbn+gjYVvkbmmTm4Azko3Dpzh84ZMh0WCtSvp\\\\\\\/k0zK03CpgG5dsJjeGQY9ucJSxSJhG2\\\\\\\/esnv6sDX\\\\\\\/X6MNj+oPvGys+fHvRiudlEaC\\\\\\\/3Ll00WC2mZ39Lt27ZdVo16g06WLq9z0Qewb+cDKRYJFQQDW0lJgHbLazG8m1OlMqeZKdWbf61rgBeTMsvu8t7g6PVbGX\\\\\\\/yREyMIrhCHX6mZT1eHeFmnsC+tTLbfQ2SF2xYh4p8GSW8s6+4\\\\\\\/bYCqi\\\\\\\/aNu1VrZlXfcKg\\\\\\\/UPg6qBiUjousJD5TDmdD8Py8aFDEhcT2YdeaeLRreywYH7rr1pzgmNa\\\\\\\/FfBDTL9lda8VLYBmOUNb\\\\\\\/Fzr+XUKWxn7Iq84S8F5a6+Lw3LcScNYvfzLucpOs2OLOJpaj6oBkjZ6QGhVD7eVPhPrA7iKfjQjer8wtJAEHCrsp0RnCjsgEbpGxaHSWfjXOWYZBeC4bgSO2+FvtEJ0xMpXn6JCLgquoYCnVyHTwMlUb6nA2lTixDt\\\\\\\/lifFjOJ9+pfMjxJIgvMkyFysJkIc9O3P7K22oSI4OMcQrTYhtYEXjKfrQMZi4nXUrdr+ABS9Ma6HSJHVH4EmatAuG8NXjBlylwDc+FvDCC9VgCVhBZDnyK6v7BDVgWJjwkkqJYm5qMC4XP8lkmNbMYY7cCrhTKk01uKMILJNJu8RTSbLjYijjHXRJKM8RxRWkuuftiCXSlKrxjGYrVAEGkgSm4OtLTEv9BXQJ90LGY2VOqa75ME5982vF4cPgIk2ETXSLxzApOJMkzJY7WAf9VSOdtOCr0Jnw8t6919I8Ya+A4dD85gAK3XPpq4PWI3sj\\\\\\\/v9zAGd6yW2ORRvUW1qgnG2ZTpZ1AsVVQZYQ5X7\\\\\\\/icElkDyEIFyROExlzhV9TvwnQj4t8n9ixiLOMKbc3NfyzSoXy1B6Tc0hCX81iUWEAWBzZVK8cYOvln5I0HxqC1yacGb7bDKO9hyfy8PHP8jaBOcQptGoNsPejr\\\\\\\/V9WYwcfOMzznzP7t1KtCBdMlpL9yumHxGQMdYsrdtig3P2vHChdojIqNIAHr5llNN\\\\\\\/VlPht97lu6\\\\\\\/GgsLd\\\\\\\/83FNBYzRuk4paKNS\\\\\\\/tzscB\\\\\\\/J4B4z2pcscOcGItkwboWsHlcBaClPxUAZt5IGgubMwbIrT17g0G7\\\\\\\/ro1FMzNrCgSVGf8b\\\\\\\/GXS8AUWzf+mrFr8yJXeTymGl4zQS33R7OaLF2l0\\\\\\\/5NvhD+J\\\\\\\/D6d2BdJPBx+Ls2N+5abrJGk9FJyNmcp3Z0VEJ0kFPT44+2iY+GIqznBEMjNJNPvhOhDVGjH24uh5Adp7yHfcc9eT7BTsjmxYtzcSmx7u\\\\\\\/Hj+vcFtUQlWlAShUKAadotyvyQi+8XT4MntiodMT0cERtKP3OONlYQICxD1pMK+rfrTpDiWBF5ZMmIvlozalPlKN\\\\\\\/XhjwpB+KMMlBJDG5Solne6a6k6VritxpKeKG7CUGa8VH5VCvaL9SZdRgaMrEqvAyZ5DOB6I0W4TL2no1mdZNq+0G3wLOzRxrd3sDSm5Uz8Glpm\\\\\\\/txjjb+qxfcpesND11WVWKfO\\\\\\\/JFMCXMe+MDHIMQl36Kq9sG03QxHRMGrzQHZRNp4O3hhzRkJ6+Iiu9ohAcFWK0zrYfJExuMbV253LVW5EKrXpEP\\\\\\\/2P02TmvSflMrh3bvaMNs3VlXFUq5ctxSsg7JXKEnGX98VRVTsbGo6zKwLp9k8qOcN9v+5rsO8BeIHKPzLoyQ\\\\\\\/1DRcaAkxg7DGBY\\\\\\\/QURfjf9VToPSUV8YUf2tR4oUOefNqSY8ot3P5\\\\\\\/fmeqqrmu0\\\\\\\/7LZ3WhGXKon8pDl4j5j\\\\\\\/d0O74VdLeZCZdxMFacNSEUNncV2F1AV3o4Y4idcUgc25bunI3Nj0xaKX68Q7D0\\\\\\\/uXYL+ZuZSz7cgaafQ3QfBcpZohMK+vw8sBtE+F\\\\\\\/Q2P7PsxCzuEHxEzxf2rASFvZzmSYO4yh1c0fSzOwcv31OORfoX4JmfmCPcc+OKEwIpXVvwN6BaI9Ch5SLOLN9N3kslPQejQtTrl8CfSmeCg4BrLbYoOhcTuBIdrbyZBlZs+LMazgvP\\\\\\\/ITtlPNRq4KmFoun8qYNgvNiBhCfeedu2QzYhdWlO0SvgOqqtjDVtwlsHCupT+TJi5pjrrN3eTR\\\\\\\/AOmdKpcFDI1RdRofoqtGkgdQT1spXgJ+O3uH9lEE8kEjaWDndhhf7LLO76auEeXjzMIbF8i\\\\\\\/P5tAz9ukXXA9rLwaNMilXNjaWlCAwJtxEHFzLwvcxjgJEK+rlZofS9YoNfLEyub50NUDcEjMJy6l3c9UBgvj2nd\\\\\\\/BKIpqyDfmijQpSW2k8IkE7YFaELL4f5SlzBdiECvlKHVz39ad5ib8JzbMWqYzG2Ckqm95Jur2MgResCD+dvyempnNWNIM6JaAHYD1dcv5kpVq8KHFFiXVIk5Y1BC8xXLUbwnTQWTkA+MzSMqrurAwjFp5G6GtDBIqXk7sIN4a9kxmdyX93DXs7X4JGLGbJtmDtTHvwjzt6pHcaA2pStrLP8kQcD3lsZ90Wozv2gBM8PER+sAUPjCsFcWuxPapmZdotlVMnzZZtGI5wmgnwFO1\\\\\\\/SvzHvCxZfrNg3A98faJGPfrm5WXnbprcN2sbddSA1Q73QPtukZtYDSfgY+GkQeIRc2xmDVgmcEYsHsa7TbjRV9UmJEh3azMwdFD2XVGQFXy9vso\\\\\\\/GcoCcQiXnfDMDCLQbleHjo3HDkhnKcoSkfGfK9vLngcTXNf38dnxTucuSoOk37dJk6eQ1eYNP0uVXjaJFyxnznK7mA51cF5kr1gD2\\\\\\\/7XdemTDJQg\\\\\\\/dXm7bJvRT\\\\\\\/I9YBTqsU8Qaq20dIxLZZ1OPB4pqYocWRaRqW2iqcFUnRrZNypZUGl2WyC8i2Ij60Nd3LgZxxr7RzUCsB\\\\\\\/+HIzyMMuJJQ5ywDXArrEFdVPokO6a1XwPOzgxLjrKs8fGh2FcgXsG4wOhEWLuW66txyLj7CTkyU7+aLB542o\\\\\\\/XOAiTUKZhsXTfotZsFs6dYEbbDKulttiC7o3aDXJfpGQfpIujfxuoM+jUUH2mXf4XnbOSaZho3WOVQ3dCpuv8QRdMY51ytuKuG\\\\\\\/\\\\\\\/bTbRmqXr\\\\\\\/RK3TKtvmYW0Y7aJp8yDgykcKVhyNd6ARfaeiEzh5Y3ZkwQD45jITVMwA7TV+mGCh7pKfDN93krGpXahLXlr30IyTA4NDu4UIRLLL+R\\\\\\\/9rfVDVpr0XpxsuwlXAXjAE6P2fub3qq2w\\\\\\\/b16IH4DRmxRXvCCNLRdXfSiUN4wT2lgEeWopYV8Wuy9JT55FgWTnCKtssp\\\\\\\/tqyT0aKhypHrdCJH5dYXdv82Imd5dt+tmTebNiC4XgBnSV\\\\\\\/Omb0XLoPf8zDaYgLXxNHB7HSiUzlrIkPNK6QzlULrH43679WqYTh1tjKyGDORRO4mote5PHc2dL7cIKATYqJ4W1jSVpmqZkHOEE4AeFsrCTKWQIIUZRsrdzB9lCO06kUhL3J9VvJTVP7uty8oJxNXClsLx8hWAZ3eafQ1RwZpqXnvfRP6unE9+Mmxg8y9nOcJGzaNx4vmSLuU+tJ5HcN8ro9bM7l8tdhKen55gG\\\\\\\/VQabNT2fcjx1XydXWyisgMTd73zUgQIug7J94+Wjgma5ikxnV2UUzSQE28TmgQ6U2HHyLcGT0Exxw9WkZWUqlCzZOIkD4ALPSSv3E28nBfauyH5lcBXJsynhdNqGKYlJiNjXuieYmoiZMCg20lGiTU4wvvxvIEitJ1WcR0I5MI72gf1CFQNgzMJzWspVodK4aEsHXfJnI1LhaF484oVX\\\\\\\/bjSBIdBch0DomJ5j77j8H+Eb\\\\\\\/JUMwOmx+UIho1i0xQWmlMJosV80t1QKDnzij5LUYUxpNMy0WlWh\\\\\\\/QTamsXOVh3gmADWYwcmpW9wwkKoqyM8UFrQfalx2KCu63LnsfimW3\\\\\\\/I8\\\\\\\/OJEynXsRwgd2ntYEpBc57ixRAV1R6gGwfUkWrej\\\\\\\/0rElyWZD0Y5628qsePXqv7oQKYIzdozXzEMn+gTnMU9CJ9DLoay3v1RuawoEagf3FbXqXJq5tJHQYep9437bI\\\\\\\/RJr+1GjRkdmxR\\\\\\\/1+tMI3Y5ddFwNsnXmsCmAi7xr6P8z\\\\\\\/BbIRVmmxvWfifo3lTD0NqBqamIkvEiQ2fBJ79x6s\\\\\\\/drDymxnFYIU4X0KCMburTiUeCbJdpQQALDYFgh5S1NuQzwJdoQPMM+z503dTU1h89AK6Io+lyah9J8ppKtTzzSaTJ1Lqt1sTWIpw7UeseIr7K5DutqNJ693BglbBHia7fFP0uHR5x98Dvb5s6hlSFaLVHLueG0XgyCeQ3mGibXcI4sj0+BGtVWBg8JG5YBKLaphOHOkNdkTF6C7vcsgKOm5rdJ8vyO5aQKWafbieNn\\\\\\\/PRRikVDksnEOvAxUATPzHUiPKI3MfB3vxyMU7whM2hkgjsCtEzzJQv6MCenm01J049CQZvvGgF20lRr3zijoWlFd7Oda8euV2V5J5cZUVCuxTpOSSq1AmLSPQ9rL+ZhxtdCoAL7eFH3ZOR6OK84+CThdEfmww8FwGlHzi1W7IE2kCsTR4C6bTLfmiM43XxBVeP6OgQwj+KDlwHmgESFDNMEIjIaq39Sil74jEZvoFdhnurAzDFmn\\\\\\\/PCVZIbiG80k8EhBIydDuThdDUyinXE8VacL50IFGHXzTXf5qArVqpG3pMasD3PqrkugCbGDrcrO7vJFAw6bydsvn0zPHEUy9GbS1XYQyjLK4JP1eIUBpfq3XL\\\\\\\/lr99A0iST8OIv11HnoL17MqHxGSOpF6cyuOSCQTTx6UKQQLmewGZfKd15geEFDIq6uaS79SSixtBtfuJivIzgWbEAryHx+2tXqm1v\\\\\\\/\\\\\\\/dsUYi9iWrz4TVyLCcf265ILLZqa6J5orQYq2K+CBLiAW+2ahgcZkXZJQAh9G6OzApRydLO5cZOaYwfMfpTBDihkg7Mf1FohjC+trWJBJjgw4hXtxzHWGDY7WwzmU1uAvk5LFzxiSHLO4\\\\\\\/grAdH+szigjb7XxNOt9UA0reXSbZHmDIohXv5YjgmMUavNWwwRHcEBJAXsbRdewdTONG3CdZ5lZ\\\\\\\/9LrmaUfWC3AnwHbqCRVCj7SxWD3ziOyT\\\\\\\/wHeW\\\\\\\/6MEm25LzlmWa+A99XjRTgZqfQQcM4uVPcFVL267NyyGRoaagxUEHE+ePfN8vF3EkkFt\\\\\\\/QLMUdN\\\\\\\/V5S9jTFafPpDfzGkKMQ69V2niBcOmkxy8bacq0NabSHyfqNallAB3Pb5CBavgMbr9Zk+JveKEC5i6QYuTac4Lsx7ShZhzdONphpdx4AgbIQSSq8aEbRTHj7n2X++Y7fiPmqG6yaYUbSmAZGqxzSXZZvSPFNqHs7xPOMUKVU6UhKKD0+0UiQFTBSILUfVGQbVlj2cZPvh2s2m576N7fbxmKfyYaecEkDj2fanDcu4O52n6FYNU4ZxDg67DWlHtmAxzxjY29cKuk1R3QVZPw4DgEEJJj7rEPlIWIQcxgzOrBviYBZV4w5wK1Hb2oY0ZsyH+9wi\\\\\\\/dq73nFW2gXwdyVg4Uo787Fa+n4ypxtsj9VrS4SNuWr4biAxnsOJxdR7b93erRAQhH\\\\\\\/QmaTirszcAzJ\\\\\\\/TyVfXG4zPIh7hyoc13Iwa7TJiaXoqs5utxLRLKeHh5rdG5y5Q1jm6LHDp+dFkono1g7rj2u9SJyEo9Z2t+ip7EFoALGgJj1SfdC6iErZqGM0Vv\\\\\\\/VBJPIDmQ3KXdZLK66RY4vxQFKiiWluKSJfIf93cEEzLoCgtK2RepZ6ueNUkLbrEm9mkPL82nBrLwWE2aeuJsNn0tD\\\\\\\/jgrDovjBdmIYeTBMwSOwAybEFM94N1XU2dNGN0GUdaeV6vhCe21bO1Jojz30MHEDp506RvKtKt5vW5Ynk6JLCIw6pbOhrNHFk+RlOMRfaO2HCAjsJX\\\\\\\/xFHdhnItwYNkR0bV+8Q23avas5ihySs5HV0\\\\\\\/pcxvL4u9ZeaubiIrV0B7Tz\\\\\\\/VMLc9sIq1w8xKubf8xUR\\\\\\\/+A+TCVDInuIbx2dBrcjbh4Zu6s\\\\\\\/vHpk5pevSgr8ISYO0Zss3Ue2xGoOHb4cg5HdAtucEBLlKCd9UpbJGS1tdqV3KDOyLt7sNdtveEtpf5LGUODyeLvex\\\\\\\/041jzELVHP6R7HAqsHKfQ9SHUZRILcJ2agCjQJJac1XIduRCYtCXormdRi6F+i6tnJTJWvBzzsPdkS8gjfbJC0GwJnT1mRHwGrFt9e1UcObx6to4I9XjhKeDU33sem9mCc4HcxJXNRA1X2HHBF\\\\\\\/bHTIWvi7n9cxPZx1jEIsC0VdonhB4Y8cjIYBbpSbs4Qsv\\\\\\\/tkrVu+34wHUZDC6JAAQaGT48VA\\\\\\\/ZISlYHRP7\\\\\\\/PhHofnmlMfKuHcfakhHj\\\\\\\/yhGyez1RMaGPYX4a0hTUREWn3ZSlbz6gdVFfiqJ+WqJkY7G81v+aerBXixt2RmRA6TbOKnVNBeF\\\\\\\/pOoXcFny1hxnluh7R0QxshY3URdFFYr\\\\\\\/RILwp\\\\\\\/Brij6TkO2Xvmq\\\\\\\/yCEF+OurGHALl5z1FRkCKoyaLlxHwulvhcdApiQa+qxJYspf+G8tb63s7DbQgYFcJXiOgH\\\\\\\/TqCQRnQb5cAt8Wc11x1\\\\\\\/gNQozMk+f6aYx\\\\\\\/ltZqUfnU4uEUAbAFr1H4sHjAmxZ\\\\\\\/5VxX2DhJLNcZC1yj7AR5HqOvLjqugsDdypu4YTdkSbsN40m\\\\\\\/tWlJBdHQo6EChaPyrfHYbK6FRKCVLP4vvC4p2OHGJScxrZnf7C8J5WF+C+r778Wjzm254Sp\\\\\\\/SPmZes411as0i\\\\\\\/wYkEaOP2LPDUN2J30xtFMgLtXys1\\\\\\\/FpsLu1+td1tEXWiPxrCZmJwLR39C2oYAdfconOO0hnq7wIXvp+yHjl4jAS3X7S6GDYzdKBTd2xg8nA498HWG+66KHtS7PO3Gs3V0+J\\\\\\\/\\\\\\\/iWqsQLN9Z2Q2B4QScGKIy1rsnHnx+2WGF79m1q726lFQ+BcrcwqrDTAmZSOq4sjdvQF\\\\\\\/V5SOE+AGYOjS26L97BgHpzaPyLGQBHbV7Sp7YRfDbCXKT7k1R\\\\\\\/nX0IoUCegBC8\\\\\\\/AaXoki16hnoOb2Dd1rzPSKCPFEtujgi6KVYwgu8snnh1aTTa34P0SNPAtovT9K+BkB7UK011oU7BQ5mOtXCwF2QoCglctqJ7vbXt0sfsIkgonUL9dYpsLgHAubOR22JCW5byXU\\\\\\\/J4j9\\\\\\\/B21k+VxIislykKAJQJVyVMbJlaLdp\\\\\\\/btPX4prIj5h5JjvrSsjCym83yfUby4N5BpdfRxB74KAdebRNZrYfQnw2s4nyqkHVK+FFE0n\\\\\\\/qOC0xS5HfB\\\\\\\/PPfLn5v+6gJNQE3+AzgbHfaXcQJFzN15f9a32m9kPGyvhZ1ORZLRWIzqJlouTYY5IZ2HlGHLWtPeheYuYOrufZq0LZ5lfNvcClpw3d2i5G67uJvetvE\\\\\\\/11uZ22TPPztXkNtS1dXDBNgAzgBUaCWxMlKwOGdE3dKIwRvOeiraUgqmX0jRMJNIGZhOd8yjqgSeocInM2kvT+yjGeAF9BgQVxfR9wMGdGtm1\\\\\\\/ZF7GuOhYQ8MYXQObLI4L\\\\\\\/O+0aLbPlgyYtCRIoJACWfM79rwkZRPnnfKjcW2Uo6Z63dnI4bLnxvTCipWH89FnztxLC7Mjohzw\\\\\\\/IJLMugGtYqjnbzHY3jqQwDxzTuqSEpxJJUCzqhXfVuzgCxKu5KacYHwSFWwhH12dNvKGiHNjd2oQ1pBxc+JbOQPTIqyCad8bngMMkFgRxXEPoGeZ2B3xnMvF3ALdyccly3tEUNh\\\\\\\/vEtJ9DkxFdCOhvWiq7tpFRdkUmXyQgIJRfGqX1Y6RM3uyqpZkvHUwqDigzwkVc\\\\\\\/5e+o+L0LPw\\\\\\\/6j8ifPTNY+XLQMoMDFx0K+AtyxLwsU4qEaHdjHR0lR5Bhc4aRrOCEkZzElPTNLdtpV3IbIhOY1xvD1KW15LU0fW3SWVqkcxWJgXu9DkGou6o72q2LMPLFLqnH+4rPMFeVUM9U5H\\\\\\\/OyxrBHd523T86CnwKAhMQkWj6lXYvgnb8FNRJR7e8S1xYaLSJvFxdIro4pIWJnDcYgLu\\\\\\\/iJ5nQvd5MydnjzLT12nZvjixDCEaKYeIvuJ9vEz6\\\\\\\/Z974yv7\\\\\\\/oGszHuff6NsYlToCaIKuw57KCwUMLFHLK7vrXBiWgddO2mqNLS9b7ueBTA3ORo6MwNzsWDDUo4w1tIuGA3+Vf1Iv35UZOJVg4PJhNUHAq5Bq6FDtnAOqztu8O3D6ws\\\\\\\/AhpMZlrxcbRPUwCRxbUPatyqO72VOltj0FG8xi30AqeZ6+FClx0QJA5IH0RYFX\\\\\\\/I0vpOSmWmF7VCYYizcqyIH\\\\\\\/CK8fbNQ3BdeWg4WE4iQ2+vaubLkwXA8hUjFb7hC6Kc8gRDeyJx1fvqQq3ORxfsUeEznnicxm6qk4\\\\\\\/LJ0HLBGgUmtjWpy\\\\\\\/wA6gmRrNsi4WteWpX\\\\\\\/W+wyogqVUtiZJR0A96X0\\\\\\\/DeqQtPTfUflaarq0hdnjwd7XmUPIcY+XK52\\\\\\\/V02iWUqbw5Q3xHHH7dbAeUMFyKhgjfYs49BMs5V0+zuyFIRTbOg9pjP1D0kGsCCOKJmbbGIeQYyAocWXdv+S5QxKkfgJRpAn6rWOwfrFLnd4okaY7p+ZCm9VQuidQrQPjUcuYULu\\\\\\\/PdnDhpk2BG\\\\\\\/SdsfCbFjcfmdGCjDJz6KpQgn+dQVmATSKKrtmm4mV1hu0lTF8D7wOX70NKkmZvFItZ46UPnNFZ65RiMtSeNK1TjCHMFMwTMnm+jmWCvzaix+yIGPnmrsK9VKkAYikAjIsPl5TSWEHYI942aSDfeSuoFk6OtEp+y1vkGK2cDanMGIv8Te+wYPH2Ex\\\\\\\/5PCcbt31nEqlxE9MBy9fPMqMCWVVu0kOSDSu\\\\\\\/9SJ1Dd5BmyaeqkG7TPQPEOOIhy7tVLjsDPi7YlLMscVLSAIfiQc8+pcW0kYMLYLWqnQW6wJhQ5B2wuUItQGdKRu8HlgPjvOmVQHLR8pgUpn266eMup+6HAURQOHF\\\\\\\/ILO212Ud+42i6ZyNVX5Vn51n5VSwRIY6RUKPQXM4ZHpQA7U5ffGltmZBwMUUKxT5LcksoPyvRXyTGoGXksKqiECTpslt4ogElije2LZWQqLWM0c6QK+2X9i4pk6u9KsTfe7bYUhqplnw6ZNLofOe1stepv+4cIvuUuv4zhHM3b9Pio9LIle\\\\\\\/UYQPDAATywJCbdSk8btgVgLPUAGpzvQ86PgoGVduFKfWxTTxrTpFW4X5MUv2XeYV4EXMK1tmzGD1aEgWY\\\\\\\/ABxvfqmMapBKx1+FJXRmCJPu0JXunVoVZferF9mMsdh6UHylzI4niTOaVuA44\\\\\\\/X0nFYAXEzXwGvpzzOQmQefDrCbKG+XvcS7nGibHdfobwPjickOvqWuGroFYGOZSm8U1Tuk82+2G4nR7qEH+baWnDmrG4cjNq+DRiwv0u2CMmNAJAurJ6S9I62XmWVGTtT0IluoSQsTjQKBQZPefQ4YN9UR96cDYONpqWR8lqv70JyXqRnU\\\\\\\/aJ1fyNAkYTjHKF\\\\\\\/rin5jf+2f3LZjDtmNkRU6CLK68135FOcwogc4ij1VTOUQTiItR1doiC7\\\\\\\/DI2+g2hh63pcszXn1gqPZspEsdKh5OThY90mbw4RD1Uz5jsFwEmsnLOjN\\\\\\\/CGjGH3BRGSmhHnjX8G+9JePC34gYb56oHn0NkRp3tgmQIUI9C09iqhfdjL6OeflcFKedEMCPr6Jc2GIqF0l5eM5tHCg1B1+8\\\\\\\/8YOAq8xDmDZ+Zv2G\\\\\\\/f6VROGSoS5Cf53syH2RFU7bO69dSoanSZVDfXwszhwf7Dd26nGX8LZh25h9W58J1pVzib+VJmgYvmZ6dj3Oh1kUtnna1yDQxAUiPXZSXhnZYZ316EDHpoC+4f1phItkSUUXeEMzrr5xz2L0goc5L0CYmbpKVEI1+FKrWZ6ZTQ9M+hsOXNpwyhNV0y7kpTAupbJXZ7ISgEOrzDu1gFNFsSUA7rO2u9GCsTrKgAsgvrWQLrR0klAqdko5qddyZhVwq6uWq5DBDCbcO+S6LqpqVwJH7hVVNBrft6yDKJshCrYCPecCYnFNcsEZQobfRvlizQbL3C5qjmdyISDAV0eDJTaxYIwDMgPe3+hSL1FH+9cLiI8yz5jlZbtLTA1mlhHOTYJ7Ug3mabel8stcIlOtv7tDZ\\\\\\\/NvrTgJliwrWBHcfQvwE7y9+bjXOOHeAMWw+C3zXfIzt5cdd69pNFQQgLPo37qJ1sLnciZWyNWEwmgCFWQ8c7Rm7S2Vhjybv+tE4c8JknY+Ma7EnCR\\\\\\\/baScu7RnyHks1OJOgLK2xfYRno71XEsc\\\\\\\/vnxn0w71acGnQ3DqnYK4iBPingVi+JG+\\\\\\\/EnT04fcSMRPZ\\\\\\\/g+D9YOCL6vsBd9hAJsuGpnhLMfF+cTHiDexo06nlIS15UW5ygN\\\\\\\/8bpsoKJO3tTvxCPx08o\\\\\\\/jSWb5xMulDKtsjbcO3bdXAOW8m5VR9mQPC\\\\\\\/oiyBUp\\\\\\\/Q+cDw5rsO08TfSIkuhtxOr9dbGn4bahrvVl8KEAVxoJi6sbE4r1xa6qU804XLitj6C2GFDEQ2pVIfTKEGx6pXRQaFad2CLMiYpy3vFiVdh3rjRZWkm4P9P6XnvClo6UV6MnTWhRAcSCGaBni8oj8JwgD8weQ6femthTmU8DTCvOn7O+T1EVazMVGN30h7szAMXmDBlzifR2TyqJnw2ttlKeO7d77EFulI1UA63wGiTufoomOk\\\\\\\/eRuqzec\\\\\\\/dOZHfb1h4DwTPzqT8TBWfKWLV5B5Tjq2AyuK5DTY7GaqkURP1H7xILkcT4oHSfzdCguetKmOtszzltvBqARDZf\\\\\\\/CWztXAS48+6j0W2GeXICESe0TcDnP3676wDh8QDeqXZ8vaQkUVYN9VS\\\\\\\/dPNx0FG4gBSL6k\\\\\\\/v6qzwsOSYCK6joP5V0hRy2oqtPdvJwK23MZ1Hrowp6dZX4G70HdmwUnCKtaZFdVjfjla5VRGTALX0l7xEH\\\\\\\/BNB7p3notcrZkrPMjxasuw==\\\u0022,\\\u0022iv\\\u0022:\\\u002202hbIWf3STJ77YnvYjiyuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c456017b23db4989a389e7a20d71db8db01a1c51f992b793d31c330ed5d1586e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002276nCXo9aj0GRyCCs045c4O8iBKWQXsJCEQeXszrXSpDKR0JhyIbrfnU4zGroz7dKqc5hwDLkGhcL3wYzhTuUo\\\\\\\/nMj1AGjVit4Zu6YnbCO46vNMYUYzVttjXfk+yjMV7L2\\\\\\\/BRpGZ2PFHb1+87iE11RMKRpdSxk4Un+T5vXQrTqhd\\\\\\\/lAKD4diyrfHtEwC2gAfsZmo9QnCXlXzQUvtZFigJ9Phauunthjvr+W\\\\\\\/1Qn4drYMt0kMqBpxYd+adrHhwFkP+hb\\\\\\\/Idt1MfZzJN59RYha0LE5uRrJSde7T6LDBlbGrNDix4Y8ou90PUakREQotAeRv7xNYqimY3dDJDkgRMsnLkZyDxj9FkiBzoOhS2bB2maOiI\\\\\\\/uui8BKhMeIiFtbodxYbop3uwLxZc\\\\\\\/Z0qWL9b3okqGzMWCCQruJ+sv5P0ULSp3ga2rRwQ0EM+ZfEtw6ESTxBjrIkFpl\\\\\\\/RyCa0bvt+\\\\\\\/fPPoihY5JrTiMXJJcl1\\\\\\\/Y4fjKyZ8iJ1mxbtvbmECLtDts86TUMi6h7RFe8O2WleBxWl0jAbKzVMVdalUBN9fIa4mj9rVEjUPTcIvS7XPvVGZ2NLvC\\\\\\\/MmvayeKvHjl+8Y6FRc267wfMKxAZiaLBGDlkPpSha18xQhd2MK+z6M\\\\\\\/0IJGEW5CM7MLkjy8R75MW8S2y40ApULt9ELebAWDlK1glzxN4UXfwjktT+DT8x2CyKfXHFGexlgTnEDgPIZjEY2XmQrJv4qsJn9GFoFlV8zrXZUPfxWnfNNU8zaiNf1bnGv1kZmAPrx+CtWFtJEKm73v74D\\\\\\\/vMEEsFR6K3I374gVA5Vz\\\\\\\/T+eHBSvSmEtNYjXdQIFxm0EDhbeO\\\\\\\/LJ+dt7RIu7FtbZWhlC+tPVNBfGnkU\\\\\\\/TXzKaV3SSguSAMkrB7C\\\\\\\/NjN2d9WXJKS4aGLrISiZoarWnIjYy8SlQsMOC0+Qoj0HoCZoN+J2wPtPs7SSLd\\\\\\\/9tIrPXH6z+3wQ9uZI4M9mWpS9Lj2e09FO+SHM+OgiCFiPLzDO56Aoa5p4OmJoPu1G1Xf3kUmp1jxncXQWOAY8opMc5tV\\\\\\\/7I7tiSif\\\\\\\/PedVIg4WnHvFhhXV8ZclZMKKvcAmhxX4E1En760AOU6RdHLCee\\\\\\\/nOEMu54bJkkgkkdSmH3\\\\\\\/32Hn4g3Hm+8eVkzWvQHJN0fFy2xU4Qg063wCB3bXDERz+HYvwx8X5Otjlf+BcG1eumtS\\\\\\\/JdU2Ahbo1104PLxLLBXqOdP8+sVDnxQjo4HBB8MXChSKDI\\\\\\\/wfq8yjaMepG41XYOIOurg+pcbPfoOShq5gg9W9OfXR9\\\\\\\/2Rqe6Z3Q5\\\\\\\/NvF3zSNeRuxJRe3pIUk53Ras4ce8WPqu1HAOzCy7p+dzcb52\\\\\\\/MF8yR1nKSwAGw4NkRQmC1XRwPG9HFQ6WB6j8YOdMkVwpwc0jK4HYa02drRSokr3ftO81wgMz9AViAsKtkI42kQhaSSOReogt5JaflRwsV7Z3+Wi0SZo9Qj8lx1LH6x9okvRyEoTpgVQhsdtLrrksOBO6uRfklBQgPiXDhrRGdU\\\\\\\/PZw9lA\\\\\\\/HA2qw\\\\\\\/1nRmMclH3mDnVTzMjPAbBxldZC\\\\\\\/ZmJ09pdQINk0MPCwu3oUKwa4NoXIJ3ZzT7b2o4D30+M37M6+sNFjhb3o+MKfLgCrn607+t4o8swZRAJxIhZy46jsl86HAme70xEDR0Hu4QJwwFU4D\\\\\\\/chEzMYNUJ9SnwY8YvJDpoMdCfy+cbxPqTRBd5gciDakFQ1pmW02d\\\\\\\/SAC3aI6moaJDD6MnW1NkNQ0dbJvT6LjCq8fA7vndclxzv3HfDW1H3yruhm3oExpVo34KPzOc9ExPIU4pxu5QKFXx8A+sgKWPQ8ExMX1qvAIFjNvhVxXmCuf+K3RcesPGMAhlYswdWsoHNh2s\\\\\\\/2sYERlxy0Quf3HDs4ToYMM199KoqEWmi+AQHz+\\\\\\\/vYl8zszlhc8cyx9W\\\\\\\/pQwDBGREKIdBbhM3hDiq4aYPCQNrMmMRmthmgi+iAs5id8VQW39sUgOcgKmCRgYCof2i4Bdvjq263cMQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Iw733BA960+ycdZv8j8jtQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b5bd67e894638d613d1499e5226fa72241a2dbc449bc9a5757f4393d82257bc0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491561 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3745" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fSj6CR8JMnRXFYWDfKTgiDEYqdU0oE-PdA5wl0_7bG0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3745" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e-Kt5NfnGC-Fd-BJ1a4MkouxTL-sgEvVyp2hkobE7JY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3737" title="Highly Continuous Genome Assembly of Eurasian Perch (Perca fluviatilis) Using Linked-Read Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3757" title="vqtl: An R Package for Mean-Variance QTL Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491561" id="node491561" data-pisa="ggg;8/12/3745" data-pisa-master="ggg;g3.118.200552" data-apath="/ggg/8/12/3745.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rishi De-Kayne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philine G. D. Feulner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3745-3755;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491561" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rishi De-Kayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rishi%2BDe-Kayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De-Kayne%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARishi%2BDe-Kayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philine G. D. Feulner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philine%2BG.%2BD.%2BFeulner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feulner%20PG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhiline%2BG.%2BD.%2BFeulner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philine G. D. Feulner</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:philine.feulner@eawag.ch" class="" data-icon-position="" data-hide-link-title="0">philine.feulner@eawag.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3745" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491561" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3745.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491561" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3745.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491561" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3745.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491561" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3745.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491561" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic datasets continue to increase in number due to the ease of production for a wider selection of species including non-model organisms. For many of these species, especially those with large or polyploid genomes, highly contiguous and well-annotated genomes are still rare due to the complexity and cost involved in their assembly. As a result, a common starting point for genomic work in non-model species is the production of a linkage map. Dense linkage maps facilitate the analysis of genomic data in a variety of ways, from broad scale observations regarding genome structure <em>e.g.</em>, chromosome number and type or sex-related structural differences, to fine scale patterns <em>e.g.</em>, recombination rate variation and co-localization of differentiated regions. Here we present both sex-averaged and sex-specific linkage maps for <em>Coregonus sp. “Albock</em>”, a member of the European whitefish lineage (<em>C. lavaretus</em> spp. complex), containing 5395 single nucleotide polymorphism (SNP) loci across 40 linkage groups to facilitate future investigation into the genomic basis of whitefish adaptation and speciation. The map was produced using restriction-site associated digestion (RAD) sequencing data from two wild-caught parents and 156 F1 offspring. We discuss the differences between our sex-averaged and sex-specific maps and identify genome-wide synteny between <em>C. sp. “Albock</em>” and Atlantic Salmon (<em>Salmo salar</em>), which have diverged following the salmonid-specific whole genome duplication. Our analysis confirms that many patterns of synteny observed between Atlantic Salmon and <em>Oncorhynchus</em> and <em>Salvelinus</em> species are also shared by members of the Coregoninae subfamily. We also show that regions known for their species-specific rediploidization history can pose challenges for synteny identification since these regions have diverged independently in each salmonid species following the salmonid-specific whole genome duplication. The European whitefish map provided here will enable future studies to understand the distribution of loci of interest, <em>e.g.</em>, F<sub>ST</sub> outliers, along the whitefish genome as well as assisting with the <em>de novo</em> assembly of a whitefish reference genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/coregonus" class="hw-term hw-article-keyword hw-article-keyword-coregonus" rel="nofollow">Coregonus</a></li><li class="kwd"><a href="/keyword/rad" class="hw-term hw-article-keyword hw-article-keyword-rad" rel="nofollow">RAD</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li><li class="kwd"><a href="/keyword/salmonidae" class="hw-term hw-article-keyword hw-article-keyword-salmonidae" rel="nofollow">Salmonidae</a></li><li class="kwd"><a href="/keyword/sex-specific-linkage-maps" class="hw-term hw-article-keyword hw-article-keyword-sex-specific-linkage-maps" rel="nofollow">sex-specific linkage maps</a></li><li class="kwd"><a href="/keyword/synteny" class="hw-term hw-article-keyword hw-article-keyword-synteny" rel="nofollow">synteny</a></li></ul><p id="p-3">Although advances in sequencing technology continue to increase the yield and lower the cost of genomic data acquisition, the curation of this data into a usable format can still be challenging (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ellegren 2014</a>). Understanding the relative positions of genetic markers is often essential for the detailed analysis of genomic datasets and is carried out in many model organisms by mapping reads to a reference genome (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sarropoulou 2011</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wolf and Ellegren 2017</a>). However, marker ordering in the absence of a reference genome can also be carried out using a linkage map, which provides a measure of recombination distance rather than a physical distance, and as a result their production has become a common early step in the analysis of large genomic datasets (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lander and Green 1987</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lander and Schork 1994</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gross <em>et al.</em> 2008</a>). Linkage maps are produced by observing recombination events which have occurred in parents by sequencing many offspring of that parental cross. Recombination events, which break up parental combinations of alleles, are used to assign markers to, and then order within, linkage groups, elucidating the relative location of thousands of markers along the genome (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sturtevant 1913</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rastas <em>et al.</em> 2013</a>). The resulting maps hold information on the broad genome structure <em>e.g.</em>, number and length of linkage groups (<em>i.e.</em>, chromosomes) and can be used to evaluate synteny with related taxa to investigate genome evolution (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Sarropoulou 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hale <em>et al.</em> 2017</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Leitwein <em>et al.</em> 2017</a>). Linkage maps can be used to associate phenotypes and genotypes through quantitative trait locus (QTL) mapping (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doerge 2002</a>). Linkage maps also hold the information to investigate the colocalization of regions under selection <em>e.g.</em>, F<sub>ST</sub> outliers identified from genome scans and the recombination landscape itself (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>). Empirical evidence has shown recombination to vary between species, populations, sexes and even individuals, highlighting the importance of its investigation in existing and new study organisms (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Smukowski and Noor 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kawakami <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stapley <em>et al.</em> 2017</a>).</p><p id="p-4">Linkage maps have become an essential tool in investigating evolution in non-model systems, providing information about the relative locations of markers along the genome and assisting in the assembly of new <em>de novo</em> genomes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ellegren 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">da Fonseca <em>et al.</em> 2016</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kubota <em>et al.</em> 2017</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sun <em>et al.</em> 2017</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhigunov <em>et al.</em> 2017</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Matz 2018</a>). Many non-model organisms have specific ecological and evolutionary characteristics which make them particularly interesting for asking targeted evolutionary questions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Matz 2018</a>). These features can include high speciation rate, remarkable numbers of species living in sympatry, high phenotypic and genomic diversity within or between populations, and unique ecological characteristics (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Garvin <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ekblom and Galindo 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hornett and Wheat 2012</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Matz 2018</a>). Carrying out studies to understand the genomic basis of these phenomena relies upon the development of new primary genomic resources in these non-model systems (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Matz 2018</a>). Linkage maps are therefore an ideal starting point to study evolution in new systems and open the door for the future production of more complex genomic resources including <em>de novo</em> genomes. Scaffolds produced during <em>de novo</em> genome assembly can be anchored to a linkage map, improving the contiguity and accuracy of the assembly (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fierst 2015</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Feulner <em>et al.</em> 2018</a>).</p><p id="p-5">Salmonids are a particularly interesting family of teleost fishes in terms of their ecology and evolution, having colonized and adapted to a huge range of habitats, reflected in their diverse life history strategies (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Nelson <em>et al.</em> 2006</a>). They also have an interesting evolutionary history, influenced by a whole genome duplication which occurred 80-100 Mya in the shared ancestor of all salmonids (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Macqueen and Johnston 2014</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lien <em>et al.</em> 2016</a>). The family Salmonidae comprises of two main clades, which diverged ∼52 Mya (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Macqueen and Johnston 2014</a>). One clade is made up of the subfamily Salmoninae which includes salmon, trout and char species and the other contains the two subfamilies Thymallinae, containing grayling, and Coregoninae, containing whitefish and ciscos (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Near <em>et al.</em> 2012</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Macqueen and Johnston 2014</a>). Following the salmonid-specific whole genome duplication the genome-wide pattern of rediploidization has varied across the genomes of different members of the Salmonidae family (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>). Many regions underwent cytological rediploidization in the ancestor of all salmonids and are referred to as ‘Ancestral Ohnologue Resolution’ (AORe) regions (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>). However, around a quarter of each salmonid genome rediploidized at a highly delayed rate, such that the major salmonid lineages (subfamilies) had been permanently separated by speciation before rediploidization was completed and those regions are known as ‘Lineage-specific Ohnologue Resolution’ (LORe) regions (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>). As ohnologue divergence depends on rediploidization, LORe regions have diverged into two duplicates independently in the different salmonid subfamilies, and consequently Atlantic Salmon and whitefish, for example, do not share direct orthology (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>).</p><p id="p-6">Whitefish exhibit remarkable phenotypic diversity and high speciation rates, with multiple sympatric species having evolved post-glaciation in the last 15000 years (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lu and Bernatchez 1999</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kottelat and Freyhof 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hudson <em>et al.</em> 2011</a>). Two main whitefish species complexes exist, one in North America and the other in Europe. The North American whitefish complex comprises of <em>C. clupeaformis</em> species including sympatric ‘dwarf’ and ‘normal’ morphs which have arisen since the last glacial maximum (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernatchez and Dodson 1990</a>). The European species complex was previously described under the umbrella term ‘<em>C. lavaretus</em> species complex’, however ongoing work to formally describe the many species which are found across Europe is being undertaken by taxonomists (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Douglas <em>et al.</em> 1999</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Østbye <em>et al.</em> 2005</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kottelat and Freyhof 2007</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hudson <em>et al.</em> 2011</a>). In Europe, whitefish are naturally found as far north as Finland and as far south as the Alps, with a particularly speciose monophyletic clade known as the Alpine whitefish which are distributed throughout Switzerland and its surrounding countries (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Østbye <em>et al.</em> 2005</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Hudson <em>et al.</em> 2011</a>). Over 30 whitefish species have been described based on morphology in Switzerland alone (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Steinmann 1950</a>) and recent studies have identified additional cryptic diversity among sympatric whitefish, using genetic data to identify reproductively isolated species which have very similar morphology (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hudson <em>et al.</em> 2017</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doenz <em>et al.</em> 2018</a>). Some lakes continue to harbor up to six sympatric whitefish species despite the reduction of genetic and phenotypic differences between many species and the extinction of others following lake eutrophication in the 1980s (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vonlanthen <em>et al.</em> 2012</a>). Sympatric whitefish species are each-others closest relatives and thus monophyletic within unconnected Swiss lake systems and occupy a variety of ecological niches and exhibit a range of morphological differences (including body size, gill raker number and spawning season and depth; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Douglas <em>et al.</em> 1999</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Hudson <em>et al.</em> 2011</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Vonlanthen <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hudson <em>et al.</em> 2017</a>). It is the repeated ecological differentiation in sympatry that makes Swiss whitefish a particularly interesting radiation in which to study the genomic basis of adaptation. Although multiple studies have investigated the genetic basis of adaptation in other salmonids, those carried out on the European members of the Coregoninae subfamily are comparatively scarce.</p><p id="p-7">The complex evolutionary history of salmonids, specifically the effect of the salmonid-specific whole genome duplication (Ss4R; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Lien <em>et al.</em> 2016</a>), makes the genetic basis of adaptation difficult to study in this family. Dense linkage maps have been produced to address these difficulties for a variety of Salmoninae, including Arctic Charr (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nugent <em>et al.</em> 2017</a>), Brook Trout (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hale <em>et al.</em> 2017</a>), Brown Trout (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Leitwein <em>et al.</em> 2017</a>) and Chinook Salmon (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">McKinney <em>et al.</em> 2016</a>). These studies typically pair the use of dense linkage maps with the Atlantic Salmon (<em>Salmo salar</em>) reference genome to improve the genomic resolution of their analyses. However, due to the ∼50 million-year divergence time between Salmoninae and Coregoninae, and the limited number and density of whitefish linkage maps, the analysis of genomic whitefish datasets to answer questions about the physical distribution of loci and their function is limited (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rogers <em>et al.</em> 2001</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rogers and Bernatchez 2004</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rogers and Bernatchez 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gagnaire <em>et al.</em> 2013</a>). Only one whitefish linkage map produced using a restriction-site associated digestion (RAD) sequencing approach is available and was produced using data from North American whitefish (<em>C. clupeaformis</em>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gagnaire <em>et al.</em> 2013</a>). It includes 3438 single nucleotide polymorphism (SNP) markers resolved into 40 linkage groups (matching the karyotype of <em>C. clupeaformis</em>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Phillips and Rab 2007</a>) and was successfully used to investigate expression QTL in <em>C. clupeaformis</em> (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Gagnaire <em>et al.</em> 2013</a>). However, studies which later described synteny patterns between salmonid genomes struggled to confidently resolve the relationships between lake whitefish linkage groups and other salmonid chromosomes using this map (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>). The use of this map for investigating the remarkable European adaptive radiation of whitefish is further limited, due to the specificity of RAD markers and limited knowledge about genetic differentiation between <em>C. clupeaformis</em> and European whitefish species (<em>C. lavaretus</em> spp. complex) (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Østbye <em>et al.</em> 2005</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Hudson <em>et al.</em> 2011</a>). The production of a European whitefish linkage map is therefore essential to study genome evolution within these extraordinary radiations.</p><p id="p-8">In this study we produce a detailed linkage map for Alpine whitefish using a RAD sequencing approach. We produced both sex-specific and sex-averaged linkage maps for <em>Coregonus sp. “Albock</em>”, one member of the Alpine whitefish clade, from one F1 lab-bred cross. Here, we describe the sex-averaged and sex-specific linkage maps of <em>C. sp “Albock</em>” and use our sex-averaged linkage map to identify synteny between <em>C. sp. “Albock</em>” and Atlantic Salmon (<em>Salmo salar</em>). We identify rearrangements present between the two species which reflect the occurrence of fission and fusion events following the Ss4R whole genome duplication, some of which were confidently identified to be shared only between members of the Salmoninae subfamily in past studies. We also discuss the results of our synteny mapping in the context of the rediploidization history of salmonids. This <em>Coregonus</em> linkage map will facilitate future research regarding the genomic basis of adaptation in the adaptive radiation of Swiss whitefish and assist with the ongoing <em>de novo</em> assembly of the whitefish genome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental cross</h3><p id="p-9">One F1 family consisting of two parents and 156 offspring was used for linkage map construction. Both parent whitefish were sexually ripe, adult, <em>Coregonus sp. “Albock”</em>, a formally undescribed species which is one member of the European whitefish lineage (<em>C. lavaretus</em> spp. complex). <em>Coregonus sp. “Albock</em>” likely originates from an introduction of whitefish from Lake Constance into Lake Thun and taxonomic description of the species is in progress. The parental whitefish collected from Lake Thun in December 2016 were crossed <em>in vitro</em> by mixing sperm and eggs (obtained from the cantonal hatchery) together before adding cold water to harden successfully fertilized eggs. Fertilized eggs were then placed in a flow-through system which ran 5° lake water over the eggs for 11 weeks until they began to hatch. Before larvae had fully utilized their yolk sac they were sedated and killed with MS222 (50 mg/l for sedation; 200 mg/l for euthanization; buffered with sodium bicarbonate 500 mg/l) and preserved in 100% ethanol (February 2017; Animal Permit number LU03/15).</p></div><div id="sec-3" class="subsection"><h3>DNA extraction, library preparation and sequencing</h3><p id="p-10">DNA for both parental whitefish was extracted from muscle tissue. Progeny DNA was extracted following the digestion of 176 whole larvae. Both parent and progeny DNA was extracted using DNeasy Blood and Tissue extraction kit (Qiagen). The DNA concentration of each extract was measured using the Qubit 1.0 Fluorometer (Thermo Fisher). In total five RAD libraries were made, with 44 F1 samples pooled into each of the four offspring RAD libraries and the two parental samples pooled into a fifth library. Each library was produced following the protocol of <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baird <em>et al.</em> (2008)</a> with slight modifications. The DNA concentration of each individual was normalized prior to the restriction enzyme digestion step to ensure 1 µg DNA was included for each F1. Since the parental library contained only two individuals, to achieve higher sequencing depth, 18 µg DNA from each parent was used for the digestion. Pre-digestion DNA integrity and the success of enzyme digestion was confirmed by running a subset of samples on a 1.4% agarose gel before and after enzyme digestion. The restriction enzyme digestion was carried out using the <em>Sbf-1</em> enzyme, which has been shown to digest salmonid DNA effectively (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Gagnaire <em>et al.</em> 2013</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>), before the digested genomic DNA was ligated to individual-specific barcodes and the forward Illumina adaptor. Size selection after shearing took place using a SageELF to retain only DNA fragments between 300 and 700 base pairs (bp). Fragments were then amplified in a PCR after the ligation of the reverse Illumina adaptor. Each library was spiked with PhiX DNA (∼10% of reads) before being single-end sequenced, each on a single lane of Illumina HiSeq 2500 with 100 cycles at the Lausanne Genomic Technologies Facility (Switzerland).</p></div><div id="sec-4" class="subsection"><h3>Sequence processing and genotyping</h3><p id="p-11">The first step of processing the 100 bp sequenced reads was to remove all PhiX reads using a Bowtie2 mapping approach (using default parameters except for the number of allowed mismatches which we set to 1; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Langmead and Salzberg 2012</a>). Next, all reads from the parental library were filtered for quality using Trimmomatic v.0.35 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>). Bases were trimmed from the beginning and end of reads if they were below quality 3, a sliding-window approach was used with a 4 base wide window to trim bases below a quality score of 15. Reads were only retained if they had an average quality of 30 and if they were longer than 50 bp. Reads from the parental library and four offspring libraries were then demultiplexed and offspring reads were trimmed to 90 bp using the <em>process_radtags</em> module in Stacks version 1.40 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Catchen <em>et al.</em> 2013</a>). Next, 20 offspring with &lt; 1 million reads were discarded to leave both parents and 156 F1 offspring for analysis. A <em>de novo</em> reference assembly was produced by combining only reads from both parents, running the <em>ustacks</em> module in Stacks (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Catchen <em>et al.</em> 2013</a>) to identify putative SNP loci present in the parents of the cross (with a minimum coverage depth of 20) and the concatenation of these consensus stacks (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Catchen <em>et al.</em> 2013</a>). An index of this reference was then produced with Bowtie2 (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Langmead and Salzberg 2012</a>). Both parental and all offspring FASTA files were aligned to the parental <em>de novo</em> reference assembly using Bowtie2 (using default parameters except for the number of allowed mismatches which we set to 1) resulting in individual alignment files. The GATK <em>Haplotype Caller</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Poplin <em>et al.</em> 2017</a>) was used to call genotypes, producing a VCF file retaining only SNPs genotyped with a minimum base quality score of 20 and a minimum confidence threshold of 20, <em>i.e.</em>, p-error 0.01. The use of GATK allowed us to further filter this genotype file with VCFtools (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Danecek <em>et al.</em> 2011</a>) to leave 20635 biallelic SNPs with a minimum phred quality score of 30 with indels removed. Since only one generation of offspring are included in an F1 linkage map, the most informative loci are those that are heterozygous in one parent and homozygous in the other (<em>e.g.</em>, maternal Aa, paternal aa or maternal aa, paternal Aa). Offspring can therefore be heterozygous or homozygous (<em>e.g.</em>, Aa or aa in an expected ratio of 1:1) and the phasing/origin of each allele is known. In addition to these highly informative loci, loci for which both parents are heterozygous can also provide information in the offspring in certain linkage mapping programs (<em>e.g.</em>, maternal Aa, paternal Aa). In these cases, three offspring genotypes may be observed <em>e.g.</em>, AA, Aa, aa in an expected ratio of 1:2:1 with only homozygous offspring being informative since we know that one copy of each allele is from each parent (<em>e.g.</em>, AA offspring or aa offspring have received one A from each parent or one a from each parent, respectively). Heterozygous offspring genotypes are uninformative since the origin of each allele is unknown (<em>e.g.</em>, Aa offspring may have received A or a from either parent). Loci were then filtered in R (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">R Core Team 2014</a>) leaving only informative loci segregating in these two ways as well as removing any loci with missing data in either parent. All SNPs from RAD loci with more than three SNPs were removed and one SNP was chosen at random from those RAD loci with two SNPs. Remaining loci with over 20% missing data were also removed using R (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">R Core Team 2014</a>), leaving 9757 loci for linkage mapping.</p></div><div id="sec-5" class="subsection"><h3>Linkage mapping</h3><p id="p-12">Linkage map construction was carried out using Lep-MAP3 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rastas 2017</a>). First custom R and python scripts were used to convert the VCF file containing informative loci to Lep-MAP3 format before it was converted to a genotype likelihood table using the script linkage2post.awk and the <em>Transpose</em> module (Lep-MAP2; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rastas <em>et al.</em> 2015</a>). Next Lep-MAP3 modules were used starting with the <em>ParentCall2</em> module identifying 7800 informative markers. The <em>Filtering2</em> module was then used to remove markers with significant segregation distortion (dataTolerance = 0.001). Linkage groups were then identified using <em>SeparateChromosomes2</em> with a logarithm of odds (LOD) score of 16 (lodLimit = 16) and the minimum number of markers per linkage group set to 25, resolving 40 linkage groups (corresponding to the 40 whitefish chromosomes identified by karyotyping; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Phillips and Rab 2007</a>) containing 5395 loci before within-group ordering of markers was carried out (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rastas 2017</a>). Due to the slight stochastic variation in marker distances between runs, the <em>OrderMarkers2</em> module was used, specifying a sex-specific map (sexAveraged = 0), three times on each linkage group to produce a male and a female linkage map. This procedure was then repeated specifying a sex-averaged map (sexAveraged = 1). The marker orders with the highest likelihoods for each linkage group for each type of map were combined to produce the final most likely male and female sex-specific maps and one final sex-averaged map, each positioning the same 5395 SNP markers. A custom R script was used to calculate differences in the marker densities and lengths between maps and the sex-averaged map was plotted using MapChart (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Voorrips 2002</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">R Core Team 2014</a>).</p></div><div id="sec-6" class="subsection"><h3>Synteny analysis</h3><p id="p-13">To identify synteny between the 29 Atlantic Salmon chromosomes and the 40 whitefish linkage groups, the <em>de novo</em> assembled RAD loci which were produced using the reads of the two parents of the cross, were mapped to the <em>Salmo salar</em> genome using Stampy v. 1.0.22 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lunter and Goodson 2011</a>) to produce an alignment file for all reference loci. Since whitefish and Atlantic Salmon are ∼52 million years divergent and transcript analysis has shown them be 93% similar, a divergence percentage of 7% (substitution rate = 0.07) was specified during mapping (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koop <em>et al.</em> 2008</a>). A custom R script was then used to match the 5395 RAD loci within the complete sex-averaged map to the corresponding loci in the reference whitefish - Atlantic Salmon alignment file, extracting the salmon chromosome, base pair position and mapping quality. Mapped loci were then stringently filtered by their mapping quality score (MAPQ &gt; 30) and the salmon chromosome with the most hits was noted. Linkage groups were then ordered to reflect their synteny with salmon chromosomes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and renamed with the prefix ‘W’ to match salmon chromosome ordering. Synteny was visualized using the <em>circlize</em> package (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gu <em>et al.</em> 2014</a>) in R plotting all links from reads with MAPQ &gt; 30 to the corresponding salmon chromosome arm and position within each chromosome arm (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). To investigate the distribution of mappings within the salmon genome, specifically why some chromosome arms had few mappings, the rediploidization history of those arms was taken into account. Chromosome arms were classified as either AORe (n = 30) or LORe (n = 14) based on when in the salmonid lineage rediploidization occurred (from <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>). Chromosome arms which had some minor proportion of LORe within a largely AORe chromosome arm (Ssa3p, Ssa5p, Ssa9qb, Ssa13qa, Ssa15qb and Ssa23) were excluded. An expected number of mappings was calculated for each chromosome arm based on the arm length relative to the sum of all arm lengths and the total number of mappings included in our synteny map. A ratio of expected/observed mappings was then calculated for each chromosome arm and plotted (with the exception of Ssa8q because of its infinite value resulting from 0 observed mappings), grouping chromosome arms by their mode of rediploidization (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A Wilcoxon rank sum test was carried out to test whether expected/observed mapping ratios for AORe and LORe chromosome arms were significantly different.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491684/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491684/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Table comparing statistics for the sex-averaged, female and male <em>C. sp. “Albock</em>” linkage maps. The results of synteny analysis are included, showing the homologous Atlantic Salmon chromosome (Ssa) for each whitefish linkage group (Calb) and the re-ordered whitefish linkage group name (W)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-14">Fastq files for all 156 offspring and both parents are deposited in the NCBI short read archive (SRA accession PRJNA478121). All R, Python and bash scripts used can be accessed at <a href="https://github.com/RishiDeKayne/">https://github.com/RishiDeKayne/</a>. Supplemental material including the genotype file (VCF), the Lep-MAP inpute file and all three linkage maps are available at Figshare: <a href="https://doi.org/10.25387/g3.7093799">https://doi.org/10.25387/g3.7093799</a>.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Linkage mapping</h3><p id="p-15">Our F1 cross was produced by crossing two wild <em>C. sp. “Albock</em>” adults. Both parents and 156 F1 offspring were successfully genotyped using a RAD-seq approach. In total 9757 SNPs were retained following stringent quality control and loci filtering steps, with 7800 identified as informative in Lep-MAP3 (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Rastas 2017</a>). Finally, 5395 SNPs were assigned to, and arranged within, linkage groups in both sex-averaged and sex-specific maps (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). With the LOD score of 16, 40 linkage groups, corresponding to the 40 chromosomes observed in karyotype studies of the closely related European whitefish (<em>C. lavaretus</em>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Phillips and Rab 2007</a>), were formed with an average of 135 markers per linkage group (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Map lengths varied from 2293.86 cM in the sex-averaged map to 2460.10 cM and 2263.05 cM in the female and male maps, respectively. All three maps produced in this study were considerably shorter than a previously published <em>C. clupeaformis</em> linkage map containing 3438 RAD markers, which had a total map length of 3061 cM (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Gagnaire <em>et al.</em> 2013</a>). Our sex-averaged <em>C. sp. “Albock</em>” map had an average linkage group length of 57.35 cM with the female and male sex-specific maps showing average linkage group lengths of 61.50 cM and 56.58 cM, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coregonus sp. &#x201C;Albock&#x201D; (European whitefish species complex) linkage map showing the grouping and position of 5395 SNPs within a sex-averaged linkage map. The length of each of the 40 linkage groups is indicated by the scale in cM with linkage groups ordered by marker number from highest to lowest." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1759089494" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Coregonus sp. &#x201C;Albock&lt;/em&gt;&#x201D; (European whitefish species complex) linkage map showing the grouping and position of 5395 SNPs within a sex-averaged linkage map. The length of each of the 40 linkage groups is indicated by the scale in cM with linkage groups ordered by marker number from highest to lowest.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3745/F1.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3745/F1.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child"><em>Coregonus sp. “Albock</em>” (European whitefish species complex) linkage map showing the grouping and position of 5395 SNPs within a sex-averaged linkage map. The length of each of the 40 linkage groups is indicated by the scale in cM with linkage groups ordered by marker number from highest to lowest.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The number of SNPs per linkage group varied from 31 to 253 and the lengths of linkage groups varied from 15.20 cM to 83.57 cM in the sex-averaged map. Two linkage groups, Calb38 and Calb39, were comprised only of male-informative loci and therefore had lengths of 0 cM in the female map, with the longest linkage group in the female map being Calb02 at 101.33 cM. In the male map linkage groups vary in length from 7.41 cM to 88.06 cM for linkage groups Calb40 and Calb07.</p><p id="p-18">Our sex-averaged map has high resolution, with a low average distance between adjacent markers of 0.46 cM, varying from 0.27 cM in Calb04 to 0.77 cM in Calb34. The linkage map of the close relative <em>C. clupeaformis</em>, a representative of the North American whitefish lineage, had a marker resolution across the map of 0.89 cM, around half the density of our <em>C. sp “Albock</em>” map. In the female map the average inter-marker distance was 0.48 cM varying in linkage groups (only considering linkage groups &gt; 0 cM) from 0.31 cM in Calb04 to 0.99 cM in Calb35. The average inter-marker distance in the male map was 0.46 cM with the smallest and largest ratios found in Calb12 and Calb39 respectively with 0.18 cM and 1.05 cM.</p><p id="p-19">Sex differences can be observed by comparing our sex-specific linkage maps for <em>C. sp. “Albock</em>”. Comparing total map lengths for the female and male maps gives a female:male recombination ratio of 1.09, however, this does not account for the two whitefish linkage groups which have length 0 cM in our female map (Calb38 and Calb39). Calculating this female:male recombination ratio for each linkage group separately, including only those &gt; 0 cM in both maps, results in a ratio of 1.25. Salmonid species have been shown to have sexual dimorphisms in recombination rate with published female:male recombination ratios varying from 1.38 in Atlantic Salmon (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lien <em>et al.</em> 2011</a>) to 2.63 in Brown Trout (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gharbi <em>et al.</em> 2006</a>) and therefore sexual dimorphism in whitefish appears to be low in comparison to other salmonids. However, since each sex-specific linkage map represents the recombination landscape in one individual, in our case each parent of the F1 cross, more than one linkage map is required to disentangle individual variation in recombination rate and consistent sex specific recombination rate variation (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Moen <em>et al.</em> 2004</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lien <em>et al.</em> 2011</a>). Although our female:male recombination ratio does not conclusively show variable recombination rates between females and males it still reveals a striking difference in map length considering the inclusion of the same set of markers for each. Studies on other teleost species, including stickleback, have also reported detailed empirical evidence of sexually dimorphic recombination rates, calculating female:male recombination ratios of linkage map lengths to be 1.64 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sardell <em>et al.</em> 2018</a>). Future work should aim to compare and contrast the recombination landscape of whitefish to the detailed sexually dimorphic recombination patterns observed in drosophila, mice, deer and various fish species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dunn 1920</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lenormand and Dutheil 2005</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kubota <em>et al.</em> 2017</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Sardell <em>et al.</em> 2018</a>).</p></div><div id="sec-10" class="subsection"><h3>Synteny analysis</h3><p id="p-20">Synteny analysis was carried out to investigate broad scale genome structural variation, such as fission and fusions of chromosomes or chromosome arms, within the Salmonidae family. Stringent filtering of mapped RAD loci to the salmon genome was applied to identify synteny while excluding uncertain mappings. From 5395 loci included in our linkage map we retained 839 mappings of high quality, which were spread across all 40 whitefish linkage groups (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Synteny between salmon chromosomes and whitefish linkage groups was determined by identifying the most common salmon chromosome the markers on each whitefish linkage group mapped to. We also investigated the distribution of mappings along the Atlantic Salmon genome based on how rediploidization is thought to have proceeded following the Ss4R whole genome duplication at the finer chromosome arm level (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). In ‘Ancestral Ohnologue Resolution’ (AORe) regions salmon and whitefish have conserved patterns of rediploidization, which occurred in their shared ancestor resulting in a 1:1 orthology between ohnologs (<a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>). However, in ‘Lineage-specific Ohnologue Resolution’ (LORe) regions, specifically the large duplicated collinear blocks ’Ssa2p-Ssa5q’, ’Ssa2q-Ssa12qa’, ’Ssa3q-Ssa6p’, ’Ssa4p-Ssa8q’, ’Ssa7q-Ssa17qb’, ’Ssa11qa-Ssa26’ and ’Ssa16qb-Ssa17qa’ (highlighted with red links in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) identified by <a id="xref-ref-61-7" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> (2017)</a>, rediploidization has proceeded independently in salmon and whitefish and ohnologs share a 2:2 orthology. As expected we identified that LORe regions had statistically fewer mappings than expected compared to AORe regions (Wilcoxon rank sum test: W = 0, <em>P</em> = 5.468x10<sup>−11</sup>) and conclude that this is the result of the mapping parameters we used (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). These parameters, aimed to identify single best mapping positions, work well in AORe regions, where we calculated that the observed number of mappings is close to the expected number (<em>i.e.</em>, a ratio of 1), meaning mappings are evenly distributed between AORe chromosome arms. Mappings to chromosome arms which make up collinear LORe blocks are not expected to be unique, lowering the mapping confidence (<em>i.e.</em>, mapping quality score) of loci there, which resulted in the filtering out of these mappings. Confident mappings within LORe regions are therefore scarce because these regions do not follow the 1:1 ohnologue orthology that we required through our mapping parameters to keep markers.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny plot identifying homologous whitefish (C. sp. &#x201C;Albock&#x201D;) linkage groups and Atlantic Salmon (Salmo salar) chromosomes. The outermost track on the Atlantic Salmon side (left) of the plot shows the locations and names of chromosome arms (alternating in white and gray). The next track inwards shows whitefish linkage groups (right) and salmon chromosomes (left) and linkage group-chromosome synteny is denoted by the same coloring of linkage groups and chromosomes. Black salmon chromosomes Ssa02 and Ssa26 represent chromosomes with no homologous whitefish linkage groups. Salmon chromosome Ssa08 is colored in white and had no significant mappings. The innermost track highlights the location of the 839 RAD markers in the whitefish linkage map (right) which confidently map to the salmon genome (left). Those markers which map to the identified homologous chromosomes are colored in gray and those which deviate are colored in black. Links represent the mappings of 839 markers within the whitefish linkage map which were successfully mapped to the Atlantic Salmon genome. &#x2018;Lineage-specific Ohnologue Resolution&#x2019; (LORe) regions within the salmon genome, identified by Robertson et al. (2017), are shown with broad red links between salmon chromosome arms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1759089494" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synteny plot identifying homologous whitefish (&lt;em&gt;C. sp. &#x201C;Albock&lt;/em&gt;&#x201D;) linkage groups and Atlantic Salmon (&lt;em&gt;Salmo salar&lt;/em&gt;) chromosomes. The outermost track on the Atlantic Salmon side (left) of the plot shows the locations and names of chromosome arms (alternating in white and gray). The next track inwards shows whitefish linkage groups (right) and salmon chromosomes (left) and linkage group-chromosome synteny is denoted by the same coloring of linkage groups and chromosomes. Black salmon chromosomes Ssa02 and Ssa26 represent chromosomes with no homologous whitefish linkage groups. Salmon chromosome Ssa08 is colored in white and had no significant mappings. The innermost track highlights the location of the 839 RAD markers in the whitefish linkage map (right) which confidently map to the salmon genome (left). Those markers which map to the identified homologous chromosomes are colored in gray and those which deviate are colored in black. Links represent the mappings of 839 markers within the whitefish linkage map which were successfully mapped to the Atlantic Salmon genome. &#x2018;Lineage-specific Ohnologue Resolution&#x2019; (LORe) regions within the salmon genome, identified by Robertson &lt;em&gt;et al.&lt;/em&gt; (2017), are shown with broad red links between salmon chromosome arms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3745/F2.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3745/F2.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491574" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Synteny plot identifying homologous whitefish (<em>C. sp. “Albock</em>”) linkage groups and Atlantic Salmon (<em>Salmo salar</em>) chromosomes. The outermost track on the Atlantic Salmon side (left) of the plot shows the locations and names of chromosome arms (alternating in white and gray). The next track inwards shows whitefish linkage groups (right) and salmon chromosomes (left) and linkage group-chromosome synteny is denoted by the same coloring of linkage groups and chromosomes. Black salmon chromosomes Ssa02 and Ssa26 represent chromosomes with no homologous whitefish linkage groups. Salmon chromosome Ssa08 is colored in white and had no significant mappings. The innermost track highlights the location of the 839 RAD markers in the whitefish linkage map (right) which confidently map to the salmon genome (left). Those markers which map to the identified homologous chromosomes are colored in gray and those which deviate are colored in black. Links represent the mappings of 839 markers within the whitefish linkage map which were successfully mapped to the Atlantic Salmon genome. ‘Lineage-specific Ohnologue Resolution’ (LORe) regions within the salmon genome, identified by <a id="xref-ref-61-8" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> (2017)</a>, are shown with broad red links between salmon chromosome arms.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot highlighting the higher expected/observed ratio of markers mapping to the 14 &#x2018;Lineage-specific Ohnologue Resolution&#x2019; (LORe) chromosome arms compared to the 30 &#x2018;Ancestral Ohnologue Resolution&#x2019; (AORe) chromosome arms. The null expectation of expected mappings/observed mappings is indicated by the dotted line where expected/observed = 1. Three asterisks denote the significant difference between the expected/observed number of mappings ratio between AORe and LORe regions (Wilcoxon rank sum test: W = 0, P = 5.468x10&#x2212;11)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1759089494" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Boxplot highlighting the higher expected/observed ratio of markers mapping to the 14 &#x2018;Lineage-specific Ohnologue Resolution&#x2019; (LORe) chromosome arms compared to the 30 &#x2018;Ancestral Ohnologue Resolution&#x2019; (AORe) chromosome arms. The null expectation of expected mappings/observed mappings is indicated by the dotted line where expected/observed = 1. Three asterisks denote the significant difference between the expected/observed number of mappings ratio between AORe and LORe regions (Wilcoxon rank sum test: W = 0, &lt;em&gt;P&lt;/em&gt; = 5.468x10&lt;sup&gt;&#x2212;11&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3745/F3.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3745/F3.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3745/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Boxplot highlighting the higher expected/observed ratio of markers mapping to the 14 ‘Lineage-specific Ohnologue Resolution’ (LORe) chromosome arms compared to the 30 ‘Ancestral Ohnologue Resolution’ (AORe) chromosome arms. The null expectation of expected mappings/observed mappings is indicated by the dotted line where expected/observed = 1. Three asterisks denote the significant difference between the expected/observed number of mappings ratio between AORe and LORe regions (Wilcoxon rank sum test: W = 0, <em>P</em> = 5.468x10<sup>−11</sup>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The prevalence of delayed rediploidization is likely the reason that three salmon chromosomes, Ssa02, Ssa08 and Ssa26 were not identified as homologs to any of our whitefish linkage groups, with Ssa08 having no significant mappings at all. All three of these chromosomes, specifically the Ssa08q, Ssa02p, Ssa02q and Ssa26 arms, are LORe regions and the lack of markers mapped to these regions in our analysis is likely caused by an abundance of 2:2 orthology between salmon and whitefish. Markers which might have mapped to these salmon chromosomes have likely been filtered out due to their poor mapping scores. This may also underpin the similarly uncertain assignment of synteny between the <em>C. clupeaformis</em> linkage map and these regions, carried out by <a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> (2016)</a>.</p><p id="p-24">Only a small number of markers on each whitefish linkage group mapped to a different salmon chromosome than the identified homologous chromosome (indicated with black lines on the innermost track in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and evidenced by the low abundance of non-parallel links from each linkage group in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). A large proportion of non-parallel links identified in our synteny analysis connect to LORe regions. However, the largest of these deviations is a series of links (16) from W02 (which was identified as homologous to Ssa01 with 18 links) to Ssa19, an AORe region. Due to the similar abundance of links to two different salmon chromosomes and the fact that rediploidization patterns in this region are shared by salmon and whitefish this series of mappings might rather reflect a whitefish specific fusion of two Atlantic Salmon chromosome arms, Ssa01qa and Ssa19.</p><p id="p-25">While multiple salmonid linkage maps, including those of <em>C. clupeaformis</em> and Rainbow Trout, identified synteny from two linkage groups to one salmon chromosome for Ssa05, Ssa06, Ssa14, Ssa17 and Ssa19, we only identify synteny from one European whitefish linkage group to each of the salmon chromosomes (Ssa05-W08, Ssa06-W09, Ssa14-W21, Ssa17-W26 and Ssa19-W29; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-73-5" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>). Although this pattern of synteny could suggest unique genome structure in <em>C. sp. “Albock</em>” (namely that each of these linkage groups in <em>C. sp. “Albock</em>” is a fusion of two other linkage groups present in other salmonids) the patterns of synteny we observe as well as those identified by <a id="xref-ref-73-6" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> (2016)</a> may be complicated by rediploidization history as indicated for multiple Atlantic Salmon chromosomes. It is now known that chromosome arms Ssa05q, Ssa06p and Ssa17qa and Ssa17qb fall within LORe regions (<a id="xref-ref-61-9" class="xref-bibr" href="#ref-61">Robertson <em>et al.</em> 2017</a>) and therefore the establishment of synteny relationships to these regions is challenging, especially when using a mapping approach with RAD data (90 bp only). Further work should therefore identify whether our one linkage group to one salmon chromosome pattern of synteny is consistent for W08, W09 and W26 but this would require the availability of longer sequences for synteny analysis. However, both Ssa14 and Ssa19 are within AORe regions with expected/observed ratios of mappings close to 1 and our identification of synteny from one linkage group to each of these chromosomes (W21-Ssa14 and W29-Ssa19) should not be affected by rediploidization. This pattern may therefore reflect European whitefish-specific chromosome fusions, although the mapping of some markers from W10 to Ssa14qb and similarly some markers from W02 map to Ssa19qb (as discussed above) suggests that the confident assignment of synteny between these regions will require a denser marker set.</p><p id="p-26">We also identify one possible European whitefish-specific fission event with markers from both W38 and W39 mapping to Ssa28, an AORe dominated chromosome which is homologous to only one linkage group in each salmonid species compared by <a id="xref-ref-73-7" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> (2016)</a> including <em>C. clupeaformis</em>. It is therefore possible that a fission event has occurred in the European whitefish lineage, however, due to relatively low number and density of markers on W38 and W39 future investigation should aim to clarify this pattern.</p><p id="p-27">We identified two salmon chromosomes which were each homologous to three different whitefish linkage groups; Ssa01 to W01, W02 and W03 and Ssa09 to W11, W12 and W13 (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). These Atlantic Salmon chromosomes have been identified to map to three linkage groups in other salmonids including Brook Trout, Arctic Charr, Coho Salmon and various <em>Oncorhynchus</em> species, however, synteny with <em>C. clupeaformis</em>, the only member of Coregoninae included in these comparisons, was less clear (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-73-8" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Hale <em>et al.</em> 2017</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Nugent <em>et al.</em> 2017</a>). This syntenic pattern has been attributed to fusion events which were unique to the Atlantic Salmon lineage only. Here we add to the evidence provided by the <em>C. clupeaformis</em> linkage map that this synteny is also consistent with Coregoninae despite their significant divergence from members of the Salmoninae.</p><p id="p-28">Synteny analysis between members of Salmonidae also identified a number of Atlantic Salmon chromosomes which each show synteny with two linkage groups (<a id="xref-ref-73-9" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Hale <em>et al.</em> 2017</a>). We find a similar pattern of synteny between <em>Salmo salar</em> and <em>Coregonus</em> for many of these salmon chromosomes including Ssa03 (to W04 and W05), Ssa10 (to W14 and W15), Ssa13 (to W19 and W20), Ssa15 (to W22 and W23), Ssa16 (to W24 and W25), Ssa18 (to W27 and W28) and Ssa20 (to W30 and W31) (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). In addition to these, our synteny analysis also identified Ssa04 as homologous to W06 and W07 and Ssa11 as homologous to W16 and W17. However, links from W07 and W17 map to the LORe regions Ssa04p and Ssa17qa, and Ssa11qa and as with other salmon chromosomes within LORe regions this complicates the assignment of synteny. Although we can be confident that W06 is homologous to Ssa04q and W16 to Ssa11qb, since both of these chromosome arms are AORe regions, the dominance of LORe in Ssa04p and Ssa11qa complicates the assignment of synteny with W07 and W17. We also find that the multiple one to one relationships between salmon chromosomes and salmonid linkage groups identified by <a id="xref-ref-73-10" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> (2016)</a> are also consistent with our map including those to Ssa12 (W18), Ssa22 (W33), Ssa23 (W34), Ssa24 (W35), Ssa25 (W36), Ssa27 (W37) and Ssa29 (W40; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-29">Two salmon chromosomes, Ssa07 and Ssa21 were shown by <a id="xref-ref-73-11" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> (2016)</a> to have synteny to two linkage groups in <em>C. clupeaformis</em> but only one linkage group in all other salmonids. Our <em>C. sp. “Albock</em>” map identifies synteny from only one linkage group, W10, to Ssa07 and similarly from W32 to Ssa21 suggesting the pattern of synteny may not be conserved between <em>Coregonus</em> species. Since Ssa07q is a LORe dominated chromosome arm the lack of synteny identified to a second whitefish linkage group may be the result of the lack of 1:1 ohnolog orthology and therefore a lack of confident mappings. The pattern of Ssa21 on the other hand most likely represents a difference between <em>C. cluepeaformis</em> and <em>C. sp. “Albock</em>” since Ssa21 has an expected/observed mappings ratio of 0.94 (close to 1) and a high density of markers. Further work must therefore be carried out to better identify potential genome structural variation between <em>C. sp. “Albock</em>” and <em>C. clupeaformis</em>.</p><p id="p-30">Both broad and small scale structural variations, including inversions, duplications and deletions, have been observed between closely related species and the mis-segregation which can occur during meiosis as a result of these variations is thought to be able to play a role in the speciation process (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Feulner and De-Kayne 2017</a>). It is therefore possible that European and North American whitefish lineages (and even species within these lineages) have unique structural variations which may underpin reproductive isolation in sympatry. Without more detailed information on genome wide synteny and the occurrence of structural variation between these two lineages it is difficult to determine whether the observed variation in synteny patterns to the Atlantic Salmon (<em>e.g.</em>, with regards to Ssa14, Ssa19, Ssa21 and Ssa28) represents true variation between these species or variation in linkage mapping resolution and accuracy. A comparison of synteny between our <em>C. sp. “Albock</em>” map and the Atlantic Salmon (using our synteny mapping approach) and the <em>C. clupeaformis</em> map to the Atlantic Salmon (compared by <a id="xref-ref-73-12" class="xref-bibr" href="#ref-73">Sutherland <em>et al.</em> 2016</a>) can be found in Table S1.</p></div><div id="sec-11" class="subsection"><h3>The development of genomic resources for European whitefish</h3><p id="p-31">A wealth of genomic resources used to study adaptation and speciation are now available for a variety of systems. Multiple species from popular model radiations including Galapagos finches (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lamichhaney <em>et al.</em> 2015</a>) and Lake Victoria cichlids (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brawand <em>et al.</em> 2014</a>) now have highly contiguous, well curated and annotated, reference genomes. These resources provide the opportunity to ask specific questions about intra and inter-species genomic differences with many studies focusing on understanding the genomic basis of adaptation and reproductive isolation. Studies can now utilize high throughput whole-genome sequencing to achieve high depth of coverage and are able to map these reads to a reference genome to understand the distribution of genomic variation along the genome. However, many interesting organisms including the many ecologically diverse salmonids have only a handful of highly contiguous and well annotated reference genomes available. Current well annotated salmonid genomes include those of Atlantic Salmon (<em>Salmo salar</em>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Lien <em>et al.</em> 2016</a>) and Rainbow Trout (<em>Oncorhynchus mykiss</em>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berthelot <em>et al.</em> 2014</a>). However, recently assemblies of Chinook Salmon (<em>Oncorhynchus tshawytscha</em>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christensen <em>et al.</em> 2018</a>), Coho Salmon (<em>Oncorhynchus kisutch</em>; NCBI BioProject: PRJNA352719), Arctic Charr (<em>Salvelinus alpinus</em>; NCBI BioProject: PRJNA348349; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Christensen <em>et al</em>. 2018</a>) and Grayling (<em>Thymallus thymallus</em>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Varadharajan <em>et al.</em> 2018</a>) have also been published. Although these genomes expand the diversity of salmonid genomes available dramatically, they are still relatively distantly related to the diverse whitefish subfamily Coregoninae.</p><p id="p-32">Our linkage map fills a gap in the resources available to analyze European whitefish genetic data allowing investigation into this species rich, ecologically diverse, lineage. The patterns of synteny between European whitefish and Atlantic Salmon reported here should be further investigated once whitefish genomes become available to identify synteny at a finer scale, identifying chromosome fission and fusion events and possible inversions also within the <em>Coregonus</em> genus. Our linkage map can also be paired with future resources to investigate the outcome of whole genome duplication including estimations of the rediploidized proportion of the genome, already calculated in Atlantic Salmon. Future work should further aim to identify regions of the genome which may underpin reproductive isolation in whitefish to better understand the speciation mechanism in this adaptive radiation.</p><p id="p-33">In conclusion, we have produced the densest <em>Coregonus</em> linkage map to date, with a total sex-averaged map length of 2293.86 cM containing 5395 SNP loci. We have found evidence of sex-specific recombination rate variation within <em>C. sp. “Albock</em>” by calculating the female:male recombination ratio <em>i.e.</em>, a ratio of female and male linkage map lengths. The level of heterochiasmy inferred by this ratio is reflected in other species with known sex-specific recombination variation, including other salmonids (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gharbi <em>et al.</em> 2006</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Lien <em>et al.</em> 2011</a>). We also show that <em>C. sp. “Albock</em>” linkage groups exhibit synteny with Atlantic Salmon chromosomes, in some cases following a pattern of synteny shared with other salmonid species. This linkage map will facilitate a host of future studies into the genomic basis of adaptation in Alpine whitefish including those on the identification of QTL for traits of interest, the interpretation of genome-wide divergence data and the colocalization of regions under selection <em>e.g.</em>, F<sub>ST</sub> outliers identified from genome scans. It also has the potential to assist in the ongoing assembly of Alpine whitefish reference genomes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">Thanks to Benjamin Gugger and team from Lake Thun whitefish hatchery for providing us with the breeding pair of <em>C. sp. “Albock</em>”. Also, thanks to Anna Feller, David Frei, Andreas Taverna and Erwin Schäffer for their help breeding and maintaining the whitefish larvae and Oliver Selz for his taxonomic expertise. We also thank two anonymous reviewers for their valuable contributions which improved the clarity of the paper. This project is funded by the Swiss National Science Foundation (SNSF project 31003A_163446/1 awarded to PGDF).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-35">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7093799">https://doi.org/10.25387/g3.7093799</a>.</p></li><li class="fn" id="fn-2"><p id="p-36"><em>Communicating editor: R. Houston</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-29"><span class="received-label">Received </span>June 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-03"><span class="accepted-label">Accepted </span>October 3, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 De-Kayne, Feulner</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.1" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD Markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodson</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Allopatric origin of sympatric populations of lake whitefish (<em>Coregonus clupeaformis</em>) as revealed by mitochondrial-DNA restriction analysis.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D44%26rft.spage%253D1263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.3" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms4657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.5" data-doi="10.1038/nature13726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brawand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malinsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic substrate for adaptive radiation in African cichlid fish.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">513</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13726</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D513%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13726%26rft_id%253Dinfo%253Apmid%252F25186727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25186727&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341814900051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.6" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.7" data-doi="10.1371/journal.pone.0195461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakharani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biagi</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkley</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Chinook salmon (<em>Oncorhynchus tshawytscha</em>) genome and transcriptome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0195461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0195461</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0195461%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0195461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0195461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.8" data-doi="10.1371/journal.pone.0204076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rondeau</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nugent</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Arctic charr <em>(Salvelinus alpinus)</em> genome and transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0204076</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0204076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D13%26rft.spage%253De0204076%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0204076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0204076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.9" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.10" data-doi="10.1002/ece3.4375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doenz</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bittner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonlanthen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rapid buildup of sympatric species diversity in Alpine whitefish.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> (<span class="cit-comment">in press</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.4375</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.4375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.4375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.11" data-doi="10.1038/nrg703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping and analysis of quantitative trait loci in experimental populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D52%26rft.atitle%253DMapping%2Band%2Banalysis%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Bexperimental%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg703%26rft_id%253Dinfo%253Apmid%252F11823790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823790&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.12" data-doi="10.1046/j.1365-294x.1999.00581.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Do assemblages of <em>Coregonus</em> (Teleostei: Salmoniformes) in the Central Alpine region of Europe represent species flocks?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">603</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294x.1999.00581.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D8%26rft.spage%253D589%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1999.00581.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1999.00581.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1920</span> <span class="cit-article-title">Linkage in mice and rats.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDunn%26rft.auinit1%253DL.%2BC.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D343%26rft.atitle%253DLINKAGE%2BIN%2BMICE%2BAND%2BRATS%26rft_id%253Dinfo%253Apmid%252F17245945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI1LzMvMzI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.14" data-doi="10.1038/hdy.2010.152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Applications of next generation sequencing in molecular ecology of non-model organisms.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DEkblom%26rft.auinit1%253DR.%26rft.volume%253D107%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DApplications%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bin%2Bmolecular%2Becology%2Bof%2Bnon-model%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.152%26rft_id%253Dinfo%253Apmid%252F21139633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21139633&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291635500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.15" data-doi="10.1016/j.tree.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing and population genomics in non-model organisms.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D51%26rft.atitle%253DGENOME%2BSEQUENCING%2BAND%2BPOPULATION%2BGENOMICS%2BIN%2BNON-MODEL%2BORGANISMS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139972&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330148000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.16" data-doi="10.1146/annurev-ecolsys-110411-160327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Evolutionary Genomics of Birds.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ecolsys-110411-160327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D44%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110411-160327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110411-160327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.17" data-doi="10.1111/jeb.13101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De-Kayne</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome evolution, structural rearrangements and speciation.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1488</span>–<span class="cit-lpage">1490</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.13101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D30%26rft.spage%253D1488%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.13101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.13101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.18" data-doi="10.1534/g3.118.200207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarzer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haesler</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A dense linkage map of Lake Victoria cichlids improved the <em>Pundamilia</em> genome assembly and revealed a major QTL for sex-determination.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2411</span>–<span class="cit-lpage">2420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200207%26rft_id%253Dinfo%253Apmid%252F29760203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNy8yNDExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.19" data-doi="10.3389/fgene.2015.00220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fierst</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using linkage maps to correct and scaffold <em>de novo</em> genome assemblies: methods, challenges, and computational tools.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00220%26rft_id%253Dinfo%253Apmid%252F26150829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26150829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.20" data-doi="10.1016/j.margen.2016.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Fonseca</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Themudo</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Madrigal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sibbesen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Next-generation biology: Sequencing and data analysis approaches for non-model organisms.</span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.margen.2016.04.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BGenomics%26rft.volume%253D30%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.margen.2016.04.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.margen.2016.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic architecture of reproductive isolation during speciation-with-gene-flow in lake whitefish species pairs assessed by RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2483</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D67%26rft.spage%253D2483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.22" data-doi="10.1111/j.1755-0998.2010.02891.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garvin</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saitoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharrett</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Application of single nucleotide polymorphisms to non-model species: a technical review.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">934</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02891.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D10%26rft.spage%253D915%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02891.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02891.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.23" data-doi="10.1534/genetics.105.048330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A Linkage Map for Brown Trout (<em>Salmo trutta</em>): Chromosome Homeologies and Comparative Genome Organization With Other Salmonid Fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2405</span>–<span class="cit-lpage">2419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.048330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048330%26rft_id%253Dinfo%253Apmid%252F16452148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI0MDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.24" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar)</em> genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.25" data-doi="10.1073/pnas.0806238105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheid</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synteny and candidate gene prediction using an anchored linkage map of <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">20106</span>–<span class="cit-lpage">20111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0806238105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806238105%26rft_id%253Dinfo%253Apmid%252F19104060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMTA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.26" data-doi="10.1093/bioinformatics/btu393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eils</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brors</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">circlize implements and enhances circular visualization in R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2811</span>–<span class="cit-lpage">2812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu393%26rft_id%253Dinfo%253Apmid%252F24930139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24930139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343082900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.27" data-doi="10.1534/g3.117.300317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Using Linkage Maps as a Tool To Determine Patterns of Chromosome Synteny in the Genus <em>Salvelinus</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3821</span>–<span class="cit-lpage">3830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300317%26rft_id%253Dinfo%253Apmid%252F28963166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjcvMTEvMzgyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzc0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.28" data-doi="10.1186/1471-2164-13-361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hornett</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheat</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative RNA-Seq analysis in non-model species: assessing transcriptome assemblies as a scaffold and the utility of evolutionary divergent genomic reference species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHornett%26rft.auinit1%253DE.%2BA.%26rft.volume%253D13%26rft.spage%253D361%26rft.epage%253D361%26rft.atitle%253DQuantitative%2BRNA-Seq%2Banalysis%2Bin%2Bnon-model%2Bspecies%253A%2Bassessing%2Btranscriptome%2Bassemblies%2Bas%2Ba%2Bscaffold%2Band%2Bthe%2Butility%2Bof%2Bevolutionary%2Bdivergent%2Bgenomic%2Breference%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-361%26rft_id%253Dinfo%253Apmid%252F22853326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22853326&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.29" data-doi="10.1111/eva.12446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundsgaard-Hansen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonlanthen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Managing cryptic biodiversity: Fine-scale intralacustrine speciation along a benthic gradient in Alpine whitefish (<em>Coregonus</em> spp.).</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D10%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.30" data-doi="10.1098/rspb.2010.0925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonlanthen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rapid parallel adaptive radiations from a single hybridogenic ancestral population.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2010.0925</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2010.0925%26rft_id%253Dinfo%253Apmid%252F20685705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMToiMjc4LzE3MDIvNTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.31" data-doi="10.1534/g3.117.044198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (<em>Cervus elaphus</em>).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2859</span>–<span class="cit-lpage">2870</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.044198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.044198%26rft_id%253Dinfo%253Apmid%252F28667018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yODU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.32" data-doi="10.1111/mec.12810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4035</span>–<span class="cit-lpage">4058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4035%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12810%26rft_id%253Dinfo%253Apmid%252F24863701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24863701&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340419800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.33" data-doi="10.1534/g3.114.012294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative Mapping Between Coho Salmon (Oncorhynchus kisutch) and Three Other Salmonids Suggests a Role for Chromosomal Rearrangements in the Retention of Duplicated Regions Following a Whole Genome Duplication Event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1730</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.012294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012294%26rft_id%253Dinfo%253Apmid%252F25053705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNzE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.34" data-doi="10.1186/1471-2164-9-545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Schalburg</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieph</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A salmonid EST genomic study: genes, duplications, phylogeny and microarrays.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKoop%26rft.auinit1%253DB.%2BF.%26rft.volume%253D9%26rft.spage%253D545%26rft.epage%253D545%26rft.atitle%253DA%2Bsalmonid%2BEST%2Bgenomic%2Bstudy%253A%2Bgenes%252C%2Bduplications%252C%2Bphylogeny%2Band%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-545%26rft_id%253Dinfo%253Apmid%252F19014685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014685&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3745.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kottelat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freyhof</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Handbook of European freshwater fishes</span>, <span class="cit-publ-name">Publications Kottelat</span>, <span class="cit-publ-loc">Switzerland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.36" data-doi="10.1111/are.13415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kubota</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longloy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singhabun</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khammee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessuwan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Quantitative trait locus mapping of growth-related traits in inter-specific F <sub>1</sub> hybrid grouper (<em>Epinephelus fuscoguttatus</em> × <em>E. lanceolatus</em>) in a tropical climate.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">5913</span>–<span class="cit-lpage">5927</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/are.13415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D48%26rft.spage%253D5913%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fare.13415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/are.13415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.37" data-doi="10.1038/nature14181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamichhaney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berglund</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almén</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of Darwin’s finches and their beaks revealed by genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14181%26rft_id%253Dinfo%253Apmid%252F25686609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25686609&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.38" data-doi="10.1073/pnas.84.8.2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Construction of multilocus genetic linkage maps in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">2363</span>–<span class="cit-lpage">2367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.84.8.2363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D84%26rft.issue%253D8%26rft.spage%253D2363%26rft.epage%253D2367%26rft.atitle%253DConstruction%2Bof%2Bmultilocus%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.8.2363%26rft_id%253Dinfo%253Apmid%252F3470801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NC84LzIzNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.39" data-doi="10.1126/science.8091226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic Dissection of Complex Traits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">2037</span>–<span class="cit-lpage">2048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.8091226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLander%26rft.auinit1%253DE.%26rft.volume%253D265%26rft.issue%253D5181%26rft.spage%253D2037%26rft.epage%253D2048%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8091226%26rft_id%253Dinfo%253Apmid%252F8091226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjUvNTE4MS8yMDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.40" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.41" data-doi="10.1534/g3.116.038497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitwein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinand</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pouzadoux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Dense Brown Trout (<em>Salmo trutta</em>) Linkage Map Reveals Recent Chromosomal Rearrangements in theSalmoGenus and the Impact of Selection on Linked Neutral Diversity.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.038497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038497%26rft_id%253Dinfo%253Apmid%252F28235829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNC8xMzY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.42" data-doi="10.1371/journal.pbio.0030063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination Difference between Sexes: A Role for Haploid Selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLenormand%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De63%26rft.epage%253De63%26rft.atitle%253DRecombination%2Bdifference%2Bbetween%2Bsexes%253A%2Ba%2Brole%2Bfor%2Bhaploid%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030063%26rft_id%253Dinfo%253Apmid%252F15736976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15736976&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.43" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-615</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.44" data-doi="10.1038/nature17164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature17164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Correlated trophic specialization and genetic divergence in sympatric lake whitefish ecotypes (<em>Coregonus clupeaformis</em>): support for the ecological speciation hypothesis.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D53%26rft.spage%253D1491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.46" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.111120.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.47" data-doi="10.1098/rspb.2013.2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20132881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2013.2881</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2881%26rft_id%253Dinfo%253Apmid%252F24452024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3NzgvMjAxMzI4ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.48" data-doi="10.1016/j.tig.2017.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Fantastic Beasts and How To Sequence Them: Ecological Genomics for Obscure Model Organisms.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.11.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D34%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.49" data-doi="10.1111/1755-0998.12479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Uchida</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An integrated linkage map reveals candidate genes underlying adaptive variation in Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">783</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D769%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.50" data-doi="10.1111/j.1365-2052.2004.01097.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyheim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munck</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Raya</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A linkage map of Atlantic salmon (<em>Salmo salar</em>) reveals an uncommonly large difference in recombination rate between the sexes.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2004.01097.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DMoen%26rft.auinit1%253DT.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D92%26rft.atitle%253DA%2Blinkage%2Bmap%2Bof%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Ban%2Buncommonly%2Blarge%2Bdifference%2Bin%2Brecombination%2Brate%2Bbetween%2Bthe%2Bsexes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2004.01097.x%26rft_id%253Dinfo%253Apmid%252F15025566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2004.01097.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15025566&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220273100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.51" data-doi="10.1073/pnas.1206625109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Near</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eytan</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornburg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resolution of ray-finned fish phylogeny and timing of diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13698</span>–<span class="cit-lpage">13703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1206625109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206625109%26rft_id%253Dinfo%253Apmid%252F22869754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0LzEzNjk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3745.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grande</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M. V. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Fishes of the world</span>, <span class="cit-publ-name">John Wiley and Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.53" data-doi="10.1534/g3.116.038026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nugent</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A SNP Based Linkage Map of the Arctic Charr (<em>Salvelinus alpinus</em>) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.038026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038026%26rft_id%253Dinfo%253Apmid%252F27986793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi81NDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.54" data-doi="10.1111/j.1365-294X.2005.02737.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Østbye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naesje</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himberg</span>  <span class="cit-name-given-names">K.-J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindar</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionary history of the European whitefish <em>Coregonus lavaretus (L.)</em> species complex as inferred from mtDNA phylogeography and gill-raker numbers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4371</span>–<span class="cit-lpage">4387</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02737.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOstbye%26rft.auinit1%253DK.%26rft.volume%253D14%26rft.issue%253D14%26rft.spage%253D4371%26rft.epage%253D4387%26rft.atitle%253DEvolutionary%2Bhistory%2Bof%2Bthe%2BEuropean%2Bwhitefish%2BCoregonus%2Blavaretus%2B%2528L.%2529%2Bspecies%2Bcomplex%2Bas%2Binferred%2Bfrom%2BmtDNA%2Bphylogeography%2Band%2Bgill-raker%2Bnumbers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02737.x%26rft_id%253Dinfo%253Apmid%252F16313599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02737.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16313599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.55" data-doi="10.1111/j.1469-185X.2000.tb00057.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rab</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-185X.2000.tb00057.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D76%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2000.tb00057.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2000.tb00057.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.56" data-doi="10.1101/201178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruano-Rubio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Scaling accurate genetic variant discovery to tens of thousands of samples.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/201178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F201178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyMDExNzh2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzc0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-web" id="cit-8.12.3745.57"><div class="cit-metadata unstructured">R Core Team, 2014 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.58" data-doi="10.1093/bioinformatics/btx494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Lep-MAP3: robust linkage mapping even for low-coverage whole genome sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3726</span>–<span class="cit-lpage">3732</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3726%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.59" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of Ultradense Linkage Maps with Lep-MAP2: Stickleback F2 Recombinant Crosses as an Example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.60" data-doi="10.1093/bioinformatics/btt563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvinen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lep-MAP: fast and accurate linkage map construction for large SNP datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3128</span>–<span class="cit-lpage">3134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt563</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt563%26rft_id%253Dinfo%253Apmid%252F24078685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24078685&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327714700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.61" data-doi="10.1186/s13059-017-1241-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundappa</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grammes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hvidsten</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redmond</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1241-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1241-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1241-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.62" data-doi="10.1093/molbev/msm066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetic architecture of ecological speciation and the association with signatures of selection in natural lake whitefish (<em>Coregonus sp</em>. Salmonidae) species pairs.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm066%26rft_id%253Dinfo%253Apmid%252F17404398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17404398&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247207700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.63" data-doi="10.1111/j.1365-294X.2004.02396.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">FAST-TRACK: Integrating QTL mapping and genome scans towards the characterization of candidate loci under parallel selection in the lake whitefish (<em>Coregonus clupeaformis</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2004.02396.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D14%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02396.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02396.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.64" data-doi="10.1023/A:1013773600304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Combining the analyses of introgressive hybridisation and linkage mapping to investigate the genetic architecture of population divergence in the lake whitefish (<em>Coregonus clupeaformis</em>, Mitchill).</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1013773600304</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DRogers%26rft.auinit1%253DS.%2BM.%26rft.volume%253D111%26rft.issue%253D1-3%26rft.spage%253D25%26rft.epage%253D41%26rft.atitle%253DCombining%2Bthe%2Banalyses%2Bof%2Bintrogressive%2Bhybridisation%2Band%2Blinkage%2Bmapping%2Bto%2Binvestigate%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bpopulation%2Bdivergence%2Bin%2Bthe%2Blake%2Bwhitefish%2B%2528Coregonus%2Bclupeaformis%252C%2BMitchill%2529.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013773600304%26rft_id%253Dinfo%253Apmid%252F11841170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013773600304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11841170&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173270000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A Microsatellite Linkage Map of Rainbow Trout (<em>Oncorhynchus mykiss)</em> characterized by large sex-specific differences in recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSakamoto%26rft.auinit1%253DT.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1331%26rft.epage%253D1345%26rft.atitle%253DA%2BMicrosatellite%2BLinkage%2BMap%2Bof%2BRainbow%2BTrout%2B%2528Oncorhynchus%2Bmykiss%2529%2BCharacterized%2Bby%2BLarge%2BSex-Specific%2BDifferences%2Bin%2BRecombination%2BRates%26rft_id%253Dinfo%253Apmid%252F10880492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzEzMzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sardell</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dagilis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitano</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-comment">Sex Differences in Recombination in Sticklebacks. G3 (Bethesda). g3.200166.2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.67" data-doi="10.1016/j.cbd.2010.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative genomics in teleost species: Knowledge transfer by linking the genomes of model and non-model fish species.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part D Genomics Proteomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cbd.2010.09.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BBiochem%2BPhysiol%2BPart%2BD%2BGenomics%2BProteomics%26rft.aulast%253DSarropoulou%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D92%26rft.epage%253D102%26rft.atitle%253DComparative%2Bgenomics%2Bin%2Bteleost%2Bspecies%253A%2BKnowledge%2Btransfer%2Bby%2Blinking%2Bthe%2Bgenomes%2Bof%2Bmodel%2Band%2Bnon-model%2Bfish%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbd.2010.09.003%26rft_id%253Dinfo%253Apmid%252F20961822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbd.2010.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20961822&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.68" data-doi="10.1038/hdy.2011.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukowski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination rate variation in closely related species.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">508</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.44</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSmukowski%26rft.auinit1%253DC.%2BS.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D496%26rft.epage%253D508%26rft.atitle%253DRecombination%2Brate%2Bvariation%2Bin%2Bclosely%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.44%26rft_id%253Dinfo%253Apmid%252F21673743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673743&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297214900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.69" data-doi="10.1098/rstb.2016.0455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smadja</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in recombination frequency and distribution across eukaryotes: patterns and processes.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2016.0455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0455%26rft_id%253Dinfo%253Apmid%252F29109219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvMTczNi8yMDE2MDQ1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzc0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Monographie der schweizerischen Koregonen.</span> <abbr class="cit-jnl-abbrev">Schweizerische Zeitschrift für Hydrol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSchweizerische%2BZeitschrift%2Bf%25FCr%2BHydrol.%26rft.volume%253D12%26rft.spage%253D340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.71" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span> <span class="cit-article-title">The linear arrangement of six sex-linked factors in <em>Drosophila</em>, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1400140104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.72" data-doi="10.1186/s12864-017-3830-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Construction of a high-density linkage map and mapping of sex determination and growth-related loci in the mandarin fish (<em>Siniperca chuatsi</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3830-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D446%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3830-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3830-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.73" data-doi="10.1093/gbe/evw262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">evw262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varadharajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillard</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tørresen</span>  <span class="cit-name-given-names">O. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulugeta</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> The grayling genome reveals selection on gene expression regulation after whole genome duplication. Genome Biology and Evolution evy201<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy201</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.75" data-doi="10.1038/nature10824"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vonlanthen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bittner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Eutrophication causes speciation reversal in whitefish adaptive radiations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">362</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10824</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVonlanthen%26rft.auinit1%253DP.%26rft.volume%253D482%26rft.issue%253D7385%26rft.spage%253D357%26rft.epage%253D362%26rft.atitle%253DEutrophication%2Bcauses%2Bspeciation%2Breversal%2Bin%2Bwhitefish%2Badaptive%2Bradiations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10824%26rft_id%253Dinfo%253Apmid%252F22337055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10824&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22337055&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300287100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.76" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/93.1.77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/93.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12011185&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3745.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175627300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.77" data-doi="10.1038/nrg.2016.133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Making sense of genomic islands of differentiation in light of speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D18%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3745.78" data-doi="10.1186/s12870-017-1127-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhigunov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulianich</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebedeva</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Development of F1 hybrid population and the high-density linkage map for European aspen (<em>Populus tremula L</em>.) using RADseq technology.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-1127-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-1127-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-1127-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3745.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3737" title="Highly Continuous Genome Assembly of Eurasian Perch (Perca fluviatilis) Using Linked-Read Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3757" title="vqtl: An R Package for Mean-Variance QTL Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3745" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JpDeOOENPXmK8dm2n088no8Q67YAeO6zadLvS3DlnDg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3745" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3745" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3745" class="active" data-icon-position="" data-hide-link-title="0">A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491561" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mQb-XyhYdNg2h7j1nZZ-Cx1uf7SL22beJqCLCJezfrs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3745.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3745" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-97AJyOcAXkNUhpBb9x0zAFo933LCI1IIrvdq2r-YM6g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3745" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3745.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491561" id="node491561--2" data-pisa="ggg;8/12/3745" data-pisa-master="ggg;g3.118.200552" data-apath="/ggg/8/12/3745.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rishi De-Kayne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philine G. D. Feulner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3745-3755;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491561--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rishi De-Kayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rishi%2BDe-Kayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De-Kayne%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARishi%2BDe-Kayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philine G. D. Feulner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philine%2BG.%2BD.%2BFeulner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feulner%20PG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhiline%2BG.%2BD.%2BFeulner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philine G. D. Feulner</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:philine.feulner@eawag.ch" class="" data-icon-position="" data-hide-link-title="0">philine.feulner@eawag.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3745" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3745&amp;text=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3745&amp;t=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3745&amp;title=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491561/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491561" id="node491561--3" data-pisa="ggg;8/12/3745" data-pisa-master="ggg;g3.118.200552" data-apath="/ggg/8/12/3745.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rishi De-Kayne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philine G. D. Feulner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3745-3755;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491561--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rishi De-Kayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rishi%2BDe-Kayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De-Kayne%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARishi%2BDe-Kayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philine G. D. Feulner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Fish Ecology and Evolution, Centre of Ecology, Evolution and Biogeochemistry, EAWAG Swiss Federal Institute of Aquatic Science and Technology, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Aquatic Ecology and Evolution, Institute of Ecology and Evolution, University of Bern, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philine%2BG.%2BD.%2BFeulner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feulner%20PG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhiline%2BG.%2BD.%2BFeulner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8078-1788" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philine G. D. Feulner</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:philine.feulner@eawag.ch" class="" data-icon-position="" data-hide-link-title="0">philine.feulner@eawag.ch</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3745&amp;rft.epage=3755&amp;rft.atitle=A%20European%20Whitefish%20Linkage%20Map%20and%20Its%20Implications%20for%20Understanding%20Genome-Wide%20Synteny%20Between%20Salmonids%20Following%20Whole%20Genome%20Duplication&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=De-Kayne%2C+Rishi&amp;rft.au=Feulner%2C+Philine+G.+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491561/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491561/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491561/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491561/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491561/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491561/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491561/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491561/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491561/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491561/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491561/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491561/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3745" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3745.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3745.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3745.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3745.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
