<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3961" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3961.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3961.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3961.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3961/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3961.atom" />
<meta name="HW.pisa" content="ggg;8/12/3961" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" />
<meta name="DC.Identifier" content="10.1534/g3.118.200760" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Rincent et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.25387/g3.7243256" />
<meta name="DC.Description" content="Genomic selection - the prediction of breeding values using DNA polymorphisms - is a disruptive method that has widely been adopted by animal and plant breeders to increase productivity. It was recently shown that other sources of molecular variations such as those resulting from transcripts or metabolites could be used to accurately predict complex traits. These endophenotypes have the advantage of capturing the expressed genotypes and consequently the complex regulatory networks that occur in the different layers between the genome and the phenotype. However, obtaining such omics data at very large scales, such as those typically experienced in breeding, remains challenging. As an alternative, we proposed using near-infrared spectroscopy (NIRS) as a high-throughput, low cost and non-destructive tool to indirectly capture endophenotypic variants and compute relationship matrices for predicting complex traits, and coined this new approach ”phenomic selection” (PS). We tested PS on two species of economic interest ( Triticum aestivum L. and Populus nigra L.) using NIRS on various tissues (grains, leaves, wood). We showed that one could reach predictions as accurate as with molecular markers, for developmental, tolerance and productivity traits, even in environments radically different from the one in which NIRS were collected. Our work constitutes a proof of concept and provides new perspectives for the breeding community, as PS is theoretically applicable to any organism at low cost and does not require any molecular information." />
<meta name="DC.Contributor" content="Renaud Rincent" />
<meta name="DC.Contributor" content="Jean-Paul Charpentier" />
<meta name="DC.Contributor" content="Patricia Faivre-Rampant" />
<meta name="DC.Contributor" content="Etienne Paux" />
<meta name="DC.Contributor" content="Jacques Le Gouis" />
<meta name="DC.Contributor" content="Catherine Bastien" />
<meta name="DC.Contributor" content="Vincent Segura" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection - the prediction of breeding values using DNA polymorphisms - is a disruptive method that has widely been adopted by animal and plant breeders to increase productivity. It was recently shown that other sources of molecular variations such as those resulting from transcripts or metabolites could be used to accurately predict complex traits. These endophenotypes have the advantage of capturing the expressed genotypes and consequently the complex regulatory networks that occur in the different layers between the genome and the phenotype. However, obtaining such omics data at very large scales, such as those typically experienced in breeding, remains challenging. As an alternative, we proposed using near-infrared spectroscopy (NIRS) as a high-throughput, low cost and non-destructive tool to indirectly capture endophenotypic variants and compute relationship matrices for predicting complex traits, and coined this new approach ”phenomic selection” (PS). We tested PS on two species of economic interest (&lt;i&gt;Triticum aestivum&lt;/i&gt; L. and &lt;i&gt;Populus nigra&lt;/i&gt; L.) using NIRS on various tissues (grains, leaves, wood). We showed that one could reach predictions as accurate as with molecular markers, for developmental, tolerance and productivity traits, even in environments radically different from the one in which NIRS were collected. Our work constitutes a proof of concept and provides new perspectives for the breeding community, as PS is theoretically applicable to any organism at low cost and does not require any molecular information.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3961" />
<meta name="citation_id" content="8/12/3961" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3961" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3961.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3961.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3961.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200760" />
<meta name="citation_pmid" content="30373914" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3961" />
<meta name="citation_lastpage" content="3972" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Renaud Rincent" />
<meta name="citation_author_institution" content="GDEC, INRA, UCA, 63000 Clermont-Ferrand, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0885-0969" />
<meta name="citation_author" content="Jean-Paul Charpentier" />
<meta name="citation_author_institution" content="BioForA, INRA, ONF, 45075 Orléans, France" />
<meta name="citation_author_institution" content="GenoBois analytical platform, INRA, 45075 Orléans, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6029-0498" />
<meta name="citation_author" content="Patricia Faivre-Rampant" />
<meta name="citation_author_institution" content="EPGV, INRA, CEA-IG/CNG, 91057 Evry, France" />
<meta name="citation_author" content="Etienne Paux" />
<meta name="citation_author_institution" content="GDEC, INRA, UCA, 63000 Clermont-Ferrand, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3094-7129" />
<meta name="citation_author" content="Jacques Le Gouis" />
<meta name="citation_author_institution" content="GDEC, INRA, UCA, 63000 Clermont-Ferrand, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5726-4902" />
<meta name="citation_author" content="Catherine Bastien" />
<meta name="citation_author_institution" content="BioForA, INRA, ONF, 45075 Orléans, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9391-6637" />
<meta name="citation_author" content="Vincent Segura" />
<meta name="citation_author_institution" content="BioForA, INRA, ONF, 45075 Orléans, France" />
<meta name="citation_author_email" content="vincent.segura@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1860-2256" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=M. Abasolo;citation_author=D. J. Lee;citation_author=C. Raymond;citation_author=R. Meder;citation_author=M. Shepherd;citation_title=Deviant near-infrared spectra identifies Corymbia hybrids.;citation_pages=121-131;citation_volume=304;citation_year=2013;citation_doi=10.1016/j.foreco.2013.04.040" />
<meta name="citation_reference" content="citation_journal_title=J. Near Infrared Spectrosc.;citation_author=O. E. Adedipe;citation_author=B. Dawson-Andoh;citation_author=J. Slahor;citation_author=L. Osborn;citation_title=Classification of red oak (Quercus rubra) and white oak (Quercus alba) wood using a near infrared spectrometer and soft independent modelling of class analogies.;citation_pages=49-57;citation_volume=16;citation_year=2008;citation_doi=10.1255/jnirs.760" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=F. M. Aguate;citation_author=S. Trachsel;citation_author=L. G. Pérez;citation_author=J. Burgueño;citation_author=J. Crossa;citation_title=Use of Hyperspectral Image Data Outperforms Vegetation Indices in Prediction of Maize Yield.;citation_pages=2517;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2017.01.0007." />
<meta name="citation_reference" content="citation_journal_title=J. Sci. Food Agric.;citation_author=D. Bertrand;citation_author=P. Robert;citation_author=W. Loisel;citation_title=Identification of some wheat varieties by near infrared reflectance spectroscopy.;citation_pages=1120-1124;citation_volume=36;citation_year=1985;citation_doi=10.1002/jsfa.2740361114" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=L. Cabrera-Bosquet;citation_author=J. Crossa;citation_author=J. von Zitzewitz;citation_author=M. D. Serret;citation_author=J. Luis Araus;citation_title=High-throughput phenotyping and genomic selection: the frontiers of crop breeding converge.;citation_pages=312-320;citation_volume=54;citation_year=2012;citation_issue=5;citation_pmid=22420640;citation_doi=10.1111/j.1744-7909.2012.01116.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Covarrubias-Pazaran;citation_title=Genome-Assisted Prediction of Quantitative Traits Using the R Package sommer.;citation_pages=e0156744;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156744" />
<meta name="citation_reference" content="citation_journal_title=G3: Genes|Genomes|Genetics (Bethesda);citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_title=Genomic Prediction of Gene Bank Wheat Landraces;citation_volume=6;citation_year=2016;citation_pmid=27172218;citation_doi=10.1534/g3.116.029637" />
<meta name="citation_reference" content="citation_journal_title=Zootaxa;citation_author=R. H. Cruickshank;citation_author=L. Munck;citation_title=It’s barcoding Jim, but not as we know it.;citation_pages=55-56;citation_volume=56;citation_year=2011;citation_doi=10.1073/pnas.1232347100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting Quantitative Traits With Regression Models for Dense Molecular Markers and Pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Comput. Electron. Agric.;citation_author=M. P. Diago;citation_author=A. M. Fernandes;citation_author=B. Millan;citation_author=J. Tardaguila;citation_author=P. Melo-Pinto;citation_title=Identification of grapevine varieties using leaf spectroscopy and partial least squares.;citation_pages=7-13;citation_volume=99;citation_year=2013;citation_doi=10.1016/j.compag.2013.08.021" />
<meta name="citation_reference" content="citation_journal_title=J. Cereal Sci.;citation_author=D. Diepeveen;citation_author=G. P. Y. Clarke;citation_author=K. Ryan;citation_author=A. Tarr;citation_author=W. Ma;citation_title=Molecular genetic mapping of NIR spectra variation.;citation_pages=6-14;citation_volume=55;citation_year=2012;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Tree Physiol.;citation_author=S. Y. Dillen;citation_author=N. Marron;citation_author=M. Sabatti;citation_author=R. Ceulemans;citation_author=C. Bastien;citation_title=Relationships among productivity determinants in two hybrid poplar families grown during three years at two contrasting sites.;citation_volume=29;citation_year=2009;citation_pmid=19483184;citation_doi=10.1093/treephys/tpp036" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=M. Ecarnot;citation_author=F. Compan;citation_author=P. Roumet;citation_title=Assessing leaf nitrogen content and leaf mass per unit area of wheat in the field throughout plant cycle with a portable spectrometer.;citation_pages=44-50;citation_volume=140;citation_year=2013;citation_doi=10.1016/j.fcr.2012.10.013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=J. Near Infrared Spectrosc.;citation_author=J. A. Espinoza;citation_author=G. R. Hodge;citation_author=W. S. Dvorak;citation_title=The potential use of near infrared spectroscopy to discriminate between different pine species and their hybrids.;citation_pages=437-447;citation_volume=20;citation_year=2012;citation_doi=10.1255/jnirs.1006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=P. Faivre-Rampant;citation_author=G. Zaina;citation_author=V. Jorge;citation_author=S. Giacomello;citation_author=V. Segura;citation_title=New resources for genetic studies in Populus nigra: genome-wide SNP discovery and development of a 12k Infinium array.;citation_pages=1023-1036;citation_volume=16;citation_year=2016;citation_pmid=26929265;citation_doi=10.1111/1755-0998.12513" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Feher;citation_author=J. Lisec;citation_author=L. Römisch-Margl;citation_author=J. Selbig;citation_author=A. Gierl;citation_title=Deducing Hybrid Performance from Parental Metabolic Profiles of Young Primary Roots of Maize by Using a Multivariate Diallel Approach.;citation_pages=e85435;citation_volume=9;citation_year=2014;citation_pmid=24409329;citation_doi=10.1371/journal.pone.0085435" />
<meta name="citation_reference" content="citation_journal_title=Metabolomics;citation_author=O. Fernandez;citation_author=M. Urrutia;citation_author=S. Bernillon;citation_author=C. Giauffret;citation_author=F. Tardieu;citation_title=Fortune telling: metabolic markers of plant performance.;citation_pages=158;citation_volume=12;citation_year=2016;citation_doi=10.1007/s11306-016-1099-1" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. Ferrio;citation_author=D. Villegas;citation_author=J. Zarco;citation_author=N. Aparicio;citation_author=J. Araus;citation_title=Assessment of durum wheat yield using visible and near-infrared reflectance spectra of canopies.;citation_pages=126-148;citation_volume=94;citation_year=2005;citation_doi=10.1016/j.fcr.2004.12.002" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=S. Fischnaller;citation_author=F. E. Dowell;citation_author=A. Lusser;citation_author=B. C. Schlick-Steiner;citation_author=F. M. Steiner;citation_title=Non-destructive species identification of Drosophila obscura and D. subobscura (Diptera) using near-infrared spectroscopy.;citation_pages=284-289;citation_volume=6;citation_year=2012;citation_pmid=22885252;citation_doi=10.4161/fly.21535" />
<meta name="citation_reference" content="citation_journal_title=Oecologia;citation_author=W. J. Foley;citation_author=A. McIlwee;citation_author=I. Lawler;citation_author=L. Aragones;citation_author=A. P. Woolnough;citation_title=Ecological applications of near infrared reflectance spectroscopy - a tool for rapid, cost-effective prediction of the composition of plant and animal tissues and aspects of animal performance.;citation_pages=293-305;citation_volume=116;citation_year=1998;citation_doi=10.1007/s004420050591" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Fu;citation_author=K. C. Falke;citation_author=A. Thiemann;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Partial least squares regression, support vector machine regression, and transcriptome-based distances for prediction of maize hybrid performance with gene expression data.;citation_pages=825-833;citation_volume=124;citation_year=2012;citation_issue=5;citation_pmid=22101908;citation_doi=10.1007/s00122-011-1747-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. C. Gaynor;citation_author=G. Gorjanc;citation_author=A. R. Bentley;citation_author=E. S. Ober;citation_author=P. Howell;citation_title=A Two-Part Strategy for Using Genomic Selection to Develop Inbred Lines.;citation_pages=2372;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.09.0742" />
<meta name="citation_reference" content="citation_journal_title=Ind. Crops Prod.;citation_author=M. N. Gebreselassie;citation_author=K. Ader;citation_author=N. Boizot;citation_author=F. Millier;citation_author=J.-P. P. Charpentier;citation_title=Near-infrared spectroscopy enables the genetic analysis of chemical properties in a large set of wood samples from Populus nigra (L.) natural populations.;citation_pages=159-171;citation_volume=107;citation_year=2017;citation_doi=10.1093/nar/gks989" />
<meta name="citation_reference" content="Genz, A., F. Bretz, T. Miwa, X. Mi, F. Leisch, et al., 2018 mvtnorm: Multivariate Normal and t Distributions. R package version 1.0–8." />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Gorjanc;citation_author=M. Battagin;citation_author=J.-F. Dumasy;citation_author=R. Antolin;citation_author=R. C. Gaynor;citation_title=Prospects for Cost-Effective Genomic Selection via Accurate Within-Family Imputation.;citation_pages=216;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.06.0526" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Gorjanc;citation_author=J.-F. Dumasy;citation_author=S. Gonen;citation_author=R. C. Gaynor;citation_author=R. Antolin;citation_title=Potential of Low-Coverage Genotyping-by-Sequencing and Imputation for Cost-Effective Genomic Selection in Biparental Segregating Populations.;citation_pages=1404;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.08.0675" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Gorjanc;citation_author=J. Jenko;citation_author=S. J. Hearne;citation_author=J. M. Hickey;citation_title=Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.;citation_pages=30;citation_volume=17;citation_year=2016;citation_pmid=26732811;citation_doi=10.1186/s12864-015-2345-z" />
<meta name="citation_reference" content="citation_journal_title=Tree Physiol.;citation_author=J. Guet;citation_author=F. Fabbrini;citation_author=R. Fichot;citation_author=M. Sabatti;citation_author=C. Bastien;citation_title=Genetic variation for leaf morphology, leaf structure and leaf carbon isotope discrimination in European populations of black poplar (Populus nigra L.).;citation_volume=35;citation_year=2015;citation_pmid=26224105;citation_doi=10.1093/treephys/tpv056" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. Guo;citation_author=M. M. Magwire;citation_author=C. J. Basten;citation_author=Z. Xu;citation_author=D. Wang;citation_title=Evaluation of the utility of gene expression and metabolic information for genomic prediction in maize.;citation_pages=2413-2427;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2780-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Hayes;citation_author=P. M. Visscher;citation_author=M. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant Breeding with Genomic Selection: Gain per Unit Time and Cost.;citation_pages=1681;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=J. Near Infrared Spectrosc.;citation_author=P. R. G. Hein;citation_author=G. Chaix;citation_title=NIR Spectral Heritability: A Promising Tool for Wood Breeders?;citation_pages=141-147;citation_volume=22;citation_year=2014;citation_doi=10.1255/jnirs.1108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjálmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=C. Lang;citation_author=D. R. Almeida;citation_author=F. R. Costa;citation_title=Discrimination of taxonomic identity at species, genus and family levels using Fourier Transformed Near-Infrared Spectroscopy (FT-NIR).;citation_pages=219-227;citation_volume=406;citation_year=2017;citation_doi=10.1016/j.foreco.2017.09.003" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=D. Ly;citation_author=S. Huet;citation_author=A. Gauffreteau;citation_author=R. Rincent;citation_author=G. Touzy;citation_title=Whole-genome prediction of reaction norms to environmental stress in bread wheat (Triticum aestivum L.) by genomic random regression.;citation_pages=32-41;citation_volume=216;citation_year=2018;citation_doi=10.1016/j.fcr.2017.08.020" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. F. C. Mackay;citation_author=E. a. Stone;citation_author=J. F. Ayroles;citation_title=The genetics of quantitative traits: challenges and prospects.;citation_pages=565-577;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19584810;citation_doi=10.1038/nrg2612" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. Madec;citation_author=F. Baret;citation_author=B. de Solan;citation_author=S. Thomas;citation_author=D. Dutartre;citation_title=High-Throughput Phenotyping of Plant Height: Comparing Unmanned Aerial Vehicles and Ground LiDAR Estimates.;citation_pages=2002;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.02002" />
<meta name="citation_reference" content="citation_journal_title=J. Near Infrared Spectrosc.;citation_author=R. Meder;citation_author=D. Kain;citation_author=N. Ebdon;citation_author=P. Macdonell;citation_author=J. T. Brawner;citation_title=Identifying hybridisation in Pinus species using near infrared spectroscopy of foliage.;citation_pages=337-345;citation_volume=22;citation_year=2014;citation_doi=10.1534/genetics.114.172106" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Accelerating Improvement of Livestock with Genomic Selection.;citation_pages=221-237;citation_volume=1;citation_year=2013;citation_pmid=25387018;citation_doi=10.1146/annurev-animal-031412-103705" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="Muñoz, F. and L. Sanchez, 2017 breedR: Statistical Methods for Forest Genetic Resources Analysts. R package version 0.12–2." />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Oakey;citation_author=A. P. Verbyla;citation_author=B. R. Cullis;citation_author=X. Wei;citation_author=W. S. Pitchford;citation_title=Joint modeling of additive and non-additive (genetic line) effects in multi-environment trials.;citation_pages=1319-1332;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17426958;citation_doi=10.1007/s00122-007-0515-3" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. M. O’Reilly-Wapstra;citation_author=J. S. Freeman;citation_author=R. Barbour;citation_author=R. E. Vaillancourt;citation_author=B. M. Potts;citation_title=Genetic analysis of the near-infrared spectral phenome of a global Eucalyptus species.;citation_pages=943-959;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0607-y" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian Lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=ISPRS J. Photogramm. Remote Sens.;citation_author=K. Y. Peerbhay;citation_author=O. Mutanga;citation_author=R. Ismail;citation_title=Commercial tree species discrimination using airborne AISA Eagle hyperspectral imagery and partial least squares discriminant analysis (PLS-DA) in KwaZulu-Natal, South Africa.;citation_pages=19-28;citation_volume=79;citation_year=2013;citation_doi=10.1016/j.isprsjprs.2013.01.013" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H. Posada;citation_author=M. Ferrand;citation_author=F. Davrieux;citation_author=P. Lashermes;citation_author=B. Bertrand;citation_title=Stability across environments of the coffee variety near infrared spectral signature.;citation_pages=113-119;citation_volume=102;citation_year=2009;citation_issue=2;citation_pmid=18971953;citation_doi=10.1038/hdy.2008.88" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Rimbert;citation_author=B. Darrier;citation_author=J. Navarro;citation_author=J. Kitt;citation_author=F. Choulet;citation_title=High throughput SNP discovery and genotyping in hexaploid wheat.;citation_pages=e0186329;citation_volume=13;citation_year=2018;citation_pmid=29293495;citation_doi=10.1371/journal.pone.0186329" />
<meta name="citation_reference" content="citation_journal_title=Spat. Stat.;citation_author=M. X. Rodríguez-Álvarez;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_author=P. H. Eilers;citation_title=Correcting for spatial heterogeneity in plant breeding experiments with p-splines.;citation_pages=52-71;citation_volume=23;citation_year=2017;citation_doi=10.1016/j.spasta.2017.10.003" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. Rohde;citation_author=V. Storme;citation_author=V. Jorge;citation_author=M. Gaudet;citation_author=N. Vitacolonna;citation_title=Bud set in poplar--genetic dissection of a complex trait in natural and hybrid populations.;citation_pages=106-121;citation_volume=189;citation_year=2011;citation_issue=1;citation_pmid=21039557;citation_doi=10.1111/j.1469-8137.2010.03469.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Sargolzaei;citation_author=J. P. Chesnais;citation_author=F. S. Schenkel;citation_title=A new approach for efficient genotype imputation using information from relatives.;citation_pages=478;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-478" />
<meta name="citation_reference" content="citation_journal_title=Anal. Chem.;citation_author=A. Savitzky;citation_author=M. J. E. Golay;citation_title=Smoothing and Differentiation of Data by Simplified Least Squares Procedures.;citation_pages=1627-1639;citation_volume=36;citation_year=1964;citation_doi=10.1021/ac60214a047" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. A. Schrag;citation_author=M. Westhues;citation_author=W. Schipprack;citation_author=F. Seifert;citation_author=A. Thiemann;citation_title=Beyond Genomic Prediction: Combining Different Types ofomicsData Can Improve Prediction of Hybrid Performance in Maize;citation_volume=208;citation_year=2018;citation_pmid=29363551;citation_doi=10.1534/genetics.117.300374" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=F. Seifert;citation_author=A. Thiemann;citation_author=T. A. Schrag;citation_author=D. Rybka;citation_author=A. E. Melchinger;citation_title=Small RNA-based prediction of hybrid performance in maize.;citation_pages=371;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4708-8" />
<meta name="citation_reference" content="signal developers, 2014 signal: Signal processing." />
<meta name="citation_reference" content="citation_journal_title=Applied Spectroscopy;citation_journal_abbrev=Appl Spectrosc;citation_author=C. A. Teixeira Dos Santos;citation_author=M. Lopo;citation_author=R. N. M. J. Páscoa;citation_author=J. A. Lopes;citation_title=A Review on the Applications of Portable Near-Infrared Spectrometers in the Agro-Food Industry;citation_pages=1215-1233;citation_volume=67;citation_year=2013;citation_issue=11;citation_pmid=24160873;citation_doi=10.1366/13-07228" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. Ward;citation_author=M. Rakszegi;citation_author=Z. Bedő;citation_author=P. R. Shewry;citation_author=I. Mackay;citation_title=Differentially penalized regression to predict agronomic traits from metabolites and markers in wheat.;citation_pages=19;citation_volume=16;citation_year=2015;citation_pmid=25879431;citation_doi=10.1186/s12863-015-0169-0" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=V. Weber;citation_author=J. Araus;citation_author=J. Cairns;citation_author=C. Sanchez;citation_author=A. Melchinger;citation_title=Prediction of grain yield using reflectance spectra of canopy and leaves in maize plants grown under different water regimes.;citation_pages=82-90;citation_volume=128;citation_year=2012;citation_doi=10.1016/j.fcr.2011.12.016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Westhues;citation_author=T. A. Schrag;citation_author=C. Heuer;citation_author=G. Thaller;citation_author=H. F. Utz;citation_title=Omics-based hybrid prediction in maize.;citation_pages=1927-1939;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2934-0" />
<meta name="citation_reference" content="citation_journal_title=Genetical;citation_author=J. Whittaker;citation_title=Marker-assisted selection using ridge regression.;citation_pages=351-367;citation_volume=75;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=S. Xu;citation_author=Y. Xu;citation_author=L. Gong;citation_author=Q. Zhang;citation_title=Metabolomic prediction of yield in hybrid rice.;citation_pages=219-227;citation_volume=88;citation_year=2016;citation_doi=10.1111/tpj.13242" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Yamada;citation_author=Y. Itoh;citation_author=I. Sugimoto;citation_title=Parametric relationships between genotype x environment interaction and genetic correlation when two environments are involved.;citation_pages=850-854;citation_volume=76;citation_year=1988;citation_doi=10.1007/BF00273671" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=C. Zenke-Philippi;citation_author=M. Frisch;citation_author=A. Thiemann;citation_author=F. Seifert;citation_author=T. Schrag;citation_title=Transcriptome-based prediction of hybrid performance with unbalanced data from a maize breeding programme.;citation_pages=331-337;citation_volume=136;citation_year=2017;citation_doi=10.1111/pbr.12482" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. M. Dekkers;citation_author=R. L. Fernando;citation_author=J.-L. Jannink;citation_title=Factors Affecting Accuracy From Genomic Selection in Populations Derived From Multiple Inbred Lines: A Barley Case Study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection - the prediction of breeding values using DNA polymorphisms - is a disruptive method that has widely been adopted by animal and plant breeders to increase productivity. It was recently shown that other sources of molecular variations such as those resulting from transcripts or metabolites could be used to accurately predict complex traits. These endophenotypes have the advantage of capturing the expressed genotypes and consequently the complex regulatory networks that occur in the different layers between the genome and the phenotype. However, obtaining such omics data at very large scales, such as those typically experienced in breeding, remains challenging. As an alternative, we proposed using near-infrared spectroscopy (NIRS) as a high-throughput, low cost and non-destructive tool to indirectly capture endophenotypic variants and compute relationship matrices for predicting complex traits, and coined this new approach ”phenomic selection” (PS). We tested PS on two species of economic interest ( Triticum aestivum L. and Populus nigra L.) using NIRS on various tissues (grains, leaves, wood). We showed that one could reach predictions as accurate as with molecular markers, for developmental, tolerance and productivity traits, even in environments radically different from the one in which NIRS were collected. Our work constitutes a proof of concept and provides new perspectives for the breeding community, as PS is theoretically applicable to any organism at low cost and does not require any molecular information." />
<meta name="og-title" property="og:title" content="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3961" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection - the prediction of breeding values using DNA polymorphisms - is a disruptive method that has widely been adopted by animal and plant breeders to increase productivity. It was recently shown that other sources of molecular variations such as those resulting from transcripts or metabolites could be used to accurately predict complex traits. These endophenotypes have the advantage of capturing the expressed genotypes and consequently the complex regulatory networks that occur in the different layers between the genome and the phenotype. However, obtaining such omics data at very large scales, such as those typically experienced in breeding, remains challenging. As an alternative, we proposed using near-infrared spectroscopy (NIRS) as a high-throughput, low cost and non-destructive tool to indirectly capture endophenotypic variants and compute relationship matrices for predicting complex traits, and coined this new approach ”phenomic selection” (PS). We tested PS on two species of economic interest ( Triticum aestivum L. and Populus nigra L.) using NIRS on various tissues (grains, leaves, wood). We showed that one could reach predictions as accurate as with molecular markers, for developmental, tolerance and productivity traits, even in environments radically different from the one in which NIRS were collected. Our work constitutes a proof of concept and provides new perspectives for the breeding community, as PS is theoretically applicable to any organism at low cost and does not require any molecular information." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__49R9wsCpiKJS6e6ZOiXRsC0989bO7FF1XdNTikSRKGM__k1m2vpL1Nu9n43bJnV24EQ0Z_UWo-Iq0zZ2FkQLL-iU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"G0-wTJGfKAprhCgXw0uFRBmwTjphHItoJ1TUNMjTWUY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491721","apath":"\/ggg\/8\/12\/3961.atom","pisa":"ggg;8\/12\/3961","ac":{"\/ggg\/8\/12\/3961.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3961.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3961"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3961"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3961":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mRJ78XdxoYPsPGVm\\\\\\\/LCU114VKhyMkz381nEvW\\\\\\\/QViGlffxVZhLGqqsid78b21ptu7w\\\\\\\/jhJDY4t5eUHwtJYIYyOu2N1+FyKyJnflibTD14mX6N7OGfkJ80YngBVKxepluf9BfDBYj5V6h2xGnYy98zu8hpQupi7zVNAwc8046yHaTe02X2bJbdwxpEZIvpe8npFHqqS5SG5dhPloWx2DyhutDgeFm7zCmrzEl6zERFuYqASpwaCkTG9BJUTUHE1XgSZ+2qLXUowE7ULoP0Lm1re4ljWV7\\\\\\\/xqa370pPz7sFpmBD1087z45D4K48\\\\\\\/TtUskmunOOOQ0JiGnzWuoAosgILkTpyJCJJ5TopkOoVEtpuMI8agU1Pwb8V3CMA7oyi74rXN9cayjn6b7VXKV9kWNA2gM04r80HmQmQgulJB04+nnlSi4tmsH8yVU+PlkP13zYL6wRFsKiLrKSmDNH+mAS\\\\\\\/hBiFHfjNfVDggCMW2ZhUdcb3yuCJnKguzAE1fl45mYSKZqqM9K9cAi0uHezE2WIGiK0hMmog5WYWEmtwzimvwfghPlnaJXTj0Gkkfbq5Y\\\\\\\/1J\\\\\\\/mF58VkCHAlQugsRKOdaFuC\\\\\\\/35Re3I2BO2HxN7jL515zteCtADAarinN1e532m0ZsvRaY9cya9HXUQep0kZH1UDwUUsbqpdWQvRrCX5bq+RqTuguPEgeQbvHbCt6TekoDv+MTvjw41SJFmG1AKtQ2SM\\\\\\\/7dG5fwLIvHVJKMn+fvA+hWZsCJRaxL0jxDsbnvolvmKK86kKy+L563o4tj2uE6e31FrqSfIAsdUNjxr7Z\\\\\\\/BdrLo7l9TvBEBfiFeRLPextKAomMwUKkLR1c2HVxxSMierJzzakF9mRAPKTUbfh0JAbHn8fWZYKY5czyDEmFnxueGzLB0yycdw0dE2uOSbJXzblNfVOpDnxl+0IEqeyYTa0SjUsY5nvXe8Xkal\\\\\\\/W9siYxaCqbdS2j1ai3EaxL0t6I1hYfBOUQFU0SbbbpYnieS265v\\\\\\\/CyRv1bq+GUVsYQU3HoHqjKzLzpPyXfCYMYePx5EsF4PTnuILQhNg1cSY1o7GZKqwAym9CJJZOXH3IrQ\\\\\\\/1cUrRa0yaYXfFIMKi\\\\\\\/Ih1qgXlj7SyNoW+hJ\\\\\\\/bCGxu\\\\\\\/83ivGW66ELbDVA0zcjB20YSm8q2q2NkY\\\\\\\/aXy5rAGwtkKwjA5dCUnX1z8BY++330+uel8\\\\\\\/1zme\\\\\\\/F\\\\\\\/hrj8RTKDdCEjIceOdvCV0Ehwww2vqVbCqAYZ1L5ri52O49xxi2kCpwkBnhno4e\\\\\\\/U4eXEq6E3\\\u0022,\\\u0022iv\\\u0022:\\\u0022v8IM2+mBzAgZz3fvLON53Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222872cb4ca8312c9256cb99b69921172382a092675f42084fd60e63f45bfe93e6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PGLQuYclajF6XwZuAfyEuTwVyQJqaUpWUulr6Pg1sZzLjZpewBlJVJxqWEW1tubjiR1M\\\\\\\/WZfkmldtAF27b4S4tDLevHx8gmNZORfc9Nrbx3MSuKCJGdPfQDViIxAY2DlyT3XVlbF711M\\\\\\\/2908pDUQufRyaYhyRbqT1VSHXMOLjPN0bSKTRtWCIB8lWY6fBcw1BqUt0lY2UMRSOu7JIzkztzQtT0nhLhRscMtE\\\\\\\/l5RpGAo3\\\\\\\/+E4jOWDow0KjTrsHbsiA4C+wrjPYYEC3u2TQm3Xm0E02tMKDUHtlIlLlU1Rn\\\\\\\/QkKyvI4uSqBMmvWAnRQcRen5GixNTDHzhV\\\\\\\/AnldyVIuzZ6lCVW05tGSp8y1Q3PGgroDl82bYaJkDW2q0THx\\\\\\\/Wp58bU7T5o6Qi9zbuF4a7ABNBRtr5Lab3Cqw0fxQ7gmkKS4SddcufijEXAg+EiIL9WRp2WOys38zm5uLFHkOOchpUr+CEYPPiY9vQvrUeIzyMDGo20osj3xnBZz2SDWCP0gp5r2TyOXXDLaeVT+H0yP48utLp\\\\\\\/Nj\\\\\\\/M4jsBVbFdLRlOMfpFK2tAz8URj9rYTNVzX7mDXbwIS38Dp4QVYcUH9ug00mKWiygwAdLHa1Qrep5G3tvWipyzhdQCM8KsQksvfP8GLXIRC22A8pJkYEw512jBYuM7C3Yh\\\\\\\/ccydizDJFpBoOLhIpwSdL17VGC1CGIIMTLqrXUjQV6ZVtaJ9LwFvW6mjYTtmLZ0WcRVhuRRB0vKzqPY9w7cHM\\\\\\\/D9c30JRt6ecJ2f44caAxFwcqK04\\\\\\\/6kJK7nCEH\\\\\\\/g3x2gs0b+Kiv8xLUCOsofQOEW0btQOXNopPya9cDveFmWN0YtQiUrlA4POzLL3W7qR9pqqSDMm9XQ7\\\\\\\/jtB2Mjw7Z\\\\\\\/JCF4uXnA3xEIjOgb2P9E8sMns1LT5pNzTZYlOe4\\\\\\\/I2HRN4\\\\\\\/4DgG4vd28kFNlAoZNso9yodfS0xBqTbDUsh+Vc\\\\\\\/7Wy9p7eMgwAACVu8NZYGeQNA9vPb+laBHiYL4K5PFThJEI4wurie7N3uPrGsdPmCqgNeHi7YnMeTE\\\\\\\/veFUMM7EzgHmTCmXXHAC03WRR4xSOSnDxZCzZSlW0Pj09ccOGz9fXdN0zaoV\\\\\\\/TdKpcsj\\\\\\\/ctQTRvDbm8rGRU88pL5XGVzlyg3kaGIkyimiaDC8I4\\\\\\\/7mpuZoLICYDDwMQhwvFfSrF17gwXxRElwKBqlYeRlFZcy3bBGZLeLFDB2GdBy3gQqJ6j09Hsly\\\\\\\/CfaEy030k9M+Utdr6L5\\\\\\\/iambRK2ji0vsMEFkOa8wrSsPiCyqo8CzUho3aqLfcavbucDJZnziOr0vCMjnvxwpIooSkS378MpOlKZ5SYFz0QvTM\\\\\\\/S4VCe9RUNHOWVZNypBz5O5qkrD7jDd6LzngH\\\\\\\/GKWviZ+\\\\\\\/pDZijhLBI2r10U4tc8OXTQ\\\\\\\/6Mt2ydohnvPUEORJ4ndODNkC8Hy6kE0Vi6pVvtBrqXXZpaW2ZyaHatvs+EvVcQ2cwEjvrf2bD2HRcNcjOHZ\\\\\\\/OBDII62+Px+snmmlOIWQNnkxUGJ9lYLZdPPjfGa6WyY0es0GPu5Q6Oagnadj6qeA4GxoQsFEeYnlSQ1sfTmgYkszH9zh+gOTJtqeMgEqBiZWQZOXN+ItwDSi8t8xRxjTBQKoPiq99I\\\\\\\/LRwgWDSllftmGZgE9VtSg2YFMDEkOiSiS\\\\\\\/vyS\\\\\\\/Xk8OVN4\\\\\\\/y1krESGfrpAGwSa8n2fEGzwuQpghUkM5wViBEOQ8GNUmU9\\\\\\\/aG2IIufYj7Dm2FSPn65+QnO0Wt9YunCll1IFOIU24hChlc2Fa1Ji3RhotUDRNXO\\\\\\\/HDmDDjTgYVPma7Qy2pY3kInCkPS6uYbfL\\\\\\\/2SOtn6IB821QDGCtv\\\\\\\/+BPqIOYibcoO4zwm\\\\\\\/paYmo1QB\\\\\\\/tMX2owm16pDbJCA3Dks+5UBAY3CjD+hNVf+LxMmCWpdOs5JaOYQ1OEmHiL3qj3hTTUco2gnCHlZjFhrpZrPzKILSSGdvG40FO07sGQfbLOFtgiBB8bu25jiWU1Zkt+q1iOSzTff4gO8va\\\\\\\/2Chy48r3tvHsJtP5AmsScjcw5bk4K3ZAvb+XWZ\\\\\\\/TMuaNT0L5KAZuVown3HJKPtcffe02epUQqbmdsLk4bRx5owqhox0D\\\\\\\/jCCVvgU4OAOLaCwI9fJf7nWUABqipjJy7T3qevnBQa79AC93ZATDdGzBE3pgPIqYtRgb6WSmgXNXpunCCfr1C7hAQATh+EnSCD6CfqYI+UxUPxuFaz6YHzviKntFwP0JOGVUUOpHmHVhmV4DUdo1qdCVfiKOwNasPpynSkjZrnyuBudKd0AZKSiqGiK0W2c+0rEVDlITpyZaIEqBvrO9\\\\\\\/H9nKHt8Wt+774fG8I5AjBN5\\\\\\\/xSTnT3wj6ie41s4PAEqJtMWt26JlDvsSTqBCLux579plZvw+4Xj85UeTOKWPeLeJr4nOWKsiwJzO8YPTkSrXLkdSuf7yFGMBVBcdWI9ilj0z4Zo0p60Sv+t\\\\\\\/fQsSl4pYaX4anD4wHD2NtA2qmxGmkh\\\\\\\/Bdo20uA26hboCJQt2EWrhPw\\\\\\\/F64j4T8A4jHb77PJU3SNMyZoplyqJFWnjYZ98kLJpPoSXloDcKfmg0FrLXSAlLOSaovxDe78Jy82ShilT5FpZ\\\\\\\/ThZnPMMEI45Obj7uV77JZXKr11BTjfto4nIDW9qVvGOnjnAbj4eMS+BW8ibct3mjQoAN6AgyoyZxSISK7o1PRZGLKHnsBElZP2f4tOaW86uR1TDJPOeodxJgEqILJSiA1776\\\\\\\/y2FJ2lz7M6dJn4f4rDL3wy4ZrUXKTQhzdeLjC1\\\\\\\/yCEGOO8Bi6U+eyoQiwpxRX+p9GlPb2niWwponq8XYeBHgbblqIjjMj7fjJaZVO+TWug9Ph\\\\\\\/IQe5gBtZoyC+4V0s65\\\\\\\/TEmXdkkZAkP2B9SH6GSanwrvivAKzsC6VIQDdCy4qAmjEq0JrvR+vhCeI\\\\\\\/rjN56hAmEqoaQWyJFx+Ds1NnMt\\\\\\\/+LYjieoVbjKE6pADmoxLw9afsVs8QX7N3oRscjxl9kD+noZrGYx\\\\\\\/LSPLiP3vRi0b4OW5cuZK6NvrB7UZqmuvezUjzRxAqP0gEtL0CPbOEx5543hIaLomOXdcy99pLqzfo\\\\\\\/qHXn07QD6Mo9EueCD08XsZNjdfuKj0VDQbkYJD6J\\\\\\\/uon91ncZNIUj15EaIUEwCrpdNH0JHEc4FEQtXu3ErY8enGm6xQsf1+YIp1Blys4tzoJ\\\\\\\/Zcpn1CVpNi4t2liPCX6FGbd2H8xrCDJwMK6W210C9AmrEn+54OX+zzBjJjSS2ZDw+bdNeaAIzaf+iI4GnarUt2fC81JldChoczJAKlTSKr8ZTVM81HzysZk03QFUOLLoEUrJNltzx2aWtrG361caT2lDqYY6UTcpptA+6pn\\\\\\\/2L\\\\\\\/jjXB0hCEDVuO3uI24WWDfEEK2NtfhCZNTmTzj7XvXyw7M0ZKo1Sho4PwBHkI8ip1u2MBdmi88IoODXq70nA+WVqutx6jzbjxchoqwO8hHINmvZFwXLh8eJIao3\\\\\\\/V3bOvwnl+SgToBdw5VAotAQBqU5luZed1KRlF54BaUWNM\\\\\\\/JQYlx2k+0fMBpXAdAW5z9Z7oyfViGCfnlXUCw4QsfVBC94Efrv09\\\\\\\/ur4tFEbR4spg2GFtnsiZ9rOg6JuJ16dE01zfHu2HVTMKOx997prnvINA\\\\\\\/d6AmmbjIvI+S7pXWLbEgds7pQC66L6WJa8noj1ooErAT5rV0RBOepBJRLnUribQhyo+dZblQXxG\\\\\\\/XS7gzQWs83sg3EN6ZIaKF2qJy939wvnQ4+DUypksBne3BMWV4lBcH8gol8uEr5+q8f1zgVs\\\\\\\/8VSrlFmAAS0JjWIP2TaBMYJeffxa1uizHnqPbi3rkMcwGWEYB99cr76dhNJl+aVteMij8OAplQh\\\\\\\/3m5nnLE8NXmkysb9\\\\\\\/6NfK5\\\\\\\/vUquJOQJKJSlgMCY8LZ9\\\\\\\/IbpPHzK5+JqH8pOYp1AcmXLDOBc8sJSSXVe9BebuHx+\\\\\\\/rnhVzMcMSWc9Mx7niz3wRtgexRn7n3d2x+vGmUTfTuiUwcBCiDZXh6wsiwgfK1HV+3vdcLuTdQsiThDSJXwllTZFZi8ewrtRUETns1Xoi6jzNFftX+m2TQ0Q0U7S6PrL38GT4ZOUuWjE5pfNACpYJTV+\\\\\\\/EjbRt+aw5USzZVH2EvarjW74tx\\\\\\\/JsGHykMlMoVzM1ZK+VCFsoqPZdbyJQ5wVjwxqRtR318yUdrSts4MebWqLnZ61p+iiXXyCYYKgsFGwOJkPsIxmLXlqKnMBPRsUtLkyCBfMl5CDcjnikzhJOGHLekgnZ2FcLjYNddQU1fJPvFno49RJdrmJWdqkdBDhUtn\\\\\\\/CXQp68LjU2EM3uzfP43qjkW3m+OrFMDOTrO2lu6mJEGDzzF0Ao4LkuR87pl6spbcao\\\\\\\/1HdzEzSzVT4U2lZOsfyYF3RpuCM4EcMWpgRATb92SU0Av4muky0uqvvMGzMTCr2en9KGcL0v30c14kVtgzZt5Xyrr754y9jniX\\\\\\\/v4l8QTGBr+CxHQkDknhzBtGYx8OuMwSUxWZuO8T+6MQz1EB2lBhLUcL7w7ntZ1M+lc57Yu4CMWxk7xTTl5vFQvwN9z0bNShBeORWtcXVR1MouXl29vr+9ZBmLSEuEO4s3lej1hyerCC2YTsHlKmjA3TaLIQO5Q0hu2KAqCaGAAXATHfoRlJKI3mVPIFJvzTovSXdT549sqPnKKy0XEqbJ5lbODdF4D3pao73dPRYfYheEfMb+8K2kIp+jD8fhfQ5VtQlEEDE4ucEwWk+K7pcbGkm5Z6gFG+BrzT\\\\\\\/JAcamB2rDDvdz7zOu7rx26jAFm7NoFDolSF6jQvI\\\\\\\/0WvS\\\\\\\/XmJl5PORPkMP7LrHyj3Pado64d\\\\\\\/QrIQb\\\\\\\/Mzo92dsW9nYtHaRALkYuxZJiW6aK6LfIoWE1J+WLyFAd26tfN+K2eBtbfWtq4sCsIIaiQGKc0Ha75yHYTWSQJey\\\\\\\/nSlVPJYGiW5Eb9B5NwjJAuZm7OXV6MnZlXHmFwA2EwFNJKW83dKGt20c26DskypehtRCksUtBA3L6e9rIGjcb4NwYbEYBjqk6hC3XP\\\\\\\/sQKgAsxy5dlYN0SnWRCyAX6FaWBJT\\\\\\\/waqM+z4NzXPp2hGPFvDrRLmvvs9fZn59IjAVonefOcXnkSH4Pyq9vmcGgsssKbVEUkNFoiPCBr1csKM09TG5C+Mx1sRvpiM6tn8fbuCgtZFs6CbBfPgNnqbSwYvZKQU\\\\\\\/3rY8vdph87wJcz4MBDsmPkXo9bzMg9HZ6iRlWpuXr6Z88sJHiX99KNDL1ND2nADrPaNA0So7xUXQibh9bZ627dpqXWN3PofMHrHnPlKa6pswgcRPZ5pj6ghOYVYGKqSE1YckcbeU9s9qFH8KI6GzbevCpmQjcpBEjgpQI\\\\\\\/4v\\\\\\\/0Y76szTi61cOOlpBDJ32OKJNOrfQm+JlxW62JohtaU5TtBeCg75PyYWbjEPsbcA+TUbSMQdDSaUEtzs4vZGoQ1OIt4PMED0CtRfRDTzdA01W+XGp7GzUSmp9F4S7Vb9o6OmvtshProj8ZO46\\\\\\\/m0XRGh1vq7IHKpYPNSgvf65t9NrzvVKR6AUcN6a2z23rzQioa+S6aMKfVB6+O4KncLIXcINNjjv4aqlesc6eFXguJ439haxEJl3tnDiMeRa8wP6brp7VCmKQhN3azSFOfJ6dTD9\\\\\\\/o5dfDDzePE5BEeqeqpu7IY\\\\\\\/ZZvYO63KO01OFlOT80Dn8t\\\\\\\/UYkFHx+fqrPsWB3K2HqtRmQgCrJrOfHSvfmLYU8M+gp045p3y0Ff+EzZEB0qRz\\\\\\\/ihDKK83cfrr9CUE28V7UaypqCQ05rG0wbn8MVpGOeulNZ\\\\\\\/nmQojz+ynK\\\\\\\/R7zr0ewRVdwvk+MYutRWC9yhlwdHnVL9Hvec997bHX01QIOzBwZwNMjhMlypW1XVX5IBTR7k9EgExeX7pEvTBgeYi8UkOqTnr3wK3iQ90Ql6b8tyl82rIYVU7\\\\\\\/eIDeRj2f13SFhK54\\\\\\\/h7JXlkz9wsC7VRlrGkPjD5VsbQ4I6\\\\\\\/DSkN6N\\\\\\\/PSNy1zcuczWUB8rRZSbI\\\\\\\/rxXeFXGxPqt9vH1Uz3djoKYj7wg5K3ik11wHF6i4GwPvpDWlKMhFV4VjuFzf4RmDG7vSp5UYQOlr7B0GRS1qJg04OVl01NgMmWWo2NiqGhMqojVAZiiwf22lz+ZPs4RAUMQVFiP\\\\\\\/Ql3qTNaCEHy68miLieii+zPmKHTKBHydfw9m4UMh6DXlRusHOedbS4PSbaYwmICDX2BlAoOOwr+fdFX+kZIrXs9po67q\\\\\\\/5D8aQ91jb8BjHl0xVgwcvBV3tIhDuPEzH7zosMVXAmqpivsGwQYNRX47CT6o9gRIGGFl\\\\\\\/p8MldZYDWCCmPGrITxMrcj+Mjb92DMneJImlYAOpSKo\\\\\\\/WeVWVNqLiSvbU4aa8NCTZJjWtqcxfWAIis\\\\\\\/BmmzW9b3Bre04jjM\\\\\\\/4brZkePngubuNbeb4Bpc\\\\\\\/F9dm8nMTJwrr7T120WmqO\\\\\\\/3BCZv5x7haY9XCfcSJEjUTyzcGhryqH7RvDqw7Op\\\\\\\/+hjH98V8dGm4ZUevp8cpkFOKcsg1cLKAux1RPoBdZZJ6kQJYNZ9Vyigvne7joX3mVnTI\\\\\\\/l6BvZ4+6GVl69m6YWUCnicPe2FaMUd5ytAf10g8surFi9EgOImYTsstq0ExMcIOoMorbzyuqXqkEKPPw+\\\\\\\/9+Ci\\\\\\\/MOsDCJOo9cRpOzjJYsmBkUuE6p1o0Msqx37KGHU9AAfOfMnITu6hKFhj+aBHdfBWc0hqhMwFQNEYYtilMjTd\\\\\\\/YdmcdEC5QW3VVxhimadF2FP+zl46yCSMD6zsBsuwSQgQx62RFUeMNKIDm2ThC8IOorDPRmjWsJG7JDarV+JZFo\\\\\\\/wMSeAYqhFPW8H7vPV9XLwZS5eiJPWZonoGooM9TgC77UBxrtG4ntGSJhBbzaS0p74ijoKa7YiG\\\\\\\/U9fJQqWAYrayvoNjQtzcXS\\\\\\\/3o\\\\\\\/VP3nBmV8jsyIC5RKMUct8VQB0YYRQwapusbgPLE5foYIH5VnRUm4NvH\\\\\\\/ltetvyDM1W+WiZkFjQAjz9IQKg8LHzABFKHr97821ha6yQ0VnyCddbtHfQDvysXvWaADvaVdCoZcRRBJZe5b\\\\\\\/XPnIKIuU7QyEEal71SFd+S6BMR4a8gKInlU7AGsW3k7IQVF7FzrjCzycvjgp7JGTZ+5\\\\\\\/CvAFiVxlJIoN7xQ38kdCO768SnUVSvfQ55pk4nKGrthS8pSvADxWtw6L7ODJI4+FqUyRLVZna3wtnjL1nhieXWaeKsEaOw+QwuIOTNhrAinRiRxz0s5Yn9\\\\\\\/3kqMVW2AcOPQIbmeBxDfMmnTOkLinfPcck97SN3vXeVmjuD1EwLCPBon2k8jLJ5ZaqNC4P7l4v8aBHmtiZ0z\\\\\\\/KlwQWQ2B2RGbIkZ7cKnk9+fKIbkf3Un0wWclcPW6Dx+nNif+1DGEYJfO6qQCnPCRtQn\\\\\\\/uTPoY5phVgcwmAWhyMn+1k3s72YJbSFkKV7pEUBdhUAZNhT+fUBfiPk7G5hUqJXs1Db1iI6\\\\\\\/dfw70ps8SVUskZ6Pngdq40pe6+JZzUDErZaMXEUd2LCIVW9KJjvD6MPlzCaIA\\\\\\\/g43NcEJmuitU0+49YBVVf2IX33ybV7WHVUFzi\\\\\\\/XL7LXq6n43i0SlHNz5SoV78bZMHO9hZhwEppCnpbLPX3BOUJxcPvLkML7uHYWjm3ihJIbksiIcVXy7XZ0uAlEsRWwawCoiI\\\\\\\/bV7qtDRlRRQgX\\\\\\\/MNSJvbyeg+c\\\\\\\/ItBk25Vv0lX\\\\\\\/O5Ep2qYLNfgUBtOJLS1\\\\\\\/YX0Jdf7vRLKbNz3IHTSTUhQ00pPopZ\\\\\\\/WEXZ\\\\\\\/beiE5xiBYTqaEgy3svKDagZrvCAS1lCQd6pMdh+k7zra0i5bzhF77ElBEXJkxYOpLV\\\\\\\/tio7St8JYmvZPwGkNAu8M5MvSu0ULVMaLrS07bFdfj5DFYE0jTrHyjO5RAgSvVFPcgy0o4NCrIEJvoY\\\\\\\/l\\\\\\\/hvjT7z58j0PrJAZBMwBKEyx6Vw7oGASGDEnoe\\\\\\\/MhD0mIJKnRTIkogOEjZfiUaBpQWI389ONIzIouXKU0Fy8gebUqnmTJTfF\\\\\\\/2adfstNkr2Ne8FkCm6EgWJtLiW+4oLpwK3M4NxTFo8YW0ApAMMCl2A1SJoJpj\\\\\\\/7Cw+lbJJ2NTMKR+jCvR+KF+ew0o4nBczEpy4sEIp15UZqh1CqM6UNcpzIDpk\\\\\\\/+krLemgWOo5wp7sXIln2zv+N+ipWgzvf2nO4QcA+2phUR42XWGHoR\\\\\\\/OmCw4MXEh0X0I+zqQAdo1udh8KrL8C1+puwwtpSKTSseQ8YOjH9HsrPAMkTqU1RZABiZRo4S3DRXqiPJ3xX6Vna51PUfHLfxEm+zfL26N5H1wU7BXsfidXhr7YP5TPMg4arvdCH+M5g\\\\\\\/GZLYcx\\\\\\\/pDMTgJr0ZqhTABfs4qXUUx7xbKBNgOXjZWymAdzmDL+mGAIMwOeQGHt\\\\\\\/1DgwqJ69ZdFUI9pKMdVrDxjvh7iXaX+MvwyFSg25nfZu8NjJEtCtPguN2N202u3HNJDxM26d2NyI92UDMoqYfEcn9TTS7PIvtwcw2SegmD+QeIS4B8IONd85aXZqNgF13mLbCwBH\\\\\\\/qcovZQbbRzdd9741R75CsvtyVA2h1gLbTEEjllwmNl1NgDmV7tuHRNlKg9qHG3R+bPzQvT0lMUHRQug2awK2k9Or2C7GnHbuYcVotcmaXEnrPaRSX0WyFifcWPCSLqitvmE6IGnZBsSoIVMYsj33w2UcaE\\\\\\\/a03zeWzdZ5s1k5o+fOpuIhQsg3oXFs9Wt7lp0I2\\\\\\\/b4nw5ExfUbyrmdfK4tyvOdnGqhSDwinKlP588S2E7VE2yFjfqIIN1\\\\\\\/xEYIHsj3\\\\\\\/0HQmAduTrl2bP8KygUpfpluK4tjtNEADZuDHlrHLqHeq8xo1RP\\\\\\\/qlcP3\\\\\\\/U0v7RzoMUJCQm4PvSsQ34effBoa\\\\\\\/D9jMoYIkYTGi1bX4x++andA6gtUg3sFYmvkUzGdztcO\\\\\\\/zd4+ddsuIealrejtP7cQWsIvnNNiJWO0vEDCUC5wKK7ua0R+TSl9DYfCRONJrT7ofzGdgIAIyPo4p504zxe+D1kohmh3\\\\\\\/NS4tCQg8HPayiVOnetNTnydJQlBIHq74hCAt\\\\\\\/z2vCvORcVOS1y403dhQeE9xsEZCGubhBF3wQW5+337cKzo9Ee2N0v+UktqmgGWHXs8oB2pDpCWYrAbAfuCeCxsgGyJaTnHaPcVEEUpnKufckVv6KLjJ69r50MiV1J8lHSvzcZQ1cQnCZdB0R\\\\\\\/1hd0\\\\\\\/Wr3WvD2GeZZPjhBKOohkXuy9O25NZCa5IIfgC48ja6MFGkCScGbGkL2nALHRF8L6oCeogB8j1Zde1om+rfEJduhbcfurEJWkIyWjFTmPLig9XRyldYtR0raqkkiCe6YRBT5nWR2ecA+w\\\\\\\/MBjcZzbp5ASnLVz1YxteL3FOw4r4XTTHP2KT76AgsY2QdeE7uRcKCtc0pxXgwepQqO0o+zo+5Rr8juDcTUpUYSepHoCTEif9A3\\\\\\\/iRmO7QRH4JO210I0\\\\\\\/i7hB82Nb52VytKHv+e\\\\\\\/lkDXJIA\\\\\\\/wGAEjMKsxT\\\\\\\/QoA5rfevqQzNNk1uUgLp22gMGVKt5Gc\\\\\\\/KRHiV3M5lOlJExOZUK87sqqaq8dedY3JYAF1nmxjD\\\\\\\/ygi\\\\\\\/6u17PGa\\\\\\\/2ukJ6ssUPl6m2ssepCmqh8\\\\\\\/er5qCCWGBs7deLieQFAKBIg1G\\\\\\\/l7ImZUgVzz6T7yyvPZ+A8WD4VZSPD4ZF0HrFI4ieoyf31j8rvD15KqafGokxIw8fORGzgH09kYv2Z9tOI5cXb0qZLTf+RdUKa4+5K5MnbUrCi2QjwsoG0ubwvcBoVeUqY2ZaV8loLGKceJxXK0A\\\\\\\/uKb\\\\\\\/DdnTtIdIClj\\\\\\\/+2De0bv5yyKITi+7rZv9Ut7IqOZOF3WsdGvP58TfXGBam8hCa6y0FbVZjmmJEmAvx5DFHBnFv7YAK05buE1pf3PtSwLZlYV73KZMB9mso1O8NddCAErVGQZHbGNEsaBlyo9UbpxS95XjkETxQ8O3AAZCF9HHw3u\\\\\\\/I2g3ZTwE6DHK8IS+mkpNcNc29kBwzfMOKdF9P6lliCrxPobfud3xdYFT32TILJMB\\\\\\\/l6syNxG\\\\\\\/nlooOn09YrAun05oPiSIbzR39H8o30BjW3if+RazXWqk3bRGin6\\\\\\\/EwMot6\\\\\\\/FVVCXmAF9tA2q6fjGcw+h3DxOIJ4xax2WaBpC8pepxSx9JzGDbehB48AV3mh380\\\\\\\/YqjW2bqZ5QptQViRbWw107\\\\\\\/FtwkUC8\\\\\\\/aGLsQpwSViz37sLVcUPgPtrWTlFy1XyfcDQNd0oT\\\\\\\/HwZEL2WYxfW+BOnWs8yqQpQA4+CVtlzmifbSfDKzZFIFovR5TjZC8f5Zw2IN75VnrD5b3QCGsiRa4DBPeFCxVFILaPSnkmaZmFTUqFESKYo\\\\\\\/L0UE7oRae34SRDe2Wt1cL9ZMETOzRV\\\\\\\/JJKyRyytKjbX78uWmmknh3StQuGIEn0XQMoLCQWyYp2LWL+l4XnWbupkfF06lKIxt7qVli2fcrzzU0jofUo3DWkwIZxV+\\\\\\\/k+jniN1no6tMAv32CUs1K2pTHk1RtJCbQEr1+RSrtLfYvH8WX0wOnCjfQdjtX8c8+pAOMRGU7AVX4V6EnbzdDKHAsOXqemnTY2QSmVxESxieMUliQlq\\\\\\\/ReMc\\\\\\\/8bZ2P1BFXBnM6WU22vDaM0zTV3tJknFhIC9bT3B8MWf90ySdKFK6DhTaHKtRbmiuD3BcHLX\\\\\\\/aZ3xUTE\\\\\\\/ybH1rSmftV2CkgBsIL\\\\\\\/A\\\\\\\/soYn0zsCS5zYuu01faFmx4x6Vq694G83\\\\\\\/90d3JK\\\\\\\/f0xWkA0biN\\\\\\\/aA51aQdbaPSHcknQsAp1IRrluoKSMrbtqBqV3WnZxw1ZQoldvSF8R0CH5xFN4yokpNda3lwsLBo0OdG+lw96vgSB1ZW2pUMDCOjOoI7gyckXzl+cqu0ZMF1SUpcCfu9Z2\\\\\\\/qMhDPsRegc4WRX5y\\\\\\\/pBGUonD2KwYVoi1BcKoCKUnPrDG59hoA2OTcGmbjMLZ21d3HO1CqQAckqI\\\\\\\/060rFKJ75Tg6yuvUHniICbZ9BRfnBiXBtz4dxaGQrONzW0OsvdiWzhMvqR3LmGAtm7UVF7\\\\\\\/hUTucldSCEUHFaoJknjAnviOX2ULEgW5a3MrpGED4gIzb90lb6ue+h5DxwARU8J++onq+r7U63lZlWRgnejr8QEXjyihiRMlk6ewpOPq4uJq2Faugsa\\\\\\\/ixjNKYDyQCEs3oGaraUS+8C+AT1pqNdhC695EGlgJ1uZUFLHKo6g6ZTwgStvUOcVHuI6gdtdJ8NINoTO9\\\\\\\/qW7mDTZxSxVGVDv2qBWBqPhbW1S4o2Dh6Sit+zhrKq\\\\\\\/JpoVIA2E9NVTGrfAF3ly5etWGsMZYnrDQkDvdHBcWUmSIh859GqfR7+yi6fhNVl23jTCUO52myHHSO8LyeloYO0qtl869NjfdfqmZWIF1toMlr9tu2HVIy7psdZGDRz1IufsCFfgfSjycMo1iVS5FmcYjH\\\\\\\/IE43uhet8qIesFb3tjQENK63UcaYQALSbCK5I6Ud\\\\\\\/HmY5XylYUvWhH53AFnSb3gVfASu8D4F7EbDCBYFyrxVrrMOuy5AIBhduVa8UwDtIH\\\\\\\/rkgLvq9r6JbQLNi1dJbQgkftBwVRWv2F3g3GPxaQYGsARMEP5aah8lvDbQgQzft2FOP9jFllIVosQZ6PKe2TFzcm\\\\\\\/Tz4g36yoztlhUo9PKgd19\\\\\\\/F9l\\\\\\\/1jsjg64VJldPoYO2IMiMmfsD\\\\\\\/47JpAD2plLGLL+hU9kVsBWybfQhStDlLlqHbz82cBBn91JlSQBzLXDl7jST2FVCB\\\\\\\/5676nFRdwj9aUlXVs5JXAjl46EPPbE183cgJG7FB\\\\\\\/ftkCJ9itHAlQdo1ovYUkuWPa28oBpJNJ\\\\\\\/8calgBePQK0eR56v7IIAMxSXquQoxihEjGRK54+i2JBMrHh3PO9r\\\\\\\/Ovj71+tYt6DsPSP3FISyEdfma4p8Ghc\\\\\\\/vs4ejQ1ulghpZLXu2\\\\\\\/2QVBQvGhoIJPNqn4z3pPrS0I8pMMFh+KUv6oXyVlwYqvaZGrbht22D23PrtAA7b6S9epcEr9vwFZtOu79cglo3Q3oHw2TYTFicsyrlOu\\\\\\\/Y4+v3EAtPSy36Zbp1uFyu2lVcjs06kwYV9gx20IMhJmp2oTUOAuE41pkLGoRriRwOnUhr\\\\\\\/pObMx1a+uiq6tpLEr\\\\\\\/jR3s\\\\\\\/lfgtcypRoH8PalzHzPCFiKBmpshAqk8dhw0Dj0OSRY\\\\\\\/5UhTgI60lPANCJYxaqLuB3ughcYpihtWowcGagH3Todin\\\\\\\/MZuS4QE89zUvOW7vwFU7XHKD6Kfda+\\\\\\\/3PMFS2WbiWrwZL\\\\\\\/M4g3VnWbBo0OS6G8dUjq+63jh3Hm\\\\\\\/y0zdrDIUZKPTIH2ETDVxOJw\\\\\\\/1T1rH6I0xlJUxqhvIWqCEA4jtYQXDNVhoy3hbko5YaXMR3DWCG7pjZM47S2dis\\\\\\\/ovt5hkBU1XS51q\\\\\\\/lHyusQ8eAzgvsn4lfoUf9uK6yXzSupY31WCITLCoofqomB9LU6VM4eUJ0kZeYUpIkNg126MM1FDTxOruU37J3AT3bEZZCqWDsZ+3v1a+Taji0LC6tFoEhkytN7NF9jpmtI0QC3i6Yvby7YV7iWK5MCtCvkMtyhd\\\\\\\/m97qUuOuBwd+nzFhHs6dysSzyFd\\\\\\\/4BddI+XsQPIf8jX33SMQMoeNyMKbFQ+BKWU4wyv+hp4\\\\\\\/AaMpSKAR2we0x0dfpv\\\\\\\/tzbMDuzjTnO7Lah2ks6p1Q7bNseDkPvmL2\\\\\\\/+JwUtMJslAXFHXYcULiQBAxYlpJDqOf\\\\\\\/FgEtaAUAOZyqqIoNGaFPha8BjyizX9iMa+rMdWdttFgx7tdaR9wCPEKYO65CZgEuoFmK4lRtDWc0ae5F8BvHU2eZnY7tIHx5TafTD87V6wsPV20EhVFh0DOn6yS1A8++F214IH9jsIa5eWyYISBeDzG66+WSvETt3yGQ9MEXUqDhBcdMs\\\\\\\/ibFuSNbNlEb28IcWIv0CaRXo7V1ykN15qSAGdg5nPDb8Kcp7MTdoplx4HWsbMGa\\\\\\\/46hKhDJIUJsCqo4F5vkWYSBbFOhP0RTwubBvH\\\\\\\/V1MOZYC7fjqNMcRXlldR0P+NL0Eu8TNs3jYA2S6eOk+Q3pIFrN+lN3iZzwH6R\\\\\\\/QVNM+zXJYUso5ifnN5yBvryi0Y9HMVI+7MXoFFZRsLopswXFWFd0CcGLOPcKmJNqOnqkxGW8UeA85J9twB5jbqhSQ3ApRzecmk6ivGbtA+IAxvgxtVCIHlRfXrwTFYnWj+9vMmTEu5ZccUlyb3bom0eQHexMQf8Aw\\\\\\\/RN2HxgCGl6ooge\\\\\\\/AXjhube57GKDquP5dFUWQjQIL\\\\\\\/HvQeg\\\\\\\/QMSJkTeZxTdE3NE07Ed77A6h7cP5+N\\\\\\\/0U7mzPgV\\\\\\\/R\\\\\\\/8IbEtKsgzNNg\\\\\\\/dxwEfiPXnLw5DsMihRHIECAaZKnqkPgoxli7kTX4JA3\\\\\\\/iXitb6PIAIv+zozd95vzN8\\\\\\\/AKii3tpu4Nly2vdWdqcnlTFiRltuF5xH0COhOMwreFjliDUE3jSpXiESFeF4bn7GWnsE2H8R7b6qNwOw0kA5nM+FRlVsh+2EZ5BlWDBuw3q+xv5N1xXItWpuCDCxAnmJdaA\\\\\\\/plNObUHAprNwwPr6YbmsGdIAjjZFIok5Q1bcExXMo7n7ZXdXgWKOwMJK0UrifuRMLggzCKcampNN4nORTvcXYl+IyAAeOLjM7RlDwP6y6gngLZKisOzNIHUI+sErMYlqKy\\\\\\\/Q3qD5jWu5Pb+N0iApNNAgc0dvqEJlqY77zLJ+J9Ml2nQmPAbww6vqVZNgUzYxzMIX\\\\\\\/a+XPN1THjuM3ZfwWYK+WKNsZQJsSJ1crOH3KV5wgeLlSzrOjg5KUBfGD4t6IzsA9T1Rl1LpK5irchhXLO1pepqTuA\\\\\\\/LuHnR5c3lse\\\\\\\/ntlzyp+8YJe\\\\\\\/\\\\\\\/88Opzk7DhODLOMa3E9Y4Ee1Bo2FrORDEAQt5GEs3gpAjBQwdnveIym1Z52cRKDlEzcHm5RIYEwusA6RyOrF9B3pg2wwatKIZv41jtSYXw9LZXAxLmyX7qX6bZEqmPs2vMqcoSh+mvXCE7XBqsRoyyYd3u5EpX7nNhV8iwA3akRQVrFR64glqPgAWsiFWgmjSRGm8mLHondNCArjtajWPUxaDkS2Qkt\\\\\\\/Qvq7FLhHiPYfDArldKoeSuM8XnSCyePrXUOsT4WQGY2GoODjVCtPvoN2W2nS2owKbYG7aH2c6YQH0lmJ1FGagJlOv8JSMcZPn1E0W03KjLRv5v\\\\\\\/k+d5UcXv5xMSj07vrIaZU80NLPaFDiYBvrWPsALgoG5kqtR91B4oYeL7M+itxPujs2Ax\\\\\\\/sjHWOxj1Hppg5wMmYGPvYJUV40ieGyFyvp7EyAZaey+bYh4eX3gp8H4zSarRD02Skcn6aVVkgYibqB+0IiaUxi8wd2LsyUBgo8b8d5CRpajR+CfEx3etuKHa0yjb0tyVd+OWlb6XCE4+b9pePdZ3zu4Zp\\\\\\\/SwXYgPfnf9tIL8TtOJeFa1THLNYOvpXRLo+fx4P0VGepafe8dosUqZWFPDBxQK5bopZ1e1V6Lm+mIX49XtYkFUxQAOQjxSy3b+yr46NUmnpIxJlQgnou6BJjd3uBbylBXdgXDYGdXVMYZgF7d3hUuZ0Wvp9U7OIoX1UwzSWJWKDIwg1I4mI0mAXtD\\\\\\\/espqorm6okCj4yaBkqOhC95Dugt\\\\\\\/AV+UBTGCr1yk8RkiFyDG6huveLZYLNRWrtcbaSE+QPX96Yofmcavungh76N2fK9tMAVQ0Ws4py7elkRFrYyIklMtwMCM\\\\\\\/c7sGepVYNu5SJDt3K0RAR41ZRCBC5LU3rtvZvCdtbmq7pt68Sv1Q3nmFaGYdQSehLk558xSHdLy4DPaQXQlOC3LB+mOp0Feduw1LRqbRUoHCNrfkcvIw+k1mjA76IVyIhOecBwTpFmZa7T3jmqbrV3pjAYCwhpK52BWTBEFwfxx48MUDxY\\\\\\\/YPueSyYIY+3VZmHBhB82goEixibjTBe\\\\\\\/u4Yw\\\\\\\/Xigz77ngJWekOTpyozYLyTBzYiVF1oCU6YOm8AolVlZViab6O3SOIZopN3QZGzU4at8Qll9jWSmgjcbuX8Wvz3+r7KWeAZfA+9X31YrVuqEdRRULEtdpiZEHcS6\\\\\\\/EtDJ6cEeTJp9nyqwQBH236tWLD\\\\\\\/\\\\\\\/MX1Fnr90AA2fjtp8\\\\\\\/b6kGxJGX+2d7OoW7IPEn\\\\\\\/VgWwfTuJnrzIwTYjc6J4UOZMWojHU5Ib5iFW5CdJ3aEvUJg986JW+mLToL8ui2SjUW8V0Rap2GVdlBghSeoKJG1nCJDvBjvgSI96SHln1P7BFeqSf3wX4DZ4uIjVArmMl52l+xG7F0Y23UJxp\\\\\\\/cL4v1eQGtVqxm2TFo7qCBmmDek1CzUR38jz94ZxzlQsnGfwMNGW9znDsj99DVJXvVmWxjbMtzc22H2oU\\\\\\\/FlFG84fQdr+t2aJoeCRzHg5ItkwrOoHVWIZRy\\\\\\\/AioTyj\\\\\\\/1m0O0sILOztCEvhXD0mmYhDaSG9XDNkTWNjKEebWZOUXUSh8dYbASeeRJqIF9hBNOEuU9N31BQq3kGQSM274rsa3mg4vmrvYU2pxrNfoihduP8MurLX77p0zuAIfByFE8p+Iue+HXvDDDfmqLSgmbi1Bs0KS3hlvzluiTdj+Lx9l3sRSVeUkquitokw5YhtgxLaF6IaB92NUYGMcNGnDnBo3JfA7YPuGiiRyAZqLOro2gLLdfzTngT6SzqPgYuiaK79QAvYvo9dKrDKUt\\\\\\\/9aNfzsDAsiW\\\\\\\/FLXe+4+n8cLeX\\\\\\\/zdILE9lDdhR+w9FRH2+qQk308lL\\\\\\\/gumJ7BfA1xnuM5geua6269LgnXOGwXK7ujRDLKLQraqJ4QLkVB2tfBD4odUsJ4fMwvcS0vhsX7RbCO+Skj0WFvHvrD6kWx5YrROk8LT185YeFCVPfd62d50s3uaOE5BmlPSirvbJoN1ubYu0gD2+BLrDKJ2QZUTMRwZnP5TGikZKRR+Oxxuc4kj9jGZYe3gXCSL7rG0T4ovzfBajDkNEcs+gW+prBhgB2FnfweME2erkLQrmEM+jhJH2VHysauziv2AthoEqDvtYrmzpdhDjcnqDECaadB12umC\\\\\\\/zmvYkB\\\\\\\/diBTW7q\\\\\\\/IYXE+nngE+e3jm8xeC+sWoJ8Y1rQPRIJu\\\\\\\/mzo7A3Sy1221o\\\\\\\/JZQZyeCqqigBZDaIeGXBjuleQuPjmFB32DxjUALVz4N\\\\\\\/HSyYt5fpjaN8Q1WZrPrYgKqaYoT+KtFAh5iQvhgL21LjYZy7xWnA9dZYxTYM1K9xmQVtJKfxJqLo5ULv+Vn9uHQ9NIh1pJX1iqilCZGaKQteyPCKISxamxsDZDO\\\\\\\/tqjaoa4QLhnc79ouvNWM4HP3pETdYudycJE1qQGa8ZF\\\\\\\/0gJ4lu0IeoKjy8DIAFwobAmXXvGZoN3IPUrxQsESM4x1otaeMEqO5H+MkcTN2pRZcH8AkpjQ\\\\\\\/OoKyhjbF0OsMQv+WZrx23qTHVPhdKPRDxuZmf7T\\\\\\\/\\\\\\\/RaZ6V5RkZpKDiPZPcsjtQ0qzvAWLlW3nJbkOupX1pvIHbufGdFE8bl3QEQI4AbMwu9MTL6hBEMBLlcSbN1l4mQtkoyA8gap7azfKynd\\\\\\\/OiYhM6Qo2PXDzBU+D9yA4oZL3eXCAqWBcUUIL3lEtsgCcg79uIOSS\\\\\\\/7bAszwtJH8sTlx3yhj5AGV0EoPSNwBt1xZsOUkJlenwF\\\\\\\/zdpYUANtJoQnAGKN7mJuG0uKzWF\\\\\\\/wDK4zDijHMfdhIb3aAo\\\\\\\/KeKZ8dNpWCprHw5Ta96JBPdmKGZDBTQpQUnbA\\\\\\\/8QfLFU2NqdR0NmxExVMflYUC9DvC0l5fD2G06ertUqM1k5XtGN8iVgUALGIkz7PO2k\\\\\\\/7d6wZBcE4uvlbTSl86JJWIV+V8me8TRw8B8PbVw1Me\\\\\\\/F0+CsjYfifoEmYKzbv52Gq\\\\\\\/O2qkAdPyjUAHRroZ+gDA0Nv9O2CEkkp9KgUO4cd07iBA9ANcTBOgcbAbbVeGiUK1VqOlweWSxmhV4nYUKzoJmom9qsWPfZLIoRRRJO8w1LlM6r8YV1wM3Vy76v8PWfaAAVEBstpGBT8glA5LdN5ZOOxcU6NVVfSHFzQ9pJUT1m3cXkQHWCd99bPN0EXKZltaX+dYMaSkIgOClw6mxhuvEL3mShD4CZU8i6Usrn7lrNO4\\\\\\\/zVkoO8Fr2kYlp8AlwPO2lcgGO6ofkjh8+9nSkA+ZA8\\\\\\\/pdjxs9eu8Mi2dxo0CGIUu8sf+YtokVyNmmsO30MpmzJ7E8+kNbwuVjjhmcaaVAjq\\\\\\\/l9b8URBYMOvlWHdzUeE9M\\\\\\\/QbC4AuT5BYdiR5xG5\\\\\\\/hWiMc0lZQkz6KanzstKqt6lA0bvqvxo0KcGJ8J7ayd6dNF1UVHn85if17Ki06qXSETZ877Iz\\\\\\\/ts2dCyEC2G5p1SIlx9Wdmo+G9hIJS8cn+HZb9z7IzsDZsJ7zMofFKz53ZSpZk2BgQB0j6hYUhZrdSMLMINAev9YXTZge6n50m1SK\\\\\\\/7K\\\\\\\/1bt8ME0RF4QnqCF+8NQ28dQyuD53fPQ4iRycgcNoycu3h1HeU4DlpB06eGwURcDR47Xg78HYRHqSOEARJBFkVMGkYq0h\\\\\\\/uBHU8SDBN2X6Ia+GNu0F3G6++oNGHszRj9lb77tzTnPvVuomsbf6xttt3witMBBY+0+LFaieDayFgR1zIlf4\\\\\\\/OUFHGgD6NXsgwNYjn+wNaDzJMsFomXzpG5IOQzzwZgcVYqqAakzwKjzGsTqMqCXpNAHuhK6TrTodjoHK3AJnd\\\\\\\/TaEXhE7lSXmElch1EEqANZlEHtS4H0Ll6fyEJ5QttXqaeJRMwnFyPI7HgpHblaP2R10LnN8rhZI06jG2n54qWfDHFHCIiXcGyBw7oWnuMGg8nxgsYqNvCgsRB85D7UeQlcLXh3utFf9Ftb\\\\\\\/UwLR4LPhort3h3hQepPlbw5Qj8mbICBvmxBHE4DjNDANl0aHEG4OIeDXX8TibWRaOk2OFZZej228Uvggi+bj4WOhkEnzoi5hbbQWEAjEAes\\\\\\\/ciGPS9rlap7u5V\\\\\\\/50Mi+m8LsANA2DhRFt5\\\\\\\/ys1Wy0jNnY79MTJiOD1QkVX4Hjcz4SHGJMYqqjUijsT5qGYkps6QUJI4GDS79JsneVLp19GXdRH\\\\\\\/AG1CeEP81ME9Fs9j1e1dRzR5AxzjWV8W7gv8YSdKUYpYos9uPLmC0W1NpWvLJjwlTy92DowRKJoapaJbxxuW1Rk108Vkm\\\\\\\/zSKEpOyLpCoWZ6+BYovLE876GcJmXeIauXsiq0JiAqrskSO5bjQ8Irdih6\\\\\\\/4InK9FOmdYWq\\\\\\\/VcMMI66Abgi0iZJGeRlT6DnJoNjs9Kc5cJbYnAx1qVPrlbSHZsBVdqr6UkCsHMvVJt6byXXP+M8OfstgvF64F8QyQVkvf3u+bm+IHy8RXH\\\\\\\/yQ3mKABLJmc33ERewHwnc6xdmTaUXnoXYuloqvowBRBLV7uVhgN1zxm6NKKQyPXnjV57qG3rVWDdHr6WNlY7fgMgivY78vxUjTbXtef0bhuTe1n89d8+AvaZYn2jZQvqmp9WLWx+2Kyg9Ntg5YKGi9Cxi2ZsNhMEnSvfrBIohZ7+BDYRvG2yTSh7JPpj1mzeqUKCju2N8X7BzP8qfDJzYEU4eC+c2WJVaKFher1VywQrV8vDck7EAbLbyEOMiRs9m9NjIJ6Q7VBK+srvx+3BzKkHm6ERRN08xwmTGStUXIQ7ZKlPp1lzhwlqzB9M8zSoD4r7xkSGaUVoYWS8WvsKei5BrOa0lT8f5cz3Czi7j94MyEtIDVTHJWDGM0xEQlTxJbgp4wV9YM0qReqrwjyYyQiIDY7Wz9BeI6cLw5OImgyyZLOEuQU9mz70YMJy8\\\\\\\/AbsGv+gRn3lbKpqG1+NQ3YwxYimpLt5k2FYdgu696chHki+l5pz2hZvzwaBBbG4kgY7moHMSJYNRSuDXydanu4pace0\\\\\\\/4U2FdcgpjEolf7Osio14zOa\\\\\\\/Cr2mpVpi9IF8p5ahtJCA6WggUMiy8bbyPeQXahdRyysBRkNZn0aeiWkCBhO\\\\\\\/08II52ORgTh6aLRnhAWoE34TR8ai5y+VO8xCJj1yH94rD2J9aRUYkpOvfEw4uZ9ta0MdGZKjQEXmnZRFOjKp5HMI6cQcXl3304SqkvdItDF5i718BtT95D1A0M359+S3uImdHKBPuI3EtY4uI8kJQYDwrhA1WxyIK3RvB4SxPNtz\\\\\\\/tBujj5fjqCGq\\\\\\\/dTZZ8RkDl4S3joHIPQm033WcCZRyL8WQgbJNWKs1QRJTlrhxKR7LeDx99iEPFwOyCnRLyA9fRB5Z7KV\\\\\\\/TY++oE7QjmdJsWwQYEAM0OtyCyZCAzweqYOupfs64FJu\\\\\\\/DKtttVuy2fyUMhpJjdwuf0FGbp2lT11liNFBjWN7nRiLfm3jVebglA1GhvEGqrXZYzzIHHRrxaR4VYMRBO1tZvkqdER3YStaMfZ\\\\\\\/Gh9TiZ\\\\\\\/zQ3YqYnJcGUzmJwswjTb0XWH29ZEfKnF0DAwcKOzo6B3ey4lnnALHAL5vUwSlz1X241tBaCBElWHZs22lndzYoCxW9ACK6VQSTUR61arSHeSeuaVBygHiLuOM1mIuuKK674LKh+XJwVRGQDuq9GeSZUI9LdfngL\\\\\\\/vSNS\\\\\\\/hmoyDhs8nkb4Ijaw0MNFRMG6+p0whhLp2+5wvaPeSD+JDkM6c0a6U+3c4tFdNxFGWNZPrfY93Uruyi9Z1E+XTicdGkUlRE+OeodeG5xWdfhL9sNFgDo2L7RxBaceBgt1SMBalo0i0o2Q4tauc7+8fWWgU9jWn5rkQTFun7vBssg9fvEwTx9lSn0DXiqg8VWvqg0hecqyEgsgvZp3T2UkdInR1M3Y+QTlggSLaHmV9VBS2fZR1Ps2apcUedvyXEQk0RNnYf4KV6pW8mzGKzrERiBd+xVzH82qRMz9mdIsGE18Mm6QkY9kqZ9XUsAf8VdHCjHA5XTqzt\\\\\\\/VIeXseIgL3NpM4TUMGOYGGc3wmor7dGJNWs+4NuMxHC8tCkxfQPBBdzrluyLMZE3pyuDrwOpeOGwINIxlLmer0sIg9FF+g\\\\\\\/XzpekLrS0y3UOUv+7YdDnaXbGKPl9\\\\\\\/sWsYmHHeIvftm8oBM9GKdqD5xyC1Ql+sjIdlOgINF7fwHrWTeJoW5QPo974+ICBhbYO4FaL1BTmiFoGRfmghyELRQ3GIJqcwAJJeScHdNKjPgdj5G83XBAb2ea0r6YI9ijuWzsYuNQ8LOTtBRGJ9Dp3oeGms31CIGE60P+1kEcXnPDZkXqVkkQdeORYIyHZCBiQ\\\\\\\/Sp0l2agprFQk3mnfhJs25\\\\\\\/vgum7ntGsOohzwxJIVWKCvE1jQQr3qT9B5+1+0YMxRHiQFtan\\\\\\\/LFrzSazZBPYRTXmdJvz7xm6sEqqw+Jvlc4Z3XRO0bbTJryeZpFyIsHGBjdOm03LNaOk9D4vxiLbpwVryB1VdDUH5IdQTDCrbVKHrAEYe3CvTr8NoFQMH936LNtlD5KVrS3LKkoXuCLyq2rJRhTV2elyilerqSL40cPEQ8jj60hNpAhLSXlwb2hLGyzknTrZutMHNoQIEMRREmuCqlSyWTS4DXRVil1rmI9H9CBG8SDrepjrVpLb4P2Eh1cMkXj\\\\\\\/uIX8\\\\\\\/MF5+yNW23346\\\\\\\/ZnaGN9hBsGrK\\\\\\\/q+0kFlUofJczhQTujUVlvxoipqoF9cGyhhqRpM0oDbH\\\\\\\/jKAFCS4a4X9yxtXvq9nTqUQzRraQk\\\\\\\/+BrWPP503UwzpYYYlYwm9V2XxefjoyQKa08QuWGOT0AqOub4sud6NMT+iRw6dcy4\\\\\\\/QV3ygaw\\\\\\\/M\\\\\\\/sFq\\\\\\\/2RRfswLXVKxGCeXe8oYqVr7hGQUBTmxSiHR3I8v3Ez87crwH2qoimAvFTXjhmpLWy\\\\\\\/vZOED1xjjpZuBiLNeRS1QWLzFXq9g9ctwdng5g4NnSO3SgDAcyYZK\\\\\\\/9WuvwAU6mRDv0xN2jJ4w6DfcWhbXqoElTfPzz11lP3t6Rejx3Wr7zPO5wQ\\\\\\\/qSVg7I72TDzbRo6iZ+pSlbIbWYvD3kaVCIcAKLSP2v6PmTK31BxK\\\\\\\/3roBmEUpbEVMS1ckQX7RjTMxIAvvszL3mqaubZ2mBFkbczqAMeixBLUoYlFkdH5N7MQIxrGxWnwCU1R\\\\\\\/d5yCuhsJz4tW4VcG4MXXIdZ5Mt\\\\\\\/m1Sako7lQEt25gQKgGvAm6hZImcA2ptHHsVXepf5dxikhQe1q2z8V+J4PDy5\\\\\\\/rOmTElxQjiV00lCN6uZTNaJ40ryNTsXp1NNgVXQocCkDiWLSGUZlWtMLthth0jMtY3EaR0+qeeK\\\\\\\/gcu7OfUZsir07TnIK6ko4bRBc9XtwAU68o3+hZMzDwClpgHf6yg1t1jLNtqwDI5sqKGbVaW87Bq6XOw+eOvz3zWEdQm+15nA7q7L85uor8xH+ywTgTsYxY\\\\\\\/CRl8sgjl06xd2ht9o\\\\\\\/A4joG+uwMQpQzFk28z98xi7HJQPvu+69pMVN\\\\\\\/iWAfPd++1ZYu7T2mZ9jcIB21zcq5sbn9QJvFdH6Bw6FT0cyMieoWQUGAp\\\\\\\/7ifSUOxQNtVw\\\\\\\/UnXIbcCeBNfZKC+KctaVP8rWrooW2ZDUXhRXvqnqjpBWT9XVPNBx6HVQqawkxDz\\\\\\\/dp6CbXL7PEQO8Z\\\\\\\/cBopYrwx5GB\\\\\\\/1iY2ZAPqid\\\\\\\/sXa\\\\\\\/AayF\\\\\\\/FNPNuM\\\\\\\/RO7ANnAhPjKkpdhkknO0I3v4DVgpkQB6LO8bUm5isU9LO3KxDTQ0Bx88dVPNFGR4CY2eQ2THFuAx96SCwJjyn17kq5UCLMlvitKYCmp4CQ3bloi8Bg9v1VTEiBj71cGe6NhAWh85y9pWMvMOci29+nQOArBQFsPw4pMcdmy2S8gYHPwEj\\\\\\\/Qc5TiW0XgqO8kuI2VK3BF2nBy8zXqElY\\\\\\\/WxP4gKc8sktN5nKaB6hQLUK8PeTEoBupFr2IoByaqCYfluem9x2kbuysEmkxNmO0NHTaUEs7LVg4gPFHlB8E+sZvFFMwoxkl8foQqXTjrYHzvl2HO16t3JMrkXiAcFLwwcX6\\\\\\\/MRndsyk+WRWL7NZ\\\\\\\/O5OiXIs362YBqoWwqnCuCB5COVYMd0+H+2I7hmiDjvUiqUFProyw2mRoK3ujI5GPJk\\\\\\\/KLAR\\\\\\\/\\\\\\\/Aed7vwQ13za1wJ2cfStEDXSL+OAEKDeaP4OwHylSk5GOEUjLFkn+kgcCZ\\\\\\\/a6XlfBHcSfLkD8atku6\\\\\\\/loR80B6HGErGj6dYYM6VBWVtzLBFza7n4ufyhRKF5px6Llp\\\\\\\/\\\\\\\/MlYijB\\\\\\\/Fvt\\\\\\\/6HbqQUEfuJPGurWWDcWwAPkdsnwGvUqyExWOatJb+EyzH+XOcYt0FwqzM07op\\\\\\\/3uyJlhLd5dmdKgHzLGNyfLT4g+DqFjb42D8qwOXi9\\\\\\\/jlx9bp9CgKic2o6D\\\\\\\/ZoPefbmhfqPSXKZpJwgfsYh2Ttl2k8Op695+SYI3KoOWMQ5\\\\\\\/kNR+o9HrERpDKRpp3VUhiZ5xV+H2S6xzQ5YissBVYH1pqFFi+mvH+mfcK6jdw8bqv5M2N5nyLN9Aogwleq61FztrpopSpkUYntAQk+Dqm2Hy1Sxi0nQ2GhgDUMNtwaUYMgO+ao8vVQmhffC4AIWwVw2c\\\\\\\/hsQouHCtXLlrIQMAVoSQZknztUnaepVbMPXvsTiOqjAfkgKxz2nULSNYLtI7IWbyFBcKLdBRRc6M3+TsZlzqkPip9+IIKiACjwa29meNQUeikEfqYPnsL8lYmyizWNfpTqQy0QnF5ZTyq6vb0tgRR2u81VS7U9O++GhzCqsgx5h3iMg2KCejfUYT\\\\\\\/wikx3WTMQzyvwbN5y9WRJk6l8oaevO3QUrunu1sdIGL1buqIHaTzRgKgc+WG8gh4xad\\\\\\\/txQascd\\\\\\\/KyFsm\\\\\\\/HPJtZd2DepatCqY\\\\\\\/V8i2Rt7jtMlHUn0NZn3WFCm+EZC1JheJgcKhMsbwdt+pjUK7cZYwyaIUl8JGMuIiCB62GDMLKIDVx6I4FvaEtb0O7Hy3i5tngkj9oRrdgpn0JJGmUp0mgNRYgcsqqykRMyB8wrITTWVF6zc3Yd7B3p4\\\\\\\/o2vIw8WIqy1JJfh3imIEb41dIaPYu\\\\\\\/gT8WcLiZj4wBEDvn5o6wWsmwZnSM0r0501UG+bP+X\\\\\\\/wRyTRnJo6zYgG1R1bjYyyQBsOmKonUIe3whORhGSXuAzpeCgKNbfGsm0cUxmVyn9x3bZHKEQUR7nKQSEZZ\\\\\\\/TlU9FRgmKHigG\\\\\\\/qC7Q6LR9nk48oFCciMWk0tQUFjDDFn7c75zI9FktWSrOmUhztvsdlpTj+a+yMlxOjNAXJSfbZRZSBSHZ+HyhlB5SoLhbX\\\\\\\/5hHYeABo5PFKe22I8Lv0NocGG5JxxsxY8fGrXDjFXlQ7x9HbQx0GY\\\\\\\/lY57nlsKMiNfHDiwHm3upZT0vD8IEeVDANQm8AwhnitKVvDZXHBBsWTzVZ6rimSayuER6KEMdizgMKSmk5HbYngqItwPtfqbKttRdVmVuo4ulVvi8WoPixPQZ1UCfLW5R6qN\\\\\\\/G8KLB0DOy4BDEoyrtFG79VG\\\\\\\/eoQWVJir7Bh2+ssQpRMQZgIX2gEeozZaaSXyXo81QasIv5LJYze0CaRIMgmWjjXWsN\\\\\\\/QUytAeCiHjPAfmKy+4uwBWiZog0iRo\\\\\\\/GIU+4K463VcS7z5I1kWMnDV4Fkuu6v3S0Dv9Qlaeg9laMtsvxiMAeZyrxD7AphqyDFjNO+lYG+hiQ4NGRgWao2DEAF3gOJa5vywFZzwyboDu3VvyYYABELztnXInPOwoFeIVSEOV27PzT2GYyyEZN6mavSj5ZG2YdLh1Ckmvg2gvNOZBEASkO+\\\\\\\/bhDDjuOUyFPTpcOgrLhzTTQNBP1J7IeXJFrnRPaAgUKpCYM3eZlwN4+YK1ZBW+E3llbYpemtO0Vgn4CmPMmgKKq701Lqrs\\\\\\\/CElFgZAWsBuUzhX8EWrAoD3RlotkRzo6AKorDUXEvfSWF8Q5o333\\\\\\\/sMeZ5wQxrC7G4v+xxOMcnZ\\\\\\\/M+TBE1Em\\\\\\\/VS1knekBkChy4YmKSqYUixwyEF8U37MXFNHi2wkUP4aRMFTg1oy3Rv0Gon0+St9xgbC8uXH8O1xmSCJpATNguVOBelsUeL3lAbadvRz2pZvsXrKR8n0Cml3J2CAvWtMAEsC8DzV39Qky7WmbZWz3N9QM4DD\\\\\\\/Fb0a5OnvmGQslhvODOulvQEU1BstTqkO7NTpzfZYKxHE+X5s3fw9TuuPNzmZIcAYE6gAOWjKHwOCrIkjAaMgtbbiIUK9GUoXmivIxLwOjbbB9CkCikMX2KUrgzJWVahaFJouAViGRS8\\\\\\\/p1STapwfseoP7LOXffrJqdy0S4eD4wW7\\\\\\\/Vq2N46o\\\\\\\/I2sZhSKxoZHqzYInJYdcxIqmlNCPLugxWunTaCkH3TpnLHgtLQ39JzqEOFPvuSuISPJMszOwt59hAmivtTuBwobrrF3T4LX9hYgpFTvzndsVcvF51QfKY6o8e9JK2Y2Mccs8seh34W7TeUJ8F2PZyjMVxP7UK6TKtia6l8LTD6b\\\\\\\/qNk4tdM0tZL4zI9GI9StoXzuhajNW2PHEys4NHK529g8hHkGddyt56bKHfvYUW7QxJvi90SbDD1bT7LRbHBbikqskUfsmsLuVVjXZmfCPUSwie\\\\\\\/aUIbrLqpXV9OPFf0tW44yDtKdqgrIUYq6TIjT4VXWpSUqWGCN5La7AYf3WpgIjGwcy219028qA0i4j9IO0e0tfcV95Wf8nqb10PFmt3VcIZeKlE\\\\\\\/yaCmLhhXbBkrEtCYX5\\\\\\\/sD4B8z3HW91NKueI8i28ggD+A4M7yRxiEpjrD5C8Ggs1H1wv721I7o0hpiZ0CtcNnYBAhpx8u65HzRBFJ4qzLpmau8JR5YeSFRo382Xy9oBcHVEF3bsvvYauWcYI\\\\\\\/Tn9uYFyVOLZwRgMM5C1Y2t2uzqhV1gTd5reSv3oO63yARNkdNV3AfHqSMDQPk2MYtiR3coW+uFTZL0mP8WtYlRDayTDXY3B3SQx4l2q5LU+99\\\\\\\/xuWvKl86xr++rGhbLewo9jFBSaHrqcwzRngOT8AZYYU+D62Vr4TDVqEbw0+DR934U6h0CndbQzOJb6pesj5ScOmYtOsghvrT2Nqcjg45n8qTXJ3b5TtFaqbe4wd3xnmFh6RrUiBK9QqElx5IbRvWrVDbhXhNV62qo0Zj11Kt27eJEfCoKkcOv9WYjXuhX5gwXjBROCw6pOC3Wl4rTCbvzlbFCwJPcfjEiO0h7Su3olV\\\\\\\/CB4NRXWUiuVEGagzG5nzC37Vk8rnSxjXawZ7\\\\\\\/Sf5ArEh\\\\\\\/fY7IbXnbSOlZ+KYM1L8YNaqJ\\\\\\\/16\\\\\\\/fMyC+QqhEusQ77xu0VNLLVGpgtD4Y510bILgdJ+MXn5X09zc43lBhZxENFu9x8ZhbYClj1v5GgGCBSZaLebNxzNL9de+IIjwcv+QYTfWDHkkb8Oi0+UTHaEeo35QNFdS1bH+pT91lRE4ru4yr3Lcv0T8p6b6ShlppWON\\\\\\\/HngzMugTw35Qt0HuJmtbfNFckAv3J+At3eKJmO72Ub0QCuImkrydB0cLV9bGnd5YSTO67xV6qyGvcqF089Jw8639uZ\\\\\\\/+LKdsmQzig4hUTGcThG\\\\\\\/22qhhwGUljIm5Lf74yzFAKXAR57zTayh7PuM5xX3oALVQGc1yXWqSJW3gJCLvc3ddYPOdf4fYuUc+hiFjj4fCShN5MXSrDYe9NuCUx8mvtFkKbrlKtlogKVv5YRExXVmayj0uauYjc1xvgxjlLjOACy04MikB\\\\\\\/D21rHAywbb9qqwtlvdXBlwp2yLDl8ROSOV5eiUuOmvmt8ocQOzKc1uHX8aCS7h+MBDeksFdEfwduuJ76fnAgWaKkp4ChwTU2tJ8dbMNALsElqmcGN+v35q\\\\\\\/zBHXVz3xUepA\\\\\\\/+fEHoAQptK3W3uUaOfWagJN8W3Ixw54dTsQ5PNr7+Ragw\\\\\\\/sVf6FCZOIRLcEIw01kX5AiWIqn9Vl8GaTsH0N9Ii87Cqvmqhl5ByPtZXYUX+wDdr9BcJ8SvnW4yojLwdvTwDrf1FecmYioH3jvkPck3vFt9jWyVVJyf8U0a6lM8ZYAJuAblBMr3jzU3k1FH2IJj50n8CvG4ik43JcqeO2idvorqYUlI54Mshh6qhqF\\\\\\\/uxmARLDKOtwaOrFiZekQA5pUgrwU4k8sHgvJd\\\\\\\/2w0HauIz+LXtYoOx7xlgkRRXD9MmcnE+EFu0HPOMAbLlS1YbrH9BpLyqDv4gnitureW50AdKGPXGQ2yoB5U8sCPVf3BSoLBJCcTddDGqgvwN7evo1AdHsNZvulZezFMweXSA5uj\\\\\\\/0LHZRk\\\\\\\/K5fD2qV+OYKB+P5QKj6dcvL3FXrVlGDwyVsk0A2WlQAqXWNuogS03RXduaUCmrib0wNxSBsR4gf2\\\\\\\/\\\\\\\/KNdKQqtE1YVnEMRK2WEobfVbjzasDu7j02PZFvwgtswvIyJNzy8ZFAzh9Ut+lkeDQOBuxoy8f++I86m5T2RP2kndjumPydDUdwGfdK\\\\\\\/5Rv0vAWZl0if3klfnS\\\\\\\/WSO76Oph0ndO26F1Q6HTBTvPvgMv\\\\\\\/rzfMvYahGJqOFtjgBUYW2EalRHHYXSAdYwAlTwpkT0xChbrt258igXWRFK7+ahhx+JXVdB\\\\\\\/Qn3RT2c222GKCX4KB5joKpmvfvprp\\\\\\\/YqLap5ACGAyZFwEQhWZ9t4blxpwbuEHD+hZIc1ZWPJVANUuP1fGI\\\\\\\/SgHUzRaNY62\\\\\\\/IyDTCdsbWaSluzotQcD+smIA9MXxOP1a3\\\\\\\/GZMbCYvc6wogaGi5RhKh5uRUuCBeMaj1a9On9WqHk0DjvUO4vrsWbp1RF5ZpaausNDwIHKdvs06kNUeRY6qMPsw7ZQRyX0PYujyj5yaRWr\\\\\\\/m0TOj\\\\\\\/n9dqVKE1vm2PqZIwHjA5A5qcIaXei3zMAVQtBos0GvBvgGqPrYhNlKIdhqjV7WybhH6jzpIsPEd9Z\\\\\\\/xmCXNrDi4t3IlUUbgISPQtTjOXt\\\\\\\/dyQlLMkR5grP+y3bTQCamoSy29uHPyaHrrqpBNZ\\\\\\\/LTPi8GWR7VFW+1pfPTzFN36Bhb5EFaEPGl4ZmvnzgpbYNLFOXxjOrk\\\\\\\/zwtr479av7qAlmRPSHrjeyepOynwmprgy+\\\\\\\/WvQarn42D8xROYjCN8G3nI3\\\\\\\/jpnvRyh+LKRSE0SXWkMu2sifP\\\\\\\/8h7FwjYYGRFBR\\\\\\\/u0o0m\\\\\\\/PaSj3uS94+fAmbNymm+HEhWw6bbgFT9O61He\\\\\\\/36NPnFOo\\\\\\\/xMGev94Keia2jaH5KsyDNkuWN0VhLxT6lwiV8NAcffdHHGkus3xvkoEf2c3uGJDd4lAAgsgpWA\\\\\\\/BLrtwYCba2PYOZCCo9DjH\\\\\\\/heWsqnuRZAEg7N2ijLogYUMViRTEsboANCk6JBnoNb4FZLCgbHbmjm2U28klINFUQrTUL88o7DWZztVj2YkFJRxYB6r8vcZgXAuiueER3Yvu+UghGtWJEXGCmCqyvJv7d3uN0hOaoVy4JrVNKQ7ZABrJM1sjwSc8jl6zUNzJQDrp4d+8yUYd49vpJAB28EHg7O2Z\\\\\\\/iJhY20LhVkgEj0GCVDTm2n5WV\\\\\\\/I6FDBpCoVxoJfw\\\\\\\/j0A3+jK12qpwGYILKSrksTz1OcEYh1OLk0NTU9\\\\\\\/AwNDEHdzgPHIi3yEvOBe35hPEE09KJqCkYYGyQz3PtrSD\\\\\\\/WHGJWLgOkkKA8wiuKoRQ5g\\\\\\\/pb8QYwMVyqEa+2zdgKikHxHP\\\\\\\/bhUrj29rm2o1NidlhcQhx\\\\\\\/jAO0ygaCabFuZZ4lwQFnc8F1NdbBWjC51ViNxcIGdmrvdpwhF+\\\\\\\/zoLj2yRGevzAokikcS8SyjpOoq7\\\\\\\/DBeD3COnlvKJSVe3umRWizcAJK\\\\\\\/rjvdqX9UrtZir10cPgovdh9wOLc\\\\\\\/KSSwRl8Yg1Lbl9b2b1mOgNqtdiBQyrANPRm5DFN1pdozbpicZGzV2jDb8Kshe7hHWbJ5NZbDDYe\\\\\\\/toVcQ7YWT1UxkF8SZkumblOtJiXkS6y9oVxkrrYhzJEA4BVn09qvWsljekRdHiM+aCZbMaer1UpXxacShV2v\\\\\\\/kRNiESaiWXQc2gCbjzcD\\\\\\\/MGgf\\\\\\\/QEHxLHyaE1x8Q1gl57Hl9t9H1LNUE6tNEC9xLM454\\\\\\\/jehNkt0pNl+ldUeDYkubk39qoXD\\\\\\\/4zrwhvkQKptO15doaz2qjx9bn2F1fKlvlOvC+e2QqN+B\\\\\\\/WyfYGvDp\\\\\\\/uF3prmGY9ZbibxJVxD0f0zM3jusezWp+weFmG\\\\\\\/j2x44RLN3bnQNTIJQnIBEOJ+WzawMxnxiUR9wPfJpUz5ASrg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Qg8bAG5\\\\\\\/joYK6rqm0ZaQhA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c7234b7881d971e82314d4d696950496a8362bf640b9a647d4765d958f14e48\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BB9eVjU3xO9ZuvNvf\\\\\\\/sojtFeagZha3ATO2i51ZPTCHDHqKQtWx0akB0lXTonzjg8dGPkB6kLLK2UECIACPk11VMqwrBTL63qU1G9eg8MowA\\\\\\\/7b86fDV\\\\\\\/hfN6MgxfZI9abu3SSh5vM9OvigsrEM51YOVASJ0VYbmlrPBJpY\\\\\\\/IhbTnvmYCCGPZOxKLYx0q3jxFRluk2ygZ+60WHhz4UBBsbAOyn9QQhO9EnB8zv+Xf82VTl2b+RVVpdILKk1Otsw5IfCFM\\\\\\\/e3ZEvr6UqAusDSGZoDV5KNSMmhVlpk95l0bdsNbRITAD07pQDlAM39wHtEdzLieOV\\\\\\\/7nlwejJjEhKoPUfRIlzHBwWIpL2faTsoEZM32PDnx7AqowtExJhcDTUJwEJyTfgH5PXSmtFQcm2oYdd3l4d6rP5n++rNFAFeXCrQEKAe9ZNzxvXZl2lu3QnKiDqSyjyC9U9OPuMJyZFi\\\\\\\/CEo5FAMrqHQ3L\\\\\\\/I6dv60QIWBtwoL9zKw5YjJqCgqLXcIRnB6aIpCOVBwNnB5TpbVbbChAVB9zUujeRJlZ4Q\\\\\\\/mMd5wfOvZV5aazz1VbdsmCiChrl0CuoirGte33yFCKOwCuZhV3XfjvaotN7jP4St5S61FBxhZmhucRCW34rsGPMV4uKE3g52WZigdPZc\\\\\\\/h0qOs3A5WvOKmFTSV2GwnENGP3kpNgh4wXH8vWky8JqzYSTVeCjlVGwDhotEmLmYZWNn97RNnWAZhDL8jOq850C5\\\\\\\/7SuV+b82YKsxjfJojw1XxTw6DXqheUH3l4FROQKHU9buyOvyK1xcGzrgJI05rCcvqkFIFkZA1WILvS+3YjiybpVDs+cWTvN8fsMdfXbUOqZdu4c5MOqIv+60n6u3ZOi5e1vcySp91IRlcug0lsnfJm06dVKue6f7uLnyUD6iVGRwvr12R1wUM3xAMQIk0SJQmwSTnI3jrAyHNZtS2x6HUN41iGssTvGX6fmElTLTwvRW\\\\\\\/6uyFt9Y3tNBtdjzfcFOkdMYNsMIRUr2uRZ5CDbBuawaERvIHvRwvS3P\\\\\\\/Kc6YReTiwCOOiw0f3jWz2CC6XQkjPcViAhpmsYjWAbn02DN0vZeApJd3m070nFn4snMdo3yB3L0DYcRr6GgqH2NMMWE1dScX+9JFgK77DLnQcIRiRIHRHBSzWkOLDSR+bztWcJoAkebUZ1UaJWzd7dF7Cn1EooY9uwThlKhz2WtxcoLmHungvuSPaX1+ogJxSyJNLeUj\\\\\\\/5qc99M\\\\\\\/G2JVQp4vUbtoJOBl1ORT7x5WODCztRV2qQTzJOIGyn8j\\\\\\\/iXE+HDGwflVVw5jA7xX3+mc6dGmVOhilEFZJ23aqUhapnHFPiCQKYCzVgdt35HKoueahFXbz6kfLo8D3ECF2nZ+8xXMOWp1Dgqw4wROf\\\\\\\/atdo1Iir83xjEWowBf8EENH9m1taSY3I2Dv2stWYSWhob+90pESWaCGoXoH5xR+oXOBAEWxlrPLQM3wJ2vtD11mFQNjaT+ibcZN9GxN2ynMJXKRASLEDKl75BL4wfmFa\\\\\\\/418LQPHPIOqoHom3qcaHKbt+zGlwTiExP1CIUbx9an3\\\\\\\/4POlQWJNx7IqBxc5H6weoq\\\\\\\/bNq+nkh1D5x5ELMj5FzO2MxBB7KrZ7jGBhUalLIo4nsjQ9uJDdAPB9PlNBmYCewJ946iQafP\\\\\\\/LK0qWNKJJIpvpxVkAcO67w4cWF996dSHqrtxmlCk38ELJKOw9d3TidsqQ4k\\\\\\\/DcqXsAc9HkfDwXGiJrJZ4R7PGwxXetVBx8duHmp0s9llW12VcXxwYooGxsnFepcq8iIz0cOo97Alll75pr8AGzGWVrmpfDIe0scSqXkyT6eRfylud7dDJ3tfGDJiNS\\\\\\\/T5TAvIRVnzBlwE6ykBBlP9okAbZWGuAzbsRUoQybhEsCeg+Gg1lEGFDUCcZ6Qk1q1MT3T3+keA+rmmoYRJVDJK6+jeRs9tTC5oEApMiTs36WHrz1djmj06f\\\\\\\/2mWa6CoPAETjWEvvwJ7hUv5Dl0EXDdc0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022t2ElkCiH1md\\\\\\\/Md8DA7GI6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d88979d0687c25799eb1719a5ce31ce4876039e0a3bc90bdc42c6bc906aa79c5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491721 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3961" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IFf_hmcDvP1DaThzeO73TAaSYCTbfqbnYExaYqKwmx4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3961" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rbvUcqLjA9zjZtUifDafaRDq4BBc26IPiPyftG6t8FQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3953" title="Shortened Lifespan and Other Age-Related Defects in Bang Sensitive Mutants of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3973" title="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491721" id="node491721" data-pisa="ggg;8/12/3961" data-pisa-master="ggg;g3.118.200760" data-apath="/ggg/8/12/3961.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renaud Rincent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Paul Charpentier</span>, <span class="highwire-citation-author" data-delta="2">Patricia Faivre-Rampant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Etienne Paux</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacques Le Gouis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine Bastien</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincent Segura</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3961-3972;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200760 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491721" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Renaud Rincent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renaud%2BRincent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rincent%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenaud%2BRincent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renaud Rincent</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Paul Charpentier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>GenoBois analytical platform, INRA, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BCharpentier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charpentier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Paul%2BCharpentier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Paul Charpentier</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia Faivre-Rampant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EPGV, INRA, CEA-IG/CNG, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BFaivre-Rampant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faivre-Rampant%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BFaivre-Rampant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Etienne Paux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Etienne%2BPaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paux%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEtienne%2BPaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Etienne Paux</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacques Le Gouis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLe%2BGouis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Gouis%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacques%2BLe%2BGouis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacques Le Gouis</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Catherine Bastien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BBastien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastien%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BBastien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine Bastien</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Vincent Segura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSegura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segura%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincent%2BSegura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincent Segura</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vincent.segura@inra.fr" class="" data-icon-position="" data-hide-link-title="0">vincent.segura@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3961" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491721" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3961.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491721" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3961.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491721" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3961.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491721" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3961.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491721" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491721/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection - the prediction of breeding values using DNA polymorphisms - is a disruptive method that has widely been adopted by animal and plant breeders to increase productivity. It was recently shown that other sources of molecular variations such as those resulting from transcripts or metabolites could be used to accurately predict complex traits. These endophenotypes have the advantage of capturing the expressed genotypes and consequently the complex regulatory networks that occur in the different layers between the genome and the phenotype. However, obtaining such omics data at very large scales, such as those typically experienced in breeding, remains challenging. As an alternative, we proposed using near-infrared spectroscopy (NIRS) as a high-throughput, low cost and non-destructive tool to indirectly capture endophenotypic variants and compute relationship matrices for predicting complex traits, and coined this new approach ”phenomic selection” (PS). We tested PS on two species of economic interest (<em>Triticum aestivum</em> L. and <em>Populus nigra</em> L.) using NIRS on various tissues (grains, leaves, wood). We showed that one could reach predictions as accurate as with molecular markers, for developmental, tolerance and productivity traits, even in environments radically different from the one in which NIRS were collected. Our work constitutes a proof of concept and provides new perspectives for the breeding community, as PS is theoretically applicable to any organism at low cost and does not require any molecular information.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/poplar" class="hw-term hw-article-keyword hw-article-keyword-poplar" rel="nofollow">Poplar</a></li><li class="kwd"><a href="/keyword/wheat" class="hw-term hw-article-keyword hw-article-keyword-wheat" rel="nofollow">Wheat</a></li><li class="kwd"><a href="/keyword/breeding" class="hw-term hw-article-keyword hw-article-keyword-breeding" rel="nofollow">breeding</a></li><li class="kwd"><a href="/keyword/endophenotypes" class="hw-term hw-article-keyword hw-article-keyword-endophenotypes" rel="nofollow">endophenotypes</a></li><li class="kwd"><a href="/keyword/near-infrared-spectroscopy-nirs-0" class="hw-term hw-article-keyword hw-article-keyword-near-infrared-spectroscopy-nirs" rel="nofollow">Near InfraRed Spectroscopy (NIRS)</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">To meet the world’s current and future challenges, especially in terms of food and energy supplies, there is a great need to develop efficient crop varieties, livestock breeds or forest materials through breeding. Until recently, the selection of promising individuals in animal and plant breeding was mostly based on their phenotypic records. This approach was a strong limit to genetic progress as the high costs of phenotyping strongly constrain the number of candidates that can be evaluated, especially when there are interactions between individuals and environments that necessitate the evaluation of selection candidates in various environments. Another strong constraint - typical in perennial crops, trees or animals - is that it can sometimes take several years to evaluate phenotypes, which increases the duration of selection cycles. These limitations are some of the main reasons why genomic selection (GS) has become so popular in the last two decades. Its principle is based on a combination of phenotypic records and genome-wide molecular markers to train a prediction model that can in turn be used to predict the performances of - potentially unphenotyped - individuals (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Meuwissen <em>et al.</em> 2001</a>). We can thus select more individuals faster, which increases genetic gain. The development of high-throughput genotyping tools at decreasing costs has made GS possible for many animal and plant species. It can be used both in pre-breeding to screen diversity material (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2016</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yu <em>et al.</em> 2016</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gorjanc <em>et al.</em> 2016</a>) and in breeding to make the schemes more efficient (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Heffner <em>et al.</em> 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gaynor <em>et al.</em> 2017</a>). However, a great number of species are still orphans of any genotyping tool, and for many others, genotyping costs remain a limit to the implementation of GS in pre-breeding and breeding. In addition, genotyping thousands to millions of individuals (potentially each year) is a challenge that consequently remains inaccessible for most species, even if the cost efficiency has improved thanks to low coverage genotyping by sequencing (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>) and imputation (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gorjanc <em>et al.</em> 2017a</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">b</a>).</p><p id="p-4">One of the reference GS models is the ridge regression BLUP (RR-BLUP, (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Whittaker 2000</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Meuwissen <em>et al.</em> 2001</a>)) in which a penalized regression is made on all markers simultaneously. This model assumes that the genes affecting the trait of interest are spread across the whole genome and that all of these genes have small effects. Despite its simplicity, this model has been proven one of the most effective in many situations, except for when major genes contribute to trait architecture. Interestingly, this model is equivalent to the genomic BLUP model (G-BLUP, (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Goddard 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhong <em>et al.</em> 2009</a>)) in which markers are used to estimate a realized genomic relationship matrix between individuals, also called kinship. This framework means that we can compress genome-wide information from numerous molecular markers into summary statistics (kinship coefficients between individuals) without diminishing prediction accuracy. Considering this fact, we should ask the question: are there more efficient alternatives than genotyping to estimate the kinship matrix? In the last years, it was proposed to use endophenotypes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mackay <em>et al.</em> 2009</a>) such as transcripts (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Guo <em>et al.</em> 2016</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zenke-Philippi <em>et al.</em> 2017</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Westhues <em>et al.</em> 2017</a>), small RNAs (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Seifert <em>et al.</em> 2018</a>) or metabolites (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feher <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Ward <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fernandez <em>et al.</em> 2016</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Xu <em>et al.</em> 2016</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Guo <em>et al.</em> 2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Schrag <em>et al.</em> 2018</a>) as regressors or to estimate kinship. These endophenotypes correspond to different molecular layers between the genome and the phenotype, which permits the integration of interactions and regulatory networks. These kinds of variables have proven to be efficient to predict integrative traits using the same statistical models as those classically used in GS. These regressors have the advantage of capturing expressed genotypes, but they remain too expensive to be routinely applied on the large scales typically dealt with by breeders. It is interesting to note that even with a small portion of the transcripts or metabolites sampled on a single tissue in a single environment and sometimes at very early stages, it was possible to compute kinship matrices allowing to reach predictive abilities similar to those obtained with molecular markers (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Xu <em>et al.</em> 2016</a>). One could thus consider the possibility of using cheaper and easier techniques to capture endophenotypic variations.</p><p id="p-5">Near-infrared spectroscopy (NIRS) is a high-throughput, non-destructive and low-cost method routinely used to estimate reflectance of a sample for numerous wavelengths. This reflectance is mainly related to the presence of chemical bonds in the analyzed tissue and as a result is expected to be related to endophenotypes. We suppose that the reflectance at each of the numerous wavelengths can be considered as an integration of numerous endophenotypic variations. We thus propose to evaluate the efficiency of NIRS to make predictions with G-BLUP (or equivalently, RR-BLUP) using these traits instead of molecular markers. Numerous studies have demonstrated the usefulness of NIRS for barcoding samples and discriminating species or varieties (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bertrand <em>et al.</em> 1985</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adedipe <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Espinoza <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fischnaller <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abasolo <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">O’Reilly-Wapstra <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Meder <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lang <em>et al.</em> 2017</a>) and have thus suggested that NIRS could be considered as a genetic marker (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cruickshank and Munck 2011</a>). Moreover, some studies have shown that NIRS can capture some genetic variability by estimating the heritability of absorbances along the spectrum and even mapping corresponding quantitative trait loci (QTL, (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Posada <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Diepeveen <em>et al.</em> 2012</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">O’Reilly-Wapstra <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hein and Chaix 2014</a>)). However, to the best of our knowledge, no studies have proposed using NIRS to perform ”phenomic selection” (PS), which we define as the use of high-throughput phenotyping to obtain numerous variables which can be used as regressors or to estimate kinship in the statistical models classically used in GS. We emphasize that the concept of phenomic selection is radically different from the classical use of NIRS prediction. In the classical methodology, NIRS is collected on a sample to make prediction on that particular sample for traits of various complexity (from chemical composition (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Foley <em>et al.</em> 1998</a>) to yield (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ferrio <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cabrera-Bosquet <em>et al.</em> 2012</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Weber <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aguate <em>et al.</em> 2017</a>)) using a formula that has previously been calibrated. On the other hand in PS, NIR reflectances are considered in the same way as genomic or endophenotypic regressors, at the genotypic level rather than the individual level, which allows making predictions in any environment without having any environment specific NIRS. In PS we suppose that once NIR reflectances are analyzed in one experiment (collections of seed, a nursery, a trial or a controlled experiment) they could be used as regressors or to estimate a kinship matrix to make predictions in any other experiment, as long as relevant phenotypic data are available to calibrate the statistical model, like in GS with molecular markers.</p><p id="p-6">There are several advantages to this approach. One can obtain NIRS for any plant or animal species at a lower cost than genotyping and potentially without particular treatment of the samples prior to the analysis such as DNA or RNA extraction. One can also obtain NIRS directly in the field thanks to portable devices (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ecarnot <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Teixeira Dos Santos <em>et al.</em> 2013</a>) or autonomous high-throughput vectors, such as phénomobiles (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Madec <em>et al.</em> 2017</a>) that generate hyperspectral images (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Diago <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Peerbhay <em>et al.</em> 2013</a>). NIRS can even be obtained non destructively on seeds before sowing. As a result, prediction-based selection would be possible for any species and at a low enough cost to make it interesting to implement, even if its results are less accurate than those of GS. As a proof of concept of PS, we report an evaluation of the usefulness of NIRS for predicting quantitative traits of economic interest within two different species, a cereal (winter wheat) and a tree (poplar) using various tissues (grains, leaves, wood) and under different environments, and compare the results to those of a GS prediction based on several thousand SNPs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><div id="sec-3" class="subsection"><h4>Genetic material and experimental designs:</h4><div id="sec-4" class="subsection"><h5>Wheat</h5><p id="p-7">The panel was composed of 228 European elite varieties of winter wheat released between 1977 and 2012, 89% of which have been released since 2000. 72.8% of these varieties are in the panel introduced in <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> (2018)</a>. The full panel was sown in one trial in Clermont-Ferrand (France) in 2015/2016. This trial was an augmented design with two treatments: one drought treatment under rain-out shelters (DRY), and one irrigated treatment (IRR) next to it. There was a difference of 223 mm in water supply (rainfall and irrigation) between the two treatments at the end of the experiment. For both treatments, the panel was divided into eight blocks of earliness with one replicate within the same block for 64 varieties and no replicates for the other 164, except for four checks, which were replicated three times in each block. Phenotypes and NIRS were collected in these two reference environments. A subset of 161 varieties were sown and phenotyped in six independent environments located in Estrées-Mons (France, 2011/2012 and 2012/2013) and Clermont-Ferrand (France, 2012/2013) with two treatments corresponding to two levels of nitrogen input (intermediate and high). This subpanel was divided into six groups of earliness and each group was repeated in two blocks. Four checks were present in each block.</p></div><div id="sec-5" class="subsection"><h5>Poplar</h5><p id="p-8">The population was an association population comprising 1,160 cloned genotypes representative of the natural range of the species in Western Europe and previously described (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Guet <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Faivre-Rampant <em>et al.</em> 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gebreselassie <em>et al.</em> 2017</a>). Clonally replicated trials of subsets of this association population were established in 2008 at two contrasting sites in central France (Orléans, ORL) and Northern Italy (Savigliano, SAV), with 1,098 and 815 genotypes at ORL and SAV respectively. At each site, a randomized complete block design was used with a single tree per block and six replicates per genotype. Growth data collected in each design clearly indicated that the Italian site was more favorable than the French site (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Guet <em>et al.</em> 2015</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gebreselassie <em>et al.</em> 2017</a>).</p></div></div><div id="sec-6" class="subsection"><h4>NIRS data:</h4><div id="sec-7" class="subsection"><h5>Wheat</h5><p id="p-9">NIRS data were obtained on flag leaves and harvested grains from the two treatments of the drought trial in Clermont-Ferrand (France) in 2015/2016. For each variety in each treatment, twenty flag leaves were sampled on one plot at 200 degree days after flowering. The samples were oven dried at 60° for 48 h. Leaves were milled (Falaise miller, SARL Falaise, France), and the powder was analyzed with a FOSS NIRS 6500 (FOSS NIRSystems, Silver Spring, MD) and its corresponding softwares (ISIscan<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-1.gif"/></noscript></span></span> and WINisi<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-2.gif"/></noscript></span></span> 4.20). For each variety in each treatment, 200 g of grains harvested at one plot were analyzed with a FOSS NIRS XDS (FOSS NIRSystems, Silver Spring, MD) and its corresponding softwares (ISIscan<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-3.gif"/></noscript></span></span> and WINisi<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-4.gif"/></noscript></span></span> 4.20). For leaf powder and grain, absorbance was measured from 400 to 2500 nm with a step of 2 nm. 5 varieties were removed from the dataset because their leaf absorbance was abnormal because of a technical problem, resulting in a final panel of 223 varieties. The resulting spectra were loaded into R software (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">R Core Team 2018</a>) to be pretreated using custom R code. They were normalized (centered and scaled) and their first derivative was computed using a Savitzky-Golay filter (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Savitzky and Golay 1964</a>) with a window size of 37 data points (74 nm) implemented in the R package signal (Signal Developers 2014). In the end, each variety in each treatment was characterized by a transformed spectrum (first derivative of the normalized spectrum) of flag leaf powder and a transformed spectrum of grains.</p></div><div id="sec-8" class="subsection"><h5>Poplar</h5><p id="p-10">NIRS was carried out on wood from stem sections collected at 1 m above ground on 2-year-old trees for 1,081 genotypes in three blocks at Orléans (total of 2,860 samples) and 792 genotypes in three blocks at Savigliano (total of 2,254 samples). After harvest, the wood samples were oven dried at 30° for several days, cut into small pieces with a big cutter and milled using a Retsch SM2000 cutting mill (Retsch, Haan, Germany) to pass through a 1-mm sieve. The wood samples were not debarked prior to milling. After stabilization, wood powders were placed into quartz cups for NIR collection with a Spectrum 400 Fourier-transformed spectrometer (Perkin Elmer, Waltham, MA, USA) and its corresponding software (Spectrum<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-5.gif"/></noscript></span></span> 6.3.5). For each sample, the measurement consisted of an average of 64 scans done while rotating the cups over the 10,000 cm<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-6.gif"/></noscript></span></span> - 4,000 cm<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-7.gif"/></noscript></span></span> range with a resolution of 8 cm-1 and a zero-filling factor of 4, resulting in absorbance data every 2 cm<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-8.gif"/></noscript></span></span> The resulting spectra were loaded into R software (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">R Core Team 2018</a>) to be processed using custom R code. They were first restricted to the 8000 cm<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-9.gif"/></noscript></span></span> - 4000 cm<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-10.gif"/></noscript></span></span> range because the most distant part of the spectra (8000 cm<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-11.gif"/></noscript></span></span> - 10,000 cm<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-12.gif"/></noscript></span></span>) appeared to be quite noisy. Then, the restricted spectra were normalized (centered and scaled), and their first derivative was computed using a Savitzky-Golay filter (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Savitzky and Golay 1964</a>) with a window size of 37 data points (74 cm<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-13.gif"/></noscript></span></span>) implemented in the R package signal (Signal Developers 2014). Finally, these normalized and derived spectra were averaged by genotype at each site.</p></div></div><div id="sec-9" class="subsection"><h4>SNP data:</h4><div id="sec-10" class="subsection"><h5>Wheat</h5><p id="p-11">The 228 wheat varieties were genotyped with the TaBW280K high-throughput genotyping array described in <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rimbert <em>et al.</em> (2018)</a>. This array was designed to cover both genic and intergenic regions of the three subgenomes. Markers with a minor allele frequency below 1%, or with a heterozygosity or missing rate above 5% were removed. Groups of identical markers were identified, and for each of them only one marker was kept. Eventually, we obtained 84,259 SNPs, either polymorphic high resolution or off-target variants, with an average missing data rate of 0.83%. Missing values were imputed as the marker frequency.</p></div><div id="sec-11" class="subsection"><h5>Poplar</h5><p id="p-12">The poplar association population was genotyped with an Illumina Infinium BeadChip array (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Faivre-Rampant <em>et al.</em> 2016</a>) yielding 7,918 SNPs for 858 genotypes. Missing values were rare (0.35%) and they were imputed with FImpute (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sargolzaei <em>et al.</em> 2014</a>). The data were restricted to the subset of 562 genotypes with SNP data and NIRS data at both sites. Within this set, SNPs with a minor allele frequency below 1% were discarded, yielding a final SNP dataset of 7,808 SNPs.</p></div></div><div id="sec-12" class="subsection"><h4>Phenotypic data:</h4><div id="sec-13" class="subsection"><h5>Wheat</h5><p id="p-13">The 228 wheat varieties were phenotyped for heading date (HD) and grain yield (GY) at the two environments in which the NIRS analysis was conducted (drought experiment in Clermont-Ferrand 2015/2016). The subpanel of 161 varieties was phenotyped for the same traits in six independent environments. In each environment, the phenotypic data were adjusted for micro-environmental effects using the random effect block and when necessary by modeling spatial trends using two-dimensional penalized spline (P-spline) models as implemented in the R package SpATS (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rodríguez-Álvarez <em>et al.</em> 2017</a>). Broad-sense heritabilities were computed following <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Oakey <em>et al.</em> (2007)</a>.</p></div><div id="sec-14" class="subsection"><h5>Poplar</h5><p id="p-14">The poplar association population was evaluated at each of the two sites for the following traits on up to six replicates by genotype: height at 2 years at Orléans (HT-ORL), circumference at 1 m above ground at 2 years at both sites (CIRC-ORL and CIRC-SAV), bud flush at both sites (BF-ORL and BF-SAV) and bud set at both sites (BS-ORL and BS-SAV) as discrete scores for a given day of the year (see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dillen <em>et al.</em> (2009)</a> and <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rohde <em>et al.</em> (2011)</a> for details on the scales used) and resistance to rust at Orléans (RUST-ORL) as a discrete score of susceptibility on the most affected leaf of the tree and on a 1 to 8 scale. Within each site, the phenotypic data were adjusted for micro-environmental effects using random effect block and/or spatial position when needed following a visual inspection of spatial effects with a variogram as implemented in the R package breedR (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Muñoz and Sanchez 2017</a>). Finally, the adjusted phenotypes were restricted to the subset of 562 genotypes with SNP and NIRS data for computing an averaged genotypic value for each trait by genotype within each site for further analyses.</p></div></div></div><div id="sec-15" class="subsection"><h3>Genomic heritability and partition of variance along spectra</h3><p id="p-15">The estimation of genomic heritability was based on the following bivariate statistical model across environments:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-16.gif"/></noscript></span></span> are the phenotypic values (absorbance for a given wavelength) in each environment, <em>β</em> is a vector of fixed environment effect, u is a vector of random polygenic effect with <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-17.gif"/></noscript></span></span> K being the scaled realized relationship matrix (see below), e is a vector of independent and normally distributed residuals with <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-18.gif"/></noscript></span></span> X and Z are design matrices relating observations to the effects. SNPs were used to estimate the genomic relationship matrix (A) between individuals, following the formula of VanRaden (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">VanRaden 2008</a>)<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-21.gif"/></noscript></span></span> are the genotypes of individuals <em>i</em> and <em>j</em> at marker <em>l</em> (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-22.gif"/></noscript></span></span> or 1 for homozygotes, 0.5 for heterozygotes), <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-23.gif"/></noscript></span></span> is the frequency of the allele coded 1 for the marker <em>l</em>, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-24.gif"/></noscript></span></span> is the average empirical marker genotype variance. K was obtained by scaling A to have a sample variance of 1 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kang <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Forni <em>et al.</em> 2011</a>). Genomic heritability was estimated for each wavelength within each environment (<em>m</em>) as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-27.gif"/></noscript></span></span> the REML estimates of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-29.gif"/></noscript></span></span> obtained with the Newton-Raphson algorithm implemented in the R package sommer (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Covarrubias-Pazaran 2016</a>). Following <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yamada <em>et al.</em> (1988)</a>, the variance/covariance estimates from the previously defined bivariate mixed-model were used to compute estimates of genetic (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-30.gif"/></noscript></span></span>), genetic by environment (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-31.gif"/></noscript></span></span>) and residual (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-32.gif"/></noscript></span></span>) variances across sites as follows: <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-33.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-35.gif"/></noscript></span></span></p></div><div id="sec-16" class="subsection"><h3>Association mapping on NIRS absorbance</h3><p id="p-16">Association mapping was carried out along spectra considering the absorbance at a given wavelength as a bivariate trait (corresponding to the two environments) and using previous estimates of genetic and residual variances (EMMAX philosophy as previously proposed in the multi-trait mixed-model approach (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Korte <em>et al.</em> 2012</a>)).</p></div><div id="sec-17" class="subsection"><h3>Phenotype prediction with genomic and phenomic information</h3><p id="p-17">The efficiency of genomic and phenomic predictions was evaluated by cross-validations in two types of scenarios (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In scenario S1, NIRS analysis and cross-validation were applied to the same environment (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1 a</a>). In scenario S2, cross-validation was applied to independent environments: the environment(s) in which NIRS was collected and the environment in which the cross-validation was applied (calibration and prediction) were different (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1 b</a>). In S1, the objective was to limit expensive or labor-demanding phenotyping to a calibration set of reduced size and to predict the remaining individuals using NIRS. In scenario S2, one experiment (or a nursery) was dedicated to collecting the NIRS of the calibration set and the predicted set, and a multi-environment trial was dedicated to phenotyping the calibration set. The main difference between S1 and S2 was that in S1, we can expect NIRS to be more related to the phenotypic data than in S2.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the concept of phenomic selection, including the two scenarios tested in the present work: (a) S1, where the calibration model is trained with true values (TVs) and NIRS data collected at the same - reference - site and (b) S2, where the calibration model is trained with NIRS data collected at the reference site and TVs from other(s) environment(s). In both scenarios, the outcome of the prediction consists of predicted values (PVs)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2022872062" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of the concept of phenomic selection, including the two scenarios tested in the present work: (a) S1, where the calibration model is trained with true values (TVs) and NIRS data collected at the same - reference - site and (b) S2, where the calibration model is trained with NIRS data collected at the reference site and TVs from other(s) environment(s). In both scenarios, the outcome of the prediction consists of predicted values (PVs).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3961/F1.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3961/F1.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491703" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Schematic representation of the concept of phenomic selection, including the two scenarios tested in the present work: (a) S1, where the calibration model is trained with true values (TVs) and NIRS data collected at the same - reference - site and (b) S2, where the calibration model is trained with NIRS data collected at the reference site and TVs from other(s) environment(s). In both scenarios, the outcome of the prediction consists of predicted values (PVs).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">For both scenarios, 5- and 8-fold cross-validation procedures repeated 20 times were used for poplar and wheat, respectively. A larger fold-number was considered for wheat in comparison to poplar because the sample size in the wheat dataset (n = 223 in the panel, and n = 161 in the subpanel) was lower than the sample size in the poplar dataset (n = 562). We would like to emphasize that in both scenarios, the predicted set was only characterized by genotypic and NIRS (obtained in the reference site) data. In particular in scenario S2, the only information available from the predicted environment is the phenotypic data of the calibration set (Figure S1). Predictive ability was computed as the Pearson correlation between the predictions and adjusted means. For genomic predictions, we tested two complementary reference models: G-BLUP and Bayesian LASSO (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Park and Casella 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>). The underlying assumptions of these two models are that the SNP effects are normally distributed for G-BLUP, whereas Bayesian LASSO allows for departure from normality (<em>i.e.</em>, SNPs with bigger effects). G-BLUP and Bayesian LASSO were run with the R packages rrBLUP (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Endelman 2011</a>) and BGLR (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>), respectively. For Bayesian LASSO, the chain was composed of 30,000 iterations with a burn-in of 5,000 iterations, and the hyperparameter λ was chosen as recommended in <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> (2013)</a>. For phenomic predictions, we used RR-BLUP but considered NIRS data instead of molecular markers. Prior to the analysis, the pretreated NIRS matrices were centered and scaled for each wavelength. For wheat, we also tested to include the spectrum of the two tissues (or of the same tissue but collected in the two environments) in the statistical model. For this we simply joined the two matrices of spectrum into one single matrix.</p></div><div id="sec-18" class="subsection"><h3>Expected genetic gain with genomic and phenomic selection in a simple example</h3><p id="p-20">We ran simulations to illustrate the expected genetic gain with GS and PS that would be achieved in one cycle of selection for various combinations of costs and reliabilities. Reliability was defined as the squared correlation between true breeding values (TBV) and the genomic or NIRS predicted values (PV).</p><p id="p-21">We considered a situation in which a given budget (200,000 €) was available to predict the performances of selection candidates with NIRS or genotyping. Depending on the costs of the methods (DNA extraction and genotyping for GS or tissue sampling and NIRS acquisition for PS), we computed the number of selection candidates (N) that could be analyzed. The TBV and genomic or NIRS PV of these N individuals were then sampled from a multivariate normal distribution with means equal to 0, variances equal to 1 and covariance equal to the square root of reliability (R package mvtnorm (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Genz <em>et al.</em> 2018</a>)). The expected genetic gain was then computed as the difference between the average TBV of the 400 individuals having the best PV and the average TBV of the population (equal to 0). We selected 400 individuals because for many species, it is feasible to apply heavier phenotyping (multi-environment trials) on a few hundred individuals. We considered two situations; in the first situation, the expected genetic gain of GS and PS was computed for various genotyping and NIRS costs with a reliability set to 0.4. In the second situation, the reliability of GS and PS varied between 0.3 and 0.6, and genotyping and NIRS costs were set to 50 € and 4 €, respectively. For each combination of parameters (reliabilities and costs of GS and PS), the simulation procedure was repeated 1000 times to obtain stable results. Because genotyping and NIRS costs are highly dependent on the species and the number of samples analyzed, we let the genotyping costs (DNA extraction and genotyping itself) vary between 25 € and 100 € and the NIRS costs (sample treatment and NIRS analysis itself) vary between 1 € and 8 € in the first situation.</p><p id="p-22">To provide concrete examples, we applied this simulation process with the reliabilities and costs that we experienced for wheat and poplar. GS costs were between 35 € and 50 € per individual for wheat and poplar, respectively, and PS costs were between 3 € and 2.5 € per individual for wheat and poplar, respectively. Reliabilities were estimated as the square of predictive abilities estimated by cross-validation divided by the heritability of the adjusted means. For each combination of trait, scenario, and NIRS data considered (tissue, environment), the increase in expected genetic gain using PS instead of GS was computed with the best performing GS model as a reference.</p></div><div id="sec-19" class="subsection"><h3>Data Availability</h3><p id="p-23">The datasets generated during and/or analyzed during the current study are available in the INRA Dataverse repository (<a href="https://data.inra.fr/">https://data.inra.fr/</a>). They can be accessed with the following link <a href="http://dx.doi.org/10.15454/MB4G3T">http://dx.doi.org/10.15454/MB4G3T</a>. R functions used for comparing the predictive ability of SNP and NIRS through cross-validations have been deposited on github (<a href="https://github.com/visegura/PS">https://github.com/visegura/PS</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7243256">https://doi.org/10.25387/g3.7243256</a>.</p></div></div><div class="section results" id="sec-20"><h2 class="">RESULTS</h2><div id="sec-21" class="subsection"><h3>Genomic heritability and partition of variance along NIRS</h3><p id="p-24">We first sought to characterize the ability of NIRS to capture genetic variability by estimating genomic heritability and partitioning the variance into genetic (<em>G</em>), genetic by environment (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-36.gif"/></noscript></span></span>) and residual variances (<em>ϵ</em>) along the NIR spectrum collected on a panel of winter wheat (leaves and grains) and a population of black poplar (wood) grown in two contrasting environments. For both species and tissues, genomic heritability was highly variable along the spectrum with peaks above 60%, showing the existence of strong polygenic signals for some wavelengths (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Figure S2, and File S1). For a given species, the proportion of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-37.gif"/></noscript></span></span> variance was on average across all the wavelengths equal to 8% (poplar), 10% (wheat leaves) or 16% (wheat grains) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and File S1). It is interesting to note that for at least half of the wavelengths, the cumulative proportion of <em>G</em> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-38.gif"/></noscript></span></span> variances was above 15%, showing that the NIR signal was often partially related to genetics. The kind of tissue analyzed by NIRS seemed to matter, as shown by the comparison of variance partition along spectra obtained on wheat leaves and grains. <em>G</em> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-39.gif"/></noscript></span></span> variances were higher and more stable along the spectrum for grains than for leaves.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of genetic (red), genetic by environment (green), and residual (blue) variances along the NIR spectrum of (a) winter wheat leaves, (b) winter wheat grains and (c) poplar wood. NIRS was performed on plant material collected on genotypes grown under favorable and unfavorable environmental conditions. The median normalized and derived spectra, along with their first and third quartiles across the genotypes under study, are indicated in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2022872062" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of genetic (red), genetic by environment (green), and residual (blue) variances along the NIR spectrum of (a) winter wheat leaves, (b) winter wheat grains and (c) poplar wood. NIRS was performed on plant material collected on genotypes grown under favorable and unfavorable environmental conditions. The median normalized and derived spectra, along with their first and third quartiles across the genotypes under study, are indicated in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3961/F2.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3961/F2.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491715" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Proportion of genetic (red), genetic by environment (green), and residual (blue) variances along the NIR spectrum of (a) winter wheat leaves, (b) winter wheat grains and (c) poplar wood. NIRS was performed on plant material collected on genotypes grown under favorable and unfavorable environmental conditions. The median normalized and derived spectra, along with their first and third quartiles across the genotypes under study, are indicated in gray.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We ran association mapping along the NIR spectrum to identify wavelengths associated with major QTL (Figure S2). In poplar, the signal appeared to be mainly polygenic with very few QTL detected, and the largest SNP <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-40.gif"/></noscript></span></span> was below 0.025 for any wavelength. In contrast, in winter wheat, we detected numerous large-effect QTL. For some wavelengths, a single SNP could have an <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-41.gif"/></noscript></span></span> of 0.23 for leaves and of 0.11 for grains, and this SNP could be in spectrum regions of high or of low genomic heritability (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). This finding means that depending on the wavelength, NIRS could capture highly polygenic relationships (wavelengths with high genomic heritability) or could tag specific regions of the genome (major QTL). These two kinds of wavelengths can be useful for making predictions because they can potentially track the two main factors responsible for GS accuracy: relatedness and linkage disequilibrium.</p></div><div id="sec-22" class="subsection"><h3>Comparing predictive abilities obtained with markers and with NIRS</h3><p id="p-27">We estimated the efficiency of GS and PS to predict the performance of new individuals within a cross-validation framework. The performances of the individuals in the validation set were predicted with genotypic information in GS (G-BLUP and Bayesian LASSO models) and with NIRS only in PS (RR-BLUP model). We considered two scenarios: in S1, NIRS analysis and cross-validation were performed in the same environment (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1 a</a>), whereas in S2, the environments in which the cross-validation was applied were different from those in which NIRS was obtained (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1 b</a>). The broad-sense heritabilities of the adjusted means were above 0.8 for all traits in each environment (Table S1).</p><p id="p-28">In wheat, the predictive abilities of PS were highly variable and appeared to be dependent on the predicted trait and on the environment and tissue in which NIRS was measured (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3 a, b, c, d</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). While combining NIRS collected in different environments or different tissues increased the predictive ability, this increase did not occur systematically. One major result is that for both traits, NIRS could lead to better predictions than molecular markers, even in the six independent environments (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3 c, d</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The gain with NIRS in comparison to molecular markers in S2 was up to 34% and 22% for heading date and grain yield, respectively. In each S2 environment and for both traits, there was always a type of NIRS that performed better or as well as the best GS model (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The gain was even stronger in S1: NIRS led to an increase in predictive ability of up to 53% and 117% for heading date and grain yield, respectively. In poplar, the predictive abilities with NIRS were always lower than those with SNP, except for growth traits under S1 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3 e</a>). In the other cases, the predictive ability with NIRS varied depending on the trait and scenario considered, but they were always significantly greater than 0. In general, they were higher when the spectra were collected in the same environment (S1) than when spectra from another environment were used (S2), except for bud flush evaluated in one site and bud set evaluated in another site. Interestingly, irrespectively of the scenario, for some traits apparently unrelated to wood chemical properties, such as resistance to rust or bud set, NIRS predictive abilities were fairly high ranging between 0.34 and 0.53.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive ability of SNP (G-BLUP or Bayesian LASSO (BL) models) or NIRS (RR-BLUP model) when predicting the phenotypic values of individuals within a cross-validation in winter wheat (a, b, c, d) and black poplar (e). Two scenarios were considered for NIRS prediction: in S1, the RR-BLUP model was trained with NIRS data and phenotypes that were collected within the same environment (a, b, e), whereas in S2, NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments (c, d, e). For wheat, two traits were considered: heading date (a, c) and grain yield (b, d). The bars of a, b, c, and d are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment), and the bars of e are labeled with the combination of trait and experiment (HT: height, CIRC: circumference, BF: bud flush, BS: bud set, RUST: resistance to rust, ORL: experimental design in Orl&#xE9;ans, France, SAV: experimental design in Savigliano, Italy). For S1 in wheat (a, b), PS was done with NIRS collected in the environment in which the phenotypes were collected (D or I), or with NIRS collected in both environments (D &amp; I). For S2 in wheat, each boxplot represents the summary statistics over all the accuracies obtained in the six S2 environments with a given type of NIRS (tissue and environment). The medians of the accuracies obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2022872062" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predictive ability of SNP (G-BLUP or Bayesian LASSO (BL) models) or NIRS (RR-BLUP model) when predicting the phenotypic values of individuals within a cross-validation in winter wheat (a, b, c, d) and black poplar (e). Two scenarios were considered for NIRS prediction: in S1, the RR-BLUP model was trained with NIRS data and phenotypes that were collected within the same environment (a, b, e), whereas in S2, NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments (c, d, e). For wheat, two traits were considered: heading date (a, c) and grain yield (b, d). The bars of a, b, c, and d are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment), and the bars of e are labeled with the combination of trait and experiment (HT: height, CIRC: circumference, BF: bud flush, BS: bud set, RUST: resistance to rust, ORL: experimental design in Orl&#xE9;ans, France, SAV: experimental design in Savigliano, Italy). For S1 in wheat (a, b), PS was done with NIRS collected in the environment in which the phenotypes were collected (D or I), or with NIRS collected in both environments (D &amp; I). For S2 in wheat, each boxplot represents the summary statistics over all the accuracies obtained in the six S2 environments with a given type of NIRS (tissue and environment). The medians of the accuracies obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3961/F3.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3961/F3.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491717" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Predictive ability of SNP (G-BLUP or Bayesian LASSO (BL) models) or NIRS (RR-BLUP model) when predicting the phenotypic values of individuals within a cross-validation in winter wheat (a, b, c, d) and black poplar (e). Two scenarios were considered for NIRS prediction: in S1, the RR-BLUP model was trained with NIRS data and phenotypes that were collected within the same environment (a, b, e), whereas in S2, NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments (c, d, e). For wheat, two traits were considered: heading date (a, c) and grain yield (b, d). The bars of a, b, c, and d are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment), and the bars of e are labeled with the combination of trait and experiment (HT: height, CIRC: circumference, BF: bud flush, BS: bud set, RUST: resistance to rust, ORL: experimental design in Orléans, France, SAV: experimental design in Savigliano, Italy). For S1 in wheat (a, b), PS was done with NIRS collected in the environment in which the phenotypes were collected (D or I), or with NIRS collected in both environments (D &amp; I). For S2 in wheat, each boxplot represents the summary statistics over all the accuracies obtained in the six S2 environments with a given type of NIRS (tissue and environment). The medians of the accuracies obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Details of the predictive abilities obtained in scenario S2 for heading date (a) and grain yield (b) for wheat. In S2, the NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments. The bars are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment). The medians of the predictive abilities obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2022872062" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Details of the predictive abilities obtained in scenario S2 for heading date (a) and grain yield (b) for wheat. In S2, the NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments. The bars are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment). The medians of the predictive abilities obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3961/F4.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3961/F4.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491708" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Details of the predictive abilities obtained in scenario S2 for heading date (a) and grain yield (b) for wheat. In S2, the NIRS and phenotypic data used to train the RR-BLUP model were collected in distinct environments. The bars are labeled with the origin of the NIRS data (I: irrigated treatment, D: drought treatment). The medians of the predictive abilities obtained over repeated cross-validations are reported as the height of the bars together with the first and third quartiles as confidence intervals.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Expected genetic gain with genomic and phenomic selection in a simple example</h3><p id="p-31">To further evaluate the potential of PS with respect to GS, the expected genetic gain with both approaches was compared in a simple scenario in which a budget of 200,000 € could be spent to genotype or analyze the NIRS of selection candidates. The difference in efficiency between GS and PS was highly dependent on the genotyping and NIRS costs and on the reliability of the two approaches (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In the scenarios that we considered here, the expected gain of using PS instead of GS was between 11% and 127%. In extreme scenarios in which genotyping was cheap (25 €) and NIRS was expensive (8 €) or in which GS reliability (0.6) was much higher than PS reliability (0.3), PS was still better than GS. We applied the simulation process with the reliabilities and costs obtained in the wheat example (35 € for genotyping and DNA extraction and 3 € for sample treatment and NIRS acquisition). The increase of expected genetic gain with PS in comparison to GS was between +60% and +127% for heading date and between -10% and +222% for grain yield, depending on the tissue and environment used for NIRS acquisition and scenario considered (Table S2). In poplar, considering genotyping and NIRS acquisition costs of 50 € and 2.5 €, respectively, as well as the reliabilities estimated with cross-validation predictive abilities, the expected gain in genetic progress varied depending on the trait and scenario considered (Table S3). It was mainly positive for growth traits (-2–93%), bud set (-6–25%) and rust resistance (-10–21%), whereas for bud flush, NIRS prediction did not seem to provide any advantage over regular SNP-based prediction.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Theoretical increase of expected genetic gain (%) by using NIRS instead of genotyping. a: Expected genetic gain for various genotyping and NIRS costs for a reliability of 0.4, a budget of 200,000 &#x20AC;, and a selection of 400 individuals. b: Expected genetic gain for various reliabilities, a budget of 200,000 &#x20AC;, genotyping and NIRS costs of 50 &#x20AC; and 4 &#x20AC;, respectively, and a selection of 400 individuals. For each scenario, true breeding values and estimated breeding values were simulated thanks to multivariate normal distributions with a covariance adapted to the chosen reliability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2022872062" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Theoretical increase of expected genetic gain (%) by using NIRS instead of genotyping. a: Expected genetic gain for various genotyping and NIRS costs for a reliability of 0.4, a budget of 200,000 &#x20AC;, and a selection of 400 individuals. b: Expected genetic gain for various reliabilities, a budget of 200,000 &#x20AC;, genotyping and NIRS costs of 50 &#x20AC; and 4 &#x20AC;, respectively, and a selection of 400 individuals. For each scenario, true breeding values and estimated breeding values were simulated thanks to multivariate normal distributions with a covariance adapted to the chosen reliability.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3961/F5.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3961/F5.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3961/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491711" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Theoretical increase of expected genetic gain (%) by using NIRS instead of genotyping. a: Expected genetic gain for various genotyping and NIRS costs for a reliability of 0.4, a budget of 200,000 €, and a selection of 400 individuals. b: Expected genetic gain for various reliabilities, a budget of 200,000 €, genotyping and NIRS costs of 50 € and 4 €, respectively, and a selection of 400 individuals. For each scenario, true breeding values and estimated breeding values were simulated thanks to multivariate normal distributions with a covariance adapted to the chosen reliability.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-33">In typical plant breeding programs, breeders have to select among thousands to millions of individuals. For most individuals, this selection is often based on a very small amount of phenotypic information because it is too expensive or simply impossible to make a precise phenotypic evaluation. It is also difficult and too expensive to genotype all individuals to apply GS, despite important economies of scales. Alternative approaches based on endophenotypes such as transcriptomes or metabolomes have been proposed to predict phenotypes (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Feher <em>et al.</em> 2014</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Ward <em>et al.</em> 2015</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fernandez <em>et al.</em> 2016</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Guo <em>et al.</em> 2016</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Xu <em>et al.</em> 2016</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Zenke-Philippi <em>et al.</em> 2017</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Westhues <em>et al.</em> 2017</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Seifert <em>et al.</em> 2018</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Schrag <em>et al.</em> 2018</a>), but their relatively low throughput and high costs are still likely to hamper their deployment at a large scale. To increase genetic progress in this context, we propose a new approach in which we use NIRS as high-throughput phenotypes to make predictions at low costs. The basic idea of this approach, which we call ”phenomic selection” (PS), is that the absorbance of a sample in the near-infrared range is mainly related to its chemical composition, which depends itself on endophenotypes and genetics. Therefore, NIRS is supposed to capture at least part of the genetic variance, and as a result, one could use it to make predictions of traits unrelated to the analyzed tissue or in independent environments. The process of PS is similar to GS, but instead of reference material and selection candidates being genotyped, they are analyzed by NIRS.</p><p id="p-34">We applied PS to the NIR spectrum of different tissues sampled on an association population of poplar and a panel of elite winter wheat. By estimating the extent of genetic variance along the NIR spectrum of poplar wood and winter wheat leaves and grains, we could show that most wavelengths displayed genetic variability (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). This result agrees with previous findings with eucalyptus wood (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Hein and Chaix 2014</a>), but whether this will still be true within pedigrees with a narrower genetic basis remains to be assessed. <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">O’Reilly-Wapstra <em>et al.</em> (2013)</a> have shown that NIR spectra collected on eucalyptus leaves could differentiate full-sibs, even though the extent of genetic variation captured was lower than at the inter-specific level. Still these results suggest that NIRS could be valuable to capture some Mendelian sampling and that PS would work within pedigrees, but this hypothesis should clearly be tested in future work.</p><p id="p-35">In the present work, the NIR spectra were specific to the environments in which they were obtained, but when they were analyzed jointly, we observed that <em>G</em> variance was larger than <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-42.gif"/></noscript></span></span> variance for most wavelengths in both species. <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Posada <em>et al.</em> (2009)</a> also reported a similar trend with coffee grains. This finding shows that even if the absorbances were partly environment specific, it should be possible to make predictions in independent environments. This result was further demonstrated by the good predictive abilities obtained with PS for most phenotypes in both species in scenario S2, <em>i.e.</em>, when the environment in which we trained the calibration model was different from the environment in which we collected NIRS. For both species, PS abilities were in the same range as GS abilities, sometimes performing better and sometimes performing worse than one another. For wheat, the results were very encouraging as we always found a situation (combination of environment and tissue analyzed) for which NIRS performed better than GS, even in six independent environments. More importantly, even when the correlation between the S1 and S2 environment was as low as 0.16 for the predicted trait (Table S4, GY in Mon12N-), PS could produce better predictions than GS (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). In other words, a prediction model based on NIRS obtained in one specific environment could be used to make predictions in completely different environments. These promising results obtained in scenarios S1 and S2 open the way to important opportunities in the plant breeding community. As revealed by our theoretical computations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), we expect PS to be able to generate large gains in genetic progress in comparison to GS, even in pessimistic scenarios. In the realistic scenarios that we experienced, the expected gain brought by using PS instead of GS could be up to 81% for wheat grain yield in scenario S2 (Table S2).</p><p id="p-36">Nevertheless, these simulations have to be considered with caution, because of the strength of the underlying hypotheses. Our work has shown the interest of the proposed PS approach within a given generation that may clearly be applicable within plant breeding programs to assess the performance of the candidate for selection. But, more work is clearly needed to establish the proportion of the variance along NIRS (and for endophenotypes) that is heritable in the narrow-sense and thus transmitted to the next generations to be further used at different stages of the breeding programs or in different breeding contexts. Indeed, similarly to endophenotypes we expect NIRS to capture non-additive genetic effects which may overestimate the expected genetic progress across multiple generations. Nevertheless on the other side it can be highly valuable to predict phenotypes, including the effect of interactions and regulatory networks, at key steps of the breeding schemes. In plant breeding, one major objective during the first few generations is to produce numerous individuals with the same genotypes (by self-fertilization, doubled haploid techniques or clonal reproduction) to allow for field evaluation in multi-environment trial (MET). Because this field evaluation is the most expensive step in the breeding schemes and because it is applied on replicable genotypes, PS would be of major interest to select among all candidates the genotypes that will be evaluated in the MET. In this situation, predicting phenotypes instead of additive values is clearly an advantage as the same genotypes can be replicated many times. Another important question related to the efficiency of PS across multiple generations is about the frequency of formula update to maintain a sufficient level of accuracy. This question, which is also relevant for GS, must be addressed in future work. We also believe that future work should assess the efficiency of PS with NIRS obtained from tissues collected on young plants. Ideally, we would like PS to be efficient with NIRS collected on the youngest possible plant to have the information as early as possible (similarly to what can be achieved with GS) and at low cost. We could show for wheat that for fixed material, NIRS collected on seeds, so before sowing, was efficient to run PS, which offers very interesting perspectives for this species. The studies on endophenotypic variations in maize (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Guo <em>et al.</em> 2016</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Schrag <em>et al.</em> 2018</a>), rice (<a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Xu <em>et al.</em> 2016</a>) and wheat (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Ward <em>et al.</em> 2015</a>) also demonstrated that the characterization of germinated seeds or seedlings was efficient to estimate kinships resulting in accurate predictions. These results are promising, but this needs to be tested for other species and on other datasets. The choice of the tissue on which to collect NIRS is also important in regard to the amount of work required. It is clear that it would be much easier to take NIRS on seeds in the lab than on individual leaves in the field. Further work is necessary to evaluate the feasibility of PS in breeding schemes. One last important point that needs to be addressed in future work, is the way NIRS collected in different environments should be combined to run PS. In the scenarios that we described here, the NIRS of the calibration set and the predicted set were obtained in a same environment. But in practice, breeders collect NIRS each year on different materials, and so it would be necessary to combine NIRS obtained in different environments, which could reduce the prediction accuracy of PS. There are practical ways (such as repeating checks each year, or implementing management practices to homogenize the environments) and theoretical ways (such as Single Step GBLUP (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Legarra <em>et al.</em> 2009</a>)) to deal with this issue, but these need to be tested.</p><p id="p-37">There are various applications of PS, which we see both as a complement and as an alternative to GS depending on the situation. The first obvious application of PS is its use when no genotyping tool is available at a reasonable cost, which is still the case for many orphan organisms, even if important progress has been achieved thanks to genotyping by sequencing that makes genotyping more accessible and cost effective (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Gorjanc <em>et al.</em> 2017a</a>). For these species, PS could potentially be a new efficient breeding tool to increase genetic progress. As mentioned before, a second application would be to use PS to screen nearly fixed material or clones, as PS (in the same manner as selection on endophenotypes) is likely to capture non-additive genetic effects. Even if the prediction accuracy is low, PS can be used to filter out a given proportion of selection candidates. One should define this proportion with respect to PS accuracy: the higher the accuracy, the more confident we are at filtering out many individuals without losing the best candidates. Note that even if PS is less accurate than GS, it could nevertheless be interesting to filter out the worst individuals considering the low cost of NIRS acquisition, and the fact that NIRS is often already routinely carried out (for example, in cereals or forest trees to predict quality traits). In a second step, one could use GS to make complementary predictions on a limited number of selection candidates. A last major application of PS would be to help conservation geneticists manage diversity collections. The use of genotyping to organize seed banks and to screen and define core collections is strongly limited by its cost. PS offers a new opportunity to manage seed banks because it allows distance matrices to be computed cheaply and reliably.</p><p id="p-38">Considering that PS gave interesting results for both a tree and an annual crop regarding various traits related to development, productivity and tolerance to disease and using tissues of a completely different nature (wood, leaf, grain), we can expect PS to work in many other plants and possibly in animal species using NIRS on organic tissues or fluids. Our work constitutes a proof of concept and a first attempt at PS, which clearly opens new perspectives for the breeding community. Indeed, one could further optimize many parameters to increase PS efficiency. The differences observed here between the PS efficiencies reported for wheat and poplar could represent a first direction for improving the approach. Indeed, PS appeared to be more efficient in wheat than in poplar and several hypotheses could be proposed to explain this result. First, spectra were acquired on different spectrometers resulting in a broader wavelength range in wheat, which also covered the visible part of the electromagnetic spectrum. Consequently, the information brought by the spectra on wheat tissues was potentially richer than the one brought by the spectra on poplar. Second, we could see that in wheat a larger proportion of <em>G</em> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3961/embed/mml-math-43.gif"/></noscript></span></span> variance could be captured by the spectra regardless of the tissue sampled and that this was especially true for the lowest wavelengths (including the visible part), which were absent in poplar. Third, the tissues in which NIRS was collected differed, and this difference seems to be an important parameter as highlighted by the differences in predictive ability between leaf and grain in wheat.</p><p id="p-39">Another possibility for the improvement of PS efficiency could be the optimization of the growing conditions of plants in the reference experiment. In wheat, it was typically better to use NIRS collected on plants grown in unfavorable conditions than in favorable conditions. This result might be explained by more pronounced dissimilarities between genetically distant individuals in conditions of stress. Therefore, there is a clear need to optimize these conditions. Once the NIRS data are collected, one could also try to improve the pretreatment of the signal and the statistical model of calibration. In our case, we choose as pretreatment the first derivative of the normalized spectrum, but other options could be tested, and these options might not necessarily be the same depending on the species considered, environment, tissue sampled or target trait. For calibrations, we have used RR-BLUP, but one might test other techniques, such as those typically allowing non-additive effects or involving feature selection, to improve the accuracy of PS. These points clearly indicate that there is great room of improvement of PS, which will likely constitute in the near future an active field of research. Finally, the recent advent of portable NIR devices as well as of hyperspectral imaging allows this technology to be used in the field. Unmanned vehicles and robots are currently being developed and can already be used to automatically collect reflectance at an industrial scale (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Madec <em>et al.</em> 2017</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Aguate <em>et al.</em> 2017</a>). These new developments will considerably increase the throughput and conversely decrease the cost of NIRS data. We thus expect that these technological advances will reinforce the advantages of the proposed PS. However, whether these technological advances will have the same predictive ability as NIRS remains to be tested and is likely to be the subject of active research in the near future.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors gratefully acknowledge the staff of the INRA GBFOR experimental unit for the establishment and management of the poplar experimental design in Orléans, the collection of wood samples in each site, and their contribution to phenotypic measurements on poplars in Orléans; Alasia Franco Vivai staff for management of the poplar experimental plantation in Savigliano, and M. Sabatti and F. Fabbrini for their contribution to phenotypic measurements on poplars in Savigliano. We acknowledge the staff of the INRA GenoBois platform for the preparation of samples and collection of NIRS on wood samples and the staff of EPGV and BioForA for their contribution to obtaining SNP data on poplar. We would like to thank J. Messaoud for NIRS acquisition on wheat samples, V. Allard, B. Adam and D. Cormier for implementation of the rain-out shelter experiment (Phéno3C, INRA Clermont-Ferrand), and E. Heumez (UE GCIE) for the experiment in Estrées-Mons. We would also like to thank A. Chateigner, L. Sanchez, G. Charmet, V. Allard, P. Martre, L. Inchboard and S. Bouchet for useful discussions and comments on the manuscript. We are grateful to the Genotoul bioinformatics platform Toulouse Midi-Pyrenees (Bioinfo Genotoul) for providing computing resources. Establishment and management of the poplar experimental sites until harvests were carried out with financial support from the NOVELTREE project (EU-FP7-211868). NIRS measurements on poplar wood samples were supported by the SYBIOPOP project funded by the French National Research Agency (ANR-13-JSV6-0001). Management of the wheat multi-environment trials was financially supported by the French National Research National Agency under Investment for the Future (BreedWheat project ANR-10-BTBR-03) and by FranceAgriMer. The Phéno3C platform was financially funded by the French National Research National Agency under the Investment for the Future Phenome project (ANR-11-INBS-12) and by the European Regional Development Fund (AV0011535).</p><p id="p-41">V.S. and R.R. designed the study, analyzed the data and wrote the paper with input from J-P.C., P.F.R, E.P., J.L.G., and C.B. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7243256">https://doi.org/10.25387/g3.7243256</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-26"><span class="received-label">Received </span>September 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-20"><span class="accepted-label">Accepted </span>October 20, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Rincent <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.1" data-doi="10.1016/j.foreco.2013.04.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abasolo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deviant near-infrared spectra identifies Corymbia hybrids.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.foreco.2013.04.040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D304%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2013.04.040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2013.04.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.2" data-doi="10.1255/jnirs.760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adedipe</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson-Andoh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slahor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Classification of red oak (Quercus rubra) and white oak (Quercus alba) wood using a near infrared spectrometer and soft independent modelling of class analogies.</span> <abbr class="cit-jnl-abbrev">J. Near Infrared Spectrosc.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1255/jnirs.760</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNear%2BInfrared%2BSpectrosc.%26rft.volume%253D16%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.3" data-doi="10.2135/cropsci2017.01.0007."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguate</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trachsel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Use of Hyperspectral Image Data Outperforms Vegetation Indices in Prediction of Maize Yield.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">2517</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.01.0007.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D2517%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.01.0007.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.01.0007.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.4" data-doi="10.1002/jsfa.2740361114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loisel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Identification of some wheat varieties by near infrared reflectance spectroscopy.</span> <abbr class="cit-jnl-abbrev">J. Sci. Food Agric.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1120</span>–<span class="cit-lpage">1124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jsfa.2740361114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSci.%2BFood%2BAgric.%26rft.volume%253D36%26rft.spage%253D1120%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjsfa.2740361114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jsfa.2740361114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.5" data-doi="10.1111/j.1744-7909.2012.01116.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serret</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luis Araus</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-throughput Phenotyping and Genomic Selection: The Frontiers of Crop Breeding ConvergeF.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7909.2012.01116.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DCabrera-Bosquet%26rft.auinit1%253DL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DHigh-throughput%2Bphenotyping%2Band%2Bgenomic%2Bselection%253A%2Bthe%2Bfrontiers%2Bof%2Bcrop%2Bbreeding%2Bconverge.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01116.x%26rft_id%253Dinfo%253Apmid%252F22420640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01116.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22420640&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.6" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Assisted Prediction of Quantitative Traits Using the R Package sommer.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0156744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.7" data-doi="10.1534/g3.116.029637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic Prediction of Gene Bank Wheat Landraces</span>. <abbr class="cit-jnl-abbrev">G3: Genes|Genomes|Genetics (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%257CGenomes%257CGenetics%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029637%26rft_id%253Dinfo%253Apmid%252F27172218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xODE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.8" data-doi="10.1073/pnas.1232347100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munck</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">It’s barcoding Jim, but not as we know it.</span> <abbr class="cit-jnl-abbrev">Zootaxa</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D56%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1232347100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1073/pnas.1232347100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting Quantitative Traits With Regression Models for Dense Molecular Markers and Pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.11" data-doi="10.1016/j.compag.2013.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diago</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardaguila</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melo-Pinto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of grapevine varieties using leaf spectroscopy and partial least squares.</span> <abbr class="cit-jnl-abbrev">Comput. Electron. Agric.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.compag.2013.08.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BElectron.%2BAgric.%26rft.volume%253D99%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compag.2013.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compag.2013.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.12" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diepeveen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">G. P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular genetic mapping of NIR spectra variation.</span> <abbr class="cit-jnl-abbrev">J. Cereal Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCereal%2BSci.%26rft.volume%253D55%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.13" data-doi="10.1093/treephys/tpp036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marron</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceulemans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Relationships among productivity determinants in two hybrid poplar families grown during three years at two contrasting sites.</span> <abbr class="cit-jnl-abbrev">Tree Physiol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">987</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/treephys/tpp036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftpp036%26rft_id%253Dinfo%253Apmid%252F19483184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tpp036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19483184&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268117200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.14" data-doi="10.1016/j.fcr.2012.10.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ecarnot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Compan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roumet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assessing leaf nitrogen content and leaf mass per unit area of wheat in the field throughout plant cycle with a portable spectrometer.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.fcr.2012.10.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D140%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2012.10.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2012.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313317800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.15" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.16" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.17" data-doi="10.1255/jnirs.1006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espinoza</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodge</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The potential use of near infrared spectroscopy to discriminate between different pine species and their hybrids.</span> <abbr class="cit-jnl-abbrev">J. Near Infrared Spectrosc.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1255/jnirs.1006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNear%2BInfrared%2BSpectrosc.%26rft.volume%253D20%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.1006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.1006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.18" data-doi="10.1111/1755-0998.12513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faivre-Rampant</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorge</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giacomello</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">New resources for genetic studies in Populus nigra: genome-wide SNP discovery and development of a 12k Infinium array.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1023</span>–<span class="cit-lpage">1036</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1023%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12513%26rft_id%253Dinfo%253Apmid%252F26929265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26929265&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.19" data-doi="10.1371/journal.pone.0085435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Römisch-Margl</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selbig</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Deducing Hybrid Performance from Parental Metabolic Profiles of Young Primary Roots of Maize by Using a Multivariate Diallel Approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0085435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85435%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085435%26rft_id%253Dinfo%253Apmid%252F24409329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24409329&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.20" data-doi="10.1007/s11306-016-1099-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urrutia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernillon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giauffret</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardieu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fortune telling: metabolic markers of plant performance.</span> <abbr class="cit-jnl-abbrev">Metabolomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11306-016-1099-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetabolomics%26rft.volume%253D12%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11306-016-1099-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11306-016-1099-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.21" data-doi="10.1016/j.fcr.2004.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villegas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Assessment of durum wheat yield using visible and near-infrared reflectance spectra of canopies.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2004.12.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D94%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2004.12.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2004.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232196400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.22" data-doi="10.4161/fly.21535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischnaller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlick-Steiner</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Non-destructive species identification of Drosophila obscura and D. subobscura (Diptera) using near-infrared spectroscopy.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">289</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.21535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.21535%26rft_id%253Dinfo%253Apmid%252F22885252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.21535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22885252&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311076700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.23" data-doi="10.1007/s004420050591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIlwee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawler</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aragones</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolnough</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Ecological applications of near infrared reflectance spectroscopy - a tool for rapid, cost-effective prediction of the composition of plant and animal tissues and aspects of animal performance.</span> <abbr class="cit-jnl-abbrev">Oecologia</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">305</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s004420050591</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.volume%253D116%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004420050591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004420050591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076034600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.24" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.25" data-doi="10.1007/s00122-011-1747-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falke</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiemann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Partial least squares regression, support vector machine regression, and transcriptome-based distances for prediction of maize hybrid performance with gene expression data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">833</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1747-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFu%26rft.auinit1%253DJ.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D825%26rft.epage%253D833%26rft.atitle%253DPartial%2Bleast%2Bsquares%2Bregression%252C%2Bsupport%2Bvector%2Bmachine%2Bregression%252C%2Band%2Btranscriptome-based%2Bdistances%2Bfor%2Bprediction%2Bof%2Bmaize%2Bhybrid%2Bperformance%2Bwith%2Bgene%2Bexpression%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1747-9%26rft_id%253Dinfo%253Apmid%252F22101908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1747-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101908&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.26" data-doi="10.2135/cropsci2016.09.0742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Two-Part Strategy for Using Genomic Selection to Develop Inbred Lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">2372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.09.0742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D2372%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.09.0742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.09.0742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.27" data-doi="10.1093/nar/gks989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gebreselassie</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ader</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boizot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">J.-P. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Near-infrared spectroscopy enables the genetic analysis of chemical properties in a large set of wood samples from Populus nigra (L.) natural populations.</span> <abbr class="cit-jnl-abbrev">Ind. Crops Prod.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInd.%2BCrops%2BProd.%26rft.volume%253D107%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3961.28"><div class="cit-metadata unstructured">Genz, A., F. Bretz, T. Miwa, X. Mi, F. Leisch, <em>et al.</em>, 2018 <em>mvtnorm: Multivariate Normal and t Distributions</em>. R package version 1.0–8.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.29" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: Prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.30" data-doi="10.2135/cropsci2016.06.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumasy</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Prospects for Cost-Effective Genomic Selection via Accurate Within-Family Imputation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.06.0526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.06.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.06.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.31" data-doi="10.2135/cropsci2016.08.0675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumasy</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Potential of Low-Coverage Genotyping-by-Sequencing and Imputation for Cost-Effective Genomic Selection in Biparental Segregating Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.08.0675</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D1404%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.08.0675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.08.0675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.32" data-doi="10.1186/s12864-015-2345-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenko</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2345-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2345-z%26rft_id%253Dinfo%253Apmid%252F26732811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2345-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26732811&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.33" data-doi="10.1093/treephys/tpv056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabbrini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fichot</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic variation for leaf morphology, leaf structure and leaf carbon isotope discrimination in European populations of black poplar (Populus nigra L.).</span> <abbr class="cit-jnl-abbrev">Tree Physiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">850</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/treephys/tpv056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftpv056%26rft_id%253Dinfo%253Apmid%252F26224105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tpv056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26224105&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.34" data-doi="10.1007/s00122-016-2780-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of the utility of gene expression and metabolic information for genomic prediction in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2413</span>–<span class="cit-lpage">2427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2780-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2413%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2780-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2780-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.35" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.36" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672308009981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.37" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant Breeding with Genomic Selection: Gain per Unit Time and Cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.38" data-doi="10.1255/jnirs.1108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">P. R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaix</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">NIR Spectral Heritability: A Promising Tool for Wood Breeders?</span> <abbr class="cit-jnl-abbrev">J. Near Infrared Spectrosc.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1255/jnirs.1108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNear%2BInfrared%2BSpectrosc.%26rft.volume%253D22%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.1108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.1108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.39" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.40" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.41" data-doi="10.1016/j.foreco.2017.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Discrimination of taxonomic identity at species, genus and family levels using Fourier Transformed Near-Infrared Spectroscopy (FT-NIR).</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">406</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.foreco.2017.09.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D406%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2017.09.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2017.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.42" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.43" data-doi="10.1016/j.fcr.2017.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauffreteau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Touzy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Whole-genome prediction of reaction norms to environmental stress in bread wheat (Triticum aestivum L.) by genomic random regression.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2017.08.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D216%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2017.08.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2017.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.44" data-doi="10.1038/nrg2612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of quantitative traits: challenges and prospects.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2612</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D565%26rft.epage%253D577%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bquantitative%2Btraits%253A%2Bchallenges%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2612%26rft_id%253Dinfo%253Apmid%252F19584810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584810&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.45" data-doi="10.3389/fpls.2017.02002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madec</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baret</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Solan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutartre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High-Throughput Phenotyping of Plant Height: Comparing Unmanned Aerial Vehicles and Ground LiDAR Estimates.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2002</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.02002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D2002%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.02002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.02002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.46" data-doi="10.1534/genetics.114.172106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebdon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brawner</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying hybridisation in Pinus species using near infrared spectroscopy of foliage.</span> <abbr class="cit-jnl-abbrev">J. Near Infrared Spectrosc.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNear%2BInfrared%2BSpectrosc.%26rft.volume%253D22%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.114.172106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.47" data-doi="10.1146/annurev-animal-031412-103705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerating Improvement of Livestock with Genomic Selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-animal-031412-103705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D1%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-031412-103705%26rft_id%253Dinfo%253Apmid%252F25387018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-031412-103705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387018&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.48" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3961.49"><div class="cit-metadata unstructured">Muñoz, F. and L. Sanchez, 2017 <em>breedR: Statistical Methods for Forest Genetic Resources Analysts</em>. R package version 0.12–2.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.50" data-doi="10.1007/s00122-007-0515-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oakey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitchford</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Joint modeling of additive and non-additive (genetic line) effects in multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0515-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DOakey%26rft.auinit1%253DH.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1319%26rft.epage%253D1332%26rft.atitle%253DJoint%2Bmodeling%2Bof%2Badditive%2Band%2Bnon-additive%2B%2528genetic%2Bline%2529%2Beffects%2Bin%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0515-3%26rft_id%253Dinfo%253Apmid%252F17426958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0515-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426958&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.51" data-doi="10.1007/s11295-013-0607-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly-Wapstra</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbour</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic analysis of the near-infrared spectral phenome of a global Eucalyptus species.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-013-0607-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D943%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0607-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0607-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.52" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian Lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214508000000337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.53" data-doi="10.1016/j.isprsjprs.2013.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peerbhay</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutanga</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Commercial tree species discrimination using airborne AISA Eagle hyperspectral imagery and partial least squares discriminant analysis (PLS-DA) in KwaZulu-Natal, South Africa.</span> <abbr class="cit-jnl-abbrev">ISPRS J. Photogramm. Remote Sens.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.isprsjprs.2013.01.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISPRS%2BJ.%2BPhotogramm.%2BRemote%2BSens.%26rft.volume%253D79%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.isprsjprs.2013.01.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.isprsjprs.2013.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.54" data-doi="10.1038/hdy.2008.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davrieux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lashermes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Stability across environments of the coffee variety near infrared spectral signature.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2008.88</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPosada%26rft.auinit1%253DH.%26rft.volume%253D102%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D119%26rft.atitle%253DStability%2Bacross%2Benvironments%2Bof%2Bthe%2Bcoffee%2Bvariety%2Bnear%2Binfrared%2Bspectral%2Bsignature.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.88%26rft_id%253Dinfo%253Apmid%252F18971953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18971953&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3961.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.56" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.57" data-doi="10.1371/journal.pone.0186329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rimbert</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darrier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choulet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">High throughput SNP discovery and genotyping in hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0186329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0186329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0186329%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0186329%26rft_id%253Dinfo%253Apmid%252F29293495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0186329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29293495&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.58" data-doi="10.1016/j.spasta.2017.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Álvarez</span>  <span class="cit-name-given-names">M. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilers</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Correcting for spatial heterogeneity in plant breeding experiments with p-splines.</span> <abbr class="cit-jnl-abbrev">Spat. Stat.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.spasta.2017.10.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSpat.%2BStat.%26rft.volume%253D23%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.spasta.2017.10.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.spasta.2017.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.59" data-doi="10.1111/j.1469-8137.2010.03469.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohde</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storme</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorge</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaudet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitacolonna</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bud set in poplar - genetic dissection of a complex trait in natural and hybrid populations.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2010.03469.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DRohde%26rft.auinit1%253DA.%26rft.volume%253D189%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D121%26rft.atitle%253DBud%2Bset%2Bin%2Bpoplar--genetic%2Bdissection%2Bof%2Ba%2Bcomplex%2Btrait%2Bin%2Bnatural%2Band%2Bhybrid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03469.x%26rft_id%253Dinfo%253Apmid%252F21039557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03469.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21039557&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284900100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.60" data-doi="10.1186/1471-2164-15-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesnais</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new approach for efficient genotype imputation using information from relatives.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D478%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.61" data-doi="10.1021/ac60214a047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savitzky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golay</span>  <span class="cit-name-given-names">M. J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Smoothing and Differentiation of Data by Simplified Least Squares Procedures.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1627</span>–<span class="cit-lpage">1639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/ac60319a045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem.%26rft.volume%253D36%26rft.spage%253D1627%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac60214a047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac60214a047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.62" data-doi="10.1534/genetics.117.300374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westhues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiemann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Beyond Genomic Prediction: Combining Different Types ofomicsData Can Improve Prediction of Hybrid Performance in Maize</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">208</span>: <span class="cit-comment">genetics.300374.2017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300374</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300374%26rft_id%253Dinfo%253Apmid%252F29363551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwOC80LzEzNzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.63" data-doi="10.1186/s12864-018-4708-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seifert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiemann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Small RNA-based prediction of hybrid performance in maize.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4708-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4708-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4708-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.12.3961.64"><div class="cit-metadata unstructured">signal developers, 2014 <em>signal: Signal processing</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.65" data-doi="10.1366/13-07228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teixeira Dos Santos</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Páscoa</span>  <span class="cit-name-given-names">R. N. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A review on the applications of portable near-infrared spectrometers in the agro-food industry.</span> <abbr class="cit-jnl-abbrev">Appl. Spectrosc.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1215</span>–<span class="cit-lpage">1233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1366/13-07228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BSpectroscopy%26rft.stitle%253DAppl%2BSpectrosc%26rft.aulast%253Ddos%2BSantos%26rft.auinit1%253DC.%2BA.%2BT.%26rft.volume%253D67%26rft.issue%253D11%26rft.spage%253D1215%26rft.epage%253D1233%26rft.atitle%253DA%2BReview%2Bon%2Bthe%2BApplications%2Bof%2BPortable%2BNear-Infrared%2BSpectrometers%2Bin%2Bthe%2BAgro-Food%2BIndustry%26rft_id%253Dinfo%253Adoi%252F10.1366%252F13-07228%26rft_id%253Dinfo%253Apmid%252F24160873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1366/13-07228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24160873&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.66" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.67" data-doi="10.1186/s12863-015-0169-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakszegi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedő</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shewry</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differentially penalized regression to predict agronomic traits from metabolites and markers in wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0169-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0169-0%26rft_id%253Dinfo%253Apmid%252F25879431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0169-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25879431&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3961.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.68" data-doi="10.1016/j.fcr.2011.12.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prediction of grain yield using reflectance spectra of canopy and leaves in maize plants grown under different water regimes.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2011.12.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D128%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2011.12.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2011.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.69" data-doi="10.1007/s00122-017-2934-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westhues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Omics-based hybrid prediction in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1927</span>–<span class="cit-lpage">1939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2934-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1927%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2934-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2934-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge regression.</span> <abbr class="cit-jnl-abbrev">Genetical</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%26rft.volume%253D75%26rft.spage%253D351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.71" data-doi="10.1111/tpj.13242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Metabolomic prediction of yield in hybrid rice.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D88%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.72" data-doi="10.1007/BF00273671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Parametric relationships between genotype x environment interaction and genetic correlation when two environments are involved.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">850</span>–<span class="cit-lpage">854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00273671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D76%26rft.spage%253D850%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00273671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00273671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988R487200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.73" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.74" data-doi="10.1111/pbr.12482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zenke-Philippi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiemann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transcriptome-based prediction of hybrid performance with unbalanced data from a maize breeding programme.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D136%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3961.75" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors Affecting Accuracy From Genomic Selection in Populations Derived From Multiple Inbred Lines: A Barley Case Study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3961.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3953" title="Shortened Lifespan and Other Age-Related Defects in Bang Sensitive Mutants of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3973" title="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3961" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5teXKM1FkW_olZg6nsp9THMEixz83tqSezGC4bSt2UA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3961" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3961" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3961" class="active" data-icon-position="" data-hide-link-title="0">Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491721" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1YtaegyqN-VcCG1HmZXu5-OSdFL5fObJg1UTmahL9T8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3961.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3961" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZHWvYtD8KKt2pM-LW1iIvdvsmC_Jfo2ALZQ0_gzt_fU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3961" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3961.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491721" id="node491721--2" data-pisa="ggg;8/12/3961" data-pisa-master="ggg;g3.118.200760" data-apath="/ggg/8/12/3961.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renaud Rincent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Paul Charpentier</span>, <span class="highwire-citation-author" data-delta="2">Patricia Faivre-Rampant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Etienne Paux</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacques Le Gouis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine Bastien</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincent Segura</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3961-3972;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200760 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491721--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Renaud Rincent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renaud%2BRincent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rincent%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenaud%2BRincent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renaud Rincent</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Paul Charpentier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>GenoBois analytical platform, INRA, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BCharpentier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charpentier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Paul%2BCharpentier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Paul Charpentier</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia Faivre-Rampant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EPGV, INRA, CEA-IG/CNG, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BFaivre-Rampant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faivre-Rampant%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BFaivre-Rampant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Etienne Paux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Etienne%2BPaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paux%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEtienne%2BPaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Etienne Paux</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacques Le Gouis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLe%2BGouis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Gouis%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacques%2BLe%2BGouis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacques Le Gouis</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Catherine Bastien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BBastien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastien%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BBastien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine Bastien</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Vincent Segura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSegura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segura%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincent%2BSegura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincent Segura</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vincent.segura@inra.fr" class="" data-icon-position="" data-hide-link-title="0">vincent.segura@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3961" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3961&amp;text=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3961&amp;t=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3961&amp;title=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491721/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491721" id="node491721--3" data-pisa="ggg;8/12/3961" data-pisa-master="ggg;g3.118.200760" data-apath="/ggg/8/12/3961.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renaud Rincent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Paul Charpentier</span>, <span class="highwire-citation-author" data-delta="2">Patricia Faivre-Rampant</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Etienne Paux</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacques Le Gouis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine Bastien</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vincent Segura</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3961-3972;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200760 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491721--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Renaud Rincent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renaud%2BRincent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rincent%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenaud%2BRincent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0885-0969" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renaud Rincent</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Paul Charpentier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>GenoBois analytical platform, INRA, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BCharpentier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charpentier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Paul%2BCharpentier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6029-0498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Paul Charpentier</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia Faivre-Rampant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EPGV, INRA, CEA-IG/CNG, 91057 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BFaivre-Rampant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faivre-Rampant%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BFaivre-Rampant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Etienne Paux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Etienne%2BPaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paux%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEtienne%2BPaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3094-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Etienne Paux</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacques Le Gouis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GDEC, INRA, UCA, 63000 Clermont-Ferrand, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLe%2BGouis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Gouis%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacques%2BLe%2BGouis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5726-4902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacques Le Gouis</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Catherine Bastien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BBastien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastien%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BBastien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9391-6637" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine Bastien</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Vincent Segura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BioForA, INRA, ONF, 45075 Orléans, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BSegura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segura%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincent%2BSegura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1860-2256" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vincent Segura</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vincent.segura@inra.fr" class="" data-icon-position="" data-hide-link-title="0">vincent.segura@inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3961&amp;rft.epage=3972&amp;rft.atitle=Phenomic%20Selection%20Is%20a%20Low-Cost%20and%20High-Throughput%20Method%20Based%20on%20Indirect%20Predictions%3A%20Proof%20of%20Concept%20on%20Wheat%20and%20Poplar&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rincent%2C+Renaud&amp;rft.au=Charpentier%2C+Jean-Paul&amp;rft.au=Faivre-Rampant%2C+Patricia&amp;rft.au=Paux%2C+Etienne&amp;rft.au=Le+Gouis%2C+Jacques&amp;rft.au=Bastien%2C+Catherine&amp;rft.au=Segura%2C+Vincent"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491721/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491721/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491721/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491721/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491721/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491721/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491721/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491721/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491721/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491721/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491721/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491721/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3961" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3961.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3961.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3961.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3961.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
