<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3829" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3829.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3829.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3829.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3829/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3829.atom" />
<meta name="HW.pisa" content="ggg;8/12/3829" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits" />
<meta name="DC.Identifier" content="10.1534/g3.118.200728" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-trait and multi-environment data are common in animal and plant breeding programs. However, what is lacking are more powerful statistical models that can exploit the correlation between traits to improve prediction accuracy in the context of genomic selection (GS). Multi-trait models are more complex than univariate models and usually require more computational resources, but they are preferred because they can exploit the correlation between traits, which many times helps improve prediction accuracy. For this reason, in this paper we explore the power of multi-trait deep learning (MTDL) models in terms of prediction accuracy. The prediction performance of MTDL models was compared to the performance of the Bayesian multi-trait and multi-environment (BMTME) model proposed by [Montesinos-López et al. (2016)][1], which is a multi-trait version of the genomic best linear unbiased prediction (GBLUP) univariate model. Both models were evaluated with predictors with and without the genotype×environment interaction term. The prediction performance of both models was evaluated in terms of Pearson’s correlation using cross-validation. We found that the best predictions in two of the three data sets were found under the BMTME model, but in general the predictions of both models, BTMTE and MTDL, were similar. Among models without the genotype×environment interaction, the MTDL model was the best, while among models with genotype×environment interaction, the BMTME model was superior. These results indicate that the MTDL model is very competitive for performing predictions in the context of GS, with the important practical advantage that it requires less computational resources than the BMTME model.

 [1]: #ref-21" />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Carlos M. Hernández-Suárez" />
<meta name="DC.Contributor" content="Javier Martín-Vallejo" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-trait and multi-environment data are common in animal and plant breeding programs. However, what is lacking are more powerful statistical models that can exploit the correlation between traits to improve prediction accuracy in the context of genomic selection (GS). Multi-trait models are more complex than univariate models and usually require more computational resources, but they are preferred because they can exploit the correlation between traits, which many times helps improve prediction accuracy. For this reason, in this paper we explore the power of multi-trait deep learning (MTDL) models in terms of prediction accuracy. The prediction performance of MTDL models was compared to the performance of the Bayesian multi-trait and multi-environment (BMTME) model proposed by Montesinos-López &lt;i&gt;et al.&lt;/i&gt; (2016), which is a multi-trait version of the genomic best linear unbiased prediction (GBLUP) univariate model. Both models were evaluated with predictors with and without the genotype×environment interaction term. The prediction performance of both models was evaluated in terms of Pearson’s correlation using cross-validation. We found that the best predictions in two of the three data sets were found under the BMTME model, but in general the predictions of both models, BTMTE and MTDL, were similar. Among models without the genotype×environment interaction, the MTDL model was the best, while among models with genotype×environment interaction, the BMTME model was superior. These results indicate that the MTDL model is very competitive for performing predictions in the context of GS, with the important practical advantage that it requires less computational resources than the BMTME model.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3829" />
<meta name="citation_id" content="8/12/3829" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3829" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3829.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3829.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3829.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200728" />
<meta name="citation_pmid" content="30291108" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3829" />
<meta name="citation_lastpage" content="3840" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática" />
<meta name="citation_author_email" content="oamontes1@ucol.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_author_email" content="oamontes1@ucol.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Carlos M. Hernández-Suárez" />
<meta name="citation_author_institution" content="Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México" />
<meta name="citation_author" content="Javier Martín-Vallejo" />
<meta name="citation_author_institution" content="Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=B. Alipanahi;citation_author=A. Delong;citation_author=M. T. Weirauch;citation_author=B. J. Frey;citation_title=Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.;citation_pages=831-838;citation_volume=33;citation_year=2015;citation_pmid=26213851;citation_doi=10.1038/nbt.3300" />
<meta name="citation_reference" content="citation_author=C. Angermueller;citation_author=H.J. Lee;citation_author=W. Reik;citation_author=O. Stegle;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Rev. Ceres;citation_author=I. Bertan;citation_author=F. I. F. Carvalho;citation_author=A. C. Oliveira;citation_author=J. A. G. Silva;citation_author=G. Benin;citation_title=Efeitos da heterose e endogamia em caracteres de importância agronômica em trigo.;citation_pages=753-763;citation_volume=56;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Evolution : GSE;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods. Genetics, Selection;citation_pages=26;citation_volume=43;citation_year=2011;citation_doi=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26" />
<meta name="citation_reference" content="citation_journal_title=Pesqui. Agropecu. Bras.;citation_author=A. F. N. M. Castro;citation_author=R. V. O. Castro;citation_author=A. C. O. Oliveira;citation_author=J. E. Lima;citation_author=R. C. Santos;citation_title=Multivariate analysis for the selection of eucalyptus clones destined for charcoal production.;citation_pages=627-635;citation_volume=48;citation_year=2013;citation_doi=10.1590/S0100-204X2013000600008" />
<meta name="citation_reference" content="citation_title=Deep Learning with R. Manning Publications, Manning Early Access Program;citation_author=F. Chollet;citation_author=J. J. Allaire;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Ruminants;citation_author=J. J. Colleau;citation_author=S. Fritz;citation_author=F. Guillaume;citation_author=A. Baur;citation_author=D. Dupassieux;citation_title=Simulating the potential of genomic selection in dairy cattle breeding. Rencontres, Recherches;citation_pages=419;citation_volume=16;citation_year=2009" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez-Rodríguez;citation_author=J.M. Hickey;citation_year=2013;citation_pmid=24022750;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_year=2016;citation_pmid=27172218;citation_doi=10.1534/g3.116.029637" />
<meta name="citation_reference" content="citation_title=Applied Multivariate Data Analysis;citation_author=B. S. Everitt;citation_author=G. Dunn;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. Gilmour;citation_author=R. Thompson;citation_author=B. Cullis;citation_title=Average Information REML: An Efficient Algorithm for Variance Parameter Estimation in Linear Mixed Models.;citation_pages=1440-1450;citation_volume=51;citation_year=1995;citation_doi=10.2307/2533274" />
<meta name="citation_reference" content="citation_title=Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python;citation_author=A. Gulli;citation_author=P. Sujit;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. He;citation_author=D. Kuhn;citation_author=L. Parida;citation_title=Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.;citation_volume=32;citation_year=2016;citation_pmid=27307640;citation_doi=10.1093/bioinformatics/btw249" />
<meta name="citation_reference" content="citation_author=M. Huang;citation_author=L. Chen;citation_author=Z. Chen;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_year=2015;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. P. Menden;citation_author=F. Iorio;citation_author=M. Garnett;citation_author=U. McDermott;citation_author=C. H. Benes;citation_title=Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties.;citation_pages=e61318;citation_volume=8;citation_year=2013;citation_pmid=23646105;citation_doi=10.1371/journal.pone.0061318" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. Toledo;citation_author=O. Pérez-Hernández;citation_year=2016;citation_pmid=27342738;citation_doi=10.1534/g3.116.032359" />
<meta name="citation_reference" content="citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C.M. Hernández-Suárez;citation_year=2018;citation_pmid=30291107;citation_doi=10.1534/g3.118.200740" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. J. Qi;citation_author=M. Oja;citation_author=J. Weston;citation_author=W. S. Noble;citation_title=A unified multitask architecture for predicting local protein properties.;citation_pages=e32235-e32235;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22461885;citation_doi=10.1371/journal.pone.0032235" />
<meta name="citation_reference" content="R Core Team, 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL http://www.R-project.org/." />
<meta name="citation_reference" content="citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=S. Mondal;citation_author=E. Autrique;citation_author=J. Crossa;citation_year=2016;citation_pmid=27402362;citation_doi=10.1534/g3.116.032888" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=L. R. Schaeffer;citation_title=Multiple trait international bull comparisons.;citation_pages=145-153;citation_volume=69;citation_year=2001;citation_pmid=11337117;citation_doi=10.1016/S0301-6226(00)00255-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet;citation_author=A. W. Schulthess;citation_author=Y. Zhao;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (Triticum aestivum L.).;citation_year=2017;citation_doi=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7" />
<meta name="citation_reference" content="citation_journal_title=Ieee Acm T Comput Bi;citation_author=M. Spencer;citation_author=J. Eickholt;citation_author=J. L. Cheng;citation_title=A Deep Learning Network Approach to ab initio Protein Secondary Structure Prediction.;citation_pages=103-112;citation_volume=12;citation_year=2015;citation_doi=10.1109/TCBB.2014.2343960" />
<meta name="citation_reference" content="citation_title=An Introduction to Multivariate Statistics.;citation_author=M. S. Srivastava;citation_author=C. G. Khatri;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=N. Srivastava;citation_author=G. Hinton;citation_author=A. Krizhevsky;citation_author=I. Sutskever;citation_author=R. Salakhutdinov;citation_title=Dropout: A Simple Way to Prevent Neural Networks from Overfitting.;citation_pages=1929-1958;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Sun;citation_author=J. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J. L. Jannink;citation_title=Multi-trait, random regression, or simply repeatability model in high-throughput phenotyping data improve genomic prediction for grain yield in wheat.;citation_pages=1-12;citation_volume=18;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. Tavanaei;citation_author=N. Anandanadarajah;citation_author=A. S. Maida;citation_author=R. Loganantharaj;citation_title=A Deep Learning Model for Predicting Tumor Suppressor Genes and Oncogenes from PDB Structure.;citation_year=2017;citation_doi=10.1101/177378" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_title=ICML;citation_title=Multitask learning for subspace segmentation.;citation_author=Y. Wang;citation_author=D. Wipf;citation_author=Q. Ling;citation_author=W. Chen;citation_author=I. Wassell;citation_pages=1209-1217;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Xavier;citation_author=B. Hall;citation_author=S. Casteel;citation_author=W. Muir;citation_author=K. M. Rainey;citation_title=Using unsupervised learning techniques to assess interactions among complex traits in soybeans.;citation_pages=200;citation_volume=213;citation_year=2017;citation_doi=10.1007/s10681-017-1975-4" />
<meta name="citation_reference" content="citation_journal_title=Knowl. Base. Syst.;citation_author=T. Xiong;citation_author=Y. Bao;citation_author=Z. Hu;citation_title=Multiple-output support vector regression with a firefly algorithm for interval-valued stock price index forecasting.;citation_pages=87-100;citation_volume=55;citation_year=2014;citation_doi=10.1016/j.knosys.2013.10.012" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=Y. Yan;citation_author=E. Ricci;citation_author=G. Subramanian;citation_author=G. Liu;citation_author=O. Lanz;citation_title=A multi-task learning framework for head pose estimation under target motion.;citation_pages=1-14;citation_volume=38;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Med. Image Anal.;citation_author=X. Zhen;citation_author=Z. Wang;citation_author=A. Islam;citation_author=M. Bhaduri;citation_author=I. Chan;citation_title=Multi-scale deep networks and regression forests for direct biventricular volume estimation.;citation_pages=120-129;citation_volume=30;citation_year=2015;citation_doi=10.1016/j.media.2015.07.003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Multi-trait and multi-environment data are common in animal and plant breeding programs. However, what is lacking are more powerful statistical models that can exploit the correlation between traits to improve prediction accuracy in the context of genomic selection (GS). Multi-trait models are more complex than univariate models and usually require more computational resources, but they are preferred because they can exploit the correlation between traits, which many times helps improve prediction accuracy. For this reason, in this paper we explore the power of multi-trait deep learning (MTDL) models in terms of prediction accuracy. The prediction performance of MTDL models was compared to the performance of the Bayesian multi-trait and multi-environment (BMTME) model proposed by [Montesinos-López et al. (2016)][1], which is a multi-trait version of the genomic best linear unbiased prediction (GBLUP) univariate model. Both models were evaluated with predictors with and without the genotype×environment interaction term. The prediction performance of both models was evaluated in terms of Pearson’s correlation using cross-validation. We found that the best predictions in two of the three data sets were found under the BMTME model, but in general the predictions of both models, BTMTE and MTDL, were similar. Among models without the genotype×environment interaction, the MTDL model was the best, while among models with genotype×environment interaction, the BMTME model was superior. These results indicate that the MTDL model is very competitive for performing predictions in the context of GS, with the important practical advantage that it requires less computational resources than the BMTME model.

 [1]: #ref-21" />
<meta name="og-title" property="og:title" content="Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3829" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-trait and multi-environment data are common in animal and plant breeding programs. However, what is lacking are more powerful statistical models that can exploit the correlation between traits to improve prediction accuracy in the context of genomic selection (GS). Multi-trait models are more complex than univariate models and usually require more computational resources, but they are preferred because they can exploit the correlation between traits, which many times helps improve prediction accuracy. For this reason, in this paper we explore the power of multi-trait deep learning (MTDL) models in terms of prediction accuracy. The prediction performance of MTDL models was compared to the performance of the Bayesian multi-trait and multi-environment (BMTME) model proposed by [Montesinos-López et al. (2016)][1], which is a multi-trait version of the genomic best linear unbiased prediction (GBLUP) univariate model. Both models were evaluated with predictors with and without the genotype×environment interaction term. The prediction performance of both models was evaluated in terms of Pearson’s correlation using cross-validation. We found that the best predictions in two of the three data sets were found under the BMTME model, but in general the predictions of both models, BTMTE and MTDL, were similar. Among models without the genotype×environment interaction, the MTDL model was the best, while among models with genotype×environment interaction, the BMTME model was superior. These results indicate that the MTDL model is very competitive for performing predictions in the context of GS, with the important practical advantage that it requires less computational resources than the BMTME model.

 [1]: #ref-21" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"C3k83V8wuOONgTrMKUT6I8wYTHhYwUGPYTS9GLK-HI8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491518","apath":"\/ggg\/8\/12\/3829.atom","pisa":"ggg;8\/12\/3829","ac":{"\/ggg\/8\/12\/3829.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3829.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3829"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3829"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3829":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022G4YWkjSOKx+zshuZvTU2C3aLEXL+BpkyZid\\\\\\\/okhYfh3UqhsJS+\\\\\\\/o1VmOTrHjEu7Pmmcq5hEOX\\\\\\\/jZN29ORxHALFQyF+R8PmGkJM6cqBrP4vh2wFOP\\\\\\\/a2HsFtVeHacW00ldudIwWItfPojPjxjwIUFIpfA0K7xMmXEbEYtrUtLrfkETwM5\\\\\\\/jkfUD3tKn3h+XuJgawwIVgstA3fMtBhZe1xTPazNpVxfVeHcuBySfh9EUrtST3O+PKtSUyqwMBHc0fSQRVPdCA6HH7Sv8N\\\\\\\/+7pa5QjTJOSO5r0iTNJIXzrsDQ5aBVeYxvrFZYGTULBsDMF5D85dum0omTxjzrTy3SHnOvI72EwH2zoVgR+q5WK5FwTAGjzzLCHcwOqHnOBJ9PBxTuU\\\\\\\/fGTQFq1Qu\\\\\\\/tle6IPtUVhnvrVo4WmIsH7RICy6WOrPtt4E00oFT+q7F1Ujl5uYgyD0k\\\\\\\/we8GVFiQKyHWkTgKeSsVlXCxVloBZ+XXQZBujMDeHNLEyGln5BTi9Cg5YdDa56BIyKJ283Xfss0SyDhpKTf0vXm41BOFCAl0pxu5zFC7SRBZwjLOl4y96QMwEzlCFYntiW0ELwOlu1d6fVMi4Kgj56NFF+PQTpRjLWxdseE661HSkfmYvwdCveWc0qi3q6dBPDlwMJ3m8lptwGYXln7VFfdXjXyIwBX09nldEd9gtcPhmuUeCD7fQCUunZZJkoMhdXonIoCBAYHiADcVirG6vlx8mDuJoHp2aQg1YI89gtdTo0krRwh1VhHhq9i0\\\\\\\/+JB\\\\\\\/MWtYSzlsRTtbVNs6O31cdd31Mwkgwre5ptMT8ZosUVgIcfp2QaktJfHLvFZlaJmkKxnn57MTwqETysgWjKKgZ3STgfs7L\\\\\\\/gPuNZVizs90yMsvjFaDwOD8bVmr3ZXIt+0EktSHU7ZYG2sFQGT6khlnDBV9gNnntHTjX+j3a1y7R3wd7d0kc7OkLqWsTLfUor6GxGE\\\\\\\/dESAld0jlrjt8FyiK0qHZtf1auOgw8wqBUdU8xIIeGAWOaor2L9Fbq4Ma0lvLC+27CHi4Cq6z9B\\\\\\\/+xMJAFwg4zZbJA76B\\\\\\\/DmbrUh7eCFx7h6IhGna1t3dtLRno4f+PO3rpzG9E52PWhV\\\\\\\/EUalqFiMxogYhhgC8Y8o0t9ROGkz4wUJ7YF52tXgIKtRMJ0098vCpPRbHC6rf7j74BcnXI+wqyRgmGl57BdDPaIbuk9F0eq5+pjaUk5+nqWag0H2Jb\\\\\\\/SBlosqDHng8KvNPq4CnHtlZnCYbW9agcipuEG8yPZBehp9WbHH9\\\u0022,\\\u0022iv\\\u0022:\\\u0022J\\\\\\\/+aqsA+Xv2GezFfziX3TQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d6870eaa873fbc6ea19844d34fa278a7ca305097496eb4ed8dddc17e0449b5db\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223ZtMhmnFB97H1AbPbnimNlzlZUNkP1W\\\\\\\/6HfCTCxUMlDvIPV6wF8I6FJ4Ibm5Vbt+iBFL9yZZJMbsklwAz2d8BtWhZIQTlKqxF1YQvbTznW\\\\\\\/CE6WUiFzrYXA6ak7F6kglffeeFLv1IRH6M11eRpB+4\\\\\\\/w4i7Hnu\\\\\\\/ftB02YR+IBZB8WCc7I8OcOEkiUw5Kos5nV9K4\\\\\\\/\\\\\\\/3ugQTOudgRJ9mjddgG7HltF4HBM2D2LFQJcOswrm0BA17Vfb9iYMmdgpuu16RrI9gvRDMX+40GeZaZM6SEldQBxktkYOTJjhcRjyl+mqYgem0IJlnwK7\\\\\\\/7tgMlsZqq+2gzKEWET9T+ZOKOjDhGZ8et\\\\\\\/O714+7GGwUpEIDWINMLpg+j3+i1pbidZtOxcG2UNfUXwZhUum1MvaKVptwwFS5DQHseTG8aOJ6lJESlJ85yD5mdr3qWjxSZSVXqUbbCwQpp2Io2G8NL8SEUQ9fC4DoGUZ8lnFbu6fF4YHQ+D\\\\\\\/v0XIRxbgikiyEFS7xarpaZdIkJJ8sDwHPzJSDIocpJT5hxfIDW3N4DTbNvVvBz6GR1riWLGl43McWakXFN6UP43ClokaqGYbFVAVC+sCaKLHalhi968uhX1iPM2LS4UmyfmVDtAJE0caEK\\\\\\\/04dcYFI3Q25\\\\\\\/\\\\\\\/OFesF+v5P\\\\\\\/G2kW0VARXx+ThEaXRxJQtsdHgu7pWk5aHHLCAzlvtlXo2+V0DnF+IltsrRCSi2Nf7u7Xh\\\\\\\/VmViYpl3\\\\\\\/Ffw+pY4\\\\\\\/JdVm3v7SISjsauuKoCXqvdfSo3J8sdWY4mDbHZQC3TIzQP6bQ8jRG5oPmWOiiRtkSyH4CAdDR5V69o\\\\\\\/wkwCZdhyzxpvGKOyC4JmBp1VaBrBRvJ4zNzlP8r2nyepY7XyDoFJ3R8\\\\\\\/57h4ALxN7PgPO7Njvrc90ApIEVD1ajfcxoXSagAks0gURM6porLt0b8PVfQFdS12L2gjAtU02rvxS\\\\\\\/jpAKoH58AqWcbKT7SGiCFKL\\\\\\\/TnPWeNLgy9HsizI+e68SM7N+qwoU0AS3KxoOo2\\\\\\\/c6Y58jAtS\\\\\\\/+2ujReW1XmtwEAqDBPsvSHaTHRXyAJyy0TiE0O+sIg5F5yx4G5rL6t5mTw\\\\\\\/FOVqJqxtBc+YQ9QkHcybd1pBF\\\\\\\/FWg3mJqo24NyEUy9nSNG10\\\\\\\/G11QmXshOUKWg\\\\\\\/O8eLfffcMGxTiMCRdKR7CxIHn8zCCAPGwzA9\\\\\\\/Vjy+8aKmPNNjrDb\\\\\\\/XHVQ9pMhnSBGbU7guM+7DHEoMoJruV0AcTyUTIYr04ZEFGI7T0YGkzcLKICZ1g830R49jef3xuSgmjEAHwGlkcCCEfiyioO\\\\\\\/2rxg4YNjouA8USZzdI7KMmgfzvsNdwV8pU+86HMj7vNLG+7Ken7VHgOzxK0znuUmS8dywZ6mVyY8z4PGsE7awm6z0h0m9iO738rL9MbBpz76hP\\\\\\\/S6759e8LnJan5gBMn6akzBm2As2WuOiEpY87U+KzJu\\\\\\\/p85Qty30OdzRGlyuWmMaF2Bx0gpBOmuKKSgwjaZSJWtwElayaGqIbXJtxszs5r2s4fTygHO1c8lIpj16mFxPp+D9MYGncSASBrHnQY1T5wf7l80XhwABD8VoCQ5D89skQa1\\\\\\\/+AaOgCbjIpjGEsfdEZcEZLXZpW3DR2dLiB\\\\\\\/OtzsGTy1yxSAlm+FmcCZiT\\\\\\\/4ZcC8HyavlqmNa1ZsMRzbifWfIZk0WD7wNRQu3hRLV0Frr2zLFkMPjtN0atA65wjEv4k5kzQDyB76mQtQf7UPL+qyQ2q6SSquHisYgzdm6Txb8XUPtHOXbz80R\\\\\\\/MuoTCS\\\\\\\/e21+diuGF\\\\\\\/5LZ6I57hjyfih\\\\\\\/Pt\\\\\\\/Hv\\\\\\\/bauKf2R8oRhMDQlvzFszeglsALX5NeZj47somUpyCE4DJwpswUXCC7akRzvpMi6EXprxItYBdlX1EWjas4xTqvKzurCk8Y3T3PWWnUiyanYIqnmNx6BmxhuZr+9i\\\\\\\/zCZdHhnJVwvF0nwKzNnVUutvto0KT2WeBd5\\\\\\\/7beUrdGC8L7Ky2Y+ujSkC9cUZJ3m0kN5HwGrAUq\\\\\\\/ZShPSfGtzSrvfn2eD01I8g0hOH6ojvetX\\\\\\\/jnrgddZ+tTGluG9\\\\\\\/Hu8jXgqJU0ZHrMuuQd2YD+BElFQks8+FAcwQbPdAvXYtwaThLCaV7ifrkJD\\\\\\\/FDZOC1ZPnnE6GPXFqTDv26azjCXYO2caOu2h8zGxSf4tapVn8sRaDtwGCHoOv6+gacFOCbT0a9fIEhxFbmrdpUK\\\\\\\/MyFNw+mJCVwn5JttyxvikOlkWy13x6pZDVo5vLPS+BLLzp\\\\\\\/Hot44GCRkw+75j\\\\\\\/dNnIm6aYWTdUs+F03YJ8qgGUXftR+wKgRH67C12VtTt+hhd\\\\\\\/GNrCVjr1aLwuIhXwE\\\\\\\/RzuxFwsHh2P6d0caEhM9C+nHEpAfnlJTzdQxP2Uf0F\\\\\\\/Dycoo4rrcS4r7X4CnBK31+VIgDEk5OQ6cNMEADw6kd7Xs\\\\\\\/A41pAzJIZMqVSTMxHEs1GZNfdYWtvdIdNK00tMt2C5Y\\\\\\\/bf+YVK1oiC+XsAvyewcJVOjxAAkwRf+QNBx\\\\\\\/aH2rbuC\\\\\\\/YEHtPs45aLqAUcuCBmh1vTpNWgiKfoeAwdJWqZRukQjbsqIM0GRMjjxTTzX+7aNILZC3pYXfzlxLKP64GqelBIX2ftQoKWFvYSV5FTPhiU\\\\\\\/aYFS2aYGF16HcrpiNVbQkE1\\\\\\\/UMsY3kxzOWPQp+4UXJlrt+ziZhy\\\\\\\/f9mWrR0JpAhwOC14g8PIPglSwnL6KeA3mEgkIhiLj+17qh+Fr4Ct2OF++G5fqCJjrt5R8fWI\\\\\\\/DxBrbiHY94zFnzCuBFIH729QwrUM6OF0+mNBqtu7mjdDTRXMUbowgy4VzxmSC7ZM2zsuygaZXtCjaAgSodP4O7A13YdFloWMjftEflzO+UkH454TEZC9hCLvDIwEJ57\\\\\\\/YfYbG\\\\\\\/1k871y3kJ3a+h9nrcoK40s7AqCAFfmL\\\\\\\/2G4icXE7+CbRX4OLKTNZVVvnkp5SPj+sQKSltU6ZKur\\\\\\\/Wyq+vQ5pVN0OGg3J0EPg6MBJqV8Pj7dKprgoh\\\\\\\/7lIwRwIZjZ4y51J28ycI0E0Ev+D\\\\\\\/1Uzbn83dBdTiAXwN7K2VIFiGdYkhk5A0yKNpSmSJplamWF4v0iruxvY\\\\\\\/C7oKGtpsEYlSK23h+6CgIBMXmrFOgKiwBKXaim\\\\\\\/g5nPhFTIiTQ61V5PZRZa+Dr3DtZUWuKV1E+LwaEM6LChRVGjm+gsXqHtyA7MbG6+2LYulyCmBaSVn2aTwlKO+zCQgkGvl1UQCT9WfBmaHTtlnUgfegDxA+i1LuDAxmhEDhqlyTp08qx8KsyOa6RY4efv57W2XkyQhVYot0DoK0Rc\\\\\\\/HAFTB0pFxVkasUXTab91VaQYlN6pCUe+90pRQmk5COz04Qt6WgW0u78vavk9CqFP772VALdWjomRCu21kqGtCDrR3X8LMNojqZsytCAUXmnoQhDKjDvjzTSuEw6IRcgsLMe9FGDh+RiteGSzCYjJwxV043pCV3Es9i0+rbnR1ZNVt7wIq5TpB2is9AOi8NjEph7LuC39ZypdY\\\\\\\/VKDrsykP77Lnxez9ALclZcNE3m73vuTruaCOGTv4fIryQL4Hq0C\\\\\\\/xPGvminMRMdoxAopI8KN+p4Ut0d9X8+OkABp6I0ohKzfbQibeV9KwD1L3g\\\\\\\/Sh4DL93sQux0myuNVVI9f9Js1vnsKPq4\\\\\\\/12Z1ORm8glqIxnofbgTN1uygvyckyj4OFdLupoHU90eedx2B9R1WLvh2yGDisUaT3rzOV+15VTopOd+lcnS4govE5a6xpALUUDab2w2cUVX02s05RFVkEyvMbzLAeJqHaSftvPUTa2QFN0tiT2iXy9yLk4Aj4JujbMKUhxH5qYIzwNlC\\\\\\\/2t4XN2jAcTy8ddgi8z3Fwu2bUTGDcOVtaAC8K07r76Dksbmis\\\\\\\/fWoSdudG1viZYlisnqccJluOLLp8NDt2O0GKXAhmn4gkNRQhYJqCuDD0qxtg\\\\\\\/srLKxsIXHDzAAX0EgM\\\\\\\/xSpiHM2xkX8ZMsuEsK+7BYo1XBFtiOih84esUAf0W0rW4p9mZFsL02P4cVuSJsGD6E0JtnEWl23hkud4txuP950fAYdbi63BKyS1uGlECQXRJTgjyXgfGhcs\\\\\\\/q8HGINE005OMiRXspfkgq8JmieSMZwn6Ir4ydJnu8T+ElqeJI2REplERp+dJ\\\\\\\/uErRJyi3BbgqfUCRnUoP4H4XVzase1IUj9FKSXKnhHQzx0CT\\\\\\\/SXKTIiGLSLkGZnpY2qk1ZKjvFyr5Ke3LWLyWE0ANDkav96GrFLbR0Vk41NzMjqdXi881oZw6RjyqtKq6zbT2Y62UQJ79kl94al18vcupWD21AJRc78DSs2NmT2GTJnYdrFLCJQ1KN\\\\\\\/l6lOR+r33PmkUAITkjEJsgg3LZjsKtQli9LaCs6KglYJsCAR5LRCprd28dirEp3QaK4Ok86adbxXM+u\\\\\\\/AuBI8BWgXhreY5wSjPteKQ0aOoYMYlC\\\\\\\/9kkDhJAgmGF1ZkI1jfMZPARE5KDLdvtYq8I8PK5zwv\\\\\\\/1B+flsl1R8iCMw8PRr3\\\\\\\/mq2VjSdvt6e502dUIFeZFUDO4t4peTAdSBKGAQKzo7\\\\\\\/LjQcmhLi\\\\\\\/9WTMy\\\\\\\/Uf4908I0rhM9BRQyBf9wtpCGyywX6JaxDHhL2P29u7qaDnUXOGpiZle22Xn7ACq\\\\\\\/4Xyab\\\\\\\/xLuZixUX3IZ2wDKsmSr29COTTrm7KXiqii\\\\\\\/i2jxyL04RivnwfMC88egeRgKZ8OdDTXKvxbSCZTntIWWgNYOwgarbK31u85spsuivsswrtT+JvkiM4PRMwj8fBaWBaC5Oy\\\\\\\/q+JonYhQeo+rCu4xZBb2jk2pz5\\\\\\\/MYoFVoI\\\\\\\/TOFBG6F9FrvMQKHEho4y016lZ4It4yc0ljJF561\\\\\\\/WSaewmZ7hm9zYJceEFtu\\\\\\\/5sAPjVHfsYXeLwpBypEgBmhiflQfg7Q+NbLUOqnCwpy2hI1YzhYTZT1vV9MbJ9ek4bAy4hjqO\\\\\\\/2ozrYBHMzqQt11f1jFraWxy1DJfxPg4isxLsezquRuryPQ5qjdn42O41Hc1tl1aLwH6CGfb\\\\\\\/m+HzVIDW59wFk8+ZD2jO89pYUbhPBNghEK\\\\\\\/MiOpV+WorqgylBR4hmajNU4b2HNHKfGqCdphv5hap\\\\\\\/9aCwB\\\\\\\/jMTkfz0ftMMIsJsiLXza6\\\\\\\/Ogh97Ux\\\\\\\/G7RZ9bsZD3QYR7cC0SG4fx6FzhH+sITmgE7VLakMUCNsSMWlBqQz0gnwU9zJu9SF8mu2xcTZQE7FHHo39716m5+ScS89hoD0In9rFrbHCcd76BqKBZPCdgkPXNFUnsqxWJCsPkrsoAJpJd8B1yWkvUPyow5+0CeLKO1nNU7TrCISS0\\\\\\\/D+zzaWaRFotiPRQjyaWTq6lmnq7Xl+LnsiFobs5NJctdq0ZHWH7dhSv3KcavrLNPLH5eSzDWypMei24V9m\\\\\\\/kMuuRYdf1fXDTnNT4TsNZF236zsw14STZJJRoXE6bYQHzGA\\\\\\\/mVmNmqXc7j3LirhLdGaZzEtoFGBn2F8inDdRdMg4Sxc3MkaCMu9i7d4zRRRj7O7c+t7j2r\\\\\\\/jmQrlgAVbMc7Zo9WDhq77pVEjm05LIRMwH09xQLuNHFE1F31GjYRC\\\\\\\/JZPMu\\\\\\\/VmR6AV6sIrtsaQxqtyzIlqrX4HquDrdY5AW3J6dH8kjnWbVRXgsBiwxwBoyteXLoE9tMRgqyB5iUJRrV9\\\\\\\/ydk+v6gBSi4pE4StRAUiyk5dEFLJFh4GJQ\\\\\\\/hFLaj2VAAw19jD6drQm2TKAuSKZGQHRTSgvJ0i\\\\\\\/33A6QNL2b4m7lrZwbiLw1WnvxuP0cAzDXeqQNnn7MMJiEv64XmJw1IOedlNh7r4P0lP0Kr\\\\\\\/nJtkcAr0T6uGhhSf0+xRG4FWfwO2rFMuLT9dxIW8ewJviU+9T6IqSD252o18i8zUDHc+087fCoAB09Q5sfyE0tJh7Gy+weCC7YDeJuQgGXeRM0UrM9DkUuidwNzQKJV0d3xFsuw2bIn6T7dYsQDSiss8L0g2j2rtuENpfa7vrGsvS\\\\\\\/P\\\\\\\/ued77knSV0R511iVqiehv72YyX8vcGBF4tCw9c4GnFSFpjUYu4lofwpfER8uI4rQ0G2pDGyNgY7ryA83HOOTAk1bT79HPotnA35UK06jAVLxuPuc4kYmKqE4Ay6Z11SGnjTWV6\\\\\\\/NN6Qnsx1stYVEbOT5TfAOcq\\\\\\\/5eygAaST3X0R84e28Gaj3Ev2Rt\\\\\\\/I9\\\\\\\/M9hcDKmrCOqMfj4YAVvXC+T\\\\\\\/KqJZnCCF\\\\\\\/Q1BAVLV+nJaZXB6gZrZ3Lizxqz9GjqyTdn1gRu6IZeYu5SxKlCn1oEzJ+5BFDka\\\\\\\/9sLGBQlwwlsdFqvds9WSVc2TkAMxtlynAQXSxTsQQzXfUjhc93fX8rk7FuU2VjwFbJlQR\\\\\\\/Y7go1xUO3Cj7WVf1mrv1YHUOH7aauAlssS+VF\\\\\\\/9yTcU6uYbIcqsq5lErsMsGyAkA3fCCkLeyfbVpAkz3j9oqjknzGb4ys9sn5FEqTc2B4QaEaHq1DV8re7Sso2AAlJHrVTHZFgkLhs6TZkmEPFGsBvmp8tWvISFu7fkrII0MUE0NWP\\\\\\\/VET9gfIKEXZkC0ZKQWMvjkP59Kxy+vf8jHuaOEkrmTVCe6HAp7xr+t5ZxrF0jXHxv+WpWombM7AIjzp6TsaAI719l\\\\\\\/VWynmR+aKlqZgS395h3wXd2oocaC64qLodxqjAYPWgI+XE7EHOMPin2Tu6\\\\\\\/PVbyHNBLi9k2s3EQbwKgUaB4z\\\\\\\/Ta34p0bBgIMvCqT5X96qluOF5Fm9fclooDYMNpqfCP2rFIW+FGgKkt6Mn1fk7SGacdII+AXaIBAdzenlGadAt049YemQx3DlR7ndLFixJXjj6Q3Zr1yPl6bwbScDMaa1KOJCACm\\\\\\\/Hh9zVIuvKZo2NT0BEMU39G9\\\\\\\/snu+10BIItZf4Tni7l8+Po6dD4eqqkkAuTaTfv4iTpD\\\\\\\/fqhg1rB4x4DTP79Z+QmwnDhvpZEaP3PdnYkBwC1yWtuEqkKPHhTevRXAstVu1VQgws\\\\\\\/ozz0m3G1CjycwHKCEF4bY2hyxhAcZqGLz6ZKGpzwLF6OpytDz5sC8SooTqSgVcIN855XV6TXVPFqYPs3O1JYEG7nzhGX34eXtHqsi3U\\\\\\\/u67OlbIIPZsyZe9iC6Al4Re9l5b3BZjwsiZy1KRkdRmO80A8jH2OO\\\\\\\/baYCFRp5p9fEjpFKs8ZT77y1SVmQWyf2k103iAXSatuLeQrMqvHCdNTcwbV5zYLqHND1Scl22tlDTTq88AshLVJAA3M78NILP3NEoex1PtQ78Qjp3tnt+WTV+GPCiFYcMDsyZFMRYDC06CiQ\\\\\\\/RfoKABBdRGH\\\\\\\/itoSv+HuIEgX7aSbTDwBkPzUad\\\\\\\/JgEs7zNJVbv4PEaZnyKILIxucpSftJD3+18+bfN9NRbVwaDhgeYusuPdcB+eFot4BB\\\\\\\/jptf6Avo6UpIm6WI\\\\\\\/f22Pu3sEZ1hive2DaSupEFplkf4x3sQSKijBuWZ+cYD9NGcB+AabLMQqR\\\\\\\/ZDfgJDVeHLfC+7p2dDPvBsYtHLDoiqkYfpZ\\\\\\\/GlSeaQdoDk1zfkh45wGBNtMyzYz2DmCRcpnrOiULGzDPHlYeAJ7oYyleaNfefV1LJraeBD\\\\\\\/QPVTpYMFz3C0TeOzBJPHKHxML8YPmA7Jzq+ubtWYwJ31euaRNW0gEpRqcQTdln70Qkuvra3rmDswQ52K9llls1cMtifErxQjApqFCXq6kgKmNafJoINubQwjrREYbsWR2cjgMBjRfgfDaXW6wDHlHkA+1DCKz5LhtFhLT33TKjKjahqSHs5udmGGjKlutC2iKe5ycxgjln0Dor4iwWYr6wH6jErysti6\\\\\\\/Isx76GEL+OvIGZOxVCnfhWwFWpTj12tFe8iK1eNP03tBHMKhKM2pbU\\\\\\\/njo\\\\\\\/wcQGH2F+gq\\\\\\\/swv6f7Qbyrt6TEAOP1ZfscKsgj8GEh2OhLA0tUJIUfuOspOEOQQinOIA8HnSQhY6c1mMB\\\\\\\/KLVb\\\\\\\/TIS38RBa8h3ZKOSvwYd+92eHMWm7xOW\\\\\\\/dPJ0k1og5nxMKopGILGaWAHtEZg+xfLM80lMBhZnqdmEJKVP8HqXM61mm+zTtHphmoJ3JT9sKyRNOr1bE0BmBWhRnz1w3LryljXIuV36VzncDNgcitz1SDgchIaea4Z68unT+2DdaaskeYenar2R8qDW+O31+97g8b8lCkUdYpDWhsGBlUpwD7B5sTSiCoiecbKPXP\\\\\\\/Uudqtv1lmVBttyR98JNsY11Sosn\\\\\\\/kq9Tuom0NT+VZlGCKNKdBG0+MpyMYOEmSBd4gOob8UvRTrISX9DEv6mhfJ2YINBRd4zTn+WNHBQfnOOk7DrNeBS5+0a\\\\\\\/m+EDXDLMSQ0fQpL4wPvKH1gQ8lR+6IVvpwwXMaFEeIQYhOUC9C7vW+Mp0VSEeLKx7eMeH6q+aVRU0rgZqflIY409YRUx1UITZ2CrDGFmgjygfRZARZSVmokNf9oTle1MJNP+01PBtmtC1OAjnviONBupkCL3LO9u9LJ8tw4I9\\\\\\\/KyfRLPnedzSB6mDwg+2KDwW4JyB5RHJMbBJf6KVj41hmFy\\\\\\\/gNy2ebxMVwW3m\\\\\\\/tUqqto0qV0AO7i1YGhdMPRjdec0xYknN918B7fENOPEJnTpdCB7J8el5do3x9L9AgPPLt44AomBV0VqrIkw4nid+CH+zDZK+rYzi7UMO2kaUW2u7daNLZeB1w95SIi29NXM45VRMWj7IUu7fHr69cwlrpHt1NkOPmuh90FjViXp6eM2ApwVcq6AC5J9fLtTh6He8FngtXnMH3K+5ye4P8kQUK4jph2pvlK\\\\\\\/0iWqUvjOvNb7bsde2WphrNcaLDJ9p4KJ0VeW5VvzYXfdYqnOOs9y+vyBYhJ9inYP7PQTv9cH0jRcHW6yGQeQaWv\\\\\\\/N5rI6DA\\\\\\\/iLLXASVjmsqJrl0RcFhXZTuiWB+SGeSm8VuC\\\\\\\/2p99RmYhjcCIKeyX4YbDpCac8Kv8lic46ac0mcsLXfYMZVvurDZz7Ihx05Iq\\\\\\\/oBEsgI1GoD109AVQ9hsgeHMkVMUrJZHzY6VIES8vh4PBoGEk184HVPg2Ip2syVIA1WQC0z589aaHvCP9\\\\\\\/cnlaTfL20CE1LlYwhqJnHkiCW+7SPvg6nGbitF00gYnPkdwoPTeOzqnIJMf\\\\\\\/FKX\\\\\\\/Dw93S+gBnsx1awCicO5q1YbTBe1ZOMfRLu425epNcxhlh9wdeV30PE1a0TLz4E3Y2KmrnEYWiA6XfycZn3vyUEMArgtoxrnUGUOyhzqzP6dBO4GhBhLVYpOl8p20Jm1k0lpeUxwQaElhUbQph9TlgKUSOfBoLMYhdFLYaU7SBg5uma0sB7itFOzbeFlblu6qPPG9E4EPyPkA3tkTn97yLMfMBClEuZXaUUlyJOGSovpxvluoakK+ViY\\\\\\\/yw1fjMaN2FGdSwmW6cY88GQtCBJnAmNFgN0yNYfy6YmS4WbsWjlCuoIp1yra36xIp0jhaw9zRdadooT7g74FjmH0rERSoiQnOAfoi0ZEwH6ig\\\\\\\/KqIwRs9m9VitWnokucfY8e0Pv1GykNBP9wEB0N4frT3tHJJoeJWtSb1vO0e6fl9FoVvdo0TpLvH6Eft6TDw5cQeYp6xuOGbo060EY8AyIfC9319j5b3kKX13FFdT64IrXsdpjumEleHujcgckDlY5Bpz07oSTSQt9YgU\\\\\\\/NlcGOwW\\\\\\\/FpHm\\\\\\\/N6T31VQQ3nI5OUZD7gduygCym\\\\\\\/lEEhnrA9GQje5XpvcuGfQIiqDs\\\\\\\/BiipsrL0XOf6JQlgP5An6ShedcBudociehsNnNQRdimRRWuz1ECbcYReTXSYyI1LP+gXrMoXItbejEToWEHdiKx6Y38XxTSltV6wkFQT5VkOFJicsga6I0sm3uQx1KohOd5tPVunSGJD\\\\\\\/JYAlR3S\\\\\\\/oCU8dOQh8lFzCO+ZXOvGHKWbGvErqtQxhvcQM3dqdFx6OP3j2+qCNd1eDtDWuRs\\\\\\\/g7LKv7oZslMEgox8JC1IJ47lhVz2Xe59uigcn7a2ZzsmoRV3jKNqPhQ19qC0mrYlpluvgt4eF5tUMgSuL34ueu\\\\\\\/EegKI+ush+U4V3Nn5jR\\\\\\\/O\\\\\\\/7dYgirEoDA+m1Igf5ZOIUWoZ8HCYieSAY7HFqKm2CDOLFvnv8W3r4U8beWE7vN3vkhsTYUKWQUeSylYWnkcFOexIhXtQh3nsxBRvFzFKUhrj7D+8P3UittHhGIm0sjIUeYfhDjLGG9GgMwtUb3Fgy+3gGvExfaixMOU9zMqTxXYl+NEr9bmxKMCXBf8adsogwnOdMC+mQMyRlL5jVAkZoTmL+pzEP2glKMbEXmGv\\\\\\\/y0GciB+IxtCHEb60+QTGjliOkt\\\\\\\/0mli+5P753vesFcUn3wCbtyTzdlZHsvIwZEhFJKVh3IkNECWxTCDCTUoji\\\\\\\/NNdNgK2N4Aec6M8f6xV3\\\\\\\/aZ37xdRhEkMaWVgwP2lUrGz3R6foo38S1AnwNmF+bUXlMQq2FVTINiRXggZsrm97JGJZfA8PGYTy9y7O59e0K7Swbfrx+qfZ9qEH65gV\\\\\\\/p7MBabK1FI2pWiLBB4oRJ25tUBIqZL+NiBIYmXAB3sCpPeVAXWuVB6SCCB+ILSSSArXGCh\\\\\\\/s\\\\\\\/xSYbAVw6opdEqK0jtwSxXzWitZM4zpoingd45RgStZYIr\\\\\\\/dWCVzYnWT+YwlTqHSmp2GP4brPQMpil1axMiP9a\\\\\\\/RJM0BF4zcuxQb44dui2qVTldxYW8cm9kxQvpquyXdrBos2tBBEd\\\\\\\/p27nH8ZxQIPLWCMx3bkOe6hOctqMpUMoWNfkCxq8N7tSrsOyhzPbhawQz2OPsS\\\\\\\/cHerJgt9HXDkmowwjySx4hiuKkkffM746XLDbaPcCGbCHPD1v6iYDV1treRp0CTiStFS0W8X3YKX+XnBEB70+yoKcHMW1GNeZ5gBFfsrtVnXPxmIUCuJnS1548hPoF8+1Tl4DY9KgQLWIZoH9aY\\\\\\\/PrKSdB9tOJy8FQfNqItIsrdQ1lHYFpIqxcJZNr\\\\\\\/SGBeQy7pLm6d5slbDNCqdoKIRWPNkMDJD1KmjwGl6A22HjczCHW4y+nlnOclaGvT7FZdJR8DL5IG\\\\\\\/cXEvxVKPE32cQZPEywQ2rI2zfQZuAmBlVqOHTqjibGXh8Dn1iKYhbwH8+YO1BemTawCvnorMhQUMvQDy9DpMBrPawiJsZcnftHb+B09YvSmUx+T49T6OITR537we4P3EvJpdZdIvIqNZ+Y8Du3j6f\\\\\\\/B5K4g\\\\\\\/9WOvmyS4WTB5Aw2KnxAYPyylEGs8mm1MzEqqwWHqimKmKyJER4NGNS\\\\\\\/UzyQGXrwPLnvilqw2QT34nijs0pr9\\\\\\\/UHyUjsg5UeC5DbsUTn4gMpmfCObPGvtpYaI\\\\\\\/XHMWGEhg9bmnxo5SzWEPjAVuzPl5NvzZ9piKl8cYLdVR0sCSL6c3112fhZOnFPHgJGOfL1eG88jK9D\\\\\\\/kpjalQlHDdQB8104tvUT15ismOeIpG1lTJNGD491AzdM7leronCuxpUVVA8kjJiAQQjsjbAlMZ7MnaHwzSZRs9bbDPmnoEnbURE\\\\\\\/\\\\\\\/sFTC2FtuD345K2OI\\\\\\\/utL\\\\\\\/lyvh2f96JrJ\\\\\\\/ntqxhvTkkeWHg9NoTLyvSVFQqw\\\\\\\/MDQrUx+L3UmjNnjPvlOg6++AmoCD5ioNArOqihCjFtXdcCSXTB2k6f4vGLvD8O8fh2OrEvcFpX1f\\\\\\\/WG\\\\\\\/Tth9zvOun+Dt5Qz8i\\\\\\\/TWEPBVWHmioDW\\\\\\\/DpQoVNFEZq+qlJepNC+KMOjKWi4fYeL0NioFlpKA4rhjxdCkfKsMAfylgkuBq3EWShUW0qB6MsV5mUeUdxuFHG8g14rNQhF91HYPu5jcKIAzfiwsvu6KT77EqlMkcWE4rhzgz\\\\\\\/c6W4hHqxA88FAtALffsl\\\\\\\/RDWuVJbYfDONW6dfAvh79GJT7QIJQmRrrkwbRd3ZF7edVwvPjMF5VYIrdsWVHT2u9THoCaM0+pr3qHQszSkZmbYbxg3rEJIrZhrP5q8FrMq\\\\\\\/pJpcFN3foq4Oiby74EYixc+ef9LUJOHC0AJSr2h37VTB\\\\\\\/h4DpkfgoCQf\\\\\\\/3HBNmJBOQFGJuUxSVLAS+BWW597UH+MuBnXu72VEGEjZPN6fvQKv6+aC8tO7O7hERt4P3Pjpnpbed5Uj39H1AgWjNmdQbnZJ4K18CWat5ABBvaERUo5IbH+sL+xSs3iEnaabjq6X4kdGteoth6nmH7au9NROrH+hg5wa6a+TgG1QjzHgPXXffFYjZmiD6\\\\\\\/UF4Iy5rYTQssJXS49d+cdz0P3u3a05Vbmh8iOBq+ENv\\\\\\\/7Nz4RcWQ5WK8+l7gEiWIOO37DBVBvRg0mz02KSMJ5a4m3vn1NEfgN0Bvhk5ju3sVP\\\\\\\/q6iwQ3soDrri9kKYngJT14z3\\\\\\\/KggXYpc7lNHPFZjiLFP4wLIketEoSXA+hge6ITzKcRVbJrgL12JxDH5d3G+G7MPEqtc555mpqQnretqAHUDrUKzWx95gFPKkCD69oGCqzd8sxKGP70am07XI+8k4n0XcvoX0gjS24biUU20J3dAHt969vqgTAM9lEoIlhNj44o8pz6BmXRN5PYrdpB7WudLuKyO8KmMCmNOeun+t8Q0CFpS4gLP3eyuobmkHWm88mmtrtkfTvkCfY0CXjm0QOLaZizQwvwUkkdRSMe6ZTtohdAHSU0iP8D7O7f1NMGVvMMhEuYgewtkcfCBJLvQUC1jXK4d\\\\\\\/hFsdnMFKcxwF8GXgcrDGid1+HEFYuvPJrOCdcGajfIrnyvVro2LOnBGeyOw89LjnhObihFUre8NX4m+wkJFt1ILT6uMHDMCkIj6+BRIc4ZHi6CzWISP6pMyyZaVszUeY6vAGIcSRtchmxF0Iya5ZYZmBCDOpH+MvgfsGBnoI111nBckk9RFJ67HT2QU9KAhnDiS3R7BA3bBFtKaE1FrKRmb+uRNqf9RXo\\\\\\\/f71yB04Rwk5RMa5QqZNK02v5Oeot2f+wgNJtmuTI3F8JhX6C4Opwct4HuGHkT3K8SVTHDiQdLyIIdOTWZ+mQ3JZc869ER4V8+7i\\\\\\\/F1Ad5M1JWLogF4hrIWzG3wZFjBDUB6jRxxLZFGEepzY3c8a2ei1YoFQv0DiK2pPxUo5yID756Mx2aRm1du0rDSLRuaoZJS3iLjx+jeW1amIKzOCMIzWAgtUexhU\\\\\\\/+z7uyfqGfgnr8tb2f7BmLhhAvpjnE12nGzX78QZUjrJQn4ocpLJD764Yk8Owu7fLoIkuOfY8D+Q4ndQtsouThjovC4lrgB23JOgm+IruMmWeCDYtHp\\\\\\\/wWEhX\\\\\\\/roK4d9Vs9zyx96KcFG4IQzGG0Katc8OqA4RapfjFIZxRvyK1XH1wMnDwPNaKIHR1XYYFj4p\\\\\\\/0C41dVp2naG5yjaSeB3B+3dES83sV9L5X8M6RWkGdoZWMHjwH+HlFW+aXrVo+xDidsCEU+VEF5C3e8WOoF+eunYMPf5DZKW47QeJ7BDRhQsCxxcdtOAnLUcrUdl9uSOg\\\\\\\/CugOEfR\\\\\\\/v3GcvL3Irr17Pv1WPSrNN1zahWyUjuc9gkkUopLyI77IQ5U+\\\\\\\/D7Y7k9pZcZRriw6KZXsBNRL3aZjXDBpp8xojMQR1MmNBmdWrwaTuYgDCFk9FA4luWENrGtLjD87z5J2hg+EKXM3ylTG7MhTor3Mol+amJZXm0YEzka1Ya6n\\\\\\\/iEzAdhZRHN\\\\\\\/xuB1Qsxt0Bqr7eEFFFNr2TvpS4YqP9\\\\\\\/P\\\\\\\/J5IViZvITTeJpxXlFoCEopyLwZtEo42yBAEGHf\\\\\\\/eMGv41R\\\\\\\/kQQvdDj8u3zEDLXJxrAE+gtGsg2ZmgjWSllUMVXpqI1HyY4VG2tehj6NGve\\\\\\\/S3aKcXOHO8zT6gK7Jsqk3wBC\\\\\\\/NW1EdskIEl6tjXfbUB4fv3EjwEdPLKSWicxksBLSSdh9tIC6mZm95lXmf1piKLzqj\\\\\\\/BP6A4ikeUPI27qk\\\\\\\/3OY\\\\\\\/IzytpSIUGR1jxby4A2CkBGlKuA\\\\\\\/k3fOE5sbZ+AQllpv+PUL\\\\\\\/ZdImbGgCBAkbyPGarvWuMEdal5c0q9GFGWpWEKbcsHtr\\\\\\\/LM2Xn7nxYrOIHxz9lw4gNgQpr194cxzwgi36hBh8b1mqKyQ3nNlx4aIV6SNtlxaQKgHNoF\\\\\\\/JBh3Zx+yp1H8oMBe+gMYP3fuLLqVe1sG2adlm4oEJ+nTRf8iYmDw\\\\\\\/hUIMq\\\\\\\/HEvcP86HN11476e\\\\\\\/RN+Foy1BKQ4mCAcs2ws+eqd0m3oCqNlHQb9WDB70nycg8\\\\\\\/cPB0rH+n8BAaJbD0xQtK4lVtszsVsNdZsCx6rykS18HKBiEVmuwbMuUqSGj2j+SeDQoxUGBrILyItjiRAHnKX1AcjApMiHgXy5Jz9OB\\\\\\\/NRqudv5+XFf+C0Ybvoq5bTVhF3qz7QxtYYyRN5tQRSpgzWjux9n08o9bvCMnqup60lIRizDkdKG2YWFcccDF+HV74GHy2yA0M5YmfvSjDBoCDza6ka66YR15HTuyg7LRWG1Oa2ewtcbqk2fnKJyHzNA0WozoxncHTDmJ4XevcVNCvmNEb0AooXRrCWsXYwUYswFv5zObFy1p3h6dp5bqq6tTsl6WBpicvaAPAAMH2wRUX9nh20UTTRHqrmGomw9KQyZoQiCX0dWyoTgPe5Nl20ewZb0vbELpPJAeqKcGFHfU0BFRK9j5W+sM1PMrFsTGJBfGfpjwEvdwFgK8kCerESsjdlc0SRiZkWgY\\\\\\\/ngRYfzy0H5bVdtarK1oPB+EmzQBSEmut6fzYnWHA+mzODs22\\\\\\\/LAp78d9HqGgEBGPJWtpxRa2tWFT5JxnhTjfMFdqdhvJSX64rP0sXua5C4k4mE79ykV1B5lAe7wVFWwtt+I47VCnf7jBD3RzpOZseT5VWQzneQ0CHMDYVtLyhmg+vu6UjEq1uF2+Ypvj8nhSqGAHmfzkQnBc+3RtEtGzr14owQxNscU0MowR1J0igL5OeV3dJfSl6XpQWnHS3AVRSK3ouHDYPhPhXervEA0p1IVCLR5EEdD\\\\\\\/mlfBRyoCx5FxzyH1TLZPdBWQIF50UYmB63l6PzwXnETZAX2PhlntYqPUB0xLAv7Yxo+GjcqsN\\\\\\\/hbTAGk8ZXhUf73sXFj0C5JVdyOg9ATuo0oAoY5EfjcvoJ4BHLOCavUtTRQGC8ZHS85DgTNVGMDb4Hf5tRgI\\\\\\\/Q+i8cvI22TTB2TlpSvOmLuBIWGmm1bvTn3XDSrFZkjcXYxVFuDlYYU8OHwNyD0FSxpQPe6DEA9xenQfSgdGW\\\\\\\/tPueuaXxHuXmtxjbRHsp4j9T3C5093lrNBioKTImZQ6dcWA6jhRvKM4GWw01giJc7Vbdzx43FTV6VYSaa411qm\\\\\\\/NRZeAEN5JNyyCi0Ukvp48pjmB9TvyMCFIwjKWqL86MjbpH+Wx57fKIOXqsd20JDFcFaCr+JnHAXJ\\\\\\\/ULTu0Ner0MVWuNJRXYa2Fec0yN6GJksm3AEXpLdd8BOcrde9Ysz5ZrR032STS5b5dj1Zcx8xf4JqFpw1v31Vlk20zpLg3f+UcEO1wj\\\\\\\/arrdMGD8oeJbcTVsu2tFVtuqvIoTZllNS\\\\\\\/9jL4m0K60IRpJWKQ7pkhz1zHYdeGANvrzlur3r1KAm9tCA1wPFqLqA2KTP8BEIZQ\\\\\\\/Wf3huoOYpVDuVjEtVihE5h0yIETGUcjylTp2nnv\\\\\\\/Wtz8ABPoq59DDpWjWBk3RePovznlKIzu100QG+uPqqDlyaxF\\\\\\\/e4a\\\\\\\/3qZqmHoMg4fI8cS39YptdeXtpMDs5F+ZIH1N2Tc6RweYSGXEsdid823Pqxsef+wU4\\\\\\\/XHKmphaSPVQJhG6gBFy24D6FaT+fSuhtnM4vjprydlOGXNu+q5AosPJO85iuj1q\\\\\\\/6QUq2X\\\\\\\/R2kMGy9gEukT5jjRFe3NlImAlAW7LutDxPbU0kbrGzP01dwOyArdP+7Tjue5oxBjs24394wmAA1YIPF\\\\\\\/KR1yfylN8BgWxGHmPj341zY+AWyP\\\\\\\/e8cxOKaXwUQpyH5QwhOs6BKwaVMMKngvWSBLLmmD+u2jw6v+wt3x\\\\\\\/D91cHaQ6FuZASHlS3bRFS5ikt2qS4X1ljipU\\\\\\\/oZvEYnuqvWrzCxFhu5DIhjHRlHYrV0cOFZ27wmxdB+vghDffXgJyPH8+pTArt9I1Xr8LtEhPMqsqtw+e4txm2sR7SbjZigai0bSz1mEAJUw\\\\\\\/Rg8YElmByXysLosC6P0ZZxFy3Db2d+KmYhqQR8XmjTusKRbYCp3bf434X1iY\\\\\\\/1hGgzitzjOVCn+RaR30PTW49UUeRhsU+XcCRpZcPOEim48yOlM8KIQG8pKYjmzJScN0hcmvambUdZwmdmluB\\\\\\\/Ee66l1lhrOAGVAsu+nLAA+thyx0INd2EjrCjGodogfUgz88\\\\\\\/gdGt+o45D6wnDNvhgnl4\\\\\\\/NqGyL5X5S8Y7q7aQw49w5mQRU\\\\\\\/b2fLo9NRoQeP8ZiLkHWVL4gOOHy\\\\\\\/icLa3InaDxWu9feZAioBQJLE6CGAENkcxFq2WIX6LWk4Ol1DxSBZrDwwMuhxUuwc4OotxOTymiL2rllU8VWRmNlUFZxshQDdlYbooPMzBCG7gvpd7E44oR6mCYj07zqRp4n84NrazhgdiiaRfGpepI8otIcbX17PhBM9k8t+XiPnU6JsXoqi8MJTftlrTDkG5bLXpADoCrPSkqNzI18EN\\\\\\\/UugtXgrHTQAIx3GmUhS+MDgV7RHMKNzqvEVIiCh+EJlqADuzVZ34N8uYH3sUfOuvDA8T+ZWacSOQIDomnHk94tI+41caEnXk4Q47alxBBwy0wTo33cxZK2n+NThNjVUGz7XCPp50WieGn+UFxMb9P6N1IuRyq3PyIgF7Oocbfes2wzTSvgXNRU65ntskkF5MT4RMa2gpf2P3ZW5ZPJYGZKGj\\\\\\\/b649sGUKf+Wu5DJLJ3IJCFzAkSL9mo8ieUMMenB36tGbDOmTcpEfrt5ypfRNWy75LxNujj+jbWZJ7go5gLOcvdgi3A7yzwKgl4380P0nN3T0m3duZ6MY37CdkVoaMPdyhWRZyA+4aioUbLXBMUSxd3xqFgPbmzntW7AUHEvHCQC\\\\\\\/bo34Ux9y83Qc7u7Qeix8MCvNxVxsI1egEfn39IFL1bg4e2wJ6Df3KicjAyQwnYco+N6kv150dHAUTGzHGwo5rcNV1rZ0iZStiZeCFm0ftKQzcr39HVWH6soK5DRBGzxBFe6assaZnl\\\\\\\/ok0ZwmGpv0YJoliuY5Bwrsi\\\\\\\/5NXR9E3QzLco3fBh+x39ujOB7oAbmoGYmm8E0FKx0pQyERe0mFJmIJyS2S5zn9WjKwg3PJ7DZH1rBt+0DA\\\\\\\/pfoS+\\\\\\\/nDMmJSU8qB2RA08Ie07JweUJ8Rf0vbqfG20FDhv+Bk5JNeaSZJk6DO7cTmN+M148EfnReJzbaDFxw2BcOcE33usQow8EhzBZdIoGloT1bSRNS2HlA02uW8s2WXwEMm7rK5Dd9ZpCwMqJMJcgvJMFrdbyvmrQ5rYHwApf9KL9vLijWEg5O15iwsNcrpPoATZVoIR6KMIPIDZApkev91WcWANUYihL8Fh8REhWoSyVRGlmjnodgKiOzylwyBnTwVQ9fh6ZHZ+2ibh1WHCUwRTlngacstjwVSzrJo3tMV\\\\\\\/ElP9TN6lhSutUa4QBTOSZJ\\\\\\\/mrKp3rx2DIhA\\\\\\\/XKY2GE2EmXk9TAshDdvYKZO+E6EbJhZDthd+d6qZjivZJFpdMIatRYn+0Fa5w2FiLhJaGIssHUmFpKjptUWzTHrLc4XKVZV6qx1IS0aUH2IKIrrJTuyFdMczQviF\\\\\\\/zWE3DxR+Y8aknDM5MPNho2\\\\\\\/xOIOqwy5172raL9pAmBJGMMw2c9\\\\\\\/NgKhO8fPGQ\\\\\\\/FDo5kpcQTU4DOsl1C\\\\\\\/Bk5jpDhJSWO3T0Q0voPMAayxL0VBiTWkTksBs97VWy9lfT47o0iTo39gT82juGIviYuYbk3qoi+Z8trb++whX9sZUDDTAMAwAFzNW0AtWSdif3Ss1ij8Oe4jshEs0V3EzCVcnyhmNOUGPFt1ES+TOXlIDSxb8\\\\\\\/Xk4jijHYGZaTeQvZYVUP1kmTCgIAVSg8oYQNygEEbJzVb6DVI6vMAQoAu7j9SQRW9sI9TFFpg+u9Qx6yj1Fv5US0S43FCwHlxzzqTCQ9I5kEC98eaxFKj5XC1wJny5Qj1J6a8ZXucJhvFHb3yTjkjCDKTr1Ro8lQTz05+aEsDumeX4xaz1ZQIkw6yeiLO5vTah4Ahobrq3ZxeW2axENWn7pxOGjfQSxqaDw7APqx4UYuocCoKvcAf5++l7kTQR2+v4JTopi2YdNRuG642FdqDv4bWxqTy8vpHzLFFrQcpiW3ygC\\\\\\\/wKNCXiGKxbs5odoFR0EI\\\\\\\/oGBnNtcxBG+ouQfbpbvIHPheX8CHasnIma6JTueS7E1RCNrE8Cx3oQe4WDZ8swCo96WtMHYXpsiC8ZPlOTbIHyTlH\\\\\\\/fCWDQWSBv1KwskLJDbiXA\\\\\\\/frrBWnBVvDbSRkuWPRE4JR9NXyyY1uTJ\\\\\\\/KjiVx7PM1jZ5bQ3zy7GbJWFzoCbDeyKhZzH8QazxbG74PAIfVNOu1DEFf67URY4KtPZyVlC0lAXSoXaAZCO5SRuz4AlHo31G79CWs9LUAhh6FlSN7FGhV3rD841ED2elM3eRAhFDxxMX5h6H\\\\\\\/zV22QUvWG+fcujC5RyjSgUTUqAjUE28zG\\\\\\\/fPQ0HyTOZ6eevx6zNSGIvimtFtzXmZnBH8ZXJTYpXVcityRcVCCj5gIxsCvyeJO0l6v2o0sUza6d7IDqS9opBIJmRBq3aIm0XL29orR8ZjNnXwNyt1bEsbUdPaLVko\\\\\\\/R1vxFxVMhaNLmRKvrsykrw5qQ8FY5uh1+pGjI2EFLLWmKX6vkBqzeMtaxg0laybarFa8+V6lt\\\\\\\/\\\\\\\/kcWI1dcgFPETQQ4QdP\\\\\\\/v3HeGjSxImR1UWvp9vT3i4bc\\\\\\\/xAkQ8ApTPZIGjOenQ1ggOrLwaRrhtPgtLcWWj5xMQ\\\\\\\/S7nzyyRd8iBe809QFovvnXv4V\\\\\\\/AHatsdf9J18du5zIH+rZy8JeeL\\\\\\\/wpoDgEad9FSXo1qPo2y6RBTowDphodYvSS\\\\\\\/PAIk6uMDAAPixPAjsDZkNsVW2ti063XEbVyywN9\\\\\\\/oIxS7zcKHfUejQMcB1bSEHgX3fGFmaoM1FW9vWP7fVM1CwGxqk8gl0d4R22F0r+63e5bMOfuRJTklLkxd8V7dlkXgX\\\\\\\/faMbVm\\\\\\\/pySbt2SdZSis+qpuvVeTwwJLgVCIeUg0AGmMH79MLFVsUe6\\\\\\\/P\\\\\\\/8at7GgnPWkgbtoLKod0wFKY7u11KQp7rUkHza2ibMndkJJGEouvPJq\\\\\\\/VkgEMDTYuKtWXlhBsBVIXIdBmGDUrQ7XF5+mlj4C5262TH0NgefgGgaY8BKhdHuHJhkzUIgu8+JnjjPsrEkCeQAnZAqmdSHtIQ5CAxR8zTOQSC3j8429MptSmkms5Oto32J8RCh79YShwXKxThesydnkc+yql6KrYPsnYIgNrth9Xe5Z7a8qCYwwc5meH4yQhJzvSA+sZAfOyc8ncfmTptF4iMDZ7YHkL83xvuCbYV4IIvqi9Xgc2mfHnn0RNww4KwGW8HzYcQucAKFjE1NcDsfCTTGR59BjfXN\\\\\\\/fcHujPo7nXPGFDjMeMGvNhVckD\\\\\\\/yeMR5bdnhy2ZDokMFPD+RqTyFIGFl4gtuISKl43a9ZLb3zq8tiiLrUubGQQlLecW2Z3mQVMcXo5fFG1lAt7dk2cxpQKiV4G6jmyfDnjmq7zdTwC3uBokEyRCuHq1cWgJaBG2IK9huuv1H3hcr4s3RnFw3YePiJUxBHtpktkDX7\\\\\\\/LhksCgGY2NCgC5XGccmbQEKims9CUelxstbskgRiaOWdZlVRQVREz1Lhisd70NfpHOI5Ej7Q0Y+HaZ2CMGkrHQDbsKhDOjk1pUctU0g0f55k5QRdm1+Kc+wE+t0g6zJYqt\\\\\\\/udT\\\\\\\/UzCw8PrtdzquaLkr+tnmmV0TpD4Z1V9cDgF5\\\\\\\/8\\\\\\\/O8yxBjg0qsRaRlyvMajgs+8cdla0avbo0NcnPkfuzk3sRx+OBeJEgNwYpxasRJpZfiHzS8kPGTJyoUgV+eCPB3QhrUhTtj5lj1pkPl024+SFMLqKUFuJf+VSeE6DweBtHRtj2pqBRHvvigMLgqoJg9w+dfHKSPVRyN\\\\\\\/KnE4fmlc3EugXZDYm4\\\\\\\/caziUpDTWwpZvFf3iDpDSzXlm0La53krYKyLytJpOOatOasgA64W6En0eiED5eWWXTptbmg+r7v8CAlXzm8w5VVZ+UxoyZ7gSjIPUo31vf\\\\\\\/RdXeTvmBrrq5k0LYuOB36SCV6o\\\\\\\/alblBxG50XMK8khr0DC\\\\\\\/daocYdwA8JqFITmkyb7lLqRqC+Pa7qU+8Z3WLWac3ZV8+1WYNX29lbluXlfnyG1nXWRydvxn5HRSx3gqwqLWoiXQR8PpPLSM8sgmTZ\\\\\\\/ZYp7YjC2ODsmUl\\\\\\\/\\\\\\\/YDHUGWDpj8ywCW002L9vuju\\\\\\\/VHUm0Vh2JniwSDeJoCFcr+7GlVAc6sMRIjDWPcoahO+uA2w8T4KhOnklawN+mSdTg01+DXtZ1fZSXOd2zQX2gL6DgIoLv99v8uh5\\\\\\\/zJmCALAquRGf22Wmslu+\\\\\\\/aOS4gHszoNraS8jmM555tiZ9ulEvvMXxqc8a8aMlw7c+NlInoXSM4ePkh5DofumYxDmpJCxQxfXYYbUt4pZlv+IqtOlEzSIJ6vryKyBTTW3K7lrXZ4kkGRYj5SvXjynUCab38uzze60\\\\\\\/y\\\\\\\/72nOaObL4VkxcPPO4MLXmtGnqYSqlk0m3xgo\\\\\\\/bYfGYY04UAdv6dI\\\\\\\/qie8GoiX\\\\\\\/WwkGRMOjgDH96rCB0W75MK9xtSI6cyq+CYAv2Lxrc7saWVbFZnLfbzLp7pC71BW\\\\\\\/nhd1a28reiiPClZoOyffMGhs1L\\\\\\\/Kx2o4JF6Ev9BKLd2Mn23mPlqZayiwdT+VzXn0V19DObRG\\\\\\\/fDsUrV+VCwUw0Qo5KsazDjC0YPTeqTWqiufHjiOdQLpgdRosIvK0JXbczVwYBVo00EdY+LalnGAf+qlNn\\\\\\\/pXEtWVfUm\\\\\\\/uM7buUIERVsFgQ85aU4imr3v2Q2Nc1mv4Nq\\\\\\\/Q\\\\\\\/+LEOu8tO1IsaALImn9chCwDDrSWQUF8zTuFugN8AMaDbhMfuAzKWJqjzvtriy8rh\\\\\\\/jdts7rafgbJqFDzQbfvTeScy4+EN4fgsotQURZOYM4h0nALh4Jce27QSC6lHDfSqv\\\\\\\/E82fn9qSDFBBnKU7opcQJ8jHOFIbiw0gfIAqxbEJ6O5uR4\\\\\\\/BxRfM5FGdTBYyH0OUznw8D8WyLK7hEnJN5InMsUlwvwrPyRO7Z2EewZGUssnNmQ8P6UcmvrD+NeiD8+l1rzg9D2UbgjV3VMvZst6joUiFPL+QY0EijQs9RCrUtMzYo0jkGxVEo\\\\\\\/sb4luaR\\\\\\\/eo5vTa3KycBH6vG0wB8buwshwBbi6GIbEEti7D88R\\\\\\\/18PH1Ivat7gCsHlI0pShPUDnNEmfH5WXLUSufjMPJeFcA5uct2Zerdy\\\\\\\/T5Sr7fxUOCWYl3IcPiPFIuaXdY8JJSURU7VAQqDrumS0+MupGMOTN9IKmVX0gNCl5m1s2N6rNz4+dKexz\\\\\\\/CadhwkEI16l20fBes2aEUlrgJIwX1hwSCdx2Df1\\\\\\\/l5gCzEJ80A88Gpu+GyeHv5SetdajoBGwAvsekjf20IZOdIfypXmz67N0L5VJjDIb2aXOe9pk6aUGOQiHeNbGdW4KSpDcdWv32\\\\\\\/1WHz4S7Ooe+\\\\\\\/+LjcTrWBR\\\\\\\/XO+1K7jnLnXA\\\\\\\/JgMXdrNFBNi5zWCk8YnvyyE2VP5k1+wzF4Wx5wSxmt9ii8fvdzCxXw7JtCDY8sbDxAwsdKOL3fWViShIRy\\\\\\\/zm6azbnMU0o\\\\\\\/++syno4ddVQOmJ9yqaWp6GDKT6j0\\\\\\\/t0X\\\\\\\/BaQ\\\\\\\/9mkNwPk8blO1OaF1JMp75sObw2RxX2rM1wVdWoSkDvmJJ3GDBCYFWrHLThek23dtuRTdjTNxU8MU5dx9VeoIOQxCk+zmFzPGK5BGZINFj25pu6r6znZPJ\\\\\\\/W4odbJcpSRsgVtils7rStYaPN8NE+KLla8V29hkcJkGR6Sgfi4dyFLbOrl\\\\\\\/YdfqDbCx6UNsH0pGrvdOS\\\\\\\/KwlPJe\\\\\\\/ZPS6m7D9\\\\\\\/TUVBiHhDdITYwhro\\\\\\\/rUO3inDzPc6fQ+Dmbb0dr70jbn2erzuo0qEQuJlO+qLFChAEWkFZKQagd0lqhEsl1RTXT1UQbJWJPkOLcRzaKaFbSeNoU50fvzkc8zTNojILhij5VY30fQLH\\\\\\\/rYwCf1kRtYIuqGw3+5avhd3thEqmjMqIFp0kKz9ASF50wEcccCfydkETw8EIwrShleZrNbzcvYaLpvG4w6gEMofnIjdo5u0qk8piV+QOOWaW\\\\\\\/+y5xE5tqryXEUjbJ7wNdWhS91oiJqKF7WN5HMMxvLXsFlgcTf0EOqHvLmVLvP41aQxoMNuhzNnofuDqW8IhB1bimiL63csK3d9rPUmSrbLTrawK54P4MLrFNvuftN7kxpRxv6awgfivKX2mq3MmcYacjHnLP+yVWijLahaV7v+Djm5h2lAyjBNm+SRkKzAgV\\\\\\\/iJYyuDnVAGVITH\\\\\\\/IbF\\\\\\\/aXZjJug0\\\\\\\/xRLZj+8khZzwAjCXHtZbbt1BeYtZlTP9fDcPpvFUKf3F1DlUgfy0UvTaQ1CLX+bTPpFXd8IztFH38WTuct12o0wvxs\\\\\\\/KkeDNTqYVbeUNQTcdQtEhKStM1jVn\\\\\\\/lio6uvCCx9zZIbJGAlq9xvZhq7CHtSl\\\\\\\/sUevQHIBiBn9BdeYVn7+yrdWAGT3Ws44ez2e2xyUrvhoW6o8041fs1+kYk9QEhjIum6S73+DafXDh0flVqoYO+ZdAgau0Cugc2Acum78a8lGTVX\\\\\\\/3GdkwMUtLLDTS5N8n1o6pAC6\\\\\\\/HMFEQDzVNlRgoEQnrkUP2C6RCge0UJB+C6YsmtKrzsAthZFHh+jxCPZ1O6f0wNzggrN5Ol8FcKROzGDOOD0XgcqyqlFQhXnXTscOGdStkGAXA9C3VOSrIlYPU3HrmUSUzBs8zM8Mt14nJxyvoNkogHfNJ4G4\\\\\\\/i0z2jBGwlp+2\\\\\\\/nFwBGrUuwXFlwprem4Ouh+2wgl2uGcfQq2CXvxYaUPU8EcC\\\\\\\/6fDZZTomoL6rIldZfIYLhFFtW42MntxXr7Dp5+kk6JLpYiMKli4BlN1yk3IQwvizgEh7uCrqIDATBAEaDPrenF42OKdCakZdsqeMi+Gze3PkCkMvni2tSBc1PABuVkjlhpwLkXE5pCY3MqBvbqp77E4Tm5MO1w3lkw0CJVz+XGbXrSpeQJjEyeOQUG7ss8eR5Q5AJAsYqCiqyddkMMtVZgAKPLusG9+lUBiBPmF6CEhmUKOSOa+n50\\\\\\\/zVkQQ8E5B44D3BAi523Pj1h29WLp9ZMrYaeh\\\\\\\/cGHRKUytf+8RabAAmbIj8nvYS9eG\\\\\\\/ifUDqPwlOPjMi9kXK6GdDsEnCj2fo6pziLLuJQ9R5P8\\\\\\\/nJAepSTg0\\\\\\\/7HiJ7Nn+h0T6nm4NZYkHd7p0fsgv8okqA9Dyb4gv7ZFrYzmyWHMbSto197uk+b6V+TgMEq8O+U68vIRpy1R\\\\\\\/5sKhGWR2uNpZCZfudZqccydIclS1uGDxdTlolvJ7X7PA0fg6GLYE05kdcMInkFpn7xyUCGP0\\\\\\\/uc+Nw0+t3brnmwylSsLe930NtZgSdRaj0Jr4d4I3Waz5ilUF6IjUZ3VbBzHndinNCBx8l7lTY9n5TIzMC6CbMWNEcXP02ATiXvqIMMiBkdC\\\\\\\/Qt7VwCjeTJJSyJYoY8Wj+gBjeBrTVEm9xL4qddHjX5pZWijBf2V6jUPijl57jUI0GjErvfiedAbEkxT+11UrY6paNfZZg+Kdk2hQ0tfD6ElyvbtSu4enczgFdxb8\\\\\\\/7OU4tFGzGxj6X+7fMHWDfvvw1b6UyzE43Lopz0BgqIne5KQbptPxjYyQvMmePvK9Haxm7gew2z6kfWHlg09xAe5BodK60g+d8zBGJcfacnZyPnvzAcGToVzVZkiXGdJkmxwOGpzdORIg2cSjUmsSW6Q+0DGBsacCUFFu2cIyyWGUFjCVKLvFAa+kTGJ7Y\\\\\\\/dK8GHz11VMFjl0KFkVs84NzROu2ipPQTZDc4QCwEk0s3f4fM4MAZcN732W7q6Tyc5o99zlCwE5FLJFnl0O\\\\\\\/PE+8mUYcXVks3ipZPQeG6IbLCzy\\\\\\\/0V8iKDoE1rrRf7qNmsC49DSH\\\\\\\/8KNQYQBZJ2oRepwtHN2F4OhyRHgddcgSF2cTBC4pL3wiJJgC0gz7Z4Oh8hxSwkeBCz+NtALzIHJkIBIoozLKvwmHSqycI+ovUxR0Stj+fiUFOf3Dj5uFYMeO76G8DL\\\\\\\/PberXW0fxvc+1Y4F4vYFwDPxh1Jkdu31dUzTtwNWLOE5Ka9gcWNOQbDNmDObR+J1w2X\\\\\\\/xh\\\\\\\/37nHVCZvc4SOiTQ4lNFKhdZ\\\\\\\/\\\\\\\/03RdOhgJk9evm+\\\\\\\/PZUzEsYJrgjy383n0qKY3jgZdxXQLoRsrIJaRks1UT77NwyQRiy4TqPK1zz5mN0Fgxf6kcwVXqLIhDLcoIoDU\\\\\\\/U8n8TPJYGkqkzjNzztxqIMYA1Uoq9ctwDafxmPMLli68CErCwmdi+zA6B009p6Q5FoikS09krJ08T8bP9LUwuOUKGSvOeCZAdRpIR2xm6nVLY\\\\\\\/mRPK76UvNEsPXLX+PJK0uyxmYDY7pSbsnaXvjYLGhBpK\\\\\\\/niBqo6+l2kfzPejo+ocgQnhlVqo9ItvTLn6xPrVOdgB2uxEg0+2Y3xtUNJpxOyt9fxnunVVHDYBYilyWB4DjM8Dga3eV7eN8R7pm5gRcLoc5JLEarHnCJ9Z1Csn7qwuG5rLwYyzJS65EOfs379usBmnPyU8Hy41kv94ASkNH9LeXgIbOoln\\\\\\\/Smty48OZ6WcVtmq319vxpbGAbZUiAbnpy+7\\\\\\\/0YFL6OxwvpiIa9NJJsIE7z4Y3tPevQQM5ubD9+NWyw9OP5Ixc\\\\\\\/aGNS\\\\\\\/bOcYJGARtOy5JNanR0LNXmWxA4mt+Wlhh5n7gZOPVUIxHTw2usfqA7d\\\\\\\/VMBljwPZ8OPnUQtoRTKUVbz1JTMi5G7F4w9lwaSjk1rx5IigF6E1Ix7XkKhlqq4gZXkImM6uPDtPvgTuPd4RZp1P3u7x099yGsXdTWIaBwJj6SD+rM5Xviq+\\\\\\\/fSdkzp2H4\\\\\\\/P0r809Kbw1LtXkQ0Lxw2NNjrMRE6BQd3MfN9h2VQw5R9ajuDoyFIk81IO8AuV8cWH3WWaEZS+5gqjYS9TSJJV+y6aJ3R+H5I5BZqKAj7uxu\\\\\\\/fHVaf9JoVAa\\\\\\\/nalAFyxXLtiStM8pcqVdOphgPRG7HiSbyHkzgHU88vLWfha1WcBWegT2T92EDqgJFYnM1FMyHIwmicV+w8TIufc1AO\\\\\\\/bLlpJ3uIDL\\\\\\\/ksVZitdZtMqtHc54NQcUHqv4Dzne3fsaznwIYhAcKIn\\\\\\\/WdrmW+bAGysRVb42MDma5pWUhwqdQrGBhOa5sJUk\\\\\\\/lXYQf6gZM3Mf0cLVLZpQQzXdbRoOfZfXEV69NpiHxeFFv4MfuXBhUADDDgX3BQQnmPJcXy3Z9TqH5BM8Ke2KYsoyU1\\\\\\\/SScxaWLZpTdRUT6+wau56r73sKcKabBYO1lAjH0z6nH8q6+An5OjEhWa57JsqD\\\\\\\/yOvgPkFOlhcIQPdy1IGGD9fg11HlH3SrgTx1Sc9a2ZfQQf74BqgFEy6YsqO08ap\\\\\\\/+dx7coIl7Gi10nTG\\\\\\\/Bq+gP+foXD0jk4opgnN3aBa5mvJ1EwOVIiYPM+Y7oP2A7qHabNe6bH61fO6xX4PF6M6gGX2nzXJtZruolN\\\\\\\/C6r\\\\\\\/W8eHjRGPWMDTaLh\\\\\\\/uhBQvlMJHtgx\\\\\\\/TLucYMaFOKZ1bHVNY315wXdJZEPCe8fQBYKyd7hzH+BeQLCwJYF\\\\\\\/vi+R8fO2kzp7qLwhNFwi6a+9RKT4xpBHm10644zd+tGMPhN2MX9\\\\\\\/U3TUEB4K0uVEkEr3KURg8BcTz\\\\\\\/10lWbMzRl5jM9JrjED0f80meXWMqHU2ar8cMPFHgSx64NenzYe6+fzFrHVmz5\\\\\\\/FKVBbmgtTntVpRB18Yh1xlPEP6INgyehap5vykWHxqgF3pVf5+9iD8LlAw9KXm314ge\\\\\\\/8+eIFECxYS48236vULhbXOJ2zYwgcyBNzHQAhzQDo8jHWyD+9TUq2GidE\\\\\\\/+k0szSYmmKzDSrG+WnjPc2hXVXjIpBrgeukEf9uRbakAgOEQ\\\\\\\/NBub\\\\\\\/UUSxe1Xgo8NBP1zEq++XK9hk3jbEEPl2av4vgSv3MF9YrlKkZ7bAthFSXmV4My\\\\\\\/wCFg1vuLc5q7oEnREjE6DbRz5ttPwbrXQ1xuQAvQQFEpBiHJffOi0j8LSZBIJdudzUuGfjOGNnDeIhuw6QEnuHSx4V5fN2mjrh8qLwPsY14IgS9tx01XwaCeDBv6AMWwK1brw\\\\\\\/LkzU9iqBkKCc7cZ\\\\\\\/QUrXe6qGW7a8SBCGIwsHQWl2DHWJwvZqMCF+opwAOt35k+oxYd8NjgekraqlBlJDSMs400n7zdpPpSbga0s1ZzLxIVRMPh\\\\\\\/b06SY7QrgqWnQdf6wI9t1ByPrZe9vOkJRIxXiEEabZ69mzn7xqq47WBrUaD6W4HnmAi3bpcgdYyDLMjiNDJNB2zCj8Wp6GuRu+yJ+IFsi47oSKTlKCtSJaSmcZO+qY7RWqDfjZkFfpM4\\\\\\\/BVFYMMKidxK9oI+zLSy4ZRDJECvr4AY\\\\\\\/JRhozJAHqghxpDUWzVsEbYxuJQxirAkAS+8s0o2PNRXD11ANs5AHNZNPKpzDaJ5uOh\\\\\\\/jFYHZVCxpV9ksB9V1z9TneH4VsDYeP75aQ+biDt4NK1I+LEkQGgyjswiWv1fx3H68kZmuAZxXxKFQ7bkCVeXWVF4HtBytJLMz1wEiUdBFYZB2e7teOES390zxg2OS8xVvGU0HTwRS7+Dp72RTZyLtKL2c+4d\\\\\\\/L+VVj1ahQEUxWeKs+pLp7lp3gR1HGR\\\\\\\/Ivvyk64H1rG7xuOeYK6hn52qFuVTOIY9B6kY87MR6w1PmAzkSq5p8qySMwPAN+zuB5Nj3mddAHH0T3O8dVr6ZkZaZ5v90a+gAlhQCNwKKRLm8kZtsFVI8+0u+6ZR7\\\\\\\/CaKNpeLfZAO6NLADS6X+aRj61Y\\\\\\\/spRzoVkCBPvTdTD7rSmiYk0ZmFYd6onxWoh+tlFHP6jxOImwzaA6D9EcTMQ9a73Rkq3n8J1aeZWJ8RHI3RxiJrECc0s8tO0gEthL5UWL\\\\\\\/ikyXtJdqq2v3mjL6xzIrLJ5F63xfdohUebyKF+XC9tZtPdnZRwzqlVlPFHRGmI15Ke3ZcHc+hnYjkIu\\\\\\\/f9oJhgBAgrL5JoT+dAVpgYvC7exQmsqFu\\\\\\\/Am\\\\\\\/LAjGFrYgsXdK306V4Vn+PNFJC6jPtmKO+m2bFQd+EASeQXI5zsl6ZalQL3bazsXBWbREYjkENICnE6U8PS0ZB+C2JmXlWHlZ4f3moEghQBZY4vdYyD3knCxOqhDZF+c0Nol1o5NvDH4Dcq0hATsRp4LzrUMYjQs8Wt7lvEJGNJ7qfHe+sffuSpEiFOmsGT9KndlIkCeqsVMq3sbLhB2xgiaR20tDDlUOoYZcoq9t\\\\\\\/tPdQ8P7DnKrwC1hpa2m5in\\\\\\\/m2wALinEX24OT75I7\\\\\\\/JWiieys2miPEGhI0ZXeUC2otmANEtValupBRRPbdZuZRH56rzgOA3hOhAoncIHzyHQ25CGwaiGuX4m7grj+8E\\\\\\\/qkS7UK+Ovbi7HRygY3poSi0oPRw1zRpcYMUfkV0jVxZHvkAhhlW8uR7WOngLCkQjtxSeCFwwCEkgGpQIIyWpxKo+4JBCtrGDWoebhXakd5nndANlDpLZ2jvgXCQI7cq7TmqcGgN3XmgflOoeYhlJszFWUo9TwsOVrpBeL+pXaq4eVFdZqSE6AGShhkPZGGzOcDEr\\\\\\\/MdnpSW3\\\\\\\/C74Pf1glC\\\\\\\/q3lUTQVO\\\\\\\/na6CCSDDrM\\\\\\\/3DCmVqZg2X93oTTHGUzfor\\\\\\\/1CXgFbe6s1T12jcruwAF6srml692j6KpTJljZ3NFBflU20wBUM4bRR275oQhA7rpIRDS5q6uNQf53synP6B4dVeJKdfZI3knakzC+Zle6B8rVUTxJkC311I08kgH05K7XT1aQ8a3j018fLHEnAWNPylwnBL+UJM6tbuGhnAiMym1JJprsPDkCaHKK9tDKd\\\\\\\/C+vhl0lpZa0MQ0tGuLW3XXYnYDrnmaxcnxxXDnv9uo4lyyr9dT9nfA2siFqf9WCdV6KQljc80wkx7flIc2Ah\\\\\\\/6Iz55q69O4vvshVemdQMzQZsI9nk7VZOnvIVw7ntOJqXfO0xbMu2W9qdURTNjKHOHhFE81LtZ9d\\\\\\\/Pw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228+5XYDYJfXn9uFezvQQtXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d2187735120e29a0d523d71d9f0dbd403e2459a700d9c93bc02f1d8112e77caa\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DbGqbaZpFyelmLDLRDRsh0sVHb8Znqwgepb3NoD2E42aPxPinOw8e+7KOmWbxX\\\\\\\/niQiscaSba8Wr0DoT79K\\\\\\\/1Zj0vdMmoMtzh2HBYcyRAktuQyNgJyj52GbBmmaXZ5soq8ATtgsDFOWe4c0yShgFRXFNxPDSKmhljIkcnmVMFXvtgqpT4hegaQ5CPpge2y7cE\\\\\\\/XU+mxG9X36tLfvMBX3XKWDsNq5RQp8v9UZGSSetYl1SMJhSa+1Z6kn4jVe2gGQtiQNgINiwi1vTGvztSAcFxN9u\\\\\\\/HQqj1gfFHMSkttxu65uH5E1vg+juSbicWvvCkS1LNwzbPRlqMfcl00sEIvz5R\\\\\\\/MKO1amAQfbGCuP1y3LjK0Z6wcAlDKsI7feW2cHMmYqQeGpRQQjLRanSVVm3w5Vo21y1cAohTKmoZecsKOcKW63UTd2P8wGf996\\\\\\\/Vt6nRxS3sJVCABZ0g+l\\\\\\\/ZtT\\\\\\\/t0iQpYKTAPzi32fNednhL4HP02lE0WZ3qCqrx4oTpk4VBS3t0xZcV7lIBSV9t9FHLuq3Bb5ZUC1k965cRpHVB\\\\\\\/cekigrr1cUsjBoRSO\\\\\\\/\\\\\\\/CSNozvD3ClF4QlDeSMYMmthy9IUkKQaCBJvXAAWmq6GOQzlBKV9sbkAo5SgM2vaEFqnVARc3l5KfEWAjp+as90tCtKcDHUOc6Lm28ICxAtXbF2ddV3jEweVj1fx7rVsM3F1IfqqR99Hp7O1nJ8+LQFDwEEviD5EXD6xseQAhk6JA7DUEra+JwaU9zFtA7lLr5GUs\\\\\\\/hYj26tgOmJD6B9YI6dQJU9EWe\\\\\\\/3q8oSO+N0UOYyPTwpz+KFSMYGFkvCPGzpIyGlUMEzBpzEchBmtiJCgdRQ4uFe3JMtAiI5eleGH0EIdbFJxKGPI3RgJnF1mekXO2Ymrf\\\\\\\/q\\\\\\\/SgRHQkuEnLtZypGXo70mBimneoam19NsfvWBylhVvzvHxVbHMGFpc2kdUhlx+ZOomMyK+YgQxeCHy5O1hyQxR6\\\\\\\/fbZ6CbnfW1friekC4u2ff3GKxg\\\\\\\/mPW48pO814QtzwmlnJlqMNjGNV66t6bpaGqmBdZE9Mps8YIFMrwgOycv6T1qaQN+XUVLCochwYcUDqDrJlfE6CZuOsGg8Vh8WsJcgs3dJ9xeB1iboYAu3jIZqMyXr4ZDbH7d1Ovd59l+8Q7Bo9LI0C8I\\\\\\\/T3x8sCYiTEVXMdxLogJ57eHscQBQsNGHzvO3x\\\\\\\/8HWCpTde9vG2Qdg9TT04ii\\\\\\\/jsLvkHRZVeWpkaIfSDX7O3LzFEC8Q+l\\\\\\\/VVkjZfQ8nVXUNdKA4n555aBhZIHSUdTYD90mVx5RJJE7bilfTFcp2PTFtYQS8wza+PrAp3GR\\\\\\\/q+6gKBVKHUHTT7GyWsxcu+hgDhREkTmERWVESGwaZXFsIEAIBFSaKE2i36Lcr2uozhcoxWEdfof61IMeoRDh8X\\\\\\\/bnIbXr+MmCA3cwGkpEhGurx\\\\\\\/wEh7xJplI8tO4Rt6I1+1U6KliPlcE+HOsA93kgTlRwhUXrhO6ihK9SoZ9MQFFcj0+Z2F07yYm8pEx5DTITtMYN1JNVQpUMQ6h6XuNX\\\\\\\/YePXNi3j7t04UFePh\\\\\\\/FTLHfEx5QSJIuA53oXnXMEF6Sak+5Bi8ZBQtvH6Z7vNxoC5kxqhiT1UIJV1Y4lFgi\\\\\\\/UTRIAFhRbwiC3Fawb0AHnrecNqRlZCMw9K82Q7tCQ43HBFFCuAkfBA0f1C1\\\\\\\/3PL783f49Lzn+DKWTfzaev90klBB6sdloRXtimdz0JLNCjTVqKViqEI21BexdiAXbcd4vqhrgLQE4eSTRnVLKUNdNUU6bolIgea7yvmaFgJbSLkicQtw84ieOthu8ce+TKgj96IUM7V2IHbgrBJLxd4M8qkAa2kMVY8Ki1+cuK7luJVkLoTG+k+PMHlChN86MMepKETolANRUoiE6LmUbJUFQGfz5XiEXuj31TcADREfG0iTUYznk2yfcTjZdfqPGoaJtlNkSemfwJhlTFtvaZ5v37ckFj5TCJabu\\\\\\\/usyXbIwEhoDerpDKYkLg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/\\\\\\\/BJlwejH4rMBS6WEdcukw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002258b291cd4240144c9467d626634d9f28605d077e8b2d5bb13d6820a1a7c54a51\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491518 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3829" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CCGYhE7zbIB7Iwj8qAvNeookV1qPMOSa5dD0cGUCHqc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3829" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HNWtOcdsXW57TBs3pTkeiR7uce_FHHxsH72m5EL7Vuc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3813" title="Multi-environment Genomic Prediction of Plant Traits Using Deep Learners With Dense Architecture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3841" title="Regulation of Root Angle and Gravitropism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491518" id="node491518" data-pisa="ggg;8/12/3829" data-pisa-master="ggg;g3.118.200728" data-apath="/ggg/8/12/3829.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span> and <span class="highwire-citation-author" data-delta="5">Javier Martín-Vallejo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3829-3840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491518" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3829" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491518" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3829.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491518" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3829.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491518" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3829.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491518" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491518/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-trait and multi-environment data are common in animal and plant breeding programs. However, what is lacking are more powerful statistical models that can exploit the correlation between traits to improve prediction accuracy in the context of genomic selection (GS). Multi-trait models are more complex than univariate models and usually require more computational resources, but they are preferred because they can exploit the correlation between traits, which many times helps improve prediction accuracy. For this reason, in this paper we explore the power of multi-trait deep learning (MTDL) models in terms of prediction accuracy. The prediction performance of MTDL models was compared to the performance of the Bayesian multi-trait and multi-environment (BMTME) model proposed by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a>, which is a multi-trait version of the genomic best linear unbiased prediction (GBLUP) univariate model. Both models were evaluated with predictors with and without the genotype×environment interaction term. The prediction performance of both models was evaluated in terms of Pearson’s correlation using cross-validation. We found that the best predictions in two of the three data sets were found under the BMTME model, but in general the predictions of both models, BTMTE and MTDL, were similar. Among models without the genotype×environment interaction, the MTDL model was the best, while among models with genotype×environment interaction, the BMTME model was superior. These results indicate that the MTDL model is very competitive for performing predictions in the context of GS, with the important practical advantage that it requires less computational resources than the BMTME model.</p></div><span class="kwd-group-title">Key words:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">deep learning</a></li><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">multi-trait</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/bayesian-modeling" class="hw-term hw-article-keyword hw-article-keyword-bayesian-modeling" rel="nofollow">Bayesian modeling</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The key principle of genomic selection (GS) is to build an accurate prediction model based on a training population consisting of individuals with both genotypic and phenotypic data. Existing GS prediction models can be grouped into two main categories based on the number of traits analyzed: univariate-trait (UT) models and multi-trait (MT) models. Most of the time, UT models are trained to predict the value of a single continuous (or categorical) phenotype in a testing data set. When there are many traits (or variables), breeders need to use more complex analyses in order to obtain all the necessary information from the data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Everitt and Dunn 1992</a>). For this reason, these situations are handled by a generalization of univariate models, which involves predicting multiple traits; the generalized models are known as MT models.</p><p id="p-4">MT models are concerned with the simultaneous prediction of multiple traits based on the same set of explanatory input variables. It is assumed that MT data sets are generated by a single system, most likely indicating that the captured outputs have some structure. MT models are designed to more efficiently capturing the complex relationships between traits, and most of the time they produce more accurate parameter estimates and better predictions than the UT models. MT models exploit not only the correlation between lines, but also the correlation between traits, which improves its efficiency. UT models, on the other hand, eliminate any possibility of learning from the possible relationships between traits because a single, independent model is trained for each trait separately. Another advantage of MT techniques is model interpretability (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Xiong <em>et al.</em>, 2014</a>). A MT model is much more interpretable than a series of single-trait models because it not only exploits the relationship between lines, but also among the traits themselves. In addition, a MT deep learning model has the advantage that it does not increase the computational time exponentially when going from the univariate to the multivariate version, which makes MT deep learning very attractive.</p><p id="p-5">MT models have recently become increasingly popular in GS due to their great capacity to predict multiple traits simultaneously and also because they help increase prediction accuracy when the traits are correlated. MT models are also popular in other fields like ecological modeling, energy forecasting, data mining (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wang <em>et al.</em>, 2015</a>), computer vision (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yan <em>et al.</em>, 2015</a>), water quality monitoring, forest monitoring, load/price forecasting and medical image analysis (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhen <em>et al.</em>, 2015</a>), due to their great effectiveness in solving challenging problems in a broad range of applications. Also, as one of the reviewers suggested, there is a great need for multivariate models in emerging fields like high-throughput phenotyping, where various traits are produced and not exploited to their full capacity due to the lack of adequate multivariate techniques that can exploit the genetic association among phenotypic traits. For dealing with multivariate data for high-throughput phenotyping and longitudinal interdependencies, we suggest reading the work of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sun <em>et al.</em> (2017)</a>, and for studying the relationship among traits with multivariate mixed models, we suggest reading <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Xavier <em>et al.</em> (2017)</a>.</p><p id="p-6">In the context of GS, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jia and Jannink (2012)</a> and <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jiang <em>et al.</em> (2015)</a> showed that the larger the genetic correlation between traits, the better the prediction accuracy of MT analysis compared to UT analysis. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Calus and Veerkamp (2011)</a> reported modest improvement in the prediction accuracy of MT analysis with regard to UT analysis. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a> found modest improvement in the prediction accuracy of MT analysis for correlated traits in comparison to data with null correlation between traits. Along these lines, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">He <em>et al.</em> (2016)</a> found that MT analysis improves prediction accuracy for correlated traits as compared to UT analysis. <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schulthess <em>et al.</em> (2017)</a> also found that in terms of prediction accuracy, MT analysis performs better than UT analysis, and pointed out that MT models are better when the degree of relatedness between genotypes is weaker. Also, there is evidence that MT analysis improves accuracy when classifying and identifying superior genetic constitutions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em>, 2018</a>) and helps scientists understand the type of allele interaction involved in heredity and the relationships between the traits under study (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bertan <em>et al.</em>, 2009</a>). In addition, it increases the precision of genetic correlation parameter estimates between traits, which helps breeders perform indirect selection. In general, MT analysis is a powerful tool for clarifying the relationship and the effect of each studied variable and for building more efficient prediction models (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Castro <em>et al.</em>, 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang <em>et al.</em>, 2015</a>).</p><p id="p-7">It is documented that MT models have some advantages over UT models, including: (a) higher prediction accuracy for individual traits because there is more information (direct or indirect) and better data connectedness (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Colleau <em>et al.</em>, 2009</a>), especially when traits with varying heritabilities are analyzed jointly; this is true if genetic correlations are significant or substantial with low error correlations; (b) simplified index selection because optimal weight factors for the total merit index are the economic weights (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Colleau <em>et al.</em>, 2009</a>); and (c) procedures for obtaining genetic and residual covariances and incorporating these in expected breeding value (EBV) estimates for across-location, across-country or across-region evaluations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schaeffer 2001</a>).</p><p id="p-8">Genomic prediction plays a significant role in the selection of the best candidate lines for which there is only measured genomic information. Achieving accurate phenotypic prediction using genetic information is a major goal in GS and plant breeding programs. Accurate prediction models will have great impact on selecting the best lines and on breeding program strategies. Various approaches for modeling MT data have been proposed in the context of GS. The most popular approach for MT prediction modeling in plant breeding is the use of mixed models under a frequentist and Bayesian approach. Selection by mixed models focusing on multivariate analyses is a powerful tool for selecting cultivars under the Bayesian approach of mixed models. One of these models is the Bayesian multi-trait and multi-environment (BMTME) model proposed by <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a>, which is a MT version of the genomic best linear unbiased prediction (GBLUP) univariate model. Under a frequentist approach, the restricted maximum likelihood (Software ASREML; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gilmour <em>et al.</em>, 1995</a>) is one of the most popular models in the context of mixed models.</p><p id="p-9">Deep neural networks referring to artificial neural networks with more than two hidden layers, have been explored for prediction in many domains. Deep learning (DL) is often presented in the media as a field that appeared from nowhere during the last decade. However, the key concepts of DL have been developing for more than 60 years, since DL is a specific subfield of machine learning that deals with neural networks with more than two layers. The applications of DL cover many areas, for example, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Qi <em>et al.</em> (2012)</a> developed a unified multi-task, local-structure predictor of proteins using DL networks as a classifier. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> (2018)</a> used DL models to accurately predict blood glucose trajectories. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Spencer <em>et al.</em> (2015)</a> developed an iterative DL network for protein secondary structure prediction. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tavanaei <em>et al.</em> (2017)</a> used a DL model for predicting tumor suppressor genes and oncogenes. DL models have also made accurate predictions of single-cell DNA methylation states (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Angermueller <em>et al.</em>, 2017</a>). <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alipanahi <em>et al.</em> (2015)</a> used DL with a convolutional network architecture to predict specificities of DNA- and RNA-binding proteins. <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Menden <em>et al.</em> (2013)</a> applied DL models to predict the viability of a cancer cell line exposed to a drug.</p><p id="p-10">In a recent study, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a> explored DL models with densely connected network architecture on nine extensive genomic data sets, including genomic×environment interaction, and compared the results to those of the GBLUP model. Results showed that the DL models appeared to be competitive, since they had higher prediction accuracy than the GBLUP in 6 out of the 9 data sets in scenarios that ignored the genotype×environment interaction term. However, the authors applied univariate modeling and did not attempt to add other traits to the prediction accuracy.</p><p id="p-11">Based on the previous considerations and the need to adapt DL methodology to the application of GS in plant breeding, in this article we propose a MT deep learning (MTDL) model for genomic-enabled prediction of multiple response variables simultaneously. To evaluate its performance, we compare the MTDL model against the BMTME model, which is a Bayesian model for predicting multi-trait and multi-environment data in the context of GS (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em>, 2016</a>, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">2018</a>). Prediction performance was evaluated on three real data sets using 10 random cross-validations and measuring the prediction accuracy based on Pearson’s correlation between the observed and predicted values.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Implemented models</h3><div id="sec-3" class="subsection"><h4>Bayesian multiple-trait multiple-environment (BMTME) model:</h4><p id="p-12">This model was implemented by <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a> and for a complete understanding of the description of the BMTME model, first we provide the notations used for the matrix-variate normal distribution that plays a key role in building the BMTME model. Matrix-variate normal distribution is a generalization of the multivariate normal distribution to matrices. The (<em>n</em>×<em>p</em>) random matrix, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-1.gif"/></noscript></span></span> is distributed as matrix-variate normal distribution denoted as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-2.gif"/></noscript></span></span>, if and only if, the (<em>np</em>×1) random vector <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-3.gif"/></noscript></span></span> is distributed as multivariate normal as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-4.gif"/></noscript></span></span>; therefore, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-5.gif"/></noscript></span></span> denotes the (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-6.gif"/></noscript></span></span>) dimensional matrix-variate normal distribution, H is a (<em>n</em>×<em>p</em>) location matrix, Σ is a (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-7.gif"/></noscript></span></span>) first covariance matrix, and Ω is a (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-8.gif"/></noscript></span></span>) second covariance matrix (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Srivastava and Khatri 1979</a>). <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-9.gif"/></noscript></span></span> and ⊗ are the standard vectorization operator and Kronecker product, respectively. Unlike in a multivariate normal model where the data are concatenated into a single vector of length <em>n</em><em>p</em>, in a matrix-variate normal model, the data <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-10.gif"/></noscript></span></span> are in an <em>n</em>×<em>p</em> matrix where each column is a trait (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em>, 2016</a>). Therefore, the proposed BMTME model is defined as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where Y is of order <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-12.gif"/></noscript></span></span>, with L the number of traits and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-13.gif"/></noscript></span></span>, where J denotes the number of lines and I the number of environments, X is of order <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-14.gif"/></noscript></span></span>, β is of order <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-15.gif"/></noscript></span></span>, since <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-16.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-18.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-19.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-20.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-21.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-22.gif"/></noscript></span></span> and contains the genotype×trait interaction term since <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-23.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-24.gif"/></noscript></span></span> is the effect of genotype×trait interaction term for <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-25.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-26.gif"/></noscript></span></span>.<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-27.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-28.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-29.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-30.gif"/></noscript></span></span> and contains the genotype×environment×trait interaction, since <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-31.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-32.gif"/></noscript></span></span> is the effect of genotype×environment×trait interaction for <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-33.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-34.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-35.gif"/></noscript></span></span>. Vector <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-36.gif"/></noscript></span></span>is distributed under a matrix-variate normal distribution as <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-37.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-38.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-39.gif"/></noscript></span></span> and represents the Genomic Relationship Matrix (GRM) and is calculated using the <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">VanRaden (2008)</a> method as <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-40.gif"/></noscript></span></span>, where p denotes the number of markers and W the matrix of markers of order <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-41.gif"/></noscript></span></span>; and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-42.gif"/></noscript></span></span> is the unstructured genetic (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-43.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-44.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-45.gif"/></noscript></span></span>is an unstructured (co)variance matrix of order <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-46.gif"/></noscript></span></span>and E is the matrix of residuals of order <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-47.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-48.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-49.gif"/></noscript></span></span> is the unstructured residual (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-50.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-51.gif"/></noscript></span></span> is the genomic relationship matrix described above (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em>, 2018</a>).</p><p id="p-13">The BMTME model resulting from <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a> was implemented by <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a>. Next, we use the modified version of the Gibbs sampler of the original BMTME model proposed by <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a> that was implemented in <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>. It is important to point out that model (1) takes into account the genotype×environment terms in the (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-52.gif"/></noscript></span></span> term and for comparison purposes, we also ran the model in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a> without the (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-53.gif"/></noscript></span></span> term to study the effect on prediction performance with and without the genotype×environment term.</p><p id="p-14">Outlined below is the Gibbs sampler implemented by <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a> for estimating the parameters of interest in the BMTME model. While the order is somewhat arbitrary, we suggest the following:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-15">Step 1. Simulate β according to the normal distribution given in Appendix A (A.1) of <a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-2"><p id="p-16">Step 2. Simulate <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-54.gif"/></noscript></span></span>according to the normal distribution given in Appendix A (A.2) of <a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-3"><p id="p-17">Step 3. Simulate <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-55.gif"/></noscript></span></span>according to the normal distribution given in Appendix A (A.3) of <a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-4"><p id="p-18">Step 4. Simulate <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-56.gif"/></noscript></span></span>according to the inverse Wishart (IW) distribution given in Appendix A (A.4) of <a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-5"><p id="p-19">Step 5. Simulate <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-57.gif"/></noscript></span></span>according to the IW distribution given in Appendix A (A.5) of <a id="xref-ref-22-11" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-6"><p id="p-20">Step 6. Simulate <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-58.gif"/></noscript></span></span>according to the IW distribution given in Appendix A (A.6) of <a id="xref-ref-22-12" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>.</p></li><li id="list-item-7"><p id="p-21">Step 7. Return to step 1 or terminate when chain length is adequate to meet convergence diagnostics.</p></li></ul><p id="p-22">The main differences between this Gibbs sampler and that given by <a id="xref-ref-21-9" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a> are: (i) the modified Gibbs sampler was built using the matrix-variate normal distribution instead of a multivariate normal distribution; (ii) this modified Gibbs sampler assumes a general or unstructured variance-covariance matrix for environments, that needs <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-59.gif"/></noscript></span></span> parameters because every term is different, while the original BMTME model assumes a diagonal variance-covariance matrix for environments that only needs L parameters since all off diagonal elements are zero; and (iii) the original BMTME model used non-informative priors based on the Half-t distribution of each standard deviation term and uniform priors on each correlation of the covariance matrices of traits (genetic and residual). The priors implemented for the Gibbs sampler described above are given in Appendix A of this article.</p></div><div id="sec-4" class="subsection"><h4>Multi-trait deep learning (MTDL) model:</h4><p id="p-23">Popular neural network architectures are: (a) densely connected networks, (b) convolutional networks, and (c) recurrent networks. Details of each type of network, its assumptions and input characteristics can be found in <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gulli and Sujit (2017)</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Angermueller <em>et al.</em> (2017)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chollet and Allaire, (2017)</a>. In this study we implemented a type (a) network, which does not assume a specific structure in the input features. In general, the basic structure of a densely connected network consists of an input layer, L output layers (for multi-trait modeling) and multiple hidden layers between the input and output layers. This type of neural network is also known as a feedforward neural network. The implementation of this neural network is challenging because it requires the following hyperparameters: number of units (U), number of layers, number of epochs (E), type of regularization method and type of activation function. Based on the literature review, we decided to use the rectified linear activation unit (ReLU) as activation function and the dropout type of regularization method for training the models (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gulli and Sujit 2017</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Angermueller <em>et al.</em>, 2017</a>; and <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chollet and Allaire 2017</a>). The range of the remaining hyperparameters was determined by a few initial studies of a single fold (80% for training and 20% for testing), randomly selected from each data set; with these initial values, we implemented a full factorial design with 6 levels for the number of units and epochs and 3 levels for the number of layers. For more details on model selection in DL models, we suggest the companion paper of <a id="xref-ref-22-13" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018)</a>, where the authors evaluate the prediction performance of univariate DL models for multi-environment data. In Appendix B we provide the R code for implementing MTDL models, while the R package BMTME that is still under development is available at the following link: <a href="https://github.com/frahik/BMTME">https://github.com/frahik/BMTME</a>.</p></div></div><div id="sec-5" class="subsection"><h3>Experimental data sets</h3><p id="p-24">Three real data sets were analyzed, one data set comprising maize lines and two data sets comprising elite wheat lines. The three data sets included several environments.</p><div id="sec-6" class="subsection"><h4>Maize data set 1:</h4><p id="p-25">This data set was used by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2013)</a> and <a id="xref-ref-21-10" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2016)</a> and is made up of a total of 309 maize lines. Three traits were evaluated: grain yield (GY), anthesis-silking interval (ASI), and plant height (PH); each of these traits was measured in three environments (Env1, Env2, and Env3) on the same 309 lines. Phenotypes of each trait were pre-analyzed and adjusted for the experimental field design. The genotyping was done with Genotype by Sequencing (GBS) technology with a total of 681,257 single nucleotide polymorphisms (SNPs); after filtering for missing values and minor allele frequency, we used 158,281 SNPs. Markers that had 80% of the maize lines with missing values were removed, and markers with a minor allele frequency lower than or equal to 0.05 were deleted.</p></div><div id="sec-7" class="subsection"><h4>Wheat data set 2:</h4><p id="p-26">This wheat data set is composed of 250 wheat lines that were extracted from a large set of 39 yield trials grown during the 2013-2014 crop season in Ciudad Obregon, Sonora, Mexico (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rutkoski <em>et al.</em>, 2016</a>). The measured traits were plant height (PH) recorded in centimeters and days to heading (DTH) recorded as the number of days from germination until 50% of spikes had emerged in each plot, in the first replicate of each trial. Both traits were measured in three environments and on the same 250 lines. Phenotypes were also adjusted by experimental design. Genomic information was obtained by genotype by sequencing (GBS) and we used a total of 12,083 markers that remained after quality control. Single nucleotide polymorphism calls were extracted and markers were filtered so that percent missing data did not exceed 80%. Individuals with 80% missing marker data were removed, and markers were recorded as -1, 0, and 1, indicating homozygous for the minor allele, heterozygous, and homozygous for the major allele, respectively. Next, markers with 0.01 minor allele frequency were removed, and missing data were imputed with the marker mean.</p></div><div id="sec-8" class="subsection"><h4>Wheat Iranian data set 3:</h4><p id="p-27">This data set consists of 2374 wheat lines evaluated in a drought environment (D) and a heat environment (H) at the CIMMYT experiment station near Ciudad Obregon, Sonora, Mexico (27 ° 20 ′N, 109 ° 54 ′W, 38 meters above sea level) during the 2010-2011 cycle and was used in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2016)</a>. Two traits were measured: days to maturity (DTM) and days to heading (DTH). Both traits were measured in the two environments and on the same 2374 lines. The number of markers used was 39,758 that remained after the quality control process from a total of 40,000 markers originally used.</p></div></div><div id="sec-9" class="subsection"><h3>Experiments evaluated</h3><p id="p-28">In this empirical evaluation, we compared the prediction accuracy of the two proposed models: the MTDL method and the BMTME model. Both methods were implemented in the R statistical software (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">R Core Team 2018</a>). The MTDL model was fitted with the Keras package (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gulli and Sujit 2017</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chollet and Allaire 2017</a>) with a densely connected network architecture. In both MTDL and BMTME, we used two different sets of independent variables: the first set was composed of information on environments and genomes (that takes into account genomic information) ignoring genotype×environment interaction, while the second set included the genotype×environment term in addition to the main effects of environments and genomes. Under the MTDL model, we implemented a grid search for choosing the hyperparameters; this was done using a full factorial design with the following three factors: (a) number of units (U), (b) number of epochs (E), and (c) number of layers. These three factors are hyperparameters under the MTDL model. We studied the following values for the U and E hyperparameters: 50, 60, 70, 80, 90 and 100, and for the number of layers we used 1, 2 and 3. These values were chosen after the previous experiments were conducted. Thus 6×6×3 = 108 experiments were run for each data set with densely connected MTDL models.</p><p id="p-29">It is important to point out that the 108 MTDL experiments used dropout regularization, which is one of the most effective and commonly used regularization techniques in neural networks; it was developed by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Srivastava <em>et al.</em> (2014)</a> at the University of Toronto. Dropout regularization is applied to a layer and consists of randomly dropping out (setting to zero) a number of the hidden layer’s during training. In our case, the dropout rate was 0.3 (30%); this meant that the percentage of features that were set to zero was 30% in each hidden layer (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gulli and Sujit 2017</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chollet and Allaire 2017</a>).</p><div id="sec-10" class="subsection"><h4>Evaluation of prediction performance With cross-validation:</h4><p id="p-30">The prediction accuracy of both MTDL and BMTME models was evaluated with 10 random cross-validations (CV): the whole data set was divided into a training (TRN) and a testing (TST) set; 80% of the whole data set was assigned to TRN, and the remaining 20% was assigned to TST set. In our random CV, one observation can appear in more than one partition because we used sampling with replacement. However, the same observation is never included simultaneously in the training and testing sets of a random partition. In the design, some lines can be evaluated in some, but not all, target environments, which mimics a prediction problem faced by breeders in incomplete field trials. For this reason, our cross-validation strategy is exactly the same as that denoted by the CV2 proposed and implemented by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2017)</a>, where a certain portion of tested lines (genotypes) in a certain portion of tested environments are predicted since some tested lines that were evaluated in some tested environments are assumed missed in others. Since <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-60.gif"/></noscript></span></span> denotes the total number of records per each available trait, then to select lines in the TST data set, we fixed the percentage of data to be used for TST (PTesting = 20%). Then 0.20<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-61.gif"/></noscript></span></span> (lines) were chosen at random, and subsequently one environment per line was randomly picked from I environments. The cells not selected through this algorithm were allocated to the TRN data set, while the selected cells (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-62.gif"/></noscript></span></span> were assigned to the TST data set. Lines were sampled with replacement if <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-63.gif"/></noscript></span></span>, and without replacement otherwise (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">López-Cruz <em>et al.</em>, 2015</a>). The metric used to measure the prediction accuracy of both models was the Pearson’s correlation and it was calculated from each trait-environment combination for each testing set of each random partition; thus the average of all partitions was reported as a measure of prediction performance. This explained cross validation method is called the outer CV and was applied for both models. However, in the DL model we also applied an inner CV strategy for tunning the hyperparameters using the grid of hyperparameter values defined above (108 experiments). The inner CV strategy consists in spliting each training set of the outer CV, here 20% of data were assigned for testing-inner and 80% for training-inner.The training-inner data set was used to train the DL model using the grid of hyperparameters values. This inner CV strategy was facilitated by using the internal capabilities of Keras by means of the validation_split argument on the fit() function. The predictve power is assessed in the second part of the data set (testing-inner). With this, a set of best-fitting hyperparameters (best combination of units, epoch and layers) from the inner CV loop is obtained. Finally, these set of hyperparameters were used to predict the performance in the independent testing data set (testing-outer). Since different traits in different environments have different heritabilities, we report the accuracy in terms of Pearson’s correlation divided by the square root of the heritability corresponding to each trait-environment combination.</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-31">The phenotypic and genotypic data used in this study can be downloaded from the following link: <a href="http://hdl.handle.net/11529/10548134">http://hdl.handle.net/11529/10548134.</a></p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-32">Our results are given in four sections. In the three first sections we report the performance of each real data set analyzed, and in the last section we compare the prediction performance of both models for all data sets. In each of the first three sections, first we present a figure with prediction performance in terms of Pearson’s correlation under both models BMTME and MTDL with (I) and without (WI) the genotype×environment interaction term.</p><div id="sec-13" class="subsection"><h3>Maize data set 1</h3><p id="p-33">Next we describe the prediction performance of the BMTME model without genotype<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-64.gif"/></noscript></span></span>environment interaction (WI). The prediction accuracy of this model ranged from 0.222 to 0.451. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> gives the predictions under the MTDL model without the interaction term. Here Pearson’s correlation ranged from 0.305 to 0.540 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Next we present the prediction performance of the models including genotype<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-65.gif"/></noscript></span></span>environment interaction (I). First, under the BMTME model the predictions ranged from 0.321 to 0.619 in terms of Pearson’s correlation. The prediction performance of MTDL is shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>; with the interaction term ranged from 0.266 to 0.501 in terms of Pearson’s correlation. The standard errors (SE) of the average Pearson correlation of this data set and of the following data sets are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table C1</a> of Appendix C.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maize data. Mean Pearson&#x2019;s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotypeenvironment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-14699600" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Maize data. Mean Pearson&#x2019;s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3829/F1.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3829/F1.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491528" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">Maize data. Mean Pearson’s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype<span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/F1/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/F1/embed/mml-math-66.gif"/></noscript></span></span>environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype×environment interaction (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Wheat data set 2</h3><p id="p-35">Next we describe the prediction performance of the BMTME model without genotype×environment interaction (WI). In terms of Pearson’s correlation, the range of predictions under this model was from -0.140 to 1.00 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Under the MTDL model without the interaction term, the predictions ranged from 0.613 to 1.00 and are given in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. When genotype×environment interaction (I) was considered under the BMTME model, the range of Pearson’s correlation was from 0.202 to 1.00 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table C1</a>). With the MTDL model the predictions ranged from 0.601 to 1.00 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wheat data. Means Pearson&#x2019;s correlation for each environment-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype&#xD7;environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-14699600" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Wheat data. Means Pearson&#x2019;s correlation for each environment-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype&#xD7;environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3829/F2.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3829/F2.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Wheat data. Means Pearson’s correlation for each environment-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype×environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype×environment interaction (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Wheat Iranian data set 3</h3><p id="p-37">The prediction performance of the BMTME model without genotype×environment interaction (WI) is presented first. The predictions of this model in terms of Pearson’s correlation ranged from 0.331 to 0.763 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> gives also the predictions under the MTDL model without the interaction term for each environment-trait combination; the range of predictions with Pearson’s correlation was from 0.574 to 0.784. Next the predictions with genotype×environment interaction (I) are provided. First, under the BMTME model, the range of predictions in terms of Pearson’s correlation was from 0.998 to 1.00. The prediction performance of MTDL is shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, with the interaction term they ranged from 0.738 to 1.00.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Iranian data. Mean Pearson&#x2019;s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype&#xD7;environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-14699600" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Iranian data. Mean Pearson&#x2019;s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype&#xD7;environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype&#xD7;environment interaction (WI).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3829/F3.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3829/F3.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Iranian data. Mean Pearson’s correlation for each environments-trait combination for the MTDL and GBLUP models. The top horizontal sub-panel corresponds to the model with genotype×environment interaction (I), and the bottom horizontal sub-panel corresponds to the same model without genotype×environment interaction (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Comparing the BMTME model to the MTDL model</h3><p id="p-39">To obtain a meta-picture of the prediction performance of the MTDL model against the BMTME model, we compared the average predictions across environments-traits of the MTDL model <em>vs.</em> those of the BMTME model for each data set. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows that when the genotype×environment interaction term was ignored in the three data sets under study, the best predictions were observed under the MTDL model, but when the genotype×environment interaction term was taken into account, the best predictions were obtained with the BMTME model. Finally, upon comparing the predictions for each data set with the four models resulting from the two models (BMTME and MTDL) and the two types of covariates used in the predictor (with and without the interaction term), we found that in maize data set 1, the best prediction of these four models corresponded to the BMTME model with the interaction term; this model was 4.61% better than the second best model, <em>i.e.</em>, MTDL without the interaction term. For wheat data set 2, the best prediction corresponded to the MTDL model without the genotype×environment interaction term; this model was 7.31% better than the second best model (BMTME with the interaction term). For the third data set, wheat Iranian data set 3, the best prediction corresponded to the BMTME model with the genotype×environment interaction term; this model was 16.32% better in terms of prediction accuracy than the second best model, MTDL with the interaction term.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean Pearson&#x2019;s correlation across environments-traits for the GBLUP and MTDL model conducted with (I) and without (WI) genotype&#xD7;environment interaction for each data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-14699600" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean Pearson&#x2019;s correlation across environments-traits for the GBLUP and MTDL model conducted with (I) and without (WI) genotype&#xD7;environment interaction for each data set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3829/F4.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3829/F4.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3829/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491566" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Mean Pearson’s correlation across environments-traits for the GBLUP and MTDL model conducted with (I) and without (WI) genotype×environment interaction for each data set.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-41">The key objective of GS is to build an accurate prediction model based on training populations consisting of individuals with both genotypic and phenotypic data. For this reason, many univariate prediction models have been built and evaluated in the GS context. However, breeders usually select, at the same time, multiple traits that are often genetically correlated, with correlations that range from weak to strong. For this reason, in this paper we explored MTDL models for selecting candidate genotypes in GS that allow the simultaneous prediction of many traits measured in each experimental unit. We found evidence that MTDL models are very competitive in terms of prediction accuracy compared with BMTME models, since the prediction performance of MTDL models was competitive to that of the BMTME models. In general terms, the best predictions were produced by the BMTME model with the genotype×environment interaction term.</p><div id="sec-18" class="subsection"><h3>Computation efficiency of MTDL vs. BMTME</h3><p id="p-42">Additionally, we found that the MTDL model is more computational efficiency than the BMTME model since for the maize and wheat data set, on average the MTDL was 11.71 and 3.39 times faster, respectively, than the BMTME model. For the maize data set the BMTME model required 42.222 hr for its implementation, whereas the MTDL only required 3.604 hr. For the wheat set BMTME required around 13.86 hr, while the MTME model only need 4.079 hr. However, it is important to point out that this difference can be due to the fact that the MTDL model automatically parallelized the jobs and its efficiency depends on the number of available cores. Furthermore, when comparing the BMTME model with the univariate Bayesian GBLUP model the BMTME model is considerable more demanding in terms of implementation time; however, we have not performed a formal comparison in this respect.</p></div><div id="sec-19" class="subsection"><h3>Models Without genotype×environment interaction</h3><p id="p-43">For the three data sets, when genotype×environment interaction was ignored in both models, the best predictions were observed with the MTDL models compared to the BMTME model. This can be explained in part by the fact that DL models use hidden layers that automatically capture complex interaction terms without the need to pre-specify <em>a priori</em> covariates that include the interaction terms. This means that, unlike the BMTME model (and most statistical mixed models), DL models can capture not only two- or three-way interaction terms, but also interactions of larger order without the need to specify in the predictor the covariates corresponding to these interactions. However, the interpretability of DL models is not as transparent nor as easy as the interpretability of some genomic models such as the GBLUP and BMTME.</p></div><div id="sec-20" class="subsection"><h3>Models With genotype×environment interaction</h3><p id="p-44">We found that in the three data sets when taking into account genotype×environment interaction in both models, the BMTME model was better than the MTDL model in terms of prediction accuracy. This means that although we found evidence that the MTDL model was very competitive in terms of prediction accuracy, the BMTME model was superior. Our results indicate that conventional prediction models based on mixed models (the BMTME model, in our case) are very powerful tools, with the advantage that they offer a transparent building process and interpretability, and avoid the time consuming and tedious task of tuning the hyperparameters that DL models need. However, we need to point out that our results are valid only for the hyperparameters used in this research.</p></div><div id="sec-21" class="subsection"><h3>General comments</h3><p id="p-45">The performance of the MTDL model depends on the set of hyperparameters used; these hyperparameters are data dependent, which was corroborated in the three data sets used in this research. These different combinations of hyperparameters for each data set corroborate the difficulties found when choosing an appropriate set of hyperparameters in the implementation of MTDL models and of course we are aware that exploring other sets of hyperparameters the prediction of the MTDL model can be improved.</p><p id="p-46">It is important to point out that the models (MTDL and BMTME) used in this research are only appropriate for multiple continuous traits; of course, each trait can be on a different scale. However, to successfully implement both models, we recommend rescaling the response variable for each training data set and, after getting the corresponding predictions for each testing set, transforming each variable back to its original scale. There are many ways to rescale the variables, but we used the standardization method (subtracting the mean and dividing by the standard deviation). However, other types of rescaling methods can be implemented and many times produce similar results in terms of prediction accuracy. The rescaling process is more important in DL models than in the BMTME model, since the BMTME works well even when the traits are in different scales.</p><p id="p-47">One possible explanation for why multi-trait models often perform better than univariate models is that they effectively increase the sample size that is used to train the models. As different traits have different noise patterns, a model that learns two or more traits simultaneously is able to learn a more general representation. Learning just trait A bears the risk of overfitting to trait A, while learning A and B jointly enables the model to obtain a better representation by averaging the noise patterns.</p><p id="p-48">Some explanations of why MTDL models are less efficient in terms of prediction accuracy than BMTME models are: (a) it is documented that DL in general requires a large amount of data for training to be successful, (b) MTDL models do not have a strong known theoretical foundation, (c) determining the architecture and the hyperparameters for DL is a challenging process and the way to proceed so far is just by trial and error, with no theory to guide the researcher, (d) lack of interpretation of results is indeed a problem, and (e) the system is prone to overfitting the training data. However, DL models also have some advantages: (a) they are easy to adapt to classification or numeric prediction problems, (b) they are capable of modeling more complex patterns than nearly any algorithm with less intense computer requirements, (c) they make few assumptions about the data’s underlying relationships, and (d) they are easy to adapt from univariate trait analysis to multi-trait analysis even with mixed phenotypes (binary, ordinal and continuous).</p><p id="p-49">Finally, we would like to point out that our results are valid under a densely connected network architecture with the grid of hyperparameters used in the 108 experiments conducted. For this reason, we believe that with other types of network architecture and hyperparameters, it would be possible to achieve a better performance in terms of prediction accuracy of the MTDL models. For these reasons, we encourage conducting other studies to evaluate MTDL models in the context of genomic selection while exploring other network architectures and sets of hyperparameters.</p></div></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusions</h2><p id="p-50">In this paper we propose using MTDL models for prediction in genomic selection. We found that the best predictions correspond to the BMTME model, but the predictions of the MTDL model were very competitive with those of the BMTME model, since in one of the three data sets, the prediction accuracy of the MTDL model was better than that of the BMTME model. We also found that the MTDL model performed better when the covariates corresponding to genotype×environment interaction were ignored. The implementation of the MTDL model also required less computational resources than the BMTME model. For these reasons, we have empirical evidence in favor of the MTDL model and suggest including these models in the toolkit of breeding scientists for predicting multiple traits simultaneously in GS. Although training the MTDL models is challenging due to the number of hyperparameters involved, we found that implementing MTDL models is feasible and practical in the GS context. However, more research is needed to increase the reliability of our findings.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-151">We thank all scientists, field workers, and lab assistances from National Programs and CIMMYT who collected the data used in this study. We thank financial support from CIMMYT CRP (maize and wheat), Bill and Melinda Gates Foundation as well the USAID projects (Cornell University and Kansas State University) that generated the CIMMYT maize and wheat data analyzed in this study. Funding for this work was received from the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. The authors highly appreciate Dr. Emma Huang and anonymous reviewers for their time reading, correcting and suggesting changes that substantially improved the quality and readability of the article.</p><p id="p-152"><em>Note added in proof:</em> See Montesinos-López et al. 2018 (pp. 3813–3828) in this issue, for a related work.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A</h2><div class="section" id="sec-23"><h3>Setting the hyperparameters for the prior distributions of the BMTME model</h3><p id="p-51">The hyperparameters for the BMTME model were set similar to those used in the BGLR software (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pérez and de los Campos, 2014</a>). These rules provide proper but weakly informative prior distributions so that we partitioned the total variance-covariance of the phenotypes into two components: (1) the error and (2) the linear predictor. First we provide the variance-covariance of the phenotypes<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-67.gif"/></noscript></span><span class="disp-formula-label">(B1)</span></span>Therefore, the variance-covariance of row i for <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-68.gif"/></noscript></span></span> of Y is equal to<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(B2)</span></span>Therefore, the average of the n rows of equation (B2), called total variance, is equal to<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-70.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-71.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(B3)</span></span>where: <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-73.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-74.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-75.gif"/></noscript></span></span>, and</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-76.gif"/></noscript></span><span class="disp-formula-label">(B4)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(B5)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(B6)</span></span><div id="sec-24" class="subsection"><h4>Setting the hyperparameters for <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-79.gif"/></noscript></span></span>:</h4><p id="p-52">Since <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-80.gif"/></noscript></span></span>)=<span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-81.gif"/></noscript></span></span> and mode<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-82.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-83.gif"/></noscript></span></span>. Therefore, from equation (B4),<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(B7)</span></span>Thus, if we replace the left-hand side of equation (B7) with the mode of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-85.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(B8)</span></span>From (B8) and solving we get <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-87.gif"/></noscript></span></span>. Then by setting <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-88.gif"/></noscript></span></span> as the proportion of the total variance-covariance (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-89.gif"/></noscript></span></span>) that is explained <em>a priori</em> by the traits, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-90.gif"/></noscript></span></span>, we have that<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-91.gif"/></noscript></span><span class="disp-formula-label">(B9)</span></span>Once we set <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-92.gif"/></noscript></span></span>, we can set <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-93.gif"/></noscript></span></span> as in (B9) and we only need to compute the phenotypic variance-covariance matrix (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-94.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-95.gif"/></noscript></span></span> and set <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-96.gif"/></noscript></span></span> as the proportion of variance-covariance that is explained <em>a priori</em> by the traits. We set as default <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-97.gif"/></noscript></span></span>.</p></div><div id="sec-25" class="subsection"><h4>Setting the hyperparameters for <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-98.gif"/></noscript></span></span>:</h4><p id="p-53">Also, since <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-99.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-100.gif"/></noscript></span></span> and mode<span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-101.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-102.gif"/></noscript></span></span>, and using equation (B5) and (B6), in a similar way as before<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-103.gif"/></noscript></span><span class="disp-formula-label">(B10)</span></span>Thus, if we replace the left-hand side of equation (B10) with the mode of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-104.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-105.gif"/></noscript></span><span class="disp-formula-label">(B11)</span></span>From (B11) and solving for <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-106.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-107.gif"/></noscript></span><span class="disp-formula-label">(B12)</span></span>for which we only need to compute the phenotypic variance-covariance matrix (<span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-108.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-109.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-110.gif"/></noscript></span></span> and set <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-112.gif"/></noscript></span></span> as the proportion of variance-covariance that <em>a priori</em> is explained by the traits in the genotype×trait and genotype×environment×trait interaction terms. We set as default <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-113.gif"/></noscript></span></span>. In the balanced case, <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-114.gif"/></noscript></span></span>, so to complete the setting value of <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-115.gif"/></noscript></span></span>, we take <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-116.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-117.gif"/></noscript></span></span> is the phenotypic variance of trait l.</p></div><div id="sec-26" class="subsection"><h4>Setting the hyperparameters for <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-118.gif"/></noscript></span></span>:</h4><p id="p-54">Also, since <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-119.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-120.gif"/></noscript></span></span> and mode<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-121.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-122.gif"/></noscript></span></span>. Let<span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-123.gif"/></noscript></span></span> be the variance-covariance matrix of the matrix of phenotypic responses, <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-124.gif"/></noscript></span></span>, that resulted from accommodating the information of the matrix of phenotypic responses (Y) of order <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-125.gif"/></noscript></span></span>, with n = IJ, as a matrix of order <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-126.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-127.gif"/></noscript></span></span>, that is, the columns of <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-128.gif"/></noscript></span></span> correspond to environments instead of traits as in Y. Then, in similar fashion, we can define<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-129.gif"/></noscript></span><span class="disp-formula-label">(B13)</span></span>Thus, if we replace the left-hand side of equation (B13) with the mode of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-130.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-131.gif"/></noscript></span><span class="disp-formula-label">(B14)</span></span>from (B14) and solving for <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-132.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-133.gif"/></noscript></span><span class="disp-formula-label">(B15)</span></span>with <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-134.gif"/></noscript></span></span></p></div><div id="sec-27" class="subsection"><h4>Setting the hyperparameters for <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-135.gif"/></noscript></span></span>:</h4><p id="p-55">Also, since <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-136.gif"/></noscript></span></span>)=<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-137.gif"/></noscript></span></span> and mode<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-138.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-139.gif"/></noscript></span></span>. Therefore, in similar fashion to the above hyperparameters, we set</p><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3829/embed/mml-math-140.gif"/></noscript></span><span class="disp-formula-label">(B16)</span></span></div></div></div><div class="section app" id="app-2"><h2 class="">APPENDIX B</h2><div class="section" id="sec-28"><h3>R code for implementing MTDL models</h3><p id="p-56">setwd(“C:/TELEMATICA 2017/Deep Learning Multi-trait”)</p><p id="p-57">rm(list = ls())</p><p id="p-58">######Libraries required#################################</p><p id="p-59">library(tensorflow)</p><p id="p-60">library(keras)</p><p id="p-61">#############Loading data################################</p><p id="p-62">load(“Data_Maize_set_1.RData”)</p><p id="p-63">ls()</p><p id="p-64">####Genomic relationship matrix (GRM)) and phenotipic data######</p><p id="p-65">G=G_maize_1to3</p><p id="p-66">Pheno=Pheno_maize_1to3</p><p id="p-67">head(Pheno)</p><p id="p-68">###########Cholesky decomposition of the GRM######</p><p id="p-69">LG=t(chol(G))</p><p id="p-70">########Creating the desing matrices #################</p><p id="p-71">Z1=model.matrix(∼0+as.factor(Pheno$Line))</p><p id="p-72">ZE=model.matrix(∼0+as.factor(Pheno$Env))</p><p id="p-73">Z1G=Z1%*%LG</p><p id="p-74">Z2GE=model.matrix(∼0+as.factor(Pheno$Line):as.factor(Pheno$Env))</p><p id="p-75">G=data.matrix(G)</p><p id="p-76">G2=kronecker(diag(3),G)</p><p id="p-77">LG2=t(chol(G2))</p><p id="p-78">Z2GE=Z2GE%*%LG2</p><p id="p-79">###Defining the number of epoch and units#####################</p><p id="p-80">units_M=50</p><p id="p-81">epochs_M=50</p><p id="p-82">##########Data for trait GY#################################</p><p id="p-83">y = Pheno[,3:5]</p><p id="p-84">X = cbind(ZE,Z1G,Z2GE)</p><p id="p-85">head(y)</p><p id="p-86">#############Training and testing sets###################</p><p id="p-87">n=dim(X)[1]</p><p id="p-88">nt=dim(y)[2]</p><p id="p-89">Post_trn=sample(1:n,round(n*0.8))</p><p id="p-90">X_tr = X[Post_trn,]</p><p id="p-91">X_ts = X[-Post_trn,]</p><p id="p-92">y_tr = scale(y[Post_trn,])</p><p id="p-93">Mean_trn=apply(y[Post_trn,],2,mean)</p><p id="p-94">SD_trn=apply(y[Post_trn,],2,sd)</p><p id="p-95">y_ts=matrix(NA,ncol=nt,nrow=dim(X_ts)[1])</p><p id="p-96">for (t in 1:nt){</p><p id="p-97">y_ts[,t] =(y[-Post_trn,t]- Mean_trn[t])/SD_trn[t] }</p><p id="p-98"># add covariates (independent variables)</p><p id="p-99">input &lt;- layer_input(shape=dim(X_tr)[2],name=“covars”)</p><p id="p-100"># add hidden layers</p><p id="p-101">base_model &lt;- input %&gt;%</p><p id="p-102">layer_dense(units =units_M, activation=’relu’) %&gt;%</p><p id="p-103">layer_dropout(rate = 0.3) %&gt;%</p><p id="p-104">layer_dense(units = units_M, activation = “relu”) %&gt;%</p><p id="p-105">layer_dropout(rate = 0.3) %&gt;%</p><p id="p-106">layer_dense(units = units_M, activation = “relu”) %&gt;%</p><p id="p-107">layer_dropout(rate = 0.3)</p><p id="p-108"># add output 1</p><p id="p-109">yhat1 &lt;- base_model %&gt;%</p><p id="p-110">layer_dense(units = 1, name=“yhat1”)</p><p id="p-111"># add output 2</p><p id="p-112">yhat2 &lt;- base_model %&gt;%</p><p id="p-113">layer_dense(units = 1, name=“yhat2”)</p><p id="p-114"># add output 3</p><p id="p-115">yhat3 &lt;- base_model %&gt;%</p><p id="p-116">layer_dense(units = 1, name=“yhat3”)</p><p id="p-117"># build multi-output model</p><p id="p-118">model &lt;- keras_model(input,list(yhat1,yhat2,yhat3)) %&gt;%</p><p id="p-119">compile(optimizer = “rmsprop”,</p><p id="p-120">loss=”mse”,</p><p id="p-121">metrics=”mae”,</p><p id="p-122">loss_weights=c(0.3333,0.3333,0.3333))</p><p id="p-123"># fit model</p><p id="p-124">model_fit &lt;- model %&gt;%</p><p id="p-125">fit(x=X_tr,</p><p id="p-126">y=list(y_tr[,1],y_tr[,2],y_tr[,3]),</p><p id="p-127">epochs=epochs_M,</p><p id="p-128">batch_size = 50,</p><p id="p-129">verbose=0)</p><p id="p-130"># predict values for test set</p><p id="p-131">Yhat &lt;- predict(model, X_ts) %&gt;%</p><p id="p-132">data.frame() %&gt;%</p><p id="p-133">setNames(colnames(y_tr))</p><p id="p-134">predB=Yhat</p><p id="p-135">y_p=predB</p><p id="p-136">for (s in 1:nt){</p><p id="p-137">y_p[,s]=y_p[,s]*SD_trn[s]+ Mean_trn[s]</p><p id="p-138">y_ts[,s]=y_ts[,s]*SD_trn[s]+ Mean_trn[s]</p><p id="p-139">}</p><p id="p-140">#################Observed and predicted values############</p><p id="p-141">Y_all_tst = data.frame(cbind(y_ts, y_p))</p><p id="p-142">Y_all_tst</p><p id="p-143">########Prediciton accuracy with Pearson Correlation######</p><p id="p-144">Cor_Mat=cor(Y_all_tst)</p><p id="p-145">Cor_Traits=diag(Cor_Mat[(nt+1):(2*nt),1:nt])</p><p id="p-146">Cor_Traits</p><p id="p-147">########Plots of observed and predicted values############</p><p id="p-148">plot(Y_all_tst[,1],Y_all_tst[,4])</p><p id="p-149">plot(Y_all_tst[,2],Y_all_tst[,5])</p><p id="p-150">plot(Y_all_tst[,3],Y_all_tst[,6])</p></div></div><div class="section app" id="app-3"><h2 class="">APPENDIX C</h2><div class="section" id="sec-29"><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491673/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491673/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table C1.</span> <span class="caption-title">Standard errors (SE) of both models, BMTME and MTDL for each data set. WI denotes the scenarios that ignore the genotype×environment interaction term, while I denotes the scenarios with the genotype×environment interaction term. Env_Trait denotes the environment-trait combination</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-153"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-07-30"><span class="received-label">Received </span>July 30, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-03"><span class="accepted-label">Accepted </span>October 3, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.1" data-doi="10.1038/nbt.3300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alipanahi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3300%26rft_id%253Dinfo%253Apmid%252F26213851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26213851&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angermueller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reik</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite> <span class="cit-pub-date">2017</span>  DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning. Genome Biology, 18:1:13.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">F. I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J. A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Efeitos da heterose e endogamia em caracteres de importância agronômica em trigo.</span> <abbr class="cit-jnl-abbrev">Rev. Ceres</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BCeres%26rft.volume%253D56%26rft.spage%253D753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.4" data-doi="10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution : GSE</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297–9686–43–26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%253F%253A%2BGSE%26rft.volume%253D43%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297%2526ndash%253B9686%2526ndash%253B43%2526ndash%253B26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.5" data-doi="10.1590/S0100-204X2013000600008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">A. F. N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">R. V. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">A. C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multivariate analysis for the selection of eucalyptus clones destined for charcoal production.</span> <abbr class="cit-jnl-abbrev">Pesqui. Agropecu. Bras.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S0100-204X2013000600008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPesqui.%2BAgropecu.%2BBras.%26rft.volume%253D48%26rft.spage%253D627%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS0100-204X2013000600008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S0100-204X2013000600008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3829.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chollet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source"><em>Deep Learning with R. Manning Publications</em>, <em>Manning Early Access Program</em></span>, <span class="cit-edition">Ed. 1st</span>. <span class="cit-publ-name">MEA</span>, <span class="cit-publ-loc">New Delhi, India</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colleau</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupassieux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Simulating the potential of genomic selection in dairy cattle breeding. Rencontres, Recherches</span>. <abbr class="cit-jnl-abbrev">Ruminants</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRuminants%26rft.volume%253D16%26rft.spage%253D419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.8" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <em>et al</em>. <span class="cit-pub-date">2013</span>  Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3: Genes|Genomes|Genetics (Bethesda) 3, <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft_id%253Dinfo%253Apmid%252F24022750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTkwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzgyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.9" data-doi="10.1534/g3.116.029637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2016</span>.  Genomic Prediction of Gene Bank Wheat Landraces. G3: Genes|Genomes|Genetics (Bethesda) 6, <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029637%26rft_id%253Dinfo%253Apmid%252F27172218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xODE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zODI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3829.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everitt</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Applied Multivariate Data Analysis</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.12.3829.11"><div class="cit-metadata"><cite><span class="cit-comment">Fox, I., A. Lynn, J. Mamta, P.-B. Rodica, and W. Jenna, 2018  Deep Multi-Output Forecasting: Learning to Accurately Predict Blood Glucose Trajectories. In KDD ’18: The 24th ACM SIGKDD International Conference on Knowledge Discovery &amp; Data Mining, August 19–23, 2018, London, UK. ACM, New York, NY, USA</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/3219819</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.12" data-doi="10.2307/2533274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Average Information REML: An Efficient Algorithm for Variance Parameter Estimation in Linear Mixed Models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1440</span>–<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2533274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D51%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TQ15600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3829.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gulli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujit</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python</span>, <span class="cit-publ-name">Packt Publishing Ltd.</span>, <span class="cit-publ-loc">Birmingham, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.14" data-doi="10.1093/bioinformatics/btw249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">i37</span>–<span class="cit-lpage">i43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw249%26rft_id%253Dinfo%253Apmid%252F27307640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27307640&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite> <span class="cit-pub-date">2015</span>  Diallel analysis of combining ability and heterosis for yield and yield components in rice by using positive loci. Euphytica, 205: 3 7–50.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.16" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.17" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM4MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.18" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.19" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, Poland, <em>et al</em>. <span class="cit-pub-date">2015</span>  Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3: Genes|Genomes|Genetics (Bethesda), 5: <span class="cit-fpage">569</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM4MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.20" data-doi="10.1371/journal.pone.0061318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menden</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iorio</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garnett</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDermott</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e61318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0061318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De61318%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061318%26rft_id%253Dinfo%253Apmid%252F23646105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23646105&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.21" data-doi="10.1534/g3.116.032359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Hernández</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2016</span>  A Genomic Bayesian Multi-trait and Multi-environment Model. G3: Genes|Genomes|Genetics (Bethesda), 6: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032359%26rft_id%253Dinfo%253Apmid%252F27342738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yNzI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zODI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.22" data-doi="10.1534/g3.118.200740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C.M.</span></span></li></ol><cite> <span class="cit-pub-date">2018</span>  Multi-environment genomic prediction of plant traits using deep learners with a dense architecture. G3: Genes|Genomes|Genetics (Bethesda) 8:.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200740%26rft_id%253Dinfo%253Apmid%252F30291107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTIvMzgxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzgyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.23" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zODI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.24" data-doi="10.1371/journal.pone.0032235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oja</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Unified Multitask Architecture for Predicting Local Protein Properties.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DQi%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De32235%26rft.epage%253De32235%26rft.atitle%253DA%2Bunified%2Bmultitask%2Barchitecture%2Bfor%2Bpredicting%2Blocal%2Bprotein%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032235%26rft_id%253Dinfo%253Apmid%252F22461885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22461885&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-8.12.3829.25"><div class="cit-metadata unstructured">R Core Team, 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.26" data-doi="10.1534/g3.116.032888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2016</span>  Predictor traits from high-throughput phenotyping improve accuracy of pedigree and genomic selection for yield in wheat. G3: Genes|Genomes|Genetics (Bethesda), 6: <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032888</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032888%26rft_id%253Dinfo%253Apmid%252F27402362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yNzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zODI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.27" data-doi="10.1016/S0301-6226(00)00255-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multiple trait international bull comparisons.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(00)00255-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D69%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252800%252900255-4%26rft_id%253Dinfo%253Apmid%252F11337117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(00)00255-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11337117&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.28" data-doi="10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–017–3029–7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B017%2526ndash%253B3029%2526ndash%253B7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.29" data-doi="10.1109/TCBB.2014.2343960"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eickholt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Deep Learning Network Approach to ab initio Protein Secondary Structure Prediction.</span> <abbr class="cit-jnl-abbrev">Ieee Acm T Comput Bi</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TCBB.2014.2343960</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIeee%2BAcm%2BT%2BComput%2BBi%26rft.volume%253D12%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2014.2343960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2014.2343960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3829.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatri</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source">An Introduction to Multivariate Statistics.</span> <span class="cit-publ-name">Elsevier North Holland</span>, <span class="cit-publ-loc">New York, NY, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krizhevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutskever</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salakhutdinov</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dropout: A Simple Way to Prevent Neural Networks from Overfitting.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1929</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D15%26rft.spage%253D1929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.32" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multi-trait, random regression, or simply repeatability model in high-throughput phenotyping data improve genomic prediction for grain yield in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.33" data-doi="10.1101/177378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavanaei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anandanadarajah</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maida</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loganantharaj</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Deep Learning Model for Predicting Tumor Suppressor Genes and Oncogenes from PDB Structure.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/177378</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F177378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxNzczNzh2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzgyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.34" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3829.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3829.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wipf</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wassell</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>  <span class="cit-article-title">Multitask learning for subspace segmentation.</span> In <span class="cit-source">ICML</span>, 2015, pp. <span class="cit-fpage">1209</span>–<span class="cit-lpage">1217</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.36" data-doi="10.1007/s10681-017-1975-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casteel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Using unsupervised learning techniques to assess interactions among complex traits in soybeans.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-017-1975-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D213%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-017-1975-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-017-1975-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.37" data-doi="10.1016/j.knosys.2013.10.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple-output support vector regression with a firefly algorithm for interval-valued stock price index forecasting.</span> <abbr class="cit-jnl-abbrev">Knowl. Base. Syst.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.knosys.2013.10.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DKnowl.%2BBase.%2BSyst.%26rft.volume%253D55%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.knosys.2013.10.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.knosys.2013.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A multi-task learning framework for head pose estimation under target motion.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D38%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3829.39" data-doi="10.1016/j.media.2015.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhaduri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-scale deep networks and regression forests for direct biventricular volume estimation.</span> <abbr class="cit-jnl-abbrev">Med. Image Anal.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.media.2015.07.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed.%2BImage%2BAnal.%26rft.volume%253D30%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.media.2015.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.media.2015.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3829.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3813" title="Multi-environment Genomic Prediction of Plant Traits Using Deep Learners With Dense Architecture" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3841" title="Regulation of Root Angle and Gravitropism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3829" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0CcR1Z57MLWb_tZqdY67kgdODX_jRqjo90vpepW3w1o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3829" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3829" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3829" class="active" data-icon-position="" data-hide-link-title="0">Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491518" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kqwAVJ3ZwLKvrz1iJYVG3V5EC0dd_Gni1Nyg936mMqg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3829.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3829" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ykhJV29bSurFOIki6F5Giw4CTyAeUidErzxrflFo_WI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3829" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3829.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491518" id="node491518--2" data-pisa="ggg;8/12/3829" data-pisa-master="ggg;g3.118.200728" data-apath="/ggg/8/12/3829.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span> and <span class="highwire-citation-author" data-delta="5">Javier Martín-Vallejo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3829-3840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491518--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3829" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3829&amp;text=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3829&amp;t=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3829&amp;title=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491518/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491518" id="node491518--3" data-pisa="ggg;8/12/3829" data-pisa-master="ggg;g3.118.200728" data-apath="/ggg/8/12/3829.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait, Multi-environment Deep Learning Modeling for Genomic-Enabled Prediction of Plant Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span> and <span class="highwire-citation-author" data-delta="5">Javier Martín-Vallejo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3829-3840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491518--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Guadalajara, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3829&amp;rft.epage=3840&amp;rft.atitle=Multi-trait%2C%20Multi-environment%20Deep%20Learning%20Modeling%20for%20Genomic-Enabled%20Prediction%20of%20Plant%20Traits&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Hern%C3%A1ndez-Su%C3%A1rez%2C+Carlos+M.&amp;rft.au=Mart%C3%ADn-Vallejo%2C+Javier"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491518/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491518/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491518/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491518/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491518/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491518/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491518/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491518/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491518/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491518/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491518/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491518/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3829" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX B</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX C</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3829.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3829.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/12/3829.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
