<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3757" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3757.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3757.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3757.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3757/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3757.atom" />
<meta name="HW.pisa" content="ggg;8/12/3757" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="vqtl: An R Package for Mean-Variance QTL Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.118.200642" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Corty, Valdar. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We present vqtl, an R package for mean-variance QTL mapping. This QTL mapping approach tests for genetic loci that influence the mean of the phenotype, termed mean QTL, the variance of the phenotype, termed variance QTL, or some combination of the two, termed mean-variance QTL. It is unique in its ability to correct for variance heterogeneity arising not only from the QTL itself but also from nuisance factors, such as sex, batch, or housing. This package provides functions to conduct genome scans, run permutations to assess the statistical significance, and make informative plots to communicate results. Because it is inter-operable with the popular qtl package and uses many of the same data structures and input patterns, it will be straightforward for geneticists to analyze future experiments with vqtl as well as re-analyze past experiments, possibly discovering new QTL." />
<meta name="DC.Contributor" content="Robert W. Corty" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="vqtl: An R Package for Mean-Variance QTL Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We present vqtl, an R package for mean-variance QTL mapping. This QTL mapping approach tests for genetic loci that influence the mean of the phenotype, termed mean QTL, the variance of the phenotype, termed variance QTL, or some combination of the two, termed mean-variance QTL. It is unique in its ability to correct for variance heterogeneity arising not only from the QTL itself but also from nuisance factors, such as sex, batch, or housing. This package provides functions to conduct genome scans, run permutations to assess the statistical significance, and make informative plots to communicate results. Because it is inter-operable with the popular qtl package and uses many of the same data structures and input patterns, it will be straightforward for geneticists to analyze future experiments with vqtl as well as re-analyze past experiments, possibly discovering new QTL.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3757" />
<meta name="citation_id" content="8/12/3757" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3757" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3757.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3757.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3757.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200642" />
<meta name="citation_pmid" content="30389795" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="3757" />
<meta name="citation_lastpage" content="3766" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Robert W. Corty" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Bioinformatics and Computational Biology Curriculum" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6787-9051" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Ayroles;citation_author=S. M. Buchanan;citation_author=C. O’Leary;citation_author=K. Skutt-Kakaria;citation_author=J. K. Grenier;citation_title=Behavioral idiosyncrasy reveals genetic control of phenotypic variability.;citation_volume=112;citation_year=2015;citation_pmid=25953335;citation_doi=10.1073/pnas.1503830112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Y. Cao;citation_author=P. Wei;citation_author=M. Bailey;citation_author=J. S. K. Kauwe;citation_author=T. J. Maxwell;citation_title=A versatile omnibus test for detecting mean and variance heterogeneity.;citation_pages=51-59;citation_volume=38;citation_year=2014;citation_doi=10.1002/gepi.21778" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=O. Carlborg;citation_author=L. Andersson;citation_title=Use of randomization testing to detect multiple epistatic QTLs.;citation_pages=175-184;citation_volume=79;citation_year=2002;citation_issue=2;citation_pmid=12073555;citation_doi=10.1017/S001667230200558X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=V. Kumar;citation_author=L. Tarantino;citation_author=J. Takahashi;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. Dudbridge;citation_author=B. P. C. Koeleman;citation_title=Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.;citation_pages=424-435;citation_volume=75;citation_year=2004;citation_issue=3;citation_pmid=15266393;citation_doi=10.1086/423738" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Dumitrascu;citation_author=G. Darnell;citation_author=J. Ayroles;citation_author=B. E. Engelhardt;citation_title=Statistical tests for detecting variance effects in quantitative trait studies.;citation_year=2018;citation_doi=10.1093/bioinformatics/bty565" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=B. Efron;citation_title=Large-Scale Simultaneous Hypothesis Testing.;citation_pages=96-104;citation_volume=99;citation_year=2004;citation_doi=10.1198/016214504000000089" />
<meta name="citation_reference" content="citation_journal_title=Ann. Hum. Genet.;citation_author=D. S. Falconer;citation_title=The inheritance of liability to certain diseases, estimated from the incidence among relatives.;citation_pages=51-76;citation_volume=29;citation_year=1965;citation_doi=10.1111/j.1469-1809.1965.tb00500.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. K. Forsberg;citation_author=M. E. Andreatta;citation_author=X. Y. Huang;citation_author=J. Danku;citation_author=D. E. Salt;citation_title=The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.;citation_pages=e1005648;citation_volume=11;citation_year=2015;citation_pmid=26599497;citation_doi=10.1371/journal.pgen.1005648" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=H. A. Mulder;citation_title=Genetic analysis of environmental variation.;citation_pages=381-395;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429270;citation_doi=10.1017/S0016672310000546" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=X. S. Zhang;citation_title=Effects on phenotypic variability of directional selection arising through genetic differences in residual variability.;citation_pages=121-132;citation_volume=83;citation_year=2004;citation_issue=2;citation_pmid=15219157;citation_doi=10.1017/S0016672304006640" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Hulse;citation_author=J. J. Cai;citation_title=Genetic variants contribute to gene expression variability in humans.;citation_volume=193;citation_year=2013;citation_pmid=23150607;citation_doi=10.1534/genetics.112.146779" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=N. Ibáñez-Escriche;citation_author=L. Varona;citation_author=D. Sorensen;citation_author=J. L. Noguera;citation_title=A study of heterogeneity of environmental variance for slaughter weight in pigs.;citation_pages=19-26;citation_volume=2;citation_year=2008;citation_issue=1;citation_pmid=22444959;citation_doi=10.1017/S1751731107001000" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Kumar;citation_author=K. Kim;citation_author=C. Joseph;citation_author=S. Kourrich;citation_author=S.-H. Yoo;citation_title=C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response;citation_pages=1508-1512;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357318;citation_doi=10.1126/science.1245503" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=S. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantitative traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="Marchand, P., 2017 rslurm: Submit R Calculations to a Slurm Cluster. R package version 0.4.0." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=O. Martínez;citation_author=R. N. Curnow;citation_title=Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.;citation_pages=480-488;citation_volume=85;citation_year=1992;citation_pmid=24197463;citation_doi=10.1007/BF00222330" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=H. A. Mulder;citation_author=P. Bijma;citation_author=W. G. Hill;citation_title=Selection for uniformity in livestock by exploiting genetic heterogeneity of residual variance.;citation_pages=37-59;citation_volume=40;citation_year=2008;citation_issue=1;citation_pmid=18096114" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. Paré;citation_author=N. R. Cook;citation_author=P. M. Ridker;citation_author=D. I. Chasman;citation_title=On the use of variance per genotype as a tool to identify quantitative trait interaction effects: A report from the women’s genome health study.;citation_pages=1-10;citation_volume=6;citation_year=2010;citation_doi=10.1371/journal.pgen.1000901" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Detecting major genetic loci controlling phenotypic variability in experimental crosses.;citation_volume=188;citation_year=2011;citation_pmid=21467569;citation_doi=10.1534/genetics.111.127068" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.;citation_pages=63-63;citation_volume=13;citation_year=2012;citation_pmid=22827487;citation_doi=10.1186/1471-2156-13-63" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=G. K. Smyth;citation_title=Generalized linear models with varying dispersion.;citation_pages=47-60;citation_volume=51;citation_year=1989;citation_doi=10.1093/biostatistics/kxn030" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=D. Soave;citation_author=L. Sun;citation_title=A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.;citation_pages=960-971;citation_volume=73;citation_year=2017;citation_doi=10.1111/biom.12651" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Sorensen;citation_author=G. de los Campos;citation_author=F. Morgante;citation_author=T. F. C. Mackay;citation_author=D. Sorensen;citation_title=Genetic Control of Environmental Variation of Two Quantitative Traits of Drosophila melanogaster Revealed by Whole-Genome Sequencing.;citation_volume=201;citation_year=2015;citation_pmid=26269504;citation_doi=10.1534/genetics.115.180273" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. G. Stephenson;citation_title=evd: Extreme Value Distributions.;citation_pages=31-32;citation_volume=2;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=J. Flint;citation_author=R. Mott;citation_title=Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.;citation_volume=172;citation_year=2006;citation_pmid=16361245;citation_doi=10.1534/genetics.104.039313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Visscher;citation_author=R. Thompson;citation_author=C. S. Haley;citation_title=Confidence Intervals in QTL Mapping by Bootstrapping;citation_pages=1013-1020;citation_volume=143;citation_year=1996;citation_issue=2;citation_pmid=8725246" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. J. Wang;citation_author=B. A. Payseur;citation_title=Genetics of genome-wide recombination rate evolution in mice from an isolated Island.;citation_volume=206;citation_year=2017;citation_pmid=28576862;citation_doi=10.1534/genetics.117.202382" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=W. H. Wei;citation_author=J. Bowes;citation_author=D. Plant;citation_author=S. Viatte;citation_author=A. Yarwood;citation_title=Major histocompatibility complex harbors widespread genotypic variability of non-additive risk of rheumatoid arthritis including epistasis.;citation_pages=25014;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep25014" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=W. H. Wei;citation_author=S. Viatte;citation_author=T. R. Merriman;citation_author=A. Barton;citation_author=J. Worthington;citation_title=Genotypic variability based association identifies novel non-additive loci DHCR7 and IRF4 in sero-negative rheumatoid arthritis.;citation_pages=5261;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-05447-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Yang;citation_author=R. Loos;citation_author=M. Goddard;citation_author=P. M. Visscher;citation_title=FTO genotype is associated with phenotypic variability of body mass index.;citation_pages=267-272;citation_volume=490;citation_year=2012;citation_issue=7419;citation_pmid=22982992;citation_doi=10.1038/nature11401" />
<meta name="citation_reference" content="citation_author=A. B. Yoo;citation_author=M. A. Jette;citation_author=M. Grondona;citation_year=2003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="vqtl: An R Package for Mean-Variance QTL Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="We present vqtl, an R package for mean-variance QTL mapping. This QTL mapping approach tests for genetic loci that influence the mean of the phenotype, termed mean QTL, the variance of the phenotype, termed variance QTL, or some combination of the two, termed mean-variance QTL. It is unique in its ability to correct for variance heterogeneity arising not only from the QTL itself but also from nuisance factors, such as sex, batch, or housing. This package provides functions to conduct genome scans, run permutations to assess the statistical significance, and make informative plots to communicate results. Because it is inter-operable with the popular qtl package and uses many of the same data structures and input patterns, it will be straightforward for geneticists to analyze future experiments with vqtl as well as re-analyze past experiments, possibly discovering new QTL." />
<meta name="og-title" property="og:title" content="vqtl: An R Package for Mean-Variance QTL Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3757" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We present vqtl, an R package for mean-variance QTL mapping. This QTL mapping approach tests for genetic loci that influence the mean of the phenotype, termed mean QTL, the variance of the phenotype, termed variance QTL, or some combination of the two, termed mean-variance QTL. It is unique in its ability to correct for variance heterogeneity arising not only from the QTL itself but also from nuisance factors, such as sex, batch, or housing. This package provides functions to conduct genome scans, run permutations to assess the statistical significance, and make informative plots to communicate results. Because it is inter-operable with the popular qtl package and uses many of the same data structures and input patterns, it will be straightforward for geneticists to analyze future experiments with vqtl as well as re-analyze past experiments, possibly discovering new QTL." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>vqtl: An R Package for Mean-Variance QTL Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TS8lLHlL3oECjHU0fc83FXYkNy3sCPdFm-_qfWBT3Lg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491624","apath":"\/ggg\/8\/12\/3757.atom","pisa":"ggg;8\/12\/3757","ac":{"\/ggg\/8\/12\/3757.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3757.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3757"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3757"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3757":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uDJpHnyTR0tjX7r8NjKGO\\\\\\\/O5I8H83X3Mw1dRega1koTCCzmaTxFwVJVqlYafD4UHfW\\\\\\\/X\\\\\\\/aZmitVw+TypWqoWb5BMDmcPtXqkLR7SUayKvKw3RnqqnZhwbZTclN4D\\\\\\\/l9RgvOJghp0rZvc+loYoJUi87+oZNoDItj3lbAaKvUTdlwcpqrv6D8Oy1CQFPzg9TD56pjzEZ933WLO3vJLNdwj7cSyaQXJrWQAr2to3VHNgnsJAOADxnI2EfcPq+JUBqphDk1pcJcbsI4kzrSVYo3RWVPiDSmiESjZdH+Q1fnF6Nz0fWg4VI52L+pS+flSVyCnqyhzQPU1GB6GH6iS8VNaFDbRubj+Tj3uKF9Mx6Wiyxj4VZBAT0AoOND1h3Jjf8FSYxj+oPEZYqVmORIpcvApAjX7b1tjED5\\\\\\\/KNgMv7qvRicyDfGL5uRv2p6fnP5edCKnd5XdcdY42rp32iVRCU0z\\\\\\\/v30LI8Ehs5SSrznnBug8SdELrzkuyQDH+ASabDXyEvbk1HkQXOdPUsdKtoVd+ghrHPbJCB0dIcrZxgxNgpLen8zj4R99rTGtuwTT3CiVnzXdEfja2V9idcCe4sAlXUHCxwVSUzfzJr8SUhRunNKHLf3ruuP1crtndTnikLqu8TK8hPLHZ4mgbusuQZTOqHe7YPv\\\\\\\/sraGZNaDUo\\\\\\\/JHABHJYMJm30Q2Y\\\\\\\/bebQ8erLEA1lePB3a8VKy4BBr0yJFEsCLSzd+IfmuyrEvEqzphC1yQJp3yvCx38Tq6DxQAGnMzMpPTDMCUHXE+FC6jd2\\\\\\\/PuOv7V3G8Kh1a1doNvIKJtHOByy6Ko+2QRq+UAWZrh+8nrjLtm\\\\\\\/iHMqWnTY9jsRW3Ud+\\\\\\\/DKlvk+lPX+j\\\\\\\/MZj14\\\\\\\/+P+2QNMHmlk7cjnknrjkiOIsQEDJkJ12vRC40mJVlBTHFHZ5TY9tBn8ffTeC5i\\\\\\\/y4AVP3QwIw5WMgsfSjU+6sHteKOLhR6Pkg5aexGNmPWVqum4WDyHioO6PIj2iUPwTgtU9gW80VFOOFJGkc18I6zyBh52cYO4FVpcwvniZLwf96O1NWGcT8ni8WvhKobUFXkoN2RxVfZumDw6B3RQqkw0uV9S7mIHTLMmertB6ieMX\\\\\\\/r7vMtpnoLSPJ9mD0OeN554XAz\\\\\\\/vSN9uUQ4NMQ7dYDmo+yVNYaEAcRpC+62TXljXbUn2N5LOygIFrL11MxeYVDgdVej51y3G2p5ZoZuyNQF+0y0i5NcpjSU\\\\\\\/iNRODAlakIFAI0l1QniLUcnGlzYLaoW197dhUPViCzjM8WKWmYNC\\\u0022,\\\u0022iv\\\u0022:\\\u00229ba+8iYT\\\\\\\/+KdBBIiNSsoHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022362672786aacbed60cc0cd54df0fdfebec6d76a1bf43788e7d1a95c0c901feb3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SPKYBMTLjL2EZTtx8KDEUhgToMsRTf5uJwDBkpu7U0q7JxKcgDkwyVORqek1BVTAwe3I7rW+hmebNfspGTGrdHtgzHWDT21Z10TrBZ7osHzTaoRGzGsfOxr8hvQccXYP1Gz6wi9otpumKThEKfCwdjOPc4F+E\\\\\\\/kIaUbnJZ9+9WVRk3cwrv+vIb3qm0DIiJ1K+czRVWSPZrvgSDsNebHYSehOyfS2bbauATj9W8RGPX7MIbfBO7fk4ium5GU\\\\\\\/qll8pcCbXuWQ5W0NwUpVDgxmbUAdA5MZr7wOULS7azNLJHirrwjmPyAAUvTIaxU8o2RnPFBi5BdA9uhtY74lrfMVFt39VrTjFB0j6zvX0ZolR+mSC5YuiRIvY+CICxRUtwC13h8jUJsalSi+C4ZahukrqNh9\\\\\\\/dFiu0SqyIvsaPQMTpSol2KMe2wMjj5WlhvlyYzGWPWRBzj723tzqkRrCUdcNMo9hVKTnTqGmnY9dsyAq7Jt0cOXnpGd2KKElzK\\\\\\\/toRst2j2lBIgfE+5+1Hx76Pjy0dSDVYPxinCLh\\\\\\\/w\\\\\\\/XJ9sTy5o9U3zws+sB1W4p\\\\\\\/YR+4azd09+id0aENXBNyJoe8vWdKOK6T+v2\\\\\\\/2vDsTgW30G422+GpOE2EoVCgx4Hd6qphmUePXoZph66uDCzDihRMN1bmjyS6i2O1sCTqdzNitgCX9+RYGRa\\\\\\\/ZAiC0R4ODXcbuWqXSSL3LCG3jByRFqV+rGwUMvhLBAQLr21bajhrbYkSZjN64ir0NG5lD1cGXK5vTqZwe4ScGkVxN3ARaO+sNM602nUwyp8R+CefvXhU5xW2bHA5r1krq14WKEYJI3a6wyfZ143YRx4yWUra4STMZzgJRoXPhxKhOhRMs9s4ruRofmr\\\\\\\/SCnyAY9DlUs\\\\\\\/hdahAL2mCvAyHSA1m4c7iyJR946XYJd9N\\\\\\\/ZZQwDwEIGYlt\\\\\\\/IPF8b4TGx4a\\\\\\\/CvLyH7sNXjhymB+q+jeImNORQB\\\\\\\/vXdSpMe5IJZ0cMJk\\\\\\\/dpdc8Ur8BRr+qxi1AJEF5czJSi2KprZHpwazCJjPxSUcDtm4htsWjjHbb6kqKccqUm6UvEOMzv+SROkV2cNMtkDml1r\\\\\\\/G4yF1ZzVZXFMGC1q8qlCWzSFjw1QCVVCz37d27tjtb1H\\\\\\\/qZgSW49Rfjdp8UE\\\\\\\/gURneBI09xy78ACnfHy76kk9cdk8\\\\\\\/I5szLrn9GmGJPikOltGuSKoDQouiIIL7Vp5+a0AePTXwPpTU2AyX6WFqLAHN0OnyfuI0lVDoCNClKyY43iy0ZrKT53hP2yTQF3NBZyCWC3bps0LzpnB3Tkat1vTHz6um2CLO7ph7GhF8DZbp2J08HQdfWGYPuOj\\\\\\\/EO3IKOhejvsTMxJoDuPRASkjLSH1lTbU+Ae2J6ZjbDB14xwn7V4fSJRg6n6oMlsPNHbFEGJBtax9etG5F+gT2B9dn9nHkdb8a1BKh27DuXSATPionkV0j0IMSG4Q9Y1uVuU7GKCv8A6u8bF2DP+219dxUGOhfxSLnFMvVnWsFf7s49\\\\\\\/lH+v4mpZ5oLFdSCDfXoI4CH0KLxzWLUnuOn2ifaM\\\\\\\/+kkX0LcXI67NPFVH6d+clRGyK2iWLsEr0Tz+VWoodRAQrui3teY+bj7nn4l3g1m3J5808IhoE6Kj6o+DlsCQBQsqli4553QT0O71oT\\\\\\\/bhax+R+js61i3a8YPj\\\\\\\/4n9CBfgdphnD8AnLoUYeAwURyCZHS\\\\\\\/43huRB5V\\\\\\\/brINF4Xn6VEtx4Bx7cv77IP8HoP\\\\\\\/hMj64iuw1rO1XTXaQ1eYelCFeEViX\\\\\\\/dnQMWGNbeHCAiqgeYt2iZVjKH24E0AjdGLjHkucUSVJpgfwBMlGVqJTqxvqX6xptK7ur4XgLIhhUGhTToPkG5mgYeg5OopSCweezv1bfOfYMIT7H+kzSiXl\\\\\\\/1T5dl\\\\\\\/0EFOGeAtNRslw0YHi4LTcCAxrg4kBQP38MbDB\\\\\\\/tBOV7WbkZ+tJHq1ZCiwUT53aaa9BfgcbgUp6+N6Zu\\\\\\\/iTzMRD8+S7\\\\\\\/\\\\\\\/qRXVhR+idEFFiFe5Fpq23DX+A1LLVefWNnuhyPiYk0rxGAIKvR9ktxkRdNxAJ4ImcBM1HqqZ4+1giA4NvMUsPBhF1VxEVKNzGgomC7P\\\\\\\/ccdaHkbQEjkfvyAUh7r6ET+u3b0JsL1+rQhe9UbnrZkcbrE3Azz4EeRE6NxvrKml8YmwHyobP5OlAGEYl\\\\\\\/495wi065fJUASpV\\\\\\\/\\\\\\\/LpZpz\\\\\\\/LmHhqrPrbFhsynnJX5jU3YZUNN6gGQEQKGNEoW234v2hlCOGzCFYFaF7USw62Rqd9JsbbfjHEU8eaSxGloDp8E2XdRZM+6jJUlSg2tpMFRVIQLsdXnPAoZskii4PwH5\\\\\\\/uoqsv69Ppu0ZrtwelfShOAL51IcOEnJfEDh2ycMIRJnNGjdDuYxoTAztndkeKvvMzmuSqXJ7qD1YstZiB7QZJquqkDxY\\\\\\\/ywQk1XLa1KB8k0kiHfZYT46pXsmE3wMKkyrKU0ULgKfLp4W1anUwL\\\\\\\/HU+VksTXpGhgVfh5E8POTIMU0k0kfodrv0e0370+HNuOgAig8zfpUDaL4q4J9nBpzD+HvdX9iGUo1Ykh0xBomGegJCT4Lknc\\\\\\\/VSWTUApmwvBeiHpiogUlXRpykYrPViI5zwxr0RR3vJyJaEnepeQe3QlC4mEztGTwvpyudEfhLUA8Y+p+DJ9prdz8BZM9hZGB3HjmBGKXtIT2MjVygOIjxStkfQYkM9avJ2sCuFgkrcpZM96OY9flpdNdNOs\\\\\\\/yH7lr1P+6OlSThwr+B6cj5VAu89XLosnN\\\\\\\/aL0Qks2SAyy8SE2zOI6IuMuIAAbc\\\\\\\/rro14+yd8XNKHAY7GsCp8G1vw4zNG\\\\\\\/BRSIF0rYLj1Fse1PWS0MxaK6Q9ZF3ERK36lumv2XYxF7aylEy3nNx593M+nxd\\\\\\\/QNTPEZX3H+aR0drPfsTw4Vb6f+7XCO\\\\\\\/rBnesBSDp77\\\\\\\/8\\\\\\\/wAEetV3xZt+S9CRZ8zL2h5XINateJrG5u5VLcImjdCg+M470yvLab3kWrSD+mhmcg0r8202kdL90sB4IYK6scamuXJURvy7JZc+tSPR\\\\\\\/lNDik\\\\\\\/rExkkDZ1ojrsshfjmJcbP67FeXaSyAWRvDNPKqvO9w1hQHcEqJFt7\\\\\\\/JaM3rs6tyOrlwKahdRdLU5qhBL0Zdf592mwUaoRw4pBAtpfUoOgw9EQBTi7Do+Cjc5KWYaZdFeednxyu0Wn6amgOQpl5h2OsUo190OmdHlAnR+YldTi1gKdUIz3PNc9SYA0t8SzDyCwz7sGJjttp4tE0WdcsggKMGrLrPeQnw\\\\\\\/2hTbVv9Nknka\\\\\\\/OJncrXoF17GCFNe0\\\\\\\/ahWY9MsT69BMNYJYNO1ZlzJaYEErY9\\\\\\\/JAFtb0eVoYhFH8P8Frl1U0xyqEqEvHpFI+\\\\\\\/eXnBCBa9ownh5jjhcpyh2U38XvEI\\\\\\\/G\\\\\\\/Iw6XaphNnjvoLGp1MGRoMczcL8CyKjLjRUyjhhIEM7nAEr4Doayj9pIEkPIyHnyM0ikdk6jzawehryRC63LtwvfBg2hT+vrt94eXF+w5\\\\\\\/hyds9ewrNRRYNDTOfgWPcd+OhEz3dFph5tbctT9CCLfh7fLhzUnxuOCuBfP8FvauZFkyfbYEQqmQdk5xVxk4aPshkB+MSd7EpLwxd8fxeGetEiGw2hdFBlb8iV7iZJgWs9njnUOVf4S+zif2r3oezuz5NABydi\\\\\\\/8CEuD2paL+BIAyc65JzT+DR1rG4iRgfgMYF2vZyEOlVnHTfzT485HZcAeYMs1YuN3OVEpVEzzYGOQUULxUjQuEiXKqcmTwx+ivqcFuuD\\\\\\\/QXk4fg8U1QQbnpBWzWYDHpnC+i10FNnWAdNwkJejlaGnf2JTS6Pwk0k1s92O1Qqv26OzBglYbi\\\\\\\/z6FLXgTLGSH3JQf2BGzPqbbVbOm1VWFbJkiqxUW9b+Bfwcv0iT6GQWYLRkQnU8ZKO0u2iRboGmE\\\\\\\/FxVzdxjR70NYEoOgG8dRcHWLamB9Ne11fPCg6q++RXZ\\\\\\\/jGFgxftQGGFNtOs36jDJe\\\\\\\/Fj9KBl\\\\\\\/kwt+Bd9IRePEQ0DwV1vQ5mET4L1exZrvtFxPbUEOZHEoorYCBfIGu21GPHMyH7jm1zAIZLgempPAQ7B9wkiDxc3S42p7OADKBrRbJEEmnZAmzjRiwshd1mMI4RFYqe+hmyrYNIKVYve41OaTZ1edlstigSfcLgSiXojxYKKak95ETO8WBjqy13CIcaJySqWihgzzwEnvswcW1xSLkU9uey+Ln3aqblx04pZaa4mW4Y1dgv4uxFWOumCrbCtvOIh99OcAxm6kZbTfL8NSgJufG\\\\\\\/3WOGB099DERQ8WekjynuBRNsNadGEsAaooYk4mrjsekU2smH0DJ7W8FRRoF0nLXHruQyzFfiNxH37BNlq36pBlPTS1GulSlHfIuICUoliEEers8QgQLe9PWr01iQbsISc8+EXzMtd6ZrTGYGkalCJDBz\\\\\\\/rvEUKM4CtuGR\\\\\\\/xDKRHMpxwI5bS2ZE3drwT4Us+aKtGLTeN3UVooR2u\\\\\\\/pmcn27y0ogEN2M+F7vFQMKfyRrcIF6PDhE9UltNqVnAV8u4gUrcHSy3nBhSgoOHlMw8KVMa\\\\\\\/+5dgk2eUX8qbSzw5O1lccq2LhTbhBiubOrqAPJyrYcRE3hgqJ6LMZdCvqxeK7ps2LI4pE4yXN8Krn4tT8pmoFB5TC5f5MG9uI12R550rsNMJzaWqOw2p8\\\\\\\/l2AaDwOl4pl1sshXIQgBe1uDLKh\\\\\\\/aVMmNcsT7dcxp9FGAE5JsAsXfNhaze0JRq\\\\\\\/UD1kyAE7b1kEZrs1R53oZB0Ctn+I1cvDo4FPGHOcf41e5+\\\\\\\/kICa4nNJvcSLJpiIsu1NBjv05FdCgi6tpHmj8tnPIbYCTasAn9CwVKU5uVCLrLHAVfNOtVQpaMZG7L9tI\\\\\\\/OwufqgCSoPCiOWe\\\\\\\/j\\\\\\\/FfYq6g92rIWF7UEl7jWWdq\\\\\\\/il4kzTZrEZm6\\\\\\\/\\\\\\\/vR+fNbBznyfE+YSq6+MkZckwluRY5fR41HLgcwpULkikMh5Gs7nCkVF6QtZk8XGOrEGUHYR3IfsqtUK6\\\\\\\/uSDGdWX523kYnrv42LiMtDoy13u2ujaCVtbwmIEbBgGiDJMyDMOD+cl2Vt6Fplpqz8QN\\\\\\\/XsawSV8kNDhbQiM8tGousikrE4gZ8dLQvi\\\\\\\/Opjc1bgAQJt72cJggWKFuIqpM6+dUYzUhFodxzBjqcKu49NvDRToRDjabGiZ1p+31vkXuLh8UcrApnXM+oOvuX5S0CtU2LeFJ4tL\\\\\\\/+uO3P5CJ6eOVpOPShPR6HNtBb8fkuQlcp7uqJGeJrceYzqZ1mdqefxMH5L2nqvvrahLa6q9zWYyx3Zk9KQRtiBkv0prm+V2fhUhhjwXCZs2AM+xzNb3N1h4ddtL5PFaHlL8nsmOrhraZfED3t7yiLAR7g60zxUK\\\\\\\/ilIE8hcK5w4QPUpnupH8Aivum33gVAF\\\\\\\/jgobGhbqLu6wPkgiJoSqQGD64PjN5BBEs7m3T3JfNsa8q3h5bwu5Cnc3aAPIMmGB8RNxbKa380oRs+bBTtFvWSVS\\\\\\\/686jsGNDzz2Vk4JoEUmLCuw2Weor8dbiAA3RmTl9BzdY05PH3w6+dUMiJ3MX+UPmoWP62YKlW1z2i20R77IPZ7UCzLaPzOujy+y35es4QDCCSIZK7trOWzARzcLf8IgOqFTN\\\\\\\/SN2CbkeJCAdTALTuw6gYnmlzfXWvr8Xg\\\\\\\/KgTyCzYyRPu3EAn5wTmQmfxPu+maYfY0ObgasMWUzcRivjoJBhZ1IipRSNsndebLZpspXf4NfRXOktpfM68pgY+DYI0ExJel3U+N3Kf+O6wva9MmWT4pURwCOEkxVbp1PuZ4N16ylQzs\\\\\\\/oQ0Uc6il+Aw9aJD0xiA4k3ypCAT9mHR11vs5XdkzzrjcA6Ju0FcWR0qq7qFf9vEjbcjzmmZR5XE0qIMpbUN9JP8fXntBvHrGuVq2hcB4yI09v62iRKpem1V3fWI\\\\\\\/+\\\\\\\/jOlxtLetUGDph0sZYwUuS6gSpEZCjGteqkLG1crg9ABy3DFyLKGfnqL+MIr5FGOM+KcuGhhVmVT67T\\\\\\\/PxymfmWscGH101aMkC0Y80nCh\\\\\\\/uKcE6x7fpfxHb0ZRishB7yoGI3mb+2zXw7B1wJQNn7TDWzB\\\\\\\/oig1GU4z6VJhQV2uU0aKd7yaKG8A0zrzveQ+3QHwySQQzK6Es30hZ946gZgejp\\\\\\\/r7NoroQHghAP\\\\\\\/1h68E7eI2WC5h8VAa+obh1zCMGApfmqBUqXGcztmQGtldyaiFaHGvJVKV5pBspt6v+Xj8ucRU0YmYZlLTA5uRkP86eu+Kpiy0Aus+dXVQOLQAwWazQNpTLrbcKuLmDI5c0NI6v41fopiTfdwN90VuVU2jK2P9F8jjHfi++mGTK0Gn+wNwCbY6GVAUGc2I6sHwuxpWE4bf34SKYazSk6ZjspWgucHxsH0TSIRB+\\\\\\\/9My+2RX0I5C9zUgHjGPBz768Ls88GDREK8vEl+TNKDOsfn+F6bYDsJW5OVu++O8\\\\\\\/FBUM6K8sPDfhBmkQIzb0sBBzhlcwiVqUr2ECORlmOL6cYzjra6\\\\\\\/Uxd\\\\\\\/hBgH+T96InSiqArDaJKSgf3B+a1mH+nOT+HTN4nvsWAeXW84KMz2BcoXDtNCg+swi97lMNq+9mnX2AgEfpqerSIEVksD\\\\\\\/SqBI\\\\\\\/iUZZKGVzPwbeRGlAIuhanAQuGXq\\\\\\\/lvrVQoB7zuqNAw0OrGt5GMqOSpnGJvYnCDg4MKqFfo1iAhc0KoOCwyN6uMqUM5gGICCpWmymnd8bwyxtkPpcpeOjps\\\\\\\/\\\\\\\/LBrAc9266pqDQALW+up71DyOyfWd4qBnAM9EXDL45QHXxzyjpflVt5PICXLlEAFt6Cegboth\\\\\\\/ttU7DcaG3TqDed1bEpodTlPbubNSjWoqYU9p\\\\\\\/cMFYyanCws+guUNeaIjiNA4xSOoelEZQDIx8oc3JBcvMBvJy2gBQ+DksqSRFN+LAXfMY1G2oTD7P03kyJvWnUV7Gi\\\\\\\/eIzyU3h3UaeOpGOR7kStQYfwmeBP6JXdasJDrJ7OFKUH0fuThE8YBDzQsvFqG+mNAcRNg04pnHAEnOP9DJJuDfbcOSRVUIoyhWgLG9l\\\\\\\/3fHWVhv\\\\\\\/PwU200bl4WgAFEJDNyp\\\\\\\/h\\\\\\\/BazEmhX5rhZlbLqNPtungriwjZBtvOc7ObafbqTwq\\\\\\\/eD7LEsj+zy2KupvIXzmy4BbP0GPUH8FXLYEjpQmi7SiNQrWO2TKnrjkHHKjMhB0ZqvPaI1EgHmVxQy4HunyxqlmqSWVZNz7KFzcPQmuUzAZa8yL4E7sdeBFmOpCdviuYbI4WWco6YtnCO\\\\\\\/l8PxI1UjuiM6+aNUhXZS4dy4\\\\\\\/VKegr962YdQ3mxssBn8pIZ7fv2y+YXTxanBlwN8Mzq3\\\\\\\/u+fvIiKen5n035gyP81vZrEmQThEii9rVxszEMvCQY\\\\\\\/HEmgHCtN3JwxkjFgYbILx9lmZSQAfAGjagXAU94niP68VQmJTyaM0IvbLt6BXffgEWeVyyhiz+9o59y8QHjdqEDAyCg\\\\\\\/WxVQbG0lnr0uMtLh+ho9CI+BZesyIq23X+QswqY6N5PCoCNG3t05UUw95\\\\\\\/8SHwRzF9CtCWBwBjM9e26ZaXWDrd+am2IwL13Dv5eKU6SGuFl\\\\\\\/fsIkd8NO\\\\\\\/pGV83P6kxgilBjzCMZ1+w\\\\\\\/jXXphxTpX5SJbJA4kfWCTBoSDNsMKjZ5cluTL3SgrJ54StgFz7bef\\\\\\\/5S3JP4soxOMdchWh9qEim2MONYbWi99PWWH6Brups1RntBsAU+WEqTG5I2DvF3tue0N68Kulde+x0H\\\\\\\/4S27MUR+DrUTky4WWXqg+BtEH2BGxv3KyxXKAyWzCQxsJytRNtgasFkNBCaPpcQ1cHLsjWD+L6iLVtEdWSx8pGdR9iFOFKovbguUQN9uSfsqiI+\\\\\\\/XUuXRLSdnDhnodxxDlm0wOoXlxLsmlLAwoUglAW2uSDRiatSUMW3661\\\\\\\/DRWotMoVd\\\\\\\/p8OLIBDMfcgxCxQHh2moWMuvwwmtmx6k+NfKCVs6cwka0N+GOs1tJ9KIxJ+78iN5DN5usMxpImUdQUpqDEcSUzoFr++\\\\\\\/IgryfNEjxT1qCrHLBDtAqeLrxW\\\\\\\/K0V70WrTu495gcELzCOQDa7RzTMjOFELr+PJec3ZgsTFt9iP+iFmY3wBRxx42FP9T3lMK\\\\\\\/YHoqr5w3RBKR1gGKLijsdgR5UybdZCL96hHt6mZH4z0Go0+kOpLtj+sgy9MrhmswCxcYZ4sF4WN0nWP0PJpAz5HGqSfRhG72zKSykVKZEwBO49UVkrLtJIwOUIo11aQ4wJlzSBfFOcgZ9SjJlmf1lrE+p5K3uJbO+wo3lLwoidTPAM8WSFoebptLZAeFjqKbCjPxm7X\\\\\\\/AyoA4Z5M+l6d1qeKsAkM9I40vW63PFXZJQYoOd8xEK2oIpKtXfzFIEIZhKKGXUlVsIWpyQGm\\\\\\\/HTcTGl0VrG4teQbO99Kc8dcFrTAxhmWmT5rYZX7SN7dnLlog2WD9pkjo+LF10fB\\\\\\\/giG1xtK5bYmJVo3bUQyXV2ETlaO4ZOkUrkQUYeo0LMgmN4J6icKcqRvHSlgg0DYnt5NqabI2jasux\\\\\\\/m97+UoezJXmTQjAR7rOS73l+c8MDLBnqVTW+SA1IchO+NRFXRPXvSYQaPH2CCmA49WGGEarwrxf0gIqc0jnLG5PD9Y1q7xsC\\\\\\\/iDn81x0q6vrVy+rx13gGnIFCQ8vjuWDUGwgggw1hrp55V0w+5puaKRMjWONVJmL65lbDmiAf4bkRzKPgFKw9arMZGKMQsI+UP8hNNQp\\\\\\\/qKi7U2jq5Gq\\\\\\\/l9ccJBi4nqFL\\\\\\\/RM40xQI1KINA6QkPrQkCPD7bz+sNAIWRottd\\\\\\\/HgiR08STCn1yuO065GA\\\\\\\/x4S2Q7xzIjjwowWuGXQrAYZq8ql9edV83T7QMLD1IFqnPe\\\\\\\/Z7gEVX0bhpwoC2ElmMK1we5YYSqvx\\\\\\\/wZ+LfuiRJrTv9fscuvnjDLSOOwMuJhj4bto2ntkIu8v\\\\\\\/ZJYUNJNONmQNod98Di38OpPF7U6jkf4HshwQQjCvBZ\\\\\\\/QiODKsTbWalrB5IwolUTc0VJrK9RVV052ive\\\\\\\/crSHeEGAPwTOfQYSaU8AEsDqFEfwI46iMLlEzSK1SV3SFIR3EXqWtwCk2FKAp8gr+aGbltABZQCeTL4g4SpsLNNWlh7ZYGvb+J4Ob81GBamqlxgs5XH2dbYkvqROBaWipn0mgSkyuLwRjZ88axcon0BGNn4k6F0iA82cun36XEUZQRr169eAyfhFHNxG3UFer\\\\\\\/fxqxO3U2fplqd2ykpImszaO\\\\\\\/beIrSOwUI4bu150V5IJra8DFEvm+hdmV9\\\\\\\/zJLH8WwMS6lgr5\\\\\\\/cedLfwrt6XY3rBD\\\\\\\/xSEj8vEJQRIJ1rLl\\\\\\\/aqCBCyKIHOXS2HX7Kbfm5zNXS+ftWXTU1h1N5LxwFNJrnw4Ypq4aRlzNAvkcaDvbL\\\\\\\/IfSRFOqJi4iAHKNVPhe23S3wfdtgCKp4INhny1P7HDKMWMCT6o+F8ndxqM1unh6pVmZNXTa20PRv4gKdfBMxkafT7W1KhAZLRwPENzykuK9zKting63xaQObfbKaTcA7gCwh81e+UgYt7mBISRH2iRm8sU\\\\\\\/bFvCo11quAMReSeQMxeb\\\\\\\/cMVuIvx+VKuxjnZpw3fDRtvpHB6tN1LntsSq98kYfqR7QnTelgk3wpKY0mGYHeBWx\\\\\\\/8x0CcHQutesKE4ZYk7u3esEv7opi7FU27Rh7++YriehZsLUKaZL98GfO3560\\\\\\\/heX860vgSxoP+GG3vA4c07dbckpD899giErVKUAl0vZN3kP\\\\\\\/SEYF3qPw5tBtMFuT+GKcqhY0+LT0Z3+Vugz8a2AtmGrW85oTqxNoUmvYMW0QZLD0ko\\\\\\\/h2slfNSDm40+Vl12GIJi5Ot1cXN274anbktQgc256LNr\\\\\\\/Pwe4DAjAgcFfXwOw9TCJ2YT36SZqNDVE+uR6PcjFuCwY+CJ6w4q5SSJQ\\\\\\\/qTv6JfdWwUQ68oTb4iDmoPe8JdOBsZKcrrYf1zgR2YJ7Yce8r0G1DNCqzXUCrNPqSYA6wiCX7CfgqOba7HsLjqGY4NhK9NoNzsmafAcQSU5VpSCrCSjUAHZw597hFIgmyEU3vYUF3\\\\\\\/dFrE9bGCXWDkJMM5niYqAPaxHurg6KOirLhD3W3ZA8rLF468mIoJPCutq2+SPlLo0EyHs8a6V6IVzr81V0nzRAKV6nkvEX\\\\\\\/SPURGKonTlQyBSHaDOob3Kmdz5HJ0Zxvsv0ltHu6SpgOuKn1gwfvjIhu52Zu20xcESWluDoc+NQjWYnMhWSSPe4Mo4sBTjrL+GyR9VppTR7UTQaa8WdIo88ZmGX\\\\\\\/Q17oQKnuJXPInvgLHA7GUp8sXB2zB5EK+WBxWSFkGhzC+7zDnVPatU3CXr\\\\\\\/GFT2waDTMjIyMGtQYMkzRMMHMw2LpCRmq9qldbNk\\\\\\\/JUXJ6PstRaqdYHWyJ7yT9P9C5QLmRQ63MUu8VkpmcxjqkslqaT\\\\\\\/ihyOaGeuE2T01AGytkyT3kBlx\\\\\\\/avSLpNXTuaUzQiypCi5p7wvyKFmyu8fIFS4TQeHySPkCrm2d699o2whe6HUq9aUycvUS+a\\\\\\\/3WoDw0NN0\\\\\\\/7uhJqI94Zp+UYYn9UXyI8bSPDKdoY0mCEYoPXsScFS0f7a8NO0nAb9i4nPP2Lo7J1xGOvIM+hNUTDaFIErtCDo+5ICdwq2geDSn1+xEIosNtshgc+0hjt0gO7iXOKci3v2hhVkF\\\\\\\/EkxPXo5oCFQnhhvlGYbPvRflX3\\\\\\\/NStDFS8KVHuAX0gfcnYPe5sl8X+yjWGhI8bt+t+7JIC0Sz7EbCngepHZzhgHcG6FF17XMNZ+zbwFZUIMUVPnRIyPHHCmlaikEelt7Z0VnrQO16JnxS5phSovj1zvnAyIIoGsy2mRE8gF5a9TquZeeO7YPpHdXrDOSanmvaFsSGihMHxPhR4mHNTWuVSLWvOLf8Ro5ofoUNuDf322YgMmcxe1vpfRqqXreThd5BShAS6ylsE+Xd2+YqyN6\\\\\\\/hgliWXGr5oxrs7l9cI4ApbzmIHf5Bpg\\\\\\\/sfpC6JbLUfmvGrdi2Us+B3gJZfpLqKXTyF7srHO42tgjzILQrvkpIynC+MKOGPF\\\\\\\/e3hU5pe2tJJTKbGVoW6YmZL9TTyexIFwMu24CmMfwiWdS93hTSFNQZmWS57SKgnfgptJwDhZ9YSpESf\\\\\\\/pJtBfThqVoNfsijZyCgXMEe\\\\\\\/vYswx\\\\\\\/Myf2SjFAxgIFfXGE4qL7GBV8ndKjDcQgk4AE+o8KH1SRCVAPmsQY1dmKgrbAgjoS7hC10hGeo\\\\\\\/wLUIB3+ORgNgSUCjX9vYava+XQa\\\\\\\/wt\\\\\\\/yxNuKANfuxilG+7U3xZVDV9p5IKoXBe+aINL2fSixAYdqec2CjGiS8DaYA\\\\\\\/xcW4d+wd9eiXCUWCF35Op8MOFbzh5xEDCVoX431fs3svHGSG31w4Wzf5aMra1GmI1KFb8iKsJqDe74HbhZUWduCXill9+AqmX2ssgMnc0oajVjZ86dZs7kTpI\\\\\\\/onUemLAnZiVQw4U+wv\\\\\\\/wRrQJwvFMr348OoHabM0YNloIVZB7J\\\\\\\/9QfyZ1dS\\\\\\\/N4s\\\\\\\/jTZ732VjjH6v66sjccRqDsNCbsYzBj\\\\\\\/dIhZZ1j8Ee309PKjA3N1uyisDfsi2fqZ2537h5G+lpLU7nCo4LOQV8yUn0UXFXYv1fYKt4zibLZZ9x9ffWHDfuRmOaMXTNHDDuRyOeZR7SP3yHq444LsM3f7ra6CJAyKPANDQzQtP\\\\\\\/8FVEfip9D79k5X5xokveA\\\\\\\/7x\\\\\\\/6KhpVfb+wwIucBE1B613xTihTEn7kcsWEdfFnFtgOwVx9cwHJ9dIw8lAlAnPsMhr1h18bdQ27gxyMTudPlcFvHP9gijYCVTDq1NmifhTmGXz7Ak\\\\\\\/1CiGMKmE+AbTeRbePGOOO+zMzADZYSJRRWkRen9ycmHPT5XusVw3reD+EpQ6U67RU4aEazWjtti4WVDu01tRr8etjck59pFHcjSrlllvg6tZ5EFdIqGpBs+TPonwZ9XKqqvaGglbCnB7UwZhN\\\\\\\/Pg+y9DmE9GuxymC3Rihr2VmyXazNKV6xhZg2ahggn6FmkzaUpOPrO7qyBGsoFuCZyEBmNfcxqHbhvk\\\\\\\/2eDF7E5B4pZN78gu5ehVGyBy3q+ICw9pSfPR\\\\\\\/45zVcPQmDLH9\\\\\\\/4SIs1uiYrwZUTD5jjoGzxFRyMth+yvKhtCqc4mTBWCxbXFehL3bQmEFnrXe4MJDCN6DkSq7r4NEgQywMM6\\\\\\\/vfkXqM6XuSd4oUMzTSMhsrDMqlulm5aQJYmN1cHGKIRzbBAj4xDCA+q\\\\\\\/2OwIh+Y8PfQnbEigtSQPfhom8ZADxcWpwXQUEbQ6kXYU5iIRIQ+TEqGAKyjii91JNmXvxtG7tZU1DXszCYegvJxOKS63NmTTdl4WoQQiVdM8UCM6\\\\\\\/Op7k38cTqoEuKfm19aYMAyi28P4kWoNEDfTOuUQE\\\\\\\/3kl3SuqZNpg0ej7VuXLedrc+2LZzEQ\\\\\\\/Czc9DJQYXY3\\\\\\\/vVWBB4RpvpBM5q\\\\\\\/TndqOHlX2eAheUZRGhAT9WeK8lKj5it3fscDBsHTXKI8XbgXtHSOmNM4cgjGbbNYxmqFKJKk7Oe3ufmYs5nMRzwMS1rJ761ha89pkv3yuDPi9zBZ7HY37uSsDl8xP0fd8qXOkIK0m3hP2xnDIPrxIBkEdftIaI8lpnUkavBl14WKFadHUaQmP0UkDkT3Ig9ta3\\\\\\\/gPcv\\\\\\\/0fGQaho4ZO5gi\\\\\\\/I4OPY4Zr3gycwosREdXG5wED7ovywUjVHDmGoxSTe1CTXLaiHL945sclZkT44FOJJs17R7ShnqyUEF9AJF+0\\\\\\\/AUDud1Lz0AQF25LFscrGG9P+KrkwXjFQtgAgY7mjj5rDzFyb9qkMrQyG1VyTv\\\\\\\/pmVv70J7Qx012VGfowffM0+7ySgHraIw17sE26MuyXTFTudPSZTDwqXRxZqnLFEO3pmvOR5ExRkj6hGeieYCXRdQr+5Xkul4iU62wmd8T+nDDySxfKwbihU4xLBggvMR6IQIfGWOmz+XI4E+v0afzMQB0PbnIipqBc43QkcYXe1acvUboxX4T2ahLq5GmOqcn\\\\\\\/USgQnk7o1uynkQw6RzMNTljAhr9i\\\\\\\/EtFX0wYS2vfetYs9ydVFOXB1chmO0I581t2WuR7fMTJqNDrMHot8cTZhd2fOFPq1gloiDbHtE5FMsiINwIikoztOfn8ncySoPkSYPnOHGqCJcxjiStNBx\\\\\\\/YR6LXzmYtuJg1q6p4dxSo9OrXu4jmWfNSmrdRzlZYSPl\\\\\\\/6pm02CgePQqrMTbim5WfDNmK3ANAcq+k4JOrZcWidiQMY9KHhEJiMGuiKffh0X1CNjNbIYlT6Vrlh\\\\\\\/if6OvnqS5Hkl8Wgf1NgnO5DFsl0QwgNSZCW84Tcv6TI+KpSNB2glVs3PWmOJBhDKqiSseRKTs2+OuIiNtajIDyYvr3pxJdOPvk\\\\\\\/P3TKG8X6EExjFwfuCAnTAncUniUanTTFXlIvmxVN+nJh82qpPn6j086ZcX0jppz4XtZvkWzp9dlmKb1VQEiFT\\\\\\\/DVEmTmmVp\\\\\\\/ocWqh3opYrwti\\\\\\\/BDEpIjacOJbcBQ+o3gisKlpOzjUz0Oj6bPTaGt91h\\\\\\\/ttWtZkGiLeLVz17xZX1zo8z9sbeZX\\\\\\\/XId1yQaGZTTy9yX7T0NCJq4Ae1Ezz6RXJJSw7Jf9oWP3yf6vWt5fVuNcIbNUNFg0kz+ePip2bFAlv\\\\\\\/hXKxCwjHdR97iwv\\\\\\\/hEdVpduEy3nuKRktbXAZMSDUxIpoYZheQgevolwZ4H+wYgjx4E794bLL7dFP3jRyZ77yTyTaDkYmZwVi7ATdqMSnsicCh6HRdHpz99z3Kh\\\\\\\/gJJl88xWEAV+doBHo\\\\\\\/ApyYWvRpxU5xlIsmCWuvI\\\\\\\/U6MiZrvEac81Q9jMgJkfp+di1JTxUcbEvzI+BSBo5cL3jgAGs6DcJLxJW6fp4+Oy5REkQoOIYq9g4LDwzyzdsaOFY\\\\\\\/Ar8HtqR1FivclftVr7\\\\\\\/KHN\\\\\\\/SGLtOloHkRc4HIU5xBEWmT4a+jiZ+lvsZjgUm0iY7aOpPEtS6lvJw27MBYjl0rvu1k926ZNgR2BRFAl1n5VxhFVt5SyO34Pows7LKbeJXxPeX0WLB+Njvui5GgCA2kIGwiBqDx+xN0ofqKfw\\\\\\\/oaiVPwfje7I4xeNI0eITjYir6PFck0SzOe+mdTadI2u29idSPXiSIN+YASpYhSlPxCE4pcpxYhoYdxqvASu4ozPYqcMLOfuSWaCFjKOb+kCeyZ0P0QAK+wAsTda4O2q5YStlZqMsQb5F2a9h36ebIs5l8aW30qwK19k+yY\\\\\\\/I9g1YLQD1RnijLt5PclDdySBZadQiQdvjOLSp3n8CBx3Qp8hAM\\\\\\\/icQP\\\\\\\/UNp6iwWhQ4x9BQAjXlwmO1LD0PMZyMlqS\\\\\\\/alI86hPny8EYkWMoGdxyjobGUzcenUMjq52LwH0IHaKlGRYJMUjck6DpPUala\\\\\\\/33WbKP8mfoJxrl74LvgwPe6Rgq2cEWsm16ItrpFdOHJNIsWI1mSjJj9i4OVTKTg+JPJZJV\\\\\\\/VBaluRuxSIykp0sPlI89p3Ha\\\\\\\/Kk1fUfUU0WMoWQw7VJAi3airINe516KowDDtXwDSNR2S+aJeFUY1rRCDFSqiPTCDW6d6KHAAwCbu9Cq\\\\\\\/s4TeIlWvBCzZgOYI1BtVIh1a1iePq1EYP1vvGSMiCLJ9BZyOnTQBBShkXfM6XSH+\\\\\\\/PivKtVQZol\\\\\\\/bi7Y64RFF9Cz5JDSXZ5ehlBAnIzVgbOyr5b+R3vPLnNL0sWq6TozipALm+b2ZnazO8CmCon4nKZn\\\\\\\/YVf27qh6aBsEmM4mHM6PTbmX\\\\\\\/lpP2ySrLV77T6hITJV4FrMdLVddHEnQEMeWxIFWjm9JXnU2+9T4KvFjrceH83SuROBCzsQQUEmtaED0ndhCgBUJ73f0yNTd71Gd5y7BG9YE2WuCJPf2QDNkrZPcIpLo3aDkhYRhB+0\\\\\\\/A8L3KBaxfym1GLjHpwXz5qOmhuoa08T1NVelCqu6IJDdz1Q169022REDBBhLgTnKO9ivtlU0wOr1gXTEonnnXEf\\\\\\\/voHadMl3Przqsht56BTfjULDo\\\\\\\/DEZ9y5jwOAzj7Dh5br0d4qoNhhPZ9BuaJeSM3KvOu6w+4NGH339nOiY0t3xevgDW1gXYZiwkfDS5QG2KKOpCwy0sFy0MLxRMrYKC8FrxJJ30I6oTu7imZR5l2Vkpu2U1ESLFNo+Y1TfRL1UODmEwyRL2Ohcj17WvMzla64cUoNOHXnH0gB793GZS8Xg8DzuVkUOmorWFbfkni0u\\\\\\\/9y6X198Yb+srDOg3VoGqh3XLEVzKgtr+aPmQHVfITlSt1ZZFQXvmrK4yCT6IWVmah+nQcvGqjxdFmq8TviLZqUYvqEPCLamOjtqej4uFNc81IH5r5Od3ZQ55YIRrJuFvjWdhuqe7JF9yXVZ+RYx7uEXDZeCOKdbAgV57wEeliFVYjaR\\\\\\\/qt3Z5t2Sxh4DxyYFJlit0whyFOF0sAHy78PaA155Nic32t2w1QBdV3FurBcB7hy7rKI5mt9tJJvfze9mu6CbcD67hnB5OVpyE9TOJTM9qhzMAh0rS00af85mAC8GpCY1cFnEpswBM0sMfQ7TNUTxgh490z8IXgHuQcSgVWeOYgT1hH8UNuHw97Z9W9cSc\\\\\\\/v\\\\\\\/qAYhUr3o+EZXlUYMiz8V7OSDZoNxyyeGDzArfDs5BUb+NMHEIEZLED0ohTOykc7h\\\\\\\/yL1pAFzN8AXK1V1FZzjRsd9jRPja99PpMSCUWRbMdOH3qWzzOjNR1Wg72UR1GwQmqv2Fu1bG9j3WXtJS0+PPBxPxSl4\\\\\\\/nd4YkQ8LJbY5mChd7VGKW2Z2zNsH2beHI+Teqo+7NVAdy+ktmw3yBi0CXNLi+VnHL2ksYxM4cnrpWtTh1+zSQEKoEVCSllbTFEuIHk8+q5mBx7AoJUCIYLdeqKGlHHMsPk7P2g+TtbEIiHsFpd3U04iH3AbgWE7eRGCPDMtmM9vc8d+UpvBIzw9yqNGz5wShANg7LVU+oY4FYQzYu0KK9R\\\\\\\/JrN4b\\\\\\\/33MQnMTOo7rOvUl3P9NuQK4EIx3cnkhNOcj3qXW+4k\\\\\\\/9jy21CfUowhQFij4Qxgx0A+cUuYDhfaEUhZzWoJlGNvasjz4NSut9ymL1yITH84Y8YJr6AJGE+4eCvxw2TCTHh6Rd9okGjsuPm23rJ3eNq6qLOF8C\\\\\\\/xf9cFae0iysyUH4P4Vo8lbvMQ6NBEqpGZkENZlpnALHSnOYvFqaUELJn2\\\\\\\/E\\\\\\\/J76bxt+SSeH3dSe16Ell2plm9XHipgCkH0VXfJiKn6rExTJpF4ewYYJKK804csijL\\\\\\\/89O6\\\\\\\/CgFQSHqmklTKofy\\\\\\\/70LcpI6DX3GhzzXKSaPQs8mqrlaK5Y0oW3HDV5I3zsNKLH9H76o5xRYmPgpPALNHoPvuO1jgDeegQ7eM12feoQmVVycpDdfKTCsBwf8xd68aBVwCQjSebBuowZ18beYFTtJ6PTqVWDiljvymWA3iOJtLwQfazF+75uS+8h7JiCTEZJEuZJOhPMmAI2ThlPtPc8TTDDZsbRaxHEekhhXyqWa2BdKAEcw4T+w659u6ufjR8azHz4Svh8QWLz5CYovg0C+ufoiOiGL8rL09uYBQrItsSY73RMlM6J53dnMcV\\\\\\\/SIQ\\\\\\\/bcO+oR\\\\\\\/WffTGPR0x0NpwRIvdxwa8SDghzS6reBGZ+9mda+qridng\\\\\\\/b8RQSVlUNTys3DTrWzC2NUKuQOHmiSGrhrCn0fFHK5Qr4iHSSBuYV3NJ0AWbm+1IoyjNK9fOcjM\\\\\\\/\\\\\\\/YHugFyV+bmW1hFzQwd\\\\\\\/byDa2+E3KyqJgmHr6EVI9wcZpodTMOVss4MKugl74AtaGIrZD8k1tfczly\\\\\\\/S9Y+Kf6xWAvgsXSsCNwqMBI0fkRpPuFKz4kdz+4MPli6Gu7q3AClZvBQiuCcanDvSNgO8t\\\\\\\/NLlxC+WuVUOldcFT+3j\\\\\\\/aJoYEoclYPWLsMAjUROddTOoSKJOt23++VmDE9CuwjfyL63IOP3gkI6vMNiWsLMejYxmdisM6LP8yPZZqqs6S5znwgrNPOLvqGfzLCmf5alpmIXmxFaxv\\\\\\\/B4RQt\\\\\\\/QCkhgpQ\\\\\\\/VIsQgwF56brXqw85X7QihOpprFcgzdbdLUXTZ6Mbr1Jbagm0ceSVWb3T3aLCIO9Fo8RRwNEYLy9oa64DdF9EiXO0\\\\\\\/ak0l8tExYnC+cdJxlwEM71tRxIg4gGailx8r\\\\\\\/s+KogGJEUx9o770n0o6GhjBNDW9fS3bniRl\\\\\\\/qEuyQ2DDkD2\\\\\\\/\\\\\\\/5DLWSRqcQx59XnjogPXEz7wzkyPzz5j++1u7nGj5\\\\\\\/CDKW\\\\\\\/CK\\\\\\\/TCfYsJiOMOsN51OalnleLDZ4jfqVtE4QRs2N3wkmmU9r8E21yIHYWpn7LRA15MPfhL2+8oqkJxnfIL4QN9TnEHDXMUCnk\\\\\\\/9aMJUxGHYYl\\\\\\\/h\\\\\\\/VEPGJipL1uPg\\\\\\\/YeeGF4yHrgeZLcWEqNRLvCCJ5Yh7OQZNdfu+829J3ogQgGhfiiu4\\\\\\\/OZPuIWfxE65ytxvKkPD666yUi3SQeA0frXeXLli6MhAKIqmnk860uHZpoMrpJoA4LpPNA\\\\\\\/tVHJagKVkGNuZwx8i2vWmxgNf2H20Sky6wpP4O2\\\\\\\/1sGVb\\\\\\\/sJ56+wiPZ9o6B708gvsVfvvhby6kCvXfLC7x+\\\\\\\/K2dScWGbR3PrIzJK8lxDl8dgEBGEqIkrzCB+1ix5uVWt3Y\\\\\\\/qbT+Cx92+o3m1\\\\\\\/hkEvYW9sX4I2Hs6v0hL+9MxBr3IKPWEybkAi9bSmXfIeBolnhVdpxv62TNGNdVTLifi69IZCm\\\\\\\/3MVQxoq8ZLdJwB42CAzCbzrpIl1\\\\\\\/5WW3CBgTkRwh+zfeFOdwzg25YqPJ\\\\\\\/Gr7OvMbuu7FP6kn898CsjP05PblbzGho1QRu0UbdJiqcQNYbSkbSGARX9efhBNxx5APu87c2icbOkaLJnFZLk0N6trkAWdbuV970owSk\\\\\\\/IZVMNNdwCJVGzp\\\\\\\/C5tv82RSw4N8qcd+3tj1ptshGgvA\\\\\\\/l2bXDw9k+d8Ir\\\\\\\/NKK6nGAyzWiBi8uNZO2fy8uyT0ypEfmvbb70GJdArXFZjxDyR0O\\\\\\\/MjHVX3kcC5Oxcl78F0FZ8+gekdmEcegE6p5NxrJbuXrUa+E4pYkMTjc3YUmcHw6l8Sb4hMTdWnBBiwZ6QKDDt0l6Foci36GqoSIk1dmWgv0d0gwq8YRvpDpnGm\\\\\\\/x9hJ0qqc05ZSf32KnS4jWP8cTPSUEMDLWId1d3Wjr6M\\\\\\\/fbkRPSJ0dKtdfFqP8bpw5GbLxOxzA8P4QqdrMy8Yu75xsECAAYgMQgs\\\\\\\/Lw3OEhotL3NvJogBToWku7850GSTKyUeo2361nNKnU+pKFwcBkf9MhzfR\\\\\\\/Vn5rGj0JfEOIQgZI6qpidlmW7+lLiXvC5yjF2mm1k6F+EtkRnVk2JJfw1HUzvEmWkqh2JMR20Vd3HkhAVWJKjYsvMerUjRLAtAc9TFKCncE8VHGC4XZWvtrBRiLmzi1p7qXRPpBtYZjWB5UPj+hUHse2BZZv8bCpitci27WdqcLDP59UNcc8yuTkvOZuJZMdnCdUnKxHLNMurk17UmGtYJZcWFutv3whe2KDXbnSjjFcFQZAy\\\\\\\/412VY5\\\\\\\/A6olrf+sUDdLufid\\\\\\\/2mfnxxB2yayf0VMJ\\\\\\\/YZeawxL0uDPK452ovox+JoBfzg6y14lG2beA+TiBFWXCAul5ivWB9o2vp22T6udXN6o9M8UlfL7cqYV9BIdOtClGwHZQHsAUYP1lxoIiyvX4RejFjK1B2\\\\\\\/Inwhm5KhwurUUSrO1xN3smZYTqp8iHNEY36OBDTQoh5sYvsRfTF3SM4cIf1TPltbNdbGTSbZ\\\\\\\/haBFV21yF\\\\\\\/6LpdKjmMRB\\\\\\\/crtxwUEJbfXVNAuY32xPpsEtiTWMupxX7CukDn+YSDkcuKiJJop9RXhyw\\\\\\\/RmWczKVKiVs01oJkmRh5jQGSslNFzc4ozJFaX4QVGmBDU941CDNjL6d\\\\\\\/5hxcjK\\\\\\\/u0ySvb1I6FYeJiVfhG3R2fdahQoxKv0EOioFbPcvPJi2mmB5CRfUJG0m0TFLQS70pG649VAJ3EBFjoRKidVcG4FGff9NVna20q5CJEOcGYIXzl7\\\\\\\/w9Mj6CmP+YaLqwA1xT1apdcu0bcKhpL2FbGsiZO5dbAYdjPM7BBuQMRBuczp9v2mGLsNTXGxe9IC67KkGI3wS7GpNzALlCMTpZfKo61r3BQauGOONeQXJhKmVi\\\\\\\/a7dZrbdpP4neLMSFCgnkdel7dINTaga\\\\\\\/mjORfSus3sDl25OT4yfP4NiSRL5VG2T4OjOIwHk8q+eM27za\\\\\\\/Iq\\\\\\\/+9kGuGPhhpDH\\\\\\\/oX0bfwvZ3drCq3VLn9NB4MjlLaF2JBCAsi8qZcNoguenqGKFYDkmyGFHZCTtHcDUR9kB\\\\\\\/bvzjA6cfj4Fl7k99nGMW\\\\\\\/A35m0Sz+rqYmRSZHhNQBbHOL3xV8QGK+0bc5ZhbOM4MBY7XBTnCrWGUh7FV7yHHF03PddPg7u63mKuhjq\\\\\\\/io4hPazP07w2tZc51tW55v07N4JE5VLFQt\\\\\\\/ZCsP8+cKE3ZE1JVs1LfMIG5cqhcOU7wYJffrS6Tn\\\\\\\/WGngck3R0JC3NeIVnx9ymqDHzvdlFnNF8Qv9fF8VS2snxs6jYXROOFCcSjvw+gZ37gAq2OiePuJ2McwgjjOGZh56V+WADjXUW5iQANRfPEHVrm20fevMXOg9cDXERCKOJGmpfYQ\\\\\\\/vsALcCdSPHbvtSvOvFi9TxPLyK5vl5bdvJTEiTV6ZZhyQN\\\\\\\/NxCTbDIk0ZqzJ2mLV9En\\\\\\\/pXZo49gBNymFFY1aaOP5geFF5oW5+UYFEfPXRL\\\\\\\/6+saowza3BCPln6im9jnfNYCyq5PmUw0HLXy\\\\\\\/+3B3RPUajHBXH+r2Pt51Ny2aOWTf6CRxjIKpTrd1s3y\\\\\\\/5MrRn63dMWHZ8J+SxnQCcRM8peIGOW9ls713lhbv572BKSWwCFut8xKp4qXH72quWYG21N1qSMNUmFYBOBw3ADnk0e6fU\\\\\\\/1YVNLGCAR26jwxcZGch9vyUZBPuv5mbY0Bvdy4QGabO1qBzhMjsXxSpDfu9AxJGTs1TyxEyNJpQ2V\\\\\\\/fLqjheWv4uUi1puQYwZhPz6u+3bZll4sfKELGnEptvgbaHRNHuRwtH3Lu9ib3A8S6rOGYWrbJaK2NO8KbfnR\\\\\\\/xHD6h21SEmPRzQ8BURkjQVlsHOkSmQK7M2eTfZQk5v4ew+D4XbTAJOb+t0VzXD5Rf\\\\\\\/T0VZrKlirUTrqX+WDQbxglYjgRlLkoTwQiDtx9ISXAD1Btrm9NwMv4fS2YFgPjgJ\\\\\\\/K4czhzr4mkGo61+xc7\\\\\\\/5x9UFvm4u\\\\\\\/HJsVtcD3fclAR+RRqMu4ET1ECwEU12CFkdf998da3nAIGE54ISBEUXNKsDmiZhBqtLAL2FN0kEzjEr40mSh7x4BBuko8CZYBngSSRyS80xMmGBf4\\\\\\\/S7J\\\\\\\/SFzEqr0eMeHK9LoKN+GTYMl8h9+gjw3zuW85fxtvp4J1C\\\\\\\/OSyUpDJFEfvfu4zXB55JTq8Y7XzUNSGAJ758B+OikCI2sswyWZ+7Z+0X7bD6dfqi2I8DsJY5xGNxS0opmGQB1OFxhkp69jTpqyv6xL2fJ79y5Iz\\\\\\\/YRwyfB4Po0AnFiNGiqVtZNbXglNpPKNMG4PCz0wPr6W1FbOXun6rYhjxtDVIqZDS0Xx13o4ciiy9ha08jzFh7PZWNxwfXWhpNLzNx+cil2THduP9lNxb0cYdBYSwyNOgDYcL4TlgVSkYMQRafBQFbOK7FtIf45zXTzlRVHxaaQquXFeSU\\\\\\\/bbdK6bc9IS\\\\\\\/dQBC8RVDKwauJC3fP6pE4XKVG5BOD3U6DGhz9RdkirEg88g0sUWmEPBH4S8ZPsB88Xw\\\\\\\/+RQZk9SPxbiaJA\\\\\\\/ujF\\\\\\\/9+q2hbxB7PTN26nc27QKX4hWgnsOgGEhykUqnyxtbufUUTR6i8MARImU0BgV9hMy26wjhCTgWXpa224X0DRMq49uD26GtFhyUkeuv8873+1dVxR48y7Bp4TSIXkCY12NsaS6GlRi+SkDyoXdJL3e\\\\\\\/W0LSe9kQVUkDBFZ3IpappSwBSTowBEcAYEBev1k99aZ7E+ZqIs\\\\\\\/pvKdfGt1OknQoepg+whFlbIdtyb9F3Obs\\\\\\\/rsk0X7R7hTSKrjVL7RrHax3MQHJPCwVXKJT8wCYoMZVIJynQGDNadOjm6D+nE\\\\\\\/gsMK7M3FLQ0\\\\\\\/WpQbNxP2yvrAJoVybIGKhfP+xAzBt6jnY\\\\\\\/o+12PgrjindaBpEcfCtUO3ZAM\\\\\\\/D+0DJsxA4PX2ew80oWQGV0bJDcgsWCUG81cRIJ6FPh1ZPaiDQ5HxVytZvcslRNtXFi0uamjAUae1f\\\\\\\/dfemmj8HUwystxaLX3Ji8K5hk539TmAiKpu8+OOPvqZr3WE31OLYFqa941hFS2Fg3Sw6+nyjxlhhjFIPhw7Qqk6QOTilaQa\\\\\\\/gb6QiuUtsXJPKEZoa\\\\\\\/5yctr8C11MZaR2O0CwVj22ZFgEURVaUbX\\\\\\\/bDm5LyiSWj4xLLZWCdMaQ+j9Sl3hx\\\\\\\/7rtMLDYOQpLnktup\\\\\\\/c6q9Q68TyHhbEhVL\\\\\\\/5PygoypuPOinDqatWrzRjx302SWTC5qeWSyo538mcnucU4gdtBoi2uWmMtgUbH+\\\\\\\/z\\\\\\\/chzAZtbXRTlR0iUNyT8PiQH2RwIF55Xx+nlI8rHmefKSW6\\\\\\\/5GUPoHGWMCkE2MQRfzmKj7Yfsz0vOzWLkJXz2bSSas91qjE4a7kByxmagcYINyLUlDkWc\\\\\\\/VZXSd0WpNkLGUXg\\\\\\\/GEJWOrIDqfthMr9pKxBXr3BGgWZKMKUKEjOeJlAsgTXsLOm48urysz6YJ4nbs0qDN51NwoPfZsIPApk6Dq3Ml0TlQBwjNG8A+AcegIahN8yuWyC1c703MLV3m9+0K1KABrtUBtDqs0LLNUvHKbUU58qacoi+3QqaiprHr4bDj5PrZqDoHWqLSYB\\\\\\\/jpsDhd1SsKSYE5HuTdE+F8PntnCMxiq6\\\\\\\/jIQBkBT\\\\\\\/Byg90PVmgPTQ4fKUtmxbLzDhVqIZToyOsWYZmW1N\\\\\\\/CVYQ+zA14VIWgydXqy4iQCdo0VfpBnOfomq3Ke3yKndcVh7vYF1O\\\\\\\/e0EcFWsmmT\\\\\\\/mTjHr6Xec86r0e2qyjCsmPXWjsKXxBFpDwKWhH16EkAyVX6bYdrSupWk+l1VeUNcAjh0oukw+12377xYya72diAHPmKjNGQDnOqb0p4SuBH6lQhlxgfvDvLEy6d4r\\\\\\\/XXR2yR8VjiKoSi6lPvdS1+1cc\\\\\\\/SQC14sbHPukI3vVlJ\\\\\\\/OVlLdqNHPPWKIpvUaTQVMVg7Ag+E1QY4A\\\\\\\/sBg+9QXTEgRSX83peqJd7fyoEfHRVGDnJIxzbc06P\\\\\\\/EtU2ccf2rb9KT7pC3OBm8WlFO7netnKE0CX5FVa5PO7FHy3xQUMCVerN2q0RN64oJdNOQ\\\\\\\/xO22FNGe7lin78ZQlbEL6mGw86UxlUe27NPzMXmDrAkbShRTWA1t+I7YLv5Cm4vIDPeSwYaSMqUtkOkkUKxsfbh7MIK3iq4jjhMMvf70hCZA9rnEW7pFs1TE007ifemmRiFq31QT+U+YpCrYJNJ5WYtLk46qPXSQ6C\\\\\\\/Q1UgXrqXf301ztkIL9T+9Vh3xmG+TAdgWDzk1ToIddvw6KxxHVc2IwmXfoDHalj0uQGEI47X41bS1MTQI7ycuznP1D6pj2ctgcrGY8dWzRbySzU8EzgLPFaNBWDZg6N8c625DXdP2HVUyvyQH1v9nhw7U4nCmti55ajgyF1DDNDaQBxhBSseNqqNQI8XvStO1TW4WXEmB1qvel8rc2uvre8cxiSXMFYZ+x10utGpXkTvZFyX5iLuXumF\\\\\\\/BbxJfA+nmhDg\\\\\\\/NSq84BaIX7sKbpjUPWzQ0q8dz+\\\\\\\/ez7KoOB9Op1u8\\\\\\\/lZMaldpJcV2bzc7RaKMyIg8i4RXIFDDPrdHUWrUrobLK4vbGrdL8sl\\\\\\\/ZouQLQHsT6DiP4cO\\\\\\\/NLPG6nN\\\\\\\/tXK906i6UFur6dK8\\\\\\\/BQYxinzMf\\\\\\\/AlU8bQWQwYGVXTb3vZvKAeEA8YkuLFSG7gxtoklvTNXulYhhfEMdI\\\\\\\/kO9MSv7qMFSIvBCh1SGCJcn3s2h5sEzatuU9pJHU9HcusvlauEVRZBdVlIIOWPL+dXb48CRN+W0C5fI4+zBzHdkJjwpp7cfA\\\\\\\/2CZG+MwLJjonm\\\\\\\/1G2uZTqMeXrywV6JQLR1dcLqyfyWVka3dDwTLSurteXKO6jX0j9lF59LgwDiQD4WWbsB0ZPFo\\\\\\\/a86EgjzEWZ2fM24Ox+ipkjweGkjfie5xkTmS2dc9oAGCtu8WEOqa5l54UOW1R5PyRgFIPVkRbl\\\\\\\/0OEvIQxFy608nZsZDcWd3dNrYtNPIQwnImBMW3umSOLTPvI3YbHtyUuRIemtvdwU8FZy0Zm3XqcMI\\\\\\\/Dek\\\\\\\/kiTPbHK59XuhhHRg+NDPZoFPIyC3Al3PiogpErh6692EpGMY\\\\\\\/PmZyjdwDeiZvsW8ldX9CQZwHNt2mDIWYycMHQ7ATqmBhRloTNyvJjRGYAY0V5KzqZLzzT2EIeGbn7CIusmEdBzbWyJ+7Fh0N6lOH\\\\\\\/kxcDfePL\\\\\\\/GxLkMv6uFLgkmBoTifpcAV4r8vbKCEnt0GNm2HECISO4LLm3ZwuM7cCErzaAlIUsrDOyF10Ge6f8PuguxJz0eTDearMgNdCd7K7gQbjusUnpZHTUhLQ2rrYUgVVJaf8HjSCJYV7Y4JKX2PyfxjlEGogbo77VCpVvPUSd5\\\\\\\/ffppteqF68GYFNW3u5\\\\\\\/dH2ebnbaXIiDwiTre1WuWSG0IgwGKCRt1mLJb\\\\\\\/Hn3sp\\\\\\\/Ue1po4ktqP5EdqaOPhExjhDX4oQbHQs1wDz4826AinZLzy66LrO22xIBqXMOURYVgmNoqHBi4ZtPnLAyvwsRsyP6U9B5gw\\\\\\\/cVRIAuVUOUfEbBMbWxybyruX1ONPPgR3pf2pGZxOG8E7mhsyH8uz3sPyAZv5iuS4ac\\\\\\\/CWWJO\\\\\\\/0oT8sL16rfKk0Rz9O9eDQ9AXebTbWVTJ6HnlcFzA5T8QbLbpkvhziC9wd8Pux\\\\\\\/73c\\\\\\\/kzFpoti7QLdJeIg9E2nxOwtue4PR0FaKEWQK5pRyL1AavRQ6cK4gduQDfTrBXADtg+QA0ZYaenLGdgIbpFTt8iKQCMDeUuPgP+g5RJ+cUDSjx3Oz7PtJayIVuMl7pq3kwYs8M+i9CLbWJanUapM6ud\\\\\\\/8aByQHczepGqODRGWHdJ7xMAknb5Jx4bbRefdTPIkxgmmVaQTChNHbv875nQ+pkFBuQ05\\\\\\\/zmI2apKhqYutI3Ce0P97T0\\\\\\\/9oTzG1cnbb7\\\\\\\/w0SZan4GyIK1E9rkrUvdOYwc+v0rEuk7AEcRs7doYctZQ40SJkiXoc+Q0mP7XfcSQl8glvPJcmiFnWjSrTL+9TZ+gDITgBzqYml+xgzATAJ4iZ8ZgRPT+ef3l0tTu0Q5wHjBizcwOlGr4jnUYgv4+zbPWSY8rYGqS0dXiupz3ZAigolnNgnxHdJxBsjbb6mweQo8qzm9FItnxUx1lu1e2c6NrKPkz+ZyxoXM01XHrH+be8o8MmZXmsSAMBgRggnobAqgbyy6\\\\\\\/8X35WXPa1OvnK+fq5jW13b\\\\\\\/d9xE9Bh7U2MwsL3\\\\\\\/YXgfr\\\\\\\/gnBKTnzT2Cz02pkFPKp4Wye1FiOy+Gs63W3SOQ+mEMyHfXfSNsDNm5yjXBS2JdQCtVPRLp69ZSUr7sNJD5sNEuEdb\\\\\\\/viZY4ne34\\\\\\\/vGjfbVb+E5RvSunoRVdQ15aW0VmS4EDKsqOBMkmWX6GAjVlxJRZ5k+C9wmQEJo3xrwSoAiydcEynk9JjYoNnHCK+Z77\\\\\\\/9VEn1IsAxrLiOjvdSuet3iixbKyjalXpXCbVqS16HvHsYhDq6JxoqGYj+8y1TxpoLXqzAtvaKR\\\\\\\/ziEiwPy+B+hd\\\\\\\/Bc7sxuKrW01B1uXUbtOIJTetsjE06AiP1FkJAtzKglerWGsaxP6ZUDRZtjXmdMFWBBYb\\\\\\\/s1xV48U+EGiZeoALspyacvwuw6yOqui5qGOF3QqeiZneXl42PJgltOzXR\\\\\\\/lgk4mg5fcadvrzV\\\\\\\/JlKmcVQePNU20HTEtgbH17tQxkEaaLlypxdDgd\\\\\\\/a4lBS1RhDECKxc842x8VVrV8yqPq9pTjnMLjIQMGsjJbBfib+ygDvDmEd4pQjQy2DR3EGtcCobKirVc\\\\\\\/LT4SHuh8l\\\\\\\/zVnjvk3srYy1FvQsKmQX3ay6F6136e0Ma0o4+U9aZQ6TwcRPkLmMQobACzjY\\\\\\\/ieVE1\\\\\\\/vuS83b0pF2IyoNBppuGcOECdPD9ddVsTCZVyWHakpHyn5sDiuOg2lVChc3AlkZiy0Dk09MFxGU6YltfFG43k9c+0Tlg9PXS85V2g5+iuMmHca6bR0M2Dlh\\\\\\\/uScrqBo5g2Ef93WMBO9jCnpt\\\\\\\/nsKceAx6V1GeVeTHBzZG5e4olFzc2o1D+bBii7AlZr\\\\\\\/g1ivPB+kjERFdYXHAzqi+8zPYgI8ldrigE7fEemKI3am1dORlCjYXTamJn3MLl4uq34d55nsZosjF2QZOsBVasjz4Q5mbZ+xNbxOPR6YRZGEJbmkd6TmAr5ekEoUVrnK\\\\\\\/432HC5WXzxEs2n+fNg793aEVb3dk5TZirB2m92JaDHRPS6lF8vL\\\\\\\/w09hsckFS6M51TxZZj6xCCzTTZnlBwDzxti5LHkkH\\\\\\\/fI7rRd7jBy4n44LkpcNBpM4zjfVLF5NxOyQ5rQ2ynzFByc8gPVcAZQmwd65\\\\\\\/RbGCpOOHLc3nJAdkG7d5fwXA8Qa+fUVXN\\\\\\\/KtVk5PkQ3IiwhtxuoEg8xBhT5Q9JaRweXWk4bxTtWa32DTlRg+f9hOTGW1x9yGXKigFqo9S279OqWa1co9brJrzKX0q2BU4Mtn7+VSCjR2WCuRjfx93WgYRWuWEZckTwpLxrlpRnKm2WHwUnmXZIn4d8OX0+gwgG3bthFIhOn+uNvf11UCCIDlXusmchqOkPnjixGr0m5iu7oBh3riMKlgaBGIq39oiEVLOomw5LWuBhGBPOy5q0g+VIqPRFcJv\\\\\\\/0wgUnCvQzzaEKOplzNZkWliiZMDJCVIMjnBrt1soEF6ER5jO6\\\\\\\/WDwTBHczZc8af6SA8ldTGNEhWvv1YGSbbZa6Wvs2CiXsxgW0YIpO7kbSCA21KGJiqF47D\\\\\\\/NWzLf82kL9SMVOxgbh1yxI3K5jp+UNJos44z6\\\\\\\/HZkJQJXZw9BekhvkoWIEDJXmMZzWS7r6j9ScAw1ihB89kuChPi5\\\\\\\/BltAabmUWmQpwMMsa6nLRdi2s0pgM66o+atGv4O9NqRpbXS+VqB6QXHN1C9cjsWeJZpv4Cklg0nWCey9SIGk\\\\\\\/wxc\\\\\\\/kya9qTb42XBgzcEmFzPnetT6qWNHEiur+2ebyVqlbY9j+1Xcr8gEHdeAP5bydptj33rkWAVKQgaWqm75qCDM\\\\\\\/JR3TeXEPobmBSzuBcPSDqBpiSS4rrOqtZSXjBfYnLgKXnyLr53nVSVciP2pOLVODMeXSe1d3yhIbkSGON69wEPTVhYAJg2UP0YQ63bYFfgdNpu2Rz3z96kc0Aj802QGBElVXYZnFK\\\\\\\/ybTrukvTCfAcPv5nzwx2Uef9EJuQg5DzV0QAxqJlBdduL13EI8GJfLNPejS9Smr3q9u7KHmrIhJuzWbakDRrW0y1y6C6xHepAcNj2JXk1+ML9m8KssR5VVsY0Urt0Y\\\\\\\/6wMhmISlgB4R5KyOnyS8pFVlVVo\\\\\\\/YfTLU35+Y7oYhsr9y35EghUHSoZG3m\\\\\\\/P4QSiqflK8RvZvKeNpsaCQUVtcScbhO3bOdiDcIkML4CBXs9OkuEIA1QktBrrx49HVg\\\\\\\/y51t1Q9TyOyPg0K71ZQaC0KGmLj6gC9HuZyAYFzqfWNqCCePx+F8nzeIXE\\\\\\\/zwvW7\\\\\\\/LHu1KdrqmlHvOG5wowBs\\\\\\\/tUGm4CB1cjCojGSqcScS4Ecn\\\\\\\/IlTq7OOKFKJRCrLuUKNzpHh\\\\\\\/nPWPZG2hFw6heZm5teoremaYxua67EoO2Loy30mn4d9yXEdVIUF9Rvkx86pTKvouptuAqQWI7PpnF3as555sgaOEPr0wqDA5M20FB7OSbl3zfzmRt\\\\\\\/BedzL4pw43xLlFR+OurBQrGX3bgfhtAWiNwAEg7Kiy5JG6AGSLKM1ARsFiyXtBcWFX\\\\\\\/a+WAkeMLW1LqkLf91OEs3ed3ZcplXTDKcQhNy3ZFqfDVdDOg4TPYb\\\\\\\/qjaUA3MO3SEFwemaDFtb57ywX2dlkol1aztpcAxf4+Jym4IIy11n0I74dSkyNeTpRikwhWNAjjvfdOzJ\\\\\\\/WxYWM2I0Idx05xaj6S2zQ\\\\\\\/GT5QlRFMPWqyphriR8rx7LEgtw8\\\\\\\/GRhW4onu3fn7i\\\\\\\/n4Yexeh2jpD3mxyIb82PGT3BqJ\\\\\\\/xrM77vZ09HlfDmihHD6zNFmmNnZLnG+mdRMhWx9SH+\\\\\\\/nixRK9m0yNbxJFmFhKGYJMeBIZWsnRAF3CrDx6JtAkYORJGEgA6Tahmk3go14RIYxw8qBcDgmSqB1Fnn54FdnAOFw5GJBwU9H\\\\\\\/T7LvTm\\\\\\\/stG\\\\\\\/ATM6CV5Q0k9ax2+tLbJY7VmYKwWbv0cnFt5Ub\\\\\\\/bvshTwktmng6cMRfuYMh0NdeKaT0RnriBB1oxmfSHjpLvenSB2kJJUV0TZdGQJxq2l0+wF+U0u+FxSi\\\\\\\/zLKZTluiTjY7lxnTHWGa\\\\\\\/MxkIUMohie8OkI2fu7cYzypFQ5LymMdyDgM5jpjTLRmK6eMQ2SKJ4sNj+06my+sTV5NlJe02yY7mAMookskuNPFcnbDEL0e975Fb\\\\\\\/wrJl1Z\\\\\\\/qeDjO0ARGWd8FrJzpaq9fFtiFcqMFSBzy2RAFEIr0QnWxMRG576P\\\\\\\/75ywSLEKfTQwHrZAcV0ZQ2qICnJRs58OryzbdU+53RXeTxbpQ7uoPfr8ZLISp5rsg+1wsMawHVYL5Na4DNVjGBUXHHtydTOR8TYJdLSntTfxaI68eTrCgkg9sjAi42w0UWffAtjqZ4NM6+OJCl8YkzlInv4KYXXniMsMfKPtJZKK4z3sm26ELFmTB4UD2nLFUjWXm1QoatLfX\\\\\\\/96P0toehNTTZA1h3KK2nWhZ4L9o4Zg4cj\\\\\\\/psN+kcwjTjyPTCybKA7JWSlSEPMO16HM5o2mnMhwwh3wyuf6Ui1hu6q9ZqQRGCkXhxIUT\\\\\\\/Z2H9nm4QdfQqvWX01CjwOPRnouoONS6uJA9zVzI2LqbtceFzjBW43ivRXVE7QyMYQMdZjKHbcxmI0BJ4iXEkQS497BDmcqsRwqImRfUOY+mIYB+\\\\\\\/REwOF6wPU\\\\\\\/iRn0bNlYHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sVqgrLo3OXeEfQz\\\\\\\/Gi5WnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222fb0f286f65bff74bc65dcd3cd6df3a08d214c6c370273e2b218188d7bef7661\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T\\\\\\\/POET9kEeClLj5uVS0QrtPlpNXvVHRZDraRAo488+kF9xNjM+Jfv2QQlJ\\\\\\\/ulqC8SmdVbvGHOH62RFF3uIAYe6K9F8gy962UUHfSlddhQdZxHjm33vp\\\\\\\/C51oC\\\\\\\/gGVr4lojs0bsE4n3qdKeX0YwODdgb5bJzQfi1p\\\\\\\/W+rFeegItMg1KV9rqwaeapCKMIK0OJGEDZhNkcDXbCEdDwOsgkWfQ2kqx9w4ivGsW\\\\\\\/OCA3EettbTaDkzgVdpYMK1IonzuSTPIHh9OjAfDD9tR09K5VZb7EF+htBNAKFKMf8Y2V7pYx41WeHL0uRvAI7ryZ8\\\\\\\/axfvsNOGDqT2IlmDS7dlM9U27hBcI6hIoqxpSMN3ffvpjsN\\\\\\\/fXJCYiuYMPVgOuMGcNG7u+52n2zfxYjDMyyrOIvMnbSXon7SCYtBm7GZRtuYJUDrei1A5dSlP1+17zBF3+0y1u4tPjKI40Ryjv93iDctvWGt6TAEoIFvsFRBKHubSaNLKAv\\\\\\\/7NNuHMHtccdvuxHp2WQ1mylmKWqorunudzQDsthV\\\\\\\/T1bUcJUXV38948FCxdTektHFfC3XqonhUiZXgSUXtXz7\\\\\\\/UpJHTL0YRJ3+zi0jZw1H+sJk5wb4ulPb3j0soMHOQ6VArFouqG8N7aKDrfZG9BxDPJs\\\\\\\/7B3O3t3mDki3sFNTzJ7MKpLYddkRPJPoyosJ1VBy2PaxjvMhG+HuxLKUlvF3tCkCdFJbm1FY6+H54YO\\\\\\\/6uelgipS7nyHObar6PopEPCwj3sF4nvnBfreOi5XqaB8BNNIIoWKWYcoFZ22ZVAaUyPC11vpBHyPdtDtaeArimaI8Wy3k\\\\\\\/+XAtDB1erF9oVHRQ6ACsQpOcq\\\\\\\/bBSHHXRdq0AnhABBhNmnWFmfswD34seXc\\\\\\\/W38XyxSwYR+LrfQMxfcj\\\\\\\/c\\\\\\\/Vyqisk\\\\\\\/k5Pi2Vg1iHI29Nx2yg17R2CCECkLLX3crqyQv60+1afi2WWo\\\\\\\/qsz7dBAV9gCTJ1vvNaihLRLFI5LF4oc2bGkPcG\\\\\\\/fr9DMmEtQS588voF9ml3RAcMyfpJXWPUQlUXFhZ1n1g+BtP\\\\\\\/1hoIo3alGoq5JEDZZsRrGyvwp1rhvMUF+QORfGhriZpEpenkyL3RK\\\\\\\/XIdPHFc7VVZzHSKxBufSb5xpQq5XOad2P9SDMWmurf1KgcYfk652f0gDXpMZqI+Fa6nYKrr5wMoniKR58JA5nXqGVJCarcskeSljo4vHiApzHMh4xOEtLNkgId0IMF8tljMTyN37esy9X\\\\\\\/50n0cXFiMwUQKldqs+IKRRZJV44GKYDZpnropgRNZ7mUY6H225g6DF3vYQGLsU7hZYmTWZSv4MNXmW7nhuhpWeh2Aatz3Vw4I2whtt8Q9Fbgx92WpykNiK4XyPVjjxMC2mBwjoQcUUTKs0VHxEPuaQJOQAM3yFYKRETLUizUk8hDUUmJua7hMIzj5wx7f3gFk1CCMjZUIDio7ik6jkw3Rvc+nkMqiG6Tua8GoAvkwAMnVXSgGqSZdWYZv+UW8gkCMxLAwYSH1ejH1w8lCep0MI\\\\\\\/CaQqnXIxIdU0WA64GexTuV4JPiKW\\\\\\\/vHuOsXJIyWrOqVU4xFd1PSFCaRxiB1bgkBNdc5YV1WEmxFBvVJkCMl1s6uG9KbmCBFsUKna0KgRrKV5OkRXuEOVX90wZwFwRUEbdegIuGsglvcEGTBysH9ZsCRAzV9zNEyzVsiqUFw0XxHZXyx2zyyFWDXd+wIUvqfrbeBJoV2Suo2WzE6GpPe4AudDLTlOFAwNal1Ew2R1qpRAd+oyT144EF3aDTq5hjkSSm1ZSL5sbT8nhSZzWON9zeglCzcpnZjGYMGeNlM6G\\\\\\\/qZ0LRELonqe75z2yjc\\\\\\\/2zqwDsAcaMH\\\\\\\/sY4r2ou+MPcvkUSDAKcgto9SXkVf7EPWWfxeUk6CzwFC3M0\\\\\\\/2d2Mn+rM1lC0ZDWyg5euSmqQKKEZ0vLRomQonvXL7IaTeobR0O2936ubO3KzWD441onnOX6GI5kGOVa\\\\\\\/\\\\\\\/uatrVjNZH9yhSeSAk9umOQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022obJh1PzpL0hiJ0Rkhai\\\\\\\/Uw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcff220918dfb9053227cb0c1ce62dd604c2a05ad593aa676bdd76c6e5248262\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491624 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3757" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CcDeNG_v6M0tuo4N73Io9szYZkuIMFNbxC5Dy-iH90w" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3757" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LGcEuEf0m0gW76OiHsVLK3Dbue2t9LcfZP6sUwnaIJk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3745" title="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3767" title="QTL Mapping on a Background of Variance Heterogeneity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491624" id="node491624" data-pisa="ggg;8/12/3757" data-pisa-master="ggg;g3.118.200642" data-apath="/ggg/8/12/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">vqtl: An R Package for Mean-Variance QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3766;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200642 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491624" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3757" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491624" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491624/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3757.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491624" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491624/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3757.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491624" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491624/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3757.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491624" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491624/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We present vqtl, an R package for mean-variance QTL mapping. This QTL mapping approach tests for genetic loci that influence the mean of the phenotype, termed mean QTL, the variance of the phenotype, termed variance QTL, or some combination of the two, termed mean-variance QTL. It is unique in its ability to correct for variance heterogeneity arising not only from the QTL itself but also from nuisance factors, such as sex, batch, or housing. This package provides functions to conduct genome scans, run permutations to assess the statistical significance, and make informative plots to communicate results. Because it is inter-operable with the popular qtl package and uses many of the same data structures and input patterns, it will be straightforward for geneticists to analyze future experiments with vqtl as well as re-analyze past experiments, possibly discovering new QTL.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/heteroscedastic-regression-mvqtl" class="hw-term hw-article-keyword hw-article-keyword-heteroscedastic-regression-mvqtl" rel="nofollow">heteroscedastic regression, mvQTL</a></li><li class="kwd"><a href="/keyword/variance-heterogeneity" class="hw-term hw-article-keyword hw-article-keyword-variance-heterogeneity" rel="nofollow">variance heterogeneity</a></li><li class="kwd"><a href="/keyword/dglm" class="hw-term hw-article-keyword hw-article-keyword-dglm" rel="nofollow">DGLM</a></li></ul><p id="p-3">Traditional quantitative trait locus (QTL) analyses have focused on discovering “mean QTL” (mQTL), regions of the genome where allelic variation drives heterogeneity of phenotype mean, while assuming that the residual variance, that is, the intrinsic stability or noisiness of the phenotype, is identical for every individual in the mapping population. It has long been recognized, however, that the residual variance is itself heritable (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Falconer 1965</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lynch and Walsh 1998</a>), a phenomenon that has been described theoretically (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hill and Zhang 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hill and Mulder 2010</a>), demonstrated in inbred model organisms (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> 2015</a>) and crops (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2012</a>), and exploited in livestock improvement efforts (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mulder <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ibáñez-Escriche <em>et al.</em> 2008</a>). Correspondingly, several groups have proposed statistical methods for mapping QTL controlling the extent of this residual variance, these sometimes termed “variance QTL” (vQTL) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Paré <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rönnegård and Valdar 2011</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cao <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Soave and Sun 2017</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dumitrascu <em>et al.</em> 2018</a>). Nonetheless, although detection of vQTL has started to enter the mainstream of genetic analysis (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hulse and Cai 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ayroles <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Forsberg <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wei <em>et al.</em> 2016</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang and Payseur 2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wei <em>et al.</em> 2017</a>), statistical tools for this purpose remain heterogeneous.</p><p id="p-4">We have developed a standardized method for QTL mapping in experimental crosses, in particular F2 intercrosses and backcrosses, that simultaneously models mean and variance effects in order to detect mQTL, vQTL and a generalization of the two that we term “mvQTL”. Our approach, which we term “mean-variance QTL mapping”, is based on a double generalized linear model (DGLM) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Smyth 1989</a>), following the proposed use in this context by <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Rönnegård and Valdar (2011)</a>. In the first of two companion articles, we characterize the method and competitors in the setting where variance heterogeneity is driven by a background factor, such as sex, batch or housing, and show that modeling these (external) variance effects improves power to detect mQTL, vQTL and mvQTL (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Corty and Valdar 2018</a>). In the second companion paper, we demonstrate the approach on two existing datasets and discover new mQTL and vQTL (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Corty <em>et al.</em> 2018</a>).</p><p id="p-5">Here, we provide a practical guide to the approach using its associated R package vqtl, which is currently suitable for F2 intercrosses and backcrosses, and is inter-operable with the well-established mean QTL-oriented package for this purpose, r/qtl (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>). First, to generate illustrative data, we simulate an F2 intercross and four phenotypes: one phenotype determined entirely by random noise, and one with each of the three kinds of QTL. On each phenotype we then conduct a genome scan using standard approximations to interval mapping (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lander and Botstein 1989</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Haley and Knott 1992</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Martínez and Curnow 1992</a>), and mean-variance QTL mapping, which includes a test for mQTL, a test for vQTL, and a test for mvQTL. The association statistics of all four tests are initially plotted in LOD score units, with drawbacks of this plotting unit discussed; then permutation scans are used to determine empirically-adjusted <em>p</em>-values, and plotting in these units is shown to to make the results of the four tests more easily comparable. Plots are then described that communicate effects that led to the QTL’s detection, and the bootstrap is used to estimate its confidence interval. Last, we benchmark performance, using one of the datasets examined in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Corty <em>et al.</em> (2018)</a> to report how computation time varies with marker density and number of permutations.</p><div class="section other1" id="sec-1"><h2 class="">Example data: Simulated F2 Intercross</h2><p id="p-6">To illustrate the use of the vqtl package, we first simulated an example F2 intercross using the R package qtl (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>), on which vqtl is based. This cross consisted of 200 male and 200 female F2 offspring, with 3 chromosomes of length 100 cM, each tagged by 11 equally-spaced markers and estimated genotype probabilities at 2cM intervals with qtl’s hidden Markov model. We then generated four phenotypes:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-7">phenotype1 consists only of random noise and will serve as an example of negative results for all tests.</p></li><li id="list-item-2"><p id="p-8">phenotype2 has an mQTL that explains 4% of phenotype variance at the center of chromosome one.</p></li><li id="list-item-3"><p id="p-9">phenotype3 has a vQTL at the center of chromosome two.</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-10">This vQTL acts additively on the log standard deviation scale, and results in residual standard deviation of [0.8, 1, 1.25] for the three genotype groups.</p></li></ul></li><li id="list-item-5"><p id="p-11">phenotype4 has an mvQTL at the center of chromosome three. This mvQTL has a mean effect that explains 2.7% of phenotype variance and a variance effect that acts additively on the standard deviation scale, resulting in residual standard deviation of [0.85, 1, 1.17] for the three genotype groups.</p></li></ol><p id="p-12">We additionally consider phenotype1x through phenotype4x, which have the same type of genetic effects as phenotype1 through phenotype4, but have the additional feature that females have greater residual variance than males. All the same analyses and plots that are shown for phenotype1 through phenotype4 are shown for phenotype1x through phenotype4x in the appendix.</p></div><div class="section other2" id="sec-2"><h2 class="">Scan the Genome</h2><p>The central function for genetic mapping in package qtl is <kbd>scanone</kbd> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>). Analogously, the central function for mean-variance QTL mapping in package vqtl is <kbd>scanonevar</kbd> building on an early version of <kbd>scanonevar</kbd> in package r/qtl. It takes three required inputs:</p><ol class="list-ord " id="list-3"><li id="list-item-6"><p id="p-14"><kbd>cross</kbd> is an object that contains the genetic and phenotypic information from an experimental cross, as defined in package r/qtl.</p></li><li id="list-item-7"><p id="p-15"><kbd>mean.formula</kbd> is a two-sided formula, specifying the phenotype to be mapped, the covariates to be corrected for, and the QTL terms to be fitted, with keywords <kbd>mean.QTL.add</kbd> and <kbd>mean.QTL.dom</kbd></p></li><li id="list-item-8"><p id="p-16"><kbd>var.formula</kbd> is a one-sided formula, specifying the variance covariates to be corrected for as well as the QTL terms to be fitted, using keywords <kbd>var.QTL.add</kbd> and <kbd>var.QTL.dom</kbd>.</p></li></ol><p id="p-17">For example, to scan a phenotype named <kbd>p1</kbd> we run:</p><p id="p-18"><kbd>scanonevar(</kbd></p><p id="p-19"><kbd>cross = test_cross,</kbd></p><p id="p-20"><kbd>mean.formula = p1 ∼ sex + mean.QTL.add + mean.QTL.dom,</kbd></p><p id="p-21"><kbd>var.formula = ∼ sex + var.QTL.add + var.QTL.dom</kbd></p><p id="p-22"><kbd>)</kbd></p><p id="p-23">At each locus in turn, this function tests for the presence of an mQTL, a vQTL, and an mvQTL. The basis of these tests is a comparison between the fit of an alternative model of the form<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-2.gif"/></noscript></span></span>with a null model that omits specific terms: for the mQTL test, the null model omits locus effects on phenotype mean; for the vQTL test, the null omits the locus effects on phenotype variance; and for the mvQTL test, the null omits locus effects on both mean and variance. (Note that the mQTL test in mean-variance QTL mapping is different from the traditional test: the traditional test does not have variance predictors of any kind in either null or alternative models.)</p><div id="sec-3" class="subsection"><h3>LOD scores and nominal p-values</h3><p id="p-24">Each type of test (mQTL, vQTL, and mvQTL) yields two association statistics: the LOD score, and the (nominal) <em>p</em>-value. The LOD is a raw measure of association equal to the base 10 logarithm of the likelihood ratio (LR) between the fitted alternative and null models. Higher values indicate greater association when considered across loci for the same type of test; but LOD scores between different types of tests, namely between mvQTL test <em>vs.</em> either mQTL or vQTL tests, are not readily comparable. The <em>p</em>-value, which is comparable between different types of tests, transforms the LOD score to take account of the number of parameters being fit: it is calculated from the asymptotic distribution of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-3.gif"/></noscript></span></span> under the null model, namely the <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-4.gif"/></noscript></span></span> distribution with degrees of freedom equal to the difference in the number of parameters between the alternative and null models.</p><p id="p-25">The <em>p</em>-values described above, however, are nominal: they do not take into account multiple testing across the genome. They also rely on asymptotic theory that assumes the underlying phenotype being residually normal; this may not always be the case and when violated will lead to inflated significance. More robust <em>p</em>-values that are corrected for genomewide significance via control of the family-wise error rate (FWER) can be obtained empirically, through a permutation procedure described below.</p></div><div id="sec-4" class="subsection"><h3>Robust, genomewide-adjusted p-values</h3><p id="p-26">To calculate the empirical, FWER-controlled <em>p</em>-value of each test at each locus we advocate use of a permutation procedure (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Corty and Valdar 2018</a>). Like previous work on permutation-based thresholds for genetic mapping (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Churchill and Doerge 1994</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carlborg and Andersson 2002</a>), this procedure sidesteps the need to explicitly estimate the effective number of tests.</p><p id="p-27">In brief, this approach involves conducting many genomes scans on pseudo-null data generated through permutation to maintain as much of the character of the data as possible, while breaking the tested phenotype-genotype association. Specifically, the design matrix of the QTL is permuted in the mean portion of the mQTL alternative model, the variance portion of the vQTL alternative model, and in both portions of the mvQTL alternative model.</p><p id="p-28">For each test (mQTL, vQTL, and mvQTL), the highest observed test statistic is extracted from each permutation scan and the collection of statistics that results is used to fit a generalized extreme value (GEV) density (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stephenson 2002</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dudbridge and Koeleman 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Valdar <em>et al.</em> 2006</a>). The observed LOD scores from the genome scan are then transformed by the cumulative distribution function of the extreme value density to estimate the FWER-controlling <em>p</em>-values. This approach is implemented in the function, <kbd>scanonevar.perm</kbd>, which requires two inputs:</p><ol class="list-ord " id="list-4"><li id="list-item-9"><p id="p-29"><kbd>sov</kbd> is the <kbd>scanonevar</kbd> object, the statistical significance of which will be assessed through permutation.</p></li><li id="list-item-10"><p id="p-30"><kbd>n.perms</kbd> is the number of permutations to conduct.</p></li></ol><p id="p-31">The object returned by <kbd>scanonevar.perm</kbd> is a scanonevar object with two additional pieces of information: an empirical <em>p</em>-value for each test at each locus and the per-permutation maxima that were used to calculate those <em>p</em>-values. These FWER-corrected <em>p</em>-values are straightforwardly interpretable: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3757/embed/mml-math-5.gif"/></noscript></span></span> for a specific test at a specific locus implies that in 5% of similar experiments where there is no true genotype-phenotype association, we would expect to observe some locus with this much or more evidence of association in this test.</p><p id="p-32">Accurate estimation of the FWER-controlled <em>p</em>-values requires many permutation scans: traditionally recommended is 1,000 (<em>e.g.</em>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Churchill and Doerge 1994</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Carlborg and Andersson 2002</a>), although the efficiency gain of using the GEV rather than raw quantiles means that fewer may be adequate in practice (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Valdar <em>et al.</em> 2006</a>). These permutation scans can be run on multiple processors by specifying the optional <kbd>n.cores</kbd> argument, which defaults to the total number of cores on the computer minus 2. On an Intel Core i5, running 100 permutations on this dataset takes about five minutes. When many phenotypes are studied, or if faster runtimes are needed, these permutation scans can be broken into groups with different values for <kbd>random.seed</kbd>, run on separate computers, and combined with the <kbd>c</kbd> function. This function combines the permutations from all the inputted scans, re-estimates the extreme value density, re-evaluates the observed LOD scores in the context of new extreme value density, and returns a new scanonevar object with more precisely estimated empirical <em>p</em>-values.</p></div><div id="sec-5" class="subsection"><h3>Reporting and plotting genome scans</h3><p id="p-33">The results of <kbd>scanonevar</kbd> can be plotted by calling <kbd>plot</kbd> on the <kbd>scanonevar</kbd> output object. This produces a publication-quality figure that shows the association of the phenotype for each location in the genome as different colors for type of test, with <em>y</em>-axis scale being specified by the user, via option <kbd>plotting.units</kbd> as the LOD (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), nominal <em>p</em>-value, or, provided permutations have been run, empirical, FWER-controlling <em>p</em>-value (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Of the available <em>y</em>-axis scales, we recommend using the FWER-controlled <em>p</em>-values since this scale puts all tests on a level-footing (unlike the LOD), and allows direct identification of genomewide significance and thereby relevance (unlike the nominal <em>p</em>-value).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For each of the four simulated phenotypes, the genome scan shows the LOD score of each test &#x2014; mQTL, vQTL, and mvQTL &#x2014; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;For each of the four simulated phenotypes, the genome scan shows the LOD score of each test &#x2014; mQTL, vQTL, and mvQTL &#x2014; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F1.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3757/F1.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491545" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">For each of the four simulated phenotypes, the genome scan shows the LOD score of each test — mQTL, vQTL, and mvQTL — in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For each of the four simulated phenotypes, the genome scan shows the -log10 of the FWER-corrected p-value for each test &#x2014; mQTL, vQTL, and mvQTL &#x2014; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test. A value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;For each of the four simulated phenotypes, the genome scan shows the -log10 of the FWER-corrected &lt;em&gt;p&lt;/em&gt;-value for each test &#x2014; mQTL, vQTL, and mvQTL &#x2014; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test. A value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F2.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3757/F2.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491647" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">For each of the four simulated phenotypes, the genome scan shows the -log10 of the FWER-corrected <em>p</em>-value for each test — mQTL, vQTL, and mvQTL — in blue, red, and black, respectively. The traditional test is in green and globally similar to the mQTL test. A value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Calling <kbd>summary</kbd> on the output of <kbd>scanonevar</kbd> produces a summary of how the scan was conducted and what the results were.</p></div></div><div class="section results" id="sec-6"><h2 class="">Communicate Significant Findings</h2><p id="p-37">Having identified interesting QTL, we want to visualize the their estimated genetic and covariate effects. Because the <kbd>vqtl</kbd> package models effects for both mean and variance, existing plotting utilities are not able to display the entirety of the modeling results. To understand and communicate the results of a <kbd>vqtl</kbd> scan at one particular locus, we developed the <kbd>mean_var_plot</kbd>. This plot illustrates how the mean sub-model and variance sub-model of the DGLM fit the data at a given locus.</p><p id="p-38">In each <kbd>mean_var_plot</kbd> in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, the location of the dot shows the estimated mean and standard deviation of each genotype group, with the mean indicated by the horizontal position and the standard deviation indicated by the vertical position. The horizontal lines extending to the left and right from each dot show the standard error of the mean estimate, and the vertical lines extending up and down from each dot show the standard error of the standard deviation estimate. There are two types of grouping factors considered by the function <kbd>mean_var_plot_model_based</kbd>: (1) <kbd>focal.groups</kbd> are groups that are modeled and the prediction for each group is plotted. For example, a genetic marker is the <kbd>focal.group</kbd> in each plot in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>; D1M1 in the top left, D1M6 in the top right, D2M6 in the bottom left, and D3M6 in the bottom right. (2) <kbd>nuisance.groups</kbd> are groups that are modeled, but then averaged over before plotting. When there are many grouping factors thought to play a role in determining the mean and variance of an individual’s phenotype, such as sex, treatment, and batch, we recommend putting just one or two in <kbd>focal.groups</kbd> and the others in nuisance.groups for clarity, cycling through which are displayed to gain a thorough understanding of the factors that influence the phenotype.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="mean_var_plot's show the estimated genotype effects at a locus with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the p-value was not statistically significant at any locus. For phenotype2, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL. For phenotype3, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL. For phenotype4, the pattern of two-dimensional separation illustrates an mvQTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;kbd&gt;mean_var_plot&lt;/kbd&gt;'s show the estimated genotype effects at a locus with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the &lt;em&gt;p&lt;/em&gt;-value was not statistically significant at any locus. For phenotype2, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL. For phenotype3, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL. For phenotype4, the pattern of two-dimensional separation illustrates an mvQTL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F3.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3757/F3.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491553" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child"><kbd>mean_var_plot</kbd>'s show the estimated genotype effects at a locus with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the <em>p</em>-value was not statistically significant at any locus. For phenotype2, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL. For phenotype3, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL. For phenotype4, the pattern of two-dimensional separation illustrates an mvQTL.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Additional plotting utilities, <kbd>phenotype_plot</kbd>, <kbd>effects_plot</kbd> <kbd>mean_var_plot_model_free</kbd> are described in the online documentation, available on CRAN.</p></div><div class="section other3" id="sec-7"><h2 class="">Establish a Confidence Interval for the QTL</h2><p id="p-41">Last, to assess the genetic precision of a discovered QTL for bioinformatic follow-up, the function <kbd>scanonevar.boot</kbd> estimates confidence intervals via the non-parametric bootstrap (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Visscher <em>et al.</em> 1996</a>). This function takes, as arguments, a <kbd>scanonevar</kbd> object, the type of QTL detected, the name of the chromosome containing the QTL, and <kbd>num.resamples</kbd>, the number of bootstrap resamplings desired. As with <kbd>scanonevar.perm</kbd>, the <kbd>n.cores</kbd> argument can be used to spread the bootstraps over many computational cores and defaults to the number of cores available minus two, and bootstraps can be run on separate computers and combined with <kbd>c</kbd> to increase the precision of the estimate of the confidence interval.</p><p id="p-42">We recommend 1000 resamples to establish 80% and 90% confidence intervals. With the datasets simulated here, it takes 20 min to run 1000 bootstrap resamples on an Intel core i5.</p></div><div class="section other4" id="sec-8"><h2 class="">Performance Benchmarks</h2><p id="p-43">By far, the most computationally-intensive step in the mean-variance QTL mapping process is the assessment of genome-wide statistical significance by permutation. The original genome scan is much faster, because it involves only a single scan, and the bootstrap is much faster because it involves only a single chromosome.</p><p id="p-44">For the first benchmark, we ran <kbd>scanonevar.perm</kbd> on the data from <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kumar <em>et al.</em> (2013)</a> and <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Corty <em>et al.</em> (2018)</a>, which contains 244 individuals and 582 loci, varying the number of permutations desired and the number of computer cores used. For a given number of cores, the relationship between time and the number of permutations is linear (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), the slope depending on the number of cores and ranging from ≈ 6.3 sec per permutation with 4 cores to ≈ 1.2 sec per permutation with 32 cores.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time taken to run scanonevar.perm on the data from Kumar et al. (2013) which contains 244 individuals and 582 loci, varying the number of permutations desired and the number of computer cores used. For a given number of cores, there is a linear relationship between number of permutations conducted and time required. The slope the the line indicates time required per permutation and is dependent on the number of cores, ranging from &#x2248; 6.3 sec per permutation with 4 cores to &#x2248; 1.2 sec per permutation with 32 cores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Time taken to run &lt;kbd&gt;scanonevar.perm&lt;/kbd&gt; on the data from Kumar &lt;em&gt;et al.&lt;/em&gt; (2013) which contains 244 individuals and 582 loci, varying the number of permutations desired and the number of computer cores used. For a given number of cores, there is a linear relationship between number of permutations conducted and time required. The slope the the line indicates time required per permutation and is dependent on the number of cores, ranging from &#x2248; 6.3 sec per permutation with 4 cores to &#x2248; 1.2 sec per permutation with 32 cores.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F4.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3757/F4.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Time taken to run <kbd>scanonevar.perm</kbd> on the data from <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kumar <em>et al.</em> (2013)</a> which contains 244 individuals and 582 loci, varying the number of permutations desired and the number of computer cores used. For a given number of cores, there is a linear relationship between number of permutations conducted and time required. The slope the the line indicates time required per permutation and is dependent on the number of cores, ranging from ≈ 6.3 sec per permutation with 4 cores to ≈ 1.2 sec per permutation with 32 cores.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">For the second benchmark, we ran <kbd>scanonevar.perm</kbd> on simulated data, always conducting 1000 permutations and using 32 cores, but varying the number of individuals in the mapping population and the number of markers in the genome. For a given population size, there is a slightly curvilinear relationship between number of markers and time required (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), which reflects a linear increase in the time taken to conduct the permuted genome scans plus an increase in the time taken for “bookkeeping” tasks like organizing and reshaping genetic data. The slope (minutes per locus) depends on the population size, ranging from ≈ 1.4 sec per locus with a population of size 100 to ≈ 3.3 sec per locus with a population of size 800.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time taken to run 1000 permutation scans on 32 cores on simulated data using scanonevar.perm, varying the number of individuals in the mapping population and the number of markers in the genome. For a given population size, there is a slightly supra-linear relationship between number of markers and time required. The average slope of the line indicates the average time required per locus and is dependent on the population size, ranging from &#x2248; 1.4 sec per locus with a population of size 100 to &#x2248; 3.3 sec per locus with a population of size 800." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Time taken to run 1000 permutation scans on 32 cores on simulated data using &lt;kbd&gt;scanonevar.perm&lt;/kbd&gt;, varying the number of individuals in the mapping population and the number of markers in the genome. For a given population size, there is a slightly supra-linear relationship between number of markers and time required. The average slope of the line indicates the average time required per locus and is dependent on the population size, ranging from &#x2248; 1.4 sec per locus with a population of size 100 to &#x2248; 3.3 sec per locus with a population of size 800.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F5.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3757/F5.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">Time taken to run 1000 permutation scans on 32 cores on simulated data using <kbd>scanonevar.perm</kbd>, varying the number of individuals in the mapping population and the number of markers in the genome. For a given population size, there is a slightly supra-linear relationship between number of markers and time required. The average slope of the line indicates the average time required per locus and is dependent on the population size, ranging from ≈ 1.4 sec per locus with a population of size 100 to ≈ 3.3 sec per locus with a population of size 800.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Based on these benchmarks, the workflow presented here is practical for QTL mapping F2 intercross and similar populations on modern, multi-core scientific computers. Populations with many recombinations, where dense genotyping arrays that interrogate &gt; 10,000 loci, could not be practically analyzed with package vqtl in this way, although it is likely that statistical and computational steps could be taken to make such studies feasible: statistically, techniques could be used that allow for large-scale analysis without permutation testing (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Efron 2004</a>); computationally, the software could be modified to run on a computer cluster, rather than on a single computer (Jette and Grondona 2003; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Marchand 2017</a>).</p></div><div class="section conclusions" id="sec-9"><h2 class="">Conclusion</h2><p id="p-49">We have demonstrated typical usage of the R package vqtl for mean-variance QTL mapping in an F2 intercross. This package is appropriate for crosses and phenotypes where genetic factors or covariates or are known or suspected to influence phenotype variance. In the case of genetic factors, they can be mapped, as illustrated in one companion article (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Corty <em>et al.</em> 2018</a>). In the case of covariates, they can be accommodated, which can increase power and improve false positive rate control, as illustrated in another companion article (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Corty and Valdar 2018</a>).</p></div><div class="section other5" id="sec-10"><h2 class="">Resources</h2><p id="p-50">The scripts used to simulate genotypes and phenotypes, conduct the genome scans, and plot the results are available as a public, static Zenodo repository at DOI: 10.5281/zenodo.1336302. The package <kbd>vqtl</kbd> and its documentation are freely available on CRAN at <a href="https://CRAN.R-project.org/package=vqtl">https://CRAN.R-project.org/package=vqtl</a>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">This work was funded by National Institutes of General Medical Sciences grants R01-GM104125 (RWC,WV), R35-GM127000 (RWC,WV), T32-GM067553 (RWC); National Heart, Lung and Blood Institute grant R21 HL126045 (RWC,WV); National Library of Medicine grant T32-LM012420 (RWC); and a National Institute of Mental Health grant F30-MH108265 (RWC).</p><p id="p-55"><em>Note added in proof:</em> See Corty and Valdar 2018 (pp. 3767–3782) and Corty et al. 2018 (pp. 3783–3790) in this issue, for a related work.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-11"><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the LOD score of each test &#x2013; mean, variance, and joint &#x2013; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mean test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the LOD score of each test &#x2013; mean, variance, and joint &#x2013; in blue, red, and black, respectively. The traditional test is in green and globally similar to the mean test.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F6.medium.gif" width="384" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A1" src="https://www.g3journal.org/content/ggg/8/12/3757/F6.medium.gif" width="384" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491560" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A1</span> <p id="p-51" class="first-child">For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the LOD score of each test – mean, variance, and joint – in blue, red, and black, respectively. The traditional test is in green and globally similar to the mean test.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the -log10 of the FWER-corrected p-value of each test &#x2013; mean, variance, and joint &#x2013; in blue, red, and black, respectively. Thus, a value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the -log10 of the FWER-corrected &lt;em&gt;p&lt;/em&gt;-value of each test &#x2013; mean, variance, and joint &#x2013; in blue, red, and black, respectively. Thus, a value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F7.medium.gif" width="388" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A2" src="https://www.g3journal.org/content/ggg/8/12/3757/F7.medium.gif" width="388" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A2</span> <p id="p-52" class="first-child">For each of the four simulated phenotypes with background variance heterogeneity, the genome scan shows the -log10 of the FWER-corrected <em>p</em>-value of each test – mean, variance, and joint – in blue, red, and black, respectively. Thus, a value of 2 implies that the quantity of evidence against the null is such that we expect to see this much or more evidence once per 100 phenotypes with no true QTL.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="mean_var_plot's show the estimated genotype effects at a locus, with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1x, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the p-value was not statistically significant at any locus. For phenotype2x, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL on a background of variance heterogeneity. For phenotype3x, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL on a background of variance heterogeneity. For phenotype4x, the pattern of two dimensional separation without either total horizontal or vertical separation illustrates an mvQTL with neither mean nor variance effect strong enough to define an mQTL or vQTL on a background of variance heterogeneity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1326029807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;kbd&gt;mean_var_plot&lt;/kbd&gt;'s show the estimated genotype effects at a locus, with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1x, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the &lt;em&gt;p&lt;/em&gt;-value was not statistically significant at any locus. For phenotype2x, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL on a background of variance heterogeneity. For phenotype3x, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL on a background of variance heterogeneity. For phenotype4x, the pattern of two dimensional separation without either total horizontal or vertical separation illustrates an mvQTL with neither mean nor variance effect strong enough to define an mQTL or vQTL on a background of variance heterogeneity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3757/F8.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A3" src="https://www.g3journal.org/content/ggg/8/12/3757/F8.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3757/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A3</span> <p id="p-53" class="first-child"><kbd>mean_var_plot</kbd>'s show the estimated genotype effects at a locus, with mean effects on the horizontal axis and variance effects on the vertical axis. Horizontal lines indicate standard errors for mean effects and vertical lines indicate standard errors for variance effects. For phenotype1x, the pattern of overlapping estimates and standard errors is consistent with the fact that there are no genetic effects, and the <em>p</em>-value was not statistically significant at any locus. For phenotype2x, the pattern of horizontal, but not vertical, separation visually illustrates the identified mQTL on a background of variance heterogeneity. For phenotype3x, the pattern of vertical, but not horizontal, separation visually illustrates the identified vQTL on a background of variance heterogeneity. For phenotype4x, the pattern of two dimensional separation without either total horizontal or vertical separation illustrates an mvQTL with neither mean nor variance effect strong enough to define an mQTL or vQTL on a background of variance heterogeneity.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-56"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-14"><span class="received-label">Received </span>February 14, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-23"><span class="accepted-label">Accepted </span>October 23, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Corty, Valdar</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.1" data-doi="10.1073/pnas.1503830112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skutt-Kakaria</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Behavioral idiosyncrasy reveals genetic control of phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">6706</span>–<span class="cit-lpage">6711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1503830112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503830112%26rft_id%253Dinfo%253Apmid%252F25953335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY3MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.2" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.3" data-doi="10.1002/gepi.21778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauwe</span>  <span class="cit-name-given-names">J. S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A versatile omnibus test for detecting mean and variance heterogeneity.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21778</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D38%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.4" data-doi="10.1017/S001667230200558X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Use of randomization testing to detect multiple epistatic QTLs.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S001667230200558X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D175%26rft.epage%253D184%26rft.atitle%253DUse%2Bof%2Brandomization%2Btesting%2Bto%2Bdetect%2Bmultiple%2Bepistatic%2BQTLs.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667230200558X%26rft_id%253Dinfo%253Apmid%252F12073555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667230200558X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12073555&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175701000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical Threshold Values for Quantitative Trait Mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarantino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice. G3 GenesGenomesGenetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Mean-Variance QTL Mapping on a Background of Variance Heterogeneity. G3 GenesGenomesGenetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.8" data-doi="10.1086/423738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudbridge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeleman</span>  <span class="cit-name-given-names">B. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/423738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDudbridge%26rft.auinit1%253DF.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D424%26rft.epage%253D435%26rft.atitle%253DEfficient%2Bcomputation%2Bof%2Bsignificance%2Blevels%2Bfor%2Bmultiple%2Bassociations%2Bin%2Blarge%2Bstudies%2Bof%2Bcorrelated%2Bdata%252C%2Bincluding%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F423738%26rft_id%253Dinfo%253Apmid%252F15266393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/423738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223272200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.9" data-doi="10.1093/bioinformatics/bty565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumitrascu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Statistical tests for detecting variance effects in quantitative trait studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-comment">bty565</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.10" data-doi="10.1198/016214504000000089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Large-Scale Simultaneous Hypothesis Testing.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214504000000089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D99%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214504000000089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214504000000089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220638200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.11" data-doi="10.1111/j.1469-1809.1965.tb00500.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">The inheritance of liability to certain diseases, estimated from the incidence among relatives.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BHum.%2BGenet.%26rft.volume%253D29%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1965.tb00500.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1965.tb00500.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19656735800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.12" data-doi="10.1371/journal.pgen.1005648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreatta</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danku</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salt</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005648%26rft_id%253Dinfo%253Apmid%252F26599497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26599497&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.13" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.14" data-doi="10.1017/S0016672310000546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of environmental variation.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D381%26rft.epage%253D395%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Benvironmental%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000546%26rft_id%253Dinfo%253Apmid%252F21429270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429270&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.15" data-doi="10.1017/S0016672304006640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Effects on phenotypic variability of directional selection arising through genetic differences in residual variability.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672304006640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D83%26rft.issue%253D2%26rft.spage%253D121%26rft.epage%253D132%26rft.atitle%253DEffects%2Bon%2Bphenotypic%2Bvariability%2Bof%2Bdirectional%2Bselection%2Barising%2Bthrough%2Bgenetic%2Bdifferences%2Bin%2Bresidual%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672304006640%26rft_id%253Dinfo%253Apmid%252F15219157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672304006640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15219157&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221730200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.16" data-doi="10.1534/genetics.112.146779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic variants contribute to gene expression variability in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.146779</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146779%26rft_id%253Dinfo%253Apmid%252F23150607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkzLzEvOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.17" data-doi="10.1017/S1751731107001000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibáñez-Escriche</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguera</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A study of heterogeneity of environmental variance for slaughter weight in pigs.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S1751731107001000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DIbanez-Escriche%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D26%26rft.atitle%253DA%2Bstudy%2Bof%2Bheterogeneity%2Bof%2Benvironmental%2Bvariance%2Bfor%2Bslaughter%2Bweight%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731107001000%26rft_id%253Dinfo%253Apmid%252F22444959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731107001000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22444959&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253223300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.18" data-doi="10.1126/science.1245503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kourrich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1508</span>–<span class="cit-lpage">1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1245503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKumar%26rft.auinit1%253DV.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1508%26rft.epage%253D1512%26rft.atitle%253DC57BL%252F6N%2BMutation%2Bin%2BCytoplasmic%2BFMRP%2Binteracting%2Bprotein%2B2%2BRegulates%2BCocaine%2BResponse%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1245503%26rft_id%253Dinfo%253Apmid%252F24357318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNTA4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3757.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantitative traits</span>, <span class="cit-edition">Ed. 1st</span>. <span class="cit-publ-name">Sinaur</span>, <span class="cit-publ-loc">Sutherland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3757.21"><div class="cit-metadata unstructured">Marchand, P., 2017 rslurm: Submit R Calculations to a Slurm Cluster. R package version 0.4.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.22" data-doi="10.1007/BF00222330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curnow</span>  <span class="cit-name-given-names">R. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222330%26rft_id%253Dinfo%253Apmid%252F24197463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24197463&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KE52200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection for uniformity in livestock by exploiting genetic heterogeneity of residual variance.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMulder%26rft.auinit1%253DH.%2BA.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D59%26rft.atitle%253DSelection%2Bfor%2Buniformity%2Bin%2Blivestock%2Bby%2Bexploiting%2Bgenetic%2Bheterogeneity%2Bof%2Bresidual%2Bvariance.%26rft_id%253Dinfo%253Apmid%252F18096114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18096114&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.24" data-doi="10.1371/journal.pgen.1000901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridker</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chasman</span>  <span class="cit-name-given-names">D. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of variance per genotype as a tool to identify quantitative trait interaction effects: A report from the women’s genome health study.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.25" data-doi="10.1534/genetics.111.127068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting major genetic loci controlling phenotypic variability in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127068%26rft_id%253Dinfo%253Apmid%252F21467569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.26" data-doi="10.1186/1471-2156-13-63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-13-63</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D63%26rft.epage%253D63%26rft.atitle%253DRecent%2Bdevelopments%2Bin%2Bstatistical%2Bmethods%2Bfor%2Bdetecting%2Bgenetic%2Bloci%2Baffecting%2Bphenotypic%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-63%26rft_id%253Dinfo%253Apmid%252F22827487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827487&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.27" data-doi="10.1093/biostatistics/kxn030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Generalized linear models with varying dispersion.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biostatistics/kxn030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D51%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxn030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxn030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.28" data-doi="10.1111/biom.12651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soave</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/biom.12651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D73%26rft.spage%253D960%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbiom.12651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/biom.12651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.29" data-doi="10.1534/genetics.115.180273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Control of Environmental Variation of Two Quantitative Traits of Drosophila melanogaster Revealed by Whole-Genome Sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180273%26rft_id%253Dinfo%253Apmid%252F26269504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">evd: Extreme Value Distributions.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.31" data-doi="10.1534/genetics.104.039313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.039313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039313%26rft_id%253Dinfo%253Apmid%252F16361245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE3ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Confidence intervals in QTL mapping by bootstrapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D143%26rft.issue%253D2%26rft.spage%253D1013%26rft.epage%253D1020%26rft.atitle%253DConfidence%2BIntervals%2Bin%2BQTL%2BMapping%2Bby%2BBootstrapping%26rft_id%253Dinfo%253Apmid%252F8725246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8yLzEwMTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.33" data-doi="10.1534/genetics.117.202382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetics of genome-wide recombination rate evolution in mice from an isolated Island.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1841</span>–<span class="cit-lpage">1852</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.202382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.202382%26rft_id%253Dinfo%253Apmid%252F28576862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi80LzE4NDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.34" data-doi="10.1038/srep25014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viatte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yarwood</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Major histocompatibility complex harbors widespread genotypic variability of non-additive risk of rheumatoid arthritis including epistasis.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">25014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep25014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D25014%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.35" data-doi="10.1038/s41598-017-05447-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viatte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriman</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worthington</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genotypic variability based association identifies novel non-additive loci DHCR7 and IRF4 in sero-negative rheumatoid arthritis.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">5261</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-05447-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D5261%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-05447-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-05447-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3757.36" data-doi="10.1038/nature11401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FTO genotype is associated with phenotypic variability of body mass index.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D490%26rft.issue%253D7419%26rft.spage%253D267%26rft.epage%253D272%26rft.atitle%253DFTO%2Bgenotype%2Bis%2Bassociated%2Bwith%2Bphenotypic%2Bvariability%2Bof%2Bbody%2Bmass%2Bindex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11401%26rft_id%253Dinfo%253Apmid%252F22982992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982992&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309733300051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.12.3757.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jette</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grondona</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> SLURM: Simple Linux Utility for Resource Management. In: Feitelson D., Rudolph L., Schwiegelshohn U. (eds) Job Scheduling Strategies for Parallel Processing. JSSPP 2003. Lecture Notes in Computer Science, vol 2862. Springer, Berlin, Heidelberg.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3757.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3745" title="A European Whitefish Linkage Map and Its Implications for Understanding Genome-Wide Synteny Between Salmonids Following Whole Genome Duplication" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3767" title="QTL Mapping on a Background of Variance Heterogeneity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3757" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YtST7TzJeyznGBUXVN0LM9sllTnOnKN7TDWNvppO5ts" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3757" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3757" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3757" class="active" data-icon-position="" data-hide-link-title="0">vqtl: An R Package for Mean-Variance QTL Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491624" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-I5O1baKXzUSjVxcW4pNAx-S8qzx4e_hPmj8YdfWmuGg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3757.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3757" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-cebEGyIrP3piEXOLmD4s8IX-BPXmb5tnr5MRx6PoAOg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3757" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3757.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491624" id="node491624--2" data-pisa="ggg;8/12/3757" data-pisa-master="ggg;g3.118.200642" data-apath="/ggg/8/12/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">vqtl: An R Package for Mean-Variance QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3766;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200642 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491624--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3757" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3757&amp;text=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3757&amp;t=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3757&amp;title=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491624/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491624" id="node491624--3" data-pisa="ggg;8/12/3757" data-pisa-master="ggg;g3.118.200642" data-apath="/ggg/8/12/3757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">vqtl: An R Package for Mean-Variance QTL Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3757-3766;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200642 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491624--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3757&amp;rft.epage=3766&amp;rft.atitle=vqtl%3A%20An%20R%20Package%20for%20Mean-Variance%20QTL%20Mapping&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Corty%2C+Robert+W.&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491624/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491624/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491624/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491624/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491624/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491624/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491624/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491624/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491624/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491624/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491624/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491624/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3757" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Example data: Simulated F2 Intercross</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Scan the Genome</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Communicate Significant Findings</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Establish a Confidence Interval for the QTL</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Performance Benchmarks</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Resources</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3757.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3757.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/12/3757.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
