<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3903" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3903.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3903.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3903.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3903/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3903.atom" />
<meta name="HW.pisa" content="ggg;8/12/3903" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" />
<meta name="DC.Identifier" content="10.1534/g3.118.200710" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7242851" />
<meta name="DC.Description" content="Cassava production in the central, southern and eastern parts of Africa is under threat by cassava brown streak virus (CBSV). Yield losses of up to 100% occur in cases of severe infections of edible roots. Easy illegal movement of planting materials across African countries, and long-range movement of the virus vector ( Bemisia tabaci ) may facilitate spread of CBSV to West Africa. Thus, effort to pre-emptively breed for CBSD resistance in W. Africa is critical. Genomic selection (GS) has become the main approach for cassava breeding, as costs of genotyping per sample have declined. Using phenotypic and genotypic data (genotyping-by-sequencing), followed by imputation to whole genome sequence (WGS) for 922 clones from National Crops Resources Research Institute, Namulonge, Uganda as a training population (TP), we predicted CBSD symptoms for 35 genotyped W. African clones, evaluated in Uganda. The highest prediction accuracy (r = 0.44) was observed for cassava brown streak disease severity scored at three months (CBSD3s) in the W. African clones using WGS-imputed markers. Optimized TPs gave higher prediction accuracies for CBSD3s and CBSD6s than random TPs of the same size. Inclusion of CBSD QTL chromosome markers as kernels, increased prediction accuracies for CBSD3s and CBSD6s. Similarly, WGS imputation of markers increased prediction accuracies for CBSD3s and for cassava brown streak disease root severity (CBSDRs), but not for CBSD6s. Based on these results we recommend TP optimization, inclusion of CBSD QTL markers in genomic prediction models, and the use of high-density (WGS-imputed) markers for CBSD predictions across population." />
<meta name="DC.Contributor" content="Alfred Ozimati" />
<meta name="DC.Contributor" content="Robert Kawuki" />
<meta name="DC.Contributor" content="Williams Esuma" />
<meta name="DC.Contributor" content="Ismail Siraj Kayondo" />
<meta name="DC.Contributor" content="Marnin Wolfe" />
<meta name="DC.Contributor" content="Roberto Lozano" />
<meta name="DC.Contributor" content="Ismail Rabbi" />
<meta name="DC.Contributor" content="Peter Kulakow" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cassava production in the central, southern and eastern parts of Africa is under threat by cassava brown streak virus (CBSV). Yield losses of up to 100% occur in cases of severe infections of edible roots. Easy illegal movement of planting materials across African countries, and long-range movement of the virus vector (&lt;i&gt;Bemisia tabaci&lt;/i&gt;) may facilitate spread of CBSV to West Africa. Thus, effort to pre-emptively breed for CBSD resistance in W. Africa is critical. Genomic selection (GS) has become the main approach for cassava breeding, as costs of genotyping per sample have declined. Using phenotypic and genotypic data (genotyping-by-sequencing), followed by imputation to whole genome sequence (WGS) for 922 clones from National Crops Resources Research Institute, Namulonge, Uganda as a training population (TP), we predicted CBSD symptoms for 35 genotyped W. African clones, evaluated in Uganda. The highest prediction accuracy (r = 0.44) was observed for cassava brown streak disease severity scored at three months (CBSD3s) in the W. African clones using WGS-imputed markers. Optimized TPs gave higher prediction accuracies for CBSD3s and CBSD6s than random TPs of the same size. Inclusion of CBSD QTL chromosome markers as kernels, increased prediction accuracies for CBSD3s and CBSD6s. Similarly, WGS imputation of markers increased prediction accuracies for CBSD3s and for cassava brown streak disease root severity (CBSDRs), but not for CBSD6s. Based on these results we recommend TP optimization, inclusion of CBSD QTL markers in genomic prediction models, and the use of high-density (WGS-imputed) markers for CBSD predictions across population.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3903" />
<meta name="citation_id" content="8/12/3903" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3903" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3903.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3903.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3903.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200710" />
<meta name="citation_pmid" content="30373913" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3903" />
<meta name="citation_lastpage" content="3913" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alfred Ozimati" />
<meta name="citation_author_institution" content="National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda" />
<meta name="citation_author_institution" content="School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York" />
<meta name="citation_author_email" content="ozimatialfred@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0503-8107" />
<meta name="citation_author" content="Robert Kawuki" />
<meta name="citation_author_institution" content="National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda" />
<meta name="citation_author" content="Williams Esuma" />
<meta name="citation_author_institution" content="National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda" />
<meta name="citation_author" content="Ismail Siraj Kayondo" />
<meta name="citation_author_institution" content="National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda" />
<meta name="citation_author" content="Marnin Wolfe" />
<meta name="citation_author_institution" content="School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York" />
<meta name="citation_author" content="Roberto Lozano" />
<meta name="citation_author_institution" content="School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0760-4977" />
<meta name="citation_author" content="Ismail Rabbi" />
<meta name="citation_author_institution" content="International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria" />
<meta name="citation_author" content="Peter Kulakow" />
<meta name="citation_author_institution" content="International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7574-2645" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York" />
<meta name="citation_author_institution" content="United States Department of Agriculture, Agricultural Research Service (USDA-ARS) R.W. Holley Center for Agriculture and Health, Ithaca 14853, NY" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_reference" content="citation_title=EMMREML: Fitting mixed models with known covariance structures;citation_author=D. Akdemir;citation_author=U. G. Okeke;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. I. Sanchez;citation_author=J. L. Jannink;citation_title=Optimization of genomic selection training populations with a genetic algorithm.;citation_pages=129-10;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0116-6" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Alicai;citation_author=J. Ndunguru;citation_author=P. Sseruwagi;citation_author=F. Tairo;citation_author=G. Okao-Okuja;citation_title=Cassava brown streak virus has a rapidly evolving genome: implications for virus speciation, variability, diagnosis and host resistance.;citation_pages=115;citation_volume=6;citation_year=2016;citation_doi=10.1016/S0065-2660(08)60048-6" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=T. Alicai;citation_author=C. A. Omongo;citation_author=M. N. Maruthi;citation_author=R. J. Hillocks;citation_author=Y. Baguma;citation_title=Re-emergence of cassava brown Streak disease in Uganda.;citation_pages=625-29;citation_volume=91;citation_year=2007;citation_doi=10.1051/forest:2006042" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=G. Beyene;citation_author=R. D. Chauhan;citation_author=M. Ilyas;citation_author=H. Wagaba;citation_author=C. M. Fauquet;citation_title=Natural selection and genetic variation for reproductive reaction norms in a wild bird population.;citation_pages=1362-1371;citation_volume=7;citation_year=2017;citation_issue=6;citation_pmid=16050111;citation_doi=10.1111/j.0014-3820.2005.tb01785.x" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=485-126;citation_volume=98;citation_year=2016;citation_doi=10.1007/s00468-012-0802-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=1421-10;citation_volume=6;citation_year=2011;citation_doi=10.1111/1365-2745.12604" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_volume=4;citation_year=2011;citation_pmid=28483868;citation_doi=10.1098/rstb.2016.0138" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_volume=41;citation_year=2009;citation_doi=10.5344/ajev.2011.10116" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=195;citation_volume=9;citation_year=2014;citation_doi=10.1016/S0301-6226(02)00096-9" />
<meta name="citation_reference" content="Goiffon, M. D., 2016 Optimal population value selection: A population-based selection strategy for genomic selection. A thesis submitted to the graduate faculty in partial fulfillment of the requirements for the degree of Master of Science of Iowa State University." />
<meta name="citation_reference" content="Haffliger, I., 2016 Genomic predictions including known QTL for reproduction traits in swine. Master’s thesis submitted to Norwegian University of Life Science." />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Rapid changes in plasticity across generations within an expanding cedar forest.;citation_pages=553-563;citation_volume=92;citation_year=2009;citation_issue=3;citation_pmid=19170817;citation_doi=10.1111/j.1420-9101.2008.01662.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: A comparison of models.;citation_pages=166;citation_volume=52;citation_year=2012;citation_doi=10.1111/geb.12526" />
<meta name="citation_reference" content="Hillocks, R. 2004 Research protocols for cassava brown streak virus. Fao.Org: 1–7." />
<meta name="citation_reference" content="citation_journal_title=Int. J. Pest Manage.;citation_author=R. J. Hillocks;citation_author=D. L. Jennings;citation_title=Cassava brown streak disease: A review of present knowledge and research needs.;citation_pages=269-234;citation_volume=49;citation_year=2003;citation_doi=10.1016/S0304-3800(98)00205-1" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=R. Hillocks;citation_author=M. Maruthi;citation_author=H. Kulembeka;citation_author=S. Jeremiah;citation_author=F. Alacho;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=164;citation_year=2016;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=Root;citation_author=R. J. Hillocks;citation_author=J. M. Thresh;citation_title=Cassava mosaic and cassava brown streak virus diseases in Africa;citation_pages=18-8;citation_volume=7;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Pest Manage.;citation_author=R. J. Hillocks;citation_author=J. M. Thresh;citation_author=J. Tomas;citation_author=M. Botao;citation_author=R. Macia;citation_title=Cassava brown streak disease in northern Mozambique.;citation_pages=439-181.;citation_volume=48;citation_year=2002;citation_doi=10.1139/cjps-2015-0253" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=J. Isidro;citation_author=J. L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Relationships between milk yield and somatic cell score in Canadian Holsteins from simultaneous and recursive random regression models.;citation_pages=1216-1233;citation_volume=128;citation_year=2015;citation_issue=3;citation_pmid=20172242;citation_doi=10.3168/jds.2009-2585" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=R. S. Kawuki;citation_author=T. Kaweesi;citation_author=W. Esuma;citation_author=A. Pariyo;citation_author=A. Ozimati;citation_title=Eleven years of breeding efforts to combat cassava brown streak disease.;citation_pages=944-571;citation_volume=66;citation_year=2016;citation_pmid=25810896;citation_doi=10.1111/2041-210X.12225" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=S. I. Kayondo;citation_author=D. P. Del Carpio;citation_author=R. Lozano;citation_author=A. Ozimati;citation_author=M. Wolfe;citation_title=A quantitative genetic model for growth, shape, reaction norms, and other infinite-dimensional characters.;citation_pages=429-450;citation_volume=8;citation_year=2018;citation_issue=4;citation_pmid=2769086;citation_doi=10.1007/BF00290638" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Legg;citation_author=S. C. Jeremiah;citation_author=H. M. Obiero;citation_author=M. N. Maruthi;citation_author=I. Ndyetabula;citation_title=Analysis of the inheritance, selection and evolution of growth trajectories.;citation_pages=979-993;citation_volume=159;citation_year=2011;citation_issue=4;citation_pmid=2323560;citation_doi=10.1016/j.virusres.2011.04.018" />
<meta name="citation_reference" content="Legg, J., E. A. Somado, I. Barker, L. Beach, H. Ceballos et al., 2014 A global alliance declaring war on cassava viruses in Africa. 231–248." />
<meta name="citation_reference" content="Lorenz, A. J., S. Chao, F. G. Asoro, E. L. Heffner, T. Hayashi et al., 2011 Genomic selection in plant breeding, knowledge and prospects." />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Lozano;citation_author=D. P. Del Carpio;citation_author=S. I. Kayondo;citation_author=T. Amuge;citation_author=J.-L. Jannink;citation_title=Leveraging transcriptomics data for genomic prediction models in cassava.;citation_pages=235;citation_volume=92;citation_year=2017;citation_doi=10.1007/BF00022850" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Lozano;citation_author=M. T. Hamblin;citation_author=S. Prochnik;citation_author=J. L. Jannink;citation_title=Identification and distribution of the NBS-LRR gene family in the Cassava genome.;citation_volume=16;citation_year=2015;citation_pmid=19748909;citation_doi=10.1093/aob/mcp214" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol.;citation_author=M. N. Maruthi;citation_author=R. J. Hillocks;citation_author=K. Mtunda;citation_author=M. D. Raya;citation_author=M. Muhanna;citation_title=Transmission of cassava brown streak virus by Bemisia tabaci (Gennadius).;citation_pages=1-312.;citation_volume=153;citation_year=2005;citation_doi=10.1111/j.1439-0434.2005.00974.x" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. V. Masona;citation_author=N. J. Taylor;citation_author=A. I. Robertson;citation_author=C. M. Fauquet;citation_title=Transferring a cassava (Manihot esculenta Crantz) genetic engineering capability to the African environment: progress and prospects.;citation_pages=32-48;citation_volume=120;citation_year=2001;citation_doi=10.1016/j.fcr.2017.08.020" />
<meta name="citation_reference" content="citation_journal_title=Namikonga and Albert. Theor. Appl. Genet.;citation_author=E. A. Masumba;citation_author=F. Kapinga;citation_author=G. Mkamilo;citation_author=K. Salum;citation_author=H. Kulembeka;citation_title=QTL associated with resistance to cassava brown streak and cassava mosaic diseases in a bi-parental cross of two Tanzanian farmer varieties;citation_pages=144-2090;citation_volume=130;citation_year=2017;citation_doi=10.1016/j.foreco.2017.03.006" />
<meta name="citation_reference" content="citation_journal_title=Virol. J.;citation_author=W. Mbewe;citation_author=F. Tairo;citation_author=P. Sseruwagi;citation_author=J. Ndunguru;citation_author=S. Duffy;citation_title=Variability in P1 gene redefines phylogenetic relationships among cassava brown streak viruses.;citation_pages=92-7;citation_volume=14;citation_year=2017;citation_doi=10.1186/s12985-017-0790-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of total genetic value using genome-wide dense markers maps.;citation_pages=860-1829;citation_volume=157;citation_year=2001;citation_doi=10.1016/j.foreco.2009.03.021" />
<meta name="citation_reference" content="citation_journal_title=Adv. Virol.;citation_author=I. U. Mohammed;citation_author=M. M. Abarshi;citation_author=B. Muli;citation_author=R. J. Hillocks;citation_author=M. N. Maruthi;citation_title=The symptom and genetic diversity of cassava brown streak viruses infecting cassava in east Africa.;citation_pages=837;citation_volume=2012;citation_year=2011;citation_doi=10.1016/j.foreco.2008.05.041" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=W. A. Monger;citation_author=S. Seal;citation_author=S. Cotton;citation_author=G. D. Foster;citation_title=Identification of different isolates of cassava brown streak virus and development of a diagnostic test.;citation_pages=768-775;citation_volume=50;citation_year=2001;citation_doi=10.1111/j.1574-6968.2007.00661.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=W. Mulimbi;citation_author=X. Phemba;citation_author=B. Assumani;citation_author=P. Kasereka;citation_author=S. Muyisa;citation_title=Genetic Analysis of Milk Urea Nitrogen and Lactose and Their Relationships with Other Production Traits in Canadian Holstein Cattle;citation_pages=2468-2479;citation_volume=26;citation_year=2012;citation_issue=5;citation_pmid=17430951;citation_doi=10.3168/jds.2006-487" />
<meta name="citation_reference" content="Mware, B., E. Ateka, and J. Songa. 2009 Transmission and distribution of cassava brown streak virus disease in cassava growing areas of Kenya. J. Appl.864–870." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. H. Nielsen;citation_author=A. Jahoor;citation_author=J. D. Jensen;citation_author=J. Orabi;citation_author=F. Cericola;citation_title=Genomic prediction of seed quality traits using advanced barley breeding lines.;citation_pages=1944-19;citation_volume=11;citation_year=2016;citation_doi=10.1111/evo.13003" />
<meta name="citation_reference" content="citation_journal_title=Cogent Biol.;citation_author=M. K. Njoroge;citation_author=D. L. Mutisya;citation_author=D. W. Miano;citation_author=D. C. Kilalo;citation_title=Whitefly species efficiency in transmitting cassava mosaic and brown streak virus diseases.;citation_pages=93-11;citation_volume=3;citation_year=2017;citation_doi=10.1890/13-2355.1" />
<meta name="citation_reference" content="Nweke. 2004 New challenges in the cassava transformation in Nigeria and Ghana: Environment and production technology division international food policy research institute. Food Policy 67: 1–118." />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Nzuki;citation_author=M. S. Katari;citation_author=J. V. Bredeson;citation_author=E. Masumba;citation_author=F. Kapinga;citation_title=Genetic Parameters for a Multiple-Trait Multiple-Lactation Random Regression Test-Day Model in Italian Holsteins;citation_pages=1564-1574;citation_volume=8;citation_year=2017;citation_issue=3;citation_pmid=17297130;citation_doi=10.3168/jds.S0022-0302(07)71642-9" />
<meta name="citation_reference" content="citation_journal_title=GM Crops Food;citation_author=J. Odipio;citation_author=E. Ogwok;citation_author=N. J. Taylor;citation_author=M. Halsey;citation_author=A. Bua;citation_title=RNAi-derived field resistance to cassava brown streak disease persists across the vegetative cropping cycle.;citation_pages=453-19;citation_volume=5;citation_year=2014;citation_pmid=24642491;citation_doi=10.1086/675302" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=E. Ogwok;citation_author=J. Odipio;citation_author=M. Halsey;citation_author=E. Gaitán-Solís;citation_author=A. Bua;citation_title=Transgenic RNA interference (RNAi)-derived field resistance to cassava brown streak disease.;citation_pages=133-1031;citation_volume=13;citation_year=2012;citation_doi=10.1111/j.2041-210x.2012.00261.x" />
<meta name="citation_reference" content="citation_journal_title=J. Gen. Virol.;citation_author=B. L. Patil;citation_author=J. P. Legg;citation_author=E. Kanju;citation_author=C. M. Fauquet;citation_title=Cassava brown streak disease: A threat to food security in Africa.;citation_pages=72-968;citation_volume=96;citation_year=2015;citation_doi=10.1016/j.dendro.2016.07.001" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res. Mol. Res;citation_author=L. Peixoto;citation_author=L. Bhering;citation_author=C. Cruz;citation_title=Determination of the optimal number of markers and individuals in a training population necessary for maximum prediction accuracy in F 2 populations by using genomic selection models.;citation_pages=17-29;citation_volume=15;citation_year=2016;citation_doi=10.1186/s12864-015-1765-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Trop Plant Biol;citation_author=S. Prochnik;citation_author=P. R. Marri;citation_author=B. Desany;citation_author=P. D. Rabinowicz;citation_author=C. Kodira;citation_title=The Cassava Genome: Current Progress, Future Directions.;citation_pages=88-94;citation_volume=5;citation_year=2012;citation_issue=1;citation_pmid=22523606;citation_doi=10.1007/s12042-011-9088-z" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria;citation_pages=900051;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=I. Rabbi;citation_author=M. Hamblin;citation_author=M. Gedil;citation_author=P. Kulakow;citation_author=M. Ferguson;citation_title=Genetic mapping using genotyping-by-sequencing in the clonally propagated cassava.;citation_pages=101-1396;citation_volume=54;citation_year=2014;citation_doi=10.1016/j.theriogenology.2017.04.052" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=P. Ramu;citation_author=W. Esuma;citation_author=R. Kawuki;citation_author=I. Y. Rabbi;citation_author=C. Egesi;citation_title=Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation.;citation_pages=35-963;citation_volume=49;citation_year=2017;citation_doi=10.1016/S0301-6226(03)00151-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: Comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=M. Roorkiwal;citation_author=A. Rathore;citation_author=R. R. Das;citation_author=M. K. Singh;citation_author=A. Jain;citation_title=Genome-enabled prediction models for yield related traits in chickpea.;citation_pages=63;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. De Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_pages=2090-1553;citation_volume=183;citation_year=2009;citation_doi=10.1139/X07-075" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=H. H. Storey;citation_author=R. F. W. Nichols;citation_title=Genetics and evolution of function-valued traits: understanding environmentally responsive phenotypes.;citation_pages=637-647;citation_volume=25;citation_year=1938;citation_issue=11;citation_pmid=22898151;citation_doi=10.1016/j.tree.2012.07.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=L. Janss;citation_author=M. S. Lund;citation_title=GENE EFFECTS AND VARIANCES IN HYBRID POPULATIONS;citation_pages=1279-1286;citation_volume=97;citation_year=2014;citation_issue=6;citation_pmid=17248353;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=P. M. VanRaden;citation_title=Efficient methods to compute genomic predictions.;citation_pages=1103-4423;citation_volume=91;citation_year=2008;citation_doi=10.1111/j.1365-2745.2006.01176.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=H. Wagaba;citation_author=G. Beyene;citation_author=J. Aleu;citation_author=J. Odipio;citation_author=G. Okao-Okuja;citation_title=Field level RNAi-mediated resistance to cassava brown streak disease across multiple cropping cycles and diverse east African agro-ecological locations.;citation_pages=1939;citation_volume=7;citation_year=2017;citation_doi=10.1139/X09-111" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=Q. Wang;citation_author=Y. Yu;citation_author=J. Yuan;citation_author=X. Zhang;citation_author=H. Huang;citation_title=Genetic Correlations Between Milk Production and Health and Fertility Depending on Herd Environment;citation_pages=1765-1775;citation_volume=18;citation_year=2017;citation_issue=5;citation_pmid=16606748;citation_doi=10.3168/jds.S0022-0302(06)72245-7" />
<meta name="citation_reference" content="citation_journal_title=J. Gen. Virol.;citation_author=S. Winter;citation_author=M. Koerbler;citation_author=B. Stein;citation_author=A. Pietruszka;citation_author=M. Paape;citation_title=Analysis of cassava brown streak viruses reveals the presence of distinct virus species causing cassava brown streak disease in East Africa.;citation_volume=91;citation_year=2010;citation_pmid=20071490;citation_doi=10.1099/vir.0.014688-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. D. Wolfe;citation_author=D. P. Del Carpio;citation_author=O. Alabi;citation_author=L. C. Ezenwaka;citation_author=U. N. Ikeogu;citation_title=Prospects for genomic selection in cassava breeding.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2017.03.0015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. D. Wolfe;citation_author=I. Y. Rabbi;citation_author=C. Egesi;citation_author=M. Hamblin;citation_author=R. Kawuki;citation_title=Genome-wide association and prediction reveals genetic architecture of cassava mosaic disease resistance and prospects for rapid genetic improvement.;citation_pages=342-356;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.11.0118" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=1-7;citation_volume=2;citation_year=2016;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Cassava production in the central, southern and eastern parts of Africa is under threat by cassava brown streak virus (CBSV). Yield losses of up to 100% occur in cases of severe infections of edible roots. Easy illegal movement of planting materials across African countries, and long-range movement of the virus vector ( Bemisia tabaci ) may facilitate spread of CBSV to West Africa. Thus, effort to pre-emptively breed for CBSD resistance in W. Africa is critical. Genomic selection (GS) has become the main approach for cassava breeding, as costs of genotyping per sample have declined. Using phenotypic and genotypic data (genotyping-by-sequencing), followed by imputation to whole genome sequence (WGS) for 922 clones from National Crops Resources Research Institute, Namulonge, Uganda as a training population (TP), we predicted CBSD symptoms for 35 genotyped W. African clones, evaluated in Uganda. The highest prediction accuracy (r = 0.44) was observed for cassava brown streak disease severity scored at three months (CBSD3s) in the W. African clones using WGS-imputed markers. Optimized TPs gave higher prediction accuracies for CBSD3s and CBSD6s than random TPs of the same size. Inclusion of CBSD QTL chromosome markers as kernels, increased prediction accuracies for CBSD3s and CBSD6s. Similarly, WGS imputation of markers increased prediction accuracies for CBSD3s and for cassava brown streak disease root severity (CBSDRs), but not for CBSD6s. Based on these results we recommend TP optimization, inclusion of CBSD QTL markers in genomic prediction models, and the use of high-density (WGS-imputed) markers for CBSD predictions across population." />
<meta name="og-title" property="og:title" content="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3903" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cassava production in the central, southern and eastern parts of Africa is under threat by cassava brown streak virus (CBSV). Yield losses of up to 100% occur in cases of severe infections of edible roots. Easy illegal movement of planting materials across African countries, and long-range movement of the virus vector ( Bemisia tabaci ) may facilitate spread of CBSV to West Africa. Thus, effort to pre-emptively breed for CBSD resistance in W. Africa is critical. Genomic selection (GS) has become the main approach for cassava breeding, as costs of genotyping per sample have declined. Using phenotypic and genotypic data (genotyping-by-sequencing), followed by imputation to whole genome sequence (WGS) for 922 clones from National Crops Resources Research Institute, Namulonge, Uganda as a training population (TP), we predicted CBSD symptoms for 35 genotyped W. African clones, evaluated in Uganda. The highest prediction accuracy (r = 0.44) was observed for cassava brown streak disease severity scored at three months (CBSD3s) in the W. African clones using WGS-imputed markers. Optimized TPs gave higher prediction accuracies for CBSD3s and CBSD6s than random TPs of the same size. Inclusion of CBSD QTL chromosome markers as kernels, increased prediction accuracies for CBSD3s and CBSD6s. Similarly, WGS imputation of markers increased prediction accuracies for CBSD3s and for cassava brown streak disease root severity (CBSDRs), but not for CBSD6s. Based on these results we recommend TP optimization, inclusion of CBSD QTL markers in genomic prediction models, and the use of high-density (WGS-imputed) markers for CBSD predictions across population." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DyYFDh0Ga0rzdSoqeX8X2bTr--7tHNl6SqWGUWQAcz4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491667","apath":"\/ggg\/8\/12\/3903.atom","pisa":"ggg;8\/12\/3903","ac":{"\/ggg\/8\/12\/3903.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3903.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3903"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3903"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3903":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/xsSr0DjTc1eEOFbKbM1EDLj+LuQwDa55s0ystSs8+daKhxHEM+bhSw0vFTcIc8YgjQsWLRUxoTg8sMU+5ijnBrgc6tnrycenvq6Ta2jh3u0vOPQLIk1NaMEofhZ+2FJHETqahQtn8YeDddHysAzpynGFew0b61G\\\\\\\/tv6thNKdAM0XYUaTtWJIO7WGLofRyX3pJTrD5ugixkVBkhrNd+SZ6DVIO\\\\\\\/nTRTJBw6x9PiGMfY0FA+dstQ7baiYLxKu54vn\\\\\\\/\\\\\\\/xMTlR9Uh5UyyVSBbaUMfznMzrcBIUY+uR4HUA6ztKkV3plvlcK4HJSeAha3KkvXoY79jKbu13dkRfKml8MEvkwRtAVvSpCP48mQryiHvboeQoy6hf\\\\\\\/BORLiwI7n\\\\\\\/5ozHi+IfuGevDn8Wz3sGxS9sWMyq8jcnYSYPGXhWTZtiGkRbmZ4WAp2phWGbAto9XgEY7vUnM8ItsDyEYwxz7gZwSA8LCx6xWePTh\\\\\\\/2IztZu4pAGfAaCUlAlC\\\\\\\/FMN7nyNENtD1xsKASJ1FsaRY++5YmTNma2jNtGTXG9jbCkq2VYqY5UMxmJi2UYdeBmwtvZlxwYR0RTK9aHRHRu4s2dMnRRL+S5n0hGbKL8AxgSq\\\\\\\/MPHUYMrjGcUDtiJ6IhHtmR7VuN6Lqx94LxoF+I3jI4aP57XxQG3Sww7K3UY\\\\\\\/TETV71H51BViM8S\\\\\\\/TrACnOZx4Kr9SNHfZOYPNWQPEnqwiToRBwnQFyDPrnKjwzH6WBm26MOUnta1sPWV1t40IrOihPbzRwXKlrLTHmxxm8+z8l0mzRKKFIHFwTD+j+fpBE2J+2lwYdcuW7bpGNNPyn4B\\\\\\\/KsfpIkXhllGCYXPTiWpZeFM294c8mZTr10w5V\\\\\\\/UvqbkeUcJiGWvY4UtiQCgp+TkqEsSl2qgA6Yt1VQDc3kezQpmTMz5CQdv1WzaWddMUI2tp+4z31R51RDPCEJK0nrppRv4XaJnHMOOPKNPIsP3zcEeBuTZ1NUwL7ajIbRVNDeoa6SttdEqmd3h7b87CZ\\\\\\\/qQtOxcAMSh3BQgXFS0mSQ5l9QtjFw63IrUHm0PoEv2QvfTtXM5PKtE1ExUNcUvNrqmSTLNRtWvSutFL1ydPpnFKeJn8AFtNfsB23pwHkbEsgo8ICJcLmoMeCS33NzgGqtXcJo5HQLORI85HwNm2+qBX5leJtUnGluMrq4me8DO2kg90ecxBj4ltZknOuJ58cxGDAU9qteL3Nqx7p\\\\\\\/deIjYceN\\\\\\\/YpTIwKNgqjf1VIF5sGG3HfCJddl7nTLqcYOV\\\\\\\/RtWFba\\\u0022,\\\u0022iv\\\u0022:\\\u00225gc1BTXZwM84eHBznb9udQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291c6064462b707389f19a8630700491ada97cf56a7e0c7bf3a3e7ae49f927096\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j4g5moMwK90KJ+BUSgpfrwl7o9zXAbHCyzKtbgAZWWN7ZgpN+pP1p9\\\\\\\/V96Mtj7dPOwRn44mkJRmVQURGbHP7NXeXuXc3+0I8UmcOvTuDsMDi4dqXG1CP4GbEFQGOfNTMM63RP+Q4+P0o0+xQi2aIPo4037luFstl+GXojqJJRO2mroRIGlrqFYGMkmSLkLm+S\\\\\\\/PScRf\\\\\\\/Nd5VTRE8oLLf+5WFWNdDj3i3DnDtPGvKncXTdL9uJQLx4ZQ6tUZmujE5Zvf5g1KB71pdSRD\\\\\\\/M8L7KmlIcEcLg0iCBBl7W0IcZCNUoaBwBqBJMF2H\\\\\\\/YKsKlsbbRtnkGmA8LSwQuBcoCWUWX20eqVloNnsEvnrcAh\\\\\\\/MpUvwCnZTfAOsp2tl1OYljDDjYB2CTT3bC1\\\\\\\/KOYGF0XRhGzPCqz71Hjl\\\\\\\/JEa+\\\\\\\/FbforNkpFBEPx1I1GfqljqMYYfTShpyCstTEkx4BiHm85RgKMkzEf+snlMIrTcO3XMPhVrpogIa1r7aXg7uclSfYp+DvKwoG0b\\\\\\\/pedk4ZNJSRj6xL1q710rJKd8zM8bZZDfATZp9dqghjhwRpNFrKJ95Df67dOc2dBBQRL4ZPs9X3UnPi\\\\\\\/9K0T8RjSGQOciovotOzJs+rI27Lv15MWj8NyhI726sg3Ic1ZDTe1F4UmyPHT40U4at\\\\\\\/QrWyDmrOAbEl82uNZdNYAXA+5GoJEIbKEADggj0T0m4piV2PNbLr1+uhJEl3K38Jt4bhkiOgCbIL8x013rPbZY4B9k8TuR2s\\\\\\\/kpJpUrGvF+ZgyYZmhAYhDLraySWCh9vLYy2yncmQYbN05OHCdpLhUTFOMSJjaukn8UyXk6ZFLgUbY+WshKnqpiwUi60t5+OaT99uPImY6grB9zPScZhwitioOaqKMuc2acyyHRn6HhlfKTqZOTsnOEvCZtCfIyaVwWMPAl8wlMwVz7bVrTXMcqVYin9\\\\\\\/JrJbdpojI6oCqyOWoBiiCkRWIO2jmZEm+hZ2Rg394qC0pWfr8AlabH8wRoOCHmuAAylwWyrl1iCbSaxSymuCpXg3N0vaEB\\\\\\\/BzBZfWL1lhRiIvDYIc53DvOz0U9N\\\\\\\/ruaBEi5LlwEtyW39+m6V0jaDw1t7l3BC3zDnY3Y3Bklx9RDamkPfNE03\\\\\\\/0JLTMDLnlL7JiYyTwrYGFxqcteKxuHbhrGQoJ7JrAt8mfoo64K8Zd3zbM2s4ivhQiygbAunNLgzm7mI3PER\\\\\\\/GVOng8xvgagRu5VthHASOmXXyOmwwxUlTredhxJfIcvLFRHk2RPkOeybb+FA\\\\\\\/zEsSJAzLV7oUB4tLr8GFCUp3e9oQwpPRpgySSrWynw3K2fIB9Xg0MrrWaWNZVTMM8JAzeoKpQijdXQKCd0aehSSkxuNcLEg4BjnYxG2BlKKv+JD\\\\\\\/JiToDiP+7ZQ+RQvWlE7E215zlH0rgZg4W4fZqoMA9LkM\\\\\\\/5s9vuaYfCXhaiB1xpT6Vi9HqSL1072LE06+2HDODn32Jp1TGUYYV6ejSeb1uj61zJ2Sq9Do9kr8rNRXu5Sv73g2vOKeh5Smi3RarTmtt4hYXKWWjQkrUbMgFLKQtLfz9gIm58eiQ4+QhF2IVpfDEyTnLmrUKiQKMns7ioiRvAv5zb+lLMXLCf5EDuj13SwYkYkdGQyXx+z5DtgthWMAe1kYlgmD95pdL1HQGMnNpv91AMlF\\\\\\\/oqo4TMed\\\\\\\/\\\\\\\/6k7mrtxS0O6NqSMCMVpskoxvS29ic5ZFdlpyHrcSC\\\\\\\/Rmv1mveddRUE\\\\\\\/DhfAoScePK2g8yPAf8D6h3vxJIwX6HKtjJGvW7090h9EqEJSrybj8hEchaMAkymyLezVrv8KGz9v\\\\\\\/d5rH7+RT6B5Dga+49muHAAp+6vJ0XL5GQbcIc5Ft0o0cIzjn3ShEnjwdPiughTaH\\\\\\\/iAXuf0JfzU0ti3e3TPsantW6tUrW5SaVAM7TCzzFvXtMJldYjaInNBnNFQ8Px5i4TmXC5H8u6j3+eDbKFFkbJqav11YepVLePr3zC\\\\\\\/zA6VoeJwK9q\\\\\\\/eqc+v7hPMwJc6Ldfnecqw3NEYW\\\\\\\/52nMp30gbaOm\\\\\\\/g1uE+EqgtJeueel1eetpF7p+v5PjMbdMRu94faxZxeMf\\\\\\\/TwQwdwDVhRNgwxqqy2Zsa34DXwE2tj0F6YBAfpqs8VC1AQqM7Yc4oAlr9E4VtN89z2kJPKYdL5bumCqUmTjlzKu2TAdPv+G0iOKqEWd2bakeUc+Ru8GaTcprufreSr72ZXioU73glZMTZ3h9yYiLdj1gS\\\\\\\/4YpN+hLC4HFtYXaP\\\\\\\/V3t1kco+hYhLe993wZ3H7SRbQ2JuCP2nXBZivL1cJyzcJ6kgeR2DRae4GnLynxWB9vKhOHBrkuv3c2KaOb0GAs6AEtqjndbkC7GYE0kuUwcvoy1i+JUH27NZl0i14C7NzBfiZTwL\\\\\\\/kpB3DKe5TMxV\\\\\\\/GrY\\\\\\\/q0QF31fHeR5XrVRozMW3sz3RH0czKy0h\\\\\\\/y4kxaJIMAvlVcef134gOhWpj\\\\\\\/0Nhiqc7rOjkpyCyztDaLGAjC2U7yFtZxdfSxOXFIAv2XCxZ0IrVn0sqLGOmDHQMqvSttaqybchBoIBoIlEXKZtkVZT7l85Tsm3ZxE7DoRtoqG7VrHF5e6rwhpDZJvnwQvh43Ec0pbDSWla5sJWg1VnoVUZeSTBJZbyrsw8sMlVuSzH7kB4UByeFzpo4EnYL8SISjEsuq3j6AxK3MlNolniCwdfkFsmJ393HlXvLG5w85y6gTQOcY4rDQhmhnti4WA\\\\\\\/jkPjSD473Rnc5mJKFOdxS8B9rd6lF9QGHpZWaToY6\\\\\\\/TsKGPP6eRVf+Ag\\\\\\\/w9xfJ2lhYuYZgsS764VQOs6S6+OKSCtQRPBimYhLdJuJmsKjrSO975PJWTVdgYuMMqjzgcrx9jrQMSJz2OQoEwfhTsm+QY5ezTzcHxFrpYO7xApEDE4US9HUgCH31eqhld4H\\\\\\\/RaL8kmrnddUo9RvV9xDLkmnD8NaXtRBWWioXOCaxUon3AmgPB5eTuRn1uz6OxmaAWIukuv+NUfRjTU5w9hVYi752YJMk8Tx\\\\\\\/PMn98IdJlgydpGbENMfWJzgLFsbsotCQOcx1Sw+gADsD+xrm3Tf8xXgbtc\\\\\\\/rz1VZYfzvpyBPQ3rqz9kZElJjrJbmWjSbtzNiCxzTbGJf8rQVPRWdkT1SVK1SkQhAqqPdZlPOBFrPxxSaptWYXmeXvpxZ1FlxO7Pg48WKuIqd5hlTEedu9ayB\\\\\\\/yOIPmDZfnTj7De9D29lKFfgfQDh5dE3PsOYyBeuKa5yRVgLZeRndZy3REMuLu2wN7cfsRyWy9KUelfNvFVJrxWesUGjosifMR2xdZLj5nKTn\\\\\\\/fcLaeqdRSIcdvQ7A9r8Xg4BK5BxfJsXSqmgknUOTpzmRJAZSr3Q17CW5YfIxyLnQtBvHHz8QsP95NkcFsGFDWYmus70J20qAcoitzgbDuegQ85\\\\\\\/j64cK9YvvnSV4z985TzbfR1nMtgpqBiAjLlVaCyYdn\\\\\\\/6pr5xTZEX1k9YotTwUsKxQLlJEvSaLezvPeLrIqMaveEpg1FKEreFYoD8og+bM1w8V8MDfXj\\\\\\\/vYdyOckiYyrIY0pGdqHfoPDpFK3sCsUhH07tUSpJoVFgzuToO28b1IY8D\\\\\\\/YYai3nn82UaXEifFcJ5NMUzQhD0N7kerDzGHSQhN4q1TgP\\\\\\\/cwZ1m0+4N2Rreoji7r1kP3rVrq6RQdAtDbgUcs13Nk9tjb9Rtq2nTdfXC7uRBn2DstKzq9WswkcdvOoXIAKnFZ0a3iMe\\\\\\\/qNr8xcQ4CqfxIXVCsvsOtZdwhBTXlnJV8abcc\\\\\\\/zpFZLPZYodKBEs3\\\\\\\/dO\\\\\\\/yLKNwdFCha3anqFaMnf0Kw+lVUEl5JG\\\\\\\/ty1HrrCZOcl3aebt2ZoQGO+80TU67jKL64YoyGFd1W7QoKIRI1WkqdFEa4qMr8YkHm8qVJc297Q4t\\\\\\\/0VanbUt5wqkIfJfb+ZamvqgIII\\\\\\\/Oa3wO2NYOjX9fcG3nhOGRaPJVEYyYFM72g\\\\\\\/NxehsgMj1VZl2IEW1d4dku45mDEyeQfmW3kWdxFM+61k\\\\\\\/XB4ZmoixMXiS+YRt8e14ecOnOokdfjUTbmFXjzTmX5zrf7icVZp0UWBFmBfxnGlIwAlfwwt+jOAC9CMu4M9atKCDjgDLMCWWFQqNmoqD+se30RycljcER4lAEiQk+ujlYpJUtnbeeBqrL9GDJroJP3whaSxljjL9qJZVAhagBv7CzPNfAQAmrUBzxpJjVy11paBiianKIT0oBRLGTJrZNSkf4yaJWUHCi3pTAgo8\\\\\\\/HVnty3jl8kXD3Lslck0SMFuoki6vChOCHCPqEOBV1p76sOPqf+O8FJ6dE0YnEk\\\\\\\/Xg80pc3pRpEpAuMtQdVozdo8yg4xY88BN08zmigeKM99BI1ft4ZSg7FnbuHPgwFY6t5zn7CgvV75Wb\\\\\\\/+t71pxCYyqxxRbaS66A+PsH2aRgGBr90Rz5TTn4PUxsSgcT9oLSIXjT1DhHATWzWL7uMPmyq2QZgS5r0GZnvtYSt6iOZ1UkJ8ytg62FUohZkA+2CgG6XXLnBMP95x30Zn4iFrFBg8UjocmLZkGbYL6vNQOV0+IOyw8HTt4v8cbTmCozbA0zCLfmfert8PjAgf1kaKN09QDjwhAYf10amdSqQIEmlAUMzyna+2sEfxmcOmR\\\\\\\/Kqu+uGWPhgYzQGk7cv2HqHTPcIETRjJB5\\\\\\\/2nTsbVWNeZw5vaIcT63dpDHfZPw7FJRcd+oB3LfR5R7\\\\\\\/CZ1yRvlKaqMdjunppSGyu3tMIMsx2qYQnwJuA3EpwObSLM4r4SgXsvUr6U18Qf7TQnvc\\\\\\\/qIxJTJ+e1fp9NATPnG9T4fVL7EFY4z7TxEe0IDim2Th2R3+FfBrb7WIZ\\\\\\\/xxsDHquhYWyUTx8uPuhJ8RJRrO\\\\\\\/CGS9UsXPgLwPIATIRWjbHmaxrsMrGAEjS5WUqGC0Vy8OVEH7cFI2U5LCy6cTOXzTdPzXQsKJnbj7PidvkH4n+3FW0bvqkcDVM3x+nbjvbcbdoh7ykKPiTpjrVp5PVlZgjVCEF28ZdQkshZ72RjCaUFHd7CWWUb2u8iegwwxKkXo066cClcSrSPLcFHDM5WYjyjNuw6UWRI4zs3j9GHH4TKfipusjob79nW1jTExASO2rbfLEXZIW3KxwBGPmMdYY6jWl2kkm1djmjdE4NSBQPCbpaT3QbZuVLwe\\\\\\\/jXdI1yCe+zhhosHJWYLqOO2mzUd1Goa9cAwluEeHKAYySZ7OpdlEiEP1WXIV5zieFDUIINdrgWfUwRIyvb2\\\\\\\/\\\\\\\/t1\\\\\\\/9aTlRTSKRAzePU6rVh4EtxNSctbFbATN6Z0A34ROvrkmCEC\\\\\\\/GKo9CBZDQAsIfaOsQN\\\\\\\/I4qPUgLNnCN55Njvr2rYDoMMbXWJ0EEBKx4JJYDu+Orxbg5QiZcpVzvvsfQrB5SI4ooeju5UogJ3P98JoYz0cfz6kjuucbccRE+c9uFt3SZSSLmsFZ9FehToUOfEE5MvHGxu\\\\\\\/7giE30IigZBrVroLHDnvCLHybfecEvSm9588CSlyOgKEwn4cuktJxHnYSUmR+ZEwWpEZLe1u8sSUZGQNT0XZY7sQddN\\\\\\\/gy\\\\\\\/urnHNtcaGZz\\\\\\\/xGnMVQP3o0bwi9QEIDdUcnYb\\\\\\\/2ezijdnIViddfeBOavtWf7dBG+1wXaDDjfbu+pWJeuzBY5qikOHlBaJSKCa2tLEvsqXN84ZDy54Ty44A+0+7aPLL8Zw1+aT+6PHvfj7Ww0kxONMNii0CGe7ezxeXwyjYY\\\\\\\/ewo8sFwDqnsInHm5vH2Q3yNjRluoDDJ\\\\\\\/zRU5xkWqMrxqiXRjZ6VLAdro1SX1OpVCFne4bNHPe33rVTx+pWN1XlS6s0MJmmBBeuvICRHNWeMQ083qx+oZWL8+Hms+j5Iqr2+Sunc68pANXD+IcgGaEOBfmWdLC0n4UPSCFtHuIf2L755JPJetLpk1UZ+VQtWjc6HiofuNEVsmbwFb8vVyM5MhqzMwTN3pKUZWz1Y3V3vwaRa6\\\\\\\/oXkvRczxiql+lCKczRD2m0ee0VhTdqfrnkkti1tJlQFQE0GPjdtyCqa54LKW0glez7a5kU1ZfoQfTIY9o36KJgHlZiEAkOVqaMmUkZ0RZoEwwLcd+ambDdcfUZ4R4+zrSar4LsTxwY2jcnku9Ba7Jd5xAR\\\\\\\/\\\\\\\/LrdVgM5NCQJDMeVeyz37Qgg9WjhiDMmBz3m4l9Dr7eSbADr5tKgHRZ6biGSBjyWK7GGQeVlsaPdQ4xSBpURh7X6T7NGrlD5bRY\\\\\\\/\\\\\\\/0MffGP\\\\\\\/kTTpeMGGo1bNaAWdOENEOUFXzcc7TkfoStiGqPY1TYVJV6HbdTkg9wkya0tobokH1eoibwzZvxk8Rk82i441Te6WXhaT8x\\\\\\\/V0eFMw6rb2cHEy4C+6tnGAlerAPZf\\\\\\\/EdEzYbpkJWujhKnWeZtF5Me1l5ezuRPH66GGxWPMJssKAmBmNL4GxICHiOPlYgYwrDziuhLZDA3BtngdZaWS4R5oGd8tuPMLlZKPgbYMrXECkPp+jw2X8aoL4U1MW1dVO7dVtFbXyCpb2NNfmGE0Gc6ZZykNAm7eXnAwcbtTKdbVeYy9QWj3CZAs2OHoAI3nEwoWL2zsIvaViFQ0x\\\\\\\/pmFllXXDviqAC4OisatsgG6fbFb7\\\\\\\/JLHSSGj6jzQCiY\\\\\\\/iWgJtmNzial8mz+gg5mSDKqkx9yNsrzgPSGXNJxjKCqSYexZQnANLAWx2316R0AVE8rDpXq6YjpMI280z3rim4k3I+BiAGSP5afFe\\\\\\\/h3fsuNvoicoWLpFtarMGnJQXlcVDMFKWgT3Vqitv5r+KjY3Qt7DTJu8NdELYnx\\\\\\\/VjXJAhEY\\\\\\\/5aiHguoNIXa7l4NvGv9AxRU9tcTvj3RyZaI+w3Ub2ATijJkNjRoWKU6BvkZrcb8Qc7nhOon\\\\\\\/QqXUjLlMnjndbvWu3eXzML3m11QJwXAj9BRgiCqb+oVXcXJGCsFsZ85SpPudeY\\\\\\\/A\\\\\\\/2YHbOP6vKnp0fUkIu2bW1rMdbaUod8\\\\\\\/Ci+dJJ0HJ22h7P\\\\\\\/i7rrJ2C3JnUASiiBha3bbvwjwZCrBH8M8o\\\\\\\/BIrowC8sAvYDEJumpLJJiXZlz5k70C2E1c9GYSG+sFjvRDddX0auJJvvQfah4GNBxFH8KfkqBjelQsXSgy1j1MNG0NncOfx8ISDtXcxGC8E8E\\\\\\\/KXisKferuECBLwrN+s+SXxP5Dhd44wPhLR1apxWFXSe2PC5qnTF4TAd\\\\\\\/Xj5iCph\\\\\\\/6c3t2oGEzmM3gMeF9FPuR0NMioTnjTHLeIcPWEXkv7xI400E\\\\\\\/Fum\\\\\\\/ERgEEdCpdzxALhJIZggZczhIj2wN37KW0IlkKc5LoMUzt6RSAHyvhoMDnZLTV2Air+sc6xF2M2WzajK90Lh1BPDIPGtCxx7jmwX3d0x8KcLBPZyaG16GGqJ4caGkwEa7YT5qnXJiWjg6xxAP7hUQBb4wKXkx+GCQBHjXjqUgmEq3NOWz2wkhRVR+2PZDJ+SAB86IiktG529sRGAdkYYQa1k51JmvBkRlEfQGJIXR9Otrewi7+9+ajzAT9PQI0bTsmm4nb4w6ByY3pNk29Eb8EIclF2XDKklX1UpL22s\\\\\\\/KhElurEGu6JmtxN60Obhsv1Me+i1UowCf\\\\\\\/etYuH3IoLbo+5Wc2wwNKs2EX9miO25IkLbAVAR75rZBETr\\\\\\\/sD+q0em96yNi14UPvgCs5za1t6zvPe3gd4uAUbX+zi3OJHw0RQUDSXNb104lAZhIuuG1hxbWCcoY0rhXPTwXm0Ns6m8xip7ZzZTCKwo\\\\\\\/KFborNHqJHRrIuF2N3amKim5TSxZExZF4XEF4w946paytBqWCFoTSzQlpMKCj5V8GRusLjvZWx2sOXj7Fxrju0pJHC\\\\\\\/aREm7DxhgFsiBL3z9Uh9wU5jJE68AmIJ8\\\\\\\/9X\\\\\\\/6fgE2YCi\\\\\\\/nMAg67UWTNzhe4zmAPC8u31v0fnvRPHfF3gsoPYQr3XmtMz15m7R4Ayyg2ICiKwVIbZ\\\\\\\/RKSTFIMMte8ORI5j8uKNso9chhb4Kye3V+bt2W+5med7fTn3tVl6\\\\\\\/R37bbQn1fReWgaVPSB1qPVYqiJBG8268fCMRGahzZsJ6TlTKrcXDR0Z\\\\\\\/34YLqZGNojY0vv1LGPs\\\\\\\/4AvpWU1A01Oe\\\\\\\/pUU4HuKgGaJXLLvlicYdcnM\\\\\\\/ZtedlRk\\\\\\\/FenzN70OfmpSTVm7aGryOzh1GEkPwvpEa3x2VkZ7WVnQCTYXIPsxGk5CN97ePXupB6TDXe8NICruU\\\\\\\/YCKjNEk8gCFnL2jjmZSwSxtrVNr99twPWz9\\\\\\\/TsLNQg\\\\\\\/BD8B3OtaBZEgzA3l7ADR2vNZdAi+JTfx2eKjLJAxJfrXXDPxpNDs4BYXpgZGT\\\\\\\/hAW0FDlVXtTtB3H46HpAyJJmV7NBb62E06dXWIlSElsP\\\\\\\/Z2nAbZ8FA6miWxDME88WCFbOHPDrhYbMsFUMRS8WIhRyNuACf3Y5AMrxiGehXSmjWazwqVt68bSSn43H7PEP7g6YE2e6Y7YoJdpx7+bjUM9XWDoAoANTXUcJhgJfYSeTO6bkUAetMn2vO\\\\\\\/yhv2Iyq4BgM9W1Q8mUpH0MMUqNdLwo9xIRge4Cm0N5O\\\\\\\/pP0e5UQcYr9mbdRwcji9mAZFMo1Kkls3KfqRQ9kHoL4LhB+1s\\\\\\\/wUSMmY7KcJmYDxCK1aAXy5tHTaxA92B8VAdOB6QkFbv\\\\\\\/SB4elMUwDyI2+C10ZZ+IAPF5nPOo1fXm5nKcNqjZFIrYhaUZ8qmdv7ppVYHugNy4Lr1QqaYOgBgkxnq+vqYFUYXyE+LSTOD8p1+uN4N7j2TQQpySOeMwPlH\\\\\\\/Wi83WBxbW7n\\\\\\\/XLUbbIZA3nti0Rbt1uOke0s+WowSEIns+MctYT59LQjnoBfYnVA9n+MfoOqqWJW3UvauYOhRE4WDurK+c7ct2yKEAa\\\\\\\/HOH6H61WYha+dM3fCriXYJihw0\\\\\\\/6mJuVj0W5OEhSX2EHofgYZt0JDrc82Q4nXBIiXuS0zD9NUqHKQn6a1rXdoQFLCF3vOZNLp8kDSaoyt9eY65tUUGDSUOy0wcq7UTL6ce\\\\\\\/AjfOsB5w9eaVJwOglwQSfBRHDbudvxILen2epDRt4tigTgBJZv5Cap00gE4Ryl6czfdPgHRsMnVsdktfN30tM+xF4o8q3ygiXQ0wpSHMXaLPdU\\\\\\\/Qpkqnkbh3UvBwfI5uPsCbVAWjjC0h\\\\\\\/MLnP21iAH45RZ3T3IgttE3U4REwDNW4VvkjwScHd2Ag0dPpNMmUv0shPfLGvHo3lwRdjTnAskCZupJrw7sqLYHimShErYVM+iW1Sg13a5eVOohsGF59PkFHqk9hlu8dSJVuUYdWbySc01Js7UmYuhOz8+TcDWiF63JvJb5agijb4qXGXk2CPZz6RjLlLRj0AnwkxMd5pNS7T835thmVQRlODDMr3VI28hvWJQOa5jmI0wzPUITX2h7dJhz6wtstv+3Kv72fGb+zNxr2ear3C+8lTVxync18rwFKCW6t2aALdditzrfzvfNJat9n8xQul8G7ZtxgjStMeMXNQI4d7L1uBC0+EV7CtaXtr8urK504WXdDJTctgvQM3ld08XmgQFPdDZrhr5fALLFUA2oUSrgJK1dnD5iV1LYmvwEzr2oG\\\\\\\/\\\\\\\/VfQKpZzvnUZS2F0TT7uKY1I2sWLTtzVWipf0OjHwAfuFmYrv8tnAuiRR5XgtJ\\\\\\\/6lwtW2BT\\\\\\\/oEcQAmltmWo8kdcdrhcsMP63HUMamLB368UdrVZr\\\\\\\/mNUeFDoSJ0YDmFdKtOAlx0JKxCsyaIGlWarWC89KeeYuv1pE+BJS1sEUMnm6WaUuOhv73ZF0VvfcEAibZhE0NS0if7QiCi5uNBERtA0R2gm3scRysCK47iDLxzl\\\\\\\/3keSy04lerqcOque+ykzlMcydR1wcfMqjlsG5aoeprBirMQgkadIQErAVVSIEIrmY3\\\\\\\/L6vOg2FE62sadzRIf\\\\\\\/OIj0EbGId\\\\\\\/GOcqmF8VuSFT3nDJ3WFXRJrYD066EeTwaENraphMt12rTqXc0LzM9an\\\\\\\/deEqZJCy2xBXlRNr4ED61o0bfPjkjw9\\\\\\\/zOetM5JhwvJlKFf4mY16m\\\\\\\/JuW4UhZeq1NN8DW2n1ebCAGNVHcXDg5hdRH58O9BFUv3lz715NlnmyU6lV7npPRspo4Qvq8xSCoZST\\\\\\\/RDB+PX5E\\\\\\\/EyRG7MQL5+4NoiyNkBTXyWAqYxrWiyxZLrDGAYA8pQXy4MNAm9eFOFJh2IaSIobzspWILLZ+KxrN6vFd0VHMAtMfInTm8OR4GAqaCOsXaXL43lzEXGzoJw9IJPMZjLEIuYDi7SsnUmSVCihtNkEh39zLMjsVE2JBnA7LuMcFD7rCNU8A1Yzuqv8gBgch7HNzO9oDdQSR\\\\\\\/UPns74+V9bNru9DoIJ5iwweO9XP5Og7isPliq4PhtW997lh1A78KeXyhXCPveXAPfe723Q0iB60R4fQw0Cd\\\\\\\/7PUhWy3KydnO80tsgtEr4+JV3oVwGpwW86DEFWXW4GV3FvCjbPePMflUo64Gks2C3WfEwClUMcTyK6SKDeIPNzenAHPr+HeXUGAYed2cOctbUg3Y9EZ+vjMDWXMfboh5DD1U0g38eWJsyDdrAJQcjhfPfIOGS1lqj1Qi95PRey3vPhtaxL7NeP3xdPX7O0OtrOw8GAYhHrOFVhRrbPES9Np7eTjRH6r7yOlb2Ey\\\\\\\/wR9hk6sPsB0miRlJezKjRZRMoKaiAQPQo6ZAhm87bkokN0CREKUWRH7hsbPodpZoM9qTdyKE1lgesOO4eG2VWrmILNyMQox1if9MgnX1Fq+bDYx3IdQDjwgq1fGw3WpQDoA+EJHLzRH\\\\\\\/15ps\\\\\\\/VhNTwZiK0LYrs99l80PCSSVPAS6RZX6Ab03zkVlu9lO8kYt8PjvQDwur\\\\\\\/T8obf1mCNu1\\\\\\\/mZY7ISIlV\\\\\\\/jO3Y0ke9yvJUTRwt\\\\\\\/FUF37O2pb91UXj85CnU6BYocG6nuC6EePfM8YggwEfm5g2ZYFKzqZA8dWJ+XUdp0iP7YPvZ+X+UZ7SdvbB8qqknFCw2Bd8f3App8ac\\\\\\\/Lysj41ZYFH1UfMn0e+VweDax+PV+lCY8K79YVdstUDa0UmmRggH0JbHE4DRzUQStTEv0exKnMAVXdxSyI20WF7ORmX8lPYFQOLM1o2R5ShJZVM7r1s7FB6XW+dKxN7FQlber3kix6Q+kM0Z4jxKji260zbJHPEH1wP78sms90pGVhTQHdBJlvHTv1t0QY0el8xjKTKFVBd1OIlyJwHIPeRhCnbsDccKCRC2RUNbC6tDgu8GcR8pRDSJW0wLNZ4aMA7QWoWcpT6oQFUEQ5L2nODSq0KzRYRWgfcRuL4co1lXidQmikTBulyfJvNJdYGyAKoSoauG2nO4hYPBFviuMezysf1bRQTc5ZtDo2QzB1sP6r\\\\\\\/jEFfjnqzwSg1reIkfW5hyYWy85P9VMz+e9O0Xv0JBrZDxmocQ22Wgdx0gt9S\\\\\\\/4ffwECW88oP5jF63SQnVlO5Cbtoc13TTwy7o+JCgz+YSu725E1+syjxif2YJC6AhJtFvnaOt\\\\\\\/itJZX\\\\\\\/k67nBQPU1WA3zqFEMXOv4vMJR85l672CcLmTkNh+3urq\\\\\\\/ENl4U7WZkbH5pkbcsYjUc+ysT3i1ODUzuttoeEwr28scsjNj1LmKSpuV0cqiI0RD0Ou84MDP82v3ofOyQrrWUVF0AxhnIk3coYIXOcb2HSprF9PZxAxTBl3j6SzWHOAPbU8u\\\\\\\/KZACr6kyFOz9cLup0Q+rX4bkiR60KcPPSLCuw52GdDW2uIEiM1a048rmFp0SRuknIU8PfEARnUAE+wyWUrx3wSY\\\\\\\/Gd\\\\\\\/Wj8ZdrJyctGZ0nNhylkmK455ZA7bsTSxE1FVDzNnuTrYyoQNgBcYRZM50Ihm3Qs8+chgS4yctPzbx5lfMThMhTPoYjAqyTDiUoGFwbSJhS8dZpiZjwCV3F\\\\\\\/5uTLFJ7TJJKnrYxILNDgnfWKAWEkONrpuKJsIfnOSbkhrYcnU3VIPNoY602PMf75179cDkCA87RHaNIxw2gkKuo3JdKUhwxBxEXwISrX6x2wtOQMloev5NUpmy6P8bO0GdiX2s9uvhjbbk2w9soT926fFZGR5wp\\\\\\\/s49aYK9J3BogNFD+QykTUhMIKqwArLomjGxirFj3VB39lQBeE22nkCyWkPQfTk1B5m6ZB1Z4JNnNACqp2aVRamJXen2b4G6iYMC5ss5rwdpEQ1HxqpxjGM0AHmx8f20IMVRpvB0mD0HUbADUvxRqYl4o66qygc0qBxMPq4Y4d7\\\\\\\/53QoMf26R9+ap\\\\\\\/h7nT2YxujXuCYVwqCgyNWzWDK36ea6n7OZxqDqZPzPyv0iPqFztYjy9ZF8SN1iD+QD+LcXl+9YIMwOourGJErtbY1XZ6Ez8GihI0fOp\\\\\\\/IJQGpN5YmRccmvzYZmTDQhJxToYPdRn0f\\\\\\\/Rx+mgQvy3uwAmFqr+FzrVRgGw2YeQVdVibdDXUVuibc8ay46wTGfQGovS5HH1udsOt8dBjkY+tMGoAJYjZguTOfyfa77Vxu2AnRmKxFNfdRghTjFS7dpW8RYwpgk9sDUAhm6jKyjG1zdIRNYUd890KgdGBCh3QqdbVN+TZmA1rUG4mmctzdyKlQg199Jz+8WP3zwUIzmBR7ropS\\\\\\\/8vUD+8MMoOLGQ0KNL+gStzSkC8T1j2naaRv5+N9nqnIcUugvpcwF5BhYc31901TVMh1cqL8ra0Lj66h4e+Yt+1syuosoAzSXamyg77VDi1ius9DSrnjOXHzdnca3HNofC\\\\\\\/PJJejFO2XwfL0xtYARlemiV2ng48AEwz4cbMJI\\\\\\\/xF\\\\\\\/cjb8dsZ9vTqspneEe5v4iazwCgpVuhre4n6f3+OnvcuM9t9OgCVqmmqtPSovVW0mvm1Vp8iuqAuWxrOpbTFHAe4P8wn6vLlkC3baZZ6f5T0qL+kLuRvyJZSI+VqpAT1nA6QXesrDlgIBtfT4eXwLS2dbnn7lRRdZc2hKMITb6a+rgCyOC4eKuxwNzcRVgnXV0jP8\\\\\\\/STxeTiS9qY7GSHrvVwkjakWB+ihW4pWGtvUMo6+Zx3JE84USaSI6HUqy+o3xFNUwlhQqtZuiUDEGNITV6Sw85cOagF8HNW5W1IXJ69YVrFibBjeKJftZhTu7lOCRqo\\\\\\\/+fVQ06\\\\\\\/nCAjbNbpvF0TS2splpFyU0HFzs2MTZDLvamZKE3VyMq4ViQWgbRrmCjjq2U7S0R86DAc7pdAT9DmHoBHjk8amTysqVBdV9Oegv\\\\\\\/CH\\\\\\\/rk+SvizioPgNuexJpC81\\\\\\\/S5BnVEoNrF9yySsNSEUZy87DKpLIWyzGLe7GCJg+IbGc6YFzHNf1MM4O1xoiyEEcuN1VdVlpwaSq8wMIeGWBs4fhReYy1FxmSmt9yu0wtcIvY+8lJXa1l2v2UkoBOk+xa94zRbLs4d2rNKIDS6w\\\\\\\/F56D48i5tFar4ayvjD5M7E8PrGNoe+VMfXUyFAl65i2vYtwL+u6NRVhhdzHZPvkl1WNT37YWh1i+uRdTCM+LBcKp5cr\\\\\\\/wZ8htSiscS++Gx74rR859l8CvGExvNtt8GylQz2nkPVKlDWGamNPPKLz+W9bDXZtgWGdgiWekVNfO5UdSP42UOW+O82\\\\\\\/m89U4iyUzbO8\\\\\\\/ieFm7fFBcYv0OqID2ZGDzzYdCPI1jmKORlftRcodqMwvycZlEvQLn0hfFeyLYAqGXDZeyXlkFr1P5Hkx42MmOq\\\\\\\/yh9Zb5aYUCpIh9K+iwKVdhAhZmIcDart59MllHwBX7Hs7C2hIXOoPHqrqet\\\\\\\/5UM0HNyV7xjGE+iA7DjeBdfvwdrBg198476+XQRPZC8KBYjp4im7kH96Lr8wMiKQdz7fHuBPowC8mFPcxJAiWber5gYEKGXMbHINWqNVJ\\\\\\\/s\\\\\\\/6YagRKSOWBv3Lx9XcFaUf94A2CZ4b3Jf+2lEZ70G84MMA2wHO2kmB3J341nh+RsAWff8Xg0mEH\\\\\\\/4uXEeY8kGsiJYJnUaEszkRnL29CR\\\\\\\/sYnIcWjFyuqMQej59m6Y3CkPklbvtd2pr6K+yiigOrepCx+lQJzF6JiJmXS\\\\\\\/6lKU3Brh\\\\\\\/dLpiMgO+eBGagywNPunSEbUilkJQanLpldbUUAgJpmzgXVrsUv2MRJl5naYMmcFEUaLOnBNbBugZ+SyjzyOfhsIl8LNJYycTy+yB+ax+cxL6VRRc4cfVpCgwE6ZFGa7coNK0k+rOJv\\\\\\\/3Ve6Jv7\\\\\\\/j9pJZxi2mNKvhfMUZ0nUt8v8+NtculS+M4VDG0FgKa\\\\\\\/zSdwyYDhktXWmiVq6eHLCiU1OO9aOaF7oO5mShtDlN7g4XI8pBs4PJ7v04vlKjSf1ipcsLFNIse62V7Ymw8rmYgoLV5DF79tXaW9VM4+Cq5egyFB7K3HlfM6RIuKlBZWGVUY78iPsWFIoqavISYJpjGWb3rhOWT1tVw\\\\\\\/rXGINHhy74RT0V5oUT8bfuDtNEHKaGNTq\\\\\\\/hC0VoxVCI6z+gqo6cqq2dvqY\\\\\\\/lQJZfsBZ3rrPaDST\\\\\\\/uHKcFIQIHavlaQip0Qd3uwToJzkdTCxPi5uDVWufr4kY0tJfRh2fWMKrwIXiH4yzOANAGXCLJKK7zRunVEWYUZAoWVokEMLSMGxo5OelQ\\\\\\\/KwJJD2hKsNYyb070Frww3RmwqVsZM7fd2qZVgzG4pU464toEDnUJO+ADhz5NKNLDO1pqBstFrL\\\\\\\/THri2r\\\\\\\/EO4ja+CMoG0+ToEb3PyGK8oaRWeJ6gelaSpP7UO9I3CuauZa0h1WhJzPa8LUKxIrSHYrDJrQbfvS3\\\\\\\/um8o4wFB96My\\\\\\\/ONSoS8WKXSCdRa1zdderBE0Xkl9K+Ak2Bt6VoEUBnBVQctHzAecOtHsHWQ1vF\\\\\\\/dGPIGNQyM8ugFJPWl5m1FHV1REoAtp3I8BkE4hP+p2g+aJ6xjetgqCS2xE2ACa54HIxtSR0ERQbCEE\\\\\\\/evQyuO8BG8bNdI4aWEASPW5qbO3\\\\\\\/otVFOEKjtHdeQqzTNOy8oGSaTMyXLMQdqkSrv3wgpX2pn8JYab+pCm7RGzHNLqdE0S7RrW0ItJt7OtGycRSKXQfNt1IyjeEDS6A3GQpCIRx+o7\\\\\\\/4srtmqqJW1YoLBVTTpe6SCdxa52ucfG7toHIjfDc\\\\\\\/R2kKT+n2KnPANB0amQWt0aLGvfdlxcfMrXnDSMvxcvR4RZ2396VzMBibyBimPNmHv8VZbyB3bG9Q6wNFkQTEC5r7NCow7oDt0DafRZLVbavJKUgtw2dvATP+49r5hJ75E6\\\\\\\/bMCS4jnri8cVULqx0mm\\\\\\\/XpxSSakyuVU7HTKpp72vkMhZm8FJmVw1Ysg9UHhmd2IXNDcH+8KcTy7ElzMLJlYpRfkDRXWolYr7xPAJ6AFACDRmLlDCVZDWUIE33y\\\\\\\/nsfRYOGhFp\\\\\\\/oxzDf0zKCHg8VMZEVPRQdHt\\\\\\\/WGy6bBWrnJsJ8N8nFrEB01icVuBm9uz7282WXsUw0yROibNl1BrMDfPgotTuTAAjBFnqrK59Vh7gcxHI40f4czXn7xaYmYdY4r+Cg0r5eZWabDwgsrP45DR2yXkgTcLOmnX8YnuzCrLqFX0UdpZurDwWb1d9QiORocUsP+FbrZtBarB12omD8QsJ+KyaLBglVmWmyQYCjmUSitFIMmSlkOSl0UDE6NqBjggRJ7\\\\\\\/da2E1ZdUlBqUjPK9bE+g3fMj2ZSTvwmhU7QX4\\\\\\\/ve4YS2Cx2CIzV7PaboCVviF+QCRGke2GMLciV1h9uiyJRTzSDEKj9fD3RdmWXltPvq1EMHlACt589QIdcx3wbtLo3zf1kpQ7a610+ekh3311HsGL5idfTVXD63PJ2Fd0hgNvKSfRGFgACc5oWsPWrRrkJ6oVEf1mI7GQKI5GmoW8weS+GHSt9mxNBiXPlJLj4mporCFXue5f8o0rUljzb\\\\\\\/APuva3PWOkZ86vHbw+m9O1yzQedWED1GoXDZwT7y\\\\\\\/zkgsDu6AO+O9YXz\\\\\\\/sbh0Jy4QNBP1RgXcWlUtxkN13xuryAxy\\\\\\\/Lf6Lv9PcdHw6iZKPrnK5UWuhcnSr3f71OrFdXFokkoU8Q2d7zaNk0ZmPOAMPMz9MZieLZ04M7sU6wm0k7G0PTi9FJ+9sJzER5cSVr2sGNwHo4OnnK6IyR4I\\\\\\\/AYNRFyOi9wtB31\\\\\\\/oTXrQ5mY\\\\\\\/TEUKdqf7EKOBQ3\\\\\\\/nbJK1a7Yhg1YZ8dgCIOsTUlR+N\\\\\\\/PDcZ0B3m2Znz0WJXLWtk6rUyhBdqvHGYVguKeSvYBW4s53ZzgRJZFQ4jue5r+M6SOjI10+eSFymGEmBTXOxi298LjOyzCY73EKNOlQf\\\\\\\/hT8aW5fpBFjQ9mCIrjvDPHjHWRzXNvrF697ufrlou79iaxQuF5y0FZ+nGYGhGQeGT4LkUzUpGcCFGn9raz\\\\\\\/IbC6kjOJwqQZjU7Ue2qotIgwBWLSI+Ir5PFeDdtpHjfjxZGlpYQO+aPCSnNjTxG+8KPkbV0SCS5ETWdOW2FUxRcJ8gt\\\\\\\/OhlzmYq37LuzkRM9tqlOY9FdnbFlYFJNMYwpvbS2TLCuoE\\\\\\\/dC\\\\\\\/wzpjdPcxvYJOCwrgPcSNeR1ciBU9cPYqUEEEw3G4683zbo0auWbc6BGm6\\\\\\\/MLRfaY0Dh9W5RnlsIex6uDBhweE\\\\\\\/sO2fHX1Qfu9r+AYRX81yooYF90YisFnV31n4bOm6UlTjS4\\\\\\\/Q6StTkBPiQSnfYaUlWcFsO2x2NWWDeZRXNvqT8bCeYIBYuOHU2sPsp6Qy4J+1EbMJXFogJKpkuAfOelUUikV4GAN4fa5VfQjommi1sHcGnjaB9B6MAbIWxX8otLabhQjT1rQ331lvH+HKxZ\\\\\\\/9ZfPSR9LioDmY0WONWqJe6FuxNMIHsPc\\\\\\\/laldUlIKOVoYM7sl6Pb+\\\\\\\/aCY2\\\\\\\/WNRUfSHCf4e5wgJKdAzL\\\\\\\/kYTceRrbEAk+U31lV+PjKMlVhKWDurT0KflhkKLNywgu23i8XPuFFQSDNdSzQWDy0PG1ca\\\\\\\/6fz1BTQSMrjm8AkRPhpUwhhL5SrHPfSEcFzgSXY89y69lLGQV2WbeL+TmM0M6xENDA0RtfZAbFFIYdfaWjcNN8xsvGc7OciIrt\\\\\\\/OvRn7Rla\\\\\\\/Vmb+9bgcp\\\\\\\/vcmzzkKJTKhaCL3AO1XME+IotgnNsOlhAtBiPTRUm9tyqjUsZCOatj6NquA3FV9i2GTm46UmDU\\\\\\\/FBOcxdAI+xPUfCZz3jCyn\\\\\\\/3jt0nsdXYnAKNlUmAOxJCKeU7NZMxtmdCXGDRq5im7qmRmBeHRGThcIHEywagc+aAtOFn38cqcpdIRyBo0oBm+IQ9acqaNKNYnBSLtgrERvEi1P98Fq7n6cWt5NWoX8H9n4m1CN7k5NohIcCbW5vjEncQHN0DvJ72XFZIK1j8tGbCMT94vwGCoWabpGsnqHONpnoUb4I1zQjIqUJru\\\\\\\/XJDoEiZoC09Kq11LmwR2+ZIG0sBlE2DnW5dC4lwDk0Y2M8cve0BUlgbfQG7dxZy5eon3+Q+xNVhPTBTdw1wslJPfcD+w6qaEw3gaLlvsxTPAtpo9GsA4PRgQm2UV0y+g80jUEE9hJmyRPkvXlxfonchjDl+qpHEIS6v9wJ8Dr5JeI\\\\\\\/Ytc15G+rftDbskFO1hzOOkgjpfoXpzxVMpotSJXJ0I9MZ96tvKALp\\\\\\\/Xjt1LSUGO3NqIjAJXwFiqtiPBk1hMS\\\\\\\/yZlsIka3ywklzzC3VyrIti\\\\\\\/TrsOoY\\\\\\\/+HARuFOBUNvOrqfR9G5DcrvOMunIdQ1DRFmzpzZ2K\\\\\\\/n\\\\\\\/u4jhiL29kfWDLWkIAi2SXK\\\\\\\/iiFNZHPq16ljB6dQ2rf41kazBbYwP5DkaMAsIqYWJIyblXPvhDxuS8sX7ubxsrfsBPcy7F2CwH+gz6VS8SfZRDXqj1eIvrFtPb17o1SDEQcG8xIB7bPSXpN5DaR4sQn+HRpZ2wEif217nK4H3\\\\\\\/kBIkZ7j5SEyLk9r9SuQfRrz8rwLPz3G8dJfbmjV8JXI07C1zKhhnRVNmGmDj9qVbOzt4batZiCEQDwpo7E4j6fUHMhvif3MDx7BgHrTDHINeVCSNKYBlbxjaBrbWSKgy1b3THZ8gbCcpxkZn3oEUkj6gSvpa2OsPw\\\\\\\/ixcSH6qttEqAo2vVnWAhnwP5wgXJqBAdNnTpY3kErvve0rcqriBZDsupt+PpAopH5WzYISn2kUXrVDD9Zpc8OPHusKXpiOWPhgkXmizut5CIG9sgg\\\\\\\/lJe5AhMMqUordtD3h5sZeHGbx7JB8Ml2CWARbAVKk6M0PxTQleB\\\\\\\/zG4yKBBfhl+60nGAhWKHEshfGII1fU0oj+to3ACEbQc35ebzfvnknnFdm5aRzN1ugUOs+7Wp6hTcJRJkrv1IS2bUpG5QPCBnCV8U3gM2rxFIhyzO4GXTSEtAfkrwrrih2dpTVz4V2HHkdyj56BxOvW+N0hhS37EBS9F3LWtB43LMq9qj\\\\\\\/TliFZSB0lef7oAh+sth1f9aORd5LqWej2bG+xwlKVzByfQ1UwBW9kKs4WD7BtNvCwYCV2ysd1W4sbqFX5z7CGljLS6zr1Q6LCHcowleqH\\\\\\\/GlH89nmMCFsDw2\\\\\\\/PJr9BuhVywPjzGpN95nXZNd77eAgf4NIaAqhOstiDQ25rA61oqH1EviKCoGm07OO3sUtoxPSpfszVwrwWLSZRjXWx9dPcSvo9gye5TkTWqSo32HIBdpUoocL7p9acROQzMDAXrJl8JX6G0MJV6vGni1W1qy8uGid83M3N4tKLQvA06+QPh6SGcJqX2sIVLnYqRIg6D2GvzQL3vYD\\\\\\\/7HYzJpU01FAb2qwfix9n6OVuT6wcxM370IVsDbHngSicplVP1SgNUWYVVZXSUf6n+SK98goaTZJ8V7cFQnJyG3FwVNKZy7Hd8GjdkmgvsJq4fHJ4l07a5Sg9yQ\\\\\\\/HWXMCXyxhjRNxF8ssOSx\\\\\\\/YK2z9Igmd2tDGo6jef\\\\\\\/yIkgs+0w1s3PmV5qv3biz4BNg\\\\\\\/TPJPajkeK\\\\\\\/uNAWUjMC6BhGwvlfidq5bJ2LgohvaWy+3asZ4EaTWgWW+DxCGRe2v8sXBoUGiMKUN5Zi\\\\\\\/3hWwif1WutJZ\\\\\\\/uDU4goABryt95T6l8FsdOZhj2s4XrbsMhnHMI8w7WN2LbM3OJjJbJ0W5JHzHMtTOVB2Z6LqFENAB+MeZPnm9R2cLP1F7XS+Sz1kEqUGO\\\\\\\/QgdbftRla9KraV1x4d3DpdR8jkV9rnzfgXSihStmEomKD6bMjPic6y\\\\\\\/oK7QgXpYQ8Yso8B85byMRU22m7Q1LSSMDgjKWo3cl7caH1BFpxqPg1985MLg7CGnNT4WXGGyV7p54lHQRbEHhn63i7SI9kVqaxUFIloFe0DDox4av1b4qf7VO5wjdoZa+sWAtezjgn6enjlHa16n7410BI55Y0iAza+WroZn1PQtW40JCeBTtneVCdpnkEWwoZ+kbjgfq6\\\\\\\/BHLO57tAn6H7RkyV73RCxeUQrF6jAw4nTQgWS4MSsVSVwvq\\\\\\\/8HG8901YKtjCVMS5M7yI\\\\\\\/ALcXS3aXNllix0Mhr9v1EzqWSEGzY9gmoMeAYvpv3Jay7CRMPT+FiHtBvqwul7ynhyu51GggRbBIrnVfzR1df6V3P2DgjD7i66tilQSEUK908kUDjzQE7ngVJ2tZOWhOmFcLpM7Z79oTRXiM6azBYUvoQOAYVGedxuBUtR29p5SdZMtnBC9I4FpgYIgOieUD2tlJnEUrkrzTW\\\\\\\/6p8R3aTzMpmHVlfk+ltnzjwNE1HSBBAnHOIwPCO4+a7Bn8F9PMdqv0+8xXc7un5AKgkW2Qcx1pLyWPk+m2EOzn1wVcZN1Ma6amH05HTptdccdWaKF7WCAS0Ic5uqvlthSGhtQa\\\\\\\/wJgwkxp9vSl3wenOn6b2pQzdn1KqwEInO1+nxXtXYBPi8bO4ko6LUeqX2J0eeSvWWCF4Jno2vhEogYTB\\\\\\\/tloU\\\\\\\/Sao2miHV3b2eP0VVPONyzPd1OCWOOFrZZVGjpYMbA4mO9wrW+TEj1rOZ5knY1sWemunbNps2NIYWghVbZXQrCq4vmyeSySSAta1Y3U9cucMa9CfAxTE4I+afI7VSArmWXMx6208L3cEa0zQzXaG7Ply4k57XJ0ibvnPf3Oryf2udcMx0rtzYdVMo5XhslqBVos9QDPaYPv+Ma\\\\\\\/iaI7OVDlHUfBS8Bg9rvirNKKhpiZGsZOYlvLmZ13jc0f9PwyyPYxHc5FNRIOv8NUJhdAsUPLHyGRXwLC4tuiTSxcaZ+1ABfyn96CH1opGGqzbXZy4WCuzzt9oBpE3tA\\\\\\\/vvltmyiqU6GcojP963feZU\\\\\\\/Qil\\\\\\\/+pPqEbPBvrLqhq3\\\\\\\/SRdJDa3r5lr2m7PVmmoV07kiD+TYGJ7TW5mYU5sbriDwBSd8UoFsVSKi8yj827tJdwwlCWyfrYxH3w57ZaHDbQDyF6O33kt9GtMKSh+HUv3NzB0eNKyBBwC2tePGkVcXsLsxkVvnKXZDK+PfBEJ6zv9d\\\\\\\/YoCwc7XyMfqxhkbyiVLjysiJe0MwePc04rDT0MjAxSiZAJMBomYeMNGllAVTg\\\\\\\/4SEoqvtspv6dGFUMs32946mEbl7XUIusV4NAgQ1dsu0APpnFk6fXQeAIsqU+\\\\\\\/ndgC9a7ogDo28bXGuDaSKf8Kny9hF4poyyePRasjqAu7u4NYKEe8jTF+HbJPl1yHuPTKfbTigvpVOxourio3gsQCVd8Dn01zhycsb+Sq24MaYUFEnXDVc1PgUBz\\\\\\\/YBObDzz\\\\\\\/4Ec8eDvZZOToE1yeFiJ+0Rz0\\\\\\\/nl74LKZ1hOmBp1x9y8IoaNI7b9rYi0p3jT3EZTWCzliq3Vwip5Lr5opxzX8V4IADEdnihJKsumQqH7TtVYzW8CNFTN6D8yOuLixvo6E+Zm+KT5kQVFeCoZK2MPay6Jh8lamMecltJZ+oDq9VPllqe5\\\\\\\/RNKUvq6HUBw+UOb5mjmd7Rek82jOTTe\\\\\\\/z0dI57LaGzrJJxz0DF1bDfq+HKIqSjxyIphZecL0TDfG0FmDPAxsJq3fFRsU+yHX5YebdYS+EtGXnpBRdfj\\\\\\\/\\\\\\\/CWJkg1XlwA3YiITqDqjo\\\\\\\/fApZipuEFATWgiAU6MysTinm7oOaqn7ihsn0HiX+4dZZX9Hi9GxaiywpOqqklbj7xClhrgRIOitE3M\\\\\\\/u9dm+q+cp4oyeAjBVwfXy\\\\\\\/nEe8O02AiPzVJ1MVWi5tTafkd3XPKuig5MEHZ9GGzvAdGRW4sdOQKSzv2Gsw6mXhseZbsCwTYNLEbBZe9Plras9Zx140QhQ2FVGq8sNvF5x4gTu6dzy4FjtmeXaDNCAxsT1DOfdjMYibeiF2KgHW2p63uYNinncu3CpV0bEMbaop1Df489COlViTNckRjbfxvyZ4mA1NB8kRoB0MHuRpgkNsdNPNXGwsG8VBY7AvshZWECdzm7QIpddxhEJPF77zr5hx4yMtJ8AxdZ2uI5U026uhCXmhuIv\\\\\\\/X8zRhnPyIlfqsgaxCCR+wsR1NllhFtUq3Kn1WpdnBib4VlnFjnVWtSkH6JU53xlAJALFOmCjcsfjVO\\\\\\\/FrDUvAalVpoq2NXQYvsdt1UGJyfK5207BDUDVhFX++FE0Axrctx8\\\\\\\/Y74Ztc8n3uA1s50F7ldDE5WmK1M883Y\\\\\\\/WVMgERzGV\\\\\\\/9lfGln7HVlX3+1YlMsKz4edMa36kJZDKWMzZgYQfI3YDqfvOtyun6ftkk5DGdWxzymFxN6PapT6+GZhCE+DvrLa6vAa5x8NZX1Nau5k2atlehcMN8JC8lxeAiO2UGAljP4gxNcHGoucuLmG4iwL5d0J8YZa2Pz27MT1MwlXA4j9kpNLOWAiU+yn7aQHI2M6Mt+b1foqQ9bCmHpM5HhMltjAHPRrkKNkRYjtamunhkd9JHeJlimH5vGhfNbBh2JErFPFlwzHOwruDV0+NXtp9R\\\\\\\/Lt9ELFP1oJE4jwNzPUreIHeKxgDAA\\\\\\\/rvp1IhSONvJQGVhjUCYfeQC3GryVoBiJ49d5MQ5CrTD6wIhx5ctKUGZshw3M+PFgeNpfF2v7K792ULeqj5quKHWb47BbuLhrj1rBcD6ws8YfUrqKHXv7HnVGBWoP\\\\\\\/C0uSsXalyHjqHIQETC63mbuGRo2SgtYCObEmj3\\\\\\\/NxBRt\\\\\\\/3mmBraYzwM\\\\\\\/Yxv0YLMVyzCuN7QxTMAC1SrIpWQ2qwBdgFv1Fdi+Lpt7MD\\\\\\\/6eLLZx81ep4N3Uwl9SPY0Ru9YxujoRwJJt0wIYqQcP2LfAMmHPf6oPFIJCBeTQ21tZhrWeYrsZPXiB\\\\\\\/FAIydhAjkiDuK+xyy3WAPPbWsmmbqELTyyca\\\\\\\/DA3a3Eb\\\\\\\/h3glbOso2jFQJB07tawW88Qwb8n4hIO5MMoQVi21fgCCCY5GFooI74X1TpkkA94im+sDLppsHNzuHcNs9NXsA3fOdfKxI9nfd1+PeuwnEuElCU45RQOyJGoVnJqkV0ejXaEzpetg1qSZnQ873Ayh5+Pj9SUNp7ALwb8QvaHA+rY2+QJROE3XAT7PIqJokpElw5WsUTvxD6Fi+NOwsSzJsGCvc6cyZZgvHYHXruiqJYiOP40cgMJGB5\\\\\\\/fRzMRBTdBj\\\\\\\/Pu11YKLyHH83pBy3jgk7iNpRNdbSvwGIYBEAEG2ts70sL+vyqmMEBYKEgerFtPjtpTgTWcHRVWre3ckUbWZP\\\\\\\/NOxiNljkP7VHdRCWzHIub0DuM5ctVQS22fJBTT6bs419G+FIPcU5U35lY+1rLrOys2W\\\\\\\/HTdO8XrrrYWfvwTCHmlAeB3P5uhKVih+0hu0M6phSb8uS50dSgiXS35IAmgbZH7XyzYkE7pTpA8MSxlipHwbzNl+rDU5xQExNmCG7pyV1c7EZHDlo64\\\\\\\/zjfYytpgFF9jX2mAA1NXbR+eyWqUttOf1bZ\\\\\\\/Qpo8cY3FxBotSkRp7mDJ1806HvbggQcc8NIfQCsQyiJJVtieSbFQbym54Akx8U\\\\\\\/S\\\\\\\/JYBowWi0huP4avPQXYo3u\\\\\\\/xQlcEcY9xPB19wrqjCfloHZWBfjxz50zHixu+NxuBfD2a+KOBSMBDMT1m+SdxEJ+peGg+fuu77pOX\\\\\\\/C8tOMxLmMe+3sex\\\\\\\/kbECTpOT3xamSBY+D92SbLiqCAwWpXIx7OfhR0vfqg4RRkFyamGGR7OpJCoqhd2UBChz94XlN\\\\\\\/ipDjZmM9Qe\\\\\\\/M0GSVZdW7DbV4bIygh4tzTLjyYc5qiFz0gqL5W5DewB\\\\\\\/HpaON7UcOWnyfN4q7rGyfWjCWUlXRe2MGd2PrQQA4XktHgSdTomHjKzq2tbsPI9K1H0ik5BwdZWR8lQzqDxztyEKfxIueUdGmyCdaJQYc14cNNN+Quq+Z\\\\\\\/gOY0oojyjooC72FucUWnBhxz1\\\\\\\/EiFx1mHJ\\\\\\\/Y24SRJYxzrqD2AINWXl2qvOYlO9lTChaup0yDsaBf3UfkrL+KLhPLcqbGTaWsEAfr08QxEM1uBA+JK1yZPCkYM0uB2ox7yG+M2dthJv98Fe3DZ70Gst48N6++Z8J3WmiY8x6iPFii8qJeAFjm4CI4AMuBK1Njdmwqu9Owi2ryrOk+MJbjYjQam7QO6ryduTwIlJKwkbC4fyS45LzWA\\\\\\\/krE8JcjNL5ssyo10orKFCmliHhE5QL8ooAyntCdS9U0XN8gISOq8CktpTH4FGBCBYzMATFFvKPXL4mhptvbLIbk1EK\\\\\\\/QTrYoyoqKWaH17rRf9MLVrSPosn9D+rDtftmf1UoyeQ17pDB1NguZl\\\\\\\/mlSTHYfZdgaOFiTJHy31y7PrZK0kkOz0ESWwQijgvLDEJkN0RPZmg5cA51hCZIaXMhvBN\\\\\\\/uXqkJdXK8KYKRwxRxszs2c8q\\\\\\\/3c0sjVpSfuT4yxJqizedGBlNNM6XqVDmclVFvnLdhRCEJWZWC2xwUBRSs3sIXcx7NSL\\\\\\\/DCajR610MUdf6Ze6XXx12oJSVQO0w1xqHdCHifhXiicF7BdCoz5zVwOIrWa4Zjo9Bhh4Hal2Gcdq67N0n1sNtZx2IEWNW72SpVAuWok\\\\\\\/cqA+jrKYV8pl5f\\\\\\\/NABuiPqSMJNmiLlHyJX9bkAtpne9GxlgWFjE7gBtJeAy\\\\\\\/bV5hgF8gV8r7pxIo+hmAJcStwA0GjQpnx\\\\\\\/7uwojB5hq8VI4C1GfvvPNDLCoPAV2Y\\\\\\\/\\\\\\\/79uga3kuO+eDuKeg\\\\\\\/H8AE9KeAJE5dbIi9Qp1lRwUtNvXtoca2arDf6J7uVIQs6bPP1w\\\\\\\/kSblZsTi20h6eeo0mND\\\\\\\/YOzvUd3uTstJY\\\\\\\/ri22v584ZSYOBT2Da6MxKbJz7hFUTgogivaEKQU7PAXEuTfiDN8eMK9QNNuZjAps0oXpedbOFZdeYu7g+MCIwIeK2Le+KxmSv+mWwqf0byoYBAH+vMcSw3S+QhEaj2Eoo7w\\\\\\\/udGQj3EiISOmFLbo2ioDhcg969pHBV+X8R1enI1c789\\\\\\\/AcfEAXIWPlP2ivaT14kw\\\\\\\/rWSYVGcTjhB+n6K9R7eodQoHc7CMs9\\\\\\\/NNU4SiOEvbIwWND9LkgkUgz5vP5i9U\\\\\\\/YpCxZh8SCZbeske30hLYFU2YzEWGTXkKKDWVVt6iSrB\\\\\\\/h2jjXr6cciE1aIBtq3YWxReczHOJClbZLmvADbeqH8Y69sINknFS\\\\\\\/uIOdwPfcVMWLKgiGjMcuLDLV5tBu3ipJg4UIQDXDNQ9rmKp9O6ydmq9fGscJToOrHZuF8fZx6Zzjdtisr2AJXeXRlIRWQFO0d564SHTu9PRerIiduck4rvRsBWOSHA13+D9rgavxGranTZrF\\\\\\\/5H5DaAQIFyPvjpBjSOYdIISVR0JNP\\\\\\\/nji62+5jU\\\\\\\/\\\\\\\/ySlxpNdFhb0lZz6dvwt7L4g6CYy22pifIA49oUMkyW6XWlpjDVBIxewmyg\\\\\\\/f+z4NpHwOcdf+Tzb12Bfwf7bPqOdgobCqJaNlyzUic8HmnfwvstPtQFLiaf5YHmv7zns2q5Y9YSHTWBHVqTrZE4c07kF2VfdPQ\\\\\\\/8YJ6XmAXjLJid+I2mcTVw0nI+y\\\\\\\/1pIoWCAhEaYLJSa3u9BAJeUqMFlPqfJpYWAQNCL3coGrGS8WnncAGuHUqN7U+z8xaeTBRLmhPmmXXCwx1UkegFozMGFgO3nIYkB+LIjaOFiCQE3ygfBstoQ\\\\\\\/S6PA\\\\\\\/qIkd814ukqbWrHz2fcFHzMXyAYkiRujFQLhSegkRCPc6pBp1Maeo0x0Zyr+saYROH5vpyoL6b2i8bBCbulHeMf2o550mAjv26Y09lMT6mNsUawFSMtwiTV3G7dK8MM15dAW4FREJXudPeqyC3IAkzmdjxx03+fMJnAkAGrw7ZJ1WICh7WickYQmbzCUmm0+lq64Z2ixYqkx2xJ4DnRWC44us8W+vBpG9fPSIs5wfG85V5TEeTnSmin73DayafYMskLGtZ+P6YFUAzdvsi9H2wa+dlIpYSuIFPNYx+m5ZhTUQrxmq0GAwAFzFIiBXuV5dN7CP8xOVfTN2QyNuy\\\\\\\/n\\\\\\\/hckRC0BWQPvTj8cGYORhODN8DzfXgHPg3SG9ilp5RY4Uix9IM7qlkfDTVirG\\\\\\\/eTOlzj2M2DB+aKO4sdlo0Ts7x+3CpBK8eKVi2rhrBob5pvrkkTBB5dFjUOEqt5Q2WC9E5Elhcbp7oE8qYdW3XY5Rkg8Oddg3wDrO8\\\\\\\/zOSwCz99VssBeFEVnWo7xATJbTq3luL5i8TpTRC0SEmU1mgVgMjuhtGV3APbSBp8+YZAu3GT0sZe2o7NYthSVLoLcJQ3zDaF9LGVlwtRrhANThipnd7SRUGKPRQ2R5pS9cGfI74FFw7qY1gOL9c2vRG4WHLJ2XVpCzHn3D\\\\\\\/mksCXSM7rBnPBgxkmWhzMNpZ4+LBUcvHS9kZ2XeQT+sducy9nONhGgZcGcBPFxqkIbQOz+UW3jhOeWhtVuWK+16UPjQPWV5EdUScQDhldEn9jKqSNJWoy1zWE6YVvf3LrduSAXAw4Xun\\\\\\\/kcCbwqUi9sydiw2quRuREwBqZLqtJhgEcht9cFT9oyNhT8SaV6ZPmshWIfqDSQPt5Ua3w1kCIvbkK2zt6GivKrTTCjbnaNO7T8KAhQfR85m0GJGQiIXNCfq3pk6TEXLZapwYUPE1hw8Q4yqE50fLJj13z+2u42Hx6bgYx9o88B1fD7MnyHzrTVthHtsflJCAWvNrL1wx5ctx\\\\\\\/7ca3W5heWL5L1nJR+EVS4RWpa2EKFsOS6ZU6Cn5M+ANhHVEYMwWZTdMOFCA2vid7efwwMQL8pacHed7RnHu\\\\\\\/xx+j+C9r1n8dBRSVos9bXVmgQJZBHxgLgUwxPyp2SSwmdZmxBOjBK+JaCyxfaqg44sTj+n5ojRfzsR3Z64ubtgL6E\\\\\\\/\\\\\\\/5Si8TBc7J6Zy2DQviiQTb9xT1B+oJ4QcSy2FCOWl52xlounZh66xDIP9JdNmaBTN2m1HrMgAvhdVhy6i9xDCduPku0g4ellYUtQdv42nzxwE9L2CeymdXGFe+vVfW65sbvJ41+4yUiV0IueB5JGCtPxGw8qFue7WgyDDb+35D1GpyqURLIvgYqHEiWZ81MDjpkiYRKD10AG9ECOa7SZdCN1WIWXG57P9zn8YvBSRjL7iy3ODnzMeJReEkfjtnraOrOzDe7Wbi\\\\\\\/egWYos+mTSPFMo\\\\\\\/SCWp5tdS4l3sX2AAbk1EYfm5qGgcwceK4nzRgW2t8vij3VM5wBvzc+JPoiET+njzxmwoyiFVwIIv\\\\\\\/+cwOYuxvpyrOegoWCfKU+VlvqwmP3t9HmgFt1Hx44euxhUIFvJImOsji4jSQB3q9SHQErL0H\\\\\\\/KfAD38kKA+QcVxeN49wSO8yy7kdXd8Q18hdgNsZCN1+u1PhvP0RR0Y0P51fs4SvODG2Q8Sciq+7kRd+ULdbGR8KzTtO4n3Z3+GHwJ8POPwXevR7uprCZjz52iQJbyosyjiTR7DUAk7v3jWaMlRoFopQ9Pz5QtxEe7wIO1otaBCvIGbupTaWRn8wclAe8D94CnXy97\\\\\\\/fAunx66jMLEa5pkTKD8pCiqM\\\\\\\/zwaBNeOlr3JiYBH03mpzSfjt7hD\\\\\\\/0K+r3VtUz3RBT8j+tE1vlMH2i48wEVr9tTjDhTyprrA8dc2YOqQowoOFZQ8ngacehCgnbR3gOZlTRGb5quz6KI60wAYy1\\\\\\\/59dFB2zaSu3kYTlc8tWb3z7HztAuuS7nts3iCcpryAqRjonoLBa9ZUug5KAv3ZYADvSU4fnsrty1bx\\\\\\\/2FIbo0GjuZF0MfdoDZZezzdeaR7hFdhXKvOKOf3RwjpfryLo7vS2N2T7Q5xynpb5o9Mx7y3iZvuTD94uUL8ORmXZODeoz2E8YwCjGVLGoPuHJw02rIMr4wuoRQrqX8B75i33y5sP0ZP3RePNtGulxOHgMhv9wUXD5zWBFCxl8KXy6N7Nuy6+FMr41SLuC98M41f9SkYVV6wPu7Zvz1ftASDKkQsbw9saYhRHc3UesKqh\\\\\\\/o+8vCcKPtioOWOfHLBjlnU1NK5bhzD\\\\\\\/trN4ubDJrqBEHWENOuihf2YvVoupGR5v7shCDbuGa6ERimUxSnABbR5r8z7qTVmV9O4G28fuQTdfmn9fPxPzMe\\\\\\\/S4i3kLJc8DN2QKQOeowjfgChv9Bthcc9BM5un89Hnq4gyyTtjph48T5ffRYAJct9hpOAA2oytgkvc1r5MCGfhgmG4MTwGGjVkHbhmHtAB7hU8J+kCvP\\\\\\\/Iz61lx\\\\\\\/ZBgGC6ggAZAwKQFDgX+5oG0k\\\\\\\/QsAuo6cwWvF5tN9tJfzhk9xQ4Y473snRgToYILWGD0upqeLGlEgKGqcinQI9OceQh0O8MFlggZ6o\\\\\\\/9LX7FV8SOGDerEEx50+XVUL1XnNp+YYBHLNsYo0ov9TJNWUVUE9COtuaCuAk4WS2tfh0s7Qcw88kR2YTRm5vF7KaAc+WL24bkTmZoOr9mgMvcrLLLlIuML53r8vCzZrRfpC\\\\\\\/ybQciocLbgd9urOvbAWfsK17c2oKlW5B4em5E053pN5XJ+1fcAKkJziI57xmJuLsBafrx9Dad5QBij+jrtnGmUsUW1k006rJbmzIX+SGwegrfeFs\\\\\\\/43R0b0Ux7OwHosONbn+RsFKzwbCmiP+I8IFFrdIvRVHJGAYwfFSlgI6wsJBqylUsR2OY6WN7Z3f9s6Pi54saCJ3+r0B4DEnPAgIeDPNlTpX5BQnxzXBLd8AFvfL5HAQqKVJBVeUVLbYLhScdSlwKKCWIwNTb\\\\\\\/nBehGYMqNqgbawdh3FPi+z+IqCrwYK1XpD0lNmf6QSzRwUyCWK0NoJVSNHhC6ZQ5bAZEpFx7HO\\\\\\\/gt+5qsJF6Vrbre3Ud\\\\\\\/2WuDoZJuVOLqZiTir1LNsVaaJ89YLBA6zQEvsG\\\\\\\/koM23N9nMNmmxLn+jvxqPGDg\\\\\\\/3JB1KX+8+HpYvweEiZYzqOs3JOcTDNSoWRQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/NbBTWROuiIpKCuE1DNqGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228706700f768df90d9f36347659af4a68864469827a38b0c3244df47c2d87de45\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TdnhVLZUR+i0RetHDo5EOOGSDcIn7Y0i7CEtEyRgkXXcDGXsm5X\\\\\\\/OEDtmrot+Ql+5emp3Ko+gLKRIk4cvKHCmm51tyrJOZK8I7H35hDOWxcA9zes5fmhkANqtzYP+A74YC459hb\\\\\\\/1o79l0qllrGZ\\\\\\\/szmtod35U5u3R3aNdNfZGUYdjEVYy1SJOCq+bbqWwmyEmtWU9\\\\\\\/6P0mSexgxdGD6SXnXmsB+ssmi2u58WBqt4SyikgXikYqRSVonz8zelTx5SYJcE38S8fz0eb2L8lpKBHdmV0eJJZl9JwzdInM\\\\\\\/6nTSqUGDzwRbvBtaD4z0spPq\\\\\\\/lKYSiJN6Lge9kKhUBoHTA5BYhoBPfrJYeliOhWtEkncF4fDy5BVIJwXmAgHUlTWIegm7L\\\\\\\/tlNfGs9j\\\\\\\/aqJrcfdIwtCb2tc6IIC8Bane4hjTzr0psDHrfI7KaDu6u\\\\\\\/9\\\\\\\/W94CYro0fFszjktzIHEfdYGw5EmuDrsK8ped0MSA1MRoZ0QUbbFB31TfSB7n0XPfMLeko2srcV92URbXEdQB2ehNcEaTANvGXtnFVKsCXDZGzcwfKagUGq9E8zHRhaWSQstnetKlfnvz7YHrG01hQ2tU5Dt+lj42fb9aL3c2n+58sC8EGg7AlrG02eOqGlZkRXpV4HAIHV1Xj5xkVLu6DI0M8yGkyK2sdo9ZzOE6C6XN0fYI24D79JjCiCoYcvsb223rkYepjzxt26cn5vEXXauNRed9dXR7ZXJILhxRWLHfk3cR0uNDFRGYLXGyd7qJLrMSJ0bTPV4oXqThsLFtkeLbsh+OP2pwSiI\\\\\\\/Rcm1drQ0BH5AnDc\\\\\\\/MYZBuPAZE2ZGE0QyYFfIM3TudmMV\\\\\\\/gPjeUk8PrARsrA2YB07XBuN1FY8\\\\\\\/F2K1yNtt+EyH\\\\\\\/Z5MTZFJrtetBb1lE4XaJWGpDydRF+BVaiLSg2G9r+5KvvexpiTFijV7QAIqukuAofYAYvwFLumtE5bJ9\\\\\\\/i3VPQaNzZHrKp5iqv2bJiNroFYkAjMdNjFtp2x35toD8XzcNEkMGw1LtWnfcnhuX+zGvl233maykjpOR+GpU8y4FQHykR02qx7DPI5wSUSZIczeCd9c5wI2s0CrOuYYHqHxIBAzHjYwI0sI\\\\\\\/pMF53tyx\\\\\\\/RP0ANjUbQky7+RmM3hHy70yO5\\\\\\\/ODNvGzZDf7AuWSyFI9ARcttvtrCxn8cOgcnCz0OAruJf4ReLrKOmvitp91I0bO4\\\\\\\/0I8au+Pzg5WkHeXK0ha8VuHugstrdZPduh0Ct1yM0WCtOxiGAvZn+Ene+Q8ViPgPFFbYB1X5zH6qXiveFeeo4MBOtYKtC2yQDXj5J7FWo8nF9AN0piByFXGRhAzlsPuJsw7Mg0ThfhVsWZ8kiv1vAg8YmQROQkrZMIH2CYYoe1zOAzLpVj51yFf3C6DiGY9Jk7O4TCgveYj7U7PpxghQh57j+KaxIVcpT5V3SOFOAWa0YcwP17+EmCWzVpX0zZ17Bfb\\\\\\\/+pv4Shh9MGMWG8UBJ3neAc+Oq8IPpFY3+LoGTk7J5cZuOdAmH0s8EZeyFlF0IWve48zkKpqZjVdLwlwC1kB+cwpF0gJ\\\\\\\/G\\\\\\\/USWxuoFlY12xXobhUNQWpiuSFmXFBLRFAgQQ3oeafymeDjWlYJ77N0SY0QXds3h9EZBByr3O6uQfXTF50Ggh2NCSuPVeitP3h1A4MltS9NlXW2KLjc0iR\\\\\\\/AZtijKmBGWOWvC1sL+OAL2JxnaFwdUfQiS907GzCJqknrLiD+FJZVoahtj75KVi+RC7qsd2S29yOL4q2XEPHsQcLTRmd8UeP9mV4d89fiXptdDYERhpJdbDOlB0lAFz5FnGJimqvwYmNQUJKsCPjFGaUlaV1YHPtQbYhO6ruD0Zg68UE7WoW09EwuS7AdPUn0\\\\\\\/xpJta+mlwEMTNwItropjK85Zdf34gCYOQwtbjVYR7IJOa8bGXGH6pDbTOAO676eeF3ElThyVqoJiWqX7iq2f83Mo+\\\\\\\/yNWEc9H3NO8N4ST1rgr+4FA9WqvrcZpDAdDg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kc37uDSDivBM7qCgu1vufA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c065c913b41b5faa1cd85408ae863d22c12c5d805403df05f0d6660fa02cb1b7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491667 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3903" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OaODChq7u6v1u0nOv53fIJuPLnneM0EAcEJB-vK6A7I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3903" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-26hcly3EwxKHPNHbYvZHDcNRulzTijnyuP3iAPdKDBU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3891" title="Genetic Contributions to Ectopic Sperm Cell Migration in Caenorhabditis Nematodes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3915" title="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491667" id="node491667" data-pisa="ggg;8/12/3903" data-pisa-master="ggg;g3.118.200710" data-apath="/ggg/8/12/3903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alfred Ozimati</span>, <span class="highwire-citation-author" data-delta="1">Robert Kawuki</span>, <span class="highwire-citation-author" data-delta="2">Williams Esuma</span>, <span class="highwire-citation-author" data-delta="3">Ismail Siraj Kayondo</span>, <span class="highwire-citation-author" data-delta="4">Marnin Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author" data-delta="6">Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3903-3913;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491667" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alfred Ozimati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alfred%2BOzimati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozimati%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlfred%2BOzimati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alfred Ozimati</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ozimatialfred@gmail.com" class="" data-icon-position="" data-hide-link-title="0">ozimatialfred@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Kawuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKawuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawuki%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BKawuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Williams Esuma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Williams%2BEsuma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esuma%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliams%2BEsuma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ismail Siraj Kayondo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BSiraj%2BKayondo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kayondo%20IS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BSiraj%2BKayondo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marnin Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarnin%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Department of Agriculture, Agricultural Research Service (USDA-ARS) R.W. Holley Center for Agriculture and Health, Ithaca 14853, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3903" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491667" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3903.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491667" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3903.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491667" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3903.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491667" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3903.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491667" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491667/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cassava production in the central, southern and eastern parts of Africa is under threat by cassava brown streak virus (CBSV). Yield losses of up to 100% occur in cases of severe infections of edible roots. Easy illegal movement of planting materials across African countries, and long-range movement of the virus vector (<em>Bemisia tabaci</em>) may facilitate spread of CBSV to West Africa. Thus, effort to pre-emptively breed for CBSD resistance in W. Africa is critical. Genomic selection (GS) has become the main approach for cassava breeding, as costs of genotyping per sample have declined. Using phenotypic and genotypic data (genotyping-by-sequencing), followed by imputation to whole genome sequence (WGS) for 922 clones from National Crops Resources Research Institute, Namulonge, Uganda as a training population (TP), we predicted CBSD symptoms for 35 genotyped W. African clones, evaluated in Uganda. The highest prediction accuracy (r = 0.44) was observed for cassava brown streak disease severity scored at three months (CBSD3s) in the W. African clones using WGS-imputed markers. Optimized TPs gave higher prediction accuracies for CBSD3s and CBSD6s than random TPs of the same size. Inclusion of CBSD QTL chromosome markers as kernels, increased prediction accuracies for CBSD3s and CBSD6s. Similarly, WGS imputation of markers increased prediction accuracies for CBSD3s and for cassava brown streak disease root severity (CBSDRs), but not for CBSD6s. Based on these results we recommend TP optimization, inclusion of CBSD QTL markers in genomic prediction models, and the use of high-density (WGS-imputed) markers for CBSD predictions across population.</p></div><span class="kwd-group-title">Key words:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/cassava" class="hw-term hw-article-keyword hw-article-keyword-cassava" rel="nofollow">Cassava</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/training-population" class="hw-term hw-article-keyword hw-article-keyword-training-population" rel="nofollow">training population</a></li><li class="kwd"><a href="/keyword/and-cassava-brown-streak-disease" class="hw-term hw-article-keyword hw-article-keyword-and-cassava-brown-streak-disease" rel="nofollow">and cassava brown streak disease</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Cassava (<em>Manihot esculenta</em> Crantz) is ranked the fourth most important source of calories in the developing world, after wheat, maize, and rice, and is estimated to feed a population of about 700 million people directly or indirectly (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Legg <em>et al.</em>, 2014</a>). Reports on global cassava production in the 1960s positioned Brazil as the leading producer in the world, however in the 1990s Nigeria became the world’s largest cassava producer, accounting for half of the world’s total production (Nweke 2004). Other African countries where cassava is a major staple food crop include Uganda, Tanzania and Kenya in eastern Africa, Malawi and Mozambique in southern Africa, Democratic Republic of Congo (DRC) in central Africa, and Ghana in western Africa (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hillocks and Jennings 2003</a>). Cassava is popular in Africa as a food security crop, because of its resilience under drought and poor soils, and its ability to be easily propagated through stem cuttings (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Masona <em>et al.</em>, 2001</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Legg <em>et al.</em>, 2014</a>).</p><p id="p-4">Yields of cassava have remained low (8-12 tons/ha) in Africa compared to Asian countries such as Thailand and Vietnam where yield averaged are up to 20 tons/ha (Nweke 2004). Reasons for relatively low yields in Africa include both abiotic (low soil fertility and socio-economic factors such as lack of access to improved varieties) and biotic factors (Nweke 2004). The most devastating biotic stresses today are the cassava brown streak (CBSD) and cassava mosaic (CMD) diseases (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maruthi <em>et al.</em>, 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mware <em>et al.</em>, 2009</a>). Of these two virus-induced diseases, CBSD is the most important constraint to cassava production in central, eastern and southern Africa as it causes yield losses of up to 100% (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alicai <em>et al.</em>, 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hillocks <em>et al.</em>, 2016</a>).</p><p id="p-5">Phylogenetic analysis of complete viral RNA genome sequences taken from CBSD symptomatic plants, sampled across eastern and southern Africa, revealed two clades of distinct CBSD-causing virus species that were named: Uganda cassava brown streak virus (UCBSV) and cassava brown streak virus (CBSV) (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Winter <em>et al.</em>, 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mohammed <em>et al.</em>, 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Patil <em>et al.</em>, 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alicai <em>et al.</em>, 2016</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mbewe <em>et al.</em>, 2017</a>). The two species belong to genus <em>Ipomovirus</em> within the family of <em>Potyviridae</em>, and share an identity of 70% and 74% at the level of nucleotide and polyprotein amino acid sequences, respectively (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Monger <em>et al.</em>, 2001</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Winter <em>et al.</em>, 2010</a>). Cassava brown streak disease symptoms on cassava leaves manifest as feathery chlorosis around secondary veins, which may disappear when new growth starts after a period of drought-induced leaf abscission (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hillocks 2004</a>). While on the roots, CBSD symptoms externally present as radial constriction, and internally as brown necrotic lesions on part or all of the starchy root, making it inedible (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hillocks 2004</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hillocks <em>et al.</em>, 2016</a>).</p><p id="p-6">Although the first incidence of CBSD was reported in 1930s (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Storey and Nichols 1938</a>), little attention was paid to it, because geographically CBSD was confined to the low altitudes of east African coastal region (less than 1000 m.a.s.l). Nonetheless, CBSD has spread rapidly to other countries including; Uganda, Burundi, DRC, Mozambique and Rwanda in the last two decades to cover wider range of altitudes than previously reported (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hillocks <em>et al.</em>, 2002</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Alicai <em>et al.</em>, 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Legg <em>et al.</em>, 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mulimbi <em>et al.</em>, 2012</a>). Cassava brown streak disease is commonly spread through sharing of infected stem cuttings for propagation, in addition to super-abundant whitefly <em>Bemisia tabaci</em>, as a vector (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hillocks and Jennings 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Njoroge <em>et al.</em>, 2017</a>).</p><p id="p-7">Officially, genetic materials can move from W. Africa to E. Africa, but movement in the reverse direction is prohibited to prevent accidental introduction of CBSD-causing viruses in W. Africa. Nevertheless, the free movement of planting materials across farming communities has led to increased fear that CBSD could spread to other regions, including West Africa (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Legg <em>et al.</em>, 2014</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Patil <em>et al.</em>, 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em>, 2017</a>). Given the current impact of CBSD on cassava production in endemic countries, effort needs to be in place to avert or minimize future CBSD impact in W. Africa, especially Nigeria the world’s leading cassava producer. Among other methods, <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Legg <em>et al.</em>, (2014)</a> proposed pre-emptive breeding for CBSD resistant clones in W. Africa.</p><p id="p-8">High levels of field resistance to CBSD have been reported from genetically transformed plants with coat protein of UCBSV and CBSV, compared to non-transformed plants (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ogwok <em>et al.</em>, 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Odipio <em>et al.</em>, 2014</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beyene <em>et al.</em>, 2017</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wagaba <em>et al.</em>, 2017</a>). However, the transgenic CBSD resistant clones are still within research confinement, because of unclear regulatory frameworks regarding field production of genetically modified organisms (GMO) in Uganda and east Africa at large. Other efforts to breed for CBSD resistance in E. Africa are geared toward identification of quantitative trait loci (QTL) for CBSD resistance, with the aim of developing molecular markers to implement marker-assisted selection (MAS). A number of QTL mapping studies for CBSD resistance in E. African germplasm have been conducted, and the studies pointed out both unique and overlapping QTL regions for which markers could be developed for MAS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Masumba <em>et al.</em>, 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nzuki <em>et al.</em>, 2017</a>). One of the highest effect QTL detected involved a bi-parental mapping population from a cross between Kiroba and AR37-80 that explained 18% of total phenotypic variance (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Nzuki <em>et al.</em>, 2017</a>). However, using bi-parental QTL to develop markers for MAS is only feasible if the QTL are validated in other breeding populations. Furthermore, the recent genome-wide association studies conducted by Kayondo <em>et al.</em> (2018), using same training populations, confirmed the polygenic nature of CBSD resistance previously reported (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kawuki <em>et al.</em>, 2016</a>).</p><p id="p-9">Genomic selection, proposed by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> (2001)</a> provides an option for using DNA markers for traits that are truly quantitative, where no single causal locus accounts for a major fraction of the variation for selection decisions. Genomic selection (GS) relies on a genome-wide distribution of markers to ensure all QTL have at least one marker in high LD, enabling selection on highly polygenic traits. Genomic selection is typically done using a phenotyped and genotyped training population to estimate genome-wide marker effects (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em>, 2009</a>). The genomic estimated breeding values (GEBV) for all genotyped individuals can then be computed as the sum of marker effects multiplied by the marker genotypes across the whole genome (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Meuwissen, <em>et al.</em>, 2001</a>). These GEBVs aim to capture all QTL accounting for variation in target traits (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hayes <em>et al.</em>, 2009</a>).</p><p id="p-10">Although GS has reportedly outperformed traditional selection methods such MAS and marker assisted recurrent selection (MARS) for quantitative traits (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Goiffon 2016</a>), successful implementation of genomic selection depends on a number of factors including: trait heritability, marker density, the size of the training population, the relationship between the training population (TP) and the selection candidates (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jannink <em>et al.</em>, 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heffner <em>et al.</em>, 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nielsen <em>et al.</em>, 2016</a>).</p><p id="p-11">Increases in prediction accuracy have been reported by composing training populations from optimal subsets of individuals chosen to minimize the expected prediction error variance (PEV) of the selection candidates compared to using random subsets or even the full set of available individuals (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rincent <em>et al.</em>, 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em>, 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Isidro <em>et al.</em>, 2015</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yu <em>et al.</em>, 2016</a>). Furthermore, studies have shown increased prediction accuracies with inclusion of prior QTL information in genomic prediction models. For example, a study by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Haffliger, (2016)</a> for reproductive traits in Swiss pig breeds revealed a significant increase in prediction accuracy for piglets when previously detected reproductive trait QTL markers were included in the prediction model.</p><p id="p-12">Thus, this study aimed to evaluate the use of genomic predictions of West African clones using training data from a Ugandan population as a pre-emptive breeding strategy for CBSD resistance. Specifically we tested CBSD prediction accuracies for (i) different sizes of training populations across genomic prediction models (ii) random and optimized training sets, (iii) models with incorporation of prior CBSD QTL, and (iv) high and low density marker panels.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Constitution and evaluation of training population</h3><p id="p-13">The training population comprised 922 clones, combined from two experimental trials. For consistency, we refer to the trials as training population 1 (TP1) and training population 2 (TP2). A total of 400 clones (TP1) were generated from crossing diverse parents that were assembled from International Center for Tropical Agriculture (CIAT), International Institute of Tropical Agriculture (IITA), Tanzania, and National Crops Resources Research Institute (NaCRRI), Uganda. The introductions from CIAT targeted improvement for quality and yield traits, while the germplasm from the IITA, Tanzania and NaCRRI breeding programs targeted resistance to CBSD.</p><p id="p-14">Crosses were made among the progenitors to generate TP1 in 2009-2010, from which both controlled crosses and open-pollinated seeds were harvested. After seedling evaluation, the first clonal evaluation for TP1 was done at Namulonge in 2012-2013 by conducting an un-replicated experiment, and afterward expanded to three sites (Kasese, Ngetta and Namuloge) for the second year of clonal evaluation, planted in alpha lattice design, in single row plots of 10 plants, replicated twice.</p><p id="p-15">The second training population (TP2) comprised 522 clones, generated from open-pollinated seeds that were harvested from the first clonal evaluation trial of TP1. Similar to TP1, after a year of seedling evaluation (2013-2014), TP2 was planted for the first clonal evaluations in 2014 at two sites (Namulonge and Kamuli). In 2015, TP2 was replanted for the second year of clonal evaluation, with the trials expanded to three sites (Namulonge, Kamuli, Serere). Thus, Namulonge was the only overlapping evaluation site between TP1 and TP2. The clonal evaluations for TP2 were established in an augmented incomplete block design with six common checks per block, and each plot within a block containing 10 plants established in a single row. Planting of all the trials was done at spacing of 1 m × 1 m adopted within and between rows, while blocks were separated by 2 m alleys.</p><p id="p-16">Data on foliar CBSD severity was collected at three and six months after planting (MAP), while the roots were evaluated for CBSD severity at 12 MAP. Foliar severity for CBSD was assessed on a scale of 1-5 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hillocks and Thresh 2000</a>), where: 1 = no symptom; 2 = slight foliar chlorotic leaf mottle with no stem lesions; 3 = foliar chlorotic leaf mottle and blotches with mild stem lesions, but no die back; 4 = foliar chlorotic leaf mottle and blotches with pronounced stem lesions, but no die back; and 5 = defoliation with stem lesions and dieback. To assess root necrosis severity, each root was sliced transversely 5-7 times and the cross-sections scored for necrotic symptoms on a scale of 1-5 (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hillocks and Thresh 2000</a>), where: 1 = no necrosis, 2 = ≤ 5% necrotic; 3 = 6–10% necrotic; 4 = 11–25% necrotic and mild root constriction; and 5 = &gt;25% necrotic and severe root constriction.</p></div><div id="sec-3" class="subsection"><h3>West African genetic materials and evaluation</h3><p id="p-17">In 2015, we received a total of 95 clones that constituted part of IITA, Nigeria genetic gain population for implementing genomic selection (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wolfe <em>et al.</em>, 2017</a>). These clones were shipped to Uganda in the form of tissue culture plantlets. The first set of 30 clones was received in February 2015 and the second lot of 65 clones was received in June 2015. The plantlets were multiplied in tissue culture and further hardened in a screen house for three months.</p><p id="p-18">In August and November 2015, the first set of 30 and the second set of 65 clones were planted in the field at Namulonge. This was done to generate adequate stem cuttings for establishment of replicated field trials. In September 2016, we established a trial for the first set of 27 clones that survived, in a randomized complete block design (RCBD) replicated twice, with each plot containing 10 plants in a single row.</p><p id="p-19">For the second set of 65 clones, unfortunately we lost more than half of the clones due to drought that occurred a month after their first field exposure in 2015. The remaining 22 clones that survived were planted in November 2016, again using an RCBD, replicated twice. In contrast to the first set of 27 clones, there was only enough planting material for five plants per plot. Cassava brown streak disease phenotyping was conducted as described previously. All infections occurred under natural conditions.</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and genotyping</h3><p id="p-20">Approximately 100 mg of fresh tissue was collected from tender apical leaves of TP1 and TP2 clones for DNA extraction. DNA was extracted following the protocol for the QIAGEN DNeasy extraction kit and quantified using the PicoGreen DNA quantification kit to ensure the required concentrations were obtained for sequencing. The extracted DNA samples were shipped to the Cornell University Genomics Diversity Facility for genotyping, using the genotyping-by-sequencing (GBS) approach (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em>, 2011</a>). The GBS libraries were constructed using the ApeKI restriction enzyme as described previously (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rabbi <em>et al.</em>, 2014</a>).</p><p id="p-21">Marker genotypes were called using TASSEL GBS pipeline v4 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Glaubitz <em>et al.</em>, 2014</a>), after aligning the reads to the Cassava reference genome v6 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Prochnik <em>et al.</em>, 2012</a>). Using VCFtools, Variant Calling Format (VCF) files were generated for each chromosome. Genotypes with less than five reads were masked before imputation. Similarly, markers with more than 60% missing calls were removed. Only bi-allelic GBS SNP markers were considered for further processing. Missing markers were imputed using Beagle 4.1 software (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2016</a>), with default parameter settings. In all, 46,760 SNPs remained, which we referred to as the “GBS” markers.</p><p id="p-22">In addition, we used a second set of markers, which we referred to as “whole genome sequence” imputed (or “WGS-imputed”) markers. Using IMPUTE2 software, the GBS samples were imputed to a marker set equivalent to what we would get from actual whole genome sequencing. The details of the imputation procedure are described by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lozano <em>et al.</em>, (2017)</a>. Briefly, the WGS imputation relied on the cassava HapMapII, a collection of 241 genome-sequenced samples (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ramu <em>et al.</em>, 2017</a>) as a reference panel. This reference panel comprised mainly of improved cassava clones under cultivation and a few wild relatives, and contained ∼28 million SNP markers (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lozano <em>et al.</em>, 2017</a>). The set of markers referred to as “WGS-imputed” hereafter, included ∼5 million SNPs, after filtering for minor allele frequencies (MAF) ≥ 0.01.</p></div><div id="sec-5" class="subsection"><h3>Statistical analyses</h3><div id="sec-6" class="subsection"><h4>Analyses of phenotypic data:</h4><p id="p-23">Because of differences in trial design for TP1 and TP2 as well as the IITA clones, two-step genomic prediction analyses were done. In the first step of the analyses, linear mixed models accounting for each trial’s design were fitted and de-regressed BLUPs were obtained for TP1 and TP2. For TP1, we fitted the model:<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-1.gif"/></noscript></span></span>, using the <em>lmer</em> function from the <em>lme4</em> R package (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">R Development Core Team 2008</a>). In this model, <strong><em>β</em></strong> defined the fixed effect for the population mean and location, with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-2.gif"/></noscript></span></span>as the corresponding incidence matrix. The incidence matrix <strong>Z<sub>clone</sub></strong> and the vector <em>c</em> represented random effect for clones <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-3.gif"/></noscript></span></span>, and I represented the identity matrix. The range variable, which was the row or column along which plots were arrayed, was nested in location-replication and was represented by the incidence matrix <strong>Z<sub>range(loc.)</sub></strong> and random effects vector <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-4.gif"/></noscript></span></span>. Block effects were nested in ranges and incorporated as random term with incidence matrix <strong>Z<sub>block(range)</sub></strong> and effects vector <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-5.gif"/></noscript></span></span>. Residuals ε were distributed as<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-6.gif"/></noscript></span></span>.</p><p id="p-24">For TP2 (522 clones), we fitted the linear mixed model <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-7.gif"/></noscript></span></span>, where <em>y</em> was the vector of raw phenotypes, <strong><em>β</em></strong> included a fixed effect for the population mean and location with checks included as a covariate. The incidence matrix <strong>Z<sub>clone</sub></strong> and the vector <em>c</em> were similar for both TP1 and TP2. The blocks were also modeled with incidence matrix<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-8.gif"/></noscript></span></span>, and <strong><em>b</em></strong> represented the random effect for the blocks. The best linear unbiased predictors (BLUPs) of the clone effects were extracted as de-regressed BLUPs following the formula proposed by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Garrick <em>et al.</em>, (2009)</a>.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-9.gif"/></noscript></span></span>Here, PEV represented the prediction error variances for the BLUPs and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-10.gif"/></noscript></span></span> was the clone variance.</p><p id="p-25">For the IITA clones, we fitted the following mixed model:<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-11.gif"/></noscript></span></span>. Where <em>y</em> was a vector of raw phenotypes, <strong><em>β</em></strong> included a fixed effect for the population mean. The incidence matrix <strong>Z<sub>clone</sub></strong> and the vector <em>c</em> represented random effect for clones <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-12.gif"/></noscript></span></span> and I represented the identity matrix, and the replication nested in the trials was modeled with incidence matrix<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-13.gif"/></noscript></span></span>, with random effect <strong><em>b</em></strong> representing replications nested within trial for the first set of 27 and second set of 22 clones, with 14 overlapping clones between the sets. The best linear unbiased predictors (BLUPs) were extracted from the model and subsequently used as the validation data for estimation of genomic prediction accuracies of CBSD for the 35 unique IITA clones. In addition, variance components were extracted from the model to compute plot based broad-sense heritability estimates.</p></div><div id="sec-7" class="subsection"><h4>Population structure:</h4><p id="p-26">To assess population structure, we used the GBS markers of TP1, TP2, and the 35 IITA clones. These markers were filtered to have MAF ≥ 0.01 and formatted as a dosage matrix with SNP genotypes coded as -1, 0, or +1. Principal component analysis (PCA) was done on the SNP matrix, using the <em>prcomp</em> function in R. The first two principal components (PC) were used to visualize population structure.</p></div><div id="sec-8" class="subsection"><h4>Cross-validation prediction accuracies for IITA clones:</h4><p id="p-27">Weestimated prediction accuracies for foliar CBSD severities evaluated at three (CBSD3s) and six (CBSD6s) months, and root severity at 12 months (CBSDRs), using a fivefold cross validation scheme, replicated 10 times for IITA clones from a single-step genomic best linear unbiased predictor (G-BLUP) model. For each replication in the cross-validation scheme, the 35 IITA clones were randomly divided into five groups of 7 clones each (folds). Four groups at a time were used as the training population to build the prediction model, while excluding the fifth group, which was used as the model validation set. This was repeated for all the 5-folds for each of the 10 replications. Prediction accuracies were computed as the Pearson correlation coefficient between the genomic estimated breeding values predicted for the validation set and the corresponding BLUPs obtained from the first-step of the analysis for 35 unique individuals in the test set (IITA clones).</p></div><div id="sec-9" class="subsection"><h4>Genomic prediction of CBSD for IITA clones:</h4><p id="p-28">We tested genomic prediction accuracies under four scenarios: (i) optimized training populations across genomic selection models (ii) optimized <em>vs.</em> random subset training populations for G-BLUP only (iii) models with inclusion of kernels defined by chromosomes on which CBSD QTL have been found (single and multi-kernel G-BLUP models), and (iv) high and low density marker panels for G-BLUP model.</p><p id="p-29">To optimize the training population, we used the selection of training population with a genetic algorithm (STPGA), <em>GenAlgForSubsetSelection</em>, from the R package STPGA (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em>, 2015</a>). The algorithm identifies a subset of a specified size from a larger pool of potential training individuals. To do this, STPGA finds the set of individuals that minimize the mean prediction error variance (mean PEV) expected for test set, using molecular marker data.</p><p id="p-30">For STPGA training population optimization, we used the first 50 principal components (PC’s) of the eigenvalue decomposition of the marker matrix as a dimension reduction approach. The pool of potential training individuals was the combined TP1 and TP2 (N = 922) described above and the target or test set were the IITA clones. We optimized 20 training populations within each size of training population specified in STPGA. In scenario (i), the optimum training populations for each training population size (100, 200, 400, 800 and full set = 922) were used to predict CBSD with four genomic prediction models namely; G-BLUP, Bayes-A, Bayes-B and Bayesian Lasso (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lorenz <em>et al.</em>, 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em>, 2012</a>).</p><p id="p-31">Under scenario (ii), we tested the performance of STPGA by comparing the optimized sets from scenario (i) with random subsets of the same size. We chose to compare optimized and random sets for population of sizes of 200 and 400, based on results from analyses in scenario (i), and for each training size we compared 20 sets for both random and optimized TPs, using G-BLUP model because of it robustness and computational efficiency.</p><p id="p-32">In the single kernel model, all GBS markers were fitted with one realized genomic relationship matrix K, according to the formula described by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">VanRaden, (2008)</a>. The relationship matrix was constructed using <em>A.mat</em> function in rrBLUP package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Endelman 2011</a>). The model was specified as: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-14.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-16.gif"/></noscript></span></span>, where y was the vector of de-regressed BLUPs, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-17.gif"/></noscript></span></span> was an overall population mean, Z was the design matrix linking observations to genomic values, g was the vector of genomic estimated breeding values for each clone, and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-18.gif"/></noscript></span></span>was the vector of residuals. We assumed, <em>g</em> had a known covariance structure defined by the realized genomic relationship matrix <em>K</em>.</p><p id="p-33">Previously, QTL for CBSD have been reported on chromosomes 4 and 11 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kawuki <em>et al.</em>, 2016</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>). We used all the markers on the two chromosomes (Chr.4 and 11) because significant markers covered essentially the whole of Chr. 4 for CBSD6s and about half of Chr. 11 for both CBSD3s and CBSD6s (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>). Therefore, we also fitted a multi-kernel G-BLUP model with two realized genomic relationship matrices, constructed using <em>A.mat</em> function as described above. In this model, the first genomic relationship matrix incorporated all markers from both chromosome 4 and 11, while the second genomic relationship matrix was derived from the rest of the genomic markers. The model was: y = <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-19.gif"/></noscript></span></span>. Here, y was the vector of de-regressed BLUPs, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-20.gif"/></noscript></span></span> was an overall mean, <em>Z</em> was the design matrix linking observations to genomic values, <em>q</em> was the vector of genomic values captured by combined QTL markers linked to CBSD resistance, <em>r</em> was the vector of genomic values captured by the remaining set of genetic markers, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-21.gif"/></noscript></span></span>was a vector of residuals. The random genetic effects for both kernels with their variance-covariance structure <em>K</em>, and the residuals were assumed to be normally distributed as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-22.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-24.gif"/></noscript></span></span>.</p><p id="p-34">Furthermore, we fitted a multi-kernel G-BLUP model with three genomic relationship matrices, where the first and second realized genomic relationship matrices were defined by all the markers on chromosomes 4 and 11 respectively, while the third kernel contained markers from the remaining 16 chromosomes. The model was: y = <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-25.gif"/></noscript></span></span>. Here, y was the vector of de-regressed BLUPs, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-26.gif"/></noscript></span></span> was an overall mean, <em>Z</em> was the design matrix linking observations to genomic values, p and s were the vectors of genomic values captured by QTL markers on chromosome 4 and 11 respectively, <em>r</em> was the vector of genomic values captured by the remaining set of genetic markers, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-27.gif"/></noscript></span></span>was the vector of residuals. The random effects, including the residual-term were assumed to be normally distributed as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-28.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-29.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3903/embed/mml-math-31.gif"/></noscript></span></span>.</p><p id="p-35">For both single and multi-kernel G-BLUP analyses, we used the two EMMREML functions, <em>emmreml</em> and <em>emmremlMultiKernel</em> to fit single and multi-kernel G-BLUP models respectively (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Okeke 2015</a>). Lastly, we tested prediction accuracies of CBSD3s, CBSD6s and CBSDRs using high-density (WGS-imputed) markers and compared that to low-density (GBS) markers used in the analyses described above. For the high-density set, we fitted the single- and multi-kernel G-BLUP models described above, using training populations of 200 and 400 clones that were optimized using either the GBS or the WGS markers. Because the results of these two optimizations were quite similar, we only reported results from the GBS optimizations.</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-36">All the raw phenotypic and genotypic data are available at the link provided for references. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7242851">https://doi.org/10.25387/g3.7242851</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Population structure, heritability and cross-validation within IITA clones</h3><p id="p-37">Principal component analyses on the SNP marker matrix showed no genetic differentiation among the TP1, TP2 and IITA clones. This was supported by PC1 and PC2 explaining only 8.75% and 5.69% of the total genetic variations, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Figure S6). Estimates of plot-basis broad-sense heritability (H<sup>2</sup>) were computed for CBSD3s, CBSD6s and CBSDRs for the 35 IITA clones (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Broad-sense heritability estimates spanned from 0.42 to 0.64 for CBSD3s and CBSDRs respectively. In addition to broad-sense heritability, we estimated narrow-sense heritability for IITA clones using a single step G-BLUP model.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of PC1 against PC2 for Eigen value decomposition of GBS markers for IITA (green), NaCRRI-TP1 (black) and NaCRRI-TP2 (red) clone." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2066890317" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of PC1 against PC2 for Eigen value decomposition of GBS markers for IITA (green), NaCRRI-TP1 (black) and NaCRRI-TP2 (red) clone.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3903/F1.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3903/F1.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491556" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-38" class="first-child">Plot of PC1 against PC2 for Eigen value decomposition of GBS markers for IITA (green), NaCRRI-TP1 (black) and NaCRRI-TP2 (red) clone.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491672/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491672/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Variance component and plot-basis heritability estimates for IITA clones</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">The lowest and highest narrow-sense heritability of 0.35 and 0.69 were recorded for CBSD3s and CBSDRs, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The average prediction accuracies from fivefold cross-validation replicated 10 times for the IITA clones were 0.40, 0.21 and 0.08 for CBSD3s, CBSD6s and CBSDRs, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). We did not do cross validation within the training set here, because the training population was previously cross-validated (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>). Previous predictive accuracy for CBSD-related traits, had mean values across methods of 0.29 (CBSD3s), 0.40 (CBSD6s) and 0.34 (CBSDRs) for cross-validation within NaCRRI training set.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies for fivefold and 10 reps using G-BLUP model, and SNP-heritability estimates for CBSD3s, CBSD6s and CBSDRs in 35 IITA clones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2066890317" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracies for fivefold and 10 reps using G-BLUP model, and SNP-heritability estimates for CBSD3s, CBSD6s and CBSDRs in 35 IITA clones.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3903/F2.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3903/F2.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Prediction accuracies for fivefold and 10 reps using G-BLUP model, and SNP-heritability estimates for CBSD3s, CBSD6s and CBSDRs in 35 IITA clones.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Predicting CBSD in IITA clones using Ugandan training population</h3><p id="p-42">In general, the mean CBSD prediction accuracies were higher for foliar than root necrosis for the different optimized training population sizes across genomic prediction models (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For CBSD3s, the prediction accuracies ranged from 0.24 (Bayes-A) to 0.36 (Bayesian Lasso). Prediction accuracies spanned from 0.14 (Bayesian Lasso) to 0.28 (G-BLUP) for CBSD6s. For CBSDRs, accuracies included negative values, ranging from -0.29 (Bayes-A) to 0.11 (Bayes-A) across different optimized training sets.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491696/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491696/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average prediction accuracies (r) for four optimized subsets of TPs and full set across genomic prediction models</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">The models did not differ much in terms of their prediction accuracies for three traits (CBSD3s, CBSD6s and CBSDRs) across optimized training populations of 100, 200, 400, 800, and full set of 922 clones. Surprisingly, Bayesian Lasso consistently had higher prediction accuracies than the other three prediction models (G-BLUP, Bayes-A and Bayes-B) for CBSD3s across the optimized TP sizes, but performed worse than those three models for CBSD6s across optimized TPs (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-45">Prediction accuracies for optimized training populations across the four models tested increased from 100 to 400 for CBSD3s to attain a plateau and declined as the optimized training population was increased to 800 and the full set of 922 clones. However, no clear trend in prediction accuracies were observed for CBSDRs for the different sizes of optimized training population (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-46">We compared CBSD prediction accuracies from random and optimized training populations of size 200 and 400 clones using the G-BLUP model. We chose these two sample sizes because they maximized prediction accuracies for CBSD3s and CBSD6s (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). For both 200 and 400 clones, the prediction accuracies were higher for optimized training sets than for the random subsets for CBSD3s and CBSD6s (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; Table S6). For example, at training population size of 200, the mean prediction accuracies for CBSD3s and CBSD6s were 0.27 and 0.28 compared to 0.11 and -0.01 for the corresponding random subsets. Similarly, at training population size of 400 clones, the mean prediction accuracies for CBSD3s and CBSD6s were 0.32 and 0.19 relative to 0.10 and 0.04 for the random subsets (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>; Table S7). We observed markedly lower standard errors as measures of variation in prediction accuracies across the traits for the optimized training populations, compared to the random subsets (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). However, no strong differences were observed for CBSDRs (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies and the standard error bars for 20 replications of optimized and random training population size of 200 and 400." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2066890317" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracies and the standard error bars for 20 replications of optimized and random training population size of 200 and 400.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3903/F3.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3903/F3.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491578" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-47" class="first-child">Prediction accuracies and the standard error bars for 20 replications of optimized and random training population size of 200 and 400.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Accounting for CBSD QTL with chromosome-specific effects or kernels</h3><p id="p-48">In general, foliar CBSD prediction accuracies for training population size of 200 and 400 were higher for multi-kernel models (K_2 and K_3) with separate kernels fitted for CBSD QTL chromosome markers than single kernel (K_1) G-BLUP models (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Prediction accuracies for CBSD3s increased from 0.27 for the single kernel G-BLUP, termed as “K_1” model to 0.31 for two-kernel G-BLUP model referred to as “K_2”, and to 0.32 for the three-kernel model referred to as “K_3” in the optimized TPs of 200 clones (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; Table S8). Similarly, for CBSD6s, prediction accuracies increased from 0.28 for single kernel G-BLUP model to 0.37 with three-kernels (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>; Table S8). No such increase was observed for CBSDRs. Notably, the mean prediction accuracies for CBSD3s and CBSD6s from multi-kernel G-BLUP models were statistically significantly different (<em>P</em> ≤ 0.05) from zero. Nevertheless, no differences were observed for CBSDRs prediction accuracies between single- and multi-kernel G-BLUP models at TP size of 200.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G-BLUP model to compare prediction accuracies for varying number of kernels for CBSD measured at 3, 6 and 12 MAP for size of TP 400 and 200. K_1= Single kernel G-BLUP model. K_2= Multi-kernel G-BLUP, the first kernel is defined by combined markers from chromosomes 4 and 11. The second kernel is defined by the remaining markers. K_3= Multi-kernel G-BLUP, the first and second kernels are defined by markers from chromosomes 4 and 11 respectively, and third kernel is defined by the remaining markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2066890317" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;G-BLUP model to compare prediction accuracies for varying number of kernels for CBSD measured at 3, 6 and 12 MAP for size of TP 400 and 200. K_1= Single kernel G-BLUP model. K_2= Multi-kernel G-BLUP, the first kernel is defined by combined markers from chromosomes 4 and 11. The second kernel is defined by the remaining markers. K_3= Multi-kernel G-BLUP, the first and second kernels are defined by markers from chromosomes 4 and 11 respectively, and third kernel is defined by the remaining markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3903/F4.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3903/F4.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491550" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-49" class="first-child">G-BLUP model to compare prediction accuracies for varying number of kernels for CBSD measured at 3, 6 and 12 MAP for size of TP 400 and 200. K_1= Single kernel G-BLUP model. K_2= Multi-kernel G-BLUP, the first kernel is defined by combined markers from chromosomes 4 and 11. The second kernel is defined by the remaining markers. K_3= Multi-kernel G-BLUP, the first and second kernels are defined by markers from chromosomes 4 and 11 respectively, and third kernel is defined by the remaining markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">For the optimized training population size of 400, a similar trend of increased prediction accuracies was observed from single- to multi-kernel G-BLUP models for both CBSD3s and CBSD6s. The mean prediction accuracies for CBSD6s were not significantly different from zero. Prediction accuracies did not vary much for CBSDRs between single- and multi-kernel G-BLUP models (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>; Table S9).</p></div><div id="sec-15" class="subsection"><h3>Comparing prediction accuracies for high (WGS-imputed) and low (GBS) density markers</h3><p id="p-51">Single kernel G-BLUP prediction accuracies for CBSD3s and CBSDRs were higher for WGS-imputed, than GBS markers for both optimized training population sizes of 200 and 400 clones (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). For CBSD6s, however, predictions accuracies were lower for high-density (WGS-imputed) at both training population sizes. For single kernel G-BLUP, prediction accuracies for CBSD3s and CBSDRs increased from 0.27 to 0.35, and -0.03 to 0.18 from low to high-density marker sets at the optimized training population size of 200 clones (Table S11). Similarly, predictions accuracies for CBSD3s and CBSDRs increased from 0.32 to 0.39, and -0.01 to 0.16 from low-density (GBS) and high-density (WGS-imputed) markers for the training populations of 400 clones (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>; Table S12).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of prediction accuracies for the CBSD-related traits under high density, whole genome sequence imputed (WGS-imputed) and low density genotyping-by-sequencing (GBS) markers for optimized training population sizes of 200 and 400 clones using single kernel G-BLUP model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2066890317" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of prediction accuracies for the CBSD-related traits under high density, whole genome sequence imputed (WGS-imputed) and low density genotyping-by-sequencing (GBS) markers for optimized training population sizes of 200 and 400 clones using single kernel G-BLUP model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3903/F5.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3903/F5.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3903/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491558" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">Comparison of prediction accuracies for the CBSD-related traits under high density, whole genome sequence imputed (WGS-imputed) and low density genotyping-by-sequencing (GBS) markers for optimized training population sizes of 200 and 400 clones using single kernel G-BLUP model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">Fitting multi-kernel G-BLUP models including kernels defined by markers on CBSD QTL chromosomes 4 and 11 for high-density markers did not always increase prediction accuracies (Tables S11 and S12). The highest prediction accuracy of 0.44 (CBSD3s) was recorded from the multi-kernel G-BLUP model, fitted with high density (WGS-imputed) markers for the optimized training population of 400 clones (Table S12). In other cases, prediction accuracies actually dropped from single- to multi-kernel models. For example, prediction accuracy for CBSD3s dropped from 0.35 for single-kernel to 0.32 for multi-kernel (three kernels) at the training population size of 200 clones (Table S11). Overall, the prediction accuracies for high and low-density marker sets were similar between the multi-kernel models regardless of the optimized training population size (Tables S11 and S12).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-54">Cassava brown streak disease (CBSD) caused by Uganda cassava brown streak virus (UCBSV) and cassava brown streak virus (CBSV) has continued to be a major threat to cassava productivity in southern, eastern and central parts of Africa. Recently, CBSD causing viruses were declared the leading biological enemy to cassava productions in CBSD endemic zones of Sub-Sahara Africa (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Legg <em>et al.</em>, 2014</a>). Concerted efforts such as quarantine, disease surveillance, and breeding for resistance have taken center-stage to prevent further spread of CBSD to W. Africa, especially Nigeria, the world’s largest producer and consumer of cassava. In this paper, we leveraged genome-wide prediction approaches as a potential means to enable pre-emptive breeding for CBSD resistance in W. Africa.</p><div id="sec-17" class="subsection"><h3>Impact of different sizes of optimized training population across models</h3><p id="p-55">For optimized training populations of 100, 200, 400, 800 and 922 clones, the highest prediction accuracies were observed at the training population sizes of 200 (G-BLUP) and 400 (Bayes-B) clones for CBSD6s and CBSD3s respectively. Our findings were similar to that of <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Wolfe <em>et al.</em> (2017)</a>, where prediction accuracy of 0.37 for CMD was observed for both the smallest and largest optimized training sizes of 300 and 900, respectively in cross-population prediction, suggesting that accuracies similar to that of the full set can be obtained with a small but carefully selected TP in relation to the test set.</p><p id="p-56">Overall, the cross-population prediction accuracies for IITA clones, based on optimized training populations and various prediction models (spanning 0.24 to 0.36), were comparable for CBSD3s to those reported previously for the cross-validation within NaCRRI training set, ranging from 0.27 to 0.32 (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>). In contrast, for CBSD6s, our prediction accuracies (0.14 to 0.28) were lower than accuracies reported by <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em> (2018)</a>, which ranged from 0.40 to 0.42. The similarity in foliar CBSD prediction accuracy for CBSD3s indicates some genetic signal for CBSD foliar symptom expression for IITA clones was captured by optimal NaCRRI training subsets. Unfortunately, our cross-population prediction accuracies for CBSDRs for optimized TPs were generally lower than the accuracies reported for cross-validation within NaCRRI training population for CBSDRs. In part, the negative prediction accuracies for CBSDR could be explained by GxE interaction for TP1 and TP2 (Table S13 and Figure S4). However, the GxE variances relative to genetic variances were low and therefore unlikely to explain fully the poor prediction accuracies observed for CBSD root necrosis in W. African clones. Also, we observed stronger linkage disequilibrium (LD) in NaCRRI training set than in the IITA test population (Figure S7), suggesting some differences in the LD decay rate between the two populations, which could partly explain the low prediction accuracies observed for CBSDRs in W. African clones. In such a case, there is need to first phenotype clones of W. African descent (<em>i.e.</em>, belonging to the W. African subpopulation) in E. Africa, and subsequently use the data generated for predicting CBSD resistance in W. African clones. One option would be to send many W. African clones to E. Africa as tissue culture plantlets. As observed in this study, cost and mortality are high for this option. Another possibility would be to send botanical seeds of W. African clones to E. Africa for evaluation.</p><p id="p-57">We did not observe consistent superior performance for any of the prediction models that we tested or for any of the CBSD traits analyzed. Several studies have reported similar results in that most prediction models perform similarly (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jannink <em>et al.</em>, 2010</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em>, 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Roorkiwal <em>et al.</em>, 2016</a>). Even though the models tested in the present study assumed different distributions of marker effects (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em>, 2001</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lorenz <em>et al.</em>, 2011</a>), their similarity in prediction accuracies could be interpreted as approximation to optimal genomic prediction models, where all the models capture the same or similar QTL effects across the genome (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Su <em>et al.</em>, 2014</a>). In such a situation, the choice of GS model would be less important than the actual design of the training population for across-population predictions.</p></div><div id="sec-18" class="subsection"><h3>Comparison of prediction accuracies for random and optimized training populations</h3><p id="p-58">Prediction accuracies can be improved by targeting more informative individuals in the reference panel used to generate the predictions and this has been demonstrated in several crop species (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Rincent <em>et al.</em>, 2012</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em>, 2015</a>). In general, we observed higher prediction accuracies for CBSD3s and CBSD6s from optimized compared with randomly selected training set of the same size. For example, at TP size of 200 clones, our prediction accuracies for CBSD3s was 0.27 with the optimized compared to 0.11 from the random subset. Similar findings were made by <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Wolfe <em>et al.</em> (2017)</a>, where STPGA-optimized training populations performed better than random subsets for a number of important cassava traits, including dry matter content (DMC), harvest index (HI), mean cassava mosaic disease and plant vigor. Our results, therefore, serve to further stress the importance of training population optimization for cross-population prediction.</p></div><div id="sec-19" class="subsection"><h3>Weighting prior biological information for CBSD prediction across population</h3><p id="p-59">Studies have shown increased prediction accuracies with inclusion of prior QTL information in genomic prediction models. For example, a study by <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Haffliger, (2016)</a> for reproductive traits in Swiss pig breeds revealed a significant increase in prediction accuracy for piglets when previously detected reproductive trait QTL markers were included in the prediction model. From the training population used in this study, two recent studies identified CBSD QTL on chromosomes 4 for CBSD3s and CBSD6s, and 11 for CBSD6s and CBSDRs (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kawuki <em>et al.</em>, 2016</a>; <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>). In addition, bi-parental mapping studies have had similar results (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Masumba <em>et al.</em>, 2017</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Nzuki <em>et al.</em>, 2017</a>). In an attempt to improve across-population prediction accuracies for CBSD symptoms, we chose to directly model the Chromosome 4 and 11 (Chr.4 and Chr.11) QTL by incorporating random effects for the markers on those chromosomes into our prediction. Prediction accuracies increased for CBSD3s and CBSD6s, but not for CBSDRs. The benefit was greatest for prediction accuracy of CBSD6s which increased by 9%, when three realized relationship matrices (Chr. 4 + Chr. 11 + the rest, optimized set of 200) were modeled. Although the percentage increase in prediction accuracies was less for the optimized TPs of 400 clones, we still observed increased prediction accuracies for CBSD6s, again when three relationship matrices were fitted. We observed a much higher increase in prediction accuracies for G-BLUP models that included the CBSD QTL as separate random effects, compared to the only marginal increase in prediction accuracies of 1.7% for CBSD3s and 2.5% for CBSDRs reported previously (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lozano <em>et al.</em>, 2017</a>).</p><p id="p-60">The higher prediction accuracies we observed by accounting for the CBSD QTL suggests that the development of genomic resources for cassava (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Prochnik <em>et al.</em>, 2012</a>), the identification of QTL by GWAS (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wolfe <em>et al.</em>, 2016</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Lozano <em>et al.</em>, 2017</a>; <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Kayondo <em>et al.</em>, 2018</a>) and candidate genes by bioinformatics (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lozano <em>et al.</em>, 2015</a>) can provide benefits for genomic prediction, particularly in across-population prediction scenarios.</p></div><div id="sec-20" class="subsection"><h3>Comparing prediction accuracies of high and low density marker panels</h3><p id="p-61">In the present study, prediction accuracies for CBSD3s and CBSDRs were 8% and 18% higher for high-density (WGS-imputed) markers than low-density (GBS) markers from single kernel G-BLUP model for the optimized training population of size of 200 clones (Table S11). Several studies have demonstrated increased prediction accuracies as a function of increase marker density (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Peixoto <em>et al.</em>, 2016</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang <em>et al.</em>, 2017</a>). In a recent study, using NaCRRI training population, prediction for CBSD-related traits, in a single kernel G-BLUP model was not improved by whole-genome imputation (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Lozano <em>et al.</em>, 2017</a>). On the other hand, in a simulation study for across population genomic prediction in dairy cattle, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">De Roos <em>et al.</em>, (2009)</a> reported higher prediction accuracies, similar to the improvement we observed for CBSD3s and CBSDRs, when more markers were included in the model. The study concluded that the reliability of genomic predictions across populations is determined by the consistency of marker–QTL allelic phase between the populations. The more diverged the populations are, the denser the markers must be to ensure preservation of marker–QTL phase across the populations. Increased prediction accuracies for CBSD3s and CBSDRs in this study, could therefore be a result of whole genome sequence imputed markers more reliably capturing the correct marker-CBSD QTL phase across the two populations. Since the only additional cost incurred in generating WGS-imputed markers is computational time, we believe that imputing the GBS markers to higher-density would benefit even poorly resourced breeding programs.</p></div><div id="sec-21" class="subsection"><h3>Conclusion</h3><p id="p-62">We have presented the first empirical validation of genomic prediction for cassava brown streak disease across populations. Based on our results, training population optimization provided a benefit of increased prediction accuracies over random subset and full set of training population for foliar cassava brown streak disease. More importantly, inclusion of prior CBSD QTL information in our genomic prediction models reasonably increased foliar CBSD prediction increased for W. African clones. Furthermore, whole genome sequence imputed markers increased prediction accuracies for CBSD3s and CBSDRs. Future efforts to better predict CBSD resistance in W. Africa clones could focus initially on testing progeny from W. African germplasm, and later use the progeny evaluation data to train CBSD prediction models in W. African. Lastly, further research should target a much larger number of W. African test clones than we used in the current study.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">We acknowledge the Bill and Melinda Gates Foundation, and the Department for International Development of the United Kingdom for funding this work through the “Next Generation Cassava Breeding Project”. Thanks to the technical field staff of NaCRRI Cassava-breeding program for generating the phenotypic data and the team from biosciences laboratory for the DNA extraction work. We also thank the International Institute of Tropical Agriculture (IITA), Ibadan, Nigeria for sharing their germplasm with us for this study. Great thanks to Cornell University Institute of Genomic diversity (IGD) for genotyping the clones.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-64">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7242851">https://doi.org/10.25387/g3.7242851</a>.</p></li><li class="fn" id="fn-4"><p id="p-65"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-03"><span class="received-label">Received </span>September 3, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-13"><span class="accepted-label">Accepted </span>October 13, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3903.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okeke</span>  <span class="cit-name-given-names">U. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">EMMREML: Fitting mixed models with known covariance structures</span>, <span class="cit-publ-name">R Repository CRAN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.2" data-doi="10.1186/s12711-015-0116-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of genomic selection training populations with a genetic algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0116-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0116-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0116-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.3" data-doi="10.1016/S0065-2660(08)60048-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alicai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndunguru</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sseruwagi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tairo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okao-Okuja</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cassava brown streak virus has a rapidly evolving genome: implications for virus speciation, variability, diagnosis and host resistance.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">36164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep36164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D13%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960048-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60048-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.4" data-doi="10.1051/forest:2006042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alicai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omongo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthi</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baguma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Re-emergence of cassava brown Streak disease in Uganda.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PD-91-0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D63%26rft.spage%253D625%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fforest%253A2006042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/forest:2006042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.5" data-doi="10.1111/j.0014-3820.2005.tb01785.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauhan</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilyas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagaba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Virus-derived stacked RNAi construct confers robust resistance to cassava brown streak disease.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.02052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBrommer%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D59%26rft.issue%253D6%26rft.spage%253D1362%26rft.epage%253D1371%26rft.atitle%253DNatural%2Bselection%2Band%2Bgenetic%2Bvariation%2Bfor%2Breproductive%2Breaction%2Bnorms%2Bin%2Ba%2Bwild%2Bbird%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2005.tb01785.x%26rft_id%253Dinfo%253Apmid%252F16050111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2005.tb01785.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16050111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230077500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.6" data-doi="10.1007/s00468-012-0802-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D27%26rft.spage%253D485%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00468-012-0802-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00468-012-0802-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.7" data-doi="10.1111/1365-2745.12604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D104%26rft.spage%253D1421%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1365-2745.12604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1365-2745.12604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.8" data-doi="10.1098/rstb.2016.0138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0138%26rft_id%253Dinfo%253Apmid%252F28483868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvMTcyMy8yMDE2MDEzOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.9" data-doi="10.5344/ajev.2011.10116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.5344%252Fajev.2011.10116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI2Mi80LzQxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.10" data-doi="10.1016/S0301-6226(02)00096-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D78%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252802%252900096-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(02)00096-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-thesis" id="cit-8.12.3903.11"><div class="cit-metadata unstructured">Goiffon, M. D., 2016 Optimal population value selection: A population-based selection strategy for genomic selection. A thesis submitted to the graduate faculty in partial fulfillment of the requirements for the degree of Master of Science of Iowa State University.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-thesis" id="cit-8.12.3903.12"><div class="cit-metadata unstructured">Haffliger, I., 2016 Genomic predictions including known QTL for reproduction traits in swine. Master’s thesis submitted to Norwegian University of Life Science.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.13" data-doi="10.1111/j.1420-9101.2008.01662.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DFallour-Rubio%26rft.auinit1%253DD.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D553%26rft.epage%253D563%26rft.atitle%253DRapid%2Bchanges%2Bin%2Bplasticity%2Bacross%2Bgenerations%2Bwithin%2Ban%2Bexpanding%2Bcedar%2Bforest.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01662.x%26rft_id%253Dinfo%253Apmid%252F19170817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01662.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19170817&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.14" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome.2010.12.0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.15" data-doi="10.1111/geb.12526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: A comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D26%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgeb.12526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/geb.12526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3903.16"><div class="cit-metadata unstructured">Hillocks, R. 2004 Research protocols for cassava brown streak virus. Fao.Org: 1–7.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.17" data-doi="10.1016/S0304-3800(98)00205-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennings</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cassava brown streak disease: A review of present knowledge and research needs.</span> <abbr class="cit-jnl-abbrev">Int. J. Pest Manage.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/0967087031000101061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPest%2BManage.%26rft.volume%253D116%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0304-3800%252898%252900205-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0304-3800(98)00205-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080302200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.18" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulembeka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeremiah</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alacho</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Disparity between leaf and root symptoms and crop losses associated with cassava brown streak disease in four countries in eastern Africa.</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jph.12430</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thresh</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Cassava mosaic and cassava brown streak virus diseases in Africa</span>. <abbr class="cit-jnl-abbrev">Root</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoot%26rft.volume%253D23%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.20" data-doi="10.1139/cjps-2015-0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thresh</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cassava brown streak disease in northern Mozambique.</span> <abbr class="cit-jnl-abbrev">Int. J. Pest Manage.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">181.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/09670870110087376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPest%2BManage.%26rft.volume%253D96%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjps-2015-0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjps-2015-0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.21" data-doi="10.3168/jds.2009-2585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2418-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DJamrozik%26rft.auinit1%253DJ.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D1216%26rft.epage%253D1233%26rft.atitle%253DRelationships%2Bbetween%2Bmilk%2Byield%2Band%2Bsomatic%2Bcell%2Bscore%2Bin%2BCanadian%2BHolsteins%2Bfrom%2Bsimultaneous%2Band%2Brecursive%2Brandom%2Bregression%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2585%26rft_id%253Dinfo%253Apmid%252F20172242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20172242&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.22" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.23" data-doi="10.1111/2041-210X.12225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaweesi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esuma</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pariyo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozimati</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Eleven years of breeding efforts to combat cassava brown streak disease.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">560</span>–<span class="cit-lpage">571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.16005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D5%26rft.spage%253D944%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12225%26rft_id%253Dinfo%253Apmid%252F25810896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25810896&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.24" data-doi="10.1007/BF00290638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayondo</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Carpio</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozimati</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide association mapping and genomic prediction for CBSD resistance in <em>Manihot esculenta</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-19696-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DM.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D450%26rft.atitle%253DA%2Bquantitative%2Bgenetic%2Bmodel%2Bfor%2Bgrowth%252C%2Bshape%252C%2Breaction%2Bnorms%252C%2Band%2Bother%2Binfinite-dimensional%2Bcharacters.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00290638%26rft_id%253Dinfo%253Apmid%252F2769086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00290638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2769086&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AG84900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.25" data-doi="10.1016/j.virusres.2011.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legg</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeremiah</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obiero</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthi</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndyetabula</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparing the regional epidemiology of the cassava mosaic and cassava brown streak virus pandemics in Africa.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.virusres.2011.04.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DM.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D979%26rft.epage%253D993%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Binheritance%252C%2Bselection%2Band%2Bevolution%2Bof%2Bgrowth%2Btrajectories.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2011.04.018%26rft_id%253Dinfo%253Apmid%252F2323560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzQvOTc5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3903.26"><div class="cit-metadata unstructured">Legg, J., E. A. Somado, I. Barker, L. Beach, H. Ceballos <em>et al.</em>, 2014 A global alliance declaring war on cassava viruses in Africa. 231–248.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3903.27"><div class="cit-metadata unstructured">Lorenz, A. J., S. Chao, F. G. Asoro, E. L. Heffner, T. Hayashi <em>et al.</em>, 2011 Genomic selection in plant breeding, knowledge and prospects.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.28" data-doi="10.1007/BF00022850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Carpio</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayondo</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amuge</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Leveraging transcriptomics data for genomic prediction models in cassava.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/208181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D92%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00022850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00022850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.29" data-doi="10.1093/aob/mcp214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification and distribution of the NBS-LRR gene family in the Cassava genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1554-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcp214%26rft_id%253Dinfo%253Apmid%252F19748909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcp214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19748909&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271106700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.30" data-doi="10.1111/j.1439-0434.2005.00974.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruthi</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mtunda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raya</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Transmission of cassava brown streak virus by <em>Bemisia tabaci</em> (Gennadius).</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">312.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0434.2005.00974.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol.%26rft.volume%253D13%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0434.2005.00974.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0434.2005.00974.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.31" data-doi="10.1016/j.fcr.2017.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masona</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Transferring a cassava (<em>Manihot esculenta</em> Crantz) genetic engineering capability to the African environment: progress and prospects.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D216%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2017.08.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2017.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.32" data-doi="10.1016/j.foreco.2017.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masumba</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapinga</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mkamilo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salum</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulembeka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">QTL associated with resistance to cassava brown streak and cassava mosaic diseases in a bi-parental cross of two Tanzanian farmer varieties</span>. <abbr class="cit-jnl-abbrev">Namikonga and Albert. Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2069</span>–<span class="cit-lpage">2090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2943-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNamikonga%2Band%2BAlbert.%2BTheor.%2BAppl.%2BGenet.%26rft.volume%253D392%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2017.03.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2017.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.33" data-doi="10.1186/s12985-017-0790-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mbewe</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tairo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sseruwagi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndunguru</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variability in P1 gene redefines phylogenetic relationships among cassava brown streak viruses.</span> <abbr class="cit-jnl-abbrev">Virol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12985-017-0790-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol.%2BJ.%26rft.volume%253D13%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-017-0790-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12985-017-0790-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.34" data-doi="10.1016/j.foreco.2009.03.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense markers maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D258%26rft.spage%253D860%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2009.03.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2009.03.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.35" data-doi="10.1016/j.foreco.2008.05.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammed</span>  <span class="cit-name-given-names">I. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abarshi</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muli</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillocks</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthi</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The symptom and genetic diversity of cassava brown streak viruses infecting cassava in east Africa.</span> <abbr class="cit-jnl-abbrev">Adv. Virol.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2012/795697</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BVirol.%26rft.volume%253D256%26rft.spage%253D837%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2008.05.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2008.05.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.36" data-doi="10.1111/j.1574-6968.2007.00661.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monger</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of different isolates of cassava brown streak virus and development of a diagnostic test.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D50%26rft.spage%253D768%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2007.00661.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2007.00661.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.37" data-doi="10.3168/jds.2006-487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulimbi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phemba</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assumani</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasereka</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muyisa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">First report of Ugandan cassava brown streak virus on cassava in Democratic Republic of Congo.</span> <abbr class="cit-jnl-abbrev">New Dis. Rep.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMiglior%26rft.auinit1%253DF.%26rft.volume%253D90%26rft.issue%253D5%26rft.spage%253D2468%26rft.epage%253D2479%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BMilk%2BUrea%2BNitrogen%2Band%2BLactose%2Band%2BTheir%2BRelationships%2Bwith%2BOther%2BProduction%2BTraits%2Bin%2BCanadian%2BHolstein%2BCattle%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2006-487%26rft_id%253Dinfo%253Apmid%252F17430951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2006-487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17430951&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3903.38"><div class="cit-metadata unstructured">Mware, B., E. Ateka, and J. Songa. 2009 Transmission and distribution of cassava brown streak virus disease in cassava growing areas of Kenya. J. Appl.864–870.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.39" data-doi="10.1111/evo.13003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahoor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orabi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cericola</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of seed quality traits using advanced barley breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0164494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D70%26rft.spage%253D1944%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.40" data-doi="10.1890/13-2355.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Njoroge</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutisya</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miano</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilalo</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Whitefly species efficiency in transmitting cassava mosaic and brown streak virus diseases.</span> <abbr class="cit-jnl-abbrev">Cogent Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCogent%2BBiol.%26rft.volume%253D85%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1890%252F13-2355.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/13-2355.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.12.3903.41"><div class="cit-metadata unstructured">Nweke. 2004 New challenges in the cassava transformation in Nigeria and Ghana: Environment and production technology division international food policy research institute. Food Policy 67: 1–118.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.42" data-doi="10.3168/jds.S0022-0302(07)71642-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nzuki</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katari</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bredeson</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masumba</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapinga</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">QTL mapping for pest and disease resistance in cassava and coincidence of some QTL with introgression regions derived from <em>manihot glaziovii</em>.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMuir%26rft.auinit1%253DB.%2BL.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D1564%26rft.epage%253D1574%26rft.atitle%253DGenetic%2BParameters%2Bfor%2Ba%2BMultiple-Trait%2BMultiple-Lactation%2BRandom%2BRegression%2BTest-Day%2BModel%2Bin%2BItalian%2BHolsteins%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252807%252971642-9%26rft_id%253Dinfo%253Apmid%252F17297130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(07)71642-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17297130&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.43" data-doi="10.1086/675302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odipio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogwok</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bua</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNAi-derived field resistance to cassava brown streak disease persists across the vegetative cropping cycle.</span> <abbr class="cit-jnl-abbrev">GM Crops Food</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/gmcr.26408</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGM%2BCrops%2BFood%26rft.volume%253D183%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1086%252F675302%26rft_id%253Dinfo%253Apmid%252F24642491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/675302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24642491&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332923300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.44" data-doi="10.1111/j.2041-210x.2012.00261.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogwok</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odipio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaitán-Solís</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bua</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transgenic RNA interference (RNAi)-derived field resistance to cassava brown streak disease.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2012.00812.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft.volume%253D4%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210x.2012.00261.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210x.2012.00261.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.45" data-doi="10.1016/j.dendro.2016.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legg</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanju</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cassava brown streak disease: A threat to food security in Africa.</span> <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">968</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/vir.0.000014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol.%26rft.volume%253D40%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dendro.2016.07.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dendro.2016.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.46" data-doi="10.1186/s12864-015-1765-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peixoto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhering</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Determination of the optimal number of markers and individuals in a training population necessary for maximum prediction accuracy in F 2 populations by using genomic selection models.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res. Mol. Res</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%2BMol.%2BRes%26rft.volume%253D15%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1765-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1765-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.47" data-doi="10.1007/s12042-011-9088-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marri</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowicz</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The cassava genome: Current progress, future directions.</span> <abbr class="cit-jnl-abbrev">Trop. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTrop%2BPlant%2BBiol%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D94%26rft.atitle%253DThe%2BCassava%2BGenome%253A%2BCurrent%2BProgress%252C%2BFuture%2BDirections.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12042-011-9088-z%26rft_id%253Dinfo%253Apmid%252F22523606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12042-011-9088-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523606&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000208980100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3903.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2008</span> p. <span class="cit-fpage">900051</span> in <span class="cit-source"><em>R: A language and environment for statistical computing. R Foundation for Statistical Computing</em>, <em>Vienna</em>, <em>Austria</em></span>, <span class="cit-publ-name">R Found. Stat. Comput</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.49" data-doi="10.1016/j.theriogenology.2017.04.052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gedil</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic mapping using genotyping-by-sequencing in the clonally propagated cassava.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D98%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.theriogenology.2017.04.052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.theriogenology.2017.04.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.50" data-doi="10.1016/S0301-6226(03)00151-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esuma</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egesi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D86%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252803%252900151-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(03)00151-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.51" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: Comparison of methods in two diverse groups of maize inbreds (<em>Zea mays L</em>.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.141473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rathore</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-enabled prediction models for yield related traits in chickpea.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D24%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.53" data-doi="10.1139/X07-075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D37%26rft.spage%253D2090%26rft_id%253Dinfo%253Adoi%252F10.1139%252FX07-075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/X07-075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.54" data-doi="10.1016/j.tree.2012.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">R. F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">Studies of the mosaic disease of cassava.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">790</span>–<span class="cit-lpage">806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DStinchcombe%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D637%26rft.epage%253D647%26rft.atitle%253DGenetics%2Band%2Bevolution%2Bof%2Bfunction-valued%2Btraits%253A%2Bunderstanding%2Benvironmentally%2Bresponsive%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2012.07.002%26rft_id%253Dinfo%253Apmid%252F22898151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2012.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22898151&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313607200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.55" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus-specific variances.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6547</span>–<span class="cit-lpage">6559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-8210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D1279%26rft.epage%253D1286%26rft.atitle%253DGENE%2BEFFECTS%2BAND%2BVARIANCES%2BIN%2BHYBRID%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F17248353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI1NC82LzEyNzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.56" data-doi="10.1111/j.1365-2745.2006.01176.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D94%26rft.spage%253D1103%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2745.2006.01176.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2745.2006.01176.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000240920400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.57" data-doi="10.1139/X09-111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagaba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aleu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odipio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okao-Okuja</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Field level RNAi-mediated resistance to cassava brown streak disease across multiple cropping cycles and diverse east African agro-ecological locations.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.02060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D39%26rft.spage%253D1939%26rft_id%253Dinfo%253Adoi%252F10.1139%252FX09-111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/X09-111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.58" data-doi="10.3168/jds.S0022-0302(06)72245-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effects of marker density and population structure on the genomic prediction accuracy for growth trait in Pacific white shrimp Litopenaeus vannamei.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0507-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWindig%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D89%26rft.issue%253D5%26rft.spage%253D1765%26rft.epage%253D1775%26rft.atitle%253DGenetic%2BCorrelations%2BBetween%2BMilk%2BProduction%2Band%2BHealth%2Band%2BFertility%2BDepending%2Bon%2BHerd%2BEnvironment%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252806%252972245-7%26rft_id%253Dinfo%253Apmid%252F16606748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(06)72245-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16606748&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.59" data-doi="10.1099/vir.0.014688-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koerbler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pietruszka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paape</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of cassava brown streak viruses reveals the presence of distinct virus species causing cassava brown streak disease in East Africa.</span> <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/vir.0.014688-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.014688-0%26rft_id%253Dinfo%253Apmid%252F20071490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.014688-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20071490&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.60" data-doi="10.3835/plantgenome2017.03.0015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Carpio</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ezenwaka</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeogu</span>  <span class="cit-name-given-names">U. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Prospects for genomic selection in cassava breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.03.0015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.03.0015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.03.0015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.61" data-doi="10.3835/plantgenome2015.11.0118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egesi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association and prediction reveals genetic architecture of cassava mosaic disease resistance and prospects for rapid genetic improvement.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.11.0118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D342%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.11.0118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.11.0118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3903.62" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3903.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3891" title="Genetic Contributions to Ectopic Sperm Cell Migration in Caenorhabditis Nematodes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3915" title="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3903" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XLh-j9Wsh4xJ62sJFtW1JRsV6UtOWF8lmZdSFjbjw7U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3903" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3903" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3903" class="active" data-icon-position="" data-hide-link-title="0">Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491667" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-K75OF8HpHXUsFBffwrJ_yzNO7nWT_wAeN53KHHpUOnA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3903.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3903" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qKTtA6m6DrE98zEgPrJ0aHegvAUn4FFgXiYMwnBdzPE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3903" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3903.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491667" id="node491667--2" data-pisa="ggg;8/12/3903" data-pisa-master="ggg;g3.118.200710" data-apath="/ggg/8/12/3903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alfred Ozimati</span>, <span class="highwire-citation-author" data-delta="1">Robert Kawuki</span>, <span class="highwire-citation-author" data-delta="2">Williams Esuma</span>, <span class="highwire-citation-author" data-delta="3">Ismail Siraj Kayondo</span>, <span class="highwire-citation-author" data-delta="4">Marnin Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author" data-delta="6">Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3903-3913;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491667--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alfred Ozimati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alfred%2BOzimati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozimati%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlfred%2BOzimati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alfred Ozimati</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ozimatialfred@gmail.com" class="" data-icon-position="" data-hide-link-title="0">ozimatialfred@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Kawuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKawuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawuki%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BKawuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Williams Esuma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Williams%2BEsuma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esuma%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliams%2BEsuma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ismail Siraj Kayondo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BSiraj%2BKayondo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kayondo%20IS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BSiraj%2BKayondo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marnin Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarnin%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Department of Agriculture, Agricultural Research Service (USDA-ARS) R.W. Holley Center for Agriculture and Health, Ithaca 14853, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3903" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3903&amp;text=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3903&amp;t=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3903&amp;title=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491667/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491667" id="node491667--3" data-pisa="ggg;8/12/3903" data-pisa-master="ggg;g3.118.200710" data-apath="/ggg/8/12/3903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alfred Ozimati</span>, <span class="highwire-citation-author" data-delta="1">Robert Kawuki</span>, <span class="highwire-citation-author" data-delta="2">Williams Esuma</span>, <span class="highwire-citation-author" data-delta="3">Ismail Siraj Kayondo</span>, <span class="highwire-citation-author" data-delta="4">Marnin Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Lozano</span>, <span class="highwire-citation-author" data-delta="6">Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3903-3913;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200710 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491667--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alfred Ozimati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alfred%2BOzimati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozimati%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlfred%2BOzimati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0503-8107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alfred Ozimati</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ozimatialfred@gmail.com" class="" data-icon-position="" data-hide-link-title="0">ozimatialfred@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Kawuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKawuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawuki%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BKawuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Williams Esuma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Williams%2BEsuma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esuma%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliams%2BEsuma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ismail Siraj Kayondo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Crops Resources Research Institute (NaCRRI), P.O. Box, 7084 Kampala, Uganda</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BSiraj%2BKayondo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kayondo%20IS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BSiraj%2BKayondo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marnin Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarnin%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Lozano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BLozano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lozano%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BLozano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0760-4977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Lozano</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Institute for Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Integrative Plant Science, Plant breeding and Genetics Section, Cornell University, Ithaca, New York</div><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Department of Agriculture, Agricultural Research Service (USDA-ARS) R.W. Holley Center for Agriculture and Health, Ithaca 14853, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3903&amp;rft.epage=3913&amp;rft.atitle=Training%20Population%20Optimization%20for%20Prediction%20of%20Cassava%20Brown%20Streak%20Disease%20Resistance%20in%20West%20African%20Clones&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ozimati%2C+Alfred&amp;rft.au=Kawuki%2C+Robert&amp;rft.au=Esuma%2C+Williams&amp;rft.au=Kayondo%2C+Ismail+Siraj&amp;rft.au=Wolfe%2C+Marnin&amp;rft.au=Lozano%2C+Roberto&amp;rft.au=Rabbi%2C+Ismail&amp;rft.au=Kulakow%2C+Peter&amp;rft.au=Jannink%2C+Jean-Luc"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491667/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491667/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491667/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491667/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491667/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491667/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491667/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491667/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491667/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491667/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491667/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491667/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3903" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3903.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3903.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3903.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3903.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
