<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3783" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3783.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3783.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3783.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3783/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3783.atom" />
<meta name="HW.pisa" content="ggg;8/12/3783" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" />
<meta name="DC.Identifier" content="10.1534/g3.118.200194" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Corty et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We illustrate, through two case studies, that “mean-variance QTL mapping”—QTL mapping that models effects on the mean and the variance simultaneously—can discover QTL that traditional interval mapping cannot. Mean-variance QTL mapping is based on the double generalized linear model, which extends the standard linear model used in interval mapping by incorporating not only a set of genetic and covariate effects for mean but also set of such effects for the residual variance. Its potential for use in QTL mapping has been described previously, but it remains underutilized, with certain key advantages undemonstrated until now. In the first case study, a reduced complexity intercross of C57BL/6J and C57BL/6N mice examining circadian behavior, our reanalysis detected a mean-controlling QTL for circadian wheel running activity that interval mapping did not; mean-variance QTL mapping was more powerful than interval mapping at the QTL because it accounted for the fact that mice homozygous for the C57BL/6N allele had less residual variance than other mice. In the second case study, an intercross between C57BL/6J and C58/J mice examining anxiety-like behaviors, our reanalysis detected a variance-controlling QTL for rearing behavior; interval mapping did not identify this QTL because it does not target variance QTL. We believe that the results of these reanalyses, which in other respects largely replicated the original findings, support the use of mean-variance QTL mapping as standard practice." />
<meta name="DC.Contributor" content="Robert W. Corty" />
<meta name="DC.Contributor" content="Vivek Kumar" />
<meta name="DC.Contributor" content="Lisa M. Tarantino" />
<meta name="DC.Contributor" content="Joseph S. Takahashi" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We illustrate, through two case studies, that “mean-variance QTL mapping”—QTL mapping that models effects on the mean and the variance simultaneously—can discover QTL that traditional interval mapping cannot. Mean-variance QTL mapping is based on the double generalized linear model, which extends the standard linear model used in interval mapping by incorporating not only a set of genetic and covariate effects for mean but also set of such effects for the residual variance. Its potential for use in QTL mapping has been described previously, but it remains underutilized, with certain key advantages undemonstrated until now. In the first case study, a reduced complexity intercross of C57BL/6J and C57BL/6N mice examining circadian behavior, our reanalysis detected a mean-controlling QTL for circadian wheel running activity that interval mapping did not; mean-variance QTL mapping was more powerful than interval mapping at the QTL because it accounted for the fact that mice homozygous for the C57BL/6N allele had less residual variance than other mice. In the second case study, an intercross between C57BL/6J and C58/J mice examining anxiety-like behaviors, our reanalysis detected a variance-controlling QTL for rearing behavior; interval mapping did not identify this QTL because it does not target variance QTL. We believe that the results of these reanalyses, which in other respects largely replicated the original findings, support the use of mean-variance QTL mapping as standard practice.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3783" />
<meta name="citation_id" content="8/12/3783" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3783" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3783.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3783.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3783.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200194" />
<meta name="citation_pmid" content="30389793" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3783" />
<meta name="citation_lastpage" content="3790" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Robert W. Corty" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Bioinformatics and Computational Biology Curriculum" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6787-9051" />
<meta name="citation_author" content="Vivek Kumar" />
<meta name="citation_author_institution" content="The Jackson Laboratory" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6643-7465" />
<meta name="citation_author" content="Lisa M. Tarantino" />
<meta name="citation_author_institution" content="Bioinformatics and Computational Biology Curriculum" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9984-2600" />
<meta name="citation_author" content="Joseph S. Takahashi" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Neuroscience, University of Texas Southwestern Medical Center, Dallas, TX 75390" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0384-8878" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Bioinformatics and Computational Biology Curriculum" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Ayroles;citation_author=S. M. Buchanan;citation_author=C. O’Leary;citation_author=K. Skutt-Kakaria;citation_author=J. K. Grenier;citation_title=Behavioral idiosyncrasy reveals genetic control of phenotypic variability.;citation_volume=112;citation_year=2015;citation_pmid=25953335;citation_doi=10.1073/pnas.1503830112" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=J. S. Bailey;citation_author=L. Grabowski-Boase;citation_author=B. M. Steffy;citation_author=T. Wiltshire;citation_author=G. A. Churchill;citation_title=Identification of quantitative trait loci for locomotor  activation and anxiety using closely related inbred  strains.;citation_pages=761-769;citation_volume=7;citation_year=2008;citation_issue=7;citation_pmid=19130624;citation_doi=10.1111/j.1601-183X.2008.00415.x" />
<meta name="citation_reference" content="citation_journal_title=Neurobiol. Aging;citation_author=G. Banks;citation_author=I. Heise;citation_author=B. Starbuck;citation_author=T. Osborne;citation_author=L. Wisby;citation_title=Genetic background influences age-related decline in visual and nonvisual retinal responses, circadian rhythms, and sleep.;citation_pages=380-393;citation_volume=36;citation_year=2015;citation_pmid=25179226;citation_doi=10.1016/j.neurobiolaging.2014.07.040" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. A. Bogue;citation_author=S. C. Grubb;citation_author=D. O. Walton;citation_author=V. M. Philip;citation_author=G. Kolishovski;citation_title=Mouse phenome database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice.;citation_pages=D843-D850;citation_volume=46;citation_year=2017;citation_doi=10.1093/nar/gkx1082" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Y. Cao;citation_author=P. Wei;citation_author=M. Bailey;citation_author=J. S. K. Kauwe;citation_author=T. J. Maxwell;citation_title=A versatile omnibus test for detecting mean and variance heterogeneity.;citation_pages=51-59;citation_volume=38;citation_year=2014;citation_doi=10.1002/gepi.21778" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Cox;citation_author=C. L. Ackert-Bicknell;citation_author=B. L. Dumont;citation_author=Y. Ding;citation_author=J. T. Bell;citation_title=A New Standard Genetic Map for the Laboratory Mouse.;citation_volume=182;citation_year=2009;citation_pmid=19535546;citation_doi=10.1534/genetics.109.105486" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=W. Q. Deng;citation_author=G. Pare;citation_title=A fast algorithm to optimize SNP prioritization for gene-gene and gene-environment interactions.;citation_pages=729-738;citation_volume=35;citation_year=2011;citation_issue=7;citation_pmid=21922538;citation_doi=10.1002/gepi.20624" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. Dudbridge;citation_author=B. P. C. Koeleman;citation_title=Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.;citation_pages=424-435;citation_volume=75;citation_year=2004;citation_issue=3;citation_pmid=15266393;citation_doi=10.1086/423738" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Dumitrascu;citation_author=G. Darnell;citation_author=J. Ayroles;citation_author=B. E. Engelhardt;citation_title=Statistical tests for detecting variance effects in quantitative trait studies.;citation_year=2018;citation_doi=10.1093/bioinformatics/bty565" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Feenstra;citation_author=I. M. Skovgaard;citation_author=K. W. Broman;citation_title=Mapping quantitative trait loci by an extension of the Haley-Knott regression method using estimating equations.;citation_volume=173;citation_year=2006;citation_pmid=16702423;citation_doi=10.1534/genetics.106.058537" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. K. Forsberg;citation_author=M. E. Andreatta;citation_author=X. Y. Huang;citation_author=J. Danku;citation_author=D. E. Salt;citation_title=The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.;citation_pages=e1005648;citation_volume=11;citation_year=2015;citation_pmid=26599497;citation_doi=10.1371/journal.pgen.1005648" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Biotechnol.;citation_author=K. A. Geiler-Samerotte;citation_author=C. R. Bauer;citation_author=S. Li;citation_author=N. Ziv;citation_author=D. Gresham;citation_title=The details in the distributions: Why and how to study phenotypic variability.;citation_pages=752-759;citation_volume=24;citation_year=2013;citation_pmid=23566377;citation_doi=10.1016/j.copbio.2013.03.010" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kapushesky;citation_author=I. Emam;citation_author=E. Holloway;citation_author=P. Kurnosov;citation_author=A. Zorin;citation_title=Gene expression Atlas at the European Bioinformatics Institute.;citation_volume=38;citation_year=2009;citation_pmid=19906730;citation_doi=10.1093/nar/gkp936" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. H. Kim;citation_author=S. M. Park;citation_author=M. R. Kang;citation_author=S. Y. Oh;citation_author=T. H. Lee;citation_title=Ubiquitin ligase MKRN1 modulates telomere length homeostasis through a proteolysis of hTERT;citation_pages=776-781;citation_volume=19;citation_year=2005;citation_issue=7;citation_pmid=15805468;citation_doi=10.1101/gad.1289405" />
<meta name="citation_reference" content="Kumar, V., K. Kim, C. Joseph, S. Kourrich, S.-H. Yoo, et al., 2013 C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response. Science (80). 342: 1508–1512. 10.1126/science.1245503" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=S. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lein;citation_author=M. J. Hawrylycz;citation_author=N. Ao;citation_author=M. Ayres;citation_author=A. Bensinger;citation_title=Genome-wide atlas of gene expression in the adult mouse brain.;citation_pages=168-176;citation_volume=445;citation_year=2007;citation_issue=7124;citation_pmid=17151600;citation_doi=10.1038/nature05453" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Ma;citation_author=G. Hoffman;citation_author=A. Keinan;citation_title=X-inactivation informs variance-based testing for X-linked association of a quantitative trait.;citation_pages=241;citation_volume=16;citation_year=2015;citation_pmid=25880738;citation_doi=10.1186/s12864-015-1463-y" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=O. Martínez;citation_author=R. N. Curnow;citation_title=Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.;citation_pages=480-488;citation_volume=85;citation_year=1992;citation_pmid=24197463;citation_doi=10.1007/BF00222330" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. McWilliam;citation_author=W. Li;citation_author=M. Uludag;citation_author=S. Squizzato;citation_author=Y. M. Park;citation_title=Analysis Tool Web Services from the EMBL-EBI.;citation_volume=41;citation_year=2013;citation_pmid=23671338;citation_doi=10.1093/nar/gkt376" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=R. M. Nelson;citation_author=M. E. Pettersson;citation_author=Ö. Carlborg;citation_title=A century after Fisher: Time for a new paradigm in quantitative genetics.;citation_pages=669-676;citation_volume=29;citation_year=2013;citation_pmid=24161664;citation_doi=10.1016/j.tig.2013.09.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Detecting major genetic loci controlling phenotypic variability in experimental crosses.;citation_volume=188;citation_year=2011;citation_pmid=21467569;citation_doi=10.1534/genetics.111.127068" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.;citation_pages=63-63;citation_volume=13;citation_year=2012;citation_pmid=22827487;citation_doi=10.1186/1471-2156-13-63" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=X. Shen;citation_author=M. Pettersson;citation_author=L. Ronnegard;citation_author=O. Carlborg;citation_title=Inheritance beyond plain heritability: variance-controlling genes in Arabidopsis thaliana.;citation_pages=e1002839-e1002839;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22876191;citation_doi=10.1371/journal.pgen.1002839" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=X. Shen;citation_author=L. Ronnegard;citation_title=Issues with data transformation in genome-wide association studies for phenotypic variability.;citation_pages=200;citation_volume=2;citation_year=2013;citation_doi=10.12688/f1000research.2-200.v1" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=K. Shimomura;citation_author=S. S. Low-Zeddies;citation_author=D. P. King;citation_author=T. D. Steeves;citation_author=A. Whiteley;citation_title=Genome-Wide Epistatic Interaction Analysis Reveals Complex Genetic Determinants of Circadian Behavior in Mice;citation_pages=959-980;citation_volume=11;citation_year=2001;citation_issue=6;citation_pmid=11381025;citation_doi=10.1101/gr.171601" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. M. Simon;citation_author=S. Greenaway;citation_author=J. K. White;citation_author=H. Fuchs;citation_author=V. Gailus-Durner;citation_title=A comparative phenotypic and genomic analysis of c57bl/6j and c57bl/6n mouse strains.;citation_pages=R82;citation_volume=14;citation_year=2013;citation_pmid=23902802;citation_doi=10.1186/gb-2013-14-7-r82" />
<meta name="citation_reference" content="citation_journal_title=Soc. Ser. B Methodol.;citation_author=G. K. Smyth;citation_title=Generalized linear models with varying dispersion. J. R. Stat.;citation_pages=47-60;citation_volume=51;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=D. Soave;citation_author=L. Sun;citation_title=A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.;citation_pages=960-971;citation_volume=73;citation_year=2017;citation_doi=10.1111/biom.12651" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=X. Sun;citation_author=R. Elston;citation_author=N. Morris;citation_author=X. Zhu;citation_title=What is the significance of difference in phenotypic variability across SNP genotypes?;citation_pages=390-397;citation_volume=93;citation_year=2013;citation_pmid=23910463;citation_doi=10.1016/j.ajhg.2013.06.017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=J. Flint;citation_author=R. Mott;citation_title=Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.;citation_volume=172;citation_year=2006;citation_pmid=16361245;citation_doi=10.1534/genetics.104.039313" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=S. Xu;citation_title=Iteratively reweighted least squares mapping of quantitative trait loci.;citation_pages=341-355;citation_volume=28;citation_year=1998;citation_issue=5;citation_pmid=9926616;citation_doi=10.1023/A:1021617618150" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="We illustrate, through two case studies, that “mean-variance QTL mapping”—QTL mapping that models effects on the mean and the variance simultaneously—can discover QTL that traditional interval mapping cannot. Mean-variance QTL mapping is based on the double generalized linear model, which extends the standard linear model used in interval mapping by incorporating not only a set of genetic and covariate effects for mean but also set of such effects for the residual variance. Its potential for use in QTL mapping has been described previously, but it remains underutilized, with certain key advantages undemonstrated until now. In the first case study, a reduced complexity intercross of C57BL/6J and C57BL/6N mice examining circadian behavior, our reanalysis detected a mean-controlling QTL for circadian wheel running activity that interval mapping did not; mean-variance QTL mapping was more powerful than interval mapping at the QTL because it accounted for the fact that mice homozygous for the C57BL/6N allele had less residual variance than other mice. In the second case study, an intercross between C57BL/6J and C58/J mice examining anxiety-like behaviors, our reanalysis detected a variance-controlling QTL for rearing behavior; interval mapping did not identify this QTL because it does not target variance QTL. We believe that the results of these reanalyses, which in other respects largely replicated the original findings, support the use of mean-variance QTL mapping as standard practice." />
<meta name="og-title" property="og:title" content="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3783" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We illustrate, through two case studies, that “mean-variance QTL mapping”—QTL mapping that models effects on the mean and the variance simultaneously—can discover QTL that traditional interval mapping cannot. Mean-variance QTL mapping is based on the double generalized linear model, which extends the standard linear model used in interval mapping by incorporating not only a set of genetic and covariate effects for mean but also set of such effects for the residual variance. Its potential for use in QTL mapping has been described previously, but it remains underutilized, with certain key advantages undemonstrated until now. In the first case study, a reduced complexity intercross of C57BL/6J and C57BL/6N mice examining circadian behavior, our reanalysis detected a mean-controlling QTL for circadian wheel running activity that interval mapping did not; mean-variance QTL mapping was more powerful than interval mapping at the QTL because it accounted for the fact that mice homozygous for the C57BL/6N allele had less residual variance than other mice. In the second case study, an intercross between C57BL/6J and C58/J mice examining anxiety-like behaviors, our reanalysis detected a variance-controlling QTL for rearing behavior; interval mapping did not identify this QTL because it does not target variance QTL. We believe that the results of these reanalyses, which in other respects largely replicated the original findings, support the use of mean-variance QTL mapping as standard practice." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Hj2_jqsnPJU8vhvDJzWKNdUehBX3ExFDZkU6W-HtZVo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491519","apath":"\/ggg\/8\/12\/3783.atom","pisa":"ggg;8\/12\/3783","ac":{"\/ggg\/8\/12\/3783.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3783.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3783"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3783"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3783":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ts8qPMtvGB2yK\\\\\\\/OEGQZ+myHT0NQX4OYHnJk6UICTxhAdQjEWvCZ71ry6AadK8DVlIRPITdpS9YgqV8mVIapysqGVDJjflvFj\\\\\\\/SNhouLXttmgg8Rni0b9ojswBHdlnI1sdsppmiYhSn3PAmPzBIF\\\\\\\/\\\\\\\/tJQK703FyzbhUhKiqBt2MCyHP3xePYYr+sV88X5HA0auTNlbaQmemNEzUDCZld1JC5n8cXZVlmBenB\\\\\\\/+8qdVdrUBy2m4vPqC5baVTwEtD1rJcGojCKMKk4\\\\\\\/tbc3rG1XS90EHhcFRkrA9jr\\\\\\\/zddDsiWglom0HHmXkYWmhmWIFhUD2GNYcfFFI9Ms7BCvUTNBKXsyzf9k76odaAhqO5XqRQ9H+p\\\\\\\/bksl2z160xkpF5SZd+bBj7fOUKzDlJLAT8su6dpMd99V\\\\\\\/232cYCAqVH2Ef7Unqns8B1rZWZgxJYKx1EWhu45N95SkgvMEXRiRReU6gZTk80kLC1LmeDpzijQzjN7Axb9rwGsWtiazKbOhhLI6hFsqMVwFo5hFzCJhYiPb6IKPEzTt2ekiw4+vROkvB+6z77Ar779jAx36RkiujyEwFpJeQKke1lgUcBjZtHKWSUhm8gk475Ak9l1AJ7WGv4pnbtZtnVxggmRIX201BjgoQQZzKHzjkticaoFhof1rZpXaQpChxnW\\\\\\\/+AWtKWi5U+u1m0KWAEOeLGFF\\\\\\\/AxYgYBuIEiNY4WLGqQ+\\\\\\\/wqQbXHa1FPD72PQFENbFH7Crl6orBz7UuMJsdOcqbYPQUWoTUJYBaYt3wmyY2es0DxJ490dE568JPCDBXe1GP0zPXOCggNyLvnIbfPJqvxIRIEmHmGSYTGCPW3A4TjyKTFSKwvEf5SNwZzXCVNmnTJIGPtFjnfBNZSyZyK\\\\\\\/w\\\\\\\/Z9AoX\\\\\\\/rcBXORCUUagP2222XScTNW4mS5aclPlfggfu7AAcJE46TsWjHxvaN5EAC9BNg03fxY0d0GI0mY9IXYbkU7gg46tie1qxRr3uF4H6SjW+RXg87dF6S99MGO4GJGBVHJtQ6TDNJEPZiat3rLL2wIhPa\\\\\\\/U1noH0UF1EC\\\\\\\/\\\\\\\/Z1YhSusyU\\\\\\\/QbzHHUDb0uWeENEa4O1sStB+mWstavSVcpLfHRnnI7LRUNbKaBmriNml9lX2WqEz8SeXWsB4+QVxG8tZdHHHnRePocn1ekvDjo7TaaOXVOeO+vLna7PanOMRAR7zlhmsbeKO+eJ5SMkoK75WumAPSMnKZYxHBmz4tACzWEVYw5XB1K6cHkxqk\\\\\\\/W7UMiDGB8gv1FET2L30NDaqb7b8rXBdgW\\\u0022,\\\u0022iv\\\u0022:\\\u00227dSPgXmxV2mwWZ4psn+15Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a420cb9d82a65a046185b560f136472472f50c327912fb7ddfdc7fb5dc5e467f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KMkTL6kOZmbG9VHLUcI5parrMyzpadIrmWGqdmFHjq21FAaqL2GWWne8EdPtdLxd+Ykj76DzB7cHtqHSkr3nb\\\\\\\/n+LOn\\\\\\\/wgqRTwm3D5YvHc6eYcTPiqMa1qWHRMA6wbCNXT\\\\\\\/AosSEIZ3IZoUIpRPMyC7qnAH0JLzCaAHH8qbP+O1eBpTQMOuchaM8BxdlALOeNTTb1c6g2NeN7Vqh1yJaFgu1+5QJf0a2ssK9z4W0DroO\\\\\\\/\\\\\\\/kqD1NG6GrtPDrmsHxZqgYoIGcnlleSsX4Wtdzz6p11mQOZfG1+mOEiWPuZK6Htbk8K1rhkODMLlcW6S9+5yTSA63YIfQg1QJdq4IOTZI2BqBNwU6QWK8Ek2ickBovjbnujvXWgL3yCRzZ4wB8\\\\\\\/1sEJnI9+BirqgfcpZNsyR6W\\\\\\\/E0Cnkg8\\\\\\\/haLDdNoJ003rJRq0ZEPkUk3HylzVxpoC8jpAMD\\\\\\\/u3VXaiX3mIk7lJmNIA5YJyMz\\\\\\\/3NM2tSslapCXpnMzLY22cnThnwrJlCBeRRVG6ZsWJagn7qyNx1yFOPMPFAPkTgeI4bGQKYbiepKKRg6p6GbP0eqhTUWvzTF+hIoF6gBXyf8fiqUQNFnK4fvE0ce5T5\\\\\\\/YFWJ+n4wD7c0Xqlj3AZtpV6Y+FcHb8l37SzvFSxThUbRTFYXtAzMM9U8YaD26Xh8fPyOKZA0rCHx4V97SuT6hNYJbSrqp4G82HsFtrwmlbstUTfVguRsOAKEIciA5PmR4WcdW7LSI3YZeSm1p2\\\\\\\/TYFjQeITkBWZKj5slYTzaQSYr08SOF2+c6GDjMGfF8CUVkrzsqrTflaA6FXwiS\\\\\\\/XXUI00B\\\\\\\/BWx3760Z\\\\\\\/aKmVVai\\\\\\\/9EmfdOhTHrbB09pb8kogQyVw9k0Ok7iTAwq+MzJMiCDM\\\\\\\/ukE1rZoJJgsiKbbbP4gEicrfXvCO9hjYQgGYzp+f\\\\\\\/bp8b4i74LRivhwCBin9ORINulXmts71H4iiYlLwsFpaEPeZ6YzGNagKYS+DQAKw7mefBD1pu5qgTt7QHMvFPUyuAkUVo1DDRt\\\\\\\/KUQktjaRcXlRL+C+Udpbb52tSNSILB\\\\\\\/Oh+L7bPdkgab+KrhxoljRUzyE\\\\\\\/5wZQyuV5ip5KysrqmE3mt1kwaveq17WREovHwph2o2ulzc9zghcH8JGI0pwqfnegyhFtEmKGkH4SQn2Z9L7MS4qtecfDjfOWiHg5VqpjA+qj1wZE8Ocv1hHCtQOgPRJnyeXGak3dSbZLmJrgk8JocpaxETWTNqYw2IjTC\\\\\\\/XATpQWmOBf+CWMQsz7dyOQBzvds4okS49tNKmBRlzizZCzVKaLmo0SG\\\\\\\/mEQeX\\\\\\\/as4JIMJIB2NybmzWPp7J8LxG4jKge1CHWUt\\\\\\\/NceUAIbH6fc27j9D3KUfi7oSVD5w1Xea8+y6eVyT455\\\\\\\/x6phwnu6GWGQc1tjWrD7VvME8TKBFqiaEYblm1M9HZrDs2CXSMi1a2P4t9dcqXSLfTzrkZsnTZvaEUPN9AF9dYuWvFAx5tFtLhQ65qji0EPXDfXwrtCjN4pUfC8BtNqbuedalO0KJ1yZGRwXt5c\\\\\\\/WKVKjekJs9XQYaUWQwevsOu9A9H4uB8FaxnVCcZ5hcn\\\\\\\/ut5F\\\\\\\/IqLCyKE7E877ie3MUItsR6P3He1tldJqT9A2Ob6Vja+pfaSu61ozS4hU\\\\\\\/XA+fUxoBNAn1dL8XoYamFJ+gUfRRXuMpn3vOhHZTpNU+TZaxahhcT5nJNeSKYivPIDelRPixSPIR1cxPnamHuFInnw\\\\\\\/X7V8Cj5nk00tveqLSoAmEAUrJwhjjXzrKHAZKRLzFiEXVE6AQI8xNJ7cfGdQZJg7iMinw9XdK7j9uEQjzQkYDSnkL4U5Ps1lKmbdZauZwtNlb6la3ZHeAyrkFrAoxZ45deSiSdX3WZegrMZbCIdt5b6Li1FEYfOG3s7U41teqtStUN8g6YStslCYExOX3dKqY04xKwPJ8rFjqFrQ58XVA5I\\\\\\\/t\\\\\\\/XuTZRXiL7E1tAx7ZbeKEQGfs0RD2fpNlUbfXtv2mH1R+4eftpVEpFHjtaIwJPR7dbX8DjdtPvFexRZdXAeyhOhIsBGu7pm83qYNSlK+lVvxzQrrUQ+DtY\\\\\\\/9sk+6ENjJIKAIPfwA9XG4EyRF3MXWVeiSBGNB\\\\\\\/KARTn\\\\\\\/WN76MIALLLc4PkHsSZQKNnmJRKjckI2VyyJ\\\\\\\/IpFTdWGg0P7r3wQZbFr8pMhqz5mjCKlWZ8b+womUSRDNPEfSHEmHePHaqE\\\\\\\/JV6Jla3UqLAe3r8ctamDsswXtPlaKVd3M1H+3\\\\\\\/j+X7RL9ow2vqIXuHSyCYV5vy7Xdo7BLPwWscneUnnbgXGU24csC4mubNq\\\\\\\/\\\\\\\/2krUUUTY0t7n1MBWEPh4OcSPz+w36PGzg+W2ncnsBh0TqSIFnnnMAMckDKY4FsoYSTFS3YSvGgcjiD0d2IcLZCkq0yew0ULy7wyguj9t0rILjCnusVbnV3kLWXYosajxAg2QNNvM7IPcle9tdoeZbXqIeuXD6jjPCGRYREzms9GCUA91MD623ajsfkjOMqKTcCUifZW0H1Rnsw2swsT+4C+Rmh6C0QAeOVTMIT0V9y8z9RLTCNXzUuYv\\\\\\\/z8658k46GGT8rQyVHAI2CkGFKZSBnMxhyK0QRyRuLsjuOGf+Hbqw+VJqWVcc\\\\\\\/rrQuo8qgKBTekdWfBiAJEJBqdbs1ezKouDrfImcjDW\\\\\\\/p4LMSlEFPnWZ8TtfUH2n7yKazwXurxq0XpGaY3KqduzIi696CU\\\\\\\/kiKBm13MgMkK5urAtBgi0E3fgXT8JZvueB8qjNar81pXMXon2g02+TUw156dn5g6KBujQLUqDd7dLjEzE7+mG8MJBzbUQcCVO4sC+xvG8ImEnQAKOPXsw2WeFUK\\\\\\\/0FrnZLYAzNkQv3k1aWnLlgPdw4bM\\\\\\\/yDcrBNo57dQ02bLXtGASnFgVSKe1VVbWZo0flHZS6k+ktQ553GN3Ii3geHs\\\\\\\/gqNgZ5Fp9eN4qXb5JUL+XVGeXA87aU4dGW7CGLfEJSRcmiqya4frrvh27LKskMf46qs0\\\\\\\/e8Mv3+VE\\\\\\\/LQC2xdalvUahLlpE0FXZbZ3CKN3xPZGdmqOItt6J7Lo+s1wV4Hoz7KviHBoX6Wmwv9SaUsIE49ss4f4ry1oSr+2YuppfimPPHqy\\\\\\\/gmVwFYs8Iqn4eKQN5Y9NdlfhJQY4nL\\\\\\\/5oD\\\\\\\/\\\\\\\/s7KIf3eunQAWiXzs\\\\\\\/emE8Qqa8YPMlrCXJFzqXzBjsmjcyACYjmJg6FEtfMG3t1qWqzzYsufztVAFNMSrTSYoQSS1oZ1T2ZVbCEpj\\\\\\\/DdnaHnCDGfYZjJXbdYxnRvzIx+IU9\\\\\\\/oKas8uqcLYhwzrgYq1TsM0PySXhnoWu60Q1C\\\\\\\/QJ73RcaAbhUrdXL2eFA9agYLdQrHx55l24\\\\\\\/QGVaSUv7aG6koF0hV4D9NPHfcH6OGUjDdk0npng+FmH+QeZCyRYHMDfo7\\\\\\\/adh\\\\\\\/b7kWuSn1V\\\\\\\/gwUh6U7FaEaCo61lO5fMJqQvXTAFIGOfIPvrlO2BqpqM7tz8SdUAZQtVnUGcG+7U9zcYUbrHbs35bJmHJTmQ4zqa1Uz\\\\\\\/cG3hsRpA7o8Tk5+qadtRuJC7RiaDG69wIgnxrEfTnExYDKEIipwUn9TCqcn6GNKr+LwfFB0IThEHgNNIdW+n4Ejs8cXofuBSG\\\\\\\/e0LWHDRSlSzvo\\\\\\\/9jrqAtR43gf35whF5CiA25senjdoAVC3MhWMxZt1awjzt8wRlSjAP7uQzSZk4BPlhukKV\\\\\\\/xEERYx2DttdtqKUOWVDuQCRrCldOTBsrLBINepw6JIeOcjGzleWcIXbmbi6\\\\\\\/Tj5YkXTXd6PqlTQGfiacsGa\\\\\\\/dR35\\\\\\\/uIBMKrAk1PYO8M2GnFEkL39MRqzmldiZwkXH9eYpTdNHZb\\\\\\\/TIraaMLswkx+lKL3HvEPYWPfHKLub0l6TkGELpASFVujsiYovzDWGxlH7bi8A6CA4pcN5ZEPxi+fCKG\\\\\\\/\\\\\\\/KUEGZu1RM2hGU8PTLIegb3hMkoLHcBpzSW+HXF3iwbFybptPsQ1FzKBjg6WKgx+LEr0krBzQhXMn5IUa74CBRmqphePJHfvgvePIHdQ9JdDpQWEWm9oX83KWTfvC84JnPp6q9uu5ji0G7TzhdXeZcINTfsrm4XE2OskxGqU33YZ6ZKYKBg6EAVhlD+RJ8VLXQO1PuM\\\\\\\/V1VS\\\\\\\/wJI+FtUDO35UF+sQMbzTQUTHQXGiPrCjGdYHHOBD6WZJViJaqlwF2TXlIz0xWxbc2HkmLXDXkQOgIXDk2\\\\\\\/lgdNEVrLAJgxUNlpy66wX82iB3+XOLUXMz5oL03R4a8tOpSuM4bUFqsSC0S73CzeKNBc5u9cIjhZgGUZDVvrIo67X+FtJgVWP1GndsTidYqQRaoADQ2wOeB38llz1jHN1WGvPqsQb1BpfKRIA\\\\\\\/WYZCs5TFPgQ\\\\\\\/h66AHFS5nJzqPgy7DVKg5jTlIJV3MLMxPLAttAxFXCWCZDQRUqrmQRgCgruasL6zorlI68cIQP\\\\\\\/wSYPY8ALmBJQJb9wXrCYy8lAMtalMLXggIgmUCgRZ8mfCF9QgdHVxgzqDUpwknv0MSPHp\\\\\\\/KK\\\\\\\/N1+PwuLhdIXPQM+zTg6ry2rbYePSdgdTFERWpTrLiS6O5OJVMr4QNl1kQXCvZgfYtNlyz4J6MmCgIyXdwlxtF7zB0weNOzZ84fIs\\\\\\\/Nm9eAYLcmc4qGU2av+pakXFMKETBW1k\\\\\\\/5wAmWcgy0N4MvFGI0RajC8J1C9xv4Apw49wnUDSpg2Wfqbbqt+nL2aaAs02ZA6DiCwPsIKo\\\\\\\/HzqpKTOQVzX7RmdybdG1LVj7XrdrcaQGu5KO9gY2+C1NFe4V\\\\\\\/05\\\\\\\/wBJsSHRmkgnAzRWY4W8OOLEQhQQV+2hjbWJlqGh3YpkmocctrcqVCZi7GtPVRQhFctS\\\\\\\/HJvyqNutg37IAbQRcfGCzbdPPYipJkMmw5n\\\\\\\/H3ijDpOw9vqVNWEV5eVGdoWYl3dHQf109hbUurc8jdvuItFxtopjcrfpJFY\\\\\\\/yQ74ktNBpsAHZMB6ueaMWYjKNSYIGXKzT60WhwZ9uMv3FWo9GXl+vWalMIVM9AjciF8njIkfxsvc5ImVkKjuokKLg8DSoeIJ5m\\\\\\\/813iau2GgAnX\\\\\\\/NVQWn\\\\\\\/fdKZsICCu+7P8FYeu\\\\\\\/UOnm0ORUgJl1umOSOP6kDG+kaZKXm0P9hEOrvjb9tZAUa0SaZJbAGIch91H7Jwe1d\\\\\\\/kDkzBbe1oo92IdjblQZknDXjJdY1fkBVSznHfq0ALZl\\\\\\\/qp+mNm7UsQYENePZEVXm5DayJoABMWF3tAQu4zruiwj+NSh9PCZsQlY8ijM+4OKJY+3X7p0R66p5SqYW\\\\\\\/LzHI+ot1HadRIocBa56UfZkJfAD5TAebNFSa5tpy36f8aT0HzTULm0e3GSc7hjzFKIH\\\\\\\/LGA3wmH0GgQBg73R6f1L5urdKExuko6yWqEp2ZyKljNaHb382fc7CR11PbsvZks2KJbDwCJKbqhkAkG2tlcoxL\\\\\\\/tJC+yfPDZqSpyhCLlMl8LQKG+75P3Y9UM+cg84NjLvETkrrIlHDVxLlzdmWRlREClT9SkHEd2u9vNKRX3OFC3X1B1Asl2l\\\\\\\/PIKQz2GGBhz\\\\\\\/FOjEE8EEQkzkEo5EKq4vtoER2OMNc3iBVYnEoJe4Ef3xrMq9tqP+UCnzPv5xnRx\\\\\\\/VI+7XdYlCrPzUvLHJYn81vcBNxCzAZn1UMhF2Swf66vbWa3QN423Oy2eQ\\\\\\\/lP5NGl016l6LENlW1\\\\\\\/M4Pj7Jm6xcnl8wU1rghT9OHhaA5OF3nmzzaeEDKabZHt25iOMkGDKdFA39inDR8r3BwzYltFQKis4Sb+u0Wz7+s80zfrlowmnqNm7OOmyyaIpLQNaNl+dBoJNykAsONYAToOA\\\\\\\/k4ah9kPNk5Kk515twPm9ySSJTdjaWI34D8jNYz0OSHGl9XeqVv6WBqZiO5xeFNgv5FyjYxy+0zh00MFjPF5uZ8ULR5P5L3SBc7QzVPVc3b5IIeYWMVKD+AJjBjpMFPFNRTbB27gRzWEMJa+6mAssfLxSKzhDwBk+HaX1j1E\\\\\\\/VH5Ew2h9i2qI1W3AhshnLu3Mo6gJVen9c0sNTS7PdiUQvEsKL4XvWnzvJ9hjTVd0GQOWRSSluE\\\\\\\/\\\\\\\/XD2jz61tkMHIfQTKVyMN6umrWziv71TtVHHxdK20Mg3lvWs46d5z6AnRpRNUAZIXRUjV5uUlNFv7R1GQ1dG+MYyi\\\\\\\/+Pc\\\\\\\/dM185AgQ6ZYZJuqf5jDH08VZ35K+IIWT02dQY7EPmNWAp\\\\\\\/PHFz0R8BXjxzz\\\\\\\/1zOwJygB24HoYE5fI5bsF+SiImWwwxf56BB8xeSRjy5u3fUuOgBRLloUtTHNmJZZkPtnlaYBcxn652bNND9kTIBRPNmn2tV6mVlrfECxWKn7mJqZ1X7gKrsrm+rgpGhEo6KdiN7ln5M370ZbXUcTCl9gcfc4Hyah3Oyn8HOvJ+RrpQkNlrk4BoagHI9iDAOjvHrV0z9AgLVdTYS3iQDEN7AOF97Um8q0zEknq8GOh2eV7S9OLNUm9RyRSlltSvt7RflXqRqNcs3Lc+7FNMrADMUP0A4R0P1ih6WBRtUW72pCW76n7LqQ8u98FcDK086hFi8NH7NfoIMIZKNPpP4T63glkCsYMJE1Nk7nM+IVWRuAfgKlbjB+Ogm7w6mLtGKnW0OtxusZQ44dzeJHdBHU5e+iwbgeHb+igr9RuMjsTf+5EukO1Li7522z+kSP1BZovAJ7OEe1KoJ\\\\\\\/nKcnU504j8DS1wg+DyyKaGAWAZ6TYPjtrDUfvar47hx2l7Y07FaHkbINBa7f6WGcgxQOOBBVTZipPpK6vWKF1Mr9MObK0VBPAIxPfB8Pc0ASqQSi9zXR0IR5oyY99XOo6PVd2pTVQElEek4HcV0tuAs2t80JF+qE17hT70alV+ri+Sgmygoe\\\\\\\/HXRKK9v7ZXxV440pqyLQYZHPH8nB6\\\\\\\/K\\\\\\\/o4XgXDHj39+EC7MhNHD+eAhisj3L\\\\\\\/rY4rYiAcr2XxmBiTIM1HTtTuaxn0KOLMJ2Zomdo9tnigEH9Z0gBKAi85U4xHn6dUzP4yvN1qOh7qPUtrx\\\\\\\/XUWGyBmt4f4ObTw4K3w3RTP\\\\\\\/1RUy0LEDYx9Y8iGl3cBYsJy2fZJ7JSZd3ON1YaRa4j6bYgGyDHJl\\\\\\\/aJQbtMWAvt87SythoAFxAmJ7W3s1vJrdxOimrQmMkZqymTB8nBXc4ZAzIcfb\\\\\\\/HFXbQZfyBv\\\\\\\/LrXumbeqVdxzItY7fl+UKGeO5oiZcEeF\\\\\\\/7HxE6OiKRWC8bSrZhMbNOsIZXqyd+9wsglC3g1w26fOIKVCnDRYx3+mDtZTi1wEK0nq86ehTCRyCBnhSPF9rvQZxVPPG6oTFPl0dMdqPN2RpVtUfB2Vx3gDKAseHJLfS5Ea4m6mUxox5LM\\\\\\\/Z26Yj4J1+HaLMXHP+Qo5zxzfJdQ9zorGLZ6yrdoREpsgJ3muNxdaBKQZl4rTFCC80hSrU00P9pxoDKgQ+82\\\\\\\/LGsJ4IVEmESLKxDoPwFAikVqz9w5NsQyRHG0JBDzRu6jXlaCl0nsVSeJqt4jQYmINbA2BxSrRXa3lVsx5NM0n4gFqLl8qDDUuPPPKYPP4HLmVeSsRaxwEfAzLvPGXHkDYOp7U7q\\\\\\\/x+dHIjSXDf0PVcS2CIZPw4y+68F\\\\\\\/hdlfLbAzWB\\\\\\\/nN9aj8PxMOMugD7WqJ39\\\\\\\/a0KgmjPGQWUCu9\\\\\\\/A8sEKaoiD+EwCbQRykzwsVFXyNooPRkxaOk5Ap56uB+qbaSBbK8i3zUi0Pibs4z5srPKCHAWT3aFueKwxkENRbpckg2wLSQ7OwIVvpyjvgfpwI4ACGYIGoGd7drirmgRYjnkQkdc6aGr+6VPMtk+F2MWUUWiMZV6\\\\\\\/+Jdb+JnFCH1kAcrwmsw53d+CT\\\\\\\/K9PwcHkFjauJSCMmPtBnAOuL6UQtfWN5L6chkM7uqjNlAMdOpoxpJCnKKGbR\\\\\\\/VAz05\\\\\\\/gsqYJLy6y4dBgdQg329tmkFNf6mr\\\\\\\/3tFI+vU61X6I7Qp4l1\\\\\\\/oRqni9zXcDSgzOviwVk0yprIkPwW\\\\\\\/At7ia1IhMPofF7FUHd7494A7vGRl5b+8hWsk6L+2jiuSTddjVoz7Sq4YHF\\\\\\\/AIyfUV2zMJTvJO8ZY1b+nBiYEPrVPgH+bTMhObeVwN6YOKbrOG14vRq8pP5LpNrXHHyU3gQRv7b9cNURxS\\\\\\\/hX5rjEOQ8ELWId+Z0UDUafAB9RN1vyiHy+4\\\\\\\/KGm9LXdmTUHDPM00BAEL6qxWomSlS\\\\\\\/KLJF+DnnAvzxxTPaTUsz2EUJtiLtLbvUwashIDfXERGCJkBVG7tmN1eXLNUKwt0NGXQWdWh\\\\\\\/CXJn3Nb7SImtRDUmysiNtjN\\\\\\\/N8wpY9Lv1AeS9LmWG5i1\\\\\\\/zDue3o2lA\\\\\\\/lqNw66x2T598ApL9X9\\\\\\\/D9GvQbgqoKElvD7WfaHyH+HuJdte4c83j5wCU8\\\\\\\/25PYgcwxByDO48aM8AJSvUwxOR7+Q6k+qz5hPa1StLmkAm4jKQFQMGcur8VaPA0hebcGNSYsvAniYBis5eojv6C5Nno0evPpQsgjBurx0GZvwiYhm2FI9YY1aXZ9+HDQTuBhiFdMG2qjWa41+7wAp6vo1ntbml4NJUtjs1wNNM4Fxy3af7YFIBCTpSyPX\\\\\\\/tmW6oqAA1Xd3ZtmYEWCbL\\\\\\\/OqHlrbq6WC1kTK6WD6yCucmeN6EkOTmpuumnEtt5HBTjcEd+As6kX15TRdXFM1\\\\\\\/l+VGM7dz8Th48IFoTjltd7NwHnKGFoMzGagA5IXXSgOdsSOu6YLZ2pQW\\\\\\\/AI2fMlEeucOLyV0y4ns9mxU9gXzEcjILeiIiM4XY2L9zqR99sNgQBFO5zT9iiJZ6irMl1sP9Drde59frrsVHGRM\\\\\\\/3NcD4OJbhO8j8P22st9JgsOfRmhMssIsKPw\\\\\\\/gDc2VdMnfLMN7YOHS9hLr\\\\\\\/k0i6I2qBvELh4Pa4H0qTNqLlniu\\\\\\\/Ld8bqRtd0nJvoRoB6Rf2r1K8jax1LexBnriVaLxCThc82oHr94Civ4hcKVdVP1uXFjrveIZpcfv\\\\\\\/QF1eyFQj3TbIpnkRUPckEq3hAWIIZiczR4P4WCxVeRDY1mTY0xT0gtpiWSsDU8a5zDp8R6\\\\\\\/iYUaFdxafIkNKLdkMbg0l8WUs\\\\\\\/Pp5yir17yRVMY4dUT+hUsPv9b77qfL0YljpSgex6rGEo4GgVZs3GvVcy2vwgZBNgcJa3NuthOCnQVeZ5SjL3HZkrQ8+o5HTOFHZ0Ls0nMB\\\\\\\/SALvtrG1g53YpuMCg9g36v1ARBc2GH\\\\\\\/ZdP08nONyhMPujkNocK2vB+4XIfIcbs\\\\\\\/YhJqLltTkmjQGQTCeUgUIbydFjA6oTX18zlLvs96P8ynMs63TiltO2+vtpEqPJbuHequvZB9lg\\\\\\\/KNdDrFx5jRop8dlmcjA9LbMvxbq22L2mlaWVoSnvsg+sW1r3yxi7IRbaila5Ib+6gWMZgmaPbeMWSY20Cz05Zil4J2FJnFxqj38aaAnUo0yWJK0nHlFeWmFYMEMkTe6lcE\\\\\\\/Ovl1DfOBb9b+cOMZaSljmoIT9q7jPgLBhUPEbZ5TSqtoUWuxjriSDX8jDn6MShihjThXj2+PJMjsWIR8cYH8arg07w52aDxTKF4HLALRKAOTBFZKwjBk77dbZECjjptyPEKqMkLLU89h4J3cB1r9WAyJ6D70nB96y++ARxmBz9PldEiJ3mPHztbzxIBqQonZYLOameqd3tknGNSt1DWS5xM8JovGarpXWZ1A9rErDW9WfMSOn\\\\\\\/l9Ao6HUKyxYLyQg6b6xIE37c6URnzDbqKMzJETlxL7ZBXcM6aztmbO7hPmz9BuVF6kyMFotZ6xeq4GKCcj5zHSOKbpfBEfzgFFDvyFCp4FdSb8JHTnQCeGrqa6J7SG29+Omp2dvvF\\\\\\\/gWrhLXGwkwED2F5fMf52vwAbrYvhdMlP1rP16Kr28sTTeEjiDX2lGzqsjO3BIj0expNHDSI1luDXlb5WQl9VDJD3c+KCbLzU8EAg+2COqCxCfrZu4hoXwbhmVOS70ZptK4W4fM7NU8Sz\\\\\\\/bOT1QJhwtnmugwlxjRL9cIunWBd9gRDoOOLxymi5lqdZXqihLkIKo\\\\\\\/8+RVVoSjOrRhtnessg6pqBIpycTjGrPD3ggF1gn+HG06tWtmRugTPXn6t0pCghsdhA6M6vv4GvYKbC1JzmPBoMbF7DfnTGUd6xGpchUq4A8NTQAOyfgZfF3MT7ILCVhBlcUhGyZk4Pxyol2vK1d9sPudfELoUuY7B2Yipx3SEFL2sq0hrhEOlRAQ8RZsnwFuwpJ9mSE2+cbScb1kdvsd8TJ4cNnqNEvxkN+ecX2s6lPjrSKMJ0y3iL5Wz3h7cKb9YiMy2h8OzwFAcWpkyadc+mABr3bBGFzJq1Nz\\\\\\\/e1CbLWcQU1HNE0z0xGJzntlJ3Z+LejAKuDYnsVqYWZ\\\\\\\/w2+6ysZ5vpHHrKmDhu\\\\\\\/lTgO2p4jlrTCEIsnWDvwa6xnLRREcCBCPIlQzfNd3P9zByzwMa+8f7PNS7LwrXrggIKmbjCOs4IhLNFLKdqQ0IAC0LgzemUjAkqW\\\\\\\/ltryM+2rscKBqjjYYkEt2loQu43kdJ\\\\\\\/NKaKmtxcGE1thmBhuS8CQOTuhvnUtcyXN5uCZGS7R7BZh87kA89Y2yYH0jG2YmokvigRceGszsOYsBmUPRDsRP\\\\\\\/2n1FYZd00BTTW1UCXNX9zxOiCwfl05Ux2RAPY+XMOwvxTv4Sdx3XPjw1dQ1kOZE4RLkBj\\\\\\\/YoPkKcWMxhgtUeX\\\\\\\/qMZDFtSba3PMHxoh5+EOLFUxKBjFa+8Zud0yDUqK4UCeawudz1c51RrQKt5z9vUd0oq950pu5WvBxDyiP8Fds5hVzpPNgJu2jDkAr1LUZ0kf3EXTZh4zkffj9fI8Dva7+m1NIDGdv0LwRx1gP6fAbSWbCkrpfLXqMtmGJV6W7addsfC4wMmU3zwh19h46p34F4dI5u+8Z8uc8+MM4abYskESlfK\\\\\\\/6LlezdlQ5E74aeoHJjBeX7M3Ndo2NVBMGg7af5m+ioxzUFbS9D+bg95hLBk6SCeEJzkbQH7V0JKJhNBYez3UMuNtIQE0xB3yGEUQHYLT9vMiJSO2fOWl\\\\\\\/227Ryc7B0JiGErmuDoSd\\\\\\\/gvlacL6A8S+ycU0\\\\\\\/6599IZGgNTcs29EgGhlLGqPFLOUCw0xvOs5NUbQs5H14BxXPLNfiyTNiCg\\\\\\\/UPDwL9xg29GPOM4kIwZenvpGH\\\\\\\/vYhRPVcpEnS3Ecapw3MTezmKWiRM8NZ3tOzluUrhCDMFDEapZs2lW5VVUYVPhDT+ixFC2KvtR5JwAVJo3yQkCHsELH9dhAkNNUiJuqJT9XTNcFevKGn50T4t5L0BQpY4aKp4462sojso4zT6b47teYBLN1GmoUhCp7qf2kM76J3A5ZmKA9ZCr1LmjYwyhsHuYqbPK6Ulyo3DOLlfRdwmFmKvjiuuQZENSXkyLNsPM4bNXxQHaM3d7Q1VzHvKcDDJuKPmO0tVrQATvLbUP7B8cE4wLYSjTFeUKxk7kJ2Y45e+iqB2fi6bst4VI56Mh6KkrI++sQ1hbIt4oQweaKcDJA1lUHszqHVse3aurE6kAYbazh8qT49lp7Y3Q7qxrG7kzFWGHBT8YKhzJ+HiYZT3yL2Vf3WMHQT8AhYbDBjQvg7AqyQ92fxSLcBSYeIY17Cdyu+UB0bibqCfJUO336YOszQWIUCY1\\\\\\\/h8VW8bhpgj59K4lWvXpi6bV1YREOiXq9qXZ8+AjW8zgqeHgkq3xBmnkANo4TWB7\\\\\\\/0GCDOJRNOvWt7A1o0uzQNxydNUruuqfxl0iIryscgRsQ+DUnncu548iWDx2KE\\\\\\\/vpsPMcP2Ytxm9hAX48bM98bRRZ6R+RcvLrcvommvmsjFTk8zMXvlv5PSlFQ1h46Qm5CsK3KgMseM2Ke9VrDizyCNrUvkxmHfeCpbhMorogD2C1uhljjVrvINh+fIhf0AxNz0pPpTXXVqf\\\\\\\/svEJkYPw9Iw03k8z0Hxt914QOZvHtKhMNUI0YUoqqv+0Vocc8CwZdM2eu8UxqHYj6iujeSNSJdDaqfoJEItxYk2iynbpGHQOmNe4ogb27mHv\\\\\\\/EwqO0vqnvF7aJ65d66qbKalRbVfKeAxy4CfQC7nsl\\\\\\\/1wLaNTNegg0O9Kb+uQtrNiVMca3QavqVKcdyJo3ggU8CxEYbFFIPa4xsldFJy92vdadKrtQaFWuIw8cJi5FaWAWwMMNpTY0SD88+90CT8fDb5ucNvh1k9Ot3pE978AY2TrfYEBctC3xU6B5Vm43KgJp75aelBT6JuBY4bao9e+q3tsKZw4xpZugIkl5byYg2gDHmh+g8ZldLUYieUJcOLN0gBfDalFdDYsEkvDwMuoV\\\\\\\/Re7eESW\\\\\\\/t5OlZMFOyys8nrLWvmL8W3V7D1WIVDpUsfHkQevPN4w8QecXT1OEMDknid5JPAnELDz4JBmrMcp+GT0dcl6JCV7iGZntckUg98lca8gs3HIXRDQFGRB\\\\\\\/7AjDQmi+PXCThKW3AQO+cspzbXO4f6DFkQghHjbgwNBXMF5GuB3ZfgQTMJNhwJY65571T7AsVj32a\\\\\\\/wAbnxwTGiELJHvP7YA2KJy+QFsUWL8mBK\\\\\\\/WXFf80ynguODdXf4LRLN94jjKc+MURw9PkPZPml6mf8WCTpBfc9fGM14JZtNOgXGitkEGy+Sb1\\\\\\\/38bFwIv54ZikxASxP87PuE4K+UAHv4nUyBfJdnKw29HUIAIMOpjMoS80FbRMOa80TS4IWbrfZ8NzGy5bIC+P+zNSLyQgxUmlyfQq4NK1ZEoD+Un6F\\\\\\\/urSdG8pjUsUB3laEf0n+4FtJAwGJ90JzkDS+tOUOGlUKXKtmyoYOqLseU2N3lCkbUk4stgfjLvlJVIwTCJa58Wkc9Mk+T1+W53eQQrrnxrN008mrXT3NrgDyduY+X4YYVmnSN8I\\\\\\\/cjM2UM9BwzNbDfBQl7MJ+H4gB6jW23f3G9pNYV0eKhz2r0BenLuLdNPS+aWDHoYX4FF\\\\\\\/6w56St0LlcEKlRcgqACotLjSPk0goS1V3gSMpSmehuf8PxdN12DfaSh4IZwqcAGi5Mo7W4PP8K+xMnW9RRUXpsRj89z56XWcdC8PenfRQXeyyqEVAJ6aWR3uCoqcjfEZxLdzEnnfj3rNmpxdImLTmlkQRBUq3VZy5y+gch1flMksJbdjJ0kMQHmSwDIofyDgWihDLJ46RxJOAP6xLWVGc8ZaGuhVrZDDqXA\\\\\\\/qZYWawANc7cE4VBLfWl94YviFSy4C\\\\\\\/QpETJO2cuhKY+T+lTwpAgW6q5X1INszw9YOCVMgcYLJlLUmlTTtLlBJJtLmG6Jlz743Fpk\\\\\\\/rDZcTFiIw4vrs5PBV+FwIM69nRUzFKGEiuRgjFVD\\\\\\\/mDqidp988iZZa0A1d\\\\\\\/T1+4M+8SktHHMBp8R3TejDsuYqdf47fRlCE+E8zJV6n\\\\\\\/pGHa4s9act59C+IN0mXVZRm2SDo6LyvbQBL+sKpNAT5bTARtxb7lHP8h\\\\\\\/QdfeDy3T7oqvzChGgZKE7hGSoejExqwuKT+hjQl4ugUIV\\\\\\\/WEuVpD2dg9jp6yL\\\\\\\/wCJtaAUVwMKpvjrtQvjCQS4DmOIGUCvl4NQLo5a\\\\\\\/ojb3dZzVSYpx7riIv59Yi9ED5Gm03fs6aoquAXrnAoVYt1eBIGH3hm2Fn3SuexwIT23nyVrD6rCgiQCXUYFxFv8Ezk0xj0ml\\\\\\\/QaLcyhrwBduU6x1kAGtPE\\\\\\\/W9EbaLIzYT\\\\\\\/Y6FYXyeXKetyK0MN9kpe8iY4ZclrKKEK2AlA7mLgZ\\\\\\\/IESaFrvh9RRsdvW+lIt4P1P4zO9gP2VtstnuYFreKHGAp3Vp7cmEzQYGQMUlvruhRYm4EGt7ClG9fD1mzLMejcsCZAMB3GOn4f\\\\\\\/hVEIcSffNuHLmZX3+iadOnRieoQLl5QE8XtexRCJIxtRqRc\\\\\\\/m\\\\\\\/YbLmk7xpDSttC4mAduY9cMXrgStZ673OyctF8k1wcl4F2joVHHlZG2JxbtluOMoLjBMZPmtnTe1qS0TqMzLIo9b6l36JXNA8xoIx8CelPRTKOim0J0PpbmNm\\\\\\\/KRk+VTjmStW9DlhO+UlB4LHpO5QcGWl5y5y2RJOw\\\\\\\/P6HNUkMNaGsYwDVs6BFua9buf8P23BhkeJ7lHMADwq4qaap9v1JCgnpGrWeqOjkGhAYInexXhXBuf8\\\\\\\/QgMANuT0BgL1LiEGk3lAghOB6G0IKwac\\\\\\\/S8W5kcxiya7WNhO8pvVO4\\\\\\\/aLSCIwsLWa8x0rqCGpjUt9w2fY5g\\\\\\\/3ugT4iQo\\\\\\\/VVfP3lkIZ7EuxhlAGLBVCdO9qdjUddkIDzPxvVtb0WvY05F\\\\\\\/zn9uFpwIDylTIWhXGPjMPjHFVfl5sLWELqi7DCGT4adc66ub0wyXblFIyqqF07\\\\\\\/dAb7y\\\\\\\/NKPfEOf203\\\\\\\/fi\\\\\\\/u2QgTFBjjPZA+\\\\\\\/KUSwoEwbs74\\\\\\\/twmPihAXmGHpGaz+Z5\\\\\\\/NoBBTfKMMPlyOsp6ww7Ef2QGpcil\\\\\\\/aP1rmu4ZRlQ6pGrwxKJSZhZzy+dToGBxeqMf2l22VZJebkVSXiLHMOI6zaL9EmSov8oi6G7Jrytkh1Kl0IO1in1uGzHsJikxe4VqildwPjX8po\\\\\\\/E\\\\\\\/fctvWs8i25CBfY5twciQ6t1XCaIqH3MptJTLg6L1+iFlXvaeDHbjDv1NIgq2OW0rQjxirFYvi0eYAmm9KTGFKyLE8cDR06bwqf3JUVw\\\\\\\/ifn41eSc\\\\\\\/8UtbYgbMrdNyjqHQWg5oKg9zrrOmpVsxcL5StnAAu87xht1V\\\\\\\/vsMpM6GPHVcBKJEdYp7z9P0avPQ7eFhb5gB\\\\\\\/tv0FKw2LVdeWBF33nnY\\\\\\\/DYC1zsypkJZjuMyq9I2D5Jh4yS8DNis8M2fJfIQSYNRMX\\\\\\\/lqVN+lEvqkuVxMXDAKn2ClS26PJw5Do6bWUJbKQO0jw2LHgFciDo7JplSNx45cm4rv9XZJVbEFJUP3BejVEn+8T3Qe4YHtnr0YF4ENzcdMYgYYiOW\\\\\\\/g4r7Sf9y+FKjT7o4ZAFQPhVNoKYAO1FdgAgSl\\\\\\\/GJKm4xgT+mmAgzCsMGbL7ayQATOZd9k4fKLoGxaj2267EeqjTxF1fCNiv82OS7UsPOJMNHeRGSIRtMErH5Ro\\\\\\\/yEEAMzGAr8FfZGF1mowScGDp33+e8DRNCzWUOSf1Ajt6GHZ1kvN2Muu\\\\\\\/B4vh5\\\\\\\/XSGaobPKL7CLSkqycgi97O8mbpu8jJNOHSxhr\\\\\\\/9gqlESg2Sq7jr6hr7v7GzWBJuwZHNKlv8nShHDWiugui4XtucyntvknbCJrDMf8rssDXGAmOjGbVGOc0Ejxn2j3kc++gg+Blmxrs10647peuD69q9W7UE7zmmVEG2B4y7uygZFDAukF1o3jCUpq5ihK25PnTKSV8li1jNUBdeKNaIXxHWYlzVK1kVfmUST3E72xUe0L\\\\\\\/d15ueAP+h9jNmD5eVNAQECSJPkrwqx41HnmVwbaafDkHrPBF4PNG2RxBQ6rsCxFJj44TBgRvU\\\\\\\/4neySomzUCLp3X8RMaSJyIGwUxXHUTZQ+xDxdID5FQSZ4NjEp7+wMSuMAs3l2A3+y7JpK9DXooa5hWLYEpD67vmK1C7sHo1ZlGGf1Y7Qyugb5Qc7174eO6uNj6sxZf2d00Owu5gGsPI0hkV6FMf4NfLYnxE5U5Ruxv2qn1G1BmBT7tNbP2ANejp4\\\\\\\/IOU\\\\\\\/S7qwsWKPHHQnQeMJJw7u6qnUax4SWPI4muUhh6DHqLxWgeWmBO+EkMx1R+KckLinWfTiLfuCm6Te0pvSR7o7Yu8tI7WvcFC0pVjssaoWAhkGmBWtDB4pfTvY2mVAKtR5o7mOW5CoR4tmBQVfp7+cIHbR3kCY7OCq6ukIuhUk0NiOC39RCone5fFjCa6VSMS+hgxBBQcOcZ9IGtAH+UvBKLf5tVT5l31neMIaDtWZBFAaoL9J9ZQ+xYmoel0dBz+icD4up8+0X27LtjJGeeFe\\\\\\\/+mNzjbUMiaDuXd42BrkP2bsir1Gl51LdlFgjZxSgOF2Jqw6+5LIZwrViQOosE5QxlraRa\\\\\\\/xgglkf9hY\\\\\\\/BnoSvpoYOhyMxw6pjY03JB0SUVA6jNRKWeVr0mqyVb9gnkXYYLqPOBTzvy2qTvflhH4d6vOx\\\\\\\/F2vqgzhFMlpahkJCxMVmdRAtKsy02Ozgm\\\\\\\/PT0y6h+\\\\\\\/jBfG5fASTO\\\\\\\/vDKOxsjKcbfFXANhh9kaBtKxQZxlRimSetctGN13TdsoxGmedMQrEfUm\\\\\\\/BAVOzYBOmDZwH3e7BEJVCeBi4W\\\\\\\/9agNJlDO73YkKJc6B0Rxr\\\\\\\/oTBRpIkgrz2yUDXQVSSPnG4Kr3cO3Bpqhin6qDwZsrrjnUIW+QMJDZcpGDYmQ4yqcTBh4\\\\\\\/ZAq\\\\\\\/pdlbsXXpLSykpQYSdyq8WnDwHyk2g\\\\\\\/Yfn85idPvNxjX3tVjoZoWg9Q5BIKhLM2sgHsbliE61mToDyc4Tu+7J+5cK2bJjF+I7W1Tvwl1h1XQqf5HxQ6WjEPpG1+XkFvXbK4PktNBa3nD\\\\\\\/Wndv5DgNHzZK2KaUCr9q2rnnYKRAjvL+kUa3\\\\\\\/8w1vw29QANKFDTF6vwBlpFlB596z1sIPXdIK+ux9GdU16l3LbkgHO0fOKFoMPxtMdeBOTzJLpFFXHjXRW5kuTLvRYlFSDcDyT7IDBfxHEztiHY0jNE\\\\\\\/xRT9nkxVy\\\\\\\/2gal7Pp7amm\\\\\\\/CSCHbWdDhuvZxwsEo3y4ltdkeGmH7BmUmwahtVbYa6h6tOZjv6ls4UcIpKNPt6xeHZSevkIBBP5bRo39Lc6NLWqbcs9dxyECDwtz+Eay4GYsPzFL8G8bbdM0+gK\\\\\\\/WTJXKhKgCz6Jd4DenKMOmSl4VJsDP8l5BYwwtrLOnp1uDp0PtUW86qZdfrG2WfhNbcw6OuVQXVX+pxj2lFHsby7a9vW1oGCeaed8GBU3TqsYOnDHIJShtN8Rgl6b9OLXZ81wcZ0fD1RwbSbZ2jymT4JXoH+zZbNWON1b7\\\\\\\/UJk67acde1JvqdcnQd+YDzj1wW+ImZj\\\\\\\/cMXv7uUXzPnc\\\\\\\/LtFho6XtBr6JgkFGHUVWNklXmd5xf5tSUWst\\\\\\\/k0skD\\\\\\\/04TTpQM1NmRzyU\\\\\\\/t9id+hOKQBDc56aRaI7nRUDa2J8qW3W1nol+P9k3wlbNJgSUCkwjmwQ2GoMkx5IDo4UQSi6Aq1XG47MwqdR\\\\\\\/d6B0kXx1sItplCF4S9X83IqjcTBHDf7jJGXHVPzTda9nkeSmLa\\\\\\\/2Zx+1LpgRkzIqCV9uWhoAITJdUGTXTY63r22255IMVlhyu+oVc5Lbs1QOIhwGh+R19Gt7BAmWF6d5vOer4G0OLraC+PQq9KDQZ43kA+Q2F0heiHjZPobBBTVe+uSqf\\\\\\\/RR3j1u4rdL8k9PhRYE422CLSpk3QPahkbL1Trho7UtlI4lcBm\\\\\\\/\\\\\\\/+3DDn3Z9zEa9pZgxh73GZ+KdC2Rmj2SRe9k4nVYOFuedBS25pPj4LiHRf17N1Jj2LoH1lqc6TK5HbvYPu4YwiRlQ3p4Nc+Eg9K1VSTa7Ldcmvsa+hJanrqcgihqJdMm1aD20fQcvpusxav\\\\\\\/eU\\\\\\\/9sRz\\\\\\\/8Vx0f7b9l7ZYxgSp+ySl7VGpRWOOiiq0TEreiSnru\\\\\\\/XvLw+Oiz61La+tdS1iCVixaBk\\\\\\\/nrPLNwO9XohNTb3+87ek21JZYaGCgs+yaDgarqDUfM5uRBHnHG7i9ofccBofT2AamNmc+bvkZhMmpUzya6Ewno687bTKcUJqqVNNu18J8n9wGKWnNST9N7hxmzzmlum3FPEedhrycs5sH2\\\\\\\/tRCfuLiUMqwRUIhOWRSaKyAKp7j8vylvAXsDOW9esg9yu3r85fZpFxgpmf3uRO5HB6V0u9qi69LRloSSQPK5T09cgEt8dU\\\\\\\/CJR5BxoNiZQ+KCpE8BFGe2lqSfh6uWqhaxsEAvq1HZ1S07V6M9jOL6g1hdZwAJqRiGfAxzHla0Mqd1BLf8cTg1822K6DBvIp0Gi380kt7dSwbAXfh\\\\\\\/19\\\\\\\/3Ta2OjW7e3NlkK1HI9FkHoSyis2Bipqgft6SYH\\\\\\\/CeaIOUyfPu2knW2HdYVxH5DODzPIePSglY9zBug9op6C\\\\\\\/1mBprN9mY6k2sfCaIF5qe8cd2qOOinCusLNvLrtqiYakvADD5c8y3zhSJdBluLAkTTxCuzRWzeaW7SSCdpzT1XiN3SnVjlN1ky+WEoi083ctA+oe3oSw3XOQ8ESNhNdzDtJnRIUMEhtmgvWNPpSgMEQVMdofCyDsvYz9GVjYMnofITsux\\\\\\\/HYSshfLztW3cyUswFbbY21jEcM2pJ1C6aIWkYywpzddum6SWzNQwnd1aoKW8BuviW2WH5VwX7qbkyCA5IWNP8JpAoPYq8kO9kuIJur5e4PRgNVosPsELh4ckpPYb1IchyJnHZtWkA6vWTreMmAM3RcxGRMRLQIs6NC62TRNxpY2MG8I+chuRB8GnGbnwvgW3AgWmtgQBoQ3pkWpIfbzU10FiPV2I8cMbVrZ+Os9N4BA+D5RHzzYsDecE3gg0oknSoMmyWrA17FMCywVtji9ogxy+dK75Y5475SCP42DQuJU6+THQyi3+FpAeynvIl4L9WR+h6UeSEssCtvyPGdgZOLHjJTgWSu8QW3VGIZTKzt3i\\\\\\\/33aLqAkYq+fyNwBUiYdD1CryGG0YOpB8DnP46567XjYkld6GPtxOTQlshjTVdwrJKtAtavfBgKL6fzhTpl7lAshf9Doc7HKE6PyNEgIcSfacT5VPq3rSBJn3Cc2id9iDgUPGol7pEN28OIkFRq\\\\\\\/8Ko8tKOriq3Xl\\\\\\\/lxKbYPeoRsO86LdgIh+K\\\\\\\/iDigXcql3TYvinRMan435p4CK5sN6iGPO69F8dlRp50ylNO2euWby78Zhq9\\\\\\\/RdXonBhNxRA6pZLxPplHP3k1C2C\\\\\\\/nQ6ew5Z01B49U7r2Ie3oWrvslwUUJ0+zBe6HhWKwngv\\\\\\\/cXL0bk5WZuTnF0N\\\\\\\/9\\\\\\\/JDY4BtZHnBrGjq\\\\\\\/Be3dgmNDOqTxXZ6h\\\\\\\/EnmrH5rDs0ykBQFIRuPDWu+2FdHRIK3rvoiXAorqhDoiZ+LQHu\\\\\\\/8izpFRi0qUMHZfV6rZRH214pW66PW4Dh60VLDaOrqW8P9PButzj+AqUXHhdBnqhv0r83PgdmHphuqv53h6k8pK1KBa\\\\\\\/vE2XyEtpfWIDGbB4wzQpZaJS0mxmRQUrgaPHAlkppC+ehoPtNwNAPLOZWDOteolBpCTXt1HdUl6vhSpPeVJ7VCEvBpVjIo4xDAaQ39aF87QUOBsyyg+lefSnC57wZsQWggCdnDtuuNtIuQKCV9+9+RDS6g7mZ1oETWVG\\\\\\\/sqN6bGo4IRCEbo2eV1bBY59VT7TjHrniHr9F6iTAtZGawz2sQvmIYh9nRJKvB34nyNBTtG7ETs\\\\\\\/csU1h1UlXDhX51BKxLklJSZDWBrykniy3laH9\\\\\\\/8\\\\\\\/PXx6BdhH7uTKh\\\\\\\/wlIRHT2Gy72CVndAVtsDLLL0ibDaoiEioNWDUXuJCOw36MAmB39hhMiubPgpNjCP4sXydhqS5Tm2kUUHDd\\\\\\\/uwWeOrS54trOHoL06\\\\\\\/dlnRcHxAWNJXinfEfzkU+qUdRzb5GKpmTy4GdtH4Y+0WfVm8j7UCMi7S\\\\\\\/zcaK2iUVkPaiSbE6i8GIEiGQ26RnGmr0mvB9nHDFJJclNymq7C7kI0TgTBsvvkBIuJw4L09RmfN42VA+8JbB1YT6Tn55K67Y6\\\\\\\/KJaxm\\\\\\\/sPXpitQ+eeL0Z8WivHiDbaNNjlVPwgsEjTSenIGIPkmJ8SAfhYnG5aWT7wL\\\\\\\/0ei936BucOabndbiQDBMkFSIcxMh1OZcfBF5hfGNZWXCuCYF\\\\\\\/IoF4qPXz8I98AkhfCPqkwrZUDnP50TBk6LyoBhx+Yg4JzgnmUP0GzN0TfLhoQ++MdtNiZL86RixSj3eDE8G5tqaBDUDeucnP+TIsFyZmZ+tmGirsay5NcFeKiJODGj\\\\\\\/TqIXoRbiob7Dprr2ehpC77MRUrpvPUoYIfMsRAHk751q7AlZ9HgvmOc\\\\\\\/eFeWVnepRUv00kyvcAS0jf90vx4+AUfpqT3ciXKs7zRLlwRiolL8QiuJ4bDJLzu6ivsOlCOFj+T8d9cBb5NCRcO25e2QMlK8j1W3SyDm22FnfUJOF7W0BJlhVaB32nRAxbST2OMVm7\\\\\\\/xcK6MkyXwKQlnr9PjEygrX1Pt0yVkVFOlyDez0VnQW15LftfsD\\\\\\\/eIAYRY8ic4sUf0CKeNi6eGskE6BWMTa75kWunFEN8iNSZO5jEPif7dOKDc5fZW7PB8HnRte3MMlXijejA9CWuDgXiToI3LzY\\\\\\\/FS\\\\\\\/2SPDJ0vRrxujtM9jU5Qv7a\\\\\\\/PKOqk0kf2tV8qBzCHrrV3O5mgB8Yl+wzBh6hBUHg5mfQCoqlOCbXeILFy\\\\\\\/l\\\\\\\/1ERwod+kOzIwAEKO3GZxa8SBnRQuzfxFEx2nzGQCtEn6wZgpa99P8nI4wmn0f4\\\\\\\/RccMUdV0hHa1iNEHFY\\\\\\\/hJLkKIYof6E3nLiFHlWO\\\\\\\/rbJPrR7xMeDR3rnfqrse5nTMvlj\\\\\\\/p5zU89zGcrUC8kNCbm3kulXZrqFAT66lp04uGJjFqHzQh3O2uHYo1F\\\\\\\/E9sDMTRz8tHci5H+A1Vcf\\\\\\\/0KQNCDg+N+UEAF+EtUPrPGZeCrbHSPeF6IGc0WdKIoQGU4shqHPJPVozDUOY8Oq5dW6wiu4MJ327rPh56OOIjsfVeGp+zeHQGUWk\\\\\\\/qIH3jG3Wman3qfTUpJp8PQOfsB9HGYWx457h+5Wm3shj5BRbKzTNGMWy4g3Q78FGl7SBNKyoZj4c4YL8ocHxudz1RVyfjkR8jha2LjQ2iSN2PwlXnFKVM0lKtUmuxYt3SmZO9CywraUMr4uH4RrC0A3jKWYff76uVmGchJcDbvUggUmMjIEcrbipi2Rge\\\\\\\/rnxIiweWa201w8annGXJoQA+4u8hPyjgxMdwjAz7DFrdiAEQlwOQ\\\\\\\/4xXGvIwbHeiyIEU9xLRPBjsZoCJnPSEqHzg\\\\\\\/xt+pISHrCGHOvtMwwsslBrW8bESFDHkAuZfU8V+x1exzvCkyN++GBpyBgjCpm\\\\\\\/bJWRBWrNJnzic7MUSVeQB8uCUwYsFRDPEenPRQimevgsGi98JOnFMrltHxQarl9j\\\\\\\/VSHkjH1eJjs\\\\\\\/9yj4wZDKrv9Hj88NZczCQpfcEkpwphr6+MPAOhOmY6OeOJbvNtICoOJlNoKy24ipOjJGSxOmqi1ZNlf2s3yd7rsuiO96ogkbhmTdblFNO+T\\\\\\\/tFko9LZsGE2urL+dNICf7O+Ncg9d9zPoOMfe6sN601GNu53aMwhEBGrfv0qBVBDkbnZBQZ+ksJ2aJUgN+AGJoXc8dsGH0wtm\\\\\\\/pMBSorv6G7kMwHujUese4OScDTVPP5ha73gEkJ\\\\\\\/Zv1xKqSI9XNfScIuC+\\\\\\\/6NF5agTPChqFXelQUmNxovus+e2aPJr6vEK4pd87U8azYmFb1jA8NJPUkO3FqcYLzboJdD2erU\\\\\\\/Air3Pful1iYIyLbKXLInEwpE9ubY1H\\\\\\\/v3jcJe4Qc1zUVOBqQQB0tvGuPmWOTossBz\\\\\\\/ccTvdP8h5yAs9HjZUJHmLyZzc\\\\\\\/3Oqck+AMwZGmNvpAjuJCeHnNQJCGSzSuGCgKv3VLrhvdpS\\\\\\\/NfnLEwYaJCxWBTtcyuHBtf4ymV\\\\\\\/HIRKDLhTiaagHAVgf1IdLsVUfqMhpsEYZ7Ied68LYqikHcu6rPEzvGw59x\\\\\\\/6wHzzJtMY8BCvGdv8r+uRtp1pj+3dtjAwTuGxV3zE5b5bVMhOHWxS+uKJKIeCu6laypR7Ej666G7iBmUp4h2VqkEwp+2tTF1cviZuIMrBQdd4+fb3CYSvmzUUwxPaz80dOULQg4joR5nQVrTyfafUZJv+5Z4QKpj+tPw53UGiIPRhp3Uecj8DBiege+z9nhUrrGBmnN8joKQc7tAPqXbt+hWVtUIVJGZUpahXt+PJrqmd7eBZpjBX0tZF+z9AIQOp7eGbAQeIRejB\\\\\\\/vwLb51wQlOpnmraGR7xitryfg\\\\\\\/XPOjkOXB8VpJvQ8tSIG8dc4ZrlemcmWIhVFDJU97FXeix7pjM6YiLN4\\\\\\\/HJ+jizJpKh3ceCqXxZT7j7mW78CDl6UOopKdtnvzSDcU858PH6qCx5YkuTZ1ZwdveWcB8lg9tHCXbGy1\\\\\\\/5yGKpu+0OWhl9aTxJ+1qeeBt04whDr8nsLluNcBQFPNmIWVxh07wraNuAslvAYV2mrUFn4N3aSZ+1H0Lu0HAMn9Xg+X4tMyyDKKHNXt+IuW9gmzYPb2+ACZrnWZZYTSVc5ukLNs16WMuPvbLnnR1UPKL2DCgNiDmK9Ssn2iDpFNk1zIZu4CwnB6HtOp1TgdppbQ9Mz3y8d0nVzZ3XUUvoTE3niQFrduWzJbRWCM6DeHZ5BPkYuxFMasPjxl7aNAXUUM0Nw2oyj0AJsCTR7XykLM5Ts\\\\\\\/RdoayK\\\\\\\/7vTtmqF1n9ZQ8cLSeCUI9LRI08IZGCwOsQyosjUIBJ3hgRfAIBh63sF1cNMQigqZxEGlGezb0UfCNdsflF+y3ig\\\\\\\/me25RX4TGWDnSgrIC\\\\\\\/mrG7zLoCNEV0DE+JRigFS8unCz4fgv0RL+umoLmLB\\\\\\\/oWc7CE9GtjjumWzBEjzoGxLIsSmSrshIAL8zNBpgKc9MxdeBZJC5q\\\\\\\/kqduXxw9IYug9KhDvFV9QnsrKw0xHc1T8v6nnnGJvIdCO5MSJKbADV4voNeRng\\\\\\\/LZLqwwIzqt0EMrhTGfILXKeKYI4wlKgcZLP8neKxDlGdTBBM16SmvcJHC\\\\\\\/MdEYLP4uMv2A\\\\\\\/53z7SO6ZYZw8KLd6FtPraOGI+S0SzS6zl13ZFbAgQw7sPG7BQU1xSOQB9DJJw7e+WxzByO8060RlB7+ccCgi6N1XVWK6DNndzDkkxTYLCnoxvnEA1XMzONKXMgexyFUJQy04JyrIWDQAuLmucyuzvjoZ\\\\\\\/0mlUq315\\\\\\\/gk\\\\\\\/rTQFkNLJlV2n9ytbaLswK6B0+rH+DcNKcbFRylQQr7EpBbZSfYfe09AixWSkq3kLSPHL\\\\\\\/oa9HtKHl109NMsTCPXx443BDEKF8CK47AwD6AxU3mzVwQQv4sLX8OYtpuSYWtqCqOFQ4YxKbYV59xuYEysskij8GjENVugcVjgf7ClPdCJzy\\\\\\\/TFa3zLWK594OkL\\\\\\\/RzwYh6QlvleyMSe4UBKUCy+M9607ZlBWG5UbfhbKebK\\\\\\\/KL90rgfGmuFM6wd7RmG4Ym5GSccBq9+YvSFNeIqnDL7pTn\\\\\\\/uAqN9EPRn0LCk\\\\\\\/7rjQftQB6S2719ZITZvGQY5v4Smsspwr0dNbF4LhV28AjsC1yaOODTlhZeDuDtIiABnlXnHgNKklp9xHtCYgquhlbtYx2DzzfliOjQMZE\\\\\\\/A\\\\\\\/sA1PGTWEmOSuShTwglWBDIKUx+BWYTAQqorVhOpqpuX130V5fTcZTeqQ1G7p6loNWWkuujdXZTnpREgY03DYJQiq6hceZASKJz8F3EMKHL5AV5dZPCecJJRMILjRA7ORnYO9dQK5YkPQvFqDGEmDkyNPwmE0VuT+nC9x2evVVO4Lpa4QrKKasrgb3nU\\\\\\\/8C5mjhnLsFcOx2jyx9BKQpPOsezFwpz2HPFIa1whvU8vMS3HU0O+1ndBQC8koW9W6jl5WlV24Ge1J8HK5kPLkq9DHpF8BfG4UjI7rH6j54XrXrw6Ti4MWpP8Ln74WmQIp44MIG+cKYzXKx56yIU6rQkreXUz5UYppYykqJWLxBpkJBca1KIwdZYc5J01datkkRwmYCSRWqnI9cF7CQTKX13OjHvo94zzKhzJPizcoFdzeLuKAJHaH14J11DuAQeRV1JTTzy6RsAV62cbHdBOeLLPqP5oKouLuEAQGtSOn9nMr6KAGtULk1r\\\\\\\/EAkOu9NVX4tc\\\\\\\/X\\\\\\\/dXothE0RHZjyS3nQ\\\\\\\/L9r0IXQ92GS6X6hhnkQqOQoxP+bjaQv8zp\\\\\\\/PkJJEePGwsVNfpfoI+WjYwOgrX\\\\\\\/+AANoPOuogrsg3N8YAWg05SVvDVosv3tO2P2Y+C2g\\\\\\\/8VcmI1LdBzBL1SUOCaE8ySNrtSGEyycdi8KvHn54KMi\\\\\\\/HGwbGTC1GXtUlwiyFJhof9LHUHOo4hFO2VLRBja9s1+YLjVL3QOtdRq8LDrpvgtXCVysoJXdP\\\\\\\/62jJ7lty\\\\\\\/96iRMqvD0VANDjwi+B4TUbEtxlfLkjCpwlHC3BLDEUo1jrfSAipawrNDDMdc8K1hHn8TbWOhMRcpsFnTP5LD06dgeXj7SyduGL6llss+cTWI96IS3M5yl3QZUrY8gl5fTP4zYN7K8P9m5UVCth2L\\\\\\\/amVTp2IQyw64Dho0m\\\\\\\/+dNdsnI5GBQLiAhm6MEAHPjjE4jKkmgsCRYN9lmml36fHjSORVpA90JFNDmu7Vv2Dob9rEa2X2MEYFwAWLXxAz\\\\\\\/YeBbtnALHnV57sOxcrIHnAvSp4mMCOQZbMwe+pJb4Mv6Hhk85W6Cb3eL6bV6kVH+w5xA6\\\\\\\/uAagj7bjBjHLam2BobLvr2tUSjhJ0c03ZN8ZPVLlzy\\\\\\\/V9JI95pX2LZxUZraeL80RPrcsO4j5pCfuzVAtmTbM+oFOoQt4MwAXAUYbbRbk219yMuQJuG7KwAX3aRa+gIGhZMPJZGAVE6IPdpgfQF5fL09d+YoRBeG+aeS4EaGGUErEOlTqyWYRrJWm40N3afw4k\\\\\\\/7ivc52U7+lmku72sls1boEG+fU0LhrWx2A1QUn0K5QXOB05vXx0F9bxOkjZ77NeV1Pj\\\\\\\/8LGKLccfrtrQtbUJNxmdwifvsjMILTN7DrVbhnD7+8DnByYTfaSoIITFCuhal7+UtgTQVkQ2AGOPCM1mAMtAsA8unydU2\\\\\\\/NUfsF9DGL8r7INLYAaD5slrViN0ogUpl\\\\\\\/C1DBurBFgWrly7Rm0HhRJjiFNMQ\\\\\\\/AtqV8KZxlc6ud4u+sP8rchNxtTJEFNq2zo7U4l1gQ0vxq7FQ3Em8OZXm+qmIvI\\\\\\\/Eie71jC5Tgz3Nz6ICfLiPZ\\\\\\\/W3\\\\\\\/AckAKqbWiDJgDo4rm6nxMC5VBQyuokXJvrjZk+yTuf+FmFTS8KsvIGHTEKNWPUVG4u8dco6mgm3FWIBBVgiwUbzVxrKCstejfZsTHlzJcSk4Z+gr5O85xkwqKnTBZP6gCQDxsSug6gXDZawLuFpwOho6Z+Ysvr9HU7ARKzUWe3vUEtdhxgNP28O9BS1DHrdmGhn2vA83qhTZLmu4qqzVszsgrp03XPWHmCmPKhOD2VxBaI7k\\\\\\\/BOebKOL4V+RU3ZIOqig2yo8z20AxCzq5Rsk\\\\\\\/D4uGp03Qm2VhWMRQTZzI3Ln8VSNgGjnsIXi1pFfXioC0Yos5\\\\\\\/5V\\\\\\\/M6+YR3muturvp0D5EzhiGFbEMF2YJua0XXKeQo4HruCB7emqA+NpsPdPacl\\\\\\\/1jPwL2meJU\\\\\\\/m9odysyTODgO0P5pJ7kNnE4BnxPURvz1eNLIYVb4\\\\\\\/SWLt+1ztQ4NpHugGliAIWC1i2aiH705zUNfgUa+\\\\\\\/1DZpqVjqQcDqLUG9PoxobQXW6Ohm2YfOnRJCraxU8cnTK6liXwmUBcwR+0YqXIFubvADY9tGT0IwdcitkMzlax+F6V95Zp4LxoV0ofmXOlAeveMyewDg6R+aZDNgSkr032eId4kL\\\\\\\/XcDrjPgKnSI1TbEBlwkZ+eYayjdD+CZKV2RfZmHTcA7TsHoy+7JiQdVpPcJmR4aGTsnem9aYWcReMAlBwqzx19iQiJJZZ4++hejTgeWt0aJY2W84AaEtCIZ5kjPzJioPegVs3lap6ksH2QtUCcZCCObXk7hCAr4TAykWeU8ISumJ2zc0Rc44azxi+oPV4gzSFE3XTfQgptsHup+Dmg75qXQcmOGR+4CR97hkZ9GJV40HEQX9WauJ6ElDQHTp6auOjD4afQXv9yP3WDAYNLsVmmXw76DaCD\\\\\\\/L9X18Qg\\\\\\\/m9GS1lIMdBPB+mNBntTxcQvSOu1FxBL8+1QuP6qNeA0tynoNzaneiVtT\\\\\\\/FF1LmPZKeTGG\\\\\\\/fJUvllfpdlz6Rg\\\\\\\/Mf39wWmAV4Md+RCKtekjfl2lLhfMIOpc1SkmZzZFjT38IaJReQsn7YpKOeluu34XDqh3NJH3+gNtuVMclOa855Mu5S3e\\\\\\\/bG4VqBc5m3RzNsTLBd2rivTrHn89coXyI730m5kb2pMgh+tmouabXe0izVQSc6uYtxfqH2Gw0wbaiBZo9cFvhG89FZCAjUzTdSloyc+f0aIAF39RhN115PVUEpl8AXl\\\\\\\/Dgzs\\\\\\\/f5GltZHqgi7xfH\\\\\\\/PJ37YWLhEp1xHdax5XgNJM6XlyPuHvLVKwq8+cFedSgyDfD4lX7++GvNnuJtzUjGk1+xMe7WXvq7HXtFvThJo4g36QDLJ0Y\\\\\\\/WqsfIUsUavxen7tlekVZD9Z44CiPJ3pCOvRUlQ9hFsqrTGwc7n2jBbH4+67KgGlZIx13+Ln0u41IDnahai3gvSw5VFnSaMK1XS3vHPqveBX4aIPphYzfuSZuiQ1RCiiuwkABMuBZWAIqQ0P+UaU7Em6V3+1ZZQiL\\\\\\\/Qls0SXb5ldZlf\\\\\\\/34lu4cyguekAOSNh\\\\\\\/m8Sv8+NimzjnDTroAmJDis4IPs3XPd8Huq4RlktngknUETQ5eh764muUN7HFXJr5k+kfav4NZ0S9hG7cyThhJJnOA+pAgH7S6zmVhS99mOhkfOGmSUVrPwA4L55soPdSTBnnECV\\\\\\\/FAHB1\\\\\\\/ZWVFuqj7ALZiyHSAcyPF84kIfxQ1yZOMfM3ZA7Lnhfgn0npreK2A3YGq2Z8wk66\\\\\\\/DftfcHjTyOsHD6nYq15yj0i\\\\\\\/iF7MUMvadxRmyxaEe\\\\\\\/bh8oD1AGzue9Mb92QREL5h7wdzdhpTRxIfifYqJxuVnQZK0GhMD5hhiSNTHJ\\\\\\\/4vI9VL0HJNMS1GFru1mCjxV2ovfzrntlLbgyshh38kBeZeLN0wfa\\\\\\\/3vMzksCXzpc0SKE\\\\\\\/CX14F5Tvnw7Xvv4xMjYHBGIRZjQHi7tVS\\\\\\\/EVrC2OxwUObAMJNESPkorsrG\\\\\\\/ZHPpNzGrB\\\\\\\/doBxvd8iQzykurSp2i0qQ4nujMZLx3inIBLSsGi9o1g48Nz93B\\\\\\\/dZgEfPw8Kg4Vmyaf\\\\\\\/yQcAJXFOe0YR6xznrDlleEL6DWxd+DpjAJ29Jn4R0A0L61NAvHOc+PGJzK696NzsbOAya5r9uYtyGoTE0Bi7siGTdD3BYxav\\\\\\\/C4wR+ShvJ5YxVg2f7vTaIwQPLgx6ue2U\\\\\\\/eQ+y0kpow3d7h3vdthu4ciExSlJ51vIf9CIx0IkEi9tvVvLU8VOT17qfrk75FI2K7OCAN7Hq96BbSzJ2+Xsh152TfpUJVc4vVe465Bilxtwg4b\\\\\\\/7C44D8yG7p6iveLNm6FjYJcY7L1\\\\\\\/KF1eYDp+2TEQ83+Q\\\\\\\/OZ4Q6KmQMX+GYpljWJ05V6dmNj\\\\\\\/X7m7fXbC+1Ac9Ua0th1gdSu08F0Fjos29cm\\\\\\\/zpTCq1aMuga8TGrJcI\\\\\\\/y3HUfvw\\\\\\\/eKNNtIrrMqIX5I3rydmDfUaRefiImIhk5UFhxth99cUrmVfyXSNJlejC3ja7p0b3WLB2MUhkTMKPIKS0oLkS5Nu0zWxxq1e4PUuuHMWVKCi49TaWdiKrr7ZZJNjQC3UdcdU+Wd945mMnSs+ccKyZBUo7N37G95kdItEQdjmhjutJugxlvh2r268duchPdWqWqQzgX4++r8EzGY5dGnV\\\\\\\/7x1FcoET08VVsqawogOxTN77DD\\\\\\\/P07K2DHSzuSVLtqoniEmv0AhUx1BU58eUc6A4P276EAI9AQ7IoHMZbG8MxG0UYsHCZE1F2DUgI1ivau4nxy3GcLUa8AaLR3epPA0Y19o4Bh4oF+TGUE66jXtaVBv2setedsRURAMrfmBicT128IW4oFbiWIRpUB5FjuzNhtnFwdP\\\\\\\/LF\\\\\\\/9bg8BdFvFQR08Q6fTJ1dtkBdNjgyRh6q6szs4ddc0r1tqhDLqitLNFIHpMW0K+o3TX9cMQfYePmXPsHt2526+ViTQ4yKLRGBNlaPJNGQMta0Ic2qnmHZ+t4dEriOk6x0JA17zf9l5yQ6bgM46k\\\\\\\/Ym\\\\\\\/hyck9NDHTysD3N+mX5wz0QE8pzQroROiK0ABB\\\\\\\/9dWDWl0BPIAQY2Hdkie6i7qwNaS4YQPAnl9qV77kGhmZ47vKrEtOZiO\\\\\\\/MaDnT\\\\\\\/RdXYU1VtBHXg7APXXHF20aZHU3hbXGergwyi9bWTgzl3FSFnIPTOhUXWsS\\\\\\\/sSxb6jUO5Jokf87PRVEPr2rrbitML7i6xkHi4RodpE9xn6cYhnN9nxe65vocH5pa7f3uydKoqe9ATB\\\\\\\/zPpHMBxh2+YQxUbsnl2FkZyFFGS9qQBs0\\\\\\\/zc1fP70c7m1EQn+OgSWQMJVMKuokxe7j1TjOFw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228c7CpYegEFKbH0yv31QzLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bf5b201707911905d91addf44bb139a89c890012a890e2f5d54e21e42df50f63\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223HcYeytsXzkL7MW6OSikV97UOonYqxyQfgzzc\\\\\\\/gw17bE+T4ETUrPNX6XRNlNuU1wbz8gK+EdCKnfOoYFIhdtiORcHZBQ22wZw7H\\\\\\\/vMPsZYbtKxRrRXdZmea8NOL8ls2SEH2FaXx667MT2flf5vPsRSLfcc7p1iOSuL9qUdyFmEoC5L3w3fBUp8perK8KbcveBRzOYw5PyC8zvCRB5cbeBCT2DBREe7L6PuvbPnv3Q\\\\\\\/8FcpuTEFwqbD970NEDw\\\\\\\/e6uO0xcLiLRvFM46nb5DoVc827DAZXI+ocFJKnbHBFKurrdW1eOYmBGWBoYl\\\\\\\/In+656e5F5nD8HGsyqRinNfnYVXsrdW+I286UOTdlahS1o2vxbpcYi7PKxlSw3vRpgZPYItsWdbRTisZOvga9wbXmz+vU7gCc27bqoMlpifw3Ti7ADrTjdpUnvBvGqGEVp\\\\\\\/MwSwEUeVHnJiDfa6xA6EHol9cR\\\\\\\/GGfzMGvHxkiFEbvnB0G7V6k8hw5CZmB3J94nTX\\\\\\\/hOHiQ2i5aWcjeJAbOGd+PPW5eICSh3Wl4de71fAFPDvCwGWOuNkzvXIqz6aX7goYUQ9ot+E3G\\\\\\\/xKtOhhNoxLlhTWk1XO6Te0JQZa3FGdadfrGS6LRtDinQd84Cajj3Iguea6JL+U7IDw+6MqLDJ0k3oKDnacoH\\\\\\\/B1jwXqmRdedHBdgYC99NG03yvCvKwEsbIKuxQTRci\\\\\\\/Xim62NHAtW64HLuGFCRgKERxOi8G9Cw7eFRsjLX\\\\\\\/CbrS04ElPupIzVWWIBEOgcMEESNCLhfvcEjtEcAGHI4iNakE63rpdk5VK9ksNJy+rMJQqH5RxhSQqN12X+xwZIu7RhwB+UASrYeFOtVOjQ6PejXrrSS1DOq3mk4D4ugHXy7Oks8kv0MFn1osoVbl7cjV\\\\\\\/Mj4mBzvN7KPEE07Odz7jY6HCernpWjOnwmvJvuRd8xbh9CGZ1\\\\\\\/62+UkXp+tYXqyHegg0vVktq0RBP3B4\\\\\\\/NTGWNRcg26xX\\\\\\\/QTkeL+3BWiH7t21JvxdQI5KXrx3MXVaX5TZM0BNs+xCN6ezB59qwynpKvGVKO8CPgDR3FxU0UB0ERizk3+QoeoWkB85wt6emNmgT5rAC4\\\\\\\/TQv2Xu8u+IRMwucLY36t++Xm7C1i\\\\\\\/VPpU6N18eMmecAGQs6AGM0BppWWbDZnZY9FveueR7DE6Y+UifxFYYw3vU\\\\\\\/u0xo2ZugcurMktGoIIYgB5xNlWAIdsZp62V6z\\\\\\\/njuzQU6nDTfTBorOAAAIXyML6icZyyyT1BszhEUVkEGDkxbU3BCIVwCUBlQW0AFqaaFbiT++TFyhhbxlTYnSYs8aGlBbt+ZK37uPNwtX2JXhiQCg4IYQH2v89ADxK0IebnFFNZVlzONReqIcA4F79sQJrGu8fXqcyc9B8YTtxfLdGyyWThMWFyK3l1DNN3Dax7dflz+p+WmnqXg+qTOUjzXtxudAkMlmhjtleMC495jlDQOG3sh15a8gDCumjmAm3D5A8Lx3b4tjvytAYLgoa22w+wb13wHpBGuX\\\\\\\/xTCPz\\\\\\\/XyMZi9ML67z+ARLuXp2JAP3hWBV5DGqeGKTc+UokrXvtUTEcUMYbEuXYbWb1WcQiXZopTHQZOUyhthegl1RXW7+azG8z0DI2qJgIVDVBNZVsvCW+AsZuha0yVxZvUJC5k9IUx9DadjmzZyroV1vrQ+\\\\\\\/DGzluNoQP9tdm4zIaGvWUN5LQhCT\\\\\\\/1OjGjkdUWhmV\\\\\\\/xEPRTn6H8sg0HPXIViN8iJ4qI\\\\\\\/f6Ot2\\\\\\\/3XuyHMs8ApaO3HPLH9JQItKeg\\\\\\\/\\\\\\\/fl+Db3De7Keu\\\\\\\/ljkWefDMjrzzobFtUqDqi0LhXQldc+mq5g\\\\\\\/6UmdOuLvkot0pVTvi8H3S5ycaFULDCV2x6mOoWmchatemiR8brL+ZtBrTK\\\\\\\/BU7ebE\\\\\\\/6TyFOfGlCdgH86J2t\\\\\\\/WueOG51WZOtSog1DHUe1vEm4AqKaCVhFiV06duy\\\\\\\/fjXSGeHZx3xi7R\\\\\\\/8kHcHyKs9TaG7OYCxLzq5NCW0K8UtpJS6rJtg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YkV7yy9C\\\\\\\/v6Ml5UXnbcjMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253c142f21b89c4a0f6183522beab55efd3810690726fdafe6c12aca36d3d95fd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491519 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3783" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AelO47gdBMs4tc-ftJpvggbG1ZS5uAp1_UjyjtTFiyA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3783" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zi1nmG7IvO38iun34aJtpl6Qd3kOfohPDAtP1n4BE2w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3767" title="QTL Mapping on a Background of Variance Heterogeneity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3791" title="PIE-1 Translation in the Germline Lineage Contributes to PIE-1 Asymmetry in the Early Caenorhabditis elegans Embryo" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491519" id="node491519" data-pisa="ggg;8/12/3783" data-pisa-master="ggg;g3.118.200194" data-apath="/ggg/8/12/3783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vivek Kumar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph S. Takahashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3783-3790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491519" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vivek Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Jackson Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivek%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVivek%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vivek Kumar</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph S. Takahashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute, Department of Neuroscience, University of Texas Southwestern Medical Center, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BS.%2BTakahashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahashi%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BS.%2BTakahashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph S. Takahashi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3783" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491519" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3783.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491519" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3783.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491519" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3783.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491519" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3783.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491519" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491519/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We illustrate, through two case studies, that “mean-variance QTL mapping”—QTL mapping that models effects on the mean and the variance simultaneously—can discover QTL that traditional interval mapping cannot. Mean-variance QTL mapping is based on the double generalized linear model, which extends the standard linear model used in interval mapping by incorporating not only a set of genetic and covariate effects for mean but also set of such effects for the residual variance. Its potential for use in QTL mapping has been described previously, but it remains underutilized, with certain key advantages undemonstrated until now. In the first case study, a reduced complexity intercross of C57BL/6J and C57BL/6N mice examining circadian behavior, our reanalysis detected a mean-controlling QTL for circadian wheel running activity that interval mapping did not; mean-variance QTL mapping was more powerful than interval mapping at the QTL because it accounted for the fact that mice homozygous for the C57BL/6N allele had less residual variance than other mice. In the second case study, an intercross between C57BL/6J and C58/J mice examining anxiety-like behaviors, our reanalysis detected a variance-controlling QTL for rearing behavior; interval mapping did not identify this QTL because it does not target variance QTL. We believe that the results of these reanalyses, which in other respects largely replicated the original findings, support the use of mean-variance QTL mapping as standard practice.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/variance-heterogeneirty" class="hw-term hw-article-keyword hw-article-keyword-variance-heterogeneirty" rel="nofollow">variance heterogeneirty</a></li><li class="kwd"><a href="/keyword/dglm" class="hw-term hw-article-keyword hw-article-keyword-dglm" rel="nofollow">DGLM</a></li><li class="kwd"><a href="/keyword/mqtl" class="hw-term hw-article-keyword hw-article-keyword-mqtl" rel="nofollow">mQTL</a></li><li class="kwd"><a href="/keyword/vqtl" class="hw-term hw-article-keyword hw-article-keyword-vqtl" rel="nofollow">vQTL</a></li><li class="kwd"><a href="/keyword/mvqtl" class="hw-term hw-article-keyword hw-article-keyword-mvqtl" rel="nofollow">mvQTL</a></li></ul><p id="p-3">Over the last 25 years, quantitative trait locus (QTL) mapping based on the assumption of homogeneous residual variance and its associated statistical methods (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lander and Botstein 1989</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Martínez and Curnow 1992</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haley and Knott 1992</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Churchill and Doerge 1994</a>) have successfully identified many QTL that influence the mean value of important complex traits. It is well-appreciated, however, that this approach fails to capture all the complexities of the relationship between genotype and phenotype, and recent interest has expanded the scope of genetic analysis to also look for QTL that cause the residual variance to be higher (or lower) in some individuals than others (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Deng and Pare 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rönnegård and Valdar 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Shen <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Geiler-Samerotte <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nelson <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ayroles <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Forsberg <em>et al.</em> 2015</a>). It has also been noted that modeling QTL effects on the mean, to be effective, can sometimes require accounting for sources of variance heterogeneity—for example, genotype uncertainty (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Xu 1998</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Feenstra <em>et al.</em> 2006</a>), X-chromosome inactivation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ma <em>et al.</em> 2015</a>), and background genetic or environmental factors more generally (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Corty and Valdar 2018a</a>)—suggesting that joint models of mean and variance could make QTL mapping more powerful, both for detecting mean-controlling QTL (mQTL) and variance-controlling QTL (vQTL).</p><p id="p-4">Several such methods incorporating joint models of mean and variance have been developed in this context. Two closely related efforts have been the application of the double generalized linear model (DGLM; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Smyth 1989</a>) to QTL mapping (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rönnegård and Valdar 2011</a>) and the omnibus test of <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cao <em>et al.</em> (2014)</a> for genetic association. [See also refs in <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Rönnegård and Valdar (2012)</a>, and recent developments of <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Soave and Sun (2017)</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dumitrascu <em>et al.</em> (2018)</a>.] Both of these approaches can detect mQTL, vQTL, and QTL influencing some combination of phenotype mean and variance (mvQTL) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Corty and Valdar 2018a</a>). Yet despite the potential of these methods to detect QTL traditional methods overlook, they remain underutilized.</p><p id="p-5">Barriers to widespread adoption include a lack of proven potential in real data applications, as well as the absence of software that is interoperable with existing infrastructure. Apart from those barriers, one reasonable concern is that a novel approach might fail to identify known QTL, adding needless complexity to the interpretation of already-reported QTL. This concern should be largely allayed by the nature of the DGLM as an extension of the linear model, simplifying to the latter when variance heterogeneity is absent [with similar arguments holding for the omnibus test of <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cao <em>et al.</em> (2014)]</a>. In fact, rather than add complexity, the DGLM automatically classifies QTL into mQTL, vQTL, or mvQTL, clarifying the genotype-phenotype relationship.</p><p id="p-6">Here we demonstrate, with two real data examples available from the Mouse Phenome Database (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bogue <em>et al.</em> 2017</a>), that QTL mapping using the DGLM, which we term “mean-variance QTL mapping”, largely replicates the results of standard QTL mapping and detects additional QTL that the traditional analysis does not. In two companion articles, we demonstrate typical usage of R package vqtl, which implements mean-variance QTL mapping (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Corty and Valdar 2018b</a>), and describe the how mean-variance QTL mapping and its associated permutation procedures reliably detects QTL in the face of variance heterogeneity arising from background factors (<em>i.e.</em>, genetic or non-genetic factors outside the targeted QTL) (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Corty and Valdar 2018a</a>).</p><div class="section methods" id="sec-1"><h2 class="">Statistical Methods</h2><div id="sec-2" class="subsection"><h3>Traditional QTL mapping based on the standard linear model (SLM)</h3><p id="p-7">The traditional approach to mapping a quantitative trait in an experimental cross with no population structure (<em>e.g.</em>, an F2 intercross or backcross) involves fitting, at each locus in turn, a linear model of the following form. Letting <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-1.gif"/></noscript></span></span> denote the phenotype value of individual <em>i</em>, this phenotype is modeled as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-2.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-3.gif"/></noscript></span></span> is the residual variance, and the expected phenotype mean, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-4.gif"/></noscript></span></span> is predicted by effects of QTL genotype and, optionally, effects of covariates. In the reanalyses performed here, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-5.gif"/></noscript></span></span> is modeled to include a covariate of sex and additive and dominance effects of QTL genotype, that is,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-6.gif"/></noscript></span></span>where <em>μ</em> is the intercept, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-7.gif"/></noscript></span></span> is the sex effect, with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-8.gif"/></noscript></span></span> indicating (0 or 1) the sex of individual <em>i</em>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-10.gif"/></noscript></span></span> are the additive and dominance effects of a QTL whose genotype is represented by <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-12.gif"/></noscript></span></span> defined as follows: when QTL genotype is known, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-13.gif"/></noscript></span></span> is the count (0,1,2) of one parental allele, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-14.gif"/></noscript></span></span> indicates heterozygosity (0 or 1); when QTL genotype is inferred based on flanking marker data, as is done here, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-16.gif"/></noscript></span></span> are replaced by their corresponding probabilistic expectations (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Haley and Knott 1992</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Martínez and Curnow 1992</a>). The evidence for association at a given putative QTL is based on a comparison of the fit of the model above with that of a null model that is identical except for the QTL effects being omitted. These models and their comparison we henceforth refer to as the standard linear model (SLM) approach.</p></div><div id="sec-3" class="subsection"><h3>Mean-variance QTL mapping based on the double generalized linear model (DGLM)</h3><p id="p-8">The statistical model underlying mean-variance QTL mapping, the double generalized linear model (DGLM; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Smyth 1989</a> and <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Rönnegård and Valdar 2011</a>), elaborates the SLM approach by modeling a potentially unique value of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-17.gif"/></noscript></span></span> for each individual, as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-18.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-19.gif"/></noscript></span></span> has the same meaning as in the SLM, but now <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-20.gif"/></noscript></span></span> is linked to its own linear predictor <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-21.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-22.gif"/></noscript></span></span>where the exponentiation ensures that <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-23.gif"/></noscript></span></span> is always positive, though <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-24.gif"/></noscript></span></span> is unconstrained. The linear predictors for <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-26.gif"/></noscript></span></span> are modeled as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-28.gif"/></noscript></span></span>where <em>μ</em>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-29.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-31.gif"/></noscript></span></span> and the <em>β</em>’s are as before, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-32.gif"/></noscript></span></span> is an intercept representing the (log of the) “baseline” residual variance, and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-33.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-35.gif"/></noscript></span></span> are the effects of the QTL and covariates on <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-36.gif"/></noscript></span></span></p><p id="p-9">The evidence for a QTL association is now defined through three distinct model comparisons, corresponding to testing for an mQTL, a vQTL, or an mvQTL. In each case, the fit of the “full” model in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 1</a> is compared with that of a different fitted null: for the mQTL test, the null model omits the QTL effects on the mean (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-37.gif"/></noscript></span></span>); for the vQTL test, the null model omits the QTL effects on the variance (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-38.gif"/></noscript></span></span>); and for the mvQTL test, the null model omits QTL effects on both mean and variance (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-39.gif"/></noscript></span></span>). These tests are detailed in <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Corty and Valdar (2018a)</a>.</p></div><div id="sec-4" class="subsection"><h3>Genomewide significance and FWER-adjusted p-values</h3><p id="p-10">The model comparisons described above constitute the SLM test and the three DGLM-based tests and each produces a likelihood ratio (LR) statistic. These LR statistics are converted to <em>p</em>-values that are adjusted for the family-wise error rate (FWER) across loci, <em>i.e.</em>, <em>p</em>-values on the scale of genomewide significance. This adjustment is performed separately for each test by calculating an empirical distribution for the LR statistic under permutation, much in the spirit of <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Churchill and Doerge (1994)</a> but with some modifications, namely that different tests have differently structured permutations. Briefly, let <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-40.gif"/></noscript></span></span> be the full set of genetic information for individual <em>i</em>, that is, the genotypes or genotype probabilities across all loci. For the SLM and mvQTL tests, we define a permutation as randomly shuffling the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-41.gif"/></noscript></span></span>’s across individuals; for the mQTL test, the permutations apply this shuffle only to the genotype information in the full model’s mean component; for the vQTL test, the permutations apply the shuffle only to the genotype information in the full model’s variance component. For a given test, for each permutation we calculate LR statistics across the genome and record the maximum; the maxima of over all permutations is fitted to a generalized extreme value distribution, and the upper tail probabilities of this fitted distribution are used to calculated the FWER-adjusted <em>p</em>-values for the LR statistics in the unpermuted data [see <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dudbridge and Koeleman (2004)</a>, and, <em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Valdar <em>et al.</em> 2006</a>; more details in <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Corty and Valdar (2018a)</a>]. An FWER-adjusted <em>p</em>-value can be interpreted straightforwardly: it is the probability of observing an association statistic this large or larger in a genome scan of a phenotype with no true associations.</p></div><div id="sec-5" class="subsection"><h3>Data Availability</h3><p id="p-11">All data and scripts used to conduct the analyses presented here and plot results are archived in a public, static repository at with DOI: 10.5281/zenodo.1453905. Specifically, the raw data files are:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-12"><span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-42.gif"/></noscript></span></span> The phenotype and genotype data from <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a> that was reanalyzed. This dataset is also available from the Mouse Phenome Database (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bogue <em>et al.</em> 2017</a>) at <a href="https://phenome.jax.org/projects/Kumar1">https://phenome.jax.org/projects/Kumar1</a>.</p></li><li id="list-item-2"><p id="p-13"><span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-43.gif"/></noscript></span></span> The phenotype and genotype data from <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> (2008)</a> that was reanalyzed. This dataset is also available from the Mouse Phenome Database at <a href="https://phenome.jax.org/projects/Bailey1">https://phenome.jax.org/projects/Bailey1</a>.</p></li><li id="list-item-3"><p id="p-14"><span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-44.gif"/></noscript></span></span> The raw data on circadian activity from <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a> that was used to plot actograms</p></li></ul><p id="p-15">The analysis and plotting scripts are:</p><ul class="list-unord " id="list-2"><li id="list-item-4"><p id="p-16">2_run_Kumar_scans.R This script runs genome scans with R/qtl and R/vqtl on the data from <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a>.</p></li><li id="list-item-5"><p id="p-17">3_plot_Kumar_scans.R This script plots the results of the reanalysis of <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a>.</p></li><li id="list-item-6"><p id="p-18">5_run_Bailey_scans.R This script runs genome scans with R/qtl and R/vqtl on the data from <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> (2008)</a>.</p></li><li id="list-item-7"><p id="p-19">6_plot_Bailey_scans.R This script plots the results of the reanalysis of <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> (2008)</a>.</p></li><li id="list-item-8"><p id="p-20">7_prune_big_files.R This script strips out redundant information from the results to make the file size smaller to share more easily online.</p></li><li id="list-item-9"><p id="p-21">8_power_simulations.R This script runs the power simulation comparing the DGLM to the SLM at the QTL identified in the Kumar reanalysis.</p></li></ul><p id="p-22">The results of running the analysis and plotting scripts are:</p><ul class="list-unord " id="list-3"><li id="list-item-10"><p id="p-23">Kumar_scans_1000_perms.RDS This file contains the results of the reanalysis of <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a>.</p></li><li id="list-item-11"><p id="p-24">Bailey_scans_1000_perms.RDS This file contains the results of the reanalysis of <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> (2008)</a>.</p></li><li id="list-item-12"><p id="p-25">Kumar_plots This directory contains the figures generated by 3_plot_Kumar_scans.R (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a>, and S1).</p></li><li id="list-item-13"><p id="p-26">Bailey_plots This directory contains the figures generated by 6_plot_Bailey_scans.R (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>, S5, S6, S7, and S8).</p></li></ul><p id="p-27">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7284575">https://doi.org/10.25387/g3.7284575</a>.</p></div></div><div class="section other1" id="sec-6"><h2 class="">Reanalysis of kumar et al. reveals a new mqtl for circadian wheel running activity</h2><div id="sec-7" class="subsection"><h3>Summary of Original Study</h3><p id="p-28"><a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a> intercrossed C57BL/6J and C57BL/6N, two closely-related strains of C57BL6 that diverged in 1951, approximately 330 generations ago. Due to recent coancestry of the parental strains, this cross is termed a “reduced complexity cross”, and their limited genetic differences ensure that any identified QTL region can be narrowed to a small set of variants bioinformatically. The intercross resulted in 244 F2 offspring, 113 female and 131 male, which were tested for acute locomotor response to cocaine (20mg/kg) in the open field. One to three weeks following psychostimulant response testing, the mice were tested for circadian wheel running activity.</p><p id="p-29">Analysis of wheel running data were carried out using ClockLab software v6.0.36. For calculation of activity, 20 day-epoch in DD was used in order to have standard display between actograms. Analysis of other circadian measures such as period (tau) or amplitude were carried out using methods previously described (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shimomura <em>et al.</em> 2001</a>). All animal protocols were approved by the Institutional Animal Care and Use Committee (IACUC) of the University of Texas Southwestern Medical Center</p><p id="p-30">Traditional QTL mapping with the SLM, reported in <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a>, detected a single large-effect QTL for cocaine-response traits on chromosome 11, but no QTL for circadian activity. A later study by another group nonetheless observed that the circadian activity of the two strains showed significant differences (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banks <em>et al.</em> 2015</a>).</p></div><div id="sec-8" class="subsection"><h3>Reanalysis with traditional QTL mapping and mean-variance QTL mapping</h3><p id="p-31">For the cocaine response traits, traditional QTL mapping and mean-variance QTL mapping gave results that were nearly identical to the originally-published analysis in <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2013)</a> (Figure S1).</p><p id="p-32">For the circadian wheel running activity trait, however, traditional QTL mapping identified no QTL (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> in green) but mean-variance QTL mapping identified one QTL on chromosome 6 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> in blue, black, and red). In this case, all three tests were statistically significant, but the most significant was the mQTL test (blue), so we discuss it as an mQTL. The most significant genetic marker was rs30314218 on chromosome 6, at 18.83 cM, 40.0 Mb, with a FWER-controlling <em>p</em>-value of 0.0063. The mQTL explains 8.4% of total phenotype variance by the traditional definition of percent variance explained (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome scan for Kumar et al. circadian wheel running activity. The horizontal axis shows chromosomal location and the vertical axis shows FWER-controlling p-values for the association between each genomic locus and circadian wheel running activity. Line colors indicate the test type, with traditional (green) representing the traditional SLM-based QTL association, and the mQTL (blue), vQTL (red) and mvQTL (black) tests calculated using (the DGLM-based) mean-variance QTL mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969551112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome scan for Kumar &lt;em&gt;et al.&lt;/em&gt; circadian wheel running activity. The horizontal axis shows chromosomal location and the vertical axis shows FWER-controlling &lt;em&gt;p&lt;/em&gt;-values for the association between each genomic locus and circadian wheel running activity. Line colors indicate the test type, with traditional (green) representing the traditional SLM-based QTL association, and the mQTL (blue), vQTL (red) and mvQTL (black) tests calculated using (the DGLM-based) mean-variance QTL mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3783/F1.medium.gif" width="440" height="83"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3783/F1.medium.gif" width="440" height="83"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491681" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Genome scan for Kumar <em>et al.</em> circadian wheel running activity. The horizontal axis shows chromosomal location and the vertical axis shows FWER-controlling <em>p</em>-values for the association between each genomic locus and circadian wheel running activity. Line colors indicate the test type, with traditional (green) representing the traditional SLM-based QTL association, and the mQTL (blue), vQTL (red) and mvQTL (black) tests calculated using (the DGLM-based) mean-variance QTL mapping.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Understanding the Novel QTL</h3><p id="p-34">Though they test for the same pattern, the mQTL test of mean-variance QTL mapping identified a QTL where the traditional QTL test did not. This discordance may arise when there is variance heterogeneity in the mapping population. In this case, mice homozygous for the C57BL/6N allele at the mQTL have both higher average wheel running activity and lower residual variance in wheel running activity than mice with other genotypes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) Average wheel speed (revolutions/minute) of all mice. It is visually apparent that female mice had higher circadian wheel running activity than male mice and that mice that homozygous for C57BL/6N had higher circadian wheel running activity and less intra-genotype variation. Large dots indicate the mice whose activity is shown in actogram form (males in Figure 3; all in Figure S2). (b) Predicted mean and variance of mice according to sex and allele at the QTL. What was visually apparent in (a) is captured by the DGLM. The estimated parameters relating to mice that are homozygous for the C57BL/6N allele imply a higher expected value and a lower residual variance than the other two genotype groups. Black x&#x2019;s indicate the estimates from the SLM, very similar to the DGLM estimates in the horizontal (mean) axis, but homogeneous in the vertical (variance) axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969551112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) Average wheel speed (revolutions/minute) of all mice. It is visually apparent that female mice had higher circadian wheel running activity than male mice and that mice that homozygous for C57BL/6N had higher circadian wheel running activity and less intra-genotype variation. Large dots indicate the mice whose activity is shown in actogram form (males in Figure 3; all in Figure S2). (b) Predicted mean and variance of mice according to sex and allele at the QTL. What was visually apparent in (a) is captured by the DGLM. The estimated parameters relating to mice that are homozygous for the C57BL/6N allele imply a higher expected value and a lower residual variance than the other two genotype groups. Black x&#x2019;s indicate the estimates from the SLM, very similar to the DGLM estimates in the horizontal (mean) axis, but homogeneous in the vertical (variance) axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3783/F2.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3783/F2.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491552" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">(a) Average wheel speed (revolutions/minute) of all mice. It is visually apparent that female mice had higher circadian wheel running activity than male mice and that mice that homozygous for C57BL/6N had higher circadian wheel running activity and less intra-genotype variation. Large dots indicate the mice whose activity is shown in actogram form (males in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; all in Figure S2). (b) Predicted mean and variance of mice according to sex and allele at the QTL. What was visually apparent in (a) is captured by the DGLM. The estimated parameters relating to mice that are homozygous for the C57BL/6N allele imply a higher expected value and a lower residual variance than the other two genotype groups. Black x’s indicate the estimates from the SLM, very similar to the DGLM estimates in the horizontal (mean) axis, but homogeneous in the vertical (variance) axis.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Double-plotted actograms illustrate the variation in wheel running activity of male mice based on their genotype at rs30314218. On reading a single actogram: An actogram illustrates the activity of a single mouse over the course of an experiment. Each day of the experiment is represented by a histogram, with bin width of six minutes. Histograms are stacked vertically. Additionally, each day is shown twice (repeated horizontally) so that there is no time of day that is illegible due to the plot edges. Yellow box indicates when lights were on. On reading this six-actogram plot: Recall that the DGLM estimates a unique mean and standard deviation (SD) for each genotype. The mice whose actograms are shown here had an activity level that is one genotype-specific SD greater than (top) or less than (bottom) the genotype-specific mean. The difference between the two is much less in the C57BL/6N homozygotes than in the other genotypes, reflecting the decreased phenotype variance among C57BL/6N homozygotes. The animals shown in this figure are marked with large blue circles in Figure 2a. A larger figure that also includes female mice as well as the ID&#x2019;s of all plotted mice are in the supplement (Figure S2 and Table 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969551112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Double-plotted actograms illustrate the variation in wheel running activity of male mice based on their genotype at rs30314218. On reading a single actogram: An actogram illustrates the activity of a single mouse over the course of an experiment. Each day of the experiment is represented by a histogram, with bin width of six minutes. Histograms are stacked vertically. Additionally, each day is shown twice (repeated horizontally) so that there is no time of day that is illegible due to the plot edges. Yellow box indicates when lights were on. On reading this six-actogram plot: Recall that the DGLM estimates a unique mean and standard deviation (SD) for each genotype. The mice whose actograms are shown here had an activity level that is one genotype-specific SD greater than (top) or less than (bottom) the genotype-specific mean. The difference between the two is much less in the C57BL/6N homozygotes than in the other genotypes, reflecting the decreased phenotype variance among C57BL/6N homozygotes. The animals shown in this figure are marked with large blue circles in Figure 2a. A larger figure that also includes female mice as well as the ID&#x2019;s of all plotted mice are in the supplement (Figure S2 and Table 2).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3783/F3.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3783/F3.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491645" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Double-plotted actograms illustrate the variation in wheel running activity of male mice based on their genotype at rs30314218. On reading a single actogram: An actogram illustrates the activity of a single mouse over the course of an experiment. Each day of the experiment is represented by a histogram, with bin width of six minutes. Histograms are stacked vertically. Additionally, each day is shown twice (repeated horizontally) so that there is no time of day that is illegible due to the plot edges. Yellow box indicates when lights were on. On reading this six-actogram plot: Recall that the DGLM estimates a unique mean and standard deviation (SD) for each genotype. The mice whose actograms are shown here had an activity level that is one genotype-specific SD greater than (top) or less than (bottom) the genotype-specific mean. The difference between the two is much less in the C57BL/6N homozygotes than in the other genotypes, reflecting the decreased phenotype variance among C57BL/6N homozygotes. The animals shown in this figure are marked with large blue circles in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2a</a>. A larger figure that also includes female mice as well as the ID’s of all plotted mice are in the supplement (Figure S2 and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The identification of this QTL by mean-variance QTL mapping but not traditional QTL mapping can be understood by contrasting how the DGLM and SLM fit the data at this locus.</p><p id="p-38">For the SLM, a single value of the residual standard deviation <em>σ</em> is estimated for all mice. Approximately 25% of the mice are homozygous for the C57BL/6N allele, so <em>σ</em> is estimated mostly based on heterozygous mice and homozygous C57BL/6J mice. The SLM estimates <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-45.gif"/></noscript></span></span>, a slight underestimate for some genotype-sex combinations, and a drastic overestimate for the homozygous C57BL/6N of both sexes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2b</a>). With <em>σ</em> overestimated for the C57BL/6N homozygotes, the addition of a locus effect to the null model results in only a limited increase in the likelihood, one that could reasonably be caused by chance alone. For the DGLM, six different values of <em>σ</em> are estimated, one for each genotype-sex combination (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2b</a>). With a better-estimated (lower) <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-46.gif"/></noscript></span></span> for the C57BL/6N homozygotes, the addition of the locus effect to the null model results in a greater increase in the likelihood, one that is very unlikely due to chance alone.</p><p id="p-39">A simulation based on the estimated coefficients shows that at a false positive rate of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-47.gif"/></noscript></span></span>, relevant for genome-wide significance testing, the SLM has 61% power to reject the null at this locus and the DGLM has 90% power (See file 8_power_simulations.R).</p></div><div id="sec-10" class="subsection"><h3>Variant Prioritization</h3><p id="p-40">Reduced complexity crosses allow variant prioritization to proceed quickly because of the number of segregating variants is small. Using 1000 nonparametric bootstrap resamples, the QTL interval was estimated as 13.5-23.5 cM (90% CI), which translates to physical positions of 32.5 - 48.5 Mb using Mouse Map Converter’s sex averaged Cox map (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cox <em>et al.</em> 2009</a>). Since this interval contains no genes or previously identified QTL shown to regulate circadian rhythms, we prioritized candidates by identifying variants between C57BL/6J and C57BL/6NJ based on Sanger mouse genome database (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Simon <em>et al.</em> 2013</a>), which yielded 463 SNPs, 124 indels, and 3 structural variants (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491540/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491540/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genetic Variants in QTL interval for circadian wheel running activity</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491538/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491538/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The characteristics of the mice plotted in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Of these variants, none of the indels or structural variants were nonsynonymous. Two SNPs were predicted to lead to missense changes (T to A at position 6: 39400456 in <em>Mkrn1</em>, and A to A/C at 6:48486716 in <em>Sspo</em>). The variant in <em>Sspo</em> was a very low confidence call and therefore likely a false positive.</p><p id="p-42">The <em>Mkrn1</em> (makorin ring finger protein 1) variant is a mutation in C57BL/6J that changes a highly conserved (Figure S3 and Figure S4) tyrosine to asparagine with rsID rs30899669. It was determined to be the best candidate variant in the QTL interval. The Mkrn1 protein is a ubiquitin E3 ligase with zinc finger domains with poorly defined function (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2005</a>). It is expressed at low levels widely in the brain according to Allen Brain Atlas and EBI Expression Atlas (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lein <em>et al.</em> 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kapushesky <em>et al.</em> 2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McWilliam <em>et al.</em> 2013</a>). Functional analysis will be necessary to experimentally confirm that this variant in <em>Mkrn1</em> is indeed the causative mutation that led, in a dominant fashion, to the decreased expected value and increased variance of circadian wheel running activity observed in mice with at least one copy of the C57BL/6J haplotype in the QTL region in this study.</p></div></div><div class="section other2" id="sec-11"><h2 class="">Reanalysis of bailey et al. identifies a new vqtl for rearing behavior</h2><div id="sec-12" class="subsection"><h3>Summary of Original Study</h3><p id="p-43"><a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> (2008)</a> intercrossed C57BL/6J and C58/J mice, two strains known to be phenotypically similar for anxiety-related behaviors, as a control cross for an ethylnitrosourea mutagenesis mapping study. The intercross resulted in 362 F2 offspring, 196 females and 166 males. Six open-field behaviors were measured at approximately 60 days of age in a 43cm by 43cm by 33cm white arena for ten minutes. All phenotypes were transformed with the rank-based inverse normal transform to limit the influence of outliers. The authors reported 7 QTL spread over five of the six measured traits, but none for rearing behavior.</p></div><div id="sec-13" class="subsection"><h3>Reanalysis with SLM and DGLM</h3><p id="p-44">SLM-based QTL analysis replicated the originally-reported LOD curves. Significance thresholds to control FWER at 0.05 were estimated by 10,000 permutations, using the method described in the original publication, but found to be meaningfully higher than the originally-reported thresholds. Of the 7 originally-reported QTL, 3 exceeded the newly-estimated thresholds (Figure S5).</p><p id="p-45">The DGLM-based reanalysis was initially conducted with the rank-based inverse normal transformed phenotypes, to maximize the comparability with the original study. This reanalysis largely replicated the results of the SLM-based analysis and identified a statistically-significant vQTL for rearing behavior on chromosome 2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and Figure S5). The top marker under the peak was at 38.6cM and 65.5Mb.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome scan for Bailey et al. rearing behavior. The x axis shows chromosomal location and the y axis shows FWER-controlling p-values for the association between each genomic locus and the Box-Cox transformed rearing behavior." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969551112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome scan for Bailey &lt;em&gt;et al.&lt;/em&gt; rearing behavior. The x axis shows chromosomal location and the y axis shows FWER-controlling &lt;em&gt;p&lt;/em&gt;-values for the association between each genomic locus and the Box-Cox transformed rearing behavior.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3783/F4.medium.gif" width="440" height="83"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3783/F4.medium.gif" width="440" height="83"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491594" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Genome scan for Bailey <em>et al.</em> rearing behavior. The x axis shows chromosomal location and the y axis shows FWER-controlling <em>p</em>-values for the association between each genomic locus and the Box-Cox transformed rearing behavior.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">There are well-known and well-founded concerns that inappropriate scaling of phenotypes can produce spurious vQTL (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Rönnegård and Valdar 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sun <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shen and Ronnegard 2013</a>). Therefore, the rearing phenotype was analyzed under a variety of additional transforms: none, log, square root, and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-48.gif"/></noscript></span></span><sup>th</sup> power (the transformation recommended by the Box-Cox procedure). Because the trait is a “count” and a positive mean-variance correlation was observed, the trait was further analyzed with a Poisson double generalized linear model with its canonical link function (log). In all cases, the same genomic region on chromosome 2 was identified as a statistically significant vQTL (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3783/embed/mml-math-49.gif"/></noscript></span></span>) (Figure S6, Figure S7 and Figure S8). Though all transformations yielded similar results, we highlight the Box-Cox transformed analysis recommended for transformation selection in <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Rönnegård and Valdar (2011)</a>.</p></div><div id="sec-14" class="subsection"><h3>Understanding the Novel QTL</h3><p id="p-48">In this case, the DGLM-based analysis identified a vQTL, a pattern of variation across genotypes not targeted by traditional, SLM-based, QTL analysis. The phenotype values, when stratified by genotype at the top locus, illustrate clear variance heterogeneity (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5a</a>). The effects and their standard errors estimated by the DGLM fitted at the top locus corroborate the impression from simply viewing the data, that the locus is a vQTL but not an mQTL (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5b</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) &#x201C;Total Rearing Events&#x201D;, transformed by the Box-Cox procedure, stratified by sex and genotype at the top marker. (b) Predicted mean and variance of mice according to sex and allele at the top marker." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969551112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) &#x201C;Total Rearing Events&#x201D;, transformed by the Box-Cox procedure, stratified by sex and genotype at the top marker. (b) Predicted mean and variance of mice according to sex and allele at the top marker.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3783/F5.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3783/F5.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3783/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491630" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-49" class="first-child">(a) “Total Rearing Events”, transformed by the Box-Cox procedure, stratified by sex and genotype at the top marker. (b) Predicted mean and variance of mice according to sex and allele at the top marker.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-50">We have demonstrated through two case studies that mean-variance QTL mapping based on the DGLM expands the range of QTL that can be detected, including both mQTL at loci that exhibit variance heterogeneity and vQTL. In an era where ever more complete and complex data on biological systems is becoming available, this modest elaboration of an existing approach represents a step toward the broader goal of characterizing the wide array of patterns of association between genotype, environment, and phenotype.</p><p id="p-51">In the reanalysis of Kumar <em>et al.</em>, mean-variance QTL mapping identified the same QTL as traditional, SLM-based QTL mapping for cocaine response traits and one novel mQTL for a circadian behavior trait. Such an mQTL would likely have been detected by a traditional QTL analysis with a larger mapping population: Through simulation, we estimated that the additional power to detect the mQTL was equivalent to the power increase that would have come from increasing the sample size by ≈100 mice, from 244 to ≈350 (See file 8_power_simulations.R). Given the considerable effort and expense associated with conducting an experimental cross or expanding the size of the mapping population, there seems to be little to be gained by omitting a DGLM-based analysis.</p><p id="p-52">In the reanalysis of Bailey <em>et al.</em>, mean-variance QTL mapping identified a novel vQTL for an exploratory behavior. A vQTL such as this would not be detected by the traditional QTL analysis no matter how large the mapping population because the pattern is entirely undetectable by the SLM.</p><p id="p-53">The identification of a vQTL raises important issues related to phenotype transformation and the interpretation of findings, but both are manageable, as we have illustrated here. The criticism that a spurious vQTL can arise as the result of an inappropriate transformation is based on the observation that when genotype means are unequal, there always exists a (potentially exotic) transformation that diminishes the extent of variance heterogeneity (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sun <em>et al.</em> 2013</a>). Thus, any other transformation (including none at all) can be seen as inflationary toward variance heterogeneity. In this context, however, an “inappropriate transformation” leads not to the misclassification of a non-QTL as a QTL, but an mQTL as a vQTL.</p><p id="p-54">To the extent that the goal of QTL mapping is to understand a trait’s genetic architecture, this criticism is valid and should be addressed by considering a wide range of transformations, alternative models, and parameterizations. To the extent that the goal of QTL mapping is to identify genomic regions that contain genes and regulatory factors that influence a trait in any way, we argue that such a misclassification is often irrelevant. Whether we pursue bioinformatic follow-up to identify QTN in a region because it was identified as an mQTL or a vQTL need not change our downstream efforts.</p><p id="p-55">In summary, we advocate for the use of mean-variance QTL mapping not as an additional flourish to consider after conducting an SLM-based QTL mapping effort, but rather as a drop-in replacement. This approach should not be too alien — when variance heterogeneity is absent, it simplifies to the well-known SLM-based approach. Full-featured software that implements this approach is described in a companion article (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Corty and Valdar 2018b</a>).</p><p id="p-56">Last, we note an additional benefit conferred by mean-variance QTL mapping not discussed in depth here. Variance heterogeneity can also derive from factors acting in the “background”, that is, arising from experimental or biological variables that are outside the main focus of testing but that nonetheless predict phenotypic variability and thereby inform the relative precision of one individual’s phenotype over another. In the case studies presented here, the only background factor considered was sex. But, more generally, any factor that a researcher considers as a potentially important covariate that should be modeled can be included not only as a mean covariate (as with the SLM) but also as a variance covariate. As described in a companion article, <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Corty and Valdar (2018a)</a>, this practice has the potential to both deliver increased power to detect QTL (mQTL, vQTL, and mvQTL) as well as increase the reproducibility of published QTL.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">This work was funded by National Institutes of General Medical Sciences grants R01-GM104125 (RWC,WV), R35-GM127000 (RWC,WV), T32-GM067553 (RWC); National Heart, Lung and Blood Institute grant R21 HL126045 (RWC,WV); National Library of Medicine grant T32-LM012420 (RWC); and a National Institute of Mental Health grant F30-MH108265 (RWC).</p><p id="p-58"><em>Note added in proof:</em> See Corty and Valdar 2018 (pp. 3757–3766) and Corty and Valdar 2018 (pp. 3767–3782) in this issue, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7284575">https://doi.org/10.25387/g3.7284575</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: D. J. de Koning</em></p></li><li class="fn" id="fn-3"><p id="p-61">Copyright © 2018 by the Genetics Society of America</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-27"><span class="received-label">Received </span>February 27, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-11"><span class="accepted-label">Accepted </span>October 11, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Corty <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.1" data-doi="10.1073/pnas.1503830112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skutt-Kakaria</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Behavioral idiosyncrasy reveals genetic control of phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">6706</span>–<span class="cit-lpage">6711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1503830112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503830112%26rft_id%253Dinfo%253Apmid%252F25953335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY3MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.2" data-doi="10.1111/j.1601-183X.2008.00415.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabowski-Boase</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffy</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of quantitative trait loci for locomotor activation and anxiety using closely related inbred strains.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-183X.2008.00415.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DBaile%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D761%26rft.epage%253D769%26rft.atitle%253DIdentification%2Bof%2Bquantitative%2Btrait%2Bloci%2Bfor%2Blocomotor%2B%2Bactivation%2Band%2Banxiety%2Busing%2Bclosely%2Brelated%2Binbred%2B%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2008.00415.x%26rft_id%253Dinfo%253Apmid%252F19130624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2008.00415.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19130624&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.3" data-doi="10.1016/j.neurobiolaging.2014.07.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heise</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starbuck</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisby</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic background influences age-related decline in visual and nonvisual retinal responses, circadian rhythms, and sleep.</span> <abbr class="cit-jnl-abbrev">Neurobiol. Aging</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neurobiolaging.2014.07.040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiol.%2BAging%26rft.volume%253D36%26rft.spage%253D380%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neurobiolaging.2014.07.040%26rft_id%253Dinfo%253Apmid%252F25179226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neurobiolaging.2014.07.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25179226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.4" data-doi="10.1093/nar/gkx1082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philip</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolishovski</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mouse phenome database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D843</span>–<span class="cit-lpage">D850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD843%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3783.5" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BGuide%2Bto%2BQTL%2BMapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.6" data-doi="10.1002/gepi.21778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauwe</span>  <span class="cit-name-given-names">J. S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A versatile omnibus test for detecting mean and variance heterogeneity.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21778</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D38%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical Threshold Values for Quantitative Trait Mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> Mean-Variance QTL Mapping on a Background of Variance Heterogeneity. G3: Genes, Genomes, Genetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> vqtl: An R package for Mean-Variance QTL Mapping. G3: Genes, Genomes, Genetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.10" data-doi="10.1534/genetics.109.105486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A New Standard Genetic Map for the Laboratory Mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.105486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105486%26rft_id%253Dinfo%253Apmid%252F19535546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEzMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.11" data-doi="10.1002/gepi.20624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pare</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast algorithm to optimize SNP prioritization for gene-gene and gene-environment interactions.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">738</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20624</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DQ%2BDeng%26rft.auinit1%253DW.%26rft.volume%253D35%26rft.issue%253D7%26rft.spage%253D729%26rft.epage%253D738%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bto%2Boptimize%2BSNP%2Bprioritization%2Bfor%2Bgene-gene%2Band%2Bgene-environment%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20624%26rft_id%253Dinfo%253Apmid%252F21922538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20624&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21922538&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.12" data-doi="10.1086/423738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudbridge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeleman</span>  <span class="cit-name-given-names">B. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/423738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDudbridge%26rft.auinit1%253DF.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D424%26rft.epage%253D435%26rft.atitle%253DEfficient%2Bcomputation%2Bof%2Bsignificance%2Blevels%2Bfor%2Bmultiple%2Bassociations%2Bin%2Blarge%2Bstudies%2Bof%2Bcorrelated%2Bdata%252C%2Bincluding%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F423738%26rft_id%253Dinfo%253Apmid%252F15266393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/423738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223272200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.13" data-doi="10.1093/bioinformatics/bty565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumitrascu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Statistical tests for detecting variance effects in quantitative trait studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-comment">bty565</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.14" data-doi="10.1534/genetics.106.058537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feenstra</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skovgaard</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping quantitative trait loci by an extension of the Haley-Knott regression method using estimating equations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2269</span>–<span class="cit-lpage">2282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.058537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058537%26rft_id%253Dinfo%253Apmid%252F16702423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIyNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.15" data-doi="10.1371/journal.pgen.1005648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreatta</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danku</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salt</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005648%26rft_id%253Dinfo%253Apmid%252F26599497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26599497&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.16" data-doi="10.1016/j.copbio.2013.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geiler-Samerotte</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The details in the distributions: Why and how to study phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">759</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.copbio.2013.03.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BBiotechnol.%26rft.volume%253D24%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2013.03.010%26rft_id%253Dinfo%253Apmid%252F23566377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2013.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23566377&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323298200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.17" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1992.131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.18" data-doi="10.1093/nar/gkp936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapushesky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurnosov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene expression Atlas at the European Bioinformatics Institute.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D690</span>–<span class="cit-lpage">D698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp936</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp936%26rft_id%253Dinfo%253Apmid%252F19906730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19906730&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276399100109&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.19" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.20" data-doi="10.1101/gad.1289405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ubiquitin ligase MKRN1 modulates telomere length homeostasis through a proteolysis of hTERT.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">776</span>–<span class="cit-lpage">781</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1289405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKim%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D776%26rft.epage%253D781%26rft.atitle%253DUbiquitin%2Bligase%2BMKRN1%2Bmodulates%2Btelomere%2Blength%2Bhomeostasis%2Bthrough%2Ba%2Bproteolysis%2Bof%2BhTERT%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1289405%26rft_id%253Dinfo%253Apmid%252F15805468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTkvNy83NzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3783.21" data-doi="10.1126/science.1245503"><div class="cit-metadata"><cite>Kumar, V., K. Kim, C. Joseph, S. Kourrich, S.-H. Yoo, <em>et al.</em>, 2013 <span class="cit-comment">C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response. Science (80). 342: 1508–1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1245503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKumar%26rft.auinit1%253DV.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1508%26rft.epage%253D1512%26rft.atitle%253DC57BL%252F6N%2BMutation%2Bin%2BCytoplasmic%2BFMRP%2Binteracting%2Bprotein%2B2%2BRegulates%2BCocaine%2BResponse%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1245503%26rft_id%253Dinfo%253Apmid%252F24357318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNTA4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.23" data-doi="10.1038/nature05453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lein</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawrylycz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayres</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide atlas of gene expression in the adult mouse brain.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05453</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLein%26rft.auinit1%253DE.%2BS.%26rft.volume%253D445%26rft.issue%253D7124%26rft.spage%253D168%26rft.epage%253D176%26rft.atitle%253DGenome-wide%2Batlas%2Bof%2Bgene%2Bexpression%2Bin%2Bthe%2Badult%2Bmouse%2Bbrain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05453%26rft_id%253Dinfo%253Apmid%252F17151600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17151600&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243384300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.24" data-doi="10.1186/s12864-015-1463-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">X-inactivation informs variance-based testing for X-linked association of a quantitative trait.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1463-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1463-y%26rft_id%253Dinfo%253Apmid%252F25880738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1463-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25880738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.25" data-doi="10.1007/BF00222330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curnow</span>  <span class="cit-name-given-names">R. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222330%26rft_id%253Dinfo%253Apmid%252F24197463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24197463&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KE52200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.26" data-doi="10.1093/nar/gkt376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uludag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squizzato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis Tool Web Services from the EMBL-EBI.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">W597</span>–<span class="cit-lpage">W600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt376%26rft_id%253Dinfo%253Apmid%252F23671338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23671338&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323603200095&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.27" data-doi="10.1016/j.tig.2013.09.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A century after Fisher: Time for a new paradigm in quantitative genetics.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2013.09.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D669%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.09.006%26rft_id%253Dinfo%253Apmid%252F24161664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24161664&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.28" data-doi="10.1534/genetics.111.127068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting major genetic loci controlling phenotypic variability in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127068%26rft_id%253Dinfo%253Apmid%252F21467569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.29" data-doi="10.1186/1471-2156-13-63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-13-63</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D63%26rft.epage%253D63%26rft.atitle%253DRecent%2Bdevelopments%2Bin%2Bstatistical%2Bmethods%2Bfor%2Bdetecting%2Bgenetic%2Bloci%2Baffecting%2Bphenotypic%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-63%26rft_id%253Dinfo%253Apmid%252F22827487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827487&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.30" data-doi="10.1371/journal.pgen.1002839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronnegard</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inheritance beyond plain heritability: variance-controlling genes in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002839</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DShen%26rft.auinit1%253DX.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002839%26rft.epage%253De1002839%26rft.atitle%253DInheritance%2Bbeyond%2Bplain%2Bheritability%253A%2Bvariance-controlling%2Bgenes%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002839%26rft_id%253Dinfo%253Apmid%252F22876191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22876191&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.31" data-doi="10.12688/f1000research.2-200.v1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronnegard</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Issues with data transformation in genome-wide association studies for phenotypic variability.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.2-200.v1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D2%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.2-200.v1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.2-200.v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.32" data-doi="10.1101/gr.171601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimomura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Low-Zeddies</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steeves</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiteley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genome-wide epistatic interaction analysis reveals complex genetic determinants of circadian behavior in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.171601</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShimomura%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D959%26rft.epage%253D980%26rft.atitle%253DGenome-Wide%2BEpistatic%2BInteraction%2BAnalysis%2BReveals%2BComplex%2BGenetic%2BDeterminants%2Bof%2BCircadian%2BBehavior%2Bin%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171601%26rft_id%253Dinfo%253Apmid%252F11381025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjExLzYvOTU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.33" data-doi="10.1186/gb-2013-14-7-r82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenaway</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gailus-Durner</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A comparative phenotypic and genomic analysis of c57bl/6j and c57bl/6n mouse strains.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-7-r82</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR82%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-7-r82%26rft_id%253Dinfo%253Apmid%252F23902802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-7-r82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23902802&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Generalized linear models with varying dispersion. J. R. Stat.</span> <abbr class="cit-jnl-abbrev">Soc. Ser. B Methodol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSer.%2BB%2BMethodol.%26rft.volume%253D51%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.35" data-doi="10.1111/biom.12651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soave</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/biom.12651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D73%26rft.spage%253D960%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbiom.12651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/biom.12651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.36" data-doi="10.1016/j.ajhg.2013.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">What is the significance of difference in phenotypic variability across SNP genotypes?</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.06.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.017%26rft_id%253Dinfo%253Apmid%252F23910463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910463&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.37" data-doi="10.1534/genetics.104.039313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.039313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039313%26rft_id%253Dinfo%253Apmid%252F16361245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE3ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3783.38" data-doi="10.1023/A:1021617618150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Iteratively reweighted least squares mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1021617618150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D341%26rft.epage%253D355%26rft.atitle%253DIteratively%2Breweighted%2Bleast%2Bsquares%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021617618150%26rft_id%253Dinfo%253Apmid%252F9926616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021617618150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9926616&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3783.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3783.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3767" title="QTL Mapping on a Background of Variance Heterogeneity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3791" title="PIE-1 Translation in the Germline Lineage Contributes to PIE-1 Asymmetry in the Early Caenorhabditis elegans Embryo" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3783" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bCOsNwyMLHRRzr7gM6oGy_-ajpmFuxyn0hEUSbcx86M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3783" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3783" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3783" class="active" data-icon-position="" data-hide-link-title="0">Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491519" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CH_mhJReyB4W4Zd_cLTgQzG0yTK3JcfiixfOr0JGfO0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3783.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3783" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VXrbkiRJksmV0iGC03Kd4jx3HoZzWY-rqcUoF6D9fQQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3783" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3783.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491519" id="node491519--2" data-pisa="ggg;8/12/3783" data-pisa-master="ggg;g3.118.200194" data-apath="/ggg/8/12/3783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vivek Kumar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph S. Takahashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3783-3790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491519--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vivek Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Jackson Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivek%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVivek%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vivek Kumar</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph S. Takahashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute, Department of Neuroscience, University of Texas Southwestern Medical Center, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BS.%2BTakahashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahashi%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BS.%2BTakahashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph S. Takahashi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3783" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3783&amp;text=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3783&amp;t=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3783&amp;title=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491519/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491519" id="node491519--3" data-pisa="ggg;8/12/3783" data-pisa-master="ggg;g3.118.200194" data-apath="/ggg/8/12/3783.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vivek Kumar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph S. Takahashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3783-3790;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491519--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vivek Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Jackson Laboratory</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivek%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVivek%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6643-7465" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vivek Kumar</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph S. Takahashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Howard Hughes Medical Institute, Department of Neuroscience, University of Texas Southwestern Medical Center, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BS.%2BTakahashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahashi%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BS.%2BTakahashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0384-8878" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph S. Takahashi</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3783&amp;rft.epage=3790&amp;rft.atitle=Mean-Variance%20QTL%20Mapping%20Identifies%20Novel%20QTL%20for%20Circadian%20Activity%20and%20Exploratory%20Behavior%20in%20Mice&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Corty%2C+Robert+W.&amp;rft.au=Kumar%2C+Vivek&amp;rft.au=Tarantino%2C+Lisa+M.&amp;rft.au=Takahashi%2C+Joseph+S.&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491519/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491519/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491519/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491519/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491519/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491519/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491519/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491519/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491519/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491519/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491519/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491519/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3783" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Reanalysis of kumar et al. reveals a new mqtl for circadian wheel running activity</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Reanalysis of bailey et al. identifies a new vqtl for rearing behavior</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3783.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3783.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3783.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3783.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
