<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3973" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3973.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3973.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3973.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3973/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3973.atom" />
<meta name="HW.pisa" content="ggg;8/12/3973" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" />
<meta name="DC.Identifier" content="10.1534/g3.118.200729" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Silke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.25387/g3.7243256" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7081094" />
<meta name="DC.Description" content="We present an RNA-Seq based approach to map 3′ end sequences of mature 16S rRNA (3′ TAIL) in bacteria with single-base specificity. Our results show that 3′ TAILs are heterogeneous among species; they contain the core CCUCC anti-Shine-Dalgarno motif, but vary in downstream lengths. Importantly, our findings rectify the mis-annotated 16S rRNAs in 11 out of 13 bacterial species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes). Furthermore, our results show that species-specific 3′ TAIL boundaries are retained due to their high complementarity with preferred Shine-Dalgarno sequences, suggesting that 3′ TAIL bases downstream of the canonical CCUCC motif play a more important role in translation initiation than previously reported." />
<meta name="DC.Contributor" content="Jordan R. Silke" />
<meta name="DC.Contributor" content="Yulong Wei" />
<meta name="DC.Contributor" content="Xuhua Xia" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We present an RNA-Seq based approach to map 3′ end sequences of mature 16S rRNA (3′ TAIL) in bacteria with single-base specificity. Our results show that 3′ TAILs are heterogeneous among species; they contain the core CCUCC anti-Shine-Dalgarno motif, but vary in downstream lengths. Importantly, our findings rectify the mis-annotated 16S rRNAs in 11 out of 13 bacterial species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes). Furthermore, our results show that species-specific 3′ TAIL boundaries are retained due to their high complementarity with preferred Shine-Dalgarno sequences, suggesting that 3′ TAIL bases downstream of the canonical CCUCC motif play a more important role in translation initiation than previously reported.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3973" />
<meta name="citation_id" content="8/12/3973" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3973" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3973.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3973.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3973.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200729" />
<meta name="citation_pmid" content="30355764" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3973" />
<meta name="citation_lastpage" content="3979" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jordan R. Silke" />
<meta name="citation_author_institution" content="Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1578-1933" />
<meta name="citation_author" content="Yulong Wei" />
<meta name="citation_author_institution" content="Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1218-2086" />
<meta name="citation_author" content="Xuhua Xia" />
<meta name="citation_author_institution" content="Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5" />
<meta name="citation_author_email" content="xxia@uottawa.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3092-7566" />
<meta name="citation_reference" content="citation_journal_title=G3 Genes|Genomes|Genetics (Bethesda);citation_author=A. Abolbaghaei;citation_author=J. R. Silke;citation_author=X. Xia;citation_title=How Changes in Anti-SD Sequences Would Affect SD Sequences in Escherichia coli and Bacillus subtilis;citation_pages=121-1615;citation_volume=7;citation_year=2017;citation_doi=10.1016/j.foreco.2013.04.040" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=49-410;citation_volume=215;citation_year=1990;citation_doi=10.1255/jnirs.760" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=S. Anders;citation_author=D. J. McCarthy;citation_author=Y. Chen;citation_author=M. Okoniewski;citation_author=G. K. Smyth;citation_title=Count-based differential expression analysis of RNA sequencing data using R and Bioconductor.;citation_pages=2517-1786;citation_volume=8;citation_year=2013;citation_doi=10.2135/cropsci2017.01.0007." />
<meta name="citation_reference" content="citation_journal_title=DNA;citation_author=L. Band;citation_author=D. J. Henner;citation_title=Bacillus subtilis Requires a “Stringent” Shine-Dalgarno Region for Gene Expression.;citation_pages=1120-21;citation_volume=3;citation_year=1984;citation_doi=10.1002/jsfa.2740361114" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=High-throughput phenotyping and genomic selection: the frontiers of crop breeding converge.;citation_pages=312-320;citation_volume=30;citation_year=2014;citation_issue=5;citation_pmid=22420640;citation_doi=10.1111/j.1744-7909.2012.01116.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. V. Boni;citation_author=D. M. Isaeva;citation_author=M. L. Musychenko;citation_author=N. V. Tzareva;citation_title=Ribosome-messenger recognition: mRNA target sites for ribosomal protein S1.;citation_pages=e0156744-162;citation_volume=19;citation_year=1991;citation_doi=10.1371/journal.pone.0156744" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=G. A. Cangelosi;citation_author=W. H. Brabant;citation_title=Depletion of pre-16S rRNA in starved Escherichia coli cells.;citation_volume=179;citation_year=1997;citation_pmid=27172218;citation_doi=10.1534/g3.116.029637" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. H. Choi;citation_author=C. H. Hagedorn;citation_title=Purifying mRNAs with a high-affinity eIF4E mutant identifies the short 3′ poly(A) end phenotype.;citation_pages=55-7038;citation_volume=100;citation_year=2003;citation_doi=10.1073/pnas.1232347100" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=M. P. Deutscher;citation_title=Twenty years of bacterial RNases and RNA processing: how we’ve matured.;citation_volume=21;citation_year=2015;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Duval;citation_author=A. Korepanov;citation_author=O. Fuchsbauer;citation_author=P. Fechter;citation_author=A. Haller;citation_title=Escherichia coli Ribosomal Protein S1 Unfolds Structured mRNAs Onto the Ribosome for Active Translation Initiation.;citation_volume=11;citation_year=2013;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Edgar;citation_author=M. Domrachev;citation_author=A. E. Lash;citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository.;citation_pages=7-210;citation_volume=30;citation_year=2002;citation_doi=10.1016/j.compag.2013.08.021" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.;citation_pages=6-194;citation_volume=8;citation_year=1998;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Open Biol;citation_author=A. J. Hockenberry;citation_author=A. R. Pah;citation_author=C. J. Jewett;citation_author=L. A. N. Amaral;citation_title=Leveraging genome-wide datasets to quantify the functional role of the anti-Shine-Dalgarno sequence in regulating translation efficiency.;citation_volume=7;citation_year=2017;citation_pmid=19483184;citation_doi=10.1093/treephys/tpp036" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. E. Jones;citation_author=A. L. Brown;citation_author=U. Baumann;citation_title=Estimating the annotation error rate of curated GO database sequence annotations.;citation_pages=44;citation_volume=8;citation_year=2007;citation_doi=10.1016/j.fcr.2012.10.013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. V. Komarova;citation_author=L. S. Tchufistova;citation_author=M. Dreyfus;citation_author=I. V. Boni;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=187;citation_year=2005;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=250-258;citation_volume=80;citation_year=2009;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Lagesen;citation_author=P. Hallin;citation_author=E. A. Rodland;citation_author=H. H. Staerfeldt;citation_author=T. Rognes;citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes.;citation_pages=437-3108;citation_volume=35;citation_year=2007;citation_doi=10.1255/jnirs.1006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G.-W. Li;citation_author=E. Oh;citation_author=J. S. Weissman;citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.;citation_pages=1023-541;citation_volume=484;citation_year=2012;citation_pmid=26929265;citation_doi=10.1111/1755-0998.12513" />
<meta name="citation_reference" content="citation_author=H. Li;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=Y. H. Lin;citation_author=B. C. Chang;citation_author=P. W. Chiang;citation_author=S. L. Tang;citation_title=Questionable 16S ribosomal RNA gene annotations are frequent in completed microbial genomes.;citation_pages=158-47;citation_volume=416;citation_year=2008;citation_doi=10.1007/s11306-016-1099-1" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. Lister;citation_author=R. C. O’Malley;citation_author=J. Tonti-Filippini;citation_author=B. D. Gregory;citation_author=C. C. Berry;citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis.;citation_pages=126-536;citation_volume=133;citation_year=2008;citation_doi=10.1016/j.fcr.2004.12.002" />
<meta name="citation_reference" content="citation_author=M. Martin;citation_pages=284;citation_volume=6;citation_year=2011;citation_pmid=22885252;citation_doi=10.4161/fly.21535" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Nakagawa;citation_author=Y. Niimura;citation_author=K. Miura;citation_author=T. Gojobori;citation_title=Dynamic evolution of translation initiation mechanisms in prokaryotes.;citation_pages=293-6387;citation_volume=107;citation_year=2010;citation_doi=10.1007/s004420050591" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Nakagawa;citation_author=Y. Niimura;citation_author=T. Gojobori;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=45;citation_year=2017;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. Nussinov;citation_author=G. Pieczenik;citation_author=J. R. Griggs;citation_author=D. J. Kleitman;citation_title=Partial least squares regression, support vector machine regression, and transcriptome-based distances for prediction of maize hybrid performance with gene expression data.;citation_pages=825-833;citation_volume=35;citation_year=1978;citation_issue=5;citation_pmid=22101908;citation_doi=10.1007/s00122-011-1747-9" />
<meta name="citation_reference" content="citation_title=Curr. Protoc. Mol. Biol.;citation_author=D. O’Neil;citation_author=H. Glowatz;citation_author=M. Schlumpberger;citation_title=Ribosomal RNA depletion for efficient use of RNA-seq capacity.;citation_pages=2372;citation_volume=Chapter 4;citation_year=2013;citation_doi=10.2135/cropsci2016.09.0742" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. A. Osterman;citation_author=S. A. Evfratov;citation_author=P. V. Sergiev;citation_author=O. A. Dontsova;citation_title=Comparison of mRNA features affecting translation initiation and reinitiation.;citation_pages=159-486;citation_volume=41;citation_year=2013;citation_doi=10.1093/nar/gks989" />
<meta name="citation_reference" content="citation_journal_title=J. Gen. Virol.;citation_author=R. Prabhakaran;citation_author=S. Chithambaram;citation_author=X. Xia;citation_title=Escherichia coli and Staphylococcus phages: effect of translation initiation efficiency on differential codon adaptation mediated by virulent and temperate lifestyles.;citation_volume=96;citation_year=2015;citation_pmid=25614589;citation_doi=10.1099/vir.0.000050" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=X. Qu;citation_author=L. Lancaster;citation_author=H. F. Noller;citation_author=C. Bustamante;citation_author=I. Tinoco;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=109;citation_year=2012;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Schurr;citation_author=E. Nadir;citation_author=H. Margalit;citation_title=Identification and characterization of E.coli ribosomal binding sites by free energy computation.;citation_pages=216-4023;citation_volume=21;citation_year=1993;citation_doi=10.2135/cropsci2016.06.0526" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Shine;citation_author=L. Dalgarno;citation_title=The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites.;citation_pages=1404-1346;citation_volume=71;citation_year=1974;citation_doi=10.2135/cropsci2016.08.0675" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Shine;citation_author=L. Dalgarno;citation_title=Determinant of cistron specificity in bacterial ribosomes.;citation_pages=30-38;citation_volume=254;citation_year=1975;citation_pmid=26732811;citation_doi=10.1186/s12864-015-2345-z" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=F. Sievers;citation_author=D. G. Higgins;citation_title=Clustal Omega, accurate alignment of very large numbers of sequences.;citation_volume=1079;citation_year=2014;citation_pmid=26224105;citation_doi=10.1093/treephys/tpv056" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Starmer;citation_author=A. Stomp;citation_author=M. Vouk;citation_author=D. Bitzer;citation_title=Predicting Shine–Dalgarno Sequence Locations Exposes Genome Annotation Errors.;citation_pages=2413;citation_volume=2;citation_year=2006;citation_doi=10.1007/s00122-016-2780-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sulthana;citation_author=M. P. Deutscher;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=288;citation_year=2013;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=T. Tuller;citation_author=Y. Y. Waldman;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=107;citation_year=2010;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=BMC Mol. Biol.;citation_author=V. Vimberg;citation_author=A. Tats;citation_author=M. Remm;citation_author=T. Tenson;citation_title=Translation initiation region sequence preferences in Escherichia coli.;citation_pages=1681;citation_volume=8;citation_year=2007;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=141-63;citation_volume=10;citation_year=2009;citation_doi=10.1255/jnirs.1108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Wang;citation_author=M. Weiss;citation_author=M. Simonovic;citation_author=G. Haertinger;citation_author=S. P. Schrimpf;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=11;citation_year=2012;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Wang;citation_author=C. J. Herrmann;citation_author=M. Simonovic;citation_author=D. Szklarczyk;citation_author=C. von Mering;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=15;citation_year=2015;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=M. S. Waterman;citation_author=T. F. Smith;citation_title=RNA secondary structure: a complete mathematical analysis.;citation_pages=219-266;citation_volume=42;citation_year=1978;citation_doi=10.1016/j.foreco.2017.09.003" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=Y. Wei;citation_author=J. R. Silke;citation_author=X. Xia;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=7;citation_year=2017;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. R. Williams;citation_author=A. Baccarella;citation_author=J. Z. Parrish;citation_author=C. C. Kim;citation_title=Trimming of sequence reads alters RNA-Seq gene expression estimates.;citation_pages=32;citation_volume=17;citation_year=2016;citation_doi=10.1016/j.fcr.2017.08.020" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. R. Woese;citation_author=G. E. Fox;citation_author=L. Zablen;citation_author=T. Uchida;citation_author=L. Bonen;citation_title=The genetics of quantitative traits: challenges and prospects.;citation_pages=565-577;citation_volume=254;citation_year=1975;citation_issue=8;citation_pmid=19584810;citation_doi=10.1038/nrg2612" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. R. Woese;citation_author=L. J. Magrum;citation_author=R. Gupta;citation_author=R. B. Siegel;citation_author=D. A. Stahl;citation_title=Secondary structure model for bacterial 16S ribosomal RNA: phylogenetic, enzymatic and chemical evidence.;citation_pages=2002-2293;citation_volume=8;citation_year=1980;citation_doi=10.3389/fpls.2017.02002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Xia;citation_title=A Major Controversy in Codon-Anticodon Adaptation Resolved by a New Codon Usage Index.;citation_pages=337-579;citation_volume=199;citation_year=2015;citation_doi=10.1534/genetics.114.172106" />
<meta name="citation_reference" content="citation_journal_title=G3 Genes|Genomes|Genetics (Bethesda);citation_author=X. Xia;citation_title=ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data;citation_pages=221-3848;citation_volume=7;citation_year=2017;citation_pmid=25387018;citation_doi=10.1146/annurev-animal-031412-103705" />
<meta name="citation_reference" content="citation_author=X. Xia;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=X. Xia;citation_title=DAMBE7: New and Improved Tools for Data Analysis in Molecular Biology and Evolution.;citation_pages=1550-1552;citation_volume=35;citation_year=2018;citation_doi=10.1093/molbev/msy073" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="We present an RNA-Seq based approach to map 3′ end sequences of mature 16S rRNA (3′ TAIL) in bacteria with single-base specificity. Our results show that 3′ TAILs are heterogeneous among species; they contain the core CCUCC anti-Shine-Dalgarno motif, but vary in downstream lengths. Importantly, our findings rectify the mis-annotated 16S rRNAs in 11 out of 13 bacterial species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes). Furthermore, our results show that species-specific 3′ TAIL boundaries are retained due to their high complementarity with preferred Shine-Dalgarno sequences, suggesting that 3′ TAIL bases downstream of the canonical CCUCC motif play a more important role in translation initiation than previously reported." />
<meta name="og-title" property="og:title" content="RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3973" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We present an RNA-Seq based approach to map 3′ end sequences of mature 16S rRNA (3′ TAIL) in bacteria with single-base specificity. Our results show that 3′ TAILs are heterogeneous among species; they contain the core CCUCC anti-Shine-Dalgarno motif, but vary in downstream lengths. Importantly, our findings rectify the mis-annotated 16S rRNAs in 11 out of 13 bacterial species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes). Furthermore, our results show that species-specific 3′ TAIL boundaries are retained due to their high complementarity with preferred Shine-Dalgarno sequences, suggesting that 3′ TAIL bases downstream of the canonical CCUCC motif play a more important role in translation initiation than previously reported." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XblP1joa_9HfDUACp7hCVGkmcdEkqRoKcx6UJbBXkS0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491516","apath":"\/ggg\/8\/12\/3973.atom","pisa":"ggg;8\/12\/3973","ac":{"\/ggg\/8\/12\/3973.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3973.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3973"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3973"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3973.full":true,"\/content\/8\/12\/3973":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QA8P\\\\\\\/Y9OXr+hmimCcrJHCg9pgZQ2AF7Tw3NOY3ZLs9ICfgg1LexoJRPbUPLkocDrmwaEOVl\\\\\\\/KBPYcidDR8vd0SyiY2sxgloZLbo3d9XzrovjT0ujAC4NXFO0r0Dh\\\\\\\/\\\\\\\/TEWSwW0IdzjCEKTTjMVJlGDYkw1190Smjhqp+xAVmNOdHw1kWJUYKxAMNhYLtLRYnVqWVAGcywIAqAu99biKMAfYD7nAV5EgTEgSzLLVotid0X6QqUyPGzI1y4\\\\\\\/c9Mim8SnzaXF3M4g8gQoDLUHfmi\\\\\\\/r6ucNuJdgd1bRcBxJv7RKIfqmXWM+XaVJ4+03t5703T9zZU8kVIY7xeNuNir18D\\\\\\\/XGC\\\\\\\/Oa+2cME4T4GDUqNwgjvImkVM548i6rwLNbsHJDYL6RFJyZsjqApnx1RIBbh\\\\\\\/e9WNWXJKnJ1gZgbLbprUt7W2+Wr0CcJg5alEJe9Yc\\\\\\\/5xuudOaEsJcBk6ri5t6FJE3XQYyKIx\\\\\\\/XLPR\\\\\\\/wkNHlf\\\\\\\/6n24HIvGH0PsjU4Dyk3fouThPllmhd50zALM3OZLshZ1vkcvIeJTUZGM3B9PJKKtTuReEz5jE8Sb6s520c8WR15uhPtLNpALa28BKxcjSwWRntuhFXlPaV0DxQ3NAUsxs0FsRpLTJIKrn+P0vR0CMT6sz\\\\\\\/0YDztZcJb3zD3aQsOaSwWewsW2MIQiRzYEubCLSpIFIYqBJsWP4pXWP4XIS8qsQsB2pm4QRFLfkwKw\\\\\\\/rvS\\\\\\\/jEzC2Qt8QA6u2Bt6iHuZwV6G4MZmNeEotM8C4K4wVE\\\\\\\/WBqLu48BHzJI3fZGYDnYylewSsqL3ryvb8dNLv4JPaEBLRGdqEJwGlrALb65yn1nfc5Y4CJqYzIHa4cLfFGxDvnDRZfBFBepGdMWSJ\\\\\\\/jX+OgVTUVJZeSfX8HRyZXFVtI2x9mIN5mOZE0iulcgCzrcGRSkIkLNqKNZXAbmHB5oKtCEQHIhYdmUh1kCQD+d+gL6ozgBF+k2ZI+6n\\\\\\\/WoavtY5sdNjmUsihiPwMrnpuWDkSC8mh6qdveCewA75O6xQGJ\\\\\\\/4R7r5W3pC7XXLBAvS2zH7QmXW08qMI9EjTKtWVN\\\\\\\/0Ahe4zU7xnc0qvi8HRor1t7xh2lztvEvIMdqfGw1lveX06Qf7GaIWl6glD9QvJbLyruwoxIaO6ftL5g+7EB5GXj5jxLLmzjf6pP7M5ei\\\\\\\/NqGn\\\\\\\/XxRs6a0jSRQ34oRIFOLOX2w7ZckD1pGY3TrSIChqJSFtfK19p05Ah0+H9GbQ+A6mCR3dFLChQszdz9gQlZFM+juetqbCBZKdcyjPfeR\\\u0022,\\\u0022iv\\\u0022:\\\u00226KMiR4ruEgRwPnf6CQEKdA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ac8878bad67df3d1c0e57e5985f1de1eb3ca78513f0fbb79bc08432cd2ddf69\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022k9zwAyByWLOrpTt3D1zSBxzo8+OukfTW4Pn\\\\\\\/SkijRuvA69rojeiIXKhbGhwrtEnuQxorqjInKV\\\\\\\/NJSHz74IJ4sBG6cdboeQb\\\\\\\/nIKmG\\\\\\\/BBVrgM44WGrvg22nI0fu\\\\\\\/jjL6bF86+4FTedKkN4s5C7vpTt9lA2lEEdoufxM\\\\\\\/VAyTuV3MZX1odEW7jVLnn8h87vUNXjEAJoWTKbN\\\\\\\/O+1ZRTAa4T5ZwLU3ADMX8t4dlKH8Sv2iwjRPWSDi8WdU0usDJSXQaGVRuGZeY\\\\\\\/vvVuC3Fw0OTh8qWmhUv9QfKcXcI\\\\\\\/Z4orJHfUGHz5G4l1QaGAOXdTw9xoolVKwe+EfopfC++XmKVmaXJ5pwvuI418qWl\\\\\\\/XmXik8da1o15JgGJPc1LnQavwZckbn+I6rN2SimKQsmUezjFMndpnmzq5UmNLDp5QRfVxhVB510idIWMvN09S051I2yWOxTWp9LmmNj4K5zLUDA4R75PYqzVqZiENr8fC\\\\\\\/IHmN1sC89vFNlJAz+gL6CJqZ0rEFaw\\\\\\\/7p21TTA\\\\\\\/MwdrGl+6AsNtIMiit\\\\\\\/2Ab+zCe+jpxjbIRVxwIoqSvFTqeLP80olZg+NbP5F9oPHystY8JavUZ22dTqprEM\\\\\\\/MPgYU1tYs6Ha+EfaOvrUjU3oIUh8S8Gf39u0A+t5wcbeCK51FdCnFuaLFRwhAG6Ns60NvrTAbjZbt73frN2SexRsQUqEVrBePGdUtmuhS0cbxeSMe79EGDOqtVProhD0G8MWWbQqsFriKvtxRjltOgiysxyODHS5rGY\\\\\\\/Ul\\\\\\\/Hoa48BzhuHMXMFahmMGvI18u1SqUFu5CvaSCJGRiODFvT+V5IwXEDGDnec3se4Aks636klHiHS8+Il1fGgLkWrS+cb1AXo0m4zJRSkV1UpQWSMC8ycIV\\\\\\\/McrHiQTS6Pw29UjroLWp2jwg5FiEMSTQVTXVUbjL5EFew+TRdK79TEteVANwLDZt4pS131LZ1ZqCHW263QmoASkCAeu3Nm\\\\\\\/FO2IcRwDJogftQG5q+adrK9uLFdxNHa+gJDG5W0bVBYh1LYZMaVMWIsug3SINFdti\\\\\\\/89XKgJghHe2GL4URqeMDSDPbCysF4\\\\\\\/HltD1fTS2OpHWHHOKTa1b\\\\\\\/TwqMazMNAGDgb21HkIfN0Vte9FN3agLYNbR4q6cdoT+NRLQrehyigK8rOE+YpvmdGxxK+di59CVJ\\\\\\\/c1xHjQY0893BRuae6V+oMDFjWkBeybmN3oS1z9Qzz\\\\\\\/+XuZkIlJTfYLqqCix1a4sJQyDtzDNNo6Gega+x8VyvSuPcT4G246nK1raQO6Nh47uPNLah8YmgcAQj3jaQy5d2TDbvs5K6oDQo0\\\\\\\/xD9KEPos1XJbAe9Jy8UbABUcnKhx3id0E5cANjR77Jt4CrPOKm64+5xtq1YRm0Q9dfNdK60JdZkMF9jm8XHUVnsoTbt2H1UleLLN\\\\\\\/Z+uwMTfUcdDhXL11\\\\\\\/PfdGBzGHElHCwxIHScd3whHP1EhoomNQ0hr1VsK1Dkt8fMOFWSh9iJgHq\\\\\\\/Wm0Fbigmv5e7baqWM6wCzo5Lbz81UjILmAZQAvN04wC6vLRWNpxLGH2dsLbLxukG\\\\\\\/nP7DaOq0H5zIehr8XMjDgGLyGhqlbkCmNw+Szynj0qLiuUkTIkNrJNTP58vmd3nqEYvx\\\\\\\/SbaNI4OF6Ileh+clsqnCqzGSrxVE1dZcJJmv74rUyKxGm7Uz127+JncFrruRf6LzAqoQpHPqJXXH52gtcxHW9ijV8uV1A02T3DF\\\\\\\/8UlJvr11AIu10sANoeAM5znAGWhKPFO9A\\\\\\\/uQP\\\\\\\/Rxor4WCTDBSDn4u7S+dGGivUIduGhpV4XyHk8gYLY3q07ugEKK9Q23YoZIzhYS9cHdY6Oxjw0brWTH2jpxg3BP2QSwpTwVEdY84zRZBX8tAy1AipbUGYwLvN1CwtebUgwxsnIiO5xxZ+I8Xh2ErAFggaMPdF7dC6X4OaN1puEe4n+k9nsips5dgTmT0ajF+oP6dV9v5O0iwivw2aVDDXr4\\\\\\\/ZVSncUCsdMprwubRGzvM0MWO047JUoYsLTin4rHvps8I9QpIzRBGRCPM9BPJYwF9nudLE8tGzRKtFQvTPxLL04LOpUpkNBmu1HMV18\\\\\\\/PmlAwC89xqstH+CbAtyhM81Bp2uKV5WEMCmRPMSriXSOR+dgExtpVoL1NR9IIGurR8gBLlJjtvKDLOMj2udJ4eHjtjEuJtRM4rct7GIaVYZNhK7UEJBwqgbglgaDvk0H0exyJIw8NMPev8L5jJFaDIi\\\\\\\/eXrV23sLuknx7XwTxTGlX7T4g4x\\\\\\\/RInSKuHb1cc2ELQC29f9M0fLZNiNU7ynus4nd6DqnZxJ7zP3SzECpzfDegVyy54+l5QENmRAdhJrzykCOr7BaJGKjXC3e58VjauMqF2W4A2gXWAPxnfaPCvIBSFijPgBjVfo+Sq2tDnPi9+pnbOa\\\\\\\/IVlpmx6sd7ZGhbGgqHqCRxGTR5\\\\\\\/ASJ3jQwfTDXhCyX2cPYGRSPtXAbhNd+HtdO5fZmdLFrHY55OtzVOJHA5jAeXdXg6yOUKmH\\\\\\\/os0e58V+mRUb0T+HNM37KCXat1zyScw9TT9hvv5KETyAuOK8FgA5Eoghxee6oB9VXxbiye2\\\\\\\/qBVSbEFekeIRrnEkK3UcpiPoy\\\\\\\/ikNnd8imEdbXIb+Vo\\\\\\\/Fj6UEC4iUoL1PckTnHvgqYB30iMBIvQV3E3w2DckpJTzg8GR3HgVyhgZXcnLrxh3+qEck\\\\\\\/nK0kxOk2luY1bPLFkbsJWVvsjX+Pyy2FtUthKNK3UkGJz26CpiV0rrvYz4qNpnz\\\\\\\/fA7\\\\\\\/DpkHLuCNbpevl+XHn01nxMOw4MQjsrUCHop57EQ7ZtYscDpiBjPRwTFfcTdkkTcx9D9IsTb2QZfNirTyDi6YdLmAKxZ\\\\\\\/bInG88mdgZS6ekuNTGNhq\\\\\\\/0AW17y4DTSk7q8G5MJ68gMh6xyJRdq\\\\\\\/uzpQhgKNlK0HRRNwlI48cWOByqObM0uNUgKBHmM013AqzK3NLgkFq7bnHmQl9bqjsfiPqBsIPQKe7v31PCRYoBldPSKLuUcf8KDWfx6snjLFHTkVSDGylIxEheMWayWwyIbrmqymgv1vYe2p9KCrExf+C4ryRF+Yu8h2nb3qzPC9Pd60+7+Ts4pC4WJrq\\\\\\\/HrPEFTeHD\\\\\\\/ukxeIhqK5HTTCbPPq7AtsYSlvTYf+p\\\\\\\/M89T0H9TPXDfCqQ3Y3BwN5kRJDTZEwRYAPVWBAZ26OD+486zd1PPHZMtzp8Ml5bbrZtQvxsyI9s80G5R5xAumhu5RvPgmGnP+yPjKmrRccEZZbPhSLQuRZS\\\\\\\/GFPpTvwSZgoxMlDwLh272y5BvR1WPfVB6T31akjavVO0v6A26l7avV80u+WBzVVrFDT0weSRMZu1KJtX75z6NFa0CvPvc4BbBVQAL5\\\\\\\/3UJgRhs+0VhDmnwPAIqjWhzUDiPfD0WqqN5rX9IK3bOUduwN2lWe0++37Gd+6t2A0YP1OBYTUD0di5mlEFe5cQxkOdqSpb9TVnTxHpkBFs96Y3f5t7h87WlFQVPbee36YPi3InRD5+8zlNMtCwIduKOK3zMDn+jP4vAFvxf8JtD9BrYpr+\\\\\\\/4Qr8RBJVrlhuTV1c1gsMzP3Ref8G5FHY+Wr7Iz5wc3RrZKPp+A8azFV+h5sPXKJtVKs3fmiR1MkvUPoAwAUDXU\\\\\\\/\\\\\\\/DRINsx6iO7TR4zib+bm8DJq27OQBNORrWCXI1hUumNfL\\\\\\\/z5UEhONQY5vFREsJGuZsASXww10mWpHE6ToD0eAPDuf\\\\\\\/5A6xeqdPN6sKkaNEHdKTl872LP6Q0DNdpncgFvgYfroRGLBLiar5W\\\\\\\/+LZOWUbpb+INEJ6wNaLy8QD+QGYPL2lRll6Txz+24KT2zxhxhdGUDoHiHQnBO7+LyWb585nfRyltJewbbrhcmbxyysQhdgbqO68dxaxw3KWQgBaYQwKAJfM\\\\\\\/h73rY01cy4f2wOgWLg0Vg3ScqCu56YrnQeM+5QZImnBK2+t\\\\\\\/HX+nnITIpExeTtvhHa5iYvXoX7HNxfcjBYdYgEpa0trPkVEOcSb4DjRXRpUvyZ8YyZhApfrMYywxsu5mu\\\\\\\/RAR5RlcgEi2Eeapy9e\\\\\\\/D11vWZr7nxHkxrFk\\\\\\\/vAkXJQC8ent927SFLjrrT46FQiYG\\\\\\\/9OvDKaihyxafbZM95SQe\\\\\\\/S+bs1vWnczMqTfmVFjnjx8YD25tDtDh+2l0SDejbTW3aUEv57GxjpEu0VyDNVRhdo2KqnFD654fIQD6WOSFBKZLyhhazthLhjk\\\\\\\/U3MHFI36nGmGsfJH+d1qsiUvR15EjFGgrPi8JSomUsXeI\\\\\\\/cY717HU1duAp5XT6Pz6kqPMrxFMabgGTBnIC0Dbn6JZrvVQ0N\\\\\\\/iqhm6ybjyyU8m0A85b\\\\\\\/hALD6ldiw5pB1EOmhxwwIy7iiyVrdybiPJM9POJ9dtwfCpJfzROGx4OMhxILKrelallYhKdSDn6FWDvlCrT5yPv\\\\\\\/2SjbHCos7\\\\\\\/Al22Dvd2ZVVT+561sClEHaXcFMh8QeQgfZZD4AxT03\\\\\\\/MnJw5AF3UhANNEpY\\\\\\\/kert5HcFyKkJPux2q0Aq5ARkd1zsRy0Xj6AFEFWyti1yrnVNZMNsi6hQN\\\\\\\/TbYALwZhIV\\\\\\\/YC0g5OuQ23DSa9BRaIUaoG5lC32tnvakHr81CoOfmI+bxTps08B4TjCzWv+ef0U9v44ReYBvafej5in54+6S3taf5SR34KfefP4Dq+IcvG5dSL5gHfSncYUsPTluJ6TPoQXcwV7eeKVRiSwsxMq6mioRKyBzyNMEYdUk\\\\\\\/PxNs2lO+6OQ9g\\\\\\\/lbqv6l3VZsr6gSEwVQKnDnRAtazd2FuRDPEk4U0cFY5O9JfW7pRxDN531VMORYx8IUMw9X1cesOSab4SplPpuE2uUQd75d\\\\\\\/Z9a6egqw7QPLA+zXFpL8yBaUut7XX4EP5suvbS3BlfbPk6oolP0ujvEZ4GPOcZA\\\\\\\/TBHBHkLDOdCHBjPeMSWLXynCixTCs94JpMvTEkjf0mvgZBnUisXrfJYiTjEA2hOCQjR9DPD9gaVGCzKtvaU6h1HWGr+z+meCuTGgWL58EPSJ2jPiyx49HrgcWUhM3bvh1vO6Mct5NBg4KiD9cnkR5U99wvjPnyPGjfllP6vPMeCx7FhR\\\\\\\/0xIZO0yyEwADMv06ytW9tPcqeQlIEXwN2q17ntUqdGxszaKVVX3gOkK4bvVeBuqZ+enLe\\\\\\\/\\\\\\\/cjZFIg6gREsm8ZesTL9k\\\\\\\/UkLm\\\\\\\/eqx0\\\\\\\/gKkeyUw2upww2Gom5HCNI+eXGyzh4IVMmllWQiynqXlbNlmk3sL+J8sYoDCbDxRPK+bOn048+02F1evmI6eLcwSMX0wEhPGDG9INKWQrWGVj+nSdcX3BQObSR97\\\\\\\/fYKh88AhQL0uN0N1IJDqZjVKPZHgCylKp8CnfSgB1bJ85+hGqjFg8q65Eei7CftBzkZcSROQi7zzUYLDzZsqFWaOBFzBmlofzotXrX3cUV4g5kPRC9X6\\\\\\\/nSTcS6ghuosn+pvFcK86t9HqJzaWXqlXW\\\\\\\/MfVOrdlpgm4zHEvhpD5lNtXWF\\\\\\\/TB7jvzrVvZekCJG+yZTcMgx7uTX4UBMK\\\\\\\/VJfAl5kZX274+Yn8GOOAqE51023Lvjj+34ikWtbSuRUed2bkB0TM0APv9ZqbtAPVnajqQNPYtXzFyUspftmWzjd9uCeteqMsmuwVoWPaa+2jh7cSV0QyKN90gZFhVoD2hhgSgT1cBk26Lbdy+eo+nYNhWENmYc4StQU\\\\\\\/2K0bxDz6ZDsbkfKuTKruYolJWPUCR9CNvA8sPgGn6At8Axlmv5CqTBEou2ZYf960haRfKHbsEQnWRd7tRs6vUw1bSG0b8dHOxMjIQrNBNzKpdHHa1+qWod+yyCw8LML6NIiLaNISGfCo3kVxIxTYHKMd5R81e266q8GJAl2sjDWBkKZ+4WESg44YpNv5gosZ9z\\\\\\\/NasGl93oQgD1VbSiNEs5fUbhHtOAZZSekYB2sf4xMpWAl97LzcRopXH9GGWXoA5IZl7wODefVUnuP6gCzp3NKOkFDRH\\\\\\\/oxxFuykrsUz\\\\\\\/P0hfn65p3FZrGAsw6zSNhtkSLpYwFw0ssZo7HimXIkQOT\\\\\\\/8lvRo9Q4677Q7B+26rjn7PxrYYvtZOr0psGp7tBIP\\\\\\\/b5tM5AqxoWtVxvMGh\\\\\\\/8XByFDDanbK9IYzya3VFXhkNSlV5uw05iULLuDMaUjQSG8+OIk8Znxzw\\\\\\\/2uuqyb0vM2rzvhjTJDtoJKBnxGSmfIJbkSNPzuA+Dx5bFkq7vIKB00VlSPrvUHzvYvw7iz7ISCX+PFypk1Qp9kBi3LqHzuiYK11jPBRYqwaG+BoBhgZl4Ek4c4FkUjjVUE8\\\\\\\/\\\\\\\/oFOShzooJm1+qSnXPao39Ceqa1vYjRc40h9JxMLwZs\\\\\\\/PqpxNB1aOR53qPYyZixKpaHkARKcVYRwbz+BvEI\\\\\\\/SwLWNqhIMpddtNdJ5hd5fe9SeoekA6bjX7nMehN\\\\\\\/Vfek\\\\\\\/2KWvTvfhe55jeEJZRy2fQbwXlbss6nav9P8HbBHnuHZumboU9HHfkpbGe7KbT4tCN94HAdyrlJ6c+4wFdTioZKn9T9YgGWVDrbNMB\\\\\\\/fMX4RrvJ6xGVNSILEjykZ6F14kV20IssxjbLwXmti8U1NuHzlWEY43VcAIWXAd2TUGUDxm1iE67BDetVF74\\\\\\\/k7GimNAcyT7VnVwBoNmO7WsEjPhrwQOhJfLB5T6mrYK8MuNU76mF0wsUg5hem2ttjniYf7V5uYSGk3ZWH3Ooia7PE7izbhWoEgaf4DBz5JTTyME5qwr5O9HthRMbhDACUt8ZLm60088yu5gacqZ72dhHEwmB\\\\\\\/O6itKV5xRS6CMmSNHUAjwQqLgrGXDtBgZUoEaS2FiLK7pkWxDtbJ1ScL78knIpSZNul3nKlAQ8pZglmKqUCsg+aX7DZai4oN2qn8uMOt7KsLvKeW2QCc6CJ8\\\\\\\/yvsBYFPV4vEUwGqBszXlsnBbelwJvF+Vip53bDWGvHp5qZMZKmTLIAFHZZh8J\\\\\\\/Xiz8hJWltvjltXVHK7vjcz5neznyKUnUNPmD+75\\\\\\\/l4ulv77mii98yk0EWd6tAMu4k2HSVVnvSa9h3+zpZONUkh7DmSGSEJFmKHwBt7QYo1xWYLvPNEBQbctAQryEcTnn+O1R+H8VKXsYPTDfA1y7hIHoiBx1PoMFXvYw5+2GFB\\\\\\\/Ys2xqQF1DBMxS0YZvSbYQEe\\\\\\\/X5IttK3JWNlTq\\\\\\\/BmTnxH8vW43Rvksl9Q+wfr9uBTX0bV6bUYzZ6bAQaNij6u8WV3+vLM\\\\\\\/RuPNE8tr+Nu4q5VOYFABoIJ\\\\\\\/KCc5G\\\\\\\/bdIsQNvJve146cKvZOJmBJmDmbUTnqpY7JucKX\\\\\\\/M9+\\\\\\\/APF9ACdCB50FGkM\\\\\\\/DnM8g6VMJJJMIhHpSKDec5+tGozJ2+l3S53Qcrvm0SUAdJDjh5t\\\\\\\/TS8GUVyOGNZJg3imzZv7kDGssJQFo3yiVJCFTSz9ZX0LXe291U2dl2\\\\\\\/S3Pja9nwFJDD40xzwDYQ7KH5K7liMxbykLbObEDrGPNfKSsSegsV3TB3VGpKjWhcVjL+oSymH+KNPDkPfdV4TmDd94XIPhP2+c7d4o4s1ZmCN0uu2hjHYJjkjXKjBSgvF+Co4s6L7mlcSJ+qu8Mk4SqfaPwFjG7xhSmWEIYN25N9pzrQ4YAaQbIXEjStO7biAJekMbLmwjbvMZslqIZdB\\\\\\\/gl5JaiReP6FfCgDpQ0NblidOzROLymshQbMixwEV4mkGf7WlmG\\\\\\\/J6sv2JBOPqTp3wA807bXB7bKiCsUYzCiljU\\\\\\\/LQoJdtpt21XfvvVHjZhkU0dtE2KOFuvS8ldO017lFWd6PMzGVDtRhKaJvQg6256rwbl0sYBRG+ZZzzPomputOEl5ls12EvrkfvrSi0MeIwn\\\\\\\/PnUAaGw\\\\\\\/b3e5iTqvEAWkJ0Wfkb2UPQUH7trCovyrY2ZlSayX4IgzMEr8nzOWlN1A3fC52+TJ+MUNhHnnkCx8Pbp0iGMLBilUmw9VdDXNrxlpnL3JXkJkHwdGMCZHLrA1lJwBRPmsa1+AFrKG9jkVgQDOuaOGFYJu5GYp0gm\\\\\\\/K5ZUjcHnXtUcs2pOLbRfIysGB+kKDa+CpQP5uyqzTyN5t08SpOdPEcyXdNM2S1PJ1eMn1OqPzrmfVf10RGPSITpU18dVf+5VhM2ggJJQDdE4nTwcO3LNG0LM\\\\\\\/cJAoNCrKy+kD6XZrkmhzpX04\\\\\\\/GPBnQvhYP0S9J5bXFAWeyv+Joki5BFBG\\\\\\\/SeGKDgEfjsmS4unyimsgdaIwPXOcr6tjCYjmq3Y+9NHRcJH1pITfyqo3L3jU37TWTWPUPuVByV2VwKKjEEsq9+QMcSodwXtt4hULOe2Vf3K6co+eSK\\\\\\\/k0F2DgrY0k5e51vftg4\\\\\\\/TMRY1KTB9eIIp5PIObhU2Fh4ZKVuSVPLJcH1UqU36hQ3LMBEnocO\\\\\\\/OPNKZxQR00lm+YfeHpCrYjRpIVQ81SeYNM8OLWIAR9dz\\\\\\\/TvfQ+l8Rg+kyTbBcipw6hV9nI27yzSaJVz7tR4RYRHn\\\\\\\/5kQTvKQPgEfovJ2wRTt+Jz2\\\\\\\/nHPVW77YaxNiiTtpYfZqGSZnUdWwoa4piKanTmtTw1+eEU+RbApTRoqjcqnNyBMMGYlERYvJjHqNg9nkq+LImfDw0PP8WbfPweDWEHIbJq42XQ4oBoFtotnOAj1Tofjb9Vqp6gE\\\\\\\/\\\\\\\/hBWoXLzw6nzK+wlcaihWcPQ5t\\\\\\\/yF5kmwWiUcUiJhOnxpNzZ8qXwyGAnvPOpb8EKybVy65McGSEN7d0WrYyivoHspgHjr6cINz4HuzDzH2xFtsRUW\\\\\\\/fj52fzHYptYzVthoWZwWp6g+WeRgLfGMfv6OrVnoFN3jQG7vMUfkDNS+6LJCluZQ7hHMxlYl3nxPJ83yb7efeBMri4Zrmdw+a91I\\\\\\\/NHb9r6YNIVHbr9tCd2snbfjtIfTB11UQt05yhvInT3BMXCf30eQg\\\\\\\/4NIjbgI3NmTiHs4hIe2fTd5Q+b0TVnNN2uzJhyGHM\\\\\\\/U\\\\\\\/XDQv1xwxLgZb9QfF2GypxtvbI6oYpN4FDaeGZGOaeU3502\\\\\\\/oAxmj11kjvrkqPdFuP2vzlqvlkMHL+DDoQLoIZ5sLSswDzUB4y3lVr8vAxj3HvYdPBmKUuDtmECrppv3Wy51cmnuA1swtfX2imx\\\\\\\/ImoXjP\\\\\\\/pDOWy0b2+DlG6InkfnolzCy6\\\\\\\/aMlF401ZN6wb+UIYUtPyvs2wO528Okg60i33ahygO27ogAV0a1BPWz5IRB7aCpQONlf9N3GpfilPfYQBRKwKQVA+GhPduSfIFMjh\\\\\\\/4n32MjjdNinxdBuwlzqAHXi4CgDlJC+Z6f7fWsQRQtYSqHzYz4hK5BxWs+qgAHRsf3CYjiJZtwHsVuMTxyXFDKg\\\\\\\/wISPQUM71Frz4uEXur2wezNClOEmcAQ23a+oqKosxzVNSY6sbbm8m3Wco6QG2q48lAoA14QHzclPamyZvuvFbjGbfWxY03mEPAUq0QTBJCC5uLd3lt30DJeMGL\\\\\\\/NjN6FvEohNDVN\\\\\\\/cuCbXDhmuFC6c\\\\\\\/3D9mH7Hwk0tvyLahGWYBMtqHP5yKAcFyDCY8thnW009LFfM2zmCHFjLJuyUwwMHEP9IZsRWH5Kvm4VLHXL4K+zMXSMcvtrM8\\\\\\\/T2jBY6QSM+O5TZS3wfxTlrYvn109oJ2HfKD87MHFBg8A0xQrmlVVaB4TWhBDkLEyBsh917Xm\\\\\\\/uSBoZ9\\\\\\\/sAfAuSOoeE4GXEfUjkY2UPEQTLvyIt7zJT+xlQYGA6NU\\\\\\\/wqfB3gGblZUWpafWfJikbe1\\\\\\\/vsL9Fu8y4OMRO7IV779kyvv8+NnHCocDUlEtzuHInrxGgq0BfQnj8\\\\\\\/8zBAqjcIgYF8wyTFhpAmx3RkBYRIRqLOkzSQTv0wWENTAoVjGZzWF1VjrNeiCdDP6qgDb4IMPOgeNurc\\\\\\\/bAD1vf6jtS243jsAkGjl\\\\\\\/eOr7cGp+7ChtwQp3UCEL\\\\\\\/kiqN+UBgszlxMfY+o36Q+bNi5caHyiotWr8SPgG9suD5aIbZpuj3GHUxCrIKgyxjdPQaTgcx9O0IBVglC96\\\\\\\/bdCStNrk6S3TFgrZXkKdBm0L4keLutGBY5TP+\\\\\\\/1M5kMkSR2usaljkTuluKi+yEQufnB9\\\\\\\/SmMCn0U9rGJkLWT7T4tatUADSxHON6j0QVvzOdA8yc4jE9SofceaAH2Dzd9qwcbxe4LEKTUgmOUVS0\\\\\\\/X2pz19lN0xMasHnAq4KmPtL5HkQg4eZur3z1KNN\\\\\\\/0WYBeMPUGi\\\\\\\/soZPs9aC09u440bIx4uqe5XvnqDhd0T4QUkkmF3caGcQ+kl1kVCVmJl7aMSMbOySC4T6ZWurgQVj\\\\\\\/fEZ3JcHIB\\\\\\\/z\\\\\\\/iEjau38AFDVaHsYmJCvvulBZ+JslbalxX\\\\\\\/Fybz6BayabQSpn94+oIvPNZiiPxa5XQD6DTXgcCyaOX\\\\\\\/LnjxpQOsmAy\\\\\\\/+K6qjivrkDUYExO8icKcc7dfmKcVJrUCgK85JTsHR2aItkkTRgmX32j8Yun+9dQDFsWf8bjxRRg7knWbPv0zw3I3Z1q6pXQWpvSw\\\\\\\/173tuqW9dHaE+x2lgyr1H3AbT4+c+KQ\\\\\\\/2QlzLOUH4StqQoxxUsUJyLrV0DF7xEjFwVO8N\\\\\\\/mEvl7A7YahgnpOBRCbIBcm52FAeBlK8nheqfx+elstXSRHsuwcVeQJhzRXpfGpADJwodqgSVY5ZkmvshMw5c2mN+dfq+p\\\\\\\/RFXtQUvbFuuuxmOaVeIXRdoO5ZBVK7LxKLcSbHwksuTC2lTcg2GpvUOrt6tdxZCUii\\\\\\\/bH4pdPBK42Salw+yFqs508WV4PKcNeA4zXPkTCxaPDbEybuAQ5IKFrVjwDKMF1WPqkef4nSVojbyfh2p9GeaOygDJbejzxtMquq0TDmw55eOuQcsIAvKMiv\\\\\\\/CJyXXVKvyKLVOCuz1hvehInXCXfMyPwEdvkQ0uTDtBwKewGaWEXzoev1iwtframxNMTK+R22W01HIMyxoU\\\\\\\/cDYtlSPdwzjBgvD8WVbo7RYqhYNHzS3Y+3AOu8BjrehBDmASQ8q5VqTuOwww4o3caBX\\\\\\\/ikWtZX8+QMWAlun1s\\\\\\\/iYbp04zzkwqOdfoS6V5IqEMr0pkSvI2h79GnuqfTVOKCaMxKOdbRq46XN7awdv3t1Oc+NeI+CTCT4rCKQp95ThMkJ4w5F9eaiROLaJxETbk2nkzYAB2NJGnMHWPjyeQfo6\\\\\\\/hRYMxfbITKg7r3qXlfiaASH2hWxOOn06TgPcDutu8ZMfclOLl030AI6beWjJHGecVV42qmMRwjy6RAX55i7UOoXo+wWXXy4L1fj7fNvhJ0nSoErr39B2nR\\\\\\\/bFxIBhHF5+FRalGhmsTwWwTs2Hq0s0NjDu8HJ0grFKoQTyMwoWcVRZuEhxfVNprIAQeg2wv9h1ODdfpjymI3RFnUVXUFy2pNvYX6V9t5aR0e+AYvggl8L3iF3aSrRN\\\\\\\/pv4gh6oClQJo4t\\\\\\\/j2iAqupZCfaBIi7ikAyfxBoQ2EW1m3NSVjGNh0RyjCEqdXBdR9p+VVcGGTvJmqwYNqzkWxmiqJGzi8qYE\\\\\\\/Lxv2tkGShyCvX7+4qy\\\\\\\/uN\\\\\\\/bgfT3PytBlAGBc9PrEpNA45CBK7Nmv9tOyq9n5rwDWJ5kmmTTls5MnGsBgFs74jIMuT6i5s3Pt+6HL6QBqa0\\\\\\\/dMdQt5A2\\\\\\\/CVQudoNzMVqo42Wjg1zRAjYk6dI5KlVNoSh5tXZ\\\\\\\/\\\\\\\/yX1p7ajF9ncOLJBbcIqGMBvZZb4Gou+Mq37TPmq7BCF0EZhzpJ24\\\\\\\/EWa4m6\\\\\\\/F71uTblja3Dg3AIW\\\\\\\/rH3+nEQqOca9NLj+A+J5NhMcczaZtZwzxpgsV0+jthDdOwksP6xzUm6tI\\\\\\\/+uA3gHqQ28VlAdxVmDbO2dcmWnmfphWmTPklcUjuyb+TpPuP1kKDPBG90b7hsxuhxOsEoLkwxH4hsHxKlMaToXxyQ8cxFrM0SmjCHXzO4C9Y3VUZe3JpTQiuNxPL4Vl0g3pQRISyC4WjtzAU6aHdJoVJJmRq8TAqGX2hO9Q9FuGNGA\\\\\\\/85ddyO9XkeWLtxvIOOva23rf3J52\\\\\\\/FegyXR\\\\\\\/HAp3lBbEIkyGz0c+DO9wLRtQKKbujYxg5CCXLgl9fN7jS7ZQtdUxXnjev0nhpzXQjorDYVBQ+UfxvVPrTfLyY4KPjFMKzetnSUhgxJLjaMQMU87dhvPPHIhkNIK1yKP1ZKEXNWpL8qRbmyq7jjOghuo8h+HQaz4BBwPdjmKDH\\\\\\\/A9+j6Q8xKGKU9soWtrEQe1ANHr2R1BWKcEZkpaHOEHGDw+GcGwtDn1f8RdVSlOPnfL+JHvy0UYDqR6J0D84\\\\\\\/m05j1WXTScMrSAxIfvJVlRZbbAN48lWbT6AfBjmK0aQ+W3JQq3\\\\\\\/86c\\\\\\\/89XK1yum\\\\\\\/136Vz8nAEP8R71IBxLbXGaydhq4yahViDG8tNqrvMSOZL+hD16U\\\\\\\/7KzJzWUyPfkMInGwQWm9MbwFm8i91gA\\\\\\\/ZSa9V5WQbEEpbgIj5NWxfCNaZn7deT+hDjvfowZLFWliz81xvy\\\\\\\/HH+KpV6WdSSHvKQEdQqUkhcBJPYmhxcs+UNSmDpQpTMCeT5PDBINZ2W1WHXQxIFM5\\\\\\\/SaI729+Vkp0mRdHPlZem5rnE\\\\\\\/N9nw2MUpURLVatZWSof266rkRERouUsS79g+uHk104z08O874saSmsFL1B2zCwhtrTKkGW9tlw9BCD1RUArdp0egAfHSgT3F5ydwF+W40ELZTar3jfruh38wslTgt5Zt3dPdKEZVXi4Iut5\\\\\\\/+U9ZYcdq2zqHZkGfV84c7RWG68Oevji16sEBpmPOKiQ8E5RCNoZByqu8M3k7Q4pgoa3pGDngoIib+fpncERzX1UR12nXikLddUjB7yTnZmk8lPrFRe+quKWQzAiLznbQTeUxA2A0Q4QMrJzxIHwjwrXgqotpB6dO47WjQDudKItZ7CxhZtBr5X5HiSnkB1OCSVIfmdsiL7vQivTwjdtN2QCLQ+m+wQKLOfTJILI6SreVCEHe96\\\\\\\/WYviFB\\\\\\\/9H23kICXidTH1ykM4xTQ7QejepgQRYKfDKeag+ZTsUIXpe5CWQEPdUD8lryxx2xWau1P9Lb83BsdWLGcYcYvOC5N2MFOCxRoyO8\\\\\\\/QVxPnSS2ZIBr0BKiCQmSZMiDVxCE53Gdv4jy035A+4qVGru5k8+MU7pBPkIrrSgpEuvJUps7ZUk1ufdEPQG4wMJpB+90Cu\\\\\\\/jZeH+vxfNwmIBIeISiePzWlByD3ngCAqBn4ITrg6AfL6xvL3rz7PswvmKlJ6MPZg\\\\\\\/drZse\\\\\\\/VEwr+MPf8haKr24miKlrjA\\\\\\\/D4NZ7du56zROosWiLTxbPnEXOrfYH\\\\\\\/zo9YOwAa12MAmHbto7azB+CnKBisuv97+gSOBhCDVjEh9YefIxhx\\\\\\\/ESZ6XSlSRpNjafVSUKlVRSwA2WhZsL0\\\\\\\/k1OqFu1BKyTswYYhmKSnQ2FzbMF0xC7DKpPjI\\\\\\\/fsRedn\\\\\\\/n4UXRobGpN+EzF0Lt+Z9z4pfG+ZVp5hx891oG0rAQcOrw96PuhuT8CtSnA0KRZdUVvQp+Z90vdTqXdBAozU0BRoCZoK65aToFzv9yhOILia3DBaW4KoKDoSvpt8XFblj3lX+Ws1ygL+bdjPj0n0jm7+jdW98f1ZYjHn+AMyOC4wRrfpWInlb+yIDClZc03QYQK8IZ845fpu7pi4EAZtiVC8H5\\\\\\\/e2AhO5iNQEKoBkiHvzDuOkhqzECXVUDMeP9SxFK8oU+yyLYz8MOl05NrF\\\\\\\/uPu9p2e87lSFTLi2Y30bFtfaJQl4bA8Z9OydAkTpuRrj1JUgxhpCT2ZmdmVaxBuQFZ1FaXztYpAn95Bp6I3PJMiMR0lCFlMBS4Hatrs+xLS4ly0QpiTXiN2so+933HyNs+jfrGc3xnedycSHFh4us15pCvn4+eHd4egm4SUH\\\\\\\/wEfWk4FUtwf1F9qDZfwApJ7JRi0\\\\\\\/KCyfV6f4smlEw2yEku44Vj\\\\\\\/pU7Ib5kstXn9YJpzA2Xr+nj4oxe1ZaTpd\\\\\\\/u9Gw5Jv45tuKxHq3dRSByc1fv7bd5+UI8teeGIyIePMzQju4feDbwghaQLILwqccPr\\\\\\\/MKVINpXUA25muJhvoMZxO\\\\\\\/C5JA2\\\\\\\/W+wZHgsBwSH0YUWmHhysRH\\\\\\\/7Qfl067DxCxwCvJPdxuMEjnbPD6+hXHwn3i3ZM5ByxAoTxXdPv7KS\\\\\\\/KTQlRrhRCUANvVc0YLr6Dm\\\\\\\/clel91MUy92Apskkqw55jMBwlWDkd2rCwiQcmXi25vfX0yFqy2P0f8Tw7O7vQ3UX4iwfHsN9Vykr7kVf+9SJquRvr57wsmPyXyQDIPOwjWWgBGm0rTXmX9DQ+XYEwedm4USLquCBlwLw4MSpJM+1Bp6zM6EVH7VmbyExqxmZRsKhXjic7Xfj1w+qN+jSUandaKLFbTZaiENZdCLKGa5i3Khc5\\\\\\\/umT\\\\\\\/0hA5dYReOOg8OaMqaxFsCiRvcduMhZMss3jfIN0xO74cMp2jl7RSgewSCUGEiIEvHhUMzkEDGWzV48FDLPf1q0wZjXEYtH\\\\\\\/KaXCmWhInpROkgIPOpMy7K+a63vcvzLCeroIkW050HZpmk\\\\\\\/LXJbpUeVA+2R881dFV0wawH4bOXFPr8aqnSNMxfxtivXSIABQLhuT9ygk7x35qfpSjYABGLfCC4PylISuaBbg52D9K9HXbVIssq5meUxb8QLdxjt441\\\\\\\/Cjfkk4hs4+Km860cQpnOmFlPedRWDSI+fLXg+tth9yElizgkIOS0mJ0f695G5VQxfc0O+NsCBq4EMsakKnbQrno+YRXrqZgLdUtRWwR2KFZgRWXNJnr6k3DPmusgSpzYau7NDJaXNWRn7ZdSRguTDIoFoHWziVcyHfZiKWqklVyBT\\\\\\\/7\\\\\\\/F1CQSmoar9dyucWATRFH581kObESD1izyZjL843URvqFVPBoN3whH+5sSjGjOtD9Zr9ylVeNKHPRsQRBdHfiVOj6rFRDvbCtHSerNq58MBOmE9qAPKgL1QBiphNbzI1vuofp0\\\\\\\/tcRIIWXUIQcT+dfu0TB7N9n+w9\\\\\\\/\\\\\\\/4TEwN2spMPHFsUtP1xktWfEmjBTd\\\\\\\/SKyOxgMwJXpC7UedIqyWcRIJgh1koeqyzw49pwzjkYayNmzDU9e86CUu1ipm4z9POYkQyTS6idhrlBeJNpFD2JmdRUzZwsnnL54TVioxxlyJkk6Jvz\\\\\\\/NUQtabatjqkrfBYEsUFtPpe8TCD06KHfzpuYk0H0gD9MtRs0M7SaOOfw5LPR4Ixk+6g96\\\\\\\/3iUsobFGleqPTHq0wMQOFdYRQ+51hdppq+Wnj7l2difMlb4YPIWZkVTuYAET8z6pGZS0\\\\\\\/B8MHTvLT7yg87rgTycQ94Ec\\\\\\\/DFrUBonCLPEf3On1DR1U\\\\\\\/GrD+rnzz8CkBR2CeziVoLOcQh1dVg3Z8YtVWyzLKmgoOJ01Nyl\\\\\\\/oWmowzu7A66HKymvYtSp8yj02EAR\\\\\\\/mEZDUzCF+MV2wx7db5uKcyIyZ3Ko0p2vQjDEcimk3T9AxGrgGZJnrjviWbkMhcYF+21yqPLt1uJKphde1Tp0IQulK5FgoYTehOKox\\\\\\\/Y1UtOzJigMyn0OnYpO\\\\\\\/urCx6L5QivheurFw3\\\\\\\/\\\\\\\/S6x6KrTQyUAFXxuIh1jyMHg9b5GDi2Cp83pMgdslnhg+zPw6KFJoFftfk6kqJXAV056+1UcgzZjHdZLtQPZIOofLiRTGv1SrQvWRz+Ulotb0Fw+dcdSqzmkyH\\\\\\\/IfKqdKZG4ZsJc8lqRUP8ic8PzSOkueOdFG\\\\\\\/NiG6bdJJRbG7dp3Mj+hagjD\\\\\\\/LBvN3x6aWEs3a2Nd+DEhCLNa0\\\\\\\/wEnUb9s82fgq0VxF\\\\\\\/\\\\\\\/nlsDm0PC8J\\\\\\\/P0CGJXga4xFzLX13aqpogq1mA7SyYdatM3mOcOJHPFnUw4mI8iMhbrbpDk1sMhgVmcXDtL67hp1KtmZN\\\\\\\/Lx6Yj6bQ0hTnGn5aIGkAWa4LvoLaj15n1u1aVMdZHRVdFW6H\\\\\\\/Y1d2UadTwPivPcmC23U3yASW91IwQ9DgMrDpjCFQs+i9ZLlFRR9U\\\\\\\/RldQSU+OgU44FqKc\\\\\\\/GCNVLpSLna95nJ35DJlzWFXOjPaOzKOtYcOhPoUkfvJeHlwneXn4HC5B+YF\\\\\\\/E\\\\\\\/jtBwScHtMkl5vHz9c8hUqs5keB3wwvJ3CTxcLvYkxCoD8pbQdDxMTryTftm40paLJFg7itV3N3ZE\\\\\\\/wzlXK8F59PurXpvxgAFhRyj9qomHOhq7k65AIGAnSmJchWJRYn2Sq22mrsAlGVTKTYHlAa+DgNbwIH2P3\\\\\\\/YgcEaa2bb9rn4Kr6d7NBN2qjhD1A6GCWpmkeAWu7nDNCF3XAu9i4AIBHnN4Xq4RGQ5liCKBBe+eeQWeEyt+PBomgqZWSNtW+RrEcTGdsdbFIsWYUN1bIkW1pv4Qc9vTNHmvmetb1XqJxZf94zkO8UgV8Ufn\\\\\\\/rCJIzJnBhsmLyj32dL3rzgWYoWC\\\\\\\/KDBaiNLkVZOuDBww1CvirmAtiQDrOFMVEYoo1469sODi0V3AwXR0pa\\\\\\\/Hf4RbuDBSSeRUA8TNGn67yhv639J23qqFlHNwFMTw04h7gAw1drWJPsVblACA\\\\\\\/lHkvn91N5ZUPMLa9t2uU3eshbbRIfdSXRDtTCbnZmjSHbZdkppU1doV04W2ZljHFcp7W2XKfb1Zvr8oTuvqZhcjoPgexQCXBh18qdNfS+lpFOofQkxWzhsowyN30ctZstWE568SAAXr+MCSghOKM9zfR9EkxkDqxRgLP4REgCgWFTcN3\\\\\\\/3XgRRTLTBLxbIvd9KGsNnB9J88wYn\\\\\\\/w768epvAoeGmIzDiFo72satSvBYX44t9h0gnTX6xIuvQkfeEASsrLF\\\\\\\/n5B1wigxqKV63XUtYOlBse\\\\\\\/9oKkKuKl2MEdHq9ekvVHhFXv+vi\\\\\\\/s4FULwXTteEg+EnJp70l+pgiTjjrieizBEPHlFiMnsLLxgiBsTDnk81mQ7d4VJbAFvYP2EzIiyXefXew53ACW\\\\\\\/vTbV+b7HFDhoFrCieVEZVHSoOMjmqUY9RGKDSvOhaSrR3Opn0czBREiPxTSou+h9pawJNGvu9EirYq0zwcCZMk7l+UZdgaK8u1WjhKpwylCZ3Vv74CuZSVTE5Llo2JanJ\\\\\\\/\\\\\\\/yGfxMSfF71XUd5+Nu3hYYaWxPaqn66JZPlr830sbY2G6ZoFzIR9HAOECZ7l8d8YfS55\\\\\\\/7TVRV9oWcd94DHc82iHX2NlY2Xjvtm8R0pyiw1aSED26baL+1jBEFxmrB1x95zGSG7XQAP5uiwdYDXs6a3l0E2v5nwwD57yyYPivMFnSu\\\\\\\/puQ71Rz2wOtijRwcAd1XVTm7i3fXoEpjUZEppWQjfGFCKXs78BRVJ3E1nX8XiE1am2gnf3jyDwLTD\\\\\\\/W4aOaZ8srfOCgFeMjPDA3NU1DjqbIbw90IyQy0E+fLl6HjdgA6+f9IyDv0Bem\\\\\\\/KEuh55OlHshdPEO0SHcOfnwZPKsNxfFs1TrDkey7nBS0WWjKvZgbHH74wBuY8RcqmJZVCWcyO7S6EbgpZS+xRk3OqFVfnUGTO7q\\\\\\\/QK7s9BZUp2ErdmyXSOhqOW08vxwbM5ibrDOZQc\\\\\\\/RXUaLEbHJoDPCxrweMQouCto3Z+Zjs8IH1uAkoE256xO+h55Ra9Ulje18ypvjY7wh+hC3yegsUYIC0etPcKDWBnMfZ4I5hWOsbeiQuv\\\\\\\/fa8sGmK7piIApde+u3apKp9O8+e+BLUei4Xv1Z53uLliEoqUdJaUP1vZBcKYk1F0vIeJNwPr+gME4XcNmb7gaVhMhJJ2PUBvIk3VkB9MqZg8B9RA1z7EAcfD84Iv5kylK4\\\\\\\/NP7GHoTQty4ETBWz8qgl4SGLAYE1IQSLe3sUMYodkxKPOXyYutwPc8lupSjBKpu880Gx5kGTWPw1\\\\\\\/T1P5LkM2yZgyJkdNv7ET\\\\\\\/6ZqB8eHofTsO9oPFj+W3SSFXsWsmu+oloaqoiVNHE0K1Uu8H2q7gi7DivE7tIl7fBcRxDL6qaoMEM27awJmDpsRcFfnLvMYvN9Pi+AJCTNGJiAQfepORzg59tJ6kgJi05JutcO\\\\\\\/CIxdWa+\\\\\\\/bfPYbEmqlCThAnDHJk\\\\\\\/JJ\\\\\\\/j+9NBlGka+uU7peRBIVT4v6gCE1ch7KaI5Pa79Px\\\\\\\/vdiv9PzLaR024AMF9oiFYydF4X9agrVNOhk8QM0RaDiSiGT5TXcTsNusuxT0GpsOiwgalUnWZNGhxKpiom6SwXB6EAheqpxSPaoIBBzQUwTCMg6EMokhulRhJiS5ZUcoS6iIDhMAmMd0\\\\\\\/v3lM9AN4+aV\\\\\\\/la9mYHtnl\\\\\\\/g12x1b95v0Odrgsc7u3Mdz+2Od68M+9SMY6zWDqpBOVMCaCJ4q7BibUlp66jBjbTWhRL6nLrULFLIukKtpQ+kEv3R7VkjHYdRbHUw4Is1L1Ku9AsBgPpGpnhj+8\\\\\\\/Uzt5HkZuJ851B+G9pyug4I52T1nZvd7SmB+XZGVDk7d259FyKY4kJRaHwmHNNicwXDxX+Swgo0bUtHEUZijyyYx7y44PWhmPaeIM6ue0q683JjcZmDL\\\\\\\/y4keO3LVIRGV6N0wxLSRbYygqeDl4QRGeSIqyKRFXkyIcvNbYzilRlsHdenWruo8ZrMLScgtkEFo7AimraWejgEQFmYHAk22yToV+fxuW316DOmW5lMdByuPUhkED8GB1nyr08DV45QjtmiT3u1WG7l4KghzrX4DWbLjwokmyQbxeXTn\\\\\\\/nPqa+Df9fOjT+wQO4A4iHLeQXWYpoBEZtqwrjHtD4VFrdfxYlsWTxih\\\\\\\/INiYjA\\\\\\\/WvBxMTjzD+JdXjORE7dQp\\\\\\\/UlFSY8HYLQttjQ\\\\\\\/VzJPZv3dQdxQUba8o3JSmZ1uG3HuQEOngw6+XOjT+Zrf2MTFzyrleLHETCJwAiANwVLmjNJgQo\\\\\\\/GG4zceyoesP7o3FREBmJ0rgf6cOzoT3dEJPZoUmM3oCzVkFh39ALXCTGlP7APwsG0r5Jgv34mgfeu8EDOf\\\\\\\/z6aHLvr76MrSQdrBgeCtxHfkoVRMIC34qBoDJvfRVZLLE8HbceEDJxkvVJwRV6zTqNaMNm1t+CrayRhTRzhSMJiJMSvAgrv8LKjZUXN2Y6Z3pDaYe7yafZtOHA5CtUWEY93Zlkbe2AlGuQMYHdH5syHgH3ybD0bwIFOLHxnlxGT+ju9yzIgse7H\\\\\\\/ROi+kvCi9yiLrqtO2q1jSqaInFAlgX2dxuMi\\\\\\\/6dj+9wmC96qOA+V3KiudNl8rUBTlNXmHatCPrQiymBtUYDy8rO23QABYTc\\\\\\\/kedanUYpP1mGJLQgyROShVzzhogxM+AzQaLdF8Lqbq3t1J3KMCybzTjoIOD813Tmc\\\\\\\/XHtExjI\\\\\\\/E43AXveB2r95pFLaaFR4InKjv8Y36JKPVjQG8C7oDeHVctK8DlJ1gP9nZz531jmgVUDWeIRMcBHHjDklPUmw9THv0kvJVltOUxc1ecxBVmFhCgnbkuXBt+v0\\\\\\\/Px3BiS2ZbP1iNEyQUC2SZWyqMmKb74F\\\\\\\/uzchuqJdXGiIaNNbo386vyJEu4LqYc8VBajlRY9iZAXfpPd4CQyIHcTipBAamOfDAkTmh+mCF+QeoaKgkXRxnaH2\\\\\\\/sz13AHpu4Za9YHrB9DhnGEJ9hLBufakPmiqy9hQ7sOsKbebBU87SQ0cf62iTXZmgpvKlakKkO9C1nHoSmCt7wNN4MR9I+XmkET2RIco1XpbUkDswAd4qtYl\\\\\\\/QqaXqc0C9WX\\\\\\\/FxleIMLkoY0fhjBnZFLohwBl+L2H8OfDzMXVZozT5lDAFcb+3tr8kDtPxZrRrckEcB+mNvFk0vtNUDfXs5RQgwwc7RCde+SDOvaqnVfqeOA6v9Qp9CxUtmlyn8eeffSytbecRSJYg0vgiORaAWxAt2vXgLKEOVLR3OO82i9XrCzb7mkd\\\\\\\/RswcLk62mnjmMOofEKxLAHbswl0IK+5unzCekMA2RZrBbRbEagfCtP\\\\\\\/bx1XrcJsQDoHDHkohlprn4qPzEfuwwRozFy6EZLwGb5iZzL3LcB\\\\\\\/J3C1IF\\\\\\\/E1gzwC30X8etDH03L8XWbOhw6LEPOxr77H4vsZwAwJ\\\\\\\/eK5RoZYUtveBfy44GvovGFsxtADIapxOfWNaIPgQyGc9eHYJTztt6X+ESy0mMTsa73owKBYwsYU\\\\\\\/tQkhaaDswQX8h3Xc6JOiTeXvCVZ6gMU3EeVGu5yD4u4kSaAR4dvobspa9runQR97mcLN+Cr\\\\\\\/P1Go9Vhp4QJ9f8p4a0gA2FHHcV9Lrm6DoTeCDqgnKLhF+7YRL6MhfA0oVtDysd4MCdc4fDDHyV+a0E88XYirR1jqU3LT0KdWPLmI\\\\\\\/swx5mqC5Q5o\\\\\\\/aheK9KfCTzdIlh\\\\\\\/613tOLwfOU8UChMaYBHOmRboQZFvIAZWaSOnGaSPBj2hkdf7Vmjr87+eCslFgF0V26t3VG7i2T3iJdL\\\\\\\/fXjHEfZrAgebklxSoKTRREE5tQSvGs7puZZQjtpyKcehhiW4KWQ+1SyW34AnIqD2ev56534tqazgTmjxqIF9cg0WfA55bT94Wqhe2C1q1+TQXSoSBxtlpadRGQq+7\\\\\\\/+P5dOwKXTinFX5RyL3J8mELFWVwdvbSUoWtm5LIEAH3NgLSu2PDBRedmb1TGjeNz+AFTOpfkil9iLMAkda4vZ1JVV\\\\\\\/d892ydQa4XkwWbcL41vv+s\\\\\\\/ffe4HotAE5teZ7i\\\\\\\/AX5Iyiv06O1\\\\\\\/QjuH7T4VEsHrQ6QP5OOZl1384fEqgPET1Bl7JLcOlPPzEFXZOPdPPD5lbHmEKbVagGTLrSZZbvSSdrVKozH5dCTUhs5UTqROplreSiMnfbdHLofQrEFLZZVgi8+TKARAMSF\\\\\\\/o\\\\\\\/d5x1DA+ANWxbqFwIsMjH0Z98uuR6GOsOcEhQwugIAk1qkoDh4SQnHX01dDWXsfMw++vwJ0EY1IrH9yt8toCGCAS7Nk2J0VpeTmZFRbk1Qj8+xiqnv+RRgUeNTHGXjNTT0e8bZ8n5fErAzJVGvjS\\\\\\\/WSdaCQ0UbeS7nYCxCha9TNytPYYGtvtmyo7Ja4w9+B5mhijexSyH0OEndc7Dl9n6kUaortbPEhsbHtKjEoAgHIGmLwUmjlLMy4Gsa\\\\\\\/RfaX2G77pLlc84f8TQ5TixvBkS4AyML3bhq3JHW2kyfplFIZn+d2IrEIlQ+krLhYueJmWHXZG7XKzI9pYwF0vzBtMyn8dXwnfo6eqcppnq96VY7QmfekM+bX8dXT6Nv7UV2vF3Iux2VzARrRv130uFp\\\\\\\/k8QLW7kIcAT4Y0h\\\\\\\/2T\\\\\\\/Mr24LrpfvlekYGli73JQEh3E0TAC5VEPFvH8xomMNbq93Zj0SakTcZzgycDAvZW82g0K+WpAp3Y6mwPGH\\\\\\\/KvOSDp2QOPCMfu+RW6Tijf++PxgZ2FCV7EcVyOeSNVTZDnoKzzh4RZfe4uv3fCGw80wbO1Dk8nKce6uwA+0CZ9zM9iN719xv2K5hEvD0Yh0QHuetNJ23AcXWaTyn6Bqs\\\\\\\/k6IrtsnDpz\\\\\\\/aemvFGbwMUdI00hT8emIt5CysqYTZr55d4qe1pndUyAkz3QA3u0lK05hMw28ki4gWx6zvve3NWSxO2VpihOQKCPLihohpdf2ZpeQX7iXbVUelsQCn0ZBGFcoO1HtwxEfu+Ns0XeFqYuEXxpNjeJbMxAkA+1ZBuJBBYFZ1yFCpecy\\\\\\\/de9bZe+asw14k7xHupVNoWwe9KFE\\\\\\\/otyQ2rBh7NV5CqnfExyjQ3hG6h2TXJ4nuM2Wth7D0WQLQJlCz+eNFWSfwGoK\\\\\\\/gxDqm6hfv2M0D5kFIYfU32EdfdYzUuHc37OPW6gTn\\\\\\\/8naqqnWWg96zhvCvbgmOoTrdDguvayoRfQyYX1DNnxUwU+8Jehy6Qba8bkKx\\\\\\\/zDvbYd+SpwX+MH67oezExMWafqouL3K3+IlnFUdJqSNz2\\\\\\\/KZ\\\\\\\/gIYyzv+ojcRGP2886LjgWw7\\\\\\\/DpLjjEsNR0\\\\\\\/NpUU\\\\\\\/MmS+l+oO9392hiluO0jR2EDPmLllFEAphr0\\\\\\\/GPL1oGqjDa6bzQdMg4Y5Gg7o3vawVkmbWfvUU3xz5Kd6rzPNs39y+G0MkAShgnU4CXYplq2TlqoieLs4D1wBr8TET+FO5cLh4faPCpGVjITIaVVsscYJ+BRgv2+2NKT4mZBGv17uYj2Ri6LBzpEc5rLpq9qjpBjdk0Xz7qG6L47th\\\\\\\/7u9Xw8goSUCd8QzF7VbJha0ctsgS5GJ2TpwqWSQqKzQpSGFNCoPSuRFbwLQ7wmBVl\\\\\\\/fdTq\\\\\\\/D8Tme1VLyjzaD171VpPjLL8nwoNi8R02mpEJci1I7kghkUbV25t7BCZYaqoVEX6x+XAArQsJ\\\\\\\/AwyvSlBGplf7zmaVVTyY1o0DEzY3KXHtT\\\\\\\/4HI+sJGswB9CGUG7pRYMPOwkHzQW22Uhy6bR+XhqSvoKCbEo3z7oaqv6rFb0fdKOxUa0uwHuDKp72SkfSmBwUaUck\\\\\\\/xnf419cYaEpUqlSPD\\\\\\\/3Z1IEOpXolabxrZrFz8x49DXXHJ8FsYHR0sqvNR4cqu3di31E69WuFbIxBmewd6EuvLA7444uUsXiKT9YjFUcobJCmpH3qEqSZjDztwN5dSUxYMkjcROOe8SIR4aCP1R2Uyc88aoWFuejAy2\\\\\\\/yLK8V+BWX5STPVQSYV5FEd526zJ7KPitz7xYVFpmYvwxhDLIj6Ab62glsWAWzyHDtfD2\\\\\\\/oaIXwhycC1MW5pYspPGKKAyt9hmCtbb31OaqV6Tu\\\\\\\/8oq7Im\\\\\\\/+wkzEhG6WPf8Ws\\\\\\\/yzlGJ4ke0BKWF3aOmiXx35YBcommsjP7\\\\\\\/0SYny6+kkjES73nBOnzp2GZiphf3++9bHi4joq9yZjWMgzoM2BDpY1XRVH\\\\\\\/8jb8yX+5gVxUD8WLEIEtR6isBVSwVFaMDRtj48EgTOkxpjsO9HSiAX7TWGAMFEOrBXKNM1QSTcUArCx4splxtP1ITa202LyCNa7sthnRyoCGKI\\\\\\\/49kaFyHDPdcGV53\\\\\\\/ItEUgrmj6jyHM7IqucTtjH0hMcOH52BdSbxt\\\\\\\/ClQZb6rDXsl7mtHB7o0qxE5iLoiQbBg9QYGDaPRD2x1kW68A2IeSlT1bRNMHwVres2A1vg58qbmg4MVDyrwa+VgCh+XlzHgZf+BCtE4BUtlnXPGAr4we71azClW0IMagrzor6gtwJNmSPPl+fHmE5IYRFyhIwroqTVAUnHLKin527NzZNPwXvfK97eVhI9dzhasm7WiwNjFL05vNSwxWgJcgxihRp4UMlGaLRQ1P7uOuTKPR21JRsWHAzV\\\\\\\/SDy9rqz38GbMM3Ukz\\\\\\\/teqIMzNABOx7yhfGPke1p2or5iAUbnmm6pxP7oPtcUaknFgo8nyyAxmjjErN+ftKksQb7S6vwbxzYGaN1DgJlbfDKU5h8F+vd0MgxAzJ6+1lpVzn7RVZZ96eEHRAqZrA5sfQTaabwe0C7vVIeJfdweP\\\\\\\/7bt9Yv8fDAYHMbuIz66m2R4hy4Hz5pTQZQLYs\\\\\\\/YJQo29U7vwasnwLEUvBXTBpve95HRKeRz1iRBkgikjZKY4thFGTfiR2AOSSzME9h09Ff8Ks5\\\\\\\/AhZTnT5chKUt7bLGiiTrWttPgvbYKkewf71pNhsXyDzcNGKpRP+QisecS15le65LniEGs8WGUQdL+p7tNb+RwIjc55Sz6mOr9iBnUAtOR\\\\\\\/BygmWD0ZC3+fYknlE\\\\\\\/ZPK9S+RHPQjXa9fe2wTnPtuJjtShVw8QbTZgfcCDilcNE0KYEbaOtxwBIfu4KKp6W1G1mA42SqgPMZolsbD0byi8IXMCo4rAoE5TUVXHK0OBZ\\\\\\\/SfSPkDVD+FT4WlsnIRFj84XIOT+Nfk\\\\\\\/ChadwLGrX3OuVJjnrm4rvUiBbXsuyQL3SuJQEw8nweRmlm6jN22df4idHhyjPDSYqvbpUCBGZdm0jXPzl3Nent4jxRPCFwTUBpRFQGfUnBxbGLky8NswaKp05VEoxMgLLldSDSag02Kgqzj2+L6QmCt4Jw60BOwVjqzAkS88bptVWts36pgOtsVgN9ZHbsUqIeFZCXYaSoibHJFxSRlIzCcMP0cF2DatOQtcgN6VIw6m8HY8XHb76RE5JcsAZNytxlmOfX1T7qjiNlfF\\\\\\\/xpmYMdrV0Q4CXaQESi+iXo871gU0uokSj6UmU3qE6UnqIdCcowIsW994prT7DBl9ACqbq73aS33tXUr3n9j1z2iqsvNCpf1P+oEmgMvnKalZ7NJRbp7fRXTjInaUAnZIwS\\\\\\\/gOKnZ+BlqmbnfazdnGBfZGyn2PVCZsdv1GJWlUyfJb5Y5tFtWAzmPx0gUz411tpDfRNUfWfmxtbbQ4kPUNo+nJkwPZyJ7K1JlhyAY3xVixGmEWoawFNkR6NGVaflEl+ESg1f8hDpf055DWJgwgwRbpNv+jyexF50xNGK+hg6rL506RkhChlyrgKnoQTSQtnPxSzQxtnfMokdJxkKES1008gWyFOsChonRDcUmIfNNB0UyybZY8atdlUg5Fr\\\\\\\/UAht4Qdf5UfKnNUiJzkcfaSclkdFkNtmF5DOlsKBCIg7fSqfSrxdZKjqYotJVPcb8A\\\\\\\/gDUqGD6RzMqhjz0j+6qVcBwfW957B\\\\\\\/7ie7bJsox+Hvp7xRiei5vbuMuPWJ1y84eBqxfT1j6HrPePOMm+UsKZmhQDYdZlY8XN\\\\\\\/JqW6EDc24LwSmtv5gTvrE2t+hFpLjsvrhXRGYiptD8HlmiZw9DjbVJ39vJcE9EtUMAsIINVjLrArKuX5TX4m2ohwdf9q\\\\\\\/nxRdHCxTGOvG9yO9Mnro4V+VEGTKVkbCfhN7FSQO\\\\\\\/DW2lhaNbbtU3u4\\\\\\\/BU8fAUo1Gnvac1HSRqxwgoJna0jI4ne9OgkcSGA05NIP6BKsmjj3EllIE6\\\\\\\/KbEfijxQ0OyOviPgaycIgRQqzHpJ9rGqMTKZ793YJCrHznNnW5umaaLvYD2J+G+6DzGDAuGtt0G1VHPlS3rX46nXP6oI67mkAk7D\\\\\\\/G7Rq23ZQZKhWUzQ4jGTkKKdHyRquIszlCF1JcstAYbmFfND7OynymOxtVswAbkjnHAcmxGVnNFoqV5XTeTtyQ1YoQySEZwjjh3mDTW0XhnO+wBYm97ebvnLPrjX4qGT\\\\\\\/rkfO2B4BKhudk11q0R1NjFKUmww7whRN8yTomlj0bKOy6u0h+9bXYgBbrL\\\\\\\/4Bd6RbbjCnM8Z6RlbEGSqflkLudl\\\\\\\/J1AEr40nzN\\\\\\\/0KqRLIKqdcu1TCLPlt\\\\\\\/AAAky8QQ0BxHObDx8rxX33wzzW2Vou4ABBXLRBc5Bhx8o8HIMLjb5ogRNRqASeBf4P1oeGPO34A77vQhyAApVMp9WQpxxCTtZEVAr5OIZwHUOm0BjdDzhRQuHW65rpNYfd5mM6gNfQhw5gxuMUBw93QtMHgY4XppBfEYf6rg0Mip31EBr35oPrZtuUz2Chb8v7FtUtvu2PSXIZMXaVkwvZgyGPpknQTbHo5dIZROogRZgGkmi9KayuRWT\\\\\\\/HJ2FqsXW8ctDZb\\\\\\\/niPLDvXqCX7Ua34vpD7up6bdYqxJb56MmsHl16I7E\\\\\\\/RsSqqElKCsqudivLpmaiF6wg0p5M7Wj\\\\\\\/tpbqo8ujRIOuRl324ljgqvauSu5PIQch75SrS7ul9TH\\\\\\\/y2LOT12y6Mc\\\\\\\/dq09JhW3pmLRUQ5tUiAmkLXFECf8QB+yiUZHRfEQeF1OCS0ZvBH4YwaXgBZ+5tM3p9vqm0U7rJfrLwsmAgWGqPCV96JGB0wuZLAg6f0t\\\\\\\/8+Rz73ORVRyDAXeqbkZqmQ3NhOIxsQLlerafPM9AZYzMgXRNK+mmjSw6Kc3czGzUkOAdZN9vvBaq51vLFyhclF9\\\\\\\/FYDDTAd71EkGd12KoI58yyLQugq6qkDj954FKWUZWKR+PwODt0ZKD2dbUTUU3\\\\\\\/q7lv2uzZfsJd6bwuoKyRko4DyGz\\\\\\\/ap+j5ZwjjW9xIt6gBUO25jcM2BgUtbZ303wDyM5yGRtdHQB9Gb1Vk6mJlf5Jx\\\\\\\/s1CxNxExYNQVhqjkPa4r6xDzYa8GI4+yaGU4f\\\\\\\/tbC60+H17PM4nV7jSgBF7CGqgY7n4UAnO7v2wBZbfHqGdc4hQUI015MmC+\\\\\\\/Emw3z1aZ04mF\\\\\\\/Z85pmXAqdY34T+OffqZHOoYXPfGT3iqWiot7sPMIIeGDNxySArmGhcMIFpUTS49IYr2JxlumWLEbBrjrJ9BD6r7bpmwgxI7zdG8utl0bhrn3s26dJpHoDt6IUdd7vSF1HkLx1o+tj6LoEGjA4HxpNnDnV0NwJbPmxb0NHbLdxPxrlZVtxT9kFOnKMXfGeLHu8JkjIN4meZjBaN8m7M2wgNWXWtcvn3aizVCMHCjHCiJvvbv5zJiVisFLYMIVc1XeunNBl3DvyNwz6TM1Eju7BLbGuA7N\\\\\\\/Gaftz0ElN6\\\\\\\/\\\\\\\/6dBNiLMm3Y+m+Cz8ub7Lv3\\\\\\\/E50CFbIEeaue445uEjlCCwE\\\\\\\/Pul30gbNyRbAKe\\\\\\\/Pvul9SyeaCi\\\\\\\/ao99um8qmx38f620mG4gaqj850pmpHogoI\\\\\\\/p0sApq3dc1lBn8N6QsWX4OznPkKzBLu0nI\\\\\\\/u4wAzhalrJgpJ5nevoGuj\\\\\\\/TEAmOWpQo4s40zlJLI6QBk36Pn4QTtklzBp730AZe76TUU9Z\\\\\\\/ABGY9Bn2mwUFebersiO2J7Aeei4YtWFPZDHR0ImSrFGdCbR1ybwFYnAgkMOgs47Nufrd7LRq81\\\\\\\/KRqrtvvbo84EtrFag\\\\\\\/pwTn0Y8gfV1OO\\\\\\\/oyfWilWZbgXQSCY0npv8xtzlwtOHAtUDrkUJtV3A46wckLfXa9+E\\\\\\\/bycJKHBbpXyw012mXeQYQWp61OyF1Tsf8yhM7B3flaXnrirD55yqC8ZL9dv4D0hzCMGdEkxb52xtWQtRk9evK0nxYEsQkKnjJMP41+E29V5WD9SfleOgOSHa31ckM5GL4ofZmcve4HkP2\\\\\\\/3lLz9EuH6\\\\\\\/AeWEh5SWmanBrzn9IdkpfdjFGvU4B67UQOYs4msRKSpKVWmNmTAeZSghr1qX+r8rimbBKNCGn07tZXt65KOWMzTQWFu6Opt0agLb1GSVi\\\\\\\/Hga\\\\\\\/vlmKksQViVhIW2DkI+TCGcwmM\\\\\\\/bFba\\\\\\\/EQFaZYupUmCW4ENt9hSzZakfNxqMDjqnajz9t2iMbCFcSsHEspwPrrFFWc9I1EZOiJhnBA2AHufYIojRbatWE4F2mmkxdZBuqWvk+soOZjlcSzZsxv2fQPGTVFta3npeGogsLb4aEbJIgt6am7QW5vEUyEEMW+LQdsV3Xsykt6VDHuCD6qUJ+AxxAMpV89M91VgRond9YezAeDEiutTbI+Qdzcf1D8IFCaV4hrJKh0CsDca\\\\\\\/4B\\\\\\\/AVpXuV37r30PFS3EhTcTq8nDwixzluYEOqzsXRCsbMB3vTKzNPGH0WEBjyZLJ5l0BHfFdwurofzruiQsvXrhVXHXZ1Xf9hscWxE+tGIlXpO3N9pMo0lCn4Undck1nqWskcbuuH2M9vRzuRVUaYkpKIC1GOpHCqIMmbDWbYn0MzvAz5BPI0mpXYXwJRmBvTWePOfWtRh3Zkec8X0cYSBCXUWLMNUdCOaKBN+rjEKXhIgBnoQS6wbAAe9uphzSrciDvXrm8QEE1l+BtDgOIPPGbd0Uhmp5GgLwP8uUdRPSCAMqgYs\\\\\\\/tI\\\\\\\/pEZbaROe\\\\\\\/AZiYZzXjuBZaGhpAK+ZM3InbVn\\\\\\\/LyTumymzVASH3J2mpHR\\\\\\\/DHT+7Pf+hxkd7uxpoUKNHwyItjlRenB1IDmz0Kvn+MKi9vm6SANmGD+51+oR9xqNsVM1gsPnPHQkx\\\\\\\/QEbi8FoitgQYoLajcrQUectKUmT+XXkI7CGB0y7XiOxKn46C4qXcAqTMW28Gv3wI0YyCTbKgHFefvSio0it4yrh9mdsI\\\\\\\/m\\\\\\\/Gkl6feCrV9veWAeGWm\\\\\\\/fu2RXPwcdhkNEpmsbESnQBFMMYVZkmzfI9aZ\\\\\\\/KXtkPXssTvALUQgOFDXE\\\\\\\/o\\\\\\\/sg1BK97W7zA5Z6qooHKKhMein0bKe\\\\\\\/UByux9Z0gQj9ZKFLOpsrrw98olh6BuZsGu1dqnx8JWFiyDMyyFEOAc8c3f0HlzApfO0UFdqVhKnUdX9Av58CS9J8F93Rs3m8u1MqrW2pLQErhUDZSZlpGGyfsHM2pLCAsy1KrbI7g6btylg8kn+el8j\\\\\\\/qnHlHX3JKGymv6oVxbJJNw+QULpN3URt8y\\\\\\\/adwRaIl7OinXZzMNn8AOAEVBNrSrRoImZjIb1FV1OTlRj1UhZkskJtqAe3RAf82f0P6ITPUS77e7PMF5EZ4985Yg0EcWzFIkIeJoI5TbmJ2CEBgFu8C\\\\\\\/nbu0\\\\\\\/VSo1bmPZDKiKGQp34kekGHaq8tHTv7c11eJg4J4yJSHTuYg==\\\u0022,\\\u0022iv\\\u0022:\\\u00220IzRsEmhB6MjZ+6madllPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d4a7ca9d18e6e0fe65c45a6486c3fabe708dabc3173aa614822875f7e2cae8fd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yisf\\\\\\\/p0Y2bUoOYqHnQoaeg3mnm2sHoV7kJxuusCupH8SUIiWf7wegeMIS7DG1v3\\\\\\\/UJMOC\\\\\\\/zN41lt8jxr8n\\\\\\\/qMhmp+qfnhoUR0maj2QJwD1xUKe8ZsBytKSB\\\\\\\/mtaCCIU94E+x0vcD5IoxFArGe1kKeFPb6YMAQYSLT6DwXCAOooiqejjH4O\\\\\\\/sdVulfuEtgz111\\\\\\\/Zf6XB0mS9cMBuY+\\\\\\\/19FiAQODHjnP\\\\\\\/eBkA8LxMT16H6VmgYMJgxrd6Mpy9AKP97BwNTKilTuNPdPJyJw1NtxYyzDmITqLikTW9zzuWfZ2HC3tPKfhyk7s4cQ+hNKxNex0gYA5fZzsWcwp8FlHNNLVNhHOKVAbz1VxKlDGJNu59lm\\\\\\\/541RNCN6kI5n9tqxdsZYtRBcoDU0Cz8Ec87I8Ffjco7MOfx30t8KihZ77Z2aeHZmhVk\\\\\\\/s+6ST3Iks1E2rbIB1hbdgo9Z\\\\\\\/YKeheL0SkNAWOZ4th9I1ujSn+cbdNPq6E7O93JVug4ZDlV0XadRk61YBdGjjEj4s4ZjxyLUI32MrphN1ntLjCwnpbZTn8UVB0pcECsmc6zCeJ+lzyQZyPt4zKycoz9ZEVb24dN+c2kp3ZjCDtekOXnj3TYGlneM6VhTrRodQymk\\\\\\\/ARnpFNJZ7v036+dHD7huqyw1PPy\\\\\\\/uOLotoNN4lNN7AXIOyyTrkRdki9ow5vtj1OFi\\\\\\\/FhM\\\\\\\/pKuW4NC9JHhEWSrw87H4YZJi7A2Qig4UqJUAR914WTbgJIWv6pFGQvK\\\\\\\/EGsU5PxCa+fIlCWNqCqFCOYK4PfePai2jsn9Yak1\\\\\\\/Wy2atzQpMpmqSAL7Oad8h1Fm6+BiaeS94q5\\\\\\\/w5W0DddWQ20Vy1JGmmrs2jwuJDdWVlWF2FFgg41V1jlm+7ebvaFuZiN7oC2B\\\\\\\/gW5CC8vRZ3U8CZa5nMnQgSxKWSZES+WQ3kNbV3swM7BmA9Pv9mIYTx+yd5zegYJtk5D\\\\\\\/8QspfCesQcqy+gV7hucK\\\\\\\/\\\\\\\/8xDV\\\\\\\/nibsPzH3FRtMAB29V2jZIa87RlYB473zpgrpFuLIAMKO3miKj\\\\\\\/TqOeB7DWWlcbSz0EDqR9ZskpJJulllQhXeV\\\\\\\/kWpaYNSGZEnpvF93\\\\\\\/8+KcIr1wH6QE5ebqgCEoi2q1IVnLugdVx00oeRQftvpPidkMStk8EHPQ8gYDjfZtEr7nVqKWsgQBFUP3q30qlsaDCbeCFVSCv4OdJwSclTkD4ZyNvYBDhj6NywZ5sKAfh3z0XXlgzmcpYdRgCysem3UmchMKttGNYGTiD8VAJ1XfN4ak\\\\\\\/gpMdY3uj0C9iOMBjkhjLcUOMTpLbgBLqqBo64qs7CRMk5OsBJwKI\\\\\\\/c5kRwXhnV6+AT66G2\\\\\\\/bD1ktmQPO+RYJ2hfbSAopIeJGkXbHM0c\\\\\\\/+kSjBstLJAvu4jHGRnWV8Hr4k3YKptBLVsvqqVTNrQWn\\\\\\\/sr5sWok+iRWNmcO5kxngSPYvkXzzhhz2rn\\\\\\\/5NgT4fpMn+M4HJgTvTMahi6+JMJvpZyxG9W7cYmAMflZamZnOc8Z6qqBSoPZJPPTAvXpfwaJA3ymcUGfsD4AzUlJB8iBX+vAzMWmT5RujV74RBtkp6c+TDzOQvCdUZLHEhfp6lkP8nXSjA+1hFFeb0dJMTzWojT6VmNG4auoIkGiXpEMVgy80CmHDibkshGtMS5b3G5nTr6Mh2F\\\\\\\/1+\\\\\\\/x6pVOLFTYACQe\\\\\\\/yArBTmqRbaHQ9N0Mqdi5i7ypp6t9ZzRbPKkiPjqsF4GvuDY54xo3O1LWrbnXFWWnrPYoAY2C8VPA+XavV6E3MosJl0MkcJdUf769uOPoeZLTrdLAD4gE6AhYMpWch5yN066EOTScAMbU6FqmLboX1CZLDfbwDVsVYvZzNZ6BAWSHGSL1NVLpJm8tRQ9tA8+TnzNuuq7WCnaqcxVazf1rrRi9zvyX0011z5OlUY19QrIUj8HPEuJKoAZfjcaPcXAypcS0S6bJZbqyrhk4Stg4Nulvhw9AcpKvGcExH5Sgngj08g8EGPnr2452namy8om\\\\\\\/1rt4ZAg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MupO2vCMnTANqeTQ\\\\\\\/\\\\\\\/lD9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ec8bf6e059c7971487473434ce0b21216e68f284115488cd905b7961db5aba66\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491516 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3973" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PD2wHGgL2bSsnLc_EJCvr5AtXB5xqEoMkwygimq1n60" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3973" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2jdYs6060gOuKgaKgTXPoHrd1kit87UG0nL-RzM9ImI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3961" title="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3981" title="Integrative Analysis of Transcriptomic and Methylomic Data in Photoperiod-Dependent Regulation of Cucumber Sex Expression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491516" id="node491516" data-pisa="ggg;8/12/3973" data-pisa-master="ggg;g3.118.200729" data-apath="/ggg/8/12/3973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordan R. Silke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yulong Wei</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3973-3979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200729 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491516" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordan R. Silke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BR.%2BSilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silke%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordan%2BR.%2BSilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordan R. Silke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yulong Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yulong%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYulong%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yulong Wei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3973" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491516" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3973.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491516" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3973.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491516" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3973.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491516" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3973.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491516" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491516/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">We present an RNA-Seq based approach to map 3′ end sequences of mature 16S rRNA (3′ TAIL) in bacteria with single-base specificity. Our results show that 3′ TAILs are heterogeneous among species; they contain the core CCUCC anti-Shine-Dalgarno motif, but vary in downstream lengths. Importantly, our findings rectify the mis-annotated 16S rRNAs in 11 out of 13 bacterial species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes). Furthermore, our results show that species-specific 3′ TAIL boundaries are retained due to their high complementarity with preferred Shine-Dalgarno sequences, suggesting that 3′ TAIL bases downstream of the canonical CCUCC motif play a more important role in translation initiation than previously reported.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">Gene Expression</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li><li class="kwd"><a href="/keyword/translation-initiation" class="hw-term hw-article-keyword hw-article-keyword-translation-initiation" rel="nofollow">Translation Initiation</a></li></ul><p id="p-4">Understanding bacterial translation is important to pharmaceutical industries seeking to optimize protein biosynthesis (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Xia 2018a</a>). In this process, the rate-limiting step is generally considered to be initiation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kudla <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tuller <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xia 2015</a>) and the most prominently cited mechanism of initiation in bacteria (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shine and Dalgarno 1974</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">1975</a>) involves an interaction between a pyrimidine-rich anti-Shine-Dalgarno (aSD) sequence at the 3′ end of the 16S rRNA (3′ TAIL) and a purine-rich Shine-Dalgarno (SD) sequence in the mRNA translation initiation region (TIR) of protein coding genes. Pairing between these two sequences helps the ribosome dock near the start codon.</p><p id="p-5">Efficient SD-mediated translation initiation requires optimal SD:aSD binding location and pairing potential (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schurr <em>et al.</em> 1993</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Osterman <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Prabhakaran <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abolbaghaei <em>et al.</em> 2017</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hockenberry <em>et al.</em> 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>). The canonical core aSD motif, CCUCC, is widely believed to elevate initation efficiency because of its strong complementarity with SD sequences and conservation across phyla (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Shine and Dalgarno 1974</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Woese <em>et al.</em> 1975</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Schurr <em>et al.</em> 1993</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Starmer <em>et al.</em> 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Vimberg <em>et al.</em> 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>). Yet what constitutes ideal SD:aSD complementarity remains a subject of debate. Some researchers contend that there is weak association between SD:aSD binding affinity and initiation efficiency (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2012</a>), but others suggest that intermediate binding affinities optimize initiation efficiency in <em>Escherichia coli</em> and <em>Bacillus subtilis</em> when a broader range of SD:aSD interactions is considered (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Vimberg <em>et al.</em> 2007</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Osterman <em>et al.</em> 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hockenberry <em>et al.</em> 2017</a>). Furthermore, when a SD sequence that binds to the <em>B. subtilis</em> 3′ TAIL is substituted with a shorter SD sequence pairing with <em>E. coli</em>’s 3′ TAIL, interferon plasmids’ expression levels decrease drastically (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Band and Henner 1984</a>). These findings emphasize the importance of characterizing the full extent of the 3′ TAIL.</p><p id="p-6">The 3′ TAIL boundary remains ambiguous for most bacterial species because the precise 3′ maturation process of the 16S precursor sequence remains unclear (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sulthana and Deutscher 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deutscher 2015</a>), and only a few mature 16S rRNA sequences have been experimentally verified (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Woese <em>et al.</em> 1980</a>). Consequently, determination of the 16S rRNA is frequently automated based on sequence similarity (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lin <em>et al.</em> 2008</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>). However, this process is often unreliable (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Starmer <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jones <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lagesen <em>et al.</em> 2007</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lin <em>et al.</em> 2008</a>) and many such 16S ribosomal RNA sequence annotations have been discontinued in NCBI’s Gene database. For example, 16S rRNA entries for <em>Streptococcus pyogenes</em> (NC_002737), <em>Bacillus anthracis</em> (NC_005945), and <em>Legionella pneumophila</em> (NC_005823) are all truncated such that their annotated 3′ ends do not encompass the canonical CCUCC motif.</p><p id="p-7">To circumvent the aforementioned problem, we devise strategies to map RNA transcripts from high-throughput RNA sequencing (RNA-Seq) data (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lister <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anders <em>et al.</em> 2013</a>) to the 16S rDNA genomic sequence with single base specificity. The feasibility of this approach was shown recently in a study (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>) where we successfully recovered the <em>E. coli</em> and <em>B. subtilis</em> 3′ TAILs documented in literature (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Shine and Dalgarno 1974</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Woese <em>et al.</em> 1975</a>). Our present objective is to advance our RNA-Seq framework to characterize the 3′ TAIL in any bacterial species, especially those that have not been experimentally verified.</p><p id="p-8">The challenge associated with our approach is the limited availability of suitable data. There is a complete lack of publicly available RNA-Seq data in GEO DataSets for many species, such as <em>Acidithiobacillus ferrooxidans</em>, <em>Microcystis aeruginosa</em>, <em>Shigella flexneri</em>, and <em>Yersinia pestis</em>. Furthermore, many experiments remove rRNAs prior to sequencing (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">O’Neil <em>et al.</em> 2013</a>) in an effort to enrich the target RNA molecules, such as mRNAs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Choi and Hagedorn 2003</a>). Fortunately, our findings suggest that ribo-depletion is often incomplete, and enough 16S rRNA reads will persist to allow for 3′ TAIL characterization. The inclusion of 13 species studied herein (covering Cyanobacteria, Deinococcus-Thermus, Firmicutes, Proteobacteria, Tenericutes, and Spirochaetes) is thus predicated on the availability of usable RNA-Seq datasets in NCBI’s GEO (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edgar <em>et al.</em> 2002</a>) database (see Materials and Methods for additional details). Additionally, the availability of protein abundance data in PaxDb (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2012</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2015</a>) for all species studied allow us to investigate the effect of SD:aSD complementarity on protein production in real genes.</p><p id="p-9">Comprehensive comparative sequence analyses (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2017</a>) claim 5′-CCUCCU-3′ is the functionally constrained 3′ TAIL terminus. In other words, the motif is conserved among bacterial species because it pairs with SD sequences effectively. However, several bases further downstream are conserved in the genomic sequences of closely related species. We suspect that this is the result of functional constraint imposed by the SD:aSD interaction further downstream of 5′-CCUCCU-3′. Accordingly, we hypothesize that downstream bases are retained in 3′ TAILs because they effectively interact with species-specific SD sequences as previously observed for <em>E. coli</em> and <em>B. subtilis</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Band and Henner 1984</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abolbaghaei <em>et al.</em> 2017</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>).</p><p id="p-10">Our findings corroborate previous studies suggesting that intermediate binding affinity is preferred (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Osterman <em>et al.</em> 2013</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Hockenberry <em>et al.</em> 2017</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>). The 3′ termini downstream of the core CCUCC are heterogeneous among species, but fall within the conserved boundary at the genomic level. Furthermore, terminal bases are preferred in SD:aSD binding in most species, albeit having weaker binding affinity than CCUCC. These findings demonstrate the importance of considering bases downstream of CCUCC in SD:aSD binding.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Processing genomic and RNA-Seq data</h3><p id="p-11">The annotated genomes of 26 species in GenBank formats were retrieved from the National Center for Biotechnology Information (NCBI) database (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). Next, the NCBI annotated 16S rRNA was retrieved. In the case where multiple 16S rRNA entries exist, the first one listed is selected.</p><p id="p-12">High-throughput RNA-Seq SRX runs of wildtype species were downloaded from GEO DataSets in FASTQ format. The FASTQ files were first converted to FASTQ+ format using ARSDA 1.1 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Xia 2017</a>), grouping identical reads under a single ID while also indicating the number of copies (SeqID_# of copies), in order to reduce the size of the datasets prior to adapter trimming. The FASTQ+ data were then processed using CutAdapt 1.17 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Martin 2011</a>) to trim off the 3′ flanking adapter sequences. In experiments that use the oligo(dT)-adapter primer, RNA fragments are first poly-adenylated at the 3′ end, we thus set CutAdapt to recognize “AAAAA”. In others that use specific sets of primers ligated to random hexamers, we set CutAdapt to recognize all possible adapters in the kits’ index, with 10% error rate. Regardless of whether poly-As or barcode adapters were trimmed, we only retained reads that were 25 nt or longer after the trimming process to mitigate bias in expression levels (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Williams <em>et al.</em> 2016</a>). Next, we used Trimmomatic 0.38 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>) to remove poor quality sequences with average Phred scores lower than 20 (1% probability of a base calling error) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ewing and Green 1998</a>). Since adapters were trimmed after reads were grouped in FASTQ+ format, sequences that were previously unique due to the presence of adapter nucleotides may become identical (such as for SeqGr176560_1 and SeqGr558077_1, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1d</a>). The processed FASTQ+ datasets were subsequently converted into FASTA format for multiple sequence alignment.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The count of mapped 3&#x2032; ends of RNA-Seq reads (A, C) and sequence alignments (B, D) for Lactococcus lactis and Deinococcus deserti. Mapped regions start with the last C of CCUCC as the first site, extended by 30 nt downstream. The 3&#x2032; ends of sequence alignments represent local reads mapped to the single major peak in L. lactis and the two major peaks in D. deserti. The complete length of the query genomic sequence is 205 nt long." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-657151478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The count of mapped 3&#x2032; ends of RNA-Seq reads (A, C) and sequence alignments (B, D) for &lt;em&gt;Lactococcus lactis&lt;/em&gt; and &lt;em&gt;Deinococcus deserti&lt;/em&gt;. Mapped regions start with the last C of CCUCC as the first site, extended by 30 nt downstream. The 3&#x2032; ends of sequence alignments represent local reads mapped to the single major peak in &lt;em&gt;L. lactis&lt;/em&gt; and the two major peaks in &lt;em&gt;D. deserti&lt;/em&gt;. The complete length of the query genomic sequence is 205 nt long.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3973/F1.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3973/F1.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491577" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">The count of mapped 3′ ends of RNA-Seq reads (A, C) and sequence alignments (B, D) for <em>Lactococcus lactis</em> and <em>Deinococcus deserti</em>. Mapped regions start with the last C of CCUCC as the first site, extended by 30 nt downstream. The 3′ ends of sequence alignments represent local reads mapped to the single major peak in <em>L. lactis</em> and the two major peaks in <em>D. deserti</em>. The complete length of the query genomic sequence is 205 nt long.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Aligning RNA-Seq reads to annotated rRNA sequences</h3><p id="p-14">We next mapped reads in the FASTA files onto the 16S rDNA genomic sequence. The FASTA+ files were converted into BLAST databases using the “Create BLAST DB” function in ARSDA. The BLAST query sequence was selected using genomic sequences 100 nt upstream and downstream of the core CCUCC motif (205 nt total query length). For each species, the query sequence was searched against BLAST databases using the BLAST function (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) implemented in ARSDA. We used an E-value cutoff of 10<sup>−5</sup> (with the exception of <em>Bacillus anthracis</em>, for which we used an E-value of 10<sup>−3</sup> due to the relatively shorter average read length and smaller database size) paired with a minimum word length of 12 to balance the quantity and quality of hits, as well as search speed, against the &gt;= 25 nt reads in the ribo-depleted datasets. Then, sequence hits were retrieved from the FASTA files using seqtk (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li 2012</a>) and complementary strand sequences were eliminated. Finally, remaining hits were aligned to the query sequence using multiple sequence alignment (Clustal Omega algorithm (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sievers and Higgins 2014</a>) implemented in DAMBE, default parameters).</p></div><div id="sec-4" class="subsection"><h3>Determining putative SD sequences based on pairing potential, location, and binding affinity</h3><p id="p-15">For each species, our characterized 3′ TAILs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were used as the complementary sequence in identifying putative SD sequences. To ensure that determined putative SD sequences are from real genes, we map protein IDs in PaxDb 4.0 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2012</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">2015</a>) to Gene IDs in NCBI and only use CDSs that have protein expressions. Using DAMBE7 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Xia 2018b</a>), we followed the methods used in previous studies (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nussinov <em>et al.</em> 1978</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Waterman and Smith 1978</a>): 30 nt upstream of start codon of all CDSs were extracted and matched against the annotated 3′ TAIL with ‘Analyzing 5′UTR’ in DAMBE, with minimum SD length =4 nt and maximum SD length = 12 nt. Site-specific observed and expected aSD usage values were retrieved from the DAMBE when SD sequences are determined.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491702/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491702/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The RNA-Seq corrected 3′ TAIL in 13 bacterial species. RNA-Seq determined 3′ TAILs are shaded gray. The NCBI annotated 3′ TAILs are in black fonts, extensions revealed by RNA-Seq data are underlined and ambiguities in bold</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Data Availability</h3><p id="p-18">Supplementary file S1 contains RNA-Seq BLAST hits and file S2 contains the list of genes with protein abundance data that were used to determine putative SD sequences in all species studied; Figure S1 contain the 3′ TAIL map for the remaining 11 species. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7081094">https://doi.org/10.25387/g3.7081094</a>.</p></div></div><div class="section results-discussion" id="sec-6"><h2 class="">Results and Discussion</h2><div id="sec-7" class="subsection"><h3>Characterizing the 3′ TAIL in bacteria using an improved RNA-Seq-based approach</h3><p id="p-19">We improve upon our method of 3′ TAIL characterization (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>) by processing the RNA-Seq data more rigorously. To ensure quality and single-base specificity for reads mapped to a reference genomic sequence, we used CutAdapt (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Martin 2011</a>) to trim adapters flanking raw RNA-Seq reads because these sequences obscure the true end of RNA fragments (see Materials and Methods for more detail). We subsequently filtered out poor quality reads by discarding those with average Phred scores ≤ 20 using Trimmomatic (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>); in other words, we retained reads with average base-calling error rates of &lt; 1% (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ewing and Green 1998</a>). A caveat of using poly-adenylated RNA-Seq datasets for <em>Neisseria meningitidis</em> is that we cannot distinguish between 5′-CCUCCUUUCU-3′ and 5′-CCUCCUUUCUA-3′ as the 3′ TAIL; it is unclear whether the first adenosine is associated with the 3′ TAIL or the poly-A chain (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-20">To map the 16S rRNA, we generated a BLAST library using the quality filtered datasets and performed ungapped local similarity search using BLAST (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) between RNA-Seq reads and a 205 nt genomic sequence with the canonical CCUCC motif at the center (100 nt extending from each side). We next aligned the BLAST hits by multiple sequence alignment (Clustal Omega algorithm (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sievers and Higgins 2014</a>) implemented in DAMBE (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Xia 2018b</a>), default parameters) against the reference genomic sequence. In all species, we define the terminus of the 3′ TAIL using two criteria: 1) it must contain the canonical CCUCC, and 2) it is the most mapped site at or near CCUCC. The underlying assumption for the second criterion is that the mature 16S rRNA is more abundant than precursor transcripts, as is the case in <em>E. coli</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cangelosi and Brabant 1997</a>), because precursors are continuously degraded by exoribonucleases (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sulthana and Deutscher 2013</a>).</p></div><div id="sec-8" class="subsection"><h3>The 3′ TAIL termini are heterogeneous but functionally constrained</h3><p id="p-21">Following our two criteria, we have characterized the 3′ TAIL in 13 out of 26 species in PaxDb (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows the sequence map and alignments for <em>Lactococcus lactis</em> and <em>Deinococcus deserti</em>. The sequences mapped for the 11 others are present in Supplementary Figure S1. Two others, <em>E. coli</em> and <em>B. subtilis</em>, were previously determined (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>). The remaining 11 species could not be characterized because of the aforementioned absence of data in four species (<em>Acidithiobacillus ferrooxidans</em>, <em>Microcystis aeruginosa</em>, <em>Shigella flexneri</em>, and <em>Yersinia pestis</em>), and because no convincing peaks were observed in the region of interest (up to 30 nt downstream of CCUCC) in the remaining seven species (<em>Bacterioides thetaiotaomicron</em>, <em>Bateonella henselae</em>, <em>Helicobacter pylori</em>, <em>Mycobacterium tuberculosis</em>, <em>Pseudomonas aeruginosa</em>, <em>Staphylococcus aureus</em>, and <em>Shewanella oneidensis</em>), likely due to effective ribo-depletion in their RNA-Seq datasets. We considered a peak to be convincing when the counts mapping to that site were at least 3 fold higher than background (counts of any four flanking sites on either side). Importantly, in the characterized 13 species, we made corrections to annotations in eight species (NC_002662 <em>L. lactis</em>, NC_002163 <em>Clamylobacter jejuni</em>, NC_000911 <em>Synechocystis</em> sp., NC_003112 <em>N. meningitidis</em>, NC_<em>012526 D. deserti</em>, NC_017504 <em>Mycoplasma pneumoniae</em>, NC_003198 <em>Salmonella enterica</em>, NC_002937 <em>Desulfovibrio vulgaris</em>), and redefined the 3′ TAIL in three others (NC_002737 <em>S. pyogenes</em>, NC_005945 <em>B. anthracis</em>, and NC_002942 <em>L. pneumophila</em>) that were certainly mis-annotated due to their failure to incorporate the canonical CCUCC. Resultantly, the annotated 3′ TAILs of only two out of 13 species, NC_003210 <em>Listeria monocytogenes</em> and NC_005823 <em>Leptospira interrogans</em> were left unchanged. In short, the 3′ TAILs can variably extend up to six bases downstream of CCUCC in the majority of species studied.</p><p id="p-22">The 3′ TAILs vary among species, but bases downstream of the CCUCC motif are conserved among bacteria. The 16S genomic sequences are largely conserved for several bases beyond CCUCC: <em>e.g.</em>, 5′-GAUCA<span class="underline">CCUCC</span>UUUCUA-3′ in Bacilli and 5′-GAUCA<span class="underline">CCUCC</span>UUA-3′ in Beta- and Gammaproteobacteria. This conservation suggests that the regions downstream of CCUCC may also be important in SD:aSD pairing. Importantly, 3′ TAIL terminal bases downstream of CCUCC are species-specific, but do not extend past the conserved genomic boundaries, in all species studied except in <em>L. interrogans</em>. In other words, the 3′ TAIL falls variably short of 5′-GAUCACCUCCUUUCUA-3′. This finding further suggests that both CCUCC and downstream bases are conserved regions that are preferred in SD:aSD pairing in most species.</p><p id="p-23">To offer a plausible reason for the unexpected length of the 3′ TAIL in <em>L. interrogans</em>, it is worth noting that the dependence on the SD:aSD interaction for efficient translation is dynamic (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Nakagawa <em>et al.</em> 2017</a>). In genes that have strong secondary structure within the TIR, ribosome recruitment is facilitated by RPS1 (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Osterman <em>et al.</em> 2013</a>). This protein binds U-rich regions (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boni <em>et al.</em> 1991</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Komarova <em>et al.</em> 2005</a>) to unfold double-stranded RNA (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Qu <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duval <em>et al.</em> 2013</a>). Furthermore, RPS1’s domains appear to be under higher functional constraint in species possessing few SD-containing genes, such as <em>L. interrogans</em> (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">2017</a>); the reliance on RPS1 reduces the dependence on a SD:aSD interaction and may relax 3′ TAIL boundary constraints.</p><p id="p-24">Notably, four species (<em>D. deserti</em>, <em>M. pneumoniae</em>, <em>L. pneumophila</em>, and <em>L. interrogans</em>) have a secondary peak of mapped reads within 20 nt downstream of CCUCC (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, Supplementary Fig. S1). We propose that the secondary peak farther downstream is the pre-16S rRNA; it is too far downstream of CCUCC to be considered as the mature 16S rRNA 3′ end based on sequence conservation (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Nakagawa <em>et al.</em> 2010</a>). The prominence of this second peak may be due to the accumulation of the endoribonuclease cleaved pre-16S rRNA intermediate, because the localization of exoribonuclease to this precursor sequence is a rate limiting step. However, the intermediate sequence is rapidly continuously degraded once it is targeted by these enzymes. This would explain the lack of sequences mapped between the mature 16S rRNA and the intermediate sequence (the two peaks) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1c</a>, Supplementary Fig. S1).</p></div><div id="sec-9" class="subsection"><h3>The 3′ TAIL terminal bases are preferred in SD:aSD binding</h3><p id="p-25">We define an aSD site to be preferred if the observed number of times the base is involved in SD pairing is greater than expected. In the absence of SD usage bias, a putative SD sequence of 4 to 12 nt can be expected to pair anywhere within the boundary of the aSD sequence, as long as complete complementarity is achieved. Here, we designate the aSD sequence to constitute the 3′ TAIL, beginning with the conserved 5′-GAUCA-3′, followed by the core motif CCUCC, and extended by variable lengths of terminal bases characterized herein (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, <em>e.g.</em>, in <em>L. monocytogenes</em>, the aSD sequence is 5′GAUCACCUCCUUUCU-3′ and the terminal bases are 5′-UUUCU-3′). Then, taking <em>L. monocytogenes</em> as example, the maximum number of possible pairs at the first complementary aSD site (<em>aSD_1</em>) by the total pool of 4 nt to 12 nt putative SD sequences is calculated by <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a>, with N<sub>m</sub> denoting N number of putative SD sequences of length m:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3973/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3973/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>However, the number of possible base-pairs resulting in perfect complementarity varies. For example, a 12 nt putative SD sequence may start pairing at the first, but not the sixth, base on a complementary aSD sequence that is 15 nt long, and the maximum usage of the sixth aSD site (<em>aSD_6</em>) is calculated instead by <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equation (2)</a>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3973/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3973/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The expected usage is then calculated by taking the relative proportions of maximum usage at each site (adding up to 1) multiplied by the total number of observed putative SD sequences of various lengths. These calculations are implemented in DAMBE (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Xia 2018b</a>) under the ‘Analyze 5UTR’ function.</p><p id="p-26">As defined, a preferred aSD site will have an observed to expected SD:aSD usage ratio (O:E) &gt; 1. Since expected SD:aSD count is calculated in absence of any selection bias in SD usage, an O:E &gt; 1 suggests presence of selection bias in observed SD usage. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows an average O:E &gt; 1 at CCUCC for all species, with the exception of <em>L. interrogans</em>. Indeed, CCUCC is preferentially used in SD:aSD pairing. Meanwhile, average O:E is &lt;1 for 5′-GAUCA-3′ in all species except <em>M. pneumoniae</em>; hence, this conserved region is avoided by SD:aSD pairing in most species. Lastly, in keeping with expectations, conserved regions downstream of CCUCC have an O:E &gt; 1 in all species except <em>L. monocytogenes</em>. These observations indicate that downstream bases are retained because they are preferred in SD:aSD binding, despite their weaker binding affinity than CCUCC. This result corroborates recent studies suggesting that intermediate SD:aSD complementarity increase initiation efficiency (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Osterman <em>et al.</em> 2013</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Hockenberry <em>et al.</em> 2017</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Wei <em>et al.</em> 2017</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean ratio of observed over expected SD:aSD complementarity (O:E ratio) in 13 species at conserved 5&#x2032;-GAUCA-3&#x2032; (blue), and 5&#x2032;-CCUCC-3&#x2032; motifs (red). The average O:E ratio is also shown for the characterized sequences downstream of CCUCC (green) in the nine bacterial species that have extended ends." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-657151478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean ratio of observed over expected SD:aSD complementarity (O:E ratio) in 13 species at conserved 5&#x2032;-GAUCA-3&#x2032; (blue), and 5&#x2032;-CCUCC-3&#x2032; motifs (red). The average O:E ratio is also shown for the characterized sequences downstream of CCUCC (green) in the nine bacterial species that have extended ends.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3973/F2.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3973/F2.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3973/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Mean ratio of observed over expected SD:aSD complementarity (O:E ratio) in 13 species at conserved 5′-GAUCA-3′ (blue), and 5′-CCUCC-3′ motifs (red). The average O:E ratio is also shown for the characterized sequences downstream of CCUCC (green) in the nine bacterial species that have extended ends.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In this study we present an RNA-Seq based approach to characterize the 3′ end of mature 16S rRNA in bacterial species across different lineages. There is weaker 3′ TAIL conservation at the RNA level than at the DNA level. Nonetheless, the presence of 3′ termini bases downstream of CCUCC falls within the conserved boundary at the genomic level. Furthermore, the usage of terminal bases is favored in SD:aSD binding. Alternatively, RPS1-mediated initiation may relax the functional constraint at the 3′ TAIL of <em>L. interrogans</em>, explaining its exceptional length. Our findings complement previous studies investigating the role of CCUCC in translation initiation and suggest that transcribed bases downstream of this canonical motif also play an important role in translation efficiency.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-29">This work was supported by the Discovery Grant of Natural Science and Engineering Research Council of Canada to X.X. (NSERC, RGPIN/2018-03878), and the Ontario Graduate Scholarship 2018-2019 to Y.W. The manuscript was substantially improved by the comments of two anonymous reviewers, and we are grateful for their insight.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Equal contribution.</p></li><li class="fn" id="fn-4"><p id="p-30">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7081094">https://doi.org/10.25387/g3.7081094</a>.</p></li><li class="fn" id="fn-5"><p id="p-31"><em>Communicating editor: D. Baltrus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-12"><span class="received-label">Received </span>September 12, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-21"><span class="accepted-label">Accepted </span>October 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Silke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.1" data-doi="10.1016/j.foreco.2013.04.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abolbaghaei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silke</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">How Changes in Anti-SD Sequences Would Affect SD Sequences in <em>Escherichia coli</em> and <em>Bacillus subtilis</em></span>. <abbr class="cit-jnl-abbrev">G3 Genes|Genomes|Genetics (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.039305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%257CGenomes%257CGenetics%2B%2528Bethesda%2529%26rft.volume%253D304%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2013.04.040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2013.04.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.2" data-doi="10.1255/jnirs.760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D16%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.3" data-doi="10.2135/cropsci2017.01.0007."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okoniewski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2013</span> <span class="cit-article-title">Count-based differential expression analysis of RNA sequencing data using R and Bioconductor.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1765</span>–<span class="cit-lpage">1786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2013.099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D57%26rft.spage%253D2517%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.01.0007.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.01.0007.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.4" data-doi="10.1002/jsfa.2740361114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Band</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henner</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title"><em>Bacillus subtilis</em> Requires a “Stringent” Shine-Dalgarno Region for Gene Expression.</span> <abbr class="cit-jnl-abbrev">DNA</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/dna.1.1984.3.17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%26rft.volume%253D36%26rft.spage%253D1120%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjsfa.2740361114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jsfa.2740361114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.5" data-doi="10.1111/j.1744-7909.2012.01116.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DCabrera-Bosquet%26rft.auinit1%253DL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DHigh-throughput%2Bphenotyping%2Band%2Bgenomic%2Bselection%253A%2Bthe%2Bfrontiers%2Bof%2Bcrop%2Bbreeding%2Bconverge.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01116.x%26rft_id%253Dinfo%253Apmid%252F22420640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01116.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22420640&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.6" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boni</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaeva</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musychenko</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzareva</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Ribosome-messenger recognition: mRNA target sites for ribosomal protein S1.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/19.1.155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.7" data-doi="10.1534/g3.116.029637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cangelosi</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brabant</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Depletion of pre-16S rRNA in starved <em>Escherichia coli</em> cells.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">4457</span>–<span class="cit-lpage">4463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/jb.179.14.4457-4463.1997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029637%26rft_id%253Dinfo%253Apmid%252F27172218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xODE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.8" data-doi="10.1073/pnas.1232347100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagedorn</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Purifying mRNAs with a high-affinity eIF4E mutant identifies the short 3′ poly(A) end phenotype.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">7033</span>–<span class="cit-lpage">7038</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1232347100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D56%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1232347100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1073/pnas.1232347100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutscher</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Twenty years of bacterial RNases and RNA processing: how we’ve matured.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.049692.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duval</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korepanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsbauer</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fechter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Escherichia coli</em> Ribosomal Protein S1 Unfolds Structured mRNAs Onto the Ribosome for Active Translation Initiation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pbio.1001731</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.11" data-doi="10.1016/j.compag.2013.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domrachev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lash</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/30.1.207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D99%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compag.2013.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compag.2013.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.12" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.8.3.186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft.volume%253D55%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.13" data-doi="10.1093/treephys/tpp036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hockenberry</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pah</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">L. A. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Leveraging genome-wide datasets to quantify the functional role of the anti-Shine-Dalgarno sequence in regulating translation efficiency.</span> <abbr class="cit-jnl-abbrev">Open Biol</abbr>. <span class="cit-vol">7</span>: <span class="cit-fpage">160239</span>. <a href="http://dx.doi.org/10.1098/rsob.160239">http://dx.doi.org/10.1098/rsob.160239</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftpp036%26rft_id%253Dinfo%253Apmid%252F19483184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tpp036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19483184&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268117200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.14" data-doi="10.1016/j.fcr.2012.10.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumann</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Estimating the annotation error rate of curated GO database sequence annotations.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D140%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2012.10.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2012.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313317800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.15" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komarova</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tchufistova</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreyfus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boni</span>  <span class="cit-name-given-names">I. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">AU-Rich Sequences within 5′ Untranslated Leaders Enhance Translation and Stabilize mRNA in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">1344</span>–<span class="cit-lpage">1349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.187.4.1344-1349.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.16" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-Sequence Determinants of Gene Expression in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1170160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.17" data-doi="10.1255/jnirs.1006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodland</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staerfeldt</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rognes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3100</span>–<span class="cit-lpage">3108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D20%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.1006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.1006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.18" data-doi="10.1111/1755-0998.12513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10965</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D16%26rft.spage%253D1023%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12513%26rft_id%253Dinfo%253Apmid%252F26929265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26929265&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> seqtk Toolkit for processing sequences in FASTA/Q formats.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.20" data-doi="10.1007/s11306-016-1099-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Questionable 16S ribosomal RNA gene annotations are frequent in completed microbial genomes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2008.02.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D12%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11306-016-1099-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11306-016-1099-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.21" data-doi="10.1016/j.fcr.2004.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonti-Filippini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2008</span> <span class="cit-article-title">Highly integrated single-base resolution maps of the epigenome in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">536</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2008.03.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D94%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2004.12.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2004.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232196400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.22" data-doi="10.4161/fly.21535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal; Vol 17, No 1 Next Gener. Seq. Data Anal. </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D6%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.21535%26rft_id%253Dinfo%253Apmid%252F22885252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.21535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22885252&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311076700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.23" data-doi="10.1007/s004420050591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic evolution of translation initiation mechanisms in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">6382</span>–<span class="cit-lpage">6387</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1002036107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D116%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004420050591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004420050591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076034600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.24" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative genomic analysis of translation initiation mechanisms for genes lacking the Shine–Dalgarno sequence in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">3922</span>–<span class="cit-lpage">3931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx124</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.25" data-doi="10.1007/s00122-011-1747-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nussinov</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pieczenik</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griggs</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleitman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Algorithms for Loop Matchings.</span> <abbr class="cit-jnl-abbrev">SIAM J. Appl. Math.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1137/0135006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFu%26rft.auinit1%253DJ.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D825%26rft.epage%253D833%26rft.atitle%253DPartial%2Bleast%2Bsquares%2Bregression%252C%2Bsupport%2Bvector%2Bmachine%2Bregression%252C%2Band%2Btranscriptome-based%2Bdistances%2Bfor%2Bprediction%2Bof%2Bmaize%2Bhybrid%2Bperformance%2Bwith%2Bgene%2Bexpression%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1747-9%26rft_id%253Dinfo%253Apmid%252F22101908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1747-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101908&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3973.26" data-doi="10.2135/cropsci2016.09.0742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glowatz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlumpberger</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ribosomal RNA depletion for efficient use of RNA-seq capacity.</span> <span class="cit-source">Curr. Protoc. Mol. Biol.</span> <span class="cit-vol">Chapter 4</span>: <span class="cit-fpage">Unit 4.19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471142727.mb0419s103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol.%26rft.volume%253D57%26rft.spage%253D2372%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.09.0742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.09.0742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.27" data-doi="10.1093/nar/gks989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osterman</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evfratov</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sergiev</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dontsova</span>  <span class="cit-name-given-names">O. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of mRNA features affecting translation initiation and reinitiation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks989</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D107%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.28" data-doi="10.1099/vir.0.000050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prabhakaran</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chithambaram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Escherichia coli</em> and <em>Staphylococcus</em> phages: effect of translation initiation efficiency on differential codon adaptation mediated by virulent and temperate lifestyles.</span> <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/vir.0.000050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.000050%26rft_id%253Dinfo%253Apmid%252F25614589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.000050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25614589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.29" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noller</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinoco</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ribosomal protein S1 unwinds double-stranded RNA in multiple steps.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14458</span>–<span class="cit-lpage">14463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1208950109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.30" data-doi="10.2135/cropsci2016.06.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schurr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadir</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margalit</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Identification and characterization of <em>E.coli</em> ribosomal binding sites by free energy computation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4019</span>–<span class="cit-lpage">4023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/21.17.4019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D57%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.06.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.06.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.31" data-doi="10.2135/cropsci2016.08.0675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shine</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalgarno</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The 3′-terminal sequence of <em>Escherichia coli</em> 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1342</span>–<span class="cit-lpage">1346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.71.4.1342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D57%26rft.spage%253D1404%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.08.0675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.08.0675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.32" data-doi="10.1186/s12864-015-2345-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shine</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalgarno</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Determinant of cistron specificity in bacterial ribosomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">254</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/254034a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D17%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2345-z%26rft_id%253Dinfo%253Apmid%252F26732811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2345-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26732811&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.33" data-doi="10.1093/treephys/tpv056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Clustal Omega, accurate alignment of very large numbers of sequences.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1079</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-646-7_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftreephys%252Ftpv056%26rft_id%253Dinfo%253Apmid%252F26224105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/treephys/tpv056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26224105&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.34" data-doi="10.1007/s00122-016-2780-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Starmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stomp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitzer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting Shine–Dalgarno Sequence Locations Exposes Genome Annotation Errors.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0020057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D129%26rft.spage%253D2413%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2780-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2780-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.35" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulthana</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deutscher</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple exoribonucleases catalyze maturation of the 3′ terminus of 16S ribosomal RNA (rRNA).</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">12574</span>–<span class="cit-lpage">12579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.C113.459172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.36" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldman</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translation efficiency is determined by both codon bias and folding energy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3645</span>–<span class="cit-lpage">3650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0909910107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.37" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vimberg</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tats</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Translation initiation region sequence preferences in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2199-8-100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMol.%2BBiol.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.38" data-doi="10.1255/jnirs.1108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D22%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1255%252Fjnirs.1108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1255/jnirs.1108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.39" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haertinger</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrimpf</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">PaxDb, a Database of Protein Abundance Averages Across All Three Domains of Life.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">500</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/mcp.O111.014704</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.40" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Mering</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Version 4.0 of PaxDb: Protein abundance data, integrated across model organisms, tissues, and cell-lines.</span> <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3163</span>–<span class="cit-lpage">3168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/pmic.201400441</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.41" data-doi="10.1016/j.foreco.2017.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">RNA secondary structure: a complete mathematical analysis.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0025-5564(78)90099-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D406%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2017.09.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2017.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.42" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silke</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Elucidating the 16S rRNA 3′ boundaries and defining optimal SD/aSD pairing in <em>Escherichia coli</em> and <em>Bacillus subtilis</em> using RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">17639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-17918-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.43" data-doi="10.1016/j.fcr.2017.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baccarella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Trimming of sequence reads alters RNA-Seq gene expression estimates.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-0956-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D216%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2017.08.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2017.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.44" data-doi="10.1038/nrg2612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woese</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zablen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uchida</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1975</span> <span class="cit-article-title">Conservation of primary structure in 16S ribosomal RNA.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">254</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/254083a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D565%26rft.epage%253D577%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bquantitative%2Btraits%253A%2Bchallenges%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2612%26rft_id%253Dinfo%253Apmid%252F19584810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584810&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.45" data-doi="10.3389/fpls.2017.02002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woese</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magrum</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1980</span> <span class="cit-article-title">Secondary structure model for bacterial 16S ribosomal RNA: phylogenetic, enzymatic and chemical evidence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2275</span>–<span class="cit-lpage">2293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/8.10.2275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D8%26rft.spage%253D2002%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.02002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.02002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.46" data-doi="10.1534/genetics.114.172106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Major Controversy in Codon-Anticodon Adaptation Resolved by a New Codon Usage Index.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.172106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D22%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.114.172106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.47" data-doi="10.1146/annurev-animal-031412-103705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data</span>. <abbr class="cit-jnl-abbrev">G3 Genes|Genomes|Genetics (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3839</span>–<span class="cit-lpage">3848</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%257CGenomes%257CGenetics%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-031412-103705%26rft_id%253Dinfo%253Apmid%252F25387018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-031412-103705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387018&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3973.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> Bioinformatics and translation initiation, pp. 173–195 in <em>Bioinformatics and the Cell</em>:<span class="cit-comment"> modern computational approaches in genomics, proteomics and transcriptomics. Springer, Cham.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3973.49" data-doi="10.1093/molbev/msy073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">DAMBE7: New and Improved Tools for Data Analysis in Molecular Biology and Evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1550</span>–<span class="cit-lpage">1552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D1550%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3973.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3961" title="Phenomic Selection Is a Low-Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concept on Wheat and Poplar" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3981" title="Integrative Analysis of Transcriptomic and Methylomic Data in Photoperiod-Dependent Regulation of Cucumber Sex Expression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3973.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FvEN2CZEZQXF3_Vga-_qwRPrXwrcHbg9QrmYt_hFj7M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3973" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3973" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3973" class="active" data-icon-position="" data-hide-link-title="0">RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491516" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BjDZEXJ7tLBqVTpej10mUCXgrW5Y-0flYVysTlDh_Hg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3973.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3973" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-prIhgvDaLnDY0sdps_9QNmjNhi6Ui4l_JEfC6spKDeg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3973" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3973.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491516" id="node491516--2" data-pisa="ggg;8/12/3973" data-pisa-master="ggg;g3.118.200729" data-apath="/ggg/8/12/3973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordan R. Silke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yulong Wei</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3973-3979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200729 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491516--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordan R. Silke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BR.%2BSilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silke%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordan%2BR.%2BSilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordan R. Silke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yulong Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yulong%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYulong%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yulong Wei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3973" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3973&amp;text=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3973&amp;t=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3973&amp;title=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491516/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491516" id="node491516--3" data-pisa="ggg;8/12/3973" data-pisa-master="ggg;g3.118.200729" data-apath="/ggg/8/12/3973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-Seq-Based Analysis Reveals Heterogeneity in Mature 16S rRNA 3′ Termini and Extended Anti-Shine-Dalgarno Motifs in Bacterial Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jordan R. Silke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yulong Wei</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3973-3979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200729 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491516--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jordan R. Silke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordan%2BR.%2BSilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silke%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJordan%2BR.%2BSilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1578-1933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jordan R. Silke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yulong Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yulong%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYulong%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1218-2086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yulong Wei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ottawa Institute of Systems Biology, Ottawa, Ontario, Canada K1H 8M5</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3973&amp;rft.epage=3979&amp;rft.atitle=RNA-Seq-Based%20Analysis%20Reveals%20Heterogeneity%20in%20Mature%2016S%20rRNA%203%E2%80%B2%20Termini%20and%20Extended%20Anti-Shine-Dalgarno%20Motifs%20in%20Bacterial%20Species&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Silke%2C+Jordan+R.&amp;rft.au=Wei%2C+Yulong&amp;rft.au=Xia%2C+Xuhua"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491516/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491516/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491516/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491516/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491516/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491516/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491516/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491516/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491516/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491516/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491516/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491516/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3973" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3973.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3973.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3973.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3973.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
