<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3915" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3915.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3915.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3915.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3915.atom" />
<meta name="HW.pisa" content="ggg;8/12/3915" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" />
<meta name="DC.Identifier" content="10.1534/g3.118.200553" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Ohkura et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7242851" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6721511" />
<meta name="DC.Description" content="Aspergillus flavus , the primary causal agent for aflatoxin contamination on crops, consists of isolates with two distinct morphologies: isolates of the S morphotype produce numerous small sclerotia and lower numbers of conidia while isolates of the L morphotype produce fewer large sclerotia and abundant conidia. The morphotypes also differ in aflatoxin production with S isolates consistently producing high concentrations of aflatoxin, whereas L isolates range from atoxigenic to highly toxigenic. The production of abundant sclerotia by the S morphotype suggests adaptation for long-term survival in the soil, whereas the production of abundant conidia by the L morphotype suggests adaptation for aerial dispersal to the phyllosphere. To identify genomic changes that support differential niche adaption, the sequences of three S and three L morphotype isolates were compared. Differences in genome structure and gene content were identified between the morphotypes. A &gt;530 kb inversion between the morphotypes affect a secondary metabolite gene cluster and a cutinase gene. The morphotypes also differed in proteins predicted to be involved in carbon/nitrogen metabolism, iron acquisition, antimicrobial defense, and evasion of host immunity. The S morphotype genomes contained more intact secondary metabolite clusters indicating there is higher selection pressure to maintain secondary metabolism in the soil and that it is not limited to aflatoxin production. The L morphotype genomes were enriched in amino acid transporters, suggesting efficient nitrogen transport may be critical in the nutrient limited phyllosphere. These findings indicate the genomes of the two morphotypes differ beyond developmental genes and have diverged as they adapted to their respective niches." />
<meta name="DC.Contributor" content="Mana Ohkura" />
<meta name="DC.Contributor" content="Peter J. Cotty" />
<meta name="DC.Contributor" content="Marc J. Orbach" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Aspergillus flavus&lt;/i&gt;, the primary causal agent for aflatoxin contamination on crops, consists of isolates with two distinct morphologies: isolates of the S morphotype produce numerous small sclerotia and lower numbers of conidia while isolates of the L morphotype produce fewer large sclerotia and abundant conidia. The morphotypes also differ in aflatoxin production with S isolates consistently producing high concentrations of aflatoxin, whereas L isolates range from atoxigenic to highly toxigenic. The production of abundant sclerotia by the S morphotype suggests adaptation for long-term survival in the soil, whereas the production of abundant conidia by the L morphotype suggests adaptation for aerial dispersal to the phyllosphere. To identify genomic changes that support differential niche adaption, the sequences of three S and three L morphotype isolates were compared. Differences in genome structure and gene content were identified between the morphotypes. A &amp;gt;530 kb inversion between the morphotypes affect a secondary metabolite gene cluster and a cutinase gene. The morphotypes also differed in proteins predicted to be involved in carbon/nitrogen metabolism, iron acquisition, antimicrobial defense, and evasion of host immunity. The S morphotype genomes contained more intact secondary metabolite clusters indicating there is higher selection pressure to maintain secondary metabolism in the soil and that it is not limited to aflatoxin production. The L morphotype genomes were enriched in amino acid transporters, suggesting efficient nitrogen transport may be critical in the nutrient limited phyllosphere. These findings indicate the genomes of the two morphotypes differ beyond developmental genes and have diverged as they adapted to their respective niches.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3915" />
<meta name="citation_id" content="8/12/3915" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3915" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3915.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3915.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3915.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200553" />
<meta name="citation_pmid" content="30361280" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3915" />
<meta name="citation_lastpage" content="3930" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mana Ohkura" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Peter J. Cotty" />
<meta name="citation_author_institution" content="USDA-ARS, School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Marc J. Orbach" />
<meta name="citation_author_institution" content="School of Plant Sciences, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author_email" content="orbachmj@email.arizona.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7714-1352" />
<meta name="citation_reference" content="citation_journal_title=AMB Express;citation_author=B. N. Adhikari;citation_author=R. Bandyopadhyay;citation_author=P. J. Cotty;citation_title=Degeneration of aflatoxin gene clusters in Aspergillus flavus from Africa and North America.;citation_pages=62;citation_volume=6;citation_year=2016;citation_doi=10.1186/s13568-016-0228-6" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=1-410;citation_volume=215;citation_year=1990;citation_pmid=25595431;citation_doi=10.1186/s12711-014-0081-5" />
<meta name="citation_reference" content="citation_journal_title=J. Basic Microbiol.;citation_author=B. Barna;citation_author=E. Leiter;citation_author=N. Hegedus;citation_author=T. Bíró;citation_author=I. Pócsi;citation_title=Effect of the Penicillium chrysogenum antifungal protein (PAF) on barley powdery mildew and wheat leaf rust pathogens.;citation_pages=36164-520;citation_volume=48;citation_year=2008;citation_doi=10.1038/srep36164" />
<meta name="citation_reference" content="citation_journal_title=Clin. Microbiol. Rev.;citation_author=J. W. Bennett;citation_author=M. Klich;citation_title=Mycotoxins.;citation_pages=24-516;citation_volume=16;citation_year=2003;citation_doi=10.1094/PD-91-0024" />
<meta name="citation_reference" content="citation_journal_title=J. Stored Prod. Res.;citation_author=K. S. Bilgrami;citation_author=A. K. Choudhary;citation_title=Impact of habitats on toxigenic potential of Aspergillus flavus.;citation_pages=1-355;citation_volume=29;citation_year=1993;citation_doi=10.3389/fpls.2016.02052" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathology;citation_author=C. H. Bock;citation_author=B. Mackey;citation_author=P. J. Cotty;citation_title=Population dynamics of Aspergillus flavus in the air of an intensively cultivated region of southwest Arizona.;citation_pages=116-433;citation_volume=53;citation_year=2004;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Boeckmann;citation_author=A. Bairoch;citation_author=R. Apweiler;citation_author=M. C. Blatter;citation_author=A. Estreicher;citation_title=The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003.;citation_pages=1-370;citation_volume=31;citation_year=2003;citation_pmid=21949733;citation_doi=10.1371/journal.pone.0016163" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_pages=250-2120;citation_volume=30;citation_year=2014;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=F. Bringel;citation_author=I. Couée;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=6;citation_year=2015;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. C. Bruen;citation_author=H. Philippe;citation_author=D. Bryant;citation_title=A simple and robust statistical test for detecting the presence of recombination.;citation_pages=e90346-2681;citation_volume=172;citation_year=2006;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=D. Bulgarelli;citation_author=K. Schlaeppi;citation_author=S. Spaepen;citation_author=E. V. L. Van Themaat;citation_author=P. Schulze-Lefert;citation_title=Structure and functions of the bacterial microbiota of plants.;citation_pages=807-838;citation_volume=64;citation_year=2013;citation_pmid=23373698;citation_doi=10.1146/annurev-arplant-050312-120106" />
<meta name="citation_reference" content="citation_journal_title=Lett. Appl. Microbiol.;citation_author=K. A. Callicott;citation_author=P. J. Cotty;citation_title=Method for monitoring deletions in the aflatoxin biosynthesis gene cluster of Aspergillus flavus with multiplex PCR.;citation_pages=60-65;citation_volume=60;citation_year=2015;citation_doi=10.1111/lam.12337" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=E. K. Cannon;citation_author=S. B. Cannon;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=2011;citation_year=2011;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutiérrez;citation_author=J. M. Silla-Martínez;citation_author=T. Gabaldón;citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_pages=65-1973;citation_volume=25;citation_year=2009;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. I. Castillo-Davis;citation_author=D. L. Hartl;citation_title=GeneMerge–post-genomic analysis, data mining, and hypothesis testing.;citation_pages=146-892;citation_volume=19;citation_year=2003;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Morbidity and mortality weekly report;citation_title=Outbreak of aflatoxin poisoning–eastern and central provinces, Kenya, January-July 2004.;citation_pages=790;citation_volume=53;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=J Genomics;citation_author=K. G. Chan;citation_author=T. M. Chong;citation_author=T. G. Adrian;citation_author=H. L. Kher;citation_author=C. Grandclément;citation_title=Pseudomonas lini strain ZBG1 revealed carboxylic acid utilization and copper resistance features required for adaptation to vineyard soil environment: a draft genome analysis.;citation_pages=225-28;citation_volume=4;citation_year=2016;citation_doi=10.1080/0967087031000101061" />
<meta name="citation_reference" content="citation_author=C. Ching’anda;citation_author=J. Atehnkeng;citation_author=R. Bandyopadhyay;citation_author=P. J. Cotty;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=P. J. Cotty;citation_title=Virulence and cultural characteristics of two Aspergillus flavus strains pathogenic on cotton.;citation_pages=1-814;citation_volume=79;citation_year=1989;citation_doi=10.1094/Phyto-79-808" />
<meta name="citation_reference" content="citation_title=The genus Aspergillus: from taxonomy and genetics to industrial application;citation_title=Agriculture, Aflatoxins, and Aspergillus;citation_author=P. J. Cotty;citation_author=P. Bayman;citation_author=D. S. Egel;citation_author=K. E. Elias;citation_pages=1-28;citation_year=1994" />
<meta name="citation_reference" content="citation_title=Etiology and management of aflatoxin contamination. Mycotoxins: detection methods, management, public health, and agricultural trade;citation_author=P. J. Cotty;citation_author=C. Probst;citation_author=R. Jaime-Garcia;citation_pages=287-299;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Mycol. Res.;citation_author=P. J. Cotty;citation_title=Aflatoxin-producing potential of communities of Aspergillus section Flavi from cotton producing areas in the United States.;citation_volume=101;citation_year=1997;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Open Forum Infect. Dis.;citation_author=F. Courjol;citation_author=T. Jouault;citation_author=C. Mille;citation_author=R. Hall;citation_author=E. Maes;citation_title=β-1,2-Mannosyltransferases 1 and 3 participate in yeast and hyphae O- and N-linked mannosylation and alter Candida albicans fitness during infection.;citation_pages=560;citation_volume=2;citation_year=2015;citation_doi=10.1270/jsbbs.16005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=N. T. Perna;citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.;citation_pages=1;citation_volume=5;citation_year=2010;citation_doi=10.1038/s41598-018-23276-8" />
<meta name="citation_reference" content="citation_journal_title=Virus research;citation_journal_abbrev=Virus Res;citation_author=R. de Jonge;citation_author=B. P. Thomma;citation_title=Comparing the regional epidemiology of the cassava mosaic and cassava brown streak virus pandemics in Africa.;citation_pages=161-170;citation_volume=17;citation_year=2009;citation_issue=2;citation_pmid=21549776;citation_doi=10.1016/j.virusres.2011.04.018" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Delmotte;citation_author=C. Knief;citation_author=S. Chaffron;citation_author=G. Innerebner;citation_author=B. Roschitzki;citation_title=Community proteogenomics reveals insights into the physiology of phyllosphere bacteria.;citation_volume=106;citation_year=2009;citation_pmid=19805315;citation_doi=10.1073/pnas.0905240106" />
<meta name="citation_reference" content="citation_title=History of Modern Biotechnology I;citation_title=The natural functions of secondary metabolites;citation_author=A. L. Demain;citation_author=A. Fang;citation_pages=1-39;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci., India, Sect. B Biol. Sci.;citation_author=M. L. Dotaniya;citation_author=V. D. Meena;citation_title=Rhizosphere effect on nutrient availability in soil and its uptake by plants: a review.;citation_volume=85;citation_year=2015;citation_doi=10.1101/208181" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.;citation_pages=1-1797;citation_volume=32;citation_year=2004;citation_pmid=25553907;citation_doi=10.1186/1471-2164-16-1" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=K. C. Ehrlich;citation_author=P.-K. Chang;citation_author=J. Yu;citation_author=P. J. Cotty;citation_title=Aflatoxin biosynthesis cluster gene cypA is required for G aflatoxin formation.;citation_pages=307-6524;citation_volume=70;citation_year=2004;citation_doi=10.1111/j.1439-0434.2005.00974.x" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Microbiol.;citation_author=K. C. Ehrlich;citation_author=J. Yu;citation_author=P. J. Cotty;citation_title=Aflatoxin biosynthesis gene clusters and flanking regions.;citation_pages=43-527;citation_volume=99;citation_year=2005;citation_doi=10.1111/j.1365-2672.2005.02637.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=2069;citation_volume=16;citation_year=2015;citation_doi=10.1007/s00122-017-2943-z" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Faddeeva-Vakhrusheva;citation_author=M. F. Derks;citation_author=S. Y. Anvar;citation_author=V. Agamennone;citation_author=W. Suring;citation_title=Gene Family Evolution Reflects Adaptation to Soil Environmental Stressors in the Genome of the Collembolan Orchesella cincta.;citation_pages=1-2117;citation_volume=8;citation_year=2016;citation_doi=10.1186/s12985-016-0669-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. D. Finn;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_author=S. R. Eddy;citation_author=J. Mistry;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=44;citation_year=2016;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=J. Basic Microbiol.;citation_author=L. Galgóczy;citation_author=T. Papp;citation_author=E. Leiter;citation_author=F. Marx;citation_author=I. Pócsi;citation_title=Sensitivity of different zygomycetes to the Penicillium chrysogenum antifungal protein (PAF).;citation_pages=10-141;citation_volume=45;citation_year=2005;citation_doi=10.1155/2012/795697" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=L. Galgóczy;citation_author=T. Papp;citation_author=G. Lukács;citation_author=E. Leiter;citation_author=I. Pócsi;citation_title=Interactions between statins and Penicillium chrysogenum antifungal protein (PAF) to inhibit the germination of sporangiospores of different sensitive Zygomycetes.;citation_pages=768-115;citation_volume=270;citation_year=2007;citation_doi=10.1111/j.1574-6968.2007.00661.x" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_author=L. Galgóczy;citation_author=T. Papp;citation_author=I. Pócsi;citation_author=N. Hegedus;citation_author=C. Vágvölgyi;citation_title=In vitro activity of Penicillium chrysogenum antifungal protein (PAF) and its combination with fluconazole against different dermatophytes.;citation_pages=11-470;citation_volume=94;citation_year=2008;citation_doi=10.1007/s10482-008-9263-x" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=R. K. Garber;citation_author=P. J. Cotty;citation_title=Formation of Sclerotia and Aflatoxins in Developing Cotton Bolls Infected by the S Strain of Aspergillus flavus and Potential for Biocontrol with an Atoxigenic Strain.;citation_pages=940-945;citation_volume=87;citation_year=1997;citation_issue=9;citation_pmid=18945065;citation_doi=10.1094/PHYTO.1997.87.9.940" />
<meta name="citation_reference" content="citation_title=Aflatoxin: Scientific background, control and implication;citation_author=A. Goldblatt;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=S. Guindon;citation_author=F. Delsuc;citation_author=J. F. Dufayard;citation_author=O. Gascuel;citation_title=Estimating maximum likelihood phylogenies with PhyML.;citation_pages=4-137;citation_volume=537;citation_year=2009;citation_doi=10.1007/978-1-59745-251-9_6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. S. Guttman;citation_author=A. C. Mchardy;citation_author=P. Schulze-Lefert;citation_title=Microbial genome-enabled insights into plant-microorganism interactions.;citation_pages=1-813;citation_volume=15;citation_year=2014;citation_pmid=28220127;citation_doi=10.3389/fpls.2017.00023" />
<meta name="citation_reference" content="citation_journal_title=BMC Microbiol.;citation_author=M. Hilber-Bodmer;citation_author=M. Schmid;citation_author=C. H. Ahrens;citation_author=F. M. Freimoser;citation_title=Competition assays and physiological experiments of soil and phyllosphere yeasts identify Candida subhashii as a novel antagonist of filamentous fungi.;citation_pages=16;citation_volume=17;citation_year=2017;citation_doi=10.4161/gmcr.26408" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=A. A. Hoffmann;citation_author=L. H. Rieseberg;citation_title=Transgenic RNA interference (RNAi)-derived field resistance to cassava brown streak disease.;citation_pages=1019-1031;citation_volume=39;citation_year=2008;citation_issue=9;citation_pmid=22845735;citation_doi=10.1111/j.1364-3703.2012.00812.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of General Virology;citation_journal_abbrev=J. Gen. Virol.;citation_author=C. Holt;citation_author=M. Yandell;citation_title=Cassava brown streak disease: a threat to food security in Africa;citation_pages=956-968;citation_volume=12;citation_year=2011;citation_issue=Pt_5;citation_pmid=26015320;citation_doi=10.1099/vir.0.000014" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. A. Hongo;citation_author=G. M. Castro;citation_author=L. C. Cintra;citation_author=A. Zerlotini;citation_author=F. P. Lobo;citation_title=POTION: an end-to-end pipeline for positive Darwinian selection detection in genome-scale data through phylogenetic comparison of protein-coding genes.;citation_pages=17;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1765-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Trop Plant Biol;citation_author=C. M. Hull;citation_author=O. Bader;citation_author=J. E. Parker;citation_author=M. Weig;citation_author=U. Gross;citation_title=The Cassava Genome: Current Progress, Future Directions.;citation_pages=88-94;citation_volume=56;citation_year=2012;citation_issue=1;citation_pmid=22523606;citation_doi=10.1007/s12042-011-9088-z" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Jia;citation_author=A. R. Raphenya;citation_author=B. Alcock;citation_author=N. Waglechner;citation_author=P. Guo;citation_title=CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database.;citation_volume=45;citation_year=2017;citation_pmid=27789705;citation_doi=10.1093/nar/gkw1004" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H. Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=InterProScan 5: genome-scale protein function classification.;citation_pages=1384-1240;citation_volume=30;citation_year=2014;citation_doi=10.2135/cropsci2013.07.0482" />
<meta name="citation_reference" content="citation_journal_title=Arch. Microbiol.;citation_author=L. Kaiserer;citation_author=C. Oberparleiter;citation_author=R. Weiler-Görz;citation_author=W. Burgstaller;citation_author=E. Leiter;citation_title=Characterization of the Penicillium chrysogenum antifungal protein PAF.;citation_pages=959-210;citation_volume=180;citation_year=2003;citation_doi=10.1038/ng.3845" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=E. M. Kellner;citation_author=K. I. Orsborn;citation_author=E. M. Siegel;citation_author=M. A. Mandel;citation_author=M. J. Orbach;citation_title=Coccidioides posadasii contains a single 1,3-beta-glucan synthase gene that appears to be essential for growth.;citation_volume=4;citation_year=2005;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=N. Khaldi;citation_author=F. T. Seifuddin;citation_author=G. Turner;citation_author=D. Haft;citation_author=W. C. Nierman;citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters.;citation_pages=1666-741;citation_volume=47;citation_year=2010;citation_doi=10.1016/j.fgb.2010.06.003" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Kirkpatrick;citation_title=How and why chromosome inversions evolve.;citation_volume=8;citation_year=2010;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_pages=790-434;citation_volume=173;citation_year=2006;citation_doi=10.1111/j.1744-7348.1938.tb02354.x" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=M. A. Klich;citation_title=Biogeography of Aspergillus species in soil and litter.;citation_pages=6547-27;citation_volume=94;citation_year=2002;citation_pmid=25129495;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. A. Klich;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=C. Knief;citation_author=N. Delmotte;citation_author=S. Chaffron;citation_author=M. Stark;citation_author=G. Innerebner;citation_title=Metaproteogenomic analysis of microbial communities in the phyllosphere and rhizosphere of rice.;citation_pages=2060-1390;citation_volume=6;citation_year=2012;citation_doi=10.3389/fpls.2016.02060" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=A. Kombrink;citation_author=B. P. Thomma;citation_title=LysM effectors: secreted proteins supporting fungal life.;citation_pages=1;citation_volume=9;citation_year=2013;citation_doi=10.1186/s12863-017-0507-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_volume=5;citation_year=2004;citation_pmid=20071490;citation_doi=10.1099/vir.0.014688-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Lanfear;citation_author=P. B. Frandsen;citation_author=A. M. Wright;citation_author=T. Senfeld;citation_author=B. Calcott;citation_title=PartitionFinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses.;citation_volume=34;citation_year=2016;citation_doi=10.3835/plantgenome2017.03.0015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=342-359;citation_volume=9;citation_year=2012;citation_doi=10.3835/plantgenome2015.11.0118" />
<meta name="citation_reference" content="citation_journal_title=Adv. Exp. Med. Biol.;citation_author=J. B. Leducq;citation_title=Ecological genomics of adaptation and speciation in fungi.;citation_pages=1-72;citation_volume=781;citation_year=2014;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Lee;citation_author=G. A. Helt;citation_author=J. T. Reese;citation_author=M. C. Munoz-Torres;citation_author=C. P. Childers;citation_title=Web Apollo: a web-based genomic annotation editing platform.;citation_pages=R93;citation_volume=14;citation_year=2013;citation_pmid=24000942;citation_doi=10.1186/gb-2013-14-8-r93" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=S. E. Lindow;citation_author=M. T. Brandl;citation_title=Microbiology of the Phyllosphere;citation_pages=1875-1883;citation_volume=69;citation_year=2003;citation_issue=4;citation_pmid=12676659;citation_doi=10.1128/AEM.69.4.1875-1883.2003" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=X. Liu;citation_author=J. Fu;citation_author=Y. Yun;citation_author=Y. Yin;citation_author=Z. Ma;citation_title=A sterol C-14 reductase encoded by FgERG24B is responsible for the intrinsic resistance of Fusarium graminearum to amine fungicides.;citation_volume=157;citation_year=2011;citation_pmid=21436218;citation_doi=10.1099/mic.0.045690-0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. S. Lundberg;citation_author=S. L. Lebeis;citation_author=S. H. Paredes;citation_author=S. Yourstone;citation_author=J. Gehring;citation_title=Defining the core Arabidopsis thaliana root microbiome.;citation_pages=86-90;citation_volume=488;citation_year=2012;citation_issue=7409;citation_pmid=22859206;citation_doi=10.1038/nature11237" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=E. Lyons;citation_author=M. Freeling;citation_title=How to usefully compare homologous plant genes and chromosomes as DNA sequences.;citation_pages=661-673;citation_volume=53;citation_year=2008;citation_issue=4;citation_pmid=18269575;citation_doi=10.1111/j.1365-313X.2007.03326.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Machida;citation_author=K. Asai;citation_author=M. Sano;citation_author=T. Tanaka;citation_author=T. Kumagai;citation_title=Genome sequencing and analysis of Aspergillus oryzae.;citation_pages=1157-1161;citation_volume=438;citation_year=2005;citation_issue=7071;citation_pmid=16372010;citation_doi=10.1038/nature04300" />
<meta name="citation_reference" content="Maddison, W., and D. Maddison, 2017 Mesquite: a modular system for evolutionary analysis. Version 3.2. http://mesquiteproject.org." />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=D. A. Martinez;citation_author=B. G. Oliver;citation_author=Y. Gräser;citation_author=J. M. Goldberg;citation_author=W. Li;citation_title=Comparative genome analysis of Trichophyton rubrum and related dermatophytes reveals candidate genes involved in infection.;citation_pages=e0025912;citation_volume=3;citation_year=2012;citation_doi=10.1128/mBio.00259-12" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Marx;citation_author=H. Haas;citation_author=M. Reindl;citation_author=G. Stöffler;citation_author=F. Lottspeich;citation_title=Cloning, structural organization and regulation of expression of the Penicillium chrysogenum paf gene encoding an abundantly secreted protein with antifungal activity.;citation_pages=167-171;citation_volume=167;citation_year=1995;citation_issue=1-2;citation_pmid=8566771;citation_doi=10.1016/0378-1119(95)00701-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=A. Mathew;citation_author=L. Eberl;citation_author=A. L. Carlier;citation_title=A novel siderophore-independent strategy of iron uptake in the genus Burkholderia.;citation_pages=805-820;citation_volume=91;citation_year=2014;citation_pmid=24354890;citation_doi=10.1111/mmi.12499" />
<meta name="citation_reference" content="citation_journal_title=Ann. N. Y. Acad. Sci.;citation_author=H. L. Mehl;citation_author=R. Jaime;citation_author=K. A. Callicott;citation_author=C. Probst;citation_author=N. P. Garber;citation_title=Aspergillus flavus diversity on crops and in the environment can be exploited to reduce aflatoxin exposure and improve health.;citation_pages=7-17;citation_volume=1273;citation_year=2012;citation_pmid=23230832;citation_doi=10.1111/j.1749-6632.2012.06800.x" />
<meta name="citation_reference" content="citation_journal_title=Food Additives &amp; Contaminants: Part A;citation_author=N. J. Mitchell;citation_author=E. Bowers;citation_author=C. Hurburgh;citation_author=F. Wu;citation_title=Potential economic losses to the US corn industry from aflatoxin contamination.;citation_pages=540-550;citation_volume=33;citation_year=2016;citation_doi=10.1080/19440049.2016.1138545" />
<meta name="citation_reference" content="NCBI resource coordinators, 2017 Database Resources of the National Center for Biotechnology Information. Nucleic Acids Research 45: D12–D17. 10.1093/nar/gkv1290" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=W. C. Nierman;citation_author=J. Yu;citation_author=N. D. Fedorova-Abrams;citation_author=L. Losada;citation_author=T. E. Cleveland;citation_title=Genome sequence of Aspergillus flavus NRRL 3357, a strain that causes aflatoxin contamination of food and feed.;citation_pages=00168e-15;citation_volume=3;citation_year=2015;citation_doi=10.1128/genomeA.00168-15" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=M. B. Pildain;citation_author=J. C. Frisvad;citation_author=G. Vaamonde;citation_author=D. Cabral;citation_author=J. Varga;citation_title=Two novel aflatoxin-producing Aspergillus species from Argentinean peanuts;citation_pages=725-735;citation_volume=58;citation_year=2008;citation_issue=3;citation_pmid=18319485;citation_doi=10.1099/ijs.0.65123-0" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Food Microbiol.;citation_author=C. Probst;citation_author=R. Bandyopadhyay;citation_author=P. J. Cotty;citation_title=Diversity of aflatoxin-producing fungi and their impact on food safety in sub-Saharan Africa.;citation_pages=113-122;citation_volume=174;citation_year=2014;citation_doi=10.1016/j.ijfoodmicro.2013.12.010" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=C. Probst;citation_author=K. A. Callicott;citation_author=P. J. Cotty;citation_title=Deadly strains of Kenyan Aspergillus are distinct from other aflatoxin producers.;citation_pages=419-429;citation_volume=132;citation_year=2012;citation_doi=10.1007/s10658-011-9887-y" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=C. Probst;citation_author=H. Njapau;citation_author=P. J. Cotty;citation_title=Outbreak of an acute aflatoxicosis in Kenya in 2004: identification of the causal agent.;citation_volume=73;citation_year=2007;citation_pmid=17308181;citation_doi=10.1128/AEM.02370-06" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=C. Probst;citation_author=F. Schulthess;citation_author=P. J. Cotty;citation_title=Impact of Aspergillus section Flavi community structure on the development of lethal levels of aflatoxins in Kenyan maize (Zea mays).;citation_pages=600-610;citation_volume=108;citation_year=2010;citation_issue=2;citation_pmid=19674186;citation_doi=10.1111/j.1365-2672.2009.04458.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=C. Raeside;citation_author=J. Gaffé;citation_author=D. E. Deatherage;citation_author=O. Tenaillon;citation_author=A. M. Briska;citation_title=Large chromosomal rearrangements during a long-term evolution experiment with Escherichia coli.;citation_pages=e0137714;citation_volume=5;citation_year=2014;citation_doi=10.1128/mBio.01377-14" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Rausch;citation_author=T. Zichner;citation_author=A. Schlattl;citation_author=A. M. Stütz;citation_author=V. Benes;citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis.;citation_volume=28;citation_year=2012;citation_pmid=22962449;citation_doi=10.1093/bioinformatics/bts378" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. D. Rawlings;citation_author=A. J. Barrett;citation_author=A. Bateman;citation_title=MEROPS: the database of proteolytic enzymes, their substrates and inhibitors.;citation_volume=40;citation_year=2012;citation_pmid=22086950;citation_doi=10.1093/nar/gkr987" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=Z. Rengel;citation_author=P. Marschner;citation_title=Nutrient availability and management in the rhizosphere: exploiting genotypic differences.;citation_pages=305-312;citation_volume=168;citation_year=2005;citation_issue=2;citation_pmid=16219070;citation_doi=10.1111/j.1469-8137.2005.01558.x" />
<meta name="citation_reference" content="citation_title=Mycotoxins: risks in plant, animal and human systems;citation_author=J. L. Richard;citation_author=G. A. Payne;citation_author=A. E. Desjardins;citation_author=C. Maragos;citation_author=W. P. Norred;citation_pages=1-199;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=R. P. Ryan;citation_author=S. Monchy;citation_author=M. Cardinale;citation_author=S. Taghavi;citation_author=L. Crossman;citation_title=The versatility and adaptation of bacteria from the genus Stenotrophomonas.;citation_pages=514-525;citation_volume=7;citation_year=2009;citation_issue=7;citation_pmid=19528958;citation_doi=10.1038/nrmicro2163" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=F. Ryffel;citation_author=E. J. N. Helfrich;citation_author=P. Kiefer;citation_author=L. Peyriga;citation_author=J. C. Portais;citation_title=Metabolic footprint of epiphytic bacteria on Arabidopsis thaliana leaves.;citation_pages=632-643;citation_volume=10;citation_year=2016;citation_pmid=26305156;citation_doi=10.1038/ismej.2015.141" />
<meta name="citation_reference" content="citation_journal_title=Proc. Jpn. Assoc. Mycotoxicology;citation_author=M. Saito;citation_author=O. Tsuruta;citation_author=P. Siriacha;citation_author=S. Kawasugi;citation_author=M. Manabe;citation_title=Distribution and aflatoxin productivity of the atypical strains of Aspergillus flavus isolated from soils in Thailand.;citation_pages=41-46;citation_volume=24;citation_year=1986;citation_doi=10.2520/myco1975.1986.24_41" />
<meta name="citation_reference" content="citation_journal_title=Frontiers in medicine;citation_author=D. Sanglard;citation_title=Emerging threats in antifungal-resistant fungal pathogens.;citation_pages=11;citation_volume=3;citation_year=2016;citation_pmid=27014694;citation_doi=10.3389/fmed.2016.00011" />
<meta name="citation_reference" content="citation_journal_title=Clinical Microbiology Reviews;citation_journal_abbrev=Clin. Microbiol. Rev.;citation_author=H. Schmidt;citation_author=M. Hensel;citation_title=Pathogenicity Islands in Bacterial Pathogenesis;citation_pages=14-56;citation_volume=17;citation_year=2004;citation_issue=1;citation_pmid=14726454;citation_doi=10.1128/CMR.17.1.14-56.2004" />
<meta name="citation_reference" content="citation_title=Aerial Plant Surface Microbiology;citation_author=J. Schönherr;citation_author=P. Baur;citation_title=Cuticle permeability studies;citation_year=1996;citation_doi=10.1007/978-0-585-34164-4_1" />
<meta name="citation_reference" content="citation_journal_title=Applied microbiology;citation_journal_abbrev=Appl Microbiol;citation_author=H. W. Schroeder;citation_author=R. A. Boller;citation_title=Aflatoxin production of species and strains of the Aspergillus flavus group isolated from field crops.;citation_pages=885-889;citation_volume=25;citation_year=1973;citation_issue=6;citation_pmid=4197766" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=M. F. Seidl;citation_author=B. P. H. J. Thomma;citation_title=Sex or no sex: evolutionary adaptation occurs regardless.;citation_pages=335-345;citation_volume=36;citation_year=2014;citation_pmid=24531982;citation_doi=10.1002/bies.201300155" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Soderlund;citation_author=M. Bomhoff;citation_author=W. M. Nelson;citation_title=SyMAP v3.4: a turnkey synteny system with application to plant genomes.;citation_volume=39;citation_year=2011;citation_pmid=21398631;citation_doi=10.1093/nar/gkr123" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.;citation_volume=22;citation_year=2006;citation_pmid=16928733;citation_doi=10.1093/bioinformatics/btl446" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=E. H. Stukenbrock;citation_title=Evolution, selection and isolation: a genomic view of speciation in fungal plant pathogens.;citation_pages=895-907;citation_volume=199;citation_year=2013;citation_pmid=23782262;citation_doi=10.1111/nph.12374" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biomolecular Techniques;citation_journal_abbrev=J Biomol Tech;citation_author=B. J. Stucky;citation_title=SeqTrace: a graphical tool for rapidly processing DNA sequencing chromatograms.;citation_pages=90-93;citation_volume=23;citation_year=2012;citation_issue=3;citation_pmid=22942788;citation_doi=10.7171/jbt.12-2303-004" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=R. R. Sweany;citation_author=K. E. Damann;citation_author=M. D. Kaller;citation_title=Comparison of soil and corn kernel Aspergillus flavus populations: evidence for niche specialization.;citation_pages=952-959;citation_volume=101;citation_year=2011;citation_issue=8;citation_pmid=21405994;citation_doi=10.1094/PHYTO-09-10-0243" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=W. H. Trejo;citation_author=R. E. Bennett;citation_title=STREPTOMYCES NODOSUS SP. N., THE AMPHOTERICIN-PRODUCING ORGANISM;citation_pages=436-439;citation_volume=85;citation_year=1963;citation_issue=2;citation_pmid=13994057" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. G. Tringe;citation_author=C. Von Mering;citation_author=A. Kobayashi;citation_author=A. A. Salamov;citation_author=K. Chen;citation_title=Comparative Metagenomics of Microbial Communities;citation_pages=554-557;citation_volume=308;citation_year=2005;citation_issue=5721;citation_pmid=15845853;citation_doi=10.1126/science.1107851" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=T. R. Turner;citation_author=E. K. James;citation_author=P. S. Poole;citation_title=The plant microbiome.;citation_pages=209;citation_volume=14;citation_year=2013;citation_pmid=23805896;citation_doi=10.1186/gb-2013-14-6-209" />
<meta name="citation_reference" content="citation_journal_title=Clinical microbiology and infection :  the official publication of the European Society of Clinical Microbiology and Infectious Diseases;citation_journal_abbrev=Clin Microbiol Infect;citation_author=M. F. Vicente;citation_author=A. Basilio;citation_author=A. Cabello;citation_author=F. Peláez;citation_title=Microbial natural products as a source of antifungals.;citation_pages=15-32;citation_volume=9;citation_year=2003;citation_issue=1;citation_pmid=12691539;citation_doi=10.1046/j.1469-0691.2003.00489.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=B. M. Vincent;citation_author=A. K. Lancaster;citation_author=R. Scherz-Shouval;citation_author=L. Whitesell;citation_author=S. Lindquist;citation_title=Fitness trade-offs restrict the evolution of resistance to amphotericin B.;citation_pages=e1001692;citation_volume=11;citation_year=2013;citation_pmid=24204207;citation_doi=10.1371/journal.pbio.1001692" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=J. A. Vorholt;citation_title=Microbial life in the phyllosphere.;citation_pages=828-840;citation_volume=10;citation_year=2012;citation_issue=12;citation_pmid=23154261;citation_doi=10.1038/nrmicro2910" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Wang;citation_author=D. Coleman-Derr;citation_author=G. Chen;citation_author=Y. Q. Gu;citation_title=OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species.;citation_volume=43;citation_year=2015;citation_pmid=25964301;citation_doi=10.1093/nar/gkv487" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=J. M. Whipps;citation_author=P. Hand;citation_author=D. Pink;citation_author=G. D. Bending;citation_title=Phyllosphere microbiology with special reference to diversity and plant genotype.;citation_pages=1744-1755;citation_volume=105;citation_year=2008;citation_issue=6;citation_pmid=19120625;citation_doi=10.1111/j.1365-2672.2008.03906.x" />
<meta name="citation_reference" content="citation_journal_title=The American Journal of Clinical Nutrition;citation_journal_abbrev=Am J Clin Nutr;citation_author=J. H. Williams;citation_author=T. D. Phillips;citation_author=P. E. Jolly;citation_author=J. K. Stiles;citation_author=C. M. Jolly;citation_title=Human aflatoxicosis in developing countries: a review of toxicology, exposure, potential health consequences, and interventions;citation_pages=1106-1122;citation_volume=80;citation_year=2004;citation_issue=5;citation_pmid=15531656;citation_doi=10.1093/ajcn/80.5.1106" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=G. Winter;citation_author=C. D. Todd;citation_author=M. Trovato;citation_author=G. Forlani;citation_author=D. Funck;citation_title=Physiological implications of arginine metabolism in plants.;citation_pages=534;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.00534" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: a program package for phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Yin;citation_author=X. Mao;citation_author=J. Yang;citation_author=X. Chen;citation_author=F. Mao;citation_title=dbCAN: a web resource for automated carbohydrate-active enzyme annotation.;citation_volume=40;citation_year=2012;citation_pmid=22645317;citation_doi=10.1093/nar/gks479" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=B. Zhu;citation_author=M. Ibrahim;citation_author=Z. Cui;citation_author=G. Xie;citation_author=G. Jin;citation_title=Multi-omics analysis of niche specificity provides new insights into ecological adaptation in bacteria.;citation_pages=2072-2075;citation_volume=10;citation_year=2016;citation_doi=10.1038/ismej.2015.251" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Aspergillus flavus , the primary causal agent for aflatoxin contamination on crops, consists of isolates with two distinct morphologies: isolates of the S morphotype produce numerous small sclerotia and lower numbers of conidia while isolates of the L morphotype produce fewer large sclerotia and abundant conidia. The morphotypes also differ in aflatoxin production with S isolates consistently producing high concentrations of aflatoxin, whereas L isolates range from atoxigenic to highly toxigenic. The production of abundant sclerotia by the S morphotype suggests adaptation for long-term survival in the soil, whereas the production of abundant conidia by the L morphotype suggests adaptation for aerial dispersal to the phyllosphere. To identify genomic changes that support differential niche adaption, the sequences of three S and three L morphotype isolates were compared. Differences in genome structure and gene content were identified between the morphotypes. A &gt;530 kb inversion between the morphotypes affect a secondary metabolite gene cluster and a cutinase gene. The morphotypes also differed in proteins predicted to be involved in carbon/nitrogen metabolism, iron acquisition, antimicrobial defense, and evasion of host immunity. The S morphotype genomes contained more intact secondary metabolite clusters indicating there is higher selection pressure to maintain secondary metabolism in the soil and that it is not limited to aflatoxin production. The L morphotype genomes were enriched in amino acid transporters, suggesting efficient nitrogen transport may be critical in the nutrient limited phyllosphere. These findings indicate the genomes of the two morphotypes differ beyond developmental genes and have diverged as they adapted to their respective niches." />
<meta name="og-title" property="og:title" content="Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3915" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Aspergillus flavus , the primary causal agent for aflatoxin contamination on crops, consists of isolates with two distinct morphologies: isolates of the S morphotype produce numerous small sclerotia and lower numbers of conidia while isolates of the L morphotype produce fewer large sclerotia and abundant conidia. The morphotypes also differ in aflatoxin production with S isolates consistently producing high concentrations of aflatoxin, whereas L isolates range from atoxigenic to highly toxigenic. The production of abundant sclerotia by the S morphotype suggests adaptation for long-term survival in the soil, whereas the production of abundant conidia by the L morphotype suggests adaptation for aerial dispersal to the phyllosphere. To identify genomic changes that support differential niche adaption, the sequences of three S and three L morphotype isolates were compared. Differences in genome structure and gene content were identified between the morphotypes. A &gt;530 kb inversion between the morphotypes affect a secondary metabolite gene cluster and a cutinase gene. The morphotypes also differed in proteins predicted to be involved in carbon/nitrogen metabolism, iron acquisition, antimicrobial defense, and evasion of host immunity. The S morphotype genomes contained more intact secondary metabolite clusters indicating there is higher selection pressure to maintain secondary metabolism in the soil and that it is not limited to aflatoxin production. The L morphotype genomes were enriched in amino acid transporters, suggesting efficient nitrogen transport may be critical in the nutrient limited phyllosphere. These findings indicate the genomes of the two morphotypes differ beyond developmental genes and have diverged as they adapted to their respective niches." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eJpJksIsaPv1eSQCybqrKJo5iuylWl3uCT32P4GAUhk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491669","apath":"\/ggg\/8\/12\/3915.atom","pisa":"ggg;8\/12\/3915","ac":{"\/ggg\/8\/12\/3915.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3915.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3915"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3915"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3915":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PB+Il4Be2OFid1plAjgbWLS1Dj3t463NfzrWYzPr0E\\\\\\\/p+vuq3\\\\\\\/4xekYBvopTwZVIWfFLIeY59wj\\\\\\\/ukKQ08ICL+4ZGS5DMkLlDVID2LsDpnWPY4Sw7hxTCJGq7ZB0lUnIqLPGZ\\\\\\\/7lWdcWysRM0MDGYLaIta4a\\\\\\\/fdPwjWkJ5YflUdrf+aRn2+rfEyqxoOJGeNjs5f5bZyvjE\\\\\\\/rOI2pT2J8F5CxxOL\\\\\\\/YiYSOQ01H5dABlQ3Lb+7TcZDzrTibBT9TsVJS2R+VT2uIkhCe3hriYQTZ\\\\\\\/u+5qA7hHSvWYOeA05N1v29GB++4kruhPElbcY3jRuYDCJ9HihJKvhcNidFTQh29mt\\\\\\\/TGUS\\\\\\\/paZj5KG3m4LRwmm\\\\\\\/fj9sI6O\\\\\\\/qPLn9Dv6wKko6x82ZL3+z99utaKtZcbRkkdjZWTHi381p\\\\\\\/+4weMiTIBWKc7eZ2g8K9VH45AExvGvvnsuCSKsOKO5Lf8QbMsKkgDQ1Iu2NuObXxl8I4Xqn9x56ubkD30FDNoK5eurfdPYkp0u9iCt\\\\\\\/Bc4Fnq5s0pDCjFuQ7lFSDA22V5p7AZdyW9ScjQ98SADbiDdxobQXuVeqnzI7zsZiYM4PadznRLDxTXJWnz3pYYVdBUtHzLLbUbu67\\\\\\\/m65Vce3gdTgYDzpxN6o8\\\\\\\/853poaSXgWBWzUnnmdrmxuEXZQKKcHf6m0q4f0V72dAng2OabxeZDe1DJawC86G2YvdTU4bmOHqMHM+ZTX4ABPPWd8KUj6gnmgm4kEmy0brqmkIwBTIr\\\\\\\/Iu5YQ1xnmjkZ2uO8o1Eu0JWCqQN1AIuoYDgDMGp93GoTef7BZS5AUREypBuKH43mp5Wow\\\\\\\/MrJR2j2APVW03jfUBE+gfT4XwKOgMFzANHuuI3+baNxWZ7WVt0UCRAJ8TpPlOm\\\\\\\/HErR8xelCgIYnYIJbCCblozj9bNICJknPNDTFOtCra07\\\\\\\/kIh+z+8y3RJceipkJ+z2M2XbDSRgegH3wyrjmIyFBB4SwHz7deqviUiKIbUvD+bO4YUyge5uLBE2TdjGmakupI10oDSNlBFb8PN4mxsSSggiSdBDVwRTY7WE0MQCXUFDi0FvexwXxTNZN12Aw6Dif2bbb1jsCQ8XQkcP9lgMpmpBmr7GTI0xT94xyGdXx6aVcoJFmy234+8KoopOqD1dzT0WkfDc2s5FbYIj\\\\\\\/9c61OMRwcOWSzp5XYg1mJfYD5ylM4kBJ9LfZtJxSk6TXd\\\\\\\/HtVnKKrcSLtmf8TQHSXrQl2Ap+UX6JA+KX4L6Srj\\\\\\\/5EpssHLKwG5\\\\\\\/bpr281f5+N3J\\\u0022,\\\u0022iv\\\u0022:\\\u0022VcVf30GmAgPafW9xJkcaSQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002274536f67bfbd2c160b0ae6b3d5d7db2f4e90e8212941ca3281657e0e1d9425bc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aXvMtRIFk77jFM7PWLooZUZDtIN5Jeh0RDaAIAkJlVlW0ICe6OJ6KLQt4lGQpRF1cDvD\\\\\\\/+w5u\\\\\\\/cflDsF2hDFNzgosBTQwcnShVg5NBJkFS+9LwE\\\\\\\/Xl0Et2dxk3QRNLJY92Sj8IHmKUi9GR9Ri4SI8jQOfxRLJMUtLONLnf+yWEfn+6PfyM+GwXptKMTpZXpDaER15H\\\\\\\/+HEVbyXK1llm4PjhOcQyBkqptLzc02VTsRLksUYNeC3TJc9pkX7ZyvPladCqeEiwNtjUP31exaTS9kj7nQhAlg227mb73UsF1TQ12vQvw8SzECFlXEZ4IBxT0Xhj1qNcsSJfxm\\\\\\\/87i840yfx3vmPChhwIzy8ACTi5H3nXFjDG6Uvyrr27eVMnbjBRt1PxCi2fzamaDSWzVmos7TM5gzYIFbpQjWyNtH\\\\\\\/REnMx30C\\\\\\\/StYaS6TWJeli8C3\\\\\\\/bM0I8YOiMsXK0MF4vciW6yhLQiFbymi+9YnUKUQR82gK62Lj67e27+RdRa2wZ9sUi0SPtIMSflygqfymCpmEfBEPKjsVkDqVPmxoo2iuyIGjwY\\\\\\\/HnALRqwLQfcAfHxMGY8YXA3U74GD5t2I4zHy7BP8RHN44mXz+sRoqsLQvEJLKQjhAjkmJ+RDEQ\\\\\\\/2gT4eupQsZUWcHBB8jfghReg+HUy+n8\\\\\\\/ZHtG8Y9SRCsdCDkKePJ2bO2uXyefXGL9lMx\\\\\\\/ZHAAQPwNCEk+f8KmmFLLvGuf3HyWynWoC2E7CyjD4WcS5IbsCwZp+8OGz35\\\\\\\/tDycZBh2CBGU81KjeeXrKW\\\\\\\/wgSh4AmMG4rKYvXK6SSLu6agbWZmH4Fe\\\\\\\/fHsY2vBrBwVtdpS0au\\\\\\\/a+Snb4ochdjoE0nV4b8fmdgXjuOox\\\\\\\/WjrW3TX93h5bCnLsNZd3yhaVHany800Kw2hk\\\\\\\/vlgiPva6wzBWL6u+0qHyw6QepUjDAv+m9YHgLyl0jQB2AHcs0EYeDDfp\\\\\\\/K4LpoD\\\\\\\/OWhCZWO+nr9LjZNO1mvZ5SBV190ribuvK7Jadn4dKlCP6LJOrr\\\\\\\/zzsAt+xIeugla19A1s3j7M6tcnLT5JigPcY9pA4rRMEAgzPY+T1lG20AY3Dv7urNNkRK4jtjSk7DcBx+8uiyngkTv+qzxQzAi+w3\\\\\\\/OkJw4iZdhhsJDL8SPLtrdhYF0ls1PQq50qwUsQejnCqIYrSeb1ItjA5gTF0YXCIG5isoFIyTDONA3dYQafgXfp6pmyFcIc3oNf29esnY7IndHL\\\\\\\/43U2BJUgaR6f3GDwfUeqlsU4majIb8wfJAEF+fy8I4b58uvGX\\\\\\\/6EqLg5L6g3J6ZYyiu7xCdx797T5P60IduMrcwoYvJL0\\\\\\\/vBJAriZCCXIIWFHm59pBVK5XmjPJ\\\\\\\/R\\\\\\\/78zvkHnHs7MXcFP\\\\\\\/Sx3EVRwxu2CTplq4cgjV7N0HZpJG6AuGGVX2e7Do3xD\\\\\\\/VXhDFWsTnx8K5r08Gl0e7hmT+QZhco1xt7rblxFCySNytARRbYnGS82ORkoWllxOjytG7RfiYpncUVz7laM58DYgRG9QA0GgJ+yXJwM7nAALwJKqOXMxbK7JsGjgG+dLWrUE7Yr\\\\\\\/\\\\\\\/eOAuVIyjd2yQzuI8fpLKJDupYfS1RDAxW7O6aLqacC50VbDgH4mXwCOl0ejbtiehnCBZswT71OIMPcRkUNkkwNLCmjtSEE3SzkdSGQNBFuAlqrJ1IlD8l5Evx+fjtPaAudOsV0z1QCqVQ+pK3JeZPEjHoMuTe6od7woT9njyHiFi5k7XBco8V9pvO0Ouyt4YEKbErng4QlSJ1B6QKpWWD1b5BXFhsV\\\\\\\/ZYOfxnujC\\\\\\\/sxaH4WImfiCpp476iaWBsLY2sO3fgsw5XbQxB9HA8UkkrDEClcyGfApTkCR1tHQNn57Mg2GwaUThT\\\\\\\/UJOmf0udLpbBLug8mLY96XYwMLvmS26trFAZSIx3eSVmcQveEFVKqY5wIkSV\\\\\\\/mChK5fqndLNWlIzKlzj7wefOc7D4uqaJ6YVV367PuTZbkQnBdJZQwdmVlHTxyHCKzC2KkVyTj4Mm7ijTxCqmsFwVA\\\\\\\/0Zry0Bbq1WPwm3ximtmZsNgUSNK7p3FqzQ8imH76o3UyqWKJrQsW3t8EsgVl8GU79MrgPI5E3zv1l96iG4Q5oXqWFsMLOkLjZ8LvgiocSCwGmhLjOdeAshB9Nuv4b+3nKarkNUU4chPEK+6EOntRdHkHVdJcN56c1xhVqTkAQBexNLsywandgNsqcf2mCbB83xS6f5gGj9TUWkQOZCnYW6TosKcTmorWK86zyvHOtyJv5mNMafFlPWP9B3pFxcQfw1AidLgEGhw\\\\\\\/yWS8zJXl35kNV\\\\\\\/htmJmxZBNzioFa5TajqNq1OB0REztvO9JG98OshefDW2qodRDTViXLn6USpDVC4kI2tt0t43iqIEN\\\\\\\/E\\\\\\\/ElrrMi4tLGahUYdojjtfPI0meP4W5qWFEl22jgJbZtv\\\\\\\/xJ5fqt4cgjPV5lsNVGVI475DjD5oJirhMC1dI75QVrrufVYu\\\\\\\/4PjC7UCfWWxFFyXqyWJQ7Fzwm6E65rirgkYrOGpgp5v5ebyP46TamgKCIJSOoYmywuC2DQldmcK7CxRcxDqs4fjZOmQXBj0GhxYBd8sG0tQ4hSrjWuVHUMAouZFgkE6XWQAjbFPikm+Z9n\\\\\\\/hpd4HDZNUi2bMNy1EdPqREXe3gHh5UAQzKutqcyaScxz5FYAMhARi45NKYf2HaKIcP1q+Qe9PUE5Gm8lMWFH1QCSI4dpvrKPEN35V1nHCry\\\\\\\/fzXjP7X3u5kbq3Af+t0ArEaNkBeuHcmaiuNPeb+puGAIOKc\\\\\\\/LNPxj0M5X8mEBvlRXRtgHi84gl3qvj4U4YbGn2us3DTUyO55baA+XiXDg1EqE+OIc979Io6KCEyKxQ+mn0R\\\\\\\/L0IjKIzVRaZwMfTMf54ngkl5GC87nsrRrgY3Vn1m9JlJ8yWTjDtJTaOY0WJwbeoN0lUAC66P870dSVTqAdSg0wGHBySQbiEpRMQI9fNf\\\\\\\/6uQk\\\\\\\/WjSrKm6JfLbHCcAE8CbdLLous3wKZ8W7TjEtvLkq0oBFmEDB868szfLBtFN2\\\\\\\/6LJ8Pv2nkjMdWhWnHqhRhH1DgXyJEGnv\\\\\\\/Tl8DACXklH+HdPIhF1hvViYeiH75VWBov+D5FGu+sxcTF5b+ISU+khXxquhDfc7fw5T9zXNxthXbIDttUSXv6kcfeY2664ODejwdRVtzYEUJa3jFbVW21YfZTzAJjE37EunjRyHx1gHXR2rCMNSm9RNDQyIMzA8qoC8PHzQbamUIne3X3kGenUHU1VVv2CRbLNPYSYVYJEu7AdFUW8suN0wkpf5rvEis5vsw\\\\\\\/g8vZkIrFPKylDoiaq+tQvrj3YRMJZoZsLbZloLxL4rYuiNhwl4Uj6ic4RUwFA\\\\\\\/mxlfKtzQ80JfpI6lhrz99JiuKUPYG6EemUZRlnnVA0Ur+UcGiYWlg72otBOQ1j79+KLeA1IMRQsM9v5tgXL+2xIxvK+VfXotMppNHKL9ECkCeVGmRTpdjksZsHydr\\\\\\\/jdxSZpytQMvztgHXb5jeLxptGjiThxrd+9f818Ele+X7zfKXhrCnZYTKZTZbCVZnqKjaMzLYSxl9fn5MSo08CgjyIJPBoLnibaX6U0O5mKvuKTCjzJohCpJ3PCsS4\\\\\\\/AQBpxP2irSpgkyTo273TeSJXDcWMtROwcDuJ\\\\\\\/N7NVWXhzQ3FnrUn1RtiCRO3AmI4KUfpGL5Cs2HXwgZsZQ6WsbEGSuXd8EIMGmwHO\\\\\\\/4F3v1zWSnakf3Vr2q7uVI+O69tTheH2LKwsrUaR5ZLT0mn9NaPxOPhGQERe70yyruR3yX36tMmD25SAvX\\\\\\\/lrvIDBPUx6n6QEgeclgmyzPZLmomC9OSuxHJMn0EDes6aCtMK8ljzP7WIe4zN2InokGJWT\\\\\\\/I6drluUHeLrt8VJ7iFmSSRVq+9ZgBtXXufcnfxZ1hmk9Sm+jcvc\\\\\\\/j7xl5CpE9GcDHvffB7Vq1d\\\\\\\/wrYksP2rj2MryIIrZ1r60xHFJFKiNZ382R7mlh+LohssbpQ46WNTJU6BkO4nVpCiGiLnIDpMOemGWP2OVR2KSntqzBEKiSirsYN7SXBHl475Tp+lM71iGCK1o6ZD+O0hVCqQxEjqYo3hRqVWBU7w+Rv+BZTBSW84tuTECFIGQJ8BtQ3Iaexf+o8o8WEqLMVk+s29PscLgzvGP+EicHrzJPVGq5dzS7yEJ\\\\\\\/h7yFFDkio8agOgp8xbH4mnMx4zGzxiILqMEilF8jqU7xMfpC9Hh\\\\\\\/wcbkvoQbFqnM\\\\\\\/sbhudw07qq8Km+SaFyfcTu6xRiB5tbgdSYfdKfNT9azbfFRuakWfN6LcXYwYNCLjpIVcdmZDpMqDu7be6TjdjrMhU\\\\\\\/\\\\\\\/GuTs1dY+j\\\\\\\/AwtZFocNb8FkWoKkUHINK\\\\\\\/UTwjT00Sm\\\\\\\/pMDMut0u5Z1Mm1\\\\\\\/NKHZ3FYdytwiO1lfONOCrihi8tCcx9JuhvWD1NmBgMEjhP4VwR2OfGptGjK\\\\\\\/1hWx7NPNmorPdKn\\\\\\\/XVI+zAOcpKJB\\\\\\\/usYz\\\\\\\/kYnvArJ7XZnfrNthJIPi30GOMuNQAVjezzgssmTHhox9ws\\\\\\\/Zh5Rc+fppRMv8XG9ewscAQAfiqRfheWUGh+FlV27y1iW4sIWEeiPCymViJ6an3m1ERyjwn6s3p3xpXfnpTHr8P1v6sTqylg7KBJSPEjtBbhFw9A8SAHweqC0L9NfN0ldkT3BhBNHbXVETR++CKHTRsAuGmcleh9ZhqV4KJGq9ZdGObkPBSm7DOxFprMvnXjhE+l5ZBNAqHridrfk8g\\\\\\\/6dkYPaUTn4QiV2qHyvn\\\\\\\/\\\\\\\/tXsNvVFHtiZfI5Tc2J8WV7KtDMee9llGPK9mHu1tiOq+JKEXjMXW81CT6okVRWSLk2HN6KyEeti9XYn4Io9\\\\\\\/6W3cQCSBL26oi6t3LrCbtExM4VjxYtATs7oziEvlBl1t2bkw5co\\\\\\\/01DW3FFojqy3ifSaAQUR0VOTNqeyJcWmHw8DcHRbhfeTBRm3PNNAivbJatW\\\\\\\/RVOiMjyOY41948ODeSMHSjGCYiWjsYn7V85vJspmEBPbL9h\\\\\\\/xvHll3qZAV+IZvXoR9yFFtGf7akX6DsqYbq9N1e+s\\\\\\\/Imzfn0s44ckV542jHW9uzzt42kmjYryYqFDX3MQVpgHFCEEEer4eFmPeRzLOnLFLW43vfezRG12QgAyHURuwCCoqIWzylTsxoQH1WZ8l0A\\\\\\\/lsfKNxFH0xn45ctr+yMNluvlFARTB5uckQFZtzMxvv8LUNm8hUyyPGoaR2OCByiYUchlOBsz6zyoU6vookSYCz3ETatzRM4JUitt3kmXebDA18gaZUyWZBIkDVW1rKHevhC6UE3mBcJREZtQX\\\\\\\/faoyo2u4v1W3MYT4I1A1WQf9cKdLU3triOtovyCToLU63tZPUmh1DXmc+tsWugaUOKoY0ILe+Km66ukTfV8nvelYJHWZeFOxuIJJ34oFXcagII+g8wsi71vGHRYS2a2S9AmU1vOuhCV+mt+3qOwZh33mjvsyixdpbQTa122oYyC7OR25QgHmb5csfX6jrscL6sNTrcx+p4CRXWAvZl0Rh2a4rxY\\\\\\\/2dVTwojlMxOJTc2MZ6K0E8YIVS4KIQ\\\\\\\/3pC1NFT8uLud5YbnFfgEfv5kQVq0yCrcSijFYZyqyS9kXlnj7c6Y29v3dj9Awu+y13uOTFBuindl9tVyzN8Zcf4wunjtgxQP458N8dWozaUATEBWxATgcffo9jZiS5l5txNiqntV3oaf2dz9nnbFdjiLd4OMP9Sx5Nz9YGD2Vf9+LY4+jlzOe+dcR1miU+SWRKrkGwspNGTLMpIqDt3wjEYXv9cMYvpMvOwD4Jk1550bEPbGvhYtCG87pDdu2ioldagDTzy0mkzoe+7PlKqY1UtLVSb5dIULftU7TQDTWw\\\\\\\/soYJl+IzTvUoYh57FrJXIYH1+W7XAAupz8ojfqDCOpx2EcN1T\\\\\\\/PUID4YZz5dMvrQzzPaHZwBQKg\\\\\\\/CtQsdV+fTf2H16vF+s406ijR8RoVvO+g2\\\\\\\/8qpxAfrAh0etkcrJ7LSqTWRlYczYo1SU3jyAOfKCYwP9wtO9BW+8NY8NqY5UPOuqBs7iGu7+sXe1yLFeUhc2Ozq7xnDQWwLH3n2SDv3sM7DGFK+diuln8Ag3D0Zs9qlMD26TScVcXY\\\\\\\/xk+qGhfbFfAppQe6yvUJlBs5ey02huE+s6Q649tvapKDkWrgFx74o409Iay+utp3cuBpDRMityPtrEHsesapWNGnOV5aJlIwu8Ae+NkFOgDHJ7bKILA7vbFWhg+nfr3R4Zz1B76BsQ+vfkf6jslU3xFzMGmrJ06eH4RD4HX25WB4jljjtqTpCiH7KwseLb1YfTchg7elPD+vHwqFvKZKC30OQvcyMvA1kC8bX1N4P9OsT6dl0hlBdRLI\\\\\\\/v03tf0QeGjbeyRBabMbc6FCTnJ9LNNU0ubcgCPod19efk+R0EmdmImF61uhCatQCJHtvtDam5ELCIDg0QLMj9VqIgJ1ZA2Ng+xa7slBUI6hliKBPZBaeKP\\\\\\\/Mea6XapQsosCkXD8YLzyXGQ57GQxL8JbnOaEpOCxlfixHTzABjLUbW2zKXM9UYqG7UhD+5Lhgdwc23n32fOi5vVV8BtoFEpEMbOAg7X8YbeXtGEAW4g81xBuDNUT0u5N1C2q2nhDB3XNoGeqnW80qUnCyeWyzZjwJUuKeyaedO7MWSJhFNel38DpW1StnWidj0m0Y253hZIsjH\\\\\\\/humcTGVc6CY7sbe5JKvoS4ir6sAsOy5LVTIFyLV+BuUi2vdsy3Sm2yeAzFQbkyJeHkwD7T07LPZY7htH2T3rD1b8iuM\\\\\\\/goJcSgt7OlR1xgR9X2in9KwKlfiaY0VQQwD\\\\\\\/z8U1yYveC81eYZVwSazJNlGyHR7DExlWg9p+mW4rB92A6fhuNInEqhIGjAizPux+Y7iIAxW\\\\\\\/Pb80XeQSX+YlG6KZQSoB994CFG+v5WcqZkCwpFvUFxsg39QIxW7SZTdB9UQ5bsX3U2\\\\\\\/i7WDnEZWf3OlKA7mJQ53\\\\\\\/0LHdeAZ6fiEuZ4+ARCfcMqDiO6bqx8rfXThFHO52rcXzs0V3nK43LYWhG\\\\\\\/FcwFfXy5CbBBTfksfhD\\\\\\\/cFHW8yEhhAHlbkWUOOnZMZc+yTK57x7RdPbTdT6jGQqGsbmcNuB++O45N6UJiy0EBEFWxqUklxxCS92eF4lKD26WXBBgv18Qmvpjsi8TIFj4ofadLTDlQV5312CgrXRD7I+CTDixlPT0ne2jk7vHRpKng\\\\\\\/Shhpg4FOU77TCtucDU8iW1jAxbD0x00Zyeg+klVXdmImO1yXyeD\\\\\\\/CbtAge8cfOB\\\\\\\/KfBkR1v02ClJY+pqulvEt+464CDbbOyhCuNsW4PhIUoA9ZJFjv8YIuQMXu0vxgc\\\\\\\/xiZfU02DVtQi8Yt8op51SSde+1gC1hgFJp2deBZSiIM6JpLECkh1s2jfFH8cendxN+Fyeo2nKVRUZmTceFTxAMDsIIZ0oA3BWCcKn2NqS\\\\\\\/LVS1pFC25zt3n54vXI+RQO4hxe4WcLsd0icbctmYHFhIS97v841M11kQjAoSSJ0qtqky5ydKgBYKj4IoNiVXx++hxraGkoxLdu4rI761wIUil239iIRM1TUMhjjizdt0nAERXrOgJB2o9JjYY+A+PHDhXVzmr3DDaJaK3zLLk2YZwVv8C9OEgrvLLU97w4j\\\\\\\/CccXakNReEc+5H8QSKEVWTDHqII7His7Tx7IzvB4T+UVQu2OT9rskaE6YAFutzk99G1gtisNdqUMiZoLcSl4uS6M9\\\\\\\/+nWX8ehmXzN+KO3L0gvbFbiI213G6ZsqO2V8KRA7DynElHw9aMB\\\\\\\/SA2KixSwnMmsU+D5IcqqcZUNZ5zGQevqAodEiUxc9wG7eJNsTjd2SUAjic5TIcragdbE620a4GnVqNqNs2TiGcFrxggI8yR2Ko5xIJr39hsM1LJVAyIbiH2F7e7+p08PvWpZ5UXgHgu6aIdB\\\\\\\/iGoetPo+ImIcVpALx+I\\\\\\\/wi8IHLH+leOo8GdsnqBXznBvydSrJS9O2DzcfXNWRztjfknhiGKQrg+dyzM1qcYITJN08ofGqOcs3doycOBuhKr\\\\\\\/NfFp91PacouCF2vucVAwnsgy3Q1Dyq1n2kFN4cfHTTH1URrxHXhVIextI+98XT0GYfQQdaUhE\\\\\\\/0SHAEGha6krYsqWozWMSHeH3VMnwaZfDSBPOjYyNxDu7+bFOCOXC3+2bXCJMZgvEUi1XOyoXwYCc2Tmop3CxZQPNm0Q1QMi0tUlP82QeIeifzfpXSd30lizC9\\\\\\\/t6Zud9ZxvYySbnc3\\\\\\\/J8DfSjuT7UlwVAwCTgAo6d2zRZ1UVq\\\\\\\/7RUSd9VZ19NQVbSLZUepOK2Gr5oP+3RAa7bUNPU4ZXmEXOV\\\\\\\/C0Sv0qzmG26jglYJKRMnB6wJoUy9CRT0t8bAqLYgt1lTDgTlRFutXpydcve+B6XI8nimYai2w34BA6I1Y21ypoDRQfzSKtg4QXA3OxwovXjzbwcWlqX9yVTwohC8PYt8ZXv\\\\\\\/+dtucLsQF6\\\\\\\/lHx1aeiHwpc9gxNPSZL\\\\\\\/MusjfZicBIMaJQFc1SjTPXg907dRbRCDc\\\\\\\/b5zgUwi6TAlThHmL1Ng\\\\\\\/z9yA8Xd9wa7wwrVx9IMAwthH3ddsBxTCiovynJX3i3r9E10UVR90t0tmnX3tYcsgiOP68\\\\\\\/ZVAeA9eXyblh2eGvWh9yHB9kf636pZZ+nRnuc7ka4f08FJmu+PxZ3rbNrgieMSeMuxKyP\\\\\\\/l73DgPU8sMAvWByLdhx9+0zPFvNTVPNeG7VhaMAYlwIZkWT3FfQWRrzkrM99++hdMXOCSuLPYLXMX92Qa9dbk4l0nNOSCzsKHIiQG6\\\\\\\/9B7brcHzxE0\\\\\\\/7SmX6A\\\\\\\/W7O2kJFIUo4xuqvmHM98rxfaU8VeQ5em8vl+5mcb0x03P+jXKap1FPDzH3OUWBzTfQ\\\\\\\/dKJITO6Ci1GGGd5rNv7MCH49OTjQ9jN1fbbwjTjO0Qf+jWJdNK\\\\\\\/BxW0c0cLOXRtMo5+kV6qKj8L5ypIojBUKqVBJ8AoRagUK\\\\\\\/FlAmGtM+leZTtuhmb0tdNstb+zOjL42VHnHwzecsokgLBXfvVTUben5O7UyGOQgs3oGH63erN46Ulah8TRtxRjEUkEHJAmX1ad+w2DBsBpamNhSl0TQ6evuxVtFRhIrGm7DD15ZsXdaiAHneKobXbOjeDHeFhbnmoVRLfrLSddAQ2QR5bsvPjTvnpTCfy7\\\\\\\/TMI+ajwqo\\\\\\\/nhdqLvWGky\\\\\\\/MbUgvUxDqFU01miuEYmTm1qn7Zssubu15\\\\\\\/mO0\\\\\\\/0NhS2M1ZlKVP6fSdrCVVBKEgSzv3ocnhIO99ZVQcYzzLImS6RLsUWW5cvFK6wBMhv4dNa7j49j75zTmecIIs47fbnGptDM8BpZmobLmwkj+ARLhp4ccnf+uvURZqK243WWliHEOtE9YYbj286EPCUlIpZUa6WoIdJqBVkgo9bKZSIpNHH6s2B86I3C\\\\\\\/CugB6jALM7OnZpM8v97JVwF9siAZ02ilc6enejxw8SlbU5RxmNKaA2296Lw6rINlyXEuDCFV\\\\\\\/2fAuuG+cumG7ctbaw8SsuvSfN7lmAu+UICllUGhy9PToTIJS0dGPx3fYbOuQ77YB\\\\\\\/bI6vWNCLugDPf4N6MIrNd11j1zXbr5EbXk8WEbFToJWkY1cS8nuRhPGjRhoOfJ9Zk+fzD+T7m2AdIYMf7dNfNKpPNtYkrxQ30rQdzKG8uJKU+BxM0uD4j081N\\\\\\\/oKSgKGrpv2cCuAYGF+I7nqCO5IA2ZK6t2uVt7cQRSXpxXNSM5HZYg+PTZfoB9j54FtJfxpUzSYyp8BiSUU8A1ZTzQfG24QO9qiNnbOHK1kDzEXa0+UdSVgVdRc++nrBepUlVo1Bju1xVqwiXkedOm\\\\\\\/xaAvSSSprWx184V40pI3FI52aGirjTRQe+5ZnplabWOTzJe3xpgC4CfCQW0ZerN8db19P+E\\\\\\\/EiA2kPBhWV2UR+qkl3FZ3QqYKV41w3qdt0RMfMNkvX7t2ZdXnfBnJlKNDwkx079wT8bxoZ382cTsBshkpUuWg9\\\\\\\/oCZF0dzl9XElYA1E3yhbWKl2p9EQmmgzHkMXNjqYYxEjoQk3\\\\\\\/6x6Z8AxJ4yckGxti2Hnb31qSDdj\\\\\\\/nl73IlLWeIk0qWhXBWcAHRf09FqJmWvkExpivS7Fkx9g2p9Hvi5+XoZWhpYaxGgO7h3\\\\\\\/8ze1J7HWrWdb8UFrRa1foTcoOZf6BzTRK2OPbXTcINDuJ6zfZFa7Vkep6paiqeszmBflBEclSrmd5jotnnQFoYwcDqG08yQz0kx985gbzXkyE0XBqP76y6E0wHlAgqS1HjPSH1Hl4bMe0EXM8RLACkhJfKbcc6hgoNDl0z4V1J3kfFuAlSTIkgZZcgj7Tb7ApIKhIrp2aUAyB3H\\\\\\\/bk0cyhBWvA22P+OVLRAz6pfJ6CxcD9ZvhI3vj3cp\\\\\\\/ciKah37eFi8PCHB0S7\\\\\\\/vLQfOz+mDrxcdAZMIw0QipdPq5eYvmOBx6V4b\\\\\\\/atP4a7jCj6jQO\\\\\\\/uikXRRppIhJb2wtxXCAYWDcP+oXvSne78gfUlGjpZIrKYxFfckz4FoXbGNDvVkWEwzpnl3hQcfiiOBdZCYgCTdtZsA51dztSYTNlQjd+vA3dglP4cDmU34rt5pttuPEQGnb2TmbTXzoD8M2mWa99LAv3rBYSFa1NCmlmr+8mgnaug6cB4CXTQ\\\\\\\/MmsA6Dl1YYEsJoutJu6PWh8KinrNQBqKexL214AjOC5KMYtgqGBE\\\\\\\/JUnbeQqEVXqGy2UKWRu5EXcuwFCmPGw\\\\\\\/ylAP0ILY\\\\\\\/+luHnziHuj1WYyO\\\\\\\/VyWM7ix7RlxqgB9pm9pThnKGROhB3JfbDYPg99990ZgpDLpQ9cPHY+7W3ieca9GfQQoARRt9CjkwAigR3zIaOzbAbw\\\\\\\/XTr6YxxuKdpgECGNnUV6ydGBZJ1VfpQPiGhtccn53G7Fey\\\\\\\/zEbqLrJo3xZ3o1xPdoztx36kEh4JQ2ZdloP+YSk++Aj1Axl8XQVpy+fvUnWKO82JVZWYyF\\\\\\\/QCZE1NyH0CSRdoUBBVZ8nQ3+TehEQZOzZW+pOJtPU3cm2gJ89HV0Su0Lqmx9AMIY0PRLClbnU\\\\\\\/YTITkGCnRktMZDeS1kMllflVS6kB2t8cscAPCVWbwnKpxFLpFrx49ygtDtPaypZ8V73MbvSKOCVILadtK2hyiSAT2fXEg1mizvbfAsleA2w4ZIXpDHd7mDvRskn8w6Z0BvRdI\\\\\\\/o0d7o6COF0VLP4oi4nUJJU5ToMb6bGggGDky8U9mOVe2+KwcOHxhfZqTmJrBA4UWqIMgzEVl9BrubMTJt0UX4DFfleoxSTGDPycHhxJcZ1ukp95SeaE1ZSFpIr0Fv56JVKgbp5IJOo41m\\\\\\\/VXGeiIx3n\\\\\\\/Z46g\\\\\\\/fHAMnpLASQu\\\\\\\/uKxlA7Q6tiiP0mG+nP6AEipfjXRRHdhqBLJ8EJd+\\\\\\\/zwaz+NAj5puU9wquVxTzu0bTkfzjHMVtRvfGk4LO6I6S4hL0p6G6tDgcP6BsJxc8TVAABrcuK5BR8Br3uxh5+VmsF\\\\\\\/raRobFIFSQybNJxCerByAjyh2Fh2czE7+WilqlymU7bBOPojfg9qMp57urUn+RNvUhGmM\\\\\\\/LyIihJmCM1H0qhUw5h6JoefMkFH3fiYTP5UIv8LrXHq70W4vZ44D5sZqWqhyZzmZ6Vqv5neBccryAWyaQrwrYEd3oFv7efuM4xiYSXyoZMtdVhXbBlioYmgeffIfQ6\\\\\\\/Np2UpriR4eElYRbpwledaPGvTKfdH1WrG1J4GH\\\\\\\/nC40+Xjdp2RoROd2x3BxpT2irF8dDIzxMidc5MJnZViEVofALTbCBOjGwayH7\\\\\\\/OJzyQo18TchjCSyBvRy5Bu3DI3PcX\\\\\\\/o7ZQrr6Xz5XYHy4ShwPwqzqAkAHzhpx9BJeIl6d\\\\\\\/sSXsDub05o9iCT+4HP6AYL35O6kDV5Vi2qKxUjsAf2LS9VdIwq8rOdpBdlh4w4EZOStEJy3U8O+\\\\\\\/JXS4vGcOMdmErQ9zggx9W70TrT8HkfsFumG\\\\\\\/cxFxGevZJYnkam62D3xx57wwnoHED5o9Dc7d91W+JpdFpxNk3XO7z2\\\\\\\/siEElxeeBYsd9RjL6wjlaux0oX8\\\\\\\/V1ab8cTcLbmLFnpxkFZ6aPOzBKF6BqcTbLmlZFjS6fVz+O9J8ruwzmFd60dokE0Hl126Dk8QCJRZ1QPSFpalbcfUTZBTh\\\\\\\/\\\\\\\/y7tAz4EST30k5QiNDzlezZOD1G3V2TH\\\\\\\/4iHqC7sq40AjyKSi9MgjR8CXZfR2kr2lbOKZfmgknciEcpUcQ+udKAwfCAlYxlJrCgZCAOBWqhCvw05+FTaJYlxTsEQAUNWk78diZncgLGiJzkaceJUtUDREiPttIAnLur60h4axejRglh2vbWrWcnUd3Tai8kqG7D1\\\\\\\/hnQvrNP5hyFRcyHOK4FTKuQjuSjzsHBgBMZR\\\\\\\/HYzKZRBzSVUi7frG9+mXz2v2YWrWedDquVAMXm9818S6IUoVjC4zzE+iVJXGp4ZIp9bL5zPtJT1oLMU9cAJ8Y7dH26NBRIq3QxX7MFy6LJcEDMnHfcafL0OpsPeCrqf\\\\\\\/adD0e3IEgbSAy98cuZXLsdyp\\\\\\\/mHFvWQgy8qqy9HhxD9zzetyeKU8cPVugau7JB5odTSkhQM9il2Monjpk+a3URxgk2bbfjvpNz2KpDK4fcyyHTwJXWydq52c9uS+MV4n5HPVBxqgYx+8cuZFNBHwscRaQFFBA2kDyvCMe74CydVSUf468kxPgzBEv\\\\\\\/oSk0ZCGAkB+sUz6jnTrYD9RvoFBRKAmY2jBuLUzvurHrq6YRR2PI\\\\\\\/BMmXJYwJlA9PPj9o+9N2dZblgcUvFeAYAYwtB5crK6vVCO19z+P3mxMTlZNAFY17GgOYseTe5iqBsLH\\\\\\\/jMurfh5ArGrgxDZArB\\\\\\\/m5yTcGIf5ODa0pf1n2ipE0pPoH7zNauv9wrUJ5JISE1L8oAIxlcmzcIiJlWUvRCRyWJMfbr3U\\\\\\\/OMc0cEATswtdQjfKEqLN9GprI4xvELdW2NmrxZoP8LLvluMtX2pwltMhCj9+Cv2NoJcoRU75dVcUV7alFdz2k0W9oajpS+8GQlUnioOLg2gUMS3Dh581V\\\\\\\/w\\\\\\\/hmoQB490amiz3IM027Yz73Nq3p3K7KvgWbJOxRr8csXsv0cKdWEkJPyLUwEMk63IlCulSbU2BhPsRLIqdaF\\\\\\\/8U2\\\\\\\/S68YXjtq6g\\\\\\\/sxySEso2C3\\\\\\\/+MniOVKXnolwNSnk3ab6aUXzPBuh4xiXiBFco7stZNyIt+S14YI+v+dk+yvxm8u20GTJyhXxcnOcpNBSxMyWJTj9\\\\\\\/akxS5ZdJQM7feZ4dwCs3hx9XNZzsXaK6D+qt3posCp7alsVXPYL963ListDC8uVMcZ7c99BKpZ6Ae3VTl6ktc801rSDkw4a+X1lvqiyWEpwN54MU1sdC\\\\\\\/kRImmq4k6RIC+1rQHUQSpsgSF0EtWeaeJMJ71e3+splzXw2JE3LplZ1mBXBlw8Gpz0OrXqnuJVamWHJBY5Kv+mgsShZNnyCM\\\\\\\/U3zT1GZCKuN1Mj17llpMPkvXh9B7LWqu4iIChtIAPwLFKWDUo\\\\\\\/fCPJAHXAmojAg7wILOmFWZiWqknQbM7m0kVUdSx8okempl\\\\\\\/h0dwxb8JAOYCPzXCh9OtksTtq1QkRb7WizBrI0xwjz0qPZ\\\\\\\/7A\\\\\\\/vlTVsCyl7QfBLPyU5JadnjTWwtIa9tYKKM2W7sCXlVI5uxxPhnKI+iZvcZ+1\\\\\\\/JzJlWfBZRoa942VlxMsDF9P2tiDSLq8RwPA2BLm+rNX7ubR0BWxw7GNzPYAHe6gkHyJyzdjGE1lmaOTDQJA0RCLs8Jh+lNMLlCwaAvDKdLuMZa6EL3Zjj2QvQ\\\\\\\/GCwkaUyz+kizLlsa1DOSfQilELD0AmJifBjou2F\\\\\\\/LefLdM1FXiZ\\\\\\\/zr0JumjeltqjK7\\\\\\\/pmUdK5j85j4Z8dy\\\\\\\/Ee\\\\\\\/h+bBh09ZQOP5wtx5opKKBzBDxT6k17yn+purAUxElVVHcoQ02fZgvtmWiT7iDSJtz2i6vn9lL8k9CRYN2jPguMxMc1xVBF6IzsngN2YnRS\\\\\\\/Zf7d6WnYRmLSx0QtE7Qr0cKv36yUP89YDF09+nOO\\\\\\\/fz67PXzF6y5J+8ttFZVSSJdqAuMXxpacQv1OxkwIuuxN\\\\\\\/2lFp7PUqGMNRSUc2bxEM9I6TIX2uFhzZgiekREf9aTK5lujJkR75BOfthDkvs7aGafj80N92VkkZQ9Uvp1GpOCzisJhN0WoFy8UM0yMqxm4yAlHAAFeRrD605Oth7NfqtrBgB8C8R8yYBc8EVA5xCZUDqgTlhnOhEH80AceL6ceHE51EdGXhd79GHiqw24b\\\\\\\/KH07curG84loH4YT5AzbSpKIoiZuMRSAFoT7PFSBzpMIqG2OvwonhE4DFGeqrv0zorl7tVD2W6m2IGYKMUI9EVu+M\\\\\\\/LAsaUkAsqtlCpLf1dRV9Yuu4TaIHaNvWPtB8DnK7uPzQTYjNKDjNfMuBhfovnIpcu3yz1EO5RQS\\\\\\\/yFauL7SBdLJcpamLagHBAqh4VfcrASIK\\\\\\\/JB1SYsgqRaMBoj3pj1QK76cpiJFMtKxfrlzjooJraM+S1IDtMqHCHxBrOUjAhoaNnb441tnBVDxNeNWOYpg6I+SmrlzNPhWwObbCPuA\\\\\\\/eZ0uftnYA6Z8hd1YJubVI0sC4qvdvp1jlA7FSy7WSI\\\\\\\/hugstJRyCMcpdjVACmxIVn+lkn4E96OBW7HvScTCcEFRKPGjZEwqPwdp2PGpD3Fc6TLTOoM6pbBfcOEnjs6Z94VwMWJI56SWzK0qVK+ofDy4kd3qUfSkVPhu1EDMDL6RwpBALe12hBdDJIn1lJZ1486U3SqLKb9a1gHcnYtTsiG\\\\\\\/h\\\\\\\/bg7fUSw9ArVuq3XFeySNtqX6NvPZdIXu8JloqyE9SQRoacKkav\\\\\\\/DXcNcPNv2s5TmdnV2Z6ZheZedo50vDbGTdvH\\\\\\\/yaYT0xMfN5Ash5zCbohMACYwysQ+yuUNDrqIMZYMUlvmfd3jSEEAKlaSlB7dZrKLYO\\\\\\\/vh6x643ta+8hRmr3w\\\\\\\/rfWESjhVd0Wsh169HM945mQK9eTXUT2jxWSPbm+0nJgKVnf8SnGDk6O68Rj7LjAWWowToFfUn7kCGNZSqHwJR43nONnfne1mg92W6UHr0o1JtT3\\\\\\\/nzUBRkEeeHv7NEnWbbOCXHv1N5W+2aVKc+luP6orBWYRsscKhh2PT30PyGrWktGqj5l24Wt+2+pq9EETdAEGqJKi9s1RAxaUCWyUrhmGZmXRk75tUGgO1cHIpLuP4vUCLKBn1XkzsbJuQh7AD2zXLzBYP1Q6AJV+Wr66pNhwaa3pJ+lGGfZwxl8KnJKf8\\\\\\\/tQ\\\\\\\/A50sDWtzUiz6HOWaZTtvux3y10ehIQGDfEtIfsW0zZouILh98Y0PakWeyIMIVGF1YyGnNcP22YvGM3O+PhfibsXz0je7pTETrNNo6gACZxU1aMQ06UIeaX1jI3+9LfoAklKNmZvf9WwYKAepMVVCtpmN0imI2Dq3BoQgINPz7gkgEHKL\\\\\\\/bQSOtpRnDZeM\\\\\\\/qqeXNrPEkRke9b\\\\\\\/XeghBu1oC7MHOFLMWQLj0UHmsOZ4Z3uJczaXwLU2PgXSqa8BhDr0LSFTKjCgD5BztODCG37nG+FKEBz3A5Oh5mC+hwpDpIrzxs2UnnAWRCBMLwGFYD4HlxEjioxRCsbzk\\\\\\\/2UbYH7bsESnt+lBdLDtIb7\\\\\\\/ovfkVFxM79YGydBowGc1VU5va3VMjr39Gxofd60FoN6jpuA2ckamE4XQ1kZtvmwKV7x\\\\\\\/wBHleko8jKm9Kx2UboGAQGdN8rFXfaajz02IA\\\\\\\/tKkDLCL5Zf2ash+mA3seti0AUNteVS8RJYZox2QAJVDbYub3FSKfLlUABsBr3o3MocCfED3w8rvUoVEWCxZkFI2mnQgMyxOUmUgY5l9pPDYmI\\\\\\\/SQ8rsOvECvS6UbFMGmOanSC2H2C0w+L+IOZGiYwoIiG28rKLUxMWz5UNWaqUzFQEA3UFmAogmuE5Z2jGxsw+ZyPBjMGe+0lJYGiPZdSTQ7GjGogW5KtD3K5OF9nSAc0qYyDImf4OIxMAKtN7\\\\\\\/oHd6uqqnwgZ8IeKtQAAR6T10safn1b5NDMYIak13lKSiiqcxMzhSoksV1c5jJI+Jum6vQaAdwVtdlURd6jQ6HyLxaGVuNw75NlBVEMugRekgPUpvuN0KD1bBT5jl84kgS\\\\\\\/dEcbhimy9tO+xeUGDsBhqiNXEOOEiTzEQktSkfXRrXYVpgxSFFu4RRwLHAQmi\\\\\\\/3XzIPDqvr86raP+aob+ufglnTOztCdN\\\\\\\/z0VZOma3wNCVCx4sH0HKVsHnE9Q5t00Jcr1\\\\\\\/KHEK37KF27QKJ39A5SXsDrSoxmPk6sSdk8X4v7bp1sTlwQboG0BkyTXfq9S\\\\\\\/YymjDZJN0T\\\\\\\/QJtmG8dOrXvVqQLIcFXBJH8uLNfka5tjKmiogBpkNW+X62J5DqjwlLP4898mEE+TJE0HtObGiktdti+XBuAwdXjs+oHQIYgbnwlYY7cgYslaAfiWtk40TXy4iIdvBMgXFa0kOfd6w6t1y\\\\\\\/aqtX0hl+HFeM+f\\\\\\\/AhDij4CFdIq9eN+9LDzYx1qhKAMsM3iSezAFclwa1oIG0Tdo4HqxfLMEt3oddMS3orfhwqXVM405wv8d9sLJUwKBWCrREOY4m5Q2j+EPbtiNdzBbGNTm3BSTfNRl9sNmC6USchHa2xwmvWqZfVT4xqP\\\\\\\/hNVpYKV8X2aIt1TwmFlKoJl5F3lTTnhp48GUJ8O5zcee7LEt4v6r9xIVmcQdjmQBdU65rR6QjZVptuoihfACn6LWsRpYNAO1\\\\\\\/GDTCOzKQhBLJhuMvAK7VM39N5T4FWKNM3eUeVyrNOZfTl5VlN5tqV4XGuxV3NiSmfN+GDls8Z7WpMHgFYxiP2vtoJFgqaSRb0IHqphpOyAJrO6GLNP10ZpYwnWyChVRcMHG5c4lIGHRL5KNo8XS8RRBsaZydpQxsE6Wnskjai1rNv8im180KjN6IYHErHrZ8zMWerlZWI0176nLOIuKaPSVQcl2eXrtWjuw5iwxktD0bmkR9vXOugdmGLPFGSv6c+GU7AWMuHVJyfhls83ZdN6fLUyWO5Ll4tFacWktek\\\\\\\/IcdGt40iug8lDf7kVGzOPoyPHLrPCTcMDa2naZbwYFn4erOivol81Dn3LQ204Mc3JTMJlLRJcpvKJXbOQ7ml0rokEsxThp29+TDgmP2R+QFP2PKY2puL+ei7d9o\\\\\\\/q568RMjN609NU1R2HUYNblmaQju85qxLcjZ522AlHete7tP2YW00\\\\\\\/Q2w4hNZymZYRgcvI+dmBfzyAx98FUc9cwqvqUa3nVnoDgo4sqp21iGu7oRC1sYkkrJeGpbhjdRO6nbGfJZWCMk+J2fmgr8BQF+CaHpS8mh33RxiPNby87HbOJHJ5SoaPkWETN6nf0vzOzLgOeCpmA7IUhQ9AOJkMAhqlbWB5+yhRfE6Wz2cWflGZzf4nDBmoOmpFa\\\\\\\/Cz908YKWmtLFHsDbKw1YII5ohIjJABgZ+M2hP\\\\\\\/ZzflabHg\\\\\\\/lMSsB6zpZh73TjL2hNx2eLNun\\\\\\\/lN5XiJBRpzj8P6c08VEEK+XZ3LZe18O+\\\\\\\/kKfcN\\\\\\\/SeGKUWe0fUoTHX4q4bUfomIllKXjQoWPCyeqdnUTm7v3WGB1H7exex\\\\\\\/TX3mzLsPupdF2u+Xnh5KUKRRNVS+XPKaiOHyI2heyPxJu+nFR\\\\\\\/G14rBamXtA9Nv2sLDZuaCMC\\\\\\\/HT+uRG7g4d0cYII84ztrFQZ696tKton0Vei2RtjkLpYeodo6A4iW8w2xAiy0+b8duwhhG7cQcNTyHKr618ezTsghW+7vvF6xcYfsgYVL1LY\\\\\\\/rhvL0hG\\\\\\\/+JISQglqw33M37hItzjwMUGRKzC3+cxe0JD7O9DngNPy3dgRmOG9XA9as33ulYRiA4HHJb0BtOIvccyyElkq\\\\\\\/dHRv7f7TGzXxXVXov5rmDlc+55lNnBFQF8xrxHJciw8bIPL6mo6uoDIRe6+mBTFcGPC6LeR\\\\\\\/hdDQ+AwkEwRQ0jyPBZqo57AhxfHRC0kKHwyxqc6e4+gr2RQxYXskBxVB5mnu+2LHeSMi9fL4cWMWoTT7egzec9prXlBKZCrfvxwKR+9TcQSiPoMrEO\\\\\\\/bzfEUxGkVzHqw9FSHxJ8grl0yxDtKsu+uDluw7Lr6dtsvsECv+\\\\\\\/eTrGOj0WmWGXtIyVQ+Bvoiv7kc7OCey66Zr80vsae3BF5+u6PwTipSwBOkoykPFauuCkZATGiE4rI+UxX+osj30yEsdCv+IYraRPyvnaslaY4xHhwNEgVVj48jKTzjYzNj1fLqlp56UNCbHoJwLd5y8B6Z0KoQyxW1E+5UOs3sGEJeahZ6UOZtpoLnklgF+KZc1JRRog\\\\\\\/3phcW3gLOhA7SQFPGX6shF6HYF2GQz4D4aKZh8Z7OZ9NqHPdev8zEv+O23sc5ns0ss6DHptpleZZq2Caq7Sv9sTYLJ6w+yTa\\\\\\\/Ndp6tmo7c+JgupKbooLiY3\\\\\\\/iZqrAslpHzSSPf\\\\\\\/+tRXeoI6DB+tMdyTUyr6arogGHRYUirGABj\\\\\\\/ce5OOcsOeIHPSPW9\\\\\\\/wGIyGafIBmyAWgfOjNdzDvaICLVmhhRjLNhhsW2VIj1buM5lH7t3WBbvpt3wx9y+XULqCG7aUgESeUAefRNgmSa+VMsrTt7dEO29NEqneY50bNQh+5PjGLy+yTPbapH3O6a6RociqAaYLE\\\\\\\/JJH5D+siawEq7xUyIWIgzL0v8o3llDE3YGD6zPHbRbE4g2xZnHcfXnAZHXOe\\\\\\\/fWYi+BJ+JnAmQDyvhqTt2EaHo2GA7\\\\\\\/g6bYV2eI5doC\\\\\\\/ZXmgnu6f5I9qt5v5C9mzWCiAtG+XGRt+g33fB0jFPC9jrHPgBnCAQiarz3cgPZL7xVwk5aanQv82LRE9liiJy3Iy9ltI\\\\\\\/UTqZEnWHwOcvDyl9MKGRhsW\\\\\\\/jlQFWQ7dj0tMooc1Ro9jLQX1X2PNokNNZFGbTU4P7lyL7LE+eI8+o5mNMDzHVXuFqu9yT5+4gAivnCSXmkNBTIo8nm\\\\\\\/5Shn9BG54Zz+JLzkZ4NPWPzMdnQM5nzoBo\\\\\\\/KhxLtJEpmpeezYgtj3s5eMn0\\\\\\\/ZHQV8g4CakIl06o5Q\\\\\\\/qJUIXw1oLetj+zWimrlWQn8xDjHt1AnT5k0\\\\\\\/M1f7xT\\\\\\\/uSfabSxUZKQy8dGhJUPNUuYmTvC8sZJ4\\\\\\\/AkBZHva5AxCURZ543u4Ipzc7y6FG9Q4afJ3G1+qNHu3sHKyW1xoO8xVCYuckAgwkWb+BQs2YerlooMDPlb0gl7AOgoLZrjBMIQJMtAZN1uAYYBhnz3Nkmenz5rA+QuHUz5Hh6K7q4MZkbtP9lBwopQnXkRqVsAI\\\\\\\/8PtMSLboIr2sZ0hdnUMtRAarm8HG8YqgimBED8QS3Uuwsy9G1VaP5S7\\\\\\\/In\\\\\\\/VulH5OxCT4TjKaWjrsmbG+2s1rJW4rBAql8hnuki3VlpFkFzyp9iMfNSw6czDqunkLhllJnpi2oJxb98kdGLvvjQNhO3NJhoHTuehL3oQvzN8moH6uXnmeB\\\\\\\/OzifELDMyAXbKc8M\\\\\\\/Ly9VUj4G4K0emxPw3a7HzmEknm27b+4DftyOU8FhGuPzGEAZ9YGL3CDXB1Mi\\\\\\\/QD29H9lQy0j4\\\\\\\/xd9vjpMcEA3G3M0l4I1fxbAT7wDqr2giNIW7XO2OeDQjYQReF96s7zJjotL0DXOFS6EI2JMOSNzCvaBcubkBYHRxuA5\\\\\\\/wLug4CsJwq62j3YLTyzlBVCvM\\\\\\\/nnHg2tu11XEex8oGBFn3np7t5gCORIDsxu6xZGJ810eL0ufs9vsCXJgshRMEGmZYvpoHz1qTI24tDQBJt+o4HDiEnmbNCoauEYPhlF0xrOLJy3Ujw7mLgtv9B8BmcHSAZHu8gfVIPy50iO\\\\\\\/SM0W3RJ0giyemo4VgTDbQLepLnDWIqntgxIrVTC0lS6AqDSKKepK2KN8PQI\\\\\\\/AL6vmiSdmGcUf7iZPKd12v2mAkZjTQPquOPOSf+vMUQyw\\\\\\\/CiYytZ9d16vu\\\\\\\/DJKgGfynVVLNoPlKd5V7RrcotkRu\\\\\\\/cMyweCNLRq7O27kIDMPr388TQqtIn2LtekuF6w+2VbpX46uM541R5wSIU0g2AuIFj8VKHkpI78CusyENleXkb0Wp9pWN2Cc9jmyTqC2eZ+GTUzr6\\\\\\\/CBV6V9DTTLileXz8tuH81YVVbw7RGPEq4s9Y8sgKrdoznASJ+SfpNHdohkKNTpD2EibdjI9XMbiRObHqDYp4mkArc4ws3+xvbmqzdo3bc7l+apioN9dpcFvCeGApXPpjGeF67mWM6oBHRjyu1mmmISr8kHqf3daKKQ9oGjFLFgrZwirYJVcaD08PMl\\\\\\\/zlST+BaHCPzcrY6tbudhnuWeYcaHP2QQanjKv+KPMNO8v6SYcSEJwopQcHf5imW6DMTgY3tw7RRVM\\\\\\\/VpyqOTnRITGFJdQz6DY0JA2xBKRtzcpCijn2ngkr8GbuZ2aoXu77P\\\\\\\/XOF4pbuhaAFrMDQg7yMpwgwhSEPwILWO24oEhSAGO3uF56xK38R6fab7ujYLmFVVKqT4KzEX8JXiSvSaDOwJ3cP1ZrzTG6hwNHrIBcegNch0tN8ah+asArtS2PWHT+kJdII50imU\\\\\\\/YNh+YSeKKclmlyduxiarvSW6pb+rxkGp47tssLz1slDyD\\\\\\\/mgJ0C8oD9NDKkjSFd93sptAkRzUwbY0Cxb3B01oTLjJkuOINvmwYlBCZkS3EuEIRIjW3kWutIEb77sB6EKFuXVXWJnV5wOI4qrQ+FGYnR+op7x1eKtdsnEt67piB9w287v2geqvrov30zNuWixQ1KaDwHfni\\\\\\\/rnDpLUthYO4\\\\\\\/PGfd5qI2eboTgQKD1cc6fbT24tL9\\\\\\\/FjGUflQ+1GXAT63lAq29gL1J7\\\\\\\/EEsG6VnIH3sCTzc7EiLpSx3gSW5x9Ps0cYNDgkFmJhzVU0EpTxpcBFBAyUcCtbMwMDJmEqhuHvAZCMbe2sfocDYdBOqEjZ1OkSnjyHibau7QkHhLt9rZGCBofh7ohc2GP764ii2Wnb1ow7vSGUIqLNAkftdVRzg4Ls3sjlT3MVoo7sqW+agCccMxjAFFfmnFcAjObfNiN0Yj3G14iUVn+D4pTyG4XdvvFGJbK1Khq0r5NCFu1i6AARqRSTcYs1e+s1rrkxJXEhzGDrj3Rw\\\\\\\/2gvM50HKZPC5XM55cz7HEYfbEqT6bfG\\\\\\\/a5F4x4pEvRPZGB9gXEP23fF7voDpxar6LeWLw7dLMYrkJteJN9i3OUQLF1xUOMkr5dnlBZyLxQkzuwUZ74yq6hEFly+PWxZ02uoCtKEWP3b\\\\\\\/d2elofXzRvHnQDq9QJpOhFTk8wptzVlZuMxDl7x9XpbpEnay1AZJaXJ1uzrZo3utVRN30HL854A62d808xoQCwGMZpKXErte3aMwY7+MWzXXsKVsEihOgjbqBfFqojcZxD07xVbkoqAF4TxC5mMyDBd\\\\\\\/6NYIsrWmDSn1j06aDLOFxgTdkgAoTQIEDUsH7BJxDqp2NqqLd\\\\\\\/+w8zYkumArC0bzR9m7hRzABzJpPb5\\\\\\\/WVRQVJDvOvSl2U4qQB0QLcojS3MsTRXWgrwgbjKvIu0zeBH4Bo060HNbtoSi5nkedU1ou807GvltRXsDPgwj1by491\\\\\\\/QwMe6ZdUDyfBU3ADPQHRyc6FAQ+mb2nBS06pknRSBWPhm5+fdspebbxEFIqbixHhAU5qiKYViMQIOILSiC72HHVPlE+pOLyyb8\\\\\\\/IqmpDvVcCnU1OWD1MZK3s7sPr+BEPar3FB5SHDjO3g5i2NMWcrv9wkOJpnjA83qc0LA9revkfBXdHBdxPzd+wTT5bnnY2eERPwbI0TI7IZZ3ZrJ2U8YOGzk\\\\\\\/npLYP0rt33SjEedRXMVVWjRedvdgfhM9Gm093iPaXo+C+rKn+yk1QoW8G+VJ2bK6ueg9vIOp1lnSfjlEBjsCOcZxuHE0nbh+jgVtRyjE8RWMJSbhIrug\\\\\\\/q1rcKfBfUaiju6+ypSc8RoZA0PRz\\\\\\\/3PB5dSPsHJ7ixlRr5tu+eHnmod3uLrbWN9h8GgQRWvHX8\\\\\\\/8jVChrpacly6vzViQ8gpaeJqPzd8cAM1M6TI2QUZRRrfngHwGgO\\\\\\\/O5dDLBZXco8LekXUCYzCdTsJ7jfYTZbrt0502HyYqaNjQJrJaUfNYI4L+Mr5h9tYMj9P2aZa2aXK3HMgdQwOnewUYDBB74PFL1rxA+68L\\\\\\\/Ncb4xCiSAzmiWBjELNZesvX06S2YRuIW2Bookj0NU\\\\\\\/Y9ZMImCX950Cvs2g+Yi4ynaF8ywoUZzyffrKkTgD2Tz1mxokl4nVBRutoW8RKr\\\\\\\/ea\\\\\\\/INzFLzXShZcf2+\\\\\\\/jLoKB4sPcWzr1n9\\\\\\\/Ug8KP0okdEdL2OZJj58OpZSeWXHhg1rfzIEeKx2h3euZ4ipE\\\\\\\/qxj3zxpr0iUtvAFwOgHanDHvVzEkM2+2bExq1KPlsrXnNszRUM5vsxq7fX99fVFiwfm6MTtT0eTqreaSHxz4EO2nu4FPOGhLy40ag1kWeCxrm1UF18H\\\\\\\/V3qKgai3RXLDDyZFtI5yqKkiNEPWtb5yOjq2s\\\\\\\/horTd9fnVfdLVaKbSK\\\\\\\/vtNsYwHLMmc8AoZKpFQOE00APTku82AA6kwaNayN+uFBkDjC8fhjk6EgbuWTGP621tHWcy\\\\\\\/1rpwNYWl0n2IcnvF6iH0dQHFmoBeyrdLXRKmVTAyy5Qv7GjLOiaLRyPSSgvB1xiJ7vQO5AahxrJhrlidjE8IoWbrMTpJs6ngVOop2sagh2Xod3iFdeUNFnp05Myy2K6vvu8SIwYhyo5HOr38EGT7aHj5Uft2gPLFGJ\\\\\\\/+kQwNemOHjYg3p7o\\\\\\\/zKac154pnt7RYerGn08GwLmQ30QHf5QsR2Zd+SRvOqPEGAids+WqypUjnig3ApFbEuVQwg1On4NeGRDG9+MJtVapk3cmu7H+FVgokwjf\\\\\\\/dW2E6y5w6PPDvIfmeuGyn1mqhnvglb2Jy+hqi0wSZ1RO+LRVxRn+5pXR79vFDjWjY99SheQQQIWxN710L4fTXQV7xnXPpiXwpcOT47cApCGZV9B75gBsqTaCRQdusA5cmveNEJzEDJ9otVsQDm2wtakMD7RzAy9pSJpS9q+4ErZ5XGH5TrrrmbBvEE0QhBQYCIt3dYEdR8d9oFzfpaPXEVox2n8WtoUfzWmyB9bl3XGqkW6+7Sb8qB8oLL4grtH8Ghpb864ChyTQJq7MerGCbmADQ5gA\\\\\\\/bVsHWfrhtMgziacuvTYFNZgQsch+RBq5Hx\\\\\\\/N3PUuvgpH82CoFC7Dz0Q+\\\\\\\/r46qs816ymvJNCTPIj+2Sm8LLKw4WCwM3xNsbuoh+qr\\\\\\\/Ab9isPY5KDGQ2Z63CxhDLBpUWoqpMUqnYN47vfk3z55S4G9A0OQrql8zVkYT5MezgIb64hwoZHFlRtiyl9VRUG66EorjQ26qhtV+FwCyUyuL7Q2uSL2ForTutohyabnuPi607g0BHJXezVPS9OBlgCfBhgRFedVQ8jGS1e9K\\\\\\\/RqpBDV+LJ+V3DwmCk9Ev\\\\\\\/GZIIyEMO8u\\\\\\\/9ezQE1sllsjjg\\\\\\\/MRX\\\\\\\/sixXM3cbrb45xutJ4yBDQe4+qqnj6cRFtVUCYvn5ZIVOtWdX43pFbvFkT\\\\\\\/CYK77zzqqivQtE5iny1eItKd3p+do30Gok358iSz6pDNsIpErEIOZLWEfqlARiXveqaehRa4PUcQZuAhcdiAtXOZmAB0uZ2HPzdZZSrigoyxrvzhFGwIYu2SSh\\\\\\\/ybzUFmn71tsNibiHPBGDJ6QCutKuKsh39thUCCn7zSQgVdkhrvf3CYkLMdvrRdUWNDIY4eqihHR7OvR7Tm7NZlTE8CUFiGQUrwXT1nV\\\\\\\/F4kMokBbYKC6jSwlrNaVa3ySy6qeEVmMHb8tIPDvQLplqv2lwmAXk08BlSolEBKEgKQQ9mGVWP5wX4LKK8+SVeRWBGPMA3ivML+yFXWkBmxucxOzevw2mG7udPnbXWodV6Tsm\\\\\\\/X4TIaNF6aJwsiQDMAinphMIcXk9qsRdU\\\\\\\/HS037ZUGqfJzVgpet\\\\\\\/X15RYOUcEuscEP6z9nz6pH0BRB7EHRcOF22gQZVnVcmVCAL4gBOLrfhpVzICYLVxKfvlrYZtrXbjsL8dRcDfxMOwkYX+fUGawQKkGWREnFrks33HSXGVcnYZXHyQQqLnWB+HCEbM3fT6f+9tIkk0ZsBvUvVS4nf4bldUH41jRqDNE+qKFeqbPF\\\\\\\/WLk2Kt3maT+KfqkXrxRVdqMF6hVGIxM+3CMbBGE3V4HWSWDHil+b+EXIbnX2ndVyMFqDZp+etE\\\\\\\/zweEV1efypqzrMnvDJacmaI6ReJPwQNVvFkBIZwpbsEQGK6sSuPGwZhUonPq+OLFz\\\\\\\/jz48f6kS7PxCtBC8F8LxLEmeozeFYa34DjcX0zmfC1CBaYzJUfJcDWiVzqzY50Yavu4SZtvbDgyKuJ+S+2hrE3ovkFFjC4nFiRdwEbTljqrPUoOo+ekQzAj2Bt87GWGSkZm04A\\\\\\\/M2Sak\\\\\\\/8O7twVgM2z1KO5yRQUpRaTFt0bZ0XU3f2qz1+knAvvRSCkAPFarlThl1vXZFbonFcgtzGGQN2YvTsM5OkoXAZNNzkBHIhVtkzruiIwwxNKe5DjnmpONso7ilUzC5LVgpBCD6vHH0DvdtDcGUMQwzTlQY7aqQaboLH6FOMumV28dRdM32IR2XD7ktA2PoGIRqB0dofnFDp7MNJYDBApHxg3KHtSgt9NMKDqp5XfDA0NJhGKRyi0duTs+W0KpKGV3qLhlCn49w3VswQW7hHvgIvaoasPtSHcEkoCEEIhtAxvEcbl8N5Ll1EQjA0yKsa7QJQ9xy2K7EBbTiwY+RqtVEHQyC8fIYWL2efmKHQiIbRWJyWHch83PXR8+bY+S5R5eDC\\\\\\\/aYNDLUwXu5PQ7POGvWvSk+L30GAtUUnTSrHK7eRLQIKIWO269iOOS1grxMNw1TIG6RiVxstctxnRx8rsol8qxZAelvtMkt3OdPDfkv0icipvfqAmmi\\\\\\\/N+nuTtKTwTVZO8Co\\\\\\\/vZOtaCY0slfZ8qaE\\\\\\\/FgsWruYiZ+crJKrk3K0oLPW1\\\\\\\/HmUE4mErRSXI3Mk7SuyALjv7qrFvYcCUnyu9lrgFX5mLirsZ4x22askV9QZNmr25bdKr6SrcFbSgiA+DQnDFw5vLtz7bmAOHu6hNQoJq9v42RN8vxiqV5L2IgOL9rjgeVecWvw2oJJhgyqpev+VTjBrsnMj26njjSCh51Fc9+28rbWX2G1QIcFo5E8eTzqcEaDUPQPMpxPEvPOl+Jq9kalr0mgP55llk9VP+tQCNWMjJUT8NfgN\\\\\\\/ROSmswSJGUn5ATY\\\\\\\/3jq6WgXgcgB3X1z5\\\\\\\/Uabrz2QGEiHLQUprjodUOH568m2wHCUmCg6r+MofrfWrNRmCbqQUlDQDSm4HbZvIlrw92QO0UT7nwdqFEWM6I8JZslEiYKVojC63aAy8OIueiLK+K15OzvMtH8IekbiPr+cheYCOE5BTJPbpGAWjW2lktFC3v3L6Bwe0SaKMou2+0ZxoBQWGI0dUPEwCg5OnHvXuB6CxphO4RfsBQCYULjKdui1F03WDTxbyWAV9nh94ALc5HgkmRavcSuZgl2yi8AiHgYWhlEFiRQpzZUbSoqUonXnl8K0RQOUW5Q0KpLOi2EcvGZ6DRUeN\\\\\\\/cSFUQ4AE3qDT\\\\\\\/hjvEnTF2i6JJXaneYd2BY5W+Ibt1MR0I2j9GKeF9nlDgn6oipHfGzGPFaBU1DdnjG7rYj1Zsb4Zs0XEMY5r3+VX0SYMrN2mwVNhxvoQ3eBIuyMBtYRV0V\\\\\\\/jGnE9iUk8YCZSCFAPJCPUf4BF0wz6f2IVsnRXqHBzOotFWzIvOls0AJ8HKLEdiqPMTuAldp\\\\\\\/gmn8YD2saG3+LfBPFNLSMbaVDtG219WYDFYny32hnan7tCXtXM8hTbTLaB9aduJ5jrMqUr4x8+Wimo2ssTIFMN+9ZWEx0AK4iAWABs4YtjJ1Rlcd4sRCIClZeVO0+yVveCHEtDKvOoT2LXSy40fxPAOeiUVJVzhAHPW2ciUHwrvxDN78AMDk\\\\\\\/KHToGGrEUma0QkPB3GbAxhN8TRox72N9StVb3mpH6sYhswU8IvGm09fusSrKOOLw1EnRe\\\\\\\/XrTvMnPwjBwXRABEbLKH5VeLQ8a8qOZApNoz6eD6KSvBNha7qQjhC8exstTUyncUjdrtY2Buiwp9g0qgjrCeS+X9qbIRyPLjj9uL7AjS2VfdmudFh3p5nRCErYhQIWO7ORwef4V2\\\\\\\/nQYaHH7HEcKHaOjj58OUyJF3oqIttGNXbXjOFszEhPzbHGsVJBsc+ewnpYHaOggyHvq84f1yKTCut6wArICIJmzB8wQ6ATg4Cm0+TbTAy3QNe3LIuISk3db\\\\\\\/kaCRIMRs2Wa1Kaj\\\\\\\/vHSngmKQduD9smoNo9+7\\\\\\\/u3FL2RoW60iGA1vofv3BjWMcXaidTYkijYLJS0IDpn9s\\\\\\\/cZDWDdJagJfwalS+ueI+aV2IRqquErO7K\\\\\\\/ByISuLHqIHcJ1W8w4A\\\\\\\/uVx6asqQmpoY4JzoGtsIQiDcvdpLnFNuLJZtsz6VsiSqxiXiIPRmex6gCBsOIxz6p3HpWon6KVypPzZ1IHrG98bdVs8ZNgyAHW+OwNqSUhCEQSyaEIv722sxKbZcTebO9QKN0tst\\\\\\\/2dDjc+GGn5OLGZfYaHIZ+Nw5Hevq1LdgRdwDL\\\\\\\/pj6mdCQSNQlXEHpCGSeJzG5FlEpUoIYFg1QpuV+Fljsdrk\\\\\\\/Uqv3YOCB4luOUse6zcuE6ygClHCiLE3omH5hR\\\\\\\/lcgvT922uDcKV7wY4wTntsPt90Ax7g5BgTSppKoSGae1QvnfFsKjeg6JuLTbglMyN1CpozyvhmZ7wjM0AtuSAOKnr5NvmhytJv2lEEgUZNlTxCosJeXzMe6DLryTt4+4lupZt1k9XVv8q2us\\\\\\\/YoXzlS3j8NB9sxVA7R2nxuG5vTcY4iHSK6uJqTtU5eNvjMgW321OOMY5311Rh6MbXXMI8MQt\\\\\\\/DJuqWuFYQuknyK44Yf\\\\\\\/LlfCM2sy3gkydCTlnMi3yx+Hr4n6VcrN0Hq3ml8vGRhPUOkudjDkeewvvA6L+FVoHBtYZtk2sJ1TF2OEDNkvfZayZk05TnXIxF+u\\\\\\\/gTXge+M9ojk9CJMTGWNmfFHbsk0eGWkIpeUf3Jr5xPC4s4S9bKakadYs\\\\\\\/\\\\\\\/WNUVUY96izpzIKIVQBdfrpckWdSTiyxpABMyX9vT50Eo0ONSqoU5YtseUCSRTNi4W3eIdWRxXaj3RPTHrRGjO2\\\\\\\/wB1sCaqyf4EoqS2GOHfD0JzK61sn0B7xDvtgMWgH6P9xVvFIZy4sM97tGHRXAkGdhII5KphwyN7R3f7ngGkdljyzjnWVIICKTAzSqdiqAk2N1c8k9hSfaE+6O1dgxvlyGxmT65bo\\\\\\\/KpGydJLm0p9YWCYhhnDhZy7mQlx\\\\\\\/tY9G1qMF\\\\\\\/sSOnsl1YQzeNSalfmFmQM0+LXTqQOs6CX8VD5ayFXaweuA4RTP0l98gPmAf9Mp6\\\\\\\/9uw6nm8yXE62Ma6CtpZAWro3zFFJ4FYtWS\\\\\\\/tNa+iBrvbmK3d75ZWEmI3GeGAOLWntv8Vo\\\\\\\/YzzzEQv2mBEdFiGYtGR\\\\\\\/lcOnhGt+xhTh70BWGg9VI7JfzMk8by5igsOI\\\\\\\/VYfhfTbuAQZ97uASI01OxraBueD+IcX1mjaOIKEQGHSGtUwoZomwumd2wdReiS\\\\\\\/E77PJJNdZEcc3fg0JTGFykwRbIQPPWUhLFTUz1MlcQysX1SHHllC4yBqXwVTQ16TrCje3O4Ro0UnZEBi1WoTAkDFlCaT2e90w0bDfhtObP3siKv4AHDw+Y3CE8j4QDJZ7QySwRAW7BMQaLH9lAPDv40zOERLSt2C0JJyhtploFzE4YBpQYn\\\\\\\/2U0IowqHykESPxpybned+LS5JJZItvlFN8wdM9c4ooRH0XwzjHOJCPP7VMIja3T4iWVTERwLQSCiiEKS+sRLCtP0BVzmpdfHlw++hwrdyXuYQyu\\\\\\\/YTnklsDvEd0GwAkdof+c6E6sxiDOVYcJjTg9P1CcE6OeXfLBUgg23NKH3r8NA6A4Yd873ALAD30btI3xrCp7HrgaDUAjgktpCPfwAOfamfOMslP7X\\\\\\\/agYxlkPJUFfPLwLfTWuEeOlXmJDKlR8Ru0cYMpoB+JL5QyHAkyR76CvMBCYJjjiIReH6j5yvmL0enHso5ehiAtkU0Ci1bWlbLFKuPKx85eql\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kt01Phz7riGr2uPk+EHlGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022784a6fd64e275c307f39e9518ac4967c2f89d3c37ea6e10025004f68f441f782\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FKUF4v\\\\\\\/ELstuvsnKaifYMTEo+bPuflTHTkcz+cC5FPKgJ6kYw8E8KKDCyw\\\\\\\/VdaxWNKIz8G9+IFRArNYuhUa1zWL8+4+uKwFA8SZg4LXFNODXwwyMDrxhYxbJLXqrmPi3A0WoL4AgYSNeYPhk1ycku3tX\\\\\\\/TQh3NgvzPwWgvQWPmAV\\\\\\\/eBA7bECz2NcabZoMrhTc3bNKoadq+fZmlOOBTW8qzs+iKcxc4ff6yLTllzjGNh8xh00G\\\\\\\/GwSzK7kYYVWwf6uD1YUB7SBgv1M\\\\\\\/G1vpa+9V6jCu4YCnAr2ptyur8M3GNGOwypMNOUNjpJtzYrVUe0L8EJ55j0tJHRpMCynkqDLPTuwmq0I77CqRSRwneXUMP9r\\\\\\\/VKDYUMjlHpy+TvuP3sXlQqGPPJY1B0\\\\\\\/wtD0Jg+6LO\\\\\\\/7+Q16vVgtfgIH\\\\\\\/QwL3XMz0x2Zc72036kpylSo9a6lQHSQ+u58Ce2MWAus4i0\\\\\\\/9CRZhektToIVfMna3kvZPPX3lqnhTSIG4V4BIqIq7gAS6R7XRacSnaOYg3AzEoFCBpRCyd5N12nQTH1uoNvqICjac0dP0ngbCwsT84e9wsShysJy9dtBjLtVEum0Ao3+kI\\\\\\\/1qRSb7tWqn2CeOtwjbEogNyAx\\\\\\\/jILAeNXBMZyOpSrgh2xUV9AOzo4CDX5QXnNM8m4loHKzZp0VAVbPmiQBKrGRGrUlbh\\\\\\\/97GCHEB4NV84WjOu\\\\\\\/Bkf7Z8CEXyfRLbJKTNnWMFTK4K4ofHiVVSkmpnDhmaE2+wYdyKGmpNrZPMi5Y4tdwsNf78An6jRvlsG6ntsHzhxxuu3dP9Qn6rCl7JpOyAk0vdNJU3OuD84Wc3MOQywv5COQepHX+U\\\\\\\/60FxNhUY0Qz3auLJlDgIGDjeSwPnfvSnTYbd07WLbkUZnI2FPHQ6lQvJCgz7uTWanB2+O0li0AlumfgWiofMRVYYpX+hdmDIimAkiBZvT5ARZXXoCSwG3sP0xAPBq8CmbR8zLHMXsWo3yaFtc4WHkLbN3sZirRXvJAC9F\\\\\\\/n8GFP5bZanlR1QRS5Akx+wFBIxzOTl5Xd9RO49jm\\\\\\\/jlmvZcj2GLi897uKKDHqkpdoHsn1Z\\\\\\\/FfzCsz8yUNs6y3bpPOspTVmj\\\\\\\/QthYC+A8rim0aA1pGVXANF2SfrJSgcgZ9O8R7iNt449pgTI76mUri5UWzr9iHrXApeTldG8J47TvQMqQ5C7fYlugyCDTi\\\\\\\/omF0Eri6HAZC2KgNI+L1GFp26QKrW9W0kLxMZEtxRj+NX3pDLvnJV+iiWlE7rD4gawTuWR2TfVHGPQxItEwAUFmSdbMEDY9SpjrkV4oiRNyP\\\\\\\/JKoG50\\\\\\\/ZD3MrdTlC8EQZ7ciEZxF21\\\\\\\/Z85nEoRNtETNaiA\\\\\\\/jeRyRms4JnV8kV1qA+bYZt95Z1NxUIAScXnOHkNBupWsBaLFefQR4qiknGDmsiJFv5qm8pomTxEQADSCicB0M+qogfhIvPfSwfLmc0C+PKqiJBlC\\\\\\\/AtT1afanSo6K4HlvD+jadE5\\\\\\\/mRk\\\\\\\/bH+Lz6a7\\\\\\\/S5AY1mlehWadIX4U3XgSVsKH1XiJOMBIA1ShQzEZqNWdqOR41UPODN1hSWGqoN\\\\\\\/RnkJ1XWp309vJbvTlQ4SX8CwiD1K8REpWdnWG52YFeXfX3p2pRyaX6qp4sbQiJM0ATpopoEgJrsHuTZ4wmqCwNo8p0tzA1TLcnQwu9bgu\\\\\\\/VqRb0YzSE2z3xOvgpmGe8A6OZSc5EaJmf7IxoXTpn+jJCebc4UF2\\\\\\\/2vqGstf4dH7mcTKv0LMT1PvIvl5kNiIkBDGajzNG0PQiKXJQ7Hu7u+dxw5k4nSL338cg7RO8QVApQfGI6hbUtCJ6IGFREAOwj7duKuiI64R1aIIEbctTscUOyNN2Rflg55YS5cDwnnfgdE0vxP5mSJkcB24j+d7zlP\\\\\\\/i46foDpyyXq3BuGe\\\\\\\/\\\\\\\/Iyqja5SMdHHrUUj5jmW\\\\\\\/lf+qOOcJtsHt8darsDmHXc8EuvNMSsdo80RvEhuDZFH6IAIAnnWTnV\\\\\\\/oFouidgAXgymWr\\\\\\\/DYdp7Wry0GrmoX9Vft\\\\\\\/3gNuVImEFC0gAojDWICT0K1uv0\\\u0022,\\\u0022iv\\\u0022:\\\u0022CH8GH+keelVwd77VBR5MLQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d8de7332c242c098981729745e8a91db84e067493fef1927b9129764d600b557\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491669 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3915" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M24QZBIRE6uCr0yohV8G3CvYXSwO2YIXDjShabI9j8M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3915" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c8ltgJnkCEUwfA85Y5_Vs6RPUF7QITouKGRHfLQdhKM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3903" title="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3931" title="The Yeast DNA Damage Checkpoint Kinase Rad53 Targets the Exoribonuclease, Xrn1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491669" id="node491669" data-pisa="ggg;8/12/3915" data-pisa-master="ggg;g3.118.200553" data-apath="/ggg/8/12/3915.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of <em>Aspergillus flavus</em> S and L Morphotypes Yield Insights into Niche Adaptation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mana Ohkura</span>, <span class="highwire-citation-author" data-delta="1">Peter J. Cotty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marc J. Orbach</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3915-3930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200553 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491669" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mana Ohkura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mana%2BOhkura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohkura%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMana%2BOhkura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter J. Cotty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BCotty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cotty%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BCotty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marc J. Orbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BJ.%2BOrbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orbach%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarc%2BJ.%2BOrbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marc J. Orbach</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orbachmj@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">orbachmj@email.arizona.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3915" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491669" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491669/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3915.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491669" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491669/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3915.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491669" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491669/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3915.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491669" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491669/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3915.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491669" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491669/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Aspergillus flavus</em>, the primary causal agent for aflatoxin contamination on crops, consists of isolates with two distinct morphologies: isolates of the S morphotype produce numerous <span class="underline">s</span>mall sclerotia and lower numbers of conidia while isolates of the L morphotype produce fewer <span class="underline">l</span>arge sclerotia and abundant conidia. The morphotypes also differ in aflatoxin production with S isolates consistently producing high concentrations of aflatoxin, whereas L isolates range from atoxigenic to highly toxigenic. The production of abundant sclerotia by the S morphotype suggests adaptation for long-term survival in the soil, whereas the production of abundant conidia by the L morphotype suggests adaptation for aerial dispersal to the phyllosphere. To identify genomic changes that support differential niche adaption, the sequences of three S and three L morphotype isolates were compared. Differences in genome structure and gene content were identified between the morphotypes. A &gt;530 kb inversion between the morphotypes affect a secondary metabolite gene cluster and a cutinase gene. The morphotypes also differed in proteins predicted to be involved in carbon/nitrogen metabolism, iron acquisition, antimicrobial defense, and evasion of host immunity. The S morphotype genomes contained more intact secondary metabolite clusters indicating there is higher selection pressure to maintain secondary metabolism in the soil and that it is not limited to aflatoxin production. The L morphotype genomes were enriched in amino acid transporters, suggesting efficient nitrogen transport may be critical in the nutrient limited phyllosphere. These findings indicate the genomes of the two morphotypes differ beyond developmental genes and have diverged as they adapted to their respective niches.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/aspergillus-flavus" class="hw-term hw-article-keyword hw-article-keyword-aspergillus-flavus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Aspergillus flavus</em></a></li><li class="kwd"><a href="/keyword/aflatoxin" class="hw-term hw-article-keyword hw-article-keyword-aflatoxin" rel="nofollow">aflatoxin</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/sclerotia" class="hw-term hw-article-keyword hw-article-keyword-sclerotia" rel="nofollow">sclerotia</a></li></ul><p id="p-3">The saprotrophic fungus, <em>Aspergillus flavus</em>, is the primary causal agent of contamination in food crops by aflatoxins, highly carcinogenic secondary metabolites produced by members of the genus <em>Aspergillus</em> section <em>Flavi</em> (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Schroeder and Boller 1973</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Klich 2002</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Klich 2007</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Mehl <em>et al.</em>, 2012</a>). Aflatoxin contamination causes both economic losses and health problems in both humans and animals. This is a serious problem in developing countries where aflatoxin contamination is not regulated; aflatoxin-contaminated food and feed result in loss of export sales and health problems ranging from stunting of growth, immune suppression, cancer and in acute cases, death (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Goldblatt 1969</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bennett and Klich 2003</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Probst <em>et al.</em>, 2007</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Richard <em>et al.</em>, 2003</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Williams <em>et al.</em>, 2004</a>). Repeated aflatoxicosis outbreaks have occurred in Kenya, and in 2004, one of the largest documented outbreaks resulted in a total of 317 cases being reported and 125 deaths (CDC, 2004; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Probst <em>et al.</em>, 2007</a>). In developed countries, regulations for aflatoxin levels in food protect consumers from aflatoxin poisoning, but great economic losses are incurred as contaminated crops must be destroyed or de-contaminated (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Richard <em>et al.</em>, 2003</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Mitchell <em>et al.</em>, 2016</a>).</p><p id="p-4">Although <em>A. flavus</em> is notorious for its ability to produce aflatoxin, isolates within the species vary in levels of toxin production and atoxigenic isolates are not uncommon. Within <em>A. flavus</em> are isolates with two distinct morphologies, the S (<span class="underline">s</span>mall sclerotia) and L (<span class="underline">l</span>arge sclerotia) morphotypes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cotty 1989</a>), characterized by sclerotial size. S morphotype isolates produce numerous small sclerotia and conidiate poorly, whereas L morphotype isolates produce relatively fewer large sclerotia and abundant conidia (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cotty 1989</a>; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The morphotypes differ in toxigenicity as well: S isolates are consistently toxigenic, whereas L isolates vary greatly in toxin production ranging from atoxigenic to highly toxigenic isolates (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Saito <em>et al.</em>, 1986</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Cotty 1989</a>). On average, S isolates produce higher levels of aflatoxin than L isolates (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Cotty 1989</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cotty 1997</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Probst <em>et al.</em>, 2010</a>) and have etiological implications as causal agents in outbreaks (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Cotty 1997</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Garber and Cotty 1997</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bock <em>et al.</em>, 2004</a>; <a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Probst <em>et al.</em>, 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cotty <em>et al.</em>, 2008</a>). Atoxigenic L isolates are of particular importance as successful biological control agents that outcompete toxigenic isolates from contaminating crops (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Garber and Cotty 1997</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Cotty <em>et al.</em>, 2008</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Mehl <em>et al.</em>, 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Colony morphology of A. flavus isolates used in genomic analyses of this study. Cultures were grown on complete media at 30 &#xB0;C in the dark for 10 days. (A) Colonies of S morphotype isolates displaying limited conidial sporulation and abundant production of small sclerotia. From left to right; AF12, AF70, AZS. (B) Colonies of L morphotype isolates displaying abundant conidial sporulation and limited or no production of large sclerotia. From left to right; BS01, DV901, MC04. (C) Closer view of small sclerotia produced by AZS (left) and large sclerotia produced by DV901 (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Colony morphology of &lt;em&gt;A. flavus&lt;/em&gt; isolates used in genomic analyses of this study. Cultures were grown on complete media at 30 &#xB0;C in the dark for 10 days. (A) Colonies of S morphotype isolates displaying limited conidial sporulation and abundant production of small sclerotia. From left to right; AF12, AF70, AZS. (B) Colonies of L morphotype isolates displaying abundant conidial sporulation and limited or no production of large sclerotia. From left to right; BS01, DV901, MC04. (C) Closer view of small sclerotia produced by AZS (left) and large sclerotia produced by DV901 (right).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F1.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3915/F1.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491601" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Colony morphology of <em>A. flavus</em> isolates used in genomic analyses of this study. Cultures were grown on complete media at 30 °C in the dark for 10 days. (A) Colonies of S morphotype isolates displaying limited conidial sporulation and abundant production of small sclerotia. From left to right; AF12, AF70, AZS. (B) Colonies of L morphotype isolates displaying abundant conidial sporulation and limited or no production of large sclerotia. From left to right; BS01, DV901, MC04. (C) Closer view of small sclerotia produced by AZS (left) and large sclerotia produced by DV901 (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Developmental and biochemical differences between <em>A. flavus</em> S and L isolates suggest they are adapted to different niches: S isolates produce abundant sclerotia that are advantageous for long-term survival in the soil, whereas L isolates produce fewer sclerotia, but large numbers of conidia that are advantageous for aerial dispersal to the phyllosphere (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Mehl <em>et al.</em>, 2012</a>). A study by <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Sweany <em>et al.</em> (2011)</a> provides support for this differential reproductive strategy: they report that more than 95% of isolates recovered from maize kernels in a study in Louisiana were of L morphotype, whereas only 44% of soil isolates were of L morphotype with the remainder consisting of S morphotype. A similar pattern was observed when comparing the frequency of S isolates and L isolates recovered from maize kernels and groundnut in Malawi (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ching’anda <em>et al.</em>, 2016</a>). Differences in aflatoxin toxigenicity observed between the morphotypes also suggest differential niche adaptation; S isolates are consistently highly toxigenic, while L isolates vary in their toxigenicity including atoxigenic and low toxin-producing isolates. This suggests selection pressure to maintain aflatoxin production varies between the soil and phyllosphere with pressure being high in the soil, leading to the retention of toxigenic S isolates, while it is lower in the phyllosphere allowing the survival of atoxigenic or low toxin-producing L isolates (<a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Mehl <em>et al.</em>, 2012</a>). This is also evidenced by lower proportions of atoxigenic isolates recovered from soil than from the phyllosphere (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bilgrami and Choudhary 1993</a>; <a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Sweany <em>et al.</em>, 2011</a>).</p><p id="p-7">In this study, we compared the genomes of S and L morphotype isolates to test the hypothesis that there are genomic differences that reflect their adaptation to the soil and phyllosphere respectively. The soil and phyllosphere are contrasting environments in terms of ambient conditions, nutrient availability, and microbial competition. The soil is buffered from changes in the atmosphere in contrast to the phyllosphere where microbes are exposed to high levels of ultraviolet light and fluctuations in temperature (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Lindow and Brandl 2003</a>; <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Whipps <em>et al.</em>, 2008</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Vorholt 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bulgarelli <em>et al.</em>, 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bringel and Couée 2015</a>). Nutrient availability is greater in the soil than the phyllosphere due to the presence of root exudates and degradation of organic matter by microorganisms (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Rengel and Marschner 2005</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Turner <em>et al.</em>, 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Dotaniya and Meena 2015</a>). In contrast, the leaf surface is covered by a waxy cuticle layer that limits passive diffusion of water and nutrients (<a id="xref-ref-109-2" class="xref-bibr" href="#ref-109">Whipps <em>et al.</em>, 2008</a>; <a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Vorholt 2012</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bringel and Couée 2015</a>). In recent years, metagenomic studies of the soil/rhizosphere and phyllosphere microbiomes show evidence that microbial communities that inhabit these niches are different as well (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Delmotte <em>et al.</em>, 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Knief <em>et al.</em>, 2012</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Lundberg <em>et al.</em>, 2012</a>). The soil environment is rich, not only with microbes, but also with small animals such as insects and worms . On the other hand, species diversity on leaf surfaces is relatively low, with bacteria being the most abundant organisms (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Lindow and Brandl 2003</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Delmotte <em>et al.</em>, 2009</a>). Therefore, if the S and L morphotypes are adapted to the soil and phyllosphere respectively, they would require different mechanisms to withstand abiotic stresses, acquire nutrients, and compete or defend against organisms they encounter. Here we identify genomic differences between the <em>A. flavus</em> morphotypes that we propose have allowed them to selectively adapt to their respective niches.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><div id="sec-2" class="subsection"><h3>Genome sequencing, assembly, and annotation</h3><p id="p-8">The genomes of three S morphotype isolates (AF12, AF70, and AZS; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; Table S1) and three L morphotype isolates (BS01, DV901, and MC04; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>; Table S1) were 2 × 100 bp paired-end sequenced using the Illumina HiSeq 2000 (San Diego, CA) platform at the Arizona Genomics Institute. Nucleotide coverage for each genome was 40x (AF12), 42x (AF70), 40x (AZS), 120x (BS01), 44x (DV901), and 123x (MC04). The genomes of BS01 and MC04 were resequenced due to low coverage in the first sequencing run. Genomic DNA was extracted from cultures grown in complete medium (1% glucose, 0.3% yeast extract, 0.3% casein hydrolysate) for 3 days at 31°. The extraction was performed using a modified protocol from <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Kellner <em>et al.</em> (2005)</a> by harvesting mycelia by centrifugation and mechanically lysing the cells using a pestle. Sequence reads were quality filtered (phred score &gt; 15 for 4-base sliding window) with Trimmomatic v0.22 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bolger <em>et al.</em>, 2014</a>). Optimal k-mer values to use for assembly were determined using VelvetOptimizer and by assessing the completeness of the assembly against the <em>A. oryzae</em> RIB40 genome (a domesticated species of <em>A. flavus</em>) using QUAST (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gurevich <em>et al.</em>, 2013</a>). Genomes were assembled with Velvet v1.2.10 using following k-mers: 55 (AF12), 51 (AF70), 53 (AZS), 61 (BS01), 57 (DV901), and 63 (MC04).</p><p id="p-9">The genomes were annotated by MAKER v2.31 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Holt and Yandell 2011</a>) using following parameters: A training set of genes was obtained by SNAP (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Korf 2004</a>), which was trained using the <em>A. oryzae</em> model available at <a href="https://github.com/hyphaltip/fungi-gene-prediction-params/blob/master/params/SNAP/aspergillus_oryzae.length.hmm">https://github.com/hyphaltip/fungi-gene-prediction-params/blob/master/params/SNAP/aspergillus_oryzae.length.hmm</a>. AUGUSTUS (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Holt and Yandell 2011</a>) predicted many gene fusions and thus was not used to predict genes for training. For protein evidence, proteins from <em>A. oryzae</em> RIB40 (Aspgd version s01-m08-r20; 11902 proteins) and <em>A. flavus</em> NRRL3357 (Uniprot proteins that were reviewed, have protein or transcript evidence, or have homology to other organisms; 1843 proteins) were used. To avoid fragmented gene calls, genes were only predicted on contigs larger than 1 kb. Single exon genes were allowed and only gene models with protein evidence were accepted. To obtain the final set of gene calls, SNAP (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Korf 2004</a>) and AUGUSTUS (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Holt and Yandell 2011</a>) were trained on the training set of gene models, and MAKER (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Holt and Yandell 2011</a>) was run using the same parameters as the training round, but allowing prediction of genes without protein evidence. Gene calls were visualized using Apollo (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Lee <em>et al.</em>, 2013</a>) to confirm they aligned with the protein evidence. InterProScan 5.14.53 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Jones <em>et al.</em>, 2014</a>) and SwissProt 2014_08 release (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boeckmann <em>et al.</em>, 2003</a>) were used to predict functions of the gene models.</p></div><div id="sec-3" class="subsection"><h3>Identification of structural variations between morphotypes</h3><p id="p-10">To identify large structural variations between the morphotypes, contigs of each genome were assembled into pseudogenomes based on the chromosomes of <em>A. oryzae</em> RIB40 using CoGE Synmap (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Lyons and Freeling 2008</a>). Synteny analysis was performed by comparing each pseudogenome against the remaining five genomes using Symap v4.2 (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Soderlund <em>et al.</em>, 2011</a>). In addition, the inversion on chromosome 8 was confirmed by PCR using primers listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; amplification of the 5′ half of <em>tsr1</em> was used as a positive control.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491642/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491642/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Primers used for phylogenetic analysis and confirmation of the inversion on Chromosome 8</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">To identify smaller structural variations between the S and L morphotypes, paired-end and split read analyses were performed (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Rausch <em>et al.</em>, 2012</a>). Contigs of AZS and DV901 were used as references for the S and L morphotypes respectively, because they had the highest N50 values indicating the best assemblies. Trimmed reads of AF12, AF70, BS01, DV901, and MC04 were mapped to contigs of AZS, and trimmed reads of AF12, AF70, AZS, BS01, and MC04 were mapped to contigs of DV901 with Bowtie2 v2.2.4 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Langmead and Salzberg 2012</a>). DELLY v0.6.1 (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Rausch <em>et al.</em>, 2012</a>) was used to identify deletions, duplications, transpositions, and inversions from the mapping output. Structural variants with at least one pair of mapping reads were kept, because structural variations specific to each morphotype were determined by its presence in all three genomes of the morphotype. To identify structural variants that are only present in the genomes of the S morphotype, first, the multiIntersect feature in BEDtools v2.23 (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Quinlan and Hall 2010</a>) was used to identify structural variants that AF12, AF70, and AZS had in common against DV901. Next, out of the structural variants that may be unique to the S isolates <em>vs.</em> DV901, those that were present in comparisons between DV901 and other L isolates (BS01 and MC04) were eliminated. Only S isolate variants that were not present in the L to L isolate comparisons were defined as S-specific. Similarly, to identify structural variants that are only present in the genomes of the L morphotype, first, the structural variants that BS01, DV901, and MC04 had in common against AZS were identified, and next, out of the variants, those that were present in comparisons of AF12 and AF70 against AZS were removed; thus, only L isolate variants that were not present in the S to S isolate comparisons were defined as L-specific. Deletions larger than 5 kb and all remaining structural variations identified by DELLY were confirmed manually using CoGe GEVO (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Lyons and Freeling 2008</a>) as well.</p></div><div id="sec-4" class="subsection"><h3>GO term enrichment and gene family expansion/contraction analyses</h3><p id="p-12">GO term enrichment analysis between genomes of the two morphotypes was carried out by the hypergeometric test (p-value &lt; 0.05) in OrthoVenn (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Wang <em>et al.</em>, 2015</a>). Protein families were determined by Pfam annotations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Finn <em>et al.</em>, 2016</a>) from InterProScan (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Jones <em>et al.</em>, 2014</a>). Protein families that were unique to each morphotype were confirmed by both BLASTn analysis (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em>, 1990</a>) of the genes against the genomes of the other morphotype and via the CoGE genome browser (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Lyons and Freeling 2008</a>): if the nucleotide sequence for the protein was present in genomes of the other morphotype, this indicated a possible misannotation in that morphotype, thus the protein family was removed from the list of unique proteins. Functional enrichment of Pfam categories between the two morphotypes was carried out using the hypergeometric test (p-value &lt; 0.05) in GeneMerge v1.4 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Castillo-Davis and Hartl 2003</a>).</p></div><div id="sec-5" class="subsection"><h3>Identification of morphotype-unique proteins</h3><p id="p-13">Morphotype-unique proteins were defined as those encoded by a gene in the same position in all three genomes of one morphotype that lacked orthologs in all three genomes of the other morphotype. Proteins from the six genomes were clustered using Orthofinder (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Emms and Kelly 2015</a>) and orthologous genes that are positionally conserved were clustered using progressiveMauve (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Darling <em>et al.</em>, 2010</a>). Clusters defined by both clustering methods were extracted and filtered to those that contain proteins from only the three S morphotype genomes or only the three L morphotype genomes. These clustering approaches can fail to include orthologs that are truncated or misannotated from its appropriate cluster; therefore, the clusters were further filtered by BLASTn analysis (e-value &lt;1e-10; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em>, 1990</a>) against all contigs of the other morphotype. Clusters that contained hits with ≥80% query coverage per subject and ≥80% identity were removed and remaining clusters were designated as morphotype-unique proteins. The chromosomal locations of these morphotype-unique proteins were visualized on their pseudogenomes using CViT v1.2.1 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cannon and Cannon 2011</a>).</p></div><div id="sec-6" class="subsection"><h3>Identification of genes under selection between morphotypes</h3><p id="p-14">To identify genes under selection, the POTION 1.1.2 pipeline (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hongo <em>et al.</em>, 2015</a>) was implemented on clusters from Orthofinder (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Emms and Kelly 2015</a>) that contained single-copy orthologs from all six genomes with &gt;80% sequence identity. Genes with evidence of recombination were identified with PhiPack (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bruen <em>et al.</em>, 2006</a>) and removed using default parameters. Multiple sequence alignment was performed using MUSCLE (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Edgar 2004</a>), protein-guided codon alignments were created using a subroutine within POTION (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Hongo <em>et al.</em>, 2015</a>), sequences were trimmed with trimAL (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Capella-Gutiérrez <em>et al.</em>, 2009</a>), and phylogenetic tree reconstruction was performed with PhyML (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Guindon <em>et al.</em>, 2009</a>). Likelihood ratio tests for selection were performed using the m12 (codeml M1a/M2) and m78 (codeml M7/M8) nested site models with PAML codeml (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Yang 2007</a>) to compare neutral models with models that allow positive selection. Genes under selection in each genome were identified relative to the three genomes of the other morphotype using a q-value &lt; 0.05 cutoff (corrected p-value for multiple testing) in either the m12 or m78 model test. Genes under selection that are morphotype-specific were identified as genes selected in all three isolates of a morphotype within a cluster.</p></div><div id="sec-7" class="subsection"><h3>Additional annotation of proteins: Secondary metabolite clusters, antibiotic resistance genes, CAZy, and peptidases</h3><p id="p-15">Identification of secondary metabolite clusters was performed by SMURF (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em>, 2010</a>). Proteins that are predicted to confer antibiotic resistance were identified using BLASTp provided in the comprehensive antibiotic resistance database (CARD; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Jia <em>et al.</em>, 2017</a>) using ‘strict’ as the cutoff. Carbohydrate-active enzymes (CAZy) in each genome were identified by analyzing protein sequences with HMMER3 against the dbCAN database (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Yin <em>et al.</em>, 2012</a>). Identification of peptidases and peptidase inhibitors was carried out by the MEROPS 10.0 batch BLAST program (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Rawlings <em>et al.</em>, 2012</a>). Proteins with morphotype-unique CAZy and MEROPS categories were confirmed by BLASTp analysis (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em>, 1990</a>) against the NCBI nr database (NCBI Resource Coordinators 2017) as well: if proteins with similar sequences in the database were annotated with a function different from that suggested by the CAZy or MEROPS categories and the NCBI annotation was functionally validated, the NCBI annotations took precedence.</p></div><div id="sec-8" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-16">To evaluate whether the &gt;530 kb inversion may have led to the divergence of S and L morphotypes, a phylogenetic analysis that includes isolates from diverse geographic regions was performed using DNA sequences from the 5′ half of <em>tsr1</em> (ribosome biogenesis protein), 3′ half of <em>tsr1</em>, and <em>cmd</em> (calmodulin) genes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Genomic DNA of eight <em>A. flavus</em> S isolates, 22 <em>A. flavus</em> L isolates, and four outgroup isolates (Table S1; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Cotty 1989</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Probst <em>et al.</em>, 2012</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Probst <em>et al.</em>, 2014</a>) was extracted from 7 day-old cultures grown at 31° in the dark on V8 agar (5% V8 vegetable juice, 2% NaCl, and 2% agar, pH 5.2). DNA extraction was performed using the method of <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Callicott and Cotty (2015)</a>. Polymerase chain reactions (PCR) of each locus were performed using the primers and annealing temperatures listed in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. The reactions were performed in 25 μL volumes using 3 ng of genomic DNA and Promega GoTaq Green Master Mix (Madison, WI), following the manufacturer’s protocol. Amplicons were sequenced at the University of Arizona Genetics Core using an Applied Biosystems 3730 DNA analyzer. Forward and reverse sequences were edited and assembled using SeqTrace 0.9.0 (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Stucky 2012</a>). For each gene, sequences were aligned in MUSCLE (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Edgar 2004</a>) and manually edited, trimmed, and assigned codon positions in Mesquite (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Maddison and Maddison 2017</a>). The three loci were concatenated and submitted to PartitionFinder v2.1.1 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Lanfear <em>et al.</em>, 2016</a>) to infer the best-fitting model of evolution among those available in RAxML (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Stamatakis 2006</a>) based on codon position and locus. The concatenated dataset was partitioned into five subsets, which were then implemented in RaxML for maximum likelihood analysis (<a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Stamatakis 2006</a>) using CIPRES (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Miller <em>et al.</em>, 2010</a>). Outgroup taxa consisted of <em>A. minisclerotigenes</em><sub>,</sub> and support was assessed using 1000 maximum likelihood bootstrap replicates.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-17">Genome sequences can be accessed at NCBI under the following accession numbers: NLCN00000000 (AF12), NLCM00000000 (AF70), NLCL00000000 (AZS), NLCK00000000 (BS01), NLCJ00000000 (DV901), and NLCI00000000 (MC04). The genomes are listed under BioProject PRJNA393333. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6721511">https://doi.org/10.25387/g3.6721511</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Genome assembly and annotation</h3><p id="p-18">A summary of the genome assemblies and annotations are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The genome assemblies of the S isolates ranged in size from 38.1 Mb – 38.3 Mb (N50 values of 0.86 Mb – 1.14 Mb), and those of the L isolates ranged in size from 37 Mb – 37.5 Mb (N50 values of 0.93 Mb – 1.03 Mb). The predicted number of genes ranged from 13,368 – 13,374 for the S morphotype genomes, and 13,256 – 13,292 for L morphotype genomes. On average, the S morphotype genomes were larger by 1 Mb and contained 99 more genes. These assemblies are similar in size and in gene numbers to the previously sequenced genomes of <em>A. flavus</em> NRRL3357 (L morphotype; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Nierman <em>et al.</em>, 2015</a>) and <em>A. oryzae</em> RIB40 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Machida <em>et al.</em>, 2005</a>) that are both 37 Mb in size with 13,485 predicted genes and 12,074 predicted genes, respectively. Predicted repetitive DNA content was slightly higher in the S morphotype genomes ranging from 1.7–1.86% relative to the L morphotype genomes that ranged from 1.29 – 1.49%. These values are considered to be an underestimate of the actual repetitive DNA content because repetitive regions do not assemble well from short sequence reads.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491634/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491634/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of genome assembly and annotation</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Structural variations between morphotypes</h3><p id="p-19">Synteny analysis revealed an inversion on chromosome 8 between the S and L morphotype genomes that is ≈531 kB and ≈551 kB in the L morphotype and S morphotype genomes respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2</a> and S1). Using AZS and DV901 to represent the S and L morphotype genomes respectively, the inversion contained 213 genes in AZS and 208 genes in DV901. Most genes in the inversion had orthologs between AZS and DV901, except for three genes that were identified as S morphotype-unique proteins (morphotype-unique protein results are presented below); one that was identified as an L morphotype-unique protein, and one that was present in S isolates in a region deleted in the L morphotype genomes but that was not detected in the morphotype-unique protein analysis. In the S morphotype genomes, the borders of the inversion affect two secondary metabolite gene clusters, one on each end. The inversion starts at the end of a polyketide synthase (PKS) gene (gene c and inversion breakpoint A in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and ends at the end of another PKS gene (gene l and inversion breakpoint B in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). In the L morphotype genomes, the inversion starts at a PKS gene that is a fused product of the two PKS genes that are at the margins of the inversion in the S morphotype genomes (gene c’ + l’ and inversion breakpoint C in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), and ends within a cutinase gene that is not encoded in the S morphotype genomes (gene p and inversion breakpoint D in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). An exception to this occurs in the DV901 genome, where the cutinase gene was not predicted due to a SNP that resulted in a premature stop codon. The presence of the inversion was confirmed by PCR using the primers depicted at each inversion breakpoint in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and listed in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. Primer pairs used to detect inversion breakpoints A and B (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>) only yielded amplicons in the S morphotype isolates and primer pairs used to detect inversion breakpoints C and D only yielded amplicons in the L morphotype isolates (data not shown).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic diagram of the chromosomal inversion on chromosome 8 using genomes of AZS and DV901 to represent the S and L morphotype, respectively. Genes in the vicinity of the inversion breakpoints are shown and regions of the inversion are indicated with gray bars; inversion breakpoints are designated as A, B, C, and D to match the in-text description. Genes located at the inversion breakpoints are shaded darker than the other genes. Arrows indicate the locations of PCR primers listed in Table 1. Orthologs between the two morphotypes are designated with the same lower case alphabet (a &#x2013; o). At the inversion breakpoints in the L morphotype, gene c&#x2019;+l&#x2019; represents a PKS gene that is a fused product of the two PKS genes (genes c and l) that are at the margins of the inversion in the S morphotype genomes, and gene p is a cutinase gene that is not encoded in the S morphotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A schematic diagram of the chromosomal inversion on chromosome 8 using genomes of AZS and DV901 to represent the S and L morphotype, respectively. Genes in the vicinity of the inversion breakpoints are shown and regions of the inversion are indicated with gray bars; inversion breakpoints are designated as A, B, C, and D to match the in-text description. Genes located at the inversion breakpoints are shaded darker than the other genes. Arrows indicate the locations of PCR primers listed in Table 1. Orthologs between the two morphotypes are designated with the same lower case alphabet (a &#x2013; o). At the inversion breakpoints in the L morphotype, gene c&#x2019;+l&#x2019; represents a PKS gene that is a fused product of the two PKS genes (genes c and l) that are at the margins of the inversion in the S morphotype genomes, and gene p is a cutinase gene that is not encoded in the S morphotype.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F2.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3915/F2.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">A schematic diagram of the chromosomal inversion on chromosome 8 using genomes of AZS and DV901 to represent the S and L morphotype, respectively. Genes in the vicinity of the inversion breakpoints are shown and regions of the inversion are indicated with gray bars; inversion breakpoints are designated as A, B, C, and D to match the in-text description. Genes located at the inversion breakpoints are shaded darker than the other genes. Arrows indicate the locations of PCR primers listed in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>. Orthologs between the two morphotypes are designated with the same lower case alphabet (a – o). At the inversion breakpoints in the L morphotype, gene c’+l’ represents a PKS gene that is a fused product of the two PKS genes (genes c and l) that are at the margins of the inversion in the S morphotype genomes, and gene p is a cutinase gene that is not encoded in the S morphotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In addition to synteny analysis, DELLY (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Rausch <em>et al.</em>, 2012</a>) revealed 31 deletions that are common to the S morphotype genomes relative to the L morphotype ranging in size from 307 bp – 12,939 bp, three of which were larger than 5kb. Manual inspection of deletions larger than 5kb revealed that the largest deletion was also present in L isolate MC04, and thus was removed from consideration. For the L morphotype genomes, there were 50 deletions relative to the S morphotype genomes that ranged in size from 151 bp – 14,979 bp, seven of which were larger than 5 kb. Manual inspection confirmed that all deletions larger than 5 kb were only present in the three L morphotype genomes.</p><p id="p-22">L morphotype genes that were at the margins of, or within deletions larger than 1 kb in S morphotypes are listed in Table S4 (genes present in the L morphotype but missing in the S morphotype) using genes in DV901 to represent the L morphotype. Only one deletion in the S morphotype genomes overlapped with secondary metabolite clusters predicted by SMURF (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em>, 2010</a>) and this deletion is unlikely to impact differential gene expression, because the gene is non-functional in both S and L isolates. The <em>cypA</em> gene of the aflatoxin gene cluster, that encodes a cytochrome P450 monooxygenase required for aflatoxin G production (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ehrlich <em>et al.</em>, 2004</a>) has deletions in both morphotypes and the deletion is larger by 575 bp in the S morphotype. L morphotype genes that were deleted in the S morphotype genomes also included genes that are predicted to encode proteins involved in carbon metabolism (a member of the glycosyl hydrolase family 12 and an alcohol dehydrogenase), nitrogen metabolism (a glutamine synthetase and a member of the NmrA-like protein family), and amino acid transport (a member of the proton-dependent oligopeptide transporter (POT) family) (Table S4).</p><p id="p-23">S morphotype genes that were at the margins of, or within deletions larger than 1 kb in L morphotypes are listed in Table S5 (genes present in the S morphotype but missing in the L morphotype) using genes in AZS to represent the S morphotype. Seven deletions in the L morphotype genomes occurred within secondary metabolite gene clusters predicted by SMURF (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em>, 2010</a>). S morphotype genes that were deleted in the L morphotype included genes that encode proteins predicted to be involved in secondary metabolism (two polyketide synthases) and detoxification or biosynthesis of compounds (four cytochrome P450s). A schematic diagram of the two largest deletions in the S and L morphotype genomes are depicted in Fig. S2.</p><p id="p-24">In addition to the deletions, DELLY (<a id="xref-ref-85-4" class="xref-bibr" href="#ref-85">Rausch <em>et al.</em>, 2012</a>) analysis revealed four inversions between the S and L morphotype genomes and one duplication in the L morphotype genomes; however manual inspection of these structural variants could not confirm their presence. There were no translocations that were detected by DELLY analysis (<a id="xref-ref-85-5" class="xref-bibr" href="#ref-85">Rausch <em>et al.</em>, 2012</a>).</p></div><div id="sec-13" class="subsection"><h3>GO term enrichment and gene family expansion/contraction analyses</h3><p id="p-25">GO terms that were enriched in the genomes of each morphotype are listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. S morphotype genomes were enriched in functions involved in secondary metabolism, such as isoquinoline alkaloid biosynthetic process (GO:0033075), (S)-stylopine synthase activity (GO:0047052), and N-methylcoclaurine 3′-monooxygenase activity (GO:0050593) that are involved in alkaloid biosynthesis or metabolism, as well as, branched-chain amino acid biosynthetic process (GO:0009082) and dihydroxy-acid dehydratase activity (GO:0004160) that can play a role in glucosinolate biosynthesis or in other pathways. S morphotype genomes were also enriched in processes generally involved in lipid metabolism including fatty acyl-CoA biosynthetic process (GO:0046949) and medium-chain fatty acid-CoA ligase activity (GO:0031956). In L morphotype genomes, enriched GO categories were exclusively involved in amino acid transport.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491698/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491698/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">GO terms enriched in S and L morphotype genomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Comparison of protein families between S and L morphotype genomes revealed 68 protein families that are unique or expanded in the S morphotype genomes with six of these being unique to the S morphotype (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The hypergeometric test (p-value &lt; 0.05) showed the DUF3589 protein family of unknown function (PF12141; p-value = 0.016) was significantly enriched in S morphotype genomes, with one protein containing two DUF3589 domains in the S morphotype, and none in the L morphotype genomes. CAZy annotation of the DUF3589 family protein predicts it is a β-1,2-mannosyltransferase and this is supported by sequence similarity to a β-mannosyltransferase of <em>Candida albicans</em> (KHC29884; 86% query coverage, 30% identity). S morphotype-unique protein families included proteins predicted to be involved in pyruvate metabolism (PF01326; pyruvate phosphate dikinase, PF00391 and PF028966; PEP-utilizing enzymes), an antifungal protein (PF11402), and a heme oxygenase (PF01126). S morphotype genomes also contained more copies of dehydratase (PF00920), chromate transporter (PF02417), ABC-2 transporter (PF01061), and ERG2 and Sigma 1 receptor like protein (PF04622) families.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491725/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491725/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Protein families (Pfams) unique to or significantly expanded in S or L morphotype genomes. P-values (&lt; 0.1) from the Pfam enrichment analysis are listed in the last column; p-values &lt; 0.05 are in bold. The full list of protein families with expansions in each morphotype is listed in Table S2 and S3</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">In L morphotype genomes, 59 protein families were expanded of which none were unique to the L morphotype (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The hypergeometric test showed the L morphotype genomes were significantly enriched in the LysM domain family (PF01476; p-value = 0.009) and Ankyrin repeats (3 copies) family (PF12796). LysM domains bind peptidoglycan or chitin, and in some plant pathogenic fungi LysM-containing proteins constitute effectors that allow evasion of host immune systems (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kombrink and Thomma 2013</a>). Among protein families expanded in L morphotype genomes were the POT domain (PF00854) and amino acid permease (PF00324); the latter finding supports the GO analysis that showed enrichment in amino acid transporters. In addition, there were more copies of protein families predicted to be involved in nitrogen metabolism, and these include arginase (PF00491), serine aminopeptidase S33 (PF12146), peptidase family M28 (PF04389), prolyl oligopeptidase (PF00326), and trypsin (PF00089) protein families.</p></div><div id="sec-14" class="subsection"><h3>Morphotype-unique proteins</h3><p id="p-28">To identify proteins unique to the S morphotype, clustering with Orthofinder (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Emms and Kelly 2015</a>) resulted in 304 clusters that consisted of proteins found only in the S morphotype genomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), and out of these, 279 were encoded by genes that were positionally conserved on contigs (positional orthologs). These clusters were filtered further by eliminating those that contained genes with significant BLASTn hits (≥80% identity and ≥80% query coverage) (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em>, 1990</a>) against contigs of L morphotype genomes. This reduced the total to 191 proteins unique to the S morphotype genomes. Similarly, to identify proteins unique to the L morphotype, clustering with Orthofinder (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Emms and Kelly 2015</a>) resulted in 189 clusters that consisted of proteins from only L morphotype genomes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), and out of these, 164 were encoded by positionally orthologous genes. Filtering the clusters further by BLASTn analysis (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em>, 1990</a>) reduced the total to 82 proteins unique to the L morphotype genomes. Mapping the morphotype-unique proteins on the pseudogenomes of AZS and DV901 revealed they are distributed across all chromosomes in both morphotypes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Numbers of morphotype-unique proteins identified using this approach is likely an underestimate, because morphotype-unique proteins that are due to pseudogenization of genes in one morphotype from the accumulation of SNPs will not be detected due to retention of high sequence similarity between the genomes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams representing the clustering results from Orthofinder to illustrate the number of proteins unique to and orthologous in each genome. (A) The number of clusters containing proteins unique to and orthologous in all six genomes; (B) the number of clusters containing proteins from only S morphotype genomes and (C) only L morphotype genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagrams representing the clustering results from Orthofinder to illustrate the number of proteins unique to and orthologous in each genome. (A) The number of clusters containing proteins unique to and orthologous in all six genomes; (B) the number of clusters containing proteins from only S morphotype genomes and (C) only L morphotype genomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F3.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3915/F3.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Venn diagrams representing the clustering results from Orthofinder to illustrate the number of proteins unique to and orthologous in each genome. (A) The number of clusters containing proteins unique to and orthologous in all six genomes; (B) the number of clusters containing proteins from only S morphotype genomes and (C) only L morphotype genomes.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal location of genes encoding morphotype-unique proteins (shown in red). Figures were created using CViT (Cannon and Cannon 2011). (A) Genes encoding S morphotype-unique proteins mapped on the pseudogenome of AZS. (B) Genes encoding L morphotype-unique proteins mapped on the pseudogenome of DV901." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosomal location of genes encoding morphotype-unique proteins (shown in red). Figures were created using CViT (Cannon and Cannon 2011). (A) Genes encoding S morphotype-unique proteins mapped on the pseudogenome of AZS. (B) Genes encoding L morphotype-unique proteins mapped on the pseudogenome of DV901.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F4.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3915/F4.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491671" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Chromosomal location of genes encoding morphotype-unique proteins (shown in red). Figures were created using CViT (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cannon and Cannon 2011</a>). (A) Genes encoding S morphotype-unique proteins mapped on the pseudogenome of AZS. (B) Genes encoding L morphotype-unique proteins mapped on the pseudogenome of DV901.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Morphotype-unique proteins with predicted Pfam, CAZy, MEROPS, and SMURF annotations are listed in Table S6 (S morphotype-unique proteins) and Table S7 (L morphotype-unique proteins). Below, we highlight morphotype-unique proteins that may play a role in niche adaptation, such as genes predicted to be involved in environmental response, gene regulation, secondary metabolism, and nutrient metabolism.</p><p id="p-32">The S morphotype-unique proteins included proteins proposed to be involved in environmental responses; one is in the SUR7/PalI family (PF01349; a putative environmental sensor), and another has a CS domain (PF04969) predicted to bind to Hsp90, suggesting it is also a putative environmental response protein. S morphotype-unique proteins also included several involved in gene regulation; four are likely transcriptional regulators (containing the fungal specific transcription factor domain; PF04082), two may be involved in post-transcriptional regulation (belonging to the Shwachman-Bodian-Diamond syndrome protein family; PF01172, involved in RNA metabolism), and two may play a role in epigenetic regulation (an RNA dependent RNA polymerase; PF05183, and a protein with a SET domain; PF00856, involved in histone methylation). Other S morphotype-unique proteins were involved in nutrient metabolism. Proteins that play a role in carbon metabolism included an alcohol dehydrogenase (PF08240), an isocharismatase (PF00857), two glucose-methanol-choline (GMC) oxidoreductases (PF05199, PF00732), a lactate/malate dehydrogenase (PF02866, PF00056), and a pyruvate phosphate dikinase (PF01326). Proteins that play a role in nitrogen metabolism included an astacin (PF01400), a DJ-1/PfpI peptidase (PF01965), a fumarylacetoacetate (FAA) hydrolase (PF01557), and a carbon-nitrogen hydrolase (PF00795). In addition, the S morphotype had unique proteins that indicate a role in microbial competition. Those that may confer antimicrobial activity included a polyketide synthase (PF14765), an antifungal protein (PF11402), and a MAC/perforin (PF01823). Others that may play a role in detoxifying antimicrobials or toxic compounds produced by themselves or by others included six cytochrome P450s (PF00067), a glyoxylase/bleomycin resistance protein (PF00903), and a lipopolysaccharide kinase (PF06293).</p><p id="p-33">The L morphotype-unique proteins contained a different set of regulatory proteins: transcriptional regulators including seven fungal specific transcription factors (PF04082, PF011951), putative post-transcriptional regulators (DEAD box RNA helicase; PF00270, PF00076), and epigenetic regulators (lysine methyltransferase; PF10294). Some L morphotype-unique proteins were involved in nutrient metabolism including carbon metabolism (alcohol dehydrogenase; PF08240), nitrogen metabolism (arginase; PF00491, trypsin; PF00089, and FAA hydrolase; PF01557), and lipid metabolism (acyl-CoA dehydrogenase; PF00441, PF02770, PF02771, and enoyl-CoA hydratase; PF00378). The L morphotype also had unique proteins that may confer antimicrobial activity (a protein with a Snoal-like domain; PF12680) or may detoxify compounds (RTA1-like protein; PF04479, cytochrome P450; PF00067).</p></div><div id="sec-15" class="subsection"><h3>Genes under selection</h3><p id="p-34">Analysis by POTION (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Hongo <em>et al.</em>, 2015</a>) identified 12 genes under selection in S morphotype genomes and 2 genes under selection in L morphotype genomes (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Genes under selection in S morphotype genomes were predicted to encode two fungal specific transcription factors (PF11951, PF04082), a polyprenyl synthetase (PF00348), a HET protein (PF06985), a eukaryotic cytochrome b561 (PF03188), a eukaryotic elongation factor 5A hypusine (PF01287), and other proteins with unknown functions. These genes were distributed across chromosomes 1, 2, 3, 5, 6, 7, 8, and an unknown contig (assembled from contigs that did not map to <em>A. oryzae</em> RIB40). The eukaryotic elongation factor 5A hypusine gene under selection was located within the large inversion identified on chromosome 8. Genes under selection in L morphotype genomes were predicted to encode a sugar transporter (PF00083) and a protein with an unknown function. These genes were located on chromosomes 3 and 5.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491728/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491728/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Genes under selection in S morphotype and L morphotype genomes and their q-values (corrected p-values for multiple testing) for the likelihood ratio tests using m12 and m78 nested site models</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Differences in secondary metabolite clusters, antibiotic resistance proteins, CAZy, and MEROPS peptidases between morphotypes</h3><p id="p-35">The number of secondary metabolite clusters and their backbone genes that were predicted by SMURF (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em>, 2010</a>) are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>. There was no notable difference in the number of secondary metabolite clusters between morphotypes, but S morphotype genomes contained 1 – 2 more copies of PKS-like proteins and L morphotype genomes contained 1 – 2 more copies of HYBRID PKS-NRPS proteins. Ten S morphotype-unique proteins belonged to five secondary metabolite clusters and three L morphotype-unique proteins belonged to three secondary metabolite clusters, indicating compositions of several secondary metabolite clusters differ between morphotypes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differences in copy numbers of proteins involved in secondary metabolism, antibiotic resistance, carbohydrate metabolism, and peptidase activity between S and L morphotype genomes. (A) Number of secondary metabolite clusters and their backbone genes predicted in each genome by SMURF (Khaldi et al., 2010). Abbreviations: prenyltransferases (DMAT), nonribosomal peptide synthase (NRPS), polyketide synthase (PKS), hybrid NRPS-PKS enzymes (HYBRID). (B) Number of proteins predicted to confer antibiotic resistance in each genome by CARD (Jia et al., 2017). Abbreviations: multidrug resistant ABC transporter (AbcA), aminocoumarin resistant (AlaS), Bifidobacteria intrinsic ileS conferring resistance to mupirocin (IleS), &#x3B2;-Glycosidase involved in Streptomyces venezuelae self-resistance (DesR), Mycobacterium tuberculosis ethA with mutation conferring resistance to ethionamide (EthA), M. tuberculosis katG mutations conferring resistance to isoniazid KatG, S. cinnamoneus elongation factor thermo unstable mutants conferring resistance to elfamycin (EF-Tu). (C) CAZy categories (Yin et al., 2012) with differences in copy numbers between S and L morphotype genomes. Abbreviations: carbohydrate-binding modules (CBM), carbohydrate esterases (CE), glycoside hydrolases (GH), glycosyltransferases (GT). (D) MEROPS categories (Rawlings et al., 2012) with differences in copy numbers between S and L morphotype genomes. Abbreviations: PfpI endopeptidase family/cysteine peptidase (C56), aminopeptidase Y family/metallo peptidase (M28), chymostrypsin family/serine peptidase (S01), D-Ala-D-Ala carboxypeptidase B family/serine peptidase (S12)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differences in copy numbers of proteins involved in secondary metabolism, antibiotic resistance, carbohydrate metabolism, and peptidase activity between S and L morphotype genomes. (A) Number of secondary metabolite clusters and their backbone genes predicted in each genome by SMURF (Khaldi &lt;em&gt;et al.&lt;/em&gt;, 2010). Abbreviations: prenyltransferases (DMAT), nonribosomal peptide synthase (NRPS), polyketide synthase (PKS), hybrid NRPS-PKS enzymes (HYBRID). (B) Number of proteins predicted to confer antibiotic resistance in each genome by CARD (Jia &lt;em&gt;et al.&lt;/em&gt;, 2017). Abbreviations: multidrug resistant ABC transporter (AbcA), aminocoumarin resistant (AlaS), Bifidobacteria intrinsic &lt;em&gt;ileS&lt;/em&gt; conferring resistance to mupirocin (IleS), &#x3B2;-Glycosidase involved in &lt;em&gt;Streptomyces venezuelae&lt;/em&gt; self-resistance (DesR), &lt;em&gt;Mycobacterium tuberculosis ethA&lt;/em&gt; with mutation conferring resistance to ethionamide (EthA), &lt;em&gt;M. tuberculosis katG&lt;/em&gt; mutations conferring resistance to isoniazid KatG, &lt;em&gt;S. cinnamoneus&lt;/em&gt; elongation factor thermo unstable mutants conferring resistance to elfamycin (EF-Tu). (C) CAZy categories (Yin &lt;em&gt;et al.&lt;/em&gt;, 2012) with differences in copy numbers between S and L morphotype genomes. Abbreviations: carbohydrate-binding modules (CBM), carbohydrate esterases (CE), glycoside hydrolases (GH), glycosyltransferases (GT). (D) MEROPS categories (Rawlings &lt;em&gt;et al.&lt;/em&gt;, 2012) with differences in copy numbers between S and L morphotype genomes. Abbreviations: PfpI endopeptidase family/cysteine peptidase (C56), aminopeptidase Y family/metallo peptidase (M28), chymostrypsin family/serine peptidase (S01), D-Ala-D-Ala carboxypeptidase B family/serine peptidase (S12).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F5.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3915/F5.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491547" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Differences in copy numbers of proteins involved in secondary metabolism, antibiotic resistance, carbohydrate metabolism, and peptidase activity between S and L morphotype genomes. (A) Number of secondary metabolite clusters and their backbone genes predicted in each genome by SMURF (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Khaldi <em>et al.</em>, 2010</a>). Abbreviations: prenyltransferases (DMAT), nonribosomal peptide synthase (NRPS), polyketide synthase (PKS), hybrid NRPS-PKS enzymes (HYBRID). (B) Number of proteins predicted to confer antibiotic resistance in each genome by CARD (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Jia <em>et al.</em>, 2017</a>). Abbreviations: multidrug resistant ABC transporter (AbcA), aminocoumarin resistant (AlaS), Bifidobacteria intrinsic <em>ileS</em> conferring resistance to mupirocin (IleS), β-Glycosidase involved in <em>Streptomyces venezuelae</em> self-resistance (DesR), <em>Mycobacterium tuberculosis ethA</em> with mutation conferring resistance to ethionamide (EthA), <em>M. tuberculosis katG</em> mutations conferring resistance to isoniazid KatG, <em>S. cinnamoneus</em> elongation factor thermo unstable mutants conferring resistance to elfamycin (EF-Tu). (C) CAZy categories (<a id="xref-ref-113-2" class="xref-bibr" href="#ref-113">Yin <em>et al.</em>, 2012</a>) with differences in copy numbers between S and L morphotype genomes. Abbreviations: carbohydrate-binding modules (CBM), carbohydrate esterases (CE), glycoside hydrolases (GH), glycosyltransferases (GT). (D) MEROPS categories (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Rawlings <em>et al.</em>, 2012</a>) with differences in copy numbers between S and L morphotype genomes. Abbreviations: PfpI endopeptidase family/cysteine peptidase (C56), aminopeptidase Y family/metallo peptidase (M28), chymostrypsin family/serine peptidase (S01), D-Ala-D-Ala carboxypeptidase B family/serine peptidase (S12).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The number of proteins predicted to confer antibiotic resistance in S and L morphotype genomes is shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>. Comparing annotations to the comprehensive antibiotic resistance database (CARD) revealed that S morphotype genomes have an additional copy of DesR; a protein involved in self-resistance to antibiotics produced in <em>Streptomyces venezuelae</em>, and 1 – 2 additional copies of AbcA; an ABC transporter that can confer resistance to methicillin, daptomycin, cefotaxime, and moenomycin (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Jia <em>et al.</em>, 2017</a>). The remaining CARD categories had the same number of copies in the genomes of both morphotypes. None of the proteins annotated by CARD belonged to the morphotype-unique protein set, but this is likely because a conservative approach was used to identify morphotype-unique proteins.</p><p id="p-38">Carbohydrate active enzyme (CAZy) categories that were present in different copy numbers in S and L morphotype genomes are represented in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>. Comparison of CAZy annotations revealed that S morphotype genomes contain a unique β-1,2-mannosyltransferase (GT91) that was also identified in the morphotype-unique protein analysis. In addition, the S morphotype genomes contained more copies of three carbohydrate esterase families: CE1, CE5 that contain cutinases, and CE10 that acts on non-carbohydrate substrates. The S morphotype also contains more members of three glycoside hydrolase families: GH10 and GH11 that contain xylanases, and GH25 that contain lysozymes. L morphotype genomes did not contain any morphotype-unique CAZy categories, but contained more copies of carbohydrate-binding module family 50 domains that bind to peptidoglycan and chitin (CBM50).</p><p id="p-39">Peptidase and peptidase inhibitor (MEROPS) categories that were present in different copy numbers in S and L morphotype genomes are represented in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>. Comparison of MEROPS annotations revealed that the S morphotype genomes contained more copies of PfPI endopeptidase (C56) family, whereas the L morphotype genomes contained more copies of aminopeptidase Y (M28E), chymotrypsin (S01A), and D-Ala-D-Ala carboxypeptidase B (S12) families.</p></div><div id="sec-17" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-40">A maximum likelihood tree with clade support values (&gt;50%) is presented in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The S isolates used in the current genomic analyses along with S isolate Yuin20 formed a distinct clade from most other <em>A. flavus</em> isolates with high support. The major clade (indicated with the arrow in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) included all L isolates except Sukhothai16, as well as some S isolates; five of these S isolates formed a subclade. The three L isolates used for the genomic analyses are present in this major clade, distinct from the S isolates used in the genomic analyses. The number of polymorphic sites was low among all the <em>A. flavus</em> isolates and thus resolution within the major clade was limited. Two isolates, L isolate Sukhothai16 and S isolate Sanpotong22, placed separate from these described clades.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood (ML) tree based on concatenated sequences of the 5&#x2032; half of tsr1, 3&#x2032; half of tsr1, and cmd (3.4 kb total). ML bootstrap values (&gt;50%) are indicated at the nodes and the arrow indicates the major clade referred to in-text. Aspergillus minisclerotingenes was used as the outgroup; isolates used in this analysis are listed in Table 1. Isolates of A. flavus are indicated with an S or L in bold to represent S or L morphotype, respectively, and isolates used in the genomics analyses are both in bold and marked with an asterisk." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-600309575" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum likelihood (ML) tree based on concatenated sequences of the 5&#x2032; half of &lt;em&gt;tsr1&lt;/em&gt;, 3&#x2032; half of &lt;em&gt;tsr1&lt;/em&gt;, and &lt;em&gt;cmd&lt;/em&gt; (3.4 kb total). ML bootstrap values (&gt;50%) are indicated at the nodes and the arrow indicates the major clade referred to in-text. &lt;em&gt;Aspergillus minisclerotingenes&lt;/em&gt; was used as the outgroup; isolates used in this analysis are listed in Table 1. Isolates of &lt;em&gt;A. flavus&lt;/em&gt; are indicated with an S or L in bold to represent S or L morphotype, respectively, and isolates used in the genomics analyses are both in bold and marked with an asterisk.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3915/F6.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/12/3915/F6.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3915/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491730" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Maximum likelihood (ML) tree based on concatenated sequences of the 5′ half of <em>tsr1</em>, 3′ half of <em>tsr1</em>, and <em>cmd</em> (3.4 kb total). ML bootstrap values (&gt;50%) are indicated at the nodes and the arrow indicates the major clade referred to in-text. <em>Aspergillus minisclerotingenes</em> was used as the outgroup; isolates used in this analysis are listed in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>. Isolates of <em>A. flavus</em> are indicated with an S or L in bold to represent S or L morphotype, respectively, and isolates used in the genomics analyses are both in bold and marked with an asterisk.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-42">Although <em>A. flavus</em> is generally considered a soil-borne fungus, the differences in asexual reproductive strategy and levels of aflatoxin production between S and L morphotypes suggest that the S morphotype, producing large numbers of sclerotia and consistently toxigenic is adapted to survival in the soil, and the L morphotype, with large numbers of conidia and variable levels of aflatoxin is adapted to aerial dispersal to the phyllosphere (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cotty <em>et al.</em>, 1994</a>). The closest sister taxa to <em>A. flavus</em> are <em>A. minisclerotigenes</em> and <em>A. parvisclerotigenus</em> (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Pildain <em>et al.</em> 2008</a>), both of which have a morphology similar to the S morphotype. Therefore, we believe the morphology of the S morphotype is ancestral to <em>A. flavus</em>, and it is speculated that the two morphotypes diverged 1-3 million years ago (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ehrlich <em>et al.</em> 2005</a>). To test the hypothesis that S and L morphotypes are differentially adapted to the soil and phyllosphere respectively, we sequenced the genomes of three S isolates and three L isolates and identified both structural and gene content differences that we propose play a role in differential niche adaptation.</p><p id="p-43">These analyses identified 191 proteins unique to the S morphotype and 82 proteins unique to the L morphotype. This is likely an undercount as our approach was conservative to avoid false positives due to misannotation. Morphotype-unique genes were present on each of the eight chromosomes, indicating acquisition was not through a single recombination or transfer event. Morphotype-unique proteins included those involved in environmental response, transcriptional regulation, post-transcriptional regulation, epigenetic regulation and metabolic potential. These differences would allow the two morphotypes to selectively respond to conditions critical to the soil and phyllosphere, which differ in nutrient availability, competition within the niches, and their ambient conditions.</p><p id="p-44">Morphotype-unique proteins in the S morphotype are involved in microbial competition, resilience against toxic compounds, and nutrient acquisition; these traits are advantageous for survival in the highly competitive soil environment. One of the protein families that the S morphotype gained is an antifungal protein with high amino acid similarity to the well-studied protein Pc24g00380/PAF in <em>Penicillium chrysogenum</em> (98% query coverage, 68% identity, e-value 4e-38; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Marx <em>et al.</em>, 1995</a>) that has antifungal activity against diverse fungal taxa (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kaiserer <em>et al.</em>, 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Galgóczy <em>et al.</em>, 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Galgóczy <em>et al.</em>, 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barna <em>et al.</em>, 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Galgóczy <em>et al.</em>, 2008</a>). Unlike the microbial diversity present in soils, the phyllosphere community consists predominantly of bacteria, with the diversity and numbers of fungi expected to be lower (<a id="xref-ref-107-3" class="xref-bibr" href="#ref-107">Vorholt 2012</a>); therefore the antifungal protein in the S morphotype would be more advantageous for soil fungi. In support of this, yeasts isolated from the soil have greater antagonistic activity against filamentous fungi than yeasts isolates from the phyllosphere (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hilber-Bodmer <em>et al.</em>, 2017</a>).</p><p id="p-45">Heavy metal tolerance is also important to the adaptive evolution of soil dwelling organisms (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Ryan <em>et al.</em>, 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chan <em>et al.</em>, 2016</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Faddeeva-Vakhrusheva <em>et al.</em>, 2016</a>). S isolates have two or more additional copies of the chromate transporter family (PF02417) than L isolates, which could confer increased tolerance to chromate. In addition, the S morphotype may have gained increased resistance to antifungals by having two copies of the C-8 sterol isomerase (PF04622; ERG2 and Sigma 1 receptor-like family) in contrast to the single copy in the L morphotype. Ergosterol biosynthesis enzymes, C-8 sterol isomerase and C-14 sterol reductase, are targets for amine antifungals produced by many microbes including soil bacteria (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Trejo and Bennett 1963</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Vicente <em>et al.</em>, 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hull <em>et al.</em>, 2012</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Vincent <em>et al.</em>, 2013</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Sanglard 2016</a>). In <em>Fusarium graminearum</em>, possession of a second copy of C-14 sterol reductase confers increased resistance to amine fungicides (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Liu <em>et al.</em>, 2011</a>). Similarly, the additional copy of C-8 sterol isomerase in the <em>A. flavus</em> S morphotype may enhance resistance to antifungal compounds produced by soil microbes enabling it to compete in the soil niche.</p><p id="p-46">In the soil, nutrient availability is influenced by root exudates that contain organic acids, sugars and amino acids, as well as by degradation of organic matter by microorganisms (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Rengel and Marschner 2005</a>; <a id="xref-ref-104-2" class="xref-bibr" href="#ref-104">Turner <em>et al.</em>, 2013</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Dotaniya and Meena 2015</a>). Among the protein families that were gained in the S morphotype were a pyruvate phosphate dikinase and a lactate dehydrogenase encoded by adjacent genes, which could allow the fungus to take advantage of diverse carbon sources in the soil. Lactate dehydrogenase catalyzes the conversion of L-lactate to pyruvate, and pyruvate phosphate dikinase phosphorylates pyruvate to produce phosphoenolpyruvate. These two enzymes could allow the S morphotype to perform gluconeogenesis from lactate to utilize organic acids in the soil as an energy source. Although the soil may have more available carbon sources compared to the phyllsophere, iron is a scarce nutrient due to lack of mobility (<a id="xref-ref-87-3" class="xref-bibr" href="#ref-87">Rengel and Marschner 2005</a>). Therefore, iron acquisition strategies independent of siderophore scavenging are adaptive traits that support adequate iron uptake in soil (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Mathew <em>et al.</em>, 2014</a>; <a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Zhu <em>et al.</em>, 2016</a>). Although both morphotypes have siderophore genes (data not shown), only the S morphotype has a heme oxygenase, which catalyzes the release of iron from heme providing increased capacity to competitively acquire iron.</p><p id="p-47">In contrast to the S morphotype, we hypothesize the L morphotype is adapted to the phyllosphere, which is protected by a cuticle layer that limits diffusion of compounds and thus is scarce in nutrients (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Schönherr and Baur 1996</a>; <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Lindow and Brandl 2003</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bringel and Couée 2015</a>). In epiphytic bacteria, the ability to scavenge for limited substrates has been implicated in adaptation to the phyllosphere and their metagenomes contain a prominence of transporter genes (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Delmotte <em>et al.</em>, 2009</a>). Similarly, L morphotype genomes have a significant GO term enrichment of amino acid transporters as well as an expansion of POT (proton-dependent oligopeptide transporter) family proteins. In addition to transporters, the L morphotype possesses a second copy of arginase, which catalyzes the breakdown of arginine into ornithine and urea. Arginine constitutes a major storage and transport form of organic nitrogen in plants (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Winter <em>et al.</em>, 2015</a>) and is shown to decrease significantly on the phylloplane in the presence of certain epiphytic bacteria (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Ryffel <em>et al.</em>, 2016</a>). Therefore, the presence of an additional copy of arginase in the L morphotype may aid in competitive nitrogen acquisition in the presence of these bacteria, allowing the fungus to take advantage of the proteinogenic amino acid with the highest nitrogen to carbon ratio (<a id="xref-ref-111-2" class="xref-bibr" href="#ref-111">Winter <em>et al.</em>, 2015</a>). The L morphotype’s ability to scavenge for nitrogen sources was also reflected in the expansion of three MEROPS peptidase categories.</p><p id="p-48">In addition to the prominence of unique proteins involved in nitrogen metabolism, proteins from the L morphotype were significantly enriched in the LysM domain (PF01476), which binds peptidoglycan or chitin. Proteins containing LysM domains can be categorized as either proteins that contain one or more LysM domains plus a chitinase domain or as proteins with multiple LysM domains and no chitinase domain. The first type is involved in fungal growth by creating plasticity in cell walls and the second type serves to evade triggering microbe-associated molecular pattern (MAMP) elicited immune responses by binding chitin particles released from hyphae (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">de Jonge and Thomma 2009</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Kombrink and Thomma 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Martinez <em>et al.</em> 2012</a>). L morphotype-unique LysM proteins contain two to three LysM domains, but lack a chitinase domain indicating they belong to the latter type. These proteins could allow the L morphotype to colonize the phyllosphere while evading the host immune system.</p><p id="p-49">Chromosomal rearrangements have been proposed to play major roles in evolution that lead to differential adaptation (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Schmidt and Hensel 2004</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kirkpatrick 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Stukenbrock 2013</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Guttman <em>et al.</em>, 2014</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Leducq 2014</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Raeside <em>et al.</em>, 2014</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Seidl and Thomma 2014</a>); A notable structural difference between the S and L morphotype genomes is a &gt;530 kB inversion on chromosome 8. Inversions are major drivers of adaptation and speciation in various organisms (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hoffmann and Rieseberg 2008</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Kirkpatrick 2010</a>) by impacting homologous chromosome pairing during meiosis that can result in reduced fertility and reproductive isolation (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Hoffmann and Rieseberg 2008</a>). Our phylogenetic analysis of <em>A. flavus</em> S and L morphotype taxa from diverse geographic regions does not resolve whether the morphotypes form monophyletic clades due to limited sample size, but does indicate the S and L isolates used in the genomic analyses belong to two distinct clades. The inversion may have resulted in limited gene flow between the morphotypes leading to distinct lineages. Inversions can also result in fortuitous changes via gene disruption or altered gene regulation at the breakpoints (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Kirkpatrick and Barton 2006</a>). In the S morphotype, there are two secondary metabolite gene clusters at the inversion breakpoints; if the compounds produced by these clusters are advantageous for microbial competition in the soil, the S inversion configuration would be advantageous for soil survival. In contrast, in the L morphotype, one of the inversion breakpoints encodes a cutinase gene, which may be favorable for nutrient acquisition in the phyllosphere. Finally, inversions can suppress recombination within the inverted region leading to maintenance of divergent allele combinations that could be advantageous in contrasting niches (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Hoffmann and Rieseberg 2008</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Kirkpatrick 2010</a>). In both the S and L morphotypes, the inverted region contains genes under selection and morphotype-unique genes. In the inverted region of the S morphotype, a eukaryotic elongation factor 5A hypusine gene is under selection-and four S morphotype-unique proteins are present (a cytochrome p450, a hexapeptide repeat of succinyl transferase, a UbiA prenyltransferase, and an unknown protein). The L morphotype has one unique protein (protein with an FAD binding domain) in the inverted region. The region may also contain genes with adaptive allelic differences between the morphotypes that were not identified to be under selection. Suppressed recombination within the inversion could maintain these gene and allelic differences between the morphotypes. However, elucidation of mechanisms and extent to which these differences play a functional role in niche adaptation will require validation.</p><p id="p-50">In addition to the inversion creating differences in secondary metabolite clusters, the number of deletions within secondary metabolite clusters also differs between the morphotypes with more intact clusters present in the S morphotype. The L morphotype contains seven deletions within secondary metabolite clusters relative to the S morphotype, including the deletion of two PKS genes. In contrast, the S morphotype contains one deletion within secondary metabolite clusters relative to the L morphotype; however, this deletion is within the <em>cypA</em> gene of the aflatoxin gene cluster in which both S and L isolates have deletions with the S isolates having a larger deletion. Therefore, both morphotypes have defective <em>cypA</em> genes resulting in the loss of aflatoxin G production (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ehrlich <em>et al.</em>, 2004</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Probst <em>et al.</em>, 2012</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Probst <em>et al.</em>, 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adhikari <em>et al.</em>, 2016</a>). The differences in deletions within secondary metabolite clusters suggest the S morphotype is under higher selective pressure to maintain secondary metabolite production, consistent with the pressure to maintain aflatoxin production. Polyketide biosynthetic genes are enriched in metagenomes of soil bacteria as well (Tringe <em>et al.</em> 2005). The extra intact clusters in the S morphotype may produce antimicrobial compounds, signaling molecules, or chelating agents (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Demain and Fang 2000</a>) that could be advantageous for competition against the more plentiful and diverse microbes in the soil compared to the phyllosphere (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Delmotte <em>et al.</em>, 2009</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Knief <em>et al.</em>, 2012</a>).</p><p id="p-51">In summary, we have used comparative genomics to test the hypothesis that the S and L morphotypes have genetic differences that could play a role in differential adaptation to the soil and phyllosphere, respectively. Our genomic comparisons indicate there are differences in gene regulation, antimicrobial activity, resistance to natural compounds and toxic chemicals, carbon and nitrogen metabolism, iron acquisition, and secondary metabolite production between the morphotypes. We have proposed how these differences could be advantageous for niche adaptation and have provided a foundation for experiments to determine the relevance of the genomic differences in niche adaptation. Functional validation of these genes is necessary as well as examination of additional S and L isolates to extend these results. Furthermore, many of the morphotype-unique proteins did not have predicted functions or domains, therefore, functional analysis of these via knockout or knock-in experiments would help in defining their potential roles in niche adaptation of the two morphotypes.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6721511">https://doi.org/10.25387/g3.6721511</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: J. Dunlap</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-29"><span class="received-label">Received </span>June 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-15"><span class="accepted-label">Accepted </span>October 15, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Ohkura <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.1" data-doi="10.1186/s13568-016-0228-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adhikari</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Degeneration of aflatoxin gene clusters in <em>Aspergillus flavus</em> from Africa and North America.</span> <abbr class="cit-jnl-abbrev">AMB Express</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13568-016-0228-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAMB%2BExpress%26rft.volume%253D6%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13568-016-0228-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13568-016-0228-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.2" data-doi="10.1186/s12711-014-0081-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D47%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0081-5%26rft_id%253Dinfo%253Apmid%252F25595431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0081-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25595431&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.3" data-doi="10.1038/srep36164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barna</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedus</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bíró</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pócsi</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Effect of the <em>Penicillium chrysogenum</em> antifungal protein (PAF) on barley powdery mildew and wheat leaf rust pathogens.</span> <abbr class="cit-jnl-abbrev">J. Basic Microbiol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jobm.200800197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBasic%2BMicrobiol.%26rft.volume%253D6%26rft.spage%253D36164%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep36164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep36164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.4" data-doi="10.1094/PD-91-0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klich</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mycotoxins.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">516</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/CMR.16.3.497-516.2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BMicrobiol.%2BRev.%26rft.volume%253D91%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPD-91-0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PD-91-0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.5" data-doi="10.3389/fpls.2016.02052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bilgrami</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choudhary</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Impact of habitats on toxigenic potential of <em>Aspergillus flavus</em>.</span> <abbr class="cit-jnl-abbrev">J. Stored Prod. Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-474X(93)90051-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStored%2BProd.%2BRes.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.02052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.02052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.6" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackey</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population dynamics of <em>Aspergillus flavus</em> in the air of an intensively cultivated region of southwest Arizona.</span> <abbr class="cit-jnl-abbrev">Plant Pathology</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathology%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.7" data-doi="10.1371/journal.pone.0016163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeckmann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bairoch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blatter</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estreicher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2003</span> <span class="cit-article-title">The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016163%26rft_id%253Dinfo%253Apmid%252F21949733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949733&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.8" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.9" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bringel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couée</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Pivotal roles of phyllosphere microorganisms at the interface between plant functioning and atmospheric trace gas dynamics.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2015.00486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.10" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruen</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A simple and robust statistical test for detecting the presence of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.048975</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.11" data-doi="10.1146/annurev-arplant-050312-120106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulgarelli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlaeppi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spaepen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Themaat</span>  <span class="cit-name-given-names">E. V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze-Lefert</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Structure and functions of the bacterial microbiota of plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-arplant-050312-120106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D64%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-050312-120106%26rft_id%253Dinfo%253Apmid%252F23373698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-050312-120106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23373698&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321699500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.12" data-doi="10.1111/lam.12337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callicott</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Method for monitoring deletions in the aflatoxin biosynthesis gene cluster of <em>Aspergillus flavus</em> with multiplex PCR.</span> <abbr class="cit-jnl-abbrev">Lett. Appl. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/lam.12337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLett.%2BAppl.%2BMicrobiol.%26rft.volume%253D60%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1111%252Flam.12337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/lam.12337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.13" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosome visualization tool: a whole genome viewer.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2011</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2011/373875</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.14" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutiérrez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silla-Martínez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.15" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castillo-Davis</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">GeneMerge–post-genomic analysis, data mining, and hypothesis testing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">892</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.12.3915.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Centers for Disease Control and Protection (CDC)</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Outbreak of aflatoxin poisoning–eastern and central provinces, Kenya, January-July 2004.</span> <abbr class="cit-jnl-abbrev">Morbidity and mortality weekly report</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMorbidity%2Band%2Bmortality%2Bweekly%2Breport%26rft.volume%253D53%26rft.spage%253D790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.17" data-doi="10.1080/0967087031000101061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adrian</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kher</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandclément</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>Pseudomonas</em> lini strain ZBG1 revealed carboxylic acid utilization and copper resistance features required for adaptation to vineyard soil environment: a draft genome analysis.</span> <abbr class="cit-jnl-abbrev">J Genomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7150/jgen.16146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGenomics%26rft.volume%253D49%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1080%252F0967087031000101061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/0967087031000101061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ching’anda</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atehnkeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Diversity of aflatoxin producing fungi in Malawi (Abstr.). Phytopathology 106: S4.45.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.19" data-doi="10.1094/Phyto-79-808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Virulence and cultural characteristics of two <em>Aspergillus flavus</em> strains pathogenic on cotton.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/Phyto-79-808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-79-808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-79-808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egel</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elias</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Agriculture, Aflatoxins, and Aspergillus</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span> in <span class="cit-source">The genus Aspergillus: from taxonomy and genetics to industrial application</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Renwick</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Peberdy</span>  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Boston, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaime-Garcia</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>, pp. <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span> in <span class="cit-source"><em>Etiology and management of aflatoxin contamination. Mycotoxins: detection methods</em>, <em>management</em>, <em>public health</em>, <em>and agricultural trade</em></span>, <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">Wallingford, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.22" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Aflatoxin-producing potential of communities of <em>Aspergillus</em> section <em>Flavi</em> from cotton producing areas in the United States.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0953756296003139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycol.%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.12.3915.23" data-doi="10.1270/jsbbs.16005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Courjol</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouault</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mille</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2015</span> <span class="cit-article-title">β-1,2-Mannosyltransferases 1 and 3 participate in yeast and hyphae O- and N-linked mannosylation and alter <em>Candida albicans</em> fitness during infection.</span> <abbr class="cit-jnl-abbrev">Open Forum Infect. Dis.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">ofv116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/ofid/ofv116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BForum%2BInfect.%2BDis.%26rft.volume%253D66%26rft.spage%253D560%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.16005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.16005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.24" data-doi="10.1038/s41598-018-23276-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0011147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-23276-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-23276-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.25" data-doi="10.1016/j.virusres.2011.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jonge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomma</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungal LysM effectors: extinguishers of host immunity.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tim.2009.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DLegg%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D161%26rft.epage%253D170%26rft.atitle%253DComparing%2Bthe%2Bregional%2Bepidemiology%2Bof%2Bthe%2Bcassava%2Bmosaic%2Band%2Bcassava%2Bbrown%2Bstreak%2Bvirus%2Bpandemics%2Bin%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2011.04.018%26rft_id%253Dinfo%253Apmid%252F21549776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2011.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21549776&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.26" data-doi="10.1073/pnas.0905240106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delmotte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knief</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaffron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innerebner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roschitzki</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Community proteogenomics reveals insights into the physiology of phyllosphere bacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16428</span>–<span class="cit-lpage">16433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0905240106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0905240106%26rft_id%253Dinfo%253Apmid%252F19805315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2NDI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demain</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The natural functions of secondary metabolites</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">39</span> in <span class="cit-source">History of Modern Biotechnology I</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Fiechter.</span>  <span class="cit-name-given-names">I. A.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin Heidelburg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.28" data-doi="10.1101/208181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dotaniya</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meena</span>  <span class="cit-name-given-names">V. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rhizosphere effect on nutrient availability in soil and its uptake by plants: a review.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci., India, Sect. B Biol. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s40011-013-0297-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%252C%2BIndia%252C%2BSect.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1101%252F208181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyMDgxODF2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.29" data-doi="10.1186/1471-2164-16-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-16-1%26rft_id%253Dinfo%253Apmid%252F25553907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-16-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553907&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.30" data-doi="10.1111/j.1439-0434.2005.00974.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrlich</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">P.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Aflatoxin biosynthesis cluster gene cypA is required for G aflatoxin formation.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">6518</span>–<span class="cit-lpage">6524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.70.11.6518-6524.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.volume%253D153%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0434.2005.00974.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0434.2005.00974.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.31" data-doi="10.1111/j.1365-2672.2005.02637.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrlich</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Aflatoxin biosynthesis gene clusters and flanking regions.</span> <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2672.2005.02637.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BMicrobiol.%26rft.volume%253D120%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2005.02637.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2005.02637.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.32" data-doi="10.1007/s00122-017-2943-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0721-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D130%26rft.spage%253D2069%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2943-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2943-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.33" data-doi="10.1186/s12985-016-0669-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faddeeva-Vakhrusheva</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derks</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anvar</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agamennone</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suring</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2016</span> <span class="cit-article-title">Gene Family Evolution Reflects Adaptation to Soil Environmental Stressors in the Genome of the Collembolan Orchesella cincta.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2106</span>–<span class="cit-lpage">2117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-016-0669-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12985-016-0669-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.34" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Pfam protein families database: towards a more sustainable future.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D279</span>–<span class="cit-lpage">D285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.35" data-doi="10.1155/2012/795697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galgóczy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pócsi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sensitivity of different zygomycetes to the <em>Penicillium chrysogenum</em> antifungal protein (PAF).</span> <abbr class="cit-jnl-abbrev">J. Basic Microbiol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jobm.200410512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBasic%2BMicrobiol.%26rft.volume%253D2012%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F795697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/795697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.36" data-doi="10.1111/j.1574-6968.2007.00661.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galgóczy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukács</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pócsi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2007</span> <span class="cit-article-title">Interactions between statins and <em>Penicillium chrysogenum</em> antifungal protein (PAF) to inhibit the germination of sporangiospores of different sensitive Zygomycetes.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6968.2007.00661.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft.volume%253D50%26rft.spage%253D768%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2007.00661.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2007.00661.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.37" data-doi="10.1007/s10482-008-9263-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galgóczy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pócsi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedus</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vágvölgyi</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>In vitro</em> activity of <em>Penicillium chrysogenum</em> antifungal protein (PAF) and its combination with fluconazole against different dermatophytes.</span> <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10482-008-9263-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.volume%253D26%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10482-008-9263-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10482-008-9263-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.38" data-doi="10.1094/PHYTO.1997.87.9.940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Formation of sclerotia and aflatoxins in developing cotton bells infected by the S strain of <em>Aspergillus flavus</em> and potential for biocontrol with an atoxigenic strain.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">940</span>–<span class="cit-lpage">945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO.1997.87.9.940</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DGarber%26rft.auinit1%253DR.%2BK.%26rft.volume%253D87%26rft.issue%253D9%26rft.spage%253D940%26rft.epage%253D945%26rft.atitle%253DFormation%2Bof%2BSclerotia%2Band%2BAflatoxins%2Bin%2BDeveloping%2BCotton%2BBolls%2BInfected%2Bby%2Bthe%2BS%2BStrain%2Bof%2BAspergillus%2Bflavus%2Band%2BPotential%2Bfor%2BBiocontrol%2Bwith%2Ban%2BAtoxigenic%2BStrain.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.1997.87.9.940%26rft_id%253Dinfo%253Apmid%252F18945065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.1997.87.9.940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18945065&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldblatt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-source"><em>Aflatoxin: Scientific background</em>, <em>control and implication</em></span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.40" data-doi="10.1007/978-1-59745-251-9_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delsuc</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating maximum likelihood phylogenies with PhyML.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">537</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-59745-251-9_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D3%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-251-9_6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-251-9_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.41" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.42" data-doi="10.3389/fpls.2017.00023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mchardy</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze-Lefert</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microbial genome-enabled insights into plant-microorganism interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">797</span>–<span class="cit-lpage">813</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00023%26rft_id%253Dinfo%253Apmid%252F28220127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28220127&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.43" data-doi="10.4161/gmcr.26408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilber-Bodmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahrens</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freimoser</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Competition assays and physiological experiments of soil and phyllosphere yeasts identify <em>Candida subhashii</em> as a novel antagonist of filamentous fungi.</span> <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12866-016-0908-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiol.%26rft.volume%253D5%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fgmcr.26408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/gmcr.26408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.44" data-doi="10.1111/j.1364-3703.2012.00812.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revisiting the impact of inversions in evolution: from population genetic markers to drivers of adaptive shifts and speciation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.39.110707.173532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DOgwok%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D1019%26rft.epage%253D1031%26rft.atitle%253DTransgenic%2BRNA%2Binterference%2B%2528RNAi%2529-derived%2Bfield%2Bresistance%2Bto%2Bcassava%2Bbrown%2Bstreak%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2012.00812.x%26rft_id%253Dinfo%253Apmid%252F22845735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2012.00812.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22845735&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.45" data-doi="10.1099/vir.0.000014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPatil%26rft.auinit1%253DB.%2BL.%26rft.volume%253D96%26rft.issue%253DPt_5%26rft.spage%253D956%26rft.epage%253D968%26rft.atitle%253DCassava%2Bbrown%2Bstreak%2Bdisease%253A%2Ba%2Bthreat%2Bto%2Bfood%2Bsecurity%2Bin%2BAfrica%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.000014%26rft_id%253Dinfo%253Apmid%252F26015320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoidmlyIjtzOjU6InJlc2lkIjtzOjExOiI5Ni9QdF81Lzk1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.46" data-doi="10.1186/s12864-015-1765-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hongo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cintra</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerlotini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobo</span>  <span class="cit-name-given-names">F. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">POTION: an end-to-end pipeline for positive Darwinian selection detection in genome-scale data through phylogenetic comparison of protein-coding genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1765-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1765-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1765-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.47" data-doi="10.1007/s12042-011-9088-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hull</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Two clinical isolates of Candida glabrata exhibiting reduced sensitivity to amphotericin B both harbor mutations in ERG2.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">6417</span>–<span class="cit-lpage">6421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AAC.01145-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTrop%2BPlant%2BBiol%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D94%26rft.atitle%253DThe%2BCassava%2BGenome%253A%2BCurrent%2BProgress%252C%2BFuture%2BDirections.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12042-011-9088-z%26rft_id%253Dinfo%253Apmid%252F22523606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12042-011-9088-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523606&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000208980100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.48" data-doi="10.1093/nar/gkw1004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raphenya</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcock</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waglechner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2017</span> <span class="cit-article-title">CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D566</span>–<span class="cit-lpage">D573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1004%26rft_id%253Dinfo%253Apmid%252F27789705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27789705&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.49" data-doi="10.2135/cropsci2013.07.0482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2014</span> <span class="cit-article-title">InterProScan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D54%26rft.spage%253D1384%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.07.0482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.07.0482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.50" data-doi="10.1038/ng.3845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaiserer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberparleiter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiler-Görz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgstaller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2003</span> <span class="cit-article-title">Characterization of the <em>Penicillium chrysogenum</em> antifungal protein PAF.</span> <abbr class="cit-jnl-abbrev">Arch. Microbiol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">204</span>–<span class="cit-lpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00203-003-0578-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch.%2BMicrobiol.%26rft.volume%253D49%26rft.spage%253D959%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.51" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellner</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orsborn</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orbach</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Coccidioides posadasii</em> contains a single 1,3-beta-glucan synthase gene that appears to be essential for growth.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.4.1.111-120.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zOTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.52" data-doi="10.1016/j.fgb.2010.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khaldi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifuddin</span>  <span class="cit-name-given-names">F. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nierman</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2010</span> <span class="cit-article-title">SMURF: genomic mapping of fungal secondary metabolite clusters.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">736</span>–<span class="cit-lpage">741</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2010.06.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D7%26rft.spage%253D1666%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2010.06.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2010.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.53" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How and why chromosome inversions evolve.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.54" data-doi="10.1111/j.1744-7348.1938.tb02354.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.047985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D25%26rft.spage%253D790%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7348.1938.tb02354.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7348.1938.tb02354.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.55" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klich</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Biogeography of <em>Aspergillus</em> species in soil and litter.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15572536.2003.11833245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.volume%253D97%26rft.spage%253D6547%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F25129495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-8210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25129495&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.56" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klich</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Aspergillus flavus</em>: the major producer of aflatoxin.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">722</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2007.00436.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.57" data-doi="10.3389/fpls.2016.02060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knief</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delmotte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaffron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innerebner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Metaproteogenomic analysis of microbial communities in the phyllosphere and rhizosphere of rice.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1378</span>–<span class="cit-lpage">1390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ismej.2011.192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D7%26rft.spage%253D2060%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.02060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.02060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.58" data-doi="10.1186/s12863-017-0507-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kombrink</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomma</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">LysM effectors: secreted proteins supporting fungal life.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1003769</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0507-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0507-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.59" data-doi="10.1099/vir.0.014688-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.014688-0%26rft_id%253Dinfo%253Apmid%252F20071490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.014688-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20071490&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.60" data-doi="10.3835/plantgenome2017.03.0015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lanfear</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frandsen</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senfeld</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calcott</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PartitionFinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">773</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.03.0015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.03.0015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.61" data-doi="10.3835/plantgenome2015.11.0118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D9%26rft.spage%253D342%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.11.0118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.11.0118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.62" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leducq</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ecological genomics of adaptation and speciation in fungi.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">781</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-007-7347-9_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BExp.%2BMed.%2BBiol.%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.63" data-doi="10.1186/gb-2013-14-8-r93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helt</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz-Torres</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childers</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2013</span> <span class="cit-article-title">Web Apollo: a web-based genomic annotation editing platform.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-8-r93</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR93%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-8-r93%26rft_id%253Dinfo%253Apmid%252F24000942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-8-r93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24000942&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.64" data-doi="10.1128/AEM.69.4.1875-1883.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindow</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandl</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Microbiology of the phyllosphere.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.69.4.1875-1883.2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DLindow%26rft.auinit1%253DS.%2BE.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D1875%26rft.epage%253D1883%26rft.atitle%253DMicrobiology%2Bof%2Bthe%2BPhyllosphere%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.69.4.1875-1883.2003%26rft_id%253Dinfo%253Apmid%252F12676659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY5LzQvMTg3NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.65" data-doi="10.1099/mic.0.045690-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A sterol C-14 reductase encoded by FgERG24B is responsible for the intrinsic resistance of <em>Fusarium graminearum</em> to amine fungicides.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1675</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.045690-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.045690-0%26rft_id%253Dinfo%253Apmid%252F21436218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.045690-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21436218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.66" data-doi="10.1038/nature11237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebeis</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paredes</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Defining the core <em>Arabidopsis thaliana</em> root microbiome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11237</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLundberg%26rft.auinit1%253DD.%2BS.%26rft.volume%253D488%26rft.issue%253D7409%26rft.spage%253D86%26rft.epage%253D90%26rft.atitle%253DDefining%2Bthe%2Bcore%2BArabidopsis%2Bthaliana%2Broot%2Bmicrobiome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11237%26rft_id%253Dinfo%253Apmid%252F22859206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22859206&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307010700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.67" data-doi="10.1111/j.1365-313X.2007.03326.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">How to usefully compare homologous plant genes and chromosomes as DNA sequences.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2007.03326.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DLyons%26rft.auinit1%253DE.%26rft.volume%253D53%26rft.issue%253D4%26rft.spage%253D661%26rft.epage%253D673%26rft.atitle%253DHow%2Bto%2Busefully%2Bcompare%2Bhomologous%2Bplant%2Bgenes%2Band%2Bchromosomes%2Bas%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03326.x%26rft_id%253Dinfo%253Apmid%252F18269575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03326.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18269575&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252931800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.68" data-doi="10.1038/nature04300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumagai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome sequencing and analysis of <em>Aspergillus oryzae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">1161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMachida%26rft.auinit1%253DM.%26rft.volume%253D438%26rft.issue%253D7071%26rft.spage%253D1157%26rft.epage%253D1161%26rft.atitle%253DGenome%2Bsequencing%2Band%2Banalysis%2Bof%2BAspergillus%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04300%26rft_id%253Dinfo%253Apmid%252F16372010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16372010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234111500051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-web" id="cit-8.12.3915.69"><div class="cit-metadata unstructured">Maddison, W., and D. Maddison, 2017 Mesquite: a modular system for evolutionary analysis. Version 3.2. <a href="http://mesquiteproject.org">http://mesquiteproject.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.70" data-doi="10.1128/mBio.00259-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gräser</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative genome analysis of <em>Trichophyton rubrum</em> and related dermatophytes reveals candidate genes involved in infection.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00259–12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.00259-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D3%26rft.spage%253De0025912%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00259-12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00259-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.71" data-doi="10.1016/0378-1119(95)00701-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reindl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stöffler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lottspeich</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1995</span> <span class="cit-article-title">Cloning, structural organization and regulation of expression of the <em>Penicillium chrysogenum paf</em> gene encoding an abundantly secreted protein with antifungal activity.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0378-1119(95)00701-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMarx%26rft.auinit1%253DF.%26rft.volume%253D167%26rft.issue%253D1-2%26rft.spage%253D167%26rft.epage%253D171%26rft.atitle%253DCloning%252C%2Bstructural%2Borganization%2Band%2Bregulation%2Bof%2Bexpression%2Bof%2Bthe%2BPenicillium%2Bchrysogenum%2Bpaf%2Bgene%2Bencoding%2Ban%2Babundantly%2Bsecreted%2Bprotein%2Bwith%2Bantifungal%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252895%252900701-6%26rft_id%253Dinfo%253Apmid%252F8566771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(95)00701-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8566771&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TQ46000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.72" data-doi="10.1111/mmi.12499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberl</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlier</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A novel siderophore-independent strategy of iron uptake in the genus <em>Burkholderia</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">805</span>–<span class="cit-lpage">820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mmi.12499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D91%26rft.spage%253D805%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12499%26rft_id%253Dinfo%253Apmid%252F24354890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24354890&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.73" data-doi="10.1111/j.1749-6632.2012.06800.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehl</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaime</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callicott</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Aspergillus flavus</em> diversity on crops and in the environment can be exploited to reduce aflatoxin exposure and improve health.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1273</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1749-6632.2012.06800.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1273%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2012.06800.x%26rft_id%253Dinfo%253Apmid%252F23230832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2012.06800.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23230832&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.12.3915.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Creating the CIPRES Science Gateway for inference of large phylogenetic trees. Proceedings of the Gateway Computing Environments Workshop (GCE) pp <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/GCE.2010.5676129</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.75" data-doi="10.1080/19440049.2016.1138545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurburgh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Potential economic losses to the US corn industry from aflatoxin contamination.</span> <abbr class="cit-jnl-abbrev">Food Additives &amp; Contaminants: Part A</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/19440049.2016.1138545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BAdditives%2B%2526%2BContaminants%253A%2BPart%2BA%26rft.volume%253D33%26rft.spage%253D540%26rft_id%253Dinfo%253Adoi%252F10.1080%252F19440049.2016.1138545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/19440049.2016.1138545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.12.3915.76" data-doi="10.1093/nar/gkw1071"><div class="cit-metadata"><cite>NCBI resource coordinators, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Database Resources of the National Center for Biotechnology Information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D12</span>–<span class="cit-lpage">D17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1071%26rft_id%253Dinfo%253Apmid%252F27899561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899561&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.77" data-doi="10.1128/genomeA.00168-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nierman</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorova-Abrams</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Losada</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome sequence of <em>Aspergillus flavus</em> NRRL 3357, a strain that causes aflatoxin contamination of food and feed.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00168</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/genomeA.00168-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D3%26rft.spage%253D00168e%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00168-15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.00168-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.78" data-doi="10.1099/ijs.0.65123-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pildain</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisvad</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaamonde</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabral</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varga</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2008</span> <span class="cit-article-title">Two novel aflatoxin-producing <em>Aspergillus</em> species from Argentinean peanuts.</span> <abbr class="cit-jnl-abbrev">Int. J. Syst. Evol. Microbiol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/ijs.0.65123-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DPildain%26rft.auinit1%253DM.%2BB.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D725%26rft.epage%253D735%26rft.atitle%253DTwo%2Bnovel%2Baflatoxin-producing%2BAspergillus%2Bspecies%2Bfrom%2BArgentinean%2Bpeanuts%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.65123-0%26rft_id%253Dinfo%253Apmid%252F18319485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.65123-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319485&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254603700036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.79" data-doi="10.1016/j.ijfoodmicro.2013.12.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diversity of aflatoxin-producing fungi and their impact on food safety in sub-Saharan Africa.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ijfoodmicro.2013.12.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BFood%2BMicrobiol.%26rft.volume%253D174%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijfoodmicro.2013.12.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijfoodmicro.2013.12.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.80" data-doi="10.1007/s10658-011-9887-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callicott</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Deadly strains of Kenyan <em>Aspergillus</em> are distinct from other aflatoxin producers.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10658-011-9887-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D132%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-011-9887-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-011-9887-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.81" data-doi="10.1128/AEM.02370-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Njapau</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Outbreak of an acute aflatoxicosis in Kenya in 2004: identification of the causal agent.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">2762</span>–<span class="cit-lpage">2764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.02370-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02370-06%26rft_id%253Dinfo%253Apmid%252F17308181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjczLzgvMjc2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzkxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.82" data-doi="10.1111/j.1365-2672.2009.04458.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotty</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of <em>Aspergillus</em> section <em>Flavi</em> community structure on the development of lethal levels of aflatoxins in Kenyan maize (<em>Zea mays</em>).</span> <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2672.2009.04458.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DProbst%26rft.auinit1%253DC.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D600%26rft.epage%253D610%26rft.atitle%253DImpact%2Bof%2BAspergillus%2Bsection%2BFlavi%2Bcommunity%2Bstructure%2Bon%2Bthe%2Bdevelopment%2Bof%2Blethal%2Blevels%2Bof%2Baflatoxins%2Bin%2BKenyan%2Bmaize%2B%2528Zea%2Bmays%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2009.04458.x%26rft_id%253Dinfo%253Apmid%252F19674186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2009.04458.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19674186&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.83" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.84" data-doi="10.1128/mBio.01377-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raeside</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaffé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deatherage</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briska</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2014</span> <span class="cit-article-title">Large chromosomal rearrangements during a long-term evolution experiment with <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e01377–14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.01377-14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De0137714%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01377-14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01377-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.85" data-doi="10.1093/bioinformatics/bts378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zichner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stütz</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">DELLY: structural variant discovery by integrated paired-end and split-read analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">i333</span>–<span class="cit-lpage">i339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts378</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts378%26rft_id%253Dinfo%253Apmid%252F22962449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22962449&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.86" data-doi="10.1093/nar/gkr987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MEROPS: the database of proteolytic enzymes, their substrates and inhibitors.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D343</span>–<span class="cit-lpage">D350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr987%26rft_id%253Dinfo%253Apmid%252F22086950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22086950&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.87" data-doi="10.1111/j.1469-8137.2005.01558.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rengel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marschner</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nutrient availability and management in the rhizosphere: exploiting genotypic differences.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2005.01558.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DRengel%26rft.auinit1%253DZ.%26rft.volume%253D168%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D312%26rft.atitle%253DNutrient%2Bavailability%2Band%2Bmanagement%2Bin%2Bthe%2Brhizosphere%253A%2Bexploiting%2Bgenotypic%2Bdifferences.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2005.01558.x%26rft_id%253Dinfo%253Apmid%252F16219070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2005.01558.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16219070&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232441600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maragos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norred</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2003</span> <span class="cit-source"><em>Mycotoxins: risks in plant</em>, <em>animal and human systems</em></span>, pp <span class="cit-fpage">1</span>–<span class="cit-lpage">199</span>. <span class="cit-publ-name">CAST - Council for Agricultural Science and Technology</span>, <span class="cit-publ-loc">Ames</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.89" data-doi="10.1038/nrmicro2163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monchy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardinale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taghavi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">The versatility and adaptation of bacteria from the genus <em>Stenotrophomonas</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DRyan%26rft.auinit1%253DR.%2BP.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D514%26rft.epage%253D525%26rft.atitle%253DThe%2Bversatility%2Band%2Badaptation%2Bof%2Bbacteria%2Bfrom%2Bthe%2Bgenus%2BStenotrophomonas.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2163%26rft_id%253Dinfo%253Apmid%252F19528958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19528958&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.90" data-doi="10.1038/ismej.2015.141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryffel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helfrich</span>  <span class="cit-name-given-names">E. J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiefer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peyriga</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portais</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2016</span> <span class="cit-article-title">Metabolic footprint of epiphytic bacteria on <em>Arabidopsis thaliana</em> leaves.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ismej.2015.141</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D10%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2015.141%26rft_id%253Dinfo%253Apmid%252F26305156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2015.141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26305156&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.91" data-doi="10.2520/myco1975.1986.24_41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siriacha</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manabe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1986</span> <span class="cit-article-title">Distribution and aflatoxin productivity of the atypical strains of <em>Aspergillus flavus</em> isolated from soils in Thailand.</span> <abbr class="cit-jnl-abbrev">Proc. Jpn. Assoc. Mycotoxicology</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2520/myco1975.1986.24_41</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BJpn.%2BAssoc.%2BMycotoxicology%26rft.volume%253D24%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.2520%252Fmyco1975.1986.24_41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2520/myco1975.1986.24_41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.92" data-doi="10.3389/fmed.2016.00011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Emerging threats in antifungal-resistant fungal pathogens.</span> <abbr class="cit-jnl-abbrev">Frontiers in medicine</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmed.2016.00011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bmedicine%26rft.volume%253D3%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmed.2016.00011%26rft_id%253Dinfo%253Apmid%252F27014694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmed.2016.00011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27014694&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.93" data-doi="10.1128/CMR.17.1.14-56.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hensel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Pathogenicity islands in bacterial pathogenesis.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/CMR.17.1.14-56.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DSchmidt%26rft.auinit1%253DH.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D56%26rft.atitle%253DPathogenicity%2BIslands%2Bin%2BBacterial%2BPathogenesis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.17.1.14-56.2004%26rft_id%253Dinfo%253Apmid%252F14726454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvMTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3915.94" data-doi="10.1007/978-0-585-34164-4_1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schönherr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Cuticle permeability studies</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span> in <span class="cit-source">Aerial Plant Surface Microbiology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">C. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Nicot</span>  <span class="cit-name-given-names">P. C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Ngyuyen-The</span>  <span class="cit-name-given-names">C.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-585-34164-4_1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAerial%2BPlant%2BSurface%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-585-34164-4_1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-585-34164-4_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boller</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Aflatoxin production of species and strains of the <em>Aspergillus flavus</em> group isolated from field crops.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Bmicrobiology%26rft.stitle%253DAppl%2BMicrobiol%26rft.aulast%253DSchroeder%26rft.auinit1%253DH.%2BW.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D885%26rft.epage%253D889%26rft.atitle%253DAflatoxin%2Bproduction%2Bof%2Bspecies%2Band%2Bstrains%2Bof%2Bthe%2BAspergillus%2Bflavus%2Bgroup%2Bisolated%2Bfrom%2Bfield%2Bcrops.%26rft_id%253Dinfo%253Apmid%252F4197766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4197766&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.96" data-doi="10.1002/bies.201300155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomma</span>  <span class="cit-name-given-names">B. P. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sex or no sex: evolutionary adaptation occurs regardless.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201300155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D36%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300155%26rft_id%253Dinfo%253Apmid%252F24531982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24531982&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.97" data-doi="10.1093/nar/gkr123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soderlund</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomhoff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SyMAP v3.4: a turnkey synteny system with application to plant genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e68</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr123%26rft_id%253Dinfo%253Apmid%252F21398631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21398631&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.98" data-doi="10.1093/bioinformatics/btl446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2688</span>–<span class="cit-lpage">2690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl446%26rft_id%253Dinfo%253Apmid%252F16928733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16928733&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241629600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.99" data-doi="10.1111/nph.12374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution, selection and isolation: a genomic view of speciation in fungal plant pathogens.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12374</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D199%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12374%26rft_id%253Dinfo%253Apmid%252F23782262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782262&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322598700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.100" data-doi="10.7171/jbt.12-2303-004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stucky</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SeqTrace: a graphical tool for rapidly processing DNA sequencing chromatograms.</span> <abbr class="cit-jnl-abbrev">J. Biomol. Tech.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7171/jbt.12-2303-004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiomolecular%2BTechniques%26rft.stitle%253DJ%2BBiomol%2BTech%26rft.aulast%253DStucky%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D90%26rft.epage%253D93%26rft.atitle%253DSeqTrace%253A%2Ba%2Bgraphical%2Btool%2Bfor%2Brapidly%2Bprocessing%2BDNA%2Bsequencing%2Bchromatograms.%26rft_id%253Dinfo%253Adoi%252F10.7171%252Fjbt.12-2303-004%26rft_id%253Dinfo%253Apmid%252F22942788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7171/jbt.12-2303-004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22942788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.101" data-doi="10.1094/PHYTO-09-10-0243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweany</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damann</span>  <span class="cit-name-given-names">K. E.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaller</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparison of soil and corn kernel Aspergillus flavus populations: evidence for niche specialization.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">952</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO-09-10-0243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DSweany%26rft.auinit1%253DR.%2BR.%26rft.volume%253D101%26rft.issue%253D8%26rft.spage%253D952%26rft.epage%253D959%26rft.atitle%253DComparison%2Bof%2Bsoil%2Band%2Bcorn%2Bkernel%2BAspergillus%2Bflavus%2Bpopulations%253A%2Bevidence%2Bfor%2Bniche%2Bspecialization.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-09-10-0243%26rft_id%253Dinfo%253Apmid%252F21405994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-09-10-0243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21405994&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trejo</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title"><em>Streptomyces nodosus</em> sp. n., the amphotericin-producing organism.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DTrejo%26rft.auinit1%253DW.%2BH.%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D436%26rft.epage%253D439%26rft.atitle%253DSTREPTOMYCES%2BNODOSUS%2BSP.%2BN.%252C%2BTHE%2BAMPHOTERICIN-PRODUCING%2BORGANISM%26rft_id%253Dinfo%253Apmid%252F13994057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiODUvMi80MzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.12.3915.103" data-doi="10.1126/science.1107851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tringe</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Mering</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative metagenomics of microbial communities.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">554</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTringe%26rft.auinit1%253DS.%2BG.%26rft.volume%253D308%26rft.issue%253D5721%26rft.spage%253D554%26rft.epage%253D557%26rft.atitle%253DComparative%2BMetagenomics%2Bof%2BMicrobial%2BCommunities%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107851%26rft_id%253Dinfo%253Apmid%252F15845853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDgvNTcyMS81NTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.104" data-doi="10.1186/gb-2013-14-6-209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The plant microbiome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-6-209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253D209%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-209%26rft_id%253Dinfo%253Apmid%252F23805896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23805896&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.105" data-doi="10.1046/j.1469-0691.2003.00489.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicente</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basilio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabello</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peláez</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Microbial natural products as a source of antifungals.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1469-0691.2003.00489.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bmicrobiology%2Band%2Binfection%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BEuropean%2BSociety%2Bof%2BClinical%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.stitle%253DClin%2BMicrobiol%2BInfect%26rft.aulast%253DVicente%26rft.auinit1%253DM.%2BF.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D32%26rft.atitle%253DMicrobial%2Bnatural%2Bproducts%2Bas%2Ba%2Bsource%2Bof%2Bantifungals.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-0691.2003.00489.x%26rft_id%253Dinfo%253Apmid%252F12691539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-0691.2003.00489.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12691539&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.106" data-doi="10.1371/journal.pbio.1001692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherz-Shouval</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitesell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fitness trade-offs restrict the evolution of resistance to amphotericin B.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001692%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001692%26rft_id%253Dinfo%253Apmid%252F24204207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204207&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.107" data-doi="10.1038/nrmicro2910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vorholt</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Microbial life in the phyllosphere.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">828</span>–<span class="cit-lpage">840</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2910</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DVorholt%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D828%26rft.epage%253D840%26rft.atitle%253DMicrobial%2Blife%2Bin%2Bthe%2Bphyllosphere.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2910%26rft_id%253Dinfo%253Apmid%252F23154261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23154261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.108" data-doi="10.1093/nar/gkv487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman-Derr</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Y. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoVenn: a web server for genome wide comparison and annotation of orthologous clusters across multiple species.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">W78</span>–<span class="cit-lpage">W84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv487</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv487%26rft_id%253Dinfo%253Apmid%252F25964301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25964301&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.109" data-doi="10.1111/j.1365-2672.2008.03906.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whipps</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hand</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pink</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bending</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phyllosphere microbiology with special reference to diversity and plant genotype.</span> <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1744</span>–<span class="cit-lpage">1755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2672.2008.03906.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DWhipps%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D105%26rft.issue%253D6%26rft.spage%253D1744%26rft.epage%253D1755%26rft.atitle%253DPhyllosphere%2Bmicrobiology%2Bwith%2Bspecial%2Breference%2Bto%2Bdiversity%2Band%2Bplant%2Bgenotype.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2008.03906.x%26rft_id%253Dinfo%253Apmid%252F19120625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2008.03906.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120625&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261125300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.110" data-doi="10.1093/ajcn/80.5.1106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jolly</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiles</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jolly</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2004</span> <span class="cit-article-title">Human aflatoxicosis in developing countries: a review of toxicology, exposure, potential health consequences, and interventions.</span> <abbr class="cit-jnl-abbrev">Am. J. Clin. Nutr.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">1106</span>–<span class="cit-lpage">1122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/ajcn/80.5.1106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BClinical%2BNutrition%26rft.stitle%253DAm%2BJ%2BClin%2BNutr%26rft.aulast%253DWilliams%26rft.auinit1%253DJ.%2BH%26rft.volume%253D80%26rft.issue%253D5%26rft.spage%253D1106%26rft.epage%253D1122%26rft.atitle%253DHuman%2Baflatoxicosis%2Bin%2Bdeveloping%2Bcountries%253A%2Ba%2Breview%2Bof%2Btoxicology%252C%2Bexposure%252C%2Bpotential%2Bhealth%2Bconsequences%252C%2Band%2Binterventions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fajcn%252F80.5.1106%26rft_id%253Dinfo%253Apmid%252F15531656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpjbiI7czo1OiJyZXNpZCI7czo5OiI4MC81LzExMDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM5MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.111" data-doi="10.3389/fpls.2015.00534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trovato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forlani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funck</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Physiological implications of arginine metabolism in plants.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D534%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.112" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: a program package for phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.113" data-doi="10.1093/nar/gks479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">dbCAN: a web resource for automated carbohydrate-active enzyme annotation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">W445</span>–<span class="cit-lpage">W451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks479%26rft_id%253Dinfo%253Apmid%252F22645317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645317&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3915.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306670900073&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3915.114" data-doi="10.1038/ismej.2015.251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multi-omics analysis of niche specificity provides new insights into ecological adaptation in bacteria.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2072</span>–<span class="cit-lpage">2075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ismej.2015.251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D10%26rft.spage%253D2072%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2015.251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2015.251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3915.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3903" title="Training Population Optimization for Prediction of Cassava Brown Streak Disease Resistance in West African Clones" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3931" title="The Yeast DNA Damage Checkpoint Kinase Rad53 Targets the Exoribonuclease, Xrn1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3915" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-euwHnC2xKDLOVkIGPNkgxOFjLGP9Tnv1i8831TBI0SU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3915" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3915" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3915" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genomics of Aspergillus flavus S and L Morphotypes Yield Insights into Niche Adaptation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491669" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aqQfQaxNc4JfttiCFKVFZrvsb_cUowMg9m2e-ZPCIkc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3915.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3915" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-t6Ek93FyCH6_cz5YrSzKea_iWO-imy16o1cei-r4vdo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3915" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3915.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491669" id="node491669--2" data-pisa="ggg;8/12/3915" data-pisa-master="ggg;g3.118.200553" data-apath="/ggg/8/12/3915.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of <em>Aspergillus flavus</em> S and L Morphotypes Yield Insights into Niche Adaptation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mana Ohkura</span>, <span class="highwire-citation-author" data-delta="1">Peter J. Cotty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marc J. Orbach</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3915-3930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200553 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491669--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mana Ohkura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mana%2BOhkura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohkura%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMana%2BOhkura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter J. Cotty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BCotty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cotty%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BCotty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marc J. Orbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BJ.%2BOrbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orbach%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarc%2BJ.%2BOrbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marc J. Orbach</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orbachmj@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">orbachmj@email.arizona.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3915" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3915&amp;text=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3915&amp;t=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3915&amp;title=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491669/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491669" id="node491669--3" data-pisa="ggg;8/12/3915" data-pisa-master="ggg;g3.118.200553" data-apath="/ggg/8/12/3915.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of <em>Aspergillus flavus</em> S and L Morphotypes Yield Insights into Niche Adaptation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mana Ohkura</span>, <span class="highwire-citation-author" data-delta="1">Peter J. Cotty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marc J. Orbach</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3915-3930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200553 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491669--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mana Ohkura </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mana%2BOhkura%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohkura%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMana%2BOhkura%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter J. Cotty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS, School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BCotty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cotty%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BCotty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marc J. Orbach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Plant Sciences, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BJ.%2BOrbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orbach%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarc%2BJ.%2BOrbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7714-1352" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marc J. Orbach</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orbachmj@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">orbachmj@email.arizona.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3915&amp;rft.epage=3930&amp;rft.atitle=Comparative%20Genomics%20of%20Aspergillus%20flavus%20S%20and%20L%20Morphotypes%20Yield%20Insights%20into%20Niche%20Adaptation&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ohkura%2C+Mana&amp;rft.au=Cotty%2C+Peter+J.&amp;rft.au=Orbach%2C+Marc+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491669/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491669/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491669/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491669/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491669/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491669/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491669/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491669/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491669/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491669/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491669/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491669/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3915" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3915.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3915.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3915.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3915.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
