<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/12/3767" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/12/3767.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/12/3767.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/12/3767.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/12/3767/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/12/3767.atom" />
<meta name="HW.pisa" content="ggg;8/12/3767" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="QTL Mapping on a Background of Variance Heterogeneity" />
<meta name="DC.Identifier" content="10.1534/g3.118.200790" />
<meta name="DC.Date" content="2018-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Corty, Valdar. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Standard QTL mapping procedures seek to identify genetic loci affecting the phenotypic mean while assuming that all individuals have the same residual variance. But when the residual variance differs systematically between groups, perhaps due to a genetic or environmental factor, such standard procedures can falter: in testing for QTL associations, they attribute too much weight to observations that are noisy and too little to those that are precise, resulting in reduced power and and increased susceptibility to false positives. The negative effects of such “background variance heterogeneity” (BVH) on standard QTL mapping have received little attention until now, although the subject is closely related to work on the detection of variance-controlling genes. Here we use simulation to examine how BVH affects power and false positive rate for detecting QTL affecting the mean (mQTL), the variance (vQTL), or both (mvQTL). We compare linear regression for mQTL and Levene’s test for vQTL, with tests more recently developed, including tests based on the double generalized linear model (DGLM), which can model BVH explicitly. We show that, when used in conjunction with a suitable permutation procedure, the DGLM-based tests accurately control false positive rate and are more powerful than the other tests. We also find that some adverse effects of BVH can be mitigated by applying a rank inverse normal transform. We apply our novel approach, which we term “mean-variance QTL mapping”, to publicly available data on a mouse backcross and, after accommodating BVH driven by sire, detect a new mQTL for bodyweight." />
<meta name="DC.Contributor" content="Robert W. Corty" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2018-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="QTL Mapping on a Background of Variance Heterogeneity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Standard QTL mapping procedures seek to identify genetic loci affecting the phenotypic mean while assuming that all individuals have the same residual variance. But when the residual variance differs systematically between groups, perhaps due to a genetic or environmental factor, such standard procedures can falter: in testing for QTL associations, they attribute too much weight to observations that are noisy and too little to those that are precise, resulting in reduced power and and increased susceptibility to false positives. The negative effects of such “background variance heterogeneity” (BVH) on standard QTL mapping have received little attention until now, although the subject is closely related to work on the detection of variance-controlling genes. Here we use simulation to examine how BVH affects power and false positive rate for detecting QTL affecting the mean (mQTL), the variance (vQTL), or both (mvQTL). We compare linear regression for mQTL and Levene’s test for vQTL, with tests more recently developed, including tests based on the double generalized linear model (DGLM), which can model BVH explicitly. We show that, when used in conjunction with a suitable permutation procedure, the DGLM-based tests accurately control false positive rate and are more powerful than the other tests. We also find that some adverse effects of BVH can be mitigated by applying a rank inverse normal transform. We apply our novel approach, which we term “mean-variance QTL mapping”, to publicly available data on a mouse backcross and, after accommodating BVH driven by sire, detect a new mQTL for bodyweight.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/12/01" />
<meta name="citation_mjid" content="ggg;8/12/3767" />
<meta name="citation_id" content="8/12/3767" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/12/3767" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/12/3767.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/12/3767.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/12/3767.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200790" />
<meta name="citation_pmid" content="30389794" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3767" />
<meta name="citation_lastpage" content="3782" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Robert W. Corty" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Bioinformatics and Computational Biology Curriculum" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6787-9051" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=M. Abney;citation_title=Permutation testing in the presence of polygenic variation.;citation_pages=249-258;citation_volume=39;citation_year=2015;citation_pmid=25758362;citation_doi=10.1002/gepi.21893" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=H. Aschard;citation_author=N. Zaitlen;citation_author=R. M. Tamimi;citation_author=S. Lindström;citation_author=P. Kraft;citation_title=A Nonparametric Test to Detect Quantitative Trait Loci Where the Phenotypic Distribution Differs by Genotypes.;citation_pages=323-333;citation_volume=37;citation_year=2013;citation_pmid=23512279;citation_doi=10.1002/gepi.21716" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Ayroles;citation_author=S. M. Buchanan;citation_author=C. O’Leary;citation_author=K. Skutt-Kakaria;citation_author=J. K. Grenier;citation_title=Behavioral idiosyncrasy reveals genetic control of phenotypic variability.;citation_volume=112;citation_year=2015;citation_pmid=25953335;citation_doi=10.1073/pnas.1503830112" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. J. Barton;citation_author=S. R. Crozier;citation_author=K. A. Lillycrop;citation_author=K. M. Godfrey;citation_author=H. M. Inskip;citation_title=Correction of unexpected distributions of P values from analysis of whole genome arrays by rectifying violation of statistical assumptions.;citation_pages=161;citation_volume=14;citation_year=2013;citation_pmid=23496791;citation_doi=10.1186/1471-2164-14-161" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=T. M. Beasley;citation_author=S. Erickson;citation_author=R. Public;citation_author=H. Building;citation_author=D. B. Allison;citation_title=Rank-based inverse normal transformations are increasingly used, but are they merited?;citation_pages=580-595;citation_volume=39;citation_year=2009;citation_issue=5;citation_pmid=19526352;citation_doi=10.1007/s10519-009-9281-0" />
<meta name="citation_reference" content="Berrett, T., Y. Wang, R. Foygel Barber, and R. Samworth, 2018 The conditional permutation test. ArXiv e-prints ArXiv:1807.05405." />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=G. E. Box;citation_author=D. R. Cox;citation_title=An analysis of transformations.;citation_pages=211-252;citation_volume=26;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=M. B. Brown;citation_author=A. B. Forsythe;citation_title=Robust Tests for the Equality of Variances.;citation_pages=364-367;citation_volume=69;citation_year=1973;citation_doi=10.1080/01621459.1974.10482955" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Y. Cao;citation_author=P. Wei;citation_author=M. Bailey;citation_author=J. S. K. Kauwe;citation_author=T. J. Maxwell;citation_title=A versatile omnibus test for detecting mean and variance heterogeneity.;citation_pages=51-59;citation_volume=38;citation_year=2014;citation_doi=10.1002/gepi.21778" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Naive application of permutation testing leads to inflated type I error rates.;citation_volume=178;citation_year=2008;citation_pmid=18202402;citation_doi=10.1534/genetics.107.074609" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=C. J. Clopper;citation_author=E. S. Pearson;citation_title=THE USE OF CONFIDENCE OR FIDUCIAL LIMITS ILLUSTRATED IN THE CASE OF THE BINOMIAL;citation_pages=404-413;citation_volume=26;citation_year=1934;citation_issue=4;citation_doi=10.1093/biomet/26.4.404" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc.;citation_author=W. G. Cochran;citation_title=Problems arising in the analysis of a series of similar experiments. Suppl.;citation_pages=102-118;citation_volume=4;citation_year=1937" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=V. Kumar;citation_author=L. Tarantino;citation_author=J. Takahashi;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_author=R. W. Corty;citation_author=W. Valdar;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. Dudbridge;citation_author=B. P. C. Koeleman;citation_title=Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.;citation_pages=424-435;citation_volume=75;citation_year=2004;citation_issue=3;citation_pmid=15266393;citation_doi=10.1086/423738" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Dumitrascu;citation_author=G. Darnell;citation_author=J. Ayroles;citation_author=B. E. Engelhardt;citation_title=Statistical tests for detecting variance effects in quantitative trait studies.;citation_year=2018;citation_doi=10.1093/bioinformatics/bty565" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=M. Felleki;citation_author=D. Lee;citation_author=Y. Lee;citation_author=R. Gilmour;citation_author=L. Rönnegård;citation_title=Estimation of breeding values for mean and dispersion, their variance and correlation using double hierarchical generalized linear models.;citation_pages=307-317;citation_volume=94;citation_year=2012;citation_doi=10.1017/S0016672312000766" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. K. Forsberg;citation_author=M. E. Andreatta;citation_author=X. Y. Huang;citation_author=J. Danku;citation_author=D. E. Salt;citation_title=The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.;citation_pages=e1005648;citation_volume=11;citation_year=2015;citation_pmid=26599497;citation_doi=10.1371/journal.pgen.1005648" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=S. K. Forsberg;citation_author=Ö. Carlborg;citation_title=On the relationship between epistasis and genetic variance heterogeneity.;citation_pages=5431-5438;citation_volume=68;citation_year=2017;citation_doi=10.1093/jxb/erx283" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. B. Fraser;citation_author=E. E. Schadt;citation_title=The quantitative genetics of phenotypic robustness.;citation_pages=e8635-e8635;citation_volume=5;citation_year=2010;citation_issue=1;citation_pmid=20072615;citation_doi=10.1371/journal.pone.0008635" />
<meta name="citation_reference" content="citation_journal_title=J. Bus. Econ. Stat.;citation_author=D. Freedman;citation_author=D. Lane;citation_title=A nonstochastic interpretation of reported significance levels.;citation_pages=292-298;citation_volume=1;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Freund;citation_author=A. M. Brandmaier;citation_author=L. Lewejohann;citation_author=I. Kirste;citation_author=M. Kritzler;citation_title=Emergence of Individuality in Genetically Identical Mice;citation_pages=756-759;citation_volume=340;citation_year=2013;citation_issue=6133;citation_pmid=23661762;citation_doi=10.1126/science.1235294" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. Gibson;citation_title=Decanalization and the origin of complex disease.;citation_pages=134-140;citation_volume=10;citation_year=2009;citation_issue=2;citation_pmid=19119265;citation_doi=10.1038/nrg2502" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=P. N. Gonzalez;citation_author=M. Pavlicev;citation_author=P. Mitteroecker;citation_author=F. Pardo-Manuel de Villena;citation_author=R. A. Spritz;citation_title=Genetic structure of phenotypic robustness in the collaborative cross mouse diallel panel.;citation_pages=1737-1751;citation_volume=29;citation_year=2016;citation_doi=10.1111/jeb.12906" />
<meta name="citation_reference" content="citation_title=Permutation tests: a practical guide to resampling methods for testing hypotheses;citation_author=P. Good;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. C. Grubb;citation_author=C. J. Bult;citation_author=M. A. Bogue;citation_title=Mouse Phenome Database.;citation_volume=42;citation_year=2014;citation_pmid=24243846;citation_doi=10.1093/nar/gkt1159" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=H. A. Mulder;citation_title=Genetic analysis of environmental variation.;citation_pages=381-395;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429270;citation_doi=10.1017/S0016672310000546" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Huang;citation_author=M. A. Carbone;citation_author=M. M. Magwire;citation_author=J. A. Peiffer;citation_author=R. F. Lyman;citation_title=Genetic basis of transcriptome diversity in Drosophila melanogaster.;citation_volume=112;citation_year=2015;citation_pmid=26483487;citation_doi=10.1073/pnas.1519159112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Hulse;citation_author=J. J. Cai;citation_title=Genetic variants contribute to gene expression variability in humans.;citation_volume=193;citation_year=2013;citation_pmid=23150607;citation_doi=10.1534/genetics.112.146779" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=E. V. Ivarsdottir;citation_author=V. Steinthorsdottir;citation_author=M. S. Daneshpour;citation_author=G. Thorleifsson;citation_author=P. Sulem;citation_title=Effect of sequence variants on variance in glucose levels predicts type 2 diabetes risk and accounts for heritability.;citation_pages=1398-1402;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3928" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=B. W. Kennedy;citation_author=M. Quinton;citation_author=J. A. van Arendonk;citation_title=Estimation of effects of single genes on quantitative traits;citation_pages=2000-2012;citation_volume=70;citation_year=1992;citation_issue=7;citation_pmid=1644672;citation_doi=10.2527/1992.7072000x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=S. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=L. J. Leamy;citation_author=D. Pomp;citation_author=E. J. Eisen;citation_author=J. M. Cheverud;citation_title=Quantitative trait loci for directional but not fluctuating asymmetry of mandible characters in mice.;citation_pages=27-40;citation_volume=76;citation_year=2000;citation_issue=1;citation_pmid=11006632;citation_doi=10.1017/S0016672300004559" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=L. J. Leamy;citation_author=D. Pomp;citation_author=E. J. Eisen;citation_author=J. M. Cheverud;citation_title=Pleiotropy of quantitative trait loci for organ weights and limb bone lengths in mice.;citation_volume=10;citation_year=2002;citation_pmid=12118102;citation_doi=10.1152/physiolgenomics.00018.2002" />
<meta name="citation_reference" content="citation_title=Contributions to Probability and Statistics: Essays in Honor of Harold Hotelling;citation_title=Robust tests for equality of variances;citation_author=H. Levene;citation_pages=278-292;citation_year=1960" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Lin;citation_author=Z.-X. Chen;citation_author=B. Oliver;citation_author=S. T. Harbison;citation_title=Microenvironmental gene expression plasticity among individual drosophila melanogaster.;citation_volume=6;citation_year=2016;citation_pmid=27770026;citation_doi=10.1534/g3.116.035444" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=J. S. Long;citation_author=L. H. Ervin;citation_title=Using Heteroscedasticity Consistent Standard Errors in the Linear Regression Model.;citation_pages=217-224;citation_volume=54;citation_year=2000;citation_doi=10.2307/2685594" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=G. B. Makumburage;citation_author=A. E. Stapleton;citation_title=Phenotype Uniformity in Combined-Stress Environments has a Different Genetic Architecture than in Single-Stress Treatments.;citation_pages=12-12;citation_volume=2;citation_year=2011;citation_pmid=22645526;citation_doi=10.3389/fpls.2011.00012" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=O. Martínez;citation_author=R. N. Curnow;citation_title=Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.;citation_pages=480-488;citation_volume=85;citation_year=1992;citation_pmid=24197463;citation_doi=10.1007/BF00222330" />
<meta name="citation_reference" content="citation_journal_title=R. Soc. Open Sci.;citation_author=D. J. Mitchell;citation_author=B. G. Fanson;citation_author=C. Beckmann;citation_author=P. A. Biro;citation_title=Towards powerful experimental and statistical approaches to study intraindividual variability in labile traits.;citation_volume=3;citation_year=2016;citation_doi=10.1098/rsos.160352" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=M. Mosedale;citation_author=Y. Kim;citation_author=W. J. Brock;citation_author=S. E. Roth;citation_author=T. Wiltshire;citation_title=Editor’s highlight: Candidate risk factors and mechanisms for tolvaptan-induced liver injury are identified using a collaborative cross approach.;citation_volume=156;citation_year=2017;citation_pmid=28115652;citation_doi=10.1093/toxsci/kfw269" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Paré;citation_author=N. R. Cook;citation_author=P. M. Ridker;citation_author=D. I. Chasman;citation_title=On the use of variance per genotype as a tool to identify quantitative trait interaction effects: a report from the Women's Genome Health Study.;citation_pages=e1000981-e1000981;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585554;citation_doi=10.1371/journal.pgen.1000981" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=T. J. Rao;citation_author=M. A. Province;citation_title=A Framework for Interpreting Type I Error Rates from a Product-Term Model of Interaction Applied to Quantitative Traits.;citation_pages=144-153;citation_volume=40;citation_year=2016;citation_doi=10.1002/gepi.21944" />
<meta name="citation_reference" content="Roach, J., and W. Valdar, 2018 Permutation tests of non-exchangeable null models. ArXiv e-prints arXiv:1808.10483." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Detecting major genetic loci controlling phenotypic variability in experimental crosses.;citation_volume=188;citation_year=2011;citation_pmid=21467569;citation_doi=10.1534/genetics.111.127068" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.;citation_pages=63-63;citation_volume=13;citation_year=2012;citation_pmid=22827487;citation_doi=10.1186/1471-2156-13-63" />
<meta name="citation_reference" content="Sachs, M. C., 2017 plotROC: A Tool for Plotting ROC Curves. J. Stat. Softw. 79: c02." />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=X. Shen;citation_author=Ö. Carlborg;citation_title=Beware of risk for increased false positive rates in genome-wide association studies for phenotypic variability.;citation_pages=93;citation_volume=4;citation_year=2013;citation_doi=10.3389/fgene.2013.00093" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. Shen;citation_author=J. De Jonge;citation_author=S. K. Forsberg;citation_author=M. E. Pettersson;citation_author=Z. Sheng;citation_title=Natural CMT2 Variation Is Associated With Genome-Wide Methylation Changes and Temperature Seasonality.;citation_pages=e1004842;citation_volume=10;citation_year=2014;citation_pmid=25503602;citation_doi=10.1371/journal.pgen.1004842" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=X. Shen;citation_author=L. Ronnegard;citation_title=Issues with data transformation in genome-wide association studies for phenotypic variability.;citation_pages=200;citation_volume=2;citation_year=2013;citation_doi=10.12688/f1000research.2-200.v1" />
<meta name="citation_reference" content="citation_journal_title=Soc. Ser. B Methodol.;citation_author=G. K. Smyth;citation_title=Generalized linear models with varying dispersion. J. R. Stat.;citation_pages=47-60;citation_volume=51;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=E. C. Snell-Rood;citation_author=E. M. Swanson;citation_author=R. L. Young;citation_title=Life history as a constraint on plasticity: developmental timing is correlated with phenotypic variation in birds.;citation_pages=379-388;citation_volume=115;citation_year=2015;citation_doi=10.1038/hdy.2015.47" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=D. Soave;citation_author=H. Corvol;citation_author=N. Panjwani;citation_author=J. Gong;citation_author=W. Li;citation_title=A Joint Location-Scale Test Improves Power to Detect Associated SNPs, Gene Sets, and Pathways.;citation_pages=125-138;citation_volume=97;citation_year=2015;citation_pmid=26140448;citation_doi=10.1016/j.ajhg.2015.05.015" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=D. Soave;citation_author=L. Sun;citation_title=A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.;citation_pages=960-971;citation_volume=73;citation_year=2017;citation_doi=10.1111/biom.12651" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Sorensen;citation_author=R. Waagepetersen;citation_title=Normal linear models with genetically structured residual variance heterogeneity: a case study.;citation_pages=207-222;citation_volume=82;citation_year=2003;citation_issue=3;citation_pmid=15134199;citation_doi=10.1017/S0016672303006426" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Sørensen;citation_author=G. de los Campos;citation_author=F. Morgante;citation_author=T. F. C. Mackay;citation_author=D. Sorensen;citation_title=Genetic Control of Environmental Variation of Two Quantitative Traits of Drosophila melanogaster Revealed by Whole-Genome Sequencing.;citation_volume=201;citation_year=2015;citation_pmid=26269504;citation_doi=10.1534/genetics.115.180273" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=M. V. Struchalin;citation_author=A. Dehghan;citation_author=J. C. Witteman;citation_author=C. van Duijn;citation_author=Y. S. Aulchenko;citation_title=Variance heterogeneity analysis for detection of potentially interacting genetic loci: method and its limitations.;citation_pages=92-92;citation_volume=11;citation_year=2010;citation_pmid=20942902;citation_doi=10.1186/1471-2156-11-92" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=X. Sun;citation_author=R. Elston;citation_author=N. Morris;citation_author=X. Zhu;citation_title=What is the significance of difference in phenotypic variability across SNP genotypes?;citation_pages=390-397;citation_volume=93;citation_year=2013;citation_pmid=23910463;citation_doi=10.1016/j.ajhg.2013.06.017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=J. Flint;citation_author=R. Mott;citation_title=Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.;citation_volume=172;citation_year=2006;citation_pmid=16361245;citation_doi=10.1534/genetics.104.039313" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=P. M. Visscher;citation_author=D. Posthuma;citation_title=Statistical power to detect genetic Loci affecting environmental sensitivity.;citation_pages=728-733;citation_volume=40;citation_year=2010;citation_issue=5;citation_pmid=20428936;citation_doi=10.1007/s10519-010-9362-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Visscher;citation_author=R. Thompson;citation_author=C. S. Haley;citation_title=Confidence Intervals in QTL Mapping by Bootstrapping;citation_pages=1013-1020;citation_volume=143;citation_year=1996;citation_issue=2;citation_pmid=8725246" />
<meta name="citation_reference" content="citation_title=Bayesian and Frequentist Regression Methods;citation_author=J. Wakefield;citation_year=2013;citation_doi=10.1007/978-1-4419-0925-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Wang;citation_author=E. Yang;citation_author=C. L. Brinkmeyer-Langford;citation_author=J. J. Cai;citation_title=Additive, epistatic, and environmental effects through the lens of expression variability QTL in a twin cohort.;citation_volume=196;citation_year=2014;citation_pmid=24298061;citation_doi=10.1534/genetics.113.157503" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=A. Wolc;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_author=N. P. O’Sullivan;citation_title=Genome-wide association analysis and genetic architecture of egg weight and egg uniformity in layer chickens.;citation_pages=87-96;citation_volume=43;citation_year=2012;citation_pmid=22742506;citation_doi=10.1111/j.1365-2052.2012.02381.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=J. B. Wolf;citation_author=D. Pomp;citation_author=E. J. Eisen;citation_author=J. M. Cheverud;citation_author=L. J. Leamy;citation_title=The contribution of epistatic pleiotropy to the genetic architecture of covariation among polygenic traits in mice.;citation_pages=468-476;citation_volume=8;citation_year=2006;citation_issue=5;citation_pmid=16925682;citation_doi=10.1111/j.1525-142X.2006.00120.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Yadav;citation_author=K. Dhole;citation_author=H. Sinha;citation_title=Differential regulation of cryptic genetic variation shapes the genetic interactome underlying complex traits.;citation_volume=8;citation_year=2016;citation_doi=10.1093/gbe/evw258" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Yang;citation_author=R. Loos;citation_author=M. Goddard;citation_author=P. M. Visscher;citation_title=FTO genotype is associated with phenotypic variability of body mass index.;citation_pages=267-272;citation_volume=490;citation_year=2012;citation_issue=7419;citation_pmid=22982992;citation_doi=10.1038/nature11401" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=F. Yates;citation_author=W. G. Cochran;citation_title=The analysis of groups of experiments.;citation_pages=556-580;citation_volume=28;citation_year=1938;citation_doi=10.1017/S0021859600050978" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=D. Shriner;citation_author=S. Banerjee;citation_author=T. Mehta;citation_author=D. Pomp;citation_title=An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects.;citation_volume=176;citation_year=2007;citation_pmid=17483424;citation_doi=10.1534/genetics.107.071365" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=B. S. Yandell;citation_author=G. A. Churchill;citation_author=D. B. Allison;citation_author=E. J. Eisen;citation_title=Bayesian model selection for genome-wide epistatic quantitative trait loci analysis.;citation_volume=170;citation_year=2005;citation_pmid=15911579;citation_doi=10.1534/genetics.104.040386" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=N. Yi;citation_author=D. K. Zinniel;citation_author=K. Kim;citation_author=E. J. Eisen;citation_author=A. Bartolucci;citation_title=Bayesian analyses of multiple epistatic QTL models for body weight and body composition in mice.;citation_pages=45-60;citation_volume=87;citation_year=2006;citation_issue=1;citation_pmid=16545150;citation_doi=10.1017/S0016672306007944" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Zou;citation_author=Z. Xu;citation_author=T. Vision;citation_title=Assessing the significance of quantitative trait loci in replicable mapping populations.;citation_volume=174;citation_year=2006;citation_pmid=16888335;citation_doi=10.1534/genetics.106.059469" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="QTL Mapping on a Background of Variance Heterogeneity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
<meta name="twitter:description" content="Standard QTL mapping procedures seek to identify genetic loci affecting the phenotypic mean while assuming that all individuals have the same residual variance. But when the residual variance differs systematically between groups, perhaps due to a genetic or environmental factor, such standard procedures can falter: in testing for QTL associations, they attribute too much weight to observations that are noisy and too little to those that are precise, resulting in reduced power and and increased susceptibility to false positives. The negative effects of such “background variance heterogeneity” (BVH) on standard QTL mapping have received little attention until now, although the subject is closely related to work on the detection of variance-controlling genes. Here we use simulation to examine how BVH affects power and false positive rate for detecting QTL affecting the mean (mQTL), the variance (vQTL), or both (mvQTL). We compare linear regression for mQTL and Levene’s test for vQTL, with tests more recently developed, including tests based on the double generalized linear model (DGLM), which can model BVH explicitly. We show that, when used in conjunction with a suitable permutation procedure, the DGLM-based tests accurately control false positive rate and are more powerful than the other tests. We also find that some adverse effects of BVH can be mitigated by applying a rank inverse normal transform. We apply our novel approach, which we term “mean-variance QTL mapping”, to publicly available data on a mouse backcross and, after accommodating BVH driven by sire, detect a new mQTL for bodyweight." />
<meta name="og-title" property="og:title" content="QTL Mapping on a Background of Variance Heterogeneity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/12/3767" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Standard QTL mapping procedures seek to identify genetic loci affecting the phenotypic mean while assuming that all individuals have the same residual variance. But when the residual variance differs systematically between groups, perhaps due to a genetic or environmental factor, such standard procedures can falter: in testing for QTL associations, they attribute too much weight to observations that are noisy and too little to those that are precise, resulting in reduced power and and increased susceptibility to false positives. The negative effects of such “background variance heterogeneity” (BVH) on standard QTL mapping have received little attention until now, although the subject is closely related to work on the detection of variance-controlling genes. Here we use simulation to examine how BVH affects power and false positive rate for detecting QTL affecting the mean (mQTL), the variance (vQTL), or both (mvQTL). We compare linear regression for mQTL and Levene’s test for vQTL, with tests more recently developed, including tests based on the double generalized linear model (DGLM), which can model BVH explicitly. We show that, when used in conjunction with a suitable permutation procedure, the DGLM-based tests accurately control false positive rate and are more powerful than the other tests. We also find that some adverse effects of BVH can be mitigated by applying a rank inverse normal transform. We apply our novel approach, which we term “mean-variance QTL mapping”, to publicly available data on a mouse backcross and, after accommodating BVH driven by sire, detect a new mQTL for bodyweight." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12.cover-source.jpg" />
    <title>QTL Mapping on a Background of Variance Heterogeneity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"m5kkZik8rtNpsLbsmC_tOGJp-7p4v-Jc5ltJ564dxz8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491653","apath":"\/ggg\/8\/12\/3767.atom","pisa":"ggg;8\/12\/3767","ac":{"\/ggg\/8\/12\/3767.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/12\/3767.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3767"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/12\/3767"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/12\/3767":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227yKmYRuYSEymBHsdQspHggL2sz+iGuk6DsMbGr+cY\\\\\\\/tHjd7pqmF5+UN+7cVNQX5dk5G0rO5CRE9jypedwxFt7tFY4u\\\\\\\/sKbn6sc4CQLkmUhB5w3C077z1gZK+Twvc3YEv2Tdof5SoQmWZYLs363\\\\\\\/y7od0bbdICYNTvylxNyv7s4FRuh8eRx2F4xWhpkj3Qsj01KuqAIuu\\\\\\\/18E7wn6sv9aSvw7\\\\\\\/MlA4llo1rUndWUSKBU2jwSfSLzuBPmwHWWOMSaOoi9wqYRMRWF5MYEFjWIzWH+1NUb+RCnvg6jMEVVjsHbuKQ3p5kxjoEuFldMtRlPPRF7TSS2PYcMJlTNNv14bbphfjjU5bbzyJUtbcAZtp0kpMthXoWJEgpII6ISs9ZaCjjqZ8e4Mo1zmG0q1ExUvwuwp\\\\\\\/igx2RltW+lQsyR4jCzPtTt5IyWhR3m1WdVDbbu1zcblWYUKY3ALMSqLBZ0BYldpDCV5yJSRd1YhgshTIU032X3zTfVdRa96tuJePLlvHl1gjV2vgMgAVa3+8ouC\\\\\\\/E8qTf6vWqbV22aqegU8ecsfaycC\\\\\\\/tmknf9Ql99\\\\\\\/+xPyMAi9GjghNI8mBNoKmJ8iBhRcD4P2cGa\\\\\\\/eZUYBcBGchWBGKdUqsuoCP66U17oIxhfAYpkt4yRaOATv29DPjdgLtAYrQNf3Jy7XWJUIF63YtAQYy07dSsVybbCNu1fgvxZuBtBhUkPkiyqaud5zRHJ+Wri9naBVy6Nxhi9Bt1SZGWSb\\\\\\\/ag2ZsTaNXXHAAS+nqhwrnhEAs7PGYSC\\\\\\\/iAoaj00pd6zSjTZCaiIRGcnhygnOmeI451pcwNsng\\\\\\\/v9gr+aoSis6ISzcM8tPAMywecfVregB2HUurB\\\\\\\/YczpwMNtRPTlCmoQV22W9C0xE1dX6eDqe7+b3mmSlCwzIa\\\\\\\/VudadlFPfbIT4lidsUbfOPxOsS2zqLYb8k+c23CjlXmpqJHwiPKRwwEYhSqFnRAQL4JHZWHtrv+oGKKx0mKY1629rLJNMb5z9rUNrSc37VHyvavzDePWrmkANKIPg\\\\\\\/qIerPkbwPph40L9gns3axqAGDqh3FJ4kivUenaBtELTd5M8+ZylweRd6gz8gSk7BNdgORdy\\\\\\\/fnuvCAJ0whsHeDayVG+aRGFU8uJc\\\\\\\/Fc6X1Udbqa2YBIdtDx8oVIPzsuMWiKbyyPYRdzTqw12cNq\\\\\\\/2F4zrDO7kt+S\\\\\\\/gARcWgTljsC9DbjDR1Qivq3eeVVMAG5bW4eOgx\\\\\\\/6EAPyWWdlR37FhR5GThgq1G\\\\\\\/v3PC9nYyYX6tSWPp\\\\\\\/COcU\\\u0022,\\\u0022iv\\\u0022:\\\u002249jRQ2UDKfNjKKDx25C0WQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022704337de6caca2b3ed17805d2d0f97c0fd298ea6e9259e93248a23f88cc8226b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rP0E7xOaOihir1j9t3rW6f\\\\\\\/S+S4KsC9\\\\\\\/gO8qDgsdJgIWstWNeA62UPdUXccbWUtuG+y8OyJRK1A0tdHluGe9IQ5YXmYUwOzFBnSJNTtTflM9WGNlTm1jzUHnzRuonpk6YHnF9P9xDrNUKFl+oGilIGRSw2yEzMEMdAmq5YSYuXBUmCB0k\\\\\\\/5Gfn9AzPqaPXa9iq28FkU5tWCYFnpGILdsx7EbpYjvGYzdQPmuxghQYBqMTEXLAg3gu3wEWOUMq3cyKmbxOuaUywh6OsUe5w0WvGoFIWDA\\\\\\\/Oj44IwbfSFDqcO2t7Or6blnWNcQ\\\\\\\/CA5EuNP5wNFJtiEy9RyLFL\\\\\\\/09gUCF0a3u7vrGGr9EX24ylDJkl3menPLD3BaahF+IZuhN7c0fJZ+SC8R3BIW69vjEj+sxEV04XAnoldNBaSIB+Oou+JCP50WBYKmJC9o9D0oPHPqdbGeCfUHhLJF9fK63kwWcUzP0IR+hjOciePf6QUFOTRu6EQNHDTsKKU07rTGqiZzous8zlZPmYjS4zdsdZZI4MMDM6izOCIYyWYRKRLf+5f6pObeQWIOgnKoS7Td7oKwHJlzt7jlOyDUHbDU6f6hAJBFdcv4AmFLPDK4Fyuqfw+YmHfxxCtztJfwaFc\\\\\\\/CvpETG+PCQ9wzH8esIOEBEGKVKYe+rgxmE9EdsOx7TAgkpHc\\\\\\\/DZG2qD0vKIHhw6YxaLn6nOTMfTOntRlnq1YOKYqHR+kYHfOPlijo\\\\\\\/WzxEnTXWFY1RWjI6\\\\\\\/RWf\\\\\\\/rAUCz3L1x+LSHR8ZmfRox0Q3\\\\\\\/QfV3Bajfbw7f7v6xI7IofiykzlDcOlVc878TWZH76Q3ZRalwGtcSwlHj9jP9GOxoXrAY8GHUVxbOydGbxJYkpfItQYA5C+HQAKq4eeuTXOs+986Mm0ZE0cUxjND0v0nxAZa3dV7uGe2WXtLZVtqEjqHQCEYhJqIef7eCcsrUtcdmCj2k7BaoXaJpH4du6vA3vrkYmL8LjWLwSn3VwTvtMHSCCPr5BeTZFZyMZ6d595FBK6ekJ8cQCnyUisBTmW0BgkkzM1eY0Pc5o9gb\\\\\\\/Q5aDcW\\\\\\\/vTR3FCU2qHi3LU9mWGxWG505LwECfShU\\\\\\\/wajBVZFLjTaJOFSO4RhHDP6BXWXConKximp1JP5ush0+t489xXpIwi4yhnTRD1KHYpS0m4tbVsWyTruBbCjVQ\\\\\\\/imnVDkDhvxbjX9+UAB0xLQ4xaSGWzr3FVWNinkN5euwTf+h1qt4lRhMyXsWGzoYjJvEgyG3vHVoEAINP+nLaJbHXJ3nYVwa1zCfNgf2dAWDFgNlSP4fIPuqi0Htu5e1eFvelJU7Va\\\\\\\/YX0uU14FfiI\\\\\\\/9+Va\\\\\\\/gd0G8UalLMdLhVQg2468qc\\\\\\\/M5k6wuGzScRk9UU3JcvDSamibONJmGp+SeP+g7XeK28gGkcnhapFSrAbe1V2fIO\\\\\\\/WQeaKt6VLDgGNboPnxl8nON16qTUFOrJiHa5ujf9sna2tLjB5aT9OKXM4fmByvVQ4PJD+MdG0W3WmVx49Tl6x+NAZ16IyMCY8wACu0RVCkUSloI\\\\\\\/UHVhqkxDzbLOYXGjiXqR4EAtRiTzxxgt+uHsjJ8rFXbXtnSIhnikP2r7hE1CqdJZp0Jd0vCcXnf+dSI09jc15APZcTxC62momsWy2maaA8QbIWonQ2n0xxiry3q2mR9ftjzEuwQiwnMyMmzZBz0OlKNCLFX+RJzNOojJNf8spOCRuiQHMhb2\\\\\\\/pZzXGd83Lr6dkytjeFF8rFBNkXzQP967ft+5auLnnx+p4PBY2iwsa2XgNgn28JTPLTdZcr3QhEt0pu+KmAXWoBUmuCW100w\\\\\\\/p1LBXvivah04QNkO5DBwwlR0TT74GqGlB7taKLmKNu8Kf9e45IMuVH0BhwAC1bsUNqTVLznEl1tDSGhGQkDHrov\\\\\\\/H0zy5bnz3mJ7EGZYcRPbg+NC+jFa3KfhXWAFKVxfRuBUMqsvIbLbqsDAJmKvroWp\\\\\\\/ULoiqSKhY\\\\\\\/JkEzJgymaxjb7JAuQij9f2hPYv2O5eBSvHk3A8fsKSxHlErB4vEjQD38rl4V4Xl6VOsgKTUjE8tG\\\\\\\/ev1vmZYEYeMLmGYWRcRAJc2qMDCXVpC\\\\\\\/uHkze8v0SMb3aplEFe2oaWCM7WuUoR30TB8b+JAhwflVsksrVEOxU4P7mWrPC+dph\\\\\\\/1lTQjL1\\\\\\\/KfLA6GyF23pWWfqme7J\\\\\\\/h04RFY0c06BMSEcd6u5IcWv32TlGrOy1a8sNATRMjZl9p5KQUKGMhIySbk\\\\\\\/h6398dJIhIN8+cFAUeEYKVg2T5C+FyQYpCK5SzQ1RdpX\\\\\\\/Az7mRGpHxqyJYIEa0SOS2aYwwc6ljrBVm2StT8CmOrYS06ifTo+\\\\\\\/1hn7csBF3M8hyPm8VkDRFslbmzg3WH8l2pmDZ0F9M5mPET1ZmQYs3XMkLtuDEKCQnlgvmpWOEn6YzmTe24fv4dPEih9wcwcPPGkpbJqkSOvPNWY1MpWtsFFvvERXmRut5s9ey+Hn8KgsmjsogeooUL4DxioYjagpYKS1VCLaooQ95Lc8HttMWpncXTRG7kbEAdT\\\\\\\/bzh6QVGC7aBlawY2Zcq\\\\\\\/CNvSlMQ+gzvH534GvlZCeYx3PF65Iaz7QAHo6\\\\\\\/YRrZaRlPm5tz5kLH\\\\\\\/5V0pA7hRks7Cl7xdmRVkUHopCVvDRx\\\\\\\/WCvMDsd8OJcp7JQHIjMw7\\\\\\\/t+oG6mjhZkWaCXg1lrm+F6N3kqYAX2y+JoyzQVBF0ssM1BylS4EkpZECsnCuFRBrB6T6ye7WyxBfJgKA8xR50\\\\\\\/7aG4FvBO7zzDkuMVeZW\\\\\\\/ATUsM\\\\\\\/GHqnxyr9CyG1xPt+GGN5Zi8LLUXEERpHoy7PgGgaL\\\\\\\/j1reH2drZZr7b\\\\\\\/Qn50ndcN6wMeQ3xUrYHovuGBEISsp9URmzIweGVrakizpowhduPhrX84x9Xcah2Q0+60q8FtZZZD8bHzBoBDyEXUu59jgOmP\\\\\\\/D8an3i\\\\\\\/KS4isyT0kcloA4GaQBiJ04QR4anJlMO791\\\\\\\/\\\\\\\/wcSZ0RnjlXau1S1JxkUCdoykAzbh2ntwTQsc+15fUkY780gs+vqfDxGWCScxJk2bedOQdXGSXrEcJm4Jy4rB5s1tBAnj2yMUzIGn4gT\\\\\\\/bJEphqayghmUaWPDJvB+UM4AxJeEyctv4mTb1dzt71hmt0CAkJsxZK5Vi6fIaTxdc5vABC5ZRWXeed5FyVLk1QjbEymD0HFShrNDRZtwKWUX0lrDPWyzlbmPeD38VLEDg3vmfKfHVwO\\\\\\\/dgVxTt26SoGbewUs82EEZ5JRqZ8IEM11iTScyAFeYZgCMcyvgAi4mhVVAz7kV88F4uYlJBn8mkR5FTVtBMvf7cUaUSsc5iSjk8M6d1LHTSWLOrENNiuZjJPB0iuf9ql9SP\\\\\\\/GHtjz7RvPp4ajYUPwvH5VhPlrbXMpenzqZbIFJv2iRU9ZrU+whjAesR2gtFXU66QpqKGUGBbnaUvDYXb4j+sAFCWDbrXUHZLnigywjkcu2lAxpm8wcUZSlbj1BYPm9NffZk5sogN2ozbqb92wTRqScbtbU4gXXUGNyVhTzQDzUHgLli52o0ERGW5tucQuOFusLFHMFa1C90F964vIasjBi+HwUCC\\\\\\\/HmOxdFCwyx4I+LKfPOi9N5ctD0YRJ84FeH7FBQVAbZFEszPxDFJDKDilUvR8a13s5UbX+LIYFk8s9FYEEE\\\\\\\/9HT4tUqreax1us93H7zm5NfQWO5lw24iMJeL9h42pcgLsg6of2VhceVW7ApFBz9OpOQzdm9gVMWVA\\\\\\\/pghYE2f3QoajSajiwmksjaCtPt6b\\\\\\\/7jfbPlfR\\\\\\\/Eugu37Ub7YYSFsL\\\\\\\/9Pkrr1e2xM6bhk5D0B7ZGtCGU8vuUeoPibIv7OlG9feVqlberMwfOkJKK9guOR9iwNOYN73TGNdd5Mgc+Yn\\\\\\\/iYWOexW8oW48znT2C\\\\\\\/dU+c4j8kvK\\\\\\\/NghbkQj2Kzbu5IFd39Bo1TyirP6VrCSKg2Mll2FTQtzN8IcfDsq2pX47Xz6iWDfPSrRqXbtabeb7qlvf1TvnUNOco1p0iZsNYyb90SXdRBsTQRg5kFodavTz9AtkDAzGcrME2B83YwisKFYPBrWrgVabWwSMnK4M1Nr7gixBZ82YMWjUBPisrybYTjY8nAbCvodNUjhFXtEkOpMZMxH\\\\\\\/8eqkFjWw4qgeQjmblx0lJWYe555VGLGJ8DFjPNFWLRTew6jCv0FC53ODMv4hRqbIDv03QO0\\\\\\\/qEVA2yWR9R3mk3DD0WYm9pg0hC76+TD4ZIxBWixFs8g8IfWmCUPHjCccdbuXzKKP6jfuKts8\\\\\\\/z0gTiJaMRMuYqjJU645bwW20FVbtBM32wlOcfOqJ\\\\\\\/gfMMDbKvwzJX4qjIO\\\\\\\/UynoD3DBLayqZtsT7mhghqFUqH9Xllh8cFGaE2Yr+SarkIPxgpCcAQ3934M720v65XGdvaPLL4aAIdlV47XuhD\\\\\\\/ZI8qh0d7nRTdqKM6sICaZDzjuTe3ZOa58wCOJck+O3oOxdJt\\\\\\\/ARPVKhuodz9NmjgkW40Vg8dChRVBFrIuekn9p1VgCu0kqAuJKu1zw9iYOnno2NtCMYvqmgQGwtpc9KBtdQB9w6N0pJaUxL\\\\\\\/LLPXsMlPyF\\\\\\\/gF6iu0ccumK4MNMEfXbrtWDbGDeNidtwE9LiHh6dp1XI1ux3\\\\\\\/NbkTPpASfbQ\\\\\\\/\\\\\\\/qixAcfAr1bV4R2zQlZGYZABajxEh2R5tUz3HtFBwxXbaw+hHoE4gCjRx576DRE1Cq2wBLj+GRr7l1inGqpSHCG6DTfrQvYnM94OuI81Cd8vUzvT+mQv548wY1aiQTECLXE76IduBOE5AL10l3Gtq6DzS0ACLzkZTd3En4T7hE5AP4LFquW6dJiNw0b7pkCCwBUaxgcKMM+935fXMCxttPh+qXhb9ldw\\\\\\\/\\\\\\\/qZMf0G\\\\\\\/KLU1\\\\\\\/rZi+aT4APq5XiBt06dwBItPb7VnO\\\\\\\/IQNV+WASRhrDYfsvaJJSs0UCz6RXwWJVxLXIb6mKUwzvAyx1ocneFaKUOufr\\\\\\\/sjm2z40IebMApr6JkFKz71diGoHW52I7gDNipmCIKJ8bitgDyNctRs36qTToBEViXvYOgzDU0jGK7bHiMx48HLCbvppkli\\\\\\\/L64tV2CJSKfbNU6p8d9gHh4pE484qlfQYNH4tsDOJFz7AGkUfFqKpDqk7QN2ZPJYNMeXWwS3PgLu9FZINH7Wlqd2mjxwftLIJpDBKBc8CfZtVn+Uiq+TGyUzgdLXWsnWvh+Nu7CKCVMgiJlt5ZYSSn\\\\\\\/GxJ8KlQWy39sWGWD7g9Z3oiMBw1q7Jlssf0ny6Udi7SR8s4L27bGkXmHa0Tfzn9bHHp9mg5Z56WYAIEH2MMqzPjsRWNuJBmwBHddJmd+Wf224yv5wQ7A6LnzEecB+tiUoYCRtZf9VCPg0Yj8Oq6tebhZ4MvJ0at+vQkvDhkgVwTGe5zsMy7EB8eFCWnBiqA3sAeZCEKBdRDy9kUtAGKXaoSm9JKjmvXdog\\\\\\\/0ULwGVm8PSkHdRpzGCUTbkkl6zmrZuOi4BYrpqkvBsinpCeE\\\\\\\/3REZZKKdH191T8dpQ\\\\\\\/p4Ooa\\\\\\\/bowtxd2nPQYGloE9buKAHfGWN2odOeeYfF9HpJYwlgZb2r37Jnt1ySQasgRy29NUSDCbh+L0gP7JYdYFXCk9I\\\\\\\/aijAnINwgbyIsCGwritf2rVess2+cv7hBKxHKIXFv2Ft0TNx6ONkUhUlXVJ4kaDv6AeKc7fRHmYLhNVcsQzMB+2QktBillKM9ew4WHirOlrD6cYHmwH0gU9z76OwsI05FqTw8keRbmF\\\\\\\/QIowlglq6Dhdzn2y1qcHXYWrDwqKmetLfKYRc25DxKMr\\\\\\\/PVxALUVP5jPxd9PLI\\\\\\\/50RurEfjg5JndqqYiw2MbnZCgwu1iReZ6L52lIkXhxeb9NDgI3sUrqG\\\\\\\/qDxZxxRBLY\\\\\\\/ae3oTFEwJ64vDZ+LFuyaZ4CSMx+2vJNJ0G\\\\\\\/zNbLPO6lsYJclQJdIEUENXtCLOkK+Abanh\\\\\\\/4lbT+BmkRPx8ZAsikdtnG++hibtAyHV0w1E8m4MPur2\\\\\\\/ZBkN1SaurMXgagfnBYIhvgleX\\\\\\\/oiXNr+ZurMdHHmBnSASFZSfNGdgGV42qf\\\\\\\/RHydiRFcJqRA+8iX\\\\\\\/lj9I6Cn\\\\\\\/c2ftWvw\\\\\\\/Q5U6YsFvKSIwOlNYhjO4dK+0ulFlSd+3euyr4B+\\\\\\\/y9xHuBUu1I+ZEgkmHf301uRVFONsc5uojDN1OcWf87Clax47PNgEghUphbe9MJfajre5GeemewvMOIhuM198wgnhXejiMDvAEuV3kwEnJYZh9a3O8s9oFP+ev366EGSzc7Qtm\\\\\\\/oRtc3gREPMAXLtX6m9GLwCxGwlsDWIS\\\\\\\/YhGiTvNCR8xpF+JoueGCUAX3Y7TnVSNhsM3pDbZar3xRcpC8LnfAzqHjw1adVF5SKJoGVophNlWUmAJKSmyqTpPswWwF1QOGqoKHNstkLjfowiIUKUceIzVp2ElBrWpNvndDD2njlh7WsxkYjXUNWTGxS5eRMRiZZa0X8Jxxziqs28KSCG4RFiBZgW\\\\\\\/cmaD6Skm4f5dTLH7o08Xza+jWs9EPItrawcwukgiyqzMOpxxW+CpyDfvI+wKLmfIeIEc4lthOkfxkVSYJIYmjfqHSzKOfsjS6ub3UPU18Xq0k12p28RqoET4Nn8UaLPXLhE+hjR4WYmMk81M0odk+w7JincHXS7askRlIBor0J+7D01UciN6yjRu5vymeK3aHVRlrB5qx15VKWCTvgTWosUQSGXiDJhioXzX2JSqCG3snNRah3IAQX3hhoNiMqiFmkZSn31EUqr4VNg4T7yauyXjGs9yUeTaUGTT1FChqryXBzviusprlq5BjKplNIy8a596piEcWCBfxORaPgUZ8Czo2zjvTLEmbo4f0ZjJAIqqoy\\\\\\\/2p\\\\\\\/eegy3fN13bIitLdhqcFS19eUHxU\\\\\\\/vgr7WqIHBkMDdCZRe2aZnDNahi5\\\\\\\/y2NIE8fV5EW6ZxOdRFHK\\\\\\\/YJPdpaTMaOARfSK1DtK5tPuHC3RQ2kLyz3yS\\\\\\\/Ek7uLHPILhIVO44V9TASXJ3NOjH7P\\\\\\\/4ztm8KEMHHTfjzEcb7pXfbjRb+LnS7Fo7z6Df9oEVbsGg+VTil5QR3oSK2YCY6bgvDH9nRP4aykF6zhk+WIa2JNBsucIfmIFSIwRLGBiM9Y5Xdfsj0v\\\\\\\/7vgOsZHzjzCsOxyo04CBC\\\\\\\/Q2yGmdwHGEK3GtPempYRyjLK8bvpOXxb0puBalxVlxpQk4YhHKK0J0p\\\\\\\/YBK78hWpJ+H1UauTda5SAMfWKMLU\\\\\\\/5XDJ3s8xGskktXrV5y8QAXA3DwwtQRDMDQeAFNYg6X01OzyKsP1JHMMdpairWUOCZ+Bd8EsKkr72ALPRthRa5rNAVg2GBVBn6cMd3\\\\\\\/sjNAdaxv8CTzjFLAPQLh8zflk6MPEDw2D7OdchoG2p1kMoGRtkOw4H4+ooyOOG2aHxYS180pcBbltXH1oaWGI+mM7gIHJYGuvThs5xPt8uGCNevWdxoEFpOwP1HPZ9ayE+0L2Af5pkWzE3erdnV+vn+\\\\\\\/UOAP3w0PsfhRMcFe72yhUD0+LPzLzzgo8BRaupW3E2t80dWSD7zGiLJu5ga8ie3al8xl9J9pef8DecRgYJzCTra\\\\\\\/nP1PY2iQvMwulnM1ennftLm48VWHo6DE6MpV1Sd5auJQs1EQ5gkN7O9j8TJy+DI7qNe0kIHilJCwkek80CfsSwbXXFBIpYrCwS8GPCOXIH7cEgOWhTtDIrEpzRbsL+1v6kZUJqVXLPA9BOCwgnvHwS60JvmgDrt+fFtnnMHrmmQYNf\\\\\\\/fmgR9aXMfV7ibGVeR7bNwrQflNvDOPxlorCDI94sCUfcQzSJXB1UPX5ce5V9gwWfBBP6C86radMv\\\\\\\/5Jd+K3rj6mXmtHhXwWbCL8\\\\\\\/+hV02PzZcblKui+3bARbTf3MLRglkApWLrmf582Ftcw5q4PGebHaIN09vZDe\\\\\\\/kCwzRflB6HogsNU8HQa8mvI07FHUYqQ7LvLb3xt7XqoWz4WdZUs2JdSFgu7DoqZKc0BZ0j9e+VftiJue\\\\\\\/Kd92uU\\\\\\\/K94WWhfepaPWg\\\\\\\/GP\\\\\\\/0ElAeCBO6EvIIQr1tW8yJ1T\\\\\\\/\\\\\\\/qCzRCSuUYSAhOtViIVQJhYxiWTaqBWJe0+DXm9mJMrp5B+Fm1loh2BQ3e5x8sl2Dp1bEWNh63YkiVk3CmNNhjoGDY5IXUWb2idEvw3V2T1CmDBrqcSMkZKOBowJ0I49u2ME4Y54EZmu0GUjCRpzpwpEo4kXJJNEaTPGJbogZgVZrurBfo8ipM66giXXtks88m7otsGPEZooxJ1nZvilKltDPySuLa0xrkWtQT\\\\\\\/63CfOlPNurIKxgg7hQTyC3aQ3ADSNDDPQIw2i2rG0HYD1zqrYFVN0KQLhNHyHY99LPx6mkuJoOkRlxOjanMO\\\\\\\/QR5GzyjmLNO3j\\\\\\\/77KKzT4a14vQ+DtTiR6Pxqr8FlsKBMfsrZKuoqot+SGou7WMEGg8CY\\\\\\\/s5jQ+qWbnvJuJR12iGlf6YbAStRas3MlXvkhrBGW+cqnjP7BfZQ1YOAUduytp\\\\\\\/XeEFvKNUXiX56WWlCdIMh2n1gy4s1Us0\\\\\\\/Nbpnynnp21PMy6LENRzm9HuFIYirYz099ApOchlGKahHEmWpqEPyIJx5ylM16pujnPpkgCvaEW9jefbcbMdXVqhEPUfQfY2K3DirH\\\\\\\/skyKUjC0EqHhfCuS07tBWfVdpKD6HJfxs5k5jKG3S\\\\\\\/ILC3o1kjICid++0BQJH8W4EDTTaxUk6Krsr3zVV12UU0dIMra+eDHCzzviaV46K8cDIPYFytzB3LAaOLwF9tlvvxti5Q3I\\\\\\\/+fox4qbpme5G4w1JYh5\\\\\\\/jFseFYdW5+gWh8lx6CU73igKBNVr9h6VbHWbY2ZMy3VPqeINqVLm\\\\\\\/iJX195uJgXbbJMTVi4xcy6HzKNFy8qG5EnERBT8i0Tz6x3PrDNHqXo\\\\\\\/Ys3pCmUQmNLn6MZHQLEPFnTvivSanGXHsmVu1uPCRyyXNHk8FnZcxKMEdENG0o1tRB\\\\\\\/e8\\\\\\\/rRbC6lefscMWY5xyaBgox8ZhnECg03CKivTf\\\\\\\/KU25CXx2O4RdkePuS+2sqWPV02RXkyLrFitbYTL3QBN5SwZ3MUIjp06aTg1y\\\\\\\/RYb6oSGUXUhNL9F+j9OPqBdzJx65Hg3G7gz6pc3S34oOGcH8FTN4aS42Vpjs9u5GBkn+LXyLpJVmvqpPgWMtEuvRqeC9eIUv1cNtg4pF5ELGUdBi\\\\\\\/XCR2wGNZ292VKDDm2SVtnnprNJqG9a\\\\\\\/bWN1tlsZhKGKN+1NHHhqJIYtKIuM9Ja4jXPeJIe9vvU+OVMoH4zIEQWUlRn49KhWL\\\\\\\/Qeslk6n89xzcuN9U2NnLBse7kvpgxx2KPix8knSZlfZMZQyyX+nOocaNS1gYVPMljctXGXBpQKklTfV4A\\\\\\\/LjMwn0qPcE5qOH5bZA0oKuaHw3tWtjlprILdj8KrcdktDNuvTeRIsGCLJOa7MOVYQT1uExFUTgSoNn3pnvcMR\\\\\\\/dCpbuLwtFlzNtS17iH4ITb\\\\\\\/ktL8VelTygti8s0fOmZIk96rrTvpUlET3+N795w8Opfe27K2QkUL5LU4DLI1+TdW2UiErFMMcIoyvM7lFJMQesDu2I9095UD43JewwmeYw7ISSMxA\\\\\\\/VjnVy1zmeFnOXwC3fJ9k3RQ8NqyEOCSKT1cOCdTZi\\\\\\\/Z5r332LSPrGC57FchguhcFDRq07at4b0z58HPnE8zW\\\\\\\/O759KIGeWYAuuwCD+r4AMc6BkSuPEwBpS8KYNFft1c5ky+h0eEYA38sw0AB8QcIYwn+PO\\\\\\\/R5VmZ+dRIxUvs\\\\\\\/9Uj\\\\\\\/+Np94dXiIElSkylLMJeKczPKFbV9r8I7MTmnuXezVZ+YDkBY5huUr1+0xCV657IPFeN+OWRVreSvnDOYUylvt6YV92yeH2qasqJ1U8r862VkJh23AILQMLmpssirUQxJ4A0ClTCB\\\\\\\/BJBNN1meLN7SDMAmftbw6M0UFUaiZxdEwJNWQ2gICMK\\\\\\\/cNbmWP8yFh2rHxSl1onLmnI8bIEW3TZu50p02kfl5rF\\\\\\\/F84aDZOAG0jlM0FGHoB6a9qnTm7\\\\\\\/SI2xuKJZgoZYEREFgbXohRRw3DDfxtuLb1NKHtETXMC8tsLPVHMbtcVsP3IOchwWobdyD\\\\\\\/Eg\\\\\\\/KmkH5iAVNG\\\\\\\/y78T3khJ3Wa+gmECIpLBgfAgSHqmQjMs\\\\\\\/gUQxdQFb6Y2jlLIrgpaKVC32S79eDaCfiDam9pj86kHZA+o10G\\\\\\\/7HR+7Ohgo6mmwTRaIKq93VziSGobPNeE3kJ9Dckxi8IMI8oEjg73H43UBHg7pnHWGCt7vKgY7l9fgHgAk1J12quCCC8sGWiwZFL+Uhs2tk4HA3vwKzSx3C6lk8\\\\\\\/tbUc7NS7eXJodTh1ZD30KBwU3TKwdAqY9Ljpnz0bnh0c06zbd+xxbBeWyOacpOBHkkiJT2nLtacac6B8JDodmfJ\\\\\\\/OwFh4LcMidG3a7CmNBxBofeeLNzhWyRGacQ4UXXHNfb8MUXKVsjf\\\\\\\/WDGLuNovgglzPwfX0cjRG5Kvo+EFJr7e7t3KAUe+B4lhYRdaY+lLRShf7npHSBuij48JwEQb5jtQmW6ktfR3kAKazrZxeKD\\\\\\\/cQbg29VptEaQZ+APBREKN3dAp0DcTcGK2EuvA7g7OLR8wnh1mcaRJ9sYwDKTQQjID\\\\\\\/zTm8rR79RpKcNphVddLPNk5OcQjGsq7yQS+GUJCTYIOaKaj4Ejy2Gr8lHLn4HweDGnw40MnVIJrWyiHvKDxqAmeuHyLtr67r4htnYfx\\\\\\\/Keobuog7W5yMWnD8IAtgNCK5wJYh0zsRAzslSfoTdA9OLppnHTCfDqM7zZXy\\\\\\\/yjC7Q+rPDrOkTwJQxxJY4PXmkZsgRRL7V5MUGRuuu70Q5Y2040fD7\\\\\\\/S5CODuzXBDRDlfgqmaGtCBma7jJHyVqBZUVqW8qcVnSWFOQweghXzA31qmGCGeaIkD\\\\\\\/xfE+Lq19XjuJFQDOuthzEZAbjpTf+5q+rSp9yMVYE3kyE1KI3SnDpSoCigeiQkCFDzSQtWMVfPTyy07SKqYmLjxLNaUkBG0Cp3tm7RRxq3d32piXqeCm5r0HT\\\\\\\/GRCkytTYOyG\\\\\\\/+6SevGkq6wJ59rXSP4fHsIg0Ww12E7dGvsqLwfb2HI6TpwaoDvqeX1cydVkJPdWVRZAg7j1TasBSNNCtLL7p7EYaKuMORF16RefG21irJn7P8HEvuPrdm7KDwrMUGLJqpI4lGyWF5REH8c4RTbi\\\\\\\/mFF6gotnGvMffUEtAJo2HlynP1IOfuIdHJKORWGPTNy1BWgr18sCKaSFntLUbNlTNkXZjyhVcTLZ9zKd\\\\\\\/DUzI+CAIP+IDNk+G+5PUIono+9uXV36jrwGnuV\\\\\\\/v7qc\\\\\\\/j42QEuMXoNmklwVRpimGf+JGMZc2FOgrtq51ftpXaqCNDisqVREkwDVM0OC3YcpePN6G2fCeycmoS\\\\\\\/LZnbxo\\\\\\\/hu3OwSb\\\\\\\/nNSGEXNdC3fGXmh\\\\\\\/4HcWPxhKA0L7Xkz5AdTxQulgHdC4HvuFjy6vyDNt\\\\\\\/WaO2ENrIS\\\\\\\/7OhMAw7esflKJhOBWvorpsqvyGPHSw0c+8Z9d0mOvaozc0GYqmhv5mw53Z36SW5XLea9i5PVtVzdWbu7DsuzG1mflVMXEVz6inVw4p6GIedc0sT+temCA+3ql3s65BnsDbTICOefEgjUGD6lxdDOmZuHn7EGZwUagsYLbMtlodBqoJzPpHyrkJVf5vlAma3eQ1WvB7nUp7b98HLUb66JnHwFtoutPmNgm\\\\\\\/tjRDFl0Wmt1pmoqBkfAtdmOLDZGyTeNAbBpmfd54JwRmJ5mC\\\\\\\/SiuTECm8sVKBZQEhj1dJ0lkxmjcnPWYtWiKLgvIYF21O6xW+vfgbc+d\\\\\\\/7DyS6mZ0S2qFEvlQifTCLf6SYn2k0oHzU0\\\\\\\/eTPThXmlls3uOlnbvZPZz71TC5WOGMTyDvlscZTf9NBroYWyW8d5tRnLEWwSquFPmPvR4ROezPzSMhhVRf6NrxQ1OjNb8CwXeyczi95xZ0yR+3kNDo7eneaw1aPUnIUAhbDz3Vut62G3rHJA0V8fIxHyrtnMVn4NkKbTY7j71eEkbPFOmADlASIdx04LYebAoKhhDbF4lb4eN\\\\\\\/q8U4RJfxb0Qz5rJ6GOsdGbR1j4S+B6jcnAuzaWnbItZMvpxqN5xYaupCc5ZCkFckjfkK5RJDbe4ys5cEOLRWCBeOa384v5BBdFRXTto3vzJCHqFatqjLmwf0VhhtDKO3InWniCbcDBvCO5XGEWwOHuXQc43kUVrLQ0pvIvxRPX7+nqTxAMZZ6Megq51HG57+VxjV7YDLlVXMWFwhuP0MTqTEttt7DTxt2SGxNuBp7OdtXP+7nsf\\\\\\\/TOraQIzkbS\\\\\\\/NKJAerJXcPLh\\\\\\\/0YtOfVdQ2+dQ1tjDY80+L+HIz11aMKvyYvbA46saG96gfLr177dIGxZStrg3sBHBzG7ZIog+9n6eJMab1HJjd6fZEXuJc+0mQ8DHtQ0LQN5NKV06IrJzAaIY3sXGxY56Zgiwzg0NZVs2DU1e323f\\\\\\\/DXgoytWE8iClsBP06HkLfrkB6XAI3ghXJRSC8thHCcrepH6A222SGjphOSBYOju0RX\\\\\\\/I70JJrGE1C4i29nD0SEh3kEBSaWf32o2Rg7niE3U68ghyqK6+vjhTbmhiR\\\\\\\/bJQsevhab0nSXuByYXGogdZkbG+chOGu0NYfkSpu7UDQuXKNTs3FaZqUrui6ZizWLOvQn4vRzO0Aj7+jRz9EFYAAyl6pX6JUYNV9IBYWadruM986iMbSSXzliiLi+yA4h7Me6xTKl2+cz3chx\\\\\\\/Lqzskt7OuqvJWIbp\\\\\\\/gOsYIzivW3VnqHqC0oALVqOSsVYk1zGvhIxQp+k\\\\\\\/JK4MdBd08vGt8VNBOQCoxw6axU1zhI4LBT4eW4fk3l+TUnkxrwNiykiR0HvcA+LUQfKl1NKF4Jo81J3aKp2Q07zX\\\\\\\/UVyvn9MfPuSspKRGO3vyy9\\\\\\\/E2zGINrpqyWpJqjpUGepxpxv+d5ZLhBWm1an3lHpD+Us6f2ALiLJq7TwnPbjwoX5hqZ8Mk3epycpX2hremhovYWGZ3BmzMWjEvysv\\\\\\\/Gh8MNHxRpKq3YfsfN66\\\\\\\/vAvJE3\\\\\\\/4q0YzaPfS4K5UBHjYs+4lLSUICPDn8pPGX+uM4iH41S81U4RLb+l3gfxYda+MrCaoQp9+QxtTi0YshCKeElmOc21+1G21Tf8+mDuZff3VjPkfxoD9TPexdycdChryvD\\\\\\\/ug97uqEd0FyydW39Diw6vnNC1ZpMlFXTxEd0h3OeFYM7TXliGcxULoFrlHQsqEi+ExfihRIHvggvPLNDFgWdZdb4Qa8t4UT3YchOE8uxE+0Pm9ZmqmwiGbHqn0vPQUTqpuU89HrxHFdIVM3f7\\\\\\\/Unyd7KHVd74RQ8CI9EB9TIPzE+hy4SGfgE5O0bGxBYbkScHmRXjKz9mxXtLZ8\\\\\\\/DcEaCb7nIchp9CUpVDpDqxfkuuf+hy6BsM\\\\\\\/B\\\\\\\/61Gin2F\\\\\\\/\\\\\\\/sNESvTlXYIUMDrztJSW8m87Z2FYqUf5pT2awiR6sjMOb0uWEDS48vSE02M\\\\\\\/wdNP0luCDz\\\\\\\/CD3QewLHbrh7OlN6RIpVk+StDA0OdxwiPrLFIbcffDiPYltuatFaOrWZMON25AFVAPmzOiPpAtYfpwR46lR7oHuaHeQr2GcGU7Wa8a6kVrtjAraryQgzMNd6jELwi58WDgaU6U+7EG\\\\\\\/mXbZB\\\\\\\/o5X2Pkgjyy7T1km40EYnyIRmwH60PW28CoQq5KjgDSuc\\\\\\\/LkbFWAWr\\\\\\\/4NB5xUAFeeKkhStGFZPR308jkgdbiFvXrXkqGRsLsfgbMI45jGg0wcyFPHgLO9MmIe72yjwI3Eu0JGJxf6QMzO7BJu9NaYcGH3t0RMA9b15mj9n9gTeY2Huo1iUw+nIBMkawG6joiubjzQEgfoe6bwxVvyipXj\\\\\\\/jlVmPnS5ymGKz5XTGHB5ORPK7yAh9Cr5ZsoccnrtVG5S4IrL8lwoOscHx3M5uKwqGtPMQ057xdy2dOyzLRBlDB4t8kcLzgbb4etCDqk65+2+Iwn4V8rXjvI+4bVC8kJsixSnctUy8Tcuxto7Q4etFUKigt8lDBCsJW1\\\\\\\/3AX\\\\\\\/ZvjsdWZoN8\\\\\\\/Pf8Z4BpMEopS9qLJxLt57MDEljGkU4zLFZRPfakmxFWMRpDx\\\\\\\/c4zxIBHsjeFAC7oAq9GF+qo6lwXYovIoMQqsuVo6K5Pcjkt8BEE2akmk4PF8\\\\\\\/tjeFLqSpJs7XzVZsmGruuD4Kkjn1RkccswiIctihoJxFzKJdyT1lS7EIgkwbvfYzPrl\\\\\\\/nikLhpgpqT2NBpMveFNGFTtc5Jn+QtTT2R7bMu82EWwH9sbh6613QNk01UVNof2MKvYYaG\\\\\\\/e39BSpw5Yr0PSyPFePB2QNyajD5vyMgxx7tc1oxEvLPxdDjz7OG5F1UchMt7uQ2NnJS9TIE4AIuQuHWb0I7LSqs5wa9bckHUfcCkCYR3tBYaS70UJN5zbzZSdrlNs5REkpGOpr1Zs3A+BgEZXSg0YrULrWqWUTtLCWQB3TbhlCihjqz+2K71dZJsYbie8MR0oSfO0CqtWWyAm9elr+dXGMcT5eCYP7ZCpp9L7UAg2wFe+2TUKjpPwZHgtjTmxNclOGuGm1rwtZfK8J0C5M5v2O1g+br8OesO3S2vHe3CrtcUwaq+2HGlIwqLjLM4FEIo8RQinLclHO5K28Zd3y58LGV+mans7WUXjEwoIAMwlQ1uqZmoW4VNKOEbUXOLeXe5OTSnUkYFCb5E7ZPKvjmBHdOx41xzfJGaPV9cxNMLqDHhv2Mqv95qhFSKppA9WTw7VdOfoquTnDqvfZxVwFb9pYzhv3bul665AwXWzEoSGSKYg3Txx8JM\\\\\\\/botHMwWhDL89pG3\\\\\\\/VKDT8V4W1D1exrXVcKPtK0e1ixTS27vwCaswMlfC\\\\\\\/V64cDQWw1QsEnUicmzTLnowM5ZPQq3Sks\\\\\\\/ih223haGyUbycVTg7QiohPTxcwPaOzTPfZ5B++8ppEl2Cp5VSktH58Wor7bu+d7KtbVt\\\\\\\/todQ3zYGfOCvzE8qHTL7gR0OVTIenAO8pSxH\\\\\\\/xsRuszf178NfFF2CRcpJwmeMMVvGg\\\\\\\/+sHyrqJwBFJDIxBejdcCTsiaMS0jFSFweK\\\\\\\/IDHkIEFRsnSU15yoUYyp4plIi0q0X+uAWCdE9MIAjC+AHpcqmZaLN2bgQxMbKYBKMRkbq2d948CC5Lc1QJcroVm3IFt0ZS9cQ6RrM0jNiHvgSlGAmedrAkDz468cHiguq8L7ktbhDlTTDZWJ14kTJMPVWQATpGB\\\\\\\/NKLHR3pLeDUlR4O0cQbXNFhTKWjgeVcCySY2RF4qMRtKELXUDdTLmg3ZukTdhupazv\\\\\\\/KaZoYGpfQJEF6gzORWlvnGbxDC6MWgOxpYV77FNQt1ESvZMvWrp+QaxX\\\\\\\/eTS1X55SeMoRy+R3An+o7WBnv2XkWXSxNE7\\\\\\\/wyu1qY7fmxK3EqnIryuTn6zxjCHsGUzBPdURSJLZqBC72BzZC0xutjxs510T0oqTKgJ8KKKybdaN4T5v4SsoGHp10OWm4+JBmNPtyInSDgosej\\\\\\\/gsowouGz3z\\\\\\\/1vkd7EYWgg1uR0xcz0Hms80oByavDV8+kk904a7WdD5tO6eFl9xXVCONX1BtkVaCiaOWvQt30OVtU3W+yqHFZ\\\\\\\/5iITN1uDod2EEzNndxHq641z\\\\\\\/djmfeAVEeiRCNSUVxgYLfIOsWOGBjAEPP4FTAkSXNocS00NTx3CrGC+ez7AIwPmyQL0F7cZcLkU0TESicBQVC4tNudqBggxRS0g4EJB75h0RKPp+Xi7buTKL4FE\\\\\\\/oMTbhREQdbvVlKkgQTd4zQeuOV77q5Wv1kHK5zALuAJAc2Wjxw8WMLxCU3UqatG1jAcjOcRjO7UxNK0DnC0gdrSgGxM4lQBKH1UIWoKrM+qmqp24ZGuh92hn0JxM2\\\\\\\/rOlOKhElZql4FQvJ7fA5b6bG9Rhi1t8JjOFXcLPFAOfoKxRfjz9qcyxvwl5k1pBa+E2hdPhjt52CELc7GMSrFQZf2b7s8kCElUQ88JKvkdc2B79P3qaFwKO9oCIB2sy1B6QGgeD984FEadLQcatg8M+2YmCDk\\\\\\\/7SlzPu0vc3RnPL9c+4WHkNhckdEWqTu5R3lH5zv7ec510TlCqS+taP9dg7ajs0Fn+TTkrtu65xmPtXYnoDw9r4Pca2gXVkij8XP+MCGnhM+gUN2ZVB1w\\\\\\\/CGJkYX7knTd1bng4gDgMBEw+Typza\\\\\\\/f80s5HLWiTZEcnwKM0bfwQfA6srJUjlX\\\\\\\/sjET2z74vU4SJIQoFjeF\\\\\\\/Ji9abzcGV+AR9fvfcRXjDqAAiDaKYVSkxU6tj4+lP1cMX4m0VCtp5kujKERHUtAl\\\\\\\/hDPvkv8fgh30nNhoS2c0Fsv9+sC6yGKN2l+npxKwPMEgdHEvRfJG2c\\\\\\\/WpNexXAq3teBgIh6o9Ycpi1QuE85xiKyOaLevwZ3oQlopLzK+K3BMqP346ROUPngWtZeeW0VX8V02y\\\\\\\/of7\\\\\\\/B1INYrj0x2M6OcNaDiNP\\\\\\\/9viKUkSTFPFcnr2PvPf4+eTNG0va3DUfwP7cAvCMzlOmUDW5Y00E28V\\\\\\\/B4l2VwDFZZVoLJr7007cYIhIpYxy52n1+PDHhB0O5uD62xOoZPK4O8f1sNBEE0c0\\\\\\\/5fPTADYXa\\\\\\\/mkLlX0vbK2VEXL\\\\\\\/m3L4dwVapsgjyG0L0JGv45bp6tkX4+YyU5b9L9Hjy4gYHbLIlj3PrR6qoGidExCoWPM8XGsfD3QNEt\\\\\\\/DXLGHwXV6Lj\\\\\\\/CWMDQTMMVYRvXFoS2I6lGf1JiewNUO7XZ6NEfKx2HbSrzrLHWSdjDyeURFlVLSDZSnuvvga9mM\\\\\\\/F3mItvY6kFX3gNxNFkdlw2zInWFjXw+nnE77ebBP2moFjfxinHLmcO9HSG34iFgsHuxNNugosPBMAE7TRT0zzh4pv88s5h1HpDm+A2ZJA4sbz\\\\\\\/WJ2ANdO\\\\\\\/Ce8nC\\\\\\\/1HJuuHu+BwawsXVm5mrMEFU2SSNgqcxzG8Y2A8AlaBJnWvbN7HzNK5bd29FU6FnoFMQ53rG1h72gUg1e2QcKULgCxlLOuhLaOZr8S\\\\\\\/uIvgfhfmFfCjxx1BaCW0PxQLxe7W\\\\\\\/W5PPUsKroKUuvanCs3SUMUS4oB3PCgEv\\\\\\\/eXODTFb1TdyLQ0Tb3+P5moQLBWh0PodK4BjGoItyPJeo0hNXViMUhSaT9mRlCN7swYp7vtv01XkcyjSHCFlSJFWnd2GsnWPW5yqK3ac2HJpM7B\\\\\\\/oGh9syEzGDnZF5pCRI26UVx7uMxRb\\\\\\\/ncoBztVSP9l5LrTy1fB2ZHST6q0f5EwlD19OL5fY+1j+cca2gAu03Ppu8CZK0oFERPn++0M7RwneEseq3tpOUPPEDupLWSNPSJaNz3t5AkyHzNLikjM\\\\\\\/vPPY2NR7dGKYTrc48OD7+aH+EAs\\\\\\\/a7NvD9en3OZVcaqmRbAav64CT0OondD0AXWCfsoPkaJEnEjZ4hOb663lUYAQWaoGoJREPWa3BIQQtsyj+2LsgFy8R1HCRmulBtpy6FsvUOMchhbvAKLS4On5JppQi6uuRjFsmXErkxcb1MuJQhWA7y3BlPJXSHdSiLtSSdvByriHzEV3hH+a0QF6wNJJSxqbSn64xBFev0AGlDhiGjirnLMOjefSZH33wbWlMij+bbTIYNyOaay1lW8bvoSJh86tjNTUUIaCW7y8TYfTPfLPU4S0Hj8zY3pr2aHfwhKqM\\\\\\\/DVEM589gb24YJEsNYqi5LWy1a7BnyEk6TT5\\\\\\\/4q1mpg\\\\\\\/daFhnlGWlhek0eenXGSX3bMtiVqBMU7C0HqxONbu6aV2Bm5Ocnskvn7iYoo3Xp9YxrfhKPH30Mrh\\\\\\\/BF4mfO5zekyvuxqOoVUMh0kt3Qno6hd2BPMLY86vWbiTDHwR7qDl\\\\\\\/L4igo3kFALXu29ubnT6MB1hdeCDGAy5Mzd2lgOXxI+QPXqFaRoRk3j\\\\\\\/52kLFHGvIVS\\\\\\\/u9rkNorldbMxBxafr9oAVba\\\\\\\/RJwUTxVVVPTHrfoky\\\\\\\/Ozl1hhSCiHk68CfADg6bMMxIIrSMLyUjBrKhKntI5dQKf2MaKlShSRHBwESIB226ZMhCpmVZb37G4aeMjD+wdE1VVklRTCRz5iG6gSZLPLLe\\\\\\\/pw05ndEQA57RYXUIVD+bEc1TA0TlvoATENO1gJvb\\\\\\\/2ncimU+3+v49X5OrLt70EBNdFus3h9\\\\\\\/Z10ych7i9lwaS9zBJFMpZ8lPo1d11DKgvN2+ga97J4BCk+1BvjRIZ+HJ6fE1hXSkP7GMy7\\\\\\\/gwvhndIBKSmdtu1T1KDqF7oD2RlaGFKaWhpWmIgM+AkWY6GUIptUjx50U9+2Xr6En+Zzw\\\\\\\/hl8RbcXBYCZCf3ISSH8eLf2Z2sgadkHhtpsPdgkGqxYb7QDHWFh+us5HwQ1wkq6dBTBVwHvKM5V2EyIBkB\\\\\\\/28RIjaPrGpD6cuuy0sIWVKaBPK3a0wcUa8rmOo1gi\\\\\\\/CeKyLpuhcGUDrOTAwNwcbXR5LDz968APeawqJNwFfCV+z\\\\\\\/QTfNN5jwvdo4eU8Krtnghtsz09HVuMEsdurP9kcLxpLbj82vimOhwRBo4tQuROOUhCV6KcwuOLyukSfWFGmujJdX1Ev0dEzDZDdNhzqGh7Uk2tuZBJMwhBS7mFB9g5VFGyEtqjtBY6NLxFIMI0vvO7NVqIfVI+Rl3WLc2mEZQjObSow6T\\\\\\\/PBQTPRpiVd7N7CL5D+WJiXK20GGIb\\\\\\\/SJHVi\\\\\\\/qVLTrt28kLtjo99D+VDiZEDCv8\\\\\\\/K7G8LndC2G74ibFnRvAyx8jB0bdjxJbd0XCk0bsluIpIGafaJWwPgT7M6OqswO3pnTjNuNiv\\\\\\\/3+J1RbQMvpM992KfcdxjjAKOKGgQRqSxUCSsuuCzuYbeICMQmw\\\\\\\/wFtKSX9+cSsWZ3r5H0nfNkyiN5UdyN\\\\\\\/99xIf4RRP3AXm0zCilWS4BvLZzgjCxWvHNryF6rNiRxKrX63klXqNhPXIruKJVF7T9Zlyb4LgTe4iuNtmIqcyUMTRVf2xp2e0mflF6XttZuGfGwM3kXPIS\\\\\\\/qepcdWii\\\\\\\/hHvHHdrYUVxycT+EAgWXVoYWI5N5XoqmPZgnBon6Z7u8geqvJSB9poeiGXYZc3ot8G5E8bY0LLlbDDg8Bxe2ddZgcAJUfJne5sFuChHaDHmyhz+dOEk5iJYtOBq3xfufgSeXoeyoHnDBWfMNrARaYensWlJU6ALErZKqmYznEwirARXnvKI0fIcQkqV9V+obhvUtKsZkCRbOyYR+iL\\\\\\\/sECJudn0I\\\\\\\/yL6uUCJmtpYCCQScBm\\\\\\\/Rw4DgDtCXP7OKDlNxB\\\\\\\/Yl3rc+akWJZtNYxtKpGl1vVMBXAwrSa9Y6FXUtErhxOCLMcx7uH6wLtBdkFFprbIdh58zTNKOtlDWb+kn\\\\\\\/+RiJMbCU21N78ojuwTLl+lg\\\\\\\/QBItsHPlTBsLYxV82lN6ZCfR9W0fI3JT+2NyKuwoVU+q9c+fgYHVNF8EIFjSsTJwU8yUEI1xx4Q21MTtXFSipiywx26+P2quKv1yIQQiX86DRIazgjTuT62KlADa2R5SN8aOw\\\\\\\/u5oi0M6uQsTTnaAhyGhXsteJOR8bQIw6VEYdvNQ14L6wCpTGTt2vHsas9IHXfUW6qpjg2\\\\\\\/OjcERDHHS3D41aSoEnmMynXd7T\\\\\\\/DMs17NM8jgqDjR1UJTb0s+E4VEp\\\\\\\/T38NlmUsruB+2WY3Rm9FKMPbqiTYIk5G5mdssuGS\\\\\\\/CnEb+f1Ng7ptPVMiiqGUM0Dqx8hX2sLLGJXNZuUdoxmWsjF6cYSfHqpW8Agj3mVinFxwBSC+avHP2KcDmB\\\\\\\/XZpFC5jLr0iOycKExkAoP640EkpTN\\\\\\\/lVSPEcX1a3xP4hy9\\\\\\\/GiZ73K6Cy9Y+deauzOGDk\\\\\\\/3xtIrAH6BJwAqWWAGbOTbbfyqgjjqoyJua4Mh9MaCiS3DXJs6hsaByN8KOLGI7J+ITZ4sxE6UEeWqVh9UdfIclhdFWkguZ1jXXKxB6XwRb5G1TcMcNvipEUfWBbCVRN1eIE4xvVRcRX0E9DCrCWGRkGBIw1\\\\\\\/W0EPLhuElUDYUYkk4VcG\\\\\\\/EB9n7ocxQIDizllyIMDHwU9jebTwXzpLAYmtrAR4bzzcVGt5QwPD7Z0zyRl3L7rqWp\\\\\\\/qzsazFCPqun7ufUs2G+WWhFYfdi31QKkZmrFdm1HpC79sXEetjMnQ3Xo9YopKFTULFN1PLJqsFj\\\\\\\/QCupgQV7wuaJcKmD0ywK9HUjt40aFZq\\\\\\\/8BCAqmmggUQ6ujDZmX6K+XE7Ne+481DbA6vFpTS2DRljA2DUA62+lbnlwNgLCkSvH6+kE4D2ZE3ruJgTRcPkrTHjfx4nSMETCqRMze+JVTaVs6N73KC+2dInNEpUe0HXYWcQJ6g5muEvK8ATRyuklcnaRHJ8YwqXh+lA2dPyPm8C+jgZna80CkMr5\\\\\\\/NvqTT54BvkSgun2\\\\\\\/xlQm52+W5ZuI1Iq8Y31P7DSyAHZ85pXLDwvQADgdAZgW\\\\\\\/D5bQS1o4Fg8A8fsK1\\\\\\\/hGZdQrqgDbwohkZjxIre9HyizdmFsiaeQ4ncdx7RFxnt8pwKkvQlvylSaQUL3dKYEgeSsvh+ddEx8MG4UuhEFvLaL6CnmbINavgaClUDuBOLgiViCKE9CjZ9zty2Wvj5URaTSOo2DfwN5qx2jSeiGx2SKL724EbJVkhkyJsi064HjPZlfkEScZ5W2J6bYFWqtzGxE2KUevqbMnxM90HWflXqiDFyPR4dXKQcJnpkE\\\\\\\/1WnTAkhUBaYVAu\\\\\\\/x7gnPYYoU\\\\\\\/CfPhYx5evIT6ltoko4VvTfJDUwUbYfsP0yOp9+WP7+b4BTuCQoEkouv9VJnbvzAAOF7rd3MJxDvA2hVrHTPNdN7bFrytfsVQI8EIvV3WXZWH50nolGL1gvBDEdhqhohSlLIpJgHVA2Z+IQaDFj2qK8DK4+nlW705UKdISXKvkty+3eb9730EJxTMUniPuoY\\\\\\\/t20LOFXSJ53nvy0Zgt6mNwK7Ldh7FsevbFlOtqf2kopp9kpUFEV8GDNa8ooWn42Wn5UNuy9cfaN9UMlD01eezagOSsoARDWs7kvhtTYvMd9dc7k0YWKRWq\\\\\\\/8tPh3XO8YQ5Ak1tsQSOoK4VRF3Vi\\\\\\\/gkG88sG42XFSgKf6UQB940dkk954Mn5m5KuUnTBYfZFWY11\\\\\\\/907uKFM5nqT91rUHJHtMWoi+HTe+FWlRcb6xdgpLpRO+R0tDz57zFec0DYcZrqF8AW78sH\\\\\\\/3m\\\\\\\/2iYgs8YOncC446sszwPAHH6z68FrAbaZNhrrwlsBNklWFC84Ps6UC35x90XXqokIle53cpI3Mu+nSg+\\\\\\\/nA2uc+LuRc5\\\\\\\/Vz22uozJzMlk267LhYPiAtn83e2WQp6m+McNGGlvRdA4\\\\\\\/i2czykfIAvFu8Swuf4ZNQwLGDQzAQ78+dvrNckOpCuXoSMG5Ju9YHWoeZt6j58p6e1qmJxGEmSrjRt77c7SOYKxKQk+pJWIt0CbFxkLlcJjJxFKRwKSnKPp+PD6fujnaiyDavnU8jqcHReA6WCHQ5bJfBBkrYyz7ZryZ6RKF1lBAV4BZ3FxQJ7CV84TTmSaGp+EVi14boDwirBJ9rVY5O6Ez8CfyUuBMk2\\\\\\\/bjNwLaqJ0HGSaOuGBBduwi8DaP67r0P7HkE+nF2k6IO1\\\\\\\/3cs2EwQQCDaJIruoLqMo+Hf28o4ab5sVHJqAfFrBolw7rVSmmTHcJIUpxGiVJTL74o6Y4DwrdNKEFhBK4P3suDFih44lqRuUip0aMl1\\\\\\\/NgCpvWJnw\\\\\\\/ZoIySSMqYhA\\\\\\\/Qp7slmnp\\\\\\\/EHd6QamebTYh\\\\\\\/mICSVoEFZB90C07d2S9HxEBA3mfMIInGugJPjce2Ta3HZa3E2BeMCJcrRStd5bNvFZ0cINt9ooKO+AXNShW9RfOnW1Trksv\\\\\\\/NpuRUnGUvrupWf0W1QOTFeM40ewa3MSY1fhOJXhKyNrw2NywAZJCC1Wv4parWkZgc04HDiPoGLj5IiLZqgjl\\\\\\\/ojqc9thmiloQmBD89\\\\\\\/1vwrjib5KgOSdc08eZdmwoYPVedpZUMzqsK\\\\\\\/EF509T7NjR5oh2IXN8GJLM5L0ItoKd+LX\\\\\\\/fWpY9emDPI4O\\\\\\\/2rnTeIdeG3nq85IIPgE9xh0U65uEgtPT61Fr1aZ3rcX0xglVyuyv5v+wUeTCqWIZUN7Pchass1mmdf5oAl3MbmawjPfLhP07\\\\\\\/L8KzpbeS\\\\\\\/vhWU55oVQCwQoJOwX+QquU0n\\\\\\\/ld5xJbcYzqTmy1cR8KuyWO0GnRNsCJKWSzuiMhQEhD17RQr3N+wMvW30jb6+H94a5Tz2j0ZYfyhs9HQ2YgbDzHzD64Y+emp0WAgMsrdaMmCb0\\\\\\\/IZ5z9Re2bpGILntuErhSiy+JaQH4FlP\\\\\\\/yWMMc4PWe2Gh+nALab580Rv107nckjjIV+HPtaCD8NN3cdwuckcA+p7wVMwVy7sxLmE0u7Yt1XcQKHJxajQKD\\\\\\\/f8tI+RlPwP1G4HwUMWpkTfhu4zDYVgSI8bgQ+w5J\\\\\\\/T8dplRuGphxwJkKPVdYSdKfqWn9DCF+V1DiFv6fjXV50xj9qDEWw8hSsa5sgRVQQDVZiMokLI2mAKUGYA8rciTavnYugw5kYsRx054IdH0l2C3wf7uN31HeLtu8b1BlF98FubXQqJH12avntEyENaBqHVFjgu7CVhPClLI5pfGjVClZv02otFILj1OyPyEV3oq8NhUdRoXlQ6xJRHrBHIdbB6\\\\\\\/x7HTTGAra9O+gE8J1BZB7V1UBnH0tVaHGe+JnWTVD712dKdF\\\\\\\/35jlsxt2PP7t49+R5c6F\\\\\\\/CAsP8sWdm\\\\\\\/tu42wGikXSyGxX\\\\\\\/w0GpjqdLmGm07Xvt9bOLX2364eOP4GYLzPNtxUq2facaRfzVjGr13XGGFd2yBfto8VCnE1Pdt17Ub8fwI23wrq3zggXF+4szFI57b4NsWk1FXT0gyyp3rB1gM9\\\\\\\/orFHXSzKgxVR8Ag\\\\\\\/yOlK9oUo6uHnCFMBhH3W5KwVG\\\\\\\/Kl4B1OMHyvrTB4XcPuj0UXsIh52jkv1w9BpEPJANgZXgC9dDYEQ4a4b1xoJ8XoQHQ3sn9pYtzH+EBXTsnBuvfTQ\\\\\\\/4LKr+27VmXd\\\\\\\/rgkRh1et\\\\\\\/LsI6FFf415rHUHdcB\\\\\\\/rmEXYoiclEJi76fIfah3iSrJAkO7Z7RP7fEYmWJKlGg3iK\\\\\\\/iE+s6AVJITJw8PZxAKSeGR8b0DevyjvC0pi4QhT6IfHgE9y1\\\\\\\/QmAhaLMIortqOed5FrEfQnlyD6xnt4cVxnbSx4iUI+CpUUTGjdG7JQ1TDcyakZ3Wss3w83KGnhC6iqfxW1f+E4vk7i+xlnhN85kyQ6MrXLahGsfORJAT4Ii1RKZASmmbc2JyMKOyLeVaqPfkIOXZufTTjr8NjUvsdLi2vtDICZG8WDNxUMcrIeGPCOW99MHrQbJnYbRv+HY0lBBTWosqgnChxXuaXJD55J9n2wh0PdHfyM5djPi6WI\\\\\\\/wzJV1ntF2Ob3HRAUyOcyKwRczzdXjS408SL0\\\\\\\/53DpmTZ4Gyym3kc0R0z2mmG6jaDcsoHEVEKyzs\\\\\\\/8nw42pGHw3spF7+ZDzuJ09Vw6TgWsFaRzK7+1\\\\\\\/g\\\\\\\/cYIzIB2MU3tcOnl8knDK3lV4tB0MvLe+K0cuXsqOQDA3V4fDvzibEFEAW07+cEnBNZn38RB1A3LpxNTDYY83gy6uXO2U8PTCjKlb63jbCfLpQFMFP331NXWi+WrbQ51EB37GruBYEQ8p6imLaH5Z+jTlsA4Eh1+oEf4ehX8k7pR51ZTzKC+FjMlgoD469L74KCdaBjqOV4r+Af1IEKq7GsSScz5+HWpotELFqSKepp5dbR5uutgF7KysRVoSZnbv2aXz8FOtn7YSRY7EpGuZzMxHn92uZiZpXvxSD9Vh27qaVSa8xu\\\\\\\/JcHKV2XKIsvVEhBvc1xc+H2CEUcchQVWCb54nXgFBr1IbkE0GTSXdP6UeGbyRfilB3g3AhKHcEd5IA6ICzgxz3cHVLnvv0QWOsUAYZnb1ph3f\\\\\\\/L6di+Y2af8faTa8JHaEjyat0YaMr\\\\\\\/ZihULVeXWHyo+lNIdbykO5xaankISKcH1Z+FOGjJxspl8cRVQtDXtAXidVy45qOxghFA86ywxbLVAryrIQ4JrFhxxCumD0qMIVa6eQYg\\\\\\\/Fqiad+98wZFk2Blyn9EXg3QFAAK7bSIkUSI46HKNNldACLWDDZQ3mMCcu26YjVuYjEazKAzdpuQGQDDXg4ZkreEw8ji3R7+dK+RzXYXHpYoiJK3atJJf+8Coumf7ftgnf1LuMI3ELw5XVZZj6uz+xooP5uwG0MwG65WQaDqJN53WpOwf4iMMKAGtg7ecPH52rfJmoVSca6YfR6U8OK4NDFEt5EIvd3zUMVJuuQ9nT\\\\\\\/OldhV\\\\\\\/53jtB7qe+6GVWSS16kkcooBytZSQi0oQc5r9T\\\\\\\/hFBXDun5ugBbdDcn1BpTHCqtUnqjIJ+jxX6sSzHUTEk5FTrYc9m2lq66ScZQlWNcJSTwsoKqWJup2aSLIPe9UCRQkiMpywXGKB7yXFufqKyg8HNhIkW5YSyrzq7vcXum+cT\\\\\\\/LYg+qnedOV4ZVel\\\\\\\/KXN3BlEsH7LdKpXyvFJytuf6xv2EsZ8mfXIzxKfYc\\\\\\\/XDu6jCTHvzUCweJ2abeqbgkmnru7oIrTG2ud33ToF9rMmzjfb4edDXQJ+wO6yXy3KfxLXFeqysnBOQTzNg\\\\\\\/eGWmpkqyuezTTU\\\\\\\/C3wDuj+n1kwvZvRTUl1bbWoNCdk2\\\\\\\/j2EC68EnuAuAuBjnl+n2Rd1MBk2p+UjbSMuOxFa3\\\\\\\/lhH0tcMQodUJGNg0hpZU5TKlDLp3b4iaB16M1F1Yub+bh2TtxD3CZfFqgstLMI4uRNNjeBaAg8WBuptXBuEYAFYw2SNtqXexrWIV1aqhf1+WKOT82I5\\\\\\\/Cgi0L1ImPD0cfqCfq5ubW7MN+Nj0VUbna5zuxkQDqmTzuSD8LmzMdVCnK+hk\\\\\\\/bZGjR5VpQpTpFLPfuKlslbDFX2FT78oN3YZzLadAK\\\\\\\/LAHpOtYg7TmDB+\\\\\\\/BvnVzjlwjCN+HLcPxyZmmLdoqojgQLdPciXyGhfyJvZe2zsDH5tSwXZqYBQAO50gxBEw8ttUxAa5p\\\\\\\/EG8STI5gtqagiRebAwkatKlDWauV4k1AWLQzM\\\\\\\/IBPkB7oGkH3yhug4DRBR0qj4z9B3+GnfLWafYQkzkeW4G8f6v7Krxtd++af6ZlnM\\\\\\\/PoAfN5K6wv7AikJSfgqMMDyYTZLVVPqh6bpbU297uFlN8c5tTyHb81S\\\\\\\/BMYUMamMOS9mbtJgTXw5C2S3dwtdSAuZnBAQZUBAyPo7uI0iJy2b2Yk1hQHqvCPF0xobXfEN1x2xI3CvRYeaF50sB8z6JaRP00PWQ7WyenDIDq64AAICqX23uNcriQGetzFBUtBgYTswfpvzNxRSDu3xXXyASiuilnw5EErxH8byevlJ5+U92GWy8IOur+bDHRyK7eAxAokMZAvP0LNFvvM2D77Qk0xNKpjOGpe3mMXohieWb8WXFVr+ge4OuWuUpSBc7xCUPEObDdiZ+p7dKcMK0vGSP+Vdyu\\\\\\\/1Zt7fOcmQVI9H8NYIF7A\\\\\\\/9VZHIHvao9LBbWQvhXqjwhu4M5i\\\\\\\/vdU61jvokFbiky6VewLgkmbwMtjBOUt2UcVKImHnwjvuqbgxBUJM+KkrQYmEiPvey2wfZV+l9N75c0ceH0Hjy3S+v4h9wiejFqIwsgegLNiEzVsbl49MmB1kalbdv0qOi9WKUH3+4fktqyL2MpX71hDoe9uqHVEhu1mFTIbNhnsPvqQKiHkSNyRTzHALL8yzAq78CCF02b+SRm\\\\\\\/qatlkiiffc3K+dimEQ6q9i0s1g1twiTZG+mTTZQj5oHHQmQBCKRmkQlyE+rj77qAe02ZjZw\\\\\\\/JuzGu5s8MhBBj63UPdxi8EnEe65ngMKbIdYSeqciMR3HmDKGFJ3BWBCfcqgeel5ME3pKtjN0IW1P3cmicmcKl7\\\\\\\/J8s6G67tfnbbSdT2Knf9\\\\\\\/wahFKf9kRnJXIswOnMLVRrAOKNLvomztGzV8QkAp1bXKKPBRIIxhmvCiVhhJVbfbuIiCAsTqL+Wh3EMveXnGpuyX\\\\\\\/9aGs7elx0aJGzHMmHFvLZfKnk8HhQWx8h5mRR1ODYH4cxzAa3KKJM+hHyH2VL+ivkS4wGmSmWux01HM\\\\\\\/EiVb9ff+dNKppT5ivPY5tE2r3jwTHAQ7yMHArUV2orCBt\\\\\\\/gczgWeuWfC8x1dOWnga48f+2G\\\\\\\/zOztiP6pLf95AKUNpgvDepMogTldZMGTb51E7ereg82OHbg4THFMSHx5Yb959Boio6Ptdb96sABjB8os1J1MpxZ9AAMf3GcpxNXfXr+6SLJ3N8qflWPMF9RqGw\\\\\\\/vDaQ90FwxEYeXu2mqvDSHkkI8pqI6txBCB9bICH1erUX60HB7gM33hiN698vK5VT8jxbq8haUZsStzmFpXRKIHMhfZIY0JuZV25UklShuW+B+s1zbnvfhUtHZEdLLvjkC7YTtGjEt3oCpSrd6oBnuu285XU2wKFTiICOVPhTTizKqRHp3y6BLY6qfUxYZiIAqwu\\\\\\\/McqlWF2yBJomNV7aSByXxe3zeC+5pQ27w8qms+mW5vJDwQnLDpkDJprGqry+5YMyc+48yukrvhc\\\\\\\/7a5dKz9rJHsGl2+w12K2fw3hPciOd3f6H+3oa3mNCVXD1N0RGQsymGsB\\\\\\\/OZSEeAeT38atDt5fSRcEzHurtuArq74sbbg9wVEfNn7CT+lRfs1Ms+pSiLhZuz22KknqzRScjp\\\\\\\/hjwniO0yMUjC0vimp7W42WBenP52iOM7yY+J6ZwWeKU8wXTFlwnvCqLNmCDhEHpgaSwydGeLdirIPF3Ja4Iolfj0rnDKQSnjQYcmPIL7Ax+\\\\\\\/8kTWYf3xeOG5d9flDIyHyOapd7vSOZ5HnEMf+OVaIer+qHHLAQGsHRu6pvwg8IENPpU+j\\\\\\\/wKH8Cm5tiARZ5ODVd3jmeEkTsQDmkMtsglfthtNJRNINIgVqdidPv43f8CBMlf24npi56kSjp0xKbe+J\\\\\\\/rDY974zzekefJYgQXvnXn9hZy+yMvHC+10Jdv7TIjWkaVWkpgl5eH2PmpUwMFKDdKuiwsgNoSrz7+BtoPQlxn9F7o56Gn1McPSB1XOR9IWdr4EHvp94ZsvC8V437LoWcwmonFy2wR+Qu7XKL01rQEfE+tcFwzKxRerqm3TpVyx0T9j5XOviakLYhBnTZRzRkbGe2q1O+UvbOsUa40k9dG9oeGKZVIruYVjDznqU3CF+ZQsis1DMYRPcJzfJuQKrZWaRw2T2mNiqMNPU6JBev3CCjtNoYSxJob99gmjy+zlQMoYXAffv4LrlFPNqYhyB\\\\\\\/FIVI12+ijT38j4ylZM6SOusGRTTGSWm01YQCWw\\\\\\\/XHHEvINi9cvvVXHcjXjMqaJazYL0+xt4uge\\\\\\\/GVqNzeymKODDh1X8U8UtGp2rvDXWxLCiGt9hW0HOFpYjL374bL2v+3ATxdqhZ7j1rR\\\\\\\/vKabPYnEMmkBaCllfdE30fq1b+vpMid\\\\\\\/cNxboVbsoktEDF9bOUL9s0zR+Q70fwB+eu+dEJKd2UKn1ee96Elh5j6HHLFoL9EC80Ot\\\\\\\/NcJTZKRPBOAK+H3\\\\\\\/VIcYgBtzMVSrBsJkc6UfxSz6m8gkbnhue6JaaGSAwYjbUaoTwHxSh9gTjnhx6S5T\\\\\\\/rXAY08cDKwcF1D7ROhT82lXbOv7Co3QZ5XKy4gWHboCaINjE2z+SSJ3bjbpGddfmtiFV20dyR3qCt2FAF4HRDLkacFaN59RU21ilRT9g5snIuz9EINns3dv+kckAVoR6dkoMy0KdnpczaSB1pjqYSlMRs0mfmaSP6rzV1mvy1R0rPy3uJVlUQ5eRVcMj3TBCZlOWSFMUBzmXiE4PV+DTHpQrLWhh82tKQs9EM7t+zt3w2xvA8LRMLygDVT34DQzbj6efX\\\\\\\/DJewLmffajrpFl4tjE2P68dx+2M8msEyrrTpqghnQdhgaV6TpkNKiaLIP9ftk53D6COVYewQK8rjVr5W4cjVr2pGTwrEmU9FJLryOjB0d9dkeB22KX16Wq3mesrj9xMlNQKgwA6ZeCPAL0m3w7IS5nbImnqtEkIWgMnwUtbmPQzFgQtJrY9bw2w155U9x3EvBAORlyQnO7Iu2JGk6KbuBkVbxRWoE\\\\\\\/Unv4OBNy8x3girLd0OJ5PCNExOFOiceWmF6pzXs6NsmDrzA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KDalwsCjzekc8I4+rcIO2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002290c9668c4397125b56c44d97056be44b4bdbf6ea5833918ce003e48a0132bd26\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mdeq\\\\\\\/oShjoIhW\\\\\\\/\\\\\\\/BNlpuoGSVXY74AMuOUnNPQ0+sGq10UiUVeeev+7cLd6J4Snn7at+48mp3ZOj\\\\\\\/Q2tmYpShnUmcSK5q6h2VH3xppGc9GZX4fi2siDdihi5uR0u5MrtqWkwvZcKkE7q7AHUw836yWhwf25Gxc4c1DsQhkD2wsABvxM7XvErzkP8p\\\\\\\/LoSacF7hRh5YKe8q+YZv6z7JNmAl0vUMoifMHkpTVFNOUXkunZWr9FMiu2s0pJRrM89ZXtLKMA09+k\\\\\\\/aL7s3BRZTOCpWbQcOhH6qi+KF+olcnNqXFCWlqf6NeG+VfUkcj2bRh24gNW1K34UEhdhSrVde9xsjrOjhXO7E5fXOyYUPpWSJfrUoxLIbU+hujsA9UGjOJ5LNvAxvaELAK0+Ut79Oy6VdSJBbXVcj74FCmHTu+3dwMG0GsZ3XIctgrs5JC1DxbxHvOD5bH7vY2ADNuPx4v3KBudMepTUj\\\\\\\/1LB6yx17UapNhnAMOyNLcINTLVjGu6vNVfqFbrByRLPmzQR1XunCaoa3PdxBLIeE\\\\\\\/QcjtGCht9t02GXb+4BI1yr7Xb\\\\\\\/dRY6xjUiz4XJSphj\\\\\\\/6IQe5klvYa1tL2TGZlBP3NIQSK2DhtB6kgVV8Gcd3wvV8BfwgLTafH\\\\\\\/gXWSKe3G4KFmnIdFrXrEJI5agx9irnMnyTwwGwkzDUhYpb+\\\\\\\/Bdis5yMI1hk6QhiAYVJia6BRWCtUixYWhEZI7jfAehCSlQ4UsLgSyoEBrlQeSPJna1RC+DYPiCuHrob2ToeiioBItGSntz9TWTQtkcnlqJqNT04kRB8zERWIjRxn1EaxEf827yLA0MC1vd5TRHuSdu4SXSeZrGXZfGVf92bcsOKFQl116wL7Q7n4mTkGyff\\\\\\\/4O06wBLMAHAY8HvcjrOKx0Ae+D5lyq0KCU5WGizfMnJ7Z1cPa86kHvyJ11CHXcHTknprL8BslTe1vHKXWjungE+8bjgaACGPWVdF+5v4\\\\\\\/7ky7kth1\\\\\\\/hgiZEQX+F\\\\\\\/oJ+0\\\\\\\/mwLfe722wOXJlYcsJuadr3c1BenruYnrUo47fkF9eM7HuWPTqaDvzkHabY9JijiCGCbCs30MOu5WzExASzEjf2kxlfBCELTFWrn\\\\\\\/ESsYR5OOAFfOazJKRgkwuZE0nbhDAq9QJKg2Lc956sTvB2vSR72Ehasj3aBSsGG8Yzs9ZHfGEaN\\\\\\\/2TM715xLrgjjy+x69AlUM8\\\\\\\/vQAjEWeMeWLZtUbiqAK70wetZJsqoYlX5u2uwH6RFmypPu6Hbiigw8OmgINHoMmdKxWD1\\\\\\\/WNpR\\\\\\\/bTWN+eZLpiLdex9tPf\\\\\\\/sT\\\\\\\/K8tPvQw1\\\\\\\/54g5v0ZRkAzSTodkoBBhsmsLFfT1XOruQfzWxYqV7qj3QAyvK\\\\\\\/FFCx\\\\\\\/EZnlq55L4qOtUxYTcqj\\\\\\\/\\\\\\\/noeaqITJPlaXLZGp7NeZLWH+Ap2G8qFXMShH1xuOnuOQ9W40MfO9\\\\\\\/sN59BLmuEjwNjFq60iQhiI+7bNJ4obt70rvhUGRC0tTEddB0IzmRDSlHFPoYfZkoFD2xFpbU2\\\\\\\/wuYIsL0fSvr309zhoffcvd0JO8Dx9c61QxsZN9mGpMDXBoaAgTz9x2JYaludmpf9f4QWielh9EHVJ6jl69fS8w1mmrP55TbgQbskezbfZfyRMq8zNbdbqB3ygjL222gV3yQul+UjJ6C2e1nTw2bZ6p\\\\\\\/DZm82F19HLTd2xynrgZiyP96EErVFaSrKeJnKW1P+eGN8pUdJXFIYQnNmpDDdX4fnXuAHS+Z\\\\\\\/UB7YTSpnQQRuIr3RcMpWjSwleHqp3v60f7ckI6QIol5kWtTLfma6Be2IKzTOXYBo6veIeBQpYEnCNTqm9UIBFD4mvCb6PNZA4Qce0s91PD76wMw27VajSDurKgA4YPzE3nHPWb7r92wzcP11Fk5gHey7UcIxNvbz0Wl3sAk4hbZ+TdC0A1ghG3DYO1ZINjrQYyUZNekymgJOPXjcmm7s6ZWvGGUpc6e9uEWE3O\\\\\\\/wKx+kDmzPk2urg6EocsvTwpoM3Dh43Mzw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EVQjNJReJgsg8izeXxw8JA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002235a9793ca3d5bca468a16c52f883c94b307cc0d1e7715f81f7b25d8c1f35b38a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491653 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3767" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QHyFyzwMm678iD9R-aHFWlnBA6zClc2SZY3VJT1z96A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3767" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0Nejf_kphqBTYI0vI7av-WH774dBb4imQRbA-nGSSQ4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3757" title="vqtl: An R Package for Mean-Variance QTL Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3783" title="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491653" id="node491653" data-pisa="ggg;8/12/3767" data-pisa-master="ggg;g3.118.200790" data-apath="/ggg/8/12/3767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTL Mapping on a Background of Variance Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3767-3782;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200790 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491653" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/12/3767" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491653" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3767.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491653" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3767.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491653" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/12/3767.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491653" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/12/3767.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491653" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491653/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Standard QTL mapping procedures seek to identify genetic loci affecting the phenotypic mean while assuming that all individuals have the same residual variance. But when the residual variance differs systematically between groups, perhaps due to a genetic or environmental factor, such standard procedures can falter: in testing for QTL associations, they attribute too much weight to observations that are noisy and too little to those that are precise, resulting in reduced power and and increased susceptibility to false positives. The negative effects of such “background variance heterogeneity” (BVH) on standard QTL mapping have received little attention until now, although the subject is closely related to work on the detection of variance-controlling genes. Here we use simulation to examine how BVH affects power and false positive rate for detecting QTL affecting the mean (mQTL), the variance (vQTL), or both (mvQTL). We compare linear regression for mQTL and Levene’s test for vQTL, with tests more recently developed, including tests based on the double generalized linear model (DGLM), which can model BVH explicitly. We show that, when used in conjunction with a suitable permutation procedure, the DGLM-based tests accurately control false positive rate and are more powerful than the other tests. We also find that some adverse effects of BVH can be mitigated by applying a rank inverse normal transform. We apply our novel approach, which we term “mean-variance QTL mapping”, to publicly available data on a mouse backcross and, after accommodating BVH driven by sire, detect a new mQTL for bodyweight.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cao%E2%80%99s-tests" class="hw-term hw-article-keyword hw-article-keyword-cao&#x2019;s-tests" rel="nofollow">Cao’s tests</a></li><li class="kwd"><a href="/keyword/reweighting" class="hw-term hw-article-keyword hw-article-keyword-reweighting" rel="nofollow">reweighting</a></li><li class="kwd"><a href="/keyword/vqtl" class="hw-term hw-article-keyword hw-article-keyword-vqtl" rel="nofollow">vQTL</a></li><li class="kwd"><a href="/keyword/variable-transformation" class="hw-term hw-article-keyword hw-article-keyword-variable-transformation" rel="nofollow">variable transformation</a></li><li class="kwd"><a href="/keyword/background-variance-heterogeneity" class="hw-term hw-article-keyword hw-article-keyword-background-variance-heterogeneity" rel="nofollow">background variance heterogeneity</a></li><li class="kwd"><a href="/keyword/heteroskedastic" class="hw-term hw-article-keyword hw-article-keyword-heteroskedastic" rel="nofollow">heteroskedastic</a></li><li class="kwd"><a href="/keyword/heteroscedastic" class="hw-term hw-article-keyword hw-article-keyword-heteroscedastic" rel="nofollow">heteroscedastic</a></li></ul><p id="p-3">A standard modeling assumption in quantitative trait locus (QTL) mapping is that all individuals, regardless of differences in their phenotypic mean, have the same residual variance. In reality, the residual variance—sometimes termed the environmental variance, and in general relating to the apparent noisiness of the phenotype—can differ between individuals. These differences in residual variance can arise from many sources, both extrinsic, such as environmental factors, and intrinsic, such as sex, or, more broadly, genetics. Environmental sources of residual variance heterogeneity have been well-documented, and include, for example, soil nitrogen and irrigation (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Makumburage and Stapleton 2011</a>), temperature (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shen <em>et al.</em> 2014</a>), and even the age at which young birds begin to experience the environmental insults outside of the nest (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Snell-Rood <em>et al.</em> 2015</a>). Genetic sources of residual variance heterogeneity have attracted increasing interest, with multiple studies finding instances of the residual variance being heritable (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sorensen and Waagepetersen 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hill and Mulder 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sørensen <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gonzalez <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mitchell <em>et al.</em> 2016</a>), and in some cases substantially attributable to allelic variation in individual genes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Paré <em>et al.</em> 2010</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wolc <em>et al.</em> 2012</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hulse and Cai 2013</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ayroles <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Forsberg <em>et al.</em> 2015</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Yadav <em>et al.</em> 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ivarsdottir <em>et al.</em> 2017</a>).</p><p id="p-4">The presence of residual variance heterogeneity, however, regardless of its source, can be problematic for analysis protocols that disregard it. Differences in residual variance between groups of individuals affect the precision of estimated means and, in turn, tests of significance or association (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cochran 1937</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yates and Cochran 1938</a>). In the context of QTL mapping, ignoring such differences discards information that could be exploited to increase the power to detect QTL; and in the case of mapping vQTL, it can covertly increase the false positive rate to well above the nominal level.</p><p id="p-5">Specifically, the background presence of a major variance-controlling factor (<em>e.g.</em>, sex, housing, strain, a vQTL, etc.) implies that inferences about any other effect (<em>e.g.</em>, that of a QTL elsewhere in the genome) occur against a backdrop of systematically heterogeneous residual variance. This “background variance heterogeneity” (BVH) acts to disrupt the natural observation weights: rather than every individual being subject to equal noise variance and therefore meriting equal weight, with BVH present some individuals’ phenotypes are inherently more (or less) noisy and so due less (or more) weight. And just as reweighting accordingly should lead to a more powerful analysis, then assuming all weights are equal (<em>i.e.</em>, variance homogeneity) risks overleveraging outliers and increasing the potential for both false negatives and false positives. This is likely to be true not only for studies detecting mQTL but also those detecting vQTL, which rely on the accurate attribution of residual noise.</p><p id="p-6">Nonetheless, consideration of variance effects—whether as the target of inference or as a feature of the data to be accommodated—has thus far remained outside of routine genetic analysis. This could be in part because vQTL are sometimes considered of esoteric secondary interest, intrinsically controversial in their interpretation (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sun <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Shen and Ronnegard 2013</a>), or <em>a priori</em> too hard to detect (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Visscher and Posthuma 2010</a>). But it is also likely to be in part because standard protocols for finding and reporting vQTL are currently lacking, and because the advantages of modeling heterogeneous variance, even when targeting mQTL, remain under-appreciated and largely undemonstrated.</p><p id="p-7">A number of statistical models and methods have been developed or adapted specifically to detect vQTL. These include: Levene’s test (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Struchalin <em>et al.</em> 2010</a>) and its generalizations (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Soave <em>et al.</em> 2015</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Soave and Sun 2017</a>); the Fligner-Killeen test (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fraser and Schadt 2010</a>); Bartlett’s test (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Freund <em>et al.</em> 2013</a>); and methods based on, or related to, the double generalized linear model (DGLM) and similar (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rönnegård and Valdar 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dumitrascu <em>et al.</em> 2018</a>). Tests have also been developed to detect genotype associations with arbitrary functions of the phenotype, for example higher moments, and these include a variant of the Komolgorov-Smirnov test (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aschard <em>et al.</em> 2013</a>) and a semi-parametric exponential tilt model (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hong <em>et al.</em> 2016</a>).</p><p id="p-8">Of the above methods, the ability to accommodate BVH of known source is limited to the DGLM of <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Rönnegård and Valdar (2011)</a> (as well as a very recent Bayesian counterpart, described in <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dumitrascu <em>et al.</em> 2018</a>), which can include variance effects of arbitrary covariates as well as those belonging to the target (or foreground) QTL.</p><p id="p-9">When the source of BVH is unknown, strategies to protect against it are less obvious. Since the threat manifests through sensitivity to distributional assumptions, possible remedies include side-stepping such assumptions via non-parametric approaches, <em>e.g.</em>, permutation testing, or reshaping the distribution prior to analysis through variable transformation. Both have been considered in the vQTL context, with permutation used in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hulse and Cai (2013)</a> and <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> (2012)</a> and transformation in <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Rönnegård and Valdar (2011)</a>, <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> (2012)</a>, <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sun <em>et al.</em> (2013)</a>, and <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shen and Carlborg (2013)</a>, but not specifically for controlling mQTL or vQTL false positives in the presence of BVH.</p><p id="p-10">Here we examine the effect of modeled and unmodeled BVH on power and false positive rate when mapping QTL affecting the mean, the variance, or both. In doing so we:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-11">Describe how the DGLM can be used develop a robust, straightforward procedure for routine mQTL and vQTL analysis, which we term “mean-variance QTL mapping”;</p></li><li id="list-item-2"><p id="p-12">Compare alternative proposed methods for mQTL and vQTL analysis;</p></li><li id="list-item-3"><p id="p-13">Show how accommodating BVH with the DGLM can improve power for detecting mQTL, vQTL, and mvQTL compared with other methods;</p></li><li id="list-item-4"><p id="p-14">Show how sensitivity to model assumptions can be rescued by variable transformation and/or permutation; and</p></li><li id="list-item-5"><p id="p-15">Demonstrate the discovery of a new QTL for mouse bodyweight from an existing F2 intercross data resource (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Leamy <em>et al.</em> 2000</a>).</p></li></ol><p id="p-16">In two companion papers, we describe R package vqtl, which implements our procedure (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Corty and Valdar 2018</a>), and in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Corty <em>et al.</em> (2018)</a> apply it to two published QTL mapping experiments detecting a novel mQTL in one and a novel vQTL in the other. In particular, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Corty <em>et al.</em> (2018)</a> demonstrates a principle investigated here: that when an mQTL also has variance effects, those variance effects induce a type of proximal BVH, and modeling them explicitly therefore improves mQTL detection.</p><div class="section methods" id="sec-1"><h2 class="">Statistical Methods</h2><p id="p-17">This section reviews the tests and evaluation procedures that we studied through simulation. First, we describe eight statistical tests that can be used to model the effect of a single locus on phenotype mean and/or variance: the standard linear model, Levene’s test, Cao’s three tests, and three DGLM-based tests. We also describe four procedures for evaluating the statistical significance (<em>i.e.</em>, calculating p-values) of these tests—a standard asymptotic evaluation and three procedures that reasonably could be expected to provide protection against violations of model assumptions.</p><div id="sec-2" class="subsection"><h3>Definitions</h3><p id="p-18">We start by defining three partially overlapping classes of QTL:</p><ul class="list-simple " id="list-2"><li id="list-item-6"><p id="p-19"><strong>mQTL</strong>: a locus containing a genetic factor that causes heterogeneity of phenotype mean,</p></li><li id="list-item-7"><p id="p-20"><strong>vQTL</strong>: a locus containing a genetic factor that causes heterogeneity of phenotype variance, and</p></li><li id="list-item-8"><p id="p-21"><strong>mvQTL</strong>: a locus containing a genetic factor that causes heterogeneity of either phenotype mean, variance, or both — a generalization that includes the other two classes. [Note: this usage is distinct from that of <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Yadav <em>et al.</em> (2016)</a>]</p></li></ul><p id="p-22">In addition, since we restrict our attention to QTL mapping methods that test genetic association with a phenotype one locus at a time, we distinguish two sources of variance effects:</p><ul class="list-simple " id="list-3"><li id="list-item-9"><p id="p-23"><strong>Foreground Variance Heterogeneity (FVH)</strong>: effects on the phenotype variance that arise from the locus under consideration (the focal locus);</p></li><li id="list-item-10"><p id="p-24"><strong>Background Variance Heterogeneity (BVH)</strong>: effects on the phenotype variance that arise from outside of the focal locus, <em>e.g.</em>, from another locus or an experimental covariate.</p></li></ul></div><div id="sec-3" class="subsection"><h3>Procedures to evaluate the significance of a single test</h3><p id="p-25">In comparing different statistical tests and their sensitivity to BVH, namely the effect of BVH on power and false positive rate (FPR), it is important to acknowledge that various measures could be taken to make significance testing procedures more robust to model misspecification in general and to BVH specifically. The significance testing methods considered here are frequentist, involving the calculation of a test statistic <em>T</em> on the observed data followed by an estimation of statistical significance based on a conception of <em>T</em>’s distribution under the null. BVH, however, will often constitute a departure of distributional assumptions, and in any rigorous applied statistical analysis when departures are expected it would be typical to consider protective measures such as, for example, transforming the response to make asymptotic assumptions more reasonable, or the use of computationally intensive procedures to evaluate significance empirically, such as those based on bootstrapping or permutation.</p><p id="p-26">Nominal significance (<em>i.e.</em>, the p-value for a single hypothesis test) is evaluated using four distinct procedures. The first two rely on asymptotics:</p><ol class="list-ord " id="list-4"><li id="list-item-11"><p id="p-27">Standard: The test statistic <em>T</em> is computed on the observed data and compared with its asymptotic distribution under the null.</p></li><li id="list-item-12"><p id="p-28">Rank-based inverse normal transform (RINT): As for standard, except observed phenotypes <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-1.gif"/></noscript></span></span> are first transformed to strict normality using the function <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-2.gif"/></noscript></span></span> where Φ is the normal c.d.f. and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-3.gif"/></noscript></span></span> is gives the rank (from <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-4.gif"/></noscript></span></span>) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beasley <em>et al.</em> 2009</a>).</p></li></ol><p id="p-29">The second two determine significance empirically based on randomization: the test statistic <em>T</em> is recomputed as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-5.gif"/></noscript></span></span> under randomizations of the data <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-6.gif"/></noscript></span></span>, and the resulting set of statistics <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-7.gif"/></noscript></span></span> is used as the empirical distribution of <em>T</em> under the randomized null. Two alternative randomizations are considered:</p><ul class="list-simple " id="list-5"><li id="list-item-13"><p id="p-30">3. Residperm: we generate a pseudo-null response <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-8.gif"/></noscript></span></span> based on permuting the residuals of the fitted null model, (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Freedman and Lane 1983</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Good 2013</a>), a process recently applied in the field of QTL mapping by <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> (2014)</a>.</p></li><li id="list-item-14"><p id="p-31">4. Locusperm: we leave the response intact, instead permuting the rows of the design matrix (or matrices) that differentiate(s) the null from alternative model.</p></li></ul></div><div id="sec-4" class="subsection"><h3>Procedure to evaluate genomewide significance</h3><p id="p-32">In the context of a genome scan, where many hypotheses are tested, we aim to control FPR genomewide through a family-wise error rate (FWER), the probability of making at least one false positive finding across the whole genome. This is done following the general approach of <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Churchill and Doerge (1994)</a>, which is closely related to the locusperm procedure described above, and which we refer to as genomeperm. Briefly, we perform an initial genome scan, recording test statistics <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-9.gif"/></noscript></span></span> for all <em>L</em> loci. Then for each randomization <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-10.gif"/></noscript></span></span> and for only the parts of the model that distinguish the null from the alternative model, the genomes are permuted among the individuals; the scan is then repeated to yield simulated null test statistics <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-11.gif"/></noscript></span></span> of which the maximum, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-12.gif"/></noscript></span></span> is recorded. The collection of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-13.gif"/></noscript></span></span> from all <em>R</em> such permutations is then used to fit a generalized extreme value distribution (GEV) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dudbridge and Koeleman 2004</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Valdar <em>et al.</em> 2006</a>), and the quantiles of this are used to estimate FWER-adjusted p-values for each <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-14.gif"/></noscript></span></span></p></div><div id="sec-5" class="subsection"><h3>Standard linear model (SLM) for detecting mQTL</h3><p id="p-33">The standard model of quantitative trait mapping uses a linear regression based on the approximation of <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Haley and Knott (1992)</a> and <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Martínez and Curnow (1992)</a> to interval mapping of <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lander and Botstein (1989)</a>. The effect of a given QTL on quantitative phenotype <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-15.gif"/></noscript></span></span> of individual <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-16.gif"/></noscript></span></span> is modeled as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-18.gif"/></noscript></span></span> is the residual variance and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-19.gif"/></noscript></span></span> is a linear predictor for the mean, defined, in what we term the “full model”, as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>μ</em> is the intercept, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-21.gif"/></noscript></span></span> is a vector of covariates with effects β, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-22.gif"/></noscript></span></span> is a vector encoding the genetic state at the putative mQTL with corresponding mQTL effects α. In the case considered here of biallelic loci arising from a cross of two founders, A and B, the genetic state vector <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-23.gif"/></noscript></span></span> is defined as follows: when genotype is known, for genotypes <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-24.gif"/></noscript></span></span> the additive dosage is <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-25.gif"/></noscript></span></span> and the dominance predictor is <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-26.gif"/></noscript></span></span>; when genotype is available only as estimated probabilities <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-29.gif"/></noscript></span></span> following <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Haley and Knott (1992</a>) and <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Martínez and Curnow (1992</a>), we use the corresponding expectations, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-31.gif"/></noscript></span></span></p><p id="p-34">The test statistic for an mQTL is based on comparing the fit of the full model, acting as an alternative model, with that of a null that omits the locus effect, namely,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Since the regression in each case provides a maximum likelihood fit, the test statistic used here is likelihood ratio (LR) statistic, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-33.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-35.gif"/></noscript></span></span> are the log-likelihoods under the alternative and the null respectively. For the biallelic model, the asymptotic test is the likelihood ratio test (LRT) whereby under the null, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-36.gif"/></noscript></span></span> (Note: Alternative evaluation using the F-test is in general more precise but for our purposes provides equivalent results.)</p><p id="p-35">The residperm approach to empirical significance evaluation of <em>T</em> proceeds as follows. We first fit the null model (<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>) to obtain predicted values <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-37.gif"/></noscript></span></span> and estimated residuals <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-38.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-39.gif"/></noscript></span></span> Then, for each randomization <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-40.gif"/></noscript></span></span> we generate pseudo-null phenotypes <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-41.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-42.gif"/></noscript></span></span>where if <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-43.gif"/></noscript></span></span> is a vector containing a random permutation of the indices <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-44.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-45.gif"/></noscript></span></span> is its <em>i</em>th element, mapping index <em>i</em> to its <em>r</em>th permuted version. The null and alternative models are then fitted to <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-46.gif"/></noscript></span></span> to yield <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-48.gif"/></noscript></span></span> and hence <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-49.gif"/></noscript></span></span></p><p id="p-36">In the locusperm approach to empirical significance, the response is unchanged but permutations are applied to the locus genotypes. For each randomization <em>r</em>, the full model <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-50.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-51.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-52.gif"/></noscript></span></span> is as defined for residperm above. This full model fit yields <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-53.gif"/></noscript></span></span> and then <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-54.gif"/></noscript></span></span> Note that <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-55.gif"/></noscript></span></span> need not be recomputed after randomization because because only the rows of the design matrices that are unique to the alternative model are permuted and thus <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-56.gif"/></noscript></span></span></p></div><div id="sec-6" class="subsection"><h3>Levene’s Test (LV) for detecting vQTL</h3><p id="p-37">Levene’s test is a procedure for differences in variance between groups that can be used to detect vQTL. Suppose individuals are in <em>G</em> mutually exclusive groups <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-57.gif"/></noscript></span></span> Let <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-58.gif"/></noscript></span></span> denote the group to which individual <em>i</em> belongs, denote <em>g</em>th group size as <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-59.gif"/></noscript></span></span> and <em>g</em>th group mean as <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-60.gif"/></noscript></span></span> Then denote the <em>i</em>th absolute deviation as <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-61.gif"/></noscript></span></span> the group mean of these as <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-62.gif"/></noscript></span></span> and overall mean <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-63.gif"/></noscript></span></span> Levene’s <em>W</em> statistic is then<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-64.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>which under the null model of no variance effect follows the <em>F</em> distribution as <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-65.gif"/></noscript></span></span> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Levene 1960</a>). Note that replacing means of <em>y</em> with medians gives the related Brown-Forsythe test (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown and Forsythe 1973</a>), and replacing all instances of z with <em>y</em> in <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 5</a> gives the ANOVA <em>F</em> statistic.</p><p id="p-38">Levene’s test does not lend itself naturally to the residperm approach because it does not explicitly involve a null model to split the data into hat values and residuals. We therefore use the null model from the SLM (<a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>) to approximate the residperm procedure with Levene’s test. To execute the locusperm procedure, for each randomization <em>r</em>, the group labels are permuted among the individuals, which is equivalent to replacing all instances of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-66.gif"/></noscript></span></span> above with <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-67.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-68.gif"/></noscript></span></span> defined as above. A corresponding genomewide procedure, although not performed here, would ensure that each randomization <em>r</em> applies the same permutation <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-69.gif"/></noscript></span></span> across all loci.</p></div><div id="sec-7" class="subsection"><h3>Cao’s Tests</h3><p id="p-39"><a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> (2014)</a> elaborates the SLM to have a variance parameter that differs by genotype, <em>i.e.</em>,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-71.gif"/></noscript></span></span> is the linear predictor, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-72.gif"/></noscript></span></span> is the variance of the <em>i</em>th individual. These are defined in what we term the “full model” as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-73.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-74.gif"/></noscript></span></span> indexes the genotype group to which <em>i</em> belongs, and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-75.gif"/></noscript></span></span> are the variances of the <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-76.gif"/></noscript></span></span> genotype groups. Thus an individual’s variance is entirely dictated by its genotype, and that genotype must be categorically known (or otherwise assigned). <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> (2014)</a> fits this model using a two-step, profile likelihood method, which in our applications we observe to be indistinguishable from full maximum likelihood (Figure S8).</p><p id="p-40"><a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> (2014)</a> describes tests for mQTL, vQTL and mvQTL based on comparing a full model against three different null models; we detail these tests below in our notation, denoting them respectively CaoM, CaoV, and CaoMV.</p><div id="sec-8" class="subsection"><h4>Cao<sub>M</sub> test for detection of mQTL:</h4><p id="p-41">The Cao<sub>M</sub> test involves an LRT between Cao’s full model and Cao’s no-mQTL model:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>To execute the residperm procedure for CaoM, pseudo-null phenotypes are generated using <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-79.gif"/></noscript></span></span> from Cao’s no-mQTL model (<a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 8</a>). The locusperm procedure respecifies the full model (<a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Equation 7</a>), leaving the variance model unchanged and specifying the mean predictor as <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-80.gif"/></noscript></span></span></p></div><div id="sec-9" class="subsection"><h4>Cao<sub>V</sub> for detection of vQTL:</h4><p id="p-42">The Cao<sub>V</sub> test involves an LRT between Cao’s full model and Cao’s no-vQTL model:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-81.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where the unsubscripted <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-82.gif"/></noscript></span></span> is a single, overall residual variance. This null model is identical to the alternative model in the SLM (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>).</p><p id="p-43">To execute the residperm procedure for Cao<sub>V</sub>, pseudo-null phenotypes are generating using <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-84.gif"/></noscript></span></span> from Cao’s no-mQTL model (<a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation 9</a>). The locusperm procedure respecifies the full model (<a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">Equation 7</a>), leaving the mean sub-model unchanged and specifying the variance predictor as <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-85.gif"/></noscript></span></span></p></div><div id="sec-10" class="subsection"><h4>Cao<sub>MV</sub> for detection of generalized mvQTL:</h4><p id="p-44">The Cao<sub>MV</sub> test involves an LRT between Cao’s full model and Cao’s no-QTL model:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>This null model is identical to the null model in the SLM (<a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>).</p><p id="p-45">To execute the residperm procedure for Cao<sub>MV</sub>, pseudo-null phenotypes are generated using <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-88.gif"/></noscript></span></span> from Cao’s no-QTL model (<a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation 10</a>). The locusperm procedure specifies the mean predictor as <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-89.gif"/></noscript></span></span> and the variance predictor as <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-90.gif"/></noscript></span></span></p></div></div><div id="sec-11" class="subsection"><h3>Double Generalized Linear Model (DGLM)</h3><p id="p-46">The DGLM models the phenotype <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-91.gif"/></noscript></span></span> via two linear predictors as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-92.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-93.gif"/></noscript></span></span> predicts the phenotype mean and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-94.gif"/></noscript></span></span> predicts the extent to which the baseline residual variance <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-95.gif"/></noscript></span></span> is increased in individual <em>i</em>. In what we term the “DGLM full model”, these are specified as<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-96.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>where <em>μ</em> is the intercept, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-97.gif"/></noscript></span></span> is a vector of covariates (which may be identical to <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-98.gif"/></noscript></span></span>), γ is a vector of covariate effects on <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-99.gif"/></noscript></span></span>, and θ is a vector of locus effects on <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-100.gif"/></noscript></span></span>.</p><p id="p-47">As with Cao’s full model, the DGLM full model can be compared, in a likelihood ratio test, with various null models to test for mQTL, vQTL (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Rönnegård and Valdar 2011</a>), or mvQTL. A full maximum likelihood fitting procedure for the DGLM was provided by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smyth (1989)</a>.</p><div id="sec-12" class="subsection"><h4>DGLM<sub>M</sub> for detecting mQTL:</h4><p id="p-48">For detecting mQTL, we use an LRT of the DGLM full model in <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">Equation 11</a> against the no-mQTL model:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-101.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>where the LR statistic has asymptotic distribution <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-102.gif"/></noscript></span></span>.</p><p id="p-49">To execute the residperm procedure for DGLMM, pseudo-null phenotypes are generated using <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-104.gif"/></noscript></span></span> from <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation 12</a>. The locusperm procedure respecifies the mean predictor as <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-105.gif"/></noscript></span></span> and does not modify the variance predictor.</p></div><div id="sec-13" class="subsection"><h4>DGLM<sub>V</sub> for detecting vQTL:</h4><p id="p-50">For detecting vQTL, we use an LRT of the DGLM full model in <a id="xref-disp-formula-13-2" class="xref-disp-formula" href="#disp-formula-13">Equation 11</a> against the no-vQTL model:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-106.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where the LR statistic has asymptotic distribution <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-107.gif"/></noscript></span></span>.</p><p id="p-51">To execute the residperm procedure for DGLMV, pseudo-null phenotypes are generated using <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-108.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-109.gif"/></noscript></span></span> from the <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation 13</a>. The locusperm procedure does not modify the variance predictor and respecifies the mean predictor as <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-110.gif"/></noscript></span></span>.</p></div><div id="sec-14" class="subsection"><h4>DGLM<sub>MV</sub> for detecting mvQTL:</h4><p id="p-52">For detecting mvQTL, we use an LRT of the DGLM full model in <a id="xref-disp-formula-13-3" class="xref-disp-formula" href="#disp-formula-13">Equation 11</a> against the no-QTL model:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-111.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>where the LR statistic has asymptotic distribution <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-112.gif"/></noscript></span></span>.</p><p id="p-53">To execute the residperm procedure for DGLMMV, pseudo-null phenotypes are generated using <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-113.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-114.gif"/></noscript></span></span> from the <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">Equation 14</a>. The locusperm procedure respecifies the mean predictor as <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-115.gif"/></noscript></span></span> and the variance predictor as <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-116.gif"/></noscript></span></span>.</p></div></div></div><div class="section methods" id="sec-15"><h2 class="">Simulation Methods</h2><p id="p-54">The eight methods and four significance testing procedures described in the previous section, amounting to 32 test-procedure combinations in total, were compared by simulation. The simulations examined the performance of each combination, in terms of false and true positive rate, under eight distinct scenarios relating to the presence or absence of a QTL (and if present, then what type), and the presence or absence of BVH. We describe the general simulation setup below, followed by a detailed description of the eight scenarios and then describe the metrics by which performance was judged.</p><div id="sec-16" class="subsection"><h3>Simulating locus and covariate</h3><p id="p-55">Each simulated experiment consisted of 300 individuals, where each individual was defined by one single-locus genotype, one covariate, and one phenotype.</p><p id="p-56">The genotype for individual <em>i</em>, denoted by <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-117.gif"/></noscript></span></span>, was simulated according to a random process to mimic an F2 intercross:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-118.gif"/></noscript></span></span>The covariate for individual <em>i</em>, denoted <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-119.gif"/></noscript></span></span>, was specified as a five-level categorical factor, with levels assigned to individuals as<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-120.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-121.gif"/></noscript></span></span> is an indicator vector such that, for example, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-122.gif"/></noscript></span></span> denotes membership of level 1. This covariate, which was fixed across simulations, was intended to mimic a generic, fixed aspect of experimental design in a typical QTL mapping study (for example, batch, technician, housing, etc.) that could plausibly influence the precision of the observations. When BVH is simulated, it is driven by this covariate.</p></div><div id="sec-17" class="subsection"><h3>Scenarios</h3><p id="p-57">We conducted simulated experiments under eight different scenarios. These scenarios varied conceptually across two dimensions. First, we considered four types of locus:</p><ol class="list-ord " id="list-6"><li id="list-item-15"><p id="p-58">null locus: The locus has no effect on phenotype.</p></li><li id="list-item-16"><p id="p-59">pure mQTL: The locus has an additive effect on the phenotype mean.</p></li><li id="list-item-17"><p id="p-60">pure vQTL: The locus has an additive effect on the log of the residual phenotype variance.</p></li><li id="list-item-18"><p id="p-61">mixed mvQTL: The locus has both an additive effect on phenotype mean and an additive effect on the log of residual phenotype variance.</p></li></ol><p id="p-62">Then, we considered whether or not BVH was present, <em>i.e.</em>:</p><ol class="list-ord " id="list-7"><li id="list-item-19"><p id="p-63">BVH absent: The covariate does not influence the residual variance of the phenotype.</p></li><li id="list-item-20"><p id="p-64">BVH present: The covariate influences the residual variance of the phenotype (in addition to the locus, if a vQTL or mvQTL).</p></li></ol><p id="p-65">The resulting eight scenarios (<em>i.e.</em>, all combinations) were realized <em>in silico</em> with three parameters: the effect of the locus on phenotype mean (<em>α</em>), the effect of the locus on phenotype variance (<em>θ</em>), and the effect of the covariate on phenotype variance (γ). Values assigned to these parameters are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The rationale for selecting values of <em>α</em> and <em>θ</em> was as follows:</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491614/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491614/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Eight scenarios were simulated, as determined by the values of three parameters. <em>α</em> indicates the additive effect of the locus on phenotype mean, <em>θ</em> the additive effect of the locus on phenotype variance, and γ the effect of the covariate on phenotype variance. The two possible values of γ are <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T1/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T1/embed/mml-math-123.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T1/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T1/embed/mml-math-124.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div><ol class="list-ord " id="list-8"><li id="list-item-21"><p id="p-66">pure mQTL: The effect size of the pure mQTL was chosen so that it always explains 5% of the phenotype variance, which is consistent with smaller effect sizes typically sought and identified in QTL mapping experiments. Such an mQTL is detectable with approximately 70% power at a 5% false positive rate by the traditional mQTL test (the standard linear model) when 300 individuals are simulated, a typical population size for QTL mapping experiments.</p></li><li id="list-item-22"><p id="p-67">pure vQTL: vQTL analysis is much less established, so the vQTL effect size was chosen to match the detectability of the mQTL. Thus, the vQTL effect size was defined such that the traditional vQTL test (Levene’s test) has 70% power at 5% FPR in a population of 300 individuals in the absence of BVH.</p></li><li id="list-item-23"><p id="p-68">mixed mvQTL: The mvQTL effect sizes were chosen such that the mean and variance signals are equally detectable, and the aggregate signal is detectable by Cao<sub>MV</sub> and DGLM<sub>MV</sub> with 70% power at an FPR of 5% in a population of 300 individuals in the absence of BVH.</p></li></ol><p id="p-69">The values of γ used for simulating BVH were <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-129.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-130.gif"/></noscript></span></span>. The former chosen to ensure constant residual variance for simulations where BVH is absent; the latter to mirror the extent of BVH we noted in experimental data, while having a concise expression as equally spaced effects centered at zero. In null locus and mQTL simulations, <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-131.gif"/></noscript></span></span> results in group-wise standard deviations of approximately <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-132.gif"/></noscript></span></span>. In vQTL and mvQTL simulations, <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-133.gif"/></noscript></span></span> and <em>θ</em> combine additively on the log standard deviation scale and result in fifteen unique variances as detailed in the Supplementary Materials.</p></div><div id="sec-18" class="subsection"><h3>Phenotype simulation</h3><p id="p-70">For each of the eight scenarios, we conducted <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-134.gif"/></noscript></span></span> simulated experiment. For scenario <em>s</em>, the phenotype for individual <em>i</em>, denoted <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-135.gif"/></noscript></span></span>, was simulated from a normal distribution based on the genotype and covariate (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-136.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-137.gif"/></noscript></span></span>) and the scenario parameters (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-138.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-139.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-140.gif"/></noscript></span></span>) as:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-141.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-142.gif"/></noscript></span></span>, and<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-143.gif"/></noscript></span></span>(Further details in Supplementary Materials.)</p></div><div id="sec-19" class="subsection"><h3>Testing significance</h3><p id="p-71">To each simulated experiment, eight tests were applied, and four procedures were used to assess the statistical significance of each test, for a total of 32 test-procedures.</p><p id="p-72">The eight tests comprise three tests for detecting mQTL: SLM, CaoM, and DGLMM; three for detecting vQTL: Levene’s test, Cao<sub>V</sub>, DGLM<sub>V</sub>; and two for detecting mvQTL: Cao<sub>MV</sub> and DGLM<sub>MV</sub>. These tests are detailed in the Statistical Methods and summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491674/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491674/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The eight tests that were evaluated in the simulation studies. FVH: foreground variance heterogeneity (<em>i.e.</em>, variance effects at the QTL). BVH: background variance heterogeneity</span><div class="sb-div caption-clear"></div></div></div><p id="p-73">The four procedures for evaluating the statistical significance of results were: standard, RINT, residperm, and locusperm, as described in the Statistical Methods. The RINT procedure was selected because it returns any phenotype distribution, no matter how exotic, to a standard normal distribution. The fact that it is commonly used in genetics research demands that its properties, and its effects on QTL mapping, be better understood. The residperm was selected because it was recently proposed for use in mQTL, vQTL, and mvQTL mapping studies (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> 2014</a>). The locusperm procedure was developed in response to suspected shortcomings of the above robustifying procedures.</p></div><div id="sec-20" class="subsection"><h3>Evaluation of tests and procedures</h3><p id="p-74">Tests and procedures for assessing statistical significance were evaluated based on their empirical false positive rate (FPR) and power at a nominal FPR of 0.05. The empirical FPR of a given test-procedure combination in a given scenario was taken as the fraction of null simulations (where the phenotype was simulated with no dependence on genotype) that resulted in <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-144.gif"/></noscript></span></span>. Similarly, the empirical power was computed as the fraction of non-null simulations that resulted in <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-145.gif"/></noscript></span></span>. These quantities are naturally considered as estimates of a binomial proportion, so their standard errors were calculated by the method of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clopper and Pearson (1934)</a>.</p><p id="p-75">The above evaluations focused only on the cutoff of <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-146.gif"/></noscript></span></span>. Also considered, however, were all possible cutoffs, using QQ plots and ROC plots, which allow examination of the empirical FPR as a function of nominal FPR and the empirical power as a function of empirical FPR, respectively; these illustrate the spectrum of trade-offs that each test makes available, but do not meaningfully change the overall interpretation of the results, so we relegate them to the Supplementary Materials.</p></div></div><div class="section other1" id="sec-21"><h2 class="">Data and Software</h2><div id="sec-22" class="subsection"><h3>Leamy <em>et al.</em> summary of original study</h3><p id="p-76"><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Leamy <em>et al.</em> (2000)</a> backcrossed mice from strain CAST/Ei, a small, lean strain, into mouse strain M16i, a large, obese strain. Nine F1 males were bred with 54 M16i females to produce a total of 421 offspring (208 female, 213 male), which were genotyped at 92 microsatellite markers across the 19 autosomes and phenotyped for body composition and morphometric traits. We retrieved all available data on this cross, which included marker genotypes, covariates, and eight phenotypes (body weight at five ages, liver weight, subcutaneous fat pad thickness, and gonadal fat pad thickness), from the Mouse Phenome Database (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Grubb <em>et al.</em> 2014</a>), and estimated genotype probabilities at 2cM intervals across the genome using the hidden Markov model in R/qtl (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2003</a>).</p><p id="p-77">This mapping population has been studied for association with several phenotypes: asymmetry of mandible geometry (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Leamy <em>et al.</em> 2000</a>), limb bone length (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Leamy <em>et al.</em> 2002</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wolf <em>et al.</em> 2006</a>), organ weight (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Leamy <em>et al.</em> 2002</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Wolf <em>et al.</em> 2006</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Yi <em>et al.</em> 2006</a>), fat pad thickness (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yi <em>et al.</em> 2005</a>, <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">2006</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">2007</a>), and body weight (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Yi <em>et al.</em> 2006</a>). The most relevant prior study to this reanalysis, <a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Yi <em>et al.</em> (2006)</a>, used standard methods to identify QTL for body weight at three weeks on chromosomes 1 and 18. However, we were not able to reproduce this result, despite following their analysis as described.</p></div><div id="sec-23" class="subsection"><h3>Availability of data and software</h3><p id="p-78">Analyses were conducted in the R statistical programming language (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">R Core Team 2017</a>). The simulation studies used the implementation of the standard linear model from package stats, Levene’s test from car, Cao’s tests as published in <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Cao <em>et al.</em> (2014)</a> and the DGLM tests in package dglm. Files S1, S2, and S3 contain the R scripts necessary to replicate the simulation studies and their analysis, relying on the plotROC package to make ROC plots (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sachs 2017</a>). File S4 contains the data from <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Leamy <em>et al.</em> (2000)</a> that was reanalyzed. File S5 contains the attempted replication of the original analysis (<a id="xref-ref-76-5" class="xref-bibr" href="#ref-76">Yi <em>et al.</em> 2006</a>) and file S6 contains the new analysis, using package vqtl (Corty and Valdar, 2018).</p><p id="p-79">The reanalyzed dataset is available on the Mouse Phenome Database (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Grubb <em>et al.</em> 2014</a>) with persistent identifier MPD:206. The entire project, including data and all analysis scripts, is available as a public, static Zenodo repository with DOI: 10.5281/zenodo.1455184. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7290146">https://doi.org/10.25387/g3.7290146</a>.</p></div></div><div class="section results" id="sec-24"><h2 class="">Results</h2><div id="sec-25" class="subsection"><h3>Simulation study on single locus testing</h3><p id="p-80">Simulations were performed to examine the ability of the eight tests listed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> to detect nonzero effects belonging to their target QTL types (mQTL, vQTL, mvQTL), and to control the number of false positives when no such QTL effects were present. Simulations were conducted in the presence and absence of background variance heterogeneity (BVH), and for each test, with p-values calculated by each of the four significance assessment procedures (standard, RINT, residperm, locusperm). The full combination of settings is listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, which also lists results pertaining to a nominal FPR of 0.05, and described in more detail in Simulation Methods section.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491568/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491568/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Empirical positive rates of all tests under all significance assessment procedures in all scenarios based on 10,000 simulations, 1,000 permutations each to estimate empirical null distributions (residperm and locusperm), and a nominal false positive rate (FPR) of <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T3/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/T3/embed/mml-math-147.gif"/></noscript></span></span>. Entries in column 1 and 5 through all rows, columns 3 and 7 in the top third, and columns 2 and 6 in the middle third represent empirical FPR. Where the empirical FPR is within one standard error of the nominal FPR of 0.05, it is written in normal font. Where it is overly conservative, it is underlined. Where it is anti-conservative, it is in boldface. The entries in the rest of the table represent power. Given the sample size of 10,000, the standard error for the values in this table are all between 0.005 and 0.01. Generally, values near 0.05 have a standard error near 0.005 and values near 0.5 have a standard error near 0.01. All standard errors are listed in Table S2 and plotted visually in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Testing for mQTL with BVH absent: SLM and Cao<sub>M</sub> outperform DGLM<sub>M</sub></h3><p id="p-81">In the absence of BVH, SLM and Cao<sub>M</sub> accurately control FPR under all significance assessment procedures (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). DGLM<sub>M</sub> was slightly anti-conservative under the standard and RINT procedures with FPR = 0.057 and 0.055, respectively. With either permutation procedure used to assess significance, however, DGLM<sub>M</sub> accurately controlled FPR.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical false positive rate (FPR) of all tests and significance assessment procedures at a nominal FPR of 0.05, as assessed through simulation of non-associated loci and phenotypes both with and without BVH. Dot indicates point estimate and line indicates 95% confidence interval. The vertical line indicates the ideal empirical FPR of 0.05. Some test-procedure combinations led to FPR outside the plotted range. In such cases the FPR is written on the left edge of the plotting area if the value was too low to plot, and the right edge if it was too high. An un-zoomed version of this plot is available in Figure S7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical false positive rate (FPR) of all tests and significance assessment procedures at a nominal FPR of 0.05, as assessed through simulation of non-associated loci and phenotypes both with and without BVH. Dot indicates point estimate and line indicates 95% confidence interval. The vertical line indicates the ideal empirical FPR of 0.05. Some test-procedure combinations led to FPR outside the plotted range. In such cases the FPR is written on the left edge of the plotting area if the value was too low to plot, and the right edge if it was too high. An un-zoomed version of this plot is available in Figure S7.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F1.medium.gif" width="219" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/12/3767/F1.medium.gif" width="219" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491640" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-82" class="first-child">Empirical false positive rate (FPR) of all tests and significance assessment procedures at a nominal FPR of 0.05, as assessed through simulation of non-associated loci and phenotypes both with and without BVH. Dot indicates point estimate and line indicates 95% confidence interval. The vertical line indicates the ideal empirical FPR of 0.05. Some test-procedure combinations led to FPR outside the plotted range. In such cases the FPR is written on the left edge of the plotting area if the value was too low to plot, and the right edge if it was too high. An un-zoomed version of this plot is available in Figure S7.</p><div class="sb-div caption-clear"></div></div></div><p id="p-83">SLM and Cao<sub>M</sub> had indistinguishable power in the detection of mQTL under all significance assessment procedures (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). DGLM<sub>M</sub>, however, had equal power to those tests only under the standard and RINT procedures, which have inflated FPR. Under the permutation-based procedures, DGLM<sub>M</sub> was less powerful than the other test-procedures.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical power of mQTL tests to detect mQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of SLM with the standard significance assessment procedure, the standard against which all other test-procedures are compared." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical power of mQTL tests to detect mQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of SLM with the standard significance assessment procedure, the standard against which all other test-procedures are compared.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F2.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/12/3767/F2.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491658" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-84" class="first-child">Empirical power of mQTL tests to detect mQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of SLM with the standard significance assessment procedure, the standard against which all other test-procedures are compared.</p><div class="sb-div caption-clear"></div></div></div><p id="p-85">These results reflect the reality that, when a simple model is exactly true, a more elaborate model tends to be less powerful. Additionally, they highlight the capability of the permutation-based procedures to accurately control FPR even when the standard and RINT procedures fail to do so (as in the case of DGLM<sub>M</sub>).</p><div id="sec-27" class="subsection"><h4>Testing for mQTL with BVH present: DGLM<sub>M</sub> dominates:</h4><p id="p-86">SLM and Cao<sub>M</sub> accurately controlled FPR under all four procedures to assess statistical significance (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). As in the absence of BVH, DGLM<sub>M</sub> exhibited a slightly inflated FPR under the standard and RINT procedures (0.059 and 0.057, respectively), but accurately controlled FPR under the permutation-based procedures (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-87">Under all four procedures, DGLM<sub>M</sub> was more powerful than SLM and Cao<sub>M</sub> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The two procedures under which DGLM<sub>M</sub> accurately controlled FPR had power of 0.822 and 0.818, greatly exceeding the power of Cao<sub>M</sub> and SLM, which were in the range [0.694, 0.719] (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-88">Based on the results of these simulations, DGLM<sub>M</sub>-residperm and DGLM<sub>M</sub>-locusperm are the recommended test-procedure combinations for mQTL testing in the presence of BVH.</p><p id="p-89">For each mQTL test-procedure combination, the AUC (Table S1), standard error of the positive rate at <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-148.gif"/></noscript></span></span> (Table S2), QQ plots illustrating the empirical FPR at each nominal FPR level (Figure S4), and ROC curves illustrating the spectrum of trade-offs between available FPR and power (Figure S1) are provided in the Supplementary Materials.</p></div><div id="sec-28" class="subsection"><h4>Testing for vQTL with BVH absent: Cao<sub>V</sub> and DGLM<sub>V</sub> outperform Levene’s test:</h4><p id="p-90">In the absence of BVH, all vQTL tests had nearly-accurate FPR control (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). All tests had FPR within one standard error of 0.05 under both empirical significance assessment procedures (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and Table S2) But under either asymptotic procedure, Levene’s test was slightly conservative. And Cao<sub>V</sub> and DGLM<sub>V</sub> were both slightly anti-conservative under the standard procedure and conservative under the RINT procedure.</p><p id="p-91">Despite the variation in FPR control among the test-procedure combinations, Cao<sub>V</sub> and DGLM<sub>V</sub> had more power to detect vQTL than Levene’s test under all procedures. Specifically, under the well-calibrated (empirical) procedures, Cao<sub>V</sub> and DGLM<sub>V</sub> had power in the range [0.698, 0.721], while under those same well-calibrated (empirical) procedures, Levene’s test had power in the range [0.664, 0.665] (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-92">Thus, in the specific situations simulated here, the empirical procedures of Cao<sub>V</sub> and DGLM<sub>V</sub> are the preferred vQTL tests in the absence of BVH. The additional power of Cao<sub>V</sub> and DGLM<sub>V</sub> relative to Levene’s test is consistent with the fact that they make strong parametric assumptions that are exactly true in these simulations and Levene’s test does not.</p></div><div id="sec-29" class="subsection"><h4>Testing for vQTL with BVH present: DGLM<sub>V</sub> outperforms Levene’s test and Cao<sub>V</sub>:</h4><p id="p-93">In the presence of BVH, there were three test-procedure combinations with major departures from accurate FPR control (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Cao<sub>V</sub> under the standard procedure was drastically anti-conservative with FPR of 0.135 (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). DGLM<sub>V</sub> under both the RINT and residperm procedures was drastically conservative with FPR of 0.023 and 0.020, respectively. Additionally, DGLM<sub>V</sub> under the standard procedure was moderately anti-conservative with FPR of 0.058. The remaining test-procedure combinations accurately controlled FPR, namely Levene’s test under all procedures, Cao<sub>V</sub> under the RINT, residperm, and locusperm procedures, and DGLM<sub>V</sub> under the locusperm procedure.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical power of vQTL tests to detect vQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of Levene&#x2019;s test with the standard significance assessment procedure, the standard against which the other test-procedures are compared." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical power of vQTL tests to detect vQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of Levene&#x2019;s test with the standard significance assessment procedure, the standard against which the other test-procedures are compared.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F3.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/12/3767/F3.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-94" class="first-child">Empirical power of vQTL tests to detect vQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of Levene’s test with the standard significance assessment procedure, the standard against which the other test-procedures are compared.</p><div class="sb-div caption-clear"></div></div></div><p id="p-95">Of the tests that accurately controlled FPR, DGLMV under the locusperm procedure was uniquely powerful, with power of 0.708, while the others had power in the range [0.539, 0.576] (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-96">Direct interpretation of these results might lead one to consider the trade-off between DGLM<sub>V</sub>-standard and DGLM<sub>V</sub>-locusperm. DGLM<sub>V</sub>-locusperm requires considerable computational effort and serves only to reduce the FPR from a modestly-inflated level of 0.058 to accurate control at 0.052. Application of the (computationally non-intensive) DGLM<sub>V</sub>-standard, however, comes with a caveat: if there were some additional, unknown (and therefore unmodeled) BVH-driving factor, DGLMV-standard would be anti-conservative anti-conservative–similar to Cao<sub>V</sub>-standard with BVH present. The locusperm procedure, in contrast, ensures accurate FPR control whether all BVH-driving factors are modeled (as in DGLM<sub>V</sub>) or not (as in Cao<sub>V</sub>). DGLM<sub>V</sub>-locusperm therefore emerges as the most robust test-procedure for vQTL mapping in the presence of BVH.</p><p id="p-97">For each vQTL test-procedure combination, the AUC (Table S1), standard error of the positive rate at <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-149.gif"/></noscript></span></span> (Table S2), QQ plots illustrating the empirical FPR at each nominal FPR level (Figure S5), and ROC curves illustrating the spectrum of trade-offs between available FPR and power (Figure S2) are provided in the Supplementary Materials.</p></div><div id="sec-30" class="subsection"><h4>Testing mvQTL with BVH absent: Cao<sub>MV</sub> and DGLM<sub>MV</sub> similar:</h4><p id="p-98">Continuing the pattern from the vQTL tests, in the absence of BVH most mvQTL tests accurately control FPR (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). The exceptions are similar to the vQTL tests as well, with Cao<sub>MV</sub>-RINT slightly conservative and DGLM<sub>MV</sub>-standard slightly anti-conservative (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-99">The standard version of Cao<sub>MV</sub> and DGLM<sub>MV</sub> were similarly powerful (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), both exceeding the power of the other mvQTL test-procedures.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical power of mvQTL tests to detect mvQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of CaoMV with the standard significance assessment procedure, the standard against which the other test-procedures are compared." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical power of mvQTL tests to detect mvQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of CaoMV with the standard significance assessment procedure, the standard against which the other test-procedures are compared.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F4.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/12/3767/F4.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491526" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-100" class="first-child">Empirical power of mvQTL tests to detect mvQTL under four significance assessment procedures. Dot indicates point estimate and line indicates 95% confidence interval. Darker gray rectangle indicates the confidence band for the power of CaoMV with the standard significance assessment procedure, the standard against which the other test-procedures are compared.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h4>Testing mvQTL with BVH present: DGLM<sub>MV</sub> dominates Cao<sub>MV</sub>:</h4><p id="p-101">In the presence of BVH, Cao<sub>MV</sub> accurately controlled FPR with the RINT, residperm, and locusperm procedures, whereas DGLM<sub>MV</sub> did so only under the locusperm procedure (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-102">Of the test-procedure combinations that accurately controlled FPR, DGLM<sub>MV</sub>-locusperm was the most powerful with power of 0.790 as compared to the others in the range [0.632, 0.650].</p><p id="p-103">As with the vQTL tests, the DGLM<sub>MV</sub>-standard is attractive is terms of computational effort and good statistical properties, but it is expected to have drastically inflated FPR in the presence of any unmodeled BVH-driving factor, similar to Cao<sub>MV</sub>-standard. DGLM<sub>MV</sub>-locusperm therefore emerges as the most robust test-procedure for mvQTL testing.</p><p id="p-104">For each mvQTL test-procedure combination, the AUC (Table S1), standard error of the positive rate at <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-150.gif"/></noscript></span></span> (Table S2), QQ plots illustrating the empirical FPR at each nominal FPR level (Figure S6), and ROC curves illustrating the spectrum of trade-offs between available FPR and power (Figure S3) are provided in the Supplementary Materials.</p></div><div id="sec-32" class="subsection"><h4>In the presence of BVH, the rank-based inverse normal transformation fails to correct anti-conservative behavior of DGLMM and overcorrects that of DGLM<sub>V</sub> and DGLM<sub>MV</sub>:</h4><p id="p-105">A consistent feature of the simulations involving detection of variance effects, whether vQTL or mvQTL, is that FPR control and power is affected, for better or worse, by applying the RINT to the response.</p><p id="p-106">In the presence of BVH, DGLM<sub>M</sub> under the standard procedure was anti-conservative (FPR = 0.059 at <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-151.gif"/></noscript></span></span>). The RINT procedure had little efficacy in returning this test to accurate FPR control (FPR = 0.057).</p><p id="p-107">In the case of vQTL detection in the presence of BVH, Cao<sub>V</sub> under the standard procedure had a drastically inflated FPR (0.135) and the RINT procedure slightly over-corrected it (FPR = 0.046). Similarly, the RINT procedure disrupted DGLM<sub>V</sub>, which was modestly anti-conservative under the standard procedure, causing overly conservative behavior (FPR = 0.023).</p><p id="p-108">As always, in the presence of BVH, the mvQTL tests exhibited a mixture of the patterns observed in mQTL tests and vQTL tests. Both Cao<sub>MV</sub> and DGLM<sub>MV</sub> were anti-conservative under the standard procedure, illustrating their relations to Cao<sub>V</sub> and DGLM<sub>M</sub> respectively. In the case of Cao<sub>MV</sub>, the RINT procedure corrected the FPR, but in in the case of DGLM<sub>MV</sub>, it resulted in an over-correction into the realm of over conservatism (FPR = 0.049 and 0.038 respectively).</p><p id="p-109">In summary, the RINT procedure was unhelpful in the context of the DGLMM: it did not repair the modest FPR inflation present under the standard procedure. But, in the context of vQTL testing with BVH, it had one useful and important property: pre-processing the phenotype with the RINT, led to vQTL tests that were conservative rather than anti-conservative, decreasing the probability of false positives at the expense of false negatives.</p></div></div><div id="sec-33" class="subsection"><h3>Genomewide reanalysis of bodyweight in Leamy <em>et al.</em> backcross</h3><p id="p-110">To understand the impact of BVH on mean and variance QTL mapping in real data, we applied both traditional QTL mapping, using SLM, and mean-variance QTL mapping, using Cao’s tests and the DGLM, to body weight at three weeks in the mouse backcross dataset of <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Leamy <em>et al.</em> (2000)</a>.</p><div id="sec-34" class="subsection"><h4>Analysis with traditional QTL mapping identifies no QTL:</h4><p id="p-111">We first used a traditional, linear modeling-based QTL analysis, with sex and father as additive covariates and genomewide significance based on 1000 genome permutations (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Churchill and Doerge 1994</a>). Although sex was found not to be a statistically significant predictor of body weight (<span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-152.gif"/></noscript></span></span> by the likelihood ratio test with 1 degree of freedom), it was included in the mapping model because, based on the known importance of sex in determining body weight, any QTL that could only be identified in the absence of modeling sex effects would be highly questionable. Father was found to be a significant predictor of body weight in the baseline fitting of the SLM (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-153.gif"/></noscript></span></span> by the likelihood ratio test with 8 degrees of freedom) and therefore was included in the mapping model.</p><p id="p-112">No associations rose above the threshold that controls family-wise error rate to 5% (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, green line). One region on the distal part of chromosome 11 could be considered “suggestive” with FWER-adjusted <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-154.gif"/></noscript></span></span>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FWER-controlling association statistic at each genomic locus for body weight at three weeks. The linear model (green, &#x201C;traditional&#x201D;) does not detect any statistically-significant associations. The mQTL test takes into account the heterogeneity of both mean and variance due to which F1 male fathered each mouse in the mapping population and detects one mQTL on chromosome 11." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;FWER-controlling association statistic at each genomic locus for body weight at three weeks. The linear model (green, &#x201C;traditional&#x201D;) does not detect any statistically-significant associations. The mQTL test takes into account the heterogeneity of both mean and variance due to which F1 male fathered each mouse in the mapping population and detects one mQTL on chromosome 11.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F5.medium.gif" width="440" height="104"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/12/3767/F5.medium.gif" width="440" height="104"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491635" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-113" class="first-child">FWER-controlling association statistic at each genomic locus for body weight at three weeks. The linear model (green, “traditional”) does not detect any statistically-significant associations. The mQTL test takes into account the heterogeneity of both mean and variance due to which F1 male fathered each mouse in the mapping population and detects one mQTL on chromosome 11.</p><div class="sb-div caption-clear"></div></div></div><p id="p-114">To test the sensitivity of the results to the inclusion/exclusion of covariates, the analysis was repeated without sex as a covariate, without father as a covariate, and with no covariates. No QTL were identified in any of these sensitivity analyses.</p></div><div id="sec-35" class="subsection"><h4>Analysis with Cao’s tests identifies no QTL:</h4><p id="p-115">The same phenotype was analyzed with Cao’s tests, again including sex and father as mean covariates, and using the genome permutation procedures described in Statistical Methods were used to control FWER. No statistically significant mQTL, vQTL, nor mvQTL were identified (Figure S10).</p></div><div id="sec-36" class="subsection"><h4>Analysis with DGLM-based tests identifies an mQTL:</h4><p id="p-116">The same phenotype was analyzed with the DGLM-based tests. In a baseline fitting of the DGLM, sex was found not to be a statistically significant predictor of mean or residual variance (mean effect <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-155.gif"/></noscript></span></span>, variance effect <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-156.gif"/></noscript></span></span>, and joint <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-157.gif"/></noscript></span></span> by the LRT with 1, 1, and 2 d.f.). But father was found to be a statistically significant predictor of both mean and variance (mean effect <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-158.gif"/></noscript></span></span> variance effect <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-159.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-160.gif"/></noscript></span></span> by the LRT with 8, 8, and 16 d.f.). Therefore, following the same reasoning as in the mean model described above, both sex and father were included in the mapping model as covariates of both the mean and the variance. As with the other tests, the genome permutation procedures described in Statistical Methods were used to control FWER.</p><p id="p-117">A genomewide significant mQTL was identified on chromosome 11 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, blue line). The peak was at 69.6 cM with FWER-adjusted <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-161.gif"/></noscript></span></span> with the closest marker being D11MIT11 at 75.7 cM with FWER-adjusted <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-162.gif"/></noscript></span></span> Nonparametric bootstrap resampling, using 1,000 resamples (after <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Visscher <em>et al.</em> 1996</a>), established a 90% confidence interval for the QTL from 50 to 75 cM. This region overlaps with the “suggestive” region identified in the traditional analysis.</p><p id="p-118">By the traditional definition of percent variance explained, following from a fitting of the standard linear model, this QTL explains 2.1% of phenotype variance. Though, given the variance heterogeneity inherent in the DGLM that was used to detect this QTL, this quantity is better considered the “average” percent variance explained. The ratio of the QTL variance to the sum of QTL variance, covariate variance, and residual variance ranges from 1 to 6% across the population, based on the heterogeneity of residual variance.</p></div><div id="sec-37" class="subsection"><h4>Understanding the novel QTL:</h4><p id="p-119">The mQTL on chromosome 11 was identified by the DGLMM test, but not by the standard linear model or Cao’s mQTL test. The additional power of the DGLMM test over these other tests relates to its accommodation of background variance heterogeneity (BVH).</p><p id="p-120">Specifically, the DGLM reweighted each observation based on its residual variance, according to the sex and F1 father of the mouse. This BVH is visually apparent when the residuals from the standard linear model are plotted, separated out by father (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Residuals from the standard linear model for body weight at three weeks, with sex and father as covariates, stratified by father. It is evident that fathers differed in the residual variance of the offspring they produced. For example, the residual variance of offspring from fathers 1 and 2 is less than that of offspring from fathers 8 and 9. Here, points are colored by their predicted residual variance in the fitted DGLM with sex and father as mean and variance covariates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Residuals from the standard linear model for body weight at three weeks, with sex and father as covariates, stratified by father. It is evident that fathers differed in the residual variance of the offspring they produced. For example, the residual variance of offspring from fathers 1 and 2 is less than that of offspring from fathers 8 and 9. Here, points are colored by their predicted residual variance in the fitted DGLM with sex and father as mean and variance covariates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F6.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/12/3767/F6.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-121" class="first-child">Residuals from the standard linear model for body weight at three weeks, with sex and father as covariates, stratified by father. It is evident that fathers differed in the residual variance of the offspring they produced. For example, the residual variance of offspring from fathers 1 and 2 is less than that of offspring from fathers 8 and 9. Here, points are colored by their predicted residual variance in the fitted DGLM with sex and father as mean and variance covariates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-122">Some fathers, for example fathers 2 and 7, appear to have offspring with less residual variance than average, whereas others, for example father 1, seem to have offspring with more residual variance than average. The DGLM captured these patterns of variance heterogeneity, and estimated the effect of each father on the log standard deviation of the observations (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Based on these estimated variance effects, observations were upweighted (<em>e.g.</em>, fathers 2 and 7) and downweighted (<em>e.g.</em>, father 1). This weighting gave the DGLM-based mapping approach more power to reject the null as compared with the SLM.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The predictive mean and standard deviation of mice in the mapping population based on father and genotype at the top marker, D11MIT11 on chromosome 11. The genotype effect, illustrated by the colored ribbons is almost entirely horizontal, indicating a difference in means across genotype groups but no difference in variance, consistent with the identification of this QTL as a pure mQTL. The father effects, illustrated by the spread of colored crossbars, have both mean and variance components. For example, father 1 (red) has the highest predictive mean and lowest predictive standard deviation. His offspring were upweighted in the QTL analysis based on their low standard deviation. Father 9 (pink) has an average predictive mean and the highest predictive standard deviation. His offspring were downweighted in the QTL analysis based on their high standard deviation. Note: the effect of sex on phenotype mean and variance was modeled, then marginalized out for readability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1742452415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The predictive mean and standard deviation of mice in the mapping population based on father and genotype at the top marker, D11MIT11 on chromosome 11. The genotype effect, illustrated by the colored ribbons is almost entirely horizontal, indicating a difference in means across genotype groups but no difference in variance, consistent with the identification of this QTL as a pure mQTL. The father effects, illustrated by the spread of colored crossbars, have both mean and variance components. For example, father 1 (red) has the highest predictive mean and lowest predictive standard deviation. His offspring were upweighted in the QTL analysis based on their low standard deviation. Father 9 (pink) has an average predictive mean and the highest predictive standard deviation. His offspring were downweighted in the QTL analysis based on their high standard deviation. Note: the effect of sex on phenotype mean and variance was modeled, then marginalized out for readability.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/12/3767/F7.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/12/3767/F7.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/12/3767/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491539" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-123" class="first-child">The predictive mean and standard deviation of mice in the mapping population based on father and genotype at the top marker, D11MIT11 on chromosome 11. The genotype effect, illustrated by the colored ribbons is almost entirely horizontal, indicating a difference in means across genotype groups but no difference in variance, consistent with the identification of this QTL as a pure mQTL. The father effects, illustrated by the spread of colored crossbars, have both mean and variance components. For example, father 1 (red) has the highest predictive mean and lowest predictive standard deviation. His offspring were upweighted in the QTL analysis based on their low standard deviation. Father 9 (pink) has an average predictive mean and the highest predictive standard deviation. His offspring were downweighted in the QTL analysis based on their high standard deviation. Note: the effect of sex on phenotype mean and variance was modeled, then marginalized out for readability.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-38" class="subsection"><h4>Other phenotypes:</h4><p id="p-124">For brevity, we described in detail only the results of the DGLM-based analysis of body weight at three weeks; but, of the eight phenotypes from this cross available on the Mouse Phenome Database, the mean-variance approach to QTL mapping discovered new QTL in four. Five of the eight phenotypes — body weight at twelve days, three weeks, and six weeks, as well as subcutaneous and gonadal fat pad thickness — exhibited BVH due to father, and for each we performed both traditional QTL mapping using the SLM and mean-variance QTL mapping using the DGLM. This reweighting changed the results in three cases: For body weight at three weeks (Figure S15) and six weeks (Figure S16), we identified one new mQTL and two new vQTL respectively. For subcutaneous fat pad thickness, we discovered one mQTL and “undiscovered” one mQTL (Figure S17). That is, after reweighting the observations based on the observed variance of each father, one locus that was overlooked by SLM was identified as an mQTL and one locus that was identified by SLM as an mQTL was no longer found to have a statistically significant association with the phenotype.</p></div></div></div><div class="section discussion" id="sec-39"><h2 class="">Discussion</h2><p id="p-125">Since the recognition that variance effects can be attributable to individual genes, a growing body of research has asked questions about the prevalence of such effects (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Huang <em>et al.</em> 2015</a>), their evolutionary origins (canalization, robustness), their ramifications (decanalization in disease, increased variation) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gibson 2009</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Freund <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2016</a>), and how the identification of such genes can provide a signal of, and thereby serve as a route to identify, higher order interactions such as epistasis or GxE (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Struchalin <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rönnegård and Valdar 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Forsberg and Carlborg 2017</a>). These studies have promoted detection of variance heterogeneity as path to new biological discovery. But less attention has been paid to this corollary: if a phenotype is subject to variance-controlling factors, then, whether or not identifying those factors is of direct interest, they will induce background variance heterogeneity that can affect inference of more standard targets, including mean-affecting QTL. In other words, interest in identifying sources of BVH may be of most interest to a subset of researchers, but interest in accommodating it should be more widespread.</p><p id="p-126">Our simulation studies showed that modeling BVH when it is present increases power to detect mQTL, vQTL and mvQTL. Our reanalysis of the Leamy <em>et al.</em> dataset demonstrated that accommodating BVH can lead to detection of mQTL that would otherwise be overlooked.</p><p id="p-127">In both cases, of the methods compared, the most powerful were those based on the DGLM, with the most robust versions of those using the locusperm significance procedure. These results should not be too surprising. The DGLM was the only method examined that can accommodate variance effects arising from both the locus and from other covariates; and the locusperm method (and genomeperm, its genomewide analog) is least reliant on parametric assumptions. We would expect other methods that allow flexible modeling of covariate effects on variance to be competitive in these regards, <em>e.g.</em>, the recent Bayesian hierarchical model of <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Dumitrascu <em>et al.</em> (2018)</a>.</p><p id="p-128">Beyond advocating any particular method, however, our results can be used to draw attention to a number of more general points about 1) the relationship between increased residual variance, observation weighting and downstream inference; 2) how knowledge of variance effects can be exploited in experimental design, analysis and reanalysis; 3) the sensitivity of variance effect detection to distributional assumptions and how this can be mitigated by strategies such as variable transformation or permutation; and, 4) how to report quantitative genetic parameters under heteroskedasticity.</p><div id="sec-40" class="subsection"><h3>Residual variance, weighting, and inference for mean effect QTL</h3><p id="p-129">The additional power of mean-variance QTL mapping to detect mQTL in general—and of DGLMM to detect mQTL in the presence of BVH in particular—can be seen as deriving from how data are reweighted. Consider heteroskedastic data modeled as <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-163.gif"/></noscript></span></span>, with known weights <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-164.gif"/></noscript></span></span> and known baseline variance <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-165.gif"/></noscript></span></span>. The log-likelihood can be written as <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-166.gif"/></noscript></span></span>, such that the key quantity to be minimized in a maximum likelihood fit is<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-167.gif"/></noscript></span></span>the weighted residual sum of squares, that is, the squared discrepancies between the observed phenotype <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-168.gif"/></noscript></span></span> and its predicted value <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-169.gif"/></noscript></span></span>, weighted by <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-170.gif"/></noscript></span></span>. The weights therefore affect how much, relatively speaking, each data point contributes to the likelihood: highly imprecise measurements, such as from individuals whose phenotypes are expected to have high variance, have low weight and diminished contribution, whereas as more precise measurements are correspondingly upweighted. In the DGLM, the weight of each observation is determined in the model-fitting process based on the phenotype, the experimental covariates, and the QTL genotype. In the SLM, weights can be specified, but they cannot be co-estimated with covariate and QTL effects. The improvement of the DGLM over the SLM and Cao<sub>M</sub> under BVH stems entirely from its greater ability to capture this additional information, and thereby give more credence to phenotype values that are more precise.</p><p id="p-130">We note a related approach to correcting inference of mean effects in the face of heteroskedasticity not considered here is the use of heteroskedastic consistent covariance matrix estimators (HCCMEs) [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Long and Ervin (2000)</a> and refs therein]. Also known as “sandwich” estimators, these use estimated residuals from the SLM to characterize heteroskedasticity empirically and thereby estimate adjusted, heteroskedastic-consistent versions of the effect standard errors. Importantly, HCCMEs do not require the source of heteroskedasticity to be identified, and they have seen recent use in genetic association [<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barton <em>et al.</em> (2013)</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rao and Province (2016)</a>]. However, this comes at a cost: when a variable that does predict heteroskedasticity can be identified, HCCMEs will tend to be inefficient compared with a model-based estimator (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wakefield 2013</a>), such as the DGLM.</p></div><div id="sec-41" class="subsection"><h3>Implications for experimental design, analysis and reanalysis</h3><p id="p-131">The possibility that some individuals could be predictably more variable than others has clear implications for experimental design. A key parameter in the design of experiments is the number of replicates, typically specified to provide adequate precision of, and thereby power to detect, an estimated effect. But foreknowledge that residual variance will differ for certain groups suggests a more nuanced approach that explicitly weighs replicates against intrinsic variability.</p><p id="p-132">For example, when designing an experiment on a population that happens to have a known, segregating vQTL that is not itself the focus of interest but would induce BVH, it may be preferable to allocate a disproportionate share of the replication to individuals in the high-variability genotype class. In such cases, it then becomes additionally helpful to understand at what level(s) the heterogeneous variance manifests. Specifically, increased variability could arise from greater between-individual variation or greater within-individual variation [cf more levels of variability described in Table 1 of <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Rönnegård and Valdar (2011)</a>]; whereas the between-individual case warrants additional biological replicates, the within-individual case could be addressable (potentially more cheaply) with additional technical replicates.</p><p id="p-133">Alternatively, the recognition that some individuals are predictably high variance may be a reason to exclude them entirely, or, more generally, to opt for conditions and population subsets for which residual variance is predicted to be minimal. If such a variance-minimizing population can be achieved without changing the genetic effects present, it would have an improved signal-to-noise ratio and provide better power to detect genetic effects.</p><p id="p-134">A more standard situation is that a vQTL (or other BVH factor) is not recognized until the experiment is first analyzed. In this case, it would make sense to perform a re-analysis, with the vQTL included as a variance-affecting covariate. Doing so should increase power to detect both mQTL and other vQTL.</p></div><div id="sec-42" class="subsection"><h3>vQTL mapping: pros and cons of the rank inverse normal transformation</h3><p id="p-135">The presence of BVH can be disruptive to a test for a vQTL. A simplistic test compares a heteroskedastic alternative model with a homoskedastic null. BVH confuses the comparison by making the true null heteroskedastic. In doing so, it increases the false positive rate for asymptotic tests that disregard BVH and reduces power when FPR is empirically controlled (see, <em>e.g.</em>, Cao<sub>V</sub> results in <a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-136">In this context it is therefore interesting to consider the crude—but often used—device of the rank inverse normal transformation. The RINT reshapes away any kurtosis (fatter tails), a key signature of heteroskedasticity, without any reference to its source. As such, it is logical that in the detection of vQTL it would have both beneficial and harmful properties.</p><p id="p-137">In the case where there is no known driver of BVH, represented by the simulations examining Cao<sub>V</sub>, the RINT procedure acts as an insurance policy: if there truly is no BVH, the test suffers a modest decrease in power; but if there truly is BVH from an unknown source, it averts the drastic FPR inflation under the standard (<em>i.e.</em>, non-empirical) p-value procedure.</p><p id="p-138">In the case where researchers are confident that, after correcting for known BVH drivers, the residuals are homoskedastic (represented by the DGLMV simulations), the RINT procedure is unnecessary, costing power with its conservatism in the absence of BVH and paradoxically creating even more conservative behavior in the presence of BVH.</p><p id="p-139">The aforementioned disadvantages of RINT, however, assume the phenotype data has an underlying normal distribution, either as given or after a deducible transformation [<em>e.g.</em>, via the Box-Cox procedure or similar; (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Box and Cox 1964</a>)]. When the data are highly non-normal, both the RINT and the locusperm procedure would provide valid inference, and perhaps the most robust approach would be to use the two in combination. Nonetheless, where normality approximately holds, whether as given or after a simple transformation, we strongly prefer the locusperm procedure without RINT: across all simulation scenarios it exhibited at worst slight conservatism when applied to DGLM-based tests and represents a useful step toward FWER control.</p></div><div id="sec-43" class="subsection"><h3>Permutation schemes for other populations</h3><p id="p-140">Our preferred permutation scheme, locusperm (or its genomewide equivalent, genomeperm), is applicable to populations in which genotypes under the null are exchangeable. As such, it holds not only for F2 and backcrosses but also, for example, in approximately equally-related recombinant inbred line panels such as the Collaborative Cross and other similar replicable multiparent populations. For example, in the (mQTL) study of <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mosedale <em>et al.</em> (2017)</a>, the use of locus genotypes (or genotype probabilities) would simply be replaced by founder haplotypes that could then be randomly exchanged across lines.</p><p id="p-141">In non-exchangeable populations, however, such as those requiring polygenic random effect terms [<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kennedy <em>et al.</em> (1992)</a>], although the DGLM could be applied via its random effects generalization, DHGLM (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Felleki <em>et al.</em> 2012</a>), the permutation scheme may need revision. In particular, a permutation scheme in which all permutations are equally likely may not comport with a reasonable null, and it may be more appropriate to allocate higher probabilities to permutations that preserve overall genetic similarity (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abney 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Roach and Valdar 2018</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berrett <em>et al.</em> 2018</a>). Although we not have a specific solution, we suspect that the necessity of such revisions, at least for the DGLMV test, will depend on the extent to which the observed heteroskedasticity is polygenic.</p></div><div id="sec-44" class="subsection"><h3>Percent variance explained</h3><p id="p-142">Variance heterogeneity complicates the notion of percent variance explained (PVE) by a QTL. Assuming the QTL has the same effect on the expected value of the phenotype of all individuals, it will explain a larger percent of total variance for individuals with lower than average residual variance, and vice versa for individuals with higher than average residual variance. In light of this observation, the percent variance explained can either be reported as “average percent variance explained” or can be calculated for some representative sub-groups. For example, if there is variance heterogeneity across sexes, it would be reasonable to report the PVE of a QTL for both males and females, or if a vQTL is known to be present elsewhere in the genome, report the PVE for each vQTL genotype as in <a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> (2012)</a>.</p></div><div id="sec-45" class="subsection"><h3>Guidelines for detecting and QTL mapping in the presence of BVH</h3><p id="p-143">To select the right test and procedure to assess significance, it is important to establish whether there is any BVH present. We advocate fitting the DGLM with all potential BVH drivers as variance covariates, then including any that are statistically significant as variance covariates in the mapping model to improve power to detect QTL. Then, given that</p><ol class="list-ord " id="list-9"><li id="list-item-24"><p id="p-144">The DGLM-based tests dominate all other tests in the presence of BVH,</p></li><li id="list-item-25"><p id="p-145">the locusperm procedure accurately controls the FPR of the DGLM-based tests in the presence of BVH, whether the source is known or not, and</p></li><li id="list-item-26"><p id="p-146">the locusperm procedure can be extended into the genomeperm procedure to control FWER, </p></li></ol><p id="p-147">we advocate for the analysis of experimental crosses that exhibit BVH with the three DGLM-based tests (DGLMM, DGLMV, and DGLMMV) and, where the individuals in the population are exchangeable (as in an F2 or backcross) or where partial exchangeability can be suitably identified [<em>e.g.</em>, see (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Churchill and Doerge 1994</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zou <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Churchill and Doerge 2008</a>)], the use of our described genomeperm procedures, which permute the genome in selective parts of the model, to assess genomewide significance.</p><p id="p-148">Because this procedure involves three families of tests rather than one family as would be typical with an SLM-based analysis, an additional correction may be desired to control experiment-wise error rate. DGLMM and DGLMV are orthogonal tests (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Smyth 1989</a>), but DGLMMV is neither orthogonal nor identical to either, so the effective number of families is between two and three. One reasonable, heuristic approach to control experiment-wise error rate is simply to lower the acceptable FWER, <em>e.g.</em>, replacing the standard 0.05 with 0.02.</p></div><div id="sec-46" class="subsection"><h3>Conclusion</h3><p id="p-149">In summary, we demonstrate the effect of BVH on QTL mapping of both mQTL and vQTL, and the value of accommodating it using the DGLM. In doing so, we propose a standard protocol for mapping mQTL, vQTL and mvQTL in standard genetics crosses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-151">This work was funded by National Institutes of General Medical Sciences grants R01-GM104125 (RWC,WV), R35-GM127000 (RWC,WV), T32-GM067553 (RWC); National Heart, Lung and Blood Institute grant R21 HL126045 (RWC,WV); National Library of Medicine grant T32-LM012420 (RWC); and a National Institute of Mental Health grant F30-MH108265 (RWC). The authors thank Larry Leamy and two anonymous reviewers for helpful comments.</p><p id="p-152"><em>Note added in proof:</em> See Corty and Valdar 2018 (pp. 3757–3766) and Corty et al. 2018 (pp. 3783–3790) in this issue, for a related work.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX</h2><div class="section" id="sec-47"><h3>Calculation of an additive effect to explain a given proportion of total variance in an F2 intercross</h3><p id="p-150">The variance attributable to a genetic factor with alleles (AA, AB, BB) at frequency (0.25, 0.5, 0.25), additive effect <em>a</em> and no dominance effect is: <span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-171.gif"/></noscript></span></span> For a genetic factor that explains a fraction <em>p</em> of total phenotype variance: <span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-172.gif"/></noscript></span></span> Combining and solving for <em>a</em> gives <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/12/3767/embed/mml-math-173.gif"/></noscript></span></span>.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-153">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7290146">https://doi.org/10.25387/g3.7290146</a>.</p></li><li class="fn" id="fn-2"><p id="p-154"><em>Communicating editor: D. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-27"><span class="received-label">Received </span>February 27, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-10-28"><span class="accepted-label">Accepted </span>October 28, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Corty, Valdar</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.1" data-doi="10.1002/gepi.21893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Permutation testing in the presence of polygenic variation.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D39%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21893%26rft_id%253Dinfo%253Apmid%252F25758362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25758362&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.2" data-doi="10.1002/gepi.21716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamimi</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindström</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Nonparametric Test to Detect Quantitative Trait Loci Where the Phenotypic Distribution Differs by Genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D37%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21716%26rft_id%253Dinfo%253Apmid%252F23512279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23512279&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.3" data-doi="10.1073/pnas.1503830112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skutt-Kakaria</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Behavioral idiosyncrasy reveals genetic control of phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">6706</span>–<span class="cit-lpage">6711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1503830112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503830112%26rft_id%253Dinfo%253Apmid%252F25953335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY3MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.4" data-doi="10.1186/1471-2164-14-161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crozier</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lillycrop</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godfrey</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inskip</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Correction of unexpected distributions of P values from analysis of whole genome arrays by rectifying violation of statistical assumptions.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-161%26rft_id%253Dinfo%253Apmid%252F23496791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23496791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.5" data-doi="10.1007/s10519-009-9281-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beasley</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Public</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Building</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rank-based inverse normal transformations are Increasingly used, but are they merited?</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10519-009-9281-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DBeasley%26rft.auinit1%253DT.%2BM.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D580%26rft.epage%253D595%26rft.atitle%253DRank-based%2Binverse%2Bnormal%2Btransformations%2Bare%2Bincreasingly%2Bused%252C%2Bbut%2Bare%2Bthey%2Bmerited%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10519-009-9281-0%26rft_id%253Dinfo%253Apmid%252F19526352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10519-009-9281-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19526352&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268995200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-8.12.3767.6"><div class="cit-metadata unstructured">Berrett, T., Y. Wang, R. Foygel Barber, and R. Samworth, 2018 The conditional permutation test. ArXiv e-prints ArXiv:1807.05405.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Box</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">An analysis of transformations.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D26%26rft.spage%253D211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.8" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.9" data-doi="10.1080/01621459.1974.10482955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsythe</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Robust Tests for the Equality of Variances.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1974.10482955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D69%26rft.spage%253D364%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1974.10482955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1974.10482955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.10" data-doi="10.1002/gepi.21778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauwe</span>  <span class="cit-name-given-names">J. S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A versatile omnibus test for detecting mean and variance heterogeneity.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21778</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D38%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical Threshold Values for Quantitative Trait Mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.12" data-doi="10.1534/genetics.107.074609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Naive application of permutation testing leads to inflated type I error rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074609%26rft_id%253Dinfo%253Apmid%252F18202402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNjA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.13" data-doi="10.1093/biomet/26.4.404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clopper</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1934</span> <span class="cit-article-title">The Use of Confidence or Fiducial Limits Illustrated in the Case of the Binomial.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/26.4.404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DCLOPPER%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D404%26rft.epage%253D413%26rft.atitle%253DTHE%2BUSE%2BOF%2BCONFIDENCE%2BOR%2BFIDUCIAL%2BLIMITS%2BILLUSTRATED%2BIN%2BTHE%2BCASE%2BOF%2BTHE%2BBINOMIAL%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F26.4.404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/26.4.404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-article-title">Problems arising in the analysis of a series of similar experiments. Suppl.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%26rft.volume%253D4%26rft.spage%253D102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarantino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice. G3 GenesGenomesGenetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corty</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> vqtl: An R package for Mean-Variance QTL Mapping. G3 GenesGenomesGenetics (Bethesda) xxx-xxx.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.17" data-doi="10.1086/423738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudbridge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeleman</span>  <span class="cit-name-given-names">B. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/423738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDudbridge%26rft.auinit1%253DF.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D424%26rft.epage%253D435%26rft.atitle%253DEfficient%2Bcomputation%2Bof%2Bsignificance%2Blevels%2Bfor%2Bmultiple%2Bassociations%2Bin%2Blarge%2Bstudies%2Bof%2Bcorrelated%2Bdata%252C%2Bincluding%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F423738%26rft_id%253Dinfo%253Apmid%252F15266393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/423738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223272200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.18" data-doi="10.1093/bioinformatics/bty565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumitrascu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Statistical tests for detecting variance effects in quantitative trait studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.19" data-doi="10.1017/S0016672312000766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felleki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of breeding values for mean and dispersion, their variance and correlation using double hierarchical generalized linear models.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672312000766</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D94%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.20" data-doi="10.1371/journal.pgen.1005648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreatta</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danku</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salt</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Multi-allelic Genetic Architecture of a Variance-Heterogeneity Locus for Molybdenum Concentration in Leaves Acts as a Source of Unexplained Additive Genetic Variance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005648%26rft_id%253Dinfo%253Apmid%252F26599497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26599497&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.21" data-doi="10.1093/jxb/erx283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">On the relationship between epistasis and genetic variance heterogeneity.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">5431</span>–<span class="cit-lpage">5438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erx283</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft.volume%253D68%26rft.spage%253D5431%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferx283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erx283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.22" data-doi="10.1371/journal.pone.0008635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The quantitative genetics of phenotypic robustness.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e8635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0008635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFraser%26rft.auinit1%253DH.%2BB.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De8635%26rft.epage%253De8635%26rft.atitle%253DThe%2Bquantitative%2Bgenetics%2Bof%2Bphenotypic%2Brobustness.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008635%26rft_id%253Dinfo%253Apmid%252F20072615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20072615&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freedman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A nonstochastic interpretation of reported significance levels.</span> <abbr class="cit-jnl-abbrev">J. Bus. Econ. Stat.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBus.%2BEcon.%2BStat.%26rft.volume%253D1%26rft.spage%253D292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.24" data-doi="10.1126/science.1235294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freund</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandmaier</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewejohann</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirste</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kritzler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Emergence of individuality in genetically identical mice.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">759</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1235294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFreund%26rft.auinit1%253DJ.%26rft.volume%253D340%26rft.issue%253D6133%26rft.spage%253D756%26rft.epage%253D759%26rft.atitle%253DEmergence%2Bof%2BIndividuality%2Bin%2BGenetically%2BIdentical%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1235294%26rft_id%253Dinfo%253Apmid%252F23661762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDAvNjEzMy83NTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.25" data-doi="10.1038/nrg2502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Decanalization and the origing of complex disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGibson%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D134%26rft.epage%253D140%26rft.atitle%253DDecanalization%2Band%2Bthe%2Borigin%2Bof%2Bcomplex%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2502%26rft_id%253Dinfo%253Apmid%252F19119265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19119265&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263091500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.26" data-doi="10.1111/jeb.12906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitteroecker</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spritz</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic structure of phenotypic robustness in the collaborative cross mouse diallel panel.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1737</span>–<span class="cit-lpage">1751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D29%26rft.spage%253D1737%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3767.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Permutation tests: a practical guide to resampling methods for testing hypotheses</span>, <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.28" data-doi="10.1093/nar/gkt1159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mouse Phenome Database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D825</span>–<span class="cit-lpage">D834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1159</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1159%26rft_id%253Dinfo%253Apmid%252F24243846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24243846&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800121&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.29" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1992.131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.30" data-doi="10.1017/S0016672310000546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of environmental variation.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D381%26rft.epage%253D395%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Benvironmental%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000546%26rft_id%253Dinfo%253Apmid%252F21429270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429270&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3767.31"><div class="cit-metadata"><cite>Hong, C., Y. Ning, P. Wei, Y. Cao, and Y. Chen, 2016 <span class="cit-comment">A semiparametric model for vQTL mapping 73: 571–581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/biom.12612</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.32" data-doi="10.1073/pnas.1519159112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic basis of transcriptome diversity in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E6010</span>–<span class="cit-lpage">E6019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1519159112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519159112%26rft_id%253Dinfo%253Apmid%252F26483487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ0L0U2MDEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.33" data-doi="10.1534/genetics.112.146779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic variants contribute to gene expression variability in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.146779</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146779%26rft_id%253Dinfo%253Apmid%252F23150607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkzLzEvOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.34" data-doi="10.1038/ng.3928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivarsdottir</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinthorsdottir</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daneshpour</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulem</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effect of sequence variants on variance in glucose levels predicts type 2 diabetes risk and accounts for heritability.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1398</span>–<span class="cit-lpage">1402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1398%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.35" data-doi="10.2527/1992.7072000x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of effects of single genes on quantitative traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2000</span>–<span class="cit-lpage">2012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1992.7072000x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DKennedy%26rft.auinit1%253DB.%2BW.%26rft.volume%253D70%26rft.issue%253D7%26rft.spage%253D2000%26rft.epage%253D2012%26rft.atitle%253DEstimation%2Bof%2Beffects%2Bof%2Bsingle%2Bgenes%2Bon%2Bquantitative%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1992.7072000x%26rft_id%253Dinfo%253Apmid%252F1644672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1992.7072000x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1644672&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JC17400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.37" data-doi="10.1017/S0016672300004559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative trait loci for directional but not fluctuating asymmetry of mandible characters in mice.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300004559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DLeamy%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D40%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bfor%2Bdirectional%2Bbut%2Bnot%2Bfluctuating%2Basymmetry%2Bof%2Bmandible%2Bcharacters%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004559%26rft_id%253Dinfo%253Apmid%252F11006632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11006632&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089181400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.38" data-doi="10.1152/physiolgenomics.00018.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Pleiotropy of quantitative trait loci for organ weights and limb bone lengths in mice.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00018.2002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00018.2002%26rft_id%253Dinfo%253Apmid%252F12118102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00018.2002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12118102&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176749000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3767.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levene</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Robust tests for equality of variances</span>, pp. <span class="cit-fpage">278</span>–<span class="cit-lpage">292</span> in <span class="cit-source">Contributions to Probability and Statistics: Essays in Honor of Harold Hotelling</span>, <span class="cit-publ-name">Stanford Univ. Press</span>, <span class="cit-publ-loc">Palo Alto, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.40" data-doi="10.1534/g3.116.035444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Microenvironmental gene expression plasticity among individual drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4197</span>–<span class="cit-lpage">4210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.035444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035444%26rft_id%253Dinfo%253Apmid%252F27770026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTIvNDE5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTIvMzc2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.41" data-doi="10.2307/2685594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ervin</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Using Heteroscedasticity Consistent Standard Errors in the Linear Regression Model.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D54%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2685594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2685594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165952200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.42" data-doi="10.3389/fpls.2011.00012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makumburage</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phenotype uniformity in Combined-Stress environments has a different genetic architecture than in Single-Stress treatments.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2011.00012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DMakumburage%26rft.auinit1%253DG.%2BB.%26rft.volume%253D2%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DPhenotype%2BUniformity%2Bin%2BCombined-Stress%2BEnvironments%2Bhas%2Ba%2BDifferent%2BGenetic%2BArchitecture%2Bthan%2Bin%2BSingle-Stress%2BTreatments.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2011.00012%26rft_id%253Dinfo%253Apmid%252F22645526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2011.00012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645526&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.43" data-doi="10.1007/BF00222330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curnow</span>  <span class="cit-name-given-names">R. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222330%26rft_id%253Dinfo%253Apmid%252F24197463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24197463&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KE52200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.44" data-doi="10.1098/rsos.160352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fanson</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biro</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Towards powerful experimental and statistical approaches to study intraindividual variability in labile traits.</span> <abbr class="cit-jnl-abbrev">R. Soc. Open Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">160352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsos.160352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR.%2BSoc.%2BOpen%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.160352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveW9wZW5zY2kiO3M6NToicmVzaWQiO3M6MTE6IjMvMTAvMTYwMzUyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.45" data-doi="10.1093/toxsci/kfw269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mosedale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Editor’s highlight: Candidate risk factors and mechanisms for tolvaptan-induced liver injury are identified using a collaborative cross approach.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/toxsci/kfw269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfw269%26rft_id%253Dinfo%253Apmid%252F28115652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfw269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28115652&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.46" data-doi="10.1371/journal.pgen.1000981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridker</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chasman</span>  <span class="cit-name-given-names">D. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of variance per genotype as a tool to identify quantitative trait interaction effects: A report from the women’s genome health study.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000981</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPare%26rft.auinit1%253DG.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1000981%26rft.epage%253De1000981%26rft.atitle%253DOn%2Bthe%2Buse%2Bof%2Bvariance%2Bper%2Bgenotype%2Bas%2Ba%2Btool%2Bto%2Bidentify%2Bquantitative%2Btrait%2Binteraction%2Beffects%253A%2Ba%2Breport%2Bfrom%2Bthe%2BWomen%2527s%2BGenome%2BHealth%2BStudy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000981%26rft_id%253Dinfo%253Apmid%252F20585554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585554&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3767.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.48" data-doi="10.1002/gepi.21944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Province</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Framework for Interpreting Type I Error Rates from a Product-Term Model of Interaction Applied to Quantitative Traits.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">144</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D40%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-8.12.3767.49"><div class="cit-metadata unstructured">Roach, J., and W. Valdar, 2018 Permutation tests of non-exchangeable null models. ArXiv e-prints arXiv:1808.10483.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.50" data-doi="10.1534/genetics.111.127068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting major genetic loci controlling phenotypic variability in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127068%26rft_id%253Dinfo%253Apmid%252F21467569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.51" data-doi="10.1186/1471-2156-13-63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-13-63</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D63%26rft.epage%253D63%26rft.atitle%253DRecent%2Bdevelopments%2Bin%2Bstatistical%2Bmethods%2Bfor%2Bdetecting%2Bgenetic%2Bloci%2Baffecting%2Bphenotypic%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-63%26rft_id%253Dinfo%253Apmid%252F22827487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827487&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-8.12.3767.52"><div class="cit-metadata unstructured">Sachs, M. C., 2017 plotROC: A Tool for Plotting ROC Curves. J. Stat. Softw. 79: c02.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.53" data-doi="10.3389/fgene.2013.00093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Beware of risk for increased false positive rates in genome-wide association studies for phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2013.00093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2013.00093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2013.00093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.54" data-doi="10.1371/journal.pgen.1004842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jonge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural CMT2 Variation Is Associated With Genome-Wide Methylation Changes and Temperature Seasonality.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004842%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004842%26rft_id%253Dinfo%253Apmid%252F25503602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25503602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.55" data-doi="10.12688/f1000research.2-200.v1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronnegard</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Issues with data transformation in genome-wide association studies for phenotypic variability.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.2-200.v1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D2%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.2-200.v1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.2-200.v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Generalized linear models with varying dispersion. J. R. Stat.</span> <abbr class="cit-jnl-abbrev">Soc. Ser. B Methodol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSer.%2BB%2BMethodol.%26rft.volume%253D51%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.57" data-doi="10.1038/hdy.2015.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snell-Rood</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Life history as a constraint on plasticity: developmental timing is correlated with phenotypic variation in birds.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">388</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.47</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.58" data-doi="10.1016/j.ajhg.2015.05.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soave</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corvol</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panjwani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Joint Location-Scale Test Improves Power to Detect Associated SNPs, Gene Sets, and Pathways.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.05.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D97%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.05.015%26rft_id%253Dinfo%253Apmid%252F26140448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.05.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26140448&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.59" data-doi="10.1111/biom.12651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soave</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A generalized Levene’s scale test for variance heterogeneity in the presence of sample correlation and group uncertainty.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/biom.12651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D73%26rft.spage%253D960%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbiom.12651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/biom.12651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.60" data-doi="10.1017/S0016672303006426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waagepetersen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Normal linear models with genetically structured residual variance heterogeneity: a case study.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672303006426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSorensen%26rft.auinit1%253DD.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D222%26rft.atitle%253DNormal%2Blinear%2Bmodels%2Bwith%2Bgenetically%2Bstructured%2Bresidual%2Bvariance%2Bheterogeneity%253A%2Ba%2Bcase%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672303006426%26rft_id%253Dinfo%253Apmid%252F15134199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672303006426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15134199&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220642000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.61" data-doi="10.1534/genetics.115.180273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Control of Environmental Variation of Two Quantitative Traits of Drosophila melanogaster Revealed by Whole-Genome Sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180273%26rft_id%253Dinfo%253Apmid%252F26269504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.62" data-doi="10.1186/1471-2156-11-92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Struchalin</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehghan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witteman</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance heterogeneity analysis for detection of potentially interacting genetic loci: method and its limitations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-11-92</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DStruchalin%26rft.auinit1%253DM.%2BV.%26rft.volume%253D11%26rft.spage%253D92%26rft.epage%253D92%26rft.atitle%253DVariance%2Bheterogeneity%2Banalysis%2Bfor%2Bdetection%2Bof%2Bpotentially%2Binteracting%2Bgenetic%2Bloci%253A%2Bmethod%2Band%2Bits%2Blimitations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-92%26rft_id%253Dinfo%253Apmid%252F20942902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20942902&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.63" data-doi="10.1016/j.ajhg.2013.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">What is the significance of difference in phenotypic variability across SNP genotypes?</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.06.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.017%26rft_id%253Dinfo%253Apmid%252F23910463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910463&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.64" data-doi="10.1534/genetics.104.039313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.039313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039313%26rft_id%253Dinfo%253Apmid%252F16361245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE3ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.65" data-doi="10.1007/s10519-010-9362-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posthuma</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Statistical power to detect genetic loci affecting environmental sensitivity.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10519-010-9362-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D728%26rft.epage%253D733%26rft.atitle%253DStatistical%2Bpower%2Bto%2Bdetect%2Bgenetic%2BLoci%2Baffecting%2Benvironmental%2Bsensitivity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10519-010-9362-0%26rft_id%253Dinfo%253Apmid%252F20428936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10519-010-9362-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20428936&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Confidence intervals in QTL mapping by bootstrapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D143%26rft.issue%253D2%26rft.spage%253D1013%26rft.epage%253D1020%26rft.atitle%253DConfidence%2BIntervals%2Bin%2BQTL%2BMapping%2Bby%2BBootstrapping%26rft_id%253Dinfo%253Apmid%252F8725246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8yLzEwMTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-8.12.3767.67" data-doi="10.1007/978-1-4419-0925-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakefield</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Bayesian and Frequentist Regression Methods</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4419-0925-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2Band%2BFrequentist%2BRegression%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4419-0925-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4419-0925-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.68" data-doi="10.1534/genetics.113.157503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkmeyer-Langford</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Additive, epistatic, and environmental effects through the lens of expression variability QTL in a twin cohort.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.157503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157503%26rft_id%253Dinfo%253Apmid%252F24298061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvNDEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMi8zNzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.69" data-doi="10.1111/j.1365-2052.2012.02381.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arango</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association analysis and genetic architecture of egg weight and egg uniformity in layer chickens.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2012.02381.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D43%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2012.02381.x%26rft_id%253Dinfo%253Apmid%252F22742506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2012.02381.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22742506&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.70" data-doi="10.1111/j.1525-142X.2006.00120.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The contribution of epistatic pleiotropy to the genetic architecture of covariation among polygenic traits in mice.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">468</span>–<span class="cit-lpage">476</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1525-142X.2006.00120.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DWolf%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D468%26rft.epage%253D476%26rft.atitle%253DThe%2Bcontribution%2Bof%2Bepistatic%2Bpleiotropy%2Bto%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bcovariation%2Bamong%2Bpolygenic%2Btraits%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1525-142X.2006.00120.x%26rft_id%253Dinfo%253Apmid%252F16925682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1525-142X.2006.00120.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16925682&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239939800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.71" data-doi="10.1093/gbe/evw258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhole</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Differential regulation of cryptic genetic variation shapes the genetic interactome underlying complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3559</span>–<span class="cit-lpage">3573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.72" data-doi="10.1038/nature11401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FTO genotype is associated with phenotypic variability of body mass index.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D490%26rft.issue%253D7419%26rft.spage%253D267%26rft.epage%253D272%26rft.atitle%253DFTO%2Bgenotype%2Bis%2Bassociated%2Bwith%2Bphenotypic%2Bvariability%2Bof%2Bbody%2Bmass%2Bindex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11401%26rft_id%253Dinfo%253Apmid%252F22982992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982992&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309733300051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.73" data-doi="10.1017/S0021859600050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The analysis of groups of experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859600050978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D28%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600050978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.74" data-doi="10.1534/genetics.107.071365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1865</span>–<span class="cit-lpage">1877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.071365</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071365%26rft_id%253Dinfo%253Apmid%252F17483424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE4NjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.75" data-doi="10.1534/genetics.104.040386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bayesian model selection for genome-wide epistatic quantitative trait loci analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.040386</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.040386%26rft_id%253Dinfo%253Apmid%252F15911579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzEzMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.76" data-doi="10.1017/S0016672306007944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinniel</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartolucci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian analyses of multiple epistatic QTL models for body weight and body composition in mice.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672306007944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DYi%26rft.auinit1%253DN.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D60%26rft.atitle%253DBayesian%2Banalyses%2Bof%2Bmultiple%2Bepistatic%2BQTL%2Bmodels%2Bfor%2Bbody%2Bweight%2Band%2Bbody%2Bcomposition%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672306007944%26rft_id%253Dinfo%253Apmid%252F16545150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672306007944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16545150&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F12%2F3767.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.12.3767.77" data-doi="10.1534/genetics.106.059469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vision</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assessing the significance of quantitative trait loci in replicable mapping populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1068</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.059469</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059469%26rft_id%253Dinfo%253Apmid%252F16888335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8yLzEwNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEyLzM3NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/12/3767.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/12/3757" title="vqtl: An R Package for Mean-Variance QTL Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/12/3783" title="Mean-Variance QTL Mapping Identifies Novel QTL for Circadian Activity and Exploratory Behavior in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/12"><span>Volume 8 Issue 12, December 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/12.cover-source.jpg?itok=V2_jeDSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/12/3767" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y5PLDjQSonpbF_QlJlFKt1KqithZKs2sQNifbtTVJZc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/12/3767" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/12/3767" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/12/3767" class="active" data-icon-position="" data-hide-link-title="0">QTL Mapping on a Background of Variance Heterogeneity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491653" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ALAH3L5sTaLCFa8rZC-17PLykt5uCbbN1jXPhI6C1a0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3767.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/12/3767" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hfSnUlLwUS4LD4w-2igQiVMsVSLI8tFp7NymGFLakWs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/12/3767" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/12/3767.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491653" id="node491653--2" data-pisa="ggg;8/12/3767" data-pisa-master="ggg;g3.118.200790" data-apath="/ggg/8/12/3767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTL Mapping on a Background of Variance Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3767-3782;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200790 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491653--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/12/3767" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/12/3767&amp;text=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/12/3767&amp;t=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/12/3767&amp;title=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491653/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491653" id="node491653--3" data-pisa="ggg;8/12/3767" data-pisa-master="ggg;g3.118.200790" data-apath="/ggg/8/12/3767.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTL Mapping on a Background of Variance Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert W. Corty</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3767-3782;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200790 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491653--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert W. Corty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics and Computational Biology Curriculum</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BW.%2BCorty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corty%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BW.%2BCorty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6787-9051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert W. Corty</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3767&amp;rft.epage=3782&amp;rft.atitle=QTL%20Mapping%20on%20a%20Background%20of%20Variance%20Heterogeneity&amp;rft.volume=8&amp;rft.issue=12&amp;rft.date=2018-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Corty%2C+Robert+W.&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491653/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491653/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491653/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491653/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491653/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491653/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491653/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491653/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491653/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491653/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491653/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491653/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/12/3767" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Simulation Methods</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data and Software</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-39" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/12/3767.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/12/3767.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/12/3767.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/12/3767.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
