<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2463" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2463.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2463.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2463.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/11/2463/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2463.atom" />
<meta name="HW.pisa" content="ggg;5/11/2463" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" />
<meta name="DC.Identifier" content="10.1534/g3.115.020222" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Limborg et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic recombination is fundamental for generating new genetic variation and for securing proper disjunction. Further, recombination plays an essential role during the rediploidization process of polyploid-origin genomes because crossovers between pairs of homeologous chromosomes retain duplicated regions. A better understanding of how recombination affects genome evolution is crucial for interpreting genomic data; unfortunately, current knowledge mainly originates from a few model species. Salmonid fishes provide a valuable system for studying the effects of recombination in nonmodel species. Salmonid females generally produce thousands of embryos, providing large families for conducting inheritance studies. Further, salmonid genomes are currently rediploidizing after a whole genome duplication and can serve as models for studying the role of homeologous crossovers on genome evolution. Here, we present a detailed interrogation of recombination patterns in sockeye salmon ( Oncorhynchus nerka ). First, we use RAD sequencing of haploid and diploid gynogenetic families to construct a dense linkage map that includes paralogous loci and location of centromeres. We find a nonrandom distribution of paralogs that mainly cluster in extended regions distally located on 11 different chromosomes, consistent with ongoing homeologous recombination in these regions. We also estimate the strength of interference across each chromosome; results reveal strong interference and crossovers are mostly limited to one per arm. Interference was further shown to continue across centromeres, but metacentric chromosomes generally had at least one crossover on each arm. We discuss the relevance of these findings for both mapping and population genomic studies." />
<meta name="DC.Contributor" content="Morten T. Limborg" />
<meta name="DC.Contributor" content="Ryan K. Waples" />
<meta name="DC.Contributor" content="Fred W. Allendorf" />
<meta name="DC.Contributor" content="James E. Seeb" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic recombination is fundamental for generating new genetic variation and for securing proper disjunction. Further, recombination plays an essential role during the rediploidization process of polyploid-origin genomes because crossovers between pairs of homeologous chromosomes retain duplicated regions. A better understanding of how recombination affects genome evolution is crucial for interpreting genomic data; unfortunately, current knowledge mainly originates from a few model species. Salmonid fishes provide a valuable system for studying the effects of recombination in nonmodel species. Salmonid females generally produce thousands of embryos, providing large families for conducting inheritance studies. Further, salmonid genomes are currently rediploidizing after a whole genome duplication and can serve as models for studying the role of homeologous crossovers on genome evolution. Here, we present a detailed interrogation of recombination patterns in sockeye salmon (&lt;i&gt;Oncorhynchus nerka&lt;/i&gt;). First, we use RAD sequencing of haploid and diploid gynogenetic families to construct a dense linkage map that includes paralogous loci and location of centromeres. We find a nonrandom distribution of paralogs that mainly cluster in extended regions distally located on 11 different chromosomes, consistent with ongoing homeologous recombination in these regions. We also estimate the strength of interference across each chromosome; results reveal strong interference and crossovers are mostly limited to one per arm. Interference was further shown to continue across centromeres, but metacentric chromosomes generally had at least one crossover on each arm. We discuss the relevance of these findings for both mapping and population genomic studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2463" />
<meta name="citation_id" content="5/11/2463" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2463" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2463.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2463.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2463.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020222" />
<meta name="citation_pmid" content="26384769" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2463" />
<meta name="citation_lastpage" content="2473" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Morten T. Limborg" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="National Institute of Aquatic Resources, Technical University of Denmark, Vejlsøvej 39, Silkeborg, Denmark" />
<meta name="citation_author_email" content="morten.limborg@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7718-6531" />
<meta name="citation_author" content="Ryan K. Waples" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Fred W. Allendorf" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of Montana, Missoula, Montana 59812" />
<meta name="citation_author" content="James E. Seeb" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics of Fishes;citation_title=Tetraploidy and the evolution of salmonid fishes;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_pages=1-53;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Allendorf;citation_author=R. G. Danzmann;citation_title=Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout;citation_pages=1083-1092;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093860" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. W. Allendorf;citation_author=J. E. Seeb;citation_author=K. L. Knudsen;citation_author=G. H. Thorgaard;citation_author=R. F. Leary;citation_title=Gene-centromere mapping of 25 loci in rainbow trout;citation_pages=307-312;citation_volume=77;citation_year=1986;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=F. W. Allendorf;citation_author=S. Bassham;citation_author=W. A. Cresko;citation_author=M. T. Limborg;citation_author=L. W. Seeb;citation_title=Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.;citation_volume=106;citation_year=2015;citation_pmid=25838153;citation_doi=10.1093/jhered/esv015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Anderson;citation_title=CROSSING OVER IN A CASE OF ATTACHED X CHROMOSOMES IN DROSOPHILA MELANOGASTER;citation_pages=403-417;citation_volume=10;citation_year=1925;citation_issue=5;citation_pmid=17246282" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. H. Barton;citation_author=B. Charlesworth;citation_title=Why Sex and Recombination?;citation_pages=1986-1990;citation_volume=281;citation_year=1998;citation_issue=5385;citation_pmid=9748151;citation_doi=10.1126/science.281.5385.1986" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Basu-Roy;citation_author=F. Gauthier;citation_author=L. Giraut;citation_author=C. Mézard;citation_author=M. Falque;citation_title=Hot regions of noninterfering crossovers coexist with a nonuniformly interfering pathway in Arabidopsis thaliana.;citation_volume=195;citation_year=2013;citation_pmid=24026099;citation_doi=10.1534/genetics.113.155549" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. S. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_author=K. A. Naish;citation_title=A dense linkage map for Chinook salmon (Oncorhynchus tshawytscha) reveals variable chromosomal divergence after an ancestral whole genome duplication event.;citation_volume=4;citation_year=2014;citation_pmid=24381192;citation_doi=10.1534/g3.113.009316" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. W. Broman;citation_author=J. L. Weber;citation_title=Characterization of human crossover interference.;citation_pages=1911-1926;citation_volume=66;citation_year=2000;citation_issue=6;citation_pmid=10801387;citation_doi=10.1086/302923" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_author=L. B. Rowe;citation_author=G. A. Churchill;citation_author=K. Paigen;citation_title=Crossover Interference in the Mouse;citation_pages=1123-1131;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901128" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=M. C. Calderon;citation_author=M. D. Rey;citation_author=A. Cabrera;citation_author=P. Prieto;citation_title=The subtelomeric region is important for chromosome recognition and pairing during meiosis.;citation_pages=6488;citation_volume=4;citation_year=2014;citation_pmid=25270583;citation_doi=10.1038/srep06488" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. L. Campos;citation_author=D. L. Halligan;citation_author=P. R. Haddrill;citation_author=B. Charlesworth;citation_title=The relation between recombination rate and patterns of molecular evolution and variation in Drosophila melanogaster.;citation_volume=31;citation_year=2014;citation_pmid=24489114;citation_doi=10.1093/molbev/msu056" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Reproduction, nutrition, development;citation_journal_abbrev=Reprod Nutr Dev;citation_author=D. Chourrout;citation_title=Thermal induction of diploid gynogenesis and triploidy in the eggs of the rainbow trout (Salmo gairdneri Richardson).;citation_pages=727-733;citation_volume=20;citation_year=1980;citation_issue=3A;citation_pmid=7349440;citation_doi=10.1051/rnd:19800415" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. C. Colombo;citation_author=G. H. Jones;citation_title=Chiasma interference is blind to centromeres.;citation_pages=214-227;citation_volume=79;citation_year=1997;citation_pmid=9279014;citation_doi=10.1038/sj.hdy.6881930" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. F. Cooper;citation_title=Recombination speeds adaptation by reducing competition between beneficial mutations in populations of Escherichia coli.;citation_pages=e225-e225;citation_volume=5;citation_year=2007;citation_issue=9;citation_pmid=17713986;citation_doi=10.1371/journal.pbio.0050225" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. E. Cruickshank;citation_author=M. W. Hahn;citation_title=Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.;citation_pages=3133-3157;citation_volume=23;citation_year=2014;citation_pmid=24845075;citation_doi=10.1111/mec.12796" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. D. Cutter;citation_author=B. A. Payseur;citation_title=Genomic signatures of selection at linked sites: unifying the disparity among species.;citation_pages=262-274;citation_volume=14;citation_year=2013;citation_pmid=23478346;citation_doi=10.1038/nrg3425" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. L. Demarest;citation_author=W. H. Horsley;citation_author=E. E. Locke;citation_author=K. Boucher;citation_author=D. J. Grunwald;citation_title=Trans-centromere effects on meiotic recombination in the zebrafish.;citation_volume=187;citation_year=2011;citation_pmid=20980237;citation_doi=10.1534/genetics.110.124081" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=C. M. Elfstrom;citation_author=C. T. Smith;citation_author=J. E. Seeb;citation_title=Thirty‐two single nucleotide polymorphism markers for high‐throughput genotyping of sockeye salmon.;citation_pages=1255-1259;citation_volume=6;citation_year=2006;citation_doi=10.1111/j.1471-8286.2006.01507.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ellegren;citation_author=L. Smeds;citation_author=R. Burri;citation_author=P. I. Olason;citation_author=N. Backstrom;citation_title=The genomic landscape of species divergence in Ficedula flycatchers.;citation_pages=756-760;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23103876;citation_doi=10.1038/nature11584" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. V. Everett;citation_author=M. R. Miller;citation_author=J. E. Seeb;citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.;citation_pages=521-521;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23031582;citation_doi=10.1186/1471-2164-13-521" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Falque;citation_author=L. K. Anderson;citation_author=S. M. Stack;citation_author=F. Gauthier;citation_author=O. C. Martin;citation_title=Two types of meiotic crossovers coexist in maize.;citation_volume=21;citation_year=2009;citation_pmid=20040539;citation_doi=10.1105/tpc.109.071514" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. A. Gagnaire;citation_author=E. Normandeau;citation_author=S. A. Pavey;citation_author=L. Bernatchez;citation_title=Mapping phenotypic, expression and transmission ratio distortion QTL using RAD markers in the Lake Whitefish (Coregonus clupeaformis).;citation_pages=3036-3048;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12127" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=F. Gauthier;citation_author=O. C. Martin;citation_author=M. Falque;citation_title=CODA (crossover distribution analyzer): quantitative characterization of crossover position patterns along chromosomes.;citation_pages=27-27;citation_volume=12;citation_year=2011;citation_pmid=21251248;citation_doi=10.1186/1471-2105-12-27" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Giraut;citation_author=M. Falque;citation_author=J. Drouaud;citation_author=L. Pereira;citation_author=O. C. Martin;citation_title=Genome-wide crossover distribution in Arabidopsis thaliana meiosis reveals sex-specific patterns along chromosomes.;citation_pages=e1002354-e1002354;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072983;citation_doi=10.1371/journal.pgen.1002354" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Harris;citation_author=R. Nielsen;citation_title=Inferring demographic history from a spectrum of shared haplotype lengths.;citation_pages=e1003521;citation_volume=9;citation_year=2013;citation_pmid=23754952;citation_doi=10.1371/journal.pgen.1003521" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Hyma;citation_author=J. C. Fay;citation_title=Mixing of vineyard and oak-tree ecotypes of Saccharomyces cerevisiae in North American vineyards.;citation_pages=2917-2930;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12155" />
<meta name="citation_reference" content="citation_author=S. E. Johnston;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. B. Kaback;citation_title=Chromosome-size dependent control of meiotic recombination in humans.;citation_pages=20-21;citation_volume=13;citation_year=1996;citation_issue=1;citation_pmid=8673097;citation_doi=10.1038/ng0596-20" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. B. Kaback;citation_author=V. Guacci;citation_author=D. Barber;citation_author=J. W. Mahon;citation_title=Chromosome size-dependent control of meiotic recombination;citation_pages=228-232;citation_volume=256;citation_year=1992;citation_issue=5054;citation_pmid=1566070;citation_doi=10.1126/science.1566070" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Kirin;citation_author=R. McQuillan;citation_author=C. S. Franklin;citation_author=H. Campbell;citation_author=P. M. McKeigue;citation_title=Genomic runs of homozygosity record population history and consanguinity.;citation_pages=e13996-e13996;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21085596;citation_doi=10.1371/journal.pone.0013996" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Kodama;citation_author=M. S. Brieuc;citation_author=R. H. Devlin;citation_author=J. J. Hard;citation_author=K. A. Naish;citation_title=Comparative mapping between Coho Salmon (Oncorhynchus kisutch) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.;citation_volume=4;citation_year=2014;citation_pmid=25053705;citation_doi=10.1534/g3.114.012294" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. E. Koehler;citation_author=C. L. Boulton;citation_author=H. E. Collins;citation_author=R. L. French;citation_author=K. C. Herman;citation_title=Spontaneous X chromosome MI and MII nondisjunction events in Drosophila melanogaster oocytes have different recombinational histories.;citation_pages=406-414;citation_volume=14;citation_year=1996;citation_issue=4;citation_pmid=8944020;citation_doi=10.1038/ng1296-406" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=H. Komen;citation_author=G. H. Thorgaard;citation_title=Androgenesis, gynogenesis and the production of clones in fishes: A review.;citation_pages=150-173;citation_volume=269;citation_year=2007;citation_doi=10.1016/j.aquaculture.2007.05.009" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=N. E. Lamb;citation_author=S. L. Sherman;citation_author=T. J. Hassold;citation_title=Effect of meiotic recombination on the production of aneuploid gametes in humans.;citation_pages=250-255;citation_volume=111;citation_year=2005;citation_issue=3-4;citation_pmid=16192701;citation_doi=10.1159/000086896" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_author=F. G. Lhuissier;citation_author=H. H. Offenberg;citation_author=P. E. Wittich;citation_author=N. O. Vischer;citation_author=C. Heyting;citation_title=The mismatch repair protein MLH1 marks a subset of strongly interfering crossovers in tomato.;citation_pages=862-876;citation_volume=19;citation_year=2007;citation_doi=10.1105/tpc.106.049106" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=M. T. Limborg;citation_author=R. K. Waples;citation_author=J. E. Seeb;citation_author=L. W. Seeb;citation_title=Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.;citation_volume=105;citation_year=2014;citation_pmid=25149250;citation_doi=10.1093/jhered/esu060" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. R. Lindner;citation_author=J. E. Seeb;citation_author=C. Habicht;citation_author=K. L. Knudsen;citation_author=E. Kretschmer;citation_title=Gene-centromere mapping of 312 loci in pink salmon by half-tetrad analysis.;citation_pages=538-549;citation_volume=43;citation_year=2000;citation_issue=3;citation_pmid=10902719;citation_doi=10.1139/g00-016" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. K. Mable;citation_title=Polyploids and hybrids in changing environments: Winners or losers in the struggle for adaptation.;citation_pages=95-96;citation_volume=110;citation_year=2013;citation_pmid=23321773;citation_doi=10.1038/hdy.2012.105" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Mary;citation_author=H. Barasc;citation_author=S. Ferchaud;citation_author=Y. Billon;citation_author=F. Meslier;citation_title=Meiotic recombination analyses of individual chromosomes in male domestic pigs (Sus scrofa domestica).;citation_pages=e99123;citation_volume=9;citation_year=2014;citation_pmid=24919066;citation_doi=10.1371/journal.pone.0099123" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=K. Mather;citation_title=Crossing‐over.;citation_pages=252-292;citation_volume=13;citation_year=1938;citation_doi=10.1111/j.1469-185X.1938.tb00516.x" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_title=Meiotic models to explain classical-linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.;citation_author=B. May;citation_author=M. E. Delany;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Fish. Res. Board Can.;citation_author=B. May;citation_author=J. E. Wright;citation_author=M. Stoneking;citation_title=Joint segregation of biochemical loci in Salmonidae: Results from experiments with Salvelinus and review of the literature on other species.;citation_pages=1114-1128;citation_volume=36;citation_year=1979;citation_doi=10.1139/f79-156" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. R. Miller;citation_author=J. P. Brunelli;citation_author=P. A. Wheeler;citation_author=S. X. Liu;citation_author=C. E. Rexroad;citation_title=A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.;citation_pages=237-249;citation_volume=21;citation_year=2012;citation_issue=2;citation_pmid=21988725;citation_doi=10.1111/j.1365-294X.2011.05305.x" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=S. Ohno;citation_title=The enormous diversity in genome sizes of fish as a reflection of nature’s extensive experiments with gene duplication.;citation_pages=120-130;citation_volume=99;citation_year=1970;citation_doi=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. P. Otto;citation_author=T. Lenormand;citation_title=Resolving the paradox of sex and recombination.;citation_pages=252-261;citation_volume=3;citation_year=2002;citation_issue=4;citation_pmid=11967550;citation_doi=10.1038/nrg761" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=C. Parisod;citation_author=R. Holderegger;citation_author=C. Brochmann;citation_title=Evolutionary consequences of autopolyploidy.;citation_pages=5-17;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20070540;citation_doi=10.1111/j.1469-8137.2009.03142.x" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. Phillips;citation_author=P. Rab;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Phillips;citation_author=K. Nichols;citation_author=J. DeKoning;citation_author=M. Morasch;citation_author=K. Keatley;citation_title=Assignment of rainbow trout linkage groups to specific chromosomes.;citation_volume=174;citation_year=2006;citation_pmid=16951085;citation_doi=10.1534/genetics.105.055269" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Pool;citation_author=R. Nielsen;citation_title=Inference of historical changes in migration rate from the lengths of migrant tracts.;citation_volume=181;citation_year=2009;citation_pmid=19087958;citation_doi=10.1534/genetics.108.098095" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. C. Presgraves;citation_title=Recombination enhances protein adaptation in Drosophila melanogaster.;citation_pages=1651-1656;citation_volume=15;citation_year=2005;citation_issue=18;citation_pmid=16169487;citation_doi=10.1016/j.cub.2005.07.065" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Renaut;citation_author=C. Grassa;citation_author=S. Yeaman;citation_author=B. Moyers;citation_author=Z. Lai;citation_title=Genomic islands of divergence are not affected by geography of speciation in sunflowers.;citation_pages=1827;citation_volume=4;citation_year=2013;citation_pmid=23652015;citation_doi=10.1038/ncomms2833" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Rockmill;citation_author=K. Voelkel-Meiman;citation_author=G. S. Roeder;citation_title=Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in Saccharomyces cerevisiae.;citation_volume=174;citation_year=2006;citation_pmid=17028345;citation_doi=10.1534/genetics.106.058933" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Roesti;citation_author=D. Moser;citation_author=D. Berner;citation_title=Recombination in the threespine stickleback genome—patterns and consequences.;citation_pages=3014-3027;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12322" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=J. E. Seeb;citation_author=L. W. Seeb;citation_title=Gene mapping of isozyme loci in chum salmon;citation_pages=399-402;citation_volume=77;citation_year=1986;citation_issue=6;citation_pmid=3559166" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Lond. B Biol. Sci.;citation_author=J. Segura;citation_author=L. Ferretti;citation_author=S. Ramos-Onsins;citation_author=L. Capilla;citation_author=M. Farré;citation_title=Evolution of recombination in eutherian mammals: insights into mechanisms that affect recombination rates and crossover interference.;citation_pages=280;citation_volume=20131945;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. G. Storer;citation_author=C. E. Pascal;citation_author=S. B. Roberts;citation_author=W. D. Templin;citation_author=L. W. Seeb;citation_title=Rank and Order: Evaluating the Performance of SNPs for Individual Assignment in a Non-Model Organism.;citation_pages=e49018-e49018;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185290;citation_doi=10.1371/journal.pone.0049018" />
<meta name="citation_reference" content="citation_journal_title=Z. Indukt. Abstamm. Vererbungsl.;citation_author=A. H. Sturtevant;citation_title=The behavior of the chromosomes as studied through linkage.;citation_pages=234-287;citation_volume=13;citation_year=1915" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. B. Talbert;citation_author=S. Henikoff;citation_title=Centromeres convert but don’t cross.;citation_pages=523;citation_volume=8;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Canadian journal of genetics and cytology.  Journal canadien de gÃ©nÃ©tique et de cytologie;citation_journal_abbrev=Can J Genet Cytol;citation_author=G. H. Thorgaard;citation_title=Sex chromosomes in the sockeye salmon: a Y-autosome fusion.;citation_pages=349-354;citation_volume=20;citation_year=1978;citation_issue=3;citation_pmid=570442;citation_doi=10.1139/g78-039" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. H. Thorgaard;citation_author=F. W. Allendorf;citation_author=K. L. Knudsen;citation_title=GENE-CENTROMERE MAPPING IN RAINBOW TROUT: HIGH INTERFERENCE OVER LONG MAP DISTANCES;citation_pages=771-783;citation_volume=103;citation_year=1983;citation_issue=4;citation_pmid=17246124" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Tine;citation_author=H. Kuhl;citation_author=P.-A. Gagnaire;citation_author=B. Louro;citation_author=E. Desmarais;citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.;citation_pages=5770;citation_volume=5;citation_year=2014;citation_pmid=25534655;citation_doi=10.1038/ncomms6770" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. K. Waples;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (Oncorhynchus keta).;citation_year=2015;citation_doi=10.1111/1755&amp;ndash;0998.12394" />
<meta name="citation_reference" content="citation_title=Isozymes: Current Topics in Biological and Medical Research;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes;citation_author=J. E. Wright;citation_author=K. Johnson;citation_author=A. Hollister;citation_author=B. May;citation_pages=239-260;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Zhao;citation_author=T. P. Speed;citation_title=On Genetic Map Functions;citation_pages=1369-1377;citation_volume=142;citation_year=1996;citation_issue=4;citation_pmid=8846913" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic recombination is fundamental for generating new genetic variation and for securing proper disjunction. Further, recombination plays an essential role during the rediploidization process of polyploid-origin genomes because crossovers between pairs of homeologous chromosomes retain duplicated regions. A better understanding of how recombination affects genome evolution is crucial for interpreting genomic data; unfortunately, current knowledge mainly originates from a few model species. Salmonid fishes provide a valuable system for studying the effects of recombination in nonmodel species. Salmonid females generally produce thousands of embryos, providing large families for conducting inheritance studies. Further, salmonid genomes are currently rediploidizing after a whole genome duplication and can serve as models for studying the role of homeologous crossovers on genome evolution. Here, we present a detailed interrogation of recombination patterns in sockeye salmon ( Oncorhynchus nerka ). First, we use RAD sequencing of haploid and diploid gynogenetic families to construct a dense linkage map that includes paralogous loci and location of centromeres. We find a nonrandom distribution of paralogs that mainly cluster in extended regions distally located on 11 different chromosomes, consistent with ongoing homeologous recombination in these regions. We also estimate the strength of interference across each chromosome; results reveal strong interference and crossovers are mostly limited to one per arm. Interference was further shown to continue across centromeres, but metacentric chromosomes generally had at least one crossover on each arm. We discuss the relevance of these findings for both mapping and population genomic studies." />
<meta name="og-title" property="og:title" content="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2463" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic recombination is fundamental for generating new genetic variation and for securing proper disjunction. Further, recombination plays an essential role during the rediploidization process of polyploid-origin genomes because crossovers between pairs of homeologous chromosomes retain duplicated regions. A better understanding of how recombination affects genome evolution is crucial for interpreting genomic data; unfortunately, current knowledge mainly originates from a few model species. Salmonid fishes provide a valuable system for studying the effects of recombination in nonmodel species. Salmonid females generally produce thousands of embryos, providing large families for conducting inheritance studies. Further, salmonid genomes are currently rediploidizing after a whole genome duplication and can serve as models for studying the role of homeologous crossovers on genome evolution. Here, we present a detailed interrogation of recombination patterns in sockeye salmon ( Oncorhynchus nerka ). First, we use RAD sequencing of haploid and diploid gynogenetic families to construct a dense linkage map that includes paralogous loci and location of centromeres. We find a nonrandom distribution of paralogs that mainly cluster in extended regions distally located on 11 different chromosomes, consistent with ongoing homeologous recombination in these regions. We also estimate the strength of interference across each chromosome; results reveal strong interference and crossovers are mostly limited to one per arm. Interference was further shown to continue across centromeres, but metacentric chromosomes generally had at least one crossover on each arm. We discuss the relevance of these findings for both mapping and population genomic studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6s2eIcCDblEy8rjeiy_8Qc5MzCr2N8bqhlFvKSqRY5U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460182","apath":"\/ggg\/5\/11\/2463.atom","pisa":"ggg;5\/11\/2463","ac":{"\/ggg\/5\/11\/2463.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2463.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2463"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2463"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2463.full":true,"\/content\/5\/11\/2463":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZLzCpY9f4ZqJwG+Pgs3NiDcsYUCNsegllwa1S\\\\\\\/qI04OKCWbeO+dXyLyURNqumFPVSvBRfmReKw3tJq01i3G8QBENKcW15uBwC2hElu1W3qF4aFwA01umv\\\\\\\/Vf8WkwvBeNwOx2YAiF1X9b9I4h1JqKzX\\\\\\\/DRiA9ID7yh9IkLx+aHyycrvtCosOSXfrxWRNTohrS47q8NAh81CO90AkRDekmOnxHxihBIQ1\\\\\\\/JlXf0TEWzavaycSu4rsTPLnkJ5p9MxpY2S6sfK7oQ8PXqJtgEsJOKbRd0A5EzlK\\\\\\\/DOUidEYYuxw8WNsrvMkrLuUEYQ66W6JEGwNOohbFAI6jd9m2LI88TdgCvERfzH5yMpxYm46XbUAAEKpZIZ8BEUwbQFN4sigsFQGDMdqoBRcjroaJbRjwq8dyrq8exPedoD+Nwo8bPnvGtYtk6uD0MPfcfQcQjinHlmDpP75v2xJxSGMtZZfp1HfVXGXCWnMLflZEuLpPS84MrcjZ1d9Ke6CPa0tFKMSx1PpHEwdEMNWoXQDAiEXediDYNLYPPOLauqdyZQ8tmb7RSP4TktaGjfjV8+zJYxiyra7j6PVyFS1XimRVRlNm4rD7f\\\\\\\/v\\\\\\\/gZ4zqCGhoh44aEQG7CXsMFfQK+3j1e9Y8UZ49hxpAOPyKSbzp2lZ4o63rpAOYEuXizcX0KkNXASOnu9TyOZjPtuIXJ79Q+X\\\\\\\/mv\\\\\\\/15bPBAFT99+5\\\\\\\/EQ2MAJ0tfiiiK+A2+IJyv8LkkTr2MD5AuvhRISLnY8zOjH6ln4P\\\\\\\/sg4fv4vN4kQiPQQFK3bJ0iDZKVa+iVGtQFEGyBa1pbJ6BTJEFtUVznSdElutS4vkxeQxUKS486AFK4BE+IdFTjfdlafyh1luwD4l6cxm1cUGSvK6mJ2Vv\\\\\\\/1LzKYZIoWPXJCjXAt\\\\\\\/f4GuWxMVUilpk\\\\\\\/oF8esLVFzDzBwcRjmbysRxI3XP0ryfR\\\\\\\/hAAp07G6\\\\\\\/uW8\\\\\\\/tpBc12H\\\\\\\/wl\\\\\\\/+DVNe12sng+yAqQ\\\\\\\/4c0kGqHsZc8AAX7RbhqR0z\\\\\\\/PQdt6TFduq8OyYl77NN\\\\\\\/ShMy1RNaHTF8v3ir9mWd3PDVb8lWBNMROn1FDyWlHNKDit0LgczVKw8rSk0v8z606+FCvcRke+\\\\\\\/AAIAbIfencOCipAuGF85IEKMt1ajzp8vl4o8OImpILSf8hy\\\\\\\/aCykXFAaoZIprGMNePG4xNiGDJNApqGQ5gW16aIOdorpIxkicnNAVDSYQ\\\\\\\/Co3\\\\\\\/R3Wqe3l+UCAh9LFsYljOACUHp3oiPuVC5qCH7qej9DgkMlrg3IDQpQYiLwCNSB\\\u0022,\\\u0022iv\\\u0022:\\\u0022JBuFVjs2M9LHTp\\\\\\\/3uN\\\\\\\/QQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b6811bf82831f5876948e169836f4b583ea09524626525de5fc5964696784ad3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222Os1yxz1wOALZPLA4BjmCZtM\\\\\\\/l4Tu5+IH\\\\\\\/klv1u73pMqBf2eKoINUqOIRmRj0ROajRBeT\\\\\\\/xLWwGvWrvssqzpL2FR5AjKWQYgUDXYj3qnliVhWklzb2YKwpDs3FH1tLOH\\\\\\\/4e0KJXPT7AH8BVP8a7fq2nhblr0fiPUE93yUeDWUbQZC7+LODd\\\\\\\/8WsnHKFQhQZnInZOVLlcgWpGv38uVYudUF+k30zdU+UAbcVqs4PVv+wbU8JOB8+\\\\\\\/6R56ypxEhAswUVh7ZMxsdLuID0CTSaom8in6mvnksMBD2gMQI4KUEkZqn\\\\\\\/CeVP\\\\\\\/Q0lYCOw6yFT7KR+cc0UsJgAmRsnrhxsG5kLWPI1fmYwf7OXpKTrbKCahMeOO9UWPjrxN3GkuicKN+8LRXDstKButjUHvtCmH9uesraANBQ4cOQEx4tJnKh6OWxDqnU\\\\\\\/hdmIlYBLypIiMh37jiw6fPY\\\\\\\/V6z0Kz9rjZ+fVmI1xyv91+M+LFPhMOhMciDJosiH0x53DR4afF3Y0IvimUmuYargUxCQqSoJruqCGMb2YtjKY4y9EB5nliABG+DooIHdIv0UCfzgg0xYgtjEhI5kSF5BjlQmkSzu\\\\\\\/EYkCg00HhcRsEKxS3oj+N7xz9hKkNtrNG8OSf0u+yZMj2y+0EGeoDBcqfyTPusxsCpv7UjnZtd3hzkV5u6QVP6+CbHzPce2TZKnYkcRT\\\\\\\/7vd2wUJdmWo4dGM9+Fa2Pgc7144jSJ4d6ChDzc3MqiV9hr+6MQc50T\\\\\\\/CUrnVbx8sbGgp2WXj1JM+AhykobnDuf1n0SsGdI\\\\\\\/8dSTmWZtbsdiW8XZHsE1iyGRZcfz7rETPikrFtNl0dwJ14owJP\\\\\\\/9+4T4DmJeZ9JfACUyoJAm47OcPLAkV9x\\\\\\\/ByzXYXzcebrNlGZtfSLCzpNX1U8qE5XXtvXnWC\\\\\\\/ApZ93wa6oexs+gMQsMDL\\\\\\\/NNkn01gjzebKsujXUISSYwlPnfnShhW1\\\\\\\/7anV+KE\\\\\\\/SGFkAIYB\\\\\\\/uEhM40pTeLYSaLgL0g7hrSghk4EaTVCukSFsBGFebxs0RmwIytptowVYVTqJm1suUsT5EDuYu7OzkyoO59+l30pdldbzsUGafEO8HgACkTR9sCeJAiNw5Oj753UviuPHgvynpd6wAitoI\\\\\\\/PZc+0Z53E3IjSSMCNpH8n2VL\\\\\\\/fKpYPYlTr07WdfIX7CjCSCCAnSNCtdit\\\\\\\/C+\\\\\\\/80sfWrEzs9dUod7rR9PV6yES3iF9hwbKeaodmhCLqPrgkm\\\\\\\/tD1h+RLpLukAS6NBIMOVcC8Thlfyaelstk3o0ieC5diFQpXJS6S8vgU1RedkhJVunFgnpxWvo4Cq+Ux7RFxzqOD7NiFA4r0b2f8jPn60T+Pgyr\\\\\\\/HNip9KIFr4tfp55UfgLxydBPNbSNnjS2CcXGVULi7MRDydBspq1wyPNALEFASc2u2awgkfsMI1IQSJ+iV2xg17Grf+X\\\\\\\/fKb1jPW5Dq+h5KOVIoXYjNBDxw7lAtzUq3SVtVdCnRIbk0MSW9dWGXNtOC\\\\\\\/p2i6kX33kR4nGBC8UXm5oaqjSFII5rWzlCVIUJmlrPTr9yxbapBqjf\\\\\\\/sw+ZWsCSwrafcXkTmFy6m9X5apqW3dvtgNtWCBPq8yAnl0ejh8SXflCGArIh2dSjeXKm8SS\\\\\\\/dNQScc4L84uMs6XhneesEl22DcnOef732V\\\\\\\/YsnABBCfFYGeZKzun+vHm\\\\\\\/Hk28++BOfrvhhFop6Ze+uu5OGy\\\\\\\/rmGkdZt\\\\\\\/DUvCqvKedjePLSJzps4JeUoNh3QALTGVN0wy2XhttFi3CeEepgJ2SWjfk0KP8STwvXNuiz1nty1Zki63Zq+NSyhEMjBDFh+zpox5k0RwqkAtJK18XoQpR1m1Xmx+4nS+g2TSj3xvfeOdJJsXe9o33lwxaXiPXUCjU\\\\\\\/CPp6163uCTVFV\\\\\\\/QfxQmp4EsVz4OBlhgFoKZe\\\\\\\/5+I0lRQyljJivEFMrbpZn5MjB\\\\\\\/djwB9uF7vE38aqvjdOoaZHbGzc13sax+FwBEFlW1QD2v\\\\\\\/+xrZ\\\\\\\/zTnT524yKrPfQ633MlX9Y2A8rCWKM3r4yKWZZuWXWLjZI3ONGPeKvZ\\\\\\\/9qqPFhDeBfUiqu23ZCLK+ya\\\\\\\/7v4rFywRzxINSs8ZUJtWti8sNjsd2n9tyc51vzdrDqqv134VrZxC5rsbnd8MAs04\\\\\\\/ySiT22htDkxRib\\\\\\\/aRgJVUvQXqcxao1To\\\\\\\/Yg0ehii82orPfiG0AmNtJIA7wrFShCSqSUcnLJE45tWlRiy2dH3ft54jyeHFCdfrzZP1Ro\\\\\\\/awqmC3Jzph+26GAR7DH10a63MeLe0Dec3wAbc7agHVqN3HYpxXhKeKNxUxKinAODp1sIVfs\\\\\\\/Hpo99MT938XdPqEedrUhSwmvkEawBmEa+8NDXv\\\\\\\/pz1chx44hUbZIOa3t\\\\\\\/0qHBUktDDbCi8Fj6Yoo\\\\\\\/EKseCJAW+CDbjkfurwSigTgVOsUzvI4vUqyPANurwY8DNkPq5CL+090lVfiH81UGzealJu2pKqAZAKz8bjNrUZFz6I0uoM3sZxwPmKoIMDxVlN\\\\\\\/OWflPM4VbEUT0VanA6Cl5JdLoue3e+xiOdXUqyhBU0Ph4Vdz2nfM+o0q5SHDtHtzbOOo9oOtC2rTVimhnnmJ4QoSHZvfQccz8uiGsT2JhR3C\\\\\\\/GEzOoePY07B6QRCOhelkGQj9QiJx\\\\\\\/rAlhyC6ql+EfTs\\\\\\\/eLpYCRZCOgfVFbzITcyO57jYhkmv4G0mRPYFBTggXpfdqs7kaKIFBRnqOjG46OEsRluYo8OhVod858+lXcBzYkNcGG9E67fk2MBHwGXbJq9QQcSDQDXJTeDiNQ3gxfjex0dY6vj8GdIZCBjdeRFzC3pnX6CsiPhdoDeUL7EzTt23sg1CjI4cPbJqx7mNbzP8t5y\\\\\\\/5iUVBtnpR0mNfeyz0dDXjpkdSU3d7flfbnzPA989bmCXfK9lLpBz3v58TlLrh9w0cVHgXbS+pOgxHhmM+EV\\\\\\\/zOzrXbfuNi9dSiC9kThKsxuRvOn92DkwXDewhWC0vrRxxaCVblRRWcuKAMPsvLA167pUaKx0r0u6zlTnwccGvwzlvHhEGR54JXAKpX58wc7U2WfILIuvQqRBG3YIy989SVlminwyJGX7JncKirbq5Y2nK024GO0JfJhkIMp48e4ZfULWfI21d5Zc2I5SqSSaAkoxezDu8gzWZrC9CFRxyGwX1h2Lq1CFCiIJW8PFBVlwh9NyCJ4nMLUkI5sMEZlV8mDY59VfmLRz4SSghBbJaFxvEPSIFiMZSbjgiq5wRpKSu\\\\\\\/f0T+LMzqXAYNK953yUfjoMW\\\\\\\/mTulAVM7AaFwu6+fWXkxNxwiLXp5Prd5mGJ5vDjPLmW\\\\\\\/jvAloj60psAKuiV4gCmPXAxjDSNwrep89kFgxtQVoOEUAQh48y\\\\\\\/vsjEb8lBy1UnQbJvBvKZG3d7LrguJnqdZlNlfmdLriX6tvZEdT63BaKLIT7nWFe5W4loY0ku6wGDGJ\\\\\\\/UmCxpApOxQXskqDeuWT2OemKyiqYiSbHRCZqi0daMsj03m2oeJg+VuGwfWGzbr6LclCd6rVhTtPqk6rSS0ezgMUQyq8+pvtaAhErY3xnUpu\\\\\\\/6ZEtlUNXi0bs62wibsas0RwJcezwdRE49+ajKJ+spfsD15NvftNeGVvvBuvp9Rp0VcyrWaFmbcrd73bn+Zv+Dmjt7vJkn7aOnKhZqmrFXDnYCB1pa4TjeQSBXJK0elkiIHBZZbEwWT3ZUuuHphrrffQGu8EGZ\\\\\\\/niR2RUprIaTjzCszYRcnch3cYMUvFrS9PWO8qfiGgX1xpxz4BBvy+mICp9BIUw29t\\\\\\\/expgZJn\\\\\\\/X6lohW2L1CkFw\\\\\\\/TRFzEZZQ+TzqQZgWjMCbHh8wVi\\\\\\\/Q0Tatp69WyPoAeLWH3df7xI3JWLlT9YGLfL+8kBg82xPM3AZ+C305qLCF6Y2czwAfWqyVphwfuvgDlhCnIhwiMNj5d2omGBCa8ln2tvptHBB7UzqKT2XdRZt4\\\\\\\/wAYXWhPaRNHHMj34w68LsGbb89nYCy1B6Gg5l5GG3fZjQB23\\\\\\\/kaXJ4+8FzI+XG\\\\\\\/pImxe5gYRnf14gm+i0uKp0w5YVnfNa\\\\\\\/WGP7UwxZf5nX2Xl\\\\\\\/DTIPiNfKwMPJDVPK5IjhLPm77X8WymFqrZwZxRros\\\\\\\/auYRs35BJZFCf46TKpZ1ragKi6fG\\\\\\\/V\\\\\\\/OT0rtht8Hy1o6llshfkiXMPUmyQsw4m6NJA5+uDunfiEft97tmpx+3zXoGNAhv1RU\\\\\\\/NSVHThOl3YJhkFInmMWTU6NRHqDbbU11nNLMfB72UkeqEsBJWxH7cJn9G2Rvi2rFQWNJuRmSDBU0Hk3N4Aucy\\\\\\\/\\\\\\\/f\\\\\\\/jQAh6EO6svPv7u+dff5U6Ir8ZkcVu6f5HrVCNqc+94EPsad\\\\\\\/WUugk6L0hlItTPyV6K\\\\\\\/8YwEQYk0UoX4lDJr7m6XzkZxYTFGlppq3wk+jrifZHyAxKhN\\\\\\\/lGsd+PdAMiZxRm3+y\\\\\\\/Xf1r\\\\\\\/z63xRAbyoTXvFZzr0TNRX10XhCYi6W6eltf2BsooCKB96QOHqFNq4uUZTvWASImXcpmkhCCve+cLzGd0XI0TT7B\\\\\\\/BeuVI8GFr+tkHbSXhUMhZYRSk7Vvd0+\\\\\\\/+1Rjv7reBuJe8wVXdF7N150oaykLPVig1UOlNXByjmXQiGwh4W5jWV3nK2fsJaE2h6bvR3o6xl3t2SMAdThQpVnu6+qnPGIRS\\\\\\\/w2cCyFX\\\\\\\/l7X2YstGr8QDEoD0+AoZnM8SXMPxLWmfsSeqbGlYjwCsW2lt5PRZXpzpnexhae1rl81Q0TecT7WXIcdrq+JYiGQaN6HAE1gNExCi0v3dww1fuiyL6bAnEMRd0XEuyhWJQMRu2vk14rXrpGrpMwEB8Z1C2O6Yoi1p1rVgfW2fuFmNFXSHWC1A6zly++v7FueNXsD+CwTdxtaAq9mk+6QlJ+tx\\\\\\\/91fhA5cUrSQ9\\\\\\\/N1LZCOiv\\\\\\\/SjrL71GVtfukiRY0I5ygWJmtIvW6XmBmDZEBHCF\\\\\\\/oKBcgmy209UGv3y4fupHzel6fsVWvqIWHomWDDHJPwqWkcw2NGq1EvAnJDWf9N1jw0RRckae7kZQ6aO7IPoHm\\\\\\\/UwKGSeajC1vKmCttOBpC6eMFcB3ZbyqViXciRe7Bd+1yxnETAiuujwywpq+ejDXOIwxIRu2x18WYhgNwdVbEEqNvnO28Kl8\\\\\\\/dgjQSPVJAjeHDALU68TtvZxASe7trH+54BSGoCTlAiGJ5iITaU6gJgbmCcyQbiArV4gixheC1m+UULgdEvu9AE5zX988udF0ywOW0qhxfRohyj+i\\\\\\\/Dsp8zGsGD6v3tUw81dPbIsP32w5\\\\\\\/l4E8gmOZ4\\\\\\\/lr\\\\\\\/KHrzOoBC0qjOcxwG7Dvvnj72OvhBlHnnmi2JJ9wMFmjcWVAUYI0O07vQ0Xum8a2TgJZkN2Fc\\\\\\\/FkgMHNpJWf97vngFTUEUP1ssPDCiq\\\\\\\/nq8AwGv4OIgRQJ0DyJ1678PCeCi2+0+klcmgQg5UyF4AWs2+YNNr6NtnD7ibBFuXaCoylKnlzNfCk5jz6zfMOu2VPccD5AFfvtFuYMQJ4P0gtpmNGmYacR2HbOe79\\\\\\\/lhW7p3X3mDsnwuKFCVsiB\\\\\\\/ilbSG58K53vV\\\\\\\/EktEnwH6EXufnhTYjKiECCM54l+yg5WsvidNjwJW8VhFnryI37Y8K9BUCQJk+Gnpv\\\\\\\/7d63WnyD7HqC9VggQtfvFukuUnH9A0o6Bi6XQ9DvSNkpqn7d0s+SkKfUt5lMsszx3\\\\\\\/Yir0zFb\\\\\\\/6g0ViKUGA3iq91\\\\\\\/eunbXLFTQcxblJxQwZrw7T6s+LN1CgfaUTO+ox6nS4dn66SHAVFVWIwjTTv5feno6o0CU8tVWwXPU3xjt8Yf+TZQDYHGJDH54Qwf4U03BVYre7f5dRztdFYLd5SIoLxSBwl9mfqxWFTs15n15lzzq+N2ztF\\\\\\\/fZB89aiI1YJ\\\\\\\/Cedy4hGXQIkNnRUUXdDbUkDoO5LDpkWo59aKmOGYrsUbtJidMidDAt7K6qWXvvw5KaYXNP6WrRACmoYSkfUBxRYGHyASmBEgJ4DfCxIEVEkVN+Fe2HEP+bCOsUG7sm5v+KofXN0mH3jMnfN0pA6VAKC8IiovxA2+3GBbExKbVTdLfiBhMbWOb6QHQEDwFTUMHl+64X2wrOqV2GrUW3zdtb+XgJ0btRGkLywpyvShPc7Mm+kaMbXMi3n8VO9kmDLxhtdoB+DxcRoUfdv\\\\\\\/79CPg2kIapn0MRfF5nOar\\\\\\\/jzwBUOLe8tUpYlQGeiDsfBRNaFFdo3V+IVqZkliMQGTrUbw9Ox6TpfDhnAXRITqmvpsrfwmRpAduQT8i3NSFKmqF4sRTWx8mpvnuRKCYwA6kjL7gA8bxy5pzQB5gay2GhrINvT22apm9\\\\\\\/MiGj1nX8W1nA6BYJ9rKzcrV5vvMZGdhYd58YEawMRsPZs0L2Dfnz6Qq6nRIyzMorVU7hWFfwSX4Z0crWr12YuYi39FG0Ahnonnn3KXPS1RB+fxMoRUoZ5AfnkhCYlO7NwlS3jzfvpYwZUb1yReDuVzSEwI2RAHyzPDWYFGSobPu\\\\\\\/P8CGDawPdTbrs5YS8J47Q0qyfT3E24CBYpChJQHhdBJO26FyNdf2FOmR3J58wFst7hYeHmDRt5OOLd1KieFgU0RWDY57deZRBdr6xr87Qh2Vp5sZtMTsw\\\\\\\/Bbh6+B1zKZkFZEKuqvE\\\\\\\/chgXTBNxHj0fjE0SpV213XflUfOEUpFO4lj4uGZ3l0frMlG+SxynzyJUG02N\\\\\\\/H\\\\\\\/KD0rmqlHQn94et4KifGb4eMrdE85BZ6QJRnrcGIvQJgdF8axbErfa2zYzKUrHZ5RIgor8BtjwKy5AeiKfgJXQfbE9XFpyqxl0hfU2ehZUQbw4ClO22kG0NJPjkeTIkT0kuUt5a4Vfz4I9dqPiCBLw5AoJ\\\\\\\/P6+tKkp4RuGm3zG21NH\\\\\\\/QcEbZGr\\\\\\\/SarunJaDgkgHziX4vDoOLbJBlgNYY95x6Jiufl2cv2UDwudczeEzA8BMH6h92CbdH27VQGzN63J0xACpLOR262hYEoNvl\\\\\\\/p\\\\\\\/iQWdUM1PHWa27sHuIq\\\\\\\/M7wwqNsYDhmYMyqPFdghALACchhjrqMksh1j6kcbqhs9GhM1kLvfi3rD7MmifcFPJ\\\\\\\/3coVYbAoEGDW6CJ6uaS1VuNJupUohhk6Gd0rs+jN8Xy3VEo\\\\\\\/+agPG7GOKoINDX6UGjVE9qpX8v\\\\\\\/vskHI5MiDQaX39CCrhOGAjsyYfrMCeebDiJSMEGb\\\\\\\/FL+f9sPdRb+bzjouVLw+UTmDBjFoWJ\\\\\\\/Gz\\\\\\\/9KcTr96o\\\\\\\/blliXH3rUmnAFrY3lcTj3gRCN3BdNK1jnaq\\\\\\\/XV+FKH\\\\\\\/wQjjCk8UMCXROJ7O\\\\\\\/iLSQpv8h53qSg0Ykg7xJFzxAr8175oGJH7O7ww0hkCe0J5FCU5X34fD8VbpiydwAG8Mj5JSmiwsDX+ttE\\\\\\\/jYpVRFsvjvHdwlurM8V+8hmq+7vmHiKru8zp57jPGvjDy4Zsp7mZ+NTzwFXxdDiN7Gp9NrJsn5oNTcG\\\\\\\/s5Gpxco+JTAY3y4KWwzJwzKgTTEUD9\\\\\\\/taxKBwQRXF4ofFgPI8nqaK3Tk8NdvLSOz7ZxhJeU2SXSH\\\\\\\/oJfpxnw8bP\\\\\\\/BfyUI2wEi+11t1fc9iFzpmn4rXMux6+jeTRuA8q9JFz3tb3FkRkYc40Xsz+X+taGIfpcyAzv9QhsD1pd4DB1IRn+dKAWhwMsf44DWRw3UB+jTGWWf2L7f7CGLZN8AcT1Q0Of3NpDwdFIbENkLqFlHES5r7tuJBfhMYHeFeucb0HIWZc+dHXF5cxIB0z5HMlPCfCtY0kHMjUsTC95lhWYSps\\\\\\\/dE7\\\\\\\/SMOeO9TnB\\\\\\\/+6LVkTwV+3aZE9P8R5uelc2jQnr6U4cIbj+8ZTBnWxcaKfwMyYG\\\\\\\/KvYirO8lUfDpTjiT3QE0pNdfr8qCdfk0z4eP6IzYEUERcIGCvr2sEZBHFOOaWIjuSeONee1dBItYmHWAVT5gjjoVIpItF9ZI55jHtIHMjSFO0DJik9JuE7LvMBY+EGDziTLDKVPc7Q0c4rPo9VVSdysKBPXMgqcHYqtTS5emu0kZ5nRNcWrmg\\\\\\\/+WPLvWJYeNTS1uxFYlGa6rFphTQVrFcMlMjtOUrV4pJz7eacmklm\\\\\\\/gLUgsxOMghO1ApxVTZ33PlJm9plMgYB60K37YltOHCVcvqO0zmhd3jxXMLsh+08tl8V67bl0gsvYLwl7E4GyLkE7TX3UNHPFxSbmhdXvnHTnGJMxC4nyK+pJ+beTvz3GpsLr6yrMoatisbK0Ld1Z7yfRMRkWBF+fpr0irD4g4eyK2hLLe9btmY70srRWtXZISdQNfrHPeXLPZwm66x7MqkZ75n2Y9RtUCvtIbjPTaZ\\\\\\\/\\\\\\\/4uNFo9BAutj3oWYhHGIYs7EicJiNZeZ2Wkf0DduH+oQYceGKPkH7MrqHB16FIbbhnoWdWA2eSxxr5l47TrILBdk03rWN98f5h9hF280WW6UQIxTFwPu7CvYYBtDAYvBICKmYUBTUdCWOK+FslLIkkfv9utYikITiBnNz5ivAdwMPQJRPajTrTztFtJNqKcCNNDaKnRoDSaF62HqgBh7vpTZcnSOHlQLpPvtmy\\\\\\\/Gu0D8IMHdNzbdlrgPo9NJBFpLWeDEehmUhXyZvQukpKFGn\\\\\\\/tBOAZ86kxUkA6Q3BR9rvFkhhG2pelas4QKk38vS0xgsmbenJqHU8jF4xh4cIkuXtRkURSoXAQv\\\\\\\/aiv0VF7ckiPK3GQ3qOj6wl8hNwM\\\\\\\/G32XDYTzRRk7vQL9xkENeDw73YBfPROS8GYC0eDAX1lzhbLBgdm95lnvOr2bC6hSMnppdwILhgQLr6JUItPBtG8HpScz5BKsP1826sg1OPavSbT9V16zAghkSqmFxDJTWcu9aZK+cR2\\\\\\\/oIy9MADftiGbQ2pJ9pthNzKPru7LGYdmwMiJ37FS6+BITNnr2Ya9aOHNFm59+\\\\\\\/qxamkINTa7ahTYmvMiAMexdDtrRo8hxjhOyJxEmvUunFe3diAMkJC29J45+xJLtdrlSdsKPfTw1eQFQVK6apjAnJoCt6D6KiUmGYOYqT2nJAjdzjNupoq8W9spWvOnd8viOgGr4f\\\\\\\/+CWX3+FtzJxsxEH6PLqNeYsepklYc19vXOZArkaR9msSGfaZScxkQxPEQMM+pgM3AvEDS96mbMVuKmTAnSgcso1lIwyb4gI2ihR6spSA+e30x8hPuTMQ31lWh42OF6ROS8gDzNcwTH63Dv6\\\\\\\/SqgPprLHjIweb4TpTUuXWKR\\\\\\\/77vNgn4lHQ90N0\\\\\\\/w4VsdT0xxkcOP58mSOwEI9YCfkBz2NES\\\\\\\/XdMcHvN\\\\\\\/9\\\\\\\/PwreMw7wF\\\\\\\/QcLY1crPntJsf9dyQ722rlwviOXA8ao6o63lDih+C9OBODF51z9QWXfo\\\\\\\/nwV5rWTTzrFTBM1\\\\\\\/HKWhxcA2l\\\\\\\/T\\\\\\\/YU5YH5N4EgeuElRacenSfpWTqWd0vxqFppwtsfO6C9MYNh4VL4\\\\\\\/ZiMD1XCAfk8cm+LIvJxbIun4J572U2wuFDOKMsVaFFJgtuqPeedtpzsTVWmNRWL+ZiKxVZJxEZcjdZ98aeqRKNSET4SQfV8MyPcCMdGX8\\\\\\\/p7oJN3WhG7x0E+Z3YOSx0G0\\\\\\\/quAyP7gynoO6y\\\\\\\/Iy4w47h9EQV+LMj0x9\\\\\\\/RgMCmauosPzEv5hyNjA8KCcpDHWN7MDntOjdLklmV2LFRDfr3RzFSB8w2xHL2Y6UyAYBltrGBHbbBS7ySslKknWHpnpBIn0LNg45fW9RLGegWhDpQ5lqpq3I\\\\\\\/uCcTmqyvHFADUPlfM9MNKyQWYNtevrkYVD6K6lN7YyTNYl7XnM3luKI5q4C8qF52a1W3kigINhSJ8R2Gxh\\\\\\\/S9rCAih8INzLNBsRmLdIdyS1SKY4l+raUGYxCkXjOpG9neRZ4Pka32Pb5MrfsmnB7GaKL9ylNYVeXf8g+t8Si7lKTH1xGRiJDT0ne9667uWjkk1\\\\\\\/jlVgyB8ZlRkqEa3emiSKRH73BvxJRb1jHTCIWeuS3vf8ft9RbzuRFRL1f9EDrAbBaL+zxphxOaHjl71egV46zlrciH7eZFc7MDgLmU5+ZXw0vJrfJy\\\\\\\/fzm7OErBPZzXJ11WchPEowaGCshDfSvYG2GVDcrEBPlpznFyfCcbUHcoiSRYLjsdYjeGX0yzhrJuWcbr5\\\\\\\/Im212wDnWwbuF3G4C2VOHQzihLjjko7vN3qIz8v7GMSsB0gILXktFwZvrloZtV2rGYebnA57BAykluz9c3t7SUgFEOQrDTGgANyhoIXHiFsjlxE1k5LVuioTE3pZ5Dw2UHMeVyMa1xaFju5TJeJLX0HRVUqV6g06IuG86ziIQi702cmIIw63VEsJoLyLXPq\\\\\\\/fGRCs6G6MmE7nuZo5A2Ramx2QomMXvSI7QxYMPK+5o5lFItn+3eIkWMbsCr2cO2AN8O1EiKVTh2YM\\\\\\\/QqSi2YQw0eIgW2eAnpuXQILAxo1QAB6zIIW64nbYy\\\\\\\/bHjsj5Y3ftZWWMaixAH3szO0zkqkdPpA4URMFylmWi8hE\\\\\\\/4bOnlDqrp2w1jDM7gekQONh2VEGaBDiBAM9H2Fxpqn8LCaHo93LQBKPuuIuhUOhexbqVNQGpc\\\\\\\/AU3y+M8b4+Lf4Yg\\\\\\\/qCXKX83FIcrvfpXj+KRNHbesNdCTZ36dZFFsTkY90XKDdqumvA1YjF93EAs5k1RLFHj2x6GCYzF0sDRatzNiL67Z6O5HlyQuy5ELVoE+7gZeoqAlTtOtgSOiEqd\\\\\\\/ECmVenDtXGq9RbgESTYPo6ruboEG0R\\\\\\\/RGlX4uQGVc\\\\\\\/UpL4Dts5wL9AXuY6lH5CtzWi30Y6FJcwzH+P5cu5b5UkDBHAibqbBdKslWV7quGERAtufmXuCcAh5KYV93z+M\\\\\\\/xsdItIHbvdXDrFiykGSMQPjUYEWQOQcluK\\\\\\\/gGHvBU3fFp1Ws4ASVdPD+nY8ShoQNucb+7x+oqKFuPAIH+tVG76UuTQN+K\\\\\\\/q7jt15FtHr6vr+udFqdwjHQKLExKe8E7f7d1oqMLm66Kw1zR6SCC\\\\\\\/Uubt07uWZNvWKxqIHnhy0QENnhwXHR1jFf9BlplJKcxGKABfQ5dV2Cj2iyplJvH1UkBeTee97j8+jdfq9IiP1o2WQxqqWipkGpSSwaDSq5CUs\\\\\\\/jBpo1hY3IVXosjS7k2ZiEdkDRKVbwuF6e1ej9KiwLWuJ+AEoKDIbYAX9ZFWhUkCQ3Q\\\\\\\/d8QbsJ4IPGhamrD2Hi4lye0\\\\\\\/83XuntIlJC1D8tzTMIUPm\\\\\\\/aKtDM9ZDJ7qWCzzvRC1+D1TurAV6LzjJlYYaxaOUMtfaERFXZzX5OzNepVscV49iZqC8CTBy2U9vSzbo7zyrVUkb\\\\\\\/SlFSNiu7Mq+uCXd\\\\\\\/B2D\\\\\\\/wDFPi5TNodSpIQ14+kqolDAgvaV+scy1g+Q+4SSBPPm56B4QxYgJMGtFNhQhVPTq2ZBbUYkdGasijWeaPUJNY+ZICFnTKgf22vzgC6DWwLHFzsuWW+3CbP7rF5txoE1YRy42pul404MV8b0FRAcK7krYQNkPqjHdYNsTCcC\\\\\\\/7nw+jETfRvd7Yzn0L1H0EtYKdYQEBTWvyq1FC1XZsPgWEDTFSQ2ie8CNhTzI9LPZzOlloLzwXPwtBIG+Hq7VzT\\\\\\\/SFHlwUSDKRhDfgfYwogNntwqNx0WmYM3JLsbcHlBvOVT6gYTJcIRFZNmJOqcRMjSC3AtBrlLA8a0L+2AlVtGNLrwiwk9Y8sXtiOwFg7bjYTL5HrKiOSnLjp1PcNmwJKPQDpyqdwurnLXicyCD2puyEUaWUwfDudprCeQP0do6YUa3yU07i5Md6msNrJvsKUwEWh03LLX7Ep9DBoNl8KN5K2bF3kdTUfb+BqdXt5FbKTeKSdJcS80BtYbhpY6yaiiuXoLAUBcEnpdn8N5\\\\\\\/Uf\\\\\\\/qdvy6ZiuCcwPIKTXu73DidO4u060ZO6mbX4jpXiCk9YtRMVrNoIEojMYbPC1DPORuvc4Y73pbfliWaZOcgukeGtHeGvJKMgiYGbqDPoPVNw6fqAlXBtgme0LoXG38fECebgjWMyvPnLRW6UQy3lHFOegdtdjlnDoi4fIJkhUtee3X5BfvBiwg0fNxKRDHH4zypL\\\\\\\/\\\\\\\/wUPNsRkZCuPKhCVL\\\\\\\/Tl8i0QvqwWO9X\\\\\\\/vTZAgR59DUcPac06Bu+OBtPe9MEbATdslh1LvNLL57NJyC0Wmdt52BZ8wIV6eJ9bH4jWi4YVN0fqb70cg9h9ZlliM1HUhlHsdCiZrX8uqAKqEYu+evJOpusc54HuIuTUQrM0TOCveI3ob4eTqamkRX7RZn+15I+ITC1UBzTIxLpS3qJBievbx80ZhN5w2cb4l4hmCnfQPOe1puCEUAHSmK79VYi5JQCEZTA8FimYSPdFcU\\\\\\\/eAKRNEwb\\\\\\\/+zVbfpOOs12EGitV+r8nJztDeI3x3HxTd7z5HSSnV1IOEOGaPYo\\\\\\\/9zIjChBNZglp2TogB3HxWrtnlir5FT0uqb937sYqI\\\\\\\/qb5iagEI2uIzTrSxNIHEZkEO2Rg53ywgLwIMvNvtBuD1teZig4f\\\\\\\/gzviB2plnOtHwK5XosH2zOtJSrKYGxdC7DTCx05GbEcW3NPBop76QRWCalecLhDLL8g3\\\\\\\/yc8o+uqPoi1k4U+TA5fHdHDHq9NxruSAUOm\\\\\\\/f5ex0GlHNv0EbUVzFQyhTGGR9V\\\\\\\/wIpU2zVAsyfLZaan27QvSFgu+\\\\\\\/VvVpvwU4fC3nPhn3ftPo6ZEx4ahK\\\\\\\/T0nG1wXivBfwyHNtlQtjZkIooGSbFVQh5wtsflsks1rTJ6LwT3j8slSMKeNjyUqj7bcQ3aHyadjzVMc4XFFYbo6Huedh9ZwZAVVTGN5c6Q\\\\\\\/u0HDDqWnrWdn3w31oUg2HvG2UTvPVlJOa+IRsa7PY\\\\\\\/x4Dfsfhr1HZp8d47t3YQBy3LzfFXaASKf5DWUQpqApmfOyyJlipP2YE\\\\\\\/396TIrd7Ch2091yTYLvbR6S7ThltVBaJM7mKlIQvjWX0nhQYZ6U8NfxvxsASm\\\\\\\/R4\\\\\\\/Ozp7mBK8YNR08VIalCffg9tIqpihkSGQEtMbH7OGdKI5IbfNLe7Gw3r0Cw1clW0zIDW7WIchD67TdoJfNF0DhyXWWkOYRqYDTP8zA41D0pi\\\\\\\/HxmhdAIX4kzaHnMxHbaiulJEyyA1eWWxEfw6USrj6J+T21oHpHJ8yMUhD64qE1KKH+VFMmUTv6CGUXbdknlK1CepIyOdyZtJX+raUIri\\\\\\\/FoEU+Zpz34I7Y2EDd6uE3Qji58Znf1eG1ZTuEepoQJf3iPM6g2UMoxsjqV1JeH2yHfkXtNSWm0axqLQgsjLPUo10dcUwbHk46UNDeXxc5wuFhyKFxPqYysjzOaNCX1vZe8LyS7Ra83YPh4gQ+kb1QG2YglG35e5v63lbN0rGuq5HUK0J8kMMqn+HKWkF7\\\\\\\/vMiZeC8XONEuM\\\\\\\/QUOvCnnGROtoyZyni\\\\\\\/3Yi9ED1h60PjWBXeZMRxU81nV4HrLe0gpiyNkQ6r8r6BD15NqpJkdstRXM+zFtn46+EPgwaxqQ74bs1He91hsNhme8ZnKKVtA2dXrgt5C8qeQWBdhzAdZIzmNUeil1FTSNbODM84vNNFm5A9kLJP46SZVLRHRqHtQ3u76oe3RXF1g\\\\\\\/noS7SbHBDQonm4iVbgTb+CbQTQQp+9BnjOQOhC\\\\\\\/+frxYLLPF8PRTa5ACJsbmsGjTrOZTkY6phvOoePWRFD6vB9ZrDcxOifWAetjaLLbGm19mpUT\\\\\\\/BqiVuR30n5jAOepuH2utmYKsYlpQmopeD5Kd6QvS\\\\\\\/rh3P4bv2z9S0f13RvaNntGjd3q7y3Vw8RIhC3Bsmn8N9IZ7dwe8P6mZpXYHLw3C534HM1cPXqMF8y\\\\\\\/9jgv7sDeWBo2isqmMzT+nOETJLGXwa67x6hir6RyX9qDYKvIqK6YssC5d8eMdCSzVGoyUScGF4S8\\\\\\\/hSwGuDD5AbwasJnMLMy9Mexjk58XNl\\\\\\\/UePeo\\\\\\\/CUtH7Bt\\\\\\\/57Wo4PNYPvJKnVRFQzGLBtsaIPqEpBvkzZd+G9vS+S66HYkLeMEzX4ZKhNX3n+ytkWcKXl3QaZ3ula0Xnl8nnJEJ3gYXJJGViacJKOhUQmioSH2Hk\\\\\\\/k3B\\\\\\\/wenO2PsBbuGcvG5ucD+5J1sjW22i7esoFlM5VI7LloEJbHHkGR46kyoA6Vi2CXSi6OmV8F4Py3r8pceS8ClofV4MMd0EmkbJ4VKD\\\\\\\/flr16zvUuRnJqnqSzhGwOSm93Qj66yczOotA9\\\\\\\/1CKw4zi2LdT+dAepLJ4rKK9NwpsGDl3EFZImg4DWuZdEa1uxnPU\\\\\\\/UJKpicTc+7CMEzqUzXBJFCucNBt0xbtFd2S22cg\\\\\\\/ahqZBGlVRkcxGGFW6g3ZlJhDfExxjvwyBep0oInbDxouWDZzBXGY2T7vm1D\\\\\\\/CaE+3ZOiTvMU7SKNdmR5kbXNRh1uFiZuA5kO3HMWBvOSlfMUh66ihC9QSDEnr+Geza5ykCryGUoBB17bWiAE12QJGKl5VGpvgcjwb4uENFxeGWbgSiHPwHQaSA9yAg5xbtulcX37hVYv9JC99wTz4sRw9mGRbdOQZbWuSwTnUpw\\\\\\\/S6TfBHE9gMDHsaiRUmLPt5QrUNBm0CPEgRUnbpgQoZdscwszn3krHwSVRVuDzkJC2702kVdK8Mch4P2ttw1uQ7PCORPxGs6\\\\\\\/81SV8q19T0N8L\\\\\\\/GdWLDvSZZL3HCGTDOlZ+WLw47qr4+SyjStgMRboe0YvVFkRCzsMXYFxrk8lGM3UI60vRuVOV85mLitmLAsVjA\\\\\\\/qqUSvc4+ZbspTh+a9XXiBzAIBJ0JvFy6bdIeio8Iff51HPs57LOqZgJUi\\\\\\\/XL4BUAm\\\\\\\/y+ZQ14VFtFnaiqHjzVBEEaw959DoiUFQZPjVqq1LP7hAdG6lVZ0G\\\\\\\/wsKCWmDX2LQziFzNjGyehpDtUfufLT4Vu4YzzBfdEcv4DpR8Re3T6IIjb87S4nrVpytpce5ptuFkpvPII+ZvHs9SytLM8fYfmFaqIjO7M3j\\\\\\\/h3iSWu7vklwg5yA0YyxomCiCSA6vHvEXl+S2rJSrGWI0PwmS3oqBn\\\\\\\/YDzHcYEnvx0CtGLCXekRHpvdsLjie7QOPLbRajmv230635q3Jy+1sWDC8gvaNO8FbQ039cfloIPKowSkV6zRQglQ6yGFvPzgWfKgubAcDBpYG4YinIBxeYqv977WnA+JCxl\\\\\\\/VZrT3fkEj7qwyXljckjrJUkeTPE2KX3IzZIbFQoZXKKTRoO37k\\\\\\\/4ulkLXaH\\\\\\\/57+se+Y75u45B5TZQN8fIM0uwAL4akF152PPJmQzkWFOebVMy2bOsTBUUyNA9\\\\\\\/XsTUAA4RWC+WcpgMjLPPwDtbuCvsMmMdYdrfrwHdWgQrveGJU67uqKZUm64+o5a7JThLqx8xu9f653AaX5YOs7kbHUKo+zq9JLoczXRFkrq84k+b9mZrGWVevpUkqxibcyMauiGlG2tFFviBGYDKN\\\\\\\/N83V1r+LO648sZ5UcSRT+9yjFMXSOwJrPNyeGE3TMG6FnbkaPrVxowAuGBOa656IhxWihXE+tcGiUwWdEXX8IrvEw7yzPn32ASAox7dQeJ9hfWP3cwLEbIm99FbrUGf8h2XjTsuQFSCJJ8KC\\\\\\\/6pK70b8cftqrTpRPCgUbRFw5Tq09iavqsESfXYkenxP9uJRkkK0PNmP4uQ1c0nVvnw3601Waw3BIhlDHicLfODopk3QzvEbCXwjd9oqHqn7d9UaJ5BeAeBN0uwB7arUgP\\\\\\\/v5cOoB3VzshMu47Mtgz0zpEpv6g\\\\\\\/TJIjLxHRm7+auk4gT6DQPy1OVRCj3+010Rqn+OGTwG7BNk2gUCPFIjstuGEd1rKlb826UCIUKiYfMwBBeuZ9FcDnQRiWlPXAzsFERmsLodp8+Lb1H3E\\\\\\\/sIgRBtCnnPiXQ7I5sfKofVDhY95k\\\\\\\/YM9pDApsCev0QTAlBLviRHJP4tUfHNGNNWk9I8u2+6j8QYsFkW3UgfDln4yDxOTmv0NvdKAIOK8pdEUmx8Q0mJj9t6kUz03TlxX4er+U014oe124FXmxA7ESD17zCJ9wsc6tFRHWFzx1XeWvMfIp7OKgGoruBB+lOG7fDnkPE99Je3+THqNzoWlpY9L4q+fUJFI6NSmy52DxMgjIPb7jSWLe2cwjRvZZi34L\\\\\\\/xKVt38ywMyDb8W\\\\\\\/xCcMmyXTmGPmfMrkM+XR66\\\\\\\/rhMvL4YfIS\\\\\\\/a42FRvIqQDFTTIBQ\\\\\\\/o4euMbCCtF3Hd2SlKWujlanxnHuwudJNPbGRAz51VeR1LH\\\\\\\/kmLvhkEZYcPU3eCVzD8K6XPp\\\\\\\/Nx\\\\\\\/ZeRQ5d8E4lwIWAc+iXB1c19309NSfyvUjNIM2wwk14TekjBjjGC++c3b9lLmCxSrgAysyJYm3vDCa2KGdSzsgILUykeoB4NRdAHCpza3Wmacr\\\\\\\/vkTPUBkkLHr8RlcnWP5q37Pz+Fi\\\\\\\/XNZZZBhLbCGUeYmpeV0u8oUqvDwAVzpk\\\\\\\/936vOl6x902+41LTZbR37v6SM6w4MTcvi3bdpypevsRqjGcjUyhj0xWJDbIKO+3ivxMR0DiRnoASnwre\\\\\\\/d\\\\\\\/mg0HodoxL\\\\\\\/i079NKaj4VbvMrhxZcdBsyD5jBvNQ7V4JEPeiknlg2oia\\\\\\\/n7uFycogVqpin5e6exkzEdmlvrIOMx+5jO4YHOxSYsROoZBApLkEkIxsw2ycgIOtXJFSfhoOzBD1RqdKvDjYncdqIuiVeN4L6rcSUblDbBGGMWVd0cgHV+Xh3+XX95swX3kT5HcDmvfDh9EbalwOhQwOloMYa\\\\\\\/AS6kuNXhYi4cfXxX8xg1K1x49bs0kNh+FaoGQ9P7DrR\\\\\\\/3sKhkBbb2ZfSCWviI+o2tOCbqJV1fV0UekBs411AgTPHqCaZTgdyDQSd8xuGOYWUQeROVM1UAlfPILe5LXIDHbmHqSAlWJxo6TY2el+6Z5LG22qhZgAJROaTTaTVPnLYqbxMLJIhaSnQTuzRm8HuL9gKAZaFm1lwDnxWTHlmIHo4fYV+ErrZ6mz5vWXPs3V6vFqzXWxtTS5TXF4uWc6p2if2cxcNmPkbCuAKcFerhD2620aBwQEafg\\\\\\\/QOSxuDJCz\\\\\\\/+2QUNQSuvufqJhlWpnsftI3HE82G\\\\\\\/V4LMhJh+CCd20Z4a9auXRtSXNmgFl36czUYCFGwcXZieHyfIgEP5OkJBCJSCDi1QrPBzBtOY39oOTfloFyX1l4B6OcHIRW6bZPwPJd2WgbiS5VTCzX56ZYNcEqEXeqYUzUVZqKFoiGy5guZDzikqJ5xv5hIV+nTr2yKPatvOVWbplMoP13MwbnB3eoCg6pM2CbqszxD7p4NbKplCN3o0Nymntzg67H6VrGI5rEQcnX961XUf8kUp7W8cDqyN7BXn3kcJw6AffZTQFeXcOTT8YMWQqHS2rA9+bgYn2QwH36IMzW0UFIGkPup4XnMlTe7iBPYskvv1bkTkfAzhkwbbB6Fi7doyH4CWFUqyYl9AUr8vhOTB3sDjAoMn8IBvQonrbRChyCuTx\\\\\\\/UCA81KmB7B8SJi6BYksllEqv4Qu+c8fRIIxHTYF2EoZJkwWrfTiosewB9DwW\\\\\\\/GBf11FfnQ2sA1Z3Sum5Ja5dH5PiV99SsisCL+s1yoD6EpRQNlwOnqPfKbdBufoyZdaVzpB2NH2xDOtYh9gvoK91FE1jL7JN4HNSv\\\\\\\/m7V8WFoi6r6ff7J0k0AhT3zgSqjxxwfLjb8GaBghreR5aPqVPUlo6eWl8MeuPOrh7TqthPDcORbKmKAThJ7qnujTmM7OvWn6zYv+Ah6nUuhGaxms3t29laz4WCOwSQXaXSyO3NqW7Zx++fhDeFKa\\\\\\\/1O7oK1aTN0kIj6Pq3OA653WMHuJNNtWAAN+kkC\\\\\\\/8g7WUbs\\\\\\\/uTJ0X2K6YWwHwsZM8kTfd3NXUnKW4v5VEgIUvxdtVKCtajeUSaKCXksFgkmnjax1kCjC4\\\\\\\/eIEjG3OVkKsOd89vss2UWiXWqMDIhT6tT1QZi63nGlmUa3\\\\\\\/3P3zBIDzjkWJ3B3\\\\\\\/+c0+Ruj48xCgdq\\\\\\\/WFhAHg10oLvXgakux9U4vVzgjUbAT9mZnA703Jtd7SZ3wHPt4OPilRv2iOguOEBWawREaoRcLJ0BbjqP\\\\\\\/TRnbtzOISqM4BjELIlRUbNUcRLJ4dPfOKtAF+0\\\\\\\/bFOgdGJCSAFmm2ATDd1NYzUnxa0u5cdhXJKgZzYG9bCryr4DAkhRNZDVSIxRDiM9+2zipcoG+cssGGhvUvX9Nujdw8MrRPWEzzx6Gfp\\\\\\\/tb2Ia\\\\\\\/5STW0D8YDQ3mz1XMQNVL8gD0EoXyZQj2zrehYSUE+25VXuOdYhiIujxkCM69GkDofm\\\\\\\/1FUuaKA\\\\\\\/A3L8UWE6D9H84PsHF3zZvX91mjx9bFmTJIVJuyJoqG5MxmPkVx48yDFFYEDRTtPrJOmqmXQfU8VHSAbJ7Xck6rChVUN80MNELMKdkgOwiraJHRLGOSwSBXfgUN7qUjdEKFo8xm10bQeyyksoSEtxuomLFzu7x3HgudICf1PelPMg6kJL97eL1dCrkQ\\\\\\\/LFuRoRoNmsn40czY2fBrp036eE3jY5oml+zH0n9kv1hn+qLM80TrHl\\\\\\\/EdscYTpCmQOpPiIvbuqZ7996hWLWdpt71IkQ35y62Je0kLZiV9fRPHNrcrzJYet1wsYVXSoNPcV9qg1JFQc7BgSFeNq4MgX0UDfGNHTs9he5mzW6SW+2NYjTjZv4V+htMYX3Q66pdj9++IOU7ruNq459GRSFDYim2kwrZ\\\\\\\/TDuxnjdUk4WrAxA0UJaiWDPAJI7c0v6XE4vIo+s8c+N6BjFkbT90VlqcVxcDJu9DhSwiDx\\\\\\\/+WMT6jVjaq1FH9taPYXP7yVC\\\\\\\/LV2fStS1KlAxDt0qAcA38yqD8vqmJgfcbquTIhiE1L43GQ5OPP1PdxE0jjlnDbNDxDqg\\\\\\\/ok2xirW6cfSpU29WI1GwXZcRpYy22kt+XIoldsFt7z56hDZAu7TSuwlGaF4Vus81GCS9U0S4AkedWoy7iVWVB97mxdiraelZOO7rfQLH3OIAbwMh+E1WuAG8yy8Ad4fbj7ppNyDvs1i4H4mYCBFoGi+mJYgC3BuKztwRSZ1JRyKLhzc44K3jXZjPqsoJdxA72SU145Kz9aYPIFRp9u9pIvHHA9EfQXLD0IBh7u8CD1RLQmyz7CRUaHTOqbj2fvjE+cfqMpm\\\\\\\/btdJH5IFMpOX\\\\\\\/sqPpCE9bnZhgT73Fxnheuq0UV3\\\\\\\/yWyCNl\\\\\\\/PpFMEQTgYbpem9LoiGHFi+v1TvGyMGfQkvoobZv9pBkt3YohPNq0ep3oP0eNRJgI4DnSAg2jt2CdbrJg2y4v5bAA8g6QKl0rY8T17+NgWWMwWmGdCNTl6E5nu+deVcac\\\\\\\/eEG4AM53BAdssoii4pNlvu41oQjTTMMafm0kHKb7AP18E25noHfhngFDenZMiz+5yeq8C9KRKl2mBzD8Mw8lD2wuSfVnWh5A7SXVdd9rbiE1u1OdHGHZT\\\\\\\/ckhLGb8fz6hItjHhOuCyozIatGIVTWGeq7IH8LRPr2Ls1podppoRMHPnUx5lKpSm60Xx92p2svFqjOhTSHPR\\\\\\\/8jrbk65wtKmmCNtPBXrlLmnY3EJM4vn5\\\\\\\/RadEx+WHFu\\\\\\\/c9vnGk2Mm9Emf+HNRT2OyDpVTujSRmBTY72nIQU4KYJ2vSSGFvrPp3vIFhjYZJ9ThcETvnw9zOH6JInxkxHns23APw\\\\\\\/60rZIAOcmvWhm\\\\\\\/eVliHkHQfSbjSNEzdi9ai8bY+8cqa0i87rJIJ7tTDxeIK5t+SJMhvkzqF2rmo3KJ+kNbZEsUz3BIlolAL3n+FI00xA5VQSRxo+aoM8PFXvSgNluG+WIrrk2u+ZX0AdmP6Htn\\\\\\\/3K2f9aAG2WqmFFYkjKja3cOZMirdGl\\\\\\\/L61wIuEPlNxQ+az9RXfa+57RSkuFl2GVKsee9k2t0EBp\\\\\\\/B6cZZq8vz0qeGQlphfEGm88EFr44JrvqRywODotlIVOrvf9bbwP1YE0Q\\\\\\\/xLezr52TEC0AwH1fAEXnhMmQuClcB1ygB39DqLpPmlFymdKsjzDilBZFXGEATrk1VwAQ20ud\\\\\\\/L+nN1Q4jaQcW1WKuFbD26JL+3ccoh\\\\\\\/PFA00ISTYXag\\\\\\\/q2Y++0JdNhiu\\\\\\\/Y3s22fSLLqPpGohB1hAm9lItCMF5llOp3AKulgNXxylxpiHBhACjLklLxqDxIp3pdESgJv\\\\\\\/c6VSdS1fSrCe9Jwi7f+lcU4ODwwi4fGl7hAl2YxZwX4X1vjcM6uA4fQ3Md+fSKflABI0U94O48UmiVZsyNCHNuyaENqdz81vHgls\\\\\\\/PiWPtuQ0Hxdw85fSbDGK8cmwNQy9VE9wNDm6D6MGZ33HV7\\\\\\\/YbUOHFQMs2gE0fVDdcqdOYj7fvUOi3b\\\\\\\/w7aTkoZtSMQtMWtlqKo0idquV47sWe+EIQNyT3U27uG01\\\\\\\/sMLPabZtVX6nOonR7pDRyE3sIVjt1G9ZO\\\\\\\/x1eO9AFhCm3nRNuMB\\\\\\\/s1CvM80BWJ47MyQFc1iDtevQt+cpWAfgb8EhRr8KAmUJsDfqYGcDJHp0AOo40vk0m2h2cZEDUbhvJnyvFavJ4R2Lv7hwSkQ7E\\\\\\\/BV74K0MwjvjU3tVSi2kDqOnUYOcbRjLwbflH8iqwOhzIq+lAsRGAF63VKkGYd3Bb5srX6gWW7M8TQxh5OuzviZa24NU\\\\\\\/R0CoUm5ELKjBcS8dg33CnZLRZ2hqDQ\\\\\\\/LI8NtIPWryaVOIFx3vL0z+o4M2v1glP6cZ610+zPanx4TMnMbcBuX6JMSDWpnwN+TqNS83MYV2IYTVMz2ZHOAvsSTyRlV+jrexfaYHE48W8gJcxeymUAc\\\\\\\/UDvWc4wJyUXTdR67MIhdFa3I+rMhLRK98pF0+x\\\\\\\/CsXwb74ChmbkCnfgm3\\\\\\\/rikMyGVVizE8TqT5tQa5DiBpfW\\\\\\\/L1+USt+yN10+pcAPP9mkhuN1mHAb9qLuRsT\\\\\\\/v9jUJSKe7O\\\\\\\/dzwoUHzBaqgMuya8NitViA3GuYyDt\\\\\\\/yuKyxjrbLYlWOHCrjfBwAkgdJFIpNPyPNTFDKytPkb0IcJ6RQx2qpXgb0YPj3IEI7Pp7q7AChrzfA1GsTXOir2pn8LAu7IJr1XCQzpmp9EwzYJaEC7yU5X0laRvq+qVR2b4oNB\\\\\\\/nMPvrzSjE\\\\\\\/VCCLIUU\\\\\\\/mzIkfZj1mb5QM8sqMwORgQ1BBfybPs9IlDw91sIM6v+7kqCyWifDZcnZi61R9GkkDzue18f8vQ5fGjB9gTLmtemKrEHTUM1RHCNplqK2X1jnBHoowLiN\\\\\\\/640Ak05CdQAC2eLLxx14OL1+57LTcEDWwEUiyz1E3Qd\\\\\\\/ZwHtwp48O2gILE9sVAC7Hl1HQeYQrcy+atB6AtLWH6gYSvxi9WRIvsj6K6CPwKzilqqo7Ng42Z82WqXU8hMDoZ7xlhqF1BAfkxDQSze59CSb7o3f02FtLhM2O+nLgjz6X+\\\\\\\/Td8+RwB4956APwjLKM50ZsJx7qb94pbdZk88DqaS5\\\\\\\/euROxURedk+9hb+\\\\\\\/27TGEloWgD0qJibJS7CUr0\\\\\\\/3Cih91kNcIutnHIfEleRvzbLG4OkHIcuN+l4zL5DTJWwzgMqcJtqg8rJLVzOvdwnmjjY6LDMorsT5AMAGgzAuTLrybC8RrFAXx6oCOKb6gjoAx1mmcn5mpiL\\\\\\\/mCjmAB9va0tBmJoADRiwPPrCsnKcqNN9Vitxsd9qBZijtVbji7JRtD4m0FJNVgvabkLOWaTAHN8IsWmiyG\\\\\\\/l58p09DcqSPicbURJ3JJCKMiOqUsgczWym+MYQdUE9pkb0mm3Jt41kZj\\\\\\\/TWx+lw9JyyEREA6cdLI33CCL1LTgzpIg4M4xlLU2EG7d3bmZh1tCuzHRotjBN\\\\\\\/cDdUN3djmgYRaTwUK3SPSa0QIhgAWJ8Dmdhfu6QuGIJdAOdJK+6p87TiFhCZJBGC9fd71IZFarOZqVlWGsQZrdPT5F6NRHYf9TdwaFr0\\\\\\\/rKbtsFtYNAgVwCBt3cMs+cslOXSxKsTeOIufUqTMRZ0wlGt8RaRxQJknV418gqJAnQp4hHLA0mK0veDocO6AYCLu9tFBdS36\\\\\\\/CvaPE\\\\\\\/YLRTf0OtfrHOXu0SvrwaAZnXHL0Q3EXo9L\\\\\\\/hTx0\\\\\\\/oVb36\\\\\\\/0aLpkSvX0aQe6jSkMfusUW1KN9TlJNZLggmVeChoUq+1Ad75tGIxUVahihO6nOhZRr4QMAANEuuAQ5xYWOoAeTxJJ7pG9e\\\\\\\/SFWxvYvYqJFhJmlLBm131KI3C1cF1ZwXIH+QhsXN5EhBTS+qy7tKXrZRtfxQM3ag+GzHtmoeeSmnrVwTtEdwJk91g1Ij78c7t\\\\\\\/kjcMauBhfG4amw54Gcl70v66In\\\\\\\/lOwJVAm0piZ5bylWhtcS6LTw+Bd+kcx5xpxm6jyHMGMBcHoutX2fpMG4IOX8spcTGJzM+oDw8GL1CrXZYbYljJFPlZnIOuo1uSDDwavqVA9fgkp3pmAA682CZ9BT7pRSA0dvhF3QAlmC5W4JPKHFybHH7uKAbRw4uL2imPyOaKtAN+zdCu07sw59jfXlr9hxrB4JEyIAhkwFqhuvxWZBEnl9xX8VFnxWdA184YgSdLQuxfWCdDRgWM1EjoWS0pV\\\\\\\/7dwCCiSEfibHA\\\\\\\/DGAafRaJUx3Sbrnu0QsxRZ5pnYDR1bqPeb\\\\\\\/E\\\\\\\/vPV8\\\\\\\/7T2zpbHUwZDOnLsghapcvNMSxntof5hwXjfh2RiwGVW4XHghQsjB+3TknibMWUDv02ZFM4crm71DJWMz2R8gJFRkiktsj4THG+OslwIr8l8QQAh8vHU3SI4uiDrXBueceryK6GoQDmp7Fr+U6gFgyEoNTZxFtre\\\\\\\/Ym\\\\\\\/1o8vh82tqL4FhUkPENqfRtmxCOX45Jshmf1w\\\\\\\/QmL5Sok\\\\\\\/ZsrXgJ83xqFjmHXK9JhEMxGmSVaifr2Zd1J5Qj+2pFk7FiNwWY2IyNCebMQvKR3nfm7D6OZ1YsZDbS8cm3\\\\\\\/6EACH7QcJAIoxj6JBFtvHoQeaK9rvnNJytpERZmd3QyE3CD4tYSs96IYeXEq+AjZgQ8kXd7+4pJycWauYlt\\\\\\\/fKQPBdkIEJdsQHzyIxJQ58jhEz96uMB6UgX+Iy2dq\\\\\\\/l6NcgZBhVQXpea3JWlicxuDmTLrqbvFY60FpsemW8g5jwBWbAWtnI\\\\\\\/Dhh8CT\\\\\\\/xOYXPbyMNo2GnKJe85rGjPJddMZvzbm3BcgvO6LVKdDKf\\\\\\\/ejQ8D49IVejhWALN9ApHtbA\\\\\\\/\\\\\\\/Lsit3ix3JDSQM3ufmK5a9uivXrFhLj5AMODySnxyzksU8uxkvGBPyFbuN0D1Uk9X4NBl19ushNiEQ70yWVDNWZwuOTc16FqHiFsxqGm+kO3wV82iWLliYRo9K90F9QGregEXL49fNk97vzEyq8mNr2HS1547qtd4lckUaAw1MV2UU9Bo2f47Sc4AgSTcPI\\\\\\\/9Ndjbv2ArJgxVwJkPUC3Z0fjxBfvam8AqHNSwYAFYvgGw2wLBKtfpDVILvSgOzqfSklLm8AeOKtMFHIx2iqhviIB\\\\\\\/6YDhwWZQhFqMU5SFx04RqxP3OyoQeklP1Ae48UMzDzIDfiXF+2L2VreydvhZB0x3x1mdRV8HsbCRtrlXtwDPHsMZLd\\\\\\\/bucpXPc4OMvbghKC+vXVjlgATOu52kW5dadYx2MzIiBOqZb2GuPAzPJ24BTtXTD\\\\\\\/uw6ZnEiF6GzjEBzSXRO\\\\\\\/9xO4yYdWrhuqBgTTmJAD\\\\\\\/6+EHxzTUDYrDmtVb2ArUDjL4qsP+U5EuofQBWI72+MWKehRsJlpmFat7ijpu8VotYB\\\\\\\/i+p5NBL3NmzJQChjEwQyTpWuPQZplI7wCQPF+QWZlUed7UCL+0ynDeX0RipVJ8KSnvNDrovH5Xzq9IaM+0aBs+OMzDHnW8LEFAprSdOEjsI32PgeFUtBHxop0+a35CrK+NeO16nm5DXXLjflljVtU5QyZKIftr1NkycBxWdMNI8OLDMky8w8DImCwCuU0lrbK1\\\\\\\/H6rxvAZj1K\\\\\\\/wb0czJYEDVS9TmN9Son1\\\\\\\/z5rekDZtkyqX3ujmVPUaT1voQ3iPjsAMY\\\\\\\/kjDiKK3o24t+N7r+VG4uaFjpotVcp6PCBLNewVP01tlBpma29NziFLwmcUS383qstxac3uIWfn1RQ3Xxwf7hAhzY4rxpE3gCle+ALZwIs7LTkncO\\\\\\\/h5w3IHmw52\\\\\\\/RSeVkUw3STukvWn4r3DoO5vTZ6ORoFQ8HIxSHbmFuiDvH4mFC4UX4WJ6uJtg3hXkDdAx\\\\\\\/ZuqVHlGGgro8LEhMLx5\\\\\\\/Snu7BMKnZgHMn\\\\\\\/IePle9IKFHOtTJsKJ9OtcgfzkD2KZDfWAlO+sS3hzuRRmD\\\\\\\/Z49wOUvxhqQFp94E0iDs9oKDPYAaAgLQtaQE65AICdH8CK8mPhgYq\\\\\\\/8FosR\\\\\\\/LcjDDls6xQ0QPddECY48RCKuNmy+DjrP9H7GaZM8HD1aQmQU1VI+IttuhuYWKH8iTHHtbvCczq9lWlxleFoXEB23\\\\\\\/UClZ7N1gSZ0o9inAPuVY4fX67r7AEiCtqZxjau\\\\\\\/+ja+ZcQllhVRYA7lr+CdoUgL+wN6xjueqj42dwtJ5u1jEkPYszeX9KeqfnE6raNp8EvB9oURV2bI2fNJeCn24bd73c8BUD9Zj4FirYMen1osofxe9xlEnMJfkHH8Zc1Mil3I96+8Iq1rfjt\\\\\\\/8xXmZcw\\\\\\\/QSUy9WvYqnK5aUSHiwWYguGpFLYC0zSlvoyMn9bcrNdmEplw+0RgJfxg2s2spf3G1tfKpkmrHAJrrd4bl2hnXdOcVIF\\\\\\\/RSNYQmGs8hOW6nFEKTxoQ+slImmFm\\\\\\\/oSCLOEgwCa7qKxFq8L8o2eEIJ1z8sbNJcOoPMY5GC6vi8ot08mLaxxCzd55znBhJWEM+ApVyBZ2j9NvUMq6Q4KoUUagubnEhFMIbZrwW7g4YvFiaz74DeS++mDHWetjL1zfqQvSsH+tR446qPuPoMyHl\\\\\\\/1UpOWMpp+p5VygMSf3KJs91jkuOvDHd7HxI0AIUr+bjPzCwiM0u\\\\\\\/eKuyGA8V3o8JXQZOBLLEQJF3ofumvm0yHJNOagiqmVYRgRtJxuypIi0fNXhz4q2P2QTSVZ2PDkzGAGJH3bsbVRbqj3AHsnS5+6dbBaY1+gDT\\\\\\\/C8pkH+NGMoL6vNNHXysS49fEdmIDAFTirJxLiZ4VyQzPk94gLXQHJ\\\\\\\/2i4xLkbYe5I6AXmyro5aAKbgF8VX7jj74U5CTUY82mZmvSFob9mm+sjExyMw5kpkbr\\\\\\\/nTcej7Zck8mlMSex0OwZi1nqovfftydz115B4N9ThYHOD8k0GbZmxA2cUAHfpSTtMohyNt5P+\\\\\\\/tyhpBFUXv5rq\\\\\\\/Xy7xyr5SCu8VJAHEOvtW35LN7l\\\\\\\/iPOf8+Pnm9psje6zFA\\\\\\\/bYV9LTZCX8cgw80mQuur6a+ESirmYc8eOEK1RFaPYnc7Z6ZMAVwqcXUTYk7YEIK4vmjZ\\\\\\\/jJHTM5WjiccIg3KUjse6+gEcorHY1LYdmQ+UhcGH7d+IT\\\\\\\/vbCULnMO4HYsZUwugsCl3TZNZ\\\\\\\/wawhuEobqUB5ImrKFHGHXDqodpY0wMUFfSQ+iI\\\\\\\/4FqtIotrVt+lOuiCnBRrzYR1FiA9CiDZ3vDjgB7AWeCdoROPo+gPxlTxFyIGncFV3WVchNj6GT2pv5IAJ5lBI\\\\\\\/0z5n7XNIE35kBubVG9+AHURfIpdo11kTLUtb1awAp9Qgxq3BRLFg5XdNawIQeCSP2E7l7PqAwu1c+PuqeOMpB7VnvSBIPdzsAQSiKLuXsQTthne25EreU0wa9wA8xQbyNYM5CTcLUbJGgPqt74ebNmjqPokxuvPqCCPN7ykC5aGDCFnLeWuhC9nLop+YWILfphNpuzDssW13SmmSOjct7rnWs5MOVom5NXx8CcotvrGBpTBGpg3ADNQNHC1T52hiGJLw+IzKOs0mX2k0hIAhF6a3EhzPBJUt9eVNgEeLz9a5vkkMxZfTH0zptRFqXwTyvEoUTQEB5sTwSzRAI8P1\\\\\\\/5TSAm8+Ef\\\\\\\/xGyBJM1bzqWb4JbqWgQ1\\\\\\\/DnsR1AylXdry1xN\\\\\\\/QPAUx8HgoCtZ2Bq6ytPscw\\\\\\\/qK0jl2qWiLn5OWYnybwZrkzqlR5zIykkywnmLKS6Ifj1KfXy2rZD4tPVcZSX5OwS5UC4HmcNoqtRAb2hSW0752ucmHETrj+lbfMyxXymIfZOmuzU\\\\\\\/\\\\\\\/n3BC+Gyb+6YRr4VRk19OZ6EFgF\\\\\\\/JNwtYLHE48zKgWk5jaC+Q9UmgOMr5g9+VSsUCCUGMhrQ50WyhnHh7ETw6L5MpyIH5xm0RWDl73Cg3VwVhTEDtqxwDvC4cJ2pTi7PAqhUQMA+iJlJzqlZ0AaXuIufu94osAqDHAdFnlMUcps7KYcnPk4mXYBXD6LHvvNapmHpHFyhKzEH8OGy7rR1zyWGTqpnDCbjdwEcPqAYpKUaflsyUtdrjIOmr8i6k\\\\\\\/DKRdUEYO3fganfypILzhVurFIyczJ3hErbsfCqjChl6JxMVaZ5yPnw6CfzfVVETQZ7xYvBuJh\\\\\\\/atz1guhHeCeXz7rwAU+Ey88wledW4ymp8pVOOKIEZuFGYC7Lg1lC7SNLG+rn13ERGeRuhUezOqcN6veJuAv0gnxhAF95DCoQzUv4rJ4y\\\\\\\/fmrfoqV12cITMjGvpWH+OEM6atOp+Euwprd34+e3A3jUnkxyC0HjX71FcdUtB6\\\\\\\/jFLIoGH\\\\\\\/eCW1Mb6JFZnha52QMCFcX24Ey4ukObBCRFlUj7WjV\\\\\\\/JbA4zhGzjRHWM0Hz1VBCr3rU74Ix4aKIfR0jzuQZ2\\\\\\\/RQy9Fe1QzxRAQaR9LPtKVJC6+EhnQJObMmXB9qyzugBiojv0DKLLlC721Ken9vUba9C3Soyl9jHVU\\\\\\\/3spkFTWsJ6isdfxQkNM69GdVMx5sxre67V733SUA1oMafmcsYGOoEBQS8Q+Uya6Rpi\\\\\\\/TIJrYtimKHyBwYJgu9JRR3uu6Cbpchc94RkTGMUNl9tcs4bYxK7ka9QubzbmHQDm6j4Y63\\\\\\\/egqnM5112\\\\\\\/hvexyLlRIZ1+NXWi7GYCkd4IvijEQHCAzimcLSABt++2fQRV7bfZV3sKfjyjPtUK4iUIjQ5Roh35tfXfRBSV8OwHyk7WglmQCBwNlhV4Lp6DgKMqKe4jmipTHPOdggCFwKVfUEuFWOM+UHYVsEN+w2Dsv9IsEA2Z7eal8547+Di4Hv1IHu\\\\\\\/B0QvfYfwHtyoiJO5fMhnoo6X+Dbgv7PbfqkIQ1sMPL6E65EA0bif+3qinRzLHXsbUecLt9P8wQIfCCEi9nDg5Wvz0zvcGh2v3vHvkivtp7\\\\\\\/f7yX9VGw9MaqnY9z\\\\\\\/01t8YGSDcGXpWdFMsjUNkft7l98BZlyoRCzznrvJlRMvlp92BrSOWeY0sn1lV6wkBH3fQz\\\\\\\/ojVmue+L1T8rAg2PNjNgVjdERRds39g4IfeP09q5J4Rr0oY5bRoL7k8yg3LHzE2L0LR9bm1vX6muJILtGudDpiieDT33\\\\\\\/FuTCd4L4AUCUcgUE25Tlu12F4d7V6t5z6NMw7WO\\\\\\\/0BhcRGC1k7mM20o9GseJmP8xVSWh+Q33rtdjM3RL850prd1NMI5nm4u4XSJjbR+YlR3OG71Npj+YxuFlF8N\\\\\\\/\\\\\\\/1gWl1GJlscEgO+Ye2nMUXsgHT7+ggnE\\\\\\\/Bt+3V2BpdKx6An3y3LKqqzxn\\\\\\\/l+pkqXDjbxErTg9LkOFVQSu9WYOm6gpejzWzehI22vTKwfpOgzlgVbul1YzrdNKivP\\\\\\\/jFgfagD5pqT\\\\\\\/MRK3Tv279yl+B43VejoZk3PvbX5wx1247ER1g8GkoEJOzUVgOgq4js9qRzQmvAutiNwwWfDh4OcmOBEAXrccsqf3gnYZBsjlhTJwOSJrOGioFUSb5A4AJpR2iBfZLtNPfFLEe4f+1lR7dfOL8wyY+pSJMGzTv6jGI9KDXMh0w6DD7KJCSVFEIXORh1F9DwkijBQxYzfutSft4mQh\\\\\\\/RiBwVxjAl1RH12aGeIP1NR42HgIlIBqROAkC\\\\\\\/xjNCrMoH2gAA2BgiXWwiCY+78DjeDiPdMJdl3tFlL4cs6coAhAZNgXYSO+toqS8xYyTeZYEtgvnfcQoyDm9jMwdAJlLwd+dF77Nf2oZxzhLMdzxJO9h6AZggREz3djDB1hahJkx+z25xCH+bCAEY8qpEc8dwEUc6IWJdEiLyfmWoyxkj9843A8\\\\\\\/faZZ2lXncWx6UK+arB1+OFc3T0K9R0hN23rC\\\\\\\/fR5+Vq5Oj2CzKR7IjgvgF9zTFHKPN1rs\\\\\\\/2Nbrd5A4gUo4ujfwjEH6AFFcGqhUlRrbpFuRQLcsXgeHRa0ftZ2EZCWCBWZTRpeGlX2EMClAJ3lJEV9jPjprPdo0E4Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wTz1ied2Tv9CvFyU5KHBxA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224f9a41ebda0c0d67fd80d25b8a9f75316f8b5170bd4f53c1e9c0ca588b225799\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w06Yi0NoVeeShmffUO8fQkmZW8+w8uDQ\\\\\\\/53y+ESt971w6KN2vQvaRzd9eI+52kUqm+NYv9Er5PVDkhzY5XVtdvbu4kdl38hQBY2kPu55jMi24uNbdpItxmYaK0IjiYXLpkyT9hTFp2pX4Cuukvs5p0+hX41dZ9nBlnKtqwCzNjZElyZfcBQhYGV9vQ2F8MmwQdXVvUvGDx3T+XYyKPJg8wODl03Q7b0z\\\\\\\/InrIV\\\\\\\/FMO8kVgToR4J4zyTzcXzWkJ6SkYK7DVq5CHhdfkLAbUBmpHP1p1jYQgU4Bc9LsiE0T64sfFYc6YAxcQB3JHK8t8M+qhhduGtg6l6Ju71X72kv0hbAIgL110USJUiNT\\\\\\\/klbdkPbBDd4zhsiJa94N8zMROzMiXZP+N0sMfAdrwbv24Ve+MpY0S90RIlYUvOAC77A9cGtYuLy3NUyC6LVzIpMb6Q49rhuRxkMmmsSHtcfNtSpCMBHkvIxz9AhAGbmiOaXaa+ez7jpLqOXC+3lK02Xp2xSHgMAHctpeaoeZWEJOC0FyeI3XZ2QMslHaJMaYnK3f0PDe10zGucmX+j1fmud0qop8ZIq+coOAFI\\\\\\\/MrhH9iFvEGUPs2LFzcNwRCvUVrtywq88kNLZsnYVOZoug3Y1BQQV9BgM\\\\\\\/\\\\\\\/QTruwxsJQwj5PXB7GM3pupUt8auDiDTbwq\\\\\\\/kx0RdmvtuqEC2RDmEDY3ILTDhOH4jzn1TPY3wVMrVAEa9ASACoPviLoVXs2se0hrmxwao48jPcpIM6TxGLG\\\\\\\/CKAZLYGOo9Be+WAooAftpKFiXZ31Nc\\\\\\\/ec0Ud0iY4+KV7AKzTIzfkiTlmw2Fx\\\\\\\/YNFaY8o61PvrB\\\\\\\/8Y6tXci4wK8HQa48KEGo8uWMHZt0GeAMx6g4nxAYIbNOn8lA4KE5BlDxIjlZK1mwOK6c32UgqHoQGsD3Q1tLpdqO9\\\\\\\/PQenhRmLJrkO8EIjP441wF3GhDSvY6DI0oa\\\\\\\/0Id+D5Jo6zWGxmbq4XK72lDOwxASYCH4VGIrmpDGQUhttEF0KCMKL5itgln4CV8LQZ2Y\\\\\\\/35n8SFU4pFz2\\\\\\\/8S5VN\\\\\\\/Mge5qauLQu+XHpeBlb5GGa7KIPS+uxjryp+BFVBUtFNl3wOLwO6d9NOixGc0J6r9QrR+UtDYK2F52vS++8L\\\\\\\/YE7DcSsPqeCwBUzg8MDlrYjga19YK\\\\\\\/qERSX\\\\\\\/YYTIgBZPYn5oEb3KxmuLBnGC9+yy0CbID+20EP\\\\\\\/zbRbg\\\\\\\/XLOYybzEHg\\\\\\\/bnW0mT92r3j\\\\\\\/sJXOnFP\\\\\\\/fmQSd\\\\\\\/guG1TgFivvP7wuy0dXI3BTBdEVmTg1gt5TUqXJaFj7pS+Dk+on4a\\\\\\\/ZqN+Nkjfghj0SlbafvUpnFpjQ+X2PY05CwtXwesIWMqIVY9I1pBrfPq+Yx3aMWNQhNm4VXM\\\\\\\/oVRRMURGN+uan5\\\\\\\/aBtsmKwr5JpsZo\\\\\\\/zme2xI3Q2zebvocRQoyOlSuj7J57Mw5+2TNR1MhGqyKls3EzY9fxrXBju703aIsrRo4Iq8dj3tNxPqWORjHrMdaTVyy3E4tfOkS3OK4KsAe4qPQUi\\\\\\\/H8OOy3hb0KFxjp4NnrBlhWQ5E+2AWlEtCBsFGeobTdHqoVjbHiVuINBbGWWnhOuhBqBpMR4nqvBv1EsJsnzegs0iD58hbM5o+FXRtKj9vPZV8aK7yBMgj0cJW0TuYDpLhGdOPV8xzLpidudQ2EgHUSAoqLXOHysB9lyv3WCB5XKjiio2fjCHgeBRC2Z+kplrJN+AGIRmk7ZJTQlzU7p+hyFLYkLytZUNBFLF5PEAXXNIo5ie5YjyhTUi53YiS6q2V9gVFfq8WOa63928tYL4eUzfxYg1RT0fVGFre21yUZVnrhMGzjg8Q+0jeImkYQvzjjSyj9gF3JOU1Y0\\\\\\\/oYjDX46m6Fjmi5bDyVW9W12BuasoJh85vVJsxgzACifGePfD8ctbyLi4Og4d4A57q0Le9\\\\\\\/d0nJeLPn08Z6LX3ntQtL6rUKUhSTpO44L+HNh5CEzYFk\\\\\\\/cJfAOuxEPjGwRiHmEPzYGBCoyDFeg==\\\u0022,\\\u0022iv\\\u0022:\\\u002211R747pD51WRKL3HZ53jRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221bba760ad85767705bc4140595aa5881313d41ee290e5228bedbde32919ee630\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460182 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2463" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3SLZXJ0jzLLNKrVkVR-TCOIRRp_sW4Z3X32cjFBrKrs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--fFNiUpDIebOeHtzAV08hNVC9SB1nYFDA0BmqZfkf30" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2453" title="Whole-Genome Sequencing Suggests Schizophrenia Risk Mechanisms in Humans with 22q11.2 Deletion Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2475" title="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460182" id="node460182" data-pisa="ggg;5/11/2463" data-pisa-master="ggg;g3.115.020222" data-apath="/ggg/5/11/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morten T. Limborg</span>, <span class="highwire-citation-author" data-delta="1">Ryan K. Waples</span>, <span class="highwire-citation-author" data-delta="2">Fred W. Allendorf</span> and <span class="highwire-citation-author" data-delta="3">James E. Seeb</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020222 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460182" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morten T. Limborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute of Aquatic Resources, Technical University of Denmark, Vejlsøvej 39, Silkeborg, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BT.%2BLimborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Limborg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorten%2BT.%2BLimborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morten T. Limborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morten.limborg@gmail.com" class="" data-icon-position="" data-hide-link-title="0">morten.limborg@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan K. Waples </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BK.%2BWaples%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waples%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BK.%2BWaples%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred W. Allendorf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Montana, Missoula, Montana 59812</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BW.%2BAllendorf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allendorf%20FW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BW.%2BAllendorf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2463" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460182" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2463.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460182" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2463.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460182" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2463.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460182" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2463.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460182" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460182/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Meiotic recombination is fundamental for generating new genetic variation and for securing proper disjunction. Further, recombination plays an essential role during the rediploidization process of polyploid-origin genomes because crossovers between pairs of homeologous chromosomes retain duplicated regions. A better understanding of how recombination affects genome evolution is crucial for interpreting genomic data; unfortunately, current knowledge mainly originates from a few model species. Salmonid fishes provide a valuable system for studying the effects of recombination in nonmodel species. Salmonid females generally produce thousands of embryos, providing large families for conducting inheritance studies. Further, salmonid genomes are currently rediploidizing after a whole genome duplication and can serve as models for studying the role of homeologous crossovers on genome evolution. Here, we present a detailed interrogation of recombination patterns in sockeye salmon (<em>Oncorhynchus nerka</em>). First, we use RAD sequencing of haploid and diploid gynogenetic families to construct a dense linkage map that includes paralogous loci and location of centromeres. We find a nonrandom distribution of paralogs that mainly cluster in extended regions distally located on 11 different chromosomes, consistent with ongoing homeologous recombination in these regions. We also estimate the strength of interference across each chromosome; results reveal strong interference and crossovers are mostly limited to one per arm. Interference was further shown to continue across centromeres, but metacentric chromosomes generally had at least one crossover on each arm. We discuss the relevance of these findings for both mapping and population genomic studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gynogenesis" class="hw-term hw-article-keyword hw-article-keyword-gynogenesis" rel="nofollow">gynogenesis</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/interference" class="hw-term hw-article-keyword hw-article-keyword-interference" rel="nofollow">interference</a></li><li class="kwd"><a href="/keyword/genome-data" class="hw-term hw-article-keyword hw-article-keyword-genome-data" rel="nofollow">genome data</a></li></ul><p id="p-3">Recombination plays a crucial role in creating novel genetic variation in sexually reproducing species (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barton and Charlesworth 1998</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Otto and Lenormand 2002</a>) and for securing proper disjunction of sister chromatids and chromosomes during meiosis. Chiasma interference (hereafter simply referred to as interference) is a fundamental process that influences crossover locations because the formation of a chiasma reduces the chance of a nearby recombination (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sturtevant 1915</a>). Interference was thought to only occur within independent chromosome arms because the centromere acted as a barrier to interference (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mather 1938</a>).</p><p id="p-4">Another important role of recombination includes crossing over between homeologs; these crossovers slow rediploidization of polyploid-origin genomes resulting from recent whole genome duplications (WGD). Pioneering studies considered allozyme loci in lake trout and brook trout hybrids to characterize segregation of duplicated loci in males and presented a meiotic model for explaining this residual tetrasomic inheritance (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">May <em>et al.</em> 1979</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wright <em>et al.</em> 1983</a>). These meiotic models were recently synthesized to further promote awareness of these processes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">May and Delany 2015</a>). Indeed, WGDs have played an important role in the evolution of many polyploid taxa that often serve as complex, but enlightening, models for understanding genome evolution (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Comai 2005</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Parisod <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mable 2013</a>). Knowledge about how meiotic recombination affects the evolution and structure of genomes is crucial to better understand and interpret genomic data.</p><p id="p-5">Historically, basic meiotic processes have mainly been described in model organisms, but today advances in sequencing technology have catalyzed dense genome mapping in nonmodel species. Linkage maps have informed population-based studies, including many of polyploid-origin taxa. However, detailed interrogations of recombination patterns in nonmodel species still lag behind surveys from a few select model organisms (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Basu-Roy <em>et al.</em> 2013</a>). Insights on recombination patterns from a broader range of taxa, including polyploid-origin species, are warranted.</p><p id="p-6">Understanding how recombination shapes genetic diversity across genomes is also crucial for interpreting signals of population divergence in genomic data. The number and distribution of so-called islands of elevated divergence have received immense attention in recent literature on ecological speciation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ellegren <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jones <em>et al.</em> 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tine <em>et al.</em> 2014</a>). Recombination rates have been invoked as an important factor explaining the size and genomic location of such islands (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Renaut <em>et al.</em> 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cruickshank and Hahn 2014</a>). A better understanding of how recombination mechanisms vary across a genome will help explain the relative contribution of neutral drift <em>vs.</em> divergent selection in creating local regions of elevated divergence among ecotypes or populations within a species (see also discussion in <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Roesti <em>et al.</em> 2013</a>).</p><p id="p-7">Salmonid fishes are an excellent system for studying the role of recombination in genome evolution for a number of reasons. Single pair matings produce thousands of embryos, enabling the examination of large numbers of meiosis from a single individual. Also, details of ploidy manipulation are well worked out; the use of haploid or gynogenetic diploid families greatly enhances genotyping and mapping capabilities (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Komen and Thorgaard 2007</a>).</p><p id="p-8">Early inheritance studies in salmonids, although often based on fewer than 50 allozyme loci, suggested strong crossover interference: in females, extended regions between telomeric loci and the centromeres invariably had a single crossover (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thorgaard <em>et al.</em> 1983</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 1986</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lindner <em>et al.</em> 2000</a>). These early findings were restricted to only a subset of chromosomes and did not include genome-wide interrogations of interference.</p><p id="p-9">We know that homolog recognition and pairing initiates at the telomeres (reviewed in <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Calderon <em>et al.</em> 2014</a>) and interference occurs across centromeres in humans (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Colombo and Jones 1997</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>) and zebrafish <em>(Danio rerio</em>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Demarest <em>et al.</em> 2011</a>). However, detailed genome-wide descriptions of interference are scarce for nonmodel organisms, and the few that exist reveal significant interspecific differences (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Segura <em>et al.</em> 2013</a>).</p><p id="p-10">Further, the salmonid ancestor went through a recent WGD (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ohno 1970</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>), and the rediploidization process is not complete. Early inheritance studies in salmonids described a complicated pattern of both disomic and residual tetrasomic inheritance for a suite of isoloci duplicated genes that share alleles (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">May <em>et al.</em> 1979</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf and Danzmann 1997</a>). These isoloci were found to reside mainly in telomeric regions (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Thorgaard <em>et al.</em> 1983</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Allendorf <em>et al.</em> 1986</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Seeb and Seeb 1986</a>). More recently, studies in a few Pacific salmonid species have combined genotyping by sequencing and the use of haploid mapping to map isoloci (<em>Oncorhynchus tshawytscha</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brieuc <em>et al.</em> 2014</a>; <em>O. kisutch</em>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kodama <em>et al.</em> 2014</a>; <em>O. keta</em>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Waples <em>et al.</em> 2015</a>). Results suggest that eight syntenic pairs of homeologous chromosome arms remain duplicated across species. These findings are further supported by insights from the rainbow trout (<em>O. mykiss</em>) genome sequence that shows conserved sequence identity and gene order between paired paralogous regions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Berthelot <em>et al.</em> 2014</a>). Data from additional species will contribute to a more complete understanding of how the WGD has shaped genome evolution in salmonids and other polyploid-origin species.</p><p id="p-11">Our objective is to use genetic mapping to improve understanding of recombination and interference. We combine the use of genotyping by sequencing data from gynogenetic haploid and gynogenetic diploid progeny from a single female sockeye salmon (<em>O. nerka</em>). We then: (1) produce a dense genetic map and locate the centromeres and retained duplications; (2) test for the occurrence and strength of interference; and (3) test for the occurrence of interference across centromeres.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Gynogenetic mapping families</h3><p id="p-12">We produced two families of gynogenetic progeny as described in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chourrout (1980)</a>. One gravid female and one male sockeye salmon from the Lake Sammamish population were stripped for eggs and sperm at the Issaquah State Salmon Hatchery (Washington, USA) in December 2012. Fin clips were taken from both parents and stored in ethanol. All progeny were produced by fertilizing all of the eggs with sperm that had been genetically inactivated with 10 min exposure to UV light (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A haploid family for linkage map construction was created by placing half of these haploid embryos into the incubator with no further treatment (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). We produced a second family of gynogenetic diploids (half-tetrads) by exposing the remaining embryos to 10 min heat shock at 28° to induce retention of the second polar body (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The first embryo hatched after 86 d; remaining embryos from both families were immediately sampled and stored in ethanol.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic overview of the construction of gynogenetic families used for linkage map construction. Gametes were collected from a single set of parents followed by UV irradiation of sperm ensuring that only maternal DNA was incorporated into developing embryos. After fertilization, half the gynogens were allowed to develop into gynogenetic haploids and the others were heat-shocked to produce gynogenetic diploids (Chourrout 1980). Both families were RAD sequenced for linkage map construction and recombination analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic overview of the construction of gynogenetic families used for linkage map construction. Gametes were collected from a single set of parents followed by UV irradiation of sperm ensuring that only maternal DNA was incorporated into developing embryos. After fertilization, half the gynogens were allowed to develop into gynogenetic haploids and the others were heat-shocked to produce gynogenetic diploids (Chourrout 1980). Both families were RAD sequenced for linkage map construction and recombination analyses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F1.medium.gif" width="440" height="158"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2463/F1.medium.gif" width="440" height="158"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467043" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Schematic overview of the construction of gynogenetic families used for linkage map construction. Gametes were collected from a single set of parents followed by UV irradiation of sperm ensuring that only maternal DNA was incorporated into developing embryos. After fertilization, half the gynogens were allowed to develop into gynogenetic haploids and the others were heat-shocked to produce gynogenetic diploids (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Chourrout 1980</a>). Both families were RAD sequenced for linkage map construction and recombination analyses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">DNA was extracted from fin clips and embryos using DNEasy-96 kits (Qiagen, Valencia, CA, USA) following the manufacturer’s directions. Whole embryos were dissected from the yolk and chorion and added directly to the lysis buffer. Both parents as well as 142 putative haploid and 141 putative gynogenetic diploid progeny were genotyped for 96 EST-derived SNPs using 5′-nuclease assays (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Elfstrom <em>et al.</em> 2006</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Storer <em>et al.</em> 2012</a>). Genotypes were called using the BioMark software v3.0.2 (Fluidigm, South San Francisco, CA). Embryos with paternal alleles signaled failure of the UV treatment and were excluded from further analysis.</p><p id="p-15">Success of the heat shock treatment was evaluated by screening the same 96 SNPs from the 5′-nuclease assays in putative gynogenetic diploids for the occurrence of completely homozygous embryos. Fully homozygous embryos, for loci segregating in the female parent, would signal failure of the heat shock to incorporate the second polar body; those individuals were also excluded. Genotypes for the 5′-nuclease assay SNPs that were segregating in the gynogenetic families were also used for mapping after filtering for Mendelian inheritance and segregation distortion (see below).</p></div><div id="sec-3" class="subsection"><h3>Sequencing</h3><p id="p-16">Genotyping was performed by sequencing restriction site–associated DNA (RAD-seq). All sequencing was done at the University of Oregon High Throughput Sequencing Facility using an Illumina HiSeq2000. We generated sequencing libraries using the restriction enzyme <em>SbfI</em> following methods previously described (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> 2012</a>). We ligated unique barcodes (6 bp) to digested DNA following the work of <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Miller <em>et al.</em> (2012)</a>. The female parent was sequenced together with gynogenetic haploid and diploid progeny to 101 bp reads (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Samples were sequenced on five lanes: two lanes each included the female parent and 47 gynogenetic haploids and three lanes each included 32 gynogenetic diploid progeny.</p></div><div id="sec-4" class="subsection"><h3>Detection and genotyping of polymorphisms</h3><p id="p-17">We used the <em>Stacks</em> software package v1.04 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Catchen <em>et al.</em> 2013</a>) to identify polymorphic loci and to assign genotypes. We used the <em>process radtags</em> program to remove reads characterized by low-quality, uncalled bases, or with ambiguous barcodes. Retained reads were demultiplexed and trimmed to 94 bp by removing the barcode and terminal base.</p><p id="p-18">We then identified duplicated loci based on the segregation patterns of alleles in the offspring following the approach of <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Waples <em>et al.</em> (2015)</a>. More details about this approach are given in <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/020222SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FileS1.pdf">File S1</a>. Duplicated loci contain sequences from at least two distinct genomic locations and have segregation patterns deviating from disomic Mendelian inheritance (<em>i.e.</em>, confounded loci <em>sensu</em>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Waples <em>et al.</em> 2015</a>).</p><p id="p-19">RAD loci, including duplicated loci, were then genotyped for the haploid family and combined with genotypes for the 5′-nuclease SNPs for linkage map construction. Strict segregation distortion tests served to detect and discard any mis-specified loci. We excluded progeny with less than 0.5 × 10<sup>6</sup> reads because genotypes could not be reliably assigned with lower sequence coverage.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-20">The haploid family was used to construct the linkage map and the gynogenetic diploid family was used to locate centromeres (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-21">The map was generated by using the minimum spanning tree method implemented in the program MSTMap (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wu <em>et al.</em> 2008</a>). Duplicated loci were mapped following the work of <a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Waples <em>et al.</em> (2015)</a>. We compared our map with previously published linkage maps for sockeye salmon (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> 2012</a>) and rainbow trout (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Miller <em>et al.</em> 2012</a>) to aid construction and annotation of linkage groups (LGs). We name LGs by considering the numbers given by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> (2012)</a> preceded by a ‘So’ prescript (see <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS1.pdf">Table S1</a> for more details).</p><p id="p-22">Half-tetrad analysis of the gynogenetic diploid offspring was used to place centromeres. Raw Illumina reads were processed in <em>Stacks</em> as described for haploids. Only haplotypes for nonduplicated loci were exported. The need to accurately call heterozygote genotypes in the diploid progeny necessitated a higher cut-off threshold of 1.5 × 10<sup>6</sup> reads compared to the haploid progeny. We exported haplotype alleles from <em>Stacks</em> to call genotypes and merged these with genotypes for SNPs detected with the 5′-nuclease assays. Loci with &gt;10% missing genotypes were discarded.</p><p id="p-23">We estimated the frequency of second division segregation (<em>y</em>) for each locus following the work of <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Thorgaard <em>et al.</em> (1983)</a> to identify centromeric regions on each LG. Values of <em>y</em> are scored as observed heterozygosity in the gynogenetic diploid family and range between zero and one. A heterozygote genotype indicates that an odd number of recombination events have occurred between the locus and the centromere; a homozygous genotype indicates zero or an even number of recombination events. Centromeres are nonrecombining regions that occur as a single location on a linkage group; therefore, adjacent loci are expected to have low <em>y</em> values because recombinations will be rare over the short locus-centromere distances. For loci with low <em>y</em> values, it can be difficult to identify their orientation in relation to the centromere. Therefore, centromeric regions were conservatively defined as the shortest interval on an LG including all markers with <em>y</em> &lt; 0.10.</p><p id="p-24">Finally, loci were assigned to specific chromosomal regions, arms or centromeres, to complete the linkage map and to enable downstream analyses of interference. We labeled loci with <em>y</em> &lt; 0.10 as centromeric (c), and, for acrocentric chromosomes, loci with <em>y</em> &gt; 0.10 were assigned to reside on arm a1. For metacentric chromosomes, loci with <em>y</em> &gt; 0.10 were arbitrarily assigned to reside on arm a1 if located before the centromeric region [<em>i.e.</em>, smaller centimorgan (cM) value on the map], or to reside on arm a2 if located after the centromeric region. Finally, it is important to note that even with a few thousand loci, our coverage of the genome will often be insufficient to detect the short p arm for some acrocentric chromosomes.</p></div><div id="sec-6" class="subsection"><h3>Interference</h3><p id="p-25">First, we identified recombination events in both haploid and diploid gynogenetic families along each linkage group. We infer phase changes within haploid offspring to be the result of recombination using the parental phase inferred during linkage map construction. We count offspring with zero, one, or two crossover events per arm. In gynogenetic diploids, homozygotes and heterozygotes reflect different phases of the recombinant chromatid resulting from meiosis I. Accordingly, homozygotes and heterozygotes represent distinct maternal phases. Duplicated loci were not considered because alleles could not be assigned to a unique map location. This framework was then used to count number and location of recombination events within each offspring and for each LG (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Crossover events were detected as phase changes observed using a sliding window that recorded the mean phase over 11 consecutive marker locations. Up to two crossovers were placed along each chromosome arm. Because double crossover events were rare, we discarded putative third crossovers because these are likely to be the result of genotyping error at one or a few terminal loci (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brieuc <em>et al.</em> 2014</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of crossover patterns for four different gynogenetic haploids along LG So2. The two maternal allelic phases are shown on the y-axis. The gray area identifies the centromeric region (see text). Upper left plot shows a haploid offspring with no recombination events, an occurrence in approximately 50% of meiotic products when all four tetrads are sampled. The upper right shows an offspring with one crossover on each arm. The lower left plot depicts an offspring with one crossover on arm a1. The lower right plot shows an example of a haploid offspring with two crossovers on arm a2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of crossover patterns for four different gynogenetic haploids along LG So2. The two maternal allelic phases are shown on the y-axis. The gray area identifies the centromeric region (see text). Upper left plot shows a haploid offspring with no recombination events, an occurrence in approximately 50% of meiotic products when all four tetrads are sampled. The upper right shows an offspring with one crossover on each arm. The lower left plot depicts an offspring with one crossover on arm a1. The lower right plot shows an example of a haploid offspring with two crossovers on arm a2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F2.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2463/F2.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467053" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Examples of crossover patterns for four different gynogenetic haploids along LG So2. The two maternal allelic phases are shown on the y-axis. The gray area identifies the centromeric region (see text). Upper left plot shows a haploid offspring with no recombination events, an occurrence in approximately 50% of meiotic products when all four tetrads are sampled. The upper right shows an offspring with one crossover on each arm. The lower left plot depicts an offspring with one crossover on arm a1. The lower right plot shows an example of a haploid offspring with two crossovers on arm a2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We estimated the genome-wide distribution of <em>y</em> to assess the potential occurrence of interference. With no interference, and therefore fully independent crossover locations, the maximum expected value of <em>y</em> is 0.67 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Anderson 1925</a>). In contrast, with complete interference, <em>y</em> is expected to reach 1.00 at distal locations because there will always be exactly one crossover between distal loci and the centromere.</p><p id="p-28">Next, we estimated the strength of interference by fitting a gamma model to the distribution of observed interchiasma distances using CODA (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gauthier <em>et al.</em> 2011</a>). The shape parameter (ν) of the fitted gamma distribution is referred to as the interference parameter; values between zero and one indicate negative interference (<em>i.e.</em>, crossover locations are more tightly clustered than expected at random). A value of one signals no interference and that crossover locations are independent of each other (<em>i.e.</em>, follow an exponential distribution), and values above one signal positive interference, with higher values indicating stronger interference. Interchiasma distances were only recorded for the diploid progeny to avoid any potential bias from merging observations with the haploid progeny that were used to generate the map distances. The distribution of interchiasma distances was then fit using the biologically realistic two parameter model that allows some fraction (<em>p</em>) of observed crossovers to follow a noninterfering pathway (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Falque <em>et al.</em> 2009</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gauthier <em>et al.</em> 2011</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Basu-Roy <em>et al.</em> 2013</a>). As recommended by <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Gauthier <em>et al.</em> (2011)</a>, we used the hill-climbing algorithm to search a parameter space with ν = [1:20] and <em>p</em> = [0:1] to find the best fit for each LG. For LGs where estimates of ν equaled 20, the analysis was repeated with increased boundaries. The CIs around estimates of ν were calculated using the Fisher information matrix (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Gauthier <em>et al.</em> 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Interference across centromeres</h3><p id="p-29">We tested for the occurrence of interference across centromeres on metacentric chromosomes by considering all offspring showing at least one crossover on both arms. We followed the method described in <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Colombo and Jones (1997)</a> by using the Spearman correlation function:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2463/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2463/embed/mml-math-1.gif"/></noscript></span></span>Here, χ<sub>a1</sub> and χ<sub>a2</sub> are the genetic distance from the midpoint of defined centromeric region to the first crossover location on arms a1 and a2. We considered different interval sizes around the centromeric midpoint (<em>d</em>) ranging from 10 cM, minimum distance allowing enough observations, and sequentially increasing <em>d</em> by 1 cM. A negative correlation indicates interference across centromeres, because a crossover close to the centromere translates into a larger-than-expected distance to the first crossover on the opposite arm. Data were pooled across all metacentric LGs because our data have too few observations to consider individual LGs. We estimated <em>ρ</em>(<em>d</em>) for both haploid and diploid families; families were treated independently for the reasons given above.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-30">Raw sequence data are deposited in the Short Read Archive (SRA) with accession number SRP063568. Genotypes for both haploid and diploid progeny are deposited on Dryad (doi: 10.5061/dryad.q675s).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Linkage map</h3><p id="p-31">An average number of 2.5 × 10<sup>6</sup> reads for 93 haploids and 3.0 × 10<sup>6</sup> reads for 77 diploids (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS2.pdf">Table S2</a>) resulted after quality filtering, barcode recovery, demultiplexing, and discarding individuals with low coverage. A total of 3496 loci remained after excluding loci uninformative for mapping (monomorphic in the female parent or with &gt;25% missing genotypes in the offspring). Of these, 868 loci were classified as duplicated. Adding the 31 polymorphic 5′-nuclease SNPs resulted in 3527 loci available for linkage map construction.</p><p id="p-32">Initial linkage group construction using the haploid family produced 30 LGs containing between 21 and 184 markers each (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS3.xlsx">Table S3</a>). Comparison to an existing but less dense map for sockeye salmon served to validate and name LGs (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FileS1.pdf">File S1</a>). In two instances, two of our linkage groups matched the same LG in the map of <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> (2012)</a>; these were denoted by A and B after the LG number. Two LGs corresponded to LG9 in <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> (2012)</a> and syntenic comparison with rainbow trout identified these LGs to be the two female sockeye salmon sex chromosomes X<sub>1</sub> and X<sub>2</sub> (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FileS1.pdf">File S1</a>); these were named So9A_(X<sub>2</sub>) and So9B_(X<sub>1</sub>) following previous descriptions (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Thorgaard 1978</a>). For the LGs So18A and So18B pair, So18B does not have a centromere (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>); here, we consider LGs So18A and So18B to be the two arms of a single metacentric chromosome (LG18 in <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Everett <em>et al.</em> 2012</a>). The total map length was 2839 cM and included 2640 nonduplicated and 605 duplicated loci (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS3.xlsx">Table S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated y values plotted along each linkage group. Each plot represents an LG with mapped loci presented by circles in the top with nonduplicated loci shown as gray circles and duplicated loci shown as red circles. Estimated y values are plotted on the y-axis for each nonduplicated marker and shown as black circles. The horizontal dotted line equals the maximum expected value of y (0.67) without interference. Loci with low y values are expected to locate proximal to centromeres, and higher values are expected for loci with increasing distance from centromeres. Overall, these plots serve to effectively distinguish acrocentric (e.g., LG So1) from metacentric (e.g., LG So2) chromosomes and define regions that include the centromere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimated &lt;em&gt;y&lt;/em&gt; values plotted along each linkage group. Each plot represents an LG with mapped loci presented by circles in the top with nonduplicated loci shown as gray circles and duplicated loci shown as red circles. Estimated &lt;em&gt;y&lt;/em&gt; values are plotted on the y-axis for each nonduplicated marker and shown as black circles. The horizontal dotted line equals the maximum expected value of &lt;em&gt;y&lt;/em&gt; (0.67) without interference. Loci with low &lt;em&gt;y&lt;/em&gt; values are expected to locate proximal to centromeres, and higher values are expected for loci with increasing distance from centromeres. Overall, these plots serve to effectively distinguish acrocentric (&lt;em&gt;e.g.&lt;/em&gt;, LG So1) from metacentric (&lt;em&gt;e.g.&lt;/em&gt;, LG So2) chromosomes and define regions that include the centromere.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F3.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2463/F3.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467038" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Estimated <em>y</em> values plotted along each linkage group. Each plot represents an LG with mapped loci presented by circles in the top with nonduplicated loci shown as gray circles and duplicated loci shown as red circles. Estimated <em>y</em> values are plotted on the y-axis for each nonduplicated marker and shown as black circles. The horizontal dotted line equals the maximum expected value of <em>y</em> (0.67) without interference. Loci with low <em>y</em> values are expected to locate proximal to centromeres, and higher values are expected for loci with increasing distance from centromeres. Overall, these plots serve to effectively distinguish acrocentric (<em>e.g.</em>, LG So1) from metacentric (<em>e.g.</em>, LG So2) chromosomes and define regions that include the centromere.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The gynogenetic diploid family was used to place centromeres; 2562 of the 2640 nonduplicated loci on the linkage map were successfully genotyped for the diploid progeny. Centromeric regions were located by plotting <em>y</em> values along the linkage map (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Acrocentric chromosomes were characterized by a linear pattern of <em>y</em> along linkage groups (<em>e.g.</em>, So1), and clearly distinguishable from metacentric chromosomes characterized by a V-shaped <em>y</em> plot (<em>e.g.</em>, So3) with values below 0.10 defining centromeric regions. After centromere placement, the 30 LGs (2n = 60) translated into a total of 102 chromosome arms (NF = 102; <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS3.xlsx">Table S3</a>) which is within the range of previously reported karyotypes for sockeye salmon (NF = 100–104) (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Thorgaard 1978</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Phillips and Rab 2001</a>).</p></div><div id="sec-11" class="subsection"><h3>Distribution of retained duplicated regions</h3><p id="p-35">Duplicated loci were not randomly distributed across the genome. Mapping of two paralogs from 40 duplicated loci enabled identification of six pairs of homeologous regions located on 11 different LGs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The majority of duplicated loci (457, 76%) mapped to these 11 LGs, including both arms on So21. Within these 11 LGs, duplicated loci represented between 22 and 66% of all loci and concentrated towards telomeres within distinct arms (red circles in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>; <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS3.xlsx">Table S3</a>). Twelve pairs of paralogs colocated within 10 cM on the same LG and likely represent local duplication events that do not originate from the WGD (<a id="xref-ref-71-5" class="xref-bibr" href="#ref-71">Waples <em>et al.</em> 2015</a>). Another extended, but unpaired, region dominated by duplicated loci occurred on arm a1 of So27; remaining duplicated loci, possibly segmental duplications, were dispersed across the genome (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467128/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467128/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pairing of homeologous chromosome arms in sockeye salmon</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Interference along chromosome arms</h3><p id="p-38">The number of observed recombination events for each arm reveals a pattern of strong, but incomplete, interference. Among haploid progeny, only 64 of 1902 (3.4%) observations showed two crossovers within a single arm (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Similarly, we observed 178 of 3777 (4.7%) arms with more than one crossover among the diploid progeny (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Further evidence for interference was revealed by a high fraction of distal loci with <em>y</em> values above 0.67 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467098/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467098/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Observed numbers of haploid and diploid gynogenetic offspring with zero, one, or two crossovers are given for each linkage group and for each arm</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide distribution of y values estimated for all nonduplicated loci in the gynogenetic diploid family. Loci with values near zero are expected to be proximal to centromeres, and loci with higher values are expected to be more distally located. Values of y above 0.67 are only expected with the occurrence of positive crossover interference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide distribution of &lt;em&gt;y&lt;/em&gt; values estimated for all nonduplicated loci in the gynogenetic diploid family. Loci with values near zero are expected to be proximal to centromeres, and loci with higher values are expected to be more distally located. Values of &lt;em&gt;y&lt;/em&gt; above 0.67 are only expected with the occurrence of positive crossover interference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F4.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2463/F4.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Genome-wide distribution of <em>y</em> values estimated for all nonduplicated loci in the gynogenetic diploid family. Loci with values near zero are expected to be proximal to centromeres, and loci with higher values are expected to be more distally located. Values of <em>y</em> above 0.67 are only expected with the occurrence of positive crossover interference.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We used the observed distribution of interchiasma distances to estimate the strength of interference across chromosomes. Estimates of ν equaled the upper limit of 20 for five LGs (So9A_(X<sub>2</sub>), So9B_(X<sub>1</sub>), So18A, So25, and So26). When the upper limit of ν in the searched parameter space was increased, the ν estimates continued to equal the maximum allowed value (<em>i.e.</em>, ν → ∞). Common to these five LGs is that they represent one chromosome arm with few, if any, double crossovers; this likely impeded the fitting of a gamma distribution for the few observed interchiasma distances (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Therefore, we do not report estimates of ν for these LGs, but conclude that the data signal strong interference as well (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Basu-Roy <em>et al.</em> 2013</a>; <a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS4.pdf">Table S4</a>). The average point estimate among remaining LGs (ν = 7.3) was considerably larger than the value expected without interference (ν = 1). These observations were further supported by low estimates of <em>p</em> (<em>p</em> = 0.00–0.16) (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS4.pdf">Table S4</a>) indicating that most crossovers are affected by interference (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Falque <em>et al.</em> 2009</a>). <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows estimates of ν for each LG. All LGs show positive interference, although CIs for So4, So16, and So28 include one, the expected value with no interference. Further, CIs around ν did not include the genome-wide average for seven LGs (So5, So6, So13, So15, So17, So27, So28), suggesting that strength of interference varies among chromosomes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). No clear correlation was found when plotting ν as a function of genetic length of each LG (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum-likelihood estimates of the interference parameter &#x3BD; from the gamma model (black circles). Vertical gray lines show the 95% C.I. around each &#x3BD; estimate. If interference does not affect crossover location, then &#x3BD; will take a value of 1.0 (horizontal dotted line). The genome-wide average (&#x3BD; = 7.3) is shown by the solid horizontal line. Estimates of &#x3BD; were not plotted for six LGs (So9A_(X2), So9B_(X1), So18A, So18B, So25, and So26) because too few double crossovers on these linkage groups impeded reliable estimation of &#x3BD; (see text)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum-likelihood estimates of the interference parameter &#x3BD; from the gamma model (black circles). Vertical gray lines show the 95% C.I. around each &#x3BD; estimate. If interference does not affect crossover location, then &#x3BD; will take a value of 1.0 (horizontal dotted line). The genome-wide average (&#x3BD; = 7.3) is shown by the solid horizontal line. Estimates of &#x3BD; were not plotted for six LGs (So9A_(X&lt;sub&gt;2&lt;/sub&gt;), So9B_(X&lt;sub&gt;1&lt;/sub&gt;), So18A, So18B, So25, and So26) because too few double crossovers on these linkage groups impeded reliable estimation of &#x3BD; (see text).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F5.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2463/F5.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467121" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Maximum-likelihood estimates of the interference parameter ν from the gamma model (black circles). Vertical gray lines show the 95% C.I. around each ν estimate. If interference does not affect crossover location, then ν will take a value of 1.0 (horizontal dotted line). The genome-wide average (ν = 7.3) is shown by the solid horizontal line. Estimates of ν were not plotted for six LGs (So9A_(X<sub>2</sub>), So9B_(X<sub>1</sub>), So18A, So18B, So25, and So26) because too few double crossovers on these linkage groups impeded reliable estimation of ν (see text).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We detected striking drops of <em>y</em> values toward telomeric regions of some LGs; this pattern was particularly pronounced on So2 (arm a2) and So22 (arm a1; <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). These observations result from an increased number of gynogenetic diploid progeny that show two recombinations along these arms (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) because loci located distal to a second chiasma will reduce estimates of <em>y</em>. Arm a2 of So2 showed an increased number of double crossovers in the haploids as well, whereas this was not the case for So22 arm a1 (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). In general, occurrence of double crossovers was more common on longer chromosome arms, but absence of double crossovers in some of the longer arms suggest that genetic length is not the only factor determining the frequency of multiple crossovers (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of observed double crossovers plotted against arm length. Symbols are shown in transparent gray to highlight over plotting. Results are given for haploid and diploid families separately because observations for diploid offspring do not include duplicated regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of observed double crossovers plotted against arm length. Symbols are shown in transparent gray to highlight over plotting. Results are given for haploid and diploid families separately because observations for diploid offspring do not include duplicated regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F6.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/11/2463/F6.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466989" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Number of observed double crossovers plotted against arm length. Symbols are shown in transparent gray to highlight over plotting. Results are given for haploid and diploid families separately because observations for diploid offspring do not include duplicated regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Interference across centromeres</h3><p id="p-45">We estimated the Spearman correlation coefficient (<em>ρ</em>) between distances from centromeres to the nearest chiasma location on both arms for the 20 metacentric LGs. For the haploid data, no values of <em>ρ</em> were significantly different from 0, whereas estimates for diploids were significantly negative for <em>d</em> &lt; 15 cM (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Nevertheless, the haploid data showed a similar pattern of negative correlations for <em>d</em> &lt; 15 cM. Lack of significant correlations in the haploid family may be due to reduced statistical power from the limited number of observations; the similar patterns observed between the haploid and diploid families lead us to conclude that interference does affect crossover patterns across centromeres.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interference across centromeres. Estimates of the correlation function &#x3C1;(d) = corr(&#x3C7;a1, &#x3C7;a2|max([&#x3C7;a1, &#x3C7;a2] &#x2264; d) for different windows of genetic distance (d) in both directions from the midpoint of defined centromeric regions. Estimates of &#x3C1;(d) were obtained for each integer value of d ranging from 10 to 75 cM. The gray area denotes the 95% C.I. around &#x3C1;(d). Observations are for all diploid offspring and pooled across all 20 metacentric LGs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916832429" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interference across centromeres. Estimates of the correlation function &lt;em&gt;&#x3C1;&lt;/em&gt;(&lt;em&gt;d&lt;/em&gt;) = corr(&#x3C7;&lt;sub&gt;a1&lt;/sub&gt;, &#x3C7;&lt;sub&gt;a2&lt;/sub&gt;|max([&#x3C7;&lt;sub&gt;a1&lt;/sub&gt;, &#x3C7;&lt;sub&gt;a2&lt;/sub&gt;] &#x2264; &lt;em&gt;d&lt;/em&gt;) for different windows of genetic distance (&lt;em&gt;d&lt;/em&gt;) in both directions from the midpoint of defined centromeric regions. Estimates of &lt;em&gt;&#x3C1;&lt;/em&gt;(&lt;em&gt;d&lt;/em&gt;) were obtained for each integer value of &lt;em&gt;d&lt;/em&gt; ranging from 10 to 75 cM. The gray area denotes the 95% C.I. around &lt;em&gt;&#x3C1;&lt;/em&gt;(&lt;em&gt;d&lt;/em&gt;). Observations are for all diploid offspring and pooled across all 20 metacentric LGs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2463/F7.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/11/2463/F7.medium.gif" width="361" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2463/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">Interference across centromeres. Estimates of the correlation function <em>ρ</em>(<em>d</em>) = corr(χ<sub>a1</sub>, χ<sub>a2</sub>|max([χ<sub>a1</sub>, χ<sub>a2</sub>] ≤ <em>d</em>) for different windows of genetic distance (<em>d</em>) in both directions from the midpoint of defined centromeric regions. Estimates of <em>ρ</em>(<em>d</em>) were obtained for each integer value of <em>d</em> ranging from 10 to 75 cM. The gray area denotes the 95% C.I. around <em>ρ</em>(<em>d</em>). Observations are for all diploid offspring and pooled across all 20 metacentric LGs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-47">We demonstrate the power of using both haploid and diploid gynogenetic offspring from a single-pair mating to describe detailed patterns of recombination in a duplicated salmonid genome. Results reveal extensive distal regions dominated by duplicated loci as well as strong, but varying, levels of interference for most LGs. Our use of a single female parent comes with some limitations for drawing general conclusions: our data only reflect information from heterozygous loci in the single female parent. Likewise, variability among populations, individuals, and sexes are not captured (<em>c.f.</em>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Johnston <em>et al.</em> 2015</a>).</p><div id="sec-15" class="subsection"><h3>Extensive subtelomeric regions remain duplicated in sockeye salmon</h3><p id="p-48">Linkage mapping revealed the occurrence of numerous regions with high levels of retained sequence identity between putative homeologs in sockeye salmon. These observations are consistent with an ongoing rediploidization of the sockeye salmon genome where some homeologous chromosomes still undergo residual tetrasomic inheritance (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">May <em>et al.</em> 1979</a>), a pattern that is shared among many extant species within the salmonid family (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Allendorf <em>et al.</em> 2015</a>).</p><p id="p-49">We detected six pairs of homeologous regions dominated by duplicated loci as well as an unpaired region on the So27 arm a1. A total of eight conserved pairs of duplicated homeologs have been described for a range of <em>Oncorhynchus</em> (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kodama <em>et al.</em> 2014</a>) and <em>Salmo</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lien <em>et al.</em> 2011</a>) species. Although we cannot rule out the possibility that some of these regions have fully diploidized in sockeye salmon based on these data, a more parsimonious explanation is that we have failed to identify another two pairs because we considered too few meiosis (progeny).</p><p id="p-50">Many genomic studies of polyploids, including salmonids, exclude duplicates to avoid inclusion of loci that deviate from standard assumptions such as Hardy-Weinberg proportions (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gagnaire <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hyma and Fay 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Limborg <em>et al.</em> 2014</a>). Consequently, extended regions that remain duplicated are not included in most population genomic studies. The filtering of duplicated loci is especially pertinent to polyploid species where extended telomeric regions are excluded in downstream scans for selection. Important genes located in these duplicated regions will remain undetected, leading to an incomplete understanding of the genetic basis of local adaptation (see discussion in <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Allendorf <em>et al.</em> 2015</a>).</p><p id="p-51">Finally, our study demonstrates how comparative mapping among species represents a powerful tool for validating <em>de novo</em> linkage maps in nonmodel species. Indeed, conserved restriction enzyme cut sites among genomes in related species make RAD sequencing, and related techniques, particularly useful for identifying orthologous chromosomal regions through comparative mapping (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/FileS1.pdf">File S1</a>); a feature that is expected to enrich mapping studies among related species.</p></div><div id="sec-16" class="subsection"><h3>Recombination patterns reveal strong interference</h3><p id="p-52"><a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Thorgaard <em>et al.</em> (1983)</a> first reported strong interference across the salmonid genome based on data from only 10 allozyme loci, three of which had <em>y</em> values at or close to 1.0. We expand the evidence for strong interference by mapping 3245 loci, finding loci with <em>y</em> values approaching 1.0 on nearly all linkage groups. It is important to note that we only used nonduplicated loci for estimating <em>y</em>. Thus, the proportion of loci with <em>y</em> values above 0.67 is likely to be an underrepresentation of the true number (<em>c.f.</em>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lindner <em>et al.</em> 2000</a>).</p><p id="p-53">Interestingly, similar to our finding, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Lindner <em>et al.</em> (2000)</a> also observed an overrepresentation of loci with low values of <em>y</em> (see <em>y</em> ≤ 0.1 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This peak can be explained by stronger crossover suppression near centromeres, which has been observed in a number of model species (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Koehler <em>et al.</em> 1996</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lamb <em>et al.</em> 2005</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rockmill <em>et al.</em> 2006</a>). One explanation involves selection against crossovers proximal to the centromere that has been shown to destabilize meiotic segregation and increase occurrence of nondisjunction (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Talbert and Henikoff 2010</a>). Assuming an even genome coverage of loci, our observation of an increased number of loci with low <em>y</em> values supports a similar model of selection against crossovers proximal to centromeres in sockeye salmon.</p><p id="p-54">Our study adds new insights about genome-wide interference in a salmonid genome by presenting the first quantitative estimates of the strength of interference across each chromosome. Positive interference for most, if not all, chromosomes described for sockeye salmon in this study is common to a range of other eukaryotes including plants (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lhuissier <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Giraut <em>et al.</em> 2011</a>) and mammals (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Broman <em>et al.</em> 2002</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Segura <em>et al.</em> 2013</a>). Nevertheless, most existing data come from model species, and the strength of interference varies greatly among taxa (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Giraut <em>et al.</em> 2011</a>), illustrating the need for obtaining species-specific estimates. Importantly, we demonstrate that data from mapping studies that use genotyping by sequencing in nonmodel species can be used to obtain quantitative estimates of interference. Because no additional data are needed, we advocate a routine execution of interference analyses in future mapping studies. This will lead to a deeper understanding of variation across more taxa and of how recombination interference is affecting genome evolution in general.</p><p id="p-55">Observations of more than one crossover, coupled with subtelomeric regions showing declining values of <em>y</em>, illustrate that interference is not complete, a pattern supported by observations in both haploid and diploid families. The occurrence of double crossovers was generally more common on longer chromosome arms (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Our measurement of recombination length, without knowledge of the corresponding physical length, complicates interpretation. It is tempting to posit that interference simply erodes with chromosome arm length, increasing the chance of observing multiple crossovers for longer chromosomes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Broman <em>et al.</em> 2002</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Giraut <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mary <em>et al.</em> 2014</a>). However, significant outlier LGs belie this interpretation (see <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>): LG So17 is a short acrocentric (42 cM), yet it has 15 double crossovers; So14 and So27 have two of the longest arms (≥70 cM), yet they have no double crossovers. Clearly the recombination length of chromosome arms is not the only factor affecting interference, and recombination patterns are likely dictated by a complex blend of different meiotic mechanisms.</p><p id="p-56">Varied strength of interference among chromosomes would have strong implications for interpreting genome data; unfortunately, there are little, if any, data on this issue in nonmodel taxa. Although no clear correlation between strength of interference and genetic distance is observed here, the two shortest LGs, So17 and So28, also have the lowest estimates of ν (<a href="http://www.g3journal.org/content/suppl/2015/09/18/g3.115.020222.DC1/TableS4.pdf">Table S4</a>). Evidence from yeast and humans also shows that interference is weaker on small chromosomes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kaback <em>et al.</em> 1992</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kaback 1996</a>). However, our observation of interchromosomal differences has to be interpreted with care. All LGs are shorter than 150 cM and contain few double crossovers that inevitably translate into uncertainties because ν estimates are conditioned on the observation of interchiasma distances (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>). Nevertheless, we do detect a genome-wide pattern of strong, but incomplete, interference with some observed variation in the strength of interference among chromosomes; these results have important implications for the analyses of genomic data (see below).</p></div><div id="sec-17" class="subsection"><h3>Interference occurs across centromeres</h3><p id="p-57">We found evidence that interference occurs across a region spanning ∼15 cM on either side of the centromere; this implies that crossovers occurring near the centromere affect recombination events on the opposite arm. Transcentromere interference has been reported in other species groups, including insects (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Colombo and Jones 1997</a>), mammals (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Mary <em>et al.</em> 2014</a>), and fish (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Demarest <em>et al.</em> 2011</a>). In our data, the effect declines rapidly with increasing distance from the centromere and disappears at distances greater than 15 cM. This rapid decline of transcentromere interference contrasts with patterns observed in humans (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>) and pigs (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Mary <em>et al.</em> 2014</a>).</p></div><div id="sec-18" class="subsection"><h3>Implications for analyzing genomic data</h3><p id="p-58">Recombination is an important evolutionary process that generates novel genetic variation. Compelling theoretical constructs predict an evolutionary advantage of increased recombination rates (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Felsenstein 1974</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Barton and Charlesworth 1998</a>), and empirical evidence for this theory has been found in, for example, <em>Escherichia coli</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cooper 2007</a>) and <em>Drosophila melanogaster</em> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Presgraves 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Campos <em>et al.</em> 2014</a>). Yet, it has proven nontrivial to broadly demonstrate such an effect because studies have been restricted to a few model species and because recombination rates vary among chromosomes and the few taxonomic groups studied (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cutter and Payseur 2013</a>). In this study, we demonstrate how data generated for linkage mapping in nonmodel species can also be used to estimate interference. These methods can be used to estimate species-specific rates of interference whenever a linkage map is created, leading to a more complete understanding of how this fundamental meiotic process shapes genomic data across species.</p><p id="p-59">Our results will also aid interpretation of new analyses facilitated by genomic data. One such method involves examining length distributions of genomic runs of homozygosity (ROH) to infer migration or inbreeding history (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pool and Nielsen 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kirin <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Harris and Nielsen 2013</a>). The length of the ROH is affected by recombination patterns, because shorter runs are expected in regions with higher recombination rates. Interference is also expected to affect the length distribution of ROHs, especially if the strength of interference varies among the individuals and populations being compared (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>). The concrete effect of strong interference on these analyses remains unclear but deserves attention in future developments of these methods. A first step could include estimates of interference within each of the populations being compared; if the strength and pattern of interference are similar among populations, then it will be easier to justify conclusions based on ROH comparisons.</p><p id="p-60">Assumptions of interference strength are crucial to genetic mapping functions. Models assuming different levels of interference have been shown to produce different results based on the same data (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhao and Speed 1996</a>). Our genome-wide estimate of the interference parameter (ν = 7.3) in sockeye salmon is closer to that assumed by the Carter-Falconer mapping model (ν = 7.6) than the more often used Haldane (ν = 1.0) and Kosambi (ν = 2.6) functions (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Zhao and Speed 1996</a>; <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Broman and Weber 2000</a>). With the accumulation of more accurate estimates of species-specific interference levels it will be possible to improve species-specific mapping efforts; such improvements will have implications when mapping genes that control important phenotypic traits.</p></div></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">We thank Carita E. Pascal for assistance with molecular analyses. We also thank Darin Combs and colleagues for providing assistance and hatchery facilities for conducting the family crosses at Issaquah Hatchery (WA, USA). Wesley A. Larson and Lisa W. Seeb provided constructive comments on earlier drafts of this manuscript. M.T.L. was supported by the Danish Council for Independent Research’s career program Sapere Aude (grant #12-126687). F.W.A. was partially supported by U.S. National Science Foundation grant DEB 1258203.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-62">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020222/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020222/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-63"><em>Communicating editor: W. S. Davidson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-07"><span class="received-label">Received </span>July 7, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-14"><span class="accepted-label">Accepted </span>September 14, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Limborg <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2463.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tetraploidy and the evolution of salmonid fishes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">53</span> in <span class="cit-source">Evolutionary Genetics of Fishes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Secondary tetrasomic segregation of MDH-B and preferential pairing of homeologues in rainbow trout.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1083%26rft.epage%253D1092%26rft.atitle%253DSecondary%2BTetrasomic%2BSegregation%2Bof%2BMDH-B%2Band%2BPreferential%2BPairing%2Bof%2BHomeologues%2Bin%2BRainbow%2BTrout%26rft_id%253Dinfo%253Apmid%252F9093860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzEwODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leary</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Gene-centromere mapping of 25 loci in rainbow-trout.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D77%26rft.issue%253D5%26rft.spage%253D307%26rft.epage%253D312%26rft.atitle%253DGene-centromere%2Bmapping%2Bof%2B25%2Bloci%2Bin%2Brainbow%2Btrout%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijc3LzUvMzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.4" data-doi="10.1093/jhered/esv015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv015%26rft_id%253Dinfo%253Apmid%252F25838153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNi8zLzIxNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1925</span> <span class="cit-article-title">Crossing over in a case of attached X chromosomes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DE.%2BG.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D403%26rft.epage%253D417%26rft.atitle%253DCROSSING%2BOVER%2BIN%2BA%2BCASE%2BOF%2BATTACHED%2BX%2BCHROMOSOMES%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F17246282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxMC81LzQwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.6" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.7" data-doi="10.1126/science.281.5385.1986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Why sex and recombination?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">1986</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D281%26rft.issue%253D5385%26rft.spage%253D1986%26rft.epage%253D1990%26rft.atitle%253DWhy%2BSex%2Band%2BRecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5385.1986%26rft_id%253Dinfo%253Apmid%252F9748151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODEvNTM4NS8xOTg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.8" data-doi="10.1534/genetics.113.155549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basu-Roy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giraut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mézard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hot regions of noninterfering crossovers coexist with a nonuniformly interfering pathway in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155549%26rft_id%253Dinfo%253Apmid%252F24026099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNzY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.9" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.10" data-doi="10.1534/g3.113.009316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A dense linkage map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009316%26rft_id%253Dinfo%253Apmid%252F24381192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy80NDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.11" data-doi="10.1086/302923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of human crossover interference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1911</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D66%26rft.issue%253D6%26rft.spage%253D1911%26rft.epage%253D1926%26rft.atitle%253DCharacterization%2Bof%2Bhuman%2Bcrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302923%26rft_id%253Dinfo%253Apmid%252F10801387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10801387&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088373800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paigen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Crossover interference in the mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1123%26rft.epage%253D1131%26rft.atitle%253DCrossover%2BInterference%2Bin%2Bthe%2BMouse%26rft_id%253Dinfo%253Apmid%252F11901128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzExMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.13" data-doi="10.1038/srep06488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calderon</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prieto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The subtelomeric region is important for chromosome recognition and pairing during meiosis.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">6488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D6488%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep06488%26rft_id%253Dinfo%253Apmid%252F25270583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep06488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270583&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.14" data-doi="10.1093/molbev/msu056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The relation between recombination rate and patterns of molecular evolution and variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1010</span>–<span class="cit-lpage">1028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu056%26rft_id%253Dinfo%253Apmid%252F24489114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzQvMTAxMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.15" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.16" data-doi="10.1051/rnd:19800415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chourrout</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Thermal induction of diploid gynogenesis and triploidy in the eggs of the rainbow trout (<em>Salmo gairdneri</em> Richardson).</span> <abbr class="cit-jnl-abbrev">Reprod. Nutr. Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%252C%2Bnutrition%252C%2Bdevelopment%26rft.stitle%253DReprod%2BNutr%2BDev%26rft.aulast%253DChourrout%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D3A%26rft.spage%253D727%26rft.epage%253D733%26rft.atitle%253DThermal%2Binduction%2Bof%2Bdiploid%2Bgynogenesis%2Band%2Btriploidy%2Bin%2Bthe%2Beggs%2Bof%2Bthe%2Brainbow%2Btrout%2B%2528Salmo%2Bgairdneri%2BRichardson%2529.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Frnd%253A19800415%26rft_id%253Dinfo%253Apmid%252F7349440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/rnd:19800415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7349440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.17" data-doi="10.1038/sj.hdy.6881930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colombo</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chiasma interference is blind to centromeres.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">214</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D79%26rft.spage%253D214%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6881930%26rft_id%253Dinfo%253Apmid%252F9279014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6881930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9279014&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XP83400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.18" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.19" data-doi="10.1371/journal.pbio.0050225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination speeds adaptation by reducing competition between beneficial mutations in populations of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCooper%26rft.auinit1%253DT.%2BF.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De225%26rft.epage%253De225%26rft.atitle%253DRecombination%2Bspeeds%2Badaptation%2Bby%2Breducing%2Bcompetition%2Bbetween%2Bbeneficial%2Bmutations%2Bin%2Bpopulations%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050225%26rft_id%253Dinfo%253Apmid%252F17713986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17713986&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.20" data-doi="10.1111/mec.12796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reanalysis suggests that genomic islands of speciation are due to reduced diversity, not reduced gene flow.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3133</span>–<span class="cit-lpage">3157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D3133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12796%26rft_id%253Dinfo%253Apmid%252F24845075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845075&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338014900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.21" data-doi="10.1038/nrg3425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic signatures of selection at linked sites: unifying the disparity among species.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3425%26rft_id%253Dinfo%253Apmid%252F23478346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23478346&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.22" data-doi="10.1534/genetics.110.124081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demarest</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horsley</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunwald</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trans-centromere effects on meiotic recombination in the zebrafish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.124081%26rft_id%253Dinfo%253Apmid%252F20980237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMzMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.23" data-doi="10.1111/j.1471-8286.2006.01507.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elfstrom</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Thirty‐two single nucleotide polymorphism markers for high‐throughput genotyping of sockeye salmon.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D6%26rft.spage%253D1255%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2006.01507.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2006.01507.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.24" data-doi="10.1038/nature11584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backstrom</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic landscape of species divergence in <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D756%26rft.epage%253D760%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bof%2Bspecies%2Bdivergence%2Bin%2BFicedula%2Bflycatchers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11584%26rft_id%253Dinfo%253Apmid%252F23103876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103876&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.25" data-doi="10.1186/1471-2164-13-521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEverett%26rft.auinit1%253DM.%2BV.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D521%26rft.atitle%253DMeiotic%2Bmaps%2Bof%2Bsockeye%2Bsalmon%2Bderived%2Bfrom%2Bmassively%2Bparallel%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-521%26rft_id%253Dinfo%253Apmid%252F23031582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23031582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.26" data-doi="10.1105/tpc.109.071514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Two types of meiotic crossovers coexist in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3915</span>–<span class="cit-lpage">3925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.071514%26rft_id%253Dinfo%253Apmid%252F20040539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8zOTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.28" data-doi="10.1111/mec.12127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping phenotypic, expression and transmission ratio distortion QTL using RAD markers in the Lake Whitefish (<em>Coregonus clupeaformis</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3036</span>–<span class="cit-lpage">3048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3036%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.29" data-doi="10.1186/1471-2105-12-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CODA (crossover distribution analyzer): quantitative characterization of crossover position patterns along chromosomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DGauthier%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.spage%253D27%26rft.epage%253D27%26rft.atitle%253DCODA%2B%2528crossover%2Bdistribution%2Banalyzer%2529%253A%2Bquantitative%2Bcharacterization%2Bof%2Bcrossover%2Bposition%2Bpatterns%2Balong%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-27%26rft_id%253Dinfo%253Apmid%252F21251248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21251248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.30" data-doi="10.1371/journal.pgen.1002354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide crossover distribution in <em>Arabidopsis thaliana</em> meiosis reveals sex-specific patterns along chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGiraut%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002354%26rft.epage%253De1002354%26rft.atitle%253DGenome-wide%2Bcrossover%2Bdistribution%2Bin%2BArabidopsis%2Bthaliana%2Bmeiosis%2Breveals%2Bsex-specific%2Bpatterns%2Balong%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002354%26rft_id%253Dinfo%253Apmid%252F22072983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072983&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.31" data-doi="10.1371/journal.pgen.1003521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring demographic history from a spectrum of shared haplotype lengths.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003521%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003521%26rft_id%253Dinfo%253Apmid%252F23754952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754952&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.32" data-doi="10.1111/mec.12155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mixing of vineyard and oak-tree ecotypes of <em>Saccharomyces cerevisiae</em> in North American vineyards.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2917</span>–<span class="cit-lpage">2930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2917%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> A genomic region containing RNF212 is associated with sexually-dimorphic recombination rate variation in wild Soay sheep (<em>Ovis aries</em>). bioRxiv DOI: <a href="http://dx.doi.org/.10.1101/024869">http://dx.doi.org/.10.1101/024869</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.34" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.35" data-doi="10.1038/ng0596-20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chromosome-size dependent control of meiotic recombination in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKaback%26rft.auinit1%253DD.%2BB.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D21%26rft.atitle%253DChromosome-size%2Bdependent%2Bcontrol%2Bof%2Bmeiotic%2Brecombination%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0596-20%26rft_id%253Dinfo%253Apmid%252F8673097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0596-20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8673097&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH43000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.36" data-doi="10.1126/science.1566070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahon</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Chromosome size-dependent control of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKaback%26rft.auinit1%253DD.%26rft.volume%253D256%26rft.issue%253D5054%26rft.spage%253D228%26rft.epage%253D232%26rft.atitle%253DChromosome%2Bsize-dependent%2Bcontrol%2Bof%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1566070%26rft_id%253Dinfo%253Apmid%252F1566070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTYvNTA1NC8yMjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.37" data-doi="10.1371/journal.pone.0013996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McQuillan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKeigue</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic runs of homozygosity record population history and consanguinity.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKirin%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De13996%26rft.epage%253De13996%26rft.atitle%253DGenomic%2Bruns%2Bof%2Bhomozygosity%2Brecord%2Bpopulation%2Bhistory%2Band%2Bconsanguinity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013996%26rft_id%253Dinfo%253Apmid%252F21085596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013996&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085596&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.38" data-doi="10.1534/g3.114.012294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative mapping between Coho Salmon (<em>Oncorhynchus kisutch</em>) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012294%26rft_id%253Dinfo%253Apmid%252F25053705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNzE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.39" data-doi="10.1038/ng1296-406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koehler</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Spontaneous X chromosome MI and MII nondisjunction events in <em>Drosophila melanogaster</em> oocytes have different recombinational histories.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKoehler%26rft.auinit1%253DK.%2BE.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D414%26rft.atitle%253DSpontaneous%2BX%2Bchromosome%2BMI%2Band%2BMII%2Bnondisjunction%2Bevents%2Bin%2BDrosophila%2Bmelanogaster%2Boocytes%2Bhave%2Bdifferent%2Brecombinational%2Bhistories.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1296-406%26rft_id%253Dinfo%253Apmid%252F8944020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1296-406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8944020&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VV73000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.40" data-doi="10.1016/j.aquaculture.2007.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Androgenesis, gynogenesis and the production of clones in fishes: A review.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D269%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2007.05.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2007.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000249143600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.41" data-doi="10.1159/000086896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Effect of meiotic recombination on the production of aneuploid gametes in humans.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DLamb%26rft.auinit1%253DN.%2BE.%26rft.volume%253D111%26rft.issue%253D3-4%26rft.spage%253D250%26rft.epage%253D255%26rft.atitle%253DEffect%2Bof%2Bmeiotic%2Brecombination%2Bon%2Bthe%2Bproduction%2Bof%2Baneuploid%2Bgametes%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000086896%26rft_id%253Dinfo%253Apmid%252F16192701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000086896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16192701&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232134400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.42" data-doi="10.1105/tpc.106.049106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lhuissier</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Offenberg</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittich</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vischer</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heyting</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The mismatch repair protein MLH1 marks a subset of strongly interfering crossovers in tomato.</span> <abbr class="cit-jnl-abbrev">The Plant Cell Online</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.volume%253D19%26rft.spage%253D862%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.106.049106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1105/tpc.106.049106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.43" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.44" data-doi="10.1093/jhered/esu060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesu060%26rft_id%253Dinfo%253Apmid%252F25149250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjEwOiIxMDUvUzEvNzQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.45" data-doi="10.1139/g00-016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habicht</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kretschmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene-centromere mapping of 312 loci in pink salmon by half-tetrad analysis.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLindner%26rft.auinit1%253DK.%2BR.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D538%26rft.epage%253D549%26rft.atitle%253DGene-centromere%2Bmapping%2Bof%2B312%2Bloci%2Bin%2Bpink%2Bsalmon%2Bby%2Bhalf-tetrad%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg00-016%26rft_id%253Dinfo%253Apmid%252F10902719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g00-016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10902719&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.46" data-doi="10.1038/hdy.2012.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Polyploids and hybrids in changing environments: Winners or losers in the struggle for adaptation.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.105%26rft_id%253Dinfo%253Apmid%252F23321773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23321773&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.47" data-doi="10.1371/journal.pone.0099123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mary</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barasc</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferchaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meslier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Meiotic recombination analyses of individual chromosomes in male domestic pigs (<em>Sus scrofa domestica</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e99123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De99123%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0099123%26rft_id%253Dinfo%253Apmid%252F24919066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0099123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24919066&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.48" data-doi="10.1111/j.1469-185X.1938.tb00516.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">Crossing‐over.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D13%26rft.spage%253D252%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.1938.tb00516.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.1938.tb00516.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic models to explain classical-linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> pii: esv056.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.50" data-doi="10.1139/f79-156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Joint segregation of biochemical loci in Salmonidae: Results from experiments with <em>Salvelinus</em> and review of the literature on other species.</span> <abbr class="cit-jnl-abbrev">J. Fish. Res. Board Can.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish.%2BRes.%2BBoard%2BCan.%26rft.volume%253D36%26rft.spage%253D1114%26rft_id%253Dinfo%253Adoi%252F10.1139%252Ff79-156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/f79-156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.51" data-doi="10.1111/j.1365-294X.2011.05305.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunelli</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BR.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D249%26rft.atitle%253DA%2Bconserved%2Bhaplotype%2Bcontrols%2Bparallel%2Badaptation%2Bin%2Bgeographically%2Bdistant%2Bsalmonid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05305.x%26rft_id%253Dinfo%253Apmid%252F21988725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05305.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988725&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298845600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.52" data-doi="10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">The enormous diversity in genome sizes of fish as a reflection of nature’s extensive experiments with gene duplication.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D99%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1577%252F1548-8659%25281970%252999%253C120%253ATEDIGS%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.53" data-doi="10.1038/nrg761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Resolving the paradox of sex and recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D252%26rft.epage%253D261%26rft.atitle%253DResolving%2Bthe%2Bparadox%2Bof%2Bsex%2Band%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg761%26rft_id%253Dinfo%253Apmid%252F11967550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11967550&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174739800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.54" data-doi="10.1111/j.1469-8137.2009.03142.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochmann</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary consequences of autopolyploidy.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DParisod%26rft.auinit1%253DC.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D17%26rft.atitle%253DEvolutionary%2Bconsequences%2Bof%2Bautopolyploidy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03142.x%26rft_id%253Dinfo%253Apmid%252F20070540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03142.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070540&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.55" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rab</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.11.2463.56" data-doi="10.1534/genetics.105.055269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeKoning</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keatley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assignment of rainbow trout linkage groups to specific chromosomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.055269%26rft_id%253Dinfo%253Apmid%252F16951085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE2NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.57" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.58" data-doi="10.1534/genetics.108.098095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inference of historical changes in migration rate from the lengths of migrant tracts.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098095%26rft_id%253Dinfo%253Apmid%252F19087958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgxLzIvNzExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.59" data-doi="10.1016/j.cub.2005.07.065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presgraves</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination enhances protein adaptation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPresgraves%26rft.auinit1%253DD.%2BC.%26rft.volume%253D15%26rft.issue%253D18%26rft.spage%253D1651%26rft.epage%253D1656%26rft.atitle%253DRecombination%2Benhances%2Bprotein%2Badaptation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.07.065%26rft_id%253Dinfo%253Apmid%252F16169487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.07.065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16169487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232235500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.60" data-doi="10.1038/ncomms2833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grassa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence are not affected by geography of speciation in sunflowers.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D1827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2833%26rft_id%253Dinfo%253Apmid%252F23652015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23652015&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.61" data-doi="10.1534/genetics.106.058933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058933%26rft_id%253Dinfo%253Apmid%252F17028345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzE3NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.62" data-doi="10.1111/mec.12322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination in the threespine stickleback genome—patterns and consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3014</span>–<span class="cit-lpage">3027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3014%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Gene-mapping of isozyme loci in chum salmon.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DSeeb%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D77%26rft.issue%253D6%26rft.spage%253D399%26rft.epage%253D402%26rft.atitle%253DGene%2Bmapping%2Bof%2Bisozyme%2Bloci%2Bin%2Bchum%2Bsalmon%26rft_id%253Dinfo%253Apmid%252F3559166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijc3LzYvMzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capilla</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farré</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution of recombination in eutherian mammals: insights into mechanisms that affect recombination rates and crossover interference.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">20131945</span>: <span class="cit-fpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft.volume%253D20131945%26rft.spage%253D280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.65" data-doi="10.1371/journal.pone.0049018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storer</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascal</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rank and order: Evaluating the performance of SNPs for individual assignment in a non-model organism.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DStorer%26rft.auinit1%253DC.%2BG.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49018%26rft.epage%253De49018%26rft.atitle%253DRank%2Band%2BOrder%253A%2BEvaluating%2Bthe%2BPerformance%2Bof%2BSNPs%2Bfor%2BIndividual%2BAssignment%2Bin%2Ba%2BNon-Model%2BOrganism.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049018%26rft_id%253Dinfo%253Apmid%252F23185290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185290&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1915</span> <span class="cit-article-title">The behavior of the chromosomes as studied through linkage.</span> <abbr class="cit-jnl-abbrev">Z. Indukt. Abstamm. Vererbungsl.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZ.%2BIndukt.%2BAbstamm.%2BVererbungsl.%26rft.volume%253D13%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Centromeres convert but don’t cross.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253D523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.68" data-doi="10.1139/g78-039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Sex-chromosomes in sockeye salmon—Y-Autosome fusion.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2Bjournal%2Bof%2Bgenetics%2Band%2Bcytology.%2B%2BJournal%2Bcanadien%2Bde%2Bg%25C3%25A9n%25C3%25A9tique%2Bet%2Bde%2Bcytologie%26rft.stitle%253DCan%2BJ%2BGenet%2BCytol%26rft.aulast%253DThorgaard%26rft.auinit1%253DG.%2BH.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D354%26rft.atitle%253DSex%2Bchromosomes%2Bin%2Bthe%2Bsockeye%2Bsalmon%253A%2Ba%2BY-autosome%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg78-039%26rft_id%253Dinfo%253Apmid%252F570442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g78-039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=570442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Gene-centromere mapping in rainbow trout: high interference over long map distances.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThorgaard%26rft.auinit1%253DG.%2BH.%26rft.volume%253D103%26rft.issue%253D4%26rft.spage%253D771%26rft.epage%253D783%26rft.atitle%253DGENE-CENTROMERE%2BMAPPING%2BIN%2BRAINBOW%2BTROUT%253A%2BHIGH%2BINTERFERENCE%2BOVER%2BLONG%2BMAP%2BDISTANCES%26rft_id%253Dinfo%253Apmid%252F17246124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTAzLzQvNzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.70" data-doi="10.1038/ncomms6770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5770%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6770%26rft_id%253Dinfo%253Apmid%252F25534655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534655&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.71" data-doi="10.1111/1755&amp;ndash;0998.12394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (<em>Oncorhynchus keta</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755–0998.12394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755%2526ndash%253B0998.12394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755&amp;ndash;0998.12394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2463.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes</span>, pp. <span class="cit-fpage">239</span>–<span class="cit-lpage">260</span> in <span class="cit-source">Isozymes: Current Topics in Biological and Medical Research</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Rattazzi</span>  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Scandalios</span>  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">G. S.</span></span>. <span class="cit-publ-name">Alan R. Liss</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.73" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2463.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">On genetic map functions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">1377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZhao%26rft.auinit1%253DH.%26rft.volume%253D142%26rft.issue%253D4%26rft.spage%253D1369%26rft.epage%253D1377%26rft.atitle%253DOn%2BGenetic%2BMap%2BFunctions%26rft_id%253Dinfo%253Apmid%252F8846913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Mi80LzEzNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2463.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2453" title="Whole-Genome Sequencing Suggests Schizophrenia Risk Mechanisms in Humans with 22q11.2 Deletion Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2475" title="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2463.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--SD1lJyq8BCJcCIDOoVyWhghWl3dXMnaalWJTaW3Iwk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2463" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2463" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2463" class="active" data-icon-position="" data-hide-link-title="0">Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460182" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XPmiCI8Gh7J1qiZub9j_QEhXTAxiWohwIzhRw5jfPEg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2463.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2463" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wWOQGmMu2O-FJhU2p6NrH2VEa50C7ws26mtQx5VUXNk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2463" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2463.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460182" id="node460182--2" data-pisa="ggg;5/11/2463" data-pisa-master="ggg;g3.115.020222" data-apath="/ggg/5/11/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morten T. Limborg</span>, <span class="highwire-citation-author" data-delta="1">Ryan K. Waples</span>, <span class="highwire-citation-author" data-delta="2">Fred W. Allendorf</span> and <span class="highwire-citation-author" data-delta="3">James E. Seeb</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020222 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460182--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morten T. Limborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute of Aquatic Resources, Technical University of Denmark, Vejlsøvej 39, Silkeborg, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BT.%2BLimborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Limborg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorten%2BT.%2BLimborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morten T. Limborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morten.limborg@gmail.com" class="" data-icon-position="" data-hide-link-title="0">morten.limborg@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan K. Waples </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BK.%2BWaples%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waples%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BK.%2BWaples%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred W. Allendorf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Montana, Missoula, Montana 59812</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BW.%2BAllendorf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allendorf%20FW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BW.%2BAllendorf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2463" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2463&amp;text=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2463&amp;t=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2463&amp;title=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460182/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460182" id="node460182--3" data-pisa="ggg;5/11/2463" data-pisa-master="ggg;g3.115.020222" data-apath="/ggg/5/11/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morten T. Limborg</span>, <span class="highwire-citation-author" data-delta="1">Ryan K. Waples</span>, <span class="highwire-citation-author" data-delta="2">Fred W. Allendorf</span> and <span class="highwire-citation-author" data-delta="3">James E. Seeb</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020222 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460182--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morten T. Limborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute of Aquatic Resources, Technical University of Denmark, Vejlsøvej 39, Silkeborg, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BT.%2BLimborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Limborg%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorten%2BT.%2BLimborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7718-6531" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morten T. Limborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morten.limborg@gmail.com" class="" data-icon-position="" data-hide-link-title="0">morten.limborg@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan K. Waples </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BK.%2BWaples%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waples%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BK.%2BWaples%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred W. Allendorf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biological Sciences, University of Montana, Missoula, Montana 59812</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BW.%2BAllendorf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allendorf%20FW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BW.%2BAllendorf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2463&amp;rft.epage=2473&amp;rft.atitle=Linkage%20Mapping%20Reveals%20Strong%20Chiasma%20Interference%20in%20Sockeye%20Salmon%3A%20Implications%20for%20Interpreting%20Genomic%20Data&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Limborg%2C+Morten+T.&amp;rft.au=Waples%2C+Ryan+K.&amp;rft.au=Allendorf%2C+Fred+W.&amp;rft.au=Seeb%2C+James+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460182/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460182/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460182/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460182/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460182/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460182/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460182/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460182/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460182/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460182/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460182/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460182/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2463" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2463.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2463.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2463.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2463.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
