<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2383" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2383.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2383.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2383.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/11/2383/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2383.atom" />
<meta name="HW.pisa" content="ggg;5/11/2383" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms" />
<meta name="DC.Identifier" content="10.1534/g3.115.021667" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Money et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Obtaining genome-wide genotype data from a set of individuals is the first step in many genomic studies, including genome-wide association and genomic selection. All genotyping methods suffer from some level of missing data, and genotype imputation can be used to fill in the missing data and improve the power of downstream analyses. Model organisms like human and cattle benefit from high-quality reference genomes and panels of reference genotypes that aid in imputation accuracy. In nonmodel organisms, however, genetic and physical maps often are either of poor quality or are completely absent, and there are no panels of reference genotypes available. There is therefore a need for imputation methods designed specifically for nonmodel organisms in which genomic resources are poorly developed and marker order is unreliable or unknown. Here we introduce LinkImpute, a software package based on a k -nearest neighbor genotype imputation method, LD-kNNi, which is designed for unordered markers. No physical or genetic maps are required, and it is designed to work on unphased genotype data from heterozygous species. It exploits the fact that markers useful for imputation often are not physically close to the missing genotype but rather distributed throughout the genome. Using genotyping-by-sequencing data from diverse and heterozygous accessions of apples, grapes, and maize, we compare LD-kNNi with several genotype imputation methods and show that LD-kNNi is fast, comparable in accuracy to the best-existing methods, and exhibits the least bias in allele frequency estimates." />
<meta name="DC.Contributor" content="Daniel Money" />
<meta name="DC.Contributor" content="Kyle Gardner" />
<meta name="DC.Contributor" content="Zoë Migicovsky" />
<meta name="DC.Contributor" content="Heidi Schwaninger" />
<meta name="DC.Contributor" content="Gan-Yuan Zhong" />
<meta name="DC.Contributor" content="Sean Myles" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Obtaining genome-wide genotype data from a set of individuals is the first step in many genomic studies, including genome-wide association and genomic selection. All genotyping methods suffer from some level of missing data, and genotype imputation can be used to fill in the missing data and improve the power of downstream analyses. Model organisms like human and cattle benefit from high-quality reference genomes and panels of reference genotypes that aid in imputation accuracy. In nonmodel organisms, however, genetic and physical maps often are either of poor quality or are completely absent, and there are no panels of reference genotypes available. There is therefore a need for imputation methods designed specifically for nonmodel organisms in which genomic resources are poorly developed and marker order is unreliable or unknown. Here we introduce LinkImpute, a software package based on a &lt;i&gt;k&lt;/i&gt;-nearest neighbor genotype imputation method, LD-kNNi, which is designed for unordered markers. No physical or genetic maps are required, and it is designed to work on unphased genotype data from heterozygous species. It exploits the fact that markers useful for imputation often are not physically close to the missing genotype but rather distributed throughout the genome. Using genotyping-by-sequencing data from diverse and heterozygous accessions of apples, grapes, and maize, we compare LD-kNNi with several genotype imputation methods and show that LD-kNNi is fast, comparable in accuracy to the best-existing methods, and exhibits the least bias in allele frequency estimates.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2383" />
<meta name="citation_id" content="5/11/2383" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2383" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2383.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2383.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2383.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021667" />
<meta name="citation_pmid" content="26377960" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2383" />
<meta name="citation_lastpage" content="2390" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel Money" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada" />
<meta name="citation_author_email" content="daniel.money@dal.ca" />
<meta name="citation_author" content="Kyle Gardner" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada" />
<meta name="citation_author" content="Zoë Migicovsky" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada" />
<meta name="citation_author" content="Heidi Schwaninger" />
<meta name="citation_author_institution" content="USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456" />
<meta name="citation_author" content="Gan-Yuan Zhong" />
<meta name="citation_author_institution" content="USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456" />
<meta name="citation_author" content="Sean Myles" />
<meta name="citation_author_institution" content="Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada" />
<meta name="citation_reference" content="citation_title=Genet. Genomics Breed. Grapes;citation_title=Genome sequence initiatives.;citation_author=A.-F. Adam-Blondon;citation_author=O. Jaillon;citation_author=S. Vezzulli;citation_author=A. Zharkikh;citation_author=M. Troggio;citation_pages=211-234;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Altshuler;citation_author=M. J. Daly;citation_author=E. S. Lander;citation_title=Genetic Mapping in Human Disease;citation_pages=881-888;citation_volume=322;citation_year=2008;citation_issue=5903;citation_pmid=18988837;citation_doi=10.1126/science.1156409" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Antanaviciute;citation_author=F. Fernández-Fernández;citation_author=J. Jansen;citation_author=E. Banchi;citation_author=K. M. Evans;citation_title=Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array.;citation_pages=203-203;citation_volume=13;citation_year=2012;citation_pmid=22631220;citation_doi=10.1186/1471-2164-13-203" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Gibbs. R. A.;citation_author=J. F. Taylor;citation_author=C. P. Van Tassell;citation_author=W. Barendse;citation_author=K. A. Eversole;citation_title=Genome-Wide Survey of SNP Variation Uncovers the Genetic Structure of Cattle Breeds;citation_pages=528-532;citation_volume=324;citation_year=2009;citation_issue=5926;citation_pmid=19390050;citation_doi=10.1126/science.1167936" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Chagné;citation_author=R. N. Crowhurst;citation_author=M. Troggio;citation_author=M. W. Davey;citation_author=B. Gilmore;citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.;citation_pages=e31745-e31745;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363718;citation_doi=10.1371/journal.pone.0031745" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=K. M. Gardner;citation_author=P. Brown;citation_author=T. F. Cooke;citation_author=S. Cann;citation_author=F. Costa;citation_title=Fast and cost-effective genetic mapping in apple using next-generation sequencing;citation_volume=4;citation_year=2014;citation_pmid=25031181;citation_doi=10.1534/g3.114.011023" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Han;citation_author=J. S. Sinsheimer;citation_author=J. Novembre;citation_title=Characterizing bias in population genetic inferences from low-coverage sequencing data.;citation_volume=31;citation_year=2014;citation_pmid=24288159;citation_doi=10.1093/molbev/mst229" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Altshuler. D. M.;citation_author=R. A. Gibbs;citation_author=L. Peltonen;citation_author=E. Dermitzakis;citation_author=S. F. Schaffner;citation_title=Integrating common and rare genetic variation in diverse human populations.;citation_pages=52-58;citation_volume=467;citation_year=2010;citation_issue=7311;citation_pmid=20811451;citation_doi=10.1038/nature09298" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=O. Jaillon;citation_author=J.-M. Aury;citation_author=B. Noel;citation_author=A. Policriti;citation_author=C. Clepet;citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla.;citation_pages=463-467;citation_volume=449;citation_year=2007;citation_issue=7161;citation_pmid=17721507;citation_doi=10.1038/nature06148" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=Y. Li;citation_author=C. Willer;citation_author=S. Sanna;citation_author=G. Abecasis;citation_title=Genotype imputation.;citation_pages=387-406;citation_volume=10;citation_year=2009;citation_pmid=19715440;citation_doi=10.1146/annurev.genom.9.081307.164242" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=K. A. McClure;citation_author=J. Sawler;citation_author=K. M. Gardner;citation_author=D. Money;citation_author=S. Myles;citation_title=Genomics: a potential panacea for the perennial problem.;citation_volume=101;citation_year=2014;citation_pmid=25326620;citation_doi=10.3732/ajb.1400143" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Myles;citation_author=J.-M. Chia;citation_author=B. Hurwitz;citation_author=C. Simon;citation_author=G. Y. Zhong;citation_title=Rapid genomic characterization of the genus vitis.;citation_pages=e8219-e8219;citation_volume=5;citation_year=2010;citation_issue=1;citation_pmid=20084295;citation_doi=10.1371/journal.pone.0008219" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. E. Rutkoski;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Imputation of unordered markers and the impact on genomic selection accuracy.;citation_pages=427-439;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Journal of toxicology and environmental health. Part A;citation_journal_abbrev=J Toxicol Environ Health A;citation_author=H. Schwender;citation_title=Imputing missing genotypes with weighted k nearest neighbors.;citation_pages=438-446;citation_volume=75;citation_year=2012;citation_issue=8-10;citation_pmid=22686303;citation_doi=10.1080/15287394.2012.674910" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. C. A. Spencer;citation_author=Z. Su;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=Designing genome-wide association studies: sample size, power, imputation, and the choice of genotyping chip.;citation_pages=e1000477-e1000477;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19492015;citation_doi=10.1371/journal.pgen.1000477" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. J. Stekhoven;citation_author=P. Bühlmann;citation_title=MissForest—non-parametric missing value imputation for mixed-type data.;citation_volume=28;citation_year=2012;citation_pmid=22039212;citation_doi=10.1093/bioinformatics/btr597" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Stranger;citation_author=E. A. Stahl;citation_author=T. Raj;citation_title=Progress and promise of genome-wide association studies for human complex trait.;citation_volume=187;citation_year=2011;citation_pmid=21115973;citation_doi=10.1534/genetics.110.120907" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=O. Troyanskaya;citation_author=M. Cantor;citation_author=G. Sherlock;citation_author=P. Brown;citation_author=T. Hastie;citation_title=Missing value estimation methods for DNA microarrays;citation_pages=520-525;citation_volume=17;citation_year=2001;citation_issue=6;citation_pmid=11395428;citation_doi=10.1093/bioinformatics/17.6.520" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. Velasco;citation_author=A. Zharkikh;citation_author=J. Affourtit;citation_author=A. Dhingra;citation_author=A. Cestaro;citation_title=The genome of the domesticated apple (Malus Ã domestica Borkh.).;citation_pages=833-839;citation_volume=42;citation_year=2010;citation_issue=10;citation_pmid=20802477;citation_doi=10.1038/ng.654" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. A. Ward;citation_author=J. Bhangoo;citation_author=F. Fernández-Fernández;citation_author=P. Moore;citation_author=J. D. Swanson;citation_title=Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation.;citation_pages=2;citation_volume=14;citation_year=2013;citation_pmid=23324311;citation_doi=10.1186/1471-2164-14-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C.-W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Obtaining genome-wide genotype data from a set of individuals is the first step in many genomic studies, including genome-wide association and genomic selection. All genotyping methods suffer from some level of missing data, and genotype imputation can be used to fill in the missing data and improve the power of downstream analyses. Model organisms like human and cattle benefit from high-quality reference genomes and panels of reference genotypes that aid in imputation accuracy. In nonmodel organisms, however, genetic and physical maps often are either of poor quality or are completely absent, and there are no panels of reference genotypes available. There is therefore a need for imputation methods designed specifically for nonmodel organisms in which genomic resources are poorly developed and marker order is unreliable or unknown. Here we introduce LinkImpute, a software package based on a k -nearest neighbor genotype imputation method, LD-kNNi, which is designed for unordered markers. No physical or genetic maps are required, and it is designed to work on unphased genotype data from heterozygous species. It exploits the fact that markers useful for imputation often are not physically close to the missing genotype but rather distributed throughout the genome. Using genotyping-by-sequencing data from diverse and heterozygous accessions of apples, grapes, and maize, we compare LD-kNNi with several genotype imputation methods and show that LD-kNNi is fast, comparable in accuracy to the best-existing methods, and exhibits the least bias in allele frequency estimates." />
<meta name="og-title" property="og:title" content="LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2383" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Obtaining genome-wide genotype data from a set of individuals is the first step in many genomic studies, including genome-wide association and genomic selection. All genotyping methods suffer from some level of missing data, and genotype imputation can be used to fill in the missing data and improve the power of downstream analyses. Model organisms like human and cattle benefit from high-quality reference genomes and panels of reference genotypes that aid in imputation accuracy. In nonmodel organisms, however, genetic and physical maps often are either of poor quality or are completely absent, and there are no panels of reference genotypes available. There is therefore a need for imputation methods designed specifically for nonmodel organisms in which genomic resources are poorly developed and marker order is unreliable or unknown. Here we introduce LinkImpute, a software package based on a k -nearest neighbor genotype imputation method, LD-kNNi, which is designed for unordered markers. No physical or genetic maps are required, and it is designed to work on unphased genotype data from heterozygous species. It exploits the fact that markers useful for imputation often are not physically close to the missing genotype but rather distributed throughout the genome. Using genotyping-by-sequencing data from diverse and heterozygous accessions of apples, grapes, and maize, we compare LD-kNNi with several genotype imputation methods and show that LD-kNNi is fast, comparable in accuracy to the best-existing methods, and exhibits the least bias in allele frequency estimates." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ifmTZaR02X5WDdrSNxtJFjaELWZj235yw4tlzDnTxnY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460179","apath":"\/ggg\/5\/11\/2383.atom","pisa":"ggg;5\/11\/2383","ac":{"\/ggg\/5\/11\/2383.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2383.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2383"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2383"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2383":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vPCf3dtt2g1PYMDyw0OyjeKdy8z7oZ\\\\\\\/fWsp3BQQUyMJX5a9qMLm52cli4jQLr\\\\\\\/B4qgeXPoqckyYKgAXSknSQPK7YCrrBg1rS80QCyeH6RMlx9Ik8RsF131\\\\\\\/tjfnYajZkuE9dIDcc1TCekEFhfi1sPdvHvLnvaJpSvitNHxh4\\\\\\\/pdXmglA9DoH3BOk9q6prHR6NTliGTkJxtMmPAorsvevrspMWoOxR5beox1nT4BrDUnNZTgcUkZyim766ogw5caREYG4rS++Sa4tFP2tHyQzI\\\\\\\/1SClc0NO1rRoENd07W+yf3QsQ8FCWLpkc1eZRXsGFasJqDcrbjQWXO4VQvcXnFk\\\\\\\/Le2qZwdnfyXNML9AzESY9ZGgKLi7GD53U5QMdoKwDu\\\\\\\/70Ld0cdoB6q6moaOfomWaxchiRRGsVpgp0BIB5SPfRWKfND7MPNTmOy2CnFIesPCf\\\\\\\/\\\\\\\/mIFvCY\\\\\\\/CsCR4zGPXtMrBNMGZnl85ghihZ9Nq6EQ4q10oOBG5Eqo4EfbbYOT+5giKWy5zvyUG4ZoeQqs0b+V4CI0vjKcbZdzPEJUd4uwUl7fr+yBcjdZC2O0u10Aka7QzIdmJfW51FkUU\\\\\\\/9lEm0SLc6cqIQfkpSvdGurQ3l48FuuD2P15sFhRPVAKLWlylLnHM02cGZ7XK9UdF7Cu0iWlrEc1RhybZ4kLOqmn132JlFvQ0U0PH0BdNrJclw9SXcpYCLxsMln6XAaJvnOHLrCEOF6b+dvRqr0PuvuIU5bfqLnvUm6COt+6dV+vTQwulF2SriZ91STVtRhyV680UkFHurQ9zwALHBklQ5Qw+i2gXIVr8xCdGwrDOgMlBamt9tVs5nNn+uHYoJIkQ6J84ZPvMzS2rzkmJNCETkGkAQJ0k+GjOeE7CGkladjWCg2UzDJ0NtWQp1\\\\\\\/+SuEldbcYRJAAgJk7zOLadM2ZaVuoy1LXEVWhe6arBKj+lwhP7bYm93vCutKEzjQlV2adNsPpCqjfkr13uweBgesMaS1B0a\\\\\\\/CXg0+W9JqeZ9k0vcoPuoT\\\\\\\/sbX6rA0AZBKhx7ue61u4FYnN0ORFNoDgFSam4eQpDQAq4UT2maC0CL3nqrJU4ew7BgnZaQ5ghz8nQXREOzfo7pzVyAcw4z8FrWV6zvvGU+EXtyV+tiqDPLFnb+PGaiiEuoKjvwkwBX+WYqXykjfGq+TEedwhyOI554A5PhPdaex++AblirukxRiIHOPH991tp29YmB3BChmXkKf8NEtntLavfvxhMD9VLlCPsBrhFyjkBmZ9QguuBg99UyCu+tZ\\\u0022,\\\u0022iv\\\u0022:\\\u00222TP8i0abJuXqR\\\\\\\/zVb69sOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220155c7cb132c881883f064b11068460258714901af642a4af8b1084d4cb922c1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g2JoCY1lpEDqTspjmtUVHRkWXw9NuZaAQ7iGqcAWbEoa7eA4XtSCOD\\\\\\\/oEXt0FRe\\\\\\\/t5nUEYaSzllB6gypTtvCrThRMd0r3jndrpwd6OeJD18NYfM5OPtERgvN34D+ygHzNUbJy+bIhxAjX++kk9nKwhI6yRcnO34n6uGYEDJaJxRFkWiLuHVVCwQ\\\\\\\/lSAkSaXCU49pJIUU2pli3fFdtVmacXNSuoSLZRTvnJb9qLfIpqYTwznEseCxISHd7+ohJsOW+T8ZQd97z8zONndP2TOoGx0Mikypr+RnYZOuxfvX8PepjiHGFQ\\\\\\\/F9xb4D5cUCuKxx6BaC9y6uPKANzFJ3RFhM5JMwpxICAM+0LCPrMG9ayaNl9jtzNYqBLqOp+cgnvRlZiXCGYc07bZEEMUD5UlJ0YKPbvhS4FFPwEfimXDh5Nv0nNDsvu4Mq0q1nXhWmYEqsa+AcinIomZnDQTvYknT+J5Em+ddvApl8knOswUDcoENdSWQynk\\\\\\\/o+PjDbEjra00KfpLnrRgddlPvdmAqdsXuUPcVwXh+rlSnRkkNSJfqaAt+PClSoD6inNUD0Ba8myfn5zz\\\\\\\/JmCMTbqZH\\\\\\\/bXzZh6XyIFZesSrLoBfPjcTLjoDUw6UoAwHKVtlCQdtIh5BdxtAwIwt0M\\\\\\\/SHtg+vB4fGoeQ1pO025ouTXunR\\\\\\\/2kKNEn59jcqpJcDxGEb5qVZi+0iuzFDYGHvlrwjh5fhiRYbPDpzXhKnakDqYDUiX3ZMbkqazIEAPjd1aDjPlpg5IN+nuX540dAKq3y4c+q5t+LoqGMCtky6ApGI7M+dC9GgddFojLnbI2R4cyvFC\\\\\\\/hZwhRbZGxvc0zXUJEgh\\\\\\\/+a8P6W9hvGxqL1jRYTp8JrwQwWeAhF5EHIsztohFGNuEcQUTuXBy2klw5AwEnsWrT7BkpR+hpfSzDD2FC\\\\\\\/dgp83S8MvB0gpHefsIlTDP+FsKIxmpZ0GFUpCiUPGyOAW9IgpuNNfUnwv5BA+hQoKjruy232Mm4GYfrPcgLBXaTPIz9yY5RKlSnSeY+yNhAZ3eQY8Xf71KUBALOeL48jiEqSgwEwZK3e89nVYRPwM\\\\\\\/SDUENNRCAxyJ0cDdU\\\\\\\/8vU7r4CFPiEydbeCA4sqgGFCdsR5ShA07k0bJuZNmDMPt220oECxh5\\\\\\\/t8pKyzcc10kN87obTGz4dPxefLF\\\\\\\/\\\\\\\/i3yrtLj+di4+Ik1wQKInxgq07meJoINGztUXS\\\\\\\/7sLcKg2LJQ7AFTk7rfyMFjJ6AdOo8Xj08AvtZYnSZpj9JDyygrgP54pAWpyj6vvPqhJN+6Np3mOOoBxg644c5xobKVwc6B2dkqCyIIR6JpT0Qb5\\\\\\\/bVHU8BESepWY5gpf\\\\\\\/IoYD8GLv\\\\\\\/fviWrIGX1pi1v4zQngBpJ2R6Rc06nFP\\\\\\\/EoAOK2zI51mrJcujZM3dCOF75qoZmFtnXkCwB1JPeC2rS0ICc1ElFHDOA2l4nYg1D\\\\\\\/MiYeR+0l3JDL91Sfvk\\\\\\\/MYOIQb3j7Oy+kXZRDL+xOoSHrIAlZiyqpUwrt6w2L2R93Q9u546bPZ+2s\\\\\\\/8MQ+jiraUrNnAUCpGO\\\\\\\/wTAPsIXsOUrDzXv4\\\\\\\/9BTHikYlarCLqE0zDwFbV07qIBpgVcXmXNp4bIoOoe4OU47lU1I1A4dF15nr4H24Q9wGZECI5SNcVixXJ76xeJ7hpOoCR8BVa6igihf2sQQR\\\\\\\/tns91gCzXKcyEyULp4OwTTFJiK+BPgbzNaSi6xwFzjYSoyz0gq49vyrKZLQ2lyLBMgpvoSDdFHjv1N\\\\\\\/S0B+cU3+AgVHMeq+05EH7W4LElMzRsqiU1JLYDXg7TE98gIBPdKBgKAMjg9ycvRSJDBUY8A\\\\\\\/\\\\\\\/NBSziRL6N03ywj93YJ3n1JRQT9yqj9nLfJeqEQEnDpRD7\\\\\\\/fkfDyNe+SCznYrwrgAXs+tEtze2WDj3DOz6q0gqq5XgAcUTwON3ZZrrsLirK3bETNu7j+qynnbc9rK6JyT7onu1BCtOUJqoL8XyIWgpK2hwnMFgxvaRtfXoEe7E\\\\\\\/z9RfL8SxihAAjs+vpzHykid1LPl8ffFQOak\\\\\\\/LgZrh4hZ1qpuShKN032IExSZ8JLOg4Hp\\\\\\\/kRzFk60AaWfNsvsF\\\\\\\/QVGdb\\\\\\\/\\\\\\\/PTegy1cbDGa2FJhFE9Ivip6\\\\\\\/+lgT25qV5OTgAO72C8jaRf4X38e3g6jtsXtfqMwPTNYhbQ5pO6dOf47\\\\\\\/fWqMwOtsnyQ9uIZCFsFI0DaKnfCzVLj54SDRX84Ig1olX29\\\\\\\/IRcQ252pkp66WQbLQUdA3xHz8I0pl5Xps0drFTDXlavLXX5OQQYaHRJuvpINrHYfGfKY1S55l+putkTNs4IUbjeQpIj+RzB8WG5bOzu0VVJAHThJUPPI5FcAWyx\\\\\\\/q8cI8OrBi7WrJ643UOk4nIHPTRqHNMhYR8VRULPfLrLiSdSfnciDMnc47pi9RUg8vZChZuESvMSSM8ldwMtCfpOTHPYCVvqol1C8o5cFa05Ko+6WPPGUnYyLsreMX09tkCSAf3WYCHHjJtrp25AD1jywISKHoUxlZfAoQAoUckaNlXm0O0ynthw0ewwIP3bIEzt7i0cXidReJ+jOl6W5EaeGKrLj4T1qnyXCACEZgRQzrcVj+AUE2Peqv09Vg+RL5fHScHddCw+pFvbwXkO0iHcVDXVGYZtvZlS1Iy\\\\\\\/NaFN0zFizrVuw4d8oXhsZrKtFOszqY\\\\\\\/ONexiAPs8OLrDeS1hzAluZiwGrkxX+vkyOO+nlzZhlyI0JE+xknUJrh8bgan1QX3rakUoPQjOCjK4FUM915XMG541XCQcORIGOIpZj1sfH3iJVGuMTZi\\\\\\\/9zVOnGsuIp8L8x1ATMcAoe8Cx9UdzQmophu3SLCa1oQYhHb4jsQyt5xrvoabB2Hkq8qfqDIGw5FSmQpxMZ6t7ODhOWOrjTJCHI2TcU6oMdM5r2TSr8iz1\\\\\\\/jYLGsIYkeEpiZGed9IAsYkTOLSECD4lYLCIfkDr7dba142y2FyA5pswrrCnS5ayDpL9LjM+K9gnM43i4xWGAVjpWyTcj1Lnl00CcJOp3jBbXR\\\\\\\/10pGyDmWFqa8IvjonrqZNCbuHyKJHyUBsGpEhHq67diunevIqJfBE9Ifr4wplZpm1chRmTwEZQ3Hc1AkDjH0dvuQ3fUKsNAgtEnsWIrVK+2pIJabgLZRawQo0BFc92rcb6OXBttTCQGKD6MeLHgepGWuHrQVlpScgRn5i4g3zrACf2rddDYJblAIsB34cdkqh9CECVhiMlo3LsS9Q\\\\\\\/ZjCWGLMI94QoZCIRMrKv+7MW9oC6osrXXB75AsVEqUtNLH31WO1MiNsaOE+lIhXRqQ2uAV9GKYU5VUi5qgoVoFTX0jDfiuKVn8REOxquM0aR5VtqYVfYAdRH1NL1MXAFoocunPz7q7d5Vo4MKqQ4v2xOOscnKA7VaL0F+NsSsl+w0XosVUvGc7dQ7N\\\\\\\/XXmXsKo2+2mUdS\\\\\\\/RwnAG6lAL2pVTSTHylAWbRn2aPdUD9HN48WJyy8Yrd\\\\\\\/ABKSHbEyNkvoZObGo+PM5n+mblMARTjxOj5vQ1UjOS5lSK22wXGEL8MeiWeOUAtSU7ciPuGmhqJB1QqmhOOlN\\\\\\\/w4P7pQZTLLe7E2TB727eOOPXn\\\\\\\/iaYmq7G3VIOJRxH3zM3WYTVetqnBPrEbKpzOYKfVSNhRQHuUWwnql69MQGPHjOR+5L6XFs+Lx+K8wemyV4DXrJw\\\\\\\/ixfbQpr2HnALreBV1H6ydZVhDXtTgddCCbIi\\\\\\\/wp6NG0CYyHp1OYaVH4l1sqgNMAh92VauHm\\\\\\\/67Cw6bc9QLR+vQ1p6v9vEnTfL\\\\\\\/12F308lfS+7Wdsj1ilCqK2KGoT1UDttCagF2kXZucF+aIgTCU2iv6py+DVjSA1Ti7bnOIBdVlCkXjq9iy9ATUEzjszWH5od++BEP0yvEIQ7rZtE6ztmKVe4SNKizl3omqzrbOMWrC3JqNfIflxQmoVW1pLq49hQaLcP6z4SLtEiWcNtZ3VnymHReXsxPfJAHX49Tcm+wTz9piUqPWxRluGl\\\\\\\/NFHP7G9AtUVMptvO04B86\\\\\\\/uqknpf0LRKDmM1lLZe3\\\\\\\/3hxh5omjkVDf986vWFNSZT2hNTn3PddjnIFK7BrfYB8+kbPDv7RReF9pB\\\\\\\/SDRDjVeHa8lU7anve5CST58ujpezW2Xc7vsufL0KDUrcIc+m2WLwgPh\\\\\\\/5Tj8GSFA+0gtR0TaDTbeieq\\\\\\\/DxUUtrFIi4YvL1QOjHnczDWHg4qc8XaysU4h2UR0BqC\\\\\\\/6+pRQSPLNurojc1dEm8zXykL5B52mbbnSCmPk+EJZBPBQG2EspM+zdQDM+ljoS\\\\\\\/KbBWLPamHtDrfBXgv0aFTGc1ojEL5KZTYza4zMGhCyi0c6sesQi3buEI6x5q+skE+cipfiqmCSVxZ0kyR0F6wR7IVvMQ623F8A654JRt7YG5JGeZcSNVX5e2+0soWChVzjTwxm+MhpfU\\\\\\\/O7FJ6AplEr1mVNH12vDN9zboXXe5+3Dxy34uY7t\\\\\\\/oHsnZrewCDBTOulqorAW53ERqfWvPG1FPRDI9Ez+s1fNNBVI8tchoIeunQLeAeyP4VzZiXXjcD\\\\\\\/HJlgM9ibZvMuKAUciBYhP+y6Kj8LVVNSdn\\\\\\\/Z34SSMr4\\\\\\\/XegDlZnXgZQAGars6QAkIdmzO+IAkM4gVsvrnPxVc4dp1Y2Lo8GHqbOS1TyN6yz1\\\\\\\/91k8Q5hUP4+CVu7Z7dzdCoJfy4eNb\\\\\\\/RM6hITgyfwnqKEG\\\\\\\/68kSp4SPlqW1ketDtsiYymc3bVYJLyvSbLK4uSA6jpYxC0yUBTIHqoMzFgWSFk1C4Thf06041\\\\\\\/y68o2hFQYogBSXDPfy2YgUSi0qgrGC+pJOsfutGvgglCPkP9WyIeLTgW86\\\\\\\/\\\\\\\/KwDIT3DYVV4d604aHuWSmH4TFZ+koyeXlg3Llb8ZHk5zS0c5WcsYH8Lq36sTeL+ra4GgTHBOoTd05gO1PxEQ6m6fhmexdzrLmb6fhukwu\\\\\\\/KegTDhhGELMkunHePIYNt9L6wz8O388My4TZlEzYteVEfHpVsEkSVKRvkI3PdysRym\\\\\\\/6k\\\\\\\/NGdfWr2aYn2SUeWFb1MwvcwLAPDhh\\\\\\\/QZtUO5OKc6SQZfxpoUnqx++HvaHwnm0VZc9DnVdV5NKFDgWRcSudOtgkgVP+hpk2oJL6Q900lXdlTbai4cNa1sn1y4C5MOmEtGNkscbkjwGDxO\\\\\\\/w0H3mD\\\\\\\/twIwvN2D3KaYcftGBKialQPXbj\\\\\\\/nXdDMPttwAylKka8zu6rSP91nFfRvsEKkQaYSHwVS8wlcKPTySu8s7k50x9P6Nhxq0FTzA+xF4sM\\\\\\\/oKrxWgf88DUrWF\\\\\\\/fbh6uEOngzAq26Rk5j8YP+G4yu+oAsT2P\\\\\\\/l8WQkETerjGNSZOwG1xSntLE3q4I3nGwSl7C9EX0fwfyjJp8K4HbkhVw7ZfF4STpnuv+9GsVw8Jt9P8hj9YKwgoMnvEwVZjTxNljAGav51FUpp09KeukMMSy33GVZRsBdIeT1Kf2V7tqvwke9pCqOhxNnz\\\\\\\/ZK+aeW0960o\\\\\\\/yWtreUR+p291pUkIruC1g0oBse8vd0MvrYAzz+GQYDF58uOlowPA69Y6kOZU0EIxHulmPYkGvFF+21O\\\\\\\/uetxHJX9YKlgkm14jAkrpOIAkPdEXSjL9doEPrYV+W68FiRULfphgVacBWePSMmKQwyHF8fc+PbsCZJYA1rL6bVYvKP2gY4BSXqEsMSrpLa5z\\\\\\\/viURMjhaYpQi67ijxX8z3du38WlADwcSMYxdhYrUx+46KqxL3Cx9jsqT5Qf9NBXK6ZTOZh8FAecuq4k1dEFxT4cbrmOHoJzk6dAx4IdYo8vHsn1giWhV6cOE8KjbGySXYpSU+foxVPIQDfnQ6VH97MQlTo\\\\\\\/SaMV8W7XWkRz8qvmfcKY\\\\\\\/vR8ItJ7DdQLQYacqyPwBol18GQSZwmgBGUyXWn+e5i+Kz0KIhoVT9WR0\\\\\\\/wKGWan1gQU\\\\\\\/uLomkG\\\\\\\/GUokKBoo111AiupcNs4H1TWiI8JIfFD7xxrBML\\\\\\\/HnMO\\\\\\\/yzlFZBmK\\\\\\\/h9pvFGPcqyovQqRYOad3Z1AfsxmCAVqMa4mnhSkmp\\\\\\\/MLNbI7PBddEWmk2vGMr\\\\\\\/Zw1RXIwML2KmsllwwbC6rjSyXsMFOBa1J6iKMqSMjdAtswLz8UIVOaVUB7lo5hJKLkTf2ZYIIDtsP6SbA9iXCr1+WOqI7WAow\\\\\\\/KT1a5p8+ddkgazVu9oQHskzYPJBdsiPXM54bmRhCtErKmF2GMh75wtnd1KrXiJlxMYJQx3Vb1wCWZZAzwc8Trm7XbaNIPaYQHCEZuJqQynnFtelFgdaeUVLSvawg91XbHyoCSypxpc1\\\\\\\/9L8QMaIRr2nO3arJIzDwVDTBuTzO2VhJ6Ww10OeJzVUDHkutP3Vw14o6NPzM7+1DWmiW8i+stY6g5+tR4U2k3SKh6+jWPrZxLOZGPwJT2zXnkuI7JXVI3SPWitRkx8SoaqVnyc3iMwQL85i9\\\\\\\/T9h8aQs4H3OJQDpiaXRRT1TnCTaMhQ0G8ZI5jtiJJ1+sLHDEdJ+BjwWYmn\\\\\\\/O5AFPM5C+aCbRko5h\\\\\\\/A8yk3Fs3PwxPLydnHBgiNH+f8VORg++Kp3GokCAq\\\\\\\/DnYBeFtoFObnmew++SBoFzjR4a4qNKSsGULVQb6ryjzCckAciwlLCqrTtynt8m\\\\\\\/pJhPWMzpDPcqNeKg6lJUrz\\\\\\\/FkKJ6IxqKlcMrqFIVxPxmSBfOmddca1s1CvWtmhkeCBYcZKWImc8RoVX01YKS8l2VyjGKFFZHDBzRdTVqbdCoysKTbS1JassrxmTAOvjC6acBJav8yn+kz5wA06HGDjXVc40L79+60uMAa9e8D3b6MPfoi7Xt7rAEKKVIPCqvQ7aL4JshCQzPYtvzW603CIy8Y5lqFkgRgao8+5w3YEq9H+AzlKN3TrMrwDy7VT7ABr6vT5bxe91MdVTRy0ebOeLLUT5NqH7BhNcUW2V8nBua2Esrack6WsypY3Yr6S8klv80j5h2bZZ2T+9YQU47aYVnFEZBYXVAZt6KUk+VsiEQfpnZQR4hKoYcgBu6wEDEHI60nz11f6H0y9kxcVXqyblzkv0zvY7+tSssv59+\\\\\\\/qjnnu4lAp1PABM7b76ecK20q3h22tKfi\\\\\\\/Ps4uHS\\\\\\\/MZbuOiR1cAK8u4aBNKTfAz8kjpVXDNMCJF3ppShl3voYfNfZplT9DY8AJSZ+mT9HPGjpBCkLkdOnnH0S\\\\\\\/7gQ5AVaqIsGyap+I8xbW43KoIrJ7\\\\\\\/HDtGRz1s2B6c6GbPbuhWXcdOvVsWOhizDewT6HdRyH+WaVSWdgd6DWCqB4aFbxXxLkMulHgGjVtUfEPoutojk3oEkCrFmoIonoKxKZ74tgD4dgprcEsjBL1G2z80e0WvZiIqtlDQ2UXgqiWMinxF\\\\\\\/nrGkO8YtpGcAANi4+jLL\\\\\\\/j7ZNHn69DAvfRA33VJ5IZTSzJ2Q22MoqFSE1Zx060FBcBl7EdgvvFVWdDfeAGieI1QyE+TmLjVAYEYTrKnLC40ybHhxcKrhuBouEfhXOa6UUhJ3awI3He7UYRwhxHzsmJa9ZmBv6lBhWxUUZwgbbNP5N0EuFHf+\\\\\\\/pdj9roWqEeqSAnyJOBvB346luo6BIJ7dqPpcegb3x9BMbAZ2RIt73\\\\\\\/GyayHlxLsWAw6+NVgf0Dn6SmnhVTnPRp12DN9HB8rRAatIFFU0y3TZNGyaEMwwerpLc3zwiB8wgsPAsUIa8fIH\\\\\\\/BpkDUUi9WkkqAPjNOP7ppSwEYbuztk0tj8LaeF8WBV3hK4+OhHhzY2nR6q0vAa+MCyDjPSN+Tnr054vilNwn8NqKjbr\\\\\\\/CdqyLoM4x7AbUFPDOJZQDhzDdUouEMUTB7A+nm66CeG6Ag9RvTNukh3Xr9zyjOA7rwN99R0F3RWsBeEoHh5L5qnsuv5HQbEbrHf1Q+V8ynADm\\\\\\\/bPTcAEH\\\\\\\/u63Hum2Rx+at\\\\\\\/8ftRCHu\\\\\\\/cTKG90ufq6TTyFA2XegAK1rknx+8AURbbS67DIuCLovwr\\\\\\\/UPvNgrWkgJ+njEDQva05JphgRJ8xDVxjAn3bjg8IbiLmMJhAoex5zJZSZZ7nTKsLkfySQGzTMhO1HqxVaO2ivdqm6UqCe11nq7N\\\\\\\/UwupAjO1dTKlo1J7QCwbnRcS7cb2ms1Pvgkqu9oNdr8Ebwf3ZjsrOE1+Pvg93GQKDgjj1aMMPxia8ogc8TGk70zKf9dMJOF041tXDBksjt9loZj+KFT0tGc5f4I4QH8OTSpgxANAfB4nzClaiklPyhOgXzlOY3MOLrw\\\\\\\/rX+0L3ev9R5QF6s3J8UZQCaXgG8rWuPQkl7X0I0P\\\\\\\/bi1P6TiossVnUXaeHIAToFfKHcatFng97gniPf0qWnvsdifT1qw+p3TNbLLe\\\\\\\/r2FCWc2E2wo7xsrv0zx63E1QKaRjSZSYayFJ5SjtrznmZQNKpWPcwmrTpw8DF7lWeZ9WkDSkbGQOEjWCf9C3GSVwQzLOYuK7lPvDGHvoSfdugC\\\\\\\/acL9ynigvx5W0UJxGTF2F3Bkclw7QlRX6Eu75Y2tt3NCkPYnOoYLJaM2n+CcAJzGmzEhE9V4V1kaWbc1w9sEYfvrzwb9m6Lrv893U+hoBaj+QJAKTxaqbT7rFgpOoGvxxXlMkK5Mrap3VdwbY0s0\\\\\\\/Sz0HOmkSnLJSHhq\\\\\\\/g+AEzPSSZjG+oX1CwT9ERtwhoiX7yJnYbOUMseSbXWnPP7cL8hIwds5Yuqwf5B0eWyKjs\\\\\\\/JEtfeZFSAZyP8R7iM9LELJVZDlB9BrF9C7rnxOo7268vqvtzHB8JhNPajz9RZS30GbNoKy+bNV\\\\\\\/znghq9euzXXljD6Z2Yz7HbMudVG\\\\\\\/tgYyAyq2+uBuUn9jjXqBwPowkALJWjjQxQirjWEPClmNmg3MHPlOw9J6iB8q8UO\\\\\\\/JFcCQcUhPFzb4mV99nTuAZRAdbW3PioCQDPsoDmT1OpLuJDDfK0opG0aQsqS9VLDZYwQpvIwvtcUL+\\\\\\\/jHZ3nRLnIiW8eRiZsC3IkoDfU8u9pVMHotIzEKsmNwz49t\\\\\\\/4KaKTf45L4p\\\\\\\/vPZfJ7cvdtvt+NouTNT9M1XIPeuHNB+vGCDpa6dkxDjQQKGI82RmCaOSnbPSDQTVMulGLVpyUZ1d\\\\\\\/QK7oFwau4BW4mD+bNITdxfeFe8qWDyxX+HS7vVluh+43EqAVD6PTM7dYR8T5eTSty+sVqUL2wCbgpQE8XnhJTnvT7LsXYb6rc6bpA2TVdS+E2oDMmaaGytJj34Xc4JG3F+MOoXE8HOoe8rkjAyhTHU\\\\\\\/xxWauVMys1dQpqobJjN36Mq94T77Du7QdAHlgQn9puiOAy0h2xSi7Wvv7OQ9ESbpDzgoEl1Rru71\\\\\\\/xg1htMHiLXf1lX5NgiuonQ7Pjt+lahZtWa0Jut1ruM82mObpuyP8GDDNMvQvB5eahxC7cCnpez+AZ8OCRhtSOzBm1wK\\\\\\\/yxDe75QaLGYKp\\\\\\\/yPru6hF9EuOC6aARumBqUzZcI06PgZnB0FyMUldsecBoXk9S+kBUXzqyjwXyOQapR9FTnBf7SuOtTvWF\\\\\\\/zIQVr1EReVxncLzIl2AWAK9yskjxItrnxlFugnZGNaXJd3aTtiBg\\\\\\\/9c8sQTYt3toiWTokUcVF4mrszzwkx42dJ6tKxurTOMyUZgt2gHxLLwrI8gYVaGOKnXkLgNkueHVqdyaVgTOD2YT27b7baJPeaqKK\\\\\\\/8kbuUiwgs5zCsZncTTOgshyIbOIAXqMmPnepCdtjxdnYzRIdHlnkJ31nr3EQVAfjgNHkHwkQnLbNH7u0WcMclPxSGxF5o2pt4cUoggr+7nKXeGaVPbViCFFqv7LpVmuhcC9vn+tfgAkLSHRLkaxHlT8RyttAvqVzbGnE9R6wiEG1dLd86kgE9pz1locZCSyWmnHScRBeWT\\\\\\\/bAhYwouQDp4FTwyyggahwd0q98yLs52zJ9v5LAcCgCEo8Y3VIxoXyxXQHys3orSF1eA6iM6xVoqB0heYbkbyZiamINYIlGYN70sxmwULtRApeSwzWl\\\\\\\/h\\\\\\\/ojSnRFvZODrLKj5oJ1Qq9b3nbmh1qU7QYlHVIOcNhZnteDqMBufl4XdjX58XTXszbIOte0fqZIOuE2WJqSPzZCwUB7OOVXdRFiV3wlLX9za93TGRBQT8pmFGNrYBWNcHGNl1E5FW0HNxw6If4mmUPl5q+Uyfx7krXRFB6WJR7khx4P5l9JjrWge8H0EQ+NEiursbrWBT+S0DgAZJnSAfdORYxVTAVKp8Eyyyweor09rkOfxFJHqALQQKpd0VUlh27Yqr3HjR82LtJ1so5XpyL0qprEmGVCfyDWONCogsiyTEJ4tqE7rlO\\\\\\\/gs+xI0dPwVtdbhxYT4xTVKFQfRdSreG9FmfIlpzFKJLx0vNNoXQ1Ypl+mwi0DqKUIYAmwmKcFol11Z7rfGvFJpMhU22kKeJsiPoEUSfGah+t\\\\\\\/8OY7Wpe7V3tQ1zBQtcBGGuWT6WQ7m7bhcSC7+v5zCVtE5XWv1+FJXuC2jdcXBxOUSzFvaEqLusHl5oXvDjOacrQ2jvxo00Tj939QLo7kGHsGpqYB+p9B1kjwP+qdBHiYlV0uhLvSW0fMCYzvUvWQuH0x6v1Imp5MexAfVItUsYJa2TBkqIpo0qsCI337e6yczh8iYVB0EnlymBD\\\\\\\/ajNoUmZEP3hP2P7qwhJuTR6A0e8EJKIz+7zH828woS2ONYUoKP9SxGMUXE7rsszhS+GtCLzU0HaIU6oHCOA9IGyQX2hYQNyRzLooawe8TQglC51ScVbVbpetA0uqGMfIgzVTOXHRcSEuiCKhYm5sZ4Z+nO7UNzQeQqLehEcKmK1772RgErS7XE4AQGMfIb\\\\\\\/aojz70mqSr\\\\\\\/mXeyrGH449XJXQyJxBSTU4+O+0PaTM5JjvubCdSBDjkBofwihZhRPKztcGA9AlGn7oOOlgsJ6M0bzYXod6LorG67gD7dXeCrupMjHBHcz5fAyYkg6bXRdhgEfck7hki0FrA0\\\\\\\/U6G\\\\\\\/E3Ct7m60uHkJe\\\\\\\/by\\\\\\\/c27vXDLkjh1PH\\\\\\\/NzLA3lm\\\\\\\/6y8cmkUxnaUtBo7DivnC9B3\\\\\\\/s1CI25QwGJV+tAAFRG3kzBmyd6FM4OXOchubbSkzkMiES4T4wS3GtXifBHCkMzr9g+\\\\\\\/us58ACrd9\\\\\\\/a2I3qeHR2LqDacvUnCGMlcatoxgL59V2GcsLjh3W77duBYyHQz2zqCZ0qg3IS0IkS\\\\\\\/FERb5QNfCXJLohBmPA6tB3wEfsSogokl2F\\\\\\\/qlNOZxlQ5yKV2hWCzjJK3YVU0UOOvDScZcDe0bR7TjRJSzkao9hX5jkoYQghX7A6TtSYg7pvLjEMWAuQ8X++WPs1+AtfKxGPpsgMdRWgG1wdGdrPEnYLpyuZIMsToCP1YCDFt6P2WPmxAHprA2wvtG0dYaUglWw6hlfiJ7yM2L8Opk0AbFBC\\\\\\\/vN5JnZCDDXq1LYpb\\\\\\\/0bHCS4nec0mxQ\\\\\\\/H9fq39pWC0NZjUiJoNlQio\\\\\\\/\\\\\\\/u8GXczSFOJK9u18E8xrnYiG7SDu\\\\\\\/AwEUMAV29LwKONv06k5e2eWiHYDnWGd657pCgHzOlvzwryNp\\\\\\\/btzGpaQEWNSThajjl3ASQy\\\\\\\/PGq8uQ3FsWtatgTKUuXF8hscYr2ECUD8jyyTFFVaQ6JI4+Em3QfEqQ+xm0OxQ5CPNY54KXqEzHM1gqi+ISACOItG1Q1Twkdrufvhstqp4beqYVQEuF\\\\\\\/29YnxHCSOFHseqli7q6lJeh3UMRAA0XUdsBexkuED36nxggprAmeW4AvOitqJOrP2j4GUV+2sFFaOZxcTRBy\\\\\\\/JZFBfVKI15wLMN4WeklAszAyDuIjQH9BKrC0iPML5I7veDCZBNqtpgs0RrZYrQI0VDx0FmDiCrpnhWmE0XOlHlev4KaZkGzE0zZ\\\\\\\/NzqdW9WidbYrtd+NuHDHjDJWn6noRUPag7pNhU3E5L7DF5aD5XnghmmFTn6lvNW4BWdssUOvRGAP7KHGpHQvsvf\\\\\\\/5VE30BnRQE9Op2qZ8c52CuROXyvTTrtBMOpuo4cz17cNQL6dyrD85iyWKs\\\\\\\/044SZPGHRaUfn4BEStXP\\\\\\\/u\\\\\\\/n72tSirTZfjGiv6gj5GLVJzOySdNMu9HuuKN5p9eXqAzClnmpCiKTA7ZPdfuXYTligKLxxdiaiv+ji\\\\\\\/3AZmLJ22HK4wGcNlIZ5ZHDNSJmrThZJSrUSUHdCrw45i7TVThsBLMBgCFyU35hACKBWFHb48Ytz1RE2htkzro5kE2nLdur\\\\\\\/h5wwGxoBB3vJRVbQWNqAu4FdiVyPWzBbxh3dGbAcwUffYT697C6umBRIkgMrMpQSGZhOv7SdD882BMO\\\\\\\/6BNeKdiWCwRyBWFRFgyQ28w+RPoHHRqhfamvNFsgHVu9Jc6SDLwDclB6eF\\\\\\\/rlaAzYCj4Tybs7lCzGbHmwYF91gaYrLpKcM1mJCOD\\\\\\\/Kuj2Poj+021umO+4zkDM35SyoLlUriFdBpguI9b\\\\\\\/GxAR0UNLSB46wVw+eJPdsZ7ZncAZqKDSKasxLCjWg0duiLXUVbnt5yRKq\\\\\\\/cRw0JDVVdIgCN8oFouOG\\\\\\\/KOLOxQIfEXeatZygcaNKMv+SKEMlsAcOBJeL+DofIkuH5wTAQ+g\\\\\\\/XpvYD1v2IySoPzkAgFs\\\\\\\/HnGqfHWGwojChRYCz2GPRNZhMzFl44BJYrV4p6lrNJLyiFFz9E6493Hewa\\\\\\\/plumeLUhdGgxDs8mOcPo9pH61p24PQVh1gD5AhQMcNa5PLrHdWaObL1KqjqKpZyBeasv4lajAX7jJzeQ5FFgCSLA6hKY4g\\\\\\\/ZKk4Z3i4zqLBiHEOQAsdHvOSekbSOKdUnwJvi+CRhfTScu\\\\\\\/q0mqxaTjzLTRzXf65TCyNLBy4IclZrpuD1ACVImn7VZMkQH92OYXUAL4yVkbKyHO1yoX2SSOa6+NK70JTeMV3CryHlMTzRSGHat5M8bChaxdC9FXsj0WlUZiyubDZmXHJVaIIZ7pvkVkJ7QrFFBQSAdg54omxGFmy11xZZrrZvz3YaWzSu+S0scmKkG6DLyqb8wtrZQs449JoReCTQTWMeaBhZ5QjX5d\\\\\\\/5vrFlEih7qx7egG+q4QrOUAkbEVniMeHV5z30Kvmw3tMKh\\\\\\\/hsN4qpFtnD872Bs6mWznhAy+qJ00eabpt9VCukK5y4EX7LQoXvcQISluphpQGt2WoNQGmdhXoHAWsLImgy26gH8eXNm55RtsEo8BaxFUS6wfemInkNZAC6Z0JGn9YPAkxqpUqhIOUpnFeFIfBMWU1m3XX5MiuKcKjvglNqKEfO2K0vgI0ky3WTpoHGMgGHpMqWXJePScNtajVf5Y5gyPbxDwp+fUZwFlwozXsZFhpKUjgaw\\\\\\\/oW54pFIEZ2LzxueoDD9bJkad7PH1zSmRXarAO1o9Sd8kZ0LXyD9LSJNZCZJ2LXxlXoUcKjX7oSaQtkv0Tcqw7edu7fh\\\\\\\/0lQ83Ef75FdSHPbXnM2wdzgqOj4ih1oHAHsNBEmp2klwKqV0s+HlEyZsZmtggcDFG0xVsz9oItwoCKpZy10oWBTBciGvVy8Hm7CQV794aB6iiGdbuddS6zfG0FkTkhYRGSdTmgB5LMQdDmaHrBG45hhAXu+5yLbLbujvnq8Ue1KqEEX\\\\\\\/Tt7TPvcaJafmIS5UNgUy04DdD1PBb\\\\\\\/f7eodhn3KodCyywBBnpt55JrykdwrOvdSFphrbMF\\\\\\\/KD8YIrxOkp7eWDLRTGiWqVHO8E250TdTX3MI1+YAnUm5+uQ7FvMiHWrCPF9R3kM1nUhDgeWUj2vlYeKE5QnGOoHogbFGYlFqReib+smppBhJLL6T04DP+V3PVtpLhD9g\\\\\\\/beEHuvaTRQWT0JzXiW\\\\\\\/\\\\\\\/7+BG++mdnXrXsEAyfg63aXu6yXLXJV4vYaanTDLCTHx3D4kuzafNx8y2aTs6xM7P\\\\\\\/HMQyUMpiyp9QFd1lXVak38qcZYG9BYYCqO\\\\\\\/hnW0o6J5Q3TFE005vhZHAts4ZbQ27BPhDWnWgh8IZwdDpWGDBv3ToQuDHSdyuCZix+3sEpCMt7oI3PYleAlTfjzYQ6eyoLffsKuRZt\\\\\\\/d0im7S8p2qrzGttooCJjO2PYJhZ9\\\\\\\/Y+5Y+3\\\\\\\/rpyLEYChSSz1\\\\\\\/rJ5I3e\\\\\\\/IKmVPMtvuBpLfAg2PN033Ei0dZbJcQkJ4RxTbHs32injM8BTT+aRUSzselFqGjg0Fl2UYAHceRcrO4UNLDqkT\\\\\\\/mpRd+hR9M6fLEUmfT9AYSa\\\\\\\/tkRoKDJ5kxe8\\\\\\\/WMHu8kpZHsHFcewb6LPIZhI2UhPVJQXHozZGal2\\\\\\\/SM5bmlz9mlAIhaZz2TtHkOacD5VoVt+EzB5imHhg6YNAxgdeNYHsolGx8kLjJIOaTj05kbYZtN+wMO+ZBlM9iT6OUIHQOM15c7rdnbpOlDbwFLWaKgHHgb4HOtX2ZNCaF0v934O136USRg1Aqp5X2sD3o8XcgQxbA5vEkOTRvSyIKjAEKVZQpfE3bGzHwxF6ifw4LtIRZi0LUDHh3w8wlIb1LHNtJUe4fE895shHBUHOfn66KM4VhrYGN8elxkDwwwHX\\\\\\\/Ufv65ehc79MXGTsouY3G8hGL3IZdGN1J9l+PD16vVdXwpgo05uPUmPpv+M2wPA9fPfJ7HS8TIVsgZeZbUe7jPb2rn2axAEY+1voupRUGRlbeFzYOATgFr2pX6vcDrBliDV4fGyN3dMA9lyTVPtINvStWfJIypOoXjah664TETbTMbyOP4R7FxrnUKGmmW9DzOkPwXn036\\\\\\\/lpSaTq50km1vPS7kSl+UJi5Rmv0\\\\\\\/QrX8x7BVbJVuKc46SYX7ykDVbTvO6oKg8ShxlzUx4kAVLyS5\\\\\\\/fcFYWCiBDQUIJ+xXcpUJ4W8bmaQ76E\\\\\\\/BmitIw6PDYTBD9I1p6YUxyhuI9jNnoh4UwKZqkjcHpE6ooyA4WXjlTT08sdRB3jv7ILc4wsk6enrXLqF+IgnWzpDb3DOuULY8zwW+3mi1Qntbm\\\\\\\/GnzQ1v1oNKODTkZsoVZ6rPgF5HNqSEQ1tIeZbvSYbhhm6lyrjwnntgIAuqVRGk+Xk2pcFecgCv\\\\\\\/xVEmUGavSb8v16wUdPcxTcIAh\\\\\\\/6y3CpgaKlhz5V7zokjTli07CvB5bv22XnBktupDrSCLKOSgwjfEPWkyS5N3OA6CsaeZExNVvB7\\\\\\\/qhIEtruMEXlrkZ4WqHTyqeTJ0YDCqTm4B+MHKCKx\\\\\\\/8O3C6E1f8IVBI5BR4HoPHbumgWTcIjqvW9A20TX7MlvGjOPH2vtiVGxGlQOrHOyCq6hiSiv3Su91LmghFkpytj4vxTeQ47cvDOFHaAz\\\\\\\/0s32g0MUklv71IGtF+PfnT\\\\\\\/FrBLc\\\\\\\/X8QaY3\\\\\\\/9+2lagJfbWckSY41cqoDtS7TUCX6t8ZgeJaEjYWKlYRtOiig4ZvtMUzfxxNkmzmgv1TKEFspcvfrWtRbmuyNUsWHyFNgLnqWLWEEwzFDZQJCBOcZWBJQATrus3lBqQ5809MG+BphBksOEhJ6mWgPkjD6mtK9ngW7zJvAGptvlKxnf9Q3J9UJ+78k1TFT9\\\\\\\/ZZ67FtNJLkvD96dl4YhAwpF0kF4k4zW6u8g\\\\\\\/rJUnX3UK3SqMV50fJ4o1\\\\\\\/HPFFXN1fFj4zYGxystcCCRfgpYdHBue+qidW5Tjxs\\\\\\\/7FzPB2X7mX9fiqXkSsK\\\\\\\/pJA1ilswxx7j\\\\\\\/5YKDC+rBqdl\\\\\\\/ufriDrluaJKlHOvfIkt9yaEIwn6XgYwUvQ2UuKjGlaYChygJc6zA9Kmcvz94qUrcHfNMQmOYZXOhBpIdcUZqJfWVipaBDEOhuNV\\\\\\\/S0bIv8LErFS+Pf+TF\\\\\\\/1obwA\\\\\\\/3L4M8aiEn93nGHh3PMBYbNSXY3wO5rWOwdBaMFqEPVgGf2sZWiYk1HNRmtH\\\\\\\/Kkx4uPrAI1MeHtJsg1wtoYFqohJ7aw5oUSGmMmt2Ia9rUk8Md+Z8k7m0tbt7RmlgrtDNSFTFBey0p54juGQ4mOczDPxzDwhp9xRk5HB7keaE08TM0yIY51e58bl7WX98F+mVG1EiXy3YAxC8CfxxFXgjjioxDmH36RxgKZ0Wu1TexhMxekxu+cN0sFAamZZE0vHoQ\\\\\\\/ItAWIimufcmLMhRv7IwthznDwnIdTZYFsUVpMzc1JY8yXGR+4\\\\\\\/yN1GfTtIQywNP4N3\\\\\\\/HicHosYYQy8K6hGaucDG5IDz2Ilez7Dmc4F0sO76q4LuOzbdbI+OXxCywUdFMzaY9XCtsKob7XrZG4L\\\\\\\/M4U+cKXpX7aUfo44FqansZj3HldIsRy41++iJJ6wFAfc\\\\\\\/vs6O3ZUEoQQZ6hcIPEKqylNuTYZfDBkpbxKtxqC9JPM3KuJvJluMLuIgcuFUiJ1EMmOgj9FrvkOmoAC29DVzSKToxWuk9xQCbzS21IS5x+XAYN4ZalqDYOjdGi0xD2JetppSwioNnuNTbO2CJfVXwQ9FEYoCwtVqT\\\\\\\/AwU\\\\\\\/R12KI7lFbIkfwsIISEIqAEmXozFVA6w5nTTPAV8cWhdOFWCmPsR5qJwAk+LsF0Lb7fKW3ncrLt1iDyauOzW4sfDSfEDG4\\\\\\\/wOJySfbcMhOeybSBy9NsTuQSQ3ywsfOP858a7YLn7jLfMpv0P82TrZ55gO3KnCbNg56QGw6cdDDMQMnK+1OmJP8NlLHCx39ri6hovc9vkW8GPIYYLImrDUVLH7z0\\\\\\\/4k2qK2uBnGZtlqNK0PFsxzodJ1Pk5wFKMlTgPWAot7W31krZL+ofPCJVH4y6ICpEwzJ1uqRFRAi6wZ4AoXmYAuSA5TtcKP6kYZaXt1kaBMPSwepYyFzbBNBwlspPTPg6ll1W0exyYeep7BhNOB9Hg3mk\\\\\\\/duaL5g6iYoh9vQOnfSQgkbunsBOF4FVarbjVenL61jNEucpEA0j32zVfweZCJscE7Qcqzsblh8J748NpgEdr3ukafxC1hIelPmd86UPdmJvfEWbJB5dSmSQ76QjnIRg8Ol9ZKwIvhMmW\\\\\\\/gacAN9HRGJl\\\\\\\/n6L6RVQqqSd8WwwU2HYj3Xwkl0OKjImIaCjh4on\\\\\\\/P3qaPl2X6Xjp6Bk5QCghn79GHGoFiBBzA4chjVhorPdTOGzMLQBeZPMFx\\\\\\\/54WswhBQknjzKb4\\\\\\\/ACWZegdDfFaI0mz2VeiPKQBcjpuIqgNIZ2ZunvvEo4jDl0ZPXDNaZHKoBOZbdtnrqkm4IPhchPJIG2YiUGX9iw9LE2sBGHjmj89lMHf4iFEciRkxPrmtfJQrTFaHP40KHRf3tdbWtqfE4hs0GHnIPgTSWhxCipfiw\\\\\\\/XIQP2efyeSCHmuHBfIAO6o7J4lI\\\\\\\/hxQQfJV8TTlHpvBMgVzKqmcz9zagpTlMBMpni5q9jb8QaKCvME900pvL4BIxzdQdqvtUsw8OLFQ5wicKunlEz\\\\\\\/j+q2udzmX7VujIFu7D3XYpJFyNG9\\\\\\\/t4FBhJS1RCTVx+nOtvi9QwiBVv9vrJR2nS64zYX1mi+76HwCyO9IhF3Jm2J6yjCaKYse3q\\\\\\\/RvFvvlJdsu5myV2r46a6rnNY2R+oGNyLFjbImP9owZDKpJTzUdIbD735HAGe2gBg1KaUvjuzTVx7jpPFR0Lj8VtuzeabmnqZgQXvwB4HazVxmPsfbgMDtefn2sXu\\\\\\\/F8UPIJ0eCrKZb3qIStnUnpJ2M4qPPfzlVMR17J4Hw2l\\\\\\\/ZwCZbZ89gqRvJNeF6q+U\\\\\\\/6um4yXkcLzRDuSVd8m5yZRpFHGElwULIVdDP2\\\\\\\/UBYwqFXdP0QYCkRSmkVzGmoNLfsHPrYTM2r6otDnqd7GeBZ9qmR4FFQyHB6oM2bhzb5hoGL\\\\\\\/ZS4gNh8xSbrO9wN5rXtjuhQk4Cfq78mICcL3pqpuYSXXpYiqD\\\\\\\/shaXqdktQ3z7rtd9l\\\\\\\/EQ8SUNIqkto\\\\\\\/ERoMR6BpFfSxSc3yy2z+HAZ5NfAh+gfVsPjxX5VFueRBb9fuUTQRJ0Y2zJEUAmrUC4oT51F\\\\\\\/Qps2EuOHDOgDK5cVkx\\\\\\\/c9tz8cAeSMWL1tr4x2mCaj97DikFxbDNCk5KNUjSmAuUJxChF7EkxmMhq5sHOD29GjNgUUm\\\\\\\/0zywRNOV1DZJg2LVOFDIPY7cHEvIzdBu++TQnye2YH99N\\\\\\\/IwUEu6QeJ6ee4fV06MOSFfU3UpIbpbS9QTVbG3xL4+Ebr2DPLCW4shWJvWpkZI6YSJOjMlY8lhkD1gYo5W4JHVztamOUAMDsHCm6iHiS5p0THpThWqYqv4a3JB3bYgWguJKXCTn1klwfkcL7hDQ1bGK49LvY1BLU27JYbeQb+Qni1Et1zP5f4wgaQKAz8BfUWly9WP2eFFc9Rx3he1\\\\\\\/emrzquMqbjmUzFbbggKjPSs9qUYCJC0SxZsJDjJiQ3nYjFMhYYg0UBlsjMbZnjVXtfjHq9\\\\\\\/jmn9yasVODCHEj1SswEQmhPFY35M9gCXxDkCDHuihEPf6lwz1uTmfbeo5qvOcgC0wXaCox8IVglD\\\\\\\/TS7j7q8oOnUlgNo03UIOGJGjUfqQIX8vc4DMiRAaBZKZVTeSzHvYNTS8X6NbD7arFB03avsMdNLvSZeNTXmUCXY\\\\\\\/N2lUmV005t8YgMrklyB153pr8jg3gItlQeKHVWYASJh3kb2w5jh5y4WCzbetIaQv3zyV2crY+nLrZ3EQ\\\\\\\/irDvhh8qbj7yvA\\\\\\\/h1mfQ645Hk\\\\\\\/9fJYrXWXr6OZnDiQDPLY\\\\\\\/iT6WG\\\\\\\/inznClO+GxHD\\\\\\\/fobcpT9OqKqHQITenNx+asjlR\\\\\\\/ReKkii6aKW4B3A0ikRhWbW4bQKmK4wW364heIjKXMkA5rB3yt7p2zr0lWp+g+EoMHmbWwvWqhIbBKBWWqxUyuVkC5gZJ5HDSjb08f6bWPruCNDO9idoQwB69vikB5rKEK3XnWTY+gdZHICJLWGZLdraN2yWAgxiyRGsWrA3CCKcf5ExhT\\\\\\\/1BDCvt7L\\\\\\\/OVnir82PYVsVntJ7dbMTO\\\\\\\/eU+CiD48rnH+aOTBGW6r41T5OBoKNti7urXP8OgQtEOVAE+qr\\\\\\\/RtaoIRCogKC91iM9M\\\\\\\/DHDpdMMX\\\\\\\/ea+uVfW6RdgDyjItHnUxEI+DRNgDP1LnZszq1JqpL8ZS19suODkrFUofxMo2U+sP9GONTOSiIwi2UbkpvEoKE4nGdhlC30QwlCIJEWd4D3cFtG\\\\\\\/8+27w5IkcfIPJcJ9tA4RNAWGlNp8VEgOwb6vDNtqrtRoYBiK\\\\\\\/tmPvJRVgXRjuNhvAcz2TzBrG9D7VM1qsNhoL5SXxo6zCbJ5MSi7XI7pI+M09VFRyJ94tMyKCVWsN7YD5OgOMN3IWuR5SqLACaAkqWN+05wkYCd+O94OPtiWPyatEuKGMgSP2plHU92JJtDdsMwodU49tYIEE97\\\\\\\/7Ze2KMOF2pkYal24ITZWASJla5ch7nvyglQ5EeNWrSNRceF0MlON2S5JkYoYqctBTBp621YcPUhhDBtm\\\\\\\/Imq5Wxvb+Fcf7m8lhAAoppNlrEjDk0xSMnvsnzyjOzajOaPm+19mhYOQq66Xih+VIOT6A+YWv0HhQbFgqEMyt0pkOgZlnMwGloG1f8yconZ38eMstLpYETHfzE+4U1dcIsq5WpTaJ\\\\\\\/B\\\\\\\/9kgDWfnHW6e5cAqnMeVeLJ3WuUzarZQkNF9in59\\\\\\\/qMufv4JIY4JwO+9HNJCYoeEI1h2VKJjBTKFR2WjUValQ2mf7cvqPVlyCKmQVhKZbYvAo8d9C\\\\\\\/igE1J9uHKZeQGbvjH+in5mk9guQTMOZ9P7Cm2lYjWodJMhl4XbnvB8iScHx0\\\\\\\/qrrqyntcedDEZXJIg+hRIEPrrVWb+69Ut+HnC0G4CJWnStH+PPbtJyXr\\\\\\\/+1duqgAJ6ZeE7MDaNFZkvtWEWINDwxqrgA7Va0wdMQwz2VtaPs+ij7oPKQUj+9913rP88Ie+QPqLeNc9Bq9UyYnSg9pLhWzJ1gxFWb85VZjxkJMW\\\\\\\/eNt6Zr2x5\\\\\\\/4TeFsxQHToNliN1y3QXcvgo4Mwg\\\\\\\/ifAhhKg8lIydWTnfPlaf5NzzP1oVEm3H5juKaTw5LuvomC7uajMj7IuhDwdo+XYB+GuCte9BCNYcUaZ90zWq1j7BTo5HFSzT+apC\\\\\\\/2W6d5YvoNr7Y3A6WcMqKXNH+QCrtSohRqMJIFJbVQiuAi0NRK3ycjEGzm3m5Nv27U1EfPnj87U12sZNrxvKjLRL28hJ5eC9uIzQq1\\\\\\\/SypZetbDiin1CraqOGlEuEKBV\\\\\\\/OQuNRxUYhJw5wWzEQUY7ZSI4GQD9mmzC13d3+z8BvGtVPxpHZIMfrhjb1AjiJFjFzXV9CZR6l6KcaehpueHiqpl+KR76KpqhKWC5lH+IeXzaRrNc\\\\\\\/OR7DW9\\\\\\\/sxoCmdbCWgxGtk\\\\\\\/PdS5PIHBTLvVNzqBZifarOPTRHy+b4LHFmlXn5\\\\\\\/SATYyCRFmUGJIrZNH\\\\\\\/iDjVAzI9c9G9YvRvOkcec0P3zpzLSkoX0Np+oQbKguW57zjHTOHsgtLx0hKb4DDkakkbfEy2GxJnwtBpfzfL2eC7RPO4FyG5T4vWP9X3qKdLW+gK7RBMLCXpuZiG5xxGime73NR+PZXbhLLVIMqQFVcKP30qeOrZ+1miBImCxnGsqEyPPZLN77\\\\\\\/naRftwEP5EvBML6\\\\\\\/jriejuJQMn2ReHXxNJRfQEg8yqzv+db+HzfuN9GW0CKSGfrCO3MCFqOUNLSa3jMS+CKQLlVbXEWuE4HFzzjY0DDK4mVjvGe6b9gpe2JzSO6kQO16XmGn0NdEVxL3o4CkUDszwjxkynZO51F80HPFUHAxGVFKccTfJ34icKV9za6Yt0d\\\\\\\/+wTsRm7clTHbu6f6k\\\\\\\/K1dLtiJH6edVO+zAiAyP\\\\\\\/r4rBQNdJfShDSmXgfiYnWzA8DRmkKwV2UEJ+\\\\\\\/EK0jpFhZjGNBJrbGUpye7FKBKr1AqtALcSrtdOOZCLhb82I+7qkT7nC4BGb0X5Zd7bqiJJmncMPv2M5mAux612wjVGs3CNXLegxIzGOe+iXzoSPApX35BfAJNThxC3BxmeWCZqHsItvFZCMFAquB0hz3Cg84yDO9orIIN7crtzx68JjluTv3xf9BLPIku0rT9JjmgEAcBdnHAX2+0JjGTSyIFEAQ\\\\\\\/GsJia\\\\\\\/mfeC1\\\\\\\/AGUqERIu49x6lsI+8r5hSqYwQPu3\\\\\\\/mbGpkf3RZGk7PXrFUkifTXXjOdEggDXclF4np42UJWD21WUx1erUjMqM3bIn30x\\\\\\\/Mo55wF4wv7QLE\\\\\\\/+MWfU7vkeXObaKSa1ewig0ZZ3atJWDzyNWV5E1NnFqsdZfZ2GcjVHeZcQMb8514LLdz\\\\\\\/Kl3UDpR8XwgEhficmsLbZTBz44LQGLK4DPFPHmNJxL8kUDKUsyTTaQjVwD296qLXRZ7vdGLVzlP9LD7HVE8FFUFVlc31FzHtweN8KZdEU34PFqfIy3NE69zlcBb4bevJsPhOWsJVVEapl\\\\\\\/GKDmiuVgSz36QWds5LCGAtb3vjLeZZ7h+YCXEvNdKQ7Mnx2kmuqIJIjnj+d+OBPDI7xNp7t1jqhLxCK\\\\\\\/3CpQmav8sQZ4fkCSi\\\\\\\/0yt0k+aOkRKH48xs9RdCy0mo0h0jPdy+0S66z9nhNZDcgitYq1tJWMdiIvN0G0gw1sCCyac+WjGnuABgH8DRZTYvxglfs0VVt6sUyCI7kSGgDv\\\\\\\/oT6UHONo+yE3Pe+8WaRQO+AqYRH8yq\\\\\\\/bskDpSIzSiIRQ8dcQRwXq\\\\\\\/x\\\\\\\/kdfamWW4wtYZ6O4sOI+RmUR2TGb7CIMfU4rR9tfRzD4aKriTfpMInys8Man6HSoCpR1EBsy1O0Q9xAwZD+GQUQIRtLom+oTor0IhsHqlm1HYIgtaQJJSFBGjnbaAV4yYzrauqxTvcmXNZiR6LFvWWWgVcM2r13dfx+e63QbhlL+yf0AH2B9pYCbI1XL5h1+k82m+0NMc6XkhfQyDdVpvQH5jXXoTjSNTf8AgBP9EuLRiwVVKBXZdibvDDKC4mLI7u7UlJpVpU0APxmj24\\\\\\\/s5WX7n96egzjhYHygis7gW3QfXbMvtG+rp5X\\\\\\\/eQBp\\\\\\\/OS5Kay6I8+FRvwk4Cn\\\\\\\/ePf8WjH5H0O0zaaGAsdA4V65UQt5ds\\\\\\\/luOZ546NJCCKhneq3nYYLMAHhcn5CJo8FERdGq8O11lAoLPthy0phLFIsWQfftGV+BuS24TikHxqvLEoQjAgQzOtCPrNDd7gRrU5PlQ7Ro7kT5SLLlOjfQ\\\\\\\/b2RFrrHXIrzHlRBPpMx9Q5S7hAOjvA7EPAND91IuzSZu40X9A\\\\\\\/uHhnWEXeDiuE4nxMyCwp1PhWdNQZ1SGnfAVze9GdDfuUIy05JmsdzaZCbV6540Sr+YqjXYH7i4\\\\\\\/ygvkiHoWSDMIzsK5u\\\\\\\/opPWR+w\\\\\\\/\\\\\\\/riPDIpfKartv0O3PuMZuNxCqbJdtw5TnSiyjWOlmdROVq\\\\\\\/Ci86t5FH+3L3V7DCZgWvl2cQadVwMO6xHAXmXF\\\\\\\/e0MOAyqtN9OG\\\\\\\/qoYbwg21rY2ZdkHmznQQBGaMWHbbvJMq+uePqipD\\\\\\\/mGmnekeIQgsHFMCvIDMHk\\\\\\\/nIvS7r4\\\\\\\/TT9om2lxfA3p+zV2gtjvPTH8cTjFVzwjYfFiC3VAFt7o4ZcGSJIJVXZnunZLdCA+3HUcr+5VuweGRt\\\\\\\/F7L6TmryR8FjnTTf+ajjKLYDtAvLf9DPjweTYCdku4OvRXQH7ImBXxTKkjYiDRZaxPoCEGKi+XUVzX9I41yihMik59eSwjGXs8adAOfAk82B396BdOzVC1XRa0zC8aketJ5hdcKxTCz7RpquMZARSJT1vsca4Op2s7FQffsTW\\\\\\\/parSmluYusRjheVKjaDyRBMgIY9H2Xy02aOaABp1yKJRjuQ7axcjd28YsrXGZbPDuSQUwjgxri5dEdggSMt5P6T+XH8\\\\\\\/28JAn6dUtydQs5Lig9UIFNfYoVN1C\\\\\\\/2PraXLFVKA1pXlj0AQfCJzbuKrYHqPC4L7ixCQmn4Un1elpak6VyaE27RQhXsjTnm4VJLan1nglTwTqk1Nr2cMJW7KDFctlDVagAUcRfw2XUToK68\\\\\\\/5ZXLl8zMyR5p2xuGsXLmKJP9kDpAtdqh23RUY+awG\\\\\\\/M5xEdxdoxizP5uWk\\\\\\\/ASZwvrkrTHNlWWiymk6U5nfUV8mqOlj0LXeJ8vN9U07DxJqNxt7o0X4vvA8Lkn9WCMuvRmZETypaJD0MXcrFmhZVHAqCKPRMa\\\\\\\/b3r57RHHlcQskWYtWKUWmrdtzLMJU8gj0yIwLzioeCR7fLWW3VgJWYMLQGcl6mSzsFdYfU0pXsEIz7QcOGSPd\\\\\\\/hqEvN96gBXjYjKBeP2+EYr46HEW1cWXIeGa8g8J6aWgqjAHgCWnj664AQ8pYsQUP+9QIVZgL+W+\\\\\\\/Ws1P0lQH9kseuOnTnzQMOZsISVZwo1C6CV0ERWddQS1oZ7lsm634zzSxp13haUimXPcHnQ7AjhN2HW0pUEAuGYnQBqnYRw+YdVD\\\\\\\/cRiJ32T0L6cy+SXw6d+UIJz\\\\\\\/xf10JIsAi3gi+uL13CZ2bLxRHieX13\\\\\\\/dPSvSk0uWE7ZlfBjA+oJyyLz5BPrA0ZAZyZPxmVF0cYNjw\\\\\\\/2\\\\\\\/Sur1Dir7V1+F0l1RnRhUL6PDgdmwwR6t3ueXs+W\\\\\\\/OLTDDuDj7wj697893U1C\\\\\\\/XZ8OQFOydTfbd9h622sAP3uPwZgNPDPl20I+19yvZqKCc+JD30OTY0W09\\\\\\\/ig84DTF1JEMxYnr8aNSMt9A9BvZnlUostuwLde3cK+tTYC65TupzzjFI4+eYcR\\\\\\\/kknncpuVcspOfi8Hv1i2k4CE6R\\\\\\\/LaZRhPU4hVc6iEbiVJlt6nmG+iIIRzlGGyhcxCB2FF9Z4kQoR90OlTDetRQMAMmNwS\\\\\\\/HLLtQgDaIFibQIrpn\\\\\\\/\\\\\\\/d5GM8NhW8FAef5mM2G9FnlXvwNU9m73xHtQ0JDMFTyBMsiQivpKCvCJlwCAuqS7H2ijh2LtuBMTAoVbYm0d4\\\\\\\/TJe88XVlEXAwiIO9VEbAi8h6dQtVVP9VER2zxsS3v1PmYymSF5wGYxHbpC38MivoWWNd7praIazEOGlfglX9Xukw8R\\\\\\\/ds5cYtSWxaSzRXjMlTU0KAhV\\\\\\\/IZUlXcAuGT+B\\\\\\\/FEBylCVr6tPovOLP71d7CwhOqJR+ObNo0C3q2TqiJ8u5gYMM1ycYQzUayhj4fHzOasBFEK0zRppbCoqK577dia071LbDA+ICHrX61qkclPSCCgb09VL4Gox\\\\\\\/5iHKdVqWEghGyUjF1\\\\\\\/r5Xqj5RELvvjhhqMKI9H9K3vbOasFYpUP\\\\\\\/nicNbqEW+uBycbJu0aZSJLTY18yAFfsystDkppDyOqJ3sfZq5QVfyogRxuZaQAkjMdvG0PoggBCdT3l5K0kzaz2JAJgsgWZFs8MMIKycGdhI7mSzCeHNbZMY967VLvGCF9v92E85apIiAEixvSYUxKfB0E1II38IwSn1HGAW7EXcBApuyywX9NFj566z911S8YjLjdssYweR79SdLsLaOMxvDjRX2hxSHlKr01WmrzqtXlYvQrVpA9djkecvjMigKp7wNIpleyF+GPS4hWqUycwIbo0puvYoBGmVQMwJqMWlug4wMDgguTmyJ0mXW1sYmyHwEQiG\\\\\\\/ZiwlaMwVlcUEzJAh3y9PXhar6DtIgxsHoYaIDM2Zx2KfbbjNw0QiUQYm+\\\\\\\/gMHNV\\\\\\\/zFiC8Jq37Bu\\\\\\\/\\\\\\\/A4Tr7MWSRCvjkvmI8DnrrLECl2ul8ChzjD0h1wG\\\\\\\/Jhsa7VJrUC+j52uS8YjhNktQdBZw+Vzgjf\\\\\\\/9NnoHjBj1dH8OP6wJDIWyNaJC5VShPAzg75aVxu5KrebwUj3Ng0lKsKn9z5JS3LyTRzqS9wdiHGcMBU4DOKK\\\\\\\/Apx0RYNwu2Lbq7tLOXqg+lyXyWj6+3WoDojkBjFb+2v2jvy5I4Giw3SjfMRpzHjIc6rtjM5Lzu1upY3HsIr4UXvFp2pHKHx\\\\\\\/18st9jtgzxrbJR6nYGlM2gFT1rpEGO4gXnk8C6s5VMvLrHtQu404FAfWSiuq0nTGBG\\\\\\\/+Wz+sD2ZqHN1gNgzluQLi1qWqtk3FwO5BD4YGawaJihT0CVOssSWI\\\\\\\/9AqkEWwfahNf\\\\\\\/i4FcEQtUtVM0lGkqvmhe0l5D7i+q3rDzdLGolmYI3KtqhraLh6lWygLnZN5JBfkoGGk9OwOL64VAEpUVK3oHzMUhQmStXYHIXTpzoFiiQfI5iUwkH9Wd16Qd25mV6kZ4z9yIlS+1IzTrtiql3Nnuhp9eQKlfAmCnVaNyiSJHeL3lCIlauii46aUdI8cLA84wvSymukpVJQDCd2MiiToGlGamSBc5uPn6FdhaI\\\\\\\/FornDZqBG5Rf\\\\\\\/FG7AWOk+INAYENBAKPm\\\\\\\/3RGARU1J2ZKa1kpVDHQFoNN+IQENxXAW4dA6yfwIgNjaMK\\\\\\\/NNZhGXf0HpF2Ee4tbjPdksgF01aAKVpkaGVPLskPs1IPEdcYcP8U4eat4ehdGv4IYiisq7W2MZxUHGTcedOG79r4iW3J9KQhU9\\\\\\\/1qagh6l09Uyam6HAoPtA19RPwXqTti0Be+jVfSm5zlOSS9EN058FmeFFwNL+tTZgajFA4SmotDSaxoUsErkJeNT0H0JZicfaBWDiZhxgO6K1WjwsfcW\\\\\\\/P\\\\\\\/RO8\\\\\\\/lAkVdlx4epH8g6dToiHfg7WI2fCJlgLqZ7NFEVy+rjdkMtoA7yFEC0iXhMK7QE4fCGAeB7n2Y3lSLVwFxnupt9Yi8AzYZjE2AQWSldE0JR60FV+AaO2XSB4XGlix6lTfbWJtXDYF\\\\\\\/0PTvdD1SuQWQQ70xqiSjV52w\\\\\\\/08HKTR1PiqSkJkGRk5lWtMf8bCPgQUb9qCUXDz87EX3ZH4KSHrBmeKli7wmpVPgtRzkgGuTt\\\\\\\/ygyoR2qT6m6IN6GHOL66DLNLdmWlDBXK8VbIHNk1A\\\\\\\/atLBEbiByJb8AVeapU9QmNJi+F\\\\\\\/LtqZL+pfy0Z2qmewn9mJF3VvY2WfoVzEruRqAA5oLXFLQryqqx9iM4lezmb9Xhc1aAriMkqff7ktiVnNOuIhS4RIEndQuDCWW5nKZQurODflRn5M6S8qHDZrsr2pj6lddNVb6pJJPTujKY9Hu5G5R38oNH\\\\\\\/bRlNRuL1XkYJ+\\\\\\\/KLNISzMJAFtJTLQ72jRjuswkV8qVR2X5Evmq+etxARP0PAhdKtn4crQ76pDg\\\\\\\/hqrOF6pcMr7MVsYuT9UvnRzYSd2ZkQzApNOwoGoe80Rv5HB8\\\\\\\/nS8GHxt2o4eUH1W3YYKPhu9f7DYOjXUcIN2rtxreZVb7nxF8kBGd2hMSsm7zB8g4D0SbQbkaX5\\\\\\\/kDQaAXe4DVWtT+JU+aKwdOQJV96HP0tyWeakn5kL63FB0CqBk5hlq+nFhccvD1wx8CLhT1Uq7Veq+qumrSOiqsns07mADh2Ny7q6ScmiqPsT\\\\\\\/aYeOpzH2aZ0dy9iLPsJCXrTZue3bjSppC0TBejUKWmGxLEnkC6NkHNlwkSJfxoyUdsOV01AiFoU3ZqakuOwtDubD9LJ6HhCg6Ld1lKh6ssz\\\\\\\/hw0GfMk3mhWfOZcvf4gSGp+su9Eh79544FO4hULSUJC3N4UNo1azqxtHZ8VBlEa62Tv1GVgofVFzgMixteRwm7TOPMpnBnN0TUlh42OVpEy15BWSvq8SFZp3OIiQc1q3psLKiv0sNvVqmnBxnX\\\\\\\/fkx1N9Lew6vo74NDtIweze4FGM4PndCzxlZ9qKNfl0\\\\\\\/MMyA4D+dW\\\\\\\/As+CEvMEUIUZiFgEpLqrsQ9zA48pvX0DUg8OjPgrb+HUfKgsJs5JDC9qOGoCQgqZkLJE1fOMXEbsOtSkZPOF1O68h1HGkmYDtTyikOEtesC3qHLkxhNDbE7FOGUOygSaRYTzqZcMrSDibOOyKwk+jakHSQbxfAFg2GfGFWlAX2jBaJf9E4KlmtHHzH7SO\\\\\\\/0RAQn2E9TnNHmSv2uL6PZ5yOgnThvmcMd\\\\\\\/tD6OQNoqy7UWuiLARVyFhpAwmt3jnHU9u8pchcBqB\\\\\\\/H79fOoYkeJiLjuoXXbX9qVi+2S5B7ITDvYyi7ABYIx4BO3C4xt6E3A5u8wngC3ZtbZb5U21Qptf8+lI+derooCCl1yzr12kN2KWTDSx6e9VSAdze2zkomJ4dsscXvpKq9pxkT7CPClrsxh1JkmZVOY5Oac5+htD03qt6RIU9jnEJ7t2R7GpdUC8\\\\\\\/Us+dlSz1i2aMX417gQziLv0POrPCnhNLJmtZElkZaOXkWCqsURwML0LNIXnuxG5E7Xuq4u1vSF+45BeU+Wq0H4nED0DqRHcgTJ+WxDdiFUkaadByJ2zbOGaHZdFSOfW7ygcP3rHwd22vrDJ4tenuhmBrRys5QfDiR2AHt48miP2Z1ZofrKS2IR5KedgHZ7LDs\\\\\\\/pSJzotWG2mAkpduf5mgCMD9TRTauPdhOESZq0XwJRyGf+JuzWhS2nenXJ8fzz0x\\\\\\\/nDUztV03NdgzfVcRux6pFIt8YmHo4EKlcvwuYPxQHp172TfuNF+K9UGf+mCnsGfQIn\\\\\\\/89HGZbh000G+fGq5Ly5LJtq63I9W+qJoDWP9Q6zU7es6ZLxVW4T0ZUuFTNxRlDAKT4nMuuhxIU+RQXS533wE17G45VKaym3az8G2bXf9RluWO6Mh0dFGohnJJm28bD3GnzcEO69NLDeLHbqGOs3JfFV+PnZVIihOQ\\\\\\\/ZgAGGsB47EZUr5sHTwOgVvIlHbScIPSjVrcQlqBz+JxGb2SbgUQSPXIK2hYgP1tickq\\\\\\\/ne8AvOJBB+oDcR\\\\\\\/gejfrmhMoZLEHdE4H5lmMW0a9qXTHYm8SqezWEXyqrhHxBQNuPX1aJRxFmlLDIyh8MkSat96oshwfiUtuXxLCa6wOLQoez4w0nQUvZP84xYba0wYYqbJUmKo\\\\\\\/j\\\\\\\/15YqFFTwcFPMlhji4LGk3y3pYMpgorTULpv\\\\\\\/57\\\\\\\/2c\\\\\\\/gKMyWnPq8reSMWieAtkm3eraoFHwM279o3lfKPL9VYbYFuXhZR6+FPSKE+933ILDNPSdbH3GKTsMPSnul\\\\\\\/fxM+VN+V3ElrKM+XihKUmw\\\\\\\/fKmblCU9h5VCoMw7tJRxXBpwOV4dDvOnQHRCa4fQPinYQR5Yz8dyCCBS5mrhhIXSbxknokGiKwKYVPvLX1F5rE5DdMK20kk9MFSyYOmUWzGDnvzWm6+xtWvaX\\\\\\\/LmbeetZfjpPgyVsdTfJLY2eDSHcjHl7ZY7hUAt88M+5ho3DHUCEPY+HDoXim73qCcLbpZcWl3fGF7wieGNERyS\\\\\\\/jr0E8789wS38VvA9gxGR1jey4mK8ar2tITdxYnr9EnokzvB7pn9TEOz4EhXCmpK\\\\\\\/bt6JdA0nS+JkTJzkMprQIFu6KLxXrXvfCoohNrsjXprhgORQxUf5013kOqUrbjXT0w4Oz\\\\\\\/pktI8ckDSfCXL4gbsIxFsIUAqPtMU1Gydf4w==\\\u0022,\\\u0022iv\\\u0022:\\\u00228wmnTU1WzHfaRFSa+kU99Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e6c81a32de9d97b46a37bbcc858401bc35a9b003d5f81af9bbfe382cfb93d33\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z8raJR2+\\\\\\\/8oFpP98Q5eNEL\\\\\\\/vU7+wkRQwMzsF9DJWt3NjKCYkQkzk1kOJxwXO5Wbj4NqSM+9ZW318ZuLb07Az3JFASxZK+lj8gP3Mhc1VCe7zWJajwhhnkw8x7AQ3UH8RqbkxeIJWFTmWvzf0MC85s8SwBFZbPel2c2rT6KhUVgZhMTBPtAqyV5nIxFLwN7k4gpHbejSCRi6QJrvyC2Z9EtKOzIpz5uhiM\\\\\\\/rGN74Bl10XFVT+wY\\\\\\\/TPtkH48XSXXIYBH22ocFGXvpBGSUi+JGKSImjeTBTlSnZTgBcwyBWK\\\\\\\/VtAPLbPp7a64v\\\\\\\/EQLTzHtai+PfM5Z3LcyqkPAbg8Fl+79bKwIP8FEZSVYInT4\\\\\\\/p8fi+HZCD1eQvP3BQvaNoyIAiu5c4enxI8Xq9Vww2YPQ7fTV8yt4rDGH6mTwXNI1DnB0btQNqehrCTPUkFdgrf6IpI6ivkd2\\\\\\\/Mn\\\\\\\/fdyxozBLd+gB3OW9pWfuYIcAnN2+\\\\\\\/jY2bCe0Dd13tTd9HskN8q7JJ5uyZZg9c1vcNWJHnuoc3G2NkmK6sPW63A6bLa9eOjwm2zLp\\\\\\\/EKAZ+YOwfcYz040NEEfaOfncZsuqwNTxKQ4GPIEbDDkoCguKEA7yPpf2Q4OxI+LGUKSXdOMgBAknHwpzYjpfBC6baExA0nDe8vU0dDORwoLM15lI4G3tXxnmmfjZwk5bi60NfmCSxl+VRrcydrD95OoItfzGzEhLbBB8TWaMj49X5gHfZZrx+EtwoUWuUZ5QzootLEsjtY56engm+Z\\\\\\\/w6qfSMaaSikt41pst+Y2RVyHx\\\\\\\/wkXxDvB6i0bDs4hbElvfw6wdkQQI2eWbWPH6xTrUzfRJ5Nt7KhVFEZ2\\\\\\\/0fLGHIYjmIjLXCNQDuPaG8xq0Do1cDp540riY1hNxUkhG\\\\\\\/LO4plIuS3kv9rD9cDUYBgrCXHU3Q7pjXpm6hnN9TvO0sPXCxyboBUUsr8T587R5Q0J1cMebUvwxaEpMaY2awaPWkOQ3eX8yltGKCDEZGSzJZJSZzmCSpCO+IqfQmWz9bx3uj++qhh+F8ijNQTF4SWkDKSlzBSW3rBPWayb\\\\\\\/EubzZqcRhgLvwIvyblS7kMrUq4xPe0VoxKYM0CTrn1RjIWTF3BaR7jhAESY2bNPTQ02\\\\\\\/DcIWo6k8MzV0Bgch7rKaCjNTTtT5hJtVgoV25J2sWCStqnH8BgrBZ5OtI2aEtCaHdfUl9xsF5JfUPWfXtius1aH9zdmk\\\\\\\/fOlWKCO+YkNIbfY8RM0kZAEEcAQjpAAmYGShT+UeTfIo0ZiKmsvW+GSEzYvmbpP5x9ajAU099ugTz2Z5sTbmNNL1bSrlqAyV6SN4q\\\\\\\/QBmdc8NCElWLNgJQ4ME0HTHyw69j55xH2JKyrQBhw4SMlsKc9YUnfSUAb4SCQQbC1VnJFk2MM6E58A0eUf3qf0lTUcA0nXRNgX98XdOoLbsxw7vh8CPtdhe+bB8snRnxeUR7ox4L5onBLP3N5KPUZDdtM9hXuZ86LJR3JpiAbqAXpco3Zk\\\\\\\/EzgILHvwyfiz5Gw5jtGmWACTKIFRtT3DrRB4ceAQXNcrDtPgT3bXvo56Cyk+d\\\\\\\/VVGRzaxmQxkh2ZssW3YXQXROyHgSETX2xhl3l3vTjOg85bojF7IdC06USTcn0fWTX\\\\\\\/xkQt3Au5CRQ7zv+948EOZZJQ0Lg1KHM5WGZJdctd+ewnsRzrJb8FXUKTGAHU8IcJAblHKKfKxZ3YGUVHoI+qWElcEXLExgjkcDb3WTw4LG5++1X1pNVwnTLrfyhY+0c\\\\\\\/p3HboiNV3S6KsYOIPmFtSouHtCNlOn7O9TqYb9Mo+rZBi\\\\\\\/zmsutE2dc6n6E5AGs\\\\\\\/JbQfdjem5vfTdwxkdS3\\\\\\\/KKulEepNiQQNL+GdYPW1UH2ed\\\\\\\/o\\\\\\\/gRDHUrWZphzIt7HA5N3OMKcCkgaSnSSQw5iHxXQqThqNUBH+WWggzeHEsLxuBBcWhreYMNWsMqaPhJhlk9POVNwTcNIvC+kMQCahqe7DEuXDUS2co1E3hkQlwd0BU9KsA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BIgTMfiQG8MLY2eXmBwn\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223d64a2dec3f632aa6e2812bbdea62634ac3ddf1508a2373aa09c074a55e31d76\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460179 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2383" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CNWPrKiFtmJlfG94gO-IWt0MK8f8ndd6AkRw-T77WAU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2383" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BQwmcwDpUu3aMc0Jm9O6y_-XWObbHy38oiAqBrQdUoY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2375" title="Post-transcriptional Mechanisms Contribute Little to Phenotypic Variation in Snake Venoms" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2391" title="Genome-Wide Association Study of Ureide Concentration in Diverse Maturity Group IV Soybean [Glycine max (L.) Merr.] Accessions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460179" id="node460179" data-pisa="ggg;5/11/2383" data-pisa-master="ggg;g3.115.021667" data-apath="/ggg/5/11/2383.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Money</span>, <span class="highwire-citation-author" data-delta="1">Kyle Gardner</span>, <span class="highwire-citation-author" data-delta="2">Zoë Migicovsky</span>, <span class="highwire-citation-author" data-delta="3">Heidi Schwaninger</span>, <span class="highwire-citation-author" data-delta="4">Gan-Yuan Zhong</span> and <span class="highwire-citation-author" data-delta="5">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2383-2390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021667 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460179" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Money </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Money%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BMoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:daniel.money@dal.ca" class="" data-icon-position="" data-hide-link-title="0">daniel.money@dal.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zoë Migicovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zo%C3%AB%2BMigicovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Migicovsky%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZo%25C3%25AB%2BMigicovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Heidi Schwaninger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heidi%2BSchwaninger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schwaninger%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeidi%2BSchwaninger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gan-Yuan Zhong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gan-Yuan%2BZhong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhong%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGan-Yuan%2BZhong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2383" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460179" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460179/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2383.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460179" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460179/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2383.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460179" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460179/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2383.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460179" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460179/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2383.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460179" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460179/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Obtaining genome-wide genotype data from a set of individuals is the first step in many genomic studies, including genome-wide association and genomic selection. All genotyping methods suffer from some level of missing data, and genotype imputation can be used to fill in the missing data and improve the power of downstream analyses. Model organisms like human and cattle benefit from high-quality reference genomes and panels of reference genotypes that aid in imputation accuracy. In nonmodel organisms, however, genetic and physical maps often are either of poor quality or are completely absent, and there are no panels of reference genotypes available. There is therefore a need for imputation methods designed specifically for nonmodel organisms in which genomic resources are poorly developed and marker order is unreliable or unknown. Here we introduce LinkImpute, a software package based on a <em>k</em>-nearest neighbor genotype imputation method, LD-kNNi, which is designed for unordered markers. No physical or genetic maps are required, and it is designed to work on unphased genotype data from heterozygous species. It exploits the fact that markers useful for imputation often are not physically close to the missing genotype but rather distributed throughout the genome. Using genotyping-by-sequencing data from diverse and heterozygous accessions of apples, grapes, and maize, we compare LD-kNNi with several genotype imputation methods and show that LD-kNNi is fast, comparable in accuracy to the best-existing methods, and exhibits the least bias in allele frequency estimates.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping by sequencing</a></li><li class="kwd"><a href="/keyword/apple" class="hw-term hw-article-keyword hw-article-keyword-apple" rel="nofollow">apple</a></li></ul><p id="p-3">A primary goal of genomic research is to establish associations between genotypes and phenotypes. Our understanding of human disease has been significantly accelerated through studies linking genotypes with disease phenotypes, primarily through the use of genome-wide association (GWA) (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2008</a>). Such associations also are essential for accelerating breeding in agricultural species. For example, genome-wide, single-nucleotide polymorphism (SNP) data are used routinely in cattle to accelerate improvement with marker-assisted selection and genomic selection (GS) (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hayes <em>et al.</em> 2009</a>). The discovery and exploitation of genotype−phenotype associations in an increasing number of agricultural species has the potential to dramatically accelerate food improvement (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">McClure <em>et al.</em> 2014</a>).</p><p id="p-4">No matter what the focal species is, the discovery of novel genotype−phenotype relationships most often requires genome-wide genotype data from a large number of samples. To date, genotyping microarrays have been the technology of choice for acquiring these data. Arrays are widely used for GWA studies in humans (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stranger <em>et al.</em> 2011</a>) and have also been developed for several agricultural species [<em>e.g.</em>, cattle (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Matukumalli <em>et al.</em> 2009</a>), rice (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhao <em>et al.</em> 2011</a>), grape (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Myles <em>et al.</em> 2010</a>), and apple (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chagné <em>et al.</em> 2012</a>)]. Although arrays have proven to be effective in a few species, next-generation DNA sequencing is becoming the method of choice for generating genome-wide genotype data in many organisms. Methods that use reduced representation libraries and multiplex barcoding have been developed to generate genome-wide genotype data from next-generation DNA sequencing that are similar to the data generated from arrays and are suitable for GWA and GS. For example, genotyping-by-sequencing (GBS) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Elshire <em>et al.</em> 2011</a>) produces genotype data useful for GS in wheat (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Poland <em>et al.</em> 2012</a>) and drastically reduces the cost of linkage mapping in apple compared with arrays (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gardner <em>et al.</em> 2014</a>).</p><p id="p-5">Although genotyping microarrays produce some missing data, GBS produces an extremely sparse genotype matrix with mostly missing data: often hundreds of thousands of SNPs are discovered, but only a small fraction of the SNPs (<em>e.g.</em>, &lt;10%) pass missing data thresholds (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gardner <em>et al.</em> 2014</a>). Imputing the missing genotypes has been shown to improve the power of methods such as GWA (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Li <em>et al.</em> 2009</a>) and so genotype imputation is becoming an increasingly important component of studies into genotype−phenotype relationships, especially when data are collected using methods like GBS.</p><p id="p-6">Most existing genotype imputation methods, such as Beagle (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2007</a>) and fastPHASE (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Scheet and Stephens 2006</a>), rely on SNPs ordered according to a genetic or physical map. These methods are most often used to impute genotypes in species with large amounts of high-quality genotype data, including reference genomes and reference genotype panels, like humans (<em>e.g.</em>, International HapMap 3 Consortium 2010) and cattle (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bovine HapMap Consortium 2009</a>). These methods first phase the genotype data and take the phased haplotype information into consideration when inferring missing genotypes. Generic imputation methods that do not rely on phasing, such as <em>k</em>-nearest neighbors (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Troyanskaya <em>et al.</em> 2001</a>) or Missing Forest (MF) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Stekhoven and Bühlmann 2012</a>), are not specifically designed for genotype data but can be used to impute missing genotypes. Although there have been comparisons between generic imputation methods (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Stekhoven and Bühlmann 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rutkoski <em>et al.</em> 2013</a>) and between genotype-specific methods (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Browning and Browning 2007</a>; <em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Marchini and Howie 2010</a>), there has been little comparison between the two groups of algorithms to date.</p><p id="p-7">Here we introduce LD-kNNi, an imputation algorithm based on the <em>k</em>-nearest neighbors imputation (kNNi) method (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Troyanskaya <em>et al.</em> 2001</a>), which takes into account the linkage disequilibrium (LD) between SNPs when choosing the nearest neighbors. Critically, our algorithm does not require ordered SNPs, unlike most existing genotype specific methods such as Beagle and fastPHASE. We compare the performance of our new method to several existing methods by using genome-wide SNP data from apple, maize, and grape.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>LD-kNNi</h3><p id="p-8">kNNi is a commonly used imputation method that has been used previously for genotype imputation (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Troyanskaya <em>et al.</em> 2001</a>) and has recently been extended to categorical data (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schwender 2012</a>). In this work we only consider biallelic SNPs and code the genotypes numerically as 0 (homozygous major allele), 1 (heterozygous), and 2 (homozygous minor allele).</p><p id="p-9">To impute a genotype at SNP <em>p<sub>j</sub></em> in sample <em>s<sub>i</sub></em> the categorical kNNi algorithm (henceforth simply kNNi) first calculates a distance from the sample to every other sample. We use the taxicab distance, where the distance, <em>d<sub>n</sub></em> (<em>s</em><sub>1</sub>, <em>s</em><sub>2</sub>) between any two samples <em>s</em><sub>1</sub> and <em>s</em><sub>2</sub> is given by the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>P</em> is the set of all SNPs and g (<em>s</em>, <em>p</em>) is the genotype of sample <em>s</em> at position <em>p</em>. It is possible that either, or both, of <em>g</em> (<em>s</em><sub>1</sub>, <em>p</em>) or <em>g</em> (<em>s</em><sub>2</sub>, <em>p</em>) are unknown, in which case this SNP is ignored in the summation. To account for the fact that the distance between samples may therefore sum across a different number of samples, we include a normalizing term, 1/<em>n</em>, where <em>n</em> is the number of SNPs actually included in the summation. Distance measures other than the taxicab distance could be used, the most obvious alternative being the Euclidean distance. Using the Euclidean distance rather than the taxicab distance did not noticeably change performance (data not shown).</p><p id="p-10">The algorithm proceeds by picking the <em>k</em> nearest neighbors to <em>s<sub>i</sub></em> that have a known genotype at position <em>p</em> and then imputing the genotype, <em>g</em><sub>i</sub> (<em>p<sub>j</sub></em>, <em>s</em><sub>i</sub>), as a weighted modal average of these genotypes (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Schwender 2012</a>). That is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>N</em> is the set of <em>k</em> samples nearest <em>s<sub>i</sub></em> , which have a known genotype at SNP <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-3.gif"/></noscript></span></span> is an indicator function that takes the value 1 if <em>g</em> (<em>s</em>, <em>p<sub>j</sub></em>) = <em>a</em> and 0 otherwise.</p><p id="p-11">Standard <em>k</em>-nearest neighbor relies on the assumption that the most similar samples across the whole genome will be the best samples with which to impute any genotype. However, we reasoned that the best samples to use for imputation are those that share an evolutionary history at the SNP to be imputed, and that these samples may not be the most similar genome-wide. Genotype imputation methods like Beagle and fastPHASE use a similar reasoning: they use information from neighboring SNPs because these SNPs likely share a history with the SNP to be imputed due to physical linkage. Beagle and fastPHASE rely, however, on ordered markers and sufficiently dense genotype data to enable haplotype reconstruction. In the absence of a known marker order, we reasoned that the most informative samples for imputation may be those that are most similar at SNPs that are highly correlated with the SNP to be imputed. These highly correlated SNPs may be physically linked to the SNP of interest, but they may also be correlated without being physically linked. Regardless of the reason for the correlation, we reasoned that these SNPs would be most informative for choosing the nearest neighbors with which to perform imputation.</p><p id="p-12">We therefore introduce LD <em>k</em>-nearest neighbor imputation (LD-kNNi), where only the <em>l</em> SNPs most in LD with the SNP to be imputed are used to determine the nearest neighbor and the weightings to be used when imputing. The “LD” in our algorithm name, LD-kNNi, does not necessarily refer to physical linkage but rather to the correlation between any two SNPs in the data.</p><p id="p-13">Thus, <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> becomes:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2383/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <em>L</em>(<em>p<sub>i</sub></em>) is the set of <em>l</em> SNPs in strongest LD with the SNP to be imputed. The algorithm then continues as for kNNi with one minor exception. As we are now using a smaller number of SNPs to calculate distance, it is possible that there could be no genetic difference between a pair of samples. To avoid a distance of zero, which would result in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> being undefined, we add a constant to the standard taxicab distance. We set this constant to one as adding different values has little effect on imputation performance (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/021667SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FileS1.zip">File S1</a>).</p><p id="p-14">We implement LD-kNNi in our program LinkImpute (available from <a href="http://www.cultivatingdiversity.org/software">http://www.cultivatingdiversity.org/software</a> under the GPL version 3 license). LinkImpute allows parameter values (<em>k</em> for kNNi, <em>k</em> and <em>l</em> for LD-kNNi) to be either fixed or optimized. To optimize the parameter values, LinkImpute randomly samples 10,000 known genotypes. For each set of parameter values, each one of these genotypes is masked and then imputed to calculate accuracy. LinkImpute optimizes the parameters by assuming the parameter space is unimodal, bounding the search space and then searching the space for the optimal values.</p><p id="p-15">When comparing different imputation methods on the apple dataset, we used manually optimized parameter values (<em>k</em> for kNNi, <em>k</em> and <em>l</em> for LD-kNNi). For kNNi, <em>k</em> = 8 was chosen. For LD-kNNi, <em>k</em> = 5 and <em>l</em> = 20 (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS1.pdf">Figure S1</a>). When comparing imputation programs on different datasets, we allowed LinkImpute to choose optimal parameters.</p></div><div id="sec-3" class="subsection"><h3>Data</h3><p id="p-16">We collected GBS data from a collection of 1995 accessions from the genus <em>Malus</em> from the US Department of Agriculture apple germplasm repository in Geneva, NY. The samples were processed with two different restriction enzymes (ApeKI, <em>Pst</em>I/EcoT22I) in separate GBS libraries and were sequenced using Illumina Hi-Sequation 2000 technology. Genotypes were called using a custom GBS pipeline described in <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gardner <em>et al.</em> (2014)</a>. Briefly, 100-bp reads generated from both enzymes were aligned to the <em>Malus domestica</em> reference genome version 1.0 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Velasco <em>et al.</em> 2010</a>) using the default parameters in BWA (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li and Durbin 2009</a>). Genotypes were called using GATK (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">McKenna <em>et al.</em> 2010</a>) with a minimum of eight reads supporting each genotype. The final genotype matrix was filtered to contain only samples from the domesticated apple, <em>Malus domestica</em>, and ≤20% missing data per SNP and per sample. SNPs with a minor allele frequency (MAF) of &lt;0.01 were then discarded. Finally, the data were pruned to exclude clonal relationships: if two or more samples had IBD &gt;0.9, they were considered clones and the sample with the least amount of missing data from the group was retained. This resulted in a dataset of 711 samples and 8404 SNPs.</p><p id="p-17">To test the accuracy of our imputation method we created a “masked” dataset by setting 10,000 random genotypes to missing. This created “truth known” genotypes to which our imputed genotype calls were compared. We limited our testing to 10,000 masked genotypes, which represents 0.17% of the genotype matrix, in order to maintain a dataset with a reasonable amount of missing data while providing enough masked genotypes to be able to estimate imputation accuracy.</p><p id="p-18">Biased allele frequency in imputed data has been shown to affect downstream analyses (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2014</a>). To determine how well each imputation method estimates allele frequencies, we filtered the genotype matrix to contain no missing data. This resulted in a matrix containing 1001 SNPs from 459 samples (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS2.pdf">Figure S2</a>). We masked and then imputed 20% (91,952 genotypes) of the genotypes at random and compared the allele frequency estimates from the imputed data to the allele frequency estimates from the complete genotype matrix. As most imputation methods make use of other SNPs to aid imputation, we imputed using all 8404 SNPs in the dataset so as to provide more information to these methods. We then restrict our analysis to the 1001 complete SNPs.</p><p id="p-19">We also tested the performance of our method on genome-wide SNP data from maize and grape. The maize data were downloaded from the International Maize and Wheat Improvement Center (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hearne <em>et al.</em> 2014</a>). We reduced the data to biallelic SNPs with &lt;20% missing data and a MAF &gt;1% and then discarded samples with &gt;20% missing data. This resulted in 43,696 SNPs from 4300 samples.</p><p id="p-20">To generate the grape dataset we collected GBS data from a collection of diverse samples from the genus <em>Vitis</em> including commercial <em>Vitis vinifera</em> varieties, hybrids and wild accessions from the USDA grape germplasm collection. The samples were processed with two different restriction enzymes (<em>Hin</em>dIII/BfaI, <em>Hin</em>dIII/<em>Mse</em>I) and were sequenced using Illumina Hi-Sequation 2000 technology. We then used the 12X grape reference genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jaillon <em>et al.</em> 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adam-Blondon <em>et al.</em> 2011</a>) and the Tassel / BWA version 4 pipeline to generate a genotype matrix (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Li and Durbin 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Glaubitz <em>et al.</em> 2014</a>). Default parameters were used at each stage except for the SNP output stage where we filtered for biallelic SNPs. We then removed any genotypes with fewer than eight supporting reads using vcftools (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Danecek <em>et al.</em> 2011</a>). Using PLINK (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Purcell <em>et al.</em> 2007</a>), we removed SNPs with &gt;20% missing data before removing samples with &gt;20% missing data. We then removed SNPs with excess heterozygosity (failed a Hardy−Weinberg equilibrium test with a p-value &lt; 0.001) and finally SNPs with a MAF &lt; 0.01. This created a dataset of 8506 SNPs and 77 samples.</p></div><div id="sec-4" class="subsection"><h3>Other imputation methods</h3><p id="p-21">We compared LD-kNNi, as implemented in LinkImpute, with several other imputation methods and programs that do not require a reference panel:</p><div id="sec-5" class="subsection"><h4>Generic imputation methods:</h4><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-23"><em>Mode:</em> The modal value of all other samples’ genotypes at the SNP of interest. Implemented in LinkImpute.</p></li><li id="list-item-2"><p id="p-24">k <em>Nearest Neighbor:</em> As described above and implemented in LinkImpute.</p></li><li id="list-item-3"><p id="p-25"><em>MF:</em> As implemented in the R package MissForest (version 4.6-10) with maxiter set to 10 and ntree to 100 and all other parameters set to default values. MF could not be run on the entire genome at once because the run time was prohibitive. Instead it was run one chromosome at a time (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Stekhoven and Bühlmann 2012</a>).</p></li></ol></div><div id="sec-6" class="subsection"><h4>Genotype-specific methods:</h4><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-27"><span class="list-label">4. </span><em>Beagle:</em> Version r1230 using default settings (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Browning and Browning 2007</a>).</p></li><li id="list-item-5"><p id="p-28"><span class="list-label">5. </span><em>fastPHASE:</em> Version 1.4.0 using default settings except K (the number of clusters) is set to 200 (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Scheet and Stephens 2006</a>).</p></li></ul></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-29">The datasets used in this study are available in <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/021667SI.pdf">Supporting Information</a> <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FileS1.zip">File S1</a>. LinkImpute is available from <a href="http://www.cultivatingdiversity.org/software">http://www.cultivatingdiversity.org/software</a> under the GPL version 3 license.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Imputation accuracy</h3><p id="p-30">We first compared the accuracy of imputation by using the large apple dataset where we randomly masked 10,000 genotypes. We compared the imputed genotypes with the actual genotypes and calculated two measures of accuracy. Genotype error is the proportion of genotypes called incorrectly and allele error is the proportion of alleles called incorrectly. The methods rank similarly for both measures (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), although allele error is approximately half of genotype error. This is to be expected, because all methods that are likely to impute one allele correctly are unlikely to impute both alleles incorrectly.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Performance of the different imputation methods on the apple dataset</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Our results show that LD-kNNi performs slightly better than Beagle and fastPHASE, which have the greatest accuracy of all the other methods tested (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). MF performs noticeably worse than these methods, although this may, in part, be due to having imputed on a per chromosome basis. kNNi performs significantly worse than any of these methods, only slightly out-performing Mode imputation. We investigated the difference between LD-kNNi and kNNi further by computing, for each imputed genotype, the number of neighbors that are shared between the kNNi and LD-kNNi methods, using <em>k</em> = 5 in both cases. We found that in 56% of imputations the two methods share no neighbors (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This finding suggests that in many cases kNNi is imputed using samples that, although similar across the whole genome, may not be informative for the SNP we are imputing. <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS3.pdf">Figure S3</a> further supports this hypothesis by measuring the average distance, using the LD-kNNi methodology (<em>d<sub>l</sub></em>, <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>), from the sample to be imputed to the neighbors being used in the imputation. This shows that the average distance, again using <em>k</em> = 5 in both cases, is much greater in the case of kNNi (average distance: 6.4) than LD-kNNi (average distance: 1.8).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The number of shared neighbors between the k-nearest neighbors imputation (kNNi) and linkage disequilibrium k-nearest neighbors imputation (LD-kNNi) methods. The value of l was set to 5 for both methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930975817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The number of shared neighbors between the &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation (kNNi) and linkage disequilibrium &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation (LD-kNNi) methods. The value of &lt;em&gt;l&lt;/em&gt; was set to 5 for both methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2383/F1.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2383/F1.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466945" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">The number of shared neighbors between the <em>k</em>-nearest neighbors imputation (kNNi) and linkage disequilibrium <em>k</em>-nearest neighbors imputation (LD-kNNi) methods. The value of <em>l</em> was set to 5 for both methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Unsurprisingly, we found that the performance of LD-kNNi is dependent on the level of LD between the SNP to be imputed and the SNPs used to find the nearest neighbors. Where the average LD between the SNPs used and the imputed SNP is high, the imputation error is lower (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS4.pdf">Figure S4</a>). Although the apple reference genome is not used in LD-kNNi, we exploited it to investigate how often our nearest neighbor calculations used SNPs from chromosomes other than the chromosome on which the imputed SNP is located. To do this, we calculated the probability of being on the same chromosome as the imputed SNP for the 20 SNPs in greatest LD with the imputed SNP. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that for the SNP with the highest LD, there is a probability of 0.7 of being on the same chromosome and that this drops off to 0.31 for the 20th-ranked SNP.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The probability of a single-nucleotide polymorphism (SNP) being on the same chromosome as the imputed SNP as a function of linkage disequilibrium (LD) with the imputed SNP. SNPs are ranked according to LD, with the SNP most in LD with the imputed SNP ranked one." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930975817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The probability of a single-nucleotide polymorphism (SNP) being on the same chromosome as the imputed SNP as a function of linkage disequilibrium (LD) with the imputed SNP. SNPs are ranked according to LD, with the SNP most in LD with the imputed SNP ranked one.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2383/F2.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2383/F2.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467119" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">The probability of a single-nucleotide polymorphism (SNP) being on the same chromosome as the imputed SNP as a function of linkage disequilibrium (LD) with the imputed SNP. SNPs are ranked according to LD, with the SNP most in LD with the imputed SNP ranked one.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">We investigated the performance of the different imputation methods based on the MAF of the imputed SNPs. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the genotype error rate of the different methods stratified by MAF. While the error rate noticeably increased with MAF for Mode and kNNi, the increase is small for the other four methods.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy as a function of the minor allele frequency (MAF) of the imputed SNP for each of the six imputation methods. MAF is binned in 5% bins and the number of SNPs in each bin is shown in parentheses. kNNi, k-nearest neighbors imputation; LD-kNNi, linkage disequilibrium k-nearest neighbors imputation. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930975817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation accuracy as a function of the minor allele frequency (MAF) of the imputed SNP for each of the six imputation methods. MAF is binned in 5% bins and the number of SNPs in each bin is shown in parentheses. kNNi, &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation; LD-kNNi, linkage disequilibrium &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2383/F3.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2383/F3.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467031" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Imputation accuracy as a function of the minor allele frequency (MAF) of the imputed SNP for each of the six imputation methods. MAF is binned in 5% bins and the number of SNPs in each bin is shown in parentheses. kNNi, <em>k</em>-nearest neighbors imputation; LD-kNNi, linkage disequilibrium <em>k</em>-nearest neighbors imputation. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Run time</h3><p id="p-41">Comparing the run time of the various imputation methods (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), we note that both MF and fastPHASE took significantly longer than any of the other methods: these two methods take on the order of 10 hr compared with only a few minutes or less for the other methods. Further analysis of the run time suggests that, as both the number of samples and SNPs increases, LD-kNNi will continue to have a shorter run time than Beagle (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS7.pdf">Figure S7</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS8.pdf">Figure S8</a>).</p><p id="p-42">Comparing the performance of LinkImpute and Beagle on multiple datasets (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) we note that LinkImpute has a similar run-time to Beagle on all three datasets while achieving slightly better accuracy.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466951/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466951/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Performance of LinkImpute and Beagle on different datasets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Accuracy of allele frequency estimation</h3><p id="p-45"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows a bubble plot of actual and incorrectly imputed genotypes for each of the six imputation methods. This shows that all six methods have a bias toward imputing the major allele. This allele bias is pronounced for Mode and kNNi and is less severe for the other methods.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bubble plots of the actual and imputed genotypes for each of the 10,000 masked genotypes using each of the six imputation methods. Bubbles are not shown for the correctly imputed cases. The size of the bubbles is proportional to the frequency of observations in that category. kNNi, k-nearest neighbors imputation; LD-kNNi, linkage disequilibrium k-nearest neighbors imputation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930975817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bubble plots of the actual and imputed genotypes for each of the 10,000 masked genotypes using each of the six imputation methods. Bubbles are not shown for the correctly imputed cases. The size of the bubbles is proportional to the frequency of observations in that category. kNNi, &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation; LD-kNNi, linkage disequilibrium &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2383/F4.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2383/F4.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466976" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Bubble plots of the actual and imputed genotypes for each of the 10,000 masked genotypes using each of the six imputation methods. Bubbles are not shown for the correctly imputed cases. The size of the bubbles is proportional to the frequency of observations in that category. kNNi, <em>k</em>-nearest neighbors imputation; LD-kNNi, linkage disequilibrium <em>k</em>-nearest neighbors imputation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The allele bias observed in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> is expected to affect allele frequency estimation. We investigated this further by using our smaller dataset. For each of the six methods, we calculated the MAF across the 1001 SNPs without missing genotype data using both the observed genotypes and the imputed genotypes. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows that every imputation method biases the MAF downward. This finding is consistent with our observation of allele bias in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. The resulting bias is least pronounced for genotype specific methods, which all bias the MAF downward by 0.5% as opposed to a minimum of 0.6% for any of the other methods.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Minor allele frequency (MAF) computed by the use of actual and imputed genotypes for each of the six imputation methods. kNNi, k-nearest neighbors imputation; LD-kNNi, linkage disequilibrium k-nearest neighbors imputation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930975817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Minor allele frequency (MAF) computed by the use of actual and imputed genotypes for each of the six imputation methods. kNNi, &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation; LD-kNNi, linkage disequilibrium &lt;em&gt;k&lt;/em&gt;-nearest neighbors imputation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2383/F5.medium.gif" width="266" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2383/F5.medium.gif" width="266" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2383/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467071" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">Minor allele frequency (MAF) computed by the use of actual and imputed genotypes for each of the six imputation methods. kNNi, <em>k</em>-nearest neighbors imputation; LD-kNNi, linkage disequilibrium <em>k</em>-nearest neighbors imputation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49"><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> shows the tendency for the MAF to be underestimated when calculated using an imputed dataset no matter what imputation method is used. In addition, LD-kNNi outperforms every other method in estimating MAF: the points cluster much closer to the line for LD-kNNi than for any of the other methods. Moreover, LD-kNNi’s most extreme deviation (3.8%) from the observed MAF is lower than any of the other tested methods (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS5.pdf">Figure S5</a>). The two groups in the Mode plot are caused by the two different modal values for SNPs (0 or 1). The bottom left group is where the modal value is 0, the top right group is where it is 1 (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS6.pdf">Figure S6</a>).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-50">LD-kNNi performs well compared with the most commonly used imputation methods. On our apple dataset it results in both superior imputation accuracy (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and more accurate allele frequency estimates (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Accuracy results on the two other tested datasets are similar, and the results presented here suggest that performance should be comparable on other similar datasets. In particular, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> suggests that the MAF distribution should have little effect on the relative performance of LD-kNNi.</p><p id="p-51">The run time of LinkImpute also compares favorably with existing methods. Only two of the methods studied here have both high accuracy and reasonable run times, namely Beagle and the LD-kNNi option of LinkImpute. Of these, our method is slightly faster. In addition, as the number of samples and SNPs increases, LD-kNNi is expected to outperform the other methods (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS7.pdf">Figure S7</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS8.pdf">Figure S8</a>), which is particularly noteworthy because increasing sample size is critical to augmenting the statistical power of GWA studies (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Spencer <em>et al.</em> 2009</a>).</p><p id="p-52">A recently developed imputation algorithm was designed for heterozygous species without a reference genome and was applied to raspberry (genus <em>Rubus</em>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ward <em>et al.</em> 2013</a>). However, this method applies only to biparental populations and relies on the construction of a genetic map. The primary advantage of LD-kNNi over existing methods is that it does not rely on ordered markers and can be applied to diverse and heterozygous populations (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS9.pdf">Figure S9</a>), not just biparental crosses. Although we called SNPs using the apple reference genome, LD-kNNi makes no use of this information during imputation. Indeed <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> shows that in many cases our algorithm is using information from SNPs that are not on the same chromosome as the imputed SNP. It is worth noting that linkage group assignments from apple F1 populations conflict with reference genome locations for 14–18% of SNPs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Antanaviciute <em>et al.</em> 2012</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gardner <em>et al.</em> 2014</a>). It is therefore likely that a significant number of sequences are anchored incorrectly in the version of the apple genome used here. Thus, the values in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> may be upward biased. Nevertheless, LD-kNNi clearly often makes use of information from SNPs on other chromosomes and the quality of the apple reference genome has no effect on its performance.</p><p id="p-53">We demonstrated that the performance of LD-kNNi improves as the LD between the imputed SNPs and the SNPs used to find the nearest neighbors increases (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS4.pdf">Figure S4</a>). This suggests that, as the SNP density of a dataset increases and more SNPs are in LD with one another, one can expect improvements in the imputation accuracy of LD-kNNi. One way of obtaining more SNPs would be to allow greater levels of missing genotypes, although the increase in missing data are likely to have a negative effect on imputation accuracy. Whether this negative effect is offset by the positive effect of increased SNP density is an area that warrants further study.</p><p id="p-54">Like most other imputation methods, LinkImpute is applied to a table of genotypes that have been called by a genotype calling algorithm. In many cases, a genotype without sufficient sequence coverage is set to missing in the table even though it has several supporting sequence reads from the original data source. In such cases, the information from those reads is lost and remains unused during imputation. By including the information from these reads during imputation, we are likely to improve imputation performance. In turn, this should enable greater confidence genotype calls from lower read depths thereby significantly increasing the total number of genotypes called. Moreover, incorporating imputation and SNP calling in this manner should help improve genotyping error rates, especially in cases of low read depth. This is an active area of research and future improvements are expected to increase both genotype quality and quantity.</p><p id="p-55">Our results suggest that LD-kNNi produces more accurate allele frequency estimates at the cost of a slight decrease in imputation accuracy. Biased allele frequencies are known to adversely affect downstream analyses (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2014</a>), whereas increased imputation accuracy does not always lead to improved phenotype prediction (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Rutkoski <em>et al.</em> 2013</a>). For many studies, an imputation method with less bias in allele frequency estimation, such as LD-kNNi, may therefore be preferable to a method with slightly increased accuracy. It is worth noting that, in cases where one is only interested in the MAF, one can simply estimate it from the nonmissing genotypes. We show that such an estimate is indeed unbiased and that it is more accurate than estimating MAF after imputation (<a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS5.pdf">Figure S5</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/15/g3.115.021667.DC1/FigureS10.pdf">Figure S10</a>). The relationship between imputation accuracy, allele frequency bias and their effects on downstream analyses warrants further investigation.</p><p id="p-56">Genotype imputation is a crucial step in many genomic studies as all existing genotyping methods result in some missing data. Most imputation algorithms rely on physical or genetic maps, either directly or in the generation of ordered SNPs, and are not suitable for use in non-model organisms with poor or underdeveloped genomic resources. Our novel genotype imputation method, LD-kNNi, does not rely on physical or genetic maps and imputes genotypes as accurately as the best existing methods that require ordered markers. In addition, it is fast and outperforms other methods in its ability to accurately estimate allele frequencies. Thus, LinkImpute is a valuable tool for improving genome-wide analyses in nonmodel organisms, especially for GWA and GS in highly diverse and heterozygous organisms.</p></div><div class="section" id="sec-13"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Patrick J Brown and Kate Crosby for providing useful discussion. This work was supported by a Genome Canada Bioinformatics and Computational Biology grant; the Canada Research Chairs program; and the National Sciences and Engineering Research Council of Canada.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-58">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021667/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021667/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-59"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-15"><span class="received-label">Received </span>June 15, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-08"><span class="accepted-label">Accepted </span>September 8, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Money <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2383.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adam-Blondon</span>  <span class="cit-name-given-names">A.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezzulli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequence initiatives.</span> <span class="cit-source">Genet. Genomics Breed. Grapes</span> <span class="cit-fpage">211</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.2" data-doi="10.1126/science.1156409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic mapping in human disease.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D322%26rft.issue%253D5903%26rft.spage%253D881%26rft.epage%253D888%26rft.atitle%253DGenetic%2BMapping%2Bin%2BHuman%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1156409%26rft_id%253Dinfo%253Apmid%252F18988837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMy84ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.3" data-doi="10.1186/1471-2164-13-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antanaviciute</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Fernández</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAntanaviciute%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D203%26rft.epage%253D203%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bof%2Ban%2Bapple%2Brootstock%2Bprogeny%2Busing%2Bthe%2BMalus%2BInfinium%2Bwhole%2Bgenome%2Bgenotyping%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-203%26rft_id%253Dinfo%253Apmid%252F22631220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22631220&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.4" data-doi="10.1126/science.1167936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Bovine HapMap Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">R. A.</span>  <span class="cit-name-given-names">Gibbs</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eversole</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BBovine%2BHapMap%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D324%26rft.issue%253D5926%26rft.spage%253D528%26rft.epage%253D532%26rft.atitle%253DGenome-Wide%2BSurvey%2Bof%2BSNP%2BVariation%2BUncovers%2Bthe%2BGenetic%2BStructure%2Bof%2BCattle%2BBreeds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1167936%26rft_id%253Dinfo%253Apmid%252F19390050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNi81MjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.5" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.6" data-doi="10.1371/journal.pone.0031745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e31745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChagne%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De31745%26rft.epage%253De31745%26rft.atitle%253DGenome-wide%2BSNP%2Bdetection%252C%2Bvalidation%252C%2Band%2Bdevelopment%2Bof%2Ban%2B8K%2BSNP%2Barray%2Bfor%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0031745%26rft_id%253Dinfo%253Apmid%252F22363718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0031745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363718&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.7" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.8" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2383.9" data-doi="10.1534/g3.114.011023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and cost-effective genetic mapping in apple using next-generation sequencing</span>. <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.011023%26rft_id%253Dinfo%253Apmid%252F25031181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.10" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.11" data-doi="10.1093/molbev/mst229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterizing bias in population genetic inferences from low-coverage sequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst229%26rft_id%253Dinfo%253Apmid%252F24288159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzMvNzIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.12" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2383.13"><div class="cit-metadata unstructured">Hearne, S., C. Chen, E. Buckler, and S. Mitchell, 2014  Unimputed GbS derived SNPs for maize landrace accessions represented in the SeeD-maize GWAS panel. Available at: <a href="http://data.cimmyt.org/dvn/dv/seedsofdiscoverydvn/faces/study/StudyPage.xhtml;jsessionid=6dede0d2bfdf0cb29ddb610981cc?globalId=hdl:11529/10034&amp;tab=files&amp;studyListingIndex=0_6dede0d2bfdf0cb29ddb610981cc">http://data.cimmyt.org/dvn/dv/seedsofdiscoverydvn/faces/study/StudyPage.xhtml;jsessionid=6dede0d2bfdf0cb29ddb610981cc?globalId=hdl:11529/10034&amp;tab=files&amp;studyListingIndex=0_6dede0d2bfdf0cb29ddb610981cc</a>. Accessed September 25, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.11.2383.14" data-doi="10.1038/nature09298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap 3 Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D. M.</span>  <span class="cit-name-given-names">Altshuler</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peltonen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrating common and rare genetic variation in diverse human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%2BM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D52%26rft.epage%253D58%26rft.atitle%253DIntegrating%2Bcommon%2Band%2Brare%2Bgenetic%2Bvariation%2Bin%2Bdiverse%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09298%26rft_id%253Dinfo%253Apmid%252F20811451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20811451&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281461200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.15" data-doi="10.1038/nature06148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Policriti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clepet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D449%26rft.issue%253D7161%26rft.spage%253D463%26rft.epage%253D467%26rft.atitle%253DThe%2Bgrapevine%2Bgenome%2Bsequence%2Bsuggests%2Bancestral%2Bhexaploidization%2Bin%2Bmajor%2Bangiosperm%2Bphyla.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06148%26rft_id%253Dinfo%253Apmid%252F17721507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17721507&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249724800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.16" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.17" data-doi="10.1146/annurev.genom.9.081307.164242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genotype imputation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D387%26rft.epage%253D406%26rft.atitle%253DGenotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164242%26rft_id%253Dinfo%253Apmid%252F19715440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.18" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.19" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.20" data-doi="10.3732/ajb.1400143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics: a potential panacea for the perennial problem.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1780</span>–<span class="cit-lpage">1790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400143%26rft_id%253Dinfo%253Apmid%252F25326620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjExOiIxMDEvMTAvMTc4MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.21" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.22" data-doi="10.1371/journal.pone.0008219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurwitz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid genomic characterization of the genus vitis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e8219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMyles%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De8219%26rft.epage%253De8219%26rft.atitle%253DRapid%2Bgenomic%2Bcharacterization%2Bof%2Bthe%2Bgenus%2Bvitis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008219%26rft_id%253Dinfo%253Apmid%252F20084295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20084295&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.23" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.24" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2383.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of unordered markers and the impact on genomic selection accuracy.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.26" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.27" data-doi="10.1080/15287394.2012.674910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwender</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Imputing missing genotypes with weighted k nearest neighbors.</span> <abbr class="cit-jnl-abbrev">J. Toxicol. Environ. Health A</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Btoxicology%2Band%2Benvironmental%2Bhealth.%2BPart%2BA%26rft.stitle%253DJ%2BToxicol%2BEnviron%2BHealth%2BA%26rft.aulast%253DSchwender%26rft.auinit1%253DH.%26rft.volume%253D75%26rft.issue%253D8-10%26rft.spage%253D438%26rft.epage%253D446%26rft.atitle%253DImputing%2Bmissing%2Bgenotypes%2Bwith%2Bweighted%2Bk%2Bnearest%2Bneighbors.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15287394.2012.674910%26rft_id%253Dinfo%253Apmid%252F22686303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15287394.2012.674910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22686303&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.28" data-doi="10.1371/journal.pgen.1000477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C. C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Designing genome-wide association studies: sample size, power, imputation, and the choice of genotyping Chip.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSpencer%26rft.auinit1%253DC.%2BC.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000477%26rft.epage%253De1000477%26rft.atitle%253DDesigning%2Bgenome-wide%2Bassociation%2Bstudies%253A%2Bsample%2Bsize%252C%2Bpower%252C%2Bimputation%252C%2Band%2Bthe%2Bchoice%2Bof%2Bgenotyping%2Bchip.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000477%26rft_id%253Dinfo%253Apmid%252F19492015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19492015&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.29" data-doi="10.1093/bioinformatics/btr597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MissForest—non-parametric missing value imputation for mixed-type data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr597%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC8xLzExMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.30" data-doi="10.1534/genetics.110.120907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Progress and promise of genome-wide association studies for human complex trait.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120907%26rft_id%253Dinfo%253Apmid%252F21115973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzIvMzY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.31" data-doi="10.1093/bioinformatics/17.6.520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Missing value estimation methods for DNA microarrays.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTroyanskaya%26rft.auinit1%253DO.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D525%26rft.atitle%253DMissing%2Bvalue%2Bestimation%2Bmethods%2Bfor%2BDNA%2Bmicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.6.520%26rft_id%253Dinfo%253Apmid%252F11395428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNy82LzUyMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.32" data-doi="10.1038/ng.654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhingra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome of the domesticated apple (Malus × domestica Borkh.).</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVelasco%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D839%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bdomesticated%2Bapple%2B%2528Malus%2B%25C3%2597%2Bdomestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.654%26rft_id%253Dinfo%253Apmid%252F20802477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20802477&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282276600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.33" data-doi="10.1186/1471-2164-14-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangoo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Fernández</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-2%26rft_id%253Dinfo%253Apmid%252F23324311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324311&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2383.34" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2383.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2383.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2375" title="Post-transcriptional Mechanisms Contribute Little to Phenotypic Variation in Snake Venoms" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2391" title="Genome-Wide Association Study of Ureide Concentration in Diverse Maturity Group IV Soybean [Glycine max (L.) Merr.] Accessions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2383" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vlg2WnRPXonMMn9IcQuVk4kuluUm-Jq3i2oVTQQVFxw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2383" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2383" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2383" class="active" data-icon-position="" data-hide-link-title="0">LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460179" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-J3PIJ9cAs3JA9j1m3hUO-FHZJr6tEtv9ZkuAnYG-aNk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2383.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2383" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DQyO9wmU5ppHk4L2p8abVFXZkCSGLvt2NbN7v2YI6dM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2383" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2383.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460179" id="node460179--2" data-pisa="ggg;5/11/2383" data-pisa-master="ggg;g3.115.021667" data-apath="/ggg/5/11/2383.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Money</span>, <span class="highwire-citation-author" data-delta="1">Kyle Gardner</span>, <span class="highwire-citation-author" data-delta="2">Zoë Migicovsky</span>, <span class="highwire-citation-author" data-delta="3">Heidi Schwaninger</span>, <span class="highwire-citation-author" data-delta="4">Gan-Yuan Zhong</span> and <span class="highwire-citation-author" data-delta="5">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2383-2390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021667 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460179--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Money </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Money%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BMoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:daniel.money@dal.ca" class="" data-icon-position="" data-hide-link-title="0">daniel.money@dal.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zoë Migicovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zo%C3%AB%2BMigicovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Migicovsky%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZo%25C3%25AB%2BMigicovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Heidi Schwaninger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heidi%2BSchwaninger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schwaninger%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeidi%2BSchwaninger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gan-Yuan Zhong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gan-Yuan%2BZhong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhong%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGan-Yuan%2BZhong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2383" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2383&amp;text=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2383&amp;t=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2383&amp;title=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460179/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460179" id="node460179--3" data-pisa="ggg;5/11/2383" data-pisa-master="ggg;g3.115.021667" data-apath="/ggg/5/11/2383.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Money</span>, <span class="highwire-citation-author" data-delta="1">Kyle Gardner</span>, <span class="highwire-citation-author" data-delta="2">Zoë Migicovsky</span>, <span class="highwire-citation-author" data-delta="3">Heidi Schwaninger</span>, <span class="highwire-citation-author" data-delta="4">Gan-Yuan Zhong</span> and <span class="highwire-citation-author" data-delta="5">Sean Myles</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2383-2390;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021667 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460179--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Money </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Money%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BMoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:daniel.money@dal.ca" class="" data-icon-position="" data-hide-link-title="0">daniel.money@dal.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle Gardner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BGardner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gardner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BGardner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zoë Migicovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zo%C3%AB%2BMigicovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Migicovsky%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZo%25C3%25AB%2BMigicovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Heidi Schwaninger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heidi%2BSchwaninger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schwaninger%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeidi%2BSchwaninger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gan-Yuan Zhong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Plant Genetic Resources Unit, Geneva, New York 14456</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gan-Yuan%2BZhong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhong%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGan-Yuan%2BZhong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean Myles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Animal Sciences, Faculty of Agriculture, Dalhousie University, Truro, Nova Scotia, B2N 5E3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BMyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BMyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2383&amp;rft.epage=2390&amp;rft.atitle=LinkImpute%3A%20Fast%20and%20Accurate%20Genotype%20Imputation%20for%20Nonmodel%20Organisms&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Money%2C+Daniel&amp;rft.au=Gardner%2C+Kyle&amp;rft.au=Migicovsky%2C+Zo%C3%AB&amp;rft.au=Schwaninger%2C+Heidi&amp;rft.au=Zhong%2C+Gan-Yuan&amp;rft.au=Myles%2C+Sean"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460179/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460179/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460179/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460179/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460179/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460179/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460179/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460179/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460179/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460179/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460179/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460179/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2383" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2383.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2383.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2383.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2383.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
