<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2329" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2329.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2329.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2329.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2329.atom" />
<meta name="HW.pisa" content="ggg;5/11/2329" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" />
<meta name="DC.Identifier" content="10.1534/g3.115.021592" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Crow et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies (GWAS) have been successfully applied in humans for the study of many complex phenotypes. However, identification of the genetic determinants of hearing in adults has been hampered, in part, by the relative inability to control for environmental factors that might affect hearing throughout the lifetime, as well as a large degree of phenotypic heterogeneity. These and other factors have limited the number of large-scale studies performed in humans that have identified candidate genes that contribute to the etiology of this complex trait. To address these limitations, we performed a GWAS analysis using a set of inbred mouse strains from the Hybrid Mouse Diversity Panel. Among 99 strains characterized, we observed approximately two-fold to five-fold variation in hearing at six different frequencies, which are differentiated biologically from each other by the location in the cochlea where each frequency is registered. Among all frequencies tested, we identified a total of nine significant loci, several of which contained promising candidate genes for follow-up study. Taken together, our results indicate the existence of both genes that affect global cochlear function, as well as anatomical- and frequency-specific genes, and further demonstrate the complex nature of mammalian hearing variation." />
<meta name="DC.Contributor" content="Amanda L. Crow" />
<meta name="DC.Contributor" content="Jeffrey Ohmen" />
<meta name="DC.Contributor" content="Juemei Wang" />
<meta name="DC.Contributor" content="Joel Lavinsky" />
<meta name="DC.Contributor" content="Jaana Hartiala" />
<meta name="DC.Contributor" content="Qingzhong Li" />
<meta name="DC.Contributor" content="Xin Li" />
<meta name="DC.Contributor" content="Pezhman Salehide" />
<meta name="DC.Contributor" content="Eleazar Eskin" />
<meta name="DC.Contributor" content="Calvin Pan" />
<meta name="DC.Contributor" content="Aldons J. Lusis" />
<meta name="DC.Contributor" content="Hooman Allayee" />
<meta name="DC.Contributor" content="Rick A. Friedman" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies (GWAS) have been successfully applied in humans for the study of many complex phenotypes. However, identification of the genetic determinants of hearing in adults has been hampered, in part, by the relative inability to control for environmental factors that might affect hearing throughout the lifetime, as well as a large degree of phenotypic heterogeneity. These and other factors have limited the number of large-scale studies performed in humans that have identified candidate genes that contribute to the etiology of this complex trait. To address these limitations, we performed a GWAS analysis using a set of inbred mouse strains from the Hybrid Mouse Diversity Panel. Among 99 strains characterized, we observed approximately two-fold to five-fold variation in hearing at six different frequencies, which are differentiated biologically from each other by the location in the cochlea where each frequency is registered. Among all frequencies tested, we identified a total of nine significant loci, several of which contained promising candidate genes for follow-up study. Taken together, our results indicate the existence of both genes that affect global cochlear function, as well as anatomical- and frequency-specific genes, and further demonstrate the complex nature of mammalian hearing variation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2329" />
<meta name="citation_id" content="5/11/2329" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2329" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2329.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2329.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2329.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021592" />
<meta name="citation_pmid" content="26342000" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2329" />
<meta name="citation_lastpage" content="2339" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amanda L. Crow" />
<meta name="citation_author_institution" content="Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Jeffrey Ohmen" />
<meta name="citation_author_institution" content="House Ear Institute, Los Angeles, California 90057" />
<meta name="citation_author" content="Juemei Wang" />
<meta name="citation_author_institution" content="Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Joel Lavinsky" />
<meta name="citation_author_institution" content="Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Jaana Hartiala" />
<meta name="citation_author_institution" content="Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Qingzhong Li" />
<meta name="citation_author_institution" content="Department of Otolaryngology, Head and Neck Surgery, Eye &amp; ENT Hospital of Fudan University, Shanghai 200031, China" />
<meta name="citation_author" content="Xin Li" />
<meta name="citation_author_institution" content="Clinical Laboratory Department, First Affiliated Hospital of Nanchang University, Nanchang, Jiangxi Province 330006, China" />
<meta name="citation_author" content="Pezhman Salehide" />
<meta name="citation_author_institution" content="Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Eleazar Eskin" />
<meta name="citation_author_institution" content="Department of Computer Science and Inter-Departmental Program in Bioinformatics, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Calvin Pan" />
<meta name="citation_author_institution" content="Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095" />
<meta name="citation_author" content="Aldons J. Lusis" />
<meta name="citation_author_institution" content="Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095" />
<meta name="citation_author" content="Hooman Allayee" />
<meta name="citation_author_institution" content="Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author" content="Rick A. Friedman" />
<meta name="citation_author_institution" content="Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033" />
<meta name="citation_author_email" content="rick.friedman@med.usc.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Bennett;citation_author=C. R. Farber;citation_author=L. Orozco;citation_author=H. M. Kang;citation_author=A. Ghazalpour;citation_title=A high-resolution association mapping panel for the dissection of complex traits in mice.;citation_volume=20;citation_year=2010;citation_pmid=20054062;citation_doi=10.1101/gr.099234.109" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=C. Bonnet;citation_author=M. Louha;citation_author=N. Loundon;citation_author=N. Michalski;citation_author=E. Verpy;citation_title=Biallelic nonsense mutations in the otogelin-like gene (OTOGL) in a child affected by mild to moderate hearing impairment.;citation_pages=537-540;citation_volume=527;citation_year=2013;citation_pmid=23850727;citation_doi=10.1016/j.gene.2013.06.044" />
<meta name="citation_reference" content="citation_journal_title=The Neuroscientist;citation_journal_abbrev=Neuroscientist;citation_author=R. L. Davis;citation_title=Gradients of Neurotrophins, Ion Channels, and Tuning in the Cochlea;citation_pages=311-316;citation_volume=9;citation_year=2003;citation_issue=5;citation_pmid=14580116;citation_doi=10.1177/1073858403251986" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet. 23: 110–115.;citation_title=Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment.;citation_author=E. Fransen;citation_author=S. Bonneux;citation_author=J. J. Corneveaux;citation_author=I. Schrauwen;citation_author=F. Di Berardino;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=R. A. Friedman;citation_author=L. Van Laer;citation_author=M. J. Huentelman;citation_author=S. S. Sheth;citation_author=E. Van Eyken;citation_title=GRM7 variants confer susceptibility to age-related hearing impairment.;citation_volume=18;citation_year=2009;citation_pmid=19047183;citation_doi=10.1093/hmg/ddn402" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Ghazalpour;citation_author=C. D. Rau;citation_author=C. R. Farber;citation_author=B. J. Bennett;citation_author=L. D. Orozco;citation_title=Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.;citation_pages=680-692;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892838;citation_doi=10.1007/s00335-012-9411-5" />
<meta name="citation_reference" content="citation_journal_title=European archives of oto-rhino-laryngology :  official journal of the European Federation of Oto-Rhino-Laryngological Societies (EUFOS) : affiliated with the German Society for Oto-Rhino-Laryngology - Head and Neck Surgery;citation_journal_abbrev=Eur Arch Otorhinolaryngol;citation_author=Q. Huang;citation_author=J. Tang;citation_title=Age-related hearing loss or presbycusis.;citation_pages=1179-1191;citation_volume=267;citation_year=2010;citation_issue=8;citation_pmid=20464410;citation_doi=10.1007/s00405-010-1270-7" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=K. Ishibashi;citation_author=M. Suzuki;citation_author=S. Sasaki;citation_author=M. Imai;citation_title=Identification of a new multigene four-transmembrane family (MS4A) related to CD20, HTm4 and beta subunit of the high-affinity IgE receptor.;citation_pages=87-93;citation_volume=264;citation_year=2001;citation_issue=1;citation_pmid=11245982;citation_doi=10.1016/S0378-1119(00)00598-9" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=K. R. Johnson;citation_author=C. Longo-Guess;citation_author=L. H. Gagnon;citation_author=H. Yu;citation_author=Q. Y. Zheng;citation_title=A locus on distal chromosome 11 (ah18) and its interaction Cdh23 ahl underlie the early, age-related hearing loss of DBA/2J mice.;citation_pages=219-225;citation_volume=94;citation_year=2008;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Lindblad-Toh;citation_author=E. Winchester;citation_author=M. J. Daly;citation_author=D. G. Wang;citation_author=J. N. Hirschhorn;citation_title=Large-scale discovery and genotyping of single-nucleotide polymorphisms in the mouse.;citation_pages=381-386;citation_volume=24;citation_year=2000;citation_issue=4;citation_pmid=10742102;citation_doi=10.1038/74215" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. N. Mahajan;citation_author=B. S. Mitchell;citation_title=Role of human Pso4 in mammalian DNA repair and association with terminal deoxynucleotidyl transferase.;citation_volume=100;citation_year=2003;citation_pmid=12960389;citation_doi=10.1073/pnas.1631060100" />
<meta name="citation_reference" content="citation_journal_title=Histochemistry and cell biology;citation_journal_abbrev=Histochem Cell Biol;citation_author=P. Mamczur;citation_author=A. J. Sok;citation_author=A. Rzechonek;citation_author=D. Rakus;citation_title=Cell cycle-dependent expression and subcellular localization of fructose 1,6-bisphosphatase.;citation_pages=121-136;citation_volume=137;citation_year=2012;citation_issue=1;citation_pmid=22057438;citation_doi=10.1007/s00418-011-0884-1" />
<meta name="citation_reference" content="citation_journal_title=Hearing research;citation_journal_abbrev=Hear Res;citation_author=Z. F. Mann;citation_author=M. W. Kelley;citation_title=Development of tonotopy in the auditory periphery.;citation_pages=2-15;citation_volume=276;citation_year=2011;citation_issue=1-2;citation_pmid=21276841;citation_doi=10.1016/j.heares.2011.01.011" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=R. Mott;citation_author=J. Flint;citation_title=Dissecting quantitative traits in mice.;citation_pages=421-439;citation_volume=14;citation_year=2013;citation_pmid=23834320;citation_doi=10.1146/annurev-genom-091212-153419" />
<meta name="citation_reference" content="citation_journal_title=J. Assoc. Res. Otolaryngol;citation_author=J. Ohmen;citation_author=E. Y. Kang;citation_author=X. Li;citation_author=J. W. Joo;citation_author=F. Hormozdiari;citation_title=Genomic-wide association study for age-related hearing loss (AHL) in the mouse: a meta-analysis.;citation_pages=335-352;citation_volume=15;citation_year=2014;citation_issue=3;citation_pmid=24570207;citation_doi=10.1007/s10162-014-0443-2" />
<meta name="citation_reference" content="citation_journal_title=Acta Otolaryngol. Suppl.;citation_author=R. J. Ruben;citation_title=Development of the inner ear of the mouse: a radioautographic study of terminal mitoses.;citation_pages=221-244;citation_volume=220;citation_year=1967;citation_pmid=6067797" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. J. Son;citation_author=L. Wu;citation_author=H. Yoon;citation_author=S. Kim;citation_author=J. Y. Choi;citation_title=Developmental gene expression profiling along the tonotopic axis of the mouse cochlea.;citation_pages=e40735-e40735;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22808246;citation_doi=10.1371/journal.pone.0040735" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Welter;citation_author=J. MacArthur;citation_author=J. Morales;citation_author=T. Burdett;citation_author=P. Hall;citation_title=The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.;citation_volume=42;citation_year=2014;citation_issue=Database issue;citation_pmid=24316577;citation_doi=10.1093/nar/gkt1229" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Wiltshire;citation_author=M. T. Pletcher;citation_author=S. Batalov;citation_author=S. W. Barnes;citation_author=L. M. Tarantino;citation_title=Genome-wide single-nucleotide polymorphism analysis defines haplotype patterns in mouse.;citation_volume=100;citation_year=2003;citation_pmid=12612341;citation_doi=10.1073/pnas.0130101100" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Yalcin;citation_author=J. Fullerton;citation_author=S. Miller;citation_author=D. A. Keays;citation_author=S. Brady;citation_title=Unexpected complexity in the haplotypes of commonly used inbred strains of laboratory mice.;citation_volume=101;citation_year=2004;citation_pmid=15210992;citation_doi=10.1073/pnas.0401189101" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. A. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. O. Yariz;citation_author=D. Duman;citation_author=C. Z. Seco;citation_author=J. Dallman;citation_author=M. Huang;citation_title=Mutations in OTOGL, Encoding the Inner Ear Protein Otogelin-like, Cause Moderate Sensorineural Hearing Loss.;citation_pages=872-882;citation_volume=91;citation_year=2012;citation_issue=5;citation_pmid=23122586;citation_doi=10.1016/j.ajhg.2012.09.011" />
<meta name="citation_reference" content="citation_journal_title=Hearing research;citation_journal_abbrev=Hear Res;citation_author=Q. Y. Zheng;citation_author=K. R. Johnson;citation_author=L. C. Erway;citation_title=Assessment of hearing in 80 inbred strains of mice by ABR threshold analyses.;citation_pages=94-107;citation_volume=130;citation_year=1999;citation_issue=1–2;citation_pmid=10320101;citation_doi=10.1016/S0378-5955(99)00003-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies (GWAS) have been successfully applied in humans for the study of many complex phenotypes. However, identification of the genetic determinants of hearing in adults has been hampered, in part, by the relative inability to control for environmental factors that might affect hearing throughout the lifetime, as well as a large degree of phenotypic heterogeneity. These and other factors have limited the number of large-scale studies performed in humans that have identified candidate genes that contribute to the etiology of this complex trait. To address these limitations, we performed a GWAS analysis using a set of inbred mouse strains from the Hybrid Mouse Diversity Panel. Among 99 strains characterized, we observed approximately two-fold to five-fold variation in hearing at six different frequencies, which are differentiated biologically from each other by the location in the cochlea where each frequency is registered. Among all frequencies tested, we identified a total of nine significant loci, several of which contained promising candidate genes for follow-up study. Taken together, our results indicate the existence of both genes that affect global cochlear function, as well as anatomical- and frequency-specific genes, and further demonstrate the complex nature of mammalian hearing variation." />
<meta name="og-title" property="og:title" content="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2329" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies (GWAS) have been successfully applied in humans for the study of many complex phenotypes. However, identification of the genetic determinants of hearing in adults has been hampered, in part, by the relative inability to control for environmental factors that might affect hearing throughout the lifetime, as well as a large degree of phenotypic heterogeneity. These and other factors have limited the number of large-scale studies performed in humans that have identified candidate genes that contribute to the etiology of this complex trait. To address these limitations, we performed a GWAS analysis using a set of inbred mouse strains from the Hybrid Mouse Diversity Panel. Among 99 strains characterized, we observed approximately two-fold to five-fold variation in hearing at six different frequencies, which are differentiated biologically from each other by the location in the cochlea where each frequency is registered. Among all frequencies tested, we identified a total of nine significant loci, several of which contained promising candidate genes for follow-up study. Taken together, our results indicate the existence of both genes that affect global cochlear function, as well as anatomical- and frequency-specific genes, and further demonstrate the complex nature of mammalian hearing variation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"h2cwo0xhz2bAl-n2Gvf1DS4_OqjB89-lHw0Zyct6ItY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460172","apath":"\/ggg\/5\/11\/2329.atom","pisa":"ggg;5\/11\/2329","ac":{"\/ggg\/5\/11\/2329.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2329.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2329"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2329"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2329.figures-only":true,"\/content\/5\/11\/2329":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zQANs+qvkSzy7cZs+R3fyCsXxffS2z3h6+0qCVuLfO1WtyjpedyMJDu379TeM1p0uyP6GvHDcu0cT18GPzHxw7RtcOaZ5DLl\\\\\\\/2wE4E8v7bAPZF+w8IyPuenvQCJ6vocl\\\\\\\/8MkBJ\\\\\\\/CHjYqcW0D9NZy50aQRYe8jX4epCdgxrD8YwLBzhXtIgKFZpSNF10U+pVBbLQJLeEy+0eniDn0F3gyBDvlGl64ZFHVQeUBm0ITED5JH4QM03TrXjW8O9vc6GXzvjQgsEJdmQDncKXyKb626Id+PLMcNBV5oLkEor8FXAZuTGHINoEzwAyJ\\\\\\\/iWEsR72aZ7g4YixLOijD4yA3Xfa4c+c6ZNrLBvBr5mXE8YD5LaDzsK9Z\\\\\\\/dwJ4EDVyfx28JK7hL+V7TIRHt\\\\\\\/R2d2P+eblo0nF6OC3hzkMu4BK0RvRhrmGGmF1cyHby\\\\\\\/AniQC8jePKWw6YdOFxNYfhrYZWIRmg9IkzNPHczyRIO9YI0wmdX41yxc5NFHyXuTxOeN6IFCQ6ugD\\\\\\\/wsUePmHBFDE3713P3THJEh2gd0cbmy59PRhxM7VMlBaHi7lZeulgjuRGRuWu+TbGo6ZESMf4HACwZhNdkJr2vNxrBRNam+yQG1QroC\\\\\\\/X5qJoi9V53eaAXHeQMv+2iPGfGYvvy\\\\\\\/UgheprOBGWye0zy7M2afC9bMg3v2nAdplDY4ZE0sj4V14DkuxYncoNcreU7EfFlxsAbLqESp0L2iNdeoCjXHJhN8UDxhCMgXRTPYcdjGmY\\\\\\\/5TBnZAO3mLcdHoDB5YfvcBIpJ04mRLEPICeujIpKGVzHRJaK9DOqbP6P6BHe4\\\\\\\/MWpjwLxdBqg9hmPbyr+VVMOQo7gbpTvbnAvD+5MDU9UpMeS7eGN7G3QNHlIqbobH70+6IX8Za5\\\\\\\/bdpXdiIvJTDBqhlJK1x1vh0Y4x0jH3UiAH1J+xV9QkH1odktQPfRD6GMzQ2ogV4PvBo1xLlPuE82vYrZcrutgR1QucZHtthUP8S7LEbnepUnZ+ewFU1JXb6HH2h2AWLIMtwGuJKWEBpHRSYsLvAmnvEvgJuWACg1m2E0O1u0pm32bv\\\\\\\/mfxe1KJ6zrXx6\\\\\\\/MFMIDb2WJt\\\\\\\/vHa5C9RuzzJGGLS0nqHRcAH2D8zGS+Qh18bZXeQ+lcVLcjk+bTBoS3B7nHRYpf4C+yHt6Cz4xVPLEUDdr4c7QUbvFLxrbIwdrjf1a6F+hFB6vlXqDIcK0aEEXA3jHyDpyzs5IyUx\\\\\\\/M1eQVMgBBajx49b9OaCti6qHikE1TUE8rPUBbJkUti5BYeYZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022H2cqVdR2psw5bhrpdh07jg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228fac25e4b07e2525017d6df191e22cf8b1cfdbd48ad45805babbd58d44019cc1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zFEPkZlni2nLn0SD5t+z4Lq2iDRpTMy9AecQDix1lUhqnsEEGu+QpV4lIHdI0hZCEWRw5lGj2JrKCyqaGYeaYrglQrmtj32Pyb3K+Adsn8B+U84\\\\\\\/R2VRp\\\\\\\/b2uZzOzH7otCaixKgr4P2R6gB1bAyFfY3bp13N13YhyG5nvFWCwSO+isjI3T+I8krbBrsKh1Jx4l\\\\\\\/QPKo41mLnSCYW1E7zhiV5NZHPLNGRbTEq1BS7iNqJx1Pz5PKIiOMSDUSHQKTApkvHm20Ucfr53ZneA3vp7WHXGoCPtIPCECfChlStmBmRuga6bz89TWFPq55VOy5GpHbvGtbe0UjEfD1u24pDsGof6gutCByhuDW\\\\\\\/e5\\\\\\\/bt8gcHXycTiZOBb1fUV3Ga7MkFfK4fnG\\\\\\\/zdTnc07WspbW0+qUP0pZabqZmn2vKNsBikZ+uyHjmvbddhNM7c+oMrhvrChezNOxyS3+HtFUdx0OC9hN7wnC0s5aS3IETha1l46dsvbGS5eHkcv\\\\\\\/GuGJY6BpXq98TLESGXS2IZY6PbtwcOQPdGhxOE2JgBrTIt9px9uGPFP0ZN8JUQs4S4R16RHiUddC5hlZg+XiMzi18yVdk54\\\\\\\/eWiI5Y8A+QARCv8SbLM21E6G2bY\\\\\\\/8djeFZazOfw9IZsEI0dIz5WOVlPsP3N4Et7U7TcF1nbpmipkhAgjXp6zgmWVyOqPVtQbdVHtHhuVBtBrp\\\\\\\/xiMY7S6z0tCy3nY82EYRSu5uckINiLJI4FK4JmNPdUNVODjuz8PMEEDEyNf3cpLWzvdkgZYDIUt+CMEjLVQcLGn+zxw\\\\\\\/JQrm2NoAgAjTHocLT6eqeKTmejRhVfYQCElROYLxjonfOcuPTJgloAHNhDXEyLmuiVhMPntBWE+0LFeXSNlNxc87H3jE67MXj+afDLiH0e\\\\\\\/iin2XQSumwi7bsE0\\\\\\\/kAyOac\\\\\\\/o1O4uaQ3pbkOjmW8aekAk93yFCaDDSJG9+3cIvHMmvj41pdcaXZnwf\\\\\\\/Z\\\\\\\/tf7LOJuowhbABygQa+\\\\\\\/RBD3V+JJyeam5OE4Z1tGGtuxeFXiSkgR4hB\\\\\\\/hjnwkpqwk9jOCp1gGPBx23L+\\\\\\\/NFpkogSEsAFHd3TGmUFipuvr+AXu2dm336Ux6dfn818BTGq0rKt3h23DoQJHLq0VVgrVcU4pOuYQLT2renEfhXsONGREKUwmgECt0P6B\\\\\\\/XhIDtlWeL9a0aa2MiY1GLr+6Qrfsaf9HIbaoehBwWUudsoZ5uP59IXb0fJHXzcpYSIr2LxGST\\\\\\\/e7uwaz\\\\\\\/NaJMx2yQOLQSEndIrwqevZvWTnaqw+8vfpwvexHG\\\\\\\/0Iux9cqHL+N\\\\\\\/oSmK1BcnBTQM83fpa5P2XSoDYlPJVqRUuvKh493XzGqSJqqD\\\\\\\/238nRVHxP+\\\\\\\/Ty7JGWf1t8pDeTAQLVOGsdwFp2Ujng5AVqp7hRQw+tGssKZUGhme3YH+VulQK1sfl4UL4hG5HMowY\\\\\\\/zCY65ca\\\\\\\/Mbd+1mu5xnn4ZkIzsucenmktT+9PnMdizQ6lb10UpPp1\\\\\\\/zC+Rc476CjHlM+16w8vHs3S21eLXoEAPz2m8hlPeztT8eV6s2gYLMdvPiIkQrSUAhfmHF0CWRBuJ2OyytvBbfxApnzsOAjW7Mi12TED1n7+OX6ylDcukHYGIIIYHPvEb4AakrFycc\\\\\\\/q2w8TpC8o3rp6tFHLLas5\\\\\\\/Zorvdh1JujZvTgstMeh3iWfyVe0rHvN9F9K7dsCRhcGTojpyJL20N3iZJkMF4neWTUMTHZZ1K8Yu3BZP3TEIkV8yZdAmDb1hTQCC\\\\\\\/Ziw81N6EqC9LcKpe3Zf0872aqeCFxY0aj6bnIZpXBLeDj9rkDcCEp7SM1QYVKwxUHYxmlOUmd+hVWe0SoieEpYk0lBWCAlofO38b58XuceLfbYCTH+Wk30PM4r2CM5AMs4jBsvoeG4m3X2GGoZzn2DRc2+MUbDH2JrGmYEHh\\\\\\\/cJ+3V+KdwxTxnvWThNDlKRtACiOq0HAHqRmCfLggi1r1shaO\\\\\\\/2uc6cVkMQaWGwguY43JsVC2G2f0q\\\\\\\/r\\\\\\\/SiYgV3Bl0Pvjr+m3n9kOEiBB2KDtLLr2yVzEEBV7LjtoKmpT3BMtz9M6uyozW4W1kQCg6mBxxI4dT7FZ0LlmMpA5wr1DuygYEmo1mLtnvcaziFTRSrUxVkK5KgjwfqfdEqU0DSllknjJJuRuuSuyuuvrp7cHmDbqGioaqcvtvobopeDoxfLNnuFgmYeNxH9VxifoLoK6SQOherjpljpRqKb29UGI+GmznHj5kbz\\\\\\\/s+yiB0E7N7qpgKmV\\\\\\\/z+RIOcOe+N\\\\\\\/PtWqcSc8TfS9BNFmrfjiAWg29\\\\\\\/epqG4mp2HEMDhdsKD2QLiYz6R0JOsYxB6VByTT2TfoNjxoAhm8eN3clzZ8Xq6MmxdxYS5FSiNfB5\\\\\\\/lng46gZ9PLiGwdV6ll+3isJDp0VzLoPCrtXp6g+iRVj6sytGIuTwo\\\\\\\/oohsIRxLyqPLN3JhtPF0ByJwuiGd3Ti9SB3R4dIpiUEy5af2XBYwIH77+RBb7L3pwohQ4k4ckf1n4pCEfvVM\\\\\\\/542NWXhGm5ztrVZ6oX+ZlRs49Ksm4SZp+G6vwPdfN+QcYmSE4m3Lg2Q7irilbyH8ylL+Ve5p96rdozgf92gucnR++TVDrDcqL8JB4OCc+tp06s5QJWFjsnjhN77AEkRF1sNF\\\\\\\/NGKrkvguMGlLHflnwrWTKhhvCiTAbvuL9xW7CeJU6BxCoPHDjJLOSIdK7OM8HdiqNTWjSwopeq9huO5P4RKkXOjxLob\\\\\\\/2E\\\\\\\/m+tKaiIdn3GF78jLZ3yWlgZm3Zv6F3jJ7dYvvYZALR3zyBEsm3K4QT5VWR82Wsfz8fhvvFjaDep3L2e3oyVxGkWF7k2uHjKiAnmLBRTEqDs6HqYvzE52Ljkoe+NxYxNI3EHLuUS4ZBMWpTGY5rKGQl2buKcCfe4k\\\\\\\/UpIb6DXuBTammNCxNRm496DiUP03UELowGrfDA6TWYT3GmW+uRIqigtG\\\\\\\/6WoPQBd6UyP0kOSUhf\\\\\\\/npW74EoMuPiUQbcLtb8+Bgx5h8cr2ZlvB7mPA87lhFSvdM+73xtYWelQGjVZ0smltPQM59gck9pFGSePPybAhPaF\\\\\\\/yAyquPGXxn9wh8hfHkdTjtHOMCwtKdOmbemMNnf328xS5E2Ga2hLcqW5JfV1O5zE5KEcA0zG\\\\\\\/tD7CrzG2MBiJol0ZJGe7XOp5lofNlIKGAKhCo9lOxCaHyhfrHSyiJ9wHtzqSb3GD2xoECnh+PcZQTWOrUJNm2Nj\\\\\\\/E8VO+T5A4d3zNzw2XrwEw6Pjtczc5tsO7cisnswSpQJJuImBfami3YM9yMcDrpxdz\\\\\\\/U4oFvzUUx1J147jFHeL8oDi\\\\\\\/dGpzaXodvTRudMmH8DCqD63dWhMZQ8X+8r3XaO5JDfhWm5m9p22ftHA\\\\\\\/9q5I0ntlLZjW2SGcNQhkZ5vpMFJQbXBZlkgVWSx50k5dxYp5fP1Lx8Bk5LDvts8j6h+PlqbWC1REA4UXU8gBwWyuLJ5ja1gysV9ZYImWOwlEKTTaRopQuLTh6G4XqwodVzRJWnp1g2eYZiljhh1u4XTI9+4xx2cS+3Wgy4k0YPp1\\\\\\\/p8OmuBwgVDQYKFEhGun5SqYZtbjujhG9H2Sox2pBVr0Kx1VXF+GvQ6UeLAze0vjnP92glHIef7s5ox3A8XD7rojEkSS61E0BOwgnyCxO\\\\\\\/VzKRGs3H3G56m6kirf6GTswb7PntXjue\\\\\\\/yetmDgY5w7rzqaHxHxqCJpW+ehlh\\\\\\\/+0TlA3T0xIhE0mY\\\\\\\/\\\\\\\/iEjpNg8GTl5IW0qBJJDPREijjSKRm2SxQfVyxEz9vkEKnxliMKcJxR\\\\\\\/iPu5g8Cqp5dY2+z9ODHqKUAidDbqdnuh46cMYs0cTkh58IBQWVFyuNudTruZe\\\\\\\/C6CIka\\\\\\\/yHhm75mhH+4JJmUGhqil8VH2ZA8ZagYoBVgnan7j11z5BmDN95hFyATvcY\\\\\\\/05wshCp1nQtqG4HA692UeNArNfEuoMlSLCjbh37N0q18AkxrpLXj4pfuaBUV9j5p\\\\\\\/RN4gAlvb130NLES62hCViC4W3edqOK9CMKAuW5IdKobSXX0v6XnFQsxdrFUvGCNP2FlL\\\\\\\/5k7HKaevb5mMrIOZO0FNGk3vvbykpLAovBA7uuBKYlBikACmSvO5\\\\\\\/GfFPWrc1MGRtCXjH4b4sWyqNbIDtWMpbnvFpAVbKS5no1mIiafW6G3fu2KCLV2hBGaKv+Y4oOmGhtvwglG2cWhBOQLEPfhVL0Jnr\\\\\\\/dfdwZIZlKOTykADe819QT2\\\\\\\/dmnS66sfBS1iVpuJyDTelPs1eVsuxl9E9qL6uuQHp0vlgKT5pWuKsfkUNl6diBye54qpamDpS4Hr13gW6UGiFtp3CUXekhmi7xiW8YSjZPg3F3n3F3rXMdr1Rsby09V2rUd6xaNnRRcEDBmGY+BFAAusZ4HiNYEJDqSe3bCIqTc6dLxWbBo6AQbKrMK8VRK1OoeoKy\\\\\\\/h65jsABNnBcWW6MzrjpTqbiefOkm4vFXiK8mp1OFxbxsIwR8ZAmM6tC9Ry8oPaOihbrLilEzgbCTVC3vslDVKn5MdzTM5IsqpCmBaFt2\\\\\\\/DSGqyCU0aGw1qPycqgWHRWqT67jK9xu9lJJPvU6RP8YCkT+YkFz7V2+4s9hsFbYSz42wATFIrRua5gdgqVud4raD2xilwXzsRzjc4gm9ATHo5t725K97gJmgnvu9ATJwdHnAm3uJSzxiw3OHNQXk03SBFKR\\\\\\\/AYso6YgbiCgCalTxp0UF4OU8UWRzaZ2nuMjQjD+6i1ZF9+w4fA4OQy+nWigMRkQAcq2mA\\\\\\\/MVjfbxBy9GHTcvRmi7GEBpJawdSxiswcKJ7jfRtuk\\\\\\\/mMJF0RhRm7kPaqejnbb+jJTXBSiXfIQgxhpfEPdJUpUmqQnpn+D8E7\\\\\\\/ZfOfH7aMMCykPolM3LHWUPBl+QODbM9FI8c\\\\\\\/8C8x3BKPATEiDPVJ8ozzBAUI2Xo52TY3XGy7vyLkKwpqSPtTP99d49aH6cukW6k5Jy7tzTdyaN0cnOrP5g+8rR26\\\\\\\/FO3\\\\\\\/8PnYM9XekfdcNXjySoXcXMkFep72avHRG1\\\\\\\/Z3pjz0MgPcu3e5pF7LNYmFzy3QLOPMazNNTIFFtW1TF9qEX\\\\\\\/BOXJkFyYIERwSUVRZFy3lSpXyQxwGBaRwtGzmKZaraF5JzP3NdaI8Uo1F0rN99jersn5eUhupo2bF1LIFBtOSZp2vKGmtWOPV0yVUBdQTsNvqd08PK8jg+vACEZi+uUGD9dz+j63dfT1M8k2izdyqj2utKbCfExleOsGtJqVW0VfdjYfOCKdeizHrbv5VV7d4a5NIO9NeS6LAZhx4eK\\\\\\\/lE1B0SVHYU9rUCakAxOKPaoeAi997IGb2QZ5569lOp4Ezkb9mpzYh\\\\\\\/3gRIwc+oWbieJRHaJSBALKf71BjnjawMnSb8REJOwN+s\\\\\\\/UAaCBbUXCs9Yktpj30f4dVtreTVkmKYHRdfBBoe\\\\\\\/jityTSzcpgCiUDawOAZc6kvfIUKQddnJ2ntfOGS6oUUIrgD9dpSDjWd0dvyh38bExwFZSRX8ai0CDaNQnrxeOOYgAy+Vffue31UthmLF7rzH+AsDolas1fbTe7RsfiQTOPYo11QGgXlhgcl7\\\\\\\/+fO6kSVleRw6cv9jeAfGK2oSlowfBM9M3uPnaD3moQx7n3\\\\\\\/teKg4PR0jMwudAujKucnuQ9zPSosGDfU62ebqwc7LQlYIJcOBQOXuYr22W83hmzJdR9H2\\\\\\\/KIgRSK7fQ80qraAw9X33bnxBVoEgPJ9bdkEnmS7lpA5aGVubPmv\\\\\\\/mQYkGZXItH6XzxfjHjRoXx9EdZ5wavvyzGCcsSWhyjh3SYRfGoz7+2DkoV0eNhigSulgbyFzVpRtXzRtsXBELr3cSQJvs4x9+QQ5IQrDah0usqmEvrWUH\\\\\\\/L5dz6I8P8FsECx+AQraH++1U0n0rV9Q27Y4lyZZSNYWIKVC2ug2rFHXitHIG0o7wW\\\\\\\/0cnxTchXfk9RBr\\\\\\\/L3JIiXRTzpGBhGFgZcBvNYvYstPGoM5UEE7VdDgcxeeS426QIyDnE19oTWxUjCG8NsX\\\\\\\/r3331FlUQXZpuTZA21lcQT\\\\\\\/DiZesQkrs\\\\\\\/2UJNYnsFxNIjAYTdz2z1qiCiy3Rx6Mi\\\\\\\/VCMv9mzm9gbxuRPY5NPqYRRKuSWY9yuVgQVH22L+vRF1tNv2Lx\\\\\\\/PyeVlxSl64Yk4LYgBnHJ5+trM9q4H5UeMRQUrCbkkWIxDGNQ6HN70kATDU\\\\\\\/Lscmjo1+NCdTRdDKZbFeumvIFjAw4nn85e4HZyLuYfexfCJCJyvOHa4AW7thRxby3+toAcTl+U5xj71GsWgMfRVM\\\\\\\/n\\\\\\\/I7tpaBIqZN7BAxF4rAjSqZv+LmyLrPLxmXNJkh76NmcgNjFUKMrweJjdlBBg4ZCnXdrUJS014vuRvpW\\\\\\\/s\\\\\\\/5qQCl0+Hw5DjSOc1IXS6dln9ZtUSBkPb8bere1Rj+eek7WjlKc839eLYqs6OWKzM0gQpWXVj6ZbEUljlTL0NThG1w4fTKf76FUsyEXexGEGpVn6DgdqOfWxXByvAcLuIdTbCfxqRWjQsHG6cdOGQEpWme2GI10vfOPrLyJaAXeYitgAHFm\\\\\\\/MuAbZwO8HLEaPs6q94UvcXWeugVl6ctEaLDtkbcqpU9VkrAYfGwtSMsKjY1TEUdDjkcM\\\\\\\/GaB7NCMu5pe41GxLFiIxJhYvfoQrlQvrcojG6JkXXalAzHIjcwscSVe82bclCtJxj0RaDbM86lj9xmUtAKrBDwUoxu788zm8\\\\\\\/j4ev6dHOuKYnB7eF4rIexeUPFKVklwiw2PNm6i5Ml2WSw67baRo6rd7KGRuSBHC7bqYGUUYIy23ewP1ytuWsrykZdXwp9O2q6sMkK049ZkUBssG0urOZoplOwu0n0iqZZhWWHxRL+kWgU3mduAYb6wHHhpYJbz80FKBEEEzNHpq+iQTRvYcuR\\\\\\\/adPRKUREC4OsDsJQ7\\\\\\\/6aEEf3xbwTT7FO9ftrcJnu2eWlOz03VKFnyt+LU9MmYbKT4exVF+Sd+N60rdhPZGk1FVEaiPfl7rJFOEb8k23JWxY4l1V3pw79BMyglztf3N6HErNpH3EgNJiebTfm4etUuFTKbpUcOJyUgTYUkYw3kPC4oWUh+RLSop5OnbomLYrGIqpA1HR7Gj1vRLO3ywzRlEAaURupiQN2XpSpkeV8gfs+3cB+wF0iML+OTxM\\\\\\\/i8H5HCJNhiRVJyvoM5n0xXYTV2rx+WMp2gKUieKgiTdoXhxv3Acx09VIiZ6EUHDKTr6xu0dJ0zjpxyPYJRoXr8PCqDFc0qK9zKW6JIjvjLUwFp87LAVEYRq5gd0NQ2pEqOsnJq8mDoOWLLEl1zoXlA7AFXuseBf+dC4d9nJ\\\\\\\/cqg1ZdBGtljY0AZTKt94cbwasQvRt8eXyTzMgloo1jYW8XZOO5AbGY5LX2xjRht9xgVErpisHX\\\\\\\/zG8eF013d81hmSHmIgh8Q+kwRLxYG5Lq1pbbpwMkNwcq66pPoW5bfeSDtR1Aeg5chXvUNW9fAxZ8Axht1AbASuyQnVSCIjE3v661JaJsul2ZM3wRaU6mVYgjOqEn+pDMP7ELH\\\\\\\/ZyIpv5Q4\\\\\\\/XSuI7IgSkIITe5WNKyOmETJBEU\\\\\\\/5aB+mvjgwtOTL365J+VVAy9S82TNWijesGBacbgdUYIWeYCWlcU8aOWm\\\\\\\/gkqOtjUKKiNLwcJHvtUS8p70lDPZ4eRJBtKBr5qBhvffSzn6l6T3ll8y7cST6Cpbcv2NzVGrsMVS45Xj6cKbUd1cEkfe6dG\\\\\\\/2WbKUjJvViVArLzezqHZReuEs0YWwZNoCowJ9Di2aimWtkyMy7rGCSRP3SZBfNBexl+CREcVkYK5kCWSA6fAmuCy+4rM\\\\\\\/rsknoA3Eb8AhaD7qoeD5+mh2UezuIgdiIKgwPFxL2QpTDKVxmFU5LX\\\\\\\/cwzHS8r69q4zaCtov2w6GGrr\\\\\\\/DcPjZfBAq7t3JLJW5jFrAH740GlGPJ\\\\\\\/4xTIvuGEW9QgpFaRTX\\\\\\\/cCTVneHlu8Ov1MJITKj7a3dLfayktdnvOk+7mfD6JIbzlL7dqYhypciC4q\\\\\\\/o+Ah6DodHAqoxdtder31vKLvGhqjkyeFubC2JEs\\\\\\\/VKiUtaptHz10nSPHMrOQHLf\\\\\\\/0t8ZRPQq3CDmGZRI6qxp5S287WvFf815g8\\\\\\\/ALV2bkYbkyFSeqMs0U5AJEiJjzjn+FqorB2an8LZfLS0cLi7HbBx6+8X43B2k0OCS6NOZD7GK0ZC2LEAuDXtxuzlJavLXu4LLVxZcBu\\\\\\\/YhBu9JCTJsw8VzWce1A\\\\\\\/+UeTspwtnK+eOckDcPVaH9EKjhMwqN95i2ns\\\\\\\/JwWKQ8OoDt\\\\\\\/nqx1Y43uDR\\\\\\\/o68rUIH6bx+yZaT1g2TM503tsD\\\\\\\/Hj5XbT2GZzQ2QVONVwxuRc9\\\\\\\/wiotJ\\\\\\\/n1dmWQB2Kk66yBwaBWB9VLnu\\\\\\\/SuSt95pumHrLOD\\\\\\\/CvyuOXRozuSn\\\\\\\/znwStvlbGyHiX8D5BugFjn30hmXZPFb2nVNm40TRQmYfOGa+iylxcYRYgF7uPFZw86K9ls2FB6rGUzXIHMT8TKQdZAfhg7kLXu03IUBrekHrchQS\\\\\\\/yrDXxPZG8LqAMfYGwzTAaDc\\\\\\\/9cqqF2H\\\\\\\/hk8XGBN3q+EJdJV41ksRP66U021dT1K7JGtXcit\\\\\\\/4oX93U3tbGiH7lntwBSo6D54xB4nWNEHO3MEH6zpV21zigpiJQesXcnYiLmAOR+qwtAxy9VeJXJ0THNn5sJQlZr6hjLDMZ5sMdhK6G94XKZ41Y+yn54j+CNyvu7gJ22V9h\\\\\\\/gRz6NQrx4iMWiGziILNd1h0rI2yLJJHTvoYN6AiPTCREcmtqmpBu43YPnoYI8B+yx\\\\\\\/3IdXHi16oggWHLC+A+RAenzpcg5gocUPwz9CrI26yRSQiQWKe5S\\\\\\\/qrqKDzg1fafdHlTwiSpg6p4DmYR5k5yfrwyAg702M90c31vWZL5QlBNyCGg44yusBURxuQ7D2XmzECs+KceS9fbOtRjMFrU0FY90HORqZEtzyk+NW5TLK111rSUPVwcK0flymudkJuXv5NbyhUAK5idCbf71vGdDsNaV6BdiTarQpvwGe625YPRArYP\\\\\\\/8qkpgF916lOp2zV9MfjTv7YTKIQ6R774nmOc7FOZQzRnoQkyqkQ6wc0j5Ld7aoeQ3+s3nZrf2OpDMCCUiTF1gzsK8ZkfEmKaw0I5J7lpuZk8ZaM5hQyMUNJupiluobpPq0zo\\\\\\\/bnKn2UepHI6EhALNr+4X1trBVz5jOas3UKz9QUxOVw5gJocHohUMheCH9i\\\\\\\/0sA36V5LPKfwljsUxynSejd1myJHYPOYa8142XrhTgMFjA7sKHCDmM\\\\\\\/F3yL65MVXIowgE5BykI0liLOOyirxj4B6zEINlpuj\\\\\\\/ug6rlPuMHAHjcStkiCTQ70646EW3d7vfNRXz5WPMsHubGqT1CnQUtNLbVdqi5d7jE+rlPCr7xYT4UpLf0GjUH6lycqwlPovHgeX\\\\\\\/MLtJqNeQoK3HEyZhlj98b9P+aszRLiM9lXQSZLBtTnkMleky7n4U7nuTSZO+vTogyqnQt0HKnCmwgWzUqotvemnn1JOkQ339NwWy6tLEh3ps5C0BW0q+9xHhLkrS+pbYl6Fq1h14lIuehdju+qi8HKLqHxJkSvn1YohzC65mrlImsmYifXYpM2u1wp8L9ADS1OJEc5sbc0l\\\\\\\/X3HS18MonQd3kXhc1MN9Uq+r++89a55wxuGBSZQndKBqd2vrnS5xoEou2X1uRVBHDl9A0+XGwWeDyrBQq17NCnw5qWePfBmwmsiKLVfF1INap1mmVpnm13\\\\\\\/pjKTcIyzj90q6m+KBZK0U5LniUfn38GMA+Q8c6anL26aBVKPjEzS+KTZaMYn\\\\\\\/e5+byEBoF8pCFiT14phV2DE+J0xTe4+gTc\\\\\\\/V4exezGKkJu8J+VcJZ4NvaU1LHY\\\\\\\/HNgDo7v\\\\\\\/0Xo3IWU4xiSIPX+jquDsZYnF8oV2GOcNJGSfeonzGL2vbQ1pvFIDkXB2vVZTtv\\\\\\\/Jv\\\\\\\/R+LuRERwBUaY+YkYeyWVZ\\\\\\\/GjZLUoCsHJdwdKbla9sUeOGNFccJXA2D\\\\\\\/UNifg5ayPXSXTQilWzaI4lFGIYQFvno2Jl4VTh+V2RpCh\\\\\\\/hKGGgZ7q\\\\\\\/fKR5OwKKl\\\\\\\/M7wYDfWUrYijc7+Iy94rL08kMW0q9k74E66k0QTXeSSF3Fe7bvvJ9mLJyWuneuk6\\\\\\\/AUgql0ViIVF6j1t5q0eHwQJZrpDY9UIoxEj56PhbQNfWeIzk4khWfie2RIXa2Vf6z+Ou2406Ub7Yy1WAjgp+suPj6iUBhO1TZuTdcRPyrWVlmNdQNbsQ1WIEfTVZPf7R\\\\\\\/\\\\\\\/gu7fG5nFrqWFO7njD9b2keyH87AR1gX5RkH5M9X89pbf\\\\\\\/dNEqqruLlQCBmJy3q+z5ch47sMEN02UT99LeBaSaJuNFZ3SEHkNP4U9Keao+I54HTaJFQfNWLMmMFjodMw7GllpIN2aiqqhQqv3ItFXF7MpqaBzpT3eL9c\\\\\\\/AoZd53Pqx+SjhbYXXPsp0LIo04ap+g7Qwp\\\\\\\/FMyXnD4If\\\\\\\/W21LlfytgUy221z2D6ZDqW684SrXspsDaTV2dG1qv+8sb7zB\\\\\\\/jmSpkuyZcUNdWJ4mJTfhnIcVQVK\\\\\\\/0ShspYXmv8M34z0Fydo6ANU3L6eiDLt73IA63eyNHxqyJ9qe5REkrtj24OwJ0MoZ7rIdLNhX6EgdMwdBd+8R+nONE+mA34SfjngW+tJeX1\\\\\\\/bn+AIgBtN2+Y9NjjGwWICuIEv3NPHBTFYYisnXoall3MfRhMyYjmGouIwwRcD2OxoqaPOBSQbcaAp5UOPWZZqtzvtvy+IIX6RvVKDUFWHEesYHvKmh8RBdWozq7zZMN7V4A8mv3mdpI0rwmARImUGMlP3Kt6XM0aSE+XWPfGtrPS9Dfzr6dsDLCpsndslPyN4FYtVnSzhchaIUojJFqe1KazFJ0tU3m4np01Xre320BnzGWGwqmPHWv1SL1ie+9h8L4ADavy7siFcQgoNi6hgs54Ad35n26UulgQqVYa3aMtsBILGArGWgys3c82+L09SvWqS30cnJuTuZ\\\\\\\/oxdTT8wUt3KgZY8Bj44raDStHZhxPwYHT4q1MddvRRqKGke\\\\\\\/yxAlxCdH\\\\\\\/TBbNOpLZgbyIhb\\\\\\\/H1iHKXr4vQ3e61OlMrnRNCC+HvucK8djTfSWfz+gai+w+PGLNY06cexdj4z+V+8J4jKkeygAbYomrtQHAe5e4j08yazAguKOlEd0Py82B2EQaNnJa7kKkgZpJhxz\\\\\\\/txMJCyHwhm7bI0pu1DZLMRbLucLgz7ESviaGbXPd1ZJptnCZQi\\\\\\\/hRmUlWGFRYWogSFabY8myfrT6LW5q9gVtT3TFxGJvNAhAxFVKZ26GYCD39+0Agkqp5lEt8qQ5xdHbYR\\\\\\\/3Y5NEuWVj7H12fXQflrPWwj9Ri5Ul+bFyWMRafBqiYhal\\\\\\\/Z32DGjoezuba6vVrURbjgvUi8RNwtiLWawlcOicEtWn6gJ6\\\\\\\/ZgXW3GwpADUSVM6C6bYj86Jp7qdxMV1+W2RrGD3oDOMg99DuRUBgJYeMh113zo8vL2Qa309izPmea9b7l59GvY4VrGkYG4GO5YJsSNflTymPNJmkOi4B7r7FOqkHR8QcDYso2WbOPOcQ8\\\\\\\/V10qSTpdCaUSEogZTxYGCdYIUyP2xCMILlGmN9QQ6CNRBuBHdSTlwO0bB\\\\\\\/YAR7gRA3dDQ6EXE8OvLSPj2IEd2TOhySCr1RvxqsG9QmxUmKlCiB\\\\\\\/vmee8jhaTx+wanhE8QlMs4aq9h4zYuR5TC8+KWuiefnRDbErA7HCp2jwc4ChH872K360\\\\\\\/kq\\\\\\\/jEUdcWv\\\\\\\/k8DfS9k\\\\\\\/HlUu2Li3lUfIAabJEBHxQblni1vzruMi6SYNh9WpB\\\\\\\/KLd+cIAPoZXxWOVv9es8gl+IAvNTxOGGncd6XSBPRsRhw7olUJHTmpCLJnbtUCPUGcshA4\\\\\\\/Kf70GUEGWabRY4yoa3Qv5xsl7VsWGTdQO+1RNzA0Al60atZm3ugHNvbwICDNcQGyKCOCQxG2Kclv6GX2EkzhGgS4KpU9UoTyk3K74Otkc14EEKZig09X8I5s7fpUgb1ir7aoW2w1dS4NvFxClCl+Ls0xuFiKqimQa6B7hF3TJKKj5mz+STvNmOLTGxlYr6o6aG2\\\\\\\/NMqBppG4+b4Frxm\\\\\\\/JWGJfwT+PixPdtzdfiXToemJyWjBcQyC8kS9gjk0XSq7utYtTGKcn4Nmwuu8tdEcg4PtuuslmRL3105UvLqg93emYajLQ\\\\\\\/hwcPHMArt2GfFYvzo\\\\\\\/QvJuJGqS+bPP8yGh55qcInaJh0HwK+OOEvJBWY1gIaMb83XTNyCwqz0wzSDRcXoNPOWqigqN0jzHU1InHTO8bImvI8EMSuJuoXp53x+bjvBoZhikp1sdpKvFlEl1DedrQ462sdECy0RxaHKyo\\\\\\\/H\\\\\\\/U1FrVXOZCfukPe7a+soAAe03NdkZ85r6gy2bZRH6d\\\\\\\/ojv++weaCfkIUwv18fVIl19ujYGY1bMvICl12P8Q1VzFMMKkH8NA8yp\\\\\\\/9TW0TFlO8cfFFNyQ8uPGvqMCXgVGim76eZWcmdfY5y\\\\\\\/crvxrCevrkTDjX6XcznOqruVNyv0LOpsbo8TlZ62APda\\\\\\\/ZPbSlmRQRqTRldliKuk4alCma8PUG0obXPIwXaez7SDedmj5o51LX0XjAbaIW60JHlQTW+1zJHKISK6t1TyT9VJxDCMwm9SrHF\\\\\\\/ei5vZprdnqkoCxQUEhxpIS3hyIIOggCzZfPc6ysBpo2\\\\\\\/26bvYxyk+w+\\\\\\\/cdHwjbGAh9aZ5fLguK2Pz+G6MDg9T7ktmiORYSt8UmIz0w\\\\\\\/PCB7q5q+q0ii87KnCIa6PfjRsx7Lv0qB5O7tdr+++JjRV0uJXUSZe9Lch9nRqBfWwuHYcLsofWzMCzLuK7tNEs8esJxBFglPWiTIpcOxMaElxN9pixZxd4VR\\\\\\\/G5EtOimtAdcSW29keToIBLCAG0E7wQ36gHdGpA6xy05XPQuLmmhkBCHpiVAUi+Im1ZX2+bT7qSWOFiSasAIJx7H1rAWuKMvrL9FkaRE9ps7vW11UvLSkS\\\\\\\/ixcl1gPS94+nhVDTlHS01XRaa3gYE\\\\\\\/HCGfFXvuuAiWb9Gipxva05okGLtmO\\\\\\\/Acl\\\\\\\/tJ6AUjy9klThA8x3CgOldRFhw7l+J0cYukTiHjMbB\\\\\\\/\\\\\\\/89ZlK2eK76o4GA0zDl3wrcoAo3mnoYOCZa7kpmszFkG\\\\\\\/lOE+49sZ\\\\\\\/wS4fqv4h1DAIq4ZhUALYYnwCZSAArsdMAaK+zvjI\\\\\\\/qCVjxDZFOruNix\\\\\\\/gj11AHMZ7HpBYl8GH6fkfG\\\\\\\/mKbviTLrgqtyXXLbu8AV0Baq++z9rQB2hQeWnYzQQdAtD7J\\\\\\\/\\\\\\\/L27xst24QwTwZF3L7S04Ok1aMywjEEPE2nmOm+k2oAkENc8oiC7yc029jqUnfqCd9+egBgEzS5NgHLAy7SzuSMFp1JBvGU9tfeJwUO\\\\\\\/J7ipWtjXhj2mcb\\\\\\\/k\\\\\\\/MEOQ31sbwop0vCgGSd898g+p82HIkRSc7RHjRMNKQ0RYdB068sxqeS3qx+jw2kMNfN49cJLz+lhjSlC\\\\\\\/kYxsb206SJ8ugGOlrggGXu9E7BDei8Pd7ldN8rdRkhMPijLdnH7t5I8Obo1RUDY7a7GblLwUtlc1vuZqJOEv2JariE5SGCw0QKkgFyM1CZ5+Gd9mWkoCeqsBmh2jH6owNHJLRQnJgJQyRCjIxiIXMa4qy7baofSdSLrEonME55zJZYDvfS0B1cJtVqf4XqyhR3IunneQ\\\\\\\/p9pzsqx23uO1DELZ2oZH8kfII8cPfwlPyuC31Rq+RrbxRyx0xeacrH0adWKMiDamzyHcyAEk1FjcK8mk1x6zcGc43j5upLVWIbI9F5+8rUUZkQB+COZOFU5js2YlBQEheObXUve8taL4odR4UV\\\\\\\/3+UKhOD54a+hS3rCFGf3v6Dx4X4CU1c8SF\\\\\\\/q645fziTbTvbb9mbiGSWQZrrPjlav+ckTQc8gMb\\\\\\\/XgPH9Q3cVSs9Qo02zgZtfv8GcONAvR6lPbXOAoMsAsnUlM5lCJcVn62B9dEoPubEP84NwJaNHjZNGolbMiLvGDexJP7AxDoS4nqsfop+fwCWwse8Inm5mdue0TpUJANKJuOSJRdrBM7yB7O8jsFN8BdJ6cmBZshYg3ioexZyfJVXbJVEeUUO5fWppX0wCaYhrW+me0wdzrr8uJFXQQbCWoowt5WUcmF9vZR+OA1K\\\\\\\/7ynpDx2QDl7SwaOGMhnN8CvO6O0tfjd9IhofPfelv2LGCP0cfn0Iw0c5nfvQ1DhH2dRVEQK5Ib3b4F99HP3VNjorZpAU7aXO+wxjhgV8I+57wKXw7QoemPQnNAr+5PgfNcHQbDo+Xvk3LQrrUQAZaCIQFIIGVFsyqgRIjPM3MK1mcZpJ6UcMcYLtx8upIlECsu2KcIojS5f2F+rD3ucuOy56kqd5ZwQtHAUqsbhji9aEyh54tipZ38tCA2l5MPuDkfEBEnr1bnQ6lwqHkufmoD\\\\\\\/0KhumvgRjyW0nOT5xC6tbHxs4xzpeEgFeBOsw1im+zA9gHlxV4jd8xoGwBiEbvTJl4c70W3wrnD2HgpK3Ajq4VjkbxKCv3auz12uun9rAg\\\\\\\/nzmkj9rxjBkyt9IjsYqV6tT3UYRft7riGWqtT1YfZMaiG6tK04TyJdemZVwcR22Jvy6O16+Ia2wGf++i3TYJm\\\\\\\/wb0oRds0BQ56ecol4T1dhOAO3GZp1htPJF5huDZRwQiKISboluOgqv5fhARMooD6XSus3MS4YnS8iq6Ng0KflrQaYsXKk99Bpx44d7DsuW\\\\\\\/GAeC0sg6Sut4+gCDN5BCxNGStenX33VyxgnOEGuZ8nqns5wmW5VXHMBBs4LTVs911IiQOOgFMDiYyfBC6nhtowi4q9t9Tu3ulijMscjGqWd2wpKet0Z\\\\\\\/MPlU3a7Nd4seZLoFl3XoyvIMyTJdsYRGjWV9+uQ19LVP9z9O\\\\\\\/0OhSt0ghdaD2FVljFeOrxkNswlNUFZxROSZn4obfFo3WTxPP0\\\\\\\/uJYlbI\\\\\\\/ImBI6BpFOYLmLaItP+N\\\\\\\/0YO175vS10pFx9UiX4lMZ2MCHEsWghjj919R0FDS8BVOTX+\\\\\\\/wnTe5mo+r\\\\\\\/Mc9I\\\\\\\/77ey3QdCnuXgn0EJIYkhLych3afHRDAxirQvw08h9HNFmkSDml5eE+QaKP2+asoM+4tuDZvdXX1vbMmcWT0UTFWDqeYnhXDK04TYDvNxMHCMxLF1Kgxqod\\\\\\\/994+qWbk91WlTNByh7kmQ6gxJICoyVsgtIo7\\\\\\\/bgXc4wNNRuDis8SiolbQQ10ka9ZAjkvg6yWQH0oKW5tAiRZBJDLeUyyLLOpxTcQCZf9KGKecr63037QOPq43g9SOsOVtpxhTRKsRpo9Y8FwGc5u76hAfHVyZO2kQnZmLvb6tY9eSksgrgbrOH7NX4hpz+c19lF2pymhyihkD5rFQsLtehc7S+FmGIdQ2ZnbLCnmfkf3nx7T5nJRhVb1A99usXzRsxUu7nXkRpZ1qcxFh4183Sj4WNN56hSVYdi689guA0ia1Oy+6up\\\\\\\/0ZJAQIlg29yshwqyTlyNmYy7dyCMJLMiLIHla1fpip83h0K33Taso6\\\\\\\/nSidFS905w8e3QquUU9UgWDMaloRTx0IjadmptkZNR9HQuLxdsNgjDywWEghS1lAIa19zcNs0eqM77WFfF+O2fLz8iglQzT5MxiyrswRQ8\\\\\\\/SvtYELObrmUo9xCTlWMleP2Q9GX6WomLHoYUNVcHOIXYC1AUaqJA5QbYHio07leK7Ipt5Vx145ssxPmb+v\\\\\\\/3Z6Gz2OO3pQxrKXM85CdWR7khYhJuG1+xzx2h4K7QQv5w3HEr9G0sqWkZ+ZgcylxrhX7ZNZ\\\\\\\/+0yGBfWzVAkbGoTy7QmSZNsJiKiWRfA2O4\\\\\\\/obfNC0vreu3EkLdhTQ2p1WK2TGqjLICbXvoiGdzBEmPeDqPUnXNuxXPMVOu6gC0Kz5ZEeimVEvr3zzjILYyIEMbSIDVo8HJsKmSLqGWf3PxRzTf7jIoEMtoDmGX4Vi+b0OmYK1cyAhj5Md3k5fQlA6tlYB151Isv0HAVN\\\\\\\/632c6uILOcn\\\\\\\/lotxfhu4eHiSa2RhSh+sh\\\\\\\/bB6jowSnqXOeIy0FtYHIyUZiB+lMPqgO+C1+jt5ONg7CG5foMhBT0Xb1c70njWHH6V+rTN+T6sG8C5BGSBgl\\\\\\\/\\\\\\\/NvtfSCw6bUt0A\\\\\\\/5EpHwpJGkFkjEsVYWWXjJiKws81s+apiap7q+Q5r90HYIvrPeux3Vc7A5DL+NfUoCZpGkeopobAtl5wRLL1DFUdSdPoLWS5UA\\\\\\\/\\\\\\\/hWgbTe+3bKNrFRLe8xqChYttrZbkJirz1IDU67WT0\\\\\\\/r1FAJcsmz0Nrn4iu0RB7jEk1330Nzoo5BF9MwaA3WGUhCie+BID8dysJXiTQACUTz4AfqrJaGu8PTgY0H4iVIRnPjwJD0+AbvFvlg7HkcoYsL8w3aaLPiOO1lm3\\\\\\\/w7p0VyRnef5\\\\\\\/pPO+pS\\\\\\\/poTnqabN+JqS9maFuRPMfAgARPqp\\\\\\\/rgCVEWLeSk0LnACzgM68xfIrTHmSppJe0hhUhzaziG+Ha2QcU\\\\\\\/qUfKB1923OCBRyq66hHinCGN5FBACQy4vXv3hbEzW2Dop+JzJwtof871AOwKnFMyX8NTbcjfzMa2aqhby75Dw1uOApHa2XTIRcEEE0POabzH9UOIohKNlRxqzpEyvtlQOYtoM0elXjVhTqQ\\\\\\\/v5UiEwA6djVyGgXJUpMnFcJkTOIgnwgte89UR+I4z6O1cFPsR8c9AxiWmw5bSwFRdnNQ+FFFtzJQf1Qz+qJbyZ+cjMbSYPhLBM1Pt5QmcWT0D0mk9lRxbavSd3NHpzkIa2bn1YM\\\\\\\/lHC6QXQB3wqYYB9JC3ZCvqPWmh8jgaEIY8lsbopuiu0zngmCvJ+QWPbKnGZpsV\\\\\\\/Ll46UnMjZaPOz93J+vjCxbv4NEAoehtVIWb0UCGUQusoGhNKez4tZC7fFtticcI7TruRNJTPO370mKsFYoTJKoNNueVieEGrCzlIRD\\\\\\\/CPjI8\\\\\\\/CpCQ9rf5Y9heGaYo2bzrOYqFsxvRgbj0tSvDemYfzCSooMOk\\\\\\\/WodonxqQ9u2\\\\\\\/8AhtrgxcO13gVBs7W3lZtx5e9ox7+qbIRYaphb0q3Gd2qnYAWA8hqWAM6I6hULnSbI+9iq0BAfwj0h7bU369r8jDNbidHcetio8gVAXx1eq8+eXnpvseErffJ+pBGHOkbwyzGQEuRhyqcbsFyA9g2Ht\\\\\\\/y0rycZdVrwtdvpwQ5Ag7psbIiUyU9sP8AxOAjSATtoqnHqdxe\\\\\\\/983ibc+Fqb9sD92nNJzIMKH4rb\\\\\\\/PSaIg3Rmcr2LswfuoYxQej6QxtOhbrf5qxnDVdnLAPyEsV9eHk8rFSQZQsZcmz9F1WFCFj+y8iXBnr5eHYaqQrWFcvaPAibzpxdDoAqtGejp6SXa7NNQA9dbcgBU4ebSngpqspJpI\\\\\\\/o3v7WBXdhke1TE336ju3Cg9L2QAe6761\\\\\\\/yAjKhq5B\\\\\\\/2YZPTUw75v58wxJQEfkj\\\\\\\/zQaNSIBSFxEh5UmhwfuZjgMDV4V1vpiH+rUEjKZSm2AlP2teJw86y\\\\\\\/HpQL9z1SCnF9yCGbaLC+NsMNlt9+yy2G+7XpHzulM8DEd5M\\\\\\\/h6ZnzQFLKMyvtQrW4WtPb7e0u4obedFyf0WNTsN2242aUaZalpSHxX2SdtP0QlicPSbFYNvEQ0hisehqhidEiD+3NQPN0TcWlcxE9HDKzHT4qJKAxQQihrrzVqmtUQqkYBa\\\\\\\/I7FvpG\\\\\\\/sKW9Mrc3l9cN7olvg7iqTl8T1Ue3GkBNDaJ0mXmuDqFsBjc8H\\\\\\\/1wxkcZo37N8IZN+ysmOE0TxYEz1Ws3i1s57frYRAPwPMH5GD66ZvjBI7yuBvWaDYBLO0UpYzOceQkcmTItSYZdYvU9eiR9EYbVg1lcogWE12lfOvJBQbFqYgDnVpFxzkxJ1jwLBPlhuezEouII9wZmUm8IfOZ8lFJo+FABuGOyvLTutJjpeWPr8LQInuuhuYiLJ+Lv6Ywvh1N+exiM1sLbHhqkCvBJOKfjgmlp1ir4qq0PR9BnmD+c\\\\\\\/ya9KfNCctllmhVoA9CQxoN6meYqc3lm+lCwq1sypbbjkJK\\\\\\\/TU74xJZcDSoPK5r5YfZa1pcOiZmJg4Ao6F\\\\\\\/uaAbcmbHRyTsQtdBM41bw9IIUyZyL\\\\\\\/WRpo1ONfXqP4LxDJYK1RsRdiIkNi3d220Iecnhqbbrs5p\\\\\\\/xVM7aAxW0Kx2Jj2uiUWueHRnWbn0C\\\\\\\/DWz+xm+QKkwcJsWMLISQ3P1CTnR6FoPpr+4LudH48PjA5521BsfCqoARrLXqpJd0KEuRALDjLw3M0NIcKxF5bsCVwrKwZiVnsBzIREp5Qg9w5DMEj5u+bpkUGWOjYbFH0YrxarB5gqllM3JvPkYU0VpllLbs2wYQUd3PddqqO5mR+N8+LDWDTkr05rPrXguXLb2wfUTTwo3UdGSMk7a+GQqH9AMChqCa2BofhpW7Zoep6oqPi97RdCReLKj6Kw4yODIo21o1+lbG0wS\\\\\\\/khzEaBD8xhE\\\\\\\/lSfnz402IldrIHOpad67hwJmJxcD\\\\\\\/IfWya8tn6hRphTe4eeHOJ1bw3fp6m9xG1Sq1o9z+P3iKWQbgOjoSXmwk2j9UlJcqxtH4iDclRtlXLHtMlcMm1I6cLxt79AmumhBlp+G0yPg8mDgy668+y5aBB8WC5kFqusNzEMflTvySbrmClfrOwAfqe5\\\\\\\/EAO+BPzOfQig96iFRzKPSMtkEHTvVn21UDuk+OH4kN0sCB5rQTrC1afqvy5pzWAtr05x0xetjaySsELEdxDfYHvz00024lymGYzMfNdXp5pQpvLaoJhsP6wPNqORzTNAYTpFRnej5EzO7465vg0\\\\\\\/sW50y4oD5upmD\\\\\\\/TAkCzGCFNiwlHmgDEtO60KnTDany1dPbUDLghgQeGZUFRK7KqIjl7GbFPAGLpMqMvaEwefrt05n0MKrMOJKxUS81osMvG9g8scBmYgxg5gZpUlD01kPttT8Jxe8NiknYkdpIdICCY+EbLdSL1GWyPdnhcUmlrV35wiwWlSrMphIUgjnF\\\\\\\/9aEvFdtxnuRQLeAmjTIn9hEvwZcnFJky9eBhoHcCWL07cecnBGVwV\\\\\\\/h3TeKX+ctOW0sgYn+Bk1XLjtXXOXWzKOCUFjYpxjPEXWI9fUQ28bQhBhuRpaTtzB+iyjIYOxKX3RuKLHp88TnRgoKnVw5tEN5hPmRaQ+pFjj6hGChKM3598u3W3CAk6eA\\\\\\\/KwpXwqtew7yZTOY\\\\\\\/o8ZuVPHtp+L2oCqdCbhq946lO1E3gQHiKlxgcmFi4wvgMdn5E7AxGNvMt1AK+wApy74XecXHP4LB4xd8BYoC9+oYNoRtz7i9Xha3zYHorpe6gToifvwrfbkcciGUXPMmAVdCwogl1Y0dc8oOwljoXbFFNH3ORy4sGrXBdEFO05pbiGKwUEzU2sBw4MVdxxRNV4j3B2PbbSgl4eSF5T1dP\\\\\\\/JEkN8yFVYjMnXOWC9Kkq7Ben6wexoqKoxDOGTqVgXQVn\\\\\\\/X8Sf2nN+X+BZg+92LHv2KP0HkrfAvpL7z03bdiQb9h3W\\\\\\\/\\\\\\\/4bAKwbdett97bRrEX972IVxaP4YR0WG0BHl1ranCkEQ2s+heGBB\\\\\\\/xBgfuvO5YP3mt6B+Sdk2IUMvd6jGudyalx7WQTek9KuCKzlRRRUbSMM7Dj3gn1ABiWXbGI6wd5ABQ1WsBzvn0Hpu8ZVCaDavy8msJ3YGh+5EpmtRMv7Jwfi1VimFYGaCYMpIQav3eb6goBliPofkix3mdGmh2UBfVnUMDfj\\\\\\\/njLhZiWPI0pxMhE2qviaQDLtEiAK64YgXqunH5PuBG6pX+BhPAmlp7LVzkOqKllCjkTrKhf8QQnMF1Tmt05BNi8BP+qnqJGU9g8LicyTSUrEwg5XwZcF7MJTVXz+vErHHyvV8cYhtNXCLR+PtH+dzbof8jT8mVDL2jorYoOq6vYuZ+wxkWh2L9lI4mhie+sY4dKaETJJ799EO5G+rsr0PylM9g4UTiWph1lrbEgpUlBCZtU+6Li4DxPeKCTsgpYjg2luQt89wqj4O4JBMaCV6BKUFReeA\\\\\\\/ugSUzZgjonT8GkT8ea21yIfju+AMNLHF1fixkqOkYUDdPUwWeMDFuLBAFvsPStVKyMe7DUP2omcSPOr95nLOKvR5MFtRTwMBiI4PY0MZGz6g156mIx4wMBvTIeoMtQpSa\\\\\\\/Pz+d3sgYolyQb9j2wHPxALXkZ\\\\\\\/5PJY\\\\\\\/UU9JemdjNcmv5N1bJCVYVoJ13psXwMkJG1Drt322ZL3xurDmxIgLMKf\\\\\\\/QRUWXhnXqvhKGsAFX13CGtIg\\\\\\\/ekEj03S1vfIWtvZOdc40Gmi8LcfEvKZMJuwTxJ\\\\\\\/F2DcQiYg94RtdYYVm876RFaI+0CL5qP7sTM\\\\\\\/RpG\\\\\\\/MkoC5mXCB\\\\\\\/IgxHO9oQxNxFqn\\\\\\\/8PKa\\\\\\\/Eo3xd1x1+u6SF4uCvzJ\\\\\\\/rxsjKulHZlWikfOsrP5+wN1l\\\\\\\/c9v9lz0aW4unnZZ5KZGfXue5x0nyespyWaMU7uf09kmcy1Q9hG0suNJTcjxdrg+t+ubqA3VbufBwfkJ0Uk7URpl5dZVNwYuhT3rTGryMaGo\\\\\\\/h7HAsNPFAO3luUDKcZ+TMBf5ltIxAA2WIpHBxKa9WoCJWbwdMSyehts\\\\\\\/QdN34Qa6T7O\\\\\\\/N3drDHWU68FZsqdTTlrHYIo4rgBydVgbzjbNg975gfYE\\\\\\\/mw86hO5YE4gERPkMIyXKNGQVoSjtiTahWFVXbb8S8hkfPZ94Uh9tN8\\\\\\\/AL3WpB0l2ClMbfqLiAjDgSpw7IXRXMkCkcgU6IXUJ1LYdw6TJoJ6F18HgvC57ZarPK3uX3mF\\\\\\\/RmagYxrLO+bigr5dqATgAG9UKMbd0H86HUf55S9OD8oaEX7uNSANzrytG4bAxRABnuALyUX+6pGr0kCpAbUYYOg+O\\\\\\\/ENp4phPtJ9LdakXuifiO5m1CfkdbYJrPubduJFtMcg2qb4Pp9BG9rJoB8pCvaq\\\\\\\/c2FOv1MgW++timmHG\\\\\\\/HT8ukTDA1OOfYK\\\\\\\/wKD8CLaRUToExKXJH4xNrZzV3osdreGw2NdR6oyPNsOXJ9iHiBJBYClpsWQYt8o0he2rAH3tO5h457IGYj9h98APW6+e7S28EXS5UNQ8jGfi7+Thni6ATu5deD01wfHGJ8x4kZ7GbEH5LwBfR6PXt+cOyaxUfXbBUoJdH0Nzm79f24Q63ZplE+xeFEvhuhFeeooeONGbH1PfEXe5Qj5owaALAsOWiKCEpwgKxY+ZawCNPW1l1+l5XqZq+Nrn\\\\\\\/OLMvFHORi2csUl66Bv\\\\\\\/T\\\\\\\/ThbdFotgxzp+BsIQWj+D7U+BUG6QoI+kqY7nOvltbY60\\\\\\\/3NOaTI7\\\\\\\/\\\\\\\/dVYjSYwlnRpGqAqSEuiQT4V+q6pHo02TmiSn7k0u1LaPfnhE3CbCvpOzCXJDPDl0BXqNiHonDSvf2BTTMNt3QeBNpnZb5xuQddrk4GaJrfSXLFQjYQjSYTHJ+xLmYQCsGigZX4laJErJKBeuyDlPaBr3frT8SlWS6YDnEgLWJ5o8D3hy9JUwJ44CtqQJavUbuAGRYEZX5iAr4dboklPTkyPictSw3UEk+2+8nti2Nf2x1b0evsGskiPU6ZUGMrB1AJzOLf3gsjso1P6a95TUp32mSJRNH\\\\\\\/O1gpK1zfGYC\\\\\\\/\\\\\\\/4ghbhbQ\\\\\\\/2tbeO6DdkiQ0IZxBRxPhxKVrANTh2En5WnS6GRKKeqlvMGhQIqBmmEE5BtkmJGTz6hmP8GtJgPIlwuxPfJkT3RFFI5wv1npmVvxyCijCuphoT7HhhjDlCJzz9lEMP4J\\\\\\\/+fNgvRizAAyM\\\\\\\/vIEqkfSGkUZODIGyXll0\\\\\\\/SDDMsUy2UfrB2LQ3mUUliLCfDlla4DPVoWcG4RpLZCXlte0D6rKyx2530JnnIBkVn2lLTQa0p2F4wckvjyhrnaZD\\\\\\\/AHN5RZWVdVlyFtlE2PazxbR8LoxQsTkwqCoqy3XyNPy55t80xcFLXkuTDXdnjYFQ+8eFxAqwuI1b4YCYaCP\\\\\\\/yI3y3Ns7UaxEVRmzTdqNmEForSNSW4WzkOOfqmUzxPFFHhQnlq\\\\\\\/uBRubMUwjKPhMO5HVdCu2bc6EAohTRC5gUm1xfYRhNDarr6l03Xfqk4O7kBOm0JwaQshdx+BgoNRcdi4OwE7492OwjB0SKdgySJ5iagqZ95jhwZhHPaFIQ7rQExwXc3yikmbLZQUeh0koRUp8jJcWvfYsA9beIb5eDD5XN8BqOvTom4c1L8DnXJyqYI7jKJrPnD2lO3d05WqByPck6nfcJGBzS3GDw2a4xH2ZtPjABpfK3hnvuQ23yVeVv0rrvWwloUpgmf70Jd3YvfOAwFZgcBmptu0QVmhvU1QWezSL\\\\\\\/ReZmjueYj3zcchQse76y20M+fNqNC3FG\\\\\\\/AnlJ+V23qkStl91hlbXTYqrzo2xj7h4iJMw5cgMjZwYuBJ+dp5oNj1R616XdNx7tK0snAHaw2hoZOUIUz1AhnAOHEIPufLlca7oU\\\\\\\/zN5QJu1oZH6JP6r6gLJDaC4EGA02yBvKUFDm29kvjqqR5xY0JfzfZ\\\\\\\/o0M2KKlYBVQ8xmP9yCgxsuNOJZA2fF6OorJxK\\\\\\\/5elgUaBw6P+bZ2lYcQ89jaaXGEFLXt0CAo\\\\\\\/9KpI3EqwD338ZJtjIX4SRMuImliG+2vl5mDhXAfKIxMhLZCoqhWHfxLk0pdriDZmrYxmD1PN3aaHjmMfTFDz1I\\\\\\\/TeuqYU282ISC0E+pMyiczND6BrzwSEmpP\\\\\\\/oKM4+JkoaRhPSJQHHs4A4EQ2vBNpglaEECtKhL9GMjhbXKKY8eNB7iJnUKXlkXe1YSOClTX5nUDCk\\\\\\\/Rx8ewEx5eO5sgCpoqIjsYSNLkxYfxpnLk5qrzba+IGYOLNnb1IdtrINWzg\\\\\\\/J0+H9Imdw610T7+80bbBEBHjpTPO2HOnoixppu4Bn702LSGiaKwiPOOBRFuXML6TNmubW80Q5AiBM1Dyz5fBVtLf46LpsifAq10DKlMMJXSQQwPoEUaR0Npm6VcXtvi4lUVeYLAPh8G2tDJzHKM2911bDbs3+3BHdIvIKJct3C6MJMzwkdntAVRUXmcWIByBdhtPjJySUOUvTHRdA44R2Xx8OnFFrt4sOJVCF70sCIVLU\\\\\\\/7RJ9w1EREtgkYUUMJfa9g+8XqkOLJfOkf0ZKv0LV1XM4hhr2nHodlk9bbx6OtcnOdsUNaCfQNwE+KmQUzTirHSmGIx\\\\\\\/OrjyL5oblLuDVmdshZsht5IkJX5M\\\\\\\/uAV7cDMWw\\\\\\\/U59QHhEu\\\\\\\/KsOeamHicJs1ZNPNs2t3DSgpU\\\\\\\/uRNVgdPUS2TyaOmjjMKV\\\\\\\/S46+tB\\\\\\\/ued0RA8ZnokQFamagPWCUNBiwzKW46h1w0rCRyUmZG4X3RG4bi9ooIIzl1hwYy98A4yZD9UVQzS0VlqowYY\\\\\\\/FmP8vUiqafBhEyogFoMUID3y6xMxU8IvyhMbp+r0jcw2bd8dDkM5rtbwq2F8yMoHpy6syXBTnI0YX9gg6m8uLMyw7ahkthQFpLdiuAs4eerNsmdkxkzFP17iNsjpfngagdcIk6xcao5PUiS9\\\\\\\/qlY54H8vBe4KB5sdhxa9oJQRA\\\\\\\/vXZ1tX8hUZE9T4YgQARkY3R+Q4hfVhEv48MEezKiPiPr9jsbvDkW7UThE+AdXxRoryhK0sZpMf3XEdfgdPBL+mXcd2cu9GJsgh336HmBazZkPRiu\\\\\\\/fXjlKu2Lbzez+HWAEy9bW8EUaX61FEjQ\\\\\\\/Ysft1qpICDhNtaOmhtmISxTJeuPJ5nKzZeaqoGYQKV3RRxvu6qz145J00bz2ra7OP1nQ\\\\\\\/hx8CIoguLURrZNywuUEH206MX79LoXB62O3hLkq593SrpD9bldaBodYYbwpUC2fZYOcDZAzfVOE+GgmMMEdg39+tGBkR5ZyO\\\\\\\/OJvFLyZE8HgLllUJXFnODwCEV7M77CqJ794xHUzKiMg4GWjo7oQTtWic0YTWqZv9EPCaQ4XJTvDWrDb073I3j3Q9McMrA\\\\\\\/bipoxE2TzKpg8T7tbr1fS\\\\\\\/Edcmk7l8IhFCpMlNLesHuxWGlGAEiF7GANfP7j7DmcF7mXainTybq54XmVHM+MeYmwTFERNJ5yA9wiFkKNCPbU+GuHpVkIxXNOkFoId4v2wYxJmUxOdFh6Y+39BwlmSJ9GOXboZSNSPzyv2sA+cWPavlKZW3R5RsVgePQvVgi5W3G0iHOlz7hkEduJFspNzTO1T3ffydH0Z3ZdbkO6\\\\\\\/vD5Qe64LD0E5w63G7urmzUo4WCRV2i7nDzl6FNc4I1gsH\\\\\\\/gDrGGI1iYxxYIOB3pWqsrh484Ip0x+U5sPB9qCSfHi9J\\\\\\\/mcgx03qkgX0QCCGnw71DGeVakbbL2gxF65P4u74U8FA79svVHVHlvvU0t\\\\\\\/qNJGesNzJNBVMaxI57GTT4LN\\\\\\\/NlNVtygVD+6Toy3D6t6w5IdCI67c63MOggvnqTmhlohWNh47EVF4W8T0xgUSsLldUvNeRXRM+IQ75s9PbVFijQtxyvlqU+HHSxiY2YI93tstT0\\\\\\\/7yfYAPEBOC+yzCWyAWi3SmPcvjIfLfmZvkB+AXQ4SaC1WJe3Hz5HCxsCQKlFzPMTfYQ0RjKsB4HEdsLJhKfSeIcKL5gEj3rtm94GreqXLXt8xhOhJv77M2eHKwhcgcrjtJHUEu7qe9TDzJKkqFWg+vCQ2XJLMY3UyaVslzfIts3YBImwBQ5DtTw0dfWHVaB4PY7luZ3bj5CxT08yH23+TUxDBmdTRU04r0uxvY6nu4ZphU4lc8B4oF39PRz9h99v\\\\\\\/pp5SkRbV+Jugf7NWN7HZLoEspSIFBDeCPieLSNgZQoncA5X5fBCnSiq0lvAH6w+9vuoP3Udd4FUuuCQZI2OgKgZzcKNuV\\\\\\\/+Uo7S56Ac2u\\\\\\\/a8VGRwDbngw3Z+VLM1AqqisgOAvvno8IzMn2\\\\\\\/30IUhXBBTo04i9aJ35SbpRFRd1l1YAbGxm3uSGsRjgrVrbmJeTnIQDAMVF1QGFDi9e5m21JIuERURrWmyBw7fK9S5rzhMrGlRNeEeYnGPFkJYKuKbJoQl9DtKaAEAGVAKuNgMrw9j90ksegC6gP0PbiK\\\\\\\/ICJNmvPbgogQ9fITStmMzLMb6LrM1b8gYisK\\\\\\\/N64gbnoyF9psR1GeDeX8lLcSi\\\\\\\/XKPfzo+dv1QmCzjC56E18xBMlPitx42y1JY\\\\\\\/zY3v\\\\\\\/uVSaTklTnVcgz4REoiG8BVOqtfgLdazRbJPskypslsZbjSroT+yY1mz4ThehfGYdFdDn+NW7x+lSVEKF1YeWDhO5Omxxb4NnIxfiBtYLVQXJs9xcB1fWB0n1vHP6\\\\\\\/0lPH9xaLuxGCtGVG2KslILynQhGe6uVs45XVR5\\\\\\\/lM3NMbJepueNIRA8wEagl6LPWJLE5\\\\\\\/fcXCKO8prZn08OiFCMMRuajMyFiUsaqOVjbEOgH1ZgHoavWBo6sMC5\\\\\\\/D3JTdfDTi3WLn15doOGf4P2VyoaL560\\\\\\\/0tI2vzgouYzF0021LYWTSmUxSpzIVEuysU+sgnu61t7wih6eKQeyj55gKMRVYN7Ri0pe5pmsCN2WffeUNxb99nkiiGJV7nsc1qZ5BBIn6HKnZYaPJNSlO8mbXjL7sxxIwQDUD1DSW+RLSkeW1TGubc2EAZt7KwxhlVKOHW7FYFg6d\\\\\\\/V5dVhttyRm4erxvli3Qo4LDX7\\\\\\\/j5M9QKjkbiGW7YQ1MXmRJAMAz4jLVkiLsd5RpbuOS6XlDCAkibh0hBx41n81Zs1qbYqJqsNXkCgH2LOOBi1F8vK8o2k\\\\\\\/eBvEQ\\\\\\\/cQy1OMp\\\\\\\/T1AAkD6nnM1pIQkglQ2Yo8KBqFLjnAQnMtsbnG5Iy\\\\\\\/C\\\\\\\/oFp4x4CabErukLYI9bYEVxdg2obw0aDzc3qJ0\\\\\\\/nh\\\\\\\/VYuaYiWHhMe63NuGNn3R7Xv+VxgXOtyIkT+1zeFbGy+5lsbsMfE3\\\\\\\/323GtplzkIJeZTDPECYCOs\\\\\\\/nCdU\\\\\\\/\\\\\\\/Oqygn+1pTs4yjUokdxIYBdVfnEgiIwpu5zgd9\\\\\\\/po8pOvFJLt2PgP1IllH0ZeS09BlcrhXYR3ElD2kyn4mCpmx6gFu2\\\\\\\/SugmN8CqmZCtheKKQp4qBW6c+KRceY\\\\\\\/t0g7blXRyh0ev6Gcr+QVxROvuwWzK8nKOuGJqoGtW0gd7vYALv7Jf5eALGqbjKIm9ebktHcnPMplTFdSIFgr3+zKNBtzAt+xUNbLroFcIHfIhEzjItHaIVJTa2P1HNjkEC+kfDxUGYUY9obQ24bgTyy\\\\\\\/EHRYroVGhtJcS8Gmof10rsdeuu8LEOStzAm\\\\\\\/8TNyH8zgTAINvIB9LGTwAOnZvmYMbkoxux0mi7VFA8I6ZE00Zc8lYtbA4mmqgMu9bvXd+j99d9f0x2cZIL8\\\\\\\/zFeLBS1167Q7uT1LWHLn62DqowGgzRfp3E+Xa71l2ONxdUT7OLe9gPELTrdHoaEEYhcSaSPhQKVUrbBy3ExJH03Qqjafg3y3BTGcmPzOLKRuDJBMOqUUf\\\\\\\/qqLbSwYJsiRQu\\\\\\\/Q2Vuy0teotylJyA1RhuemyAHllwB8Iuw8BMvrnctcN1Q4jdvkaEJtBGJDcXQGE7s0lbtIYHZY6QfQfAPR4c7KoWykzw2iZ\\\\\\\/qxOEmY27K1ruzUzcc9DgusKcstxRKpvusbYPlG6bTGS1HIDh3bdn3gw2X9NA2uye\\\\\\\/zk\\\\\\\/x093tvvgjDkswg8cOX08lDbdki3GpVl2Shbvn1HVOebv7wC+oxV0N6gQaV3Z3SD8HKMIp+LdQ5dmhcus5XxkNqkfPbLBHyWmt0a5o\\\\\\\/QBc43GAavitjY+yeYDvJ\\\\\\\/OBZHYOtq8MdCtlRSTf3HdZeL1yZkEaNlUHuYDEprj15CwtVtamL1RIcRWDl6v53Zklkoy8mcj0rov\\\\\\\/l3e\\\\\\\/skQqD5I+BOWdIBWcuMMSnF4fKlIUz8bx2R3mlMFn2yQQakZTighzK7rJxIfAI19jEPuepRhaRxclsV1nTZiBfBont9p1Mrl2GTG7Y7XsotOGs91VgKg\\\\\\\/4kfsdrdVEBklKE1Fl\\\\\\\/UWAL9pqt81lYs8fglSRwzlpyKg91sx83r8AOhdLIlPKkBGy47vSuHeJfyaO\\\\\\\/fuLPjuRLWMK86skMA7o0YaV3PIWsTPg+5Gfy7+TcbsIChpkfi+CiN\\\\\\\/DlVH4EF7H+HMr\\\\\\\/xzcZJkzqG6X1aIknJOBQHSZElQkNq7Ua0wwG7ZckfwF43fBN2kGJrw57NmdB3JFpqqiv3vjqbvWE7Odu6XXazZjPFM97pGIP\\\\\\\/rTMNZLrTfEklU1XCi18XmijSNyzLbh\\\\\\\/Hr+8HLuSWvMp5RF0qNNBknB\\\\\\\/u1Mo89uLLmvYKpT7q+T\\\\\\\/1to2xd31KgDoXpSB5IGc3fs\\\\\\\/vMf9nS5PFSrHKz+ISKI8ZNIV9y6jNn\\\\\\\/DLOOXW2AV+VmmM+Wwue+pA9MPLq22sfP2UiiY1NfiKA5kOZpiZa3xN9JyYkQhiqBEBlorwv\\\\\\\/DemXiWkDuqWpEK63j4pRSd3BMVGFhn+dxTEokcLvkllzhB78omGDenhlnBO2f7tnCRcV+RuL4hB305hkQyx4caB+bVSOp772e8ucfwIFuluXZDB7bpVBnJvXy++k4QgA0aypxYSyZCsg8fVSqpsk\\\\\\\/i\\\\\\\/pNevhx57nplj4D3ECSsVgtyrFf+ABuRF5TahT9Jh+LynXct8ofMJNSf6q7Vt+Rd+jRS9w11x\\\\\\\/UfHOn2ItMNAAHGgxBuLXZ56VakUuloLYhhMO4smX54V\\\\\\\/qK4hoW65mSk1uClEzTLHO0EPNbTh9Wu9nF4Kc662MXjAQnikBMjtp952BP5uJDI0UDDcdxebkGhiEWsKvEL9DllkU3m+WUwR52dMhHgnvQ3VYbNYGzwRuamRxV2rvC678iuVY81aHDlYHGHi3OQ2mnWDls9ACeCWCTXqvIomcWyGVaV1ms7Mq2R5x9m6o\\\\\\\/UhLXDY7XrkwgnFDAGGd4uU+FqZSkZpa0DiN6+7dNC5Ra2V8WocuvbCjnDRsZXgr0\\\\\\\/OJXVOy\\\\\\\/8qFHPCrz4SUFdAAjfZcpM\\\\\\\/Znxa+XVNSQlZag9xcTozBjbvvWgQOhVTZSW517LupDEHGR02BqRmyX5jReDKoTh99yV28hWw3lGUO95Po4gkKmPiIQNaqoyfQi\\\\\\\/JdhiNNQO6afbG6eeWUEQ4mJaQq+RhjeaJ5Ofg2BSaMS6\\\\\\\/wZUNYKczkwKDYqDH5OSz4JafR5DxBVHavrLAyDFj6EYLm\\\\\\\/Wn+N9\\\\\\\/8ke2wZ8QzPZYdtoNDGJ4myJk7MJ6yJjIYUAmQNK1qu\\\u0022,\\\u0022iv\\\u0022:\\\u0022aUd9nOkM2w5UtpZ5Bkxt2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aa991eeedd84dcada45ca8e1af6d2a707bc7ef10035c64f3411b024afda62e55\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GVOEaZ22cTV32azSVh3u8+n1J+euZJZaWaQyhWoJ3nzp+u3zJ7uF1QT7GFL0aicYZxb1jkMrMlAM41OXwaQCm2Cezbp5jVFGWRkt07anbDoVHf5v30kVCo7SmXWSPbT7FXJuQGPxFwLQUvyaHrcbOjUqSx0qSoKPBVmDO6E+axIbWrArCQKVFJQPiQZOhS3pBhzNRFDxpKuxHAL7hHp3rRlNZ\\\\\\\/RcicKxhfhNTANe9c5Z0tB\\\\\\\/j39sCOhN1eKwNYrebfXnAyu6p1k89FNDSu7ZElXEOVfQEZYcibAMb\\\\\\\/kkJ2fBJ2nfycF0c8FSBwRcRKcLJFSG+Hh5M80BN3Kgsa0WlDIYRBlq\\\\\\\/fEq3pHnbIyTX7XtP6yWODBwqXjH04iSrbWdh9CzGa3RNWOmKhGTG9pYFfzH7Xk\\\\\\\/wKzaobc\\\\\\\/CH7jcP\\\\\\\/A4kvmfEvOfKiZ69GcUisGLFiPOmIyQ8Dx0gzpScn5uLcKBP3jONSaBJl+q52EVNTK9n3JG53n90frghTAk7YTSQ\\\\\\\/WhLutcLEQIOcqJoHMnJmHiT7anwCKrJ9AuBUbjtfMKLCuYpOiQ0l2CJSnOt7HtNtjcCUcdfwTp7Ncp85nAzjRnFm8H3x6nCO0yVhQ\\\\\\\/32ETLStbDxz3YKmd6X06LFgosk\\\\\\\/mFG0mFdQL7Yu+hdjTv7LhRdyCOq0uvWzxYWwxRxLLYDMsI0k1GL06K4j2LbYhNLWCg5C\\\\\\\/kTNTMftU0735Gatxj80nysEjNpDwZvspdaeBJfXnY6bdQNFyoMJ4ytYv2+nC77jeQmhS2vsf4qHWvV5QTTHKDOCJqNySkOn27Ox0\\\\\\\/vYgOysCPrcQmK+2NCuWjRwybYs9SO37HfZhWTZH3ViYeAijWkSoVpB8GsECskFm2vaRhxvTLkxuUZMwKf0G9bChH6gdeutD8AMu7XSx4mxASYJmtJDTU+0NnxvXj0RvtTuiNsGZx0nzee616NQ\\\\\\\/xLw14OpWgANC84yr7CoSapeCnWUTOM\\\\\\\/kBZNsMhbd22AZBQY6AJRdSkqoLA0pFVUMTra\\\\\\\/bQgxRk5f7WO6Evg8sMHpCRoYxb3GwLsLbhOzVa8XPOVIerVFKZ+DrNWiH0NWm7ldmhvCnV\\\\\\\/wuXU9HTXZW2Tews7T9MaoK6AzjpLoaAvtfkpMsp0fYpsbfVXXqza4inpE0lhUByYGqdhq4CkqnS7hxILIjIOVEqikgIF7OfFAKEdk+r7ZzQQpXTdvbAc9D5cPIU7aBaz2poU0IGtQzZwrbuV2Rp5W2+6qVn8lij70+qUXAwHBTsRZmonOKr3n++2kvwjlG966ojTD3u237Nt2XL3ClJ23I6W2y6sw4XmJjNUqZLpr10IiZKPcLJlK1g0gPm8H3RzUMgJh5aq4QAUzTiLYTdPXp0+T4Ji586wrz+jREeXfCU7TDxLdsrfC1Z1+DxEBT15NBFgzx63FJMWO0eV6\\\\\\\/nB2flUs+Sd4leX4hsJ6BayvUbFVd+Wq1c+8BQSmmQt1EmspP0bYJ6OtNRtINQJMLOEeefnT4ycoppxuD8fGh28EjVsZpxCmVGlfaVrOSRxqkVLNtaPmDLLAk+D6LeR4x9VvKV2IE+VQI2ogjMS6XN2ugiXUg3yGLVNXAuHSOSrTxKvPLeETyyacc7BHNHJsXI\\\\\\\/\\\\\\\/20dEu7nFLE8Sum2WDkHv3lDThiq4sP1AW7xMhbdq7YSUyDemfi5vHFPHLuhnvwK+qNY41iLuL9OE8FkS9EadWS3Nj1ROQFYe8KW7KAhU5j7Lya4xjIAf3XLq5DRwaNaNZCqDXJoc3v6wh2WGlxlhA\\\\\\\/aLkScPFP\\\\\\\/Q1kuw6m5H3oL21kp5x5ykBcPks2ALzDmRHYfbIm0OTKw5nl9ApCgnKvN+xgkIV\\\\\\\/seBvxBk0ZrQM8ojavegWrbZCa88iYPUQu7s++asLcxZbeAYoF6Cp4gBWghm0jyu4N+So1Gnrbc8n\\\\\\\/htYv4loCE33jabWzjp2alfsciVomTblsLz492ZPiVm6vglfjVLBOmoKVTi+fkF1LJT+1\\\\\\\/B6Jo\\\\\\\/p66XeVj1GnYUC2rqqL2wU1h0ZbaEBJC\\\\\\\/5fetIwWcfI\\\u0022,\\\u0022iv\\\u0022:\\\u0022wlY8JNKD0MauleGxzXu25Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c476d8d2bf1631dd306a0b2439db5d6f0a3af0ebeb99055b4da075d3096e680c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460172 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2329" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CH4CTZxH8xFyPmQK9agUdygWr4sQ9OFU6DdGuRyq6ck" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2329" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6W-klMQYNnxK_bLYvnWZ1NMeVTRtE-mtFbJMNNv1Nx0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2323" title="bz-rates: A Web Tool to Estimate Mutation Rates from Fluctuation Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2341" title="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460172" id="node460172" data-pisa="ggg;5/11/2329" data-pisa-master="ggg;g3.115.021592" data-apath="/ggg/5/11/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amanda L. Crow</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey Ohmen</span>, <span class="highwire-citation-author" data-delta="2">Juemei Wang</span>, <span class="highwire-citation-author" data-delta="3">Joel Lavinsky</span>, <span class="highwire-citation-author" data-delta="4">Jaana Hartiala</span>, <span class="highwire-citation-author" data-delta="5">Qingzhong Li</span>, <span class="highwire-citation-author" data-delta="6">Xin Li</span>, <span class="highwire-citation-author" data-delta="7">Pezhman Salehide</span>, <span class="highwire-citation-author" data-delta="8">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="9">Calvin Pan</span>, <span class="highwire-citation-author" data-delta="10">Aldons J. Lusis</span>, <span class="highwire-citation-author" data-delta="11">Hooman Allayee</span> and <span class="highwire-citation-author" data-delta="12">Rick A. Friedman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021592 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460172" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda L. Crow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BL.%2BCrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crow%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BL.%2BCrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ohmen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>House Ear Institute, Los Angeles, California 90057</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BOhmen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohmen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BOhmen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juemei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juemei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuemei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Lavinsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BLavinsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavinsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BLavinsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaana Hartiala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaana%2BHartiala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartiala%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaana%2BHartiala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingzhong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Otolaryngology, Head and Neck Surgery, Eye & ENT Hospital of Fudan University, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xin Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Clinical Laboratory Department, First Affiliated Hospital of Nanchang University, Nanchang, Jiangxi Province 330006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pezhman Salehide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pezhman%2BSalehide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salehide%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APezhman%2BSalehide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science and Inter-Departmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Calvin Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calvin%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalvin%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Hooman Allayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hooman%2BAllayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allayee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHooman%2BAllayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rick A. Friedman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rick%2BA.%2BFriedman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedman%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARick%2BA.%2BFriedman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rick.friedman@med.usc.edu" class="" data-icon-position="" data-hide-link-title="0">rick.friedman@med.usc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2329" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460172" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460172/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2329.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460172" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460172/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2329.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460172" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460172/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2329.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460172" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460172/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2329.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460172" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460172/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-wide association studies (GWAS) have been successfully applied in humans for the study of many complex phenotypes. However, identification of the genetic determinants of hearing in adults has been hampered, in part, by the relative inability to control for environmental factors that might affect hearing throughout the lifetime, as well as a large degree of phenotypic heterogeneity. These and other factors have limited the number of large-scale studies performed in humans that have identified candidate genes that contribute to the etiology of this complex trait. To address these limitations, we performed a GWAS analysis using a set of inbred mouse strains from the Hybrid Mouse Diversity Panel. Among 99 strains characterized, we observed approximately two-fold to five-fold variation in hearing at six different frequencies, which are differentiated biologically from each other by the location in the cochlea where each frequency is registered. Among all frequencies tested, we identified a total of nine significant loci, several of which contained promising candidate genes for follow-up study. Taken together, our results indicate the existence of both genes that affect global cochlear function, as well as anatomical- and frequency-specific genes, and further demonstrate the complex nature of mammalian hearing variation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association-study-gwas" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study-gwas" rel="nofollow">genome-wide association study (GWAS)</a></li><li class="kwd"><a href="/keyword/hybrid-mouse-diversity-panel-hmdp" class="hw-term hw-article-keyword hw-article-keyword-hybrid-mouse-diversity-panel-hmdp" rel="nofollow">Hybrid Mouse Diversity Panel (HMDP)</a></li><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">genetics</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/abr" class="hw-term hw-article-keyword hw-article-keyword-abr" rel="nofollow">ABR</a></li><li class="kwd"><a href="/keyword/hearing" class="hw-term hw-article-keyword hw-article-keyword-hearing" rel="nofollow">hearing</a></li><li class="kwd"><a href="/keyword/cochlear-function" class="hw-term hw-article-keyword hw-article-keyword-cochlear-function" rel="nofollow">cochlear function</a></li></ul><p id="p-3">Genome-wide association studies (GWAS) have been successfully applied to numerous complex traits in humans (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Welter <em>et al.</em> 2014</a>). These GWAS leverage natural genetic variation in an unbiased approach that is advantageous for studying complex traits. We and others have shown that audiometric threshold elevation secondary to aging in humans is continuously distributed throughout the population (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Friedman <em>et al.</em> 2009</a>) and that the human cochlea has a perceptual frequency spectrum of 20 Hz to 20 kHz. Heritability studies have shown that the sources of this variance are both genetic and environmental, with approximately half of the variance attributable to hereditary factors (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Huang and Tang 2010</a>). As such, a major impediment to progress in the discovery of risk loci for hearing in humans is the lack of control of the many environmental factors that affect hearing during the lifetime, and, to date, only a limited number of large-scale GWAS for hearing phenotypes have been undertaken in humans. We recently reported an association between age-related hearing loss and SNPs within the GRM7 gene and subsequently demonstrated, in a more comprehensive study, the polygeneic nature of this disease (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fransen <em>et al.</em> 2015</a>). Despite these successes, association studies in humans (particularly for common forms of hearing loss and baseline hearing ability) are still plagued by small sample sizes and phenotypic heterogeneity.</p><p id="p-4">To harness the same investigative power of the GWAS approach in a more controlled environment that addresses some limitations of human studies, several groups have proposed mouse GWAS (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Valdar <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bennett <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Flint and Eskin 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ghazalpour <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mott and Flint 2013</a>). For obvious reasons, mouse models have several advantages over human studies. The environment can be more carefully controlled, measurements can be replicated in genetically identical animals, and the proportion of the variability explained by genetic variation is increased. Complex traits in mouse strains have been shown to have higher heritability, and genetic loci often have stronger effects on the trait compared to humans (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lindblad-Toh <em>et al.</em> 2000</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wiltshire <em>et al.</em> 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Yalcin <em>et al.</em> 2004</a>). Furthermore, several recently developed strategies for mouse genetic studies, such as use of the Hybrid Mouse Diversity Panel (HMDP), provide much higher resolution for associated loci than traditional approaches with quantitative trait loci (QTL) mapping (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bennett <em>et al.</em> 2010</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Ghazalpour <em>et al.</em> 2012</a>). The genetic and functional similarities of the mouse and human ear coupled with the ability to control environmental exposure make the mouse the ideal model for the study of age-related hearing loss.</p><p id="p-5">Both naturally occurring and experimentally induced mutations in mice have provided much of our current understanding of many diseases of the ear. It has long been our hypothesis that common forms of hearing impairment in both mice and humans is a complex trait resulting from susceptibility loci throughout the genome that contribute to the natural variation in hearing over time. Based on literature suggesting that most common disease phenotypes result from sequence variation, often regulatory, in genes that are different from those underlying Mendelian traits, <em>e.g.</em>, nonsyndromic hearing loss, we undertook a study to characterize the genetics of hearing by performing an association analysis that exploits the natural genetic and phenotypic variation of hearing levels in inbred strains of mice.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-6">Female 5-wk-old mice of 99 inbred strains (n = 3–8) from the HMDP (Jackson Laboratories) were obtained and used in accordance with the University of Southern California Institutional Animal Care and Use Committee (IACUC) guidelines. Mice were housed in sterilized microisolator cages and received autoclaved food and water <em>ad libitum</em>.</p></div><div id="sec-3" class="subsection"><h3>Hearing assessment using auditory brainstem response thresholds (ABR)</h3><p id="p-7">Mice were anesthetized with an intraperitoneal injection of a mixture of ketamine (80 mg/kg body weight) and xylazine (16 mg/kg body weight). Mouse body temperature was maintained through the use of a TCAT-2DF temperature controller and the HP-4 M heating plate (Physitemp Instruments Inc., Clifton, NJ). Artificial tear ointment was applied to the eyes during anesthesia. Finally, mice recovered from anesthesia on a heating pad. Stainless-steel electrodes were placed subcutaneously at the vertex of the head and the left mastoid. A ground electrode was placed at the base of the tail. Test sounds were presented using an Intelligent Hearing Systems speaker attached to an 8-inch-long tube that was inserted into the ear canal. Due to time and equipment constraints, only the left ear was assessed.</p><p id="p-8">Auditory signals were presented as tone pips (4, 8, 16, 24, and 32 kHz) in the form of a hamming wave with a 0.3-ms rise and fall time (total time of 1 ms). These signals were presented at a rate of 40 per second. They were then sent to an amplifier and then to the sound transducer from Intelligent Hearing Systems. Physiologic responses were recorded with a 20,000 analog-to-digital rate and sent to an eight-channel 150-gain AC/DC headbox, and then onto a secondary Synamps signal amplifier of 2500 gain before analysis. Filter settings were set at a low pass of 3000 Hz and a high pass of 100 Hz with an artifact rejection of signals with amplitudes exceeding ±50 μV. Three thousand waveforms were averaged at each stimulus intensity. Tone bursts were first presented at a high intensity to elicit a waveform. Next, the intensity was decreased by 20 dB until nearing threshold. Intensity was then decreased in smaller steps of 10, 5, and 2 dB as threshold was approached. Hearing threshold was determined by visual inspection of ABR waveforms and was defined as the intensity at which two peaks could be distinguished. Experiments were duplicated at low intensities when the peaks were not apparent.</p></div><div id="sec-4" class="subsection"><h3>RNA isolation and microarray analysis</h3><p id="p-9">We isolated cochlea from 64 HMDP strains (2–4 mice per strain) mice at age 6 wk. RNA was purified after decapitation, the inner ear was microdissected, all soft tissue was removed from the cochlear capsule, and the vestibular labyrinth was removed at the level of the round and oval windows. The microdissected cochleae were frozen in liquid nitrogen, ground to powder under liquid nitrogen, and RNA lysis buffer was added (Ambion). The sample was incubated over night at 4°, centrifuged at 12,000 × <em>g</em> for 5 min to pellet insoluble materials, and RNA was isolated following manufacturer’s recommendations. This procedure yielded approximately 300 ng of total RNA per animal. Gene expression data were generated using Illumina’s Mouse whole genome expression BeadChips (MouseRef-8-v1 Expression BeadChip). All amplifications and hybridizations were performed according to Illumina’s protocol by the Southern California Genome Consortium microarray core laboratory at UCLA. In brief, 100 ng of total RNA was reverse-transcribed to cDNA using Ambion cDNA synthesis kit AMIL1791 and then converted to cRNA and labeled with biotin. Then, 800 ng of biotinylated cRNA product was hybridized to prepared whole genome arrays and allowed to incubate overnight (16–20 hr) at 55°. Arrays were washed and then stained with Cy3-labeled streptavadin. Excess stain was removed by washing and the arrays were dried and scanned on an Illumina BeadScan confocal laser scanner.</p></div><div id="sec-5" class="subsection"><h3>Association analyses for hearing phenotypes and tissue eQTLs</h3><p id="p-10">For eQTLs in the liver, we used previously generated expression data in the HMDP (<a href="http://geneeqtl.genetics.ucla.edu/">http://geneeqtl.genetics.ucla.edu/</a>).</p><p id="p-11">GWAS analyses for transcript abundance in the cochlea and hearing phenotypes in the HMDP strains were performed using genotypes of ∼450,000 SNPs obtained from the Mouse Diversity Array (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> 2009</a>). SNPs were required to have minor allele frequencies &gt;5% and missing genotype frequencies &lt;10%. Applying these filtering criteria resulted in a final set of ∼200,000 SNPs that were used for analysis. Association testing was performed using FaST-LMM (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lippert <em>et al.</em> 2011</a>), a linear mixed model method that is fast and able to account for population structure. To improve power, when testing all SNPs on a specific chromosome, the kinship matrix was constructed using the SNPs from all other chromosomes. This procedure includes the SNP being tested for association in the regression equation only once. Genome-wide significance threshold in the HMDP was determined by the family-wise error rate (FWER) as the probability of observing one or more false positives across all SNPs per phenotype. We ran 100 different sets of permutation tests and parametric bootstrapping of size 1000 and observed that the genome-wide significance threshold at a FWER of 0.05 corresponded to <em>P</em>=4.1×10<sup>−6</sup>, similar to what has been used in previous studies with the HMDP (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Ghazalpour <em>et al.</em> 2012</a>). This is approximately an order of magnitude larger than the threshold obtained by Bonferroni correction (4.6×10<sup>−7</sup>), which would be an overly conservative estimate of significance because nearby SNPs among inbred mouse strains are highly correlated with each other.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-12">Strains and genotype data are available from Jackson Labs (<a href="http://jax.org">http://jax.org</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Variation in ABR thresholds in the HMDP</h3><p id="p-13">As part of our effort to comprehensively characterize the genetic basis of strain variation in hearing, we recorded ABR threshold data in 5- to 6-wk-old female mice from 29 common inbred (CI) and 70 recombinant inbred (RI) mouse strains from the HMDP. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A–F</a> demonstrates the results for ABR thresholds after tone burst stimuli of 4, 8, 12, 16, 24, and 32 kHz in the HMDP. Notably, among the 99 strains characterized, ABR across the frequencies tested varied by two-fold to five-fold, strongly suggesting a complex underlying genetic architecture. Furthermore, ABR thresholds were highly correlated, particularly among those across frequencies at the same range of the spectrum (<a href="http://www.g3journal.org/content/suppl/2015/09/04/g3.115.021592.DC1/021592SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/04/g3.115.021592.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/04/g3.115.021592.DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in ABR in the HMDP. The auditory brainstem response (ABR) to tone burst stimuli at six different frequencies exhibits two-fold to five-fold variation among 99 HMDP strains. Each dot represents an individual mouse from the respective strain and the mean values are indicated by the horizontal black bars. Female 5-wk-old mice were exposed to auditory signals at frequencies of 4 kHz, 8 kHz, 12 kHz, 16 kHz, 24 kHz, and 32 kHz. The ability of individual mice to hear these signals was assessed using an ABR test. ABR is represented by the decibel level at which hearing threshold was reached, determined visually by an ABR waveform." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variation in ABR in the HMDP. The auditory brainstem response (ABR) to tone burst stimuli at six different frequencies exhibits two-fold to five-fold variation among 99 HMDP strains. Each dot represents an individual mouse from the respective strain and the mean values are indicated by the horizontal black bars. Female 5-wk-old mice were exposed to auditory signals at frequencies of 4 kHz, 8 kHz, 12 kHz, 16 kHz, 24 kHz, and 32 kHz. The ability of individual mice to hear these signals was assessed using an ABR test. ABR is represented by the decibel level at which hearing threshold was reached, determined visually by an ABR waveform.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-1.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-1.medium.gif" width="404" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in ABR in the HMDP. The auditory brainstem response (ABR) to tone burst stimuli at six different frequencies exhibits two-fold to five-fold variation among 99 HMDP strains. Each dot represents an individual mouse from the respective strain and the mean values are indicated by the horizontal black bars. Female 5-wk-old mice were exposed to auditory signals at frequencies of 4 kHz, 8 kHz, 12 kHz, 16 kHz, 24 kHz, and 32 kHz. The ability of individual mice to hear these signals was assessed using an ABR test. ABR is represented by the decibel level at which hearing threshold was reached, determined visually by an ABR waveform." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variation in ABR in the HMDP. The auditory brainstem response (ABR) to tone burst stimuli at six different frequencies exhibits two-fold to five-fold variation among 99 HMDP strains. Each dot represents an individual mouse from the respective strain and the mean values are indicated by the horizontal black bars. Female 5-wk-old mice were exposed to auditory signals at frequencies of 4 kHz, 8 kHz, 12 kHz, 16 kHz, 24 kHz, and 32 kHz. The ability of individual mice to hear these signals was assessed using an ABR test. ABR is represented by the decibel level at which hearing threshold was reached, determined visually by an ABR waveform.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-2.medium.gif" width="399" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-2.medium.gif" width="399" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Variation in ABR in the HMDP. The auditory brainstem response (ABR) to tone burst stimuli at six different frequencies exhibits two-fold to five-fold variation among 99 HMDP strains. Each dot represents an individual mouse from the respective strain and the mean values are indicated by the horizontal black bars. Female 5-wk-old mice were exposed to auditory signals at frequencies of 4 kHz, 8 kHz, 12 kHz, 16 kHz, 24 kHz, and 32 kHz. The ability of individual mice to hear these signals was assessed using an ABR test. ABR is represented by the decibel level at which hearing threshold was reached, determined visually by an ABR waveform.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>GWAS for ABR threshold variation at each tested frequency</h3><p id="p-15">Based on the phenotypic data in the HMDP, we next sought to identify the genetic factors affecting hearing by performing a GWAS for ABR thresholds at each tone burst stimulus frequency. These analyses used an efficient mixed model algorithm (EMMA) that takes into account the underlying population structure and genetic relatedness of the HMDP strains (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kang <em>et al.</em> 2008</a>) and has been used successfully for association mapping in the HMDP. These GWAS analyses yielded several interesting observations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2, A</a>–<a id="xref-fig-2-2" class="xref-fig" href="#F2">2F</a>.) First, we identified at least one locus for each frequency that achieved genome-wide significance, with a total of nine regions distributed on chromosomes 3, 4, 9, 10, 13, and 19 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Second, with the exception of the 24 kHz stimulus, each analysis identified at least one locus that was frequency-specific. Third, a locus on chromosome 3 (rs30259360) that was significantly associated with ABR threshold after a 12-kHz stimulus also demonstrated a suggestive association (<em>P</em>∼10<sup>−5</sup>) with nearly all other ABR traits (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Fourth, loci that were associated with ABR threshold across multiple frequencies tended to cluster at the same end of the spectrum. For example, the chromosome 19 locus (rs30354441) that was associated with ABR threshold after a 4-kHz tone burst also demonstrated association after the 8-kHz stimulus but not the other frequencies. Conversely, a locus on chromosome 13 (rs52344209) was only common to ABR threshold after 24-kHz and 32-kHz tone bursts. Finally, one locus on chromosome 10 (rs29362366), which was associated with ABR threshold only after a 16-kHz stimulus, maps to <em>Otogl</em>, a gene that has been implicated in a Mendelian form of human deafness (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Yariz <em>et al.</em> 2012</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of GWAS results for ABR at each tone burst frequency. (A) ABR at 4 kHz was significantly associated with a locus on chromosome 9 and a region on chromosome 19. (B) A GWAS for the 8-kHz tone burst revealed two significantly associated loci on chromosomes 10 and 19. (C) Three significantly associated loci on chromosomes 3, 13, and 19 were identified in the GWAS analysis for the 12-kHz tone burst. (D) One significant locus on chromosome 10 was associated with ABR at the 16-kHz tone burst. (E) The GWAS for ABR at 24 kHz identified two different significant loci on chromosome 13. (F) Two significant loci on chromosomes 4 and 13 were associated with ABR at 32 kHz. For each significantly associated locus, the gene(s) nearest to the peak SNP is indicated. The GWAS analyses for each of the six tone burst stimuli included 182,270 SNPs, whose genomic positions are shown along the x-axis with their corresponding -log10 P values indicated by the y-axis. The genome-wide thresholds for significant (P=4.1&#xD7;10&#x2212;6) and suggestive (P=4.1&#xD7;10&#x2212;4) evidence of association are indicated by the horizontal red and blue lines, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of GWAS results for ABR at each tone burst frequency. (A) ABR at 4 kHz was significantly associated with a locus on chromosome 9 and a region on chromosome 19. (B) A GWAS for the 8-kHz tone burst revealed two significantly associated loci on chromosomes 10 and 19. (C) Three significantly associated loci on chromosomes 3, 13, and 19 were identified in the GWAS analysis for the 12-kHz tone burst. (D) One significant locus on chromosome 10 was associated with ABR at the 16-kHz tone burst. (E) The GWAS for ABR at 24 kHz identified two different significant loci on chromosome 13. (F) Two significant loci on chromosomes 4 and 13 were associated with ABR at 32 kHz. For each significantly associated locus, the gene(s) nearest to the peak SNP is indicated. The GWAS analyses for each of the six tone burst stimuli included 182,270 SNPs, whose genomic positions are shown along the x-axis with their corresponding -log10 &lt;em&gt;P&lt;/em&gt; values indicated by the y-axis. The genome-wide thresholds for significant (&lt;em&gt;P&lt;/em&gt;=4.1&#xD7;10&lt;sup&gt;&#x2212;6&lt;/sup&gt;) and suggestive (&lt;em&gt;P&lt;/em&gt;=4.1&#xD7;10&lt;sup&gt;&#x2212;4&lt;/sup&gt;) evidence of association are indicated by the horizontal red and blue lines, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-3.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-3.medium.gif" width="332" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of GWAS results for ABR at each tone burst frequency. (A) ABR at 4 kHz was significantly associated with a locus on chromosome 9 and a region on chromosome 19. (B) A GWAS for the 8-kHz tone burst revealed two significantly associated loci on chromosomes 10 and 19. (C) Three significantly associated loci on chromosomes 3, 13, and 19 were identified in the GWAS analysis for the 12-kHz tone burst. (D) One significant locus on chromosome 10 was associated with ABR at the 16-kHz tone burst. (E) The GWAS for ABR at 24 kHz identified two different significant loci on chromosome 13. (F) Two significant loci on chromosomes 4 and 13 were associated with ABR at 32 kHz. For each significantly associated locus, the gene(s) nearest to the peak SNP is indicated. The GWAS analyses for each of the six tone burst stimuli included 182,270 SNPs, whose genomic positions are shown along the x-axis with their corresponding -log10 P values indicated by the y-axis. The genome-wide thresholds for significant (P=4.1&#xD7;10&#x2212;6) and suggestive (P=4.1&#xD7;10&#x2212;4) evidence of association are indicated by the horizontal red and blue lines, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of GWAS results for ABR at each tone burst frequency. (A) ABR at 4 kHz was significantly associated with a locus on chromosome 9 and a region on chromosome 19. (B) A GWAS for the 8-kHz tone burst revealed two significantly associated loci on chromosomes 10 and 19. (C) Three significantly associated loci on chromosomes 3, 13, and 19 were identified in the GWAS analysis for the 12-kHz tone burst. (D) One significant locus on chromosome 10 was associated with ABR at the 16-kHz tone burst. (E) The GWAS for ABR at 24 kHz identified two different significant loci on chromosome 13. (F) Two significant loci on chromosomes 4 and 13 were associated with ABR at 32 kHz. For each significantly associated locus, the gene(s) nearest to the peak SNP is indicated. The GWAS analyses for each of the six tone burst stimuli included 182,270 SNPs, whose genomic positions are shown along the x-axis with their corresponding -log10 &lt;em&gt;P&lt;/em&gt; values indicated by the y-axis. The genome-wide thresholds for significant (&lt;em&gt;P&lt;/em&gt;=4.1&#xD7;10&lt;sup&gt;&#x2212;6&lt;/sup&gt;) and suggestive (&lt;em&gt;P&lt;/em&gt;=4.1&#xD7;10&lt;sup&gt;&#x2212;4&lt;/sup&gt;) evidence of association are indicated by the horizontal red and blue lines, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-4.medium.gif" width="332" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F2/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Manhattan plots of GWAS results for ABR at each tone burst frequency. (A) ABR at 4 kHz was significantly associated with a locus on chromosome 9 and a region on chromosome 19. (B) A GWAS for the 8-kHz tone burst revealed two significantly associated loci on chromosomes 10 and 19. (C) Three significantly associated loci on chromosomes 3, 13, and 19 were identified in the GWAS analysis for the 12-kHz tone burst. (D) One significant locus on chromosome 10 was associated with ABR at the 16-kHz tone burst. (E) The GWAS for ABR at 24 kHz identified two different significant loci on chromosome 13. (F) Two significant loci on chromosomes 4 and 13 were associated with ABR at 32 kHz. For each significantly associated locus, the gene(s) nearest to the peak SNP is indicated. The GWAS analyses for each of the six tone burst stimuli included 182,270 SNPs, whose genomic positions are shown along the x-axis with their corresponding -log10 <em>P</em> values indicated by the y-axis. The genome-wide thresholds for significant (<em>P</em>=4.1×10<sup>−6</sup>) and suggestive (<em>P</em>=4.1×10<sup>−4</sup>) evidence of association are indicated by the horizontal red and blue lines, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466985/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466985/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Loci identified in HMDP for ABR thresholds at various tone burst frequencies</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Bioinformatics analyses to prioritize positional candidate genes</h3><p id="p-18">We next used bioinformatics approaches to prioritize candidate genes at each locus. First, we used publicly available expression data for several tissues in the HMDP (<a href="http://geneeqtl.genetics.ucla.edu/">http://geneeqtl.genetics.ucla.edu/</a>) to determine whether variation at the identified loci had <em>cis</em>-acting effects on gene expression. Expression QTL (eQTL) were considered local, or <em>cis</em>, if the lead SNP mapped within 2 Mb of the peak GWAS SNP. We identified four genes as having <em>cis</em> eQTL in the liver: three (<em>Higd1a</em>, <em>Ccbp2</em>, and <em>Ano10</em>) are located in the chromosome 9 locus identified in the 4-kHz GWAS, and the fourth <em>cis</em> eQTL in liver is <em>Naip2</em>, which is located within the second chromosome 13 region that was identified in the 24 kHz GWAS (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467016/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467016/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>cis</em> eQTL for GWAS Loci at Various Tone Burst Frequencies</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">Because we are interested in hearing, we wanted to interrogate if any of our genes exhibit <em>cis</em> eQTL in a more relevant tissue, namely, the cochlea. Therefore, we performed association analysis on microarray-generated expression data in the cochlea from a subset of 64 HMDP strains. Notably, several of our GWAS loci exhibited <em>cis</em> eQTL in cochlear tissue (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For example, <em>Prpf19</em>, which maps approximately 700 kb upstream of our peak GWAS SNP for ABR thresholds after 4 kHz and 8 kHz stimuli, yielded a significant eQTL in the cochlea (rs30899404; <em>P</em>=2.7×10<sup>−6</sup>) but not in other tissues. Additionally, an eQTL for <em>Ms4a6d</em> (rs30768936; <em>P</em>=2.7×10<sup>−8</sup>) was located ∼160 kb downstream of our peak GWAS SNP for the 12-kHz burst in a different region on chromosome 19 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Finally, <em>Fbp1</em> had a highly significant <em>cis</em> eQTL (rs13481847; <em>P</em>=6.1×10<sup>−21</sup>) in the cochlea that maps ∼100 kb upstream of our peak GWAS SNP on chromosome 13 for ABR threshold after a 24-kHz tone burst. We also examined <em>in situ</em> imaging to visualize where in the P1 mouse cochlea <em>Prpf19</em> and <em>Fbp1</em> are expressed (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), as expression in different regions of the organ can indicate different biological functions with respect to hearing. The sensory epithelium and spiral ganglion demonstrate <em>Prpf19</em> mRNA expression (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>); further, <em>Fbp1</em> mRNA expression was detected in the basal layer of the striavascularis in P1 mouse cochleae (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In situ images exhibiting cochlear mRNA expression of GWAS candidate genes. (A) Antisense RNA probe for Prpf19 in P1 cochlea demonstrates expression in the cochlear epithelium, represented by the black circle, and spiral ganglion, denoted by the black arrow. Stria vascularis indicated by the yellow star. (B) Antisense RNA probe for Fbp1 in P1 cochlea demonstrates expression in the basal layer of the stria vascularis, represented by the black oval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In situ&lt;/em&gt; images exhibiting cochlear mRNA expression of GWAS candidate genes. (A) Antisense RNA probe for Prpf19 in P1 cochlea demonstrates expression in the cochlear epithelium, represented by the black circle, and spiral ganglion, denoted by the black arrow. Stria vascularis indicated by the yellow star. (B) Antisense RNA probe for Fbp1 in P1 cochlea demonstrates expression in the basal layer of the stria vascularis, represented by the black oval.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F3.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2329/F3.medium.gif" width="286" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child"><em>In situ</em> images exhibiting cochlear mRNA expression of GWAS candidate genes. (A) Antisense RNA probe for Prpf19 in P1 cochlea demonstrates expression in the cochlear epithelium, represented by the black circle, and spiral ganglion, denoted by the black arrow. Stria vascularis indicated by the yellow star. (B) Antisense RNA probe for Fbp1 in P1 cochlea demonstrates expression in the basal layer of the stria vascularis, represented by the black oval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">In addition to eQTL analysis, we also used the Ensembl genome browser to determine whether any of the positional candidate genes harbored amino acid substitutions among the inbred mouse strains that were sequenced. Of the candidate genes examined, <em>Otogl</em> had an Asn2170Ser polymorphism that was predicted to have functionally deleterious consequences by both SIFT (score = 0.0) and PolyPhen2 (score = 1.0). <em>Otogl</em> is 87% identical to the human protein at the amino acid level and Asn2170Ser is located in a region of high homology that is evolutionarily conserved across seven mammalian species (data not shown). Further analysis revealed that <em>Otogl</em> harbored four additional amino acid substitutions (Ser177Leu, Ala833Thr, Met1107Ile, Ser1258Pro) among the available inbred strains, but these were similar in their physiochemical properties, computationally predicted by SIFT to be tolerated, or were located in regions of the protein that were not as evolutionarily conserved as the region containing Asn2170Ser (data not shown).</p><p id="p-23">Based on these data, we reasoned that one or more of these variants may be driving the association signal for ABR at 16 kHz at the <em>Otogl</em> locus. Because none of the five amino acid substitutions were included in the panel of SNPs used in our GWAS analyses, we used the publicly available sequence data for ∼25 inbred strains to determine the extent of linkage disequilibrium (LD) between the variants and the peak <em>Otogl</em> SNP (rs29362366). This analysis revealed that the Ser177Leu and Ala833Thr variants were in perfect LD with rs29362366, whereas Met1107Ile and Ser1258Pro were much more weakly linked. The Asn2170Ser polymorphism was also only present in strain DBA/2J. Thus, the association at <em>Otogl</em> could be due the Ser177Leu and Ala833Thr variants, even though they were not predicted to have functional effects, or other unknown polymorphisms that are present on the haplotype containing Ser177Leu, Ala833Thr, and rs29362366. Alternatively, the association signal could be due to the segregation of Asn2170Ser among DBA and the BXD RI strains. To distinguish between these possibilities, we performed an association analysis with rs29362366 with and without DBA/2J and the BXD strains. Notably, exclusion of these strains increased the significant association of rs29362366 with ABR at 16 kHz by one order of magnitude (<em>P</em>=2.5×10<sup>−8</sup>) compared to the analysis that included all HMDP strains (<em>P</em>=2.3×10<sup>−7</sup>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Conversely, rs29362366 was not associated with ABR at 16 kHz in an analysis that included only C57BL/6J, DBA/2J, and the BXD strains (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Taken together, these results suggest that the association signal at the <em>Otogl</em> locus is due to the haplotype harboring Ser177Leu, Ala833Thr, and rs29362366.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ABR at 16 kHz varies by genotype at rs29362366. (A&#x2013;C) The effect of Otogl peak SNP rs29362366 on 16 kHz ABR. (A) Genotype effect when all strains are included in analysis. (B) Genotype effect when DBA/2J and BXD RI strains derived from C57BL/6J and DBA/2J matings are excluded from analysis. (C) Genotype effect when only C57BL/6J, DBA/2J, and BXD RI strains are included in analysis. Haplotype containing allele C is present in C57BL/6J and haplotype containing allele T is present in DBA/2J." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-861873935" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;ABR at 16 kHz varies by genotype at rs29362366. (A&#x2013;C) The effect of &lt;em&gt;Otogl&lt;/em&gt; peak SNP rs29362366 on 16 kHz ABR. (A) Genotype effect when all strains are included in analysis. (B) Genotype effect when DBA/2J and BXD RI strains derived from C57BL/6J and DBA/2J matings are excluded from analysis. (C) Genotype effect when only C57BL/6J, DBA/2J, and BXD RI strains are included in analysis. Haplotype containing allele C is present in C57BL/6J and haplotype containing allele T is present in DBA/2J.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2329/F4.medium.gif" width="440" height="117"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2329/F4.medium.gif" width="440" height="117"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2329/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467132" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">ABR at 16 kHz varies by genotype at rs29362366. (A–C) The effect of <em>Otogl</em> peak SNP rs29362366 on 16 kHz ABR. (A) Genotype effect when all strains are included in analysis. (B) Genotype effect when DBA/2J and BXD RI strains derived from C57BL/6J and DBA/2J matings are excluded from analysis. (C) Genotype effect when only C57BL/6J, DBA/2J, and BXD RI strains are included in analysis. Haplotype containing allele C is present in C57BL/6J and haplotype containing allele T is present in DBA/2J.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-25">We have used a GWAS approach with correction for population structure to map several loci for hearing traits in inbred strains of mice. Our results identify a number of novel loci and demonstrate two critical additions to the literature. First, with the exception of the <em>Otogl</em> locus, which was identified in humans, none of the loci identified in the GWAS with the HMDP overlap with Mendelian loci identified for hearing loss in mice. This observation supports the notion that variation in hearing among inbred mouse strains has a complex genetic architecture. Second, just as the cochlea has a frequency-specific functional place map, we demonstrate that the genetics of hearing sensitivity is frequency-specific as well.</p><p id="p-26">We recently published the first meta-analysis for age-related hearing loss in mice using data from several data sets, including a subset of the HMDP strains (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ohmen <em>et al.</em> 2014</a>). The meta-analysis identified five significantly associated loci, including Fscn2, the causative gene at the Ahl8 locus identified in a cross between the DBA/2J and C57BL/6J strains. However, there was no overlap between the loci detected in the meta-analysis and those in the present study. For example, the meta-analysis contained only 35 of the 99 HMDP strains and predominantly powered by the inclusion of N2 mice from a DBA/2J × C57BL/6J backcross generated by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Johnson <em>et al.</em> (2008)</a>. By contrast, the present study sought to identify the genetic determinants for common forms of hearing with the entire HMDP and would thus be more likely to detect allelic variation that is present across all 99 strains. Finally, our current study assessed hearing at 4- and 24-kHz tone burst stimuli, in addition to those at 8, 16, and 32 kHz that were used in our prior study.</p><div id="sec-12" class="subsection"><h3>There exists frequency-specific genetic variation in hearing thresholds in inbred strains of mice</h3><p id="p-27">One of the most important findings from our analysis is the apparent existence of strong frequency-specific genetic variation in hearing ability across inbred mouse strains. Ruben demonstrated through an autoradiographic study of the mouse cochlea that terminal mitoses in the developing cochlear epithelium were first detected in the cochlear apex (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ruben 1967</a>). From this study it was concluded that the differentiation of the cochlear organ of Corti occurred from the base to the apex. The mammalian cochlea has numerous specializations throughout its length that contribute to the tonotopic representation of perceived sound with the basal portion responding to higher frequencies than the apex (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Davis 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mann and Kelley 2011</a>). Critical to the frequency tuning of the cochlea are the morphological and mechanotransductional properties of the hair cells, the basilar membrane, and the spiral ganglion neurons along the length of the cochlear spiral. The molecular bases for these tonotopical specifications along the longitudinal axis of the cochlea remain largely unknown.</p><p id="p-28">The initial observations made by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Zheng <em>et al.</em> (1999)</a> that different strains exhibit different threshold sensitivities to particular stimulus frequencies were poignant and were a preliminary insight into the strain variation and genetic variation of sensitivities along the length of the cochlear duct. <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Son <em>et al.</em> (2012)</a>, using microarray technology, also demonstrated gradients of gene expression in the adult and developing mouse cochlea, a subset of which (approximately 5% of those genes assayed) exhibited clear spatial and temporal variations in expression along the cochlear duct. Utilizing an association-based approach, we build on these prior studies to elucidate the genetic architecture of hearing sensitivities along the length of the cochlear duct in inbred strains of mice. For example, the finding of at least one specific genome-wide significant locus for each frequency in our comprehensive analysis supports this notion. This was true for each tone-burst stimulus tested except 24 kHz. Additional support for frequency-specific, or cochlear regionally specific, genetic determinants comes from the identification of a locus for the lower-frequency apical cochlear spectrum on chromosome 19 (4 kHz and 8 kHz) and of a region on chromosome 13 for the higher frequency basilar cochlear spectrum (24 kHz and 32 kHz). Although a portion of this frequency-specific genetic association can be explained by issues related to study power, we also detected a genome-wide significant locus on chromosome 3 for the 12-kHz tone burst that demonstrated a suggestive association with nearly all other ABR stimulus traits (<em>P</em>∼10<sup>−5</sup>).</p></div><div id="sec-13" class="subsection"><h3>Relationship to alternate forms of hearing loss</h3><p id="p-29">To date, approximately 18 loci for AHL and noise-induced hearing loss (NIHL) have been described in mice using traditional QTL mapping strategies (<a href="http://hearingimpairment.jax.org/table2.html">http://hearingimpairment.jax.org/table2.html</a>). However, none of the loci we identified overlap with these previously identified regions. There may be several reasons for this observation. First, we tested ABR in mice only at age 5 wk, and many strains used in the previously reported AHL mapping studies do not develop hearing loss until a later age; thus, we would not detect impaired hearing in these strains at 5 wk. Second, prior QTL studies were performed using F2 crosses of two strains of mice, resulting in approximately 50% frequency of each allele in the population of mice for mapping. With the HMDP, however, the effect allele may not be frequent enough among the strains used to provide sufficient power with the linear mixed model mapping algorithm. Despite the limited overlap between previously reported mouse hearing QTL and the association signals in this study, our results may still have clinical relevance and do suggest a connection with hearing phenotypes in humans. For example, bioinformatics and stratified genetic analyses at the chromosome 10 locus implicated <em>Otogl</em> as a strong positional candidate for ABR threshold after a 16-kHz stimulus in the HMDP. Interestingly, mutations of the human ortholog, <em>OTOGL</em>, have been identified as the cause of Mendelian forms of moderate high-frequency nonsyndromic hearing loss in humans (Yariz <em>et al.</em> 20120; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bonnet <em>et al.</em> 2013</a>).</p></div><div id="sec-14" class="subsection"><h3>Hearing impairment in mice: natural variation</h3><p id="p-30">Our GWAS for hearing phenotypes in mice revealed nine statistically significant candidate loci; of these, bioinformatic analysis suggests prioritizing at least four positional candidate genes for functional validation. As described above, <em>Otogl</em> is a strong candidate for the 16-kHz locus and, based on cochlear eQTL analyses, <em>Prpf19</em>, <em>Ms4a6d</em>, and <em>Fbp1</em> represent strong candidates within their respective intervals. <em>Prpf19</em> is a pre-mRNA processing factor that has been implicated in mammalian DNA repair (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mahajan and Mitchell 2003</a>) but has no known function in the auditory system. Consistent with the eQTL results, RNAseq data demonstrate a two-fold greater expression level in cochlear hair cells compared to nonsensory cells (N. Segil, personal communication) and our <em>in situ</em> data indicate expression in the sensory epithelium and spiral ganglion. Array data similarly confirms two-fold greater expression of <em>Prpf19</em> mRNA in the spiral ganglion neurons of mice in comparison to the vestibular ganglion beginning at E16 and extending into adulthood (<a href="http://goodrichlabmicroarrays.hms.harvard.edu">http://goodrichlabmicroarrays.hms.harvard.edu</a>).</p><p id="p-31"><em>Ms4a6d</em> is a member of a multigene four-transmembrane family related to CD20, a hematopoietic cell–specific protein and a high-affinity IgE receptor beta chain (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ishibashi <em>et al.</em> 2001</a>) with no known function in the mammalian auditory system. <em>Ms4a6d</em> demonstrates nearly four-fold greater levels of expression in nonsensory cells of the inner ear and similar expression levels in the spiral and vestibular ganglia that peak more than 2 wk postnatal, near the time of onset of hearing in mice (N. Segil, personal communication).</p><p id="p-32"><em>Fbp1</em> has recently been demonstrated to localize within the cell nucleus in a cell cycle–dependent manner and may be involved in RNA processing, nucleosome assembly, and cell cycle regulation. Like the other putative positional genes described above, it has no known role in hearing in mammals (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mamczur <em>et al.</em> 2012</a>). RNAseq data also revealed a two-fold greater expression of this gene in nonsensory cells of the inner ear and variable levels of expression within the spiral and vestibular ganglia of mice during prenatal and postnatal periods); additionally, our <em>in situ</em> data place <em>Fbp1</em> mRNA in the striavascularis, which is critical to the maintenance of the endocochlear potential. Although a specific function has not been ascribed to the basal cell layer, there are widespread gap junctions between the basal cells and the basal cells and intermediate cells. This suggests that these cells make up a functional syncytium similar to that seen in the myocardium, supporting the notion that this gene may play a role in hearing in mice. Taken together, these results demonstrate that <em>Prpf19</em>, <em>Ms4a6d</em>, and <em>Fbp1</em> are expressed in various cochlear cell types and that the loci harboring these genes contain functional genetic variation with respect to their expression. Additional functional studies will be required to determine whether these genes play a role in mammalian hearing.</p><p id="p-33">In conclusion, we have performed a comprehensive analysis in mice to elucidate the genetic architecture of hearing in response to tone burst stimuli. We identified multiple novel loci that are, for the most part, frequency-specific and illustrate the complex nature of mammalian hearing. By combining systems genetics with bioinformatics, we have also identified plausible positional candidate genes at several loci that can be pursued for functional validation. Importantly, the overlap between the effect of the <em>Otogl</em> locus on hearing loss in mice and nonsyndromic hearing loss in humans reinforces the concept that the underlying biological pathways are likely to be conserved among mammals. Furthermore, the results of such genetic studies in mice can potentially be leveraged toward development of novel prevention and treatment strategies.</p></div></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">This work was supported by California Institute for Regenerative Medicine grant TG2-01161 (A.L.C.); NIH grants T32ES013678 (A.L.C.), R01ES022282 (E.E.), P01HL30568 (A.J.L.), P01HL28481 (A.J.L.), R01ES021801 (H.A.), 3R01ES021801-03S1 (H.A.), R01ES025786 (H.A.), and R01DC010856 (R.A.F.); pilot project awards from the Southern California Clinical and Translational Science Institute through NIH grant UL1TR000130 (H.A.) and the Southern California Environmental Health Sciences Center through NIH grant P30ES007048 (H.A.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-35">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021592/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021592/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-36"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-07"><span class="received-label">Received </span>April 7, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-30"><span class="accepted-label">Accepted </span>August 30, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Crow <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.1" data-doi="10.1101/gr.099234.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A high-resolution association mapping panel for the dissection of complex traits in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.099234.109%26rft_id%253Dinfo%253Apmid%252F20054062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.2" data-doi="10.1016/j.gene.2013.06.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loundon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michalski</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verpy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Biallelic nonsense mutations in the otogelin-like gene (OTOGL) in a child affected by mild to moderate hearing impairment.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">527</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D527%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.06.044%26rft_id%253Dinfo%253Apmid%252F23850727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.06.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23850727&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.3" data-doi="10.1177/1073858403251986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gradients of neurotrophins, ion channels, and tuning in the cochlea.</span> <abbr class="cit-jnl-abbrev">Neuroscientist</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNeuroscientist%26rft.stitle%253DNeuroscientist%26rft.aulast%253DDavis%26rft.auinit1%253DR.%2BL.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D311%26rft.epage%253D316%26rft.atitle%253DGradients%2Bof%2BNeurotrophins%252C%2BIon%2BChannels%252C%2Band%2BTuning%2Bin%2Bthe%2BCochlea%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1073858403251986%26rft_id%253Dinfo%253Apmid%252F14580116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3Bucm8iO3M6NToicmVzaWQiO3M6NzoiOS81LzMxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.4" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fransen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonneux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corneveaux</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrauwen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Berardino</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association analysis demonstrates the highly polygenic character of age-related hearing impairment.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet. 23: 110–115.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.6" data-doi="10.1093/hmg/ddn402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Laer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huentelman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheth</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eyken</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GRM7 variants confer susceptibility to age-related hearing impairment.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddn402%26rft_id%253Dinfo%253Apmid%252F19047183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvNzg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.7" data-doi="10.1007/s00335-012-9411-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D680%26rft.epage%253D692%26rft.atitle%253DHybrid%2Bmouse%2Bdiversity%2Bpanel%253A%2Ba%2Bpanel%2Bof%2Binbred%2Bmouse%2Bstrains%2Bsuitable%2Bfor%2Banalysis%2Bof%2Bcomplex%2Bgenetic%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9411-5%26rft_id%253Dinfo%253Apmid%252F22892838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9411-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892838&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.8" data-doi="10.1007/s00405-010-1270-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Age-related hearing loss or presbycusis.</span> <abbr class="cit-jnl-abbrev">Eur. Arch. Otorhinolaryngol.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Barchives%2Bof%2Boto-rhino-laryngology%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BEuropean%2BFederation%2Bof%2BOto-Rhino-Laryngological%2BSocieties%2B%2528EUFOS%2529%2B%253A%2Baffiliated%2Bwith%2Bthe%2BGerman%2BSociety%2Bfor%2BOto-Rhino-Laryngology%2B-%2BHead%2Band%2BNeck%2BSurgery%26rft.stitle%253DEur%2BArch%2BOtorhinolaryngol%26rft.aulast%253DHuang%26rft.auinit1%253DQ.%26rft.volume%253D267%26rft.issue%253D8%26rft.spage%253D1179%26rft.epage%253D1191%26rft.atitle%253DAge-related%2Bhearing%2Bloss%2Bor%2Bpresbycusis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00405-010-1270-7%26rft_id%253Dinfo%253Apmid%252F20464410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00405-010-1270-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20464410&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.9" data-doi="10.1016/S0378-1119(00)00598-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishibashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of a new multigene four-transmembrane family (MS4A) related to CD20, HTm4 and beta subunit of the high-affinity IgE receptor.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DIshibashi%26rft.auinit1%253DK.%26rft.volume%253D264%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D93%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnew%2Bmultigene%2Bfour-transmembrane%2Bfamily%2B%2528MS4A%2529%2Brelated%2Bto%2BCD20%252C%2BHTm4%2Band%2Bbeta%2Bsubunit%2Bof%2Bthe%2Bhigh-affinity%2BIgE%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900598-9%26rft_id%253Dinfo%253Apmid%252F11245982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00598-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11245982&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167534500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longo-Guess</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A locus on distal chromosome 11 (ah18) and its interaction Cdh23 ahl underlie the early, age-related hearing loss of DBA/2J mice.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">219</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D94%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.11" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.12" data-doi="10.1038/74215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winchester</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Large-scale discovery and genotyping of single-nucleotide polymorphisms in the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLindblad-Toh%26rft.auinit1%253DK.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D381%26rft.epage%253D386%26rft.atitle%253DLarge-scale%2Bdiscovery%2Band%2Bgenotyping%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74215%26rft_id%253Dinfo%253Apmid%252F10742102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742102&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086192800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.13" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.14" data-doi="10.1073/pnas.1631060100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Role of human Pso4 in mammalian DNA repair and association with terminal deoxynucleotidyl transferase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10746</span>–<span class="cit-lpage">10751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1631060100%26rft_id%253Dinfo%253Apmid%252F12960389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE5LzEwNzQ2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.15" data-doi="10.1007/s00418-011-0884-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mamczur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sok</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rzechonek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakus</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cell cycle-dependent expression and subcellular localization of fructose 1,6-bisphosphatase.</span> <abbr class="cit-jnl-abbrev">Histochem. Cell Biol.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHistochemistry%2Band%2Bcell%2Bbiology%26rft.stitle%253DHistochem%2BCell%2BBiol%26rft.aulast%253DMamczur%26rft.auinit1%253DP.%26rft.volume%253D137%26rft.issue%253D1%26rft.spage%253D121%26rft.epage%253D136%26rft.atitle%253DCell%2Bcycle-dependent%2Bexpression%2Band%2Bsubcellular%2Blocalization%2Bof%2Bfructose%2B1%252C6-bisphosphatase.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00418-011-0884-1%26rft_id%253Dinfo%253Apmid%252F22057438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00418-011-0884-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22057438&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.16" data-doi="10.1016/j.heares.2011.01.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of tonotopy in the auditory periphery.</span> <abbr class="cit-jnl-abbrev">Hear. Res.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHearing%2Bresearch%26rft.stitle%253DHear%2BRes%26rft.aulast%253DMann%26rft.auinit1%253DZ.%2BF.%26rft.volume%253D276%26rft.issue%253D1-2%26rft.spage%253D2%26rft.epage%253D15%26rft.atitle%253DDevelopment%2Bof%2Btonotopy%2Bin%2Bthe%2Bauditory%2Bperiphery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.heares.2011.01.011%26rft_id%253Dinfo%253Apmid%252F21276841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.heares.2011.01.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21276841&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.17" data-doi="10.1146/annurev-genom-091212-153419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissecting quantitative traits in mice.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft.volume%253D14%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-091212-153419%26rft_id%253Dinfo%253Apmid%252F23834320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-091212-153419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23834320&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326658500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.18" data-doi="10.1007/s10162-014-0443-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohmen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joo</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic-wide association study for age-related hearing loss (AHL) in the mouse: a meta-analysis.</span> <abbr class="cit-jnl-abbrev">J. Assoc. Res. Otolaryngol</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">335</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAssoc.%2BRes.%2BOtolaryngol%26rft.volume%253D15%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10162-014-0443-2%26rft_id%253Dinfo%253Apmid%252F24570207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10162-014-0443-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24570207&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruben</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Development of the inner ear of the mouse: a radioautographic study of terminal mitoses.</span> <abbr class="cit-jnl-abbrev">Acta Otolaryngol. Suppl.</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BOtolaryngol.%2BSuppl.%26rft.volume%253D220%26rft.spage%253D221%26rft_id%253Dinfo%253Apmid%252F6067797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6067797&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.20" data-doi="10.1371/journal.pone.0040735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Son</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developmental gene expression profiling along the tonotopic axis of the mouse cochlea.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e40735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSon%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40735%26rft.epage%253De40735%26rft.atitle%253DDevelopmental%2Bgene%2Bexpression%2Bprofiling%2Balong%2Bthe%2Btonotopic%2Baxis%2Bof%2Bthe%2Bmouse%2Bcochlea.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040735%26rft_id%253Dinfo%253Apmid%252F22808246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22808246&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.21" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.22" data-doi="10.1093/nar/gkt1229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacArthur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D1001</span>–<span class="cit-lpage">D1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1229%26rft_id%253Dinfo%253Apmid%252F24316577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi9EMS9EMTAwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.23" data-doi="10.1073/pnas.0130101100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pletcher</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batalov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarantino</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide single-nucleotide polymorphism analysis defines haplotype patterns in mouse.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3380</span>–<span class="cit-lpage">3385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0130101100%26rft_id%253Dinfo%253Apmid%252F12612341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzYvMzM4MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.24" data-doi="10.1073/pnas.0401189101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fullerton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keays</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Unexpected complexity in the haplotypes of commonly used inbred strains of laboratory mice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">9734</span>–<span class="cit-lpage">9739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401189101%26rft_id%253Dinfo%253Apmid%252F15210992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzI2Lzk3MzQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.25" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.26" data-doi="10.1016/j.ajhg.2012.09.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yariz</span>  <span class="cit-name-given-names">K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seco</span>  <span class="cit-name-given-names">C. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dallman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutations in OTOGL, encoding the inner ear protein otogelin-like, cause moderate sensorineural hearing loss.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYariz%26rft.auinit1%253DK.%2BO.%26rft.volume%253D91%26rft.issue%253D5%26rft.spage%253D872%26rft.epage%253D882%26rft.atitle%253DMutations%2Bin%2BOTOGL%252C%2BEncoding%2Bthe%2BInner%2BEar%2BProtein%2BOtogelin-like%252C%2BCause%2BModerate%2BSensorineural%2BHearing%2BLoss.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.09.011%26rft_id%253Dinfo%253Apmid%252F23122586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23122586&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2329.27" data-doi="10.1016/S0378-5955(99)00003-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erway</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Assessment of hearing in 80 inbred strains of mice by ABR threshold analyses.</span> <abbr class="cit-jnl-abbrev">Hear Res.</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">94</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHearing%2Bresearch%26rft.stitle%253DHear%2BRes%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%2BY.%26rft.volume%253D130%26rft.issue%253D1-2%26rft.spage%253D94%26rft.epage%253D107%26rft.atitle%253DAssessment%2Bof%2Bhearing%2Bin%2B80%2Binbred%2Bstrains%2Bof%2Bmice%2Bby%2BABR%2Bthreshold%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-5955%252899%252900003-9%26rft_id%253Dinfo%253Apmid%252F10320101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-5955(99)00003-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10320101&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079691600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2329.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2323" title="bz-rates: A Web Tool to Estimate Mutation Rates from Fluctuation Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2341" title="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2329.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B94U5NIFXSjUM_IaZxgYZbqdQv4ZJXBRUlfsOnlC_ak" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2329" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2329" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2329" class="active" data-icon-position="" data-hide-link-title="0">The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460172" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nxyI7eTByko3wr1y2L1n9hLs7dlB0uHxMlFVB_oOuPE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2329.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2329" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0t8hRFhB6KDC39CSFf4Y7AQpmzlLEeVa62QfojoBTLw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2329" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2329.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460172" id="node460172--2" data-pisa="ggg;5/11/2329" data-pisa-master="ggg;g3.115.021592" data-apath="/ggg/5/11/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amanda L. Crow</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey Ohmen</span>, <span class="highwire-citation-author" data-delta="2">Juemei Wang</span>, <span class="highwire-citation-author" data-delta="3">Joel Lavinsky</span>, <span class="highwire-citation-author" data-delta="4">Jaana Hartiala</span>, <span class="highwire-citation-author" data-delta="5">Qingzhong Li</span>, <span class="highwire-citation-author" data-delta="6">Xin Li</span>, <span class="highwire-citation-author" data-delta="7">Pezhman Salehide</span>, <span class="highwire-citation-author" data-delta="8">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="9">Calvin Pan</span>, <span class="highwire-citation-author" data-delta="10">Aldons J. Lusis</span>, <span class="highwire-citation-author" data-delta="11">Hooman Allayee</span> and <span class="highwire-citation-author" data-delta="12">Rick A. Friedman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021592 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460172--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda L. Crow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BL.%2BCrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crow%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BL.%2BCrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ohmen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>House Ear Institute, Los Angeles, California 90057</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BOhmen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohmen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BOhmen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juemei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juemei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuemei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Lavinsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BLavinsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavinsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BLavinsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaana Hartiala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaana%2BHartiala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartiala%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaana%2BHartiala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingzhong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Otolaryngology, Head and Neck Surgery, Eye & ENT Hospital of Fudan University, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xin Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Clinical Laboratory Department, First Affiliated Hospital of Nanchang University, Nanchang, Jiangxi Province 330006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pezhman Salehide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pezhman%2BSalehide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salehide%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APezhman%2BSalehide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science and Inter-Departmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Calvin Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calvin%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalvin%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Hooman Allayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hooman%2BAllayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allayee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHooman%2BAllayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rick A. Friedman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rick%2BA.%2BFriedman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedman%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARick%2BA.%2BFriedman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rick.friedman@med.usc.edu" class="" data-icon-position="" data-hide-link-title="0">rick.friedman@med.usc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2329" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2329&amp;text=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2329&amp;t=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2329&amp;title=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460172/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460172" id="node460172--3" data-pisa="ggg;5/11/2329" data-pisa-master="ggg;g3.115.021592" data-apath="/ggg/5/11/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amanda L. Crow</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey Ohmen</span>, <span class="highwire-citation-author" data-delta="2">Juemei Wang</span>, <span class="highwire-citation-author" data-delta="3">Joel Lavinsky</span>, <span class="highwire-citation-author" data-delta="4">Jaana Hartiala</span>, <span class="highwire-citation-author" data-delta="5">Qingzhong Li</span>, <span class="highwire-citation-author" data-delta="6">Xin Li</span>, <span class="highwire-citation-author" data-delta="7">Pezhman Salehide</span>, <span class="highwire-citation-author" data-delta="8">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="9">Calvin Pan</span>, <span class="highwire-citation-author" data-delta="10">Aldons J. Lusis</span>, <span class="highwire-citation-author" data-delta="11">Hooman Allayee</span> and <span class="highwire-citation-author" data-delta="12">Rick A. Friedman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2339;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021592 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460172--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amanda L. Crow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BL.%2BCrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crow%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BL.%2BCrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey Ohmen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>House Ear Institute, Los Angeles, California 90057</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BOhmen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohmen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BOhmen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juemei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juemei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuemei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Lavinsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BLavinsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lavinsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BLavinsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaana Hartiala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaana%2BHartiala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartiala%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaana%2BHartiala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qingzhong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Otolaryngology, Head and Neck Surgery, Eye & ENT Hospital of Fudan University, Shanghai 200031, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xin Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Clinical Laboratory Department, First Affiliated Hospital of Nanchang University, Nanchang, Jiangxi Province 330006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xin%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXin%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pezhman Salehide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pezhman%2BSalehide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salehide%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APezhman%2BSalehide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science and Inter-Departmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Calvin Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calvin%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalvin%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Departments of Human Genetics, Medicine, and Microbiology, Immunology, and Molecular Genetics, David Geffen School of Medicine at UCLA, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Hooman Allayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Preventive Medicine and Institute for Genetic Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hooman%2BAllayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allayee%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHooman%2BAllayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Rick A. Friedman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Otolaryngology and Zilkha Neurogenetic Institute, Keck School of Medicine, University of Southern California, Los Angeles, California 90033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rick%2BA.%2BFriedman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedman%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARick%2BA.%2BFriedman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rick.friedman@med.usc.edu" class="" data-icon-position="" data-hide-link-title="0">rick.friedman@med.usc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2329&amp;rft.epage=2339&amp;rft.atitle=The%20Genetic%20Architecture%20of%20Hearing%20Impairment%20in%20Mice%3A%20Evidence%20for%20Frequency-Specific%20Genetic%20Determinants&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crow%2C+Amanda+L.&amp;rft.au=Ohmen%2C+Jeffrey&amp;rft.au=Wang%2C+Juemei&amp;rft.au=Lavinsky%2C+Joel&amp;rft.au=Hartiala%2C+Jaana&amp;rft.au=Li%2C+Qingzhong&amp;rft.au=Li%2C+Xin&amp;rft.au=Salehide%2C+Pezhman&amp;rft.au=Eskin%2C+Eleazar&amp;rft.au=Pan%2C+Calvin&amp;rft.au=Lusis%2C+Aldons+J.&amp;rft.au=Allayee%2C+Hooman&amp;rft.au=Friedman%2C+Rick+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460172/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460172/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460172/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460172/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460172/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460172/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460172/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460172/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460172/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460172/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460172/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460172/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2329" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2329.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2329.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2329.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2329.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
