<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2341" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2341.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2341.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2341.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2341.atom" />
<meta name="HW.pisa" content="ggg;5/11/2341" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" />
<meta name="DC.Identifier" content="10.1534/g3.115.020040" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hill et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Our ability to assemble complex genomes and construct ultradense genetic maps now allows the determination of recombination rates, translocations, and the extent of genomic collinearity between populations, species, and genera. We developed two ultradense genetic linkage maps for pepper from single-position polymorphisms (SPPs) identified de novo with a 30,173 unigene pepper genotyping array. The Capsicum frutescens × C. annuum interspecific and the C. annuum intraspecific genetic maps were constructed comprising 16,167 and 3,878 unigene markers in 2108 and 783 genetic bins, respectively. Accuracies of marker groupings and orders are validated by the high degree of collinearity between the two maps. Marker density was sufficient to locate the chromosomal breakpoint resulting in the P1/P8 translocation between C. frutescens and C. annuum to a single bin. The two maps aligned to the pepper genome showed varying marker density along the chromosomes. There were extensive chromosomal regions with suppressed recombination and reduced intraspecific marker density. These regions corresponded to the pronounced nonrecombining pericentromeric regions in tomato, a related Solanaceous species. Similar to tomato, the extent of reduced recombination appears to be more pronounced in pepper than in other plant species. Alignment of maps with the tomato and potato genomes shows the presence of previously known translocations and a translocation event that was not observed in previous genetic maps of pepper." />
<meta name="DC.Contributor" content="Theresa Hill" />
<meta name="DC.Contributor" content="Hamid Ashrafi" />
<meta name="DC.Contributor" content="Sebastian Reyes Chin-Wo" />
<meta name="DC.Contributor" content="Kevin Stoffel" />
<meta name="DC.Contributor" content="Maria-Jose Truco" />
<meta name="DC.Contributor" content="Alexander Kozik" />
<meta name="DC.Contributor" content="Richard Michelmore" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Our ability to assemble complex genomes and construct ultradense genetic maps now allows the determination of recombination rates, translocations, and the extent of genomic collinearity between populations, species, and genera. We developed two ultradense genetic linkage maps for pepper from single-position polymorphisms (SPPs) identified &lt;i&gt;de novo&lt;/i&gt; with a 30,173 unigene pepper genotyping array. The &lt;i&gt;Capsicum frutescens&lt;/i&gt; × &lt;i&gt;C. annuum&lt;/i&gt; interspecific and the &lt;i&gt;C. annuum&lt;/i&gt; intraspecific genetic maps were constructed comprising 16,167 and 3,878 unigene markers in 2108 and 783 genetic bins, respectively. Accuracies of marker groupings and orders are validated by the high degree of collinearity between the two maps. Marker density was sufficient to locate the chromosomal breakpoint resulting in the P1/P8 translocation between &lt;i&gt;C. frutescens&lt;/i&gt; and &lt;i&gt;C. annuum&lt;/i&gt; to a single bin. The two maps aligned to the pepper genome showed varying marker density along the chromosomes. There were extensive chromosomal regions with suppressed recombination and reduced intraspecific marker density. These regions corresponded to the pronounced nonrecombining pericentromeric regions in tomato, a related Solanaceous species. Similar to tomato, the extent of reduced recombination appears to be more pronounced in pepper than in other plant species. Alignment of maps with the tomato and potato genomes shows the presence of previously known translocations and a translocation event that was not observed in previous genetic maps of pepper.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2341" />
<meta name="citation_id" content="5/11/2341" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2341" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2341.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2341.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2341.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020040" />
<meta name="citation_pmid" content="26355020" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2341" />
<meta name="citation_lastpage" content="2355" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Theresa Hill" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Hamid Ashrafi" />
<meta name="citation_author_institution" content="Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="Sebastian Reyes Chin-Wo" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Kevin Stoffel" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Maria-Jose Truco" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Alexander Kozik" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Richard Michelmore" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="avandeynze@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=D. Altshuler;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=R. M. Durbin;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Ashrafi;citation_author=T. Hill;citation_author=K. Stoffel;citation_author=A. Kozik;citation_author=J. Yao;citation_title=De novo assembly of the pepper transcriptome (Capsicum annuum): a benchmark for in silico discovery of SNPs, SSRs and candidate genes.;citation_pages=571-571;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23110314;citation_doi=10.1186/1471-2164-13-571" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=L. Barchi;citation_author=J. Bonnet;citation_author=C. Boudet;citation_author=P. Signoret;citation_author=I. Nagy;citation_title=A high-resolution, intraspecific linkage map of pepper (Capsicum annuum L.) and selection of reduced recombinant inbred line subsets for fast mapping.;citation_pages=51-60;citation_volume=50;citation_year=2007;citation_issue=1;citation_pmid=17546071" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.;citation_pages=519-520;citation_volume=356;citation_year=1992;citation_issue=6369;citation_pmid=1560824;citation_doi=10.1038/356519a0" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=A. Ben Chaim;citation_author=Y. Borovsky;citation_author=G. U. Rao;citation_author=B. Tanyolac;citation_author=I. Paran;citation_title=fs3.1: a major fruit shape QTL conserved in Capsicum.;citation_pages=1-9;citation_volume=46;citation_year=2003;citation_issue=1;citation_pmid=12669791" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Ben-Chaim;citation_author=Y. Borovsky;citation_author=M. Falise;citation_author=M. Mazourek;citation_author=B. C. Kang;citation_title=QTL analysis for capsaicinoid content in Capsicum.;citation_pages=1481-1490;citation_volume=113;citation_year=2006;citation_issue=8;citation_pmid=16960715;citation_doi=10.1007/s00122-006-0395-y" />
<meta name="citation_reference" content="citation_title=Peppers: Vegetable and Spice Capsicums;citation_author=P. W. Bosland;citation_author=E. Votava;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=A. Brand;citation_author=Y. Borovsky;citation_author=S. Meir;citation_author=I. Rogachev;citation_author=A. Aharoni;citation_title=pc8.1, a major QTL for pigment content in pepper fruit, is associated with variation in plastid compartment size.;citation_pages=579-588;citation_volume=235;citation_year=2012;citation_pmid=21987007;citation_doi=10.1007/s00425-011-1530-9" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=E. S. Buckler;citation_author=J. M. Thornsberry;citation_title=Plant molecular diversity and applications to genomics.;citation_pages=107-111;citation_volume=5;citation_year=2002;citation_issue=2;citation_pmid=11856604;citation_doi=10.1016/S1369-5266(02)00238-8" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=M. C. Calderon;citation_author=M.-D. Rey;citation_author=A. Cabrera;citation_author=P. Prieto;citation_title=The subtelomeric region is important for chromosome recognition and pairing during meiosis.;citation_pages=6488;citation_volume=4;citation_year=2014;citation_pmid=25270583;citation_doi=10.1038/srep06488" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. B. Chaim;citation_author=I. Paran;citation_author=R. Grube;citation_author=M. Jahn;citation_author=R. Van Wijk;citation_title=QTL mapping of fruit-related traits in pepper (Capsicum annuum).;citation_pages=1016-1028;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220000461" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=O. Cohen;citation_author=Y. Borovsky;citation_author=R. David-Schwartz;citation_author=I. Paran;citation_title=CaJOINTLESS is a MADS-box gene involved in suppression of vegetative growth in all shoot meristems in pepper.;citation_volume=63;citation_year=2012;citation_pmid=22859675;citation_doi=10.1093/jxb/ers172" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. A. Deokar;citation_author=L. Ramsay;citation_author=A. G. Sharpe;citation_author=M. Diapari;citation_author=A. Sindhu;citation_title=Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly.;citation_pages=708;citation_volume=15;citation_year=2014;citation_pmid=25150411;citation_doi=10.1186/1471-2164-15-708" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Fernandez-Pozo;citation_author=N. Menda;citation_author=J. D. Edwards;citation_author=S. Saha;citation_author=I. Y. Tecle;citation_title=The Sol Genomics Network (SGN)—from genotype to phenotype to breeding.;citation_volume=43;citation_year=2015;citation_pmid=25428362;citation_doi=10.1093/nar/gku1195" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_abbrev=Bioinformation;citation_author=E. Góngora-Castillo;citation_author=R. Fajardo-Jaime;citation_author=A. Fernández-Cortes;citation_author=A. E. Jofre-Garfias;citation_author=E. Lozoya-Gloria;citation_title=The capsicum transcriptome DB: a &quot;hot&quot; tool for genomic research.;citation_pages=43-47;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22359434;citation_doi=10.6026/97320630008043" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. V. Grigoriev;citation_author=H. Nordberg;citation_author=I. Shabalov;citation_author=A. Aerts;citation_author=M. Cantor;citation_title=The genome portal of the Department of Energy Joint Genome Institute.;citation_volume=40;citation_year=2012;citation_pmid=22110030;citation_doi=10.1093/nar/gkr947" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. Hackett;citation_author=L. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. A. Hill;citation_author=H. Ashrafi;citation_author=S. Reyes-Chin-Wo;citation_author=J. Yao;citation_author=K. Stoffel;citation_title=Characterization of Capsicum annuum genetic diversity and population structure based on parallel polymorphism discovery with a 30k unigene pepper GeneChip.;citation_pages=e56200;citation_volume=8;citation_year=2013;citation_pmid=23409153;citation_doi=10.1371/journal.pone.0056200" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=H. Hirakawa;citation_author=K. Shirasawa;citation_author=K. Miyatake;citation_author=T. Nunome;citation_author=S. Negoro;citation_title=Draft genome sequence of eggplant (Solanum melongena L.): the representative Solanum species indigenous to the old world.;citation_volume=21;citation_year=2014;citation_pmid=25233906;citation_doi=10.1093/dnares/dsu027" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Huo;citation_author=D. F. Garvin;citation_author=F. M. You;citation_author=S. McMahon;citation_author=M.-C. Luo;citation_title=Comparison of a high-density genetic linkage map to genome features in the model grass Brachypodium distachyon.;citation_pages=455-464;citation_volume=123;citation_year=2011;citation_issue=3;citation_pmid=21597976;citation_doi=10.1007/s00122-011-1598-4" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_journal_abbrev=Biostatistics;citation_author=R. A. Irizarry;citation_author=B. Hobbs;citation_author=F. Collin;citation_author=Y. D. Beazer-Barclay;citation_author=K. J. Antonellis;citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data;citation_pages=249-264;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12925520;citation_doi=10.1093/biostatistics/4.2.249" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=D. Jeifetz;citation_author=R. David-Schwartz;citation_author=Y. Borovsky;citation_author=I. Paran;citation_title=CaBLIND regulates axillary meristem initiation and transition to flowering in pepper.;citation_pages=1227-1236;citation_volume=234;citation_year=2011;citation_pmid=21773792;citation_doi=10.1007/s00425-011-1479-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Kang;citation_author=S. Nahm;citation_author=J. Huh;citation_author=H. Yoo;citation_author=J. Yu;citation_title=An interspecific (Capsicum annuum× C. chinese) F2 linkage map in pepper using RFLP and AFLP markers.;citation_pages=531-539;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220051678" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Kim;citation_author=M. Park;citation_author=S.-I. Yeom;citation_author=Y.-M. Kim;citation_author=J. M. Lee;citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species.;citation_pages=270-278;citation_volume=46;citation_year=2014;citation_pmid=24441736;citation_doi=10.1038/ng.2877" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=V. Lefebvre;citation_author=A. Palloix;citation_author=C. Caranta;citation_author=E. Pochard;citation_title=Construction of an intraspecific integrated linkage map of pepper using molecular markers and doubled-haploid progenies.;citation_pages=112-121;citation_volume=38;citation_year=1995;citation_issue=1;citation_pmid=18470157" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=V. Lefebvre;citation_author=S. Pflieger;citation_author=A. Thabuis;citation_author=C. Caranta;citation_author=A. Blattes;citation_title=Towards the saturation of the pepper linkage map by alignment of three intraspecific maps including known-function genes.;citation_pages=839-854;citation_volume=45;citation_year=2002;citation_issue=5;citation_pmid=12416616" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. J. Lercher;citation_author=L. D. Hurst;citation_title=Human SNP variability and mutation rate are higher in regions of high recombination.;citation_pages=337-340;citation_volume=18;citation_year=2002;citation_issue=7;citation_pmid=12127766;citation_doi=10.1016/S0168-9525(02)02669-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=K. Livingstone;citation_author=G. Churchill;citation_author=M. Jahn;citation_title=Linkage mapping in populations with karyotypic rearrangements;citation_pages=423-428;citation_volume=91;citation_year=2000;citation_issue=6;citation_pmid=11218078;citation_doi=10.1093/jhered/91.6.423" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. D. Livingstone;citation_author=V. K. Lackney;citation_author=J. R. Blauth;citation_author=R. Van Wijk;citation_author=M. K. Jahn;citation_title=Genome Mapping in Capsicum and the Evolution of Genome Structure in the Solanaceae;citation_pages=1183-1202;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388833" />
<meta name="citation_reference" content="citation_journal_title=Sci. Hortic. (Amsterdam);citation_author=F.-H. Lu;citation_author=M.-Y. Yoon;citation_author=Y.-I. Cho;citation_author=J.-W. Chung;citation_author=K.-T. Kim;citation_title=Transcriptome analysis and SNP/SSR marker information of red pepper variety YCM334 and Taean.;citation_pages=38-45;citation_volume=129;citation_year=2011;citation_doi=10.1016/j.scienta.2011.03.003" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=F.-H. Lu;citation_author=M.-C. Cho;citation_author=Y.-J. Park;citation_title=Transcriptome profiling and molecular marker discovery in red pepper, Capsicum annuum L. TF68.;citation_pages=3327-3335;citation_volume=39;citation_year=2012;citation_issue=3;citation_pmid=21706160;citation_doi=10.1007/s11033-011-1102-x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S. Mallard;citation_author=M. Cantet;citation_author=A. Massire;citation_author=A. Bachellez;citation_author=S. Ewert;citation_title=A key QTL cluster is conserved among accessions and exhibits broad-spectrum resistance to Phytophthora capsici: a valuable locus for pepper breeding.;citation_pages=349-364;citation_volume=32;citation_year=2013;citation_doi=10.1007/s11032-013-9875-3" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=Y. Mimura;citation_author=T. Inoue;citation_author=Y. Minamiyama;citation_author=N. Kubo;citation_title=An SSR-based genetic map of pepper (Capsicum annuum L.) serves as an anchor for the alignment of major pepper maps.;citation_pages=93-98;citation_volume=62;citation_year=2012;citation_pmid=23136519;citation_doi=10.1270/jsbbs.62.93" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=L. A. Mueller;citation_author=T. H. Solow;citation_author=N. Taylor;citation_author=B. Skwarecki;citation_author=R. Buels;citation_title=The SOL Genomics Network. A Comparative Resource for Solanaceae Biology and Beyond;citation_pages=1310-1317;citation_volume=138;citation_year=2005;citation_issue=3;citation_pmid=16010005;citation_doi=10.1104/pp.105.060707" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. W. Nachman;citation_title=Variation in recombination rate across the genome: evidence and implications.;citation_pages=657-663;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433578;citation_doi=10.1016/S0959-437X(02)00358-1" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=R. P. Naegele;citation_author=H. Ashrafi;citation_author=T. A. Hill;citation_author=S. R. Chin-Wo;citation_author=A. E. Van Deynze;citation_title=QTL mapping of fruit rot resistance to the plant pathogen Phytophthora capsici in a recombinant inbred line Capsicum annuum population.;citation_pages=479-483;citation_volume=104;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=E. A. Ogundiwin;citation_author=T. F. Berke;citation_author=M. Massoudi;citation_author=L. L. Black;citation_author=G. Huestis;citation_title=Construction of 2 intraspecific linkage maps and identification of resistance QTLs for Phytophthora capsici root-rot and foliar-blight diseases of pepper (Capsicum annuum L.).;citation_pages=698-711;citation_volume=48;citation_year=2005;citation_issue=4;citation_pmid=16094437" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=I. Paran;citation_author=J. R. van der Voort;citation_author=V. Lefebvre;citation_author=M. Jahn;citation_author=L. Landry;citation_title=An integrated genetic linkage map of pepper (Capsicum spp.).;citation_pages=251-261;citation_volume=13;citation_year=2004;citation_doi=10.1023/B:MOLB.0000022526.30914.31" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S.-W. Park;citation_author=J.-K. Jung;citation_author=E.-A. Choi;citation_author=J.-K. Kwon;citation_author=J.-H. Kang;citation_title=An EST-based linkage map reveals chromosomal translocation in Capsicum.;citation_pages=963-975;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0089-0" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome sequence and analysis of the tuber crop potato.;citation_pages=189-195;citation_volume=475;citation_year=2011;citation_issue=7355;citation_pmid=21743474;citation_doi=10.1038/nature10158" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. P. Prince;citation_author=E. Pochard;citation_author=S. D. Tanksley;citation_title=Construction of a molecular linkage map of pepper and a comparison of synteny with tomato.;citation_pages=404-417;citation_volume=36;citation_year=1993;citation_issue=3;citation_pmid=18469998" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Qin;citation_author=C. Yu;citation_author=Y. Shen;citation_author=X. Fang;citation_author=L. Chen;citation_title=Whole-genome sequencing of cultivated and wild peppers provides insights into Capsicum domestication and specialization.;citation_volume=111;citation_year=2014;citation_pmid=24591624;citation_doi=10.1073/pnas.1400975111" />
<meta name="citation_reference" content="citation_title=Plant Gen.;citation_author=W. Z. Rehrig;citation_author=H. Ashrafi;citation_author=T. Hill;citation_author=J. Prince;citation_author=A. Van Deynze;citation_title=CaDMR1 Cosegregates with QTL Pc5.1 for Resistance to Phytophthora capsici in Pepper (Capsicum annuum).;citation_pages=2;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Roselius;citation_author=W. Stephan;citation_author=T. Städler;citation_title=The relationship of nucleotide polymorphism, recombination rate and selection in wild tomato species.;citation_volume=171;citation_year=2005;citation_pmid=16085701;citation_doi=10.1534/genetics.105.043877" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S.-C. Sim;citation_author=G. Durstewitz;citation_author=J. Plieske;citation_author=R. Wieseke;citation_author=M. W. Ganal;citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.;citation_pages=e40563-e40563;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22802968;citation_doi=10.1371/journal.pone.0040563" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=T. Sugita;citation_author=T. Kinoshita;citation_author=T. Kawano;citation_author=K. Yuji;citation_author=K. Yamaguchi;citation_title=Rapid construction of a linkage map using high-efficiency genome scanning/AFLP and RAPD, based on an intraspecific, doubled-haploid population of Capsicum annuum.;citation_pages=287-295;citation_volume=55;citation_year=2005;citation_doi=10.1270/jsbbs.55.287" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=O. Sy;citation_author=R. Steiner;citation_author=P. W. Bosland;citation_title=Recombinant inbred line differential identifies race-specific resistance to phytophthora root rot in Capsicum annuum.;citation_pages=867-870;citation_volume=98;citation_year=2008;citation_issue=8;citation_pmid=18943204;citation_doi=10.1094/PHYTO-98-8-0867" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. D. Tanksley;citation_author=R. Bernatzky;citation_author=N. L. Lapitan;citation_author=J. P. Prince;citation_title=Conservation of gene repertoire but not gene order in pepper and tomato;citation_pages=6419-6423;citation_volume=85;citation_year=1988;citation_issue=17;citation_pmid=16593975;citation_doi=10.1073/pnas.85.17.6419" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=M. J. Truco;citation_author=H. Ashrafi;citation_author=A. Kozik;citation_author=H. van Leeuwen;citation_author=J. Bowers;citation_title=An ultra-high-density, transcript-based, genetic map of lettuce.;citation_volume=3;citation_year=2013;citation_pmid=23550116;citation_doi=10.1534/g3.112.004929" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Environ. Biotechnol.;citation_author=H. T. H. Truong;citation_author=K.-T. Kim;citation_author=S. Kim;citation_author=Y. Chae;citation_author=J.-H. Park;citation_title=Comparative mapping of consensus SSR markers in an intraspecific F-8 recombinant inbred line population in Capsicum.;citation_pages=193-206;citation_volume=51;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Van Os;citation_author=P. Stam;citation_author=R. Visser;citation_author=H. Van Eck;citation_title=RECORD: a novel method for ordering loci on a genetic linkage map.;citation_pages=30-40;citation_volume=112;citation_year=2005;citation_issue=1;citation_pmid=16228189;citation_doi=10.1007/s00122-005-0097-x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. West;citation_author=H. van Leeuwen;citation_author=A. Kozik;citation_author=D. J. Kliebenstein;citation_author=R. W. Doerge;citation_title=High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis.;citation_volume=16;citation_year=2006;citation_pmid=16702412;citation_doi=10.1101/gr.5011206" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=F. Wu;citation_author=S. D. Tanksley;citation_title=Chromosomal evolution in the plant family Solanaceae.;citation_pages=182-182;citation_volume=11;citation_year=2010;citation_pmid=20236516;citation_doi=10.1186/1471-2164-11-182" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Wu;citation_author=N. T. Eannetta;citation_author=Y. Xu;citation_author=R. Durrett;citation_author=M. Mazourek;citation_title=A COSII genetic map of the pepper genome provides a detailed picture of synteny with tomato and new insights into recent chromosome evolution in the genus Capsicum.;citation_pages=1279-1293;citation_volume=118;citation_year=2009;citation_issue=7;citation_pmid=19229514;citation_doi=10.1007/s00122-009-0980-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=S. C. Yarnes;citation_author=H. Ashrafi;citation_author=S. Reyes-Chin-Wo;citation_author=T. A. Hill;citation_author=K. M. Stoffel;citation_title=Identification of QTLs for capsaicinoids, fruit quality, and plant architecture-related traits in an interspecific Capsicum RIL population.;citation_pages=61-74;citation_volume=56;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=W.-W. Zhang;citation_author=J.-S. Pan;citation_author=H.-L. He;citation_author=C. Zhang;citation_author=Z. Li;citation_title=Construction of a high density integrated genetic map for cucumber (Cucumis sativus L.).;citation_pages=249-259;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21971891;citation_doi=10.1007/s00122-011-1701-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Our ability to assemble complex genomes and construct ultradense genetic maps now allows the determination of recombination rates, translocations, and the extent of genomic collinearity between populations, species, and genera. We developed two ultradense genetic linkage maps for pepper from single-position polymorphisms (SPPs) identified de novo with a 30,173 unigene pepper genotyping array. The Capsicum frutescens × C. annuum interspecific and the C. annuum intraspecific genetic maps were constructed comprising 16,167 and 3,878 unigene markers in 2108 and 783 genetic bins, respectively. Accuracies of marker groupings and orders are validated by the high degree of collinearity between the two maps. Marker density was sufficient to locate the chromosomal breakpoint resulting in the P1/P8 translocation between C. frutescens and C. annuum to a single bin. The two maps aligned to the pepper genome showed varying marker density along the chromosomes. There were extensive chromosomal regions with suppressed recombination and reduced intraspecific marker density. These regions corresponded to the pronounced nonrecombining pericentromeric regions in tomato, a related Solanaceous species. Similar to tomato, the extent of reduced recombination appears to be more pronounced in pepper than in other plant species. Alignment of maps with the tomato and potato genomes shows the presence of previously known translocations and a translocation event that was not observed in previous genetic maps of pepper." />
<meta name="og-title" property="og:title" content="Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2341" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Our ability to assemble complex genomes and construct ultradense genetic maps now allows the determination of recombination rates, translocations, and the extent of genomic collinearity between populations, species, and genera. We developed two ultradense genetic linkage maps for pepper from single-position polymorphisms (SPPs) identified de novo with a 30,173 unigene pepper genotyping array. The Capsicum frutescens × C. annuum interspecific and the C. annuum intraspecific genetic maps were constructed comprising 16,167 and 3,878 unigene markers in 2108 and 783 genetic bins, respectively. Accuracies of marker groupings and orders are validated by the high degree of collinearity between the two maps. Marker density was sufficient to locate the chromosomal breakpoint resulting in the P1/P8 translocation between C. frutescens and C. annuum to a single bin. The two maps aligned to the pepper genome showed varying marker density along the chromosomes. There were extensive chromosomal regions with suppressed recombination and reduced intraspecific marker density. These regions corresponded to the pronounced nonrecombining pericentromeric regions in tomato, a related Solanaceous species. Similar to tomato, the extent of reduced recombination appears to be more pronounced in pepper than in other plant species. Alignment of maps with the tomato and potato genomes shows the presence of previously known translocations and a translocation event that was not observed in previous genetic maps of pepper." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RmkiNq9qAz0azUCQeh8DTwXJYBrWbK3LQywOdIsR6tU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460164","apath":"\/ggg\/5\/11\/2341.atom","pisa":"ggg;5\/11\/2341","ac":{"\/ggg\/5\/11\/2341.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2341.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2341"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2341"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2341":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KetKVsBCpVbbPtH9omUBw1lWgdtnpRNzU2NAGWdRNIH2hHlze8tZTs\\\\\\\/9asZG5Gz5\\\\\\\/\\\\\\\/EFjyc3W+c3KY8ULKx\\\\\\\/3491ScUeQtE\\\\\\\/WcrMA8ayq\\\\\\\/m58vvmKbbOAlPJrBrqi8DEcbPY9sgKfRrOjMG5P1pcdrC0GWX7dx7\\\\\\\/7\\\\\\\/NBXdIATGDFcW8JBNCv3jzPdEOP+BoHcxljns0pZ4kSB0k5qXYfks0rKr1kA\\\\\\\/TNgnEwgwiuroAISoPZeDmu8BIwnJvyI6WLKW7cCLEzBJaXiYCbOny3QyP4x4O4U4bb8Ns\\\\\\\/qdc4pqUhnEGUZHdD8rVzVZlKYdCZEFmSSiYViiWmoOALdDgq4H8FkqP069QiRCuNb7ymCxqAplNknhjmiy9as5UuFFODA1tCURA9Lk\\\\\\\/Lki4LtTp6qz2vnyqrbixB\\\\\\\/8H0u3CAwFyGd6xphscEH\\\\\\\/Rl2CrRGw4xqpOMdR8HEMS8nHnw75h7nALBDqTxUxQ4LPTmA1n0Qmj9lghGph8QIy+zq9\\\\\\\/rS+SpbcX4kMZ20eeDFZMt03smZHQC8wD2QCt5zRvGxH1EF9SS\\\\\\\/+dsyIFZ85O3BQTnKk\\\\\\\/F46kftpkNj0L\\\\\\\/BIt1140+XsV6yAp1PuEtzer2A6g9uh6rSUG1tc0g2cX\\\\\\\/lrO8jMA\\\\\\\/gjin+iAieIbIQUzOiKT4GgSFqOckdDNaGEhocfo7hRM6px8kIJF5cWD1E0ZR8PzNOQRwQpHRUcK1b2cPx+QnSBgjqEiCtNtzBDUgMuyI8\\\\\\\/krUbSop5FLGNQGQWxgdvhjCSQzINh2JvEiDNCuOj\\\\\\\/c6j+s2A3kNhiiUc7sZWt6fyGpHY7Y\\\\\\\/tc0oTJu9ix5WRzCoZRoAtdA8H+nSPQ8HiLQwodPaCw9pIMJnI+J9D1T3mDZnKtHyzdnCwzbwgir9c1cnqp87pCwfNHERfSWssEQqxbnsWXO7TDXzVMDl3MPqvhFtz0hAwfsvMwxQcU7zC\\\\\\\/ROfkUgL+MW+G1bpbFuOFJNDkKzsTkX5tE3hHrSXCiyLTyumZQhM2LzdYYuwV8p2wG1o5NPQjY\\\\\\\/A\\\\\\\/pbDb9iW3fFgK\\\\\\\/7NxPa2b+igdx\\\\\\\/HzEzJHGM+sUC\\\\\\\/v60hHsWIaIcSAMf8dWrRPDenL9thRx\\\\\\\/SaPP3aXIlGoYmKdWcAeYOLj\\\\\\\/PYkV2JdLAzIW7nBRJnaAflN2Yy\\\\\\\/IAItHq1WwWY3mAASEDOsTcmMnVtp8oixJi9BklYqLDmTh0tRQADzRBd2zx3Hr1L94od27lEp6OvVQwX\\\\\\\/0lauLfUrZIB08\\\\\\\/+caVo7Mo52Ha1t+2cwKZIlQnb1sCBJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022dOw866AJL9n996o8CydrCA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022649137e5c37502a18810c40e74a1c81f1a54d4a52701cd425e51ea951e8d7e0b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cF\\\\\\\/eqsHoLbTbpc2BPpuEsU3TbjakAsFaHl44rlYivPpShYlG3UocgITfb0CDMbzkdnQH7RkOuaA+VOm79Djgj2TxfVCP0Jx4dMY9R+6f3aasyINnYM\\\\\\\/xbWqiCBYfdFhi3kqhL8U45eE9rfn5FuDA4i9H0bFvyWF3ICVrsleH3kiJs88JY\\\\\\\/QHu4Y2qXuVTOOfglCwP3koRBh6K82QSXEy2B68JZWvA1bmID\\\\\\\/\\\\\\\/fKuOgSDW4gUorgHpQ+4kwNdkPPUK2WS0ugRgyacadBxl3v5uaNNav6Co5BquckQwjjiTVrqjGfIB0yAgJU8USGCdCKiiqy+0cnRQ02wuHYtJ8xQMWWeL1hXYmBfoqiOQPPgRey\\\\\\\/vxCPcyvk1f\\\\\\\/2lT8afoYFw0v1bx34iBdtSKNAZjNuto5PLQV6bBZCUpIfFZkuzt0pVB8\\\\\\\/6CifYeOttk9SZCVemiocKBhyDwYWI5vnVdm6W8OMbuLNCcye88It4x5sshYzVue\\\\\\\/G4\\\\\\\/IqMXkNW1ghUSFauJSZt3TPXaT3gf4PU2TMHj8K4c5s0+kI65jvw2ApjhRpWyCaKglBycBVhZN7drx7bv3b89EM2\\\\\\\/gQsktQudsMu++QhFbglPjKq3sgZaeLkomjh\\\\\\\/3t13JgsMndwOynnkoEgiczLyg5qD5KC9BrjAEOdIt2T27FCUEYCsHt47OyDgVORK3ArQZ\\\\\\\/\\\\\\\/ZclHGI6+jvMgkP7j+Cnywxt\\\\\\\/gQKegBOA2pGg2EYwdL3\\\\\\\/c+jyKYLf0znkNEGCJxGcGthi9FtzbXVo3hPPD1SGMgtGZX8Rj5kmDQCpuvgz45JZ1AOIRzqnNGBT5maQz6YgHLggNba0PLIIQfJnN+RIyb0oI6ZfzOFcFvjvhTNnCjQ1HvSm8RW+\\\\\\\/oDaZIR3CH1XuudnYd5aacmg0TT2yapuoq7BaOxIP1WV7+t6MDee9KYMz3XCLVK\\\\\\\/PCWmT9gzTrAQhFYpAgsdZYfMur1ToInwJHzK8LIKIaPnM4bTG1WKYM8CSLvsclWUrfWs2ctMJbrgrg+nnJum40fMEsVccL\\\\\\\/9NqAVBN5nrGh+9S2xotc0mLncmfyHKf7NIZmkI3AyWCtqFbKa1dd76F\\\\\\\/FHUGJ5qDd8foIM5DNOcsI1noBurFJKWk9bj+IwAJ2WDfglZ\\\\\\\/UenUU+bIjqwZ24ZuWnOGoifkmF607F4Jb\\\\\\\/wMQrPW0YPGcVLYR6Pnc3pXyA6cuXjGTPWvOWnox1KSrmJZ7LeqxF7LrCkFRdBtk2p1vSRkb0fnk6qzCl204j0JGuPvol5tlUgz1irA0gV813ICbZUxtXEShCjHOfB1uvO0zJxdKUIS97Cp0hxF9MPg07WRpInh7bEp0+Aq\\\\\\\/n1tzbUjJdBZh4eseiA24Qrv2qNdMXq76WEjkMMPE\\\\\\\/Rz8vQk9rG7eksHW3PhRPNQclgMeDKaYWfY6uEI6VpPrYbDqi4gPi65\\\\\\\/eEjypefbyLVsbelahYwAnGCJDZu6hDtsjbgMPDebz5JY+NIc7E8ECswzJ81ipoYhnvJFTsnxZy7JXs95JZn5YKqhu3QdQmXphp9mV0iGfGxgcWwjXF3wRr4kGumdQeVqjxxKPJH4CLnXHODpdfWTNrq4zUzJHR\\\\\\\/t045HkDzrDvf+3oFwvM8lOAElyKEwtARMHhUDUa9MrvMIKeRHCcuEzBk78nA1HNZEYF383K5zooTCAyOOpP5Ej894E2Ob\\\\\\\/J79frnJu4EAVB0XzoiH6V4XJfHI1tFfONfU7dASSJDovgy7OdRSNkNw4wrSNGqH2b5oAWZN0ReDKf3ws3B5mPz3ckCBd2cGf68VJoIOpL9PndJdveh+XHxtOrbyLIXXnDslTywFDKs9iQTganncoDzOIa2pYtACQCe2fYeKN0AF4\\\\\\\/hM78pwAIu\\\\\\\/aaxEXDT9Ew1e\\\\\\\/KhnUtnmSSuidPgwvLJr\\\\\\\/CBKHFGDqo\\\\\\\/AkHdHdVnTYob6n8sU1brKZwXXTKQab6yzSSRbVkWOJH1Vo\\\\\\\/85WpjNyFxTDvZuMEstJqOkDQC6nO+i3SfM2DmRVk\\\\\\\/FRkeF2GPbhHIMHElU\\\\\\\/AL6J9s3Fn+EZ6S07BB0RZI4\\\\\\\/eaDGAkUF1AMNWbmBLjrcIG0Yr0tSIfyoeE+nYASLtRPnAmMnxV\\\\\\\/nmxRpAFgjnd0v5tPBxX2jn62TQnZvvlyohEWDRk4gV0ImNq\\\\\\\/iNvXtc6ZFxqB9wWF8JCe\\\\\\\/AF4Bf4TCEhWahF38SvMCxtlvCm1NabGVjFQnnWhLTrdeQockkwUp0cc+5DAZG3vRt1titXG+gtq2zShyBiaes2PAq\\\\\\\/qpd7DA8SAMPX\\\\\\\/FGUxA2NnVe9F8IbUxWKErJ9Z71ijJvnSuXGywlma65zfvUUShkykAOKzmrPrFQsmtlJjZrnBMJaMtZhLd3V4RRhtiwDtlLUP4YXSRu\\\\\\\/yb6SaSUDbkhvhK96eSekCe4BrfuPO8CvbFclPj1T1\\\\\\\/5MTxHGDPlTnnw41cKFsneLsXZljTsl9JZ9WLqAOWx51ntBcQ2cwhk01fyI3e1NctNwkk2RQKFMS6sLSlpU1Qw4s\\\\\\\/WZm9iUp4gdJ6XXEbYzNyux6h6xVUk42GkNkZV6VONSLu1J\\\\\\\/1V7zgp5Jnq5WlLEWX3HxawlRvHY9XJLMc4QHPIcAfpsrbQ68Ao5COJapXsIuJPQeksyZjDfaQNU4INxnEADn1ngkZPpVuhUwTVuCXdRrqLN+MSgAIqebjS0S0CRsRy9FUALzLL2skTm9BgFdi+2CTvcfBqhg5Bt7dF3+0IA8lvA9OBDcLekxX7E8V6sz3oIDQPjHXQy\\\\\\\/C7RqOXp1vdpw9exGWiKhM1T6gfZbUAMc+kBhmNhYasvXR5Y3fbNiNJywh43BbeHdDII6rh08ExxezO1pL27oilO0Tyr\\\\\\\/Fm0GCEVq9zfDKYMJ5PsGdaX\\\\\\\/AzS+nLUFnpCXLRPPhlXsO2nQNqvI5WnzqANY0+GURHxVHbuMM1kSnUFfgmG87Wxayg1DeScbu0s55JVoF0ITjW5mD3Pc6EWFtx\\\\\\\/sUHoY6ZTqRGndtcc7riUgjs9SOUvPRxYJaLAXM+CefQwocNrNru8jmTimdyAsMYtTlpJknkBYCDHkiIOHSDZm62lfNUg\\\\\\\/HcKzAzUsYmgCejMhGtEqM6BaUaWnTbZ1+ihey513guzRGPSumNtoZewS+9ZawEnQ26eSFAiT3iTgS4VLnOvf\\\\\\\/aBmS7Y2CmY2PeOzoMcGSgbTk8+RsL941YseZiV97q6HkFaVAO66hlmSPzg3Lq0vIeaFcaWBTXRAzPKb9UeOV00j0ySqhImJVE1W8my4tQZcK1SKq5xaAIL5RAkeuJeMlhsGCaq8kzQ3eeJMx+dyXh5V37SRG9vWAv7FOv4CtwItB9aPMsygTiM4PFeVPjteYlGZss\\\\\\\/wPjQ3oVwaeDNg9+t\\\\\\\/I5+W7h\\\\\\\/IeQSHq3jEaz8ajfgcRvA0m1EHAIYRMWNtawzhEjMNOUmbEa68HXa5WD84Z79RdMVL8+k7H759QEK8GQTcQZpQw4K\\\\\\\/ManBYy8Kj1MfEgm9ftOV98v4M2mEb6XJ3lV6PQAQFD5TdEcqE4XU0Q5a+l59Jp51zyvPDnwpDdjaw\\\\\\\/NgnsJAApsupri+BQVm51lN64EqAJPJ4RBRrbLd86vmnrKaAIYSx1FLS8\\\\\\\/B8Z3TftME312WotLKMaKwGz7jW9Hj4+LgQi\\\\\\\/oP7Zca3XaPgkNzoa3xK4BjP3tRBomSL4mW2SGzZb1DSd9P6nDU6B\\\\\\\/8MjbW6vxrvA4dh0k+rCpY425bl0DAAauPj7sUBkezlCrPqUW3i93tF8gwBfzU2pOMbK76teJjU+QfMB3PIGzGceZCCEMIrW\\\\\\\/M1r0tMEu90DMie6nslMV7RA4EJGDZchasllP8M6ZVAFiMUXwJ\\\\\\\/i7CfnU5MIQrt7bJRcz+V+t8Olp+gL3MZY3C11WiZRaOXpbr8xpF3z0K0QAJCU2p6aXM2tlnwP2n0LJY6ZL7oJwaf5kqcl\\\\\\\/LjLoAen8tWmb\\\\\\\/KI6qo0dmNltr00GDz+oXSQ4l1mt9\\\\\\\/o01aP5ek6ABGWtCssR9k8a2UUTg\\\\\\\/C7SlzmH6\\\\\\\/RzmRr+9zlpLnlWwNXOPKXtP7WdBoNtz9mxJPr\\\\\\\/YlvAnyAeul5gO1wdchYgC3evVnBcjVYx6GsabCAEq9HCgPBmo7Zm68ZZmDSmu2y5yLDNnqhPIgjVx7TnYnzIgUpjH6dLLjmnbS\\\\\\\/EEYz2C0jHeZ1gA7N1LVByCmWDBvjFujb\\\\\\\/IATP0EklECk7zYf4x0f+uzVesuOZpaeXEC5myh1nCsRqgBEZEKNcsSpFbc2NmUWWEQa4WvqtLYN7JBIe7x3n0QbTbq\\\\\\\/OkZ6P3W4Xtndhn1wkecdQ73u59f6+3yglTnv0tWZPNUJeje0FZ1HAtANdjzDws0JejRMJMCEVuqRka7RQmSkZSDJjBs0gqVsWQY\\\\\\\/+vfEqzRZKe2YLrJkoPFQ15M9TpZD2QSv3j6sNQkCle2zEEZ8Zn8HooE82yo6VBtbA\\\\\\\/HnSOk3wnPVzO85LFeTLnEcgxsKldPrCpLjK0SIVeac3tq8fTLonm0pzbUWW+zQaliOL5qfny87xN3v\\\\\\\/Mzsc+dWW81FwRNGKu1gB0qI33pSHKQWEamp7hgTLm5YrGA5APm99SfztavpPTrxmGdd1+ih0KtH68JIIi4AW1RYeWbGWWhFeq5c3hZiT8IApdSuELGEerhBRl0dTJyOiqWWq9MbZ\\\\\\\/hUb9FXi4hlRJ23SsmvwXLHnsJg1JZ7X6yJKHpdzmiukFlg754fQROjm3II8pwaUHHSVhQKnwUiGmqKzMuhnJJBqqNVox\\\\\\\/eisvN9\\\\\\\/8EnDd\\\\\\\/o+qiqQnZjsUvpN5CKGdm47nfIm1iHQe4KbPt6QXCLlxbns7kiZp2J+ZYq85xRzpTiQq6PeSY7MMKwDYphaaO1vmbzLHOYFwVw4ZLlLpIs+81dC5zBHzlKbvWcLK3Q8WVBynDDhWDHyRzn90CPZsFNhbEmUMaLEV82OPnIfzKDdIuOekBHOIc48ivlru+ZtVtYlqml6Wv4aDsJFT+4Ov3mR0a2+txNq2e03XcVKZoRmQ\\\\\\\/H6YXMlLjQkL3fmRzqCnqm+9gUd5e8NIWZ34mm65I47X19YtvPKuwbYXg09NuhVR4fFA4rOhQS2q+ijRCcXuU81fBK0sYsqQbxiCFDWV3\\\\\\\/Guoq0NxdJOwrL5fs8IWHO5QTPgH7xMzuhimeExqFDa9\\\\\\\/61AGwge3mchJ094qFRJS2dyTr1TiLkPv31W00C09Z5aCISLtk95lQY5U7OOEvRIRBdR6KkByXqdnSUV3fAwbhA8lEFVPRt+Izh6hN2\\\\\\\/YKHinWVXL1\\\\\\\/vxL0sRAOvuPHZ+nQLW3k7q0OoW0Eu3SLdt\\\\\\\/uA9K9crLqKFoNi\\\\\\\/KLTp4b3emG\\\\\\\/hfgYtWHDwTiqwkgKcIr0nQQyTxJqKmiy1rp+VB+lNXE69TvOxVh9iNXWW+X0kuMACskkfB\\\\\\\/cbS0GEK5wFV348J6++Ka2XhaJTj4ijbj8ywqhXYsehJWFYYR\\\\\\\/Ysbhe84wFx4fdaBxavcYfBdKyibQvWUbxufF9VKxUM0C5Jd90vTK5E7zTmlH6FK3EU72iPtsOeIyj+pwZ8l6QDOe4ciSSFREW7eFseChiSf2ayFjTZqvRL3ddvuL+qmelSwKJv+WJ891wAa\\\\\\\/uvf3cEdCHT6ecaJj5Fpidxe6yxt1PsRk\\\\\\\/cMzrpTep0r4l0MW3XJJ3k\\\\\\\/f3+3QttrG8jNfkkhCxrM2x+VCH4oFaaoysOszxNsTzD667HeyzzblwS8OUmnmy2AnlUbUcTIf1iCIiVDvBi+icWYU99+UykBGQb3O7FwGD5OBJiSU72LyY8xO98Ypn8+uWvfJAyQJexB56xNn0zulyFadG3XROoQsHLwbKfO9tfiL+IiG32PEBlRKjeHuUtkC7JreBl6QNUDDHNOSt7UkudSceJ+m6bdafVHQ7bVt3yobUuNzTpszjFOFgnYdIalzZOIzQ4tPxvtJK0eN219gdPtr5uuNzws\\\\\\\/\\\\\\\/\\\\\\\/ZSAn4hoGRTI8O5uI9e0DEmRbhY9I93050eE+GQZkOpUmFHyk43fWEsmTsoi6U6o\\\\\\\/c\\\\\\\/G7gCKBoZrIv+G8hQqLDMzJ31zAgYFKTEA9i6DUgkZ8xfW5JWXXXUBslOlpBjNqSGn6QrVd\\\\\\\/8nk4FHLwB3r1dHoJiUJnOQYBDGRUGCCSxcgblrMb2LH96t5wjUvL3SsQe\\\\\\\/jaEhGFJNmXLmbwD\\\\\\\/DBtKYpmPn+U4z53xOunVn0WkeXK5ONp1iuS\\\\\\\/ai4H\\\\\\\/RQeBOp+LbQt\\\\\\\/2x6VDYj\\\\\\\/0bwt2qrqSCsQg0J8WdhfCdQdu7e4HOYcN+87i3TxtCPNY6V0Gk5JcXoIrp+tEema8RPHvZpxqK9yTHC42IlJN3ZdWHSnb8eogYR9y0vzI\\\\\\\/pTdR9VA5lU1ehUD5xpSBy4y+v9Od3ZCMNz7gGpTj8TR04Vi+HCvcDNNQzFaBEMehtH2tHZMGim31LxVKOjJHLSQxhXEAuimN58uC9DYpiYDw6h89tR5+HC6OXHuFEiLDVFrd7joZoAq4zjb\\\\\\\/WBvOOIWo+SJ+TOB7pVPNzgdYRUcR7t48b9E0fUtg\\\\\\\/yETg7ONrl6myKhkrTlSIM8s1zVTOQI8w9Dz56IiFO4eC5l1MGayp8QIgw42AtU3jSp9migwyzOpUVsF9LI3KWByfcNHr1elYeVoyNKz5VN5fEpV3iAH7KgPHj+dDBCI3OuhsYhXznmSPe4+8HNd888V25JR1luD9FeomWKwLa5kL57IBnwGOIEqzbeHfa7qEcxM7\\\\\\\/\\\\\\\/52xwtnDcG47gt3\\\\\\\/FwB9SBfydAubumFpdYRc5uMegCJSvuETBVDPDTHvLrKnPQsTM2WeSwGRz\\\\\\\/jo\\\\\\\/gMXa8tgRO3jGrjUj\\\\\\\/ef+UCJowL8rRmlGn\\\\\\\/205MItHZO\\\\\\\/HsiGPEJAJDxMhCEC5Em7HNFVPN13RVEFu\\\\\\\/z+sf55A\\\\\\\/eSPQXP8GQGp5iQgl2OwXcHdvGXj4\\\\\\\/Nnu6KskmjwlrpGvvLJB1Hb3EUrlwoStb9xG7Yt9T1CJroR+\\\\\\\/9A4F8COzMJr7aS3SfhK8z4F5bNWkE3HDXSKFVoGQDTtEtc++0cdD9pY8dFISO7rvGboG4kpWYkgd56It5QzLe\\\\\\\/JQlvmLwla59IBo1cAAs3UIUhihl7SvAxOIcpKyHA8+\\\\\\\/v0h9PDLq0GHXKP3yFmaodl1azovcLxac+ub3kPRyOmlp60mIITWTc29ZlpCilFHv6AR2UqtoOptNvG5qLyj6MxGzg79FD+ooy37zcyrPXBXEg73oPL1S7BmgpHR+O4CLyjK1AXeHXHDiSS97McjS9eeCQqojE84bsNaRQS7i65RvrJfaKazsJstHiRrSkiTEAO2e2afoUdBx+ZyyVjRrRdrdgzohMFMCIsf9LAqWIu3xU8jkUsZch1HgLg3I0AtlhHQpg7Yc9fsS0sHcq65mDfAFOzhXKfSxnFB1Wm9qqO3UuOB+aRSH7zcI30vOgvpBUTh0q4CRcJ3F8ojQAevI0pgFctTK1tTSECKvs+kWdaxP+dyv7Hs3uebQyUKBupXwxFwMlbr83p7d2fYjnUHPng3YOjvCqVbi2mbLoq+qxep6LAphFkiVXrkhDewME+Cgu3kcZeuuIm48x8w4AJxIQ7yCceEwzVV6fAflvDbzEexljNycLqq5Bmd0TRk+QAglhXWU9XWyrBnsrR4W33EsH48ORlA5yTg3FapFW6M5pZk1SFlunLdyKQ2PaYr7WmoTZnT1keHucr9ge+DskS0b+woD97JCyNv8oSvIWAVS2XW1cAerVeQ5u70VZIbgINePU5MhOHj0bcMfFZCkR7x1ZO\\\\\\\/XFp3+JD4R1CIx8UC7xDSeO0z8\\\\\\\/uxl0QJDNnR6SEoyBuH3DNrIYp5Rg+ECQi+NQ+o9geEoHj7pj5AsiOjJt3gFUz334LbFM3jXG+tAUA5txRuVeQl+CfAqche3pTPJ4BUyykuaP9Ysp4g9dsT4B93+MhRWZRg25DAQo\\\\\\\/BA+rpSHcvAY1s8e38kdgyV1h54piU5qxCAJsRhBg4y6eLZOlte3Zz8ikuAPPFjQp69JQqW5LBcqAcDo0yfS7H2aR71XkttPvNEjE26O5lChPwOuU9BReq0+M4wYHcIMfoaqcWN9VvNOL0T9qvy7AAoFdkHQYsHeFFC3i8o26zo\\\\\\\/RJy9adUty3JWAGYQT+7ADOGz4fv0K+6WnytMr6W+xX+s6ilIdJMsmj4FBav8XVEoMqkb\\\\\\\/D4RjiS9Csc3Ob3xTIiuqmJk8+jLK22BrW4DF83Y31Zg3214jEDn\\\\\\\/2iSFOh+2r1\\\\\\\/gXEW8NvOXd2UNGDVAqXQg5DI5RAOQ0ygzwTjGc3iATADlcYf\\\\\\\/8j1HoMivL1VMtZTu4qgjQRI73p\\\\\\\/DouiDrXWmgvLCdrIOo\\\\\\\/bHfilQvUHyXVcCrTFFIUnCVNqhpqm+L07TQWDSunGCAddQ+8yiVzvtAS9YmMtjRf7z+yuKMVy5EYy501ckno9RIB8+YgnZAWATkUYAX0E12zcWkGMDQBviBKDLRmdS9uoR7msM\\\\\\\/x78J5rfLVw3PFbjL7ySY7DLr1GuAwGvZztFnJqwHIvEH2Waq8T\\\\\\\/fyI86Q66lMqGhlXR1\\\\\\\/54oqkfceIHjhm7xQvpBouyJdhBC4q4vNr4dl+YUUI6GTD8f3l7WI7hL5SPZywSEbPs0NCdMffAXR5pq9aiAox0YWQI7ywxSBTWwh7T8HStc0bJY9lU7gyG1qVZimJwCLbzz9oP1z\\\\\\\/LWSg1NIUMUe24rcbMzxVcbpLC0GRJtfVNBD+wxjO4uRwOYoSrwS0Y6GY\\\\\\\/CT6m0aeRFMweEDsgDN0frrExQPpTiwLALFK+KWmXhrEBxsFuK9pyJW+a5IgpYMEmKd0Gigvn0CjP54tqUPy+8rS\\\\\\\/G4Izlg4XSiqa7y7MMqunu9dJmrA6+xkakUrIVeooodKTkbbVao8fPT6rlvqGC6VsUMzbqzsaxBXe718Siv2BSPjg+1a\\\\\\\/GqdzlHRpSg3IUskXfZadeH6cPcZGVDH1p\\\\\\\/eWWS\\\\\\\/lzuKHqgfXELk9kAZ\\\\\\\/ptjQn1t57O7OHrib8sU6P67JenMzhwkOlr5fwzjE4ICyN8hc95O\\\\\\\/hZf\\\\\\\/42EjhF2sQdGOLFTdo6JOLPnkAATueq1LQM4HBTGbPTAo0wlUDytjLNvHHQTHrrG0N8u5yFyN4VXYB6pqokuBmGzv5XjsZqz\\\\\\\/ompPAE4EOOT1g+yiRG\\\\\\\/WtwTHz8qC37xu4yeTWgXBRhWxTONRGLgrco7gRYT+B3w2T+xXPnNhZimcwPb2bbXfmY7jTCFpFsdbPHqCFNIhh+ZKiCiM6lwikwvk6SMwNQr6028CYW1dd4bWrTyx1S5WPIObEZ2T8B8qeio4tc29I\\\\\\\/\\\\\\\/T6h8e7Kd\\\\\\\/+TGMgEY6x4kfVhYg8hC6ZmWpgUCkMEjj+92cHnUUyszaSpagpHQ4KBnQ\\\\\\\/DhW9J3n0yMclNtGY7LENmtPRe6v\\\\\\\/jYkOvuap0Ryhmu+VHT1g9GLUHXZKcDtRhZqFwbCBfJMbna+R7JRcZmk8fkoNnqrdqbopqxt5EvJGqLk4x0tYRcGauky\\\\\\\/3Q+GtBH\\\\\\\/LiUMG4groFxllELEqtAN4rvTi\\\\\\\/YcERHuAGMmaMLDA3gs\\\\\\\/vvau5oHvs1QImfxA3jXgpLoYrczr8UlDxJDsBlPyQlRn+2LmTKIcD\\\\\\\/\\\\\\\/1u5DhEM+WOsKV4YaNueU8bWk2+DDmFHDQCnSsrbzJe4WmJE47oyF3SP8EbVSk48djOEOVQYZxNnTO4BsA\\\\\\\/yhc8ofux9VwpzTPXf4Rgmb464zbPhwVUDlFpinm7bgh+s800qhyQYhx9dhuiKBW5znjYc7vlkCN36R4qVx8PNifAB9oeNra\\\\\\\/ZH\\\\\\\/2mu2b3PliMO48f\\\\\\\/84\\\\\\\/lOMLNhQRKX4dV+d0d+GzQFHtvyHtmTsywX\\\\\\\/ZguexM\\\\\\\/r5zBFHKHnRPv49o2hUzYiV1YezjPrZCz+\\\\\\\/oNL7RdZi\\\\\\\/TpU7L4r1N4z8ZrAFhGrQpiZS0e5Lyl3xkOiX0CPEAwqKU4jWolctrdJDCAZknocD13bp2QuAd\\\\\\\/GvcZkFo2zINuSMJptlaGMOuWVbU5O7djoQsmYQh\\\\\\\/uP7Zx\\\\\\\/E8ZImSPSzPHs0LupkE6kRqJLrReKObJHZLZbMKhNXUTke3C66m4RBPYExb0MYjHc9SHKHg2Mq7pO1gDmakhQYX\\\\\\\/HYRWYWG+abr5zeQ6jo\\\\\\\/+xxlJSl4ojdTioZqY\\\\\\\/TS7JscoyxHd6GdyJTkpapXKlxAQkaHo\\\\\\\/otjZfm5pvOSbEXitU3P1j+zFYb\\\\\\\/ocCQ+33I0JEYldvyb4xKvlQfBN4KAWBJOQ7NI89BKE8TYmBonW1hd+5AOlX8wD+3\\\\\\\/z6ctseCBJekjrFCjdr8ail8rVk+N3kRVfUdTom96tqZkQKfJsXsOSk2JKLv\\\\\\\/10n3xl8J6LSUAx70V4rOQr6DOFEra7Fgp8h1LHI2FFryFX+f\\\\\\\/5X73BT+DR1sS3RTY85cACukr0Lk+FDy+x5bkA8LjeCtWWjIeaVL7hh4gk8560VPYrRZOkmWvuuwx27taYTCQMa0vwAjIQSElhYTTwBIApU\\\\\\\/TZNvLUUKK7ohvEK\\\\\\\/GG6tFRNjJ1kLHBu9wU7qKxGmMVwlmb5Ayw0MzDczvnzEMrxQVUrM5Gi7I4+QjdNnGAmoD4Q2vPhr9M6sUH3y+OvZMmGs7hS5RBPSsOlBNbxyUvcmhN376ZViAcfCAY\\\\\\\/00hSUpadgKHkIltGFuVsiBl55TH+xcTjktb8dG\\\\\\\/uOe\\\\\\\/QwTKgF9z+\\\\\\\/823T\\\\\\\/r+\\\\\\\/9AuAXQMx72rvEfd9sfmAXYx0V1U\\\\\\\/gVTGJnk\\\\\\\/ecE2uXqap+kae8FVNHfNN\\\\\\\/Mn7vY0H2dUvc46RhrPadCFC5M+eZEEKAhkspjOGBvJsDQa4Ex2PRiuXVVTiy0HFXROcYh\\\\\\\/c8qcVptIYhMeD7RWqb+ekaV\\\\\\\/XGDhrRIs1RhD3BKojyVvMWUJcAb6yaJb9IQul5EQlA95H9+9NaKvyAEM25Oe5QAjwKn9il+WSq9vdJKRAII+TPOQYFl5DKozQtvjInmyv6ubG3A1cBZaajuE5l3oP4sYD22oQgC8DzfNRjkKbSv9c56aKVTVqtsRCUBPdDdOXRltAot61d\\\\\\\/cu2UWTZg+Jo8zFJgwEdE0X8mQsRzb56w7acoTDkb0nkQx3BA4gTncEcblBsAauhwOdSbeQ8BSIdBgol8kju+8Uk\\\\\\\/xJ8ANJ94L3aWR2cxgVzLXy+edT7O7hPZhg\\\\\\\/TA71JiQcspefS\\\\\\\/ldnvp9TM10+KeyElfe3+HOsmiM523WXakmtS1bfkE2VHAt3G2arnGbqf3rsLB1dzwKlokS5NxqdnChTBMRgJYSnZ+tMYUBc0UaFMLBZIGwWo158OWI4XxUamEfUnnSMbm6E13O6wCnph29C1uXV\\\\\\\/1YWcj80pOO\\\\\\\/48Ulp9QwGdqhw26ifhns8NvXZS43svMsFroAt2fcm7dK+coRLC0Bh4apOSlIwWTZdpTDbILLJkGeM50QLyPInq3Bccv1A2P\\\\\\\/8NPJ+zdSW8iL3kxR4TzrDyjgtSkkcXO7OHbVG9FgEIF3X+7JfQoJm4JgwRhJmN+cYK6L0PwKJD\\\\\\\/yAf8DudVi8k98o2vfZtaPCN54uJpatr0qnBze\\\\\\\/AmsTbs1CI03a1Q7749DKBIzySZ0VBNoMIcy5UR8mYQoeW9GFIoSionvy2VWRslogzndCZ+4CqbLN4yBehjhQnKj1DNFV\\\\\\\/XpxkGHmE7W47EP1ZWDiDP0PP4DBvTL\\\\\\\/kwBjOG1Rk2ZaCuGMVbNUFy6aVNDWHXZ7Q0yHzwfUbCrmiNtLBApD\\\\\\\/nr+DK0eOrNxg01pxaY4YtygoRMdYd7Wl8RTflHewkxLtP6SJWrO6sE0uJ6S\\\\\\\/2CKXW8Je3xC\\\\\\\/xlA3vEFnakwFwFs43SoAC614EyavF5RT3QSTcarO0MEhSpCmzQxdgFJHYnSG7YTuPXOKl3OXAoPV8ERrxxxdsv\\\\\\\/ku3qheriQnW\\\\\\\/3c2Gp\\\\\\\/ymvNu+pcifR6kSHrxwzHE2XvahYOJ8viec45J7J5OeCT700dSFQeUI61yQtnNvtIfgrinL59jsXCbhcOroLQioNo1X+4wK1A\\\\\\\/qbZniiR\\\\\\\/LccZkFWvWCdcegWTsdjKhx\\\\\\\/GaK8vl9TCNateo3\\\\\\\/Yup3fm0LVH6bH8bxheZ3XWddzEOCTowlQfISRVFejqojycDp+\\\\\\\/1KFW7smQFL25GYFgr2VuzzDYZNUPtXyKtSQCWhppfiGszwQHTDFj7E24EDSVqvh6oisaaZuiguOj3uQJQhwBG3Y5LRyKD03FnKOw7dQwIZ8T0STEQ45DeYnF\\\\\\\/xAal8vmDoaEbWllYcNI+l76sOspznzTh5uWxmKFiyDXgYRAyOPFUt4VqYwKsxFYRHbtnMAVi63JFEzRY3lRSVHCS7i8kkQ2l5BGgW\\\\\\\/lAeLi56ox7YGf629wMDw\\\\\\\/N4xCvVCC\\\\\\\/CEmITdUS61yLmc6Hcr86iqrv8vTrgP\\\\\\\/Oq3ZqkN819C4qpri2ynxMlktw5bpVrgZodHvDwptINYDhrNxVs3o16wWTN+IN50JWq5X8bOESLNqpNPq\\\\\\\/d9+Ej7ZXQj0p5B3THtTA09LZWEs8lXkXZqZfkP4uEKej3NxG07boXUSihdUzpiKXKb3dcIavbwfvG03Q5f6lV2s+hReFKLw2NP5OnRBy9WxHDp00\\\\\\\/TEEknbskmq0HUwwSZD8nWNptsWXSE5NJXogN2nPAuPO4s8bs12iLJ8HhZG+JkiQhtKytt7iq+Xp\\\\\\\/leB0ZzRVQb667BXAhabkWmu4u7IHaOoSr1mst1gjWRp9gl7YCv1W9we+yr3T9tO2CwkihstA8fGlc77gN03JNBuVjivAmO8FG4HTdYtVEX+jEzx\\\\\\\/HGErhBR9eR2iVfPngOHXHTyxyB9wjwi2NoW3TMAVDVWfkH\\\\\\\/g\\\\\\\/UR7I5yd8cZZJKW5WR1HGSiR0SQQvJSlCYedtF8FkmUoh\\\\\\\/mZ80unCvQzEZ+qis3EatTP+sxaCuwY4LuNmtAh3l1KRtfqeJMfPpXwO0FfbQC649JU6DWAsiJxGgDgzQ5cDEgfpIOqu4grDeYbc8fnDR05MigmDw\\\\\\\/dz5xMqc25h7xSGUECS8HrgBFd11EDHkkOqyyigCe96\\\\\\\/4+1MLb1xYQ9Ngd10Bi0LuRdeCVEWaSlEV9AxSN6vL1f3wJ5mhLQ+dNIrjcM1YewhNU3F45nnUyAjzPOKm49g\\\\\\\/3WnPV8ghPTtfqbNmXf1t8E+KZYgyYBHPLiUrsJM\\\\\\\/DX68pT0IUvtAXa592dIEMJtZd4qQBfpuf9IxLTyF5LG7RAKbL4jKnGwVXQXx9m\\\\\\\/syPKCxuoC\\\\\\\/BZrkUNTIgYf3XyqpK0tyxw\\\\\\\/ha1zkKd7iGStdJ9AZLlYVsEkXGUphsZPJVer52QgLTiu0NgpFwuFMOZum3BwEeKkHgwmEUgqXGLwDO1k62SuhYF\\\\\\\/3X\\\\\\\/v\\\\\\\/j+FhPIbMEKw3gBLOK3eR7\\\\\\\/DFw3ufbV3ocsptgydP+kzRe8+2PMlMI19HXDP7Xt1r3wJ5Q5LY8idQ\\\\\\\/f33pM3fh2J5P3nhUd9RzZfupP\\\\\\\/Icocf0kHO9V+gpPo8abSAPMXE0KFkOS20q0ViFdgQige1+Sava16ANPZgz0WJURPBCol4Fc5rCAl720TY0kaMb1sc75IT0KQ8njs1sagPkSycmsEhPCVWeA2nrpU9xvti6WojoU7hpp61KKkdiH1ylxavQ91n\\\\\\\/\\\\\\\/QzPNwOMAR5flC0C8mPtlYXybpdwi5M6DGNWaqt58tlbl79A2vC0\\\\\\\/N9bGqP+FMonKoQIda7Of4BeO2givKfC8kaMh\\\\\\\/h43TCI3W4o\\\\\\\/dDG8aKq49oTtUfoMeML0\\\\\\\/uwvpITb\\\\\\\/IWTy4EW5rp2fJlxW8\\\\\\\/5pRbzQ7vnyCviZcQ0z9FUPmOD2ZJbqI82Kje4WMn9zT6C3kwS+9Qhl8dvLc9Kn6zFKJKmazXO\\\\\\\/A+h7DREee7D6pqRFBK8a6VzNWvte+D4ww4CACN4asm62YUKpZZLGbwoh2SZpLFoGZrAY2LqZb2N0LARFnnm5rRfaV5M6bMyx3oAgHNxNglNFukpRscIeHEL\\\\\\\/HikSv\\\\\\\/ZU6qnoMGbswwZnCRtgGJWo2eabWinFMUe9ksTf0iS8690ub+Ko0PXSYHY15cGLUNtuIOxDsKkHSwOlQziZBdy4iVJ2UAvFMWCWQL0yl0GsZa3tg6ot7au8cGuLXQFvm8usrn9ozzCggTX3LSL\\\\\\\/kIKCrzLHfgV5y7xqAw3tiDCOIVZK2d7VrPqyCQPS3Bw6SV0OGq362UiwRxft\\\\\\\/7dF9o4t0qmqzqQdlbgkoWbM28QNs9enILUpaeUm84xk5kGN5l\\\\\\\/SMhcx6BMkGX3tkRQYlWvrUOoSQMejVP6Cf+e8vUQV\\\\\\\/j10DzhhJAVLQ1J1EdNJAHZn1K0wy+u1UJjbW+Mtgx3x+Qn8ONy\\\\\\\/TVJgLg0apnrJO7M6Sj01NA8kHc1Z5YS3jadXBi0xbMYXnh\\\\\\\/SSmvoCD42qSaD4cfZQ0XiclQXdcZMtqFDIoLs9mPiRslDIdbrX55e3BoddY8O21WhAc5bywTrc8M\\\\\\\/rkux9CJE6hcyl5wBykUTSVe3Tg0RPm80mA826rSkNeEW5fz1N3lilP+tdHtAB5H\\\\\\\/17yYGx\\\\\\\/FAUaoqNJpLIXsfDap1P36TcwFjZ4e6vYopVC4jwDQG2oBU\\\\\\\/1iIqb7lOmiEspmuOM1OA0K\\\\\\\/rWbmpfmd23PJT\\\\\\\/fkpO1NeHHtGR6aiujNKaX1b+j3pTaw\\\\\\\/j3oP1jFv9oAyJBbXRfgll406fqtUhNZqbiV7S0fNpE5I6MXqr8o3WNFST5PX6AEqXEMSjxJBYeWiG+RWVTVLFoeCQCJCcBpPVjb2s6m3OkVwAOK1\\\\\\\/oYUJNsya\\\\\\\/PwFnzy0LHMTvPdhy6tQmcIxssPNcfsxXEIvok2skvEQdmCLxmybzHHSJRKjsxXETWgdsdMgilIgJ2b9p6EavOyF\\\\\\\/cPdsJSyaVLGVPEiP3u3qxNCzw7fK8YLYvZqPDZJJz3b0ppkAsFEr+3HNnXr077c0LPKB5vj2R2riMMJlsiSrX2SONi4xYOH3nNfO161bW9AlC66LhLY6tXMFjqEUzb1U0\\\\\\\/CWrbc\\\\\\\/kOFGH+nCwmUORJDcSR+oBT+xul3qdDxmyCTsdX+he+WH1HMKKuj6pmbr2ai9luzLK7ntfYuEsE9Rt9A7OdBHCxS7d9eWoi+0nuqmPWiSLpRrou0\\\\\\\/bqPpXlquoTMxI3vHvjTiatz\\\\\\\/2XKwEKN6HSxkI0S4Clp9FyMP6LLHQ0GcfihwRfgxa+UvnVfqQbnlEk9SwPgDUiq7agMgg5+dFID7vq6sGWNhCYL5GKWEc05HdiL0BgbRLw7SJBMgRGBnpBrPhGd8CiXE8NyDuDYld8DAOKJ15FAkUu7LEWXDe80VYJ9jHpdPQM8xRvrd7IZQR7RmkhH3FmnKdusYn4YKK5AuIJZ58xmmt0nByxaXCE8NSXrAv6FaJT4kCR2\\\\\\\/W+u5wOIhu3kF6pn9OJ9tgQ66mx937ShJeGGM0fhNo3Xz1nZCOrtYSyP46ryWOE01KiUSuMRYLkPCn1qs+CQqjuH5PiMbLeVjCkVaqdFptSwRpK7brTvIKS097NTuVIypaqzFb14oBxfWytqJ0ttrQLOUpQ3q94zHWiD43EeqgneyDRMZu\\\\\\\/Hsb7xWz5Jy970lsuQTCSYWQBQ+vBXqG2OY1rkEnt5t3pNWYZQpkMIASXH1Xea06IeKihgm26ZI622KksSt1grIo6yfXiDa+NW5O4xM2j\\\\\\\/WqEb8YkQFOZWWekJvGVRNovt2g0MvygsfPb8kzfnezjMLrChUF\\\\\\\/boyEVtMBtncuyQbxu3jCl8RfBfy9QpUx8jIZbz+e4kl6qsgtxgJWZllrhkgtxdtpAOvgbbrrK9FSZcGoqwSb2c4NQWIGBR1tn79E08D\\\\\\\/LzcQfVvHQDTZssGZxBoms80gWdjXiX8R2UzG3kPEeOJVgai+qgO9GKEp8VBcfcikUTudGMvtgZPEU2aBXrUyOLBnNGjLJCXRyQf9zRUwI7JD4VbA+f6xaV7ezNRCrAA1knMNVdOeEAqKMpmb8XXBBi0ukbGz\\\\\\\/1SL5K1O+etuqx1azQFFoQFXXPeYZMLhWC7U\\\\\\\/XCejpgmAenn67wJayzwLGs87bW2mUeeoqzaBHOxgUPqagvr9Hm\\\\\\\/f\\\\\\\/OknFNXdlUnnnU7yc9EFCyYF73B0AEPzUiLnk\\\\\\\/KQhWSehSR8w7eveVQWIDXd05+ZkH+CYMxVSiA4tlnoxCclOJLXdD5Of4cGpPSa1UiOJGVI1Wpdd\\\\\\\/4olDe60X9RU6e7QLpZRIiNF4b+skA4dNaIVeMYkNCMn7MnEiZya2nEN23sKX3Y1mR8xtM9XKFSZy7G6NAEqpBuErMRgkPW87GYf+EKI2dNJxqBTnYZjbs1miR3Ge4EgXGhkQamz52nlodsRmLIqS9SxifenoFNqTYff+fHVn84fFPGROMhIqa3qTtcqhqZS2I9Ueh8NudSQobi+PJiWELXFbPN+p3OjbNsgwUdIH3\\\\\\\/T2EhZel6UAAXNL299iLdKbbRK5Aiut2vbuwXuv+315uzn4D\\\\\\\/WLcXZeoBSIhjRczlxZC0EHr0ZL69dhe4uyVod8GCue8XUsbo1um5V4NLYmeQHGPdLws8KDMYi1MV0wsOIQykWG9SwgHLOG34Mbo8SrYH0xLc0o\\\\\\\/3Xml25DM4faQ9cRyVmgkzr84o+KuUS68s2i5kh\\\\\\\/2nOmQyypDikbsY\\\\\\\/j9fnWdA\\\\\\\/683kFXxtBjVt8eqeADpe\\\\\\\/jiK1MhgxYBYlaMPnd0t30nEK2US4iSOk7JbqI8ia4VbJBkPDMWGu7ikLh5lD4MPMsPNqQ1ovxyZD75RCmK5LTvNS40DRnCxux1gxurveqcnhHtX1doJSHTAC0zO\\\\\\\/7yDltaj61eHIrUHX1DO0pwnooJTe+\\\\\\\/rqJKSNUTogiCKG8xVaLS9sguyFI0WxDmdWdoddfDxSdTa0qcY5ueBwu1MqWi1cqQYJLl88Gw4EQ0RXZrzQvOYDzX8if7slnYf\\\\\\\/h7Qifp98U16+gvTzOdtlYQhE3ijBykXqHZUq6M7SI1XAbx4Evl2yuYW\\\\\\\/ugEIncsf+\\\\\\\/mM+VBR1cs6JxyEmhilitzMX82lfRocgDIe5f5OsK6sjJuSP2pLdYTNj0khDUCn66ZTm9S0vn\\\\\\\/au9l9Rq3cKo5JICun3LGDxivEWjQBdmDRVqVxpATEU4FiMr8NNdHP3PFtTrFudhgtY15EkQA7NfA310UxpJ2\\\\\\\/kVcR8QCq3o+jmudfrapiGaY63IZ9PPQoPicVTqhqJ2x+B\\\\\\\/7WGJZhnBvGkSLRtDX3YTsi4T\\\\\\\/GyXaPUc2koPlAZ3fqvOZzXZL\\\\\\\/+mhuxHqIPmsJT2Y6fNBbK63dlIx9rs1Wzdb1ZPk4mT2KMLifEPUHmFsYXeWawhZW+f3AYbxk0NOEib19f2N8\\\\\\\/Wq6S0CYxRjvcheyWkA8ZJSBnXzlmWOOEM0+tEb1sYCRa3buQefuhDSd2o8ihSc3etszxCKF72Kqcylt5hXNQE8a8Ns9AdPbZ7CsqHKTX2s1+csa8kew8Y\\\\\\\/WcNXCclOUnnsK24\\\\\\\/DWtDtLOquVCNck+25a3wSNDeqIH\\\\\\\/v\\\\\\\/pBlkilNVjSKkR5ezHuoLuHJ5eJuoX7CqkPvtMWi9sB3GrVmge0qcU2A4VX9bXgsALrCIZ\\\\\\\/8mU0cZbWvslCEvJrTqit5ScS7iD1+8zYDxHgOZ3FhVkp8csunJmhRSAzkawdlQA2yXfjdZVyZXm2uWMcEFvObAnVoaJFCDZJnCfu1ZCKSkfm5CK17qNFkhIDexUwJwQC29+EY8o3im1dLF30DT12GdO\\\\\\\/NYYh5DooQFMog6w7FL5aRgIiQah0ktqbMYYBvoQArz6P9Skmrj8\\\\\\\/KXB01qd+yWL3YrNLFcxx7pN9CaAT9RXIUeBN8nYAlOPuIYbjJb+FnRnlEyEM0iOxrhmwb+Ltg1aI0wr7ezgvDdZfZBnzL\\\\\\\/sqZKxq5+FDgRxRiWbzsdrebKbnvkU+ycvQiOfB2zmNj1v3oJkAxe7UhRx0qOStktTraMzIkSH1zXsbApIoysJQ2UbRhTmYnYDDG3stc5Rmvnav0dBdODGNtoVrCg+n387VYtma80U568tScZePOpV6sEClIJ1hCug5stmdLHuGG2QRgHJID6fjh4lPyuGhOVMFMlDy81UEeklDJ4l1+PZWUR6WshX5D2s5TULCOTVVsQY4j6zt3zQpT24hYgGsm\\\\\\\/3x9ECwhf4gnGZHKpuc9DAwWo8\\\\\\\/Cm6RAsLWuz44r5hg\\\\\\\/g+grfHHc\\\\\\\/Bvma9Kbb6VtpqCGWKFI0C+LBt7ovsj0fh7TsYZzvzd9nSrkkCjtsxv7QxJ2OFC7JgMEJHlODdmoUbYUe92v36aTb7DDyTl9r8VHHQcuXQ15Q+Us4gIF9Ia5UZTuhPR6oNM2T38uUEkqu7nwi6fTGuzc3y4PmwsNp\\\\\\\/ePOIgTDVUgEJPfPUocD9J8qdKqLTwIL\\\\\\\/gLOaQTh0C+FWFx9gxzk4nuhWNLdwo9lCffT5yGRI3HLSqewQG0\\\\\\\/pihZ1\\\\\\\/G0U18KxwP5gtPJioglXZ4U1qkQEBCtG4xPnLgLv1Ga+2nnz2iXNKUaXPlbqLyrlm5RzZqTVJYqjA5Ebj+rOzuT9SZ2XQbQIsm37fa3tMiGzzRjN7xAh21AIx7xvtZxQLmXwA9Y2ew\\\\\\\/f9I21KE1dW4t\\\\\\\/2KjOa3C02qXmBZoOhpXCJebHeQeWmbtnBG+38Zg7ObjYQRFOaFueCqbTtLk+yTJPGhky0p86BUndP0v74KjUY4LQCNHpJWpAEtUMPOuWj\\\\\\\/jVzK64+YTHpzem3N++nqLFqqsBEwlTt6bxMgD6OzEUxIUpBzgCTa\\\\\\\/il6DYgsxz7+MFGwDikrLyTQ6ZKVv0v7rKDg2XvNxihKZenPsILexe+U4CoH1zxc2kV31vvcp0AKwD6iQTRKonH+wyjXHh+00O+zn3JcKcePbJfId+vNd2j38x6ztnVSKrhg2BFqiFSppppuuo8Hxxp3nRcswugOnCALlAj7bLg2PETHJtqgr3CxVeqXjJWF6eNexLqlrgFhJ08sblo5LgQ8PEfAd\\\\\\\/GoQC4tT+9E1cCtKpMoE3EIIkurJgu1DxmytqS4KBRwP7vGdQLy2cw0H20cHtdXPwlf6H7NQAg34iqSvvro3NzEuCzaD2NM9yMQpVOsSegx36t808iRDXr3S6tyBTj3\\\\\\\/wPOsuPh1NofcLK3bR79HCGvaQLKLKegR69XYvw5kc9pHLl5wTqlqZ+UofeXiqVad6GxRnWfk4p4NdF2dFmYYuYnkOw5mvv7CzmKln5nqMbZhjpr+0MncJrr3sp6abdW88AQNjPPK6uImxLagAmHjA6iYUYXKu\\\\\\\/flkeLUTpMX+JdN+WUUerzabjVJJe40l+Cq\\\\\\\/gxsb5MGxrl2z5hT+OI3yWfK+MZ5WeAcYUtcBFcw80W40bvQQYWP98J1qhRVhgzYT85Q2Pkf4ef\\\\\\\/8lMyeBMouPtV8+i\\\\\\\/sgWv5rqGyI7Cf\\\\\\\/F\\\\\\\/2TDToBh3vJ4xLvKKQKX1ntvx5Qk8MJR4f0OVIIO\\\\\\\/rIfWifjf91qXowEvgGCbhyj9lQgkXyz07d\\\\\\\/DGrYiCpoWinGzXALUb6tOL8wbCanWRZRudVDWfgTQZmiGlwiQvRDM8llK7DbvJs2FUBY8jXyPO5LiT04sDnkUeQ7rwjpZNz0Z5cIpS9uxGgvKl4W5mYKUUfXN6qjJaVC+Mp1pURDX3LFFOwmjVMDd0tffTbO9j541xZQ6lPYE2BU53igIiBaV1vO6c4yKaBx0Xqg9yK9+nHvmbwytCJCkVZqRv2yIfBgNnpals8CJCDFmgFaKNKbQ4FDGJYCCG4SrkdnpDwiKhxPIreZDjJH409HKGcNnCL7UzSNt5JOZAAe2tDGWgZxGDbSg\\\\\\\/Ps3jHgv5Oaj02Nz5DatGHS9Kj0gDJp\\\\\\\/3rhtGJjunrkrudSoTV5dN6\\\\\\\/dTNLCFR5G319WsL9fhmkd4znObhOcluQAGcCmqptd41cMh3+aRHgm50LiJM3A6ZdmkngNsN94R9uRH2ovDYNZ53VYedf9n7qotNvVVwev3Hnl3vJUA84EEtbyaV22zJaTXOTp6uI7U2LCueajbuTElshn86bUO\\\\\\\/EAziY1Zbb470KGdim4ZdO8TXuCXYRCdDUhpD6GYJheN4HGPKk9zHmuzLNn\\\\\\\/LYxZ7AH18\\\\\\\/53ZSJ9W3WlXyeFcG0zejSvjIKSYEjoVudbhSCRLLZ\\\\\\\/Pa5eZWcs8KXGqswqANWexOW7\\\\\\\/gWIVFIrKI3mtxkgnFA3m+ivGpxDdkIjKvIBzl848iDM26UklMrMskOMKivNnEpa9ghUckZ9FUP1WylWxMVyq2M+GeEcoC6gCO9FuloC\\\\\\\/7T4UGUFhfpMCB8ZFJ6GkeevN3Lxdsa9WaGp4EhvuqZBDEpLr6e06L9cQ4VMzAhGClncpL90QkgKvI0MQpSiDNpB3cWxHTRajaYcO1j8PcoNwxWpeaB3Npsmzy7A0u24AoxQBnz7JUkGOPvXnH8DVjuM23A7nLaF0\\\\\\\/7gQcQj3hlytVJ0+6IVcnm5p94e+8PIei6rQ7O0NESMpPvMCCoenV2mEWq9MZc0oYp\\\\\\\/neqG05V5j4ILfz0KLGvYjJ1Y\\\\\\\/Pqey5AsYJA7IDGYNdpAw5mx8fgWPwSTVNHSam2GXxIJRp+0bzWPKOQmnAZt18iOuR5+s0jIosCLf05bBOyJ8H34f6mUB2otQBf1Zw\\\\\\\/+GoSylEetfSDBYVy3+oWzQVYuBldYJMmzd62vwPcsyowHy+iyvSrIDp+wD37UlHNMdkwajM\\\\\\\/3nX1ojNOEYvxp52w8+fKkqd00XKPKgBpLALP51ejLBPEdVQkaQ0k651fd1cEUyKF7bnUsCfMfnb3fj5JeBOuz5KPitayYSrcBL0081rMibFtfVMrregn\\\\\\\/3veu7KDUQLs3EqlNI+oltvECQ9PDSUF11DeICAzt3BMXiepyN0OmciFPEInMCRzx47yKQZ7hJ7KHtx3iLTmLcKZqOq0ViOFsUBKwfx8eH79f8ETNqiGola4YQHzukKWeOISeSXsCrk2fyrsllEMLVq8OOzjgEu4t47ViyC0i+Z6ZnhhfCgAssLef2KjrTSnJ7fiI5b3\\\\\\\/CTXgzEgAJW9OMHkmExXl6y9BS+A5vXU\\\\\\\/qBUmmztDPPteHLLmLtCxRsyDtX3jYO2P4dPDSLSX9nnoOH4LVEfK+zmCujPeddhzbTj86U+UxWpmtIIgZyUQc2Pv6g9s0oGjCXiPnCEDcGhV1V2qQYIAC\\\\\\\/E62trGVg2QNjYIZ09MWeRE2qxwwdYcUeOuzDjLqvQkN86SVdwE0wdccNBhsH3y4Ur8InEv\\\\\\\/iv9WJGLSf5Wu3KqyCS4C+Rv+D4tZ3SeBWCNsQHcWosPcox5zcwp5htUuD0+EbnHbqOq\\\\\\\/P+KJcVWb4E9guTAOGowhYyxasdOoLAuhPNRQ4zOUfhx81ZxpqgSHIjN0cjHe8Dll44ueXSQYoWql36vMXeFKMNZcOPJg9CzqeEwWWwftTVRTfukPW5Qcf\\\\\\\/Al2KHtW\\\\\\\/AUBT7i\\\\\\\/0FN+QZcq5vLURZw4TjXE2\\\\\\\/a5p+56F9xuJDkRWfi\\\\\\\/rjb9Qtx1ccUF91Z9lyUbWleM3YlM2iQUk8UWQzbw+L4iN9Tn3vwH1odPpMyX6w9sI46QzLVzBJqQuG+GwwVrHga1t6Egh6pYznBmHCEmYhni2sFnvPpGcxKDJ51yv1kNDRREagvYw\\\\\\\/Hx1gqoWLkz2FLkggj\\\\\\\/V0V0HgBe2ynsDftGp69pzO2Ft4i3wzO8LQTurUAF1qOKJm+PtQ6ZSn2eII9BmRN5lu1zxvHgq3uxCJBwmcpfIsfNL7Ag4EKvQplGfMjt7SBqhbrvlim6RI2hwCeoFJtVE68xFRUNY8tHiGYuUdX866eNr1WzCs+e9d1H8BS\\\\\\\/O8Ju0S+IEwm+Ult+oeIvUeYr9fXd996sEcrCrkea2LKozaFkiWh+zLwS+lwbSwXSg3vNfJpA28X4TYzkBNpM7DgoG88hZKNxBgWTR0zaL2SmTnsR4ySy80YU+PzhO7JguTF7TGDpcR392HVZJfSZxvLcIXIj41QX5HeiEoXxqoEvGv0wQW5hskoK3MTJorDFS0jcztTtFeBcL+Xo4vyAsGrx9u\\\\\\\/Lb4mqgnk6hfyXka0aUw6ght3O0GSuhyQkg0TVKw9J1ydgw13l6\\\\\\\/rjZkVWeYOsqAlgg6vY3joqAcCZ0SiBOolHvr18f78frx0DOzOVJ0VrcdbTscuH3szqFy907ovstzFWls9aQVUrsnBOnWxe0idtgphvsBcBiCHAe1B6TJo4Yks21R2VB1qfEcttR5or\\\\\\\/40W7F+0yUa8g+kDb8FjI8pA27XLszkUYK0e1xO76SuGSKySnKwYA0nDfC8lCqXXcHwUBjYAO\\\\\\\/yXxPTq4e024U7RvbIvKt9GUhctUivnLm+wkS68QYZCXGFxbnsC2MCpmCtKT2gLegDBiH7VUGv38rPAr2GFcXib\\\\\\\/2Q5NCNYCLjWWZXIoQbtt5jJMBw+Vh9Gr8mDUqyI2bk2+aoX4cF3P8Vq8aWxIisYUbnKUsKX6BG14fw3FgKfFXlNpTxh4y\\\\\\\/ketK5FtfRQjFdDe3A0sgOfxvFzUjcipVzSrFZli+97\\\\\\\/jnKTB2gNOU9QZLojY4AqInuHFCFh4YAUksrfAPmIrQVpVcWF6nOvGJugQ3Qu1t1lyPRd18nRMijWpME\\\\\\\/fCdAdWs5jf3lCYBW6AwOwzryHMfphUN1N5ygRWx0DDbHSuR74\\\\\\\/IXu53v1ZjF45Aqcit7uXo4QysS\\\\\\\/DgZsVA1yvsQj+RvrzP2ednB2fXryT2CdfVkpx8Li1xdgORxm2XX0f3enR+o7xXbiEnv9Zm486iMb4NAhD25FCxPTKylK4I33GJI7mTpAx0aoVgrp2RuermRM6yfculwoCrjiGBjYsolFsz8zucHDaF7C3QS515CIWBkhEpDHU9SxQoAqQHDbu7Bson4Ajg+3FWDjodapFALABwZ5siWfnMFBx5p61GHd76FW198HdVPARSdRthIpFyMz64SyNpcplKDhJ61em6UDMkGbGhvZcSkkXt85l\\\\\\\/o\\\\\\\/vyhG42E0qRmGcx1YnhHrvSixfwKZfZ3nBLw\\\\\\\/hEcqUb4eYipUMLC96nB6KIWzg6+1HhKpNp+pC0T0dpcnatKoTIaWETKw+pNnc4FsLf72BlMlp+D69OmB3EvaZqgW5IqZ\\\\\\\/xlJP5sbNdQ3jX\\\\\\\/aALlntP2Foc6mayZfPJpklpETaGBql5x0ofQLFRE4Mxm5uL53mw4HSbaaPyidktR3TrjN1HRRwMKz7OsXs4P2SV9isLuNsd5A2zvbwzF4VUV+q4AOc4mNui0ptlLAYNKoSbV3WABCSJZeZMwd6EbGpymdPNoknNyjKTos\\\\\\\/9nPzcS9vOAgV4DQoCE\\\\\\\/6NEDtwLKZ8Gw+GF04jPixqQ4pPAMS939+PNnIBGcVWEoKeVoVn6PkrQ6wJyQ\\\\\\\/pSeAhhovCSJt2\\\\\\\/OONXRNvfHi2VKqEh1vwy2B6uMK\\\\\\\/9xjcdxWl+0szaxW765iufdZFB3l4MzMZTXix8zSxJuSYT+M0tDyWVCxIU6dPmVpBi1HCbx3jZQrhNwL\\\\\\\/EWkc7cO18UsHoM9orYaXEnU4Tk1J0NY4bXZ+VnsDwE+OMFvCQR0+cTR0qyW8TsACEtzTepYJ0j6PcccGw+4hV4BIDjDkbRGUIptP\\\\\\\/lDsYV4JZrJV0TADFGJMBqgdbQByrqMtH5DRGA4ztJmvSMRj\\\\\\\/KmRJ8Y3BTZg04EPVXT5DrQmGVo34KHFN4pLILHlj68kJSrtS4XmpDlSj+0CaQJDoTLmKR6cvctgV6c+b6EzFABK8VC2+1nxnAcOyKIVMmNneW0t63YbnDKQLxuEnGXPK54kElOfJ+7Ko72rS4l\\\\\\\/DB+rmrWRXq9j5ERuKBi0EQF8SdtIc9dBIFNI6ejqllwtj1dc+Y5dVTZmlWF7qzAIDq504rZejmDwIRVJ9AW9cT4rJqBD1scNngLAc1sAJvXY30sz7jWhxlrhIxeSaK4sC8tL3Cyv5l8lgMpUEgiDE\\\\\\\/3Z9b1rcDAPZFO3tlr0Y+wJivnbVwidiOJpWUsNUsc9+3bAMdvLYQ+2\\\\\\\/Pp9aQ4+oZw4gEr\\\\\\\/6NHyWx\\\\\\\/za\\\\\\\/s4iSobVKUdtGywk\\\\\\\/33\\\\\\\/GosCri7aD6fYgy8X8El5HPos8l9CqcJVjJbhquECSPX4a31ROwNiKqF7lw1dJzYGnuD1zpARcO2XOMbGRp\\\\\\\/ZGm33Ubyrz5Kd17leeG+3N42avLKZ6CI0XB8mrLOUk6EW5+WU+gRN4pTCInlAzba+AzuoZ4eQGDruL82T8hgPqqQfuXYFdtUD\\\\\\\/mbGpqVqRHtlb6ojByiuv0x96sfsn9q92dfVqLLtlU2Q3VAMj0TTpnd2MjnV5zjcvugp2Hy\\\\\\\/mGAhFPXfDdi5E0P2PLTK0v1WtuI7Z4mApTkuMFIbHdW7xqH1lUj22Nq6HrFnF9\\\\\\\/cwkgpgeiup8YWeWcen1CDy+Aly3Ije\\\\\\\/flQS+fpgu86Nhe1uThkfDV4e9O4GtEWIZaVbv78B+AsG90aW+ten+eef839Dw+PuYLr9bogOcgj0HfveD\\\\\\\/sYmQzv3TwHn8tMvUNoe\\\\\\\/\\\\\\\/pOMfzMW5ss7EEVrwVefEwsmhCBFuvjzyWvhT+pL6gNpXjV598HpRRv\\\\\\\/dQxC66wM13z9TbB51a7MQC+o6XFCu9LWcKxSOjsJ2r1SWmZLbXeRBBKkpDuO+wDLPKqi3RVro0G8iI21ilaCFMbfwN\\\\\\\/6lrV\\\\\\\/3rMIMoxwXHaUjh+km6O0knUGCjephzBkevkAcbaF4rlBetLSmVNJ42CD3DChxb85du3RKraciilqHrmeZp6hdJUU4jnJKjMuULiwDj3z5O26H9D71hQJnQzRyMmANfOAmRbYWuJ5gZOk5AJqZSbc6HAB5JBboNWeuOFAh3y\\\\\\\/UPqPkmiboM7QUXGG1sTt9q5eBEA0ZMthf3ETJm5feb9fT6RbtjY9ZFOzXCfufLxrgi8G7wFy3zyfypgBW3myqaaNN7dtacpu1hdNvw7naiA5UVQf6Ed9Hs3b7NnrkSl0jqXv5zbJObfvTpw1BIVB6LJqoZ\\\\\\\/VDUT6p7cPsUQb8eFVxRyoqp7pz6nOEyEMkxu4feiFJA42jgsSpUmQ7odLkJFgXXa6jFRxHhpG+K9l8IirNV8kZDpah8WqI3qvvYuAgaQs0GoLNQYf1HK4Ra2L6Ox2tsboxE2INyrzY2y1pq2PrJVA8DOhvtXllT3ofGxssQlH3WQV\\\\\\\/2qyNBN\\\\\\\/pTUAbEgKdVr8ebYUjMGNQ4DZidrfiq8Qwk6jKW5rZQfDfJsV4VERKR7Bl9mdGZRS6H3Mlc+H7Ex7PI\\\\\\\/J4MnA1IwVGVOwsYCM2pz6AZ0zigQKsjG0\\\\\\\/tBY\\\\\\\/tQ0IO+Of4\\\\\\\/qvSxbMq8ruUodWxWvKGNUg89GmF2aGvBlNyqqlKnYbdtvpyOzeI2jHAirK+Ro8v97ipm23Kstz4DpjNPCW5q8NKmbO\\\\\\\/LTGl5GvNaPY9L\\\\\\\/ztgsJSsjSQqR+oC4wde0J9pO0UOhCkV+rY1HH2yjWo3lmHuDmQUc1z0Htht\\\\\\\/o6LQTnCgLpcccYSSfxgvgQ867Dv8vGSg6EoMwPRNSUknbSvLh0rHhpSokkXDOyF23KEFlzL2rNSxvV\\\\\\\/mmCJqtk90C3Q9zV\\\\\\\/BcPSGxR5gy3bC13gtfheh4VayMmrUsrsqQo1TPGUQDGWnqLQXvnyQQ2sHW0xNbzly5dHR1T7FljdHLPktkz7EbtULLexs9MFssxPqZATN1tfEO6q3hswa8lVgFnSXfFim7steyiBhl5FJtW489qgKztNp5l1IOfhdHrHc5\\\\\\\/Biqvt3t0xR461LAhBc9GYmfV5ukiLoKLBV+CBgdPaZU6QyYOTmJoiIAkzS3u4TGqKrkCv+0N5K+6subSHj1MlkKGoxEOCufZBcooFCoOMecytlCutnLrgtMTTMlBCRWjsr40f3dqcb8zY4a3Q3MuIydTwLj+AqD9b7UPuZ81mY9CG9wA8F70jucJRDOWQIZP12ppvhjhfCT4D9xLIpOv3sqNGUEKSlNhHSfEJ0DNttuTgf0tWLW+FU7+aZ0t4bp3jfIyLVSxHPDYPnoa9aHGoNcqIlF2SMvn1CydWVlu\\\\\\\/ho6pjpVbXBQ1S6zPQ7Olfmr2qQWo+tMM8kSIsOKH17xUhgp8HNe9ih5qnNSkYgiW9iX7uj3xXm3BBSwE3SBjmXRpgz8i+u2CAVL2O4S+uJ6oDfovHnuOM7Cvx8GFlkWs0OIewIS8v8GKtJ2nqG9ttAw9\\\\\\\/YKdv8KGzPsuppl\\\\\\\/y\\\\\\\/\\\\\\\/MoVKFLXVDdK4LoF8wLBPhbLljZv7fiV\\\\\\\/psWXfeOD\\\\\\\/l05WOXwvYAkNzqzgfTwlNgpc\\\\\\\/FmP75dRFx3EP5YJ8K8Kh6k69IH5JfOhjlt9GDbuuA2DLAQOu7CuEAVTVMHWcXATSWhlBtSHDu9qMrswG9+t8B\\\\\\\/MP8bac6LGDoMLXbBQa1RwwCAdVqeqNDPu0ZoxmwCBSfsRxdTCPPxuC9oVmzx0iMGK9vlViOOW6CVadDlG9Y1+88lvi2Iqnsm9EhkFoOrg\\\\\\\/NpR8FAQgpwcwey6swMYBUwAX\\\\\\\/1owfpTEqB7fmi9\\\\\\\/Et4Y4RHJk0iXlFP1XGHPCZ04uF88uaXPb6hfm9puEb2n0LZP0WjLPrqCuFkZBfJcnTmJDFpno82OKrF7gpX2MYFqZafeNLb8EG5pUyJu8AFNL\\\\\\\/9STKYz2WIwedYr0wxo0ZvX6MAkAGZawBd94\\\\\\\/57AjLHg0BD7QxdnBMmRR8U7+CgGZrWo5uicX98EcEpE0qZfpRmk5DsXY4eZHGL6EPbWl48JvHRXOOAEG2W1ECbuNvE6ar1DWONHzVW60quKt6SS0UOjum76lL7i6wd7p9BlmXPlfk22qXOvbGPs\\\\\\\/ExSNScH+LDPpfl0+q4sxH5SfASyn8DUzNdBhzY5Dj7UJSSarAbK2feH0Gbn38dPMs+CYNQHGkGohkB03k4hxbTvcsoY6Xj2Ta0BwdIRfCFq+160MBFqw6bZBSpMlMXC+TUnTBOVvDT4Pp3qdglVUfg7j+YfJ9OZZSr\\\\\\\/lm0TFlAhUAg8Is7tTaRST0IGAySkV8BnTxlgg3sYHQ+N1W2oF+CcU0PlevTrixU+oJjIwY6shMwLJHxdp9xEsuVsAdgNyGg1uvXwiwORlfAOmcJ5L61UEmUPx3eCAuSW5XREAAUi6EK\\\\\\\/QP\\\\\\\/\\\\\\\/hn30bqi\\\\\\\/ycwgn2U7zwUUNdAPvhdtDM+BPQff35OxkjM37XaAek1xQVQU+bzhTrBT0DCMhVrVt+ize6VXaB8Wovt6ariaPaDReO69HimqsxMKQBVftbseeN+CzMXnFs2FHa\\\\\\\/l3S0cMrmaAK6hZvYLH4pmffGQz6AaoBmq2ivoHDq1DF2D3gFxr82TFzcRyJMrgngWjXNaqBiIvTmKXwmsb1sCF1jEfQ1rYnvXUBAS2\\\\\\\/EhlqtmKUMIjugwxBm7hXP7\\\\\\\/V0PMBqvwSJp2EhL8n95xe7z\\\\\\\/Av\\\\\\\/5aRaOgIvRHQ3gGBN5gkoNNirtneOL8Ggj3qJHgZPDi0Hpp27Ckfd6IaRIYQMLHRLcEs+uvHQY2oMkY9W2YVyOxX4xzdaq+lfsJo1ZaxVkeA6izHMhL0SFSozImR4WBLMdY7M6nZFmCQcUDcRss2bgeJzJwd0PuPPIbUBE1ZseFoPGLuNDyyNJfEOOHxwHU\\\\\\\/aKv12mZk+6nAoaVK\\\\\\\/XX5IurhzERatchKEWS6JSK\\\\\\\/rZXRpPosJ0YNOyb\\\\\\\/b2UgKWqx39Ou0LGhvZ56egKNg8Wz2dzPyUaY5Oczk2nTibvjBQfKS2TMoYSrC8FkwqjGsDgKGAUXPFBqZIsx6DaKl3jsGuInQZ4sZ4Q2lF2jQbCscsLmk7kI7WXMlDUrbLt4oVLO1fsDW8gG\\\\\\\/SB4bZVfsltg0xAiiXHLiv9FBrJXFokibfhFF\\\\\\\/OjYsL\\\\\\\/THzJe\\\\\\\/kKCNwkQk3dyUWknXjmAcIuW5PGVUp7KXqMx4sKcUoYmsyHKMoY4efRR2ELiHg9ylo9d3ptFt+BvqiOL0JnYbcdlrqH7AJHR2bnOaWO3bjgpo2ynWKB3kmiAOznaHaDVdIxkvf66BwkHDqx7FIhnsU6A4+LKFkgpq\\\\\\\/i8QPeJRXbOz9hi44ObXRUpdM3QIGDFQNPk74mlsXozfDyveEcHDE6E2+KdOkvYMYzku6TMMKr6gA8w2v+049U4b5Z+s5HoJZnSMwSsXqofilkU0RPVWbv6vt1C\\\\\\\/GbQumtaQtR\\\\\\\/maz9aVN9Vw5C\\\\\\\/gtbflnhEQyGkC\\\\\\\/XR\\\\\\\/kX5ryOe5wHri0OxSmrGtX1GJsO\\\\\\\/xupPe1\\\\\\\/CUIZIw==\\\u0022,\\\u0022iv\\\u0022:\\\u00220UWU\\\\\\\/l7W8qhzVoczmegLsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225488611ddcff6e1e6aacdd7ed0c89b44b0a363b03a29061ec00e82f83b24d853\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UmNlCRrd0Doh8uzPy\\\\\\\/kKxyjDXIcLlPo9T75CWx9N7+aVtGI6qyUOVUZbNM2SuDiTcMUnnYuC1DdMnp02SYDhGNXXmvgIzLp7qmBizgBTrwFILpp3RZ6Ym23O4fphsaeAsLzNjquGR4rezjAesMogOpkcX2tSamZ\\\\\\\/G7MQ8Uf2cvwaxH3aZvb4jeHlo6j71t6rK1igK899IdXqI6Tc8gBjJ6olAzqPgyXU2OuZDCJMK3XEDbMpEhAVRZBNiAupk3RVMK1IzSxMKgjhIcpdynctakSSGpkNZqmk4\\\\\\\/+5CmU1IJwuJSnJnnS\\\\\\\/8T5ELhtbdA4XYSw9mxHTp8gakpSEYqJ7NV4rPQLH2voIANcgo8RdSoQWyyM7gO6DNEV7ONyN9n++8PxW8Hoc+rxD4GOqX9OCuuJb3IphvKnAjo8YdmaRbUVknA760k5vv\\\\\\\/SwZM12YcHnmTFoyivFur4FiYk4pcjhj6z79qxnIio158Tw79VdJJZ93ZxbvszaUZnW9ufXTUI90i54zHAkW5+5RQofryPBz0Vcbw2iYxr7zdlXvyMWj8BApoAtgXhjd9710PTxbV1rqCggF+Tb2Z7IAW9VNF3HDse3Zpi3coeYe9Akc\\\\\\\/6Gab4riSmxDFJzGGYEbQJF\\\\\\\/cghgWzO75iIGsKLTOBGUEzLNGXECKGJ77WZCFLqIWCG5q7sMSawvK4RRWnQBfOVb1HEUyaWdYo0JNFohih\\\\\\\/xGkgmErYtcp5aUdquPzFLqpfiM3XeehMjC1AlYrWPlAFxs8lp8RzzDiEss7qq+GW0Q64d6zywbFiacoFY1fM7mJfqjO5+CtLXT+mi\\\\\\\/vAZ86tmzeIXFk8YxQuQ+NIY001r6Y2wWn645T+xCmWa9\\\\\\\/C\\\\\\\/jR+ROSyHdu7BLNRKBZmvHA7W8efTv0VXvgUcI\\\\\\\/uSZD5h2edGL2HpsWSVJb4fbSvsT2g8SGP3SceErWR7dd56kCIGjmZmm9oBGS7DdSgAKDuYZMHBFIi8vrvsIycrvcrvELcWiqHMyWfKaPuvyXO3X27yz3foQQ1dZUiOXxO5k\\\\\\\/TSaZd72K26BLnZ25nAkqxSfkDdz+b\\\\\\\/cKqEwRYJLJfh5uxTxM8n\\\\\\\/i8fibB1rzXNM3rTCbYEQX\\\\\\\/HvzVGnRG+sB8xJwQVrNZLStMSgVUu87WDR\\\\\\\/wW0AOQE5pud+tBmoo9o7usFTxjrBvyAjc8VfdgQBYiVH\\\\\\\/TsNILj4wG8exaNFeNLcUhd3bJJzCuBTDGJivtwNoHV98WivMIWlbN2W64CZBILGFi0qCDSc8ykO7EsVWlK+D8TVU5DMKg\\\\\\\/y\\\\\\\/8yvaYkVwfgKPMb02kIshdA0i4XBS5ue3dsBiP3KSoChhO08SiRxhQEq1K1sWi4rndBDRW8Pmb2nxWPIF1OX0R6czhAZ7rr9ylc87KM6xW\\\\\\\/92BbhNs0P+b7YKZQlwz+mWzh1f9VxImKa1X+TqtgQn0SORbtTDvnwg9shPpAmBAUKW6cK422Zr5saL28nsIPWQEFly8TX6IBfR44H8MLwIeYeuy6tWAM8dE\\\\\\\/S3TzHoRrVuTu4UjUwmK770gfonjwoNAesCPjjrg06Az4vEc3KDha0lZLg\\\\\\\/gK0T49TrLtLnl9ABF0SHEilrxOdJGlxYkQbm9oI0F060XDpwgxXF7tHcpdKgINgXVkt6yXdjpjb1kCimXiq4ZRNtspjw6BAQ61gfZDjmcIvhYMv464qFma1wbwQtgBFQsbLJTbGyRHfwSVPcaMzTXW0ZxdvSaWthgff4w2Kj1MwJoV9mVuhNORwTtmNOdKs8EKAQw8jjcyAQ+ie3cZLsb2iLH27HsEX+mTI\\\\\\\/he9Zyx6F7ROynluQh+ZcoG4crYZ59pUQxuGtOfIVlemmg4Hvn4So3Rr80+LEjQFVl6a+Z7+HK4+d\\\\\\\/LP2amnvEUyhI30W0WbZbbKkRgh1S9XPfaH+NA2xfcoo4zVCTu5+zxfb8H\\\\\\\/2nEYos9ZtZXtlwNjBHb+RbMykZEUBVrGPLJ+LrJEHA\\\\\\\/FBOkmQzCJBmUaNHlRKtjYrprZUVw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sT8zNzOLiX5jVTWbXnWsEA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9452f324127d29030e68f71327bcefecb01b4477cb6c6cad5965229a88ec802\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460164 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2341" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xrvd266yw9FKfiIBwh2QGisH-AsVDgPrRSnLJzL1-WU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2341" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FkmBPTIKn2UVivbTwqhZ1wbOUhH-5Y9H5kBAduqoEus" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2329" title="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2357" title="Genetic Linkage Mapping of Economically Important Traits in Cultivated Tetraploid Potato (Solanum tuberosum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460164" id="node460164" data-pisa="ggg;5/11/2341" data-pisa-master="ggg;g3.115.020040" data-apath="/ggg/5/11/2341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Reyes Chin-Wo</span>, <span class="highwire-citation-author" data-delta="3">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="6">Richard Michelmore</span> and <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2341-2355;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020040 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460164" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Reyes Chin-Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin-Wo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chin-Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin-Wo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:avandeynze@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">avandeynze@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2341" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460164" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460164/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2341.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460164" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460164/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2341.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460164" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460164/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2341.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460164" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460164/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2341.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460164" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460164/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Our ability to assemble complex genomes and construct ultradense genetic maps now allows the determination of recombination rates, translocations, and the extent of genomic collinearity between populations, species, and genera. We developed two ultradense genetic linkage maps for pepper from single-position polymorphisms (SPPs) identified <em>de novo</em> with a 30,173 unigene pepper genotyping array. The <em>Capsicum frutescens</em> × <em>C. annuum</em> interspecific and the <em>C. annuum</em> intraspecific genetic maps were constructed comprising 16,167 and 3,878 unigene markers in 2108 and 783 genetic bins, respectively. Accuracies of marker groupings and orders are validated by the high degree of collinearity between the two maps. Marker density was sufficient to locate the chromosomal breakpoint resulting in the P1/P8 translocation between <em>C. frutescens</em> and <em>C. annuum</em> to a single bin. The two maps aligned to the pepper genome showed varying marker density along the chromosomes. There were extensive chromosomal regions with suppressed recombination and reduced intraspecific marker density. These regions corresponded to the pronounced nonrecombining pericentromeric regions in tomato, a related Solanaceous species. Similar to tomato, the extent of reduced recombination appears to be more pronounced in pepper than in other plant species. Alignment of maps with the tomato and potato genomes shows the presence of previously known translocations and a translocation event that was not observed in previous genetic maps of pepper.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genechip" class="hw-term hw-article-keyword hw-article-keyword-genechip" rel="nofollow">GeneChip</a></li><li class="kwd"><a href="/keyword/pseudolinkage" class="hw-term hw-article-keyword hw-article-keyword-pseudolinkage" rel="nofollow">Pseudolinkage</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/solanum-tuberosum" class="hw-term hw-article-keyword hw-article-keyword-solanum-tuberosum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Solanum tuberosum</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/solanum-lycopersicum-solanaceae-genomics" class="hw-term hw-article-keyword hw-article-keyword-solanum-lycopersicum-solanaceae-genomics" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Solanum lycopersicum</em> Solanaceae genomics</a></li></ul><p id="p-3">Peppers, <em>Capsicum</em> spp., are, with few exceptions, diploid (2n = 2x = 24). The five domesticated species, <em>C. annuum</em> (<em>e.g.</em>, bell, Jalapeño, paprika, pimiento, chile), <em>C</em>. <em>frutescens</em> (<em>e.g.</em>, tabasco), <em>C</em>. <em>chinense</em> (<em>e.g.</em>, habanero), <em>C. pubescens</em>, and <em>C. baccatum</em>, are used as a spice to add pungency and flavor to salsas, curries, and dressings. Additionally, nonpungent, sweet <em>C. annuum</em> types are consumed as fresh or cooked vegetables, providing a rich source of Vitamins A, B, and C, iron, potassium, and magnesium. The diversity of uses for peppers has led to the development of individual types that have been selected for specific sets of consumer-driven fruit traits such as degree of pungency, flavor, color, shape, fruit wall thickness, and drying ability (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bosland and Votava 2000</a>). Coincidently, peppers are bred for horticultural traits that allow them to be efficiently produced and withstand abiotic and biotic stress in temperate and tropical environments. World production of peppers has increased 32% from the years 2000 to 2010 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fao 2012</a>), valued at to $27.8 billion in 2011 (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fao 2012</a>).</p><p id="p-4">Pepper species are interfertile to varying degrees and are amenable to classical and molecular-genetic analyses. An integrated genetic map now includes more than 2200 mostly anonymous markers (AFLP, RFLP, SSR, and RAPDs) in 13 chromosomal linkage groups based on the integration of data from six intra- and interspecific populations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Paran <em>et al.</em> 2004</a>). Doubled haploid and recombinant inbred line (RIL) populations increasingly have become available for mapping functional loci for disease resistance (<em>pvr</em>) and fruit quality (capsaicin, erect fruit habit, fruit color, fasciation, fruit shape) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lefebvre <em>et al.</em> 2002</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ben Chaim <em>et al.</em> 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ogundiwin <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barchi <em>et al.</em> 2007</a>) as well as candidate gene analyses (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jeifetz <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brand <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cohen <em>et al.</em> 2012</a>). Meta-analysis to identify quantitative trait loci (QTL) for <em>Phytophthora capsici</em> resistance leveraged these resources to refine the location of resistance loci (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mallard <em>et al.</em> 2013</a>).</p><p id="p-5">Genomic resources increasingly are becoming available in pepper and related Solanaceae, such as tomato (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tomato Genome Consortium 2012</a>), potato (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Potato Genome Sequencing Consortium 2011</a>), and eggplant (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mueller <em>et al.</em> 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hirakawa <em>et al.</em> 2014</a>). Transcriptomes from multiple varieties and tissues have been sequenced and assembled for pepper as sequencing technologies advanced. For example, transcriptomes for fruit of two pepper parental lines (CM334 and Taean) and their hybrid (TF68) were sequenced with 454 technology (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lu <em>et al.</em> 2011</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2012</a>). The Capsicum Transcriptome DB was created for <em>C. annuum</em> var. Sonora Anaheim and Serrano Tampiqueño with 32,314 contigs based on Sanger and 454 data (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Góngora-Castillo <em>et al.</em> 2012</a>). A composite assembly of normalized transcriptomes (from root, leaf, flower, and fruit tissues) of a <em>C. annuum</em> landrace, Criollo de Morelos 334 (CM334), Early Jalapeño, and a nonpungent blocky type, Maor, was generated with Illumina technology, totaling 123,261 contigs harboring 22,000 putative single-nucleotide polymorphisms defined at high stringency (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2012</a>). Transcriptome sequencing efforts also have facilitated the annotation of the recently assembled pepper genomes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>).</p><p id="p-6"><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> (2012)</a> also described the assembly of 31,196 unigenes from Sanger sequences that served as the basis for the design of a high-density Affymetrix microarray for pepper. This array was used to analyze polymorphisms in 40 <em>C. annuum</em> lines selected by commercial breeders worldwide as representing the primary breeding germplasm plus three additional species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> 2013</a>). Breeding germplasm was separated into four major clusters and landraces. An analysis of population structure showed that nonpungent types were the least diverse and derived from pungent types. In this paper, we describe the genetic mapping of these unigenes in one interspecific and one intraspecific mapping population, relating it to the draft genome sequence of pepper, and show synteny with other Solanaceae crop genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7">Two populations of RILs were used to construct genetic maps. The intraspecific NM06 population, provided by Dr. Paul Bosland (New Mexico University, Los Cruces, NM), consisted of 66 F<sub>6</sub>:F<sub>7</sub> RILs and was derived from a <em>C. annuum</em> ‘Early Jalapeño’ × <em>C. annuum</em> ‘CM334’ cross (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sy <em>et al.</em> 2008</a>). The interspecific FA07 population was developed by Dr. Molly Jahn at Cornell University (Ithaca, NY) and coauthors at UC Davis, was derived from a cross between <em>C. frutescens</em> accession BG2814-6 × <em>C. annuum</em> ‘NuMex RNaky’ and consisted of 123 F<sub>7</sub>:F<sub>8</sub> RILs produced from single seed descent from F<sub>2</sub> lines described by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ben-Chaim <em>et al.</em> (2006)</a>. We will refer to the NM06 and FA07 populations and maps as NM and FA, respectively.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and polymorphism detection</h3><p id="p-8">DNA was extracted from each RIL and parent as described <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> (2013)</a>. An Affymetrix GeneChip designed for detection of polymorphism in pepper was used for <em>de novo</em> identification of polymorphic markers segregating in each population. The pepper GeneChip contained sequences from the 31,196 unigene expressed sequence tag (EST) assembly (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2012</a>). After probe selection, 30,815 unigenes were represented on the GeneChip, including 642 nonpepper COSII gene sequences and 30,173 unigenes derived from pepper transcriptome and promoter sequences (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> 2013</a>). Duplicate hybridizations of total genomic DNA were performed for each RIL, and three replicate hybridizations were carried out for each parent as described (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> 2013</a>). The raw hybridization data (CEL files) was subjected to robust multiarray average (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Irizarry <em>et al.</em> 2003</a>), background correction, and normalization and quality controls were as in <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> (2013)</a>.</p><p id="p-9">Polymorphic positions were detected within each RIL population as described previously for RIL genotyping with the similarly designed Lettuce and Pepper GeneChip arrays (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">West <em>et al.</em> 2006</a>; <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2012</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> 2013</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> 2013</a>). Briefly, weighted hybridization intensities of all probes spanning each 2-bp window for each unigene were used to calculate the SPPdev value at each 2-bp position for all the hybridized GeneChip arrays. The SPPdev ratio is a measure of the hybridization difference between modes of a bimodal distribution of SPPdev values for a given position across all samples. If no bimodal distribution was found, the position was considered nonpolymorphic. For each polymorphic position, an allele call (A, B, or −) was assigned for each GeneChip. This was carried out iteratively for all positions on all unigenes. A genotype was assigned to that 2-bp position based on the SPPdev values relative to the GeneChips hybridized with parental DNA; the “A” genotype was assigned to alleles from the cultivated parents <em>C. annuum</em> ‘Early Jalapeño’ (NM) and ‘NuMex RNaky’ (FA) and the “B” genotype to alleles from ‘CM334’ and <em>C. frutescens</em> acc. 2814-6. Both replicates of an individual RIL had to have the same genotype for an assignment to be made. If the replicates were inconsistent, the haplotype was treated as missing data. When contiguous 2-bp positions were polymorphic, they were summarized as one SPP range. Summarized SPPs were filtered on the basis of several criteria, including minimum average SPPdev value of 1.1, maximum percentage of missing data of a haplotype of 10%, minimum number of valid probes (those hybridizing to more than 90% of the antigenomic probes) of two, and minimum number of bases spanning an SPP of two. The number of SPPs per unigene ranged from 1 to 191, with averages of 7.1 and 9.9 SPPs per unigene for NM and FA populations, respectively. Consensus haplotypes were called from multiple SPPs within a contig.</p></div><div id="sec-4" class="subsection"><h3>Linkage map construction</h3><p id="p-10">The haplotypes were determined for each RIL individual in the population with the methodology of <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> (2013)</a>. Briefly, multiple SPPs within the same contig were summarized into consensus unigene haplotypes with a Python script (<a href="https://github.com/huaxudavis/xuhu-rwm-map">https://github.com/huaxudavis/xuhu-rwm-map</a>). If the SPP calls within a unigene varied within a RIL, the two most likely consensus haplotypes were calculated and retained as markers for mapping (<a href="https://github.com/huaxudavis/xuhu-rwm-map/blob/master/split_spps.py">https://github.com/huaxudavis/xuhu-rwm-map/blob/master/split_spps.py</a>). Four iterations of mapping were used to determine genetic bins and assign positions. For the first iteration, unigene markers were assigned to linkage groups based on consensus haplotype(s) using MadMapper (<a href="https://github.com/alex-kozik/atgc-map">https://github.com/alex-kozik/atgc-map</a>; <a href="https://github.com/alex-kozik/atgc-map/blob/master/Python_MadMapper_V254_RECBIT_V131010.py">https://github.com/alex-kozik/atgc-map/blob/master/Python_MadMapper_V254_RECBIT_V131010.py</a>; <a href="https://github.com/alex-kozik/atgc-map/blob/master/py_matrix_2D_V254_RECBIT_V090710.py">https://github.com/alex-kozik/atgc-map/blob/master/py_matrix_2D_V254_RECBIT_V090710.py</a>). Pairwise recombination values among all markers were calculated for each linkage group with JoinMap 4.0 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Van Ooijen 2006</a>). Markers with zero recombination were considered to be one genetic bin. Therefore, within each line, the alleles for all markers within a bin should have originated from the same parent. The assignment of markers into genetic bins was not always unequivocal because of missing data. Adjacent markers had to be assigned to the same genetic bin when there was no recombination for some combinations of markers but there was recombination between other markers within the same bin (see <a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> 2013</a>). Genetic bins consisted of single unigene markers (singletons), multiple unigene markers with the same haplotype, haplotypes differing only by missing data (marker bins), or haplotypes that having A, B, and missing calls for a given RIL where recombination could not be unambiguously determined (<a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Truco <em>et al.</em> 2013</a>). The marker with the least missing values within a genetic bin was used for mapping iteration one. Markers within a linkage group were ordered using RECORD (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Van Os <em>et al.</em> 2005</a>). CheckMatrix heat plots were used to visually inspect and adjust marker grouping and order. Genetic distances (in centimorgan) were then calculated with JoinMap 4.0 (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Van Ooijen 2006</a>). Based on bin map positions, all unigenes were ordered and graphical genotypes were inspected for obvious errors in unigene placement.</p><p id="p-11">Because the SPP detection algorithm does not enable the determination of heterozygous alleles, mapping iterations two through four adjust for this by inputting heterozygous calls. Previous analyses have demonstrated that the SPP algorithm assigns heterozygous alleles as missing and A and B calls at a 2:1:1 ratio (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Hill <em>et al.</em>, 2013</a>). Regions of heterozygosity in individual RILs were identified by looking for contiguous regions with abnormally high rates of missing data compared with the same region in other RILs and high frequencies of apparent double crossovers. With the results of mapping iteration one, all SPPs were aligned with map positions. Graphical genotypes of the SPP calls were sorted by map position and visually inspected. SPPs with haplotypes inconsistent the bin haplotype, and other SPPs within their respective were removed. The missing data and allele frequencies across all remaining SPPs within each bin were then used to calculate the bin consensus allele for each RIL. Minimum requirements of &lt;80% missing and 66% A or B among nonmissing calls were used to assign an allele. All other cases were considered missing. Homozygous haplotypes were distinguished from heterozygous haplotypes by the use of a sliding window of 5 cM. Heterozygous regions were defined as those where missing values, and multiple crossovers were detected over multiple contiguous 5-cM regions. For mapping iteration two, the alleles within the heterozygous regions were converted to missing values. Ordering and positioning of markers within each linkage group was determined with RECORD_WIN for iterations two through four. Consensus bin haplotypes were then recalculated for bins/unigenes that were collapsed into new bins by RECORD_WIN during iteration two. For iteration three, heterozygous regions were refined where the order of bins within heterozygous regions had changed after iteration two. The regions determined to be heterozygous were given heterozygous calls, not missing as in iteration two, then all remaining double crossovers were identified and changed to missing data and marker order and positions were calculated. Following iteration three, allele assignments by bin were again recalculated. Heterozygous calls that were not contiguous were refined and double crossovers supported by multiple SPPs at a 100% A or B frequency with no missing data were included for the final iteration with RECORD_WIN. All RECORD_WIN determined bins that showed not recombination were combined into the final genetic bins to represent each map.</p><p id="p-12">For the FA map, linkage group assignment and orientation were based on conserved ortholog set II (COSII) markers between the parental lines <em>C. frutescens</em> var. BG2814-6 X <em>C. annuum</em> cv. NuMex RNaky that were previously assayed by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> (2009)</a>. Linkage group assignment for the NM map was based on markers common to the FA map. Because SPP detection parameters allowed a maximum allele frequency of 90%, skewed markers were included in the genetic maps. A χ<sup>2</sup> goodness-of-fit test for 1:1 segregation of each bin marker was performed to identify significantly skewed genetic bins at <em>P</em> &lt; 0.01.</p><p id="p-13">Collinearity between maps was determined from common mapped unigenes by the assignment of rank order to the markers in each map. When there were multiple markers within a genetic bin, rank order within the bin was assigned on the basis of the order of those markers in the second map. A bivariate fit of rank order was performed and a linear fit model was calculated using JMP software (SAS Campus Drive, Cary, NC).</p></div><div id="sec-5" class="subsection"><h3>Comparison of genetic maps to genome assemblies</h3><p id="p-14">The location of EST unigenes sequences in the pepper CM334 (v1.5) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>), pepper Zunla-1 (release 2.0) (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>), tomato (SL2.50, <a href="http://solgenomics.net/organism/1/genome">http://solgenomics.net/organism/1/genome</a>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fernandez-Pozo <em>et al.</em> 2015</a>), and potato (Stuberosum_206_v3.4, <a href="http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=Stuberosum">http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=Stuberosum</a>) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grigoriev <em>et al.</em> 2012</a>) genome assemblies were determined by the use of GMAP with default settings (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wu and Watanabe 2005</a>). The BLAT output files were filtered to select high-confidence hits. The pepper genome matches were filtered at 98% identity, minimum alignment length of 200 nucleotides (nt), and a maximum of 50 unaligned nt at either end of the unigene. If there were multiple hits with the same score for a given EST after filtering, the position of the EST in the genome was considered ambiguous and dropped from further analyses. The tomato and potato genome hits were filtered at minimums of 80% identity and 75% coverage with a minimum length of 200 nt. When there were multiple hits for a given EST, the hit with greatest identity and coverage was retained. If there was no single hit with both greatest identity and coverage, the EST was excluded from further analyses.</p><p id="p-15">Recombination rates (centimorgan per megabase pair) were calculated by taking the first derivative of the smoothed curve representing genetic <em>vs.</em> physical position (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ganal <em>et al.</em> 2011</a>). The smoothed curve for each linkage group was calculated using the Kernal Smoother function in JMP.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-16">The full set of SPP marker haplotypes, genetic maps (NM06 and FA07), comparison between maps, and mapping of unigenes to the Capsicum, Tomato, and Potato genomes have been added to the Sol Genomics Network FTP site (<a href="ftp://ftp.solgenomics.net/manuscripts/Hill_2015/">ftp://ftp.solgenomics.net/manuscripts/Hill_2015/</a>). Additional information, including the GeneChip assembly, GO annotations for assembled unigenes, single-nucleotide polymorphisms, SSRs also is available (<a href="https://pepper.ucdavis.edu/public/data.php">https://pepper.ucdavis.edu/public/data.php</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genotyping mapping populations using the Pepper GeneChip</h3><p id="p-17">The Pepper GeneChip genotyping array, containing ∼6.5 M 25-nt probes representing 30,815 unigenes, was used to detect polymorphisms within the intraspecific NM and interspecific FA RIL populations. The NM population was selected because it segregates for broad-spectrum resistance to <em>Phythopthora capsisi</em> from CM334 (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Sy <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rehrig <em>et al.</em> 2014</a>). The FA population was derived from the same cross producing F<sub>2</sub> individuals previously used to generate a 299 marker COSII pepper map that was aligned with tomato linkage groups (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>). Polymorphic positions (SPPs) within unigene sequences were detected on the basis of differential DNA hybridization values segregating within each population (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> 2012</a>; <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> 2013</a>). After polymorphism detection, three NM and four FA RILs were excluded from further analyses due to a high percentage of missing/ambiguous calls. Among the 63 NM and 119 FA RILs, 39,515 SPPs in 5587 unigenes (18% of those assayed) and 173,149 SPPs in 16,780 unigenes (55% of those assayed) were detected, respectively. Unigene consensus haplotypes (unigene markers) were used to generate genetic maps. The percentage of missing calls was 2.4% and 2.7% for NM and FA markers, respectively (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/020040SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS1.pdf">Table S1</a>)</p></div><div id="sec-9" class="subsection"><h3>Construction of genetic maps</h3><p id="p-18">To construct the FA map, unigene markers clustered into 11 groups using MadMapper. Following marker ordering within clusters, visual inspection with CheckMatrix demonstrated that one cluster was interrupted by a sharp reduction in recombination indicating this cluster may include two linkage groups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The two groups clustered together due to linkage of markers in the middle of the large linkage block to those of the smaller linkage block. This pseudolinkage suggested that this cluster included the chromosomal translocation pair P1 and P8, as observed in previous interspecific maps (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tanksley <em>et al.</em> 1988</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Park <em>et al.</em> 2014</a>). This was confirmed by linkage group assignment that used FA COSII markers. A high proportion of FA markers, 96% (16,154 unigenes), were assigned to genetic bins (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). There were 13 mapped FA unigenes having multiple haplotypes, and each of these mapped to two genetic bins. Each unigene with haplotypes mapping to two bins were considered as two distinct makers, resulting in a total of 16,167 markers in the FA map (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.xlsb">File S1</a>). The percentage of missing and heterozygous calls was 0.6% and 4.4%, respectively (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS1.pdf">Table S1</a>). The FA map consisted of 2105 genetic bins with a total distance of 1380 cM. There were 62−303 genetic bins and 242−3137 markers per linkage group. The average and maximum intervals between genetic bins were 0.66 and 11.05 cM, respectively. To construct the NM map, 1452 NM markers were clustered by MadMapper into 21 groups. These 21 clusters were aligned with the FA linkage groups by the use of 1312 common markers resulting in 12 NM linkage groups. In addition to the 21 clusters, there were 115 unassociated markers. FA map markers were used to assign these markers to linkage groups where possible. Therefore, NM markers or NM marker clusters that did not have common FA map markers could not be assigned to linkage groups, and the markers were excluded from the map. The ordering of markers by RECORD at a critical gap size of 30 cM followed by visual inspection confirmed linkage group assignation with the use of a critical gap size of 30 cM. Linkage group assignment followed by marker ordering and calculation of genetic distances resulted in 3876 (69%) NM markers assigned to map positions. Two unigenes had markers that mapped to two genetic bins, resulting in a total of 3878 NM map markers (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS2.xlsb">File S2</a>). The NM map consisted of 783 genetic bins with a total genetic distance of the 1399 cM (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The average and maximum intervals between genetic bins were 1.81, and 10.48 cM, respectively (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS1.pdf">Figure S1</a>). The number of genetic bins ranged from 53 to 102 and markers from 143 to 527 per linkage group. Linkage group P1 was the longest at 177 cM and P5 the shortest at 93 cM. The poorest marker coverage was found on P7, which had the fewest number of bins, total markers and the largest average marker interval.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat plot showing recombination between markers. MadMapper cluster including FA linkage groups P1 and P8. The arrow indicates the position of a clear break in linkage between P8 markers (upper left) and P1 markers. The heat plot also indicates the pseudolinkage between P8 and P1 markers (black dashed line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heat plot showing recombination between markers. MadMapper cluster including FA linkage groups P1 and P8. The arrow indicates the position of a clear break in linkage between P8 markers (upper left) and P1 markers. The heat plot also indicates the pseudolinkage between P8 and P1 markers (black dashed line).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F1.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2341/F1.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466964" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Heat plot showing recombination between markers. MadMapper cluster including FA linkage groups P1 and P8. The arrow indicates the position of a clear break in linkage between P8 markers (upper left) and P1 markers. The heat plot also indicates the pseudolinkage between P8 and P1 markers (black dashed line).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466959/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466959/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Statistics for <em>C. frutescens</em> acc. BG2814-6 × <em>C. annuum</em> ‘ NuMex RNaky’ (FA) and <em>C. annuum</em> ‘Early Jalapeño ’ × <em>C. annuum</em> ‘CM344’ (NM) maps</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Singletons accounted for 37% of both the NM and FA genetic bins (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS2.pdf">Table S2</a>). Although there were few large gaps, greater than 5 cM and 2 cM between genetic bins for the NM and FA maps respectively, marker density per cM was highly variable (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS1.pdf">Figure S1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Genetic bins having the largest numbers of markers were found on FA P1 at 137.5 cM with 624 unigene markers and NM P9 at 59.1 cM with 280 unigene markers (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS3.pdf">Table S3</a>). Five linkage groups had 15% or greater difference in size between the FA and NM maps (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). P3 (21%), P4 (15%), and P9 (16%) were longer in the FA map, and P10 (27%) and P12 (28%) were longer in the NM map. Markers with as low as 10% allele frequency were allowed during SPP detection and mapping. Markers having significant segregation distortion (<em>P</em> &lt; 0.01) mapped to several linkage groups (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS4.pdf">Table S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS5.pdf">Table S5</a>). The NM map had distorted markers on nine linkage groups and seven distorted regions spanning more than 5 cM, four skewed toward the Early Jalapeño allele on P6, P7, P9 and P11 and three toward the CM334 allele on P3, P5, and P6. The FA map had skewed markers on 10 linkage groups, with 19 regions skewed toward the cultivated, NuMex RNaky, allele and six toward the <em>C. frutescens</em> allele. There were large regions (≥10 cM) distorted toward the NuMex RNaky allele on P1, P2, P10, and P11 and two regions on P4 and P9. The most highly skewed bins (<em>P</em> &lt; 0.001) occurred across large regions at the tops of FA P2 (22 cM) and P12 (25 cM) toward the <em>C. frutescens</em> allele. Distorted markers were dispersed across different regions between the two maps with the exception of P9 where the pattern of distortion toward the cultivated allele appears similar between the two maps. Phenology and whole plant traits measured in the FA population by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yarnes <em>et al.</em> (2013)</a> identified QTL for days to maturity, stigma exsertion, and branching density that lay within regions of skewed segregation (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS6.pdf">Table S6</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker density on FA (red) and NM (blue) linkage groups. Histograms show the marker density per centimorgan across all 12 Capsicum linkage groups. Bars represent the number of unigene markers and those labeled with values indicate the size of centimorgan bins greater than 200 (FA) or 75 (NM) markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker density on FA (red) and NM (blue) linkage groups. Histograms show the marker density per centimorgan across all 12 Capsicum linkage groups. Bars represent the number of unigene markers and those labeled with values indicate the size of centimorgan bins greater than 200 (FA) or 75 (NM) markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F2.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2341/F2.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Marker density on FA (red) and NM (blue) linkage groups. Histograms show the marker density per centimorgan across all 12 Capsicum linkage groups. Bars represent the number of unigene markers and those labeled with values indicate the size of centimorgan bins greater than 200 (FA) or 75 (NM) markers.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele frequencies of markers used to generate the genetic maps. Scatter plots show the allele frequency of each bin marker by genetic position for (A) the FA parent NuMex RNAKY and (B) the NM parent Early Jalape&#xF1;o. Vertical lines delimit linkage groups indicated by the numbers at the top. The solid and dash horizontal lines represent allele frequencies resulting in significant &#x3C7;2 values (P &lt; 0.01 and P &lt; 0.001, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Allele frequencies of markers used to generate the genetic maps. Scatter plots show the allele frequency of each bin marker by genetic position for (A) the FA parent NuMex RNAKY and (B) the NM parent Early Jalape&#xF1;o. Vertical lines delimit linkage groups indicated by the numbers at the top. The solid and dash horizontal lines represent allele frequencies resulting in significant &#x3C7;&lt;sup&gt;2&lt;/sup&gt; values (&lt;em&gt;P&lt;/em&gt; &lt; 0.01 and &lt;em&gt;P&lt;/em&gt; &lt; 0.001, respectively).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F3.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2341/F3.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Allele frequencies of markers used to generate the genetic maps. Scatter plots show the allele frequency of each bin marker by genetic position for (A) the FA parent NuMex RNAKY and (B) the NM parent Early Jalapeño. Vertical lines delimit linkage groups indicated by the numbers at the top. The solid and dash horizontal lines represent allele frequencies resulting in significant χ<sup>2</sup> values (<em>P</em> &lt; 0.01 and <em>P</em> &lt; 0.001, respectively).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Collinearity between FA and NM maps</h3><p id="p-26">There were a total of 2667 unigenes in common to both the NM and FA maps (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS7.pdf">Table S7</a>). On the basis of on the relative genetic positions of the unigene markers on common linkage groups, there was a very high conservation of marker ordering between maps (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, and <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS2.pdf">Figure S2</a>). The overall coefficient of collinearity among unigenes mapping to common linkage groups was 0.99, and collinearity coefficients by linkage group were 0.99−1.00, with the exception of the translocated distal arm of P1 cultivated and P8 wild at 0.96 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Minor obvious differences in collinearity between FA and NM maps were on P5, P6, P10, and P12 (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS3.xlsb">File S3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466928/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466928/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Collinearity between common mapped markers</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regression of the order of markers on common linkage groups between the FA and NM linkage maps. The circles show the ordered 2108 markers common to both maps found on linkage groups 2&#x2013;7 and 9&#x2013;12 based on their map positions for both maps. Rank orders within each linkage group were used for regression analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Regression of the order of markers on common linkage groups between the FA and NM linkage maps. The circles show the ordered 2108 markers common to both maps found on linkage groups 2&#x2013;7 and 9&#x2013;12 based on their map positions for both maps. Rank orders within each linkage group were used for regression analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F4.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2341/F4.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Regression of the order of markers on common linkage groups between the FA and NM linkage maps. The circles show the ordered 2108 markers common to both maps found on linkage groups 2–7 and 9–12 based on their map positions for both maps. Rank orders within each linkage group were used for regression analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regression of marker order between the FA (wild) and NM (cultivated) P1 and P8 linkage groups. The order of 483 markers common to both maps found on linkage groups P1 and P8, where a translocation has occurred between C. frutescens and C. annuum. Common markers were ranked based on their map positions for both maps then rank orders within each linkage group pair (P1 cultivated and P1 wild, P1 cultivated and P8 wild, P8 cultivated and P8 wild) were used for regression analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Regression of marker order between the FA (wild) and NM (cultivated) P1 and P8 linkage groups. The order of 483 markers common to both maps found on linkage groups P1 and P8, where a translocation has occurred between &lt;em&gt;C. frutescens&lt;/em&gt; and &lt;em&gt;C. annuum&lt;/em&gt;. Common markers were ranked based on their map positions for both maps then rank orders within each linkage group pair (P1 cultivated and P1 wild, P1 cultivated and P8 wild, P8 cultivated and P8 wild) were used for regression analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F5.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2341/F5.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467069" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Regression of marker order between the FA (wild) and NM (cultivated) P1 and P8 linkage groups. The order of 483 markers common to both maps found on linkage groups P1 and P8, where a translocation has occurred between <em>C. frutescens</em> and <em>C. annuum</em>. Common markers were ranked based on their map positions for both maps then rank orders within each linkage group pair (P1 cultivated and P1 wild, P1 cultivated and P8 wild, P8 cultivated and P8 wild) were used for regression analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The large chromosomal translocation between the bottom arm of P1 and P8 between <em>C. annuum</em> and other <em>Capsicum</em> species was known to be segregating in the FA population resulting in P1 and P8 being represented by four chromosomes which have been designated P1 wild, P8 wild, P1 cultivated, and P8 cultivated (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Tanksley <em>et al.</em> 1988</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>). A comparison of P1 and P8 markers common to both maps demonstrated a clear break point on P1 wild (FA map) within the genetic bin at 137.5 cM (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). All common map markers distal to 137.5 cM on FA P1 wild map to P8 cultivated (NM map). Similarly, all common markers distal to NM 133.8 cM map to FA P8 wild. A number of markers mapping to P1 cultivated at 114.4−131.6 cM map to P8 wild at 0−12.1 cM in the FA population, however, several of these makers lack collinearity between the two maps indicating the association with P1 may be due to pseudolinkage between P1 wild and P8 wild markers.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination heat plot and painted chromosomes showing recombination between markers on P1 and P8 and common markers between FA and NM. The heat plot upper left triangle shows recombination between markers on FA P1 (wild) and P8 (wild). The lower right corresponds to NM P1 (cultivated) and P8 (cultivated) marker recombination. Chromosomal regions are colored according to translocation pairs. These regions are boxed in the same color on the heat plot. The nontranslocated portion of P1 (P1 wild/P1 cultivated) are shown in blue. The translocated arms P8 wild/P1 cultivated are in aqua and P1 wild/P8 cultivated are in yellow. Gray corresponds to the region of pseudolinkage between P1 and P8 wild and red indicates the region where the translocation break point occurred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Recombination heat plot and painted chromosomes showing recombination between markers on P1 and P8 and common markers between FA and NM. The heat plot upper left triangle shows recombination between markers on FA P1 (wild) and P8 (wild). The lower right corresponds to NM P1 (cultivated) and P8 (cultivated) marker recombination. Chromosomal regions are colored according to translocation pairs. These regions are boxed in the same color on the heat plot. The nontranslocated portion of P1 (P1 wild/P1 cultivated) are shown in blue. The translocated arms P8 wild/P1 cultivated are in aqua and P1 wild/P8 cultivated are in yellow. Gray corresponds to the region of pseudolinkage between P1 and P8 wild and red indicates the region where the translocation break point occurred.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F6.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/11/2341/F6.medium.gif" width="305" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466973" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Recombination heat plot and painted chromosomes showing recombination between markers on P1 and P8 and common markers between FA and NM. The heat plot upper left triangle shows recombination between markers on FA P1 (wild) and P8 (wild). The lower right corresponds to NM P1 (cultivated) and P8 (cultivated) marker recombination. Chromosomal regions are colored according to translocation pairs. These regions are boxed in the same color on the heat plot. The nontranslocated portion of P1 (P1 wild/P1 cultivated) are shown in blue. The translocated arms P8 wild/P1 cultivated are in aqua and P1 wild/P8 cultivated are in yellow. Gray corresponds to the region of pseudolinkage between P1 and P8 wild and red indicates the region where the translocation break point occurred.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Only 83 unigenes mapped to different linkage groups between populations, excluding P1/P8 (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS7.pdf">Table S7</a>). When queried against the Illumina transcriptome assembly of <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Ashrafi <em>et al.</em> (2012)</a> and the Zunla and CM334 genome assemblies (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>), 77 and 40 of these unigenes had multiple transcriptome and genome hits, respectively. In addition, sequences of 22 unigenes were found on both chromosome pseudomolecules to which they were mapped. This finding suggests, that unigenes mapping to different linkage groups between populations tend to belong to multigene families and are mapping to different paralogs. In total, 76 of the 83 unigenes were found on a chromosome pseudomolecule in common with at least one of the mapped linkage groups with more matches to FA, 62 unigenes, than NM, 38 unigenes, linkage groups.</p></div><div id="sec-11" class="subsection"><h3>Collinearity between genetic and physical positions</h3><p id="p-35">Of the 30,173 pepper unigenes represented on the Pepper GeneChip, 26,725 were identified at 98% identity or greater in the CM334 v1.5 genome assembly (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>). A unique match for 25,472 unigenes (84%) was identified and 18,930 (62%) had been placed on chromosome pseudomolecules (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS8.pdf">Table S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS4.xlsb">File S4</a>). There were 68% of the FA and 67% of the NM mapped unigenes placed on CM334 chromosomes with 98% (10,866) and 96% (2511) of these on shared linkage group/chromosome pairs, respectively (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS9.pdf">Table S9</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS10.pdf">Table S10</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS4.pdf">Figure S4</a>) (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>). A query against the <em>C. annuum</em> ‘Zunla-1’ 2.0 assembly (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>) found 23,848 EST unigenes that matched the Zunla genome sequences and positions for 23,332 unigenes were unambiguously identified with 21,194 (70%) placed on chromosome pseudomolecules (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS11.pdf">Table S11</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS12.pdf">Table S12</a>; <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS4.pdf">Figure S4</a>). There were 83% (13,488) and 81% (3,125) of the FA and NM markers identified in the assembly with 12,242 and 2773 found on chromosomes and 97% and 95% of these on common linkage group/chromosome pairs, respectively.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between physical and genetic positions with corresponding recombination rates. The 8825 FA and 2131 NM map markers that have been placed on chromosomes 2&#x2013;7 and 9&#x2013;12. The left Y-axis represents the centimorgan position of FA (red circles) and NM (blue triangles) unigene markers and the X-axis represents the physical position (Mbp) of unigenes on the CM334 pseudomolecules. The right Y-axis represents the recombination frequency, cM/Mbp, along the chromosomes for the FA (red line) and NM (blue dash line) maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between physical and genetic positions with corresponding recombination rates. The 8825 FA and 2131 NM map markers that have been placed on chromosomes 2&#x2013;7 and 9&#x2013;12. The left Y-axis represents the centimorgan position of FA (red circles) and NM (blue triangles) unigene markers and the X-axis represents the physical position (Mbp) of unigenes on the CM334 pseudomolecules. The right Y-axis represents the recombination frequency, cM/Mbp, along the chromosomes for the FA (red line) and NM (blue dash line) maps.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F7.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/11/2341/F7.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467013" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-36" class="first-child">Relationship between physical and genetic positions with corresponding recombination rates. The 8825 FA and 2131 NM map markers that have been placed on chromosomes 2–7 and 9–12. The left Y-axis represents the centimorgan position of FA (red circles) and NM (blue triangles) unigene markers and the X-axis represents the physical position (Mbp) of unigenes on the CM334 pseudomolecules. The right Y-axis represents the recombination frequency, cM/Mbp, along the chromosomes for the FA (red line) and NM (blue dash line) maps.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between physical and genetic positions with corresponding recombination rates for chromosomes 1 and 8. The 2240 FA and 387 NM map markers that have been placed on CM334 chromosomes 1 and 8. The left Y-axis indicates the linkage group (LG) and centimorgan position of FA (red circles) and NM (blue triangles) unigene markers. The X-axis represents the physical position (megabase pair) of unigenes on the pseudomolecules. The right Y-axis represents the recombination frequency in cM/Mbp along the chromosomes for the FA (red line) and NM (blue dash line) maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between physical and genetic positions with corresponding recombination rates for chromosomes 1 and 8. The 2240 FA and 387 NM map markers that have been placed on CM334 chromosomes 1 and 8. The left Y-axis indicates the linkage group (LG) and centimorgan position of FA (red circles) and NM (blue triangles) unigene markers. The X-axis represents the physical position (megabase pair) of unigenes on the pseudomolecules. The right Y-axis represents the recombination frequency in cM/Mbp along the chromosomes for the FA (red line) and NM (blue dash line) maps.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F8.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/11/2341/F8.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-37" class="first-child">Relationship between physical and genetic positions with corresponding recombination rates for chromosomes 1 and 8. The 2240 FA and 387 NM map markers that have been placed on CM334 chromosomes 1 and 8. The left Y-axis indicates the linkage group (LG) and centimorgan position of FA (red circles) and NM (blue triangles) unigene markers. The X-axis represents the physical position (megabase pair) of unigenes on the pseudomolecules. The right Y-axis represents the recombination frequency in cM/Mbp along the chromosomes for the FA (red line) and NM (blue dash line) maps.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">There were 3629 FA markers that matched CM334 sequences on chr00 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS4.xlsb">File S4</a>). These sequences have been assembled into scaffolds. There were matches to 520 scaffolds, 390 scaffolds had multiple unigene matches, and 346 (89%) of those were to a single FA LG. A total of 330 scaffolds matched multiple unigene markers mapping within 5 cM and 301 scaffolds had markers mapping to a single genetic bin, accounting for 2507 and 779 FA markers, respectively.</p><p id="p-39">The assignment of chromosomal positions was highly conserved between both maps and each genome, with collinearity coefficients of 0.999 and greater (<a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/FigureS4.pdf">Figure S4</a>). However, six pseudomolecules of the Zunla assembly are inverted relative to the P3, 4, 6, 9, 11, and 12 linkage groups (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>). The chromosomes were densely covered by the FA markers with only a few, small segments of physical positions not covered. On the basis of FA markers associated with the 3.02 Gb of assembled CM334 chromosomes, 1.39 Gb (46%) of the genome was nonrecombining. There were several physical segments devoid of NM markers, generally corresponding to large physical segments with little to no genetic recombination. In general, the two linkage maps displayed similar patterns of recombination with the distal regions of chromosomes showing greater recombination and the central, likely pericentromeric, regions with greatly reduced recombination. The acrocentric chromosomes 2 and 8 showed concomitant reduced recombination near the top of the chromosome.</p></div><div id="sec-12" class="subsection"><h3>Syntenic analyses</h3><p id="p-40">We analyzed synteny between each pepper map and the assembled genomes of pepper’s close solanaceous relatives, tomato and potato (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS5.xlsb">File S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS13.pdf">Table S13</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS14.pdf">Table S14</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS15.pdf">Table S15</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/08/g3.115.020040.DC1/TableS16.pdf">Table S16</a>, and <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). Alignment of the FA and NM maps to both genomes was similar, demonstrating good concordance between both pepper maps and the tomato and potato genomes. Dot plots showing relative pepper genetic positions <em>vs.</em> relative physical positions on each chromosome group are highly similar between the tomato and potato assemblies. These plots demonstrate several translocations in common between tomato and potato with pepper. In addition to the eight major translocations previously described (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Tanksley <em>et al.</em> 1988</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wu and Tanksley 2010</a>), there appears to be a translocation between the nonrecombining region on P4 and T11/S11 and possibly T12/S12 that has, to our knowledge, not been observed by syntenic analyses between pepper maps and tomato.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pepper maps vs. tomato and potato genomes. Genetic and physical positions of FA (red circles) and NM (blue dots) markers matching tomato and potato genome sequences &#x2265; 80% ID and &#x2265;75% unigene coverage. (A) Positions of 8267 mapped unigenes, 7660 FA and 1,755 NM found on tomato chromosomes. (B) Positions of 8131 mapped unigenes, 7551 FA and 1723 NM found on the potato chromosomes. Green circles represent previously identified translocations between pepper and tomato whereas the P4/Sl11 and St11 clusters (highlighted in yellow) represent a previously unmapped translocation event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638341129" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pepper maps &lt;em&gt;vs.&lt;/em&gt; tomato and potato genomes. Genetic and physical positions of FA (red circles) and NM (blue dots) markers matching tomato and potato genome sequences &#x2265; 80% ID and &#x2265;75% unigene coverage. (A) Positions of 8267 mapped unigenes, 7660 FA and 1,755 NM found on tomato chromosomes. (B) Positions of 8131 mapped unigenes, 7551 FA and 1723 NM found on the potato chromosomes. Green circles represent previously identified translocations between pepper and tomato whereas the P4/Sl11 and St11 clusters (highlighted in yellow) represent a previously unmapped translocation event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2341/F9.medium.gif" width="223" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/5/11/2341/F9.medium.gif" width="223" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2341/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467078" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-41" class="first-child">Pepper maps <em>vs.</em> tomato and potato genomes. Genetic and physical positions of FA (red circles) and NM (blue dots) markers matching tomato and potato genome sequences ≥ 80% ID and ≥75% unigene coverage. (A) Positions of 8267 mapped unigenes, 7660 FA and 1,755 NM found on tomato chromosomes. (B) Positions of 8131 mapped unigenes, 7551 FA and 1723 NM found on the potato chromosomes. Green circles represent previously identified translocations between pepper and tomato whereas the P4/Sl11 and St11 clusters (highlighted in yellow) represent a previously unmapped translocation event.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-42">We constructed ultrahigh density EST-based genetic maps for <em>Capsicum</em> using GeneChip technology. An interrogation of 30,173 unigenes segregating in two RIL populations resulted in high-quality, comprehensive <em>Capsicum</em> genetic maps: a 16,780 unigene interspecific and a 3878 unigene intraspecific map with a total of 17,365 nonredundant unigenes mapped. The maps capture variation in fairly diverse germplasm, including a small fruited <em>C. frutescens</em> variety and <em>C. annuum</em> lines from three distinct gene pools, the semidomesticated disease-resistant CM334 variety and Jalapeño and Anaheim types. These maps enable genetic studies in <em>Capsicum</em> and comparisons between Solanaceous species. We have reported previously an immediate application of this resource for molecular breeding through high-resolution mapping of QTL for multiple traits (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brand <em>et al.</em> 2012</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Yarnes <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Naegele <em>et al.</em> 2013</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rehrig <em>et al.</em> 2014</a>).</p><p id="p-43">The haploid chromosome number for <em>Capsicum</em> is 12; however, when details have been reported for construction of pepper maps, <em>de novo</em> clustering of markers into 12 linkage groups has been rare, with upwards of 20 linkage groups being identified (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Prince <em>et al.</em> 1993</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chaim <em>et al.</em> 2001</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lefebvre <em>et al.</em> 2002</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sugita <em>et al.</em> 2005</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barchi <em>et al.</em> 2007</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Truong <em>et al.</em> 2010</a>). Common pepper map markers, integrated maps, and syntenic relationships with tomato have been used to assemble multiple small groups into chromosomal linkage groups (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Paran <em>et al.</em> 2004</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Barchi <em>et al.</em> 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mimura <em>et al.</em> 2012</a>). As expected, the increase in marker density and coverage attributable to <em>de novo</em> identification of gene-based markers and the fairly even distribution of genes along pepper chromosomes (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>) resulted in FA markers clustering into 12 linkage groups.</p><p id="p-44">Initially, FA linkage groups P1 and P8 formed a single cluster due to pseudolinkage of markers on the translocated chromosomal arms between <em>C. annuum</em> and <em>C. frutescens</em>. This has been observed in previous interspecific maps between <em>C. annuum</em> and <em>C. frutescens</em> or <em>C. chinense</em> (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Park <em>et al.</em> 2014</a>). Our use of CheckMatrix to visualize recombination patterns provided an obvious demarcation between the P1 and P8 linkage group, although pseudolinkage prohibited the chromosomal assignment of a few markers near the translocation break point without the guidance of the intra-specific NM map that lacks the translocations. A comparison of both maps indicates the P1/P8 translocation break point is coincident with the most marker dense genetic bin, in an approximately 120 Mb nonrecombining region, of P1. Identifying the precise physical position of the breakpoint may be difficult without whole genome sequencing of the RILs.</p><p id="p-45">The NM map data, having less than one fourth the markers and about half of the population size of the FA, was insufficient for clustering markers into 12 groups. This also may explain the inability to map 30% of the NM polymorphic unigenes, a phenomenon common to previous pepper mapping efforts, particularly for intraspecific maps of <em>C. annuum</em> (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Prince <em>et al.</em> 1993</a>; <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kang <em>et al.</em> 2001</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Barchi <em>et al.</em> 2007</a>). The common map markers indicate apparent errors in marker grouping can occur when mapping members of multigene families. This would be more pronounced when detecting polymorphisms in more closely related lines.</p><p id="p-46">Marker ordering was carried out <em>de novo</em> for each map independently and resulted in very high collinearity between maps. The lowest collinearity coefficient was between P8 wild and P1 cultivated, consistent with pseudolinkage (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Livingstone <em>et al.</em> 2000</a>). In addition, there were some isolated differences in marker ordering, which tended to be at the top or bottom of linkage groups. Marker ordering can be influenced by missing calls or calling errors, particularly in high-density maps (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackett and Broadfoot 2003</a>). The SPP detection algorithms cannot discriminate heterozygous alleles from homozygotes, which likely resulted in both missing and erroneous calls (<a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Hill <em>et al.</em> 2013</a>). Because individual calls were collapsed into consensus haplotypes for each unigene and then consensus haplotypes for each genetic bin before calculating distance, heterozygous alleles could be determined by their high percentage of missing values and low A or B allele frequencies. Overall, missing or erroneous calls seem to have had little impact on original marker ordering, which had little change during subsequent mapping iterations where heterozygous call corrections were implemented.</p><p id="p-47">Marker distribution was similar between chromosomes for both maps; however, marker distribution on P7 had unique features in each map. The NM linkage group 7 had the largest marker interval, lowest marker density and, compared with the CM334 chromosome 7, had the smallest fraction of unigene markers per gene (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>). These features all indicate reduced polymorphism on P7 between the intraspecific NM parents. Reduced polymorphism was not apparent on FA P7. However, this was the only linkage group with multiple 1-cM bins having more than 300 markers. The two marker-dense bins were separated by more than 50 cM and corresponded to both the pericentromeric region and region of reduced recombination near the distal end of chromosome 7. Similarly, the COSII map (F2 of the FA RIL population) had a large gap (&gt;20 cM) on P7 separating two clusters of markers. The reduced recombination indicated the possibility of a paracentric inversion between the COSII parents (<a id="xref-ref-58-7" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>). However, marker collinearity between the FA and NM markers on P7 did not support a paracentric inversion between FA parents. Alternatively, the high density of markers near the distal end of the chromosome points to increased variation, resulting in reduced recombination near the subtelomeric region of chromosome 7 between <em>C. frutescens</em> and <em>C. annuum</em>. Chromosome subtelomeric regions can have high rates of sequence variation and have been shown to be important for recombination (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Calderon <em>et al.</em> 2014</a>). Whether this is related to the reduced polymorphism in the intraspecific population is unclear.</p><p id="p-48">The total lengths of the FA and NM maps were similar, close to the 1390-cM map length estimated by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lefebvre <em>et al.</em> (1995)</a>. The FA map was 19 cM (1.4%) shorter than the NM map, consistent with the expectation of reduced recombination in the interspecific population. However, there were five linkage groups that were longer in the FA map. This finding could be explained by incomplete chromosomal coverage at the ends as the result of reduced polymorphism, expansion caused by segregation distortion, or elevated heterozygosity. Differences of chromosomal coverage could be determined by comparing the relative positions of markers common to both maps at the top and bottom of linkage groups. Of the five FA linkage groups that were longer than their NM counterparts, FA markers on P2, P5, and P9 extended further by a total of 30 cM, which would result in a relative underestimation of NM length. Significant segregation distortion was observed in 10 FA linkage groups, including severe marker distortion on P2 favoring the <em>C. frutescens</em> allele at the top and the NuMex RNaky allele at the bottom which may result in the overestimation of recombination. Also, contrary to expectation, comparisons between the two maps and the maps with the genome indicate that the bottom of chromosome 4 and tops of chromosomes 3, and 6 appear to have less recombination in the NM <em>vs.</em> FA map. Overall, these features suggest that both chromosomal coverage at the ends and segregation distortion may have contributed in part to the underestimation of NM length and/or overestimation of FA length for several linkage groups. However, the linkage group with the largest expansion in the FA map was P3 and this appears to be primarily due to reduced recombination at the top of NM P3.</p><p id="p-49">Comparisons between genetic distances and physical distances showed similar patterns of recombination rate variation along the pepper chromosomes. Strong recombination repression occurs over long chromosomal regions, which likely correspond to pericentromeric regions. This pattern of recombination also has been observed in tomato and other species (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Ganal <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huo <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sim <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhang <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Deokar <em>et al.</em> 2014</a>). However, recombination repression appears to be more pronounced in pepper, most similar to what has been observed in tomato, suggesting that it may be a common feature in the Solanaceae. Despite fairly even genetic map coverage, there were several large gaps in chromosomal coverage by NM markers on all linkage groups except P1, P9, and P11. These gaps corresponded to the nonrecombining regions, also demonstrated by the absence of the extremely marker dense 1 cM bins that were observed in FA linkage groups. This shows that although there are polymorphic genes present in these chromosomal regions in the FA population, there was a smaller proportion of NM polymorphic genes in the nonrecombining regions. Regions having low recombination have been shown to have less variation in both plants and animals (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Begun and Aquadro 1992</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Buckler and Thornsberry 2002</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lercher and Hurst 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nachman 2002</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Roselius <em>et al.</em> 2005</a>). This, along with the fourfold reduction in NM <em>vs.</em> FA markers explains, at least in part, the gaps in chromosomal coverage by NM markers. Confounding this would be the potential for gaps in the genome assembly across these regions, because scaffold ordering cannot benefit from genetic maps.</p><p id="p-50">Syntenic analyses between the FA and NM maps and the tomato (Sl) and potato (St) genome assemblies confirm 10 translocations that were previously reported (<a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Livingstone <em>et al.</em> 1999</a>; <a id="xref-ref-58-8" class="xref-bibr" href="#ref-58">Wu <em>et al.</em> 2009</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Wu and Tanksley 2010</a>; <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> 2014</a>). Additional translocations were observed between P4 and SlChr11/StChr11 and P4 and SlChr12/StChr12. Both of these translocations correspond to the nonrecombining region on P4 and were only observed with the greater density FA map and the genome assembly of <a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Qin <em>et al.</em> (2014)</a> in a detailed comparison of the pepper and tomato genomes. Because the intraspecific maps used to assemble the pepper genomes had half the markers of the full FA map, and we observed reduced polymorphism in the nonrecombining regions of the intraspecific NM map, the P4 centromeric region may not be completely assembled at this time. Interestingly, there is also a translocation of the top arm of P4, including markers from the nonrecombining region that correspond to the top of SlChr03/StChr03. This finding indicates that there have been as many as three translocation events between pepper and tomato/potato within the nonrecombining region of P4, requiring five chromosomal breakages.</p><p id="p-51">Most pepper maps to-date are primarily based on anonymous markers and thus difficult to compare in many cases. Because the maps presented here are gene-based, they can be easily associated with other gene-based genetic maps regardless of the polymorphic positon(s) within the gene. Thus, trait-marker associations and candidate gene identification will be facilitated and easily assayed across populations. The utility of the maps has already been demonstrated by their use in an in-depth study of 39 phenological, leaf, fruit, morphological, and capsaicinoid traits and identification of a <em>Phytophthora</em> resistance gene candidate (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Brand <em>et al.</em> 2012</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Yarnes <em>et al.</em> 2013</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Naegele <em>et al.</em> 2013</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Rehrig <em>et al.</em> 2014</a>). In addition, the maps were used to aid assembly of the recently released <em>Capsicum</em> reference genome based on CM334 (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Kim <em>et al.</em> 2014</a>). Even so, we found more than 300 unassembled scaffolds that were each assigned to a single genetic bin by FA map markers. This gives approximate chromosomal positions for these scaffolds and their gene compliments, providing critical information for candidate gene identification. These maps will enable further trait associations and gene cloning in <em>Capsicum</em> as well as comparisons with gene-based genetic maps and genome assemblies across species.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">Our project coordinated the RILs developed by Cornell, Syngenta, Rijk Zwaan BV, and Enza Zaden BV. Thanks to Nunhems BV for genotyping the FA RILs using COS II markers. This project was supported by an award from the University of California UC Discovery Program bio06-10565 with matching funds from Rijk Zwaan BV, Enza Zaden BV, DeRuiter Seeds B.V., Nunhems USA, Seminis Vegetable Seeds, Inc., Syngenta Seeds, Inc., and Vilmorin Co.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-53">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-54"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-01"><span class="received-label">Received </span>July 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-03"><span class="accepted-label">Accepted </span>September 3, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hill <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.1" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.2" data-doi="10.1186/1471-2164-13-571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>De novo</em> assembly of the pepper transcriptome (<em>Capsicum annuum</em>): a benchmark for in silico discovery of SNPs, SSRs and candidate genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAshrafi%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D571%26rft.epage%253D571%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Bof%2Bthe%2Bpepper%2Btranscriptome%2B%2528Capsicum%2Bannuum%2529%253A%2Ba%2Bbenchmark%2Bfor%2Bin%2Bsilico%2Bdiscovery%2Bof%2BSNPs%252C%2BSSRs%2Band%2Bcandidate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-571%26rft_id%253Dinfo%253Apmid%252F23110314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23110314&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barchi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Signoret</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high-resolution, intraspecific linkage map of pepper (<em>Capsicum annuum L</em>.) and selection of reduced recombinant inbred line subsets for fast mapping.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBarchi%26rft.auinit1%253DL.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D60%26rft.atitle%253DA%2Bhigh-resolution%252C%2Bintraspecific%2Blinkage%2Bmap%2Bof%2Bpepper%2B%2528Capsicum%2Bannuum%2BL.%2529%2Band%2Bselection%2Bof%2Breduced%2Brecombinant%2Binbred%2Bline%2Bsubsets%2Bfor%2Bfast%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F17546071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17546071&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.4" data-doi="10.1038/356519a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.</span> <span class="cit-source">Nature</span> <span class="cit-vol">356</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D356%26rft.issue%253D6369%26rft.spage%253D519%26rft.epage%253D520%26rft.atitle%253DLevels%2Bof%2Bnaturally%2Boccurring%2BDNA%2Bpolymorphism%2Bcorrelate%2Bwith%2Brecombination%2Brates%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F356519a0%26rft_id%253Dinfo%253Apmid%252F1560824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/356519a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1560824&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HN23100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben Chaim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">G. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanyolac</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">fs3.1: a major fruit shape QTL conserved in <em>Capsicum</em>.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBen%2BChaim%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253Dfs3.1%253A%2Ba%2Bmajor%2Bfruit%2Bshape%2BQTL%2Bconserved%2Bin%2BCapsicum.%26rft_id%253Dinfo%253Apmid%252F12669791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12669791&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.6" data-doi="10.1007/s00122-006-0395-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Chaim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falise</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazourek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTL analysis for capsaicinoid content in <em>Capsicum</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1481</span>–<span class="cit-lpage">1490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBen-Chaim%26rft.auinit1%253DA.%26rft.volume%253D113%26rft.issue%253D8%26rft.spage%253D1481%26rft.epage%253D1490%26rft.atitle%253DQTL%2Banalysis%2Bfor%2Bcapsaicinoid%2Bcontent%2Bin%2BCapsicum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0395-y%26rft_id%253Dinfo%253Apmid%252F16960715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0395-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16960715&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2341.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosland</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Votava</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Peppers: Vegetable and Spice Capsicums</span>. <span class="cit-publ-name">Cabi Publishing</span>, <span class="cit-publ-loc">Wallingford, Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.8" data-doi="10.1007/s00425-011-1530-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meir</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogachev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aharoni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">pc8.1, a major QTL for pigment content in pepper fruit, is associated with variation in plastid compartment size.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D235%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-011-1530-9%26rft_id%253Dinfo%253Apmid%252F21987007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-011-1530-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21987007&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300850900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.9" data-doi="10.1016/S1369-5266(02)00238-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span>  <span class="cit-name-suffix">IV</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Plant molecular diversity and applications to genomics.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D107%26rft.epage%253D111%26rft.atitle%253DPlant%2Bmolecular%2Bdiversity%2Band%2Bapplications%2Bto%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5266%252802%252900238-8%26rft_id%253Dinfo%253Apmid%252F11856604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5266(02)00238-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11856604&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174097000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.10" data-doi="10.1038/srep06488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calderon</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">M.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prieto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The subtelomeric region is important for chromosome recognition and pairing during meiosis.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">6488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D6488%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep06488%26rft_id%253Dinfo%253Apmid%252F25270583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep06488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270583&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.11" data-doi="10.1007/s001220000461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaim</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grube</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Wijk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">QTL mapping of fruit-related traits in pepper (<em>Capsicum annuum</em>).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1016</span>–<span class="cit-lpage">1028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D1016%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220000461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220000461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169278600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.12" data-doi="10.1093/jxb/ers172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David-Schwartz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CaJOINTLESS is a MADS-box gene involved in suppression of vegetative growth in all shoot meristems in pepper.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">4947</span>–<span class="cit-lpage">4957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fers172%26rft_id%253Dinfo%253Apmid%252F22859675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2My8xMy80OTQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.13" data-doi="10.1186/1471-2164-15-708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deokar</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diapari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sindhu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D708%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-708%26rft_id%253Dinfo%253Apmid%252F25150411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25150411&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.14" data-doi="10.1093/nar/gku1195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Pozo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tecle</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Sol Genomics Network (SGN)—from genotype to phenotype to breeding.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D1036</span>–<span class="cit-lpage">D1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1195%26rft_id%253Dinfo%253Apmid%252F25428362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0My9EMS9EMTAzNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.15"><div class="cit-metadata unstructured">Food and Agriculture Organization of the Unite States, 2012 FAOSTAT. Available at: <a href="http://faostat.fao.org/site/339/default.aspx">http://faostat.fao.org/site/339/default.aspx</a>. Accessed September 21, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.16" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (<em>Zea mays L</em>.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.17" data-doi="10.6026/97320630008043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Góngora-Castillo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fajardo-Jaime</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Cortes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jofre-Garfias</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozoya-Gloria</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Capsicum</em> Transcriptome DB: a “hot” tool for genomic research.</span> <span class="cit-source">Bioinformation</span> <span class="cit-vol">8</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBioinformation%26rft.aulast%253DGongora-Castillo%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D47%26rft.atitle%253DThe%2Bcapsicum%2Btranscriptome%2BDB%253A%2Ba%2B%2522hot%2522%2Btool%2Bfor%2Bgenomic%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.6026%252F97320630008043%26rft_id%253Dinfo%253Apmid%252F22359434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.6026/97320630008043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22359434&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.18" data-doi="10.1093/nar/gkr947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordberg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome portal of the Department of Energy Joint Genome Institute.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D26</span>–<span class="cit-lpage">D32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr947%26rft_id%253Dinfo%253Apmid%252F22110030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwL0QxL0QyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.19" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.20" data-doi="10.1371/journal.pone.0056200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterization of <em>Capsicum annuum</em> genetic diversity and population structure based on parallel polymorphism discovery with a 30k unigene pepper GeneChip.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e56200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De56200%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0056200%26rft_id%253Dinfo%253Apmid%252F23409153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0056200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23409153&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.21" data-doi="10.1093/dnares/dsu027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyatake</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunome</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negoro</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Draft genome sequence of eggplant (<em>Solanum melongena L</em>.): the representative <em>Solanum</em> species indigenous to the old world.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu027%26rft_id%253Dinfo%253Apmid%252F25233906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvNjQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.22" data-doi="10.1007/s00122-011-1598-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garvin</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparison of a high-density genetic linkage map to genome features in the model grass <em>Brachypodium distachyon</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHuo%26rft.auinit1%253DN.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D455%26rft.epage%253D464%26rft.atitle%253DComparison%2Bof%2Ba%2Bhigh-density%2Bgenetic%2Blinkage%2Bmap%2Bto%2Bgenome%2Bfeatures%2Bin%2Bthe%2Bmodel%2Bgrass%2BBrachypodium%2Bdistachyon.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1598-4%26rft_id%253Dinfo%253Apmid%252F21597976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1598-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21597976&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.23" data-doi="10.1093/biostatistics/4.2.249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobbs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beazer-Barclay</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonellis</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.stitle%253DBiostatistics%26rft.aulast%253DIrizarry%26rft.auinit1%253DR.%2BA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D264%26rft.atitle%253DExploration%252C%2Bnormalization%252C%2Band%2Bsummaries%2Bof%2Bhigh%2Bdensity%2Boligonucleotide%2Barray%2Bprobe%2Blevel%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252F4.2.249%26rft_id%253Dinfo%253Apmid%252F12925520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjQvMi8yNDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.24" data-doi="10.1007/s00425-011-1479-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeifetz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David-Schwartz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CaBLIND regulates axillary meristem initiation and transition to flowering in pepper.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">234</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D234%26rft.spage%253D1227%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-011-1479-8%26rft_id%253Dinfo%253Apmid%252F21773792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-011-1479-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21773792&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.25" data-doi="10.1007/s001220051678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nahm</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An interspecific (<em>Capsicum annuum</em>× <em>C. chinese</em>) F2 linkage map in pepper using RFLP and AFLP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.26" data-doi="10.1038/ng.2877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeom</span>  <span class="cit-name-given-names">S.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in <em>Capsicum</em> species.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2877%26rft_id%253Dinfo%253Apmid%252F24441736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24441736&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palloix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caranta</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pochard</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of an intraspecific integrated linkage map of pepper using molecular markers and doubled-haploid progenies.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLefebvre%26rft.auinit1%253DV.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D112%26rft.epage%253D121%26rft.atitle%253DConstruction%2Bof%2Ban%2Bintraspecific%2Bintegrated%2Blinkage%2Bmap%2Bof%2Bpepper%2Busing%2Bmolecular%2Bmarkers%2Band%2Bdoubled-haploid%2Bprogenies.%26rft_id%253Dinfo%253Apmid%252F18470157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thabuis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caranta</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blattes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Towards the saturation of the pepper linkage map by alignment of three intraspecific maps including known-function genes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLefebvre%26rft.auinit1%253DV.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D839%26rft.epage%253D854%26rft.atitle%253DTowards%2Bthe%2Bsaturation%2Bof%2Bthe%2Bpepper%2Blinkage%2Bmap%2Bby%2Balignment%2Bof%2Bthree%2Bintraspecific%2Bmaps%2Bincluding%2Bknown-function%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F12416616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12416616&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.29" data-doi="10.1016/S0168-9525(02)02669-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Human SNP variability and mutation rate are higher in regions of high recombination.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLercher%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D337%26rft.epage%253D340%26rft.atitle%253DHuman%2BSNP%2Bvariability%2Band%2Bmutation%2Brate%2Bare%2Bhigher%2Bin%2Bregions%2Bof%2Bhigh%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902669-0%26rft_id%253Dinfo%253Apmid%252F12127766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02669-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12127766&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176560800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.30" data-doi="10.1093/jhered/91.6.423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livingstone</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Linkage mapping in populations with karyotypic rearrangements.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DLivingstone%26rft.auinit1%253DK.%2BD.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D423%26rft.epage%253D428%26rft.atitle%253DLinkage%2Bmapping%2Bin%2Bpopulations%2Bwith%2Bkaryotypic%2Brearrangements%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F91.6.423%26rft_id%253Dinfo%253Apmid%252F11218078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkxLzYvNDIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livingstone</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lackney</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blauth</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Wijk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genome mapping in <em>Capsicum</em> and the evolution of genome structure in the Solanaceae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1183</span>–<span class="cit-lpage">1202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLivingstone%26rft.auinit1%253DK.%2BD.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1183%26rft.epage%253D1202%26rft.atitle%253DGenome%2BMapping%2Bin%2BCapsicum%2Band%2Bthe%2BEvolution%2Bof%2BGenome%2BStructure%2Bin%2Bthe%2BSolanaceae%26rft_id%253Dinfo%253Apmid%252F10388833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzExODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.32" data-doi="10.1016/j.scienta.2011.03.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">M.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">J.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome analysis and SNP/SSR marker information of red pepper variety YCM334 and Taean.</span> <abbr class="cit-jnl-abbrev">Sci. Hortic. (Amsterdam)</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BHortic.%2B%2528Amsterdam%2529%26rft.volume%253D129%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.scienta.2011.03.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.scienta.2011.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.33" data-doi="10.1007/s11033-011-1102-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome profiling and molecular marker discovery in red pepper, <em>Capsicum annuum L</em>. TF68.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">3327</span>–<span class="cit-lpage">3335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DLu%26rft.auinit1%253DF.%2BH.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D3327%26rft.epage%253D3335%26rft.atitle%253DTranscriptome%2Bprofiling%2Band%2Bmolecular%2Bmarker%2Bdiscovery%2Bin%2Bred%2Bpepper%252C%2BCapsicum%2Bannuum%2BL.%2BTF68.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-011-1102-x%26rft_id%253Dinfo%253Apmid%252F21706160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-011-1102-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21706160&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.34" data-doi="10.1007/s11032-013-9875-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massire</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachellez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A key QTL cluster is conserved among accessions and exhibits broad-spectrum resistance to <em>Phytophthora capsici</em>: a valuable locus for pepper breeding.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D32%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-013-9875-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-013-9875-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.35" data-doi="10.1270/jsbbs.62.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minamiyama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An SSR-based genetic map of pepper (<em>Capsicum annuum L</em>.) serves as an anchor for the alignment of major pepper maps.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D62%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.62.93%26rft_id%253Dinfo%253Apmid%252F23136519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.62.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23136519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.36" data-doi="10.1104/pp.105.060707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solow</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skwarecki</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buels</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The SOL Genomics Network. A comparative resource for Solanaceae biology and beyond.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1310</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DMueller%26rft.auinit1%253DL.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D1310%26rft.epage%253D1317%26rft.atitle%253DThe%2BSOL%2BGenomics%2BNetwork.%2BA%2BComparative%2BResource%2Bfor%2BSolanaceae%2BBiology%2Band%2BBeyond%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.060707%26rft_id%253Dinfo%253Apmid%252F16010005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM4LzMvMTMxMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.37" data-doi="10.1016/S0959-437X(02)00358-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Variation in recombination rate across the genome: evidence and implications.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D663%26rft.atitle%253DVariation%2Bin%2Brecombination%2Brate%2Bacross%2Bthe%2Bgenome%253A%2Bevidence%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900358-1%26rft_id%253Dinfo%253Apmid%252F12433578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00358-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433578&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naegele</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin-Wo</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QTL mapping of fruit rot resistance to the plant pathogen <em>Phytophthora capsici</em> in a recombinant inbred line <em>Capsicum annuum</em> population.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D104%26rft.spage%253D479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogundiwin</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berke</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massoudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huestis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Construction of 2 intraspecific linkage maps and identification of resistance QTLs for <em>Phytophthora capsici</em> root-rot and foliar-blight diseases of pepper (<em>Capsicum annuum L</em>.).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DOgundiwin%26rft.auinit1%253DE.%2BA.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D698%26rft.epage%253D711%26rft.atitle%253DConstruction%2Bof%2B2%2Bintraspecific%2Blinkage%2Bmaps%2Band%2Bidentification%2Bof%2Bresistance%2BQTLs%2Bfor%2BPhytophthora%2Bcapsici%2Broot-rot%2Band%2Bfoliar-blight%2Bdiseases%2Bof%2Bpepper%2B%2528Capsicum%2Bannuum%2BL.%2529.%26rft_id%253Dinfo%253Apmid%252F16094437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16094437&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.40" data-doi="10.1023/B:MOLB.0000022526.30914.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Voort</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An integrated genetic linkage map of pepper (<em>Capsicum spp</em>.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D13%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AMOLB.0000022526.30914.31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:MOLB.0000022526.30914.31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.41" data-doi="10.1007/s11032-014-0089-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">J.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">J.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An EST-based linkage map reveals chromosomal translocation in <em>Capsicum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D963%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0089-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0089-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2341.42" data-doi="10.1038/nature10158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Potato Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequence and analysis of the tuber crop potato</span>. <span class="cit-source">Nature</span> <span class="cit-vol">475</span>: <span class="cit-fpage">189</span>−<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DX.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D189%26rft.epage%253D195%26rft.atitle%253DGenome%2Bsequence%2Band%2Banalysis%2Bof%2Bthe%2Btuber%2Bcrop%2Bpotato.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10158%26rft_id%253Dinfo%253Apmid%252F21743474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21743474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pochard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of a molecular linkage map of pepper and a comparison of synteny with tomato.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DPrince%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D404%26rft.epage%253D417%26rft.atitle%253DConstruction%2Bof%2Ba%2Bmolecular%2Blinkage%2Bmap%2Bof%2Bpepper%2Band%2Ba%2Bcomparison%2Bof%2Bsynteny%2Bwith%2Btomato.%26rft_id%253Dinfo%253Apmid%252F18469998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18469998&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.44" data-doi="10.1073/pnas.1400975111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome sequencing of cultivated and wild peppers provides insights into <em>Capsicum</em> domestication and specialization.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">5135</span>–<span class="cit-lpage">5140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1400975111%26rft_id%253Dinfo%253Apmid%252F24591624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE0LzUxMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rehrig</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CaDMR1 Cosegregates with QTL Pc5.1 for Resistance to Phytophthora capsici in Pepper (Capsicum annuum).</span> <span class="cit-source">Plant Gen.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D7%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.46" data-doi="10.1534/genetics.105.043877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roselius</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Städler</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The relationship of nucleotide polymorphism, recombination rate and selection in wild tomato species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.043877%26rft_id%253Dinfo%253Apmid%252F16085701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNzUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.47" data-doi="10.1371/journal.pone.0040563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40563%26rft.epage%253De40563%26rft.atitle%253DDevelopment%2Bof%2Ba%2Blarge%2BSNP%2Bgenotyping%2Barray%2Band%2Bgeneration%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040563%26rft_id%253Dinfo%253Apmid%252F22802968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22802968&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.48" data-doi="10.1270/jsbbs.55.287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sugita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinoshita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuji</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid construction of a linkage map using high-efficiency genome scanning/AFLP and RAPD, based on an intraspecific, doubled-haploid population of <em>Capsicum annuum</em>.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D55%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.55.287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.55.287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.49" data-doi="10.1094/PHYTO-98-8-0867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosland</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Recombinant inbred line differential identifies race-specific resistance to Phytophthora root rot in <em>Capsicum annuum</em>.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DSy%26rft.auinit1%253DO.%26rft.volume%253D98%26rft.issue%253D8%26rft.spage%253D867%26rft.epage%253D870%26rft.atitle%253DRecombinant%2Binbred%2Bline%2Bdifferential%2Bidentifies%2Brace-specific%2Bresistance%2Bto%2Bphytophthora%2Broot%2Brot%2Bin%2BCapsicum%2Bannuum.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-98-8-0867%26rft_id%253Dinfo%253Apmid%252F18943204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-98-8-0867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18943204&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.50" data-doi="10.1073/pnas.85.17.6419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatzky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapitan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Conservation of gene repertoire but not gene order in pepper and tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">6419</span>–<span class="cit-lpage">6423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D85%26rft.issue%253D17%26rft.spage%253D6419%26rft.epage%253D6423%26rft.atitle%253DConservation%2Bof%2Bgene%2Brepertoire%2Bbut%2Bnot%2Bgene%2Border%2Bin%2Bpepper%2Band%2Btomato%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.17.6419%26rft_id%253Dinfo%253Apmid%252F16593975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTcvNjQxOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.51" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.52" data-doi="10.1534/g3.112.004929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An ultra-high-density, transcript-based, genetic map of lettuce.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004929%26rft_id%253Dinfo%253Apmid%252F23550116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNC82MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">H. T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative mapping of consensus SSR markers in an intraspecific F-8 recombinant inbred line population in <em>Capsicum</em>.</span> <abbr class="cit-jnl-abbrev">Hortic. Environ. Biotechnol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BEnviron.%2BBiotechnol.%26rft.volume%253D51%26rft.spage%253D193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2341.54"><div class="cit-metadata unstructured">Van Ooijen, J., 2006 JoinMap v. 4. Software for the calculation of genetic linkage maps in experimental populations. Available at: <a href="https://dendrome.ucdavis.edu/resources/tooldocs/joinmap/JM4manual.pdf">https://dendrome.ucdavis.edu/resources/tooldocs/joinmap/JM4manual.pdf</a>. Accessed September 21, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.55" data-doi="10.1007/s00122-005-0097-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eck</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RECORD: a novel method for ordering loci on a genetic linkage map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BOs%26rft.auinit1%253DH.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DRECORD%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Bordering%2Bloci%2Bon%2Ba%2Bgenetic%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0097-x%26rft_id%253Dinfo%253Apmid%252F16228189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0097-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228189&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.56" data-doi="10.1101/gr.5011206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-density haplotyping with microarray-based expression and single feature polymorphism markers in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5011206%26rft_id%253Dinfo%253Apmid%252F16702412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.57" data-doi="10.1186/1471-2164-11-182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Chromosomal evolution in the plant family Solanaceae.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWu%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.spage%253D182%26rft.epage%253D182%26rft.atitle%253DChromosomal%2Bevolution%2Bin%2Bthe%2Bplant%2Bfamily%2BSolanaceae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-182%26rft_id%253Dinfo%253Apmid%252F20236516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20236516&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.58" data-doi="10.1007/s00122-009-0980-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eannetta</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazourek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A COSII genetic map of the pepper genome provides a detailed picture of synteny with tomato and new insights into recent chromosome evolution in the genus <em>Capsicum</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DF.%26rft.volume%253D118%26rft.issue%253D7%26rft.spage%253D1279%26rft.epage%253D1293%26rft.atitle%253DA%2BCOSII%2Bgenetic%2Bmap%2Bof%2Bthe%2Bpepper%2Bgenome%2Bprovides%2Ba%2Bdetailed%2Bpicture%2Bof%2Bsynteny%2Bwith%2Btomato%2Band%2Bnew%2Binsights%2Binto%2Brecent%2Bchromosome%2Bevolution%2Bin%2Bthe%2Bgenus%2BCapsicum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-0980-y%26rft_id%253Dinfo%253Apmid%252F19229514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-0980-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19229514&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.59" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yarnes</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of QTLs for capsaicinoids, fruit quality, and plant architecture-related traits in an interspecific <em>Capsicum</em> RIL population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D56%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2341.61" data-doi="10.1007/s00122-011-1701-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Construction of a high density integrated genetic map for cucumber (<em>Cucumis sativus L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DW.%2BW.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D259%26rft.atitle%253DConstruction%2Bof%2Ba%2Bhigh%2Bdensity%2Bintegrated%2Bgenetic%2Bmap%2Bfor%2Bcucumber%2B%2528Cucumis%2Bsativus%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1701-x%26rft_id%253Dinfo%253Apmid%252F21971891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1701-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21971891&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2341.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2341.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2329" title="The Genetic Architecture of Hearing Impairment in Mice: Evidence for Frequency-Specific Genetic Determinants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2357" title="Genetic Linkage Mapping of Economically Important Traits in Cultivated Tetraploid Potato (Solanum tuberosum L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2341" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UqGMF3cWMtr0OjEL3JyNBlwJezIM9RKLg1yDG5rA-Xg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2341" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2341" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2341" class="active" data-icon-position="" data-hide-link-title="0">Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460164" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-luv_e38IufQ1yTltk_70Y7DMGR6yo5uhDi2U35ep-HA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2341.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2341" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-GiQ31vIScezHMK6DR_zmg_Vy-4IkFit5E_mun9kFyj4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2341" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2341.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460164" id="node460164--2" data-pisa="ggg;5/11/2341" data-pisa-master="ggg;g3.115.020040" data-apath="/ggg/5/11/2341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Reyes Chin-Wo</span>, <span class="highwire-citation-author" data-delta="3">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="6">Richard Michelmore</span> and <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2341-2355;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020040 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460164--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Reyes Chin-Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin-Wo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chin-Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin-Wo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:avandeynze@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">avandeynze@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2341" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2341&amp;text=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2341&amp;t=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2341&amp;title=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460164/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460164" id="node460164--3" data-pisa="ggg;5/11/2341" data-pisa-master="ggg;g3.115.020040" data-apath="/ggg/5/11/2341.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ultra-High Density, Transcript-Based Genetic Maps of Pepper Define Recombination in the Genome and Synteny Among Related Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Reyes Chin-Wo</span>, <span class="highwire-citation-author" data-delta="3">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="6">Richard Michelmore</span> and <span class="highwire-citation-author" data-delta="7">Allen Van Deynze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2341-2355;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020040 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460164--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Reyes Chin-Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin-Wo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chin-Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin-Wo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:avandeynze@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">avandeynze@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2341&amp;rft.epage=2355&amp;rft.atitle=Ultra-High%20Density%2C%20Transcript-Based%20Genetic%20Maps%20of%20Pepper%20Define%20Recombination%20in%20the%20Genome%20and%20Synteny%20Among%20Related%20Species&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hill%2C+Theresa&amp;rft.au=Ashrafi%2C+Hamid&amp;rft.au=Chin-Wo%2C+Sebastian+Reyes&amp;rft.au=Stoffel%2C+Kevin&amp;rft.au=Truco%2C+Maria-Jose&amp;rft.au=Kozik%2C+Alexander&amp;rft.au=Michelmore%2C+Richard&amp;rft.au=Van+Deynze%2C+Allen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460164/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460164/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460164/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460164/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460164/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460164/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460164/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460164/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460164/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460164/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460164/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460164/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2341" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2341.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2341.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2341.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2341.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
