<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2405" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2405.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2405.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2405.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/11/2405/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2405.atom" />
<meta name="HW.pisa" content="ggg;5/11/2405" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila" />
<meta name="DC.Identifier" content="10.1534/g3.115.021659" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Rice et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Polyglutamine (pQ) tracts are abundant in proteins co-interacting on DNA. The lengths of these pQ tracts can modulate their interaction strengths. However, pQ tracts &gt;40 residues are pathologically prone to amyloidogenic self-assembly. Here, we assess the extent and consequences of variation in the pQ-encoding [opa][1] repeats of [Notch][2] in Drosophila melanogaster . We use Sanger sequencing to genotype [opa][1] sequences (![Formula][3]&lt;/img&gt;-CAX repeats), which have resisted assembly using short sequence reads. While most sampled lines carry the major allele opa31 encoding Q13HQ17 or the opa32 allele encoding Q13HQ18, many lines carry rare alleles encoding pQ tracts &gt;32 residues: opa33a (Q14HQ18), opa33b (Q15HQ17), opa34 (Q16HQ17), opa35a1 / opa35a2 (Q13HQ21), opa36 (Q13HQ22), and opa37 (Q13HQ23). Only one rare allele encodes a tract &lt;31 residues: opa23 (Q13–Q10). This opa23 allele shortens the pQ tract while simultaneously eliminating the interrupting histidine. We introgressed these [opa][1] variant alleles into common backgrounds and measured the frequency of [Notch][2] -type phenotypes. Homozygotes for the short and long [opa][1] alleles have defects in embryonic survival and sensory bristle organ patterning, and sometimes show wing notching. Consistent with functional differences between Notch opa variants, we find that a [scute][4] inversion carrying the rare opa33b allele suppresses the bristle patterning defect caused by [achaete][5] / [scute][4] insufficiency, while an equivalent [scute][4] inversion carrying opa31 manifests the patterning defect. Our results demonstrate the existence of potent pQ variants of Notch and the need for long read genotyping of key repeat variables underlying gene regulatory networks.

 [1]: http://flybase.org/reports/FBgn0003002.html
 [2]: http://flybase.org/reports/FBgn0004647.html
 [3]: /embed/mml-math-1.gif
 [4]: http://flybase.org/reports/FBgn0004170.html
 [5]: http://flybase.org/reports/FBgn0000022.html" />
<meta name="DC.Contributor" content="Clinton Rice" />
<meta name="DC.Contributor" content="Danielle Beekman" />
<meta name="DC.Contributor" content="Liping Liu" />
<meta name="DC.Contributor" content="Albert Erives" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Polyglutamine (pQ) tracts are abundant in proteins co-interacting on DNA. The lengths of these pQ tracts can modulate their interaction strengths. However, pQ tracts &amp;gt;40 residues are pathologically prone to amyloidogenic self-assembly. Here, we assess the extent and consequences of variation in the pQ-encoding &lt;i&gt;opa&lt;/i&gt; repeats of &lt;i&gt;Notch&lt;/i&gt; in &lt;i&gt;Drosophila melanogaster&lt;/i&gt;. We use Sanger sequencing to genotype &lt;i&gt;opa&lt;/i&gt; sequences (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;msup&gt;&lt;mn&gt;5&lt;/mn&gt;&lt;mo&gt;′&lt;/mo&gt;&lt;/msup&gt;&lt;/math&gt;-CAX repeats), which have resisted assembly using short sequence reads. While most sampled lines carry the major allele &lt;i&gt;opa31&lt;/i&gt; encoding Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;17&lt;/sub&gt; or the &lt;i&gt;opa32&lt;/i&gt; allele encoding Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;18&lt;/sub&gt;, many lines carry rare alleles encoding pQ tracts &amp;gt;32 residues: &lt;i&gt;opa33a&lt;/i&gt; (Q&lt;sub&gt;14&lt;/sub&gt;HQ&lt;sub&gt;18&lt;/sub&gt;), &lt;i&gt;opa33b&lt;/i&gt; (Q&lt;sub&gt;15&lt;/sub&gt;HQ&lt;sub&gt;17&lt;/sub&gt;), &lt;i&gt;opa34&lt;/i&gt; (Q&lt;sub&gt;16&lt;/sub&gt;HQ&lt;sub&gt;17&lt;/sub&gt;), &lt;i&gt;opa35a1&lt;/i&gt;/&lt;i&gt;opa35a2&lt;/i&gt; (Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;21&lt;/sub&gt;), &lt;i&gt;opa36&lt;/i&gt; (Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;22&lt;/sub&gt;), and &lt;i&gt;opa37&lt;/i&gt; (Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;23&lt;/sub&gt;). Only one rare allele encodes a tract &amp;lt;31 residues: &lt;i&gt;opa23&lt;/i&gt; (Q&lt;sub&gt;13&lt;/sub&gt;–Q&lt;sub&gt;10&lt;/sub&gt;). This &lt;i&gt;opa23&lt;/i&gt; allele shortens the pQ tract while simultaneously eliminating the interrupting histidine. We introgressed these &lt;i&gt;opa&lt;/i&gt; variant alleles into common backgrounds and measured the frequency of &lt;i&gt;Notch&lt;/i&gt;-type phenotypes. Homozygotes for the short and long &lt;i&gt;opa&lt;/i&gt; alleles have defects in embryonic survival and sensory bristle organ patterning, and sometimes show wing notching. Consistent with functional differences between &lt;i&gt;Notch opa&lt;/i&gt; variants, we find that a &lt;i&gt;scute&lt;/i&gt; inversion carrying the rare &lt;i&gt;opa33b&lt;/i&gt; allele suppresses the bristle patterning defect caused by &lt;i&gt;achaete&lt;/i&gt;/&lt;i&gt;scute&lt;/i&gt; insufficiency, while an equivalent &lt;i&gt;scute&lt;/i&gt; inversion carrying &lt;i&gt;opa31&lt;/i&gt; manifests the patterning defect. Our results demonstrate the existence of potent pQ variants of Notch and the need for long read genotyping of key repeat variables underlying gene regulatory networks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2405" />
<meta name="citation_id" content="5/11/2405" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2405" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2405.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2405.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2405.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021659" />
<meta name="citation_pmid" content="26362765" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2405" />
<meta name="citation_lastpage" content="2419" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Clinton Rice" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324" />
<meta name="citation_author" content="Danielle Beekman" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324" />
<meta name="citation_author" content="Liping Liu" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324" />
<meta name="citation_author" content="Albert Erives" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324" />
<meta name="citation_author_email" content="albert-erives@uiowa.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7107-5518" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. Ananda;citation_author=E. Walsh;citation_author=K. D. Jacob;citation_author=M. Krasilnikova;citation_author=K. A. Eckert;citation_title=Distinct mutational behaviors differentiate short tandem repeats from microsatellites in the human genome.;citation_volume=5;citation_year=2013;citation_pmid=23241442;citation_doi=10.1093/gbe/evs116" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. E. Andrew;citation_author=Y. P. Goldberg;citation_author=B. Kremer;citation_author=H. Telenius;citation_author=J. Theilmann;citation_title=The relationship between trinucleotide (CAG) repeat length and clinical features of Huntington's disease.;citation_pages=398-403;citation_volume=4;citation_year=1993;citation_issue=4;citation_pmid=8401589;citation_doi=10.1038/ng0893-398" />
<meta name="citation_reference" content="citation_author=M. Ashburner;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. M. Bailey;citation_author=J. W. Posakony;citation_title=Suppressor of hairless directly activates transcription of enhancer of split complex genes in response to Notch receptor activity.;citation_pages=2609-2622;citation_volume=9;citation_year=1995;citation_issue=21;citation_pmid=7590239;citation_doi=10.1101/gad.9.21.2609" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=O. Barad;citation_author=E. Hornstein;citation_author=N. Barkai;citation_title=Robust selection of sensory organ precursors by the Notch-Delta pathway.;citation_pages=663-667;citation_volume=23;citation_year=2011;citation_issue=6;citation_pmid=21963301;citation_doi=10.1016/j.ceb.2011.09.005" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=O. Barad;citation_author=D. Rosin;citation_author=E. Hornstein;citation_author=N. Barkai;citation_title=Error Minimization in Lateral Inhibition Circuits;citation_pages=ra51-ra51;citation_volume=3;citation_year=2010;citation_issue=129;citation_pmid=20606215;citation_doi=10.1126/scisignal.2000857" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=V. Biancalana;citation_author=F. Serville;citation_author=J. Pommier;citation_author=J. Julien;citation_author=A. Hanauer;citation_title=Moderate instability of the trinucleotide repeat in spino bulbar muscular atrophy;citation_pages=255-258;citation_volume=1;citation_year=1992;citation_issue=4;citation_pmid=1303195;citation_doi=10.1093/hmg/1.4.255" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. A. Bricmont;citation_author=J. R. Daugherty;citation_author=T. G. Cooper;citation_title=The DAL81 gene product is required for induced expression of two differently regulated nitrogen catabolic genes in Saccharomyces cerevisiae.;citation_pages=1161-1166;citation_volume=11;citation_year=1991;citation_issue=2;citation_pmid=1990272;citation_doi=10.1128/MCB.11.2.1161" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Brittain;citation_author=E. Stroebele;citation_author=A. Erives;citation_title=Microsatellite repeat instability fuels evolution of embryonic enhancers in Hawaiian Drosophila.;citation_pages=e101177;citation_volume=9;citation_year=2014;citation_pmid=24978198;citation_doi=10.1371/journal.pone.0101177" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. J. Brizuela;citation_author=L. Elfring;citation_author=J. Ballard;citation_author=J. W. Tamkun;citation_author=J. A. Kennison;citation_title=Genetic analysis of the brahma gene of Drosophila melanogaster and polytene chromosome subdivisions 72AB.;citation_pages=803-813;citation_volume=137;citation_year=1994;citation_issue=3;citation_pmid=7916308" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=D. D. Carson;citation_author=M. D. Summers;citation_author=L. A. Guarino;citation_title=Molecular analysis of a baculovirus regulatory gene.;citation_pages=279-286;citation_volume=182;citation_year=1991;citation_issue=1;citation_pmid=2024466;citation_doi=10.1016/0042-6822(91)90671-W" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=S. Chen;citation_author=V. Berthelier;citation_author=J. B. Hamilton;citation_title=Amyloid-like features of polyglutamine aggregates and their assembly kinetics.;citation_pages=7391-7399;citation_volume=41;citation_year=2002;citation_pmid=12044172;citation_doi=10.1021/bi011772q" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. M. Comeron;citation_author=R. Ratnappan;citation_author=S. Bailin;citation_title=The many landscapes of recombination in Drosophila melanogaster.;citation_pages=1002905;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.pgen.1002905" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_title=Population genomics of inversion polymorphisms in Drosophila melanogaster.;citation_pages=1003056;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.pgen.1003056" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. J. Courey;citation_author=R. Tjian;citation_title=Analysis of Sp1 in vivo reveals multiple transcriptional domains, including a novel glutamine-rich activation motif.;citation_pages=887-898;citation_volume=55;citation_year=1988;citation_issue=5;citation_pmid=3142690;citation_doi=10.1016/0092-8674(88)90144-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. J. Courey;citation_author=D. A. Holtzman;citation_author=S. P. Jackson;citation_author=R. Tjian;citation_title=Synergistic activation by the glutamine-rich domains of human transcription factor Sp1.;citation_pages=827-836;citation_volume=59;citation_year=1989;citation_issue=5;citation_pmid=2512012;citation_doi=10.1016/0092-8674(89)90606-5" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=J. Crocker;citation_author=A. Erives;citation_title=A Schnurri/Mad/Medea complex attenuates the dorsal-twist gradient readout at vnd.;citation_pages=64-72;citation_volume=378;citation_year=2013;citation_pmid=23499655;citation_doi=10.1016/j.ydbio.2013.03.002" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. Crocker;citation_author=Y. Tamori;citation_author=A. Erives;citation_title=Evolution acts on enhancer organization to fine-tune gradient threshold readouts.;citation_pages=2576-2587;citation_volume=6;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=J. Crocker;citation_author=N. Potter;citation_author=A. Erives;citation_title=Dynamic evolution of precise regulatory encodings creates the clustered site signature of enhancers.;citation_pages=99-99;citation_volume=1;citation_year=2010;citation_pmid=20981027;citation_doi=10.1038/ncomms1102" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. R. David;citation_author=P. Gibert;citation_author=H. Legout;citation_author=G. Pétavy;citation_author=P. Capy;citation_title=Isofemale lines in Drosophila: an empirical approach to quantitative trait analysis in natural populations.;citation_pages=3-12;citation_volume=94;citation_year=2005;citation_issue=1;citation_pmid=15329665;citation_doi=10.1038/sj.hdy.6800562" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Dosztányi;citation_author=V. Csizmok;citation_author=P. Tompa;citation_author=I. Simon;citation_title=IUPred: Web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content.;citation_volume=21;citation_year=2005;citation_pmid=15955779;citation_doi=10.1093/bioinformatics/bti541" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=Z. Dosztányi;citation_author=V. Csizmók;citation_author=P. Tompa;citation_author=I. Simon;citation_title=The pairwise energy content estimated from amino acid composition discriminates between folded and intrinsically unstructured proteins.;citation_pages=827-839;citation_volume=347;citation_year=2005;citation_issue=4;citation_pmid=15769473;citation_doi=10.1016/j.jmb.2005.01.071" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=H. Ellegren;citation_title=Microsatellite mutations in the germline: implications for evolutionary inference.;citation_pages=551-558;citation_volume=16;citation_year=2000;citation_issue=12;citation_pmid=11102705;citation_doi=10.1016/S0168-9525(00)02139-9" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. G. Fehon;citation_author=P. J. Kooh;citation_author=I. Rebay;citation_author=C. L. Regan;citation_author=T. Xu;citation_title=Molecular interactions between the protein products of the neurogenic loci Notch and Delta, two EGF-homologous genes in Drosophila.;citation_pages=523-534;citation_volume=61;citation_year=1990;citation_issue=3;citation_pmid=2185893;citation_doi=10.1016/0092-8674(90)90534-L" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=M. E. Fortini;citation_title=Notch signaling: the core pathway and its posttranslational regulation.;citation_pages=633-647;citation_volume=16;citation_year=2009;citation_issue=5;citation_pmid=19460341;citation_doi=10.1016/j.devcel.2009.03.010" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. E. Fortini;citation_author=S. Artavanis-Tsakonas;citation_title=The suppressor of hairless protein participates in notch receptor signaling.;citation_pages=273-282;citation_volume=79;citation_year=1994;citation_issue=2;citation_pmid=7954795;citation_doi=10.1016/0092-8674(94)90196-1" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. J. Fryer;citation_author=E. Lamar;citation_author=I. Turbachova;citation_author=C. Kintner;citation_author=K. A. Jones;citation_title=Mastermind mediates chromatin-specific transcription and turnover of the Notch enhancer complex;citation_pages=1397-1411;citation_volume=16;citation_year=2002;citation_issue=11;citation_pmid=12050117;citation_doi=10.1101/gad.991602" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. J. Fryer;citation_author=J. B. White;citation_author=K. A. Jones;citation_title=Mastermind recruits CycC:CDK8 to phosphorylate the Notch ICD and coordinate activation with turnover.;citation_pages=509-520;citation_volume=16;citation_year=2004;citation_issue=4;citation_pmid=15546612;citation_doi=10.1016/j.molcel.2004.10.014" />
<meta name="citation_reference" content="citation_author=T. Furukawa;citation_author=Y. Kobayakawa;citation_author=K. Tamura;citation_author=K. Kimura;citation_author=M. Kawaichi;citation_author=T. Tanimura;citation_author=T. Honjo;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=R. Galant;citation_author=J. B. Skeath;citation_author=S. Paddock;citation_author=D. L. Lewis;citation_author=S. B. Carroll;citation_title=Expression pattern of a butterfly achaete-scute homolog reveals the homology of butterfly wing scales and insect sensory bristles.;citation_pages=807-813;citation_volume=8;citation_year=1998;citation_issue=14;citation_pmid=9663389;citation_doi=10.1016/S0960-9822(98)70322-7" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. Garrigan;citation_author=S. B. Kingan;citation_author=A. J. Geneva;citation_author=P. Andolfatto;citation_author=A. G. Clark;citation_title=Genome sequencing reveals complex speciation in the Drosophila simulans clade.;citation_volume=22;citation_year=2012;citation_pmid=22534282;citation_doi=10.1101/gr.130922.111" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. L. Gómez-Skarmeta;citation_author=I. Rodríguez;citation_author=C. Martínez;citation_author=J. Culí;citation_author=D. Ferrés-Marcó;citation_title=Cis-regulation of achaete and scute: shared enhancer-like elements drive their coexpression in proneural clusters of the imaginal discs.;citation_pages=1869-1882;citation_volume=9;citation_year=1995;citation_issue=15;citation_pmid=7649474;citation_doi=10.1101/gad.9.15.1869" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ràmia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster Genetic Reference Panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. T. Kadonaga;citation_author=A. J. Courey;citation_author=J. Ladika;citation_author=R. Tjian;citation_title=Distinct regions of Sp1 modulate DNA binding and transcriptional activation;citation_pages=1566-1570;citation_volume=242;citation_year=1988;citation_issue=4885;citation_pmid=3059495;citation_doi=10.1126/science.3059495" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. A. Kassis;citation_author=S. J. Poole;citation_author=D. K. Wright;citation_author=P. H. O’Farrell;citation_title=Sequence conservation in the protein coding and intron regions of the engrailed transcription unit.;citation_pages=3583-3589;citation_volume=5;citation_year=1986;citation_pmid=2881781" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. F. Kelly;citation_author=R. J. Lake;citation_author=T. Walz;citation_author=S. Artavanis-Tsakonas;citation_title=Conformational variability of the intracellular domain of Drosophila Notch and its interaction with Suppressor of Hairless.;citation_volume=104;citation_year=2007;citation_pmid=17535912;citation_doi=10.1073/pnas.0702887104" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Kidd;citation_author=M. R. Kelley;citation_author=M. W. Young;citation_title=Sequence of the notch locus of Drosophila melanogaster: relationship of the encoded protein to mammalian clotting and growth factors.;citation_pages=3094-3108;citation_volume=6;citation_year=1986;citation_issue=9;citation_pmid=3097517;citation_doi=10.1128/MCB.6.9.3094" />
<meta name="citation_reference" content="citation_journal_title=Prion;citation_author=M. Kim;citation_title=Beta conformation of polyglutamine track revealed by a crystal structure of Huntingtin N-terminal region with insertion of three histidine residues.;citation_pages=221-228;citation_volume=7;citation_year=2013;citation_pmid=23370273;citation_doi=10.4161/pri.23807" />
<meta name="citation_reference" content="citation_author=R. A. Kovall;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. R. La Spada;citation_author=D. B. Roling;citation_author=A. E. Harding;citation_author=C. L. Warner;citation_author=R. Spiegel;citation_title=Meiotic stability and genotype-phenotype correlation of the trinucleotide repeat in X-linked spinal and bulbar muscular atrophy.;citation_pages=301-304;citation_volume=2;citation_year=1992;citation_issue=4;citation_pmid=1303283;citation_doi=10.1038/ng1292-301" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. Lecourtois;citation_author=F. Schweisguth;citation_title=The neurogenic suppressor of hairless DNA-binding protein mediates the transcriptional activation of the enhancer of split complex genes triggered by Notch signaling.;citation_pages=2598-2608;citation_volume=9;citation_year=1995;citation_issue=21;citation_pmid=7590238;citation_doi=10.1101/gad.9.21.2598" />
<meta name="citation_reference" content="citation_author=D. Lindsley;citation_author=E. Grell;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Lyman;citation_author=M. W. Young;citation_title=Further evidence for function of the Drosophila Notch protein as a transmembrane receptor.;citation_pages=10395-10399;citation_volume=90;citation_year=1993;citation_issue=21;citation_pmid=8234305;citation_doi=10.1073/pnas.90.21.10395" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. Martínez;citation_author=J. Modolell;citation_title=Cross-regulatory interactions between the proneural achaete and scute genes of Drosophila;citation_pages=1485-1487;citation_volume=251;citation_year=1991;citation_issue=5000;citation_pmid=1900954;citation_doi=10.1126/science.1900954" />
<meta name="citation_reference" content="citation_journal_title=The International journal of developmental biology;citation_journal_abbrev=Int J Dev Biol;citation_author=J. Modolell;citation_author=S. Campuzano;citation_title=The achaete-scute complex as an integrating device.;citation_pages=275-282;citation_volume=42;citation_year=1998;citation_issue=3;citation_pmid=9654009" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_title=The unfolded protein response as a target for cancer therapy.;citation_author=A. Nagelkerke;citation_author=J. Bussink;citation_author=F. C. G. J. Sweep;citation_author=P. N. Span;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Negre;citation_author=P. Simpson;citation_title=Evolution of the achaete-scute complex in insects: convergent duplication of proneural genes.;citation_pages=147-152;citation_volume=25;citation_year=2009;citation_issue=4;citation_pmid=19285745;citation_doi=10.1016/j.tig.2009.02.001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. J. Newfeld;citation_author=D. A. Smoller;citation_author=B. Yedvobnick;citation_title=Interspecific comparison of the unusually repetitive Drosophila locus mastermind.;citation_pages=415-420;citation_volume=32;citation_year=1991;citation_issue=5;citation_pmid=1904096;citation_doi=10.1007/BF02101281" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. J. Newfeld;citation_author=A. T. Schmid;citation_author=B. Yedvobnick;citation_title=Homopolymer length variation in the Drosophila gene mastermind.;citation_pages=483-495;citation_volume=37;citation_year=1993;citation_issue=5;citation_pmid=8283480;citation_doi=10.1007/BF00160429" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. J. Newfeld;citation_author=H. Tachida;citation_author=B. Yedvobnick;citation_title=Drive-selection equilibrium: homopolymer evolution in the Drosophila gene mastermind.;citation_pages=637-641;citation_volume=38;citation_year=1994;citation_issue=6;citation_pmid=8083889" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. T. Orr;citation_author=M. Y. Chung;citation_author=S. Banfi;citation_author=T. J. Kwiatkowski;citation_author=A. Servadio;citation_title=Expansion of an unstable trinucleotide CAG repeat in spinocerebellar ataxia type 1.;citation_pages=221-226;citation_volume=4;citation_year=1993;citation_issue=3;citation_pmid=8358429;citation_doi=10.1038/ng0793-221" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. F. Perutz;citation_author=R. Staden;citation_author=L. Moens;citation_author=I. De Baere;citation_title=Polar zippers.;citation_pages=249-253;citation_volume=3;citation_year=1993;citation_issue=5;citation_pmid=15335744;citation_doi=10.1016/0960-9822(93)90174-M" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. F. Perutz;citation_author=T. Johnson;citation_author=M. Suzuki;citation_author=J. T. Finch;citation_title=Glutamine repeats as polar zippers: their possible role in inherited neurodegenerative diseases.;citation_pages=5355-5358;citation_volume=91;citation_year=1994;citation_issue=12;citation_pmid=8202492;citation_doi=10.1073/pnas.91.12.5355" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. F. Perutz;citation_author=J. T. Finch;citation_author=J. Berriman;citation_author=A. Lesk;citation_title=Amyloid fibers are water-filled nanotubes;citation_pages=5591-5595;citation_volume=99;citation_year=2002;citation_issue=8;citation_pmid=11960014;citation_doi=10.1073/pnas.042681399" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. L. Pinkham;citation_author=J. T. Olesen;citation_author=L. P. Guarente;citation_title=Sequence and nuclear localization of the Saccharomyces cerevisiae HAP2 protein, a transcriptional activator.;citation_pages=578-585;citation_volume=7;citation_year=1987;citation_issue=2;citation_pmid=3547076;citation_doi=10.1128/MCB.7.2.578" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=P. Ramain;citation_author=K. Khechumian;citation_author=L. Seugnet;citation_author=N. Arbogast;citation_author=C. Ackermann;citation_title=Novel Notch alleles reveal a Deltex-dependent pathway repressing neural fate.;citation_pages=1729-1738;citation_volume=11;citation_year=2001;citation_issue=22;citation_pmid=11719214;citation_doi=10.1016/S0960-9822(01)00562-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Ràmia;citation_author=P. Librado;citation_author=S. Casillas;citation_author=J. Rozas;citation_author=A. Barbadilla;citation_title=PopDrowser: The population Drosophila browser.;citation_volume=28;citation_year=2012;citation_pmid=22180410;citation_doi=10.1093/bioinformatics/btr691" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Scherzinger;citation_author=R. Lurz;citation_author=M. Turmaine;citation_author=L. Mangiarini;citation_author=B. Hollenbach;citation_title=Huntingtin-encoded polyglutamine expansions form amyloid-like protein aggregates in vitro and in vivo.;citation_pages=549-558;citation_volume=90;citation_year=1997;citation_issue=3;citation_pmid=9267034;citation_doi=10.1016/S0092-8674(00)80514-0" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Scherzinger;citation_author=A. Sittler;citation_author=K. Schweiger;citation_author=V. Heiser;citation_author=R. Lurz;citation_title=Self-assembly of polyglutamine-containing huntingtin fragments into amyloid-like fibrils: Implications for Huntington's disease pathology;citation_pages=4604-4609;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200309;citation_doi=10.1073/pnas.96.8.4604" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=C. Schlötterer;citation_title=Evolutionary dynamics of microsatellite DNA.;citation_pages=365-371;citation_volume=109;citation_year=2000;citation_issue=6;citation_pmid=11072791;citation_doi=10.1007/s004120000089" />
<meta name="citation_reference" content="citation_author=S. Sen;citation_author=D. Dash;citation_author=S. Pasha;citation_author=S. K. Brahmachari;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=D. Sharma;citation_author=S. Sharma;citation_author=S. Pasha;citation_author=S. K. Brahmachari;citation_title=Peptide models for inherited neurodegenerative disorders: conformation and aggregation properties of long polyglutamine peptides with and without interruptions.;citation_pages=181-185;citation_volume=456;citation_year=1999;citation_issue=1;citation_pmid=10452554;citation_doi=10.1016/S0014-5793(99)00933-3" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=O. Shaya;citation_author=D. Sprinzak;citation_title=From Notch signaling to fine-grained patterning: Modeling meets experiments.;citation_pages=732-739;citation_volume=21;citation_year=2011;citation_issue=6;citation_pmid=21862316;citation_doi=10.1016/j.gde.2011.07.007" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Shimizu;citation_author=S. A. Woodcock;citation_author=M. B. Wilkin;citation_author=B. Trubenová;citation_author=N. A. M. Monk;citation_title=Compensatory flux changes within an endocytic trafficking network maintain thermal robustness of Notch signaling.;citation_pages=1160-1174;citation_volume=157;citation_year=2014;citation_pmid=24855951;citation_doi=10.1016/j.cell.2014.03.050" />
<meta name="citation_reference" content="citation_author=B. Sidorov;citation_year=1931" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. B. Skeath;citation_author=G. Panganiban;citation_author=J. Selegue;citation_author=S. B. Carroll;citation_title=Gene regulation in two dimensions: the proneural achaete and scute genes are controlled by combinations of axis-patterning genes through a common intergenic control region.;citation_pages=2606-2619;citation_volume=6;citation_year=1992;citation_issue=12b;citation_pmid=1340472;citation_doi=10.1101/gad.6.12b.2606" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. G. Snell;citation_author=J. C. MacMillan;citation_author=J. P. Cheadle;citation_author=I. Fenton;citation_author=L. P. Lazarou;citation_title=Relationship between trinucleotide repeat expansion and phenotypic variation in Huntington's disease.;citation_pages=393-397;citation_volume=4;citation_year=1993;citation_issue=4;citation_pmid=8401588;citation_doi=10.1038/ng0893-393" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. A. Soshnev;citation_author=H. Ishimoto;citation_author=B. F. Mcallister;citation_author=X. Li;citation_author=M. D. Wehling;citation_title=A conserved long Noncoding RNA affects sleep behavior in Drosophila.;citation_volume=189;citation_year=2011;citation_pmid=21775470;citation_doi=10.1534/genetics.111.131706" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=D. Sprinzak;citation_author=A. Lakhanpal;citation_author=L. LeBon;citation_author=J. Garcia-Ojalvo;citation_author=M. B. Elowitz;citation_title=Mutual inactivation of Notch receptors and ligands facilitates developmental patterning.;citation_pages=1002069;citation_volume=7;citation_year=2011;citation_doi=10.1371/journal.pcbi.1002069" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Suzuki;citation_author=Y. Nogi;citation_author=A. Abe;citation_author=T. Fukasawa;citation_title=GAL11 protein, an auxiliary transcription activator for genes encoding galactose-metabolizing enzymes in Saccharomyces cerevisiae.;citation_pages=4991-4999;citation_volume=8;citation_year=1988;citation_issue=11;citation_pmid=3062377;citation_doi=10.1128/MCB.8.11.4991" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. Tautz;citation_title=Hypervariabflity of simple sequences as a general source for polymorphic DNA markers;citation_pages=6463-6471;citation_volume=17;citation_year=1989;citation_issue=16;citation_pmid=2780284;citation_doi=10.1093/nar/17.16.6463" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=A. Tóth-Petróczy;citation_author=C. J. Oldfield;citation_author=I. Simon;citation_author=Y. Takagi;citation_author=A. K. Dunker;citation_title=Malleable machines in transcription regulation: The Mediator complex.;citation_pages=1000243;citation_volume=4;citation_year=2008;citation_doi=10.1371/journal.pcbi.1000243" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Vaisnav;citation_author=C. Xing;citation_author=H.-C. Ku;citation_author=D. Hwang;citation_author=S. Stojadinovic;citation_title=Genome-wide association analysis of radiation resistance in Drosophila melanogaster.;citation_pages=e104858;citation_volume=9;citation_year=2014;citation_pmid=25121966;citation_doi=10.1371/journal.pone.0104858" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. A. Wharton;citation_author=B. Yedvobnick;citation_author=V. G. Finnerty;citation_author=S. Artavanis-Tsakonas;citation_title=opa: a novel family of transcribed repeats shared by the Notch locus and other developmentally regulated loci in D. melanogaster.;citation_pages=55-62;citation_volume=40;citation_year=1985;citation_issue=1;citation_pmid=2981631;citation_doi=10.1016/0092-8674(85)90308-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=M. J. White;citation_author=J. P. Hirsch;citation_author=S. A. Henry;citation_title=The OPI1 gene of Saccharomyces cerevisiae, a negative regulator of phospholipid biosynthesis, encodes a protein containing polyglutamine tracts and a leucine zipper;citation_pages=863-872;citation_volume=266;citation_year=1991;citation_issue=2;citation_pmid=1985968" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Wu;citation_author=J. C. Aster;citation_author=S. C. Blacklow;citation_author=R. Lake;citation_author=S. Artavanis-Tsakonas;citation_title=MAML1, a human homologue of Drosophila mastermind, is a transcriptional co-activator for NOTCH receptors.;citation_pages=484-489;citation_volume=26;citation_year=2000;citation_issue=4;citation_pmid=11101851;citation_doi=10.1038/82644" />
<meta name="citation_reference" content="citation_journal_title=Current topics in developmental biology;citation_journal_abbrev=Curr Top Dev Biol;citation_author=S. Yamamoto;citation_author=W. L. Charng;citation_author=H. J. Bellen;citation_title=Endocytosis and intracellular trafficking of Notch and its ligands.;citation_pages=165-200;citation_volume=92;citation_year=2010;citation_pmid=20816395;citation_doi=10.1016/S0070-2153(10)92005-X" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. F. Yuan;citation_author=Y. H. Fu;citation_author=G. A. Marzluf;citation_title=nit-4, a pathway-specific regulatory gene of Neurospora crassa, encodes a protein with a putative binuclear zinc DNA-binding domain.;citation_pages=5735-5745;citation_volume=11;citation_year=1991;citation_issue=11;citation_pmid=1840634;citation_doi=10.1128/MCB.11.11.5735" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Polyglutamine (pQ) tracts are abundant in proteins co-interacting on DNA. The lengths of these pQ tracts can modulate their interaction strengths. However, pQ tracts &gt;40 residues are pathologically prone to amyloidogenic self-assembly. Here, we assess the extent and consequences of variation in the pQ-encoding [opa][1] repeats of [Notch][2] in Drosophila melanogaster . We use Sanger sequencing to genotype [opa][1] sequences (![Formula][3]&lt;/img&gt;-CAX repeats), which have resisted assembly using short sequence reads. While most sampled lines carry the major allele opa31 encoding Q13HQ17 or the opa32 allele encoding Q13HQ18, many lines carry rare alleles encoding pQ tracts &gt;32 residues: opa33a (Q14HQ18), opa33b (Q15HQ17), opa34 (Q16HQ17), opa35a1 / opa35a2 (Q13HQ21), opa36 (Q13HQ22), and opa37 (Q13HQ23). Only one rare allele encodes a tract &lt;31 residues: opa23 (Q13–Q10). This opa23 allele shortens the pQ tract while simultaneously eliminating the interrupting histidine. We introgressed these [opa][1] variant alleles into common backgrounds and measured the frequency of [Notch][2] -type phenotypes. Homozygotes for the short and long [opa][1] alleles have defects in embryonic survival and sensory bristle organ patterning, and sometimes show wing notching. Consistent with functional differences between Notch opa variants, we find that a [scute][4] inversion carrying the rare opa33b allele suppresses the bristle patterning defect caused by [achaete][5] / [scute][4] insufficiency, while an equivalent [scute][4] inversion carrying opa31 manifests the patterning defect. Our results demonstrate the existence of potent pQ variants of Notch and the need for long read genotyping of key repeat variables underlying gene regulatory networks.

 [1]: http://flybase.org/reports/FBgn0003002.html
 [2]: http://flybase.org/reports/FBgn0004647.html
 [3]: /embed/mml-math-1.gif
 [4]: http://flybase.org/reports/FBgn0004170.html
 [5]: http://flybase.org/reports/FBgn0000022.html" />
<meta name="og-title" property="og:title" content="The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2405" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Polyglutamine (pQ) tracts are abundant in proteins co-interacting on DNA. The lengths of these pQ tracts can modulate their interaction strengths. However, pQ tracts &gt;40 residues are pathologically prone to amyloidogenic self-assembly. Here, we assess the extent and consequences of variation in the pQ-encoding [opa][1] repeats of [Notch][2] in Drosophila melanogaster . We use Sanger sequencing to genotype [opa][1] sequences (![Formula][3]&lt;/img&gt;-CAX repeats), which have resisted assembly using short sequence reads. While most sampled lines carry the major allele opa31 encoding Q13HQ17 or the opa32 allele encoding Q13HQ18, many lines carry rare alleles encoding pQ tracts &gt;32 residues: opa33a (Q14HQ18), opa33b (Q15HQ17), opa34 (Q16HQ17), opa35a1 / opa35a2 (Q13HQ21), opa36 (Q13HQ22), and opa37 (Q13HQ23). Only one rare allele encodes a tract &lt;31 residues: opa23 (Q13–Q10). This opa23 allele shortens the pQ tract while simultaneously eliminating the interrupting histidine. We introgressed these [opa][1] variant alleles into common backgrounds and measured the frequency of [Notch][2] -type phenotypes. Homozygotes for the short and long [opa][1] alleles have defects in embryonic survival and sensory bristle organ patterning, and sometimes show wing notching. Consistent with functional differences between Notch opa variants, we find that a [scute][4] inversion carrying the rare opa33b allele suppresses the bristle patterning defect caused by [achaete][5] / [scute][4] insufficiency, while an equivalent [scute][4] inversion carrying opa31 manifests the patterning defect. Our results demonstrate the existence of potent pQ variants of Notch and the need for long read genotyping of key repeat variables underlying gene regulatory networks.

 [1]: http://flybase.org/reports/FBgn0003002.html
 [2]: http://flybase.org/reports/FBgn0004647.html
 [3]: /embed/mml-math-1.gif
 [4]: http://flybase.org/reports/FBgn0004170.html
 [5]: http://flybase.org/reports/FBgn0000022.html" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ECc22k7xuSzSY1l6KpW_b-dAO9utWJ81W7RXfHMBvU4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460177","apath":"\/ggg\/5\/11\/2405.atom","pisa":"ggg;5\/11\/2405","ac":{"\/ggg\/5\/11\/2405.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2405.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2405"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2405"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2405":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ROttxNr3e6KTY9+3o87Ar4Nux8KCtc+vcbBsUuQJjiaaatDKoaT+a4G7x72TxMGX9w9RoWNx7BbH2Qr6NCcKgqk1MzEqstOTcS+AL8P9LG3GitTvsTlFD9ot3JZ5Ek9lXEDF\\\\\\\/GqWv7NdhxvdRHRdHw4k0CGkSf8Z6p5\\\\\\\/kFvRoYZlPxVWknRtCSZi0CXChD19P97FKp7jqNTGdWfd8xcSpuQ4pVTIswy9A\\\\\\\/8tL751Z3RDshw6EfEXla8dTCpCGJRUEuuXHIafMrcLraHWOsFoKfqY2oW\\\\\\\/wxyaCkp5ALdDdfIJW3Siqrq732oDi83HylDFLLVyUcy2NR9WAwdHxLAdKJ7b4ux6Zrys7yyKX5l5HlJzKWkMP36vBmUpsWWEmuDtJ+A8g9R9aDW2gHjRv93ij223lRhQD9pECXmcOZCrbORUo9RoSlQAmbqoZAnnHb4tPkiVuhtUhuUuQpSjpXcYtv3pGEPBUzNp16TtCEBBxiBeeGxGZMps5oXSaPDyyEboQLB643ey+wE5Wz60ozDibGj0BsmuGl43GldW4APoev81FJBkWXdQrR000vYLkVkxLRCysZshOMgS2teAIdIlSVI2xZYC1LFSqCA9pHQcCiV5rDckK0MejeP1XhjLDoqF0+BcgLYu+U+vivn\\\\\\\/k9rFMYWvO\\\\\\\/dA7oimAH5M2B4ZezDR5TIwSDjmcNMM1yQ4BdN0PKYGa7sIHINUu7F8fpCG5KisKU2mp3W32AfHYq07rxou4Jm2hWk1IV+wEtCQZrnP6UJGDsgg+1GgVW2TpDW02fq+tU2mfMHg5+dErIMmK6o2nOQy\\\\\\\/EpdXhaHrrqPNgeR4gXUuShOUQf786PmKut1b\\\\\\\/Kvg55xwyiWrWvug7XHerkPb81cuVYolJ+56hYCmLKkzPPoYnJkLxJvWTeW6m6i+HtWybymuOLxDAksl9Ik92KZm304FljPrjmWzALKYBZW9sc6VxWh0RZqFkN62wS0D49SYO+1geyeX7XihzDb0VdYhBMm+R4XxR6slx0K+l1eAFBp3j+vfFOnrpkM7XEv70oM1PpT1ZS8Q3Ol6iiB9jVvhcvp2LjExHJDFDbYtPh8id82tquVI9Bskm6dsDIyQThqQEB8LikUvWkxbauHSRDK7CpmGULg9RauVsqm6\\\\\\\/V+8RLimiTCG3FefUoJpqp143Acd+7TIkVR0+YpKjtYw6tJ3eDC2Y+1TRHpwR7sST0Z1GUdJxn7u2TM7Ulxhnz2gn9KsExIoS9lF33NflXlhnAqrAfwy0PWDn\\\\\\\/DDS8iA4U6\\\u0022,\\\u0022iv\\\u0022:\\\u0022v7mOrHZ5wA17uqpLm3kIIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022207fff89455458c58a6a06ec4095ca6639f914907d83776bb735d7cd7e3861da\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224xCOGEdO\\\\\\\/BLFKVEbji9vgZQDB2u5qj8Hf6Zc5+amD1Acv+yDj2EGHj\\\\\\\/HnFJ3w6nxI5jFHW1WDLDyFU3bOhXa99ncbnfTzLIyrcaTZ6jPyk8j5O1P8LGcdmZGGtqv3o1fFim3ryp9HRNJ+KkuNRYrnR1icjhdaTq0F926VuTXW2DJHj11N0EMHUe8vUb56YS3DqnZ7RxG8QRu4AAhI9QwyYuiTmltWR9P0oVVjzuafDsy+j09I\\\\\\\/Oe0D3zXAmnW\\\\\\\/8BHdSY6c7gSZjaoTMXHb1bsstT0SN24LWgDOFxJijnmSk7clIzh2IyOfe10CXbHSo+PLTSgoA0Esmb4OKxhAc1IzDXBdFDP14CU7Z27EwS7NToM7CE5NjVe4lfyOwFWb\\\\\\\/zS02eJDeMgs1B1IZ+ysUAeq5B9II6HDptGKKB6I2ER8e6bYNjMThXG1NdiTyyXBoYUADI2QgeMOboV2GEdVzGNibsF\\\\\\\/9aAWLNHOS1H1+KoVMbAzjLc+akdXLOUuCsuIRlJ+UVB8aiLX2oNq9fXO6r5KfDk+zC4akigywAEG8XGJwSuGJ8Xx+pO6AdNKMWVo1ReZL2IKWstmRwxreH2kHL54JxdZLlBvYKhPHStZ\\\\\\\/YXeQsCsYcSwTJ\\\\\\\/uyPrgKrmbzHEH7r820syzbF9sA9XNf8t9PeDDOHelwhlXXSl4B2U98qVcosKCh0Y\\\\\\\/BItEndehbGtSw5jzF3Oms8YiekO8AmHVQ9mGC53l0mSAXwwe7Fa4hu8K9N5kJ\\\\\\\/RdFjcCC1XJ0z8KuOy89iMrv+KQmwsY43pj1Y2wj624plr5zdt40194CGxjCpl0Ta7pqoJJTrbiGeuuINONk+2gzpZIB3FBXNtNzEkIQLKlHHCBS9zsO5SmCS3C3+nQ4G1G2IfE2nwRoAwaZaFJrplS8AE9BAIoem9BvYSWMl8M4Yfl66hyx4C71iKWPAqWgYNb1qXROCXRyL44godtByVS9NNgsFRWQYLRQQ+UnOlxdbmXpyIWq13G6oBe3DTCzgkXSrj6HF4XlxoKqFLlxKdUpmOE+8w+U1Uzx7Reg6+5JQIzMS4EE6K0KWWywfK9o2sO9xXMrGob4znox4EkwJLBWL+m55+GFSuVblMGHYxxrkVCttT\\\\\\\/U7uBpWSknx13UPEqbwNUHWNx8apU6wW27G0SCy7fj0JOJPcEx1jsP7Q5ThMeLSrxrcmq4oqkbcWwK7W7usncqG4QeopByoIy\\\\\\\/SwBs0xaX0Dqqzh+ZNtlsJQYX5gZGoN5KcSYZIhyoYBsH9flU7X9zRDhJzuB9Xt5A\\\\\\\/jYWoilyn5j+B93LztUcQANHQvyMPcJTMGxn8rlnaGc2Yk9r6rR8FKHMQoPfQWUuQ7VMr6udlKiJ2BqMvnPj8lbEj5tYQNIefugSxmpFDE8YStTheOtJ7mAIrDkcIuhpU1Taw6trXCLm2KdLd6i4I57HE\\\\\\\/uJR0O5\\\\\\\/lR97kBvFNQXgEGzT9g4NQmnwG\\\\\\\/AN\\\\\\\/8DygAcjHD+GzPj+g\\\\\\\/LEJjIZ6MI\\\\\\\/W4N5hUfp\\\\\\\/sY15ftMZ53iR3OBtMTpCMm81X41kjTDoXSTSOPSV6zawBQ1UsxnzhopVmg6yIdglQ\\\\\\\/mz6nWWwo5yzc+2wDtrZgg706+Oh1wRWYOs82OsYOmeT+QNJzumEhgGLvJRfMkXf61Ikkw75idlL0XY1MvAQ42HBsRPUiGWbfdAn2qtwKb4i7s3ADbIqYIOqdisFuPKeqiQUoyqURuiFIwLJeZsEW09wFyD1ApMNsklw7+SajZoXx+CEiCQewdYHkL3FS78+1kE6owlEbQ91u1VRCB0srauGo14P0TAmfzvrAlPZkjNG+IENqPZpdHbhYk4qFYblLVFngR95r8WBg52HH8NGF9bHjSLSp\\\\\\\/9wiLI0hZBl0K7TefWis3qIXMyF1vpPg2x+DOE3hIFi+UVBx2OmzXowVyE8BQ6fuy6p\\\\\\\/0ZraefZK4uMQVAFDXzEEszqdPL8TW4eN9Cds6O5aAAWmQv4CkrVMliErAo+vq3uE39\\\\\\\/rpcMwoWItAG56r3XLOOm8X3sVfCrAKmYuBZXjTqztmfvyktgDjBRk6uzI2V54rTOMmcm5Z5h2NYOYRfCmJXRSPBuBeyeDtyduxhnO\\\\\\\/0g2yux0whNTR5MpyDLyjIp9MXLHWU2n6IFyLw0NXgFAFkcACFTbkbJPthcOe5aQm6E+yda8l8DFnrJJaefTNKvstQ7VvgCfRPj3cEzEMVHdmvvREAckD46v7ecBbuJgiTZiQotWho7qiPMYWplKwc7hpW2rIh0kJnVtwapRASUM9yAK7jhzTA+T4Y77zO4vlJ6MhDP0EC+3+YJsaPOr\\\\\\\/00LX1XeZdOQSCCGFC6ldH+rAaUKy\\\\\\\/4CStCFHDvj+99yfYIUEPwt3xLviU2fc8ljS7HwUpnxm0C8534ZCfM4K4nLKy8dD7+An6Cq050EF6blr01GUfCg8W7az5k\\\\\\\/EHZ11f2DnQauao9miKD058E6YCpshOoUgxIXT1GHOT7laZuKJPlHan46jl1ZZ6ACHXBTeCQDVA7\\\\\\\/ZV5cnGCq\\\\\\\/pc1UokI1tOot81J4OQhNn36dk1vkBNPVZedmKCue6uFYN4PT6s26m7ZEmnJFS8IGIuwYkUtzArEiXVch6ArUnU5jXuovmwz1dFsgO2bvSpIsExeLh8SWv6JanN+8FgFVLFkr2JHgYA3rwTVWgxJWYpADjWjr+4dIaclJeht4ULkoNy+3VqzRptVRyabivr+ItFtFVrCjzkV1XeKfp4MtGwpvcd7jwFYpy6rNFUHo5PXChBXW18zwWHZs7Ep0SERoL9VEfww\\\\\\\/il2Ny7ZMNMLGgxZ6P9ONASqi\\\\\\\/9mbd631CZ8nhlz8OZ3Bj+SsH5QfXam\\\\\\\/a40AMG06QqK60NO0jRo1wfNXwFafldLdbDKB0kSzyesiUPPhujJ\\\\\\\/e\\\\\\\/dnPFa1R\\\\\\\/TPQmErUpEl7ndPgSNAN1rI5sEiERo8n7h1y+khvIEReHjUr7VQp0kkj7gpCSQH\\\\\\\/ZbiYko05nSIdo7D6WPFWFEVQ9BHmAJmXd6naMb4mSZwifVhAiO7+EL\\\\\\\/jC9r0a\\\\\\\/PocHgDVK1Y4q8IDcD5BxemCqEFT+D5+gjtMc2uGCKTdvEMGkWXp3flDFH2JBucl9KO6cCYA0Z50fEgvpcWtIeFdc4mpYfiB0ctDUE\\\\\\\/9E7BA\\\\\\\/cL76JOGt\\\\\\\/kypLIQv520ThSEcHD5Cs9cv6HiGpe6sUnVWwa06fsTaPF\\\\\\\/MkW8Ebm10vxAPeDxf6lhPpwTx1KECq7YKpOdx81X2wMCAFGvEeOrZKpAKTW0HVOGlfOJ9\\\\\\\/uPLUSmZ6BvnOIgFFlBf7l7fYiWCMog8RsTj+9jcOU\\\\\\\/BCBSMsnNnFWeMZNkNXrZtfqNp68de8YatILM1y5aAXHhO6M3ffL1CX4WYTN\\\\\\\/N+ZAFSNg2AQ6k7T+sS0DndclBwCGxDjCTUi8kIrv5HbDfMwMMyNRvC4ztbYv8Zmyl7ZU1S0qSM1TTXe3Aq3kaZN8bsWVty0NcCFKwhF63vXVoPysxzD32I8mmIw5rSAnAtl1nKWljw313jrNt\\\\\\\/q1C5hiIZ+9r5uyUY01Jl7pk39CMlaQYpN0ne8mvoz9OjB9RhktA\\\\\\\/sUdTKLR5r62eiY+2h+G\\\\\\\/FAK9wD+YZ7UZFhcjGGWpgNd7a8ulYRNtHklXlP72YdGiXtdJTlF8ZlMlIOMd1ORvS4VsaB0Yr7CAPAXmFFerVytBnxEd83sCnMsWE9c5X+eQ9SBvzANNNSneJ3l64YJBqbpjNJgedw4mPau1se4H0sk7FAx3ARGFxfD821XA7zby15xDf1svpq+3AKVAbjFveYp\\\\\\\/9iDL6Q5BoeJ8H8dAOWgV8ORdPmHrwTICRCI1o5MRIFA4JTFzIpIJffS10gvGH1M\\\\\\\/J\\\\\\\/Xy7XgExe3KicUf5Th6RiREDQVO2oTUID9Pp+XEL0tPWWMvohFVWKzKs2YxMEkGCnA6kGKJF2XPKMw\\\\\\\/NY0X2dbFQ3GecvaFXTcJqZ914OtftP1ppIarCYU0UFvK4W807Kuk4b98F1BtK9I0xbyAnFDojkwJXBmBoR3y7QWkUcea1R0hJJ9pDfYNkx2+KJctp3\\\\\\\/aXg\\\\\\\/+A9Q+yFdVsiRsD38V9AZDI5VirOgYbmdoJMKkWcpWtmuoli\\\\\\\/1PnRjt1j8N+YmEHZ\\\\\\\/+s9zzWV6xrCaW2U6XQMMYA0Xol0i8aJg3DEoZ5CzOrgraqsRaZx9YMzYXotrExi9sy2Q8NDzUktgzH3G55ytDJDdtBnG\\\\\\\/gIIZOgLnnwKl59B9Lt+N7PCwu1XHZEWHlUt\\\\\\\/T7DM0PL20PaNb0snkxI6er97tH+3+Ix6A+ePBFh08glB9ovk2Nh8t4mzvTcxNfB47cWCd2KWuWfMq1WP+FVxZUvZx\\\\\\\/FiQVGttVt2KHOFWC1Sv82KV\\\\\\\/z3maTBPlhMahkYYrN7oyi\\\\\\\/Inq0LkR\\\\\\\/36TjLc\\\\\\\/kzFT\\\\\\\/nbnNPp8Wc8QJM9YjZFb1Kn6snDT+C6tszxoftv2FeZtBz3b1WdMiFVf187Jb4d7a90XT5lAzE4oX+ROEcTbBieH0vqptixVs2yCdmaKEb01jHDVSJurqsf3Pyc34xFK\\\\\\\/tKH24IrFGjLvh1cl4NFuF+tx8XfEe1XZtrY3uvhXeRnQVT0cOzk+fuNu3VDW5xBVvYxovjd5pEUvIwwdFoCvRvvw44dU4iKAc9gSIe2ZKCg9d6pEhRlkG8c9GfPmxA3Jxb\\\\\\\/AtShfEiDEzL5k4Umo3njPyZXZsZzLcoBaqgReWvtVmMSvThc4SPUBmc5cOkuesOBDfMTCdWvvCZ4PXtYw6hz\\\\\\\/LECOvVBz5OeZ0jlgDmTeqJtDUr\\\\\\\/p687KA81ZB6dDd8Y+jIo66H6Ghh8udYHzX50zJ88h\\\\\\\/c62X1ThDut6erL7oKonsPnIcivRu8OPyRrSQeQpB2xwprcC1ks\\\\\\\/4L2gEoezjgk5blTXdx6hvU2qs\\\\\\\/w5ODKIcAsma\\\\\\\/fVvNKSn8GQBJAZXXN2\\\\\\\/G7CWZHEz6I6Gh6isJI18fwfV3R25y4hz0zyc3asKIahpuHxHjDeZqcLebRNcQO3dqQophlFiy3dKrCGz7bt9KOrbJ74SheaCC0jjn8eR6eqm9dXegf8Rqo2qC05FY4WM3C0Fd0Au2Wtqgdc+mPKqlvMARL+sZdVf4HbZFaCzJmbxv62HpPxHaG7UB82ecEDzuJxysqnQbmgGcS1\\\\\\\/3FjLPiCsdXompgMj9JjWNflU7EAly6UJABkPBiXOp845lLof\\\\\\\/tYIL0OMQVU\\\\\\\/hyY5ecjunMu3lfxRxaJ\\\\\\\/P0LtGIODrsn1QNH8RwBGvwHAPqhj84wkSP2qD0RQ8pXvGL1t+cx5HwI\\\\\\\/VqMuRJqyQ9MkAmXuUdmHPHUQvR9jBB1kc5me+2VSmSXlPQZcJDwbjef+esxMkRATluf6VHOZ4q7Y1IFfGyQ0OjdFgM3KevrfD6EZTSizaaxeUU19E\\\\\\\/in91AkUwRnhy5aMziR5xv1D1VG2nqXEzhwM4XdP4m\\\\\\\/0KLegXch3DeNKJ4oUXKtQCvV7O\\\\\\\/lRBae+ig9iEbZ0+lj\\\\\\\/8XLI3QR09nJcxNyP+QUCul\\\\\\\/NalOjYq0B+MIE6CdBhiE\\\\\\\/f18dqu9XgCOagfwWmdenje81kZ5dptoB3PhCKKnT8ymvTm6kd8uDLQkxMsvWSH5GoLAYiKNVnd4mx\\\\\\\/5toX1sTSgxSOLPUSZxw01ddLj7dnlTLUSZm9j+jgprkVYY8fqugqIjiaIZ5Y1Ct9kJHZtLkYoz8gcGdRiOS0bcYh67xfJ9hGFEr3ioXASb6+ZbihyMkOH6UpAk\\\\\\\/xzffVKjlLPMBDMxlXxDvMXfKJYd\\\\\\\/IDCdj7sErDdNsuhWbbYmppduXT8+A6EbDyUd6y4cTZaNomGtkR4c32Hpovq7EkJjRso7yY4F+ndYLbSmeKdCDq5XaHqfrNpJVAUZ7NHFhtw4NTelJrYTPEeqgZMmPnjZd\\\\\\\/eyBI2u3VMwEpgd6kt1xjk1M6oOIv6MplMmpYF1xCJEqF5YHANaDIqUW2Li14KW9d+KQQa+IDMx\\\\\\\/IRnhAKmkKv48TN724TreOvQs4CTn50S6\\\\\\\/tJojRpgN5+r7\\\\\\\/F136ki4kLJu5uoc8j0jyuhm\\\\\\\/9CBdO96KvgeDSs0ItiR9WmJdMZbtGvWLxyhE1MahRPBsJEejH\\\\\\\/KYaR5zlDri80AMxUMsVYeePowjHagRZh3RXl4GnUvxI\\\\\\\/\\\\\\\/tkeONibpbSOxUgqmLKghvtqfA1jXZs8xPX\\\\\\\/n2f9\\\\\\\/6pkH\\\\\\\/asmHT\\\\\\\/pUgff\\\\\\\/zeg9hfbPh3hrZ9I0CyflsNrEBLx\\\\\\\/p2MIP9Y5sadZlDoHT4qh\\\\\\\/q1rYx6ZzMrr43uO9axT6KwqIDP9KG9cMY5s2gPbqCHjatx53BOUYmCduBe+HmXySGLqGXu6ZWhL7sIg4WTAhG6OnRdxtByuF7MsaeEC+zrk1RczF5BQzMA59EQMRKdm1\\\\\\\/NFS+ktjgoNDWl+nXHfclVwEHfI3WlqCnx009i+p9HEzk9DbRTDVyQTXY2qpCM1C7JzB6lsEDpwXZMLIq8IvAfPJEwOOKfWiOXZ\\\\\\\/Jz+QU8UZy7\\\\\\\/SOWS9lOLmxEdbVAhpNCippcnYl1f77y\\\\\\\/V+dWvMdNw6EZCLdO+OMMR+vg0pmldM15IVq3Ave779Uak5JBs\\\\\\\/Fhb4e+yPY3F\\\\\\\/B6x\\\\\\\/FnBo0dZFpumtlDbbcS1b3z0hr2d9+Mc8fhGBdS0cvDwO1tvbIj2b6EsQzyzX7eaaEv+6Pg92kR3gFrLZ0fIRqHCQden7Kmb7of56EF\\\\\\\/Q3XIEnaqsxUGfd\\\\\\\/uqXS10DvcblCn0RZsXu7EykZIRChIGagLk2VnYEdYnHt3w0ISfwsUNRSf8raDlURmE5hm05jz\\\\\\\/MsgVtqwBHwXnuvCdv8UEuYOdpZyLGCyFXmUbgCfUCeCohCV13vfj5zGWuGRCdqaxhZ1p+m\\\\\\\/JyjJbIAo5zOMChr36tWDPZOz8ZEfWVBkJ6Pk9s5KX17U2pObVzu8cYBPYiWASy\\\\\\\/ilrq0kexMkFQDv5XVOqseHLJc\\\\\\\/LRY3\\\\\\\/DsVmAu3Xxsp9OwMzWT4eZTosu4FY8XfV\\\\\\\/5zNWz5viEl18MmsdprKCkWyIgSjNL906UTxWCAz2w9oZ5iHOqqRaR9HN7VjWysDwMwvdfoR6Srdk9R2VpJFDA41QYt0KlTx3WO85mtMLQqFPxFPxwCWqG8GL0uAjtE9Nzh37dxP\\\\\\\/WoLJHK2SnFGszstxnxkRl147qEqiZJe3CQkhWrKv1DJM7qkbR3sGNO592SKnNin072Rgt25NHfft0YrEH4CQOZm+xSUF8+DPClL476jHIF1ana6ym6UbmxX1hZxnb5HUcW1OmCUaNpXoINyPDo3gjZofolTVUKQAuzHSQ6wbivVvyPQcBMDUH9S5zlYz8jGM2cxXXqR3dQl720ZYp5kYEv9YcXLA1rcCMiF6k57bSjJph9+xKjVTh2cKYKAkm+FgXpe40LO\\\\\\\/7Uq\\\\\\\/h1mPdmNnNwclvmVoSGQs2AGytZ66RSM26wrul6T\\\\\\\/LUBJGc01EwwWKfgxORdBc5c6znT7zz8BrFFA1HKRun8u2Kan37VWkvHrRk\\\\\\\/09ee5CgUSxBuUfjsgOD+nxzKG9TW6wp2\\\\\\\/SO86timLjzJ\\\\\\\/XHYtRXc5KKqVok37np4jPx3HIYHWjSgCZMa3vSEjzncb4NW8SOI+KV1K1Ymr1XoAbPdNLRin7i2x5Y8N2ywWob12uQ5kJr9NRS+YTOOLYbVN3dsDSdlZtdGgzWnSWzdn0MGmDLJ8oqwXNJb232qIlG\\\\\\\/538lb3inqPuajW34GCuwbPpujItffGbR+EM5uaI+PdBTLFU2tctp\\\\\\\/Iv0yss2BZ4liVlwnkQs52Tb1hoB0azsRPBdX2zBgspN3knhOHWP0ISN2c1G6TTWBMEGA\\\\\\\/ZBfBY5AiIEj2qU\\\\\\\/juKf4tvskLgoGrIB8Y821ZQt5KidMpaRKTCZdGKysRPGQu9BMtWRtaUh49J3ade7DP6aqNb9Fdu8UPcFYeFEL3i7BzWISh9OB0wY5TaGSnbbp8J11Wb63eLYAQncsWAy98dJ\\\\\\\/gdlSAWdSEVSROv\\\\\\\/CWBts+BwuIWfsKkOM1atFl3Cnlv0nWU20n\\\\\\\/ZmTeVTuUViU3vl35wTQc4IQWVu8EtjZpF+2DRh\\\\\\\/HSfNKiSVHh5LkRbAyaBqXYs8bS+SITrg9w97usXFW2XKN5PjKazJpR8IPlcReA3qILLh+\\\\\\\/9Sh4KIoAKM+fOlUHnt9e\\\\\\\/C9XhCrh0JxZnni9kaaavNpzV3Q6RKccGuBOj3iOd4Ft7GjcfzZMh8KrS69Mh7sRLbxrXFmc\\\\\\\/ApULTIFoLGNEKvWHZnQV83NCLkTc+JZvVdUNq4xi2jDo3UfQTKN8Cvwb9ugGBBhewZ6njwDNpxkxU8Vjvw1IfeMkkapoxVBdz2TyeRFKg08ZVA+QHrt4yyn8VwT6aDg4NgYsi\\\\\\\/YSk\\\\\\\/lNhQC7x9DiBHVQjynfaERlEtLoyTqBa\\\\\\\/YlCllDILOVFGT58cSbNqmBmFzbfeSNSyC2UAdXQYRqpgI9\\\\\\\/lry3DN2vM2tEGsKXOCR3asd5tZubY1MPovwUSfWDoDf0SAnhphMcOi3IcPaZgAILuueGDzTM1ZgzIZ3mYLSsWJHrjjKNQ7Z2OMJx+TDk8EgpkErthE7cFa9GhHAZOPPKF8YXgZ6qhxM5\\\\\\\/r0wKik5mOJXqmUF40wmueaeY3io6lsg+juxvDB67Zd8mSo8qdw3TdrsPjBQL3iAKEaNv8t9e7varF8XNFhcTn6kd3g0DMqlpQBupzz6h1z7PIEM5H6cg01POCk0XLCMZpdnOIIlv1kcRU4CRcCX5pOQsk\\\\\\\/CcvP8h4uCJy6XV6sNxf0GcU\\\\\\\/S+EEOKrGmk21A71h8Vi+EWn6VaaPh7334IAuNklLEXVK2lrrmDXY+F\\\\\\\/4zhEXlU+ANwW2Tb68Sbs9g5SqGmyUUXWp8zEHlzJ0alXzxtNTjLtFRo4gMUGvg\\\\\\\/ZzGnou+sChdYYxx1R6+D8J8Bi\\\\\\\/UMqktyc2IVceeetZt5P7H12Bp28IRvFR\\\\\\\/aTsTZtuAIpaIqJpgNh5c6pzXqUasXu2ajKZfn216cJmb5dsUubdH2SCVZd8YAXkLOSWLDFDqtsXCgsMAujoSOpLccC7os1EPIsXRqIq831ZGgcK3jMWymXdhhbngW630vEbpvnD3eP6HA76I51CRVXAVwsf4oyZeZ3YH1Ju+CaZxlTg5W3vQdG8sGUF6fXB8uhWP89BrQ6rZQV1+51etLJuzjiaVY5+zJ1as+kojzFory9uvuvEyDRVQolWFuIULo7amZAwi1YOY85zNOvzZoc5+LM6vjx+4nMlzgh+Jf3XY8M7ESBo9ZHb91jbeEQMR\\\\\\\/FmnYIFq1qwkfAb2xGIGFr2Sdkc7KCoWq1XFHZHEVRKO8UE8AKYzCRmFa5hb1I7etKrw0EdCAc4T+OipyXvsFydxUTAfppcWbd\\\\\\\/CiBx03Vty92rXLXcxEQN82aeiNTXNE7kvfnFVTXkn5aXSDVmKnUoF3M4ugddUwHYXgo0CeqCimnH6GCQekm3gFSBkPuxzZBv0XEwcpXevkZ1i5fI2CkDvVQxVRjjEGZFj0n9cXpPRAfl5fYtdhKdCO9ZuD\\\\\\\/yTqf6R+9QItLWE42jveIMTDorrLHFhJbN8eawXWYPyscPD6KKVFfqqzmswnzusSAXxaCRyFAlpsJep0\\\\\\\/K4lPdjOS2vQ23p4kVak39HO+sMWP\\\\\\\/jQafjLNY6bOyFip0aqGPw0N70qIUvajATzIz55SHz0grdCo3ovdWbfxzjijge4dOypkOkLjwE+OMINWIM3x8ygZqfHOWLCx20EqIT8aAPMQ9cDMXDHcmJe8GFLKB2iFkTv24Isoyz+WogEJ\\\\\\\/MPNJ+qwWnC1ykumB42rFVgeHr6FVLbs\\\\\\\/1bJDZh3HBk59cnj64XB\\\\\\\/J3sSN3AYjThiJmYthwzA63oxK6idYWVNsnMd+VzV7VFCcYUXczTLj0Q0CilUXarZjVvGL5GNBIMDCZLUK24t2mr6kyvgPL0PHt78spO4GgdNCw8Cfwdc+R8Hnix9eeil6zvwjKqqHwKpJCsytbDw8gXk\\\\\\\/\\\\\\\/NhX+jG\\\\\\\/Z3S8MV7lFQcMcPnU4craarq+2SrgIOg3SQeDBCFYdCRP4aba8vsN5cL0qqZ7CTVhvq+f74fiYyoZFN4Cw0rIPfc6TkYRFeh4IU2aZCXSEDnQMOONiWQABWtjUfzM83W2frFHp2Jb7GSi7l8y2M9ZDWIqTKj8n8K7cagCpNbKO8OEEV\\\\\\\/Y5S3jSyY+UFwKEFtvwmoqZR0TW+KL82AqEDk2JWlNOu1uXOG2uOUgjMDaW6pc+Hj4jX+fWxGCa3cvfLH6\\\\\\\/gs5ou7lRL\\\\\\\/SBJeOoZMCFLQ4GW\\\\\\\/RliPS+h7S6uHFVS+MIDFFPwO4MtjmohpQD5XPMqkzYBEwPQUg1htE5+DWENwJuKpJ9KxSiTrM89YsvvIpDvmsTNBJh1OegDlCZN1DdnKHEK9PANQA9NcuSEa2YZ12zW8KR1kAXMoJc6IPgWzkv3SgcZGvQ8eeWQGEwqCBndPLl3O68aDL5y7l1TMiEs0OM4Gs8vPa5nLDgd+h8xlPZnTnTpU7iFa0kSmGEm5cpmBaABl3Kk9pjjldaXLbjVTOQhIKtN1lOhxs7MKmIH03oHsYAF9FypcYXfehTIylmGsWmvu\\\\\\\/BArC5JJwqvkZPLFyh+nAZNGD0M6\\\\\\\/Rm1m+iv7TrRPHHp46iMbB\\\\\\\/HAeBp\\\\\\\/XKxj\\\\\\\/Z5gBWRl6GQ26dMKZG4mRIe4wxQQskLcAourqsssvZGqXd21lP9\\\\\\\/Nm7ePGNgLUPVI\\\\\\\/PU+wJencbadzyqke6vvAwxZ4AvRb6AzFbstX23vZZCKT6R\\\\\\\/pqWEUSiFj6ajAPJgv6Y8OI4YkltwCj1bXIaOHGGyeGrRgB+8dedaUTvPPkAKJkUBw4U5sJhN9hzYu7L6CxzSaXA7B+eckOG8Cc38We6PTrmxJZvits5fGp0FO7ah7GqVABphbGqYtRU4\\\\\\\/CwcR6Lc4PHHyYDU7LnJNjuBJhKZ+yWniC3ijrsw8fwbEDC97417bFdaN6\\\\\\\/lKtpbRWQtTu67Tk0U7nlRjrAQenVL1ZYHrSA3AMk4jBObAcb8CbfBDswCLzHA8994BUSB+DDWXpJDTkXo1F85awNUZCaciyjOVjofInWh8\\\\\\\/xLOajbxLxP1s6X\\\\\\\/G5Pby4hxeuR4yO\\\\\\\/NFrM3QzwMt1xpURZAEGh8NjyZPxo5wjeZZOSoV46R6UHR89vlq+\\\\\\\/tKk02otRfcFKFIAO4Enm5CwTFsraEuHcMQ9SOYBeLfp7\\\\\\\/wUw7NPUvrD69CLPwjdFF\\\\\\\/DTyV2N4RxWf6iIrDsgr+dg\\\\\\\/nDkQF0rDcSP7lY4JeYSscIV9GtRhfjQ1\\\\\\\/BxaRGhdN3nhQkGZdDsMuLo7MvV1FYtqJLcDmzD2BeWDLnD1sA7u6ELFAirnOtmCSIw38G5czIeY9BARuGR2NjQw5qaTs1r9nHRxRTGGYVImNlt72XfyHarcIdZzVA9SJc9p+\\\\\\\/x9YIcNP2cedyldJkQtGoc8FX7HIGEMqiBxe8ftJm\\\\\\\/Wk8jljZdpFREOLeCyzB7CrEBicJUH0l8XRjzVBm6lHXQdilne95zpoC9txuuiIUj\\\\\\\/+XwFy5q8DGh+RSBt1x5hYnO3YdZirr2T77aNg83eg49n+MD3t6I+kbm8KrYJDEQBpuD\\\\\\\/0YpWgauNZTW0PfxkT\\\\\\\/ysrSvFJy+5TW92d3H6NcQmU\\\\\\\/KH5NcygHZOQpIqLLO3Bo+ykNO5Zf20HblAc+PNj34KmpcWqBoCeWSy0mFZ2il7AZY3HgYwSi6rykc9SfqzjOX1nlgH2\\\\\\\/VqNfp0eheBUqxpCFa9mSCq0IJqkzOBjRLzvZ665cUykMPHqew5EScUGjg2d17pgIzlX+8siuKwdLsPe3bSbGwijc6KGFb764ltb1eSC+vS+HnCqLqI6Y9S8UdmU9jv9EyV9jim1+q73IagoFQx4ksJmbHbq7R45gCh2X0a4jp4Gcght69EaPqUClE7zZPqbMw5Xi08V1JYju+E+AJOQYtSyZycLIXmbPU2PnOI\\\\\\\/pYzMIcadkyYz7x10b7pvtGDJQY57QWTtU3unl7NVMs\\\\\\\/uVO7U3lsyZzyl7r+mbTh4hry6lYM8GOU0ZFbm+wmHwLmpLvF+9sIMF7HRhP6t7oUFjHA0MJsP4884ob+YGXqDVVAge6Sj8+IUlD2Q4rw0yYG8yqHi6cQE2cHO+ynA9q\\\\\\\/m7GcIxumfhdLZaSupGTIpOi7HdwXrPNcfrnTOEITuPmD2ILh+qDUctKyp\\\\\\\/f\\\\\\\/UE7QgGbHhY3d\\\\\\\/eAZI4LbQ88O4gJkPCg62jomh9fTmfu7bgE4RXE8kMUxqGfCqZkdYNhHzfPodTvgNbrAO3tOe3dfqpgIeWHlzsaq\\\\\\\/di5IUwDz2gB8BlAodQaSxptnGihQZua3WVaxarUuu5KaGbVr9v5qWth8nVzFUA11\\\\\\\/ayjKf7qJiVgDoAjYbEXO2mO7WaQOVQbyNdVkHsXZPCv\\\\\\\/JYB1LcRE6MqTyOIPJcoHOb6mum+H4NSQMAJn5BPUdlKuZbl+Qrz2QglfVuXNsRpQrYykG+qNgnkFADESrXKnqlUw28pOCRxPfBU+VElqTfT74JupB8TDSwy4jMcnp43pa5g6BMypvvmbW46C8JvooWjSC3R+P3\\\\\\\/9cbF8GxpNyxx0QHTCDIpJv0heTzsPKrpPgzjS0wSvHSoToJFDhPy6PJVEs\\\\\\\/HoBCIKterfFas5f77EOV6m1GvMClSmP77Mp0H0HafqwyEyd7so693VRHS4axZI0xavuVIExP4yeMdMUW1jWFn7R9r\\\\\\\/i4h0h+CS6ZQNrphbpQMysyElppvAVDDIgs85QtXVM1coFU6sZpfNrRo6WAgleboYyK4+WOfiNW+AxmgnWKjPLC7CaWt381lhDXVbnAFS\\\\\\\/qLiTtfapOPYMM7oY+DyJht4DpAoXRt6hnIPVLQPpg9x3zha3t\\\\\\\/aGAHW5bnXLoOXD7FHGhPcXCQsQfdLO9JACqB9ueykfl2AnGh39QXr4\\\\\\\/OQdccK8PPXRV0730g9dbIoDg7wNe3rQTw\\\\\\\/IyMh6BkChCWBnT0mV6oEbwbdLbdivreBedNiGmltfTbpchUYrSLYlvv3mKESwT872PkRU9YdpY+D6MuFnJX9j82VMlIAYplWvkJUL8PyVKhMuQKBtg21oqdeapU821iWhZIOMj7oedirzgXVPAjrcvuzRSTsN\\\\\\\/V6w\\\\\\\/Y2nBLec45Wwbg2jQ37z0v4xCxOeXRo80nSByWWYwrFb0SEfoaB5YW4M4BZGcJXmwJr0vgCpvFZYfi2gyH16OrXRBFcgJ+JnX2ah5rpbTLqOn8obW5UZoXe3B0BxSlKzuYz\\\\\\\/gk99jzco3q8oCdk5ZZ4I1+CHJBf0fDoNCwcIqItbFVUdgMK6SQZvaYjFf8PNpb0AI7WEWREv2DPmHqWep2eqIsvd3ksley+dHtu1hSgK\\\\\\\/CTjP\\\\\\\/Qb7\\\\\\\/mhAiCjdEj7djhq6xBpYIwWpJLwntjstMSS7SKWzyLJZqxxamxHivvsf+3+QrgYvyTjRKB1A2IniCk2of1dT45yAoND8Q2N+nYsksi4v1TYmpV4jqhhMuzEAXv5xlLeajqL4YuvEpHxbgxJZfvgwa2c7aWtPamHr+d\\\\\\\/LRQ7K8Z2GRPiQ8uVz4kixGoJ8xsc+Za0ZuLCCMyGW6oYWbJXv6MYZ0yKvFUi\\\\\\\/IlSDhGgYS5fhpw42dkxkANMx3GfmpUg9+YU9wkV4iAQ2ps6khQ0lKTAaJdlA3YM4mgoQFCq15rXe9MzItPAaloLaIG8VG2Ps7Cf4RcicZRX2jggomJYFCV2ZGEkN1mG\\\\\\\/2Ol8hbJGQintwyGgMv8ssXF+AeM6WOktTiPk9jfQR\\\\\\\/WqQ33fAne6ZpEATGNGjqHqhTtRnEgOres8f7fGYtPBA+t8rWgtfwj5FSUVJ0O8Q0D9TlLsCpDEbdE3cAJpdoxIShKJVi0P1oTLwL8rhpLpf7vfFfehrx6MEf+9d7cppbwU6o699zsI9IXdjvcgwQt1E56wg5VtqKWCpjws9GFCPMN+tHvFgCNhzKK+hDsM7uwG68+ig6mMi30SdcHBK+t2MwZPAg8JUv2apjgQBoG1Nuil+e263q\\\\\\\/DG6ZHi4CRPVciHPGvD7iZGb9YZcwryHthFqVmR\\\\\\\/ntKCmWIN5vn4gL7rITdAHL1lq+MZazMPpEhAxiu3s9rSZ3mrQPg8AzrDVp8ljRX5YjAXHzCKzlE51wI9NQ0QjLdRkq9uGZEi9g8Oq1O5LXl70nj5LS+stNFFkTZ8RFx5pjSCyDzixYqruuewyJbDN7CbzQzUvBl2zgiObBnDDMfEN7aSzbrecCSvIQT4Gi4\\\\\\\/KE5mNHjGB1gBWYFulWm83SiL4rH3E9dGMmEUFLECbph8iS2qVJjIyA6Rz5DjQbWRLxaS4RQkjSRXlDSLXBt2HKU622ixFiTl39f8pvX2iJi2eMP9Ly4TQpg4v8z3YunVtcPc1WuRqA4N+\\\\\\\/SF6x5yKAastsvE5+kbCBlOIh8+cch74lnYq3OT8uKP8EYApNFLNZOHSpbSdQD\\\\\\\/BbE0vEgjo12j4DQ8Y4tyTpEGL5rPVtF8TO\\\\\\\/Komwkd2AKxjoIJBnjIw4RAJSVbc1nk4xJ4OcPk7uNFfkrOcRR5IKQPM0evM6nTG1GlH6JLluIdsxnt8E3MCXB0ymO2AgiBVs+auI31ngqggxSyMiCTy7vQBxp40Bvi+Ia37oij6Meuow\\\\\\\/gtIP5N2wcT4t1PRT0E4FlOAW\\\\\\\/XhSZpek0QwAAlNnwvJncI3KgXt5BSBi+rIltlUBl+7qapuCAgj9GYTfCgG9B0hEvXiARGylESXtGPXtZWsYk5x5yfXPdGebW\\\\\\\/9F2E4b1JRkGntV\\\\\\\/WodVDaF28IKHEmg58lCpz7VKrghW8AQgkTinPaO5ZWL+StG0sM2MJcStZlGjdLM\\\\\\\/YEvPePhlpDsRU2vYvZTBXBsOGyHAJd5ZK8m8vQbKO5HqFikkOsJpUOi1Ej9ZnCSlDuJueUhyqF8nYN9dqACI9L1SRA+Z7mk2RnWFx5reA5B8TCofCzGqIchMwIvwORwLG2eM8pjcois05dXXJQWLQGipjvAevdRVvN9cx+o++BegdUwcP2nRZQXOmqihB9UlwXtHXC8MDp+S39IWb9GBk5yRzFluMMa+gml5a5NfFL3O5pewwENYhoFSIuR8pPwgk0Fpeh+BleigMMOIoKuL\\\\\\\/txLNMke8lD+qehCwlwpwHhx4+ltmAt3ixz9GW8gI1U0a9BoK7LbmO2A2VzmcteLa7e+NGolhDJv0+hmqJCqroDpz0RtmzD4jocWVEQqVLfyYrL\\\\\\\/CCP3ezp7TV1i0I5YqvFsb1SzxSwNCGrlOy7ul77Fy8iQ+MSK\\\\\\\/alcuAzFXTcg3XCbVsMsaREZGCowtk\\\\\\\/VZOHG9icYQxm3+NjTMyfcqZzC7bTNlNX5p3ppRGWFPNuoXHSQ7RDcBULAFQv4MFzxZhn8QTpF2PmBmLxeXEx0FaF1jpBKpZhB34hnCQV1zN9zUxoflyUfZa0nPAd\\\\\\\/bG5rplggz93sOkZ7o8fwMOQmubqvwmQ1eTqWBRNs1DT9+spiUjwIkJi0qjLjNoMt+pVti5WMJciIHkh35PDBGrWp70KB+xkhneKYBuaOS8koLR2M4xHsFiL153eUN7Ahou2bvooVldjmbUlnaMI+Ka8tPoiNupF982+FvUbBVuPn1uSBR4LkTkKV9dAFPRXBnC0p0kXHHLK5WaebpAK79Yo659iVlpOz4SoNhrxjpS7GFW6SC4IfXdkITuCYIJ9g94WiQA1Uzw7+LuSPMUOod0BsPg8C5idhm6cEqT1aG52hM9U5Nmj+jF+IWy+qLU56ldLxNZtVim7fjMvXl0kjdB8D13Xlut9Uovd30I+gvX9yeMJ0lGrdac8RNtG6nHqPWISQ0M5iozBEG308cdBiCTWSVS+61MkZbHce7yxlJ+sX13KrIH1QIOzk53ckkf4xBCQ+yv+TriAervCBiyp5PCcCDwzrg+2anxGcBGy9l\\\\\\\/t20Vb7gCqZGUROGcFKMZdWFk4yvh\\\\\\\/eIpQ3M1CwnrEe1eV2TXkbAnAV7rhStJ0lQ0R6Q9\\\\\\\/4nSWkw4VAOMWDFZlp3u1xpiZ0PmniIbpjcPV1wDVlzPDHghvLe6mSpgR7SHFMdNzQCPS+D4+ulG4SaynyoI0aCzsrQ2TRfGQu80JWblOBmUFZ74v5mD3DcN2EFoStpXVdy4LTRvZaIpPNLIntzclrGnEE\\\\\\\/R87dBHWvD+RBWmPOwgTDzUVO3iCsSImfbexCCRLQiOr9UFBYUxGqFByLRb5zGhpoYgz9089wG7SF1rgfDARfQVtOv+MZdaU3tJPPlyDFG01qXtfYRFfpAGL3vhjmZ1MupJo5UYz+NyqOC8qbQW4cvxzWjVMvbJ+X7WVzxjvQDUqjOY\\\\\\\/71UOlssGZ46y9URvfppFhwWIz0t21ymT\\\\\\\/EgYibfDMPhkfmin6Hq3YHhOuOBLljGCNaW+IVpUvap2dgPy+OnctgylJKz+WBEFv2FC7F6tDHympC564F9EjwgQ0Yup4pxMcxwsMPnEbnEuLneHkNL1LfcV65RLDVTQU8LTY7gyfEk1datLum8JuDYadE3rfAJgIjZ+3H5ti18ouQE0s29aJmvMOi9xN1n2etorfcT3XWUngwxPnave6TXpRlZqEDQH\\\\\\\/pkUXeP\\\\\\\/TxUJflUZs6Z4M3yX4ZnUT58csBaLP4aqXMhtJlImgetQjrOfwnssC5QSGCspuNIHG1b0IdSOFqWijHPh+aaIfPkyzDXR59QaMqObHFtMi+7kW3oLLhgvM2zCVB1hUZWsonRx\\\\\\\/1qHcpK2TX1hrfn6pVO39ZQTMkkEcTdPwYD31YVs8zwxdjtNGxuPjxvBCwt48DoygVh+GzouujjGaK6tewCRBAeiNIZJOopc7rFmH76Y4iePxwTmuRKdW9zEkfTojkDdEZymx5vEYZMVchaoXZ6VKIGfuaRcT6gQeqK6sYA41L4TT4NZkDHAw3Vg9upVPWFshm2WlRRmKzWdliREzjQFSIBAQxTjyQU8zmkgTISgxJZ9QHzu58PB0n4IXbnYiRnvYqk32mRAR6MG7qpEiL0uTlvWkX2bk6kLODwbzFpxcd3+FI0OSbdVYnh2F+NOF6qireJf1NmamaIWzpk\\\\\\\/Es9OtLYvS8mpxbbXwCFN2g9eHzEHlztTLL6+IvuZ5xYficQ2haH0\\\\\\\/IjHDoycBI+h8kjyqU+3SwE8l2jqKvWHEQkpREKCXi2y8AEPU\\\\\\\/+qr0\\\\\\\/BkPabEDzV8fIGWUYAhvJ8goxdkup1GvdZickiSNOLRYVBjRzcibofHx6P8XAXDI4A3IDlz5nqjx\\\\\\\/9PswIIEMkKSqGk+z\\\\\\\/Q\\\\\\\/RADBpkehYQA6gK5zS4F6oe6GYnvBH8NzmvTe22uAVxdeThuoThlVMPByUCeRH+C+odPnXJZb\\\\\\\/tKcXL1UAVcaCASGcMdwWOrLRPuVr+U+mY2Y0o4+xz\\\\\\\/W9Ev+S8Ok16NXlVG6aN1Lv5LEJP1TkeMmEmL64xm7O\\\\\\\/Swt7vV50Wr5Rq9d2Ou7iM1P1WNKYS81C\\\\\\\/mNSRTsXMIceObyRCh5cPkMwq\\\\\\\/vqMLzmrXnue2ScHj3e2PgL\\\\\\\/8cYOpuwhFbsI0saQZ9tXaS0dSS66GipEbqVWPnwtcAv1PYdA8mJh5FgkHYV+bEkdir9mfp1upKlSIHrxExzHkYb3fnSaHvqjXjfU2SiwVBCn57dojRcdfpRKj2jvoHLvL4nEg06xf9VvfbHe\\\\\\\/+q4Co5jCxOLD3Qpod1pz\\\\\\\/vSZ9SsetfVCwxPmt03\\\\\\\/4QExYCoF\\\\\\\/TuQx8exuJvz+fEpBrNp+6JmBVD3YIhR6Oh1W0993vGs8Xwe9T6sCQVgN385oc3shummhsUcrzB\\\\\\\/xF173p1LswFo3lpvciqd16d8ryEU9SHgiL7RD9XQj4NbJGKd6cSPLHKAqe8qQ0wtPtr67NZAu1E9TdHEOT7AWvW\\\\\\\/WtsBmgDw0BrTSGYOICe7prXy4pO9W8FEu8IlZQOliKCiBYqdgRPKtvNxuG7gCuI1f\\\\\\\/1VMYnf6om0iaZNqQmw8asjdd8bW4S8RjQi9MN6nO5QNcrXya7Omjsu6ihgXYqjfz\\\\\\\/ox19gMQxFK6GFhFz4kTlGlrGCmMEb8Xrd20UCU04mYW0lR7OuGe+M19EV9GO1GiMKZ\\\\\\\/lJ7gyTUrrehhdob0vMmZBYfOLPCztXu28bbq\\\\\\\/i8dO2OtIbaJo08GVViJOra\\\\\\\/G+OmOpAixxvll9fxTb+gbEUPmLfR3yHoTR1pR86qa5nQGi+ajV14++Q78v3S6ICC5r7eFo0KIstVLZurhaOvlaK9G+4SJFcajyEAY+8jUxDztK5DUr3AObCJbI0Iw1tdgQaIPvSioSLTpkfnGRH8C5a3RvMWnqkwdw21RdG7TZ3\\\\\\\/EAGips0qyVhMeRx+7XNUI3ldurpoTBYwGzeusigV66X32ne7Kz93G6dM+BxgaE7lAqBLK6ftYyFnOL3Jsl3vjfeM1YHK2\\\\\\\/mcSZNRFbemgekIeuB5FR6srZflY5512obgUQEqcg0Tp6tqequ+jbEFG2qG6bZg5QN774HP+A\\\\\\\/DHmzNQxx9HCy1YEwGlzlJNfeCTbsi6XQcktJwbR\\\\\\\/TGZAR2Dp\\\\\\\/8JMox\\\\\\\/Wr7hhtWb1Zz\\\\\\\/dceNt+Uy1iCoCcm++bXfqeOwJG77tFp13YKWyAuXi6TPayJ5vs7BqiQaf9+m7auIXHGLFQkgj+R7bvHhq2hkA1sJfpKarsHS0rPEpOVxCjynNIrgt3i1lxvJIRNCNtFq93YFc4qd1TjHAJo\\\\\\\/UfEgtPtYbexMguzZdmChva219pu4glhVcQfpLIU2aMYjWz2wH3l2IkRrIaaqEXmXXevn5ku4PfkM8ZHi1HQu8jXt3hMmVpXRSIi5BXjZLngU841XC+izTnd9IKEGH7UTHdGAkXByIBi0dMHyq6sbNm+5JsRqxHRCPVjygn4X65TNNS2OzAw+C6YMnpHj6JKssoyZwM2YvaPBpOtEPXD4HkKieiaagF4NTa\\\\\\\/Il+MbqiLBj9n1DpfJwHi6m0bu5bdkJUd+JxzlRz6DggU5QN6J1g1oNwveMhObQVRg2HDRpveFddQ0dd5pPGeNdg\\\\\\\/YGaqvCQO17OFCpSWnQz3Y8clyTd3nHVkAhJJs5aDS6mGAFiI7tG\\\\\\\/tkmhp8GDggD8m1gg23zSLx45Toc3hHQxEvKrc+mx\\\\\\\/Xyi+xX+NXCdpS6dGsMp3ALaqCfR3G9dDP7pUUIenu3ozPLNgAT5Kz42uYB2xKnhSzY0d4nultYgh+C9C5hN+gxRU5QGp\\\\\\\/qO\\\\\\\/ucgMW7xSfk3eWEvQnCjuOdcPejI85dexYTX4glNeNPTjUVMb2D7rJqaF+W7jpHLnJUy\\\\\\\/N9htLya02Jd84zZCUQu0VJ+cNvsv9pC5hB1Bnpo\\\\\\\/Y2Bws2tBJhubryY3offycgzPyoul+yuW9eClDjLlUh8xWqt9588LUYXNc1xeEQgwVojyybXAgikXky0ZhK9GIKJe+mq7YPVA+ZUjh8Z9O7g4AEls5QRKyP7gK8a18rUNRSQcke5PEsEf4mVYio3Z9J8cActzRYW2BqML08yi\\\\\\\/vPSIc7gifmbhnC9owjB94fJw1Nwf4jQnSJ6rlaywLSAgq5HnhbgMQcUn2p\\\\\\\/MLxHWEDjSyZzcPrkANj74pnZxrKAOpnAdT1EnNVWMaRHvJFxwgiG\\\\\\\/QOoWXMutO5ux933XVLOCex9GDnExOqJZxkSJlz\\\\\\\/o71Wb2zHuV85LHcCpJ4jrquMvTbMbeViJxGkNqdYb+rILTxr+eztwvbXAqLVioFR\\\\\\\/jhHsOzKYwq50oVM4cITy+tUPZ6IkTx91uKbddwcA0TOjjCPzEMtF6CRhdRilIE1M9JjEbje5ZbwxbXKzEmN62yRXYPt6Yclxa\\\\\\\/ruVyrDBgsODBFmncq5WyvpiRJ6wuLIavO88tywWZjgH+iOUHAJO6mGPS32\\\\\\\/rUVyKUdLBB8No2UT34P0YsXvVZfT6jgB1G8qq3cuOV\\\\\\\/lchPD45JpMzs3uvmp7Xlf0GiWnzFBiSUofr0DmUFsTY7Lh8JKDtBWVQEyqNbFYQiFmt35g9WO4Ptyd2OS04AIF3Yi6oqslFewax0HKIZ0hsltOO29aMTvb9zwyY0\\\\\\\/kwb3MUuWXe1sXjR68lGdkNaLHBTLE9kFr1jHI9Fd4bAY\\\\\\\/aLm+Xdw0n\\\\\\\/GrrP75B2Tfw\\\\\\\/EkhNEjcA\\\\\\\/ampzjq79JatWVvF8yRjgkEV33RSxmibGUQK02jZdyhsf0su3+Dou+ix3Amu8lQCPUNTj4+qWrz60ovVysh+9NzC2kMeiuRfenyUfFedr34fDpxQIswSpvhrPPeZj9kJU2tiBvVKXo3jIQyCmziY7w9o1fzXCN4xQZCa3vUYyRTLepezJyMT8b7N5LEsRPH9v6EcRtGUd3uK\\\\\\\/EQkVLS1kS58pKDPYAM3Y78sqEGqgpCKacBj7G4eR1f\\\\\\\/Znl12RCX1sowj4u823gGaLh3Q0JrWs51fIOhvPQQsctMy075pG2uPq\\\\\\\/mtOOJXCRcBrqwA+KWWhxkj6+jxO9irMhq2Dc+9TqEUw41yTb3SEQ3E4F40yz79aR6WHlf6XPFuqu2+N9MXrnNgIMBXstq1nyPDHp0Fam1CjKsQaXUvrVyYsDAlv18xNp7KVqDf0kr+QUCM5G3dnA8jDGYVHy7NJ0xhjYKMrKBmcUDPNfRBmr+HfEGQFTGL5yyJPoGDtDLwaAnaMjGlQDmQgc0TMr6\\\\\\\/XcPt3uxLY4NmGfRscscqTCtgchsyP1L0ONH6ON5xW4xOuVPj+WLUG\\\\\\\/6ar6\\\\\\\/sIm2wF\\\\\\\/8IGu2p7DoKMPNWMmWBxBtPRGeMNon16fFgqmjMAWapr1sTwR9MRRkGR7Pz43HlbD0Nnk6nHayWLPoVTwy807fcoxSCUPNNyCchgNls9kZZf5n44IfswGLbg93VN77afsjubaX\\\\\\\/qEqgboENc0aQ9S9E2teog8RAn4BfAVFEQe49sHnXUI1e2p9ogG79EC5NGylZoKCANPq+EB6irop53YrhLEXX+G6DG4bdlzN0TdV2\\\\\\\/wx5au\\\\\\\/XVejjJh3LAC9\\\\\\\/QM\\\\\\\/iUTNyicZQOWdtI+rNSzqgQ++XXzbn4RJ8cKfgOzbZMTrL4h8n+Fi0RH6QgraUHJ\\\\\\\/0nICncQF1CY9bn\\\\\\\/vm3y8W8pWzPeYKtdQw4V6xy8Auyhno6Xpl8u+1A9Gs9R8vNOY1ewbC3nHptWTTPhAH6V0FUHXry6387Dd2eNuCkJe\\\\\\\/Mf5JP7e2Ta8roToc2KZTnCtBzaKLE8YfTbOQsRkRTEKuJ4XHN\\\\\\\/VWFUz9UFFdhR9qv0tO6Tai\\\\\\\/bcM+NXtixjEsTcqVW9vybiGhpaWSf14oyGnQ09PwX9m+RQqXmmkCHsDMLDUzL\\\\\\\/pkOnBIok97kjDufUurqaTew6ANVgt3fPW2hqQFbALxMT8YhdDIxiCD3SIDtHNOjF\\\\\\\/yYUsxZh0fED00jDB00Hs0vcmP8qNsRdSsDRJ6jFEeB\\\\\\\/43WjulkGmN+fRIvylRK5jeS+ISFTPO\\\\\\\/wwqTnhPxzvAtSATede5FaNNqzHRS0n2K5Fj22Yc\\\\\\\/\\\\\\\/fe6GOeWUIW3ipDZXX4xPEmfCR4+16YWV6cqF1qLJaTk9hHM79\\\\\\\/HgRvnpXBfUt9GFs54NkBnrxBN0KNzaOv1oXPgbaQCBgRin24KyqW3007EowZ92lq+YwAsxpLFH62FOEAOwu81TptSqZRqc0LWTx5B\\\\\\\/x6rD25+IXVp6kD2BrAXtz+wHb6UK6gNOzBToJCYNKxCXq+9HB\\\\\\\/JPa2mXXmDCY4XWoNs1YQaxqaT5DW3nABSFxsYFI7KZBhnkvAWiFk65lQuFUHW2zUj8leJ+gGO2MTppAQOAolqtlGtlVFAyJdpecFUkSC4flTheXBVNK\\\\\\\/IlGN8FQh6Ec8TJpBKxfwPILcG6fsNOnB689in51mPX\\\\\\\/TenLObqdhj9HaZwwI4gZsdJPGgbZmq7l8jJTh+ch8BWZQNkDVa8uXCSDb08kbR6w3BK0+L+nIWPPGycW+UXHTbJMmgWzXlO0lKJthqq26QzdEGQX7qQ7wbf0OU3yJtLGsDfuRjHHKuGouhIIRamn8YYFz\\\\\\\/ZwAPC3pE9aorwiv6LZvOvYzA+l4TVc8aDouwUW13DcmXeO7ugY7Gtd6Z29biMW91Nk0Fa8ZKt1x\\\\\\\/MErIvoPnrNeyhQAOAOi+mw48PqVXGaV1uddR5i2BHIwYBhhlFCOrYXa+c5HprvCCRJ3qJa6WWuJZNfbVFhhQEqqVEDOPxN20Y+83w+Vgl8LCAXQOlZezQXLuc7w3shx3xXZN4y+JNN8Z87GZC1051nhCdc2KPq+51uAFz1a50wgH\\\\\\\/gXGAH+KOyKTynzmnm0Drxzoxf+N4tPykYlWz8u9PU0twB1P1+c9mDtRiCAjiBRIDpV1fGJn2SBoAloGc\\\\\\\/CaF7gR9KC9ymEkQiBt18ci7Z4KCvZD5TrBsMRgOlZOSMWUVX9DyaNKUGIeHBNuCP4o8UeTz5M1MG8cIZRo3HxVj\\\\\\\/PRvJAAjxpwE6B5J\\\\\\\/3ONFgnJUgVcFvdvWOVFoAvF9tFByCkaO0kcoAAyXI\\\\\\\/0ADGj2q6m+E5ee+K8sGKxyt\\\\\\\/\\\\\\\/Ivq+8IdF1Yebtxg4vUl+Nd7pPXXQJTy+nf9OR9ViYqYtsRD8zqvV4ef5kS+BtOLYD0zl4tohVMCQHHInnK5UIXZ6rYHFw5BWT\\\\\\\/qG7jzM+iwokK\\\\\\\/3Rx565narufMs5qCEnbICuMc+KH6+n6wmUrog6Bo9MZHHvsc5gTEr8t75zDA4U7RFDxyBGeSAMt\\\\\\\/uWM+ZvCnF2\\\\\\\/r+1KtF1eOltn+l\\\\\\\/vy8q5llPjmOJe2vILV1WvmYYMM1zkInZ5t1y6mHYl3X6v5ldO3HBTkpBmUQ+or+go\\\\\\\/sK9zRM3oQDiCMBb46dGlRS8NpgpyOYPjyIM2IJW7FCWRMyr\\\\\\\/eyv73TQrH4uWyjKTadGw\\\\\\\/Ux9QXDyjEHeBmR\\\\\\\/7lKFH\\\\\\\/ZmybqYJ5vco3fAjmBulDIBHM7+M7azfiTHscqJsMEnFB5x5OutpB\\\\\\\/VeRDzomRxpUEKXTBmpPQLx3+WhAFlyYpSyH3+a6\\\\\\\/iMAuIi2RUXVmLn+NgwoABQuuNNZJCq+RsSGoAR9DHFgtWKXknvXFAn6+3MAj3MnjPMLSSLBYhZRrCMf2ZC4iU8wUVTtkRN8rBCZ0tA3smnCTggBQVSGhMueGD\\\\\\\/rOKApuqIPJrEGUWczJJXemSH5iBA1LtEyaiXLzOwOSUu3lTGnO2ehKsjqUz8V8S91PXnjOLMOBUrm6zbARKDKMSs3rVRbHFfcdJ8yB88SLs9aCym0qC6eoRQIBavTSCKjlVSj9l39PeBaQdTrvYmpkJ\\\\\\\/NtkR7wTcFb\\\\\\\/utrhBKhQGEEyTDCITcvbTlBAFD4MKIewi5xnwfAZ0Z03J4cNhBfHpycuRXq5HOnsuqVDfmzZ5cfWeNrX13TEP6bSODOxbVaUb8ZJbNWFwXkoQ92JeKZp93VN2kvslt6E9ovwqNLKOhlmGqql8PBXTG765xVXzyNtgT3FJboYnFTtDHuTr92Dcn4Ibszcb48SlG5JUezIGeQvH+wdqXTJ4UDYBVD\\\\\\\/RMyj6ht88iXyh4EsL9leqenTBfI1M9cspcAvag+AJ+w2iMIu1XtHc0cMR95nnfc\\\\\\\/vL+\\\\\\\/Uko1ZoiKSLtyb+lo9KRrJr+7Sdj+FlvzDYPcCB0dNWU8qmTCQ5VfVJLuA33snz6JLfeeeft641bCILmdmAXQLxSdtWriUd4M7z8GYnNbc4sXHR4lhgf3S\\\\\\\/8mLwUaXZK7BPu6xJ6bbq5us5CaVUkPsOa171wLabxVSYpo7SOFe\\\\\\\/MaeQ\\\\\\\/CSyezpCIsLsICHs2tDl4nkeLg6X6KW7NMG5Ci4sOtKhRs66q2tufKrmU3FahWhypp01leNVfcW3rGhaCjlPWeef9XsvwByuXvA4A3kd91SzHQJ\\\\\\\/H8DX3bvumyW\\\\\\\/OWA8Hh2L0QBhxcRa0MFcRBVswZMn+XhK1gcnhGepbgIK2y74E6SaxVg6gnNqYL+1E2NX2szgzvSIc4UcPywEL8LFTagHuVjFZJirU5tfC31K1H45KsGs9DnFZ41z7YMdI7Tgnm8\\\\\\\/XDUOooLxvSS2Za6wf2njNRs8Ec2qw7uiNh1tkKOS+E3Fx6\\\\\\\/xQN\\\\\\\/DZyACvjbS4V3btqenwAC\\\\\\\/hapwvkYSCEyE8Xd57hlUiGi8gzQmlIPUq5Mp\\\\\\\/p5sWOlp2ecYyW8uqMGKa7Aqnw2LxIisUtxaVoz9j2CP+DgTpy84zFDgVwQVBr7iWiOjgQUxpZRrfNgSgxuE14cNs7XWEEm7+ha+qoQeD\\\\\\\/p6BJX\\\\\\\/f\\\\\\\/A6wjU5ff2Gw6FoBKFD1AWx0BqmBpXWf\\\\\\\/+VHEjLToAbUFtX8w\\\\\\\/TXFYDJgXRaPUSu6uPURgpHKHIKgB9LHuHRrno2bThJbpKRBXdpx3ARGtfokt7K2gXqS45JKNgwlTWQkhrWFNREfBTiXa8aGu1ZLAv0ERftbyXDOLcBpvDEHIjvC6YNFOr3Q0rssbyMoxNPjOs4II2ITI88dKWL0UnehJqq4jBHAOTFonnTnF+myxT+9yvt3KCGWNxEDbyCiTWtus5we6Sdw8n5GpeM4cl\\\\\\\/THuNfo5Lqu2UVd1lif77b2\\\\\\\/85N0p\\\\\\\/8ccyk0SUHgg5W1foTWkgfrHCB\\\\\\\/Ke9r99waQGDqytTkeJfgN\\\\\\\/KH7xpb6xG\\\\\\\/4dp0EMT6FjEbCbzGPIzHcDVD26y0aoMHB5mrRQR5TwyrO8BLensTIMotuhozozK7NBF1QzNnSuiorTpQM1wPb5YXmGwtLclJ9A6f+TRWL6spY7\\\\\\\/1yGFG0urTpjcXzKuuY4dlyKijus9rZnsayfraRJBJfiVqQIreeZV1eu4CKjhTJAyB4HvTYTgnGcOE2cNa7B5soWo2dpj+neGqfpMI1oKqfj6k\\\\\\\/1ATNL\\\\\\\/ZM+BIAcZFlunKJwyScGeHuoLNiwUzCLHyV9hbBXbMiz0MhOo8hsgBpaOgF+WsRqd0TOYKcaauUDj83qrMO8o6pWvTSVBmjuUbpNKBB\\\\\\\/ByZl4m16abUHfSW5PLDukB11C79PDnxqvrcY085BCv4x\\\\\\\/r0CjnX2V8h49KHTyIHHG+oSGiiUO8IyoDoPhK9SQrktdbtDkkBw+Z+tVPmE5qbljKvCObnIyOmI7LjWOLMyjUztPYkNQp1NLv8PdI4gOe4GJTqzBXe6WqhSMHWwMLFCFXVnlqhSk8j0fQucDqf8BFJWzuMZ5MXg1lHOInkkKSsb5ovKTcNgv8l08nmaHGNcPknRMoV+cRtBfAwP8aKdt4wjU8LT\\\\\\\/EVjwxpFSjK9nviau\\\\\\\/B5sag0yhEhLSWVnjI7CbnAfQ5Bq4YyeMIFeZ1VWNg4FN41gkoKYqYWws9ylt6axFtNmccvRAiX7Pt4WPkgc54WVeIhOPO1SaEUPPZ\\\\\\\/yVd9AWjwG+F+xQJjxi4oiQw6xsH\\\\\\\/E5h2lnER9uNpYiw8VYuNpAHfFGGqLaf0s9ywvSbVm9cIaODEm9dZkWqU\\\\\\\/j04y7zf\\\\\\\/iwWgRqzwmSiWwAM++JtYJRO5JbiE1LQbMw9mcWpuaIZVKrMT9pBr8rl+Pt9GAERiZiwjcobBNT1pbN4SAfH31zJ6ezDu0RoHwuE0KA3Iec9ei3wnKUjFl5i3blgDZ\\\\\\\/9VmlcfRfuDrSORnkhUIT3L3JUROymbtGCKnF9a9NT4vl+0i4p+evrXvbmplzrsp5Z69yrF8VGalpQcJ5G\\\\\\\/LkfCYdmWsvCky9CY9DiswZ34fuxsCH2Tdxhsg8UPZCArDOckOakKusn6ryRqzIAx\\\\\\\/Ot+abkJ5aZ6y2abDNjirzcvM3sz91Cx\\\\\\\/zJPdtq1jDP08L4uxKQYG5dCGkNpqHg7F\\\\\\\/uKjpdxgo\\\\\\\/I1O9mvm4rzxonqI\\\\\\\/\\\\\\\/J+QvQnL29RtRhU+WdUoL5GHAaaG0uZTwq4Rgg0GW4s5Z7W6Tj9r0io3uzmEieU7w17a5CmA9\\\\\\\/cUeNUP+48wobTDTu7kTthPDFriCOsc8JfKVPINxVWp\\\\\\\/0HB2DaDKTqytiwDx6NOD2pyyTUYa1eKn11biUorWkEmwH7Lwle2hYlHlHLf9\\\\\\\/YGzxisFvfNnB3dAKbnVg5L82HAqQUWCFWUqDj4iFANpeWsoKEDk+yXdoYhjACFoVoypsaUqzdC7YFJThRgEwZcSRt7BPTvcI2og+Bj5uMfkf8GRxRqNxXpCSOjmDwzhB7\\\\\\\/ekaq\\\\\\\/5lkk+JlxznMJX17XW4POSwu8G7fhBtWiFVSvxBDYoIzAlZEREczhTp+ZnsnOpmxKsFLwonojpv\\\\\\\/HSqPO0W\\\\\\\/F0hcZ7h6eCCP3N9Ma3icwgai8gO3ZGLSHTbE1UwfkVZ\\\\\\\/t1hMqisDVcrWig0BBexFZRma2DQ2cc5UNO16xsxxbu6qNJ5f8Mwplv+S3UFklsuRv1y8n5hUWLpmEXUE3\\\\\\\/iK072eyM+a+a+uglc9dzOAGUx3u6kPG1PHETy\\\\\\\/3AdFfJUQxZpProZGCVgnYvASIx0kW+i1h0xRyZhG2ncZD6ChpENpxf+Ye4z7XzVRdXRxcF6RdMCpAxQI+WVGjEUBeK3KbbwssLQBbHhEgs4huz1\\\\\\\/k8XckooP528PoHArnfEqQULEcOLe04mhNnfXAXdhvsoC0oIrO9lhSDfq6wUL4owuPbP3D444BbWNx9GTGBmYP\\\\\\\/USIwKd9P639RHLGKQmFRfE4zsG0MmRTBK8p+ZrASQcORwM7W2dJ0CP1EJ8iLgJj4FTXTkOBntc4ujZZkcjFnXUCHa2n3wKKrv7nrITaLU2n+onCfTYIh1TxPrWP1wUr7ahM8nYx6Y0u3syG6PTe4qi2aqg9\\\\\\\/Tkdp9J\\\\\\\/vJs8bR4dXw9bPAsLxpXq\\\\\\\/M5\\\\\\\/8bJjMc5N2TeGRKSj3nU6atEPDRidKc3mV6ltcCiScZHj8JC5YXg\\\\\\\/T5sI30GEruU6ixeSdGY4wNmTi2CKdhH\\\\\\\/N83\\\\\\\/B02h1es99WqJlvD\\\\\\\/ZhkmF4kHZmdmOUK6dMbJErobX1jJ4onddgAnbZCgZHWRE2IcXEw9chuan0e8IhMDDhe3b4kIILZQFH5ZnfIYt4ELu5ZUwbt8BUIZqD\\\\\\\/Z1Uo3rYuOMlvqirPSdNUnNDrfWeiqlBaXQt+1ddWTXPRYZQM4VhHdJpsq4Fxlesfc+wO6xZqM9FCMNsiVbiS5uBUqLs3AT5MolcQXbSQRDcSfnSQJxUMEvUfr8CWfAWxfcnATaRzA\\\\\\\/Zez5cp3JJm1ob178nhOx1fhQjjAhXDVQXPuy8v\\\\\\\/WLFBoj0lYStRC+o3QT0+zGOyTIjK5KDg2FhVKwoTl\\\\\\\/cQ7p5tFs\\\\\\\/XKjbzjfI5WSns8on4zskxgEFYBCwGa8VbayLCqHrbp3iVRdoiWwnLBBZiaZS4PPvy6322aYd2QwjxtDutrlp4arGfLhivcsPkjgUtgQXt56Ry59NNKfELOe53WmK5WizFf78aZ1tm0K8\\\\\\\/J+LKRP\\\\\\\/VlXOr8MN++ubzt6L+f5BMNgbCUeUxuaOyPFYZnwxoYggJTfL89tYSXjmwLRPCyKnqNTswkxaNQS9AcI\\\\\\\/XLLZyOwd7T3+kHRP0OFFJ\\\\\\\/ODL5MI8r\\\\\\\/WfPpLWjMg2WPE+geR3QGi\\\\\\\/DUDECDu2IN\\\\\\\/VlywBbhr6YID3z2meHMuZ4lDA4xJb\\\\\\\/DcvWfculQ5BV3nfrqIaPEJvXHHMvFEcBQfRgBIuRtXrVyzm0vHBDWaTxdGkpB5r80wWtF0TkXtJw3XHSwnGmIye2rsiBTR3YSsWZCFy\\\\\\\/AySCbToZu4GKDxsuDe6gXH3lXomsqjbs44S8w\\\\\\\/kshvVvHt5ofthVPgatQPiYVbP70Szo2A84RysQQIb9WB0KluBKScYGErZUrg1TZ\\\\\\\/Nq3fnBxS8yLnVpilyvUL2epXsB3ZigM1UAXkmMj5LiuRe+XRhD\\\\\\\/2Ta9+Mr2UoiKxLJx9j0lT8L6EbS0chcsyXiqjL0Ud\\\\\\\/iBn6gpCIKLuUKA3Jrcr1emaVpuoT2IDPEwp4gs9nngNYFHb8PMtxY4zITXd2Htnj1hPP8lzaDVqWl2aFId8huxEzWmf\\\\\\\/oVR4XYn\\\\\\\/4UF9uieQ4et1jG8zN3eKnGgUmCTe4E3A3PsWLVOChrZsgxKv0cT2hR5A0UFrjJb5thPokYPlIoygxf4m4F1Ro0lqGAQe2CtIjWXGDA5QbXF8HEpgedPxyOIt4fNrRjTskJNdBxFl3QGhZ+QyPz4JhU2dv++zdOWAB5ME8bqqiSn53VRCUt9rfWsnlqxyBdI0dpkWhpwKGrYoMj26OHbPtjkKZC2vT7A7bkVSJ1SY4426VA+nl0m783o+o5Cyqx\\\\\\\/NB+08s\\\\\\\/keNxD\\\\\\\/uWk6CqWLgO9oA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RoVQgQ9pWzhuXo1n3GaB9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002255d462b2a2b4038bb06c8852cef7ab13f24b52d0d7fab0ddb3fea7f6c0f5f587\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h3+lkYnmhVtm5m6J+gqoDACbCQ3IqVZmGikXEwPvYjuApI9050jS3XH8M1AlE\\\\\\\/yDwd6WQqFIOg\\\\\\\/PqlJqxbjZANnnxZqKXQ8boNHmJ23sGHT7UCmaLy8YtWIFNq5GriPXwRe7Xzs5y39fViAv+xzM8DkIuyQ9NGHt9aMcp+1BUqzGAksqfgc9NW6v2FhLTMGMAe7RL5oJP2w1j5Qtyghms4allEUybw5tM3snNeSNXI28GXSIa\\\\\\\/VbmuJ\\\\\\\/JDnnL+CcxwfN8QGAXbfPWKzon4zpTvgPU3ajM8mrQWGvRlK5HCiL9LhfaMQmkkKLG45h7qD08Lj5jXj3J5JJLDkT3qTzVKHZAXuPeL+\\\\\\\/EFha15xkTyyZdgX15klenFlivyXw6SQugcCMI44w5QGEBcgsiuIHFWbobsJTDH9vl7zC5IWtN\\\\\\\/I2GjzlqfjMleKxW7F5kmE\\\\\\\/T7k33f9sMIZhZ\\\\\\\/glXfCpIpwFjiakErX2TTRJqFXBhEKxzV6EQrVQxE0dSSUY4OB2pUQbNXr454Q\\\\\\\/O5KP\\\\\\\/qjEwzsd8d83jso3ehWv3NC+dTgnVcitLix9I8D1ObmGKalDYnf\\\\\\\/sq83HiXxCfq1JlJWMx7igEL9kbipBSms4oGmTyOprDLbd9zxJD+af8pzmXEkz4VE\\\\\\\/BGDunAZwUzVEzpvHyyMoYlIaBECW3NFIzxbhn3YnyzPSTxMQPa17b9fix2x1GwltANjwQ5q0Ip+1eTjRdymtZdXihP9uiiccaAiatO\\\\\\\/vmmpCMlQTbfIEtkIOSowgo1tY0Kuh9AG+TbItvQ7ufpZQiqUCXSQ5ORIC7xNVu6sd3MtrpmQQzvnW8bhU7PlamGas5Pi+Q+SkAPTCQwk+5X5Hr7q9mLuDQ6g1UYVPzcnOslbH+fnaV7ThFmTbmMN88wlgg\\\\\\\/XEShGkARTvYGongmzuXTbUct7v8k36+qfP7AYZvC78e3DOkhttEGnvty8le20BMt+ziqDZUP2gkvjNYDfcHl7AmIR\\\\\\\/UF0Ssi2f\\\\\\\/8rK0HuH6UDZ\\\\\\\/T7dxMbfiSTC77BKHdYpSPnnbw+qJCjQI++cvURKjDJ9+DGQGCqOGDytDi\\\\\\\/xs6DOXYqe0bDhmsS3F6RY89WuqDGJVf0jOVEoCzB6gZDHy0uvyVm0fXjW74N9wQKI21UpRXxXxq+fbJ0vYIpzyu3QceRxqSJgM7Zxh7x+DacNVglMH8c9lje\\\\\\\/rvdorlCvXRnWTlgm1Im9r4hTJ\\\\\\\/23c8MTxp7LW7OrSandL2wgcJ3at2Vw1O8VA4cQZ34qzcQocKMd5DdLCqYrv0\\\\\\\/PNqQ6UumKssGc5mKS97r859V3Dfxi\\\\\\\/KT6qU3HvVP1827yK38qsJ0mgPWYN6ETWouQadNY\\\\\\\/1R5MEzx7zFvny32cEePS2Bxdlc8K7RwRzsPeOG8t\\\\\\\/Xzp5tcViMqNic8rezAgVNdJcGAvVP6UH2zKIq+ZB8fuvIY+MhnxNWgwTXrt4NM5UQZuz2QILZdoDdrPYZpxONALKJGlYbJjGuzkUwW+MjwQpVrWYMP\\\\\\\/GV4A9mtkNT3STeC9C\\\\\\\/G7Ahihr8b5IRqw4NYlCiP3\\\\\\\/1oc4gci82A1yPfGRZbw8xiGq9fO77iWzKyyJc9tUm9T6XHV0CJQYR6YXBzNApku6AYMvVOC98W5bpCk3XYpRLN0fDKdeb9F9EfmEcWULI4Q1e5j0OpQKhQ5mtpOl9qLlKYkzoyaDQ2XPdK5Rk1rj2xHtz0KCbFV1BCzKGaUjDo3DUnAlhJ8ETR3LNOnzWRYrSOdf28nBNO8pThrbkoHlCfMf6HcDCpMDLkog2j0RPQX2SpbaAxDX+5BkGsPisoI1SuZ35xTJJ0v0L+BlB88RjiSxHzSuTo2hPF6AF8gKHMoKZO9yGXwd57GD4FwEeAT1WvdaUGjlDG0SsMYwH83vNLkLt373E9McmIT\\\\\\\/Jcd85U\\\\\\\/Cyuk6QO0YV3i3cutjuxFQaV\\\\\\\/tuNsNTnnlOMz77zk4kAunLNFzGOKLLUvyahZUGvKgQ7LDEDvLR3uFeFP0X0xrT6ccKWKx6mQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qO4KldBl45zvMmKsvgAoFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224fab610c7324a4029abb717c0ef01f58551366190c1f91dece612201e5344714\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460177 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2405" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sEC-JO5AT05YumsoA9utWMIofrF2D7c1cuu59d6xPEA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FakX--ZuJCSWAAWE-CM9ju1bK2caTwr8ImkUBvsihzM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2391" title="Genome-Wide Association Study of Ureide Concentration in Diverse Maturity Group IV Soybean [Glycine max (L.) Merr.] Accessions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2421" title="Flux Control in a Defense Pathway in Arabidopsis thaliana Is Robust to Environmental Perturbations and Controls Variation in Adaptive Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460177" id="node460177" data-pisa="ggg;5/11/2405" data-pisa-master="ggg;g3.115.021659" data-apath="/ggg/5/11/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Nature, Extent, and Consequences of Genetic Variation in the <em>opa</em> Repeats of <em>Notch</em> in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Clinton Rice</span>, <span class="highwire-citation-author" data-delta="1">Danielle Beekman</span>, <span class="highwire-citation-author" data-delta="2">Liping Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2419;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021659 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460177" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clinton Rice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BRice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BRice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Danielle Beekman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BBeekman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beekman%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BBeekman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liping Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liping%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiping%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2405" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460177" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2405.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460177" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2405.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460177" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2405.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460177" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2405.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460177" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Polyglutamine (pQ) tracts are abundant in proteins co-interacting on DNA. The lengths of these pQ tracts can modulate their interaction strengths. However, pQ tracts &gt;40 residues are pathologically prone to amyloidogenic self-assembly. Here, we assess the extent and consequences of variation in the pQ-encoding <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> in <em>Drosophila melanogaster</em>. We use Sanger sequencing to genotype <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequences (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-1.gif"/></noscript></span></span>-CAX repeats), which have resisted assembly using short sequence reads. While most sampled lines carry the major allele <em>opa31</em> encoding Q<sub>13</sub>HQ<sub>17</sub> or the <em>opa32</em> allele encoding Q<sub>13</sub>HQ<sub>18</sub>, many lines carry rare alleles encoding pQ tracts &gt;32 residues: <em>opa33a</em> (Q<sub>14</sub>HQ<sub>18</sub>), <em>opa33b</em> (Q<sub>15</sub>HQ<sub>17</sub>), <em>opa34</em> (Q<sub>16</sub>HQ<sub>17</sub>), <em>opa35a1</em>/<em>opa35a2</em> (Q<sub>13</sub>HQ<sub>21</sub>), <em>opa36</em> (Q<sub>13</sub>HQ<sub>22</sub>), and <em>opa37</em> (Q<sub>13</sub>HQ<sub>23</sub>). Only one rare allele encodes a tract &lt;31 residues: <em>opa23</em> (Q<sub>13</sub>–Q<sub>10</sub>). This <em>opa23</em> allele shortens the pQ tract while simultaneously eliminating the interrupting histidine. We introgressed these <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variant alleles into common backgrounds and measured the frequency of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>-type phenotypes. Homozygotes for the short and long <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> alleles have defects in embryonic survival and sensory bristle organ patterning, and sometimes show wing notching. Consistent with functional differences between <em>Notch opa</em> variants, we find that a <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> inversion carrying the rare <em>opa33b</em> allele suppresses the bristle patterning defect caused by <em><a href="http://flybase.org/reports/FBgn0000022.html">achaete</a></em>/<em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> insufficiency, while an equivalent <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> inversion carrying <em>opa31</em> manifests the patterning defect. Our results demonstrate the existence of potent pQ variants of Notch and the need for long read genotyping of key repeat variables underlying gene regulatory networks.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyglutamine" class="hw-term hw-article-keyword hw-article-keyword-polyglutamine" rel="nofollow">polyglutamine</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/notch" class="hw-term hw-article-keyword hw-article-keyword-notch" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Notch</em></a></li><li class="kwd"><a href="/keyword/developmental-genetics" class="hw-term hw-article-keyword hw-article-keyword-developmental-genetics" rel="nofollow">developmental genetics</a></li><li class="kwd"><a href="/keyword/dgrp" class="hw-term hw-article-keyword hw-article-keyword-dgrp" rel="nofollow">DGRP</a></li><li class="kwd"><a href="/keyword/gene-regulatory-networks" class="hw-term hw-article-keyword hw-article-keyword-gene-regulatory-networks" rel="nofollow">gene regulatory networks</a></li></ul><p id="p-4">Questions about the nature of polyglutamine (pQ) tracts in DNA-binding transcription factors and co-factors have arisen ever since the discovery of the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-2.gif"/></noscript></span></span>-CAX) triplet repeats in the <em>Drosophila</em> gene encoding <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> (<em>N</em>) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wharton <em>et al.</em> 1985</a>). This was followed by similar reports of CAG-triplet repeats encoding pQ tracts in other regulatory genes of flies (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kassis <em>et al.</em> 1986</a>), yeast (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pinkham <em>et al.</em> 1987</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Suzuki <em>et al.</em> 1988</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bricmont <em>et al.</em> 1991</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">White <em>et al.</em> 1991</a>) and other fungi (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yuan <em>et al.</em> 1991</a>), mammals (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Courey and Tjian 1988</a>), and insect viruses (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Carson <em>et al.</em> 1991</a>). The significance and interest for human genetic disorders increased after pQ tract expansion was implicated in spinal cerebellar ataxias (SCAs), Huntington’s disease, and other neurodegenerative disorders (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Biancalana <em>et al.</em> 1992</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">La Spada <em>et al.</em> 1992</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Orr <em>et al.</em> 1993</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Snell <em>et al.</em> 1993</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrew <em>et al.</em> 1993</a>).</p><p id="p-5">Study of the Sp1 transcription factor led to the proposal that glutamine-rich regions are transactivation domains associated with co-interacting factors sensitive to <em>cis</em>-regulatory binding site spacing (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Courey and Tjian 1988</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kadonaga <em>et al.</em> 1988</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Courey <em>et al.</em> 1989</a>) . Similarly, it was found that transcriptional enhancers integrating developmental morphogenic signals mediated by pQ-rich factors are sensitive to binding site spacing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Crocker <em>et al.</em> 2008</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Crocker and Erives 2013</a>). This led to a series of investigations that strongly implicated the selection of microsatellite repeat (MSR) variants in tuning enhancers targeted by pQ-rich transcription factors (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Crocker <em>et al.</em> 2008</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brittain <em>et al.</em> 2014</a>). Thus, we suspect that this MSR <em>cis</em>-enrichment at enhancers is a consequence of their being targeted by pQ-rich factors, which can also functionally evolve by MSR-related slippage in <em>trans</em>. Both <em>cis</em>-regulatory and <em>trans</em>-regulatory coding variation via MSR variants could affect the degree of pQ β-sheet interdigitation between the pQ tracts of TFs binding to adjacent sites in <em>cis</em>. Polyglutamine-rich factors have a tendency to aggregate (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Scherzinger <em>et al.</em> 1997</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2002</a>) but, more specifically, they may do so through the Perutz polar zipper (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Perutz <em>et al.</em> 1993</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">1994</a>). The Perutz polar zipper is a strong β-sheet structure created by two adjacent proteins interdigitating together. Furthermore, polyglutamine-rich regions are frequently embedded in intrinsically disordered domains (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tóth-Petróczy <em>et al.</em> 2008</a>), thereby reserving a role for a DNA enhancer scaffold to precipitate complex formation (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brittain <em>et al.</em> 2014</a>).</p><p id="p-6">To explore the natural relationship between pQ tract length and transcriptional interaction networks, we have focused on the Notch intracellular domain (NICD) for several reasons. First, this domain functions as a mostly dedicated coactivator of the highly conserved Su(H) transcription factor (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fortini and Artavanis-Tsakonas 1994</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Furukawa <em>et al.</em> 1995</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lecourtois and Schweisguth 1995</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bailey and Posakony 1995</a>). Second, Notch acts in a large number of developmental contexts involving tissue-specific enhancers targeted by Su(H) and other pQ-rich transcription factors. Notch pQ variants can thus be assayed through Notch-target reporter assays and other characterized assays (macrochaete patterning and SOP lineage specification) indicative of Notch activity. Third, NICD contains a single, long pQ tract interrupted only by a single histidine residue. The sequences from both the reference iso-1 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams 2000</a>) and Canton-S (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kidd <em>et al.</em> 1986</a>) strains feature the wild-type number of 31 consecutive <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-3.gif"/></noscript></span></span>-CAX codon repeats in the eighth exon of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>, the majority of which are <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-4.gif"/></noscript></span></span>-CAG triplets. We refer to this <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat configuration of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> as the <em>opa31</em> allelic type. The <em>opa31</em> version of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> encodes the pQ tract, Q<sub>13</sub>HQ<sub>17</sub>. Additional work has shown the existence of a neutral <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-5.gif"/></noscript></span></span>-CAG expansion in some <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> alleles corresponding to the <em>opa32</em> type encoding Q<sub>13</sub>HQ<sub>18</sub> (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tautz 1989</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lyman and Young 1993</a>).</p><p id="p-7">Here, we sample the range of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants in the inbred isofemale Raleigh (RAL) lines constituting the <em>Drosophila melanogaster</em> genetic reference panel (DGRP), additional classical <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> alleles, X balancers, and other informative stocks. We find an extraordinary range of functionally variant <em>Notch opa</em> alleles that are invisible to current high-throughput genome sequencing and assembly methods. The distribution of <em>Notch opa</em> length variants is highly asymmetric for <em>D. melanogaster</em> and features a long tail of non-wild-type alleles in the range from 32 to 37 residues with unique alleles encoding the intervening histidine at distinct positions within the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats. In stark contrast, we found no alleles in the range from 24 to 30 residues. Remarkably, a single deleterious allele, <em>opa23</em>, encodes an uninterrupted pQ tract of 23 residues. This suggests that the histidine may mitigate Notch pQ-seeded misfolding and/or aggregation, and that extreme shortening of the pQ tract can be balanced by simultaneous loss of the histidine. To determine the extent to which <em>Notch opa</em> variants are associated with <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> phenotypes, such as bristle patterning and wing notching defects, we introgressed RAL X chromosomes and X chromosomal regions differing in their <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat configuration into a common background. We find that both short and long extreme <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants are significantly associated with aberrant embryonic levels of both full-length Notch and NICD, embryonic failure, bristle patterning defects, and occasionally wing notching. Our results also indicate that these phenotypes are frequently suppressed by genomic modifiers present in inbred stocks. We propose that the <em>Notch opa</em> repeat configuration is an important species-specific gene regulatory network variable.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>PCR amplification</h3><p id="p-8">Genomic DNA was extracted from small pools of flies and individual flies of different lines. We used Invitrogen’s Platinum Taq High Fidelity, which is a mixture of: recombinant “Platinum” <em>Taq</em> DNA polymerase; a <em>Pyrococcus sp. GB-D</em> DNA polymerase with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-6.gif"/></noscript></span></span> exonuclease proofreading activity; and a Platinum <em>Taq</em> antibody for hot starts.</p><div id="sec-3" class="subsection"><h4>Amplification of opa repeats:</h4><p id="p-9">Regions of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> containing the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats of exon 8 were amplified via conventional PCR thermocycling (30 cycles of 94° → <em>T<sub>a</sub></em> → 72°) using one of the three following primer pairs and associated annealing temperatures (<em>T<sub>a</sub></em>): OPA440f (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-7.gif"/></noscript></span></span>-CAG TCG CGA CCC AGT CTA C) and OPA440r (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-8.gif"/></noscript></span></span>-CCC GGA GAT CCA CAA AAT CCA) with a 58°<em>T<sub>a</sub></em>, OPA808f (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-9.gif"/></noscript></span></span>-TTA CTT GTT ACA GGC TCG CCA TCG) and OPA808r (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-10.gif"/></noscript></span></span>-CCT CGC TCC AAT CGG AAT TCG) with a 67°<em>T<sub>a</sub></em>, or OPA720-597f (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-11.gif"/></noscript></span></span>-CCG GCA ATG GAA ATA GCC ACG) and OPA720-597r (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-12.gif"/></noscript></span></span>-AGG GCG GAT TCA TTT GAC CCG) with a 54°<em>T<sub>a</sub></em>.</p></div><div id="sec-4" class="subsection"><h4>Amplification of In(3R)K-only fragment:</h4><p id="p-10">To detect the In(3R) Kodani inversion, we used the following primers normally located on the same DNA strand: In(3R)K-R1/F1 (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-13.gif"/></noscript></span></span>-TCG AAG CCC GTG TGG TAA TC), which acts as forward primer if inversion present, and In(3R)K-R3 (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-14.gif"/></noscript></span></span>-TTC TCC CAA CGC ATC ACC AAA). We find that this primer pair amplifies a ∼1250 bp fragment if the inversion is present. When no inversion is present, amplification with R1 and In(3R)K-F2 (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-15.gif"/></noscript></span></span>-CTG GAC AGG AAG GGC GTC ATT AGC) gives a noticeably larger band than the inversion band (see <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/021659SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-5" class="subsection"><h3>Cloning and sequencing</h3><p id="p-11">PCR products were purified by gel electrophoresis (1% agarose gel), excision of the band, and gel purification (QIAquick Gel Extraction Kit, Qiagen). Purified PCR products were ligated into the pGEM-T Easy plasmid (Promega) and transformed into JM109 competent <em>Escherichia coli</em> (Promega) according to standard protocols. Blue/white screening was used to identify colonies with plasmids carrying inserts. Cells from individual white colonies were grown in Luria Broth with ampicillin overnight. Plasmids were isolated from cultures using the QIAPrep Spin Miniprep Kit (Qiagen) and sequenced (ABI BigDye3.1 and ABI 3730, Applied Biosystems) with T7, SP6, M13, and/or <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>-specific PCR primers. The RAL lines and <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em> inversion lines were genotyped by sequencing 7–10 independent clones with most lines having at least 10 clones and some lines having hundreds of clones sequenced as part of quality control (QC) experiments. The classical <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> alleles, X balancers, the wopa23, RAL-100, and RAL-105 lines were genotyped by sequencing three to seven clones, and many were also confirmed by sequencing a PCR-amplified genomic DNA.</p></div><div id="sec-6" class="subsection"><h3>QC experiments for opa genotyping</h3><p id="p-12">To ensure true genotypes could be distinguished from variant clones produced by somatic mutation, PCR mutagenesis, and/or interculture and intraculture environmental contamination, we conducted the following quality control experiments.</p><p id="p-13">To gauge the extent of PCR error, we conducted eight replicate PCR reactions using as genomic template the miniprep DNA from one <em>opa31</em> clone. We obtained sequences for two to three clones from each reaction for a total of 21 sequenced clones. Except for one sequenced clone from a reaction with three sequenced clones, all were of the original genotype. The single exception corresponded to an <em>opa30L</em> contraction (Q<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-16.gif"/></noscript></span></span>HQ<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-17.gif"/></noscript></span></span>) on the left (L) side of the histidine codon, a PCR contraction error rate of approximately 4.8%. Similar contraction error rates are seen in our other QC experiments.</p><p id="p-14">To gauge the extent of somatic variation, we used PCR primer pair OPA720-597f/r to clone and sequence 218 long (720 bp) and 276 short (597 bp) <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> clones from BDGP stock #26820, which contains a <em>P</em>-element carrying a <em>UAS-N-full</em> cDNA that is presumably nonfunctional in the absence of a GAL4 driver. Both endogenous and transgenic sequences feature a (Canton-S) wild-type <em>opa31*3</em> genotype (shortened to <em>opa31</em> here). Of the 218 endogenous clones, 94.0% were of the original <em>opa31</em> genotype (205 clones), and 6.0% were contraction variants (nine <em>opa30L</em> clones, one <em>opa29L</em> clone, and two <em>opa30R</em> clones). Of the 276 nonfunctional cDNA clones that we sequenced, 94.2% were of the original <em>opa31</em> genotype (260 clones) and 5.8% were contraction variants (13 <em>opa30L</em> clones, two <em>opa29L</em> clones, and one <em>opa27R</em> clone). Thus, both the functional endogenous locus and the presumably nonfunctional transgenic loci have identical error rates in a similar range as the PCR-based experiment.</p><p id="p-15">To gauge the extent of intraculture environmental contamination, we sequenced 360 clones from 39 individual larvae and pupae from the RAL-142 stock, which we initially identified to be polymorphic for <em>opa31</em> and <em>opa32</em> in equal amounts. We sequenced 5–13 clones per RAL-142 individual with an average of 9.2 clones per individual. Of the 360 clones, 93.6% were of the original genotypes (337 clones of either <em>opa31</em> or <em>opa32</em>), 5.8% were contraction variants (10 clones each of either <em>opa30L</em> or <em>opa31L</em>, and one clone of <em>opa28L</em>), and 0.6% corresponded to one <em>opa33L</em> expansion variant encoding Q<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-18.gif"/></noscript></span></span>HQ<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-19.gif"/></noscript></span></span> (two clones). We measured a 144:124 female-to-male sex ratio in this stock (53.7% female) and thus estimated observing 14 flies for each of <em>opa31</em>-only and <em>opa32</em>-only genotypes (males and females) and 11 heterozygous genotypes (<em>opa31</em>/<em>opa32</em> females) under Hardy-Weinberg equilibrium. We observed close to the predicted numbers with 13 <em>opa31</em>-only genotypes, 14 <em>opa32</em>-only genotypes, and 12 <em>opa31</em>/<em>opa32</em> heterozygotes if we assume that all individuals with clonal genotype frequencies of one singleton “minor” allele in eight or more sequenced clones are the result of environmental contamination from the DNA of bottle mates of other genotypes. Because there was a statistically impossible number of such true genotype calls, we concluded that these numbers indicated an intraculture environmental contamination rate of 5.7%.</p><p id="p-16">To gauge whether the shortened <em>opa23</em> allele in the RAL-646 line has an attenuated or otherwise aberrant mutational rate, we sequenced 358 clones from 38 individual larvae and pupae from this line. These individuals were picked on the same day as the RAL-142 individuals described above using the same picking tool wiped with ethanol between picks. Of the 358 sequenced RAL-646 clones, we identified six and seven clones corresponding to <em>opa31</em> and <em>opa32</em> alleles, respectively. These correspond to the two RAL-142 alleles, and an interculture contamination rate of 3.6%. These 13 clones were minor contaminants in 10 of the 38 RAL-646 individuals, all of which were found to be homozygous for <em>opa23</em>. Although this low rate of contamination did not prevent genotyping of any of the 38 individuals, we subsequently adopted more stringent control of reagents and picking tools and eliminated environmental contamination in later <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequencing experiments. QC experiments three and four were conducted at the start of this project and corresponded to the only time we ever saw evidence of environmental contamination. Putting aside the 13 <em>opa31</em>/<em>opa32</em> contaminant clones, we were left with 340 total clones sequenced from this stock. Of these, 96.5% were of the original <em>opa23</em> genotype (333 clones) and 3.5% were <em>opa22L</em> contractions (12). In summary, the contraction variants were identical to the (CAG)<sub>7</sub> → (CAG)<sub>6</sub> contractions seen in all the other QC experiments, including the PCR control.</p></div><div id="sec-7" class="subsection"><h3>Reassembly of RAL-646 opa repeats</h3><p id="p-17">To understand the discrepancy in the DGRP “Line-646” assembly for RAL-646, which was reported as a single H → Q variant but which we found to be a 24-bp deletion, we asked whether this line had changed since the time of the DGRP sequencing and assembly. For example, it could have been the case that an original change of the intervening His to Gln resulted in a toxic uninterrupted pQ tract. This nonsynonymous mutated allele could have been replaced in the stock by a subsequent more advantageous deletion that shortened this tract. To address this question, we obtained all of the original DGRP sequencing reads and aligned them to the iso-1 reference alignment (<a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS1.pdf">File S1</a>). Many of the reads in this alignment had mismatches to each other and to the reference sequence, and a few only had one or two that were plausibly polymorphisms specific to this stock. After removing the reads with high levels of mismatch, we are left with an alignment that suggests the two synonymous changes and the indicated nonsynonymous change seen in the Line-646 DGRP assemblies. Nonetheless, when we reassemble the original DGRP reads by alignment to our long Sanger sequence, we increase the number of original DGRP reads with 100% alignment (<a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS2.pdf">File S2</a>). Thus, the original RAL-646 stock was and continues to be homozygous for the unique <em>opa23</em> allele and not the reported substitution variant encoding an H → Q <em>opa31</em>.</p></div><div id="sec-8" class="subsection"><h3>DGRP line outcrossing</h3><p id="p-18">To produce our outcrossed lines, one to five virgin RAL females (from RAL-# parent stock lines) were crossed with two to three X balancer males from lab stocks of <em>FM7c</em>/<em>N<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-20.gif"/></noscript></span></span></em> (background one) or <em>FM7a</em> (background two). We performed five generations of outcrossing of virgin females to the balancer stock males, followed by two generations of inbreeding to rehomozygose the RAL X chromosome. We refer to these lines as the RALX#-bg1 and RALX#-bg2 outcrossed lines.</p><p id="p-19">To produce the <em>w</em><sup>1118</sup> <em>N<sup>opa23</sup></em> line (“wopa23”), we first crossed RAL-646 (<em>opa23</em>) virgin females with <em>w</em><sup>1118</sup> males (P<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-21.gif"/></noscript></span></span>). Second, we took heterozygous F<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-22.gif"/></noscript></span></span> virgin females and crossed them to <em>FM7c</em> males. Based on the distance between <em>w</em> and <em>N</em> and published recombination rates for this region, we expected a recombination rate of 1% (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Comeron <em>et al.</em> 2012</a>). So, third, we took 202 individual white-eyed F<sub>2</sub> males, choosing as much as possible those that had ectopic macrochaetes, and set up individual crosses with <em>FM7a</em> virgin females. Fourth, after F<sub>3</sub> larvae were visible, we then genotyped the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats from the single F<sub>2</sub> males by Sanger sequencing of amplified genomic DNA. Fifth, after approximately 50 genotypes, we genotyped our first white-eyed <em>opa23</em> male from one of the F<sub>2</sub> crosses. We took the F<sub>3</sub> virgin females and crossed them to <em>FM7a</em> sibling males. Finally, we crossed the non-Bar, white-eyed F<sub>4</sub> males to heterozygous sibling females and selected only white-eyed nonbalancer flies in the F<sub>5</sub> generation to homozygose the recombined <em>w opa23</em> X chromosome.</p></div><div id="sec-9" class="subsection"><h3>Embryonic survival assays</h3><p id="p-20">Populations of 400–600 parent flies from individual DGRP parent stocks (RAL-# lines) or outcrossed lines (RALX#-bg1 and RALX#-bg2) were raised at room temperature (22–24°) and allowed to lay eggs on a blue-dyed apple juice (AJ) agar plate at 25° for 4 hr 1–12 d after eclosion. Efforts were made to standardize the size of the parent population, as well as the age spectra. Embryos were counted in sections of the AJ agar plates so that similar numbers and densities of embryos were assessed across lines. These sections were placed in the middle of larger AJ agar plates with yeast paste smeared around the outer portion of the plate to attract larvae away from the middle. The fraction of embryos failing to hatch was determined by counting the number of unhatched embryos approximately 40 hr after the end of the laying period.</p></div><div id="sec-10" class="subsection"><h3>Macrochaete scoring</h3><p id="p-21">The ectopic macrochaete phenotypic data shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> involved scoring doubled (nonsplit) anterior and posterior dorsocentral macrochaetes (DCs), anterior and posterior scutellar macrochaetes (SCs), and posterior postalars (pPAs).</p><p id="p-22">The macrochaete frequency plots for the 13 macrochaetes of the hemi-notum involved the following scoring system applied to each of the 13 macrochaete locations. A score of 1.0 was assigned for a normal-looking macrochaete (normal thick bristle) at a correct location. A score of 0.0 was assigned for a missing bristle, and the presence of an empty socket was recorded if observed. A score of 2.0 was assigned if an ectopic bristle was observed near a normal position, or if a split bristle was observed (two bristles emanating from a single socket). Split bristles were rare and such cases were recorded if observed. A score of 0.5 or other fractional values were assigned for abnormally slender bristles that were either larger than a microchaete or emanating from a robust macrochaete-type socket. Occasionally, an ectopic bristle was observed exactly mid-distance from the anterior and posterior dorsocentrals or scutellars, and in such cases each bristle position was assigned a value of 1.5. A score of −1.0 (1.0–2 defects = −1.0) was assigned when an empty or missing socket at a correct location was observed together with a nearby ectopic macrochaete. Rarely, a score of −2.0 (1.0–3 defects = −2.0) was assigned when two empty sockets replaced a single normal macrochaete. The data for a specific bristle position for a given genotype seldom involved <em>both</em> ectopic (&gt;1.0 scores) and missing bristles (&lt;1.0 scores). To prevent the cancellation or masking of such defects using the described scoring system, only the more frequent defect (ectopic or missing bristle, but not both) was scored, with the data for the less frequent defect being assigned 1s. Typically, 40–60 hemi-nota from 20–30 adult flies of each sex were scored, but more were scored when many of the flies had wild-type macrochaete patterning.</p></div><div id="sec-11" class="subsection"><h3>Western immunoblots</h3><p id="p-23">Lysates were obtained from 50 μL of embryos collected on AJ plates and aged to either 0–2 hr or 4–6 hr after egg deposition (AED). SDS-PAGE (BIO-RAD TGX acrylamide) gels were loaded with 8 μL of protein extracts in loading buffer and typically run for 0.5 hr at 206 V using the BIO-RAD casting system and electrophoresis apparatus (Mini-PROTEAN Tetra Cell). Gel-separated proteins were transferred to PVDF blotting paper and blocked overnight at 4° with 5% nonfat milk in PBT (1× PBS + 0.1% tween-80). Blots were then incubated with one or two of the primary antibodies diluted 1:500 in PBT-1.0 (1× PBS + 1% Tween-80) for 1 hr of shaking at room temperature (RT). Mouse C17.9C6 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fehon <em>et al.</em> 1990</a>) and E7 monoclonal antibodies, which are specific for <em>Drosophila</em> Notch intracellular domain and β-tubulin, respectively, were obtained from the Developmental Studies Hybridoma Bank (University of Iowa) and used as primary antibodies. After four washes in 1× PBT (15 min per wash), the blots were incubated 1 hr at RT shaking with HRP-conjugated goat anti-mouse secondary antibodies at a 1:10,000 dilution. PVDF paper was washed four times in PBT (15 min per wash) before being incubated with ECL detection solutions per manufacturer’s instructions. The blot paper was then inserted in a plastic envelope and exposed to film.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-24">All of the RALX-bg1 and RALX-bg2 strains, which feature RAL X chromosomes introgressed into two common backgrounds, are available on request, as are the wopa23 lines carrying <em>w</em><sup>1118</sup>, <em>N<sup>opa23</sup></em>. Representative RALX-bg1 samples of three <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants are also available from the Bloomington stock center under the following stock numbers: #63522 for <em>opa23</em> (RALX646-bg1); #63523 for <em>opa31</em> (RALX440-bg1); and #63524 for <em>opa35</em> (RALX237-bg1). These were chosen for showing the least overall embryonic failure (see <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS3.pdf">File S3</a>).</p><p id="p-25"><a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS1.pdf">File S1</a> is a figure showing genotyping results for the Kodani inversion. <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS2.pdf">File S2</a> contains the reassemblies of the original sequencing reads for the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> from the DGRP Line-646. <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS3.pdf">File S3</a> is a figure showing the separate embryonic assay results for the RALX-bg1 and RALX-bg2 series.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Nomenclature for the many opa variations of Notch</h3><p id="p-26">The <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> nucleotide repeats are located in the eighth exon of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> (<em>N</em>) and encode two polyglutamine (pQ) tracts separated by a single histidine (H) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This pQ-H-pQ sequence is part of the Notch intracellular domain (NICD), which is translocated into the nucleus upon cleavage (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). These pQ tracts are intrinsically disordered and surrounded by many other similar peaks of disorder (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). While the NICD pQ tracts are immediately flanked by conserved amino acid sequence, we find evidence of multiple <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat configurations that are unique to the <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> gene of <em>D. melanogaster</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). To better relate our phenotypic results from studying variant <em>Notch opa</em> alleles, we introduce a simple nomenclature for the different <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em>-encoded pQ configurations.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The cleaved Notch intracellular domain (NICD) is characterized by a polyglutamine (pQ) tract configuration that is unique to each Drosophila species. (A) Shown is the 939-residue-long polypeptide sequence for NICD from D. melanogaster. Residues that tend to be secondary structure breakers are highlighted in cyan (p, g, s, D, N, and H), with the smaller amino acids shown in lowercase lettering. Glutamines (Qs) are shown in red. The seven ankyrin repeats are shown in alternating bold and dotted underlining. Two nuclear localization sequences, NLS1 (KRQR) and NLS2 (KKAK), are indicated with double wavy underlining. The Nedd4 ubiquitination site (ppsY) is also indicated in light wavy underlining. (B) Much of the NICD is disordered, as demonstrated by this plot of long disorder tendency based on pairwise energy content (IUPred) (Doszt&#xE1;nyi et al. 2005a,b). The ankyrin (ANK) repeats (residue positions 137&#x2013;384) and the pQ tracts (Q13HQ17 at residue positions 774&#x2013;804) are notable for having the least and most disorder tendencies, respectively (highlighted in yellow). (C) The opa-encoded pQ repeats of the melanogaster subgroup are characterized by two adjacent pQ tracts separated by a single conserved histidine. While the surrounding amino acids are conserved, each species features a unique pQ configuration characterized by the length of the pQ tracts on either side of the histidine. The assembly for Drosophila mauritiana indicates a unique position for the His residue even though the entire opa repeat sequence is still uncertain (Garrigan et al. 2012). In this study, we refer to different Notch opa configurations by appending the length suffix to the word &#x201C;opa.&#x201D; We add an additional index when more than one configuration exists for a given length as distinguished by the position of the single histidine (e.g., opa25-sec vs. opa25-yak, or opa33a vs. opa33b in D. melanogaster). Unless a species is indicated (e.g., opa26-sim), all opa designations refer to alleles from D. melanogaster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The cleaved Notch intracellular domain (NICD) is characterized by a polyglutamine (pQ) tract configuration that is unique to each &lt;em&gt;Drosophila&lt;/em&gt; species. (A) Shown is the 939-residue-long polypeptide sequence for NICD from &lt;em&gt;D. melanogaster&lt;/em&gt;. Residues that tend to be secondary structure breakers are highlighted in cyan (p, g, s, D, N, and H), with the smaller amino acids shown in lowercase lettering. Glutamines (Qs) are shown in red. The seven ankyrin repeats are shown in alternating bold and dotted underlining. Two nuclear localization sequences, NLS1 (KRQR) and NLS2 (KKAK), are indicated with double wavy underlining. The Nedd4 ubiquitination site (ppsY) is also indicated in light wavy underlining. (B) Much of the NICD is disordered, as demonstrated by this plot of long disorder tendency based on pairwise energy content (IUPred) (Doszt&#xE1;nyi &lt;em&gt;et al.&lt;/em&gt; 2005a,b). The ankyrin (ANK) repeats (residue positions 137&#x2013;384) and the pQ tracts (Q&lt;sub&gt;13&lt;/sub&gt;HQ&lt;sub&gt;17&lt;/sub&gt; at residue positions 774&#x2013;804) are notable for having the least and most disorder tendencies, respectively (highlighted in yellow). (C) The &lt;em&gt;opa&lt;/em&gt;-encoded pQ repeats of the &lt;em&gt;melanogaster&lt;/em&gt; subgroup are characterized by two adjacent pQ tracts separated by a single conserved histidine. While the surrounding amino acids are conserved, each species features a unique pQ configuration characterized by the length of the pQ tracts on either side of the histidine. The assembly for &lt;em&gt;Drosophila mauritiana&lt;/em&gt; indicates a unique position for the His residue even though the entire &lt;em&gt;opa&lt;/em&gt; repeat sequence is still uncertain (Garrigan &lt;em&gt;et al.&lt;/em&gt; 2012). In this study, we refer to different &lt;em&gt;Notch opa&lt;/em&gt; configurations by appending the length suffix to the word &#x201C;opa.&#x201D; We add an additional index when more than one configuration exists for a given length as distinguished by the position of the single histidine (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;opa25-sec&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;opa25-yak&lt;/em&gt;, or &lt;em&gt;opa33a&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;opa33b&lt;/em&gt; in &lt;em&gt;D. melanogaster&lt;/em&gt;). Unless a species is indicated (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;opa26-sim&lt;/em&gt;), all &lt;em&gt;opa&lt;/em&gt; designations refer to alleles from &lt;em&gt;D. melanogaster&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F1.medium.gif" width="440" height="386"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2405/F1.medium.gif" width="440" height="386"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467001" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">The cleaved Notch intracellular domain (NICD) is characterized by a polyglutamine (pQ) tract configuration that is unique to each <em>Drosophila</em> species. (A) Shown is the 939-residue-long polypeptide sequence for NICD from <em>D. melanogaster</em>. Residues that tend to be secondary structure breakers are highlighted in cyan (p, g, s, D, N, and H), with the smaller amino acids shown in lowercase lettering. Glutamines (Qs) are shown in red. The seven ankyrin repeats are shown in alternating bold and dotted underlining. Two nuclear localization sequences, NLS1 (KRQR) and NLS2 (KKAK), are indicated with double wavy underlining. The Nedd4 ubiquitination site (ppsY) is also indicated in light wavy underlining. (B) Much of the NICD is disordered, as demonstrated by this plot of long disorder tendency based on pairwise energy content (IUPred) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dosztányi <em>et al.</em> 2005a</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">b</a>). The ankyrin (ANK) repeats (residue positions 137–384) and the pQ tracts (Q<sub>13</sub>HQ<sub>17</sub> at residue positions 774–804) are notable for having the least and most disorder tendencies, respectively (highlighted in yellow). (C) The <em>opa</em>-encoded pQ repeats of the <em>melanogaster</em> subgroup are characterized by two adjacent pQ tracts separated by a single conserved histidine. While the surrounding amino acids are conserved, each species features a unique pQ configuration characterized by the length of the pQ tracts on either side of the histidine. The assembly for <em>Drosophila mauritiana</em> indicates a unique position for the His residue even though the entire <em>opa</em> repeat sequence is still uncertain (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Garrigan <em>et al.</em> 2012</a>). In this study, we refer to different <em>Notch opa</em> configurations by appending the length suffix to the word “opa.” We add an additional index when more than one configuration exists for a given length as distinguished by the position of the single histidine (<em>e.g.</em>, <em>opa25-sec</em> <em>vs.</em> <em>opa25-yak</em>, or <em>opa33a</em> <em>vs.</em> <em>opa33b</em> in <em>D. melanogaster</em>). Unless a species is indicated (<em>e.g.</em>, <em>opa26-sim</em>), all <em>opa</em> designations refer to alleles from <em>D. melanogaster</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The single intervening histidine (His, H) is relevant to our findings and is a convenient marker for naming distinct <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> configurations even when they share the same number of repeats. The His codon (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-23.gif"/></noscript></span></span>-CAY) is closely related to the codon for glutamine (Gln, Q) (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-24.gif"/></noscript></span></span>-CAR), and so one might expect to frequently observe His codon turnover within this tract, but this is not the case. The His codon changes position only indirectly after changes in the lengths of the two flanking pQ tracts. Thus, it is likely that the single His residue is highly conserved for a specific role. For example, His residues are known β-sheet breakers and have been found to attenuate the β-sheet forming potential of pQ peptides (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sharma <em>et al.</em> 1999</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sen <em>et al.</em> 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kim 2013</a>). Interestingly, species that are closely related to <em>D. melanogaster</em> each possess a single His residue at a unique position within the pQ tract (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-29">To distinguish the variants by repeat length, we refer to each <em>Notch opa</em> configuration by a length designation. For example, here <em>opa31</em> will refer to the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> nucleotide repeats encoding the 31 residues Q<sub>13</sub>HQ<sub>17</sub>. For cases where there were two or more observed allelic classes of the same length, we use an additional lowercase suffix letter. For example, here <em>opa33a</em> will designate Q<sub>14</sub>HQ<sub>18</sub> and <em>opa33b</em> will designate Q<sub>15</sub>HQ<sub>17</sub> (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For cases where identical pQ configurations are encoded by independently derived nucleotide sequences as determined by the unique patterns of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-25.gif"/></noscript></span></span>-CAG and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-26.gif"/></noscript></span></span>-CAA codons, we append an additional number to the letter suffix. Thus, for example, <em>opa35a1</em> and <em>opa35a2</em> both encode Q<sub>13</sub>HQ<sub>21</sub> but have been derived by independent histories of insertions and deletions (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shown is the distribution of opa-encoded pQ tracts of 41 RAL lines (boxes above line) genotyped by Sanger sequencing of multiple independent clones. Variants are shown from short (left, bluer boxes) to long (right, redder boxes), with each column representing a distinct opa allele. Heterozygous haplotypes (142a/142b and 370a/370b) are each represented by separate boxes. The distribution of opa alleles from D. melanogaster is highly asymmetric and does not harbor any alleles encoding " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Shown is the distribution of &lt;em&gt;opa&lt;/em&gt;-encoded pQ tracts of 41 RAL lines (boxes above line) genotyped by Sanger sequencing of multiple independent clones. Variants are shown from short (left, bluer boxes) to long (right, redder boxes), with each column representing a distinct &lt;em&gt;opa&lt;/em&gt; allele. Heterozygous haplotypes (142a/142b and 370a/370b) are each represented by separate boxes. The distribution of &lt;em&gt;opa&lt;/em&gt; alleles from &lt;em&gt;D. melanogaster&lt;/em&gt; is highly asymmetric and does not harbor any alleles encoding &lt;31 residues except for &lt;em&gt;opa23&lt;/em&gt;, which is much shorter and missing the His codon. Because of this asymmetry, we refer to the range &lt;em&gt;opa24&lt;/em&gt;&#x2013;&lt;em&gt;opa30&lt;/em&gt; as the &lt;em&gt;D. melanogaster&lt;/em&gt; allelic barrens, and to the range &lt;em&gt;opa32&lt;/em&gt;&#x2013;&lt;em&gt;opa38&lt;/em&gt; as the allelic jungle. Shown below the &lt;em&gt;opa-#&lt;/em&gt; axis are several classic &lt;em&gt;Notch&lt;/em&gt; alleles, X balancers, and X inversions, which we also genotyped (see text). Canton-S wild-type and &lt;em&gt;N&lt;sup&gt;Co&lt;/sup&gt;&lt;/em&gt; and &lt;em&gt;N&lt;sup&gt;nd&lt;/sup&gt;&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; mutant alleles have been previously reported (Kidd &lt;em&gt;et al.&lt;/em&gt; 1986; Lyman and Young 1993). Also shown is the distribution of &lt;em&gt;opa&lt;/em&gt; genotypes for 11 isofemale lines from different worldwide populations (Tautz 1989).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F2.medium.gif" width="247" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2405/F2.medium.gif" width="247" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466922" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Shown is the distribution of <em>opa</em>-encoded pQ tracts of 41 RAL lines (boxes above line) genotyped by Sanger sequencing of multiple independent clones. Variants are shown from short (left, bluer boxes) to long (right, redder boxes), with each column representing a distinct <em>opa</em> allele. Heterozygous haplotypes (142a/142b and 370a/370b) are each represented by separate boxes. The distribution of <em>opa</em> alleles from <em>D. melanogaster</em> is highly asymmetric and does not harbor any alleles encoding &lt;31 residues except for <em>opa23</em>, which is much shorter and missing the His codon. Because of this asymmetry, we refer to the range <em>opa24</em>–<em>opa30</em> as the <em>D. melanogaster</em> allelic barrens, and to the range <em>opa32</em>–<em>opa38</em> as the allelic jungle. Shown below the <em>opa-#</em> axis are several classic <em>Notch</em> alleles, X balancers, and X inversions, which we also genotyped (see text). Canton-S wild-type and <em>N<sup>Co</sup></em> and <em>N<sup>nd</sup></em><sup>1</sup> mutant alleles have been previously reported (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kidd <em>et al.</em> 1986</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lyman and Young 1993</a>). Also shown is the distribution of <em>opa</em> genotypes for 11 isofemale lines from different worldwide populations (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Tautz 1989</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diverse RAL nucleotide sequences for the opa repeats of D. melanogaster encode nine distinct pQ-isoforms of Notch. The opa repeat genotypes (third codon positions of -CAX shown) also demonstrate that the longest repeats of -CAG coincide with the most polymorphic regions, consistent with the effect of repeat length on replication slippage frequency and nucleotide substitution rates. We also note that the  region of the opa repeats corresponds to a repeat of repeats, (GGGGA)2-3, and derivatives thereof. This region is also a hypervariable site. Asterisks indicate three positions where synonymous substitutions were observed. Canton-S (CS) and other genotyped stocks are also listed. BDGP stock #26820, which is not listed, carries a UAS-N-FULL cDNA featuring a stable opa31*3 nucleotide sequence as determined in QC experiments (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Diverse RAL nucleotide sequences for the &lt;em&gt;opa&lt;/em&gt; repeats of &lt;em&gt;D. melanogaster&lt;/em&gt; encode nine distinct pQ-isoforms of Notch. The &lt;em&gt;opa&lt;/em&gt; repeat genotypes (third codon positions of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-CAX shown) also demonstrate that the longest repeats of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-CAG coincide with the most polymorphic regions, consistent with the effect of repeat length on replication slippage frequency and nucleotide substitution rates. We also note that the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; region of the &lt;em&gt;opa&lt;/em&gt; repeats corresponds to a repeat of repeats, (GGGGA)&lt;sub&gt;2-3&lt;/sub&gt;, and derivatives thereof. This region is also a hypervariable site. Asterisks indicate three positions where synonymous substitutions were observed. Canton-S (CS) and other genotyped stocks are also listed. BDGP stock #26820, which is not listed, carries a UAS-N-FULL cDNA featuring a stable &lt;em&gt;opa31&lt;/em&gt;*3 nucleotide sequence as determined in QC experiments (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F3.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2405/F3.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467014" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Diverse RAL nucleotide sequences for the <em>opa</em> repeats of <em>D. melanogaster</em> encode nine distinct pQ-isoforms of Notch. The <em>opa</em> repeat genotypes (third codon positions of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-27.gif"/></noscript></span></span>-CAX shown) also demonstrate that the longest repeats of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-28.gif"/></noscript></span></span>-CAG coincide with the most polymorphic regions, consistent with the effect of repeat length on replication slippage frequency and nucleotide substitution rates. We also note that the <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/F3/embed/mml-math-29.gif"/></noscript></span></span> region of the <em>opa</em> repeats corresponds to a repeat of repeats, (GGGGA)<sub>2-3</sub>, and derivatives thereof. This region is also a hypervariable site. Asterisks indicate three positions where synonymous substitutions were observed. Canton-S (CS) and other genotyped stocks are also listed. BDGP stock #26820, which is not listed, carries a UAS-N-FULL cDNA featuring a stable <em>opa31</em>*3 nucleotide sequence as determined in QC experiments (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The most frequent <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> alleles, wild-type <em>opa31</em> and <em>opa32</em>, also have the most derived variants caused by single synonymous substitutions at the nucleotide sequence level. We will usually not distinguish between these and refer to them nonspecifically with an asterisk (<em>opa31*</em>) only when discussing features at the nucleotide level. (In this study we noted three such polymorphisms and refer to them by applying the suffixes *1, *2, and *3 to the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> genotype base name).</p></div><div id="sec-15" class="subsection"><h3>Notch opa variants from the Raleigh lines (DGRP)</h3><p id="p-33">To sample a natural distribution of <em>Notch opa</em> alleles, we made use of the 205 inbred isofemale lines constituting the <em>Drosophila melanogaster</em> Genetic Reference Panel (DGRP) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mackay <em>et al.</em> 2012</a>). The DGRP stocks were founded by 1500 mated females of <em>D. melanogaster</em> caught in 2003 outside of a farmers market in Raleigh (RAL), North Carolina, and individually subjected to 20 generations of full-sibling inbreeding (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Mackay <em>et al.</em> 2012</a>). The initial bulk set of DGRP genome assemblies were based on paired short reads (Illumina 75 nucleotides), which are incapable of individually spanning the 93 base pairs constituting the typical <em>Notch opa</em> sequence and resolving alignment of its repeat structures. A subset of these was subsequently complemented with 454 reads as well as various consensus assembly strategies (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>). The DGRP Freeze 1.0 assemblies focused on identifying high-quality SNPs with significant minor allele frequencies (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>). Subsequent reassemblies were used to establish an assembly consensus to improve the sequence genotyping around microsatellite repeat structures (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>). However, in our experience with Sanger resequencing of the <em>Notch opa</em> repeats, the DGRP 1.0, and similar DGRP 2.0 assemblies are unreliable at this locus (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For example, of the 158 prefreeze PopDrowser assemblies based only on Illumina reads (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ràmia <em>et al.</em> 2012</a>), more than 40% of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequences contain ambiguous base calls, particularly <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-30.gif"/></noscript></span></span>-CAR codons. Similarly, of the 162 DGRP Freeze 1.0 assemblies based only on Illumina reads and aligned using different methods (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Mackay <em>et al.</em> 2012</a>), 33% of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequences contain ambiguous base calls.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466873/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466873/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sampling the <em>Notch opa</em> repeat genotype via Sanger sequencing</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">The difficulty of aligning lengthy repeat structures is compounded by the alternate subalignments made possible by the repeat structure (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>). This difficulty is exacerbated at the <em>Notch opa</em> region when reads are aligned to the reference iso-1 assembly (isogenic strain with genotype <em>y</em>; <em>cn bw sp</em>) because it contains a minor synonymous polymorphism (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-31.gif"/></noscript></span></span>-CAG<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-32.gif"/></noscript></span></span>-CAA; see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brizuela <em>et al.</em> 1994</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams 2000</a>).</p><p id="p-43">We performed Sanger sequencing of several groups of DGRP lines, which we chose by various strategies (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Given the DGRP1.0 and DGRP2.0 sequences, we had a prior expectation of finding few length variants different from <em>opa31</em>. We therefore chose to sequence 11 lines, which had ambiguous sequence annotated in their <em>Notch opa</em> assemblies, and one line (RAL-646), which was annotated as a simple coding substitution changing the central histidine to glutamine. We find that this set of RAL stocks, which we refer to as the Notch Dirty Dozen, carried three previously unreported alleles: <em>opa23</em> (encoding Q<sub>23</sub>), <em>opa33a</em> (encoding Q<sub>14</sub>HQ<sub>18</sub>), and <em>opa35a1</em> (encoding Q<sub>13</sub>HQ<sub>21</sub>) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-44">To assess the distribution of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants in DGRP lines with unambiguous assemblies, we obtained and genotyped 24 additional lines chosen as follows. A recent report on radiation-resistant RAL lines (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Vaisnav <em>et al.</em> 2014</a>) listed four of the Notch Dirty Dozen lines, which we had found to carry non-<em>opa31</em> variants, as exhibiting different degrees of radiation resistance (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). We speculate that extremely variant <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> lengths may lead to proteostatic stress. Aberrant pQ-related aggregation might induce the activity of stress response pathways and prime such lines to resist lethal doses of radiation (more on the phenotypes of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants later). To test this hypothesis, we genotyped 12 of the top radiation resistant lines (66/161) and 12 randomly chosen radiation-sensitive lines (within the range RAL-820–RAL-911), which constitute the bulk (∼60%) of the DGRP stocks (95/161). Altogether, we found that 58.3% of the radiation-resistant lines have non-<em>opa31 Notch</em> genotypes <em>vs.</em> 31.8% of the radiation-sensitive lines (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Furthermore, the average absolute value of deviation from the wild-type length of 31 amino acids is 1.8 amino acids for the radiation-resistant lines, whereas it is 0.6 amino acids for the radiation-sensitive lines (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Thus, nonwild-type <em>Notch opa</em> length variants are correlated to radiation-resistance.</p><p id="p-45">The strange RAL-646 stock, which is homozygous for <em>opa23 Notch</em> on the X chromosome, is one of only three RAL lines that are also homozygous for the Kodani inversion <em><a href="http://flybase.org/reports/FBab0005595.html">In(3R)K</a></em>, located in the right arm of chromosome 3 (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>). Ten additional lines are heterozygous for <em><a href="http://flybase.org/reports/FBab0005595.html">In(3R)K</a></em>, including RAL-440, which we genotyped as having wild-type (<em>opa31</em>) repeats. The Kodani inversion is a rare but cosmopolitan inversion that originated approximately 60,000–90,000 years ago, although it was recently discovered that it is almost entirely fixed in the African samples from the Oku range in Cameroon (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Corbett-Detig and Hartl 2012</a>). Because of the interesting <em>opa23</em>-related phenotypes, we genotyped the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats of the two other lines that are also homozygous for the Kodani 3R inversion as originally described (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Huang <em>et al.</em> 2014</a>), RAL-100, and RAL-105 (Kodani inversions confirmed for all three in <a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS1.pdf">Figure S1</a>). We found that RAL-100 is homozygous for <em>opa32</em>, whereas RAL-105 is homozygous for <em>opa33b</em>.</p><p id="p-46">In <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, we show all of the 41 RAL lines we genotyped as a single distribution. This distribution shows the frequencies of 10 different Notch <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> isoforms sampled in the Notch Dirty Dozen (ambiguous DGRP1.0 sequence information), 12 additional radiation-resistant lines, 12 additional radiation-sensitive lines, and two additional lines homozygous for <em><a href="http://flybase.org/reports/FBab0005595.html">In(3R)K</a></em> (individual lines per group are listed in <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). This distribution has three remarkable features. First, approximately 48% of examined RAL lines (19.5/41) do not have wild-type <em>opa31</em> repeat haplotypes and are thus not being accurately genotyped by current high-throughput methods. Second, it shows that there is a long diminishing tail of many rare <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> expansion alleles in the <em>opa32–opa37</em> range, which we refer to as the “allelic jungle.” Third, in contrast to the long tail of rare expanded alleles, there is a relative deficit of alleles shorter than the wild-type <em>opa31</em> genotype except the <em>opa23</em> variant, which is also missing the intervening histidine. Thus, the <em>opa24–opa30</em> range, which would encompass variants that are one to seven residues shorter than wild-type, is a veritable “allelic barrens” for <em>D. melanogaster</em>.</p></div><div id="sec-16" class="subsection"><h3>Embryonic failure is associated with rare Notch opa variants</h3><p id="p-47">RAL-646 flies, which are homozygous for the short <em>opa23</em> rare variant allele, are much more difficult to expand than most other RAL stocks homozygous for the wild-type <em>opa31</em> allele. We attribute this in part to a much higher rate (27%) of embryonic failure in the RAL-646 stock relative to all five other RAL stocks that we tested at 25° (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Interestingly, RAL-237, which carries the long <em>opa35</em> rare variant allele, had the lowest embryonic lethality at 3.7% of all six lines tested. To further test that the two RAL stocks with rare <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants have outlier embryonic failure phenotypes (very high in RAL-646 but very low in RAL-237), we also measured embryonic failure for the heterozygous RAL-142 stock (<em>opa31</em>/<em>opa32</em>) and the homozygous RAL-208 (<em>opa31</em>) and RAL-338 (<em>opa32</em>) stocks at room temperature (22–23°). We found that these had failure rates of 15.9%, 5.2%, and 5.2%, respectively, in the same range as all the <em>opa31</em> lines tested at 25°. Altogether, the seven RAL stocks that were homozygous or heterozygous for the two predominant alleles of <em>opa31</em> and <em>opa32</em> and raised at 22–25° had an average of 13.1% embryonic failure, which is less than half the failure seen in RAL-646.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lines with rare opa variants have aberrant Notch expression and suffer embryonic failure. (A) Among six isogenic RAL lines assayed for the rate of embryonic failure (failure to hatch as larva by all causes), RAL-646, which carries the short opa23 allele, has the highest rate of failure, whereas RAL-237, which carries the long opa35a1 allele, has the lowest rate of failure. Four different RAL lines carrying wild-type opa31 alleles have intermediate levels of embryonic failure. Bars represent averages of two replicate trials shown as dots. (B&#x2013;D) Embryonic failure is increased in outcrossed RALX lines carrying the extreme opa variants, whereas it is decreased in all of the opa31 outcrossed lines at 25&#xB0; (B), 18&#xB0; (C), and 28&#xB0; (D). Bars in (B) represent averages of four replicate trials, two from each background, except RALX021-bg1, which was conducted only once. Error bars represent &#xB1;1 SD. RALX237-bg1 and RALX237-bg2 have a consistently higher level of embryonic failure than all the outcrossed opa31 lines, but the RALX237-bg2 failure rate is much higher than RALX237-bg1. (E) The recombined wopa23 line carrying a smaller region of the RAL-646 X chromosome and outcrossed into the FM7 and w1118 background shows a higher level of embryonic failure than the w1118 stock. (F) Western blot of embryonic extracts from RALX-bg1 strains carrying different Notch opa variants from before (0&#x2013;2 hr after egg deposition, or AED) and after Notch activation (4&#x2013;6 hr AED), when the cleaved Notch intracellular domain (NICD, purple arrow) is more prominent. Gels were probed with anti-NICD antibody and anti-&#x3B2;-tubulin antibodies. Smaller NICD degradation products can be seen below NICD (smaller gray arrows)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Lines with rare &lt;em&gt;opa&lt;/em&gt; variants have aberrant Notch expression and suffer embryonic failure. (A) Among six isogenic RAL lines assayed for the rate of embryonic failure (failure to hatch as larva by all causes), RAL-646, which carries the short &lt;em&gt;opa23&lt;/em&gt; allele, has the highest rate of failure, whereas RAL-237, which carries the long &lt;em&gt;opa35a1&lt;/em&gt; allele, has the lowest rate of failure. Four different RAL lines carrying wild-type &lt;em&gt;opa31&lt;/em&gt; alleles have intermediate levels of embryonic failure. Bars represent averages of two replicate trials shown as dots. (B&#x2013;D) Embryonic failure is increased in outcrossed RALX lines carrying the extreme &lt;em&gt;opa&lt;/em&gt; variants, whereas it is decreased in all of the &lt;em&gt;opa31&lt;/em&gt; outcrossed lines at 25&#xB0; (B), 18&#xB0; (C), and 28&#xB0; (D). Bars in (B) represent averages of four replicate trials, two from each background, except RALX021-bg1, which was conducted only once. Error bars represent &#xB1;1 SD. RALX237-bg1 and RALX237-bg2 have a consistently higher level of embryonic failure than all the outcrossed &lt;em&gt;opa31&lt;/em&gt; lines, but the RALX237-bg2 failure rate is much higher than RALX237-bg1. (E) The recombined wopa23 line carrying a smaller region of the RAL-646 X chromosome and outcrossed into the &lt;em&gt;FM7&lt;/em&gt; and &lt;em&gt;w&lt;/em&gt;&lt;sup&gt;1118&lt;/sup&gt; background shows a higher level of embryonic failure than the &lt;em&gt;w&lt;/em&gt;&lt;sup&gt;1118&lt;/sup&gt; stock. (F) Western blot of embryonic extracts from RALX-bg1 strains carrying different &lt;em&gt;Notch opa&lt;/em&gt; variants from before (0&#x2013;2 hr after egg deposition, or AED) and after Notch activation (4&#x2013;6 hr AED), when the cleaved Notch intracellular domain (N&lt;sub&gt;ICD&lt;/sub&gt;, purple arrow) is more prominent. Gels were probed with anti-NICD antibody and anti-&#x3B2;-tubulin antibodies. Smaller NICD degradation products can be seen below NICD (smaller gray arrows).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F4.medium.gif" width="214" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2405/F4.medium.gif" width="214" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Lines with rare <em>opa</em> variants have aberrant Notch expression and suffer embryonic failure. (A) Among six isogenic RAL lines assayed for the rate of embryonic failure (failure to hatch as larva by all causes), RAL-646, which carries the short <em>opa23</em> allele, has the highest rate of failure, whereas RAL-237, which carries the long <em>opa35a1</em> allele, has the lowest rate of failure. Four different RAL lines carrying wild-type <em>opa31</em> alleles have intermediate levels of embryonic failure. Bars represent averages of two replicate trials shown as dots. (B–D) Embryonic failure is increased in outcrossed RALX lines carrying the extreme <em>opa</em> variants, whereas it is decreased in all of the <em>opa31</em> outcrossed lines at 25° (B), 18° (C), and 28° (D). Bars in (B) represent averages of four replicate trials, two from each background, except RALX021-bg1, which was conducted only once. Error bars represent ±1 SD. RALX237-bg1 and RALX237-bg2 have a consistently higher level of embryonic failure than all the outcrossed <em>opa31</em> lines, but the RALX237-bg2 failure rate is much higher than RALX237-bg1. (E) The recombined wopa23 line carrying a smaller region of the RAL-646 X chromosome and outcrossed into the <em>FM7</em> and <em>w</em><sup>1118</sup> background shows a higher level of embryonic failure than the <em>w</em><sup>1118</sup> stock. (F) Western blot of embryonic extracts from RALX-bg1 strains carrying different <em>Notch opa</em> variants from before (0–2 hr after egg deposition, or AED) and after Notch activation (4–6 hr AED), when the cleaved Notch intracellular domain (N<sub>ICD</sub>, purple arrow) is more prominent. Gels were probed with anti-NICD antibody and anti-β-tubulin antibodies. Smaller NICD degradation products can be seen below NICD (smaller gray arrows).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">To assess whether other issues distinguished the growth of these stocks, we measured fecundity at 25° (number of eggs laid in a 28-hr period, dusk-to-dusk). We found that the RAL-646 stock has a nearly average fecundity, demonstrating that this stock’s failure to thrive stems mainly from its increased embryonic failure, possibly caused by its <em>opa23 Notch</em> sequence. Of note, the inbred RAL-237 line again outperformed all other stocks in this regard (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466938/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466938/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">RAL lines with the shortest and longest <em>opa</em> variants have average or above average fecundity</span><div class="sb-div caption-clear"></div></div></div><p id="p-52">To investigate whether the RAL-646 and RAL-237 lines were exhibiting outlier phenotypes in the embryonic lethality assays due to their variant X-linked <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats, we assayed this phenotype and others after introgressing the different RAL X chromosomes into two common backgrounds (see <em>Materials and Methods</em>). For background one (“bg1”), we used an <em>FM7c</em>/<em>N<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-33.gif"/></noscript></span></span></em> stock. We refer to this series of introgressed lines as the “RALX#-bg1” series in contrast to their “RAL-#” parent stocks. To rule out effects due to <em>N<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-34.gif"/></noscript></span></span></em> modifying suppressors in background one, we also created a series of introgressed RAL X lines using an <em>FM7a</em> stock (“bg2”). We refer to this second series of introgressed lines as the RALX#-bg2 series.</p><p id="p-53">We found that embryonic failure increases significantly for both the <em>opa23</em> and the <em>opa35</em> outcrossed lines, whereas it becomes attenuated in eight different <em>opa31</em> introgression lines (compare A to B in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). While the <em>opa35</em> RAL-237 stock had the lowest embryonic failure in all experiments, the RALX237-bg2 has the highest embryonic failure at 60.6%. We found similar results when we repeated the embryonic failure assays at colder (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>, 18°) and warmer (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>, 28°) temperatures than 25°. Interestingly, these results suggest that the major allele <em>opa31</em> is optimal in outcrossed lines, whereas the <em>opa35</em> is optimal in an isogenized background with compatible suppressor modifiers. In contrast, the <em>opa23</em> allele is likely deleterious in most genomic backgrounds and at different temperatures, as we find that this continues to be the case when we recombine this region out of the X chromosome of RAL-646 (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E</a>) in the wopa23 flies discussed later.</p><p id="p-54">To determine whether the embryonic lethality differences in the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> allelic series is due to Notch proteostatic issues related to its pQ tract, we performed Western blot analyses comparing embryonic extracts from two time points corresponding to pre-Notch and post-Notch activation. Using an antibody specific to the Notch intracellular domain, NICD, we found that extracts from <em>opa23</em> embryos (RALX646-bg1) had substantially increased levels of full-length Notch receptor at both time points and higher levels of NICD at the postactivation time point, relative to extracts from the wild-type <em>opa31</em> embryos (RALX822-bg1) (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4F</a>). In contrast, extracts from <em>opa35</em> embryos (RALX237-bg1) had modestly lower levels of full-length Notch but oddly similar or slightly elevated levels of NICD relative to wild-type <em>opa31</em> embryos (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4F</a>). Thus, we found there are observable differences in the levels of both full-length Notch and NICD in the outcrossed lines carrying the most extreme <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants, which may be related to their differences in embryonic survival.</p></div><div id="sec-17" class="subsection"><h3>Bristle defects are associated with rare Notch opa variants</h3><p id="p-55">To determine the extent to which classic <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> phenotypes are associated with RALX outcrossed lines carrying rare <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants, we measured the frequency of duplicated macrochaetes on the notum of adult flies in both parent and the RALX introgressed lines carrying variant <em>Notch opa</em> alleles. First, we scored the frequency of ectopic dorsocentral (DC), scutellar (SC), and posterior postalar (pPA) macrochaete-type bristles (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, A–D</a>). We found that the RAL-646 (<em>opa23</em>) parent line has a high rate of ectopic macrochaetes in both males (8%) and females (23%) relative to representative controls. This effect was seen to a lesser degree in the parent RAL-237 stock, which carries the <em>opa35</em> variant of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>. In the outcrossed lines, we found that the <em>opa23</em> line, RALX646-bg1, maintains the same high rate of ectopic macrochaetes similar to the parent RAL-646 stock, whereas the <em>opa35</em> line, RALX237-bg1, exhibits an increased rate of ectopic macrochaetes that is higher than both its parent RAL stock and all of the outcrossed lines carrying wild-type <em>Notch opa</em> repeats (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). In both of the two variant outcrossed lines, the highest levels of ectopic bristles occur in females, more than three-times higher than outcrossed males, suggesting that this phenotype may be sensitive to imprecision in dosage compensation.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rare Notch opa variants manifest macrochaete patterning defects. (A) Shown is a graph of the frequency of ectopic (i.e., duplicated) bristles on the adult thorax (DCs, dorsocentral macrochaetes; SCs, scutellar macrochaetes; pPAs, posterior postalars) of male (left) and female (right) RAL stocks and outcrossed RALX-bg1 lines, carrying different opa variants at Notch. This defect is most pronounced for the RAL-646 and RALX646-bg1 lines (blue bars) carrying the rare opa23 variant, but some intensification is also seen for the RALX237-bg1 line (red bars) carrying another rare variant, opa35. Lines carrying wild-type opa31 alleles in either the parent RAL stocks or the RALX-bg1 derivative lines based on RAL-439, RAL-440, and RAL-822 (yellow bars) rarely have ectopic bristles. (B&#x2013;D) Shown are representative adult nota with ectopic bristles (arrows) in the parent RAL-646 (B) and the outcrossed RALX646-bg1 line (C), but not the RALX439-bg1 line (D). Cuticle fluff (asterisks in B) is frequently seen in the RAL-646 flies. (E) Cartoon of hemi-notum and its 13 macrochaetes, abbreviated as follows: (1&#x2013;2) upper and lower humerals (uHU, lHU); (3) presutural (PS); (4&#x2013;5) anterior and posterior notopleurals (aNP, pNP); (6&#x2013;7) anterior and posterior supra-alars (aSA, pSP); (8&#x2013;9) anterior and posterior postalars (aPA, pPA); (10&#x2013;11) anterior and posterior dorsocentrars (aDC, pDC); and (12&#x2013;13) anterior and posterior scutellars (aSC, pSC). Anterior is to the left. Posterior is to the right. Top of image is the dorsal midline. (F) Frequency (FREQ) of macrochaete occurrence in parent RAL stocks (different colored shapes) for female (circles) and male (squares) flies. (G) Frequency of macrochaete occurrence in 5&#xD7; outcrossed RALX stocks using FM7c males (bg1). Outcrossed lines are homozygous for the RAL X. The failure to form pSA is also seen in the FM7 balancer lines (data not shown). (H) Frequency of macrochaete occurrence in 5&#xD7; outcrossed RALX stocks using FM7a males (bg2). Outcrossed lines are homozygous for the RAL X. These data show that the RALX646 (blue opa23) and RALX237 (red opa35) outcrossed lines produce ectopic posterior postalar, dorsocentral, and scutellar macrochaetes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rare &lt;em&gt;Notch opa&lt;/em&gt; variants manifest macrochaete patterning defects. (A) Shown is a graph of the frequency of ectopic (&lt;em&gt;i.e.&lt;/em&gt;, duplicated) bristles on the adult thorax (DCs, dorsocentral macrochaetes; SCs, scutellar macrochaetes; pPAs, posterior postalars) of male (left) and female (right) RAL stocks and outcrossed RALX-bg1 lines, carrying different &lt;em&gt;opa&lt;/em&gt; variants at &lt;em&gt;Notch&lt;/em&gt;. This defect is most pronounced for the RAL-646 and RALX646-bg1 lines (blue bars) carrying the rare &lt;em&gt;opa23&lt;/em&gt; variant, but some intensification is also seen for the RALX237-bg1 line (red bars) carrying another rare variant, &lt;em&gt;opa35&lt;/em&gt;. Lines carrying wild-type &lt;em&gt;opa31&lt;/em&gt; alleles in either the parent RAL stocks or the RALX-bg1 derivative lines based on RAL-439, RAL-440, and RAL-822 (yellow bars) rarely have ectopic bristles. (B&#x2013;D) Shown are representative adult nota with ectopic bristles (arrows) in the parent RAL-646 (B) and the outcrossed RALX646-bg1 line (C), but not the RALX439-bg1 line (D). Cuticle fluff (asterisks in B) is frequently seen in the RAL-646 flies. (E) Cartoon of hemi-notum and its 13 macrochaetes, abbreviated as follows: (1&#x2013;2) upper and lower humerals (uHU, lHU); (3) presutural (PS); (4&#x2013;5) anterior and posterior notopleurals (aNP, pNP); (6&#x2013;7) anterior and posterior supra-alars (aSA, pSP); (8&#x2013;9) anterior and posterior postalars (aPA, pPA); (10&#x2013;11) anterior and posterior dorsocentrars (aDC, pDC); and (12&#x2013;13) anterior and posterior scutellars (aSC, pSC). Anterior is to the left. Posterior is to the right. Top of image is the dorsal midline. (F) Frequency (FREQ) of macrochaete occurrence in parent RAL stocks (different colored shapes) for female (circles) and male (squares) flies. (G) Frequency of macrochaete occurrence in 5&#xD7; outcrossed RALX stocks using &lt;em&gt;FM7c&lt;/em&gt; males (bg1). Outcrossed lines are homozygous for the RAL X. The failure to form pSA is also seen in the &lt;em&gt;FM7&lt;/em&gt; balancer lines (data not shown). (H) Frequency of macrochaete occurrence in 5&#xD7; outcrossed RALX stocks using &lt;em&gt;FM7a&lt;/em&gt; males (bg2). Outcrossed lines are homozygous for the RAL X. These data show that the RALX646 (blue &lt;em&gt;opa23&lt;/em&gt;) and RALX237 (red &lt;em&gt;opa35&lt;/em&gt;) outcrossed lines produce ectopic posterior postalar, dorsocentral, and scutellar macrochaetes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F5.medium.gif" width="350" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2405/F5.medium.gif" width="350" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467003" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-56" class="first-child">Rare <em>Notch opa</em> variants manifest macrochaete patterning defects. (A) Shown is a graph of the frequency of ectopic (<em>i.e.</em>, duplicated) bristles on the adult thorax (DCs, dorsocentral macrochaetes; SCs, scutellar macrochaetes; pPAs, posterior postalars) of male (left) and female (right) RAL stocks and outcrossed RALX-bg1 lines, carrying different <em>opa</em> variants at <em>Notch</em>. This defect is most pronounced for the RAL-646 and RALX646-bg1 lines (blue bars) carrying the rare <em>opa23</em> variant, but some intensification is also seen for the RALX237-bg1 line (red bars) carrying another rare variant, <em>opa35</em>. Lines carrying wild-type <em>opa31</em> alleles in either the parent RAL stocks or the RALX-bg1 derivative lines based on RAL-439, RAL-440, and RAL-822 (yellow bars) rarely have ectopic bristles. (B–D) Shown are representative adult nota with ectopic bristles (arrows) in the parent RAL-646 (B) and the outcrossed RALX646-bg1 line (C), but not the RALX439-bg1 line (D). Cuticle fluff (asterisks in B) is frequently seen in the RAL-646 flies. (E) Cartoon of hemi-notum and its 13 macrochaetes, abbreviated as follows: (1–2) upper and lower humerals (uHU, lHU); (3) presutural (PS); (4–5) anterior and posterior notopleurals (aNP, pNP); (6–7) anterior and posterior supra-alars (aSA, pSP); (8–9) anterior and posterior postalars (aPA, pPA); (10–11) anterior and posterior dorsocentrars (aDC, pDC); and (12–13) anterior and posterior scutellars (aSC, pSC). Anterior is to the left. Posterior is to the right. Top of image is the dorsal midline. (F) Frequency (FREQ) of macrochaete occurrence in parent RAL stocks (different colored shapes) for female (circles) and male (squares) flies. (G) Frequency of macrochaete occurrence in 5× outcrossed RALX stocks using <em>FM7c</em> males (bg1). Outcrossed lines are homozygous for the RAL X. The failure to form pSA is also seen in the <em>FM7</em> balancer lines (data not shown). (H) Frequency of macrochaete occurrence in 5× outcrossed RALX stocks using <em>FM7a</em> males (bg2). Outcrossed lines are homozygous for the RAL X. These data show that the RALX646 (blue <em>opa23</em>) and RALX237 (red <em>opa35</em>) outcrossed lines produce ectopic posterior postalar, dorsocentral, and scutellar macrochaetes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">To understand whether specific macrochaete bristles are affected by the various <em>Notch opa</em> alleles, we scored the relative presence and absence of each of the 13 macrochaete bristles of the adult hemi-notum in our RAL parent lines and outcrossed RALX-bg1 and RALX-bg2 lines (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, E–H</a>). These results show that the posterior PA, anterior and posterior DC, and the anterior SC bristles are ectopically produced in lines carrying rare, variant <em>Notch opa</em> alleles. We note a general failure to form the posterior supra-alar (pSA) macrochaete in most outcrossed RALX-bg lines regardless of <em>Notch opa</em> status. Because we see a similar effect in <em>FM7</em> flies, which we used as the background for introgressing the RAL X chromosomes, we attribute this particular bristle defect to autosomal modifiers carried in the <em>FM7</em> genetic background that likely affect the specific prepatterning signals driving proneural cluster expression at this bristle position.</p></div><div id="sec-18" class="subsection"><h3>Does a rare Notch opa variant suppress a key scute inversion?</h3><p id="p-58">Balancers are chromosomal tools that have been used in hundreds of genetic studies in <em>Drosophila</em>, including much of the work involving <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lindsley and Grell 1972</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashburner 1989</a>). In the process of genotyping the <em>N</em><sup>1</sup> and <em><a href="http://flybase.org/reports/FBal0098836.html">N<sup>TA17V</sup></a></em> stocks, which are kept over the multiply-inverted <em>FM7c</em> X balancer chromosome, we discovered that this balancer carries a rare <em>opa33b</em> allele. After sequencing many other X balancers, we found that most carry the rare <em>opa33b</em> allele while some carry <em>opa32</em> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Thus, we found that no classical X balancer chromosome carries the major <em>opa31</em> allele. This includes <em>FM1</em>, <em>FM4</em>, <em>FM6</em>, <em>FM7a</em>, <em>FM7c</em>, and <em>Basc</em>. In contrast, this <em>Notch opa33b</em> variant is rare in the RAL stocks (2/41 RAL lines genotyped, see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-59">To understand the association of <em>Notch opa33b</em> with most balancers, we first noted that all such X balancer chromosomes (<em>FM1</em>, <em>FM7a</em>, <em>FM7c</em>, and <em>Basc</em>) are linked to the <em>white apricot</em> allele <em><a href="http://flybase.org/reports/FBal0018195.html">w<sup>a</sup></a></em>, whereas those carrying <em>Notch opa32</em> (<em>FM4</em> and <em>FM6</em>) are linked to the wild-type <em><a href="http://flybase.org/reports/FBgn0003996.html">white</a></em> locus. (The latter two balancers likely swapped the <em>white–Notch</em> region during recombination events associated with creation of these hybrid X balancers.) In approximately 1929, Sidorov subjected <em><a href="http://flybase.org/reports/FBal0018195.html">w<sup>a</sup></a></em> flies to X-ray mutagenesis and generated the <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> inversion allele <em>In(1)sc<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-35.gif"/></noscript></span></span></em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sidorov 1931</a>). The <em>In(1)sc<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-36.gif"/></noscript></span></span></em> inversion was one of the founding inversions used in the “First Multiple” inversion balancers. Thus, we sequenced both <em>In(1)sc<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-37.gif"/></noscript></span></span></em> and the <em><a href="http://flybase.org/reports/FBal0018195.html">w<sup>a</sup></a></em> stock (BDGP #148) and found both carry <em>opa33b</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). To confirm the simple conclusion that the balancer <em>Notch opa</em> variant originated with the <em><a href="http://flybase.org/reports/FBal0018195.html">w<sup>a</sup></a></em> stock, we genotyped the related <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>7</sup> inversion, <em>In(1)sc</em><sup>7</sup>, which was independently produced by mutagenesis of <em><a href="http://flybase.org/reports/FBal0018195.html">w<sup>a</sup></a></em>, and found that it also carries <em>opa33b</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-60">To determine whether the <em>opa33b</em> variant of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> suppresses or enhances the <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> phenotype caused by a separation of diverse prepatterning enhancers and the SOP mother cell enhancer from <em><a href="http://flybase.org/reports/FBgn0000022.html">achaete</a></em> or <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em>, we identified an <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em> inversion allele that has the closest known breakpoint to the <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> breakpoint, <em>sc<sup>V</sup></em><sup>2</sup> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). This <em>sc<sup>V</sup></em><sup>2</sup> allele’s breakpoint is 6 kb upstream of the <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> breakpoint and separates the same set of known prepatterning and SOP enhancers present throughout the ∼105-kb-long <em>AS-C</em> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). We genotyped this inversion allele and found that it encodes a wild-type <em>opa31 Notch</em> allele (see <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). Thus, in the absence of additional modifiers and suppressors, differences in the bristle patterning defects of the <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> alleles <em>sc<sup>V</sup></em><sup>2</sup> and <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> constitute another test of the hypothesis that the precise <em>Notch opa</em> length is a functional modifier of Notch-Delta network signaling behavior as suggested by the severe shape of the allelic distribution (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>) and both the embryonic lethality (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>) and bristle defects (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>) associated with introgressed RAL X chromosomes carrying rare <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A scute inversion carrying the wild-type opa31 Notch allele manifests a severe bristle patterning defect that is not seen with an equivalent scute inversion carrying the rare expanded opa33b allele. (A) The AS-C locus and positions of the scV2 (linked to opa31) and sc8 (linked to opa33b) inversion breakpoints (light red vertical bars). Also shown are the diverse prepatterning enhancers (blue PPE boxes), which normally drive both ac and sc (blue arrows) in different proneural clusters of developing wing imaginal discs. In addition, the SOP mother cell enhancer (dark blue SE box), which drives both ac and sc, is also shown. Although each inversion is linked to a different Notch opa allele carried within the inversions, both inversions separate the same set of known enhancers. The DC PPE also coincides with the yar noncoding transcript (Soshnev et al. 2011). (B) A comparison of macrochaete patterning defects shows that scV2 has more deleterious effects than sc8, consistent with a suppressor role for opa33b expanded Notch. (C) A comparison of macrochaete defects for RAL F1 heterozygotes over FM7c, which carries the sc8 inversion allele, shows that heterozygotes with the short and long variants are severely deficient in specifying DC and pSC bristles relative to a representative wild type heterozygote. (D) RAL stocks over the sc8 inversion allele by itself (\emph{i.e.}, not in a complex balancer chromosome), shows more modest effects compared to the FM7c heterozygotes and further suggests the presence of N1 modifiers in the latter (see text)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A &lt;em&gt;scute&lt;/em&gt; inversion carrying the wild-type &lt;em&gt;opa31 Notch&lt;/em&gt; allele manifests a severe bristle patterning defect that is not seen with an equivalent &lt;em&gt;scute&lt;/em&gt; inversion carrying the rare expanded &lt;em&gt;opa33b&lt;/em&gt; allele. (A) The &lt;em&gt;AS-C&lt;/em&gt; locus and positions of the &lt;em&gt;sc&lt;sup&gt;V&lt;/sup&gt;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; (linked to &lt;em&gt;opa31&lt;/em&gt;) and &lt;em&gt;sc&lt;/em&gt;&lt;sup&gt;8&lt;/sup&gt; (linked to &lt;em&gt;opa33b&lt;/em&gt;) inversion breakpoints (light red vertical bars). Also shown are the diverse prepatterning enhancers (blue PPE boxes), which normally drive both &lt;em&gt;ac&lt;/em&gt; and &lt;em&gt;sc&lt;/em&gt; (blue arrows) in different proneural clusters of developing wing imaginal discs. In addition, the SOP mother cell enhancer (dark blue SE box), which drives both &lt;em&gt;ac&lt;/em&gt; and &lt;em&gt;sc&lt;/em&gt;, is also shown. Although each inversion is linked to a different &lt;em&gt;Notch opa&lt;/em&gt; allele carried within the inversions, both inversions separate the same set of known enhancers. The DC PPE also coincides with the &lt;em&gt;yar&lt;/em&gt; noncoding transcript (Soshnev &lt;em&gt;et al.&lt;/em&gt; 2011). (B) A comparison of macrochaete patterning defects shows that &lt;em&gt;sc&lt;sup&gt;V&lt;/sup&gt;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; has more deleterious effects than &lt;em&gt;sc&lt;/em&gt;&lt;sup&gt;8&lt;/sup&gt;, consistent with a suppressor role for &lt;em&gt;opa33b&lt;/em&gt; expanded &lt;em&gt;Notch&lt;/em&gt;. (C) A comparison of macrochaete defects for RAL F1 heterozygotes over &lt;em&gt;FM7c&lt;/em&gt;, which carries the &lt;em&gt;sc&lt;/em&gt;&lt;sup&gt;8&lt;/sup&gt; inversion allele, shows that heterozygotes with the short and long variants are severely deficient in specifying DC and pSC bristles relative to a representative wild type heterozygote. (D) RAL stocks over the &lt;em&gt;sc&lt;/em&gt;&lt;sup&gt;8&lt;/sup&gt; inversion allele by itself (\emph{i.e.}, not in a complex balancer chromosome), shows more modest effects compared to the &lt;em&gt;FM7c&lt;/em&gt; heterozygotes and further suggests the presence of &lt;em&gt;N&lt;/em&gt;&lt;sup&gt;&lt;em&gt;1&lt;/em&gt;&lt;/sup&gt; modifiers in the latter (see text).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F6.medium.gif" width="313" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/11/2405/F6.medium.gif" width="313" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466993" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-61" class="first-child">A <em>scute</em> inversion carrying the wild-type <em>opa31 Notch</em> allele manifests a severe bristle patterning defect that is not seen with an equivalent <em>scute</em> inversion carrying the rare expanded <em>opa33b</em> allele. (A) The <em>AS-C</em> locus and positions of the <em>sc<sup>V</sup></em><sup>2</sup> (linked to <em>opa31</em>) and <em>sc</em><sup>8</sup> (linked to <em>opa33b</em>) inversion breakpoints (light red vertical bars). Also shown are the diverse prepatterning enhancers (blue PPE boxes), which normally drive both <em>ac</em> and <em>sc</em> (blue arrows) in different proneural clusters of developing wing imaginal discs. In addition, the SOP mother cell enhancer (dark blue SE box), which drives both <em>ac</em> and <em>sc</em>, is also shown. Although each inversion is linked to a different <em>Notch opa</em> allele carried within the inversions, both inversions separate the same set of known enhancers. The DC PPE also coincides with the <em>yar</em> noncoding transcript (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Soshnev <em>et al.</em> 2011</a>). (B) A comparison of macrochaete patterning defects shows that <em>sc<sup>V</sup></em><sup>2</sup> has more deleterious effects than <em>sc</em><sup>8</sup>, consistent with a suppressor role for <em>opa33b</em> expanded <em>Notch</em>. (C) A comparison of macrochaete defects for RAL F1 heterozygotes over <em>FM7c</em>, which carries the <em>sc</em><sup>8</sup> inversion allele, shows that heterozygotes with the short and long variants are severely deficient in specifying DC and pSC bristles relative to a representative wild type heterozygote. (D) RAL stocks over the <em>sc</em><sup>8</sup> inversion allele by itself (\emph{i.e.}, not in a complex balancer chromosome), shows more modest effects compared to the <em>FM7c</em> heterozygotes and further suggests the presence of <em>N</em><sup><em>1</em></sup> modifiers in the latter (see text).</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">We found that the <em>sc<sup>V</sup></em><sup>2</sup> line, which is linked to <em>Notch opa31</em>, has a more severe and penetrant bristle phenotype compared to the <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> inversion, which is linked to <em>Notch opa33b</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). For example, <em>sc<sup>V</sup></em><sup>2</sup> males have extremely high rates of ectopic DCs while <em>sc<sup>V</sup></em><sup>2</sup> females have high rates of DC patterning defects. Empty sockets are not observed in females, suggesting that the defect is in underspecification of sensory organ precursors in females and overspecification in males. In stark contrast, <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em><sup>8</sup> males and females have almost normal DC bristle patterning. This result is consistent with a role for the <em>opa33b</em> allele of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> in partially suppressing <em>AS-C</em> insufficiency caused by the inversion’s localization of enhancers away from <em><a href="http://flybase.org/reports/FBgn0000022.html">ac</a></em> or <em><a href="http://flybase.org/reports/FBgn0004170.html">sc</a></em>.</p><p id="p-63">We also conducted two related experiments that compare the bristle phenotypes in heterozygous females carrying either the <em>In(1)sc</em><sup>8</sup> or <em>FM7c</em> balancers and RAL X chromosomes carrying different <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants. Genome-wide modifiers of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> variants, including other pQ-variable loci, may have been acquired in stocks carrying balanced <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> mutants. Thus, we crossed our RALX-bg1 lines to <em>FM7c</em>/<em>N</em><sup>1</sup> flies and scored the heterozygous balancer females (RALX/<em>FM7c</em>). We found that the <em>opa23</em> and <em>opa35</em> heterozygotes are similarly severely deficient in the formation of DC and posterior SC macrochaetes relative to <em>opa31</em> heterozygotes (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). At several other bristle positions, the <em>opa35</em> (lHU, pSA, pPA, aSC) and <em>opa23</em> (aNP) variants are modestly affected relative to <em>opa31</em> heterozygotes. For comparison we conducted similar experiments with RALX/<em>In(1)sc</em><sup>8</sup> heterozygotes, which would be deficient in the postulated suppressors of the <em>Notch<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-38.gif"/></noscript></span></span></em> phenotype. We found that the <em>opa23</em> heterozygotes do not show a severe bristle phenotype in this “naive” background. This suggests to us that the <em>N</em><sup>1</sup>/<em>FM7c</em> balanced stock has likely acquired suppressor modifiers of the <em>N</em><sup>1</sup> phenotype.</p><p id="p-64">We hypothesized that some classic <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> mutants may have acquired suppressors via changes in the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats after decades of maintenance at stock centers. The slightly expanded <em>opa32</em> version of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> was previously reported for the classic alleles <em><a href="http://flybase.org/reports/FBal0012759.html">N<sup>Co</sup></a></em> and <em>N</em><sup>60</sup><em><sup>g</sup></em><sup>11</sup>, but this extra glutamine is thought to be phenotypically silent (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kidd <em>et al.</em> 1986</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Lyman and Young 1993</a>). To determine whether <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> opa repeats are coevolving with the primary mutations of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> mutant alleles, we genotyped several additional classical <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> mutant stocks. Remarkably, our survey of <em>N</em><sup>1</sup>, <em>N<sup>ts</sup></em><sup>1</sup>, <em>N<sup>ts</sup></em><sup>2</sup>, <em>N<sup>nd</sup></em><sup>0</sup>, <em>N<sup>TA</sup></em><sup>17</sup><em><sup>V</sup></em>, and <em>N<sup>Ax-ts</sup></em><sup>1</sup> did not turn up a single length variant different from <em>opa31</em> (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) despite the number of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants we had found by surveying RAL stocks. This result suggests to us that perhaps changes in the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat number by contraction or expansion are generally deleterious in combination with other lesions at this locus.</p></div><div id="sec-19" class="subsection"><h3>The Notch opa23 locus gives bristle defects and notched wings</h3><p id="p-65">To rule out phenotypic effects caused by X-linked loci other than <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>, we recombined the rare and interesting <em>Notch opa23</em> allele out of its RAL-646 X chromosomal background (see <em>Materials and Methods</em>). We used the parent RAL-646 stock, <em>w</em><sup>1118</sup> fly stock and <em>FM7</em> balancer stocks to produce a white-eyed “wopa23” line, carrying a much smaller portion of the RAL-646 X chromosome spanning from <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> to some crossover point between <em><a href="http://flybase.org/reports/FBgn0000524.html">dx</a></em> and <em>ABCF2</em> (<em><a href="http://flybase.org/reports/FBgn0030672.html">CG9281</a></em>) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). We scored macrochaete phenotypes for the wopa23 flies and compared them to our series of outcrossed <em>opa23</em> lines (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). This showed that male wopa23 flies have missing presutural macrochaetes approximately 5% of the time, which is a defect not seen in RAL-646, RALX646-bg1, RALX646-bg2, or <em>w</em><sup>1118</sup> flies of either sex. Elevated levels of ectopic aDC bristles are seen in the wopa23 flies, but these are comparable to the <em>opa23</em> parent and outcrossed lines. We also found that our wopa23 male and female flies displayed the eponymous <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> phenotype of notched and nicked wing tips (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7, C–F</a>, and vertical arrows). This suggests that there were X-linked suppressors of <em>Notch opa23</em>, which were recombined away in this line. A smaller number of flies had a wing vein patterning defect in which the L5 vein fails to reach the wing margin (horizontal arrows in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7D</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The X chromosomal region carrying the rare Notch opa23 variant is associated with classic Notch phenotypes. (A) Shown is the region from chromosome X containing Notch. These genes include: yellow (y), the achaete scute complex (AS-C), Hr4, white (w), Notch (N), Deltex (dx), Sex lethal (Sxl), and ABCF2 (CG9281). Below the chromosomal graphic are the various polymorphisms distinguishing RAL-646 from other RAL stocks or RAL-646-derivative lines, including the wopa23 line in which we introgressed the region of the Notch locus (blue shaded area) and some of the nearby flanking regions (exact breakpoints of fragment lie within the yellow&#x2013;green shaded areas) into control non-RAL backgrounds. (B) Shown are the macrochaete frequencies for RAL-646, RALX646-bg, and wopa23 line in the same convention as previous figures. (C) Shown is a graph depicting the frequency of wing notching in various lines. The original wopa23 generation (wopa23-G0) produced notched and/or nicked wings in males and females at a modest but not insignificant rate. However, following six generations of selection for notched wing flies (wopa23-G6), this rate increased dramatically to 72% in males and 78% in females. In contrast, selection for notched wings in the N1/FM7c stock failed to increase the rate of notching past 53% even after 10 generations of selection. (D) Shown are a pair of wings from a wopa23-G0 female prior to selection for notched wings. Vertical arrow shows a classic wing notch. Horizontal arrows in each wing show an additional defect of incomplete L5 wing vein formation, albeit at a lower frequency than notched wings. (E, F) Wings from female (E) and male (F) wopa23 flies after six generations of propagating only flies with notched wings. These flies have more deeply serrated notches than the unselected wopa23 stock, but the incomplete L5 wing vein phenotype disappears." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2054270333" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The X chromosomal region carrying the rare &lt;em&gt;Notch opa23&lt;/em&gt; variant is associated with classic Notch phenotypes. (A) Shown is the region from chromosome X containing &lt;em&gt;Notch&lt;/em&gt;. These genes include: &lt;em&gt;yellow&lt;/em&gt; (&lt;em&gt;y&lt;/em&gt;), the &lt;em&gt;achaete scute&lt;/em&gt; complex (&lt;em&gt;AS-C&lt;/em&gt;), &lt;em&gt;Hr4&lt;/em&gt;, &lt;em&gt;white&lt;/em&gt; (&lt;em&gt;w&lt;/em&gt;), &lt;em&gt;Notch&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt;), &lt;em&gt;Deltex&lt;/em&gt; (&lt;em&gt;dx&lt;/em&gt;), &lt;em&gt;Sex lethal&lt;/em&gt; (&lt;em&gt;Sxl&lt;/em&gt;), and &lt;em&gt;ABCF2&lt;/em&gt; (&lt;em&gt;CG9281&lt;/em&gt;). Below the chromosomal graphic are the various polymorphisms distinguishing RAL-646 from other RAL stocks or RAL-646-derivative lines, including the wopa23 line in which we introgressed the region of the &lt;em&gt;Notch&lt;/em&gt; locus (blue shaded area) and some of the nearby flanking regions (exact breakpoints of fragment lie within the yellow&#x2013;green shaded areas) into control non-RAL backgrounds. (B) Shown are the macrochaete frequencies for RAL-646, RALX646-bg, and wopa23 line in the same convention as previous figures. (C) Shown is a graph depicting the frequency of wing notching in various lines. The original wopa23 generation (wopa23-G&lt;sub&gt;0&lt;/sub&gt;) produced notched and/or nicked wings in males and females at a modest but not insignificant rate. However, following six generations of selection for notched wing flies (wopa23-G&lt;sub&gt;6&lt;/sub&gt;), this rate increased dramatically to 72% in males and 78% in females. In contrast, selection for notched wings in the &lt;em&gt;N&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt;/&lt;em&gt;FM7c&lt;/em&gt; stock failed to increase the rate of notching past 53% even after 10 generations of selection. (D) Shown are a pair of wings from a &lt;em&gt;wopa23&lt;/em&gt;-G&lt;sub&gt;0&lt;/sub&gt; female prior to selection for notched wings. Vertical arrow shows a classic wing notch. Horizontal arrows in each wing show an additional defect of incomplete L5 wing vein formation, albeit at a lower frequency than notched wings. (E, F) Wings from female (E) and male (F) wopa23 flies after six generations of propagating only flies with notched wings. These flies have more deeply serrated notches than the unselected wopa23 stock, but the incomplete L5 wing vein phenotype disappears.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2405/F7.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/11/2405/F7.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2405/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466893" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-66" class="first-child">The X chromosomal region carrying the rare <em>Notch opa23</em> variant is associated with classic Notch phenotypes. (A) Shown is the region from chromosome X containing <em>Notch</em>. These genes include: <em>yellow</em> (<em>y</em>), the <em>achaete scute</em> complex (<em>AS-C</em>), <em>Hr4</em>, <em>white</em> (<em>w</em>), <em>Notch</em> (<em>N</em>), <em>Deltex</em> (<em>dx</em>), <em>Sex lethal</em> (<em>Sxl</em>), and <em>ABCF2</em> (<em>CG9281</em>). Below the chromosomal graphic are the various polymorphisms distinguishing RAL-646 from other RAL stocks or RAL-646-derivative lines, including the wopa23 line in which we introgressed the region of the <em>Notch</em> locus (blue shaded area) and some of the nearby flanking regions (exact breakpoints of fragment lie within the yellow–green shaded areas) into control non-RAL backgrounds. (B) Shown are the macrochaete frequencies for RAL-646, RALX646-bg, and wopa23 line in the same convention as previous figures. (C) Shown is a graph depicting the frequency of wing notching in various lines. The original wopa23 generation (wopa23-G<sub>0</sub>) produced notched and/or nicked wings in males and females at a modest but not insignificant rate. However, following six generations of selection for notched wing flies (wopa23-G<sub>6</sub>), this rate increased dramatically to 72% in males and 78% in females. In contrast, selection for notched wings in the <em>N</em><sup>1</sup>/<em>FM7c</em> stock failed to increase the rate of notching past 53% even after 10 generations of selection. (D) Shown are a pair of wings from a <em>wopa23</em>-G<sub>0</sub> female prior to selection for notched wings. Vertical arrow shows a classic wing notch. Horizontal arrows in each wing show an additional defect of incomplete L5 wing vein formation, albeit at a lower frequency than notched wings. (E, F) Wings from female (E) and male (F) wopa23 flies after six generations of propagating only flies with notched wings. These flies have more deeply serrated notches than the unselected wopa23 stock, but the incomplete L5 wing vein phenotype disappears.</p><div class="sb-div caption-clear"></div></div></div><p id="p-67">To understand the notched wing phenotype of the wopa23 flies better, we put various RALX-bg1 X chromosomes over an X chromosome carrying the dominant <em>N<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2405/embed/mml-math-39.gif"/></noscript></span></span></em> allele, which produces notched wings at a certain frequency. We then scored the frequencies of notched and/or nicked wing tips in the F<sub>1</sub> heterozygous nonbalancer females. Our <em>N</em><sup>1</sup> cross results show that the <em>opa23</em> and <em>opa35</em> alleles have the smallest suppression effect on notched wings in heterozygous F<sub>1</sub> females. This suggests that both contractions and expansions of the wild-type <em>opa31</em> repeat length are types of lesions at the <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> locus.</p><p id="p-68">We also performed an experiment on the <em>N</em><sup>1</sup>/<em>FM7c</em> and wopa23 flies in which we selected for notched wings over several generations. While we increased the number of notched wings in <em>N</em><sup>1</sup> females modestly from 34 to 50% after one generation of selection (<em>N</em><sup>1</sup> is lethal in males), we were unable to achieve further increases even after 10 generations of selection (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7C</a>). In contrast, we substantially increased the rate of notched wings in both males and females from &lt;10% to &gt;70% in our wopa23 stock after only a few generations of propagating only flies with notched wings (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7C</a>). These “Super-Notch” wopa23-G<sub>6</sub> flies lose the L5 wing vein phenotype that we first saw with the wopa23-G<sub>0</sub> generation, even while the severity of wing notching was increasing (compare <a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7D</a> to <a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7, E and F</a>). We also regenotyped the Kodani inversion in the wopa23-G<sub>0</sub> and Super-Notch wopa23-G<sub>6</sub> lines and found that both still carried at least one copy of the <em><a href="http://flybase.org/reports/FBab0005595.html">In(3R)K</a></em> inversion despite the three generations of outcrossing involved in the recombination screen (<a href="http://www.g3journal.org/content/suppl/2015/09/10/g3.115.021659.DC1/FileS1.pdf">File S1</a>).</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-69">We discovered a plethora of functional <em>Notch opa</em> variants by sampling the Raleigh lines (DGRP) and genotyping multiple independent clones from each line by Sanger sequencing. We found that almost half of the RAL haplotypes (19/41) do not carry the standard <em>opa31</em> variant and are not being accurately genotyped by high-throughput sequencing and assembly. Importantly, alleles characterized by extremely short or long <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats have common developmental defects, including classic <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> phenotypes affecting macrochaete patterning and wing notching. These phenotypes intensify when their X chromosomes or regions of the X chromosome containing <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> are outcrossed and/or recombined out into other backgrounds. The outcrossing effect suggests that these particular isogenic RAL lines carry both linked and unlinked suppressor modifiers. It will be interesting to determine the extent to which the 205 DGRP isofemale lines, established after 20 generations of inbreeding initially 1500 Raleigh females, are enriched for compatible alleles (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">David <em>et al.</em> 2005</a>).</p><p id="p-70">The Line-646 DGRP mis-assembly at <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> is an example of the problematic nature of genotyping the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequences and of the repeat assembly problem in general. This problem is exacerbated by the need for flanking nonrepetitive “anchor” sequence to accurately start or end the repeat alignments in the correct position, as this further reduces the remaining read length that can span into the repeats. For the <em>Notch opa</em> repeats, a single 75 nucleotide (nt) read cannot span the wild-type length of 93 bp even without anchor sequence. From what we have seen, 91% of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> length variants (10 of 11 observed variant length classes) have a greater number of repeats than wild-type. If a minimum of two flanking non-CAX codons are used for anchoring sequence, this leaves only 23 repeat CAX codons in a 75-nt read (6 + 69). Thus, even the shorter RAL-646 <em>opa23</em> repeats cannot be spanned by a single 75-nt read. In summary, we found that the important <em>Notch opa</em> repeats are best sequenced and assembled using sequence reads &gt;140 bp long (3 bp × 40 triplet repeats + 10 bp × 2 anchor sequences).</p><p id="p-71">Tautz reported hypervariability for the <em>Notch opa</em> repeats of <em>D. melanogaster</em> after seeing four length variants from 11 independent isogenic female lines established from North and South America, Europe, Asia, and Australia (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Tautz 1989</a>). These four alleles encode the pQ range of Q<sub>13</sub>HQ<sub>17-20</sub> and most commonly correspond to wild-type <em>opa31</em> and <em>opa32</em> (81.8%). Two rare variant haplotypes corresponded to an <em>opa33c</em> encoding Q<sub>13</sub>HQ<sub>19</sub>, and an <em>opa34b</em> encoding Q<sub>13</sub>HQ<sub>20</sub> (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a>). If we consider just the three North American isofemale lines genotyped by Tautz, then these give a 2:1 ratio of <em>opa31</em> to <em>opa32</em>, which very roughly approximates the 3:1 ratio we obtained from genotyping the isofemale Raleigh lines of the DGRP (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). However, this ratio falls to 0.8:1 when all the worldwide isofemale lines are considered. The single isofemale line from Japan was homozygous for <em>opa34b</em>, whereas the single isofemale line from the USSR was heterozygous for <em>opa33c</em>/<em>opa31</em>. Both of these variants have not yet been genotyped by us in the resequenced DGRP lines. Thus, it will be interesting to see if different <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> alleles are found in different world populations.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison with worldwide distribution of <em>Notch opa</em> repeat variants</span><div class="sb-div caption-clear"></div></div></div><p id="p-74">Among the new <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants, we discovered the rare short <em>opa23</em> allele that causes embryonic lethality in many backgrounds. Of all the rare <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> alleles that we discovered by sampling DGRP RAL lines and other stocks, the <em>opa23</em> allele has the greatest length change relative to the wild-type <em>opa31</em>. It is also the only <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> allele not encoding the histidine residue internal to the pQ tract. Both the parent RAL-646 and RALX646 introgressed lines, bg1 and bg2, correspond to the stocks and lines with the greatest bristle patterning defects, a classic <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> phenotype. This chromosomal region around the <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> locus is also responsible for producing notched wings when recombined into different backgrounds, presumably because it loses X-linked suppressor modifiers of <em>Notch opa23</em>.</p><p id="p-75">The distribution of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> length variants suggests extreme purifying selection against the shorter would-be alleles in the barrens range (<em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> 24–30), and somewhat more moderate purifying selection against the many different longer variants in the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> allelic jungle range (<em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> 31–38). Given the <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>-like phenotypes associated with extreme variants in the skewed distribution of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> genotypes (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>), the allelic barrens is unlikely to be an artifact of poor sampling. We propose that the existence of <em>opa23</em> is supportive of the reality of the barrens because it is missing the codon for histidine. On the far short side of the barrens, <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> alleles may become semiviable provided the interrupting histidine is missing. In the presence of an intervening histidine residue, the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em>-encoded pQ peptide may behave differently, as seen in other contexts (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Sharma <em>et al.</em> 1999</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Sen <em>et al.</em> 2003</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kim 2013</a>). In this regard it is interesting that the longest <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variant we have observed also has a contiguous stretch of 23 Qs. Thus, 23 Qs are the longest uninterrupted tracts we have observed in either short or long <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> variants. Why this should be so must be specific to the roles played by NICD.</p><p id="p-76">Perutz explained the 40 glutamine pathological threshold via the amyloid structure that is able to form from two stacked 20-residue-long rings (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Perutz <em>et al.</em> 2002</a>). In this cross-β amyloid nanotube, each glutamine side group forms a hydrogen bond with another side chain in the adjacent ring. These side-chain hydrogen bonds are in addition to the ones along the peptide backbone, typical of all β-sheet structures. However, the observed <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat distribution also suggests possible lethality of expansions past Q<sub>23</sub>. Because this is significantly below the pathological limit, we think either of two possible explanations will eventually be found to apply. First, a long pQ tract with only a single intervening histidine might behave similarly to an uninterrupted tract in some respects, such that a new Perutz amyloid threshold exists near 38–40 residues. Second, some other nonamyloid structure, possibly an interdigitated β-sheet interaction between NICD and one or more other interacting proteins, exists. These interactions may be too strong when the tract is abnormally expanded. Similarly, the allelic barrens might represent the lethality of contracting the pQ tracts below Q<sub>13</sub> (left-side) or Q<sub>17</sub> (right-side) if doing so impacts specific critical interactions.</p><p id="p-77">We find only three synonymous A/G (puRine) substitutions in all of the known <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> sequences (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). These three synonymous polymorphisms occur at specific positions within and near the long (CAG)<sub>7</sub> triplet repeats (an unspecified number of the 11 Tautz sequences have the *2 polymorphism as it occurs in that consensus). The location of these synonymous substitutions is consistent with two empirical observations. First, indel mutations scale as a function of repeat number for diverse repeat unit lengths and sequences (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ananda <em>et al.</em> 2013</a>). Second, nucleotide substitution rates tend to be elevated around indel polymorphisms (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ellegren 2000</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schlötterer 2000</a>). At a superficial glance, the occurrence of these substitutions across the diversity of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> genotypes (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) suggests they are independently recurrent mutations. Alternatively, <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> gene conversion events may be common at this locus, in which case it would be difficult to produce simple relationships between these alleles.</p><p id="p-78">Why do we find rare variant alleles like <em>N<sup>opa23</sup></em> if they are deleterious in many backgrounds? Some of the microchaete defective alleles of <em>Notch</em> (<em>N<sup>Mcd</sup></em>) encode C-terminal truncations of NICD that still have the N-terminal ankyrin repeats as well as both nuclear localization signals but are missing the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em>-encoded pQ tract (see <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ramain <em>et al.</em> 2001</a>). Such <em>N<sup>Mcd</sup></em> alleles are pupal lethal, suggesting an important role for the disordered C-terminal half of NICD and its pQ tract. This is consistent with our results, which suggest that contractions and deletions that reduce or eliminate the <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats are also lethal. Thus, one possible answer is that variant <em>Notch opa</em> alleles are continuously being generated <em>de novo</em> because of intrinsic instability associated with repeat sequences in general (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ananda <em>et al.</em> 2013</a>). These variants are likely subject to intense selection because there are at least three different ways in which <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat variability can affect Notch function. First, <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat variability may affect the absolute levels of Notch protein due to increased degradation during cotranslational membrane targeting, processing, or in the mature full-length membrane bound or cleaved NICD forms (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4D</a>). Second, <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat variability may affect the strength of interactions NICD makes with other cofactors in the nucleus as discussed above. Indeed, the Notch coactivator Mastermind, which is recruited by NICD and required for its activity (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wu <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fryer <em>et al.</em> 2002</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kovall 2007</a>), has several pQ tracts that are under substantial selective constraint (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Newfeld <em>et al.</em> 1991</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">1993</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">1994</a>). Third, <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat variability may affect the ability of NICD to assume the distinct conformational shapes that allow pQ-dependent self-association or interaction with the Su(H) transcription factor (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kelly <em>et al.</em> 2007</a>). In summary, the distribution of <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> length variants of <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> likely represents a snapshot summary of the ongoing selection acting on these unstable but functionally necessary <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeats and the precise <em><a href="http://flybase.org/reports/FBgn0003002.html">opa</a></em> repeat number.</p><p id="p-79">Interestingly, we found that almost twice (1.9×) as many of the sampled radiation-resistant RAL lines (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Vaisnav <em>et al.</em> 2014</a>) encoded non-<em>opa31 Notch</em> variants compared to the sampled radiation-sensitive RAL lines (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). This suggests to us that these lines may feature proteostatic stress in the endoplasmic reticulum (ER), where the Notch membrane receptor is cotranslated with membrane integration (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fortini 2009</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Yamamoto <em>et al.</em> 2010</a>). Similarly, upregulation of the unfolded protein response (UPR), which is induced by misfolded proteins in the ER, is known to confer treatment resistance to some cancers (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nagelkerke <em>et al.</em> 2014</a>). Enrichment for Notch pQ variants in radiation-resistant lines also suggests that <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> is one of only a few such loci that are both variable and capable of causing systemic stress. If so, then it will be interesting to identify the mechanisms for radiation-resistance in the few such <em>opa31</em> lines that we have genotyped.</p><p id="p-80">We conclude by discussing whether there is a role for a tunable <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> locus in the context of developmental canalization, particularly in light of distinct bristle patterning phenotypes by equivalent <em><a href="http://flybase.org/reports/FBgn0004170.html">scute</a></em> inversions carrying different <em>Notch opa</em> variants (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). The induction of proneural clusters for specific macrochaetes involves distinct pathways targeting a number of independent “prepatterning” enhancers (PPEs) at the <em>achaete scute</em> complex (<em>AS-C</em>) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Martínez and Modolell 1991</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Skeath <em>et al.</em> 1992</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gómez-Skarmeta <em>et al.</em> 1995</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Modolell and Campuzano 1998</a>). Importantly, mutations in genes for Notch-Delta signaling components can affect specific bristles, and indeed sensory bristle-specific regulatory modules and the <em>AS-C</em> gene copy number are dynamically evolving (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Galant <em>et al.</em> 1998</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Negre and Simpson 2009</a>). In addition, proneural cells transition from experiencing diverse prepatterning signals at different presumptive bristle positions to the same lateral inhibition regulatory network featuring Notch-Delta signaling. As the 26 symmetric macrochaete sensory organs on the adult notum are also monomorphic traits, these circuits must also function in the presence and absence of male dosage compensation of X-linked genes, including the <em>AS-C</em> and <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> genes. Thus, Notch-Delta signaling must become highly canalized to build the stereotypical macrochaete-type sensory organ at several positions receiving distinct signaling cues and operating at slightly different dosage levels. Some preliminary work has already been done to model parameter space across different mechanistic contexts (<em>e.g.</em>, lateral inhibition, boundary formation, asymmetric cell fate specification) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Matsuno <em>et al.</em> 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barad <em>et al.</em> 2010</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2011</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sprinzak <em>et al.</em> 2011</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shaya and Sprinzak 2011</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shimizu <em>et al.</em> 2014</a>). In this context, it will be interesting to determine in future studies whether there are functional associations between distinct <em>Notch opa variants</em> and other pQ-encoding loci throughout the genome. If this is found to be the case, then the <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> locus could harbor an important evolvable repeat variable that influences the signaling characteristics of Notch-Delta regulated circuits.</p></div><div class="section" id="sec-21"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-81">We thank Pamela Geyer and Josep Comeron for helpful consultations. This work was funded in part by an NSF CAREER award (to A.E.; IOS:1239673) and a 2014 Evelyn Hart Watson research fellowship (to C.R.). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-14"><p id="p-82">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021659/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021659/-/DC1</a></p></li><li class="fn" id="fn-15"><p id="p-83"><em>Communicating editor: K. S. McKim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-15"><span class="received-label">Received </span>June 15, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-08"><span class="accepted-label">Accepted </span>September 8, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Rice <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.1" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The genome sequence of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>–<span class="cit-lpage">2195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.2" data-doi="10.1093/gbe/evs116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ananda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasilnikova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Distinct mutational behaviors differentiate short tandem repeats from microsatellites in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs116%26rft_id%253Dinfo%253Apmid%252F23241442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjUvMy82MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.3" data-doi="10.1038/ng0893-398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrew</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telenius</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theilmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The relationship between trinucleotide (CAG) repeat length and clinical features of Huntington’s disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndrew%26rft.auinit1%253DS.%2BE.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D398%26rft.epage%253D403%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Btrinucleotide%2B%2528CAG%2529%2Brepeat%2Blength%2Band%2Bclinical%2Bfeatures%2Bof%2BHuntington%2527s%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0893-398%26rft_id%253Dinfo%253Apmid%252F8401589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0893-398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8401589&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LQ17800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2405.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <em>Drosophila: A Laboratory Handbook</em>. Cold Spring Harbor Laboratory Press</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.5" data-doi="10.1101/gad.9.21.2609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posakony</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Suppressor of Hairless directly activates transcription of Enhancer of split complex genes in response to Notch receptor activity.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2609</span>–<span class="cit-lpage">2622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBailey%26rft.auinit1%253DA%2BM%26rft.volume%253D9%26rft.issue%253D21%26rft.spage%253D2609%26rft.epage%253D2622%26rft.atitle%253DSuppressor%2Bof%2Bhairless%2Bdirectly%2Bactivates%2Btranscription%2Bof%2Benhancer%2Bof%2Bsplit%2Bcomplex%2Bgenes%2Bin%2Bresponse%2Bto%2BNotch%2Breceptor%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.21.2609%26rft_id%253Dinfo%253Apmid%252F7590239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8yMS8yNjA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.6" data-doi="10.1016/j.ceb.2011.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barad</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornstein</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Robust selection of sensory organ precursors by the Notch-Delta pathway.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DBarad%26rft.auinit1%253DO.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D663%26rft.epage%253D667%26rft.atitle%253DRobust%2Bselection%2Bof%2Bsensory%2Borgan%2Bprecursors%2Bby%2Bthe%2BNotch-Delta%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2011.09.005%26rft_id%253Dinfo%253Apmid%252F21963301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2011.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21963301&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.7" data-doi="10.1126/scisignal.2000857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barad</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornstein</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Error minimization in lateral inhibition circuits.</span> <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">ra51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DBarad%26rft.auinit1%253DO.%26rft.volume%253D3%26rft.issue%253D129%26rft.spage%253Dra51%26rft.epage%253Dra51%26rft.atitle%253DError%2BMinimization%2Bin%2BLateral%2BInhibition%2BCircuits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.2000857%26rft_id%253Dinfo%253Apmid%252F20606215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTA6IjMvMTI5L3JhNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.8" data-doi="10.1093/hmg/1.4.255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biancalana</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serville</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pommier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julien</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Moderate instability of the trinucleotide repeat in spino bulbar muscular atrophy.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DBiancalana%26rft.auinit1%253DV.%26rft.volume%253D1%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DModerate%2Binstability%2Bof%2Bthe%2Btrinucleotide%2Brepeat%2Bin%2Bspino%2Bbulbar%2Bmuscular%2Batrophy%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F1.4.255%26rft_id%253Dinfo%253Apmid%252F1303195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjc6IjEvNC8yNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.9" data-doi="10.1128/MCB.11.2.1161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bricmont</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The <em>DAL81</em> gene product is required for induced expression of two differently regulated nitrogen catabolic genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBricmont%26rft.auinit1%253DP%2BA%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D1161%26rft.epage%253D1166%26rft.atitle%253DThe%2BDAL81%2Bgene%2Bproduct%2Bis%2Brequired%2Bfor%2Binduced%2Bexpression%2Bof%2Btwo%2Bdifferently%2Bregulated%2Bnitrogen%2Bcatabolic%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.2.1161%26rft_id%253Dinfo%253Apmid%252F1990272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzIvMTE2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.10" data-doi="10.1371/journal.pone.0101177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brittain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stroebele</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microsatellite repeat instability fuels evolution of embryonic enhancers in Hawaiian Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101177%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101177%26rft_id%253Dinfo%253Apmid%252F24978198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24978198&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brizuela</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elfring</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamkun</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennison</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic analysis of the brahma gene of <em>Drosophila melanogaster</em> and polytene chromosome subdivisions 72AB.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrizuela%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D137%26rft.issue%253D3%26rft.spage%253D803%26rft.epage%253D813%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Bbrahma%2Bgene%2Bof%2BDrosophila%2Bmelanogaster%2Band%2Bpolytene%2Bchromosome%2Bsubdivisions%2B72AB.%26rft_id%253Dinfo%253Apmid%252F7916308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzMvODAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.12" data-doi="10.1016/0042-6822(91)90671-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guarino</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Molecular analysis of a baculovirus regulatory gene.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DCarson%26rft.auinit1%253DD.%2BD.%26rft.volume%253D182%26rft.issue%253D1%26rft.spage%253D279%26rft.epage%253D286%26rft.atitle%253DMolecular%2Banalysis%2Bof%2Ba%2Bbaculovirus%2Bregulatory%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252891%252990671-W%26rft_id%253Dinfo%253Apmid%252F2024466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(91)90671-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2024466&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.13" data-doi="10.1021/bi011772q"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berthelier</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, B. O’Nuallain, and R. Wetzel, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Amyloid-like features of polyglutamine aggregates and their assembly kinetics.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">7391</span>–<span class="cit-lpage">7399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D41%26rft.spage%253D7391%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi011772q%26rft_id%253Dinfo%253Apmid%252F12044172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi011772q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12044172&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176057300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.14" data-doi="10.1371/journal.pgen.1002905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnappan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The many landscapes of recombination in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1002905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253D1002905%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.15" data-doi="10.1371/journal.pgen.1003056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of inversion polymorphisms in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1003056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253D1003056%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.16" data-doi="10.1016/0092-8674(88)90144-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Courey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tjian</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Analysis of Sp1 in vivo reveals multiple transcriptional domains, including a novel glutamine-rich activation motif.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCourey%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D55%26rft.issue%253D5%26rft.spage%253D887%26rft.epage%253D898%26rft.atitle%253DAnalysis%2Bof%2BSp1%2Bin%2Bvivo%2Breveals%2Bmultiple%2Btranscriptional%2Bdomains%252C%2Bincluding%2Ba%2Bnovel%2Bglutamine-rich%2Bactivation%2Bmotif.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252888%252990144-4%26rft_id%253Dinfo%253Apmid%252F3142690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(88)90144-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3142690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R318500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.17" data-doi="10.1016/0092-8674(89)90606-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Courey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtzman</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tjian</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Synergistic activation by the glutamine-rich domains of human transcription factor Sp1.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCourey%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D59%26rft.issue%253D5%26rft.spage%253D827%26rft.epage%253D836%26rft.atitle%253DSynergistic%2Bactivation%2Bby%2Bthe%2Bglutamine-rich%2Bdomains%2Bof%2Bhuman%2Btranscription%2Bfactor%2BSp1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990606-5%26rft_id%253Dinfo%253Apmid%252F2512012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90606-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2512012&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989CC79800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.18" data-doi="10.1016/j.ydbio.2013.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crocker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Schnurri/Mad/Medea complex attenuates the dorsal-twist gradient readout at <em>vnd</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">378</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D378%26rft.spage%253D64%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2013.03.002%26rft_id%253Dinfo%253Apmid%252F23499655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2013.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23499655&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crocker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamori</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution acts on enhancer organization to fine-tune gradient threshold readouts.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2576</span>–<span class="cit-lpage">2587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D6%26rft.spage%253D2576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000261187900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.20" data-doi="10.1038/ncomms1102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crocker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erives</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic evolution of precise regulatory encodings creates the clustered site signature of enhancers.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DCrocker%26rft.auinit1%253DJ.%26rft.volume%253D1%26rft.spage%253D99%26rft.epage%253D99%26rft.atitle%253DDynamic%2Bevolution%2Bof%2Bprecise%2Bregulatory%2Bencodings%2Bcreates%2Bthe%2Bclustered%2Bsite%2Bsignature%2Bof%2Benhancers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1102%26rft_id%253Dinfo%253Apmid%252F20981027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981027&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.21" data-doi="10.1038/sj.hdy.6800562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legout</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pétavy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isofemale lines in <em>Drosophila</em>: an empirical approach to quantitative trait analysis in natural populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DDavid%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D12%26rft.atitle%253DIsofemale%2Blines%2Bin%2BDrosophila%253A%2Ban%2Bempirical%2Bapproach%2Bto%2Bquantitative%2Btrait%2Banalysis%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800562%26rft_id%253Dinfo%253Apmid%252F15329665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15329665&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225772400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.22" data-doi="10.1093/bioinformatics/bti541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csizmok</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tompa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">IUPred: Web server for the prediction of intrinsically unstructured regions of proteins based on estimated energy content.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3433</span>–<span class="cit-lpage">3434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti541%26rft_id%253Dinfo%253Apmid%252F15955779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTYvMzQzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.23" data-doi="10.1016/j.jmb.2005.01.071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csizmók</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tompa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">The pairwise energy content estimated from amino acid composition discriminates between folded and intrinsically unstructured proteins.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDoszt%25C3%25A1nyi%26rft.auinit1%253DZ.%26rft.volume%253D347%26rft.issue%253D4%26rft.spage%253D827%26rft.epage%253D839%26rft.atitle%253DThe%2Bpairwise%2Benergy%2Bcontent%2Bestimated%2Bfrom%2Bamino%2Bacid%2Bcomposition%2Bdiscriminates%2Bbetween%2Bfolded%2Band%2Bintrinsically%2Bunstructured%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2005.01.071%26rft_id%253Dinfo%253Apmid%252F15769473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2005.01.071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15769473&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227959200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.24" data-doi="10.1016/S0168-9525(00)02139-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Microsatellite mutations in the germline: Implications for evolutionary inference.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D551%26rft.epage%253D558%26rft.atitle%253DMicrosatellite%2Bmutations%2Bin%2Bthe%2Bgermline%253A%2Bimplications%2Bfor%2Bevolutionary%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902139-9%26rft_id%253Dinfo%253Apmid%252F11102705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02139-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11102705&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165861000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.25" data-doi="10.1016/0092-8674(90)90534-L"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fehon</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kooh</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regan</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Molecular interactions between the protein products of the neurogenic loci <em>Notch</em> and <em>Delta</em>, two EGF-homologous genes in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFehon%26rft.auinit1%253DR.%2BG.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D523%26rft.epage%253D534%26rft.atitle%253DMolecular%2Binteractions%2Bbetween%2Bthe%2Bprotein%2Bproducts%2Bof%2Bthe%2Bneurogenic%2Bloci%2BNotch%2Band%2BDelta%252C%2Btwo%2BEGF-homologous%2Bgenes%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252890%252990534-L%26rft_id%253Dinfo%253Apmid%252F2185893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(90)90534-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2185893&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DC93400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.26" data-doi="10.1016/j.devcel.2009.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortini</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Notch signaling: The core pathway and its posttranslational regulation.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DFortini%26rft.auinit1%253DM.%2BE.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D633%26rft.epage%253D647%26rft.atitle%253DNotch%2Bsignaling%253A%2Bthe%2Bcore%2Bpathway%2Band%2Bits%2Bposttranslational%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2009.03.010%26rft_id%253Dinfo%253Apmid%252F19460341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2009.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19460341&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266347100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.27" data-doi="10.1016/0092-8674(94)90196-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortini</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artavanis-Tsakonas</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The Suppressor of Hairless protein participates in Notch receptor signaling.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFortini%26rft.auinit1%253DM.%2BE.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D282%26rft.atitle%253DThe%2Bsuppressor%2Bof%2Bhairless%2Bprotein%2Bparticipates%2Bin%2Bnotch%2Breceptor%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990196-1%26rft_id%253Dinfo%253Apmid%252F7954795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90196-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7954795&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PN63000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.28" data-doi="10.1101/gad.991602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turbachova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kintner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mastermind mediates chromatin-specific transcription and turnover of the Notch enhancer complex.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DFryer%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D1397%26rft.epage%253D1411%26rft.atitle%253DMastermind%2Bmediates%2Bchromatin-specific%2Btranscription%2Band%2Bturnover%2Bof%2Bthe%2BNotch%2Benhancer%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.991602%26rft_id%253Dinfo%253Apmid%252F12050117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE2LzExLzEzOTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.29" data-doi="10.1016/j.molcel.2004.10.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mastermind recruits CycC:CDK8 to phosphorylate the Notch ICD and coordinate activation with turnover.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DFryer%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D509%26rft.epage%253D520%26rft.atitle%253DMastermind%2Brecruits%2BCycC%253ACDK8%2Bto%2Bphosphorylate%2Bthe%2BNotch%2BICD%2Band%2Bcoordinate%2Bactivation%2Bwith%2Bturnover.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.10.014%26rft_id%253Dinfo%253Apmid%252F15546612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15546612&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225249600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayakawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawaichi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanimura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honjo</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> Suppressor of hairless, the <em>Drosophila</em> homologue of RBP-J kappa, transactivates the neurogenic gene <em>E(spl)m8</em>. Idengaku zasshi 70: 505–524.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.31" data-doi="10.1016/S0960-9822(98)70322-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galant</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skeath</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paddock</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Expression pattern of a butterfly <em>achaete-scute</em> homolog reveals the homology of butterfly wing scales and insect sensory bristles.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DGalant%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D14%26rft.spage%253D807%26rft.epage%253D813%26rft.atitle%253DExpression%2Bpattern%2Bof%2Ba%2Bbutterfly%2Bachaete-scute%2Bhomolog%2Breveals%2Bthe%2Bhomology%2Bof%2Bbutterfly%2Bwing%2Bscales%2Band%2Binsect%2Bsensory%2Bbristles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252898%252970322-7%26rft_id%253Dinfo%253Apmid%252F9663389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(98)70322-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9663389&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074593600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.32" data-doi="10.1101/gr.130922.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrigan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingan</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geneva</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequencing reveals complex speciation in the <em>Drosophila simulans</em> clade.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130922.111%26rft_id%253Dinfo%253Apmid%252F22534282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTQ5OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.33" data-doi="10.1101/gad.9.15.1869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Skarmeta</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culí</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrés-Marcó</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title"><em>Cis</em>-regulation of <em>achaete</em> and <em>scute</em>: Shared enhancer-like elements drive their coexpression in proneural clusters of the imaginal discs.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1869</span>–<span class="cit-lpage">1882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGomez-Skarmeta%26rft.auinit1%253DJ%2BL%26rft.volume%253D9%26rft.issue%253D15%26rft.spage%253D1869%26rft.epage%253D1882%26rft.atitle%253DCis-regulation%2Bof%2Bachaete%2Band%2Bscute%253A%2Bshared%2Benhancer-like%2Belements%2Bdrive%2Btheir%2Bcoexpression%2Bin%2Bproneural%2Bclusters%2Bof%2Bthe%2Bimaginal%2Bdiscs.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.15.1869%26rft_id%253Dinfo%253Apmid%252F7649474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNS8xODY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.34" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ràmia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 <em>Drosophila melanogaster</em> Genetic Reference Panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.35" data-doi="10.1126/science.3059495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadonaga</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladika</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tjian</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Distinct regions of Sp1 modulate DNA binding and transcriptional activation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKadonaga%26rft.auinit1%253DJ.%26rft.volume%253D242%26rft.issue%253D4885%26rft.spage%253D1566%26rft.epage%253D1570%26rft.atitle%253DDistinct%2Bregions%2Bof%2BSp1%2Bmodulate%2BDNA%2Bbinding%2Band%2Btranscriptional%2Bactivation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3059495%26rft_id%253Dinfo%253Apmid%252F3059495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNDIvNDg4NS8xNTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kassis</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Farrell</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Sequence conservation in the protein coding and intron regions of the engrailed transcription unit.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3583</span>–<span class="cit-lpage">3589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D5%26rft.spage%253D3583%26rft_id%253Dinfo%253Apmid%252F2881781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2881781&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.37" data-doi="10.1073/pnas.0702887104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artavanis-Tsakonas</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Conformational variability of the intracellular domain of <em>Drosophila</em> Notch and its interaction with Suppressor of Hairless.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">9591</span>–<span class="cit-lpage">9596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0702887104%26rft_id%253Dinfo%253Apmid%252F17535912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIzLzk1OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.38" data-doi="10.1128/MCB.6.9.3094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Sequence of the notch locus of <em>Drosophila melanogaster</em>: relationship of the encoded protein to mammalian clotting and growth factors.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3094</span>–<span class="cit-lpage">3108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKidd%26rft.auinit1%253DS%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D3094%26rft.epage%253D3108%26rft.atitle%253DSequence%2Bof%2Bthe%2Bnotch%2Blocus%2Bof%2BDrosophila%2Bmelanogaster%253A%2Brelationship%2Bof%2Bthe%2Bencoded%2Bprotein%2Bto%2Bmammalian%2Bclotting%2Band%2Bgrowth%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.6.9.3094%26rft_id%253Dinfo%253Apmid%252F3097517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8zMDk0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.39" data-doi="10.4161/pri.23807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Beta conformation of polyglutamine track revealed by a crystal structure of Huntingtin N-terminal region with insertion of three histidine residues.</span> <abbr class="cit-jnl-abbrev">Prion</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPrion%26rft.volume%253D7%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fpri.23807%26rft_id%253Dinfo%253Apmid%252F23370273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/pri.23807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23370273&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovall</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> Structures of CSL, Notch and Mastermind proteins: piecing together an active transcription complex. Curr. Opin. Struct. Biol. 17: 117–127.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.41" data-doi="10.1038/ng1292-301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">La Spada</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roling</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harding</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Meiotic stability and genotype-phenotype correlation of the trinucleotide repeat in X-linked spinal and bulbar muscular atrophy.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLa%2BSpada%26rft.auinit1%253DA.%2BR.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D301%26rft.epage%253D304%26rft.atitle%253DMeiotic%2Bstability%2Band%2Bgenotype-phenotype%2Bcorrelation%2Bof%2Bthe%2Btrinucleotide%2Brepeat%2Bin%2BX-linked%2Bspinal%2Band%2Bbulbar%2Bmuscular%2Batrophy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1292-301%26rft_id%253Dinfo%253Apmid%252F1303283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1292-301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1303283&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KB49500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.42" data-doi="10.1101/gad.9.21.2598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lecourtois</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweisguth</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The neurogenic suppressor of hairless DNA-binding protein mediates the transcriptional activation of the enhancer of split complex genes triggered by Notch signaling.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2598</span>–<span class="cit-lpage">2608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLecourtois%26rft.auinit1%253DM%26rft.volume%253D9%26rft.issue%253D21%26rft.spage%253D2598%26rft.epage%253D2608%26rft.atitle%253DThe%2Bneurogenic%2Bsuppressor%2Bof%2Bhairless%2BDNA-binding%2Bprotein%2Bmediates%2Bthe%2Btranscriptional%2Bactivation%2Bof%2Bthe%2Benhancer%2Bof%2Bsplit%2Bcomplex%2Bgenes%2Btriggered%2Bby%2BNotch%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.21.2598%26rft_id%253Dinfo%253Apmid%252F7590238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8yMS8yNTk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2405.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grell</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <em>Genetic Variations of Drosophila melanogaster</em>. Carnegie Institute of Washington Publication.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.44" data-doi="10.1073/pnas.90.21.10395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Further evidence for function of the <em>Drosophila</em> Notch protein as a transmembrane receptor.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10395</span>–<span class="cit-lpage">10399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLyman%26rft.auinit1%253DD.%26rft.volume%253D90%26rft.issue%253D21%26rft.spage%253D10395%26rft.epage%253D10399%26rft.atitle%253DFurther%2Bevidence%2Bfor%2Bfunction%2Bof%2Bthe%2BDrosophila%2BNotch%2Bprotein%2Bas%2Ba%2Btransmembrane%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.21.10395%26rft_id%253Dinfo%253Apmid%252F8234305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjEvMTAzOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.45" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> Genetic Reference Panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.46" data-doi="10.1126/science.1900954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modolell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Cross-regulatory interactions between the proneural <em>achaete</em> and <em>scute</em> genes of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">251</span>: <span class="cit-fpage">1485</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMartinez%26rft.auinit1%253DC%26rft.volume%253D251%26rft.issue%253D5000%26rft.spage%253D1485%26rft.epage%253D1487%26rft.atitle%253DCross-regulatory%2Binteractions%2Bbetween%2Bthe%2Bproneural%2Bachaete%2Band%2Bscute%2Bgenes%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1900954%26rft_id%253Dinfo%253Apmid%252F1900954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNTEvNTAwMC8xNDg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.11.2405.47"><div class="cit-metadata unstructured">Matsuno, H., R. Murakami, R. Yamane, N. Yamasaki, S. Fujita <em>et al.</em>, 2003 Boundary formation by Notch signaling in <em>Drosophila</em> multicellular systems: experimental observations and gene network modeling by Genomic Object Net. Pacific Symposium on Biocomputing, Hawaii. Pacific Symposium on Biocomputing pp. 152–163.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Modolell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campuzano</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The <em>achaete-scute</em> complex as an integrating device.</span> <abbr class="cit-jnl-abbrev">Int. J. Dev. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bdevelopmental%2Bbiology%26rft.stitle%253DInt%2BJ%2BDev%2BBiol%26rft.aulast%253DModolell%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D282%26rft.atitle%253DThe%2Bachaete-scute%2Bcomplex%2Bas%2Ban%2Bintegrating%2Bdevice.%26rft_id%253Dinfo%253Apmid%252F9654009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9654009&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073605600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagelkerke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweep</span>  <span class="cit-name-given-names">F. C. G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Span</span>  <span class="cit-name-given-names">P. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The unfolded protein response as a target for cancer therapy.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr>.1846: 277–284.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.50" data-doi="10.1016/j.tig.2009.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Negre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of the <em>achaete-scute</em> complex in insects: convergent duplication of proneural genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNegre%26rft.auinit1%253DB.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D147%26rft.epage%253D152%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bachaete-scute%2Bcomplex%2Bin%2Binsects%253A%2Bconvergent%2Bduplication%2Bof%2Bproneural%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.02.001%26rft_id%253Dinfo%253Apmid%252F19285745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285745&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265823700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.51" data-doi="10.1007/BF02101281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newfeld</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoller</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yedvobnick</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Interspecific comparison of the unusually repetitive <em>Drosophila</em> locus <em>mastermind</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DNewfeld%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D415%26rft.epage%253D420%26rft.atitle%253DInterspecific%2Bcomparison%2Bof%2Bthe%2Bunusually%2Brepetitive%2BDrosophila%2Blocus%2Bmastermind.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101281%26rft_id%253Dinfo%253Apmid%252F1904096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904096&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FJ48500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.52" data-doi="10.1007/BF00160429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newfeld</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yedvobnick</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Homopolymer length variation in the <em>Drosophila</em> gene <em>mastermind</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DNewfeld%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D483%26rft.epage%253D495%26rft.atitle%253DHomopolymer%2Blength%2Bvariation%2Bin%2Bthe%2BDrosophila%2Bgene%2Bmastermind.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160429%26rft_id%253Dinfo%253Apmid%252F8283480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00160429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8283480&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newfeld</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tachida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yedvobnick</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Drive-selection equilibrium: homopolymer evolution in the <em>Drosophila</em> gene <em>mastermind</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DNewfeld%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D637%26rft.epage%253D641%26rft.atitle%253DDrive-selection%2Bequilibrium%253A%2Bhomopolymer%2Bevolution%2Bin%2Bthe%2BDrosophila%2Bgene%2Bmastermind.%26rft_id%253Dinfo%253Apmid%252F8083889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8083889&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NL78400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.54" data-doi="10.1038/ng0793-221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banfi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Servadio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Expansion of an unstable trinucleotide CAG repeat in spinocerebellar ataxia type 1.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BT.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D226%26rft.atitle%253DExpansion%2Bof%2Ban%2Bunstable%2Btrinucleotide%2BCAG%2Brepeat%2Bin%2Bspinocerebellar%2Bataxia%2Btype%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0793-221%26rft_id%253Dinfo%253Apmid%252F8358429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0793-221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8358429&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LJ84200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.55" data-doi="10.1016/0960-9822(93)90174-M"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perutz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Baere</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Polar zippers.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPerutz%26rft.auinit1%253DM.%2BF.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D249%26rft.epage%253D253%26rft.atitle%253DPolar%2Bzippers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0960-9822%252893%252990174-M%26rft_id%253Dinfo%253Apmid%252F15335744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0960-9822(93)90174-M&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15335744&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LL95700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.56" data-doi="10.1073/pnas.91.12.5355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perutz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finch</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Glutamine repeats as polar zippers: their possible role in inherited neurodegenerative diseases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">5355</span>–<span class="cit-lpage">5358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPerutz%26rft.auinit1%253DM.%2BF.%26rft.volume%253D91%26rft.issue%253D12%26rft.spage%253D5355%26rft.epage%253D5358%26rft.atitle%253DGlutamine%2Brepeats%2Bas%2Bpolar%2Bzippers%253A%2Btheir%2Bpossible%2Brole%2Bin%2Binherited%2Bneurodegenerative%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.12.5355%26rft_id%253Dinfo%253Apmid%252F8202492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTEvMTIvNTM1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.57" data-doi="10.1073/pnas.042681399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perutz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finch</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesk</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Amyloid fibers are water-filled nanotubes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">5591</span>–<span class="cit-lpage">5595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPerutz%26rft.auinit1%253DM.%2BF.%26rft.volume%253D99%26rft.issue%253D8%26rft.spage%253D5591%26rft.epage%253D5595%26rft.atitle%253DAmyloid%2Bfibers%2Bare%2Bwater-filled%2Bnanotubes%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.042681399%26rft_id%253Dinfo%253Apmid%252F11960014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS84LzU1OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.58" data-doi="10.1128/MCB.7.2.578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinkham</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olesen</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guarente</span>  <span class="cit-name-given-names">L. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Sequence and nuclear localization of the <em>Saccharomyces cerevisiae HAP2</em> protein, a transcriptional activator.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DPinkham%26rft.auinit1%253DJ%2BL%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D578%26rft.epage%253D585%26rft.atitle%253DSequence%2Band%2Bnuclear%2Blocalization%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BHAP2%2Bprotein%252C%2Ba%2Btranscriptional%2Bactivator.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.2.578%26rft_id%253Dinfo%253Apmid%252F3547076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMi81NzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.59" data-doi="10.1016/S0960-9822(01)00562-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khechumian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seugnet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arbogast</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackermann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Novel <em>Notch</em> alleles reveal a Deltex-dependent pathway repressing neural fate.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1729</span>–<span class="cit-lpage">1738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DRamain%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D22%26rft.spage%253D1729%26rft.epage%253D1738%26rft.atitle%253DNovel%2BNotch%2Balleles%2Breveal%2Ba%2BDeltex-dependent%2Bpathway%2Brepressing%2Bneural%2Bfate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252801%252900562-0%26rft_id%253Dinfo%253Apmid%252F11719214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(01)00562-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11719214&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172363400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.60" data-doi="10.1093/bioinformatics/btr691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ràmia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casillas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PopDrowser: The population <em>Drosophila</em> browser.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr691%26rft_id%253Dinfo%253Apmid%252F22180410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC80LzU5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.61" data-doi="10.1016/S0092-8674(00)80514-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scherzinger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lurz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turmaine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangiarini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenbach</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>Huntingtin</em>-encoded polyglutamine expansions form amyloid-like protein aggregates in vitro and in vivo.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DScherzinger%26rft.auinit1%253DE.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D549%26rft.epage%253D558%26rft.atitle%253DHuntingtin-encoded%2Bpolyglutamine%2Bexpansions%2Bform%2Bamyloid-like%2Bprotein%2Baggregates%2Bin%2Bvitro%2Band%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980514-0%26rft_id%253Dinfo%253Apmid%252F9267034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80514-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9267034&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XQ06300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.62" data-doi="10.1073/pnas.96.8.4604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scherzinger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sittler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweiger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiser</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lurz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Self-assembly of polyglutamine-containing huntingtin fragments into amyloid-like fibrils: implications for Huntington’s disease pathology.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4604</span>–<span class="cit-lpage">4609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DScherzinger%26rft.auinit1%253DE.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4604%26rft.epage%253D4609%26rft.atitle%253DSelf-assembly%2Bof%2Bpolyglutamine-containing%2Bhuntingtin%2Bfragments%2Binto%2Bamyloid-like%2Bfibrils%253A%2BImplications%2Bfor%2BHuntington%2527s%2Bdisease%2Bpathology%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4604%26rft_id%253Dinfo%253Apmid%252F10200309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ2MDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.63" data-doi="10.1007/s004120000089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Evolutionary dynamics of microsatellite DNA.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DSchl%25C3%25B6tterer%26rft.auinit1%253DC.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D365%26rft.epage%253D371%26rft.atitle%253DEvolutionary%2Bdynamics%2Bof%2Bmicrosatellite%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004120000089%26rft_id%253Dinfo%253Apmid%252F11072791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004120000089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11072791&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089909700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dash</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brahmachari</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> Role of histidine interruption in mitigating the pathological effects of long polyglutamine stretches in <em>SCA1</em>: A molecular approach. Protein Sci. 12: 953–962.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.65" data-doi="10.1016/S0014-5793(99)00933-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brahmachari</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Peptide models for inherited neurodegenerative disorders: Conformation and aggregation properties of long polyglutamine peptides with and without interruptions.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DSharma%26rft.auinit1%253DD.%26rft.volume%253D456%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D185%26rft.atitle%253DPeptide%2Bmodels%2Bfor%2Binherited%2Bneurodegenerative%2Bdisorders%253A%2Bconformation%2Band%2Baggregation%2Bproperties%2Bof%2Blong%2Bpolyglutamine%2Bpeptides%2Bwith%2Band%2Bwithout%2Binterruptions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252899%252900933-3%26rft_id%253Dinfo%253Apmid%252F10452554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(99)00933-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10452554&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081910100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.66" data-doi="10.1016/j.gde.2011.07.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaya</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sprinzak</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From Notch signaling to fine-grained patterning: Modeling meets experiments.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">732</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DShaya%26rft.auinit1%253DO.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D732%26rft.epage%253D739%26rft.atitle%253DFrom%2BNotch%2Bsignaling%2Bto%2Bfine-grained%2Bpatterning%253A%2BModeling%2Bmeets%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2011.07.007%26rft_id%253Dinfo%253Apmid%252F21862316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2011.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21862316&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.67" data-doi="10.1016/j.cell.2014.03.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodcock</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkin</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trubenová</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monk</span>  <span class="cit-name-given-names">N. A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Compensatory flux changes within an endocytic trafficking network maintain thermal robustness of Notch signaling.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1160</span>–<span class="cit-lpage">1174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1160%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.03.050%26rft_id%253Dinfo%253Apmid%252F24855951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.03.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24855951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sidorov</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> A study of step-allelomorphism in <em>Drosophila melanogaster.</em> A case study of origination of an allelomorph of <em>scute</em> producing simultaneously characters of ‘hairy wing’ [translation]. Zhurnal eksperimental noi biologii i meditsiny VII: 1–40.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.69" data-doi="10.1101/gad.6.12b.2606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skeath</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panganiban</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selegue</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Gene regulation in two dimensions: The proneural <em>achaete</em> and <em>scute</em> genes are controlled by combinations of axis-patterning genes through a common intergenic control region.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2606</span>–<span class="cit-lpage">2619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSkeath%26rft.auinit1%253DJ%2BB%26rft.volume%253D6%26rft.issue%253D12b%26rft.spage%253D2606%26rft.epage%253D2619%26rft.atitle%253DGene%2Bregulation%2Bin%2Btwo%2Bdimensions%253A%2Bthe%2Bproneural%2Bachaete%2Band%2Bscute%2Bgenes%2Bare%2Bcontrolled%2Bby%2Bcombinations%2Bof%2Baxis-patterning%2Bgenes%2Bthrough%2Ba%2Bcommon%2Bintergenic%2Bcontrol%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.6.12b.2606%26rft_id%253Dinfo%253Apmid%252F1340472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjYvMTJiLzI2MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.70" data-doi="10.1038/ng0893-393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacMillan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheadle</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazarou</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Relationship between trinucleotide repeat expansion and phenotypic variation in Huntington’s disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSnell%26rft.auinit1%253DR.%2BG.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D393%26rft.epage%253D397%26rft.atitle%253DRelationship%2Bbetween%2Btrinucleotide%2Brepeat%2Bexpansion%2Band%2Bphenotypic%2Bvariation%2Bin%2BHuntington%2527s%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0893-393%26rft_id%253Dinfo%253Apmid%252F8401588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0893-393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8401588&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LQ17800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.71" data-doi="10.1534/genetics.111.131706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soshnev</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcallister</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wehling</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A conserved long Noncoding RNA affects sleep behavior in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131706%26rft_id%253Dinfo%253Apmid%252F21775470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNDU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.72" data-doi="10.1371/journal.pcbi.1002069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sprinzak</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakhanpal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeBon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Ojalvo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elowitz</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutual inactivation of Notch receptors and ligands facilitates developmental patterning.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1002069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D7%26rft.spage%253D1002069%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.73" data-doi="10.1128/MCB.8.11.4991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukasawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title"><em>GAL11</em> protein, an auxiliary transcription activator for genes encoding galactose-metabolizing enzymes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4991</span>–<span class="cit-lpage">4999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSuzuki%26rft.auinit1%253DY%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D4991%26rft.epage%253D4999%26rft.atitle%253DGAL11%2Bprotein%252C%2Ban%2Bauxiliary%2Btranscription%2Bactivator%2Bfor%2Bgenes%2Bencoding%2Bgalactose-metabolizing%2Benzymes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.11.4991%26rft_id%253Dinfo%253Apmid%252F3062377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjgvMTEvNDk5MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.74" data-doi="10.1093/nar/17.16.6463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Hypervariability of simple sequences as a general source for polymorphic DNA markers.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">6463</span>–<span class="cit-lpage">6471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTautz%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D16%26rft.spage%253D6463%26rft.epage%253D6471%26rft.atitle%253DHypervariabflity%2Bof%2Bsimple%2Bsequences%2Bas%2Ba%2Bgeneral%2Bsource%2Bfor%2Bpolymorphic%2BDNA%2Bmarkers%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.16.6463%26rft_id%253Dinfo%253Apmid%252F2780284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xNi82NDYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.75" data-doi="10.1371/journal.pcbi.1000243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tóth-Petróczy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Malleable machines in transcription regulation: The Mediator complex.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1000243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D4%26rft.spage%253D1000243%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-3" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.76" data-doi="10.1371/journal.pone.0104858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaisnav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stojadinovic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association analysis of radiation resistance in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e104858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De104858%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0104858%26rft_id%253Dinfo%253Apmid%252F25121966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0104858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25121966&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.77" data-doi="10.1016/0092-8674(85)90308-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wharton</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yedvobnick</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finnerty</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artavanis-Tsakonas</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title"><em>opa</em>: a novel family of transcribed repeats shared by the <em>Notch</em> locus and other developmentally regulated loci in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWharton%26rft.auinit1%253DK.%2BA.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D55%26rft.epage%253D62%26rft.atitle%253Dopa%253A%2Ba%2Bnovel%2Bfamily%2Bof%2Btranscribed%2Brepeats%2Bshared%2Bby%2Bthe%2BNotch%2Blocus%2Band%2Bother%2Bdevelopmentally%2Bregulated%2Bloci%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252885%252990308-3%26rft_id%253Dinfo%253Apmid%252F2981631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(85)90308-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2981631&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AAZ2900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The <em>OPI1</em> gene of <em>Saccharomyces cerevisiae</em>, a negative regulator of phospholipid biosynthesis, encodes a protein containing polyglutamine tracts and a leucine zipper.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DWhite%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D266%26rft.issue%253D2%26rft.spage%253D863%26rft.epage%253D872%26rft.atitle%253DThe%2BOPI1%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae%252C%2Ba%2Bnegative%2Bregulator%2Bof%2Bphospholipid%2Bbiosynthesis%252C%2Bencodes%2Ba%2Bprotein%2Bcontaining%2Bpolyglutamine%2Btracts%2Band%2Ba%2Bleucine%2Bzipper%26rft_id%253Dinfo%253Apmid%252F1985968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI2Ni8yLzg2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.79" data-doi="10.1038/82644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aster</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blacklow</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artavanis-Tsakonas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title"><em>MAML1</em>, a human homologue of <em>Drosophila mastermind</em>, is a transcriptional co-activator for <em>NOTCH</em> receptors.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DL.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D484%26rft.epage%253D489%26rft.atitle%253DMAML1%252C%2Ba%2Bhuman%2Bhomologue%2Bof%2BDrosophila%2Bmastermind%252C%2Bis%2Ba%2Btranscriptional%2Bco-activator%2Bfor%2BNOTCH%2Breceptors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F82644%26rft_id%253Dinfo%253Apmid%252F11101851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/82644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11101851&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165671700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.80" data-doi="10.1016/S0070-2153(10)92005-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charng</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellen</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Endocytosis and intracellular trafficking of Notch and its ligands.</span> <abbr class="cit-jnl-abbrev">Curr. Top. Dev. Biol.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bdevelopmental%2Bbiology%26rft.stitle%253DCurr%2BTop%2BDev%2BBiol%26rft.aulast%253DYamamoto%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.spage%253D165%26rft.epage%253D200%26rft.atitle%253DEndocytosis%2Band%2Bintracellular%2Btrafficking%2Bof%2BNotch%2Band%2Bits%2Bligands.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0070-2153%252810%252992005-X%26rft_id%253Dinfo%253Apmid%252F20816395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0070-2153(10)92005-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20816395&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2405.81" data-doi="10.1128/MCB.11.11.5735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzluf</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title"><em>nit-4</em>, a pathway-specific regulatory gene of <em>Neurospora crassa</em>, encodes a protein with a putative binuclear zinc DNA-binding domain.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">5735</span>–<span class="cit-lpage">5745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DYuan%26rft.auinit1%253DG%2BF%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D5735%26rft.epage%253D5745%26rft.atitle%253Dnit-4%252C%2Ba%2Bpathway-specific%2Bregulatory%2Bgene%2Bof%2BNeurospora%2Bcrassa%252C%2Bencodes%2Ba%2Bprotein%2Bwith%2Ba%2Bputative%2Bbinuclear%2Bzinc%2BDNA-binding%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.11.5735%26rft_id%253Dinfo%253Apmid%252F1840634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS81NzM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2405.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2391" title="Genome-Wide Association Study of Ureide Concentration in Diverse Maturity Group IV Soybean [Glycine max (L.) Merr.] Accessions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2421" title="Flux Control in a Defense Pathway in Arabidopsis thaliana Is Robust to Environmental Perturbations and Controls Variation in Adaptive Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GR7JizKELC-eSscofbdf2cH4Am7gqK9yG42_vnd-1lo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2405" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2405" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2405" class="active" data-icon-position="" data-hide-link-title="0">The Nature, Extent, and Consequences of Genetic Variation in the opa Repeats of Notch in Drosophila</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460177" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Xzp-_4mHW16mlltX9DZTa-M6uGQiQxMXMmk8QXJ-fLs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2405.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2405" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ncnwXbL9YUH24miNqXsOvYQacrQd_2hECWv0Zv-rzCw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2405" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2405.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460177" id="node460177--2" data-pisa="ggg;5/11/2405" data-pisa-master="ggg;g3.115.021659" data-apath="/ggg/5/11/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Nature, Extent, and Consequences of Genetic Variation in the <em>opa</em> Repeats of <em>Notch</em> in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Clinton Rice</span>, <span class="highwire-citation-author" data-delta="1">Danielle Beekman</span>, <span class="highwire-citation-author" data-delta="2">Liping Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2419;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021659 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460177--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clinton Rice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BRice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BRice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Danielle Beekman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BBeekman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beekman%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BBeekman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liping Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liping%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiping%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2405" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2405&amp;text=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2405&amp;t=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2405&amp;title=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460177/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460177" id="node460177--3" data-pisa="ggg;5/11/2405" data-pisa-master="ggg;g3.115.021659" data-apath="/ggg/5/11/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Nature, Extent, and Consequences of Genetic Variation in the <em>opa</em> Repeats of <em>Notch</em> in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Clinton Rice</span>, <span class="highwire-citation-author" data-delta="1">Danielle Beekman</span>, <span class="highwire-citation-author" data-delta="2">Liping Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Albert Erives</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2419;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021659 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460177--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clinton Rice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BRice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BRice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Danielle Beekman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BBeekman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beekman%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BBeekman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liping Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liping%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiping%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Albert Erives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Iowa, Iowa City, Iowa 52242-1324</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albert%2BErives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erives%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbert%2BErives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7107-5518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Albert Erives</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:albert-erives@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">albert-erives@uiowa.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2405&amp;rft.epage=2419&amp;rft.atitle=The%20Nature%2C%20Extent%2C%20and%20Consequences%20of%20Genetic%20Variation%20in%20the%20opa%20Repeats%20of%20Notch%20in%20Drosophila&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rice%2C+Clinton&amp;rft.au=Beekman%2C+Danielle&amp;rft.au=Liu%2C+Liping&amp;rft.au=Erives%2C+Albert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460177/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460177/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460177/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460177/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460177/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460177/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460177/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460177/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460177/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460177/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460177/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460177/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2405" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2405.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2405.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2405.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2405.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
