<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2441" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2441.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2441.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2441.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2441.atom" />
<meta name="HW.pisa" content="ggg;5/11/2441" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes" />
<meta name="DC.Identifier" content="10.1534/g3.115.020164" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Kitchen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Many nonmodel species exemplify important biological questions but lack the sequence resources required to study the genes and genomic regions underlying traits of interest. Reef-building corals are famously sensitive to rising seawater temperatures, motivating ongoing research into their stress responses and long-term prospects in a changing climate. A comprehensive understanding of these processes will require extending beyond the sequenced coral genome ( Acropora digitifera ) to encompass diverse coral species and related anthozoans. Toward that end, we have assembled and annotated reference transcriptomes to develop catalogs of gene sequences for three scleractinian corals ( Fungia scutaria , Montastraea cavernosa , Seriatopora hystrix ) and a temperate anemone ( Anthopleura elegantissima ). High-throughput sequencing of cDNA libraries produced ∼20–30 million reads per sample, and de novo assembly of these reads produced ∼75,000–110,000 transcripts from each sample with size distributions (mean ∼1.4 kb, N50 ∼2 kb), comparable to the distribution of gene models from the coral genome (mean ∼1.7 kb, N50 ∼2.2 kb). Each assembly includes matches for more than half the gene models from A. digitifera (54–67%) and many reasonably complete transcripts (∼5300–6700) spanning nearly the entire gene (ortholog hit ratios ≥0.75). The catalogs of gene sequences developed in this study made it possible to identify hundreds to thousands of orthologs across diverse scleractinian species and related taxa. We used these sequences for phylogenetic inference, recovering known relationships and demonstrating superior performance over phylogenetic trees constructed using single mitochondrial loci. The resources developed in this study provide gene sequences and genetic markers for several anthozoan species. To enhance the utility of these resources for the research community, we developed searchable databases enabling researchers to rapidly recover sequences for genes of interest. Our analysis of de novo assembly quality highlights metrics that we expect will be useful for evaluating the relative quality of other de novo transcriptome assemblies. The identification of orthologous sequences and phylogenetic reconstruction demonstrates the feasibility of these methods for clarifying the substantial uncertainties in the existing scleractinian phylogeny." />
<meta name="DC.Contributor" content="Sheila A. Kitchen" />
<meta name="DC.Contributor" content="Camerron M. Crowder" />
<meta name="DC.Contributor" content="Angela Z. Poole" />
<meta name="DC.Contributor" content="Virginia M. Weis" />
<meta name="DC.Contributor" content="Eli Meyer" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Many nonmodel species exemplify important biological questions but lack the sequence resources required to study the genes and genomic regions underlying traits of interest. Reef-building corals are famously sensitive to rising seawater temperatures, motivating ongoing research into their stress responses and long-term prospects in a changing climate. A comprehensive understanding of these processes will require extending beyond the sequenced coral genome (&lt;i&gt;Acropora digitifera&lt;/i&gt;) to encompass diverse coral species and related anthozoans. Toward that end, we have assembled and annotated reference transcriptomes to develop catalogs of gene sequences for three scleractinian corals (&lt;i&gt;Fungia scutaria&lt;/i&gt;, &lt;i&gt;Montastraea cavernosa&lt;/i&gt;, &lt;i&gt;Seriatopora hystrix&lt;/i&gt;) and a temperate anemone (&lt;i&gt;Anthopleura elegantissima&lt;/i&gt;). High-throughput sequencing of cDNA libraries produced ∼20–30 million reads per sample, and &lt;i&gt;de novo&lt;/i&gt; assembly of these reads produced ∼75,000–110,000 transcripts from each sample with size distributions (mean ∼1.4 kb, N&lt;sub&gt;50&lt;/sub&gt; ∼2 kb), comparable to the distribution of gene models from the coral genome (mean ∼1.7 kb, N&lt;sub&gt;50&lt;/sub&gt; ∼2.2 kb). Each assembly includes matches for more than half the gene models from &lt;i&gt;A. digitifera&lt;/i&gt; (54–67%) and many reasonably complete transcripts (∼5300–6700) spanning nearly the entire gene (ortholog hit ratios ≥0.75). The catalogs of gene sequences developed in this study made it possible to identify hundreds to thousands of orthologs across diverse scleractinian species and related taxa. We used these sequences for phylogenetic inference, recovering known relationships and demonstrating superior performance over phylogenetic trees constructed using single mitochondrial loci. The resources developed in this study provide gene sequences and genetic markers for several anthozoan species. To enhance the utility of these resources for the research community, we developed searchable databases enabling researchers to rapidly recover sequences for genes of interest. Our analysis of &lt;i&gt;de novo&lt;/i&gt; assembly quality highlights metrics that we expect will be useful for evaluating the relative quality of other &lt;i&gt;de novo&lt;/i&gt; transcriptome assemblies. The identification of orthologous sequences and phylogenetic reconstruction demonstrates the feasibility of these methods for clarifying the substantial uncertainties in the existing scleractinian phylogeny.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2441" />
<meta name="citation_id" content="5/11/2441" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2441" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2441.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2441.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2441.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020164" />
<meta name="citation_pmid" content="26384772" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2441" />
<meta name="citation_lastpage" content="2452" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sheila A. Kitchen" />
<meta name="citation_author_institution" content="Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331" />
<meta name="citation_author_email" content="kitchens@science.oregonstate.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4402-8139" />
<meta name="citation_author" content="Camerron M. Crowder" />
<meta name="citation_author_institution" content="Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331" />
<meta name="citation_author" content="Angela Z. Poole" />
<meta name="citation_author_institution" content="Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331" />
<meta name="citation_author" content="Virginia M. Weis" />
<meta name="citation_author_institution" content="Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331" />
<meta name="citation_author" content="Eli Meyer" />
<meta name="citation_author_institution" content="Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Abascal;citation_author=R. Zardoya;citation_author=D. Posada;citation_title=ProtTest: selection of best-fit models of protein evolution.;citation_volume=21;citation_year=2005;citation_pmid=15647292;citation_doi=10.1093/bioinformatics/bti263" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. J. Barshis;citation_author=J. T. Ladner;citation_author=T. A. Oliver;citation_author=F. O. Seneca;citation_author=N. Traylor-Knowles;citation_title=Genomic basis for coral resilience to climate change.;citation_volume=110;citation_year=2013;citation_pmid=23297204;citation_doi=10.1073/pnas.1210224110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Baumgarten;citation_author=O. Simakov;citation_author=L. Y. Esherick;citation_author=Y. J. Liew;citation_author=E. M. Lehnert;citation_title=The genome of Aiptasia, a sea anemone model for coral symbiosis.;citation_volume=112;citation_year=2015;citation_pmid=26324906;citation_doi=10.1073/pnas.1513318112" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=E. A. Berntson;citation_author=S. C. France;citation_author=L. S. Mullineaux;citation_title=Phylogenetic relationships within the class Anthozoa (phylum Cnidaria) based on nuclear 18S rDNA sequences.;citation_pages=417-433;citation_volume=13;citation_year=1999;citation_issue=2;citation_pmid=10603268;citation_doi=10.1006/mpev.1999.0649" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Bongaerts;citation_author=C. Riginos;citation_author=T. Ridgway;citation_author=E. M. Sampayo;citation_author=M. J. H. van Oppen;citation_title=Genetic divergence across habitats in the widespread coral Seriatopora hystrix and its associated Symbiodinium.;citation_pages=e10871-e10871;citation_volume=5;citation_year=2010;citation_issue=5;citation_pmid=20523735;citation_doi=10.1371/journal.pone.0010871" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Bridge;citation_author=C. W. Cunningham;citation_author=B. Schierwater;citation_author=R. DeSalle;citation_author=L. W. Buss;citation_title=Class-level relationships in the phylum Cnidaria: evidence from mitochondrial genome structure.;citation_pages=8750-8753;citation_volume=89;citation_year=1992;citation_issue=18;citation_pmid=1356268;citation_doi=10.1073/pnas.89.18.8750" />
<meta name="citation_reference" content="citation_journal_title=Journal of Morphology;citation_journal_abbrev=Journal of Morphology;citation_author=A. F. Budd;citation_author=J. Stolarski;citation_title=Corallite wall and septal microstructure in scleractinian reef corals: comparison of molecular clades within the family Faviidae.;citation_pages=66-88;citation_volume=272;citation_year=2011;citation_issue=1;citation_pmid=21061280;citation_doi=10.1002/jmor.10899" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Castresana;citation_title=Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis;citation_pages=540-552;citation_volume=17;citation_year=2000;citation_issue=4;citation_pmid=10742046;citation_doi=10.1093/oxfordjournals.molbev.a026334" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Chapman;citation_author=E. F. Kirkness;citation_author=O. Simakov;citation_author=S. E. Hampson;citation_author=T. Mitros;citation_title=The dynamic genome of Hydra.;citation_pages=592-596;citation_volume=464;citation_year=2010;citation_issue=7288;citation_pmid=20228792;citation_doi=10.1038/nature08830" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=C. J. Closek;citation_author=S. Sunagawa;citation_author=M. K. DeSalvo;citation_author=Y. M. Piceno;citation_author=T. Z. DeSantis;citation_title=Coral transcriptome and bacterial community profiles reveal distinct Yellow Band Disease states in Orbicella faveolata.;citation_pages=2411-2422;citation_volume=8;citation_year=2014;citation_pmid=24950107;citation_doi=10.1038/ismej.2014.85" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Conaco;citation_author=P. Neveu;citation_author=H. Zhou;citation_author=M. L. Arcila;citation_author=S. M. Degnan;citation_title=Transcriptome profiling of the demosponge Amphimedon queenslandica reveals genome-wide events that accompany major life cycle transitions.;citation_pages=209-209;citation_volume=13;citation_year=2012;citation_pmid=22646746;citation_doi=10.1186/1471-2164-13-209" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet. Resour.;citation_author=G. Concepcion;citation_author=N. Polato;citation_author=I. Baums;citation_author=R. Toonen;citation_title=Development of microsatellite markers from four Hawaiian corals: Acropora cytherea, Fungia scutaria, Montipora capitata and Porites lobata.;citation_pages=11-15;citation_volume=2;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Zool. J. Linn. Soc.;citation_author=M. Daly;citation_author=D. G. Fautin;citation_author=V. A. Cappola;citation_title=Systematics of the Hexacorallia (Cnidaria: Anthozoa).;citation_pages=419-437;citation_volume=139;citation_year=2003;citation_doi=10.1046/j.1096-3642.2003.00084.x" />
<meta name="citation_reference" content="citation_journal_title=Mar. Biodivers.;citation_author=S. W. Davies;citation_author=M. Rahman;citation_author=E. Meyer;citation_author=E. A. Green;citation_author=E. Buschiazzo;citation_title=Novel polymorphic microsatellite markers for population genetics of the endangered Caribbean star coral, Montastraea faveolata.;citation_pages=167-172;citation_volume=43;citation_year=2013;citation_doi=10.1007/s12526-012-0133-4" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. K. Davy;citation_author=D. Allemand;citation_author=V. M. Weis;citation_title=Cell Biology of Cnidarian-Dinoflagellate Symbiosis;citation_pages=229-261;citation_volume=76;citation_year=2012;citation_issue=2;citation_pmid=22688813;citation_doi=10.1128/MMBR.05014-11" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. DeSalvo;citation_author=C. Voolstra;citation_author=S. Sunagawa;citation_author=J. Schwarz;citation_author=J. Stillman;citation_title=Differential gene expression during thermal stress and bleaching in the Caribbean coral Montastraea faveolata.;citation_pages=3952-3971;citation_volume=17;citation_year=2008;citation_issue=17;citation_pmid=18662230;citation_doi=10.1111/j.1365-294X.2008.03879.x" />
<meta name="citation_reference" content="citation_journal_title=Marine Pollution Bulletin;citation_journal_abbrev=Marine Pollution Bulletin;citation_author=A. E. Douglas;citation_title=Coral bleaching--how and why?;citation_pages=385-392;citation_volume=46;citation_year=2003;citation_issue=4;citation_pmid=12705909;citation_doi=10.1016/S0025-326X(03)00037-7" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. W. Dunn;citation_author=A. Hejnol;citation_author=D. Q. Matus;citation_author=K. Pang;citation_author=W. E. Browne;citation_title=Broad phylogenomic sampling improves resolution of the animal tree of life.;citation_pages=745-749;citation_volume=452;citation_year=2008;citation_issue=7188;citation_pmid=18322464;citation_doi=10.1038/nature06614" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=I. Fernandez-Silva;citation_author=J. Whitney;citation_author=B. Wainwright;citation_author=K. R. Andrews;citation_author=H. Ylitalo-Ward;citation_title=Microsatellites for next-generation ecologists: a post-sequencing bioinformatics pipeline.;citation_pages=e55990;citation_volume=8;citation_year=2013;citation_pmid=23424642;citation_doi=10.1371/journal.pone.0055990" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=B. Fuchs;citation_author=W. Wang;citation_author=S. Graspeuntner;citation_author=Y. Li;citation_author=S. Insua;citation_title=Regulation of polyp-to-jellyfish transition in Aurelia aurita.;citation_pages=263-273;citation_volume=24;citation_year=2014;citation_pmid=24440392;citation_doi=10.1016/j.cub.2013.12.003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Fukami;citation_author=C. A. Chen;citation_author=A. F. Budd;citation_author=A. Collins;citation_author=C. Wallace;citation_title=Mitochondrial and nuclear genes suggest that stony corals are monophyletic but most families of stony corals are not (Order Scleractinia, Class Anthozoa, Phylum Cnidaria).;citation_pages=e3222-e3222;citation_volume=3;citation_year=2008;citation_issue=9;citation_pmid=18795098;citation_doi=10.1371/journal.pone.0003222" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=N. Galtier;citation_author=R. W. Jobson;citation_author=B. Nabholz;citation_author=S. Glémin;citation_author=P. U. Blier;citation_title=Mitochondrial whims: metabolic rate, longevity and the rate of molecular evolution.;citation_volume=5;citation_year=2009;citation_pmid=19324654;citation_doi=10.1098/rsbl.2008.0662" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Hamada;citation_author=E. Shoguchi;citation_author=C. Shinzato;citation_author=T. Kawashima;citation_author=D. J. Miller;citation_title=The complex NOD-like receptor repertoire of the coral Acropora digitifera includes novel domain combinations.;citation_pages=167176;citation_volume=30;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Havird;citation_author=S. R. Santos;citation_title=Performance of single and concatenated sets of mitochondrial genes at inferring metazoan relationships relative to full mitogenome data.;citation_pages=e84080;citation_volume=9;citation_year=2014;citation_pmid=24454717;citation_doi=10.1371/journal.pone.0084080" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. R. Helm;citation_author=S. Siebert;citation_author=S. Tulin;citation_author=J. Smith;citation_author=C. W. Dunn;citation_title=Characterization of differential transcript abundance through time during Nematostella vectensis development.;citation_pages=266;citation_volume=14;citation_year=2013;citation_pmid=23601508;citation_doi=10.1186/1471-2164-14-266" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Karako-Lampert;citation_author=D. Zoccola;citation_author=M. Salmon-Divon;citation_author=M. Katzenellenbogen;citation_author=S. Tambutté;citation_title=Transcriptome analysis of the scleractinian coral Stylophora pistillata.;citation_pages=e88615;citation_volume=9;citation_year=2014;citation_pmid=24551124;citation_doi=10.1371/journal.pone.0088615" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K. Misawa;citation_author=K. I. Kuma;citation_author=T. Miyata;citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;citation_pages=3059-3066;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136088;citation_doi=10.1093/nar/gkf436" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=E. Kayal;citation_author=B. Roure;citation_author=H. Philippe;citation_author=A. G. Collins;citation_author=D. V. Lavrov;citation_title=Cnidarian phylogenetic relationships as revealed by mitogenomics.;citation_pages=5;citation_volume=13;citation_year=2013;citation_pmid=23302374;citation_doi=10.1186/1471-2148-13-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Kenkel;citation_author=E. Meyer;citation_author=M. Matz;citation_title=Gene expression under chronic heat stress in populations of the mustard hill coral (Porites astreoides) from different thermal environments.;citation_pages=4322-4334;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12390" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Modell.;citation_author=S. Kininmonth;citation_author=M. J. H. van Oppen;citation_author=H. P. Possingham;citation_title=Determining the community structure of the coral Seriatopora hystrix from hydrodynamic and genetic networks.;citation_pages=2870-2880;citation_volume=221;citation_year=2010;citation_doi=10.1016/j.ecolmodel.2010.08.042" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. V. Kitahara;citation_author=S. D. Cairns;citation_author=J. Stolarski;citation_author=D. Blair;citation_author=D. J. Miller;citation_title=A comprehensive phylogenetic analysis of the Scleractinia (Cnidaria, Anthozoa) based on mitochondrial CO1 sequence data.;citation_pages=e11490-e11490;citation_volume=5;citation_year=2010;citation_issue=7;citation_pmid=20628613;citation_doi=10.1371/journal.pone.0011490" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. M. Kocot;citation_author=J. T. Cannon;citation_author=C. Todt;citation_author=M. R. Citarella;citation_author=A. B. Kohn;citation_title=Phylogenomics reveals deep molluscan relationships.;citation_pages=452-456;citation_volume=477;citation_year=2011;citation_issue=7365;citation_pmid=21892190;citation_doi=10.1038/nature10382" />
<meta name="citation_reference" content="citation_journal_title=Evol. Bioinform. Online;citation_author=K. M. Kocot;citation_author=M. R. Citarella;citation_author=L. L. Moroz;citation_author=K. M. Halanych;citation_title=PhyloTreePruner: a phylogenetic tree-based approach for selection of orthologous sequences for phylogenomics.;citation_pages=429-435;citation_volume=9;citation_year=2013;citation_pmid=24250218" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=P. Kück;citation_author=K. Meusemann;citation_title=FASconCAT: Convenient handling of data matrices.;citation_pages=1115-1118;citation_volume=56;citation_year=2010;citation_pmid=20416383;citation_doi=10.1016/j.ympev.2010.04.024" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. M. Lehnert;citation_author=M. S. Burriesci;citation_author=J. R. Pringle;citation_title=Developing the anemone Aiptasia as a tractable model for cnidarian-dinoflagellate symbiosis: the transcriptome of aposymbiotic A. pallida.;citation_pages=271-271;citation_volume=13;citation_year=2012;citation_pmid=22726260;citation_doi=10.1186/1471-2164-13-271" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_author=W. P. Maddison;citation_author=D. R. Maddison;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=E. Maier;citation_author=R. Tollrian;citation_author=B. Nürnberger;citation_title=Development of species-specific markers in an organism with endosymbionts: microsatellites in the scleractinian coral Seriatopora hystrix.;citation_pages=157-159;citation_volume=1;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_abbrev=Dev Neurobiol;citation_author=H. Q. Marlow;citation_author=M. Srivastava;citation_author=D. Q. Matus;citation_author=D. Rokhsar;citation_author=M. Q. Martindale;citation_title=Anatomy and development of the nervous system of Nematostella vectensis, an anthozoan cnidarian.;citation_pages=235-254;citation_volume=69;citation_year=2009;citation_issue=4;citation_pmid=19170043;citation_doi=10.1002/dneu.20698" />
<meta name="citation_reference" content="citation_journal_title=Limnol. Oceanogr.;citation_author=C. H. Mazel;citation_author=M. P. Lesser;citation_author=M. Y. Gorbunov;citation_author=T. M. Barry;citation_author=J. H. Farrell;citation_title=Green-fluorescent proteins in Caribbean corals.;citation_pages=402-411;citation_volume=48;citation_year=2003;citation_doi=10.4319/lo.2003.48.1_part_2.0402" />
<meta name="citation_reference" content="citation_journal_title=The Biological Bulletin;citation_journal_abbrev=Biol. Bull.;citation_author=E. Meyer;citation_author=V. M. Weis;citation_title=Study of Cnidarian-Algal Symbiosis in the &quot;Omics&quot; Age;citation_pages=44-65;citation_volume=223;citation_year=2012;citation_issue=1;citation_pmid=22983032" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Meyer;citation_author=G. V. Aglyamova;citation_author=S. Wang;citation_author=J. Buchanan-Carter;citation_author=D. Abrego;citation_title=Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx.;citation_pages=219-219;citation_volume=10;citation_year=2009;citation_pmid=19435504;citation_doi=10.1186/1471-2164-10-219" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=E. Meyer;citation_author=G. V. Aglyamova;citation_author=M. V. Matz;citation_title=Profiling gene expression responses of coral larvae (Acropora millepora) to elevated temperature and settlement inducers using a novel RNA-Seq procedure.;citation_pages=3599-3616;citation_volume=20;citation_year=2011;citation_issue=17;citation_pmid=21801258" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Moriya;citation_author=M. Itoh;citation_author=S. Okuda;citation_author=A. C. Yoshizawa;citation_author=M. Kanehisa;citation_title=KAAS: an automatic genome annotation and pathway reconstruction server.;citation_volume=35;citation_year=2007;citation_pmid=17526522;citation_doi=10.1093/nar/gkm321" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Moya;citation_author=L. Huisman;citation_author=E. Ball;citation_author=D. Hayward;citation_author=L. Grasso;citation_title=Whole transcriptome analysis of the coral Acropora millepora reveals complex responses to CO2-driven acidification during the initiation of calcification.;citation_pages=2440-2454;citation_volume=21;citation_year=2012;citation_issue=10;citation_pmid=22490231;citation_doi=10.1111/j.1365-294X.2012.05554.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Nakasugi;citation_author=R. N. Crowhurst;citation_author=J. Bally;citation_author=C. C. Wood;citation_author=R. P. Hellens;citation_title=De Novo transcriptome sequence assembly and analysis of RNA silencing genes of Nicotiana benthamiana.;citation_pages=e59534;citation_volume=8;citation_year=2013;citation_pmid=23555698;citation_doi=10.1371/journal.pone.0059534" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. O’Neil;citation_author=S. Emrich;citation_title=Assessing De Novo transcriptome assembly metrics for consistency and utility.;citation_pages=465;citation_volume=14;citation_year=2013;citation_pmid=23837739;citation_doi=10.1186/1471-2164-14-465" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. O’Neil;citation_author=J. Dzurisin;citation_author=R. Carmichael;citation_author=N. Lobo;citation_author=S. Emrich;citation_title=Population-level transcriptome sequencing of nonmodel organisms Erynnis propertius and Papilio zelicaon.;citation_pages=310-310;citation_volume=11;citation_year=2010;citation_pmid=20478048;citation_doi=10.1186/1471-2164-11-310" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Philippe;citation_author=E. A. Snell;citation_author=E. Bapteste;citation_author=P. Lopez;citation_author=P. W. Holland;citation_title=Phylogenomics of eukaryotes: impact of missing data on large alignments.;citation_volume=21;citation_year=2004;citation_pmid=15175415;citation_doi=10.1093/molbev/msh182" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. R. Polato;citation_author=J. C. Vera;citation_author=I. B. Baums;citation_title=Gene discovery in the threatened elkhorn coral: 454 sequencing of the Acropora palmata transcriptome.;citation_pages=e28634-e28634;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22216101;citation_doi=10.1371/journal.pone.0028634" />
<meta name="citation_reference" content="citation_journal_title=Dev. Comp. Immunol.;citation_author=A. Z. Poole;citation_author=V. M. Weis;citation_title=TIR-domain-containing protein repertoire of nine anthozoan species reveals coral–specific expansions and uncharacterized proteins.;citation_pages=480-488;citation_volume=46;citation_year=2014;citation_pmid=24933613;citation_doi=10.1016/j.dci.2014.06.002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Pratlong;citation_author=A. Haguenauer;citation_author=O. Chabrol;citation_author=C. Klopp;citation_author=P. Pontarotti;citation_title=The red coral (Corallium rubrum) transcriptome: a new resource for population genetics and local adaptation studies.;citation_pages=1205-1215;citation_volume=15;citation_year=2015;citation_pmid=25648864;citation_doi=10.1111/1755-0998.12383" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. N. Price;citation_author=P. S. Dehal;citation_author=A. P. Arkin;citation_title=FastTree 2--approximately maximum-likelihood trees for large alignments.;citation_pages=e9490-e9490;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20224823;citation_doi=10.1371/journal.pone.0009490" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. H. Putnam;citation_author=M. Srivastava;citation_author=U. Hellsten;citation_author=B. Dirks;citation_author=J. Chapman;citation_title=Sea Anemone Genome Reveals Ancestral Eumetazoan Gene Repertoire and Genomic Organization;citation_pages=86-94;citation_volume=317;citation_year=2007;citation_issue=5834;citation_pmid=17615350;citation_doi=10.1126/science.1139158" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Quast;citation_author=E. Pruesse;citation_author=P. Yilmaz;citation_author=J. Gerken;citation_author=T. Schweer;citation_title=The SILVA ribosomal RNA gene database project: improved data processing and web-based tools.;citation_pages=D590D596;citation_volume=41;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Comp. Biochem.;citation_author=W. S. Reynolds;citation_author=J. A. Schwarz;citation_author=V. M. Weis;citation_title=Symbiosis-enhanced gene expression in cnidarian-algal associations: cloning and characterization of a cDNA, sym32, encoding a possible cell adhesion protein.;citation_pages=33-44;citation_volume=126;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Zool;citation_author=A. Riesgo;citation_author=S. C. Andrade;citation_author=P. Sharma;citation_author=M. Novo;citation_author=A. Perez-Porro;citation_title=Comparative description of ten transcriptomes of newly sequenced invertebrates and efficiency estimation of genomic sampling in non-model taxa.;citation_pages=33-33;citation_volume=9;citation_year=2012;citation_issue=1;citation_pmid=23190771;citation_doi=10.1186/1742-9994-9-33" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Riesgo;citation_author=N. Farrar;citation_author=P. J. Windsor;citation_author=G. Giribet;citation_author=S. P. Leys;citation_title=The analysis of eight transcriptomes from all poriferan classes reveals surprising genetic complexity in sponges.;citation_volume=31;citation_year=2014;citation_pmid=24497032;citation_doi=10.1093/molbev/msu057" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. L. Romano;citation_author=S. R. Palumbi;citation_title=Evolution of Scleractinian Corals Inferred from Molecular Systematics;citation_pages=640-642;citation_volume=271;citation_year=1996;citation_issue=5249;citation_doi=10.1126/science.271.5249.640;citation_georef=1996017853" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=O. Rota-Stabelli;citation_author=Z. Yang;citation_author=M. J. Telford;citation_title=MtZoa: A general mitochondrial amino acid substitutions model for animal evolutionary studies.;citation_pages=268-272;citation_volume=52;citation_year=2009;citation_pmid=19475713;citation_doi=10.1016/j.ympev.2009.01.011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Roure;citation_author=D. Baurain;citation_author=H. Philippe;citation_title=Impact of missing data on phylogenies inferred from empirical phylogenomic data sets.;citation_volume=30;citation_year=2013;citation_pmid=22930702;citation_doi=10.1093/molbev/mss208" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Ruiz-Ramos;citation_author=I. Baums;citation_title=Microsatellite abundance across the Anthozoa and Hydrozoa in the phylum Cnidaria.;citation_pages=939;citation_volume=15;citation_year=2014;citation_pmid=25346285;citation_doi=10.1186/1471-2164-15-939" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. F. Ryan;citation_author=P. M. Burton;citation_author=M. E. Mazza;citation_author=G. K. Kwong;citation_author=J. C. Mullikin;citation_title=The cnidarian-bilaterian ancestor possessed at least 56 homeoboxes: evidence from the starlet sea anemone, Nematostella vectensis.;citation_pages=R64-R64;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16867185;citation_doi=10.1186/gb-2006-7-7-r64" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_title=The genome of the ctenophore Mnemiopsis leidyi and its implications for cell type evolution.;citation_author=J. F. Ryan;citation_author=K. Pang;citation_author=C. E. Schnitzler;citation_author=A.-D. Nguyen;citation_author=R. T. Moreland;citation_volume=342;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=The Biological Bulletin;citation_journal_abbrev=Biol. Bull.;citation_author=J. G. Sanders;citation_author=S. R. Palumbi;citation_title=Populations of Symbiodinium muscatinei Show Strong Biogeographic Structuring in the Intertidal Anemone Anthopleura elegantissima;citation_pages=199-208;citation_volume=220;citation_year=2011;citation_issue=3;citation_pmid=21712228" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Sanders;citation_author=M. Shcheglovitova;citation_author=P. Cartwright;citation_title=Differential gene expression between functionally specialized polyps of the colonial hydrozoan Hydractinia symbiolongicarpus (Phylum Cnidaria).;citation_pages=406;citation_volume=15;citation_year=2014;citation_pmid=24884766;citation_doi=10.1186/1471-2164-15-406" />
<meta name="citation_reference" content="citation_journal_abbrev=Mar Genomics;citation_author=C. E. Schnitzler;citation_author=V. M. Weis;citation_title=Coral larvae exhibit few measurable transcriptional changes during the onset of coral-dinoflagellate endosymbiosis.;citation_pages=107-116;citation_volume=3;citation_year=2010;citation_issue=2;citation_pmid=21798204;citation_doi=10.1016/j.margen.2010.08.002" />
<meta name="citation_reference" content="citation_journal_abbrev=Ecol Lett;citation_author=K. A. Selkoe;citation_author=R. J. Toonen;citation_title=Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers.;citation_pages=615-629;citation_volume=9;citation_year=2006;citation_issue=5;citation_pmid=16643306;citation_doi=10.1111/j.1461-0248.2006.00889.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=X. Serrano;citation_author=I. B. Baums;citation_author=K. O’Reilly;citation_author=T. B. Smith;citation_author=R. J. Jones;citation_title=Geographic differences in vertical connectivity in the Caribbean coral Montastraea cavernosa despite high levels of horizontal connectivity at shallow depths.;citation_pages=4226-4240;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12861" />
<meta name="citation_reference" content="citation_journal_title=Coral Reefs;citation_author=T. L. Shearer;citation_author=C. Gutiérrez-Rodríguez;citation_author=M. A. Coffroth;citation_title=Generating molecular markers from zooxanthellate cnidarians.;citation_pages=57-66;citation_volume=24;citation_year=2005;citation_doi=10.1007/s00338-004-0442-6" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Shinzato;citation_author=E. Shoguchi;citation_author=T. Kawashima;citation_author=M. Hamada;citation_author=K. Hisata;citation_title=Using the Acropora digitifera genome to understand coral responses to environmental change.;citation_pages=320-323;citation_volume=476;citation_year=2011;citation_issue=7360;citation_pmid=21785439;citation_doi=10.1038/nature10249" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=C. Shinzato;citation_author=M. Hamada;citation_author=E. Shoguchi;citation_author=T. Kawashima;citation_author=N. Satoh;citation_title=The repertoire of chemical defense genes in the coral Acropora digitifera genome.;citation_pages=510-517;citation_volume=29;citation_year=2012;citation_issue=8;citation_pmid=22873809;citation_doi=10.2108/zsj.29.510" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=C. Shinzato;citation_author=E. Shoguchi;citation_author=M. Tanaka;citation_author=N. Satoh;citation_title=Fluorescent protein candidate genes in the coral Acropora digitifera genome.;citation_pages=260-264;citation_volume=29;citation_year=2012;citation_issue=4;citation_pmid=22468836;citation_doi=10.2108/zsj.29.260" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Shinzato;citation_author=M. Inoue;citation_author=M. Kusakabe;citation_title=A snapshot of a coral “holobiont”: a transcriptome assembly of the scleractinian coral, Porites, captures a wide variety of genes from both the host and symbiotic zooxanthellae.;citation_pages=e85182;citation_volume=9;citation_year=2014;citation_pmid=24454815;citation_doi=10.1371/journal.pone.0085182" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Siebert;citation_author=M. D. Robinson;citation_author=S. C. Tintori;citation_author=F. Goetz;citation_author=R. R. Helm;citation_title=Differential gene expression in the siphonophore Nanomia bijuga (Cnidaria) assessed with multiple next-generation sequencing workflows.;citation_pages=e22953-e22953;citation_volume=6;citation_year=2011;citation_issue=7;citation_pmid=21829563;citation_doi=10.1371/journal.pone.0022953" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=C. Smith-Keune;citation_author=S. Dove;citation_title=Gene expression of a green fluorescent protein homolog as a host-specific biomarker of heat stress within a reef-building coral.;citation_pages=166-180;citation_volume=10;citation_year=2008;citation_issue=2;citation_pmid=17994304;citation_doi=10.1007/s10126-007-9049-6" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=J. Soza‐Ried;citation_author=A. Hotz‐Wagenblatt;citation_author=K. H. Glatting;citation_author=C. del Val;citation_author=K. Fellenberg;citation_title=The transcriptome of the colonial marine hydroid Hydractinia echinata.;citation_pages=197-209;citation_volume=277;citation_year=2010;citation_issue=1;citation_pmid=19961538;citation_doi=10.1111/j.1742-4658.2009.07474.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Srivastava;citation_author=O. Simakov;citation_author=J. Chapman;citation_author=B. Fahey;citation_author=M. E. A. Gauthier;citation_title=The Amphimedon queenslandica genome and the evolution of animal complexity.;citation_pages=720-726;citation_volume=466;citation_year=2010;citation_issue=7307;citation_pmid=20686567;citation_doi=10.1038/nature09201" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML Version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. J. Stefanik;citation_author=T. J. Lubinski;citation_author=B. R. Granger;citation_author=A. L. Byrd;citation_author=A. M. Reitzel;citation_title=Production of a reference transcriptome and transcriptomic database (EdwardsiellaBase) for the lined sea anemone, Edwardsiella lineata, a parasitic cnidarian.;citation_pages=71;citation_volume=15;citation_year=2014;citation_pmid=24467778;citation_doi=10.1186/1471-2164-15-71" />
<meta name="citation_reference" content="citation_journal_title=Mar. Biotechnol. (NY);citation_author=J. Sun;citation_author=Q. Chen;citation_author=J. C. Lun;citation_author=J. Xu;citation_author=J.-W. Qiu;citation_title=PcarnBase: Development of a transcriptomic database for the brain coral Platygyra carnosus.;citation_pages=244-251;citation_volume=15;citation_year=2013;citation_pmid=22875536;citation_doi=10.1007/s10126-012-9482-z" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=N. Traylor-Knowles;citation_author=B. R. Granger;citation_author=T. J. Lubinski;citation_author=J. R. Parikh;citation_author=S. Garamszegi;citation_title=Production of a reference transcriptome and transcriptomic database (PocilloporaBase) for the cauliflower coral, Pocillopora damicornis.;citation_pages=585-585;citation_volume=12;citation_year=2011;citation_pmid=22126435;citation_doi=10.1186/1471-2164-12-585" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=J. N. Underwood;citation_author=P. B. Souter;citation_author=E. R. Ballment;citation_author=A. H. Lutz;citation_author=M. J. H. van Oppen;citation_title=Development of 10 polymorphic microsatellite markers from herbicide-bleached tissues of the brooding pocilloporid coral Seriatopora hystrix.;citation_pages=176-178;citation_volume=6;citation_year=2006;citation_doi=10.1111/j.1471-8286.2005.01183.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. N. Underwood;citation_author=L. D. Smith;citation_author=M. J. H. van Oppen;citation_author=J. P. Gilmour;citation_title=Multiple scales of genetic connectivity in a brooding coral on isolated reefs following catastrophic bleaching.;citation_pages=771-784;citation_volume=16;citation_year=2007;citation_issue=4;citation_pmid=17284210" />
<meta name="citation_reference" content="citation_title=Graph clustering by flow simulation;citation_author=S. Van Dongen;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. J. H. van Oppen;citation_author=J. Catmull;citation_author=B. J. McDonald;citation_author=N. R. Hislop;citation_author=P. J. Hagerman;citation_title=The mitochondrial genome of Acropora tenuis (Cnidaria; Scleractinia) contains a large group I intron and a candidate control region.;citation_pages=1-13;citation_volume=55;citation_year=2002;citation_issue=1;citation_pmid=12165838;citation_doi=10.1007/s00239-001-0075-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. J. H. van Oppen;citation_author=A. Lutz;citation_author=G. De’ath;citation_author=L. Peplow;citation_author=S. Kininmonth;citation_title=Genetic traces of recent long-distance dispersal in a predominantly self-recruiting coral.;citation_pages=e3401-e3401;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852897;citation_doi=10.1371/journal.pone.0003401" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Vidal-Dupiol;citation_author=D. Zoccola;citation_author=E. Tambutté;citation_author=C. Grunau;citation_author=C. Cosseau;citation_title=Genes related to ion-transport and energy production are upregulated in response to CO2-driven pH decrease in corals: New insights from transcriptome analysis.;citation_pages=e58652;citation_volume=8;citation_year=2013;citation_pmid=23544045;citation_doi=10.1371/journal.pone.0058652" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=V. M. Weis;citation_title=Cellular mechanisms of Cnidarian bleaching: stress causes the collapse of symbiosis;citation_pages=3059-3066;citation_volume=211;citation_year=2008;citation_issue=19;citation_pmid=18805804;citation_doi=10.1242/jeb.009597" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. M. Weis;citation_author=D. Allemand;citation_title=What Determines Coral Health?;citation_pages=1153-1155;citation_volume=324;citation_year=2009;citation_issue=5931;citation_pmid=19478172;citation_doi=10.1126/science.1172540" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Wenger;citation_author=B. Galliot;citation_title=RNAseq vs. genome-predicted transcriptomes: a large population of novel transcripts identified in an Illumina-454 Hydra transcriptome.;citation_pages=204;citation_volume=14;citation_year=2013;citation_pmid=23530871;citation_doi=10.1186/1471-2164-14-204" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Whelan;citation_author=N. Goldman;citation_title=A General Empirical Model of Protein Evolution Derived from Multiple Protein Families Using a Maximum-Likelihood Approach;citation_pages=691-699;citation_volume=18;citation_year=2001;citation_issue=5;citation_pmid=11319253;citation_doi=10.1093/oxfordjournals.molbev.a003851" />
<meta name="citation_reference" content="citation_journal_title=Developmental and comparative immunology;citation_journal_abbrev=Dev Comp Immunol;citation_author=E. M. Wood-Charlson;citation_author=V. M. Weis;citation_title=The diversity of C-type lectins in the genome of a basal metazoan, Nematostella vectensis.;citation_pages=881-889;citation_volume=33;citation_year=2009;citation_issue=8;citation_pmid=19454330;citation_doi=10.1016/j.dci.2009.01.008" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Many nonmodel species exemplify important biological questions but lack the sequence resources required to study the genes and genomic regions underlying traits of interest. Reef-building corals are famously sensitive to rising seawater temperatures, motivating ongoing research into their stress responses and long-term prospects in a changing climate. A comprehensive understanding of these processes will require extending beyond the sequenced coral genome ( Acropora digitifera ) to encompass diverse coral species and related anthozoans. Toward that end, we have assembled and annotated reference transcriptomes to develop catalogs of gene sequences for three scleractinian corals ( Fungia scutaria , Montastraea cavernosa , Seriatopora hystrix ) and a temperate anemone ( Anthopleura elegantissima ). High-throughput sequencing of cDNA libraries produced ∼20–30 million reads per sample, and de novo assembly of these reads produced ∼75,000–110,000 transcripts from each sample with size distributions (mean ∼1.4 kb, N50 ∼2 kb), comparable to the distribution of gene models from the coral genome (mean ∼1.7 kb, N50 ∼2.2 kb). Each assembly includes matches for more than half the gene models from A. digitifera (54–67%) and many reasonably complete transcripts (∼5300–6700) spanning nearly the entire gene (ortholog hit ratios ≥0.75). The catalogs of gene sequences developed in this study made it possible to identify hundreds to thousands of orthologs across diverse scleractinian species and related taxa. We used these sequences for phylogenetic inference, recovering known relationships and demonstrating superior performance over phylogenetic trees constructed using single mitochondrial loci. The resources developed in this study provide gene sequences and genetic markers for several anthozoan species. To enhance the utility of these resources for the research community, we developed searchable databases enabling researchers to rapidly recover sequences for genes of interest. Our analysis of de novo assembly quality highlights metrics that we expect will be useful for evaluating the relative quality of other de novo transcriptome assemblies. The identification of orthologous sequences and phylogenetic reconstruction demonstrates the feasibility of these methods for clarifying the substantial uncertainties in the existing scleractinian phylogeny." />
<meta name="og-title" property="og:title" content="De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2441" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Many nonmodel species exemplify important biological questions but lack the sequence resources required to study the genes and genomic regions underlying traits of interest. Reef-building corals are famously sensitive to rising seawater temperatures, motivating ongoing research into their stress responses and long-term prospects in a changing climate. A comprehensive understanding of these processes will require extending beyond the sequenced coral genome ( Acropora digitifera ) to encompass diverse coral species and related anthozoans. Toward that end, we have assembled and annotated reference transcriptomes to develop catalogs of gene sequences for three scleractinian corals ( Fungia scutaria , Montastraea cavernosa , Seriatopora hystrix ) and a temperate anemone ( Anthopleura elegantissima ). High-throughput sequencing of cDNA libraries produced ∼20–30 million reads per sample, and de novo assembly of these reads produced ∼75,000–110,000 transcripts from each sample with size distributions (mean ∼1.4 kb, N50 ∼2 kb), comparable to the distribution of gene models from the coral genome (mean ∼1.7 kb, N50 ∼2.2 kb). Each assembly includes matches for more than half the gene models from A. digitifera (54–67%) and many reasonably complete transcripts (∼5300–6700) spanning nearly the entire gene (ortholog hit ratios ≥0.75). The catalogs of gene sequences developed in this study made it possible to identify hundreds to thousands of orthologs across diverse scleractinian species and related taxa. We used these sequences for phylogenetic inference, recovering known relationships and demonstrating superior performance over phylogenetic trees constructed using single mitochondrial loci. The resources developed in this study provide gene sequences and genetic markers for several anthozoan species. To enhance the utility of these resources for the research community, we developed searchable databases enabling researchers to rapidly recover sequences for genes of interest. Our analysis of de novo assembly quality highlights metrics that we expect will be useful for evaluating the relative quality of other de novo transcriptome assemblies. The identification of orthologous sequences and phylogenetic reconstruction demonstrates the feasibility of these methods for clarifying the substantial uncertainties in the existing scleractinian phylogeny." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fmjkTmd0FX9YavoW5phXiHPP0IF8cLh79NxNN_Q_L3w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460165","apath":"\/ggg\/5\/11\/2441.atom","pisa":"ggg;5\/11\/2441","ac":{"\/ggg\/5\/11\/2441.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2441.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2441"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2441"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2441":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n5Xr1PELXheyTv9ZpaBQatwsskCVFPXFO3KXnj6QDp6PqVO6B\\\\\\\/z4CtzZBxehDA0JS2itXslrUX5B6z+KwFKc2\\\\\\\/5UuZYyv4p+iUOQbrGUs\\\\\\\/jfdRJBZ7gBvcUb00RwsFwxNUlAP2Iz3NWSSqmWOJbp2fU187Sz9EFlPwWWr8fPktn7kuBhZEivHOFwjuNmTWzBzVkVRBC94DS9DFqQCnRDfu83SRZcxo5\\\\\\\/N8S1auh7Y\\\\\\\/ST0GZ2nbzpBbVQ4SeuShIi+tMPDjT6yHiefitwITnmIG6sBX\\\\\\\/yPigy5Lo+7b74Pi1oD+g+G09BWFIgSHOB9vpGlcyP6G7vCXdh8oMW33d3XrEXep+3WWoC8DHnZ\\\\\\\/XFq2iTFO5RvLYE3tAtHXzidHvFiAasbPRDa69W1Mrs\\\\\\\/0ik+Pi\\\\\\\/rfHFx4XNy+qLr9dWmU66fuWfBwvuPYa7hbGUrEHUHtNpsUOOMb2v1EUYSLiyFVwiblL8xM5Jn35qNKpn\\\\\\\/xrYOzKEYbjPcYRby0ziO9xhpWx5X3zj0nxRweZ\\\\\\\/WDgmPBu9ymp+WkJTjlj2K6Gg95GMtt\\\\\\\/xSF6GzwOdyUdYPInzgz5QF8qpmgeEMCcIi6+Qc4pitnneqM2gR5yaur78Tvjped6jbdIJ9jalWABNWrUGB93b+f3ryYrOjrF+MZ4DDhabvbBsaYCWK6t9jVlRf6fOhCqXr4AwYEj\\\\\\\/DAOnfGmSM\\\\\\\/zU7kThhZQg7GjOps3XapnEKz1M7BackKb6sCV9BSIWdcLHeXRZH2CbNr4d\\\\\\\/\\\\\\\/zRAmNrplA+Kok1Zv\\\\\\\/WcJnD4YVP17ERl7C3vnkk00B0mKvbai3bo1NGhbVBOhYY8II2D2nEYO0VZuUTiOSQw+\\\\\\\/YF4eFbQWc1DfmZmyW02tBRj6bxpdUNd2Asx\\\\\\\/cDUqJwGU\\\\\\\/+hhHB0KTvppGPSlaQ\\\\\\\/UROZ9RwehE9H5zNkmpQPkq3ioPBb27FQ5y+gTf6qxUwXZo8ySWBLMRNwO9YAUJZJtfage6J1Oe9dD5GSQRkfZ4khY0j\\\\\\\/AXv1eXmEo4SOJDywbWSQODgj+mPL2rJDzJdGlQTBB67Zz9LaiqelfEu1sWRVPs3vLk8BhtcXgxF\\\\\\\/RwwekLQ4gFH+TJOabSld8gZOh95RN56PZPmWVa2eZkhPPEc9L4avdceIP1YhMKrkE5yZVMyw6ZFSiBEi9yoVc7LQU9h05HizVt+GxMvW5dVe0WVTBTgp1c1pNfLkLt5hBtZllo6y0OuV4lEYaNjJqOM5Vd1CWqytmYgxIhIjI3UiwK9gBoKQCFSTjquGjWbm2\\\\\\\/I7wL\\\u0022,\\\u0022iv\\\u0022:\\\u0022N85iH0AvvoFfe2ykXTIAjA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002295acf380857298a7794046bfceeabc65f8eced5264baa6f758441fd912a38204\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w8RbBviHANSrLGhvP+nC0Giv4PQno3VJ6JJu3P\\\\\\\/GIJIRcTjdXQxWvS+KxLC3jeiBTgkcOi0Uv2JQK7GX6YnFkL8RYoWxahk3jex0XrP+Nik9BHBM\\\\\\\/oClS39sBQuiK\\\\\\\/iUFsv4DkDyDhkL27WEv5wzzhu9BPXH4jW3ZJXFO+j671WLf+lT\\\\\\\/FR45eG\\\\\\\/0RstNibD\\\\\\\/l3CprJY6kAXLUKK\\\\\\\/nkPc+H88fIKTGHxm8XIkryZYOVco3tn0VXrczJsvpFWKGGXhulvyssg2GWN1Lieqmvp8KAbYNmgsPbuJx6ML78\\\\\\\/dvRZT46NJwirMrlPFnyeR7r28i4BRnTRUfHTXXkNJI5FfK4mCi0Xky9ex5GW+uHuTR2KCYBUfKOk8q6PmmST\\\\\\\/kn5TvXzlH\\\\\\\/JVD36V4anx\\\\\\\/CFFJ394a2HeOKcWXSYhkUMy9hFOiAqcp3SvAUxr5D2kOyhI9qB3kDBovglxttrm3QsBVudUY4bffFPA7rUl5TSOBI8w3wFSOiNbsb6peEO6vmdUSr1j0Hj4MekbBSRWwvlIYV3TBHD9E97XP6ku049qpWPXBfgGKEnbiHHrTABeZzY0BGXtE\\\\\\\/ochHUgvE\\\\\\\/vNoEDcTyiDZJ3HbiTOVWaPobPmhIBC9uvm1\\\\\\\/\\\\\\\/+Mvxc37nErOAdl1mXSOV2TRFGBILJ+Byd7AKZRmNywW5xd7vdeXwYq1gWQ0c3AkugOZwgLvqyXyRvDyFqk5kmpXXzZPDZfJFiNwq\\\\\\\/wDSlaH4UO+Xoe7jy0j98YS3WaK4oZ9Vk3GtKy6+y5tnlgxC4uIWOFEcukqZpdscrgJK1T04TVPfcD\\\\\\\/s5VJvXyYfVov0UofyuWSUP38uort\\\\\\\/FAB0+QnNR6vRngWjChMHyFOg30bip5gg6EcbEON+\\\\\\\/8hkVNBrjPP7ASqEJRrV3vZ4TwcQOWuBp0emH2vgZt9zGLYrHeS6z0LVq1VB9wqh0LWJB1n8A1AELly90UsdAYwMEuGhbt04PJhXzQM8lPgHbSPOUkZ0MDczcTo5dbD2\\\\\\\/3jyLA0h8r0GbtHBqQ8P0pjGgFLEOhf0JMsTDllZYBQEAX\\\\\\\/K5W5p7PAnjPj3CAJmkeQQqko3N09itNaxQ6oc3NC+NmbaIY7vzI+z8Asu+j7kfDhs0qORa04IJpwVXqvUhegx\\\\\\\/5QhwnX2gxdcsZ57q\\\\\\\/L4shQ5hUB3SwqnEuxJmJcx8o3opyO8s0YJDPXIrOlFg30mcZ+SuvRVlehfZaqOLRDBkzeOFJPh3dGsuKTlEOJQGajd8Q5d+wxduoXwCRSoi+CxHLYOM08YtB5Hsh4vmilw5iRfzTJT8PNaq9nhUtDXunF0ijEofBJAvkcsQCsoDm3BRbJIWABJDIqxQn7G1o1gFz0A8EMtEllMzrHPVteB\\\\\\\/7Ro30HXmoReKIlny3QaVXWb+UFd\\\\\\\/GSno66XUF5jWWCrXsMNTjflPgzvhgF726khHPu8SzDz6M8ktrOtuGK6WJ5LIQ7LS\\\\\\\/HpNfI946XP5vHQvla5LrQy8E8XGEkIBQ7WPPT8WbtrArKCRJNydnA4lLlSNnjzYrzEC\\\\\\\/QC6bnt0Lwy0USfMeu0dTakke6t+QTyrA8JiH94M07M+b1PCi9xKrBLUp9da3Hbp9rwtHAbszuZ6oH2OCaECuJwvkZis6DIi8a3TEpWQjZ6jCUtW26eg8FQ1s5w4V2OOUDaNYFufC1LxnyhZt\\\\\\\/EBMARas8ZdXsC16oJiT5n+YDwaU8yPs7XZ5l5fqo1TJpkf2cQTiPITKQawWfsn1W4ERhEzJfaJ9MdPUMkvP1lKGDAjMmhHzhw2UC9\\\\\\\/HAVNvxh\\\\\\\/oJesJVx7DuJRbfY23k2DFnEsElxETxyTt\\\\\\\/1MjuWb1qIQfCESZaCFW8EPhjpmD\\\\\\\/iiCn4KzMPfDcfcw4MzjZDstHTZW8Zyhc7nbX2p3mZrhHaLPB\\\\\\\/4dVfl61qdIC7JdXUsEikz2Z0xtqjNZmo1ossqrDH4f0lOEOhOAuTen6QZjQJ4ri1Fzh8V4Kk5zT2KDbmzD2bFI+YNaMVdV6l5ky\\\\\\\/ZlF9IXEeVO89lDri\\\\\\\/8BlDuI9aRMsUN9JzAiHEHXgffkS3YAbwIYDxxUqO2hxzfxZd2Axh4wcKPIaL9aQEM7kubrYtLAp0YcfpZMqdnxH4PzozDFl+9EnQ0BUb4ONwf86LZdCn0lMLHGV8knanTh6F8bRE6XJGu6b28iV6HjH\\\\\\\/RCRVeTn+RCeFuGvERfbMYLpiz+CHOfoIYNK1TlzcUJgqsmiNdbQ9bbumvqjmj4jLjaNPRuNtvIy07iY\\\\\\\/0UK3hFWwzVFFv\\\\\\\/X88MCrphNEuJ2TuAe+jHtiPgZ\\\\\\\/M8azNRQmqtpXqx1tZFaDRt6lUSgM6KPgvqgqiIGyJyKJ7oNjOa2ciaHXbAmlS9u3EvIOYfjRG769uc4\\\\\\\/q50m4Hc0pkKoMzlWp9Wq68WxnX3xui\\\\\\\/88KPHG4A5t4HBrO4bDHoo7gFABdLOuL+3LZLUq8YOU7RRALF4wwLRzWAebv6I8XqwM+h8nOgAjopyY69Okox6bSd3PH7C6Hw3deQMR4qA8u3lCzOK0kN1+hvDOWwKeqViFT+\\\\\\\/s9mf5YaHs4QhopWrTXqadwEPc6OAY95ODnFVj9xddPJIPYCiTsiRCWf2RXNNp\\\\\\\/8TuUZZrtj6H3zVs9CXwpHCP+D0K+Ewrs8l8hQlNEynAlbjvVjvS\\\\\\\/+7WpFZwiM836d8LAivBg77fP1h3gF2ctozXOe5iA7TTzdh1b3tZy1BOqgElMiF381vwudrAVWmPBYlagcuCU3WP\\\\\\\/7AoeDJdIBOZB1gXPAli5wdffNWbcft7nynuQw3T5G8qm48FzbKz1WjGoOMSLb40plk8WSCj11I+zdvLwj1mtCMy+9O8cl2uQD6RvO7ExPQu8YKk7RqfUAsnLYLjGF+VS44dLbmO3BUfqzXpvKVGiATZ1KJYvdrVw9qMb1mXVaQ38yZUtqgoXPpYK\\\\\\\/hyVHVvq7ipliTv1VhHCXQEHO4cDRFLT04RqKwe+hax0oA3kYZwXyYHCMaWXx7yaPGNHslFIblIV1nG5QX5osjarXZ4WrWbBsNjOU8FQ9qx7ieVwU11j6TlYAqqcbSTIvAl9DYwFZWhwvZU7LvPYB5lNd8+SSJgZAi4jF5DXc\\\\\\\/TyN6bvCCttEQlOQKHSbN38lK+O5TTTjPQ0ogvG17ppyFy+J9tEO\\\\\\\/+1QHSVWeBZk1dahvSa6LLSDLR2OK\\\\\\\/3OYl1qkrOIu03+lLCMckzPaXtmWLXFgucOLzIDbIzfJBMbRAbrkvqRWmNm+S9pC9agwhZmyJkBpQZpbAXTTyXgMp2W0G3vth1vp3sMF8nfw2yccDpj9j3VDuODP4H\\\\\\\/IYIkIdAz9g+3kfwEqKakmqRwlFMpX0KpeiMU2+ARV9o5qwAqSUBSK2JP+wIEpWcc3q9Subi9Rk\\\\\\\/+1y1btSdWz1TiDfM4tD+PayLYiVOCtLdpJT+cixeTSAO9UO2c1z\\\\\\\/O8qI90skAdsDkLfNIyN10Y8z6HpTTIdlezOWGuLD\\\\\\\/EFqnBCgg9FUSRhljwsPBXN9X4EC0LyxdLKb+dGlic6YdC5GvOpOOmavmjqRQ++xtIeKc25WmcZ87Qv19GKYgEsEEL324xdyCa2lsXm3SMW6ZwIh07YAkHIJAPkoAIYont7c7OJZvpjeA2zvSsrbJwrNwAezO9PfunWY3E8VK92yrep0\\\\\\\/Ay+G8qN0SwnGgngyTmKuS+m1xwPuYdCqebHugDPQJ7vBtPfDSqCzEnMP6AI+GZzegEIsO6DHe9QYHVr2LyP9go1qfk8IuGy9Qy\\\\\\\/9bHjUs\\\\\\\/ZDAbCNW4ZvodL02ZjnCdgO364CeLgVJyllx8AskOS4DsrXHeYZsYocNFqJo5VhHmXZ+yvjyt1QTRTBv7lLvZm++TGg09IkSPI7UHK+dhFvRJe93QuKFaT36VESq79grMxUBhGQ9VLdwrbaxMbMcTnbQgaQAZ8fSlEoXnA2laHYDTsr+r7pFB5Lk5FIgBvswTCA2UWrTi+W\\\\\\\/Z1p2nVBzPcifmsORI4yLP2LTIiD+LISREawOMlkG2hnzUYatvx5C7+g1WqUVisbB\\\\\\\/Y0bs2OZ89AaXrROiIFDSGplLy\\\\\\\/Hgyb9ehef+P+TEXZoPhy6h\\\\\\\/aIuCj0uoCBXLfX93RJGs7dSy1vYE67iNJM3Y5koo4sw+eqspJ1s\\\\\\\/zWHYxoPzTCHYuUB49kakYOFPVx1KW8Qq0ehC6z71k6+lDu\\\\\\\/keA67foY9MM\\\\\\\/kCO2bV20c7B6kx\\\\\\\/fOCYQ9InyriT8qSZ6xoyyxaK98\\\\\\\/ocb7YAxbi327UaCeV8dpnrxejkcCIvKfbMc6e4hi688xDfpDo1uNHuHx2L\\\\\\\/LPVoferkwYmXKoHYFwe0he6o4M1eSPJ6MRbYWZqni6EyH9FmcB3jxcoDR02kb+0kmDrhBW2LgwjkiXeBkGHvUtR29q2NlSlQoD3b9bhWcFCnsBLEdurMlosqgVIT+EYAT2M0skAqjc\\\\\\\/TnJzz+lG5nMRveTrmO01hAHqYnDrxZQA8rO\\\\\\\/sFey9TsvTd4oQtrBeGJLznLjpmJl7cFFnPpdlWzgeG\\\\\\\/6IgwA2kCrbqf86XymzNEQ38f1UYn4GxUx2E9oYDcjLQqgXN6X33a\\\\\\\/un5kaNsmQsk\\\\\\\/cUXHCEQJUC6RIlUFLhl0AXq895F5aGeLpi\\\\\\\/fV7R3pyFRztUx0iHdf81lY5Xro79c3jN+t7jKLXdXrxKltVwgxDwmN\\\\\\\/t1Dimrwts5KULi36hAO5n12QsglxQqNHBZizLchwwTlDywredqDGqswv+TLV9z0Pyd04hVn3R1YEIJlRuQo63MQ+CKj+TbjIFVGZvsSBJr7gni\\\\\\\/C3ixhlfogPmIXS7uIc2EuHxaTE4O5ejG+xG4DeyVjJ0m5kYwD80GFhLYMsf+t8REu8Igz+zCfeGwErngHt2vEzajEHKgxHE9BMHMgKowwxhs1vrGVNaUR5J9TLpbdAW+X5ITuYC8Lmz+V5TnlcOcDkjXvFwKrMJN49YxXD8UPFS8zds\\\\\\\/VlOhNtuE2OxPmnblsTaU4nWvcXS\\\\\\\/etORIUN5lXyqoKMzqBn\\\\\\\/Esh6gbxENNVZGVkQEe8zv1lpN76+m5X93JRLtKEIpz3ch2LTmm2pCi6uUY72N8Aeu+1mrDEI+7jFmKpooY5ubawjBJKbA52\\\\\\\/myG7LXOIXtFV98KoI1UrhdecZr9nh1g6q5X0e2sbvEziq\\\\\\\/27tj5BxhFSQhU7NJn0X9F818Tq0v+Ss7dMPnRI4aXG4EG8YAumBvefyR69Vw\\\\\\\/DW2DLSP5mS5hTI+SuiUUt2jdmER+Wfl\\\\\\\/F5daEzDyjmggHJg+jh2DfuwdeqFhI23\\\\\\\/V+aa6B0wLC3bYGloXLBSVoChndic2MW5K+0BJtsO9Iz\\\\\\\/9MJAwlcl+rvuX7ZEmSnC+Ma\\\\\\\/FFLkV2vCkHXMoY\\\\\\\/uzteBd\\\\\\\/fAvtk9nGtpDZ7B1Cf3jGyfZXTQ5zhOJm6FV8fRM4elFphwU8nqv69o9T9GzQOj3Pcdtpx0SLf4SEEcTixYPraVru0dPjO+vfyAwzzu4jPz1WgQTxZBRNDBdY0oCR3Q5mHFh4qd5qgab6f0vOU0MSDqCMpQ7BvSAcXiueXxX5jE0amg1IlBX\\\\\\\/iQPiG2XngvJRNiLNZOKacEDPTJ5Qa+HqRLobE0wYks8UqsMh8BexMt54KYj1AtsCvvzSEOTXTiPNnI5UjW9P6s69s098XQsiSAGcxlO50oOYx8nPI5pLbjGl1xIdlPL7FjopCmyTs3Ah7QHYUDtR\\\\\\\/0PTyYpcI3nn8hKDUMNXbn\\\\\\\/LTI5EH7hm9K9OulLyfUyrBJVG8OfAAhLiAT5NbeoL1weXottyLtxxgLAEyHqlNt\\\\\\\/fMdJsNq5FwscgupBlZC3mrKbb9GN+FoNZ9J29ox+i5Azvu1JZhB9OfWwDjgHJ90LIrsGTaQ5h\\\\\\\/yYhwa0WaQdpjQTIMUsbhWasv8UIhwaLMj\\\\\\\/JMgA2nwGK0pOJTXU6J1xHwXqDZeJXh22tMMbMaOqMhHf2jpUBrZx5J9ODFQFI\\\\\\\/z6r9KaN6zKRiUSbvtOUtItCcUT0lD1BGLxqVrUKPsFkAmXULVuMjTu7+3zxP7l4HtCtencOfNB1W\\\\\\\/bq18FEraFbxtU2L7WVW3J4K18X8jLY8kKprvePjS5w163WHc3ruXEW1BhYmCI9nQG+3N4iLE64OcbYPsHfoba757pUFw9Rl9CeKmlDdmPqjCSOSbu3MO\\\\\\\/YkbaSXio2YCZI08qiml1Lhl5dcAogM\\\\\\\/juDy+W1rCvFuB7CJbZVFn75LDoDZhJFFaB89auGrn0uA74tifyxOovqqU2JaOlgjcVFScYa0zYiVXCC2VlbkUHF4W6xdMTwMQZ\\\\\\\/PM8cW9\\\\\\\/bFYPBYz5KrYh5RtHYrnl5IilvWsnbYE3ZdG7vsuzqP\\\\\\\/gjfNTLenK8wg+tiesVMFAYsAeITOc0Mg8+W8ye1VBF1pKaVFEOvJuCcbicCAKKUpPfcMXcewwssdLfhZyr4+WQSkqwWZnlzMzIbye3IMWvJmCJOtrebt37ozGvMiu1vqYw9DIu89xMS36FfycMOIqs3XEMM98n3uXuI5stR45kFy\\\\\\\/B2CSVV63mYX1RYGEMLdORtzg2BHwp4h4Q69t2pWvyDCOAKJaHKIsseImVg1RHQJsTXtNnPT+HaxpIFdA+CjBS8sENIVehmlkfXT+F2OzJ4EsVSB7btLixgC7d2qyGsim2StcSGXj8\\\\\\\/qFJHhpDSveBq4iHQqr6DsaiPL2CHKY223bO8PpqcLHuzFBamVENtpJmFjlL5rcrHIJCV9XmGmZprSGbRRk5d4p8w+o\\\\\\\/b5BXAbxMY+YlpmUVip+2Lqm514A3gA+NIdwLJl9lpmIsFB\\\\\\\/gKHy4eo1lMk3ZG9\\\\\\\/9rH6DPr7cqhSD1dmlMGhSbZPUr2mTq9T4mKyyToR0gfoM+tHTwevLp\\\\\\\/Xg0+YNrPVWRHLGAL+QFxMTUcPOvidNCTU5Qt8QUFKt5Xf58yll5iXfwfvwORG7X4I5YAi0nIpzDeOT37NyJqqkse+8CL2c4+g8+afZ28zX8HilQKccaA3yRMOT7Y\\\\\\\/8kUBRR8XETvumjWSJ1T4g7R6pTIZa6JVxyvTj4hjUx3PomKdqUQvD\\\\\\\/0b7y\\\\\\\/qsHNEnCT+qFLQSB\\\\\\\/1F5p2knEmEcoP3LWvlVl2R2bCQmXx71b8ssRkECKvNBFpNad4BTXyPPGARK++B3ccEXwNXHDii7Zu695iODqlqFibAeoSATiuLnilFwcXo9rFs9aZIxCjujEdvb869EWsaAjkfu6q6eml2CFv7ymbFSspaJY27DIhQRHt2mGBt+m2+F2Ub\\\\\\\/iEN+4whz5\\\\\\\/5UsPZ54sZDvWL9Sbd0tSkemAyjb\\\\\\\/feChBwHNOxFYuyM5Pv9fi3SsZs8p4U+wtGwOL27iV8i8ienHBUcVO2EMVFjN+\\\\\\\/twuDjcbo70ylc52l\\\\\\\/w2mc4UFB6TGpjZ3pN8eCRbNsBIYDQQA4ucm2v8Bl83N7F95sEr+T5w0nZiVgCi9tlfS9nALuLw1HnI+DRJomAUDw0gS602cHo6IV7nUtc9wUOIZOZyRZbWrHCu8FpYI+KkQ4cJeySXaEzqMHANkYn3OKNEwcd5W21Y4Hbbgjpt+tgKOwzdo82SIGrL1O5DThZ41tQzEPWp+CwnW2ICyKj5DtGbSrA0T9X7UrJc6fl4Ewvvw0S39UA2bGWg2j7Vwk4yw10SEogaGyBXnJ79tZsNUt0woP\\\\\\\/g+kChWL1UFa3UmuyRKQ+Ij1x+0vJpE8Jm5ZNbWzedAXf+M20GeQkX4dahj8MyeeT+zTv4DG4Mpqi5Ccorr\\\\\\\/Dd0wmGh37xovB9XW160v4tYBl+GT\\\\\\\/qy83Ei3kbuPrGasLozj4e+OqUiTRSxHMBa11rka1oVScZLP\\\\\\\/OUrKZkoM7RgnYr3dqqN0SftGjLrRU0d8e+FDOTtdKSXk3MwxuIfOwhVOnCslxonSVYnVfTzAJbtwRoOf1o9ggPfrIBXycbFamUJs06P1vt6pwZ90P\\\\\\\/QsGR2so0ojzPzawp5AiawYQNcWwohs2M5xQns1UBQzBGLgmf+TOzn9nN400axTF+79CZQU9nNJGtqHGsTp9Yb5\\\\\\\/y3HoP7J3B8hVAa6edwNEYOoW\\\\\\\/+9T7klSBMmJYk7z5h9uUWtwEk9iwEDVb1j2hBO95SDjgtXWNB2b6wrGGjvorhFnqb+twwa5p0urlil\\\\\\\/KIaHXFI7RtaJwiuJiWakkT9BzscQFIxyXRuHaewdzLRlvVmBhZdpF+wlyPfNmiaUCir9aytXeLQNlUeEttbTzj3\\\\\\\/3p67\\\\\\\/96o6SeAZs4Yt9b8CBtlE\\\\\\\/ddr958OP6HBYxTHkaGletJfvVrmPyZ2M70cBo+2gZ1brGmviyykxWKUCebSITUz3hsDstFdfqIMasIlezo9ENa\\\\\\\/5USFRvQq6OALdJUm8Drhme8ECfZWjIH4hN0yAcM3D2tBVTvAQHdaWq3KYR74nIm\\\\\\\/Id8pqqBTJnrptX\\\\\\\/E5ab9fWOWHZqmTOYPzfLx+mEFrVe+B2ZucvzI\\\\\\\/RcOxHa66q0V\\\\\\\/rqYMxbsAoAdRbYKDYOcg\\\\\\\/JoU3gG2bOUHilCMSXoocZpT47ohQpmGD1PK5R27Q9AOl6f0hW\\\\\\\/66UN0oXF0OVZ1XGTRD9FtY7bFgPs6+88c9O57DzVqCb32Pmea4L8gVF8G3pZ9+FX36lczeRkvEcLMkk+PyhNpQlBrL12G\\\\\\\/6QO9ZPTDmzVPRGe4BS0MbWhvlYR1sNdjBW9XElcS9a4x9q6lyG6P04cSFfDBOUz0MJQt9uFray3JIulyprti\\\\\\\/q7iGNvK68Ei8KnWV3nwDHoRJPqDo+3wHPSC5McWvFrc7c2V0f+zwJQKCZ0CcQrVd7FbqtF4\\\\\\\/yE5N7lSCo6nKs8hnNPsKOwdi7IpAezI5KvUEd6mpXp4rkXTwj3+xjf3njrw2wyBg7qZBiCI6znnCfhbccstNv+y9DwPYEbVFUk40dtTZDB7MOsh9O8vp8BmRYwK9+atWGiHDmOwjdNtpMReHGOgKWz2DtC8QB3zbzawttq0Fx1lSuk\\\\\\\/kyrn7x+EYOC4LgdQeTorh\\\\\\\/HFhogq8MqaZnu9MWK7xYeVrw0E5SzwtzkAoSWWVfylGAuBEh6D8Qihuk0rzlL0xl+yT3dH4RE794Gr0kNx4SIYfjAy\\\\\\\/fj0J+c436fS2JlxlGsZrhNqIpGaBXbpX3aqtq+JF8xcq4SChrcczYjmP\\\\\\\/1ek+U03l0On2S7bJFCF5GRx\\\\\\\/k8J0Y89JYtcbO\\\\\\\/mbNbZm9hlp2QkLWX+oa4fLMPDq\\\\\\\/II9KGwta0UXisGBYrh2dUyPV1UjF+\\\\\\\/2Yphegp8P2HidKvYE5TdunqfXlY8zeKAIt3+31E+bhHjiRyJv8SzOLi8\\\\\\\/yvXuXfHjgQZHb45GDi4rxeG8s7RxsL5VrdC6PvcSGFvnI+AhZFRyBtWoIGVYOe76PQCfeP4H1uKMrVBpUnbSyLmBKlRrRZdCQ2\\\\\\\/3+xvirX1\\\\\\\/NlEr2z9ThojF5b3Sr2t0Q2NuUevwPRWV+Levz5ZW5Gvh1MxffWJprl303LYBu05jgaouJfkyrM05K1xLxb\\\\\\\/nQ2SHp+Hnvzg6nI+P6mDAgg0o3Z0QnUTT9r+wB8rrzfAW9wQRpigJWCIyd4l6OvXhdOCuhwBbAgC0kiGL8FZsi04GoehmeE+b7XqtjDiGUOaEemCHRaCgTz7URB+XEwIwmXvl+oD7GpXjyU6QlQrg3ewJSiPL5KjWfFVNfYiOpSiGLTsVR0x4hRAqgu\\\\\\\/GnKSOt6niu9Enw3wdOqqgkYqosShoQ8LHdd3\\\\\\\/8ioMqMSsSKUOnFUDXhnlvebFABu6yJAu6XllVe7kIstSThikhmBlbC6A3\\\\\\\/NegEMG6gtzfqY3ex1051M8gPorVjmVX37qqOa0+TdfqbhAa08I5jFUYhcioY8H\\\\\\\/InruAYO7OeFC082p227UwOuEZBa1cmESTwrjQFQun\\\\\\\/+mKbuhX787q9UMw1cMtjUoleQSs89ckuCugLWtU8zEdvBSPzCos8CxGD8zQR1nmQ3f\\\\\\\/WITz4Oihedj5XaNzXI1KHdVksFVyw\\\\\\\/+fiFFkfvnWm3XluXkb4KLh3OUYrCbFODsmiKAnEdKp6DNOh2KCkv9okVwedb908XAwbDSiwqkx2rqqc+hN6a1xe7ZmHsxJDqglLGt6zUXmZKg1tYQgEXGa4MdZYsJIvOv3W37TFec5WSOKOcN7er6OwICiPRXmFkErWi9p3acjCLVQ8Z0GsAHZhVhVJrccG03rtFbrlU87Ps02ZosSUQDVtOqgie8SkexNxiYSrCnBA2nbnVJlQPuTMNxJRIG\\\\\\\/Gpi7sLp3MRi8XPBX2WkFk18xhQx5wdexXT98t8IysXRH3HElEr191OCrOPofuFxdZl7ePHLzMCdB8WrLe3lST0+VDU9WFumGYwrMCaZL6E73RuUaO8RWCTzbsuE3cwKIz7dkm218cmsSvduDJyisHmDcrCCF\\\\\\\/zu++iRC3rDXxSdaNtRsmLPowVPhcSBL1x9Q\\\\\\\/3FBVqL8ARJSZAEz\\\\\\\/Mm4hGj6hOiDkzOQXHzRfuLIHyCT9WZAsF0BkUkLNoiSRfBZXXczAKua5kfkJO0cuHD10fhjAPAFszOxLwPcOgZsh\\\\\\\/xV\\\\\\\/GGI6HnGLGoL0q\\\\\\\/6bI6bLEiuBbNhTShPFh6JsB4udd0Z4f+M+19GAVPjdZmI5L76AEuTqaPfLK5RMyHINw8M41FKqjWDbvPJsyU0NFXXnLcwW8Z0UQb2nvMHJVSZAstQu4oEQPC7DdeyUR0C8hYnNBPwCih5ZhjfN\\\\\\\/e4L+NLoO50j7PHlQSUu\\\\\\\/\\\\\\\/9vkZg3VhB1HmY1hguwsqz9QiDNRKu4AAtr+78uVLraBSMi+2wYp9Z+8zYZm+ocf4oGjXyNlwzzGywSQ74GT1NAYyKhWsh+fqETwnZCzbOocjIiNW8uvbajy8KdBzIoh9CWc5AcYlXVFwZuV4oJd1P+v6oGjirJ7GJFTSCqhHXebn5RKle8M+QHIhTbe3o2QkPvRX4+hFNtuTV3xUJV\\\\\\\/sCEi3tSsQ4LMwCtazWB9FvpNVV6\\\\\\\/t4IjxLAXtorh0mUG3rCP6vfgLFiUwg5aWuuqEJR6H+gJtOc3QRgzshjr0XYIpLzzIY38PzeREzBe73EumVsflwveMVY9ldjEAZ5EGWoeCY3dpkxXw84xKazr1cHYOp7CTjouOTobClRLB33HOkHgagfLKhvEI0A4goPLrOyCKVQVLXVzmoI9z45cYLYEp7T1CPYEJoCMZsObRpd525Rk6ZwMvG7+1eOK3QBtBeIwmZiB1P7lSt1Dfy8w5m\\\\\\\/A4IfMlnt\\\\\\\/ANWzi0ULG4OgMAl00hpZAlv4yZm7SkUwsRDKy4DvWmVg9AjvYcYdC0Hhat6yAHIjJ0bwDzEqbFmMfpUnWuMb3q+b1dJtGNwGASAK+KyE+ENd\\\\\\\/cZcx2B2sNgWZzNzxhN3177eFRrU6ttbOC52RnIbm55a432pWCW7hIe53PC5kxGA8ffazmfrvg45Tc3FUYSVJ3Gz\\\\\\\/DlWSLxWreK4k4XQhEOlMWADQpACY76hhLVU3UCsbszourbf4kYD67zwh9emxHPLwmEw6j+aW8rXDrUqnoAAK4+IW+52MYcKkOoA3m3DiiYgDypUxb3AQEPStoZ8hKRvKyt8QDTr47cLcHZUh5OqIhZTJ2K53g0TalVf2J90Zp59bhtxG6rJltwyHOD0HGu+RRITx7Mbh6XxyKxzJLC+Gd0JWI5ChRaSoIRIov2mfRdsuZuQ0gZw8fWXTttmx4giMgTtTvhepi5OjcM\\\\\\\/dmc4TuTw+tvd8CuFG2fb6EVaYU4wmYcPBEFIRGvHG438HvkqCDqxfTkljtPDVkdunEQkYP8M4dOe9YmEANWtBEq7bnN6vQEb0y+RIHDQ2KJDK3h4PJ6F4iGhV1EnK77D+0DGe02Hep7DvON2ahLL6WrT4uFBQcD3KbX6gfLq2cwc4A2t9O7xtyA+du3\\\\\\\/FCuHRX9kpeeZTB7quzwfrkguoQruq5dti1GMtGR5bb+RCiUBWMkFKM\\\\\\\/f10yfA9ghhiQWpFgkRvQ28BASOtwCiqh5FIyIhbKTHaeGBUpT\\\\\\\/RipPKgZzHAqlr1yCBCiKkGgt6nSeY2KmP0u\\\\\\\/+v9T+XRr69cCVoE6ybJxClBoQnonHWFxOUOzgddwF10QyYtrXUQj7xpqp2IKDeahcWJSOh0dlzRH803iCn8TxO1GwCvTPZt2edkhYXHCPgtrqwJ+5xPDfbZds8o\\\\\\\/CzFFxFt234GQwexUHPZ9oSpqSBMhGNKPHX3fEdgEWH2w2ha0h3kd9fyvtnVM82RMEkG0hz5SPPb3tj6LFXY9822IQ6viS+3fE4vZoOc6TkIL3KYDMDcd4VKCB\\\\\\\/Og3EyrQYdI93dsqv+HuH7B5CiffNCr0W3NqejYVI6aqJJB1MrvWSoQZFI85zFQs0q7L9uwGvybBak1EZWqWmfwrQEIQ+5Im4k6SRAA\\\\\\\/4ggheJoRcT9vERAOUTB\\\\\\\/T5GbsGD9rAw2fQh+EdCwHXCWswZM9kRU7yTKywT5LQnTx9j7SZ+NEJtpciQRRPDBcwDjtuPliicDnXqbEkhM5cmUgrkK28t0Vp\\\\\\\/99sxQTR+AQyUdmNAYt5v+PhxdDnMjReLQRylZmSL1l1LQ5G0Gmw+S3viaL7ofJo89M5tjUY2cGTmNmCbadTSFP849wWbSzlixQkqg9uAaSgQ6j6RNUE0IC\\\\\\\/ARqavik9MrmORXuN37WqCXPUJcXhv0+NtzZuBXi3UwibTLT+JH8BILpjOe6zCtvu0jHPhzHTR\\\\\\\/Vbu7p\\\\\\\/71rhb+FigKDrp4dBIQwezoLrSbv0cg0zUm8nr+JkZ5pZ93P1FZcjsuHZJZCFgMVz0CAVdRg24\\\\\\\/Z+xATMPmMMl4gjvkCkZElLZIxXhIstSu82r6Xei3ZJHrMziUrUKl3fMrfz2tgpNKPVzSiKEY9CD8KI5p\\\\\\\/DfEsBOm7wd+pCYyxCXMTZsc7\\\\\\\/c1TWjWqvb7xS+ARZ7556Bx4Idiq9Otpxb2qRGJ3\\\\\\\/dlJ66UuiQY+2rbnBiGmMgEW3hFR0N51XrleXoo32OI\\\\\\\/wQ6cLFv1wsDtrof6k8yv+uM6vdYA9ux4MFmK9g0tVEafd46F1yBPJY65syLgQBflyTu3QuMpk\\\\\\\/4rk4PqKuFbnB+yvmY3WbtGhZFMQW7oxOdSWkWoWUZmk2Kr0RakvxKhhQFPPzLKuOyDDLxFuaYv6LKo5PkgDklcuL4KAqg4umleaMTd+8WpLuL7lX22b\\\\\\\/3HBuGErDS4PWYN7DE7VDoMjVzedgKAqUrFPOppb0JoP9wRWDxygcuEeNxGyE1CEUu8oxnE7gSql8DOzBNAvcK0vfgWg3LPobpSsqR\\\\\\\/mJuy\\\\\\\/321puEl11pNzJCO1zj4s6Y2B82LjdbrDDGmySucQOBNIADYFWx\\\\\\\/Q\\\\\\\/bNuZR6lqDv2xqY6yUv04LmtkOVYGC1dP2zSQbYu8o5LOffEu7f8FDqkvkZcF0vphJJ8n7UbdiamCNCD\\\\\\\/WkNaDvVRkNv3OGG9uJATfJp8jWAt07ETze3DhxIEWG1KLBgq0kEjgpNr91NBLsZPvDxh0dhlEqURXGFz1yIK\\\\\\\/AaNYuNLxnM07+xvH++x6WQtkYqF4nOnJLe0rZdpiRAPe\\\\\\\/STq0kxlQUlrpFvpGl8gd5KdGdGxOnqfLTzxGMBs6QkKLOMYeSbm5ILyuuO5vfS4nBE6xobkdxoKYnjHDgtzeYeVFaxrM1Go9OppwDYYPQGGwCP2YWjlt0UBOtakdG1PjW6K\\\\\\\/RL\\\\\\\/kmguVuop64UB9U3c18vaL\\\\\\\/E8cLcjBg2sK2jt7u7hM82mMTpIycw7KpDxjLGX45\\\\\\\/1l\\\\\\\/r3f0tJWUbzk95BrXg0VMT2HUr5AwFd\\\\\\\/ZaPfvgr8eGrp8cs0evxfdQEnnXp6hSNU\\\\\\\/Mu9vRVLzEDAWuhrAeaV\\\\\\\/Y7JZAzNwj6NslIbDUt6E6ccB90BN2pDSfvMvxc0ygvPRuTZlDFlF34tL\\\\\\\/rt1pkIr0OjuE5f2I9GC0jrBJK4O9r6bg4jCkmC5bhVC1O3qcwcdQw2zWtbN0A5XLTQNQsU+50qPDkFIdFUf\\\\\\\/PKkbd\\\\\\\/KBWa0fKLrq1OxefgkkX\\\\\\\/ccMKCGuYxG9bQjH0kAu1uvga1F73ZfgxKPd2X2vUuLLR1FF2ecvJakIPgMN1VPPm4pX1OGSYv6Tn4aWd1TtdlwTvWqsfZqdA3XbAenB6pZXc2m\\\\\\\/g2Uh3aiYlP43T\\\\\\\/K64YsS\\\\\\\/hkfIGpL0fCLuOilz3Jqawr6vOvPKbXv7bA4p7MXLcJQJCKgNHW3PSo4qe2qlIWwXi7Z1lkIKlIRCg8kGK5TGBnEO11ffwMBB7\\\\\\\/KIEmTF9InzU5vWJOssOgPr8LoGg3Nis7TMdNJ\\\\\\\/Pf5gJ7K27uucayMfuw0IZCxsX14zgp6NPJO+kOLGDpEkFlHcL738iu3L1a2gRyNdyzCKCf7eK5RVhrqEqbSgGEMGDI8e0Ziq79zKptnvxVmEGzSUlaSucK8jfOF\\\\\\\/e37\\\\\\\/RAEf0frDLYjCOMoD\\\\\\\/2J8R8H3YVzTHRhk9A\\\\\\\/9X0lCTOh5fX1zo75Nb2KbDuBCcMbDt3AZBLzpDYx1C15AYycoQFrkDRoQ3yetO4tjmYsuQFKMkuS84XDfjT+A947lGSfkWBqGntgUZEuBiFfcILTXNIuX6uJIhPWBh+nWbX3H\\\\\\\/gmOE9wDoGXTxAjiVHnKP+0X211WGAo9Q7j1CG5mtS1i3aauivmJ5xIGNdqLJez921XZ48gg6IL8WBC+KcpB\\\\\\\/\\\\\\\/R350AHBfMwT9YF2BW\\\\\\\/IMhhmo7leZ0qFYhVUNgG0pU20tMTGsHJ1jdkTfZq7v8MXh+\\\\\\\/UXHIsUzJz7I1oVIjR\\\\\\\/UMtsBFT+delDyOA74IRqti4\\\\\\\/5\\\\\\\/6lNebQjHBCrgDsu\\\\\\\/OfzAccVfcJ1cGvy4mU2NDJfRK25Eh8HRjN80YB035VRGWxVEGSz0S5FO1SKFBTE0KMCbpv27ETpVkOxRE7Ytr\\\\\\\/Um5sRrGsxq\\\\\\\/RFVQe5Kh3YdTtw9MfyqAssY2ilRzdfw7N4d3neHhM4rFIrcfePJFhiodYM4cswdKEReRVK1+N1XT9Mu1nyd8YuE2xMq3Rc+aErvoSD89oFpJiyUgR9+581B3IbMsqhxBXf8J\\\\\\\/fHH0Eg1soMJ8OL1TGAI+o5ypSakF3e2D4DUStn1jU9Vlzty2sbU0lhyCqHUb1ycKmKkFdJJgLhBzZSauq7tWZF0JHhIxG3cZHX\\\\\\\/Jw41D9eeoHBeOt\\\\\\\/FrsuWk3wq352FfblSd6tkurGSEulUxcJC6ROa0schbSaoqSSw25Ow1bzYn0dyLUQm+VsQ1Jx+cfQIX6fIYhXRnfkXCMLYKjb2R5DbugG2yIyaSghZCIDW5FE1kinYw79kQsX2vhDfpCtk40\\\\\\\/gjaN12t4laCPQUho8ZuebMSwi7wDDmtLhGhKDMN43XGDUvJi40cT+rlbWNCWXRr9l\\\\\\\/ai\\\\\\\/tlovZLmJtbI3budeeOnin5lSO5HaUTwkryB2HFlkGBGdgRUf03FRq5pBa2+79OZcLcX3QF3AfaNmCZAFXUJLpJPs9gWQ1X+B0i+eeruLUZt6qLHGYhVkGNucvL6HZafJzUlNn18RoF01Z5Cwfwq2dTLBoj464aojXhGJesOD97Kj3Eps41s6ATYJIhB2Rz4un6qkYUSgg9M+zLis0Jjd3XGU1yGr+GGfvKpqMC\\\\\\\/URcYA29BxWDOylaUqzKetgXE6RevaSTQI\\\\\\\/2l8fjSVaGaxfsBjIVd5rjgUdCjEuDVWoY0g3PdVbKCFwywIKN+lPftGmllHW2VMd4M\\\\\\\/QdhAkBZbtjnc\\\\\\\/JGzkGEkHMX2Yzih4xl9BjvDx\\\\\\\/12RZizRwhbvTLtgeZuJ+lQYdDPKgkgspcPSovvFl\\\\\\\/RaUcX\\\\\\\/4XxF+dtga99HSSKTNWhRQ06gQkg1gebP4H8XdcNz27ojkE+N+XGLbF7gqOdHeEJqerX8q54mcqTJDq8i8U7Qt0kOxrsQKgtTh8TCEAZ\\\\\\\/uP6OsG4GA6cQJfrAc\\\\\\\/1vOydMiOYZZZIHR\\\\\\\/oTWv64cAoT0jLLQuc6QFDhl23w\\\\\\\/wbj8TESBONzo4Y3ErPbt4+oGwPBXiZmhakWXR2zNT29QZooK+RCbWOWFTOCVW8d1u1D0Vkr0qLHXgVONG1LlJOX0pWwD\\\\\\\/tKHdTcNjO4ZaVEs14qHa\\\\\\\/iyBxL5iIAWn0gnaFCxLXclpbxsrGtm4+i9Mu8r0SARajdnBRF5Q7xvf+6q+EgHqFrfaNSLQnHs1\\\\\\\/pq6u1ed9uU+MG1ek158kyBmdDKpZ+XZmAhoFMXBw57ISaYIgjVsd8MTif3XjqYGE4NhNFnr2uM1VajZl1Kb7iRYKtyVH5SiRSqebyd98TM7PZnVFYzUj9zeed5VKh49a9lnc3HWWFgCWi35VymF1CJ3QW1hIVpbgeQaNc23qd8gkLA431KEheqf+3DOskyeKw6mPpqfEQqXSMoowgn4DGjmATrQbw2hbUkdqlaiR+sygKzxoCDFq47AWiwPkTDFqXC+Hmo58dO90Gpf5DGD6MzZnOAvC+1uq0Ig3bJ2LxRSOv9x2Uxsnhuz7LyxmC++FGnrhSpCltHggopz1N\\\\\\\/0IXsPtCN0Oiuv4NRSRAg0v2vq9hn48GoHuiYVtjGMo764mM1I0qB7H4enXqEJEJ0hz8Bf1YG1L+WHyaMYmAO4EXSDcZxziBsTxV5LCsDO3\\\\\\\/ZpMPVU3IYywOCf4iWz6MClHVsB8f15DWc\\\\\\\/twNEMTJpTxrakuNNsxo\\\\\\\/M60eIxy4a3mdCqM0L8tH12o+vPwVNJQWKdxBs6ueE\\\\\\\/hIiuVZpSDG+TTwdOzn9KdItDL8Ba6q8D2g7NforH0KfZ6OYbE4Uxk8qJJY+6Yx\\\\\\\/xaMDyBpPMMAU4CR3q8Jbqd5\\\\\\\/yK+bZVn1ZKEGn2Kc3TpatUfDULSitf8qE1o9XptRJxijPsD7cmWVqRJjXsCWL6us28ub17KfSJF7D7J1frlVD0dEsYYDxY0EELOyh05Ld19qgCQ2b6t\\\\\\\/tKL7cG5tEf7U6rw3zDfWiHaXIPOHLEsBFQqrUQpw4Ap9CGI1ICmvRE43Uml0LZPZ53VNx\\\\\\\/z6B9MsRx581v2kzHgqEpM+eO2SlfmGfcQ4Vc\\\\\\\/r7aszF3mGGE1vY2K04esm\\\\\\\/gYQ2UcgyuSD8PMmPw43yu3RyOKn84fUD6bqdOzFLd\\\\\\\/I976FN0ZR4wic\\\\\\\/Ulq1eTFnhpHIQUfz\\\\\\\/EgUwyR98kCHxSiPSyh3\\\\\\\/PZwfVOJqHL5AoJlILBnQ9pb8jfTNYZmvd1MCplqg\\\\\\\/0o7PJGT66TBKSlkS\\\\\\\/8JwDeMNePzEZ\\\\\\\/QBjV4VvF4PAmBSd1x3bbvVBsSzchd28kNMjvBGsfgirW8LTWjfJQycLQN05wy0ust4r3L6GzfTIGNhnkdrAztUlmEpXohD47PnjhWk08zuFQYsj31Ny+s7HVYBAoVtZVtiRAClIJUD3LUbJeu16PfeD0q+d6SVVMyXksiPwauxVqpbDYpF9krXKEbCqxKRu7GhIqKuQMv9s\\\\\\\/LcnML5EpUacJKbHJ21nVAzT2ZHG6k28azN8lTYHUQn\\\\\\\/Zl78vkrU8VVpuag6PU9RUEx+e0DGBeZIVkZOkTrgk7hhz0OgrB2\\\\\\\/uQOpXD3LjoUDTvkRBedpFXesi9bJDcv1bjdO3YenpZllaHwq0k394VTJA3\\\\\\\/4CLokC9OS712yx5e3owg0sAGhs2DEytTBTv9UMjH09iCNYSFvXfTdLNW+RcytjnUxAwWYk6OZzyNW9jqEtXemCrFnYKUvEcYaz6DPn501urnK4lcW+eqF4yLYND5\\\\\\\/HBnVAnOuyZyBv8\\\\\\\/nHl9cHRePbpXbggrq6AjlpQVf3NvBKrudL0Kh1CcuL\\\\\\\/mz9arH4\\\\\\\/XcGv3MgG+Dng817CQfGiknjJ7O3uEv6b1Y2DMdZfxEMGaqz8w\\\\\\\/X45ZSuUDqvQ5raNMb1kmYtimr6IaZNQY6pfYC8NTtEx4pJzZBECPylCOpgX0gxZ4AkFffwrJRW2ZJZ\\\\\\\/9vUBJl91YCpmhFWS1BPwgfHeD9hbiLzTnBu1qkA3IEVVNKWRNB5460cFzKq\\\\\\\/JRxFHD4KMZ27tGP+P0v\\\\\\\/6h7iw3zBr\\\\\\\/WVezf8MsJB8tAdD5emWzVXW2l2KjoA7qN1ceRxcY+JFxbe55HMcYCCyPx0ejKfG8rNfSjsakmTJOY+DerzzGy7o4ejnCKrrtmpUJ12ds\\\\\\\/DTBWWXTckjUYshUkRlqxYCM0P9YsVNlQcDviEuowhQMDKEuRp7aER7nApcD8bnCaUq\\\\\\\/H2EIWV7Ax9oRD3SPLsw9JGdVTW6h1eAk1AuG+4SxBtYAFJFkgqDtHPdMRIhzShfqm3b4H07tIVVA2C3UDZ3ZyWzfW2kIUE6KDaDWBI8xmjxuQfqUeHNOuizeu7O6pCToV+aiijFR62nr2HMdQ2zjIFZYnpvuQWZx6CdR\\\\\\\/\\\\\\\/UF\\\\\\\/MuLMny\\\\\\\/pWfVG202DfMghMoCy2\\\\\\\/CuU9pCF1Y+0rNpOSOaL85GoQLLijQmYpZhMKSFY7eHofIS2zKtSNdrDSrs1dVGj0y6jhSoGscc0Z91U2Y5aXVWrPRZwhx08ua19LG6k1KM5AFwAJMFYX0Y2TVysTQ7x\\\\\\\/+T++tqFgST4ytxsrZzO8GWE8RCsrMKgZnHQs8sHhwT3LqC\\\\\\\/beJ8njzIfTd6i8Nv5yRKzyb3MZtoF845bANfycerdY\\\\\\\/JUBexV5ltOo+nvbEDlemxOsU8YvFsp5AbsW00tU92tKDb6Dwsl5P3E+wvq2bHs\\\\\\\/+MhBdCjVNuSKl4I6tPWd1mg+2aez8Kre6qI1UUj\\\\\\\/tz2NsRMqlNN4jyS0DVVxmogHwNAhguVzlIjgDZPvM6Cr2lGS4U3mqu0NkOALiQ6DAg3Pny305QM7C3p3+lybi\\\\\\\/P2qotd2OfvpgBseL619+WKb9YjHNbFFcGABmiLEKM30g0d63wytT8PzWU2bE0UGJv+GUKnMmRGp4ABUNZhCIbk6dSXiUBwX86zJFmsofPK9mO0oKrgDZ86o97m4bObdIWMUQscKFGNdb8PBdtAS\\\\\\\/BHp\\\\\\\/tHcv7sNzZKrGmBzu9W0WZkR+dC2kyvgBht2sM+7wzgrMrIG1oDh+gDA+ttgkVXm3\\\\\\\/0IpQ9T3BcmZNaHSYiawQ8pgqEFA5xTZPszzh5CHYHChlDk7sMpE14\\\\\\\/4OCczrhacOkzDotrPwm1semb9vPQHUOpZXAnunJv6rPvb7MGip3FwA3XQ9iic9zX1\\\\\\\/jBS0QXJcfEk3RA\\\\\\\/mgDP7PYQR5nnNX+w2v9jkBi3oVL+g9V6OESw7cbbLMnjP54kMmuzSCDwyGBwQW6knEX5DNEa9ve0x\\\\\\\/Auq+MR2FCZWaJt80D9eLXsaYsrFKUQOoADhfa+UzyXVy3WEtoU5nl\\\\\\\/No6YBapMMHcgj8+UhmV+3zrv41YHbKaN4NvAJZxBO9kxa+19QzlUa3s66TKZVQJlWlA\\\\\\\/IIPNYUm+Z9W7h7hfSEgRt3uyZE9Edns6XYZXCnZMa4ePvuZj\\\\\\\/wu2+qh5sWyMO1DhGGP\\\\\\\/RhmnaaLEflUT5zf7u8Etya5tIAha9XMThwBZ2BK0\\\\\\\/4N4LoTqQSXhKcHxa6PNPJNdQReVPyAiNTggQZWZ6FB+J4aZUHtZoSr2W2ffndZe\\\\\\\/SGaQs3HDWLom1XcDpqSGhMjvIyQZFed19c83Knsaigi76y42Nz6pEkN3oMmEad8ipdRQFCwBtiM8ChDe3vfxATnCXZrB63\\\\\\\/A4v14Lj1yF6K1voVMYA1J1Ax0a8dSKxc0uorwNPUWc7mcjVwWqrC7jIrvpPrZcr8EnisT4UhPeRpHUxRJUdSelmYmYKGglL5l7TpIlUqJ1hY7WxHQWnANQPv8WJVtPl2Wj8epFd4yYZkrZuweGLqKJLJXmq5AM7L6azKzdOUJDJujCkCnF7I2TFcPVwC3QIPk8sa0ykS+83lLIc2SsaT5Ub+FfvypYbCV2efhE7Xue2hBLizA261VyksEJNNCnqxqdUo9kltlZoBt89okicnQBwWus6pug\\\\\\\/LbNuL4NFpNvyH0yRmf3o5y9eLhzZIfmObbfQSezwdQ4Z4YKcx0Io56KnAE03xW3KonCqr3VAZRgDUnu9fUkn8DuVx7og8mempmMt6nS6JFpLfWnpbfjELgw5bih8YqPMLbyWSte3U4MwwTvnT1ag0F\\\\\\\/uqWL7C215yHcEW\\\\\\\/NkAx9F+p8JD+rgGjxUPAxKl41RNCrZ2J2nq2CD06sfGF0mcRGuzuzHjlnEAxbxhCnMLjzKl5nPcdT14Y1hjO47DW3ja\\\\\\\/nF+ITKJ6eyjoSEWRBHwBlwePlLFCIqpDxcHabUYS\\\\\\\/50K4r04T62jElPx18oPmwRxNfLHz6cqcP7PjLLtPJE7SK6QY2asBHsjsEzTvIYTs\\\\\\\/ODE\\\\\\\/Rnf0BGlF8Ijq0AABSB4tDP0MUiaAIt66BssmeWeGCJNHkbqyWM5VGb4Y2HSl5AhTD00vbkrRcpRwNnOt8YPdhkPMZq4eH7XCzd73VK9eoRcReOuanAWmm5p34KIa+meGH\\\\\\\/4IYEqAEyWVD+YtZ\\\\\\\/OcJ3vIXIAf7cVRAhSjWx0Mqb94Uidq6qd6AAcYhY\\\\\\\/wh9EuWHsMJ6174D9R2uNjT4nIIb72y1ocO087HDnlkW9hSG6iCFcD2k5tzIl05Icsp62s+KUd39lKPIuWMTc0ffWVHp+2ccLP5+ImBkNZueMN8FpuqJJ1PtNwnNoj46Xr23LxpHIOJToeKFrsKoOwttEscNJvEm8YfjfW\\\\\\\/dTiq9t\\\\\\\/zVs3SAFHBZiGfCsIN1t4IbJFaZL\\\\\\\/ZGHfwu22GwgzBs8pAs6oGR1SfJiJds7LRI4FcSOfg7fjq8gGI23iK\\\\\\\/QlmkXHnnQNSibcu82n6n2csNi0TDzo6zvrXYJAeWwWCuz6vzngGhfwsmlgdmQNqErCFUCRHgnxNNHE+Cl4UqDkJRA9C50DHCEvv3vluYq0u+Ouyva9vxzaOBeL+\\\\\\\/QxXKQgsX0Hd23lhyv7M0ujY2a8klq+gfvHO\\\\\\\/mr2RhjEG3ljhJHuAWaORH7z+gmCnQjohYKWC+Q+R3P7YtQcjzGuxRB1qHTL\\\\\\\/y6scUF2hrZSBqiqEToa4aGKMUCZPBC95s9FOEcStvtKjmbZcqx15bvmEPowfABHTvm\\\\\\\/7r5yuKurp\\\\\\\/5OnnY+4NkdyX3CCmelzjmruCUgTI5tOfFMt1zPCT272XOV+GhIDz7NWn62AI0KuNQGsNSVicjzIiZuuhrGfrDclFEo8nTkJzYFnxfzS59ogL++lqxMFpsCu0u\\\\\\\/UEnuCc6+x5fAPfB\\\\\\\/lhYifAmz2UGtzp6gTur\\\\\\\/CMa7uXktZYVAr8QML48rinMO4eu59ab4F\\\\\\\/E920LT4j17lBDmueJrdkykpckskXKEGepfiBw0lA6Yu5TuZ07s8aGGKploO3RtpL8UaurLtBzFWnKtVsC7Veo2Wiro1QIPt6ZflfAreB8twxa2S+3+Y3e0VAGyGMYsPdwO2E8sZUSNPlDD4F41cVYe8zme\\\\\\\/UBLIYRrFPR+HD47PwJxcXrD7kMFdcu\\\\\\\/6HTS7ZCKq2aYKVAhWOqk26GZF1pYDQWZo7NJkjjhjTTJsLnZAa4dKL\\\\\\\/ka6dxAnELGsVnTM4Tc4ch3xAqmNP8OBwd6Po6xVqyXa7wUCRJO9W9mYPKpHGNQhH+CBxfdpXgJC0RYdzqid4UuX\\\\\\\/hwotATThLBz3l5U8tbQf7TIXNmMTMmYsqtPDoAw5BwL547sjJ09othLEvZqu4jJYkE0FyezKqBKoNlbg3xHwzZ8OyahG9htg\\\\\\\/IfgAFJhRbmLjJQwvC2V3iAjSxNiF4pgSAe\\\\\\\/bfxz4zMRNR0bGwM1BT7LjjcTtQsUwXiaA6fmkYsxDmQXBfkqEGN\\\\\\\/fJQQ7xQKZVucFZONEhaq1VUuHfnWjFUn+7Y+r+9kFNTcTOETzqaELy\\\\\\\/R6E9z4TP8ppJ+ac34UChAYNcb67\\\\\\\/GR8AayagLpkQY9wlbAtZ94AALsGDW8+EITKkf3gGlkUaCRnwqdPM+1hhBQ+1dDIJSJyBe1waij4xzGS8ITlLWXIe7oCTM0WZNH4xetfDcOhVIVJDSh9hbvdLEJrrjvEq7toD9Z8uGL4A88iE1xfqIz0luhfkj7wg+AsZiLejXN1SRwZqNMohJLePMduntTkSYkq3KeFDkMHeRbXkEkGRBaNrBnqRfzZ\\\\\\\/JzJ0Ap1aqpNTstBdKdUjIAUYbhfZDUAfBwbNFtB5qj+9Ac4v\\\\\\\/RvNJnaQSCI4I6jhsrZtAtl8dSQlkmpNAj0\\\\\\\/EcHE3XnKhM3eSLJx5aIEHfmH9TshQKq0o39atf+HigWPYVHVoE+M81pt0NitEj\\\\\\\/ihfrQSQb6ce4Jo9\\\\\\\/GGHleaqqLmnUdOF2pszUZUcRxsVfMK+GrYqhT9bdmbSTAPPID7q9ARZF4fPiqEZVgR8DF8RXirZOBpHobv+MmDMvmUrV5DwFm+P6N6ipMBH2OB980D4KvOlNNODTtEwkYPZdP1TbpRU+ZXL3GcICL32pmy9+no44gYifIk2Etftwc343AE+0n1ttuTTsV\\\\\\\/NBb8x2Plj6PJdVlfMaCwq\\\\\\\/E0rqa+Rm5fk3Xz0ynjO4qzABtJ2SA6e724TjFkWqmqQthV8xq5fUNK3wUAQyt\\\\\\\/yYg\\\\\\\/bjTlkNKNnTEL2ZAqZeXMUZT13xWbTdWhbsST+G0gXeYEh4iyuZI56G0tFm6VjXVUkvcKouKfvHI94Gj7jnZ+KniMQL2rMV9ebSgRekVPfx4GCrdJdDUC1qh4Wjx2UUGrewbet8Mpj+fIlyOiTO+jhvmQnLatuIKxounPbmR0kBhuCdVNBTNkfYeh27ol3MDdwtf5jfzjPN8dC+u6z8oKbT5fbtUv48I+CAAT03DUfDFl0pdd\\\\\\\/wpJIVtPcxNVic29W4IBw+Qi+ZefcFmZk4IKm8Bbvg57DkXtQv0g8RQKActtlWTQbm5CQRsGQTgGl39l2C2GLDKny4LcZI25I0Y7IhHXPCwKMGEVmeUVsQVRTCVzITQsltf46aKdOg7yGdjnngmpSPN1aXvS+sCIgEZBwKBQ16EZfVbDxbjaAJJSX0K1Og3vbdpPxVqe1go+RB2awYhQwyTGHtqOlmYW5rg+jfRsdm9boIiGlEnWGMuTg0xKecgoLNvliPydG2M6AD0wORO4yW8ZhlTbJfiY+nWIupo5Ey7bOHHPNNj2M6uOerhWmHQw7eXQ\\\\\\\/LxQtRIf8LqEq34zIW0Z4mTYkos6WOi1rK1aItkJ7a1g3iskTJ2wEvt3sewICb7YTTHU6RSnHhY8YDlVBpPMWHiUIlBOqvmY8hmep\\\\\\\/6rW\\\\\\\/XULKc6+bmlC5yd36RiD5YcEfF+KUDuV3MgnreGDlVueyIcDdtGZFk6G6sth6cCkw4XWobQbOiiAGfYF2PcDubZ\\\\\\\/YAXdeb6K7AOuVN8oftYPZh00W2xBI4EACSrsZ1XxvyWGNWbdllidyGdLvT3ZqwyOqtD0YRxeMXOwe6fekZn+LaE+f+UkuTHa86wqleNs1xh9TzdXEL7WHpUGmZo975qjB3nrVjcYAjftxhkp\\\\\\\/JBK0Qpw97k5aCqMNISpp09lTc82Jl6jbrLrj3UscPYVioSyF2cXRxS4ERJvgnGujN+1Rqc3heJ0fRorwBZp9D\\\\\\\/zDkzym4mXVRoJKJaPX3X2M72fKoTH5bIflBDttzQfut7\\\\\\\/XYNF626nUhGSOuQUD4T6jqhVbU33mbv7sTkY+hdVLz3WaFiMlrmRcxIKlyHEc6wz1lGN2wq\\\\\\\/E80XG\\\\\\\/sIZ8jhQ73ez68TIEHdoFDoJQkD9YM0JZhSWvz+FIJz+\\\\\\\/gp\\\\\\\/GvKNdPoYRewar+LaYUW0e8DELbEEAOXgDvyh6KCXtgm+7AIrYonadHegNhgcATJCH7XXbff7vLB\\\\\\\/ade4xs58GStxN2NGg23jMXZgdfOjFRLI3qqw7X4vChB4bGKb7IsjxLI8xDmrW7cmMjWU90mj\\\\\\\/XwGcXuNNk4a098ZxXaUryLlI+SLE4\\\\\\\/RoseAtro\\\\\\\/knpNlK\\\\\\\/RwFROmPGTvv90RBfaLVP\\\\\\\/Ie06Pa0\\\\\\\/caXzOhE+XpLaH91LzV2LR1d8hG2FOAd7C6WxTTv5TEhoo1dVqI0wL5QIUkLEYl\\\\\\\/Iq3chMAzI3ohka6iBcByUCIeKcutsmPJqe1uxZ7VOb4tquz0oxEabpLaHP8XDvXsvgu7weuIXBNNa9fPS2GiZ+\\\\\\\/2fanCvy5sXQdqvbZSEXOxqPOR7PRY4zck3Ms8bXnAJ4DxKZcVElW9g99PJDEF6n1b\\\\\\\/fmehb0NCez47pbh5Qe7uUHceXSgxpSq4C7WfK6MPsE60nW3fSqu2vixDEw9\\\\\\\/QkW5l\\\\\\\/c\\\\\\\/pibzdN\\\\\\\/HrzDuwpRFUmGJB1tZveIcfEhi+97Lj\\\\\\\/hNEed8MrxcWz1LnZedRGTlh6rb6jAN\\\\\\\/H9YdSYCkuy9xAF7QprOs+4AnUSsl5wvlyM9VA0Hbpgh3keDJ0VyeHakmOdNH3vvPPdJd1WD20bvak9IlNn\\\\\\\/FNs6jH\\\\\\\/HeYfJP60NsSltPFgxlTcTPaZ9Osp\\\\\\\/WMZDfftNkz48BWlGw1\\\\\\\/1HwAaGkl\\\\\\\/I0DAfG+EckB+TBelc170A0a5IqadFEUtjPb6VHOQJWt7XltsAB0bZlc8Dmikw4uOJzg93n4tPl\\\\\\\/nEk9MQR3LruV3RPD\\\\\\\/Pc9U1Vr0sZGQ5vPXlqpLlaPXJKZHFeMXdM2oPE7VYAiOtgBT3hU21258XkVn+tykzYxxrK8aEUxd3xwgR5g7Dva\\\\\\\/v8LD6aFNWpsHp6mEzyL1GUopz778bO585uxRfe2PdyI+S2QBA4UEuA6HmbEaXzJzrY3Xyxqp33J7e3thwFWF9dmh3UZuGfV6aE0eQlbZYP\\\\\\\/uoFpgp636bZWJ0Vkx6BXsVFAg43MPiNbtKNtzkEeaqjGopxZTzzo5Hr9tUGBtFe+dNTbQzcC6S6877gvhVoHX+QDJuKtts+rPJOL8uDeT5ztkLgKr8MK2D7BDKhnZmoqHfbtMcxd2UErG1Qh1PV+y7vXeoBsEzXyO4HAZOQkiVeFKuyO4NfSx82j2eBgGTnQA1yAQbKIQUQ3TN6Nk\\\\\\\/b+EsrIQ\\\\\\\/3YK5vf0x8PwuEdO4v2nS9H5YYMpna3ainaOOEvs0Gv98UxsN4SKzcCVCfIcnqxPeGar1b0w9mlXKgrd58YXlqFQXfrVafRkGwKKOeGdzKrBM8x1FIAdkRLYKeN58s7zbTW\\\\\\\/cXu6fARNLiI9W0tYspWPZRNYR37heO4ST8JTIfNJmujvW8N4Tb\\\\\\\/WhzjZZRmgf5895zqBcOImbcV5yeKSnGJDmrNAU2RTVfqVdNhAadhcLqDcjQSPb0d8sEIy2co71lNF5O6ev1fThTl1R0MLLlba+kq\\\\\\\/M4OVJ7Z65tyeLP90362FLjk\\\\\\\/i+pRZ6+NJ5OEJ+Af65Qz3NKpUT9LfKK\\\\\\\/lZP96ieV8RdhNJxwPrF\\\\\\\/S8qUYxP9JsSHiqJlB9Rak8MyX8u5DQQm1E462x71e5qJochXYGFjKbYeIBggJau+1JhRwFK9npaOu\\\\\\\/m6Rf1BMZH7H8fdMOaXL2ye00FY5JeW\\\\\\\/egwPXvPhn7Uu2xnUm+tSshCgPCfqOW9yD7B3A68PyHog9w9\\\\\\\/o\\\\\\\/kOL3tXxmasNMVPjLvRMyB2rbNrhY3mStiUE\\\\\\\/2wLpAQIB8NAuDpULF+lPV7gMfwTETUedNTdzBypUtaA51r9N79xpUOAnII\\\\\\\/l38D+moe5yjd+S5l565py60jiLHp9UQi24NwkAXF+I4kg4xs206uCZEUADn+GyM0EuyB6DNez67Xoi61BDlINElwDdE69\\\\\\\/pO2vWlcYxZLfnerK0KmzX6tOdvem6OfRkXm5HXE5H9tCh4H6vx5ha\\\\\\\/PpyGx8Um7rRIlcfTK3KnZdDGRVKQY3KmM8snB\\\\\\\/l8T4x7n6P+zfI0yYkwks3NgWynuE0695e+tg6p5NZyJJYcdXTxQf4awY3OeZttS9SmiMlxMSUyplQj3BqywAG+vCObAwTSRLOuvolu6q2sa7UylJZIlDXs\\\\\\\/moz3XHkIDQy7mz4jV1rq3kJ0HvXG67u1pxyNAb5uZO06QRxTiHouJHNoFuLUS73UpR3VbnOQYJq+jS5qiBBpotO\\\\\\\/dW2LmNUjgw6LY58taGN3YtTJEH+iF5VF3xVVK6vj5CEAhl5CcQD8oRAyanYV3ndF0rQnKb8KQuYdzOL294Qhfl+kamMMGBU2xNiEb410g\\\\\\\/bJPeZSFKlXSQyXCEezGTJAO7wwSCbBnItD3jvDqwZfRh0vk2rJcQClKVa+agDlfdS7f7Is7bj0WjoK4cbSlNEWNEs5CF0ul9shzdv9K3zeyvKwSt0\\\\\\\/k7Amrk\\\\\\\/XLczrB+GZPE+TGXnh\\\\\\\/ltD3MKdQfoC\\\\\\\/ntUXuRHEbZjwVmT19zq74BjIZE2NAmd8VwvipVJbMfsJK0oeFE9J9MWCkdS4G+bqBs0\\\\\\\/QDbEHBPe4aX\\\\\\\/6azDaJ4Vd0oBYwrNsZQpsivJ45CQangk0byVDqMpRfY2p95MbDsL1lYdMQL6l18ObuyTATgheH3Hs9HVc\\\\\\\/01klI674saDX6eT+CPlefTEnqkBlA0SvlBSPpltzTMgJk6Sy4jQeaKNjGvAbOjnl1OrQGD1tJ3UMCtIXgYNx0H0zzcBfq2AGdh4g\\\\\\\/Hzy3OS5j6T4f+yg8+WjKHJjyrqZ+l3yzsqCeRvLyqYHS40lNTUUYwNEOa2ByoJl3Be2c27O17XPSKvfdUhkiejnEVRb0FBD9xCeA9KOxn0GXBlBDviFrR1Rn8o2pdUYzvQz4VoIARnWx4cxuvF92roGm7gM6XWPCNPBTFFJ0aiH3VYaMHLxiaUrqx2DRv4\\\\\\\/zhMN\\\\\\\/\\\\\\\/0T8vuLJ5gOgaQFG5KGeEVUdRHlTyY+ALbRjn6G8CiigGgTBWaGL0vjMk4VvOWJEbtyJPofEdEzj72U2Bbk4okHJGz13HcHlvrjBsrQs1akmqH+phykQzP2o5TzMTFCFROhANlyJApFWjUnS2PbsYg7MakNMxXtIyaJZMBD1aNQCqyGLO6JPUvN7qC8wCbfebLXQKHeo5818PM4+w99U+gyuCbkOEiiaRTKeMFDyMrJJZLzOnAjQzoni43MutF\\\\\\\/qAYi\\\\\\\/sZEhe5LfiuSQrasGX7LQE0irvRAbV8sJ74afbMLk68O6FBkLzezT7kmBCL86nKdX9q6F0XdYr38rc9OWu6KxKO9isPE+vjJJ3Y3NjEbG4tqCx5wz4s32XLyO+pl3W0GWXUMUP+exucLSKFln0A\\\\\\\/cuC\\\\\\\/G4ZNxIjXItXLS4LGTX6VOrBetv32SohKsc7enJFe6R26SLubroioku8cP5T6AEqMkaDzQIu4l1Dj1pQEhSXKnHnksHlHi7+kjMC2RHGNCfXxOyo39J\\\\\\\/V3+QloJFgl0cymkJC0gRjHs0YjOLVDD0sMAGOWYnCe8ukJ0IBuCEgIUOmD3zjXH+V85NZdDVNNnGe37G8BmAh3Kwb51diZP9Hl8v1ZPF261buzjDkaVEbwRpxe2nPoKpNOsq+3Ls5\\\\\\\/A1uATF321HREVgQ0\\\\\\\/pQE5xdd+gLh6iZijqPOZubIm3cnbp4gmGVxov5QgrK2G0A4PFvkDBVCFONEC86tAA4TB0qt2AcCzHgtIb2aCrR8HaiWH2RAn4vsGAQprLFoZ\\\\\\\/OFaRwIkCjkX83pe9MxJt3a8kukvyvrzk0UfPy1\\\\\\\/sxsAPdABlNd36q+6O5auy6o\\\\\\\/Gb\\\\\\\/Q8m\\\\\\\/PJ7nt8in2ZpEF22EM+NzR53+dLnyh6GRPyff7pdkfG9Py7swAz91+IN4KN2e8mpWWVEw3wE\\\\\\\/gO7Hf\\\\\\\/jNebkD\\\\\\\/Upd14x\\\\\\\/SozmjtcFWZtSU3C7\\\\\\\/y0IIOzVceRU52fUJqGt5OCcaRwjUhIyRGgx2B5Nz6jXP8MTMvyk\\\\\\\/2GNJkUqGyV8Z89IC\\\\\\\/S28koj486tL+F8j\\\\\\\/DC4eut7g3vQibbZr0FeXd7yN0JNJijNXoCSbqkT79Xp+l64EaJw2Nkv770gs\\\\\\\/G1XOOZ5wSVb99XJZt1mOx9F5aVzd4mNMejmnSqY49EYacxB\\\\\\\/K4URsG0ptJLe4PEDqNFMer10ivkbQ5Py2nvUIXTVyZS3sJzmoh6C4+HlzhKsk68luLFUQMLvX7VXuoFxAGi9Ddw3IzlIZC5VA\\\\\\\/HzV5laGpNKWQNHNbvelaTdxtl6hHatEBsVb5rHSYaibJwBNEimS5tJhBm1Nw1lpBZ9MoWHB5vTmGDgJUMGftZ5Xza6rAoCO2Tfw2lvvKMF5OP61Ck6m2gIl4BC07Xnj8eFP4RHTdVmvO4oVR3PXasviWmZse2xUpQyApOzniByN+05JItBKKvQNlxQtjHazumLrAZecuoLHFXwOlsaQzvvnR7dzmZ+7rfYp0Y+c7z+UrrAuiaESMmliBHOtWmwdumRxB5a2Nz4EL8EMNTfpXj8WopJU137sucAAINrEp3v\\\\\\\/3jNMCpWa8YB31Ce3Mh5GMY\\\u0022,\\\u0022iv\\\u0022:\\\u00229UpnO47u8xMi4RRkW0E7KA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224eaf215c4bc3b3fd9d7f2446fc48c1ca11bd7607acce38ef508a3804c81c60fb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DRrD1H6TAmlxwT9LlTn2xrb88+nbJAdU3g0VD7TJE3nRypJnU5eHxAsYhzfuYyOI5ZbmIUcFEzrtILs+qFwxC4TN6MpiFA6awe1QlyleJejUeatscNtUPz5\\\\\\\/+g4zjafwbndmCM7rIkbcVEjPsGEbCJpxws0xu5sc7IYBbkSf8C\\\\\\\/8pZ8QVTmuokXd547BvfOEwkjB92mz52\\\\\\\/9pzANXyIGRXOvJyc5PkrW6mevbnPZLOLyItk1AY1dXtYmX3MIPCHw63d1AMXgzELYtF8l9ciNGujdDKWKw5C6k1PNXJC3O\\\\\\\/c3QTOkJY5FRBkxK6\\\\\\\/tl1TKgn1YHRsCa+wTqdft8B\\\\\\\/QhI\\\\\\\/3w17xaKDPONSIOsOCsJPRJJKTJl3bNSm20wy0AzFAcv+NaRFWf+sI9ZcaPQOw8LsqM46LdsoRF9jnLo9XGTwFdETifnJzfDHC\\\\\\\/x0Ho8cW+YZlFhzC5EDv98wlqR5X0zd9+mh5oUPYveP5pnbr\\\\\\\/hSEAQ3sZRvyLQwu76ihwyTk3ToZ4vja\\\\\\\/iaj3iGsVp29CGnB75lXCf1SGsCL\\\\\\\/YOoBJLjOxD46+zKYHI2q4GHlnfIkgWpmgwzUAkcgwSP1jYZAF7QgcsODxE\\\\\\\/tP8RJQR\\\\\\\/5mzsJMOdK9+ENOybipzw6263gfuqdOv6Z6B8eUJFuw1T\\\\\\\/idcqlAYL2fUIHNKfUi\\\\\\\/dkoI4d5vvuo6ojAv88kNPNmPvTlovNHLHHf1C7AfS66Im9m\\\\\\\/IFzn\\\\\\\/QeE1KrgnzrmaVDu6zB7EmOJ4in5JQS2xgT4BOJps7qK1maO+1Ew8ULssDPAH4ecDofR1ibOesHTGfmVz0H67BdIs6Pt9yRJGjAIhMYlBY76W9ZXMH6jcl7OV+HRIgjF2Vaq+dXIlAdrvGEM88PMDTrteGbjkenjsRbX0OAroineReOEkV1O56aPYOw1VNUAUHhMNy84ZmqelKIsDVluNOYFCY5CSXpcVKSA+ygllsoH3Sbm5TQKufwbB6RsX9tf0XrnmJESJ\\\\\\\/IwxgMjrjK6lXRoCF5qIrmFXi8KhtluwKdDyu0YAvmZjglow0nQAzzJntOu28xgR3QT6P7KaPE9QsXeWb8CT7Z5UWwSQtqKkIxkxAio+mFTYLUi8W0UoF2Q4EDlgFlYC8E0wjxTuRSor3lLwflNVyOhQdOt7BqZ9f5qBKkqxn9ObwCPksXpPy6gUx4tWxjxybZ2K5gUVHGg5IliyTF3FXjwnMZ\\\\\\\/zLfbS8NzFzFrsjIRz0jFzCmdECdp\\\\\\\/h6adrN23k14wlMzJ6t0oFVhyscigbhFw929L2ywDNKOuHD496rTUHV2H5ADl+0TyC6xvLr4fZI8zBZOKe7PbnlbDTCFRQs1D69zbtBVEK+zjbzvyjvNFcYSKmHyPD1+y3Q6y5ShO1iek1NTFQ5jkcyx4z0FDLlX10iartiqviy04VZRZ4HMYnFCfQn\\\\\\\/OEeiJBPcFljw7qsxhSRr8mtHzg1yquwh8eeDYYGc3phettwFOuNIDBgkYLhKXhUJsigUP9vclWGKH8TNDLKf6HucX1YBJJrr4SVzgNcol6kauWAsjr+cYJ5EuvcqK2tD1CZyHIJ5X6fHgDgK8cQ\\\\\\\/CyX8dLoIq5Y+KSuRCRZuHmOArognEYw2BYH4+qUoRJQscotGncu1mdQ7qQfBrpbdZ8dEojfN9LVvsxIXc4O1vXuUB7wNXsmv3ykgRe4f5XOxLN96LdF9Sd6aNEufnKOfY5UnzVmIz7g2xRtF4JHQrXokPMHPZGZQ\\\\\\\/47+TXTDsKv5+f9ecbxBE8BbAxT3QxH9unYNvQpEoTNnTJ5tVmgf5m1pPCUVr77gk0hjJSoHov1dpFETG\\\\\\\/b99qQ10Y8VSiytku9lnGEDAcdssgEAVce6pW29CKmzFBXYxcOWf\\\\\\\/YU\\\\\\\/yCcRFwz9VRKuct6saywX3T2D9I0AHnXJKaZN4F5JKvW9qdP0aXryEtQsbDOnBDelOWxjMv64UNa1GZIIavNUm2sMULWIyaNTiPrWQCj21Aipy\\\\\\\/Z+AXWIw0nbNdRRKww0h2kQ2b7HKsfzFndjPAZd2k=\\\u0022,\\\u0022iv\\\u0022:\\\u00229owHMpM7nl6ABfSbQizhJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002288f990cb3ee5984cd8364d89f5fc7c5031ae8685e62e42eadeef49890599bb2c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460165 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2441" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EO3K2UYWYiCWp8cAGLfK7tyH-lzJmwusQ_upCMTNOYo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2441" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UZ0mh5dDplLTp8t9i_CAdyA_2j6qHn5yfZ5vj5rj3yY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2429" title="Slit Binding via the Ig1 Domain Is Essential for Midline Repulsion by Drosophila Robo1 but Dispensable for Receptor Expression, Localization, and Regulation in Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2453" title="Whole-Genome Sequencing Suggests Schizophrenia Risk Mechanisms in Humans with 22q11.2 Deletion Syndrome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460165" id="node460165" data-pisa="ggg;5/11/2441" data-pisa-master="ggg;g3.115.020164" data-apath="/ggg/5/11/2441.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheila A. Kitchen</span>, <span class="highwire-citation-author" data-delta="1">Camerron M. Crowder</span>, <span class="highwire-citation-author" data-delta="2">Angela Z. Poole</span>, <span class="highwire-citation-author" data-delta="3">Virginia M. Weis</span> and <span class="highwire-citation-author" data-delta="4">Eli Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2441-2452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020164 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460165" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheila A. Kitchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BA.%2BKitchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kitchen%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheila%2BA.%2BKitchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheila A. Kitchen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kitchens@science.oregonstate.edu" class="" data-icon-position="" data-hide-link-title="0">kitchens@science.oregonstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Camerron M. Crowder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camerron%2BM.%2BCrowder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crowder%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamerron%2BM.%2BCrowder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Angela Z. Poole </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BZ.%2BPoole%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poole%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BZ.%2BPoole%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Virginia M. Weis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BM.%2BWeis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weis%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BM.%2BWeis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eli Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eli%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEli%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2441" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460165" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2441.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460165" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2441.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460165" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2441.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460165" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2441.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460165" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Many nonmodel species exemplify important biological questions but lack the sequence resources required to study the genes and genomic regions underlying traits of interest. Reef-building corals are famously sensitive to rising seawater temperatures, motivating ongoing research into their stress responses and long-term prospects in a changing climate. A comprehensive understanding of these processes will require extending beyond the sequenced coral genome (<em>Acropora digitifera</em>) to encompass diverse coral species and related anthozoans. Toward that end, we have assembled and annotated reference transcriptomes to develop catalogs of gene sequences for three scleractinian corals (<em>Fungia scutaria</em>, <em>Montastraea cavernosa</em>, <em>Seriatopora hystrix</em>) and a temperate anemone (<em>Anthopleura elegantissima</em>). High-throughput sequencing of cDNA libraries produced ∼20–30 million reads per sample, and <em>de novo</em> assembly of these reads produced ∼75,000–110,000 transcripts from each sample with size distributions (mean ∼1.4 kb, N<sub>50</sub> ∼2 kb), comparable to the distribution of gene models from the coral genome (mean ∼1.7 kb, N<sub>50</sub> ∼2.2 kb). Each assembly includes matches for more than half the gene models from <em>A. digitifera</em> (54–67%) and many reasonably complete transcripts (∼5300–6700) spanning nearly the entire gene (ortholog hit ratios ≥0.75). The catalogs of gene sequences developed in this study made it possible to identify hundreds to thousands of orthologs across diverse scleractinian species and related taxa. We used these sequences for phylogenetic inference, recovering known relationships and demonstrating superior performance over phylogenetic trees constructed using single mitochondrial loci. The resources developed in this study provide gene sequences and genetic markers for several anthozoan species. To enhance the utility of these resources for the research community, we developed searchable databases enabling researchers to rapidly recover sequences for genes of interest. Our analysis of <em>de novo</em> assembly quality highlights metrics that we expect will be useful for evaluating the relative quality of other <em>de novo</em> transcriptome assemblies. The identification of orthologous sequences and phylogenetic reconstruction demonstrates the feasibility of these methods for clarifying the substantial uncertainties in the existing scleractinian phylogeny.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/coral" class="hw-term hw-article-keyword hw-article-keyword-coral" rel="nofollow">coral</a></li><li class="kwd"><a href="/keyword/phylogenomics" class="hw-term hw-article-keyword hw-article-keyword-phylogenomics" rel="nofollow">phylogenomics</a></li><li class="kwd"><a href="/keyword/nonmodel-system" class="hw-term hw-article-keyword hw-article-keyword-nonmodel-system" rel="nofollow">nonmodel system</a></li><li class="kwd"><a href="/keyword/database" class="hw-term hw-article-keyword hw-article-keyword-database" rel="nofollow">database</a></li></ul><p id="p-4">Transcriptome sequencing provides a rapid and cost-effective approach for gene discovery in nonmodel organisms. Analysis of transcriptomes from a diverse range of invertebrates such as sponges (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Riesgo <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conaco <em>et al.</em> 2012</a>), ctenophores (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Ryan <em>et al.</em> 2013</a>), annelids (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Riesgo <em>et al.</em> 2012</a>), and mollusks (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Riesgo <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kocot <em>et al.</em> 2011</a>) has enhanced comparative and evolutionary studies of metazoans. Quantitative analysis of these sequences (RNA-Seq) has become the method of choice to profile genome-wide transcription levels. This technique provides an unbiased approach to discovering functional processes through identification and quantification of differentially expressed genes between phenotypic states including experimental treatments (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Meyer <em>et al.</em> 2011</a>), tissue types (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Siebert <em>et al.</em> 2011</a>), and developmental stages (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Graveley <em>et al.</em> 2011</a>).</p><p id="p-5">Genomic and transcriptomic resources have been developed for a variety of species within the phylum Cnidaria (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Moya <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barshis <em>et al.</em> 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fuchs <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Helm <em>et al.</em> 2013</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lehnert <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Meyer <em>et al.</em> 2009</a>, <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">2011</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Polato <em>et al.</em> 2011</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Shinzato <em>et al.</em> 2014</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Soza‐Ried <em>et al.</em> 2010</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Wenger and Galliot 2013</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Sun <em>et al.</em> 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Meyer and Weis 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lehnert <em>et al.</em> 2014</a>), a diverse group of evolutionarily and ecologically significant species that range from hydroids (Class Hydrozoa) and jellyfish (Class Medusozoa) to sea anemones and corals (Class Anthozoa). Cnidarians are among early-diverging or basal metazoans and occupy a key position as a sister taxon to the bilaterians (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dunn <em>et al.</em> 2008</a>). Many cnidarians play an important role in marine trophic cascades due to their mutualistic relationship with dinoflagellate species of the genus <em>Symbiodinium</em> that reside inside of cnidarian host cells. This relationship is based on nutritional exchange in which <em>Symbiodinium</em> spp. provide the cnidarian host with products from photosynthesis in return for inorganic nutrients and a stable, high-light environment (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Davy <em>et al.</em> 2012</a>). The paramount examples of this partnership are the reef-building corals, which form the trophic and structural foundation of productive and biodiverse coral reef ecosystems. Anthropogenic stressors, especially those associated with global climate change, are gravely threatening these reef ecosystems, including the corals themselves (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Douglas 2003</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Weis and Allemand 2009</a>). Insight into the molecular mechanisms that underlie coral-dinoflagellate symbioses and their stress response to environmental perturbation is critical for future management and conservation of coral reef ecosystems.</p><p id="p-6">To date, there are three publically available sequenced genomes from the Anthozoa: the symbiotic coral <em>Acropora digitifera</em> (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>), symbiotic sea anemone <em>Aiptasia sp.</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baumgarten <em>et al.</em> 2015</a>) and the nonsymbiotic sea anemone, <em>Nematostella vectensis</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Putnam <em>et al.</em> 2007</a>). These genomes have provided insight into the genomic complexity of cnidarians, furthering studies of gene evolution and function across basal metazoans (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Poole and Weis 2014</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Putnam <em>et al.</em> 2007</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Marlow <em>et al.</em> 2009</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Ryan <em>et al.</em> 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hamada <em>et al.</em> 2013</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Shinzato <em>et al.</em> 2012a</a>, <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">b</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Wood-Charlson and Weis 2009</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Dunn <em>et al.</em> 2008</a>). Comparison of these genomes has revealed putative symbiosis-associated genes that may function in the onset and maintenance of cnidarian-dinoflagellate symbiosis (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Meyer and Weis 2012</a>). Annotated <em>de novo</em> transcriptomes, generated using next generation sequencing (NGS) [expressed sequence tags (ESTs), 454 pyrosequencing and Illumina HiSeq technologies], have been published for eight genera of anthozoans (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Polato <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kenkel <em>et al.</em> 2013</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Meyer <em>et al.</em> 2009</a>; <a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lehnert <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pratlong <em>et al.</em> 2015</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Shinzato <em>et al.</em> 2014</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Vidal-Dupiol <em>et al.</em> 2013</a>). These resources have been used in a variety of contexts, including the study of gene family evolution (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Poole and Weis 2014</a>), symbiosis-enhanced gene expression (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lehnert <em>et al.</em> 2014</a>), and responses to environmental stressors such as elevated seawater temperature (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Meyer <em>et al.</em> 2011</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kenkel <em>et al.</em> 2013</a>), bacterial infection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Closek <em>et al.</em> 2014</a>), and CO<sub>2</sub>-driven changes in seawater pH (<a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Vidal-Dupiol <em>et al.</em> 2013</a>). These studies are adding to earlier generation omics studies [EST studies, subtractive hybridization and cDNA microarrays (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Meyer and Weis 2012</a>)] and are providing information on the mechanisms of cnidarian-dinoflagellate symbiosis and coral bleaching, a stress response that results from the breakdown of the partnership (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Weis 2008</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Davy <em>et al.</em> 2012</a>). Expression studies are therefore contributing not only to our basic understanding of cellular processes in cnidarians but also to our ability to link molecular responses with phenotypic change due to environmental perturbation.</p><p id="p-7">The available anthozoan resources are limited in taxonomic diversity and dominated by a few genera from a narrow geographic range (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Meyer and Weis 2012</a>). In addition, many resources are from aposymbiotic (lacking dinoflagellate symbionts) samples or nonsymbiotic species, which limits the study of interplay between the two partners. One goal of this work is to increase the number and diversity of anthozoan resources for comparative, phylogenetic, and functional analyses.</p><p id="p-8">In this study, we present transcriptomes from four anthozoans: the sea anemone <em>Anthopleura elegantissima</em> (Brandt 1835) and the corals <em>Fungia scutaria</em> (Lamarck 1801), <em>Montastraea cavernosa</em> (Linnaeus 1767), and <em>Seriatopora hystrix</em> (Dana 1846) in varying symbiotic states, life history stages, and geographic locations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These species are of particular interest to investigations into the molecular mechanisms associated with the onset, maintenance, and breakdown of cnidarian-dinoflagellate symbioses. We highlight how these transcriptomes can be used in applications ranging from targeted gene searches to orthologous group predictions and phylogenomic analysis. In addition, we outline a method to screen for cross-contamination between sequencing libraries that can be broadly applied to other transcriptome studies.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466992/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466992/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Collection sites, life history stages, and symbiotic states of the four anthozoans used for transcriptome assembly</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection and RNA extraction</h3><p id="p-9">All four anthozoan species examined in this study engage in symbiosis with <em>Symbiodinium</em> spp., and therefore RNA extractions typically include contributions from the dinoflagellate symbionts at some level. Here, two samples (<em>M. cavernosa</em> and <em>S. hystrix</em>) were collected from symbiotic specimens and two samples (<em>F. scutaria</em> and <em>A. elegantissima</em>) were collected from nominally aposymbiotic stages or specimens (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Larvae of <em>F. scutaria</em> were reared in filtered seawater at the Hawaii Institute of Marine Biology following fertilization and development and remained symbiont-free during development (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Schnitzler and Weis 2010</a>). The aposymbiotic specimen of <em>A. elegantissima</em> was collected in that condition in the field.</p><p id="p-10">Total RNA was extracted from <em>S. hystrix</em>, <em>F. scutaria</em>, and <em>A. elegantissma</em> using the following methods. <em>S. hystrix</em> tissue was stored in RNA<em>later</em> Stabilization Solution (Qiagen, CA) and RNA was extracted using the RNeasy Mini Kit (Qiagen, CA) according to the manufacturer’s protocol. Whole animal specimens of <em>A. elegantissima</em> (aposymbiotic) and <em>F. scutaria</em> (larvae) were collected, frozen in liquid nitrogen, and stored at −80°. RNA was extracted using a combination of the TRIzol RNA isolation protocol (Life Technologies, CA) and RNeasy Mini Kit (Qiagen, CA). The TRIzol protocol was used for initial steps up to and including the chloroform extraction. Following tissue homogenization, an additional centrifugation step was performed at 12,000 × <em>g</em> for 10 min to remove tissue debris. After the chloroform extraction, the aqueous layer was combined with an equal volume of 100% EtOH and the RNeasy Mini Kit was used to perform washes following the manufacturer’s protocol.</p><p id="p-11">A core sample of <em>M. cavernosa</em> was collected, frozen in liquid nitrogen, and stored at −80°. Total RNA from <em>M. cavernosa</em> was extracted following a modified TRIzol protocol with a 12-M LiCl precipitation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mazel <em>et al.</em> 2003</a>). Briefly, the coral fragment was vortexed in TRIzol reagent for 15 min and then processed according to the manufacturer’s instructions through phase separation. To precipitate RNA, 0.25 ml of isopropanol and 0.25 ml of a high-salt solution (0.8 M sodium citrate and 1.2 M NaCl) per 1 ml of TRIzol used was added to the aqueous supernatant. The addition of the high-salt solution removes proteoglycan and polysaccharide contaminants. The solution was incubated at room temperature for 10 min and then centrifuged at 12,000 × <em>g</em> for 10 min at 4°. After centrifugation, the standard TRIzol protocol was followed through the ethanol wash. To remove PCR inhibitors of an unknown nature that are frequently encountered in coral samples, RNA was precipitated by adding an equal volume of 12 M LiCl and then was incubated for 30 min at −20°. The sample was centrifuged at 12,000 × <em>g</em> for 15 min at room temperature and washed with 75% EtOH (1 ml per 1 ml of TRIzol), followed by centrifugation at 7500 × <em>g</em> for 5 min at room temperature. The supernatant was removed and the RNA pellet was air-dried.</p><p id="p-12">The extracted total RNA from each sample was DNase-treated using a TURBO DNA-Free Kit (Ambion, CA) to remove genomic DNA contamination. RNA quantity and quality were assessed using the NanoDrop ND-1000 UV-Vis Spectrophotometer (Thermo Scientific, MA) and gel electrophoresis.</p></div><div id="sec-3" class="subsection"><h3>Preparation of sequencing libraries</h3><p id="p-13">Polyadenylated RNA was purified from 10 µg of total RNA using the Magnetic mRNA Isolation Kit (New England Biolabs, MA). First strand cDNA was synthesized using ProtoScript M-MuLV FS-cDNA Synthesis Kit (New England Biolabs, MA) according to the manufacturer’s protocol and modified oligonucleotides in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/020164SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS1.pdf">Table S1</a>. Second strand synthesis was performed by incubating first-strand cDNA with 1× NEBNext Second Strand Synthesis Buffer (New England Biolabs, MA), 0.2 mM dNTPs, 15 units of <em>Escherichia coli</em> DNA ligase (New England Biolabs, MA), 75 units of <em>E. coli</em> DNA polymerase I (New England Biolabs, MA), and 3 units of RNase H (New England Biolabs, MA) for 2 hr at 16°. cDNA was purified using the GeneJet PCR Purification Kit (Fermentas, MA) and then fragmented using NEBNext dsDNA Fragmentase (New England Biolabs, MA) according to the manufacturer’s protocol, with the addition of 5 mM MgCl<sub>2</sub> and 1 mg ml<sup>−1</sup> BSA (New England Biolabs, MA). Fragmented cDNA was purified and the ends were repaired using NEB Quick Blunting Kit (New England Biolabs, MA) according to manufacturer’s protocol. The product was purified and A-tailed in a reaction with nuclease-free water, 1× NEB Standard <em>Taq</em> buffer (New England Biolabs, MA), 1 mM dATP, and 2 units of NEB Standard <em>Taq</em> (New England Biolabs, MA) at 68° for 2 hr. Tailed templates were ligated to double stranded adaptors prepared with oligonucleotides from the Illumina Customer Sequence Letter (version August 12, 2014, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Illumina 2014</a>; <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS1.pdf">Table S1</a>). Purified, tailed cDNA was combined with T4 DNA ligase buffer (New England Biolabs, MA), T4 DNA ligase (New England Biolabs, MA), and the double stranded adaptors, and the solution was incubated at 12° for at least 6 hr. Ligation products were purified and then amplified using custom sample-specific barcodes (“indices”) designed with a 3-bp minimum Hamming distance based on Illumina barcodes (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Illumina 2014</a>) (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS1.pdf">Table S1</a>). PCR included template cDNA, Phusion Taq polymerase buffer (Thermo Scientific, MA), dNTPs, 5′ Illumina “i5” barcoding oligo, 3′ Illumina “i7” multiplex oligonucleotide, and Phusion High Fidelity Taq polymerase (Thermo Scientific, MA). Reactions were incubated at 98° for 30 sec, followed by 17–21 cycles of the following: 98° for 10 sec, 63° for 30 sec, and 72° for 1.5 min. Reactions were amplified for the minimum cycle number required to produce a visible product on a 1% agarose gel. PCR products were size-selected by excising the 350–550 bp fraction from a 2% agarose gel. Finally, size-selected sequencing libraries were extracted using the E.Z.N.A. Gel Extraction Kit (Omega Bio-Tek, GA).</p></div><div id="sec-4" class="subsection"><h3>Sequencing, processing, and assembly</h3><p id="p-14">cDNA libraries were sequenced on Illumina HiSeq 2000 at University of Oregon’s Genomics Core Facility (Eugene, OR). All cDNA libraries were pooled on a single lane to produce 100-bp paired-end reads. Raw sequences were filtered using custom Perl scripts to remove uninformative (matching adaptors in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS1.pdf">Table S1</a>, or poly-A tail) and low-quality reads (&gt;20 positions with quality scores &lt;20) (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Meyer <em>et al.</em> 2011</a>). All custom scripts used in this study are available online at GitHub (<a href="https://github.com/Eli-Meyer">https://github.com/Eli-Meyer</a>). The high-quality filtered reads were then assembled using default settings in Trinity v2.0.2, a de Bruijn graph-based assembler that uses paired-end data to reconstruct transcripts and group these into components intended to represent the collection of transcripts originating from a single gene (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Grabherr <em>et al.</em> 2011</a>).</p></div><div id="sec-5" class="subsection"><h3>Functional annotation</h3><p id="p-15">To develop these assemblies as resources for functional studies, we assigned putative gene names and functional categories [Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG)] to assembled transcripts based on sequence comparisons with online databases. All sequence comparisons were conducted using BLAST+ from National Center for Biotechnology Information (NCBI) (Package version 2.2.29) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>). Gene names were assigned by comparing transcript sequences against UniProt protein sequence databases (SwissProt and TREMBL) using BLASTx with an expect value (E value) cutoff of 10<sup>−4</sup>. Each transcript was assigned a gene name based on its best match, excluding matches with uninformative names (<em>e.g.</em>, uncharacterized, unknown, or hypothetical). GO terms describing biological processes, molecular functions, and cellular components were assigned to each transcript based on GO-UniProt associations of its best match downloaded from the Gene Ontology website (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashburner <em>et al.</em> 2000</a>). KEGG orthology terms were assigned from single-directional best hit BLAST searches of each transcriptome on the KEGG Automatic Annotation Server (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Moriya <em>et al.</em> 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>Reference transcriptome databases</h3><p id="p-16">To enhance the utility of these resources for the coral research community, we have also developed searchable databases and made these publicly available on the author’s laboratory website hosted at Oregon State University (<a href="http://people.oregonstate.edu/%E2%88%BCmeyere/index.html">http://people.oregonstate.edu/∼meyere/index.html</a>). Databases were produced using the open-source SQLite software library and can be queried directly using a publicly accessible web form. To demonstrate the utility of our searchable databases for rapidly identifying genes of interest, we searched each database for a few genes previously studied in cnidarians, including a cell adhesion molecule (sym32) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Reynolds <em>et al.</em> 2000</a>), a cysteine biosynthesis enzyme [cystathionine β-synthase, (Cbs)] (<a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>), and a fluorescent protein (GFP) (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Mazel <em>et al.</em> 2003</a>; <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Shinzato <em>et al.</em> 2012b</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Smith-Keune and Dove 2008</a>). For comparison with these simple text searches, we also conducted a more comprehensive search for each gene based on reciprocal BLAST. Representative sequences for each gene were obtained from the UniProt database (version 2014_09, downloaded October 20, 2014), and searched against each assembly using tBLASTn (bit-score ≥45). The matching transcripts were then reciprocally compared against UniProt using BLASTx. Reciprocal matches were evaluated at the level of gene names; transcripts identified by searching with a target gene (<em>e.g.</em>, B5T1L4, GFP from <em>Acropora millepora</em>) were accepted if they reciprocally matched a different gene with corresponding annotation (<em>e.g.</em>, Q9U6Y6, a GFP gene from <em>Anemonia manjano</em>).</p></div><div id="sec-7" class="subsection"><h3>Evaluating gene content and completeness of assembly</h3><p id="p-17">An ideal reference transcriptome would include all genes present in the genome of an organism, but low or tissue-specific expression can lead to incomplete sampling of genes during cDNA library preparation. To evaluate the gene representation of our assemblies, we searched each assembly for sequence similarity with a core set of conserved eukaryotic genes (CEGMA) (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Parra <em>et al.</em> 2007</a>) and with gene models from sequenced anthozoan genomes: the coral <em>A. digitifera</em> (OIST: adi_v1.0.1) (<a id="xref-ref-81-4" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>) and the anemone <em>N. vectensis</em> (assembly version: Nemve1) (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Putnam <em>et al.</em> 2007</a>). Sequence comparisons were conducted using NCBI’s BLASTx (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) and bit-scores ≥50 were considered significant.</p><p id="p-18">An ideal transcriptome assembly would also include complete transcripts as contiguous sequences or contigs, but variation in coverage and sequence characteristics lead to fragmented assemblies consisting of partial transcripts. To evaluate the effectiveness of our assemblies in reconstructing complete transcripts, we calculated the Ortholog Hit Ratio (OHR), a metric ranging from 0 to 1 that indicates the proportion of each gene included in the assembled transcript (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">O’Neil <em>et al.</em> 2010</a>). Each assembly was compared to gene models from the <em>N. vectensis</em> genome using BLASTx to identify orthologs. We calculated OHR first with a relatively stringent approach (OHR<sub>HITS</sub>) as the proportion of each <em>N. vectensis</em> gene included within local alignments with assembled transcripts (high-scoring segment pairs in BLASTx output). Because this approach excludes divergent regions, we calculated OHR with an alternative and more inclusive approach (OHR<sub>ORF</sub>) as the ratio of the transcript’s longest ORF (in the BLASTx-defined reading frame) relative to the length of its corresponding <em>N. vectensis</em> protein. When multiple transcripts matched a single gene, we considered only the longest OHR. Distributions of maximum OHR scores and summary statistics were examined to evaluate the completeness of each assembly.</p></div><div id="sec-8" class="subsection"><h3>Screening for biological contamination</h3><p id="p-19">All species used in this study engage in symbiotic associations with intracellular dinoflagellate symbionts; therefore, RNA extracted from these specimens is expected to include contributions from both animal hosts and dinoflagellate symbionts. To evaluate these contributions we conducted a series of sequence comparisons aiming to identify the taxonomic origin of each transcript (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Transcripts were compared with a series of sequence databases using BLAST v.2.2.29 with a bit-score threshold of 45. To identify transcripts derived from rRNA, each assembly was compared with cnidarian rRNA sequences using BLASTn. <em>N. vectensis</em> sequences were chosen for this purpose because they represent the most complete cnidarian sequences in the SILVA rRNA database (SILVA: ABAV01023297, ABAV01023333) (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Quast <em>et al.</em> 2013</a>). Transcripts were compared with a cnidarian mitochondrial genome using BLASTn; for this analysis, we chose the complete mitochondrial genome from <em>Acropora tenuis</em> (NCBI: NC_003522.1) (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">van Oppen <em>et al.</em> 2002</a>). To identify the taxonomic origin of each transcript, sequences were compared with the NCBI nonredundant (nr) protein database (downloaded March 12, 2014) using BLASTx (E value ≤10<sup>−5</sup>) (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>). To avoid errors that might arise from the scarcity of cnidarian and dinoflagellate sequences in these databases, transcripts were compared with gene models from <em>Symbiodinium minutum</em> (clade B) (OIST: symbB.v1.2.augustus.prot) and <em>A. digitifera</em> (OIST: adi_v1.0.1_prot) using BLASTx. The taxonomic origin of each sequence was categorized as follows. First, transcripts matching rRNA or mitochondrial sequences were assigned to those categories. Transcripts matching <em>Symbiodinium</em> genes more closely than coral genes that did not return a metazoan hit as their best match in nr were assigned to the dinoflagellate category. Transcripts matching coral genes more closely than <em>Symbiodinium</em> genes that also matched metazoan records or lacked matches in nr were categorized as metazoan. Transcripts that showed conflicting results (metazoan in one database but nonmetazoan in the other) were categorized as “unknown.” Transcripts lacking any match to either coral or <em>Symbiodinium</em> genes were assigned based on taxonomic annotation of the best match in nr, if available. This series of decisions made it possible to classify each transcript based on origin (ribosomal, mitochondrial, other metazoan, dinoflagellate, or other taxa, which includes prokaryotes, uncertain, or no match).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annotation pipeline used to classify origins of each assembled transcript. A series of sequence comparisons was performed comparing each transcript against N. vectensis rRNA (SILVA: ABAV01023297, ABAV01023333), A. tenuis mitochondrial DNA (NCBI: NC_003522.1), A. digitifera and S. minutum gene models, and the NCBI nonredundant protein database (bit-score threshold of 45 for small databases; E value threshold of 10&#x2212;5 for large databases). Transcripts were assigned to categories by evaluating their similarity to each database in the order shown (see Materials and Methods for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285242733" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Annotation pipeline used to classify origins of each assembled transcript. A series of sequence comparisons was performed comparing each transcript against &lt;em&gt;N. vectensis&lt;/em&gt; rRNA (SILVA: ABAV01023297, ABAV01023333), &lt;em&gt;A. tenuis&lt;/em&gt; mitochondrial DNA (NCBI: NC_003522.1), &lt;em&gt;A. digitifera&lt;/em&gt; and &lt;em&gt;S. minutum&lt;/em&gt; gene models, and the NCBI nonredundant protein database (bit-score threshold of 45 for small databases; E value threshold of 10&lt;sup&gt;&#x2212;5&lt;/sup&gt; for large databases). Transcripts were assigned to categories by evaluating their similarity to each database in the order shown (see &lt;em&gt;Materials and Methods&lt;/em&gt; for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2441/F1.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2441/F1.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466910" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Annotation pipeline used to classify origins of each assembled transcript. A series of sequence comparisons was performed comparing each transcript against <em>N. vectensis</em> rRNA (SILVA: ABAV01023297, ABAV01023333), <em>A. tenuis</em> mitochondrial DNA (NCBI: NC_003522.1), <em>A. digitifera</em> and <em>S. minutum</em> gene models, and the NCBI nonredundant protein database (bit-score threshold of 45 for small databases; E value threshold of 10<sup>−5</sup> for large databases). Transcripts were assigned to categories by evaluating their similarity to each database in the order shown (see <em>Materials and Methods</em> for details).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Screening for cross-contamination</h3><p id="p-21">During preliminary analysis of the transcriptome assemblies, we observed a few orthologs with unexpectedly high sequence similarity (&gt;99%) among species.</p><p id="p-22">Because cross-contamination could realistically occur at several different stages during multiplex library preparation and sequencing, we tested for evidence of cross-contamination in our transcriptome assemblies and developed a pipeline to eliminate contaminating sequences. To evaluate the extent of cross-contamination in our libraries, we mapped the cleaned reads used to produce each assembly against that assembly using the Trinity utility <em>align_and_estimate_abundance.pl</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Haas <em>et al.</em> 2013</a>). We then compared all transcriptome libraries sequenced and prepared together using BLASTn to identify nearly identical sequences present in multiple assemblies (bit-score ≥100). This analysis identified many sequences occurring in multiple assemblies, which were highly abundant in one sample (consistent with this being their true origin) but were very low in abundance (&lt;10-fold lower) in other assemblies (consistent with cross-contamination). To evaluate the level of sequence similarity expected among anthozoan transcriptomes, for comparison with the similarity observed among our assemblies, we compared publicly available transcript assemblies produced independently in different labs [<em>Pocillopora damicornis</em>, (<a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>); <em>A. digitifera</em>, (<a id="xref-ref-81-5" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>); and <em>A. millepora</em>, (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Meyer <em>et al.</em> 2009</a>)]. To eliminate putative cross-contaminants identified in our assemblies, we first compared assemblies using BLASTn to identify highly similar sequences (bit-score ≥100). We then estimated the abundance of each transcript in each assembly by mapping and counting reads from each library against the assembly produced from those reads using the Trinity utility <em>align_and_estimate_abundance.pl</em>. To identify and remove sequences that might result from cross-contamination, we categorized each transcript based on sequence similarity and relative expression in all other assemblies. Any transcripts with nearly identical matches in more than one assembly were assigned to the assembly in which each was most abundant if the sequence was at least 10-fold more abundant in that library than any others. Alternatively, transcripts found at comparable levels (&lt;10-fold difference) in multiple assemblies were flagged as “unknown origin” and excluded from further analysis.</p></div><div id="sec-10" class="subsection"><h3>Development of SSR markers</h3><p id="p-23">Simple sequence repeats (SSRs), also known as microsatellites, are sequences with repetitive 2–5 base pairs of DNA. These molecular markers have been widely used for studies of genome mapping, genetic linkage, and population structure. Although SSRs have largely been replaced with sequencing-based approaches for single nucleotide polymorphism (SNP) genotyping, in some situations they may still be the most practical option. To demonstrate the utility of transcriptome assemblies for SSR marker development and to identify SSR markers for the four species described here, we used a pipeline we have previously described for identifying SSRs in coral sequence data (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Davies <em>et al.</em> 2013</a>). In brief, sequences containing repetitive regions (≥30 bp, ≤15% deviation from perfect repeat structure, ≥30 bp flanking regions) were identified using RepeatMasker v3.2.9 (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Smit <em>et al.</em> 1996</a>) and then assembled using CAP3 to eliminate redundancy (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Huang and Madan 1999</a>). Target sequences were further screened for redundancy using BLASTn (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) to identify unique targets within each repeat type (<em>e.g.</em>, AT, CCG, etc.). Finally, primer sequences flanking these SSRs were developed using Primer3 (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rozen and Skaletsky 1999</a>), targeting regions 150–500 bp with 45–65% GC content.</p></div><div id="sec-11" class="subsection"><h3>Identification of orthologous groups</h3><p id="p-24">To facilitate comparative studies of cnidarian gene sequences, and to demonstrate the utility of our transcriptome assemblies for phylogenetic analysis, we identified orthologous groups among the four transcriptomes generated in this study. We also compared these with sequence resources from other cnidarians and basal metazoans, including a marine sponge <em>Amphimedon queenslandica</em> (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Srivastava <em>et al.</em> 2010</a>), the hydrozoan <em>Hydra magnipapillata</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chapman <em>et al.</em> 2010</a>), the schyphozoan <em>Aurelia aurita</em> (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fuchs <em>et al.</em> 2014</a>), and a variety of other anthozoans including <em>Aiptasia pallida</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Lehnert <em>et al.</em> 2012</a>), <em>N. vectensis</em> (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Putnam <em>et al.</em> 2007</a>), <em>A. digitifera</em> (<a id="xref-ref-81-6" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>), <em>Porites astreoides</em> (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Kenkel <em>et al.</em> 2013</a>), <em>P. damicornis</em> (<a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Vidal-Dupiol <em>et al.</em> 2013</a>), <em>Stylophora pistillata</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Karako-Lampert <em>et al.</em> 2014</a>), <em>Orbicella faveolata</em> (formerly belonging to the genus <em>Montastraea</em>) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">DeSalvo <em>et al.</em> 2008</a>), and <em>Pseudodiploria strigosa</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS2.pdf">Table S2</a>). These resources varied in the types of sequencing technologies used to create them, and this resulted in differing degrees of assembly completeness, ranging from whole genomes to EST libraries (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS2.pdf">Table S2</a>). All resources were converted into candidate protein coding sequences using the package TransDecoder (transdecoder.sourceforge.net) that identifies open reading frames. Protein sequences were then processed with FastOrtho (enews.patricbrc.org/fastortho), an OrthoMCL-based program (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li <em>et al.</em> 2003</a>) that performs an all-by-all BLAST of the input sequences (E value cutoff ≤10<sup>−5</sup>) and clusters orthologous groups with the Markov Cluster algorithm (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Van Dongen 2000</a>).</p></div><div id="sec-12" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-25">The four transcriptomes from this study and other sequence resources were used to infer phylogenetic relationships from commonly used markers and newly identified orthologs. The mitochondrial gene <em>cytochrome c oxidase 1</em> (COI) has been used to reconstruct the most comprehensive phylogeny of corals (Anthozoa, Scleractinia) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>), and mitochondrial sequences are commonly used to infer evolutionary relationships of the Cnidaria (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bridge <em>et al.</em> 1992</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kayal <em>et al.</em> 2013</a>). Recent findings suggest, however, that a concatenated set of <em>NADH dehydrogenase</em> genes (ND 2, 4, and 5), called ND supergene, outperforms COI in metazoan datasets including in anthozoans (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Havird and Santos 2014</a>).</p><p id="p-26">To investigate the effect of increased gene sampling on phylogenetic inferences, we compared phylogenetic trees constructed based on the widely used marker COI, the ND supergene, and the set of orthologs identified from a comparison of our transcriptomes with other cnidarian sequence resources. All taxa used in searches for orthologous groups were included and <em>A. queenslandica</em> served as the outgroup. The Transdecoder catalog of proteins for each organism was made into a local BLAST database. Then, the mitochondrial protein sequences of COI, ND2, ND4, and ND5 were found from BLASTx searches against our local databases, UniProt database, or NCBI database (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS4.pdf">Table S4</a>). In some cases, mitochondrial genes were not recovered from the local protein databases, but they were found by tBLASTx to the original resources. These transcripts were instead translated using Expasy Translate Tool (<a href="http://web.expasy.org/translate/">http://web.expasy.org/translate/</a>) under the “invertebrate mitochondrial” genetic code. Proteins sequences for COI, ND2, ND4, and ND5 were aligned using MAFFT v6.864b (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Katoh <em>et al.</em> 2002</a>). In some cases, the mitochondrial sequences were fragmented within a single database or recovered from two separate databases (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS4.pdf">Table S4</a>). These fragments were aligned and manually combined to increase total alignment positions. Individual MAFFT alignments of ND2, ND4, and ND5 were concatenated into a single matrix in Mesquite (v. 3.02) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Maddison and Maddison 2015</a>). Protein alignments of COI and the ND genes were run through ProtTest server (<a href="http://darwin.uvigo.es/software/prottest_server.html">http://darwin.uvigo.es/software/prottest_server.html</a>) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abascal <em>et al.</em> 2005</a>) to select the appropriate substitution rate model based on AIC and BIC criterion. Phylogenetic trees were constructed using maximum likelihood (ML) in RAxML v8.0.26 (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Stamatakis 2014</a>) under the MTZOA+G+F model (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rota-Stabelli <em>et al.</em> 2009</a>). Optimal topology was selected based on ML scores from 500 replicate trees. Nodal support was assessed from 500 bootstrap replicates.</p><p id="p-27">For phylogenomic reconstruction, the computational pipeline PhyloTreePruner (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kocot <em>et al.</em> 2013</a>) was applied to orthologous groups with a minimum amino acid length of 100 from the 15 taxa identified in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS2.pdf">Table S2</a>. PhyloTreePruner is a phylogenetic approach used to refine orthologous groups identified in programs like OrthoMCL by removing predicted paralogs resulting from gene duplication or splice variants through single gene-tree evaluation (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kocot <em>et al.</em> 2013</a>). First, each group of orthologs was aligned using MAFFT v6.864b with 1000 iterations. Ambiguous or uninformative positions were removed from the alignment using Gblocks v0.91b (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Castresana 2000</a>). Then, single-gene ML trees for each group inferred with FastTree2 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Price <em>et al.</em> 2010</a>) were screened for paralogy with PhyloTreePruner and the longest sequence for each taxon was retained. The pruned orthologous groups were then merged into a single matrix using FASconCAT v1.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kück and Meusemann 2010</a>). To examine the impact of missing data on tree topology, two trees were constructed. In the conservative tree, 14–15 taxa were sampled per ortholog for a total of 397 groups (73,833 unique alignment positions). The relaxed tree allowed more missing data, requiring only at least 10 taxa sampled per ortholog for a total of 2896 groups (535,413 unique alignment positions). For each dataset, ML trees were inferred with RAxML v8.0.26 using the WAG+GAMMA+F substitution model (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Whelan and Goldman 2001</a>). Topology for each tree was selected from 100 replicate trees, and nodal support values are based on 100 and 500 bootstrap replicates in the conservative and relaxed trees, respectively.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-28">The data sets supporting the results of this article are available from the Sequence Read Archive at NCBI (Accession number: SRP063463), the Dryad Digital Repository (doi: 10.5061/dryad.3f08f), and the author’s website (<a href="http://people.oregonstate.edu/%E2%88%BCmeyere/index.html">http://people.oregonstate.edu/∼meyere/index.html</a>).</p></div></div><div class="section results-discussion" id="sec-14"><h2 class="">Results and Discussion</h2><div id="sec-15" class="subsection"><h3>Sequencing and <em>de novo</em> assembly</h3><p id="p-29">The four libraries described here were sequenced on Illumina HiSeq 2000 (each occupying 1/6th of a lane), yielding on average 26.3 million paired reads per library (range, 21.2–30.3; <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS5.pdf">Table S5</a>). A fraction of these (22% on average; range 14–28%) were removed during quality and adaptor filtering prior to assembly. Assembly of the remaining high-quality reads produced on average ∼170,000 transcripts. This is substantially higher than the number of genes in sequenced cnidarian genomes (23,677 in <em>A. digitifera</em>, 27,273 in <em>N. vectensis</em>), which likely results from redundancy, fragmentation in the assemblies, and biological contamination. Assemblies included many small contigs (on average, 47% were &lt;400 bp) that were unlikely to provide significant matches, so for analyses based on sequence homology we considered only contigs ≥400 bp (average n = 91,792). For these core transcriptome datasets used for downstream analyses, the average length ranged from 1.1 to 1.7 kb and N<sub>50</sub> ranged from 1.4 to 2.7 kb. These are slightly shorter than the expected size distribution for a complete cnidarian transcriptome (<em>e.g.</em>, average ∼1700 and N<sub>50</sub> ∼2200 bp transcripts in the <em>A. digitifera</em> genome), suggesting incomplete assemblies. Assembly statistics of the four transcriptome references developed in this study are broadly comparable to previously published anthozoan transcriptomes (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Moya <em>et al.</em> 2012</a>; <a id="xref-ref-81-7" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>, <a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abascal <em>et al.</em> 2005</a>; <a id="xref-ref-94-4" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Lehnert <em>et al.</em> 2012</a>).</p></div><div id="sec-16" class="subsection"><h3>Completeness of transcriptomes</h3><p id="p-30">To evaluate the completeness of the transcriptome assemblies from the perspective of gene content, we conducted sequence comparisons with conserved eukaryotic genes and gene models from sequenced relatives. The core eukaryotic genes (CEGMA) (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Parra <em>et al.</em> 2007</a>) are expected to be expressed in most eukaryotes (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nakasugi <em>et al.</em> 2013</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Sanders <em>et al.</em> 2014</a>) and are widely used to estimate transcriptome completeness. Sequence comparisons revealed matches for 453 (98.9%) of these conserved genes in <em>A. elegantissma</em> and 456 (99.5%) in <em>F. scutaria</em>, <em>M. cavernosa</em>, and <em>S. hystrix</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). For a more comprehensive view of gene representation, the transcriptomes were compared with gene models from sequenced relatives (the coral <em>A. digitifera</em> and the anemone <em>N. vectensis</em>). This analysis identified matches for more than 14,000 gene models in each genome (BLASTx, bit-score ≥50): 54–67% of gene models in <em>A. digitifera</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) and 48–49% in <em>N. vectensis</em>. This is comparable to the level of sequence similarity observed among anthozoans with completed genomes. BLASTp comparisons of predicted proteins from the <em>A. digitifera</em> and <em>N. vectensis</em> genomes using the same thresholds recover 35% and 42% of genes in the other genome. This is substantially lower than the optimistic estimates of representation based on CEGMA, perhaps reflecting essential functions and constitutive expression of these highly conserved genes. Comparisons with gene models of closely related taxa appear to provide a more conservative estimate of gene representation in transcriptome assemblies.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three metrics used to evaluate gene representation and assembly of complete transcripts in de novo transcriptome assemblies. (A) Percent of core eukaryotic genes (CEGMA) identified in each assembly. (B) Percent of A. digitifera gene models with significant matches in each assembly. (C) Median proportion of each N. vectensis protein aligned with transcripts in each assembly (OHRHITS). Gray = our transcriptome assembly compared to the respective reference for each analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285242733" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Three metrics used to evaluate gene representation and assembly of complete transcripts in &lt;em&gt;de novo&lt;/em&gt; transcriptome assemblies. (A) Percent of core eukaryotic genes (CEGMA) identified in each assembly. (B) Percent of &lt;em&gt;A. digitifera&lt;/em&gt; gene models with significant matches in each assembly. (C) Median proportion of each &lt;em&gt;N. vectensis&lt;/em&gt; protein aligned with transcripts in each assembly (OHR&lt;sub&gt;HITS&lt;/sub&gt;). Gray = our transcriptome assembly compared to the respective reference for each analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2441/F2.medium.gif" width="175" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2441/F2.medium.gif" width="175" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466984" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Three metrics used to evaluate gene representation and assembly of complete transcripts in <em>de novo</em> transcriptome assemblies. (A) Percent of core eukaryotic genes (CEGMA) identified in each assembly. (B) Percent of <em>A. digitifera</em> gene models with significant matches in each assembly. (C) Median proportion of each <em>N. vectensis</em> protein aligned with transcripts in each assembly (OHR<sub>HITS</sub>). Gray = our transcriptome assembly compared to the respective reference for each analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">To evaluate the effectiveness of our assemblies in reconstructing complete transcripts, we calculated ortholog hit ratios (OHR) for each final assembly. This method estimates the amount of a <em>de novo</em> transcript contained in the best ortholog from a reference genome (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">O’Neil <em>et al.</em> 2010</a>), ranging from 1 (for complete transcripts) to 0 (for transcript fragments). We calculated OHR based on sequence comparisons with <em>N. vectensis</em> gene models using two approaches. First, a relatively stringent analysis based on the proportion of each <em>N. vectensis</em> gene included in regions of local similarity (OHR<sub>HITS</sub>) produced median OHR of 63.8, 64.7, 65.7, and 58.0% for <em>A. elegantissma</em>, <em>F. scutaria</em>, <em>M. cavernosa</em>, and <em>S. hystrix</em>, respectively (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). A more inclusive analysis based on the longest ORF (in BLAST defined frame) produced similar estimates (median OHR<sub>ORF</sub>: 67.4, 75.8, 77.2, and 60.3%, respectively). Each assembly included more than 5000 reasonably complete transcripts spanning at least 75% of the corresponding <em>N. vectensis</em> gene (range, 5262–6725). Overall, these comparisons with existing cnidarian sequence resources quantify the representation and completeness of our assemblies and provide a framework for comparison with other <em>de novo</em> assemblies. These estimates compare favorably with previous transcriptome completeness estimates for cnidarians (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Sanders <em>et al.</em> 2014</a>) and several invertebrates (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">O’Neil and Emrich 2013</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Riesgo <em>et al.</em> 2012</a>) using similar methods.</p></div><div id="sec-17" class="subsection"><h3>Annotation of transcriptomes</h3><p id="p-33">Transcripts were annotated using BLAST homology searches against the UniProt databases. Approximately one-third of all transcripts matched records in UniProt (range, 30–40%) (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS5.pdf">Table S5</a>). The relatively low fraction of sequences annotated is attributable in part to sequence lengths: on average, 21% of transcripts &lt;400 bp in length were annotated as compared with 42% of transcripts 400–1000 bp in length and 78% of transcripts &gt;1000 bp. Even among the longest transcripts (&gt;1 kb), a substantial number of sequences lacked annotated matches in UniProt (range, 6647–12,090 sequences per assembly). This highlights the well-known bias in taxonomic composition of existing databases and the value of ongoing gene sequencing in underrepresented metazoan taxa for public sequence databases.</p><p id="p-34">To categorize the biological functions inferred from sequence similarity, Gene Ontology (GO) terms were assigned to transcripts matching GO-annotated records in the UniProt database. This process identified functional annotation for 77% of transcripts with BLAST matches, providing tentative gene identities for a large number of sequences in each assembly (range, 32,299–47,547 transcripts; <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS5.pdf">Table S5</a>). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the distribution of functional categories across the four transcriptomes, visualized using the Web Gene Ontology Annotation Plotting (WEGO) application. The GO terms were broadly distributed across the three domains and the percentages of sequences mapped to a given sub-ontology were highly similar for all species and comparable to other invertebrate transcriptomes (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Riesgo <em>et al.</em> 2012</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">O’Neil <em>et al.</em> 2010</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Lehnert <em>et al.</em> 2012</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Moya <em>et al.</em> 2012</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Polato <em>et al.</em> 2011</a>; <a id="xref-ref-84-4" class="xref-bibr" href="#ref-84">Shinzato <em>et al.</em> 2014</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Stefanik <em>et al.</em> 2014</a>; <a id="xref-ref-94-5" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>). The similarities in functional distributions of assemblies prepared from diverse species, developmental stages, and symbiotic states highlight the constitutive expression of a broad set of genes in cnidarian transcriptomes. These core genes should facilitate comparative transcriptome studies by increasing the overlap among incomplete libraries.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of functional categories (GO terms) in each transcriptome assembly. The percentage of transcripts with GO annotation for each category under the three main ontology domains was calculated for each assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285242733" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of functional categories (GO terms) in each transcriptome assembly. The percentage of transcripts with GO annotation for each category under the three main ontology domains was calculated for each assembly.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2441/F3.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2441/F3.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Distribution of functional categories (GO terms) in each transcriptome assembly. The percentage of transcripts with GO annotation for each category under the three main ontology domains was calculated for each assembly.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To determine taxonomic origin for each transcript, we conducted a series of BLAST searches and filtering steps outlined in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Because our assemblies were produced from symbiotic and aposymbiotic specimens, the transcriptomes contain genes not only from anthozoans but also from their associated microbial community. To investigate the relative contributions of these sources we classified each transcript based on sequence similarity (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). These analyses confirmed that metazoan sequences comprised the majority of each library as expected. Fortunately, only a small fraction of transcripts were derived from organelles (mitochondria and ribosomes): on average, 212 transcripts (range, 127–284) in each assembly matched rRNA (<em>N. vectensis</em>) and 30 transcripts (range, 16–54) matched the mitochondrial genome (<em>A. tenuis</em>). Notably, almost half of the transcripts in each assembly (range, 46.2–49.9%) lacked matches to coral or <em>Symbiodinium</em> spp. genes, or NCBI’s <em>nr</em> database (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), a range that is consistent with results from other anthozoan transcriptomes (<a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Sun <em>et al.</em> 2013</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Karako-Lampert <em>et al.</em> 2014</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Polato <em>et al.</em> 2011</a>; <a id="xref-ref-94-6" class="xref-bibr" href="#ref-94">Traylor-Knowles <em>et al.</em> 2011</a>). These "unknown" transcripts may represent lineage-specific genes ("taxonomically restricted genes") that require further characterization. Comparison with NCBI’s <em>nr</em> database revealed that the majority of sequences with matches in one or more databases (59–95%) matched a metazoan sequence better than any other taxon, suggesting they originated from the animal host rather than from dinoflagellate or prokaryotic symbionts. A negligible fraction of transcripts in each assembly (0.8–1.7%) was assigned to the “Other taxa” category, most of which matched either coral or <em>Symbiodinium</em> genes but were classified as “unknown” because of conflicting results in the <em>nr</em> search (<em>e.g.</em>, transcripts that matched <em>Symbiodinium</em> more closely than coral but whose best matches in <em>nr</em> were from metazoans).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted taxonomic origin of transcriptomes based on homology searches with BLAST. The percent of transcripts that were assigned to rRNA (purple), mtDNA (blue), dinoflagellate (green), metazoan (pink), other taxa (orange), and no match (gray) are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285242733" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted taxonomic origin of transcriptomes based on homology searches with BLAST. The percent of transcripts that were assigned to rRNA (purple), mtDNA (blue), dinoflagellate (green), metazoan (pink), other taxa (orange), and no match (gray) are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2441/F4.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2441/F4.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Predicted taxonomic origin of transcriptomes based on homology searches with BLAST. The percent of transcripts that were assigned to rRNA (purple), mtDNA (blue), dinoflagellate (green), metazoan (pink), other taxa (orange), and no match (gray) are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The contribution of algal symbionts varied widely across samples. In nominally aposymbiotic samples of <em>F. scutaria</em> and <em>A. elegantissma</em>, 2.6% of transcripts on average were classified as dinoflagellate in origin (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), which may have resulted either from unexpected presence of symbionts at low abundance in these samples or from genes lacking orthologs in the <em>A. digitifera</em> reference. The symbiotic samples from <em>S. hystrix</em>, in contrast, showed a comparable abundance of transcripts classified as metazoan (61,369) and dinoflagellate in origin (41,724). Surprisingly, the <em>M. cavernosa</em> library that was similarly prepared from a symbiotic sample showed only 7278 transcripts from dinoflagellates (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). This striking contrast in <em>Symbiodinium</em> contributions from symbiotic specimens may have arisen from differing methods of RNA extraction. For <em>S. hystrix</em>, tissue was airbrushed off the coral skeleton directly into RNA<em>later</em> Stabilization Solution (Qiagen, CA), followed by complete tissue homogenization. In contrast, the <em>M. cavernosa</em> fragment was simply vortexed to disrupt tissue, without physical homogenization. Our findings suggest that omitting physical homogenization during lysis can minimize symbiont contamination for studies aiming to focus on the cnidarian host, whereas studies investigating both components may benefit from thorough homogenization during extraction. The gene names, functional categories, and putative origin of each transcript are annotated in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS6.xls">Table S6</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS7.xls">Table S7</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS8.xls">Table S8</a>, and <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS9.xls">Table S9</a>.</p></div><div id="sec-18" class="subsection"><h3>Gene searches of the database</h3><p id="p-39">The resulting annotations and sequences are available in a set of searchable databases hosted by Oregon State University (<a href="http://people.oregonstate.edu/%E2%88%BCmeyere/index.html">http://people.oregonstate.edu/∼meyere/index.html</a>). To illustrate the utility of databases for cnidarian researchers targeting specific genes, we compared the effectiveness of simple text searches of the databases with reciprocal BLAST (RB) analysis, a more comprehensive approach that requires additional work by the end-user. Text searches targeting a few selected genes [cell adhesion molecule sym32, green fluorescent protein (GFP), and cystathionine β-synthase (Cbs)] produced comparable results as RB searches (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS10.pdf">Table S10</a>). Text searches are obviously sensitive to query phrasing; the query “fluorescent” retrieves 51 putative GFP homologs, and functionally related synonyms (“GFP”, “chromoprotein”) retrieved an additional 10. Interestingly, the Cbs homologs identified in nominally aposymbiotic samples (<em>A. elegantissima</em> and <em>F. scutaria</em>) showed greater sequence similarity with <em>Symbiodinium</em> gene models than coral (<em>A. digitifera</em>) and were classified as dinoflagellate in our assignment procedure (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>), whereas Cbs homologs in symbiotic samples (<em>M. cavernosa</em> and <em>S. hystrix</em>) included both metazoan and dinoflagellate transcripts. This unexpected observation of apparently dinoflagellate homologs of Cbs in nominally aposymbiotic samples is noteworthy because of their variable distribution among corals and possible roles in coral nutritional dependency on symbiosis (<a id="xref-ref-81-8" class="xref-bibr" href="#ref-81">Shinzato <em>et al.</em> 2011</a>). While this finding could be explained by undetected <em>Symbiodinium</em> harbored in these putatively aposymbiotic samples, the uncertainty introduced by these observations suggests that studies investigating the diversity of Cbs homologs across corals may require additional data (<em>e.g.</em>, <em>in situ</em> hybridization) to confirm transcript origins. Overall, the close agreement between rigorous computational searches and simple text searches in these examples illustrates the utility of our searchable online databases for rapidly identifying genes of interest in reference transcriptome assemblies.</p></div><div id="sec-19" class="subsection"><h3>Novel SSR markers</h3><p id="p-40">Simple sequence repeats (SSRs or microsatellites) have been widely used to study genetic diversity, hybridization events, population structure, and connectivity in anthozoans (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Concepcion <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fernandez-Silva <em>et al.</em> 2013</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Selkoe and Toonen 2006</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Ruiz-Ramos and Baums 2014</a>), and can directly influence phenotypic traits by altering DNA replication, translation, and gene expression (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Ruiz-Ramos and Baums 2014</a>). SSR markers can be readily identified from <em>de novo</em> assemblies of NGS data and emerge as a side benefit in transcriptome assembly projects conducted for other purposes. We identified and developed primers for 52, 49, 73, and 75 candidate SSR markers in <em>A. elegantissma</em>, <em>F. scutaria</em>, <em>M. cavernosa</em>, and <em>S. hystrix</em>, respectively. Primer pairs for each species are listed in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS11.xls">Table S11</a>. For three of the species studied here, varying numbers of SSR markers are already available. Previous studies of <em>S. hystrix</em> have developed 10 SSR markers (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Maier <em>et al.</em> 2001</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Underwood <em>et al.</em> 2006</a>) to study habitat partitioning within a single reef (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bongaerts <em>et al.</em> 2010</a>), dispersal and recruitment patterns across multiple reefs (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">van Oppen <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kininmonth <em>et al.</em> 2010</a>), and population changes associated with bleaching events (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Underwood <em>et al.</em> 2007</a>). Candidate SSR markers have been identified in <em>F. scutaria</em> (n = 118) from the coral host and dinoflagellate symbionts (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Concepcion <em>et al.</em> 2010</a>). SSR markers previously developed in <em>M. cavernosa</em> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Shearer <em>et al.</em> 2005</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Serrano <em>et al.</em> 2014</a>) have been used to investigate the population connectivity across depth and geographic distance (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Serrano <em>et al.</em> 2014</a>). The candidate SSR markers identified in this study provide additional markers for future studies along similar lines. To our knowledge, SSR markers have not been previously developed in <em>A. elegantissima</em>. Although the population structure of the host has not been described, analysis of their dinoflagellate symbionts revealed highly structured populations across their geographic range (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sanders and Palumbi 2011</a>). The markers developed in this study for <em>A. elegantissima</em> provide tools to investigate population structure of the host across a similar range.</p></div><div id="sec-20" class="subsection"><h3>Orthologous groups and phylogenomic reconstructions</h3><p id="p-41">With the increasing availability of transcriptomes and genomes, these datasets can now be mined to discover novel phylogenetic markers within Anthozoa and across the Cnidaria to resolve taxonomic uncertainties. Phylogenetic reconstruction of anthozoans has presented challenges because analyses based on morphology, life history, and molecular sequences have failed to adequately delineate taxonomic boundaries or evolutionary relationships (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Daly <em>et al.</em> 2003</a>). To date, molecular phylogenies for anthozoans have been based on one or a small number of markers including nuclear ribosomal 28S and 18S genes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Daly <em>et al.</em> 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berntson <em>et al.</em> 1999</a>), β<em>-tubulin</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>), mitochondrial 16S (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Daly <em>et al.</em> 2003</a>), <em>cytochrome b</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>), and COI (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>). Interestingly, mitochondrial sequences in anthozoans have extremely low mutation rates compared to the bilaterians and are therefore highly conserved, allowing for robust comparisons across distantly related taxa (<a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">van Oppen <em>et al.</em> 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Galtier <em>et al.</em> 2009</a>). Therefore, the mitochondrial gene COI has been used recently to define evolutionary relationships among scleractinian corals (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>; <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>) and to support the distinction of robust corals from the complex corals (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Romano and Palumbi 1996</a>).</p><p id="p-42">One disadvantage to single gene phylogenetic inferences is that they suffer from weak phylogenetic signals, sensitivity to hidden paralogy, and spurious tree artifacts (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Philippe <em>et al.</em> 2004</a>). Despite these potential limitations, single gene trees have advanced the field of cnidarian systematics. However, polyphyly remains a problem among several anthozoan families when using both maximum likelihood and Bayesian analyses (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>), which has led to recent shifts in taxonomic classification (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>). To expand beyond previous single-gene approaches, we performed phylogenomic analyses incorporating the four new transcriptomes and other available "omic" resources. By simultaneously increasing taxon and gene sampling, phylogenetic inference is expected to improve (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Philippe <em>et al.</em> 2004</a>) and may help resolve some of the challenges in reconstructing the evolutionary relationships of the Anthozoa and, more broadly, phylum Cnidaria.</p><p id="p-43">For phylogenomic analysis, transcripts larger than 400 bp were converted to protein with TransDecoder and clustered into orthologous groups using FastOrtho. The number of assigned orthologous groups ranged from 14,144 to 21,147 for the four transcriptomes (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/FigureS1.pdf">Figure S1</a>). Comparison of all four resulted in 6560 shared orthologs (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/FigureS1.pdf">Figure S1</a>). The three coral species shared 2045 orthologs not found in anemones and the two most closely related corals (<em>M. cavernosa</em> and <em>F. scutaria</em>) shared 1682 orthologs absent from the other assemblies. By incorporating 11 additional taxa for phylogenomic analysis (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS2.pdf">Table S2</a>), 443 orthologs were identified between all taxa. After setting a minimum protein length (100 amino acids), these orthologs were refined using the PhyloTreePruner analysis pipeline (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Kocot <em>et al.</em> 2013</a>). Filtering resulted in the identification of 397 orthologs for ≥14 taxa. These were used to construct a phylogenetic tree we termed “conservative” because loci with any missing data were excluded (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS12.xls">Table S12</a>).</p><p id="p-44">Missing data are a commonly encountered problem in phylogenomic analyses, from either reduced transcript length or gene absence from a transcriptome (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Philippe <em>et al.</em> 2004</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Kocot <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Roure <em>et al.</em> 2013</a>). However, the sensitivity of phylogenetic inference to incomplete datasets is still under investigation, with mixed results from phylogenomic analyses on large but patchy supermatrices (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Roure <em>et al.</em> 2013</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Philippe <em>et al.</em> 2004</a>). Because the resources in this study used for ortholog identification differed in completeness, ranging from EST libraries to complete genomes (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS2.pdf">Table S2</a>), we tested the influence of missing data on our phylogenetic reconstruction. To investigate this, we lowered the required number of taxa per orthologous group to ≥10, which identified 2897 orthologs (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/TableS12.xls">Table S12</a>). This second set was used to create the “relaxed” phylogeny, so called because loci with some missing data were included.</p><p id="p-45">Both maximum likelihood phylogenomic analyses reconstructed identical and strongly supported topologies (bootstrap = 100; <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/FigureS2.pdf">Figure S2</a>), demonstrating that our phylogenetic inference was insensitive to missing data (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). However, the relationship of the corals in the family Faviidae, containing <em>M. cavernosa</em>, <em>P. strigosa</em>, and <em>O. faveolata</em>, varied among the COI, ND supergene, and phylogenomic analyses. The mitochondrial ND supergene identified by <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Havird and Santos (2014)</a> produced a phylogenetic tree nearly synonymous with the accepted cnidarian taxonomic relationships and phylogenomic analyses from this study (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>), except for the placement of the <em>M. cavernosa</em> as sister taxon to <em>O. faveolata</em> and <em>P. strigosa</em>. The analysis of single gene COI resulted in a discordant phylogenetic topology (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), failing to reconstruct the complex coral clade (<em>P. astreoides</em> and <em>A. digitifera</em>), which was recovered by ND supergene, relaxed and conserved trees (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/FigureS2.pdf">Figure S2</a>). In the COI tree, the placement of the <em>F. scutaria</em>, from the family Fungiidae, as sister taxon to <em>P. strigosa</em> and <em>M. cavernosa</em> from the family Faviidae, instead of <em>O. faveolata</em> is incongruent with current taxonomic placement (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>) (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>). Furthermore, while the phylogenomic analyses placed <em>O. faveolata</em> as sister to <em>P. strigosa</em> with strong support (bootstrap = 100), this relationship was not recovered in either mitochondrial phylogeny (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.020164.DC1/FigureS2.pdf">Figure S2</a>). Overall, the tree topology from the phylogenomic analyses is consistent with accepted evolutionary relationships within Anthozoa (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Budd and Stolarski 2011</a>; <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Fukami <em>et al.</em> 2008</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> 2010</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Discordance in maximum likelihood phylogenetic reconstruction of COI compared to a combined phylogeny of concatenated ND (2, 4, and 5) genes and two phylogenomic trees. The COI phylogeny is presented on the left and the combined phylogeny is presented on the right. Topology for the ND mitochondrial set, relaxed and conservative phylogenomic trees were nearly identical. Therefore, nodal support is summarized on the relaxed tree (right). Bootstrap support at the nodes from left to right represents ND gene set/relaxed/conservative. If topologies differed in the summary tree, then the nodal support is presented - - as next to the node. Yellow solid lines connect taxon with different positions and/or relationships between the two trees, whereas black dashed lines connect those with the same position and/or relationship. Reconstructions of groups in the class Anthozoa based on Kitahara et al. (2010) are highlighted in boxes: teal= robust corals; dark pink = complex corals; and light blue = anemones. The names of species used in this study are emphasized by bold font. Scale bars indicate the amino acid replacements per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285242733" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Discordance in maximum likelihood phylogenetic reconstruction of COI compared to a combined phylogeny of concatenated ND (2, 4, and 5) genes and two phylogenomic trees. The COI phylogeny is presented on the left and the combined phylogeny is presented on the right. Topology for the ND mitochondrial set, relaxed and conservative phylogenomic trees were nearly identical. Therefore, nodal support is summarized on the relaxed tree (right). Bootstrap support at the nodes from left to right represents ND gene set/relaxed/conservative. If topologies differed in the summary tree, then the nodal support is presented - - as next to the node. Yellow solid lines connect taxon with different positions and/or relationships between the two trees, whereas black dashed lines connect those with the same position and/or relationship. Reconstructions of groups in the class Anthozoa based on Kitahara &lt;em&gt;et al.&lt;/em&gt; (2010) are highlighted in boxes: teal= robust corals; dark pink = complex corals; and light blue = anemones. The names of species used in this study are emphasized by bold font. Scale bars indicate the amino acid replacements per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2441/F5.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2441/F5.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2441/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467020" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Discordance in maximum likelihood phylogenetic reconstruction of COI compared to a combined phylogeny of concatenated ND (2, 4, and 5) genes and two phylogenomic trees. The COI phylogeny is presented on the left and the combined phylogeny is presented on the right. Topology for the ND mitochondrial set, relaxed and conservative phylogenomic trees were nearly identical. Therefore, nodal support is summarized on the relaxed tree (right). Bootstrap support at the nodes from left to right represents ND gene set/relaxed/conservative. If topologies differed in the summary tree, then the nodal support is presented - - as next to the node. Yellow solid lines connect taxon with different positions and/or relationships between the two trees, whereas black dashed lines connect those with the same position and/or relationship. Reconstructions of groups in the class Anthozoa based on <a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Kitahara <em>et al.</em> (2010)</a> are highlighted in boxes: teal= robust corals; dark pink = complex corals; and light blue = anemones. The names of species used in this study are emphasized by bold font. Scale bars indicate the amino acid replacements per site.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Conclusion</h3><p id="p-47">The annotated transcriptome assemblies developed in this study provide useful resources for genomic research in anthozoan species for which sequences resources were previously lacking. The searchable databases developed from these assemblies make it possible to rapidly identify genes of interest from each species. Our ortholog analysis demonstrates the feasibility of phylogenetic inference in corals using transcriptome assemblies from diverse stages and symbiotic states, highlighting a promising path toward resolving major uncertainties in the existing phylogeny of scleractinians. Future studies will benefit from the growing body of anthozoan sequence resources, including the four assemblies contributed in this study.</p></div></div><div class="section" id="sec-22"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We acknowledge Dr. Christine Schnitzler and the labs of Dr. Tung-Yung Fan and Dr. Andrew Baker for assistance with sample collection. In addition, we thank Sarah Guermond and Emily Weiss for assistance in sample preparation and analysis. Research funding was provided by Oregon State University, Department of Integrative Biology. Publication of this article in an open access journal was funded by the Oregon State University Libraries &amp; Press Open Access Fund. E.M., S.K., and A.P. conceived the investigation. S.K., C.C., and A.P. performed library preparation and sequencing. C.C., A.P., and E.M. assembled and annotated the transcriptomes. S.K. performed computational analyses related to transcriptome completeness, GO annotation, and phylogenetics. C.C. complied transcriptome statistics. C.C. and S.K. performed targeted gene searches. E.M. performed cross-contamination screens, identified SSR markers, and provided bioinformatic expertise. S.K., C.C., and E.M. made significant contributions to the preparation of the manuscript. All authors revised and approved the final manuscript. The authors declare no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-49">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020164/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020164/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-50">Accession Numbers: Raw data: NCBI’s SRA, accession SRP063463. Assemblies: DRYAD digital repository, doi: 10.5061/dryad.3f08f</p></li><li class="fn" id="fn-4"><p id="p-51"><em>Communicating editor: A. Gasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-07"><span class="received-label">Received </span>July 7, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-11"><span class="accepted-label">Accepted </span>September 11, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Kitchen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.1" data-doi="10.1093/bioinformatics/bti263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abascal</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zardoya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">ProtTest: selection of best-fit models of protein evolution.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti263%26rft_id%253Dinfo%253Apmid%252F15647292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzIxMDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.2" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nature Genetics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.4" data-doi="10.1073/pnas.1210224110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barshis</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladner</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seneca</span>  <span class="cit-name-given-names">F. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traylor-Knowles</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic basis for coral resilience to climate change.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1387</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210224110%26rft_id%253Dinfo%253Apmid%252F23297204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzQvMTM4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.5" data-doi="10.1073/pnas.1513318112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baumgarten</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simakov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esherick</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liew</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehnert</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genome of Aiptasia, a sea anemone model for coral symbiosis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">11893</span>–<span class="cit-lpage">11898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1513318112%26rft_id%253Dinfo%253Apmid%252F26324906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM4LzExODkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.6" data-doi="10.1006/mpev.1999.0649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berntson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">France</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullineaux</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Phylogenetic relationships within the class Anthozoa (phylum Cnidaria) based on nuclear 18S rDNA sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DBerntson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D417%26rft.epage%253D433%26rft.atitle%253DPhylogenetic%2Brelationships%2Bwithin%2Bthe%2Bclass%2BAnthozoa%2B%2528phylum%2BCnidaria%2529%2Bbased%2Bon%2Bnuclear%2B18S%2BrDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.1999.0649%26rft_id%253Dinfo%253Apmid%252F10603268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.1999.0649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10603268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083786500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.7" data-doi="10.1371/journal.pone.0010871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bongaerts</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riginos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridgway</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampayo</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic divergence across habitats in the widespread coral <em>Seriatopora hystrix</em> and its associated <em>Symbiodinium</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBongaerts%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10871%26rft.epage%253De10871%26rft.atitle%253DGenetic%2Bdivergence%2Bacross%2Bhabitats%2Bin%2Bthe%2Bwidespread%2Bcoral%2BSeriatopora%2Bhystrix%2Band%2Bits%2Bassociated%2BSymbiodinium.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010871%26rft_id%253Dinfo%253Apmid%252F20523735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20523735&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.8" data-doi="10.1073/pnas.89.18.8750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bridge</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierwater</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSalle</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buss</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Class-level relationships in the phylum Cnidaria: evidence from mitochondrial genome structure.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">8750</span>–<span class="cit-lpage">8753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBridge%26rft.auinit1%253DD.%26rft.volume%253D89%26rft.issue%253D18%26rft.spage%253D8750%26rft.epage%253D8753%26rft.atitle%253DClass-level%2Brelationships%2Bin%2Bthe%2Bphylum%2BCnidaria%253A%2Bevidence%2Bfrom%2Bmitochondrial%2Bgenome%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.18.8750%26rft_id%253Dinfo%253Apmid%252F1356268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMTgvODc1MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.9" data-doi="10.1002/jmor.10899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Budd</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolarski</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Corallite wall and septal microstructure in scleractinian reef corals: comparison of molecular clades within the family Faviidae.</span> <abbr class="cit-jnl-abbrev">J. Morphol.</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMorphology%26rft.stitle%253DJournal%2Bof%2BMorphology%26rft.aulast%253DBudd%26rft.auinit1%253DA.%2BF.%26rft.volume%253D272%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D88%26rft.atitle%253DCorallite%2Bwall%2Band%2Bseptal%2Bmicrostructure%2Bin%2Bscleractinian%2Breef%2Bcorals%253A%2Bcomparison%2Bof%2Bmolecular%2Bclades%2Bwithin%2Bthe%2Bfamily%2BFaviidae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmor.10899%26rft_id%253Dinfo%253Apmid%252F21061280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmor.10899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21061280&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.10" data-doi="10.1093/oxfordjournals.molbev.a026334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castresana</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastresana%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D540%26rft.epage%253D552%26rft.atitle%253DSelection%2Bof%2BConserved%2BBlocks%2Bfrom%2BMultiple%2BAlignments%2Bfor%2BTheir%2BUse%2Bin%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026334%26rft_id%253Dinfo%253Apmid%252F10742046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTQwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.11" data-doi="10.1038/nature08830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkness</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simakov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The dynamic genome of <em>Hydra</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChapman%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D464%26rft.issue%253D7288%26rft.spage%253D592%26rft.epage%253D596%26rft.atitle%253DThe%2Bdynamic%2Bgenome%2Bof%2BHydra.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08830%26rft_id%253Dinfo%253Apmid%252F20228792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20228792&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275974200048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.12" data-doi="10.1038/ismej.2014.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Closek</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSalvo</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piceno</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSantis</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Coral transcriptome and bacterial community profiles reveal distinct Yellow Band Disease states in <em>Orbicella faveolata</em>.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2411</span>–<span class="cit-lpage">2422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D8%26rft.spage%253D2411%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2014.85%26rft_id%253Dinfo%253Apmid%252F24950107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2014.85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950107&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.13" data-doi="10.1186/1471-2164-13-209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conaco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neveu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arcila</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degnan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome profiling of the demosponge <em>Amphimedon queenslandica</em> reveals genome-wide events that accompany major life cycle transitions.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DConaco%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.spage%253D209%26rft.epage%253D209%26rft.atitle%253DTranscriptome%2Bprofiling%2Bof%2Bthe%2Bdemosponge%2BAmphimedon%2Bqueenslandica%2Breveals%2Bgenome-wide%2Bevents%2Bthat%2Baccompany%2Bmajor%2Blife%2Bcycle%2Btransitions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-209%26rft_id%253Dinfo%253Apmid%252F22646746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22646746&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Concepcion</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polato</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baums</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of microsatellite markers from four Hawaiian corals: <em>Acropora cytherea</em>, <em>Fungia scutaria</em>, <em>Montipora capitata</em> and <em>Porites lobata</em>.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet. Resour.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%2BResour.%26rft.volume%253D2%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.15" data-doi="10.1046/j.1096-3642.2003.00084.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fautin</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappola</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Systematics of the Hexacorallia (Cnidaria: Anthozoa).</span> <abbr class="cit-jnl-abbrev">Zool. J. Linn. Soc.</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BJ.%2BLinn.%2BSoc.%26rft.volume%253D139%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1096-3642.2003.00084.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1096-3642.2003.00084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000186903100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.16" data-doi="10.1007/s12526-012-0133-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buschiazzo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel polymorphic microsatellite markers for population genetics of the endangered Caribbean star coral, <em>Montastraea faveolata</em>.</span> <abbr class="cit-jnl-abbrev">Mar. Biodivers.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiodivers.%26rft.volume%253D43%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12526-012-0133-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12526-012-0133-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.17" data-doi="10.1128/MMBR.05014-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davy</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allemand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cell biology of cnidarian-dinoflagellate symbiosis.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDavy%26rft.auinit1%253DS.%2BK.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D261%26rft.atitle%253DCell%2BBiology%2Bof%2BCnidarian-Dinoflagellate%2BSymbiosis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.05014-11%26rft_id%253Dinfo%253Apmid%252F22688813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni8yLzIyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.18" data-doi="10.1111/j.1365-294X.2008.03879.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeSalvo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voolstra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Differential gene expression during thermal stress and bleaching in the Caribbean coral <em>Montastraea faveolata</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3952</span>–<span class="cit-lpage">3971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDesalvo%26rft.auinit1%253DM.%2BK.%26rft.volume%253D17%26rft.issue%253D17%26rft.spage%253D3952%26rft.epage%253D3971%26rft.atitle%253DDifferential%2Bgene%2Bexpression%2Bduring%2Bthermal%2Bstress%2Band%2Bbleaching%2Bin%2Bthe%2BCaribbean%2Bcoral%2BMontastraea%2Bfaveolata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03879.x%26rft_id%253Dinfo%253Apmid%252F18662230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03879.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662230&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259448800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.19" data-doi="10.1016/S0025-326X(03)00037-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Coral bleaching—how and why?</span> <abbr class="cit-jnl-abbrev">Mar. Pollut. Bull.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2BPollution%2BBulletin%26rft.stitle%253DMarine%2BPollution%2BBulletin%26rft.aulast%253DDouglas%26rft.auinit1%253DA.%2BE.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D385%26rft.epage%253D392%26rft.atitle%253DCoral%2Bbleaching--how%2Band%2Bwhy%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0025-326X%252803%252900037-7%26rft_id%253Dinfo%253Apmid%252F12705909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0025-326X(03)00037-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12705909&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182968200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.20" data-doi="10.1038/nature06614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hejnol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browne</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Broad phylogenomic sampling improves resolution of the animal tree of life.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDunn%26rft.auinit1%253DC.%2BW.%26rft.volume%253D452%26rft.issue%253D7188%26rft.spage%253D745%26rft.epage%253D749%26rft.atitle%253DBroad%2Bphylogenomic%2Bsampling%2Bimproves%2Bresolution%2Bof%2Bthe%2Banimal%2Btree%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06614%26rft_id%253Dinfo%253Apmid%252F18322464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18322464&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254792500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.21" data-doi="10.1371/journal.pone.0055990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Silva</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wainwright</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ylitalo-Ward</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Microsatellites for next-generation ecologists: a post-sequencing bioinformatics pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e55990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De55990%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0055990%26rft_id%253Dinfo%253Apmid%252F23424642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0055990&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23424642&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.22" data-doi="10.1016/j.cub.2013.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graspeuntner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Insua</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Regulation of polyp-to-jellyfish transition in <em>Aurelia aurita</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.12.003%26rft_id%253Dinfo%253Apmid%252F24440392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24440392&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.23" data-doi="10.1371/journal.pone.0003222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budd</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mitochondrial and nuclear genes suggest that stony corals are monophyletic but most families of stony corals are not (Order Scleractinia, Class Anthozoa, Phylum Cnidaria).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFukami%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253De3222%26rft.epage%253De3222%26rft.atitle%253DMitochondrial%2Band%2Bnuclear%2Bgenes%2Bsuggest%2Bthat%2Bstony%2Bcorals%2Bare%2Bmonophyletic%2Bbut%2Bmost%2Bfamilies%2Bof%2Bstony%2Bcorals%2Bare%2Bnot%2B%2528Order%2BScleractinia%252C%2BClass%2BAnthozoa%252C%2BPhylum%2BCnidaria%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003222%26rft_id%253Dinfo%253Apmid%252F18795098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18795098&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.24" data-doi="10.1098/rsbl.2008.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jobson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nabholz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blier</span>  <span class="cit-name-given-names">P. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mitochondrial whims: metabolic rate, longevity and the rate of molecular evolution.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2008.0662%26rft_id%253Dinfo%253Apmid%252F19324654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNS8zLzQxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.25" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.26" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental transcriptome of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.27" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinzato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The complex NOD-like receptor repertoire of the coral <em>Acropora digitifera</em> includes novel domain combinations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">167–176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D30%26rft.spage%253D167176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.29" data-doi="10.1371/journal.pone.0084080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Havird</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Performance of single and concatenated sets of mitochondrial genes at inferring metazoan relationships relative to full mitogenome data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e84080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De84080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0084080%26rft_id%253Dinfo%253Apmid%252F24454717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0084080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454717&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.30" data-doi="10.1186/1471-2164-14-266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helm</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterization of differential transcript abundance through time during <em>Nematostella vectensis</em> development.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-266%26rft_id%253Dinfo%253Apmid%252F23601508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601508&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.31" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: a DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2441.32"><div class="cit-metadata unstructured">Illumina, 2014 Illumina customer sequence letter. Illumina, Inc., San Diego.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.33" data-doi="10.1371/journal.pone.0088615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karako-Lampert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoccola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon-Divon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katzenellenbogen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tambutté</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome analysis of the scleractinian coral <em>Stylophora pistillata</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e88615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De88615%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0088615%26rft_id%253Dinfo%253Apmid%252F24551124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0088615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24551124&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.34" data-doi="10.1093/nar/gkf436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xNC8zMDU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.35" data-doi="10.1186/1471-2148-13-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roure</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavrov</span>  <span class="cit-name-given-names">D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cnidarian phylogenetic relationships as revealed by mitogenomics.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-5%26rft_id%253Dinfo%253Apmid%252F23302374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23302374&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.36" data-doi="10.1111/mec.12390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kenkel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene expression under chronic heat stress in populations of the mustard hill coral (<em>Porites astreoides</em>) from different thermal environments.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4322</span>–<span class="cit-lpage">4334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D4322%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.37" data-doi="10.1016/j.ecolmodel.2010.08.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kininmonth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Possingham</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Determining the community structure of the coral <em>Seriatopora hystrix</em> from hydrodynamic and genetic networks.</span> <abbr class="cit-jnl-abbrev">Ecol. Modell.</abbr> <span class="cit-vol">221</span>: <span class="cit-fpage">2870</span>–<span class="cit-lpage">2880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BModell.%26rft.volume%253D221%26rft.spage%253D2870%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2010.08.042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2010.08.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284444300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.38" data-doi="10.1371/journal.pone.0011490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitahara</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolarski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A comprehensive phylogenetic analysis of the Scleractinia (Cnidaria, Anthozoa) based on mitochondrial CO1 sequence data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKitahara%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De11490%26rft.epage%253De11490%26rft.atitle%253DA%2Bcomprehensive%2Bphylogenetic%2Banalysis%2Bof%2Bthe%2BScleractinia%2B%2528Cnidaria%252C%2BAnthozoa%2529%2Bbased%2Bon%2Bmitochondrial%2BCO1%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011490%26rft_id%253Dinfo%253Apmid%252F20628613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20628613&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.39" data-doi="10.1038/nature10382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kocot</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citarella</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohn</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phylogenomics reveals deep molluscan relationships.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKocot%26rft.auinit1%253DK.%2BM.%26rft.volume%253D477%26rft.issue%253D7365%26rft.spage%253D452%26rft.epage%253D456%26rft.atitle%253DPhylogenomics%2Breveals%2Bdeep%2Bmolluscan%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10382%26rft_id%253Dinfo%253Apmid%252F21892190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892190&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295080500038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kocot</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citarella</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moroz</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halanych</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PhyloTreePruner: a phylogenetic tree-based approach for selection of orthologous sequences for phylogenomics.</span> <abbr class="cit-jnl-abbrev">Evol. Bioinform. Online</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBioinform.%2BOnline%26rft.volume%253D9%26rft.spage%253D429%26rft_id%253Dinfo%253Apmid%252F24250218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24250218&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.41" data-doi="10.1016/j.ympev.2010.04.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meusemann</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FASconCAT: Convenient handling of data matrices.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D56%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2010.04.024%26rft_id%253Dinfo%253Apmid%252F20416383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2010.04.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416383&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.42" data-doi="10.1186/1471-2164-13-271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehnert</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burriesci</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developing the anemone <em>Aiptasia</em> as a tractable model for cnidarian-dinoflagellate symbiosis: the transcriptome of aposymbiotic <em>A. pallida</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLehnert%26rft.auinit1%253DE.%2BM.%26rft.volume%253D13%26rft.spage%253D271%26rft.epage%253D271%26rft.atitle%253DDeveloping%2Bthe%2Banemone%2BAiptasia%2Bas%2Ba%2Btractable%2Bmodel%2Bfor%2Bcnidarian-dinoflagellate%2Bsymbiosis%253A%2Bthe%2Btranscriptome%2Bof%2Baposymbiotic%2BA.%2Bpallida.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-271%26rft_id%253Dinfo%253Apmid%252F22726260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726260&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2441.43"><div class="cit-metadata unstructured">Lehnert, E. M., M. E. Mouchka, M. S. Burriesci, N. D. Gallo, J. A. Schwarz <em>et al.</em>, 2014 Extensive differences in gene expression between symbiotic and aposymbiotic Cnidarians. G3 (Bethesda) 4: 277–295.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.44" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: Identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Mesquite: a modular system for evolutionary analysis. v3.02. <a href="http://mesquiteproject.org">http://mesquiteproject.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollrian</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nürnberger</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Development of species-specific markers in an organism with endosymbionts: microsatellites in the scleractinian coral <em>Seriatopora hystrix</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D1%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000174289600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.47" data-doi="10.1002/dneu.20698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marlow</span>  <span class="cit-name-given-names">H. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martindale</span>  <span class="cit-name-given-names">M. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Anatomy and development of the nervous system of <em>Nematostella vectensis</em>, an anthozoan cnidarian.</span> <abbr class="cit-jnl-abbrev">Dev. Neurobiol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DDev%2BNeurobiol%26rft.aulast%253DMarlow%26rft.auinit1%253DH.%2BQ.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D235%26rft.epage%253D254%26rft.atitle%253DAnatomy%2Band%2Bdevelopment%2Bof%2Bthe%2Bnervous%2Bsystem%2Bof%2BNematostella%2Bvectensis%252C%2Ban%2Banthozoan%2Bcnidarian.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdneu.20698%26rft_id%253Dinfo%253Apmid%252F19170043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dneu.20698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19170043&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263847700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.48" data-doi="10.4319/lo.2003.48.1_part_2.0402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazel</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesser</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorbunov</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrell</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Green-fluorescent proteins in Caribbean corals.</span> <abbr class="cit-jnl-abbrev">Limnol. Oceanogr.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLimnol.%2BOceanogr.%26rft.volume%253D48%26rft.spage%253D402%26rft_id%253Dinfo%253Adoi%252F10.4319%252Flo.2003.48.1_part_2.0402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4319/lo.2003.48.1_part_2.0402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182049800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Study of cnidarian-algal symbiosis in the “omics” age.</span> <abbr class="cit-jnl-abbrev">Biol. Bull.</abbr> <span class="cit-vol">223</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiological%2BBulletin%26rft.stitle%253DBiol.%2BBull.%26rft.aulast%253DMeyer%26rft.auinit1%253DE.%26rft.volume%253D223%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D65%26rft.atitle%253DStudy%2Bof%2BCnidarian-Algal%2BSymbiosis%2Bin%2Bthe%2B%2522Omics%2522%2BAge%26rft_id%253Dinfo%253Apmid%252F22983032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbGJ1bGwiO3M6NToicmVzaWQiO3M6ODoiMjIzLzEvNDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.50" data-doi="10.1186/1471-2164-10-219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aglyamova</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan-Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrego</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequencing and <em>de novo</em> analysis of a coral larval transcriptome using 454 GSFlx.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMeyer%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D219%26rft.epage%253D219%26rft.atitle%253DSequencing%2Band%2Bde%2Bnovo%2Banalysis%2Bof%2Ba%2Bcoral%2Blarval%2Btranscriptome%2Busing%2B454%2BGSFlx.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-219%26rft_id%253Dinfo%253Apmid%252F19435504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19435504&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aglyamova</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Profiling gene expression responses of coral larvae (<em>Acropora millepora</em>) to elevated temperature and settlement inducers using a novel RNA-Seq procedure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3599</span>–<span class="cit-lpage">3616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMeyer%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D17%26rft.spage%253D3599%26rft.epage%253D3616%26rft.atitle%253DProfiling%2Bgene%2Bexpression%2Bresponses%2Bof%2Bcoral%2Blarvae%2B%2528Acropora%2Bmillepora%2529%2Bto%2Belevated%2Btemperature%2Band%2Bsettlement%2Binducers%2Busing%2Ba%2Bnovel%2BRNA-Seq%2Bprocedure.%26rft_id%253Dinfo%253Apmid%252F21801258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21801258&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294224400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.52" data-doi="10.1093/nar/gkm321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">KAAS: an automatic genome annotation and pathway reconstruction server.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>(<span class="cit-supplement">Suppl 2</span>): <span class="cit-fpage">W182</span>–<span class="cit-lpage">W185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm321%26rft_id%253Dinfo%253Apmid%252F17526522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8yL1cxODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.53" data-doi="10.1111/j.1365-294X.2012.05554.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grasso</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole transcriptome analysis of the coral A<em>cropora millepora</em> reveals complex responses to CO2‐driven acidification during the initiation of calcification.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2440</span>–<span class="cit-lpage">2454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMoya%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D10%26rft.spage%253D2440%26rft.epage%253D2454%26rft.atitle%253DWhole%2Btranscriptome%2Banalysis%2Bof%2Bthe%2Bcoral%2BAcropora%2Bmillepora%2Breveals%2Bcomplex%2Bresponses%2Bto%2BCO2-driven%2Bacidification%2Bduring%2Bthe%2Binitiation%2Bof%2Bcalcification.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05554.x%26rft_id%253Dinfo%253Apmid%252F22490231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05554.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22490231&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303388300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.54" data-doi="10.1371/journal.pone.0059534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakasugi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bally</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellens</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De Novo</em> transcriptome sequence assembly and analysis of RNA silencing genes of <em>Nicotiana benthamiana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e59534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De59534%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059534%26rft_id%253Dinfo%253Apmid%252F23555698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0059534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555698&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.55" data-doi="10.1186/1471-2164-14-465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emrich</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assessing <em>De Novo</em> transcriptome assembly metrics for consistency and utility.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-465%26rft_id%253Dinfo%253Apmid%252F23837739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23837739&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.56" data-doi="10.1186/1471-2164-11-310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dzurisin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmichael</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emrich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population-level transcriptome sequencing of nonmodel organisms <em>Erynnis propertius</em> and <em>Papilio zelicaon</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DO%2527Neil%26rft.auinit1%253DS.%2BT.%26rft.volume%253D11%26rft.spage%253D310%26rft.epage%253D310%26rft.atitle%253DPopulation-level%2Btranscriptome%2Bsequencing%2Bof%2Bnonmodel%2Borganisms%2BErynnis%2Bpropertius%2Band%2BPapilio%2Bzelicaon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-310%26rft_id%253Dinfo%253Apmid%252F20478048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20478048&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.57" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzEwNjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.58" data-doi="10.1093/molbev/msh182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bapteste</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenomics of eukaryotes: impact of missing data on large alignments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1740</span>–<span class="cit-lpage">1752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh182%26rft_id%253Dinfo%253Apmid%252F15175415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTc0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.59" data-doi="10.1371/journal.pone.0028634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polato</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vera</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baums</span>  <span class="cit-name-given-names">I. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene discovery in the threatened elkhorn coral: 454 sequencing of the <em>Acropora palmata</em> transcriptome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPolato%26rft.auinit1%253DN.%2BR.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28634%26rft.epage%253De28634%26rft.atitle%253DGene%2Bdiscovery%2Bin%2Bthe%2Bthreatened%2Belkhorn%2Bcoral%253A%2B454%2Bsequencing%2Bof%2Bthe%2BAcropora%2Bpalmata%2Btranscriptome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028634%26rft_id%253Dinfo%253Apmid%252F22216101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22216101&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.60" data-doi="10.1016/j.dci.2014.06.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TIR-domain-containing protein repertoire of nine anthozoan species reveals coral–specific expansions and uncharacterized proteins.</span> <abbr class="cit-jnl-abbrev">Dev. Comp. Immunol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BComp.%2BImmunol.%26rft.volume%253D46%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2014.06.002%26rft_id%253Dinfo%253Apmid%252F24933613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2014.06.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24933613&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.61" data-doi="10.1111/1755-0998.12383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pratlong</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haguenauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrol</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klopp</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontarotti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The red coral (<em>Corallium rubrum</em>) transcriptome: a new resource for population genetics and local adaptation studies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D1205%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12383%26rft_id%253Dinfo%253Apmid%252F25648864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25648864&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.62" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FastTree 2–approximately maximum-likelihood trees for large alignments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPrice%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9490%26rft.epage%253De9490%26rft.atitle%253DFastTree%2B2--approximately%2Bmaximum-likelihood%2Btrees%2Bfor%2Blarge%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft_id%253Dinfo%253Apmid%252F20224823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224823&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.63" data-doi="10.1126/science.1139158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellsten</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dirks</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPutnam%26rft.auinit1%253DN.%2BH.%26rft.volume%253D317%26rft.issue%253D5834%26rft.spage%253D86%26rft.epage%253D94%26rft.atitle%253DSea%2BAnemone%2BGenome%2BReveals%2BAncestral%2BEumetazoan%2BGene%2BRepertoire%2Band%2BGenomic%2BOrganization%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1139158%26rft_id%253Dinfo%253Apmid%252F17615350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMTcvNTgzNC84NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quast</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pruesse</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerken</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The SILVA ribosomal RNA gene database project: improved data processing and web-based tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D590–D596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D41%26rft.spage%253DD590D596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Symbiosis-enhanced gene expression in cnidarian-algal associations: cloning and characterization of a cDNA, sym32, encoding a possible cell adhesion protein.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComp.%2BBiochem.%26rft.volume%253D126%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.66" data-doi="10.1186/1742-9994-9-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riesgo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrade</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Porro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative description of ten transcriptomes of newly sequenced invertebrates and efficiency estimation of genomic sampling in non-model taxa.</span> <abbr class="cit-jnl-abbrev">Front. Zool.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BZool%26rft.aulast%253DRiesgo%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D33%26rft.atitle%253DComparative%2Bdescription%2Bof%2Bten%2Btranscriptomes%2Bof%2Bnewly%2Bsequenced%2Binvertebrates%2Band%2Befficiency%2Bestimation%2Bof%2Bgenomic%2Bsampling%2Bin%2Bnon-model%2Btaxa.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-9994-9-33%26rft_id%253Dinfo%253Apmid%252F23190771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-9994-9-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23190771&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.67" data-doi="10.1093/molbev/msu057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riesgo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giribet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leys</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The analysis of eight transcriptomes from all poriferan classes reveals surprising genetic complexity in sponges.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu057%26rft_id%253Dinfo%253Apmid%252F24497032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzUvMTEwMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.68" data-doi="10.1126/science.271.5249.640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Evolution of scleractinian corals inferred from molecular systematics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRomano%26rft.auinit1%253DS.%2BL.%26rft.volume%253D271%26rft.issue%253D5249%26rft.spage%253D640%26rft.epage%253D642%26rft.atitle%253DEvolution%2Bof%2BScleractinian%2BCorals%2BInferred%2Bfrom%2BMolecular%2BSystematics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.271.5249.640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzEvNTI0OS82NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.69" data-doi="10.1016/j.ympev.2009.01.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rota-Stabelli</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telford</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MtZoa: A general mitochondrial amino acid substitutions model for animal evolutionary studies.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D52%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2009.01.011%26rft_id%253Dinfo%253Apmid%252F19475713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2009.01.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19475713&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266692800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.70" data-doi="10.1093/molbev/mss208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roure</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baurain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of missing data on phylogenies inferred from empirical phylogenomic data sets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss208%26rft_id%253Dinfo%253Apmid%252F22930702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzEvMTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2441.71"><div class="cit-metadata unstructured">Rozen, S., and H. Skaletsky, 1999 Primer3 on the WWW for general users and for biologist programmers, pp. 365–386 in <em>Bioinformatics Methods and Protocols</em>. Springer, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.72" data-doi="10.1186/1471-2164-15-939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Ramos</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baums</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microsatellite abundance across the Anthozoa and Hydrozoa in the phylum Cnidaria.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D939%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-939%26rft_id%253Dinfo%253Apmid%252F25346285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25346285&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.73" data-doi="10.1186/gb-2006-7-7-r64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazza</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwong</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The cnidarian-bilaterian ancestor possessed at least 56 homeoboxes: evidence from the starlet sea anemone, <em>Nematostella vectensis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRyan%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR64%26rft.epage%253DR64%26rft.atitle%253DThe%2Bcnidarian-bilaterian%2Bancestor%2Bpossessed%2Bat%2Bleast%2B56%2Bhomeoboxes%253A%2Bevidence%2Bfrom%2Bthe%2Bstarlet%2Bsea%2Banemone%252C%2BNematostella%2Bvectensis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r64%26rft_id%253Dinfo%253Apmid%252F16867185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r64&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16867185&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnitzler</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">A.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreland</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genome of the ctenophore <em>Mnemiopsis leidyi</em> and its implications for cell type evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: 1232592.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Populations of <em>Symbiodinium muscatinei s</em>how strong biogeographic structuring in the intertidal anemone <em>Anthopleura elegantissima</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Bull.</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiological%2BBulletin%26rft.stitle%253DBiol.%2BBull.%26rft.aulast%253DSanders%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D220%26rft.issue%253D3%26rft.spage%253D199%26rft.epage%253D208%26rft.atitle%253DPopulations%2Bof%2BSymbiodinium%2Bmuscatinei%2BShow%2BStrong%2BBiogeographic%2BStructuring%2Bin%2Bthe%2BIntertidal%2BAnemone%2BAnthopleura%2Belegantissima%26rft_id%253Dinfo%253Apmid%252F21712228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbGJ1bGwiO3M6NToicmVzaWQiO3M6OToiMjIwLzMvMTk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.76" data-doi="10.1186/1471-2164-15-406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shcheglovitova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Differential gene expression between functionally specialized polyps of the colonial hydrozoan <em>Hydractinia symbiolongicarpus</em> (Phylum Cnidaria).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D406%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-406%26rft_id%253Dinfo%253Apmid%252F24884766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24884766&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.77" data-doi="10.1016/j.margen.2010.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnitzler</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Coral larvae exhibit few measurable transcriptional changes during the onset of coral-dinoflagellate endosymbiosis.</span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMar%2BGenomics%26rft.aulast%253DSchnitzler%26rft.auinit1%253DC.%2BE.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D107%26rft.epage%253D116%26rft.atitle%253DCoral%2Blarvae%2Bexhibit%2Bfew%2Bmeasurable%2Btranscriptional%2Bchanges%2Bduring%2Bthe%2Bonset%2Bof%2Bcoral-dinoflagellate%2Bendosymbiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.margen.2010.08.002%26rft_id%253Dinfo%253Apmid%252F21798204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.margen.2010.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21798204&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284347100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.78" data-doi="10.1111/j.1461-0248.2006.00889.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selkoe</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DSelkoe%26rft.auinit1%253DK.%2BA.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D615%26rft.epage%253D629%26rft.atitle%253DMicrosatellites%2Bfor%2Becologists%253A%2Ba%2Bpractical%2Bguide%2Bto%2Busing%2Band%2Bevaluating%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00889.x%26rft_id%253Dinfo%253Apmid%252F16643306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00889.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16643306&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236799600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.79" data-doi="10.1111/mec.12861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baums</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Geographic differences in vertical connectivity in the Caribbean coral <em>Montastraea cavernosa</em> despite high levels of horizontal connectivity at shallow depths.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4226</span>–<span class="cit-lpage">4240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4226%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000341176200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.80" data-doi="10.1007/s00338-004-0442-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shearer</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez-Rodríguez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffroth</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Generating molecular markers from zooxanthellate cnidarians.</span> <abbr class="cit-jnl-abbrev">Coral Reefs</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCoral%2BReefs%26rft.volume%253D24%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00338-004-0442-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00338-004-0442-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227742400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.81" data-doi="10.1038/nature10249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinzato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hisata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the <em>Acropora digitifera</em> genome to understand coral responses to environmental change.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DShinzato%26rft.auinit1%253DC.%26rft.volume%253D476%26rft.issue%253D7360%26rft.spage%253D320%26rft.epage%253D323%26rft.atitle%253DUsing%2Bthe%2BAcropora%2Bdigitifera%2Bgenome%2Bto%2Bunderstand%2Bcoral%2Bresponses%2Bto%2Benvironmental%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10249%26rft_id%253Dinfo%253Apmid%252F21785439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21785439&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294206500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.82" data-doi="10.2108/zsj.29.510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinzato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">The repertoire of chemical defense genes in the coral <em>Acropora digitifera</em> genome.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DShinzato%26rft.auinit1%253DC.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D510%26rft.epage%253D517%26rft.atitle%253DThe%2Brepertoire%2Bof%2Bchemical%2Bdefense%2Bgenes%2Bin%2Bthe%2Bcoral%2BAcropora%2Bdigitifera%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.29.510%26rft_id%253Dinfo%253Apmid%252F22873809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.29.510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22873809&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.83" data-doi="10.2108/zsj.29.260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinzato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Fluorescent protein candidate genes in the coral <em>Acropora digitifera</em> genome.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DShinzato%26rft.auinit1%253DC.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D260%26rft.epage%253D264%26rft.atitle%253DFluorescent%2Bprotein%2Bcandidate%2Bgenes%2Bin%2Bthe%2Bcoral%2BAcropora%2Bdigitifera%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.29.260%26rft_id%253Dinfo%253Apmid%252F22468836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.29.260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22468836&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.84" data-doi="10.1371/journal.pone.0085182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinzato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusakabe</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A snapshot of a coral “holobiont”: a transcriptome assembly of the scleractinian coral, <em>Porites</em>, captures a wide variety of genes from both the host and symbiotic zooxanthellae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85182%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085182%26rft_id%253Dinfo%253Apmid%252F24454815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454815&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.85" data-doi="10.1371/journal.pone.0022953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siebert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tintori</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goetz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helm</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differential gene expression in the siphonophore <em>Nanomia bijuga</em> (Cnidaria) assessed with multiple next-generation sequencing workflows.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e22953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSiebert%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De22953%26rft.epage%253De22953%26rft.atitle%253DDifferential%2Bgene%2Bexpression%2Bin%2Bthe%2Bsiphonophore%2BNanomia%2Bbijuga%2B%2528Cnidaria%2529%2Bassessed%2Bwith%2Bmultiple%2Bnext-generation%2Bsequencing%2Bworkflows.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022953%26rft_id%253Dinfo%253Apmid%252F21829563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829563&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-web" id="cit-5.11.2441.86"><div class="cit-metadata unstructured">Smit, A., R. Hubley, and P. Green, RepeatMasker Open-3.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.87" data-doi="10.1007/s10126-007-9049-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith-Keune</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dove</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene expression of a green fluorescent protein homolog as a host-specific biomarker of heat stress within a reef-building coral.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DSmith-Keune%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D180%26rft.atitle%253DGene%2Bexpression%2Bof%2Ba%2Bgreen%2Bfluorescent%2Bprotein%2Bhomolog%2Bas%2Ba%2Bhost-specific%2Bbiomarker%2Bof%2Bheat%2Bstress%2Bwithin%2Ba%2Breef-building%2Bcoral.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-007-9049-6%26rft_id%253Dinfo%253Apmid%252F17994304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-007-9049-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994304&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.88" data-doi="10.1111/j.1742-4658.2009.07474.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soza‐Ried</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hotz‐Wagenblatt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glatting</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Val</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellenberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The transcriptome of the colonial marine hydroid <em>Hydractinia echinata</em>.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DSoza-Ried%26rft.auinit1%253DJ.%26rft.volume%253D277%26rft.issue%253D1%26rft.spage%253D197%26rft.epage%253D209%26rft.atitle%253DThe%2Btranscriptome%2Bof%2Bthe%2Bcolonial%2Bmarine%2Bhydroid%2BHydractinia%2Bechinata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2009.07474.x%26rft_id%253Dinfo%253Apmid%252F19961538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2009.07474.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961538&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.89" data-doi="10.1038/nature09201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simakov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahey</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">M. E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The <em>Amphimedon queenslandic</em>a genome and the evolution of animal complexity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSrivastava%26rft.auinit1%253DM.%26rft.volume%253D466%26rft.issue%253D7307%26rft.spage%253D720%26rft.epage%253D726%26rft.atitle%253DThe%2BAmphimedon%2Bqueenslandica%2Bgenome%2Band%2Bthe%2Bevolution%2Bof%2Banimal%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09201%26rft_id%253Dinfo%253Apmid%252F20686567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20686567&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280562500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.90" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML Version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.91" data-doi="10.1186/1471-2164-15-71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stefanik</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubinski</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granger</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrd</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reitzel</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Production of a reference transcriptome and transcriptomic database (EdwardsiellaBase) for the lined sea anemone, <em>Edwardsiella lineata</em>, a parasitic cnidarian.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-71%26rft_id%253Dinfo%253Apmid%252F24467778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24467778&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.92" data-doi="10.1007/s10126-012-9482-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lun</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">J.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PcarnBase: Development of a transcriptomic database for the brain coral <em>Platygyra carnosus</em>.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiotechnol.%2B%2528NY%2529%26rft.volume%253D15%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-012-9482-z%26rft_id%253Dinfo%253Apmid%252F22875536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-012-9482-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22875536&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.11.2441.93" data-doi="10.1038/75556"><div class="cit-metadata"><cite>Ashburner, M., C. A. Ball, J. A. Blake, D. Botstein <em>et al</em>., <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene Ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.94" data-doi="10.1186/1471-2164-12-585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Traylor-Knowles</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granger</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubinski</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parikh</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garamszegi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Production of a reference transcriptome and transcriptomic database (PocilloporaBase) for the cauliflower coral, <em>Pocillopora damicornis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTraylor-Knowles%26rft.auinit1%253DN.%26rft.volume%253D12%26rft.spage%253D585%26rft.epage%253D585%26rft.atitle%253DProduction%2Bof%2Ba%2Breference%2Btranscriptome%2Band%2Btranscriptomic%2Bdatabase%2B%2528PocilloporaBase%2529%2Bfor%2Bthe%2Bcauliflower%2Bcoral%252C%2BPocillopora%2Bdamicornis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-585%26rft_id%253Dinfo%253Apmid%252F22126435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22126435&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.95" data-doi="10.1111/j.1471-8286.2005.01183.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souter</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballment</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Development of 10 polymorphic microsatellite markers from herbicide-bleached tissues of the brooding pocilloporid coral <em>Seriatopora hystrix</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D6%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01183.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2005.01183.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235725600056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiple scales of genetic connectivity in a brooding coral on isolated reefs following catastrophic bleaching.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DUnderwood%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D771%26rft.epage%253D784%26rft.atitle%253DMultiple%2Bscales%2Bof%2Bgenetic%2Bconnectivity%2Bin%2Ba%2Bbrooding%2Bcoral%2Bon%2Bisolated%2Breefs%2Bfollowing%2Bcatastrophic%2Bbleaching.%26rft_id%253Dinfo%253Apmid%252F17284210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17284210&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244004400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2441.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Dongen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Graph clustering by flow simulation</span>, <span class="cit-publ-name">University of Utrecht</span>, <span class="cit-publ-loc">The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.98" data-doi="10.1007/s00239-001-0075-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catmull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hislop</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagerman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The mitochondrial genome of <em>Acropora tenuis</em> (Cnidaria; Scleractinia) contains a large group I intron and a candidate control region.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253Dvan%2BOppen%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D13%26rft.atitle%253DThe%2Bmitochondrial%2Bgenome%2Bof%2BAcropora%2Btenuis%2B%2528Cnidaria%253B%2BScleractinia%2529%2Bcontains%2Ba%2Blarge%2Bgroup%2BI%2Bintron%2Band%2Ba%2Bcandidate%2Bcontrol%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0075-0%26rft_id%253Dinfo%253Apmid%252F12165838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0075-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12165838&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176243200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.99" data-doi="10.1371/journal.pone.0003401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">M. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De’ath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peplow</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kininmonth</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic traces of recent long-distance dispersal in a predominantly self-recruiting coral.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dvan%2BOppen%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3401%26rft.epage%253De3401%26rft.atitle%253DGenetic%2Btraces%2Bof%2Brecent%2Blong-distance%2Bdispersal%2Bin%2Ba%2Bpredominantly%2Bself-recruiting%2Bcoral.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003401%26rft_id%253Dinfo%253Apmid%252F18852897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852897&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.100" data-doi="10.1371/journal.pone.0058652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vidal-Dupiol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoccola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tambutté</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosseau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genes related to ion-transport and energy production are upregulated in response to CO2-driven pH decrease in corals: New insights from transcriptome analysis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e58652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De58652%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0058652%26rft_id%253Dinfo%253Apmid%252F23544045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0058652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23544045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.101" data-doi="10.1242/jeb.009597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cellular mechanisms of Cnidarian bleaching: stress causes the collapse of symbiosis.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DWeis%26rft.auinit1%253DV.%2BM.%26rft.volume%253D211%26rft.issue%253D19%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DCellular%2Bmechanisms%2Bof%2BCnidarian%2Bbleaching%253A%2Bstress%2Bcauses%2Bthe%2Bcollapse%2Bof%2Bsymbiosis%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.009597%26rft_id%253Dinfo%253Apmid%252F18805804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTEvMTkvMzA1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.102" data-doi="10.1126/science.1172540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allemand</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">What determines coral health?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWeis%26rft.auinit1%253DV.%2BM.%26rft.volume%253D324%26rft.issue%253D5931%26rft.spage%253D1153%26rft.epage%253D1155%26rft.atitle%253DWhat%2BDetermines%2BCoral%2BHealth%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172540%26rft_id%253Dinfo%253Apmid%252F19478172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzMS8xMTUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.103" data-doi="10.1186/1471-2164-14-204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galliot</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNAseq <em>vs.</em> genome-predicted transcriptomes: a large population of novel transcripts identified in an Illumina-454 <em>Hydra</em> transcriptome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-204%26rft_id%253Dinfo%253Apmid%252F23530871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23530871&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.104" data-doi="10.1093/oxfordjournals.molbev.a003851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A general empirical model of protein evolution derived from multiple protein families using a Maximum-Likelihood approach.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DWhelan%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D691%26rft.epage%253D699%26rft.atitle%253DA%2BGeneral%2BEmpirical%2BModel%2Bof%2BProtein%2BEvolution%2BDerived%2Bfrom%2BMultiple%2BProtein%2BFamilies%2BUsing%2Ba%2BMaximum-Likelihood%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003851%26rft_id%253Dinfo%253Apmid%252F11319253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvNjkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2441.105" data-doi="10.1016/j.dci.2009.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood-Charlson</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">V. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The diversity of C-type lectins in the genome of a basal metazoan, <em>Nematostella vectensis</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Comp. Immunol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DWood-Charlson%26rft.auinit1%253DE.%2BM.%26rft.volume%253D33%26rft.issue%253D8%26rft.spage%253D881%26rft.epage%253D889%26rft.atitle%253DThe%2Bdiversity%2Bof%2BC-type%2Blectins%2Bin%2Bthe%2Bgenome%2Bof%2Ba%2Bbasal%2Bmetazoan%252C%2BNematostella%2Bvectensis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2009.01.008%26rft_id%253Dinfo%253Apmid%252F19454330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2009.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19454330&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2441.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2441.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2429" title="Slit Binding via the Ig1 Domain Is Essential for Midline Repulsion by Drosophila Robo1 but Dispensable for Receptor Expression, Localization, and Regulation in Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2453" title="Whole-Genome Sequencing Suggests Schizophrenia Risk Mechanisms in Humans with 22q11.2 Deletion Syndrome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2441" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L0xRv8wxN9WDotjmv2zh5M65FhkTmsJsU-puev2ILpA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2441" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2441" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2441" class="active" data-icon-position="" data-hide-link-title="0">De Novo Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460165" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Zk9RfkvCmLKaSPxyx-d4l8-oJ0nnJBnWINJY9s_GgDg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2441.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2441" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rssImfAN-M-xpAOxiL2EcwImApa0nCIYose_wWZD5og" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2441" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2441.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460165" id="node460165--2" data-pisa="ggg;5/11/2441" data-pisa-master="ggg;g3.115.020164" data-apath="/ggg/5/11/2441.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheila A. Kitchen</span>, <span class="highwire-citation-author" data-delta="1">Camerron M. Crowder</span>, <span class="highwire-citation-author" data-delta="2">Angela Z. Poole</span>, <span class="highwire-citation-author" data-delta="3">Virginia M. Weis</span> and <span class="highwire-citation-author" data-delta="4">Eli Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2441-2452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020164 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460165--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheila A. Kitchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BA.%2BKitchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kitchen%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheila%2BA.%2BKitchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheila A. Kitchen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kitchens@science.oregonstate.edu" class="" data-icon-position="" data-hide-link-title="0">kitchens@science.oregonstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Camerron M. Crowder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camerron%2BM.%2BCrowder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crowder%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamerron%2BM.%2BCrowder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Angela Z. Poole </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BZ.%2BPoole%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poole%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BZ.%2BPoole%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Virginia M. Weis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BM.%2BWeis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weis%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BM.%2BWeis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eli Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eli%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEli%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2441" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2441&amp;text=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2441&amp;t=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2441&amp;title=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460165/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460165" id="node460165--3" data-pisa="ggg;5/11/2441" data-pisa-master="ggg;g3.115.020164" data-apath="/ggg/5/11/2441.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>De Novo</em> Assembly and Characterization of Four Anthozoan (Phylum Cnidaria) Transcriptomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sheila A. Kitchen</span>, <span class="highwire-citation-author" data-delta="1">Camerron M. Crowder</span>, <span class="highwire-citation-author" data-delta="2">Angela Z. Poole</span>, <span class="highwire-citation-author" data-delta="3">Virginia M. Weis</span> and <span class="highwire-citation-author" data-delta="4">Eli Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2441-2452;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020164 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460165--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sheila A. Kitchen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BA.%2BKitchen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kitchen%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASheila%2BA.%2BKitchen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4402-8139" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sheila A. Kitchen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kitchens@science.oregonstate.edu" class="" data-icon-position="" data-hide-link-title="0">kitchens@science.oregonstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Camerron M. Crowder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camerron%2BM.%2BCrowder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crowder%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamerron%2BM.%2BCrowder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Angela Z. Poole </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BZ.%2BPoole%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poole%20AZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BZ.%2BPoole%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Virginia M. Weis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BM.%2BWeis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weis%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BM.%2BWeis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eli Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, Oregon State University, Corvallis, Oregon 97331</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eli%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEli%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2441&amp;rft.epage=2452&amp;rft.atitle=De%20Novo%20Assembly%20and%20Characterization%20of%20Four%20Anthozoan%20%28Phylum%20Cnidaria%29%20Transcriptomes&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kitchen%2C+Sheila+A.&amp;rft.au=Crowder%2C+Camerron+M.&amp;rft.au=Poole%2C+Angela+Z.&amp;rft.au=Weis%2C+Virginia+M.&amp;rft.au=Meyer%2C+Eli"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460165/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460165/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460165/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460165/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460165/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460165/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460165/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460165/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460165/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460165/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460165/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460165/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2441" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2441.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2441.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2441.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2441.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
