<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2475" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2475.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2475.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2475.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2475.atom" />
<meta name="HW.pisa" content="ggg;5/11/2475" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" />
<meta name="DC.Identifier" content="10.1534/g3.115.020180" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Methanogenesis and its key small-molecule methyltransferase Mtr complex are poorly understood despite their pivotal role in Earth’s global carbon cycle. Mtr complex is encoded by a conserved mtrEDCBAFGH operon in most methanogens. Here we report that two discrete lineages, Methanococcales and Methanomicrobiales , have a noncanonical mtr operon carrying two copies of mtrA resulting from an ancient duplication. Compared to mtrA-1 , mtrA-2 acquires a distinct transmembrane domain through domain shuffling and gene fusion. However, the nontransmembrane domains (MtrA domain) of mtrA-1 and mtrA-2 are homogenized by gene conversion events lasting throughout the long history of these extant methanogens (over 2410 million years). Furthermore, we identified a possible recruitment of ancient nonmethanogenic methyltransferase genes to establish the methanogenesis pathway. These results not only provide novel evolutionary insight into the methanogenesis pathway and methyltransferase superfamily but also suggest an unanticipated long-lasting effect of gene conversion on gene evolution in a convergent pattern." />
<meta name="DC.Contributor" content="Sishuo Wang" />
<meta name="DC.Contributor" content="Youhua Chen" />
<meta name="DC.Contributor" content="Qinhong Cao" />
<meta name="DC.Contributor" content="Huiqiang Lou" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Methanogenesis and its key small-molecule methyltransferase Mtr complex are poorly understood despite their pivotal role in Earth’s global carbon cycle. Mtr complex is encoded by a conserved &lt;i&gt;mtrEDCBAFGH&lt;/i&gt; operon in most methanogens. Here we report that two discrete lineages, &lt;i&gt;Methanococcales&lt;/i&gt; and &lt;i&gt;Methanomicrobiales&lt;/i&gt;, have a noncanonical &lt;i&gt;mtr&lt;/i&gt; operon carrying two copies of &lt;i&gt;mtrA&lt;/i&gt; resulting from an ancient duplication. Compared to &lt;i&gt;mtrA-1&lt;/i&gt;, &lt;i&gt;mtrA-2&lt;/i&gt; acquires a distinct transmembrane domain through domain shuffling and gene fusion. However, the nontransmembrane domains (MtrA domain) of &lt;i&gt;mtrA-1&lt;/i&gt; and &lt;i&gt;mtrA-2&lt;/i&gt; are homogenized by gene conversion events lasting throughout the long history of these extant methanogens (over 2410 million years). Furthermore, we identified a possible recruitment of ancient nonmethanogenic methyltransferase genes to establish the methanogenesis pathway. These results not only provide novel evolutionary insight into the methanogenesis pathway and methyltransferase superfamily but also suggest an unanticipated long-lasting effect of gene conversion on gene evolution in a convergent pattern.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2475" />
<meta name="citation_id" content="5/11/2475" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2475" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2475.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2475.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2475.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020180" />
<meta name="citation_pmid" content="26384370" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2475" />
<meta name="citation_lastpage" content="2486" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sishuo Wang" />
<meta name="citation_author_institution" content="State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author_institution" content="Department of Botany, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_email" content="wangsishuo@yeah.net" />
<meta name="citation_author_email" content="lou@cau.edu.cn" />
<meta name="citation_author" content="Youhua Chen" />
<meta name="citation_author_institution" content="Department of Zoology, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Department of Renewable Resources, University of Alberta, Edmonton, T6G 2H1, Canada" />
<meta name="citation_author" content="Qinhong Cao" />
<meta name="citation_author_institution" content="State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author" content="Huiqiang Lou" />
<meta name="citation_author_institution" content="State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China" />
<meta name="citation_author_email" content="wangsishuo@yeah.net" />
<meta name="citation_author_email" content="lou@cau.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4465-6186" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. H. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. R. Baker;citation_author=V. Hanson-Smith;citation_author=A. D. Johnson;citation_title=Following Gene Duplication, Paralog Interference Constrains Transcriptional Circuit Evolution;citation_pages=104-108;citation_volume=342;citation_year=2013;citation_issue=6154;citation_pmid=24092741;citation_doi=10.1126/science.1240810" />
<meta name="citation_reference" content="citation_journal_abbrev=Archaea;citation_author=E. Bapteste;citation_author=C. Brochier;citation_author=Y. Boucher;citation_title=Higher-level classification of the Archaea: evolution of methanogenesis and methanogens.;citation_pages=353-363;citation_volume=1;citation_year=2005;citation_issue=5;citation_pmid=15876569;citation_doi=10.1155/2005/859728" />
<meta name="citation_reference" content="citation_title=The Timetree of Life;citation_title=Archaebacteria;citation_author=U. F. Battistuzzi;citation_author=S. B. Hedges;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=B. Becher;citation_author=V. Muller;citation_author=G. Gottschalk;citation_title=N5-methyl-tetrahydromethanopterin:coenzyme M methyltransferase of Methanosarcina strain Go1 is an Na(+)-translocating membrane protein.;citation_pages=7656-7660;citation_volume=174;citation_year=1992;citation_issue=23;citation_pmid=1447136;citation_doi=10.1128/jb.174.23.7656-7660.1992" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. Borrel;citation_author=P. W. O’Toole;citation_author=H. M. Harris;citation_author=P. Peyret;citation_author=J. F. Brugere;citation_title=Phylogenomic data support a seventh order of methylotrophic methanogens and provide insights into the evolution of methanogenesis.;citation_volume=5;citation_year=2013;citation_pmid=23985970;citation_doi=10.1093/gbe/evt128" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Breuert;citation_author=T. Allers;citation_author=G. Spohn;citation_author=J. Soppa;citation_title=Regulated polyploidy in halophilic archaea.;citation_pages=e92-e92;citation_volume=1;citation_year=2006;citation_pmid=17183724;citation_doi=10.1371/journal.pone.0000092" />
<meta name="citation_reference" content="citation_title=Planctomycetes: Cell Structure, Origins and Biology;citation_title=The distribution and evolution of C1 transfer enzymes and evolution of the planctomycetes;citation_author=L. Chistoserdova;citation_pages=195-209;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=ProtTest 3: fast selection of best-fit models of protein evolution.;citation_volume=27;citation_year=2011;citation_pmid=21335321;citation_doi=10.1093/bioinformatics/btr088" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=jModelTest 2: more models, new heuristics and parallel computing.;citation_pages=772-772;citation_volume=9;citation_year=2012;citation_issue=8;citation_pmid=22847109;citation_doi=10.1038/nmeth.2109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. R. Eddy;citation_title=Profile hidden Markov models;citation_pages=755-763;citation_volume=14;citation_year=1998;citation_issue=9;citation_pmid=9918945;citation_doi=10.1093/bioinformatics/14.9.755" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=J. A. Fawcett;citation_author=H. Innan;citation_title=Neutral and non-neutral evolution of duplicated genes with gene conversion.;citation_pages=191-209;citation_volume=2;citation_year=2011;citation_pmid=24710144;citation_doi=10.3390/genes2010191" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=G. P. Fournier;citation_author=J. P. Gogarten;citation_title=Evolution of acetoclastic methanogenesis in Methanosarcina via horizontal gene transfer from cellulolytic Clostridia.;citation_volume=190;citation_year=2008;citation_pmid=18055595;citation_doi=10.1128/JB.01382-07" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Gao;citation_author=R. S. Gupta;citation_title=Phylogenomic analysis of proteins that are distinctive of Archaea and its main subgroups and the origin of methanogenesis.;citation_pages=86-86;citation_volume=8;citation_year=2007;citation_pmid=17394648;citation_doi=10.1186/1471-2164-8-86" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Goto;citation_author=P. Prins;citation_author=M. Nakao;citation_author=R. Bonnal;citation_author=J. Aerts;citation_title=BioRuby: bioinformatics software for the Ruby programming language.;citation_volume=26;citation_year=2010;citation_pmid=20739307;citation_doi=10.1093/bioinformatics/btq475" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=G. Gottschalk;citation_author=R. K. Thauer;citation_title=The Na(+)-translocating methyltransferase complex from methanogenic archaea.;citation_pages=28-36;citation_volume=1505;citation_year=2001;citation_issue=1;citation_pmid=11248186;citation_doi=10.1016/S0005-2728(00)00274-7" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=S. Gribaldo;citation_author=C. Brochier-Armanet;citation_title=The origin and evolution of Archaea: a state of the art;citation_pages=1007-1022;citation_volume=361;citation_year=2006;citation_issue=1470;citation_pmid=16754611;citation_doi=10.1098/rstb.2006.1841;citation_georef=2006077624" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.;citation_volume=52;citation_year=2003;citation_pmid=14530136;citation_doi=10.1080/10635150390235520" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Symp. Ser.;citation_author=T. A. Hall;citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.;citation_pages=95-98;citation_volume=41;citation_year=1999;citation_pmid=12460286;citation_doi=10.1046/j.1462-2920.2002.00362.x" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=U. Harms;citation_author=R. K. Thauer;citation_title=The corrinoid-containing 23-kDa subunit MtrA of the energy-conserving N5-methyltetrahydromethanopterin:coenzyme M methyltransferase complex from Methanobacterium thermoautotrophicum. EPR spectroscopic evidence for a histidine residue as a cobalt ligand of the cobamide.;citation_pages=149-154;citation_volume=241;citation_year=1996;citation_issue=1;citation_pmid=8898900" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=U. Harms;citation_author=R. K. Thauer;citation_title=Identification of the active site histidine in the corrinoid protein MtrA of the energy-conserving methyltransferase complex from Methanobacterium thermoautotrophicum.;citation_pages=783-788;citation_volume=250;citation_year=1997;citation_issue=3;citation_pmid=9461302;citation_doi=10.1111/j.1432-1033.1997.00783.x" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=U. Harms;citation_author=D. S. Weiss;citation_author=P. Gartner;citation_author=D. Linder;citation_author=R. K. Thauer;citation_title=The energy conserving N5-methyltetrahydromethanopterin:coenzyme M methyltransferase complex from Methanobacterium thermoautotrophicum is composed of eight different subunits.;citation_pages=640-648;citation_volume=228;citation_year=1995;citation_issue=3;citation_pmid=7737157" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=B. Hippler;citation_author=R. K. Thauer;citation_title=The energy conserving methyltetrahydromethanopterin:coenzyme M methyltransferase complex from methanogenic archaea: function of the subunit MtrH.;citation_pages=165-168;citation_volume=449;citation_year=1999;citation_issue=2-3;citation_pmid=10338124;citation_doi=10.1016/S0014-5793(99)00429-9" />
<meta name="citation_reference" content="citation_journal_title=Microbes Environ.;citation_author=T. Iino;citation_author=H. Tamaki;citation_author=S. Tamazawa;citation_author=Y. Ueno;citation_author=M. Ohkuma;citation_title=Candidatus Methanogranum caenicola: a novel methanogen from the anaerobic digested sludge, and proposal of Methanomassiliicoccaceae fam. nov. and Methanomassiliicoccales ord. nov., for a methanogenic lineage of the class Thermoplasmata.;citation_pages=244-250;citation_volume=28;citation_year=2013;citation_pmid=23524372;citation_doi=10.1264/jsme2.ME12189" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Innan;citation_author=F. Kondrashov;citation_title=The evolution of gene duplications: classifying and distinguishing between models.;citation_pages=97-108;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051986;citation_doi=10.1038/nrg2689" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Jaroszewski;citation_author=Z. W. Li;citation_author=X. H. Cai;citation_author=C. Weber;citation_author=A. Godzik;citation_title=FFAS server: novel features and applications.;citation_volume=39;citation_year=2011;citation_pmid=21715387;citation_doi=10.1093/nar/gkr441" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=L. A. Kelley;citation_author=M. J. E. Sternberg;citation_title=Protein structure prediction on the Web: a case study using the Phyre server.;citation_pages=363-371;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19247286;citation_doi=10.1038/nprot.2009.2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=H. Kishino;citation_author=M. Hasegawa;citation_title=Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea.;citation_pages=170-179;citation_volume=29;citation_year=1989;citation_issue=2;citation_pmid=2509717;citation_doi=10.1007/BF02100115" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Kumar;citation_author=S. B. Hedges;citation_title=TimeTree2: species divergence times on the iPhone.;citation_volume=27;citation_year=2011;citation_pmid=21622662;citation_doi=10.1093/bioinformatics/btr315" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=N. Lane;citation_author=W. F. Martin;citation_title=The Origin of Membrane Bioenergetics.;citation_pages=1406-1416;citation_volume=151;citation_year=2012;citation_pmid=23260134;citation_doi=10.1016/j.cell.2012.11.050" />
<meta name="citation_reference" content="citation_journal_title=Tellus B Chem. Phys. Meterol.;citation_author=J. Lelieveld;citation_author=P. J. Crutzen;citation_author=F. J. Dentener;citation_title=Changing concentration, lifetime and climate forcing of atmospheric methane.;citation_pages=128-150;citation_volume=50;citation_year=1998;citation_doi=10.1034/j.1600-0889.1998.t01-1-00002.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Z. Li;citation_author=A. Godzik;citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. Q. Liao;citation_title=Concerted evolution: molecular mechanism and biological implications.;citation_pages=24-30;citation_volume=64;citation_year=1999;citation_issue=1;citation_pmid=9915939;citation_doi=10.1086/302221" />
<meta name="citation_reference" content="citation_author=Y. C. Liu;citation_author=W. B. Whitman;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Maiti;citation_author=G. H. Van Domselaar;citation_author=H. Zhang;citation_author=D. S. Wishart;citation_title=SuperPose: a simple server for sophisticated structural superposition;citation_pages=W590-W594;citation_volume=32;citation_year=2004;citation_issue=suppl_2;citation_pmid=15215457;citation_doi=10.1093/nar/gkh477" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Mano;citation_author=H. Innan;citation_title=The evolutionary rate of duplicated genes under concerted evolution.;citation_volume=180;citation_year=2008;citation_pmid=18757936;citation_doi=10.1534/genetics.108.087676" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. P. Mansai;citation_author=H. Innan;citation_title=The power of the methods for detecting interlocus gene conversion.;citation_volume=184;citation_year=2010;citation_pmid=19948889;citation_doi=10.1534/genetics.109.111161" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. P. Martin;citation_author=P. Lemey;citation_author=M. Lott;citation_author=V. Moulton;citation_author=D. Posada;citation_title=RDP3: a flexible and fast computer program for analyzing recombination.;citation_volume=26;citation_year=2010;citation_pmid=20798170;citation_doi=10.1093/bioinformatics/btq467" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=H. J. Muller;citation_title=The relation of recombination to mutational advance.;citation_pages=2-9;citation_volume=1;citation_year=1964;citation_pmid=14195748;citation_doi=10.1016/0027-5107(64)90047-8" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Nei;citation_author=A. P. Rooney;citation_title=Concerted and birth-and-death evolution of multigene families.;citation_pages=121-152;citation_volume=39;citation_year=2005;citation_pmid=16285855;citation_doi=10.1146/annurev.genet.39.073003.112240" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Okonechnikov;citation_author=O. Golosova;citation_author=M. Fursov;citation_title=Unipro UGENE: a unified bioinformatics toolkit.;citation_volume=28;citation_year=2012;citation_pmid=22368248;citation_doi=10.1093/bioinformatics/bts091" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=K. Paul;citation_author=J. O. Nonoh;citation_author=L. Mikulski;citation_author=A. Brune;citation_title=“Methanoplasmatales,” Thermoplasmatales-related archaea in termite guts and other environments, are the seventh order of methanogens.;citation_volume=78;citation_year=2012;citation_pmid=23001661;citation_doi=10.1128/AEM.02193-12" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational chemistry;citation_journal_abbrev=J Comput Chem;citation_author=E. F. Pettersen;citation_author=T. D. Goddard;citation_author=C. C. Huang;citation_author=G. S. Couch;citation_author=D. M. Greenblatt;citation_title=UCSF Chimera--a visualization system for exploratory research and analysis.;citation_pages=1605-1612;citation_volume=25;citation_year=2004;citation_issue=13;citation_pmid=15264254;citation_doi=10.1002/jcc.20084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. D. Pruitt;citation_author=T. Tatusova;citation_author=D. R. Maglott;citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.;citation_volume=35;citation_year=2007;citation_pmid=17130148;citation_doi=10.1093/nar/gkl842" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Rice;citation_author=I. Longden;citation_author=A. Bleasby;citation_title=EMBOSS: the European Molecular Biology Open Software Suite.;citation_pages=276-277;citation_volume=16;citation_year=2000;citation_issue=6;citation_pmid=10827456;citation_doi=10.1016/S0168-9525(00)02024-2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. A. Richards;citation_author=D. M. Soanes;citation_author=M. D. M. Jones;citation_author=O. Vasieva;citation_author=G. Leonard;citation_title=Horizontal gene transfer facilitated the evolution of plant parasitic mechanisms in the oomycetes.;citation_volume=108;citation_year=2011;citation_pmid=21878562;citation_doi=10.1073/pnas.1105100108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=F. Ronquist;citation_author=J. P. Huelsenbeck;citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models;citation_pages=1572-1574;citation_volume=19;citation_year=2003;citation_issue=12;citation_pmid=12912839;citation_doi=10.1093/bioinformatics/btg180" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Sarmiento;citation_author=J. Mrazek;citation_author=W. B. Whitman;citation_title=Genome-scale analysis of gene function in the hydrogenotrophic methanogenic archaeon Methanococcus maripaludis.;citation_volume=110;citation_year=2013;citation_pmid=23487778;citation_doi=10.1073/pnas.1220225110" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Sawyer;citation_title=Statistical tests for detecting gene conversion;citation_pages=526-538;citation_volume=6;citation_year=1989;citation_issue=5;citation_pmid=2677599" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=H. Shimodaira;citation_title=An approximately unbiased test of phylogenetic tree selection.;citation_volume=51;citation_year=2002;citation_pmid=12079646;citation_doi=10.1080/10635150290069913" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=H. Shimodaira;citation_author=M. Hasegawa;citation_title=Multiple Comparisons of Log-Likelihoods with Applications to Phylogenetic Inference;citation_pages=1114-1116;citation_volume=16;citation_year=1999;citation_issue=8;citation_doi=10.1093/oxfordjournals.molbev.a026201" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=H. Shimodaira;citation_author=M. Hasegawa;citation_title=CONSEL: for assessing the confidence of phylogenetic tree  selection;citation_pages=1246-1247;citation_volume=17;citation_year=2001;citation_issue=12;citation_pmid=11751242;citation_doi=10.1093/bioinformatics/17.12.1246" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Soding;citation_author=A. Biegert;citation_author=A. N. Lupas;citation_title=The HHpred interactive server for protein homology detection and structure prediction;citation_pages=W244-W248;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980461;citation_doi=10.1093/nar/gki408" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML-VI-HPC: Maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.;citation_volume=22;citation_year=2006;citation_pmid=16928733;citation_doi=10.1093/bioinformatics/btl446" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=B. C. Stover;citation_author=K. F. Muller;citation_title=TreeGraph 2: combining and visualizing evidence from different phylogenetic analyses.;citation_pages=7-7;citation_volume=11;citation_year=2010;citation_pmid=20051126;citation_doi=10.1186/1471-2105-11-7" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Suyama;citation_author=D. Torrents;citation_author=P. Bork;citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.;citation_pages=W609-W612;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845082;citation_doi=10.1093/nar/gkl315" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=R. K. Thauer;citation_title=Biochemistry of methanogenesis: a tribute to Marjory Stephenson:1998 Marjory Stephenson Prize Lecture;citation_pages=2377-2406;citation_volume=144;citation_year=1998;citation_issue=9;citation_pmid=9782487;citation_doi=10.1099/00221287-144-9-2377" />
<meta name="citation_reference" content="citation_journal_title=Protein Science;citation_journal_abbrev=Protein Sci.;citation_author=B. Wallner;citation_author=A. Elofsson;citation_title=Can correct protein models be identified?;citation_pages=1073-1086;citation_volume=12;citation_year=2003;citation_issue=5;citation_pmid=12717029;citation_doi=10.1110/ps.0236803" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. S. Weiss;citation_author=R. K. Thauer;citation_title=Methanogenesis and the unity of biochemistry.;citation_pages=819-822;citation_volume=72;citation_year=1993;citation_issue=6;citation_pmid=8458077;citation_doi=10.1016/0092-8674(93)90570-G" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. V. Welander;citation_author=W. W. Metcalf;citation_title=Loss of the mtr operon in Methanosarcina blocks growth on methanol, but not methanogenesis, and reveals an unknown methanogenic pathway.;citation_volume=102;citation_year=2005;citation_pmid=16024727;citation_doi=10.1073/pnas.0502623102" />
<meta name="citation_reference" content="citation_journal_title=Biophysical Journal;citation_journal_abbrev=Biophys. J;citation_author=D. Xu;citation_author=Y. Zhang;citation_title=Improving the physical realism and structural accuracy of protein models by a two-step atomic-level energy minimization.;citation_pages=2525-2534;citation_volume=101;citation_year=2011;citation_issue=10;citation_pmid=22098752;citation_doi=10.1016/j.bpj.2011.10.024" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="Methanogenesis and its key small-molecule methyltransferase Mtr complex are poorly understood despite their pivotal role in Earth’s global carbon cycle. Mtr complex is encoded by a conserved mtrEDCBAFGH operon in most methanogens. Here we report that two discrete lineages, Methanococcales and Methanomicrobiales , have a noncanonical mtr operon carrying two copies of mtrA resulting from an ancient duplication. Compared to mtrA-1 , mtrA-2 acquires a distinct transmembrane domain through domain shuffling and gene fusion. However, the nontransmembrane domains (MtrA domain) of mtrA-1 and mtrA-2 are homogenized by gene conversion events lasting throughout the long history of these extant methanogens (over 2410 million years). Furthermore, we identified a possible recruitment of ancient nonmethanogenic methyltransferase genes to establish the methanogenesis pathway. These results not only provide novel evolutionary insight into the methanogenesis pathway and methyltransferase superfamily but also suggest an unanticipated long-lasting effect of gene conversion on gene evolution in a convergent pattern." />
<meta name="og-title" property="og:title" content="Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2475" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Methanogenesis and its key small-molecule methyltransferase Mtr complex are poorly understood despite their pivotal role in Earth’s global carbon cycle. Mtr complex is encoded by a conserved mtrEDCBAFGH operon in most methanogens. Here we report that two discrete lineages, Methanococcales and Methanomicrobiales , have a noncanonical mtr operon carrying two copies of mtrA resulting from an ancient duplication. Compared to mtrA-1 , mtrA-2 acquires a distinct transmembrane domain through domain shuffling and gene fusion. However, the nontransmembrane domains (MtrA domain) of mtrA-1 and mtrA-2 are homogenized by gene conversion events lasting throughout the long history of these extant methanogens (over 2410 million years). Furthermore, we identified a possible recruitment of ancient nonmethanogenic methyltransferase genes to establish the methanogenesis pathway. These results not only provide novel evolutionary insight into the methanogenesis pathway and methyltransferase superfamily but also suggest an unanticipated long-lasting effect of gene conversion on gene evolution in a convergent pattern." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9uCHWB0vRYOCljvka6P_AccYBpCKEHD8mn_TmP2FTwU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460163","apath":"\/ggg\/5\/11\/2475.atom","pisa":"ggg;5\/11\/2475","ac":{"\/ggg\/5\/11\/2475.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2475.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2475"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2475"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2475.figures-only":true,"\/content\/5\/11\/2475":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EFvJCd55AOnB24bfChMHNAoaNN8zxzm2BGmtDW6MT9JEfccT3qsBxfK\\\\\\\/KuOu80mZxYmkSm6IOKL2wpbv9LUuFyljgyRxbqQBlcKDi+nYzGRGChs8VNvkS5R\\\\\\\/lKAdyo3wyk5M+RplLRPM8urwLqaJiGonfSGtrAI2G020U\\\\\\\/kAgEDflDd3RXS8Y+wGPUO9wj3W0nEEYgX43w\\\\\\\/3fh68QfSwH5hCHxMkiLNjmXAl1Onn4G\\\\\\\/VTKdYVf6OPfSB3SnrcWFVEpquPh24\\\\\\\/Fqj28QzyFVA60a0mKD7z3b2LKj5XdJQ6fKhPlLxu3x6Lxv4PQcTPXbcp4WsjOLIDXGdAshwr8Gkd1xUEurJcn86SXXoP6RFLIj5CLuz4DEvoncTT3huRFtQL\\\\\\\/4btmTD10A7EudYcvB774PeonFq2fRWLQF\\\\\\\/YsdbZE6jxaiMk4N7qxJhdPiEcoJQSMqlRKgnfRLnQ0I0RLY9KIx4rUmkCkVyd4QoN4M8KtC+OyEeGn4LWBFxyTnok6MvNDb09aVFR\\\\\\\/wAKPkDgW8eaX7t\\\\\\\/US2zXvMsaG6vVK4N2fAhdWhSk4lRQxNeY6nFAkfip8nYHNv+y6in9bZEnURBZQKeOrsuNYKYqfbDzcfD1PDOTqVuvjgkt++Sm1mhfif\\\\\\\/tQPl9qdXOQ9+nJOW4Re3yHSxmF5x67X+M59QJC1pzS4JGYABDbcZBG2FG2q\\\\\\\/5qBM3N1u5SCBY+llIrINlHB6IyATRFCaOmFwscDOPoMFHTHPd9lE1U6DwcyfeefAZuKcV69CChCesBT5Cn\\\\\\\/c\\\\\\\/UxaAaQY3rDHhqxOVMVP01oZRRLkfqKvelmmWisvSS5hkvke\\\\\\\/GksTMiSy053vFhk1pfsuQt++cqLWaqc0MWk54M+7u8\\\\\\\/g3zsFEyoJtrP5sWb1PhBdn8VwB2cUD8tJBdypjw4d5Wa8xGHeugoiweN8yt5pPzkBMNAvx1kTZzZpeNuXKivLH6\\\\\\\/3TRTu7C7oymAqshEZUFEVRGl\\\\\\\/8KYKoAxD\\\\\\\/lKRWisg9nA+RdxQCyHwAj7nwxdRJkQ1SddsbCn1KUADvsw4c7exZKWbjNVABb\\\\\\\/jpGHooGtJCY5kK8\\\\\\\/rFEDlmofdGduPPNfY5M\\\\\\\/OF4HiHN9BpjXkxuXQhItMvgif\\\\\\\/Vyzz1xUUtaStERwfxKZSzKzHFSUQi+rUyeKSf6HR+5lL1JnlCDUDB51dhxslddAfj4Bt80P2\\\\\\\/o0z8O5Dg5+1nS\\\\\\\/2BBhLJJbvr9gqz+9zsnV6pmDOIiW30qdsdLjgxAUXD6Y2lyzpHGM5URL8a7MRN2TATLI7X\\\u0022,\\\u0022iv\\\u0022:\\\u0022hdp7t\\\\\\\/AwCvCYz+mJuy5Qsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022de4246697d66fe17988023bf70a988ed8c62b399b59dc292bad1a6ea6dd42d96\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BaGJfOqbc\\\\\\\/7NjrXn1OSZhYaUpURCogSKf5ebqSgyFJ1zDDacgeCYk5yjl4JVZ8nTsqoZ4G8hsfmIJptqCDtiStJe1CK6ZVD6w3\\\\\\\/3D3ya072I2JVBIY+HnQ5BEsHj9HKjKFAY0EhatXS5nrU649NZ0j7PLBpWsCD85\\\\\\\/bxtOPa587fLQNlFXlsi5UON7TVzxiqY13v6dzXfize6qVC2t1NFKBtNhWLc9X7UWTyHw5SRprQKmSjxr37dO7+aF1wIkl8GEyiBDedAwoLp4cqHxQ46S8Gs86aWJ7KAqTalAqhC\\\\\\\/M5f+WRftGhvADqDbS\\\\\\\/CkX0M\\\\\\\/6uTpv3YKQ9YJe50vEzT9s74GDcImmQls2SwDdo7lrajjBK+UWBay3MPg01uH\\\\\\\/Mj+lUxNITnYe8W0BWPJCuUg89mjRosCQqNLt\\\\\\\/DyAFpkHaJ\\\\\\\/M0UQAJeKMhWWcEzjs9Z6x7KwpasHxcvXM3jHIf9f9tkwJtTZVTByPVoVLTpWlZr9UAcqLl9KTOGJnpw5YYCJNoJjZQhVGVJ7WcuZAwHJdaAOy+4CIDAv6OC4JutodFSnyk4Zu3caCgAd67ko32Z6L4mbgjBE5jyahhGAnVlvG8vIvGjy18ZAAidb2rC+rfMfOTr47c+l0cnHFvQfTPoGeT+Iyg4wjjaBedj9o5RRDI3hsBkjnGrYfDAf\\\\\\\/5Wddn+2eqr+OSmYUQRfbArEvEQVUE9gA6bpPrRUQKNEWR0WbqLyH5Eq9bNmYuJDTs+prmBV9S39CZspBcKXAYpXJdvMtqjo6HS+mzhMw7dIFsEPRawH3AmdnMHBlolUcQ7YtWuVav\\\\\\\/lcfcqxmQY3oX9AKHAkizTRlJ+Tle\\\\\\\/yeb2RfoIlNLH52I15cXfJ6qhRCjz6ZtLNKNZmzgJZxawQFdNh4eQsGfR8F5FgUe7MTwZnPKSnesH3CoYMzbMJQQ7hqERe5vMowvhEatjVv2wbj4UtJ6EjUxMM\\\\\\\/\\\\\\\/sBGQUJ2\\\\\\\/ldusbxdfOjOPLkFFQ3e7kjT\\\\\\\/epC5+s7fN+Djy+eXq9nsScm\\\\\\\/BU7RW4CcM0bccLuDyakeJ\\\\\\\/zqKd8XREz8B\\\\\\\/7bXkiV7A5ZbPEaGgM\\\\\\\/xERUrbEbqbVbza3P3gcnFlLbT6Yh9HDJPOSpEemxsmVWb5\\\\\\\/qjjD6J9P1fojGqWEei\\\\\\\/PNxrjS\\\\\\\/zQNd\\\\\\\/CVv3NWe+X4TAefetITlk6Q9dxoDQPNjKTsqEPIXbydZwnDGlz+QDOtno\\\\\\\/60Cn6DZQzkspElhBLutG7T05M9OZbTUlaP9LFL8tGfuIZ07APf0DXhBF\\\\\\\/5Pt7P2\\\\\\\/y1Qywn6bNeJ3AT80jMqk9eak2nYVhGN+UKCCra9MfziI+wam\\\\\\\/2vPDoUiUGjZeSoV3M4uwTYpZLDGO2zvJp5s7dCsBR8WbDKg+OhmgujDhWgvTZdYPrr5IAzv5cdnLgX8\\\\\\\/DHBEmaW9S1uObGhR\\\\\\\/aTzdVgWhw7DNUOpXUI4njk\\\\\\\/kpg\\\\\\\/nSzNpx+Y1aq5iXS4rVEOq87g8rqUF0nr0MoLf83Qe43HROkHsBBGkrwe7QmEdstYYYp6p3YzqlcnpksCUv1aLAjaC0F\\\\\\\/KvE4WJ2izNjaXGQGNMRg6b3lD1aZQOhzcxnH6kHAMGCf4UBr9EcoAfB\\\\\\\/2y0UzYcQJVPlaMVdSE\\\\\\\/uJKM72OPcPZGMZZlMvY9wHc8FeY61y\\\\\\\/i2TyW3Aut6aKp+Y0Z8lEf4cMVB5x4HMx+cTJ4Yc61999rjv471yutOmiMVvRBOSXBht37GVdiqw5ct5Ee2Gc8sivZzS+u1yj6n8C8A3XLFb2UGyxgJmxprD2qDkI6y442hqrnwCbOHzukwbQ90m8PizJM\\\\\\\/6HG4NWpeDcWC9AdHzoryQnkiL034CIxByf86n3Ia7Vi8xNOFwxsBbyLxPcH5QF78n7aI7sbznDwt2jdxz64a53qhbrsAHwR91YWFsY\\\\\\\/\\\\\\\/ghlE+5G4t0vBqQlvLpPPg9Y1yX3DRGAy5KjDQSTGeHYziNj8nzHiP4rLKlz6bNHtZ0On83SWBVcAVXbB7WyisGqxVZAE9pEYwvBUKvEeqZ+D0SOIosYGTLtqRiF+jMxFINQopLd7\\\\\\\/siNJr2gbgzGVhmJW+nruv8RxDWju\\\\\\\/uCDXnwFX5M0kaOM3q4sMviYQz\\\\\\\/l3PwqBO9161hmN6HP8GCiUJ1u8muamXB3UI5ODEQ\\\\\\\/W2cByXj79SZ2jUTY2a752xgwN5\\\\\\\/Q\\\\\\\/jtv8GZyRGG0rqdxSSeJD973UGKigXv3w\\\\\\\/Gu+Rkn6cbXAvb5rUsnTSTds8eL0TNCoTPlekz1qV8AroYs5oIOstnZqYvi+KF4IuOCBsSA3RfE2YnNVTBjksE6btKXOP8PdnFt5TzjoG3o9Ry96xvpEgxyw5HgDiy0k0pykxhiCI\\\\\\\/479oAR6ZzrmiN+sF7tHyT2Z1sLQRYR8lJoECgdX9GYOofKPVvX06rlrQKZQzHEOJ1tF37Irnq1KMgfVqXHFVM7Qb17e8CxN6fyZYqJfb162aMgWPTypads4RUNztn6VOFrG5facC2v3bZMGmWnaGC1yRBBxV4rd0YCwo0SvdF4mcL9ngTzGcn\\\\\\\/EANAdnhsqGEIgTbdkZ6z8UK2bjRKbPqinoTqz7O\\\\\\\/yGjGoXYPeKRY6W\\\\\\\/LNE5yJpRsI8A03JjtwAL4IkwC3++llhYXS\\\\\\\/aI1C0HEhUvYL1+d7nUQIuKyUIfNSM4PiSGx7cLoN5G36LEh83lDmiyC06gR9DhxOS9sb6ULhMHTr0cg319ifazDe+qelGhxehYAEsxVg4ZNfh9oJI8BzDr2J+A\\\\\\\/Kg\\\\\\\/uohySqMrg1oe6u2GXcAsCZq49sd9Jj6gqWIK36dLJRR7iyyXr\\\\\\\/G13pxZlGKv0\\\\\\\/3mQQPSi6tjiif8UiAibUancPBybvVs3GlqMf0h+ES6ge8pn7G19PVk1qrpDqq+sHQ2y1xusBVVLG959qEt641PB+trl4SVRZXMAILdwcXw7kCeFAuLEqyckcpCHfa5A7GSoRtmHdB+9bRQrDTx3o\\\\\\\/ESXPH6fFxbzQzJq3mZVAV7ZA9NRAeQ2lJbo62\\\\\\\/ShB6xmi3d\\\\\\\/TksP8oj6cbtqcy73LDXaIeZX5BnSd8xMW3Mzsp5l\\\\\\\/6B7KhgFm+oFKi3qXUV7phJECBhJL\\\\\\\/mTl9axXxpferxUzvHNOueofXgMTEhZwVk8VcqZ3xNLAGtBj2d8F6ZieoYdgqMP5svlIClQlK57SRsWtddne72Ahl\\\\\\\/y8HKNiln6dmn1CY8Z8JcIRhNCHYGDWXw1FVXMzJbiZ24bldH0sc\\\\\\\/yF6OI4TK6TzCLaXm\\\\\\\/ZVe2q3DxdYkcWobJTkxoq8cAqr33tlLA1WbC6muiMMd5qo34c\\\\\\\/4Qx\\\\\\\/BcXF65cb4W+ma1PFIpTbHFl3ZWspISyzF8RrCU00+bJg7UR3urL18eYKZx1uinV8xsbaMDLUnOywYwRkPJhYJanFmg+p0V+\\\\\\\/HR3mzptXXvpvbfrztqk+jjaeFoFCaXBX1+GU\\\\\\\/Fguut78PW+k60Tl2\\\\\\\/PSdfaWXGdS5JGgIYZShFARfjqQkSGBOPAXgTGfY4xWjw4s6DT5k5c184I3Xtl\\\\\\\/IlYlsBiZAz0aOM9yYjsplw+M7HD9zk3hrvUc+5Oq61tx+NUur\\\\\\\/AIR25DkkFpJCmHOpTnlQTHwpwk9YseMALo\\\\\\\/uEboNyp83Zr47g1bMzP2kZMXMMFQH6QRPS1I5cJ3FVbhndAVA2yw2ZtdGGW43\\\\\\\/3UV6HSCmxnsAqriofl5aF6qyxqdC8A+Ow8RBIn\\\\\\\/1x+mEiEe+FKNb53KuMe537ajgd7\\\\\\\/XILA2jH3qguKAi9d3LGU3dWZhd6FY\\\\\\\/4hr\\\\\\\/iYYhhhNkBoZVqUfIs7mmYIlsIP49zoHbmWdaAbsKRnZNzWe\\\\\\\/YZwlN6znb3xIFkFDPZGRl9DaqWz3uWv4QGDggIZ1pscgLouOIbiZsIuCft5oKOPkUTQ1O8NG0UYauFx8va8faZ8YVFX+gPhe7W7QMU4est+jawWIxWaKv\\\\\\\/ZDNqF6WHB5aGZzH98zjf5gl9sFKif23NUM5s4q3WyEtbc4gxsOJ8dZlcQO8nk8C6p9btxpX1GDcl3csGUIBBbV3P2nH40QpWr0fzajKCPm8z4s+YepbA3slRIISpokYyUeihM8miCVOVDYDPUvgJNiFbLeXNT\\\\\\\/Com3p4Yf98KR5Q7ekYEQ6BBPCo9+j3znQ9wErThVhL+3lJe0+rCVbvGyT+HP5DKVCGsLc3drxr\\\\\\\/MVwszpOF4Yt613L2jKaX+N\\\\\\\/96IK0iMVjkKq6\\\\\\\/N6Lu0a2x6gY9lYBjiFKJ56YGnezfchpKOwAWl42tKtdsN0UDpf84PNdnDq1oDc2sUWSMhwGnynIqBDh3y7\\\\\\\/zcXt2sZGvcrUwENKfX\\\\\\\/UljIt3rslZhcqgQZekpJZo7rWgV8OBetx\\\\\\\/Huvwbpoe8ar9BI9yicTkOSXnuvqg2NhxHTeFwaHkWIYqn\\\\\\\/eNA8lW\\\\\\\/tWY0Jz3ZAyPKLpexMt4YjKatwl6YCLnCe+FRaN8luGqE2HNkAVE8VUIruLkarS5854SFr8AYm3YpxDLT189A3QqqdR9umL2j8Rnp9gP0upQsHziGkDvl+d4egbTwjmxGkW9HdktgddhhbrEPZ+7GjEuDp4GgidFpsvby+IY0limIZPCz\\\\\\\/jS1hxQZTOLiSSCCKNmmlSvyf116bIhq9k\\\\\\\/\\\\\\\/Yg3m9Ox71+aV1O5PRyRxEfWZKtyivQAxCt8+MLLU30jh56nV04yK9qzCEAyQ2NgCVLmSjgnvLES0WBL0L9Ptsj6D0Y3dbHjQUu0Z8Cge4EsLP5ETDW4V\\\\\\\/xKCIJ3jMIITbavykkqzjq2PLtSjz7tioO62DCbbOh9HgysSJdQwSg0IvOfz\\\\\\\/ko\\\\\\\/l729zdlaZn4XVwnDQ7dcVExVGJD3ZF67KzKL7W0arNCk37q1ABIfRclyUEq3iUxfSYi9py\\\\\\\/wvpI0M8Nsq79vOXGIiS\\\\\\\/3Z89494KhrgU3ILtZ9Z0rF0Tyg9Z1gKYwWlAtzm4aL+gngNAg2ZaT2td\\\\\\\/Zo8uHMv+4H50Bqeu0yTdte+gHck4cVczwlRM9Fl+UWjXjTY9AS00zftOStqb+2UJq\\\\\\\/FwIw2f4fV\\\\\\\/QJcOnT8pma2GpBLBvj9fwETjd4GKW7I8ZwyCOu8eNmyA7f1SYq+lGAS5NPP7eV3LRqCmcE9jjExZF8ebRE4LGr\\\\\\\/B+ufY6eX3RVWf7gA13MWbtdY71Agm8wyT2\\\\\\\/Ru5V+CTzFBqvUg5+LG0Tv84hFfqUqq0fBhpkfCYybdg\\\\\\\/fryq5SnJFLsHti4710gZo8VFFhwPFv9v+9OOsX+4bjXE2daKCmsuKvTAV5At9lY36cbkVx0xNVNb\\\\\\\/E3PElpe0Xa50\\\\\\\/z1L\\\\\\\/NDeDlk+NzP8qwvxhIvnd8BVQf6JVMabp6d+yA563dyrQ\\\\\\\/7lpQ1S6EL36z7QvcCom2gb77mTCGjvZSjc1rjDayJJcSVl5cJ4M\\\\\\\/q56fruLTo+6Ds8GJ4ASv4+Bi7+Oy1fZ0gJK895wY4W1+EKHAKz5dZ7UnDTCuV\\\\\\\/8Z0l6i0rQ2UAj9VtbjR31LomfmeIxniMcTg3CeK9jPdlRjChe7OLHANCxNLtT+CjKYfjFVWAZlhCyJ1tTB7+3sfZ1oV81MnVwWbckZUrAmbne3qJj9b5abLRhFg4GmYNSDCBvDG3BFMyXc4rIK2gl6TWqv43AU90vMM31VCVAoGqFf89MWmGm0X0uWBlkdtzR6XYcAXwkl+X3bD08F3aY2FhAiP\\\\\\\/+LXkPliqVcy5UPTZ3huRwuMZ\\\\\\\/0lzvoi4rRMfeqqzuVjzPhpD8aX0A03H13SQkyaCjnIa\\\\\\\/bkfuSxv+i3dM9wvC5zwSezy1qc4dOcO7d0\\\\\\\/xa7OHQZdTzPrKNqW+ZTB0Ug+V4gAadFhO4mR0Czhgzs3rfrH6s3Phf2FiHNFTUK5QJq9Xy0mj67XJNpW0WJ7uj70KRb0I6TPUPdmwRfIqyHZTvfpW9EHk+xPoKh3MW7zyfD3\\\\\\\/VhbaCglrjQHusL4362cxEMbiawut3JBHKS1OHCW+rovHCoSDrV+pgr9D16U0mDcyWpp\\\\\\\/lkNuAA6uK7Qe7j550mICO9jx1UM5G2SoXuf3qcSYhTX3OSSMgDZdMYRmT7u6pk6OrsCnLZwsfIyHWk1aUBP5Fsz2bJX5dNncdz5HCWMqtbphQCY\\\\\\\/bjIlyC6mMIb4kuv+6aHagB7pWmcwabtKnSw7WPHZHypey1ED+eIzVjA9CtM\\\\\\\/MlG4\\\\\\\/KwyIAzZhvaLwxqT9765lL4OVsHkbNEs1zf3NOU7l2deP9KujE\\\\\\\/IaKmDq+\\\\\\\/gn5y9T5Y0pT4Yl8\\\\\\\/jWY\\\\\\\/qshK+Bb+9v9mZmgt7cQF7iG2jYVRx63AwF58zrYFIR0drFOQQQKMo+SKYbKF5Us2SvflhpnvZnr+sEYLa2CEpI0AEDNK\\\\\\\/RL0YssQ\\\\\\\/lLratYmmkOAFcP0s9i9qvN2eStLT0l\\\\\\\/3YzyAuqBOdQOo3DUvip2aIOyskzOUDBgUMCyTUuddaLq8gbEF\\\\\\\/kCXH6C2rAfEe9gAjRb8kEMWkFAzcuNLh\\\\\\\/I8I0zTnrPkfCHsWC+0+CCIhhGAAK7h7xj49eUNOxswliy760atd4MDLQeJ0BoRjwXBqBDJ\\\\\\\/U65k2u8LFxPr6JD1zJXpgbycMdX\\\\\\\/\\\\\\\/L3OaNEdcw5c1PsD7lwsJHVN3e8KW\\\\\\\/3fkjJpq3nvNiNY6cFnpnlyXgk5hyvmKD9J3VycE7Kof35x2Mn0HBRwKWDU8myv9h20sIGJDvF0Me2MRM40i1Xhe7pkJ9A+nvG4TVSif1CYN\\\\\\\/Qblgd1nadc6sCJ45phGHC81vdjqXTRWD7D7FSuq9kaLxcSfylL8xdInP9z1k2Rew7IvoTlGOp1tEI1mS9EAc5Yerm0XwuHILagFGk3AYLfNlC0At+eLIeX0ar2Vgo9mEId5ToIcObFdKM5q8wMCKmFrzCQFRUw\\\\\\\/Yr2hCPSuFux2Vy08ZzmggfP42XAyY6AZladnOeCJDbUkHYL0a+Ha+iwA7nPNGZ7eEWn3DZGoZ8wuyFUVZHO+3ZBqYmUXFtKlg2GpRbsEFNuRRPcshfnqocB2kkKHsEY\\\\\\\/QTDvrqLn63jYTmYahrPRemiCEWI7sWwfSVRxoO0vPx05Fm\\\\\\\/l82gLGk\\\\\\\/KCnaaQer5FtxdUzpuGxnFb2w\\\\\\\/\\\\\\\/bVlgXp6vXg21dRaQI316H\\\\\\\/PfmmsEhsjKEI0fSp4OmiJgmZA+ZytX2LPRJeBlEAGNK\\\\\\\/MogmDnKmfkZCGX6hM0kOZeqf7cn7aUfShqia9f7+wq2gJmMWcWjhgrE6VXuVilv3cCB2YDoiM55meO9tgrU0YBkj2QeaPAcp9VuYx8L5YdIsBPMviCHmO3wvICDTwJ7\\\\\\\/FvJuSIXY0sOmCauajJO7HeEThiZH4wCLrTSVcCPoVTGGrx\\\\\\\/gyX21Tf+ZKJAWWwiRn1UCmzJOJA9S91N25HJarqChCzXXG1JzNLtxBJlDAvKvkGfh\\\\\\\/45AM3bAZ2iml28kIetb1f6O6QcHhHEezBZp1L+fKfeJBKRy+l0lnvCybd16yLvdghpPnbV507hIu+DbVy8R3Y4xBR99sXWDBkEQIxeKVc+s8Vr8y4t\\\\\\\/d5k4rf3XOchHfyc5EuQt3k\\\\\\\/XCwuEkCDxwPd1PKsfzGoclfRzvzg25W+V2brHsZlesSkINXEnbFBTsS2YmMLCRIKqaHFygMx745GJOG6coS2d7nY5BGpBKmvaAeAhfd+FqkOPxI92q5v\\\\\\\/WB9DJDraZkieQ6zABFtp3XJsc+EQbI7xACs21nSwmrkXxKOee+MExKpcUXvPUNfR+sRp9TRzfcHPTvwskQpCmIyTRTp71D\\\\\\\/9k5UQMdGlA6AS2J7pbdKluZzreHOHhBwKNRonlrm0o1JPtgXunBVqEBic+dY+CmuVUa\\\\\\\/l296QrlYj\\\\\\\/Njl1u5d0QTZZz7zXM9LYv8MyL7jVzOZ5czjnAsFfi87GFP1cVZ8mGa0n1a73jgTkaOhzQVbiP6A+CJQ+6ae414EeRVjkrtKtpl9fBNpzh3GNk8+ozTSEfexFu8x+6Avuou2LPas3y6p1hLwwpWxtnnbLt0hL7gRP+ByIif5RJqv103fO7kZDxsd2TDdbMLcAuHUlXWkn1q2l1fZA8GQk0KQRZqPxNxh9puF7tzFftg8oaSxZlKw7EHfr5Q\\\\\\\/zFn3GOvjkqH\\\\\\\/U28E6Hg6NZL89O3TGpubaoi\\\\\\\/NFoaRXR1V6AOI\\\\\\\/8Vvr9Ez6hC4p523LP\\\\\\\/gbA8o24WeIyfFBjSV9TxcnSgtdU2cxO0PGJakXyQmPpwr+3+ldu8ccB8yNeVT3b8ZVzNVi7JG0m34IsBD3nepzYCQ8O1naWOoM424cUEPtnhH3Bc21H\\\\\\\/RLsBX5sYYkieD5CzvhMuvhOgb0t39rESJFAEMyotWCQVwsrVMG1xf1x33\\\\\\\/T\\\\\\\/I3Zo5eKmzeJGP7IW9em4db8enjiiayy9vjbQqdD977k6BbbwO49aW4OR+ue00F7hQXryCsUlsil8osGcG3BVCYBL7MvN7Oi1MVXoyAVly23EvtRpZZSwmYVwGKkFxLi6W+oEm64zRPpKijA6bwRMmTKyWLFMqjhJm0\\\\\\\/EEOqyAQ9TZ\\\\\\\/9fulekBnlqkF9cy5RXGC9fZQYU1MBTnYsshGcDfLULcHZ+UJY9uexJJULSRcaQGTsKdaLwGGddCCAPEDlEUGdgfTkZ1PbVuw2rwMD\\\\\\\/X8mb5EAiH4vbM45wtVJsWuj4Z\\\\\\\/z4myydQOzSBq+JHs1LRfkatBa7xgtNh374mXGlOOX8nQRKMLE5FYPiDoB7baCbFIP+8RkceK7r0SLVzxd0N2GZNENDScd7ZBAF+QDoM0yNzZWA99Ob+9nFiL3GNf3E+hW\\\\\\\/r6ZjgSa1hw2KiDxVceHpctYcsLlWF2XlrSsLGQin7QQ2r+LzSsZsK8+ye1hrz8KRv1uhQ\\\\\\\/cUTxH2drW4\\\\\\\/2ojD2+tu84Plc6BftAFB1+svdPKm0Y0MzmzJBVdlr0HyYNtEQsB46c+7FoBl8p3qPqgANRHWRXzJU8s5G1L6Ivm1ROQmi63+cqbjuyPF6euDJDfigWznrPdTwNOfv6fd2WEhyibDAxRA6\\\\\\\/U70buL\\\\\\\/pj+MD3ia0\\\\\\\/7IUIm5O8oFRfohD+xLsevOZYdxgDdjPWX+\\\\\\\/iKHcWVPpGM+tfXrL4AI4X37xvvqy9f77vrW9ViGlfXj2Su0dugZ1AJ0vYpklYlteLtrnJDqYCkVm0dSfSN8+ta1HLjuLw2Sk5EPuJFwebVFqZQWL+ux4CuiCiyC7vd12lRnRsFltuVpFMUfVCb3yUfZ4DHu6PGBvBmR1rHL2sk4DmWwzKa\\\\\\\/H8nPvvNCCqkxTY+JmOK28EHRhHDzQag6VlpcrqGbmwSNHcfSQcUxDa5p2\\\\\\\/lmVB7JccKEZPVNzCK0cR8vh2wWTNOsviv5IhIgvd4I1KGEagDVQ5VTOuhbNcpFidpmIG2G+aOlWfd\\\\\\\/WlVJrgIztO11xBEIFYLILY1w4TLzzoHL1QoPI4jtwOcstdjyKVsTiRhn8Wy9dyqJwGm5aDYeYkRSrncUvgIwAUloo\\\\\\\/XHaV4Dl+\\\\\\\/nx9odIdzxQehZbZfir463bkddv6KO7y2yqfNANcd0\\\\\\\/REp0znr1dnLNHhn3WSHEPF5XJ6dj97MoOUwETTqkGVLPnLz0jLfplYp1NCraXFDXHC3kheAYkm8XWC\\\\\\\/Dg84GXm7Cn+5oXOa20dXf3nXf6D8HaFNKfiWE0j3\\\\\\\/CyyOY6cbZw5arIWCtl5c6bLqSOvMppFAYBxX09qDc5Hip9VbxrqWZEtkMPEmljyzlnja6PdsgquUFwKmcbNJNmte0Ekv9zurG1OdI9j6EBp1aLF5jz60v1WfIljnK+qM6pwubSEqywgSN6PNOI6guksGsUpMva5BV6BvbYzFQ\\\\\\\/CEM4+2tAfAkv9tIeFAksyTBKkoWOLCSaaLWwMMFpmI4hnuGonn8b8l98GsnJDB7cIte5161rdHlYzVZ6vBsLlDSKsYaBt+A+uMsQT3AxHeNX6V0raeci3WNwT9PFMOyMjlmL51MFYmTeLWaiLU2uDC9V4wNNhP8E5TWkDs6jNe+2Nv60VeDZFqjlftoCwC4+Wjkg62bkIipFM82usMEGOq\\\\\\\/7mRG3\\\\\\\/XbsFlEAhi5zbzlWXjL23tBO1QU7Qs3kcocLYf+TncqtxSAFpW70l2V8LS1EbK0Bsg0GqmCUXGEXHh60Lrn51YmZAjBMFX995HaDtKRkf+Dsei4aCYs5W10eZkT9lnKAADH4Ve6rk0njrhdc8QOKYRY72fFJH94S8w15UwxmMXbMCS5ZeT8vBPrlnUDzHAVDb8RfMsshI5aX7I9F7DYYFo5fFFB9qY\\\\\\\/\\\\\\\/Jo7BeVJHI9gkp233uzfOAz+E+Fr+5aYuh2cnxVSKx0MRvOb8exgUXFrz7NNhmSphgVME7uI2\\\\\\\/5wUDkRUKxqQYw2F5DpyMSZOZOcLFUKXmn\\\\\\\/8D1PodjHZ7xpHO\\\\\\\/DcbsY2Po2XGMMRvcehAGIGsRSJau9Q1jNbUlP4kiFW3a+\\\\\\\/bkpksS08w5+30a5z\\\\\\\/WWtH52DzdOzhP2H65ePGNXLo81+3fOU3wGEK5hvTwpxDQyKD3uf\\\\\\\/tUShBDiUn6nqMCpRNG0Gz7+dM+mOZqAX7zwwk3MSyQGAI6tV051cBcprqKua8jDZvZ7qMrmnt0oQRfglYtzmOSC0oEPSiHFjmuiKhUcIspvOEy8E++qjGOaW4RwabVdWSf6iSAqkftFaGXW0GjMpK6Py6Dh4hpHxaPKUHZ2HIW2eFjvFPB7P8XHVGsIqNNMshOVAuTWySZH0vBcS2Bi0VcpJ9vFUrw4naanRbVsFV6g\\\\\\\/6IFoIxCAd2ctTt9ROV4+Wrulxdx2KuQXph+hBAlNZPxpcUy4JbkuuHpVLS3ffhPdhCPw9LQjwMOzgtPBaqpaF5FJHeyMeZB9LlKHdQeI98iR+bPsb\\\\\\\/XA\\\\\\\/Z1IjdwhW08yPaXODgC3ZJwUKnBIDZ+WSk4efvUh0XI\\\\\\\/EPXUqzQgt+AtXExbWjGwfR9TkB5\\\\\\\/J9odGMuBV6q8YcaViN4\\\\\\\/s9EOBLk1K1xmBQi63WzHdstOk1sid3vaqpGUvC6emMA4su8eEN8+oGE81A7hYU3ycWvxbM5HTxMlUCWFp3SF4\\\\\\\/CWt+EPLNrQxE11l56D+g29LYIfs7wD71\\\\\\\/OCYr21PgQEGQ9pja7yQvcdVpJsKA3J1OVvUpGkGy5Dctgkl1byYyUWr2Rm+FXBO9Gw2pcGwxC+RaP2w3bnhGE\\\\\\\/SnMmFjCHlhsyw6n8yiLsCoj5N47UY\\\\\\\/kCh\\\\\\\/Ftf4uQsQ5QcIz4K30CgMYijFbQredfIx8tQEejjIP\\\\\\\/LBajlD05xz6jjHAUj2YloIVdX\\\\\\\/KO32RfDoFU1EptjZ06WyMIJMxIgW6kbkBAQ0Lwmqz44fSUNbhZPK8WAXH+xsF8c18BP9HJrOYtTpkog7mqt2RrPPDGRBtuPZx\\\\\\\/tyDJdNwn7XTO72dgPLQLzh47rqYwHNN0rHgiVW1Ipvnh6yVAW0boPMwRkFGRyqA\\\\\\\/PghR3NBauSfhvIhZi0V9gTr3TvTswyDF3IKElul3Y+dr\\\\\\\/9RVMW26cxXlnOfWrE7u7WRQQBGcblrTUMv+ohVAL\\\\\\\/Mhk4yQ5VNbX5MtAMRTRC2seuuNhBzkuWMY+i2PzI5DM8EKzYGNvvNpP6k2Y7x7+EFFZ6eLDwEbESCMMEvKFp6Uw3\\\\\\\/P30SV5PC4HXWRM8zi0yx5rmTNFwv+O82taW3ODiCJLXAhgf5SMUAJg6BY+kv7pGyFF9I23QOBtF35deQWTKLwDJN5BwxFnMF5JmTkCUFZMFmRH+mvDzFRaeSGNd9x+hOk+3X80tjC58yX8AhWGoGfhzQq6qhNUtwanXMQyb29e3nYh8Vu20wO7l5AJMyWXgFOASaeYFTwomwwtc53gF\\\\\\\/VGGiPMz4KQoab4P2WIdXAZ4CUV0GoAavF8rjS58sVh5dIGdlqyS\\\\\\\/x9f2Sw5zCoCtI5KHoiGo47IpQcLC51FAUYHYUtgMIgUHAto6xTHWeJoKd2+F+VVFbdB\\\\\\\/qig7\\\\\\\/T0j2Gj4iNFo1PMuwSDV\\\\\\\/xHpew8QkSk7\\\\\\\/VNkq6tSrJOWAhTtxYInUbOrtJskj3gzvyWyuTEbdJrN65f8DNHA5+7jLY+QPDC\\\\\\\/enES\\\\\\\/EsCZmYvaRmDJzHk6UfoODkeZmLeyWvu3LsMVB6klmDCxq+LKkIg8aTZQKLdB23aKErn29Vtbu4hototCJCpZogSTfgTxzufcVbMPMKJDIDP1g+ZVidt5G2Zx3r8NBtqEutUxdCTQx32h96tWV5Lzu3N\\\\\\\/EBC+NbQp9Pind+B9EZh8GcKrOYX2i5Yk+FCAIHurGqQBxP20Jc+2e8xmMl1HMjgvmtRZz2j5wE5OVMdEnNyiyWHFfMoo22txNYi4DWwNCXT1VXTv2mqKTqK\\\\\\\/nMnim2Uzs3CCgnpbgMFT4oYAhAOriLJYAeqRcwltCbgYKe3Cu3y1qsrLoGlExtT1E3MpX0gh47hKNBNqUevJaTeoO8Eb8dsMMLLj4yHseDjbHg4cgXKBVtLDIQwGqB1NZOW1lvELo56Oy5r5E6e6\\\\\\\/Qh6ZHidlTvfDBhadnJEYF+OYFoaWwVNxV2mMnz3WokQo2NmLy06zS9hNKpYL1WITqLwaU3QDtSiV9GLejr19j2j5aQFWOzzasS6Lq2xbSpUbUXu2acDFbYEvG18WK\\\\\\\/lGq2uwqqYzKYZNljgyQwY\\\\\\\/V+9bRw9+V\\\\\\\/SlRIFTIpvoR\\\\\\\/1nujNUSM68hH8rf1wkcnowR9m2WAVPF2+D2gSHPDY9HWYfZlDLqBoP03EXRdZa7\\\\\\\/i1HFrgFENKhsyUI9gTnhb3ZaHh86W2L15h4PL3srxClehD4WPDaX0LXJi0YUHeExd8DWCwr+v9\\\\\\\/1Mthr6nGwF51N8Zm5pmIBoiyejVrAjk7C02oVQhILhtYUz93D7GAIlNzSWMJIMM5nlZmokyLMFQu0qRV4uX+WKRzfuo4SK2ybzRlabZdL3e0WLA\\\\\\\/dn6k8mY2Y1EFLlCgh03qiz6ZjHCAp1vxNcIv1Nnta47LWWkK9cIBMDgh2AawgbsBhHYadnz\\\\\\\/O52W3jroQC+\\\\\\\/3o\\\\\\\/ULekEtBw\\\\\\\/Th99qOsP3rLjgut5yNDGxOwi5RWAG9CbqS5js1FZcr5BvohKVV\\\\\\\/fLdMZ\\\\\\\/gfOuB0KrtUZ2uCPuh4h6fSQ\\\\\\\/DwFonKHwJA9v3N3Tn8ETvJ\\\\\\\/UpvNFfw4U2XTcZ1yYt2WZkeWDcr6mQrBMq39FAHIvNxb++XMz7P3lCp+faXDRPNC1YOM3XiZ4eAa95uFI4BLyA32Qm2XuN1p63D\\\\\\\/ASOahaTXzB+VHI6z+psseJ4uOmBaKKUzZsXk3p4B90EO4wz++rs3jz6lBq99JRfN0mfu0j0U1v\\\\\\\/mdCCuLWVCXaGVc5D2mWq8JVqN\\\\\\\/vxlX3i3j9HUBwG\\\\\\\/AVzeP5Ii2gjib5h9yzT+jjhsJx7Acas8V133FdNuQJO317Y\\\\\\\/32XxMdeNv2c+Fgvtz5q6uKfDYiTtu705FgScyGE3E2dT\\\\\\\/LQvRsQFDmtyLY2jVEv1SOo2559XhS9MsAZX+nRRdhVViFHc6fnw2i4wH49EeCo9dsDstZfFh7JDNfJvQ9Yr+OT1WvkAt7lfaMtOqFMfwOiNr6cF6o38lUobJ51kn3cTfBSooHc3nFepVVUrO4Ap3UgTY0XM5+PS1pqTI87q8IAo8tEIQ+UqJbhpQlqJoeePhP293LardQ\\\\\\\/srWcIgE0Fck+nnZ\\\\\\\/dd7uUosHdDbKbrAFkXqQt2EzoVGpx22rpYAEUGghP0xUAYRn2IjXPi3JGgK+4avBM6z12cbbVQwqhcc\\\\\\\/7O5SSaUx1DV99Xzk4gzUdJ6VmtCYsBGZL1T2ixdW727+wBM+db+yenmJ4S8O79CmKZpSl+LbhkLkNoAeJVO\\\\\\\/YB+f4WBf0Y9HldgviR0XmGbKwWnsSjov0PuRQcJ7tTdYG0EtEI4KO8bQIL50v3m5youRj7qTuGrx6zQmzXfCEsLer7yr9Y2Bg7rn64x0oo5lWfnf+o1BcwcWizxJBoxfMQKWV7Cjyzw8pYcwsP8iPZxGkUfivp\\\\\\\/S+L0VXwUDvh9Dnai1sSm91\\\\\\\/H36wgYfvFRW9QqZoN76LonUWym5FlS785n+Hqr5jxeoiZa9am4OkynrJqdE5EkfBP86kFqy3HuUltO9ga8uJ9oMQO8r8Ic1krsG53KIgrq440mgdPa8uVdG8wz7kQxhM0VGrD\\\\\\\/d4Wt5mx+aOC6wssf7dqLa0l2LwpwMd1lG4TGl91Y+9\\\\\\\/8i5Fl9E9pLeZgr6fLHlSuRLDVlovGrtwsgd1EqIeZa7IElcP6gvVhpzkrmvKvYPNBy8BFJbVMvQdAAA41SApTOj2uF4rYENL6oqvJkcLqD23FQlzohJ0vkKpWRTLtfsANiErO0O7u25WXBWJJJ3vrU\\\\\\\/ZDN0ryCbqhdAkYdgLbGUFH7CBb7jjnXYCcui\\\\\\\/UmUdUTX+PqNLS65p1eAfZoxG3jbyGLS2ZvSRhYgMy6Wy+bHn10ViVI+9Uq\\\\\\\/vxosTgFXlRCWaDkhPhSiRkCWicVpQ1v7KEodSHa3E3Bk8s8Mjo9ad8xlE7ObwMWDFLTgzAaBnAVizBc3r2ZIl2zppSxPPROdJTArOFD\\\\\\\/y7eHJKfRTJpLY\\\\\\\/MlOPEznOrqX0KRFleqSMZIP6+eyE6M7MD7PpbUPYVn377jiSly78VJkLyi1\\\\\\\/fMGr5dEdKhKYQXSb516jBpblAWc08e0L3TEH4kfGO9pieZrs\\\\\\\/9LjlF+1udrzkkKIxPRrXKHODJ9Jij8mdDex\\\\\\\/L3KCFBLOPICHHJKhV97EXAV2OcMwpGmFNdNyMFPxFRO5orN6PRKlVuM3bSxkJNV2W9VPAQncF06QR6rxqJvvlTFbGLpgkgvoD5u7TczT27m\\\\\\\/W5735xzz8ApZI+lxUln7rEr+wLLu6IRSuIeV3OJFx6hNsmfeT0PcqHyAE8G647AVWroUhXfpM\\\\\\\/MvYbsq+XyNA5fgvykHBJ43HOMc5ZOSAYDK\\\\\\\/noTB23ttOIc80NvJiNlHzUx0MryjgnRInn4Irz\\\\\\\/TeFgVvktwDd5M7\\\\\\\/EKJqosL+sARqa5Ebjd5JyOlhY4nNN5Db+XrksBmVvB9jJ3nRLSo3T+zxZK66W4jNQLzc5z8dVJUH4J3G1MHhOANbopUtJnV+UfhjLqa7FXOQTKtvpVRZ\\\\\\\/Tftdq7Bd19tPNlx9wQC0c5EVzIbaW1bPmK7lge+Y2IxHWAGcAzjfdsA4ua2QsA\\\\\\\/eS1ARSiLjnq+SzlRWxz20V1DUXfovFUEoLzGsSAlDtBvIw6HN+jdxZ1ujPGsml7FTV0GuICcKoedoIEIXn+0wVkXJq9FHnetTgxs358Vt\\\\\\\/DlDCTvsLjmSy1EmEaL0WSVzsTeGPAQYoMFEfDFlHCBJfraFFdd27C++3cfTZON7\\\\\\\/oyKyAMnAhxQoIxyaYvkbHL487HNg2Rbs3neXCOkU1ofj0dZBnIIKLVjcKk1srw0Vgz13dwn1oWlbURFYzVaq\\\\\\\/z5Oe9gEa\\\\\\\/e0D4TqOutFJMSBLe3u6QTCY8PMOCSN2CUQTH4GscS1\\\\\\\/WH3sGsaielCVDWuEC7z61Bf3HD8mKiYBbz64bKV1pXWnN5UYP3O5sc6iT3jmrhTaRFPQhQqQdJwcBZs\\\\\\\/FxuTtYgJwkMF8HM4j\\\\\\\/9GkVKXXkrr9RRlFaUyWtuwsby8+Re+xDOK3\\\\\\\/uPy1UN5aEnFnsFCZxUiBraUoOoQPlCXky5VUsawo5ooUbX3qwpFlvXAaQxpBg798\\\\\\\/QSdDgADj72wPlHZ+HEFaoQAUaxYXthx0Vh97CsIX88EfYwGNX5Fc8VULfYiEA6jHuIZ8KIry7mjuV+FAlRA5QfwmN0eAjXGivGI8rk9hDIoVrS2eqdylU5aFh2ai1z7whwXM\\\\\\\/z2Z9hb1m\\\\\\\/Ql2zl2xAWmE8EgZs7VBrcRKR\\\\\\\/\\\\\\\/A7dTRrFDUUZ7n7g7xnUU5tK36dh4PC9Egfdyj9Qsd9boZ4OrPdnxAaJGTAuWWYwvJJN\\\\\\\/0Y6IgANPuwIXIVhoXKkRo6VrQZV8u4U+QUEWhVS\\\\\\\/YzxS2s\\\\\\\/duoXbwil5jYzz43McDWaR2wrJdGF\\\\\\\/6jY7j6jHXZXNuW0UShQKqfYsWAfnLXJfd9gZcydN4hgwksmTKwFd\\\\\\\/0h832HtKJyYrvJn9RIfGc+QBvPv2zklaYuI\\\\\\\/yhuUPHVMhVohJvfk85EvrIR03FvcW7dDcctHMC9i8vcZ\\\\\\\/T\\\\\\\/btcMxrg0aTgEtp4rFhzDsFaSLenS6QAxV8aI3G8VtIT1OY91dN\\\\\\\/99g0t0t56Bpf4+8azyppH+KvdlqMFy9bdSOuSKbbomUDwJMD7IZ1Ek+44EE38Lo9N1y\\\\\\\/a3h0GM6LEWa+ZnK1p5YdoF5nelLqU6+7SfGp+oxjKodSEnq7wJyVdK73WMjjrmk\\\\\\\/ZdDOr6e2DhrSAfgBzjV4bS7qo060HfIXi9m9xUqpct2muu\\\\\\\/iNGkBUpIjah51SRtSR9xil\\\\\\\/8BOpWs7msxwQ08mn6VCgiM7\\\\\\\/XZkNeF6CIWf1Pu9j8kgkUK49g3S+v8+hnUskX\\\\\\\/8olLNbK6NI2sUo2H5lwjhXYNzrVxa\\\\\\\/VzMiQlq4gTogUORtD6vMP7i7E4ZlwgJ5X35CtPW+R7Yeqr7jo7vdUP9VwuYccLtkgyt5v9IWUlQLi8QW\\\\\\\/xI0knV4zmZUry5t8wsX2\\\\\\\/\\\\\\\/rDSllRc002w++5Jsa7vFPBPTPDQtE\\\\\\\/D6YWwxvCTxrpXnYo57ylqKu\\\\\\\/E0qAxhn18ZjR8o7HS\\\\\\\/5JF0DxFMEblZ4x5C7OP7x0DkfkK\\\\\\\/H1nKqj16FZh4\\\\\\\/I0PIj\\\\\\\/hax5XLIza1DvVYE2Ca1lkPCfYdewHRjJt\\\\\\\/q4yW5d7\\\\\\\/SjbJ4FY9lcUVQ6gj5FrKUTNfsYlzUWdw85FlOPQq4AGt\\\\\\\/YWMu4B9FFF+CYT3iaFC0zq6u2OzlwQ0i4akB1bITKztpJlYnLCg0j2wNtieVSrGwHh2KfI91tzxQLYeTDTCqqSjS6Y65DHpadBImOEqaYvIeA3I4HBO3mBXxctOWqfALwRKWyiT3HvH0SltK5XtDu9TrHlLg5wBO6V2yFO5JzAjLihnlxDDUNGP7fpDmwm9hKLY9DOebKdHEKHXuwCmWZuTKRugVyD5esSo00rNKjv2JmrKw3e7k5kkxK0ykjY259iMMpxQZGBK3K3Qu2MzSgTG9d41Jjma9ZuaMXJQpm125nPb+V6IXXHfV26fQP7qaRD6mvN5i1d\\\\\\\/Q\\\\\\\/AQcGHRQDU7NbrIPe+byejN0bb2J8Ed+bjyAb76Hz00ku38u5vVlfGkkWWz90pPqLffiVNNlLiQk2CyMpNXfFhkVjQsKIpWgWelTZ7iwrxZ1ZFthV0Zs0T\\\\\\\/kyOH8hHmForVLj72ZSpoDnqbkKcdQV2G3BGFi4+Wo4PQ++1CRPa\\\\\\\/E+Nk5+4G2z+6F7htjSvzRFAb4XNiiV500NpLj486Zo78K\\\\\\\/LLGFYaEKXAw9v0S3Es9tM7E4Je5ucYUFfkT6VRtZpBC\\\\\\\/B2+4JZ4rUuNOg7jyHjBxrP62y\\\\\\\/acR4RKKZyR2hmp7HubN7TnbCrOdjlD3XJLB9KdrQtD9S4PQVfJyw75+dfQi\\\\\\\/f8RGixahh1+aO01TfKZ+KSfMdcCDYHzx3X9fisejAnzFPpGtC3k80mT7HRPeFQ34Va9\\\\\\\/NYt35ci7mxBjCGh5Q+JT2ceLTei7XHOE8U82EIfd9ISpP5re4fR+NqCRUrl57ObSjPsYKPFJrRYQllph\\\\\\\/Bz8IWlsonHEyf3jd8B7O8vS+HlZHswvG2GBtP5GA+deoX0mQ2N+1YRfBfY6KsxwjUGThMvPqpJz8PnYEJkFwz4fws2Q\\\\\\\/O2gILs3+qXpvXjp6dSba8TEVH6NG4\\\\\\\/N0vmNywV6fXUNd5HOh9q9jXhIPKjMPcCPwMPeyCMrs9lo3wr20aORz555YjfM3MKLhTifPzrUdnt3sXY99CDLKg82\\\\\\\/h0dy9kH93Ur7D9f7\\\\\\\/lNn\\\\\\\/uYGAn4eX1KuzeMcipo4Tz926kJyzXa7r8YiWII+Hbxw9FPMQ5tiN2UcFXLauLUS+9q9WS0OgN2uLTQHLS3Fwdd+AETkucvWZfe9yF5MbXOlD+B+oSJEqwUWxr4eQUkR8gO2VIcT4WnH5AAybrypx3NuxydovFb6Plw9vy0+ia3Ppl3gtUuNrGcAhyvxOmbG4xtG\\\\\\\/4uAc5XmjaqHbKkZB488HWbnKGapoOwBUVlB33sXcf5VqV5520ij+vc04\\\\\\\/9H6f9HjJuCNzVTiidGN8GL7vyLNl2zIqL5GeV7tTR5CmdO0dHwrz5G6r8GCfjcuMCsiio8e2yblti2PvZxD4ZVycGP7D7Z5VulyurZxJ393DXUUEjUs66hSuy1PVJ5vFMOC6VibBNl+KdN+d\\\\\\\/HFzZ8jCLSgdpvuYjn1L\\\\\\\/d5TvgE3gQqObKM2RVZJMqQRaDeZYoRWs8re8cU585cVVWYvCNCgK\\\\\\\/OhfEiYkWLP58sQMzKc\\\\\\\/eLRTJItQQQwHWCR\\\\\\\/4ULonW0sfJID1jbPif1f5HFJXcKNxiPaefRRObCvkdecU5D9BwsVNk4PcgU35SEaGJ1kxwQZpjQsUif\\\\\\\/JzEHe3H7925y6bLZaWyrR9anPIjU65v3j29JHcsHuoUCusJQg7YBxLh0Y8p2NgYqhmfzuUeZ4AxbtGO61CsCh5klW3PbS2NNxKhgV9v929bTWp70HK3mAYPTNUema0nxB1kKjsSO5ovRWiv6Hw\\\\\\\/t44O7cu3whAWB1y43btSx9SN5hpvZwcmYcgj73ZDevP4KAVSTqnmgIY18vI6lqBzBhkM36FNwaNZRFjX3JJ\\\\\\\/mRn8VfZITS5vDaaZnQMkpXnIc1IdzZCme5uNHbk8WmFgVUgzW5xd7\\\\\\\/\\\\\\\/IqLDV0Z3eTVNDYUmqldndQQ31HMTSCelZc+Y9kwx6nt0liBoCHzFcJ7VpGftB\\\\\\\/qWgq83Ar6LDNJIEEM1NCpf3noRRb\\\\\\\/8VurpwveNSaYM2ym5AwBSShpSF6DpE67ZsH26yvhS3BUF\\\\\\\/o8NJ2HbLjy34Lsgmuyd1gTluOlzTZu+0tvHfQt3EWLd0q89rMfBmThQNteF3PJOuWUjn+7mQ6amnlhnPP7BlCcpCNxtEErZwkpAgw4OCmePMQaRMyF24zMF5X65ecubqki1bWqSCbSjYanRrAWGm2D1ZNSJpd8x66JN8Rh\\\\\\\/BFLibXqTPV13ocQH4DtVy62jjlmzJkQbhu1bZB2iKE8OTK1tZur9c7Us0N2xPhRTW2iqb4Ne9VBH8zUOk2gHquTFsp19+obBKwkwzQZgFhlpAItK9cqcGoYcAxXNkqBcxgcRKns5WmJjkHDD+tW2wd4u2YWarCP6Hc4rvzGqL+6aHhqw7CaVO4J68MTm1Q\\\\\\\/oDXPERcJRTmrLfx1LaDd2cxSTnYavpGHb9DlJGrxGOWJ5697HESVaWSqNaz4VVrCiyd4OsEWec2ByhBpKWK\\\\\\\/G1s20VzGzLMAbiBPk9PlmJc7ay9mfujy2cJ2hJqTq8Zs7LNQb5CanAxbLiAKoJP3e7SeZcBPStyQ15Burk4XBdP4yC3\\\\\\\/jpLX0959XfEcRDZE2auEUrF4K2jM2anjBmXtskZS3e72GdbVgPUZQhhm7GlGRHoIo3V9kNcb8XJ03NLBABEMx9TFyAgvXTWCW8ZDj6XYg5YYj\\\\\\\/3YccoYg38KRrVGWEJAFSwlit9QjgrVezXHYqpLlgB1aPTDV6jcNaJi+XD9qiZgtV\\\\\\\/9i5aoAOhOOC6y0nyPoQK16vgYBdKT0tGoD9lUyABkb\\\\\\\/\\\\\\\/97lqc0VX\\\\\\\/qnyakvYFf3SX6AdOLrc7evL1iO3eaIufK2+\\\\\\\/HPJcKtvli77ZApegjTnFuPy31hH+oOkp2z+3ucboXFvv82jW\\\\\\\/6lrHY02Vc+lQ6aWFro98cRZKCMOuEyEaeaD2bLuSj7jfP7g5Rh0kl5X88kZscMKU5FVdp5NqcYkNdpOhLLyYONZfJsHTfl1PxCPYREuLTt9HOg3WlK9A2MJdDKS17kh8y6hjpaN2GPAondkrNgzGVeR4misFgzFxbspyljnRWmrMDlXk\\\\\\\/tU+CslMX7KpUOnAJhSPh81pPgpVxmzBvV7eXq\\\\\\\/es+Wd6SHLSKG3uC7uNGlZVijAcqx7rY+Y+9ny0O1+DEh+kMSN5t5gAKBBJ2gqTg3BVndB+s8bnM2B5jziasfoIEfvjBAvcF97rN2\\\\\\\/pkaftAPSe9O1wsgeluSXKMsDSGBld4\\\\\\\/m4RCB8+kwJloxDIbqIspK4TDfcAlCgV0QTHhc+HcBKCYfJnkv4VXdmWNZArwjk5ipgXgJR3NKdLJLgO2Q46x\\\\\\\/OizWqLaqxWQzeaJQeYh6+S0o3G1aUlQPiKvggN6Hy8P+WLUD9IaqruT9onenS+CoIrdtgvmL7iCixGYBzsH1JYjQh61RdiWbBe07ABf6vxqPNFxU+2XyTafvydyCQfiDw3\\\\\\\/GLLopRKLg1THzfVVn0S18bTH3Sx9w1HJH8VlUUQsFnrts\\\\\\\/pHGsV7UqvICAZ2h8dfuiL6vEeI4Py79i\\\\\\\/H7mb31aex9LaU45rrNM\\\\\\\/twQHRAibL6ITbRXR13CRoPC4hDD4sBKzxRVJQimCM7uWyW5yi\\\\\\\/Ek7c4h8b0LWxrja3ilx\\\\\\\/n1QuWRdeeW3TJnLJejAH\\\\\\\/ZA2uSojP\\\\\\\/nwND4yeG\\\\\\\/pdRBfSk6JsOnO3YM5ieOH9z5VaGGD1z+hHtqd2Eshur+lvu\\\\\\\/5jt9cn55e9uTJt8uxn90AIcYJquj13l80XgmxEpCQPCzWeShHV5nd61NrWRrWTRaChfPnGCt91isQ24cYxKyEc7MYqTNx6jdHJ84HlYeqVpDiXLfsEwH\\\\\\\/\\\\\\\/fiRgzzUAoC9gFVuLEQXZ6tZe7+ii6FcJHWtFyTTigP8PAHGQmxG+aL94TgWox0KWrBt3l08LCxleOlagxz1ZjFBz2WTrGiJooE052S+b0KDA70SHGnJtdDcfWHngX60Qco4wsQnmDyda3xHDXvUvW8RPHbTyWzjzjT0K5eA1jm5YXh9KPRZIaAOgdPna5rfpIHk\\\\\\\/WvC0Lzscngdjlq\\\\\\\/cMWZAIKwYSPv7E+IPkaR5Uy0+5mJQr1iJD8X2gwhKtJuSWFDaj\\\\\\\/TV8LUgIWAPON2ngSfOEq7KqHqqSCz75x6oA0WPktuqB0VTWO\\\\\\\/BIj\\\\\\\/t+IaNRjQyAWX48o\\\\\\\/uy+2ue9lEvRZI3gti8EGgrKZuNybP78ikCDp2J6t9bq3NQRxMLXXobjoBKe7QvqTXhR\\\\\\\/C5mfOx0NYZG4THoMXk6F1ID1u51k0N3dR5WAGnSGvZawgOSnpHdbU5yqyao0qjb72zwCdiCrnx2w68HwUNwWvlXF0+elFHq3zaWs6IkG00XCFkb2YSKyfru+bR2uJFmQLiUcg+A7y8I0P08YMYD2SeJDsplrKZ5MeqZi5E4zzbj7rSziv+8sJGFykeHoUqWRWFrtTD2CWPeJ108LMq0sVn+meRshxvraFtKAkoTTzvOi81ZSPTm1Ilo+J4YvNSkatUUpPO\\\\\\\/aZZ0srJ8FZzC9KE7UKKAz7LAGuGAkWNQ3R+1as9hrF0c9ajiK6BjknQIsNR7v8dlckrseoHwwwrOmV6XD67yQofu\\\\\\\/pvlzo8q8f3jRA7BOUUy5X7N14gxaX42Jq5Ei7SnJZDnmWK9lXX\\\\\\\/nPc9wFVQZOUbGzeiarZCF3duq\\\\\\\/bF02d1DTiP8rAM5DTfNAK60HDBnCnVvdnFNqDnuVhkdxwssSPFa7yVehde5p+rMYCTnvVlL5dnzSUC+0F9QPtIjE1PN1Z6fIZ1nVloXFisg4J\\\\\\\/u9i0Dbjm\\\\\\\/S50q9+1zxANi99TtBbCGk1\\\\\\\/SDyylzRtUQ338psiZsMELT+dUEY838XH9uT5sCPBB+UEz5gCGuOjBWTFtninSr4PVokYJw4ng\\\\\\\/luoEDU9eLxEu6RRFdTJi8Bjx0a\\\\\\\/ZchAuLVScWGtmeS9D0BXFoBgzI5mTT6P1AJ1NmCRi6WIwmWM8z4KMR3s8TXXYWsISmkHg6VwERHFuGV4EOsLmrLXmRLOmCysVAxyvVjO+nEuxpGQ7tS5DZIVByezQgk6rCCuzuvHzxBnOl+5tj5\\\\\\\/FadboFfNXZe1wvC51qbpGqbAqZt2bWkRO5I2OZNEmGRZllG1TYOOo95YTqHqNBio4dxvUIKV06+xDqx6NbqbatTLUiS39c+ziQPewJ7k7L9OA2LRbSLY24L5LBnP\\\\\\\/XXt+9lH5CmdoN\\\\\\\/cUT72brBwby74h5FDtjN0UJ24nSuBZ0wLuh3EhqlT\\\\\\\/m8b8s3UVNSnb98ftwCwTN0I8arvr8\\\\\\\/5Xb7j8TZjY1dzc8gawVz4s7bVcis+vaYll1zWaPnbDVgk7KkeZcBJWHSrMa4E03WFmswM7ExoEMNHKnqkHukfsGTwz0D6P+F3P3RMgEXm+owoP5adBV4VJTYkp56uGO8jDHIkGcPAn3bcfJ+S37chnC0734v0DN8yOZeFGUl5y4DCoxw+rMVFTR1ZllmNTK74BNHoa26BvAIX0Mtb6GL\\\\\\\/j\\\\\\\/M\\\\\\\/zwJCzEtGA6+aYM5paAStg82C6+TVkHRMiEUPbJSxSRo8DY59apOj6W1DvKGBdTlRx34E6ArR1m5ny1FAoOAz8vay7EsLWLZEDanMHnNOuHlJefB7bRlhqOfGFUOwnq3DyE5nrSj3bLefHO8W0NPAqhw+ACU7jS6pW00WPVJ063oLmSnZPf8YGKuHs5oT9rDZ29ee3\\\\\\\/8IfDVVXpwSizp9B8aABj+V7RM0BP59iaPRNNEmsTNvJh\\\\\\\/BjCHNNAK6UC86DBY8JLRdSgHI1+9\\\\\\\/As40KxMTXNzOyIr90DCxQgon1w8K2dPUleR7Es7sR83jEPOvBk0qTB62V5ULsZ7TNpHTYMLA6dfPDue87YIKiNblCKe5bECH2voxEqeqKOdM9gdXFUYF+TJkzM3pEx08iEKYv6Z5szruzsBusjS4KZXoBLZULVi\\\\\\\/DgX6ni+iETv0ScZGJ99i6uWq3q7T4Sw3VxuqLqOXonmrv+eDwjkn5dlgCq75sz7hvttm2FR\\\\\\\/Kk0yH7apTx6nrolvPf+SgMSssFGS6+W9N5NyBNEfeGJn3jWDi0W0LzwM5bfed511R6gVpBg1zWQ0oOZdfJ6iyueS7U+0mCDDQyLyA5RY4\\\\\\\/Ir\\\\\\\/mHs19ZwbtfVBtPeNjgGzu\\\\\\\/+S1P4Xm3XiotJeMm7oclR4pVlSCzrJZ8GqXWjjJxSe\\\\\\\/fC07So7jliLs9ZikNKmBCkFD1jqUcGRW7ieJvVdfd9Ub8IOcpa\\\\\\\/jMNj96tX+MTKuFEiUnskZXYUUAX2hZLYok26jiN6X88hiNn+fDOTE6T1b4\\\\\\\/qI8sFVoA1xfDeonKw4ahv\\\\\\\/K7QARjof5ex1cBpi1fpVC3JfmnP5EVYPYAfiGdyFgVU6gD6QtgGsp7SleM18F9l\\\\\\\/VXIBmrKglowSvHHu6fDFAw+pTXFguhkAWxfOLg2Qi\\\\\\\/AjqIojrEgqr0smVkIVqM\\\\\\\/TvqED6cPbujaE9atyGGeggrhuOccORKsjhnXlD\\\\\\\/0aEBeikqBGYBYEIJ0HSftvTnhKuG4m\\\\\\\/C8sSMyNpStjosESbUmkyhHwvUZ1egxc4ugEFUZjDbqAUIzuimwGj8+fGtb6JLe2Z8u0PHIij7BraPIs5ZAI2mIG1rPEEe7BqDpprhwsBd0gEyMtcs1Kn1ZvUCOKwdJRd2IYO98J9Zz\\\\\\\/ZvU+ifEAD\\\\\\\/St+kJMapnXwH\\\\\\\/XzkVnF\\\\\\\/zR6v4GcMNb66Gm+qHl8WCxHvGTX46CBH2hA8Rw3l4dWOlgljcNKBHYL22lGDPb4LplT5cst4xSOGHeqAiINzG28zEPGDVd6VbAGTnP6g1\\\\\\\/1GK9P6WyJqpnTCe6LV1zMefkdG9mGIZPp6SfO\\\\\\\/juohiX1PORRiCCuZWOTEY\\\\\\\/1a25DVDEYABP7gfE9m5BZnoFUcTDXafRIAw3HuuIIQcToOyvpzGBR0ViOCq1L5JSIni\\\\\\\/TkRnETGS3VzQLt6EdgC3m4O4iCmSgJ76oGHx0p4D5gtCs1iDtSpidMKDoWZ2R3itHfoKcJ+33eMyhlO\\\\\\\/CZcZPuuJRW9wUSLAr8CtltvXYePyg8Ws6pJwY53ijF7qeETXzgUWKHaPCrApyyyW6ShK589Sd7o9MH1lvSlAHX42WcAWGPhvZnV2Gr+MUWkxw20sRslig0\\\\\\\/Dr2iF6ZPIrdj43QYIIDlHgv2mtj2o6YM1AnmRjfSZQlca0WkXiG\\\\\\\/onjNyC6AnQ6afwx8p4BjzK4q5HQX36qGQwN\\\\\\\/AdBLYokuyNXqN8FY\\\\\\\/2f6ZGE2n5wdPLMg3zMAmeff5YEBsif2YPgnPx4LpZKpxohUewAICkL6mlsabzNtuFs8ayrWculXVorJH1HARyuLfGCsNrTONu5e+7w9D1j9ABSlpXu9ckXC+S4LwFlswjVH1g7J7WRD+kNlFKtgPHl6GKC2PUc7FRKI3g96woPWk+OA37Zgt9JBZZUrYe5eI\\\\\\\/9SS8Q4ZE1SYt5+S+ezAeK1CDnpfiB9QVFJtdjyFFvl5k3ztSMSeIPBOSmxCcTngN0\\\\\\\/BTEqp5fsVL17UR1ZWNWnmeWRiFFLk3gNWu\\\\\\\/WB8D\\\\\\\/8aibz6y3hAP6anCs2mAwBu6BGLAapYpEsO\\\\\\\/POlKIJyWE82s8nq5v4C2g8ohtmeHucM2\\\\\\\/dpPq0jG+sTiY2wb5jOhB2cfV0sn9KV7GR1XHF5VyoHxJrlTkArKnefrqsVt0lYyH9BDyVdxwEYhfyaIQgo85ttEGfnuFTaoBo0K0B5f4bRCLB25OGcvn9Vig3gkyP514UVWmKZ\\\\\\\/AxBbElHk9l4sdbLChRfC71ZyVNp+rj6ffz\\\\\\\/+U90hkC9\\\\\\\/O609xgIDRunDRgXZ1sbBcqqeKpFMVWqoxWQIPntaygYV68ts9aL6PHb04F9JhxKR\\\\\\\/rwywNMtgmA+Nud6+zP5XOr12TJiphC+wKGX+VIiSYGBYH+viYqrthvNzpAqnb5+LJl4p08aSncBV4NuC7WVeyyHaojtXqJWVhQXjKjZ2pcED8r84cyKad21TnsuqtyvkJIHk9t09Iic4VZBI8ceZb6oavkV\\\\\\\/SYrm77pzHW6\\\\\\\/N9IRHITg8pQKNRhW8KQWnoRFNjT7kFgCRCz1YGVO9b6PoWqM8YYrEEbfgfUIIpKK9JgIRFNlL5WM+L4x5Nq2W8XyRm1AgMhhWv9IVmqpmFj+8izjADFRgT4D+dd9ZWtSUYHJZjo1YybVk+FKtmAcA5eVIJQ7AvQXIPb3mhfhsrT\\\\\\\/MGoAis9BFfGgUY6rszehQS00Fl4U9rRqOzAWtv884qLjPkfOPB\\\\\\\/+g43PyjrooYz79\\\\\\\/R9Yc3S9RqWq+KJUYktoaOJhc+1lGYWlUP7xqv6+\\\\\\\/8sbLCXmX574i5cFZO2Vn7oLpYlSfD+xwbQbOB4IZ5t3UIo9weDpPmm2aiuGgi3CCg50VJBL34foZ+SoX4VBUcQvUhlCIdWJzPGaxi+FoNPOlD6mniWKm0\\\\\\\/mUUAhOluT7f1inLgd2SjJwMezGlMDL6PCeoq7UdqPgq9gv8kLws6sbHuzUdYBlJSMOfV8XOLCzT8HUfW92FULrSKchvIc9rdjXpukGHYQasItjynguQamD++h7v2gwdRnLTDYwmphTCzvpE8XSJR5icMTp35nWum1tZgM\\\\\\\/LtXM8wbl4oPyoTJM3JYaG9iGhzPzRC2ejTLJzvhRQXv3MgSEbagONsjNo5L+IL1HV8Ux4Umgv3WdlPAZ2uoQxadZZEYXMBGxC+ioKEyTXUXPwd5K6Kri1bW5d2RpXbdrZrcZjRKQxJK48+nptUyJTFcOasAjKTGg16cw2wlBKXAD5ITc883aMcNQSjH8DdDCNvknhh3m+Xias3wz\\\\\\\/XWsePdjnMkwmCKiX+oeCzLTHUvkf1kqX+B2pWPgYE6XfjN3SVPHZJkEoOg7bbqGfR6B\\\\\\\/OJHtqo2WKOhQGBxmoBs9ElPx0NG7yCrmab2UeT8TSTx1UQobbs\\\\\\\/XoBulZ+QROoewTrZD1otGCoecTPQ61u5ehsQHkvWlHCjtALbxnI696C6Mf4v90QXL6a1bmlQUpxtdX+7UcmdOHGSEGz4wOgNCiwXiYEbGER70sgbEqlPv4ziH5LFZDuoG\\\\\\\/DY5eA5bleWQ17oTtDKtAvc7ShWKN0t+iCtZ0Vab3Pncofcr89eX7cOm9snBeY5QrRKz+cenER844QAje+g72+si3vMs4A8wZ\\\\\\\/c+oim6yV9ss2ZV2dnGbr9XNfucSeGMWK0WgDPjDkXKfhl2X6NQrrp\\\\\\\/XYr\\\\\\\/1cTyezNmo4wUDRPiwM94KRaSmUQylWW8NaTaDohwYA8IMeyLHk5\\\\\\\/X47aeBisveS9s64O++4zjuEazhmvBzgdXVt8J\\\\\\\/pjE7Z8nmZNM511jYDRqDqLovWNKPliL\\\\\\\/nv8EXWH6N5kfnxBcge+y5fuQ4W0f7+1RpTt\\\\\\\/q3e5+oT3dfQfk8TVqeo2Iv3i06\\\\\\\/Crn0C3jnC3zrWJtdf2fwrv85nP8T0kkhYqnGLMdSISTWqlMX3IDegRkvFBZ3wl7rwHxis2wsjwyJgOU8fGXIamU+5ItB0HRIM7ikhZa7CqkF0nEjN2PJYl6fZ4A0SZCTKgo+d0Vz8nNzK5ia\\\\\\\/b86oBnRd0Xw69hNPNtgK2pYZ+58XEETwCm3fCT1P5C+fCBz7nCNTVTWEb43e14uiph3E0ScvJQGQiQibWit8134857A9UxXpmEtj\\\\\\\/3KE2A9fKOJV70LagmHkliKRh3LJHIltSq1yfdT4V3jubccYLpCbyzEChIWIspAmMvYvLA\\\\\\\/Ns+HV1crHMUrK8ar7C8ixr\\\\\\\/M5RrgiaunHgVLvUmx0AstUd9j3L7X3IlqGIk5N1+2iGMJPaJvBzHdLwoPmJddFYHJJLYVVWFu8D22LKiyrg3aRKDDDGbwOPfFecwYqUakSOThRe1M75VOuaBAhHmuc6mw7I7GO9uPHQaktKU\\\\\\\/DAxWTAtNhEI9SKHwznOo6Vk2bcAQb9fCmNlf7OEcpuBnVuWYInRZD6vNvrj+W3JjmRfU0BqHp7t+a4RkHkgfsPkw\\\\\\\/3jXigcM+dECmQaHqSjmXM3UrUAu8qWGwmzuV5BaLX\\\\\\\/7Sbo5AVcpWxUhx3xK7hY7Gd\\\\\\\/g65qXLO53p5EZbgXAj9yeIQ0WN0XmqyXWIzGy4npGsiKzNqtDeWiTHX8f0ny6MLGZ1agtQ7nkN43AYtho7LharHZVyJ+WQVd9ujbIxpvfgqj4epCDncSSk\\\\\\\/jhYm2AzvfzxbnTr9Qz\\\\\\\/EvoXJ47FhMJEHM3HBgJU4DxqqeBuaIqMu2ac2r1cKrfHx6EicrXO64TGRk5PjUxAr0BgjPzPi2wJees1gf34R\\\\\\\/6SHpzbsB+WWdq5pRyYud4Ab\\\\\\\/442pMwFeZCewzdk9YuCeRR7xyr241+K8MHsQ3+ygWmbfuxi5I1XDYhApeQ4I8JWmYn6yUKEtLJmAjGklC1Jp3I912awgpfeqXul47Ja1lBuMks5Z2ET1149tCTzTxCP2OoKr1C68cnceUe4f18\\\\\\\/2ixuMag\\\\\\\/HMm7vodzvr\\\\\\\/10wjxRRU6keVA9GxTnw1Vc957TrBKgT76s4R\\\\\\\/vHNNDJPdvOdzJW2BHM3IWNm3cdOhlBrUukCPf4QFmSY7+jCOTYh8rczS4eS2RDtpmzY\\\\\\\/yrjVgHYkkkgoI8ua8T9HiNT+Pc8+QFS+YqiKjHP2jrBSPqpaLlvA5+B4EjJU3W6k\\\\\\\/9A73dO+CtaaXhand9Kgh10OI+G8f2KGALCCTJFz6XktsAsflxGKAGGBIWfYvb9Eg3oCqpp6jqVV2NylIPt49\\\\\\\/l6Z61riyBJOrj6YJd+TXOiqfZIMfSed5cLH3pfUo+Sm95gmk7GlToijM7Aa9pFsTlgMbXdXfmocXtbokS7DU2RVmXsuXK0THJNPAUs89SYkYLRF7glDkXtkmrDeF7E86leQ6WYloJv08xqdqbOSb31Sr0ZzIawwUJSRQH75vqnee0P4ZdzI3sy9m\\\\\\\/cYlgEyb+D7gIw971+XtOESF\\\\\\\/S\\\\\\\/HgVmP0VBlSEtXDYoOaBwo2sxTYjjZCnYZVAFHDAw0iUFEsxo+Y6h9FRbPE8eEQMpuTFQcHp31dQ70DklIBePiB1rnjS\\\\\\\/6P\\\\\\\/hrm6IbNGT2P9syP+vk+9zUarmRjBmchxwU+T8ufsPnJDdYKYNOeCBL7On4sl5GduTz4yJzKg9pRTho92UcJSSo9HMZx2KbUPNTeG4zMtKQxAVTb40JqTFruRxspghTx5g8oNbqX9\\\\\\\/xYy6VdHs9SupetYsEyRUvjx1jc4oFRKs1dQqkIm0GxvskPBsW9NWOAw08qehaeR8mmzCJEnI\\\\\\\/vERkn\\\\\\\/ogy8TLq1ibG34WtMEkZq\\\\\\\/c18bjduQATahUJ15tavUn1dwDndguYrPn0yTBmA18ebEYJXc6aSvEfWTiDoSqTctEp6USnZ+pupQGRZyNjxJhg6BErhPCdZIdpULJR4Jypkw1jOo7ol9C2oIXyCg68wF10BvzOE9a6a5iXUtatGwpwbeE\\\u0022,\\\u0022iv\\\u0022:\\\u00222TJwe2oYbAs2OPdHIRf3uQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269773fbc1cfd567b1aeeb284ab557d85d9397b62953bf2eb4af1b7538b8c2f2f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rty5lStdrqJLVR+NzDyK6uebQiZNmvQEEZvfNdgCXZNU13LLc87RpeA5HKy\\\\\\\/HHysgi4WfTgp5iZsz1tpoy8Zt0chjYp84C9cf6LgHWkQRC1t3WzdmmueMB9kiWStUKs\\\\\\\/sHPS9SQILANbEQWDsUfDuO\\\\\\\/NG94Z4mhhzge27z2STdDpu9LOqGUbJaTpAGyCvT3rNpVXvOBlSAhWxOv7TPuAw9Qb75FyI5yf+YX3QVMNDrJrQdP62ENE+OAfGArPF5BEyTvgEMDRZW34VdebRxxDoGKFzMyscrpyukf7aRqraoMB\\\\\\\/uxKaUns1aKsiBsRN7NjYQYb4qSB7kTykDSePRwCk4Ruv1OXqcfqWWZ1OVN\\\\\\\/AzmmQPlu40QSxKWCyi91XSw51B2s8a6HVQAqFymb+caY9eFisOvW0wPhKsma+lqu3d831HkKXa+CUGeLEgMeRrcodV6fbDY\\\\\\\/dZd2wNpmPAJtANH9HE\\\\\\\/tU3Wujaks7z0238yvVzXxdBvtqX\\\\\\\/S0FnUp7bY99AC0PTec6kLgykY6snqF7oPDWYqQfrZRQB47pGOlwGfjVcn4ucjvFnLj9sOWXOXjq2AlKdjGmoNeHF9MNaPmgRwiU76iMhGtptLYwLmKFxSm\\\\\\\/MTY3FL+7algReiq28\\\\\\\/hoLc+KWrVgkFwMikuvMxgutJlb\\\\\\\/Ee+zMCF6w54F+W+zuZk4HJbGzcjxtyHdcHpg4ZqgjZqP8dhiWn2rYOaBq3wz9iLkFOMA1uUNn4XmVYA7J3f8TZp5YlNMPWQXw99hu+e3EIGLN8OZagX4DMn12vAe6HFsx8MGypeIKElmUltaOQsH27TUVyUUSNssDLSOmkRJhmhUILl17\\\\\\\/Co7bND5qpSzC8b14Jz3NAzSUnNkzPFND2ekaDv6HW+6ygsO8sZ7rZL\\\\\\\/2pkV8sVGpkUunkXBXkaf44J\\\\\\\/HD3oCpc4vQiyBRICxyAtzQdUzqx7AjdudgQxBeO8AwYuiK4\\\\\\\/RJn4ayw1Vd7Ey7ei1ddBrBzcm3MJ8VMINoUZcggQEgeMfRv0VJyNX9mXPH3O4HOoPgPxhUEuCKputq0Ir9J0IiqsuoG7hJcAOWLMmiUCn52zKk4MoMB8got+5IndNgBAMPAltfOwwJ9HcgyWj0i0IGo04RLZeRIlloEC333w6nevnyuRuJHjsUPlTffJSBGy1SI2mwwpQvexXEKgi3my4K32XoUGL2\\\\\\\/615RXAeEGLyo7azkwhOXs8\\\\\\\/INy2o3\\\\\\\/GvohfYMAjcn\\\\\\\/wTZGaF4fdvGyCVGF0cKjtb1nL6pTMrhza+sX3YctrMVhJr5KcRRsfX3XlbxtI7UUbOSyhLDOnNveWdjppDKmqJmWgSpvmDLLPCTxdc0j33+hhKesrKaJ93DznNRBhotCYJQbUqOdfejR3+ilWJZUGkJZUyu4f6\\\\\\\/37iqd6kHUjYvYSyNTjgtoSJJXnUvGKRFuec3caUOiKo\\\\\\\/GebPw7Ut6HFy2zpTwsw5iBHAZ7Pbn8ibScEX2MsGR9qc2QO\\\\\\\/iAnubL5GuZysG2Zml2JXsLsPBItg7FTbjAqrYDsihqv5TuUzjhxW6OmiwRajwxqofySQreFpxmxCdMwZwDn+giGQ5eD+bZ1FIPtpLVOM4QEPolLw4TkJIZ0EQr54MUocQruFsubOrPQEdAmjAaB8hSMnCM06DGCeY6HyNCMctO5W+0ztUqU28xgsYyNgQd3XghE8CO4\\\\\\\/743OeLErh02BO0R9vYkibkb\\\\\\\/lVdl1GLp3DW5UftckD6WBzV0OxZYrkXNiOJ9qbAfQBAoDCSBoFJR01hoK6KET5GWLk8Xu9Nc6NH9OnGqsmzHvxFJEHZqoRKhLnN6XwCtqQ7ibLSG0nXZv43KXdLN3x3TOgNY8\\\\\\\/4jNln34oFnr6woV3N2i+86lNtRQBxzzlLZfYVgPKO2FkOqefLYfVrvJcJkWfQaYNLya8WzPsfDzrojnk+jzFAlIU0lMQKkjC1cfx8FTCGiOP8bdt6kIqsdOL7vYwFW3Lllwg6sI2c5doLixCjf1uONGWaMSx33E8+2vo10D95gD9EJmkflaxy4GleqLiVESz0FP52fL3Go\\\u0022,\\\u0022iv\\\u0022:\\\u0022CkXFSgYPsk+zydabb\\\\\\\/cOIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221343165763dd06b628c39c15ecc63daaaf04f152df864cac391db04b3f575502\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460163 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2475" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K67m0c34w9KF55BzaQ5BD0R2MvCGUC6dl08RT3dMIrE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2475" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U0pqe4owS7omg0H4u1nMF5dAKIJVFvCoySS48LCi6s0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2463" title="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2487" title="RNAseq Analysis Highlights Specific Transcriptome Signatures of Yeast and Mycelial Growth Phases in the Dutch Elm Disease Fungus Ophiostoma novo-ulmi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460163" id="node460163" data-pisa="ggg;5/11/2475" data-pisa-master="ggg;g3.115.020180" data-apath="/ggg/5/11/2475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sishuo Wang</span>, <span class="highwire-citation-author" data-delta="1">Youhua Chen</span>, <span class="highwire-citation-author" data-delta="2">Qinhong Cao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Huiqiang Lou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2475-2486;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020180 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460163" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sishuo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sishuo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASishuo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Renewable Resources, University of Alberta, Edmonton, T6G 2H1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYouhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qinhong Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinhong%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinhong%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huiqiang Lou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huiqiang%2BLou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuiqiang%2BLou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Huiqiang Lou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2475" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460163" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2475.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460163" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2475.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460163" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2475.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460163" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2475.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460163" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460163/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Methanogenesis and its key small-molecule methyltransferase Mtr complex are poorly understood despite their pivotal role in Earth’s global carbon cycle. Mtr complex is encoded by a conserved <em>mtrEDCBAFGH</em> operon in most methanogens. Here we report that two discrete lineages, <em>Methanococcales</em> and <em>Methanomicrobiales</em>, have a noncanonical <em>mtr</em> operon carrying two copies of <em>mtrA</em> resulting from an ancient duplication. Compared to <em>mtrA-1</em>, <em>mtrA-2</em> acquires a distinct transmembrane domain through domain shuffling and gene fusion. However, the nontransmembrane domains (MtrA domain) of <em>mtrA-1</em> and <em>mtrA-2</em> are homogenized by gene conversion events lasting throughout the long history of these extant methanogens (over 2410 million years). Furthermore, we identified a possible recruitment of ancient nonmethanogenic methyltransferase genes to establish the methanogenesis pathway. These results not only provide novel evolutionary insight into the methanogenesis pathway and methyltransferase superfamily but also suggest an unanticipated long-lasting effect of gene conversion on gene evolution in a convergent pattern.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-conversion" class="hw-term hw-article-keyword hw-article-keyword-gene-conversion" rel="nofollow">gene conversion</a></li><li class="kwd"><a href="/keyword/concerted-evolution" class="hw-term hw-article-keyword hw-article-keyword-concerted-evolution" rel="nofollow">concerted evolution</a></li><li class="kwd"><a href="/keyword/gene-fusion" class="hw-term hw-article-keyword hw-article-keyword-gene-fusion" rel="nofollow">gene fusion</a></li><li class="kwd"><a href="/keyword/methyltransferase" class="hw-term hw-article-keyword hw-article-keyword-methyltransferase" rel="nofollow">methyltransferase</a></li><li class="kwd"><a href="/keyword/methanogenesis" class="hw-term hw-article-keyword hw-article-keyword-methanogenesis" rel="nofollow">methanogenesis</a></li></ul><p id="p-3">Mtr is the key methyltransferase in the biological methanogenesis process, which plays a crucial role in Earth’s global carbon cycle and has a significant contribution to global warming (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gottschalk and Thauer 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Houghton <em>et al.</em> 2001</a>). However, our knowledge of methanogenesis and its key small molecular methyltransferase remains very limited.</p><p id="p-4">Methanogenesis is an ancient process performed exclusively by a particular group of anaerobic archaea called methanogens (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Weiss and Thauer 1993</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bapteste <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gribaldo and Brochier-Armanet 2006</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu and Whitman 2008</a>). These methanogens are capable of catalyzing the formation of methane from low-molecular organic compounds like carbon dioxide and acetic acid, which is usually the final step in the decay of organic matter (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lelieveld <em>et al.</em> 1998</a>; Houghton and Intergovernmental Panel on Climate Change Working Group I 2001). Methane (CH<sub>4</sub>) is not only the major component of clean natural gas but also a potent greenhouse gas. On the Earth, approximately 10<sup>9-10</sup> tons of methane are biologically produced through methanogenesis each year, whereas 1–10% is estimated to be released into the atmosphere (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Thauer 1998</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Welander and Metcalf 2005</a>).</p><p id="p-5">Mtr, <em>N</em><sup>5</sup>-methyltetrahydromethanopterin (CH<sub>3</sub>-H<sub>4</sub>MPT): Coenzyme M (CoM), catalyzes the second-to-last step of methanogenesis C1 pathway (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gottschalk and Thauer 2001</a>). Mtr complex is composed of eight subunits (named as MtrA-H) with no homology between them (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Harms <em>et al.</em> 1995</a>). Genes encoding the eight subunits are located in the <em>mtrEDCBAFGH</em> operon (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Harms <em>et al.</em> 1995</a>). Among the eight subunits, MtrH is thought to be responsible for the energy-consuming transfer of the methyl group from methyl-tetrahydromethanopterin (H4MPT; in some species <em>N</em>5-methyltetrahydrosarcinapterin is the methyl donor) (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Thauer 1998</a>) to the corrinoid prosthetic group harbored by MtrA (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hippler and Thauer 1999</a>), a protein anchored to cell membrane (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harms and Thauer 1996</a>). The conformational change induced in MtrA via methylation and demethylation in turn enables the sodium pumper (probably MtrE) to pump Na<sup>+</sup> extracellularly and helps establish the sodium ion gradient utilized in the ATP synthesis (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Becher <em>et al.</em> 1992</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gottschalk and Thauer 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lane and Martin 2012</a>).</p><p id="p-6">In this study, through a comprehensive phylogenetic and comparative genomic study of Mtr methyltransferase complex, we revealed an unexpected recurrent concerted evolution of <em>mtrA</em> genes, likely lasting throughout the whole history of the extant methanogens (over 2410 million years), after the convergent evolution of gene structure in three different archaeal lineages. Strikingly, the transmembrane domain of the same gene undoes significant divergent evolution very likely through domain shuffling and gene fusion. Additionally, we found an ancient nonmethanogenic origin of <em>mtrH</em> from a cobalamin-dependent methyltetrahydrofolate-homocysteine methyltransferase. Also, phylogenomic analyses detected homologs of <em>mtrA</em> and <em>mtrH</em> in nonmethanogens, which are probably involved in novel methyl-transfer pathways, implicating neo-functionalization after horizontal gene transfer (HGT) between methanogens and nonmethanogens. Our findings reveal the important roles of gene conversion and gene fusion in shaping the convergent evolution of methanogenesis genes and shed new insight into the evolution of the small-molecular methyltransferase gene superfamily.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of homologous sequences</h3><p id="p-7">Protein sequences from Uniprot-Swiss annotated as subunits of Mtr were chosen as the seed sequences in BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) search. Homologs of <em>mtr</em> genes were identified with the E-value threshold of 1e-10 and BLOSUM62 matrix in all species with complete genome sequences available in RefSeq database (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pruitt <em>et al.</em> 2007</a>). PSI-BLAST (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) was used to search for remote homologs with default parameters in bacteria. The search was iterated until convergence. For homologs of Mtr proteins with a typical length less than 100 amino acids (MtrB, MtrF, MtrG), we used a different strategy to enhance the sensitivity of the search. We used seed sequences as the bait and searched for homologs in each order of methanogenic archaea. In this case, the hit with the lowest E value was then used to search for homologs in the same order and candidates with an E value lower than 1e-5 were considered as homologs. The taxonomic information of methanogenic archaea at order level was based on the work of <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borrel <em>et al.</em> (2013)</a>, in which methanogens are classified into seven orders including six traditional orders and a recently proposed order referred to as Mx in this study for current communication (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Borrel <em>et al.</em> 2013</a>). The putative false-positive homologs were minimized with hmmscan implemented in HMMER3 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eddy 1998</a>) using the HMM profile of the corresponding domain. MeTr domain containing proteins shown in the phylogenetic tree were selected based on the list of representative species across different kingdoms of life (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Richards <em>et al.</em> 2011</a>) and clustered using Cd-hit (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Godzik 2006</a>) to filter out highly redundant sequences. Sequence processing and format conversion were performed using custom Perl and Ruby (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goto <em>et al.</em> 2010</a>) scripts.</p></div><div id="sec-3" class="subsection"><h3>Phylogenetic reconstruction</h3><p id="p-8">Sequence alignment was performed using MUSCLE v3.8.31 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar 2004</a>) with default settings and edited manually with Bioedit (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hall 1999</a>). Alignment was visualized with UGENE (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Okonechnikov <em>et al.</em> 2012</a>). Prior to phylogenetic reconstruction, jModelTest v2.1 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Darriba <em>et al.</em> 2012</a>) and ProtTest v3.2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Darriba <em>et al.</em> 2011</a>) were used to find the most proper substitution model for DNA sequence alignment and protein sequence alignment using the Akaike Information Criterion (AIC), respectively.</p><p id="p-9">Phylogenetic analyses were generated using maximum likelihood and Bayesian methods separately. The phylogenetic tree of each gene encoded by mtr operon was built based on protein sequences alignments using RAxML v7.3.9 (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stamatakis 2006</a>) with 500 pseudo replicates of bootstrap sampling. To infer gene conversion events, maximum likelihood trees were reconstructed based on DNA sequences alignments with maximum likelihood and Bayesian methods, respectively. Maximum likelihood was reconstructed using Phyml 3.0 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guindon and Gascuel 2003</a>) with 1000 pseudo replicates of bootstrap sampling Bayesian phylogenetic analyses performed with Mrbayes v3.1 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ronquist and Huelsenbeck 2003</a>). MCMC chains were run for 10 million generations with sampling every 1000 generations. The average SD of split frequencies reached less than 0.002 when chains were summarized. The chains were thought to reach convergence. The first 25% of the sampled generations were discarded as burn-in and the rest were used in the calculation of posterior probabilities and consensus tree construction. Phylogenetic trees were visualized with MEGA v5 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tamura <em>et al.</em> 2011</a>), TreeGraph v2.0 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stover and Muller 2010</a>), and Figtree v3.1 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>). For DNA sequences utilized in gene conversion detection, maximum likelihood bootstrap support values and Bayesian inference posterior probability for each node were mapped using TreeGraph (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Stover and Muller 2010</a>) and checked manually. The phylogenetic history of all methanogens is based on the work of <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Borrel <em>et al.</em> (2013)</a>, in which all of the seven orders of methanogens are included. The divergence time of species was obtained from TimeTree (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Battistuzzi and Hedges 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kumar and Hedges 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Detection of gene conversion</h3><p id="p-10">To detect gene conversion events, phylogenetic trees were built using maximum likelihood and Bayesian methods based on DNA sequences separately as described above. Bootstrap values and Bayesian posterior probability were used to evaluate the confidence level of gene conversion events. Thus, if a gene is clustered with its paralogs from the same species instead of its orthologs from different species with a high bootstrap support and Bayesian posterior probability, it could be indicative of recent duplication or gene conversion at high frequency. For likelihood test of two competing topologies of the tree, we manually generated alternative tree topologies of <em>mtrA</em> for <em>Methanomicrobiales</em> and <em>Methanococcales</em> using Mesquite v2.75 (<a href="http://mesquiteproject.org">http://mesquiteproject.org</a>). The <em>P</em>-values for different tests including Approximately Unbiased (AU) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shimodaira 2002</a>), Kishino-Hasegawa (KH) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kishino and Hasegawa 1989</a>), and Shimodaira-Hasegawa (SH) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shimodaira and Hasegawa 1999</a>) tests were calculated using CONSEL (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Shimodaira and Hasegawa 2001</a>) to indicate the likelihood of different tree topologies. Moreover, GENECONV (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sawyer 1989</a>) was also applied to detect possible gene conversion events with default parameters except for 100,000 permutations that were performed to calculate the confidence for each gene conversion event. To get results with significant statistical support, only results with <em>P</em>-value smaller than 0.05 were considered to be positive gene conversion events. Furthermore, several analyzing methods implemented in RDP4 package (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Martin <em>et al.</em> 2010</a>) were used to verify the results of gene conversion, including RDP, BOOTSCAN, CHIMAERA, MAXCHI, SISCAN, and 3SEQ. The <em>P</em>-value for the null hypothesis of no recombination events is shown for each algorithm.</p></div><div id="sec-5" class="subsection"><h3>Sequence similarity and Ka/Ks analysis</h3><p id="p-11">Pairwise sequence similarity was calculated with Water program implemented in the EMBOSS bioinformatic suite with default parameters (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rice <em>et al.</em> 2000</a>). To perform calculate KA (nonsynonymous substitution rate), the nucleotide-coding sequences were aligned to the protein alignments using PAL2NAL (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Suyama <em>et al.</em> 2006</a>). Ka was estimated using CODEML from the PAML package v4.7 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yang 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>Structural analysis</h3><p id="p-12">FFAS (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jaroszewski <em>et al.</em> 2011</a>), HHpred (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Soding <em>et al.</em> 2005</a>), and Phyre2 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kelley and Sternberg 2009</a>) were used in the fold recognition analysis to detect a proper model for homology modeling. Regions for 3D structure were chosen according to fold recognition result. The predicted model was built up by using Phyre2 based on the best reference. The refinement of the structure was done with ModRefiner (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Xu and Zhang 2011</a>). The quality of the predicted model was evaluated by two widely used programs PROQ (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wallner and Elofsson 2003</a>) and ANOLEA (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Melo <em>et al.</em> 1997</a>). Mapping of consensus sequence to the surface of the structure was performed using Chimera (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pettersen <em>et al.</em> 2004</a>). The superposition of the structure of MtrH and MeTr was performed using SuperPose (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Maiti <em>et al.</em> 2004</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-13"><a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/020180SI.pdf">Supporting Information</a> contains detailed descriptions of all supplemental files and <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS9.pdf">Figure S9</a>, along with supplemental references. Genomic context analysis of mtrH homologs are listed in <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS3.xlsx">Table S3</a>. Parameters of natural habitat of all methanogenic archaea with complete genome sequences are summarized in <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS6.xlsx">Table S6</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3><em>Methanomicrobiales</em> and <em>Methanococcales</em> carry two <em>mtrA</em> paralogous genes with different structure in their <em>mtr</em> operons</h3><p id="p-14">Mtr methyltransferase complex, as the key enzyme in methanogenesis, is encoded by a conserved <em>mtrEDCBAFGH</em> operon containing a single copy of each gene organized in the certain order in all methanogens (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>) (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Gottschalk and Thauer 2001</a>), except for the recently proposed seventh order Mx (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Paul <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Iino <em>et al.</em> 2013</a>). We identified two unconventional types of <em>mtr</em> operon carrying two copies of <em>mtrA</em> in <em>Methanomicrobiales</em> and <em>Methanococcales</em> orders by surveying all complete genomes in the NCBI RefSeq database (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, C and D</a>). One copy is the normal type, <em>mtrA-1</em>, which presents widely throughout methanogens. It is composed of an N-terminal MtrA domain and a C-terminal transmembrane domain that may be required for anchoring MtrA protein to the cell membrane (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harms and Thauer 1997</a>). Interestingly, the second copy of <em>mtrA</em> is not just a product of a simple gene duplication event. <em>mtrA-2</em> paralogous gene has a structure different from <em>mtrA-1</em>. It loses its C-terminal transmembrane domain and is located just downstream of <em>mtrA-1</em>. Moreover, <em>mtrA-2</em> became fused with downstream transmembrane genes <em>mtrF</em> and <em>mtrG</em> in <em>Methanococcales</em> and <em>Methanomicrobiales</em>, which were designated as <em>mtrA-2a</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>) and <em>mtrA-2b</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>), respectively. These results indicate that <em>mtrA-2</em> acquires a transmembrane domain distinct from <em>mtrA-1</em> through putative domain shuffling and gene fusion events.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic view of genomic context and gene structure of group I mtrA. (A) mtrEDCBAFGH operon in methanogens. (B) Methanogens usually encode one copy of a normal type of mtrA (mtrA-1) composed of an N-terminal MtrA domain labeled in teal and a C-terminal transmembrane domain in navy. (C and D) Besides mtrA-1, two orders of methanogens bear a second copy of mtrA designated as mtrA-2 within the mtr operon. mtrA-2 is the fusion type containing a MtrA domain fused at its C-terminus with a different transmembrane subunit of Mtr complex (mtrA-F fusion in Methanococcales and Methanocella arvoryzae; mtrA-G fusion in Methanomicrobiales). mtrEDCB in the upstream of mtrA are omitted for simplicity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic view of genomic context and gene structure of group I &lt;em&gt;mtrA&lt;/em&gt;. (A) &lt;em&gt;mtrEDCBAFGH&lt;/em&gt; operon in methanogens. (B) Methanogens usually encode one copy of a normal type of &lt;em&gt;mtrA&lt;/em&gt; (&lt;em&gt;mtrA-1&lt;/em&gt;) composed of an N-terminal MtrA domain labeled in teal and a C-terminal transmembrane domain in navy. (C and D) Besides &lt;em&gt;mtrA-1&lt;/em&gt;, two orders of methanogens bear a second copy of &lt;em&gt;mtrA&lt;/em&gt; designated as &lt;em&gt;mtrA-2&lt;/em&gt; within the &lt;em&gt;mtr&lt;/em&gt; operon. &lt;em&gt;mtrA-2&lt;/em&gt; is the fusion type containing a MtrA domain fused at its C-terminus with a different transmembrane subunit of Mtr complex (&lt;em&gt;mtrA-F&lt;/em&gt; fusion in &lt;em&gt;Methanococcales&lt;/em&gt; and &lt;em&gt;Methanocella arvoryzae&lt;/em&gt;; &lt;em&gt;mtrA-G&lt;/em&gt; fusion in &lt;em&gt;Methanomicrobiales&lt;/em&gt;). &lt;em&gt;mtrEDCB&lt;/em&gt; in the upstream of &lt;em&gt;mtrA&lt;/em&gt; are omitted for simplicity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F1.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2475/F1.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467064" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Schematic view of genomic context and gene structure of group I <em>mtrA</em>. (A) <em>mtrEDCBAFGH</em> operon in methanogens. (B) Methanogens usually encode one copy of a normal type of <em>mtrA</em> (<em>mtrA-1</em>) composed of an N-terminal MtrA domain labeled in teal and a C-terminal transmembrane domain in navy. (C and D) Besides <em>mtrA-1</em>, two orders of methanogens bear a second copy of <em>mtrA</em> designated as <em>mtrA-2</em> within the <em>mtr</em> operon. <em>mtrA-2</em> is the fusion type containing a MtrA domain fused at its C-terminus with a different transmembrane subunit of Mtr complex (<em>mtrA-F</em> fusion in <em>Methanococcales</em> and <em>Methanocella arvoryzae</em>; <em>mtrA-G</em> fusion in <em>Methanomicrobiales</em>). <em>mtrEDCB</em> in the upstream of <em>mtrA</em> are omitted for simplicity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To test the possible origin of the second <em>mtrA</em> copy, we constructed a phylogenetic tree of all seven methanogenic orders. <em>mtrA-2</em> only distributes in the two distantly related orders <em>Methanomicrobiales</em> and <em>Methanococcales</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/020180SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1</a>), but not in the other five orders of methanogens. Such a discrete limited distribution does not favor the possibility that <em>mtrA-2</em> already exists in the last common ancestor of all methanogens, wherein at least five independent gene loss events should have occurred (Scenario I, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1A</a>). Furthermore, <em>mtrA-2a</em> and <em>mtrA-2b</em> show slightly different characteristics in gene structure (<em>mtrA-F</em> <em>vs.</em> <em>mtrA-G</em> fusion) and location (between <em>mtrA-1</em> and <em>mtrG</em> <em>vs.</em> between <em>mtrF</em> and <em>mtrH</em>) in the operon. These facts strongly suggest a lineage-specific origin of <em>mtrA-2a</em> and <em>mtrA-2b</em> (Scenario II, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1B</a>). Moreover, all 10 <em>mtrA-2a</em> orthologs were located in the syntenic genomic context with the same gene structure in the <em>mtr</em> operon of <em>Methanococcales</em>, and so were eight <em>mtrA-2b</em> orthologs in <em>Methanomicrobiales</em>. The highly conserved operon and gene structure strongly argue that the second copy of <em>mtrA</em> might be created in the common ancestor of each genus rather than in each species via recent gene duplication, fusion, and domain shuffling events. Therefore, it is more likely that <em>mtrA-2a</em> and <em>mtrA-2b</em> genes resulted from two independent duplications of <em>mtrA-1</em> followed by gene fusion and/or domain shuffling events that occurred in the common ancestors of <em>Methanomicrobiales</em> and <em>Methanococcales</em> lineages, respectively (Scenario II, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1B</a>).</p></div><div id="sec-10" class="subsection"><h3>Long-lasting convergent concerted evolution between <em>mtrA-1/ mtrA-2</em> paralogs</h3><p id="p-17">To examine the relationship between <em>mtrA-1</em> and <em>mtrA-2</em> outparalogs, we first analyzed the sequence similarity. Surprisingly, although <em>mtrA-1</em> and <em>mtrA-2</em> paralogs in the same species have distinct transmembrane domains, they bear a highly conserved MtrA domain at both amino acid (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS2.pdf">Figure S2</a>) and nucleotide levels (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS4.pdf">Figure S4</a>). Next, we calculated the pairwise sequence similarity and substitution rate of MtrA domain of <em>mtrA</em> homologs using EMBOSS bioinformatic suite (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Unexpectedly, the average sequence identities between <em>mtrA-1</em> and <em>mtrA-2</em> paralogs from the same species are 92.4% and 99.7% for <em>Methanomicrobiales</em> and <em>Methanococcales</em>, respectively, which are significantly higher than the values between their orthologs from other species (73.7% and 78.2%) (<em>P</em>-value &lt; 0.00001) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>). Meanwhile, the MtrA domain of <em>mtrA-1/2</em> paralogs shows significantly lower nonsynonymous substitution rates (0.033 and 0.002) than their orthologs (0.171 and 0.100) in <em>Methanomicrobiales</em> and <em>Methanococcales</em>, respectively (<em>P</em>-value &lt; 0.00001) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a>). These results indicate that <em>mtrA-1</em> shares a much more similar MtrA domain with <em>mtrA-2</em> from the same species than orthologs from other species.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of sequence conservation between orthologs and paralogs of mtrA gene in Methanomicrobiales and Methanococcales. The pairwise sequence similarity (upper left) and Ka (lower right) ratio of two mtrA paralogs were calculated in Methanomicrobiales (A) and Methanococcales (B). Sequence similarity is indicated in the upper left side and is colored based on the nucleotide sequence identity. Red indicates a relatively high sequence identity and green stands for lower sequence identity. Ka ratios are shown in the lower left side and are colored based on the value of Ks from blue (higher Ka) to orange (lower Ka). Average DNA sequence similarity (C) and Ka (nonsynonymous substitution rate) (D) of orthologs and paralogs in Methanomicrobiales and Methanococcales. Abbreviations: Mfer, Methanocaldococcus fervens AG86; Mvul, Methanocaldococcus vulcanius M7; Maeo, Methanococcus aeolicus Nankai-3; Mmar, Methanococcus maripaludis S2; Mvan, Methanococcus vannielii SB; Mvol, Methanococcus voltae A3; Moki, Methanothermococcus okinawensis IH1; Mign, Methanotorris igneus Kol5; Mlab, Methanocorpusculum labreanum Z; Mbou, Methanoculleus bourgensis MS2; Mmar, Methanoculleus marisnigri JR1; Mpet, Methanoplanus petrolearius DSM 11571; Mboo, Methanoregula boonei 6A8; Mfor, Methanoregula formicicum SMSP; Mpal, Methanosphaerula palustris E1-9c; Mhun, Methanospirillum hungatei JF-1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heat map of sequence conservation between orthologs and paralogs of &lt;em&gt;mtrA&lt;/em&gt; gene in &lt;em&gt;Methanomicrobiales&lt;/em&gt; and &lt;em&gt;Methanococcales&lt;/em&gt;. The pairwise sequence similarity (upper left) and Ka (lower right) ratio of two &lt;em&gt;mtrA&lt;/em&gt; paralogs were calculated in &lt;em&gt;Methanomicrobiales&lt;/em&gt; (A) and &lt;em&gt;Methanococcales&lt;/em&gt; (B). Sequence similarity is indicated in the upper left side and is colored based on the nucleotide sequence identity. Red indicates a relatively high sequence identity and green stands for lower sequence identity. Ka ratios are shown in the lower left side and are colored based on the value of Ks from blue (higher Ka) to orange (lower Ka). Average DNA sequence similarity (C) and Ka (nonsynonymous substitution rate) (D) of orthologs and paralogs in &lt;em&gt;Methanomicrobiales&lt;/em&gt; and &lt;em&gt;Methanococcales&lt;/em&gt;. Abbreviations: Mfer, &lt;em&gt;Methanocaldococcus fervens&lt;/em&gt; AG86; Mvul, &lt;em&gt;Methanocaldococcus vulcanius&lt;/em&gt; M7; Maeo, &lt;em&gt;Methanococcus aeolicus&lt;/em&gt; Nankai-3; Mmar, &lt;em&gt;Methanococcus maripaludis&lt;/em&gt; S2; Mvan, &lt;em&gt;Methanococcus vannielii&lt;/em&gt; SB; Mvol, &lt;em&gt;Methanococcus voltae&lt;/em&gt; A3; Moki, &lt;em&gt;Methanothermococcus okinawensis&lt;/em&gt; IH1; Mign, &lt;em&gt;Methanotorris igneus&lt;/em&gt; Kol5; Mlab, &lt;em&gt;Methanocorpusculum labreanum&lt;/em&gt; Z; Mbou, &lt;em&gt;Methanoculleus bourgensis&lt;/em&gt; MS2; Mmar, &lt;em&gt;Methanoculleus marisnigri&lt;/em&gt; JR1; Mpet, &lt;em&gt;Methanoplanus petrolearius&lt;/em&gt; DSM 11571; Mboo, &lt;em&gt;Methanoregula boonei&lt;/em&gt; 6A8; Mfor, &lt;em&gt;Methanoregula formicicum&lt;/em&gt; SMSP; Mpal, &lt;em&gt;Methanosphaerula palustris&lt;/em&gt; E1-9c; Mhun, &lt;em&gt;Methanospirillum hungatei&lt;/em&gt; JF-1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F2.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2475/F2.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467033" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Heat map of sequence conservation between orthologs and paralogs of <em>mtrA</em> gene in <em>Methanomicrobiales</em> and <em>Methanococcales</em>. The pairwise sequence similarity (upper left) and Ka (lower right) ratio of two <em>mtrA</em> paralogs were calculated in <em>Methanomicrobiales</em> (A) and <em>Methanococcales</em> (B). Sequence similarity is indicated in the upper left side and is colored based on the nucleotide sequence identity. Red indicates a relatively high sequence identity and green stands for lower sequence identity. Ka ratios are shown in the lower left side and are colored based on the value of Ks from blue (higher Ka) to orange (lower Ka). Average DNA sequence similarity (C) and Ka (nonsynonymous substitution rate) (D) of orthologs and paralogs in <em>Methanomicrobiales</em> and <em>Methanococcales</em>. Abbreviations: Mfer, <em>Methanocaldococcus fervens</em> AG86; Mvul, <em>Methanocaldococcus vulcanius</em> M7; Maeo, <em>Methanococcus aeolicus</em> Nankai-3; Mmar, <em>Methanococcus maripaludis</em> S2; Mvan, <em>Methanococcus vannielii</em> SB; Mvol, <em>Methanococcus voltae</em> A3; Moki, <em>Methanothermococcus okinawensis</em> IH1; Mign, <em>Methanotorris igneus</em> Kol5; Mlab, <em>Methanocorpusculum labreanum</em> Z; Mbou, <em>Methanoculleus bourgensis</em> MS2; Mmar, <em>Methanoculleus marisnigri</em> JR1; Mpet, <em>Methanoplanus petrolearius</em> DSM 11571; Mboo, <em>Methanoregula boonei</em> 6A8; Mfor, <em>Methanoregula formicicum</em> SMSP; Mpal, <em>Methanosphaerula palustris</em> E1-9c; Mhun, <em>Methanospirillum hungatei</em> JF-1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">To further test the close relationship between <em>mtrA-1</em> and <em>mtrA-2</em> paralogs, we reconstructed the phylogenetic trees of <em>mtrA</em> in <em>Methanococcales</em> and <em>Methanomicrobiales</em> using maximum likelihood (ML) and Bayesian posterior probability methods. Both trees showed that the two paralogs of <em>mtrA</em> existing in the same species in <em>Methanomicrobiales</em> and <em>Methanococcales</em> are clustered together with bootstrap values higher than 98 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). To confirm the noncanonical topology of <em>mtrA</em> trees (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS5.pdf">Figure S5A</a>), we manually created an alternative tree topology where all <em>mtrA-1</em> orthologs group together while <em>mtrA-2</em> orthologs form a distinct clade (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS5.pdf">Figure S5B</a>). Next, we compared the likelihood of two opposite topologies. The alternative scenario was statistically rejected with extremely low <em>P</em>-values for both <em>Methanomicrobiales</em> and <em>Methanococcales</em> by all topology comparison tests implemented in CONSEL (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS5.pdf">Figure S5D</a>). These data support the unusual <em>mtrA</em> topology shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>. Together, both sequence similarity and phylogenetic topology demonstrate that <em>mtrA-1</em> and <em>mtrA-2</em> paralogs within one species are more closely related to each other than to their orthologs in other species, indicating an unusual convergent evolutionary pattern of <em>mtrA-1</em> and <em>mtrA-2</em> paralogous genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unusual phylogenetic topology of two mtrA paralogs in Methanomicrobiales and Methanococcales orders. The cladograms depict the evolutionary relationship of mtrA homologs in Methanomicrobiales (A) and Methanococcales (B). For simplicity, the normal (mtrA-1) and fusion (mtrA-2) types of mtrA were labeled as -1 and -2 following the name of species, respectively. mtrA-1 is presented in blue and mtrA-2 is in green. The phylogenetic reconstruction was performed using DNA sequences of MtrA domain. The maximum likelihood (ML) bootstrap value was obtained with 1000 replicates and the Bayesian inference (BI) posterior probability was obtained with 10,000,000 MCMC generations run. Note that there are three copies of group I mtrA in Methanococcus maripaludis X1. Two normal types of mtrA gene were labeled as mtrA-1a and mtrA-1b. The support values are shown on the branch prior to each node (ML/BI). The bootstrap value and posterior probability lower than 50 are not shown in the tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Unusual phylogenetic topology of two &lt;em&gt;mtrA&lt;/em&gt; paralogs in &lt;em&gt;Methanomicrobiales&lt;/em&gt; and &lt;em&gt;Methanococcales&lt;/em&gt; orders. The cladograms depict the evolutionary relationship of &lt;em&gt;mtrA&lt;/em&gt; homologs in &lt;em&gt;Methanomicrobiales&lt;/em&gt; (A) and &lt;em&gt;Methanococcales&lt;/em&gt; (B). For simplicity, the normal (&lt;em&gt;mtrA-1&lt;/em&gt;) and fusion (&lt;em&gt;mtrA-2&lt;/em&gt;) types of &lt;em&gt;mtrA&lt;/em&gt; were labeled as -1 and -2 following the name of species, respectively. &lt;em&gt;mtrA-1&lt;/em&gt; is presented in blue and &lt;em&gt;mtrA-2&lt;/em&gt; is in green. The phylogenetic reconstruction was performed using DNA sequences of MtrA domain. The maximum likelihood (ML) bootstrap value was obtained with 1000 replicates and the Bayesian inference (BI) posterior probability was obtained with 10,000,000 MCMC generations run. Note that there are three copies of group I &lt;em&gt;mtrA&lt;/em&gt; in &lt;em&gt;Methanococcus maripaludis X1&lt;/em&gt;. Two normal types of &lt;em&gt;mtrA&lt;/em&gt; gene were labeled as &lt;em&gt;mtrA-1a&lt;/em&gt; and &lt;em&gt;mtrA-1b&lt;/em&gt;. The support values are shown on the branch prior to each node (ML/BI). The bootstrap value and posterior probability lower than 50 are not shown in the tree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F3.medium.gif" width="338" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2475/F3.medium.gif" width="338" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467081" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Unusual phylogenetic topology of two <em>mtrA</em> paralogs in <em>Methanomicrobiales</em> and <em>Methanococcales</em> orders. The cladograms depict the evolutionary relationship of <em>mtrA</em> homologs in <em>Methanomicrobiales</em> (A) and <em>Methanococcales</em> (B). For simplicity, the normal (<em>mtrA-1</em>) and fusion (<em>mtrA-2</em>) types of <em>mtrA</em> were labeled as -1 and -2 following the name of species, respectively. <em>mtrA-1</em> is presented in blue and <em>mtrA-2</em> is in green. The phylogenetic reconstruction was performed using DNA sequences of MtrA domain. The maximum likelihood (ML) bootstrap value was obtained with 1000 replicates and the Bayesian inference (BI) posterior probability was obtained with 10,000,000 MCMC generations run. Note that there are three copies of group I <em>mtrA</em> in <em>Methanococcus maripaludis X1</em>. Two normal types of <em>mtrA</em> gene were labeled as <em>mtrA-1a</em> and <em>mtrA-1b</em>. The support values are shown on the branch prior to each node (ML/BI). The bootstrap value and posterior probability lower than 50 are not shown in the tree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The convergent evolution pattern of <em>mtrA-1/2</em> paralogs could be caused by two evolutionary mechanisms, concerted evolution (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) or recent gene duplications (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The latter mechanism is much less likely because of the highly conserved syntenic genomic context and same gene structure of <em>mtrA-2a</em> and <em>mtrA-2b</em> in each lineage. <em>mtrA-2</em> was created via <em>mtrA-1</em> duplication, gene fusion, and domain shuffling events preceding the speciation of <em>Methanomicrobiales</em> and <em>Methanococcales</em> genera (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS1.pdf">Figure S1B</a>, Scenario II). Therefore, the higher sequence similarity and phylogenies of <em>mtrA-1</em> and <em>mtrA-2</em> point to concerted evolution accomplished by gene conversion instead of recent duplication in each species (Scenario A, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two different evolutionary models regarding the observed phylogeny of mtrA. For clarity, only three species are shown. (A) mtrA-2 is generated via an ancient gene duplication and domain shuffling event in the ancestor followed by frequent gene conversion (indicated as green arrow) in different lineages. (B) mtrA-2 is generated via gene duplication and domain shuffling in different lineages multiple times. The star in yellow denotes the formation of mtrA-2 through the duplication of mtrA-1. The stars in orange and light yellow denote the fusion between mtrA-2 and mtr-F/G and the loss of the C-terminal transmembrane domain, respectively. Each branch in gray represents a speciation event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two different evolutionary models regarding the observed phylogeny of &lt;em&gt;mtrA&lt;/em&gt;. For clarity, only three species are shown. (A) &lt;em&gt;mtrA-2&lt;/em&gt; is generated via an ancient gene duplication and domain shuffling event in the ancestor followed by frequent gene conversion (indicated as green arrow) in different lineages. (B) &lt;em&gt;mtrA-2&lt;/em&gt; is generated via gene duplication and domain shuffling in different lineages multiple times. The star in yellow denotes the formation of &lt;em&gt;mtrA-2&lt;/em&gt; through the duplication of &lt;em&gt;mtrA-1&lt;/em&gt;. The stars in orange and light yellow denote the fusion between &lt;em&gt;mtrA-2&lt;/em&gt; and &lt;em&gt;mtr-F/G&lt;/em&gt; and the loss of the C-terminal transmembrane domain, respectively. Each branch in gray represents a speciation event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F4.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2475/F4.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466940" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-22" class="first-child">Two different evolutionary models regarding the observed phylogeny of <em>mtrA</em>. For clarity, only three species are shown. (A) <em>mtrA-2</em> is generated via an ancient gene duplication and domain shuffling event in the ancestor followed by frequent gene conversion (indicated as green arrow) in different lineages. (B) <em>mtrA-2</em> is generated via gene duplication and domain shuffling in different lineages multiple times. The star in yellow denotes the formation of <em>mtrA-2</em> through the duplication of <em>mtrA-1</em>. The stars in orange and light yellow denote the fusion between <em>mtrA-2</em> and <em>mtr-F/G</em> and the loss of the C-terminal transmembrane domain, respectively. Each branch in gray represents a speciation event.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To confirm gene conversion events between <em>mtrA-1</em> and <em>mtrA-2</em>, we performed GENECONV analysis and obtained the statistical support for each event by 100,000 permutations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mansai and Innan 2010</a>). Only those with a <em>P</em>-value lower than 0.05 were considered to be positive conversion events. Gene conversion events were detected for all pairs of <em>mtrA-1/2</em> paralogs within the same species as identified in the phylogenetic trees (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS1.pdf">Table S1</a>). Moreover, these gene conversion events were further validated by several other computational tools implemented in RDP package as well (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS2.pdf">Table S2</a>). These data indicate that the high sequence similarity and close relationship between <em>mtrA-1</em> and <em>mtrA-2</em> paralogs are maintained by recurrent gene conversion events, favoring the concerted evolution model (Scenario A, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>).</p></div><div id="sec-11" class="subsection"><h3>Two novel subfamilies of <em>mtrA</em> gene with potential distinct functions</h3><p id="p-24">To examine the evolutionary scenario of <em>mtrA-1</em> and <em>mtrA-2</em> in a broader scope, phylogenetic analysis was expanded to all homologs of <em>mtrA</em> identified in all complete genomes. The tree obtained by comprehensive phylogenetic analysis clearly revealed four groups of <em>mtrA</em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Among them, two new groups, as defined as group III and group IV, have not been identified previously.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A more comprehensive phylogenetic analysis reveals new groups of mtrA gene. The maximum likelihood tree of mtrA was constructed with RAxML. The substitution model was determined by ProtTest prior to the phylogenetic reconstruction. Homologs of mtrA were classified into four groups (subfamilies). Group I, group II, group III, and group IV of mtrA are colored blue, green, pink, and light blue, respectively. Group I is the dominant type found in most methanogens with a C-terminal transmembrane domain. Homologs in groups II and III are only found in Methanosarcinales and/or Methanocellales. Group IV mtrA is only distributed in nonmethanogens. The number on the scale bar indicates the number of substitutions per amino acid site. The number next to each node indicates the bootstrap value. Methanocella arvoryzae, which has two group I mtrA paralogs, was marked in red square." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A more comprehensive phylogenetic analysis reveals new groups of &lt;em&gt;mtrA&lt;/em&gt; gene. The maximum likelihood tree of &lt;em&gt;mtrA&lt;/em&gt; was constructed with RAxML. The substitution model was determined by ProtTest prior to the phylogenetic reconstruction. Homologs of &lt;em&gt;mtrA&lt;/em&gt; were classified into four groups (subfamilies). Group I, group II, group III, and group IV of &lt;em&gt;mtrA&lt;/em&gt; are colored blue, green, pink, and light blue, respectively. Group I is the dominant type found in most methanogens with a C-terminal transmembrane domain. Homologs in groups II and III are only found in &lt;em&gt;Methanosarcinales&lt;/em&gt; and/or Methanocellales. Group IV &lt;em&gt;mtrA&lt;/em&gt; is only distributed in nonmethanogens. The number on the scale bar indicates the number of substitutions per amino acid site. The number next to each node indicates the bootstrap value. &lt;em&gt;Methanocella arvoryzae&lt;/em&gt;, which has two group I &lt;em&gt;mtrA&lt;/em&gt; paralogs, was marked in red square.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F5.medium.gif" width="304" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2475/F5.medium.gif" width="304" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467126" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">A more comprehensive phylogenetic analysis reveals new groups of <em>mtrA</em> gene. The maximum likelihood tree of <em>mtrA</em> was constructed with RAxML. The substitution model was determined by ProtTest prior to the phylogenetic reconstruction. Homologs of <em>mtrA</em> were classified into four groups (subfamilies). Group I, group II, group III, and group IV of <em>mtrA</em> are colored blue, green, pink, and light blue, respectively. Group I is the dominant type found in most methanogens with a C-terminal transmembrane domain. Homologs in groups II and III are only found in <em>Methanosarcinales</em> and/or Methanocellales. Group IV <em>mtrA</em> is only distributed in nonmethanogens. The number on the scale bar indicates the number of substitutions per amino acid site. The number next to each node indicates the bootstrap value. <em>Methanocella arvoryzae</em>, which has two group I <em>mtrA</em> paralogs, was marked in red square.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The biggest clade composed by the majority of <em>mtrA</em> genes was defined as group I. All group I <em>mtrA</em> genes are located within the <em>mtr</em> operon, which exists in all methanogens except the Mx order (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Borrel <em>et al.</em> 2013</a>). <em>Methanomicrobiales</em> and <em>Methanococcales</em> species carried two copies of group I <em>mtrA</em>, <em>mtrA-1</em>, and <em>mtrA-2</em> as mentioned above. Again, these two paralogs clustered together, which recapitulates the noncanonical topology in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>.</p><p id="p-27">Interestingly, the other three groups of <em>mtrA</em> lost the C-terminal transmembrane domain in group I counterparts. Furthermore, according to the phylogenetic tree, all these <em>mtrA</em> genes appeared with long branches apart from group I homologs. Group II <em>mtrA</em> was only distributed in Methanosarcinales. In addition, it forms an operon together with a conserved methanogenesis gene <em>mtxX</em> and a <em>mtrH</em> homolog <em>mtxH</em>. It has been proposed that together with <em>mtxX</em>, group II <em>mtrA</em> and/or <em>mtxH</em> might participate in the methyltransfer from trimethylamine or methanol to tetrahydromethanopterin, a methanogenesis pathway unique to Methanosarcinales (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Harms and Thauer 1997</a>).Group III <em>mtrA</em> exists in Methanosarcinales and Methanocellales lineages and lacks apparent conserved genomic context. We are able to find some <em>mtrA</em> homologs sporadically distributed in nonmethanogenic bacteria and archaea, composing group IV <em>mtrA</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The patchy distribution pattern indicates that these organisms may obtain <em>mtrA</em> homologs by horizontal gene transfer (HGT). These data show that groups II, III, and IV <em>mtrA</em> distinguish themselves from group I homologs in many aspects, including sequence similarity, gene structure, and genomic context. This suggests that in contrast to the concerted evolution of <em>mtrA-1</em> and <em>mtrA-2</em> in <em>Methanomicrobiales</em> and <em>Methanococcales</em>, other groups of <em>mtrA</em> undergo sequence and potential functional diversification.</p></div><div id="sec-12" class="subsection"><h3>Another putative convergent concerted evolution example in <em>Methanocella arvoryzae</em></h3><p id="p-28">In the comprehensive phylogenetic tree described above, we noticed that <em>Methanocella arvoryzae</em> is the only organism outside the <em>Methanomicrobiales</em> and <em>Methanococcales</em> orders encoding two copies of group I <em>mtrA</em> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, red square). Like <em>Methanococcales</em>, <em>Methanocella arvoryzae</em> has a <em>mtrA-F</em> fusion type (<em>mtrA-2a</em>) downstream of <em>mtrA-1</em> in the <em>mtr</em> operon. More interestingly, the two <em>mtrA</em> paralogs in <em>Methanocella arvoryzae</em> showed high sequence similarity and were clustered into the same clade in the ML tree, resembling the convergent pattern as <em>mtrA-1</em> and <em>mtrA-2</em> paralogs in <em>Methanococcales</em> and <em>Methanomicrobiales</em>.</p><p id="p-29">One possibility is that it is due to a recent duplication of <em>mtrA</em> followed by the loss of C-terminal transmembrane domain and fusion with <em>mtrF</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS6.pdf">Figure S6A</a>). However, it is worth noticing that group III <em>mtrA</em> is missing in this organism while it exists in all other species from the same order. This raised another possibility that group III <em>mtrA</em> might be fused with <em>mtrF</em> after a translocation event and got converted by <em>mtrA-1</em> later (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS6.pdf">Figure S6B</a>). This scenario couples the loss of group III <em>mtrA</em> and the high similarity between two <em>mtrA</em> paralogs and avoids assuming the duplication of <em>mtrA-1</em>, the loss of the transmembrane domain of <em>mtrA-1</em>, and the loss of group III <em>mtrA</em>. Thus, this scenario is more parsimonious and likely represents the third example of independent origin of the chimeric <em>mtrA-F</em> gene through concerted evolution.</p></div><div id="sec-13" class="subsection"><h3>An ancient origin of MtrH from the MeTr domain of B12-dependent methionine synthase (MetH)</h3><p id="p-30">In Mtr complex, MtrH is the methyltransferase responsible for the transfer of methyl group between CH<sub>3</sub>-H<sub>4</sub>MPT and corrinoid prosthetic group harbored by MtrA (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hippler and Thauer 1999</a>). The prevalence and crucial role of MtrH among methanogens aroused our interest to pursue a clearer evolutionary map of <em>mtrH</em> in a context of whole methyltransferase superfamily. Surprisingly, a PSI-BLAST search against the NCBI RefSeq database hit numerous methyltransferases with sequence identity of 30% with <em>mtrH</em> in the second round of iteration. The PSI-BLAST was not stopped until no more hits were detected at the e-value cutoff of 1e-3. Vitamin B12-dependent methionine synthase (MetH) and 5-methyltetrahydrofolate–homocysteine methyltransferase appeared to be mostly enriched among all of the hits. The amino acid sequence alignment of MtrH and MeTr revealed a conserved region limited to the pterin ligand binding MeTr domain of MetH and 5-methyltetrahydrofolate–homocysteine methyltransferase (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS7.pdf">Figure S7</a>). To further illustrate the relationship between MtrH and MeTr from a structural perspective, fold recognition search was performed using MtrH from <em>Methanococcus jannaschii</em>, the first sequenced archaeon. In agreement with amino acid sequence similarity, all three fold recognition analyses (FFAS, HHpred, and Phyre2) consistently reported a region with similarity to the MeTr domain in B12-dependent methionine synthase (MetH) and dihydropteroate synthetase (DHPS), thus pointing to a remote homologous relationship between MtrH and MeTr domain.</p><p id="p-31">In addition, we built a homology modeling of the MtrH via Phyre2. The N-terminal region (a.a.11-299) was used in 3D modeling because it shows relatively high similarity in both PSI-BLAST and folding recognition search. The predicted MtrH tertiary structure was supported by both computational programs PROQ (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Wallner and Elofsson 2003</a>) and ANOLEA (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Melo <em>et al.</em> 1997</a>). MtrH presented a TIM-barrel structure, which closely resembles the structural feather of MeTr domain (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). A more detailed comparison between the MtrH and MeTr domain at the 3D level revealed a highly conserved region (colored red in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>) corresponding to the ligand binding motifs of MeTr domain, indicating the potential conserved functions on ligand binding between MtrH and MeTr.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted 3D structure of MtrH of Methanococcus jannaschii. (A) The superimposition of MtrH (green) and MeTr (purple). The amino acids at the position of 50-271 of the best-aligned protein (PDB ID: 3K13) are chosen in structural comparison. The structural model of MtrH shows a TIM-barrel fold. (B) The structure of MtrH is colored based on the evolutionary conservation information. The red represents the most conserved residues and the blue indicates the least conserved residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predicted 3D structure of MtrH of &lt;em&gt;Methanococcus jannaschii&lt;/em&gt;. (A) The superimposition of MtrH (green) and MeTr (purple). The amino acids at the position of 50-271 of the best-aligned protein (PDB ID: 3K13) are chosen in structural comparison. The structural model of MtrH shows a TIM-barrel fold. (B) The structure of MtrH is colored based on the evolutionary conservation information. The red represents the most conserved residues and the blue indicates the least conserved residues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F6.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/11/2475/F6.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466997" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Predicted 3D structure of MtrH of <em>Methanococcus jannaschii</em>. (A) The superimposition of MtrH (green) and MeTr (purple). The amino acids at the position of 50-271 of the best-aligned protein (PDB ID: 3K13) are chosen in structural comparison. The structural model of MtrH shows a TIM-barrel fold. (B) The structure of MtrH is colored based on the evolutionary conservation information. The red represents the most conserved residues and the blue indicates the least conserved residues.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Furthermore, we constructed an ML tree of MtrH/MeTr domain-containing proteins. The whole MtrH group was clustered together with groups of MetH and 5-methyltetrahydrofolate–homocysteine methyltransferase (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS8.pdf">Figure S8</a>). Both MetH and 5-methyltetrahydrofolate–homocysteine methyltransferase are present in numerous organisms from different kingdoms, implying a common ancient origin of MtrH and MeTr domain-containing proteins.</p><p id="p-34">Taken together, these data suggest an ancient nonmethanogenic origin of <em>mtrH</em>, which means that MeTr domain of B12-dependent methionine synthase and 5-methyltetrahydrofolate–homocysteine methyltransferase might evolve to become MtrH and be utilized by methanogens for methanogenesis.</p></div><div id="sec-14" class="subsection"><h3>Massive bidirectional interdomain HGTs of <em>mtrH</em></h3><p id="p-35">If we took a closer view of MtrH branch, it comprises three subgroups (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Subgroup I <em>mtrH</em> is located in <em>mtr</em> operon involved in C1 methanogenesis pathway. Subgroup II forms an operon with group II <em>mtrA</em> and might participate in other methanogenesis pathways (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Harms and Thauer 1997</a>). <em>mtrH</em> homologs are also found in several distantly related nonmethanogens, which are named subgroup III. They form a separate clade in the phylogenetic tree (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). The topology of the tree and patchy distribution of subgroup III <em>mtrH</em> among very distantly related organisms tends to suggest that some nonmethanogens might acquire <em>mtrH</em> homologs from methanogenic archaea and that massive HGT events occurred among nonmethanogens.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Massive horizontal gene transfer between methanogens and nonmethanogens in the evolution of mtrH. The maximum likelihood tree of mtrH was constructed with RAxML. ProtTest was utilized to estimate the substitution model used in the phylogenetic reconstruction. mtrH homologs correspond to three subgroups. Subgroups I, II, and III mtrH homologs are colored deep blue, green, and light blue, respectively. The majority is subgroup I, which is located in mtrEDCBAFGH operon. Subgroup II mtrH is located in mtxXAH operon, with group II mtrA only distributed in Methanosarcinales. Subgroup III contains all mtrH homologs from nonmethanogens, and two homologs from methanogens also belong to this group (in the red square)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-223523816" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Massive horizontal gene transfer between methanogens and nonmethanogens in the evolution of &lt;em&gt;mtrH&lt;/em&gt;. The maximum likelihood tree of &lt;em&gt;mtrH&lt;/em&gt; was constructed with RAxML. ProtTest was utilized to estimate the substitution model used in the phylogenetic reconstruction. &lt;em&gt;mtrH&lt;/em&gt; homologs correspond to three subgroups. Subgroups I, II, and III &lt;em&gt;mtrH&lt;/em&gt; homologs are colored deep blue, green, and light blue, respectively. The majority is subgroup I, which is located in &lt;em&gt;mtrEDCBAFGH&lt;/em&gt; operon. Subgroup II &lt;em&gt;mtrH&lt;/em&gt; is located in &lt;em&gt;mtxXAH&lt;/em&gt; operon, with group II &lt;em&gt;mtrA&lt;/em&gt; only distributed in &lt;em&gt;Methanosarcinales&lt;/em&gt;. Subgroup III contains all &lt;em&gt;mtrH&lt;/em&gt; homologs from nonmethanogens, and two homologs from methanogens also belong to this group (in the red square).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2475/F7.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/11/2475/F7.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2475/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467114" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-36" class="first-child">Massive horizontal gene transfer between methanogens and nonmethanogens in the evolution of <em>mtrH</em>. The maximum likelihood tree of <em>mtrH</em> was constructed with RAxML. ProtTest was utilized to estimate the substitution model used in the phylogenetic reconstruction. <em>mtrH</em> homologs correspond to three subgroups. Subgroups I, II, and III <em>mtrH</em> homologs are colored deep blue, green, and light blue, respectively. The majority is subgroup I, which is located in <em>mtrEDCBAFGH</em> operon. Subgroup II <em>mtrH</em> is located in <em>mtxXAH</em> operon, with group II <em>mtrA</em> only distributed in <em>Methanosarcinales</em>. Subgroup III contains all <em>mtrH</em> homologs from nonmethanogens, and two homologs from methanogens also belong to this group (in the red square).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Interestingly, <em>mtrH</em> in nonmethanogens show a conserved genomic context with other methyltransfer-related genes (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS3.xlsx">Table S3</a>). The operons include homologs of <em>mta</em>, which are thought to catalyze methanogenesis from methanol. This implicates functional constraints on the structure of operons containing <em>mtrH</em> homologs in nonmethanogens. It would be postulated that <em>mtrH</em> are likely neo-functionalized and involved in novel methyltransfer processes after HGT in many nonmethanogenic lineages. More interestingly, several prokaryotic species that carry <em>mtrH</em> homologs are able to utilize chloromethane as carbon and energy source (highlighted in <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS3.xlsx">Table S3</a>). Considering the functional links between <em>mtrH</em> and its neighbor genes in bacteria (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gao and Gupta 2007</a>), it is likely that these bacteria acquired <em>mtrH</em> homologs via HGT to help reduce methyl halides.</p><p id="p-38">Finally, it is worth noting that two <em>mtrH</em> homologs from methanogens, <em>Methanoplanus petrolearius DSM 11571</em> and <em>Methanoregula boonei 6A8</em>, group with <em>mtrH</em> homologs from nonmethanogens supported by bootstrap values of 94 and 79 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Additionally, the genomic context is conserved among these species as well (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS3.xlsx">Table S3</a>). This may represent a rare case of reverse horizontal transfer of <em>mtrH</em> homologs from nonmethanogens to methanogens. Taken together, the phylogeny of <em>mtrH</em> suggests massive bidirectional gene transfer between methanogens and nonmethanogens and within nonmethanogens.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-39">Gene conversion, the process of nonreciprocal exchange between two DNA segments (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liao 1999</a>), can cause the concerted evolution of different sequences, thereby leading to functional conservation or redundancy of duplicated genes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nei and Rooney 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Innan and Kondrashov 2010</a>). In this study, through comprehensive phylogenetic and rigorous computational analyses, we found an exceptional long-lasting recurrent gene conversion mechanism that constrains the divergence of <em>mtrA-1</em> and <em>mtrA-2</em> paralogs with different gene structure in three distinct methanogenic archaeal lineages in parallel. Such a unique evolutionary process may be important for <em>mtrA</em> gene function and may be driven by adaptation to survive in a methanogenesis manner under some particular environment.</p><div id="sec-16" class="subsection"><h3><em>mtrA-1</em>/<em>mtrA-2</em> paralogs represents a unique example of convergent concerted evolution</h3><p id="p-40">In bacteria, concerted evolution of paralogs should occur through homologous recombination, which is usually considered to have a similar outcome of eukaryote’s gene conversion. Interestingly, only the highly conserved MtrA domain in <em>mtrA-2</em> is overwritten by copying the counterpart sequence from <em>mtrA-1</em>. MtrA-2 obtained a transmembrane domain distinct from MtrA-1 through domain shuffling and gene fusion, thus leading to a divergent pattern with MtrA-1 paralog in transmembrane domain. As far as we know, the evolutionary scenario of MtrA-2 reported here represents the first example that convergent and divergent evolution occur in the two distinct domains of the same gene through distinct mechanisms, respectively.</p><p id="p-41">Although <em>mtrA-1</em> and <em>mtrA-2</em> paralogs in the same species have distinct transmembrane domains, they bear a significantly more similar MtrA domain than their orthologs in different species at both levels of nucleic acid and amino acid sequence. These data point to a unique convergent evolution within a certain domain of two paralogous genes. However, the gene structure and genomic/operon context of <em>mtrA-2</em> are highly conserved at the genus level, indicating a convergent pattern of <em>mtrA-2</em> evolution in each lineage as well. Such a convergent pattern is ascribed to a concerted evolution (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>) rather than recent gene duplication (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). It is worth mentioning that although a recent duplication model is not favored by the same gene structure and highly conserved genomic/operon context at the genus level, we cannot completely rule out this possibility. However, if the high similarity between <em>mtrA-1</em>/<em>2</em> paralogs were due to recent duplication, then first we would have to assume at least 10 and eight independent duplication gene fusion domain shuffling events in <em>Methanococcales</em> and <em>Methanomicrobiales</em>, respectively. Second, we would also have to assume such a complicated process should have occurred constantly in each species belonging to the same genus.</p></div><div id="sec-17" class="subsection"><h3>Exceptional long-lasting gene conversion events underlie <em>mtrA-1</em>/<em>mtrA-2</em> concerted evolution</h3><p id="p-42">Gene conversion events should predate the speciation events in the orders <em>Methanomicrobiales</em> and <em>Methanococcales</em> because concerted pattern and gene conversion were detected in almost all organisms in these two orders. Moreover, given the fact that <em>mtrA-1</em> and <em>mtrA-2</em> share significantly high similarity in their MtrA domain at the nucleotide sequence level, it is very likely that concerted evolution is still an ongoing process in extant methanogens. According to the archaeal TimeTree (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Battistuzzi and Hedges 2009</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kumar and Hedges 2011</a>), gene conversion events between <em>mtrA-1</em> and <em>mtrA-2</em> can thus be estimated to last for more than 2410 Myr (million years) in <em>Methanomicrobiales</em> and 2216 Myr in <em>Methanococcales</em> species (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/FigureS9.pdf">Figure S9</a>). This case of gene conversion, to our knowledge, represents one of the most long-lasting concerted evolution events that have been reported so far (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS5.pdf">Table S5</a>). It is more intriguing given that it happens independently in two distinct lineages and that both <em>mtrA</em> paralogs are essential to the cell growth and/or methanogenesis (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sarmiento <em>et al.</em> 2013</a>). The MtrA domain is essential for most methanogens to thrive through an extremely ancient and highly conserved methanogenesis reaction. Therefore, we suggest that it is most likely that gene conversion may contribute to the removal of deleterious mutations and the spread of beneficial mutations (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mano and Innan 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fawcett and Innan 2011</a>). The recurrent evolution of gene conversion becomes a good strategy to maintain a highly conserved MtrA domain in two <em>mtrA</em> paralogs. This might be particularly important for archaea where recombination through sexual reproduction is absent (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breuert <em>et al.</em> 2006</a>). Also, it is known that for genes involved in protein complexes like Mtr, the existence of divergent paralogs might introduce problems as other subunits may have to adapt to different copies of the same gene (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baker <em>et al.</em> 2013</a>). The homogenization of paralogous sequences via gene conversion may thus help to avoid this problem in the Mtr complex with multiple protein–protein interactions.</p><p id="p-43">However, the C-terminal transmembrane domain only exists in group I <em>mtrA</em> belonging to Mtr complex, but not other groups of <em>mtrA</em>. This suggests that the transmembrane domain should be indispensable to Mtr complex, and thus methanogenesis capability. One possibility is that the transmembrane domain might be involved in sensing different ligands or other environmental cues important for methanogenesis. Among species with habitat data currently available, most organisms where gene conversion of <em>mtrA</em> occurred were found to be mesophiles (<a href="http://www.g3journal.org/content/suppl/2015/09/16/g3.115.020180.DC1/TableS6.xlsx">Table S6</a>). However, not all mesophilic methanogens carry <em>mtrA-2</em>, implying that there could be some unknown environmental factor(s) contributing to such extremely long-lasting concerted evolution and determining the distinct transmembrane domains of <em>mtrA-1/2</em>.</p><p id="p-44">In summary, we suggest that the unusual evolutionary pattern of MtrA paralogs represents a novel example that gene conversion may help unsexual organisms escape from “Muller ratchet” (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Muller 1964</a>) by allowing for more efficient spread of beneficial mutations and removal of deleterious mutations after the convergent domain shuffling in different lineages.</p></div><div id="sec-18" class="subsection"><h3>The nonmethanogenic origin of MtrH From MeTr</h3><p id="p-45">Another interesting finding is that the methyltransferase MtrH required for methanogenesis may be traced back to a nonmethanogenic origin. The homologous relationship between MtrH and MeTr observed in this study provides a case of the transfer of nonmethanogenic genes to methanogenesis genes and indicates the utilization of preexisting genes in the genome in the establishment of the methanogenesis pathway. This is in agreement with previous findings that two enzymes involved in the acetoclastic methanogenesis pathway may originate from <em>Firmicutes</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fournier and Gogarten 2008</a>). All these data provide evidence to support the recent hypothesis that the origin of some methanogenesis genes could be dated back to the last common ancestor of archaea (LCA) based on an extensive genomic analysis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chistoserdova 2013</a>).</p></div></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We thank Drs. Jan Janouškovec, Ence Yang, and Yinhua Huang for thoughtful discussion and critical reading of the manuscript. We thank Fabien Burki for sharing the script used to retrieve taxonomic information. We also thank Ce Shi, Yunfei Wang, and Yong Zhang for providing suggestive information. This work was supported by National Natural Science Foundation of China (31271331 to H.L.; 31200052 to Q.C.; J1103520 to S.W.), Program for New Century Excellent Talents in University of China (NCET-09-0733 to H.L.), and UBC University International Graduate Fellowship and UBC Four-Year Doctoral Fellowship (to S.W. and Y.C.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-47">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020180/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020180/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-48"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-06"><span class="received-label">Received </span>July 6, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-14"><span class="accepted-label">Accepted </span>September 14, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.2" data-doi="10.1126/science.1240810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson-Smith</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Following gene duplication, paralog interference constrains transcriptional circuit evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBaker%26rft.auinit1%253DC.%2BR.%26rft.volume%253D342%26rft.issue%253D6154%26rft.spage%253D104%26rft.epage%253D108%26rft.atitle%253DFollowing%2BGene%2BDuplication%252C%2BParalog%2BInterference%2BConstrains%2BTranscriptional%2BCircuit%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1240810%26rft_id%253Dinfo%253Apmid%252F24092741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1NC8xMDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.3" data-doi="10.1155/2005/859728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bapteste</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Higher-level classification of the Archaea: evolution of methanogenesis and methanogens.</span> <abbr class="cit-jnl-abbrev">Archaea</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DArchaea%26rft.aulast%253DBapteste%26rft.auinit1%253DE.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D353%26rft.epage%253D363%26rft.atitle%253DHigher-level%2Bclassification%2Bof%2Bthe%2BArchaea%253A%2Bevolution%2Bof%2Bmethanogenesis%2Band%2Bmethanogens.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2005%252F859728%26rft_id%253Dinfo%253Apmid%252F15876569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2005/859728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15876569&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2475.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Battistuzzi</span>  <span class="cit-name-given-names">U. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Archaebacteria</span>, <span class="cit-source">The Timetree of Life</span>, edited by B. <span class="cit-ed"><span class="cit-name-surname">Hedges</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.5" data-doi="10.1128/jb.174.23.7656-7660.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschalk</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">N-5-methyl-tetrahydromethanopterin–coenzyme-M methyltransferase of methanosarcina strain Go1 is an Na+-translocating membrane-protein.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">7656</span>–<span class="cit-lpage">7660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DBecher%26rft.auinit1%253DB.%26rft.volume%253D174%26rft.issue%253D23%26rft.spage%253D7656%26rft.epage%253D7660%26rft.atitle%253DN5-methyl-tetrahydromethanopterin%253Acoenzyme%2BM%2Bmethyltransferase%2Bof%2BMethanosarcina%2Bstrain%2BGo1%2Bis%2Ban%2BNa%2528%252B%2529-translocating%2Bmembrane%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.23.7656-7660.1992%26rft_id%253Dinfo%253Apmid%252F1447136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NC8yMy83NjU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.6" data-doi="10.1093/gbe/evt128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borrel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peyret</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brugere</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogenomic data support a seventh order of methylotrophic methanogens and provide insights into the evolution of methanogenesis.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1769</span>–<span class="cit-lpage">1780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt128%26rft_id%253Dinfo%253Apmid%252F23985970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTc2OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.7" data-doi="10.1371/journal.pone.0000092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breuert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spohn</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Regulated polyploidy in halophilic archaea.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBreuert%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.spage%253De92%26rft.epage%253De92%26rft.atitle%253DRegulated%2Bpolyploidy%2Bin%2Bhalophilic%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000092%26rft_id%253Dinfo%253Apmid%252F17183724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17183724&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2475.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chistoserdova</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The distribution and evolution of C1 transfer enzymes and evolution of the planctomycetes</span>, pp. <span class="cit-fpage">195</span>–<span class="cit-lpage">209</span> in <span class="cit-source"><em>Planctomycetes: Cell Structure</em>, <em>Origins and Biology</em></span>, edited by J. A. <span class="cit-ed"><span class="cit-name-surname">Fuerst</span></span>. <span class="cit-publ-name">Springer</span>, New York.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.9" data-doi="10.1093/bioinformatics/btr088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ProtTest 3: fast selection of best-fit models of protein evolution.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr088%26rft_id%253Dinfo%253Apmid%252F21335321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy84LzExNjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.10" data-doi="10.1038/nmeth.2109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">jModelTest 2: more models, new heuristics and parallel computing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPatel%26rft.auinit1%253DV.%2BR.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D772%26rft.epage%253D772%26rft.atitle%253DjModelTest%2B2%253A%2Bmore%2Bmodels%252C%2Bnew%2Bheuristics%2Band%2Bparallel%2Bcomputing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2109%26rft_id%253Dinfo%253Apmid%252F22847109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847109&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.11" data-doi="10.1093/bioinformatics/14.9.755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Profile hidden Markov models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D755%26rft.epage%253D763%26rft.atitle%253DProfile%2Bhidden%2BMarkov%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.9.755%26rft_id%253Dinfo%253Apmid%252F9918945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNC85Lzc1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.12" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.13" data-doi="10.3390/genes2010191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Neutral and non-neutral evolution of duplicated genes with gene conversion.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D2%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes2010191%26rft_id%253Dinfo%253Apmid%252F24710144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes2010191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24710144&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.14" data-doi="10.1128/JB.01382-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of acetoclastic methanogenesis in Methanosarcina via horizontal gene transfer from cellulolytic Clostridia.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01382-07%26rft_id%253Dinfo%253Apmid%252F18055595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5MC8zLzExMjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.15" data-doi="10.1186/1471-2164-8-86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phylogenomic analysis of proteins that are distinctive of Archaea and its main subgroups and the origin of methanogenesis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGao%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.spage%253D86%26rft.epage%253D86%26rft.atitle%253DPhylogenomic%2Banalysis%2Bof%2Bproteins%2Bthat%2Bare%2Bdistinctive%2Bof%2BArchaea%2Band%2Bits%2Bmain%2Bsubgroups%2Band%2Bthe%2Borigin%2Bof%2Bmethanogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-86%26rft_id%253Dinfo%253Apmid%252F17394648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17394648&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.16" data-doi="10.1093/bioinformatics/btq475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BioRuby: bioinformatics software for the Ruby programming language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2617</span>–<span class="cit-lpage">2619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq475%26rft_id%253Dinfo%253Apmid%252F20739307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjAvMjYxNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.17" data-doi="10.1016/S0005-2728(00)00274-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gottschalk</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The Na+-translocating methyltransferase complex from methanogenic archaea.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta.</abbr> <span class="cit-vol">1505</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DGottschalk%26rft.auinit1%253DG.%26rft.volume%253D1505%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D36%26rft.atitle%253DThe%2BNa%2528%252B%2529-translocating%2Bmethyltransferase%2Bcomplex%2Bfrom%2Bmethanogenic%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0005-2728%252800%252900274-7%26rft_id%253Dinfo%253Apmid%252F11248186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0005-2728(00)00274-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11248186&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167644600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.18" data-doi="10.1098/rstb.2006.1841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gribaldo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochier-Armanet</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The origin and evolution of Archaea: a state of the art.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">361</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DGribaldo%26rft.auinit1%253DS.%26rft.volume%253D361%26rft.issue%253D1470%26rft.spage%253D1007%26rft.epage%253D1022%26rft.atitle%253DThe%2Borigin%2Band%2Bevolution%2Bof%2BArchaea%253A%2Ba%2Bstate%2Bof%2Bthe%2Bart%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2006.1841%26rft_id%253Dinfo%253Apmid%252F16754611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjEvMTQ3MC8xMDA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.19" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUyLzUvNjk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.20" data-doi="10.1046/j.1462-2920.2002.00362.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Symp. Ser.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BSymp.%2BSer.%26rft.volume%253D41%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1462-2920.2002.00362.x%26rft_id%253Dinfo%253Apmid%252F12460286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1462-2920.2002.00362.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12460286&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The corrinoid-containing 23-kDa subunit MtrA of the energy-conserving N5-methyltetrahydromethanopterin:coenzyme M methyltransferase complex from Methanobacterium thermoautotrophicum. EPR spectroscopic evidence for a histidine residue as a cobalt ligand of the cobamide.</span> <abbr class="cit-jnl-abbrev">Eur. J. Biochem.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DHarms%26rft.auinit1%253DU.%26rft.volume%253D241%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D154%26rft_id%253Dinfo%253Apmid%252F8898900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8898900&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VL74800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.22" data-doi="10.1111/j.1432-1033.1997.00783.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Identification of the active site histidine in the corrinoid protein MtrA of the energy-conserving methyltransferase complex from Methanobacterium thermoautotrophicum.</span> <abbr class="cit-jnl-abbrev">Eur. J. Biochem.</abbr> <span class="cit-vol">250</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DHarms%26rft.auinit1%253DU%26rft.volume%253D250%26rft.issue%253D3%26rft.spage%253D783%26rft.epage%253D788%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bactive%2Bsite%2Bhistidine%2Bin%2Bthe%2Bcorrinoid%2Bprotein%2BMtrA%2Bof%2Bthe%2Benergy-conserving%2Bmethyltransferase%2Bcomplex%2Bfrom%2BMethanobacterium%2Bthermoautotrophicum.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1432-1033.1997.00783.x%26rft_id%253Dinfo%253Apmid%252F9461302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1432-1033.1997.00783.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9461302&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071370300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gartner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The energy conserving N5-methyltetrahydromethanopterin:coenzyme M methyltransferase complex from Methanobacterium thermoautotrophicum is composed of eight different subunits.</span> <abbr class="cit-jnl-abbrev">Eur. J. Biochem.</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DHarms%26rft.auinit1%253DU%26rft.volume%253D228%26rft.issue%253D3%26rft.spage%253D640%26rft.epage%253D648%26rft.atitle%253DThe%2Benergy%2Bconserving%2BN5-methyltetrahydromethanopterin%253Acoenzyme%2BM%2Bmethyltransferase%2Bcomplex%2Bfrom%2BMethanobacterium%2Bthermoautotrophicum%2Bis%2Bcomposed%2Bof%2Beight%2Bdifferent%2Bsubunits.%26rft_id%253Dinfo%253Apmid%252F7737157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7737157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QN89900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.24" data-doi="10.1016/S0014-5793(99)00429-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hippler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The energy conserving methyltetrahydromethanopterin:coenzyme M methyltransferase complex from methanogenic archaea: function of the subunit MtrH.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DHippler%26rft.auinit1%253DB.%26rft.volume%253D449%26rft.issue%253D2-3%26rft.spage%253D165%26rft.epage%253D168%26rft.atitle%253DThe%2Benergy%2Bconserving%2Bmethyltetrahydromethanopterin%253Acoenzyme%2BM%2Bmethyltransferase%2Bcomplex%2Bfrom%2Bmethanogenic%2Barchaea%253A%2Bfunction%2Bof%2Bthe%2Bsubunit%2BMtrH.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252899%252900429-9%26rft_id%253Dinfo%253Apmid%252F10338124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(99)00429-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10338124&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080081200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2475.25"><div class="cit-metadata unstructured">Houghton, J. T., Y. Ding, D. J. Griggs, N. Noguer, P. J. van der Linden, D. Xiaosu, K. Maskell, and C. A. Johnson, 2001 <em>Climate Change 2001: The Scientific Basis: Contribution of Working Group I to the Third Assessment Report of the Intergovernmental Panel on Climate Change</em>. Cambridge University Press, Cambridge; New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.26" data-doi="10.1264/jsme2.ME12189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iino</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamazawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueno</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohkuma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Candidatus Methanogranum caenicola: a novel methanogen from the anaerobic digested sludge, and proposal of Methanomassiliicoccaceae fam. nov. and Methanomassiliicoccales ord. nov., for a methanogenic lineage of the class Thermoplasmata.</span> <abbr class="cit-jnl-abbrev">Microbes Environ.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2BEnviron.%26rft.volume%253D28%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1264%252Fjsme2.ME12189%26rft_id%253Dinfo%253Apmid%252F23524372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1264/jsme2.ME12189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23524372&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319877100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.27" data-doi="10.1038/nrg2689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of gene duplications: classifying and distinguishing between models.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D108%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bduplications%253A%2Bclassifying%2Band%2Bdistinguishing%2Bbetween%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2689%26rft_id%253Dinfo%253Apmid%252F20051986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051986&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.28" data-doi="10.1093/nar/gkr441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FFAS server: novel features and applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W38</span>–<span class="cit-lpage">W44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr441%26rft_id%253Dinfo%253Apmid%252F21715387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzOS9zdXBwbF8yL1czOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.29" data-doi="10.1038/nprot.2009.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">M. J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Protein structure prediction on the Web: a case study using the Phyre server.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKelley%26rft.auinit1%253DL.%2BA.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D371%26rft.atitle%253DProtein%2Bstructure%2Bprediction%2Bon%2Bthe%2BWeb%253A%2Ba%2Bcase%2Bstudy%2Busing%2Bthe%2BPhyre%2Bserver.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.2%26rft_id%253Dinfo%253Apmid%252F19247286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19247286&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265782000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.30" data-doi="10.1007/BF02100115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKishino%26rft.auinit1%253DH.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D170%26rft.epage%253D179%26rft.atitle%253DEvaluation%2Bof%2Bthe%2Bmaximum%2Blikelihood%2Bestimate%2Bof%2Bthe%2Bevolutionary%2Btree%2Btopologies%2Bfrom%2BDNA%2Bsequence%2Bdata%252C%2Band%2Bthe%2Bbranching%2Border%2Bin%2Bhominoidea.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02100115%26rft_id%253Dinfo%253Apmid%252F2509717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02100115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2509717&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AG42200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.31" data-doi="10.1093/bioinformatics/btr315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">TimeTree2: species divergence times on the iPhone.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2023</span>–<span class="cit-lpage">2024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr315%26rft_id%253Dinfo%253Apmid%252F21622662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTQvMjAyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.32" data-doi="10.1016/j.cell.2012.11.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Origin of Membrane Bioenergetics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1406</span>–<span class="cit-lpage">1416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1406%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.11.050%26rft_id%253Dinfo%253Apmid%252F23260134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.11.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23260134&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312890300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.33" data-doi="10.1034/j.1600-0889.1998.t01-1-00002.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lelieveld</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crutzen</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dentener</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Changing concentration, lifetime and climate forcing of atmospheric methane.</span> <abbr class="cit-jnl-abbrev">Tellus B Chem. Phys. Meterol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTellus%2BB%2BChem.%2BPhys.%2BMeterol.%26rft.volume%253D50%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1600-0889.1998.t01-1-00002.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1600-0889.1998.t01-1-00002.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.34" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTMvMTY1OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.35" data-doi="10.1086/302221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">D. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Concerted evolution: Molecular mechanism and biological implications.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLiao%26rft.auinit1%253DD.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D30%26rft.atitle%253DConcerted%2Bevolution%253A%2Bmolecular%2Bmechanism%2Band%2Bbiological%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302221%26rft_id%253Dinfo%253Apmid%252F9915939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9915939&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078477400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitman</span>  <span class="cit-name-given-names">W. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> Metabolic, phylogenetic, and ecological diversity of the methanogenic archaea. Ann. N Y Acad. Sci. 1125<strong>:</strong> 171–189.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.37" data-doi="10.1093/nar/gkh477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maiti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Domselaar</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wishart</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SuperPose: a simple server for sophisticated structural superposition.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">W590</span>–<span class="cit-lpage">W594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMaiti%26rft.auinit1%253DR.%26rft.volume%253D32%26rft.issue%253Dsuppl_2%26rft.spage%253DW590%26rft.epage%253DW594%26rft.atitle%253DSuperPose%253A%2Ba%2Bsimple%2Bserver%2Bfor%2Bsophisticated%2Bstructural%2Bsuperposition%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh477%26rft_id%253Dinfo%253Apmid%252F15215457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8yL1c1OTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.38" data-doi="10.1534/genetics.108.087676"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The evolutionary rate of duplicated genes under concerted evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.087676%26rft_id%253Dinfo%253Apmid%252F18757936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.39" data-doi="10.1534/genetics.109.111161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansai</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The power of the methods for detecting interlocus gene conversion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111161%26rft_id%253Dinfo%253Apmid%252F19948889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzIvNTE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.40" data-doi="10.1093/bioinformatics/btq467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moulton</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RDP3: a flexible and fast computer program for analyzing recombination.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2462</span>–<span class="cit-lpage">2463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq467%26rft_id%253Dinfo%253Apmid%252F20798170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTkvMjQ2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.11.2475.41"><div class="cit-metadata unstructured">Melo, F., D. Devos, E. Depiereux, and E. Feytmans, 1997 ANOLEA: a WWW server to assess protein structures. Proc. Int. Conf. Intell. Syst. Mol. Biol. 5: 187–190.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.42" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The relation of recombination to mutational advance.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D1%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft_id%253Dinfo%253Apmid%252F14195748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14195748&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1964WT37300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.43" data-doi="10.1146/annurev.genet.39.073003.112240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Concerted and birth-and-death evolution of multigene families.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.spage%253D121%26rft.epage%253D152%26rft.atitle%253DConcerted%2Band%2Bbirth-and-death%2Bevolution%2Bof%2Bmultigene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.112240%26rft_id%253Dinfo%253Apmid%252F16285855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.112240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285855&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.44" data-doi="10.1093/bioinformatics/bts091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okonechnikov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golosova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fursov</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Unipro UGENE: a unified bioinformatics toolkit.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts091%26rft_id%253Dinfo%253Apmid%252F22368248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOC84LzExNjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.45" data-doi="10.1128/AEM.02193-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonoh</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikulski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brune</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">“Methanoplasmatales,” Thermoplasmatales-related archaea in termite guts and other environments, are the seventh order of methanogens.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">8245</span>–<span class="cit-lpage">8253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02193-12%26rft_id%253Dinfo%253Apmid%252F23001661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8yMy84MjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.46" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">UCSF chimera—A visualization system for exploratory research and analysis.</span> <abbr class="cit-jnl-abbrev">J. Comput. Chem.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.47" data-doi="10.1093/nar/gkl842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D61</span>–<span class="cit-lpage">D65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl842%26rft_id%253Dinfo%253Apmid%252F17130148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNS9zdXBwbF8xL0Q2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.48" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">EMBOSS: The European molecular biology open software suite.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.49" data-doi="10.1073/pnas.1105100108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasieva</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Horizontal gene transfer facilitated the evolution of plant parasitic mechanisms in the oomycetes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">15258</span>–<span class="cit-lpage">15263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1105100108%26rft_id%253Dinfo%253Apmid%252F21878562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM3LzE1MjU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.50" data-doi="10.1093/bioinformatics/btg180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DRonquist%26rft.auinit1%253DF.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D1572%26rft.epage%253D1574%26rft.atitle%253DMrBayes%2B3%253A%2BBayesian%2Bphylogenetic%2Binference%2Bunder%2Bmixed%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg180%26rft_id%253Dinfo%253Apmid%252F12912839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTIvMTU3MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.51" data-doi="10.1073/pnas.1220225110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarmiento</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mrazek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitman</span>  <span class="cit-name-given-names">W. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-scale analysis of gene function in the hydrogenotrophic methanogenic archaeon Methanococcus maripaludis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">4726</span>–<span class="cit-lpage">4731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220225110%26rft_id%253Dinfo%253Apmid%252F23487778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzEyLzQ3MjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical tests for detecting gene conversion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSawyer%26rft.auinit1%253DS%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D526%26rft.epage%253D538%26rft.atitle%253DStatistical%2Btests%2Bfor%2Bdetecting%2Bgene%2Bconversion%26rft_id%253Dinfo%253Apmid%252F2677599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjYvNS81MjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.53" data-doi="10.1080/10635150290069913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An approximately unbiased test of phylogenetic tree selection.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150290069913%26rft_id%253Dinfo%253Apmid%252F12079646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUxLzMvNDkyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.54" data-doi="10.1093/oxfordjournals.molbev.a026201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple comparisons of log-likelihoods with applications to phylogenetic inference.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DShimodaira%26rft.auinit1%253DH%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D1114%26rft.epage%253D1116%26rft.atitle%253DMultiple%2BComparisons%2Bof%2BLog-Likelihoods%2Bwith%2BApplications%2Bto%2BPhylogenetic%2BInference%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081925400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.55" data-doi="10.1093/bioinformatics/17.12.1246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">CONSEL: for assessing the confidence of phylogenetic tree selection.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1246</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DShimodaira%26rft.auinit1%253DH.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D1246%26rft.epage%253D1247%26rft.atitle%253DCONSEL%253A%2Bfor%2Bassessing%2Bthe%2Bconfidence%2Bof%2Bphylogenetic%2Btree%2B%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.12.1246%26rft_id%253Dinfo%253Apmid%252F11751242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTcvMTIvMTI0NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.56" data-doi="10.1093/nar/gki408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biegert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupas</span>  <span class="cit-name-given-names">A. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The HHpred interactive server for protein homology detection and structure prediction.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W244</span>–<span class="cit-lpage">W248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSoding%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW244%26rft.epage%253DW248%26rft.atitle%253DThe%2BHHpred%2Binteractive%2Bserver%2Bfor%2Bprotein%2Bhomology%2Bdetection%2Band%2Bstructure%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki408%26rft_id%253Dinfo%253Apmid%252F15980461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8yL1cyNDQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.57" data-doi="10.1093/bioinformatics/btl446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">RAxML-VI-HPC: Maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2688</span>–<span class="cit-lpage">2690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl446%26rft_id%253Dinfo%253Apmid%252F16928733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMjEvMjY4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.58" data-doi="10.1186/1471-2105-11-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stover</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">TreeGraph 2: Combining and visualizing evidence from different phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStoever%26rft.auinit1%253DB.%2BC.%26rft.volume%253D11%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DTreeGraph%2B2%253A%2Bcombining%2Band%2Bvisualizing%2Bevidence%2Bfrom%2Bdifferent%2Bphylogenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-7%26rft_id%253Dinfo%253Apmid%252F20051126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051126&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.59" data-doi="10.1093/nar/gkl315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torrents</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W609</span>–<span class="cit-lpage">W612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSuyama%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW609%26rft.epage%253DW612%26rft.atitle%253DPAL2NAL%253A%2Brobust%2Bconversion%2Bof%2Bprotein%2Bsequence%2Balignments%2Binto%2Bthe%2Bcorresponding%2Bcodon%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl315%26rft_id%253Dinfo%253Apmid%252F16845082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8yL1c2MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzI0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.60" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.61" data-doi="10.1099/00221287-144-9-2377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Biochemistry of methanogenesis: a tribute to Marjory Stephenson. 1998 Marjory Stephenson Prize Lecture.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">2377</span>–<span class="cit-lpage">2406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DThauer%26rft.auinit1%253DR.%2BK.%26rft.volume%253D144%26rft.issue%253D9%26rft.spage%253D2377%26rft.epage%253D2406%26rft.atitle%253DBiochemistry%2Bof%2Bmethanogenesis%253A%2Ba%2Btribute%2Bto%2BMarjory%2BStephenson%253A1998%2BMarjory%2BStephenson%2BPrize%2BLecture%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-144-9-2377%26rft_id%253Dinfo%253Apmid%252F9782487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-144-9-2377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9782487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076004700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.62" data-doi="10.1110/ps.0236803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elofsson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Can correct protein models be identified?</span> <abbr class="cit-jnl-abbrev">Protein Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DWallner%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1073%26rft.epage%253D1086%26rft.atitle%253DCan%2Bcorrect%2Bprotein%2Bmodels%2Bbe%2Bidentified%253F%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.0236803%26rft_id%253Dinfo%253Apmid%252F12717029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.0236803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12717029&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182486800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.63" data-doi="10.1016/0092-8674(93)90570-G"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thauer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Methanogenesis and the unity of biochemistry.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWeiss%26rft.auinit1%253DD.%2BS.%26rft.volume%253D72%26rft.issue%253D6%26rft.spage%253D819%26rft.epage%253D822%26rft.atitle%253DMethanogenesis%2Band%2Bthe%2Bunity%2Bof%2Bbiochemistry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990570-G%26rft_id%253Dinfo%253Apmid%252F8458077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90570-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8458077&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KU17500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.64" data-doi="10.1073/pnas.0502623102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welander</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metcalf</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Loss of the mtr operon in Methanosarcina blocks growth on methanol, but not methanogenesis, and reveals an unknown methanogenic pathway.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">10664</span>–<span class="cit-lpage">10669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0502623102%26rft_id%253Dinfo%253Apmid%252F16024727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzMwLzEwNjY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.65" data-doi="10.1016/j.bpj.2011.10.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improving the physical realism and structural accuracy of protein models by a two-step atomic-level energy minimization.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">2525</span>–<span class="cit-lpage">2534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DXu%26rft.auinit1%253DD.%26rft.volume%253D101%26rft.issue%253D10%26rft.spage%253D2525%26rft.epage%253D2534%26rft.atitle%253DImproving%2Bthe%2Bphysical%2Brealism%2Band%2Bstructural%2Baccuracy%2Bof%2Bprotein%2Bmodels%2Bby%2Ba%2Btwo-step%2Batomic-level%2Benergy%2Bminimization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2011.10.024%26rft_id%253Dinfo%253Apmid%252F22098752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2011.10.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22098752&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297143500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2475.66" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2475.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2463" title="Linkage Mapping Reveals Strong Chiasma Interference in Sockeye Salmon: Implications for Interpreting Genomic Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2487" title="RNAseq Analysis Highlights Specific Transcriptome Signatures of Yeast and Mycelial Growth Phases in the Dutch Elm Disease Fungus Ophiostoma novo-ulmi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2475.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uyc74zYrXTyVZTU6uH9kvbf6TmhaUXVD7B0ryVbDjlE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2475" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2475" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2475" class="active" data-icon-position="" data-hide-link-title="0">Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460163" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xpCtYaZgCIJJcbrIqyi_Ij_scrOeoZpydtG7H6Gmn3M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2475.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2475" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-yq8ETIaEOHDLRlk22oN_9IcRv81TgcliIOgjKA_xrsc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2475" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2475.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460163" id="node460163--2" data-pisa="ggg;5/11/2475" data-pisa-master="ggg;g3.115.020180" data-apath="/ggg/5/11/2475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sishuo Wang</span>, <span class="highwire-citation-author" data-delta="1">Youhua Chen</span>, <span class="highwire-citation-author" data-delta="2">Qinhong Cao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Huiqiang Lou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2475-2486;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020180 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460163--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sishuo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sishuo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASishuo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Renewable Resources, University of Alberta, Edmonton, T6G 2H1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYouhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qinhong Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinhong%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinhong%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huiqiang Lou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huiqiang%2BLou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuiqiang%2BLou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Huiqiang Lou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2475" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2475&amp;text=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2475&amp;t=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2475&amp;title=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460163/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460163" id="node460163--3" data-pisa="ggg;5/11/2475" data-pisa-master="ggg;g3.115.020180" data-apath="/ggg/5/11/2475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Lasting Gene Conversion Shapes the Convergent Evolution of the Critical Methanogenesis Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sishuo Wang</span>, <span class="highwire-citation-author" data-delta="1">Youhua Chen</span>, <span class="highwire-citation-author" data-delta="2">Qinhong Cao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Huiqiang Lou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2475-2486;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020180 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460163--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sishuo Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sishuo%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASishuo%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youhua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Renewable Resources, University of Alberta, Edmonton, T6G 2H1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youhua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYouhua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qinhong Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinhong%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinhong%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Huiqiang Lou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Agro-Biotechnology and Ministry of Agriculture Key Laboratory of Soil Microbiology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huiqiang%2BLou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHuiqiang%2BLou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4465-6186" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Huiqiang Lou</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangsishuo@yeah.net" class="" data-icon-position="" data-hide-link-title="0">wangsishuo@yeah.net</a>
<a href="mailto:lou@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">lou@cau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2475&amp;rft.epage=2486&amp;rft.atitle=Long-Lasting%20Gene%20Conversion%20Shapes%20the%20Convergent%20Evolution%20of%20the%20Critical%20Methanogenesis%20Genes&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Sishuo&amp;rft.au=Chen%2C+Youhua&amp;rft.au=Cao%2C+Qinhong&amp;rft.au=Lou%2C+Huiqiang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460163/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460163/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460163/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460163/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460163/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460163/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460163/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460163/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460163/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460163/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460163/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460163/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2475" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2475.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2475.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2475.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2475.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
