<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2307" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2307.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2307.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2307.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2307.atom" />
<meta name="HW.pisa" content="ggg;5/11/2307" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea" />
<meta name="DC.Identifier" content="10.1534/g3.115.021402" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Whittle and Extavour. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In protein-coding genes, synonymous codon usage and amino acid composition correlate to expression in some eukaryotes, and may result from translational selection. Here, we studied large-scale RNA-seq data from three divergent arthropod models, including cricket ( Gryllus bimaculatus) , milkweed bug ( Oncopeltus fasciatus ), and the amphipod crustacean Parhyale hawaiensis , and tested for optimization of codon and amino acid usage relative to expression level. We report strong signals of AT3 optimal codons (those favored in highly expressed genes) in G. bimaculatus and O. fasciatus , whereas weaker signs of GC3 optimal codons were found in P. hawaiensis , suggesting selection on codon usage in all three organisms. Further, in G. bimaculatus and O. fasciatus , high expression was associated with lowered frequency of amino acids with large size/complexity (S/C) scores in favor of those with intermediate S/C values; thus, selection may favor smaller amino acids while retaining those of moderate size for protein stability or conformation. In P. hawaiensis , highly transcribed genes had elevated frequency of amino acids with large and small S/C scores, suggesting a complex dynamic in this crustacean. In all species, the highly transcribed genes appeared to favor short proteins, high optimal codon usage, specific amino acids, and were preferentially involved in cell-cycling and protein synthesis. Together, based on examination of 1,680,067, 1,667,783, and 1,326,896 codon sites in G. bimaculatus , O. fasciatus , and P. hawaiensis , respectively, we conclude that translational selection shapes codon and amino acid usage in these three Pancrustacean arthropods." />
<meta name="DC.Contributor" content="Carrie A. Whittle" />
<meta name="DC.Contributor" content="Cassandra G. Extavour" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In protein-coding genes, synonymous codon usage and amino acid composition correlate to expression in some eukaryotes, and may result from translational selection. Here, we studied large-scale RNA-seq data from three divergent arthropod models, including cricket (&lt;i&gt;Gryllus bimaculatus)&lt;/i&gt;, milkweed bug (&lt;i&gt;Oncopeltus fasciatus&lt;/i&gt;), and the amphipod crustacean &lt;i&gt;Parhyale hawaiensis&lt;/i&gt;, and tested for optimization of codon and amino acid usage relative to expression level. We report strong signals of AT3 optimal codons (those favored in highly expressed genes) in &lt;i&gt;G. bimaculatus&lt;/i&gt; and &lt;i&gt;O. fasciatus&lt;/i&gt;, whereas weaker signs of GC3 optimal codons were found in &lt;i&gt;P. hawaiensis&lt;/i&gt;, suggesting selection on codon usage in all three organisms. Further, in &lt;i&gt;G. bimaculatus&lt;/i&gt; and &lt;i&gt;O. fasciatus&lt;/i&gt;, high expression was associated with lowered frequency of amino acids with large size/complexity (S/C) scores in favor of those with intermediate S/C values; thus, selection may favor smaller amino acids while retaining those of moderate size for protein stability or conformation. In &lt;i&gt;P. hawaiensis&lt;/i&gt;, highly transcribed genes had elevated frequency of amino acids with large and small S/C scores, suggesting a complex dynamic in this crustacean. In all species, the highly transcribed genes appeared to favor short proteins, high optimal codon usage, specific amino acids, and were preferentially involved in cell-cycling and protein synthesis. Together, based on examination of 1,680,067, 1,667,783, and 1,326,896 codon sites in &lt;i&gt;G. bimaculatus&lt;/i&gt;, &lt;i&gt;O. fasciatus&lt;/i&gt;, and &lt;i&gt;P. hawaiensis&lt;/i&gt;, respectively, we conclude that translational selection shapes codon and amino acid usage in these three Pancrustacean arthropods.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2307" />
<meta name="citation_id" content="5/11/2307" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2307" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2307.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2307.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2307.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021402" />
<meta name="citation_pmid" content="26384771" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2307" />
<meta name="citation_lastpage" content="2321" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Carrie A. Whittle" />
<meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Cassandra G. Extavour" />
<meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author_email" content="extavour@oeb.harvard.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2922-5855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_author=S.W. Schaeffer;citation_title=Natural Selection and the Frequency Distributions of ``Silent'' DNA Polymorphism in Drosophila;citation_pages=295-307;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136019" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Akam;citation_title=Arthropods: development diversity within a (super) phylum.;citation_pages=4438-4441;citation_volume=9;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=H. Akashi;citation_title=Gene expression and molecular evolution.;citation_pages=660-666;citation_volume=11;citation_year=2001;citation_issue=6;citation_pmid=11682310;citation_doi=10.1016/S0959-437X(00)00250-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Translational Selection and Yeast Proteome Evolution;citation_pages=1291-1303;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930740" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. Bachtrog;citation_title=Reduced selection for codon usage bias in Drosophila miranda.;citation_pages=586-590;citation_volume=64;citation_year=2007;citation_issue=5;citation_pmid=17457633;citation_doi=10.1007/s00239-006-0257-x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Behura;citation_author=D. W. Severson;citation_title=Comparative analysis of codon usage bias and codon context patterns between dipteran and hymenopteran sequenced genomes.;citation_pages=e43111-e43111;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22912801;citation_doi=10.1371/journal.pone.0043111" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=S. K. Behura;citation_author=D. W. Severson;citation_title=Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.;citation_pages=49-61;citation_volume=88;citation_year=2013;citation_doi=10.1111/j.1469-185X.2012.00242.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Beletskii;citation_author=A. S. Bhagwat;citation_title=Transcription-induced mutations: Increase in C to T mutations in the nontranscribed strand during transcription in Escherichia coli;citation_pages=13919-13924;citation_volume=93;citation_year=1996;citation_issue=24;citation_pmid=8943036;citation_doi=10.1073/pnas.93.24.13919" />
<meta name="citation_reference" content="citation_journal_title=Biological chemistry;citation_journal_abbrev=Biol Chem;citation_author=A. Beletskii;citation_author=A. S. Bhagwat;citation_title=Correlation between transcription and C to T mutations in the non-transcribed DNA strand.;citation_pages=549-551;citation_volume=379;citation_year=1998;citation_issue=4-5;citation_pmid=9628351" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. J. Betancourt;citation_author=D. C. Presgraves;citation_title=Linkage limits the power of natural selection in Drosophila.;citation_volume=99;citation_year=2002;citation_pmid=12370444;citation_doi=10.1073/pnas.212277199" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. J. Brown;citation_author=R. Denell;citation_author=R. Gibbs;citation_author=M. Klingler;citation_author=M. Lorenzen;citation_title=The genome of the model beetle and pest Tribolium castaneum.;citation_pages=949-955;citation_volume=452;citation_year=2008;citation_issue=7190;citation_pmid=18362917;citation_doi=10.1038/nature06784" />
<meta name="citation_reference" content="citation_journal_title=Cornell Experiment Station Memoir;citation_author=F. H. Butt;citation_title=Embryology of the Milkweed Bug, Oncopeltus fasciatus (Hemiptera).;citation_pages=2-43;citation_volume=283;citation_year=1949" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=O. Clay;citation_author=S. Caccio;citation_author=S. Zoubak;citation_author=D. Mouchiroud;citation_author=G. Bernardi;citation_title=Human coding and noncoding DNA: compositional correlations.;citation_pages=2-12;citation_volume=5;citation_year=1996;citation_issue=1;citation_pmid=8673288;citation_doi=10.1006/mpev.1996.0002" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. Coghlan;citation_author=K. H. Wolfe;citation_title=Relationship of codon bias to mRNA concentration and protein length in Saccharomyces cerevisiae.;citation_pages=1131-1145;citation_volume=16;citation_year=2000;citation_issue=12;citation_pmid=10953085;citation_doi=10.1002/1097-0061(20000915)16:12&lt;1131::AID-YEA609&gt;3.0.CO;2-F" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. K. Colbourne;citation_author=M. E. Pfrender;citation_author=D. Gilbert;citation_author=W. K. Thomas;citation_author=A. Tucker;citation_title=The Ecoresponsive Genome of Daphnia pulex;citation_pages=555-561;citation_volume=331;citation_year=2011;citation_issue=6017;citation_pmid=21292972;citation_doi=10.1126/science.1197761" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. A. Combs;citation_author=M. B. Eisen;citation_title=Sequencing mRNA from cryo-sliced Drosophila embryos to determine genome-wide spatial patterns of gene expression.;citation_pages=e71820;citation_volume=8;citation_year=2013;citation_pmid=23951250;citation_doi=10.1371/journal.pone.0071820" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_title=Selective and Mutational Patterns Associated With Gene Expression in Humans: Influences on Synonymous Composition and Intron Presence;citation_pages=1293-1304;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280243;citation_doi=10.1534/genetics.104.026351" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_title=The Correlation Between Synonymous and Nonsynonymous Substitutions in Drosophila: Mutation, Selection or Relaxed Constraints?;citation_pages=767-775;citation_volume=150;citation_year=1998;citation_issue=2;citation_pmid=9755207" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_author=M. Aguade;citation_title=Natural Selection on Synonymous Sites Is Correlated With Gene Length and Recombination in Drosophila;citation_pages=239-249;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872963" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Cutter;citation_title=Divergence times in Caenorhabditis and Drosophila inferred from direct estimates of the neutral mutation rate.;citation_volume=25;citation_year=2008;citation_pmid=18234705;citation_doi=10.1093/molbev/msn024" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Cutter;citation_author=J. D. Wasmuth;citation_author=M. L. Blaxter;citation_title=The evolution of biased codon and amino acid usage in nematode genomes.;citation_volume=23;citation_year=2006;citation_pmid=16936139;citation_doi=10.1093/molbev/msl097" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. Diez-Roux;citation_author=S. Banfi;citation_author=M. Sultan;citation_author=L. Geffers;citation_author=S. Anand;citation_title=A high-resolution anatomical atlas of the transcriptome in the mouse embryo.;citation_pages=e1000582-e1000582;citation_volume=9;citation_year=2011;citation_issue=1;citation_pmid=21267068;citation_doi=10.1371/journal.pbio.1000582" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_title=Embryonic development of the cricket Gryllus bimaculatus.;citation_author=S. Donoughe;citation_author=C. G. Extavour;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. A. Drummond;citation_author=J. D. Bloom;citation_author=C. Adami;citation_author=C. O. Wilke;citation_author=F. H. Arnold;citation_title=Why highly expressed proteins evolve slowly.;citation_volume=102;citation_year=2005;citation_pmid=16176987;citation_doi=10.1073/pnas.0504070102" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=M. J. Dufton;citation_title=Genetic code synonym quotas and amino acid complexity: cutting the cost of proteins?;citation_pages=165-173;citation_volume=187;citation_year=1997;citation_issue=2;citation_pmid=9237887;citation_doi=10.1006/jtbi.1997.0443" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. Duret;citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes.;citation_pages=287-289;citation_volume=16;citation_year=2000;citation_issue=7;citation_pmid=10858656;citation_doi=10.1016/S0168-9525(00)02041-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Duret;citation_author=D. Mouchiroud;citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;citation_pages=4482-4487;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200288;citation_doi=10.1073/pnas.96.8.4482" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Ewen-Campen;citation_author=N. Shaner;citation_author=K. A. Panfilio;citation_author=Y. Suzuki;citation_author=S. Roth;citation_title=The maternal and early embryonic transcriptome of the milkweed bug Oncopeltus fasciatus.;citation_pages=61-61;citation_volume=12;citation_year=2011;citation_pmid=21266083;citation_doi=10.1186/1471-2164-12-61" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=F. Garcia-Gonzalez;citation_author=L. W. Simmons;citation_title=Male-induced costs of mating for females compensated by offspring viability benefits in an insect.;citation_pages=2066-2075;citation_volume=23;citation_year=2010;citation_issue=10;citation_pmid=20722896;citation_doi=10.1111/j.1420-9101.2010.02065.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=L. Gilboa;citation_title=Organizing stem cell units in the Drosophila ovary.;citation_pages=31-36;citation_volume=32C;citation_year=2015;citation_pmid=25703842;citation_doi=10.1016/j.gde.2015.01.005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Green;citation_author=B. Ewing;citation_author=W. Miller;citation_author=P. J. Thomas;citation_author=E. D. Green;citation_title=Transcription-associated mutational asymmetry in mammalian evolution.;citation_pages=514-517;citation_volume=33;citation_year=2003;citation_issue=4;citation_pmid=12612582;citation_doi=10.1038/ng1103" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. G. W. Group;citation_author=J. H. Werren;citation_author=S. Richards;citation_author=C. A. Desjardins;citation_author=O. Niehuis;citation_title=Functional and Evolutionary Insights from the Genomes of Three Parasitoid Nasonia Species;citation_pages=343-348;citation_volume=327;citation_year=2010;citation_issue=5963;citation_pmid=20075255;citation_doi=10.1126/science.1178028" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. R. Haddrill;citation_author=K. Zeng;citation_author=B. Charlesworth;citation_title=Determinants of synonymous and nonsynonymous variability in three species of Drosophila.;citation_volume=28;citation_year=2011;citation_pmid=21191087;citation_doi=10.1093/molbev/msq354" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Heger;citation_author=C. P. Ponting;citation_title=Evolutionary rate analyses of orthologs and paralogs from 12 Drosophila genomes.;citation_volume=17;citation_year=2007;citation_pmid=17989258;citation_doi=10.1101/gr.6249707" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=General rules for optimal codon choice.;citation_pages=e1000556-e1000556;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593368;citation_doi=10.1371/journal.pgen.1000556" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=On the limitations of using ribosomal genes as references for the study of codon usage: a rebuttal.;citation_pages=e49060;citation_volume=7;citation_year=2012;citation_pmid=23284622;citation_doi=10.1371/journal.pone.0049060" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Hodgkinson;citation_author=A. Eyre-Walker;citation_title=Variation in the mutation rate across mammalian genomes.;citation_pages=756-766;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=21969038;citation_doi=10.1038/nrg3098" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=and R. A. Lempicki;citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.;citation_volume=37;citation_year=2009;citation_pmid=19033363;citation_doi=10.1093/nar/gkn923" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=and R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_title=Cricket Behavior and Neurobiology;citation_author=F. Huber;citation_author=T. E. Moore;citation_author=W. Loher;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. K. Ingvarsson;citation_title=Gene expression and protein length influence codon usage and rates of sequence evolution in Populus tremula.;citation_volume=24;citation_year=2007;citation_pmid=17204548;citation_doi=10.1093/molbev/msl212" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=P. K. Ingvarsson;citation_title=Molecular evolution of synonymous codon usage in Populus.;citation_pages=307-307;citation_volume=8;citation_year=2008;citation_pmid=18983655;citation_doi=10.1186/1471-2148-8-307" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Jankovics;citation_author=L. Henn;citation_author=A. Bujna;citation_author=P. Vilmos;citation_author=K. Spirohn;citation_title=Functional analysis of the Drosophila embryonic germ cell transcriptome by RNA interference.;citation_pages=e98579;citation_volume=9;citation_year=2014;citation_pmid=24896584;citation_doi=10.1371/journal.pone.0098579" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Jia;citation_author=S. Liu;citation_author=H. Zheng;citation_author=B. Li;citation_author=Q. Qi;citation_title=Non-uniqueness of factors constraint on the codon usage in Bombyx mori.;citation_pages=356;citation_volume=16;citation_year=2015;citation_pmid=25943559;citation_doi=10.1186/s12864-015-1596-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Kim;citation_title=Effect of strong directional selection on weakly selected mutations at linked sites: implication for synonymous codon usage.;citation_volume=21;citation_year=2004;citation_pmid=14660698" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=Z. Kontarakis;citation_author=A. Pavlopoulos;citation_author=A. Kiupakis;citation_author=N. Konstantinides;citation_author=V. Douris;citation_title=A versatile strategy for gene trapping and trap conversion in emerging model organisms;citation_pages=2625-2630;citation_volume=138;citation_year=2011;citation_issue=12;citation_pmid=21610038;citation_doi=10.1242/dev.066324" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Lemos;citation_author=B. R. Bettencourt;citation_author=C. D. Meiklejohn;citation_author=D. L. Hartl;citation_title=Evolution of proteins and gene expression levels are coupled in Drosophila and are independently associated with mRNA abundance, protein length, and number of protein-protein interactions.;citation_volume=22;citation_year=2005;citation_pmid=15746013;citation_doi=10.1093/molbev/msi122" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. J. Li;citation_author=H. Huang;citation_author=P. J. Bickel;citation_author=S. E. Brenner;citation_title=Comparison of D. melanogaster and C. elegans developmental stages, tissues, and cells by modENCODE RNA-seq data.;citation_volume=24;citation_year=2014;citation_pmid=24985912;citation_doi=10.1101/gr.170100.113" />
<meta name="citation_reference" content="citation_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=P. Liu;citation_author=T. C. Kaufman;citation_title=Morphology and Husbandry of the Large Milkweed Bug, Oncopeltus fasciatus;citation_pages=pdb.emo127-pdb.emo127;citation_volume=2009;citation_year=2009;citation_issue=8;citation_pmid=20147229;citation_doi=10.1101/pdb.emo127" />
<meta name="citation_reference" content="citation_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=P. Liu;citation_author=T. C. Kaufman;citation_title=Dissection and Fixation of Large Milkweed Bug (Oncopeltus) Embryos;citation_pages=pdb.prot5261-pdb.prot5261;citation_volume=2009;citation_year=2009;citation_issue=8;citation_pmid=20147237;citation_doi=10.1101/pdb.prot5261" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Loewe;citation_author=B. Charlesworth;citation_title=Background selection in single genes may explain patterns of codon bias.;citation_volume=175;citation_year=2007;citation_pmid=17194784;citation_doi=10.1534/genetics.106.065557" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=J. E. Mank;citation_author=H. Ellegren;citation_title=Are sex-biased genes more dispensable?;citation_volume=5;citation_year=2009;citation_pmid=19433612;citation_doi=10.1098/rsbl.2008.0732" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. A. T. McVean;citation_author=J. Vieira;citation_title=The evolution of codon preferences in Drosophila: a maximum-likelihood approach to parameter estimation and hypothesis testing.;citation_pages=63-75;citation_volume=49;citation_year=1999;citation_issue=1;citation_pmid=10368435;citation_doi=10.1007/PL00006535" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. T. McVean;citation_author=J. Vieira;citation_title=Inferring Parameters of Mutation, Selection and Demography From Patterns of Synonymous Site Evolution in Drosophila;citation_pages=245-257;citation_volume=157;citation_year=2001;citation_issue=1;citation_pmid=11139506" />
<meta name="citation_reference" content="citation_title=Emerging Model Organisms: A Laboratory Manual;citation_title=The two-spotted cricket Gryllus bimaculatus: an emerging model for developmental and regeneration studies;citation_author=T. Mito;citation_author=S. Noji;citation_pages=331-346;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli.;citation_volume=26;citation_year=1998;citation_pmid=9628917;citation_doi=10.1093/nar/26.13.3188" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. E. Neafsey;citation_author=R. M. Waterhouse;citation_author=M. R. Abai;citation_author=S. S. Aganezov;citation_author=M. A. Alekseyev;citation_title=Mosquito genomics. Highly evolvable malaria vectors: the genomes of 16 Anopheles mosquitoes.;citation_volume=347;citation_year=2015;citation_pmid=25554792;citation_doi=10.1126/science.1258522" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=F. Odegaard;citation_title=How many species of arthropods? Erwin’s estimate revised.;citation_pages=583-597;citation_volume=71;citation_year=2000;citation_doi=10.1111/j.1095-8312.2000.tb01279.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Osawa;citation_author=T. Ohama;citation_author=F. Yamao;citation_author=A. Muto;citation_author=T. H. Jukes;citation_title=Directional mutation pressure and transfer RNA in choice of the third nucleotide of synonymous two-codon sets.;citation_pages=1124-1128;citation_volume=85;citation_year=1988;citation_issue=4;citation_pmid=2448791;citation_doi=10.1073/pnas.85.4.1124" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Pal;citation_author=B. Papp;citation_author=L. D. Hurst;citation_title=Highly Expressed Genes in Yeast Evolve Slowly;citation_pages=927-931;citation_volume=158;citation_year=2001;citation_issue=2;citation_pmid=11430355" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Pavlopoulos;citation_author=M. Averof;citation_title=Establishing genetic transformation for comparative developmental studies in the crustacean Parhyale hawaiensis.;citation_volume=102;citation_year=2005;citation_pmid=15911760;citation_doi=10.1073/pnas.0501101102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. E. Popescu;citation_author=T. Borza;citation_author=J. P. Bielawski;citation_author=R. W. Lee;citation_title=Evolutionary rates and expression level in Chlamydomonas.;citation_volume=172;citation_year=2006;citation_pmid=16361241;citation_doi=10.1534/genetics.105.047399" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Qiu;citation_author=R. Bergero;citation_author=K. Zeng;citation_author=D. Charlesworth;citation_title=Patterns of codon usage bias in Silene latifolia.;citation_volume=28;citation_year=2011;citation_pmid=20855431;citation_doi=10.1093/molbev/msq251" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. W. Raiford;citation_author=E. M. Heizer;citation_author=R. V. Miller;citation_author=H. Akashi;citation_author=M. L. Raymer;citation_title=Do amino acid biosynthetic costs constrain protein evolution in Saccharomyces cerevisiae?;citation_pages=621-630;citation_volume=67;citation_year=2008;citation_issue=6;citation_pmid=18937004;citation_doi=10.1007/s00239-008-9162-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. C. Regier;citation_author=J. W. Shultz;citation_author=A. Zwick;citation_author=A. Hussey;citation_author=B. Ball;citation_title=Arthropod relationships revealed by phylogenomic analysis of nuclear protein-coding sequences.;citation_pages=1079-1083;citation_volume=463;citation_year=2010;citation_issue=7284;citation_pmid=20147900;citation_doi=10.1038/nature08742" />
<meta name="citation_reference" content="citation_title=Emerging Model Organisms: A Laboratory Manual;citation_title=The crustacean Parhyale hawaiensis, a new model for arthropod development;citation_author=E. J. Rehm;citation_author=R. L. Hannibal;citation_author=R. C. Chaw;citation_author=M. A. Vargas-Vila;citation_author=N. H. Patel;citation_pages=373-404;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=H. Seligmann;citation_title=Cost-minimization of amino acid usage.;citation_pages=151-161;citation_volume=56;citation_year=2003;citation_issue=2;citation_pmid=12574861;citation_doi=10.1007/s00239-002-2388-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Sella;citation_author=D. A. Petrov;citation_author=M. Przeworski;citation_author=P. Andolfatto;citation_title=Pervasive natural selection in the Drosophila genome?;citation_pages=e1000495-e1000495;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19503600;citation_doi=10.1371/journal.pgen.1000495" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. M. Sharp;citation_author=W.-H. Li;citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.;citation_pages=28-38;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104616;citation_doi=10.1007/BF02099948" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=T. M. Tuohy;citation_author=K. R. Mosurski;citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes;citation_pages=5125-5143;citation_volume=14;citation_year=1986;citation_issue=13;citation_pmid=3526280;citation_doi=10.1093/nar/14.13.5125" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=M. Averof;citation_author=A. T. Lloyd;citation_author=G. Matassi;citation_author=J. F. Peden;citation_title=DNA Sequence Evolution: The Sounds of Silence;citation_pages=241-247;citation_volume=349;citation_year=1995;citation_issue=1329;citation_pmid=8577834;citation_doi=10.1098/rstb.1995.0108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. E. St Pierre;citation_author=L. Ponting;citation_author=R. Stefancsik;citation_author=P. McQuilton;citation_title=FlyBase 102–advanced approaches to interrogating FlyBase.;citation_volume=42;citation_year=2014;citation_pmid=24234449;citation_doi=10.1093/nar/gkt1092" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Stark;citation_author=M. F. Lin;citation_author=P. Kheradpour;citation_author=J. S. Pedersen;citation_author=L. Parts;citation_title=Discovery of functional elements in 12 Drosophila genomes using evolutionary signatures.;citation_pages=219-232;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994088;citation_doi=10.1038/nature06340" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Subramanian;citation_author=S. Kumar;citation_title=Gene Expression Intensity Shapes Evolutionary Rates of the Proteins Encoded by the Vertebrate Genome;citation_pages=373-381;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454550;citation_doi=10.1534/genetics.104.028944" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Sueoka;citation_title=Directional mutation pressure and neutral molecular evolution.;citation_pages=2653-2657;citation_volume=85;citation_year=1988;citation_issue=8;citation_pmid=3357886;citation_doi=10.1073/pnas.85.8.2653" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Swire;citation_title=Selection on synthesis cost affects interprotein amino acid usage in all three domains of life.;citation_pages=558-571;citation_volume=64;citation_year=2007;citation_issue=5;citation_pmid=17476453;citation_doi=10.1007/s00239-006-0206-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. Trotta;citation_title=The 3-base periodicity and codon usage of coding sequences are correlated with gene expression at the level of transcription elongation.;citation_pages=e21590-e21590;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21738721;citation_doi=10.1371/journal.pone.0021590" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. O. Urrutia;citation_author=L. D. Hurst;citation_title=Codon Usage Bias Covaries With Expression Breadth and the Rate of Synonymous Evolution in Humans, but This Is Not Evidence for Selection;citation_pages=1191-1199;citation_volume=159;citation_year=2001;citation_issue=3;citation_pmid=11729162" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. P. Wall;citation_author=A. E. Hirsh;citation_author=H. B. Fraser;citation_author=J. Kumm;citation_author=G. Giaever;citation_title=Functional genomic analysis of the rates of protein evolution.;citation_volume=102;citation_year=2005;citation_pmid=15800036;citation_doi=10.1073/pnas.0501761102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Wang;citation_author=Z. W. Shao;citation_author=Y. Xu;citation_author=J. Liu;citation_author=Y. Liu;citation_title=Optimal codon identities in bacteria: implications from the conflicting results of two different methods.;citation_pages=e22714-e22714;citation_volume=6;citation_year=2011;citation_issue=7;citation_pmid=21829489;citation_doi=10.1371/journal.pone.0022714" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Wang;citation_author=M. Hsieh;citation_author=W. H. Li;citation_title=A general tendency for conservation of protein length across eukaryotic kingdoms.;citation_volume=22;citation_year=2005;citation_pmid=15371528" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. M. Weinstock;citation_author=G. E. Robinson;citation_author=R. A. Gibbs;citation_author=K. C. Worley;citation_author=J. D. Evans;citation_title=Insights into social insects from the genome of the honeybee Apis mellifera.;citation_pages=931-949;citation_volume=443;citation_year=2006;citation_issue=7114;citation_pmid=17073008;citation_doi=10.1038/nature05260" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. A. Whittle;citation_author=H. Johannesson;citation_title=Evolutionary dynamics of sex-biased genes in a hermaphrodite fungus.;citation_volume=30;citation_year=2013;citation_pmid=23966547;citation_doi=10.1093/molbev/mst143" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. A. Whittle;citation_author=M. R. Malik;citation_author=J. E. Krochko;citation_title=Gender-specific selection on codon usage in plant genomes.;citation_pages=169-169;citation_volume=8;citation_year=2007;citation_pmid=17567911;citation_doi=10.1186/1471-2164-8-169" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=C. A. Whittle;citation_author=Y. Sun;citation_author=H. Johannesson;citation_title=Evolution of synonymous codon usage in Neurospora tetrasperma and Neurospora discreta.;citation_volume=3;citation_year=2011;citation_pmid=21402862;citation_doi=10.1093/gbe/evr018" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=C. A. Whittle;citation_author=Y. Sun;citation_author=H. Johannesson;citation_title=Degeneration in codon usage within the region of suppressed recombination in the mating-type chromosomes of Neurospora tetrasperma.;citation_volume=10;citation_year=2011;citation_pmid=21335530;citation_doi=10.1128/EC.00284-10" />
<meta name="citation_reference" content="citation_title=The Evolutionary Biology of Flies;citation_author=B. Wiegmann;citation_author=D. K. Yeates;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Williford;citation_author=J. P. Demuth;citation_title=Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, Tribolium castaneum.;citation_volume=29;citation_year=2012;citation_pmid=22826459;citation_doi=10.1093/molbev/mss184" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Database;citation_author=V. Zeng;citation_author=C. G. Extavour;citation_title=ASGARD: an open-access database of annotated transcriptomes for emerging model arthropod species.;citation_volume=2012;citation_year=2012;citation_pmid=23180770;citation_doi=10.1093/database/bas048" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=V. Zeng;citation_author=K. E. Villanueva;citation_author=B. Ewen-Campen;citation_author=F. Alwes;citation_author=W. E. Browne;citation_title=De novo assembly and characterization of a maternal and developmental transcriptome for the emerging model crustacean Parhyale hawaiensis.;citation_pages=581-581;citation_volume=12;citation_year=2011;citation_pmid=22118449;citation_doi=10.1186/1471-2164-12-581" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. Zeng;citation_author=B. Ewen-Campen;citation_author=H. W. Horch;citation_author=S. Roth;citation_author=T. Mito;citation_title=Developmental gene discovery in a hemimetabolous insect: de novo assembly and annotation of a transcriptome for the cricket Gryllus bimaculatus.;citation_pages=e61479;citation_volume=8;citation_year=2013;citation_pmid=23671567;citation_doi=10.1371/journal.pone.0061479" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Entomol.;citation_author=M. Zuk;citation_author=F. Garcia-Gonzalez;citation_author=M. E. Herberstein;citation_author=L. W. Simmons;citation_title=Model systems, taxonomic bias, and sexual selection: beyond Drosophila.;citation_pages=321-338;citation_volume=59;citation_year=2014;citation_pmid=24160422;citation_doi=10.1146/annurev-ento-011613-162014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="In protein-coding genes, synonymous codon usage and amino acid composition correlate to expression in some eukaryotes, and may result from translational selection. Here, we studied large-scale RNA-seq data from three divergent arthropod models, including cricket ( Gryllus bimaculatus) , milkweed bug ( Oncopeltus fasciatus ), and the amphipod crustacean Parhyale hawaiensis , and tested for optimization of codon and amino acid usage relative to expression level. We report strong signals of AT3 optimal codons (those favored in highly expressed genes) in G. bimaculatus and O. fasciatus , whereas weaker signs of GC3 optimal codons were found in P. hawaiensis , suggesting selection on codon usage in all three organisms. Further, in G. bimaculatus and O. fasciatus , high expression was associated with lowered frequency of amino acids with large size/complexity (S/C) scores in favor of those with intermediate S/C values; thus, selection may favor smaller amino acids while retaining those of moderate size for protein stability or conformation. In P. hawaiensis , highly transcribed genes had elevated frequency of amino acids with large and small S/C scores, suggesting a complex dynamic in this crustacean. In all species, the highly transcribed genes appeared to favor short proteins, high optimal codon usage, specific amino acids, and were preferentially involved in cell-cycling and protein synthesis. Together, based on examination of 1,680,067, 1,667,783, and 1,326,896 codon sites in G. bimaculatus , O. fasciatus , and P. hawaiensis , respectively, we conclude that translational selection shapes codon and amino acid usage in these three Pancrustacean arthropods." />
<meta name="og-title" property="og:title" content="Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2307" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In protein-coding genes, synonymous codon usage and amino acid composition correlate to expression in some eukaryotes, and may result from translational selection. Here, we studied large-scale RNA-seq data from three divergent arthropod models, including cricket ( Gryllus bimaculatus) , milkweed bug ( Oncopeltus fasciatus ), and the amphipod crustacean Parhyale hawaiensis , and tested for optimization of codon and amino acid usage relative to expression level. We report strong signals of AT3 optimal codons (those favored in highly expressed genes) in G. bimaculatus and O. fasciatus , whereas weaker signs of GC3 optimal codons were found in P. hawaiensis , suggesting selection on codon usage in all three organisms. Further, in G. bimaculatus and O. fasciatus , high expression was associated with lowered frequency of amino acids with large size/complexity (S/C) scores in favor of those with intermediate S/C values; thus, selection may favor smaller amino acids while retaining those of moderate size for protein stability or conformation. In P. hawaiensis , highly transcribed genes had elevated frequency of amino acids with large and small S/C scores, suggesting a complex dynamic in this crustacean. In all species, the highly transcribed genes appeared to favor short proteins, high optimal codon usage, specific amino acids, and were preferentially involved in cell-cycling and protein synthesis. Together, based on examination of 1,680,067, 1,667,783, and 1,326,896 codon sites in G. bimaculatus , O. fasciatus , and P. hawaiensis , respectively, we conclude that translational selection shapes codon and amino acid usage in these three Pancrustacean arthropods." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kfl24qE6vWtt-NYu-gDhDOxe473MPfRfwhzg0FOZviY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460171","apath":"\/ggg\/5\/11\/2307.atom","pisa":"ggg;5\/11\/2307","ac":{"\/ggg\/5\/11\/2307.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2307.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2307"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2307"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2307":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OZss25U\\\\\\\/86BM0fpOcRu8muRs1CH7hSEpyFJhOqQe8Cl3FN5\\\\\\\/di3IYyw7t3lljdJ2sHoJYiv2T01MAr1T76lSl6pnl8aWEsNbH8mAE37Wl74+Pa\\\\\\\/UOcMgcBgRm5MObHqkIu4q5E7GKV32BPwyLcFLbRGQY3dXgbR8EN1jZ6w\\\\\\\/5nllcUGiu6GxGYYa9Rw3n+2AE5KQtJFhN3dwyWyXVXZOiDLLkf2qwNyoiOIcGo0sN3cb7FuzIrFUMSJr3WJQorZ\\\\\\\/sGWYJ7iRm6ADbMXKXULKfg2nM2UDRpYIERUXIs\\\\\\\/7hjf8EdbEzZJiNX87DV1mz96l0hIvfCakKHjm0PXmAwSrreOmYK2SYmgXT7\\\\\\\/QBhe9iC3AnoN7hWYraqRf4o9vqO84ZH2s1r2cl8IHsBsB8Tosy8l8oYfpfPSMPVnxNqGNvGYtk052P6hjeJIBRKsepyghU\\\\\\\/FhtoTYxijfBGAz1xFa2ohFWwJq0ZOisu8fnJ45A3nU54NKS7Qr3S5WvFFj2HQ4jl1LwOrWF7pITxVyqNv8fKLXg8U\\\\\\\/KC\\\\\\\/O4uwsAVOSAkIB\\\\\\\/SQjFm8sZdafXudRaMmhf2dRdurpTChmBBhhvmEKnTRRPtJOwb55x2oPSWJ7UlX5BYC1bJCg\\\\\\\/XHPPBXGlmOm0EPjN15hgbw4LdXnY3Xt3srK0Lwk8mZBkHgm5MOcnxayfAHgPripXPOn4XZDO1pfsoTDn8YjGKiEsPSdGLESwUGVEFZ6492A9M\\\\\\\/aDEYVZdpgPbk6IhT\\\\\\\/0qsVipyooxngxKjWmxdeXHqYPUqaiKMyTW7iimKbE3BVUd4acI8apa6KhWISJPAX8YrCXgkCBHq0kbi5Rjmee3\\\\\\\/zjevfn4Cgkk4F7aVjiaQqoLAKJOdS5F82UiTrtGK77rhP9i25cPF5p0RjdNp1AwlUWuictAHkpg45QB3faMeFjILpEpTrqcL7poUs3WZ4NiwPclAGsKL9\\\\\\\/XDBhmMB5cx6XxG3\\\\\\\/0hPcNAVYHEfdco9MUGgbsqrNskR7D8Dp4XbITRIEhRZVGAgNY3Js0Bz0mkelSPgF1u\\\\\\\/ctsga9M3UAZZOR0OSJ\\\\\\\/iWp2UPrT6OmR\\\\\\\/yAsuSEWgl2nofx4Egzx7uXhroAQywrkwCfFIMao\\\\\\\/e8bef6TLrL\\\\\\\/cy9JSMj8AUitm6Ih2WEJ3d+aTE3knjmTc\\\\\\\/PWRo4D1+jweQJBDqTvebd1RGGZyv1b6VkzNjrLCqOL0lVLSmwo15T8cbW99l67m2bfLyCcso3RiV0hlGY0ZplLdbEJ35AYnDY9YpxMcvmBVSRx2RKAV92Fd\\\u0022,\\\u0022iv\\\u0022:\\\u0022YPuZjpNCo3aBtZIkOyfdTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227703380ebe6b566f7fcadae6f4313e0123e2109c566a06b4c6b4cf36c6ac7d2b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rxKqNSNX5H2unjDKniG+uOG0pNg4me+vAUZ\\\\\\\/DQOe9tVi+Rtu4f2kk\\\\\\\/ZAowY6vcAws7QmJTJ6WKyhoAC2lxqnoG1RabGMvpYbNU5MzSUrDCW7QHrxKyEkyrqOuBVMJS+5mHRBtpbA1o7540yQuTf+9VMyLRGiH4LNgCZQD97Zl4Arne2XQ0pP2dYZO6wEAfbHH6NjYbyd4qub1l0y9M5iSNdX1N1LVA37DabzN2KaBGyvCA36MWHy2xKvmvSbqh7dwzxcFJRUeQJd8+7NYqWTWgjgKd6SkRbs4qlolQ3ACdho3VXr7qIP9dGjRpc6oG05zjBb5XdB0rA6z7GqSdrxbH\\\\\\\/9rNAgGblWebDthHCuaF0\\\\\\\/IXREad5HZsCNMlSUfhh5FxqN9VH\\\\\\\/2Aq16h20QJoLzcmWNUFs\\\\\\\/o0MmgVL0v3m18HanzOTHTsMahXPlrKM\\\\\\\/hauheBAskLpXl+TNJ9ijWuy1h1Fg2hIqf6F2D0N\\\\\\\/gMm3hZYovvSC+MzIIrIhaWgpEMLuPbfcEU8p1TxLZtudmT9Q5rDv01Bwm3HL7qK2ZjyjWheqz35cWHeEMmSh+\\\\\\\/b2gEBZ+lqENdxephDMWOhjDBLviVajfuiuzv09hMS+th948hFiQgNrEPXpZnoVx3qrqkj8ZnssynU\\\\\\\/M6rTQ66rVWXKn\\\\\\\/+jxrUkY+Nl0L+ZiAjf6nVTqSJqf0O08PtmMbrmn\\\\\\\/zwhDGMQPshHIWNrvDtxmWpYWQn27oLeOfG6VgpmIN5gtT3pUJ0iRhLGI8SyIWseAgs0D5oxV51U4d\\\\\\\/CWzamdxzKhuzS2Qxn5rkP0CsAZ7OkwHkKQhZdHk8kWuHFTtm5sPPkPofGLxgdr+XdTaAiniEkYXvoG5bd6xaALqLtCIYNcCn0L20+iMvz0wz+H0NRfjPui4nRf4MvlzNAvHNnXgOIuixwzU1L5YJAZst+apEdWv\\\\\\\/OjCcRenezX6xHBK\\\\\\\/+D3SkJ4whp91JjWoTq7iHD+QfP34tGcjYHMGqvWn5dO2X2IQvoqFr5kS2GiNR\\\\\\\/wDWElNEBO5w+Xsr1k\\\\\\\/7sWKF2\\\\\\\/bLvsgeltU2xSR1gJNppLX8PAj8Cje5eDycLiJS2bzoeU5myhl9zu56RIgF3AowBb7CnXJlmDUBuyS0fR094JPC9h6euFDbOxMZeA7418EbFJcEBbYUt72LpIkjET\\\\\\\/xinn2rAgFVoGNhFOA6wueGzPu2eY3krIA\\\\\\\/Ijo6MbXAV4CFwbXGxtYg0Th3U8tp9Qo8VDLg4EyeTHavcLAVaBhcEcFNavUlr6APL2uz8lTQk05BcBMzEmA6UjujRpdDddW7mLZvuhuOIvXt1+Z0WZx52Fbd1z4ZLExFCEV3ckWlyGgmyl1zvwVCVBfBCsem5nR1CPRQnVk8e3MA7qz+abM7uGa1x50OJInzZG0M6oLUsyS+yP9VqGgYbo96wwEcfP0Sx+JeE4SN4Lcna2Vwsm007e8Ea+WUdmzOCpp4TAq4oDVT5t6X9PWcNeeK2VhyPgaxXQgLwAtsibrm9FlxZBm4+zyTMqudt0TfKxa7+WEI8WetMwHhMjX9dy3kQb9zGJL2iar7IT3Hf2A6RefZD5cZ1YstvrIJzh8dJQHbfRYoLzWoAiLvKx45KtVty5hsh2rSrlkVqGrOI78UPgOfFQ9lp3tTdA5Yes17NRAO27rVsaBaGDIoJx6IYBjMtyxUCpQ7woMTlcaXgLgcAC9WmgoFQZMCbYRZQIbyvBkb91cB0pcdJAWCTL6e7Shm2pAB4Bb4G7uqv58E+nb6GLuMg7s4I4NnW1UYIWd2f\\\\\\\/ZW\\\\\\\/7a9N5QSRR4dMns3k8a6ljJ+eaFYuWZ1JMpm6ukM+cwhJ2awLgr8vn9FSCfhT1w5Ey\\\\\\\/Gqw0x6KNU7kG\\\\\\\/eJJupFStfPZDkDPSFlRYlX25o2BV3V3OKQEmD2\\\\\\\/yZiTHQQI0oeT9JPwOTCka8+Y5t8WlsherGvAs+lmYhILhC4NIiA7yRkOO3EV+Ehwbkcvf1ykIc\\\\\\\/pKu221RHp4nlHWDQhTIS5OqdAEChJ7toxwj8EaDo5NvCbq2DiZ2cYfDEZ1bWbmPEvZ4oyjrG51ycLrYajtEEzj5\\\\\\\/dQ39\\\\\\\/aEftaKB3qmKeg4SrmrDIdfMC5wd4l3DmsDaDbbgTTUukMsWaH6chiSJFSz+YMMLz90aepefYtcNcnObfL0BX2KGWOllvv7+5gRBJckQUL21wPo+nhldLWKsD8ul0P27UfwuWUH7JNuTbsk3sjAOqXpIdjFm7fCIf0CDkZz0go2t0efXHQSwi07Fr1xZFvg2eZfOn8+VLSRwinttjpqLP+4iBnl\\\\\\\/VrfYlTiGPlqapNK3LXE0aKnBoKwgCiPQevydcyAHSw\\\\\\\/uqaVByx4ZJESaefDPOZANOdah0fM\\\\\\\/i+0v3F8IChxrK+ZCNzYUPM+24Wq2i0cUHAZJRaZiy1Ve0zHwvPj4IzrhR4er8qeIcZP1Da3l2+JQpQ4cymYgVBdxbaNzaUviKdogCTNGJIZxFZX880ledGv49SigYWb2X1Lj9aZSuVcwRUS5ZXoO2x6T\\\\\\\/9jn3VwAWW0u02PeoXVEd2TKcaHhhDN5P1JjRRZag3yNs8JlTFQxkPiMowmMiKRzU3EDyaYGXFtLFuSd8JPT1Fn47DV\\\\\\\/5YhV4Q4CI1iJMLgQ\\\\\\\/d\\\\\\\/oP5KfLahC3lzudGnCglz53SASRJvc6iw0+SltppCGd\\\\\\\/iJz9AphQj2nZtPepnsmKZHQjD5dINlDi6Jhz8CcWHRPjNc9cPfGv+iSD0mlAX74JXhp2oLy8AOxru+IWGFrXysBzx5FSa16qaq4rJCyEYTSBGhfwpk\\\\\\\/dVDU42UPWOp7PZoMLJjFPxTBqSBGmVIdhfANcT7F6zvV7FaLLUVQQl1cxEvTiVAWlKuLaTSgDKZvI\\\\\\\/4fdMwqfHGKY8A\\\\\\\/PCCjDpMM5uhKhR1rVlh0a8WhtI5XgTIWL+j0B8+6nPrh9TOpTQ7+q\\\\\\\/lWZsDmtr7qT4o955eoGc4NYwDs97eYk2AuAKcGtdNsjZmtnuSoprx2Wd8p6ZbOHJt4K3dcr\\\\\\\/1DZYi2xb8H6a5Ua+l+GIiJtYbeY4iRe73bN\\\\\\\/cUG6HW3XubWCHqLt8j+FNhTEtEcXAa8T4d5Nhs8XeelE31saafYclpHuYYohCbo7iW9ZVsNAn+4\\\\\\\/BE0j4Sk8jTEGsDtkInBt2K8CmB1\\\\\\\/EECcr5JZ89ws26AIIqIQ95cCgR3a+flHt2FQ6SxzXefIXKrRQzCVVzjux7mW0M3bXGT3NovOX4Gu\\\\\\\/s21Ps7sX1HSZeMe1o0Y5oAzMT3Ve5bwbRhL9cpdk4c\\\\\\\/CihHuermFsXn3aBTfG1r1XhTXBW5HuoIWApI0rUAS9aCjJb8xwpe43pKf4GQr7afjb2oo\\\\\\\/GTqzridqXI65MTRN485cqvTijEk8GaTkBeEmZtQ652AwrYevrITyFvLglsk2XGpKtKH2xLZJlvFjzLDkekeQkpjegI+uXDcWsMuVrXAKHsPIKgp4+mXWlcvTrEu5hNElFKegh6v6k3sSVaP36uF+XL6omS4Jlz2awyDza8GekZB9IPZvZ5fDFut1mJQZ75ZwOHJrVifB8odS3Eg5\\\\\\\/g59Fji623LkmLdSIHreghRk7hYA18ZrSUaw\\\\\\\/\\\\\\\/BkYWO5lMcskBWyUx+YhJGvK5D8xLKcA1kXuzJNsELu1mFboP+l7EAkQxFdK9pt9B75xPEeSmXtC7ri4w\\\\\\\/xLvyaYBTAwSGihXmpTsNPNZ0GafK6ISa4Wmd+k4mnuRvhu+RKmXt8bLX92YAjfzQR2zhT4BfB8xqMtnDT3+of4iHr\\\\\\\/qFWsXH7tBDeXMbvtMwK+cT\\\\\\\/rl44jOcIaFDfQm04sljoACelhsEhOPsF8+1EtUq50TSq3VlZn6kto6uh4ltk8vqnD09ko1qLrWvqIeQpcDlqQ31wK+hmClnxqpPJVJg8A\\\\\\\/EsKtZNlEyz4YbBxosnmx+S5pSYqmad7M9ujVVNubFf1t26fwv1ATi9t1WWyg9gBElouQ3XsUS9MUKHvpwfqsW3sR0hed+6U\\\\\\\/7IZu\\\\\\\/3IGsraxvN8ARLKdD+ko9fDak1L+\\\\\\\/CGvala+H8Grlnm+07iWbPd9OLgnkPEafbipNYFPE9mQU8xkJQm0jhmvpiVKWgpWG4qEgY+V3LHW2csgrYwyC8bc2u8NKgzD9bcliXvB2Jw3Nrs3ws6rFyYDBS5bJceCv5vbbKQMm0AMl+XnabPcHpubb3nCKQuXv2X\\\\\\\/OherFxDBRrCd8lpo4N2Y4F7CdeCeg9SWAeuur4PkED39+qOIBEn\\\\\\\/XgW2p+jonLQ7CNH3cg+2HQ4Awc2AuVEZkgjAXswa5DX+F\\\\\\\/gdo+vErPqHQUlT2GYFSiJxl4kyMXvL6cRnaqI8ugGBjcJK8GP92MSqnCc5yy2EWzZM8Ub52ak0Zk1wnqvUWJDzfuY8EKbuPnW9gzBuHQxnRa+ovsbKgQzXqJ7l9qhZirfLJ\\\\\\\/lmwSRX1KAT3et2hGaQOLZZvThvraXXetJlHAhfYQC9f\\\\\\\/wIgZ4obosBJFG0e15YwriI+FSj9tywNQ4wY2VfyVwUbokJZMDfwWU6uvpeFznnRJ75MW9BRO52EOi\\\\\\\/mz7gTzIiWXVuQQN7WxcfrYBkqDJSnz2wBqZ2A7\\\\\\\/HcZQylhCkhm8zmf3cqY3FLsueaqvk0Sxfs\\\\\\\/3JsrRLlI9HaiaHtK9v+wazXBNQxS\\\\\\\/5G8pgPL8yDUocaJq\\\\\\\/ETylJP7o3Es0QCXOnX\\\\\\\/4uvOijV5SU1ONZA9pgAvXtjjuiMQgdrLFSStzwsPAUt2bIBSxUHB9PlGXvSmFJHyRrKM1zPL1ZS46AgnkQBKHgB8VkIRbdRQt\\\\\\\/oDXtPz5FHYs6z5gtUh\\\\\\\/izC5DFKRa\\\\\\\/\\\\\\\/1NwELJMfuaNgk5IIurvQ4hHZ\\\\\\\/PIBS+Jqt67AregAg7sgbC33\\\\\\\/EPtVM6iTcbhHqjz6dmFBXSAIPZiouEfBA0NMWT388TG85WM2dPxxmThsChomBVvuM3sQ0fWhNlRR4L5ZCtOTnaz551gYXzwTt1vaL0rPDdQ+tB+geGDPH52aXekdPCJx\\\\\\\/EQCoNAqZC0cAE931r\\\\\\\/80ghKADF4Gpad+sv\\\\\\\/rHdw5zI\\\\\\\/rkksjdd8MCkmy5F8w1hcJeNB+QZt+GI8WfWld3Wq7kU4tZadzDF7K\\\\\\\/xUIdaWdhZiD5cNSpVPsTrKqD8wkYvPo6rEv1MXk5LpIyUBtIxoVVSCUTFnQ7PcMFwNfDNzFRhJsw+YVAchzfHfhkYrp9U69141xBw\\\\\\\/OYF3XV5dgpH30Wg1O+rz0YwbKoykZZ5zEOJQLBelV1LIkPUZYib5OI7nHPDCF9j0X0EEJtCwsX68b3+ao0xqSk68msrE1ELoFs8tVl6Sfl8Dscwt46qWGzbF6uI9GuFXmd+OKVRpKTEzBgJ0KRrW2zuzOr5t5Mac4p4O\\\\\\\/L22pQYqVmfpcF1h8\\\\\\\/Jwk+naFseLq5vA6wrZnqxjL55obyraqRSKzy0KWOaUL1iod18E1J6B+Zj0u4z8GOIf1jjC3o4QfCu5tsHhbetQnGS\\\\\\\/TH\\\\\\\/sqn9UQJceYkiOkon6DGKpFQd8oozcOcA4\\\\\\\/wOZZYsW+ebrRCDsxMryWwt2oHKD65sn+MZNceIueQsGyglQnODJmkAqTtlInupbHohGVsttwLamr04t2tnaHOyErpbVRlSIa1OANpB9YilDc\\\\\\\/0u7t5Or52a+GYJcLRPGHk55lF1mwPJaj0SQZi8y1bLy0VFOqAX2FK357cty\\\\\\\/DmKgBg3RqAcolmG39REDXf0v1f1SM52g5lupChKbG25ufN7Epdy1\\\\\\\/9klyuDrN+cTcvmJlVxTSwI+aplrvdMX3Qva8PrS3xGNRnvqDTqr3U65zMdHvEVPFWwfgHHkMVDER3DV3abWtHf4wpbxmflc4tAoGYc7++KQyCSBTKSzyd6SeP9s47IDTCFSqfwn3ndpu+sKcHmiXJosjZ7ukF\\\\\\\/tNKPgibqrFCxhdJKXta8LjauVC8sWjdKEaPIpOiNT97ijflwkCKBEHlc7HAIgk0d4RwKqCqJnN6GT9rd+i9tiTVQGEcw\\\\\\\/YSzrPYR6ZXLmLKL65o2OfORjS8orcYVuI6T8NT7BBgg2P\\\\\\\/anyc7yJlC\\\\\\\/WCkNUo\\\\\\\/30AhwScQ7zLv4HQEe9VqO7WfZlUjxW+E6dwsXE5Ylf+21dAGbklmgKdHw1IYKNo3+uCYTj5UsgLbf2cZIWwyAKZphp7nfAH5pjZky6Nn\\\\\\\/JrVh26\\\\\\\/iWuQhAfD0VmUXvSy6WazpefPev8DDtZDiAhOhi\\\\\\\/z6znV+bBKozjQeArDMYpa7dhA0bzXmoeLUn\\\\\\\/crsC4WuA52Uvtp\\\\\\\/fgRWdpmynnp0TKFZ\\\\\\\/C8VcAdsxB5RJAKrRK1IEiKdr6CyEjSXMW\\\\\\\/j9RP\\\\\\\/Lg1dYXwge9uF4PnvBVxYbP9MD\\\\\\\/ZLpfhWo17Kk8CSp32yaYlq2oGJEdujnh3fDTBAmWE718xCqR0SD3x9YGZb0WcOGJC4xk8Od1cR\\\\\\\/Mc+4eOidnviT3b0q8y10uoPl7Loq7vxkoC9ciEgfMt3rmNVoXLEDSMQaqFAhH+p91uFrvt\\\\\\\/V4Cf95OrIGVkXZEkJ01NVJY+n4sBr786n15AtO\\\\\\\/BAKVWSxyN9e6meF81qwAyOuA7q4Wa4qzMNbQwSAxiF1yLeolzLr+wi3sFAm05qZn1CYU0CovDddcGKXu6EgmfxnBwRyxiBFt9fwJitsHnYoUC9cQGSfbPTjyR3XOz0UJl\\\\\\\/jCdVQ8+PTAT7I5ERnU3iaNSYYHwztU4Vfcp61uAw1jJgc1LZy1puk4VKjduFr3+KrZqdDcmiqHCPi7r3AkbC2wubGXT4pXByAZzLZ8lYUHCFKhXLy0iHdjdACJ+TorlSCyJm6vnPoYCrNcau1z\\\\\\\/Jpi6BY+Yr5SKiYdMkID+7xo3F0agi3+Nd5jR8VjPEaSbnvfmuMvhrh68MCInvJsrL\\\\\\\/tWBQvy1W9TaFYJCqlwEcyaR2qaP9rHsxg5ZJzPKAW23wIxDWOaioVE5vZVmFkQ1pW+M3RucyMPVvXg9DWw56QFd9JsBpD8NC7+wiIv7XC7SNriZnJrhmfsSrORKPcjbm4MDF6vATSPQi1lmGll3H5XseZF2QM6BotQmC4qHMherOVN8Tlz\\\\\\\/kxJbPg27N86PPLa8\\\\\\\/YTzd4Bye2iRbtIi657TBq9wH81btssgvyuLpfXUQpZyB+qk1dOIBFflL+gnV8JWJvMy3wd50hDo+KGg3ZQTUtWlq3sD3fmSzPOFvazVoamCw3y7TL1sz6d4Iv5+NP20lK9eoqN6VsNBylVzrbkBjVQBVd8t+eOBa9ANTZO0meMHFm\\\\\\\/3Rrrt9GG95kzjx8f96L3+2e9cbd+7AgoFR1NmcnLykGiKziKqxZwtBaeT3h3CfbMRKNyg1QrJoUC5BUAv3Hib7v4scO+41AsyWPr2k9bU1xQXTfKgvApAbSJgM+VHV3EypYsdr\\\\\\\/JMDlX0x\\\\\\\/aoOdSMPlQZ79fQEJH\\\\\\\/bf7TGDFqS243m\\\\\\\/ru0S3V5RIsmIvF7GQJkUOkJtx\\\\\\\/bmCadyK4NqjAzeBu9NIgKT3Ag3pJiTUjTRlG94qFocQRGmx7IbT8E28IlzLlvyTHSVfyHEAVQwI9kRRmo\\\\\\\/TPsqweleniG5VkR1mhRhmZCB5h0cEejEXCllf6txKkqZ31ZmFg9oXfozEfPMFVX87s7cIcRu3olLnwgOEBb6oPUIzjEwwoUT5qgZhd6xqiWwbT2h2FQ2LcgIyFrKe6s+NT0T\\\\\\\/4cH9ZuR19lmp4RbfxodvJ5+Ie1Ttc44\\\\\\\/S9HZdDDu2EgA5BTUKZPr2SmLXG3ALpYeEV\\\\\\\/VgcPzyJKnVuXeiTa2KSI113Q36+3nobSKVZFl0GNzzop3cHMkdYI\\\\\\\/qSYQCSf6IPV5DA87x\\\\\\\/rN16OxkOw84\\\\\\\/wl4rCBhiBoS8DtHG7CqudVBQX6pPOPhlQQPTJrSrAYxzQcwdXXLzwjOmGpxuMaIaIZIDNu3Ulp3pV9RLJiqORv7mh+La8Ngi6zNFsjh+1TSJ3xBkEY51ToFKwdtWBjePYuqu2UXnxWeW23Dhd2iySo6U0oYyMh2uychmmgT\\\\\\\/wWdl902p0y\\\\\\\/ht3LXBQzFmkh5pH3p5RbN7HO7D9hAtUMZRVj+nSDM8OEgl4LPtMIkq6OCNL+W2M8qYD+7guHXjqt5MmIJNpOPy6\\\\\\\/ynLIxtJR9kvc4C5IhPWuhcZ81\\\\\\\/AG1hPyngqW4fVmFZqDU4jSSKpH0tIFMBATKMpvOYK9rv2OAd7OXJTQIXmUmMRYt\\\\\\\/0eWPEs0Ax+UYoUTkx8S+kdrAgNqtTCsqwHvDjri9tiXjqGnXuFN4QAQsMgCC4jzHyIdrZuKkxnj7YuoTIRGTcfdgeCAVv6zT6c5V1T1CS8koem1Zg3hBZ5cPJQNlSRcFeqPJMbjmiDHHd4J3eD4s3Onnp5UNAFe4LOhmFUANDXjWwY3YNFvxvTj4ecu37dt78kiL2SpxJRSE9H\\\\\\\/Pq4466ojVhimbLHgS6SW128BQBjWdAVFeygXvhlIp8YGwI\\\\\\\/hoIXLXNvzthc6CdGH0ONkebPXaMycKEwD9YCBEhr\\\\\\\/6tTh6zRbUVuXVhu6OBhinxmbzSwlm+A0T4hZBmKAFBDqyVLXtamwaiPI7Gws0h+Fsc28snvxZ\\\\\\\/xu5FCDjulbjmniiWmGvK34NZuH4O+h29rqqpz2IdhAU4IHBh0++qpO+dI9zHn6iiEukV+b7B+EFbPB0boXwN4gEwVQOo5DiFzZWS+xS0UWWfvpFxvlmJw0Hnv1xrI5NdUe6jJTE5Wl6d2xfAYmqySGl9bzoT4oTcw16WkT2UmWwC4ZqFhQEk08WC0Yu9nHk\\\\\\\/97S5rP2SMAgCGweVmexcLRj5Y8EIgBS12oT8fkGcAk\\\\\\\/vrSpmqYaBEH0HW1cPSeWK\\\\\\\/l0WAZ8p07OzuNI3Jx1UJ+YZEqZaZpbOXh4xRuKbNbuUE90Bj1rhBIrKgmmNhR973Y4BGXjACzz915tEA+Q+hBno4Kky3XSBjF7S51Vp5Kepi82eamLBwtw\\\\\\\/xcCK2KNtpvMRu0btGh4ZMHxmzyulo7Oz3jKIsA25tQpuIIbGsLgusNZ\\\\\\\/hAFLwc7AnFzj7lXbP\\\\\\\/EnQGDNaxThriewXIbECYe\\\\\\\/ZY8sMhgOH8VGG330K8t5lM3sBIFe2fzWINENAm4km1dDt7+H67vLjelAo\\\\\\\/o9LeLAqu+QCbwfMBk\\\\\\\/Mf1cWJof6c\\\\\\\/Cs3hopWJk6VL+7yR2dw8MJHVN+mwGAy4ujBbv6NBpHhwYZkPLtu3bZ4mYZF1Ww3piaK\\\\\\\/4f0YFAkhI2MASMLzxxXt+tn2fC92My0ap6yWZobSX16j+YLM6dxgl+cBbnniJgpQlSmAEus\\\\\\\/bGil32Sv3X6d9bxC5jMXrLtrFWBDDNy0bMVj1rPPxbhcS+w5zdZTYnqfLgyt4rhVmpmhOub7WU+g8Ji6lbe7n3D6OTXweumq+XfVv2x4itFC1bi17pfEcU1b9zZItR\\\\\\\/xKbijXqdHIyamBwoWj9ojPrcy0VIRUEAXz37m\\\\\\\/7r+cd2O2bXb9cQ3c1gtZY3hjN7bZEDfKNaCdgCBN18N5i4EPmD0IWXvLJIE79\\\\\\\/eVIyrIp7mMKnw8gakSKMmKnhkx\\\\\\\/Aul7NmWKmfLO5jIaxIUe\\\\\\\/CchExdpIhh\\\\\\\/0b4YSde1Np3s3Twpb2AU1ITN6t9UH4czbEHqyXj6dUkTrAvu2gF3Gk4GJOCOtMHwyiSnOz+PvW\\\\\\\/E4+DxN+FUQc9OvzDjy5whJU1HpKYEHNIaywmCGf0DNhqohPpCFUs5v\\\\\\\/bW+QyQIJHRC8TBdpezQY1xm7zDyqA5T6ihKwqpRYlujPS\\\\\\\/kOgUn3kh2DbpPgndpDtG2cRpXYS+XO2StyVsT0gF5gR6raPFGDaHtpvvZFEO3GIOQ5RJDq20aohwynVXMxdvUVF+T+7zY04r5GnA+zM4KfZMqsN0nSC\\\\\\\/F7TR5xtywZkg559s7T6Br0pIcIqv\\\\\\\/oKq\\\\\\\/a8AOlb\\\\\\\/whxhAadIdTXBtxZG+dRaPYS1LSktii\\\\\\\/yb5Abq4AZJedx0npso1cmSELGXp5RmMqZm\\\\\\\/P28OGXXW9GMWyKJNPp9t4uyVR\\\\\\\/tzaW4Hf+8uJbT91gbR8OLWoi8hd9BE6BuramwMEyrd+NTcXA+kZda4DI6xzuhdMsNpp9mE8EtEPkF5jY8agAUgydjw2Bmch1v0kzdLHjVFi3sEOrp5wJX8nJMANkuq7E8HNLrZY8EfJZMfSFh7qvGPcqtNeAcfKpFVuZ4JEO0ItMGghfCBINSEfr4Is4srRMiueMmWjOGb6dUfg9GOpJ\\\\\\\/p4Mu4h3HcIRZQ8nb2k+QUWkrqerforlZM975s3B3CENxrxB+KTt8o+2K7oyHYztyd7+9c7upPK9wIV6xO4mfCrdt5pzMOlt5uxFP+P9XJQSwlDfk1QIs05OzH8zn\\\\\\\/UO8sewjy6b6rNfMgeKS5wjnlVMW9LLMzKEjsbRo5jBR7yETmUJLixa579rAZHRDUhBjz\\\\\\\/8t370mmq8QU3aMVTDsqFli1uR8fsniOxmQGH05aaP92ThvGzyQq+oE4Dbku1778ZNZqDfmUFTk8LORoFCy3AZDUFD12hkCgZUpfaTzyhVAc4SvRNOB+NaVT0R+T8PL1JRaJcYQ\\\\\\\/RuAJcNmKb0uDpTPUfMCbSFIQv35\\\\\\\/Hob2yuYoCdtx3PVEq9I6M20\\\\\\\/lRjNnIp44OskYugteC\\\\\\\/Jf1NmjHswMvxZcyaNLuZEn7GQfqIHoADnoFV0YgJTz3rNql7SSPctDeNLs4FOs7NmJVUjSgVp1oft6EEts\\\\\\\/C11\\\\\\\/AtfoX9EKfCiUDYvAohocfKjgL\\\\\\\/68TUZG2G63uRyk2ZThI5HqFCJ116lhzM2H+qnV6Stvqw9GbPUBmvbG3tf0MrptkS6KDh24TpOI8q0pgf5DruwYB1gSua7wqGkN5zlkjr\\\\\\\/eEUKr3UdVPZcAPwlp5NH8Halm\\\\\\\/D2CJZavY69JjIShc21GsoaOA+KIl3zEU7dxellQdEo4gYFZ32iQSEn4aXcPb5wCT5qXs7\\\\\\\/sGha1nMmSKErG2szPc5pHnLkf2dT\\\\\\\/rGmEGvMpJv9pjjDe+d0\\\\\\\/7hBsDKzLIv9o5gFv8aTyHKnzlyPuMNP8V4t2opYXViYfvfSaY30z6Z\\\\\\\/10rACtnmIULjjPoOyzaE9YRNZYDF5aD\\\\\\\/lKCOp62jDH1XCNoQvXUUDXHodvfuGR9OwTxn8xIyM5vWprwLL9kJ3qONLlITauKUFCG48m6j8O197GOjQeJGkwfHTLvWT1vEdLEyFnjRkV2RmmFWGpKRdGUDg67qH9yzHo+w54CeDeskx9DO8p7NLlPwiM5EZCHLuNPVqmDuwXCbqnF8kiTCsYd7RvirLB83DDpHnDuB8cbIYGGWXifBGd57AtknOguPIV2iWSJTSiVSKZW8jXjLOM\\\\\\\/TMhFPBiD5gyEawck9RWabvR4pyiKniS08R1w3yCH11FD\\\\\\\/K9VYyDfDTcpZqm9T5z7qYRXAl8C1ivFdBlEydPSu3sieBFdGqC4EJg7eYxC8dPEDdgZrvLsW4y5xPeYcl6rBtmQylrFaIMTkb9yYGowXNHa6VegnpDYZZ4Qo9rxEyJl19AoXvzwGxixDYIR9gUE8\\\\\\\/eVoD1v9WyAkQdONCw6RtIVQDFCF+Bl26zDCtXUeADthkcuGoNjCrMJ9k43Vn4M\\\\\\\/rjDCzpyDhjWeTkVXIflMoelIKC+eZuApguaqLsD9UQU\\\\\\\/zwEuWmy3TeDU28yzAdRqI6BClOpsWGnNRmpeSKctQdHbVQBMcVBfVN+xUU1IrIfdJd24fGVpVQ1ln2sFfzNP8\\\\\\\/lbRPT5ydsWNMauuWAlKHaYlAbmiDmNuZNA\\\\\\\/jgJvfuxfRbtM4uwSg2pj6KeAkawKac6ye3+9KXdnFzstfrgNGU+cLuYg3uJWz2BVtKjoIBiPxvFOSpf6AEn6PsQ5mda5+1lXs6fEEPB0uiZprNy3XkTREs528sxAnXLhsM4V7FXawfYWkqDfF1xXFB004dcLpXCYiVcP\\\\\\\/d5z4c0jNUryLoZvSNWZzkGK\\\\\\\/+kg+19jGQSFX59x78DDuiVeDCTvtebQ07NigFm2BKfCPOxjdQJHp4ZicvDc3UZcBkUpMS5BSI\\\\\\\/jxgoXvYlqQ85EfRSTcag\\\\\\\/GG5X8jaEdrxR8ofnBA6LAmhKUoe+W1Xuy26Z5I1JfjDhifLaJW+dxAKaMjni6nQwu5YaSdXj0BfPf668yWfDudXlsltumhekgMvZ7cH\\\\\\\/cb7RCUu\\\\\\\/pvlmYQeNrLxD9GSpiRBvCvi1dMVS+vMzChVQSztCuvZmU10+2TmY+lAZrvP1zFvUDLI+8eQp0IIBmcF3u5KPlfh7i5izSj4kMxlAzt0jQcqMaCU5WDG8OKfZpgI3Pok9U4cNXp7CH4Ryh1FjK6l6r5kPr+Sb8opgVgS71uz4qSkBjEe0EWPc8xchG41QqHYlgDqI1Xf1uUnOeKmHRzaI\\\\\\\/xODv2MOnaVRwhinHtFn4nTFFyBUlKHNpNjqGoak9tngvSubO5aLqL\\\\\\\/JrDePpmQ6Bho9T857c8mHJDqlqJNxdNKKMsXg6nB1WvM195EiifqEWVbwX993kLoXrnNU70paKTaBK9KEH+zMwOqt0YMTas6WGApex2IVal5vzhKLsHQkHYTH04eduRNvmxzvnAQASXyaa8gVlO6GjpNTFAVsx7eBM7xrQaV1LI37SwIpSTb0ii8dqvsOhkUGv3fukmW6ffj\\\\\\\/0RekVCHUzfM2TkkaI5dyJQbMUCd0C73ZMiKWfQI83+aa6Qk4t8GC5fnjihESdQ2krUFF6IGE2CnxnrH1VizanZfRYgHMrLYxHDThCxaRDog3pAvqaZ8Ko+ZDIWb\\\\\\\/Waw6T60XwEz9ls54rHsjKsKYHpdQ+\\\\\\\/1DNvI3CFy3olf0YBn\\\\\\\/KtafOX27rdmSUBvn6ujnD4C3241zbirU+OUv7YHH25K+nZdp6owy7SL5HnGPJfEJIDvhQQj875Uc7lHjE62L43zPDdIDH6zKqKw6Nu3EjzTRuBl9M8SOv8zb9\\\\\\\/R9wfFWU\\\\\\\/5woNhoMe6HaxgUPc86ZmuKs2u+Ob81l37eRkYAeUoT72ILq8j5Kt2FtmBBojBRNekk0xmhnEzUYOfXXLR7WkqoQ6yky85B8EEP4przFHG\\\\\\\/PbgtbuylO38PXEv6mrwRela0QJRs1ioQ0gGm\\\\\\\/88AIAykwtqK6F2jBf8io4bT4x356gK1Bi64aSd8cP0cNrbkfoZToNT9p8mfp2Wqk+cIT9p\\\\\\\/Og6NED1J2e8AUdXtnyC4Sxd1HvUfWQPmX+pLEg2BAZqgJy4BzspKF3unjnKM\\\\\\\/vqGokucmYsuwR798WLfPGI45kheDp5ILdQDDGThdQyR11koB6Ng+DsOqHL08ZTlKB06YEa9wx3CXxO6RiWHS2qENy\\\\\\\/1bGr7qA1jW3CLTznY7SD4YR9vln6Z3dj+jvSb14PHPFRyyj58l56zGwB0lGH4qAf+MGGCKnC0TBCUEHQuuzW6TMwhaMBbzvMzxG\\\\\\\/2BIwSNnymTeKGOMmLoIj\\\\\\\/Vb2gd8UIuFLxuLbdEVyFll3EExQiSwG+4jmoRSUDVKQfe6b+kw29m4jy+V4BhR9Ou\\\\\\\/DLj4y1NVssbBoOkIY1TX32GyX\\\\\\\/DMJhIUTCpylp75S9tuNRmCo2O+q0B5iNPu0OyhI7Y9dRkHP+g3ByXNqw8ObScBW0z+APmwCc1qNronsb0lYm8bn9GGPyRJULqPh70lAO1eY1pqTbcVrIHR66kOmCTx6cRoWTH0F0KWIDb81tsc5iTd4GWTZE5Ekmlzl1LVcj7ZF0B1gt4GsYMJ9z0FiaHFuVxXORBTtu1njjyopOm70sb2XfEa2rwfFJQr0KMKfHwSKuAAPtdR5XF5uGNQl1I01\\\\\\\/\\\\\\\/qUTab3LwCM39l0uhOwjpjZVEDdPgqhQJktdCVvRUDEQtXmIKq04NWnKdG6pHCx6XLrrgJ3pnVvUUuJ5le7S6bpU7+0fhyl8hyhGkC8e6rQhTqxh8Gp2EGueNQ7HF0BLsKp8j3HozOmtW3ROxYpRRBX4GqF80hWQNpIcSThWQFQtFaj\\\\\\\/zR8fQdubT\\\\\\\/KWhW0\\\\\\\/+PN4u6hOzUXHOi\\\\\\\/8eSrBQTVwN7qh05dTBuZjXmgYF60tPHJX+2cWtbhora6ZTXrihkNI0O2hVqINec9AwQBqdHtVCW1n125BR2KKIj4RSQvm2JteoeKM3kbEbhIRG8Ig32B1yDOARejKczVtvvw5uqEHttdXpjaT42jRovpwZ3Zm4zPdZY9\\\\\\\/CzEuUsdpLOZKKayNlRQhNl8IlcQNpG7tswVxJnahYPrSpcNFFtGLp0L1ixqpaa8cOs8WxQxVy8KpEuQ2s6wBc0Tet0Y7ZWmoCI0UkV\\\\\\\/Rf0VekoGljhYRUyGkrGG3Te3W1e9B46RUe9wTRkeKkqQee8lpoDtzwtTtDQCNS6VuHQ7MDh\\\\\\\/gbAYRrfJzfsCEoJTlD8IzR6UKkujmfUkSxGwnwU\\\\\\\/0kSI4KUJfTedBdGtPZ1ZOUbQ3oDqKEcnbZf6PKev\\\\\\\/xD3MVulMYcZ\\\\\\\/POoypWTIg+5b3RzupwZ13ktUkyIMwY6QARpkbq46+MUtgFoHWwMRn58j4Oi6e3G2DIXRKIF2IscO5imvgPicJEn1vKIf5NT0xRfssjIp3GiUtjTWPZv83zNqMbAnzsj3bCb4\\\\\\\/Z\\\\\\\/ORjKR2CdXWMV1n2tUFUp9tOiihOcNmalP1Fc+KHBiVdEY45\\\\\\\/voWxwJbKBaCrioGY9RuLOFIIJ0smSE3nJkoY269I7j\\\\\\\/QEqMdDhW3Bo6KOtvECBS2M+EfrvOwi1UC8jATkpK56OYXMNu5Vaz\\\\\\\/AnZMYsnHGbRQxUGbGVwU2UXZkHSUO7Tm+qYp17eJCtB8S3raX7g0VK5vPUA3Q00axZgjrH5FpMpa\\\\\\\/LupVtyrvKYI3lM8OIwVVSnhnatuTDyvH+JheVNC\\\\\\\/QhjdkoUpVHx3nv6MBQrVqhpJm26z+24g+Y92e1rTLhiKGxnMcp5za2xnvswKqfxrSZe4YcQWT1lSGyOXUQQsoZ5BIdTVjxHcRr0k6gUhqOEINPG2UDYbXMyd3Tv9Po42zDvJIgAR6bkpiJ57Gh\\\\\\\/GeaatMKzCq\\\\\\\/j+MQYwtaeJI3qr0jFzdHtReZUWw\\\\\\\/\\\\\\\/eQvoQIy0Y+Xfvf2wpwyjoNuQqcG2P87yykHLm4ZaXbwbs+M7oBzYsyM3e3RlvzFdDv9LTtjPeRLxGfqaZ1ESoQHtynRbOLXwwqq4pCOqiwcSzsk4mjWPYTQ\\\\\\\/wqR9wV3akp2hNe0TK5D9RIAieA\\\\\\\/bPfaF6wpLhWCd\\\\\\\/YVQjbB8vcKNAkU\\\\\\\/Lw6syiVYiDkABlvYqSoZVsrokL878X5Of1MHUP7D62mf+N6E\\\\\\\/Eeiy9VJjZ0StV3e59PO6+e+w2TNXpxA5Ga5ICpkQVvVCLc7GlTXt\\\\\\\/3JRJFiQNTmbjl9cYslT+8cVOHMvECkJ04ZLl1kJ0coHRjGtA3E7A+K6PIAlD0kpFNU4PYCQFIOPS+xmrs4cNvzOf7QH0+rvqvw281rQ623TWwA00Bdy6YtYnOXJm+TbXKtj+MdKMTYWwNG7UJdNw3UBq0wbNQXBHGkFB8JrVc76t4wLcuiTbgRJw9QN7r92XWF2s2dc8xFYFZ3gcvVbPy4J\\\\\\\/LYYEATU95PsaemcpjI4yYnu9s1JY9b9rWoHPfhARi9OGLIRWZRUbGiG3zkMLKMFskP4bBUPZTuDWNERjVkLNg+XdJPtQ5J6ddrTLDulOF3q7LTapghGnLjZLFKgc2uF+NVjO9eXjJWPvI47uk1nLugyp3mvcDWPpgU1VscOtaN3UYxwUlKGRxpBTFtsC3diib7nNAnEaKgHctrjgEzaOrU05V3l6G9td+hyjSxhwFpZLb1Ixs9\\\\\\\/4Mb8PJJEQZTtr\\\\\\\/zSrLg18KeiiGKpKPz67eTqI+hBezJ3wLfH0dCavC3yvuUZ6Fld3R7IzcwuhYJtzfqAOO8PF6jXj4LOh0JL2rX+XuS6CCOrSLcJLZX6wOkq6+VlSr6eSs\\\\\\\/hi40eWvK5lAHspMrP8EksLY\\\\\\\/Pd4XoO7B3lxL3gDmzBh2OUtzOuO4GQXuajZo8wQUZG27NE+y6rL2YJ\\\\\\\/AH2ciIuma\\\\\\\/kjg5lbqV9XJe3SaNlqqEmHKxbdQHPdeyPeYuiYtALYJ4LUWjdidYyLPMtoFddBPu9YUv19rrU+J1YjhduWz+sdalEIjRG8M5k2Mne22pqrKP\\\\\\\/tdTa7GLaLNCIuZhCt83F7idk33Svq+ILtS4xTwVBlCpNA9uyNKafglznNyheiaBcNaRduzbMgqF8JzgIgn8a4ZVWcPIMbWY70PqEVMhPtIt9ElFltJQjapsXcUHdzE43BsmyUyjYbfRfvIISipGaasVnXEMvHF+3IKK98xpGyf9AJHxOQNX24FSj\\\\\\\/BY9zv8cB9VEyUhFwP5mUHseNh\\\\\\\/JxBEC22fTAVH0K37ZIlCJYSuw5ky4V1R7UZ3QghPxGmiAo2yD10qJaeDaJ7\\\\\\\/BYFKxhKs+wakwiJuA251UrEauwBkcaq0FTkkDdTydskFbvVtqnFqoxgP0wzKCxI94qzJNLNV5gRS01nitmWQwZnv2W5aflCZMVGxw+NFl\\\\\\\/Pa1oCWTNnlpVg3BtwY7ffh8fT1g4OIsELObTr04uhwDqj7mYrM2LKrQnNIFI7FSqjY4PHWHuL0352hJ5ByzCbQwEa6pWUrZ5bB35nZkJ+wGX8A7PeJX7KglTfELHyUbFUUE+xLFBk83sQa8ASdQfXRfckhE3DP\\\\\\\/6lkeC6elVbO3HXYUGZ2XW7HLvKH4qqsRUX5xyETdH76r4362TucfIA+hFqK34xgBFRMYV1ogzBmGm85ztlLcxWfvBou7ZaLRBQp9q0RWsTiVZDaky4fAqizB\\\\\\\/Xe8H0KhpGvWfrxgQKAII68Ys2YwUz\\\\\\\/mruYvC+IWThTEwn+gwMFQvkYpgOikNSHRZrrL\\\\\\\/eAts+OQ9kafdXtRMaaYJhHXXVKy4cKFVTp3D3qtU7qF3TebccRat3GbkPFto0yrFXWdWuxtq4ZbgpnBEiJnELa+E6PxA7Imd5E4KpQ1lLMtguWZkaHsbnH1T2NLH0pKBg\\\\\\\/oK4qY2g6XRCZZ0wrt4kK5+GIIuRrfPZg+Wzv3zuEP9l4\\\\\\\/3UwAgdg9LH5+042dGLTRtVXh3GYe1gK6C\\\\\\\/WY1hoZa1NY3XhURU087KpsYETcrWHFzkMwKxCdwmfQHS2gUEwpPB\\\\\\\/yBUXmeTaIhLOusD+Iq1422Mx9NXXl+hvWKNuBNms9q94bswlmikGyBFfx4uLBEnxFpO73bkPrVwWWJB4FazbB0I7+s282cg1WQQ+bbyHcOO92UEcLdTDkdrWxfXys7ZBOJp1MbJIwe0eFVijGqBtzxkvUOd6XGHWxeU9vePMUA3wT6v24sysBQ7GKbe7OEUVKVGS5xpos0MM7q1O88FHatt7t8B9QyLwlXwztLJ1EidUhaSbWWt\\\\\\\/OEjAVcavuG7lsdsCXT\\\\\\\/KeUQpqTMrQNw8zQ9GPu5pMEgq\\\\\\\/2EBnK+D7+cw7wcjXW8\\\\\\\/+oACddV+WWVJdqsT+CEpM49AuB49NSeizxQdPrcEHIpHxHBOkldPj5kmFlw4bpg17n3AzUZkUnbk2\\\\\\\/z7zyEFkIR0dFfpLccd1rmU4j\\\\\\\/hICb7SRZitoAvQiIj5odeM2ENRnhdytU5wHLSGYC3Nf4Mu91ZqoQIgVciUDcKoITnFftNk\\\\\\\/cuG47ZQUStlo2HUDjaPYOg3ggvngABQVdIREDZEkmUUNiL2l0huUsAYI0PvNz+8heNshxWoeb0Kbpn\\\\\\\/XMGoe2DcvbEJjh8YKddcXQ48yZtOLhzECLisMQ37Urk03WLmrz81LhdFypSn+CIRQag5IkePwrSvucAyN1OCJCVCKlLp6POckhsq7IQEj7tSDQXCOdB\\\\\\\/tmZStB9Aot7WA2\\\\\\\/55yqChrsYNTkyfN08o0vM8LRnloC2y+aynu03LhewhdYs6G4xSCRuJyCvcViqPN5AqlRhuB6FOomp03X3i6qXnvtT2ni72vaZaPci9Dk1WnBoTSdDdp3AnMLP9v7J4AMbUMDA5K\\\\\\\/FsxmAzYrmwXKv4PCU4EDQRakAgpjtk3Dfqu+uFcFF5dUAgqjuqjQfxRMRJho0MB8qn1\\\\\\\/4asJfcLliYteRKI9yq\\\\\\\/andAIEbPKGaFO79m2XH\\\\\\\/zJsKf4H3wDYUYrleoCxCHRUWXORGCNMy9P5s5d1LTbHxyfPYSOEy\\\\\\\/XjffugyWxnDGQJEdRXbv1HSE7065DE\\\\\\\/pGhegUMVVyydG8KYtMEzWaH\\\\\\\/1MegyBP2Gy0j5ekNa7cqCPWMqMFLuDvSSoWdlACerZMc+7431mdZuA\\\\\\\/o2Lx9QkQfWIjNJRAflIKm1sE\\\\\\\/V4UzgJbHMUsuSXf0lnZTt\\\\\\\/u3PYv3CJJESqQEu+3LkXtoHbCKZadBjjSgg4AZ58EV9KRO2HpCUWLXZTHZU3JgpWLAlTVKfU8iOUx+3OIBgYaVj1hBVdky+Z5RXI45PZSGkSGl1WLZz+5tjVbV8GVGY9C8cJM3rKn9GyIx0hUWvOv8Hqsn3FdMXgERy4co2+dYXIWGex3WevkAL\\\\\\\/OqkHkJ+CrgluNpoEPS0hfT+42ztf4h17UK7QYksnJ06C7xH07GSi9jDlXNABOGwJDA7x791VG\\\\\\\/UvdiJrhEqqvMw2hoBP1irh0ASpvtqQoAUvFh+9F3B7PYop7GUFScJRQvu0McWEFv\\\\\\\/RWl+rnYpcBpnMQDMPGwZoMShdhMm3eV5CewpmnzgLvnlsZx1HLyErB1wMVZRLndSM55viI6624PL8rcZE1rqhFhl+J9oRe3B6xHiA+R3pCbNz\\\\\\\/e6xIWIdd7L2wZg2sl3V05ealAweVtr6I8K5hlebR2zxqONczCaTVLNswQZreWGC72vHZL6OVOLOlN6D5TX19IKrehldR0B39K7jdg8k9BPcwOKVqEnSVJYC7k1pt2AcC\\\\\\\/o3Hn8rmo+7lChUbiTuDZ0hvqPEC9pseOyUGaGnsCdfbyRqitgDDxWmz3nmBTOJ5TKRrkpRwcUXp5K9OAB8oMdJYw9DYRzaulPWuDmocTeLZ99nDbhtKHe6tGvnGjrfKtuQsu43ubKB5KDKn2lMbb\\\\\\\/4ZB1smE9s3S2W0h551uQfnv1Ypx5+gSMPX6rH1SqKnWh3VI1vnBWplakjUsyhWRMDPs93fuOuokClj0mbZnGT+itimLcOEvjjZuP8VecZiHNmJ26V8XtbAoferE6vrFxRsTaYaZr8RVG9jU5QAOzgqj2aVx0TJObm8mSGlkp2jD8rFP+qO+S6gADWk1d1raLDT26aDdPcFe66tre8QyLFH5csH2Z7MUIlYjYWkPJjhBk4toD5wVpOKWDg34\\\\\\\/ZQIWNmK1+\\\\\\\/A9r0GeBqDlg6VzZWl3OZgSfLfAvUqtGk2iGo0jXxxCZv5+IU\\\\\\\/J9lTLz8fLxoRMWvrEnQ6iNCPnXCOnP2RSOqAn1M6MbBTk65NXA15qCHX1Yeg9Rmtwy7wcs7HlUTLS1Mifiy64oIuCtO8B8e2Y1IT2pq95tOQPs+SZFUwkBaqtkZFGFKCn7N9bGGcl4MeY21O8xubK5c6GltpOeYEBRHoWNKhLGldBKjKSX9LZJuvz6FygOE5YpcH17YsnEWb1zYsFf+9ug0JwcwGF3v\\\\\\\/2aizZ97i3ISu6IURGQQ4h6mCChIDLElQyjZ+FHSz+1RMbQ8db02mEHwnp55gcPxpM\\\\\\\/IDeY809LsM7Q4nDA5ez3qIltU8CpaIBAlc9rofgOkoFOqkxhlOH9h0scYie6jNKzVYESkeOMw83tFlCfRAH68o7V9mky2rcX\\\\\\\/Bka6dennJa7vWPc\\\\\\\/sM8cMUVZTlrDUk0D\\\\\\\/7DcyhRHQzQob1P\\\\\\\/zYN5FBGX0HhDTW28dYtrxkpzlzco9mmyjC8R3g8OHIA24YgcWf4D7QwxusCMXrY1\\\\\\\/z2OROXz2IO0kErcnzeu4XCo220ne+drnyowcgl2TUVfp6bx0CcaRjXogBnPP4jBYyC\\\\\\\/dOvBW0evNGvqeKxROxGUrgnGUx\\\\\\\/\\\\\\\/kbsgkhvWZfya427D53yV1GZJ4yI5LLSx3l16eP0QWAbFOg0SPEgX3Xus1FqekHLlFbiHE+PJG+pe27E5u8fv5thn0C31q6YbRKiqUDJ4zG4I\\\\\\\/alK40ZUjL22gluRxbXl\\\\\\\/7Tawb7EkVDaAbkoYT6HUCM+lop7U5g3pehZausOq0cOLaNwgI6\\\\\\\/nvQyqCwCqz0CC4AgxbS\\\\\\\/hB6lPjzOVcpn9mscdHPGMhw0HMEhUU\\\\\\\/Ys+AI6xcgvlD+idWGrJ+uBpql0SJIb9DzVUJSQbQOOMhMZNFWZAhN2GXwPxXlCpatgJFjlaOdD58uHNEN5WZQBajhF\\\\\\\/hyVKRlxNrdVHJFjqhnvHakLO1ax+MOJtgFkgAqZwr0rP\\\\\\\/98Npu1Wfg2SvxxHNKerA2UrWILEi8k98kWTNyAuaPnSbVOKBp3soxx51DML6c3uPiaI+WA2yn56SABMSAQ8Ol3Hkuvq2x9TvxCj7S0YETKuXlA5V1vtZsBaPmTvoIdlo+B5Wvr1Xv2hBEkOQtXms3+c09ZKobJOIIdfmpUt4JK\\\\\\\/lF\\\\\\\/pJcY3XtDFnbX2II2n5ASVWqiGTZ+Xqdt9RgGt0AbGgYIfZ88lKtp\\\\\\\/0Nh8xZNjIFESShrr3ePejeWiZk7lXR1fuxjTDn74ZEMe73lkjEiKkq92BKL91N5kzNuEdMSUZe1lqusfEWxKE8o+G\\\\\\\/2WvAdWfp5mpXQEJv3jWNoAzgnFtx18v2vY155cmp7omvRGUFs7qYCVvV2+7s7H0FY17wlcCTGy7R6Q+UVDPqtiqxlw+bZKYRYaby93pmPbyHzZfj76pjEJgd+ek2VxBleSL1zuisXtxxwnrRFxF694HiVzP7tTgyz++4ruON+pUB622+sKCNbhs8W2HvXvoy7RuJQULdLHZL63DuRY3YUqGXgZ6vu6s\\\\\\\/Zd3KiN\\\\\\\/+wwn13ymqdMZyGNmBgdu6kEZ+R\\\\\\\/md1wFJA8xxZWDJXxH1YwIB6jW3GdurxauEsnZlozFja1SW79fD5U05W6oahgm2RcA28QlcHhSJNgEfl+wBCeWaYm33IPzWJWdR7ztdqFz7HA\\\\\\\/7okudc7vRDkV3OYMlVrO+hhBfptN1u3I5N3V\\\\\\\/SkTzzJrhuWmrcFkJ0iaSn7ngHlbsdpPIEeMrONfn2yVQEaVI28Alqzp1I3FPkmJNR2EPJjKPueF\\\\\\\/ic\\\\\\\/NDLbTDF2V6hy7fn9L\\\\\\\/5fr10+p1oVZH3GHthEyyC47TtqdGOWqgRPhA8WzRprXYin\\\\\\\/MjlmR0KnpoPUm699l+Z+VMzuJoPy16S35KnXBl4kjM98P5Ucerl6NG1FOJqzcKMLMRhL+gX3HjPcscwcAL2ccEONGMNr+2CvQKPys5DD1dcYLUkt6j0AU\\\\\\\/Tq0z0GfQaHCD0G\\\\\\\/dj7X1lGTm9zIjbGqLo9OFqdZfVr1zgHNvdsSwX+eCrfzu71JZa1IcHn9o2Pm2vHQ8g1RgOl2CMNqYLMXlWxuBnI3PWjF3qCrcUuCxo\\\\\\\/aC\\\\\\\/N+fdO+gt8BUWbB9KrqrmHpkoCsv7Z5ZYnRCxLlVcmgyopk91j0Lq9FQA6ImsrbRIsW3bb1FW1EZqWHz62mqTbMWMMjJa1Xu0upz0o4VcFyI9JCrixOCjGQ4RIjim6jARGu6bfzSrYCSNJ\\\\\\\/Nl70yMNmaSb2aaaREhqOpDgP+GVEwOZnnyojq3+Wx5EirtIRY085m5BXfTMY5Z\\\\\\\/GaSOSNwOnoPWVNOtyL59SGjEBlblW4Arnrj7dUavWMgWxdWjUxyl\\\\\\\/qLHGNPgkAnc+yuf8p7QNeotgCMdFDy8jsroFjk+PtDQEkswBX1a+wXbfKSJRSALJmDVxW5j7qhl\\\\\\\/k7ngZE3C4NDQ4ZLzxbRCex74lqd406igHxrBjACGErZUpMsh19agMxrcqhJlUP0BwLvTxvcw\\\\\\\/SIRfkz5v45x+zladZLhyp0xvrsRefQXq8NkOtgIetREy8bVdGIL7MkHRRGHa\\\\\\\/539OO\\\\\\\/GnEWAvDRYpz2nLAW6w4IvjjS1zqJtcbOICMQwBR7iQLPJYXwWehYGia3JDtiC0Enef1wTID+IxoFLeL81RzeLRBwH6wHzeHJgLLgW1bziky7CqNW17zAKHK5GjQ+HXhj5DjGrryDgwe6CXFaUKvRErqGiFdxvq54CHITXRvLFrlo6X4ukPss1mLllKFkNM+HKhZgYNMt09aK3rg3jxZ2pDYla0NSZtiK\\\\\\\/zHs6ODKyizZ6Qjq1IoZo63FysNPgd\\\\\\\/mEAUharS8vDkN0kKNIeeUdmPh53\\\\\\\/C+J8diM7B4GplqPqNq7lwieiA+wLBVnWLekgNtWZG2J6czonygLs8I7+i7Rz7Vq1BlRyUJ1zfPPgRvLsVlAltYy5F5bKTx7yjFdY0xSE+p\\\\\\\/m3PMKKAfF+sslMUrcG3nZ6LMEIC+90VfaiThP9go0OaxFt2+koVslmvn+O1uyNDQPqgdUcY1SJsHrm5IexopY7Y7SETDJZIOfVvN2PjlqD9M43LFzKBD0HRsMrKENAO67wfXlyAyYTXqZW45w+dZWxMuP9N8xqVEV5po\\\\\\\/VhTVVfB3ZqqFK9xYqUlAqyN8r53biovDFKhE5B5xxCD8cdx6qXpDvxpV2OTd6h\\\\\\\/Rtg8jAH+MGKwrHzKPgZh2S1uh+\\\\\\\/mCq5e5F6u32R2qghMdGwfXrfrUTneVVmFo1CEaHIxwWptUY29PVuNioH4dvVS31qeurnNzRo3ojug1P7qPKSmyF5FzKMjIgYx\\\\\\\/dDVrO\\\\\\\/+6dANiwI1fOTyO73\\\\\\\/DqEg1pMpj1VqrLOHrTXTJm1mwQvWSaoHW\\\\\\\/2ULW7TDB7GVMAw4CteD\\\\\\\/6C+HXz3qt7sIDY7F9njeuxnwU0oLuUeXEyANEWMlzcp3xxHlmLupqkpu\\\\\\\/Cds60rxn26fCJVtHA\\\\\\\/jSJXLTN\\\\\\\/zqgi20ST2lXyY3qoZoX4o6wIfFxW\\\\\\\/4JVdL\\\\\\\/QQlsaqtzRcgK5hsjceDUXzm1ihLfrW\\\\\\\/kHsi6IxtOg2Nt2MsfWlmsyRXD8uJJoQfNU1pkj9B2KKxWKJ3mHiXCwINPFm2Bnpcknw+8VLqFYfhB65u0wnxRodOsvgBEqkAegb2pqY6poibVTPLHjMPBBU894DOUQBjG7n19Qs1qx0euc9HzEHhc9R7ThPPR0Oos1VdM6625iNGGis00pAWfYLd3TURTHsmpGtmh8f\\\\\\\/h7yfHIzpeX+m1aHne2sM55gUsl3YFW0cFWdMIc74xVA2WICHnyHxOU7EnN+2q942Fzvjav+humzzy3sqZSSAIKXUFFgY7icVOK0hghG0L1Jd5lG29so8L4q9uTnqPhKdcDtGWCzdL5ujaxWCgbwsmm33LXOUaHm7qjWNMobmFEApMw6LMZ+FVqB\\\\\\\/BhegPywyZTmVCdkwEOYkrc1e6JQ1OdbBPmgb\\\\\\\/lIBERW\\\\\\\/MjW7eRtOcN\\\\\\\/8Rrl1wgDDv5eaxy94PrAuEQfEqS3xrdeOqm6j7tqltJBSnKfimHBnZXS9SGKy24vswiulrMoCBr+yJh8knsPy3QcUqmr2uN4d5l8T64HyHsX1AnqT8voXDO7LAFu3c48bsQb5jlVm3erzsWN\\\\\\\/fhJAOBDjS25CXNfZqhKPRfqBs5ZBRaYR8THNxxitzuJVYEgSrPXG5RsQ4QLW9y\\\\\\\/NwqNlOtec+PnaYFpDy8FMPdjND258pD1hNY9s83FNZ0OCQHasOzvhIztkyDeHnhorUWbD96HwbzZO9a5uFjgiW4Llo76\\\\\\\/u\\\\\\\/fZqzOl00U0rC+Xx1B0dikHgZV3nAtOTw0noZ8weRSrSRnu+G0I05EbLXuDyDnnJva9gRRQ7YdiIOpkxM144w\\\\\\\/om8+CP0I+hcb9y7649LrewvJntp0eiU8dCYGxSwtU+NIeOzE5\\\\\\\/bbM3Wsm6S3l\\\\\\\/ygiSRMMvbSDkYPHmMnhU9c4C7HgR\\\\\\\/Ilnrsftt1re3q1l7mV2Mt3s4pdwfMYFXKnxIliwRSeKbiB0EjWnDl\\\\\\\/TdzgF9VHWwhAKgL9s19YUeSXxGSHvg6GOYJIXK1JsaWXMvpcLgrGX5W63xuimp52Vw1sM8Reht4Bh3Ofq09dRD\\\\\\\/NJk5VHArdd7trXgFvepnOLNmKbkSegvYhZhvuWfPEp0Pwz5UEbh6ImPlS\\\\\\\/4QAJdK6ZWMeCeclYaq3e0OU7fp4nmA8ctBzNf7MD\\\\\\\/huFGZUcQG4s3cWi5GqgIwEfA0i\\\\\\\/Ca66qwX5G6rGoKgt2ab1uwSaxpQo+XJDQUxWeyJVHjVR2GHYr2t6ppsUkIq49Ht6g4OVAcL9yBj9V8Cp8l6ibRJNkTzS1cNrXfX32uzq+y5NS4XCl4m4\\\\\\\/JhYAp3+QcfmOtoLfVFUt4aIvXnxjaeKYdmLcE0UmS2wVwkxNJ8gYZ7x6r8ElACkHCGVdjc3EyVFV2RQXXLDG7\\\\\\\/6tZGIY1hFo+5Ihy3SYW1FZP2MDT+aM\\\\\\\/OpXr00T9S2qZO\\\\\\\/ITZz\\\\\\\/I\\\\\\\/RH4A8nZgPur5rlnLmrksk\\\\\\\/3Y+4\\\\\\\/QwkPrqUCCOnQVwD5D1DKvpil2kkmcoTdWV2kTTfDxhouqd3Bq7WlwU5pTB7vmUEnh62dTM1PkxllgHcAlDaBeQDzWihP\\\\\\\/+wTmdwbLnJgW661Dj\\\\\\\/WPqYt+SWNyLrNp6P2ChprlsfiDH5leJvLDAF1VJtWVDVHjVl4rlAdZmO0LMZMxD8CENZeUDPCldR+v3WzEO\\\\\\\/diczOQzaNy5AFsgKu5dxKfalsAvaqfdDvqySIa3shsA7jsCe4qyzP9\\\\\\\/hhwG2ejyAt5skjB8R9P3+cAFMErvGV5iQhPLC0S2ly45ROWY7k5VP1PhJbPZyGIXm+fm1IGSg41395SfEzu+M33S227VVxtIsSKMu1EG53f54u25HXXJlqljzDad88nEJzM6lsIBojgdR1gr3fyx11gwkhanYBfpHssyJT\\\\\\\/3RrD4IzPd4aKY1iMed5AjJPlITjvcSHyteXhDtQVSE29NynYFziv9+liQ799B+ATGmKKSkjtXw3TXKYv1dvysw0GlIjS6j61Xa5Zx7U+3VSWoHe8vHXFdDWfgvqJJ+Pze3hVcceaARMSHrtViU2ID3kPOTQM5J9lktatzZJxZmNVOwuRoeb1ASZ4LgZDc3ZpkOIjj\\\\\\\/m6HvQvS+AIkRNwRRjZAIZYon5Sb3VJ0tG19G7i+sSTJU3M9fC5HijIoeG7iUhtSJJgQYvO1466JZJUGwWawh+RiKIzylMeljcr1OGzNyRSkmyWBQso9brdrW\\\\\\\/Kklr1sdpRcHq61\\\\\\\/GgXdiJu33JSjgcuBbD5fW47Sgl4A979dyNiCSmC2Tn8bceOEN7GJ0idGD9H\\\\\\\/8vNHMZFnNmWoloTmrF9TtyzoqWu9oksfJuVA3KABTdAVcnPdd2IJbv5eL1K5YazO4TRDvAfGi82SMyjdZr5JGKWL+rQ+kq3iElKi\\\\\\\/1NGW8HSrIedadIhRn8wFAwLaHa6XfaLTSht\\\\\\\/bJhL2QJ5nUgtREXfsOwGxr39u34+MzTT89muwzCdiKcC6S3XHn6XiH6DxVbagJY7QN8m8W7Ds8TmdQbF64JkZoEF8nj6aRxlIv\\\\\\\/EdLl8x9vk6MKW4pc6VTOcUB0ku1kEQ1rjgjd3YKFTjQsOpMrCUkJYbfWXz4I2hEY7CO4Y9duxyP0eLCe25hJxDMZzBhsIIBaGzhen9Zs69mNYmPqPd8t668c7xG4HwrGJQXT9zADaav2JWhFD15stAuA+4oZGzDLeLYU+AmfY6Qr6UAw3I1VszYAt1jFkOx7vDBMQaYSN7wdqfkTTB3LOXrjEQGpMS\\\\\\\/td\\\\\\\/Nm0MlXUfWQGseowigvesuAbODbuethEqoLTX+RGxMcPZk8IWpVmfrnbp6WtKSWC378ywShDy3jjhOHsnM\\\\\\\/CcZOEdEcHGrsMAnVi+La98mgH6TwCbk26k+kBN2AfHCwsKbgOPBg1MP5XwdY0V0YtARDgdGttkGUWXKSnG7WDNFyhiazanFbTcjxDxLFb1Un56wQ0arv7aCzSohD+acg4FKacv+rWs2joJ\\\\\\\/EMkQsgk6PfOfMuXyxmeuBvM\\\\\\\/x+QOx+AwuoqOxgeWPbVsrbVEfubAPrBucCM3GeTbpt8mlHJrMMeBguDEeeeAql1QT324rp3JZyTaMbXOZ5Xz2w\\\\\\\/U5K\\\\\\\/YhdsPZ0i02UJDODmUTuEYqag999PTf6mXAtL9kEANoqjDQqpymakSrdfpsmw9l\\\\\\\/CZKdXFgsZx50FucpG1dT55mHOr6Msp1mB+cNQWvQw7md1ZdFt3q1gtv3db7t2Ve2h\\\\\\\/8j5Ilm2hMrm8IdbzOOCwMRi9qNg2\\\\\\\/zuPDPgPjkoei2iOZ+ubOnu\\\\\\\/E4i8o8+kYqRjnhLFsVq3rQUHDlGvMBKU85ndHa3ud+2IrhzYQBJW\\\\\\\/4e9OWMoJxpZd4Rh2LpYWreGJNRs0tCaTR5MxNkhfUw33lCYWqG6Y5BJl\\\\\\\/edFggoMV6TxCadr0Puj7f0eP46h5zZudKTfFLbit0vQm8F9V2Djcxj9SOoThcGGgq4SvRPoLJ\\\\\\\/yB0rTROt7UOV7khbq\\\\\\\/blPxaezGdUslerT0QVG7WbP\\\\\\\/2P8OzZPw3C\\\\\\\/BQ7tWDFZTk\\\\\\\/tBiH3tkVnnrHbrINLSYjBNMsSR6k4eMWzRbIWEkfEcw\\\\\\\/bO\\\\\\\/RBhhQi27pSG11lxgVT\\\\\\\/Fe7MpqPKPOOq1T3SaD9105FneJHKVsRpxR9FPxDZhV9KgOhzzwVDja72XwknKHYXALKvhhPGXDzYCHhly84f8Cf0DCjq\\\\\\\/+HRMWwvENRiEWGII7AnqHmdS3evdNadfl+\\\\\\\/RtgSFF4\\\\\\\/tXIazSjOhHjQqGUCBFbBdYbKtNHTi+ZcpradPer5NRF6dDf4pcXf7bndas0GKEUhjBzQJqxu5HovFUjpY7I120ppHKVzjx\\\\\\\/NubynCQ64dgN4xqiYaDd51KXrV\\\\\\\/uApbYQkVSWvQaHTEYfY5Dyv+inTqPGMfJ1uUfEZswxQOUdBs9tjwQadWVZZHFMypHaOrNhP0z\\\\\\\/TJozCuueiZyy20Msn6xYVZkeePcp5j7zXV57mPs5wyBzIg7tXIkTdPrwhF2I\\\\\\\/2NhEV0DWoL5JKzTYQmlZJZFAyXvkLkxyUNTfc0O5HO3M4hmeSl2t8ZKMSrgshM6LeWZsNNcKfOylwWNu9HEY8+8AprijWC8JL+h0teDQ8FVCCGxQvkuquRTykLscdHyQGlK0FSnM0q5Pg8tP6VJ\\\\\\\/JPP9hbRPtXr0+d\\\\\\\/byPQYukxXkiCk7a4pju4ntBTVfIQkHLVQwmyB+\\\\\\\/ZEN\\\\\\\/Nbd2KRR11s4bCQzZVVlBiR7NJmAPUWnyFvGXlDM4qNrwcXUKhoEMUaSg2HC4Heb3sRYrf2zRUx\\\\\\\/okAqVBqjkqKeaz7AipuzDD5arMJ1Ntg1+B2zrvVIJ8VvLTxwp969RCYuuge4lv\\\\\\\/Ti7lQnBz0UCYs59E\\\\\\\/jASy+GJ3Ey9Lmzv438E0K2prRE4WADSoWo49+8XJs5THLD3FztrjBLEkxLBkomVMJ07bp0DcbO+fS4zhhV1aoZoQzdDJRFiD16n\\\\\\\/WfufHCgVkm2l+w\\\\\\\/CNOEayYc9r8KOXYMnwXG5E2t0+CPERLEvQOKsbOoFW5agQleuJr3Z7AOfUygCbdrZulB8rklPd49IrEpXXfJ00o\\\\\\\/2VUoiRu3++XDN\\\\\\\/C1ZMYfh83ZREdi8rRoa6p5LSigPzjXXsuhslWZAXRrk0cdqV4UhlEdeHOTU+NyP0PXwru9ENuksqYmaKcCBD6caQBdJ8IND2F+Zr+LTPTiD1RidKh1SGxaPdta793irUb8R8+75TypcBX4LvHu+tmSlMe\\\\\\\/jQ96uE+9R5paWgw1fSfkKUyMiv9czz8hFHnkh5EpLJ68l7Blhb7HVncKgR60Br8LtfDCHPUmIXPvBQxEdVBs5cTA3e0kSqVh8JqE\\\\\\\/A4lMKuoLeBic2n6Ow3RiS0V\\\\\\\/4BRLBt4DnT0UUzFk544iBctKLp6niAwfcPUMMTE2IpqeC\\\\\\\/y+1eKFGNYDuRcq2q3hPr2r4gQrGF0FF7XyOv1p\\\\\\\/pdPR7zzkjV8gHU+RAQWUlTYp7TDE0H7Gm8oOdIbg3FFfsY\\\\\\\/MgHI2+iXQ5wHruSRlXjkleYBfyBHfQCuZzhql9oDllysr11OFmMq7H45Q50mxTyFHB442jAJ\\\\\\\/W+PodlNtyZDOMj6wJdEfgr19D9wda2+dgiPdN0uCacH28fq+bgMMjn8vs6tot8okGbREItdqrx4Vlu\\\\\\\/z0SQD4fI2cUhD2k2Qm1\\\\\\\/ACblwWCV5NPPz+Q8trMH0N82RFrbh2pcbS1PciczAIyh5cqxRrXXtWYho\\\\\\\/cmjivraoSbfa0zLWZJt5cTSa\\\\\\\/qkWahjJl9PhivU9Bpi6wp1znJQnJuIQdZhJRdrrtcuQNRPnYLfHmR6HkDIySzqnzld\\\\\\\/6KQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LgWlKsRQu22m7DVVZYx2Gw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223fe30f68ddbb029182f7c89bfa75e688306ffff660ca6fe9edbb6b59a41175ba\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RkWxpd3rapoDblWy9SxkFrgDZ\\\\\\\/R2yLmKNKCaNOOr0m0YE0\\\\\\\/huG+Qw3ROZW2jgkHcks6eLGSM7\\\\\\\/kxnBJTIJVAkTqPDIEKEastxL1SsMXHmoswgDnXdHtiTs5FCtN8lzZNMFZ96DT+dpViKYr1I\\\\\\\/KgJ9TNcw1gqAT1kQE89GxFzPyId1Wb6CUWOiiKrq1reOogwI8ucL9YlzZyePFD8vZRxLTOwAIBJrEVMg7Q9z76pd6Z5s+J1JAzM+2IIztNOjwTKa6iBZWylUiwEZUocV3xxDCZkZxqwjc\\\\\\\/wsdWgQjSWZ2oOBstDbpR5Ae19DE6JHeTZR\\\\\\\/5Asmo6ENj7\\\\\\\/DL5KNaz0iiUGIUZPMEk159stHmEvv8+lAf0bz\\\\\\\/1XWyFdUTUdL+cupFGWGJvxNWLDI9IZmf+FXZlrQXnFu\\\\\\\/UeYjGQJNSOAJi4ZPs9Ufv3SaMtHmfcZ+kPs7mDI4VWKClZrp\\\\\\\/oeVMhRZZv6TxBS+yX+dAPi\\\\\\\/i4dxW3Q\\\\\\\/wwgX9Aaav\\\\\\\/5jUqFLSDkyUVA3wVF5hdsihrOjQtV9UeSfqOYMsKlJXT7LKKJwCFHsIuP+oO2Sifi5ig+RRr812Wt713NtFk1M+rIM7XHUIIE8OkkcNTh+nqZtgPcDuISTUIArhVWbkn7kn6KpfkvB4Q4kknzqqN3Um+KUAMNj+T7Df6iISfaUNe2oDrIsHJHYqViPPQvP17aHH5OGoINpLu4tfSuGV1l2z3QoxEkOSkNqdO5v5c\\\\\\\/z3QpGz\\\\\\\/YGCrgkkLUTOe8xVSr7DkYtoG3ZA9Na2h+RtDQJHdroIee+7Dxa3yzCzthJGYd5d6E2QBkEKvhpRRkRDHDRLpWQvJqY\\\\\\\/o8oPkJGSrOvLjoCitXxDaoPq7TCFZkQ3KwlOl9eQzvOSpxy4SGk5f3ilwNlKNCt1MgZti9w28bkIBeQWSxy5nRHYq\\\\\\\/U39h2oag9XNdRSBO6E4ED6kgt\\\\\\\/JoCL5tVigU\\\\\\\/03nAZCeB2VG+MaDEe1F3lzEQmDxoR4ZQgB+16nKYKfpOMeB2POijFHVQ7FTMRGfWzJ0gkr9l5onRREY86mLO1PY0rxqjqgX5cUQs3\\\\\\\/jGvn79pf05MZUIdn3wg0ulSzho2ITnmrF+8KGRUDjJc\\\\\\\/7ODoOJhgyNL\\\\\\\/ukWFavaAWQjeUj7ryphlnBt7dZ3eWBOZUTkYi8RpFcvJcHVYThMbiYcywLBJIH+1i0jkwJI5wmz4NqW77of2yUMHu9nqjBqG4Zs2e6Ua\\\\\\\/wDOe5Ma8S5UW3VwhNqf\\\\\\\/QknW9bu57n8Gz5W4qUEIHeh3t1TfOgtbf0JpcH7qkjGpREXM2pXUkFShVAtcw77l\\\\\\\/KNZxK3YAUS+BKWsnbPhCG9tFVQEuqx1GlvLZwzqJTALhC7WzS2OqgTJqXNJcxarl2RpMdVPi7KYkM7SjYkqiO15+LhVvFG9wABTkgYNjDj9yetdOicXNPZR325\\\\\\\/iTja8RICShoHcwHy23wmLj1rhqFtstOT492c98gNVpQ2IH\\\\\\\/cdFwiyMZN0qUN0VMDaZGcQJxiQhCpU1SExm5AhBzFXBRvB4piOhEPniOVI1wWAibnevsYG\\\\\\\/bede4gE5o9D1vXlM1561ry4B2sdHZn7dpLenPhUOnXL4NwnsUPfkNjFbnEOLQWhx2EXAYIYz+\\\\\\\/CW1LOhS7Kk\\\\\\\/OA\\\\\\\/DdNUO0Nw7ys6zNWGDkXBiDk8jyl7zbQLMKgw7cmCLo\\\\\\\/p01qJJm3O9aw+SILNZDaQSi49K\\\\\\\/OsUI646ilHMjDmx8WDywLw9RxJhJcbHSnhGsW6VuIW69lslaeAFuhU00zqlIxPQUW2r+WJuQUuGa9tf1Fav+ir9M0ykVMR\\\\\\\/\\\\\\\/oqX4RRpmLQjFsth+hqUBneQqJ6Gy8IBpt3l+YDZJoKdzBj3b1wwGR+nY6Td+jU9Qf9xRyS70\\\\\\\/r02lzI0lzvwgrgRTHh35hC5GYhEpmc3MDIIj\\\\\\\/r0AbX5MFgudhhryDWBOfma\\\\\\\/V01PbkzQf\\\\\\\/lGTk2V3tVKSaTNEeuf5mrMrKumKQOwhG6osZU3LtdF\\\\\\\/t4n1FOW52P1yw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022a8abOt0dWBZamt269izWRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c1de0288644b7c6541cb48868106721903631585931848d41c9c7116d0cd301c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460171 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2307" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DjuW4qTgy-j56n_wRItsMh4kzHx2p37xEU7YxVuH4qs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2307" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-O9tRTRZUo-GD7mzYmM7BOnq3C8GN6FXzewD7pAtxpsI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2299" title="Persistent Activation of the Innate Immune Response in Adult Drosophila Following Radiation Exposure During Larval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2323" title="bz-rates: A Web Tool to Estimate Mutation Rates from Fluctuation Analysis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460171" id="node460171" data-pisa="ggg;5/11/2307" data-pisa-master="ggg;g3.115.021402" data-apath="/ggg/5/11/2307.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Carrie A. Whittle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cassandra G. Extavour</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2307-2321;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460171" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carrie A. Whittle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carrie%2BA.%2BWhittle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whittle%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarrie%2BA.%2BWhittle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cassandra G. Extavour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassandra%2BG.%2BExtavour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Extavour%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACassandra%2BG.%2BExtavour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cassandra G. Extavour</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:extavour@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">extavour@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2307" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460171" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2307.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460171" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2307.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460171" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2307.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460171" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2307.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460171" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In protein-coding genes, synonymous codon usage and amino acid composition correlate to expression in some eukaryotes, and may result from translational selection. Here, we studied large-scale RNA-seq data from three divergent arthropod models, including cricket (<em>Gryllus bimaculatus)</em>, milkweed bug (<em>Oncopeltus fasciatus</em>), and the amphipod crustacean <em>Parhyale hawaiensis</em>, and tested for optimization of codon and amino acid usage relative to expression level. We report strong signals of AT3 optimal codons (those favored in highly expressed genes) in <em>G. bimaculatus</em> and <em>O. fasciatus</em>, whereas weaker signs of GC3 optimal codons were found in <em>P. hawaiensis</em>, suggesting selection on codon usage in all three organisms. Further, in <em>G. bimaculatus</em> and <em>O. fasciatus</em>, high expression was associated with lowered frequency of amino acids with large size/complexity (S/C) scores in favor of those with intermediate S/C values; thus, selection may favor smaller amino acids while retaining those of moderate size for protein stability or conformation. In <em>P. hawaiensis</em>, highly transcribed genes had elevated frequency of amino acids with large and small S/C scores, suggesting a complex dynamic in this crustacean. In all species, the highly transcribed genes appeared to favor short proteins, high optimal codon usage, specific amino acids, and were preferentially involved in cell-cycling and protein synthesis. Together, based on examination of 1,680,067, 1,667,783, and 1,326,896 codon sites in <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em>, respectively, we conclude that translational selection shapes codon and amino acid usage in these three Pancrustacean arthropods.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/gryllus-bimaculatus" class="hw-term hw-article-keyword hw-article-keyword-gryllus-bimaculatus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Gryllus bimaculatus</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/oncopeltus-fasciatus" class="hw-term hw-article-keyword hw-article-keyword-oncopeltus-fasciatus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Oncopeltus fasciatus</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/parhyale-hawaiensis" class="hw-term hw-article-keyword hw-article-keyword-parhyale-hawaiensis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Parhyale hawaiensis</em></a></li><li class="kwd"><a href="/keyword/optimal-codons" class="hw-term hw-article-keyword hw-article-keyword-optimal-codons" rel="nofollow">optimal codons</a></li><li class="kwd"><a href="/keyword/translational-selection" class="hw-term hw-article-keyword hw-article-keyword-translational-selection" rel="nofollow">translational selection</a></li></ul><p id="p-3">The arthropods comprise more than 80% of animal species currently living and are a highly diverse phylum of exoskeleton organisms that includes terrestrial insects and aquatic crustaceans (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akam 2000</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Odegaard 2000</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Regier <em>et al.</em> 2010</a>). Despite the diversity of this phylum, research in genome evolution in arthropods remains focused either on species of flies and mosquitoes, primarily <em>Drosophila</em> and <em>Anopheles</em> (Diptera) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Neafsey <em>et al.</em> 2015</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Stark <em>et al.</em> 2007</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Wiegmann and Yeates 2005</a>), or on a few other holometabolous insects (those with complete metamorphosis; the so-called “higher” insects) such as beetles (Coleoptera) or wasps (Hymenoptera) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brown <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Group <em>et al.</em> 2010</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Weinstock <em>et al.</em> 2006</a>). Much remains unknown outside those insects, and a broader understanding of how arthropod genomes evolve requires expanding studies to include nontraditional models (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Zuk <em>et al.</em> 2014</a>). Emerging arthropod model systems for genomics research include two basally branching hemimetabolous insects, namely a cricket (<em>Gryllus bimaculatus</em>, Orthoptera) and the milkweed bug (<em>Oncopeltus fasciatus</em>, Hemiptera), as well as the amphipod crustacean <em>Parhyale hawaiensis</em> (Amphipoda) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Liu and Kaufman 2009a</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mito and Noji 2009</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rehm <em>et al.</em> 2008</a>). Each of these arthropods has recently acquired genome-wide transcriptome data from ovaries, multistage embryos, and/or postembryonic tissues (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al.</em> 2011</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Zeng <em>et al.</em> 2011</a>, <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">2013</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>), allowing investigation of evolutionary dynamics of protein-coding genes.</p><p id="p-4">Crickets have historically been key systems for research in functional genetics, neurobiology, and developmental biology but have more recently emerged as an important model for evolutionary biology, including topics in sexual selection and speciation (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Garcia-Gonzalez and Simmons 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Huber <em>et al.</em> 1989</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Zuk <em>et al.</em> 2014</a>). The two-spotted cricket <em>G. bimaculatus</em>, like other crickets, branches basally to the Diptera, Hymenoptera, and Coleoptera and has a large genome (∼1.7 Gb) predicted to contain more than 19,000 unique gene sequences (<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Zeng <em>et al.</em> 2013</a>). An additional basally branching hemimetabolous insect, the milkweed bug <em>O. fasciatus</em>, provides further opportunities for molecular evolutionary research (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Butt 1949</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Liu and Kaufman 2009a</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">b</a>). Specifically, more than 10,000 unique genes, including those involved in an array of developmental, signaling, and housekeeping functions, have been identified, allowing comparative genomics in this taxon (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al.</em> 2011</a>). The amphipod crustacean <em>P. hawaiensis</em> belongs to the crustacean class Malacostraca, which is one of the crustacean groups most closely related to the Hexapods (Insecta, Collembola, Protura, and Diplura) (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Regier <em>et al.</em> 2010</a>) and is common in intertidal habitats worldwide. It has extensive genetic and developmental tools available (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kontarakis <em>et al.</em> 2011</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Pavlopoulos and Averof 2005</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Rehm <em>et al.</em> 2008</a>), yet remarkably is one of the only crustaceans besides <em>Daphnia pulex</em>, with whole genome or transcriptome datasets available (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Colbourne <em>et al.</em> 2011</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Zeng <em>et al.</em> 2011</a>). Currently, few studies are available regarding the genomic traits and evolution in nontraditional arthropod models, including the evolution of synonymous codon usage and amino acid composition in protein-coding DNA.</p><p id="p-5">Synonymous codons are not used randomly in the genome. Biases in synonymous codon usage may arise from mutational pressure (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Osawa <em>et al.</em> 1988</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Sharp <em>et al.</em> 1995</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Sueoka 1988</a>) or from selective forces favoring efficient and accurate translation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Duret 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Stoletzki and Eyre-Walker 2007</a>). Findings that codon usage biases are correlated with transfer RNA (tRNA) abundance and/or gene copy number (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Duret 2000</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ikemura 1981</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ikemura 1985</a>) and with gene expression levels in numerous taxonomic groups (<em>e.g.</em>, <em>Escherichia coli</em>, <em>Saccharomyces</em>, <em>Caenorhabditis</em>, <em>Drosophila</em>, <em>Arabidopsis</em>, <em>Silene</em>, <em>Populus</em>) support the theory that selection favors specific codons that promote efficient and accurate translation of genes that are expressed at high levels (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ingvarsson 2008</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Qiu <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sharp and Li 1986</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sharp <em>et al.</em> 1986</a>). Thus, an effective method to identify codons favored by selection (optimal codons) is to compare codon usage per amino acid in highly expressed genes and lowly expressed genes (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Ingvarsson 2008</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Whittle <em>et al.</em> 2007</a>). In this regard, the presence/absence of optimal codons and factors linked to their usage may reveal whether selection for translational efficiency and/or accuracy plays a major role in an organism’s genome evolution.</p><p id="p-6">In arthropods, nearly all information to date about optimal codons is from higher insects: optimal codons have been identified in species of <em>Drosophila</em> and <em>Anopheles</em>, and other taxa such as <em>Tribolium castaneum</em>, <em>Apis mellifera</em>, and <em>Nasonia vitripennis</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Behura and Severson 2012</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2013</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>; C. A. Whittle <em>et al.</em>, unpublished data), although no or weak genome-wide trends were evident in <em>Bombyx mori</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Jia <em>et al.</em> 2015</a>; C. A. Whittle <em>et al.</em>, unpublished data). The bias favors GC3 codons in some organisms, including <em>D. melanogaster</em> (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>), but AT3 is favored in others (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Behura and Severson 2012</a>; C. A.Whittle <em>et al.</em>, unpublished data), indicating marked divergence in the presence/absence of optimal codons and types of optimal codons among these insects. Nevertheless, further data are needed about the dynamics of codon usage in arthropods outside of the higher insects, such as hemimetabolous insects and crustaceans, to answer a number of outstanding questions, including, for example whether optimal codons exist in these organisms, and if so, how they are related to traits such as amino acid composition (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>).</p><p id="p-7">The amino acid composition of proteins also appears to change with increasing expression levels (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Duret 2000</a>; <a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Several reports have indicated that amino acid frequency in proteins encoded by highly expressed genes correlates to the most abundant tRNA in an organism, consistent with selection for speed and accuracy of translation (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Duret 2000</a>), Further, studies in some eukaryotes, including yeast, <em>T. castaneum</em>, and <em>Caenorhabditis elegans</em>, have indicated that amino acids in highly expressed genes tend to be less metabolically costly (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Raiford <em>et al.</em> 2008</a>; <a id="xref-ref-94-4" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). For instance, highly transcribed genes contain fewer amino acids of large size and complexity (S/C scores; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dufton 1997</a>), and favor low cost amino acids (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-94-5" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Thus, in those studies, abundant proteins, presumably encoded by highly transcribed genes, tended to be comprised of smaller less complex amino acids, which is not only predicted to minimize the biochemical energy costs of synthesis, but might also contribute to the stability of the protein structure and its conformation (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Dufton 1997</a>; <a id="xref-ref-94-6" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Expanding this molecular evolutionary research to non-traditional model organisms will help elucidate the breadth of this phenomenon and the role of evolution of amino acid preferences in promoting translational efficiency.</p><p id="p-8">In the present investigation, we study optimal codon and amino acid usage based on recently available large-scale RNA-seq data from three emerging models of arthropods, <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em>. The RNA-seq data are available at the Assembled Searchable Giant Arthropod Read Database (<em>i.e.</em>, ASGARD) (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al.</em> 2011</a>; <a id="xref-ref-98-3" class="xref-bibr" href="#ref-98">Zeng <em>et al.</em> 2011</a>; <a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>; <a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">Zeng <em>et al.</em> 2013</a>). Using these data, we provide strong evidence that codon usage and amino acid frequency has been optimized in highly expressed genes of each of these organisms, with the strongest signals observed in <em>G. bimaculatus</em> and <em>O. fasciatus</em>, and weaker, yet significant effects detected in <em>P. hawaiensis</em>. Further, using <em>D. melanogaster</em> as a reference, we show that highly expressed coding sequence (CDS) are shorter, consistent with translational selection, and are enriched for genes involved in cell-cycle processes and protein synthesis in all three organisms. Together, the results are consistent with a history of selection on synonymous codon usage and on amino acid frequency in highly transcribed genes to promote translational efficiency and accuracy across Pancrustacean genome evolution.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Transcriptomes Under Study</h3><p id="p-9">Here, we studied RNA-seq data from ASGARD derived from ovaries, multi-stage embryos and/or postembryonic tissues for <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/021402SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS1.pdf">Table S1</a>) (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al.</em> 2011</a>; <a id="xref-ref-98-4" class="xref-bibr" href="#ref-98">Zeng <em>et al.</em> 2011</a>; <a id="xref-ref-97-4" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>; <a id="xref-ref-99-4" class="xref-bibr" href="#ref-99">Zeng <em>et al.</em> 2013</a>). Multistage embryos and ovaries comprise highly complex tissues, expressing a large component of genes in the genome (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Li <em>et al.</em> 2014</a>); thus, genes highly transcribed in these tissues are apt to be an effective tool to reveal genome-wide optimization of codon usage and amino acids in an organism (c.f. <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Subramanian and Kumar 2004</a>). The next-generation sequencing data are described in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS1.pdf">Table S1</a>. We identified all assembled transcripts (<a id="xref-ref-97-5" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>) without isoforms, to allow accurate mapping of reads to a specific single CDS to quantify expression. For this gene set, we extracted all coding regions with a start codon, and lacking any unknown sites or internal stop codons. To quantify expression levels, we mapped reads to the CDS and calculated reads per million (RPM) using the non-normalized libraries that comprise the largest (or only) dataset per species (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS1.pdf">Table S1</a>; where read number directly reflects abundance; average read length was 349, 297, and 400 bp per species, respectively) (<a id="xref-ref-97-6" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>). RPM was defined as the number of read matches to a CDS/Total number of reads matching all CDS × 1,000,000; see <em>Identification of optimal codons</em> for reads per kilobase million, RPKM). Tools and software for measuring expression and codon and amino acid usage in the various datasets are described in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FileS3.pdf">File S3</a>.</p></div><div id="sec-3" class="subsection"><h3>Data availability</h3><p id="p-10">Data are available at ASGARD (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al</em>. 2011</a>; <a id="xref-ref-98-5" class="xref-bibr" href="#ref-98">Zeng <em>et al</em>. 2011</a>; <a id="xref-ref-97-7" class="xref-bibr" href="#ref-97">Zeng and Extavour 2012</a>; <a id="xref-ref-99-5" class="xref-bibr" href="#ref-99">Zeng <em>et al</em>. 2013</a>). Details are provided in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS1.pdf">Table S1</a>.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-11">For our analyses, the number of CDS examined after excluding genes with isoforms, unknown sites, or internal stop codons for <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em> was 5284, 6161, and 6731, which spanned 1,680,067, 1,667,783, and 1,326,896 codons, respectively. The nucleotide composition varied across all CDS among organisms: it was AT-rich for <em>G. bimaculatus</em> (AT = 0.6177 ± 0.006; with similar levels of A and T) and <em>O. fasciatus</em> (0.6520 ± 0.006; similar levels of A and T) and AT was mildly higher than GC content in <em>P. hawaiensis</em> (0.5529 ± 0.0417).</p><p id="p-12">We first asked whether we could detect differences in codon usage relative to expression levels in all three study species. To study synonymous codon usage, we first determined the GC3 content among the 5% highest and lowest expressed CDS and detected striking differences consistent with selection on codon usage among these two categories. A highly effective method to identify codon preferences is to compare these preferences between genes expressed at extremely high or extremely low levels (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Ingvarsson 2008</a>; <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Whittle <em>et al.</em> 2007</a>, <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">2011a</a>). As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, we found that GC3 was statistically significantly lower in CDS with the 5% highest RPM (mean = 0.307 ± 0.003) than in the 5% of CDS with the lowest RPM (mean = 0.334 ± 0.006) in <em>G. bimaculatus</em> (<em>t</em>-test <em>P</em> = 2.0 × 10<sup>−4</sup>) and <em>O. fasciatus</em> (mean 5% highest = 0.306 ± 0.004 and mean 5% lowest= 0.349 ± 0.007, respectively, <em>P</em> = 1.5 × 10<sup>−7</sup>). These trends indicate that AT3 codons are more common under high transcription in these insects. For <em>P. hawaiensis</em>, GC3 was significantly elevated in highly expressed genes compared with lowly expressed genes (mean 5% highest = 0.507 ± 0.005 and mean 5% lowest = 0.473 ± 0.005 respectively, <em>P</em> = 6.8 × 10<sup>−7</sup>) suggesting selection for GC3 codons in highly expressed genes in this crustacean. For all genes per species, we then assessed the effective number of codons (ENC), wherein values range from 20 (when one codon is exclusively used to code for a given amino acid) to 61 (all codons used equally), and lower values denote greater biases in codon usage in a gene (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Wright 1990</a>). We found that AT3 was strongly negatively correlated with ENC in <em>G. bimaculatus</em> (Spearman’s Rank R = −0.65, <em>P</em> = 2.0 × 10<sup>−6</sup>) and <em>O. fasciatus</em> (R = −0.68, <em>P</em> = 2.0 × 10<sup>−7</sup>), thus concurring with AT3 codon preferences in those species. In contrast, no correlation was evident for GC3 and ENC across all genes from <em>P. hawaiensis</em>. (<em>P</em> &gt; 0.05), but a negative correlation evident in the uppermost expressed gene set (R = −0.20, <em>P</em> &lt; 2.0 × 10<sup>−4</sup>, for the uppermost 5%), implying marked GC3 favoritism within this dataset. Given these findings, we next assessed synonymous codon usage in every amino acid of the analyzed CDS, to test the hypothesis that AT3 in the insects, and GC3 in the amphipod, were optimal codons in these species.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The GC3 content for the 5% most highly and lowly expressed genes for each of three species under study. Different letters indicate a statistically significant difference among high and low expressed genes within each species (P &lt; 0.05 using t-tests)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542393386" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The GC3 content for the 5% most highly and lowly expressed genes for each of three species under study. Different letters indicate a statistically significant difference among high and low expressed genes within each species (&lt;em&gt;P&lt;/em&gt; &lt; 0.05 using &lt;em&gt;t&lt;/em&gt;-tests).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2307/F1.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2307/F1.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467010" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">The GC3 content for the 5% most highly and lowly expressed genes for each of three species under study. Different letters indicate a statistically significant difference among high and low expressed genes within each species (<em>P</em> &lt; 0.05 using <em>t</em>-tests).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-5" class="subsection"><h3>Identification of optimal codons</h3><p id="p-14">Although there is some variation in approaches to identify optimal codons (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hershberg and Petrov 2009</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2012</a>; <a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>), here we identified the optimal codon list in each amino acid in <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em> using relative synonymous codon usage (RSCU) relative to gene expression (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Ingvarsson 2008</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Qiu <em>et al.</em> 2011</a>; <a id="xref-ref-86-4" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-90-3" class="xref-bibr" href="#ref-90">Whittle <em>et al.</em> 2007</a>; <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Whittle <em>et al.</em> 2011a</a>). RSCU measures the observed frequency of a particular codon relative to the expected frequency if all synonymous codons were used equally. RSCU values greater than 1 indicate preferential usage, and higher values among codons within a synonymous codon family denote increased usage (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Sharp and Li 1986</a>). Optimal codons were defined as those having a statistically significant and positive ∆RSCU = RSCU<sub>Mean Highly Expressed CDS</sub> − RSCU<sub>Mean Low Expressed CDS</sub>; when more than one codon matched this criterion per amino acid, the codon with the largest ∆RSCU amino acid was defined as the primary optimal codon (<a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Ingvarsson 2008</a>; <a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">Whittle <em>et al.</em> 2011a</a>). The ∆RSCU for all amino acids is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and the mean RSCU and standard errors per expression class are shown in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS2.pdf">Table S2</a>. We report that 17 of the 18 amino acids with synonymous codons had an optimal codon with a statistically significant and positive ∆RSCU in <em>G. bimaculatus</em>. For <em>O. fasciatus</em>, we identified a total of 16 optimal codons. Both species favored AT3 codons in highly expressed genes. For instance, in <em>G. bimaculatus</em>, 15 of 17 optimal codons ended in AT, whereas 16 of 16 optimal codons ended in AT for <em>O. fasciatus</em>. In fact, for each of the eight amino acids encoded by four or more codons (<em>e.g.</em>, Ala, Arg, Gly, Leu, Pro, Ser, Thr, and Val), the optimal codon always ended in T in <em>O. fasciatus</em>, whereas six of eight (the two exceptions being Leu and Pro) ended in T for <em>G. bimaculatus</em>. Thus, there are marked preferences toward specific types of codons (T3) even for amino acids wherein an assortment of synonymous codons exist. The bias toward optimal codons ending in AT was also observed for amino acids with two or three synonymous codons (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). As <em>G. bimaculatus</em> and <em>O. fasciatus</em> each have inherently AT rich transcriptomes (see first paragraph of the Results section), codon usage is likely partly influenced by the inherent genome composition, which might be expected to contain AT-rich codons. Despite this, our data show that the usage of AT3 codons is enhanced markedly in highly expressed genes (not expected under mutational pressure; see also <em>Optimal codon usage is shaped by selection</em>), and thus is consistent with selection favoring these optimal codons in genes that experience high rates of transcription.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466969/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466969/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The difference (Δ) in mean RSCU for the 5% most highly <em>vs.</em> lowly expressed genes in <em>Gryllus bimaculatus</em>, <em>Oncopeltus fasciatus</em>, and <em>Parhyale hawaiensis</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-19">For <em>P. hawaiensis</em>, 13 amino acids also were found to have an optimal codon that was favored in highly expressed genes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). However, in contrast to <em>G. bimaculatus</em> and <em>O. fasciatus</em>, the majority (12 of 13) of optimal codons were GC3 codons (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Notably, the absolute value of ∆RSCU was of markedly lower magnitude for <em>P. hawaiensis</em> (mean and standard error, 0.0729 ± 0.0062) than for <em>G. bimaculatus</em> (0.1060 ± 0.0110) and <em>O. fasciatus</em> (0.1158 ± 0.0111) (<em>P</em> &lt; 0.016 for <em>t</em>-tests). This finding may indicate that selective forces acting on optimal codons are weaker in the amphipod than in the two insects. Nevertheless, collectively the transcript data for all three invertebrate species show evidence of favoritism toward specific synonymous codons in highly expressed genes.</p><p id="p-20">It is worth noting that genes with longer assembled CDS in the aforementioned analyses could have a greater RPM using next-generation sequencing data due to their greater size, and thus not solely result from high expression. Accordingly, as described in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FileS1.pdf">File S1</a>, we repeated our analyses using RPKM, which includes length when measuring expression level, and found the same optimal codon lists to that obtained from RPM, with the exception that P-values were weaker. In sum, we show that RPM provides the most rigorous method to identify optimal codons (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FileS1.pdf">File S1</a>), and thus use these codon lists for all analyses.</p><p id="p-21">Another complementary method used to further confirm optimal codon lists is to compare codon use between ribosomal protein genes (RPGs), which typically are highly expressed, and lowly expressed genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Heger and Ponting 2007</a>; <a id="xref-ref-86-5" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>). Thus, we repeated our analyses of RSCU using RSCU<sub>RPGs</sub>= RSCU<sub>RPGs</sub> – RSCU<sub>CDS with Lowest 5% Expression</sub> (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FileS2.pdf">File S2</a>) and found that the results of these analyses also support the existence of AT3 optimal codons in <em>G. bimaculatus</em> and <em>O. fasciatus</em>, and GC3 codons for <em>P. hawaiensis</em>.</p></div><div id="sec-6" class="subsection"><h3>Genome-wide optimal codon usage</h3><p id="p-22">Using the optimal codon lists in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, we calculated the frequency of optimal codons (Fop) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ikemura 1981</a>) for each CDS under study and found that this parameter was highly statistically significantly and positively correlated to RPM across all CDS for <em>G. bimaculatus</em> (Spearman Rank R = 0.23, <em>P</em> &lt; 10<sup>−15</sup>), <em>O. fasciatus</em> (R = 0.17, <em>P</em> &lt; 10<sup>−15</sup>), and <em>P. hawaiensis</em> (R = 0.09, <em>P</em> &lt; 10<sup>−15</sup>). The R values were each &lt;0.3, suggesting a moderate, yet highly significant, association between Fop and RPM. We subsequently binned each CDS into one of three distinct RPM categories, namely low (below the 5th percentile), moderate (between the 5th and the 95th percentile), and high (above the 95th percentile). This approach revealed an unambiguous shift in Fop, increasing progressively from the low, moderate to high RPM classes for <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), a trend consistent with a strong connection between transcription rates and Fop. Thus, although selection on codon usage has greatest effects in the uppermost expression levels, it also shapes codon usage (albeit to a lesser extent) in moderately expressed genes in those taxa. The weakest effect appears to be for moderately expressed genes in <em>P. hawaiensis</em>, which was only modestly higher than the low expression class. Thus, optimization of codon usage for the moderate expression level classes may be mild in this taxon given the relatively low absolute value of the genome-wide R value above, the similarity of Fop to the lowest class (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), and the fact that ENC showed an effect only in the highest expression category (see <em>Identification of optimal codons</em>). Given these trends, it is evident that the method of contrasting of expression among the CDS with the 5% highest and lowest RPM used herein, was advantageous in revealing the list of optimal codons. This agrees with prior research indicating that comparison of codon usage among the highest and lowest expressed CDS (known as the comparison method) provides a more effective tool (not weakened by mild correlations in the mid-ranges of expression) for revealing optimal codons than broad correlations between codon usage and expression across all CDS (known as the correlation method), at least for these particular organisms (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Hershberg and Petrov 2009</a>; <a id="xref-ref-86-6" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The average frequency of optimal codons (Fop) relative to expression level for the three species of invertebrates. Expression of coding sequence was categorized as low (above 95th percentile), moderate (between 5th and 95th percentile), and high (below 5th percentile). Error bars represent standard errors and are very small." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542393386" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The average frequency of optimal codons (Fop) relative to expression level for the three species of invertebrates. Expression of coding sequence was categorized as low (above 95th percentile), moderate (between 5th and 95th percentile), and high (below 5th percentile). Error bars represent standard errors and are very small.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2307/F2.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2307/F2.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">The average frequency of optimal codons (Fop) relative to expression level for the three species of invertebrates. Expression of coding sequence was categorized as low (above 95th percentile), moderate (between 5th and 95th percentile), and high (below 5th percentile). Error bars represent standard errors and are very small.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We also report that strong positive correlations were found between Fop and AT3 for <em>G. bimaculatus</em> (Spearman R = 0.62, <em>P</em> &lt; 10<sup>−15</sup>) and <em>O. fasciatus</em> (R = 0.82. <em>P</em> &lt; 10<sup>−15</sup>) and between Fop and GC3 (R = 0.64. <em>P</em> &lt; 10<sup>−15</sup>) for <em>P. hawaiensis</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS1.pdf">Figure S1</a>). This affirms the strong link between these traits, even in the lattermost taxon where only 12 GC3 optimal codons were identified.</p></div><div id="sec-7" class="subsection"><h3>Optimal codon usage varies among amino acids</h3><p id="p-25">The strength of bias in codon usage varied among amino acids. Specifically, the degree of biased codon usage in the 5% most highly and least highly expressed genes was strongest for the highly degenerate amino acids. For <em>G. bimaculatus</em>, the three amino acids with six synonymous codons tended to have the largest ∆RSCU among highly and lowly expressed genes: Arg (+0.277), Leu (+0.300), and Ser (+0.297) (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>; see <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS2.pdf">Table S2</a> for means and standard errors per expression category), whereas three- and four-fold degenerate amino acids (Ala, Gly, Ile, Pro, Thr, and Val) ranged between mean ∆RSCU values of +0.163 and +0.311 (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). In contrast, for amino acids with two-fold degeneracy (Asn, Asp, Cys, Gln, Glu, His, Phe, and Tyr) the ∆RSCU values ranged between +0.062 to +0.170 (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Thus, it appears the selective pressure favoring the use of optimal codons increases with greater degeneracy. Similarly, for <em>O. fasciatus</em>, the highest ∆RSCU values were found for the six-fold degenerate amino acids Leu (+0.346) and Ser (+0.359) and the four-fold degenerate amino acid Ala (+0.370), whereas the lowest values included the two-fold amino acids His (+0.102) and Phe (+0.118). Thus, as found for <em>G. bimaculatus</em>, these milkweed bug data are consistent with a tendency for greater selective pressure on use of optimal codons for more degenerate amino acids. However, this relationship is not universal in <em>O. fasciatus</em>, as ∆RSCU for Arg was +0.098 in this taxon (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). The trends in these two hemimetabolous insects could indicate a hierarchy of selection coefficients on codon usage (<a id="xref-ref-22-11" class="xref-bibr" href="#ref-22">Cutter, <em>et al.</em> 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">McVean and Vieira 1999</a>), which is greater in amino acids with higher degeneracy. For the taxon <em>P. hawaiensis</em>, the highest ∆RSCU occurred for Leu (+0.146) and Ile (+0.136), but the remaining amino acids showed no trends with respect to degeneracy. This may reflect the fact that optimal codon usage and ∆RSCU were weakest in this taxon, making differences among amino acids accordingly less marked.</p></div><div id="sec-8" class="subsection"><h3>Optimal codon usage is shaped by selection</h3><p id="p-26">It has been observed that although codons with elevated frequency in highly expressed genes typically result from selection, they also could result from mutational biases (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Comeron 2004</a>). For instance, this might occur if high transcription rates lead to a bias toward specific mutations (<em>e.g.</em>, C to T) in the DNA strand, which has been observed in <em>Escherichia coli</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Beletskii and Bhagwat 1996</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">1998</a>) and/or from biases arising during transcriptional-coupled repair such as those reported in mammals (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Green <em>et al.</em> 2003</a>). Traditionally, one method used to exclude mutational bias is to compare GC content of introns <em>vs.</em> GC3 frequency in CDS, where the absence of a relationship excludes mutational bias (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bachtrog 2007</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Comeron 2004</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Qiu <em>et al.</em> 2011</a>; <a id="xref-ref-94-7" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). As the RNA-seq data we have studied here does not contain introns, we cannot test for putative mutation bias using this method. Nonetheless, several features of our data point toward selection as a significant factor in the evolution of optimal codons in these taxa, rather than an expression-induced mutational bias. First, for <em>G. bimaculatus</em> and <em>O. fasciatus</em> the vast majority of codons defined as optimal in <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a> end in T. However, examining amino acids with four-fold degenerate sites (Ala, Gly, Pro, Thr, Val), which can end in A, G, C, or T, it is evident that there is substantial variation in the ∆RSCU of the optimal codon across amino acids. This finding is not consistent with mutational bias, as all possible types of mutations to T (G to T, C to T, A to T) can occur at the degenerate site. If any neutral mutational bias to T3 were responsible for the evolution of optimal codons for these amino acids, then such bias should affect all five of these amino acids similarly and thus should lead to similar ∆RSCU values. However, these values in fact ranged from +0.139 to +0.279 for <em>G. bimaculatus</em>, and from +0.194 to +0.370 for <em>O. fasciatus</em> (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). Second, the very large positive ∆RSCU values for four- and six-fold degenerate amino acids (as compared to two-fold) are consistent with selection (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">McVean and Vieira 2001</a>; <a id="xref-ref-22-12" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>) rather than mutational bias. Third, for the six-fold degenerate amino acid Ser, which has two codons ending in T (TCT, AGT), we only observed a strong signal of optimal codons for TCT and not AGT in <em>G. bimaculatus</em>; this was also the case for <em>O. fasciatus</em>. The presence of a C to T mutational bias would be expected to result in both TCT (from TCC to TCT mutations) and AGT (from AGC to AGT mutations) having a statistically significant and positive ∆RSCU. However, we observed no such effect at the AGT codon (+0.024 for <em>G. bimaculatus</em> and −0.081 for <em>O. fasciatus</em>, neither of which were significant; <a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>). We note that the caveat for this particular finding for Ser is that it excludes solely a C to T mutational bias in these species. Collectively the aforementioned results are suggestive that factors other than mutational bias, namely selective pressures, contribute toward shaping the optimal codons for <em>G. bimaculatus</em> and <em>O. fasciatus</em>. Nonetheless, we do not exclude that some mutational pressures, such as neighboring-nucleotide effects, could contribute to nucleotide composition (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hodgkinson and Eyre-Walker 2011</a>). Finally, for <em>P. hawaiensis</em>, where the majority of optimal codons ended in GC, the ∆RSCU for the optimal codon identified for amino acids with four synonymous codons varied extensively (+0.028 to +0.114), whereas one ended in A. Both of these observations are inconsistent with mutational bias as an underlying cause of optimal codons. In sum, our data point toward selection as a factor shaping codon usage in all three arthropods.</p></div><div id="sec-9" class="subsection"><h3>Optimal codon usage correlates to amino acid size and complexity</h3><p id="p-27">We assessed the cost of protein synthesis using the <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Dufton (1997)</a> methodology, wherein each amino acid is assigned a size complexity score (S/C) based on its molecular weight and complexity. The S/C score reflects the chemical energy investment, as well as the costs of stability in the protein’s final conformation (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Dufton 1997</a>; <a id="xref-ref-94-8" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). The S/C scores are listed in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>. For each species, we calculated the proportion of amino acids per protein with high S/C (&gt;40; Pr<sub>High SC</sub>), which include Cys, Phe, His, Met, Arg, Trp, and Tyr. We found that Pr<sub>High SC</sub> was inversely correlated with Fop for <em>G. bimaculatus</em> (Spearman R = −0.09, <em>P</em> &lt; 10<sup>−15</sup>) and <em>O. fasciatus</em> (Spearman R = −0.12, <em>P</em> &lt; 10<sup>−15</sup>) (see below in this section for <em>P. hawaiensis</em>). Given that codon adaptation indices, such as Fop, correlate with expression and can be used as a proxy to measure expression levels of genes across the genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coghlan and Wolfe 2000</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Popescu <em>et al.</em> 2006</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wall <em>et al.</em> 2005</a>; <a id="xref-ref-94-9" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>), we can infer that higher gene expression is linked to reduced S/C score in these insects. The R values are similar to those reported for expression and mean S/C in the flour beetle <em>T. castaneum</em> (<a id="xref-ref-94-10" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>) and suggest that selection for reduced S/C is not exclusive to highly expressed genes but rather exhibits a gradual decline with reduced transcription rates. Nonetheless, our data show that the relationship is not particularly strong as revealed by the absolute value of R (≤0.12), possibly indicating that the selection pressure on optimal codon usage relative to S/C varies among genes. In other words, S/C may be under relatively greater selective pressure in some genes than Fop and vice-versa.</p><p id="p-28">We further studied the relationship between expression and the frequency of each of the individual 20 amino acids. For this, we used Fop as a proxy for the relative expression level at the genome-wide level (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Coghlan and Wolfe 2000</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Popescu <em>et al.</em> 2006</a>; <a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Wall <em>et al.</em> 2005</a>), which is apt to be less noisy than RPM or RPKM at intermediate expression levels (outside the upper and lower 5%; see <em>Identification of optimal codons</em>). In addition, we wished to assess the relationship between Fop and amino acid in and of itself. The Spearman rank correlations between Fop and amino acid frequency per CDS at the genome-wide level are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The results showed that for <em>G. bimaculatus</em>, 15 amino acids exhibited a statistically significant correlation. The negative R values, which indicate amino acids used rarely in genes with high Fop/expression level, include four of the six amino acids with high S/C scores (&gt;40, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>) namely Arg, Met, His, and Trp (the latter two are nonsignificant), consistent both with the inverse relationship described above between Pr<sub>High SC</sub> and Fop (see <em>Optimal codon usage correlates to amino acid size and complexity</em>), and suggest selection against these amino acids in highly expressed genes. The positive correlations indicate that the more frequently an amino acid appeared in a CDS sequence, the more likely it was to display elevated optimal codon usage. Accordingly, the amino acids most favored under high expression (with R &gt; 0.269, <em>P</em> &lt; 10<sup>−15</sup>) included Glu and Asp, which have moderate S/C scores (between 32.7 and 36.5, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>). The amino acids Asn, Lys and Ile also exhibited substantial positive correlations with frequency in CDS and expression levels (R = 0.169, 0.151, and 0.115, respectively, <em>P</em> &lt; 10<sup>−15</sup>), and have moderate or low S/C scores (33.7, 30.1, and 16.0, respectively, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>). The two amino acids with the largest positive R values with respect to Fop, namely Glu and Asp, and those with the most negative values, Arg and Thr, are illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> (Fop values are binned into four distinct categories: Fop &lt; 0.3, ≥3 Fop &lt; 0.4, ≥0.4 Fop &lt; 0.5, and Fop ≥0.5). These results further confirm the striking shifts in amino acid frequency under high Fop/expression. In summary, it is evident that specific amino acids are preferred under high expression in <em>G. bimaculatus</em>, and these tend to be of moderate or low size and complexity.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467124/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467124/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The Spearman rank correlations between the frequency of each amino acid per CDS and the Fop</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two amino acids with the largest positive (left) and negative (right) correlation to Fop/expression level in (A) G. bimaculatus; (B) O. fasciatus; and (C) P. hawaiensis. Fop was binned into four categories as shown. Spearman R correlations in Table 2 were calculated with the use of all (unbinned) data points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542393386" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The two amino acids with the largest positive (left) and negative (right) correlation to Fop/expression level in (A) &lt;em&gt;G. bimaculatus&lt;/em&gt;; (B) &lt;em&gt;O. fasciatus&lt;/em&gt;; and (C) &lt;em&gt;P. hawaiensis&lt;/em&gt;. Fop was binned into four categories as shown. Spearman R correlations in Table 2 were calculated with the use of all (unbinned) data points.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2307/F3.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2307/F3.medium.gif" width="324" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">The two amino acids with the largest positive (left) and negative (right) correlation to Fop/expression level in (A) <em>G. bimaculatus</em>; (B) <em>O. fasciatus</em>; and (C) <em>P. hawaiensis</em>. Fop was binned into four categories as shown. Spearman R correlations in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> were calculated with the use of all (unbinned) data points.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">In <em>O. fasciatus</em>, 16 amino acids exhibited a statistically significant change in frequency with Fop/expression level (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The two amino acids with the strongest negative (Arg, Gly) and strongest positive (Asn, Lys) R values are illustrated in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>. The strongest negative correlation was for Arg (R = −0.252, <em>P</em> &lt; 10<sup>−15</sup>), which also has among the largest S/C scores (56.3), implying losses of this metabolically expensive amino acid in CDS expressed at high levels (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). However, Gly and Ala also exhibited substantial inverse correlations (between −0.188 and −0.183, <em>P</em> &lt; 10<sup>−15</sup>) and have low S/C scores (between 1 and 4.8), implying reduced frequency of these amino acids despite their very low metabolic costs. The amino acids exhibiting the strongest positive correlations with Fop/expression levels were Asn, Lys, Asp, and Glu (R between 0.211 and 0.287, <em>P</em> &lt; 10<sup>−15</sup>; <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). Remarkably, each of these amino acids has S/C scores within a narrow range from 30.1 to 36.5. Thus, similar to <em>G. bimaculatus</em>, genes expressed at high levels appear to favor not amino acids with the lowest S/C scores, but rather a narrow range of moderately sized amino acids, potentially indicating an advantage to maintaining an intermediate protein size under high expression, rather than the smallest possible size. Thus, this explains the negative association between Pr<sub>High SC</sub> and Fop reported previously (see preceding paragraph in this section), <em>i.e.</em>, an average decline in use of amino acids with extreme S/C in highly expressed genes, while indicating this decline does not necessarily favor the lowest S/C amino acids in these invertebrates. Notably, eight of the nine amino acids with positive R in <em>G. bimaculatus</em> also had positive R in <em>O. fasciatus</em>, implying similar preferences for specific amino acids among these divergent insects from different orders (Orthoptera and Hemiptera, respectively). Furthermore, the four amino acids with strongest positive R values (Lys, Asn, Asp and Glu) were identical among these two arthropods (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>), implying a shared preference for intermediate sized amino acids under high expression. In the beetle <em>T. castaneum</em> (Coleoptera), it has been reported that amino acids with moderate and with low scores were preferred under high expression (<em>e.g.</em>, Glu, Asp, Lys, Val, Ala, and Gly) (<a id="xref-ref-94-11" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>), whereas our data for <em>G. bimaculatus</em> and <em>O. fasciatus</em> reveal a strong preference for intermediate sized amino acids (excluding the lowest classes), suggesting species-specific effects on amino acid preferences.</p><p id="p-33">Together, these results from individual amino acids in <em>G. bimaculatus</em> and <em>O. fasciatus</em> indicate that the moderate negative correlation between Pr<sub>High SC</sub> and Fop is due to a shift toward increased usage of intermediate cost amino acids, coupled with a decline in both very high and low cost amino acids under high expression. It is worth noting that degeneracy was sometimes lower for higher cost amino acids (<em>e.g.</em>, Phe <em>vs.</em> Ser, <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>), which could conceivably contribute to an inverse correlation between Pr<sub>High SC</sub> and Fop (since codon bias was reduced for low degeneracy amino acids, <a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>). However, our results in <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> show that degeneracy effects on Fop did not underlie the Pr<sub>High SC</sub> and Fop negative correlation. For example, for <em>G. bimaculatus</em>, the two-fold degenerate amino acids Phe (S/C score 44.0) and Cys (57.2) exhibited positive (rather than negative) correlations with expression while highly degenerate amino acids such as Ser (S/C score of 17.8) exhibited a large negative correlation (rather than positive) with transcription/Fop level (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). We therefore propose that our findings with respect to Pr<sub>High SC</sub> cannot result from an artifactual relationship between Fop and amino acid usage. Rather, our data suggest that the reduced usage of expensive amino acids is primarily caused by a greater frequency of moderate cost amino acids under high expression.</p><p id="p-34">For <em>P. hawaiensis</em>, the relationship between Fop and Pr<sub>High SC</sub> was weakly positive (Spearman’s R = 0.050, <em>P</em> &lt; 10<sup>−15</sup>), consistent with a mild tendency for higher protein size under high expression, and implying a more complex dynamic between expression and protein size in this taxon. This is supported by the relationships between Fop/expression level and amino acid frequencies per gene. Specifically, Fop/expression level was inversely correlated with the frequency of several amino acids per CDS, including Ser, Cys, Thr, and Leu (R between −0.377 and −0.102, <em>P</em> &lt; 10<sup>−15</sup>). In turn, the strongest positive correlations were found for Asp, Tyr, Gly, Lys, Glu and Met (R between 0.103 and 0.195, <em>P</em> &lt; 10<sup>−15</sup>). Thus, high Fop/expression level in <em>P. hawaiensis</em> genes favors use of a distinct set of amino acids as compared to <em>G. bimaculatus</em> and <em>O. fasciatus</em>. As shown in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>, when binned into four distinct classes of Fop values, the amino acids with the most positive (Asp) and negative (Ser) R values exhibited a clear link to Fop/expression level, but those with the second largest value (Tyr and Cys, respectively) exhibited a much weaker effect. Further, some amino acids that were more frequent under high Fop/expression level showed no clear trend in S/C score. For example, Tyr (with positive R) has among the highest S/C scores (57), whereas Gly (also with positive R) exhibits the lowest S/C score (1) (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS3.pdf">Table S3</a>). This might indicate that a balance of large and small amino acids are preferred under elevated expression <em>in P. hawaiensis</em>, that codon and amino acid usage are subject to much weaker selection outside the highest expressed gene set (see <em>Genome-wide optimal codon usage</em>), or that factors other than protein size play a predominant role in shaping amino acid frequency in this organism.</p></div><div id="sec-10" class="subsection"><h3>Codon usage and CDS lengths in <em>D. melanogaster</em> orthologs support translational selection</h3><p id="p-35">We evaluated the role of CDS length and GO annotation category in our findings using the well-studied and annotated model system <em>D. melanogaster</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">St Pierre <em>et al.</em> 2014</a>) as a reference system. First, we determined whether genes exhibiting optimal codon usage in each of our three arthropods under study also exhibit optimal codon usage in their orthologs in <em>D. melanogaster</em>, a species previously shown to have GC3 optimal codons (<a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>). Thus, GC3 provides an effective measure of optimal codon usage in this taxon. Note that we did not cross compare the CDS sets of <em>G. bimaculatus</em>, <em>O. fasciatus</em> and <em>P. hawaiensis</em> since, given the <em>de novo</em> assembled transcriptomes that we used, apparent losses of orthologs among each pair might result from any one of expression below detection thresholds, assembly methods, or true gene loss, thereby minimizing gene sets; this ambiguity would not occur in <em>D. melanogaster</em>, which has a complete annotated gene list. Thus, we independently contrasted each gene set to their orthologs in <em>D. melanogaster</em>. The goal of this analysis was to determine whether orthologs to highly expressed genes in each of the three species under study, also exhibited elevated optimal codon usage in a divergent taxon. The number of <em>D. melanogaster</em> orthologs matching the CDS list from <em>G. bimaculatus</em>, <em>O. fasciatus</em> and <em>P. hawaiensis</em> was 3960 (74.9% of the CDS set under study), 4190 (68.0%) and 2822 (41.9%), respectively. Using the <em>D. melanogaster</em> orthologs of those CDS from the low (lowest 5% RPM), moderate (&gt;5% RPM &lt; 95%), and high (highest 5% RPM) categories in each of our three study species, we measured the ENC. We then evaluated the ENCs and GC3 (optimal codon) values across the three expression level categories. As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, the <em>D. melanogaster</em> orthologs of the <em>G. bimaculatus</em> genes in the highest expression category showed markedly lower ENC and higher GC3 than those from the low or moderate categories (<em>P</em>-value ranked analysis of variance [ANOVA] were 2.2 × 10<sup>−15</sup> and 2.2 × 10<sup>−15</sup>, respectively, Dunn’s <em>post hoc</em> tests <em>P</em> &lt; 0.05). This confirms optimization of codon usage in the highly expressed gene set across these divergent organisms, bias toward AT3 codons in <em>G. bimaculatus</em> and GC3 in <em>D. melanogaster</em> (note: higher expression in <em>D. melanogaster</em> is evident based on elevated optional codon usage (defined here as GC3), (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Coghlan and Wolfe 2000</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Popescu <em>et al.</em> 2006</a>; <a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Wall <em>et al.</em> 2005</a>; <a id="xref-ref-94-12" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). For <em>O. fasciatus</em>, identical trends were detected (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), wherein <em>D. melanogaster</em> orthologs of the high expression dataset had lower ENC and higher GC3 than those from the low and moderate classes (P-ranked ANOVA = 4.2 × 10<sup>−11</sup> and 6.9 × 10<sup>−7</sup>, respectively, Dunn’s <em>post hoc</em> tests <em>P</em> &lt; 0.05). Finally, for <em>P. hawaiensis</em>, lower ENC and higher GC3 was also observed for orthologs matching the high expression category genes, than for the low and moderate expression categories (<em>P</em> = 5.8 × 10<sup>−7</sup> and 3.3 × 10<sup>−4</sup>, respectively; Dunn’s <em>post hoc</em> tests <em>P</em> &lt; 0.05) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>), indicating a GC3 bias in each of these arthropods. Collectively, these results demonstrate that enhanced optimal codon usage in the high expression CDS dataset of <em>G. bimaculatus</em>, <em>O. fasciatus</em> and of <em>P. hawaiensis</em> is shared with their orthologs from the divergent relative <em>D. melanogaster</em>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar and whisker plots of the effective number of codons (ENCs) and GC3 content of D. melanogaster orthologs to coding sequence with low, moderate, and high expression in (A) G. bimaculatus; (B) O. fasciatus; and (C) P. hawaiensis. P-values of ranked analysis of variance " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1542393386" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bar and whisker plots of the effective number of codons (ENCs) and GC3 content of &lt;em&gt;D. melanogaste&lt;/em&gt;r orthologs to coding sequence with low, moderate, and high expression in (A) &lt;em&gt;G. bimaculatus&lt;/em&gt;; (B) &lt;em&gt;O. fasciatus&lt;/em&gt;; and (C) &lt;em&gt;P. hawaiensis&lt;/em&gt;. &lt;em&gt;P&lt;/em&gt;-values of ranked analysis of variance &lt;0.0003 for each figure. Different letters in each figure indicate paired differences using Dunn&#x2019;s contrast (&lt;em&gt;P&lt;/em&gt; &lt; 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2307/F4.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2307/F4.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2307/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Bar and whisker plots of the effective number of codons (ENCs) and GC3 content of <em>D. melanogaste</em>r orthologs to coding sequence with low, moderate, and high expression in (A) <em>G. bimaculatus</em>; (B) <em>O. fasciatus</em>; and (C) <em>P. hawaiensis</em>. <em>P</em>-values of ranked analysis of variance &lt;0.0003 for each figure. Different letters in each figure indicate paired differences using Dunn’s contrast (<em>P</em> &lt; 0.05).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">We next evaluated the relationship between CDS length and gene expression level for each arthropod under study, using lengths obtained from <em>D. melanogaster</em> orthologs. This is an effective approach as protein lengths tend to be highly conserved in eukaryotes (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Wang <em>et al.</em> 2005</a>), and was used since transcriptome assembly may not always yield full CDS lengths. Our objective was to determine whether high expression is linked to shorter/longer protein lengths. As shown in <a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>, the length (number of codons) of the <em>D. melanogaster</em> orthologs matching CDS from <em>G. bimaculatus</em>, <em>O. fasciatus</em> and in <em>P. hawaiensis</em> were shortest for the high expression category, intermediate for moderate class, and longest for the lowest expression category for each of the species under study (ranked ANOVA <em>P</em> &lt; 2.9 × 10<sup>−9</sup> for all three species). This pattern indicates that translational selection favors short proteins as a function of expression (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Taken together, our collective findings indicate that highly expressed genes encode short proteins (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>), exhibit high Fop (<a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), and preferentially use specific amino acids (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a>), each of which promote cost-efficient translation.</p><p id="p-38">Given that highly expressed genes tend to be shorter than and lower expressed genes (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>), we considered the role of genetic-linkage in our study. Selective sweeps or background selection can lead to fixation of linked nonoptimal codons, which may have greater effects in longer genes, and could reduce Fop (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Comeron <em>et al.</em> 1999</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Loewe and Charlesworth 2007</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Whittle <em>et al.</em> 2011b</a>). We thus compared Fop of long genes (&gt;500 codons) from the high expression (highest 5% expression) <em>vs.</em> those from the low expression (lowest 5%) category for each species under study. We found that, consistent with the results of our analysis for all highly expressed genes regardless of length (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), Fop was statistically significantly elevated in the high- compared with the low-expression category for long genes in <em>G. bimaculatus</em> (Mean<sub>High</sub> = 0.483 ± 0.003, Mean<sub>Low</sub>= 0.441 ± 0.007 MWU <em>P</em> &lt; 10<sup>−15</sup>), <em>O. fasciatus</em> (Mean<sub>High</sub> = 0.497 ± 0.005, Mean<sub>Low</sub>= 0.426 ± 0.008 MWU <em>P</em> &lt; 10<sup>−15</sup>), and in <em>P. hawaiensis</em> (Mean<sub>High</sub> = 0.409 ± 0.013, Mean<sub>Low</sub>= 0.348 ± 0.009 MWU <em>P</em> = 0.0002). This argues against a pervasive role of genetic linkage, which, were it the major factor shaping codon use evolution, would be expected to yield similar Fop in long genes from both the high and low expression categories. Thus, long genes appear most influenced by their expression level, and are under greater selective pressure on codon usage when highly transcribed. Accordingly, we propose that genetic linkage, if it has an effect on Fop in these organisms, is largely restricted to the lower expression classes, where CDS are on average markedly longer (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>) and purifying selective pressures are apt to be weaker due to their low transcription (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Subramanian and Kumar 2004</a>).</p></div><div id="sec-11" class="subsection"><h3>Functional ontology annotation shows highly transcribed CDS are involved in cell-cycling and translation</h3><p id="p-39">We wished to determine whether highly expressed genes with elevated optimal codon usage were preferentially associated with particular functions, biological processes or cellular components. To this end, we used <em>D. melanogaster</em> orthologs to assess GO annotation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ashburner <em>et al.</em> 2000</a>) of the CDS in all three arthropods under study. We clustered genes from the highest expression category (above the 95th percentile) in each species into functional groups using the procedures in DAVID (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Huang da <em>et al.</em> 2009a</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">b</a>). The results in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> show that high expression was associated with enrichment of genes involved in cell-cycle processes (<em>e.g.</em>, spindle organization, microtubule organization, proteosomes), translation (ribosome synthesis, RPGs) and DNA/RNA binding in all three arthropods under study, consistent with roles in rapid cell divisions in the embryo and/or reproductive tissues. Thus, the optimization of codon and amino acid usage observed in these organisms could be instrumental in minimizing the protein biosynthetic costs of cell division and protein turnover in the embryo and sexual tissues (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS1.pdf">Table S1</a>). Remarkably, translational efficiency includes optimization of codons and amino acids not only in genes involved in cell division, but also in genes controlling protein synthesis itself (RPGs). This latter finding is concordant with extensive literature showing that protein synthesis genes are typically among the highest expressed in an organism (see for example <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Sharp and Li 1987</a>; <a id="xref-ref-86-7" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466968/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466968/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Functional clustering of the most highly expressed CDS (above the 95th percentile) for each of three arthropod species under study using their orthologs in <em>Drosophila melanogaster</em> and the gene ontology system DAVID (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Huang da <em>et al.</em> 2009a</a>,<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">b</a>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">For the CDS pooled from the moderate and low expression categories, we found enrichment of certain DNA/RNA and protein synthesis genes, primarily from the upper expression levels in the moderate class. However, the CDS with moderate and low expression levels spanned a vast range of functionalities rather than showing significant enrichment for any specific GO category compared with the high expression category (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/TableS4.pdf">Table S4</a>). Such a distribution of GO categories is expected for highly complex eukaryotic embryo/sexual tissues, which are typically comprised of an array of distinct cell types and transcripts (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Combs and Eisen 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Diez-Roux <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Donoughe and Extavour 2015</a>; <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Ewen-Campen <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gilboa 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Jankovics <em>et al.</em> 2014</a>). Although the moderate/low expressed genes are clearly biologically relevant, our data indicate they are less apt to influence codon or amino acid optimization (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a>) and have longer CDS lengths (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>), likely due to their reduced transcriptional and translational levels, and thus biosynthetic costs. Together, the consistency in functionalities of the highly expressed dataset in all three arthropods studied herein reveals that optimization of codon and amino acid usage is a shared feature of highly expressed genes, particularly those involved in cell division and translation, in these divergent arthropods.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Optimal codon and amino acid usage</h3><p id="p-44">Major codon and amino acid preferences under high expression point toward adaptation for translational accuracy and/or efficiency (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akashi 2001</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Akashi 2003</a>). Here, we identified clear preferences in codon usage and in amino acid frequency relative to expression in three emerging models of arthropods, <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em>. With respect to optimal codons, preferences have been previously reported in bacteria, fungi such as yeast and Neurospora, and some animals including <em>D. melanogaster</em> and <em>C. elegans</em>, but remain rare in arthropods outside of the higher insects (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-28-10" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-91-4" class="xref-bibr" href="#ref-91">Whittle <em>et al.</em> 2011a</a>). Our results revealed AT3 optimal codons across the vast majority of amino acids in <em>G. bimaculatus</em> and <em>O. fasciatus</em>, whereas weaker yet significant GC3 preferences were observed in <em>P. hawaiensis</em>. Optimal codons have been linked to rapid and accurate translation, the efficient use of ribosomes and a reduced cost of proofreading (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">2003</a>) and may also improve transcription (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Trotta 2011</a>). Thus, the stronger bias in codon usage in highly expressed genes reported here (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table 1</a>) likely reflects the fitness effects of a higher number of transcription and translation events, and in this regard is similar to other invertebrates including <em>D. melanogaster</em>, <em>C. elegans</em>, and <em>T. castaneum</em> (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">2003</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Duret 2000</a>; <a id="xref-ref-94-13" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Given that selection coefficients among synonymous codons are believed to be small, optimal codons may not be as prevalent in organisms with smaller population sizes (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Comeron 2004</a>). Thus, our results are consistent with sufficiently large population sizes in each of these three arthropods such that Nes &gt; 1, allowing optimization of codon usage in each organism (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Comeron 2004</a>; <a id="xref-ref-22-13" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-28-11" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>).</p><p id="p-45">Both mutation and selection can contribute to codon usage. For example, in some organisms such as humans, GC3 content is correlated to GC of adjacent noncoding DNA, suggesting mutational pressures, in addition to selection, play a significant role in its codon usage profiles (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clay <em>et al.</em> 1996</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Comeron 2004</a>). It is also conceivable that AT3 or GC3 biases correlate to expression because highly transcribed genes are localized to regions that are inherently AT- or GC-rich, respectively. Our present results revealed substantial variation in codon bias among amino acids with the same degeneracy, which combined with the fact that amino acids are likely arbitrarily distributed throughout the genome, supports a role of selection herein. Further, the finding that higher degeneracy is connected to greater biases in codon usage is also consistent with selection, and might indicate that the presence of multiple tRNAs leads to greater pressure for optimization. Nonetheless, future sequence data for introns or other noncoding DNA from all three species under study will help further elucidate the precise role of selection relative to mutation in shaping codon usage in these models of the Pancrustacea.</p><p id="p-46">Selective pressure on amino acids for efficient protein synthesis is much less well studied than codon usage. Revealing preferences for specific amino acids in a range of organism could play a major role in advancing our understanding of the dynamics underlying protein evolution, and thus warrants greater attention. Although some amino acid changes might individually have minor effects on cell physiology or fitness, collectively these effects could explain proteome-wide patterns of amino acid composition and evolution (<a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Akashi 2003</a>). Indeed, amino acid size and complexity has been linked to expression in the invertebrates <em>D. melanogaster</em>, <em>C. elegans</em> and <em>T. castaneum</em> as well as yeast (<a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Akashi 2003</a>; <a id="xref-ref-22-14" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2006</a>; <a id="xref-ref-94-14" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Our findings extend these conclusions to include <em>G. bimaculatus</em> and <em>O. fasciatus</em>. However, our results differ from those of other eukaryotes in that we report that highly expressed genes exhibit a prevalence of amino acids with moderate size and complexity, which does not include the smallest classes, suggesting a benefit to utilizing mid-range amino acids, rather than using the smallest in these insects. It is feasible that these amino acids with moderate size/complexity, while more metabolically costly than the smallest, confer an advantage to protein stability or activity.</p><p id="p-47">For <em>P. hawaiensis</em>, the dynamic between Fop/expression level and amino acid biosynthesis costs is less straightforward, as amino acids with both high and low S/C scores exhibited statistically significant positive correlations to Fop (<a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). This suggests that there is a selective advantage of a mixture of large and small amino acids on protein stability or function in this taxon. However, another interpretation is that only a few amino acids are optimized in this taxon. For instance, the strong positive correlation between Fop/expression and frequency of Asp, and the strong negative correlation for Ser (<a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>), each demonstrate unambiguous signals of amino acid optimization. However, the weaker signals from the other amino acids (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>) might indicate that the main or strongest effects of expression are limited to Asp and Ser. In addition, the weaker signal of optimal codon usage in <em>P. hawaiensis</em>, which is very mild outside the highest expression class, suggests that a more complex dynamic underlies this phenomenon for this taxon. Further studies will be needed to ascertain the dynamics of codon and amino acid usage in this crustacean. Nevertheless, taken together, our data provides evidence of optimization of codon and amino acid usage in highly transcribed genes across all three of these distinct arthropods.</p><p id="p-48">The fact that <em>D. melanogaster</em> orthologs to CDS from the high expression category of <em>G. bimaculatus</em> and of <em>O. fasciatus</em> each had higher GC3 frequencies than orthologs from lower expression classes (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, A and B</a>), confirms that optimal codon usage is retained across the same gene sets among divergent insects. Thus, despite the fact these various insects have evolved distinct sets of optimal codons, selection for a subset of codons that promote translational efficiency of highly expressed CDS appears to be a conserved trait. Further, our findings showing elevated usage of GC3 optimal codons in highly transcribed CDS of <em>P. hawaiensis</em> and their orthologs in <em>D. melanogaster</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>) reveals similar evolution traits across the Pancrustacea. Functional categorization (GO) analyses indicate that for all three species studied herein, the CDS from the highest Fop/expression category primarily included genes involved in cell-cycling and protein synthesis, such as RPGs (which are typically highly expressed) (<a id="xref-ref-86-8" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2011</a>) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), suggesting that such fundamental processes are especially prone to selection to reduce biosynthetic costs (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Seligmann 2003</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Swire 2007</a>; <a id="xref-ref-94-15" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). Given that transcripts of cell-cycling and translation genes are known to be prevalent throughout embryogenesis and in later developmental phases in arthropods such as <em>D. melanogaster</em> and other invertebrates including <em>C. elegans</em> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Li <em>et al.</em> 2014</a>), optimization of codons and amino acids in these gene sets could potentially markedly reduce lifetime biosynthetic costs (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Duret 2000</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Seligmann 2003</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Swire 2007</a>; <a id="xref-ref-94-16" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>) and enhance fitness in these organisms.</p><p id="p-49">Our finding that highly expressed CDS encoded short proteins in <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and in <em>P. hawaiensis</em> (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>) concurs with previous studies in eukaryotes such as <em>D. melanogaster</em>, <em>T. castaneum</em>, humans, yeast, and some plants (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">2003</a>; <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Comeron 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ingvarsson 2007</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lemos <em>et al.</em> 2005</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Moriyama and Powell 1998</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Urrutia and Hurst 2001</a>; <a id="xref-ref-90-4" class="xref-bibr" href="#ref-90">Whittle <em>et al.</em> 2007</a>; <a id="xref-ref-94-17" class="xref-bibr" href="#ref-94">Williford and Demuth 2012</a>). The phenomenon of shorter CDS in highly expressed genes might reflect the action of selection to minimize the protein length, which combined with high optimal codon usage and preferential amino acid usage observed herein (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-2-14" class="xref-table" href="#T2">Table 2</a>), potentially reduces the costs of transcription, translation, and cellular transport, in these arthropods (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-12" class="xref-bibr" href="#ref-4">2003</a>). With respect to translational costs, reductions in protein length of highly transcribed genes should have similar advantages to the usage of optimal codons and of reduced-cost amino acids in terms of biosynthesis, suggesting potential for co-evolution of these traits (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-13" class="xref-bibr" href="#ref-4">2003</a>).</p><p id="p-50">The fact that long CDS under high transcription had elevated Fop compared with their lowly transcribed counterparts supports the hypothesis that expression level, rather than genetic-linkage, is the primary factor shaping Fop differences among expression classes. Nevertheless, it is formally possible that genetic linkage may contribute toward the observed reduced Fop in the lower expression level datasets, as longer genes appear most prone to fixation of non-optimal codons (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Betancourt and Presgraves 2002</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Comeron <em>et al.</em> 1999</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Loewe and Charlesworth 2007</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Whittle and Johannesson 2013</a>). This linkage phenomenon might be most common in genes with weak expression levels, which would be expected to experience lower purifying selection pressures (<a id="xref-ref-80-3" class="xref-bibr" href="#ref-80">Subramanian and Kumar 2004</a>). Future data on amino acid substitutions and linked fixations of nonoptimal codons with respect to CDS length among related species in each taxonomic group (<em>Gryllus</em>, <em>Oncopeltus</em>, and <em>Parhyale</em>) will help further discern the role of genetic-interference on Fop (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Betancourt and Presgraves 2002</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kim 2004</a>; <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Whittle and Johannesson 2013</a>). In addition, when available in the future, genomic data for all three of these arthropods will allow precise measures of CDS lengths in all genes under study (not only those with orthologs in <em>D. melanogaster</em>), providing a means to test fine-scale gene-length effects on codon usage within expression classes, which to date have typically shown higher codon bias in shorter genes in eukaryotes (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Akashi 2001</a>, <a id="xref-ref-4-14" class="xref-bibr" href="#ref-4">2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cutter 2008</a>; <a id="xref-ref-28-12" class="xref-bibr" href="#ref-28">Duret and Mouchiroud 1999</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Moriyama and Powell 1998</a>; <a id="xref-ref-90-5" class="xref-bibr" href="#ref-90">Whittle <em>et al.</em> 2007</a>, <a id="xref-ref-91-5" class="xref-bibr" href="#ref-91">2011a</a>).</p><p id="p-51">Together, it is evident that at a genome-wide level, high transcription is accompanied by a marked shift toward short protein lengths (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>) and this shift co-occurs with increased usage of optimal codons (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>) and specific amino acids (<a id="xref-table-wrap-2-15" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>). This points toward multifaceted aspects of translational selection, which affect several genomic traits (codons, amino acids, and CDS length) to yield efficient protein synthesis in these arthropods.</p></div><div id="sec-14" class="subsection"><h3>Fop as a predictor of expression</h3><p id="p-52">While Fop, or other codon adaptation indices, have been shown to reliably reflect the expression level in an organism (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Coghlan and Wolfe 2000</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Popescu <em>et al.</em> 2006</a>; <a id="xref-ref-85-4" class="xref-bibr" href="#ref-85">Wall <em>et al.</em> 2005</a>), it is worthwhile to note that selection at the protein level can influence optimal codon usage. For instance, at the interspecies level, proteins that evolve rapidly (displaying high nonsynonymous to synonymous substitution rates, dN/dS) (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Comeron <em>et al.</em> 1999</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Loewe and Charlesworth 2007</a>; <a id="xref-ref-89-3" class="xref-bibr" href="#ref-89">Whittle and Johannesson 2013</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Yang 2007</a>) often exhibit reductions in optimal codon usage. Adaptive evolution at the protein level may cause selective sweeps that fix linked non-optimal codon mutations, and relaxed selection on proteins often correlates to losses in optimal codons (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Betancourt and Presgraves 2002</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Kim 2004</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sella <em>et al.</em> 2009</a>; <a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Whittle <em>et al.</em> 2011b</a>). Relaxed selection on proteins or synonymous codons may arise from factors such as reduced population size or lowered recombination rates over an organism’s evolutionary history (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Betancourt and Presgraves 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Comeron and Kreitman 1998</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Haddrill <em>et al.</em> 2011</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Sella <em>et al.</em> 2009</a>; <a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">Whittle <em>et al.</em> 2011b</a>). However, the relationship between Fop and expression should not be uncoupled by these processes, but rather is likely maintained due to the fact that rates of protein evolution appear to be inversely correlated to expression level in various organisms including mice, fruit fly, and yeast (see for example <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Lemos <em>et al.</em> 2005</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Pal <em>et al.</em> 2001</a>; <a id="xref-ref-80-4" class="xref-bibr" href="#ref-80">Subramanian and Kumar 2004</a>). The conservation of highly expressed genes at the protein sequence level may be explained by their essentiality, or indispensability, to fitness (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mank and Ellegren 2009</a>). In addition, highly expressed genes might also exhibit signatures of selection on (otherwise) neutral amino acids if such residues are needed to promote translational efficiency, for example because they promote a small size or specific conformation (<a id="xref-ref-4-15" class="xref-bibr" href="#ref-4">Akashi 2003</a>). In this regard, Fop and expression level likely remain connected even for genes with rapid rates of protein evolution, such that Fop would decrease under rapid protein evolution, which is more apt to occur in genes with lowered expression. Given that we observed clear relationships between Fop and RPM, and between Fop and the S/C score and amino acid composition (<a id="xref-table-wrap-2-16" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>), we suggest that in these arthropods, optimal codon usage reflects expression level on a genome-wide level, and that optimal codon use has evolved to promote efficient translation. In this regard, the relationships observed between Fop and amino acid composition would directly reflect adaptation for fast and accurate translation in genes with high expression.</p><p id="p-53">We conclude that highly expressed genes, in particular those involved in cell-cycling and protein synthesis (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), exhibit a history of coevolution of optimal codons (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>), the usage of specific amino acids (<a id="xref-table-wrap-2-17" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>) and shorter CDS length (<a href="http://www.g3journal.org/content/suppl/2015/09/17/g3.115.021402.DC1/FigureS2.pdf">Figure S2</a>), which act in concert to promote accurate and cost efficient translation in these divergent arthropods. Our findings indicate that selection favors specific optimal codons and amino acids in three emerging models of arthropods, <em>G. bimaculatus</em>, <em>O. fasciatus</em>, and <em>P. hawaiensis</em>. For both insects, a particularly strong signal of optimal codons was identified. Further, the marked connection between Fop and amino acid size and complexity suggests an adaptive strategy to promote efficient translation, which likely allows fast rates of accurate protein synthesis. Although weaker selective effects on codon usage, with fewer optimal codons, were evident in <em>P. hawaiensis</em> compared with the two insects, this taxon also exhibits a significant connection among Fop and size/complexity, suggesting coevolution that appears to favor both small and large amino acids has also occurred in this crustacean. Thus, the dynamics of protein evolution may be more complex in this taxon, wherein protein stability or function in highly expressed genes may necessitate a wide range of amino acids. Further studies should explore the variation in amino acid and synonymous codon mutations and specific types of amino acid replacements at the population level (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi and Schaeffer 1997</a>) to reveal whether selective variation can be observed at the intraspecies level in these arthropods. Furthermore, future availability of genomes from these three species will allow further study of other facets of translational selection, including the relationship between tRNA gene abundance and expression, which is particularly relevant to understanding the evolution of optimal codons and its contribution to functional genome evolution.</p></div></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank the anonymous reviewers for valuable comments on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-10"><p id="p-55">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021402/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021402/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-56">Identification numbers of RNA-seq data at the National Center for Biotechnology Information: SRR060816, SRR057570, SRR060813.</p></li><li class="fn" id="fn-12"><p id="p-57"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-15"><span class="received-label">Received </span>June 15, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-28"><span class="accepted-label">Accepted </span>August 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Whittle and Extavour</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S.W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Natural selection and the frequency distributions of “silent” DNA polymorphism in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D295%26rft.epage%253D307%26rft.atitle%253DNatural%2BSelection%2Band%2Bthe%2BFrequency%2BDistributions%2Bof%2B%2560%2560Silent%2527%2527%2BDNA%2BPolymorphism%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9136019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akam</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Arthropods: development diversity within a (super) phylum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">4438</span>–<span class="cit-lpage">4441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D9%26rft.spage%253D4438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.3" data-doi="10.1016/S0959-437X(00)00250-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene expression and molecular evolution.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D660%26rft.epage%253D666%26rft.atitle%253DGene%2Bexpression%2Band%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252800%252900250-1%26rft_id%253Dinfo%253Apmid%252F11682310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(00)00250-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11682310&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171989300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Translational selection and yeast proteome evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1291%26rft.epage%253D1303%26rft.atitle%253DTranslational%2BSelection%2Band%2BYeast%2BProteome%2BEvolution%26rft_id%253Dinfo%253Apmid%252F12930740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzEyOTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.5" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.6" data-doi="10.1007/s00239-006-0257-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced selection for codon usage bias in <em>Drosophila miranda</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBachtrog%26rft.auinit1%253DD.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D586%26rft.epage%253D590%26rft.atitle%253DReduced%2Bselection%2Bfor%2Bcodon%2Busage%2Bbias%2Bin%2BDrosophila%2Bmiranda.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0257-x%26rft_id%253Dinfo%253Apmid%252F17457633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0257-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17457633&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246311200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.7" data-doi="10.1371/journal.pone.0043111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative analysis of codon usage bias and codon context patterns between Dipteran and Hymenopteran sequenced genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e43111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBehura%26rft.auinit1%253DS.%2BK.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De43111%26rft.epage%253De43111%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bcodon%2Busage%2Bbias%2Band%2Bcodon%2Bcontext%2Bpatterns%2Bbetween%2Bdipteran%2Band%2Bhymenopteran%2Bsequenced%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0043111%26rft_id%253Dinfo%253Apmid%252F22912801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0043111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22912801&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.8" data-doi="10.1111/j.1469-185X.2012.00242.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D88%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2012.00242.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2012.00242.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.9" data-doi="10.1073/pnas.93.24.13919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beletskii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Transcription-induced mutations: increase in C to T mutations in the nontranscribed strand during transcription in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">13919</span>–<span class="cit-lpage">13924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBeletskii%26rft.auinit1%253DA.%26rft.volume%253D93%26rft.issue%253D24%26rft.spage%253D13919%26rft.epage%253D13924%26rft.atitle%253DTranscription-induced%2Bmutations%253A%2BIncrease%2Bin%2BC%2Bto%2BT%2Bmutations%2Bin%2Bthe%2Bnontranscribed%2Bstrand%2Bduring%2Btranscription%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.24.13919%26rft_id%253Dinfo%253Apmid%252F8943036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMjQvMTM5MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beletskii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Correlation between transcription and C to T mutations in the non-transcribed DNA strand.</span> <abbr class="cit-jnl-abbrev">Biol. Chem.</abbr> <span class="cit-vol">379</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2Bchemistry%26rft.stitle%253DBiol%2BChem%26rft.aulast%253DBeletskii%26rft.auinit1%253DA.%26rft.volume%253D379%26rft.issue%253D4-5%26rft.spage%253D549%26rft.epage%253D551%26rft.atitle%253DCorrelation%2Bbetween%2Btranscription%2Band%2BC%2Bto%2BT%2Bmutations%2Bin%2Bthe%2Bnon-transcribed%2BDNA%2Bstrand.%26rft_id%253Dinfo%253Apmid%252F9628351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9628351&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073686200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.11" data-doi="10.1073/pnas.212277199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presgraves</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Linkage limits the power of natural selection in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">13616</span>–<span class="cit-lpage">13620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.212277199%26rft_id%253Dinfo%253Apmid%252F12370444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjEvMTM2MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.12" data-doi="10.1038/nature06784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klingler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genome of the model beetle and pest <em>Tribolium castaneum</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">955</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRichards%26rft.auinit1%253DS.%26rft.volume%253D452%26rft.issue%253D7190%26rft.spage%253D949%26rft.epage%253D955%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bmodel%2Bbeetle%2Band%2Bpest%2BTribolium%2Bcastaneum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06784%26rft_id%253Dinfo%253Apmid%252F18362917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18362917&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255208600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butt</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Embryology of the Milkweed Bug, <em>Oncopeltus fasciatus</em> (Hemiptera).</span> <abbr class="cit-jnl-abbrev">Cornell Experiment Station Memoir</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCornell%2BExperiment%2BStation%2BMemoir%26rft.volume%253D283%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.14" data-doi="10.1006/mpev.1996.0002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clay</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caccio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoubak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Human coding and noncoding DNA: compositional correlations.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DClay%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D12%26rft.atitle%253DHuman%2Bcoding%2Band%2Bnoncoding%2BDNA%253A%2Bcompositional%2Bcorrelations.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.1996.0002%26rft_id%253Dinfo%253Apmid%252F8673288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.1996.0002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8673288&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TZ13600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.15" data-doi="10.1002/1097-0061(20000915)16:12&lt;1131::AID-YEA609&gt;3.0.CO;2-F"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coghlan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Relationship of codon bias to mRNA concentration and protein length in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DCoghlan%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D1131%26rft.epage%253D1145%26rft.atitle%253DRelationship%2Bof%2Bcodon%2Bbias%2Bto%2BmRNA%2Bconcentration%2Band%2Bprotein%2Blength%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0061%252820000915%252916%253A12%253C1131%253A%253AAID-YEA609%253E3.0.CO%253B2-F%26rft_id%253Dinfo%253Apmid%252F10953085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0061(20000915)16:12&lt;1131::AID-YEA609&gt;3.0.CO;2-F&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10953085&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089440800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.16" data-doi="10.1126/science.1197761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The ecoresponsive genome of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColbourne%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D331%26rft.issue%253D6017%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DThe%2BEcoresponsive%2BGenome%2Bof%2BDaphnia%2Bpulex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197761%26rft_id%253Dinfo%253Apmid%252F21292972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxNy81NTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.17" data-doi="10.1371/journal.pone.0071820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequencing mRNA from cryo-sliced <em>Drosophila</em> embryos to determine genome-wide spatial patterns of gene expression.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e71820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De71820%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0071820%26rft_id%253Dinfo%253Apmid%252F23951250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0071820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23951250&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.18" data-doi="10.1534/genetics.104.026351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selective and mutational patterns associated with gene expression in humans: influences on synonymous composition and intron presence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1293</span>–<span class="cit-lpage">1304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1293%26rft.epage%253D1304%26rft.atitle%253DSelective%2Band%2BMutational%2BPatterns%2BAssociated%2BWith%2BGene%2BExpression%2Bin%2BHumans%253A%2BInfluences%2Bon%2BSynonymous%2BComposition%2Band%2BIntron%2BPresence%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026351%26rft_id%253Dinfo%253Apmid%252F15280243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzEyOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The correlation between synonymous and nonsynonymous substitutions in <em>Drosophila</em>: mutation, selection or relaxed constraints?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D150%26rft.issue%253D2%26rft.spage%253D767%26rft.epage%253D775%26rft.atitle%253DThe%2BCorrelation%2BBetween%2BSynonymous%2Band%2BNonsynonymous%2BSubstitutions%2Bin%2BDrosophila%253A%2BMutation%252C%2BSelection%2Bor%2BRelaxed%2BConstraints%253F%26rft_id%253Dinfo%253Apmid%252F9755207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUwLzIvNzY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguade</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Natural selection on synonymous sites is correlated with gene length and recombination in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D249%26rft.atitle%253DNatural%2BSelection%2Bon%2BSynonymous%2BSites%2BIs%2BCorrelated%2BWith%2BGene%2BLength%2Band%2BRecombination%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9872963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.21" data-doi="10.1093/molbev/msn024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Divergence times in <em>Caenorhabditis</em> and <em>Drosophila</em> inferred from direct estimates of the neutral mutation rate.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn024%26rft_id%253Dinfo%253Apmid%252F18234705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzQvNzc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.22" data-doi="10.1093/molbev/msl097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasmuth</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The evolution of biased codon and amino acid usage in nematode genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2303</span>–<span class="cit-lpage">2315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl097%26rft_id%253Dinfo%253Apmid%252F16936139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMi8yMzAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.23" data-doi="10.1371/journal.pbio.1000582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diez-Roux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banfi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sultan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geffers</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A high-resolution anatomical atlas of the transcriptome in the mouse embryo.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1000582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDiez-Roux%26rft.auinit1%253DG.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253De1000582%26rft.epage%253De1000582%26rft.atitle%253DA%2Bhigh-resolution%2Banatomical%2Batlas%2Bof%2Bthe%2Btranscriptome%2Bin%2Bthe%2Bmouse%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000582%26rft_id%253Dinfo%253Apmid%252F21267068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21267068&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donoughe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Extavour</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Embryonic development of the cricket <em>Gryllus bimaculatus</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> DOI: 10.1016/j.ydbio.2015.04.009.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.25" data-doi="10.1073/pnas.0504070102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adami</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Why highly expressed proteins evolve slowly.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">14338</span>–<span class="cit-lpage">14343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504070102%26rft_id%253Dinfo%253Apmid%252F16176987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQwLzE0MzM4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.26" data-doi="10.1006/jtbi.1997.0443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dufton</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic code synonym quotas and amino acid complexity: cutting the cost of proteins?</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DDufton%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D187%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D173%26rft.atitle%253DGenetic%2Bcode%2Bsynonym%2Bquotas%2Band%2Bamino%2Bacid%2Bcomplexity%253A%2Bcutting%2Bthe%2Bcost%2Bof%2Bproteins%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.1997.0443%26rft_id%253Dinfo%253Apmid%252F9237887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.1997.0443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9237887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XL78400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.27" data-doi="10.1016/S0168-9525(00)02041-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">tRNA gene number and codon usage in the <em>C. elegans</em> genome are co-adapted for optimal translation of highly expressed genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D287%26rft.epage%253D289%26rft.atitle%253DtRNA%2Bgene%2Bnumber%2Band%2Bcodon%2Busage%2Bin%2Bthe%2BC.%2Belegans%2Bgenome%2Bare%2Bco-adapted%2Bfor%2Boptimal%2Btranslation%2Bof%2Bhighly%2Bexpressed%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902041-2%26rft_id%253Dinfo%253Apmid%252F10858656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02041-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10858656&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087898200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.28" data-doi="10.1073/pnas.96.8.4482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4482</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4482%26rft.epage%253D4487%26rft.atitle%253DExpression%2Bpattern%2Band%252C%2Bsurprisingly%252C%2Bgene%2Blength%2Bshape%2Bcodon%2Busage%2Bin%2BCaenorhabditis%252C%2BDrosophila%252C%2Band%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4482%26rft_id%253Dinfo%253Apmid%252F10200288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ0ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.29" data-doi="10.1186/1471-2164-12-61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewen-Campen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panfilio</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The maternal and embryonic transcriptome of the milkweed bug <em>Oncopeltus fasciatus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEwen-Campen%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DThe%2Bmaternal%2Band%2Bearly%2Bembryonic%2Btranscriptome%2Bof%2Bthe%2Bmilkweed%2Bbug%2BOncopeltus%2Bfasciatus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-61%26rft_id%253Dinfo%253Apmid%252F21266083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21266083&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.30" data-doi="10.1111/j.1420-9101.2010.02065.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gonzalez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmons</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Male-induced costs of mating for females compensated by offspring viability benefits in an insect.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2066</span>–<span class="cit-lpage">2075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DGarcia-Gonzalez%26rft.auinit1%253DF.%26rft.volume%253D23%26rft.issue%253D10%26rft.spage%253D2066%26rft.epage%253D2075%26rft.atitle%253DMale-induced%2Bcosts%2Bof%2Bmating%2Bfor%2Bfemales%2Bcompensated%2Bby%2Boffspring%2Bviability%2Bbenefits%2Bin%2Ban%2Binsect.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2010.02065.x%26rft_id%253Dinfo%253Apmid%252F20722896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2010.02065.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20722896&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.31" data-doi="10.1016/j.gde.2015.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilboa</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Organizing stem cell units in the <em>Drosophila</em> ovary.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">32C</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D32%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.01.005%26rft_id%253Dinfo%253Apmid%252F25703842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25703842&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.32" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental transcriptome of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.33" data-doi="10.1038/ng1103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transcription-associated mutational asymmetry in mammalian evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGreen%26rft.auinit1%253DP.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D514%26rft.epage%253D517%26rft.atitle%253DTranscription-associated%2Bmutational%2Basymmetry%2Bin%2Bmammalian%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1103%26rft_id%253Dinfo%253Apmid%252F12612582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12612582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181928600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.34" data-doi="10.1126/science.1178028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Group</span>  <span class="cit-name-given-names">N. G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werren</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niehuis</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional and evolutionary insights from the genomes of three parasitoid Nasonia species.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWerren%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D327%26rft.issue%253D5963%26rft.spage%253D343%26rft.epage%253D348%26rft.atitle%253DFunctional%2Band%2BEvolutionary%2BInsights%2Bfrom%2Bthe%2BGenomes%2Bof%2BThree%2BParasitoid%2BNasonia%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178028%26rft_id%253Dinfo%253Apmid%252F20075255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2My8zNDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.35" data-doi="10.1093/molbev/msq354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determinants of synonymous and nonsynonymous variability in three species of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1731</span>–<span class="cit-lpage">1743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq354%26rft_id%253Dinfo%253Apmid%252F21191087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzUvMTczMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.36" data-doi="10.1101/gr.6249707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolutionary rate analyses of orthologs and paralogs from 12 <em>Drosophila</em> genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1837</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6249707%26rft_id%253Dinfo%253Apmid%252F17989258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xODM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.37" data-doi="10.1371/journal.pgen.1000556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">General rules for optimal codon choice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000556%26rft.epage%253De1000556%26rft.atitle%253DGeneral%2Brules%2Bfor%2Boptimal%2Bcodon%2Bchoice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000556%26rft_id%253Dinfo%253Apmid%252F19593368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593368&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.38" data-doi="10.1371/journal.pone.0049060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">On the limitations of using ribosomal genes as references for the study of codon usage: a rebuttal.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De49060%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049060%26rft_id%253Dinfo%253Apmid%252F23284622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284622&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.39" data-doi="10.1038/nrg3098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkinson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in the mutation rate across mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHodgkinson%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D756%26rft.epage%253D766%26rft.atitle%253DVariation%2Bin%2Bthe%2Bmutation%2Brate%2Bacross%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3098%26rft_id%253Dinfo%253Apmid%252F21969038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21969038&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.40" data-doi="10.1093/nar/gkn923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">and R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn923%26rft_id%253Dinfo%253Apmid%252F19033363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjY6IjM3LzEvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.41" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">and R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2307.42"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Loher</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">1989</span> <span class="cit-source">Cricket Behavior and Neurobiology</span>. <span class="cit-publ-name">Cornell University Press</span>, <span class="cit-publ-loc">Ithaca, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.43" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.45" data-doi="10.1093/molbev/msl212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene expression and protein length influence codon usage and rates of sequence evolution in <em>Populus tremula</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl212%26rft_id%253Dinfo%253Apmid%252F17204548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzMvODM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.46" data-doi="10.1186/1471-2148-8-307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular evolution of synonymous codon usage in Populus.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DIngvarsson%26rft.auinit1%253DP.%2BK.%26rft.volume%253D8%26rft.spage%253D307%26rft.epage%253D307%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2BPopulus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-307%26rft_id%253Dinfo%253Apmid%252F18983655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983655&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.47" data-doi="10.1371/journal.pone.0098579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jankovics</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bujna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilmos</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spirohn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Functional analysis of the <em>Drosophila</em> embryonic germ cell transcriptome by RNA interference.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e98579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De98579%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0098579%26rft_id%253Dinfo%253Apmid%252F24896584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0098579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24896584&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.48" data-doi="10.1186/s12864-015-1596-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Non-uniqueness of factors constraint on the codon usage in <em>Bombyx mori</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1596-z%26rft_id%253Dinfo%253Apmid%252F25943559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1596-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25943559&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Effect of strong directional selection on weakly selected mutations at linked sites: implication for synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Apmid%252F14660698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMjg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.50" data-doi="10.1242/dev.066324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kontarakis</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlopoulos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiupakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konstantinides</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douris</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A versatile strategy for gene trapping and trap conversion in emerging model organisms.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">2625</span>–<span class="cit-lpage">2630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKontarakis%26rft.auinit1%253DZ.%26rft.volume%253D138%26rft.issue%253D12%26rft.spage%253D2625%26rft.epage%253D2630%26rft.atitle%253DA%2Bversatile%2Bstrategy%2Bfor%2Bgene%2Btrapping%2Band%2Btrap%2Bconversion%2Bin%2Bemerging%2Bmodel%2Borganisms%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.066324%26rft_id%253Dinfo%253Apmid%252F21610038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM4LzEyLzI2MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.51" data-doi="10.1093/molbev/msi122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bettencourt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meiklejohn</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolution of proteins and gene expression levels are coupled in <em>Drosophila</em> and are independently associated with mRNA abundance, protein length, and number of protein-protein interactions.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi122%26rft_id%253Dinfo%253Apmid%252F15746013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIyLzUvMTM0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.52" data-doi="10.1101/gr.170100.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickel</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of <em>D. melanogaster</em> and <em>C. elegans</em> developmental stages, tissues, and cells by modENCODE RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170100.113%26rft_id%253Dinfo%253Apmid%252F24985912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTA4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2307.53" data-doi="10.1101/pdb.emo127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Morphology and husbandry of the large milkweed bug, Oncopeltus fasciatus.</span> <span class="cit-source">Cold Spring Harb. Protoc.</span> <span class="cit-vol">2009</span>: <span class="cit-fpage">pdb.emo127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DLiu%26rft.auinit1%253DP.%26rft.volume%253D2009%26rft.issue%253D8%26rft.spage%253Dpdb.emo127%26rft.epage%253Dpdb.emo127%26rft.atitle%253DMorphology%2Band%2BHusbandry%2Bof%2Bthe%2BLarge%2BMilkweed%2BBug%252C%2BOncopeltus%2Bfasciatus%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.emo127%26rft_id%253Dinfo%253Apmid%252F20147229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjE3OiIyMDA5LzgvcGRiLmVtbzEyNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2307.54" data-doi="10.1101/pdb.prot5261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Dissection and fixation of large milkweed bug (Oncopeltus) embryos.</span> <span class="cit-source">Cold Spring Harb. Protoc.</span> <span class="cit-vol">2009</span>: <span class="cit-fpage">pdb.prot5261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DLiu%26rft.auinit1%253DP.%26rft.volume%253D2009%26rft.issue%253D8%26rft.spage%253Dpdb.prot5261%26rft.epage%253Dpdb.prot5261%26rft.atitle%253DDissection%2Band%2BFixation%2Bof%2BLarge%2BMilkweed%2BBug%2B%2528Oncopeltus%2529%2BEmbryos%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.prot5261%26rft_id%253Dinfo%253Apmid%252F20147237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjE5OiIyMDA5LzgvcGRiLnByb3Q1MjYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.55" data-doi="10.1534/genetics.106.065557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loewe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Background selection in single genes may explain patterns of codon bias.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065557%26rft_id%253Dinfo%253Apmid%252F17194784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEzODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.56" data-doi="10.1098/rsbl.2008.0732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Are sex-biased genes more dispensable?</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2008.0732%26rft_id%253Dinfo%253Apmid%252F19433612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNS8zLzQwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.57" data-doi="10.1007/PL00006535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The evolution of codon preferences in Drosophila: a maximum-likelihood approach to parameter estimation and hypothesis testing.</span> <abbr class="cit-jnl-abbrev">J Mol Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D75%26rft.atitle%253DThe%2Bevolution%2Bof%2Bcodon%2Bpreferences%2Bin%2BDrosophila%253A%2Ba%2Bmaximum-likelihood%2Bapproach%2Bto%2Bparameter%2Bestimation%2Band%2Bhypothesis%2Btesting.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006535%26rft_id%253Dinfo%253Apmid%252F10368435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10368435&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080974500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Inferring parameters of mutation, selection and demography from patterns of synonymous site evolution in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DInferring%2BParameters%2Bof%2BMutation%252C%2BSelection%2Band%2BDemography%2BFrom%2BPatterns%2Bof%2BSynonymous%2BSite%2BEvolution%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F11139506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzEvMjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2307.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noji</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The two-spotted cricket Gryllus bimaculatus: an emerging model for developmental and regeneration studies</span>, pp. <span class="cit-fpage">331</span>–<span class="cit-lpage">346</span> in <span class="cit-source">Emerging Model Organisms: A Laboratory Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.60" data-doi="10.1093/nar/26.13.3188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Gene length and codon usage bias in <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae</em> and <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3188</span>–<span class="cit-lpage">3193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.13.3188%26rft_id%253Dinfo%253Apmid%252F9628917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMy8zMTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.61" data-doi="10.1126/science.1258522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neafsey</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abai</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aganezov</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyev</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mosquito genomics. Highly evolvable malaria vectors: the genomes of 16 Anopheles mosquitoes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">1258522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1258522%26rft_id%253Dinfo%253Apmid%252F25554792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDcvNjIxNy8xMjU4NTIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.62" data-doi="10.1111/j.1095-8312.2000.tb01279.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odegaard</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">How many species of arthropods? Erwin’s estimate revised.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D71%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2000.tb01279.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2000.tb01279.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000166406100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.63" data-doi="10.1073/pnas.85.4.1124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Directional mutation pressure and transfer RNA in choice of the third nucleotide of synonymous two-codon sets.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOsawa%26rft.auinit1%253DS.%26rft.volume%253D85%26rft.issue%253D4%26rft.spage%253D1124%26rft.epage%253D1128%26rft.atitle%253DDirectional%2Bmutation%2Bpressure%2Band%2Btransfer%2BRNA%2Bin%2Bchoice%2Bof%2Bthe%2Bthird%2Bnucleotide%2Bof%2Bsynonymous%2Btwo-codon%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.4.1124%26rft_id%253Dinfo%253Apmid%252F2448791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS80LzExMjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Highly expressed genes in yeast evolve slowly.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPal%26rft.auinit1%253DC.%26rft.volume%253D158%26rft.issue%253D2%26rft.spage%253D927%26rft.epage%253D931%26rft.atitle%253DHighly%2BExpressed%2BGenes%2Bin%2BYeast%2BEvolve%2BSlowly%26rft_id%253Dinfo%253Apmid%252F11430355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzIvOTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.65" data-doi="10.1073/pnas.0501101102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlopoulos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Averof</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Establishing genetic transformation for comparative developmental studies in the crustacean <em>Parhyale hawaiensis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">7888</span>–<span class="cit-lpage">7893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501101102%26rft_id%253Dinfo%253Apmid%252F15911760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzIyLzc4ODgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.66" data-doi="10.1534/genetics.105.047399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Popescu</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borza</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bielawski</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolutionary rates and expression level in Chlamydomonas.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047399%26rft_id%253Dinfo%253Apmid%252F16361241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE1NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.67" data-doi="10.1093/molbev/msq251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergero</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of codon usage bias in <em>Silene latifolia</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq251%26rft_id%253Dinfo%253Apmid%252F20855431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvNzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.68" data-doi="10.1007/s00239-008-9162-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raiford</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heizer</span>  <span class="cit-name-given-names">E. M.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Do amino acid biosynthetic costs constrain protein evolution in <em>Saccharomyces cerevisiae</em>?</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DRaiford%26rft.auinit1%253DD.%2BW.%26rft.volume%253D67%26rft.issue%253D6%26rft.spage%253D621%26rft.epage%253D630%26rft.atitle%253DDo%2Bamino%2Bacid%2Bbiosynthetic%2Bcosts%2Bconstrain%2Bprotein%2Bevolution%2Bin%2BSaccharomyces%2Bcerevisiae%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-008-9162-9%26rft_id%253Dinfo%253Apmid%252F18937004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-008-9162-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18937004&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262837600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.69" data-doi="10.1038/nature08742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Regier</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shultz</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwick</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Arthropod relationships revealed by phylogenomic analysis of nuclear protein-coding sequences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRegier%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D463%26rft.issue%253D7284%26rft.spage%253D1079%26rft.epage%253D1083%26rft.atitle%253DArthropod%2Brelationships%2Brevealed%2Bby%2Bphylogenomic%2Banalysis%2Bof%2Bnuclear%2Bprotein-coding%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08742%26rft_id%253Dinfo%253Apmid%252F20147900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147900&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275108400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2307.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rehm</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannibal</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaw</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas-Vila</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The crustacean Parhyale hawaiensis, a new model for arthropod development</span>, pp. <span class="cit-fpage">373</span>–<span class="cit-lpage">404</span> in <span class="cit-source">Emerging Model Organisms: A Laboratory Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.71" data-doi="10.1007/s00239-002-2388-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seligmann</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cost-minimization of amino acid usage.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSeligmann%26rft.auinit1%253DH.%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DCost-minimization%2Bof%2Bamino%2Bacid%2Busage.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-002-2388-z%26rft_id%253Dinfo%253Apmid%252F12574861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-002-2388-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12574861&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180787800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.72" data-doi="10.1371/journal.pgen.1000495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pervasive natural selection in the <em>Drosophila</em> genome?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSella%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000495%26rft.epage%253De1000495%26rft.atitle%253DPervasive%2Bnatural%2Bselection%2Bin%2Bthe%2BDrosophila%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000495%26rft_id%253Dinfo%253Apmid%252F19503600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503600&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.73" data-doi="10.1007/BF02099948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D38%26rft.atitle%253DAn%2Bevolutionary%2Bperspective%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bunicellular%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099948%26rft_id%253Dinfo%253Apmid%252F3104616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104616&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.74" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon adaptation Index–a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.75" data-doi="10.1093/nar/14.13.5125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuohy</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosurski</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5125</span>–<span class="cit-lpage">5143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D14%26rft.issue%253D13%26rft.spage%253D5125%26rft.epage%253D5143%26rft.atitle%253DCodon%2Busage%2Bin%2Byeast%253A%2Bcluster%2Banalysis%2Bclearly%2Bdifferentiates%2Bhighly%2Band%2Blowly%2Bexpressed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.13.5125%26rft_id%253Dinfo%253Apmid%252F3526280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMy81MTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.76" data-doi="10.1098/rstb.1995.0108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Averof</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matassi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">DNA sequence evolution: the sounds of silence.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D349%26rft.issue%253D1329%26rft.spage%253D241%26rft.epage%253D247%26rft.atitle%253DDNA%2BSequence%2BEvolution%253A%2BThe%2BSounds%2Bof%2BSilence%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1995.0108%26rft_id%253Dinfo%253Apmid%252F8577834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNDkvMTMyOS8yNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.77" data-doi="10.1093/nar/gkt1092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Pierre</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefancsik</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McQuilton</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">FlyBase 102–advanced approaches to interrogating FlyBase.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D780</span>–<span class="cit-lpage">D788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1092%26rft_id%253Dinfo%253Apmid%252F24234449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9ENzgwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.78" data-doi="10.1038/nature06340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kheradpour</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discovery of functional elements in 12 <em>Drosophila</em> genomes using evolutionary signatures.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStark%26rft.auinit1%253DA.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D219%26rft.epage%253D232%26rft.atitle%253DDiscovery%2Bof%2Bfunctional%2Belements%2Bin%2B12%2BDrosophila%2Bgenomes%2Busing%2Bevolutionary%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06340%26rft_id%253Dinfo%253Apmid%252F17994088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994088&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.79" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.80" data-doi="10.1534/genetics.104.028944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene expression intensity shapes evolutionary rates of the proteins encoded by the vertebrate genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSubramanian%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D373%26rft.epage%253D381%26rft.atitle%253DGene%2BExpression%2BIntensity%2BShapes%2BEvolutionary%2BRates%2Bof%2Bthe%2BProteins%2BEncoded%2Bby%2Bthe%2BVertebrate%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028944%26rft_id%253Dinfo%253Apmid%252F15454550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.81" data-doi="10.1073/pnas.85.8.2653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Directional mutation pressure and neutral molecular evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">2653</span>–<span class="cit-lpage">2657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D85%26rft.issue%253D8%26rft.spage%253D2653%26rft.epage%253D2657%26rft.atitle%253DDirectional%2Bmutation%2Bpressure%2Band%2Bneutral%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.8.2653%26rft_id%253Dinfo%253Apmid%252F3357886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS84LzI2NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.82" data-doi="10.1007/s00239-006-0206-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swire</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Selection on synthesis cost affects interprotein amino acid usage in all three domains of life.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">558</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFablet%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D558%26rft.epage%253D571%26rft.atitle%253DSelection%2Bon%2Bsynthesis%2Bcost%2Baffects%2Binterprotein%2Bamino%2Bacid%2Busage%2Bin%2Ball%2Bthree%2Bdomains%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0206-8%26rft_id%253Dinfo%253Apmid%252F17476453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0206-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17476453&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246311200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.83" data-doi="10.1371/journal.pone.0021590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trotta</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The 3-base periodicity and codon usage of coding sequences are correlated with gene expression at the level of transcription elongation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e21590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTrotta%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De21590%26rft.epage%253De21590%26rft.atitle%253DThe%2B3-base%2Bperiodicity%2Band%2Bcodon%2Busage%2Bof%2Bcoding%2Bsequences%2Bare%2Bcorrelated%2Bwith%2Bgene%2Bexpression%2Bat%2Bthe%2Blevel%2Bof%2Btranscription%2Belongation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021590%26rft_id%253Dinfo%253Apmid%252F21738721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21738721&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urrutia</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Codon usage bias covaries with expression breadth and the rate of synonymous evolution in humans, but this is not evidence for selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1191</span>–<span class="cit-lpage">1199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUrrutia%26rft.auinit1%253DA.%2BO.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D1191%26rft.epage%253D1199%26rft.atitle%253DCodon%2BUsage%2BBias%2BCovaries%2BWith%2BExpression%2BBreadth%2Band%2Bthe%2BRate%2Bof%2BSynonymous%2BEvolution%2Bin%2BHumans%252C%2Bbut%2BThis%2BIs%2BNot%2BEvidence%2Bfor%2BSelection%26rft_id%253Dinfo%253Apmid%252F11729162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS8zLzExOTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.85" data-doi="10.1073/pnas.0501761102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsh</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Functional genomic analysis of the rates of protein evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">5483</span>–<span class="cit-lpage">5488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501761102%26rft_id%253Dinfo%253Apmid%252F15800036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE1LzU0ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.86" data-doi="10.1371/journal.pone.0022714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Optimal codon identities in bacteria: implications from the conflicting results of two different methods.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e22714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWang%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De22714%26rft.epage%253De22714%26rft.atitle%253DOptimal%2Bcodon%2Bidentities%2Bin%2Bbacteria%253A%2Bimplications%2Bfrom%2Bthe%2Bconflicting%2Bresults%2Bof%2Btwo%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022714%26rft_id%253Dinfo%253Apmid%252F21829489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829489&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A general tendency for conservation of protein length across eukaryotic kingdoms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Apmid%252F15371528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIyLzEvMTQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.88" data-doi="10.1038/nature05260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worley</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Insights into social insects from the genome of the honeybee <em>Apis mellifera</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D443%26rft.issue%253D7114%26rft.spage%253D931%26rft.epage%253D949%26rft.atitle%253DInsights%2Binto%2Bsocial%2Binsects%2Bfrom%2Bthe%2Bgenome%2Bof%2Bthe%2Bhoneybee%2BApis%2Bmellifera.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05260%26rft_id%253Dinfo%253Apmid%252F17073008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17073008&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241523400044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.89" data-doi="10.1093/molbev/mst143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary dynamics of sex-biased genes in a hermaphrodite fungus.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2435</span>–<span class="cit-lpage">2446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst143%26rft_id%253Dinfo%253Apmid%252F23966547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMS8yNDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.90" data-doi="10.1186/1471-2164-8-169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krochko</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gender-specific selection on codon usage in plant genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWhittle%26rft.auinit1%253DC.%2BA.%26rft.volume%253D8%26rft.spage%253D169%26rft.epage%253D169%26rft.atitle%253DGender-specific%2Bselection%2Bon%2Bcodon%2Busage%2Bin%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-169%26rft_id%253Dinfo%253Apmid%252F17567911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17567911&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.91" data-doi="10.1093/gbe/evr018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Evolution of synonymous codon usage in <em>Neurospora tetrasperma</em> and <em>Neurospora discreta</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr018%26rft_id%253Dinfo%253Apmid%252F21402862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC8zMzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.92" data-doi="10.1128/EC.00284-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Degeneration in codon usage within the region of suppressed recombination in the mating-type chromosomes of <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">594</span>–<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00284-10%26rft_id%253Dinfo%253Apmid%252F21335530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMC80LzU5NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2307.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiegmann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeates</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">The Evolutionary Biology of Flies</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.94" data-doi="10.1093/molbev/mss184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demuth</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, Tribolium castaneum.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3755</span>–<span class="cit-lpage">3766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss184%26rft_id%253Dinfo%253Apmid%252F22826459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMi8zNzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.95" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The “effective number of codons” used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.96" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjMwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.97" data-doi="10.1093/database/bas048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Extavour</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ASGARD: an open-access database of annotated transcriptomes for emerging model arthropod species.</span> <abbr class="cit-jnl-abbrev">Database</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">bas048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbas048%26rft_id%253Dinfo%253Apmid%252F23180770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZGF0YWJhIjtzOjU6InJlc2lkIjtzOjEzOiIyMDEyLzAvYmFzMDQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMS8yMzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.98" data-doi="10.1186/1471-2164-12-581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewen-Campen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alwes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browne</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>De novo</em> assembly and characterization of a maternal and developmental transcriptome for the emerging model crustacean <em>Parhyale hawaiensis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DZeng%26rft.auinit1%253DV.%26rft.volume%253D12%26rft.spage%253D581%26rft.epage%253D581%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Band%2Bcharacterization%2Bof%2Ba%2Bmaternal%2Band%2Bdevelopmental%2Btranscriptome%2Bfor%2Bthe%2Bemerging%2Bmodel%2Bcrustacean%2BParhyale%2Bhawaiensis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-581%26rft_id%253Dinfo%253Apmid%252F22118449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118449&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.99" data-doi="10.1371/journal.pone.0061479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewen-Campen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horch</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Developmental gene discovery in a hemimetabolous insect: <em>de novo</em> assembly and annotation of a transcriptome for the cricket <em>Gryllus bimaculatus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e61479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De61479%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061479%26rft_id%253Dinfo%253Apmid%252F23671567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23671567&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2307.100" data-doi="10.1146/annurev-ento-011613-162014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gonzalez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herberstein</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmons</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Model systems, taxonomic bias, and sexual selection: beyond <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEntomol.%26rft.volume%253D59%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-011613-162014%26rft_id%253Dinfo%253Apmid%252F24160422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-011613-162014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24160422&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2307.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329608900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2307.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2299" title="Persistent Activation of the Innate Immune Response in Adult Drosophila Following Radiation Exposure During Larval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2323" title="bz-rates: A Web Tool to Estimate Mutation Rates from Fluctuation Analysis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2307" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-isqANku4JLjB7nT7O9f0XoZKQUuNEsk8pJBopWWPHWA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2307" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2307" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2307" class="active" data-icon-position="" data-hide-link-title="0">Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460171" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7rT8ymADPzCrCBxhnb8KjiSsDV7iiCPBg6OljzTn0Js" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2307.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2307" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Y46SWQyVYQM9F_FeoX7qnHqMfdWSyQYQAKhablD0Qaw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2307" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2307.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460171" id="node460171--2" data-pisa="ggg;5/11/2307" data-pisa-master="ggg;g3.115.021402" data-apath="/ggg/5/11/2307.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Carrie A. Whittle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cassandra G. Extavour</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2307-2321;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460171--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carrie A. Whittle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carrie%2BA.%2BWhittle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whittle%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarrie%2BA.%2BWhittle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cassandra G. Extavour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassandra%2BG.%2BExtavour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Extavour%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACassandra%2BG.%2BExtavour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cassandra G. Extavour</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:extavour@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">extavour@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2307" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2307&amp;text=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2307&amp;t=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2307&amp;title=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460171/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460171" id="node460171--3" data-pisa="ggg;5/11/2307" data-pisa-master="ggg;g3.115.021402" data-apath="/ggg/5/11/2307.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Codon and Amino Acid Usage Are Shaped by Selection Across Divergent Model Organisms of the Pancrustacea</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Carrie A. Whittle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cassandra G. Extavour</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2307-2321;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460171--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carrie A. Whittle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carrie%2BA.%2BWhittle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whittle%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarrie%2BA.%2BWhittle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cassandra G. Extavour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassandra%2BG.%2BExtavour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Extavour%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACassandra%2BG.%2BExtavour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2922-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cassandra G. Extavour</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:extavour@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">extavour@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2307&amp;rft.epage=2321&amp;rft.atitle=Codon%20and%20Amino%20Acid%20Usage%20Are%20Shaped%20by%20Selection%20Across%20Divergent%20Model%20Organisms%20of%20the%20Pancrustacea&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Whittle%2C+Carrie+A.&amp;rft.au=Extavour%2C+Cassandra+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460171/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460171/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460171/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460171/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460171/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460171/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460171/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460171/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460171/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460171/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460171/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460171/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2307" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2307.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2307.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/11/2307.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/11/2307.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
