<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/11/2365" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/11/2365.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/11/2365.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/11/2365.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/11/2365/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/11/2365.atom" />
<meta name="HW.pisa" content="ggg;5/11/2365" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation" />
<meta name="DC.Identifier" content="10.1534/g3.115.022111" />
<meta name="DC.Date" content="2015-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Graffelman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="This paper addresses the issue of exact-test based statistical inference for Hardy−Weinberg equilibrium in the presence of missing genotype data. Missing genotypes often are discarded when markers are tested for Hardy−Weinberg equilibrium, which can lead to bias in the statistical inference about equilibrium. Single and multiple imputation can improve inference on equilibrium. We develop tests for equilibrium in the presence of missingness by using both inbreeding coefficients (or, equivalently, χ2 statistics) and exact p -values. The analysis of a set of markers with a high missing rate from the GENEVA project on prematurity shows that exact inference on equilibrium can be altered considerably when missingness is taken into account. For markers with a high missing rate (&gt;5%), we found that both single and multiple imputation tend to diminish evidence for Hardy−Weinberg disequilibrium. Depending on the imputation method used, 6−13% of the test results changed qualitatively at the 5% level." />
<meta name="DC.Contributor" content="Jan Graffelman" />
<meta name="DC.Contributor" content="S. Nelson" />
<meta name="DC.Contributor" content="S. M. Gogarten" />
<meta name="DC.Contributor" content="B. S. Weir" />
<meta name="article:published_time" content="2015-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;This paper addresses the issue of exact-test based statistical inference for Hardy−Weinberg equilibrium in the presence of missing genotype data. Missing genotypes often are discarded when markers are tested for Hardy−Weinberg equilibrium, which can lead to bias in the statistical inference about equilibrium. Single and multiple imputation can improve inference on equilibrium. We develop tests for equilibrium in the presence of missingness by using both inbreeding coefficients (or, equivalently, χ&lt;sup&gt;2&lt;/sup&gt; statistics) and exact &lt;i&gt;p&lt;/i&gt;-values. The analysis of a set of markers with a high missing rate from the GENEVA project on prematurity shows that exact inference on equilibrium can be altered considerably when missingness is taken into account. For markers with a high missing rate (&amp;gt;5%), we found that both single and multiple imputation tend to diminish evidence for Hardy−Weinberg disequilibrium. Depending on the imputation method used, 6−13% of the test results changed qualitatively at the 5% level.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/11/01" />
<meta name="citation_mjid" content="ggg;5/11/2365" />
<meta name="citation_id" content="5/11/2365" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/11/2365" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/11/2365.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/11/2365.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/11/2365.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.022111" />
<meta name="citation_pmid" content="26377959" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2365" />
<meta name="citation_lastpage" content="2373" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jan Graffelman" />
<meta name="citation_author_institution" content="Department of Statistics and Operations Research, Universitat Politècnica de Catalunya, 08028 Barcelona, Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3900-0780" />
<meta name="citation_author" content="S. Nelson" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461" />
<meta name="citation_author" content="S. M. Gogarten" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461" />
<meta name="citation_author" content="B. S. Weir" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461" />
<meta name="citation_author_email" content="bsweir@uw.edu" />
<meta name="citation_reference" content="citation_title=Categorical Data Analysis;citation_author=A. Agresti;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Pediatric Research;citation_journal_abbrev=Pediatr. Res.;citation_author=B. W. Alleman;citation_author=S. Myking;citation_author=K. K. Ryckman;citation_author=R. Myhre;citation_author=E. Feingold;citation_title=No observed association for mitochondrial SNPs with preterm delivery and related outcomes.;citation_pages=539-544;citation_volume=72;citation_year=2012;citation_issue=5;citation_pmid=22902432;citation_doi=10.1038/pr.2012.112" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Epidemiol.;citation_author=H. A. Boyd;citation_author=G. Poulsen;citation_author=J. Wohlfahrt;citation_author=J. C. Murray;citation_author=B. Feenstra;citation_title=Maternal contributions to preterm delivery.;citation_volume=170;citation_year=2009;citation_pmid=19854807;citation_doi=10.1093/aje/kwp324" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2014" />
<meta name="citation_reference" content="citation_title=An Introduction to Population Genetics Theory;citation_author=J. F. Crow;citation_author=M. Kimura;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=O. Delaneau;citation_author=J. Marchini;citation_title=Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel.;citation_pages=3934;citation_volume=5;citation_year=2014;citation_pmid=25653097;citation_doi=10.1038/ncomms4934" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. M. Gogarten;citation_author=T. Bhangale;citation_author=M. P. Conomos;citation_author=C. A. Laurie;citation_author=C. P. McHugh;citation_title=GWASTools: an R/Bioconductor package for quality control and analysis of genome-wide association studies.;citation_volume=28;citation_year=2012;citation_pmid=23052040;citation_doi=10.1093/bioinformatics/bts610" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=I. Gomes;citation_author=A. Collins;citation_author=C. Lonjou;citation_author=N. S. Thomas;citation_author=J. Wilkinson;citation_title=Hardy-Weinberg quality control.;citation_pages=535-538;citation_volume=63;citation_year=1999;citation_issue=Pt 6;citation_pmid=11246455;citation_doi=10.1046/j.1469-1809.1999.6360535.x" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=J. Graffelman;citation_author=V. Moreno;citation_title=The mid p-value in exact tests for Hardy-Weinberg equilibrium.;citation_pages=433-448;citation_volume=12;citation_year=2013;citation_pmid=23934608" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Graffelman;citation_author=M. Sánchez;citation_author=S. Cook;citation_author=V. Moreno;citation_title=Statistical inference for Hardy-Weinberg proportions in the presence of missing genotype information.;citation_pages=e83316;citation_volume=8;citation_year=2013;citation_pmid=24391752;citation_doi=10.1371/journal.pone.0083316" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=An exact test for randomness of mating.;citation_pages=631-635;citation_volume=52;citation_year=1954;citation_doi=10.1007/BF02985085" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=L. Hosking;citation_author=S. Lumsden;citation_author=K. Lewis;citation_author=A. Yeo;citation_author=L. McCarthy;citation_title=Detection of genotyping errors by Hardy-Weinberg equilibrium testing.;citation_pages=395-399;citation_volume=12;citation_year=2004;citation_issue=5;citation_pmid=14872201;citation_doi=10.1038/sj.ejhg.5201164" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. C. Laurie;citation_author=K. Doheny;citation_author=D. Mirel;citation_author=E. Pugh;citation_author=L. Bierut;citation_title=Quality control and quality assurance in genotypic data for genome-wide association studies.;citation_pages=591-602;citation_volume=34;citation_year=2010;citation_issue=6;citation_pmid=20718045;citation_doi=10.1002/gepi.20516" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=H. Levene;citation_title=On a matching problem arising in genetics.;citation_pages=91-94;citation_volume=20;citation_year=1949;citation_doi=10.1214/aoms/1177730093" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_title=Statistical Analysis with Missing Data;citation_author=R. J. A. Little;citation_author=D. B. Rubin;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Stat. Med.;citation_author=V. Liublinska;citation_author=D. B. Rubin;citation_title=Sensitivity analysis for a partially missing binary outcome in a two-arm randomized clinical trial.;citation_pages=4170-4185;citation_volume=33;citation_year=2014;citation_pmid=24845086;citation_doi=10.1002/sim.6197" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. M. Nielsen;citation_author=M. G. Ehm;citation_author=B. S. Weir;citation_title=Detecting marker-disease association by testing for Hardy-Weinberg disequilibrium at a marker locus.;citation_pages=1531-1540;citation_volume=63;citation_year=1998;citation_issue=5;citation_pmid=9867708;citation_doi=10.1086/302114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. V. Rohlfs;citation_author=B. S. Weir;citation_title=Distributions of Hardy-Weinberg equilibrium test statistics.;citation_volume=180;citation_year=2008;citation_pmid=18791257;citation_doi=10.1534/genetics.108.088005" />
<meta name="citation_reference" content="citation_title=Multiple Imputation for Nonresponse in Surveys;citation_author=D. B. Rubin;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=The Journal of pediatrics;citation_journal_abbrev=J Pediatr;citation_author=K. K. Ryckman;citation_author=B. Feenstra;citation_author=J. R. Shaffer;citation_author=E. N. Bream;citation_author=F. Geller;citation_title=Replication of a genome-wide association study of birth weight in preterm neonates.;citation_pages=19-24.e4;citation_volume=160;citation_year=2012;citation_issue=1;citation_pmid=21885063;citation_doi=10.1016/j.jpeds.2011.07.038" />
<meta name="citation_reference" content="citation_title=Analysis of Incomplete Multivariate Data;citation_author=J. L. Schafer;citation_year=1997" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_author=Abecasis. G. R.;citation_author=D. Altshuler;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=R. M. Durbin;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_title=Flexible Imputation of Missing Data;citation_author=S. van Buuren;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=S. van Buuren;citation_author=K. Groothuis-Oudshoorn;citation_title=Mice: multivariate imputation by chained equations in R.;citation_pages=1-67;citation_volume=45;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis II;citation_author=B. S. Weir;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Stat. Biosci.;citation_author=B. S. Weir;citation_title=Interpreting whole-genome marker data.;citation_pages=316-329;citation_volume=5;citation_year=2013;citation_doi=10.1007/s12561-013-9090-0" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. E. Wigginton;citation_author=D. J. Cutler;citation_author=G. R. Abecasis;citation_title=A note on exact tests of Hardy-Weinberg equilibrium.;citation_pages=887-893;citation_volume=76;citation_year=2005;citation_issue=5;citation_pmid=15789306;citation_doi=10.1086/429864" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
<meta name="twitter:description" content="This paper addresses the issue of exact-test based statistical inference for Hardy−Weinberg equilibrium in the presence of missing genotype data. Missing genotypes often are discarded when markers are tested for Hardy−Weinberg equilibrium, which can lead to bias in the statistical inference about equilibrium. Single and multiple imputation can improve inference on equilibrium. We develop tests for equilibrium in the presence of missingness by using both inbreeding coefficients (or, equivalently, χ2 statistics) and exact p -values. The analysis of a set of markers with a high missing rate from the GENEVA project on prematurity shows that exact inference on equilibrium can be altered considerably when missingness is taken into account. For markers with a high missing rate (&gt;5%), we found that both single and multiple imputation tend to diminish evidence for Hardy−Weinberg disequilibrium. Depending on the imputation method used, 6−13% of the test results changed qualitatively at the 5% level." />
<meta name="og-title" property="og:title" content="Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/11/2365" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="This paper addresses the issue of exact-test based statistical inference for Hardy−Weinberg equilibrium in the presence of missing genotype data. Missing genotypes often are discarded when markers are tested for Hardy−Weinberg equilibrium, which can lead to bias in the statistical inference about equilibrium. Single and multiple imputation can improve inference on equilibrium. We develop tests for equilibrium in the presence of missingness by using both inbreeding coefficients (or, equivalently, χ2 statistics) and exact p -values. The analysis of a set of markers with a high missing rate from the GENEVA project on prematurity shows that exact inference on equilibrium can be altered considerably when missingness is taken into account. For markers with a high missing rate (&gt;5%), we found that both single and multiple imputation tend to diminish evidence for Hardy−Weinberg disequilibrium. Depending on the imputation method used, 6−13% of the test results changed qualitatively at the 5% level." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11.cover-source.jpg" />
    <title>Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DdXgReNpUp9gvlVyoGJIjGEwtFuWdskc27hskgo1JOE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460167","apath":"\/ggg\/5\/11\/2365.atom","pisa":"ggg;5\/11\/2365","ac":{"\/ggg\/5\/11\/2365.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/11\/2365.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2365"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/11\/2365"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/11\/2365":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h5RzoG1941\\\\\\\/nOq4QLzFcuo4BZvyRFE0h4Jw049o\\\\\\\/dW7jNZmCzEiL7HUkW6Mww+YOzhezeaH7QceeMz1ytVzPGISCXXhVAJJK+R339SVA2Qx5ZAIW9HWrbQJwP3rODMAWZP4t7o5aDro8ojY2+9o3wR\\\\\\\/BYpQhhuhd1ctNdCH\\\\\\\/AykLMTbV7kzGvImWExF3sO9p70F1QEZv2WRlgkKeOZM1wpATQtLYYhd9OPvVTEX3XTebXCGvvh986sn0XPnrUOuH\\\\\\\/Ohb0VXo7xcMvpQc1cEmMLu7P\\\\\\\/QnsGmL07kW2hCD8ZFPTR9IOMVrB0e+oaZ6B6DLK92Cp0Fc3pHzymTzNBNJjEHgieGpoHacMRReh7qYybI3cXiBKcLDWa4glJsSbihva24droNj\\\\\\\/wcHKvaNGVzqaXr\\\\\\\/kblpxqlRT\\\\\\\/kI+fetCNkSCAT4y2qqFdyIqK\\\\\\\/L+I+K5KY2jWn+H3BlIFOB0ZCYOTdjXzA6QsYtSnmxKMp+ifSpWFoaqprlSS+vIPuQshAR87Jl+XmEoxJsRQRgSDLzpuGuI2ZortRBEXRkaWrZghsMKtNKAiK32Mt05wsWBvGlPEQCSaQWQsQi7aYTGT6kSkOuAXBhFxvtPL68GoSFobDQrikVrO7IOlFV7lN\\\\\\\/C01nJl9oOMMWhUYBe5ssLO+O8RbVQyQ4jD6QCzWP5vIbPNwhss2\\\\\\\/\\\\\\\/mveQ98zNuG8E9J00usw1KIT2ozNmzRg8dLQqzp8Bsg7DG0WUqj6HOd01Q6nHVnKyI+yevEnNslQDbvQ9tfj43\\\\\\\/2Jk\\\\\\\/JCrnA0fM9BiGOTSdOBcHxD+KLnh9i\\\\\\\/+lPCgidcXaOo0IRyj0p1PPnRXWAZoDoCvuBJ5Bs93CXHdRu5TqcJGFojGL3epcYt6358o9Z27ng6Y12ZXuwjlbTPQApyzjTKwpf\\\\\\\/rV67vGYQOHLsbXNsyqgNi5SW3lbWfh46lvmMylauOavbSTmP+xKi6YqS3QsiGPJL8vaDW3OAKgOWPPSsk\\\\\\\/GNhfvxGJAkW+m1nwWTOJ7amsZ4EM1vvNIf3ClTVt0sxCjKrigSN02sGlgKKNv7XPy3OtIElCxWxj0fpXis3JqhIwDJrO37cT2bjn+7\\\\\\\/ppzknjxL+0hlbv+QeyMIlWxfLHvQlLAU9FiSiXLuqVDKOAQKR946VfY6lk81LG8nZQM4vFN21IZ8Iyaw\\\\\\\/ZyOCT4ouxJLXIFnnssDqj\\\\\\\/NJiZk6C55dk7iqU5htKZqlPCU0J\\\\\\\/N1vY01gM1orz3FDZQ\\\\\\\/5XvtSUNm8vYXVaYCI6Mr8l\\\\\\\/HTcfN\\\\\\\/kwCeOxoS\\\u0022,\\\u0022iv\\\u0022:\\\u0022FKXkODzr\\\\\\\/zde6kqJ7bx9DA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253ef3ef9afcfb4cf4d7bb22a92b11292518ef81b20e129bdfd8550a0f1a07093\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022k4YXUxyZhPM66k\\\\\\\/\\\\\\\/H60ESXr\\\\\\\/7cXxadCrQG1qAKkYX12rtta0rF000TKMhKOuDRR7wiHvYdWFzDJR4RzjFG4IR4aVuV4xWJR0fZ04Vo\\\\\\\/y28xDbjZC000dySJbwIchaIfmPa\\\\\\\/adh8mDlfLjcVec4JWZm0SlXzfDX6DN3iAHcQ+mGmDGXA8t9Ck2WG8LV8LJKjTPbah8jrKpROZS91tvyhqJsgDOyxx7Qn54qLK2zxlI\\\\\\\/y6Sl9vG6laYQzM8vRUZcUaDOZ2hLhN2aYDRlt+A8JqpX7f\\\\\\\/eIFJZXQlcV8vZZzoWLHAxk6hcHqP9nVp4GY\\\\\\\/QVLKir9hk6TJZGc9482C1qvF50KNP0Ww2zpugXX6a3vKbdP6DeG4i8EbExB7y+GXrnCLoNRBNw98dI3S6PA0McFeKpFWMRMDJArI+mIflAEBih4PWf3bxUZk9Y8gdhpL3Gt3f5x12INr3nshxE9rpwP3jrnnQpqBw1Lt2CVsBmdpUjt1ZQTeBB\\\\\\\/4pFTJMXrMLX1Z1gyHI5EZwRra0x1XmCVZIhPk7ySVvUoZYPOL6pdWy7FkuTSbAaLfDp2mzyvN+aOA1TVnrUHCoNiQR+E7\\\\\\\/bCD6dYbvc5qeBucfbVAx5xYA1Vl\\\\\\\/2tS70XVvs1z6zOszu+bnqtBPEGOHTUrR40f5Hc5Phdl\\\\\\\/74lbnDdPRZpfB7cDxGLPA9mLz5zpEQWR6Di8n+XPx\\\\\\\/icYq3Pi+rKwO9WQISqFvP713\\\\\\\/Py1gcWA259ufFOWDvCuANAU5JN5RueBVCsduD\\\\\\\/Nl26qSxltEEDTnH87GiMdCFPo7YDfcyWkXptFudBzVqK6hsD990epgNc0VBjZDsogYqjtMtJNn77a732OjIT\\\\\\\/AYW\\\\\\\/vUgTclL7bUTHpYV0baR1ci4XIFqaAaHluf0EiUmelQZfyzPwnOPwL8MLSnImw8qki4o\\\\\\\/dNqULKF+kto+AcSD3vMMeEr28XgblNwVrRMJqSFxGd68e3xCin+3kk5NPdxMOZQa1gJprBAdNCcIVq+Ua8Gfq\\\\\\\/L6Bz\\\\\\\/5Tk0NygFNTtZpMc6buemvL9zrzRZBTCHb1CpBTSmL11v86sz1oj4bo0cWA7+bDeKDdSc0DQfOQR4zoNzn6SZRsb7QcT6yebeVboiq5lY50DMfngGoiTIOwXnXfA8+xM5HThTilsh5MFBrfXg6ZypojvHhSU4FE1IoJX5qHyrRmoUHFD0uNnE\\\\\\\/KgqmIXnCSkJ4BDcjEd2EfdrTc9HxHimK3ccimzoHUbmDGRBRo6w\\\\\\\/tGf0VN\\\\\\\/fzlXEIjNRmzUZdVjATE0l5DNq9UlNU7HRQ\\\\\\\/Yzr3QgdhAYSu8N5m9WJBBt\\\\\\\/u0eW4jCbY0PI5hkWPBhR2\\\\\\\/\\\\\\\/Z0ReC9C7aJhVGPmrg6nMjZOGQ\\\\\\\/BK1\\\\\\\/qffzVs5MABsVIU04N7MkXkBNGRsakVfXEHY3vllddHVv7nsyPS5lNTVn51XBZQUJWFyvaxplqAiygVj2HhWyPAcBiLWvFs1Lo\\\\\\\/E9JKkHRcizVGI4jD92sh0z7Xm0XxgUO\\\\\\\/dB1ayMb9UnBHI7N72VJDgPhEY\\\\\\\/LJt4pIoJiZbjI4REP5Z33smrubatZWuvgBisIwuZTjIQE7dPJIFBMECzpb7Hl7OZbtKmnA+5FAqMXzUHGUe\\\\\\\/VZmnn4L1RetIrv6vi3WTFwxLK11swCjhTWSamKRKxe5ak1yrgoMJkFD6175GnlwBybBWjF9pPgS2SCvqNC8MWBjYZTpcprCkIgRJB6GaWU\\\\\\\/4qmZhiYrwrBpd5WYa53rdT0KS8YCI4tCqlxON4a97FOBnaXAcnYTcdCOXtewAn1\\\\\\\/0TzIszNL\\\\\\\/7jEjvML+nemKiYzjLA5NGL5Cu\\\\\\\/lh7b3Sgj7VQLdRhyI6rB5zouJ1pp8qFYnQNC8tiecNSpCYKDnAmtXbBsVKqgNwSvC\\\\\\\/3URvZP9D\\\\\\\/12BfEn5mjpqsZwPK44glcWGFkcFlmGVg7Kd7iz5zbFo7tXZLqQWyVoFFWcMELc4mSxhNb9cbt2vgSwX9Cti4VUQwZmwiuaRbYzSeAPEVA8zeo3wQh0\\\\\\\/SLD\\\\\\\/Ni4zKJuRavrNbZey9TEAI1SoIjc1mFs7AxlpE\\\\\\\/HvaMqqExCCX7ACaQUNpQlRHNsO5iQeepvRV1QxR2tAIhDfbMyt7u5Sf5W1\\\\\\\/xlVZm5r8XCKC7n6w+8zt3tRJggTj47nT9+zeTAIcIUgVM9s6WEqAqiOFSwOAKGDdg4ydsgj3D6QroATWZ9L9RZ1Sd5py6U+YcuILt6HIGn803i\\\\\\\/5DvuN5eA7Po4pqg4lE1LNGfLs+6XEmK1j4ydAE2rV3AP6RBqbMXqHbMT6avID6S2aJQ+HWAfDmCDCFDdig+4YfO932196xho3L9Sxwh3X+isX73BnfiIQuGDstSKX+nm\\\\\\\/dPKCw9iTCczBnX5sVuQsKNi6bVwiFVjRE+lnrnz8U\\\\\\\/kzBLzGWqkQkNjlLr+NVHaNLs17iVvUkrD1ffK4fIO8mMwJ90GNUR9P2Aqr5ErGB2HIyhK5bHaR73ErKA7L7d2dKTuI7bVa+bK3VW2TY7O+siCkkK4biWuQN1C2MoVoNJ+Mlwr3JlWSbLJvR0ohV6VpprEptOyOViS2QcwuohGZUbUA16fH5GEQ1eRCL9X3oaMxirdR0tc6UF7oKNyAni6HzQ9FP\\\\\\\/ovrpdRiCNVmSwLYvkUHB2gw2uz7kIBwt28s3Jyh16Zlp2+KZBLhqgHoeCb+sMzVKrgkAS9uEpydgxhl\\\\\\\/gbCX7ooXavGhWaHy0hUp0aZNlzlOyiofcgWg4R57BfCTC5jJUNiY8L3I+Ig18gjxNAGTvzcuyJoJgW+EnWHpgnFti5To+EG4YIMYnWs+kFTDtTXE2dm\\\\\\\/Um9QTgpLOpDLDOAYWYjMPCR4j3dXBjTF\\\\\\\/qnCViNKk14HCd3f0dp2YrKyU49qRzZ5lkiOT4s+aPTD2abS1S6Qt0JBVdU5gMIFqvP\\\\\\\/VgHXU6VcPzKWIVwVvCu9iVmdN4syjT88\\\\\\\/jvxK\\\\\\\/WbAUcGWlsSDtwdyPybSsF86iTHzWDg9LHctg0Z7uoSpxohmbsZYnAuppJ3C5uzwnRt5mAPwlQmfmVPp9gNYpKjhpqgs4JJ9iM6qEfJirRWdF9Ld9yjg8IPiYhsxyTlk+44NNsZMZgzdMhqTZ0IAzReETvikjpTt++JhbB8e3LQopxAXkEamxQqcbgDgYFhCXn3uKu+tEAye2VHIj79NTqpRBxDyGwrA52Z45QEs+qk4JdKDn5Lldf4Pm6X1rj9xVaKX8ytSAVZECuOZME4vs3rPBN5Uc+MQKQoYbpOSLPU324XMR6xzeUzG2\\\\\\\/px4JJDZad\\\\\\\/544j6bfDAmeqvxVZ\\\\\\\/5Zcj1wZcd0t3hzNNXE3jQN\\\\\\\/BHwcCZVXiSX8VjEz1HW15oUIETLD8bnJW2B+SnaTCDE7E\\\\\\\/T0Jcgae\\\\\\\/ecG9\\\\\\\/rAnwIiY8A2EDyOF8vye\\\\\\\/nUZBDxhaY+czpP75xcl76893qE34xyxOv4k4bZLu9\\\\\\\/lyDf3piMwlNbEuq7LW9tpVK0FVyG4T52GZSERZADVbVLa5kzwvudosGCXDr5JYw86ZsiZ0Tx7AMyaindDuUyo4aNPsGDBBWLkdYQL2TAPffyhywyLn2xDqX4MFDuaXlsDSIIQxbVGIPJeD5xmrWx3AvaI7I6xCH6QknEXVT\\\\\\\/CxMsaVhtc2Wak9a3BVUg2MxUD60xkFnSruFm\\\\\\\/ko\\\\\\\/zx+MgH6rKRM5Wajnbzopf6IW\\\\\\\/CxBXEMyKkMGOEZgoZu+vVx+H2dd1\\\\\\\/sBXp9W7Hd0dbEdKojCeWp62i4h7E50K0bTDqxadVLN8+zWZuVtlYtL7REt+LstHAObwVDZpO8tlwnK1MlbiIR5mmnu13yvoWLmbGj4WBWpXXm3YInNJ9DwHA6tDHnXqkApJ\\\\\\\/gxBcjP9GUlRrS7g3xCiuQtYoFZNiP\\\\\\\/Ke0h9A9VzZMwnUM6feW8Kr2+a4j1MgbKaHINa6vzr96WhwP2Xl7edjVBBFV\\\\\\\/P49GRAOv\\\\\\\/C4eq4mYEryKsVRq78htyiMuFEkZwtvaML5vcAcrmlnRpmOqnBBLBF\\\\\\\/ytzgrofZsIJ0ZbhZn9ZMlSMX7khMqizY5Fqdbw++0di2\\\\\\\/PPpCtpS9i3sTH\\\\\\\/buzmub0DpH79Gv0Yxe7+xMGdORmusGsOH4Dts5ZJSBA8W3qomRXp50tvOozLmAPGyChuGmvGf9jb9GntIQLVMuRWEeIxmN6W8O1ON1yLfj92Takzpgs0TVtyJdDXjgqm3ELM9gJ5OPhQTL\\\\\\\/m3GNftxN5aKSTVdZ\\\\\\\/tsn+A\\\\\\\/8+HhnHkMKX0C5ee\\\\\\\/KiRomKCgDanlXppCnNf9dltazrdsnwc57ybsmjFAA0eVd817+eDcGOx1togZV53x8pA0GNznfyDRxtvroNlZA\\\\\\\/zSlN+8O2mED6YhTJDXBoFpRttFQh3xH5J22zWoxLsnWVf+hdoDvYZN6HL3avFbhh3APlyyK0KLsJ0LXbeicncdwzvYWCFXQIw0IRJLuOh0e5pDddRWDobA5dbBoxbcs0B0PVTiIXSRiMWmx2CBP7P2z5qyMk3cTsyWoQbLwGOUT6Kqnw3RsRR6K+oBQie6bqoEeGG7GRQkWqqeIhbbtlgsEm5pzmaXQoi8EJx53zItrbFi0qLnDhqWMZ\\\\\\\/IAchKmaP7lpn7Car1GkO0oKe3O3yVFXIsX2wuCD7+yRMdWQNPYSPv\\\\\\\/yI2YwMJw4w1lPnDxzJx0NQUWZUX6F1L8gxuWzjiMMW79puj5JXjtdc\\\\\\\/Z7zotVRo64bzd\\\\\\\/sJQy0f84xutZGMnsmVeTuQ4wV8jrm1PJwZxavHlln2GfYqYa4MuG1Vf+hb4p0qr+M33TgiR\\\\\\\/jTpKGiyTEz8+OTtav6\\\\\\\/6PF55Lvb7eRJRPwc0whwW+HyMPJMOczFOBbHBvHkHgB6lX6JZjFNTdic7lVD3crrNORTHJvn5jUy2nIawe3NX3hWPmHZpL8w0VUsK3DGdTyWs8ntlBO6iHKu4563WMfijp+wEnAHtsF2GbgGtFdVmQPgfqqODtrYCqxRqVgjO6JzaDWqg4EoLM6mGg\\\\\\\/sB7dv3hmd18GBVCFgo7GVfuT2XfkWrjkItMiGnQSaTojtfoAGm0i00WiqnW2AO392SxOp7fTpFOQ6PiM5sbCnF5psr4WYYI1DKqgQQTOIpJGL4UG4PJHZR5PS2uTyDC74Fwwb1t9tKwtwFVLSvjexPdZvpePhtcnXvIUjBkLGQYuxPS2iC5IZl1EDxC6luSt5twHN1A\\\\\\\/AETQetxA1XU8t0i34WzMfFpVurLr9Fa4LFMEVmoQ5v\\\\\\\/oUEMDo6\\\\\\\/CKnGAIR+g5Z2dXaNnKcUKRn8amwuTrqRTMq4SGZmwYldkHyGl2MofzrwT3iZky00HtxXEKQFwDkTtkObCz\\\\\\\/IhQu4RZRpd\\\\\\\/Jr4vhQERnUZY\\\\\\\/hK91wfLyTLNYMb77X\\\\\\\/MKS80c7ACvgpVYzkM5nuPqIfN+zIRfCGIY4e1ZVNoGVIbk6rglnUFhCboiUMMljxJiF2aIqNG92+2nn6umBslxb4xbk0CczeL8TirXLrD1e6YoNnSp3U6xF8kAtFcCexq9BZlqDL6FxfW309oxVe0ozxcm5fpedgL6t+\\\\\\\/8s3IxfWYpS3eb0e8fojLPaWiNm8IK+9b1AmQJ1MttwSUBewbGRV\\\\\\\/WBCmiNSF2MKfP6lS1o4AJ7o6alrsLtGfSLaAPxG+tEqhSyt1SCaONmpWPEH+sVciAKC94eLa8ft5sp7KqGGPaah7CsyG+r\\\\\\\/ETr8NfBnzIrzDjt+e2bpoMeEoU8hN4H76h8438REoN7BjzJvuv4XNQjiiqZtCvwbYuQim9tXdmFjqDSJd8cnSt+zSKbRUK+wQX1XhBYW0\\\\\\\/71bz0X8SgiOnF42nckv3RXnG7DYMcgZDJmKTKP7pRZNwjCgAbfZMGev6Gc9Ny1Ze4X9wu2ZxF5f6FzM2vDhk06AGKFDKsX3Mn+wZ+2h9PyFzYcd0Pxb+gdVxtDpIV4W8O01wul8BSUEanWY3GCBVXwjNrzv2F1NO03BJ6\\\\\\\/wjvAL8qhaa5z0ZHUO5dczz0uwei4tOQ8yzyIRxd4gsl56R\\\\\\\/BeAsHgfcfaok08Df5V4vUUmKA2Qu\\\\\\\/a5iFP86D4uIvWhjz8wy465TAf+Pgvfz7LmlvtKYDNwtfXKS+JM7rZSZ4iOEzB62cc0uMv1EdHQFBK6XbF7YeyiRByOyxx\\\\\\\/hcdc2\\\\\\\/NP60tvNikBxuS1PynfQCFwhjHpDVwqm0ubCbaq6Xe5Pr8VdJ5T\\\\\\\/q1uwVOm3sbldrc6SvfoNk3EG0DbGzSXzVikZBoUm8XlUiQFAYa+2fUroiVM8HjB6qv60cMfEHBUQZSMTlqo8a300S03lbV\\\\\\\/3QlL0y914jOyaWwF0KkNSB2cwiXLceOa83HPiaytOSF40gYqSBo4DTEdn+u2WGOpGP14KA5Wd3Mn11cW17HSHyu4GpZMg\\\\\\\/PY4hmViKRD7NaZpLhlcoFT7T\\\\\\\/1vuF\\\\\\\/XlKfJeJ2D2qe+y9ibd7qNW0GmVe9gIyerpRyAIvxp2fShYGmCx9bJAMkMsvhOScEAQz57PiJLo1LWuUCYj4l6FGJKg76ihBt1QNqmU3AnMQGEQTJY1sKCde8rAjaoUzKFCEHnMfITj5tWGpRXHWzqQRUsoj3c\\\\\\\/aaRQeMb3DhqsCcFM8EqjM8fhNJviPzlq\\\\\\\/KbncrYeHt89hFylSSueGlmxgkfGqE+u9mQ+ttUqTrs357n03JK5wFZ3ejWV4NbvmWT+SxpkuW3pbbNTAsTYp3n+dfU1wWZG0+AYspzh2bhJi4fEuSrmdR1taoQBvIos9hR84GvEuBCYvZThnmQZGjXSJA2bWEmWQnRK6evcWX+I\\\\\\\/lVu7M+QE6lmFcJO1pIDiZ8\\\\\\\/UW37d7lGlca1FEeQJZFe4oAxr87p3M5xlMIttwJauGUvy7yLwVDxKqCHEWYR1\\\\\\\/5f\\\\\\\/A94toRA5wgqC66pxF1+crelVSb04XWYkDDX4ImqgiImGv1ydXKVGyNxdXM\\\\\\\/afiox\\\\\\\/nmBEe9iQ\\\\\\\/7LkjeErzSQkI6\\\\\\\/BF78eJLULHjM35WlhTM+zm6\\\\\\\/pv2szkvgqAKU0swN+7c9NQdhMVILbZfo4iuslqrCiVljtv931yZqb4pn9qWQsXnQjAjGs0nQcf2ED7WDwWgbAhWtaD7A0dstNV+TP0K6P467W6iLECJAjOkA13cwV5\\\\\\\/pQUFbCo2Q96Db\\\\\\\/VntgSPj3+u2d+7Jo9mzY3M0F2TFbEtZ69N5fMQlqAh8L8n\\\\\\\/\\\\\\\/mqzIW5eQrIQQLpSNRK5M6pMyDTfQqfoQG0WdYePiGocN\\\\\\\/Ule3sJWynyyH6yE8Dvaz94i1DIa8CPZlaPpq0nD1s0KskDf7eDqQPfhmhdR66Ay7ymqh2V4\\\\\\\/E8fS5464BGXJYwbkybLRdb\\\\\\\/4ZjIOHIy8ewYQtoPetu8oUtQJcpQovi8qYd2m0N0SU7wmSyjWD4W2odDuKgwQiona08lBGVF8hODq2nhuIyUr8OQoCrBD3YYZ4zjfuk31yC5SxgSAUCkGFrq2khbHz4h7wNK0\\\\\\\/8S3fhx9iijNphzaExvlQqokxoE6qXaLVNZvPBSzeH\\\\\\\/eYLLUg5TIOZx\\\\\\\/q5122YW+2gM97ya9nPT4YTOSBZXMPMPvWtvbw8SeX89\\\\\\\/zlmw1w\\\\\\\/AFegry6hNAGL6mBzyBuirVHH8Zvdtec9cNP+KPRr7qb3Sis8fhTmrz8wb4QGBu6ApTxM8ANSFwHSPy1bqVHaD6dce6Srz9qS40WRV34LDMfU0TDwGcM2ID\\\\\\\/ZkeMvIkxh8nk2DcAHJiU42c0wAS2vq8OrOrsyTyFrzlSJIPCTowt0m9UjJjeGifW9QS05hjKLRz4nc\\\\\\\/9RboCWgQhmoX5FpO9+zuDZz0U1MS2rLXiUaojTvEMXD1Xc001JCcA1s+ubwdsN20tUtJ0s\\\\\\\/+qBzYT6DEqCsQ0ERiw2I21y6rIvgM4jBzq3KMjv2YXcozznUm2KpQ5yXl86OYSdfeogPdaug2qGTA1hvzI6iZWGD8ABV\\\\\\\/Hhc5es5GEICqiio1pldCdUCqB+7LN9TNTTrO3WkaUML3qOBFOzP6XrlrhZU34Xa4IWwlKew5CziQZOqW4HzauYKUtjwXACRMPA5f\\\\\\\/dt6\\\\\\\/cknoV6I5nbnIqhKgXuvWVW\\\\\\\/mtlT9K3zXf3\\\\\\\/xrBFZgfiKILadOSpIwyjxzSbwufRUa16YuKIUQ7+221M0Xd145EVpFzL4+aDxoiGPMZW8GBlvpNvaCRhyofve0apjnc7OltVmy2EDRdlcEBIl+MaBCWedQvV7bB06xWo75C9Yg7ea1NtVP6SLuTNdhHPtOu2KrPzt8KMjReiRZiCfhZ38Vmhc4ik+Ex\\\\\\\/EDJe0zMYZzGFRCAkpIvFnya8OAb3QBTnUVdzX+eYsQqF09Zy5dFv3oWg6GgXA9eMPzrZKZmcnKQxwOZ5QHiF5NndOvrP3p\\\\\\\/vQQD7TdU53JUtlU5Bjrwf5mSVNDv7CoPBKrGVXBwbl0lLM4adhnvN6+lQZUbS3a5dauWwiDrax23\\\\\\\/BII4Nw5r9AdCN5lRNAznhgrJcH8oxDv5l8BjUh8o6faOpckRJp37xvCGYw86iXRduwDT3d1K+UwOG6NrexTwKFe+p0Or1I8XV3U01z2qLJwczBFK19KAlIaQoCcXCldovVlBaPIqRaFYpqrxR2L5qz9fSaS02vnOSlcn3YuLhu8FZNo4gWVehTLWktAn9sFE5lZjWCS4s+0SC9BAUk081Ab52PK2rcQ8EN0KdHwSKlA+F\\\\\\\/HKLzVNlJaImiq7tqS0+edzfasGLjr2w\\\\\\\/FCNrWnAnZvMsIE3IIyZvms+K13gTv89M9wdN9j47w72ESZSxS6hfkwqSXMm7kqtJv3vy5dl1si4e58VZV2JsEUUgbupe0MJ2YcWhYTPz3jTxb51q7\\\\\\\/dEdU4OUYuRqhN9KD+jcS4g1vNhMCJSowcCOobiABN5gWGp5NQHy75ss6\\\\\\\/kMnZVmxlK\\\\\\\/6AtuLCpNVUK6hzV80efiRdOxS4c7\\\\\\\/zo2JfGr7kDoCNoEATpVwea1Z8ZpT8RTHvHAL8jWIVU8RMqyhIYyj\\\\\\\/V1NtuKx4h4IC7mqBEEd77nHy++vTeykFpMqmcgGMc5vCk8kUyEVHqC1L2YAC6j38VNSTdodlp\\\\\\\/U4LiwO1RnRilOVXF781Y07vZChpWzkfgoVAsb7omjJ1JvAAllvSq9wYlYiFl2TSzB\\\\\\\/rZ9KBlhIbxVvXTWU6jKofM6Xpdjb31j9uiookl+nI8wkdHpT2xvYNNFGNl69kQDYAMdGo+m5pHxs0ct\\\\\\\/PR3c53YzFQxkaHzYwZqX2NsqLWLJ4ZY5UQ7hio3T40WgglDuZ557jek4m+b9rhbELtL9ZmBo0gx\\\\\\\/D7AX+90H\\\\\\\/AH4Aqa75PtGqvNfC8f\\\\\\\/\\\\\\\/x+UedYNqwi4wGo3Mp0gYVft4kpn3uGJNiZYXtlkMvT2MwqnFFDy7UfqOiDeWnAmDm7hhGsVcdb5wgOF3yVwYj1pmUQREYVeA7SQWgwjZOrILIHFgH0t4cxMDCWYlWRgo0YCcQzsaqYAtbsijuSFxtFJgjU\\\\\\\/oQHtX0bMX6Et+mVn5vBmdu6ERhcK\\\\\\\/xxELRuOgl7yjRS1HTd1vcjx4VoIlWwIWeZ8MPBgfkuuHH\\\\\\\/MsoHDz6P8mmHJdYwULo8hiJzL9pz\\\\\\\/BUSasDGtuouJZx\\\\\\\/0MRgtY6KBooxUhDOIHlBpcmifKm4TXcSGC0VNuptIEKNWBYfSdlvsXft4v685CME3Tbol\\\\\\\/uaoa0FBYiU20AHdx\\\\\\\/6lcyLUq1odLRZcdwEeFLhOfxXsa5qkyfo7CVdgySTgzRD+lYlm4b9BGNTOWssHzSQx370SUK8qhIcwS7nbJN\\\\\\\/bgnb5k\\\\\\\/kgmFFIPdATj984g\\\\\\\/in1\\\\\\\/QytHiprgAl11KNJ55DkhlZJlMi4rpy937C51NJr+TqZ6xbxN6s9TsTAyWH3YhR2nqOrehKtUSEpgHIazW4Sba6QQyi4mDcwiDq7gQpz1ZriW7AFqXBnLcduIebsHM+ylEaEti\\\\\\\/82w1VoRU2Qx0FTYDnpSE6kgnAzeJEsyGelDT4kVojFLc1xJFPBimoh3PGGDINxiIipikmRCYYPS1jJgitM5UkyLCUxjjiZOdsifi40fJXMM\\\\\\\/WwiWfzzceomuBmpJ6OXoFrzjfBY0vbQ75fZ2ZhGIdLb5hro0u2fouDQRDiey+jvbzWTEmqEFl5026cZjzMOrPdTSy22v6JOzzKiEdK6+\\\\\\\/5\\\\\\\/pjAEhb3ggJ4x1KjVLtRqTeyJUfRBB89GEMpaI1lvdPyXzkSbcvRsqDP\\\\\\\/U+Tp8hKG3wM+WGOlofRb6sj52aP8DutguZwRigjcE1xKR\\\\\\\/opIScGVHdWznT2HYGpPwPRLZRahfiGpTYuR2FsI13EjqUziaOUn7dhOulQbgInQIbGsweAnkb8fafYyKn8Eb1AeqVQlkosnA4wdTYiVz+Co5cQCtr\\\\\\\/zOO8SOOvtCWkpS9JDV7IhtZgmn9CyjlJ2fFp9fQkqPSmFW4pPsSX9badt9ucTsw6ddsDbMRyg4I\\\\\\\/FKpp\\\\\\\/EmYtrBh0Fi9wxIlRbrpdFeEPAdKkMzj7OFp0YgxZLTCWQOqjUHBsYhv0tGTCTImc6QZdmTMYV8qdM4MdIWmhbMKjUO\\\\\\\/5bS49zf1rZHhcUDaeVZHKOXceEhWOlka8yxadIeCMB1L5Kbuh1KrKKXCmoOznw5Db04IMCiXSt6AliaGO+8G5KJAJY+QM44tyIak51T8arhYLPSW+Neca17ELHuo+h+aNdXQo8k9Ziphr2jHkUPii47Ae2jsUj++phNHXA1SXY4GtDFru6LLm11h7ME44MKe4ZISpgjmfnie30xn4XEF+gmNgC3Gyo9ZyhgziyFPwumF\\\\\\\/oHhlmJyqiU5EJxkxezLvAq1+QSUKOl71IqC7Z9bcUIykbGY0yUOsT9Jb2xY+a9TkC3L736TNZh\\\\\\\/tulzjsN0AMM8mzedPHgZAeYJca6IsMtrOoWMSrg5PdlReIMjR\\\\\\\/I20zK8XTqKphyQ7iR5SM8Kq6pS0J\\\\\\\/hcT\\\\\\\/3wKD3udf7tbEHk\\\\\\\/GlWTfP0cdfXOzYyaq9yXMrqv705hlAk0bTvtH1PRl8qp1UIoE5ZSjqkvBOybe+eQU1fFNnUVwdp11ZinsX\\\\\\\/5MXnVshbwTlr9wxsO3tLwP8PUvq0N4VT6oPtoi1K7nYGz6nziB3lwQCxADYsfKHU2HP\\\\\\\/jRVxSpsqDFCGP8AiPvIC6TvSSKkkj+HYBA8S5Kv61b90egl9lmReF9+mp6sXUkPdxY0oplurgau7TC6OpqiV5uKtcZIIKnTHAsdyvPE4kDolLoaKKx4Zi9PTnlTgcOLHxK+nXDCUyFGc8UJuhUEFLhM5ejFzj2lUBTcSk+CG\\\\\\\/c3cWjO9Khj74fKdEy8n0VcItLdAi1kMfffh8IXZZTY\\\\\\\/ToXhb3yTmQAY0QQZpzAgIaxJSYGJ0VWHCt9I\\\\\\\/t5AxwEDhkzFQWrEg9qTkyUVySdUSeXoB15x\\\\\\\/4\\\\\\\/W+3Xh9wmTRMGoSzErhTlv24xxFRG3n\\\\\\\/sKBvZgmjtM\\\\\\\/zGPAbgRzewY0TDRl5Dk1IRXBAjd4VqURS04vKgbx5yKQwRl0P21foOeZ+rkw5XOS\\\\\\\/k50YR0R7SLb5rw8zz5ialIYIuw0dWsNyT1NteW8aVZ4gLzZv4psId2Q2R7L12o5\\\\\\\/BPRsg+EBmwTlpJGRVMZhOcyEyYmfNb0UIE7stcNIEFA8mg8qFQZbMKyC4hRNRswbqBNYJvgjZCUymVhfb3guxHZ4TM0dhgI2Mfo6scwl\\\\\\\/RhDX9TPnddMlgHnl4ZRkTBMHG+2mX+9HKdlrdic\\\\\\\/e3YzEsKxlLG2daKoq43UvWmR8nQdzMxmUSasoxY0XqYyfM24Wv5H3qVpT09vPVD1oSYqX6tIQwdxrQ4iE1m7m+MLL2MyhLxfTkoGq9XM\\\\\\\/0q4k0YaEW1KYfhsIUHn+8f8VVNb5deD+LgAgHOXFmp38rMX4nhOTHbnEINw\\\\\\\/Expc8psxvD4A3F0L9pbs16lRGiSsO8L1vhD\\\\\\\/paY7dRgSgudy+\\\\\\\/HwDjUQ3OxX+YJ6abfmh7RYun+WW5UEF1rNczstRAwviOmZ9LQiNQF44t4mXumDQIQFlO9G7SniJUhTEVtOY8fA5QAQcWiFlQZEBAOWiEHBEqT4uTPj9qxO3IBQErZvImnoWYXRmod1XeW1rrc4EiUAaX2BFsTDYcG3F96WpMLiwfiP9pj84PhmcjF2Hazxx4Tq1ZbvgRglgeKik+ztb7TmYHQXdcIyghkXQfB6r7CUg5UfrnWLeDqzCoHa\\\\\\\/FvC3yjYQsVj0J14NyCX5Dpue0oIG4AXtkjd+7kistj23cUk2JFSFAAckSATXC\\\\\\\/b8Cjv1B1emDnZDKCCiS1LDUWDhjXgCjEE\\\\\\\/Xf6ZlomPw2b0S+4ipuhDYeTN+spXmOJWQUqMDUvecOps1EwrGmbxpc6XHNUOX4+K7xs4HwCdbXcvr4PsNcwCj9D6OWLjFvgUIwm2NmBFrfon0xrR0kjfrjwp6gzrDB3KGRMsF3L5xWoAPNm9O9fbaeUA7ZpEYbZZFqTUdENCTIZYNi3BqKLo5M1r8pwypBWpdpSCjty82w2xm\\\\\\\/\\\\\\\/yDzvsBkGXcXhzZAEK88zDd3BQD69gVgH8Hm9DBzs6nddkAVHEG7wph6GSusHEvJSjLeToAE5WYy54CbM\\\\\\\/nbK3vl5KRd2SZ0ELWFc5Gzanrxc5hsRTHvP3b6u\\\\\\\/0Ge9Mciq+9fVU3VtDMGgLNyORn1PREwLDBfUGrQ5bF63xCjm04TxC0dkB9AqamHhirriqYTMNk5yeU\\\\\\\/Xc80c8zIJHMUul76Ly7pOzm5Po49zcyc+dIZiXwZVf3TUwXyIr3K40IeaPeOxA66npDIswh79NPcYs5wWVOVGDn5Be+G8Gt589DGPBDSofchtvy6G8jyM1i\\\\\\\/3UIqsAyWoKrZ5ek1VG0C9OG0UwT5Xu3cy+oTCeVRN4ucmcUheqnAB5SQub+JZSid+BH5hGt4MSY+R0nLX9kXUPiYMQ0zOR\\\\\\\/Ocyr3jeyNOGumJ76DLVDUNvOxXlRv1K+HiHUv1gLXpFF0UpdeBkk4rWnfpUKH2yMd3pNdkSfvIqa49jvvLiYQXzOMlGeyRvmMPGjnFJqsr0yJZ3VIFKkOOxDr66KPEofXy9hYbdLGzg3XNLX1byE1Fo\\\\\\\/iSTO9LDXoWy\\\\\\\/B3sWXcpMcvmBsyRgx33F4sloZt5P6Xbeu4jNWW45Z56M59YHOO9wOrMZ9eoZx+3+avgKNY7APsM8xnimYN1VvWhaLV8EYzfsckErlD0DFdiIezhRhoBWYcMpehxFwJfCvD4b14UY4W+6WGKZUtpDe+DXkK9Ewj5yUbWASaAljVwyIdoOlqlBvUHsKqTjKiUtsOepkgurc4wqBo6l51Go\\\\\\\/cUyIyruSEphWrOMJQj903U\\\\\\\/7Pd1McloQlkyHWDUuqt4y8VsAu88qodyKiPPq7EFHBuTc6Ept6oEmwm2AL6sa25lJng9Qb9dTLCJX6m8xxngrZW256DEVygjmUNbhDJHtfvl1LVa4FQdpOqzJ30\\\\\\\/rcHhuOqPOZ0fhBU2b3bF+IbQkIi05cCDmUHyCxCtpZ4Q2WDK\\\\\\\/gOGbWWn9griO0yPis4ifBFPxsD7uErsq5yFjbENePBZlkylc3lcgIJb6UQ+dp\\\\\\\/zNjGcsG7sIJUATCdyRKatjORJPSKwQMwN1Sl\\\\\\\/k3UuMh46oYFjGjfKRQJ\\\\\\\/anh9AXS7\\\\\\\/UAr6IsNVH6VK8E6oSIH+k5Xvu4prOllhicvzqlnD0eH0I52iigiT67Kw3WH\\\\\\\/YYLueyZkwOyqK02OyCiLfuHtS9CxQSr1TVvEV86hblzfr6m6yna6W2wJ\\\\\\\/SWbW0e8In7cS20ZZftYVqoq+8C3v4u++Zxnbr+OB20Kq+Fjxsq5t3UQaIxGxiFkEffHQ0oFm\\\\\\\/HHFw4\\\\\\\/OACbm0Btraad6RReoDI4BGYtveKkc6V79Nv5gZ7+E5l1ax9OEP0RuURI1OZ9RGi1ouGzTNsh7j6mlgTfJsT9pcg9DTz34xK93wI6uKABybq1jgHTsB4QgjAHi6Bha8sVa9Uq6uU0O3z4ZP91IH5eSquo6\\\\\\\/gx5raLKOXtfHjwQCR6AlPp+cHzyLwv9uNI1HQ6okrq4kdbew+PAmaQU3HprtbhYW+mPmlvRW55tGLlltVlZDF5AL9ZZE0qhffKh\\\\\\\/Tmvg3iAtThBiHMY7YlrA8wLAld+P35L4R04W4EcL8SFWaZkRGh4JK1h1DNLrsSBDNpkqyCj6tXAcYGXmXf8jEDT6V5zzcimJtmE48h9rrwF3EFMHVV9FW6pyTFXUEZKq9cbuWBYRenW3aSGyVHFMDJAHX6xhi9PpWL4x05rBcDmCEYT2ocKBMVwqj7s9nn7MtAsqFrMGNr7u988XYDvm+yDCGF8VLTp7+ii30rIP1aavMe0CLXfQ57E7srcUEOLVTxfT8qzko2IdN1Vy05qEIA6S8\\\\\\\/BQhAcOqhDiNqO0\\\\\\\/5uWErKS7+gJ40kiF0vQsW+jU3nuezjUuancKokLrw+2yIMKC5+CWK4\\\\\\\/zg7Fjtl43NXhfmMrXoqLpjFlBB6aVaB6ePOSCCjnaNCOSr3nz5aUXoG8badnroCUM2KCM4iqqnPKGR1XhH7jBFW2zq+OJyMIAEqNTq7qpczlXKNsrQR7H1lyaHneYIeCuwwg08GpAZOjtvccsN2fi4XLJosKDETJsjoUpu5wCjpugRY2DCQHR3Ljyy33Y4fsIlPVAZu0DYboXB2BTbh7XKF7b04HGt62t5LOtSfOIpON+tzKjMBL2+VBwjOUKqKwrpIhEwsIWB10TYre6Q62G4ZMclcTCaD9UOUDFPtSGVpNzKpLBkXCOwiU\\\\\\\/9WJn38FqhZyU+l0Qg3qUoBqtE28jaIeweOthnbS9x4a72FkigU++qSPGeBGB1Wmdv1HxvSGZA7+i+9PMQGKWDDo0i59ZX6E5XjYf0skQPh7GG2p7owNBQ4VSQchBVS8KbMwm2i77f1xLsQ9x5V1\\\\\\\/3eboqZkVk8gjYinl8QQtBqBy41t2D6\\\\\\\/HRTsFgLDMfty2bRRheqGT4DVQ4J2BA1V\\\\\\\/sxZMlBi\\\\\\\/y5DQPPh0o9gl\\\\\\\/krjvAlN6F1Icu\\\\\\\/1J5OcZx0FjkuzdrB9Gb1iY8ldZ1h3UQ2Sc4jcgKMfk2KxPoBLYTgs+PQbYPmL76shwmG1nHntEiqUgAsJbdRmmowrGRW61SBnwNnZ8putCmqXkvi78bBx3OixlwNBj2MUYbAxAcXcGHCiXiPGoG3dRAcI4W6JOpjt+L3aZvGYe8n5v6FdMeFKbG0LQ2hB9jc4H+NHo\\\\\\\/U6SNbJ6BVV27CRlg+zeTT5X9hJKdJ23esrT8vqFX7zAr5XGmcMQ2pjvyk4+2XEtLbXorFgEtbuaRvvRBPRD3AzXj0Br3nXn5mqSx+4UnWDzKBRkAwTqcAFFbiDrwinnwn\\\\\\\/4y2D+uiOAJiAG+jBXWeb5fM2Re9dsy+TfsF9pxUQKtSu04ZgGwebEuVlscsAr3ZGTCzmyFGDDiwbeGKXKi9jbkByzqeaeINrO771fbZ3Ka6Nw6872nUiPC3DILbsddWau9C8nPn+mSb+Mc\\\\\\\/WKQwBwY1A7CHrWw4wxuctVmf7UCdwdTUrsBzp0wu2a\\\\\\\/T5BEryBZO+ezo2YF0lAYGDSMwQLr+v42jf34zV7GikvPhJvvW6xiQmAFzpnpIZnrfshl3oPmcBWi04B4+fGDtb3ydTv\\\\\\\/nULJHC5VP\\\\\\\/\\\\\\\/yB0bFy6GVV8gL43ydcblHraY5Zj6aEhKLme3noZ0OnfrkcC6SwLZqBZ57U7ow4tXtTv17eioDWpRlayfl2lG9GW\\\\\\\/DuZkWxqyH5f2Vr\\\\\\\/NwtH\\\\\\\/efk0rnW3wIEAJT4KVrDv80vuQmWOWva8+3Vdl0EpVoRBt6Gpx982xCERE2JgEqyl3FM8k7+0IcLpIqrYSuPpwNqhcH7CZ3M\\\\\\\/UQiWyWf37WxRU7havZHPOHVOCR6jnnCkAq5Qb1VjLw1lX4Z8nNR+b+wTpodJYAmtv2xRp8TuoQnHo\\\\\\\/HlnW\\\\\\\/IMpXgQlKg6TMXjmhNF04ZgFqc7nnjzyzoBrRZ0t3czZSSWDYzc2OUjA\\\\\\\/x1eAp3UlJz0im\\\\\\\/F5cBZIB1kdx76CNIQkxog83cDJryOmWu+3zNikjzuCIbcVR+2c24a4B7Vo9E7Mq\\\\\\\/fR+d7tWTjkVOqwJetG7nrgouTrGxRzRUIgModaPG4UTQD+lOS4R7yuLkPs11BjKPhhuDEyNjd6KL0JYubMI60PnbJsESggVJ6UpP83T2gki8iGLMXCyW76iWBWTvr4hj6Ww4QI\\\\\\\/cZhLT7OSBZmCYNzn7tzuiU4CD324ZPXYzxxTZsaEDD4M\\\\\\\/egtXHiWx+RIaWulGhOapLMQ4P26koQ7hypoAOTCZCeEW4pPfjVwd+Ix4rV4j11ls0zMef7Eerc0R3reumm7fIdACW0dxDC+U5GYu23nEfYtCaw4LirI4SP+ss6cFHNaXL2BdgTAzAV5F06CSJ9bB5NQ4UNZHHhrFEV8kRQjdkrg4NXXGKQjksEzyEj4vhOEwqrPlfm2asFEqSjK6n89hxP1qLglvspxJbCqe426xRVAH6nBZAc1gtPwQjM8suAf4MUF24SvAZXdeEQOgWPgTAUAI63cPPSLDJeEdAGkmqpOfqf3sT4S026q13yXH2cZm2R+8LQdhdWSfv0gO+cetC+tXV1GDqwg03i\\\\\\\/Cpze2UN2xMg5r9iY75wULCvQy+rAAjZN\\\\\\\/IgQOyvNo0EfL6pAH8iLhpGE5dtv+bl7I1yOH9mJIrGuv8UtUpWtkccRBIOaYQaDjK9+47wrjsOPWngJrNCNQLimZkKLsXaIhV4kkcKTIKNolGo2LrCqSppsyHVXJqqRnqvd2bRkHVyt5MpMwkOJBTujRbV4SCBABgj0N0GYHjNisz3prUJ8ZY8gVvWL12AKfR1TKCLr8To64hgLcoT6niVdVfZZBd+k3JIjmjhRYRr1NkPp\\\\\\\/TACegbERLar6iTnSnzIOMKI1+B4PBx3rP6MNxRJh3jSTRFTwd3TS9aLMcS8o6o46kOyydKkAvzXd0qB5\\\\\\\/pmAQ\\\\\\\/pJqjZUViaTD9SPKh8baE3MFfUCPMrNhwsa7IPZ21RUAv4BYJ8PZ1fQkV1LwSx92tgOnGkEFtQuLy34Ue98C5\\\\\\\/DQdIt4+W7hLAspUOW1ZdgiXZRPBBrZuqxKAL57rWJ15SCkuhy9zIMbIJKwdiXDP2WXqX8oqg0p0cSM3AO\\\\\\\/HEx92NAAXRg\\\\\\\/wHJbDZYkI48Lo+iBomSo6WO0SSo3LWOxyygndTeyLGm9WT6ZBTAfmwmcFI3r\\\\\\\/Yg8V0OBuCho026Oxq2mUvTZ6bOzP66G9ysYg4yxHeBSODSrdCoSLjw2VT\\\\\\\/H3vG8E0ZdkebOV3fSDMvaO6mVpEEZ1WfC2a61V8Js9uyEs9ssXfo\\\\\\\/oBD6fRriABJn3S94si\\\\\\\/1KoaaVAvQoqU91sEz5eUuAKXOWhZUq2IkhTeVof5Jfpl1c0OmZ0XQInuhruhoDTXndpRwUumKKDCndWIXqUOmFYvn8cN6vF27W81gdJxI1EqIWqTGF+EaSl1jZZmO0X1OwVXFtmpcy+6K+wCJSXdQvr8oUXzbCCLTUJABhOdwOWSB+cvwxIFoDOqh3DgkdvHb4dNf+lGO01CebCA9sJzfHQKkwjPGfUMDpI7lrbHfNQBLpxW+LbrsO+YZlyi89DQeWleKwN7opINGr82NwXPomBjZc10Ro7hSKTG3lDZs5Eilez7a3ARIsy6gc\\\\\\\/7yAXkjYBXd2dymy0JZw2CfDt1JwBqfqO85HzMw3EGu4w+KRSysxqdw+Kny\\\\\\\/sndDCr26xxX5CfUjTEDmSLrmLu3A31Ja2aO\\\\\\\/WsK2\\\\\\\/3BwKWx+huPmvbVq4k7WbqNLkNKyo9ewPFHbBvJAvfvlMNrWXQ91xV8fNCa5a1ho7aWMK\\\\\\\/92JF12ANUVb7uB+bG4tatFxvtQhiP0bPXBH+3BV8VTOD+0QTu941J5EDpljl+bR4QwY1EHr7lsKzq+tHvBgB82bPJuSHzwbQb6F9TC2GW4BmNoL3OBfilEx8v9d6521hJx6bbpcvLPDxzg0a3keNpcrkBmc0fZ1si3JwVBaeEIirDUqMFSjeFhFKSlRcVp3nk7qtMZgSbhquvo9sG63QHbV7Gkletc7Pa\\\\\\\/mm9jCdeOo7GtgLXTVpgvdBVrp9Te7N34Zl8PY0J9Tc8gB2TlCCIE48QR0kuyO0UYpH+8KzA9OJEYt6KDd1qbrkPc8q8yDuUhICuNL9ECtXfhWt1uPH3ef4CiDSHdbR8y19IXlYA5qDR5ftI95u346zOpjgmxYOe71xOpt6ehU0csi\\\\\\\/Xj5a3XCPxd3NlWdTK54HMMC8uQlcaNHhHKqPXtASs76U9nDzcMQ3ECycoXUyel1i0yh5vAZ+6R63ujc\\\\\\\/UzmgnrO7NtnChAPwst\\\\\\\/RrNB1pijOiOHzP\\\\\\\/OxQGQlyE28OZaT82PhBXlgO1Z3ZyTh9sWU8NCQ9VmGR+NMiJGwfJn2EzS\\\\\\\/qP2bpgr3zokZIrqYC26\\\\\\\/kk62i15w5ft6V58SlWAW89eIOI+J9gRG4DwKFUhiFh7ec6FBNLgTlbnfmMNA4AGrEPvP7lASjW2Ne+bAPOvyW9Gro8Lz8uc4tLyOJ6QCxX3nywTIfQ93uzsw\\\\\\\/+MdfwUBAyHsi+QLDhNjOz+lXqvdsjS3t4g4\\\\\\\/LLfqPL3dJXeMn84zec20U\\\\\\\/RlGwKshzoU\\\\\\\/yWPA2qdmUf8WdNHJa54gM5uqWFQi47TqPl8ntSFWh35BIcJSHKqSoI0IGAx939nTxzz4hqQylvwwuLEDIicfU7z9JbYpKdOeFrzXBamYtxYWq\\\\\\\/GhqEXRRypU\\\\\\\/F+f2lCfatbCejR87xbqsNEl3J4GXSYhwhQH01zb\\\\\\\/6Ez25oH9axlrqVphNJo0m7YxoIyO+9ydexV2ti11llKhQ6CCRv1w2APkKk\\\\\\\/loY1HRiUFzIVWrDLbr\\\\\\\/pGR7VxVtppYT36ILfyCBF2l+v2XgDpt64sNNCYuunW1mPZY0zcpjzJzyiMOh65jsj6NOOutLICYxN84NKmhdn+CS9vM3dlQX4ezbHUOD8nLqDP9XbSCObMueS44MTxbor0LYf8NVqXd0940qNjzmrHorcoRjW079N+1Yl7BCbgGOK3nFp3ZLWIP\\\\\\\/FwU0a\\\\\\\/ksufWM+NETPImIFMMC8ctfPP1yhk3UCC8OJiSmek8ODd1hDCfhuIjNlOItXg940pBjD2SbwoMw438bnLeyOBXXmb4zAIVRcaqVfZuRLtO7pYCbffmFkKc\\\\\\\/Yz2AMedCbo7p20SPisf7OZRQI3243iNCFyCB3IGPQHtD8qITN2KvJOkJD9VW5At\\\\\\\/Url+eeEwQQHebbB2BnLlKMD3+eegQnetTCQh6fsoCOiFYTLFM7gTWII\\\\\\\/VAnerdoXips6Smws3l+WRVrffqbZ6ENQsKC6fgdvsy96lblhauI7EnuOkoZfoOQqxnvLh2GXbCc1r6vg3PwpVfy1fPsILcblrHo50FD+r6ewbciKpalqViQL1gk3X8tfclT7fmXHTXSQ8XyZ2tIcV\\\\\\\/50qwb9xDMe+R3zm2ezXTWVUFv5NQNtOyuNOFMAWWbI3bSEPaX3rR9vymgvJe6kCgpPsBPOvUsJvvdrAdXnohTFO3l5\\\\\\\/vlor6IE131HfZPmazfx6g5KGuHgW+YJ06WyMGXh\\\\\\\/63KT3\\\\\\\/qJ3MV1gbgR7hhrxweVOPx5aHJyVHkvIxpmroHrYKS+FymIFQH9GM8C9+3GMfj5JVKtIPl+Bt2iEIsLfj+hwNJbNNdfJMAJmvQiJ6IqCHi1OIeMVxviIboggRYoob+cTRHE6+Un8Rq1\\\\\\\/xpLM3zSKkCkot1FIqYqD2hd37e2H+LbHaVimJM1KU4hdN9DefuRWP9iJ1ebPktd6ymjwaFxu+MeBGHhYfla9JvzWluDrloVms7mplOxR3Iqsx9fD\\\\\\\/7DTRCA6ALbDaRAirkmrPU2THS1qgNtgi3RyA7YG72p4f4HgQxb3aduRjySR5fqBMutZnN7gGMxoWWdJnkaVxMPl6qV2eUtP1a0mQFZv0WlEGHugfpkoIXDZbC70di4Ai5pDCjGGOfKOZpGMcM8HW+4AuT1k8zfpVfVx4lXUA6AaBHzH19HMIYZFUIE7JMFQYUf9OW7+RkrVes3e\\\\\\\/wuRxjh1NVEPZNkBzouQfvask5mKety11z12Q6lYPsArNVbPo5t8\\\\\\\/ussPbG0GbXBtltEzpfuT8DjNrMOZoJ17qT74e0mHbC\\\\\\\/6wHbx13ub0yGSKvph2Fkx+UtzXTY6zZrTaBA9iQw361AP5LC0IB2uJN4eXvxO4ZCqrRMPppdgCh+I+I7rPXI2vdXaOqQWYN651I\\\\\\\/i9ShjXzFBGajWrTaZMKyMq3OS3b0MvBKzP73s\\\\\\\/DlCYH55cxfv0tEu+tY1UAIWTgNHjHzIssZ6pGUSgua24EYR57\\\\\\\/fsmFmX6QcJv\\\\\\\/tpLK9TIF5SLWWVRyEFdg5SLx3AUIkSDpxl9FKSUE5LrsNH178Uxs3D7BJUHmX6aUwIhroJKusIz4yoXK2oatp2oOaC4krlkHQr20bufSXTSWtEbykgYjueZ7mhImz6WfS2B\\\\\\\/5pHbb4jkfcHufWFyKMm2RMfhfF9ZiBy6nyOC8vRxR\\\\\\\/TtMlqlYzjJGS5QB9A8ujDD+yTPVCfvXZlOfwrGRnn+pNMrknSMVaPAlE5yQMTK6fmy91wJop9ZEBEzYaAq8NZCkycVN+snCIVUNanzX0fRUoTa3\\\\\\\/DKyKTVaUXE28CWgTVJjynWhKDFfBg6Mkh52mORjbYd7sMRi0D5STpPYO3wSY\\\\\\\/c6diZHFSp529bdAsQ+FUrV\\\\\\\/RoLlBiyTGL04l+Rei1pgOV\\\\\\\/sWOIc9X68c1ZGsvP7H9VjyEWLk2CnbBdLwMJ6UFFp7ZfHa2jK76jdc0SDhxiLAKdE97Wq4UMnJuvfpGrjy4\\\\\\\/LV4wClkik3cPwXE7faL6tf8ETcucRxdp8qEuuNiFdcPpDcLGCJoQUNJBlI57uSm2ChPl9\\\\\\\/KeeEOMhx1Mt0WxMTpV5fB4HzFdwzROkOpl\\\\\\\/eV7TkWYTzLTd01MQr3rzGp7X+SatMvBcJyDW+U9vUAPx0P\\\\\\\/iXaMc6m14X7OdomEURNQPsLQjIVWmgsTt6jZVPLp636t3Ip+uIIsS7DVKuCHzk7RqqYcfyHLWQ2Dp6Gm1PdOD6M+Qau5gDKoJ3kBJE9Al8OVGCpjszR69TtPs18QC+pMTJ4ky2m8cyz3epkw04prkOTG45OSBVCd\\\\\\\/XvF43AHD4Chf8Gn3kk+\\\\\\\/Dpras8rqyH8kowpDmvjlWXTKHRFb6hZQboKJiTuZ4U4gxXYJxcpw5LYFNF+boEySZ\\\\\\\/s50KmhgG4J2QvcSMi4WLZ0X5RfjM4sV6rwEJrF7bNWudXCsdEuakHHxmkmcb5e7h9lcgh\\\\\\\/\\\\\\\/I1K+TM8VXOj6XjB06wL53apN8hIiMicENQohJO4OdpTiwP\\\\\\\/mD5DHiPouZnmcxNp\\\\\\\/cJiS9ttx8TuIL\\\\\\\/+Wr\\\\\\\/DLGYANkO4frMnGHjKo4oh3P6Z\\\\\\\/omqhTbb06f3u4QSAgjifqtJ0J1BbGkrBoN\\\\\\\/jGaNHQjgIom4Gj1C9nH+UybOO\\\\\\\/wwQz6wjni+LLNvH+9R4K\\\\\\\/Ha9hFOdOKxobssEbaPUk7Rdu6ucROQDGyZjEyKb7NUUP3f45YAv3o102OP+r4b\\\\\\\/ZavS4eLzclKHfgejTaBuWed58eru5jthnHH9\\\\\\\/U1w5iKXK+gx\\\\\\\/akncjSI3K2IRL0OPI88TF6Mi4iVIxRGAya0eQmQWTglq5yyjpiaL5MB58iPCLxa8Q\\\\\\\/Amkl0M34LgaNgILv8CaBJX9TgrmXtRfqDR\\\\\\\/PSKqjok0a9CY2AXhXU7UhktPNEk6\\\\\\\/mclCGVz4gop+5Y55cBDu4uZZKQfle1NUUhqkLEssdn1Ro2U+HQXpFhqY5\\\\\\\/tICm++U+LhBTQKKba2yECIK7Sl+GY\\\\\\\/QzaojBXfQO1EZANERrgf9ZDk9wlHHenxbZ13fUJwRnzV6wEEYHxVVnN2qAgKaRS1QVSXN2frU6d0lF29xOGT9zEhavGF0+KxV+ufUiey3dE6oHvtYtVasBKDa\\\\\\\/3CiMSiwRPfdBHuyK7wzGtvGKhykrl7NXHcaFWAdKoqFKtwRpak6f\\\\\\\/ke2L+1XafS+2+o7edtAL7PmVdP0iRYtr2r9NLV\\\\\\\/8fMVb1Jc5GLtls+mbTor18lY9PQJQGSiUuac6H0gdJtSWLFK05m7WvfBJrnGq324M7QQYxa1M1dHUNHkkHQ433xa3f5hKtfcLiMUGRGpCqYI2ny6J0pvPzKV3GQ2vQ+RVssDS3bi8JNESYGikAO4KFYqozV380YCubylo0FJjjDyTNDmjWPNr46VX+DJuymUBdiVkSAePQqE\\\\\\\/UWDTMSeWu\\\\\\\/F\\\\\\\/7DbYwXV9wVy6BQZHKxvdL0pVCs+4ZtLih1k+BO6k7tt8GNiCVnxXfSOWHScXv5RBsPns0Zw7Ns+U4kdfC7NB98odGo1r1o73AnPh5PcM3zMDFl7wCIojTJnDTohYCNMy4lVhlwclxC9gJevJcquDuKOfOduNcfVuMnTIhXXx5ObarTLCTUn3kH2y9yutBJHNDAYe0qgs4SdGZ2xvy3foOaXRdqvJ4IULR0iyeWgfKw2+03gWyHK4PL6Xu50jWF0xx1eM6D6nugPjxfFWa2dLgtfqtOYafiLZZaBhKqiRmluCk7fHTf64XmCzzchGzqSvyDKMQlFm06SXGYQDGoAgEqwVOAKZr4AAaEtDbOfWIF9Ybk5znrgRv3MUOzo+mbEosCLZxaX\\\\\\\/lWlNvbcTJQlgpmitDrRXe\\\\\\\/+cOH1t6X2bVwp69L6JspR7lKGgncSSkjbSqJixzouhtPNVkkf7R538DHJHxqeBjBVhB5MXywQTSap+nFcXna5oOsS59FzzKqXmaXP7pxP9EJZU7RS\\\\\\\/bHS852Lq0\\\\\\\/KseAX0KfRvQaa7A3x++6MAWrXQCVDH8XKfRZPIxcsJzaW5C+04Yk22D7ObRhJwrGcOhnoSooIvUz8PvIZ3oDkQi116SkXrZCmEXNZiaO6bZsuK9\\\\\\\/rT2Fuq3ymMpN8xJ4FajYgBQJZoZ2krgFhkJsiU5beb59YjwVPKob9OBS540Oyc9cjmfGwBlAaUFGS7295PydEO7TJjPq0KYw3POvUK4Rd4aYzy71Xs00asoLZRGqEskDRLa6iPmv39EtWSaemSlxHh0H0bNj\\\\\\\/spxPyQPBjn5IDMjEfwrBz9lkX8xFem6Wl1xcJnfnCNVXmFc2BDyitCAFbFiJs3YfdN7f6UFI5SvjShvmaV3rwGZIB84wbkwc1tMIHyduuggsDev9Hp7ZuqHY\\\\\\\/qAc0ZFAi\\\\\\\/WHgPMbHZIuWoIeN0lzVHZWDmmTGCP82S7d2Irc7VXo44SJy\\\\\\\/coUEMXbY+WNbCoxyaf6Mo79yNapPzVS4DS6vN12tw9W0rfDHJHdtfo8pXiGW9tsvC04xgW4DXZpsLOHZjwNkS\\\\\\\/DYgj8PxocpbmvjVA7t+rj7k\\\\\\\/oEolcuG8X+tAXhONDxflKQkWHY1fCYdNpi2Qw0M02qVBdwekVVa7NcrCW5kS9DghguMbXiqo+wQS4Apkmo0spZsT\\\\\\\/VAvzIKgxAAFxMlJ4m7AQ8HAOnzzlHcE7Mswq0vW0DOJUronMwCw3I3sOPuR2FzUPXDbjIdcRrkfGTtV+QTDUzyzXIhtaPEgV7GQo8DrubsKm3Dj3s0ZGYY04T4fcrxMEUGWITso8GLHsVgR0Ea3Iva9e\\\\\\\/iA05VqGiLehOUuJlizk6VpNaWUHjbY4Ri0gW\\\\\\\/K\\\\\\\/p56wM7o0D37AMGMlfj3vVC5rCzUZ1I1Lic96CBGJKTgtIsu1oY7PJvnA2qwXVMlE8E2a8326LmZ1\\\\\\\/5gDeDXJdgA+FCK6iJQBf3oopGZo42jVTKNBHwH0BnbVQGnvQQ8z72Nt1rVXOQVaFIMz+AruR7SBai8I6WxEP3xnXA1cO8Z0p2hzCf6FXdFmFzhirh0T+zGTVyyZEhlYjio+P\\\\\\\/G7ngT5i6of1kup9kg3vSaJfLxpRQP9zalErPEtZFP4zSZcVqKKHb2U0hknCcQYqLfrX30B5pQJ1BN0puZllFo4Dvr5Ty8pQLyJcwMJDE0NR+SMdm11wre5p+KjlMNCSurIUDKzjsGdtoxen\\\\\\\/Lp98Uu\\\\\\\/6KOcmuH5e3duNqWognOjm+QlgD2Jkq1njuRwThkwd+pWg6HNjFCpdNuplEreLZ4FNw7xNyoxhBZggNVZ5+Je96LoEG+CWPDY3XPIxD8e2RxWI5fdkBN\\\\\\\/TokvYh0uqV5oCylTpXMM3MreDInOePxGe8taKeeGvlZ5gYGoTjBXh7d5s0jO5pj1FBkECJ56clUtwvRCIjxuln\\\\\\\/9vRC64NDgh1lECGyOOSN+oBo4zG3utg727QXJFlDCDMluRNV3x9dSVj8bNDYNJ4qpS89jESm7SGuRDgZWQKJlCyCrS8HWQdoh\\\\\\\/2ZXLSS4g8puHiKRwTPW\\\\\\\/YT5KbGGlWDPqkrqSQTzDi5r6+tJ0FQub6kRAbhI9IBx\\\\\\\/Zv2dTjBnmGWxXiE7ggWM+U8UPssYyO8wkAvD4qgMAY\\\\\\\/fkoKnVXbZvFZs0lZDRINup7JFj2k0jrPCjRJnmY2Gjq1JZy0HQSNKhl8LuNvwWJARR\\\\\\\/VZz+PSPtYq4zsOO2D2mLaS9HcjjElo31kmLukU4DwZp0bjHzQR\\\\\\\/t9P0453C4SB1gRgpAJ0Sdlpk7zYJZJW41vZHNkt0GEtuX7D+nxHY40BJGXLzGcmMNAHqAqFqfTAAWfpcoVaNLa4Y3cZg810TtJM+nhmA1MFDQYdue1zfEO7iLHrcPEeRqZlb4br0ss4g60yXWMMoekDxuh2L1RN+EhtdWYuZB+Xc1R5T2MnVHslJzW94MNGvsrQGWeywWbih13UlV4WwX4N\\\\\\\/fJRYFSVTMO9kn9uHzvhaqzvmCmBPEYzDq9WApgGEup15dwZbuAXvLZxNTd4dCxh5ZXDI5PMBBrYCXJ4FNvhOgISKTnoHi+9VBTnFmHQncsbvM5MPcXvv0ENXMEdgPrPZrxm0ntk4zt91dMDseYGbVznj5XwgEmZFZxr5G+dIDQrHcJxoBxdR3glZsqGH\\\\\\\/QoCvwWAlfzAKdhF9jAiSVOf3K2yvDwkp9sjLXRaAhMoLXa\\\\\\\/e\\\\\\\/\\\\\\\/uAE6EaHjSIrPU5zoQtXgMW7KThmOYk55K\\\\\\\/dpdX27Et4t1vNZ7hIDS7lc8PxWkHq7nOqPDUdLyXcMwCsaoxbJbOtCBJK4\\\\\\\/sfUmKVJ3HVNFwnkfHDKagTT1QFJo03Vg\\\\\\\/ClabNZ6qPeVli7oiPqGHWBu6AMGpVAplX2\\\\\\\/S3wr\\\\\\\/gloaZbJz0LoFFDX3FCutr7y9\\\\\\\/YQSZdcJf25UzDUwKMOFVCgyjV7fxO14KIgEZs41F3GMUOCuQFrxv4CPkJS7LKYKGSPgrMSX1Cm8RwUKx9c6geDm1OSruhScuBQurpT1bcMw3qTbj9bIQwkbVs5CqJg+AjstYemp1J3XvtG46Eii70IwhcMt0oBS9UY6E7Hs3WeRxtRqbjrddKTglfTUkYJlJxjDUu7ctM+TKyt2R27S71vMB8r7nfacMsmZDn8te3KFtCEMzeNWs3gVeft4A8Zoh0GWYfmZ2ZrHmtLYC08NF9BA8iLbCunV6+xoLS88EjbojfG+EOdSpbQimp6nKGsEYRPogZm4qQ3raOxnSKKmUCGClh0JJPypaPfrIRMJz5SgfF\\\\\\\/6K2m6IltyGLMCRq+trjFMcvUGbddjQt4m5MVMQvn2q\\\\\\\/iXMvH6VDoibM7i+T\\\\\\\/qVb3EJNxu9ydn2ChkJu2+7c3ABy+xVSURD9UD2Np5mTJ\\\\\\\/mYXDAgRlhUsCrBLAkw9F7lcqZeDH3s+0WAmxplVwqA3R2s7WKS94iR5C0IIfeJrfhD3bo12NoIE+F+X2fvI6bg2ENDD7uvgWzn7XMhs0ZJPNo0M8dV7je60wENs\\\\\\\/4MU2JuBi6xHYJvZd39\\\\\\\/JtXt6SyR6Ev\\\\\\\/LpqrzZzhDulSSuVErRU7FOdAtnvCL1AiYdN4H+8OzOtEpKqkhrEA\\\\\\\/HSiHiXKgYJMpMB6Ko+P797LWRODKjrDWzCU1huGfSL+yaOKdSzW+TIoq6eRz6Z3\\\\\\\/cJwHHGtzWpTuNlIi7b6vf6UGrbNqFkSZr4+NO3md0B2Cpcq7khYklS7cW3cIdwjmDUFHnp5+huLC8LDxB65W3rORbznOzeC+PhsDDCUdsxOrYHbHXw5VTr8IL5U97UOf1Hq4vmXbQrVsCbEqT34YF0S7pLpF4plNP+D4qJ1MRf4XhjW3lFxzOwY62Rn2SJadRnhgiZVLKKA41MR93X0UVFGK5OwuX9bCzh7B07MtO975QLrA3hekbh6kTEW1t\\\\\\\/zHtD6GeDuKR992uq4AiCwFGtJrDz6or4Z1NhAoC7qoRBu88qRKI3tyMqF1qIegndi6YrTLK30qEh7kfjwtF+2gREUwC01QvpmjayoMW2j8v7b8npnxZu7BACQnQ6yhBS\\\\\\\/kE9\\\\\\\/vXp\\\\\\\/JRlcydAZg7XlX6t65CjwSRKxQjjO5Hnh37vvrxvor6Xddv9+OTcZvzp\\\\\\\/KX5vJGWTWZpJEiyqGqqbY\\\\\\\/VExbQgZzBD6fAoctnXaHTcPTUgW3aaI7y+Tz50hj1Jix8FpUTKzRSzJNv4zo8tN1X1R6VeaKaHEaY9OwHQmKeFHIhD88dyZI+IZL6jVCaWOdwa9FlkKLxN0p\\\\\\\/MZlmfAsn6YtpWvRz8wu1tbBFy8NV7GT7EnfJXJYHnnrYUafrB5MFiHqkcQeC7RZTWa\\\\\\\/8j2YvILGboYMo5EMvol3bFx8iELut6GY7syIl5NjvNHFRlGTEwqXXblUfk3iI9Y2rcKk16bqwRdHGwtw0TCHCG12Iwd50RQ3AZdYgmubzyL\\\\\\\/1vjhNkxrMp9chvCDddYgePF3yxcLu2LodyKeIEyShZZ2OUPlPGZIn9HC+Fr+\\\\\\\/p7yk+lpH4+vwPe9M2LKAryp5JiBK3qey76tK\\\\\\\/LX9DfI+G0s1ytncBH5Nzmxmbpdg+V1Nmff6XZCymP2R5WhYkEQZVWFePlfGfPz6W\\\\\\\/zMe2gAWRFukTfg87R30UbdrpAP7lQowIyfqIv+uG4KtacbURM7B3rrm0p4jOTJRangzEDvGXPlYWUUvtpt+KgeETqH3mOjObHjL9XWKFUFi9PdjgO72PxDFwMPiXzI+rKR+yrLE\\\\\\\/MT1TBmFPi+NO53Lml8jpyTxGHgCqKetA18UM\\\\\\\/RebwXTDxCdbNmCjzlYIHBhSdnvowtm0JvSRY18\\\\\\\/fElYdDZipN1Z264AtM0prAj2R7kuMe\\\\\\\/VlWHSwG\\\\\\\/yDbIz\\\\\\\/D8Pty439bKS19\\\\\\\/nSji0LCkEh+\\\\\\\/go7afxXE\\\\\\\/V8LJok9EmEyxFXQ8hY7JQ9xjNVfw4kb+ifqXOuf7Td7GtuZTjBfb8Nrf1QOGoWkR+tFdYY4McB6WS21DU48gIgUNAkE2WqmAkGZ60U+jOBFBwKHOODrXwXWqtV2+LeOnjwS4\\\\\\\/kAvWS9im5Ry4bT3mmKtGd4thhR9F6RNrMi6qZkWv+W+ovBID27dR5T9jaWXvZ77BYzCyWthTKttLsgQx7JB+xLJB1EQtVisD6quDXqWq5CRE1tamRpu2J4R+Q7dmMhGn3cfXrPSu0xNqvUPJn\\\\\\\/KV976M3lDF\\\\\\\/lyCU6HITxqcps9v5yLkazBJJ9BN5WaVYo2iKkTWhTfapIL5GMKqbNP7WXJk5MHpyP+M4smPWaoSkUEWXoFigRf65pCDJpu+yNcfT9F20MdmGLLpmM6\\\\\\\/RwHZJkF+wk8k3PrV6ZidVZYZkuAk\\\\\\\/43B9rLJ7rlC3YKKGW1BI5\\\\\\\/+N+FF61wiFqCGVuF0\\\\\\\/Ozf+3h1rCLwFhhLDkQPuElfl5\\\\\\\/6hCOYkv2FNPiZJ9r9LDEuYQ4kAsgDYw9vtn5VtR6Of7F2agpbSVv0Gw9Izsf6a9rVDfHEEdf3rge7NBKv6FAlJBv5YBtSHMEU6qmuHkE0pZEJLDHUnaj187W1VZiZAfWtBTy\\\\\\\/ok0qamzMRI2rj5HwOPR0nsqeDPiDpD\\\\\\\/r8P\\\\\\\/BI0sCCAvdSPkG1Wco4sdjC3jxYDXgLr8azz8wLufBBM1\\\\\\\/s1KMEzdY3\\\\\\\/hILtZl\\\\\\\/azKe4sYjnf3Zf1aIzutn9oC8hmt649QM6P2qc9byIR0Bff36xCRy7yq0RJuHi45y0uId85YDXw3ERnfO2ptmNuMdERkBqsl9+RVYXF7qxxTWMqxNsfgAAUhPyknPXPqDtj1ypAbOW0BaOyfO0\\\\\\\/RsW8E\\\\\\\/QDmijb7svC+83F0Tpu31648WGtb2iGNd4OHx9HVA\\\\\\\/d7jA9BLcihz80OOPpGI1omU+iR9a5QPj+ql8+W\\\\\\\/4Y3FHax0WBudFt9OZtmrhQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022H9lKx31vtuIxYxVxft6\\\\\\\/Zw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e91a440e361bac6236c76529fbf8700b05c80dcb3bb10c5a28e46e9c41681498\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022omEe+FW6C\\\\\\\/tWonx2Eb2RQ1ojX6SXQ0xU3jRVWyllUV5Il36nUC\\\\\\\/efbimG0UIM42VpnYRJyIBGkm\\\\\\\/aNrjRxkHoL2dB77vaUy+4QJYqDJJNawnZ72nK6XXV3S1tOaAC\\\\\\\/b\\\\\\\/1uAnJZZbovVcWu9rGGmL5Rl4KyEmGpoF1Dv0da8SsCX7B7mzgzt6CLFjwggZ9FW4YIEJYQly5Lo1EHByohC4eCO\\\\\\\/pUAcqTD1T7wWpHjoa9X4Q0d8D4aldR1gd7dEoNwj8uznf61TLt6aHiUT0R\\\\\\\/D5uRIxCcYCjqCM+QR23\\\\\\\/Kd3IsNI1PQ4Q09Q5WgGao3tqo+hM9L6J8o8IO8iTPmCEu9SRSyxvqSjzUU9VwtGKcR19m31xFTSB033rOwBx31MiXSmU5rKR10CvaQXVaac5LyA5QhDAQ\\\\\\\/W5Fr4ykjH2g+UAZ0R9RJIMsmvFqf5IYnukYidGmtRGw3Ya90697iXz1ZgkH2Ie4jYrs50ifX66bk54vbJbBFKt8pWKwFrfhshi2DOM4B9xbnllcaxAUIv4UcB9Zk1HhkzizjQHWiXj2RgPRv3hRRiHdwhMx7c87UcUmI9rVdRdWDnmNhYBdrKI0T7laPDxjfVg4xvyznHw20GUNPk2Gt6ET7wxNOCt7xUSUwv9vL64gyMOFVl0CL+vLHSyGGTtOzfi9uZfpjilsV1xccBiHcfbLpUxcPB40yKq0Tkl+Z+FdZiOsdoQ5+N1boyJa\\\\\\\/DxNSYm3QIvESfQcRAlxVzfYDqX98O0injE4126uLM2nLDGQOotplCDL0ma3KZ23NbxcCu5UAYz+ygAYq6rZrxwGVUoy9zy3slGyAk2bOboDWw9B6uxExPDF+\\\\\\\/o6PulrK45NkkCUjA2VN9bgZ9uQx95mw\\\\\\\/5VlvURosFEINUAG1x5UKmcMhnilNgdEOKLNVaenNQFJ\\\\\\\/7lPGKCpCIfXRJ2tr\\\\\\\/7ydOW+86T5H24il0+g4iGVveJtGOnbjD19YhlOUvpknKjopB877PZhFbIE4etGYvtVzIBS1tZzp+++vcH\\\\\\\/0MJfHcRxCA5a7FmWz5srCzUIdAcH4RHOkdL8TI9u4Czyq3bq6dWr\\\\\\\/eRWcJDWPZRDCCd4GIOcX48bgkuMhT6LbVeGWLbcjsbodTUrYV06ouCMN0bd6Rxn+UyLMggSXd54RVn\\\\\\\/wTrK0LCW258XYABnLj5HsPh41bX+u\\\\\\\/TZ\\\\\\\/FkXmc9dxmmZG7djsP5pDC5teDFiIEe4VGghdkPpklTplCx3AcDf6q1Y6sAazS2CEGtY7G2fvArS6qkilkagWXs1gIfwjYILGblaq4\\\\\\\/nGomXHS+bQZ5deSo\\\\\\\/P12FuxQgh\\\\\\\/MCalueXpk0rnu\\\\\\\/y0S\\\\\\\/YsvmAhOomfSf0hTthMWji3YnYEBhMmPZxnCQzS3DPEZXCzxLBZ0N6JBzjozLjic3rXeo3GesYmev8+FJiX76mpzsLEwmo2RnxrtfU\\\\\\\/4+kz7I0BHaeF8NG68\\\\\\\/Q\\\\\\\/PzGf8tQ+Dm3C6dZKaL588To2IZbfyG1gm0JcHUgYEGa22Gr1I+qj1OF3KeDYnRv4sntBx1qhhJ+hSSG+PT+5GdFGC0OVoG+ryKg7jm0gv45w7RPJJg8NZVMQKlpJhor1eoZobUt0ur5ZxB7sDIUmeriKZhiNnPIzTzbvymcAH+5hxEC\\\\\\\/Th8Vgjv3JiuL89N+b3VqX7fajei24NQ84NN+1DY\\\\\\\/1ZHxUo8MNu1PGZcD4riA46sV9yBFHE+EBrXezuRrptGFRRBhAtOsV7dG8apt+At\\\\\\\/TmCHB1L77XOYpYoIfQL51bz7H\\\\\\\/d1FciKQlw06zJ6aI2F\\\\\\\/2lcRRjJSgVzeYM5CJXUe+eKsTZzMQZ1Aza4pKOhnrirxIkAykMiTfwfYwZHR8G59mObRlPjkyIgCD+SaMtE3gR979+HNMF4OF+ssQ7rN4tMnWgpy46VUtG0WSMP36kC6RkGegWTsDW6asMR78BTMMOUh+XU\\\\\\\/tu5pr8uT5j4vJ1sMQmU1QnT\\\\\\\/s9LF6eY8IHyuVc5FB+7BqFtHV1RrVxULHeVOX+lbIg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RbwvGGhgDdpgFYGs+c6u8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b06defecca5bd044b8590b4d0aadb6292e9ace7561d1922cfd569391e0247441\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460167 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2365" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ysBfsdrSgSdfVGVcRCNjHwKZvHiJqrC2n_VG2myPPTc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LVAW-WgHd89oxb22JVlHzMXAFU7UjGS1yrQ1FHrO1jo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2357" title="Genetic Linkage Mapping of Economically Important Traits in Cultivated Tetraploid Potato (Solanum tuberosum L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2375" title="Post-transcriptional Mechanisms Contribute Little to Phenotypic Variation in Snake Venoms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460167" id="node460167" data-pisa="ggg;5/11/2365" data-pisa-master="ggg;g3.115.022111" data-apath="/ggg/5/11/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan Graffelman</span>, <span class="highwire-citation-author" data-delta="1">S. Nelson</span>, <span class="highwire-citation-author" data-delta="2">S. M. Gogarten</span> and <span class="highwire-citation-author" data-delta="3">B. S. Weir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022111 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460167" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jan Graffelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics and Operations Research, Universitat Politècnica de Catalunya, 08028 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BGraffelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graffelman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BGraffelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan Graffelman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">B. S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB.%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bsweir@uw.edu" class="" data-icon-position="" data-hide-link-title="0">bsweir@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/11/2365" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460167" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460167/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2365.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460167" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460167/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/11/2365.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460167" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460167/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/11/2365.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460167" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460167/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">This paper addresses the issue of exact-test based statistical inference for Hardy−Weinberg equilibrium in the presence of missing genotype data. Missing genotypes often are discarded when markers are tested for Hardy−Weinberg equilibrium, which can lead to bias in the statistical inference about equilibrium. Single and multiple imputation can improve inference on equilibrium. We develop tests for equilibrium in the presence of missingness by using both inbreeding coefficients (or, equivalently, χ<sup>2</sup> statistics) and exact <em>p</em>-values. The analysis of a set of markers with a high missing rate from the GENEVA project on prematurity shows that exact inference on equilibrium can be altered considerably when missingness is taken into account. For markers with a high missing rate (&gt;5%), we found that both single and multiple imputation tend to diminish evidence for Hardy−Weinberg disequilibrium. Depending on the imputation method used, 6−13% of the test results changed qualitatively at the 5% level.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hardy%E2%88%92weinberg-equilibrium" class="hw-term hw-article-keyword hw-article-keyword-hardy&#x2212;weinberg-equilibrium" rel="nofollow">Hardy−Weinberg equilibrium</a></li><li class="kwd"><a href="/keyword/missing-data" class="hw-term hw-article-keyword hw-article-keyword-missing-data" rel="nofollow">missing data</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/exact-test" class="hw-term hw-article-keyword hw-article-keyword-exact-test" rel="nofollow">exact test</a></li></ul><p id="p-3">Modern genotyping platforms produce large databases with information on tremendous numbers of genetic markers used in gene−disease association studies. Typically, such data sets contain a considerable number of missing observations; we will refer to such instances as “missings.” The missing data problem pervades much of the subsequent statistical analysis of the data. There are several approaches to deal with missing genotype data, as we briefly sketch here. First, the simplest and often-used approach is just to ignore the missings and exclude these from the computations. For very small amounts of missing data, this action may be justified, because omitting a few missings in a large data set is unlikely to affect the results of the study (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Schafer 1997</a>). However, if the number of missings is substantial, then there is a loss of power because of a reduced sample size if the missings are ignored. Moreover, if missings are not missing completely at random, then ignoring missings may lead to severe bias in the results of the analysis. In recent work (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Graffelman <em>et al.</em> 2013</a>) we have shown that the conclusions of basic tests on Hardy−Weinberg equilibrium (HWE) can be mistaken if the missings simply are ignored. On the other hand, we also have shown (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Weir 2013</a>) that the proportion of HWE rejections can be much closer to nominal in large-scale datasets if single-nucleotide polymorphisms (SNPs) with any missings are removed from consideration.</p><p id="p-4">Second, missings can be imputed at the time the data are phased (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delaneau <em>et al.</em> 2014</a>) or by the use of external reference panels (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Howie <em>et al.</em> 2009</a>). In these approaches the missing values often are inferred once, and a single completed data set is then used in the subsequent statistical analyses. The imputed data set is being treated as if it was a completely observed data set. Reference panels have become a popular tool for imputing missing genotypes (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Howie <em>et al.</em> 2009</a>). However, one cannot be sure that the imputed values are the correct ones. Taking the imputed data set as being the truly observed genotypes ignores the uncertainty about the imputations in posterior analyses.</p><p id="p-5">Third, one can use multiple imputation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Little and Rubin 2002</a>) of the missing data by using statistical models that borrow information from correlated variables to impute the missings. This approach takes the uncertainty in the imputations into account. The goal of this approach is not to impute the missing genotypes once and for all, but to do correct statistical inference on the genetic parameters of interest. The missing genotypes are imputed many times, leading to many completed data sets. Each completed data set is analyzed separately by the method of interest, and then the results are combined.</p><p id="p-6">In this paper, we focus on multiple imputation as a tool for treating missings in genetic data analysis and use that approach to test markers for HWE. In genetic association studies tests for HWE are used as a tool for detecting genotype error (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gomes <em>et al.</em> 1999</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hosking <em>et al.</em> 2004</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Laurie <em>et al.</em> 2010</a>). In previous work (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Graffelman <em>et al.</em> 2013</a>) we described multiple imputation for inference on HWE based on the inbreeding coefficient. We argue below that the use of inbreeding coefficients amounts to using the χ<sup>2</sup> test statistic for HWE. Today, the state-of-the-art test for HWE is the exact test (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wigginton <em>et al.</em> 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Rohlfs and Weir 2008</a>) as this is the most powerful test. The purpose of this paper is to extend the previous results on inference with missings for HWE to exact test procedures for HWE.</p><p id="p-7">Single imputations based on reference panels like the HapMap project (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">The International Hapmap Consortium 2007</a>) or 1000 Genomes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">The 1000 Genomes Project Consortium 2010</a>) or produced by SHAPEIT (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Delaneau <em>et al.</em> 2014</a>) have become important tools in statistical genetics. We therefore dedicate some attention to comparing results obtained by multiple imputation with those obtained by using single imputations made by SHAPEIT.</p><p id="p-8">The structure of the remainder of this paper is as follows. First we outline our multiple imputation approach for inference on HWE with missings. Then we describe a database from the GENEVA project (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boyd <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ryckman <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alleman <em>et al.</em> 2012</a>) that we use as an example for our methodology and we present the results of a HWE analysis for a subset of SNPs from this project. The <em>Discussion</em> section completes this paper. References for software implementing the multiple imputation procedures outlined are provided.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">In this section we provide the basic notation and describe two approaches for inference on HWE when there are missing genotypes: single imputation (SI) and multiple imputation (MI). We restrict our attention to biallelic genetic markers with alleles A and B and let <em>p<sub>A</sub></em> and <em>p<sub>B</sub></em> be their respective allele frequencies. We define <em>n<sub>AA</sub></em>, <em>n<sub>AB</sub></em>, <em>n<sub>BB</sub></em>, and <em>n<sub>A</sub>,n<sub>B</sub></em> as the respective genotype and allele counts, and <em>n</em> as the total sample size.</p><div id="sec-2" class="subsection"><h3>Single imputation</h3><p id="p-10">Estimation of missing genotypes in genetic studies is linked to the problem of haplotype estimation. If the haplotypes of an individual are known or have been estimated, then the missing values of SNPs that form part of that haplotype can in principle be inferred. There are several computer programs available for imputation of missing genotypes such as SHAPEIT2 (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Delaneau <em>et al.</em> 2014</a>), MaCH (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2010</a>), IMPUTE2 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Howie <em>et al.</em> 2012</a>), or Beagle (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2014</a>). For imputation, haplotypes from HapMap (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">The International Hapmap Consortium 2007</a>) populations or the 1000 Genomes Project (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">The 1000 Genomes Project Consortium 2010</a>) are used as a reference panel. A comparison of these methods in terms of accuracy and required computational resources has been made (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Howie <em>et al.</em> 2009</a>) and is beyond the scope of the current paper. In this paper we adhere to SHAPEIT2 as a haplotype inference software for phasing complete chromosomes and use the results from this program to obtain single imputations of the missing genotype data.</p></div><div id="sec-3" class="subsection"><h3>Multiple imputation</h3><p id="p-11">MI is a statistical tool for dealing with missing values (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Little and Rubin 2002</a>). It requires a statistic that can be calculated for each imputed dataset. If HWE is tested by the classical χ<sup>2</sup> test, then multiple imputation can be performed with inbreeding coefficients (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Graffelman <em>et al.</em> 2013</a>). If the exact test is used for testing HWE, then multiple imputation can be performed using <em>p</em>-values from a one-sided exact test. We now sketch both these approaches.</p><div id="sec-4" class="subsection"><h4>Inbreeding coefficients:</h4><p id="p-12">Hardy−Weinberg disequilibrium can be parameterized by using the within-population inbreeding coefficient <em>f</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crow and Kimura 1970</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Weir 1996</a>), and under this parameterization the population genotype frequencies are given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-2.gif"/></noscript></span></span>, where <em>p<sub>m</sub></em> is the minor allele frequency min(<em>p<sub>A</sub></em>,<em>p<sub>B</sub></em>). If <em>f</em> = 0, then the genotype frequencies correspond to the Hardy−Weinberg proportions. For <em>f</em> &gt; 0 there is a deficiency of heterozygotes, and for <em>f</em> &lt; 0, there is an excess. The inbreeding coefficient <em>f</em> can be estimated by maximum likelihood (ML) by the use of the multinomial distribution for genotype counts, and the ML estimator and its variance are given by the following (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Weir 1996</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>We note that the ML estimator is related to the classical χ<sup>2</sup> test statistic for HWE by <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-4.gif"/></noscript></span></span>. For imputation we use a multinomial logit model that uses allele intensities and/or flanking markers as predictors. Imputed data sets are generated by the MICE algorithm (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">van Buuren and Groothuis-Oudshoorn 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">van Buuren 2012</a>). Multiple imputation yields a set of <em>m</em> complete data matrices of genotype information. To be able to perform statistical inference for HWE, inbreeding coefficients and their variances are estimated for all imputed data sets, and these estimates are combined according to Rubin’s pooling rules (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rubin 1987</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Little and Rubin 2002</a>). If <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-5.gif"/></noscript></span></span> is the estimate of <em>f</em> from the <em>i</em>th of <em>m</em> imputations, we write<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <em>W</em> is called the average within-imputation variance. Next, the between-imputation variance (<em>B</em>) and the total variance (<em>T</em>) are computed as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>A test statistic for HWE (<em>H</em><sub>0</sub>: <em>f</em> = 0) is then given by <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-8.gif"/></noscript></span></span>. Under the null, this statistic has a <em>t<sub>v</sub></em> distribution with ν degrees of freedom, ν given by<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-9.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>The multiple imputation <em>p</em>-value, denoted by <em>p<sub>mi</sub></em>, for a two-sided test for HWE after multiple imputation is given by<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-10.gif"/></noscript></span></span>Alternatively, inference can be performed by calculating a 100(1 − <em>a</em>)% confidence interval given by <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-11.gif"/></noscript></span></span>. We note that <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-12.gif"/></noscript></span></span> in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equations (2)</a> and <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">(3)</a> may be calculated by substitution of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-14.gif"/></noscript></span></span>, but we do not recommend this. Under the null hypothesis of <em>f</em> = 0, it follows that <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-15.gif"/></noscript></span></span>. If the sample estimates <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-17.gif"/></noscript></span></span> are used, then <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-18.gif"/></noscript></span></span> is often below <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-19.gif"/></noscript></span></span>, yielding a Wald statistic that is too liberal. We also point out that substitution of the sample estimate <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-20.gif"/></noscript></span></span> yields a zero variance for samples that do not contain heterozygotes, because in that case we have <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-21.gif"/></noscript></span></span>. For markers with a low MAF, samples without heterozygotes can easily arise. Typically such samples have one homozygote with a low count and the other one with a high count. We note further that a zero homozygote count also puts the estimate <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-22.gif"/></noscript></span></span> on the boundary of the parameter space because if <em>n<sub>AA</sub></em> = 0, then <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-23.gif"/></noscript></span></span>, and the latter coincides with the lower bound for <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-24.gif"/></noscript></span></span>. Instead of the use of inbreeding coefficients, the multiple imputation approach also can be applied to the χ<sup>2</sup> statistics (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-25.gif"/></noscript></span></span>) of each imputed data set. Assuming the null hypothesis to be true, these can be converted into standard normal variates by<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-26.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>and these are processed again by the usual averaging and pooling rules, with <em>W</em> = 1. Because <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-27.gif"/></noscript></span></span>, combining χ<sup>2</sup> statistics is equivalent to combining inbreeding coefficients and assuming that the variance of the inbreeding coefficient is <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-28.gif"/></noscript></span></span>.</p></div><div id="sec-5" class="subsection"><h4>Exact p-values:</h4><p id="p-13">The exact test for HWE is based on the discrete distribution of the number of heterozygotes given the allele count <em>n<sub>A</sub></em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Levene 1949</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Haldane 1954</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Weir 1996</a>):<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>The standard <em>p</em>-value for an exact test is obtained by summing the probabilities in (7) for all possible samples that are as likely or less likely under HWE than the observed sample. In this paper, we adhere to the mid <em>p</em>-value, recently proposed for use in exact HWE testing by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Graffelman and Moreno (2013)</a>. The mid <em>p</em>-value is defined as <em>half</em> the probability of the observed sample plus the probabilities of all samples more extreme than the observed one (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agresti 2002</a>). The mid <em>p</em>-value has been shown to have a rejection rate that is closer to the nominal level (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Graffelman and Moreno 2013</a>). We apply Rubin’s results for combining <em>p</em>-values from multiple imputed data sets (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Licht 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liublinska and Rubin 2014</a>). Let <em>p<sub>i</sub></em> be the <em>p</em>-value of the <em>i</em>th imputed data set. Then we obtain<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where <em>φ</em><sup>−1</sup> is the inverse of the distribution function of a standard normal random variable. If <em>p<sub>i</sub></em> has a uniform distribution, then <em>z<sub>i</sub></em> has a <em>N</em>(0,1) distribution. The <em>z</em>-statistics are averaged over the <em>m</em> imputed data sets to obtain<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-31.gif"/></noscript></span></span>and from here on the usual pooling rules are applied, where we obtain the between (<em>B</em>) and within (<em>W</em>) imputation variance with <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">equations (3)</a> and <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">(4)</a>, but replacing <em>f</em> by <em>z</em>. We have <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-32.gif"/></noscript></span></span> and calculate the final multiple imputation <em>p</em>-value (<em>p<sub>mi</sub></em>) as <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-33.gif"/></noscript></span></span>. The exact <em>p</em>-value <em>p<sub>i</sub></em> is often equal to one in exact tests with markers with a low MAF, giving <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-34.gif"/></noscript></span></span>. This would make it impossible to carry through the computations. This problem is neatly solved by using the mid <em>p</em>-value, because the latter is strictly smaller than one. If the <em>p</em>-value <em>p<sub>i</sub></em> is small, <em>z<sub>i</sub></em> will be large and positive. A set of small <em>p</em>-values will thus give a large <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-35.gif"/></noscript></span></span>, and <em>p<sub>mi</sub></em> will tend to be small as well. Conversely a set of large <em>p</em>-values will give a large but negative <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-36.gif"/></noscript></span></span>, and this will produce a large <em>p<sub>mi</sub></em>. This procedure is for one-tailed tests only. For a two-sided exact test, given only a two-sided <em>p</em>-value the correct sign of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-37.gif"/></noscript></span></span> cannot be inferred. Exact tests for HWE with missings in this paper were therefore performed twice: one test for heterozygote excess and another test for heterozygote deficiency. In most practical applications, two-sided exact tests for HWE are performed, and the fact that a test that accommodates missings by combining <em>p</em>-values is one-sided is a bit of a limitation. If a two-sided exact test is required in this setting, then a pragmatic solution to this problem is to perform both one-sided tests and to calculate a two-sided multiple imputation <em>p</em>-value as <em>p<sub>mi</sub></em> = 2min(<em>p</em><sub>mi,excess</sub>, <em>p</em><sub>mi,deficiency</sub>).</p></div></div></div><div class="section other1" id="sec-6"><h2 class="">Software</h2><p id="p-14">The procedures for inference on HWE with missing genotypes discussed in this paper are implemented in the R-package HardyWeinberg (<a href="https://cran.r-project.org/web/packages/HardyWeinberg/index.html">https://cran.r-project.org/web/packages/HardyWeinberg/index.html</a>). The multiple imputation part of the procedure is handled by the R-package MICE (<a href="https://cran.r-project.org/web/packages/mice/index.html">https://cran.r-project.org/web/packages/mice/index.html</a>).</p><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-15">The GENEVA Prematurity data are available for download from the dbGaP resource (<a href="http://www.ncbi.nlm.nih.gov/gap">http://www.ncbi.nlm.nih.gov/gap</a>). They are listed as “Genome-Wide Association Studies of Prematurity and Its Complications” and the dbGaP Study Accession number is phs000103,v1.p1. Only the 1939 mothers from 3886 mother-child pairs from the Danish sample were used in this work.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-16">In this section we describe a dataset from the GENEVA project and apply the proposed methods of inference for HWE with missing values. We first describe the dataset and then show inference on HWE using single and multiple imputation.</p><div id="sec-9" class="subsection"><h3>Description of the dataset</h3><p id="p-17">We use a subset of SNPs from the GENEVA project on Prematurity (<a href="http://www.genome.gov/27550876">www.genome.gov/27550876</a>). The original genome-wide dataset contains 657,366 SNPs typed for 3886 individuals. The dataset contains only 0.16% missing values overall, once completely missing SNPs have been eliminated. The percentage of missings per individual never exceeds 10%, and the percentage of missings per SNP never exceeds 20%, besides a subset of SNPs that was missing for all individuals. This database is filtered as follows. Only those SNPs are used that have at least 5% missing values. SNPs with a percentage of missings below this level mostly have only one or two missing values. For such SNPs, multiple imputation is unlikely to affect the statistical inference for HWE because of the large sample size used in this study. Only the mothers (female founders) in the dataset are considered, and for pairs with a first or second degree family relationship one individual was removed, in order to create a subset of independent individuals. Only autosomal SNPs are used, and the SNPs are selected to be at least 15kB apart, in order to obtain a set of approximately independent markers (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gogarten <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zheng <em>et al.</em> 2012</a>). No thresholds for the minor allele frequencies are used. Applying these filters produces a dataset of 1939 females and 677 SNPs. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> summarizes the HWE status of the 677 SNPs and represents them in a ternary plot with the 95% acceptance region for a χ<sup>2</sup> test for HWE (left panel). The figure shows that a large number of markers is out of equilibrium and reveals that disequilibrium is due mainly to a lack of heterozygotes. For comparison, we also show a ternary diagram of 677 complete SNPs that were randomly chosen from the dataset (right panel). Complete SNPs clearly show less disequilibrium.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Left panel: ternary plot for 677 single-nucleotide polymorphisms (SNPs) with &gt;5% missing. A total of 229 SNPs (34%) are significant in a &#x3C7;2 test. Right panel: 677 SNPs without missings taken at random. A total of 56 (8%) SNPs are significant in a &#x3C7;2 test. Significant markers are red and nonsignificant markers are green (&#x3B1; = 0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2071473282" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Left panel: ternary plot for 677 single-nucleotide polymorphisms (SNPs) with &gt;5% missing. A total of 229 SNPs (34%) are significant in a &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test. Right panel: 677 SNPs without missings taken at random. A total of 56 (8%) SNPs are significant in a &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test. Significant markers are red and nonsignificant markers are green (&#x3B1; = 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2365/F1.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/11/2365/F1.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Left panel: ternary plot for 677 single-nucleotide polymorphisms (SNPs) with &gt;5% missing. A total of 229 SNPs (34%) are significant in a χ<sup>2</sup> test. Right panel: 677 SNPs without missings taken at random. A total of 56 (8%) SNPs are significant in a χ<sup>2</sup> test. Significant markers are red and nonsignificant markers are green (α = 0.05).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Imputation results</h3><p id="p-19">We first present a few examples of SNPs whose inference on HWE is altered by using imputation. Next, we show some results for the full subset of 677 SNPs. Three ways of dealing with missing values are considered: discarding, single imputation by SHAPEIT and multiple imputation with a multinomial logit model using flanking markers as covariates.</p><div id="sec-11" class="subsection"><h4>Some example SNPs:</h4><p id="p-20">We treat some SNPs in detail in order to show how exact inference on HWE can be affected by imputation. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the plots of genotypes calls for four SNPs of the database as an example.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of genotype calls for four single-nucleotide polymorphisms (A: rs818284; B: rs13022866; C: rs3766263; D: rs2714888) with &gt;5% missings in the GENEVA project on prematurity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2071473282" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plots of genotype calls for four single-nucleotide polymorphisms (A: rs818284; B: rs13022866; C: rs3766263; D: rs2714888) with &gt;5% missings in the GENEVA project on prematurity.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2365/F2.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/11/2365/F2.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466878" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Plots of genotype calls for four single-nucleotide polymorphisms (A: rs818284; B: rs13022866; C: rs3766263; D: rs2714888) with &gt;5% missings in the GENEVA project on prematurity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">For these four SNPs, MI <em>p</em>-values (χ<sup>2</sup> based and exact) were obtained by performing 50 imputations using a multinomial logit model with two flanking SNPs as predictors. SI <em>p</em>-values were obtained by doing a χ<sup>2</sup> test and an exact test with the completed data obtained by SHAPEIT. Test results are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466998/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466998/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Hardy-Weinberg equilibrium statistics for 4 SNPs with more than 5% missing values</span><div class="sb-div caption-clear"></div></div></div><p id="p-24"><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> shows a SNP with a large number of missings close to the origin with approximately zero intensities, corresponding to null alleles. For this marker, allele intensities are of little use for imputing the missing genotypes. The inbreeding coefficients and HW <em>p</em>-values remain unaltered under SI. MI using the two closest flanking markers as covariates was hardly possible because the flanking markers had missings for almost the same set of individuals. SHAPEIT imputed almost all missings as AA genotypes, but this did not alter the HWE inference. Panel A represents a relatively common pattern for markers with missings that are out of HWE. In fact we found a successive set of more than 20 SNPs on chromosome 6 all having null alleles for the same set of individuals. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> shows a SNP with a significant deficiency of heterozygotes. Most missings border the AB cloud and are mostly imputed as heterozygotes by SI or MI. This lowers the inbreeding coefficient. An exact test for this marker is significant if the missings are discarded, but clearly nonsignificant if the missings are imputed. <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> shows a SNP with a large number of missings close to the AA cluster, and also missings close to the AB cluster. An exact test that discards the missings is significant, as there is an excess of heterozygotes. Statistical imputation with information from correlated flanking SNPs mostly imputes these groups of missings as AA and AB respectively. Consequently, both exact and χ<sup>2</sup>-based inference is altered, the inbreeding coefficient drops toward zero when SI or MI is used, and with imputation the marker no longer deviates significantly from HWE. <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a> shows a SNP that would be considered in HWE when missings are discarded. A large cluster of missings bordering the AA cluster is mainly imputed as AA genotypes, and when these missings are taken into account, the inbreeding coefficient becomes larger and the marker becomes significant.</p></div><div id="sec-12" class="subsection"><h4>The full subset of SNPs:</h4><p id="p-25"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the relationships between inbreeding coefficients obtained by discarding the missings and inbreeding coefficients obtained by SI and MI. For MI, we use a multinomial logit model that uses information from the two flanking markers: see <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Graffelman <em>et al.</em> (2013)</a> for more details. For each SNP, 50 imputed datasets were created with the MICE algorithm. For some SNPs the model could not be estimated because flanking SNPs had missings for the same individuals as the SNP to be imputed, or because of perfect relationships between the SNPs involved. In these cases the allele intensities (usually having less missings than SNPs) were used as covariates instead. Imputation mostly gives similar results as discarding the missings, and estimates correlate reasonably well. However, some degree of reversal of the test results is observed. There is a set of markers (blue upward triangles) that becomes non-significant when missings are imputed (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A and B</a>). There is also a set of markers that is significant under MI and nonsignificant under SI (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>), showing that the results of the two imputation procedures do not always coincide. Inbreeding coefficients obtained by MI most closely resemble the inbreeding coefficients obtained under discarding. SI generally seems to decrease the inbreeding coefficients in absolute value, and so diminishes the number of markers out of HWE.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between inbreeding coefficients. (A) Estimates obtained by discarding against estimates obtained by multiple imputation using two flanking markers (MI). (B) Estimates obtained by discarding against estimates obtained by single imputation (SI). (C) SI estimates against MI estimates. Plotting symbols and colors indicate the significance of the markers in two Hardy&#x2212;Weinberg equilibrium tests (&#x3B1; = 0.05). Red diamonds: both tests significant, green circles: both tests nonsignificant, upward blue triangles: significant in the test on the x-axis, nonsignificant for the test on the y-axis. Downward orange triangles: nonsignificant on the x-axis, significant on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2071473282" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationships between inbreeding coefficients. (A) Estimates obtained by discarding against estimates obtained by multiple imputation using two flanking markers (MI). (B) Estimates obtained by discarding against estimates obtained by single imputation (SI). (C) SI estimates against MI estimates. Plotting symbols and colors indicate the significance of the markers in two Hardy&#x2212;Weinberg equilibrium tests (&#x3B1; = 0.05). Red diamonds: both tests significant, green circles: both tests nonsignificant, upward blue triangles: significant in the test on the &lt;em&gt;x&lt;/em&gt;-axis, nonsignificant for the test on the &lt;em&gt;y&lt;/em&gt;-axis. Downward orange triangles: nonsignificant on the &lt;em&gt;x&lt;/em&gt;-axis, significant on the &lt;em&gt;y&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2365/F3.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/11/2365/F3.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466958" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Relationships between inbreeding coefficients. (A) Estimates obtained by discarding against estimates obtained by multiple imputation using two flanking markers (MI). (B) Estimates obtained by discarding against estimates obtained by single imputation (SI). (C) SI estimates against MI estimates. Plotting symbols and colors indicate the significance of the markers in two Hardy−Weinberg equilibrium tests (α = 0.05). Red diamonds: both tests significant, green circles: both tests nonsignificant, upward blue triangles: significant in the test on the <em>x</em>-axis, nonsignificant for the test on the <em>y</em>-axis. Downward orange triangles: nonsignificant on the <em>x</em>-axis, significant on the <em>y</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We consider the consequence of single and multiple imputation for the <em>p</em>-values of one-sided and two-sided exact tests for HWE. The relationships between <em>p</em>-values obtained by discarding and imputation of missings are plotted in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between exact p-values of tests for Hardy&#x2212;Weinberg equilibrium (HWE) for heterozygote deficiency, excess or either (two-sided) obtained by discarding missings and imputing missings using single imputation and multiple imputation using information from two flanking markers. Plotting symbols and colors indicate the significance of the markers in two HWE tests (&#x3B1; = 0.05). Red diamonds: both tests significant, green circles: both tests non-significant, upward blue triangles: significant for the test on the x-axis, nonsignificant for the test on the y-axis. Downward orange triangles: non-significant on the x-axis, significant on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2071473282" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationships between exact &lt;em&gt;p&lt;/em&gt;-values of tests for Hardy&#x2212;Weinberg equilibrium (HWE) for heterozygote deficiency, excess or either (two-sided) obtained by discarding missings and imputing missings using single imputation and multiple imputation using information from two flanking markers. Plotting symbols and colors indicate the significance of the markers in two HWE tests (&#x3B1; = 0.05). Red diamonds: both tests significant, green circles: both tests non-significant, upward blue triangles: significant for the test on the &lt;em&gt;x&lt;/em&gt;-axis, nonsignificant for the test on the &lt;em&gt;y&lt;/em&gt;-axis. Downward orange triangles: non-significant on the &lt;em&gt;x&lt;/em&gt;-axis, significant on the &lt;em&gt;y&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2365/F4.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/11/2365/F4.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466914" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Relationships between exact <em>p</em>-values of tests for Hardy−Weinberg equilibrium (HWE) for heterozygote deficiency, excess or either (two-sided) obtained by discarding missings and imputing missings using single imputation and multiple imputation using information from two flanking markers. Plotting symbols and colors indicate the significance of the markers in two HWE tests (α = 0.05). Red diamonds: both tests significant, green circles: both tests non-significant, upward blue triangles: significant for the test on the <em>x</em>-axis, nonsignificant for the test on the <em>y</em>-axis. Downward orange triangles: non-significant on the <em>x</em>-axis, significant on the <em>y</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">The plots of the <em>p</em>-values give some idea of the performance of the SI and MI approach. In general, <em>p</em>-values obtained by MI resemble the <em>p</em>-values obtained by discarding missings, in particular for the nonsignificant markers (first column of <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The <em>p</em>-values obtained by SI generally show larger differences with respect to discarding missings (second column of <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). A considerable subset of markers becomes nonsignificant when missings are imputed by SI or MI, as shown by the vertical blue stripe in the first two columns of <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>. A few markers become significant upon imputation. If we take the two-sided test as a reference, then 33 SNPs (4.9%) turned nonsignificant under MI, whereas nine (1.3%) turned significant under MI. SI using SHAPEIT produced larger changes: 79 SNPs (11.7%) turned nonsignificant under SI, whereas 7 (1%) turned significant under SI. In total about 6% of the test results changed under MI, and about 13% under SI.</p><p id="p-30">The Q-Q plots of the <em>p</em>-values using a uniform reference distribution are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. A logarithmic scale is used to emphasize the lower tail of the distribution. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5D</a> shows the expected pattern for the database under HWE and can be used as a reference graph. These graphs confirm that there are many more significant results than would be expected by chance alone. SI is seen to slightly improve the distribution of the <em>p</em>-values. None of the methods used yields a uniform <em>p</em>-value distribution, most likely because the studied subset of SNPs is not only subject to missing observations but also to considerable genotyping error.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Q-Q plots of p-values for tests for Hardy&#x2212;Weinberg equilibrium obtained by (A) discarding missings, (B) single imputation of missing values and (C) multiple imputation. (D) shows a reference Q-Q plot of the p-values for a dataset of 677 simulated SNPs with the same sample size and allele frequency distribution as the observed data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2071473282" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Q-Q plots of &lt;em&gt;p&lt;/em&gt;-values for tests for Hardy&#x2212;Weinberg equilibrium obtained by (A) discarding missings, (B) single imputation of missing values and (C) multiple imputation. (D) shows a reference Q-Q plot of the &lt;em&gt;p&lt;/em&gt;-values for a dataset of 677 simulated SNPs with the same sample size and allele frequency distribution as the observed data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/11/2365/F5.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/11/2365/F5.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/11/2365/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Q-Q plots of <em>p</em>-values for tests for Hardy−Weinberg equilibrium obtained by (A) discarding missings, (B) single imputation of missing values and (C) multiple imputation. (D) shows a reference Q-Q plot of the <em>p</em>-values for a dataset of 677 simulated SNPs with the same sample size and allele frequency distribution as the observed data.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">This paper presents an exact test for HWE that takes missing genotypes into account. For small samples or low minor allele frequencies, the exact test for HWE generally is preferred over the classical χ<sup>2</sup> test. Thus, the work presented here allows one to test HWE in small samples or low MAF samples by an exact procedure even if there is missing genotype information. The mid <em>p</em>-value is used as the test-statistic to be used in the exact test for HWE. We note that the mid <em>p</em>-value has the property that its value in a test for heterozygote excess is one minus the <em>p</em>-value obtained in a test for heterozygote deficiency. The standard one-sided exact <em>p</em>-value does not have this property. It this sense testing is akin to a student <em>t</em>-test for quantitative variables, where the <em>p</em>-value for the one-sided test with <em>H</em><sub>0</sub>: μ = μ<sub>0</sub> against <em>H</em><sub>1</sub>: μ &gt; μ<sub>0</sub> is also one minus the <em>p</em>-value for the one-sided test with <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-48.gif"/></noscript></span></span>. This property directly carries over to the <em>p</em>-value obtained by multiple imputation (<em>p<sub>mi</sub></em>). Thus, if one wants to do both one-sided tests, only the exact test for a single one-sided test needs to be calculated. This means that doing both tests does not increase the computational burden in comparison with a standard two-sided test. We also note that the proposed two-sided multiple imputation <em>p</em>-value (<em>p<sub>mi</sub></em>) cannot exceed one, precisely because <em>p</em><sub>mi,excess</sub> = 1 − <em>p</em><sub>mi,deficiency</sub>. At least for large samples, the two-sided <em>p<sub>mi</sub></em> is seen to correlate well with the two-sided <em>p</em>-values obtained by combining inbreeding coefficients. In relation with this, we note that Q-Q plots of standard exact <em>p</em>-values made against the uniform distribution as a reference distribution are often used and typically show a band of <em>p</em>-values at the value 1 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Rohlfs and Weir 2008</a>). A <em>p</em>-value of 1 occurs if the observed sample is the most likely one for the given minor allele count. This often happens for markers with a low MAF. If the mid <em>p</em>-value is used, this band of <em>p</em>-values at 1 disappears, because the probability of the most likely sample is halved and a wider range of <em>p</em>-values can occur.</p><p id="p-33">Markers that strongly deviate from HWE have, especially for large samples like the one studied here, <em>p</em>-values that are vanishingly small, leading to <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-49.gif"/></noscript></span></span>[see <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation (8)</a>]. To avoid computational problems such <em>p</em>-values were set to the smallest floating point number that makes <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/11/2365/embed/mml-math-50.gif"/></noscript></span></span> different from 0. Likewise, <em>p</em>-values whose difference from 1 is vanishingly small were set to 1 minus the smallest floating point number that makes (1 − <em>p<sub>i</sub></em>) different from 1. This guarantees that the multiple imputation algorithm produces a <em>p</em>-value close to 0 if all imputed data sets are highly unlikely under HWE, and a <em>p</em>-value close to 1 if all imputed data sets are highly likely under HWE.</p><div id="sec-14" class="subsection"><h3>Implications for genome-wide association studies</h3><p id="p-34">Data cleaning for genome-wide association studies routinely uses HWE testing to filter out SNPs of low quality (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Laurie <em>et al.</em>, 2010</a>). Because HWE generally is expected in human populations, departures from HWE are expected to indicate problems with the data rather than some biological phenomenon. However, if SNPs that deviate significantly from HWE are eliminated routinely before association analysis, there may be a loss of power for association tests. It is well known (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nielsen <em>et al.</em> 1998</a>) that the HWE test at genetic markers, when confined to cases or to controls, is actually a test for linkage disequilibrium between the marker and the disease genes. Eliminating markers that fail HWE tests also may be eliminating SNPs associated with the disease. We suggest that a goal should be to retain as many markers as possible in the association analysis and maybe focus HWE tests on those markers that do appear to be associated with the disease. Markers found to be significantly associated with a trait could be examined for the questions: 1) Do they have low MAF? 2) Are they out of HWE? 3) Do they have a large number of missings? 4) Do the genotype call plots look unusual? These questions are, in fact, part of sound analysis pipelines.</p><p id="p-35">On the other hand, many markers of the GENEVA subset studied in this paper (more than 5% have missing data) have “bad” genotype calling plots and 34% of the subset markers are out of HWE. Most of these are likely out of HWE due to genotyping error. If a marker presents disequilibrium AND it has a high missing rate, genotyping error is probably the most likely explanation for both things. But if a marker presents disequilibrium without a high missing rate then there is less evidence for genotyping error and it may be appropriate to keep the marker for association analysis.</p></div><div id="sec-15" class="subsection"><h3>GENEVA data results</h3><p id="p-36">The analysis of the GENEVA database on prematurity shows that disequilibrium is more often due to a deficiency of heterozygotes than to an excess of heterozygotes. In 79% of the cases of statistically significant disequilibrium, the latter was due to a deficiency and in 21% of the cases it was due to an excess of heterozygotes. That disequilibrium is more often due to heterozygote deficiency seems to be a characteristic of SNP data (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Graffelman <em>et al.</em> 2013</a>). An explanation for this is that heterozygous genotypes have a greater probability of being missing or being misclassified as a homozygote, basically because in the intensity plots they form a middle cloud bordering two other groups. Homozygote genotypes maybe misclassified as heterozygotes but it is less likely that a homozygote AA is misclassified as a BB or the reverse. If the sample is large, and all three genotypes are present with substantial frequencies, then an AB genotype may be expected to have double the misclassification rate and the missingness rate of an AA genotype. A few monomorphic markers with a considerable number of missings were observed. Such markers could potentially be tested for HWE if missings are taken into account and imputed with genotypes that possibly differ from the single observed homozygote. MI with the multinomial logit model was not possible in these cases because the response shows no variation. SHAPEIT assigned the single observed common homozygote to all missings in these cases. However, in the call plot sometimes the missings clustered outside the cloud of the common homozygote, indicating that a different genotype might indeed exist. This stresses the need for an imputation method that can impute genotypes that have not been observed in the sample.</p><p id="p-37">A considerable set of markers had a zero count for one homozygote. In some cases missings formed a separate cluster in the call plot, separate from the two observed genotypes, that most likely corresponded to the unobserved homozygote. This generates a situation of (false) heterozygote excess. In this situation, the MI approach with the multinomial logit model is not able to create sensible imputations, because it can impute only the two observed categories in the data. In some instances, SHAPEIT did impute the missings as the second homozygote and could diminish heterozygote excess. The fact that SHAPEIT can impute missings with genotypes that have not been observed in the data is an advantage and it explains at least in part that the inbreeding coefficients obtained with this method tend to be closer to zero than those obtained by MI. Single-imputation programs sometimes produce a best-guess value for a missing value using the genotype that has the highest posterior probability. Using the full vector of posterior probabilities for all genotypes could improve the inference for HWE, as it would take better account of imputation uncertainty. This requires a test for HWE that is based on probabilities instead of on genotype counts. The development of such a test is part of our ongoing research.</p><p id="p-38">The strength of the MI approach described in this paper lies in two points: First, MI takes uncertainty in the imputations into account by imputing many times. Second, only a few correlated variables (flanking markers and/or intensities) are needed for imputation. This is an advantage in situations where no reference panels are available, or when there are not sufficient markers for reliable haplotype estimation. A future modification of the multinomial logit model that allows for the imputation of all three genotypes would form a valuable extension of the work presented here.</p><p id="p-39">We comment on some additional aspects of the call plots of studied subset of the GENEVA database. Many call plots of the significant markers were of poor quality due to null alleles, cluster overlap, or more than three clusters. Missing genotypes often correspond to null alleles (intensities close to zero). Markers that became significant upon imputation often showed additional clusters of missings separated from the AA, AB, and BB clouds. Both imputation algorithms used here are restrictive in the sense that they force such missings to be imputed as one of the three habitual genotypes and do not account for the existence of null alleles or copy number variation.</p><p id="p-40">The database on prematurity studied in this paper has a low overall percentage of missing values and concerns a large sample. Most SNPs have only a few missing values, and inference for HWE is hardly affected by discarding or imputing the missings. Accounting for missingness becomes more interesting if the number of missings is substantial because then there is more scope for bias if missings are ignored. If no MAF threshold is applied, then the set of markers studied here contains precisely those markers generally considered poor markers. For such markers HWE is rejected more often, in particular if the missings are discarded. Often this subset of markers is ruled out from analysis by applying thresholds for the MAF and the HWE exact <em>p</em>-value (frequently used exclusion criteria are MAF below 0.05 and HWE exact <em>p</em>-value below 0.001). It is precisely for this subset that single and multiple imputation can provide improved statistical inference.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">This study was supported in part by grants ECO2011-28875 and CODARSS MTM2012-33236 of the Spanish Ministry of Education and Science, and by R01 GM075091 from the United States National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: M. Johnston</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-02"><span class="received-label">Received </span>August 2, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-06"><span class="accepted-label">Accepted </span>September 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Graffelman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agresti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Categorical Data Analysis</span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.2" data-doi="10.1038/pr.2012.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alleman</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myking</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryckman</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myhre</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feingold</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">No observed association for mitochondrial SNPs with preterm delivery and related outcomes.</span> <abbr class="cit-jnl-abbrev">Pediatr. Res.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPediatric%2BResearch%26rft.stitle%253DPediatr.%2BRes.%26rft.aulast%253DAlleman%26rft.auinit1%253DB.%2BW.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D539%26rft.epage%253D544%26rft.atitle%253DNo%2Bobserved%2Bassociation%2Bfor%2Bmitochondrial%2BSNPs%2Bwith%2Bpreterm%2Bdelivery%2Band%2Brelated%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fpr.2012.112%26rft_id%253Dinfo%253Apmid%252F22902432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/pr.2012.112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902432&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311771000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.3" data-doi="10.1093/aje/kwp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wohlfahrt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feenstra</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maternal contributions to preterm delivery.</span> <abbr class="cit-jnl-abbrev">Am. J. Epidemiol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BEpidemiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faje%252Fkwp324%26rft_id%253Dinfo%253Apmid%252F19854807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYW1qZXBpZCI7czo1OiJyZXNpZCI7czoxMToiMTcwLzExLzEzNTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D84%26rft.spage%253D210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source"><em>An Introduction to Population Genetics Theory</em></span>. <span class="cit-publ-name">Harper and Row</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.6" data-doi="10.1038/ncomms4934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3934%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4934%26rft_id%253Dinfo%253Apmid%252F25653097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25653097&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.7" data-doi="10.1093/bioinformatics/bts610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangale</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHugh</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GWASTools: an R/Bioconductor package for quality control and analysis of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3329</span>–<span class="cit-lpage">3331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts610%26rft_id%253Dinfo%253Apmid%252F23052040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjQvMzMyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.8" data-doi="10.1046/j.1469-1809.1999.6360535.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonjou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Hardy-Weinberg quality control.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DGomes%26rft.auinit1%253DI.%26rft.volume%253D63%26rft.issue%253DPt%2B6%26rft.spage%253D535%26rft.epage%253D538%26rft.atitle%253DHardy-Weinberg%2Bquality%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-1809.1999.6360535.x%26rft_id%253Dinfo%253Apmid%252F11246455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-1809.1999.6360535.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11246455&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086526800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graffelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The mid <em>p</em>-value in exact tests for Hardy-Weinberg equilibrium.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D12%26rft.spage%253D433%26rft_id%253Dinfo%253Apmid%252F23934608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23934608&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.10" data-doi="10.1371/journal.pone.0083316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graffelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Statistical inference for Hardy-Weinberg proportions in the presence of missing genotype information.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e83316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De83316%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0083316%26rft_id%253Dinfo%253Apmid%252F24391752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0083316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24391752&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.11" data-doi="10.1007/BF02985085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">An exact test for randomness of mating.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D52%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02985085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02985085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954XX18100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.12" data-doi="10.1038/sj.ejhg.5201164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosking</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lumsden</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Detection of genotyping errors by Hardy-Weinberg equilibrium testing.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DHosking%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D395%26rft.epage%253D399%26rft.atitle%253DDetection%2Bof%2Bgenotyping%2Berrors%2Bby%2BHardy-Weinberg%2Bequilibrium%2Btesting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.ejhg.5201164%26rft_id%253Dinfo%253Apmid%252F14872201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.ejhg.5201164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14872201&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220879600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.13" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.14" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.15" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.16" data-doi="10.1002/gepi.20516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doheny</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quality control and quality assurance in genotypic data for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLaurie%26rft.auinit1%253DC.%2BC.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D591%26rft.epage%253D602%26rft.atitle%253DQuality%2Bcontrol%2Band%2Bquality%2Bassurance%2Bin%2Bgenotypic%2Bdata%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20516%26rft_id%253Dinfo%253Apmid%252F20718045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20718045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.17" data-doi="10.1214/aoms/1177730093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levene</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">On a matching problem arising in genetics.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D20%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177730093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177730093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.18" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: Using sequence and genotype data to estimate haplotypes and unobserved genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.19"><div class="cit-metadata unstructured">Licht, C., 2010 New methods for generating significance levels from multiply-imputed data. Dissertation, Otto-Friedrich Universität, Bamberg, Germany.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">R. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source"><em>Statistical Analysis with Missing Data</em></span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.21" data-doi="10.1002/sim.6197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liublinska</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sensitivity analysis for a partially missing binary outcome in a two-arm randomized clinical trial.</span> <abbr class="cit-jnl-abbrev">Stat. Med.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">4170</span>–<span class="cit-lpage">4185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BMed.%26rft.volume%253D33%26rft.spage%253D4170%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fsim.6197%26rft_id%253Dinfo%253Apmid%252F24845086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/sim.6197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845086&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.22" data-doi="10.1086/302114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Detecting marker-disease association by testing for Hardy-Weinberg disequilibrium at a marker locus.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1531</span>–<span class="cit-lpage">1540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DD.%2BM.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D1531%26rft.epage%253D1540%26rft.atitle%253DDetecting%2Bmarker-disease%2Bassociation%2Bby%2Btesting%2Bfor%2BHardy-Weinberg%2Bdisequilibrium%2Bat%2Ba%2Bmarker%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302114%26rft_id%253Dinfo%253Apmid%252F9867708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9867708&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076985200029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.23" data-doi="10.1534/genetics.108.088005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohlfs</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Distributions of Hardy-Weinberg equilibrium test statistics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088005%26rft_id%253Dinfo%253Apmid%252F18791257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzE2MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzExLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source"><em>Multiple Imputation for Nonresponse in Surveys</em></span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.25" data-doi="10.1016/j.jpeds.2011.07.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryckman</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feenstra</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaffer</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bream</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geller</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Replication of a genome-wide association study of birth weight in preterm neonates.</span> <abbr class="cit-jnl-abbrev">J. Pediatr.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpediatrics%26rft.stitle%253DJ%2BPediatr%26rft.aulast%253DRyckman%26rft.auinit1%253DK.%2BK.%26rft.volume%253D160%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D24.e4%26rft.atitle%253DReplication%2Bof%2Ba%2Bgenome-wide%2Bassociation%2Bstudy%2Bof%2Bbirth%2Bweight%2Bin%2Bpreterm%2Bneonates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jpeds.2011.07.038%26rft_id%253Dinfo%253Apmid%252F21885063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jpeds.2011.07.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21885063&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Analysis of Incomplete Multivariate Data</em></span>. <span class="cit-publ-name">Chapman and Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-5.11.2365.27" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">G. R.</span>  <span class="cit-name-given-names">Abecasis</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <span class="cit-source">Nature</span> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Buuren</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>Flexible Imputation of Missing Data</em></span>. <span class="cit-publ-name">Chapman and Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Buuren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groothuis-Oudshoorn</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mice: multivariate imputation by chained equations in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D45%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.11.2365.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Genetic</em> Data Analysis II</span>. <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.31" data-doi="10.1007/s12561-013-9090-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Interpreting whole-genome marker data.</span> <abbr class="cit-jnl-abbrev">Stat. Biosci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BBiosci.%26rft.volume%253D5%26rft.spage%253D316%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12561-013-9090-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12561-013-9090-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.32" data-doi="10.1086/429864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wigginton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A note on exact tests of Hardy-Weinberg equilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWigginton%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D76%26rft.issue%253D5%26rft.spage%253D887%26rft.epage%253D893%26rft.atitle%253DA%2Bnote%2Bon%2Bexact%2Btests%2Bof%2BHardy-Weinberg%2Bequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F429864%26rft_id%253Dinfo%253Apmid%252F15789306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/429864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15789306&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F11%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228198300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.11.2365.33" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjQvMzMyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTEvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/11/2365.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/11/2357" title="Genetic Linkage Mapping of Economically Important Traits in Cultivated Tetraploid Potato (Solanum tuberosum L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/11/2375" title="Post-transcriptional Mechanisms Contribute Little to Phenotypic Variation in Snake Venoms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/11"><span>Volume 5 Issue 11, November 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/11.cover-source.jpg?itok=JmHQkVZp" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/11/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0Gdf-KjxgpUfPKZwUL3Of2g_RO6-3ALhSLhWJAkU6-E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/11/2365" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/11/2365" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/11/2365" class="active" data-icon-position="" data-hide-link-title="0">Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460167" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pGoqxbbkqO_OeyFM-EIbxDzYbBAkvJmhCY5Cg5YWIHo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2365.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/11/2365" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WynktH4js_CfWjeVbV1ofEcwzeLd6NfDC0WAd8iYlG0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/11/2365" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/11/2365.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460167" id="node460167--2" data-pisa="ggg;5/11/2365" data-pisa-master="ggg;g3.115.022111" data-apath="/ggg/5/11/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan Graffelman</span>, <span class="highwire-citation-author" data-delta="1">S. Nelson</span>, <span class="highwire-citation-author" data-delta="2">S. M. Gogarten</span> and <span class="highwire-citation-author" data-delta="3">B. S. Weir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022111 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460167--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jan Graffelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics and Operations Research, Universitat Politècnica de Catalunya, 08028 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BGraffelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graffelman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BGraffelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan Graffelman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">B. S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB.%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bsweir@uw.edu" class="" data-icon-position="" data-hide-link-title="0">bsweir@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/11/2365" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/11/2365&amp;text=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/11/2365&amp;t=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/11/2365&amp;title=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460167/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460167" id="node460167--3" data-pisa="ggg;5/11/2365" data-pisa-master="ggg;g3.115.022111" data-apath="/ggg/5/11/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exact Inference for Hardy-Weinberg Proportions with Missing Genotypes: Single and Multiple Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan Graffelman</span>, <span class="highwire-citation-author" data-delta="1">S. Nelson</span>, <span class="highwire-citation-author" data-delta="2">S. M. Gogarten</span> and <span class="highwire-citation-author" data-delta="3">B. S. Weir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2373;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022111 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460167--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jan Graffelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics and Operations Research, Universitat Politècnica de Catalunya, 08028 Barcelona, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BGraffelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graffelman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BGraffelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3900-0780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan Graffelman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">S. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">S. M. Gogarten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BM.%2BGogarten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogarten%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BM.%2BGogarten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">B. S. Weir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics, University of Washington, Seattle, Washington 98105-9461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BS.%2BWeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weir%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB.%2BS.%2BWeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bsweir@uw.edu" class="" data-icon-position="" data-hide-link-title="0">bsweir@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2365&amp;rft.epage=2373&amp;rft.atitle=Exact%20Inference%20for%20Hardy-Weinberg%20Proportions%20with%20Missing%20Genotypes%3A%20Single%20and%20Multiple%20Imputation&amp;rft.volume=5&amp;rft.issue=11&amp;rft.date=2015-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Graffelman%2C+Jan&amp;rft.au=Nelson%2C+S.&amp;rft.au=Gogarten%2C+S.+M.&amp;rft.au=Weir%2C+B.+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460167/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460167/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460167/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460167/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460167/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460167/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460167/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460167/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460167/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460167/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460167/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460167/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/11/2365" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Software</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/11/2365.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/11/2365.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/11/2365.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
