<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2187" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2187.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2187.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2187.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2187/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2187.atom" />
<meta name="HW.pisa" content="ggg;5/10/2187" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.115.021493" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Dubarry et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Three major DNA polymerases replicate the linear eukaryotic chromosomes. DNA polymerase α-primase (Pol α) and DNA polymerase δ (Pol δ) replicate the lagging-strand and Pol α and DNA polymerase ε (Pol ε) the leading-strand. To identify factors affecting coordination of DNA replication, we have performed genome-wide quantitative fitness analyses of budding yeast cells containing defective polymerases. We combined temperature-sensitive mutations affecting the three replicative polymerases, Pol α, Pol δ, and Pol ε with genome-wide collections of null and reduced function mutations. We identify large numbers of genetic interactions that inform about the roles that specific genes play to help Pol α, Pol δ, and Pol ε function. Surprisingly, the overlap between the genetic networks affecting the three DNA polymerases does not represent the majority of the genetic interactions identified. Instead our data support a model for division of labor between the different DNA polymerases during DNA replication. For example, our genetic interaction data are consistent with biochemical data showing that Pol ε is more important to the Pre-Loading complex than either Pol α or Pol δ. We also observed distinct patterns of genetic interactions between leading- and lagging-strand DNA polymerases, with particular genes being important for coupling proliferating cell nuclear antigen loading/unloading ([Ctf18][1], [Elg1][2]) with nucleosome assembly (chromatin assembly factor 1, histone regulatory HIR complex). Overall our data reveal specialized genetic networks that affect different aspects of leading- and lagging-strand DNA replication. To help others to engage with these data we have generated two novel, interactive visualization tools, DIXY and Profilyzer.

 [1]: http://www.yeastgenome.org/locus/S000004683/overview
 [2]: http://www.yeastgenome.org/locus/S000005670/overview" />
<meta name="DC.Contributor" content="Marion Dubarry" />
<meta name="DC.Contributor" content="Conor Lawless" />
<meta name="DC.Contributor" content="A. Peter Banks" />
<meta name="DC.Contributor" content="Simon Cockell" />
<meta name="DC.Contributor" content="David Lydall" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Three major DNA polymerases replicate the linear eukaryotic chromosomes. DNA polymerase α-primase (Pol α) and DNA polymerase δ (Pol δ) replicate the lagging-strand and Pol α and DNA polymerase ε (Pol ε) the leading-strand. To identify factors affecting coordination of DNA replication, we have performed genome-wide quantitative fitness analyses of budding yeast cells containing defective polymerases. We combined temperature-sensitive mutations affecting the three replicative polymerases, Pol α, Pol δ, and Pol ε with genome-wide collections of null and reduced function mutations. We identify large numbers of genetic interactions that inform about the roles that specific genes play to help Pol α, Pol δ, and Pol ε function. Surprisingly, the overlap between the genetic networks affecting the three DNA polymerases does not represent the majority of the genetic interactions identified. Instead our data support a model for division of labor between the different DNA polymerases during DNA replication. For example, our genetic interaction data are consistent with biochemical data showing that Pol ε is more important to the Pre-Loading complex than either Pol α or Pol δ. We also observed distinct patterns of genetic interactions between leading- and lagging-strand DNA polymerases, with particular genes being important for coupling proliferating cell nuclear antigen loading/unloading (Ctf18, Elg1) with nucleosome assembly (chromatin assembly factor 1, histone regulatory HIR complex). Overall our data reveal specialized genetic networks that affect different aspects of leading- and lagging-strand DNA replication. To help others to engage with these data we have generated two novel, interactive visualization tools, DIXY and Profilyzer.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2187" />
<meta name="citation_id" content="5/10/2187" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2187" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2187.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2187.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2187.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021493" />
<meta name="citation_pmid" content="26297725" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2187" />
<meta name="citation_lastpage" content="2197" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marion Dubarry" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author" content="Conor Lawless" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4186-8506" />
<meta name="citation_author" content="A. Peter Banks" />
<meta name="citation_author_institution" content="High Throughput Screening Facility, Newcastle Biomedicine, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author" content="Simon Cockell" />
<meta name="citation_author_institution" content="Bioinformatics Support Unit, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author" content="David Lydall" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom" />
<meta name="citation_author_email" content="david.lydall@ncl.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2478-085X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. G. Addinall;citation_author=M. Downey;citation_author=M. Yu;citation_author=M. K. Zubko;citation_author=J. Dewar;citation_title=A genomewide suppressor and enhancer analysis of cdc13–1 reveals varied cellular processes influencing telomere capping in Saccharomyces cerevisiae.;citation_volume=180;citation_year=2008;citation_pmid=18845848;citation_doi=10.1534/genetics.108.092577" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. G. Addinall;citation_author=E. M. Holstein;citation_author=C. Lawless;citation_author=M. Yu;citation_author=K. Chapman;citation_title=Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.;citation_pages=e1001362-e1001362;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490951;citation_doi=10.1371/journal.pgen.1001362" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Aksenova;citation_author=K. Volkov;citation_author=J. Maceluch;citation_author=Z. F. Pursell;citation_author=I. B. Rogozin;citation_title=Mismatch repair-independent increase in spontaneous mutagenesis in yeast lacking non-essential subunits of DNA polymerase Îµ.;citation_pages=e1001209-e1001209;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21124948;citation_doi=10.1371/journal.pgen.1001209" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=B. M. Alberts;citation_title=The DNA Enzymology of Protein Machines;citation_pages=1-12;citation_volume=49;citation_year=1984;citation_issue=0;citation_pmid=6597750;citation_doi=10.1101/SQB.1984.049.01.003" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=A. A. Alcasabas;citation_author=A. J. Osborn;citation_author=J. Bachant;citation_author=F. Hu;citation_author=P. J. Werler;citation_title=Mrc1 transduces signals of DNA replication stress to activate Rad53.;citation_pages=958-965;citation_volume=3;citation_year=2001;citation_issue=11;citation_pmid=11715016;citation_doi=10.1038/ncb1101-958" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. J. Andrew;citation_author=S. Merchan;citation_author=C. Lawless;citation_author=A. P. Banks;citation_author=D. J. Wilkinson;citation_title=Pentose phosphate pathway function affects tolerance to the G-quadruplex binder TMPyP4.;citation_pages=e66242;citation_volume=8;citation_year=2013;citation_pmid=23776642;citation_doi=10.1371/journal.pone.0066242" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=A. T. Annunziato;citation_title=Assembling chromatin: the long and winding road.;citation_pages=196-210;citation_volume=1819;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. Araki;citation_author=R. K. Hamatake;citation_author=A. Morrison;citation_author=A. L. Johnson;citation_author=L. H. Johnston;citation_title=Cloning DPB3, the gene encoding the third subunit of DNA polymerase II of Saccharomyces cerevisiae;citation_pages=4867-4872;citation_volume=19;citation_year=1991;citation_issue=18;citation_pmid=1923754;citation_doi=10.1093/nar/19.18.4867" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Bando;citation_author=Y. Katou;citation_author=M. Komata;citation_author=H. Tanaka;citation_author=T. Itoh;citation_title=Csm3, Tof1, and Mrc1 form a heterotrimeric mediator complex that associates with DNA replication forks.;citation_volume=284;citation_year=2009;citation_pmid=19819872;citation_doi=10.1074/jbc.M109.065730" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. P. Bermudez;citation_author=Y. Maniwa;citation_author=I. Tappin;citation_author=K. Ozato;citation_author=K. Yokomori;citation_title=The alternative Ctf18-Dcc1-Ctf8-replication factor C complex required for sister chromatid cohesion loads proliferating cell nuclear antigen onto DNA.;citation_volume=100;citation_year=2003;citation_pmid=12930902;citation_doi=10.1073/pnas.1434308100" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. K. Breslow;citation_author=D. M. Cameron;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=J. Stewart-Ornstein;citation_title=A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.;citation_pages=711-718;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18622397;citation_doi=10.1038/nmeth.1234" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. E. Budd;citation_author=J. L. Campbell;citation_title=DNA polymerases delta and epsilon are required for chromosomal replication in Saccharomyces cerevisiae.;citation_pages=496-505;citation_volume=13;citation_year=1993;citation_issue=1;citation_pmid=8417347;citation_doi=10.1128/MCB.13.1.496" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. R. Cairns;citation_author=Y. Lorch;citation_author=Y. Li;citation_author=M. Zhang;citation_author=L. Lacomis;citation_title=RSC, an essential, abundant chromatin-remodeling complex.;citation_pages=1249-1260;citation_volume=87;citation_year=1996;citation_issue=7;citation_pmid=8980231;citation_doi=10.1016/S0092-8674(00)81820-6" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. M. Chou;citation_author=P. Petersen;citation_author=J. C. Walter;citation_author=G. Walter;citation_title=Protein phosphatase 2A regulates binding of Cdc45 to the prereplication complex.;citation_volume=277;citation_year=2002;citation_pmid=12185086;citation_doi=10.1074/jbc.M207226200" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=L. Crabbe;citation_author=A. Thomas;citation_author=V. Pantesco;citation_author=J. De Vos;citation_author=P. Pasero;citation_title=Analysis of replication profiles reveals key role of RFC-Ctf18 in yeast replication stress response.;citation_pages=1391-1397;citation_volume=17;citation_year=2010;citation_issue=11;citation_pmid=20972444;citation_doi=10.1038/nsmb.1932" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=Y. Daigaku;citation_author=A. Keszthelyi;citation_author=C. A. Muller;citation_author=I. Miyabe;citation_author=T. Brooks;citation_title=A global profile of replicative polymerase usage.;citation_pages=192-198;citation_volume=22;citation_year=2015;citation_pmid=25664722;citation_doi=10.1038/nsmb.2962" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. Dua;citation_author=D. L. Levy;citation_author=J. L. Campbell;citation_title=Analysis of the Essential Functions of the C-terminal Protein/Protein Interaction Domain of Saccharomyces cerevisiae pol {varepsilon} and Its Unexpected Ability to Support Growth in the Absence of the DNA Polymerase Domain;citation_pages=22283-22288;citation_volume=274;citation_year=1999;citation_issue=32;citation_pmid=10428796;citation_doi=10.1074/jbc.274.32.22283" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=E. Eden;citation_author=R. Navon;citation_author=I. Steinfeld;citation_author=D. Lipson;citation_author=Z. Yakhini;citation_title=GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.;citation_pages=48-48;citation_volume=10;citation_year=2009;citation_pmid=19192299;citation_doi=10.1186/1471-2105-10-48" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=W. Feng;citation_author=G. D’Urso;citation_title=Schizosaccharomyces pombe Cells Lacking the Amino-Terminal Catalytic Domains of DNA Polymerase Epsilon Are Viable but Require the DNA Damage Checkpoint Control;citation_pages=4495-4504;citation_volume=21;citation_year=2001;citation_issue=14;citation_pmid=11416129;citation_doi=10.1128/MCB.21.14.4495-4504.2001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Finley;citation_author=H. D. Ulrich;citation_author=T. Sommer;citation_author=P. Kaiser;citation_title=The Ubiquitin-Proteasome System of Saccharomyces cerevisiae;citation_pages=319-360;citation_volume=192;citation_year=2012;citation_issue=2;citation_pmid=23028185;citation_doi=10.1534/genetics.112.140467" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. L. Flood;citation_author=G. P. Rodriguez;citation_author=G. Bao;citation_author=A. H. Shockley;citation_author=Y. W. Kow;citation_title=Replicative DNA polymerase delta but not epsilon proofreads errors in Cis and in Trans.;citation_pages=e1005049;citation_volume=11;citation_year=2015;citation_pmid=25742645;citation_doi=10.1371/journal.pgen.1005049" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=A. Gambus;citation_author=F. van Deursen;citation_author=D. Polychronopoulos;citation_author=M. Foltman;citation_author=R. C. Jones;citation_title=A key role for Ctf4 in coupling the MCM2-7 helicase to DNA polymerase alpha within the eukaryotic replisome.;citation_pages=2992-3004;citation_volume=28;citation_year=2009;citation_issue=19;citation_pmid=19661920;citation_doi=10.1038/emboj.2009.226" />
<meta name="citation_reference" content="citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology;citation_journal_abbrev=CRIT REV BIOCHEM MOL BIOL;citation_author=P. Garg;citation_author=P. M. Burgers;citation_title=DNA polymerases that propagate the eukaryotic DNA replication fork.;citation_pages=115-128;citation_volume=40;citation_year=2005;citation_issue=2;citation_pmid=15814431;citation_doi=10.1080/10409230590935433" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Reconstitution of a eukaryotic replisome reveals suppression mechanisms that define leading/lagging strand operation.;citation_author=R. E. Georgescu;citation_author=G. D. Schauer;citation_author=N. Y. Yao;citation_author=L. D. Langston;citation_author=O. Yurieva;citation_pages=10.7554/eLife.04988;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. S. Hanna;citation_author=E. S. Kroll;citation_author=V. Lundblad;citation_author=F. A. Spencer;citation_title=Saccharomyces cerevisiae CTF18 and CTF4 Are Required for Sister Chromatid Cohesion;citation_pages=3144-3158;citation_volume=21;citation_year=2001;citation_issue=9;citation_pmid=11287619;citation_doi=10.1128/MCB.21.9.3144-3158.2001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. H. Hartwell;citation_author=R. K. Mortimer;citation_author=J. Culotti;citation_author=M. Culotti;citation_title=GENETIC CONTROL OF THE CELL DIVISION CYCLE IN YEAST: V. GENETIC ANALYSIS OF cdc MUTANTS;citation_pages=267-286;citation_volume=74;citation_year=1973;citation_issue=2;citation_pmid=17248617" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. S. Ivessa;citation_author=J. Q. Zhou;citation_author=V. P. Schulz;citation_author=E. K. Monson;citation_author=V. A. Zakian;citation_title=Saccharomyces Rrm3p, a 5' to 3' DNA helicase that promotes replication fork progression through telomeric and subtelomeric DNA;citation_pages=1383-1396;citation_volume=16;citation_year=2002;citation_issue=11;citation_pmid=12050116;citation_doi=10.1101/gad.982902" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. S. Ivessa;citation_author=B. A. Lenzmeier;citation_author=J. B. Bessler;citation_author=L. K. Goudsouzian;citation_author=S. L. Schnakenberg;citation_title=The Saccharomyces cerevisiae helicase Rrm3p facilitates replication past nonhistone protein-DNA complexes.;citation_pages=1525-1536;citation_volume=12;citation_year=2003;citation_issue=6;citation_pmid=14690605;citation_doi=10.1016/S1097-2765(03)00456-8" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=R. E. Johnson;citation_author=R. Klassen;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=A major role of DNA polymerase delta in replication of both the leading and lagging DNA strands.;citation_pages=163-175;citation_volume=59;citation_year=2015;citation_pmid=26145172;citation_doi=10.1016/j.molcel.2015.05.038" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. D. Kaufman;citation_author=R. Kobayashi;citation_author=B. Stillman;citation_title=Ultraviolet radiation sensitivity and reduction of telomeric silencing in Saccharomyces cerevisiae cells lacking chromatin assembly factor-I.;citation_pages=345-357;citation_volume=11;citation_year=1997;citation_issue=3;citation_pmid=9030687;citation_doi=10.1101/gad.11.3.345" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Kesti;citation_author=K. Flick;citation_author=S. Keranen;citation_author=J. E. Syvaoja;citation_author=C. Wittenberg;citation_title=DNA polymerase epsilon catalytic domains are dispensable for DNA replication, DNA repair, and cell viability.;citation_pages=679-685;citation_volume=3;citation_year=1999;citation_issue=5;citation_pmid=10360184;citation_doi=10.1016/S1097-2765(00)80361-5" />
<meta name="citation_reference" content="citation_journal_title=Medical hypotheses;citation_journal_abbrev=Med Hypotheses;citation_author=Y. Klipper-Aurbach;citation_author=M. Wasserman;citation_author=N. Braunspiegel-Weintrob;citation_author=D. Borstein;citation_author=S. Peleg;citation_title=Mathematical formulae for the prediction of the residual beta cell function during the first two years of disease in children and adolescents with insulin-dependent diabetes mellitus.;citation_pages=486-490;citation_volume=45;citation_year=1995;citation_issue=5;citation_pmid=8748093;citation_doi=10.1016/0306-9877(95)90228-7" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. R. Knott;citation_author=C. J. Viggiani;citation_author=S. Tavare;citation_author=O. M. Aparicio;citation_title=Genome-wide replication profiles indicate an expansive role for Rpd3L in regulating replication initiation timing or efficiency, and reveal genomic loci of Rpd3 function in Saccharomyces cerevisiae;citation_pages=1077-1090;citation_volume=23;citation_year=2009;citation_issue=9;citation_pmid=19417103;citation_doi=10.1101/gad.1784309" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=O. V. Kochenova;citation_author=D. L. Daee;citation_author=T. M. Mertz;citation_author=P. V. Shcherbakova;citation_title=DNA polymerase zeta-dependent lesion bypass in Saccharomyces cerevisiae is accompanied by error-prone copying of long stretches of adjacent DNA.;citation_pages=e1005110;citation_volume=11;citation_year=2015;citation_pmid=25826305;citation_doi=10.1371/journal.pgen.1005110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. Kubota;citation_author=K. Nishimura;citation_author=M. T. Kanemaki;citation_author=A. D. Donaldson;citation_title=The Elg1 replication factor C-like complex functions in PCNA unloading during DNA replication.;citation_pages=273-280;citation_volume=50;citation_year=2013;citation_pmid=23499004;citation_doi=10.1016/j.molcel.2013.02.012" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. A. Kunkel;citation_author=R. K. Hamatake;citation_author=J. Motto-Fox;citation_author=M. P. Fitzgerald;citation_author=A. Sugino;citation_title=Fidelity of DNA polymerase I and the DNA polymerase I-DNA primase complex from Saccharomyces cerevisiae.;citation_pages=4447-4458;citation_volume=9;citation_year=1989;citation_issue=10;citation_pmid=2555694;citation_doi=10.1128/MCB.9.10.4447" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=I. Kurth;citation_author=M. O’Donnell;citation_title=New insights into replisome fluidity during chromosome replication.;citation_pages=195-203;citation_volume=38;citation_year=2013;citation_pmid=23153958;citation_doi=10.1016/j.tibs.2012.10.003" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. D. Langston;citation_author=D. Zhang;citation_author=O. Yurieva;citation_author=R. E. Georgescu;citation_author=J. Finkelstein;citation_title=CMG helicase and DNA polymerase epsilon form a functional 15-subunit holoenzyme for eukaryotic leading-strand DNA replication.;citation_volume=111;citation_year=2014;citation_pmid=25313033;citation_doi=10.1073/pnas.1418334111" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Lawless;citation_author=D. J. Wilkinson;citation_author=A. Young;citation_author=S. G. Addinall;citation_author=D. A. Lydall;citation_title=Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.;citation_pages=287-287;citation_volume=11;citation_year=2010;citation_pmid=20509870;citation_doi=10.1186/1471-2105-11-287" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Li;citation_author=R. Martienssen;citation_author=W. Z. Cande;citation_title=Coordination of DNA replication and histone modification by the Rik1-Dos2 complex.;citation_pages=244-248;citation_volume=475;citation_year=2011;citation_issue=7355;citation_pmid=21725325;citation_doi=10.1038/nature10161" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=Z. Li;citation_author=F. J. Vizeacoumar;citation_author=S. Bahr;citation_author=J. Li;citation_author=J. Warringer;citation_title=Systematic exploration of essential yeast gene function with temperature-sensitive mutants.;citation_pages=361-367;citation_volume=29;citation_year=2011;citation_issue=4;citation_pmid=21441928;citation_doi=10.1038/nbt.1832" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=X. H. Lin;citation_author=J. Walter;citation_author=K. Scheidtmann;citation_author=K. Ohst;citation_author=J. Newport;citation_title=Protein phosphatase 2A is required for the initiation of chromosomal DNA replication;citation_pages=14693-14698;citation_volume=95;citation_year=1998;citation_issue=25;citation_pmid=9843951;citation_doi=10.1073/pnas.95.25.14693" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=H. Masai;citation_author=S. Matsumoto;citation_author=Z. You;citation_author=N. Yoshizawa-Sugata;citation_author=M. Oda;citation_title=Eukaryotic chromosome DNA replication: where, when, and how?;citation_pages=89-130;citation_volume=79;citation_year=2010;citation_pmid=20373915;citation_doi=10.1146/annurev.biochem.052308.103205" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=J. Mejlvang;citation_author=Y. Feng;citation_author=C. Alabert;citation_author=K. J. Neelsen;citation_author=Z. Jasencakova;citation_title=New histone supply regulates replication fork speed and PCNA unloading.;citation_volume=204;citation_year=2014;citation_pmid=24379417;citation_doi=10.1083/jcb.201305017" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Muramatsu;citation_author=K. Hirai;citation_author=Y. S. Tak;citation_author=Y. Kamimura;citation_author=H. Araki;citation_title=CDK-dependent complex formation between replication proteins Dpb11, Sld2, Pol {varepsilon}, and GINS in budding yeast;citation_pages=602-612;citation_volume=24;citation_year=2010;citation_issue=6;citation_pmid=20231317;citation_doi=10.1101/gad.1883410" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Myung;citation_author=A. Datta;citation_author=C. Chen;citation_author=R. D. Kolodner;citation_title=SGS1, the Saccharomyces cerevisiae homologue of BLM and WRN, suppresses genome instability and homeologous recombination.;citation_pages=113-116;citation_volume=27;citation_year=2001;citation_issue=1;citation_pmid=11138010;citation_doi=10.1038/83673" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. H. Ngo;citation_author=L. Balakrishnan;citation_author=M. Dubarry;citation_author=J. L. Campbell;citation_author=D. Lydall;citation_title=The 9–1-1 checkpoint clamp stimulates DNA resection by Dna2-Sgs1 and Exo1.;citation_volume=42;citation_year=2014;citation_pmid=25122752;citation_doi=10.1093/nar/gku746" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. Ray-Gallet;citation_author=J. P. Quivy;citation_author=C. Scamps;citation_author=E. M. Martini;citation_author=M. Lipinski;citation_title=HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis.;citation_pages=1091-1100;citation_volume=9;citation_year=2002;citation_issue=5;citation_pmid=12049744;citation_doi=10.1016/S1097-2765(02)00526-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. A. Reijns;citation_author=H. Kemp;citation_author=J. Ding;citation_author=S. M. de Proce;citation_author=A. P. Jackson;citation_title=Lagging-strand replication shapes the mutational landscape of the genome.;citation_pages=502-506;citation_volume=518;citation_year=2015;citation_pmid=25624100;citation_doi=10.1038/nature14183" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. F. Rizzardi;citation_author=E. S. Dorn;citation_author=B. D. Strahl;citation_author=J. G. Cook;citation_title=DNA replication origin function is promoted by H3K4 di-methylation in Saccharomyces cerevisiae.;citation_volume=192;citation_year=2012;citation_pmid=22851644;citation_doi=10.1534/genetics.112.142349" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Schuldiner;citation_author=S. R. Collins;citation_author=N. J. Thompson;citation_author=V. Denic;citation_author=A. Bhamidipati;citation_title=Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.;citation_pages=507-519;citation_volume=123;citation_year=2005;citation_issue=3;citation_pmid=16269340;citation_doi=10.1016/j.cell.2005.08.031" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=S. Sengupta;citation_author=F. van Deursen;citation_author=G. de Piccoli;citation_author=K. Labib;citation_title=Dpb2 integrates the leading-strand DNA polymerase into the eukaryotic replisome.;citation_pages=543-552;citation_volume=23;citation_year=2013;citation_pmid=23499531;citation_doi=10.1016/j.cub.2013.02.011" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. Shimada;citation_author=Y. Oma;citation_author=T. Schleker;citation_author=K. Kugou;citation_author=K. Ohta;citation_title=Ino80 chromatin remodeling complex promotes recovery of stalled replication forks.;citation_pages=566-575;citation_volume=18;citation_year=2008;citation_issue=8;citation_pmid=18406137;citation_doi=10.1016/j.cub.2008.03.049" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Smith;citation_author=I. Whitehouse;citation_title=Intrinsic coupling of lagging-strand synthesis to chromatin assembly.;citation_pages=434-438;citation_volume=483;citation_year=2012;citation_issue=7390;citation_pmid=22419157;citation_doi=10.1038/nature10895" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. H. Tong;citation_author=C. Boone;citation_title=Synthetic genetic array analysis in Saccharomyces cerevisiae.;citation_pages=171-192;citation_volume=313;citation_year=2006;citation_pmid=16118434;citation_doi=10.1385/1-59259-958-3:171" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=H. Tourriere;citation_author=P. Pasero;citation_title=Maintenance of fork integrity at damaged DNA and natural pause sites.;citation_pages=900-913;citation_volume=6;citation_year=2007;citation_issue=7;citation_pmid=17379579;citation_doi=10.1016/j.dnarep.2007.02.004" />
<meta name="citation_reference" content="citation_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=T. Tsubota;citation_author=S. Maki;citation_author=H. Kubota;citation_author=A. Sugino;citation_author=H. Maki;citation_title=Double-stranded DNA binding properties of Saccharomyces cerevisiae DNA polymerase epsilon and of the Dpb3p-Dpb4p subassembly;citation_pages=873-888;citation_volume=8;citation_year=2003;citation_issue=11;citation_pmid=14622139;citation_doi=10.1046/j.1365-2443.2003.00683.x" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. Unnikrishnan;citation_author=P. R. Gafken;citation_author=T. Tsukiyama;citation_title=Dynamic changes in histone acetylation regulate origins of DNA replication.;citation_pages=430-437;citation_volume=17;citation_year=2010;citation_issue=4;citation_pmid=20228802;citation_doi=10.1038/nsmb.1780" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. Vogelauer;citation_author=L. Rubbi;citation_author=I. Lucas;citation_author=B. J. Brewer;citation_author=M. Grunstein;citation_title=Histone acetylation regulates the time of replication origin firing.;citation_pages=1223-1233;citation_volume=10;citation_year=2002;citation_issue=5;citation_pmid=12453428;citation_doi=10.1016/S1097-2765(02)00702-5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Waga;citation_author=B. Stillman;citation_title=Anatomy of a DNA replication fork revealed by reconstitution of SV40 DNA replication in vitro.;citation_pages=207-212;citation_volume=369;citation_year=1994;citation_issue=6477;citation_pmid=7910375;citation_doi=10.1038/369207a0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=S. Waga;citation_author=B. Stillman;citation_title=THE DNA REPLICATION FORK IN EUKARYOTIC CELLS;citation_pages=721-751;citation_volume=67;citation_year=1998;citation_issue=1;citation_pmid=9759502;citation_doi=10.1146/annurev.biochem.67.1.721" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Watt;citation_author=I. D. Hickson;citation_author=R. H. Borts;citation_author=E. J. Louis;citation_title=SGS1, a Homologue of the Bloom's and Werner's Syndrome Genes, Is Required for Maintenance of Genome Stability in Saccharomyces cerevisiae;citation_pages=935-945;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913739" />
<meta name="citation_reference" content="citation_journal_title=Radiation research;citation_journal_abbrev=Radiat Res;citation_author=T. A. Weinert;citation_title=Dual cell cycle checkpoints sensitive to chromosome replication and DNA damage in the budding yeast Saccharomyces cerevisiae.;citation_pages=141-143;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1438694;citation_doi=10.2307/3578518" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=J. T. Yeeles;citation_author=J. Poli;citation_author=K. J. Marians;citation_author=P. Pasero;citation_title=Rescuing Stalled or Damaged Replication Forks;citation_pages=a012815-a012815;citation_volume=5;citation_year=2013;citation_issue=5;citation_pmid=23637285;citation_doi=10.1101/cshperspect.a012815" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. T. Yeeles;citation_author=T. D. Deegan;citation_author=A. Janska;citation_author=A. Early;citation_author=J. F. Diffley;citation_title=Regulated eukaryotic DNA replication origin firing with purified proteins.;citation_pages=431-435;citation_volume=519;citation_year=2015;citation_pmid=25739503;citation_doi=10.1038/nature14285" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=C. Yu;citation_author=H. Gan;citation_author=J. Han;citation_author=Z. X. Zhou;citation_author=S. Jia;citation_title=Strand-specific analysis shows protein binding at replication forks and PCNA unloading from lagging strands when forks stall.;citation_pages=551-563;citation_volume=56;citation_year=2014;citation_pmid=25449133;citation_doi=10.1016/j.molcel.2014.09.017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Three major DNA polymerases replicate the linear eukaryotic chromosomes. DNA polymerase α-primase (Pol α) and DNA polymerase δ (Pol δ) replicate the lagging-strand and Pol α and DNA polymerase ε (Pol ε) the leading-strand. To identify factors affecting coordination of DNA replication, we have performed genome-wide quantitative fitness analyses of budding yeast cells containing defective polymerases. We combined temperature-sensitive mutations affecting the three replicative polymerases, Pol α, Pol δ, and Pol ε with genome-wide collections of null and reduced function mutations. We identify large numbers of genetic interactions that inform about the roles that specific genes play to help Pol α, Pol δ, and Pol ε function. Surprisingly, the overlap between the genetic networks affecting the three DNA polymerases does not represent the majority of the genetic interactions identified. Instead our data support a model for division of labor between the different DNA polymerases during DNA replication. For example, our genetic interaction data are consistent with biochemical data showing that Pol ε is more important to the Pre-Loading complex than either Pol α or Pol δ. We also observed distinct patterns of genetic interactions between leading- and lagging-strand DNA polymerases, with particular genes being important for coupling proliferating cell nuclear antigen loading/unloading ([Ctf18][1], [Elg1][2]) with nucleosome assembly (chromatin assembly factor 1, histone regulatory HIR complex). Overall our data reveal specialized genetic networks that affect different aspects of leading- and lagging-strand DNA replication. To help others to engage with these data we have generated two novel, interactive visualization tools, DIXY and Profilyzer.

 [1]: http://www.yeastgenome.org/locus/S000004683/overview
 [2]: http://www.yeastgenome.org/locus/S000005670/overview" />
<meta name="og-title" property="og:title" content="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2187" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Three major DNA polymerases replicate the linear eukaryotic chromosomes. DNA polymerase α-primase (Pol α) and DNA polymerase δ (Pol δ) replicate the lagging-strand and Pol α and DNA polymerase ε (Pol ε) the leading-strand. To identify factors affecting coordination of DNA replication, we have performed genome-wide quantitative fitness analyses of budding yeast cells containing defective polymerases. We combined temperature-sensitive mutations affecting the three replicative polymerases, Pol α, Pol δ, and Pol ε with genome-wide collections of null and reduced function mutations. We identify large numbers of genetic interactions that inform about the roles that specific genes play to help Pol α, Pol δ, and Pol ε function. Surprisingly, the overlap between the genetic networks affecting the three DNA polymerases does not represent the majority of the genetic interactions identified. Instead our data support a model for division of labor between the different DNA polymerases during DNA replication. For example, our genetic interaction data are consistent with biochemical data showing that Pol ε is more important to the Pre-Loading complex than either Pol α or Pol δ. We also observed distinct patterns of genetic interactions between leading- and lagging-strand DNA polymerases, with particular genes being important for coupling proliferating cell nuclear antigen loading/unloading ([Ctf18][1], [Elg1][2]) with nucleosome assembly (chromatin assembly factor 1, histone regulatory HIR complex). Overall our data reveal specialized genetic networks that affect different aspects of leading- and lagging-strand DNA replication. To help others to engage with these data we have generated two novel, interactive visualization tools, DIXY and Profilyzer.

 [1]: http://www.yeastgenome.org/locus/S000004683/overview
 [2]: http://www.yeastgenome.org/locus/S000005670/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gl0iwEfT5rADvIUfHGt1VRzipim2AFLEuCjx6Dtfl7g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460032","apath":"\/ggg\/5\/10\/2187.atom","pisa":"ggg;5\/10\/2187","ac":{"\/ggg\/5\/10\/2187.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2187.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2187"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2187"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2187.abstract":true,"\/content\/5\/10\/2187":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YkGR\\\\\\\/uUG2d29Afl5Sol+GZAv29lwPAgDNFpJ23+5yAbfG9jd6r21668VjJu2J0NDgRvauSICAUJ+mzALk6+etVRpvSlSttLFZQY1S\\\\\\\/BS10gtquR6BERl5f1xhS6EhDa+8CN+93rKBe6SgyMla2NPj74s0YvGyOCEi8J4CJxs+8bkHHtIx9ncSNZ2tCPTG3nHSaU5X4NtsnSqQRT2BcUEKgMKhJyvhG29QuzDeG6cNmOGl5FCMQ8cmDkoPUR9t8bLH6aagP+9koY1wA06O0GE3vUo5O5ZIEsUaMnHD9D4B6cRn0A75FPbIR9o6toPbZqZZBX6VuK+WzeVtLv+wxqVkTRKx2nn2QLoscCb2Q6iD26YZztgpkXfQAbN8LTqtHxgK99RYdALQLlKxSFzLZQgGx\\\\\\\/anq6U+7WkYzYkriA8ILyJHyCLYM5fW4uNZShMWAppuTx4b2BeMCEZoru4v0QmtcVlOvi6TJhOO+bwq+l1IwGZ2pP9BnNd\\\\\\\/MS2iIZSrLSv8BMdiYn5DDpNXnlex+7rWhpC8E0PWBdIuMGGpHXmrTAeaT\\\\\\\/rEyRrPqnSZW+7wv\\\\\\\/KJWEEMW\\\\\\\/8hEJWzRGAU5oBHuwcdhlmTB+LAEQvWyo13sUifvdheFDblGBA8rfpdU+X+XyZHvWE2XhlJ5YxaxPfJ\\\\\\\/5ZXJ\\\\\\\/EaKW45e6fakySMkgQdyLS8NOPmREBb7P8o7u9B3\\\\\\\/b4lLPY9CVK0Ht1JkAHEK2FZsv5twXA2eTQ\\\\\\\/2HeEQbaXYyWoTrZPYdPUYAu9sPdXCAv2kTF\\\\\\\/2SV1inm86qf6JJ\\\\\\\/cYQaX\\\\\\\/dYdpLdsES\\\\\\\/yc+DdAM4Hv72vIk6SIf9Ka83PmVS1xTgcgfyiKXTShxCAatbWxetk0bdn31ya0oN5Bkrm1ehZ2ycqx2MkeiMaErh3P\\\\\\\/Pwm3yY9vCq\\\\\\\/GgK\\\\\\\/qBG9c3Bb3ACPzVJbnbvdFd9VGbuRa1BhMZMnsqL6hDx5FluKdRZBLRO6a2LqJbq\\\\\\\/w+LHJ607hXn4hkf7VEtTty2iVe8Wh0wwkozE7B8euMfCaCIY6AeHKXJOc2JeIjUCge3V0tjJLPW7jRUOqAYq0RMnx+IOBM9Bwq8qoOwSCXkHOcoBkUWR5Gr4U\\\\\\\/cZzhnhJuKMV00bmkUXsEtS0VEvwNbNX5U8R5e8BPKIpyfSx7\\\\\\\/noKm7YKEvZ8iTH490t+vu+PcS1Bk6mEidZTIgActp\\\\\\\/FblgkL9BuDzvSC+N04t2Fx+wZMlHJeVQOtEIFadF9sFApfgGWm5pCV5gO4Cwjc8GB8eTN+vEA8MUq16USIZ09jgu\\\u0022,\\\u0022iv\\\u0022:\\\u0022w5xiAZUYqqd6a8QxuGgsBg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225461c763b25ab5137b1bb4509dd0fd29415ccb47ab8ecd1b1698fc69e39eb0c3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wRtNRdt988dBx2n4Hwy8ghalc2ipaPBMcIb45+MRqrZNagScq\\\\\\\/ce3xqAAW2yMvNmpewm4GyYK+hi\\\\\\\/SxJlVMN1OGx+LuDvq+bGhwvDBEGNpDbMQzSD+f\\\\\\\/gWM7LjCEE19a8FOjU2Y\\\\\\\/c02mdAreDYHtc3IIAGBjYqe\\\\\\\/RvPqpeKFk62\\\\\\\/inYjxzE5TTyNC02zHx+irm98sytIlSzNcZn65FaSgpkr0BxiqFXSgXEsvJ+yaEeCJf\\\\\\\/wKLvAY8FHGiQBI8oDDM8WI8pyCCIAoWn51sDQKmxZ1wQ3QO7+vml+ls2fGlYfJbJkd5GN3kiPUiSs5Nhf2p9DHyqrTHZh4HNbAlQ9HuRI+qktLCfiJB\\\\\\\/EPF1Ok21dlnbWX2IfE5+UDN6VceiYNc3kSsVMONciTsO\\\\\\\/76aRrTLPXHBBRtExNR6cMtD8QmTWeQxz0Xk3FEk4qg4MnCktvPw+g6YW4vtR9gvOB\\\\\\\/WLb413FqUEeM9oAuYJbmgwRdkmanAeY9iv7XiwB3O7SWv5krydkKnBOP7wDn8LXr34Dsot+2TWUz38J+NemzjNRMki++8DBsBLTJklLrkpOxDX\\\\\\\/koOV179AkqBYdyIsKNuUOGZeyefjlvEslahP1ex5bCUbGPpeyyCzzBTzGfnYxVcpeNsYxFobZtXrDXhi\\\\\\\/\\\\\\\/bfssLicEIuLtGnqE6TpBoqaizIW9MCBKCLFq5BRiHf92Ft3dmNdLeeuRyeGjci+Ye3u2Hv4CBK2isilnyrC2HHN+mJwGoKYYvIDRNXRlsOwZNyJIgX0K2y05w2DgFU\\\\\\\/DWgGwF2Qu19YII40pdgQ+XObW1THJfPVjZVXe7gZeIqZ27kF6ngi2Yzrr6HMX3OToN1\\\\\\\/PAgDv03yeXySq2uGUwAZ7URTeBkCaWpVjBy4zLzmwtXmRh98WX08HetDPKqCJx1i0MVkd3ivdiPf99tr1gtFMZriS4SH5LbvsECtVdW9nZDJmCL64ODaarg53s1CW7W0AmeWCg95MpBjImoAPURhg\\\\\\\/+en\\\\\\\/wgf5E4p+60CIKbHt4ctJ\\\\\\\/uJ+lV7PjHPrcxzNCzVLzKtgjrKEho3L+TwQk70BGqLXgsrsx5dLPoe5czeXF9KMpx1L5UQb3Rl\\\\\\\/53l53GCFej4RZ4\\\\\\\/VDOBPkdmlTKUELadnCbaDVWEC79iBgXep6uKhRrFrFTzi7p314bnF4lpmsoh7b9eua8ex5AdL80I1lC2SWuCBpdV2pwgHD1XVQ9gCrF5s0PV9gDCXm6Gb2oIeTVnzIw1LtyPVwIOHZe+y5pb2kPbOrdpuHkjk6vy2+UNo8UTLocepgP0VyDVOLLmsjcr0gjVwmUlsXfNWUt9NY2Pa5LgWc\\\\\\\/x3AE4G9QBV5vBmgRImUVr2Q4prYo4Nj9MQpdTTbFTNlglVIbWVGSam4g+XqZu6z5RZ2\\\\\\\/TqXSJJHaowaKlVvi7f0AObqJ75cx9ZRhHppFC7OHtBm4bLZ1G8JIE6hTJZb5Bmy7diJu24XUNTfIPmlSEBKqHK\\\\\\\/MRmG8yNmuNTtUQjcuUKTaBhLIeLgn+mU6Rp2RMPccJdUyM3KRiIVhVo4OLB+qpzq4CKrC3\\\\\\\/7UEofQ2i7s18sPSouSRvlvNaZKpO6qv1EqnXptX0w9q4UolyAdXb5QR90mjbMVRVkGiZtcSqx+WlRuj7RPQvqrh+yqEYJdiCuZM89MEwZjtvidxke3hV3XjWpbbMnV8UAlBKQCJx+vf3+6FkxGIWp+surZYuUh9+bmM78KNPWG3oXUu5hrbJ2mhdqrcIoFIr+EcwYH5h8t3FSjdBZnCkG4Js9JDsbxVfsYedAlZPqLDTBba1VTKIytxtELSgKWuxkM3fElzHPrDfF6aCvzP0kkZt+GfrGQgmlglkuqU1eRAc1xq4bBIuCNDYI4biblap\\\\\\\/1yxahtJ6HGd5pZ\\\\\\\/YDVG2zvfFVIVZjjZUNxRIkD6lWCdwfi3QR3vGwVIfjDxHAGFcLW0F5BolIRsCSn2P\\\\\\\/GKYVU+PfNiiDbP6xu6xIRD3Dl7xl9ZN8D2LqdlbPy0obHw4VAZBcuz7dtE+RobGLwlFGzd0qvjrb+xBqN0jNFa92BJhE5RdvwGe6bwVECHg8aN\\\\\\\/0oDc\\\\\\\/8+chm9tjZyGBDYHbGQ+2jNSkMHmRwALlhehmkLT7Stjc7cKceTv2INYBK+0kuFhlUurOeSTMmLAHd4l+unQRjPHOin+XSgq67cfEHuw5eSvJWJZ3ow1n6JKMhTQQxF3z6WJBbz+7IceGJtTEVpDEWhGP1YnLP+V8ik+IByuP6fF2suDVNPPxoXkeJKQaYKiWpB+Peg+LvOlC1pKHYYJlU5NaRmP1eEEHzzTb5W8m\\\\\\\/YfVhm2xjpne3Qt5afJtvTiubpt30bDbaCUemFCrZtNOOj3I2Y7QCHUhVe6qQUti1aWTi0yo45q9Kbd9o56O+S1jceyS\\\\\\\/hWf21p9QauxTKGH4cJsrZIRpdc3i64PEjAVdrPeXVOvF1txWT42j9VEZFS+qcPRhLEMgiJvD5mExCh8+RGJVwErZiK9qBKjqbqA0BBqgSEkTCqfPJE4V81377ExrjWN7YuoIGY9IBhFQh63SQyEiWCp+A+asAhwOWGKy3OMo4H1EinqHQBzU73u9PIWRwAIlnY4l4lxrBXlRfx0li6sLm+odRh3ZGoHZ1rX5fO6DG6GkJ7+95vz8liz2ypliraGTHzh\\\\\\\/xwfF3p1s7Ba4qoJK7yydWgIcDecqCI5R2y\\\\\\\/Kii9NBg+r2SgUTQT9cc1PSkiPKPI\\\\\\\/n\\\\\\\/IDHPntHfJXmeQZwyW3yN7rIp0fpuYjQ+cxnKaQ6c0jMduoGKUC91S3QohqtWisrnf2FTb1z7I+tHEFUOct6LU+muy4rZe++v36FNs+D1mWadj2QJ6A5LqRxWdHuxa0MREPj1oWn\\\\\\\/q185O2aHThUdDbm2DkPIbshl4+e6Zim6hFb3Al84J9R\\\\\\\/B2BuAJtO5dOf5AzkxgmsEW6zUxF5As1D8y53mOOBn50C\\\\\\\/yV4VA\\\\\\\/yliAehOSzyHfkNvT9EdZh+jMr1O4Ad9rM93FNKh8lZfk+FJXUsjXRFzrFqR8rjg4zTlKFl1L63thbar0xRLufu2t28+jowb43Le3mXHrQyU8Bm9VsAOgqHizDXNFZvYDU8QivpSj+6STEYbr7ZYz40pwn\\\\\\\/Ber7gQcLgiLtmy3mcLO9HQs5Jqv3UXcqfClOezYjtrky\\\\\\\/3CySqqO3U7FufaZanjdObNPqxoAwFBGZpKYeeCuORDMX7O0qEpCHs7KgidCUEW5cZY\\\\\\\/mjN4RI3EW0cSzwio9FcQ7i6EB0gTYHCDEaertLd5qYUuM2D+veuaGd0pr3ncM5sImT976CJNfYvNV8cZVaXbZU4Qqr4GfQJcQbdNPHLi4Al9UcUVFWZ1Yv464pTk9Wmd1JpHHhK0oRoz1YwAyp6mDu9Ifb1S5q3cH+Y3J3g3pxnhhfcj9uRKXj8DYX3au1hY1STqUbLyGyFmLIwxyfElOMwamO\\\\\\\/NjXeG0OCC0nSBNN2IrqDTtDHnAVIsj5nS2kQvCdPJn0\\\\\\\/81lEk9KGfaqP2nYH5Uh20lUnUVAClNWqOEf0VgIVy6Dcsz25okKfiFPfDJ2i0aPxyORa8OGCe0jBvKp0kd5xW5n0fsLHsj0hjUcZPkLzSGwsL41UlJpKzO+92yxak\\\\\\\/1YUyyATh+MA2RVR7+jYY5bQ2jWpyFqbSHZ9qTtxMCQaVuSghXl7zg2LLaXHswfZVA6dfBZh\\\\\\\/Ij6W9Jn8pzKoq6DLnhkXpGWy9pPpKSgRmXKkQ+SsWWe+Dmg+s3WRiHH3lFYwNh57kOTyb0taRo6bDbaBDgi0ct1J1TD20ZfkWPMW9pIkm3CUXRYfWxdWYeJRy6D5Hq7JBJwaF9x0q1StLDH+GSHkOMmwRAYJ1Lu4ThJjGCqn1nbsxFS62QMsQ+hqsna56\\\\\\\/F3SexMdU0qW6y\\\\\\\/CCoJmrqhr+n3yC1lKOmysIVxvW3izJ34A\\\\\\\/Uc4vz8ar9rSju+JSJF3Ok0K7KW9vtN\\\\\\\/CGmxdSzeGf3FzuhaAoCUQje135ETnWikjPymqUkVAw8LooKMDHG69C3\\\\\\\/NYEi0UAGGIAW2lUD5ZqHNEovUUnWfFS6oa0kJwd2kaB2Jyn81XzOADVxmb5l9\\\\\\\/z\\\\\\\/gK9\\\\\\\/CMlwnKkbv\\\\\\\/+FfLKaBrOl1AOcqacf5BQtRF0G6J0zRUUjImSInh4k+w\\\\\\\/ErcbMvpf9Lhimk+blE7lEyyEpOiE2eaIWI4nCqjULMfzdXScNagWNSJv7RZ21\\\\\\\/xr5vcFS6CBGMkYzt5lA\\\\\\\/7hktfOxh74sM+Bq9PyOrACKqzKvqMZoRVgtUall06tej14lZH7QSp4yLz68nUbUZHrHWIRMbX7aHJCw\\\\\\\/\\\\\\\/myfeiFSzKIzzvdTIEAt0RuDKS2jMQ0sKYWLBvPmbG9Lfz0v\\\\\\\/15m5C\\\\\\\/Kks\\\\\\\/RXyEllycd\\\\\\\/NnvWE13vijFxvSJoIgKzXpH7LKBq0VLTzzS+fOQAbutDpsaq2vKLwIjS0e4X+Jv0kxy\\\\\\\/Yea2+EjQzWu\\\\\\\/xlvUqaCt3YCut6QlKuNvgBsMtDT\\\\\\\/Qapb+pbk6iaRGA7qymdgUsX80p1qd6JtDL9Oc0Eclf0Jjy8acyYcSz0QPfJ7V2PU4uZFZqCqL\\\\\\\/cPvMr4kKKe2JEE3w3Oy4elWRLbTNeJB5nugPEHMKYFMxHjTONeRTHdfaKoU9qQlm0REr51aTqi6V\\\\\\\/qYsiG7hAzKN078epLo2x8mVjd5sM7mmCAzbOTvy58AWQp99GSENNOThFu21zX+1tB0pg22J+RhJkTc19UrLisNZqsBi5Ip87FpivMU\\\\\\\/jbz7vHCNuZpyBPP9WfHPUlixyvW7ACBWO0xsimS9qSprqrstq27lm0crRFtWl2F7zomUDh+zTx12JzoqQC1o+dEn3JLX3K99vZbtPNgOasMLT+jf5Fkjham8CMZD7+HMRcI5XD+kpcldHk\\\\\\\/ZhvoN2HkQfEANqRbwfy51pNb4WNmgVW6JClxlIYDeqDvoQXP4mlQ3ta2Zb+EgHiH887XYey9lBK1aC0Zs9+zE35QIVt395QyRAWBA7qrBA5XYJ0y4qzlSCHVrAe3iaDDcMKWegBFymYLrU9H7Kw+1yvP\\\\\\\/FFy+xwPEmYvwyut3uOOcBFkHBDJALkjXDYFs8imhqW\\\\\\\/30k\\\\\\\/Xx7iTQ4OunAkd+vqdKDJcz0APl\\\\\\\/KivSh1mvXfrrdQl29j3JzuY3vtxbnAbTcYUHMoRc3tr3Py07lrmhX3dm+0Enf9wep1AZ+\\\\\\\/VstA\\\\\\\/ajWPGbRQmGV1I92+ucZ0mJMDRL13sqOLMNiNENkh70P7AyvASr7lOvfUsj0pzIYo6pSgoYWe7fliKtf1Y5\\\\\\\/OMpsWEoQqmAUnMrozW6giY8Xj8fWgbVFMP8WNasllKsRsRPd8tgGaT9O+x18Y2QW\\\\\\\/wWGvjHIiYaDGHuW\\\\\\\/gBJo8\\\\\\\/ylmsDJTpsebsF3hn5TsuG8nQXzSS1JHetnlH4bUq9RCdgbFlboGexUeqVxyv97aM0VUWZcOm66h\\\\\\\/CuW5KHjqoMngV6xBK57x3rXwYV6TIG4uzIZQ+00Hl8k9ckXtoW6yW5kNeXmaPEewA7GnriC7y1UH6\\\\\\\/bNk+O0qgZzdmLTEK\\\\\\\/WVLuePYGibtKvHM5d\\\\\\\/Mrx33HiTHoQWXhONUspbub8Rn8gN3YTUxfLk1AGhgCb\\\\\\\/KSneFV\\\\\\\/g3ltYW9vf78GuuM6+TRde6C\\\\\\\/naSx7J\\\\\\\/CC+xBgpsEAutIjvlPmqfPexP9NjFvJLxlT26yGsPAjAY78wgpX0dFgfcsb75dAPTq6WY5oN1HnQJWQcTz08CneZZdRC1qW\\\\\\\/I2JTmImn7Jl2LP\\\\\\\/uOHH95iSjELBwI+53CF+DM2UF6OaphsAP4j4iNL365ALEg+Ev7JwhfhckMda0v8pVZJ4xP7s7Gn+3Bn9vfoCYvwq61wn05keVOUzQd8sC2AcTwW6ouRYuwJGtJsYz8aF0tWjEoslddAx5FwAQcEPTeytvcRajTLL7SCMR26LbalDmO\\\\\\\/ZgdHm1th1pl9qrk37d4Nzr+XM+kAnQmtPtU9Bzp4sGHVma33Sq\\\\\\\/K6nJOQ8GyyzE6qQG3wV639H4jMCP\\\\\\\/Y+\\\\\\\/XnG4vFjqEq\\\\\\\/7WOJ+IwAg564EiYZOEx4+xVUmpXFZQhnMx2e7DhH7fXyD1U+pWebqtaFm+NKBsomF3fBHJ1BcHfl8ziXJHTsIHcV3ffuLpW8tHZ00dOzoOi4CMWt7\\\\\\\/2wwp\\\\\\\/j7Cv7ukKyO8PEjD50X7gqZNl790K40x3xus\\\\\\\/WoxUjA2qqm\\\\\\\/A2ySrbkWi3kUEdUo9hTc7+iMJNtvhi9EL67D7tj0\\\\\\\/aP5GJQH0\\\\\\\/c\\\\\\\/ss73W4cJiFnVcHGztE5w9IOUqSm7PEDEh0ZNtPOJdOOfJQOd8kyY9ZmwWWjdLfovRKaxUmpE+YwpjH+kkWdrg86gZ6UCQgpPP\\\\\\\/Y7Sktz\\\\\\\/fvaLLzDllCnNkBeObIrQ5Umjaj3EBeVBHC0qDzWTQ\\\\\\\/TlhlV7IydP4iSLCLIb1KCb4AkxB2Or4QLzEbg1QPClBx6rR3ppDEKkyz4cErVfkC1LVEH28nwZYc+G+vWvaGZBkMiXeMV3lds2Oc17gigBPZ9xC9RL7fK\\\\\\\/9XwaobExbOo1497BG1IZmZcbjDnS6zMXcp0wF6ISl3lj3i9v3MArHtLMqIJ3VI00xE9nATMQhELHKSJJkzf+bvun3uQeDP0N6PriaUsqDG8wcMtMswv6LWh\\\\\\\/b1\\\\\\\/LFLJlslKA8JNjz5QnImhyt\\\\\\\/SVvnlAdXzVZImfsYEkFhtBj5vYNOgDBl\\\\\\\/lZs8Y+\\\\\\\/uX0lt7Zu7KyiccDVhnGMwWYJget3yZdn3kUZWeOkNkwASxG81hSD4FYrMgyuou9wTf+6bQsifcNIEMcOn3t4Bp9KRGTaJjFKpLhHcVC9ql\\\\\\\/ioBBBRNABRMcHgVTpjZy6UugzxWa6HhjxlRavlw2\\\\\\\/XQ1EUPoGbhaOqh8Yx3SBjuPesOeiIcWOexJ6o+IhnSXkHxlU2oAT2RADbqDIjwOF\\\\\\\/QZgcqYUXdNiOf1alV5WIMWLjhUAGDLHMsOdGW5WBedW+kK+FyslxLraP\\\\\\\/WyRHWmFUAOvBXLwoog5gf3W0j2BD56Jq2GJnf2Bxjfsgl8C8tx96JYZqXsheh4Cn\\\\\\\/TcB\\\\\\\/puOpfzIST7lgBmquPsYhTZ3ATJZoMAFnZcHWgBbfSxxDB9iLMRqu6bLKUU6NYp8n+YuKdbPG0PTrOXHee6lxqqDyB764GFph6CXvfN85eC+EGBEXlxxa\\\\\\\/4R7H6OKiphoddSy0UzK\\\\\\\/ym8wEZevGcAlT80lITu96MAd1E3w7szMSlYyB98+e\\\\\\\/xBP1fcvXtK9buxQdVvRab+eg0pBsvJfRBDRW4ghdaIWXHgqjxiZWC0G5c6Mu6SqwumAYrFt02cwTaafbzBfT\\\\\\\/JYYjlMzKCJJdbNgKhOVQmbj29MuBfHQE5bXlpPafuhg6wFJVz6vujTNtKdGEOaMRNrB\\\\\\\/RqTCexDWxY5E0zo\\\\\\\/bgIsk5PBuDfZXOohtNcgpOpTmSvySQQ0ACKsoG88NwUsTL1yMqS4RHMP6CJ2yrd\\\\\\\/Teeu2vDv0qjCEP2b2IlSmtL69EU\\\\\\\/+apj8+Bx3W\\\\\\\/dp9\\\\\\\/1ZUVm3+UyKa8nCnkmBJA6y15HKzgT5\\\\\\\/c64eig2oiHxRiKG0vuwUS+bPebvWqx9oWIzrJnAO658Hfi\\\\\\\/ez71M9JnB3UV3DLgjBvHzBxCOFeKnKk9EeyM8XO+T\\\\\\\/8ARcpCXiya89xLsJMmOL7HBKHfLgZUaPa+WuGgOf9nJ\\\\\\\/NoB6jYSOQ\\\\\\\/qZMHwafrgLDUMYxR+tcD\\\\\\\/Bu8KcCHAtIsXTxXTbpnn+pMPv29W17HJcykUPW96CTVP7z9Mh2yVRRnG8xNd30qKO0P+QLmPYf3GJHytvaaT+6UcxYW3zk+3ekjy36GEkOHbIgRGEmCx71sbqhUqUXreorglKX8xxahXDr+RMv8sy91vtTUy0lWw5MwUW4\\\\\\\/Jp67P3+SveboVwZLzT6BIVhclYDc7hBakxzRBStxI4UEzXvqWznFjOdw+X7+sAg1uCUp0X+j17DNeTcOQumhXMN12r5u0tjKJ6ERv4Ip3xiEeoDlpf3LaBaIjSJoTakrm6xQ0+NSoP7FXEm11hOGGM+RS4rXApFmV8kjnP3Ph+dNX+GqOR1Mm3lTv3FBkElIbZh11j45o\\\\\\\/q7GkOsxVWCxSIp3b5RT8e2tUeARXRGFI6PgSJG+RwbnJ\\\\\\\/uFqLpOEEM\\\\\\\/w22KOZfK+GV\\\\\\\/isiBi066ZT4Jyl7KvJ4HN1XcsOk5VIgNhOGq+XeRdNQwRxg5eitygO5qbzhC29qsNxNzFmBB8bk4+X4EfE\\\\\\\/ifyd8Qux86bzJhsPK0BIbJbIcIr5N9ZDxxteWUBPud3Na1zKts033ZGJFYrEouC+Lv5uhQIiZb0s3Jj5bzlWtZxQDbw4rDjVbHPChKtB6YXBfpjzaSmSoamw1qf7NZrww2nWl\\\\\\\/Q0GKpAE+YjJyQTGidU4ep2epcIZNKrjafV95GhykTMmFWGoQljNuwFx6Ds4atuIXosSdos1iDHO1iat+YqZ9M2Ys8WSfSlHXHPb0HYLHPnTfv419\\\\\\\/mI3Iqg5X\\\\\\\/MGTq2rVtFL3lG0DZribB\\\\\\\/208zTql0\\\\\\\/XB2Q6nKR0BZ1WVyPKD69OXU+qFY5AY3YnqTKsEng2WYmb\\\\\\\/uOGQ\\\\\\\/pbtqnTtwPs\\\\\\\/vGPgNdDym4swovExzRVnb+DfeYtOFMlHdg3IF\\\\\\\/dOD3I6fHI5odxUyk9Uvk\\\\\\\/5YNK+u9GyeAPMZ2XTYnwofzvEhYK0Vcp8VhBgyn+m43FCSGKAmmzpeagNkX6Q3rx+v9DFyILLoKcPwn1VDFqB2JOlhncBpEl3XGf9c1RACb3Y9poqOF0pO9SlM4ESOYY+kL+GxgOajDr410\\\\\\\/M3LaushL6fyuy0NeqyLDsTSIKtzba16mVAUK6B0+QOE70ExZU8l4GY98x1ZHFsQdwy1DBwdkY6yFe1CDsZJytEsaOfiQMVQVObmjOZVa0FHk6FWrEjrVZYtN+09cVx4B9\\\\\\\/b\\\\\\\/UYoIiU9kypdNw8XHeyrAYC0GzogqSEmwEgwqD6VRw0lE+ZXRYBFCfc4Qwe7q8i0JcRY6ahEq3Nk4AKB00rJmu294cFKV6k89IYKMiXEZwQaiLMJX56cvWF\\\\\\\/4Dpxlm49hDGcsk8ASg67DjEk013kLE9e2Rmyb3UNGILH1pUZrWGmbgNgjP7vSOxegZULCsD7TisYK94pu5jku0CC2lU9HWnZlIvC9pV4oJKLxdUsuXgBebIPDiog5do2eOD1uOOYqfzgGDAR6+wOtwTIWZw8hqg06WmtYB+JtB3p+\\\\\\\/CL77p+xYqK2wUZNDaQSJBpIdcAOfcyTvTxK5VkxLapokhZg6nMScT3NOajr4uNxzqvz5O5qtzfKAl9Ye4uQ0R+wgh6jIWD7ZPgENLR3nnkx1b9JdHYaRa0y9SPBmZVQj3ci3ak4b4xMoKyK4QD3HQO\\\\\\\/OrLjiOVW0t3obo\\\\\\\/HC7RFHQBt9DI+6cWHIoUj8cHIK\\\\\\\/uAJZ6EkCOOVBNicwSUkwmNX443NxkQIHvEStE5f1U1IdogBzCbaqVsREbpJgxWEPNpn7C0Nill2rFyVhHBkWlw70OHBo2mv4cQhd6jgOgohfpraKBr96IoD21nvboGIVmJOnSAun+HTWPNVCxrkl97kAExePfk2DHbRp73S0IPQZSiW+diCnvYJ8eL0a5ANTZRMzISEa1QXfTixxLqK30Gn9RxD\\\\\\\/dE\\\\\\\/nW55Lwil+3TLzvCs+yMFtJaEf9s7oba7eXVrQkRCgI5KWlxBhsO01l6oxBi6Xho\\\\\\\/rCUojAdBdBMSlw+oLQSmtiB87Q4NrxCJcS7Lv25tvW09MBeVIy61djHXZoV5daxgpgHmAIHUEm205bV45qQ5FGxw5UII6wy5BlKGYN\\\\\\\/3HaX86SouG0J1Fm+sQ8D1yLo5M6wwewS+1B5Dvw6XjuLYvqmuSfxMApF4Ff8pSVu7rEiKF5\\\\\\\/rabYkch4+J61AQn8Jf0fXjtgf6YVsvLTkS9XvUdsxgDO18LTSF71cEGFUEppO835oHloqmfwifj8XZAKRBhHC0+OB0l9rQWsqhQHSn6MSmwRQLlrfcgiXAWGoD7w9p71UdCrgbeN823TXQ1sNM4rRdP0pdpGWu+yTlm9sXncgZSl9SFloGpm4a\\\\\\\/nhX1mDlAOvCJqEI43EGXfrFzpmseBX3asF+J9XYxXmWT8YixvMxRedas1eEev5tdZIqSnUHfqVIOoNoVtgO7d40w86VOYlMkhr14D9t0AtfuANCsVvz6UVlY5GuQnVN3tPNu5BLsS3ymD7y9bxuWQHfZpfod7GvPKUrSnkLR3owQjF2\\\\\\\/oSKdsDgD8dVXm\\\\\\\/tHB\\\\\\\/v4BQ\\\\\\\/rLHXLyU+T0Mh4q27aw2R\\\\\\\/z\\\\\\\/jPwxYzuibTAXb3MP7NA8xTvWaomFd2wMTHtXsQVKgXU5xtHmaS+r8DjShHCKjwQxst6l4NNj9wZM4UDBeGkTK4g+2nJR\\\\\\\/71lGSYsUSHdl2DQF2JQUmKhI20vl2jfyLyuQZWkE8+032gCMXV2d6xLHonWa\\\\\\\/x0f6+at+Exwu4pZVb4KMgPY1Fwv9oZ+PX22vq2EmXvsI7iGy5QjXxTTXdqg6loYlGqp7C1\\\\\\\/j6ufv2uRpG8GHNRvUbDuxmZdTghjNuY4uRl8GdeRtFnLpQM2XD6rEZyUbTn3k5LAU964lvOvMw3cw66jNeG+bjgo0\\\\\\\/Tp8Dc2PDeMhG\\\\\\\/4TvTwvFaQL9Bl3V6kVq\\\\\\\/bdVqcnNLYCxWRTtvsUjjto4j3714\\\\\\\/VgTDD4oXMSIzgU4jWUMRgtoBcZdQYFXpypcUBJxQBsvK3ZeboTy\\\\\\\/EYiWV35nOi+K9s9vxyb1Nu6BYIqMZL4l9N2pkfyTFX3BBLPZYW2n6G+XeUhoLQhsELROaEoN+behTUCQzqlc3XBrzM9CUzZsAJ0zAkXsAObjNmkxSEAwQ3fxyxLStfTPTWEV\\\\\\\/Si+R9H5FpQ9qdbe4\\\\\\\/I66dEScsiWorC8014DAWoUyCehGM\\\\\\\/0ZcFPVnTAgucjLxPBvhFGByuNIv9jgacPTDwNw5YXbv4q7CSe79LzQJ9\\\\\\\/xjSzwScvbywgzNfxUxHNjBHLU3gBCHulbM0cBUmtQSA0zb\\\\\\\/kVvgxdGIr+ZBgV0hcgDfiZ8pZqg6SYLpgTWFONq5y\\\\\\\/vnEzDcJPuivAQI+63KJyF\\\\\\\/NKj3NYdf7d4+gHZCoqQJkpaxXqU8q8EaiAY4CP4f5LP4RzuQnU+jF04z+YjPoSyimtPN6V8mz2wxKppqOHse9IphML\\\\\\\/H1ewn5MJhNUN+Lg8F6H67mSHjCvJJecsCt3O7Jbxa+zcMl+ZimpysRC5Oqz\\\\\\\/KLF3sGDEW6Hwc8yABOMDwRZLn8wkFwfMkDE4REZToQjoo05yP6sYd5U1NeOYWhe+\\\\\\\/nCxvOalGFkqgVXPQqgUFL9pRNNzDg7ZNzZ4JW6W9GBOTv3d2wnxGJdTN++5QbP4CBCwyvn9Ocnw3XCeRp4bkJtwyNBq\\\\\\\/CoHs520KmDWSleQz\\\\\\\/8\\\\\\\/jQoI6PBa7tqUnaAOe+d\\\\\\\/3xbkCML888ZGDZry64Y5YjzySmIvUHRAFutYCPfnfWFgiiFifWnkcEdzXCayTTrKEioBX168zz9xoSrRk2QsT0tNxOwUPO7fD5rE4zK7buPQgrtG55jpWv7jqfBOkhTXsMD2538SC3A+XYcabN\\\\\\\/ba3E1FeoaG0lLGT9LGVcAFrzHKvCqnzEYapg9\\\\\\\/eGN5GYKdEtE0gOoKsxa5D2nAtutovug6GIOkFJ1BYIN\\\\\\\/+cYM28gUMP8lMBYIn2KHyMlPxPiC8H1nj5Q\\\\\\\/ciCtuOR7IjVFPLKkYAYeQ9TFCVpp\\\\\\\/BtlO8kxTIny+nuIZ5BiNM8TDg7nMBvKRgb8ahvWvRDCYUYJUzEQXG9EVN0686G7Ha8YBRtSQFEntqQx66KpP0QejEF0Pvraq\\\\\\\/JIGlO2unyqBH7GCKHWUYoYiJkFWmzqQfeP22g3gscuAqpjipy5SP0mnWAQ6NhahDBvFDoO7P+wCtYTRTg\\\\\\\/fhlO2sI3rAXhl5E\\\\\\\/HIUygHEIK9RrRo47ozMSUZn4BKTVluHe+ZKci58c2zXl\\\\\\\/aFXzRYwYCurupbS3Rr4OMDzvte26yrxIZ8pAismSqqNu6JQQCZgaXOmQlJw2JsHh3zsQz\\\\\\\/v65V64zE2vPChn4wRXub8tLHD3a8hJNhOSRt+CGNPaIs\\\\\\\/OqnXDxR9T0H6CBMJKnXFak+grLCfJjXloObjMq\\\\\\\/s+rTQqdOKJ2w3PNiZCxFx\\\\\\\/UFeA0QkyAY0s2a9Wjsmku7TlxHfBjBxQFvMhW2AYj4GsI3ZzoXnRuoKZVj16ykAmeOBNi5zA2glTeH9SOdqLttFAh4rVn9MNNVyCKqOIzEK8+ZKl3Tf3Vkn60R\\\\\\\/8D2zAYzNLylZToQU7MOT5pB0U8lqe4tfYrDLLXBPx3fpTJr\\\\\\\/5mux4r4JnjW4ZbExMnuVk6zJR0ycutdrQi9dEyOpzBYzP7EektqKaV3CrAo6WPzba+6tQe5oLeLUzEZo71BZMkecQW1T5IB0Rh+YGXkMd6+9PozLBZCAZxsb3NcqKt0uBETCMpfpd+ceJXCwpodt2m6ZvvJooCo41aoeGq\\\\\\\/et6OFWKH5hwWO3WqyiGYxqDPYUM\\\\\\\/6nCeDbwEUd0+YTx6M7gJUTAGEvKv672b8QmVmWxDQ5V+QoZ8O7O3hRGulKQdnxK2vQWQJ\\\\\\\/2nUh7MavvyejZUoLIeOdmSZLG1ErSzLaMY1Zoa+PbFwsp0vWYluPe7V3dyWDEoHuY3qoty0\\\\\\\/JQPo+qibLtsq2ctMPSSewzZE++tawU7IReSOjH0aw4ndSlbtFsQICHoeI1ZcDaoQsYBr6kciaIwWFnLsBnuDeVfYRZvU2tZN8FQNVUVmL\\\\\\\/HJEi00tXbhNFk3NxIDEYygwQEDXvpuJ5WEqmnWQ0kUPd0VoNqABeLpxgITf50GcjRe2z9ugHbXOZGo7uyjQzOcCBaeQ+GoF4iedXZM9m21uwc3oOigvmAmUQz3KObNlOTkAwNq7eMOf42bKVVftGbWpwiAizJzpe5Bq6y\\\\\\\/\\\\\\\/icUO0JqOVqaFhKIlJ6mmbeBZcCqd92JbPgrHqsRpilIEHC+OBmXLpZQea1BP2epOh0KAIRvTsV3nlAF2YYwIMC1gMf\\\\\\\/wcKKQ2y+CdTUsFXbLzDmXnGQmLBgS0a2Mlqg4SzHk9Kx7xMlqTKJ+uGe1\\\\\\\/iIPpToLDtlEiaCC1\\\\\\\/Z7+BeusREer3JFQCC8sY1VuEN1odExSID2pOM4hMBDq\\\\\\\/OsGVOnKsWSemf4aH4BDjTFc998ig5nAawPmu9VrsM8k696aGR1ziQk9mv3bTzVMZ3XwtzWe\\\\\\\/ENV2leQxgEehgB9iIV8mPFD+N87xcX336Iua69c3Au6aZyPO09qb2ZtNVacY6m8R01F3nbcB8jErUkWVdS7\\\\\\\/2rZJVY6uQ1vuO8d9Ye6K30DrOIpLU0NqdkthYq7nesp60iIBNJU2ieHOz2bJ6r+sVGJT8f9sBu7nIPZet4Ulv928mjDHe\\\\\\\/JE827JHzTp1+TrlupmAaHioWi5n1sN89nCFLFL3gAUCwx339drc45nHaf1bKj\\\\\\\/IWAS3Unxfw6k5yyWF22vR\\\\\\\/uLb4WSDZjLwhbYQm4CqDnz4FOHmyqwdQ+gUr0ZsTMTK949DWUpMNq1epQ6JoSqs4r1pDCEp9XgenMNqm1zOS4COZ3BI13\\\\\\\/0ykXUw1M39oKcmFoBIcm9JSbxzvwhH4AyucymWsH9L5DRnE1otvCaTB8FSDdHK1w3WZBfszK8cWb8c8HSvFh3XmuxMgzO+SpQ3zRdzPYUZ1IdLNxmDUFZRMT6AKYwtEhlSmkcKUWFzFx22Pwiv12JUVPDYRDhn3\\\\\\\/XGfwj\\\\\\\/xfBtzycE\\\\\\\/GHzLKqA0HB2nb0SYl8rMZ1q0XW7Mcn2P4POFMJLcGMO\\\\\\\/MUNzpboR7jnwj+9fXAhDt9LnA+2VcMZ+dtHWuamIjBe8DTMBi1sJAvYJlF1s8tRPsDyVNp23poCc5Qg3JuP0a1xqtEOATimMbUzqQgthFbCd6XP1\\\\\\\/nIe5dYoxOTTZlVuLD+VsiTnPydAmggmlW2tTr6ULSfCjrSIL0I5OBjyKU6pa2\\\\\\\/hOEHNz4Fj7b4c7W1WNaeJ83cugpqXd+hOO5c36S2DOvscDv+YR5aei0FezUlOe7kPZ8W32h\\\\\\\/1yuOOcUyREoptaYDar4AWV\\\\\\\/LeFlFZr+YkCAyTq0yELaTlTAMkf2iEdv3aDJMV5rw8rlKGr4dFlTHiRc4EbNPizZbNLmiyR8QtLjvr41qlp+NiVsoZ0EvymKnBHcGvZ0OtSKwJpxkCj0MAkLkrbyBXcVwcmmio\\\\\\\/pf7UjKdPvbOwa6wQ+ZRytLkzQsQqEYXQIjkm10EVhCN6X5Gzv9NqHtCcaEODFju5c94ELXygZds8SShcknCVH7rNy+ZlNlDREunuG2tzZfL2zMPg520Jnn8ZGuZx3UY0WTd4xVtM7jacaeOWDZkvtLnEeSwnfdjBBljZXx1BrPPn3HRdS0EQVwXF6wgp3hWPWmxWmJNFWo0k2K9JMMuINAHWfiS9XKAH8CahiHgjvaWhMzfvhE8TplcrXt\\\\\\\/cfBLg9hm7Tbu6of7aT6DKcT4PdqG2JEGTH5Kt3TLykxKXcnC+WFyrRYjwyiRPnW87UTI0J5NYofgbJwiclJ8emcnjp49qala3yzjDEANCNyAYWtLUT4ZGhGUP79DWaOO97cKF6E\\\\\\\/ENVjiJJxZpTfLUKsfOVoZNfi\\\\\\\/44BBBBBJQ81efVqvfzyAbmBXIB6boQBNhJ\\\\\\\/pYJFoTO7KC8i1mCRS\\\\\\\/NIR8IfvbTqdyzJsNVleRvVGMUCJ6MueNTT5yDlSB+3eC8UCAl7bnzAhhKhFVCIUHBdLN5ccSF\\\\\\\/RnohS\\\\\\\/Zg9aaDPVSL4xHsfxXPPgR22SF1v8FArtMswduMPhuhOYoN2z5TerO3wtILO9HDdlMsUvSdpRRJGh\\\\\\\/ZuFknkTwBfryNNs4qkuyOSvPyeMcpSMMhcN8yRqtS8gVq3qMj+HwIh5njS2efgyLlHDEqW69wUqsQfmLogx8gKUyfKlr6u5w4VIonMkLfItilfHCiAhefVCKMHSWeEGYg5s7bhkrKD8ARyeKtc4YgNUGWexc7mgS3OG3Dz7Uf13+NNtuwDqm82R2q9XOVxJsGeksR0bKj+wvq0jEmSK+kO4dZzaDq+nRkfS5eqcgY9\\\\\\\/NdHu9MnGjlVUfUkza00AIKkm9YlIDI25pfKYKT9rn2tsyet\\\\\\\/FHhjcZNwHLnlI\\\\\\\/ptWA0E3z9u5lAyJf9q4zFa0E0JVj8KtfCu0x+dd2oP1B4mlxNh8MfYSyF5fXgtG29A2W8LI4uwH8jV2cLp0LpV6dr4vgVx9rftzf9kagWbzWJP4j4TkeIEtr0IDVu8gK4YXyPd4JTe8oblCgFBS2wx\\\\\\\/kfIMYZBO0fMnrW6W3xV2FpRuSZ9QLg9EMLYO7ZzPJ6Z7K0\\\\\\\/\\\\\\\/JZitVf7iGplxn48aXk3q22HR+3hsQ7AwJsexw6DPRRpPyRQacDvUEYSnN0w1\\\\\\\/RXa2khXdIKcp1r9ML5TdAmMlOeJmynIKMvFLybmi+q1lKXGYd\\\\\\\/pnqRwWRzDpBJPkSbKmEpJGcWmXrB8meJQSYEP9JQb16Ad8ELJjQQ\\\\\\\/j4xUG3+fsLizzlUhb\\\\\\\/U0Cr7n33d2HZw5L6d1j+LEtQX7nDkAVEQ\\\\\\\/o96qA5rC1UV1YzxqxBznkuSGoxb3KFNOrF3ojZvtpaJZbhn4IOka9UWEDrWhPmV1LtsZUIfKCtuKOqA\\\\\\\/yarBC2LpUBKtjxGzLfDWrdigVAUjzIZQSbSipsmOGBJYwecs5U13aAMp4dK9Eu3Uw9qt02fSYILnDn0ZUmPNbaTojy3OtPTUJUbguoGYP2qEimZhA3Aj\\\\\\\/97xkoz\\\\\\\/WuMJh5cDMb3Qxlkwo4WrNkBW6KkogeluCe69tFVzPJI8q99yP8qEZFMbid9eOalptLF08M1Nlf5HfeGAH15kqYk3v5SYfx1Xg99eIgoZnIFl29R8K\\\\\\\/1tECmP3vK9DkilnIlZMtCEs89wgx4ge7DHgimIVdI\\\\\\\/7WUcFZhj112kRy0AEvMKD1ZKHPR9ohUpBfD4b57WiEXMx5bCTcW7\\\\\\\/hMmJqPtyZACsHNNsIdvA+CdF4MEUdUpoY\\\\\\\/ISpJiCbjXBiXGXho5Mli\\\\\\\/F20v1tjXQpwuZSAukvtpkr5ud8fSLI+\\\\\\\/p1u1hE9\\\\\\\/vGNGlmS+n9EYUElqc2dhq4jC3ju0RrTZfpE2ILSo5+G6xFTDAwDFnq3JPRxd4CRRwAl1LNwi3Kk+MH+IPxwny+\\\\\\\/M2oDvy926k9E25iDXg7eSUuKcbX4H5wIOTc8ZmtdRappf+4xiOwIbH8ZPNtRtandyy0mZxCvr3fPxT2eIXHlo1Ncs9J5+zBuCB15CG8zwitWEGHKFbmj8d\\\\\\\/OJjiqan25SUAgdngNbLZGyKoqVAmMXJIlgxGQAhWcNr0r7VCpVnMPToKA9C3Vm1jFT5oMMje61iqI0TvZumrVM41YkROChbhHyEYU8HNFec7dsVEa2p7XoTfl8pRRDda4W4cUihBa6oXWaaoTABNGBFGOORctq3AcHMEbrJoeHLscqrLbL3p97WDVA\\\\\\\/3GG6T\\\\\\\/cA94S8L1ULpnVBfOfvDp4lQ0nNrAK3k1f3vKW0fM4bPBd8qpcBjUKpSD5mULyiq+bS38tGLw+6xzqEJjuwaSBEtid9YZirixX6+AFXob7F\\\\\\\/rPI+QEZEy+YYa4BL5+g+ndkTW+Ozs5NzzTq9UgXrmC4nEe+BKF1gY7MAyZ\\\\\\\/2OqyBF78NcfFniRXROLuqzS8\\\\\\\/SkS53bmcxPQp0ix0vZXgNVIWOMkmI6JFjpApAAj0FtEo5XLjXEM9sodDrQrM1PpRe29uKWcp04TJtQ5VihRQ0TZPvS0G0ebednLsflru8jbw+a6rpYRdsaroMvL5FJD22jwuG0E+v4AgC4+sUl23mvxieV9TItuOkwF7vOZ\\\\\\\/puJKAZsNxs1jDqKG2UWN53lx0AbJzYZqweo71g4\\\\\\\/p18I8svq\\\\\\\/Ip5xo66kbzT\\\\\\\/WZ\\\\\\\/MP9gfyJeBNZ9nhdYUmcUzC0Vh8tbf0JchaxOr+NC+oGhj37WR3kG+TXvTQDN7KmF35NM1Fuo3R0XkbIXG2Wezoj2+oNwtGqgT4LdNzqSg\\\\\\\/BqvuEvWVTQVf8PHus+9JFdwHEwMmdiqX5md2EaWrWt3DLVEnx1NOlT\\\\\\\/CJY1m7NUvL1iNyWEbn\\\\\\\/cjgaEk8cyUkJXjg+u0kDa9oBmShvBprUE8WpsXjwLS7ID2uVrnoMxVRfoev+svZuh1\\\\\\\/nA2OLX2wkE+cyH4C8P3HVYaxWTxqsj5RRlwCUNpswtLhk5hDPvquQUe79fSZdzkzZJH+FzcNTrMQETahLboF3Dw8VdWRNXECopaTaP\\\\\\\/+t26X4DtcvCr3AcKXriSOhOyaDCNK0leEmoG0QnnoGNi5Lq3Sfbn3gakUfjZ1sw0\\\\\\\/aMAfGEjLnvuStLrSgS5rECBTzOCe6AGzJoFPrYcC9ySGJrWvks\\\\\\\/Bj5NtgdOIpSOPRrpJU\\\\\\\/hWuqv9naM1YZfEetB39E2cdNahkJ3zwKLwKmWvMJM6VjOzCjle8wBxyyxuTFv0\\\\\\\/bNbguoyekefspGnNRQ4HnDfp8erDRRRg38S4i9\\\\\\\/k0uaM+e4tRkfnD3Q3b4a75D17GNAy6RylNrs5nwFB9OuG8a7FjgHYJwO9ZVe19uGlbMgyXkOvUUL62IGWGHBpN554wm1hQYbyQMdcWIhO58JaPyoK1NNY8xz25NsXR0Y7XTAlmaPLp\\\\\\\/m1Bwp5eMlhUOiRe9skj\\\\\\\/PoxfAsw5d80Id2Uj4zg7QkHqXf94i5yExmMQoIkq+MzcspEFnIHGLkwYTSiZ3Jqej++SIRkJwEn1E4WetrmEH4Wc2TsstRdJa+sg2ROU1NMq88drnk\\\\\\\/eU+cPWbP3PACRGBWry1l39OsvTJLBDZYcIsEy6Vrgc0XKPJO5XDmIdbk0TCF\\\\\\\/hAFJ3JlyGmH3Hc5hkE\\\\\\\/vbxf5WTcN\\\\\\\/MdS8C4RuNxtLJV4TuEdxI2E6i0fxLl8T1B1iBUxJ9fu\\\\\\\/32EEXS5tUimq2JoUHp8lu6PwEv1fQalaRj9rkx0veufkIzgxhjA1vf\\\\\\\/VdANMiiY94O7acHceUdnSH6mexzJgskD59lLtw8qV2ukAi4KI7GQOGH66WlQa\\\\\\\/opubLQfMRjYR3FfPYoOGh7WTtK8f5FeSy4SfiacN5qP9J7ADS\\\\\\\/0IMDxvZylJo2IUtMVBn32Jv7PbIpsao2CucQzJ6SO1hiidN5WPSQjfZmusQRVAAjCZrrGBrALhgO1BETOo6IhIPkdOG73kXFKjd20x07BvgcTQYov2jFjjh9GgMRRhgyZgIgCLhCAiYw9FqX8Tj0eseoI3y1RrQOZ8nL6Is1imPcdp9rdUhMj6k0d9GtlJlLO\\\\\\\/qt5w05zC38FVxcKuUa1LFNeJ9yctCUUkisLSncdl7r5XgQftkneudvFRRxVtZCodCHKTLKTTtWKtBOSxkoqZuaJ+ytsdbkN8ToTvSDfdmnnb0iJfKIR0L0pvhrdPRBolvQpfuYQq1G9FropDcEZobn9DIlOo2B6wDjV9\\\\\\\/m7gqdgQU35y39RD+iEUn6\\\\\\\/gKr08dtoK77DJUo3N9eqFfoJDChviOGVS3r7RVELRqsDiUxp\\\\\\\/huQyJm5HhbJW4f80Rif0wcJeK8zY6lz3bzKF+oxcLo0mmxID1K5J7LaefzfCUXS83O05UrAJtHgw7jOHnyKkY1if5eBzpiSsPeuRP\\\\\\\/1fF0Rn\\\\\\\/yTPsAQKn+dmEqQ+xH6wf29y0pq7WEZLGMNJQkgWxHL5oo8UH1Hyi7BDVvnecPa2cjHHAti85SH63CknMHLaPZ3wmYpfuqqSaWDx9CfIvoDRDon5zwvT9idQvGrPUCVZ\\\\\\\/hlpZnso71zLr\\\\\\\/DLl\\\\\\\/x6Gmf7MvRG4b2kLo6ylk6OvzzLirBWx87qESZabNqfthyKPyAsrTcLfYz65qjwIJc+S7yVF70BZRkhkrRNDjlQWflAaFvSNji9BsKZIpRXr87HfuIRljnFuD1JGhgKK7zYuLpEtQVpQB1vIa+Tc6mifEA9sS+Tud8xohPPU4Wu4gdMAmVglii90jRaEvrLOtN68JiI0xa\\\\\\\/8etKQzRcsElY9nHzc\\\\\\\/6LATZ62Z+aCvDFpEAsUYnF7k7laVnCHuDnv6iRmTWvZ6Rx9m5SLmrhTZOAfuyL\\\\\\\/TbMwDInghTXpAVxCAHhcPVVnsxhHhF+DCCAE5bmAZV12caiB+jEM2RFCO5Y\\\\\\\/Rg2Uo\\\\\\\/7sB81rA\\\\\\\/w67+rIppNbvJz67Be39IQwM2kipcviamU+k3He76JowjaozZB6Expxiy1MLHvW53Hv1qnkrgA0bVFnnFIHObWLBA8qvfYAJ2siOKSxgsuAM3IS0cgJU6gAMQEDVVr3lyor4\\\\\\\/uGNKBqGA7LStZWKFyrjZnB9Rrm\\\\\\\/IhkkSmOKFxW0NVYF\\\\\\\/QNUilGePHBUzRmIiSFQ4ThRxqlcEz+dZ+nvYLKgWoIw6hPoOjCWAMt7PokMB5aibx0EwvHK9v4ZgxRpfWyr5FlquG14Ytyn6SI6knRpyW\\\\\\\/h3Qxp7l9IwoKDdYbrCOrXhKFBVscTNOZNAit3PKBjSYL7i87XD9ljkUpsCe9glgqEpxaKEHFMrwHjkhmbRnUQJkZtjlT0Eb8z1hLMfSSyk1544TgZA7JmGE6Eqyx5vsBuMm2+h0MGZc1ThMcEwwOTsJzLs\\\\\\\/yjZuZXVaBa0dW4gyv+3wnQM5Widv5slAhRB75ypw1SCfSaDWuUnbZqti3ZTwQPA9awi9pQylG44U4GaBy2Op+fdEvbmGHJmF9ZM3ZPjUbek6ZWZ4yh1lnqLKJ1XyggPO8QvRYJ+aLZ0qn0g3ud2cr3Bug\\\\\\\/3MiVgBeabQeCSUQUlaSFRKGMWkgE3pHu0rgXOU5e69HzPGXt4Wf76VI9lWdsCbCqr4OP\\\\\\\/u599bXo13woJhDAMBzl8iEaoSbrB87s0n89WeGqbglIADLpOil5kuLsV96wCOpZqGHwXr8uahhuZ5XTHT7uvOJOiSlNGUqjr8mtdUCWK2MCijKeKg\\\\\\\/GPRigvpdJVXVwoUm2KEJltoFDCaza4s0zXKAM3ISLDS4XscnnRDN0wgGb37dd3UPRsBHJQdCcD9Z0ipT4b3Iq4pvjnCgebtX7E9Q9P\\\\\\\/NZTrk2fuE8pZR4\\\\\\\/KaMDI81TryqaMImvN9Ri5DW\\\\\\\/A65AjZfVmX4krUqes06yVn2J\\\\\\\/34BfmrABtA\\\\\\\/vG9Uvcj3NWNZLdcoBjXQTGDQJTOJQ8SyqLCEKt\\\\\\\/8S0nvQVhwDyO4A+t1kkx98E0NKD7nuD4BTKbi8B9A\\\\\\\/xKHjR3P4jRQvxaVzBQqA1u8MSCLH4Isr4gTYZ6G6MrY5dHsbBC1hBxwHchy1SzS0JlaFXMXgHRK0lrWZDDgJ7Ca103832ttHYTXOFVv2RkSMc85XuEXw14DCZojtDFXdEGf\\\\\\\/zV+aN3vy9A4tBx2HHKIdM71HxustLsYeB7HDN7YD51r6a4Pg\\\\\\\/+7oXhhW5SRQjyWKs5G5qFl4+qM1xakP8TSf3aGB\\\\\\\/2MbIM3zhVmV\\\\\\\/cJMyeWclRo5CFIdl2ddmqIa9VO2owo1nJ7KZmbkAozK2kDem1YvRfTgMWxOqhRwlP4WvRME3pj+AoL5aGYSMkyQddfoAZoec9BnNMHbEInxUNa62cznTUAmmH\\\\\\\/eO4sULwoUp8FqljhhM4G2g9DEdCOJsrcYq6KHEQ+gRoOG20GGYNYLug\\\\\\\/w+9Eq5kYYOyc7WgrPxPTalRODIm90N5j8\\\\\\\/6h0DpP0x423x2J2CwqCnJMPjNttXu0iO9tFsHriEDjaoB5++FxAXzv5OVGJWW81rRsfOzioVciBYpDWnJto1zsaMw4seSe8rv+kZzH1vnnsz7+8D98EHjXFfM5lMBqcn550EgxzgF2qlAP3LNSYF8dQUCWP98HbQwt+xqWjioaYO9EzeAKEL\\\\\\\/3niehzE8YwPuzzkfUx+FemEWtir1r9OXJ+azYsVGfNaXyG2c2pha+Okh+YWlHQfSTxlJzf\\\\\\\/9QzATBnsqHe2+CiRJf1af63yRMza+cBLlAiQLhHDCwBOWlLmKyHZA7TDOeDFdvtm6Rq\\\\\\\/AqQh9vzeMg0X4gCqLTb6oWUh7JXZbmUYNQVZK\\\\\\\/f1E0iA71GqJMNlw55k7Oi1yeiv3bmisASDv580S52a8v9kb2HktQqRF8Dj4wFzCb17G301ZfDjK6VIyuTsxkKCKedRjtr5sxIVpykDxPdbbvKf5H3Xi4WdghulRpoiVpmxfcMeNX3D\\\\\\\/8GtyQeSyxIa491uABfJc3GO6V6G7ihNEWEC6PHzqbIMYRNPLDiPWDdECcgNRPciVDEBpZCe8Y544A6dseil9\\\\\\\/Cf3APwh3pk6gLu21nLLOvpDtP2zsLlqODBGIAE+qEm7b5z8ic6HZ2lPE18zNxoJ6b\\\\\\\/aXF5WO287Pir9hFgKGFnnHy9Mw11XM42EzUNMSkVM+3t++DgjdPcA\\\\\\\/n5Yr9xoUJxBEKvP8sCY7HhcTOoGpquPYdLQ48CvnVoe7bDsZdkIBHqSunSPWl22MiceAg8wyW4kzNFdsUJumGhvrYQ0\\\\\\\/rR48QXC8jBb9UJxIDxd8SZYG0z1Csh9zH+yYovrDcUQeFHl5p1\\\\\\\/iOXKVkm1G1F6PO+HUyt8+agNe8Pk1SsIfTMkcCpBBHO2KrptKM72y\\\\\\\/UDCElC9krSIB+QaMEJt4X8SSh98G6Fd0NUbQfL64OY4wlHtqKB6pjjfp357A1DJzUIe+XJzG26+JlYBY3iSdqmv7U5bF4s3hMWTHCmZFd01Zjyq86jrsPc50cgalq7JY5\\\\\\\/BnDMqwNZJ45PC\\\\\\\/ayjWbdKL36u0XSQz9cL4pe4XU3AkBudgjFtRWMxOKAw05ZtJSqJTPZXCuWMKx90b3X+sWDlNV8ZU8zP3C5CwIn7nQDJQUKUoAIt5F1nzPUxjPVsbCZmqMQplaqSZzpBZMih\\\\\\\/zzcFUBQv9MQrj+AP2p\\\\\\\/124rdObV418wL2WvbeDVI8qmn3k5CmgcnkCS3qOPBHLzvTifmCj0XK2q9Qf2JWdulypCMy07t4\\\\\\\/0QSqB5h8iOPKX7b81EQBkwlNbx9nMqS2JBAyQiDF2oQmdJcvhFC9p7mMnkHJZyZVwlt0KLP5rNLnGJ2LyEKEzxte01Mm0sIJ\\\\\\\/kR1LSuaeU8EcHEaeLLuEywJZLnlPFe\\\\\\\/VWo6YM33EC65UG79gx\\\\\\\/jCGeVcdPjvzZkZ6uWI0vn2kRx7v9mgFm6zVnevezIfiin+7zqaTANufbg\\\\\\\/mmf0rY1lVJub5GQSbDFn9X\\\\\\\/7pUXjxGSWQDXGZQvYJ\\\\\\\/X8LUanYHSSJpSGJW8QIuXE+XmiR2vGhm+diLKgf6EHTGKsHilIsdpU6Qs3X6wudV2eemvOwpTMKWBIhfEiUG5Rgz7\\\\\\\/nAN8x2KqR\\\\\\\/4YAsuoQF3uP9L9my2nq5Cd88IZj+8Kr3wN8rM4pwOIno4jtSELY5bMiWvi4JeY+kdso8tUVyIK8PfLd4le4ezLEuqfhyruqGlya9Ig2jin4qF9KgS7xzLb37Mu08tm10AQZVOicoE9pLfdf4I+ycG\\\\\\\/PQQ3F4Js7LuMFoNNKgIftZVbHHBYWE+7Hs\\\\\\\/U\\\\\\\/bNFaKKqNqWepVYset17j7k9hfpJE4PmwLaYZmSKR8vr9\\\\\\\/EZ+Xwnmomr5s5YJBxsbNwyAZZp2PcZ4tzJK35FrgRhjG+Nlr001lI7dkQ8Yovjet+3RTrlyd3jYTesA05+CshBiUhdWChXECP57v2wsAJk8w3M8B7Gg0yn\\\\\\\/9tbhoHYNO6BNqqL3TL7xqe5okZxYpC+GGVgpDMndp5pjxBgkfa+v0B9i+h2BcEiq0R1\\\\\\\/GIjoDlbRo+Uysu7HtTAYyYRjKFu5JOwCCr6AkJZOWcYWkK\\\\\\\/Hyot7uf9qEqX3W1jLL16pSodo0qWfgV6rHNNupm3CP7ciMJ8Z0M7aMb6XKw2gq6RGGkT19+CFaEsbBsMslVkx+\\\\\\\/5b0\\\\\\\/t\\\\\\\/ic8lxiIflQ7FYySM880TaHEu0w3oqV\\\\\\\/+aP+STC3XIni3MJgFiJjv\\\\\\\/RfVCqD\\\\\\\/7pcndO5msHlQAj7m77mYc7AaYyHpLGX6qBvt4FTVKlRuArdPTBK9cj7QenVzTEqwWgwUDWD+CyzsXDaJO7UGRhlD\\\\\\\/eaKL2CR3QjG4FqqvihoheC1+pi3zVl6244Tf9ACta6D7x6gMBWNM5d61Ht4WGPDBC\\\\\\\/O6HVB4N6vTzE9ZE+TCEh0YUVjF1krdq4baYeiBGuTKv04E97TtV9kyPnaTbh3DXFvNQcwCFO70KZXn9YPhU\\\\\\\/6nGBJLSsYKeLCojgvKgvstcI6EtiFyOSteRaZCEZ\\\\\\\/nqO1f0NK2jhHtwz6L2cpODMrwgx\\\\\\\/ESoTBqyV\\\\\\\/j3i\\\\\\\/bXPn3rLBlTlEB6adGdeFk7i15meDLRsHbcN5LNHE5JtwI4g8SjxXXBslkt3vvVRgygEEKyUc6PZLi8S+yBOo2kZtmyrH6SkMyOa6DpJ62324u6T7iAZ5Xo2wwINSWUJRJUCkjvb2If\\\\\\\/3m0UNGXxGjjO4v1Y9vbFbLwTXHT2+6VVCY\\\\\\\/St9\\\\\\\/SFFg7vKkTfCXqBqcGR0YR\\\\\\\/LreFBhkCd6WqRzWwWBbNHbvXB7ylchIyMMjBIb\\\\\\\/C2Jsg0b7kKidf+WVYJDBHrQdpSyjMybHj4emV0FxbB61Kd2JZQun32SY5oxN10WpLEWWdQUXPMuJ25uCpjzkVB0V\\\\\\\/gERV0NRj\\\\\\\/Hux8gPMZW+mRagOT6FTyHQdzyLCSyd0nJKp17cZUNMrzbZD0IUUb67BQecUNKdDSEaO8BdLCQ0U80yNnbxEgiFb+DJFXcpdAoF\\\\\\\/yHmmLT0rgCQe\\\\\\\/CndleYTQdZ8xIJsRWIBIWsrBD979\\\\\\\/o\\\\\\\/HhoRE9GgkdvpMf1RbzhII3+7lO4d7dFjvMBXz29DTfLWM7rN45GPLHlXuPH44M92mcriosb+U2RwIYkTTcgaRbaOPXkCRKC1ugAxgD9eUN8IGZNWG2NMvC9jrDKsp\\\\\\\/nx9lf\\\\\\\/KI8J81SldIwx2a1g0yAcLwTrSygJ+1KmfH7f9kDm6h+QPGZ\\\\\\\/d+PGmxnQsP6u8wD5QN300qnMMzkJ5U9\\\\\\\/kEE+Yi7f0\\\\\\\/rCOyN5EV2Uw3oy3sy4OMM78BXG1vrxoOa2nK9AYAf4WpnU6Sgqc+VZPByaWTc1wxbafpJgtig2i79TMmphoqz\\\\\\\/q+t6Op\\\\\\\/ED3bgvDAvwrPzRjm\\\\\\\/h4NBzL0Y46og4BMcX9r9\\\\\\\/psblCIFLdJRXwE3lIFfZr3pPz6uHCkZH6mJR7j4nlxIuaN7UGXQ3dOIXSkkpkgy\\\\\\\/jR9wsoDTYj7LyRqAOpTPab0w67Yf9Rz7oYepB6jBTZrH4o4i+SFnmu1r6w1AGYKmQC6Z9uG1CTE5dkdLMqSZOVAp\\\\\\\/FnsdNNxh2I51uBSYmpezIQQUz0q\\\\\\\/fJtKphYPPSLepId+XOMZsKccdl8zljDyLms8vJCpPJEIcNDT8imcMzKi0K0xyVEGuOZ8CiZWUhQYCono1hiaHqs5fQfvNgpsrvMZNs5E1ZS6DEfOyAq5A55vucwjTEYmJuBIpJTxEqvidBsYwouYPW0CrU\\\\\\\/VOkt8f0TQoeN\\\\\\\/M8LoFfDKT7WJZxmeNGjRT4h58j+WxWQjzeHVX2qlD4kCS7PQvubyOVr0OL4RxKjqx4ntnGNmB8urVALcdJ2j3e2Qt+e2gCDRyxcjJR7uXVfLwkDM\\\\\\\/gwNB9BkH4E7zWXTYSBRrg4uqT2zUr09OppD4qDUB3IhbvrD87uOymvkEAxwxWR6xirz4VHomDnl5FsmOMhVItfoNtbAhJSbFbmEkyrGaAH2Zsn5KAgC+FPrEFHogzAtv46BvfqYyMawmPxZ6UgTQHmrVlza5rhaVJZp5+Vc6lgOVReI2I75aCRM7YndeZDujIVuUjYwkF85\\\\\\\/BquzzvG8VvbtHsWprgnsL5XK9kX0V9FmupeXvs6eN7d3X++ACL4jZfptRJDP9ihiVEXI44Nw1S2FRMze8nXfUYqHlsb+3imqlrvhraGVF98RSpjC5lA\\\\\\\/+HlR5zSNbsqRCGlEj0U32lwAD8Q6aGQAuh1yO6cQUDqTQmMTuqrjA\\\\\\\/xsgW1dPeKpLOhctLnm+BYMaouOZo1YBTi5bhHIw\\\\\\\/iNYB\\\\\\\/oiR0Xqa9T9iKvCCLYnJJDTNMcv36p2zK3tZZeT24reHDDmkwbuOQgt2McjTKI+lwa1vYVUEe8XeIG59MTualnULJYbIUDL1M1JOo9woma8xvO\\\\\\\/jOENOHrLgutai\\\\\\\/xp+LDh0p9yr1KxVqinunj71\\\\\\\/dvoZTsaVvEeaK8aaXx0kYA0q7N8V1Ay4B96Td6a10Ajs5DpQprRanKAXRZYv2YbeMAmckb\\\\\\\/hdcyLfK67Y5Rue\\\\\\\/11oL6YnKCdjsinjrq66CF9T0SOuQX8h6csR4SWQuOvKI\\\\\\\/w8vrAP9RyAVJFIfDIcOf5LjsbIfrAYVvHrohaiQDi\\\\\\\/deHjiIsNUqZqhGDJvNnMNTbz02W5HK6Fj6FmDJT2ACYzaGjNl8d3+5LZEu96FPdipubqhp9QWyXZMEY50205nFoB5nlcPNI6TfH17toeOybsdDAKzBUBmQOnstztcFN45tOfyIf9zdHe+hGoMn6x\\\\\\\/QviCe0Wb9szOvdTR41p5\\\\\\\/ohxnC0kxoMCSzD95siwf5rBF+nfMeLdEzquOHysSNPPO6FFLgC2xOdeTdXQIDc6XVgevxl6pH\\\\\\\/3Za9vknjFhaUxNjT1c\\\\\\\/CPhYiXZNGLoXKc+I2BJmvgmSPW6XAS4U3iFPWMA1WyH8i7pfgR30wgvsMLjpRpJNXuZYwP\\\\\\\/ga6J1O9i6bm2Os4+7hZgqLNm2gvsG\\\\\\\/1TegjUuDK9X3+86U2qO8QgpdkObTPm2iSSZosPvIIgOtVs2t5vDeItprB8LWVnKoEmx1y1rhM7twn+RxMDGvmAExtzo3JjaBDXKKfK2m4uazK2j57J8fcop+pRPDSgEYKMxemuzok9C9hbwGnGbXCiLzThcjr1ws8+nbnZ\\\\\\\/E\\\\\\\/UmQPzxpF3pOTghfhj+BUZVB2czMZeXRgK9WFOdLtkVnRODna1PhayNV79p5WtLZPGtxwJt+5JKKJJhM23XP2C38wBlDz95kqTsEnf+nprGzLNLEYLw0MkmgHPJrNrrZfhRDnNgVkjJnvWGovaqWS8hGTzFU4yD1reN4xFgHVXU5OdnPsK592A2PNv7F+qnCB27G8Ykxcb9wlD6J21iyAwg50RkSF7rJoaQEq8D7CwlxmlGjrqWKLmuVJXqPdQOZ5uwWu8gLzLKH6FbjeE1F\\\\\\\/xp8uYz6J9kvDNTpH6GjQ\\\\\\\/L7It+fphHr06My9+tYbFdKH+Dw8m+6m\\\\\\\/E5hHQM0GwY+YZxDxp1fryeZye\\\\\\\/vuSspXLVps\\\\\\\/I3xq6HZEz5w4IsFQj4J40nqGbJxsefBDaPqg5ueqvNCOL3zLnI8BPKFZMdSSRbQi2u2dbQIoeumdhZfn47IArWLCavA4v7VstYrdeF67qmwbdb+9r49OSOWHoxY6YETz+lC7bYlh0tiAo7ltFXwGJdOAFB6Jt0uQSDPwi+z+sq9BymeaT81znD7q\\\\\\\/pPnmw3wbnMngSD8xVRJln3JTICu\\\\\\\/KIycooR+leN\\\\\\\/QnXtCzy1gY5yJBJ4bI34eyu0vz06b\\\\\\\/ny4KuyHsn6Mk\\\\\\\/EPCiOU1QHnp\\\\\\\/EpB4+zHzOLi3SqZYhmZwjnODUk4Rdv2MPj9zxAoICF5IdStdVqpTvp\\\\\\\/L2itT\\\\\\\/i+3NmeOM9a1VTW7sG7c4DjeQRvaUEcBxVth28j096tBdDE9kw3mkCOgvzTOZu1rbQD8pPJDuQmX5k00NAR4FVcsfv6pwhTKnkDjg3M3uC4hjQZLxbRDJ1S4M5eTjnwvMg+z6ptv0ap1Ld6KU+9Y6MVgMpM8\\\\\\\/4EafmS5lt4NHrArBQcDRdqRPaCaXe9VyaYEofuSYJu8ROgQEgqqDeoZ1o1VJXVfglWJorTst+1mXpNgpGNvorZXLSqNqkwjpVqj0YVqTfgBESlNaXychu8\\\\\\\/iYy4CNtApvdX+gm9Rz3uCqPfTI7q8LAhIt8kKaaAolH6A5DXrb7JCzVuqqG5tqbWiLO9F85LG7oEbr8OLsZU7O1JB3tB4sTvUViQZ97+W+Plp8x\\\\\\\/c0IjYVQReY9o1gleeyFBqiV30tyjPtE3KRSz8AV0a9IypGlgct1QPajS5J38AfDBR3S4Xri+H7PjUmT9OmUi4PEOxGlzOWaVuayeUldyLrzgvkaRLEHuC9jt4srUfV8zIVWRP5JA5HwkH7idK0de3X\\\\\\\/IDwSMBsnJAGTCPb+ksGdBGj2T0UNzud0gLMUTlj1AU9X7IME0q\\\\\\\/iPy9Nzqk7K0KboBIf0twZRk5IuUtab74I7VoW+m4+9\\\\\\\/d83nc7uNdLQnaMKkWDBemSXb1I\\\\\\\/atlTRZABqA6HQMZTgkIY0t87HHQgLb1xqFr1GSbNTGuYYHR6hB+SzPY7UxrX\\\\\\\/35axRs7v9kdFiO+9tCFUNaL\\\\\\\/F4eOZRZbh6tOMHJx9Y63tiE6a657pqKJn1PQ9AT2yRkQvV0wlSkj+T7DlaosoLKD5xEO3c7OesLJOYDJzt2CiOKswsExeDiptshNOCaWnTzkMCHtCGP2rRbYmvM9q6AivA5vNju8eTJi7uBZ\\\\\\\/VILxPlT1HrX4pofXhMuUYWsCI+BL0SVQkSRaUjyIKLOAlQIYMNXDzrT8+3Msn\\\\\\\/kknsvU0uLOKYxXZDmDSJTXRF0n\\\\\\\/CWC\\\\\\\/iNnQcK+ff3CWRU6j5EtwSiisiNu1Z2YYYXgPvhmdF02v9SeeREqmomt0uRSn6dShLODqLXUTtCjgYliZDrxqPVR8rtgO1Daf9Od3lNYkTVCn61aa3rp8ipBuGmHDPFC6xzT59fFpGVyUE8B0QbiTCiqgliVehA1pe\\\\\\\/slDhShsLW4Fr\\\\\\\/oSXPD94Gyvp4SS6nccNmUsCquFjyKScYk9yHVJmjK4bBamkE0N2NiNiY9eMRYBQba\\\\\\\/ie5vG+7OvxnuWbMUGUaraMbSnRm7l5Y+Ym6vdP1NugG2HUsFGAx9YGij6egkYW1ALVK5zg+xmesqEvb8OK7IP2+osO94B84eQwC2xaIm6FhdGRTObTsz7qo7Wjj4wAY2BibhE\\\\\\\/y6j0BAw7KVhpuXUW9+9UOP8KWqvNmB8Oz4Y+ls9HNDm0MSQqoIKJbMsPO5VXiA1PFfvFmWxCuB0SGPPJ8NhBvJw6RjRbolYSYi2l1RlulIVBeIqHhMTtZb5wsHEEGblYlYA3Y5RuU6vYOpXxW98WON0Ys9qum\\\\\\\/9R+leEx84Z4hlXiSa+2AWicNeA6CmwqkvcR1xRK+ikdGkUYgr+9EYEgZCGxzA0Q7W\\\\\\\/mu0AP2llIox8OZMGYHwMUoqkiN8qkSgr562zirUISDA1ADVV7Bkf8YhjVbXomXlE2\\\u0022,\\\u0022iv\\\u0022:\\\u0022h4n6mBsLc0g0AbkHcTjL5g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f93e01c9c254e225de3321d11e0e0376d5442d6e20c1e069db7998010d5aefe2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225b8oB5oeLJKmeMsXkRkct4GLHj8IOSjCoFOwSpKVBAT1TFdjpoP\\\\\\\/m7jTfRqE+J0322IFnNVv8ZD6RBCg4m4MD91xB5oocMeae87I9wzx2u9fCudg79esjcbAjERNpRpOgCrubEkQ+NO0YX6lgrrSEFw5Bw2BJWzN\\\\\\\/+FfiIABmjjM6vJznoNyVVu2f6RRLk1thQIuof0B97lS5ekz6mLRWNIvK3HqeDogpV7DisxCr3aUnXbvmXp2JnK70TjsmZomNrn0k7bsPbqGsIbB6k9bMximVGwGY\\\\\\\/rS2yw3Ht+9s1CXQpCYhS3CHNFLmSusIDgHDMSz5WZ1jz70nMqwWot3aSpHH9DysxXrYvdszq+aenCRNxyB3Q7RABiofq+VibTMEDDgRIyAyDH\\\\\\\/z\\\\\\\/xN60N\\\\\\\/D9Fad3hGkZw4lNjtawVXRaMMJGEOHDUen5qOCovqN5nBhL\\\\\\\/LbZcrK7zQQPY4XVYcXFs70KJhVKca5BXOBTQhf90h6lbOssHa9efBgxMG75B2N4ZxC66LtE6fDjrVkGPEjHp0okXH5U1uT8nPXcjOskphL2ufGrmICcI2g8inhYv8\\\\\\\/D0LEXo0dLTONmfqZt\\\\\\\/FVtvKRIKhipKQx5BqBAQV11h8KN8OkwALP3byKdv6Vj+foVfUc9+Q5bdVnCZ4UJm8+65qgWKUtYwZ4kgxznuCA4i1pmsRsPfhNz8D6mis17hJPf0R1PEUBRi2jO+CANSNGc75qZzABpeMg77a99cfE2z0nnqVuZqrjLl9UPKSHlU5MSFHBFMU\\\\\\\/fDDo8\\\\\\\/+8G7AJP63+TLsAqOTbmt7aBLesilqZR4KIWvjZQunU5vfVyabL7yTC+TznAkznfZ4A8ltUf6cLfthoo\\\\\\\/OMQ3txNdkJwLNrwbaSFZ2EU+FUjokqpVjdXFiJV6heZ7pCimUPBakuszmSUpcsUdbqnRwCFdbIvVCkxxUgL\\\\\\\/o9P4K8I6a1o2vAhFjCi5P7XfJ4yXzg+o2yM0cUHKkQ+Bjq2ERmcNn8WTfbOFm9csSXmggFomv7jLMkkSqgAGdVzjx37AGeXL1mPdA+gClx7gGcgOJL24v2YyHyMlCHm7Ln21LmQ94ioTCvuP4QFD0wmFCIbTifdEIACfE4OhjwykKZCtayklZ9d0rOSg3LZvC3znEB5agrGy65zvLPIb1ixuWunJA5XLxXmNGaQzq1ieKwTVXKFINVOPdrFT\\\\\\\/v+560WWgxRh22FC48YBBGluzCECtgiDCrkDVRfhRFTIH+f5uWFdlKwWrIey4NALBMtNBEjtu2A9pN7hCTXVuTgjQdcaQC5RPvHPoXR4S8+UGBCF4x0xRCwl2Bka7U\\\\\\\/dWxsCtBNiBjXd4CMFWwVxFYhPJfwd5I563V1gNauCgus\\\\\\\/jdKxNkSuYZiUe6JljoxRIP4JsVDcWPB8wHJl+hnQGB74afdRjSmHzwr8DfH6gXHDSvCtkNVdQaLQHNStWhbminf2Rv8amZSYH9Sm0Yo6yS7d1jYJwUt6jJqk7XBBHMyi+Xvbd4WAMDlx7yzn39GN+ijffGIEHJ6dOMUApoZ\\\\\\\/cMBunF6EFpkzS0MHPk4AL\\\\\\\/ewH6NRvwDK1OtryCl0+5YuMJb5tOVWq024sYwJIC+kL7bZJEMUGhAvZsOrdq25vDMK00v4Gc4NNZs8ah+OxGvuxjDzXI8yU1Av5QF+KK\\\\\\\/q6\\\\\\\/vgUVvrEXehbkqZq+QUH\\\\\\\/az9pOVwrGiJyzmVN0i7N39RamPKvxOUOxuofguu0viGmgN2IInLmvAizcMR3LH3+tAZRl8SpAz8rmeT8qDyabvPrO7gccTY2GUBeEL3q+TZO28HE0HWEWNtdD5K6PHMN8m3FS7q6rAeKU95DsLxR7Z+YP0jFmS0sSyl1\\\\\\\/3\\\\\\\/QX\\\\\\\/e3xWVuaXPLJUletxFm0eeS1r7lOi1Xx8RfIVd9TUNKhne4Kdo44f6Ah0SHJPlnRcarh9j52rEBd71jy9h9+xNobfEXX2klgfXXUq8EhuoFYRmq1zMZdHb9Ai4FCYaxTNMXS2cFsnJt2nylY+dkLWBqvmWifJsnLx0doJ2FGWe7MT\\\\\\\/yAG\\\\\\\/pGRuyPsKqRKz\\\u0022,\\\u0022iv\\\u0022:\\\u00220Cp5ZfXgX7kbZuF80dKFTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277e7263f6f5ea5c5c0ccda63e2a56c1bc0117d860fa7c3b3b95f0fb315803d22\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460032 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2187" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-x96F-NQHvtMdVaeml0YXtOwmMWpzYjsAkjWkQvCZgco" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2187" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Jn2YHLY4zlV7AQ3-lch8fL6PyI1u-TDRoMundouYhH4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2177" title="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2199" title="Selection Against Maternal microRNA Target Sites in Maternal Transcripts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460032" id="node460032" data-pisa="ggg;5/10/2187" data-pisa-master="ggg;g3.115.021493" data-apath="/ggg/5/10/2187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marion Dubarry</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">A. Peter Banks</span>, <span class="highwire-citation-author" data-delta="3">Simon Cockell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2187-2197;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460032" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marion Dubarry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marion%2BDubarry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubarry%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarion%2BDubarry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">A. Peter Banks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>High Throughput Screening Facility, Newcastle Biomedicine, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BPeter%2BBanks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Banks%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BPeter%2BBanks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Cockell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Bioinformatics Support Unit, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BCockell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BCockell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">david.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2187" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460032" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2187.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460032" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2187.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460032" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2187.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460032" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2187.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460032" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460032/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Three major DNA polymerases replicate the linear eukaryotic chromosomes. DNA polymerase α-primase (Pol α) and DNA polymerase δ (Pol δ) replicate the lagging-strand and Pol α and DNA polymerase ε (Pol ε) the leading-strand. To identify factors affecting coordination of DNA replication, we have performed genome-wide quantitative fitness analyses of budding yeast cells containing defective polymerases. We combined temperature-sensitive mutations affecting the three replicative polymerases, Pol α, Pol δ, and Pol ε with genome-wide collections of null and reduced function mutations. We identify large numbers of genetic interactions that inform about the roles that specific genes play to help Pol α, Pol δ, and Pol ε function. Surprisingly, the overlap between the genetic networks affecting the three DNA polymerases does not represent the majority of the genetic interactions identified. Instead our data support a model for division of labor between the different DNA polymerases during DNA replication. For example, our genetic interaction data are consistent with biochemical data showing that Pol ε is more important to the Pre-Loading complex than either Pol α or Pol δ. We also observed distinct patterns of genetic interactions between leading- and lagging-strand DNA polymerases, with particular genes being important for coupling proliferating cell nuclear antigen loading/unloading (<a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>, <a href="http://www.yeastgenome.org/locus/S000005670/overview">Elg1</a>) with nucleosome assembly (chromatin assembly factor 1, histone regulatory HIR complex). Overall our data reveal specialized genetic networks that affect different aspects of leading- and lagging-strand DNA replication. To help others to engage with these data we have generated two novel, interactive visualization tools, DIXY and Profilyzer.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-replication" class="hw-term hw-article-keyword hw-article-keyword-dna-replication" rel="nofollow">DNA replication</a></li><li class="kwd"><a href="/keyword/dna-polymerase" class="hw-term hw-article-keyword hw-article-keyword-dna-polymerase" rel="nofollow">DNA polymerase</a></li><li class="kwd"><a href="/keyword/profilyzer" class="hw-term hw-article-keyword hw-article-keyword-profilyzer" rel="nofollow">Profilyzer</a></li><li class="kwd"><a href="/keyword/dixy" class="hw-term hw-article-keyword hw-article-keyword-dixy" rel="nofollow">DIXY</a></li><li class="kwd"><a href="/keyword/quantitative-fitness-analyses-qfa" class="hw-term hw-article-keyword hw-article-keyword-quantitative-fitness-analyses-qfa" rel="nofollow">quantitative fitness analyses (QFA)</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li></ul><p id="p-3">DNA replication requires the coordinated action of numerous proteins and nucleic acids to ensure accurate genome duplication. At the core of the replicative machinery in eukaryotes are three B-family DNA polymerases. The three polymerases, Pol α, Pol δ, and Pol ε, work in concert with many other accessory proteins. For example, the six-subunit complex, the origin recognition complex, binds at replication origins throughout the cell cycle and recruits replication proteins to initiate replication. The origin recognition complex helps recruit the hexameric helicase MCM in G1, along with <a href="http://www.yeastgenome.org/locus/S000004093/overview">Cdc45</a> and GINS (CMG for Cdc45-MCM-GINS) in S-phase to initiate replication and unwind DNA at the front of the replication fork. The five-polypeptide complex, replication factor C (RFC), the sliding clamp loader, loads the trimeric sliding clamp, proliferating cell nuclear antigen (<a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a>), onto DNA. <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> helps tether polymerases to DNA to improve processivity during strand elongation (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Waga and Stillman 1998</a>).</p><p id="p-4">DNA polymerase α-primase (Pol α) initiates replication of both leading and lagging strands (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Georgescu <em>et al.</em> 2015</a>). To perform this task, Pol α synthesizes a short RNA primer that is extended first by Pol α DNA polymerase activity before other DNA polymerases take over. The most widely accepted current view is that DNA polymerase δ (Pol δ) completes lagging-strand synthesis, whereas DNA polymerase ε (Pol ε) completes leading-strand synthesis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Garg and Burgers 2005</a>). An alternative model is that Pol δ is the major polymerase for both leading and lagging strands and Pol ε is a repair polymerase (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Johnson <em>et al.</em> 2015</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Waga and Stillman 1994</a>). Pol α has limited processivity and lacks intrinsic 3′ exonuclease activity for proofreading functions (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kunkel <em>et al.</em> 1989</a>). The current evidence indicates that Pol α synthesizes about 1.5% of the genome (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Reijns <em>et al.</em> 2015</a>). In budding yeast, all four subunits of Pol α, <a href="http://www.yeastgenome.org/locus/S000001447/overview">Pri1</a>, <a href="http://www.yeastgenome.org/locus/S000001528/overview">Pri2</a>, <a href="http://www.yeastgenome.org/locus/S000005046/overview">Pol1</a>, and <a href="http://www.yeastgenome.org/locus/S000000131/overview">Pol12</a>, are essential for cell viability.</p><p id="p-5">Pol δ principally completes synthesis of the lagging strand. In budding yeast Pol δ consists of three subunits: <a href="http://www.yeastgenome.org/locus/S000002260/overview">Pol3</a>, <a href="http://www.yeastgenome.org/locus/S000003766/overview">Pol31</a>, and <a href="http://www.yeastgenome.org/locus/S000003804/overview">Pol32</a>. <a href="http://www.yeastgenome.org/locus/S000002260/overview">Pol3</a> is the catalytic subunit, and it has an associated 3′ exonuclease activity that can remove misincorporated nucleotides. <a href="http://www.yeastgenome.org/locus/S000003766/overview">Pol31</a>, like <a href="http://www.yeastgenome.org/locus/S000002260/overview">Pol3</a>, is essential in budding yeast. <a href="http://www.yeastgenome.org/locus/S000003804/overview">Pol32</a> is a nonessential subunit of Pol δ that help stabilize the other two subunits. The subunits of Pol δ play additional roles since, for example, <a href="http://www.yeastgenome.org/locus/S000003766/overview">Pol31</a> and <a href="http://www.yeastgenome.org/locus/S000003804/overview">Pol32</a> also physically interact with <a href="http://www.yeastgenome.org/locus/S000006088/overview">Rev3</a> and by this criterion are subunits of the error-prone DNA polymerase zeta (Pol ζ), which catalyses synthesis opposite DNA lesions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kochenova <em>et al.</em> 2015</a>).</p><p id="p-6">Pol ε principally completes synthesis of the leading-strand. Pol ε is a four-subunit complex consisting of <a href="http://www.yeastgenome.org/locus/S000005206/overview">Pol2</a>, <a href="http://www.yeastgenome.org/locus/S000006379/overview">Dpb2</a>, <a href="http://www.yeastgenome.org/locus/S000000482/overview">Dpb3</a>, and <a href="http://www.yeastgenome.org/locus/S000002528/overview">Dpb4</a>. <a href="http://www.yeastgenome.org/locus/S000005206/overview">Pol2</a>, the catalytic subunit, and <a href="http://www.yeastgenome.org/locus/S000006379/overview">Dpb2</a> are essential proteins in yeast, whereas <a href="http://www.yeastgenome.org/locus/S000000482/overview">Dpb3</a> and <a href="http://www.yeastgenome.org/locus/S000002528/overview">Dpb4</a> can be deleted. <a href="http://www.yeastgenome.org/locus/S000005206/overview">Pol2</a> DNA polymerase and 3′ exonuclease activities are located in the N-terminal domain while its C-terminal domain carries conserved zinc-finger motifs (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/021493SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS1.pdf">Figure S1</a>). Interestingly, the C-terminus of <a href="http://www.yeastgenome.org/locus/S000005206/overview">Pol2</a> is essential for yeast viability, whereas the N-terminal polymerase domain of <a href="http://www.yeastgenome.org/locus/S000005206/overview">Pol2</a> is not (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dua <em>et al.</em> 1999</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feng and D’Urso 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kesti <em>et al.</em> 1999</a>). The current model to explain the fact that yeast cells can survive in the absence of the leading-strand DNA polymerase function is that the lagging-strand DNA polymerase Pol δ can, if necessary, replace Pol ε polymerase to replicate the leading strand (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Daigaku <em>et al.</em> 2015</a>). Additionally, Pol δ can replace Pol ε to proofread errors created by defective Pol ε and can complete DNA synthesis on the leading strand (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Flood <em>et al.</em> 2015</a>). It is thought that the most important (essential) function of Pol ε is to help initiate DNA replication as part of the preloading complex (pre-LC), working in concert with the GINS, <a href="http://www.yeastgenome.org/locus/S000001591/overview">Sld2</a>, and <a href="http://www.yeastgenome.org/locus/S000003626/overview">Dpb11</a> proteins (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sengupta <em>et al.</em> 2013</a>). <a href="http://www.yeastgenome.org/locus/S000000482/overview">Dpb3</a> and <a href="http://www.yeastgenome.org/locus/S000002528/overview">Dpb4</a>, the nonessential subunits affect Pol ε binding to DNA and its processivity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aksenova <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Araki <em>et al.</em> 1991</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tsubota <em>et al.</em> 2003</a>).</p><p id="p-7">In eukaryotes, three multi-subunit polymerases interact to complete DNA replication, whereas in bacteria a single DNA polymerase is sufficient. The reasons for this difference are unclear but may reflect that fact that in eukaryotes, unlike prokaryotes, numerous origins are used to replicate each chromosome and eukaryotic chromosomes are linear, bringing an additional “end replication problem.” At each eukaryotic replication origin, a pair of replisomes travels in opposite directions from the origin (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Masai <em>et al.</em> 2010</a>) and will encounter replisomes coming in the opposite direction as well as other obstacles such as nucleosomes, DNA barrier elements, transcription factors, transcribing RNA polymerases, and damaged DNA, as well as chromosome ends (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Tourriere and Pasero 2007</a>). Accordingly, a large number of mechanisms have evolved to help the DNA replication machinery to deal with such challenges (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yeeles <em>et al.</em> 2013</a>). Interestingly, it seems that distinct mechanisms have evolved to regulate the stability of leading- and lagging-strand machinery (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kurth and O’Donnell 2013</a>).</p><p id="p-8">Genetic suppressor and enhancer screens are a powerful way to explore how complex biochemical processes work <em>in vivo</em> and in particular how processes are controlled and interact with other cellular components or pathways. We have previously performed suppressor and enhancer analysis of telomere defective strains to identify pathways that are important for maintaining functional chromosome ends in budding yeast (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addinall <em>et al.</em> 2008</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2011</a>). Here, we applied quantitative fitness analysis (QFA) to identify factors that affect fitness of strains defective in DNA replication. We identify numerous genes that affect general as well as specific aspects of eukaryotic DNA replication. To help others engage with this resource we also provide web-based tools that permit interaction with these and similar data sets.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Creation of the double-mutant libraries by synthetic genetic array (SGA)</h3><p id="p-9">Genotypes of the strains used in this study are described in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS1.pdf">Table S1</a>. The thermosensitive (ts) alleles have been resequenced and mutations are shown in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS1.pdf">Figure S1</a>. The SGA technique (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tong and Boone 2006</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tong <em>et al.</em> 2001</a>) was used to combine the yeast genome knock-out (<em>yfgΔ</em>) and DAmP [decreased abundance by mRNA perturbation, which affects the function of essential genes (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schuldiner <em>et al.</em> 2005</a>), (<em>yfg-d</em>)] collections with the <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005046/overview">pol1</a>-4</em>, <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12</em>, <em><a href="http://www.yeastgenome.org/locus/S000002260/overview">cdc2</a>-2</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> mutations. The recessive, ts-mutated alleles from (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2011b</a>) were flanked by the selectable <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and HphMX markers in strains bearing <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em>STE2pr-SpHis5</em> and <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a></em>::NatMX. Diploids were selected on rich media containing geneticin, hygromycin B, and nourseothricin. Diploids were sporulated on solid medium, and meiotic haploid <em>MAT</em><strong>a</strong> double-mutant progeny were isolated on SC medium containing canavanine, thialysine, geneticin, hygromycin B, and nourseothricin while lacking leucine, lysine, histidine, and arginine. Selection was based on haploid selection markers (<em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a></em>::<em>STE2pr-SpHis5</em>), double-mutant selection markers (KanMX, HphMX, NatMX, and <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>), and thialysine (<em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a></em>::NatMX). Solid agar to solid agar pinning was performed on a Biomatrix BM3-SC robot (S&amp;P Robotics Inc., Toronto, Canada) at 20° as previously described in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> (2011)</a>.</p></div><div id="sec-3" class="subsection"><h3>Spotting yeast cultures for QFA</h3><p id="p-10">Double mutants obtained from SGA were inoculated into liquid (final media: SC-ARG, −HIS, −LYS, −LEU, +canavanine, +thialysine, +geneticin, +hygromycin, and +nourseothricin) at 20° in 96-wells plates and grown to saturation. Cultures were spotted on solid agar-plates in 384-format using a Biomek FX robot [Beckman Coulter (UK) Limited, High Wycombe, UK], and growth assays carried out as in <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> (2011)</a>, except we used concentrated cultures, without diluting in water, for <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a>Δ</em> (control), <em><a href="http://www.yeastgenome.org/locus/S000005046/overview">pol1</a>-4</em>, <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002260/overview">cdc2</a>-2</em> experiments. Plates were imaged automatically, usually every 4 hr. For each genotype, we performed four independent genetic crosses and analyzed two growth curves from each cross (eight growth curves in total). In some cases, however, we examined more than eight cultures (<em>i.e.</em>, up to 144 for <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>Δ</em>, repeated across several plates) and for others less than eight (<em>i.e.</em>, down to four because of to technical errors). The specific number of growth curves analyzed for each double mutant can be found in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-4" class="subsection"><h3>QFA fitness estimates</h3><p id="p-11">Cell-density estimates are generated from time-course photographs using Colonyzer2 [(<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lawless <em>et al.</em> 2010</a>), <a href="http://research.ncl.ac.uk/colonyzer/">http://research.ncl.ac.uk/colonyzer/</a>]. Strain fitness was quantified by first fitting the generalized logistic model to growth curves:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-1.gif"/></noscript></span></span>where <em>t</em> is time since inoculation, <em>x</em> is cell density, <em>r</em> is a growth rate parameter, <em>K</em> is carrying capacity, and <em>ν</em> is a parameter controlling symmetry of the growth curve. This differential equation has an analytical solution:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-2.gif"/></noscript></span></span>where <em>x</em><sub>0</sub> is the cell density at inoculation (assumed constant for all spots on all plates). We derive expressions for maximum doubling rate (MDR) and maximum doubling potential (MDP) for this generalized logistic model, equivalent to those presented by <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> (2011)</a> for the logistic model [see qfa R package (qfa_0.0-39, <a href="http://qfa.r-forge.r-project.org/">http://qfa.r-forge.r-project.org/</a>) for details]:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-3.gif"/></noscript></span></span>Following <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> (2011)</a>, we define fitness as the product of MDR and MDP.</p></div><div id="sec-5" class="subsection"><h3>QFA genetic interaction strength estimates</h3><p id="p-12">To estimate the strength and significance of genetic interaction with a query background (<em>xyz</em>), we compared the fitness of each strain in our collection (<em>yfgΔ-d</em>) in that background (<em>xyz yfgΔ</em>) with the equivalent fitness in a control background (<em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a>Δ yfgΔ-d)</em> under the same temperature and on the same growth media. We first fit a regression, equivalent to Fisher’s multiplicative model of genetic independence [see (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>) for details], to mean fitnesses across the collection, using the lm function in R:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-4.gif"/></noscript></span></span>where square brackets denote fitness and tildes denote average (mean) values for each strain. Assuming that most strains in the collection do not interact with the query background, estimating the slope <em>m</em> allows us to predict the fitnesses of individual strains in the query background, given fitness observed in the control background:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-5.gif"/></noscript></span></span>If the mean of predicted fitnesses for a strain is significantly different from the mean of observed fitnesses for that strain that suggests that the strain interacts with the query background. We carried out unpaired, two-tailed tests of the null hypothesis that <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-6.gif"/></noscript></span></span> is equal to <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-7.gif"/></noscript></span></span> using the t.test function in R. We corrected for multiple testing by applying a false discovery rate correction to the vector of <em>P</em>-values for the strain collection using the p.adjust function in R. This gave us a vector of corrected q-values. We defined interactions with q &lt; 0.05 as significant. We defined genetic interaction strength as the difference between the mean of the observed fitnesses and the mean of the predicted fitnesses.</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2187/embed/mml-math-8.gif"/></noscript></span></span></div><div id="sec-6" class="subsection"><h3>Stripping genes from QFA analysis</h3><p id="p-13">Genes within 20 kb of query mutations were stripped from analysis because of genetic linkage (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Tong and Boone 2006</a>). In addition, strains that did not complete SGA, such as genes required for mating and the biosynthesis of histidine, lysine, arginine, and leucine, were excluded as well as genes that failed the transition between SGA and QFA (probably due to prolonged storage on plates at 4°). Lists of genes stripped in each screen are available in the <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FileS2.xlsx">File S2</a> and in the GISReports folder on the following Web site: <a href="https://github.com/lwlss/profilyzer_Dubarry2015">https://github.com/lwlss/profilyzer_Dubarry2015</a>.</p></div><div id="sec-7" class="subsection"><h3>Gene ontology (GO) terms analysis</h3><p id="p-14">A list of enriched biological processes and biological components was downloaded from the Gene Ontology enRIchement anaLysis and visuaLizAtion tool (GOrilla: <a href="http://cbl-gorilla.cs.technion.ac.il/">http://cbl-gorilla.cs.technion.ac.il/</a>) using a p-value threshold of 10<sup>−3</sup> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Eden <em>et al.</em> 2009</a>). Significant negative or positive genetic interactions for each screen were compared with all genes screened as a background gene set to identify enriched GO terms. The lists were filtered to include only terms that annotated between two and 250 genes. Final lists were ranked by “enrichment” score (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS4.pdf">Table S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS5.pdf">Table S5</a>) and filtered to include only the smallest terms of any branch ranked at the eighth level, or below, of the GO hierarchy (see <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS3.pdf">Figure S3</a>). p-value is not corrected for multiple testing. The q-value is adjusted to control for the false discovery rate (&lt; 0.05) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Klipper-Aurbach <em>et al.</em> 1995</a>). The dates when enrichments were assessed are indicated in each table.</p></div><div id="sec-8" class="subsection"><h3>Dynamic interactive X-Y (DIXY) plots</h3><p id="p-15">DIXY is an online data visualization tool for browsing QFA fitness plots, which show evidence for genetic interaction by comparing fitnesses of strains in a query screen with fitnesses in a control screen (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). QFA screen data presented in this paper along with data from (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> 2013</a>) can be visualized here: <a href="http://bsu-srv.ncl.ac.uk/dixy-pol/viz/">http://bsu-srv.ncl.ac.uk/dixy-pol/viz/</a>. Note that the DAmP alleles are missing from the data obtained in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> (2013)</a>. The DIXY Web site includes documentation to help users interact with plots. DIXY uses the Python web-framework Django (<a href="https://www.djangoproject.com/">https://www.djangoproject.com/</a>) to serve dynamic web pages powered by the javascript libraries D3 (<a href="http://d3js.org/">http://d3js.org/</a>) and jQuery (<a href="http://jquery.com/">http://jquery.com/</a>). The tool requires genetic interaction strength (*GIS.txt) files as input, as generated by the R package for QFA: <a href="http://qfa.r-forge.r-project.org/">http://qfa.r-forge.r-project.org/</a>.</p></div><div id="sec-9" class="subsection"><h3>Profilyzer</h3><p id="p-16">Profilyzer is an online tool for visualizing and interrogating the results of multiple QFA screens at once. QFA screen data presented in this paper, along with data from <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> (2013)</a>, can be visualized here: <a href="http://research.ncl.ac.uk/qfa/Dubarry2015/">http://research.ncl.ac.uk/qfa/Dubarry2015/</a>. Note that the DAmP alleles are missing from the data obtained in <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> (2013)</a>. The Profilyzer Web site includes documentation to help users interact with plots. Profilyzer is built using the Shiny framework (<a href="http://shiny.rstudio.com/">http://shiny.rstudio.com/</a>) as a wrapper around a set of custom-built R functions. Source code and data underlying this instance can be found on GitHub: <a href="https://github.com/lwlss/profilyzer_Dubarry2015">https://github.com/lwlss/profilyzer_Dubarry2015</a>.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-17">All strains and materials are available upon request. <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FileS2.xlsx">File S2</a> contain the raw data from the screens performed in this study. These information are also available through GitHub: <a href="https://github.com/lwlss/profilyzer_Dubarry2015">https://github.com/lwlss/profilyzer_Dubarry2015</a> with description of the fitness estimates (generated by the qfa R package <a href="http://qfa.r-forge.r-project.org/">http://qfa.r-forge.r-project.org/</a>) and description of the Profilyzer tool. Source code for the DIXY visualization tool can be found at <a href="https://github.com/Bioinformatics-Support-Unit/dixy">https://github.com/Bioinformatics-Support-Unit/dixy</a>. <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS1.pdf">Table S1</a> contains the list of strains used in the study. <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS2.pdf">Table S2</a> summarizes the positive and negative interactions identified (q &lt; 0.005). <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS5.pdf">Table S5</a> show enriched gene ontology terms for the DNA polymerase mutants. <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS6.pdf">Table S6</a> provides a description of all the negative interactions affecting the DNA polymerase mutants.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-18">We performed QFA to uncover genetic interactions affecting the fitness of strains containing hypomorphic, ts alleles affecting each of the catalytic subunits of the three major replicative polymerases. We examined <em><a href="http://www.yeastgenome.org/locus/S000005046/overview">pol1</a>-4</em> affecting Pol α function, <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12</em> affecting Pol ε function (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Budd and Campbell 1993</a>), and <em><a href="http://www.yeastgenome.org/locus/S000002260/overview">cdc2</a>-2</em> affecting Pol δ function (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hartwell <em>et al.</em> 1973</a>) (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-19">First, at permissive temperature, we crossed polymerase-defective strains to a genome-wide loss of function library using the SGA method (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Tong <em>et al.</em> 2001</a>). The library contained nonessential gene disruptions (<em>yfgΔ</em>, replaced with the KanMX marker) and DAmP alleles affecting essential genes [<em>yfg-d</em>, DAmP, which can partially reduce mRNA stability by inserting the <em>KanMX marker</em> into the 3′UTR of essential genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Breslow <em>et al.</em> 2008</a>)]. We indicate the combined library as <em>yfgΔ-d</em>. Fitness of strains was deduced from growth curves of cultures spotted on solid agar. Individual cultures were monitored by time course photography and curves fitted to the data allowing us to estimate MDR and MDP for each (see <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS2.pdf">Figure S2</a> for a small set of examples). Generally, for each genotype, eight curves were analyzed. Fitness was defined as the product of the MDR and MDP, as before (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). We performed QFA at temperatures that reduced fitness of strains with ts alleles to approximately 40–60% of wild-type fitness. In total, approximately 230,000 growth curves were generated (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FileS1.xlsx">File S1</a>).</p><div id="sec-12" class="subsection"><h3>DNA polymerase screens</h3><p id="p-20">Pol α-primase initiates DNA replication of both leading and lagging strands. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the effects of about 4800 library mutations combined with either a Pol α mutation (<em><a href="http://www.yeastgenome.org/locus/S000005046/overview">pol1</a>-4</em>; <em>y</em>-axis) or a control mutation (<em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a>Δ</em>, <em>x</em>-axis). It is clear that at 33°, <em><a href="http://www.yeastgenome.org/locus/S000005046/overview">pol1</a>-4 yfgΔ-d</em> double-mutants grew on average more poorly than <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">lyp1</a>Δ yfgΔ-d</em> mutants. We observed 127 double-mutants that grew significantly more poorly than expected (inverted red triangles), reflecting negative genetic interactions with the Pol α mutation. 26 positive genetic interactions (blue upward triangles) also were identified (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS2.pdf">Table S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic interactions that affect DNA polymerase functions. (A) Fitnesses of yeast strains with yfg&#x394;-d mutations in pol1-4 or lyp1&#x394; genetic backgrounds and cultured at 33&#xB0;. Each symbol shows the effect of a single yfg&#x394;-d in the two contexts, and the total number of yfg&#x394;-d genes shown is indicated by the number in top left corner of the plot. The solid gray line is a linear regression through all points, and the dashed line is the line of equal fitness (if there were no fitness cost caused by pol1-4 at 33&#xB0;). Statistically significant positive genetic interactions with pol1-4 are represented by blue triangles, negative genetic interactions by red inverted triangles, and the remaining observations by gray dots. The numbers of significant interactions are indicated by the colored numbers across the top of the plot. (B) Same as in (A) but for DNA polymerase &#x3B4; (Pol &#x3B4;) mutation (cdc2-2) at 30&#xB0;. (C) Same as in (A) but for the DNA polymerase &#x3B5; (Pol &#x3B5;) mutation (pol2-12) at 36&#xB0;. (D) A ranked list of enriched biological process Gene Ontology (GO) terms across the negative genetic interactions identified for pol1-4. The complete list of enriched terms GO terms and classifications are found in Table S3. Black dots correspond to the common terms found enriched in the DNA polymerase &#x3B1;-primase (Pol &#x3B1;), Pol &#x3B4;, and Pol &#x3B5; screens. The asterisk indicates that the GO term has been abbreviated from Table S3. (E) Same as in (D) but for Pol &#x3B4; mutation (cdc2-2) at 30&#xB0;. See Table S4 for the complete list of enriched terms GO terms. (F) Same as in (D) but for the Pol &#x3B5; mutation (pol2-12) at 36&#xB0;. See Table S5 for the complete list of enriched terms GO terms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-282554492" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic interactions that affect DNA polymerase functions. (A) Fitnesses of yeast strains with &lt;em&gt;yfg&#x394;-d&lt;/em&gt; mutations in &lt;em&gt;pol1-4&lt;/em&gt; or &lt;em&gt;lyp1&#x394;&lt;/em&gt; genetic backgrounds and cultured at 33&#xB0;. Each symbol shows the effect of a single &lt;em&gt;yfg&#x394;-d&lt;/em&gt; in the two contexts, and the total number of &lt;em&gt;yfg&#x394;-d&lt;/em&gt; genes shown is indicated by the number in top left corner of the plot. The solid gray line is a linear regression through all points, and the dashed line is the line of equal fitness (if there were no fitness cost caused by &lt;em&gt;pol1-4&lt;/em&gt; at 33&#xB0;). Statistically significant positive genetic interactions with &lt;em&gt;pol1-4&lt;/em&gt; are represented by blue triangles, negative genetic interactions by red inverted triangles, and the remaining observations by gray dots. The numbers of significant interactions are indicated by the colored numbers across the top of the plot. (B) Same as in (A) but for DNA polymerase &#x3B4; (Pol &#x3B4;) mutation (&lt;em&gt;cdc2-2&lt;/em&gt;) at 30&#xB0;. (C) Same as in (A) but for the DNA polymerase &#x3B5; (Pol &#x3B5;) mutation (&lt;em&gt;pol2-12&lt;/em&gt;) at 36&#xB0;. (D) A ranked list of enriched biological process Gene Ontology (GO) terms across the negative genetic interactions identified for &lt;em&gt;pol1-4&lt;/em&gt;. The complete list of enriched terms GO terms and classifications are found in Table S3. Black dots correspond to the common terms found enriched in the DNA polymerase &#x3B1;-primase (Pol &#x3B1;), Pol &#x3B4;, and Pol &#x3B5; screens. The asterisk indicates that the GO term has been abbreviated from Table S3. (E) Same as in (D) but for Pol &#x3B4; mutation (&lt;em&gt;cdc2-2&lt;/em&gt;) at 30&#xB0;. See Table S4 for the complete list of enriched terms GO terms. (F) Same as in (D) but for the Pol &#x3B5; mutation (&lt;em&gt;pol2-12&lt;/em&gt;) at 36&#xB0;. See Table S5 for the complete list of enriched terms GO terms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2187/F1.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2187/F1.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466760" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Genetic interactions that affect DNA polymerase functions. (A) Fitnesses of yeast strains with <em>yfgΔ-d</em> mutations in <em>pol1-4</em> or <em>lyp1Δ</em> genetic backgrounds and cultured at 33°. Each symbol shows the effect of a single <em>yfgΔ-d</em> in the two contexts, and the total number of <em>yfgΔ-d</em> genes shown is indicated by the number in top left corner of the plot. The solid gray line is a linear regression through all points, and the dashed line is the line of equal fitness (if there were no fitness cost caused by <em>pol1-4</em> at 33°). Statistically significant positive genetic interactions with <em>pol1-4</em> are represented by blue triangles, negative genetic interactions by red inverted triangles, and the remaining observations by gray dots. The numbers of significant interactions are indicated by the colored numbers across the top of the plot. (B) Same as in (A) but for DNA polymerase δ (Pol δ) mutation (<em>cdc2-2</em>) at 30°. (C) Same as in (A) but for the DNA polymerase ε (Pol ε) mutation (<em>pol2-12</em>) at 36°. (D) A ranked list of enriched biological process Gene Ontology (GO) terms across the negative genetic interactions identified for <em>pol1-4</em>. The complete list of enriched terms GO terms and classifications are found in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS3.pdf">Table S3</a>. Black dots correspond to the common terms found enriched in the DNA polymerase α-primase (Pol α), Pol δ, and Pol ε screens. The asterisk indicates that the GO term has been abbreviated from <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS3.pdf">Table S3</a>. (E) Same as in (D) but for Pol δ mutation (<em>cdc2-2</em>) at 30°. See <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS4.pdf">Table S4</a> for the complete list of enriched terms GO terms. (F) Same as in (D) but for the Pol ε mutation (<em>pol2-12</em>) at 36°. See <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS5.pdf">Table S5</a> for the complete list of enriched terms GO terms.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We next examined the results obtained with the Pol δ mutation. Pol δ primarily contributes to lagging-strand synthesis but is sometimes engaged in leading-strand synthesis (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Daigaku <em>et al.</em> 2015</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Flood <em>et al.</em> 2015</a>). Interestingly we only identified 39 statistically significant negative genetic interactions with the <em><a href="http://www.yeastgenome.org/locus/S000002260/overview">cdc2</a>-2</em> mutation and no positive interactions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS2.pdf">Table S2</a>).</p><p id="p-23">Finally, we focused on Pol ε, which is the major leading strand DNA polymerase and in addition plays an important role in the assembly of the CMG helicase at origins of replication (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Muramatsu <em>et al.</em> 2010</a>). With <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12</em>, we saw the largest number of genetic interactions, identifying 134 positive interactions and 185 negative interactions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS2.pdf">Table S2</a>).</p><p id="p-24">We interrogated the biological significance of the genetic interactions identified in the three screens by searching for enriched GO terms in the positive and negative interactions (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, D−F</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS3.pdf">Figure S3</a>). Negative genetic interactions gave enriched GO terms for Pol α, Pol δ, and Pol ε mutants, whereas positive genetic interactions only gave enriched GO terms with Pol ε mutants (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS4.pdf">Table S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS5.pdf">Table S5</a>). The two enriched GO terms found from <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12 yfgΔ-d</em>−positive interactions are related to cytoplasmic translation and ubiquitin-dependent protein catabolic process. The proteolytic and ubiquitin systems clearly contribute to DNA replication, but the many potential mechanisms of action remain to be fully understood (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Finley <em>et al.</em> 2012</a>).</p><p id="p-25">The enriched GO terms found among the negative interactions with the different DNA polymerase mutants were enriched for numerous pathways functioning in DNA replication, DNA damage checkpoint regulation, and various DNA repair pathways (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, D−F</a>). Interestingly, only two of 47 GO terms: the intra-S DNA damage checkpoint and telomere maintenance via recombination, were found in all three Pol α, Pol δ, and Pol ε screens. We conclude that defects in any of three DNA polymerases causes cells to rely on genes important for DNA checkpoint pathways and telomere integrity. However, the more specific nature of the most of the genetic interactions we have uncovered suggest that polymerase-specific interactions allow us to distinguish the division of labor between the three replicative DNA polymerases. If so, these data are a valuable resource to help to understand the numerous, complex, biochemical interactions combine together to result in efficient DNA replication.</p></div><div id="sec-13" class="subsection"><h3>DIXY plots: an interactive data visualization tool</h3><p id="p-26">To better facilitate the appreciation of and interaction with these data and to compare the effects of specific genes in the context of different polymerase mutations, we have developed an online data visualization tool called DIXY (<a href="http://bsu-srv.ncl.ac.uk/dixy-pol">http://bsu-srv.ncl.ac.uk/dixy-pol</a>). DIXY connects a series of fitness plots like the three shown in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>. It is possible to highlight genes based on their location in one plot, their name, GO function, or presence in known complexes/pathways and determine where the chosen genes lie across the fitness plots. To help highlight different genetic interactions, the points (<em>yfgΔ-d</em> mutations) move positions as the user switches between datasets (<em>i.e.</em>, all points in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a> move to different positions in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, B or C</a>).</p><p id="p-27">With DIXY, it is possible to compare the effects of mutations across datasets to identify factors that affect DNA replication in different ways. For example the <em><a href="http://www.yeastgenome.org/locus/S000001073/overview">rrm3</a>Δ</em> mutation, which encodes a DNA helicase that helps replisome progression in some regions of the genome (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ivessa <em>et al.</em> 2002</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2003</a>), seems to cause particularly poor growth in the Pol ε mutants but less so in Pol α or Pol δ mutants (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS4.pdf">Figure S4</a>). In contrast, the <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a>Δ</em> mutation, which affects a different DNA helicase of the RecQ family (homologous to human BLM) that is involved in the suppression of illegitimate recombination (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Myung <em>et al.</em> 2001</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Watt <em>et al.</em> 1996</a>), shows the opposite phenotype. Both <a href="http://www.yeastgenome.org/locus/S000001073/overview">Rrm3</a> and <a href="http://www.yeastgenome.org/locus/S000004802/overview">Sgs1</a> limit genome instability but this pattern shows that clear differences in the roles played by these helicases during regulation of DNA replication. Such differences in genetic interactions are informative about DNA replication processes and can be explored using DIXY.</p></div><div id="sec-14" class="subsection"><h3>Specificity of genetic interactions affecting DNA replication</h3><p id="p-28">To explore the differences between the genetic networks affecting the three DNA polymerase mutants, we analyzed the distribution of specific negative genetic interactions over the three screens (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The interactions can be considered in the context of a current view of the eukaryotic DNA replication fork (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Only 18 gene deletions/DAmP mutations reduced fitness of all three DNA polymerase mutants (area I, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS6.pdf">Table S6</a>). This small number is consistent with the fact that only two GO terms, the intra S DNA damage checkpoint and telomere maintenance via recombination, were found in all three screens (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>). Much larger numbers of genes were found in other areas of the plot <em>e.g.</em>, 131 gene deletions specifically interacted with the Pol ε mutation (area IV, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Thus, these data reinforce the view that there are many more genetic interactions that are specific to single polymerases than interactions that affect fitness of strains defective in all three polymerases.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Specificity of genetic interactions. (A) Venn diagram showing overlaps between negative genetic interactions that affect cells with a DNA polymerase &#x3B1;-primase (Pol &#x3B1;) mutation (pol1-4), DNA polymerase &#x3B4; (Pol &#x3B4;) mutation (cdc2-2), or DNA polymerase &#x3B5; (Pol &#x3B5;) mutation (pol2-12) (Table 1 and Table S6). (B) A simplified view of the replication fork based on Sengupta et al. (2013). Pol &#x3B5; binds to origins via Dpb2-GINS interaction. Pol &#x3B5; remains associated with the CMG helicase (Cdc45-MCM2-7-GINS) during the leading-strand synthesis. Pol &#x3B1;-primase is coupled to the CMG helicase via Ctf4 and initiates the DNA synthesis on both strands. Pol &#x3B1; is replaced by Pol &#x3B4; on the lagging-strand to extend the nascent strand. Note: for simplicity, numerous essential replication factors have been omitted (i.e., PCNA, RFC, Dpb11 etc.) (C) Fitness profiles of gene deletions affecting the DNA damage checkpoint (ddc1&#x394;, rad9&#x394;, rad17&#x394;, and rad24&#x394;) in combination with lyp1&#x394; (control), cdc13&#x2010;1, pol1-4, pol2-12, and cdc2-2 mutations and in presence of 100 mM hydroxyurea (HU). Dashed gray boxes represent the fitness range of the double-mutants in each screen. Box plots show 50% range, the whiskers represent 1.5-fold the 50% range from the box, and the horizontal black line is the median fitness. (D) Fitness profiles of gene deletions affecting the Mre11 complex (rad50&#x394; and xrs2&#x394;) as in C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-282554492" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Specificity of genetic interactions. (A) Venn diagram showing overlaps between negative genetic interactions that affect cells with a DNA polymerase &#x3B1;-primase (Pol &#x3B1;) mutation (&lt;em&gt;pol1-4&lt;/em&gt;), DNA polymerase &#x3B4; (Pol &#x3B4;) mutation (&lt;em&gt;cdc2-2&lt;/em&gt;), or DNA polymerase &#x3B5; (Pol &#x3B5;) mutation (&lt;em&gt;pol2-12&lt;/em&gt;) (Table 1 and Table S6). (B) A simplified view of the replication fork based on Sengupta &lt;em&gt;et al.&lt;/em&gt; (2013). Pol &#x3B5; binds to origins via Dpb2-GINS interaction. Pol &#x3B5; remains associated with the CMG helicase (Cdc45-MCM2-7-GINS) during the leading-strand synthesis. Pol &#x3B1;-primase is coupled to the CMG helicase via Ctf4 and initiates the DNA synthesis on both strands. Pol &#x3B1; is replaced by Pol &#x3B4; on the lagging-strand to extend the nascent strand. Note: for simplicity, numerous essential replication factors have been omitted (&lt;em&gt;i.e.&lt;/em&gt;, PCNA, RFC, Dpb11 etc.) (C) Fitness profiles of gene deletions affecting the DNA damage checkpoint (&lt;em&gt;ddc1&#x394;&lt;/em&gt;, &lt;em&gt;rad9&#x394;&lt;/em&gt;, &lt;em&gt;rad17&#x394;&lt;/em&gt;, and &lt;em&gt;rad24&#x394;&lt;/em&gt;) in combination with &lt;em&gt;lyp1&#x394;&lt;/em&gt; (control), &lt;em&gt;cdc13&#x2010;1&lt;/em&gt;, &lt;em&gt;pol1-4&lt;/em&gt;, &lt;em&gt;pol2-12&lt;/em&gt;, and &lt;em&gt;cdc2-2&lt;/em&gt; mutations and in presence of 100 mM hydroxyurea (HU). Dashed gray boxes represent the fitness range of the double-mutants in each screen. Box plots show 50% range, the whiskers represent 1.5-fold the 50% range from the box, and the horizontal black line is the median fitness. (D) Fitness profiles of gene deletions affecting the Mre11 complex (&lt;em&gt;rad50&#x394;&lt;/em&gt; and &lt;em&gt;xrs2&#x394;&lt;/em&gt;) as in C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2187/F2.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2187/F2.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466826" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Specificity of genetic interactions. (A) Venn diagram showing overlaps between negative genetic interactions that affect cells with a DNA polymerase α-primase (Pol α) mutation (<em>pol1-4</em>), DNA polymerase δ (Pol δ) mutation (<em>cdc2-2</em>), or DNA polymerase ε (Pol ε) mutation (<em>pol2-12</em>) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS6.pdf">Table S6</a>). (B) A simplified view of the replication fork based on <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Sengupta <em>et al.</em> (2013)</a>. Pol ε binds to origins via Dpb2-GINS interaction. Pol ε remains associated with the CMG helicase (Cdc45-MCM2-7-GINS) during the leading-strand synthesis. Pol α-primase is coupled to the CMG helicase via Ctf4 and initiates the DNA synthesis on both strands. Pol α is replaced by Pol δ on the lagging-strand to extend the nascent strand. Note: for simplicity, numerous essential replication factors have been omitted (<em>i.e.</em>, PCNA, RFC, Dpb11 etc.) (C) Fitness profiles of gene deletions affecting the DNA damage checkpoint (<em>ddc1Δ</em>, <em>rad9Δ</em>, <em>rad17Δ</em>, and <em>rad24Δ</em>) in combination with <em>lyp1Δ</em> (control), <em>cdc13‐1</em>, <em>pol1-4</em>, <em>pol2-12</em>, and <em>cdc2-2</em> mutations and in presence of 100 mM hydroxyurea (HU). Dashed gray boxes represent the fitness range of the double-mutants in each screen. Box plots show 50% range, the whiskers represent 1.5-fold the 50% range from the box, and the horizontal black line is the median fitness. (D) Fitness profiles of gene deletions affecting the Mre11 complex (<em>rad50Δ</em> and <em>xrs2Δ</em>) as in C).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466773/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466773/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Enhancers affecting defective DNA polymerase strains (q = 0.05)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Profilyzer: a tool for assessing genetic interactions in different contexts</h3><p id="p-31">To better understand the relevance of the genetic interactions in the three different DNA polymerase mutants, we created Profilyzer, an interactive, fitness-profiling web-tool (<a href="http://research.ncl.ac.uk/qfa/Dubarry2015/">http://research.ncl.ac.uk/qfa/Dubarry2015/</a>). This allows us to view the fitness profile of specific mutations across the three DNA polymerase screens and in other contexts. We also assessed the effects mutations in the context of <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em>, causing telomere defects, since “telomere maintenance via recombination” was one of only two GO terms enriched in all three polymerase screens and in the presence of the S-phase poison hydroxyurea (HU) [<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5, A−B</a>, (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> 2013</a>)]. Profilyzer can be used to see fitness profiles of individual genes or groups of genes and to identify genes with similar profiles.</p><p id="p-32">We first assessed the profiles of the 18 gene deletions that negatively affect all three polymerase mutants (area I, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). These mutations affect proteins that are important for sensing and responding to many replication defects and/or contribute to the stability or function of the DNA replication fork (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/TableS6.pdf">Table S6</a>). For example, DNA damage checkpoint mutations, including <em><a href="http://www.yeastgenome.org/locus/S000002625/overview">rad9</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">rad17</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">ddc1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">rad24</a>Δ</em> (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), which were all previously reported to reduce fitness of DNA polymerase mutants (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Weinert 1992</a>), were found in this group. In <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>, we observe that strains with checkpoint mutations (<em><a href="http://www.yeastgenome.org/locus/S000002625/overview">rad9</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">rad17</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">ddc1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">rad24</a>Δ</em>) are close to average fitness in control conditions, very fit when combined with <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em>, but very unfit when combined with all polymerase mutations or grown on HU. In contrast, other genes found in area I, like <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">RAD50</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">XRS2</a></em> (affecting two of three components of the <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> complex), showed a different profile. <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ and <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em> interacted negatively with all three polymerases and HU and also interacted negatively with <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). Thus, Profilyzer allows us to see that although all six genes affect fitness of DNA polymerase mutants similarly, <em><a href="http://www.yeastgenome.org/locus/S000002625/overview">rad9</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">rad17</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">ddc1</a>Δ</em>,and <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">rad24</a>Δ</em> can be distinguished from <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em> by their effects in <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> strains. Such comparisons will be useful for helping to clarify the roles played by specific genes during DNA replication.</p><p id="p-33"><em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em>, along with <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a>Δ</em>, affect the three components of the MRX complex but <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">MRE11</a></em> was not in central area I of <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a>. To investigate the reason why, we generated profiles of all three genes (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5C</a>). Reassuringly it is clear the all three genes behave very similarly, apart from in the control genetic background. Therefore, the reason why <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a>Δ</em> was not classified, like <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em>, as being in area I, seems to be that in the control experiment <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a>Δ</em> strains were unusually sick, probably because of a technical error during the SGA procedure. Thus, Profilyzer complements statistical cut off−based approaches to explore genome wide genetic interactions.</p><p id="p-34">Profilyzer also permits the search for genes that cluster similarly to any particular query mutation. When we used this function to identify the nearest fitness profiles to <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">RAD24</a></em> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5D</a>), we found <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">RAD17</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">DDC1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002625/overview">RAD9</a></em> were the top three fitness profiles like <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">RAD24</a></em>. Furthermore, the ranking and distance of these three genes from <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">RAD24</a></em> reflect the fact that <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">RAD24</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">RAD17</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">DDC1</a></em>, encoding components of the checkpoint sliding clamp and its loader, share many functions that are distinct to <em><a href="http://www.yeastgenome.org/locus/S000002625/overview">RAD9</a></em>, encoding a checkpoint mediator protein (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ngo <em>et al.</em> 2014</a>). Profilyzer can be customized to explore subsets of data. For example, if we search for the closest profiles to <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a>Δ</em> and choose to ignore the control data then <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em> are among the top 12 most similar profiles to <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a>Δ</em>.</p><p id="p-35">The other 12 genes in area I also could be classified based on their fitness in the context of the <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> mutation. Nine gene deletions including <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">rad52</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000002232/overview">bre1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000003804/overview">pol32</a>Δ</em>, behaved liked <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a>Δ</em> mutants and were unfit in all contexts (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5E</a>), whereas <em><a href="http://www.yeastgenome.org/locus/S000002411/overview">rad57</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000003461/overview">smi1</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000006134/overview">lea1</a>Δ</em> differed because they were comparatively fit in combination with <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5F</a>). Profilyzer therefore allows us to identify genes that are important in different contexts, in the specific examples highlighted, in polymerase mutants and telomere defective mutants.</p></div><div id="sec-16" class="subsection"><h3>Genes that affect specific polymerase functions</h3><div id="sec-17" class="subsection"><h4>Regulators that work in concert with Pol <em>δ</em> (area II):</h4><p id="p-36">We next wanted to understand those genes that seemed to affect cells with defects in specific DNA polymerases. We first examined the Pol δ−specific interactors (13 genes, area II, <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). We found that <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000003131/overview">rad54</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002483/overview">rad55</a>Δ</em>, from the <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">RAD52</a></em> epistasis group involved in HR repair, were in this group (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS6.pdf">Figure S6A</a>). Other, similarly functioning HR genes <em><a href="http://www.yeastgenome.org/locus/S000003804/overview">pol32</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">rad52</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002411/overview">rad57</a>Δ</em> were among those genes that affected all three polymerases (area I, discussed previously in <em>Profilyzer: a tool for assessing genetic interactions in different contexts</em>). The fitness profiles of <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000003131/overview">rad54</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002483/overview">rad55</a>Δ</em> are somewhat difficult to distinguish from <em><a href="http://www.yeastgenome.org/locus/S000003804/overview">pol32</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">rad52</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002411/overview">rad57</a>Δ</em> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS6.pdf">Figure S6A</a> <em>vs.</em> <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS5.pdf">Figure S5, E−F</a>). The major difference is that <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000003131/overview">rad54</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002483/overview">rad55</a>Δ</em> are slightly fitter in Pol α and Pol ε mutants. Clearly the boundaries between the areas of the Venn diagram in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a> are indistinct and areas I and II contain genes with similar roles. Other genes in area II of the Venn diagram seemed to affect the fitness of all three polymerase mutants somewhat similarly to those in area I (<em>i.e.</em>, <em><a href="http://www.yeastgenome.org/locus/S000000291/overview">rfc5</a>-d</em>, <em><a href="http://www.yeastgenome.org/locus/S000005945/overview">rmi1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000004730/overview">pkr1</a>Δ</em>). In summary, analysis using Profilyzer shows that most gene deletion/DAmP mutations found to interact with Pol δ alone (area II) also interact with Pol α or Pol ε and therefore, by this criterion, are general regulators of DNA replication.</p><p id="p-37">Among the Pol δ−specific interactors in area II was <em><a href="http://www.yeastgenome.org/locus/S000001447/overview">pri1</a>-d</em>, affecting an essential DNA Pol α-primase subunit. This is curious, because we imagined that <a href="http://www.yeastgenome.org/locus/S000001447/overview">Pri1</a> would be most important in mutants defective in Pol α. Interestingly when we examined the effects of <em><a href="http://www.yeastgenome.org/locus/S000001528/overview">pri2</a>-d</em>, affecting another subunit of Pol α-primase, we saw a similar-shaped profile of genetic interactions across the screens (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS6.pdf">Figure S6B</a>). <a href="http://www.yeastgenome.org/locus/S000001447/overview">Pri1</a> and <a href="http://www.yeastgenome.org/locus/S000001528/overview">Pri2</a> are each essential genes disrupted by DAmP alleles, and it has been estimated that less than half of the DAmP alleles affect gene function (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Breslow <em>et al.</em> 2008</a>). Inefficiency of the DAmP allele could explain why the effects of <em><a href="http://www.yeastgenome.org/locus/S000001528/overview">pri2</a>-d</em> are so mild in all cases. Irrespective, it seems that based on the fitness profiles, <a href="http://www.yeastgenome.org/locus/S000001447/overview">Pri1</a> is particularly important in Pol δ, Pol α, and <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> mutants and less important to Pol ε mutants. This finding suggests, perhaps, that <a href="http://www.yeastgenome.org/locus/S000001447/overview">Pri1</a>, with the help of the accessory protein <a href="http://www.yeastgenome.org/locus/S000001528/overview">Pri2</a>, plays critical roles in the switch from Pol α to Pol δ DNA polymerase activity during lagging strand synthesis and at telomeres.</p></div><div id="sec-18" class="subsection"><h4>Regulators that work in concert with Pol <em>α</em> (area III):</h4><p id="p-38">From area III, we highlight two particularly relevant genes, <em><a href="http://www.yeastgenome.org/locus/S000006339/overview">CTF4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001933/overview">RSC8</a></em>, that specifically interact with Pol α (area III, <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>) in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS6.pdf">Figure S6C</a>. <em><a href="http://www.yeastgenome.org/locus/S000006339/overview">CTF4</a></em> encodes a core component of the replication fork and is important for coupling the CMG helicase with Pol α (see <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2B</a>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gambus <em>et al.</em> 2009</a>). Strong interactions of <em><a href="http://www.yeastgenome.org/locus/S000006339/overview">ctf4</a>Δ</em> with Pol α suggest that the most critical role for <a href="http://www.yeastgenome.org/locus/S000006339/overview">Ctf4</a> is tethering Pol α at the replication fork. Perhaps as expected, <em><a href="http://www.yeastgenome.org/locus/S000006339/overview">ctf4</a>Δ</em> mutants also were unfit when combined with the other polymerase mutations and very unfit when combined with the telomere-defective <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> allele. The <em><a href="http://www.yeastgenome.org/locus/S000001933/overview">rsc8</a>-d</em> mutation causes a defect in an essential subunit of the RSC chromatin remodeling complex (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cairns <em>et al.</em> 1996</a>), suggesting that interactions between Pol α and the RSC complex are critical for coordination of replication with chromatin structure.</p></div><div id="sec-19" class="subsection"><h4>Regulators that work in concert with Pol <em>ε</em> (area IV):</h4><p id="p-39">The largest number of specific genetic interactors was seen with Pol ε (area IV, <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2A</a>). As expected, we found other components of Pol ε: <em><a href="http://www.yeastgenome.org/locus/S000005056/overview">dpb2</a>-d</em> and <em><a href="http://www.yeastgenome.org/locus/S000000482/overview">dpb3</a>Δ</em> (<em><a href="http://www.yeastgenome.org/locus/S000002528/overview">dpb4</a>Δ</em> was not significant but shows a similar profile) in this area (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). It is interesting that defects in any of Pol ε subunits affect Pol ε whereas, in contrast, defects in Pol α-primase subunits affect Pol δ function more (compare <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> with <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS6.pdf">Figure S6B</a>). Pol ε has at least two major functions during DNA replication, one as the leading strand polymerase and two, as part of the pre-LC containing GINS (<a href="http://www.yeastgenome.org/locus/S000002897/overview">Sld5</a>, <a href="http://www.yeastgenome.org/locus/S000002420/overview">Psf1</a>, <a href="http://www.yeastgenome.org/locus/S000003608/overview">Psf2</a>, <a href="http://www.yeastgenome.org/locus/S000005506/overview">Psf3</a>), <a href="http://www.yeastgenome.org/locus/S000001591/overview">Sld2</a>, and <a href="http://www.yeastgenome.org/locus/S000003626/overview">Dpb11</a> that forms off chromatin and is recruited to origins before replication initiation (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Muramatsu <em>et al.</em> 2010</a>). Interestingly <em><a href="http://www.yeastgenome.org/locus/S000002420/overview">psf1</a>-d</em>, <em><a href="http://www.yeastgenome.org/locus/S000003608/overview">psf2</a>-d</em>, and <em><a href="http://www.yeastgenome.org/locus/S000003626/overview">dpb11</a>-d</em> were also Pol ε enhancers (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). These data are consistent with the idea that GINS-Pol ε interactions being crucial for loading of Pol ε to the replisome (in the pre-LC) and/or for maintaining Pol ε at the replication fork after replication initiation (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Langston <em>et al.</em> 2014</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Sengupta <em>et al.</em> 2013</a>) (see <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness profiling of genes affecting DNA polymerase &#x3B5; (Pol &#x3B5;). (A) Fitness profiles of dpb2-d, dpb3&#x394;, dpb4&#x394; mutations as in Figure 2C. (B) Fitness profiles of components of the preloading complex: dpb11-d and psf1-d, psf-2-d (GINS) as in Figure 2C. (C) Fitness profiles of hda1&#x394;, hos2&#x394;, rtx3&#x394;, set3&#x394;, sif2&#x394;, snt1&#x394;, and ume6&#x394; mutations (Rpd3L-expanded complex) as in Figure 2C. Note: Note that the DAmP alleles are missing from the hydroxyurea (HU) experiments (Andrew et al. 2013)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-282554492" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fitness profiling of genes affecting DNA polymerase &#x3B5; (Pol &#x3B5;). (A) Fitness profiles of &lt;em&gt;dpb2-d&lt;/em&gt;, &lt;em&gt;dpb3&#x394;&lt;/em&gt;, &lt;em&gt;dpb4&#x394;&lt;/em&gt; mutations as in Figure 2C. (B) Fitness profiles of components of the preloading complex: &lt;em&gt;dpb11-d&lt;/em&gt; and &lt;em&gt;psf1-d&lt;/em&gt;, &lt;em&gt;psf-2-d&lt;/em&gt; (GINS) as in Figure 2C. (C) Fitness profiles of &lt;em&gt;hda1&#x394;&lt;/em&gt;, &lt;em&gt;hos2&#x394;&lt;/em&gt;, &lt;em&gt;rtx3&#x394;&lt;/em&gt;, &lt;em&gt;set3&#x394;&lt;/em&gt;, &lt;em&gt;sif2&#x394;&lt;/em&gt;, &lt;em&gt;snt1&#x394;&lt;/em&gt;, and &lt;em&gt;ume6&#x394;&lt;/em&gt; mutations (Rpd3L-expanded complex) as in Figure 2C. Note: Note that the DAmP alleles are missing from the hydroxyurea (HU) experiments (Andrew &lt;em&gt;et al.&lt;/em&gt; 2013).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2187/F3.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2187/F3.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466725" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Fitness profiling of genes affecting DNA polymerase ε (Pol ε). (A) Fitness profiles of <em>dpb2-d</em>, <em>dpb3Δ</em>, <em>dpb4Δ</em> mutations as in <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2C</a>. (B) Fitness profiles of components of the preloading complex: <em>dpb11-d</em> and <em>psf1-d</em>, <em>psf-2-d</em> (GINS) as in <a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2C</a>. (C) Fitness profiles of <em>hda1Δ</em>, <em>hos2Δ</em>, <em>rtx3Δ</em>, <em>set3Δ</em>, <em>sif2Δ</em>, <em>snt1Δ</em>, and <em>ume6Δ</em> mutations (Rpd3L-expanded complex) as in <a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2C</a>. Note: Note that the DAmP alleles are missing from the hydroxyurea (HU) experiments (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Andrew <em>et al.</em> 2013</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">DNA replication initiation is influenced by chromatin structure, for example, histone acetylation. Additionally, after DNA replication is completed, epigenetic chromatin marks are maintained on the replicated sister chromatids. Histone deacetylases (HDACs), particularly <a href="http://www.yeastgenome.org/locus/S000005274/overview">Rpd3</a>, delay replication initiation (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Knott <em>et al.</em> 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vogelauer <em>et al.</em> 2002</a>). Interestingly we identified genes affecting seven HDAC components as negatively interacting with <em><a href="http://www.yeastgenome.org/locus/S000005206/overview">pol2</a>-12</em> (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). This was a surprise because we might expect that reduction of HDAC activity would improve the fitness of Pol ε mutants if DNA replication is easier to initiate when chromatin is more acetylated (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Unnikrishnan <em>et al.</em> 2010</a>). Therefore, this somewhat surprising result leads us to hypothesize instead that loss of HDACs, combined with defective Pol ε, leads to failure to properly replicate and maintain heterochromatic regions of the genome. Consistent with this hypothesis, in <em>Schizosaccharomyces pombe</em>, chromatin-modifying complexes travel with Pol ε to control modification of newly replicated chromatin at the fork (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2011a</a>).</p></div><div id="sec-20" class="subsection"><h4>Genes that differently interact with Pol <em>α</em> and Pol <em>ε vs.</em> Pol <em>δ</em> (area V):</h4><p id="p-42">In area V, we found genes affecting protein complexes that modify chromatin, like the INO80-type complex (<em><a href="http://www.yeastgenome.org/locus/S000000894/overview">ies5</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000002160/overview">nhp10</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000000435/overview">swc5</a>Δ</em>) and the <a href="http://www.yeastgenome.org/locus/S000001161/overview">Set1</a>/COMPASS complex (<em><a href="http://www.yeastgenome.org/locus/S000000064/overview">swd1</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000000379/overview">swd3</a>Δ</em>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Both of these complexes have been shown to promote recovery of stalled replication forks (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rizzardi <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shimada <em>et al.</em> 2008</a>) and perhaps, therefore, this is the reason why they were in area V.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness profiling genes associated with DNA polymerase &#x3B1;-primase (Pol &#x3B1;) and DNA polymerase &#x3B5; (Pol &#x3B5;). (A) Fitness profiles of components of the Set1-COMPASS complex: swd1&#x394; and swd3&#x394; and the INO80-type complex: ies5&#x394;, nhp10&#x394; and swc5&#x394; as in Figure 2C. (B) Fitness profiles of pph21&#x394;, rrd1&#x394;, and rrd2&#x394; mutations (Protein Phosphatase 2A complex) as in Figure 2C. (C) Fitness profiles of csm3&#x394; and mrc1&#x394; mutations (S-phase checkpoint) as in Figure 2C. (D) Fitness profiles of ctf18&#x394;, ctf8&#x394;, and dcc1&#x394; mutations (Ctf18-RLC) and elg1&#x394; mutation (Elg1-RLC) as in Figure 2C. (E) List of the top 10 gene deletions/DAmP mutations that show similar profiles to ctf18&#x394; when combined with control (lyp1&#x394;), cdc13&#x2010;1, pol1-4, pol2-12, and cdc2-2 mutations and in presence of 100 mM hydroxyurea (HU). (F) As in (E) but show similar profiles to elg1&#x394; mutation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-282554492" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fitness profiling genes associated with DNA polymerase &#x3B1;-primase (Pol &#x3B1;) and DNA polymerase &#x3B5; (Pol &#x3B5;). (A) Fitness profiles of components of the Set1-COMPASS complex: &lt;em&gt;swd1&#x394;&lt;/em&gt; and &lt;em&gt;swd3&#x394;&lt;/em&gt; and the INO80-type complex: &lt;em&gt;ies5&#x394;&lt;/em&gt;, &lt;em&gt;nhp10&#x394;&lt;/em&gt; and &lt;em&gt;swc5&#x394;&lt;/em&gt; as in Figure 2C. (B) Fitness profiles of &lt;em&gt;pph21&#x394;&lt;/em&gt;, &lt;em&gt;rrd1&#x394;&lt;/em&gt;, and &lt;em&gt;rrd2&#x394;&lt;/em&gt; mutations (Protein Phosphatase 2A complex) as in Figure 2C. (C) Fitness profiles of &lt;em&gt;csm3&#x394;&lt;/em&gt; and &lt;em&gt;mrc1&#x394;&lt;/em&gt; mutations (S-phase checkpoint) as in Figure 2C. (D) Fitness profiles of &lt;em&gt;ctf18&#x394;&lt;/em&gt;, &lt;em&gt;ctf8&#x394;&lt;/em&gt;, and &lt;em&gt;dcc1&#x394;&lt;/em&gt; mutations (Ctf18-RLC) and &lt;em&gt;elg1&#x394;&lt;/em&gt; mutation (Elg1-RLC) as in Figure 2C. (E) List of the top 10 gene deletions/DAmP mutations that show similar profiles to &lt;em&gt;ctf18&#x394;&lt;/em&gt; when combined with control (&lt;em&gt;lyp1&#x394;&lt;/em&gt;), &lt;em&gt;cdc13&#x2010;1&lt;/em&gt;, &lt;em&gt;pol1-4&lt;/em&gt;, &lt;em&gt;pol2-12&lt;/em&gt;, and &lt;em&gt;cdc2-2&lt;/em&gt; mutations and in presence of 100 mM hydroxyurea (HU). (F) As in (E) but show similar profiles to &lt;em&gt;elg1&#x394;&lt;/em&gt; mutation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2187/F4.medium.gif" width="313" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2187/F4.medium.gif" width="313" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2187/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Fitness profiling genes associated with DNA polymerase α-primase (Pol α) and DNA polymerase ε (Pol ε). (A) Fitness profiles of components of the Set1-COMPASS complex: <em>swd1Δ</em> and <em>swd3Δ</em> and the INO80-type complex: <em>ies5Δ</em>, <em>nhp10Δ</em> and <em>swc5Δ</em> as in <a id="xref-fig-2-18" class="xref-fig" href="#F2">Figure 2C</a>. (B) Fitness profiles of <em>pph21Δ</em>, <em>rrd1Δ</em>, and <em>rrd2Δ</em> mutations (Protein Phosphatase 2A complex) as in <a id="xref-fig-2-19" class="xref-fig" href="#F2">Figure 2C</a>. (C) Fitness profiles of <em>csm3Δ</em> and <em>mrc1Δ</em> mutations (S-phase checkpoint) as in <a id="xref-fig-2-20" class="xref-fig" href="#F2">Figure 2C</a>. (D) Fitness profiles of <em>ctf18Δ</em>, <em>ctf8Δ</em>, and <em>dcc1Δ</em> mutations (Ctf18-RLC) and <em>elg1Δ</em> mutation (Elg1-RLC) as in <a id="xref-fig-2-21" class="xref-fig" href="#F2">Figure 2C</a>. (E) List of the top 10 gene deletions/DAmP mutations that show similar profiles to <em>ctf18Δ</em> when combined with control (<em>lyp1Δ</em>), <em>cdc13‐1</em>, <em>pol1-4</em>, <em>pol2-12</em>, and <em>cdc2-2</em> mutations and in presence of 100 mM hydroxyurea (HU). (F) As in (E) but show similar profiles to <em>elg1Δ</em> mutation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In area V we also identified two mutations (<em><a href="http://www.yeastgenome.org/locus/S000006073/overview">rrd2</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002292/overview">pph21</a>Δ</em>) affecting the PP2A complex. The <em><a href="http://www.yeastgenome.org/locus/S000001415/overview">rrd1</a>Δ</em> mutation affecting another subunit was in the Pol α−specific area of the Venn diagram (area III, <a id="xref-fig-2-22" class="xref-fig" href="#F2">Figure 2A</a>) but its pattern of interactions is very similar to <em><a href="http://www.yeastgenome.org/locus/S000006073/overview">rrd2</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000002292/overview">pph21</a>Δ</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). PP2A plays a critical role in the G1/S transition possibly by helping the recruitment of <a href="http://www.yeastgenome.org/locus/S000004093/overview">Cdc45</a>, a key component of CMG complex (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chou <em>et al.</em> 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lin <em>et al.</em> 1998</a>). The recruitment of <a href="http://www.yeastgenome.org/locus/S000004093/overview">Cdc45</a> is necessary for CMG helicase activation and for Pol ε and Pol α recruitment to the fork (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yeeles <em>et al.</em> 2015</a>). It is clear that <em><a href="http://www.yeastgenome.org/locus/S000002292/overview">pph21</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000001415/overview">rrd1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000006073/overview">rrd2</a>Δ</em> have comparatively small effects on the fitness of Pol δ mutants, suggesting that PP2A specifically affects the recruitment of the pre-LC and/or maintains stability of the core replication fork components. Pol δ is distinct to Pol α and Pol ε because it is recruited by Pol α and <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a>, rather than the CMG complex, to replication forks (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Georgescu <em>et al.</em> 2015</a>) (see <a id="xref-fig-2-23" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-45">Other enriched components in area V correspond to S-phase checkpoint components <em><a href="http://www.yeastgenome.org/locus/S000004651/overview">csm3</a>Δ and <a href="http://www.yeastgenome.org/locus/S000000566/overview">mrc1</a>Δ</em> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Alcasabas <em>et al.</em> 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bando <em>et al.</em> 2009</a>). It is interesting that, like the <em><a href="http://www.yeastgenome.org/locus/S000002292/overview">pph21</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000001415/overview">rrd1</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000006073/overview">rrd2</a>Δ</em> mutations discussed previously, the S-phase checkpoint genes have comparatively small effects in Pol δ mutants. Perhaps this is because these S phase checkpoint proteins and PP2A are particularly important for the stability of the replication fork (<a id="xref-fig-2-24" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-46"><a href="http://www.yeastgenome.org/locus/S000001234/overview">Ctf8</a> and <a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>, which with <a href="http://www.yeastgenome.org/locus/S000000521/overview">Dcc1</a>, help form the <a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>-RFC-like complex (RLC) and load <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> onto chromatin (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bermudez <em>et al.</em> 2003</a>), were found in area V. Although <em><a href="http://www.yeastgenome.org/locus/S000000521/overview">dcc1</a>Δ</em> was not identified in area V, when we examined its pattern in different mutant backgrounds, it is very similar to <em><a href="http://www.yeastgenome.org/locus/S000001234/overview">ctf8</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). <a href="http://www.yeastgenome.org/locus/S000005743/overview">Rfc1</a> along with <a href="http://www.yeastgenome.org/locus/S000003829/overview">Rfc2</a>, <a href="http://www.yeastgenome.org/locus/S000005234/overview">Rfc3</a>, <a href="http://www.yeastgenome.org/locus/S000005454/overview">Rfc4</a>, and <a href="http://www.yeastgenome.org/locus/S000000291/overview">Rfc5</a> forms the main RFC complex and the <a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>-RFC complex complements <a href="http://www.yeastgenome.org/locus/S000005743/overview">RFC1</a> in loading <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> during replication. Interestingly when we examined the effect of the <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em> mutation, part of an alternative RLC, we saw a very different (opposite) pattern of interactions to those seem by <em><a href="http://www.yeastgenome.org/locus/S000001234/overview">ctf8</a>Δ</em>, <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000521/overview">dcc1</a>Δ</em> mutations. <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em> shows a strong negative genetic interaction with the Pol δ mutation but much weaker interactions with Pol α or Pol ε mutations (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>). The opposite patterns of genetic interactions seen for <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em> may reflect the fact that Elg1-RLC unloads <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> whereas <a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>-RLC loads <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> during replication (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kubota <em>et al.</em> 2013</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yu <em>et al.</em> 2014</a>), and PCNA-Pol δ interactions are more critical to efficient DNA binding than PCNA-Pol ε interactions, because Pol ε seems to be tethered directly by CMG (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Sengupta <em>et al.</em> 2013</a>).</p></div></div><div id="sec-21" class="subsection"><h3>Coordination of PCNA (un)loading with histone deposition</h3><p id="p-47"><a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>, in particular, illustrates how different profiles of fitness measurements in yeast strains carrying related mutations (<em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ vs. <a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em>) can draw attention to different cellular functions. Therefore, we next used fitness profiling to identify the nearest fitness profiles to <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em>. The top 10 nearest fitness profiles to <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em>, of more than 4800 profiles created, are shown in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4E</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS7.pdf">Figure S7A</a>. Importantly, as positive controls for this approach, we identified <em><a href="http://www.yeastgenome.org/locus/S000001234/overview">CTF8</a></em> as the ninth closest gene and <em><a href="http://www.yeastgenome.org/locus/S000000521/overview">DCC1</a></em> was 14th closest (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4E</a> and data not shown). Other known relationships are with <em><a href="http://www.yeastgenome.org/locus/S000006339/overview">CTF4</a></em>, which as we have discussed previously (in <em>Regulators that work in concert with Pol α, area III</em>), couples Pol α with CMG at the replication fork. In addition <a href="http://www.yeastgenome.org/locus/S000006339/overview">Ctf4</a> and <a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a> are involved in sister chromatid cohesion (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gambus <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hanna <em>et al.</em> 2001</a>). <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em> (affected by both ::<em>:<a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000000566/overview">YCL060C</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">MRC1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000000566/overview">YCL061C</a></em> disruptions) and <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">CTF18</a></em> are also important for the S-phase checkpoint pathway (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crabbe <em>et al.</em> 2010</a>). This profiling approach found similar profiles for <em><a href="http://www.yeastgenome.org/locus/S000000379/overview">SWD3</a></em> affecting the COMPASS complex, <em><a href="http://www.yeastgenome.org/locus/S000004410/overview">CDC73</a></em> a member of the <a href="http://www.yeastgenome.org/locus/S000000483/overview">Paf1</a> complex required for gene expression, modification of histones, and telomere maintenance, <em><a href="http://www.yeastgenome.org/locus/S000004929/overview">ELP6</a></em>, part of the Elongator complex, <em><a href="http://www.yeastgenome.org/locus/S000006345/overview">KAR3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003287/overview">MUP1</a></em>. Further experiments will be necessary to investigate the biochemical mechanisms underlying these related patterns.</p><p id="p-48">We next investigated the closest fitness profiles to <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em>, which behaved in a very different way to <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a>Δ</em> (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4, D and F</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS7.pdf">Figure S7B</a>). Interestingly, the 3/4 of the closest genes to <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">ELG1</a></em> affect the chromatin assembly factor 1 (CAF-1) (<em><a href="http://www.yeastgenome.org/locus/S000006222/overview">RLF2</a> (<a href="http://www.yeastgenome.org/locus/S000006222/overview">CAC1</a>)</em>/<em><a href="http://www.yeastgenome.org/locus/S000006222/overview">YPR018W</a></em>; <em><a href="http://www.yeastgenome.org/locus/S000004570/overview">CAC2</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000004570/overview">YML102W</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004570/overview">CAC2</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000004569/overview">YML102C-A</a></em>), responsible for histone deposition during DNA replication. We also found <em><a href="http://www.yeastgenome.org/locus/S000002893/overview">VPS72</a></em> encoding, an <a href="http://www.yeastgenome.org/locus/S000005372/overview">Htz1p</a>-binding component of the <a href="http://www.yeastgenome.org/locus/S000002742/overview">SWR1</a> complex, and <em><a href="http://www.yeastgenome.org/locus/S000000214/overview">HHT1</a></em> that encodes for the histone H3. This leads us to hypothesize that <a href="http://www.yeastgenome.org/locus/S000005670/overview">Elg1</a> and CAF-1 orchestrate replication fork progression by coupling <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> unloading to nucleosome assembly on lagging-strand. The HIR complex also loads histones, but this activity was defined as being independent of DNA replication in mammalian cells (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ray-Gallet <em>et al.</em> 2002</a>). However, in yeast, it is not clear whether the role of the HIR complex is restrained to outside of S-phase. The fact that yeast mutants lacking CAF-1 subunits are viable suggests redundant pathways for histone incorporation during S-phase. On the basis of our data (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS7.pdf">Figure S7C</a>), we suggest that the mirrored profiles of CAF-1 and HIR complexes reflect distinct activities of histone deposition during DNA replication.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-49">Genome replication is perhaps the most critical task that faces any cell. Replication is a complex task, depending on numerous different proteins and nucleic acids that function together in what has been likened to a machine (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alberts 1984</a>). A powerful combination of biochemical and genetic approaches has succeeded in identifying the core components of the replisome. The replication machinery interfaces with the cell cycle, chromatin, and transcription machinery to ensure that the processes of DNA replication are coordinated within the cell cycle and with other cellular processes. Replication machinery can falter, for example, if it encounters DNA damage, and therefore a complex series of error correction pathways help overcome difficult situations. Although the core components of eukaryotic DNA replication have been identified our understanding how these replication components are regulated by the cell cycle, or respond to error, remains far from complete. In this paper we have performed genome-wide suppressor/enhancer interaction screens to identify genes that affect the fitness of replication defective strains. We examined strains defective in the each of the three major DNA polymerases (Pol α, Pol δ, and Pol ε).</p><p id="p-50">Our unbiased genome-wide experiments have identified many genetic interactions consistent with known connections between DNA replication, repair, recombination, telomere maintenance, chromatin modification, and transcription. Interestingly, most of the genetic interactions we identified were specific to individual polymerases, rather than shared across the three polymerases, which indicates that most interactions affect specific aspects of the eukaryotic DNA replication process and this pattern likely reflects the different roles of Pol α, Pol δ, and Pol ε during DNA replication. Pol ε plays a major structural role during DNA replication, as part of the Pre-LC that is so critical for initiating DNA replication and, with Pol α, a part of the core replication fork machinery. Pol δ is less structurally important (<a id="xref-fig-2-25" class="xref-fig" href="#F2">Figure 2B</a>). We also identified interactions of Pol ε with histone deacetylases. Histone acetylation is an important aspect of chromatin replication and assembly, and histone deacetylation has been found to occur after histone deposition to allow the newly assembled chromatin to acquire higher order structure (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Annunziato 2013</a>). How or why histone deacetylation is particularly important for leading-strand synthesis will require further investigation.</p><p id="p-51"><a href="http://www.yeastgenome.org/locus/S000005670/overview">Elg1</a> is part of an alternative RFC complex and was recently shown to be involved in the removal of <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> from DNA (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kubota <em>et al.</em> 2013</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Yu <em>et al.</em> 2014</a>). Removal of <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> is particularly important for lagging-strand replication because each Okazaki fragment uses a single <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> trimer. Consistent with these recent observations, we found that the <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em> mutation particularly affects fitness of Pol δ mutants. Furthermore, <em><a href="http://www.yeastgenome.org/locus/S000005670/overview">elg1</a>Δ</em> caused very similar effects to <em><a href="http://www.yeastgenome.org/locus/S000006222/overview">cac1</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000004570/overview">cac2</a>Δ</em> mutations in all polymerase-defective strains (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.115.021493.DC1/FigureS7.pdf">Figure S7</a>). <a href="http://www.yeastgenome.org/locus/S000006222/overview">Cac1</a> and <a href="http://www.yeastgenome.org/locus/S000004570/overview">Cac2</a> encode subunits of CAF-1, involved in chromatin reassembly after DNA replication (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kaufman <em>et al.</em> 1997</a>). Recently, it was shown that nucleosome assembly helps regulate termination of Okazaki fragment synthesis in yeast (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Smith and Whitehouse 2012</a>) and defects in histone deposition delays <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> unloading in human cells (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mejlvang <em>et al.</em> 2014</a>). Our results lead us to suggest that <a href="http://www.yeastgenome.org/locus/S000005670/overview">Elg1</a> and CAF-1 orchestrate replication fork progression by coupling <a href="http://www.yeastgenome.org/locus/S000000292/overview">PCNA</a> unloading to nucleosome assembly. We envisage that this function is specific to CAF-1 because <em><a href="http://www.yeastgenome.org/locus/S000005564/overview">hir2</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000003901/overview">hir3</a>Δ</em> mutations affecting HIRA, another nucleosome assembly complex, show an “opposite” profile to <em><a href="http://www.yeastgenome.org/locus/S000006222/overview">cac1</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000004570/overview">cac2</a>Δ</em> when combined with the different DNA polymerase mutants. However, the finding that <em><a href="http://www.yeastgenome.org/locus/S000005564/overview">hir2</a>Δ</em> and <em><a href="http://www.yeastgenome.org/locus/S000003901/overview">hir3</a>Δ</em> show interactions with polymerase mutations and are sensitive to the S-phase poison HU strongly suggests that in yeast HIRA plays a role in histone deposition during the S-phase. These types of insights, built on unbiased, genome-wide genetic interaction studies, lead us to believe that our data will be a valuable resource for understanding the processes of DNA replication in the context related processes such as cell-cycle progression, transcription, histone deposition, and epigenetic inheritance.</p><p id="p-52">In conclusion, we report thousands of genetic interactions that are informative about how the three major eukaryotic DNA polymerases coordinate their activities to replicate DNA.</p></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Charles Boone for providing thermosensitive strains, Simon Whitehall for constructive comments and critical reading of the manuscript, and the members of the Lydall lab for helpful discussion on the manuscript. M.D. was funded by an EMBO Long Term Fellowship (ALTF 218-2012). This work was supported by Biotechnology and Biological Sciences Research Council (BB/M002314/1), Cancer Research UK (C23629), and Wellcome Trust (WT093088MA) grants (to D.L.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-54">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021493/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021493/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-55"><em>Communicating editor: R. A. Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-15"><span class="received-label">Received </span>July 15, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-19"><span class="accepted-label">Accepted </span>August 19, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Dubarry <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.1" data-doi="10.1534/genetics.108.092577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Downey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zubko</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genomewide suppressor and enhancer analysis of cdc13–1 reveals varied cellular processes influencing telomere capping in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2251</span>–<span class="cit-lpage">2266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092577%26rft_id%253Dinfo%253Apmid%252F18845848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.2" data-doi="10.1371/journal.pgen.1001362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holstein</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAddinall%26rft.auinit1%253DS.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001362%26rft.epage%253De1001362%26rft.atitle%253DQuantitative%2Bfitness%2Banalysis%2Bshows%2Bthat%2BNMD%2Bproteins%2Band%2Bmany%2Bother%2Bprotein%2Bcomplexes%2Bsuppress%2Bor%2Benhance%2Bdistinct%2Btelomere%2Bcap%2Bdefects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001362%26rft_id%253Dinfo%253Apmid%252F21490951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.3" data-doi="10.1371/journal.pgen.1001209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aksenova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maceluch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pursell</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mismatch repair-independent increase in spontaneous mutagenesis in yeast lacking non-essential subunits of DNA polymerase epsilon.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAksenova%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001209%26rft.epage%253De1001209%26rft.atitle%253DMismatch%2Brepair-independent%2Bincrease%2Bin%2Bspontaneous%2Bmutagenesis%2Bin%2Byeast%2Blacking%2Bnon-essential%2Bsubunits%2Bof%2BDNA%2Bpolymerase%2B%25CE%25B5.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001209%26rft_id%253Dinfo%253Apmid%252F21124948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124948&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.4" data-doi="10.1101/SQB.1984.049.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alberts</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The DNA enzymology of protein machines.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DAlberts%26rft.auinit1%253DB.M.%26rft.volume%253D49%26rft.issue%253D0%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DThe%2BDNA%2BEnzymology%2Bof%2BProtein%2BMachines%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1984.049.01.003%26rft_id%253Dinfo%253Apmid%252F6597750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjY6IjQ5LzAvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.5" data-doi="10.1038/ncb1101-958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alcasabas</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werler</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mrc1 transduces signals of DNA replication stress to activate Rad53.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DAlcasabas%26rft.auinit1%253DA.%2BA.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253D958%26rft.epage%253D965%26rft.atitle%253DMrc1%2Btransduces%2Bsignals%2Bof%2BDNA%2Breplication%2Bstress%2Bto%2Bactivate%2BRad53.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1101-958%26rft_id%253Dinfo%253Apmid%252F11715016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1101-958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11715016&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172070300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.6" data-doi="10.1371/journal.pone.0066242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrew</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merchan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pentose phosphate pathway function affects tolerance to the G-quadruplex binder TMPyP4.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e66242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66242%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066242%26rft_id%253Dinfo%253Apmid%252F23776642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23776642&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Annunziato</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling chromatin: the long and winding road.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1819</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1819%26rft.spage%253D196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.8" data-doi="10.1093/nar/19.18.4867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamatake</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Cloning DPB3, the gene encoding the third subunit of DNA polymerase II of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4867</span>–<span class="cit-lpage">4872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAraki%26rft.auinit1%253DH.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D4867%26rft.epage%253D4872%26rft.atitle%253DCloning%2BDPB3%252C%2Bthe%2Bgene%2Bencoding%2Bthe%2Bthird%2Bsubunit%2Bof%2BDNA%2Bpolymerase%2BII%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.18.4867%26rft_id%253Dinfo%253Apmid%252F1923754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xOC80ODY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.9" data-doi="10.1074/jbc.M109.065730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bando</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Csm3, Tof1, and Mrc1 form a heterotrimeric mediator complex that associates with DNA replication forks.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">34355</span>–<span class="cit-lpage">34365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.065730%26rft_id%253Dinfo%253Apmid%252F19819872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNDkvMzQzNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.10" data-doi="10.1073/pnas.1434308100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bermudez</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniwa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tappin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yokomori</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The alternative Ctf18-Dcc1-Ctf8-replication factor C complex required for sister chromatid cohesion loads proliferating cell nuclear antigen onto DNA.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10237</span>–<span class="cit-lpage">10242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1434308100%26rft_id%253Dinfo%253Apmid%252F12930902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE4LzEwMjM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.11" data-doi="10.1038/nmeth.1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBreslow%26rft.auinit1%253DD.%2BK.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D711%26rft.epage%253D718%26rft.atitle%253DA%2Bcomprehensive%2Bstrategy%2Benabling%2Bhigh-resolution%2Bfunctional%2Banalysis%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1234%26rft_id%253Dinfo%253Apmid%252F18622397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18622397&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.12" data-doi="10.1128/MCB.13.1.496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Budd</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">DNA polymerases delta and epsilon are required for chromosomal replication in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBudd%26rft.auinit1%253DM%2BE%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D496%26rft.epage%253D505%26rft.atitle%253DDNA%2Bpolymerases%2Bdelta%2Band%2Bepsilon%2Bare%2Brequired%2Bfor%2Bchromosomal%2Breplication%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.1.496%26rft_id%253Dinfo%253Apmid%252F8417347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjEzLzEvNDk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.13" data-doi="10.1016/S0092-8674(00)81820-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorch</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacomis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">RSC, an essential, abundant chromatin-remodeling complex.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCairns%26rft.auinit1%253DB.%2BR.%26rft.volume%253D87%26rft.issue%253D7%26rft.spage%253D1249%26rft.epage%253D1260%26rft.atitle%253DRSC%252C%2Ban%2Bessential%252C%2Babundant%2Bchromatin-remodeling%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981820-6%26rft_id%253Dinfo%253Apmid%252F8980231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81820-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8980231&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WA54100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.14" data-doi="10.1074/jbc.M207226200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Protein phosphatase 2A regulates binding of Cdc45 to the prereplication complex.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">40520</span>–<span class="cit-lpage">40527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M207226200%26rft_id%253Dinfo%253Apmid%252F12185086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvNDMvNDA1MjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.15" data-doi="10.1038/nsmb.1932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crabbe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pantesco</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Vos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasero</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of replication profiles reveals key role of RFC-Ctf18 in yeast replication stress response.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1391</span>–<span class="cit-lpage">1397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DCrabbe%26rft.auinit1%253DL.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1391%26rft.epage%253D1397%26rft.atitle%253DAnalysis%2Bof%2Breplication%2Bprofiles%2Breveals%2Bkey%2Brole%2Bof%2BRFC-Ctf18%2Bin%2Byeast%2Breplication%2Bstress%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1932%26rft_id%253Dinfo%253Apmid%252F20972444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972444&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.16" data-doi="10.1038/nsmb.2962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daigaku</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keszthelyi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyabe</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global profile of replicative polymerase usage.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D22%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2962%26rft_id%253Dinfo%253Apmid%252F25664722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25664722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.17" data-doi="10.1074/jbc.274.32.22283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dua</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Analysis of the essential functions of the C-terminal protein/protein interaction domain of <em>Saccharomyces cerevisiae</em> pol epsilon and its unexpected ability to support growth in the absence of the DNA polymerase domain.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">22283</span>–<span class="cit-lpage">22288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DDua%26rft.auinit1%253DR.%26rft.volume%253D274%26rft.issue%253D32%26rft.spage%253D22283%26rft.epage%253D22288%26rft.atitle%253DAnalysis%2Bof%2Bthe%2BEssential%2BFunctions%2Bof%2Bthe%2BC-terminal%2BProtein%252FProtein%2BInteraction%2BDomain%2Bof%2BSaccharomyces%2Bcerevisiae%2Bpol%2B%257Bvarepsilon%257D%2Band%2BIts%2BUnexpected%2BAbility%2Bto%2BSupport%2BGrowth%2Bin%2Bthe%2BAbsence%2Bof%2Bthe%2BDNA%2BPolymerase%2BDomain%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.32.22283%26rft_id%253Dinfo%253Apmid%252F10428796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMzIvMjIyODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.18" data-doi="10.1186/1471-2105-10-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinfeld</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakhini</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEden%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DGOrilla%253A%2Ba%2Btool%2Bfor%2Bdiscovery%2Band%2Bvisualization%2Bof%2Benriched%2BGO%2Bterms%2Bin%2Branked%2Bgene%2Blists.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-48%26rft_id%253Dinfo%253Apmid%252F19192299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19192299&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.19" data-doi="10.1128/MCB.21.14.4495-4504.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Urso</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>Schizosaccharomyces pombe</em> cells lacking the amino-terminal catalytic domains of DNA polymerase epsilon are viable but require the DNA damage checkpoint control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4495</span>–<span class="cit-lpage">4504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFeng%26rft.auinit1%253DW.%26rft.volume%253D21%26rft.issue%253D14%26rft.spage%253D4495%26rft.epage%253D4504%26rft.atitle%253DSchizosaccharomyces%2Bpombe%2BCells%2BLacking%2Bthe%2BAmino-Terminal%2BCatalytic%2BDomains%2Bof%2BDNA%2BPolymerase%2BEpsilon%2BAre%2BViable%2Bbut%2BRequire%2Bthe%2BDNA%2BDamage%2BCheckpoint%2BControl%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.14.4495-4504.2001%26rft_id%253Dinfo%253Apmid%252F11416129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xNC80NDk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.20" data-doi="10.1534/genetics.112.140467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The ubiquitin-proteasome system of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFinley%26rft.auinit1%253DD.%26rft.volume%253D192%26rft.issue%253D2%26rft.spage%253D319%26rft.epage%253D360%26rft.atitle%253DThe%2BUbiquitin-Proteasome%2BSystem%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140467%26rft_id%253Dinfo%253Apmid%252F23028185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvMzE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.21" data-doi="10.1371/journal.pgen.1005049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flood</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kow</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Replicative DNA polymerase delta but not epsilon proofreads errors in Cis and in Trans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005049%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005049%26rft_id%253Dinfo%253Apmid%252F25742645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25742645&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.22" data-doi="10.1038/emboj.2009.226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gambus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deursen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polychronopoulos</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foltman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A key role for Ctf4 in coupling the MCM2–7 helicase to DNA polymerase alpha within the eukaryotic replisome.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2992</span>–<span class="cit-lpage">3004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DGambus%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D19%26rft.spage%253D2992%26rft.epage%253D3004%26rft.atitle%253DA%2Bkey%2Brole%2Bfor%2BCtf4%2Bin%2Bcoupling%2Bthe%2BMCM2-7%2Bhelicase%2Bto%2BDNA%2Bpolymerase%2Balpha%2Bwithin%2Bthe%2Beukaryotic%2Breplisome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.226%26rft_id%253Dinfo%253Apmid%252F19661920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2009.226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19661920&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270514000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.23" data-doi="10.1080/10409230590935433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA polymerases that propagate the eukaryotic DNA replication fork.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DGarg%26rft.auinit1%253DP.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D128%26rft.atitle%253DDNA%2Bpolymerases%2Bthat%2Bpropagate%2Bthe%2Beukaryotic%2BDNA%2Breplication%2Bfork.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10409230590935433%26rft_id%253Dinfo%253Apmid%252F15814431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10409230590935433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15814431&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228922600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Georgescu</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schauer</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langston</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurieva</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reconstitution of a eukaryotic replisome reveals suppression mechanisms that define leading/lagging strand operation.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-fpage">10.7554/eLife.04988</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.25" data-doi="10.1128/MCB.21.9.3144-3158.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroll</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> CTF18 and CTF4 are required for sister chromatid cohesion.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3144</span>–<span class="cit-lpage">3158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHanna%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D3144%26rft.epage%253D3158%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BCTF18%2Band%2BCTF4%2BAre%2BRequired%2Bfor%2BSister%2BChromatid%2BCohesion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.9.3144-3158.2001%26rft_id%253Dinfo%253Apmid%252F11287619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMzE0NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culotti</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culotti</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Genetic control of the cell division cycle in yeast: V. Genetic analysis of <em>cdc</em> mutants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHartwell%26rft.auinit1%253DL.%2BH.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D286%26rft.atitle%253DGENETIC%2BCONTROL%2BOF%2BTHE%2BCELL%2BDIVISION%2BCYCLE%2BIN%2BYEAST%253A%2BV.%2BGENETIC%2BANALYSIS%2BOF%2Bcdc%2BMUTANTS%26rft_id%253Dinfo%253Apmid%252F17248617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMi8yNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.27" data-doi="10.1101/gad.982902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivessa</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Saccharomyces</em> Rrm3p, a 5′ to 3′ DNA helicase that promotes replication fork progression through telomeric and subtelomeric DNA.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1383</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DIvessa%26rft.auinit1%253DA.%2BS.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D1383%26rft.epage%253D1396%26rft.atitle%253DSaccharomyces%2BRrm3p%252C%2Ba%2B5%2527%2Bto%2B3%2527%2BDNA%2Bhelicase%2Bthat%2Bpromotes%2Breplication%2Bfork%2Bprogression%2Bthrough%2Btelomeric%2Band%2Bsubtelomeric%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.982902%26rft_id%253Dinfo%253Apmid%252F12050116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE2LzExLzEzODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.28" data-doi="10.1016/S1097-2765(03)00456-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivessa</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenzmeier</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessler</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudsouzian</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnakenberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> helicase Rrm3p facilitates replication past nonhistone protein-DNA complexes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DIvessa%26rft.auinit1%253DA.%2BS.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D1525%26rft.epage%253D1536%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2Bhelicase%2BRrm3p%2Bfacilitates%2Breplication%2Bpast%2Bnonhistone%2Bprotein-DNA%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900456-8%26rft_id%253Dinfo%253Apmid%252F14690605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00456-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14690605&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187511600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.29" data-doi="10.1016/j.molcel.2015.05.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klassen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A major role of DNA polymerase delta in replication of both the leading and lagging DNA strands.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.05.038%26rft_id%253Dinfo%253Apmid%252F26145172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.05.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26145172&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.30" data-doi="10.1101/gad.11.3.345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Ultraviolet radiation sensitivity and reduction of telomeric silencing in <em>Saccharomyces cerevisiae</em> cells lacking chromatin assembly factor I.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKaufman%26rft.auinit1%253DP%2BD%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D345%26rft.epage%253D357%26rft.atitle%253DUltraviolet%2Bradiation%2Bsensitivity%2Band%2Breduction%2Bof%2Btelomeric%2Bsilencing%2Bin%2BSaccharomyces%2Bcerevisiae%2Bcells%2Blacking%2Bchromatin%2Bassembly%2Bfactor-I.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.3.345%26rft_id%253Dinfo%253Apmid%252F9030687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTEvMy8zNDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.31" data-doi="10.1016/S1097-2765(00)80361-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kesti</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flick</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keranen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syvaoja</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">DNA polymerase epsilon catalytic domains are dispensable for DNA replication, DNA repair, and cell viability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DKesti%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D685%26rft.atitle%253DDNA%2Bpolymerase%2Bepsilon%2Bcatalytic%2Bdomains%2Bare%2Bdispensable%2Bfor%2BDNA%2Breplication%252C%2BDNA%2Brepair%252C%2Band%2Bcell%2Bviability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980361-5%26rft_id%253Dinfo%253Apmid%252F10360184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80361-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10360184&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080573400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.32" data-doi="10.1016/0306-9877(95)90228-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klipper-Aurbach</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braunspiegel-Weintrob</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peleg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mathematical formulae for the prediction of the residual beta cell function during the first two years of disease in children and adolescents with insulin-dependent diabetes mellitus.</span> <abbr class="cit-jnl-abbrev">Med. Hypotheses</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMedical%2Bhypotheses%26rft.stitle%253DMed%2BHypotheses%26rft.aulast%253DKlipper-Aurbach%26rft.auinit1%253DY.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D486%26rft.epage%253D490%26rft.atitle%253DMathematical%2Bformulae%2Bfor%2Bthe%2Bprediction%2Bof%2Bthe%2Bresidual%2Bbeta%2Bcell%2Bfunction%2Bduring%2Bthe%2Bfirst%2Btwo%2Byears%2Bof%2Bdisease%2Bin%2Bchildren%2Band%2Badolescents%2Bwith%2Binsulin-dependent%2Bdiabetes%2Bmellitus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0306-9877%252895%252990228-7%26rft_id%253Dinfo%253Apmid%252F8748093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0306-9877(95)90228-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8748093&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TH69600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.33" data-doi="10.1101/gad.1784309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viggiani</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">O. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide replication profiles indicate an expansive role for Rpd3L in regulating replication initiation timing or efficiency, and reveal genomic loci of Rpd3 function in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BR.V.%26rft.volume%253D23%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1090%26rft.atitle%253DGenome-wide%2Breplication%2Bprofiles%2Bindicate%2Ban%2Bexpansive%2Brole%2Bfor%2BRpd3L%2Bin%2Bregulating%2Breplication%2Binitiation%2Btiming%2Bor%2Befficiency%252C%2Band%2Breveal%2Bgenomic%2Bloci%2Bof%2BRpd3%2Bfunction%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1784309%26rft_id%253Dinfo%253Apmid%252F19417103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiMjMvOS8xMDc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.34" data-doi="10.1371/journal.pgen.1005110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochenova</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daee</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mertz</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shcherbakova</span>  <span class="cit-name-given-names">P. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA polymerase zeta-dependent lesion bypass in <em>Saccharomyces cerevisiae</em> is accompanied by error-prone copying of long stretches of adjacent DNA.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005110%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005110%26rft_id%253Dinfo%253Apmid%252F25826305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25826305&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.35" data-doi="10.1016/j.molcel.2013.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kubota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanemaki</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Elg1 replication factor C-like complex functions in PCNA unloading during DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.02.012%26rft_id%253Dinfo%253Apmid%252F23499004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23499004&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318208600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.36" data-doi="10.1128/MCB.9.10.4447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamatake</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motto-Fox</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Fidelity of DNA polymerase I and the DNA polymerase I-DNA primase complex from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">4447</span>–<span class="cit-lpage">4458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKunkel%26rft.auinit1%253DT%2BA%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D4447%26rft.epage%253D4458%26rft.atitle%253DFidelity%2Bof%2BDNA%2Bpolymerase%2BI%2Band%2Bthe%2BDNA%2Bpolymerase%2BI-DNA%2Bprimase%2Bcomplex%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.9.10.4447%26rft_id%253Dinfo%253Apmid%252F2555694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvNDQ0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.37" data-doi="10.1016/j.tibs.2012.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">New insights into replisome fluidity during chromosome replication.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D38%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2012.10.003%26rft_id%253Dinfo%253Apmid%252F23153958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2012.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23153958&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317315700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.38" data-doi="10.1073/pnas.1418334111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langston</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurieva</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgescu</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CMG helicase and DNA polymerase epsilon form a functional 15-subunit holoenzyme for eukaryotic leading-strand DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">15390</span>–<span class="cit-lpage">15395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1418334111%26rft_id%253Dinfo%253Apmid%252F25313033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQzLzE1MzkwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.39" data-doi="10.1186/1471-2105-11-287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLawless%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D287%26rft.epage%253D287%26rft.atitle%253DColonyzer%253A%2Bautomated%2Bquantification%2Bof%2Bmicro-organism%2Bgrowth%2Bcharacteristics%2Bon%2Bsolid%2Bagar.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-287%26rft_id%253Dinfo%253Apmid%252F20509870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20509870&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.40" data-doi="10.1038/nature10161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">W. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Coordination of DNA replication and histone modification by the Rik1-Dos2 complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D244%26rft.epage%253D248%26rft.atitle%253DCoordination%2Bof%2BDNA%2Breplication%2Band%2Bhistone%2Bmodification%2Bby%2Bthe%2BRik1-Dos2%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10161%26rft_id%253Dinfo%253Apmid%252F21725325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21725325&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.41" data-doi="10.1038/nbt.1832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Systematic exploration of essential yeast gene function with temperature-sensitive mutants.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D361%26rft.epage%253D367%26rft.atitle%253DSystematic%2Bexploration%2Bof%2Bessential%2Byeast%2Bgene%2Bfunction%2Bwith%2Btemperature-sensitive%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1832%26rft_id%253Dinfo%253Apmid%252F21441928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21441928&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289284900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.42" data-doi="10.1073/pnas.95.25.14693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheidtmann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohst</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newport</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Protein phosphatase 2A is required for the initiation of chromosomal DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">14693</span>–<span class="cit-lpage">14698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLin%26rft.auinit1%253DX.-H.%26rft.volume%253D95%26rft.issue%253D25%26rft.spage%253D14693%26rft.epage%253D14698%26rft.atitle%253DProtein%2Bphosphatase%2B2A%2Bis%2Brequired%2Bfor%2Bthe%2Binitiation%2Bof%2Bchromosomal%2BDNA%2Breplication%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.25.14693%26rft_id%253Dinfo%253Apmid%252F9843951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjUvMTQ2OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.43" data-doi="10.1146/annurev.biochem.052308.103205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa-Sugata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Eukaryotic chromosome DNA replication: where, when, and how?</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DMasai%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.spage%253D89%26rft.epage%253D130%26rft.atitle%253DEukaryotic%2Bchromosome%2BDNA%2Breplication%253A%2Bwhere%252C%2Bwhen%252C%2Band%2Bhow%253F%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.052308.103205%26rft_id%253Dinfo%253Apmid%252F20373915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.052308.103205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20373915&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280225300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.44" data-doi="10.1083/jcb.201305017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mejlvang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neelsen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasencakova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">New histone supply regulates replication fork speed and PCNA unloading.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201305017%26rft_id%253Dinfo%253Apmid%252F24379417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjIwNC8xLzI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.45" data-doi="10.1101/gad.1883410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muramatsu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tak</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CDK-dependent complex formation between replication proteins Dpb11, Sld2, Pol (epsilon}, and GINS in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMuramatsu%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D6%26rft.spage%253D602%26rft.epage%253D612%26rft.atitle%253DCDK-dependent%2Bcomplex%2Bformation%2Bbetween%2Breplication%2Bproteins%2BDpb11%252C%2BSld2%252C%2BPol%2B%257Bvarepsilon%257D%252C%2Band%2BGINS%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1883410%26rft_id%253Dinfo%253Apmid%252F20231317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjQvNi82MDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.46" data-doi="10.1038/83673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myung</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">SGS1, the <em>Saccharomyces cerevisiae</em> homologue of BLM and WRN, suppresses genome instability and homeologous recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMyung%26rft.auinit1%253DK.%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D116%26rft.atitle%253DSGS1%252C%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bhomologue%2Bof%2BBLM%2Band%2BWRN%252C%2Bsuppresses%2Bgenome%2Binstability%2Band%2Bhomeologous%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F83673%26rft_id%253Dinfo%253Apmid%252F11138010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/83673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11138010&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166187900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.47" data-doi="10.1093/nar/gku746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ngo</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubarry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The 9–1-1 checkpoint clamp stimulates DNA resection by Dna2-Sgs1 and Exo1.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">10516</span>–<span class="cit-lpage">10528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku746%26rft_id%253Dinfo%253Apmid%252F25122752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi8xNi8xMDUxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.48" data-doi="10.1016/S1097-2765(02)00526-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray-Gallet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quivy</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scamps</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">HIRA is critical for a nucleosome assembly pathway independent of DNA synthesis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRay-Gallet%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D1091%26rft.epage%253D1100%26rft.atitle%253DHIRA%2Bis%2Bcritical%2Bfor%2Ba%2Bnucleosome%2Bassembly%2Bpathway%2Bindependent%2Bof%2BDNA%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900526-9%26rft_id%253Dinfo%253Apmid%252F12049744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00526-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12049744&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175967100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.49" data-doi="10.1038/nature14183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reijns</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Proce</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Lagging-strand replication shapes the mutational landscape of the genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D502%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14183%26rft_id%253Dinfo%253Apmid%252F25624100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25624100&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.50" data-doi="10.1534/genetics.112.142349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rizzardi</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorn</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahl</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DNA replication origin function is promoted by H3K4 di-methylation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142349%26rft_id%253Dinfo%253Apmid%252F22851644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvMzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.51" data-doi="10.1016/j.cell.2005.08.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denic</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhamidipati</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchuldiner%26rft.auinit1%253DM.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D507%26rft.epage%253D519%26rft.atitle%253DExploration%2Bof%2Bthe%2Bfunction%2Band%2Borganization%2Bof%2Bthe%2Byeast%2Bearly%2Bsecretory%2Bpathway%2Bthrough%2Ban%2Bepistatic%2Bminiarray%2Bprofile.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.08.031%26rft_id%253Dinfo%253Apmid%252F16269340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.08.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16269340&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233264300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.52" data-doi="10.1016/j.cub.2013.02.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sengupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deursen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Piccoli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labib</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dpb2 integrates the leading-strand DNA polymerase into the eukaryotic replisome.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D543%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.02.011%26rft_id%253Dinfo%253Apmid%252F23499531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23499531&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.53" data-doi="10.1016/j.cub.2008.03.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schleker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kugou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ino80 chromatin remodeling complex promotes recovery of stalled replication forks.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">566</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DShimada%26rft.auinit1%253DK.%26rft.volume%253D18%26rft.issue%253D8%26rft.spage%253D566%26rft.epage%253D575%26rft.atitle%253DIno80%2Bchromatin%2Bremodeling%2Bcomplex%2Bpromotes%2Brecovery%2Bof%2Bstalled%2Breplication%2Bforks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2008.03.049%26rft_id%253Dinfo%253Apmid%252F18406137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2008.03.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18406137&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255180000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.54" data-doi="10.1038/nature10895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitehouse</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Intrinsic coupling of lagging-strand synthesis to chromatin assembly.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">483</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D483%26rft.issue%253D7390%26rft.spage%253D434%26rft.epage%253D438%26rft.atitle%253DIntrinsic%2Bcoupling%2Bof%2Blagging-strand%2Bsynthesis%2Bto%2Bchromatin%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10895%26rft_id%253Dinfo%253Apmid%252F22419157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22419157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301771200036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.55" data-doi="10.1385/1-59259-958-3:171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Synthetic genetic array analysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%26rft.volume%253D313%26rft.spage%253D171%26rft.epage%253D192%26rft.atitle%253DSynthetic%2Bgenetic%2Barray%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1-59259-958-3%253A171%26rft_id%253Dinfo%253Apmid%252F16118434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1-59259-958-3:171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16118434&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.56" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.57" data-doi="10.1016/j.dnarep.2007.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tourriere</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasero</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Maintenance of fork integrity at damaged DNA and natural pause sites.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">900</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DTourriere%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253D900%26rft.epage%253D913%26rft.atitle%253DMaintenance%2Bof%2Bfork%2Bintegrity%2Bat%2Bdamaged%2BDNA%2Band%2Bnatural%2Bpause%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2007.02.004%26rft_id%253Dinfo%253Apmid%252F17379579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2007.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17379579&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2187.58" data-doi="10.1046/j.1365-2443.2003.00683.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubota</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Double-stranded DNA binding properties of <em>Saccharomyces cerevisiae</em> DNA polymerase epsilon and of the Dpb3p-Dpb4p subassembly.</span> <span class="cit-source">Genes Cells</span> <span class="cit-vol">8</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DTsubota%26rft.auinit1%253DT%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D873%26rft.epage%253D888%26rft.atitle%253DDouble-stranded%2BDNA%2Bbinding%2Bproperties%2Bof%2BSaccharomyces%2Bcerevisiae%2BDNA%2Bpolymerase%2Bepsilon%2Band%2Bof%2Bthe%2BDpb3p-Dpb4p%2Bsubassembly%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.2003.00683.x%26rft_id%253Dinfo%253Apmid%252F14622139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.2003.00683.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14622139&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186755700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.59" data-doi="10.1038/nsmb.1780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unnikrishnan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gafken</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukiyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic changes in histone acetylation regulate origins of DNA replication.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DUnnikrishnan%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D430%26rft.epage%253D437%26rft.atitle%253DDynamic%2Bchanges%2Bin%2Bhistone%2Bacetylation%2Bregulate%2Borigins%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1780%26rft_id%253Dinfo%253Apmid%252F20228802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20228802&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.60" data-doi="10.1016/S1097-2765(02)00702-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogelauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubbi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Histone acetylation regulates the time of replication origin firing.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVogelauer%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D1223%26rft.epage%253D1233%26rft.atitle%253DHistone%2Bacetylation%2Bregulates%2Bthe%2Btime%2Bof%2Breplication%2Borigin%2Bfiring.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900702-5%26rft_id%253Dinfo%253Apmid%252F12453428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00702-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453428&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179450600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.61" data-doi="10.1038/369207a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Anatomy of a DNA replication fork revealed by reconstitution of SV40 DNA replication in vitro.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">369</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaga%26rft.auinit1%253DS.%26rft.volume%253D369%26rft.issue%253D6477%26rft.spage%253D207%26rft.epage%253D212%26rft.atitle%253DAnatomy%2Bof%2Ba%2BDNA%2Breplication%2Bfork%2Brevealed%2Bby%2Breconstitution%2Bof%2BSV40%2BDNA%2Breplication%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F369207a0%26rft_id%253Dinfo%253Apmid%252F7910375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/369207a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7910375&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NM06700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.62" data-doi="10.1146/annurev.biochem.67.1.721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The DNA replication fork in eukaryotic cells.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DWaga%26rft.auinit1%253DS.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D721%26rft.epage%253D751%26rft.atitle%253DTHE%2BDNA%2BREPLICATION%2BFORK%2BIN%2BEUKARYOTIC%2BCELLS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.67.1.721%26rft_id%253Dinfo%253Apmid%252F9759502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.67.1.721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9759502&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075721700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickson</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">SGS1, a homologue of the Bloom’s and Werner’s syndrome genes, is required for maintenance of genome stability in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWatt%26rft.auinit1%253DP.%2BM.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D935%26rft.epage%253D945%26rft.atitle%253DSGS1%252C%2Ba%2BHomologue%2Bof%2Bthe%2BBloom%2527s%2Band%2BWerner%2527s%2BSyndrome%2BGenes%252C%2BIs%2BRequired%2Bfor%2BMaintenance%2Bof%2BGenome%2BStability%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F8913739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzMvOTM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.64" data-doi="10.2307/3578518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Dual cell cycle checkpoints sensitive to chromosome replication and DNA damage in the budding yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Radiat. Res.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRadiation%2Bresearch%26rft.stitle%253DRadiat%2BRes%26rft.aulast%253DWeinert%26rft.auinit1%253DT.%2BA.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D143%26rft.atitle%253DDual%2Bcell%2Bcycle%2Bcheckpoints%2Bsensitive%2Bto%2Bchromosome%2Breplication%2Band%2BDNA%2Bdamage%2Bin%2Bthe%2Bbudding%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3578518%26rft_id%253Dinfo%253Apmid%252F1438694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3578518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1438694&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JY58300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.65" data-doi="10.1101/cshperspect.a012815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeeles</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poli</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marians</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasero</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rescuing stalled or damaged replication forks.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a012815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DYeeles%26rft.auinit1%253DJ.%2BT.%2BP.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253Da012815%26rft.epage%253Da012815%26rft.atitle%253DRescuing%2BStalled%2Bor%2BDamaged%2BReplication%2BForks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012815%26rft_id%253Dinfo%253Apmid%252F23637285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzUvYTAxMjgxNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.66" data-doi="10.1038/nature14285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeeles</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deegan</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Early</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Regulated eukaryotic DNA replication origin firing with purified proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">519</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D519%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14285%26rft_id%253Dinfo%253Apmid%252F25739503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25739503&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2187.67" data-doi="10.1016/j.molcel.2014.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Strand-specific analysis shows protein binding at replication forks and PCNA unloading from lagging strands when forks stall.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D56%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.09.017%26rft_id%253Dinfo%253Apmid%252F25449133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25449133&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2187.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2187.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2177" title="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2199" title="Selection Against Maternal microRNA Target Sites in Maternal Transcripts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2187.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w3nmLvS0P_RHkLnErd2OWvEGNMPfeUXqC4J4ty7jdvY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2187" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2187" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2187" class="active" data-icon-position="" data-hide-link-title="0">Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460032" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rMZeXsWlNDzEbFbvBw9KIA0QpTR1IfagNpBe-7XRNXQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2187.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2187" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1LlQvKTa5R8Wkfj2PW-Seqz3OhjrSPHSgD1kmN17jug" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2187" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2187.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460032" id="node460032--2" data-pisa="ggg;5/10/2187" data-pisa-master="ggg;g3.115.021493" data-apath="/ggg/5/10/2187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marion Dubarry</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">A. Peter Banks</span>, <span class="highwire-citation-author" data-delta="3">Simon Cockell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2187-2197;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460032--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marion Dubarry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marion%2BDubarry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubarry%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarion%2BDubarry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">A. Peter Banks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>High Throughput Screening Facility, Newcastle Biomedicine, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BPeter%2BBanks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Banks%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BPeter%2BBanks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Cockell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Bioinformatics Support Unit, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BCockell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BCockell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">david.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2187" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2187&amp;text=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2187&amp;t=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2187&amp;title=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460032/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460032" id="node460032--3" data-pisa="ggg;5/10/2187" data-pisa-master="ggg;g3.115.021493" data-apath="/ggg/5/10/2187.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marion Dubarry</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author" data-delta="2">A. Peter Banks</span>, <span class="highwire-citation-author" data-delta="3">Simon Cockell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2187-2197;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460032--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marion Dubarry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marion%2BDubarry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubarry%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarion%2BDubarry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">A. Peter Banks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>High Throughput Screening Facility, Newcastle Biomedicine, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BPeter%2BBanks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Banks%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BPeter%2BBanks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Cockell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Bioinformatics Support Unit, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BCockell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BCockell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Cell and Molecular Biosciences, Faculty of Medical Sciences, Newcastle University, Newcastle upon Tyne, NE2 4HH, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.lydall@ncl.ac.uk" class="" data-icon-position="" data-hide-link-title="0">david.lydall@ncl.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2187&amp;rft.epage=2197&amp;rft.atitle=Genetic%20Networks%20Required%20to%20Coordinate%20Chromosome%20Replication%20by%20DNA%20Polymerases%20%CE%B1%2C%20%CE%B4%2C%20and%20%CE%B5%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dubarry%2C+Marion&amp;rft.au=Lawless%2C+Conor&amp;rft.au=Banks%2C+A.+Peter&amp;rft.au=Cockell%2C+Simon&amp;rft.au=Lydall%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460032/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460032/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460032/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460032/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460032/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460032/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460032/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460032/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460032/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460032/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460032/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460032/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2187" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2187.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2187.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2187.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2187.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
