<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2073" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2073.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2073.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2073.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2073/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2073.atom" />
<meta name="HW.pisa" content="ggg;5/10/2073" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ensemble Learning of QTL Models Improves Prediction of Complex Traits" />
<meta name="DC.Identifier" content="10.1534/g3.115.021121" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Bian and Holland. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Quantitative trait locus (QTL) models can provide useful insights into trait genetic architecture because of their straightforward interpretability but are less useful for genetic prediction because of the difficulty in including the effects of numerous small effect loci without overfitting. Tight linkage between markers introduces near collinearity among marker genotypes, complicating the detection of QTL and estimation of QTL effects in linkage mapping, and this problem is exacerbated by very high density linkage maps. Here we developed a thinning and aggregating (TAGGING) method as a new ensemble learning approach to QTL mapping. TAGGING reduces collinearity problems by thinning dense linkage maps, maintains aspects of marker selection that characterize standard QTL mapping, and by ensembling, incorporates information from many more markers-trait associations than traditional QTL mapping. The objective of TAGGING was to improve prediction power compared with QTL mapping while also providing more specific insights into genetic architecture than genome-wide prediction models. TAGGING was compared with standard QTL mapping using cross validation of empirical data from the maize ( Zea mays L.) nested association mapping population. TAGGING-assisted QTL mapping substantially improved prediction ability for both biparental and multifamily populations by reducing both the variance and bias in prediction. Furthermore, an ensemble model combining predictions from TAGGING-assisted QTL and infinitesimal models improved prediction abilities over the component models, indicating some complementarity between model assumptions and suggesting that some trait genetic architectures involve a mixture of a few major QTL and polygenic effects." />
<meta name="DC.Contributor" content="Yang Bian" />
<meta name="DC.Contributor" content="James B. Holland" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Ensemble Learning of QTL Models Improves Prediction of Complex Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Quantitative trait locus (QTL) models can provide useful insights into trait genetic architecture because of their straightforward interpretability but are less useful for genetic prediction because of the difficulty in including the effects of numerous small effect loci without overfitting. Tight linkage between markers introduces near collinearity among marker genotypes, complicating the detection of QTL and estimation of QTL effects in linkage mapping, and this problem is exacerbated by very high density linkage maps. Here we developed a thinning and aggregating (TAGGING) method as a new ensemble learning approach to QTL mapping. TAGGING reduces collinearity problems by thinning dense linkage maps, maintains aspects of marker selection that characterize standard QTL mapping, and by ensembling, incorporates information from many more markers-trait associations than traditional QTL mapping. The objective of TAGGING was to improve prediction power compared with QTL mapping while also providing more specific insights into genetic architecture than genome-wide prediction models. TAGGING was compared with standard QTL mapping using cross validation of empirical data from the maize (&lt;i&gt;Zea mays&lt;/i&gt; L.) nested association mapping population. TAGGING-assisted QTL mapping substantially improved prediction ability for both biparental and multifamily populations by reducing both the variance and bias in prediction. Furthermore, an ensemble model combining predictions from TAGGING-assisted QTL and infinitesimal models improved prediction abilities over the component models, indicating some complementarity between model assumptions and suggesting that some trait genetic architectures involve a mixture of a few major QTL and polygenic effects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2073" />
<meta name="citation_id" content="5/10/2073" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2073" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2073.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2073.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2073.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021121" />
<meta name="citation_pmid" content="26276383" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2073" />
<meta name="citation_lastpage" content="2084" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yang Bian" />
<meta name="citation_author_institution" content="Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="James B. Holland" />
<meta name="citation_author_institution" content="Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author_institution" content="U.S. Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695" />
<meta name="citation_author_email" content="james_holland@ncsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4341-9675" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=E. Bauer;citation_author=R. Kohavi;citation_title=An empirical comparison of voting classification algorithms: bagging, boosting, and variants.;citation_pages=105-139;citation_volume=36;citation_year=1999;citation_doi=10.1023/A:1007515423169" />
<meta name="citation_reference" content="citation_title=Molecular Dissection of Complex Traits;citation_title=QTL analyses: power, precision, and accuracy;citation_author=W. D. Beavis;citation_pages=145-162;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. Bernardo;citation_title=What proportion of declared QTL in plants are false?;citation_pages=419-424;citation_volume=109;citation_year=2004;citation_issue=2;citation_pmid=15085262" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. Bian;citation_author=J. Ballington;citation_author=A. Raja;citation_author=C. Brouwer;citation_author=R. Reid;citation_title=Patterns of simple sequence repeats in cultivated blueberries (Vaccinium section Cyanococcus spp.) and their use in revealing genetic diversity and population structure.;citation_pages=675-689;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0066-7" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Bian;citation_author=Q. Yang;citation_author=P. Balint-Kurti;citation_author=R. Wisser;citation_author=J. Holland;citation_title=Limits on the reproducibility of marker associations with southern leaf blight resistance in the maize nested association mapping population.;citation_pages=1068;citation_volume=15;citation_year=2014;citation_pmid=25475173;citation_doi=10.1186/1471-2164-15-1068" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=G. Blanc;citation_author=A. Charcosset;citation_author=J.-B. Veyrieras;citation_author=A. Gallais;citation_author=L. Moreau;citation_title=Marker-assisted selection efficiency in multiple connected populations: a simulation study based on the results of a QTL detection experiment in maize.;citation_pages=71-84;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9607-z" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Bagging predictors.;citation_pages=123-140;citation_volume=24;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Stacked regressions.;citation_pages=49-64;citation_volume=24;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forest.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Buckler;citation_author=J. Holland;citation_author=M. McMullen;citation_author=S. Kresovich;citation_author=C. Acharya;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_title=Multiple Classifier Systems;citation_title=Ensemble methods in machine learning;citation_author=T. G. Dietterich;citation_pages=1-15;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=B. Efron;citation_title=Better bootstrap confidence intervals.;citation_pages=171-185;citation_volume=82;citation_year=1987;citation_doi=10.2307/2289144" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Elshire;citation_author=J. Glaubitz;citation_author=Q. Sun;citation_author=J. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=J. H. Friedman;citation_author=B. E. Popescu;citation_title=Predictive learning via rule ensembles.;citation_pages=916-954;citation_volume=2;citation_year=2008;citation_doi=10.1214/07-AOAS148" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Glaubitz;citation_author=T. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=J. Lu;citation_author=V. Kishore;citation_author=G. Gay;citation_title=Evaluation of genome-wide selection efficiency in maize nested association mapping populations.;citation_pages=261-275;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21938474;citation_doi=10.1007/s00122-011-1702-9" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_title=Handbook of Econometrics;citation_title=The bootstrap;citation_author=J. L. Horowitz;citation_pages=3159-3228;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=F. Hospital;citation_author=L. Moreau;citation_author=F. Lacoudre;citation_author=A. Charcosset;citation_author=A. Gallais;citation_title=More on the efficiency of marker-assisted selection.;citation_pages=1181-1189;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050679" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Huang;citation_author=M.-J. Paulo;citation_author=M. Boer;citation_author=S. Effgen;citation_author=P. Keizer;citation_title=Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.;citation_volume=108;citation_year=2011;citation_pmid=21368205;citation_doi=10.1073/pnas.1100465108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Kump;citation_author=P. Bradbury;citation_author=E. Buckler;citation_author=A. Belcher;citation_author=M. Oropeza-Rosas;citation_title=Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.;citation_pages=163-168;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217757;citation_doi=10.1038/ng.747" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. Sanchez Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=The Computer Journal;citation_journal_abbrev=The Computer Journal;citation_author=J. A. Nelder;citation_author=R. Mead;citation_title=A Simplex Method for Function Minimization;citation_pages=308-313;citation_volume=7;citation_year=1965;citation_issue=4;citation_doi=10.1093/comjnl/7.4.308" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Ogut;citation_author=Y. Bian;citation_author=P. J. Bradbury;citation_author=J. B. Holland;citation_title=Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.;citation_pages=552-563;citation_volume=114;citation_year=2015;citation_pmid=25585918;citation_doi=10.1038/hdy.2014.123" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Peiffer;citation_author=M. Romay;citation_author=M. Gore;citation_author=S. Flint-Garcia;citation_author=Z. Zhang;citation_title=The genetic architecture of maize height.;citation_volume=196;citation_year=2014;citation_pmid=24514905;citation_doi=10.1534/genetics.113.159152" />
<meta name="citation_reference" content="citation_journal_title=Artif. Intell. Rev.;citation_author=L. Rokach;citation_title=Ensemble-based classifiers.;citation_pages=1-39;citation_volume=33;citation_year=2010;citation_doi=10.1007/s10462-009-9124-7" />
<meta name="citation_reference" content="citation_title=SAS/STAT 13.1 User’s Guide;citation_author=S. SAS Institute;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Schön;citation_author=H. F. Utz;citation_author=S. Groh;citation_author=B. Truberg;citation_author=S. Openshaw;citation_title=Quantitative Trait Locus Mapping Based on Resampling in a Vast Maize Testcross Experiment and Its Relevance to Quantitative Genetics for Complex Traits;citation_pages=485-498;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166171;citation_doi=10.1534/genetics.167.1.485" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Stange;citation_author=H. Utz;citation_author=T. Schrag;citation_author=A. Melchinger;citation_author=T. Wuerschum;citation_title=High-density genotyping: an overkill for QTL mapping? Lessons learned from a case study in maize and simulations.;citation_pages=2563-2574;citation_volume=126;citation_year=2013;citation_pmid=23860723;citation_doi=10.1007/s00122-013-2155-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. Navarro;citation_author=D. An;citation_author=M. Romay;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_pages=3;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=C. C. Holmes;citation_author=R. Mott;citation_author=J. Flint;citation_title=Mapping in structured populations by resample model averaging.;citation_volume=182;citation_year=2009;citation_pmid=19474203;citation_doi=10.1534/genetics.109.100727" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Q. Yang;citation_author=Z. Li;citation_author=W. Li;citation_author=L. Ku;citation_author=C. Wang;citation_title=CACTA-like transposable element in ZmCCT attenuated photoperiod sensitivity and accelerated the postdomestication spread of maize.;citation_volume=110;citation_year=2013;citation_pmid=24089449;citation_doi=10.1073/pnas.1310949110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Yu;citation_author=W. Xie;citation_author=J. Wang;citation_author=Y. Xing;citation_author=C. Xu;citation_title=Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers.;citation_pages=e17595-e17595;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21390234;citation_doi=10.1371/journal.pone.0017595" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Stat.;citation_author=G. Zhang;citation_author=Y. Lu;citation_title=Bias-corrected random forests in regression.;citation_pages=151-160;citation_volume=39;citation_year=2012;citation_doi=10.1080/02664763.2011.578621" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=Y. Zhao;citation_author=M. F. Mette;citation_author=M. Gowda;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Bridging the gap between marker-assisted and genomic selection of heading time and plant height in hybrid wheat.;citation_pages=638-645;citation_volume=112;citation_year=2014;citation_pmid=24518889;citation_doi=10.1038/hdy.2014.1" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=G. Zou;citation_author=G. Zhai;citation_author=Q. Feng;citation_author=S. Yan;citation_author=A. Wang;citation_title=Identification of QTLs for eight agronomically important traits using an ultra-high-density map based on SNPs generated from high-throughput sequencing in sorghum under contrasting photoperiods.;citation_volume=63;citation_year=2012;citation_pmid=22859680;citation_doi=10.1093/jxb/ers205" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Ensemble Learning of QTL Models Improves Prediction of Complex Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Quantitative trait locus (QTL) models can provide useful insights into trait genetic architecture because of their straightforward interpretability but are less useful for genetic prediction because of the difficulty in including the effects of numerous small effect loci without overfitting. Tight linkage between markers introduces near collinearity among marker genotypes, complicating the detection of QTL and estimation of QTL effects in linkage mapping, and this problem is exacerbated by very high density linkage maps. Here we developed a thinning and aggregating (TAGGING) method as a new ensemble learning approach to QTL mapping. TAGGING reduces collinearity problems by thinning dense linkage maps, maintains aspects of marker selection that characterize standard QTL mapping, and by ensembling, incorporates information from many more markers-trait associations than traditional QTL mapping. The objective of TAGGING was to improve prediction power compared with QTL mapping while also providing more specific insights into genetic architecture than genome-wide prediction models. TAGGING was compared with standard QTL mapping using cross validation of empirical data from the maize ( Zea mays L.) nested association mapping population. TAGGING-assisted QTL mapping substantially improved prediction ability for both biparental and multifamily populations by reducing both the variance and bias in prediction. Furthermore, an ensemble model combining predictions from TAGGING-assisted QTL and infinitesimal models improved prediction abilities over the component models, indicating some complementarity between model assumptions and suggesting that some trait genetic architectures involve a mixture of a few major QTL and polygenic effects." />
<meta name="og-title" property="og:title" content="Ensemble Learning of QTL Models Improves Prediction of Complex Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2073" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Quantitative trait locus (QTL) models can provide useful insights into trait genetic architecture because of their straightforward interpretability but are less useful for genetic prediction because of the difficulty in including the effects of numerous small effect loci without overfitting. Tight linkage between markers introduces near collinearity among marker genotypes, complicating the detection of QTL and estimation of QTL effects in linkage mapping, and this problem is exacerbated by very high density linkage maps. Here we developed a thinning and aggregating (TAGGING) method as a new ensemble learning approach to QTL mapping. TAGGING reduces collinearity problems by thinning dense linkage maps, maintains aspects of marker selection that characterize standard QTL mapping, and by ensembling, incorporates information from many more markers-trait associations than traditional QTL mapping. The objective of TAGGING was to improve prediction power compared with QTL mapping while also providing more specific insights into genetic architecture than genome-wide prediction models. TAGGING was compared with standard QTL mapping using cross validation of empirical data from the maize ( Zea mays L.) nested association mapping population. TAGGING-assisted QTL mapping substantially improved prediction ability for both biparental and multifamily populations by reducing both the variance and bias in prediction. Furthermore, an ensemble model combining predictions from TAGGING-assisted QTL and infinitesimal models improved prediction abilities over the component models, indicating some complementarity between model assumptions and suggesting that some trait genetic architectures involve a mixture of a few major QTL and polygenic effects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Ensemble Learning of QTL Models Improves Prediction of Complex Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZJ4qIsTH7bBFa9zeohEb4CgUDZcsaLs1yqTSwPDBnRA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460047","apath":"\/ggg\/5\/10\/2073.atom","pisa":"ggg;5\/10\/2073","ac":{"\/ggg\/5\/10\/2073.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2073.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2073"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2073"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2073":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BtZiBf2CiwzLHvz4vz\\\\\\\/dho3Gk9Dfv6fRvS3+\\\\\\\/7KIuPsaM316wWTcoBmepYgBGuUvwgIoDJcBi1Zp7BZKydVnxvw4VkoBxR16RmXINxW4AvuUu6go7RE3qj6zfk1HKMP+F3h0neegkuOG4\\\\\\\/3nkDfcKPApptxjCPFFYEUsoARzTZCI3g+Er5sXh17GVxy9aOpdZpFaA4H4+ecR4GFTuY19\\\\\\\/Adg4LPVwcZSGjAMpmV0ie3YLieKJiZzG1j8K7A5gSAre7qR+7PS3hPUQyrkNCgm6O2Rb+r\\\\\\\/I9E62ifcQTaZ9tTatjhWsDw1q1+yUVGioKQ6w7D1v\\\\\\\/DV3t5KqMDe2pm9ueLzfWBY1l58zLvtkmftBw5MBg8gcp1LrDj4RgZt\\\\\\\/Z7fVFY3vzGBO3QQdIl6y\\\\\\\/OXdGDH4n1YwJDjkfW+Ijv1Z9I8K2JISOVyM55NVN1ZPCxSMBFLh4bXjBt3XL7YWKRu4gFrXY919YnNLPknLwUb0QRVT0L3ojpkvfbJpC20iM5vP9i2CTAabchp2JIbM5d+6y9mULiS0G1rh0kQzG+oJrahXW7lf4HNnN8DHw3fLzoJC16NUqJO9+Zx8Mpb42NlejTTx++f3fWF7YeKQwWk3qck\\\\\\\/FOju0ol2VzWqzgUMZD2NZ55fPeGV0OOdQbFFyLikOZQ1ny5rqif1vb7eQKtnK2dW8SDuVv7fRt8HxTEzE6CPn3Qo5ybl9sX0thCWIyzFUcX7IBF+GTux11bU7TRrwJnbQxAmPiUq9cdhn6F0O9d99nC3MNUBoqtB3libbMf0PYKoShiyZ+FOu7aMkJawBA0AZwFOO4aB93jqCzpHTZW+6Hk2SpLWFg4MTWuZMNXhAL47pDYCYJwIQaehQKiB\\\\\\\/JKk0HOTyGyLhHFr27X0JoQQ2RJ9dClFuXeVb8FmcJlsxlkqBqyozCOml7p+c5nBnB+JPebWEOs893\\\\\\\/rKo5FH17GvkafaEbkdBhudNCT7lqQY8DDx1GWBwuqIF4qeBRafA\\\\\\\/mmwXDpTF4wqafJou6i8E+dXYOZVxObhJOsK3cfcKAhq1dr1qlJriMv2aNmqFjBGQooHvwaCLAYNO+W5FMcTnA\\\\\\\/5jb+Cl9pJ49ztA7wxhI5l63uUqbpqEYKfjjH3xyC9nA77oNGPjX89eCi6f0x1WY+lzpEnvi\\\\\\\/ctBMTpx4QnqNoA003v5tlX20FcgPH3RCiUK97UsNOrg17CWCbOvEkE2ZuGqyUz6cTHrDfdwXi3Pr1QG18cfVCWp6G+QG9frpvvIJKR9\\\\\\\/uC+MmjOFMyNYYM\\\u0022,\\\u0022iv\\\u0022:\\\u0022PizohsRT+igFpZcQJaGodg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022681d1d5c61df195d32c3646a1226e853fe0f1373d2639164a2cfb918b94b9c6e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pJJEU5hDLEz1NPEwTtHMag5yyMdW3ife6kMblZ9Q3NQDhe+MhX5N20dnkPBhpHa+tdiERX20CzSgUPFr8ru9TM0SI2JbY4F20k6WizEKeu7sG+z50h+vgCmEs2T06xLl9gyCcx1ddyodXRifvN2bCNPrKxY+5iEDvyHbxzVwqJQu2tx73coFoETbR0aMzuD859OUeXDpfrKGa+9P6CRRDvlCuVM6qfDcB627BinnHqUdChHGE2A2E8vnEnNleaKvXRzLbicCc928eE9G+lalIJGkKErLPcy+GrHteZe7f\\\\\\\/WMZTPjgzLvBf6xjKwcWcs14eh7SJeU\\\\\\\/H8EuRndbSVMcg7+3b15FFzzncO+DDbgWZgk9HX9u+\\\\\\\/9whXtyGDz\\\\\\\/RJN4VyrsfRdnM5+ePhOnmM\\\\\\\/LobVKG4MELLS3fBznOTwA2aSbdXJgRsfzvqlayqAjSvyJU0mwxDk0IyEU7MbYiRSfWHl5L+4YCeKac3f9sFsQSxtUZfferFEP6edr41+Yr\\\\\\\/zyLPY9NeiPaNMIvJequLDQGmVrTUIq5MFca3zxDCGIXEWiN5c2ccAosL5UUx7MXEnPXw+6w6iNgu5\\\\\\\/rM7ORigyHp3BQM86uqRhIxmKH5GQVMWBWU97CoyB6ZhlX3jZRv0oVSh6\\\\\\\/zH93n4jjuTocjSUwp9v06U\\\\\\\/+gTY3sDRxdH2tXW1FVdkuTvNvJyPYTarL42YGKC1MzhjSh6SXronvGMkIfcRMOtMNYIb9Fmp+6b6VHhZ0eODbdptG0WXQ6Yd2lj3T5l23+kI8vnjfL6ka4QvQsbeSDQwazbKbJFdJtr9pnzQ1y0bGEj+XgYMv9I7Llf3b9oNHDRLJ6FXLYZnTY9GIu3JPSJP7qhbpKoQCx1fbmfwKyqFQLHRytsIF4fodPz46RsZkp8Ba7P74JZOA8precr2W3jtCix7gOQ1YhsGG9ngQ19oQtmcJ+0O2Zrb5b+DngZFiNewQEGvPSSbq1xnLQOLrGrIWH52wD9bW3vEAHB\\\\\\\/6NYO+9BTTZ+BNw7s3P15VBdNNSuG048S2QwVXN7Vz3e3Nwc1F44\\\\\\\/515RO8HMqd+aXHoef9\\\\\\\/Pd\\\\\\\/WMiHUIe8ZMgnEwGtP5UBX49kOW0+iFiRJLFzAu7MS4P7qbvI\\\\\\\/K6TQyFT0BDZmai8HUoiuLngnjUrM9GN3mAbSbXgQYhu5tfa3d4HAd63\\\\\\\/BYIMzWk1m1G45TH9bs0dm1HBWKhYRJCjSRLlyFmW\\\\\\\/5nVBqUii\\\\\\\/tmyw3EU\\\\\\\/z9s6VmNcZbSnh3W5uW2V2Eed0\\\\\\\/5ODKNpZMaRwlsR9puRA3yMnMti4ayl1advK154kI09hkheBkW1Ozcpsl6BtD2Muz\\\\\\\/KEH6osakY12AHSxxktwhYjjO5mv+vAO60pIzuZf4\\\\\\\/VgIoTFaf2U6rOCJREVV\\\\\\\/Mc1l0\\\\\\\/OZTscFoMvUFDLGJlBaVepis6Y2Ha3WM8lZOYkJ0CSSeeyPPkMISOIuGImc7OKqBaz0GUmKHFvx7Bovpf6jDh0XOWyWr0qe7BYFqsfTHHy7Ys+pB87j9djZfrmfKtQdABP0Z\\\\\\\/hrn0TPrVhDmOf8hqB6DzsgD6fneZrlXCJWpMA4GXULM3vGgGjydL4xp\\\\\\\/9iQcFx6GnfKPKHewCjrKlRDZ8gJylSJkKepFg+yaB2tTDe+Pz3t+iRE0iS\\\\\\\/\\\\\\\/QDFwo25BWNoTHRU4jqrcFXFXFZgM1jWt4Tgemz5QzQpuH5sawoPryQ5edU6617PEf6Vs4LZaCOex6wH\\\\\\\/1upm+j58+tuPDxeeFiGFWU157EpQMQ8Zfm76QvUpC0\\\\\\\/zjVlCUr6U5Q3fd4B2uGkHfZwyXuZnhT2Px0DAmeBmeBRuJxb6QAPfODcEvxegOzfRxRrqrbsZL\\\\\\\/C74ngBBRX7xRSN91gZkcil4E\\\\\\\/+kmlMU2PQiL0+UMDQ5vCnUyW3T836ITz46SUDw0IOnoPdBH2GYEXez2WefodSibauwKlwtLGGxA4WC1aSgOsBs4jeBF\\\\\\\/CeoEWAHpO74vzaYeVxMZyOpjcZalxgUL8gb0HfTC6\\\\\\\/rl4cEW1Mbc2zoL2xLO5SKjAmP6Rf\\\\\\\/PvsWPWqfS21AENyOOCTXj7IEg\\\\\\\/N+rGfpF8wt+nyl0cowl4vVkX0DgBMzgWciOo2R312x1c8fvqpUWuQKYlsrGPoxWC+15+gQaE38vO8itLUxwh+q24ngfroNUZWcGOrs9L82\\\\\\\/8+QkkvKpRLYTeDZCHnfXABhcBdMV5SD3jG4NfYaPWefG1zWZ0E5Cp2NiHXOPt+vAZDXqMlCtUCF3iFEJ8WcSjujrGi2VedQJfCxY6ZXBL2hbJqmerTZEyOGeAfRXwZQD5MW9yykTOWaFEtrYeNRj1GFTekFlOMFdMsFRv+J+fM8TeHHLbs\\\\\\\/TYKx5XAbYp6u5rUBfj1jYCkGaQXUZTh4jnWuatMA+E++8vaV8j\\\\\\\/9u1tH5DOMFgqWwChHEEhYKWflPfbXFllhD\\\\\\\/39jhdsUtz6XkRArpU+VjYsW+Jb+3Q7yelJD\\\\\\\/aO3Hs4my4Vwwyl+36StpDtkX+OflCpue2KgmZzxnUQ\\\\\\\/ThSjDQptgatEkfnTKPl2lh+YCbo7g76yBZFi62n74quil5pXoiEhwHkp780v31JKal30H4KdKISWfYC3o5M+xIH9\\\\\\\/nPtLyVvqv09upl8bxSYshpbTYiisKCiRMoh9\\\\\\\/H0mtVaJeZ8GJy9YYJ8i+BFiuhRVN1gDz15k6amz3oHHItSD6DliStFyFTsXQwosazZTj2Parqct8c1tNLCJB3sIOGlG90Aa5xrpQmTV\\\\\\\/Q5oXqTzaodrp63g6qj68J+pkD1pVHrbQaZIIvfDLp6XGRv1kXi09KqT+x+Ga6TEW8O\\\\\\\/F+9ZDpV9CXptQXeOQ2qnupupD+gmBKVOJvnEYaXwrpECDkaRdSipyVcTCIpYQiO8lkJ8Sp6kIdVhUaZeAeNcGYF5C5tQA1\\\\\\\/P1pJMracUEhOl1dVDTPSorSIqydDkkwhT4GYQCgz+rE7LmHaZVR6RGyQlqWHVOS19K1cqEOhT+1KThkXn+HmJWNpslAfV9Z9suXW2cBQ915pHocjCbEVKy6BOipnOlBFX0hdHQdwkxEXEGN8QF19hx7S+587ZTtNrXUXudDZtOMuxNCOUH6pdfVbchfl+4u9WQxZdhYnraH2R9b2nkArLQPHw62Y3uPje0nUqOHm\\\\\\\/4Y1URhAx6jIPqFwBq1TVnmxWFee5F8tJe+40A23q8fGdWFgI5y19lGltyFt8VjNoTmN9m5oG+ODysR0RJG4KYXC8DRkWo+HyG57gcVsVFu8IIovZF5ds1iJgxHh58ynbcvB4fAlOD2NO3uNAKmQJA6ZrsQYs9ccmY77zIdPXGZJXKA3yxIZNB0a93LNH+ngFCQMAg5xXvO6tYPt7fFbTQvpjZ+HDh1jzPKa\\\\\\\/tTSSM3TAHkCRJvZRRLgYHyQOZQsfzMUt69Q2yFQEry8fGDIbGxVfmZTeMW5klVT\\\\\\\/iGVwbjskC+FZw2YAnLjANy2fD3QVbehImeWTkW0GF91PqIyQ6LWnWxAUn8taHW9m7LPufsurAqQySUYfO8vmn+bYtmuXMFVMzrTZrGAU8gLnwThfGvY+V87OrN5YN1iAdySv8T6NEp2w21hUAHhHxhiQ83lVb4nOkF+HFq1IxoRCig+3\\\\\\\/RgR\\\\\\\/7ySBCMds+LVyoLPorcNCsCNulTxnPN+XAKzGO2kyj0892jkaW+07arLmL7uUvOh+Ush7i20C8DImEmF00EPjoplqYN9UFAJyG18\\\\\\\/\\\\\\\/JHSELqTNPbH\\\\\\\/JeIWMvTxMIDBk+TTY8h9YqPQUTE94uJRnsWkPSFDKNNQdDNbW8CVYZEU4cyESt9gWcUS0MCOvPa3gXMIOCpQpiArO2uPzJp+JEsXiDjpiJ6I5EndZ7qgdDLq0glIP1Mg6nhObO\\\\\\\/X5AlwVyXjoSgX30QqVorQrkRwpipGSI0iSMa6aETGjWoequuRlFPvrI70AFUkizcN4pf9hSQCGiPdoxvnDXJ5PaMrjJwrzowu\\\\\\\/CcMy3AKY3y689r6UxMG6dnrzmZp20oRH8Y\\\\\\\/4MKbd4ssQCicfs7Ka7ln0rPyM4euYaBXpT5Red6YwXhK+dKDF2IqSz5FMfX+PAcShBrjkb2LbDUuiAKYtDExp8xzHYHNTkHipdD8zzEpn714dQqKlF8K+qB+x11RDwS5QbcTGODzbLgyLG0haWazTUhk4D3AT3\\\\\\\/3b2Lk57ktKs4GSFOu3xbT1zv\\\\\\\/MzK6ozqTCCIkEDAs20rI25uquQ9Q2Fkd4rPhsospbmk1lHnJbkTj52SFd5bpnJfVRWrNimUGHfGIIJu5opq+MUwJW0YxRz5fZmONbjWpZBlfvV2QdTuKEbT03SFqgYMInQxD7JmFg3HcPUCEz7IFkBvyDvDRKFrchSWbW4tkAhXYLr1s\\\\\\\/75h6qqgLziGY4RiHrbi66Pg0TaqiKPeB8C2YHgx5+vLWOPiEHBSwbqW2bUdEW0qB2rCBY5aV\\\\\\\/eWQOXqgikwfd7pOPRQN+iSN2ARxI1L3R69+1cLe66qbaW9Td2FqdCCgGcKpnFqu6aDnVNRPM5qSuJneJBVLYUAX53toiBSc8wFc8VpB0A51fqm8WQSYSCVkLnp7bBaUOV\\\\\\\/dtlGGubsC4RIMNl16eqIbrDjn5sU4PT1AokFMaF9QAFYr4QU2EiEPKiB07NQi2mEOL32ebLgYXGPhMeAqbDmc6dhyvsJTCZhfZ4nsZORgtGBT6lcksHSSwBNklibGwxaGOPlAbENbM9qS86M+1y9SPvbdBla+utnF1644MGMPYTtglVmbpZzFqEVNxC8otlvV5hHgQv4aykzGvPvlPnQvn75O9pLhZ4dM14LVjl0eYC1aQD8q0i0ZBM8zJ1p63djl3unCLXS4nVcRi3Gs0jcpqCg8C+PcRbpxqm\\\\\\\/b8Gj3Y2cHTJQxXDRc2xPpu4WWVuEdnBhhssAwQpDlTO9gOf1Z+XkdvrCPnQZLygEOlaa8FHYEwy\\\\\\\/OuPEGzUqc3Q53CnHOcD2soTWnhdj\\\\\\\/5QnHHleDVynVAkrMtwHpZdBor6zVaRcIuwMs6nUQV6fjmpy6NsKynLWwc6ui06I9umCWsjed5FN3ErIfhKkC62w18NbqxPc1dgKqV4SBL4pUIkb8eMfPx13J6aJodiHvMjIPe2EO9neRsff4lCk8GwXzfXzuatjzoVLfCIHN1wI5JI88noNUOQq4h4WJlmgmjHl4ae9lhiY7ioKkKXJIMqEzAIX+I16yuxKM+On2EkTmtJrQWgM2tF8KA2R1C08YP9deUBfRuWWonfoYVwPhMx73PRR7H8aPQKRBZOn54jY45gL\\\\\\\/SzvrQZTvuX6ILtBkWo5C\\\\\\\/sYibes4AgbXs31shNVe3FpF2Iu8QMzqCjF\\\\\\\/96KxbJY\\\\\\\/k+psX+PToNJaj+m\\\\\\\/3O4w03wrShg49wLEpEb6D2rqbnX0MwLTVCC6HoMVkMouP83Fv2sPs3Vs7q981\\\\\\\/QW2bKC4w18dSmxLFXqNwwkehFR7JKbMmofThmYYQCJpGxSoF0ZW5YECpj0FcG3AjH14UHR+h67\\\\\\\/SEXEpyeVrxAM+Ye+jBHzc7EWo8\\\\\\\/9xjsegBxuLgBSSgp96wUFzSNMDD5qzh4Fgz+E8GC9wcO\\\\\\\/W5R3EVOuymnknA7ZD\\\\\\\/djvb\\\\\\\/b+Dy8+77prvEe0yEVveUeDHve221TmmV3JC\\\\\\\/ySJs0REHfOJZi9JWu7sPWE1ZOHtFGe4SZ2RaOx48YlLN7kbbRwLBd\\\\\\\/+sYEpWflf\\\\\\\/1d8OQhowIqaSn2v8VAIaeEDcKW6\\\\\\\/sVcKCljq5asD+\\\\\\\/qXk4Yw5fxp1ApBSg++NOyq6zcKffmaK6iihh0XhOAaqcJ1oD2WZScMZAYVdi\\\\\\\/9qGAZzBkm7VBMxkwFLDkiNWDAhoxkgjDlI9UZP9eslV6eHlTj4VLLPFqOung411Gj01\\\\\\\/rGITmxn0njaXlg7katFUyD5YqveltAB76MKJglWQWSg2v5\\\\\\\/0hi6aonwIl\\\\\\\/l33fQn0eAWF9WgFuWAmzF\\\\\\\/Gb+7v2rysGGXiezLVPYRU5wBZdhHeu057gRXXusS3KE3M7pbcs32V5+wjVfh6YPa2x+ZnGbaYF3j6dVcNGtIPCO9yOo6zSVAoPv+ltFurz3QUYcP8V8SLkU10h6CJC+6MX1X5kQEgMad\\\\\\\/Tr8kBGHaQLtehFw973QNB99ojp\\\\\\\/RWr3j8pYywlV0D+\\\\\\\/qipoXibmT3iBu+efrLEPE4ORmMQMA7BWbPhs5O\\\\\\\/pV+FilzrimcOg5Jt+7E4WcqgmwzWF\\\\\\\/1LreykBqyZBJCAQolO0AhCZ\\\\\\\/immaXgUykCs3ix6eWG6EWudv8iwIdA1GqcSFIZdtLkoSOu5wTL\\\\\\\/owhAbujHyLWKsIV0s6NtABjYUX9QE68aorlP5\\\\\\\/XtyNEAkqHxjuNzj1H7bbKagotpoOEpTNz8xcyQPZP3ips+fcsHezdr+0pLNPbwLheSumRjj76\\\\\\\/uxlEpK2SGXVx7uEyxqbjELK5jynzETsLztRLOLTTsSO0OSTDHSrYJWNbfLVG4g5LF+MWyU0S6hg6g1heuMJiytFf40Irc1YB9qk9Mvdioeiil1j+ZFbxFkKQ6q\\\\\\\/LCO6zpOCkEzcu4g1CZ2wvgTwWhJUx0Bv3Du2vlDlWSeR22Q3p3GfS+weVN2xY0oTVp\\\\\\\/y5oQYZlQvQFkBE239mxN8qROSbdSbJjnGvHfE1NWEVsh00UGGmx0ChuYNlExX7l78wtYD8qUp4EKQZRkq6ysaALsT2yRFJ+8xHsUF2jd40oVteNRJT5IgA\\\\\\\/bPtiymXcv5fOiPKornpYBWvzH6+D\\\\\\\/0LQwhSr1u\\\\\\\/nYIPC4Hma7movNDphahAn+QN9FZPPJTUVx0Vs+HXmWFpzfI+5AQQG609UlExTIicieBnqmlFtE7mJBgLhU+R1st6yS2+UIFBUyRYmrU0BYGBgtt8MVZ3\\\\\\\/fEC5014Nq\\\\\\\/rhdNxFhq6osSv5Xz3ebcPBzqzZR5lHWNax\\\\\\\/uTG1j3bIckJPpy5Tz+nOZrZEenTTYB3zT2o9vNeLFhqkA\\\\\\\/5Sd\\\\\\\/772iWaHnIGRc1wO2xkKDRTaErhOS+\\\\\\\/yHEECYu0v0Y2xrUeCqmVfF4OfXlmDqXVxJ+WQgFttRKLIt7byn4nqNGZhGUhftsh7\\\\\\\/ySjFTYRqQxZCCZp6X1S8bRolCMyA488ylGOx4JZ4glI5JaBW\\\\\\\/UnAA1vQz1Aqe5h08WheLTSBCqGf9OZ7ZXLPTnYDe5NXzOAwzBHcZ\\\\\\\/zaLMYLRoMYCTwTT1\\\\\\\/fjhTV91T3pcu7DXDNkaKBRvxdRl65SIRy2WRBXrifInB1vi6xexlLEy1J5qQOJgmNRpehWV7Ds\\\\\\\/4D8LJrkH+p2qqthgAm8fWCtaOyEvEyQ2SXWFTDg+K\\\\\\\/5eKyKIE0YIBp42PrCu+k5h2lXExkEqVHl89o93XantpeiV4Qspdbanj8ArhrHcVAYZmPjsZT7Z1zVZqaYa0XHi0z6P\\\\\\\/DO5dOpY+SE21fMnAZ0Y8v913WySEvDAIPpRIqxfRpZGxAX070sMW\\\\\\\/a09WdulYcyHJOmBMKmD+gJ2w6fXXRCPyeIhsf8K1\\\\\\\/d4TCPsrHyU\\\\\\\/VKHVFJPpQD9WD0cyLV38szMVlNn6P6+xswj6PKfnWTyGZtUxzaGiSLFLIs9AiMabpwQ1JhHUWdee+3wln5l2EFbL\\\\\\\/6XSGc1uIF0NF0UjUK\\\\\\\/PivO6W3jUU2P7i5u4P+yL07DLk1tJV8XyqgqphLmhlh3PNbGaxKh6Y\\\\\\\/nk31R2ba8OBxfY6qzkJbffVYh8Qo3MdyJblXOpA2FP\\\\\\\/cVWHzHEHY9Afbz8RNgej\\\\\\\/I5nmSHUiBgkm6LuOPzz4JQxdZpG+JN7dfZhdHTGQBuj+eJMUwgdm3SnhEqC6do8wCBXcI8sXPJQIveUEaCX1hoY2r6QsjOqlZB9zigK5DGRjwwfPwEmfPo\\\\\\\/saYtYaivGkhnOJKCOp7CS4ljolNQXmPn555CPo+Ntj+eodKP6fFIXcdqle7kJdsVhVXsXXZqfvZEgIx\\\\\\\/Xvrim+1OCj1IZF9rCft9Li5jIwlP5A0cvukiUks+4qmy3RqStB48uS6tjPABpUj3TnQgBFNjxE29gJQHIo6JYUlrCMq3boIcJ\\\\\\\/IZaduS\\\\\\\/0EAiRjYUCCiWtiye2cZaI6K0UEuFn8nI17uliVRXRjO8dNwIHy7oVeGIBf9lS5tVtV1MRXDPqpvvc9Kj++\\\\\\\/j3Ou+plg4V4FcAhqrP6nPrJ8KyHzLOUmm9nsMNGPpbx3xzAc6kfqJl5CDDikAlRbg0DFPZ1KomTlvdFc1KaveSVDBv\\\\\\\/G8EMTPmxfdjDCMFAqY79OTU6pproIg6\\\\\\\/qdFz7vlDCkHSo9w8ua0IqLiA9TLcMKTrF\\\\\\\/fcpjhyHPuW6Mk6\\\\\\\/MjoSptvClf7WwmYa0la3GLbPXX0KLuRoTh1AynUQ6Ium3ONIHTP57pRTYQcSGcQktZ6QejigOsT4q3mGJ0VA1UMd88E5Cque43EmPfNMsCUq6ISWuLMiOug4SVy1RfrraBTb47D\\\\\\\/5odppgr1pjg2mnYIvSSncyl2a4izLRZ5aRxjg\\\\\\\/fRVOuxPK0sFMC9CJEUG45mvNP4U7a2i+iJK5ERwJQ2N2i0Vz6Cu6CoWntCRdyimluIoQwhp2fm8tVut455J83OspjDEiYktyvCRpVn1Eo5qpqi32iXk9Fy1ZrYHXr1KIlgNyye6KpEvDQlpQ5ukxr39gUeA9LR60uZnN20mLnQwvAvfzBR0kFcBd0WP4s0gXsp7WFiBvfn70Rn\\\\\\\/8uIau5JJygZH6nMis7WQzhR1MIk4I4YlMq3MYWxIh4fVeHqaNEuzccfMvUP3UVg8JZMt1dnfXX8C84j5PXxFQ71EjP5Q3YgM+V4aXWKwZcLTMQl+Z7esehVtKFOER48T0PsFK0dlLeX+hCerFB93GKQWMiO8UqsHSnl2EJg0jrak5Lm7TpIWa43Skg9pxEZ5jfFgINyjIbqrTh40Ma6zgZE1RJRQarC5dktLKVe9wagZ3jTrAGsGlOVyEVF0E0GX\\\\\\\/8j0cSBQPFjqh7PbDI9pB6em9XZkM23xYhcxCnEwZa6m566eR\\\\\\\/tjpNPalxU6ErMHWDsUoLYkcypYaCyVNUSGoqgSspKiqCmUZOI0XvPrtZukQskC3JyTnBIVtACtZD4CqJMzAyTvipthmev1+foNdNdnqMFSvqyb9NU+7BqE9o3SNLLE1Li74MBkHoKa84C5Lv7lXnHtvV1mTWV10OOJ9W+LZHZ3ZHxTr+ziZHkQSqShwznOKJdM2MhT2+pHkCTzgDTUX9KSHx7M3Z5BbI9exrSOMJLowr1xtY3FnY1\\\\\\\/fNM74ad+BsPSTj6ZXwnsimjgwOCb19siNISxlSMEz07FHQHeA2JjPBIe22WODLmfIM\\\\\\\/zpde5xCwfHqRdEX+PJqC\\\\\\\/+3TWG65hc3mMM92u8ETZcsDb9oCAJfpDskdX87L0yxmIhMPTIKABxfSdFtwIP+iKEILaJDGMAMk9HezTXXjuHhOoaLOoeJ7sLPPPHSAF8WSGLIbJWnIXQntjKgdUHFv7i0eE+4QtJ3f9WiIPnkj62XGajcXIbJZnQ07xxiublmlIkFrQaiD6PtIA\\\\\\\/1CJy0yuWu2mCfbZyMBhzacdO55wrUWqF0VjCuon9HEuKPgf6\\\\\\\/acFrOlW4nXl2jBJ4P+7ALq84WBTY0\\\\\\\/7P9BD7q1uZwPx2HnpH7Z3vd22Fr52vyZACjhvlHb4vtL5+gd5\\\\\\\/FI\\\\\\\/y183cMUUnWno98UBDrX5QE0CaqGj+sonzYdkhm6o7HL34jJ9Ec+ClzidAL6LYP6e+8bndgZH33ZlrBQbcgrXY5aqs+\\\\\\\/OgYsjFmdBUY44h1Vc6dpCW7eNouSpIkc7yxTInmLnXP2fYWwe3jmGIM7xj4D\\\\\\\/1q31S1+BXYxr+GEFWDpAdFtABepubSA2xFeYtnHyTCcd4KNowa7oL53nLbnRUWp7dnJKGft2rLiwnVhl8n6m3Jz7FYH54uMqhCr6iPWTWSIjW9Mnx48yYs+BnfgElbErJJSG2VkDPkIxwc+MQo0\\\\\\\/cPUfFSQEWtJxYUcSjXi\\\\\\\/8L49nbud6z8JvTecrX8pv5toy+dDJ9lh\\\\\\\/pT7BAAh6D5m1rE0X3dW8m6opwEcEcUuS91Vgxzg2hRGBl4srJ2luXZSo+cNLUba7TR2d5V8dKNjrfU6awXBadsD1j7fj4yYn8JCWwUZQ9cvnYzbiKwmKxpK0cO8qYYi4YjYfNW8JfwuOtb\\\\\\\/DbyYx0TtG8pbPGqWGbMlCOQgEJJpZ8w0Oji2dzXxds9CNyjbV\\\\\\\/NnyrakmWmJ9H1c9L7CTt9I2wuWDvAX0lOmsoRAvgqSncIP9Txk4H1qpICHl+RvPerkydT\\\\\\\/uwaRsjU+6l0ys4lSZT7y+uTUqD12KZE6ClwC9vAVUL\\\\\\\/uAeHMqyKJb4kfdI08P5xn9+v9\\\\\\\/NsnF6ZxRSFQ5p4LOZhor4nAlj4IqwbNXhyX\\\\\\\/bjmxwkdMfIvom4PS4QCqQss+oKtpwaFZb5wZMSgdAyxaZXFTlG8XpykXQlQx7F3VqhQ+ZdSAz32i2sxyGQNwZKQlUPpu3Adlbx1Lnmxsv+DcP7epzSX2tIISBiqNd4IzW1i5rkAlqXEx7\\\\\\\/0tL2Pw7FCyZz+3M2Lq1HJA0qLP09swW7dPDmxacGLuQG253M+AqWu7xt2h2IFPVfOXJ07H3YJ2fFj8rufZQvtVNyPMEti8cmVo2QLmKdOeuNLz+iObnM6\\\\\\\/T6+miQCBTehhVclbMBXVRG7qjKZOqm6QYlyMN3HrhywASoi\\\\\\\/S\\\\\\\/b8pEH4CWZxjNnLAxcGXZqJpRygZw0Q+EdTCg\\\\\\\/hV2qC6WKhBskC10YUSxRXY1jL+I42k3aNKMBSAoRa0W3hOq15O1i+3klz4\\\\\\\/kd1lzjUgiruQ6QwW6XDEfg79eorTt+pfvmUc\\\\\\\/7bb2k7uQ5enCYSUiXSxie2cKdkZiS3Yyebh6RyB5UaXyFXCrjipArOY0Tm8RdtxfjKobheeg8ooQQAd8MfAlAQvsUuCXXBxTp2GRMhtTCuTpVWFlXeC0CCeeFMc3poYWPjZpn7okcb1U6NQB0f8g4aC1X\\\\\\\/ATBzBEpGbsiFg3Yp\\\\\\\/nKBuhu1w6jJjxQgUfz7vtdTAgjqsHeMfoSUGHCLkMhRzF1AAv3gUSUHL+5Qza9M9waEs73thVjuQqJfuPcX9CPKPhDMgEvbiAKIpwEMZ2skzmW4kzNncrsNSSaqD8Jsvbcuv9A\\\\\\\/vOdCo5vUbh+1VTpfxUfKpMPaO4atLCyob1AbmdD\\\\\\\/MNJZ\\\\\\\/FMUFvMHnfMP2s3VWHPVhMelxE74ysUJVVL60NPPJQYM3hb9lm9vYx0ufxlrA6cC8yh5HTCh6ATh9dg0UbglXJ7zoLaHQYuT1DsmfS6MP9oMShmG7Kp\\\\\\\/v18mCpmYiI3r8iXIA9dQypHdMwOixBmzkmn3\\\\\\\/BrIVG0DFg7pqubrem1EXaDCYKmwMunaNBNLHaD7ZC1X02p4gXwyuC1U621dZaXyURkyUpSkBcpEcoe12hNfXnB8dMJU1k2fS6n2P0KNQalUo\\\\\\\/Lf5nk2aBAZhzXAYIFHkjFpWxxJlcW7xifZRMF9cOxfVZehyMaKS8q0z7boxgdITZ6q+4zcSCQTrpVu3vuFwP8Z\\\\\\\/jN94kS5MN9X49FIK0PqIJlzv1in4XwrYw4Q\\\\\\\/nq11d3JfKCmjRx9TyLwRxCZoMbMm2ahJjbJ9bPeDFMON2WFlxGxA1ReW1jp7C6b9VaWYi\\\\\\\/uhfqk2FeP\\\\\\\/VkS5XKaqf\\\\\\\/OJfebUf2\\\\\\\/ypfNc0mbidJTxquw9956VjXpkZIxNykXzHvyGMdOYl1\\\\\\\/UD4UFNxCrl877aJSP91BKUejFvonv4ylFYScg1EFV8aMRUII4qiygKpc\\\\\\\/3mUdXxoJxinCY9x7eUCe1WkefmqC8JrZcZ4QHbnCtRkjrnaSezMnzvo9PG899UiCq+4IN8EczggRFUt0EF2655uIJBw92pF7JHxhGpVrJqIc6sQ0UOvwKO9k3p6O+z93Gv4o4QJHww2zhSp6ysAeBAz2CRb6TRzYVjHMt3HtNw1I3ksE0rDPfsDfWb327nVhAfkBdkC7DlBLSUDpMNSvfOoLgOZD1cjulSZPmA4uQHP\\\\\\\/XhaCJbFl7y0lBP6YfovRddqVk4r6GY\\\\\\\/tbZoLfo+y8bPPMYn9dHfPQIlhYp2zJzVY69Yet6ejL1vap80x\\\\\\\/FWQha2KqELGQADnKDfykbl9HEm9p7lFTcSyCEVb5TYrXpup714KWewjyOzQIe6zdggm1NOdV\\\\\\\/POlAPp12ca7l5aKdY8IDvz\\\\\\\/AYIXUtlO2hpwcPppF408JTUsTFQb0H3Z5\\\\\\\/0xmEiFcS+mMde1PV4XXgcLtwIHnQXevLTKeXCtAnDR5yyYM10DTfZgLkwiJA4dR1\\\\\\\/n+QGlMOc2\\\\\\\/\\\\\\\/XmgdYje\\\\\\\/S1OaPN3Bz1FIP4jTBHJBJKe95ZLTtE5AJGS1fHp\\\\\\\/6ZbEY0uOSkjM1zSzyW5eAldb5CR4daSXkQf\\\\\\\/q97lAcSMDPBerTVnI9ANDF9JRso4oWHGqJB9QlVieFceTYB\\\\\\\/90aElu8dXE5Eo6BNyVOQK3FQetTHsKtCCvVkVaL7p20+31WcpnLTrarOGRXPFPfS9jdf10Hmx5EFTMV0oe1p95iArHVYc0gQjc1sRhoIxqWh53zcdBttyU54dAONa03l38r7YA2bqWHIJiEtIELTICB22WtNOdJpKxr4nvMRY8JKd0SbqVEkAPnTsze96llIo\\\\\\\/iUGowdvbNylhD\\\\\\\/JhkUaChXBHcG3C7eGeyPwn2RCipvXMPAWGp\\\\\\\/9q4Apj39FUWGhrZjsph1L12s3MjemjMFWBatiTjvxaXGF\\\\\\\/9y9FeTMwEkcz7hqV+o9\\\\\\\/pjdMQtW7rUs3uH5zTEKCuBtCWA1nKiUaR8jvan1byA8lKAD0+91F9yE13BsYbKHIpILdsGQfMS6TABgk0kM+72ou9nMTwFbfqn6oOqVRrN1TOP6hV3482905INk8Y3cRINN3NTBA6L3q9IZdTgYVvrMKcfnwq4E+M74wjBh8Vfr9b9ZcIUm712+UhDgEQm7PYJx6hqg3v6\\\\\\\/Pxnr14pvtg+orTWDMMKgRAouj9nSDSHm2ET8a077N+PcRPKmX1QSmL7CiIXmldRAhWGEnPw\\\\\\\/R63x7XBz2NeA7zshp+8BpAoGfyD5UKzN9bJv8+iiyb1ny3ChBASOwg8oAl46Ka2CzbWmO1sxpq9VZl6o4Wr\\\\\\\/pMleRCGZo6T6XSAwftF2IMSLjPoA5pFAxw5ALYvpPTbXzEv65saYrJZ1mwf7i5lvU89eNMEHn0qZzmFolsdIOH7g6sA3eZk9AdfIlICNIO3noQOcwY7QNkSADi0BUcnyvLyFBD0eGKwtI3XnQ+xH+ysos87\\\\\\\/F8TqD1EYkiI+lTrLsphS8qtOo0kFelPnnRO\\\\\\\/GCOyp7wpCw9RCXuzLyJlp7wR7qObX6FvuokXpQjntvrd\\\\\\\/VBhu3hT5lg\\\\\\\/ZWqgCqHZ6zqEelmcewaVfapzK8CU2xGI4IFZtz9UkDy+sWvq7QT2+FyFi7pg13qeEEdTZtrjVZWpoW9WNajTVz29A7kR2zoIsEnVxajmMOCbKxyhs+7wD413jYNaLJzEpnexOzLjMUh01af2Qcjr1+t3iU41NCbeqx5hYMYOhgX7fyD8ryuLCjSsippjOm\\\\\\\/5x+QJ2qlOMVV2NFFj1IV2EW1sXPft1FvdCZ5KughLWLajufpVskIFH4zqPfm6umyTLD5UnDCr\\\\\\\/GEj8oJY6dq3eLo8qHtueEuH6QYOFh4YsqOIdM9eH3SE2CD3cW9m3lbVUyQ8QNNuUaONV\\\\\\\/r6IvHzAf0dLmBFsQ8xl48UChKQRKb4ZYSCQJYn8ZpQJSMnIn8Ox2e4ffXiW11yCxe6zE1BsVq1cjRq7tYXT4CEfbwkUTBqIQAH6ZKiYqdKxygRlhrlQGhnUkc6TcaWnkHAi9H4Nn3yg+rTuIyfTfPP4pq7tYgaPUns1ymwIzUoG8sLHrkXujv6djUJn+lqx4CVh+RMyt0KkfoanDab9oocxwzM7X8pL2xu2TMVywGfStSIA7TdGU4wERzBLGdSCFTKv6i0dXSBe0GFRYU\\\\\\\/UFmxtBW\\\\\\\/S3rsv4lzqj+v\\\\\\\/1Iqozhgd2KZfvU8a8Khdwz5\\\\\\\/VYXTE9ZXPLnCm9En57Jqa7AA\\\\\\\/qFYfnHMyLYvX0D0kUg7Z599WwNk1LAiUTY0F+fh1Epd2SHgN80u22yHRB3BZBbtkzKX7FhrmSyItv6EzSAqHL39Z3TQXCN3T4KGHaTkHbodHoHS4fmsJKEVbwN+o29hIN3tFTzHF1ueRibXRnPs4+25lI85zXqB0\\\\\\\/w3dnQ9NIE36BWKrgas6zwMjOsUuvhGeChPfhzYwgcphmvsr3n5zlJEzfFJG2CY0jCGQv6tGKoKxyvTzFqU5S6G\\\\\\\/F\\\\\\\/NIhp+wTnnImZSUhFF6FVxmq9RPuQwoKHoOrfultvLo5Q8suVMLOet+xLXtSU\\\\\\\/Xg5c7ldGDGPwRNrIx+WNykU\\\\\\\/vn+tLZrxQuB0Xk8uV\\\\\\\/v4yOGaG3Zr\\\\\\\/7D9GQ\\\\\\\/Ds4DXXnYBZZoW3gOCu2XT8zsnTf+kdYNFdzws4y4EfHEZplpFkp0Spo5soixqF1equnPomaBpJzSxlQs3kq1jxXYr4ny+at2gTvYKmdmH36wFs8Ds543XaT1WqCtJCoR1ws\\\\\\\/soU4gCAbx25PTKWVY3WHlUl2tLRgL3Thk78zoxq\\\\\\\/uhkG6+zk1Ju7ni8113iBnOVlb7txjEab9pJqUQ8JkYFaKvOFgrV3JXqjOIg0LUj7wFvQsCn+YMHJO\\\\\\\/nv\\\\\\\/T40y\\\\\\\/oCCYCjENPmAYRVXAn+m4CUPjhs9dFdaVmuFAhm9qhyUi+TXQE+sUy1FwsYXGxP3R0yK671XJuD7865PWDr7kbl11HaOCGi6iP0t8uvQl6bdLXQxfpa6dXB\\\\\\\/Uc\\\\\\\/2msEqhB8NJkZnF74ZPsWo310W\\\\\\\/urcFzwfN6B72GDQhxNVXHA4jG2mWEatVs7vB\\\\\\\/hC+HuEShX87rvuqAxLsZxlKt6e+6o8\\\\\\\/OxLLYBB5iXOa2DBeCDVJNi6oWm0T3LTWXi4eqJbCngw86q0UjqmDZV9dkNlsGWe7MMZ9BrPlIK2I+CrfYkh0g2W3zNw78lssZF8KGY6a+FsHpd+RMEuTt1xavShSNwEM611LWK1YDCUel3d44alyBC0RGtjpSPfl1hnA2XH12lxA8QIh4tGQ58MmDZ1cwhX+6FU9+mSA5w53nLLx1hGt0S3CrytzxQWiBTsOsjeoA\\\\\\\/Z+ETTmO4U4DYKk4+BJFvJ4TARZqfbGZI36jurrZaI3YUCjkGb45ItVK26W9N\\\\\\\/rCMsNJaWpAS\\\\\\\/2goRYIvaXs5PqA\\\\\\\/N4RzplP81o3iZ+g5mSV4EqJOiIbQPKDxepjFTh\\\\\\\/TZMSbKT\\\\\\\/EyxBCL3Lg7MIrzU3FxXc7EpH\\\\\\\/hoa2crnLAqrvVCHxzF9+GEXHAj\\\\\\\/vJ2MHlk4g4dggkkUWmH4aPfCgkv1AkfB6Va7x2YRGHqwubXIgERx+mXvfgElJKkl1Q5Chz8\\\\\\\/S3pcvMvyNHyGpvi5ENhCAer\\\\\\\/pkI7Unw2ra\\\\\\\/hDRateqSox8ndmRNIHtYxhwjuCUvc\\\\\\\/35dmB13MkMElequcpgXhqZ1wUbuiGoY9WWj9w6jMnF9bTDInueUl2Jk0hw8m4rpKRZzeZ1vo29gjEo9ftmzNZm\\\\\\\/65C\\\\\\\/toyo2FFPnXNjBWGXZ0l6gBFZ3sHOHCyWJosASOSpPnDbrkHjtz+MpOBX8Xq4R8aPS5Ayrf1SHdue3vYObnDZpLGB3KNSTDbt+imuglIwDHPfQGVf3c8Ni9p4+8FOJWk4WH2E\\\\\\\/oyRtV8DSUdrNXBfcaknTKT5J5bIkhUM+LyQfkQHmZuPjBrIfbi3aai+TflCY5hy3igUFfrfxf9h5jhvRwixSMmFUo3jZolw43K\\\\\\\/mIbN7ktiaRQThwHIQMzZBAXRXMIx+n94ZC79gdDgVJ+iAdrp\\\\\\\/AeRe86ZXKc+8uZJKxXhqKNPCC\\\\\\\/t0aOSk3L4k9xnHlC2RYEqwKGyxRp4cHDBQavrnkMQyRbmgUyMpB67yUkuWAFSqjcYsATP6QzcUsAHURPUp9m2cIaf\\\\\\\/gTXyraj1nWNYYXvoI6L03d9zMCWApG1MlB9UeIISJ42oqMTTEzTNprrQKZY8bcoSZAY7VmEvhkPpoXXivVnxv3hoo9CXAAf+utmgmfVKld\\\\\\\/SlBJs3LEbKoB5bjBYrUJifZU97ivQEJm1PqL57GH06o4+ZSMNhMheDUn4PilIWAitVHCvrRKPtId\\\\\\\/\\\\\\\/NB3hn8gtjWHIGsnNv+5uu\\\\\\\/1TP6KH3KZ63Q7K3ItqWIz2Zq0CsmKI4SaWxI\\\\\\\/h\\\\\\\/H5p\\\\\\\/vQ1ZSHIQkM6O1JJY0IlARBvSOz6MO5sOHATqMaprAWS5BNLsELtSNryZeuUBEDBuW0b227jbvvF1Pp+0ObqVmRcZIteAT4FKJ5FNIAN+r1NbtkYsOr2tZ+FEkuVLP8BmPsl2Cwq0qVZLLxtP6jnXCCIlLrAmc+RL3f7Lh0i49cvxAr56O1ufIXvUauVbvOkw2NezNd1UqUTgqwHpY5MPfa73ecLWoJQjW4DvrBehT+EUA9RIlb7I0AUcq1\\\\\\\/vAoqhwpWsN9QE7V6+wBg+gPbDAybNwQZcTXV9vtUvsHrZiDCpDgWcCmm3QpokopMnnIWkUY9Tm2rMmmifrP4X6yE66UkP8EhLUKECyNH8wK3522ozvmjCjnNbNMFh+ZZQMSm5WHRGeAY4N8b\\\\\\\/oGdNlXVL4RSCGjxq0K\\\\\\\/nR5KrNNTP5pO1uzLUK97V7mHXKO7WR6ZgMCebwtr8O50o2bfhiJRzJgW9lRETE3TT+17arZZe6ZxvXnLUPaZxSHAcZrAOVo2HqOm33hpGi8Ct+rjVGWhrbh\\\\\\\/NFA9VbSRUSSD0Gm56ncGxRXxx5ZdTfoSRJ3zi+Y4+oy01mLTiANwrvSh7wZiOnA7rclsaVMqFKd5g+Q1X8H1YfuOWxmfLNx+FFS4tJlFgb0aIsuAol6XFegE5kacya4ho6lSYnwNEVblxzosbKqPXEPme+beXKTRsFR709UF709ytIv7vZD2OX94hu\\\\\\\/t+QvlHr5H\\\\\\\/+3KcpsYnSpSxEZIUgc3otr\\\\\\\/CQonRl9yeNsiM4DndPicTBWcOkPO1qwXBewEJCYxobK\\\\\\\/xE7q7QXuIrSl2cXY\\\\\\\/AbW8w53lobkua\\\\\\\/tmt\\\\\\\/dvfTZrEdumAbpuYCEHLiWtb+DS3g7MKQRnm1D\\\\\\\/OlY6UMN1TYmTwUjAKg+TuH66c1Grlu7G8I6vAWxZGDMujX3A+jQxGJcbbc7ZX5ru2aCGKa1go9sEopW7ljl0gRnnPFlpYtbCTf1K+\\\\\\\/K0V7RZelNuqGDXlxQBB+DSeP1cQ95abrvSrdgdt1mRvYsKHdX1zoY\\\\\\\/uxaD552VUI3+vVBeT6Pf82SRRxbM8CrAPfVmtwnioBCi3+KMFOEl7SaCkir6SziW2rP2xmx2lh4F+9ny6ZWkt2DxJn+LkO+mo5bsw3ttCAYCwUDd2fEKDvdDb9rI8oaCAFc\\\\\\\/lnzyRgDRx9hWeqFIMe1LcE0K0qe2XZBuVH2aZ9ZnyFWboUXZJg9umqIhwzaJ3D1hYd3D2cFuAiap7Kzr3D1kjgnv60TvUUHWB9oL7\\\\\\\/Pcp8IER+Njzu7xT491NBAF5syRuYB9E2YoRiBtJe+jcQI\\\\\\\/2ekj8cTq6ZBmKqKHYBD0f\\\\\\\/LWU+mz8XS9mq3J2pOTK7910wQIbCdCaNqWP\\\\\\\/K+HbULOqdbY\\\\\\\/98SmPzlHAOlnrH5\\\\\\\/oZ63MCCvFjfGeKqy0kuQUitpNOsa6QPS7hIcvH9W\\\\\\\/HM2jzJ5LO+P1+UhmocR6TpoUkchj7pz+KdEvyDtq+3mnwARrW97wGb6xzV\\\\\\\/FW5tWJjAYLMZlgo3sKUa9YjraLHBa+rzy\\\\\\\/ec\\\\\\\/W+68i47daqLA2CxVTIu\\\\\\\/WuJGsoNijKkLXalPMoYOosNU\\\\\\\/owvPQeXdrnnQAEoHAwTpBcXlqs3KyoNLZ5QijcT2PzwN4+yRBkJbCcEWMXRl7BoJZPXYQns\\\\\\\/2E44Vp3BlRl1xZNPO1FQDmMfvsHkFRpFxvgUp06vBdh55Tb5d5BKTSQwVjzG50ERuU16pMIMELlr7LqGv6RngMDNDuZBenrZ7l+b+9Fqrt1UYZUOf7T3n2BJq5FlaXlh5iZhWJFeUExhkQ3sFta3z\\\\\\\/UkbKqlNbCHD5D26SC0oJczhSKM6tG6fY6IMNaIFDhvAxrBOEcGIryuAVlSmEdSjUR7ASYo8uM+IeienBo0lgCrhNz9QRAcmMqRfaV9HO7I4Qs7vHic3jHyoL8dY8MXtIPS5ciydzT+WC938v3J+uMUcLOrkjgLD5XrYTU\\\\\\\/Eepg3M\\\\\\\/3QIF3tP+u+fmjyVjdUCEZ2wTo76BWAEqSEGmQxsmjM\\\\\\\/BZPwXcSpWVyQ5iFi4DmhPXp12ZRfUQW+pPu7+\\\\\\\/X521tnKASgDgyrjHTzq3LPBJDyPcuKxhXvDFfrhx04+1wQAlWOn+jwaSddXurlBmfFUIAABQUCRaAk2LvY19gvC2LmXjSdDhOfT5EMkPT\\\\\\\/9398YXDXZYgzCExC\\\\\\\/B8Kkv6pYQUl2e+j6haIjoMMCruGRuORvyHsiARIb9PHB8ehoh3ikBT1of3M50IgxYX7+SmIxRLNjWG3nC5rzQ0JfDJIcRiVJ8yzAX\\\\\\\/IJe27F9QWAUeRbhvg2xfX4CPLxuiJIpEL\\\\\\\/bEGQUgZQNeBTNTQ9g09iVBwdMV2dFaqWsPeztyJDxhHg\\\\\\\/vh79GzlulRPhFZYks1feoRMoYOojHzjgthwt9K7S3IvR83knqNjn8BlBjzvr+3vExmxCiiTQBG43Kpum0FHG0UYVMcB5r4t4D\\\\\\\/a\\\\\\\/YIInVlyzl8pXTAg65tJn1zKPv7yk8wyNxEaE\\\\\\\/nDp\\\\\\\/o\\\\\\\/9DdDhAA+sf9XOpZ2DhHUpHa6Vl7LFG8wAeKojU9ANRpfRAgeAKCHHrFASCCwQK+cuirve1ogBsp8P8eVT\\\\\\\/QYQ0JH8g7gcRjjNzUloGxD7rZHRUg2MmmB6fLEp29\\\\\\\/dOzR8bb7FIPDkLeTnZjiL\\\\\\\/IZSccIP35XU8YvNlGAHRbJXvHQq02O\\\\\\\/lzRshyqZGD2ZVG7ZNuapgkNTjo8BvGonvy3rJyVGlrQzN0UwVEcsT+CilIj9l8qcMD06czGD5H7DNCxKWtU7NV9HXhz19DPh4L9JXxAIlrAqryrgOs7iVjSE5Mhhy4iZ5QTRlgjg50db2DjDaAzbWVNNVRE8F8\\\\\\\/eeQOjo0eUS3b3DcoDeUDnEo7GzkektLvfFCv6yqIHbTLDPjA\\\\\\\/LrUEgriDV3NTZGIDCf\\\\\\\/L1BRy97UJuwbJJ3gsj5QZT44A1FkBbZLyTntvsxvE\\\\\\\/nlvwgodo0I7kMpcwQcoXypoRj0DvSRdUFFBEEEsbunkPll4uSwCY8m2f3UuT34zNxO6bU9IjeqVKI3sfG2CK+0FlRPnMBnSseoj6yQum5JLmCAS8AEgnoBpmdqbdIckrsWx\\\\\\\/UTXF\\\\\\\/osOQ2PoWAM1801IJkf3M5sGzGiYT1t\\\\\\\/lzI0kTmpd81sz5oLxSWPI6VczGrPeYo1oSHIqULo+eYuf7803rPyBF2wtoUaI\\\\\\\/yHBp9UDQB5hQitZsz1T+pNFWWdQuvbnBOGqTyt\\\\\\\/z9Ebiw72MvP4+gdywAcrWu0Hc+X4PQi3yqgqOy68\\\\\\\/7VxdNqmpKsGsvdzRPzdNAEMjOevjstIibQWL8tY12m+kczW5PdN1Gl+dKxe+ob94puTE31yI0Y6\\\\\\\/oC449Mf19LJV0gmXlnIaMhR5udvjM9qPb8I1iMO9XOuRDaISYoFw7B2Z6IKlxuMmGR8B+MJPl4md\\\\\\\/n2X0Jn5mD0bIZ6PbOeV+XruBBkVWkAON80cY6+3xEv+6gZvCNrq1wjvHn4tTTTNPVDPy\\\\\\\/C3lfLPpm5P+ET0dQE+Pl0xy4ZnCqVYjwKA1f\\\\\\\/XU9\\\\\\\/fVzyJu+IpfNzM80eJKNxCsZZZQeerj18e3bgB3uMavQELwkpNhNDMU78634Wqg3MQXwit\\\\\\\/w05ZbM2PnVGcJ4LcrluHdVkDXo1Z7LUrmJHdZAA00LYk9p7WN46Mkg3GGPnClo8qSGAK8wrLGTjnXonCgvz+HO6Uy9BRQeZ3jenD\\\\\\\/ZYYiVL\\\\\\\/mJ+lOAgvDkirHy7pyxYvrrm11gjMzJO+ro2WBA1IFbj6wZn917Czwpy7FOIpqt0giqGcuoUYtuglhgzV\\\\\\\/NrudBsqrdkmz0KU+CwT8hN4QCY3Ubc+KthhzVPtksfEnzK6TmVuQSoW6V3L8nKkMNz9gaAH9PXvvRXqiWwSYh0yWHjwWpSfqJl\\\\\\\/gfWmAHNqvNWP0tFnW50Qdbs1ngRaSjhFdJLbtEw4tHf39G+U8HQtGfHqCxzjmTiI6TbRd8AexOYcxQXI+EHeVAq3lyZxLFSZSHfhuACBaO5ZJzhJ9cAsD4xP1L11+lOxNJ3FZRMG6S7V1yNyIYkHTgtu2F96zfjRucs4HHiENRhbOtKNdd\\\\\\\/fBLys0sBD6Ii049+0sbY9sycN0rilAUC+BYox4JCBw7oHRTMHX2rLhLuuSXqow3SEYXEuBlhqjqKpirsMoyj94p0EJnhMqOeqPHq1vShKyLDcwM0DVZGzAxfo\\\\\\\/RMNeaKnWHaLis6Su+W\\\\\\\/vrie\\\\\\\/nyqXVD\\\\\\\/fzoxzhl6T9znfc3EU4O84rniuZAC+JXQ\\\\\\\/d6Q3MMtRIOdvUFD5\\\\\\\/IleBZiA3GO6bXX15z9SOCk6DuSM+AF3Ucg2BzYFGvpad+pEDtkwfSoYdEkvAZfiagEyedwwSN4vW4ywK++Qdahga0fn1+df6ZKx+Zqa2V1xGp+Tb5GiFRYSlGRA78kT0jDQbrKePbWoibT3btha87NAWkrYuyH8RaME\\\\\\\/TwxowuIZ2e7IxR0ayd1xYpVXY\\\\\\\/WnJ9gF89FEOQaWnQtWYtuL0wp5RO0M9fwk7IC\\\\\\\/SL0MBfTaE1tfWEKKH5rRmL256C7UgXgM01SsHm5FeAF5vYQvSFMkQGSuch3g+3BVZwlKjx6vWMq3heHKhJwoX5j5oJRrWOOp+HnyHaUufXv0F2Xme01sEinLdj6lpb1yTzmsNBa1jptGYI87UUa5XH+PaHtG\\\\\\\/Je9JacwH\\\\\\\/aOdpvT2Kbu8533vsAYGLRWce46R84pCWEki\\\\\\\/szt2d3kuNGj8YP+9zVAaSl3BFHVcWMBYZZ+\\\\\\\/pUDSlKN99nQLeXCDfzzmBdG1sOWmpWc9lakElckHnEOOl7IV\\\\\\\/UVOappqo\\\\\\\/wTylCf1ag4dxaWBdwGgHfdou\\\\\\\/Z73ijrfv+3Tcy+0l+HsXYrRN9B9Cx+2d0+EALNBHurvf0TKeKqW18GhoCdt4f1Lz9LYSNZP5RlgzbmQafCHBkja+iAw2J+nzOLQlLFGSHV8UrhsK89GR\\\\\\\/ZTuA5VkEl+oIDvtVGqY2u7\\\\\\\/QUgCs+VwX8rFFdi2TT4SiaXauViOOP1G9VRl2W+Of6d\\\\\\\/TVuFZ90cVWJNJz57lqWyWs7x5taAOYEtmqFubj\\\\\\\/EuJhsReGSYpbGJZspOqErBViA1s+I7wOjumM7JROEx6pf8BXTnoILiZalIit\\\\\\\/wmpcBBqoPjG8I9FZuIOYAQP1VimfcTjhQ5dfKvY4dAEdJs2i37XvO5FZ5xtY9Acu+ptQZ+6g9GryWx\\\\\\\/IhK1rQTpoKaqzl0VS67dyHeP++wQtZz7OXdSl\\\\\\\/Kj9UdiLGEcWJKR63wtXOeNjnw2LJeKJZhO3W25Vs5QJbSrVFKnCuxYBsvD1amKoChMzKiFQBZVXX8\\\\\\\/FGKvpH5M5+VouZrP+LmJv3PFiD+FZaBDjoZfVl\\\\\\\/cCGEi5f4ILlcVpPu6L76E3o2MUcPuq72u4H7ueZSiWidG\\\\\\\/MJ5P77CRDrob6q\\\\\\\/qojuXX\\\\\\\/9g91THozmzjSa+h911I3TIS2wr8khLVB00hOqv1b9af3fO8R\\\\\\\/5DnaNPHVdckqOZCRBNq9M5lCKLvjgBfgzAj2uldbnw1\\\\\\\/KG01JF4GxHw2XkrxFfXqum6Ue9Zt2pkMkfSdGW5FCnubschr2AIEuPB88iIif5SZ8dUC6YnjXPtemYiUWMxILNpwctKtXlkCkzaQXcQAvCpffHLLpOWSr6m0TvkdPTrsXdCKGXkpeAhaxVZHsG1d5zuKQdUjcuVReTCG+GFnMyrHMOsbsFkScvivVg2gZvRsZNhTwsfvJcKMF1wEmMh7DPydWAlVfVgwnSAzdQjU7Cf+8olWX6Sju\\\\\\\/7cJv\\\\\\\/4PAR+Dj9SNAiT4qEFBUae3Y3hTbyk2qmTu4WxclfAHya+BvbvWxJbpbwvb8E6aI8DrSgOB\\\\\\\/LZPvhwfqb0wAOgtRz84qmomYd1JcyWOi2sEqn5HXscJAmblBwIAWXMdlW3sXvXHH5A4rIApg+QMn5kVyz3gwas2mEVFIHakU07IYQeGqKTAd8nhAI2FZLsR3WZlEjzvACkBQtN3U5MfzZpr28E8NzlSAXUbfapiwoEla5RgMYnJ3ULNgMEH8Uq\\\\\\\/xmjOiNJJpFhqSNNYvPJMcNVn+r8h5DsTrQRAlzJYptDb+iRiuXSWYn7I0UKtipc8HG3mj9ihcJtg7jxtL9g3Qo2KrEqJqx27vGNnB6OPLwYEOshbiqAIVUPJ7\\\\\\\/OuTh7qy7zvQRnhbFK3+gsdJ4nbhHRgOCncqVSNMON2j4ONDsqpPj0gtw0JsJMSu\\\\\\\/TZO5UGbkiQYW5iN\\\\\\\/ygQsbgJNUMJdkU04MN2eycsyoIOKkuix6yCBDvpt0DwJS+SMlrTdX9Lvkv38tUtAjCNbxTSIfGbvLvvUWATXeJuYb4\\\\\\\/1NI10AE9cciH\\\\\\\/24XycPcv7Et6v1jVA9n\\\\\\\/2vlLwJ8PHUV\\\\\\\/fRSTtPql5iHNV63o1nQjgUvGxR4dOiPgtDN\\\\\\\/PHQdZDsR8e99aqvcKM89Zz3LsqiGxFx\\\\\\\/io4N0kl9jG1nYM91YKmEeetwdsMcuzZ2nSsUQkt0UO\\\\\\\/rH91t3v2Zo4j9DDSdk9KKP2uK9IWwKI1djhj2NgMoTpVp00oMN4U1Zhoxm+TM1HCBE4k7G2aurX3YRmyYyBpusIpfiPtD0G98zSc4cCw0QcWKAXbopfpEIpO1DY8HuH45pHPd3snM7ixQFPAqR\\\\\\\/L3vVAAg7BVBrStD8IdC1R3DFBvtjFHszFe2Hx9XGqoiFBWUq40IoNEFfNBVqObJhAfIfZf0Zh841XVO+2dnH15tXo4b1orW5ZO+vqWoy3NDsIt8OwT6p33IwZh3J+AqBhPIKKjVM2271SaF3nk\\\\\\\/zWKpETd\\\\\\\/k8Av9Jl9IobG166HNQTAROQhEWiu5d1no9klWM2+Pv1L3yKkVMTmbDDnOQocBAEQbWlbk\\\\\\\/v1EWBOkKotxFjeUZ74HIrHE2Jx7uNYVIg6sTMIaTm7FMUYuo+lO49eYY0sCS8U57rsYw3V79do4uklItqQkUmaSyGBExYcfSAlkcnT6MyP58W1bTVqz3JuWPmccO6Nzestf0T7bk8FDJsTDMXRb2\\\\\\\/WI0z85bnsgN+OV3azmGCrWuRdN+XlbXjTuLVn5d1mgb2+D\\\\\\\/UkpA10IaA6C+D\\\\\\\/t\\\\\\\/5\\\\\\\/WIhKXSP5nnMwLeKSZ4CT6YtJqPv4i7EFDIbU14jI9SW3QeweyyshZSVVHVZw\\\\\\\/PMAWQ9jlWHdG97bUTLR3\\\\\\\/945NeNTu8BsADYjFM3aN\\\\\\\/4aGWyJFd5cA\\\\\\\/ZJ1+v\\\\\\\/ACwhATxJS6vb+lGmadwKMzZVO0mUZk\\\\\\\/q2pPCckgQ1zDOj5MCi5oXmkhUBxrvVKKIXVeO62CYGFO21+wFA5OudY2nyeXrwo47ud7RlkzFwGevL1OnUaeY7WQ47TayT6TV3TpBUbnq7I0cbUnkwNtZzONwbb9AcIGWGeQnm1bqAS9d+5YgQJXBINYUmwuB6wlAGTuzeXn2lvX3Qeo1HI0lCOddMQcS9glyz45aRaqu93BGJFiEeJKKb95beQ6lGT+f6VE2S4X5Z7jaaDyxq4X0M\\\\\\\/ekd+dOFvxNZYUQynHv+W40T0uvaJSSsPltxZGxd2ks2GT460xzMcuDIGo3rETWFuRKWUTyngkciLUpdtOYg9tQYduykwT2lkSIhJVHEutpQsCRBLXvu5qcmzVpmrAZArUUYmDgI19lXQjieyUlAcChYaU83d0E8YW\\\\\\\/o74o1Y8z2vbB5Ow8Il+vwTNls1Cga5RJPfU9k+C6ppj7nliMbMi2F5YafPk2oGFucKr0jeM9MmFszcJbeDlgKqWcSQvy2BEKUrKb8AZh9DQBbO7ocTImdibQgAj6BVouEgg4dtm2kjS5Rjyb6tT4eMsyZ3ehd5zb1zKEfcmcLOHPpH8ZThDs9sY8KUsfmEHg7b9wXfzhZbeIfHE\\\\\\\/x93+\\\\\\\/gF2D9LyoFMae5BfLdfM4+FCOa95qC7OdcqS8Lp9ndMIi63wgQxdT\\\\\\\/mqOIVW48dHfAqqd2UYMFQfw22HfMOm2GN1C9KB5oG\\\\\\\/nD5YWJon7aO01R5r3LDpXRA1IwDd4H1yCSfKCnM1MGbdxKSX29xI0WSPkY0vdJrq+U6uyEJJUb3jj\\\\\\\/8USSs07Wiez8dkCbeQwNTCm+YAqp1Yt4ePxlZ5dJ+QMRAgucXiQO5WVLonTB6AkSUqd1SUwwn6IkT3BYMHVzlrov6msAybemieZf0bz0eRWJ8QyYpZIsHP\\\\\\\/7bXGRXD0SOaKHQNC5EDa4lmMcCyCom4aXHlnlcWlBtkJiZqZYYJUeIT6gGo5SsKnovrPJ7BBPKVNoJxg58gv41JUaJ\\\\\\\/u+cA0VvsEVoY\\\\\\\/q0815cCMWIabZLzMiisZgvGKuSNjssBy7BuTmzgYsE0aKIl456IpfHCm2TuAbnKMvr8N7cjHBK24KkK8b4PjXecyfwrW52QtrMtrwWFFCQgpvgJismuzKjs1zn8YL9fnVgs+OGbWrC+lrKuH1HuB0Cp3\\\\\\\/ADaJyE255k5ZDbJz6GklKBoP8C8DFlCLQTSCcPxvBjtURqhZLYSawgiOxo0\\\\\\\/m\\\\\\\/s7JbKOjaB80qn49bk2Hy2ku8WioOKhMwTm0DA0uKz0mSE2xn3TxnmYFRCyZe9nLwcAeS2+yHp4zeAUepP023HHHBNrHm5hY+kIVS69FkZN\\\\\\\/htBlmgAo3XnEC6AKHntU+4Qfaq1XRuoaf0GZf3+4oeOCnZg8yVzG93AjASgUPYWyef29w1PkQ646iWm7xZg4\\\\\\\/dcHi0hewmY93XBDR3dwv\\\\\\\/2m9nEE29bx8\\\\\\\/DZfzeZZqL3cuaGhH02qbf9Blt50rX8r6xggclaT0YcfPpjNAKHpLea3faiaoE188IIwz5cv5Peppf8rIwJcCmQ7of0V0tMr+vJofcjS7Yy7wiS0q5HVWnSwte2t6DsPb+PJ\\\\\\\/ullobLRwgZeFzcUDDBOzuYINszAZYIyO123SyjPHLVEoDvXFrESyj3Vc6KXTrtQvlL3jqf\\\\\\\/Q0ZcNDAN4ZPtC9DuVzCuJ+bUU5dFHwsUD5JiKpVUDonJkLAl2K0qgvIQeg2RvweKNjsTkaUOtKkHUhChNOWi\\\\\\\/3dXrJriI1CrQk5tzz472VTZvtZ9A\\\\\\\/XRW7JUTCiLdsYXC4F30rHEFZ7XJElqjqfhl4aYfgkTnpjLD+a\\\\\\\/QDrVZBU6jB0CYhZ3GaZFGHLBN9pB2u3nnyp\\\\\\\/aXfGmJDqHCDRg9OnSJXza96vXn9Vh3Pv2c26JLFdPRr21nIqPa6zjFcVjL6q1AgIvlhhprDgD6nv2c9XHWq0anYX9nUt3XLYEzFlUuDb0nftyDAARhQRYKg+qHPOePSxZYkV2paWpSzecSNRIyZgUTROBkbHOYRwM6bS8I0fhyEAb02Tns23kyefqOdfHEEbHiEkQy79XrFwME1zfBTUrEJs28diLKQf2cCLFfpcpki0Ukv2NJ+B9Ki\\\\\\\/BapFKuGFdsaNDpSyNfsKR1B1Dfqrl8mGrmteOAbS177mvqdL+EJLWIAJis3FmYkREfxqIFoSIVT9AjPXGaG38CzRqBfpHg72MN+cEzVHlpxHEU0s9Gg1pq7Dl3EnUlHd\\\\\\\/lhj+gvHh3QZJEnbKUQ6buvp7juvG0Ozgng0+S8p1h7RyNCQyPJJUAY0SgIU3aZrHr0fDQjngGvWMKDD2Du8fX0Bem2h02\\\\\\\/j1sPwQHhTTF3OxZCkFVkMwuI\\\\\\\/\\\\\\\/Yv9AmP\\\\\\\/kZrsReI6jkJ6375eaCB0n1U3nb+LY6EO6ikOPEtI61bkSWRi3q\\\\\\\/0u8CvoHLyBf0gUv4cYcxR0Gtn\\\\\\\/SWSSvQAt7LWKQTxMjCyJobUkaS56n4IZQ8qc+55z3HVGdXevtMDhIkBkxO6JFojIg8AXSuIhaZwtPj\\\\\\\/a44Ezc4lUwSEguLNXOJYAI1ByeGi+ks+d8eXnlxN5w7H255mm3HZT+W5pP1geIEhrPCR0K6QYtVdvQ7RRdPT9lFIXVTGUpU+0vrKnl\\\\\\\/OODBrrB0Q9dlPbMvDadEfW9uArp4K868Wv\\\\\\\/dvlsMIv6M+VhiRGnqzagwCKi6XPR\\\\\\\/50ZXJrJIc1PIjCnLMp+3FkFdpiAe0TuazlmO2HwTHzl8eRQyf4Q9U0kwUjl5uLADesbG97MFFDO4NhtVOw8hNA3dfjxUhBghJemeiWexoqdcOpmDt8dTIaPDtJ0K1AlZ+a0672WSsLqX+EYQZKt7uK+hsxtUAyrB64VEe9hGxhIgnlNu9M5fYW38rWBAkIcJCax2xdcRqUgJZDlAVSZtb0IgEqbB3fXc4bHQux5QjzLoRakFjnh0s3wzkdlge5DIhEnT5keiCfb3hxpJ5R+rmreudEJ\\\\\\\/decR7G7bFvxFc+G9XFBVpUBIwMCsinbihf91ww08jSusxochP7hcCMJtFdWkhb8PgTR7PMqkVS6Q5PwxX0gEBsgiraXe8atOgmDkDa3c7PjBNGtQ3gBzAKaBwVoOe2mi9zjwT9duVa5ZbjFVlT9jUqZh\\\\\\\/4LOt0zmohgPuflLRgqeTBVyms+bmyBuPQ7Kt7PwijofS\\\\\\\/y8I\\\\\\\/HsLQ8+LX446PtcBn\\\\\\\/bSJyA3ovgmI9ltNr8wCj40GT\\\\\\\/k6nRmWkks3Goe1hoGH7voGauX3QeW\\\\\\\/Iq2Dw6wwAFYut8AdVyvFb9FYPvGOwiCrNyaS9zWWme1IpNKMsA6v2ML6Y8l0lJa2dBTn8Mo6CitdyXXZFIDsFs3FBlGElAO3QiyHC01lgqMNX\\\\\\\/EfVEcAC7KjXB9Jg\\\\\\\/j55cBWZUculXpC+quB6SJasdsW5tFUZ2xJW8hWTXrNHiCR0iVRyy9Y7qQkVe0u7aUPtANr00SDiuYsjPWax2TlQAiKkewPG2ShrVkln22Mgyo4nm17QpVj8VOdfP3v5bGyMPVEsS2cUJHF6hk8CNHlRa86DK2n66pwLThznev\\\\\\\/4mUE0k6eT7SxcS7nY0R6CEGivAUvKlGwqAEfjBHUsJSQX+fAKbO\\\\\\\/Xk4iv8kcujc+CpvuL+52Bqk\\\\\\\/ELIetoY+csU\\\\\\\/asM1ev5G3MDa67b\\\\\\\/KSjaYmZbT3xET3HESFyFAFVCF2en3B0bxscTIjOAnUNEB8afBgrdPbDIVMUUftfsSnsVdArqOTYRfGH3pXlvRrQOaiLaaocC3GgFOpWx9XJZ1+7VRnsSVIm8YXHY9mhjXb8LratR34vO9wAQ2SBeRBlyFoGmGrd3oeheL6a5s6QvqWNtuYW5vJbADoMOP1Lw16IInhf5jFxYUgMTWCmcs6AWwNWk8\\\\\\\/AiCvEXnsfzPVVqy4EoQeUQlwx9R\\\\\\\/qzc5mQP5AR7O9wF7EDzIwrEcE94wo4NBvbVkJcLxCFEExQkJdyU8LUoBATTA59iYPvRfP8\\\\\\\/l\\\\\\\/84+ViWQ3atvRzaYyGMT9eJKq6mrNCdwl5inRpunZb5izkO5Xcuw23fKOPHoC7\\\\\\\/X+kpDDvPmob1KSd3lHyBrgc6vLCywDgPOVPEmIbsN1x\\\\\\\/NrUCa0PRf3nMI31gBptclkeGMMK+yVI7ZxjgdizrP8B1W9Gqdxqj0KIYW1OlDceF51fis76czKosXfjt7JoHqxlSfKHO+SD9L5svM9h\\\\\\\/wvy1JpFSpPRhXFCh2BBHxhYXIB19P9YN+2QyncGvQfMgG76Zih7N4M71ITOjYO+5NOdhzcbx66lKG9GrVHSMP1xRNDA7U4G7kpqV3Uy0sgz3\\\\\\\/9AS8B5YNcY0hKvOmVbz8n6FuYnKqhpj\\\\\\\/HVwfqEuqqmdYJ7cvGNceAY6TNGvQQbwZlIzZSVfbFc+cFjmh4PUyI1dcqQKxPwJckepOTXXF\\\\\\\/Z0umg0x4sIQXARyObSDEPwPZhLZDettxr2YRx\\\\\\\/CvKERx+njoB6cWrPM75W55Gj+WzayO+VdgSqFj+ZBYeXaKzEoZtBdbvTMGNjXQDjKErhd2M1D9bboBZ8QxXy3tJ4HSvy8\\\\\\\/oNTvHQr3lA9MJd7mOr2uSCjXA9SBTIexzHxW9h1z0d2QdLoxp7+RuK\\\\\\\/RUUffrVLXznAc+lsjRTG+cb2572rPA40NYFDTG7Nw2BZAA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WJmh7K0GWiMrk\\\\\\\/hh0mYZFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a26c7cb81369c6d2f9ab95cf37f520538ceada0d99514bb9dea45fb4b1fd2200\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DMyP+thAOi7KrYQJfDIG59\\\\\\\/Q+GnQJaqDBuRLY9q1UeSenia3Ts+vnktajqp9zIAnEevtCcXJzm58fK6JYBCRd02ZHs3iUt+aEPlCkHbwr16v3e4yz\\\\\\\/dTL9Xua\\\\\\\/Xiqyt9hxd9flLWGIBidHr3z5czDrq88gc2+ROErdsT2os2suVRCpIRV9mDbtHHd\\\\\\\/j8K7txCOOQpPnUjbKeKDrceyfuagUj+AeDrr4fh\\\\\\\/+v4UfCMA4HXzdIPTuFZInANZm6OIIQKZ\\\\\\\/T2c1JQeS1taHAaxGoqfyd2iVuoyNesABVXjP+ezxC99DWfrCvPt+fegn\\\\\\\/FdRrzLZKCkc7FPweJ+iyM1Y3vStlDP7HctvePkvbJp7TneakPL\\\\\\\/mFr+iJLSneb0d7xzqb8JQUTrlXCO9ez\\\\\\\/8IVQRohCr92zgW5o70IRWGYJTBcgGyq7VIT00aLnwHIsrha2RHjt6YKwDDAzKe8LUFkp5EupQU+1vHs6JkMmUldeCmWtPygLRVHm1gO8uXK0u1eYgHhsXioASFDhD66PtT8oQYpY9l2x0dO+fsYVxJ8YI0YkvwH9rR5OS0PBQBGVjRZQjMg+2NUMUgqGiIXkfHgbotvmJjMO9ILh0jqm4hx6H1v10ME46wfBmsfoupztrHdY6rzgEAgkyD6r+l00WEzc0Cug07WF1W4rw69JQV4dR5SzfNortL88im0iXYYVZv4aduKA4IHjAT0z7\\\\\\\/0DEbKSjVnG4c9BFcCprjFzX1HkGfVJswhRcUIcDp6uHsZSxue8eXYZDzUjMKOnXiTRkSzzVkE1J27y3iLuoQ\\\\\\\/FjXJtBxx1rceH4HqK5ybjtI\\\\\\\/nMTc5TTuCPBfc8q5foyLg\\\\\\\/Yp9P9EPoFqOjfdkCdLWnUWayKyTvsDoJrWOZWrRCevqGyZTrrE+X+b5S+ACApCYsXFW6wFztmWy5o4H8PrL2IOyhYoevbIggbcxdh4BoQPcdpfvtrd5eoUkGd\\\\\\\/YJdBVpYN6788+Ip21\\\\\\\/FXgFYtrDfpnAGKjgTJBzdiexnO8WIqMLdSFsJPoKddxiljYvQUe7UHpQfL6uI7BZwTCPZBPldj7IxZ5KF+luCI\\\\\\\/8RkWw8ikuMHThX5lPSHdXkjJ6d6fMKszF9Y4YbO67ZMTt7Z27d9UHfj22xzU7rSdraDcBWZ1jF1aKNJ9imHcwxhj5sHDZT92DhW0CtUeMkrvrAxNdO+9c19Pwqg5Fh8c1pobdPEvIQdyFdDYid2j4xztLFJ+9Am9s2UkmYB9UekfNrCgvrYLlboufFXDoi+CqicNrT6Ws2ofeLvj7ZKtnB31XReDJ5W8yGUc7TnbBi51qwaD2F+hlrB4aaAKSuVPZjBZmj0SvJd670EC1wGbO2s1LN704gcFTVO4oIerrxK6ZZzsxLspR3rpT7TBOR9kVjCihbdMvRoz5rNEpHaGXhOJ+J8uBYBbjhIrBN56CPqKtkmZCv5wYf9JHnl1yX1aRjNi1apLNxEvkKoJC1kKdViTTPldiJSEG6Gri26xSrmPfrpzEvoyW659zRI2QC21WGQ+6avh96c5fJbRcpspzA39+vANNAG7Ta1fDx9sURuTq6rhfsx5AZVHozkwNREW4oFJOJ0wubws9pCECz3KpSfcuhhRVRfP8jjipgoc4loe88XJPiFPj\\\\\\\/tYDWkCx3T\\\\\\\/2xjaxXxOoyj4FdBI9T5Z822c9Gynoj+5dFgd6Tz0epD8H00V3Q4cBaHjBHDpCbfluqg8Jk7ARlrrL5ACZtzo2j1mvphLOcUliQIR5l07Iz0Jh\\\\\\\/a4wTmX5ZLQG0IULYupPTZD5kzxRnnUiPpvj8HQLdi346rACnLkrgEkEe85GUrUE8FnWh4SbkUhUhTO8qw91u9m0XfM7461fArjmQlBlWEKHNC\\\\\\\/i44Da6l5l7gZ6LT\\\\\\\/PRLhz9Zb9a3eWYL7\\\\\\\/AmEY8uQy0iNq8OMDB8vl9DtUiDBrIxEAImEvVf3ZLIG3mCOeSUbnCLlyXiWYHS643jweP7cGVo\\\\\\\/drDWnkmIYRwyDs7UyImNf2oYcog==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AGY2AKBFpNWpIQTB7Zlf0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277445318ea075c41b541a8563d93627f9d3ce597f5dcbdc5ed05dcb6b037d310\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460047 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2073" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AEp3Pdjix5hW5iGKMqIWjSq6wSzWE-uwrkZ4e25YsDI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2073" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tWaEVCXHtFdeycY5FuL6DBl91a4AT4UMIDaZ3Oaj4OE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2061" title="Analysis of Circadian Rhythms in the Basal Filamentous Ascomycete Pyronema confluens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2085" title="Using Targeted Resequencing for Identification of Candidate Genes and SNPs for a QTL Affecting the pH Value of Chicken Meat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460047" id="node460047" data-pisa="ggg;5/10/2073" data-pisa-master="ggg;g3.115.021121" data-apath="/ggg/5/10/2073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ensemble Learning of QTL Models Improves Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yang Bian</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2073-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460047" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yang Bian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BBian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bian%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYang%2BBian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>†</span>U.S. Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James B. Holland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2073" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460047" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460047/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2073.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460047" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460047/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2073.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460047" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460047/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2073.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460047" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460047/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2073.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460047" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460047/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Quantitative trait locus (QTL) models can provide useful insights into trait genetic architecture because of their straightforward interpretability but are less useful for genetic prediction because of the difficulty in including the effects of numerous small effect loci without overfitting. Tight linkage between markers introduces near collinearity among marker genotypes, complicating the detection of QTL and estimation of QTL effects in linkage mapping, and this problem is exacerbated by very high density linkage maps. Here we developed a thinning and aggregating (TAGGING) method as a new ensemble learning approach to QTL mapping. TAGGING reduces collinearity problems by thinning dense linkage maps, maintains aspects of marker selection that characterize standard QTL mapping, and by ensembling, incorporates information from many more markers-trait associations than traditional QTL mapping. The objective of TAGGING was to improve prediction power compared with QTL mapping while also providing more specific insights into genetic architecture than genome-wide prediction models. TAGGING was compared with standard QTL mapping using cross validation of empirical data from the maize (<em>Zea mays</em> L.) nested association mapping population. TAGGING-assisted QTL mapping substantially improved prediction ability for both biparental and multifamily populations by reducing both the variance and bias in prediction. Furthermore, an ensemble model combining predictions from TAGGING-assisted QTL and infinitesimal models improved prediction abilities over the component models, indicating some complementarity between model assumptions and suggesting that some trait genetic architectures involve a mixture of a few major QTL and polygenic effects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/thinning-and-aggregating" class="hw-term hw-article-keyword hw-article-keyword-thinning-and-aggregating" rel="nofollow">thinning and aggregating</a></li><li class="kwd"><a href="/keyword/ensemble-modeling" class="hw-term hw-article-keyword hw-article-keyword-ensemble-modeling" rel="nofollow">ensemble modeling</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/zea-mays" class="hw-term hw-article-keyword hw-article-keyword-zea-mays" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Zea mays</em></a></li></ul><p id="p-3">Massive numbers of molecular markers can now be readily provided by next-generation sequencing and high-throughput genotyping (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bian <em>et al.</em> 2014a</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Glaubitz <em>et al.</em> 2014</a>). High-density marker maps (with marker intervals smaller than 1 cM in mapping populations) have been available to an increasing number of plant species, such as in <em>Arabidopsis thaliana</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang <em>et al.</em> 2011</a>), maize (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Swarts <em>et al.</em> 2014</a>), rice (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yu <em>et al.</em> 2011</a>), and sorghum (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zou <em>et al.</em> 2012</a>). Unfortunately, the dramatic increase of marker availability may adversely affect quantitative trait locus (QTL) mapping because collinearity among markers may complicate the detection and estimation of QTL. Statistical tests to declare a QTL or make QTL inference are conditional on other QTL in multi-QTL models (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zeng 1994</a>). The value of a test statistic depends on other QTL in the model if they share some proportion of genetic variation. Strong covariance between tightly linked markers may increase the risk of selecting collinear markers, bias the QTL estimates, and possibly overfit the predictive model, especially when a relaxed selection threshold is applied (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bian <em>et al.</em> 2014b</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ogut <em>et al.</em> 2015</a>).</p><p id="p-4">Ensemble learning is an alternative approach that could improve QTL-based prediction ability. Ensemble learning involves estimating multiple “learner” models on a training data set and predicting unseen observations by a vote or weighted average among the multiple learners (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dietterich 2000</a>). For example, suppose one has available a vector of some input variables, <strong>x</strong>, associated with a numerical outcome vector <strong>y</strong>. The estimation process involves finding a function of <em>F(x)</em> that maps values in the space of <strong>x</strong> to values in the corresponding space of <strong>y</strong>. <em>F(x)</em> can be estimated in various ways, one of which is by ensemble learning. The generic ensemble estimator takes the form: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-1.gif"/></noscript></span></span>, where <em>S</em> is the number of ensemble members (or “base learners”), the base learners <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-2.gif"/></noscript></span></span>are functions of x derived from the training data, and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-3.gif"/></noscript></span></span> is an ensemble learning function. As one example, the ensemble learner can be estimated simply by model averaging the base learners, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-4.gif"/></noscript></span></span>. The base learners can be trained based on random samples from the original data set or a subset of <strong>x</strong>. Common ensemble methods include bootstrap aggregation of predictions (bagging) and random forests, and several features characterize them, such as loss function, ensemble dispersions, and memory of baser learners (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hastie <em>et al.</em> 2009</a>). Bagging of regression models is trained by paralleling base regressions with no influence (zero memory) among them and minimizing squared-error loss function on bootstrap samples (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breiman 1996a</a>). If the bootstrap estimation is roughly correct, then aggregating would reduce variance without increasing bias. Random forests (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Breiman 2001</a>) increase ensemble dispersion over bagging by additionally using a randomly chosen subset of the predictors rather than using all of them, and this method typically solves for paralleled decision or regression trees (nonlinear base learners). The ensembles of base learners produced by bagging and random forests can be conducted in a parallel manner, meaning that each individual learner is trained independent of the results from others.</p><p id="p-5">Here, we developed a new ensemble learning approach used in QTL analyses, referred to as a thinning and aggregating (TAGGING) method. We thinned linkage map marker sets into submaps with equally reduced intermarker density, built QTL mapping models upon the thinned marker sets as based learners in parallel, and aggregated by averaging the predictions from the base learners to predict the test data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The TAGGING method shares some aspects of model averaging with bagging and random forest, as they all generate multiple predicted values and aggregate prediction by averaging. In contrast to random forests and bagging, however, TAGGING uses stratified sampling of the linkage map to create disjoint marker sets to generate independent discrete QTL models upon the unchanged set of observed mapping lines so that prediction variance is expected to decay faster and bias is not affected by bootstrapping samples. By comparison, bagging uses the original marker set to build prediction models on bootstrapped samples of lines, and random forests uses randomly sampled linkage markers and bootstrapped samples of lines.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data sampling, map thinning, model building, and ensemble schemes. The cross validations were repeated 10 times for Figure 2, Figure 3, and Figure 4 and 50 times for Figure 5. Step 1: Take a stratified random sample 80% of recombinant inbred lines of each nested association mapping family to use as a training set and 20% as test set. Step 2: Thin the original 0.2-cM resolution map into s sets of reduced maps with intermarker distance 0.2*s cM, and calibrate joint-family (JF) and single-family (SF) models as base learners on the same training sets with various combinations of reduced maps and p thresholds. Step 3: Independently predict the same test sets with the parameter estimates obtaining from training data. Step 4: Generate ensemble predictions for ensemble single-family (ESF) and ensemble joint-family (EJF) models by taking arithmetic means. Step 5: Generate ensemble learning and prediction of QTL-based and GP models. Step 6: Evaluate prediction R2 for all models, including individual quantitative trait locus (QTL) models, thinning and aggregating (TAGGING)-assisted QTL models, ensembles of QTL and genomic best linear unbiased prediction models, as well as subbagging models (see text for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572960335" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Data sampling, map thinning, model building, and ensemble schemes. The cross validations were repeated 10 times for Figure 2, Figure 3, and Figure 4 and 50 times for Figure 5. Step 1: Take a stratified random sample 80% of recombinant inbred lines of each nested association mapping family to use as a training set and 20% as test set. Step 2: Thin the original 0.2-cM resolution map into &lt;em&gt;s&lt;/em&gt; sets of reduced maps with intermarker distance 0.2*&lt;em&gt;s&lt;/em&gt; cM, and calibrate joint-family (JF) and single-family (SF) models as base learners on the same training sets with various combinations of reduced maps and &lt;em&gt;p&lt;/em&gt; thresholds. Step 3: Independently predict the same test sets with the parameter estimates obtaining from training data. Step 4: Generate ensemble predictions for ensemble single-family (ESF) and ensemble joint-family (EJF) models by taking arithmetic means. Step 5: Generate ensemble learning and prediction of QTL-based and GP models. Step 6: Evaluate prediction &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; for all models, including individual quantitative trait locus (QTL) models, thinning and aggregating (TAGGING)-assisted QTL models, ensembles of QTL and genomic best linear unbiased prediction models, as well as subbagging models (see text for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2073/F1.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2073/F1.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466812" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Data sampling, map thinning, model building, and ensemble schemes. The cross validations were repeated 10 times for <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and 50 times for <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Step 1: Take a stratified random sample 80% of recombinant inbred lines of each nested association mapping family to use as a training set and 20% as test set. Step 2: Thin the original 0.2-cM resolution map into <em>s</em> sets of reduced maps with intermarker distance 0.2*<em>s</em> cM, and calibrate joint-family (JF) and single-family (SF) models as base learners on the same training sets with various combinations of reduced maps and <em>p</em> thresholds. Step 3: Independently predict the same test sets with the parameter estimates obtaining from training data. Step 4: Generate ensemble predictions for ensemble single-family (ESF) and ensemble joint-family (EJF) models by taking arithmetic means. Step 5: Generate ensemble learning and prediction of QTL-based and GP models. Step 6: Evaluate prediction <em>R<sup>2</sup></em> for all models, including individual quantitative trait locus (QTL) models, thinning and aggregating (TAGGING)-assisted QTL models, ensembles of QTL and genomic best linear unbiased prediction models, as well as subbagging models (see text for details).</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">QTL mapping has proven useful for detecting major QTL with relatively large effects but may lack power in accurately modeling small QTL effects or polygenic background effects (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em> 2009</a>). QTL models typically underestimate the number and overestimate the effects of QTL, reducing the accuracy of QTL-based predictions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beavis 1998</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schön <em>et al.</em> 2004</a>). Predictability and interpretability are two competing goals and usually compromise each other for predictive machines. The parameters in some genomic prediction (GP) models are not easily interpretable in terms of genetic theory, and this problem is compounded in nonparametric models (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola and Van Kaam 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2010</a>). Such models may serve as good prediction machines but may provide no insights into the locations of important QTL or their gene actions. The TAGGING method presented here is a compromise method that aims to improve the prediction ability of QTL models while maintaining some of their interpretability in terms of QTL locations and effect sizes. Furthermore, the ensembling concept is fully general, so predictions can be obtained by ensembles of models that capture distinct components of the true genetic architecture, for example, ensembles of QTL models that are best at identifying genes with larger effects and polygenic models that capture the “background” effects of many genes of small effect distributed throughout the genome. Models that can account for two hypotheses may better approximate the true biological mechanism better than either of the two models individually.</p><p id="p-8">In this study, two QTL mapping models, joint-family (JF) linkage analysis for multifamily mapping populations and single-family (SF) QTL analysis for biparental mapping populations (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Ogut <em>et al.</em> 2015</a>), were tested within the TAGGING framework for maize complex traits. The new ensemble QTL models were examined for the number of QTL detected and proportion of genotypic variance explained by the detected QTL. We expect that for a high-bias base learner (<em>e.g.</em>, QTL models based on sparse linkage maps), subset aggregation ensures model flexibility and therefore protects against high biasness in the ensemble predictions. For high-variance base learners (<em>e.g.</em>, models in which QTL are included at low selection stringency with dense maps), ensembling might provide a reduction in variance of predictions.</p><p id="p-9">The objectives of this study were to: (1) develop a thinning and aggregating (TAGGING) method as a new ensemble learning approach for QTL analysis; (2) compare prediction abilities of TAGGING, QTL models, and standard GP genomic best linear unbiased prediction (GBLUP) models; and (3) test whether useful complementarity occurs between QTL-based and polygenic models in their ensemble learning for GP. We tested these models using a cross-validation strategy on data for three complex traits previously reported for the maize nested association mapping population (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McMullen <em>et al.</em> 2009</a>). At all QTL selection stringency thresholds and thinning intensities we tested, the TAGGING strategy resulted in a better prediction result than conventional QTL methods, implying the robustness of this new method. Results indicate that TAGGING provides information on the position and effect sizes of QTL (including their precision) while improving prediction ability compared with traditional QTL mapping procedures. With very high density linkage maps becoming increasingly available, we expect the TAGGING method will find use in genetic investigations and genetic prediction.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant phenotypes and genotypes</h3><p id="p-10">The maize nested association mapping (NAM) population comprises a set of ∼5000 recombinant inbred lines (RILs) derived from crosses between a reference parent, inbred line B73, and 25 other diverse founder inbred lines of maize (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">McMullen <em>et al.</em> 2009</a>). Three complex traits, resistance to southern leaf blight (SLB), plant height (PHT), and days to anthesis (DA), were previously studied for genetic architectures, and the predicted mean values for NAM RILs across multiple environments were previously reported (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yang <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bian <em>et al.</em> 2014b</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peiffer <em>et al.</em> 2014</a>) (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/021121SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/7dde4a3cedc611e4ac5606ec4bbcf141">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/687458c6edc611e4ae3406ec4bbcf141">File S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/b5e82c54edc611e4ae3406ec4bbcf141">File S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/93761014edc611e4b69206ec4bbcf141">File S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/c367d258edc611e496ee06ec4bbcf141">File S5</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d67633eeedc611e49c4306ec4bbcf141">File S6</a>). A second-generation NAM linkage map consisting of 7386 imputed pseudomarkers with a uniform 0.2-cM intermarker distance generated from genotyping-by-sequencing followed by full-sib family haplotype imputation (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Swarts <em>et al.</em> 2014</a>) was used for linkage analysis.</p><p id="p-11">Realized genomic relationship matrices (<em>G</em> matrices) were developed for each NAM family separately using the linkage map markers based on the first method described in (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden 2008</a>), which track within-family identity-by-descent (IBD) for genome segments (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The linkage map marker scores do not reflect IBD between families, however, so a separate relationship matrix for the whole NAM panel developed from the maize HapMap v1 of 1.6 million single-nucleotide polymorphisms (SNPs) (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Peiffer <em>et al.</em> 2014</a>) also was used here (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d6069ea4edc511e49c4306ec4bbcf141">File S7</a>). This <em>G</em> matrix measured the pairwise relationships among all lines with reference to the hypothetical population that would result from intermating all of the mapping population F1’s. To maintain identical numbers of markers for both the within-family relationship matrices and the matrix for the whole panel, we extracted 7386 HapMap v2 markers polymorphic in NAM and closest to the 7386 linkage marker in terms of the AGP v2 coordinates (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/cc9cdd00edc611e4ae3406ec4bbcf141">File S8</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/9bc11e3aedc611e4ac5606ec4bbcf141">File S9</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/71f708b2edc611e4994406ec4bbcf141">File S10</a>) to construct an identity-in-state (IIS) <em>G</em> matrix for the whole NAM panel. A total of 4354, 4359, and 4359 RILs with both genotypic and phenotypic data were available for analyzing SLB, PHT, and DA, respectively.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466833/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466833/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of prediction models compared</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Prediction accuracy calculation</h3><p id="p-13">Prediction abilities of models were evaluated both for the whole NAM panel and for within each of the 25 biparental mapping population using a cross-validation procedure. Training sets were created by randomly sampling 80% of RILs from each of the 25 NAM families. The remaining lines constituted the validation sample for the SF and JF QTL models created using the training data set. Within-family predictions were evaluated, and prediction abilities were measured as the proportion of total trait variance explained by the model (<em>R</em><sup>2</sup>) in the validation set. The <em>R</em><sup>2</sup> was used to enable direct comparison to heritabilities and was estimated by averaging squared Pearson’s correlations (<em>r<sup>2</sup></em>) between the observed and predicted BLUP values. We converted the value of <em>R<sup>2</sup></em> to have a negative sign for a few cases of negative correlations between predicted and observed line values.</p></div><div id="sec-4" class="subsection"><h3>QTL models</h3><p id="p-14">Multiple linear regression models were first fit within each family independently (SF models; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). For a given linkage maps <em>s</em>, the SF model for family <em>f</em> is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-5.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-6.gif"/></noscript></span></span> is a vector of <em>N<sup>f</sup></em> length referring to the trait phenotypic values in a given family, <strong><em>1</em></strong> is an <em>N<sup>f</sup></em> × 1 intercept vector of 1’s, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-7.gif"/></noscript></span></span> is the intercept, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-8.gif"/></noscript></span></span> is a <em>N<sup>f</sup></em> × <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-9.gif"/></noscript></span></span> matrix of marker genotypes, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-10.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-11.gif"/></noscript></span></span> × 1 column vector of the additive effects relative to B73, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-12.gif"/></noscript></span></span> is the number of significant loci in stepwise selection, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-13.gif"/></noscript></span></span> is a <em>N<sup>f</sup></em> × 1 column vector of errors. The stepwise incorporation and exclusion of markers were based on predefined <em>alpha</em> threshold (<em>p</em>). After including as many markers as possible based on their <em>P</em>-values, the model was then reduced by split-sample cross-validation. Specifically, the model selection step with minimum prediction error variance was chosen in five-fold split-sample cross-validation within the training set, using the ‘choose = CV cvMethod = split(5)’ option in SAS Proc GLMSelect (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">SAS Institute 2013</a>). The RIL phenotype values were predicted from the linear model.</p><p id="p-15">JF linkage models were trained in all 25 families of NAM populations. For a given linkage maps <em>s</em>, JF analysis for the multifamily connected populations can be described as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-14.gif"/></noscript></span></span>where <strong><em>Y</em></strong> is a vector of <em>N</em> length referring to the trait phenotypic values for all RILs, <strong>A</strong> is an <em>N</em> × <em>P</em> incidence matrix relating RILs to their corresponding family, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-15.gif"/></noscript></span></span> is a <em>P</em> × 1 column vector for the family effects, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-16.gif"/></noscript></span></span> is a <em>N</em> × <em>P</em> matrix relating each RIL’s genotype at locus <em>i</em> to its corresponding family-specific allele effect, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-17.gif"/></noscript></span></span> is a <em>P</em> × 1 column vector of the family-specific additive effects associated with locus <em>i</em>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-18.gif"/></noscript></span></span> is the number of significant loci in stepwise selection based on pre-defined <em>p</em>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-19.gif"/></noscript></span></span> is a <em>N</em> × 1 column vector of errors. The prediction ability was then evaluated for each family separately to make a comparable comparison with biparental scenarios.</p></div><div id="sec-5" class="subsection"><h3>TAGGING method for QTL analyses</h3><p id="p-16">The TAGGING QTL analysis method can be summarized as thinning of dense marker maps into a set of disjoint maps of lower density and then aggregating predictions from paralleled QTL models on the same training data sets (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The method begins by conducting a stratified sampling of the markers on the linkage map, which is simplified in the case of the maize NAM linkage map which has a uniform density of 0.2 cM between each pair of adjacent linked markers. The original map is thinned into <em>s</em> disjoint sets of markers, maintaining the linkage map position information for the markers, starting with the first marker on the first linkage group. This procedure was then repeated by starting selection at the second marker to create a new sample map and continuing to initiate selections at subsequent markers until the <em>s</em><sup>th</sup> marker. The result is <em>s</em> disjoint maps, each with <em>s</em>*0.2 cM distance between adjacent markers. Thinning is expected to reduce the extent of covariance and collinearity that otherwise occur in base QTL analyses with the dense maps and/or relaxed selection stringencies.</p><p id="p-17">JF and SF models for multifamily and biparental mapping populations were fit using each thinned map separately, and phenotype values for the validation set lines were predicted separately for each reduced map (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/44c43d7eedc611e49c4306ec4bbcf141">File S11</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/5dbb14a6edc611e49c4306ec4bbcf141">File S12</a>). The two types of QTL base learners were constructed with each subset of markers for the same training samples to estimate the regression coefficients, and then the <em>s</em> sets of SF and JF predicted values for the test sets were aggregated to form the ensemble prediction (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The ensembles of QTL models proposed here were named “ensemble joint-family linkage” (EJF) analyses for multifamily mapping populations and “ensemble single-family” (ESF) analyses for biparental mapping populations, respectively. The ensemble learner <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-20.gif"/></noscript></span></span> is formed as some linear combination of predictions of each base learner: <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-21.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-22.gif"/></noscript></span></span> being the coefficients for <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-23.gif"/></noscript></span></span>. For both EJF and ESF modeling, the coefficients were set to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-25.gif"/></noscript></span></span>. Essentially, we averaged predictions over <em>s</em> map subsets for each training set, and the ensemble predictions were an arithmetic mean of those base learners.</p><p id="p-18">To estimate precision of QTL localization, the frequencies of QTL positions detected in JF and EJF across the resampled training sets were summarized to elucidate the QTL architectures for the three complex traits. Resample model inclusion probability (RMIP) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Valdar <em>et al.</em> 2009</a>) was computed to measure the power of detection for the trait-marker associations across NAM panel. The RMIP was calculated for each marker as the proportion of data samples in which the marker was tested and selected in the model of interest at the given selection <em>p</em>.</p><p id="p-19">To compare the efficiency of the TAGGING ensemble learning method with existing ones, we implemented subsample aggregating (subagging) of both JF and SF analyses using the same base learning algothrims as in TAGGING. Subagging is a sobriquet for subsample aggregating where simple random sampling is used instead of aggregation of bootstrapped samples implemented by bagging. The basic difference between TAGGING and subagging is that TAGGING averages over subsamples of markers on a fixed set of RILs, while subagging (subsample aggregating) averages over samples of RILs for a fixed set of markers. Subagging instead of bagging was used here because subbaging is more efficient in computation, because of reduction in sample size compared with bagging, and it avoids collinearity problems induced or aggravated from the increased relatedness among bootstrapped samples, which is a concern with the dense linkage map used here. We formed 10 simple random sampling subsamples with sample size equal to 80% of each family from each original training set. We implemented subbagging of both SF and JF analyses upon the same training and test sets as used by all other analyses.</p><p id="p-20">SF and ESF models were constructed using selection thresholds ranging from <em>P</em> = 1e-4 to <em>P</em> = 0.05, and <em>P</em> = 1e-5 to 0.01 for JF and EJF. Map resolutions used ranged from 0.2 to 20 cM intermarker distances for both. Because <em>P</em> = 1e-05 was too stringent for SF, and <em>P</em> = 0.01 or greater were too relaxed for JF using 0.2-cM density map (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Ogut <em>et al.</em> 2015</a>), we did not include them in this study.</p></div><div id="sec-6" class="subsection"><h3>Prediction error decomposition</h3><p id="p-21">The cross-validation experiments allow us to evaluate the mean squared prediction error and to decompose this into terms due to the bias and variance in prediction. Suppose there is some underlying true function <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-26.gif"/></noscript></span></span> estimated from ensemble or discrete QTL models on the training set T by <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-27.gif"/></noscript></span></span>. Given a new data point in test set <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-28.gif"/></noscript></span></span> where <em>ε</em> is normally distributed with zero mean and variance <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-29.gif"/></noscript></span></span>, a function of 1 − heritability. The expected pointwise prediction error can be decomposed in a familiar form:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-31.gif"/></noscript></span></span> denotes expectation over resampled training sets drawn from the same distribution to estimate <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-32.gif"/></noscript></span></span>. Whereas the second part of the right hand side of (1) is prediction variance, the first part of the right hand side of (1) is the sum of bias<sup>2</sup> and irreducible error variance and decomposed as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-33.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>The TAGGING method estimated <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-34.gif"/></noscript></span></span> by averaging over <em>s</em> predictions from <em>s</em> sets of thinned marker maps based on a training set T used and estimated <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-35.gif"/></noscript></span></span>by averaging over all test sets that contain the data point in question. Finally, averaging over all points predicted throughout all test sets for their bias and variance gives the more accurate estimates. We refer to <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2073/embed/mml-math-36.gif"/></noscript></span></span> in (1) as “bias<sup>2</sup>” in our results, because the true underlying genetic mechanism is unknown and thus the irreducible error variance could not be disentangled from the bias<sup>2</sup> in (2). The bias<sup>2</sup> and variance in prediction were compared between single and TAGGING-based models all of which used the same 7386 linkage marker genotypes. In addition, a few predicted values representing the most severe outliers in prediction ability were excluded from prediction error calculation in order to guard against artifactual inflation of the prediction error variance simply due to high collinearity that occasionally occurred in few cross-validations. The cutoff for excluding an outlier was the mean predicted value ± 100 times the standard error of the mean.</p></div><div id="sec-7" class="subsection"><h3>GBLUP models using IIS and IBD genotypes</h3><p id="p-22">The <em>G</em> matrix derived from 1.6 million SNPs in HapMap v1 was used to create a GBLUP model for the entire NAM population HapMap v1–based genomic best linear unbiased prediction (HGBLUP) models, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) in the same cross validation setup as in QTL analyses. single-family genomic best linear unbiased prediction (SGBLUP; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) model were developed for each NAM biparental mapping family using the 25 disjoint <em>G</em> matrices developed from the original linkage map of 7386 markers. We also extracted the same number of allele calls as close as possible to the physical positions of the 7386 linkage map pseudomarkers, and with the IIS <em>G</em> matrix across whole NAM panel derived from that, we conducted the joint-family genomic best linear unbiased prediction (JGBLUP) models (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) to compare the efficiencies of using the same amount of genotypic information in different models.</p></div><div id="sec-8" class="subsection"><h3>Ensemble learning of TAGGING-assisted QTL models and GP models</h3><p id="p-23">To capture the complementary strengths of SF and JF analyses, we averaged predictions from EJF and ESF and referred to this ensemble as the EJF + ESF model. All pairs of QTL and GP models using the same number of 7386 loci were then combined in ensemble learning in a linear fashion. We explored ensemble learning by averaging predictions from either EJF, ESF, or EJF + ESF QTL models with predictions from SGBLUP or JGBLUP models to test the hypothesis that mixture of models with contrasting genetic assumptions can improve genetic prediction by better mimicking genetic architectures (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/2d33f2bcedc611e4ae3406ec4bbcf141">File S13</a>). For simplicity, the coefficients in ensembles of two component models were set to 0.5. Finally, we attempted ensemble learning that combined QTL and GBLUP models derived from both 7386 marker genotypes and 1.6 M HapMap1 SNP genotypes. We evaluated the importance of the four participating models to their ensemble prediction <em>R<sup>2</sup></em>, EJF + ESF, SGBLUP, JGBLUP, and HGBLUP, by analysis of variance in a 2<sup>4</sup> – 1 factorial experiment, in which 15 possible combinations of the four models’ presence or absence were included.</p></div><div id="sec-9" class="subsection"><h3>Pseudooptimal ensemble coefficients</h3><p id="p-24">We tested whether additional tweaking of the coefficients of the base leaners in the ensemble model can improve the ensemble prediction. The base learners can be considered as input variables to a multiple regression model aimed at predicting the unknown trait values. Here we introduced pseudooptimal coefficients for giving relative weights to each component model, which we estimated from using cross-validation with respect to the true phenotypic data of test sets to identify a fixed set of ensemble coefficients to minimize the least-square errors for the test data. The ensemble coefficients were tuned using the Nelder–Mead optimization technique (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nelder and Mead 1965</a>), in which average prediction <em>R<sup>2</sup></em> values were dynamically recorded for tested sets until they (as objective functions) converged to a plateau. The sum of coefficients was scaled to one, under nonnegativity constraints, as recommended by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breiman (1996b)</a>. The pseudooptimal accuracies should be considered as an idealistic upper limit and not likely to occur in a real prediction scenario, as those predictions were obtained using the phenotypic values in test sets to optimize the ensemble coefficients. Finally, two sets of natural coefficients were attempted in ensembles of the aforementioned three component models, EJF, ESF, and SGBLUP: equal weights of 1/3 for each, and equal weights between QTL-based models (0.25 for EJF and ESF) and SGBLUP models (0.5).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-25">Seeds of NAM lines have been deposited at USDA Maize Genetics Cooperation Stock Center (<a href="http://maizecoop.cropsci.uiuc.edu/nam-rils.php">http://maizecoop.cropsci.uiuc.edu/nam-rils.php</a>). <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/7dde4a3cedc611e4ac5606ec4bbcf141">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/687458c6edc611e4ae3406ec4bbcf141">File S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/b5e82c54edc611e4ae3406ec4bbcf141">File S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/93761014edc611e4b69206ec4bbcf141">File S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/c367d258edc611e496ee06ec4bbcf141">File S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d67633eeedc611e49c4306ec4bbcf141">File S6</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d6069ea4edc511e49c4306ec4bbcf141">File S7</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/cc9cdd00edc611e4ae3406ec4bbcf141">File S8</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/9bc11e3aedc611e4ac5606ec4bbcf141">File S9</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/71f708b2edc611e4994406ec4bbcf141">File S10</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/44c43d7eedc611e49c4306ec4bbcf141">File S11</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/5dbb14a6edc611e49c4306ec4bbcf141">File S12</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/2d33f2bcedc611e4ae3406ec4bbcf141">File S13</a> are available for download at <a href="http://downloads.figshare.com/article/public/1517823">http://downloads.figshare.com/article/public/1517823</a>.</p><p id="p-26"><a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/7dde4a3cedc611e4ac5606ec4bbcf141">File S1</a> (NAMSLB.RData) contains mean values for Southern leaf blight disease scores and linkage map marker scores for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/687458c6edc611e4ae3406ec4bbcf141">File S2</a> (SLB_genopheno.sas7bdat) contains mean values for Southern leaf blight disease scores and linkage map marker scores for NAM RILs formatted as a SAS data set. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/b5e82c54edc611e4ae3406ec4bbcf141">File S3</a> (NAMPHT.RData) contains mean values for plant height and linkage map marker scores for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/93761014edc611e4b69206ec4bbcf141">File S4</a> (PHT_genopheno.sas7bdat) contains mean values for plant height and linkage map marker scores for NAM RILs formatted as a SAS data set. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/c367d258edc611e496ee06ec4bbcf141">File S5</a> (NAMDA.RData) contains mean values for days to anthesis and linkage map marker scores for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d67633eeedc611e49c4306ec4bbcf141">File S6</a> (DA_genopheno.sas7bdat) contains mean values for days to anthesis and linkage map marker scores for NAM RILs formatted as a SAS data set. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/d6069ea4edc511e49c4306ec4bbcf141">File S7</a> (relMat.RData) contains the realized additive genomic relationship matrix for NAM lines based on 1.6M HapMap I markers (courtesy of Dr. Jason Peiffer). <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/cc9cdd00edc611e4ae3406ec4bbcf141">File S8</a> (NAMSLB_IIS.RData) contains mean values for Southern leaf blight disease scores and Identity In State (IIS) calls for HapMap markers closest to NAM linkage map markers for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/9bc11e3aedc611e4ac5606ec4bbcf141">File S9</a> (NAMPHT_IIS.RData) contains mean values for plant height and Identity In State (IIS) calls for HapMap markers closest to NAM linkage map markers for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/71f708b2edc611e4994406ec4bbcf141">File S10</a> (NAMDA_IIS.RData) contains mean values for days to anthesis and Identity In State (IIS) calls for HapMap markers closest to NAM linkage map markers for NAM RILs formatted as an R data object. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/44c43d7eedc611e49c4306ec4bbcf141">File S11</a> (ESF.sas) contains SAS code to conduct ensemble single family QTL analysis. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/5dbb14a6edc611e49c4306ec4bbcf141">File S12</a> (JSF.sas) contains SAS code to conduct ensemble joint family QTL analysis. <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/figshare.com/s/2d33f2bcedc611e4ae3406ec4bbcf141">File S13</a> (GBLUP and ensemble model prediction SLB.R) contains R code to ensemble GBLUP and ensemble-QTL-based predictions.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>TAGGING of QTL models improves QTL model prediction</h3><p id="p-27">EJF models had substantially better prediction <em>R</em><sup>2</sup> compared with the individual JF models using either the original or reduced maps for the three traits (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>). Using the thinned maps of 10 ∼15 cM intermarker distances at selection <em>P</em> = 0.01 generated the best prediction <em>R<sup>2</sup></em> in EJF for SLB (<em>R</em><sup>2</sup> = 0.475), PHT (0.476), and DA (0.445); those best EJF models outperformed the best JF model substantially [1 cM maps under <em>P</em> = 0.0001 for SLB (<em>R<sup>2</sup></em> = 0.385) and PHT (0.371), and 1 cM maps under <em>P</em> = 0.001 for DA (0.332)]. The improved prediction abilities of EJF over JF indicated that the genetic architecture can be better depicted by averaging multiple independent QTL models developed from disjoint subset maps as achieved by TAGGING. TAGGING of SF analyses also improved the predictions for the biparental mapping populations substantially compared with the regular SF analyses. The prediction <em>R</em><sup>2</sup> values generated by the optimal ESF models outperformed those from the best SF models by an average of 0.118, 0.123, and 0.116 across 25 families for SLB, PHT and DA, respectively (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>). Among the pure linkage models (SF, JF, ESF, and EJF) and across all traits, map densities, and selection <em>p</em> thresholds tested, EJF models had the greatest prediction abilities, whereas SF models always had lowest prediction ability (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction R2 for resistance to southern leaf blight in biparental and multiple-family prediction, comparing joint-family (JF), single-family (SF), and the thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) analyses using multiple map densities and under multiple selection p. The number at the top of and dot within each boxplot present the mean R2 among 25 nested association mapping families for that boxplot. The x-axis represents the densities of linkage maps used for the JF, SF, and TAGGING-assisted QTL methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572960335" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; for resistance to southern leaf blight in biparental and multiple-family prediction, comparing joint-family (JF), single-family (SF), and the thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) analyses using multiple map densities and under multiple selection &lt;em&gt;p&lt;/em&gt;. The number at the top of and dot within each boxplot present the mean &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; among 25 nested association mapping families for that boxplot. The x-axis represents the densities of linkage maps used for the JF, SF, and TAGGING-assisted QTL methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2073/F2.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2073/F2.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466765" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Prediction <em>R<sup>2</sup></em> for resistance to southern leaf blight in biparental and multiple-family prediction, comparing joint-family (JF), single-family (SF), and the thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) analyses using multiple map densities and under multiple selection <em>p</em>. The number at the top of and dot within each boxplot present the mean <em>R<sup>2</sup></em> among 25 nested association mapping families for that boxplot. The x-axis represents the densities of linkage maps used for the JF, SF, and TAGGING-assisted QTL methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">The performances of both discrete and TAGGING QTL models varied with the densities of maps and selection <em>p</em>. In general, for EJF, sparse maps and relaxed <em>p</em> were favorable in prediction, and for JF, dense maps and stringent <em>p</em> were favorable. Prediction abilities from EJF increased with decreasing <em>p</em> thresholds and decreasing map densities until the map density reduced down to 15−20 cM intermarker distances (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>). In contrast, for biparental mapping populations, moderate map densities along with relaxed <em>p</em> were advantageous in prediction using both SF and ESF analyses. In addition, using linkage maps of 1 cM or 0.2 cM density made little difference in prediction abilities for both JF and SF QTL models (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-30">The efficiency of TAGGING was compared with the previously proposed ensemble learning method subbagging, both of which were applied in the same inputs and cross validation schemes. The best prediction <em>R<sup>2</sup></em> values were 0.452, 0.440, and 0.405 using subbagging of JF models under selection <em>P</em> = 0.001 and 0.396, 0.378, and 0.391 using subbagging of SF models under selection <em>P</em> = 0.01 for SLB, PHT, and DA, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>). In all cases, the optimal prediction abilities of TAGGING-assisted QTL models were superior to the corresponding subbagging predictions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>).</p></div><div id="sec-13" class="subsection"><h3>Bias and variance</h3><p id="p-31">For all traits and under all selection <em>p</em> thresholds, TAGGING substantially reduced prediction variance in both JF and SF models (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS3.pdf">Figure S3</a>). Variance in prediction was basically eliminated when a large (≥25) number of thinned maps were averaged in TAGGING. TAGGING of JF models always reduced the bias compared with the single JF models, and the magnitude of bias in EJF models was roughly equal across a large range of thinning intensities, even when the map density was as low as 100 markers per map. The prediction bias in TAGGING of SF analyses can fluctuate compared with that of the individual SF models, depending on the thinned map densities (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS3.pdf">Figure S3</a>). When SF models suffered from severe collinearity at the relaxed selection stringency (<em>P</em> = 0.01), thinning reduced covariance among markers and alleviated model bias. When SF models were based on high stringency of marker selection (<em>P</em> = 1e-04), aggregating of the appropriately thinned maps (2 ∼5 cM inter-marker distances) seemed to slightly improve model fit by incorporating more predictors in the ensemble. In general, TAGGING seemed to be more protective against high bias for JF than SF models (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS3.pdf">Figure S3</a>). By examining the error compositions, we found that the prediction advantages of TAGGING of JF models over that of SF models were related to the lower level of bias before using TAGGING and more stable reduction of bias after using TAGGING (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS3.pdf">Figure S3</a>). In addition, when comparing across <em>P</em>-value thresholds, prediction bias in both TAGGING models decreased with more relaxed <em>P</em>-values because of greater model flexibility.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction bias2 and variance in thinning and aggregating (TAGGING)-assisted and single quantitative trait locus (QTL) models for resistance to southern leaf blight (SLB). X-axes denote the intermarker distances for the genetic maps used: &#x201C;0.2&#x201D; for single joint-family (JF) or single-family (SF) models, and others for TAGGING methods. Dot radius was scaled to within-family R2 calcualted based on all recombinant inbred lines in the test sets to permit comparisons of both mean error measures and mean prediction R2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572960335" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction bias&lt;sup&gt;2&lt;/sup&gt; and variance in thinning and aggregating (TAGGING)-assisted and single quantitative trait locus (QTL) models for resistance to southern leaf blight (SLB). X-axes denote the intermarker distances for the genetic maps used: &#x201C;0.2&#x201D; for single joint-family (JF) or single-family (SF) models, and others for TAGGING methods. Dot radius was scaled to within-family &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; calcualted based on all recombinant inbred lines in the test sets to permit comparisons of both mean error measures and mean prediction &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2073/F3.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2073/F3.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466745" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Prediction bias<sup>2</sup> and variance in thinning and aggregating (TAGGING)-assisted and single quantitative trait locus (QTL) models for resistance to southern leaf blight (SLB). X-axes denote the intermarker distances for the genetic maps used: “0.2” for single joint-family (JF) or single-family (SF) models, and others for TAGGING methods. Dot radius was scaled to within-family <em>R<sup>2</sup></em> calcualted based on all recombinant inbred lines in the test sets to permit comparisons of both mean error measures and mean prediction <em>R<sup>2</sup></em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genetic architecture revealed by marker-trait associations</h3><p id="p-33">To better understand the trait QTL architectures, the probability of inclusion of a marker in a JF or EJF models was estimated across the resampled training sets (resample model inclusion probability, RMIP; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Valdar <em>et al.</em> 2009</a>). RMIP plots visualized the enrichment of marker-trait associations within particular genomic regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS5.pdf">Figure S5</a>). In general, with the decrease in the thinned map densities used in EJF (which also reflects the increase in the number of JF models combined by TAGGING), RMIP values increased substantially and the regions containing marker associations expanded (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS5.pdf">Figure S5</a>). As the selection <em>p</em> relaxed, the RMIP values in EJF increased and the enriched regions expanded, especially for sparse maps (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS6.pdf">Figure S6</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker-trait associations identified by joint-family (JF) and ensemble joint-family (EJF) models across 10 chromosomes, using different map densities under selection p at 0.001 for maize SLB resistance in nested association mapping panel. Blue, cyan, and gray peaks denote associations with resample model inclusion probability (RMIP) values greater than 0.5, 0.3&#x2212;0.5, and less than 0.3, respectively. X-axes deonte the genetic positions (cM) across 10 chromosomes. RMIP values are summarized at the individual marker (every 0.2 cM) basis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572960335" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker-trait associations identified by joint-family (JF) and ensemble joint-family (EJF) models across 10 chromosomes, using different map densities under selection &lt;em&gt;p&lt;/em&gt; at 0.001 for maize SLB resistance in nested association mapping panel. Blue, cyan, and gray peaks denote associations with resample model inclusion probability (RMIP) values greater than 0.5, 0.3&#x2212;0.5, and less than 0.3, respectively. X-axes deonte the genetic positions (cM) across 10 chromosomes. RMIP values are summarized at the individual marker (every 0.2 cM) basis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2073/F4.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2073/F4.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466911" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Marker-trait associations identified by joint-family (JF) and ensemble joint-family (EJF) models across 10 chromosomes, using different map densities under selection <em>p</em> at 0.001 for maize SLB resistance in nested association mapping panel. Blue, cyan, and gray peaks denote associations with resample model inclusion probability (RMIP) values greater than 0.5, 0.3−0.5, and less than 0.3, respectively. X-axes deonte the genetic positions (cM) across 10 chromosomes. RMIP values are summarized at the individual marker (every 0.2 cM) basis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>GBLUP model prediction performance</h3><p id="p-35">The same marker data used for SF and ESF models also were used to construct within-family realized genomic relationship matrices. These relationship matrices were used to implement SF GP models (SGBLUP model) (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Average within-family prediction <em>R</em><sup>2</sup> values were 0.460, 0.465, and 0.450 for SLB, PHT, and DA, respectively, using the SGBLUP model. The JGBLUP model, which used an integrated relationship matrix calculated based on IIS information of the same number of marker genotypes that were closest to the linkage marker physical positions, generated best prediction results among EJF, ESF, SGBLUP, and JGBLUP models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS4.xlsx">Table S4</a>). Similarly, a previous report showed that joint analyses of several half-sib families in GP models can increase prediction abilities over family-specific GP models (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lehermeier <em>et al.</em> 2014</a>). In addition, the NAM panel was also analyzed using HGBLUP model for which the relationship matrix was calculated based on the 1.6 M HapMap v1 SNPs. HGBLUP generated the best prediction abilities of all component models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS4.xlsx">Table S4</a>) probably because of the enormous amount of genotype information used.</p></div><div id="sec-16" class="subsection"><h3>Ensemble of TAGGING-assisted QTL models and GBLUP models</h3><p id="p-36">The SF and JF QTL models were based on different genetic assumptions of genetic heterogeneity and allele effects, and may be considered complementary in describing QTL architectures (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Ogut <em>et al.</em> 2015</a>). To test the hypothesis that the combined QTL mapping results can improve prediction, we ensembled by model averaging to combine results from EJF and ESF. The EJF + ESF models did not noticeably improve prediction abilities over the EJF models (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>). The EJF + ESF results indicated that the selection <em>P</em> = 0.001 and 0.01 in EJF and ESF analyses resulted in the optimal prediction <em>R<sup>2</sup></em> when 1 cM reduced maps were used, and <em>P</em> = 0.01 and 0.05 for 20 cM reduced maps (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS2.xlsx">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS3.xlsx">Table S3</a>). We then restricted our modeling to using those <em>P</em> thresholds for the most thinned 20-cM maps in further ensemble learning.</p><p id="p-37">For ensemble learning between TAGGING-assisted QTL and GBLUP models, first, we attempted pairwise combinations of the three QTL models and two GBLUP models that used the same underlying 7386 marker genotypic information. Adding EJF or EJF + ESF models substantially increased within family prediction <em>R<sup>2</sup></em> and generated better prediction results for at least 21 (<em>P</em>-value &lt; 0.0005) out of 25 NAM families, compared with their ensemble partner SGBLUP or JGBLUP models (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS4.xlsx">Table S4</a>). Second, the best ensemble combinations of QTL and GBLUP models that used the same limited genotypic information improved prediction abilities, resulting in comparable performance relative to GBLUP models using much larger scale genotypic information. Specifically, (EJF + ESF) + JGBLUP models outperformed HGBLUP models (based on &gt;200 times more marker information) by 0.01 ∼0.02 in within-family prediction <em>R<sup>2</sup></em> for SLB and PHT (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS4.xlsx">Table S4</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ensemble between thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) and genomic best linear unbiased prediction (GBLUP) models showed their complementary effects on prediction ability. The mean within-family prediction R2 values for QTL and GBLUP models were shown in first row and column, and other cells show the R2 values for ensembles with equal weights for the two models in the corresponding rows and columns. The thinned maps of 20-cM intermarker distances were used for the QTL models, under the best selection stringency [P = 0.01 for ensemble joint family (EJF); 0.05 for ensemble single family (ESF)]. *P-value &lt; 0.0005 in the one-sided binomial tests with the null hypothesis that the ensemble model predicted the same as the corresponding row GBLUP model (&#x2265;21 of 25 families). The R2 values were estimated from 50 replicates of cross validation. Standard errors of prediction ability based on variation among families ranged from 0.016 to 0.018. JGBLUP, joint-family genomic best linear unbiased prediction; HGBLUP, HapMap v1-based genomic best linear unbiased prediction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572960335" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ensemble between thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) and genomic best linear unbiased prediction (GBLUP) models showed their complementary effects on prediction ability. The mean within-family prediction &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values for QTL and GBLUP models were shown in first row and column, and other cells show the &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values for ensembles with equal weights for the two models in the corresponding rows and columns. The thinned maps of 20-cM intermarker distances were used for the QTL models, under the best selection stringency [&lt;em&gt;P&lt;/em&gt; = 0.01 for ensemble joint family (EJF); 0.05 for ensemble single family (ESF)]. *&lt;em&gt;P&lt;/em&gt;-value &lt; 0.0005 in the one-sided binomial tests with the null hypothesis that the ensemble model predicted the same as the corresponding row GBLUP model (&#x2265;21 of 25 families). The &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values were estimated from 50 replicates of cross validation. Standard errors of prediction ability based on variation among families ranged from 0.016 to 0.018. JGBLUP, joint-family genomic best linear unbiased prediction; HGBLUP, HapMap v1-based genomic best linear unbiased prediction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2073/F5.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/10/2073/F5.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2073/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Ensemble between thinning and aggregating (TAGGING)-assisted quantitative trait locus (QTL) and genomic best linear unbiased prediction (GBLUP) models showed their complementary effects on prediction ability. The mean within-family prediction <em>R<sup>2</sup></em> values for QTL and GBLUP models were shown in first row and column, and other cells show the <em>R<sup>2</sup></em> values for ensembles with equal weights for the two models in the corresponding rows and columns. The thinned maps of 20-cM intermarker distances were used for the QTL models, under the best selection stringency [<em>P</em> = 0.01 for ensemble joint family (EJF); 0.05 for ensemble single family (ESF)]. *<em>P</em>-value &lt; 0.0005 in the one-sided binomial tests with the null hypothesis that the ensemble model predicted the same as the corresponding row GBLUP model (≥21 of 25 families). The <em>R<sup>2</sup></em> values were estimated from 50 replicates of cross validation. Standard errors of prediction ability based on variation among families ranged from 0.016 to 0.018. JGBLUP, joint-family genomic best linear unbiased prediction; HGBLUP, HapMap v1-based genomic best linear unbiased prediction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The high consistency of model prediction abilities between individual component models and their ensemble models suggested that the ensemble learning performance is likely predictable. Indeed, linear models involving terms of HGBLUP and JGBLUP or (EJF + ESF)*JGBLUP explained ∼72% to ∼84% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS5.xlsx">Table S5</a>) of the variance in within family prediction <em>R<sup>2</sup></em> among all combinations of the four tested component models (EJF + ESF, SGBLUP, JGBLUP, and HGBLUP). The (EJF + ESF) + HGBLUP + JGBLUP model (based on equal weights of 1/3 for the three components) resulted in the best prediction <em>R<sup>2</sup></em> (0.516 for SLB, 0.515 for PHT, and 0.512 for DA), which was 0.02 better for SLB and PHT and marginally better for DA than the HGBLUP model; it also predicted better in significantly more families than the HGBLUP model (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS4.xlsx">Table S4</a>). The magnitude of the increase in prediction <em>R<sup>2</sup></em> from ensembling TAGGING-assisted QTL models and GBLUP models was small but consistent across families. The traits studied are highly multigenic, so the results may be considered as a conservative case for evaluating the utility of the additional QTL information in GP models.</p></div><div id="sec-17" class="subsection"><h3>Pseudooptimal ensemble coefficients</h3><p id="p-40">We further searched for a fixed set of optimal coefficients that can maximize the accuracy from the EJF, ESF, and SGBLUP predictions. The prediction accuracy reached plateau after several rounds of Nelder–Mead optimizations. The resulting average within family prediction <em>R<sup>2</sup></em> based on the “pseudooptimal” ensemble coefficients differed by less than 0.01 from the two sets of simple coefficients (1/4)EJF + (1/4)ESF + (1/2)GBLUP and (1/3)EJF + (1/3)ESF + (1/3)GBLUP we developed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1/TableS6.xlsx">Table S6</a>). The results implied that the current ensemble learning to combine QTL-based and GP models was efficient in capitalizing on the model complementarities.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Optimizing QTL mapping parameters</h3><p id="p-41">Determining the optimal prediction model for a collection of multiple biparental families, as is commonly encountered in both academic genetic studies and commercial breeding programs, is important for maximizing precision of QTL mapping and accuracy of genome-enabled prediction. Our results demonstrate that there is an interaction between map density and QTL significance threshold on model prediction performance. The optimal threshold for a given analysis can vary according to map density. In earlier studies of prediction accuracy within biparental families from QTL models, accuracy was optimal at quite relaxed <em>P</em>-value thresholds (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hospital <em>et al.</em> 1997</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 2004</a>). Those studies were conducted by the use of linkage maps with substantially lower marker density than the current study. As genomics technology has made increasingly dense linkage maps available, however, marker colinearity becomes problematic for QTL mapping and the relaxed threshold is no longer optimal. For example, when <em>P</em> = 0.05 were adopted in SF analysis at the greatest map density, average prediction ability was 0.21, and this improved to 0.31 by increasing the selection stringency to <em>P</em> = 0.01 (the first column of <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). Using appropriately chosen selection <em>p</em> reduced the propensity to overfitting or underfitting resulting from the improper marker densities. Nevertheless, an overly sparse map used in JF or SF models may fail to capture the QTL information or compromise the precision of QTL positioning because the markers are not in tight linkage to many QTL and therefore lead to omitted-variable bias (<a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS8.pdf">Figure S8</a>). Our JF and SF model results suggest that a 1-cM density is appropriate for the level of linkage disequilibrium (LD) that exists within a biparental RIL family in maize. By comparison, previous research reported that marker densities increasing from 5 to 1 cM in a doubled haploid maize population improved neither the overall QTL detection power nor the proportion of genotypic variance explained by the detected QTL (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stange <em>et al.</em> 2013</a>). Finally, our results confirmed that the optimal selection threshold also differs by QTL mapping design. SF models are more prone to underfitting (high bias) problems than JF models, and more stringent selection threshold should be used in multiparental analysis (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blanc <em>et al.</em> 2008</a>). To conclude, it is important to consider the dynamics among map density, QTL analysis method and QTL detection stringency, the optimization is important to gain the best performance in QTL analysis based prediction (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS2.pdf">Figure S2</a>). Performances of both traditional and TAGGING-assisted QTL models varied with the densities of maps and selection <em>p</em>. However, the prediction ability of TAGGING models was greater than the respective traditional QTL mapping methods for all selection <em>p</em> and thinning intensities tested.</p><p id="p-42">Bayesian GP models represent an important class of models to consider, but the computational difficulty of conducting on many repeated samples with the millions of markers used in this study (for the HGBLUP models) precluded their inclusion in the model comparisons in this study. In addition, Bayesian models have shown little or no advantage over GBLUP models in many plant breeding scenarios (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2012</a>).</p></div><div id="sec-20" class="subsection"><h3>Marker-trait association and genetic architecture</h3><p id="p-43">The EJF models may reflect the true genetic architecture better than discrete models concerning only a few strong associations. Indeed, the greater number of marker associations contributing to ensemble predictions is one of the bases for the improved prediction ability. By using the TAGGING method, association architectures in the EJF models were equipped with the polygenic feature that can include many more markers linked to QTL, while it also involved marker selection and differential weighting of marker information. The precision of QTL mapping, however, can be compromised in ensemble mapping. For example, previous data had indicated with high confidence the presence of two linked QTL on chromosome 3 for genetic resistance to SLB (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bian <em>et al.</em> 2014b</a>), but the blurring of QTL positions in EJF resulted in a merging of the two QTL into a single broad peak in the RMIP visualizations (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The larger number of marker effects that are ensembled in the TAGGING method may offer a useful compromise between QTL detection and predictions. We focused on cross-validation comparisons based on real data here, but simulation studies would be required to determine the accuracy of QTL positition and effect estimates from the TAGGING method.</p><p id="p-44"><a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Ogut <em>et al.</em> (2015)</a> suggested that JF models (which assume common QTL positions among families) and SF models (which assume independent QTL positions among families) can complement one another by capturing different aspects of the overall genetic architecture. We tested this hypothesis by ensembling predictions from the two QTL models in a single EJF + ESF model. The ESF + EJF model appeared to offer little advantage over EJF: the relatively poor predictive ability of ESF negated any advantage of complementarity between ESF and EJF models when they were ensembled. A single integrated base learner model that more flexibly fits allele effects only within families where they are significant may more effectively achieve the goal of taking advantages of both models in depiction of genetic architectures. Development of such a model is underway.</p><p id="p-45">Marker-trait associations are found highly enriched in some genomic regions, as indicated by the disjoint TAGGING scans. For a particular region of interest <em>a priori</em>, the RMIP information surrounding that region indicates its importance and the resolution of QTL information. The association-enriched regions might represent probable intervals of QTL effects, that is, QTL or the cluster of QTL may reside in the multi-locus regions or in linkage with the loci in the regions. Leveraging association-enriched regions may better explain the underlying genetic architectures compared to the traditional point estimates of single QTL peaks, although further research remains needed to effectively define and set boundaries for the regions or factor in their kernel density. We suspect that the association-enriched regions might be important to reveal hidden genetic variation. Further research will be required to develop this method and test the model efficiency.</p></div><div id="sec-21" class="subsection"><h3>Bias and variance in TAGGING prediction</h3><p id="p-46">The prediction ability of ensemble learning is usually stronger than that of a single learner. The first reason is that the training sample size might not be sufficient for determining a single best base model. In TAGGING, the reduced maps provide similar information, because they only differ by consecutive markers. The base learners should perform similarly well on searching those slightly different hypothesis spaces to fit the same training data sets. Thus, combining these learners can join the marker information that would otherwise not be obtained by searching in the original hypothesis space with a single algorithm (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rokach 2010</a>). The second reason is that the search processes of the learning algorithms might be imperfect, especially in the “small N, large P” situations that have become common in genomics. Even if there exists a unique best set of predictors (actual functions of genes underlying the studied traits), the high dimensionality of dense linkage maps may hinder this set from being selected by efficient search algorithms. Thus, ensembles can compensate for such imperfect search processes by reducing the hypothesis space. The third reason is that, the model specification (linear models here) being tested might not contain the true target function, but ensembles can nevertheless provide better approximations to the true function than a single base learner function can.</p><p id="p-47">The proposed TAGGING framework was successful at strengthening “weak learners” (two types of QTL models) by first reducing the hypothesis space and then aggregating by averaging the base models. A rule of thumb for optimizing TAGGING in QTL-based prediction is to conduct EJF based on heavily thinned maps (more sets to aggregate) under relaxed selection <em>p</em> thresholds. Thinning alleviates the collinearity within each marker set, and this allows the selection threshold to be relaxed without overfitting. In addition to thinning, aggregating across predictions from multiple models also decreases the prediction variance. For example, averaging <em>N</em> identical independent model predictions would reduce the resulting prediction variance by a factor of 1/<em>N</em>. In most ensemble learning, including TAGGING, reduction is obviously less than 1/<em>N</em> because of dependent base predictions. Thinning by stratified sampling of markers from the linkage map takes advantage of the consistent ‘spatial’ pattern of correlation among markers, such that the subset hypothesis spaces defined by thinning represented disjoint representations of the linkage map. By decomposing the prediction error variances, we showed that the contribution of variance among prediction sets was reduced to almost zero with large samples of sparser subset maps (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Finally, the TAGGING strategy tracks the linkage structure exactly in our case of a perfectly uniform marker density. In more typical QTL mapping situations in which the markers are not evenly spaced in the linkage map, it is important to stratify maps accounting for the original marker correlations, instead of just even sampling across chromosomes. Similarly, in genome-wide association studies for which the correlation structure of marker set is always not constant, the LD between marker genotypes does not decays monotonically as their physical distances extend. Special attention needs to be paid to appropriate marker stratification schemes for these more general situations of uneven marker spacing for QTL mapping and complex LD structure for association mapping when TAGGING.</p><p id="p-48">The minimum density required for TAGGING to work was around 20 cM in the thinned maps, and further thinning caused decreased prediction ability in TAGGING. Because dense maps are expected to be more easily available for many species, the applicability of TAGGING will only increase over time. Furthermore, most crop plants regularly deal with maps with density greater than one marker per 20 cM, so there is already general applicability at this time.</p><p id="p-49">In addition to reducing the variance by model averaging, another motivation of TAGGING was to alleviate bias in prediction. First, a large collection of disjoint predictors can be considered and weighted when ensembling them, and therefore more genotypic information (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/08/13/g3.115.021121.DC1/FigureS5.pdf">Figure S5</a>) contributes to the ensemble prediction, resulting in reduced bias in prediction. Moreover, when a few dominating predictors consistently perform better than their correlated competitors (for example, markers within the same LD block), they will tend to be selected in prediction models at the expense of those weaker competing predictors, some of which may provide information about local features of the data. TAGGING thins the map into equally spaced disjoint maps, providing more opportunity for predictors to be considered without competition from the dominant predictors, possibly increasing the chance that weak local features will be represented by some of the base learners.</p><p id="p-50">We expect that bootstrapping of data samples in addition to TAGGING to generate even greater numbers of base learners will not result in a substantial decrease in the prediction error, as the variance is already nearly eliminated by TAGGING, whereas bias may increase due to the use of smaller effective training samples for each base learner. As observed here (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a> and Figure S3) and found in many other ensemble studies, the dominating error source turns out to be the bias<sup>2</sup> and irreducible error (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bauer and Kohavi 1999</a>), which suggests necessity of bias reduction in ensemble learning. A better approach may be to implement TAGGING upon greater variable (lower bias) base learners. Moreover, bias corrected estimators (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Efron 1987</a>) can be further incorporated into TAGGING, which may help reduce the prediction bias due to finite-sample bias in base learner estimators (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Horowitz 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang and Lu 2012</a>). Similar to bagging and random forest, TAGGING applied a natural model averaging weight to combine base learners and did not require a tuning process. Another direction of future work could be related to exploring sophisticated ensemble learning algorithms. Regularized linear regression on the base learners can be easily implemented (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Friedman and Popescu 2008</a>). Alternatively, meta-learning strategies such as stack regressions (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Breiman 1996b</a>) are approaches based upon the parallel training of multiple learning programs, followed by a meta-learning stage to stack them in a principled fashion.</p><p id="p-51">The prediction ability of TAGGING models is not sufficient to outperform current standard GP methods, such as GBLUP. Our results suggest, however, that TAGGING can simultaneously match the prediction ability of the GBLUP model (with a bit of complementarity that can be exploited in additional ensembling) while also providing information on important genomic regions, which can be utilized in gene discovery. The improved prediction ability of the ensemble models over conventional QTL mapping imply they can better model the true QTL architecture, for example, by highlighting important genomic regions instead of relying on point estimates of QTL effects and by ameliorating collinearity in dense genetic maps.</p></div><div id="sec-22" class="subsection"><h3>Oligogenic and polygenic model complementation</h3><p id="p-52">The high heritabilities (more than 85%, line mean based) of the three traits implied a great proportion of trait variation should be attributable to differences between maize lines after accounting for known environmental effects (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Peiffer <em>et al.</em> 2014</a>). Traditional additive QTL models nevertheless provide only moderate prediction ability. This indicates that the genetic factors underlying these high heritable traits are complex or might not be approximated well without considering more complex model hypothesis. No strong specific digenic interaction was found in NAM populations for the studied traits, although it is still possible that polygenic additive by additive effects are important, even if we have not mapped specific interactions so far. Considering the epistasis may be one piece of the missing components, fitting feasible epistasis effects that can account for moderate or large effects seems practical in improving prediction accuracy, especially in our TAGGING framework where the hypothesis spaces can now be more easily searched. Previous studies showed that infinitesimal GP model (GBLUP or ridge regression models) outperformed QTL-based model in predicting complex traits for both multifamily populations (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Peiffer <em>et al.</em> 2014</a>) and biparental segregating populations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2012</a>). The opposite results were found for traits with major QTL (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 2014</a>). In a plant breeding application, linear combinations of different GP (including Bayesian) models did not result in noticeable gain in GP accuracy (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Heslot <em>et al.</em> 2012</a>). Our results showed that the ensemble learners among well-tuned TAGGING-assisted QTL models and GBLUP models that come from the same genotypic information improved prediction, which suggests their useful complementation in prediction of complex traits. Furthermore, the factorial experiment of combining varied model predictions suggest that aggregating models that use different genotypic information is advantageous in GP in the tested NAM populations. Leveraging the complementary effects among model assumptions and/or genetic information provides one more possible solution to achieve a better model specification to approach the ideal heritable variation.</p></div></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Dr. Jason Peiffer for suggesting use of the Nelder–Mead optimization technique and for providing us with HapMap v1-based relationship matrix for the NAM panel. We thank Dr. Howard Bondell for a helpful discussion about analyses on bias and variance. We used the High Performance Computing Services in NC State University and thank Dr. Gary Howell for assistance with parallel computing. We appreciate two anonymous reviewers for their thorough evaluations. Research was supported by United States National Science Foundation grants IOS-1238014 and IOS-0820619 and the US Department of Agriculture, Agricultural Research Service. Y.B. is a graduate student at NC State University and supported by a National Science Foundation grant IOS-1238014 and a Monsanto Co. fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-54">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021121/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-55"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-28"><span class="received-label">Received </span>April 28, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-09"><span class="accepted-label">Accepted </span>August 9, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Bian and Holland</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.1" data-doi="10.1023/A:1007515423169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohavi</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An empirical comparison of voting classification algorithms: bagging, boosting, and variants.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D36%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1007515423169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1007515423169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2073.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QTL analyses: power, precision, and accuracy</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">162</span>, <span class="cit-source">Molecular Dissection of Complex Traits</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">What proportion of declared QTL in plants are false?</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D424%26rft.atitle%253DWhat%2Bproportion%2Bof%2Bdeclared%2BQTL%2Bin%2Bplants%2Bare%2Bfalse%253F%26rft_id%253Dinfo%253Apmid%252F15085262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15085262&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223011800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.4" data-doi="10.1007/s11032-014-0066-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballington</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Patterns of simple sequence repeats in cultivated blueberries (Vaccinium section Cyanococcus spp.) and their use in revealing genetic diversity and population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D675%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0066-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0066-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.5" data-doi="10.1186/1471-2164-15-1068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Limits on the reproducibility of marker associations with southern leaf blight resistance in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1068%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1068%26rft_id%253Dinfo%253Apmid%252F25475173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25475173&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.6" data-doi="10.1007/s10681-007-9607-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veyrieras</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Marker-assisted selection efficiency in multiple connected populations: a simulation study based on the results of a QTL detection experiment in maize.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9607-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9607-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254877800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">a</span> <span class="cit-article-title">Bagging predictors.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D24%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1996UZ38000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">b</span> <span class="cit-article-title">Stacked regressions.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D24%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.9" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forest.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.10" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.11" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.12" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2073.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dietterich</span>  <span class="cit-name-given-names">T. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ensemble methods in machine learning</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span> in <span class="cit-source">Multiple Classifier Systems</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kittler</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Roli</span>  <span class="cit-name-given-names">F.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.14" data-doi="10.2307/2289144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Better bootstrap confidence intervals.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D82%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987G462600027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.15" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.16" data-doi="10.1214/07-AOAS148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popescu</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Predictive learning via rule ensembles.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D2%26rft.spage%253D916%26rft_id%253Dinfo%253Adoi%252F10.1214%252F07-AOAS148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/07-AOAS148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.17" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.18" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.19" data-doi="10.1007/s00122-011-1702-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishore</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genome-wide selection efficiency in maize nested association mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D275%26rft.atitle%253DEvaluation%2Bof%2Bgenome-wide%2Bselection%2Befficiency%2Bin%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1702-9%26rft_id%253Dinfo%253Apmid%252F21938474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1702-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2073.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">The Elements of Statistical Learning</span>. <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.21" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.22" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2073.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horowitz</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The bootstrap</span>, pp. <span class="cit-fpage">3159</span>–<span class="cit-lpage">3228</span>, in <span class="cit-source">Handbook of Econometrics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Heckman</span>  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Leamer</span>  <span class="cit-name-given-names">E.</span></span>. <span class="cit-publ-name">North Holland</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.24" data-doi="10.1007/s001220050679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacoudre</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">More on the efficiency of marker-assisted selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D1181%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000071281000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.25" data-doi="10.1073/pnas.1100465108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Effgen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keizer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of natural allelic variation in <em>Arabidopsis</em> using a multiparent recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">4488</span>–<span class="cit-lpage">4493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100465108%26rft_id%253Dinfo%253Apmid%252F21368205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzExLzQ0ODgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.26" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.27" data-doi="10.1038/ng.747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kump</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oropeza-Rosas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of quantitative resistance to Southern leaf blight in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKump%26rft.auinit1%253DK.%2BL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bquantitative%2Bresistance%2Bto%2Bsouthern%2Bleaf%2Bblight%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.747%26rft_id%253Dinfo%253Apmid%252F21217757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.28" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (Zea mays L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjA3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.29" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.30" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez Villeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.31" data-doi="10.1093/comjnl/7.4.308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelder</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mead</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">A simplex method for function minimization.</span> <abbr class="cit-jnl-abbrev">Comput. J.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BComputer%2BJournal%26rft.stitle%253DThe%2BComputer%2BJournal%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D308%26rft.epage%253D313%26rft.atitle%253DA%2BSimplex%2BMethod%2Bfor%2BFunction%2BMinimization%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcomjnl%252F7.4.308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/comjnl/7.4.308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1965CLA3200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.32" data-doi="10.1038/hdy.2014.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogut</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.123%26rft_id%253Dinfo%253Apmid%252F25585918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25585918&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.33" data-doi="10.1534/genetics.113.159152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic architecture of maize height.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159152%26rft_id%253Dinfo%253Apmid%252F24514905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.34" data-doi="10.1007/s10462-009-9124-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokach</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ensemble-based classifiers.</span> <abbr class="cit-jnl-abbrev">Artif. Intell. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArtif.%2BIntell.%2BRev.%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10462-009-9124-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10462-009-9124-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273807600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2073.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SAS Institute</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">SAS/STAT 13.1 User’s Guide</span>. <span class="cit-publ-name">SAS Institute Inc.</span>, <span class="cit-publ-loc">Cary, North Carolina</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.36" data-doi="10.1534/genetics.167.1.485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Openshaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait locus mapping based on resampling in a vast maize testcross experiment and its relevance to quantitative genetics for complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchon%26rft.auinit1%253DC.%2BC.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D485%26rft.epage%253D498%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BMapping%2BBased%2Bon%2BResampling%2Bin%2Ba%2BVast%2BMaize%2BTestcross%2BExperiment%2Band%2BIts%2BRelevance%2Bto%2BQuantitative%2BGenetics%2Bfor%2BComplex%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.485%26rft_id%253Dinfo%253Apmid%252F15166171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvNDg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.37" data-doi="10.1007/s00122-013-2155-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuerschum</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density genotyping: an overkill for QTL mapping? Lessons learned from a case study in maize and simulations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2563</span>–<span class="cit-lpage">2574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2563%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2155-0%26rft_id%253Dinfo%253Apmid%252F23860723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2155-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23860723&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.39" data-doi="10.1534/genetics.109.100727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping in structured populations by resample model averaging.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.100727%26rft_id%253Dinfo%253Apmid%252F19474203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.40" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.41" data-doi="10.1073/pnas.1310949110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CACTA-like transposable element in ZmCCT attenuated photoperiod sensitivity and accelerated the postdomestication spread of maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">16969</span>–<span class="cit-lpage">16974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310949110%26rft_id%253Dinfo%253Apmid%252F24089449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQyLzE2OTY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.42" data-doi="10.1371/journal.pone.0017595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e17595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17595%26rft.epage%253De17595%26rft.atitle%253DGains%2Bin%2BQTL%2Bdetection%2Busing%2Ban%2Bultra-high%2Bdensity%2BSNP%2Bmap%2Bbased%2Bon%2Bpopulation%2Bsequencing%2Brelative%2Bto%2Btraditional%2BRFLP%252FSSR%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017595%26rft_id%253Dinfo%253Apmid%252F21390234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21390234&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.44" data-doi="10.1080/02664763.2011.578621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bias-corrected random forests in regression.</span> <abbr class="cit-jnl-abbrev">J. Appl. Stat.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BStat.%26rft.volume%253D39%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1080%252F02664763.2011.578621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/02664763.2011.578621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.45" data-doi="10.1038/hdy.2014.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mette</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bridging the gap between marker-assisted and genomic selection of heading time and plant height in hybrid wheat.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">638</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D638%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.1%26rft_id%253Dinfo%253Apmid%252F24518889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24518889&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2073.46" data-doi="10.1093/jxb/ers205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of QTLs for eight agronomically important traits using an ultra-high-density map based on SNPs generated from high-throughput sequencing in sorghum under contrasting photoperiods.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">5451</span>–<span class="cit-lpage">5462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fers205%26rft_id%253Dinfo%253Apmid%252F22859680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2My8xNS81NDUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2073.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2061" title="Analysis of Circadian Rhythms in the Basal Filamentous Ascomycete Pyronema confluens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2085" title="Using Targeted Resequencing for Identification of Candidate Genes and SNPs for a QTL Affecting the pH Value of Chicken Meat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2073" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pA-W_ylcYdmooP747IpaZuXZqgyuKkIDDwEhSrtVc5o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2073" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2073" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2073" class="active" data-icon-position="" data-hide-link-title="0">Ensemble Learning of QTL Models Improves Prediction of Complex Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460047" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vElwlcYtEP1R8CPkBGw0IK_WG1HwAKq04Eu_xhv8b_k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2073.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2073" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hLPgs3Jaa-bJpoErTJW_Mp1moyv0tnSxnnbkT3GE2pM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2073" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2073.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460047" id="node460047--2" data-pisa="ggg;5/10/2073" data-pisa-master="ggg;g3.115.021121" data-apath="/ggg/5/10/2073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ensemble Learning of QTL Models Improves Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yang Bian</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2073-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460047--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yang Bian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BBian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bian%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYang%2BBian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>†</span>U.S. Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James B. Holland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2073" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2073&amp;text=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2073&amp;t=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2073&amp;title=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460047/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460047" id="node460047--3" data-pisa="ggg;5/10/2073" data-pisa-master="ggg;g3.115.021121" data-apath="/ggg/5/10/2073.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ensemble Learning of QTL Models Improves Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yang Bian</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2073-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021121 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460047--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yang Bian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BBian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bian%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYang%2BBian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>†</span>U.S. Department of Agriculture, Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4341-9675" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James B. Holland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2073&amp;rft.epage=2084&amp;rft.atitle=Ensemble%20Learning%20of%20QTL%20Models%20Improves%20Prediction%20of%20Complex%20Traits&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bian%2C+Yang&amp;rft.au=Holland%2C+James+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460047/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460047/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460047/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460047/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460047/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460047/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460047/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460047/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460047/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460047/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460047/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460047/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496049" id="node496049" data-pisa="ggg;g3.119.400658v1" data-pisa-master="ggg;g3.119.400658" data-apath="/ggg/early/2020/03/26/g3.119.400658.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.119.400658" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Dissecting Adaptive Traits with Nested Association Mapping: Genetic Architecture of Inflorescence Morphology in Sorghum</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2073" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2073.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2073.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2073.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2073.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
